<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Copyright &#195;&#8220; 2010 Genetics Society America <br /> DOI: 10.1534/genetics.110.121822 <br />  <br />  <br />  <br /> A Targeted UAS-RNAi Screen Drosophila Larvae Identifies Wound Closure <br />               Genes Regulating Distinct Cellular Processes <br />  <br />          Christine Lesch,* Juyeon Jo,* Yujane Wu,*,&#226;&#8364;&#160; Greg S. Fish&#226;&#8364;&#161; Michael J. Galko*,&#226;&#8364;&#160;,1 <br />           *Department Biochemistry Molecular Biology &#226;&#8364;&#160;Genes Development Graduate Program, University Texas <br />            Graduate School Biomedical Sciences, University Texas MD Anderson Cancer Center, Houston, Texas 77030 and <br />                     &#226;&#8364;&#161; <br />                       Department Biochemistry, Stanford University School Medicine, Stanford, California 94305 <br />                                                         Manuscript received August 4, 2010 <br />                                                       Accepted publication August 20, 2010 <br />  <br />  <br />                                                            <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> <br />                 Robust mechanisms tissue repair critical <span id='am-8' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-9' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span>survival</span> multicellular organisms. Efficient <br />              cutaneous wound repair requires migration cells wound edge farther the <br />              epidermal sheet, genes control coordinate migrations remain obscure. This in <br />              systematic screening approach vivo identification classification postembryonic <br />              wound closure genes developed. Here, performed proof-of-principle reporter-based <br />              vivo RNAi screen Drosophila melanogaster larval epidermis identify genes required normal <br />              wound closure. Among candidate genes tested kinases transcriptional mediators Jun <br />              N-terminal kinase ( JNK) signaling pathway shown required epithelial sheet migration during <br />              development. Also targeted genes involved actin cytoskeletal remodeling. Importantly, RNAi <br />              knockdown canonical noncanonical members JNK pathway caused open wounds, as <br />              did genes involved actin cytoskeletal remodeling. Our analysis JNK pathway components <br />              reveals redundancy upstream activating kinases distinct roles downstream <br />              transcription factors DJun DFos. Quantitative qualitative morphological classification open <br />              wound phenotypes evaluation JNK activation suggest multiple cellular processes required <br />              migrating epidermal cells, including functions specific cells wound edge others <br />              specific cells farther epidermal sheet. Together, results identify new set of <br />              conserved wound closure genes, determine putative functional roles genes migrating <br />              epidermal sheet, provide template broader vivo RNAi screen discover full <br />              complement genes required wound closure larval epidermal wound healing. <br />  <br />  <br />  <br />  <br /> C    ELL migration critical feature wound <br />       healing responses (Martin 1997; Singer and <br /> Clark 1999). During postembryonic cutaneous repair <br />                                                                             allowed high-throughput identification genes re- <br />                                                                             quired closure simple epithelial scratch wounds. <br />                                                                             Further, genetic screen genes required the <br /> in humans, rodents, Drosophila larvae, highly                           mechanistically distinct process embryonic wound <br /> differentiated barrier epidermal cells assume polar-                      closure recently reported (Campos et al. 2009). <br /> ized motile morphology (Odland Ross 1968;                           However, tissue repair field lacks method for <br /> Clark et al. 1982; Galko Krasnow 2004; Wu et al.                        systematic vivo identification functional classifica- <br /> 2009). This change migratory <span id='am-86' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-87' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span>phenotype</span> es-                          tion genes required postembryonic wound closure. <br /> sential efficient repair. The identification genes                      One pathway implicated postembryonic repair in <br /> involved repair assignment specific cellular                      vertebrates Drosophila Jun N-terminal <br /> functions genes vertebrate models                          kinase ( JNK) signaling pathway (Ramet et al. 2002; Li <br /> been hindered redundancy complexity                           et al. 2003; Galko Krasnow 2004), often <br /> the vertebrate tissue repair response (Martin 1997;                         controls epithelial migrations (Xia Karin 2004). In <br /> Grose Werner 2004). Cell culture-based assays                           Drosophila, JNK implicated developmen- <br /> (Simpson et al. 2008; Vitorino Meyer 2008)                         tally programmed epithelial migrations, including dor- <br />                                                                             sal closure (DC) (Riesgo-Escovar et al. 1996; Sluss <br />   Supporting information available online http://www.genetics.org/ <br />                                                                             et al. 1996), thorax closure (Zeitlinger Bohmann <br /> cgi/content/full/genetics.110.121822/DC1.                                   1999), border cell migration (Llense Martin- <br />   Available freely online author-supported open access          Blanco 2008). Of responses, JNK signaling during <br /> option.                                                                     DC extensively studied. In canonical <br />   1 <br />    Corresponding author: Department Biochemistry Molecular           context intracellular signaling relay defined <br /> Biology&#226;&#8364;&#8220;<span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>Unit</span> 1000, University Texas MD Anderson Cancer Center, <br /> 1515 Holcombe Blvd., Houston, TX 77030.                                     follows: JNK kinase kinase kinase ( Jun4K) Mis- <br /> E-mail: mjgalko@mdanderson.org                                              shapen (Treisman et al. 1997; Su et al. 1998), JNK <br /> Genetics 186: 943&#226;&#8364;&#8220;957 (November 2010) <br />  944                                                     C. Lesch et al. <br />  <br /> kinase kinase ( Jun3K) Slipper (Stronach Perrimon             analysis assign putative functional roles these <br /> 2002), JNK kinase ( Jun2K) Hemipterous (Glise et al.          genes. Since UAS-RNAi transgene libraries cover <br /> 1995), JNK Basket (Riesgo-Escovar et al. 1996;            nearly genome (Dietzl et al. 2007) and <br /> Sluss et al. 1996). Phosphorylated Basket activates           functional annotation lines rapidly proceed- <br /> transcription factors DJun DFos (Riesgo-Escovar               ing (Mummery-Widmer et al. 2009), methodology <br /> and Hafen 1997; Kockel et al. 2001) encoded                described highly scalable. Thus, anticipate <br /> genes Jun-related antigen (Jra) kayak (kay), respec-          eventually identifying dissecting specific roles of <br /> tively. Despite intensive study, signal(s) activate      complement genes required epi- <br /> this pathway DC larval wound closure              dermis proper wound healing. <br /> remain unidentified. In larval wound healing, only <br /> JNK pathway component far shown required for <br /> healing Basket (Galko Krasnow 2004),                              MATERIALS AND METHODS <br /> architecture signaling pathway upstream <br /> and downstream JNK activation remains unclear.                    Fly strains genetics: Drosophila melanogaster strains were <br />                                                                   reared 25&#194;&#176; standard cornmeal media. We used the <br /> Similarly unclear relationship JNK                 GAL4/UAS (Brand Perrimon 1993) tissue- <br /> activation actin cytoskeletal dynamics likely        specific expression transgenes: A58-Gal4 (Galko and <br /> drive epidermal cell migration wound gap,              Krasnow 2004) UAS-Dcr-2;A58-Gal4 (see below) express <br /> although increase actin wound edge                Gal4 predominantly larval epidermis, e22c-Gal4 <br /> larvae lacking JNK epidermal sheet                 (Lawrence et al. 1995) expresses Gal4 embryonic and <br />                                                                   larval epidermis. We constructed &#226;&#8364;&#732;&#226;&#8364;&#732;wound reporter&#226;&#8364;&#8482;&#226;&#8364;&#8482; lines <br /> recently shown (Wu et al. 2009).                                  basis epidermal drivers. All involved recombin- <br />    Localized actin polymerization thought drive         ing driver UAS-src-GFP transgene <span id='am-5' about='rdfs:label' typeof='owl:Thing'>label</span> cell <br /> protrusive cell behavior observed wound healing            membranes UAS-DsRed2-Nuc transgene label cell <br /> and instances cell migration (Pollard                nuclei. w1118;;UAS-src-GFP,UAS-DsRed2-Nuc,A58-Gal4/TM6B <br /> Borisy 2003). Biochemical cell culture approaches             (A58) built follows: DsRed2-Nuc PCR amplified <br />                                                                   pDsRed2-Nuc (BD Biosciences Clontech) using 59 <br /> have identified regulators actin cytoskeletal             primer EcoRI restriction site (59-GAAGGAATT <br /> dynamics including Rho GTPases (Nobes Hall                CATGGCCTCCTCCGAGAACG-39) 39 primer an <br /> 1995) Arp2/3 complex (Welch et al. 1997)              EagI restriction site (59-GAAGCGGCCGTTATCTAGATCCGG <br /> its activators like SCAR, stimulate              TGG-39), digested EcoRI EagI, ligated EcoRI/ <br /> polymerization new rearrangement existing                EagI-digested pUAST. Transgenic flies generated by <br />                                                                   standard procedures UAS-DsRed2-Nuc inserts third <br /> actin filaments. SCAR, instance, clearly required          chromosome recombined A58-Gal4 third <br /> for polymerization new actin filaments branch             chromosome UAS-src-GFP transgene. In parallel, A58 <br /> <span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span> shafts preexisting filaments (Goley               reporter combined second chromosome UAS- <br /> Welch 2006), prerequisite lamellipodial protru-             Dicer-2 (Dcr-2) transgene (Dietzl et al. 2007) enhance RNAi <br /> sion cell culture (Kiger et al. 2003). SCAR            potency (Dcr-2;A58). The &#226;&#8364;&#732;&#226;&#8364;&#732;e22c reporter&#226;&#8364;&#8482;&#226;&#8364;&#8482; w1118;e22c-Gal4,UAS- <br />                                                                   src-GFP,UAS-DsRed2-Nuc/CyO (e22c) constructed similarly. <br /> required certain cell migrations developing            For detailed morphological analysis (see below) used <br /> Drosophila embryo (Zallen et al. 2002). However,              w1118;e22c-Gal4,UAS-DsRed2-Nuc/CyO hs-hid crossed <span id='am-7' about='protege:TO' typeof='owl:Thing'>to</span> <br /> in vivo requirement factors physi-            respective RNAi lines. UAS-basketRNAi UAS-basketDN served <br /> ologically induced cell migrations wound heal-            positive control wound closure defects, progeny <br /> ing remains unclear.                                              w1118 crossed respective reporter served a <br />                                                                   negative control. We refer parental line bearing the <br />    Here, combining epidermal reporter cell               UAS-RNAi transgene &#226;&#8364;&#732;&#226;&#8364;&#732;RNAi line&#226;&#8364;&#8482;&#226;&#8364;&#8482;. When RNAi line is <br /> morphology, larval wound healing assay, UAS-                crossed Gal4 wound reporters, refer the <br /> RNAi gene knockdown technology (Kennerdell                    progeny &#226;&#8364;&#732;&#226;&#8364;&#732;geneX RNAi&#226;&#8364;&#8482;&#226;&#8364;&#8482;. <br /> Carthew 2000; Dietzl et al. 2007) performed                     All 190 UAS-RNAi lines used pilot screen (supporting <br /> proof-of-principle genetic screen targeting primarily             information, Table S1) obtained NIG-Fly (http:// <br />                                                                   www.shigen.nig.ac.jp/fly/nigfly/index.jsp) Vienna <br /> putative JNK pathway kinases, transcription factors,              Drosophila RNAi Center (VDRC). For optimization screen- <br /> substrates (Otto et al. 2000), selected regulators            ing following RNAi lines targeting following genes were <br /> of actin cytoskeletal dynamics. Although members           selected: 1388R-1 1388R-2 (TGF- b activated kinase 1, Tak1); <br /> the canonical JNK pathway defined DC                  2190R-2 4353R-2 (hemipterous, hep); 2248R-1 (Rac1); 2272R- <br /> required normal wound closure, architecture            1, 2272R-2, 2272 #33516*, 2272 #33518 (slipper, slpr); 2275R- <br />                                                                   2 ( Jun-related antigen, DJun/Jra); 4636R-1 (SCAR); 4720 #34891 <br /> the signaling pathway larval wound healing                 4720 #34892* (Protein kinase 92B, Pk92B); 4803 #34898 <br /> differs important ways. Further, established                (Tak1-like 2, Takl2); 5336R-2 5336R-3* (Ced-12); 5680R-1 and <br /> functional roles regulators actin              5680R-2 (basket, bsk); 7717 #25528 (Mekk1); 8261R-1 (G protein g <br /> cytoskeleton. Some genes appeared function              1, Gg1); 9738R-1 9738R-3* (MAP kinase kinase 4, Mkk4); <br /> primarily leading edge cells, acted               9901R-2 (Actin-related protein 14D, Arp14D); 10076R-1 (spire, <br />                                                                   spir); 10379R-1 (myoblast city, mbc); 12235R-1* 12235R-3 <br /> farther epidermal sheet groups             (Arp11); 12530R-2 12530R-3 (Cdc42); 15509R-2 (kayak, <br /> of cells. Thus, screen capable identifying new          DFos/kay); 16973R-1 16973R-2 (misshapen, msn); 31421 <br /> wound closure genes subsequent morphological                  #25760 (Tak1-like 1, Takl1); DPXN IR N1 DPXN IR N3 <br />                                                    Drosophila Wound Closure Screen                                                945 <br />  <br /> (Paxillin, Pax). Lines indicated asterisk         For analysis JNK activation, RNAi lines crossed to <br /> ones plotted Figure 3, B C. Lines UAS-RNAi           w1118;e22c-Gal4/CyO Act::GFP;msn-lacZ/TM6B w1118;;msn- <br /> inserts targeting gene denoted x2 Figure 3,       lacZ,A58-Gal4/TM6B progeny larvae carrying msn-lacZ, the <br /> B C Tak1x2 1388R-1,1388R-2: hepx2 2190R-2,4353R-2; <br /> bskx2 5680R-1,5680R-2; Cdc42x2 12530R-2,12530R-3;            Gal4 driver, UAS-RNAi transgene dissected 6 hr post <br /> msnx2 16973R-2;16973R-1; Paxx2 DPXN IR N1,DPXN           wounding post mock wounding. Histochemistry per- <br /> IR N3.                                                               formed described (Galko Krasnow 2004) the <br />     We constructed RNAi lines, targeting different      following modifications: Samples fixed 20 min cold <br /> genes: 4353R-2,9738R-1 hep,Mkk4 1388R-1,1388R-               2% glutaraldehyde room temperature, rinsed phosphate- <br /> 2;2272 #33516 Tak1x2;slpr. UAS-Rac1-N17DN UAS-cdc42-         buffered saline, stained 37&#194;&#176; 20 min. Samples were <br /> N17DN (Luo et al. 1994) used inhibit Rac1 Cdc42          imaged 103/0.40 HC PLAN APO objective a <br /> function, viable alleles Tak1 (Tak12527) slipper          digital camera (Leica DFC300 FX) mounted Leica DM5500 <br /> (slprBS06) (Polaski et al. 2006) used test functions   B stereomicroscope. Eighteen 52 Z-stacks 2-mm depth were <br /> in larval wound healing.                                             collected in-focus information extracted using <br />     To confirm open wound phenotypes listed Figure 3          extended-depth field algorithm Image-Pro MDA v6.1 <br /> we tested additional overlapping nonoverlapping RNAi             software (Media Cybernetics). <br /> lines (the NIG-Fly VDRC identifiers strains           Quantitative analysis morphological features: Twenty- <br /> listed Table S2, columns 3 4).                                hours post wounding, various wound features were <br />    Wounding assays: Pinch puncture wounds per-              measured. For measurement wound area (Figure 5B) the <br /> formed described (Galko Krasnow 2004). Briefly,               wound gap defined area free epidermal cells. <br /> early instar (L3) Drosophila larvae pinched          Outlining wound perimeter using Adobe Photoshop CS3 <br /> blunted forceps dorsal aspect single abdominal           allowed measurement area. To quantify clustering of <br />                                                                      wound-edge nuclei (Figure 5C) defined &#226;&#8364;&#732;&#226;&#8364;&#732;front line&#226;&#8364;&#8482;&#226;&#8364;&#8482; nuclei <br /> segment, usually A4, -5, -6. Larvae maintained 25&#194;&#176; <br />                                                                      closest wound gap .20 mm2 and <br /> with exception wounding imaging, per- <br />                                                                      located ,40 mm wound margin. The center-to- <br /> formed room temperature. <br />                                                                      center distance adjacent line nuclei was <br />    Pilot screen wound closure genes: Females bearing one <br />                                                                      measured. When distance ,17.8 mm, considered <br /> of epidermal reporters crossed males harboring <br />                                                                      nuclei cluster. Clusters adjacent nuclei <br /> UAS-RNAi inserts block target gene expression larval <br />                                                                      counted open wounds resulting targeting <br /> epidermis. These sexes reversed UAS-RNAi transgenes <br />                                                                      genes classes I, III, IV. To quantify nuclear crowding <br /> on X chromosome. Early L3 progeny larvae harboring <br />                                                                      farther wound edge (Figure 5D) area occupied <br /> both reporter UAS-RNAi transgene etherized, <br />                                                                      nuclei 50-mm strip extending cell rows <br /> pinch wounded, allowed recover fly food 24 hr, a <br />                                                                      wound margin measured using Image-Pro <br /> time control wounds closed. For <br />                                                                      version 6.2. To normalize effect wound size, divided <br /> visualization, etherized larvae mounted double-sided <br />                                                                      nuclear area wound gap area wound. <br /> tape, flattened coverslips, viewed dorsal a <br />                                                                      Figure 5A provides schematic overview parameters <br /> GFP2 filter Leica MZ16FA stereomicroscope using a <br />                                                                      (described above) quantified open wounds. We <br /> Planapo 1.63 objective. Images captured using color <br />                                                                      used nonparametric Mann&#226;&#8364;&#8220;Whitney test assess statis- <br /> digital camera (Leica DFC300 FX) Image-Pro AMS v5.1 <br />                                                                      tical significance (Figure 5, B&#226;&#8364;&#8220;D). To quantify epidermal <br /> software (Media Cybernetics). Wounds scored &#226;&#8364;&#732;&#226;&#8364;&#732;open&#226;&#8364;&#8482;&#226;&#8364;&#8482; <br />                                                                      syncytium formation closed wounds (Figure 5E) we <br /> if dark gap free nuclei remained epidermal sheet. <br />                                                                      counted number nuclei syncytial cell near <br />    Statistical analysis survival rate: A minimum 30 <br />                                                                      wounds control larvae (w1118) msnRNAi x2 <br /> unwounded wounded larvae genotype ana- <br />                                                                      -expressing larvae. <br /> lyzed survival rates 24 hr post wounding post mock <br /> treatment. We performed pairwise comparisons (A58 vs. Dcr- <br /> 2;A58, e22c vs. A58, e22c vs. Dcr-2;A58) unwounded <br /> (Table S3) wounded (Table S4) larval survival rates                               RESULTS <br /> UAS-RNAi transgene using chi-square tests Fisher&#226;&#8364;&#8482;s exact <br /> tests, appropriate. In addition, compared survival          Normal progression wound closure: To provide <br /> rate unwounded vs. wounded UAS-RNAi transgenes              context analysis UAS-RNAi screen, first <br /> given reporter strain (Table S5). To account inflated        observed normal pinch wound closure control larvae. <br /> type 1 error rate, Bonferroni correction applied (P -value     The unwounded epidermis larvae consisted a <br /> for significant difference set 0.0001792 instead 0.05). <br />                                                                      regularly patterned monolayer primarily mononu- <br />    Whole mount immunofluorescence histochemistry: <br /> Dissection immunostaining larval epidermal              cleate polygonal cells (Figure 1A). Immediately <br /> mounts performed described 4% para-              wounding, gap  0.2 mm2 epidermal <br /> formaldehyde used fixative antigens (Galko           sheet apparent (Figure 1B). Cells wound <br /> and Krasnow 2004). Primary antibodies anti-Fasciclin III        margin partially destroyed intact cells farther <br /> diluted 1:50 (Patel et al. 1987), anti-phospho-Histone H3 <br />                                                                      away wound gap maintained regular <br /> (Ser10) (Upstate Cell Signaling) diluted 1:500, anti-DFos <br /> (Zeitlinger et al. 1997) diluted 1:1000, anti-DJun               shape. Four hours later, cells rows <br /> (Bohmann et al. 1994) diluted 1:2000. All primary                surrounding wound elongated gap <br /> secondary antibodies diluted phosphate-buffered              (arrows Figure 1C). In addition, wound margin <br /> saline, 1% heat-inactivated goat serum, 0.3% Triton X-100        cells showed intense Fasciclin III staining (solid arrow- <br /> (PHT) buffer. Secondary antibodies FITC-conjugated goat         heads), lacked Fasciclin III staining the <br /> anti-mouse IgG (H1L) (1:200; Jackson ImmunoResearch), Alexa <br /> Fluor 488-conjugated goat anti-mouse IgG2a (1:200; Invitrogen        wound-facing cell (open arrowheads). By 8 hr <br /> Molecular Probes), Cy3-conjugated goat anti-rabbit IgG           migrating cells covered original wound gap <br /> (H1L) (1:300; Jackson ImmunoResearch).                               cells ones appeared elongated <br />  946                                                   C. Lesch et al. <br />  <br />  <br />  <br />  <br />                                                                               Figure 1.&#226;&#8364;&#8221;Temporal progression wound <br />                                                                            closure control larvae. (A&#226;&#8364;&#8220;E) Dissected whole <br />                                                                            mounts larvae heterozygous e22c-Gal4 <br />                                                                            driver UAS-DsRed2-Nuc. Nuclei expressing <br />                                                                            DsRed2-Nuc red membranes immunos- <br />                                                                            tained Fasciclin III green. (A) Un- <br />                                                                            wounded epidermal sheet. (B&#226;&#8364;&#8220;E) Epidermal <br />                                                                            sheet wounding: (B)  5 min post wound- <br />                                                                            ing, (C) 4 hr post wounding, (D) 8 hr post <br />                                                                            wounding, (E) 24 hr post wounding. Note <br />                                                                            epidermal sheet closed. Solid arrow- <br />                                                                            heads, wound-edge cells retain Fasciclin III <br />                                                                            staining facing wound gap; open arrowheads, <br />                                                                            wound-edge cells lack Fasciclin III staining <br />                                                                            facing wound gap; arrows, elongated epider- <br />                                                                            mal cells; dots, multinucleate epidermal cells. <br />                                                                            Bar, 100 mm. <br />  <br />  <br />  <br />  <br /> (arrows Figure 1D). Further, cells multiple nuclei      transgenes label epidermal cell membranes (UAS-src- <br /> (white dots) visible. By 24 hr wounds           GFP) nuclei (UAS-DsRed2-Nuc). UAS-Dicer-2 a <br /> were closed possessed irregularly shaped epidermal          transgene reported enhance RNAi potency (Dietzl <br /> cells wound site including syncytia          et al. 2007). We established wound re- <br /> containing 2&#226;&#8364;&#8220;12 nuclei (Figure 1E). In summary,          porters identify wound closure defects when <br /> normal wound closure cells adjacent wound                viewed live. Unwounded A58 reporter-bearing larvae <br /> change shape. These cells begin migrate        possessed largely polygonal mononuclear epidermal <br /> close wound gap epidermal cells farther away          cells highly uniform size shape (Figure 2A), <br /> elongate wound. Moreover, formation               similar observed immunostained whole <br /> small syncytia wound site normal event       mounts (Figure 1A Galko Krasnow 2004). <br /> wound closure. Because immunostaining dissected              Immediately wounding, wound gap appar- <br /> larvae laborious, sought develop reporters        ent (Figure 2B). Twenty-four hours post wounding, <br /> that permit rapid live scoring wound closure.          wound closure complete control larvae, <br />    Epidermal reporters allow live assessment wound           irregularly spaced nuclei haze green fluores- <br /> closure: We built wound reporters consist Gal4        cence overlaid previous wound gap (Figure 2C). <br /> driver embryonic (e22c) larval (A58 Dcr-      Although cellular membranes retaining GFP the <br /> 2;A58) onset epidermal expression recombined            wound gap bright viewed live in <br />                                              Drosophila Wound Closure Screen                                           947 <br />  <br />  <br />  <br />  <br />                                                                            Figure 2.&#226;&#8364;&#8221;An epidermal wound reporter al- <br />                                                                          lows live scoring wound closure. (A&#226;&#8364;&#8220;C and <br />                                                                          E&#226;&#8364;&#8220;F) Live larvae bearing copy A58 epi- <br />                                                                          dermal wound reporter. (A&#226;&#8364;&#8220;D) Control (crossed <br />                                                                          w1118). (A) Mock wounded. (B&#226;&#8364;&#8220;F) Pinch <br />                                                                          wounded. (B) Immediately post wounding. (C) <br />                                                                          Twenty-four hours post wounding. Not cell <br />                                                                          membranes visible, nuclei present <br />                                                                          wound gap. (D) Dissected whole <br />                                                                          mount larva C, immunostained <br />                                                                          Fasciclin III (green). The arrowheads mark <br />                                                                          identical positions C D. Faint membranes <br />                                                                          difficult live larvae apparent. <br />                                                                          (E) Larva expressing UAS-bskDN transgene. (F) <br />                                                                          Larva expressing UAS-bskRNAi transgene exhibits <br />                                                                          wound closure defect similar UAS-bskDN. <br />                                                                          Arrows, tracheal dorsal trunks. Bar, 100 mm. <br />  <br />  <br />  <br />  <br /> dissected samples (compare Figure 2C 2D), src-            Using e22c wound reporter, retained src- <br /> GFP nuclear RFP signals sufficient un-            GFP membrane fluorescence wound-proximal cells <br /> ambiguously score defective wound closure (Figure 2, E        better reporters (data shown), we <br /> and F). Similar results obtained e22c   screened 190 UAS-RNAi lines targeting 142 genes (see <br /> Dcr2;A58 reporters (data shown). In conclusion,       Table S1 general scheme Figure 3A). Fifteen UAS- <br /> three epidermal wound reporters useful tools          RNAi transgenes showed open wound phenotype <br /> rapid live scoring wound closure defects. We used     4 transgenes did produce substantial pop- <br /> these reporters establish viable method identifi-    ulation testable larvae (Table S1). We wished to <br /> cation wound closure genes.                                determine onset expression (embryonic or <br />    Screening strategy: To test efficacy RNAi       larval) RNAi potency determining <br /> screening, asked tissue-specific UAS-      factor yield screen. To assess this, we <br /> RNAi transgene phenocopy wound closure              retested subset UAS-RNAi transgenes three <br /> defect observed expression dominant-negative        reporters. For optimization, surveyed lethality, <br /> version basket (UAS-basketDN), known wound closure       percentage open wounds, morphology of <br /> gene (Galko Krasnow 2004). Reporter-bearing               wounded unwounded epidermis. Our test set <br /> larvae expressing UAS-bskRNAi transgene phenocopied         included 29 UAS-RNAi lines targeting 21 candidate <br /> the UAS-bskDN wound closure defect (Figure 2, E F),       wound closure genes, scored positive in <br /> confirming RNAi-mediated gene knockdown,              round screening. Some surprising negatives <br /> least gene, lead wound closure de-        included preliminary assessment of <br /> fect. Assessment RNAi potency re-         false negatives initial screen. <br /> porters using UAS-GFPRNAi transgene (Levi et al. 2006)      For candidates (slpr, Pk92B, Mkk4, Ced-12, <br /> to target UAS-src-GFP revealed e22c Dcr-2;A58     Arp11) tested different independent RNAi lines <br /> reporters strong approximately equal knockdown            present UAS-RNAi transgene that <br /> that greater achieved A58 re-          showed highest percentage open wounds (Figure <br /> porter (Figure S1).                                           3, B C). The candidate genes fell categories. <br />  948                                                       C. Lesch et al. <br />  <br />  <br />  <br />  <br />   Figure 3.&#226;&#8364;&#8221;Quantification wound closure epidermal expression UAS-RNAi transgenes. (A) Workflow UAS- <br /> RNAi&#226;&#8364;&#8220;based screen wound closure genes. (B C) Percentage open wounds expression UAS-RNAi transgenes <br /> targeting indicated genes JNK pathway SAPK signaling components (B) genes involved actin cytoskeletal <br /> remodeling (C). Open bars, e22c reporter; shaded bars, A58 reporter; solid bars, Dcr-2;A58 reporter. w1118 crossed respective <br /> reporter used negative control bskRNAi positive control. Dashed line, arbitrary 15% cutoff detailed morpho- <br />                                                  Drosophila Wound Closure Screen                                                    949 <br />  <br /> The comprised genes stress-activated protein                 Jun2 kinases Jun3 kinases led nearly <br /> kinase (SAPK) signaling pathways canonical                fully penetrant wound closure defect (Figure S4). In <br /> Jun kinase relay associated transcription factors (in             conclusion, genes canonical JNK pathway and <br /> total 17 UAS-RNAi transgenes targeting 11 genes). The                 noncanonical components (Tak1, Takl2, and <br /> second contained genes involved actin cytoskeletal                 Mkk4) showed open wound phenotype least <br /> remodeling, including Rho-like GTPases genes                      reporters. <br /> involved phagocytosis (in total 12 UAS-RNAi trans-                    Actin cytoskeletal remodeling required normal <br /> genes targeting 10 genes).                                            wound closure: The second test set comprised 10 genes <br />    For hits category able            involved actin cytoskeletal remodeling (Figure 3C). <br /> confirm open wound phenotype RNAi lines                  Four UAS-RNAi transgenes exhibited open <br /> target nonoverlapping sequences gene (nine                 wound phenotype similar positive control of <br /> of lines), alternative RNAi lines target com-               bskRNAi showed .50% open wounds at <br /> pletely partially overlapping sequences (five               wound reporters. The most <br /> lines), dominant-negative transgenes (two trans-               penetrant open wound phenotype (92&#226;&#8364;&#8220;100%) was <br /> genes), larval viable mutant alleles (two                   achieved targeting Gg1, Ced-12, Arp14D, mbc, <br /> mutants) (Figure S2 Table S2). For proteins                   followed Rac1, SCAR, Arp11 (65&#226;&#8364;&#8220;72%) com- <br /> for good antibodies immunostaining larval                bination reporters. Three UAS- <br /> whole mounts exist (DFos DJun), able                 RNAi transgenes (spir, Cdc42, Pax) showed open <br /> to verify on-target knockdown targeted protein                 wounds. This result inefficient <br /> (Figure S3). Together results suggest                  knockdown targeted gene indicate that <br /> wound closure phenotype of hits highly                 genes play role wound closure (see discussion <br /> unlikely to off-target RNAi effects.                        below). <br />    Canonical noncanonical JNK pathway compo-                         Comparison wound reporters: In total, an <br /> nents involved wound closure: Genes               open wound phenotype scored 9 UAS-RNAi <br /> test category (Figure 3B) encoded canonical JNK                       transgenes using A58 12 UAS-RNAi transgenes <br /> pathway SAPK signaling components. The                      using e22c Dcr-2;A58 (Figure 3, B C). However, <br /> highest percentage open wounds obtained                   reporter lethal crossed transgenes <br /> targeting bsk, DJun/Jra, DFos/kay (92&#226;&#8364;&#8220;100% open                   targeting msn, DJun/Jra, Ced-12. In addition, we <br /> wounds). These followed transgenes targeting                  observed larval epidermis fragile <br /> the Jun4 kinase Msn (46%), Jun/SAP2 kinases                       expression Dcr-2 (data shown). When the <br /> Hep Mkk4 (38&#226;&#8364;&#8220;63%), Jun/SAP3 kinases                       percentage open wounds high (90&#226;&#8364;&#8220;100%), we <br /> (0&#226;&#8364;&#8220;52%). Interestingly, expression UAS-RNAi trans-                 obtained similar results reporters. In <br /> genes targeting members canonical JNK signaling                contrast, choice reporter critical RNAi <br /> pathway involved DC showed higher percentage                  lines medium low percentage open wounds. <br /> open wounds compared kinases                     Where differences reporters, <br /> level kinase cascade. For example, Jun2K Hep               RNAi potency major determinant the <br /> (required DC) showed 63% open wounds                    strength wound closure phenotype of <br /> SAP2K Mkk4 (not required DC) showed 38% open                      differences. For instance, targeting hep, Mkk4, <br /> wounds using e22c reporter. Among SAP3                    mbc, Rac1 showed open wounds the <br /> kinases tested, transgenes targeting slpr, Tak1,              stronger e22c Dcr-2;A58 reporters. Only SCAR and <br /> and Takl2 displayed open wounds arbitrary                    Arp14D showed open wounds e22c re- <br /> 15% threshold Jun3K Slpr (required DC)                   porter Dcr-2;A58, suggesting these <br /> showing highest percentage (52% Dcr-2;A58)                   genes early onset RNAi expression the <br /> within group. In short, downstream members                 major determinant. For nonlethal crosses, statistical <br /> JNK/SAPK pathways showed higher percentage                       comparison survival rates UAS-RNAi trans- <br /> open wounds upstream SAP kinases. It likely               genes unwounded (Table S3) wounded (Table <br /> that Hemipterous/Mkk4 Jun2K level Slip-                    S4) larvae showed sporadic cases <br /> per/Tak1 Jun3K level redundant                   choice reporter significantly influenced <br /> other coexpression RNAi transgenes targeting                 subsequent survival. Further, comparison unwounded <br />  <br />  <br /> logical analysis. Daggers indicate insufficient L3 larvae testing. Lines UAS-RNAi inserts targeting gene are <br /> denoted x2. Genes boldface type B indicate members canonical JNK pathway. The numbers scored larvae each <br /> RNAi knockdown using A58 Dcr-2;A58 B C follows: n(A58-Gal4) &#194;&#188; 82 (w1118), 56 (msnx2), 39 (slpr), 39 (Tak1x2), 31 <br /> (Takl1), 35 (Takl2), 32 (Pk92B), 31 (Mekk1), 36 (hepx2), 38 (Mkk4), 84 (bskx2), 42 (DJun/Jra), 33 (DFos/kay), 33 (Ced-12), 41 (mbc), 31 <br /> (Rac1), 36 (Cdc42x2), 36 (SCAR), 37 (Arp14D), 30 (Arp11), 46 (spir), 35 (Paxx2), 36 (Gg1); n(Dcr-2;A58-Gal4) &#194;&#188; 72 (w1118), <br /> 33 (slpr), 35 (Tak1x2), 44 (Takl1), 31 (Takl2), 33 (Pk92B), 34 (Mekk1), 35 (hepx2), 36 (Mkk4), 69 (bskx2), 65 (DFos/kay), 30 (mbc), 35 <br /> (Rac1), 40 (Cdc42x2), 33 (SCAR), 33 (Arp14D), 37 (Arp11), 46 (spir), 34 (Paxx2), 34 (Gg1); Table S1 n(e22c). <br />  950                                                   C. Lesch et al. <br />  <br />  <br />  <br />  <br />                                                                                              Figure 4.&#226;&#8364;&#8221;UAS-RNAi trans- <br />                                                                                           genes affecting wound closure <br />                                                                                           distinct wound morpholo- <br />                                                                                           gies. (A&#226;&#8364;&#8220;I) Dissected mounts <br />                                                                                           larvae heterozygous e22c-Ga- <br />                                                                                           l4,UAS-DsRed2-Nuc (red) the <br />                                                                                           indicated UAS-RNAi transgene <br />                                                                                           immunostained Fasciclin <br />                                                                                           III (green) 24 hr post wounding. <br />                                                                                           UAS-RNAi transgenes grouped <br />                                                                                           seven classes basis of <br />                                                                                           wound morphology. (A) Class I, <br />                                                                                           bskx2; (B) class I, DFos/kay; (C) class <br />                                                                                           I, DJun/Jra; (D) class II, hepx2; (E) <br />                                                                                           class III, SCAR; (F) class IV, Ced- <br />                                                                                           12; (G) class V, mbc; (H) class VI, <br />                                                                                           Rac1; (I) class VII, msnx2. See re- <br />                                                                                           sults classification criteria. <br />                                                                                           Stars indicate disorganized epider- <br />                                                                                           mal sheet; arrowheads indicate <br />                                                                                           smooth wound edge pro- <br />                                                                                           nounced fluorescence; brackets <br />                                                                                           span zones nuclear crowding. <br />                                                                                           Lines UAS-RNAi inserts tar- <br />                                                                                           geting gene denoted <br />                                                                                           x2. Bar, 100 mm. <br />  <br />  <br />  <br />  <br /> vs. wounded survival given UAS-RNAi transgene             reporter (see Figure 3, B C). Differences epidermal <br /> with particular reporter (Table S5) showed         organization apparent grouped into <br /> was case survival wounding             seven classes (detailed below) basis qualitative <br /> significantly different using e22c-Gal4 reporter. In        (Figure 4) quantitative (Figure 5) criteria. <br /> sum, e22c reporter advantages early            Class I (bsk, DFos/kay, DJun/Jra): The distinguish- <br /> onset expression high RNAi potency (Figure S1)           ing feature class I genes cells surrounding <br /> without lethality epidermal fragility complica-         wound gap cells epidermal sheet <br /> tions resulting Dicer-2 expression. Given e22c        farther away wound failed elongate and <br /> also affords better visualization wound morphology,        remained similar shape size unwounded <br /> we used e22c reporter detailed analysis          epidermal cells (Figure 4, A&#226;&#8364;&#8220;C). At targeting <br /> wound morphology presented below.                               DFos/kay DJun/Jra, defects progress of <br />    Different classes wound closure genes              wound closure, including failure leading edge and <br /> identified quantitative qualitative assessment        distal cells elongate wound edge, <br /> wound morphology: We aimed classify                 apparent earlier time points (Figure S5). <br /> wound closure genes according morphology                 Class II (hep, Tak1, Mkk4): The distinguishing <br /> epidermal cells surrounding wound. For optimal              feature class II genes smaller size open <br /> morphological analysis used e22c reporter label-          wounds (Figures 4D 5B). This phenotype resembled <br /> ing epidermal nuclei red stained           closing wounds control larvae 8 hr post wound- <br /> the membranes dissected mounts anti-              ing (Figure 1D) exacerbated coexpression of <br /> Fasciclin III. To limit scope analysis wound      RNAi transgenes targeting Jun2 kinases or <br /> morphology, compared genes open-wound             Jun3 kinases (Figure S4, C E), sug- <br /> phenotype exceeded arbitrary 15% cutoff e22c        gesting redundancy levels JNK pathway. <br />                                               Drosophila Wound Closure Screen                                                  951 <br />  <br />  <br />  <br />  <br />                                                                               Figure 5.&#226;&#8364;&#8221;Quantification wound closure <br />                                                                            classes identified qualitative morphological <br />                                                                            features. (A) Schematic parameters quantified <br />                                                                            classes genes showing open wounds at <br />                                                                            24 hr post wounding. Hexagons, epidermal cells; <br />                                                                            solid circles, epidermal nuclei; b, wound area; c, <br />                                                                            clustering line nuclei; d, crowding nu- <br />                                                                            clei near wound margin. The lightly shaded <br />                                                                            area indicates region area occupied <br />                                                                            nuclei measured assess nuclear crowd- <br />                                                                            ing. For details quantification procedures see <br />                                                                            materials methods. (B) Quantification of <br />                                                                            wound area select classes. n &#194;&#188; 35, 10, 9 <br />                                                                            classes I, II, V, respectively. ***P , <br />                                                                            0.001, Mann&#226;&#8364;&#8220;Whitney test. (C) Quantification of <br />                                                                            clusters line nuclei select classes. n &#194;&#188; <br />                                                                            35, 32, 10 classes I, III, IV, respectively. <br />                                                                            ***P , 0.001, Mann&#226;&#8364;&#8220;Whitney test. For nonsignif- <br />                                                                            icant comparison (NS) P &#194;&#188; 0.725. (D) Quantifica- <br />                                                                            tion nuclear area 50 mm wound <br />                                                                            edge. n &#194;&#188; 15, 15, 10 classes I, III, and <br />                                                                            IV, respectively. ***P , 0.001, Mann&#226;&#8364;&#8220;Whitney test. <br />                                                                            For nonsignificant comparison (NS) P &#194;&#188; 0.467. <br />                                                                            (E) Quantification epidermal syncytium forma- <br />                                                                            tion closed wounds msnRNAi-expressing larvae. <br />                                                                            The number nuclei syncytial cell near <br />                                                                            wounds control msnRNAi x2 mea- <br />                                                                            sured. The diameter bubbles reflects the <br />                                                                            number cells number nuclei. Con- <br />                                                                            trol wounds average syncytial cells <br />                                                                            small numbers nuclei (2&#226;&#8364;&#8220;12) wounds <br />                                                                            msnRNAi-expressing larvae fewer syncytia <br />                                                                            greater numbers nuclei. <br />  <br />  <br />  <br />  <br />    Class III (SCAR, Arp14D, Gg1): The distinguishing       cence wound margin, smoother wound edges <br /> feature class III genes leading-edge epidermal     (Figure 4G), larger average wound size (Figure 5B). <br /> nuclei appeared &#226;&#8364;&#732;&#226;&#8364;&#732;pearl necklace&#226;&#8364;&#8482;&#226;&#8364;&#8482;-like clusters        Class VI (Rac1): The distinguishing feature class <br /> wound margin (Figure 4E). This demonstrated           VI gene disorganization epidermal cells <br /> quantitatively measuring presence clusters con-      surrounding open wound (Figure 4H). This not <br /> taining line nuclei      surprising unwounded epidermal sheet in <br /> certain minimal distance (see materials methods)           Rac1RNAi-expressing larvae highly disorganized <br /> of (Figure 5C). This analysis revealed         (see Figure S6 unwounded larvae expressing the <br /> both class III class IV genes shared property      UAS-RNAi transgenes shown Figure 4). <br /> front line nuclear clustering.                                    Class VII (msn): Open wounds larvae expressing <br />    Class IV (Ced-12): The distinguishing feature        msnRNAi similar ones Rac1RNAi larvae class <br /> class IV gene stronger nuclear crowding           VI (data shown). However, closed wounds msnRNAi <br /> much broader band extended farther              larvae exhibited distinct morphology (Figure 4I), in <br /> the wound margin compared class III genes. Smaller          original wound area (Figure S7A) a <br /> epidermal cells region near wound edge              single syncytial epidermal cell containing tens nuclei <br /> accompanied crowding (Figure 4F). When                 (Figure 5E). To accommodate differences, msn <br /> quantified nuclear area 50-mm distance         grouped separate class, class VII. This distinct <br /> wound edge (Figure 5D), measure nuclear                morphology did appear result nuclear <br /> crowding significantly higher class IV       division absence cytokinesis no <br /> gene Ced-12 class I class III genes.           anti-phospho-Histone H3 staining observed msnRNAi <br />    Class V (mbc): The distinguishing features class V   larvae, similar controls knockdown class <br /> gene pronounced accumulation green fluores-          I genes DJun/Jra DFos/kay (Figure S8). Nor did it <br />  952                                                  C. Lesch et al. <br />  <br />  <br />  <br />  <br />                                                                                            Figure 6.&#226;&#8364;&#8221;JNK activation in <br />                                                                                         larvae expressing UAS-RNAi trans- <br />                                                                                         genes affecting wound closure. <br />                                                                                         (A&#226;&#8364;&#8220;L) Dissected epidermal whole <br />                                                                                         mounts unwounded (A J) <br />                                                                                         pinch wounded (B&#226;&#8364;&#8220;I K <br />                                                                                         L) larvae heterozygous for <br />                                                                                         e22c-Gal4 driver (A&#226;&#8364;&#8220;K) the <br />                                                                                         A58-Gal4 driver (L), JNK activ- <br />                                                                                         ity reporter msn-lacZ (A&#226;&#8364;&#8220;L), and <br />                                                                                         indicated UAS-RNAi trans- <br />                                                                                         gene. All mounts were <br />                                                                                         stained 6 hr post wounding or <br />                                                                                         post mock wounding X-Gal <br />                                                                                         detect b-galactosidase reporter <br />                                                                                         activity (blue). (A) w1118, un- <br />                                                                                         wounded; (B) w1118, wounded; <br />                                                                                         (C) bskx2; (D) DJun/Jra; (E) DFos/ <br />                                                                                         kay; (F) hep,Mkk4; (G) SCAR; (H) <br />                                                                                         Ced-12; (I) mbc; (J) Rac1, un- <br />                                                                                         wounded; (K) Rac1, wounded; <br />                                                                                         (L) msnx2. Morphological classes <br />                                                                                         indicated panels. <br />                                                                                         Bar, 100 mm. <br />  <br />  <br />  <br />  <br /> appear to misregulated apoptosis,       classifying genes identified screen, used <br /> observed control msnRNAi wounds (data            JNK reporter msn-lacZ (Spradling et al. 1999; <br /> shown).                                                        Galko Krasnow 2004) assess extent of <br />    Finally, targeting genes de-        JNK pathway activation expression UAS-RNAi <br /> scribed nonoverlapping RNAi transgenes               transgenes block alter wound closure. In un- <br /> revealed distinctive morphologies consis-      wounded control larvae, msn-lacZ expressed at <br /> tent property specific RNAi transgenes        low levels (Figure 6A), 6 hr wounding <br /> first used screen [examples misshapen (class        induced graded fashion cell rows <br /> VII) Arp14D (class III) shown Figure S7].           surrounding wound (Figure 6B). Expression UAS- <br />    JNK activation expression UAS-RNAi trans-             bskRNAi reduces activation msn-lacZ (Figure 6C), <br /> genes affecting wound closure: As final mode              extent observed UAS-bskDN <br />                                              Drosophila Wound Closure Screen                                             953 <br />  <br />                                                                                            Figure 7.&#226;&#8364;&#8221;Model epider- <br />                                                                                         mal cell behaviors normal <br />                                                                                         perturbed wound healing. <br />                                                                                         Cell behaviors wound <br />                                                                                         healing indicated sche- <br />                                                                                         matic template. On tem- <br />                                                                                         plates, select cells contacting the <br />                                                                                         wound edge marked blue <br />                                                                                         nuclei, select cells within <br />                                                                                         sheet close vicinity the <br />                                                                                         wound marked red nu- <br />                                                                                         clei. Arrows indicate direction <br />                                                                                         cellular migration the <br />                                                                                         wound gap. The <span id='am-6' about='xsp:length' typeof='owl:Thing'>length</span> ar- <br />                                                                                         row symbolizes speed the <br />                                                                                         migrating cells (longer, fast; <br />                                                                                         shorter, slow). A T-bar indicates <br />                                                                                         migration blocked. Wavy <br />                                                                                         arrows improper direction- <br />                                                                                         ality migration. The bright <br />                                                                                         green wound margin class V in- <br />                                                                                         dicates pronounced membrane <br />                                                                                         fluorescence. The green central <br />                                                                                         wound area class VII indicates <br />                                                                                         large syncytial cell. See text for <br />                                                                                         model details. <br />  <br /> (data shown Wu et al. 2009). Targeting putative       msn-lacZ reporter serves useful tool further <br /> JNK pathway components blocked msn-lacZ activation            classify genes identified originally grouped by <br /> except notable surprises. The          effects epidermal morphology. <br /> targeting transcription factors DJun DFos, <br /> which thought work tandem exhibit <br />                                                                                     DISCUSSION <br /> a similar wound closure morphology (Figure 4, B and <br /> C), did affect msn-lacZ activation equally. Expression       Morphological classification wound closure genes <br /> of DJun/JraRNAi slight effect msn-lacZ          suggests multiple cellular processes required for <br /> activation (Figure 6D), expression DFos/kayRNAi      normal closure: We genes identified in <br /> led complete block (Figure 6E), similar          screen grouped seven classes, each <br /> observed coexpression UAS-RNAi transgenes             possessing distinct morphological feature defect. <br /> targeting upstream SAP2 kinases Hep Mkk4              Here, use classes infer biological <br /> (Figure 6F) SAP3 kinases Slpr Tak1 (data       functions occur epidermal sheet bring <br /> shown). The second surprise expression            proper closure. These inferences illus- <br /> msnRNAi targeting Jun4K required DC did           trated graphically Figure 7. <br /> block msn-lacZ activation enhance              Targeting class I genes (bsk, DJun/Jra, DFos/kay) <br /> slightly (Figure 6L).                                         led open wounds epidermal cells surrounding <br />    Notably, classes genes           wound appeared largely maintain original <br /> included actin cytoskeletal modulators, classes III       shapes distribution. This contrast class III <br /> V, little effect msn-lacZ expression (Figure     genes (SCAR, Arp14D, Gg1), targeting led to <br /> 6, G I). The exception class IV      pearl necklace-like clustering epidermal nuclei <br /> gene, Ced-12 (Figure 6H), targeting led            wound edge, exaggerated version of <br /> decrease JNK activation similar observed      seen control wounds actively closing <br /> expression bskRNAi (Figure 6C). Only Rac1RNAi affected     (8-hr control Figure 1D). Comparing classes I III, <br /> JNK activation unwounded state (Figure 6J          hypothesize processes required <br /> data shown), showing higher level msn-lacZ           closure. One ability leading-edge cells to <br /> activation unwounded controls (Figure 6A). We         wound gap. The second genetically <br /> found expression Rac1RNAi (Figure 6K)         separable ability cells farther wound <br /> Rac1DN (data shown) msn-lacZ activated           edge sheet gap. We <br /> further following wounding, curiously           speculate class I genes defective these <br /> activation uniform obviously graded        processes class III genes defective the <br /> the wound site. Taken together, results suggest         ability cells gap. <br /> that increase decrease Rac1 signaling          Class II genes (hep, Mkk4, Tak1) distinguished <br /> can activate JNK pathway. In conclusion,          smaller area open wounds that <br />  954                                                     C. Lesch et al. <br />  <br /> suggests delay wound closure. Since              msnRNAi-expressing larvae closed, albeit aberrantly. <br /> pronounced nuclear crowding genes                  These closed wounds exhibit abnormal morphology, <br /> targeted, hypothesize they, like class I genes (but       initial wound gap single <br /> likely lesser extent redundancy),           syncytial cell contain .60 nuclei. Although <br /> partially deficient movement gap             syncytium formation occurs normally larval <br /> movement sheet.                                        wound closure, usually involves relatively small num- <br />    The remaining classes date defined              bers (10&#226;&#8364;&#8220;12) cells. The msnRNAi phenotype suggests that <br /> single genes. The sole class IV gene, Ced-12, charac-          syncytium formation tightly regulated process during <br /> terized extreme cell nuclear crowding               normal closure hyperactivation syncytium <br /> the wound edge. In contrast class III genes,              formation does preclude closure wound. <br /> crowding extends farther wound edge                In summary, classification scheme suggests that <br /> in excess seen normal closure. This                different processes are <br /> phenotype suggests targeting Ced-12 actually             important normal wound closure: migration into <br /> lead hyperactivation directional cell migration           wound gap, migration epidermal sheet <br /> within wounded sheet (but wound gap).            near wound, directionality migrations, <br /> Ced-12 initially identified gene required            organization wound edge, regulation of <br /> apoptotic corpse clearance phagocytosis                syncytium formation. A recent scratch-wound study <br /> Caenorhabditis elegans (Gumienny et al. 2001). Migrating          employing adherent endothelial cells migrate a <br /> epidermal cells following wounding Drosophila                  sheet reached similar conclusions the <br /> larvae extend impressive phagocytic processes (Galko              cellular processes required closure (Vitorino and <br /> and Krasnow 2004). We speculate phagocytic              Meyer 2008). Our scheme, certainly incom- <br /> activity actually positively required move-            plete level genes contained each <br /> ment gap inhibitor movement             class, likely incomplete terms the <br /> within sheet.                                                 number distinct classes, hopefully serve a <br />    The class V gene (myoblast city) leads distinct      useful framework analyzing genes emerge from <br /> phenotype larger wounds smooth wound edges                reporter-based screening candidate gene <br /> that exhibit pronounced green fluorescence. This                  analysis (Kwon et al. 2010). <br /> suggests migrating epidermal cells properly                Architecture JNK signaling pathway larval <br /> organize wound edge membrane dynamics form                   wound closure: Despite intensive study, signal(s) <br /> normal wound edge migrate wound gap.                 activate pathway DC larval <br /> Why mbcRNAi open wounds larger? One possibility            wound closure remain unidentified. Our results suggest <br /> that wound closure defect mbcRNAi severe           architecture JNK signaling pathway in <br /> than genes classes I, III, IV. A second            wound healing differs DC, does in <br /> possibility mbc RNAi wounds gape extent.          JNK-dependent cellular processes innate <br /> This possibility supported apparently weak        immunity (Boutros et al. 2002; Silverman et al. 2003; <br /> wound edge retract fold                 Geuking et al. 2009) cell death (Igaki 2009). Given <br /> original border (data shown). Such folding                unique morphological phenotype persis- <br /> cause increased green fluorescence observed            tence JNK reporter activity expression msnRNAi <br /> edges wounds.                                            (see above), unlikely kinase acts linearly <br />    Targeting Rac1 (class VI) leads disorganized              upstream rest kinase relay. Like Misshapen, <br /> epidermis unwounded sheet             Rac reported upstream JNK <br /> constitutive activation JNK signaling. The odd spac-           activation DC (Woolner et al. 2005) recently <br /> ing irregular shapes epidermal cells           reported upstream JNK activation larval <br /> the unwounded wounded epidermal sheet                     epidermis expression activated form of <br /> suggest cells lacking Rac1 constitutively             Rac1 led activation JNK signaling (Baek et al. <br /> motile sheet motility lack           2010). The notion Rac1 linearly upstream JNK <br /> the directionality usually leads successful wound         larval epidermis likely oversimplifica- <br /> closure. An alternative interpretation loss Rac1       tion loss Rac1 function also <br /> somehow affects adhesive interactions                 leads JNK activation. One gene partially <br /> epidermal cells longer maintain               required upstream JNK activation wound closure <br /> their normal shapes. There considerable evidence            Ced-12, performs similar role thorax closure <br /> other systems Rac having effects migration            (Ishimaru et al. 2004). <br /> (Monypenny et al. 2009; Wang et al. 2010) adhesion               Unlike DC, Jun3K Jun2K are <br /> of epithelial cells (Eaton et al. 1995; Chihara et al.            exclusively required morphogenesis, wound <br /> 2003) possibilities mutually exclusive.          closure positions appear redundant. Two <br />    The sole class VII gene, misshapen, shows strangest        SAP3 kinases, Slipper Tak1, SAP2 kinases, <br /> phenotype come pilot screen. Most wounds              Hemipterous Mkk4, moderate wound closure <br />                                                 Drosophila Wound Closure Screen                                                      955 <br />  <br /> defect individually targeted. Redundancy sug-            entail. RNAi-based vitro scratch wounds partially <br /> gested severe open-wound phenotype                filled gap field (Simpson et al. 2008; Vitorino <br /> double-RNAi experiments double-mutant analysis               Meyer 2008) unlikely assays <br /> for Tak1 slipper (C. Lesch, J. Jo Y. Wu,                 recapitulate complexity wound healing it <br /> unpublished observations). JNK reporter results             occurs living organism. Here, developed a <br /> show complete absence JNK activation hepRNAi-            reasonably rapid, medium-throughput methodology <br /> Mkk4RNAi expression support redundancy.                     identifying wound closure genes vivo Drosoph- <br />    Finally, results, recent studies (Campos      ila larvae. The success future scalability the <br /> et al. 2009; Pearson et al. 2009), indicate importance       screen depend critically elements: reporter <br /> of DJun- DFos-mediated transcription following               allows live visualization closure recent <br /> wounding. In study, differences morphology,               development near whole-genome UAS-RNAi libraries <br /> msn-lacZ activation, survival, fragility cuticle      (Dietzl et al. 2007). Because RNAi based, screen <br /> (not shown) suggest DJun DFos act          likely false-negative rate (29.4%) similar to <br /> together serve distinct functions wound            screens (Mummery-Widmer et al. 2009) <br /> closure.                                                         larger selection positive control genes was <br />    Importance actin cytoskeletal remodeling                available analysis. Like screens involving read- <br /> wound closure: Unlike DC embryonic wound                     outs physiological challenge (Kambris et al. 2006; <br /> closure, depend actin cable            Brandt et al. 2008; Campos et al. 2009), screen is <br /> formation contraction (Kiehart et al. 2000; Jacinto          labor intensive. However, sys- <br /> et al. 2002; Wood et al. 2002), larval wound closure pro-        tematic vivo approach screening postembry- <br /> ceeds primarily process directed cell migra-        onic wound closure defects, holds great promise for <br /> tion (Wu et al. 2009). Although actin accumulates         identifying set evolutionarily conserved genes <br /> larval wound margin (Wu et al. 2009; Kwon et al. 2010),          required tissue autonomously wound closure. This is <br /> this accumulation discontinuous filopodial            especially case given numerous recent studies <br /> lamellipodial process extension apparent          (Li et al. 2003; Galko Krasnow 2004; Mace et al. <br /> the relatively early stages closure (Wu et al. 2009). Given   2005; Pujol et al. 2008; Tong et al. 2009; Wang et al. <br /> these differences cytoskeletal dynamics DC            2009; Wu et al. 2009) point deep evolutionary <br /> and larval embryonic wound closure,                conservation epidermal wound healing responses. <br /> expect differences set genes required regulate        We thank Georg Halder, Randy Johnson, Andreas Bergmann, and <br /> actin polymerization. Indeed, observe. At        members Galko laboratory comments; Shana Palla for <br /> the level Rho-GTPases, DC embryonic wound            assistance statistical analysis; Beth Stronach mutant strains; <br /> closure require Rac1, RhoA, Cdc42 (Harden et al.             Amin Ghabrial UAS-GFPRNAi; Leisa McCord graphical help; <br />                                                                  LaGina Nosavanh Denise Weathersby help screening; and <br /> 1999; Stramer Martin 2005), larval wound             Developmental Studies Hybridoma Bank Dirk Bohmann for <br /> closure requires Rac1 Cdc42 (Figure 3C Figure            antibodies. UAS-RNAi lines generously provided advance of <br /> S2) RhoA (data shown).                               public distribution Ulrich Theopold Ryu Ueda. This work was <br />    Although Arc-p20, Arp2/3 complex component,                supported University Texas MD Anderson Cancer Center <br /> has recently shown required embryonic             institutional research grant National Institutes Health grants <br />                                                                  1 R01GM083031 R01GM083031-02 S1 (American Recovery and <br /> wound closure (Campos et al. 2009), Arp2/3 complex           Reinvestment Act supplement preceding grant) M.J.G. <br /> has extensively analyzed date DC or <br /> embryonic wound closure. Here, one <br /> component complex (Arp 14D) required for <br />                                                                                        LITERATURE CITED <br /> larval wound closure, SCAR, activator the <br /> complex. Both genes fall morphologically           Baek, S. H., Y. C. Kwon, H. Lee K. M. Choe, 2010 Rho-family <br />                                                                      small GTPases required cell polarization directional <br /> class III, suggests act positively       sensing Drosophila wound healing. Biochem. Biophys. Res. <br /> regulate epidermal cell movement wound gap                  Commun. 394: 488&#226;&#8364;&#8220;492. <br /> but movement epidermal sheet. A             Bohmann, D., M. C. Ellis, L. M. Staszewski M. Mlodzik, <br /> gene class, previously implicated control cell         1994 Drosophila Jun mediates Ras-dependent photoreceptor <br />                                                                      determination. Cell 78: 973&#226;&#8364;&#8220;986. <br /> morphology in vitro RNAi screen (Kiger et al. 2003),       Boutros, M., H. Agaisse N. Perrimon, 2002 Sequential activa- <br /> is Gg1, suggests possible connection                 tion signaling pathways innate immune responses in <br /> G-protein&#226;&#8364;&#8220;coupled receptor signaling regulation               Drosophila. Dev. Cell 3: 711&#226;&#8364;&#8220;722. <br />                                                                  Brand, A. H., N. Perrimon, 1993 Targeted gene expression a <br /> cytoskeletal dynamics epithelial cells. <br />                                                                      means altering cell fates generating dominant pheno- <br />    Future screening prospects: A systematic approach                 types. Development 118: 401&#226;&#8364;&#8220;415. <br /> for identifying genes required postembryonic                 Brandt, S. M., G. Jaramillo-Gutierrez, S. Kumar, C. Barillas- <br /> wound closure long lacking field                  Mury D. S. Schneider, 2008 Use Drosophila model <br />                                                                      identify genes regulating Plasmodium growth mosquito. <br /> tissue repair. This partly complexities the <br />                                                                      Genetics 180: 1671&#226;&#8364;&#8220;1678. <br /> wound healing response vertebrates time           Campos, I., J. A. Geiger, A. C. Santos, V. Carlos A. Jacinto, <br /> cost large-scale screens vertebrate models                  2009 Genetic screen Drosophila melanogaster uncovers novel <br />  956                                                                 C. Lesch et al. <br />  <br />      set genes required embryonic epithelial repair. Genetics 184:     Llense, F., E. Martin-Blanco, 2008 JNK signaling controls <br />      129&#226;&#8364;&#8220;140.                                                                     border cell cluster integrity collective cell migration. Curr. <br /> Chihara, T., K. Kato, M. Taniguchi, J. Ng S. Hayashi,                         Biol. 18: 538&#226;&#8364;&#8220;544. <br />      2003 Rac promotes epithelial cell rearrangement tracheal          Luo, L., Y. J. Liao, L. Y. Jan Y. N. Jan, 1994 Distinct morphoge- <br />      tubulogenesis Drosophila. Development 130: 1419&#226;&#8364;&#8220;1428.                     netic functions similar small GTPases: Drosophila Drac1 in- <br /> Clark, R. A., J. M. Lanigan, P. DellaPelle, E. Manseau, H. F. Dvorak              volved axonal outgrowth myoblast fusion. Genes Dev. 8: <br />      et al., 1982 Fibronectin fibrin provide provisional matrix             1787&#226;&#8364;&#8220;1802. <br />      epidermal cell migration wound reepithelialization.           Mace, K. A., J. C. Pearson W. McGinnis, 2005 An epidermal <br />      J. Invest. Dermatol. 79: 264&#226;&#8364;&#8220;269.                                            barrier wound repair pathway Drosophila mediated by <br /> Dietzl, G., D. Chen, F. Schnorrer, K. C. Su, Y. Barinova et al.,                  grainy head. Science 308: 381&#226;&#8364;&#8220;385. <br />      2007 A genome-wide transgenic RNAi library conditional               Martin, P., 1997 Wound healing&#226;&#8364;&#8220;aiming perfect skin regenera- <br />      gene inactivation Drosophila. Nature 448: 151&#226;&#8364;&#8220;156.                        tion. Science 276: 75&#226;&#8364;&#8220;81. <br /> Eaton, S., P. Auvinen, L. Luo, Y. N. Jan K. Simons, 1995 CDC42            Monypenny, J., D. Zicha, C. Higashida, F. Oceguera-Yanez, <br />      Rac1 control different actin-dependent processes Dro-             S. Narumiya et al., 2009 Cdc42 Rac family GTPases regu- <br />      sophila wing disc epithelium. J. Cell Biol. 131: 151&#226;&#8364;&#8220;164.                    late mode speed direction primary fibroblast mi- <br /> Galko, M. J., M. A. Krasnow, 2004 Cellular genetic anal-                  gration platelet-derived growth factor-dependent <br />      ysis wound healing Drosophila larvae. PLoS Biol. 2: E239.              chemotaxis. Mol. Cell. Biol. 29: 2730&#226;&#8364;&#8220;2747. <br /> Geuking, P., R. Narasimamurthy, B. Lemaitre, K. Basler                    Mummery-Widmer, J. L., M. Yamazaki, T. Stoeger, M. Novatchkova, <br />      F. Leulier, 2009 A non-redundant role Drosophila Mkk4                    S. Bhalerao et al., 2009 Genome-wide analysis Notch signal- <br />      hemipterous/Mkk7 TAK1-mediated activation JNK.                     ling Drosophila transgenic RNAi. Nature 458: 987&#226;&#8364;&#8220;992. <br />      PLoS ONE 4: e7709.                                                       Nobes, C. D., A. Hall, 1995 Rho, rac, cdc42 GTPases regu- <br /> Glise, B., H. Bourbon S. Noselli, 1995 hemipterous encodes                    late assembly multimolecular focal complexes associated <br />      novel Drosophila MAP kinase kinase, required epithelial                actin stress fibers, lamellipodia, filopodia. Cell 81: 53&#226;&#8364;&#8220;62. <br />      cell sheet movement. Cell 83: 451&#226;&#8364;&#8220;461.                                   Odland, G., R. Ross, 1968 Human wound repair. I. Epidermal <br /> Goley, E. D., M. D. Welch, 2006 The ARP2/3 complex: ac-                    regeneration. J. Cell Biol. 39: 135&#226;&#8364;&#8220;151. <br />      tin nucleator comes age. Nat. Rev. Mol. Cell. Biol. 7: 713&#226;&#8364;&#8220;726.       Otto, I. M., T. Raabe, U. E. Rennefahrt, P. Bork, U. R. Rapp et al., <br /> Grose, R., S. Werner, 2004 Wound-healing studies trans-                    2000 The p150-Spir protein provides link c-Jun N-terminal <br />      genic <span id='am-164' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-165' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span>knockout</span> mice. Mol. Biotechnol. 28: 147&#226;&#8364;&#8220;166.                       kinase function actin reorganization. Curr. Biol. 10: 345&#226;&#8364;&#8220;348. <br /> Gumienny, T. L., E. Brugnera, A. C. Tosello-Trampont, J. M.                   Patel, N. H., P. M. Snow C. S. Goodman, 1987 Characteriza- <br />      Kinchen, L. B. Haney et al., 2001 CED-12/ELMO, novel                       tion cloning fasciclin III: glycoprotein expressed sub- <br />      member CrkII/Dock180/Rac pathway, required for <br />                                                                                   set neurons axon pathways Drosophila. Cell 48: 975&#226;&#8364;&#8220;988. <br />      phagocytosis cell migration. Cell 107: 27&#226;&#8364;&#8220;41.                        Pearson, J. C., M. T. Juarez, M. Kim W. McGinnis, 2009 Multi- <br /> Harden, N., M. Ricos, Y. M. Ong, W. Chia L. Lim, 1999 Participa- <br />                                                                                   ple transcription factor codes activate epidermal wound- <br />      tion small GTPases dorsal closure Drosophila em- <br />                                                                                   response genes Drosophila. Proc. Natl. Acad. Sci. USA 106: <br />      bryo: distinct roles Rho subfamily proteins epithelial <br />                                                                                   2224&#226;&#8364;&#8220;2229. <br />      morphogenesis. J. Cell Sci. 112(Pt. 3): 273&#226;&#8364;&#8220;284. <br />                                                                               Polaski, S., L. Whitney, B. W. Barker B. Stronach, 2006 <br /> Igaki, T., 2009 Correcting developmental errors apoptosis: les- <br />                                                                                   Genetic analysis slipper/mixed lineage kinase reveals re- <br />      sons Drosophila JNK signaling. Apoptosis 14: 1021&#226;&#8364;&#8220;1028. <br />                                                                                   quirements multiple Jun-N-terminal kinase-dependent mor- <br /> Ishimaru, S., R. Ueda, Y. Hinohara, M. Ohtani H. Hanafusa, <br />      2004 PVR plays critical role JNK activation thorax clo-             phogenetic events Drosophila development. Genetics <br />      sure Drosophila metamorphosis. EMBO J. 23: 3984&#226;&#8364;&#8220;3994.                 174: 719&#226;&#8364;&#8220;733. <br /> Jacinto, A., W. Wood, S. Woolner, C. Hiley, L. Turner et al.,                 Pollard, T. D., G. G. Borisy, 2003 Cellular motility driven by <br />      2002 Dynamic analysis actin cable function Drosoph-                assembly disassembly actin filaments. Cell 112: 453&#226;&#8364;&#8220;465. <br />      ila dorsal closure. Curr. Biol. 12: 1245&#226;&#8364;&#8220;1250.                           Pujol, N., S. Cypowyj, K. Ziegler, A. Millet, A. Astrain et al., <br /> Kambris, Z., S. Brun, I. H. Jang, H. J. Nam, Y. Romeo et al.,                     2008 Distinct innate immune responses infection and <br />      2006 Drosophila immunity: large-scale vivo RNAi screen                  wounding C. elegans epidermis. Curr. Biol. 18: 481&#226;&#8364;&#8220;489. <br />      identifies serine proteases required Toll activation. Curr.     Ramet, M., R. Lanot, D. Zachary P. Manfruelli, 2002 JNK <br />      Biol. 16: 808&#226;&#8364;&#8220;813.                                                           signaling pathway required efficient wound healing Dro- <br /> Kennerdell, J. R., R. W. Carthew, 2000 Heritable gene silenc-                 sophila. Dev. Biol. 241: 145&#226;&#8364;&#8220;156. <br />      ing Drosophila using double-stranded RNA. Nat. Biotechnol.            Riesgo-Escovar, J. R., E. Hafen, 1997 Common distinct <br />      18: 896&#226;&#8364;&#8220;898.                                                                 roles DFos DJun Drosophila development. Science <br /> Kiehart, D. P., C. G. Galbraith, K. A. Edwards, W. L. Rickoll                 278: 669&#226;&#8364;&#8220;672. <br />      R. A. Montague, 2000 Multiple forces contribute cell sheet            Riesgo-Escovar, J. R., M. Jenni, A. Fritz E. Hafen, 1996 The <br />      morphogenesis dorsal closure Drosophila. J. Cell Biol. 149:           Drosophila Jun-N-terminal kinase required cell morpho- <br />      471&#226;&#8364;&#8220;490.                                                                     genesis DJun-dependent cell fate specification in <br /> Kiger, A. A., B. Baum, S. Jones, M. R. Jones, A. Coulson et al.,                  eye. Genes Dev. 10: 2759&#226;&#8364;&#8220;2768. <br />      2003 A functional genomic analysis cell morphology using              Silverman, N., R. Zhou, R. L. Erlich, M. Hunter, E. Bernstein <br />      <span id='am-242' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-243' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-250' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-256' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-258' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-262' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-270' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-278' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-284' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-290' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-292' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-295' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-302' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-308' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-315' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-316' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-318' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-319' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span>RNA interference</span>. J. Biol. 2: 27.                                            et al., 2003 Immune activation NF-kappaB JNK requires <br /> Kockel, L., J. G. Homsy D. Bohmann, 2001 Drosophila AP-1:                     Drosophila TAK1. J. Biol. Chem. 278: 48928&#226;&#8364;&#8220;48934. <br />      lessons invertebrate. Oncogene 20: 2347&#226;&#8364;&#8220;2364.                    Simpson, K. J., L. M. Selfors, J. Bui, A. Reynolds, D. Leake et al., <br /> Kwon, Y. C., S. H. Baek, H. Lee K. M. Choe, 2010 Nonmuscle                    2008 Identification genes regulate epithelial cell migra- <br />      myosin II localization regulated JNK Drosophila                 tion using siRNA screening approach. Nat. Cell Biol. 10: <br />      larval wound healing. Biochem. Biophys. Res. Commun. 393:                    1027&#226;&#8364;&#8220;1038. <br />      656&#226;&#8364;&#8220;661.                                                                 Singer, A. J., R. A. Clark, 1999 Cutaneous wound healing. <br /> Lawrence, P. A., R. Bodmer J. P. Vincent, 1995 Segmental pat-                 N. Engl. J. Med. 341: 738&#226;&#8364;&#8220;746. <br />      terning heart precursors Drosophila. Development 121:              Sluss, H. K., Z. Han, T. Barrett, R. J. Davis Y. T. Ip, 1996 A JNK <br />      4303&#226;&#8364;&#8220;4308.                                                                   signal transduction pathway mediates morphogenesis an <br /> Levi, B. P., A. S. Ghabrial M. A. Krasnow, 2006 Drosophila                    immune response Drosophila. Genes Dev. 10: 2745&#226;&#8364;&#8220;2758. <br />      talin integrin genes required maintenance tracheal        Spradling, A. C., D. Stern, A. Beaton, E. J. Rhem, T. Laverty et al., <br />      terminal branches luminal organization. Development 133:                 1999 The Berkeley Drosophila Genome Project gene disrup- <br />      2383&#226;&#8364;&#8220;2393.                                                                   tion project: single P-element insertions mutating 25% vital <br /> Li, G., C. Gustafson-Brown, S. K. Hanks, K. Nason, J. M. Arbeit                   Drosophila genes. Genetics 153: 135&#226;&#8364;&#8220;177. <br />      et al., 2003 c-Jun essential organization epidermal        Stramer, B., P. Martin, 2005 Cell biology: master regulators of <br />      leading edge. Dev. Cell 4: 865&#226;&#8364;&#8220;877.                                          sealing healing. Curr. Biol. 15: R425&#226;&#8364;&#8220;R427. <br />                                                          Drosophila Wound Closure Screen                                                      957 <br />  <br /> Stronach, B., N. Perrimon, 2002 Activation JNK path-              Wood, W., A. Jacinto, R. Grose, S. Woolner, J. Gale et al., <br />      way dorsal closure Drosophila requires mixed lin-             2002 Wound healing recapitulates morphogenesis Drosoph- <br />      eage kinase, slipper. Genes Dev. 16: 377&#226;&#8364;&#8220;387.                               ila embryos. Nat. Cell Biol. 4: 907&#226;&#8364;&#8220;912. <br /> Su, Y. C., J. E. Treisman E. Y. Skolnik, 1998 The Drosophila             Woolner, S., A. Jacinto P. Martin, 2005 The small GTPase <br />      Ste20-related kinase misshapen required embryonic dorsal             Rac plays multiple roles epithelial sheet fusion&#226;&#8364;&#8220;dynamic stud- <br />      closure acts JNK MAPK module evolution-                 ies Drosophila dorsal closure. Dev. Biol. 282: 163&#226;&#8364;&#8220;173. <br />      arily conserved signaling pathway. Genes Dev. 12: 2371&#226;&#8364;&#8220;2380.            Wu, Y., A. R. Brock, Y. Wang, K. Fujitani, R. Ueda et al., 2009 A <br /> Tong, A., G. Lynn, V. Ngo, D. Wong, S. L. Moseley et al.,                        blood-borne PDGF/VEGF-like ligand initiates wound-induced <br />      2009 Negative regulation Caenorhabditis elegans epidermal                epidermal cell migration Drosophila larvae. Curr. Biol. 19: <br />      damage responses death-associated protein kinase. Proc. Natl.            1473&#226;&#8364;&#8220;1477. <br />                                                                              Xia, Y., M. Karin, 2004 The control cell motility epi- <br />      Acad. Sci. USA 106: 1457&#226;&#8364;&#8220;1461. <br />                                                                                  thelial morphogenesis Jun kinases. Trends Cell Biol. 14: <br /> Treisman, J. E., N. Ito G. M. Rubin, 1997 misshapen encodes a <br />                                                                                  94&#226;&#8364;&#8220;101. <br />      protein kinase involved cell shape control Drosophila. Gene <br />                                                                              Zallen, J. A., Y. Cohen, A. M. Hudson, L. Cooley, E. Wieschaus <br />      186: 119&#226;&#8364;&#8220;125.                                                               et al., 2002 SCAR primary regulator Arp2/3-dependent <br /> Vitorino, P., T. Meyer, 2008 Modular control endothelial <br />                                                                                  morphological events Drosophila. J. Cell Biol. 156: 689&#226;&#8364;&#8220;701. <br />      sheet migration. Genes Dev. 22: 3268&#226;&#8364;&#8220;3281.                              Zeitlinger, J., D. Bohmann, 1999 Thorax closure Drosoph- <br /> Wang, S., V. Tsarouhas, N. Xylourgidis, N. Sabri, K. Tiklova et al., <br />                                                                                  ila: involvement Fos JNK pathway. Development 126: <br />      2009 The tyrosine kinase Stitcher activates Grainy head epi-            3947&#226;&#8364;&#8220;3956. <br />      dermal wound healing Drosophila. Nat. Cell Biol. 11: 890&#226;&#8364;&#8220;895.        Zeitlinger, J., L. Kockel, F. A. Peverali, D. B. Jackson, M. Mlodzik <br /> Wang, X., L. He, Y. I. Wu, K. M. Hahn D. J. Montell, 2010 Light-             et al., 1997 Defective dorsal closure loss epidermal de- <br />      mediated activation reveals key role Rac collective guidance       capentaplegic expression Drosophila fos mutants. EMBO J. <br />      cell movement vivo. Nat. Cell Biol. 12: 591&#226;&#8364;&#8220;597.                      16: 7393&#226;&#8364;&#8220;7401. <br /> Welch, M. D., A. Iwamatsu T. J. Mitchison, 1997 Actin poly- <br />      merization induced Arp2/3 protein complex surface <br />      Listeria monocytogenes. Nature 385: 265&#226;&#8364;&#8220;269.                                                    Communicating editor: D. I. Greenstein <br />                 GENETICS <br />                        Supporting Information <br />   http://www.genetics.org/cgi/content/full/genetics.110.121822/DC1 <br />  <br />  <br />  <br /> A Targeted UAS-RNAi Screen Drosophila Larvae Identifies Wound <br />        Closure Genes Regulating Distinct Cellular Processes <br />  <br />  <br />  <br />    Christine Lesch, Juyeon Jo, Yujane Wu, Greg S. Fish Michael J. Galko <br />  <br />  <br />  <br />  <br />           Copyright &#195;&#8220; 2010 Genetics Society America <br />                   DOI: 10.1534/genetics.110.121822 <br />  2 SI                                                     C. Lesch et al. <br />  <br />  <br />  <br />  <br />   <br />    FIGURE S1. &#226;&#8364;&#8221;Relative RNAi potency epidermal wound reporters. Flies bearing e22c-Gal4-, A58-Gal4-, Dcr-2;A58- <br /> Gal4-based reporters crossed w1118 (control) UAS-GFPRNAi assess level knockdown UAS-src-GFP <br /> transgene expression progeny larvae. In indicated columns larval mounts immunostained Fasciclin III <br /> (blue) visualized nuclear DsRed2-Nuc (red) membane src-GFP (green), markers (merge). In case was <br /> Fasciclin III DsRed2-Nuc expression affected. Relative w1118 controls, knockdown src-GFP expression most <br /> pronounced e22c-Gal4 Dcr-2;A58-Gal4 reporters, weaker knockdown observed A58-Gal4 reporter. <br /> Scale bar 100 &#206;&#188;m. <br />                                                            C. Lesch et al.                                                    3 SI <br />  <br />  <br />  <br />  <br />   <br />  <br />  <br />  <br />  <br />                                                                                                                                        <br />   <br />     FIGURE S2. &#226;&#8364;&#8221;Quantification wound closure alternative RNAi lines, dominant-negative transgenes, mutants. (A) <br /> Quantification wound closure epidermal expression indicated UAS-RNAi transgenes (original RNAi line, open bar; <br /> non-overlapping RNAi line, diagonal striped hash; overlapping RNAi line, dotted hash; larval viable mutants, wavy hash; <br /> dominant-negative transgenes, diamond hash). (A) JNK pathway candidate genes. Non-overlapping lines targeting misshapen, <br /> slipper, Tak1, Mkk4, DFos/kay open wound phenotypes, overlapping lines targeting Mkk4 DJun/Jra and <br /> larval viable mutations slipper Tak1. (B) Actin cytoskeletal dynamics candidate genes. Non-overlapping lines targeting Ced- <br /> 12, mbc, Arp14D, Arp11, open wound phenotypes, overlapping lines targeting Ced-12, SCAR, Arp11, and <br /> dominant-negative transgenes targeting Rac1 Cdc42. Absence bar indicates line, transgene, mutant available <br /> or tested gene. The number scored larvae RNAi knockdown using e22c-Gal4 A B follows <br /> (original RNAi lines column 3 Table S1): n non-overlapping RNAi lines: msn = 39, slpr = 40, Tak1 = 43, Mkk4 = 40, <br /> DFos/kay = 30, Ced-12 = 37, mbc = 43, Arp14D = 48; n overlapping RNAi lines: slpr = 30, Mkk4 = 34, DJun/Jra = 33, Ced-12 = <br /> 37, SCAR = 31; n mutants: slpr = 9, Tak1 = 8; n DN versions: Rac1 = 36, Cdc42 = 34. The number scored larvae for <br /> each RNAi knockdown using Dcr-2;A58 B follows (original RNA line legend Figure 3): n non-overlapping <br /> RNAi line: Arp11 = 36; n overlapping RNAi line: Arp11 = 42. <br />  4 SI                                                      C. Lesch et al. <br />  <br />  <br />  <br />  <br />   <br />  <br />  <br />  <br />  <br />                                                                                                <br />   <br />     FIGURE S3. &#226;&#8364;&#8221;Protein knockdown DFos/kayRNAi DJun/JraRNAi lines. Dissected larval mounts expressing UAS- <br /> src-GFP (green) immunostained indicated antibodies (red). (A) Control larva immunostained goat anti-rabbit- <br /> Cy3 secondary antibody. (B) All larvae immunostained anti-DFos. Controls analyzed mock wounding 6 <br /> hours post puncture wounding DFos/kayRNAi-expressing larvae analyzed 6 hours post puncture wounding. DFos is <br /> faintly expressed nuclei unwounded epidermal cells. This expression increases wounding efficiently knocked <br /> down expression DFos/kayRNAi. (C) All larvae immunostained anti-DJun. Controls analyzed mock <br /> wounding 4 hours post-wounding DJun/JraRNAi-expressing larvae analyzed 4 hours post wounding. DJun is <br /> expressed nuclei unwounded epidermal cells. This expression maintained wounding efficiently knocked <br /> down expression DJun/JraRNAi. Stars, puncture wound site. Scale bar, 20 &#206;&#188;m. <br />                                                          C. Lesch et al.                                                   5 SI <br />  <br />  <br />  <br />  <br />                                                                                                         <br />   <br />    FIGURE S4. &#226;&#8364;&#8221;Redundancy Jun2 Jun3 kinases. (A) Quantification wound closure epidermal expression the <br /> indicated UAS-RNAi transgenes using e22c reporter. Double knockdown hep Mkk4 Tak1 slpr gives higher <br /> percentage open wounds corresponding single knockdowns. Open bars, single RNAi; hashed bars, double RNAi. (B- <br /> E) Larvae heterozygous e22c-Gal4,UAS-DsRed2-Nuc indicated UAS-RNAi transgenes immunostained Fasciclin <br /> III (green) wounding (B, D) 24 hours wounding (C, E). (B) hep,Mkk4RNAi unwounded. (C) hep,Mkk4RNAi 24 hours <br /> post-wounding. (D) Tak1x2;slprRNAi unwounded. (E) Tak1x2;slprRNAi 24 hours post-wounding. Unwounded epidermal <br /> morphologies normal wounds average larger single knockdowns (see Figure 4D). The number scored <br /> larvae RNAi knockdowns targeting genes follows: n= 52 (hep,Mkk4), 32 (Tak1x2;slpr). For numbers <br /> of original RNAi lines Table S1. Scale bar 100 &#206;&#188;m. <br />  6 SI                                                     C. Lesch et al. <br />  <br />  <br />  <br />  <br />   <br />  <br />  <br />  <br />  <br />                                                                                                                         <br />   <br />    FIGURE S5. &#226;&#8364;&#8221;Early time points wound closure select UAS-RNAi lines. Larvae heterozygous e22c-Gal4,UAS-DsRed2- <br /> Nuc (red) indicated UAS-RNAi transgenes immunostained Fasciclin III (green) indicated timepoints. Scale <br /> bar 100 &#206;&#188;m. <br />                                                           C. Lesch et al.                                                      7 SI <br />  <br />  <br />  <br />  <br />   <br />  <br />  <br />  <br />  <br />                                                                                                                             <br />   <br />    FIGURE S6.&#226;&#8364;&#8221;Unwounded control epidermal sheets larvae expressing selected UAS-RNAi transgenes shown Figure 4. <br /> Dissected epidermal mounts unwounded larvae heterozygous e22c-Gal4 UAS-DsRed2-Nuc (red) expressing the <br /> indicated UAS-RNAi transgene immunostained Fasciclin III (green). DJun/JraRNAi- Ced-12RNAi-expressing larvae <br /> showed slightly disorganized epidermal sheet. In Rac1RNAi-expressing larvae epidermal sheet highly disorganized. Scale <br /> bar 100 &#206;&#188;m. <br />  8 SI                                                    C. Lesch et al. <br />  <br />  <br />  <br />  <br />                                                                                                                   <br />   <br />    FIGURE S7. &#226;&#8364;&#8221;Morphology alternative timepoints RNAi lines. Dissected larval mounts larvae expressing the <br /> indicated transgenes immunostained Fasciclin III (green) indicated timepoints. (A) Larva heterozygous e22c-Gal4,UAS- <br /> DsRed2-Nuc (red), msnRNAi x2 transgenes used Figure 4I immediately wounding. The wound gap normal and <br /> lacks nuclei. (B) Larva heterozygous e22c-Gal4,UAS-DsRed2-Nuc (red), msnRNAi (VDRC line 16973 #101517 Table <br /> S2) 24 hours post wounding. Class VII morphology multiple nuclei wound gap (star) similar NIG line <br /> (16973R-2;16973R-1) shown Figure 4I. (C) Larva heterozygous e22c-Gal4,UAS-DsRed2-Nuc (red), Arp14DRNAi (VDRC <br /> line 9901 #101999) 24 hours post wounding. Class III open wound morphology clustered nuclei (bracket) wound edge is <br /> similar knockdown class III genes (Figure 4E). Scale bar 100 &#206;&#188;m. <br />                                                            C. Lesch et al.                                                      9 SI <br />  <br />  <br />  <br />  <br />   <br />  <br />  <br />  <br />  <br />                                                                                                                <br />   <br />    FIGURE S8. &#226;&#8364;&#8221;Lack epidermal cell division expression select UAS-RNAi lines. Flies bearing e22c-Gal4,UAS-src-GFP <br /> were crossed w1118 (control) indicated UAS-RNAi transgenes stained anti-phospho-Histone H3 (red) to <br /> assess presence mitotically active cells. Expression UAS-DJun/JraRNAi, UAS-DFos/kayRNAi, UAS-msnRNAi x2 larval <br /> epidermis did affect high level staining observed imaginal tissue intense anti-phospho-Histone H3 staining <br /> was observed larval epidermis of genotypes wounding. Stars indicate pinch wound gaps in <br /> genotypes gave open wounds. Scale bar 20 &#206;&#188;m. <br />  10 SI                                           C. Lesch et al. <br />  <br />  <br />  <br />  <br />                                                     	         <br />  <br />                      <br />         <br />                                                        <br />            <br />  <br />                                           	        !                                 <br />            <br />       <br />          &quot;# <br />         <br />    <br />                                           <br /> #                  <br />    <br />   $ %                           <br />     <br />     ,,/4 *.,0,.                                            .                    !&quot; <br />   % <br />     ,..4 *2.01                                             -                     /.  <br />     ,.23 (,                                                &apos;&amp;                  &quot;   <br />     ,.23 (-                                                ,                   &quot;   <br />     ,.33 (,&amp;,.33 (-                                         ),                    , <br />     -+0/ (,                                                /                     &quot;- <br />     -+0/ (/                                                .                     &quot;- <br />     -,,/ (,                                                .                      <br />     -,,/ (-                                                ,                      <br />     -,4+ (-&apos;/.0. (-                                         )(                      <br />     -,43 *--4//                                            &apos;.                      <br />     --/3 (,                                                 )+                     , <br />     --2- *..0,1                                            )*                  !   <br />     --2- *..0,3                                            )&amp;                  !   <br />     --2- (,                                                 *&amp;                  !   <br />     --2- (-                                                )&apos;                  !   <br />     --2- (-&apos;--2- (,                                        ),                  !   <br />     --20 (-                                                 *.                    <br /> # ) <br />    <br />     -0-- (,                                                &apos;&amp;                   &quot; (   <br />     .,2, (-                                                &apos;&amp;                    , <br />     .,2, (.                          closed                 (,                    , <br />     .-,- *,.+.+                                            &apos;&amp;                    ( 	  <br />     ..2. (-                                                .                     # <br />     /+44 */441/                                            ,                     ( 	 <br />     /.3+ (,                                                -                   #!  <br />     /.3+ (.                                                -                   #!  <br />     //20 (,                                                /                   	   - <br />     /0-, (,                                                ,                    &quot;  , <br />     /1.1 (,                                                 )-                       <br />     /1.2 *,/+-                                             +                      <br />     /2-+ *./34,                                            ).                   <br />  4-  <br />     /2-+ *./34-                                            ).                   <br />  4-  <br />     /3+. *./343                                            )&apos;                     - <br />     /334 *,..0-                                            .                   $  <br />     /40+ *44.,                                             +                     /40+ <br />                                C. Lesch et al.                    11 SI <br />  <br />  <br />  <br />  <br /> *%%- #&amp;            	                                	  <br /> ( <br /> *%%- #&apos;            	                                	  <br /> ( <br /> *&apos;&amp;&apos; #&amp;            	                                <br />    <br /> *&apos;&amp;&apos; #(            	                                <br />    <br /> *((+ #&apos;               <br />                                 #&amp;&apos; <br /> *((+ #(               <br />                                 #&amp;&apos; <br /> *()* #&apos;            	                                  **  <br /> *()* #(            	                                  **  <br /> *(,&apos; #&apos;            	                                 <br />  * <br /> **&apos;- #&apos;            	                                <br />    #. <br /> **&apos;- #(            	                                <br />    #. <br /> **+&apos; #&amp;            	                                   <br /> **+&apos; #(            	                                   <br /> *+&amp;% #&amp;            	                                    #.+   <br /> *+&amp;% #&apos;            	                                    #.+   <br /> *+-% #&amp;&quot;*+-% #&apos;       <br />    <br />   <br />  	                        <br /> *,(% #&apos;            	                                     <br /> *.-- $&amp;)++)        	                               !  &apos; <br /> *..( $(&apos;-&apos;         	                                  <br /> +&amp;&apos;) $)(%&amp;         	                                     <br /> +*&amp;* #&amp;            	                                <br />    -+  <br /> +*&amp;* #(            	                                <br />    -+  <br /> +*&amp;, #(            	                                 &amp;- <br /> +,&amp;( $&apos;,,&apos;*        	                               	   <br /> +-.% $&amp;(*).        	                                <br />      <br /> +-.% $&apos;,%..        	                                <br />      <br /> +-.% $.)(&amp;         	                                <br />      <br /> +.*( $&amp;*(%         	                                  #        <br /> ,%%&apos; #&apos;            	                                   <br /> ,%%&apos; #,            	                                   <br /> ,%%&apos; #,            	                                   <br /> ,%+- #&apos;            <br />            <br />   <br />                     <br />       <br /> ,%+- #(            <br />            <br />   <br />                     <br />       <br /> ,&amp;&apos;&amp; #(            	                                <br />      <br /> ,&amp;&apos;&amp; #)            	                                <br />      <br /> ,&apos;*% $&apos;,&amp;%&apos;        	                                <br />    #+ <br /> ,&apos;*% $&apos;,&amp;%(        	                                <br />    #+ <br /> ,&apos;*% $.&apos;-%         	                                <br />    #+ <br /> ,(*+ #&amp;            	                                <br />   <br /> ,(*+ #&apos;            	                                <br />   <br />  12 SI                          C. Lesch et al. <br />  <br />  <br />  <br />  <br />     .--&apos; &amp;(+*.0     	                               $! <br />     ..(. &amp;),,)/     	                               <br />    ( <br />     ./., <br /> %*         	                              !   <br />     /&apos;0, <br /> %)         	                                  <br />     /)-( <br /> %(            <br />                               ( <br />     /)/, <br /> %(         	                                 <br />     /,.. <br /> %(         	                                 <br />  %  (  <br />     /,.. <br /> %)         	                                 <br />  %  (  <br />     /,0, <br /> %(         	                                  %. <br />     /,0, <br /> %)         	                                  %. <br />     //,- <br /> %(         	                                %    <br />     //,- <br /> %)         	                               %    <br />     //0- <br /> %)         	                              (/# <br />     //0- <br /> %*         	                              (/# <br />     0-)* <br /> %(         <br />            <br />   <br />                      <br />     0-)* <br /> %)         	                                 <br />     0--/ <br /> %(         	                               <br />  + <br />     0--/ <br /> %)         	                               <br />  + <br />     0.*/ <br /> %(            <br />                              <br />   + <br />     0.*/ <br /> %*            <br />                              <br />   + <br />     00&apos;( <br /> %)            <br />                                (+  <br />     (&apos;&apos;.- <br /> %(        	                                   <br />     (&apos;)++ <br /> %(        	                                0-   <br />     (&apos;)++ <br /> %)        	                                0-   <br />     (&apos;*-* &amp;(*+--    	                                   <br />     (&apos;*.0 <br /> %(           <br />                                 <br />     (&apos;+/( &amp;0*()     	                                (&apos;+/( <br />     (&apos;,0* <br /> %(        	                                   <br />     (&apos;-() &amp;(*,-)    	                                /*  <br />     (&apos;-)- &amp;))/+,    	                             	   <br />     (&apos;-0/ &amp;++*(&apos;    	                                <br />   <br />    <br />     (&apos;/)* &amp;(./*     	                                  <br />     (&apos;/// <br /> %(        	                               <br />  * <br />     (&apos;/// <br /> %)        	                               <br />  * <br />     ((&apos;-+ <br /> %*        	                               <br />      <br />     ((&apos;00 &amp;0).)     	                                ((&apos;00 <br />     (((++ <br /> %)        	                                &quot; <br />   <br />     ((**, <br /> %(        	                                $ <br />     ((,-( &amp;0,+)     	                                  <br />     ()(0* &amp;+0/*,    	                                ))  <br />                                   C. Lesch et al.                13 SI <br />  <br />  <br />  <br />  <br /> +,,*2 %,    --+2-     	                                .0 <br /> +,,-/ %+              	                                 ++ <br /> +,,-/ %-              	                                 ++ <br /> +,-03 )-//,.          <br />            <br />   <br />                      <br /> +,-1* ).--+-          	                                +,-1* <br /> +,/*+ )+--12          	                              	 /0  <br /> +,/-* %,#+,/-* %-     	                                ., <br /> +,031 )+-/1-          	                              	 +-  <br /> +,1/. )3//+           	                              	 .,  <br /> +,1// %+              	                               &apos;-(    <br /> +-+/2 ),..32          	                              	 .3  <br /> +-,*0 )3-/.           	                              	 .1  <br /> +-,,/ )/+-2-          	                              	 .1  <br /> +-1,0 ),11-           	                              	 1.  <br /> +-310 )+,31           	                                32  <br /> +-33/ ).,/,.          	                                +-33/ <br /> +.*,0 )20,            	                                 <br /> +.,,0 )+31+1          	                                   <br /> +.,,0 )-0-//          	                                   <br /> +.,,0 )-0-/0          	                                   <br /> +.-0* ),02*/          	                              	 22  <br /> +.1./ %,              	                               <br />    <br /> %  , <br /> +.321 ),3.,,          	                                0.  <br /> +/,3- %-              	                                +/,3- <br /> +/.-1 %+              	                                     <br /> +/.-2 %,              	                                +/.-2 <br /> +//*3 %,                 <br />                                 &amp;  ! <br /> +/0+. ).3,1           	                                +/0+. <br /> +/1.. )+*30           	                                +/1.. <br /> +/113 )+,-.1          	                               <br />    <br /> +/13- ).**,0          	                                  + <br /> +/2,/ %,              	                                  <br /> +01.* %.              	                                , <br /> +012/ %+              	                               &quot;- <br /> +02,1 %,              	                                <br />  . <br /> +02,1 %-              	                                <br />  . <br /> +030* )+.0+           	                              	 --  <br /> +030+ )1/2-           	                              	 --  <br /> +031- %,$+031- %+        <br />                                 <br /> +033, ).13.2          	                                  0 <br />  14 SI                                                  C. Lesch et al. <br />  <br />  <br />  <br />  <br />     &apos;-&amp;.* $&apos;                              	     <br />                    *           / <br />     &apos;-&amp;.* $)                              	     <br />                    *           / <br />     &apos;-((, %))&apos;(&apos;                          	     <br />                    +       	 +/  <br />     &apos;-)*. $&apos;                              	     <br />                    )           <br />     &apos;-)*. $(                              	     <br />                    +           <br />     &apos;-++/ $)                              	     <br />                    *           <br />     &apos;-,/- %*)&amp;-+                          	     <br />                    ##       &quot; <br />     &apos;--/+ %(,.&apos;+                          	     <br />                    *           ( <br />     &apos;-.&amp;&amp; %),())                          	     <br />                    &apos;             <br />     &apos;.&amp;/, %)&amp;.-)                          	     <br />                    *        <br />     <br />     &apos;.(*&apos; %*-/,,                          	     <br />                    *        <br />    $* <br />     &apos;.(*&apos; %*-/,-                          	     <br />                    *        <br />    $* <br />     &apos;.)*+ %)++-&apos;                          	     <br />                    ##           <br />     &apos;.*-* $(                              	     <br />                    +        $   $   <br />     &apos;.*-* $)                              	     <br />                    #&quot;       $   $   <br />     &apos;..+/ %&apos;))/+                          	     <br />                    *       	 &apos;/  <br />     )&apos;&amp;,&apos; %*)/.                           	     <br />                    *         /.  <br />     )&apos;(.&amp; %/+)-                           	     <br />                    +         /*  <br />     )&apos;))+ %,.&apos;)                           	     <br />                    *         /)  <br />     )&apos;*(&apos; %(+-,&amp;                          	     <br />                    %) 	     <br />   &apos; <br />     )&apos;,(( %.,.+                           	     <br />                    +         )/  <br />     )&apos;,++ %*+..)    *(.&amp;                  	     <br />                    *          <br />     )&apos;,,&amp; %-.+(                           	     <br />                    *         )&apos;,,&amp; <br />     )&apos;-(&amp; %+&apos;*,&apos;                          	     <br />                    *         )&apos;-(&amp; <br />     )&apos;-*- %*.&amp;&apos;.                          	     <br />                    (         ),  <br />     )&apos;-,&amp; %-,.,                           	     <br />                    +         )&apos;-,&amp; <br />     )&apos;-.) $&apos;                              	     <br />                    #&amp;            <br />     )&apos;-.) $(                              	     <br />                    (             <br />     )&apos;-/*   <br />        &apos;#  <br />        )           	     <br />                    %+ 	     <br />  ! <br />     )&apos;/,( %&apos;)&amp;&apos;                           	     <br />                    *         $     <br />     )&apos;/,( %&apos;)&amp;)(                          	     <br />                    +         $     <br />     )()+, $(                              	     <br />                    ##          &apos; <br />     )()+, $)                              	     <br />                    $&quot;          &apos; <br />     )(*-, $&apos;                              	     <br />                    #&quot;          &apos;* <br />     )(*-, $)                              	     <br />                    #&quot;          &apos;* <br />     ))+*( %(-&apos;),                          	     <br />                    #&quot;         ) <br />  <br />                 <br />       <br />              #&apos; ,           <br />    <br />  <br />  <br />        $&quot;&quot;+!#&quot;         <br />           $&quot;    $&quot;&quot;+  <br />  <br />  <br />    	     <br />   <br />      <br />                   <br />      <br />           <br />                                                    C. Lesch et al.   15 SI <br />  <br />  <br />  <br />  <br />    <br />                   	  <br />  <br />  <br />   	                        %   <br />    %   <br />  <br />  <br />   <br />  <br />  <br />   &quot; <br />  ( &apos;&amp;#   <br />        <br />    <br />     <br />               <br />  <br /> $       <br />     <br />             <br />  <br />                       <br />    <br />  	  !          <br />  16 SI                                                                   C. Lesch et al. <br />  <br />  <br />  <br />  <br />                                                                          TABLE S2 <br />  <br />                                                               Alternative RNAi Lines Alleles <br />  <br />  <br />  <br />               	      <br />          	         <br />   #        !   +           	  % <br /> &quot;           	          <br /> &quot;           	                     !          	          <br />  <br />                          &amp;+.,( &quot;&apos;!&amp;+.,( &quot;&amp; &amp; &apos;&apos; $  *-&apos;             &amp;+.,( $&amp;%&amp;*&amp;, &amp; &apos;&apos; &apos;                                           <br />                          **.* (++,)- &amp;   &quot;&apos;! *-&apos;                   &apos;&apos;,&apos; &quot;&amp; &amp; &apos;&apos; &apos;                  &apos;&apos;,&apos; $((*&amp;- &amp;   &quot;&apos;! *-&apos;               %+  <br />                                                                    &apos;&apos;,&apos; &quot;&apos;                           <br />                                                                    &apos;&apos;,&apos; &quot;&apos;!&apos;&apos;,&apos; &quot;&amp; &amp; *-&apos;  <br />           <br />   &amp;            &amp;(-- &quot;&amp; &amp;(-- &quot;&apos; &amp; &apos;&apos; $  *-$    &quot;&apos;! *-&apos;    &amp;-).&apos; $&amp;%&amp;(*, &amp; &apos;&apos; &apos;                                          <br />   &amp;&apos;*&apos;,  <br />                          &apos;&amp;.% &quot;&apos;!)(*( &quot;&apos; &amp; &apos;&apos; $    &quot;&apos;! *-&apos;                                                                        <br />          	  )            .,(- &quot;( &amp; &apos;&apos; $    &quot;&apos;! *-&apos;                 .,(- $&amp;%-*+&amp; &amp; &apos;&apos; &apos;             .,(- &quot;&amp; &amp; &apos;&apos; $    &quot;&apos;! *-&apos;      <br />              #           &apos;&apos;,* &quot;&apos; &amp; &apos;&apos; $  *-&apos;                                                       &apos;&apos;,* $&amp;%,.., &amp; &apos;&apos; &apos;            <br />              #           &amp;**%. &quot;&apos; &amp; &apos;&apos; $  *-$    &quot;&apos;! *-&apos;           ((.*+ $+&apos;&amp;&apos; &amp; &apos;&apos; &apos;                                             <br />             &quot;&amp;&apos;          *((+ &quot;( &amp; &apos;&apos; $  *-&apos;                       *((+ $&amp;%,*.% &amp; &apos;&apos; &apos;             ,++- %* &amp; &apos;&apos; $  *-&apos;            <br />                         &amp;%(,. &quot;&amp; &amp; &apos;&apos; $  *-$    &quot;&apos;! *-&apos;           &amp;%(,. $&amp;+%)) &amp; &apos;&apos; &apos;                                            <br />             &amp;            &apos;&apos;)- &quot;&amp; &amp; &apos;&apos; $    &quot;&apos;! *-&apos;                                                                                     &amp;&quot; <br /> &amp;,  <br />   <br />             )&apos;           &amp;&apos;*(% &quot;&apos; &amp;&apos;*(% &quot;(                                                                                         &quot;   )&apos;&quot; <br /> &amp;, 	  <br />                          )+(+ &quot;&amp; &amp; &apos;&apos; $    &quot;&apos;! *-&apos;                                                 )+(+ $&apos;&amp;.%- &amp; &apos;&apos; &apos;             <br />             &amp;)           ..%&amp; &quot;&apos;   &amp; &apos;&apos; $  *-$    &quot;&apos;! *-&apos;                                          ..%&amp; $&amp;%&amp;... &amp; &apos;&apos; &apos;            <br />             &amp;&amp;           &amp;&apos;&apos;(* &quot;&amp; &amp; *-$    &quot;&apos;! *-&apos;                 &amp;&apos;&apos;(* $&amp;%)%&amp;% &amp; &apos;&apos; &apos;            &amp;&apos;&apos;(* &quot;( &amp; *-$    &quot;&apos;! *-&apos;    <br />            &amp;             -&apos;+&amp; &quot;&amp; &amp; &apos;&apos; $  *-$    &quot;&apos;! *-&apos;                                                                           <br />  <br />                                # !        #     ), /            <br />          	     !                      	   #       <br />                                                          C. Lesch et al.                                                    17 SI <br />  <br />  <br />  <br />  <br />                                                                      <br />  <br />                 <br />    <br />     <br />               	 <br />  <br />   <br />            	    <br />             	  	                       	          <br />  <br />                                                                                     <br />      <br />      	    <br />    <br />   <br />    	   <br />   <br />    <br />      ) <br />      <br />                                                       14 &quot; (    ).&apos; 14         ..  &quot; &quot;  14            ..  &quot; &quot;    ).&apos; 14 <br />  <br />     -- %-../1 <br /> )-&amp;                                    &apos;&quot;)/,                    &apos;&quot;&apos;.*                  &apos;&quot;+// <br />     -- %-../1 <br /> )/&amp;                                    &apos;&quot;,//                    &apos;&quot;&apos;*,                  &apos;&quot;&apos;(* <br />     -0  %55,- <br /> ).&amp;                                    &apos;&quot;&apos;/.                    &apos;&quot;(.+                  &apos;&quot;/(+ <br />     0.$. %-.1/, <br /> ).&amp;-.1/, <br /> )*&amp;                         &apos;&quot;+*.                    &apos;&quot;-++                  &apos;&quot;..( <br />     )-. %1//2 <br /> ).&amp;                                     $	                      &apos;&quot;&apos;&apos;-                   $	 <br />     )-. %1//2 <br /> )/&amp;                                     $	                      &apos;&quot;(&apos;+                   $	 <br />      *  % %-11,5 <br /> ).&amp;                                 &apos;&quot;(/0                    &apos;&quot;*&apos;+                  &apos;&quot;&apos;)&apos; <br />    - %4.2- <br /> )-&amp;                                       &apos;&quot;.,.                    &apos;&quot;),-                  &apos;&quot;*0) <br />   	! *	   %..31 <br /> ).&amp;                                   $	                      &apos;&quot;()&apos;                   $	 <br />     $. %.-5, <br /> ).&apos;0/1/ <br /> ).&amp;                             &apos;&quot;&apos;&apos;(                    &apos;&quot;+/0                  &apos;&quot;&apos;(- <br />   <br />    - %33-3 +.11.4&amp;                                 &apos;&quot;)&apos;,                    &apos;&quot;&apos;&apos;)                  &apos;&quot;&apos;)+ <br />       %..3. +//1-2&amp;                                  &apos;&quot;++0                    &apos;&quot;+.-                  &apos;&quot;0*, <br />       %..3. +//1-4&amp;                                  &apos;&quot;0&apos;-                    &apos;&quot;*&apos;&apos;                  &apos;&quot;*,, <br />       %..3. <br /> )-&amp;                                      &apos;&quot;&apos;&apos;-                    &apos;&quot;0&apos;,                  &apos;&quot;&apos;&apos;/ <br />       %..3. <br /> ).&amp;                                      &apos;&quot;&apos;,+                    &apos;&quot;+)&apos;                  &apos;&quot;)(- <br />      $. %..3. <br /> ).&apos;..3. <br /> )-&amp;                            &apos;&quot;&apos;,*                    &apos;&quot;&apos;+&apos;                  &apos;&quot;0,, <br />     $. %-253/ <br /> ).&apos;-253/ <br /> )-&amp;                            $	                      &apos;&quot;&apos;(&apos;                   $	 <br />   <br />   0 %53/4 <br /> )-&amp;                                      &apos;&quot;()+                    &apos;&quot;*0*                  &apos;&quot;+,) <br />   <br />   0 %53/4 <br /> )/&amp;                                      &apos;&quot;&apos;-,                    &apos;&quot;&apos;)*                  &apos;&quot;+/) <br />    5.  %03., +/045-&amp;                                 &apos;&quot;(//                    &apos;&quot;&apos;),                  &apos;&quot;*(( <br />    5.  %03., +/045.&amp;                                 &apos;&quot;**(                    &apos;&quot;+*)                  &apos;&quot;&apos;0( <br />    $$. %       <br />   -&amp;       <br />   /&amp;                       &apos;&quot;&apos;&apos;*                    &apos;&quot;))/                  &apos;&quot;&apos;&apos;&apos;&apos;+0) <br />   <br />   - %..04 <br /> )-&amp;                                      &apos;&quot;*/+                    &apos;&quot;&apos;&apos;/                  &apos;&quot;&apos;&apos;) <br />      <br />  %02/2 <br /> )-&amp;                                      &apos;&quot;,-*                    &apos;&quot;+.-                  &apos;&quot;/.- <br />     -$. %-/44 <br /> )-&amp;-/44 <br /> ).&amp;                            &apos;&quot;,,(                    &apos;&quot;*.&apos;                  &apos;&quot;.*&apos; <br />      - %/-0.- +.132,&amp;                                &apos;&quot;-+.                    &apos;&quot;)+.                  &apos;&quot;+&apos;&apos; <br />      . %04,/ +/0454&amp;                                 &apos;&quot;++/                    &apos;&quot;&apos;&apos;/                  &apos;&quot;&apos;*- <br />       $. %124, <br /> )-&amp;124, <br /> ).&amp;                           &apos;&quot;*.,                    &apos;&quot;*+&apos;                  &apos;&quot;&apos;.( <br />      %-,/35 <br /> )-&amp;                                      &apos;&quot;)-.                    &apos;&quot;/(.                  &apos;&quot;*.. <br />       %-,,32 <br /> )-&amp;                                     &apos;&quot;&apos;0.                    &apos;&quot;(+)                  &apos;&quot;0*) <br />          %#---4&amp;                                     &apos;&quot;),.                    &apos;&quot;&apos;-/                  &apos;&quot;&apos;&apos;, <br />  <br />          #   	  <br />    <br />     	      <br />         	  <br />    <br />        	   	  <br />  	  )+                           <br />  <br />    <br />  <br />           <br />    	  	 <br />   <br />  <br />     <br />          ) <br />       	    <br />   <br />              <br />   <br />      <br />      <br />      	   &quot;   # 	   <br />   	  <br />      &quot;        <br />   	  <br /> !    <br />     $ <br />      $.  	   <br />      <br />         <br />     ).&apos; 14   <br />      <br />      	  !   <br />    <br />      <br />      	           	     	   <br />       <br />        	   	  <br /> &quot;      <br />   <br />  !        <br />      <br />   <br />      <br />      <br />      	       <br />               	     	   <br />      <br />        	   	  <br />     	      <br />      ) <br />       	    <br />   <br />               <br />     <br />    <br />  18 SI                                                   C. Lesch et al. <br />  <br />  <br />  <br />  <br />    <br /> 	          <br />              	                 <br />  $      <br />                          	    <br />   <br />    <br />       &apos;      !%&amp;&quot;     <br />        <br />        <br />      <br />                        <br />         <br /> 	     <br />  !  <br />          <br />                     <br />    <br />      <br />         #      <br />        <br />    <br />         	 <br />   <br />     	   <br />      <br />     <br />                                                             C. Lesch et al.                                                  19 SI <br />  <br />  <br />  <br />  <br />                                                                       <br />  <br />                  <br />    <br />     <br />               	 <br />  <br />   <br />            	    <br />           	  	                       	          <br />  <br />                                                                                                   <br />                  	     ) <br />     <br />                                                       14 &quot; (    ).&apos; 14          ..  &quot; (  14            ..  &quot; (    ).&apos; 14 <br />  <br />      -- $-../1 <br /> )-%                                   &amp;!+*-                     &amp;!).*                  &amp;!&apos;(+ <br />      -- $-../1 <br /> )/%                                   &amp;!+/+                     &amp;!)&apos;+                  &amp;!&apos;*/ <br />      -0  $55,- <br /> ).%                                   &amp;!*)+                     &amp;!).,                  &amp;!&apos;&apos;/ <br />      0.$. $-.1/, <br /> ).&amp;-.1/, <br /> )/%                        &amp;!/.(                     &amp;!&apos;-(                  &amp;!&apos;,, <br />      )-. $1//2 <br /> ).%                                    #                        &amp;!+(-                   #  <br />      )-. $1//2 <br /> )/%                                    #                        &amp;!-&apos;+                   #  <br />       *  % $-11,5 <br /> ).%                                &amp;!&amp;&amp;*                     &amp;!&amp;&amp;&amp;&amp;&amp;&apos;&apos;              &amp;!&amp;&amp;&apos; <br />     - $4.2- <br /> )-%                                      &amp;!(.)                     &amp;!-&apos;)                  &amp;!*+* <br />    	! *	   $..31 <br /> ).%                                  #                        &amp;!(+)                   #  <br />      $. $.-5, <br /> ).&apos;0/1/ <br /> ).%                            &amp;!&amp;&apos;-                     &amp;!&amp;*&amp;                  &amp;!,*, <br />    <br />    - $33-3 +.11.4%                                &amp;!.))                     &amp;!+&amp;(                  &amp;!,)* <br />        $..3. +//1-2%                                 &amp;!,+)                     &amp;!&apos;&amp;.                  &amp;!&amp;+- <br />        $..3. +//1-4%                                 &amp;!(/-                     &amp;!,/.                 <br /> </body></html>