<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br /> The microtubule-associated protein, NUD-1, exhibits <br /> chaperone activity in vitro <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> Regulation of cell division requires the concerted           refolding assay was utilized. Following thermal denatur- <br /> function of proteins and protein complexes that properly              ation, rabbit reticulocyte lysate and ATP were added and <br /> mediate cytoskeletal dynamics. NudC is an evolutionarily              luciferase activity measured. In the presence of NUD-1, <br /> conserved protein of undetermined function that associates            nearly all of the luciferase activity was regained, indicating <br /> with microtubules and interacts with several key regulators           that unfolded intermediates complexed with NUD-1 could <br /> of mitosis, such as polo-kinase 1 (Plk1) and dynein. NudC             be refolded. These studies represent the first functional <br /> is essential for proper mitotic progression, and homologs             evidence for a member of this mitotically essential protein <br /> have been identified in species ranging <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> fungi <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span>                 family as having chaperone activity and facilitates elucida- <br /> humans. In this paper, we report the characterization of the          tion of the role such proteins play in chaperone complexes <br /> Caenorhabditis elegans NudC homolog, NUD-1, as a                      utilized in cell division. C. elegans NUD-1 is a member of <br /> protein exhibiting molecular chaperone activity. All                  an evolutionary conserved protein family of unknown <br /> NudC/NUD-1 proteins share a conserved p23/HSP20                       function involved in the regulation of cytoskeletal dynam- <br /> domain predicted by three-dimensional modeling [Garcia-               ics. NUD-1 and its mammalian homolog, NudC, function <br /> Ranea, Mirey, Camonis, Valencia, FEBS Lett 529(2&#226;&#8364;&#8220;                     with the dynein motor complex to ensure proper cell <br /> 3):162&#226;&#8364;&#8220;167, 2002]. We demonstrate that nematode NUD-1                 division, and knockdown or overexpression of these <br /> is able to prevent the aggregation of two substrate proteins,         proteins leads to disruption of mitosis. In this paper, we <br /> citrate synthase (CS) and luciferase, at stoichiometric               show that NUD-1 possesses ATP-independent chaperone <br /> concentrations. Further, NUD-1 also protects the native               activity comparable to that of small heat shock proteins and <br /> state of CS from thermal inactivation by significantly                cochaperones and that changes in phosphorylation state <br /> reducing the inactivation rate of this enzyme. To further             functionally alter chaperone activity in a phosphomimetic <br /> determine if NUD-1/substrate complexes were productive                NUD-1 mutant. <br /> or simply &#226;&#8364;&#339;dead-end&#226;&#8364;&#65533; unfolding intermediates, a luciferase <br />                                                                       Keywords Chaperone . Dynein . NudC . nud . p23 . Plk1 <br />  <br />                                                                       Abbreviations <br /> L. M. Faircloth : P. F. Churchill : G. A. Caldwell :                  CS     citrate synthase <br /> K. A. Caldwell (*)                                                    GR     glucocorticoid receptor <br /> Department of Biological Sciences, The University of Alabama, <br />                                                                       nud    nuclear distribution <br /> Tuscaloosa, AL 35487, USA <br /> e-mail: kcaldwel@bama.ua.edu                                          Plk1   polo-kinase I <br />                                                                       RRL    rabbit reticulocyte lysate <br /> G. A. Caldwell : K. A. Caldwell                                       SEC    size exclusion chromatography <br /> Departments of Neurology and Neurobiology, <br />                                                                       sHsps small heat shock proteins <br /> Center for Neurodegeneration and Experimental Therapeutics, <br /> University of Alabama at Birmingham,                                  TPR    tetracopeptide repeat <br /> Birmingham, AL 35294, USA                                             UCS    UNC-45/Cro1/She4p <br />  96                                                                                                         L.M. Faircloth et al. <br />  <br />  <br /> Introduction                                                    contains the same two serine residues noted above; further, <br />                                                                 knockdown of nud-1 in C. elegans results in mislocalization <br /> During mitosis, regulatory proteins assist in the synchroni-    of PLK-1 whereby it is no longer localized to centrosomes <br /> zation of cellular events via association with chaperones,      (Faircloth and Caldwell, unpublished observation). <br /> molecular motors, motor-associated proteins, kinases, and          Notably, all homologs of NudC, including NUD-1, <br /> other proteins. Although many of these proteins have been       contain a domain similar to the human p23 protein and <br /> defined, the control of mitotic cell division remains an        HSP20/alpha-crystallin family of heat shock proteins <br /> unresolved problem. Studies of molecular motors and their       (Garcia-Ranea et al. 2002). Proteins that contain this p23 <br /> cargo have provided insights into this mechanism, yet a         core-folding motif often bind to Hsp90 via this region (Zhu <br /> functional understanding of the mitotic &#226;&#8364;&#339;machinery&#226;&#8364;&#65533;             and Tytgat 2004; Catlett and Kaplan 2006). This domain <br /> remains to be discerned.                                        was revealed using a combination of three-dimensional and <br />     Nearly 30 years ago, a genetic screen performed in the      hidden Markov modeling and was identified in several new <br /> fungus, Aspergillus nidulans, identified a number of nuclear    protein families. One of the new p23-containing proteins <br /> distribution (nud) genes that regulate the movement of          identified in that study, Sgt1, a yeast kinetochore assembly <br /> nuclei, a fundamental process for growth and development        protein, has since been functionally validated as an Hsp90 <br /> (Morris 1975). Among these, NudC was first identified as a      binding partner (Takahashi et al. 2003; Catlett and Kaplan <br /> <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span>mutation</span> that regulated nuclear movement, cell wall             2006). <br /> deposition, colony growth, and viability (Osmani et al.            The human Hsp90 interactome was recently character- <br /> 1990; Chiu et al. 1997). Other nud loci encode either           ized using proteomic techniques, and NudC was identified <br /> components or proposed regulators of the minus-end-             as a novel and specific binding partner (Te et al. 2007), thus <br /> directed microtubule-dependent motor protein dynein and         providing physical evidence for the predicted p23 domain <br /> its activator dynactin (Morris 2000). NudC is evolutionarily    within NudC. In general, p23 proteins and HSP20 family <br /> conserved, with homologs identified in many species             members [often referred to as small heat shock proteins <br /> including Caenorhabditis elegans, Drosophila, and               (sHsps)] exhibit ATP-independent chaperone activity and <br /> humans.                                                         cooperate with ATP-dependent chaperones such as Hsp70 <br />     Genetic analyses in A. nidulans have shown that NudC        and Hsp90 (Ehrnsperger et al. 1997; Felts and Toft 2003; <br /> is functionally linked to the dynein/dynactin motor complex     Haslbeck et al. 2005b). <br /> (Morris 2000; Han et al. 2001). Complementary functional           In this study, we investigate the in vitro chaperone <br /> studies performed in mammalian cells demonstrated that          activity of C. elegans NUD-1 as a prototypic NudC <br /> NudC co-immunoprecipitated with the dynein/dynactin             example of the important, but functionally undefined, <br /> complex (Aumais et al. 2001). An additional NudC                NudC protein family. The results of this study reveal that <br /> homolog, termed NudC-like (NudCL), also binds the               NUD-1 does possess ATP-independent chaperone activity. <br /> dynein complex and appears to be involved in the                This was tested via aggregation, inactivation, and refolding <br /> stabilization of dynein intermediate chain (Zhou et al.         assays and NUD-1 was shown to have activity equivalent to <br /> 2006). Both mammalian NudC homologs show increased              or exceeding that of Hsp90. <br /> phosphorylation during mitosis (Nishino et al. 2006; Zhou <br /> et al. 2006). The kinase responsible for the phosphorylation <br /> of NudCL has not yet been identified. However, NudC is          Materials and methods <br /> known to be phosphorylated at Ser-274 and Ser-326 by the <br /> mitotic kinase Plk1 (Zhou et al. 2003). Modulating levels of    Plasmid constructs and site-directed mutagenesis <br /> mammalian NudC via knockdown or overexpression results <br /> in mitotic failure and mislocalization of Plk1 (Aumais et al.   Whole worm total RNA was extracted as previously <br /> 2003; Zhou et al. 2003).                                        described (Locke et al. 2006), and complementary DNA <br />     We have characterized the sole C. elegans NudC              (cDNA) was synthesized using SuperScriptIII (Invitrogen) <br /> homolog, NUD-1, and reported its expression in neurons,         from an oligo(dT)12&#226;&#8364;&#8220;18 primer. nud-1 cDNA was isolated <br /> multinucleated tissues such as the hypodermis, and in           from whole worm cDNA for use in the Gateway Cloning <br /> mitotically dividing cells (Dawe et al. 2001). RNAi-            System (Invitrogen) with Nud-1GTWY5 and Nud- <br /> mediated knockdown of nud-1 in C. elegans is associated         1GTWY3 primers. All primers used are listed in Table 1. <br /> with embryonic lethality resulting from mitotic cell division   Site-directed mutagenesis was performed to change the <br /> defects such as misoriented mitotic spindles, an absence of     serine residues at positions 263 and 315 to glutamic acid <br /> midzone microtubules, and cytokinesis furrow regression         residues in order to mimic phosphorylation at these sites. <br /> (Aumais et al. 2003). Amino acid sequence alignment             Three polymerase chain reactions (PCRs) were performed <br /> between NUD-1 and NudC reveals that NUD-1 also                  using nud-1 cDNA to accomplish these mutations. The first <br />  NUD-1 exhibits chaperone activity in vitro                                                                                        97 <br />  <br /> Table 1 Primers used for cloning and mutagenesis of C. elegans nud-     1 mg/ml solution (Buchner et al. 1998). Luciferase and the <br /> 1 and pnud-1 <br />                                                                         luciferase assay system were purchased from Promega. <br /> Name                    Primer sequence                                 Oxaloacetate, acetyl-CoA, IgG, and dithio-1,4-nitrobenzoic <br />                                                                         acid were purchased from Sigma and BSA from Invitrogen. <br /> Nud-1 GTWY 5            ggggacaagtttgtacaaaaaagcaggcttcat <br />                          gtctcaatatgagcgattcgactcgg <br />                                                                         CS inactivation <br /> Nud-1 GTWY 3            ggggaccactttgtacaagaaagctgggtttta <br />                          tccaatttctgaagtccatttcagga <br /> Nud263E Fwd             gaagccagagaacgagaaattgagcgacttgg                Thermal inactivation of CS was achieved by incubating the <br /> Nud263E Rev             ccaagtcgctcaatttctcgttctctggcttc                native protein at 43&#194;&#176;C. CS (15 &#206;&#188;M) was first diluted 1:100 <br /> mutnud-1 GTWY 3         Ggggaccactttgtacaagaaagctgggtcttatccaattttagc   into 40 mM HEPES&#226;&#8364;&#8220;KOH (pH 7.5), 0.1 mM EDTA in the <br />                                                                         presence or absence of additional proteins at 25&#194;&#176;C. <br /> Primers are listed in the 5&#226;&#8364;&#178;&#226;&#8224;&#8217;3&#226;&#8364;&#178; direction. <br />                                                                         Inactivation was initiated by shifting the sample to 43&#194;&#176;C. <br />                                                                         Aliquots were taken at specific time points, and CS activity <br /> PCR used Nud-1GTWY5 and Nud263E Rev primers.                            was measured according to Srere et al. (1963). <br /> Nud263E Fwd and mutNud-1GTWY3 primers were then <br /> used to generate the second PCR product. These two PCR                  Aggregation assays <br /> products were combined to serve as the template for the <br /> third PCR. This reaction used Nud-1GTWY5 and mutNud-                    To measure CS aggregation, CS (15 &#206;&#188;M) was diluted 1:100 <br /> 1GTWY3 primers and resulted in pnud-1 cDNA, encoding                    in 40 mM HEPES&#226;&#8364;&#8220;KOH (pH 7.5) and equilibrated at 43&#194;&#176;C <br /> a phosphomimetic NUD-1 protein (pNUD-1). These PCR                      in the presence and absence of chaperones or controls. All <br /> products were cloned into pDONR221 to create a Gateway                  NUD-1 and pNUD-1 molar concentrations were based on <br /> entry vector. nud-1 and pnud-1 were each introduced into                the octameric form. Luciferase (10 &#206;&#188;M) was diluted as <br /> the destination vector pDEST17 which carries an N-                      above and equilibrated at 42&#194;&#176;C with and without chaper- <br /> terminal His tag. All PCR products and inserts were                     ones or controls. To monitor kinetics of thermal aggrega- <br /> verified by sequencing (Northwoods DNA, Inc.).                          tion, light scattering was measured in a PerkinElmer <br />                                                                         LS50-B luminescence spectrophotometer at 370 nm with <br /> Protein expression/purification                                         a spectral bandwidth of 2.5 nm. <br />  <br /> Expression of recombinant NUD-1 and pNUD-1 was                          NUD-1 analysis by SEC <br /> carried out in the Escherichia coli strain BL21-AI (Invi- <br /> trogen). Transformed cells were induced with 0.1%                       Purified NUD-1 and pNUD-1 protein samples were spun at <br /> arabinose at 37&#194;&#176;C for 4 h. Cells were spun down and                     4&#194;&#176;C for 10 min at 16,000&#195;&#8212;g and filtered through a 0.45-&#206;&#188;m <br /> resuspended in 50 mM sodium phosphate (pH 8.0) con-                     syringe filter. Samples were analyzed on a Superose 6 10/ <br /> taining protease inhibitor cocktail (Roche) and frozen at               300 GL column (GE Healthcare; 10&#195;&#8212;300 mm; separation <br /> &#226;&#710;&#8217;80&#194;&#176;C. They were then lysed by sonication, and 50%                      range of 5&#226;&#8364;&#8220;5,000 kDa) with a mobile phase consisting of <br /> ammonium sulfate was added to the resulting supernatant.                100 mM HEPES&#226;&#8364;&#8220;KOH (pH 7.5), 150 mM KCl, running at <br /> Precipitated proteins were pelleted by centrifugation                   0.5 ml/min at 22&#194;&#176;C. Proteins were detected by absorbance <br /> (15,000&#195;&#8212;g) and resuspended in and dialyzed against                      at 280 nm. <br /> 50 mM sodium phosphate (pH 8.0). These proteins were <br /> purified using HIS-select nickel affinity gel (Sigma)                   Luciferase refolding assay <br /> according to the manufacturer&#226;&#8364;&#8482;s specifications. After <br /> purification, the proteins were dialyzed against 40 mM 4-               Luciferase refolding assays were performed as described by <br /> (2-hydroxyethyl)-1-piperazineethanesulfonic acid (HEPES)&#226;&#8364;&#8220;               Lee et al. (1997) and as modified by Abdulle et al. (2002). <br /> KOH (pH 7.5), and protein concentrations were determined                Luciferase (0.2 &#206;&#188;M) was incubated with various concen- <br /> by bicinchoninic assays using bovine serum albumin (BSA)                trations of NUD-1, pNUD-1, or IgG in refolding buffer <br /> as a standard.                                                          [5 mM MgCl2, 10 mM KCl, 50 mM HEPES&#226;&#8364;&#8220;KOH <br />                                                                         (pH 7.5), and 2 mM DTT] at 22&#194;&#176;C or 42&#194;&#176;C for 15 min. <br /> Materials                                                               To prevent luciferase adsorption to walls, tubes were treated <br />                                                                         with 1 mg/ml BSA for 15 min then rinsed with water. Ten <br /> Citrate synthase (CS) from porcine heart mitochondria was               microliters of the mixtures was added to a solution pre- <br /> obtained from Roche and prepared as described previously                incubated at 30&#194;&#176;C containing 18 &#206;&#188;l of rabbit reticulocyte <br /> (Buchner et al. 1998). CS concentration was determined                  lysate (RRL; Promega) and 2 &#206;&#188;l of 0.1 M ATP. At various <br /> using the published extinction coefficient of 1.78 for a                times, 2 &#206;&#188;l of the sample was removed, added to 48 &#206;&#188;l of <br />  98                                                                                                              L.M. Faircloth et al. <br />  <br />                                                                      Fig. 2 NUD-1 and pNUD-1 <br /> 50 mM HEPES&#226;&#8364;&#8220;KOH (pH 7.5), vortexed, and diluted 10-                  form large oligomeric com- <br /> fold into luciferase assay reagent (Promega). Luciferase             plexes. To analyze oligomeriza- <br /> activity was measured with a Promega GloMAX 20/20                    tion state, both NUD-1 and <br /> luminometer.                                                         pNUD-1 (2 &#206;&#188;M) were run on a <br />                                                                      Superose 6 10/300 GL column <br />                                                                      (GE Healthcare) as described in <br /> Statistics                                                           &#226;&#8364;&#339;Materials and methods&#226;&#8364;&#65533; <br />  <br /> Statistical analysis for chaperone activity was performed <br /> using the Student&#226;&#8364;&#8482;s t test to compare NUD-1 with pNUD-1 <br /> endpoint results in various assays. Significance was <br /> determined as P&lt;0.05. <br />  <br />  <br /> Results <br />  <br /> Production and analysis of recombinant NUD-1 <br /> and pNUD-1 <br />  <br /> The nud-1 cDNA was fused to a 6X histidine tag at the N <br /> terminus and expressed in bacteria as a full-<span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> 6X-HIS- <br /> NUD-1 protein. It was purified over a nickel column and <br /> will be referred to hereafter as NUD-1 (Fig. 1). Notably,            so, to determine if its size could be altered by phosphor- <br /> mammalian NudC is phosphorylated at two highly con-                  ylation. Therefore, we utilized size exclusion chromatogra- <br /> served serine residues by Plk1; the phosphorylated form of           phy (SEC) to analyze the oligomeric state of NUD-1 as well as <br /> NudC is biologically active during cell division (Zhou et al.        pNUD-1. NUD-1 forms a large oligomer (~300 kDa) con- <br /> 2003). These conserved sites are also present in the C.              sisting of eight NUD-1 monomers (Fig. 2). It is also evident <br /> elegans homolog, NUD-1, at residues 263 and 315. Since               that phosphorylation has no effect on the oligomeric state <br /> phosphorylation is known to have an effect on sHsps, it was          of NUD-1 since there is no shift in molecular weight <br /> important that we investigate the role of phosphorylation on         observed in the pNUD-1 mutant (Fig. 2). For all subsequent <br /> the activity of NUD-1. To do so, we constructed a mutant to          assays, concentrations of NUD-1 and pNUD-1 were calcu- <br /> mimic phosphorylation at these sites by using site-directed          lated using the octameric form. <br /> mutagenesis to change the conserved serines to glutamic <br /> acid residues. The resulting mutant phosphomimetic pro-              NUD-1 suppresses aggregation of CS and luciferase <br /> tein, pNUD-1, was also produced and purified as a 6X-HIS <br /> fusion protein (Fig. 1).                                             Since many of the proteins that contain the p23 domain <br />    Many sHsps are known to form large oligomeric                     have been found to have in vitro chaperone activity, we <br /> complexes whose composition can be altered when phos-                sought to determine if NUD-1 and pNUD-1 also exhibited <br /> phorylated (Haslbeck et al. 2005b). Since NUD-1 is well              such activity. We used CS and luciferase as substrates in a <br /> within the size range of the sHsps at 37 kDa, we wanted to           common assay which examines the ability of a chaperone <br /> know if it was capable of forming a larger complex and, if           to inhibit aggregation (Jakob et al. 1995; Herbst et al. 1997; <br />                                                                      Morrow et al. 2006). Both substrates aggregate at high <br />                                                                      temperatures (&gt;40&#194;&#176;C) and aggregation is monitored by light <br />                                                                      scattering. <br />                                                                         Figure 3 shows the thermal aggregation of CS after <br />                                                                      incubation at 43&#194;&#176;C over 20 min. At equimolar concen- <br />                                                                      trations of NUD-1 (1&#195;&#8212;), CS aggregation was reduced by <br />                                                                      93.5&#194;&#177;3.0% (Fig. 3A). This high level of protection was <br />                                                                      even maintained when the concentration of NUD-1 was <br /> Fig. 1 Expression and purification of NUD-1 and pNUD-1. Crude        decreased fourfold (0.25&#195;&#8212;). This nearly absolute protection <br /> extract from E. coli BL21-AI cells containing the pDEST17            can be compared to excess BSA, a negative protein control, <br /> expression vector with either NUD-1 (lane 1) or pNUD-1 (lane 3) <br />                                                                      which still shows high levels of aggregation. To determine <br /> was collected after 4 h of arabinose induction. NUD-1 (lane 2) and <br /> pNUD-1 (lane 4) were purified by nickel affinity column chromatog-   the potential for phosphorylation to affect chaperone <br /> raphy and analyzed by gel electrophoresis with Coomassie staining    activity, pNUD-1 was tested as well (Fig. 3B). Imitating <br />  NUD-1 exhibits chaperone activity in vitro                                                                                                        99 <br />  <br />  <br />                                                                             stoichiometric concentrations, are both capable of nearly <br />                                                                             complete suppression of aggregation, but at substoichio- <br />                                                                             metric concentrations, NUD-1 exhibits greater chaperone <br />                                                                             activity than pNUD-1. <br />  <br />                                                                             Thermal inactivation of CS <br />  <br />                                                                             Although NUD-1 can prevent protein aggregation, it is not <br />                                                                             known whether the protected proteins retain activity. To <br />                                                                             further characterize the interactions of NUD-1 with unfold- <br />                                                                             ed proteins, we examined the thermal inactivation of CS. <br />                                                                             This assay measures the amount of CS activity remaining <br />                                                                             after exposure to heat-denaturing conditions in the presence <br />                                                                             of an additional chaperone. For example, in the presence of <br />                                                                             excess IgG (a negative control), CS rapidly lost its activity <br />                                                                             after incubation at 43&#194;&#176;C, decreasing to less than 10% after <br />                                                                             only 5 min (Fig. 5). This inactivation reaction followed <br />                                                                             first-order kinetics with an inactivation rate of 19.2&#195;&#8212;10&#226;&#8364;&#8220;3 s&#226;&#710;&#8217;1 <br />                                                                             (Table 2). In contrast, an eightfold molar excess of NUD-1 <br />                                                                             decreased the inactivation rate by 88%; this is similar to the <br />                                                                             effect of Hsp90 at the same molar ratio (Jakob et al. 1995). <br />  <br />  <br /> Fig. 3 NUD-1 and pNUD-1 inhibit heat-induced aggregation of <br /> citrate synthase (CS). CS (0.15 &#206;&#188;M) was incubated at 43&#194;&#176;C for 20 min <br /> either alone (filled circle), in the presence of BSA (0.75 &#206;&#188;M, triangle), <br /> NUD-1 (0.15 &#206;&#188;M, open circle; 0.0375 &#206;&#188;M, filled inverted triangle) in <br /> A, pNUD-1 (0.15 &#206;&#188;M, open circle; 0.0375 &#206;&#188;M, filled inverted <br /> triangle) in B. Aggregation was determined by light scattering at <br /> 370 nm. Data are representative of three trials and were calculated as a <br /> percentage of the maximum aggregation of CS after 20 min for each <br /> trial and are expressed as the mean&#194;&#177;SD <br />  <br /> phosphorylation in this manner had no effect (P=0.88) on <br /> the chaperone activity of NUD-1 at the 1&#195;&#8212; concentration <br /> (93.2&#194;&#177;2.0%). However, at the lower concentration, the <br /> chaperone activity of pNUD-1 was significantly lower than <br /> NUD-1 (82.2&#194;&#177;3.2%; P=0.01). <br />    To demonstrate that NUD-1 chaperone activity exhibits <br /> general substrate specificity, we verified that the protein <br /> was also able to inhibit thermal aggregation of luciferase. <br /> Figure 4 shows the aggregation of luciferase after incuba- <br /> tion at 42&#194;&#176;C over 20 min. Similar to its effect on CS, an <br /> equimolar amount of NUD-1 decreased aggregation by 98&#194;&#177; <br /> 1.4% (Fig. 4A). A reduced concentration of NUD-1 was <br /> also effective with 87.1&#194;&#177;4.8% protection. Incubation with <br /> pNUD-1 exhibited a similar result (Fig. 4B). At an <br /> equimolar concentration, aggregation was decreased by <br />                                                                             Fig. 4 NUD-1 and pNUD-1 inhibit heat-induced aggregation of <br /> 94.2&#194;&#177;4.22% and was not significantly different from NUD- <br />                                                                             luciferase. Luciferase (0.1 &#206;&#188;M) was incubated at 42&#194;&#176;C for 20 min <br /> 1 (P=0.21). However, at the lower concentration, aggrega-                   either alone (filled circle), in the presence of BSA (0.75 &#206;&#188;M,), NUD-1 <br /> tion was reduced by 69.9&#194;&#177;5.3%, which is significantly                       (0.1 &#206;&#188;M, triangle; 0.025 &#206;&#188;M, filled inverted triangle) in A, or pNUD- <br /> lower than with NUD-1 (P=0.01). These data correspond                       1 (0.1 &#206;&#188;M, triangle; 0.025 &#206;&#188;M, filled inverted triangle) in B. <br />                                                                             Aggregation was determined by light scattering at 370 nm. Data are <br /> well with the CS aggregation data and together demonstrate <br />                                                                             representative of three trials and were calculated as a percentage of the <br /> that phosphorylation has a significant effect on the ability                maximum aggregation of luciferase after 20 min for each trial and are <br /> of NUD-1 to prevent aggregation. NUD-1 and pNUD-1, at                       expressed as the mean&#194;&#177;SD <br />  100                                                                                                                      L.M. Faircloth et al. <br />  <br />  <br />                                                                         showed a slight increase in activity to 102% after the <br />                                                                         addition of RRL, but was stable throughout the refolding <br />                                                                         period (active luciferase, Fig. 6). However, when luciferase <br />                                                                         was incubated at 42&#194;&#176;C for 15 min in the absence of any <br />                                                                         chaperones, or with excess IgG, the activity decreased to <br />                                                                         less than 10% and no recovery of activity after the refolding <br />                                                                         period was observed (Fig. 6A, B). In the presence of an <br />                                                                         equimolar concentration of NUD-1, 88.2&#194;&#177;12.2% luciferase <br />                                                                         activity was regained; doubling the concentration of NUD- <br />                                                                         1 did not alter the activity recovered (Fig. 6A). When <br />                                                                         luciferase was denatured in the presence of an equimolar <br />                                                                         concentration of pNUD-1, 54.7 &#194;&#177; 8.0% activity was <br />                                                                         regained and, as with NUD-1, an increase in pNUD-1 <br />                                                                         concentration had no further effect (Fig. 6B). However, <br />                                                                         there was a significant difference in chaperone activity <br />                                                                         between NUD-1 and pNUD-1 at both concentrations (1&#195;&#8212;, <br />                                                                         P=0.02; 2&#195;&#8212;, P=0.04). From these data, it is clear that both <br />                                                                         NUD-1 and pNUD-1 can maintain heat-denatured lucifer- <br />  <br />  <br /> Fig. 5 NUD-1 and pNUD-1 decrease the rate of heat-induced <br /> inactivation of CS. CS (0.15 &#206;&#188;M) was incubated for 30 min in the <br /> presence of 1.5 &#206;&#188;M IgG (filled inverted triangle), 1.2 &#206;&#188;M NUD-1 <br /> (filled circle), or 1.2 &#206;&#188;M pNUD-1 (open circle), and CS activity was <br /> measured at various time points. The enzymatic activity of CS was <br /> expressed as a percentage of initial CS activity <br />  <br />  <br /> The presence of pNUD-1 resulted in even greater thermo- <br /> protection with a 94% decrease in the inactivation rate. <br /> These data demonstrate that both NUD-1 and pNUD-1 are <br /> able to bind to and stabilize early folding intermediates of <br /> CS under thermal denaturing conditions. <br />  <br /> NUD-1 can maintain heat-denatured luciferase in a folding <br /> competent state <br />  <br /> To verify that the partially unfolded proteins trapped by <br /> these chaperones were in a refoldable state, an in vitro <br /> luciferase refolding assay was employed. In this assay, <br /> luciferase is heat-denatured in the presence or absence of <br /> chaperones and then allowed to refold at a lower temper- <br /> ature in the presence of ATP and other chaperone <br /> machinery present in RRL. Luciferase maintained at 25&#194;&#176;C <br />                                                                         Fig. 6 Luciferase can be refolded in the presence of either NUD-1 or <br />                                                                         pNUD-1. Active luciferase (0.2 &#206;&#188;M, filled circle) was measured after <br /> Table 2 Influence of various proteins on the rate of thermal            incubation at 25&#194;&#176;C for the indicated times. All other measurements <br /> inactivation of CS at 43&#194;&#176;C                                              were made after incubation at 42&#194;&#176;C for 15 min. Luciferase (0.2 &#206;&#188;M) <br />                                                                         was incubated either alone (open circle) or in the presence of IgG <br /> Addition to                              Rate constant (&#195;&#8212;10&#226;&#710;&#8217;3 s&#226;&#710;&#8217;1)      (2 &#206;&#188;M, filled square). A Luciferase was incubated with NUD-1 <br /> inactivation buffer                      of CS thermal inactivationa    (0.2 &#206;&#188;M, triangle; 0.4 &#206;&#188;M, filled inverted triangle). B Luciferase was <br />                                                                         incubated with pNUD-1 (0.2 &#206;&#188;M, triangle; 0.4 &#206;&#188;M, filled inverted <br /> 1.50 &#206;&#188;M IgG                                          19.2               triangle). The refolding step was performed at 30&#194;&#176;C for 120 min in <br /> 1.20 &#206;&#188;M NUD-1                                         2.3               rabbit reticulocyte lysate (RRL) supplemented with ATP, and lucifer- <br /> 1.20 &#206;&#188;M pNUD-1                                        1.2               ase activity was determined at the different time points. Data are <br />                                                                         representative of three trials and are presented as percentage of <br /> a <br />  Rate constants were obtained from the inactivation kinetics shown in   luciferase activity after 15 min of incubation at 25&#194;&#176;C. Data are <br /> Fig. 5A.                                                                expressed as mean&#194;&#177;SD <br />  NUD-1 exhibits chaperone activity in vitro                                                                                     101 <br />  <br />  <br /> ase in a refoldable state that allows it to be refolded by       Abdulle et al. 2002). Notably, while pNUD-1 displayed <br /> other chaperones into a fully active enzyme, but NUD-1           high chaperone activity in this assay, it was significantly <br /> exhibits greater chaperone activity.                             lower than NUD-1, demonstrating regulation of NUD-1 <br />                                                                  chaperone activity by phosphorylation. <br />                                                                     Several years ago, NUD-1 was predicted to contain a <br /> Discussion                                                       p23/HSP20 domain via three-dimensional modeling pro- <br />                                                                  grams (Garcia-Ranea et al. 2002). It is interesting to note <br /> We describe the previously unreported ability of NUD-1 to        that while our assays indicate that NUD-1 shares many <br /> function as a molecular chaperone, with multiple substrates      characteristics with proteins in the sHsp family, there are <br /> and across a variety of established assays. The purified         distinct differences in activity and it does not fit the <br /> protein forms a large oligomer (&gt;300 kDa), corresponding         exemplar profile of either a sHsp or a p23-like protein <br /> to an octamer of NUD-1 subunits; this is characteristic of       exclusively. These proteins predominantly consist of the <br /> chaperones in the sHsp family. One of the features of some       p23/Hsp20 domain with no other known functional <br /> sHsps is that their oligomeric state can be altered by           domains present (Weikl et al. 1999; Stromer et al. 2003). <br /> phosphorylation, sometimes changing chaperone activity           However, within NUD-1, the p23/HSP20 domain is fused <br /> levels (Sun and MacRae 2005). Thus, we also used                 with additional domains that, together, likely provide the <br /> molecular mimicry of serine phosphorylation of NUD-1             proper three-dimensional structure required for NUD-1/ <br /> by substituting glutamate to study its effects on oligomer-      NudC activity (Garcia-Ranea et al. 2002). In metazoans, <br /> ization and chaperone activity (Eidenmuller et al. 2000;         NudC homologs have an N-terminal extension consisting of <br /> Longshaw et al. 2004).                                           two coiled-coil domains; further, these proteins share a C- <br />    The ability to inhibit aggregation of heat-denatured          terminal region that is phosphorylated by, and binds to, <br /> proteins is a characteristic of most molecular chaperones        Plk1. The p23/Hsp20 domain is located between these two <br /> (Borges and Ramos 2005). Both NUD-1 and pNUD-1                   domains (see Fig. 7). Therefore, it might not be surprising <br /> prevented more than 90% of the aggregation of CS and             that NUD-1 appears to have characteristics of multiple <br /> luciferase at equimolar concentrations. This is similar to the   chaperone types whereby it prevents aggregation and forms <br /> high activity of sHsps, which is due to their high substrate     oligomers similar to sHsps (instead of monomers like p23), <br /> binding affinity (Lee et al. 1995; Koteiche and McHaourab        but displays thermal inactivation properties more similar to <br /> 2003). The decrease in chaperone activity due to phosphor-       p23 (and unlike sHsps). This distinct combination of <br /> ylation is also seen with the sHSPs (Rogalla et al. 1999; Ito    observed chaperone traits may reflect the multidomain <br /> et al. 2001).                                                    architecture of NUD-1. <br />    We further examined NUD-1 chaperone activity using a             A greater understanding of NUD-1 function requires <br /> CS inactivation assay. In this context, NUD-1 protects CS        further investigation into these unique chaperone properties. <br /> from thermal denaturation by slowing the rate of inactiva-       Since it contains a p23 domain, which is known to bind <br /> tion by more than 80%. This result is suggestive of a            Hsp90 (Felts and Toft 2003; Lee et al. 2004; Wu et al. <br /> transient interaction between NUD-1 and early folding CS         2005), the relationship between NUD-1 and Hsp90 could <br /> intermediates (Jakob et al. 1995). Notably, the cochaperone      be investigated to determine if it acts as a cochaperone. <br /> p23 has also been shown to reduce the inactivation rate of       Classification of cochaperones has been traditionally based <br /> CS (Bose et al. 1996). In contrast to NUD-1 and p23, sHsps       upon secondary structural domains. However, considering <br /> such as Hsp25 and Hsp30C appear to form a more long-             that NUD-1 was identified as a p23 domain-containing <br /> lived complex with CS intermediates (Ehrnsperger et al.          protein based on three-dimensional modeling, it does not <br /> 1997; Fernando and Heikkila 2000) and are therefore              conform to families defined purely by their secondary <br /> unable to prevent CS inactivation (Ehrnsperger et al.            structure. Thus, functional studies of NUD-1 cochaperone <br /> 1997). Due to the tight binding of sHsps to unfolded             activity with Hsp90 are still warranted. While beyond the <br /> substrates, they require the addition of an ATP-dependent <br /> chaperone, such as Hsp70, and more permissive conditions <br /> before substrates can be released (Ehrnsperger et al. 1997; <br /> Lee et al. 1997; Haslbeck et al. 2005a). <br />    Using a luciferase refolding assay in the presence of <br /> equimolar NUD-1, we determined that NUD-1 was able to <br /> maintain luciferase in a folding-competent state; after the <br />                                                                  Fig. 7 Schematic representation of the three primary domains <br /> addition of RRL and ATP, ~90% of the luciferase activity         identified within NUD-1. The positions of the two conserved C. <br /> was recovered. This has also been demonstrated for many          elegans serine residues altered to mimic PLK-1 phosphorylation are <br /> sHsps (Bose et al. 1996; Fernando and Heikkila 2000;             indicated on upper portion of the figure (amino acids 263 and 315) <br />  102                                                                                                               L.M. Faircloth et al. <br />  <br />  <br /> scope of this study, the binding of NUD-1 to Hsp90 could        cellular dysfunction associated with aberrant phosphoryla- <br /> be confirmed by SEC and/or co-immunoprecipitation               tion of NudC by Plk1 may be a consequence of <br /> experiments. Structure&#226;&#8364;&#8220;function studies involving removal       mislocalization as well as chaperone activity. The chaper- <br /> of all or part of the p23 domain to examine the specificity     one activity of NUD-1/pNUD-1 may maintain the func- <br /> of binding and characterization of the ability of NUD-1 to      tionality of proteins, perhaps even Plk1, as they traverse the <br /> modulate the ATPase activity of Hsp90 could potentially         cellular milieu. <br /> advance our understanding of both these proteins. Further-         The interaction of NudC with molecular motor proteins, <br /> more, it would be interesting to determine if the N- or C-      such as dynein, as well as Plk1, underscores the pivotal role <br /> terminal regions of NUD-1, involved in protein&#226;&#8364;&#8220;protein          NudC plays in cell division. We speculate that NudC and its <br /> interactions or PLK-1 binding, respectively, are necessary      various homologs may function in the capacity of a <br /> for chaperone activity. Previous studies with p23 have          chaperone to correctly localize mitotic proteins, perhaps <br /> shown that its unstructured C terminus is responsible for its   via an Hsp90 and/or dynein-mediated mechanism. <br /> chaperone activity (Weikl et al. 1999). <br />     Many chaperones interact with Hsp90 through two well- <br />                                                                <br /> characterized Hsp90-binding domains, the p23/HSP20 and         <br /> tetracopeptide repeat (TPR) domains (Lamb et al. 1995;          <br /> Blatch and Lassle 1999). Glucocorticoid receptor (GR)          <br /> transport to the nucleus is a well-characterized function of <br /> Hsp90 that requires association with an immunophilin <br /> through its TPR domain. In this regard, the GR/Hsp90 <br />                                                           <br /> complex is linked to the immunophilin, which then attaches <br /> to the dynein complex for retrograde movement along <br />                                                                 <br /> microtubules. This interaction of a chaperone with a <br />                                                                    <br /> cytoskeletal element is not unique to this class of                  <br /> chaperones. Additional chaperones interact with motor            <br /> proteins and cytoskeletal elements; these include the sHsps,   <br /> which interact with actin filaments and tubulin (Gusev et al. <br />                                                                 <br /> 2002; Mitra et al. 2007), the UNC-45/Cro1/She4p (UCS)              <br /> family of chaperones that interact with myosin (Hutagalung   <br /> et al. 2002), and a plethora of tubulin chaperones, which            <br /> include cofactors and chaperonins (Lewis et al. 1997; <br />                                                                     <br /> Lundin et al. 2008).                                           <br />     Studies in A. nidulans and mammalian cells have shown          <br /> that NudC is associated with the microtubule minus-end           <br /> directed dynein/dynactin motor complex (Morris 2000; <br />                                                                <br /> Aumais et al. 2001; Han et al. 2001). This complex may        <br /> target regulatory and effector molecules to the midzone        <br /> during the cytokinesis phase of mitosis. When NudC is         <br /> knocked down using small interfering RNA, cytokinesis <br />                                                                    <br /> defects result, and Plk1 is no longer localized at the               <br /> centrosomes during metaphase or at the midbody during           <br /> cytokinesis (Aumais et al. 2003; Zhou et al. 2003). These           <br /> cytokinesis defects can be partially rescued by ectopic <br />                                                                  <br /> expression of wild-type NudC, but not by expression of         <br /> NudC with mutations in the Plk1 phosphorylation sites              <br /> (Zhou et al. 2003). Furthermore, when NudC-deficient cells         <br /> are reconstituted with wild-type NudC, proper Plk1 <br />                                                                  <br /> localization is restored, but not when phospho-defective             <br /> NudC is used (Nishino et al. 2006).                                <br />     Notably, in our study, we have discovered that NUD-1       <br /> chaperone activity levels can be regulated by phosphoryla- <br />                                                                      <br /> tion. In the context of these findings, the localization and     <br /> activity of NudC/NUD-1 may be dissociated. However, the            <br /> </body></html>