<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br />                                         Pivotal role of the Francisella tularensis heat-shock <br />                                         sigma factor RpoH <br />  <br />                                         Francisella tularensis is a highly infectious pathogen that infects animals and humans <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> cause the <br />                                         disease tularemia. The primary targets of this bacterium are macrophages, in which it replicates in <br />                                         the cytoplasm after escaping the initial phagosomal compartment. The ability to replicate within <br />                                         macrophages relies on the tightly regulated expression of a series of genes. One of the most <br />                                         commonly used means of coordinating the regulation of multiple genes in bacteria consists of the <br />                                         association of dedicated alternative sigma factors with the core of the RNA polymerase (RNAP). In <br />                                         silico analysis of the F. tularensis LVS genome led us to identify, in addition to the genes encoding <br />                                         the RNAP core (comprising the a1, a2, b, b9 and v subunits), one gene (designated rpoD) encoding <br />                                         the major sigma factor s70, and a unique gene (FTL_0851) encoding a putative alternative sigma <br />                                         factor homologue of the s32 heat-shock family (designated rpoH). Hence, F. tularensis represents <br />                                         one of the minority of bacterial species that possess only one or no alternative sigma factor in <br />                                         addition to the main factor s70. In the present work, we show that FTL_0851 encodes a genuine s32 <br />                                         factor. Transcriptomic analyses of the F. tularensis LVS heat-stress response allowed the <br />                                         identification of a series of orthologues of known heat-shock genes (including those for Hsp40, <br />                                         GroEL, GroES, DnaK, DnaJ, GrpE, ClpB and ClpP) and a number of genes implicated in Francisella <br />                                         virulence. A bioinformatic analysis was used to identify genes preceded by a putative s32-binding <br />                 site, revealing both similarities to and differences <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> RpoH-mediated gene expression in <br />                 Escherichia coli. Our results suggest that RpoH is an essential protein of F. tularensis, and positively <br />                   regulates a subset of genes involved in the heat-shock response. <br />  <br />  <br />  <br /> INTRODUCTION                                                                through arthropod bites, from contaminated animal <br />                                                                             carcasses, or by inhalation of bacteria. Due to its high <br /> Francisella tularensis is a Gram-negative, facultative <br />                                                                             infectivity and the severity of disease, F. tularensis has been <br /> intracellular pathogen that causes tularemia in humans <br />                                                                             classified by the US Centers for Disease Control as a <br /> and a large number of animal species. Human infections <br />                                                                             potential biological weapon (for reviews, see McLendon et <br /> can occur after ingestion of contaminated water or food, <br />                                                                             al., 2006; Sjostedt, 2006). The pathogenicity of F. tularensis <br />                                                                             is largely dependent on its ability to replicate inside host <br /> Abbreviations: FDR, false discovery rate; FPI, Francisella pathogenicity <br /> island, MFS, major facilitator superfamily; PFM, position frequency <br />                                                                             cells, and in particular in macrophages in vivo. Several <br /> matrix; qRT-PCR, quantitative RT-PCR; RNAP, RNA polymerase; TIGR,           factors important for virulence have been identified, and in <br /> The Institute for Genomic Research.                                         particular genes located in the Francisella pathogenicity <br /> The microarray data discussed in this paper have been deposited in          island (FPI) (de Bruin et al., 2007; Golovliov et al., 2003; <br /> ArrayExpress under the accession numbers E-MEXP-2048 and E-                 Lai et al., 2004; Ludu et al., 2008; Nano et al., 2004; Nano &amp; <br /> MEXP-2037.                                                                  Schmerk, 2007; Santic et al., 2007; Tempel et al., 2006; <br /> Five supplementary tables, listing primers used in this study, genes        Weiss et al., 2007), but the molecular mechanisms <br /> significantly decreased in expression after temperature upshift, genes      underlying disease are not yet fully understood. <br /> that were used in BioProspector analysis, putative binding sites found by <br /> BioProspector and used for building of matrices, and genes with a           Regulation of gene expression in bacteria occurs primarily <br /> potential s32-binding sequence in their promoter region, are available      at the transcriptional level. The association of dedicated <br /> with the online version of this paper.                                      alternative sigma factors to the core of the RNA polymerase <br />  <br /> 2560                                                                                           029058 G 2009 SGM Printed in Great Britain <br />                                                                                            Francisella tularensis heat-shock response <br />  <br />  <br /> (RNAP) provides a simple and efficient way for bacteria to       for RpoH and identified genes preceded by a putative <br /> rapidly adapt to various environmental changes. The              RpoH-binding site. Our results demonstrate similarities <br /> RNAP holoenzyme contains the subunits of the core                between the heat-stress response in F. tularensis and the <br /> molecule (comprising two a subunits, the b, b9 and v             well-studied response in E. coli, but also illuminate <br /> subunits) and a sigma factor enabling the holoenzyme to          differences between these two organisms. A number of <br /> specifically recognize promoter elements and initiate            heat-induced genes have previously been demonstrated to <br /> transcription at these sites. The regulon of a single sigma      be of importance for the ability of F. tularensis to survive <br /> factor can comprise hundreds of genes. Many sigma factors        and/or multiply inside host cells, suggesting that stress <br /> are induced in response to environmental changes and are         responses are of significance for the virulence of Francisella. <br /> often controlled by anti-sigma factors (for reviews see <br /> Gourse et al., 2006; Gruber &amp; Gross, 2003). The number of <br /> sigma factors encoded by different bacterial species varies      METHODS <br /> considerably. For example, seven sigma factors have been <br />                                                                  Bacterial strains and plasmids. Bacterial strains and plasmids used <br /> identified in Escherichia coli to date, whereas Streptomyces <br />                                                                  in this study are listed in Table 1 and oligonucleotide sequences in <br /> coelicolor is thought to encode at least 61. Some bacteria       Supplementary Table S1. F. tularensis LVS was grown on pre-made <br /> contain only a single sigma factor, but the majority express     chocolate agar enriched with PolyVitex (bioMe&#194;&#180;rieux), chocolate agar <br /> several, and the number generally relates to the size of the     plates prepared from GC medium <span id='am-161' about='xsp:base' typeof='owl:Thing'>base</span>, IsoVitalex vitamins and <br /> genome and the environmental variation that allows               haemoglobin (BD Biosciences), or in Schaedler + vitamin K3 broth <br /> growth of the organism (Kill et al., 2005; Rodrigue et al.,      (Shaedler-K3, bioMe&#194;&#180;rieux) at 37 uC or room temperature. E. coli <br /> 2006).                                                           strains were grown on Luria&#226;&#8364;&#8220;Bertani (LB) plates or in LB broth <br />                                                                  (Difco) at 37 uC or room temperature aerobically. <br /> Another ubiquitous mode of bacterial response to <br />                                                                  For F. tularensis, antibiotics were used at 5 mg ml21 (kanamycin) and <br /> environmental changes involves the function of two-              1.75 mg ml21 (chloramphenicol). For E. coli, antibiotics were used at <br /> component signal transduction systems (TCS). Each TCS            the following concentrations: 25 mg ml21 (kanamycin), 2.5 mg ml21 <br /> consists of a membrane-bound sensor that monitors                (chloramphenicol) and 50 mg ml21 (ampicillin). <br /> environmental variations and a dedicated cytoplasmic <br />                                                                  The plasmid used for overexpression of rpoH, pFNLTP6gro-0851, was <br /> response regulator that mediates an adaptive response,           constructed by amplifying a 957 bp fragment (corresponding to the <br /> i.e. a transcriptional regulation of gene expression.            sequence 78 bp upstream of the rpoH gene and the entire gene) using <br /> Strikingly, Francisella subspecies do not possess a single       primers 0851_F and 0851_R (Supplementary Table S1), followed by <br /> complete TCS pair (Mohapatra et al., 2007 and references         digestion with NdeI and NotI and cloning into plasmid pFNLTP6gro <br /> therein), and analyses of the F. tularensis genomes have         (Maier et al., 2004; Meibom et al., 2009) downstream of the groE <br /> revealed the presence of only a single gene encoding a           operon promoter of F. tularensis. pFNLTP6gro-0851 was introduced <br />                                                                  into the LVS strain by electroporation. <br /> protein homologous to an alternative sigma factor, thereby <br /> placing F. tularensis within the relatively small group of       The plasmid pBADrpoH was constructed by amplifying the LVS rpoH <br /> bacterial species that express only a single or no alternative   gene using primers 0851_F2 and 0851_R2, followed by digestion with <br /> sigma factor. The absence of any complete TCS and the            EcoRI and cloning into plasmid pBAD downstream of the arabinose- <br />                                                                  inducible pBAD promoter. The rpoH gene was verified to be in the <br /> presence of only one gene encoding an alternative sigma <br />                                                                  right orientation by PCR using primers 59pBAD and 0851_R2. <br /> factor may reflect the fact that the Francisella genome is <br /> relatively small and/or is thought to be deteriorating, based    Functional complementation. E. coli K-12 strain MG1655 was <br /> on the high number of pseudogenes and incomplete                 transformed with either pRB11 (Yura et al., 2007) or pBADrpoH. <br /> biosynthetic pathways (Larsson et al., 2005). The absence        Phage P1 was used to transduce the DrpoH : : Kan zhf50 : : Tn10 <br /> of multiple alternative sigma factors or TCS suggests that F.    <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span>mutation</span> from strain CAG48315 (Zhou et al., 1988) into MG1655/ <br /> tularensis has a limited ability to respond to and proliferate   pRB11 and MG1655/pBADrpoH. For the transduction lysate <br />                                                                  preparation, an overnight culture of CAG48315 was diluted 100-fold <br /> in diverse environmental conditions. <br />                                                                  in fresh LB supplemented with 5 mM CaCl2 and 0.2 % glucose, and <br /> Given the small number of putative regulatory proteins           grown at 37 uC with shaking for 1 h before adding 200 ml P1 phage <br /> encoded by F. tularensis, we were interested in examining        lysate into 5 ml culture. When the culture had lysed completely, <br />                                                                  several drops of chloroform were added and the transduction lysate <br /> the function of the only alternative sigma factor that may <br />                                                                  was kept at 4 uC. For the transduction, the recipient strains (MG1655/ <br /> help this organism in <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span>survival</span> and in the stress response. In    pRB11 and MG1655/pBADrpoH) were grown overnight in LB <br /> the present study, we show that the only alternative sigma       supplemented with 50 mg ampicillin ml21 and 0.2 % arabinose. The <br /> factor encoded by the F. tularensis genome, RpoH, is a           cells were then harvested and resuspended in LB supplemented with <br /> genuine s32 <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span>, and our data suggest that rpoH is an           50 mg ampicillin ml21, 0.2 % arabinose, 100 mM MgSO4 and 5 mM <br /> essential gene of F. tularensis. To gain further insight into    CaCl2. A 100 ml volume of bacterial cells and 100 ml transduction <br /> the RpoH-dependent mechanisms of gene expression in F.           lysate were mixed and incubated at 37 uC with shaking for 30 min. <br />                                                                  After addition of 200 ml 1 M sodium citrate (pH 5.5), cells were <br /> tularensis we performed a transcriptomic analysis of the <br />                                                                  incubated at 37 uC with shaking in 1 ml LB containing 50 mg <br /> heat-stress response of F. tularensis LVS and evaluated the      ampicillin ml21 and 0.2 % arabinose for 1 h, and plated on LB agar <br /> role of RpoH in the regulation of the bacterial genome in        supplemented with 50 mg ampicillin ml21, 25 mg kanamycin ml21 <br /> an RpoH-overproducing strain. In addition, a bioinfor-           and 0.2 % arabinose. Colonies appearing after overnight incubation <br /> matic analysis determined a consensus binding sequence           were verified to have the DrpoH : : Kan zhf50 : : Tn10 mutation by <br />  <br /> http://mic.sgmjournals.org                                                                                                       2561 <br />  N. Grall and others <br />  <br />  <br /> Table 1. Bacterial strains and plasmids used in this study <br />  <br />  Strain or plasmid                                             Description or genotype                                        Source <br />  <br />  E. coli strains <br />  DH5a                            F2w80lacZ DM15 recA1 endA1 hsdR17 supE44 thi-1 gyrA96 relA1                        Laboratory strain collection <br />                                   (lacZYA-argF) U169 <br />  CAG48315                        l2 phtpg&#226;&#8364;&#8220;lacZ DrpoH : : Kan zhf50 : : Tn10, pRB11                                  Zhou et al. (1988) <br />  MG1655                          K-12                                                                               Arnaud Gutierrez <br />  MG1655/pRB11                    MG1655 containing plasmid pRB11 expressing E. coli rpoH                            This study <br />  MG1655/pBADrpoH                 MG1655 containing plasmid pBAD expressing F. tularensis rpoH                       This study <br />  MG1655DrpoH/pRB11               MG1655/pRB11 transduced with P1 lysate from CAG48315                               This study <br />  MG1655DrpoH/pBADrpoH            MG1655/pBADrpoH transduced with P1 lysate from CAG48315                            This study <br />  F. tularensis strains <br />  LVS                             F. tularensis subspecies holarctica live vaccine strain                            Laboratory strain collection <br />  LVS/pFNLTP6gro-0851             LVS with plasmid pFNLTP6gro_0851 expressing rpoH from the groES promoter           This study <br />  LVS/pFNLTP6gro                  LVS with plasmid pFNLTP6gro with no insert                                         This study <br />  Plasmids <br />  pRB11                           pBR322 derivative with E. coli rpoH gene downstream of IPTG-inducible              Yura et al. (2007) <br />                                   tac promoter <br />  pBAD                            araBAD promoter, araC, pBR322 origin, AmpR                                         Invitrogen <br />  pBADrpoH                        F. tularensis rpoH gene cloned downstream of arabinose-inducible promoter          This study <br />                                   of pBAD <br />  pFNLTP6gro                      E. coli&#226;&#8364;&#8220;F. tularensis shuttle vector with groE operon promoter, KmR AmpR,          Meibom et al. (2009); Maier <br />                                   constructed from pFNLTP6gro-gfp by deleting gfp gene                               et al. (2006) <br />  pFNLTP6gro-0851                 E. coli&#226;&#8364;&#8220;F. tularensis shuttle vector with F. tularensis rpoH downstream of         This study <br />                                   groE operon promoter, KmR AmpR <br />  <br />  <br />  <br /> PCR using primers KanaRtn903_F and KanaRtn903_R. The first                   hybridization buffer (40 % formamide, 56 saline sodium citrate <br /> strain obtained after transduction, MG1655DrpoH/pRB11, was then              (SSC), 0.1 % SDS, 0.6 mg salmon sperm DNA ml21) at 42 uC for 18 h <br /> grown on LB agar supplemented with 50 mg ampicillin ml21 and                 in a hybridization chamber (Corning). <br /> 25 mg kanamycin ml21 at room temperature, 30 and 37 uC. The <br />                                                                              The F. tularensis microarrays, provided by the Pathogen Functional <br /> second strain, MG1655DrpoH/pBADrpoH, was grown on LB agar <br />                                                                              Genomics Resource Center (PFGRC; Rockville, MD, USA), contain <br /> supplemented with 50 mg ampicillin ml21 and 25 mg kanamy- <br />                                                                              2331 oligonucleotides (70-mer) with four replicates per oligonucleo- <br /> cin ml21 in the presence and absence of 0.2 % arabinose at room <br />                                                                              tide, representing all the ORFs of F. tularensis Schu S4 and F. <br /> temperature, 30 and 37 uC. <br />                                                                              tularensis LVS. <br /> RNA extraction. RNA was extracted from exponential-phase bacteria            Microarrays were scanned with a Genepix 4000B scanner (Molecular <br /> with TRIzol reagent (Invitrogen). After extraction with chloroform,          Devices). To determine signal fluorescence intensities for each spot, <br /> the aqueous phase was purified using the RNeasy Clean-up protocol            TIFF images were analysed using the Spotfinder and Midas software <br /> (Qiagen). Contaminating DNA was removed from RNA preparations                (TIGR). Statistical analyses were performed using the Significance <br /> with RNA-free DNase I (Ambion).                                              Analysis of Microarrays (SAM) software (Tusher et al., 2001). The <br />                                                                              number of genes with significantly modified expression was defined <br /> Microarray experiments and data analyses. To prepare RNA for                 by performing a one-class analysis using a false discovery rate (FDR) <br /> microarray analyses, F. tularensis LVS, F. tularensis LVS+                   of 0 %. <br /> pFNLTP6gro, and F. tularensis LVS+pFNLTP6gro-0851 were grown <br /> at 37 uC with shaking to OD600 ~0.3&#226;&#8364;&#8220;0.4. For rpoH overexpression             Two RNA extractions from independent experiments were done for <br /> experiments, bacterial cells were then harvested and RNA was                 each condition and each sample set was hybridized to two <br /> extracted. For heat-shock-response experiments, the culture of F.            microarrays. <br /> tularensis LVS was split into two: one part of the culture was placed at <br /> 37 uC with shaking, whereas the other was incubated at 44 uC with            Real-time quantitative RT-PCR (qRT-PCR). To validate the <br /> shaking. After 30 min incubation, bacterial cells were harvested and         microarray results, four ORFs were selected for qRT-PCR analysis <br /> RNA was extracted.                                                           with the same RNA samples used in the microarray hybridizations. <br />                                                                              For the heat-shock time-course experiment, RNA samples were <br /> Reverse transcription and cDNA fluorescent labelling were performed          isolated before temperature shift from 37 to 44 uC and after 10, 20, <br /> for each sample on 3 mg of RNA according to the protocol provided            30, 45 and 60 min incubation at 44 uC. For the analyses, 1 mg RNA <br /> by The Institute for Genomic Research (TIGR; http://pfgrc.jcvi.org).         was reverse-transcribed using random hexamers and Superscript II <br /> Briefly, the RNA was reverse-transcribed using Superscript II reverse        reverse transcriptase (Invitrogen) according to the protocol provided <br /> transcriptase in the presence of amino-allyl-dUTP, dNTP and                  by the manufacturer. Real-time PCR was performed with gene- <br /> random primers, followed by coupling of cyanine dye (Cy3 or                  specific primers using an ABI PRISM 7700 and SYBR green PCR <br /> Cy5). Microarray hybridization and post-hybridization washings were          master mix (Applied Biosystems). To calculate the amount of gene- <br /> performed according to the TIGR protocol. Briefly, after pre-                specific transcript, a standard curve was plotted for each primer set <br /> hybridization of microarray slides, hybridization was performed in           using a series of diluted genomic DNA from LVS. To compare the <br />  <br /> 2562                                                                                                                             Microbiology 155 <br />                                                                                                    Francisella tularensis heat-shock response <br />  <br />  <br /> transcript amounts for the different conditions, the amounts of each        Functional complementation of E. coli rpoH <br /> gene transcript were normalized to DNA helicase (FTL_1656), as this <br /> gene has been shown to change little in expression during growth            We first tested the ability of the LVS rpoH gene to <br /> (Brotcke et al., 2006). The expression of each gene was determined          functionally complement the temperature-sensitive growth <br /> from three replicates in a single qRT-PCR experiment.                       of an E. coli DrpoH mutant lacking s32, which can grow <br />                                                                             only at or below 20 uC (Zhou et al., 1988). For this, we <br /> Bioinformatics analyses. The RpoH-binding site consensus <br /> sequence was identified as described previously (Slamti et al., 2007) <br />                                                                             introduced the DrpoH mutation from strain CAG48315 <br /> using BioProspector (Liu et al., 2001) to search for conserved motifs       (Zhou et al., 1988; Table 1) into E. coli strain MG1655 by <br /> with a structure similar to those of the RpoH-binding sites in E. coli      transduction. The recipient strain contains a plasmid <br /> and Vibrio cholerae. The search parameters and input sequences were         encoding the F. tularensis rpoH gene (pBADrpoH) under <br /> adjusted until the motif identified corresponded to the previously          the control of an arabinose-inducible promoter. The <br /> predicted RpoH-binding sites upstream of F. tularensis grpE and             resulting strain did not grow on agar plates without <br /> groES (Meibom et al., 2008) (Supplementary Tables S3 and S4). The <br /> sequences 200 bp upstream of the first 400 annotated F. tularensis <br />                                                                             supplement at any temperature (room temperature, 30 or <br /> ORFs were used as the background model. The final search                    37 uC; Fig. 1, middle row). However, the F. tularensis rpoH <br /> parameters were as follows: motif, 2 blocks; width of the first motif       gene promoted growth of the mutant strain when <br /> block, 7; width of the second motif block, 7; minimum gap between           arabinose was added to the medium (Fig. 1, top row), <br /> the blocks, 13; maximum gap between the blocks, 16. Identification of       showing that F. tularensis rpoH can functionally <br /> putative RpoH-binding sites was accomplished in two steps. First,           complement the lack of s32 in E. coli. As a control, we <br /> four position frequency matrices (PFMs) were constructed. These <br /> were composed of the PFMs of blocks 1 and 2 identified by <br />                                                                             transduced the DrpoH mutation into a recipient strain <br /> BioProspector separated by 13&#226;&#8364;&#8220;16 columns, respectively, in which the        containing a plasmid encoding the E. coli rpoH gene <br /> frequencies of all four nucleotides were set to be identical. Second, the   (pRB11; Yura et al., 2007) (Fig. 1, bottom row), and this <br /> program PATSER (van Helden, 2003) was used to search the 200 bp             strain exhibited the same growth <span id='am-162' about='xsp:pattern' typeof='owl:Thing'>pattern</span> as the <br /> upstream of all F. tularensis genes for sequences corresponding to          MG1655DrpoH/pBADrpoH strain grown in the presence <br /> each of these four matrices. The minimum PATSER score threshold was         of arabinose. These results show that F. tularensis rpoH <br /> set to 9. In cases where two putative binding sites had the same block <br /> 1 or block 2 sequences but different gap lengths, the lower-scoring site <br />                                                                             encodes a genuine s32 <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> of RNAP. <br /> was discarded. <br />                                                                             F. tularensis transcriptomic response to heat <br />                                                                             stress <br /> RESULTS <br />                                                                             We next studied gene expression changes in F. tularensis <br /> Identification of a functional rpoH gene of                                 LVS in response to heat shock. For this, we extracted RNA <br /> F. tularensis LVS                                                           from bacterial cultures grown in Schaedler medium <br />                                                                             (supplemented with vitamin K3) after a temperature <br /> The in silico analysis of the LVS genome allowed us to                      upshift for 30 min at 44 uC. After cDNA synthesis and <br /> identify all the genes involved in the synthesis of core                    labelling, control (the same culture kept at 37 uC) and <br /> RNAP. As reported by Charity et al. (2007), the F.                          heat-shocked samples were paired and hybridized on the <br /> tularensis genome contains two genes, designated rpoA1                      microarrays. Statistical analysis using SAM software <br /> (FTL_0261) and rpoA2 (FTL_0616), which encode two                           (Tusher et al., 2001) indicated that 181 genes exhibited <br /> different a subunits of RNAP that differ from one another                   significant (FDR 0 %) differential expression at a twofold <br /> in regions that are predicted to be critical for dimer                      or greater level in response to a temperature upshift from <br /> formation, promoter recognition, and activator inter-                       37 to 44 uC (Table 2 and Supplementary Table S2). The <br /> action. FTL_1744 and FTL_1743 (rpoB and rpoC) encode                        wide distribution of putative functional roles attributed to <br /> the b and b9 subunits, respectively, and FTL_1533 (rpoZ)                    the differentially expressed genes indicates the extent of the <br /> the v subunit.                                                              molecular response that enables F. tularensis LVS to survive <br /> The major sigma factor (s70) is encoded by FTL_1050                         and eventually adapt to thermal stress. The transcripts of <br /> (rpoD). Strikingly, we identified only one gene, FTL_0851,                  112 genes were more abundant after the temperature <br /> encoding a putative alternative sigma factor sharing                        upshift (Table 2). Among these, 15 genes had annotated <br /> significant sequence similarity with other members of the                   functions in protein fate. Well-known heat-shock proteins, <br /> family of s32 heat-shock factors (hence designated rpoH).                   such as Hsp40, GroEL, GroES, DnaK, DnaJ, GrpE, ClpB, <br /> The rpoH sequence of LVS encodes a protein of 292 aa, with                  ClpX, ClpP and HtpG, form the major part of this group. <br /> 53.4 % amino acid identity with s32 of E. coli K-12, and is                 The major sigma factor RpoD (s70) was found to be <br /> highly conserved among the F. tularensis subspecies.                        upregulated, with a fold-induction of 4.6. In contrast, rpoH <br /> Multiple sequence alignments of s32 family members have                     did not appear to be upregulated after a temperature <br /> led to the identification of a highly conserved stretch of nine             upshift. About 40 % of the upregulated genes were genes <br /> amino acids Q(R/K)(K/R)LFFNLR that is designated the                        encoding proteins of unknown function or pseudogenes. <br /> RpoH box (Nakahigashi et al., 1995). It is of note that this                Transposases composed 21 % of the upregulated genes, but <br /> box is perfectly conserved in the F. tularensis s32 sequences,              this could reflect cross-hybridizations on the microarrays <br /> strongly suggesting that it encodes a functional s32 factor.                as these transposases are highly homologous. Interestingly, <br />  <br /> http://mic.sgmjournals.org                                                                                                            2563 <br />  N. Grall and others <br />  <br />  <br />  <br />  <br />        Fig. 1. The rpoH gene from F. tularensis functionally complements an E. coli DrpoH mutant. E. coli stain MG1655DrpoH <br />        harbouring plasmid pBADrpoH (encoding rpoH from F. tularensis) or pRB11 (encoding rpoH from E. coli) was streaked on LB <br />        agar containing ampicillin and 0.2 % arabinose, where indicated (+ara), and incubated for 24 h at 30 and 37 6C, or for 48 h at <br />        room temperature (RT). <br />  <br />  <br />  <br />  <br /> five genes of the F. tularensis FPI were found to be                     qRT-PCR analyses on four heat-shock-induced genes <br /> downregulated: FTL_0115/1161, FTL_0116/_1162 and                         (groEL, dnaK, grpE and hsp40) over a period of 60 min <br /> FTL_0124/_1170, encoding proteins of unknown function,                   incubation at 44 uC (Fig. 2b). For all four genes we <br /> and pdpB1 and iglB1 (intracellular growth locus                          observed increased expression at all times (10&#226;&#8364;&#8220;60 min) <br /> B)(Supplementary Table S2). One third of the down-                       compared with before heat shock. However, the transcript <br /> regulated genes had annotated functions in metabolism,                   levels appeared to be less elevated at the later time points <br /> biosynthesis of cofactors or ribonucleotide biosynthesis. As             (45 and 60 min). <br /> for the upregulated genes, genes encoding proteins of <br /> unknown function or pseudogenes represented a signifi- <br />                                                                          Role of rpoH in F. tularensis transcriptional <br /> cant portion of the downregulated genes. Furthermore, <br />                                                                          regulation <br /> several genes encoding proteins with transport functions <br /> were downregulated after heat shock.                                     In spite of our repeated efforts, we were unable to <br />                                                                          inactivate the rpoH gene of LVS. We tried to inactivate <br /> To verify the accuracy of the microarray data, we <br />                                                                          rpoH either by insertional mutagenesis or by allelic <br /> performed qRT-PCR analyses on transcripts of four chosen <br />                                                                          exchange at temperatures ranging from 20 to 37 uC, but <br /> genes. For these experiments, we used one of the two RNA <br />                                                                          were unsuccessful. It is thus likely (although not formally <br /> sample sets that were used for the microarray experiments. <br />                                                                          proven) that rpoH is essential in F. tularensis, as it is <br /> As seen in Fig. 2(a), the relative levels of rpoH, dnaK, groEL <br />                                                                          thought to be in other bacteria, such as V. cholerae and <br /> and hsp40 transcripts fully supported the microarray data, <br />                                                                          Neisseria gonorrhoeae (Laskos et al., 2004; Slamti et al., <br /> with dnaK, groEL and hsp40 expressed at higher levels at <br />                                                                          2007). <br /> 44 uC than at 37 uC and the expression of rpoH not <br /> changed by the temperature upshift. To investigate the                   To overcome the lack of an rpoH mutant, we evaluated the <br /> heat-shock response in a temporal manner, we performed                   effect of overproduction of RpoH on the LVS transcrip- <br />  <br /> 2564                                                                                                                      Microbiology 155 <br />                                                                                                   Francisella tularensis heat-shock response <br />  <br />  <br /> Table 2. Genes significantly increased in expression (at least twofold) after temperature upshift <br />  <br />  Locus*                        Gene product                          Gene               Cellular role categoryD                     Fold <br />                                                                                                                                   changed <br />  <br />  FTL_0034        Hypothetical protein                                       Hypothetical proteins: conserved                       2.02 <br />  FTL_0088        Acetyltransferase protein                                  Unknown function                                       2.38 <br />  FTL_0093        Chitinase                                                  Metabolism                                            10.24 <br />  FTL_0094        ClpB protein                                        clpB   Degradation of proteins, peptides and glycopeptides    5.51 <br />  FTL_0144        Hypothetical protein                                       Unknown function: general                              2.77 <br />  FTL_0163        MFS transporter                                            Transport and binding proteins                         4.31 <br />  FTL_0180        Acyltransferase                                     htrB   Metabolism                                             2.25 <br />  FTL_0186        MFS transport protein, fragment                            Transport and binding proteins                         4.01 <br />  FTL_0201        Hypothetical protein                                       Unknown function                                       2.15 <br />  FTL_0202        Conserved hypothetical membrane protein                    Cell envelope: other                                   3.29 <br />  FTL_0203        Hypothetical membrane protein                              Unknown function: general                              2.00 <br />  FTL_0219        Hypothetical protein                                       Unknown function: general                              2.86 <br />  FTL_0267        Heat-shock protein 90                               htpG   Protein folding and stabilization                      3.01 <br />  FTL_0281        Heat-shock protein, hsp40                                  Protein folding and stabilization                      7.13 <br />  FTL_0359        Type IV pili fibre building block protein                  Unknown function                                       2.55 <br />  FTL_0365        2                                                          Pseudogene                                             2.92 <br />  FTL_0390        Type IV pili fibre building block protein                  Unknown function: general                              1.99 <br />  FTL_0645        Hypothetical lipoprotein                                   Cell envelope                                          3.20 <br />  FTL_0662        Prophage repressor protein                                 Mobile and extrachromosomal element functions          3.31 <br />  FTL_0663        Hypothetical protein                                       Hypothetical proteins: conserved                       4.02 <br />  FTL_0758        2                                                          Pseudogene                                             2.24 <br />  FTL_0826        2                                                          Pseudogene                                             2.09 <br />  FTL_0844        LysR transcriptional regulator family protein              Regulatory functions                                   2.35 <br />  FTL_0867        Hypothetical protein                                       Hypothetical proteins: conserved                       2.78 <br />  FTL_0891        Trigger factor                                      tig    Protein folding and stabilization                      4.38 <br />  FTL_0892        ATP-dependent Clp protease subunit P                clpP   Degradation of proteins, peptides and glycopeptides    3.70 <br />  FTL_0893        ATP-dependent protease ATP-binding subunit          clpX   Degradation of proteins, peptides and glycopeptides    2.87 <br />  FTL_0894        DNA-binding, ATP-dependent protease La              lon    Degradation of proteins, peptides and glycopeptides    2.53 <br />  FTL_0908        2                                                          Pseudogene                                             2.07 <br />  FTL_0909        Hypothetical protein                                       Hypothetical proteins: conserved                       2.29 <br />  FTL_0911        2                                                          Pseudogene                                             2.34 <br />  FTL_0912        2                                                          Pseudogene                                             1.97 <br />  FTL_0964        ATP-dependent protease ATP-binding subunit          hslU   Protein folding and stabilization                      4.33 <br />  FTL_0965        ATP-dependent protease peptidase subunit            hslV   Degradation of proteins, peptides and glycopeptides    5.13 <br />  FTL_0996        AhpC/Tsa family protein                                    Detoxification                                         2.01 <br />  FTL_0997        Hypothetical protein                                       Hypothetical proteins: conserved                       3.13 <br />  FTL_1008        2                                                          Pseudogene                                             2.01 <br />  FTL_1048        Hypothetical protein                                       Unknown function: general                              2.23 <br />  FTL_1049        DNA primase                                         dnaG   DNA replication, recombination and repair              2.21 <br />  FTL_1050        RNAP sigma-70 factor                                rpoD   Transcription                                          4.58 <br />  FTL_1125        Transcription regulator                             hipA   Regulatory functions                                   2.75 <br />  FTL_1126        Transcription regulator                                    Regulatory functions                                   2.26 <br />  FTL_1190        Chaperone protein GrpE (heat-shock protein          grpE   Protein folding and stabilization                      6.32 <br />                   family 70 cofactor) <br />  FTL_1191        Chaperone protein DnaK (heat-shock protein family   dnaK   Protein folding and stabilization                      2.23 <br />                   70 protein) <br />  FTL_1192        Chaperone protein DnaJ (heat-shock protein 70       dnaJ   Protein folding and stabilization                      6.19 <br />                   family cofactor) <br />  FTL_1194        Pirin family protein                                       Unknown function: general                              4.17 <br />  FTL_1221        Hypothetical membrane protein                              Cell envelope                                          5.11 <br />  FTL_1222        Conserved hypothetical protein                             Hypothetical proteins: conserved                       4.58 <br />  FTL_1226        2                                                          Pseudogene                                             2.65 <br />  FTL_1257        2                                                          Pseudogene                                             2.31 <br />  FTL_1259        2                                                          Pseudogene                                             3.41 <br />  FTL_1260        2                                                          Pseudogene                                             2.27 <br />  <br />  <br />  <br /> http://mic.sgmjournals.org                                                                                                           2565 <br />  N. Grall and others <br />  <br />  <br /> Table 2. cont. <br />  <br />  <br />  Locus*                          Gene product                          Gene                 Cellular role categoryD                    Fold <br />                                                                                                                                      changed <br />  <br />  FTL_1265        2-Amino-4-hydroxy-6-hydroxymethyldihydropteri-        folK     Biosynthesis of cofactors, prosthetic groups and       5.01 <br />                   dine pyrophosphokinase/dihydropteroate synthase                carriers: folic acid <br />  FTL_1266        Lipase/esterase                                                Metabolism                                              5.82 <br />  FTL_1267        Hypothetical protein                                           Hypothetical proteins: conserved                        4.01 <br />  FTL_1269        2                                                              Pseudogene                                              2.07 <br />  FTL_1270        2                                                              Pseudogene                                              2.68 <br />  FTL_1361        Cold-shock protein                                    cspA     Hypothetical proteins                                   2.30 <br />  FTL_1365        Hypothetical protein                                           Hypothetical proteins: conserved                        2.49 <br />  FTL_1366        Membrane fusion protein, putative                              Transport and binding proteins                          2.13 <br />  FTL_1403        Conserved hypothetical membrane protein                        Cell envelope                                           2.90 <br />  FTL_1411        Hypothetical protein                                  ybaB     Hypothetical proteins: conserved                        2.31 <br />  FTL_1412        Recombination protein RecR                            recR     DNA replication, recombination and repair               2.07 <br />  FTL_1517        Hypothetical protein                                           Hypothetical proteins: conserved                        2.09 <br />  FTL_1535        2-Dehydro-3-deoxyphosphooctonate aldolase                      Cell envelope                                           1.99 <br />  FTL_1555        Hypothetical membrane protein                                  Cell envelope                                           4.32 <br />  FTL_1575        Hypothetical membrane protein                                  Cell envelope                                           4.09 <br />  FTL_1600        Periplasmic L-asparaginase II precursor               ansB     Metabolism                                              2.04 <br />  FTL_1691        Hypothetical protein                                           Hypothetical proteins: conserved                        2.02 <br />  FTL_1692        Hypothetical protein                                           Cell envelope: surface structures                       2.13 <br />  FTL_1714        Chaperone protein, GroEL                              groL     Protein folding and stabilization                       3.05 <br />  FTL_1715        Chaperonin protein, GroES                             groS     Protein folding and stabilization                       3.12 <br />  FTL_1770        Type IV pili nucleotide-binding protein                        Transport and binding proteins                          3.01 <br />  FTL_1771        Type IV pili nucleotide-binding protein               pilT     Transport and binding proteins                          3.03 <br />  FTL_1776        Hypothetical membrane protein                                  Cell envelope                                           3.01 <br />  FTL_1777        2                                                              Pseudogene                                              2.13 <br />  FTL_1778        2                                                              Pseudogene                                              3.19 <br />  FTL_1793        Hypothetical protein                                           Hypothetical proteins: conserved                        2.06 <br />  FTL_1805        ATPase                                                         Unknown function                                        2.92 <br />  FTL_1806        MFS transport protein                                          Transport and binding proteins                          2.91 <br />  FTL_1807        Histidyl-tRNA synthetase                              hisS     tRNA aminoacylation                                     3.12 <br />  FTL_1808        Ribosome-binding factor A                                      Transcription                                           2.96 <br />  FTL_1810        Transcription elongation factor NusA                  nusA     Transcription                                           4.04 <br />  FTL_1811        Hypothetical protein                                           Unknown function: general                               2.74 <br />  FTL_1814        2                                                              Pseudogene                                              2.76 <br />  FTL_1878        Two-component sensor protein KdpD                     kdpD     Regulatory functions: other                             2.14 <br />  FTL_1894        Transposase, fragment                                          Mobile and extrachromosomal element functions           4.16 <br />  FTL_1913        Sua5/YciO/YrdC family protein                                  Unknown function                                        2.06 <br />  FTL_1945        Hypothetical protein                                           Unknown function: general                               1.95 <br />  FTL_1957        Heat-shock protein                                             Protein folding and stabilization                       8.57 <br />  FTL_XXXX&#194;&#167;       Transposase                                           isftu2   Mobile and extrachromosomal element functions         2.0&#226;&#8364;&#8220;6.9 <br />  FTL_YYYY||      Transposase                                           isftu1   Mobile and extrachromosomal element functions         2.3&#226;&#8364;&#8220;3.2 <br />  <br /> *Genes in bold type have a putative RpoH-binding sequence located before the start codon (see Supplementary Table S3 for detailed information). <br /> Gene numbers marked in italic type have been identified in screens or selections for determinants required for intracellular survival and/or <br /> virulence (Kraemer et al., 2009; Maier et al., 2007; Su et al., 2007; Tempel et al., 2006; Weiss et al., 2007). <br /> DCellular role category according to TIGR. <br /> dFold change calculated as the average expression at 44 uC relative to 37 uC calculated from two independent experiments with duplicate <br /> hybridizations. <br /> &#194;&#167;A number of isftu2-type transposases: FTL_0079, 0220, 0270, 0367, 0441, 0468, 0568, 0575, 0591, 0622, 0861, 0868, 1301, 1368, 1471, 1516, 1655, <br /> 1680 and 1885. <br /> ||Three isftu1-type transposases: FTL_0128, 0417 and 0607. <br />  <br />  <br />  <br />  <br /> 2566                                                                                                                          Microbiology 155 <br />                                                                                                         Francisella tularensis heat-shock response <br />  <br />  <br />  <br />  <br />       Fig. 2. Quantification by RT-PCR of transcript levels of selected genes after heat shock. (a) Validation of microarray <br />       experiment. Transcript levels were normalized to that of DNA helicase (FTL_1656) and the fold difference (relative to 37 6C) <br />       and SDs are shown for the dnaK, groEL, hsp40 and rpoH genes. (b) Change in expression of four heat-shock genes during <br />       60 min incubation at 44 6C. Transcript levels were normalized to that of DNA helicase (FTL_1656) and the fold difference <br />       (relative to sample taken before temperature shift) and SDs are shown for the dnaK, groEL, hsp40 and grpE genes. Data are <br />       from triplicate samples. <br />  <br />  <br />  <br />  <br /> tome. For this, we compared the transcriptomes of LVS                         known heat-shock proteins (Hsp40, Hsp90, GroES, GroEL, <br /> (transformed with the plasmid pFNLTP6gro without                              DnaK and DnaJ). The transcriptional control by RpoH <br /> inserted gene) to that of an LVS overexpressing RpoH                          therefore seems to be very specific to heat-shock proteins. <br /> (i.e. transformed with the plasmid pFNLTP6gro_0851, <br /> expressing FTL_0851 from the groES promoter). We <br />                                                                               Consensus sequence for Francisella RpoH- <br /> extracted RNA from exponentially grown bacterial cul- <br />                                                                               dependent promoter <br /> tures, and after cDNA synthesis and labelling, control and <br /> RpoH-overexpressing samples were paired and hybridized                        To identify an RpoH-binding site consensus sequence, we <br /> to the microarrays. Statistical analysis indicated that only                  analysed the 200 bp sequence upstream of all 112 heat- <br /> seven genes exhibited significant increased expression                        upregulated genes listed in Table 2 using Bioprospector <br /> (FDR 0 %) in response to overexpression of RpoH                               (Liu et al., 2001). These analyses failed to identify an RpoH <br /> (Table 3). As expected, rpoH itself was upregulated 21-                       consensus binding sequence, which we reasoned was likely <br /> fold. The six other upregulated genes all encoded well-                       due to the fact that a relatively small proportion of heat- <br />  <br />  <br />  <br />  <br /> Table 3. Genes showing a change in expression in the presence of a plasmid overexpressing RpoH <br />  <br />  Locus                           Gene product*                        Gene                    Cellular role categoryD                Fold changed <br />  <br />  FTL_0851        RNAP sigma factor                                    rpoH         Transcription: transcription factors                  20.6 <br />  FTL_0281        Heat-shock protein, Hsp40                                         Protein fate: protein folding and stabilization        2.9 <br />  FTL_0267        Heat-shock protein 90                                htpG         Protein fate: protein folding and stabilization        2.9 <br />  FTL_1191        Chaperone protein DnaK (heat-shock protein           dnaK         Protein fate: protein folding and stabilization        2.6 <br />                   family 70 protein) <br />  FTL_1192        Chaperone protein DnaJ (heat-shock protein           dnaJ         Protein fate: protein folding and stabilization        2.0 <br />                   70 family cofactor) <br />  FTL_1714        Chaperone protein, GroEL                             groEL        Protein fate: protein folding and stabilization        2.2 <br />  FTL_1715        Chaperone protein, GroES                             groES        Protein fate: protein folding and stabilization        2.3 <br />  <br /> *Genes with significantly changed expression, upregulated at least twofold, are included in the table and displayed according to their functional <br /> category. <br /> DCellular role category according to TIGR. <br /> dValues are average fold change between LVS/pFNLTP6gro-0851 and LVS/pFNLTP6gro from two independent experiments with duplicate <br /> hybridization. Each microarray contains four spots for each gene. <br />  <br />  <br /> http://mic.sgmjournals.org                                                                                                                      2567 <br />  N. Grall and others <br />  <br />  <br />  <br />  <br />        Fig. 3. Consensus sequence of the F. tularensis s32-binding site displayed with WebLogo (http://weblogo.berkeley.edu). The <br />        height of each column indicates the sequence conservation at that position, while the height of symbols within the stack <br />        indicates the relative frequency of each nucleic acid at that position. N(13&#226;&#8364;&#8220;16) indicates the gap between the two blocks. <br />  <br />  <br />  <br /> regulated genes identified in our microarray analyses are              significant homology with the RpoH-binding site consensus <br /> directly regulated by RpoH, making it difficult for                    sequences in E. coli and V. cholerae (Slamti et al., 2007). <br /> Bioprospector to identify conserved binding sites. To <br /> address this problem, we repeated our Bioprospector <br /> analyses using only those upregulated genes annotated by <br /> NCBI as chaperones, chaperonins, proteases or heat-shock <br />                                                                        DISCUSSION <br /> proteins, classes of genes known to be directly regulated by           The bacterial heat-shock response results in a transient <br /> RpoH in other bacteria (Supplementary Tables S3 and S4).               induction of a set of highly conserved heat-shock proteins, <br /> This yielded several putative binding sites, several of which          including chaperones and proteases. The alternative s- <br /> corresponded to previously predicted F. tularensis RpoH-               factor s32 (RpoH) is the primary regulator that controls the <br /> binding sites (Meibom et al., 2008). A PFM representing the            transcription of genes during heat shock and some other <br /> sites identified by Bioprospector was constructed and the              general stress conditions. The activity of the heat-shock <br /> program PATSER (van Helden, 2003) was used to interrogate              proteins ensures the correct folding of proteins during <br /> the upstream regions of all 1754 ORFs in the NCBI                      normal growth, and is essential for resistance to stress by <br /> annotation of LVS for putative RpoH-binding sites. A total             minimizing the damage caused by the accumulation of <br /> of 72 non-overlapping putative RpoH-binding sites with a               misfolded and aggregated proteins. Our examination of the <br /> score of 9 or above were identified upstream of 71 genes               heat-shock response in F. tularensis resulted in the <br /> (Supplementary Table S5), including 13 of the genes                    identification of many bacterial heat-shock proteins, some <br /> upregulated by heat shock (Table 4 and marked in bold                  of which have been previously reported to increase in <br /> type in Table 2). From these 13 putative sites a consensus             amount after heat or oxidative stress (Ericsson et al., 1994; <br /> binding sequence was determined (Fig. 3) that shares                   Lenco et al., 2005) and in response to mammalian body <br />  <br />  <br />  <br /> Table 4. Heat-induced genes with a potential s32-binding sequence in the promoter region <br />  <br />  Distance*    LengthD                Binding site sequence              Locus                        Gene productd <br />  <br />      34          27     CTTGAAAAACAGCAAAGTATCCTTAAG                    FTL_0094    ClpB protein <br />      81          30     CTTGAAAAAACTAACTTCCACTCTCATATA                 FTL_0267    Heat-shock protein 90, HtpG <br />      84          27     CTTGTATTTATATAAAAAGACCTTATA                    FTL_0281    Heat-shock protein, Hsp40 <br />       7          28     CTTGAAAATGATTAGAAAAAGTATTTTA                   FTL_0663    Hypothetical protein <br />     101          29     CTTGTCATATATTTAATAATAAACATATA                  FTL_0891    Trigger factor <br />     162          28     CTTGAAGTGATGTTTCATATACCTTCAT                   FTL_0894    DNA-binding, ATP-dependent protease La <br />      48          29     CTTGTAAAATTTAATATCAATTCCATTTA                  FTL_0965    ATP-dependent protease, peptidase subunit, HslV <br />      82          29     CTTGAAAATACAGAATTAATCACTATCTA                  FTL_1190    Chaperone protein GrpE (heat-shock protein family <br />                                                                                     70 cofactor) <br />        92        29     CTTGAAAAGATTATAAATATGCCCATCTA                  FTL_1191    Chaperone protein DnaK (heat-shock protein family <br />                                                                                     70 protein) <br />        68        30     CTTGAAAATTAAAAAAAAGCCCCCAATTTC                 FTL_1265    2-Amino-4-hydroxy-6-hydroxymethyldihydropteri- <br />                                                                                     dine pyrophosphokinase/dihydropteroate synthase <br />        98        29     CTTGAAAATTTTTTTTTTGACTCAATATC                  FTL_1715    Chaperonin protein, GroES <br />        40        29     CTTGTATTTAGCATGATCGTATCATTCTT                  FTL_1805    ATPase <br />        75        27     CTTGAAATTTTAAAAACCGATCATATA                    FTL_1957    Heat-shock protein <br />  <br /> *Distance from ATG start codon. <br /> DLength (nucleotides) of the putative s32-binding site. <br /> dGene product according to TIGR. <br />  <br /> 2568                                                                                                                  Microbiology 155 <br />                                                                                            Francisella tularensis heat-shock response <br />  <br />  <br /> temperature (Horzempa et al., 2008), in addition to                 Defining an RpoH consensus box <br /> various other genes. <br />                                                                     Using the data we obtained from the transcriptional <br />                                                                     profiling, we performed a bioinformatics analysis to <br /> Role of s32 in heat-stress response                                 determine a consensus sequence for binding of s32 (Fig. <br />                                                                     3). We identified a putative s32-binding site in the <br /> Control of s32 amount and activity in E. coli is complex <br />                                                                     promoter region of 13 genes that were induced after heat <br /> and is performed at transcriptional, post-transcriptional <br />                                                                     shock. Among these are all the genes induced by RpoH <br /> and post-translational levels. The heat-shock response in E.        overexpression, or the first gene in the presumed two-gene <br /> coli is induced as a result of increased s32 levels, primarily      transcripts dnaK-dnaJ and groES-groEL, further suggesting <br /> due to an increase in rpoH mRNA translation and                     that these genes are directly under s32 control. In the <br /> increased stability and activity of s32 (Arse`ne et al.,            remainder of the heat-induced genes, we did not observe <br /> 2000), whereas control of rpoH transcription seems to be            any putative s32-binding sites, indicating that some of <br /> of only minor importance. Similarly, we did not observe an          these are not controlled directly by s32. The consensus <br /> increase in transcription of rpoH in F. tularensis after            binding sequence for s32 resembles the consensus sequence <br /> temperature upshift. In contrast, transcription of s70 was          from V. cholerae and E. coli (Nonaka et al., 2006; Slamti et <br /> increased after heat shock, suggesting that s70-containing          al., 2007), and several of the genes and putative operons <br /> RNAPs are required for the heat-shock response in                   that are preceded by a s32-binding site in F. tularensis are <br /> Francisella, perhaps in combination with gene-specific              also found in E. coli, although small variations are often <br /> regulators.                                                         observed (Nonaka et al., 2006) (Fig. 4). We observed heat- <br /> To more directly define the genes dependent on RpoH, we             mediated upregulation of the four consecutive genes tig- <br /> performed transcriptional profiling of F. tularensis LVS            clpP-clpX-lon, and bioinformatics analysis identified a s32- <br /> harbouring a plasmid expressing RpoH from the groE                  binding site upstream of the tig and lon genes (Fig. 4a). <br /> promoter and compared it with bacteria carrying an empty            However, in E. coli, only clpP, clpX and lon are induced by <br /> plasmid. This allowed us to identify six genes (in addition         heat shock, and a s32-binding site is found upstream of <br /> to rpoH itself) that were expressed at increased levels in the      clpP and lon. In both F. tularensis and E. coli, the <br /> strains overexpressing RpoH. These genes encode the heat-           transcription <span id='am-163' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> dnaK-dnaJ is induced by heat shock <br /> shock proteins Hsp40, HptG, DnaK, DnaJ, GroES and                   and preceded by a s32-binding site. In F. tularensis, these <br /> GroEL, which were all induced after heat stress as well. The        genes are adjacent to grpE, which is also heat-induced and <br /> previous studies of the s32 regulon in E. coli, V. cholerae         preceded by a s32-binding site, although in E. coli grpE is <br /> and N. gonorrhoeae identified these genes also (Gunesekere          located elsewhere on the genome (Fig. 4b). The organiza- <br /> et al., 2006; Nonaka et al., 2006; Slamti et al., 2007; Zhao et     tion of the two-gene transcriptional <span id='am-164' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> groES-groEL <br /> al., 2005), demonstrating common features of RpoH                   seems to be identical in the two species (Fig. 4d), as is the <br /> regulation in different species. However, the number of             case with the hslV-hslU genes, with the exception that the <br /> genes identified in F. tularensis was unexpectedly low when         E. coli genes are followed by a third heat-induced gene, <br /> compared with other bacterial species. To date, there is no         menA (Fig. 4c). The protease FtsH carries out degradation <br /> promoter that allows the specific induction of transcription        of s32 in E. coli and is itself induced by heat shock, and the <br /> in Francisella, and our experiments were therefore                  transcription unit contains a s32-binding site (Fig. 4e). <br /> performed under continuous overexpression of RpoH.                  Interestingly, in F. tularensis we did not observe increased <br /> Based on the knowledge of the regulation of s32 in E. coli,         expression of the ftsH gene after heat shock, and the genetic <br /> we believe this could lead to underestimation of the genes          region encoding FtsH is not predicted to contain a s32- <br /> directly controlled by RpoH. In the current model for               binding site (Fig. 4e). The region containing ftsH seems to <br /> regulation of s32 in E. coli, called the &#226;&#8364;&#732;unfolded protein          have been duplicated in the LVS strain, whereas it is only <br /> titration model&#226;&#8364;&#8482;, the stability and activity of s32 rely on the     found in one copy in the other strains of F. tularensis for <br /> level of unfolded protein relative to the chaperone level (for      which sequence information is available. Currently, we do <br /> review see Guisbert et al., 2008). When unfolded protein            not know whether the activity of s32 in F. tularensis is <br /> levels are low compared with chaperones, s32 is predomi-            controlled by a mechanism similar to the &#226;&#8364;&#732;unfolded protein <br /> nantly bound to the chaperones and is inactive, whereas a           titration model&#226;&#8364;&#8482; and whether degradation of s32 is <br /> high level of unfolded proteins will titrate the chaperones         performed by the FtsH protease. In any case, the regulatory <br />                                                                     mechanism must differ from the one in E. coli, since FtsH is <br /> away from s32, thereby rendering s32 active. In our <br />                                                                     not part of the heat-shock response in F. tularensis. <br /> experimental set-up, unfolded protein levels are expected <br /> to be low (since no stress is imposed on the bacteria),             We also observed s32-binding sites next to a number of <br /> whereas the chaperones will be expressed as a result of             genes that we did not find to be upregulated by RpoH <br /> RpoH overexpression. If the same mechanism of s32                   overexpression or after heat shock. It is possible that some <br /> regulation exists in F. tularensis as in E. coli, our set-up will   of these were missed in the transcriptomic analysis of heat <br /> lead to chaperone-mediated inactivation of s32, and this            stress or were not induced under constant overexpression <br /> could be responsible for the low number of genes we                 of RpoH. We performed transcriptomic analysis after <br /> observe to be induced in the RpoH-overexpressing strain.            30 min of heat exposure, and it is probable that some genes <br />  <br /> http://mic.sgmjournals.org                                                                                                    2569 <br />  N. Grall and others <br />  <br />  <br />  <br />  <br />        Fig. 4. Schematic representations of the organization of selected genes containing putative s32-binding sequences in F. <br />        tularensis and E. coli. Black arrows represent genes that are induced by heat stress. Grey arrows represent genes that are not <br />        induced by heat stress. Gene symbols are indicated above arrows and locus numbers to the left in each panel (numbers <br />        preceded by FTL_ designate the loci in F. tularensis LVS and numbers preceded by b the loci in E. coli MG1655). isftu, <br />        transposase; H, hypothetical protein; mfs, major facilitator transporter. The locations of s32-binding sites in E. coli are from <br />        Nonaka et al. (2006). <br />  <br />  <br />  <br /> are induced only early in the heat-shock response. Our                    intracellular multiplication and/or survival. These genes <br /> qRT-PCR analysis of the temporal expression of four heat-                 include clpB (Gray et al., 2002; Maier et al., 2007; Meibom <br /> shock genes showed an increased expression at all times                   et al., 2008; Su et al., 2007; Tempel et al., 2006; Weiss et al., <br /> from 10 to 60 min of heat exposure, although with less                    2007), hspG (Weiss et al., 2007), FTL_0663 (Kraemer et al., <br /> elevated transcript levels at the later time points. These                2009; Su et al., 2007), dnaK (Tempel et al., 2006) and <br /> results indicate that the heat-shock response is initiated                FTL_1806 [encoding major facilitator superfamily (MFS) <br /> promptly and continues for up to 1 h, but do not rule out                 transport protein] (Maier et al., 2007; Weiss et al., 2007). <br /> the possibility that some genes could have increased                      The identification of these genes, which are important for <br /> expression only early in the heat-shock response.                         intracellular survival, may indicate that stress responses <br /> Interestingly, a number of the genes containing putative                  play a central role in the pathogenesis of F. tularensis and <br /> s32-binding sites (that are not induced by heat stress) are               its ability to survive inside host cells. The intracellular <br /> induced by oxidative stress (N. Grall, unpublished results),              environment is hostile to the bacterium, and induction of <br /> indicating that not only the heat-shock response is                       stress responses may be imperative to its survival within <br /> mediated by s32. Finally, we anticipate that some of the                  host cells. <br /> putative sites do not represent true s32-binding sites,                   Given the significant number of genes (18 %) that are <br /> although they were identified in our bioinformatics                       induced under heat stress and are implicated in the <br /> analysis.                                                                 virulence of Francisella, it will be interesting to further <br />                                                                           study the molecular mechanisms underlying the stress <br /> A relationship between heat-stress response and                           response. In particular, an understanding of how the <br /> virulence                                                                 activity of s32 is controlled, including determining which <br />                                                                           proteins and signals mediate this control, will lead to a <br /> Remarkably, a considerable number of the genes that were                  much better understanding of how s32-mediated stress <br /> upregulated by heat stress (20 genes, see Table 2) have been              responses contribute to survival within the mammalian <br /> identified in screens or selections for genes required for                host. <br /> </body></html>