<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br />  <br />                          A novel pathway regulates memory and plasticity via SIRT1 and <br />                          miR-134 <br />  <br />  <br />                          <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />                                The NAD-dependent deacetylase Sir2 was initially identified as a mediator of replicative lifespan <br />                                in budding yeast and was subsequently shown <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> modulate longevity in worms and flies1,2. Its <br />                                mammalian homologue, SIRT1, appears to have evolved complex systemic roles in cardiac <br />                                function, DNA repair, and genomic stability. Recent studies suggest a functional relevance of <br />                                SIRT1 in normal brain physiology and neurological disorders. However, it is unknown if SIRT1 <br />                                plays a role in higher-order brain functions. We report that SIRT1 modulates synaptic plasticity <br />                                and memory formation via a microRNA-mediated mechanism. Activation of SIRT1 enhances, <br />                                while its loss-of-function impairs, synaptic plasticity. Surprisingly, these effects were mediated via <br />                                post-transcriptional regulation of CREB expression by a brain-specific microRNA, miR-134. <br />                                SIRT1 normally functions to limit expression of miR-134 via a repressor complex containing the <br />                                transcription factor YY1, and unchecked miR-134 expression following SIRT1 deficiency results <br />                                in the down-regulated expression of CREB and BDNF, thereby impairing synaptic plasticity. <br />                                These findings demonstrate a novel role for SIRT1 in cognition and a previously unknown <br />                                microRNA-based mechanism by which SIRT1 regulates these processes. Furthermore, these <br />                                results describe a separate branch of SIRT1 signaling, in which SIRT1 has a direct role in <br />                                regulating normal brain function in a manner that is disparate <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> its cell <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span>survival</span> functions, <br /> HHMI Author Manuscript <br />  <br />  <br />  <br />  <br />                                demonstrating its value as a potential therapeutic target for the treatment of CNS disorders. <br />  <br />  <br />                                       The mammalian Sir2 homolog SIRT1 is involved in a variety of complex processes relevant <br />                                       to aging, including the regulation of oxidative stress, metabolism control, and circadian <br />                                       rhythms1&#226;&#8364;&#8220;4. We previously demonstrated that SIRT1 gain-of-function is neuroprotective in <br />                                       overactive Cdk5 and mutant SOD models of neurodegeneration, which are relevant to <br />                                       Alzheimer&#226;&#8364;&#8482;s disease and ALS, respectively5. Moreover, SIRT1 has recently been implicated <br /> HHMI Author Manuscript <br />  <br />  <br />  <br />  <br />                                       in molecular pathways regulated by cocaine6, suggesting that, in addition to its involvement <br />                                       in neurogenesis and neuroprotection, SIRT1 has further functions in the brain that are yet to <br />                                       be described. <br />  <br />                                       To directly evaluate the physiological role of SIRT1 in learning and memory, mutant mice <br />                                       lacking SIRT1 catalytic activity in a brain-specific manner (SIRT1&#206;&#8221;) were generated by <br />                                       crossing mice carrying a floxed SIRT1&#206;&#8221;ex4 allele7,8 with Nestin-Cre transgenic mice. In <br />                                       tests of associative memory, SIRT1&#206;&#8221; mice exhibited a significant decrease in freezing <br />                                       behavior as evaluated by both contextual (Fig. 1a) and tone-dependent (Fig. S2a) fear <br />                                       conditioning paradigms. Shock sensitivity and locomotor activity did not differ between <br />                                       SIRT1&#206;&#8221; mice and littermate controls (Figs. 1b, S2b). SIRT1&#206;&#8221; mice showed a similarly <br />                                       decreased memory performance in a novel object recognition task (Fig. 1c), which relies <br />                                       upon the hippocampus and cortex9. The time spent exploring the objects during training did <br />                                       not significantly differ between groups (Fig 1d). In the Morris water maze, SIRT1&#206;&#8221; mice <br />                                       displayed significantly increased escape latencies in the hidden platform paradigm (Fig. 1e), <br />                                       while spending less time in the target quadrant in a probe trial (Fig. 1f) compared with <br /> HHMI Author Manuscript <br />  <br />  <br />  <br />  <br />                                       control mice, suggesting that SIRT1 plays a role in spatial learning. Visual function and <br />                                       swimming ability were not affected in the SIRT1&#206;&#8221; mice (Figs. S2c,d). Together, these <br />                                       results show that SIRT1 has an important role in several forms of memory. <br />  <br />                                       Next, we used a long-term potentiation (LTP) paradigm to directly determine the role of <br />                                       SIRT1 in synaptic plasticity. LTP in hippocampal CA1 neurons, induced by two &#206;&#184; burst <br />                                       (2&#195;&#8212;TBS) stimulation of the Schaffer collaterals in control mice, was abrogated in SIRT1&#206;&#8221; <br />                                       mice, demonstrating a requirement of SIRT1 in synaptic plasticity (Fig. 1g). CA1 neurons in <br />                                       SIRT1&#206;&#8221; mice exhibited normal basal synaptic transmission (Figs. S3a,b) compared to <br />                                       control mice. These results demonstrate that the LTP deficits caused by SIRT1 inactivation <br />                                       are not due to impaired synaptic transmission. <br />  <br />                                       The brains of SIRT1&#206;&#8221; mice had a grossly normal anatomy (data not shown). However, <br />                                       experiments using an antibody against synaptophysin (SVP), which labels the presynaptic <br />                                       terminals of functional synapses10, revealed significant decreases in SVP immunoreactivity <br />                                       in the hippocampal striatum radiatum of SIRT1&#206;&#8221; mice, as well as reduced SVP protein <br />                                       content in the SIRT1&#206;&#8221; hippocampus, compared to controls (Figs. 1h and 1i). Golgi <br />                                       impregnation demonstrated that the dendritic spine density of CA1 pyramidal neurons is <br /> HHMI Author Manuscript <br />  <br />  <br />  <br />  <br />                                       significantly decreased in the hippocampus of SIRT1&#206;&#8221; mice (Fig. 1j). These results suggest <br />                                       that SIRT1 regulates synapse formation, synaptic plasticity, and memory formation. <br />  <br />                                       Brain-derived neurotrophic factor (BDNF) and cAMP response binding protein (CREB) are <br />                                       two genes that play critical roles in synaptic plasticity and modulating synapse formation11&#226;&#8364;&#8220; <br />                                       13. Both mRNA and protein levels of BDNF were significantly decreased in SIRT1&#206;&#8221; <br />                                       hippocampi compared with controls (Fig. 2a). CREB binds to several BDNF promoters and <br />                                       plays a key role in the activity-dependent regulation of BDNF expression14&#226;&#8364;&#8220;17. We <br />                                       performed chromatin immunoprecipitation (ChIP) to further examine the association of <br />                                       CREB and BDNF promoters. Baseline binding of CREB to BDNF promoters 1, 2, and 4 <br />                                       was reduced in the SIRT1&#206;&#8221; hippocampus (Fig. 2b). As reported previously15,17, binding of <br />                                       CREB to BDNF promoters 1 and 4, was increased after contextual fear conditioning <br />                                       training. In SIRT1&#206;&#8221; mice, however, this training-related increase in BDNF promoter 1 and 4 <br />                                       binding by CREB was abolished (Fig. S4a). <br />  <br />  <br />                                             Nature. Author manuscript; available in PMC 2011 February 1. <br />                           Gao et al.                                                                                              Page 3 <br />  <br />  <br />                                       One possible explanation for the decreased CREB binding to BDNF promoters observed in <br />                                       SIRT1&#206;&#8221; mice is that CREB itself is downregulated. Consistent with this notion, CREB <br />                                       protein levels were significantly reduced in SIRT1&#206;&#8221; hippocampi (Fig. 2c, left). Although <br />                                       overall CREB levels were reduced in the SIRT1&#206;&#8221; mice, CREB phosphorylation was still <br /> HHMI Author Manuscript <br />  <br />  <br />  <br />  <br />                                       increased by fear conditioning training (Fig. S4b). However, in contrast to BDNF, mRNA <br />                                       levels of CREB were not altered (Fig. 2c, right), suggesting that CREB protein levels are <br />                                       downregulated in SIRT1&#206;&#8221; brains via posttranscriptional mechanisms. <br />  <br />                                       To investigate such mechanisms, we considered a well-known means of posttranscriptional <br />                                       regulation of gene expression, the inhibition of translation via microRNA (miRNA). <br />                                       miRNAs are expressed at high levels in the brain18,19, and the involvement of miRNA in <br />                                       numerous aspects of normal and abnormal brain function has been reported19,20. We <br />                                       compared the expression of miRNAs in SIRT1&#206;&#8221; hippocampi with littermate control <br />                                       hippocampi using a miRNA microarray (Exqion Inc), and found that a number of brain- <br />                                       enriched miRNAs differed in expression between the two groups (Fig. S5). We evaluated <br />                                       the expression of eight miRNAs that appeared significantly altered in the microarray <br />                                       analysis, including five upregulated and three downregulated miRNAs, using mature <br />                                       miRNA-specific quantitative PCR (qPCR). Consistent with the microarray, our qPCR <br />                                       analyses confirmed significant changes in expression of all eight miRNAs (Fig. 2d). <br />  <br />                                       Of these miRNAs, miRNA-134 was of particular interest, as it is specifically expressed in <br />                                       the brain and has been demonstrated to negatively regulate dendritic spine formation in <br /> HHMI Author Manuscript <br />  <br />  <br />  <br />  <br />                                       vitro21. Comparative genomic analyses of the 3&#226;&#8364;&#8482;UTR of the mouse CREB gene revealed <br />                                       three partial complementary binding sites for miR-134, with one site being well-conserved <br />                                       (Fig. S6a). To determine whether miR-134 directly binds to and inhibits the translation of <br />                                       CREB mRNA, we carried out a CREB activity luciferase reporter assay in which luciferase <br />                                       is driven by a minimal promoter downstream of CREB-binding elements. Co-transfection of <br />                                       miR-134 with CRE-Luc resulted in a significant decrease in CREB activity in cultured CAD <br />                                       cells, a neural cell line (Fig. 2e). As a negative control, we created a miR-134 mutant <br />                                       containing mutations in the miR-134 seed region (mut-miR-134). Expression of mut- <br />                                       miR-134 did not affect CREB activity (Fig. 2e). The expression of miR-34b-5p or miR-34c, <br />                                       two other miRNAs upregulated in brains of SIRT1&#206;&#8221; mice, had no effect on CREB activity <br />                                       (Fig. S6b), indicating specific regulation of CREB by miR-134. Finally, miR-134 loss-of- <br />                                       function using a locked-nucleic-acid (LNA)-modified oligonucleotide probe (LNA- <br />                                       miR-134), which achieves specific knockdown of endogenous miR-134, significantly <br />                                       enhanced CREB activity (Fig. 2e). These results suggest that miR-134 regulate CREB <br />                                       protein expression via a post-transcriptional mechanism. <br />  <br />                                       To verify the direct nature of CREB inhibition by miR-134, we generated a luciferase <br /> HHMI Author Manuscript <br />  <br />  <br />  <br />  <br />                                       reporter construct in which the 3&#226;&#8364;&#8482;UTR of CREB, containing all three predicted miR-134 <br />                                       target sequences, was inserted downstream of a luciferase expression cassette (WT-CREB). <br />                                       As expected, co-expression of miR-134 with WT-CREB significantly attenuated reporter <br />                                       expression (Fig. 2f). A luciferase reporter construct linked to the 3&#226;&#8364;&#8482;UTR of CREB <br />                                       containing multiple mutations within the three miR-134 target sequences (mut-CREB) was <br />                                       refractory to knockdown by miR-134 (Fig. 2f). Expression of mut-CREB alone resulted in a <br />                                       significant increase in luciferase activity, indicating a de-repression of the reporter construct <br />                                       by endogenous miR-134 (Fig. 2f). These results indicate that miR-134 attenuates CREB <br />                                       expression via a specific interaction with the target regions within the 3&#226;&#8364;&#8482;UTR of CREB. <br />                                       Moreover, overexpression of miR-134, but not mut-miR-134, in cultured CAD cells resulted <br />                                       in reduced levels of CREB protein relative to a scrambled miR control (Scr-miR; Fig. 2g). In <br />                                       contrast, delivery of LNA-miR-134 resulted in increased CREB protein levels compared to <br />                                       LNA-scrambled-miR (LNA-scr-miR) control (Fig. 2g), suggesting that, under normal <br />                                       conditions, miR-134 has a limiting effect on CREB translation. <br />  <br />  <br />                                              Nature. Author manuscript; available in PMC 2011 February 1. <br />                           Gao et al.                                                                                            Page 4 <br />  <br />  <br />                                       To investigate how SIRT1 modulates miR-134 transcription, we conducted ChIP with an <br />                                       anti-SIRT1 antibody and examined the association of SIRT1 with DNA sequences spanning <br />                                       ~5 Kb upstream of the Pre-miR-134 sequence. We found that two regions, R3 and R7, could <br />                                       be specifically amplified from SIRT1 ChIP, suggesting that SIRT1 is associated with <br /> HHMI Author Manuscript <br />  <br />  <br />  <br />  <br />                                       potential regulatory elements upstream of miR-134 (Fig. 3a). The specificity of the <br />                                       interaction was verified by ChIP analyses from SIRT1 constitutive <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span>knockout</span> (KO) mouse <br />                                       brain tissue8, which yielded a dramatically reduced pull-down of the R3 and R7 fragments <br />                                       (Fig. 3b). To determine the transcriptional modulating activities of these sites, we cloned R3, <br />                                       R7, and the control R5 fragment upstream of a minimal promoter in a pCL3-promoter <br />                                       luciferase reporter construct. Interestingly, co-expression of SIRT1 with reporter constructs <br />                                       containing either R3 or R7, but not R5, significantly reduced, while SIRT1 shRNA <br />                                       increased, the expression of the luciferase reporter (Fig. 3c). Collectively, these results <br />                                       indicate that SIRT1 inhibits miR-134 expression by directly binding to distal inhibitory <br />                                       elements. <br />  <br />                                       To verify the contribution of miR-134 to the modulation of CREB by SIRT1, we created <br />                                       SIRT1 loss-of-function in CAD cells by either shRNA-mediated knockdown (shRNA- <br />                                       SIRT1) or overexpression of a catalytically-inactive mutant of SIRT1 (H363Y)22, both of <br />                                       which elevated the level of miR-134 in these cells (Fig. 3d). The loss of SIRT1 markedly <br />                                       inhibited CREB activity, but this effect was reversed by the administration of LNA-miR-134 <br />                                       (Fig. 3e). These results provide strong evidence that SIRT1 loss-of-function attenuates <br /> HHMI Author Manuscript <br />  <br />  <br />  <br />  <br />                                       CREB activity via a miR-134-mediated posttranscriptional mechanism. We searched the R3 <br />                                       and R7 regions for transcription factor consensus binding motifs and discovered Yin Yang 1 <br />                                       (YY1) binding sites within both the R3 and R7 fragments (Fig. 3f, top), but not in any of the <br />                                       other fragments upstream of the miR-134 coding region. YY1 is a ubiquitous and highly- <br />                                       conserved transcription factor that can activate or repress gene expression, depending upon <br />                                       the cellular context23. To determine if YY1 binds to miR-134 regulatory sequences, we <br />                                       performed ChIP experiments using anti-YY1 and anti-SIRT1 antibodies after both <br />                                       overexpression and knockdown of YY1 in CAD cells. The anti-YY1 antibody <br />                                       immunoprecipitated the R3 and R7, but not the R5, fragments, suggesting an association of <br />                                       YY1 with DNA elements within these fragments (Fig. 3f). The knockdown of YY1 also <br />                                       reduced YY1 binding to the R3 and R7 miR-134 promoter regions in cultured CAD cells <br />                                       (Fig. 3f, lower left). Moreover, the binding of SIRT1 to R3 and R7 was impaired after YY1 <br />                                       knockdown (Fig. 3f, lower right), suggesting that SIRT1 cooperates with YY1 in binding to <br />                                       the upstream regulatory elements of miR-134. Consistent with this idea, in SIRT1 <br />                                       constitutive KO mice, YY1 binding to R3 and R7 of miR-134 was also reduced (Fig. 3g). To <br />                                       determine the functional consequence of YY1 binding to R3 and R7, we used the pCL3- <br />                                       promoter luciferase reporter constructs containing R3, R5, or R7 as described in Fig 3c. We <br /> HHMI Author Manuscript <br />  <br />  <br />  <br />  <br />                                       found that YY1 overexpression repressed, whereas YY1 knockdown potentiated, the <br />                                       luciferase activity driven by R3 and R7 (Fig. 3h). Conversely, YY1 abundance had no <br />                                       influence on the luciferase activity driven by R5. These results suggest that the binding of <br />                                       YY1 to R3 and R7 represses transcription. We then investigated the influence of YY1 upon <br />                                       miR-134 and CREB protein abundance in CAD cells, finding that overexpression of YY1 <br />                                       reduced miR-134 expression, while miR-134 abundance was increased following the <br />                                       shRNA-mediated knockdown of YY1 (Fig. 3i). Furthermore, CREB levels were positively <br />                                       regulated by YY1 expression (Fig. 3j). Collectively, these observations support the concept <br />                                       that SIRT1 is recruited to YY1 DNA binding elements and that the two proteins collaborate <br />                                       to suppress miR-134 expression. <br />  <br />                                       We next examined the role of miR-134 in synaptic plasticity and memory formation. <br />                                       miR-134 was overexpressed in area CA1 of the hippocampus via lentiviral-mediated <br />                                       delivery (Fig. S7a). As observed in SIRT1&#206;&#8221; mice, overexpression of miR-134 abrogated <br />                                       LTP in CA1 neurons (Fig. 4a), while basal synaptic transmission was not impaired (Fig. <br />  <br />  <br />                                              Nature. Author manuscript; available in PMC 2011 February 1. <br />                           Gao et al.                                                                                            Page 5 <br />  <br />  <br />                                       S7b). miR-134 overexpression in area CA1 also resulted in a significant impairment in long- <br />                                       term memory formation in the contextual fear-conditioning paradigm (Fig. 4b), but not in <br />                                       the tone-dependent fear conditioning paradigm (Fig. S7c), a task reliant upon amygdalar <br />                                       function24. miR-134 overexpression did not affect shock sensitivity (Fig. S7d). Thus, <br /> HHMI Author Manuscript <br />  <br />  <br />  <br />  <br />                                       miR-134 overexpression in the hippocampus closely mimics the effects of SIRT1 loss. <br />                                       These data demonstrate that the upregulation of miR-134 plays a role in the synaptic <br />                                       plasticity impairment observed in SIRT1&#206;&#8221; mice. <br />  <br />                                       To assess, in vivo, whether miR-134 upregulation underlies the impairment of LTP observed <br />                                       in SIRT1&#206;&#8221; mice, we examined the effect of miR-134 knockdown using injections of the <br />                                       LNA-miR-134 probe into hippocampal CA1 region (Figs. S8a,b). We found that knockdown <br />                                       of miR-134 restored LTP in acute hippocampal slices from SIRT1&#206;&#8221; hippocampi (Fig. 4c), <br />                                       whereas the LNA probe containing a scrambled sequence (LNA-scr-miR) failed to <br />                                       ameliorate the LTP defects in SIRT1&#206;&#8221; mice. Moreover, injections of LNA-miR-134, but not <br />                                       LNA-scr-miR, into SIRT1&#206;&#8221; mouse hippocampus markedly rescued contextual (Fig. 4d), but <br />                                       not tone-dependent (Fig. S8c), fear memory formation. All groups exhibited normal shock <br />                                       sensitivity (Fig. S8d). Examination of hippocampal lysates harvested from control and <br />                                       SIRT1&#206;&#8221; mice injected with LNA-scr-miR revealed that both CREB and BDNF protein <br />                                       levels were reduced in SIRT1&#206;&#8221; mice (Fig 4e). However, LNA-miR-134 treatment restored <br />                                       CREB and BDNF protein to levels comparable to control mice (Fig. 4e). <br />  <br />                                       Our data indicate that the impairments in memory and synaptic plasticity observed in <br /> HHMI Author Manuscript <br />  <br />  <br />  <br />  <br />                                       SIRT1&#206;&#8221; mice are, at least, partly mediated via an up-regulation of miR-134 and consequent <br />                                       translational inhibition of miR-134 target genes. SIRT1 normally functions in cooperation <br />                                       with YY1, and potentially additional proteins, to restrict the expression of miR-134 and that, <br />                                       upon SIRT1 loss-of-function, higher levels of miR-134 negatively regulate synaptic <br />                                       plasticity via the translational block of key plasticity proteins such as CREB (Fig. S1), <br />                                       which subsequently mediates the various synaptic plasticity impairments observed following <br />                                       SIRT1 loss-of-function. <br />  <br />                                       We previously demonstrated that SIRT1 promotes neuronal survival in age-dependent <br />                                       neurodegenerative disorders5. We have now found that SIRT1 activity also promotes <br />                                       plasticity and memory in a direct manner through a mechanism distinct from its established <br />                                       neuroprotective activity. This result demonstrates a multi-faceted role of SIRT1 in the brain, <br />                                       further highlighting its potential as a target for the treatment of neurodegeneration and <br />                                       conditions with impaired cognition, with implications for a wider range of CNS disorders. <br />  <br />                             Methods Summary <br /> HHMI Author Manuscript <br />  <br />  <br />  <br />  <br />                                       For detailed methods, please see the supplemental materials. The SIRT1 KO and <br />                                       SIRT1&#206;&#8221;ex4/Nestin-Cre mice were provided by the laboratory of Leonard Guarente. The <br />                                       plasmids and LNA-miRNA (Ambion) were transfected into mouse CAD cells using <br />                                       Lipofectamine 2000 (Invitrogen) according to the manufacturer&#226;&#8364;&#8482;s instructions. RNA <br />                                       extraction, purification, and quantitative PCR were performed according to the <br />                                       manufacturer&apos;s protocols. Tissue and cell lysis, protein concentrations, and western blot <br />                                       analyses were prepared as described previously5. Immunoblot data were quantified by <br />                                       measuring the band intensity with NIH imaging software and UN-SCAN-it gel digitizing <br />                                       software (Silk Scientific). Immunostaining was performed as described previously3 with <br />                                       LSMeta10 software and a confocal microscope (Zeiss). All behavioral testing was <br />                                       performed as described previously5 and elsewhere. The data were analyzed by unpaired <br />                                       Student&#226;&#8364;&#8482;s t-test. Two-way ANOVA was used to compare differences between groups at <br />                                       several time points. <br />  <br />  <br />  <br />  <br />  <br />  <br />                                       Figure 1. SIRT1 loss-of-function impairs memory and synaptic plasticity <br />                                       (a) Freezing time of SIRT1&#206;&#8221; mice and littermate controls (Cont) 24 hr after contextual fear <br />                                       conditioning training. (b) Shock sensitivities did not differ between control and SIRT1&#206;&#8221; <br />                                       mice. (c) SIRT1&#206;&#8221;and control mice were tested for novel object discrimination 24 hrs after <br />                                       training. (d) Pre-test exploration times were equal for each object. (e) Escape latencies of <br />                                       SIRT1&#206;&#8221; and control mice were examined with the Morris water maze hidden platform test. <br />                                       (f) Left panel: the swimming time spent in each quadrant in the probe trial on day 5 (T, <br />                                       target; L, left; O, opposite; R, right). Right panel: representative path tracings of the probe <br />                                       test. (g) LTP measurements were performed in the CA1 region of acute slices from SIRT1&#206;&#8221; <br /> HHMI Author Manuscript <br />  <br />  <br />  <br />  <br />                                       mice and controls. (h) SVP immunoreactivity in SIRT1&#206;&#8221; and control hippocampi. (i) <br />                                       Western blots from hippocampal lysates examined SVP in SIRT1&#206;&#8221; and control mice. (j) <br />                                       The density of Golgi-impregnated hippocampal neuronal dendritic spines was measured in <br />                                       SIRT1&#206;&#8221; and control mice. Scale bar: 10 &#206;&#188;m. TBS: theta-burst stimulation, SVP: <br />                                       synaptophysin, OD: optical density. *p &lt; 0.05, **p &lt; 0.01, ***p &lt; 0.001. All histograms <br />                                       represent average &#194;&#177; SEM. <br />  <br />  <br />  <br />  <br />                                              Nature. Author manuscript; available in PMC 2011 February 1. <br />                           Gao et al.                                                                                      Page 9 <br /> HHMI Author Manuscript <br /> HHMI Author Manuscript <br />  <br />  <br />  <br />  <br />                                       Figure 2. BDNF and CREB are downregulated, while miRNA-134 is up-regulated, in SIRT1&#206;&#8221; <br />                                       mice <br />                                       (a) BDNF in the SIRT1&#206;&#8221; and control mouse hippocampus. Left: mRNA levels. Right: <br />                                       Western blot. (b) Chromatin immunoprecipitation (ChIP) with an anti-CREB antibody was <br />                                       followed by qPCR for BDNF promoters 1, 2, and 4. (c) CREB in the SIRT1&#206;&#8221; and control <br />                                       mouse hippocampus. Left: Western blot. Right: mRNA levels. (d) qPCR of selected <br /> HHMI Author Manuscript <br />  <br />  <br />  <br />  <br />                                       miRNAs from hippocampi of SIRT1&#206;&#8221; and control mice. (e) CAD cells were transfected <br />                                       with the plasmids or LNA indicated together with CRE-Luc. (f) Luciferase reporter <br />                                       constructs containing either a wildtype (WT-CREB) or a mutated (mut-CREB) CREB <br />                                       3&#226;&#8364;&#8482;UTR region, were cotransfected with miR-134 or control. (g) CREB protein expression in <br />                                       CAD cells after transfection with the indicated constructs or LNAs. Scr: scrambled, mut: <br />                                       mutant, LNA: locked-nucleic-acid (LNA), CRE-Luc: CREB activity reporter construct. **p <br />                                       &lt; 0.01; ***p &lt; 0.001; NS, not significant. <br />  <br />  <br />  <br />  <br />                                             Nature. Author manuscript; available in PMC 2011 February 1. <br />                           Gao et al.                                                                                        Page 10 <br /> HHMI Author Manuscript <br /> HHMI Author Manuscript <br />  <br />  <br />  <br />  <br />                                       Figure 3. SIRT1 regulates CREB through miR-134 <br />                                       (a) From SIRT1 ChIP, two genomic regions, R3 and R7, corresponding to <span id='am-238' about='xsp:base' typeof='owl:Thing'>base</span> pairs <br />                                       1418~830 and 3427~2901 upstream of miR-134, respectively, were amplified from the wild <br />                                       type, but not the SIRT1 total knockout (SIRT1KO), brain tissue. (b) miR-134 promoter <br /> HHMI Author Manuscript <br />  <br />  <br />  <br />  <br />                                       regions R3 and R7 were quantified from SIRT1 ChIP with qPCR. (c) Reporter constructs <br />                                       containing R3, R5, or R7 regions upstream of a minimal promoter in a luciferase reporter <br />                                       were co-transfected with SIRT1, SIRT1 shRNA, or empty vector. (d) qPCR for miR-134 in <br />                                       CAD cells after transfection with the indicated plasmids. (e) CRE-luciferase reporter assay <br />                                       in CAD cells. (f) ChIP was performed from CAD cells with anti-YY1 and anti-SIRT1 <br />                                       antibodies. (g) Anti-YY1 ChIP followed by qPCR for regions R3 and R7. (h) Luciferase <br />                                       reporter constructs containing R3, R5, or R7 regions were co-transfected with the indicated <br />                                       plasmids. (i) miR-134 levels were measured in CAD cells with qPCR after transfection with <br />                                       the indicated plasmids. (j) Western blotting measured CREB protein levels in CAD cells <br />                                       after transfections with the indicated plasmids. shRNA Cont: scrambled shRNA control. *p <br />                                       &lt; 0.05; **p &lt; 0.01; ***p &lt; 0.001; NS: not significant. All histograms represent average &#194;&#177; <br />                                       SEM. <br />  <br />  <br />  <br />  <br />                                             Nature. Author manuscript; available in PMC 2011 February 1. <br />                           Gao et al.                                                                                      Page 11 <br /> HHMI Author Manuscript <br /> HHMI Author Manuscript <br />  <br />  <br />  <br />  <br />                                       Figure 4. MiR-134 knockdown rescues the LTP and memory impairments caused by SIRT1 <br />                                       deficiency <br />                                       (a) LTP was measured in acute hippocampal slices of mice six weeks after injection with <br />                                       Lv-miR-134 or Lv-Scr-miR. (b) Lv-miR-134 and Lv-Scr-miR-injected mice were tested <br />                                       with a contextual fear conditioning task. (c) Following hippocampal injections with LNA- <br />                                       miR-134 or LNA-scr-miR, LTP was measured in SIRT1&#206;&#8221; and control mouse acute <br />                                       hippocampal slices. (d) Freezing behavior in the contextual fear conditioning task was <br />                                       examined in SIRT1&#206;&#8221; and control mice after hippocampal injections of LNA-miR-134 or <br />                                       LNA-scr-miR. (e). Western blot for CREB and BDNF in brain lysate after in vivo miR-134 <br />                                       knockdown. Control scrambled miR lentivirus: Lv-Scr-miR. **p &lt; 0.01, NS: not significant. <br /> HHMI Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Nature. Author manuscript; available in PMC 2011 February 1. <br />   <br /> </body></html>