<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Proc. Natl. Acad. Sci. USA <br /> Vol. 96, pp. 3552&#226;&#8364;&#8220;3555, March 1999 <br /> Biochemistry <br />  <br />  <br />  <br /> Replication fork assembly recombination intermediates is <br /> required bacterial growth <br /> JOING LIU&#226;&#8364;&#160;, LIEWEI XU&#226;&#8364;&#161;, STEVEN J. SANDLER&#194;&#167;,                     AND   KENNETH J. MARIANS&#226;&#8364;&#160;&#226;&#8364;&#161;&#194;&#182;i <br /> &#226;&#8364;&#160;Molecular Biology Graduate Program &#226;&#8364;&#161;Biochemistry Structural Biology Graduate Program, Cornell University Graduate School Medical Sciences, <br /> New York, NY 10021; &#194;&#167;Department Microbiology, University Massachusetts Amherst, Amherst, MA 01003; &#194;&#182;Molecular Biology Program, Memorial <br /> Sloan&#226;&#8364;&#8220;Kettering Cancer Center, New York, NY 10021 <br />  <br /> Edited Charles M. Radding, Yale University School Medicine, New Haven, CT, approved January 27, 1999 (received review <br /> November 16, 1998) <br />  <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span>          PriA, 3* 3 5* DNA helicase, directs assem-                      (12). Single-stranded DNA-binding protein (SSB) purified <br /> bly primosome bacteriophage plasmid DNAs.                          according Minden Marians (13). The DNA polymerase <br /> Primosomes multienzyme replication machines con-                           III holoenzyme reconstituted Pol III* b <br /> tribute DNA-unwinding Okazaki fragment-                                subunit described Wu et al. (14) purified subunits <br /> priming functions replication fork. The role PriA                      described Marians et al. (15). <br /> chromosomal replication unclear. The phenotypes priA                             Preparation D Loop Template DNA. A 100-nt-long oli- <br /> null mutations suggest protein participates repli-                      gonucleotide having sequence 59-ACATACATAAAGG- <br /> cation restart recombination intermediates. We                         TGGCAACGCCATTCGAAATGAGCTCCATATGCTAG- <br /> that PriA promotes replication fork assembly D loop,                        CTAGGGAGGCCCCCGTCACAATCAATAGAAAATT- <br /> intermediate formed initiation homologous recom-                          CATATGGTTTACCAGCGC-39 annealed f1R408 <br /> bination. We DnaC810, encoded naturally                         viral DNA (16). The central 42 nt oligonucleotide are <br /> arising intergenic suppressor allele priA2::kan muta-                        nonhomologous template. The heteroduplex was <br /> tion, bypasses need PriA replication fork                            converted nicked form II DNA 42-nt-long bubble <br /> assembly D loops vitro. These findings underscore                         region incubation DNA polymerase III holoen- <br /> essentiality replication fork restart recombination inter-                    zyme presence SSB. During 2 min this <br /> mediates normal growth conditions bacteria.                                incubation, 29,39-dideoxynucleotides ddTTP ddATP at <br />                                                                                     concentrations 20-fold higher dTTP dATP were <br /> Primosome assembly bacteriophage fX174 DNA requires                              introduced reaction ensure complementary <br /> seven proteins: PriA, PriB, PriC, DnaT, DnaB, DnaC,                             strand synthesized extended further. After ex- <br /> DnaG (1). Only proteins, DnaB, replication                       traction phenol precipitation ethanol, DNA <br /> fork helicase (2), DnaC, DnaG, replication fork                             products purified electrophoresis nondena- <br /> primase, (3) known required cellular replication                     turing agarose gels. Complete form II bubble DNA was <br /> <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> chromosomal origin, oriC (4). However, strains                             recovered gel 59-32P-labeled minus-strand <br /> carrying priA null mutations constitutively induced                     oligonucleotide sequence 59-ATATAAAAGAAAC- <br /> SOS response (5). We suggested observation indi-                          GCAAAGACACCACGGAATAGTTTATTTT-39 then <br /> cated replication forks formed oriC did                            annealed form D loop form II template. The template <br /> complete replication chromosome rescued                        gel filtered Bio-Gel A5M remove both <br /> by PriA-dependent process (6). Subsequent genetic analyses                        unannealed oligonucleotide unincorporated [g-32P]ATP. <br /> demonstrated priA null strains acquired suppressor mu-                            DNA Replication Reaction. Reaction mixtures (12 ml) con- <br /> tations dnaC rapidly (7, 8) defective                           taining 50 mM Hepes&#226;&#8364;&#8220;KOH (pH 8.0), 10 mM Mg(OAc)2, 10 <br /> homologous recombination (7, 8), repair UV damage (7, 8),                        mM DTT, 80 mM KCl, 200 mgyml BSA, 2 mM ATP, 40 mM <br /> double-strand break-repair (8), induced consti-                        dNTPs, 0.42 nM 32P-labeled form II D loop DNA template, 0.5 <br /> tutive stable DNA replication (9). These results led                         mM SSB, 225 nM DnaC, 30 nM PriA, PriB, PriC, DnaT, <br /> proposal absence PriA-catalyzed replication fork                        DnaB, DNA polymerase III holoenzyme incubated <br /> assembly recombination intermediates account                       37&#194;&#176;C 10 min. Reactions terminated addition <br /> of phenotypes (7, 10). This idea supported                         EDTA 25 mM NaOH 50 mM analyzed by <br /> demonstration PriA specifically bind D loop DNA                          electrophoresis 2 Vycm 20 h room temperature <br /> (11). In report investigated PriA-catalyzed rep-                       horizontal 0.7% alkaline agarose gels using 30 mM <br /> lication fork assembly D loops using defined templates                        NaOH 2 mM EDTA electrophoresis buffer. Gels <br /> and purified DNA replication proteins.                                              neutralized, dried, autoradiographed. When used, <br />                                                                                     PriAK230R, PriAK230A, PriAK230D, DnaC810 were <br />                                                                                     present concentrations wild-type coun- <br />                MATERIALS AND METHODS                                                terparts. <br />    DNA Replication Proteins. A dnaC810 ORF con- <br /> structed means splicing overlap extension PCR                                                      RESULTS <br /> cloned NdeI site pET11C overexpression <br /> plasmid (Novagen). Overexpression purification                                 PriA-Directed Replication Fork Assembly D Loops. The <br />                                                                                     scheme preparing double-stranded circular template <br /> DnaC810 wild-type protein. PriA, PriB, PriC, <br />                                                                                     carrying D loop 39-invading strand given Fig. 1. <br /> DnaT, DnaB, DnaC purified described Marians <br />                                                                                     A replication fork assembled template use the <br /> The publication costs article defrayed page charge <br />                                                                                     This paper submitted directly (Track II) Proceedings office. <br /> payment. This article marked &#226;&#8364;&#732;&#226;&#8364;&#732;advertisement&#226;&#8364;&#8482;&#226;&#8364;&#8482; in <br />                                                                                     Abbreviation: SSB, single-stranded DNA-binding protein. <br /> accordance 18 U.S.C. &#194;&#167;1734 solely indicate fact.                       i <br />                                                                                       <span id='am-5' about='protege:TO' typeof='owl:Thing'>To</span> reprint requests addressed. e-mail: k-marians@ <br /> PNAS available online www.pnas.org.                                             ski.mskcc.org. <br />  <br />                                                                              3552 <br />            Biochemistry: Liu et al.                                                        Proc. Natl. Acad. Sci. USA 96 (1999)            3553 <br />  <br />                                                                          exhibited absolute requirement primosomal <br />                                                                          proteins PriC. Omission protein resulted a <br />                                                                          decrease DNA synthesis one-third complete <br />                                                                          reaction. This result similar observed the <br />                                                                          past replication different templates (17). Some exten- <br />                                                                          sion invading strand holoenzyme be <br />                                                                          observed, suppressed presence PriA. If <br />                                                                          invading strand omitted reaction and <br />                                                                          [a-32P]dATP included, DNA replication be <br />                                                                          observed (data shown). These observations suggested that <br />                                                                          PriA directing primosome assembly D loop. <br />                                                                             In experiment shown Fig. 2, extension invading <br />                                                                          strand result processes: (i) either <br />                                                                          assembly bona fide replication fork D loop followed <br />                                                                          elongation leading strand coupled unwinding of <br />                                                                          duplex DNA template (ii) uncoupled unwinding the <br />                                                                          template DNA leaving oligonucleotide annealed viral <br />                                                                          single-stranded DNA elongated primer- <br />    FIG. 1. Scheme preparation form II D loop template         extension reaction polymerase. We showed previously <br /> DNA. Pol III HE, DNA polymerase III holoenzyme; ss(c), single-           coupled replication fork action requires protein&#226;&#8364;&#8220;protein <br /> stranded circular.                                                       interaction DnaB t subunit holoen- <br />                                                                          zyme (18). In presence interaction replication forks <br /> 39-OH invading strand primer leading-strand            rapidly nearly 1,000 ntysec, its <br /> synthesis. To focus event, radioactive <span id='am-3' about='rdfs:label' typeof='owl:Thing'>label</span>            absence, polymerase stuck slow-moving <br /> present replication reactions 59 end            helicase replication fork progression proceeds only <br /> invading strand oligonucleotide. Extension 39 end          30 ntysec. <br /> minus strand template inhibited                   Accordingly, assessed speed elongation the <br /> presence terminal dideoxy-nucleotide.                               invading strand presence absence t (Fig. 3). For <br />    Incubation D loop template, seven primosomal               experiments used holoenzyme reconstituted from <br /> proteins, SSB, DNA polymerase III holoenzyme,                individually purified subunits. Ten-second time points were <br /> ten-subunit replicase Escherichia coli (1), resulted exten-        taken start reaction, elongated <br /> sion invading strand oligonucleotide (42 nt) full-             products examined denaturing gels. Full-<span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span> ma- <br /> length template size (6.4 kb). This activity seen quite         terial observed presence t 10 sec, <br /> clearly reaction products analyzed dena-              60 sec full-length material observed <br /> turing alkaline agarose gel (Fig. 2). The efficiency              absence. This gives rate replication fork progression <br /> reaction varied, generally 15&#226;&#8364;&#8220;30% (as determined                  presence t 600&#226;&#8364;&#8220;700 ntysec, similar have <br /> PhosphorImager analysis) invading strand                 measured past replication systems (19). Thus, <br /> elongated length 10-min incubation. The reaction            conclude bona fide replication fork assembly was <br />  <br />  <br />  <br />  <br />                                                                             FIG. 3. Bona fide replication forks formed D loop <br />                                                                          template. Standard replication reaction mixtures increased size <br />                                                                          8-fold contained holoenzyme reconstituted purified subunits <br />                                                                          presence absence t. Aliquots (12 ml) were <br />   FIG. 2. Primosomal protein requirements D loop-dependent           removed start incubation (0 sec) 10-sec <br /> DNA replication. The indicated proteins omitted standard       intervals thereafter, reaction products analyzed as <br /> replication reaction mixtures. Temp, template only; HE alone, reaction   described text. The overexposed region the gel <br /> mixture contained DNA polymerase III holoenzyme SSB.        represents unelongated invading strand arises than <br /> The markers HindIII digest bacteriophage l DNA.                1% primer extended minute reaction. <br />  3554      Biochemistry: Liu et al.                                                    Proc. Natl. Acad. Sci. USA 96 (1999) <br />  <br /> occurring D loop template presence        recombination intermediates, DnaC810 able to <br /> primosomal proteins, SSB, holoenzyme.                         bypass requirement PriA recognize D loop and <br />    Replication Fork Assembly Does Not Require PriA                nucleate assembly primosome. Accordingly, tested <br /> Helicase Activity. Although PriA DNA helicase (20, 21),          DnaC810 direct transfer DnaB the <br /> all phenotypes priA null mutations suppressed        D loop template DNA (Fig. 5). <br /> by mutated priA alleles encode PriA proteins            In presence SSB holoenzyme, combina- <br /> longer ATPases DNA helicases, catalyze primo-            tion wild-type DnaC DnaB did support elongation <br /> some assembly (22). These mutations substitutions          invading strand D loop (Fig. 5). This result was <br /> invariant Lys Walker A box nucleotide-binding motif. If        expected, data presented Fig. 2 because <br /> the PriA-dependent replication fork assembly described           SSB binding DNA inhibits DnaB loading absence of <br /> were relevant happened cell, expect           DnaA oriC PriA, PriB, PriC, DnaT on <br /> these mutant proteins substitute fully wild-type PriA       bacteriophage fX174 DNA (1). On hand, DnaC810 <br /> the replication reaction. This proved case. Three           clearly able load DnaB D loop template <br /> mutant proteins, having K230R, K230A, K230D                   absence primosomal proteins, evinced by <br /> substitutions, tested. All supported replication        elongation invading strand length (Fig. 5). <br /> the D loop template greater extent wild-type            Thus, E176G substitution DnaC810 represents true <br /> protein (Fig. 4). We observed improved activity the <br />                                                                       gain-of-function <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span>mutation</span> allows bypass DnaB <br /> mutant proteins systems (22) believe arises <br />                                                                       loading pathway involves PriA, PriB, PriC, DnaT. <br /> because remain bound site DNA binding, <br />                                                                          Interestingly, relative efficiencies replication <br /> providing better target wild-type protein, can <br />                                                                       reactions catalyzed presence DnaC810 DnaB <br /> move site DNA helicase activity. <br />                                                                       varied compared reaction catalyzed complete <br />    DnaC810 Bypasses Requirement PriA, PriB, PriC, <br /> and DnaT During Replication Fork Assembly. Strains carrying           set primosomal proteins. At 80 mM KCl experiment <br /> priA null mutations difficult grow. They rich         shown, DnaC810 reaction 5- 10-fold efficient. <br /> media-sensitive, form huge filaments, viability            However, 600 mM potassium glutamate (data shown), <br /> roughly 1y100th wild type (5, 7, 9). Suppressor           reaction catalyzed complete set proteins more <br /> mutations restore viability, ablate constitutive      efficient factor 2 (the efficiency DnaC810 <br /> induction SOS response defects homologous           reaction decreases, primosomal reaction <br /> recombination repair UV-damaged DNA, arise over-               increases). We assume difference reflects relative <br /> night transduction priA2::kan allele fresh          stability intermediate complexes formed the <br /> recipient cells (7, 9). These mutations map dnaC  <br /> </body></html>