<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Yeast Nop15p is an RNA-binding protein required <br /> for pre-rRNA processing and cytokinesis <br />  <br />  <br />  <br />                              NIFK also localized <span id='am-2' about='protege:TO' typeof='owl:Thing'>to</span> regions of condensed mitotic <br />                                                                    chromosomes, where it may interact with the forkhead- <br />  <br />                                               associated (FHA) domain of the cell proliferation marker <br />  <br />                                                                    pKI-67, and was proposed to function in mitosis (Sueishi <br />  <br />                                                                    et al., 2000; Takagi et al., 2001). A database search with <br />  <br />                                                                    this sequence clearly identi&#194;&#174;ed YNL110c as the probable <br />                                                                    yeast homologue (32% identity, 55% similarity over 174 <br /> Nop15p is an essential protein that contains an RNA <br /> recognition motif (RRM) and localizes to the nucleo-               amino acids). YNL110c/NOP15 was reported to be essen- <br />                                                                    tial for viability in systematic analyses (De Antoni et al., <br /> plasm and nucleolus. Cells depleted of Nop15p failed <br />                                                                    1997; Capozzo et al., 2000). Spores carrying a ynl110D <br /> to synthesize the 25S and 5.8S rRNA components of <br />                                                                    <span id='am-81' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-82' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span>mutation</span> either failed to bud or were arrested after the &#194;&#174;rst <br /> the 60S ribosomal subunit, and exonucleolytic 5&#194;&#162; pro- <br />                                                                    cycle, producing an abnormal bud either smaller or larger <br /> cessing of 5.8S rRNA was strongly inhibited. Pre- <br />                                                                    than wild-type. This led to the suggestion that Ynl110p/ <br /> rRNAs co-precipitated with tagged Nop15p con&#194;&#174;rmed <br />                                                                    Nop15p functions during the cell cycle (Capozzo et al., <br /> its association with early pre-60S particles and <br />                                                                    2000). <br /> Nop15p bound a pre-rRNA transcript in vitro. <br />                                                                       Recent papers have described other factors that function <br /> Nop15p-depleted cells show an unusually abrupt <br />                                                                    in both ribosome synthesis and activities required for cell <br /> growth arrest prior to substantial depletion of riboso- <br /> mal subunits. Following cell synchronization in mito-              cycle progression. Net1p/C&#194;&#174;1p regulates release and <br />                                                                    activation of the Cdc14p phosphatase, thus triggering <br /> sis, Nop15p-depleted cells undergo nuclear division <br />                                                                    mitotic exit (Shou et al., 1999; Straight et al., 1999; <br /> with wild-type kinetics, activate the mitotic exit net- <br /> work and disassemble their mitotic spindle. However,               Visintin et al., 1999), but also promotes transcription of <br />                                                                    the pre-rRNA by RNA polymerase I (Shou et al., 2001). <br /> they uniformly arrest at cytokinesis and fail to assem- <br />                                                                    Yph1p/Nop7p and Noc3p are associated with the origin of <br /> ble a contractile actin ring at the bud neck. In dividing <br />                                                                    replication complex (ORC) and play key roles in the <br /> wild-type cells, segregation of nucleolar proteins to the <br />                                                                    initiation of DNA replication (Du and Stillman, 2002; <br /> daughter nuclei occurs after separation of the nucleo- <br />                                                                    Zhang et al., 2002). However, both Nop7p and Noc3p are <br /> plasm. In these late mitotic cells, Nop15p was partially <br />                                                                    also components of pre-60S particles and each is required <br /> delocalized <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> the nucleolus to the nucleoplasm, <br />                                                                    for pre-rRNA processing, 60S subunit synthesis and export <br /> consistent with a speci&#194;&#174;c function in cell division in <br />                                                                    to the cytoplasm (Harnpicharnchai et al., 2001; Milkereit <br /> addition to its role in ribosome synthesis. <br />                                                                    et al., 2001; Adams et al., 2002; Oef&#194;&#174;nger et al., 2002). <br /> Keywords: cell division/nucleolus/pre-rRNA/ribosome <br /> synthesis/Saccharomyces cerevisiae                                 Another 60S synthesis factor Sda1p (Ba&#195;&#187;ler et al., 2001; <br />                                                                    Harnpicharnchai et al., 2001; Ihmels et al., 2002) is <br />                                                                    required to pass the `Start&apos; checkpoint at the G1&#194;&#177;S <br />                                                                    boundary (Buscemi et al., 2000; Zimmerman and <br /> Introduction                                                       Kellogg, 2001) and one allele was also reported to lead <br /> Most steps in ribosome synthesis take place within the             to defects in the actin cytoskeleton. Moreover, a high- <br /> nucleolus, a specialized subnuclear structure. During              throughput screen implicated several other ribosome <br /> ribosome synthesis, a complex processing pathway con-              synthesis factors and ribosomal proteins in determining <br /> verts a large pre-rRNA to the mature 18S, 5.8S and 25S/            the timing of Start, and therefore the size of the cell at <br /> 28S rRNAs. In addition, the mature rRNA sequences                  division (Jorgensen et al., 2002). <br /> within the pre-RNA undergo extensive covalent nucleotide              While this work was in progress, Nop15p was identi&#194;&#174;ed <br /> modi&#194;&#174;cation and assemble with ~80 ribosomal proteins.              as a component of a pre-60S particle, and a strain <br /> More than 140 non-ribosomal proteins that are required for         expressing a haemaglutinin (HA)-tagged form of <br /> ribosome synthesis have been identi&#194;&#174;ed by genetic and              Nop15p showed some reduction in 60S subunit accumu- <br /> biochemical approaches in Saccharomyces cerevisiae (see            lation (Harnpicharnchai et al., 2001). Here we show that <br /> Kressler et al., 1999; Venema and Tollervey, 1999;                 Nop15p is required for pre-rRNA processing, but also has <br /> Warner, 2001; Fatica and Tollervey, 2002).                         a speci&#194;&#174;c function in cell cycle progression. <br />    Far fewer human ribosome synthesis factors have been <br /> characterized, but the product of the open reading frame <br /> (ORF) NNP18/NOPP34/hNIFK was shown to be associ- <br />                                                                    Results <br /> ated with pre-ribosomal particles (Fujiyama et al., 2002).         Cells depleted of Nop15p are defective in 60S <br /> NIFK was reported to localize to the nucleolus (Takagi             subunit synthesis and pre-rRNA processing <br /> et al., 2001) and was identi&#194;&#174;ed in a proteomic analysis of         The predicted ORF for YNL110c/NOP15 contains 220 <br /> puri&#194;&#174;ed human nucleoli (Andersen et al., 2002). However,           amino acids (25.4 kDa), and analysis with PSORT (Nakai <br />  <br /> &#195;&#163; European Molecular Biology Organization                                                                                 6573 <br />  M.Oef&#194;&#174;nger and D.Tollervey <br />  <br />  <br />                                                                                and Horton, 1999) identi&#194;&#174;ed one canonical RNA recog- <br />                                                                                nition motif (RRM; residues 92&#194;&#177;168) and two putative <br />                                                                                nuclear localization signals (NLS), consistent with a role <br />                                                                                in nuclear RNA processing. <br />                                                                                   To examine the function of Nop15p, its expression was <br />                                                                                placed under the control of a repressible galactose <br />                                                                                promoter using a one-step PCR technique in strain <br />                                                                                YDL401 (see Materials and methods). Growth of the <br />                                                                                GAL::nop15 strain (YMO11) was not clearly different <br />                                                                                from that of the isogenic wild-type strain (YDL401) on <br />                                                                                permissive, galactose medium, but was progressively <br />                                                                                slowed following transfer to repressive, glucose medium <br />                                                                                (Figure 1). Mild growth impairment was initially seen, <br />                                                                                commencing ~6 h after transfer, with much more stringent <br />                                                                                growth inhibition after 20 h. Both the sharp in&#194;&#175;ection point <br />                                                                                in the growth curve and the complete growth inhibition are <br /> Fig. 1. Depletion of Nop15p leads to abrupt growth arrest. Growth              unusual features for strains in which ribosome synthesis <br /> curves of GAL::nop15 strains following transfer to glucose medium,             factors are depleted under GAL repression, which is never <br /> with and without expression of Rpl25p-eGFP. Strains were pre-grown             complete (see below). <br /> in RGS medium and transferred to glucose medium for the times                     To determine whether depletion of Nop15p inhibits <br /> indicated. Strains were maintained in exponential growth by dilution <br /> with pre-warmed medium. Cell densities measured by OD600 are shown             rRNA synthesis, metabolic labelling with [H3]uracil was <br /> corrected for dilution. Wild-type (open squares); GAL::nop15; (&#194;&#174;led            performed 16 h after transfer to glucose minimal medium <br /> diamonds); GAL::nop15; rpl25-GFP (&#194;&#174;lled squares).                              (Figure 2). In the GAL::nop15 (YMO14) strain, synthesis <br />  <br />  <br />  <br />  <br /> Fig. 2. Pulse&#194;&#177;chase analysis of rRNA synthesis. (A) Structure and processing sites of the 35S pre-rRNA. This precursor contains the sequences for the <br /> mature 18S, 5.8S and 25S, which are separated by the two internal transcribed spacers ITS1 and ITS2 and &#194;&#175;anked by the two external transcribed <br /> spacers 5&#194;&#162;ETS and 3&#194;&#162;ETS. The positions of oligonucleotide probes utilized in northern hybridization and primer extension analyses are indicated. <br /> (B) Pre-rRNA processing pathway. For further discussion of the processing pathway and enzymes, see Kressler et al. (1999), Venema and Tollervey <br /> (1999), Warner (2001) and Fatica and Tollervey (2002). (C and D) Pre-rRNA was pulse-labelled with [3H]uracil for 2 min at 30&#194;&#176;C and chased with a <br /> large excess of unlabelled uracil for the times indicated. Labelling was performed for the GAL::nop15 strain and a wild-type strain 16 h after transfer <br /> to glucose medium. (C) High molecular weight RNA separated on a 1.2% agarose/formaldehyde gel. (D) Low molecular weight RNA separated on an <br /> 8% polyacrylamide/urea gel. <br />  <br />  <br /> 6574 <br />                                                                                           Ribosome synthesis and cell division <br />  <br />  <br /> of mature 25S and 5.8S rRNA was almost completely                depletion of the cytoplasmic ribosomes or defects in <br /> inhibited (Figure 2C and D). The 27SA pre-rRNA was               assembly or export of the 60S subunits. <br /> accumulated, while the 27SB and 7S pre-rRNAs were lost. <br /> In contrast, synthesis of the 20S pre-rRNA and 18S rRNA          Nop15p is associated with precursors to the 25S <br /> remained robust. Synthesis of the 5S rRNA and tRNA was           and 5.8S rRNAs <br /> also little affected (Figure 2D).                                RNAs that co-precipitated with TAP-tagged Nop15p <br />     The effects of Nop15p depletion on pre-rRNA process-         (Figure 4A&#194;&#177;C, Nop15p-TAP lanes) were compared with <br /> ing were assessed in more detail by northern hybridization       a mock precipitation from the non-tagged parental strain <br /> and primer extension (Figure 3). Depletion of Nop15p             YDL401 (Figure 4A&#194;&#177;C, Mock lanes). Northern hybridiza- <br /> resulted in some accumulation of the 35S primary                 tion (Figure 4A) showed that the 7SL and 7SS pre-rRNA <br /> transcript but had little impact on levels of the 27SA2 or       components of the pre-60S ribosomes co-precipitated with <br /> 20S pre-rRNAs (Figure 3A). This indicates that the early         Nop15p-TAP, but were not detectably recovered in a mock <br /> cleavages at sites A0, A1 and A2 are kinetically delayed but     precipitation from YDL401. The U3 snoRNA, a <br /> still occur with good ef&#194;&#174;ciency. In contrast, the levels of      component of the 90S pre-ribosomes, was precipitated <br /> the 27SB pre-rRNAs were mildly reduced after 8 h and             with lower ef&#194;&#174;ciency. The 6S pre-rRNA and mature <br /> strongly reduced by 16 h after transfer to glucose medium.       rRNAs were not co-precipitated detectably. High mol- <br /> Levels of the 27SA2 + A3 pre-rRNAs, both of which are            ecular weight pre-rRNAs were analysed by primer exten- <br /> detected by oligo 020 (Figure 3A, b), were mildly                sion (Figure 4B and C). The 27SA2, 27SBL, 27SBS and <br /> increased following Nop15p depletion. This is in contrast        26S pre-rRNA components of pre-60S ribosomes were co- <br /> to the depletion of 27SA2 (Figure 3A, a), suggesting that        precipitated with Nop15p-TAP. In contrast, no co-pre- <br /> the 27SA3 pre-rRNA was accumulated. This was con-                cipitation of the 35S pre-rRNA was observed. <br /> &#194;&#174;rmed by primer extension (Figure 3C), which showed                 These results show that Nop15p is associated with the <br /> little alteration in the level of 27SA2, as shown by the         early pre-60S ribosomes. The precipitation of the U3 <br /> primer extension stop at site A2, and strong accumulation        snoRNA reveals some association with earlier pre- <br /> of 27SA3, shown by the stop at site A3. The level of 27SBS       ribosomes. However, the lack of co-precipitated 35S <br /> was clearly reduced, whereas the level of 27SBL appeared         indicates binding only to the later 32S pre-rRNA, upon <br /> unaffected, as shown by the primer extension stops at B1S        which A2 cleavage occurs. Nop15p has an RRM and may <br /> and B1L, respectively. The exonucleases Rat1p and Xrn1p          bind directly to the pre-rRNA, probably immediately prior <br /> normally digest the pre-rRNA from site A3 to site B1S            to cleavage at site A2, and remains associated with the pre- <br /> (Henry et al., 1994), and the loss of 27SBS is therefore a       60S particle until after cleavage at site C2. <br /> consequence of defective processing of 27SA3.                       To test directly whether Nop15p exhibits RNA binding <br />     Analysis of low molecular weight RNAs (Figure 3B)            activity, a gel shift assay was performed (Figure 4D). A <br /> showed that the 7S and 6S pre-rRNAs were mildly                  labelled pre-rRNA transcript was prepared, which extends <br /> depleted after 8 h and strongly depleted by 16 h after           from the 5&#194;&#162; region of ITS1 to the 3&#194;&#162; region of ITS2 and <br /> transfer to glucose medium (Figure 3Ba). In contrast to the      includes sites A2, A3, B1 and C2. Labelled transcript (10 <br /> 27SB pre-rRNAs, no clear differences were observed in            fmol) was incubated with 0&#194;&#177;30 fmol of recombinant <br /> the ratios of the long and short forms of the 7S pre-rRNA        Nop15p, which was expressed in Escherichia coli as a <br /> (7SL and 7SS), showing that the processing of 27SBL to           thrombin-cleavable GST fusion construct (see Materials <br /> 7SL is also inhibited (see Figure 2A). Little depletion of       and methods). Free and bound forms of the RNA were <br /> mature 18S, 25S or 5.8S rRNA was seen even at late time          resolved by native gel electrophoresis (Fatica et al., 2002). <br /> points. We conclude that Nop15p is required for                  Nop15p clearly retarded the migration of the RNA, <br /> exonucleolytic processing of the 27SA3 pre-rRNA to               although the bound material did not migrate as a discrete <br /> 27SBS. The alternative form, 27SBL, continues to be              species. Binding was inhibited by cold competitor pre- <br /> synthesized but fails to be converted to 7SL.                    rRNA transcript, but was not inhibited by a 200-fold molar <br />     These results demonstrate that Nop15p is a bona &#194;&#174;de          excess of tRNA. We conclude that Nop15p has RNA <br /> pre-rRNA processing factor with speci&#194;&#174;c functions in the         binding activity. Further analyses will be required to <br /> synthesis of the 25S and 5.8S rRNAs. It is, however,             determine whether this in vitro binding is sequence <br /> striking that the mature rRNAs were not substantially            speci&#194;&#174;c. <br /> depleted even 24 h after transfer to glucose medium. The <br /> mature rRNAs are stable in most backgrounds and must be          Cells depleted of Nop15p become non-viable and <br /> depleted by growth, which is very slow in the Nop15p-            arrest with abnormal morphology <br /> depleted strain at late times.                                   The association of Nop15p with pre-rRNAs and the <br />     To test for potential defects in 60S subunit export, the     processing defects seen following its depletion clearly <br /> Rpl25p&#194;&#177;enhanced green &#194;&#175;uorescent protein (eGFP)                  demonstrate that Nop15p is a genuine ribosome synthesis <br /> reporter (Gadal et al., 2001) was expressed in the               factor. However, the sudden and dramatic growth arrest <br /> GAL::nop15 strain. No nuclear accumulation of Rpl25p-            seen on depletion of Nop15p (Figure 1) is not expected for <br /> eGFP was observed on Nop15p depletion (data not                  strains carrying a GAL-regulated allele of a ribosome <br /> shown). Expression of Rpl25p-eGFP synergistically                synthesis factor and has not been seen in many other <br /> inhibits growth in some mutants with ribosome assembly           published analyses. GAL repression is not complete, and <br /> defects (Oef&#194;&#174;nger et al., 2002) but did not alter the kinetics   such strains generally continue growth at a reduced rate <br /> of growth inhibition in the GAL::nop15 strain (Figure 1).        that can be supported by the residual ribosome synthesis. <br />     Together, these observations indicate that growth arrest     The growth arrest in the Nop15p-depleted strain therefore <br /> in Nop15p-depleted strains is not a consequence of               suggests a function in addition to ribosome synthesis, <br />  <br />                                                                                                                         6575 <br />  M.Oef&#194;&#174;nger and D.Tollervey <br />  <br />  <br />  <br />  <br />                                                                             Fig. 4. Nop15p associates with pre-rRNAs in vivo and in vitro. (A&#194;&#177; <br />                                                                             C) Nop15p-TAP co-precipitates with pre-rRNAs. Lane 1, total RNA <br /> Fig. 3. Analysis of pre-rRNA processing. (A) Northern analysis. Lanes       control (5 mg); lane 2, mock precipitate from a wild-type control strain; <br /> 1 and 2, wild-type strain in RGS medium and 24 h after transfer to glu-     lane 3, precipitate from a strain expressing Nop15p-TAP. (A) Northern <br /> cose; lanes 3&#194;&#177;7, GAL::nop15 strain in RGS medium and after transfer         hybridization of high and low molecular weight RNAs separated on a <br /> to glucose medium for the times indicated. (B) Northern analysis.           1.2% agarose/formaldehyde gel or 8% polyacrylamide/urea gel, <br /> Lane 1, wild-type strain 24 h after transfer to glucose; lanes 2&#194;&#177;7,         respectively. (B and C) Primer extension analyses. Nop15p-TAP was <br /> GAL::nop15 strain in RGS medium and after transfer to glucose med-          immunoprecipitated from cell lysates using IgG&#194;&#177;agarose, with release <br /> ium for the times indicated. RNA was separated on a 1.2% agarose/for-       of bound RNA&#194;&#177;protein complexes by cleavage of the protein A linker <br /> maldehyde gel (A) or 8% polyacrylamide/urea gel (B). Probe names            by TEV protease. RNA was recovered from the released material, <br /> are indicated in parentheses on the left. (C) Primer extension using        and from a mock-treated, isogenic wild-type control strain. <br /> oligo 006, which hybridizes within ITS2, 3&#194;&#162; to site C2 (the 3&#194;&#162; end of the   Oligonucleotides used are indicated in parentheses. (D) Nop15p binds <br /> 7S pre-rRNA). Primer extension stops at sites A2, A3, B1S and B1L           to pre-rRNA in vitro. Gel mobility shift assay performed with an <br /> show levels of the 27SA2, 27SA3, 27SBS and 27SBL pre-rRNAs,                 in vitro transcribed pre-rRNA fragment, extending from the 5&#194;&#162; region <br /> respectively. Lanes 1 and 2, wild-type strain in RGS medium and 20 h        of ITS1 to the 3&#194;&#162; region of ITS2. Nop15p was expressed in E.coli as a <br /> after transfer to glucose medium; lanes 3 and 4, GAL::nop15 strain in       GST fusion with a thrombin-sensitive linker and eluted from a <br /> RGS medium and 20 h after transfer to glucose medium.                       glutathione&#194;&#177;Sepharose column by cleavage with thrombin. Lanes 1&#194;&#177;4, <br />                                                                             10 fmol pre-rRNA incubated with 0&#194;&#177;30 fmol Nop15p as indicated; <br />                                                                             lanes 5&#194;&#177;8, competition experiment. The gel shift assay was performed <br />                                                                             in the presence of 200-fold molar excess of cold pre-rRNA (lane 7) and <br /> particularly as it occurs prior to substantial depletion of the             tRNA (lane 8). Reactions in lanes 6&#194;&#177;8 contained 10 fmol RNA and <br /> mature rRNAs.                                                               15 fmol Nop15p. Complexes were resolved by electrophoresis in native <br />                                                                             6% acrylamide/bisacrylamide (80:1) gels containing 0.53 TBE. <br />    Cell viability was assessed during incubation in glucose <br /> medium by plating the cells on solid galactose medium. <br /> Cell numbers were calculated by counting in a haemocyt-                        Microscopic inspection of the GAL::nop15 strain 18 h <br /> ometer slide. At 16 h after transfer to glucose medium, the                 after transfer to glucose medium showed 63% unbudded <br /> viability of the Nop15p-depleted cells had dropped to                       cells (from 600 cells counted), indicating an arrest in G1. <br /> ~20%, 4-fold lower than the wild-type or the GAL::nop15                     However, 37% of the cells showed a distinctive and <br /> strain grown on galactose medium. (Figure 5A). In a                         unusual elongated shape, which was not observed in <br /> similar analysis performed on a GAL::nop1 strain, viabil-                   galactose medium, demonstrating a defect in cell morph- <br /> ity dropped only much later in depletion (commencing                        ology (Figure 5B). These elongated cells contained <br /> after 60 h), and a slow decline in growth rate continued                    duplicated, separated nuclei as shown by 4&#194;&#162;,6-diamidino- <br /> through 80 h of depletion (Tollervey et al., 1991). The                     2-phenylindole (DAPI) staining and had a marked con- <br /> inability of the cells to recover from depletion of Nop15p                  striction corresponding to the bud neck (indicated by an <br /> strongly suggested that reduced numbers of ribosomes is                     arrow in Figure 5B), where cytokinesis should have <br /> not the major cause of growth arrest.                                       occurred. This morphology is distinct from that seen in <br />  <br /> 6576 <br />                                                                                                            Ribosome synthesis and cell division <br />  <br />  <br /> cells defective in the mitotic exit network (MEN), which                      Institute), 35% of cells arrested with large buds and <br /> arrest with an intact, elongated mitotic spindle (Jimenez                     divided nuclei within 2 h of transfer to 37&#194;&#176;C (data not <br /> et al., 1998). To con&#194;&#174;rm this distinction, anti-tubulin                       shown), demonstrating that this <span id='am-3' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-4' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-5' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span>phenotype</span> does not <br /> staining was performed on wild-type and GAL::nop15                            require long incubation in restrictive conditions. <br /> cells after 18 h in glucose medium to observe the                                Bud neck constriction during cytokinesis involves an <br /> organization of microtubules (Figure 5C). None of the                         actin/myosin contractile ring (Bi et al., 1998). The ability <br /> elongated GAL::nop15 cells contained a mitotic spindle                        of the Nop15p-depleted cells to form this ring was <br /> (from &gt;500 cells examined), while the rounded cells                           assessed using rhodamine-conjugated phalloidin to visua- <br /> arrested in G1 displayed an apparently normal interphase                      lize the organization of polymerized or &#194;&#174;lamentous actin <br /> array of microtubules (Figure 5C, d). No cells examined                       (F-actin). In the wild-type (Figure 5D, a, d and j), F-actin <br /> showed morphology consistent with metaphase arrest.                           was observed in cortical patches and was concentrated <br />    A collection of `heat-inducible-degron&apos; strains has been                   around the bud neck in dividing cells (indicated by arrows <br /> described very recently (Kanemaki et al., 2003), in which                     in Figure 5D). The same organization was observed for the <br /> the target protein is rapidly degraded following transfer to                  GAL::nop15 strain in permissive RGS medium (Figure 5D, <br /> 37&#194;&#176;C. In a strain expressing the degron-tagged Nop15p                         b and c) and 14 h after shift to glucose medium (Figure 5D, <br /> construct (generously provided by K.Labib, Paterson                           e and f). After 18 h in glucose medium, the rounded <br />  <br />  <br />  <br />  <br /> Fig. 5. Depletion of Nop15p leads to a cell cycle defect. (A) Percentage of cells from wild-type and GAL::nop15 strains that remain viable when pla- <br /> ted on fresh, pre-warmed YRGS plates following incubation in glucose medium for the times indicated. Viability was determined by comparing the <br /> total number of cells plated with the number of colonies formed at each time point. (B) Cell morphology. (C) Staining with anti-tubulin. (D) Staining <br /> with rhodamine-conjugated phalloidin to detect F-actin. Wild-type and GAL::nop15 strains were examined during growth in RGS medium and follow- <br /> ing transfer to glucose medium for 14 or 18 h. The position of the nucleus was visualized by DAPI staining. In (C) and (D), the &#194;&#174;gures indicated with <br /> an asterisk show phase contrast images of cells in the corresponding immuno&#194;&#175;uorescence images. The bar represents 10 mm and arrows indicate the <br /> position of the bud neck. <br />  <br />  <br />                                                                                                                                               6577 <br />  M.Oef&#194;&#174;nger and D.Tollervey <br />  <br />  <br />  <br />  <br /> 6578 <br />                                                                                                            Ribosome synthesis and cell division <br />  <br />  <br /> GAL::nop15 cells arrested in G1 showed randomly                               presumably leads to the observed failure to undergo <br /> distributed F-actin patches (Figure 5D, l). However,                          cytokinesis. <br /> none of the elongated cells (from &gt;500 cells examined) <br /> contained detectable F-actin at the bud neck (arrow in                        The nucleolus segregates late and Nop15p <br /> Figure 5D, k), indicating that actin is depolymerized in                      becomes partially delocalized during nuclear <br /> these cells.                                                                  separation <br />    To analyse this cell cycle defect better, wild-type and                    Nop15p was reported previously to be localized to the <br /> GAL::nop15 cells were examined following synchroniza-                         nucleolus, based on its exclusion from the nucleoplasmic <br /> tion. Cell division was arrested at the spindle assembly                      region of the nucleus (Harnpicharnchai et al., 2001). This <br /> checkpoint by nocodazole treatment (Tolliday et al.,                          was con&#194;&#174;rmed by the co-localization in indirect immuno- <br /> 2002), which inhibits microtubule assembly, preventing                        &#194;&#175;uorescence of Nop15p-TAP with a DsRed fusion with the <br /> spindle formation and nuclear division. Nocodazole was                        well characterized nucleolar protein Nop1p, the yeast <br /> added 15 h after the shift to glucose medium, at which time                   homologue of human &#194;&#174;brillarin (data not shown) (Gadal <br /> the GAL::nop15 cells are still cycling, and both wild-type                    et al., 2001). In interphase cells, Nop15p-TAP was largely <br /> and mutant uniformly arrested as large-budded cells. After                    excluded from the nucleoplasm, which was decorated by <br /> 3 h incubation, the nocodazole was washed out and nuclear                     DAPI staining Figure 7A, a&#194;&#177;c). The localization of <br /> separation and actin ring formation were monitored                            Nop15p was dramatically altered in the mitotic cells, <br /> microscopically. In both wild-type and Nop15p-depleted                        &#194;&#174;lling the entire nuclear volume, as shown by the co- <br /> cells, mitosis resumed with similar kinetics, as shown by                     localization of the protein A tag with the DAPI-stained <br /> the high ef&#194;&#174;ciency of nuclear separation 30 min after                         region (Figure 7A, d&#194;&#177;f). A second striking observation was <br /> release from nocodazole arrest (Figure 6A and C).                             that in late mitotic cells in which the nucleoplasm of the <br /> However, while the wild-type cells carried on through                         daughter nuclei was well separated as shown by DAPI <br />                                                                               staining (Figure 7A, d), Nop15p-stained material formed a <br /> cytokinesis, the Nop15p-depleted cells uniformly arrested <br />                                                                               bridge between the nuclei (Figure 7A, e). <br /> as large budded cells (Figure 6A). Wild-type cells <br />                                                                                  To determine whether the late segregation of Nop15p is <br /> exhibited a peak of actin ring formation 30 min after <br />                                                                               a general feature of nucleolar proteins, we also examined <br /> release from nocodazole arrest, with ~29% of cells <br />                                                                               the behaviour of Nop1p (Figure 7B). In both interphase <br /> showing an actin ring (Figure 6B and C), in good                              and mitotic cells, Nop1p was largely excluded from the <br /> agreement with previous reports (Tolliday et al., 2002).                      nucleoplasm. In mitotic cells, segregation of Nop1p to the <br /> In striking contrast, no Nop15p-depleted cells with an F-                     daughter nuclei also occurred after substantial separation <br /> actin ring were detected 30, 45 or 60 min after nocodazole                    of the nucleoplasm. <br /> release (from &gt;500 cell examined) (Figure 6B and C).                             We conclude that the nucleolus was divided and <br />    The total level of actin present in the cells was assessed                 segregated to the daughter nuclei only after separation of <br /> by western blotting (Figure 6D), which revealed no clear                      the nucleoplasm. It is clearly essential that cytokinesis <br /> difference between the mutant and wild-type. The ability                      should not commence until the daughter nucleoli have <br /> of the Nop15p-depleted cells to activate the MEN was                          completely separated. This suggests a possible rationale <br /> assessed by examining the localization of Cdc14p&#194;&#177;GFP                          for linking the timing of actin ring formation to nucleolar <br /> (Figure 6E) [kindly provided by E.Schiebel (Hofken and                        components. The release of Nop15p from the nucleolus at <br /> Schiebel, 2002)]. In both the wild-type and Nop15p-                           this time may be functionally related to the triggering of <br /> depleted cells, Cdc14p&#194;&#177;GFP was predominantly nucleo-                          cytokinesis, although the mechanisms involved remain to <br /> plasmic, indicating that Nop15p is not required for its                       be elucidated. <br /> release from nucleolar sequestration. <br />    The Nop15p-depleted cells arrested correctly at the <br /> spindle assembly checkpoint and subsequently were able                        Discussion <br /> to undergo nuclear separation, activation of the MEN and                      Nop15p and ribosome synthesis <br /> disassembly of the mitotic spindle with the same ef&#194;&#174;ciency                    We show here that Nop15p is a bona &#194;&#174;de ribosome <br /> as the wild-type, showing that they do not have a non-                        synthesis factor, which is required for the 5&#194;&#162; to 3&#194;&#162; <br /> speci&#194;&#174;c defect in cell cycle progression. However, the                        exonuclease digestion that generates the 5&#194;&#162; end of the <br /> Nop15p-depleted cells uniformly arrested at cytokinesis                       major, short form of the 5.8S rRNA as well as for <br /> and failed to form an actin ring at the bud neck. We                          processing of 27SB to 7S pre-rRNA. Consistent with this, <br /> conclude that Nop15p is required, directly or indirectly,                     and with recent proteomic analyses (Harnpicharnchai <br /> for the recruitment of actin to the developing bud neck.                      et al., 2001; Gavin et al., 2002), tagged Nop15p was <br /> The consequent failure to form a normal contractile ring                      associated with RNA components of the early pre-60S <br />  <br />  <br /> Fig. 6. Cytokinesis and actin ring formation are inhibited in synchronized, Nop15p-depleted cells. Wild-type and GAL::nop15 cells were shifted to glu- <br /> cose medium for 15 h and then treated with nocodazole for 3 h, arresting the cells in late M phase. Cells were examined before nocodazole washout <br /> (0 min samples) and at times points following release from arrest. (A) Ef&#194;&#174;ciency of nuclear separation. (B) Ef&#194;&#174;ciency of actin ring formation. <br /> (C) Typical wild-type and GAL::nop15 cells visualized 30 min after nocodazole release. F-actin was visualized by staining with rhodamine-conjugated <br /> phalloidin (panels a and b). Nuclei were visualized by DAPI staining (panels c and d). (D) Western blot analysis of Act1p in wild-type and <br /> GAL::nop15 cells. Whole-cell extracts were prepared and equal amounts of protein were separated by SDS&#194;&#177;PAGE. Act1p was detected by western <br /> blotting using primary anti-actin antibody (I-19) and secondary peroxidase-conjugated rabbit-anti goat antibody. (E) Release of Cdc14p from nucleolus <br /> to nucleoplasm during late mitosis. The release of Cdc14p&#194;&#177;GFP during mitotic exit was observed in wild-type (panel b) and GAL::nop15 (panel e) <br /> cells 30 min after release from nocodazole arrest. Nuclei were visualized by DAPI staining (panels a and d). The bar represents 10 mm. <br />  <br />  <br />                                                                                                                                                6579 <br />  M.Oef&#194;&#174;nger and D.Tollervey <br />  <br />  <br />                                                                          depletion of Nop15p in a ts-degron strain (Kanemaki et al., <br />                                                                          2003) led to a similar proportion of cells apparently <br />                                                                          arrested at cytokinesis within 2 h of transfer to the non- <br />                                                                          permissive temperature. Following cell synchronization in <br />                                                                          late M-phase by nocodazole treatment and release, the <br />                                                                          defect in cytokinesis was complete, with all Nop15p- <br />                                                                          depleted cells uniformly arresting at the &#194;&#174;rst cytokinesis. <br />                                                                             In S.cerevisiae, cortical patches of polymerized F-actin <br />                                                                          support the ellipsoid outgrowth of the bud cell wall during <br />                                                                          S phase. During late mitosis, actin is partially relocated to <br />                                                                          the bud neck to form an actin/myosin contractile ring (Bi <br />                                                                          et al., 1998) that will constrict the bud neck leading to <br />                                                                          cytokinesis. Staining with rhodamine-conjugated phalloi- <br />                                                                          din revealed that none of the arrested, Nop15p-depleted <br />                                                                          cells had assembled an actin ring (from &gt;500 examined), <br />                                                                          potentially explaining their failure to complete cytokinesis <br />                                                                          (Bi et al., 1998). General defects in the actin cytoskeleton <br />                                                                          lead to defects in cell polarity and spindle formation <br />                                                                          (Palmer et al., 1992; Ayscough et al., 1997; McMillan <br /> Fig. 7. The nucleolus segregates late during yeast cell division. The    et al., 1998; Theesfeld et al., 1999), which are predicted to <br /> localization of the nucleolus was determined in wild-type interphase <br /> and mitotic cells. (A) The nucleolar marker Nop1p localized with anti-   lead to metaphase arrest. We did not see Nop15p cells that <br /> Nop1p antibody. (B) The localization of Nop15p-TAP. The nucleo-          are arrested in metaphase, and Nop15p-depleted cells that <br /> plasm was visualized by DAPI staining. The bar represents 10 mm.         arrested in G1 retained an apparently normal distribution <br />                                                                          of polymerized actin, indicating that the defect in actin <br /> particles. Weaker co-precipitation was seen for the U3                   assembly is speci&#194;&#174;c for cytokinesis. <br /> snoRNA, a component of the 90S pre-ribosomes, whereas                       The defect in cytokinesis seen in strains lacking Nop15p <br /> the 35S pre-rRNA was not co-precipitated detectably. We                  has not been reported previously for any of the &gt;100 <br /> conclude that Nop15p is likely to associate with the 90S                 characterized ribosome synthesis factors. More speci&#194;&#174;c- <br /> pre-ribosomes after the initial cleavage at site A0 and                  ally, mutations in proteins that are present in the same pre- <br /> immediately prior to cleavage at site A2, remaining                      60S particles as Nop15p (Harnpicharnchai et al., 2001; <br /> associated with the early pre-60S complexes (Fatica and                  Fatica and Tollervey, 2002) have not been observed to <br /> Tollervey, 2002) during processing at sites A3, B1S, B1L                 result in defects in cytokinesis. The pre-rRNA processing <br /> and C2. Nop15p has a good consensus RRM and showed                       defect in Nop15p-depleted cells most closely resembles <br /> RNA binding activity in vitro, and is therefore likely to                that seen on depletion of Nop7p/Yph1p (Adams et al., <br /> bind directly to the pre-rRNA. Alterations in the pre-rRNA               2002; Oef&#194;&#174;nger et al., 2002). Depletion of Yhp1p does <br /> structure induced by binding of Nop15p may be important                  result in a cell cycle phenotype, but this is due to defects in <br /> for the recruitment of the 5&#194;&#162; to 3&#194;&#162; exonucleases Rat1p and               DNA replication (Du and Stillman, 2002) and is clearly <br /> Xrn1p (Henry et al., 1994) that generate the 5&#194;&#162; end of the               distinct from the effects of Nop15p depletion. <br /> 5.8S rRNA.                                                                  Defects in another nucleolar protein, Net1p/C&#194;&#174;1p, also <br />                                                                          result in cell cycle arrest as large-budded cells, due to a <br />                                                                          failure to activate the MEN (Shou et al., 1999; Straight <br /> Nop15p and the cell cycle                                                et al., 1999; Visintin et al., 1999). Nop15p-depleted cells <br /> In cells undergoing depletion of Nop15p, the growth rate                 lacked the characteristic features of MEN defects; a <br /> was only mildly reduced until ~16&#194;&#177;20 h, at which time                    stabilized mitotic spindle and the distinctive bud projec- <br /> growth abruptly ceased. This was accompanied by a                        tions that arise from continued spindle elongation <br /> substantial decline in the viability of cells that were                  (Jimenez et al., 1998). More directly, Nop15p-depleted <br /> returned to permissive medium. Moreover, the abundance                   cells were shown to release Cdc14p&#194;&#177;GFP ef&#194;&#174;ciently from <br /> of the ribosomal subunits was not strongly reduced at the                the nucleolus following release from nocodazole arrest, <br /> time at which growth ceased. This behaviour is not                       demonstrating MEN activation. <br /> common for cells depleted of ribosome synthesis factors,                    Components of the septum formation machinery, <br /> which usually show a slow, progressive increase in                       including the septins and Myo1p, are localized to the <br /> doubling time, as the growth rate falls to match reduced                 future bud site early in the cell cycle (Lippincott and Li, <br /> numbers of ribosomes. A lack of cytoplasmic ribosomes is                 1998). In contrast, actin is relocated to form the actin/ <br /> therefore unlikely to be the principal cause of the growth               myosin ring only late in the cell cycle, following <br /> arrest in Nop15p-depleted cells.                                         nucleoplasmic separation and shortly before disassembly <br />    Microscopic examination revealed two distinct terminal                of the mitotic spindle. Several proteins that are required for <br /> phenotypes in the Nop15p-depleted cells. Around 63% of                   actin ring assembly have been identi&#194;&#174;ed (Osman et al., <br /> cells were rounded and unbudded, consistent with an                      2002; Tolliday et al., 2002), but the mechanism and timing <br /> inability to pass the `Start&apos; cell cycle checkpoint. The                 of its recruitment remain poorly understood. We observed <br /> remaining 37% showed an unusual, elongated morphology                    that segregation of the nucleolar markers Nop15p and <br /> with duplicated and separated nuclei. A constriction was                 Nop1p in dividing yeast cells lags behind the separation of <br /> present marking the site of bud formation, but the cells                 the nucleoplasm. Dividing cells were readily detected in <br /> had clearly been unable to complete cytokinesis. Rapid                   which the nuclei appeared well separated by DAPI <br />  <br /> 6580 <br />                                                                                                             Ribosome synthesis and cell division <br />  <br />  <br /> staining, but were in fact bridged by nucleolar material.                        For pre-rRNA and rRNA analysis the following oligonucleotides were <br />                                                                                used: 002, 5&#194;&#162;-GCTCTTTGCTCTTGCC; 003, 5&#194;&#162;-TGTTACCTCTGGGC- <br /> Late segregation of the nucleolus has been seen previously <br />                                                                                CC; 006, 5&#194;&#162;-AGATTAGCCGCAGTTGG; 007, 5&#194;&#162;-CTCCGCTTATTGA- <br /> in &#194;&#174;ssion yeast (Toda et al., 1981; Hirano et al., 1989), so                   TATGC; 008, 5&#194;&#162;-CATGGCTTAATCTTTGAGAC; 017, 5&#194;&#162;-GCGTTGT- <br /> this may be a conserved feature. In mitotic cells, Nop15p                      TCATCGATGC; 020; 5&#194;&#162;-TGAGAAGGAAATGACGCT; 033, 5&#194;&#162;- CGC- <br /> was partially delocalized from the nucleolus, with staining                    TGCTCACCAATGG; 041, 5&#194;&#162;-CTACTCGGTCAGGCTC; and 200, 5&#194;&#162;- <br /> seen throughout the nucleoplasm. Two regulatory net-                           UUAUGGGACUUGUU. <br /> works are implicated in the release of Cdc14p from the <br />                                                                                Immuno&#194;&#175;uorescence <br /> nucleolus. The FEAR network (Cdc fourteen early release)                       For localization of Nop15p, Nop1p and tubulin, cells were &#194;&#174;xed by <br /> is required for initial release, while the MEN is both                         incubation in 4% (v/v) formaldehyde for 30 min at 25&#194;&#176;C, and <br /> activated by Cdc14p and required to prevent relocation of                      spheroplasted. Immuno&#194;&#175;uorescence was performed as described <br /> Cdc14p back to the nucleolus (Shou et al., 1999; Straight                      (Grandi et al., 1993; Berge&#195;&#65533;s et al., 1994). To stain nuclear DNA, DAPI <br /> et al., 1999; Visintin et al., 1999). The roles of these                       was included in the mounting medium (Vectashield, Vector <br />                                                                                Laboratories). Nop15p-TAP was detected with an anti-protein A antibody <br /> networks in the triggering of nucleolar separation and/or                      coupled to &#194;&#175;uorescein isothiocyanate (FITC). Nop1p was detected with a <br /> release of Nop15p remain to be determined. The late                            mouse anti-Nop1p antibody and a secondary goat anti-mouse antibody <br /> nucleolar separation and/or the release of Nop15p would                        coupled to FITC (Sigma). Tubulin was detected with a rabbit anti-tubulin <br /> be appropriate timing markers for the initiation of actin                      antibody (Sigma) and a secondary goat anti-rabbit antibody coupled to <br />                                                                                FITC (Sigma) at a 1:1000 and a 1:200 dilution, respectively. For F-actin <br /> ring formation.                                                                staining, cells were &#194;&#174;xed in 1% glutaraldehyde for 10 min at 25&#194;&#176;C. Cells <br />    The basic mechanism of cytokinesis and several factors                      were washed twice in phosphate-buffered saline (PBS), resuspended in <br /> involved in septum formation and closure are conserved                         PBS/0.1% Triton X-100 and incubated for 10 min at 25&#194;&#176;C. Rhodamine- <br /> from yeast to animal cells (Field et al., 1999). The closest                   conjugated phalloidin (6 U) in PBS was added and incubated for 45 min at <br /> human Nop15p homologue is NIFK, which is nucleolar in                          25&#194;&#176;C. Cells were washed once in PBS and incubated with poly-L-lysine- <br />                                                                                coated coverslips before being mounted onto slides using moviol, <br /> interphase but associates with condensed mitotic chromo-                       containing DAPI. Cells containing pYE195-Rpl25-eGFP were grown in <br /> somes (Takagi et al., 2001). Functions of Nop15p, in                           SGSR-URA to mid-exponential phase, transferred to SD-URA for various <br /> addition to its role in pre-rRNA processing, may therefore                     times and &#194;&#174;xed in 4% (v/v) formaldehyde for 30 min and pelleted. Cells <br /> have been conserved in humans.                                                 were resuspended in 100 mM KH2Ac/K2HAc/1.1 M sorbitol and mounted <br />                                                                                onto slides using moviol, containing DAPI. For synchronization in late M- <br />                                                                                phase, cells were treated with nocodazole (10 mg/ml) for 3 h, commencing <br />                                                                                15 h after shift to glucose-containing medium. Cells were released by <br /> Materials and methods                                                          washing with pre-warmed medium and stained for F-actin as described <br />                                                                                above. Images were obtained with Smart Capture VP. <br /> Strains <br /> Growth and handling of S.cerevisiae were by standard techniques. GAL- <br />                                                                                Immunoprecipitation of Nop15p-TAP <br /> regulated strains were pre-grown in RGS medium, containing 2% <br />                                                                                For immunoprecipitation of Nop15p-TAP, cells were grown in YPgal to <br /> raf&#194;&#174;nose, 2% galactose and 2% sucrose, and harvested at intervals <br />                                                                                OD600 = 2 and lysed in buffer A (150 mM KAc, 20 mM Tris-Ac pH 7.5, <br /> following a shift to medium containing 2% glucose. Strains for pulse&#194;&#177; <br />                                                                                5 mM MgAc) with 1 mM dithiothreitol (DTT), 0.5% Triton X-100, <br /> chase analysis were pre-grown in minimal RGS medium lacking uracil <br />                                                                                2.5 mM vanadyl-ribonucleoside complexes (VRC) and 5 mM phenyl- <br /> and shifted to minimal glucose medium lacking uracil. Strains for <br />                                                                                methylsulfonyl &#194;&#175;uoride (PMSF) at 4&#194;&#176;C using glass beads (Sigma). <br /> immuno&#194;&#175;uorescence studies were grown in minimal glucose medium <br />                                                                                Immunoprecipitation with rabbit IgG&#194;&#177;agarose beads, TEV cleavage and <br /> lacking leucine. The degron-Nop15p fusion strain (generously provided <br />                                                                                RNA extraction were performed as described (Rigaut et al., 1999). <br /> by K.Labib, Paterson Institute) was grown as described (Kanemaki et al., <br /> 2003). <br />    Yeast strains used and constructed in this study are listed in table 1 of   Pulse&#194;&#177;chase labelling experiments <br /> the Supplementary data available at The EMBO Journal Online.                   Pulse&#194;&#177;chase labelling of pre-rRNA was performed as previously <br /> Conditional mutants under the control of the repressible GAL10 promoter        described (Tollervey, 1987b) using 100 mCi of [5,6-3H]uracil <br /> were generated by a one-step PCR strategy in the strains YDL401 and            (Amersham) for 2 min at 30&#194;&#176;C. Total RNA was extracted with buffer <br /> BMA64 (Lafontaine and Tollervey, 1996). Transformants were selected            AE/phenol&#194;&#177;chloroform and ethanol precipitated (Schmitt et al., 1990). <br />                                                                                3H-Labelled pre-rRNA and rRNA were resolved on 1.2% agarose gels for <br /> for HIS+ prototrophy and screened by PCR. TAP-tagged strains were <br /> constructed by a one-step PCR strategy in the GAL-mediated strain              high molecular weight and 8% acrylamide&#194;&#177;8 M urea gels for low <br /> GAL::nop15 (Rigaut et al., 1999). Transformants were screened by               molecular weight RNAs. RNA was transferred to Hybond-N+ Nylon <br /> immunoblotting and PCR. TAP-tagged strains were transformed with               membranes (Amersham), dried and exposed to X-ray &#194;&#174;lm for 10 days at <br /> pUN100DsRedNOP1 (kindly provided by E.Hurt, Heidelberg) to allow               &#194;&#177;80&#194;&#176;C with an intensifying screen. <br /> ready identi&#194;&#174;cation of the nucleolus. <br />                                                                                Mobility shift assays <br /> Viability assay                                                                A PCR fragment corresponding to the YNL110c (NOP15) ORF was <br /> GAL::nop15 and wild-type strains were pre-grown in RGS medium,                 ampli&#194;&#174;ed and cloned into BamHI and EcoRI sites of the pGEX-2T vector <br /> containing 2% raf&#194;&#174;nose, 2% galactose and 2% sucrose, before being              (Pharmacia) to obtain the pGEX-NOP15 vector. Nop15p was expressed <br /> shifted to medium containing 2% glucose. A viability assay was                 in the E.coli strain BL21, puri&#194;&#174;ed by using glutathione&#194;&#177;Sepharose <br /> performed by plating ~1, 2.5 and 5 3 103 cells of samples taken at 0,          according to the manufacturer&apos;s protocol and released by cleavage with <br /> 8, 16 and 24 h after the shift onto pre-warmed YGSR plates. Colonies           thrombin (Sigma). In vitro transcription of pre-rRNA (5&#194;&#162;-ITS1 to 3&#194;&#162;-ITS2) <br /> were counted after 2 days of growth and the percentage viability was           was performed with PCR products ampli&#194;&#174;ed from an rDNA plasmid and <br /> determined by comparison with the number of cells plated.                      carrying the T7 promoter region. The binding reaction was performed in <br />                                                                                30 mM Tris&#194;&#177;HCl pH 7.4, 150 mM KCl, 2 mM MgCl2, 0.1% Triton X- <br /> RNA extraction, northern hybridization and primer                              100, 20% glycerol and 1 mM DTT, in the presence of tRNA (1 mg/ml), <br /> extension                                                                      10 fmol of 32P-labelled pre-rRNA and 0&#194;&#177;30 fmol of recombinant protein <br /> RNA was extracted as described previously (Tollervey, 1987a). For high         in a reaction volume of 15 ml. RNA was heat denatured at 65&#194;&#176;C for 10 min, <br /> molecular weight RNA analysis, 7 mg of total RNA was separated on a            followed by slow cooling to room temperature, prior to the binding <br /> 1.2% agarose gel containing formaldehyde and transferred for northern          reaction. For competition experiments, a 200-fold molar excess of cold <br /> hybridization as described previously (Tollervey, 1987b). Standard 6, 8 or     competitor RNA was added. Reactions were incubated at room <br /> 12% acrylamide&#194;&#177;8 M urea gels were used to analyse low molecular                temperature for 30 min and then loaded on a 6% native acrylamide/ <br /> weight RNA species and primer extension reactions. Primer extensions           bisacrylamide (80:1)/4% glycerol gel in 0.53 TBE buffer. Prior to <br /> were performed as described previously (Beltrame and Tollervey, 1992)          loading, the gel was pre-run for 1 h and then run for 5 h at 250 V in the <br /> on 5 mg of total RNA using primers 006, 007 and 033.                           cold room. <br /> </body></html>