<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br /> Yeast ribosomal protein L10 helps coordinate tRNA <br /> movement through the large subunit <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span>                                                                        sequentially bind EF-Tu aa-tRNA GTP (ternary com- <br />                                                                                 plex or TC; in eukaryotes the elongation factor is called <br /> Yeast ribosomal protein L10 (E. coli L16) is located                            &#226;&#8364;&#732;eEF1A&#226;&#8364;&#8482;), stimulate GTP hydrolysis by EF-Tu, accommo- <br /> at the center of a topological nexus that connects                              date aa-tRNA into the large subunit, e&#239;&#172;&#8364;ect peptidyltrans- <br /> many functional regions of the large subunit. This                              fer, bind EF-G (eEF2 in eukaryotes), again stimulate GTP <br /> essential protein has previously been implicated in                             hydrolysis by EF-G and translocate along the mRNA by <br /> processes as diverse as ribosome biogenesis,                                    one codon. The TC and EF-G are structurally similar (1), <br /> translational fidelity and mRNA stability. Here, the                            and there is a large degree of overlap between their ribo- <br /> inability <span id='am-6' about='protege:TO' typeof='owl:Thing'>to</span> maintain the yeast Killer virus was used                           some binding sites. In yeast, for example, eEF2 contacts <br /> as a proxy for large subunit defects to identify a                              the a-sarcin-ricin loop (SRL, H95 of 25S rRNA), the <br /> series of L10 mutants. These mapped to roughly                                  GTPase-associated center (GAC composed of ribosomal <br /> four discrete regions of the protein. A detailed ana-                           protein L12, H43, H44), Helices 33, 34, 69 and 89, ribo- <br /> lysis of mutants located in the N-terminal &#226;&#8364;&#732;hook&#226;&#8364;&#8482; of                            somal proteins L9, L12 and the P proteins (P0, P1a, P1b, <br /> L10, which inserts into the bulge of 25S rRNA helix                             P2a, P2b) on the large ribosomal subunit, while it also <br />                                                                                 interacts with many elements of the small subunit includ- <br /> 89, revealed strong effects on rRNA structure corre- <br />                                                                                 ing h5, h15, h33, h34, h44 in the head and body domains, <br /> sponding to the entire path taken by the tRNA 3&#226;&#8364;&#8482; end                            and with ribosomal protein S23 (2). The interactions <br /> as it moves through the large subunit during the                                between these factors and the ribosome were initially char- <br /> elongation cycle. The mutant-induced structural                                 acterized using chemical footprinting methods (3). A cryo- <br /> changes are wide-ranging, affecting ribosome bio-                               electron microscopy study of the TC stalled on an <br /> genesis, elongation factor binding, drug resistance/                            Escherichia coli 70S ribosome with kirromycin revealed a <br /> hypersensitivity, translational fidelity and virus main-                        similarly complex set of interactions that included the <br /> tenance. The importance of L10 as a potential trans-                            SRL and the GAC on the large subunit (4). <br /> ducer of information through the ribosome, and of a                                We have been using a yeast-based system to dissect how <br /> possible role of its N-terminal domain in switching                             the ribosome coordinates TC binding, aa-tRNA accom- <br /> between the pre- and post-translocational states                                modation, peptidyltransfer and eEF2 binding. Molecular <br /> are discussed.                                                                  genetics and biochemical studies using mutant rRNAs and <br />                                                                                 ribosomal proteins are identifying important elements and <br />                                                                                 allosteric rearrangements involved in these processes (5&#226;&#8364;&#8220; <br />                                                                                 11). These studies have led us to focus on structural ele- <br />                                                                                 ments in the vicinity of the &#226;&#8364;&#732;accommodation corridor&#226;&#8364;&#8482;, a <br /> INTRODUCTION                                                                    passageway located along the interface between H89 and <br /> The ribosome is a complex macromolecular machine that                           the H90&#226;&#8364;&#8220;H92 structure of the large subunit rRNA (12). <br /> orchestrates multiple reactions in a highly coordinated                         Previously, we demonstrated the involvement of a central <br /> manner. Although atomic level ribosome structures                               extension of ribosomal protein L3 in coordinating TC, <br /> reveal the topologies of the functional centers, how these                      eEF2 binding and peptidyltransfer by helping to open <br /> exchange information and coordinate their activities                            and close this structure (9). <br /> remains an active &#239;&#172;&#65533;eld of inquiry. For example, during                             Ribosomal protein L3 is located on the H90&#226;&#8364;&#8220;H92 side <br /> the translation elongation cycle the ribosome must                              of the accommodation corridor. Examination of this <br />  <br /> *To whom correspondence should be addressed. Tel: +1 301 405 0918; Fax: +1 301 314 9489; Email: dinman@umd.edu <br /> Present addresses: <br /> Alexey N. Petrov, Department of Structural Biology, Stanford University School of Medicine. D105 Fairchild Science Building, 299 Campus Drive <br /> West, Stanford, CA 94305-5126, USA <br /> Sara C. Roshwalb, University of Tennessee College of Veterinary Medicine. 2407 River Dr Knoxville, TN 37996, USA <br />  <br /> &#195;&#376; 2008 The Author(s) <br /> This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/ <br /> by-nc/2.0/uk/) which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited. <br />  6188 Nucleic Acids Research, 2008, Vol. 36, No. 19 <br />  <br />  <br /> region reveals that another ribosomal protein, L10 (L16 in       MATERIALS AND METHODS <br /> E. coli and L10e in Haloarcula marismortui) is located on        Strains, media and genetic methods <br /> the H89 side of the corridor. L10 extends <span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span> the solvent <br /> accessible side near the factor-binding site down toward         Escherichia coli strain DH5a was used to amplify plas- <br /> the peptidyltransferase center (PTC) in the core of the          mids, and E. coli transformations were performed using <br /> large subunit. The N-terminal three-quarters of yeast            the high-e&#239;&#172;&#402;ciency transformation method (24). The hap- <br /> L10 is sandwiched between helices 89 and 38, and appears         loid S. cerevisiae rpl10 gene deletion strain AJY1437 <br /> to be largely structurally conserved with its bacterial and      (MATa met15D0 leu2D0 ura3D0 his3D0 RPL10::kanR <br /> archael homologs (13&#226;&#8364;&#8220;16). Inspection of these structures         pAJ392) was a kind gift from Dr. A.W. Johnson. The <br /> reveals that these proteins represent an extension of H39        L-A and M1 dsRNA viruses were introduced into this <br /> of the large subunit rRNA. In cryo-EM reconstitutions,           strain by cytoplasmic mixing (cytoduction) (25) using <br /> the eukaryote speci&#239;&#172;&#65533;c C-terminal region of L10 appears to        strain JD759 [MAT  kar1-1 arg1 thr(i,x) (L-A HN M1)] <br /> form a large unresolved mass adjacent to the L7/L12 stalk        as the cytoplasm donor. The resulting strain was desig- <br /> in association with Helix 38 (17). In Saccharomyces cere-        nated JD1293 [MATa rpl10::Kan met15D0 leu2D0 <br /> visiae, L10 is encoded by the essential single copy RPL10        ura3D0 his3D0 pRPL10-URA3-2  (L-A HN M1)]. Yeast <br /> gene (18). L10 plays a critical role in 60S subunit biogen-      cells were transformed using the alkali cation method (26). <br /> esis: incorporation of L10 into the large subunit constitu-      YPAD and synthetic complete medium (H-), as well as <br /> tes the last step of 60S subunit biogenesis (19). Speci&#239;&#172;&#65533;cally,   YPG, SD and 4.7 MB plates used for testing the killer <br /> immature large subunits lacking L10 are exported from            <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span>phenotype</span> were prepared and used as described previously <br /> the nucleus to the cytoplasm via the Crm1p pathway in            (27). Oligonucleotide primers were purchased from IDT <br /> complex with the export adapter protein Nmd3p, after             (Coralville, IA). Dual luciferase assays to quantitatively <br /> which L10 is brought to the ribosome in the complex              monitor translational recoding in yeast were performed <br /> with Sqt1p, promoting release of Nmd3p followed by               as previously described (28). The latter involved use of a <br /> incorporation of L10. Depletion studies demonstrate              0-frame control reporter in combination with  1 riboso- <br /> that large subunits lacking L10 are unable to form 80S           mal frameshift, or nonsense suppression test reporter con- <br /> ribosomes and are translationally inactive (20,21). Thus,        structs. Recoding e&#239;&#172;&#402;ciencies and statistical analyses were <br /> L10 is the key component required for the &#239;&#172;&#65533;nal activation        performed as previously described (28,29). Ten-fold dilu- <br /> step of ribosomes in the cytoplasm. In addition, at least        tion spot assays to monitor sensitivity to anisomycin (20 <br /> one rpl10 allele has been shown to inhibit degradation of         g/ml) or paromomycin (3 mg/ml) were performed as pre- <br /> nonsense-containing mRNAs, suggesting that L10 may               viously described (30). <br /> play a role in translational &#239;&#172;&#65533;delity (22). Thus, L10 is <br /> important both for ribosome assembly and post-transcrip-         Generation of rpl10 alleles <br /> tional regulatory processes.                                     A library of plasmid-borne rpl10 mutants was constructed <br />    Toward the goal of furthering our understanding of            using the error-prone PCR and gap repair method (31) <br /> L10, a primary library of randomly mutagenized rpl10             based on pJD589. To create pJD589, a 1-kb fragment <br /> alleles was screened for the inability to maintain the           containing the 50 promoter, open reading frame and 30 <br /> yeast &#226;&#8364;&#732;Killer&#226;&#8364;&#8482; phenotype, which is caused by an endogen-         untranslated region (UTR) sequences of RPL10 was sub- <br /> ous dsRNA virus that is highly sensitive to a broad array        cloned from the URA3-2  vector pAJ392 into the BamHI <br /> of changes in the translational apparatus (23). This for-        sites of pRS313 (32). Silent mutations were added into <br /> ward genetic screen identi&#239;&#172;&#65533;ed 56 new rpl10 alleles.              codons 4, 5, 220 and 221 of the RPL10 ORF to create <br /> The mutants generally clustered into four regions of the         two StuI restriction sites. Mutagenesis primers (70 nt) for <br /> L10 structure: (i) in a &#226;&#8364;&#732;hook&#226;&#8364;&#8482; of L10 that inserts into          PCR were designed to be complementary to the 50 and 30 <br /> the bulge at the <span id='am-163' about='xsp:base' typeof='owl:Thing'>base</span> Helix 89; (ii) in a cluster that           UTRs of RPL10 and include the RPL10 translational <br /> appears to form a bridge between Helices 38 and 39; (iii)        start and stop codons: (forward 50 TTCCGCAAGTGC <br /> a series of amino acids that appear to form a plane along        TTTTGGAGTGGGACTTTCAAACTTTAAAGTACA <br /> the face of L10 that interacts with the distal regions           GTATATCAAATAACTAATTCAAGATGGCTAGAA <br /> of Helices 89 and 38; and (iv) amino acids present in the        GG30 , reverse 50 AATTACTGTTTAATAAACTAGAAT <br /> unresolved region of the protein that may directly               TTAAATCAAAAAAATTTCTCTTTTAAGTTAGTTC <br /> interact with the GTPase-associated center. The current          AAATGTTTGAAAAGAACTTAGG30 ). Random muta- <br /> study focuses on the N-terminal &#226;&#8364;&#732;hook&#226;&#8364;&#8482; region of L10.            genesis was performed with the GeneMorph II PCR <br /> A series of genetic and biochemical studies led to a             random mutagenesis kit with template concentrations <br /> detailed analysis of the e&#239;&#172;&#8364;ects of two mutants in this           optimized to generate between one and four mutations <br /> region on 25S rRNA structure. These were found to pro-           per RPL10 coding sequence. pJD589 was digested with <br /> mote structural changes that map along the path taken by         StuI, and the linearized plasmid lacking the RPL10 <br /> tRNAs through the elongation cycle. When considered              coding sequence was puri&#239;&#172;&#65533;ed by Tris&#226;&#8364;&#8220;acetate&#226;&#8364;&#8220;EDTA&#226;&#8364;&#8220; <br /> along with the physical location of L10 within the large         agarose gel electrophoresis. This was cotransformed with <br /> subunit and its role in ribosome biogenesis, the &#239;&#172;&#65533;nding          the randomly PCR-mutagenized RPL10 coding sequences <br /> presented here are used to build a model for how L10             into JD1293 cells. Recombinants were selected for growth <br /> plays an important role for coordinating tRNA passage            on medium lacking histidine (-his), and cells having lost <br /> through the ribosome.                                            the wild-type RPL10-containing plasmids were selected <br />                                                                        Nucleic Acids Research, 2008, Vol. 36, No. 19 6189 <br />  <br />  <br /> for by replica plating onto 5-&#239;&#172;&#8218;uoroorotic acid (5-FOA)-         70S ribosome complexed with two tRNAs at 2.8 A&#203;&#353; resolu- <br /> containing medium (33). Note that recombination                 tion (PDB 2J00 and 2J01) (14), the T. thermophilus 70S <br /> between the PCR product and linearized vector was               ribosome complexed with a model mRNA and two <br /> designed so that the silent mutations would revert to           tRNAs at 3.7 A&#203;&#353; resolution (PDB 2I1C and 1VSA) (15), <br /> wild-type sequence in the recombinant clones.                   the E. coli ribosome complexed with three tRNAs at 3.5 A&#203;&#353; <br />                                                                 (2AW4) (13), the T. thermophilus 70S ribosome with a <br /> Identification of rpl10 alleles unable to support the yeast     model mRNA and tRNAs at 5.5 A&#203;&#353; (2HGU) (39) and a <br /> killer virus                                                    cryo-EM reconstruction of the D. radiodurans ribosome <br /> Approximately 3000 His+ colonies were arrayed onto -his         complexed with thiostrepton at 3.3&#226;&#8364;&#8220;3.7 A&#203;&#353; (2ZJR) (40) <br /> plates, replica plated onto 4.7 MB plates seeded with           were visualized using PyMOL (DeLano Scienti&#239;&#172;&#65533;c LLC). <br /> 5   47 killer indicator cells, and incubated at 208C to iden- <br /> tify Killer  (K&#226;&#8364;&#8220;) colonies as previously described (34). <br /> Plasmids harboring rpl10 alleles were rescued into E. coli      RESULTS <br /> from yeast strains that had lost the killer phenotype, rein-    Identification of translationally defective rpl10 alleles <br /> troduced into JD1238 cells and rescored for the inability <br />                                                                 Prior studies have implicated L10 with a broad range of <br /> to maintain the killer phenotype. The procedure was per- <br />                                                                 ribosome-associated functions, including large subunit <br /> formed three times in order to prevent identi&#239;&#172;&#65533;cation of <br />                                                                 maturation, subunit association and NMD. These facts, <br /> false positive strains due to spontaneous Killer virus loss. <br />                                                                 in combination with the centralized location of L10 within <br /> RNA blot analyses                                               the 3-D structure of the large subunit, provided us with <br />                                                                 the motive to pursue an in-depth molecular genetic and <br /> RNA (northern) blotting was performed as previously             biochemical analysis of L10 function. Toward to this end, <br /> described (35). Total RNA was extracted with acid               a library of randomly mutagenized rpl10 alleles was <br /> phenol/chlorophorm (pH = 4.5) from mid-logarithmic              screened for the inability to maintain the yeast killer phe- <br /> cell cultures as previously described (36). Equal amounts       notype. The killer system of yeast is composed of L-A <br /> of RNA (2.5  g) were separated through a 1% agarose-            helper, and M1 satellite dsRNA viruses (41). The <br /> formaldehyde gel, RNA was transferred to a Hybond-N             genome of the L-A virus encodes two open reading <br /> membrane (Amersham) and UV cross linked to the mem-             frames where the &#239;&#172;&#65533;rst encodes the Gag (capsid) protein, <br /> brane. Nonspeci&#239;&#172;&#65533;c binding sites on the membrane were            and the second encodes a viral RNA-dependent RNA <br /> blocked using ULTRAhyb-Oligo bu&#239;&#172;&#8364;er (Ambion), incu-              polymerase (RDRP), the production of which is depen- <br /> bated with a [32P] 50 -end-labeled oligonucleotide (50 C        dent on a programmed &#226;&#8364;&#8220;1 ribosomal frameshift (&#226;&#8364;&#8220;1 <br /> CGGGGTGCTTTCTGTGCTTACCGATACGACCTT                               PRF) (42,43). The single open reading frame of the M1 <br /> TACCGGCTG30 ) complementary to the 50 end of the                satellite virus encodes a protein toxin that is exported into <br /> CYH2 CDS, and subsequently washed according to man-             the surrounding media and kills uninfected cells, thus pro- <br /> ufacturer&#226;&#8364;&#8482;s instructions. Hybridizing species were identi-      ducing the easily detectable Killer+ (K+) phenotype, and <br /> &#239;&#172;&#65533;ed using a GeneStorm phosphoimager (Bio-Rad), and              its maintenance requires L-A to supply capsid proteins <br /> quanti&#239;&#172;&#65533;ed using QuantifyOne software from Bio-Rad.              and RDRP. M1 is highly sensitive to small perturbations <br />                                                                 in the translational apparatus, including changes in &#226;&#8364;&#8220;1 <br /> Ribosome biochemistry and visualization                         PRF e&#239;&#172;&#402;ciency (34), ribosome biogenesis defects (44) and <br /> Lysates of cycloheximide arrested yeast cells were sedi-        decreased overall protein synthesis (45). Thus, the killer <br /> mented through 7&#226;&#8364;&#8220;47% sucrose gradients and polysome             assay provides a simple and rapid method to screen for <br /> pro&#239;&#172;&#65533;les were determined by monitoring A254 nm as pre-           translational defects. <br /> viously described (37). S. cerevisiae ribosomes were iso-          The coding region of RPL10 was subjected to random <br /> lated, yeast aminoacyl-tRNA synthetases were puri&#239;&#172;&#65533;ed            PCR mutagenesis and screened for loss of the killer phe- <br /> and yeast phenylalanyl-tRNAs were aminoacylated with            notype as described in the Materials and methods section. <br /> [14C]Phe and puri&#239;&#172;&#65533;ed by HPLC as previously described            Mutant rpl10-HIS3 plasmids were rescued from Killer <br /> (6). Single-round peptidyltransfer assays using Ac-             minus (K&#226;&#8364;&#8220;) strains, passaged through E. coli, reintroduced <br /> [14C]Phe-tRNA and puromycin, and equilibrium binding            into rpl10D cells and rescored for their abilities to promote <br /> studies of [14C]Phe-tRNA binding to the ribosomal A-site        M1 virus loss. Due to a signi&#239;&#172;&#65533;cantly high intrinsic rate of <br /> were carried out using poly(U) primed ribosomes as pre-         killer loss in the JD1293 strain background ( 10%), the <br /> viously described (6). Equilibrium binding studies of eEF-      plasmid rescue and reintroduction procedure was repeated <br /> 2 were performed as previously described (9). The data          three times to eliminate false-positive results. The rpl10 <br /> were &#239;&#172;&#65533;tted to a one-site-binding model with ligand deple-       mutations responsible for conferring the K  phenotype <br /> tion using Prism Graph Pad software. rRNA structure             were identi&#239;&#172;&#65533;ed by DNA sequence analysis. From &gt;3000 <br /> analysis using DMS, kethoxal and CMCT were performed            colonies tested, a total of 56 unique mutants were identi- <br /> as previously described (9). The X-ray crystal structure of     &#239;&#172;&#65533;ed. In addition,  15% of the mutants were lethal as <br /> the H. marismortui 50S ribosomal subunit (1VQ6) (38),           determined by their inability to grow as the sole allele of <br /> the cryo-electron microscopy (cryo-EM) reconstruction           rpl10 (data not shown). The collection of K&#226;&#8364;&#8220; mutants con- <br /> of S. cerevisiae ribosomal proteins threaded onto the           tained alleles harboring 35 single mutations, 20 double <br /> X-ray crystal structure of the H. marismortui 50S ribo-         mutations and one triple mutant. These are summarized <br /> somal subunit (PDB IS1I) (2), the Thermus thermophilus          in Supplementary Table 1. A very large amount of overlap <br />  6190 Nucleic Acids Research, 2008, Vol. 36, No. 19 <br />  <br />  <br /> was observed between the single and double mutants. The                       groups, speci&#239;&#172;&#65533;cally: (i) in a hook-like structure located at <br /> high degree of overlap between the single and double                          the N-terminal region of L10 that inserts into the major <br /> mutants led us to conclude that the mutagenesis was per-                      groove of helix 89; (ii) in a space that appears to link <br /> formed to near saturation, and the large number of alleles                    helices 38 and 39; (iii) along a plane of the protein lying <br /> with single amino acid substitutions allowed exclusion of                     &#226;&#8364;&#732;atop&#226;&#8364;&#8482; of helix 89; and (iv) in the unresolved C-terminal <br /> alleles with multiple mutations from subsequent analysis.                     domain of L10 that appears to extend outward toward the <br /> Among the single mutants, multiple substitutions were                         GTPase-associated center. Given this apparent structural <br /> found at amino acid residues 7, 9, 40, 93, 94 and 145.                        complexity of L10, we chose to focus on characterizing the <br /> Areas of L10 with high frequencies of mutations were                          N-terminal &#226;&#8364;&#732;hook domain&#226;&#8364;&#8482; mutants for this study. <br /> clustered in three regions of the protein. These regions <br /> spanned amino acids 7&#226;&#8364;&#8220;17, 59&#226;&#8364;&#8220;94 and 144&#226;&#8364;&#8220;152.                                  The N-terminal &#226;&#8364;&#732;hook&#226;&#8364;&#8482; mutants affect cell growth <br /> Consistent with &#239;&#172;&#65533;ndings from the Johnson lab, no viable <br />                                                                               Ribosome-associated defects commonly promote de&#239;&#172;&#65533;cien- <br /> mutants mapping to the unstructured central loop (amino <br />                                                                               cies in cellular growth at ambient temperature. As a semi- <br /> acids 102&#226;&#8364;&#8220;112) were identi&#239;&#172;&#65533;ed in the current study, con-                      quantitative monitor of this, 10-fold dilution spot assays <br /> sistent with the suggestion that his loop may play a critical                 were performed on rich medium at 308C, revealing allele- <br /> function in peptidyltransfer (46). As diagramed in                            speci&#239;&#172;&#65533;c growth defects by the R7Q and Y11C mutants <br /> Figure 1, the mutants appear to cluster into four distinct                    (Figure 2A). Note that the large colonies in the Y11C <br />                                                                               sample appear to be escape mutants. Changes in ribosome <br />         5S                                                                    structure and function also alter the growth characteristics <br />    L4 <br />                    L10                                                        of cells at decreased and/or elevated temperatures. Thus, <br />                                                                               these mutants were assayed with regard to their resistance <br />                                   H38                  4 C-term <br />              PTC                                                              or sensitivity to low (158C) and high (378C) temperatures. <br />                      L3                                    P160T              As shown in Figure 2B, mutations of the tyrosines in the <br />                                                            P179S <br />   L2                                                                          N-terminal region of the protein bridging helices 89 and <br />                                                            F199I <br />                                                            S201F <br />                                                            K202 Q <br />                                                                               39, i.e. Y9H, Y9N and Y11C promoted enhanced growth <br />                                                                               at low temperature. In contrast, none of these mutants <br />                           2 <br />                                                                               a&#239;&#172;&#8364;ected growth at 378C (Figure 2C). Ten-fold serial dilu- <br />                                                                               tion assays were also used to probe the e&#239;&#172;&#8364;ects of small <br />         H39                                                  3                molecule protein synthesis inhibitors on the translational <br />                                                                               apparatus. Anisomycin is a competitive inhibitor of A-site <br />                                                                               binding and sterically hinders positioning of the acceptor <br />                                                                               end of the A-site tRNA in the PTC (47). All of the other <br />                               1                                               mutants were either hypersensitive as determined by <br />                                                                               reduced growth at 15  g/ml anisomycin (R7Q, Y9H, <br />                                         H89                                   Y9N and Y11C), or resistant as evidenced by the ability <br />                                                                               to grow on medium containing 20  g/ml anisomycin <br />                                                                               (R7L, R7P and Y9C) (Figure 2D and E, and summarized <br /> Figure 1. Localization of the L10 mutations identi&#239;&#172;&#65533;ed in this study.          in Table 1). These data indicate a high degree of allele- <br /> Insert shows the &#226;&#8364;&#732;crown view&#226;&#8364;&#8482;, and the general location of L10 relative <br /> to other salient features of the large subunit. Large picture shows the       speci&#239;&#172;&#65533;city with regard to this drug. Another small mole- <br /> L10 mutants mapped onto the structure of yeast L10 taken from (2).            cule drug, paromomycin, binds at the decoding center on <br /> The four spatially de&#239;&#172;&#65533;ned groups of L10 mutants are circled and               the small subunit and promotes conformational changes <br /> labeled, where group 1 represents those located at the N-terminus of          associated with formation of the codon&#226;&#8364;&#8220;anticodon helix <br /> the protein. Group 4, indicated by the pink circle, map to the structu- <br /> rally unde&#239;&#172;&#65533;ned C-terminal region of yeast L10. Neighboring 25S rRNA           between mRNA and incoming A-site tRNA (48). Thus, <br /> helical structures are indicated, and GAC indicates the GTPase-               paromomycin was used as a probe for defects associated <br /> associated center of the large subunit.                                       with interactions between the small and large subunits. <br />  <br />     A              30&#194;&#176;C                 B       15&#194;&#176;C                C      37&#194;&#176;C              D      Aniso (15&#194;&#181;g/ml)         E    Aniso (20&#194;&#181;g/ml) <br />      WT <br />     R7L <br />     R7P <br />     R7Q <br />     Y9C <br />     Y9H <br />     Y9N <br />    Y11C <br />  <br /> Figure 2. Genetic phenotypes of the L10 N-terminal hook mutants. Ten-fold dilutions of cells harboring the indicated rpl10 alleles were spotted onto <br /> complete synthetic medium lacking histidine (-his), and were incubated at the indicated temperatures (three left panels). In the two right panels, cells <br /> were spotted onto -his medium containing the indicated concentrations of anisomycin. <br />                                                                                    Nucleic Acids Research, 2008, Vol. 36, No. 19 6191 <br />  <br />  <br /> None of the mutants promoted paromomycin-speci&#239;&#172;&#65533;c                            the three rpl10 alleles that most a&#239;&#172;&#8364;ected nonsense- <br /> changes in growth (data not shown), thus indicating that                    suppression, and from control isogenic wild-type and <br /> the observed e&#239;&#172;&#8364;ects on cell growth were conferred by                        upf1D cells. While the CYH2 pre-mRNA was stabilized <br /> changes in the large subunit.                                               in the upf1D strain, none of the mutants appeared to sta- <br />                                                                             bilize this species relative to isogenic wild-type controls <br /> Effects on translational fidelity                                           (Figure 3B), con&#239;&#172;&#65533;rming their e&#239;&#172;&#8364;ects on nonsense- <br /> Alterations in ribosome structure can a&#239;&#172;&#8364;ect the &#239;&#172;&#65533;delity of                  suppression. <br /> protein translation. Changes in &#226;&#8364;&#8220;1 PRF have been shown <br /> to interfere with virus propagation by altering the ratio <br /> viral Gag to Gag-pol proteins (34), and thus altered &#226;&#8364;&#8220;1                     All of the L10 N-terminal hook mutants affect 60S <br /> PRF frequencies could be one possible reason underlying                     subunit biogenesis <br /> the K  phenotypes of the rpl10 mutants. An in vivo dual-                    L10 is the last protein incorporated into the large subunit <br /> luciferase reporter assay and statistical analysis package                  and is required for formation of active ribosomes (50). <br /> (28,29) was used to quantitatively assess &#226;&#8364;&#8220;1 PRF in the                     During translation initiation, the 43S complex recruits <br /> wild-type RPL10, and isogenic rpl10 strains. In wild-type                   the mRNA to form the 48S complex, after which the <br /> cells, &#226;&#8364;&#8220;1PRF was 8.3   0.16%. Surprisingly, &#226;&#8364;&#8220;1PRF e&#239;&#172;&#402;- <br />                                                                             60S subunit is recruited. Insu&#239;&#172;&#402;cient levels of 60S subunits, <br /> ciency was not strongly a&#239;&#172;&#8364;ected in any of the rpl10 mutant <br />                                                                             or 60S subunits de&#239;&#172;&#65533;cient in their ability to form 80S ribo- <br /> strains (Figure 3A, white bars) suggesting that &#226;&#8364;&#8220;1 PRF <br />                                                                             somes, tend to promote accumulation of 48S&#226;&#8364;&#8220;mRNA <br /> defects were not the major cause of the K  phenotypes. <br />                                                                             complexes, and this additional mass results in intermedi- <br /> The dual luciferase assay was also used to quantitatively <br /> access the e&#239;&#172;&#8364;ects of the rpl10 alleles on the ability of ribo-              ate sedimentation coe&#239;&#172;&#402;cients as monitored by the appear- <br /> somes to recognize termination codons (nonsense suppres-                    ance of &#226;&#8364;&#732;halfmers shoulders&#226;&#8364;&#8482; on the denser sides of <br /> sion). This analysis revealed strong allele-speci&#239;&#172;&#65533;c e&#239;&#172;&#8364;ects                  polysome tracings. L10 mutants a&#239;&#172;&#8364;ecting inclusion of <br /> (Figure 3A, gray bars). While nonsense suppression was                      the protein into large subunits have been shown to pro- <br /> 0.27%   0.01% in wild-type cells, it was stimulated by                      mote the appearance of these halfmer shoulders in poly- <br />  1.5-fold by the R7L mutant but not by the other two                        some pro&#239;&#172;&#65533;les (21), and 60S ribosomal subunit biogenesis <br /> mutants as this position. In contrast all of the Y9 mutants                 defects have been linked with defects in killer virus main- <br /> appeared to promote hyperaccurate termination codon                         tenance (44). Sucrose gradient analyses revealed that all <br /> recognition, decreasing rates of nonsense-suppression by                    but the Y9H and Y11C mutants promoted formation of <br /> 1.8-fold (Y9C) and 2.5-fold (Y9N). The translational &#239;&#172;&#65533;del-                  halfmers (Figure 3C), many with greater amplitude than <br /> ity data are summarized in Table 1.                                         previously observed in the temperature sensitive mutants <br />    It had been previously shown that the grc5-1 allele of                   (21). Additionally, observation of decreased amplitude of <br /> RPL10 promoted defective turnover of nonsense-contain-                      the 60S relative to 40S peaks, of 80S relative to 40S and <br /> ing mRNAs (22). Although the bicistronic reporters                          60S peaks, or an increase in the amount of material near <br /> internally control for apparent changes in translational                    the top of the gradient are also associated with 60S sub- <br /> &#239;&#172;&#65533;delity due to e&#239;&#172;&#8364;ects on mRNA stability (28,49), to                         <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> biogenesis defects. These e&#239;&#172;&#8364;ects were observed in the <br /> further exclude this possibility and to determine whether                   R7L, R7Q, Y9C, Y9H, Y9N and Y11C mutants (sum- <br /> any of the new rpl10 alleles a&#239;&#172;&#8364;ected nonsense-mediated                      marized in Table 1). These results point to 60S subunit <br /> mRNA decay, levels of the ine&#239;&#172;&#402;ciently spliced CYH2                          ribosome biogenesis defects as the most likely cause <br /> precursor-mRNA were assayed by northern blot analy-                         of the inability of the rpl10 mutants to maintain the <br /> sis using total RNA extracted from cells expressing                         killer virus. <br />  <br />  <br /> Table 1. Summary of the L10 N-terminal &#226;&#8364;&#732;hook&#226;&#8364;&#8482; mutants examined in this study <br />  <br /> Strain       Percentage of             Nons. Supp.             Anisoc             aa-tRNA Kd,           eEF2 Kd, nM           Ribo biogenesis <br />               1PRF (fold-WT)a          (fold-WT)b                                 nM (fold-WT)d         (fold-WT)e            defectsf <br />  <br /> WT           <br /> R7L        <br /> R7P           <br /> R7Q           <br /> Y9C          <br /> Y9H       <br /> Y9N            <br /> Y11C          <br />  <br />   Denotes standard errors throughout. <br /> a <br />   E&#239;&#172;&#402;ciency of  1 ribosomal frameshifting. Numbers in parentheses denote fold wild-type levels throughout. <br /> b <br />   Percent rate of suppression of an in-frame UAA codon. <br /> c <br />   Resistance or hypersensitivity to anisomycin relative to wild-type. <br /> d <br />   Dissociation constants of ribosomes for the aa-tRNA eEF1A GTP Ternary Complex (nM). <br /> e <br />   Dissociation constants of ribosomes for eEF2 (nM). <br /> f <br />  Indicates the e&#239;&#172;&#8364;ects of the indicated L10 mutants on ribosome biogenesis/subunit joining. <br />  6192 Nucleic Acids Research, 2008, Vol. 36, No. 19 <br />  <br />  <br />                          A                     1.8 <br />                                                                              -1PRF     Nons Supp <br />                                                1.6 <br />                                                                                                              B <br />                                                1.4                                                                  f1 T 7L 9C 9H <br />                                                                                                                   up W R Y Y <br />  <br />                           Recoding (Fold-WT) <br />   <br />                                                 0 <br />                                                      WT   R7L    R7P    R7Q      Y9C    Y9H     Y9N   Y11C <br />  <br />  <br />                           C WT              R7L                                                 R7P                 R7Q <br />                               40S 80S                                           halfmers               halfmers             halfmers <br />                                60S polysomes <br />  <br />  <br />  <br />  <br />                                                Y9C                     Y9H                 Y9N                     Y11C <br />                                                       halfmers                                          halfmers <br />  <br />  <br />  <br />  <br /> Figure 3. The L10 N-terminal hook mutants promote allele-speci&#239;&#172;&#65533;c e&#239;&#172;&#8364;ects on nonsense codon recognition and ribosome biogenesis/subunit joining. <br /> (A) Allele-speci&#239;&#172;&#65533;c e&#239;&#172;&#8364;ects on nonsense codon recognition but not programmed &#226;&#8364;&#8220;1 ribosomal frameshifting (&#226;&#8364;&#8220;1 PRF). Isogenic yeast cells expressing <br /> either wild-type or mutant forms of L10 were transformed with dual luciferase reporter and control plasmids (28) and rates of translational recoding <br /> were determined. &#226;&#8364;&#8220;1 PRF indicates programmed &#226;&#8364;&#8220;1 ribosomal frameshifting promoted by the yeast L-A virus frameshift signal. Nons-supp. denotes <br /> the ability of ribosomes able to suppress an in-frame UAA termination codon located between the Renilla and &#239;&#172;&#65533;re&#239;&#172;&#8218;y luciferase reporter genes. <br /> Relative di&#239;&#172;&#8364;erences in recoding e&#239;&#172;&#402;ciencies of the mutants is depicted as fold of wild-type. Error bars denote standard error as previously described <br /> (29). (B) The mutants do not a&#239;&#172;&#8364;ect nonsense-mediated mRNA decay. A northern blot of total RNA extracted from isogenic wild-type and indicated <br /> L10 mutants was probed with a [32P]-labeled synthetic oligonucleotide complementary to the yeast CYH2 coding sequence as previously described <br /> (62). Locations of the unspliced CYH2 pre-mRNA and of the mature CYH2 mRNA are indicated. (C) Most of the L10 mutants promote strong 60S <br /> biogenesis and subunit joining defects. Cytoplasmic extracts from isogenic strains were loaded onto 7&#226;&#8364;&#8220;47% sucrose gradients, centrifuged in an SW41 <br /> rotor at 40 000 r.p.m. for 180 min at 48C, fractionated and analyzed by continuous monitoring of A254 (63). The locations of 40S, 60S, 80S, polysome <br /> fractions and halfmers are labeled. The presence of halfmers is typically due to subunit joining defects or to lower abundance of 60S subunits as a <br /> consequence of 60S biogenesis defects. <br />  <br />  <br /> Ribosome biochemistry                                                                         e&#239;&#172;&#8364;ects on these two parameters. The Y9C mutant pro- <br />                                                                                               moted an  8-fold increase in Kd for aa-tRNA, and an <br /> Previous studies from our laboratory using mutants of <br />                                                                                                50% decrease in Kd for eEF2. Conversely, the Kds of <br /> ribosomal protein L3 demonstrated an inverse relation- <br />                                                                                               Y11C ribosomes for aa-tRNA were 0.86- and 1.54-fold of <br /> ship between the a&#239;&#172;&#402;nity of ribosomes for aa-tRNA and <br />                                                                                               wild-type. No signi&#239;&#172;&#65533;cant changes in peptidyltransferase <br /> eEF2 (9). Equilibrium dissociation constants were deter- <br />                                                                                               activities were observed between wild-type and mutant <br /> mined for aa-tRNA using of [14C]Phe-tRNA and non-salt- <br />                                                                                               ribosomes (data not shown). <br /> washed ribosomes, and for eEF2 by monitoring the ability <br /> of diphtheria toxin to <span id='am-5' about='rdfs:label' typeof='owl:Thing'>label</span> ribosome bound eEF2 with <br /> [14C]NAD+. The inverse relationship between aa-tRNA                                           Chemical protection studies: the Y9C and Y11C forms of <br /> and eEF2 a&#239;&#172;&#402;nities was also observed with the L10 mutants                                      L10 produce profound local and long-distance changes in <br /> (Figure 4, Table 1 and Supplementary Figure 1). In all                                        25S rRNA structure <br /> cases, increased a&#239;&#172;&#402;nity for aa-tRNA was matched by                                            In light of their strong and opposing e&#239;&#172;&#8364;ects on aa-tRNA <br /> decreased a&#239;&#172;&#402;nity for eEF2 and vice versa. In particular,                                      and eEF2 binding, the e&#239;&#172;&#8364;ects of the Y9C and Y11C <br /> the Y9C and Y11C mutants showed dramatically opposing                                         mutants on 25S rRNA structure were probed using three <br />                                                                                                                Nucleic Acids Research, 2008, Vol. 36, No. 19 6193 <br />  <br />  <br />                             800                                            20                           between L10 and these rRNA helical structures. <br />                             700                                            18                           Interestingly, the changes in H38 and H89 are similar to <br />                                                                                                         those promoted by rRNA mutants located in Helix 38 (7). <br />  aa-tRNA binding (Kd, nM) <br />  <br />  <br />  <br />  <br />                             600                                            16 <br />  <br />  <br />  <br />  <br />                                                                                 eEF2 binding (Kd, nM) <br />                                                                                                         An additional series of long-distance changes are observed <br />                             500                                            14 <br />                               ~ <br />                             300                         ~                  12 <br />                                                                                                         extending along a line from the base of the peptidyltrans- <br />                                                                                                         ferase center (G2813&#226;&#8364;&#8220;G2815) along Helix 74 and out to <br />                             250                                            10 <br />                                                                                                         Helix 88: as discussed below, this follows the path taken <br />                                                                                                         by the 30 end of the deacylated tRNA as it moves from the <br />                             200                                            8                            P-site to the E-site and exits the ribosome. The a&#239;&#172;&#8364;ected <br />                             150                                            6                            bases in this cluster tend to be hyperprotected as com- <br />                             100                                            4                            pared to wild-type ribosomes, suggesting that the Y11C <br />                             50 <br />                                                                                                         promoted disruption of the local interactions between L10 <br />                                                                            2 <br />                                                                                                         and Helices 38 and 89 has caused them to collapse into <br />                              0                                             0                            more distantly located structures. This <span id='am-164' about='xsp:pattern' typeof='owl:Thing'>pattern</span> is also simi- <br />                                                    7Q <br />   <br />                                                                                                         lar to one observed by another rRNA mutant, 	2922C, <br />                              located at the base of Helix 92 in the A-loop (11). <br /> Figure 4. An inverse relationship between ribosome binding with the                                     Additionally, one base at the tip of Helix 91 (A2901, <br /> aa-tRNA and eEF2. Dissociation constants were generated by analysis                                     E. coli U2583), which abuts the SRL, became hyperpro- <br /> of single-site-binding isotherms of eEF-1A stimulated binding <br /> of [14C]Phe-tRNA to ribosomal A-sites in the presence of GTP <br />                                                                                                         tected by this mutant. Attention is directed to four speci&#239;&#172;&#65533;c <br /> (aa-tRNA, gray bars), or with eEF2 (black bars) as determined by                                        bases. First, U2860 (E. coli U2483) helps to form one <br /> monitoring the fraction of [14C]-ADP-ribosylation of ribosome asso-                                     side of the &#239;&#172;&#65533;rst &#226;&#8364;&#732;gate&#226;&#8364;&#8482; through which aa-tRNA passes <br /> ciated eEF2 by diphtheria toxin. Error bars denote standard deviation.                                  during the process of accommodation (12). Second, <br /> The equilibrium binding curves from which these data were generated <br /> are shown in Supplementary Figure 1. <br />                                                                                                         A2818 (E. coli A2450) is located in the core of the peptidyl- <br />                                                                                                         transferase center. Third, G2815 (E. coli G2447) was hyper- <br />                                                                                                         protected by this mutant and is discussed in the context of <br /> base-speci&#239;&#172;&#65533;c solvent-accessible reagents: dimethylsulfate                                               anisomycin hypersensitivity. Fourth, G2777 (E. coli <br /> (DMS), kethoxal and carbodiimide metho-p-toluenesulfo-                                                  A2406) which is normally base-paired with C311 (E. coli <br /> nate (CMCT). rRNAs were extracted, and modi&#239;&#172;&#65533;ed bases                                                    U416) in helix 22 became deprotected, and C2775 <br /> were identi&#239;&#172;&#65533;ed by primer extension using reverse tran-                                                  (E. coli U2404) displayed an enhanced in-line cleavage pat- <br /> scriptase to detect methylation at the N3 position of uri-                                              tern, suggesting an opening of a proposed E-site gate <br /> dines and the N1 position of guanosines (CMCT), at the                                                  responsible for movement of tRNA from the P-site to the <br /> N1 and N2 positions of guanosines (kethoxal), and at the                                                E-site and release of deacylated tRNA from the ribosome. <br /> N1 position of adenosines and N3 position of cytidines                                                     Examination of the e&#239;&#172;&#8364;ects of the Y9C mutant on 25S <br /> (DMS) (51). The primers used were designed to probe                                                     rRNA chemical protection patterns reveals a distinctly <br /> functional regions of domain V (including the A-site and                                                di&#239;&#172;&#8364;erent picture marked by more local changes along <br /> P-site loops, the PTC, and the helices adjacent to these                                                the aa-tRNA accommodation corridor (Figure. 5A, <br /> structures, i.e. Helices 73, 74, 88&#226;&#8364;&#8220;93), Helices 94&#226;&#8364;&#8220;96 and                                              C and E). Proximal to the mutant amino acid, U2828 <br /> Helix 38 (9,11).                                                                                        (E. coli U2460) was hyperprotected, suggesting that an <br />    Autoradiograms generated from these experiments are                                                  interaction between it and L10 were signi&#239;&#172;&#65533;cantly altered. <br /> shown in Figure 5A. Figure 5B and C show 2-dimensional                                                  Deprotection of bases in Helix 38 (A1947, A1048) and <br /> maps of the relevant regions of yeast 25S rRNA onto                                                     Helix 89 (A2844, A2850) that appear to interact with <br /> which nucleotides having altered chemical protection pat-                                               other parts of L10 suggests that the local change may <br /> terns in the Y11C and Y9C mutants have been mapped.                                                     have radiated outward from the mutated residue disrupt- <br /> This information was in turn used to map the modi&#239;&#172;&#65533;ed                                                    ing the interaction of L10 with these helices. <br /> bases onto the structure of the yeast ribosome (2) in                                                   Hyperprotection of bases along the aa-tRNA accommo- <br /> Figure 5D and E. The chemical protection patterns are                                                   dation corridor, in particular U2861 (E. coli U2492) and <br /> also mapped onto &#239;&#172;&#65533;ve additional high-resolution ribo-                                                   U2923 (E. coli U2555), and extending outward to regions <br /> some structures (Supplementary Figure 2). Although no                                                   of Helix 95, suggests collapse of this structure into the <br /> signi&#239;&#172;&#65533;cant di&#239;&#172;&#8364;erences were observed in the locations of                                                 corridor. Finally, the strong deprotection of A2818 and <br /> bases with chemical modi&#239;&#172;&#65533;cation changes, as discussed                                                   A2819 (E. coli A2450 and A2451) indicates structural <br /> below, this analysis did reveal di&#239;&#172;&#8364;erences in the locations                                             changes in the peptidyltransferase center. The possible sig- <br /> of the N-terminal hook domain. Examination of the data                                                  ni&#239;&#172;&#65533;cance of the chemical protection patterns observed in <br /> reveals that both of the mutants confer signi&#239;&#172;&#65533;cant changes                                              the Y9C and Y11C mutant ribosomes is discussed below. <br /> in 25S rRNA structure. The Y11C mutant is marked by <br /> signi&#239;&#172;&#65533;cant changes radiating outward and parallel to L10 <br /> from the site of the <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span>mutation</span> along Helices 89 and 38 (and                                              DISCUSSION <br /> expansion segment 12, which is not present in the atomic                                                This study describes the generation and characterization <br /> resolution structure) (Figure 5A, B and D). The a&#239;&#172;&#8364;ected                                                 of a new set of rpl10 yeast alleles based on their ability to <br /> bases in these structures are mostly deprotected, suggest-                                              promote loss of the Killer phenotype. The mutations pre- <br /> ing that the Y11C mutant destabilizes interactions                                                      dominantly clustered in several regions. Eight mutants <br />  6194 Nucleic Acids Research, 2008, Vol. 36, No. 19 <br />  <br />  <br /> Figure 5. Structure probing of wild-type and mutant ribosomes. (A) Autoradiograms of reverse transcriptase primer extension reactions spanning <br /> sequences in helix 38 and expansion segment 12, helices 88&#226;&#8364;&#8220;92 and helices 94&#226;&#8364;&#8220;96. Sequencing reactions (left sides of panels) are labeled corresponding <br /> to the rRNA sense strand. Mutants are indicated at top, and bases with altered chemical protection patterns are indicated to the left. Atop each <br /> panel lane, &#195;&#184; indicates untreated ribosomes, D is DMS, C denotes CMCT and K stands for kethoxal. Open arrowheads indicate bases that are <br /> hyperprotected from chemical attack relative to wild-type, and &#239;&#172;&#65533;lled arrowheads denote bases that are more susceptible to chemical attack. The black <br /> arrow at C2775 denotes increased in-line cleavage at this site in the Y11C mutant. Yeast 25S rRNA base numbering is used throughout. (B and C) <br /> Localization of bases in yeast 25S rRNA whose modi&#239;&#172;&#65533;cation patterns were a&#239;&#172;&#8364;ected by the Y11C (B) and Y9C (C) mutants. Protected and <br /> deprotected bases are indicated by open and &#239;&#172;&#65533;lled arrowheads, respectively, and the arrow at C2775 shows enhanced in-line cleavage. (D and E) <br /> rRNA protection data mapped onto the yeast large subunit crystal structure threaded into that of H. marismortui (2). Base numbering follows the <br /> S. cerevisiae sequence shown in (B) and (C). Hyperprotected bases are colored gray, and deprotected bases are shown in black. In panel E, A2818 <br /> (E. coli A2451) is colored dark ochre. The path taken by the 30 end of aa-tRNA as it accommodates into the large subunit is denoted by the orange <br /> arrows, while the path it takes out of the peptidyltransferase center is traced by the green arrow in (D). <br />                                                                         Nucleic Acids Research, 2008, Vol. 36, No. 19 6195 <br />  <br />  <br /> were identi&#239;&#172;&#65533;ed in the N-terminal &#226;&#8364;&#732;hook&#226;&#8364;&#8482; region composed          in particular with regard to coordination of elongation <br /> of residues R7, Y9 and Y11. All these residues either con-       factor interactions. Speci&#239;&#172;&#65533;cally, in the Y9C mutant, hyper- <br /> tact the PTC-proximal bulge in the H89 or with nearby            protection of the Gate 1 bases (U2860 and U2923, E. coli <br /> residues. The second group, A64, K74, N144, K145 and             U2492 and U2555), accompanied by increased a&#239;&#172;&#402;nity for <br /> L152, appear to interact with H89. Group three appears to        eEF2 and decreased a&#239;&#172;&#402;nity for aa-tRNA is consistent <br /> directly contact or be in close proximity with the bulge in      with a &#226;&#8364;&#732;closed&#226;&#8364;&#8482; accommodation corridor/eEF2-binding <br /> H38 and includes eight mutants at six positions (K15,            site. Conversely in the Y11C mutant, deprotection of <br /> Y17, V26, Q59, P93 and F94). In particular, K15 and              U2861 (E. coli U2493) coupled with increased a&#239;&#172;&#402;nity <br /> Y17 link Helices 38 and 39, each appearing to make con-          for aa-tRNA and decreased a&#239;&#172;&#402;nity for eEF2 is consistent <br /> tact with both structures. The fourth group is located in        with the &#226;&#8364;&#732;open&#226;&#8364;&#8482; corridor/TC-binding conformation. In <br /> the C-terminal region of L10, the structure of which is          addition, deprotection of G2777 (E. coli A2406) and <br /> currently unresolved. Together with tip of the H38, this         enhanced in-line cleavage of C2775 (E. coli U2404) sug- <br /> region of the protein forms an extension that approaches         gests that the &#226;&#8364;&#732;E-site gate&#226;&#8364;&#8482;, through which deacylated <br /> the L7/L12 stalk. We attribute K40 and G81positions to           tRNA passes on its way from the P-site to the E-site <br /> this group because they topologically represent the base of      and out of the ribosome (39,54&#226;&#8364;&#8220;57), is also open in this <br /> this extension. Consistent with a previous report (46), no       mutant. The e&#239;&#172;&#8364;ects of these mutants on ribosome struc- <br /> mutations were found in the unstructured loop of L10 that        ture along the entire path traversed by tRNAs during the <br /> is thought to most closely approach the PTC, thus indi-          elongation cycle leads us to propose that L10 plays an <br /> rectly suggesting that this loop is either not important for     important role in coordinating tRNA movement through <br /> ribosome biogenesis, peptidyltransfer and/or translational       the large subunit. <br /> &#239;&#172;&#65533;delity (unlikely), or that it is so important that mutants in <br /> this region are inviable (more likely). A full analysis of all   Anisomycin resistance/hypersensitivity and <br /> of the mutants identi&#239;&#172;&#65533;ed in the current study is beyond the      ribosome structure <br /> scope of a single manuscript. Thus, we chose to focus <br /> more deeply on those located in the N-terminal &#226;&#8364;&#732;hook&#226;&#8364;&#8482;            The L10 mutants examined in the current study displayed <br /> region of L10 in the current study in order to establish a       a range of phenotypes related to anisomycin, ranging from <br /> connection with ribosomal protein L3 (see accompanying           strong drug resistance (R7L, R7P and Y9C) to hypersen- <br /> manuscript by Meskauskas,A. and Dinman,J.D.) with an             sitive (Y9H, Y9N and Y11C). Anisomycin resistance by <br /> emphasis on how ribosome structure informs function.             ribosomal protein L3 mutants was previously proposed to <br />                                                                  be due to opening of the accommodation corridor, <br />                                                                  increasing di&#239;&#172;&#8364;usion rates of the relatively large aa-tRNA <br /> Ribosome structure and elongation factor interactions: <br />                                                                  into the PTC relative to the much smaller drug (6,9). This <br /> an allosteric model <br />                                                                  model cannot explain the observations made with the L10 <br /> The TC and eEF2 are structural mimics of one another             mutants in the current study, where Y9C (corridor closed) <br /> and bind to essentially the same site of the ribosome (1).       is drug resistant while Y11C (corridor open) is hypersen- <br /> How then does the ribosome coordinate its interactions           sitive. Two more-recent studies point to changes in the <br /> with these two ligands at the correct phases of the elonga-      drug-binding site being responsible for anisomycin resis- <br /> tion cycle? The Dontsova group used mutants of Helix 42          tance (11,58). Mutation of E. coli C2452 to U (yeast <br /> in E. coli to demonstrate that activation of EF-G may            C2820U, H. marismortui 2487U) confers very strong ani- <br /> involve allosteric signaling from the peptidyltransferase        somycin resistance. Chemical protection studies in the <br /> center to the GAC, and that the di&#239;&#172;&#8364;erence between the            yeast mutant showed that anisomycin was unable to pro- <br /> EF-G and TC-binding sites may be due to di&#239;&#172;&#8364;erences in            tect this base from chemical attack, suggesting that aniso- <br /> the positioning of the GAC relative to the SRL (52,53).          mycin cannot interact with a U at this position. The X-ray <br /> We subsequently proposed a complementary model sug-              crystal structure of the analogous H. marismortui mutant <br /> gesting that ribosomal protein L3 functions as the &#226;&#8364;&#732;gate-        con&#239;&#172;&#65533;rmed this model. In the current study, we did not <br /> keeper to the A-site&#226;&#8364;&#8482; within this scheme (9). This model         detect any changes in the sensitivity of this base to chemi- <br /> has been expanded and re&#239;&#172;&#65533;ned in the accompanying                 cal modifying agents. However the increased sensitivity of <br /> manuscript. These studies propose that L3 plays a role           the two 50 adjacent bases (A2818 and A2819, E. coli A2450 <br /> in coordinating the orderly binding of the elongation fac-       and A2451) could account for the resistance of the Y9C <br /> tors and peptidyltransferase activity by participating in a      mutant to this drug. This does not address the question of <br /> series of local allosteric changes in rRNA structure.            how Y11C, in which A2818 (E. coli A2450) is also depro- <br /> Speci&#239;&#172;&#65533;cally we proposed that the &#226;&#8364;&#732;open&#226;&#8364;&#8482; aa-tRNA accom-           tected, could be anisomycin hypersensitive. A potential <br /> modation corridor conformation de&#239;&#172;&#65533;nes the TC-binding             answer might be gleaned from the anisomycin-resistant <br /> site, while the &#226;&#8364;&#732;closed&#226;&#8364;&#8482; conformation favors eEF2 binding.       G2447C and G2447U H. marismortui mutants (58). This <br /> Examination of the atomic scale ribosome structures              base contacts anisomycin through a K+, and the two <br /> reveals that L10 lies on the opposite side of the accommo-       mutants cause drug resistance by altering the position of <br /> dation corridor and PTC from L3 (e.g. see Figure 5D),            this ion. In yeast, the corresponding base is G2815; this <br /> and data in the accompanying manuscript indicate that            base and its two 50 neighbors are deprotected in the Y11C <br /> bases in H89 that interact with the N-terminal hook of           mutant (Figure 5). We suggest that this mutant may result <br /> L10 are involved in this process. The data from the current      in a change in this region of the anisomycin-binding <br /> study closely complement those from our studies of L3,           pocket that confers increased a&#239;&#172;&#402;nity for anisomycin, <br />  6196 Nucleic Acids Research, 2008, Vol. 36, No. 19 <br />  <br />  <br /> resulting in drug hypersensitivity. This idea is supported     impact on two discrete physical pathways for information <br /> by the increased a&#239;&#172;&#402;nity of Y11C for aa-tRNA, the 30 end        &#239;&#172;&#8218;ow through the ribosome. <br /> of which partially overlaps with the anisomycin-binding           Lastly, is L10 involved in the process of ribosome ratch- <br /> pocket.                                                        eting between the pre- and post-translocational states <br />                                                                (61)? Comparison of the L10 (bacterial L16, archae <br /> Ribosome structure, translational fidelity and virus           L10e) N-terminal regions between seven di&#239;&#172;&#8364;erent high- <br /> maintenance                                                    resolution ribosome structures suggest that it can assume <br /> Some of the L10 &#226;&#8364;&#732;hook&#226;&#8364;&#8482; mutants had strong e&#239;&#172;&#8364;ects on ter-       two di&#239;&#172;&#8364;erent conformations: interacting with Helix 89 or <br /> mination codon recognition (Figure 3A). We speculate that      with Helix 38 (see Supplementary Figure 2). One possible <br /> this may be explained by open/closed nature of accommo-        interpretation of these comparisons is that the hook inter- <br /> dation corridor. The closed conformation, e.g. Y9C, could      acts with Helix 89 when the A-site is unoccupied (the post- <br /> decrease aa-tRNA accommodation rates, providing more           translocation state), while it interacts with Helix 38 when <br /> time for ribosomes to reject suppressor tRNAs during the       the A-site is occupied (pre-translocation state). It is possi- <br /> proofreading phase of translation elongation (59).             ble that the N-terminal &#226;&#8364;&#732;hook&#226;&#8364;&#8482; of L10 may participate in <br /> Conversely, the &#226;&#8364;&#732;open&#226;&#8364;&#8482; mutants (Y11C, and perhaps R7L)         this process through its interactions with the PTC prox- <br />                                                                imal loop of Helix 89 (post-state) and Helix 38 (pre-state). <br /> may inhibit proofreading by allowing faster accommoda- <br />                                                                By this model, interaction of the L10 hook with bases in <br /> tion of incorrect tRNAs. Since &#226;&#8364;&#8220;1 PRF was at best only <br />                                                                the PTC-proximal bulge in Helix 89 may help to open the <br /> marginally a&#239;&#172;&#8364;ected by the mutants examined in the study, it <br />                                                                aa-tRNA accommodation corridor. Upon A-site tRNA <br /> is more likely that the killer virus maintenance defects of <br />                                                                occupancy, the L10 hook could &#239;&#172;&#8218;ip up to interact with <br /> the L10 N-terminal &#226;&#8364;&#732;hook&#226;&#8364;&#8482; mutants were due to their strong <br />                                                                H38, releasing the H89 gate base to interact with its part- <br /> e&#239;&#172;&#8364;ects on 60S biogenesis as previously described (44,45). <br />                                                                ner at the base of Helix 92, which is controlled through <br /> L10 and information flow through the ribosome                  the L3 &#226;&#8364;&#732;rocker switch&#226;&#8364;&#8482; (see accompanying manuscript by <br />                                                                Meskauskas,A. and Dinman,J.D.). By this model then, <br /> Inspection of Figure 5D shows that L10 lies along a plane      the Y9C mutant would thermodynamically favor the <br /> formed by Helix 38, Helix 39/L10, Helix 89, Helix 91 and       closed conformation while Y11C would drive it toward <br /> Helix 95. This can be compared to &#239;&#172;&#65533;ve &#239;&#172;&#65533;ngers, with L10 as      the open state. Alternatively, it is possible that no such <br /> a proteinaceous prosthesis of Helix 39. In addition, L10       switch exists, and that the di&#239;&#172;&#8364;erences in the atomic resolu- <br /> interacts with the D-loop of 5S rRNA (data not shown).         tion structures could be due to phylogenetic di&#239;&#172;&#8364;erences <br /> Why would nature select for a removable protein to             (eukaryotes and archae versus bacteria), or due to di&#239;&#172;&#8364;er- <br /> occupy this space rather than simply extending Helix 39?       ent crystallization conditions. The answers to these ques- <br /> Examination of its placement reveals that L10 is at the        tions await the availability of high-resolution structures of <br /> center of a topological nexus that interacts with: 5S          eukaryotic ribosomes and of bacterial ribosomes com- <br /> rRNA, thus connecting it to the B1b and B1c intersubunit       plexed with EF-G. <br /> bridges at the crown of the large subunit; the GTPase- <br /> associated center, connecting it to elongation factor acti- <br /> vation; the A-site &#239;&#172;&#65533;nger (Helix 38), connecting it with the     <br /> decoding center through the B1a bridge; and the peptidyl-     <br /> transferase center. We suggest that the information &#239;&#172;&#8218;ow <br /> through this region of the large subunit is too complex to <br /> be monitored by a simple rRNA helix, thus necessitating         <br /> the evolution of an informationally more complex mole-         <br /> cule such as a protein to occupy this site. Thus, we suggest   <br /> that L10 functions to collect, distribute and coordinate     <br /> information throughout the large subunit.                      <br />    The observed e&#239;&#172;&#8364;ects of the L10 mutants on ribosome      <br /> structure and function could be explained consequent to         <br /> 60S mis-assembly defects. This notion is supported by        <br /> studies of the prokaryotic ortholog L16 showing that it         <br /> is only important for late assembly steps but that it is not    <br /> involved in other ribosomal functions (60). However, in <br /> support of our model, we note that if mis-assembly were <br /> the major issue, then the Y9C and Y11 C mutants would           <br /> have displayed similar changes in chemical protection pat-     <br /> terns. This was not the case. Instead, the chemical protec-    <br /> tion patterns observed with the Y9C mutants more closely      <br /> resembled those observed for various mutants of riboso-         <br /> mal protein L3 [(9) and accompanying manuscript], while       <br /> those for the Y11C mutants more closely resembled those      <br /> observed for mutants of Helix 38 and of the A-loop (7,11), <br /> suggesting that the Y9C and Y11C mutants independently     <br /> </body></html>