<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>BMC Genomics                                                                                                                             BioMed Central <br />  <br />  <br />  <br /> Research article                                                                                                                       Open Access <br /> Chemical-genetic profile analysis yeast suggests previously <br /> uncharacterized open reading frame, YBR261C, affects protein <br /> synthesis <br /> Md Alamgir1,2, Veronika Eroukova1,2, Matthew Jessulat1,2, Jianhua Xu3 and <br /> Ashkan Golshani*1,2 <br />  <br /> Address: 1Department Biology, Carleton University, 1125 Colonel By Drive, Ottawa, K1S 5B6, Canada, 2Ottawa Institute Systems Biology, <br /> Carleton University, 1125 Colonel By Drive, Ottawa, K1S 5B6, Canada 3College Life Science, JiLin University, Changchun, 130012, PR <br /> China <br /> Email: Md Alamgir - malamgir@connect.carleton.ca; Veronika Eroukova - erukova@gmail.com; <br /> Matthew Jessulat - mjessula@connect.carleton.ca; Jianhua Xu - john_xu_bioinfo@yahoo.ca; Ashkan Golshani* - ashkan_golshani@carleton.ca <br /> * Corresponding author <br />  <br />  <br />  <br />  <br /> Published: 3 December 2008                                                     Received: 26 June 2008 <br />                                                                                Accepted: 3 December 2008 <br /> BMC Genomics 2008, 9:583     doi:10.1186/1471-2164-9-583 <br /> This article available <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span>: http://www.biomedcentral.com/1471-2164/9/583 <br /> &#194;&#169; 2008 Alamgir et al; licensee BioMed Central Ltd. <br /> This Open Access article distributed terms Creative Commons Attribution License (http://creativecommons.org/licenses/by/2.0), <br /> which permits unrestricted use, distribution, reproduction medium, provided original work properly cited. <br />  <br />  <br />  <br />  <br />                  <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />                  Background: Functional genomics received considerable attention post-genomic era, <br />                  aims identify function(s) different genes. One way study gene function investigate <br />                  alterations responses deletion mutants different stimuli. Here investigate the <br />                  genetic profile yeast non-essential gene deletion array (yGDA, ~4700 strains) increased <br />                  sensitivity paromomycin, targets process protein synthesis. <br />                  Results: As expected, analysis indicated majority deletion strains (134) with <br />                  increased sensitivity paromomycin, involved protein biosynthesis. The remaining strains <br />                  divided smaller functional categories: metabolism (45), cellular component biogenesis <br />                  organization (28), DNA maintenance (21), transport (20), (38) unknown (39). These <br />                  represent minor cellular target sites (side-effects) paromomycin. They represent <br />                  novel links protein synthesis. One strains carries deletion previously <br />                  uncharacterized ORF, YBR261C, term TAE1 Translation Associated Element 1. Our <br />                  focused follow-up experiments indicated deletion TAE1 alters ribosomal profile the <br />                  mutant cells. Also, gene deletion strain TAE1 defects translation efficiency and <br />                  fidelity. Miniaturized synthetic genetic array analysis indicates TAE1 genetically interacts <br />                  16 ribosomal protein genes. Phenotypic suppression analysis using TAE1 overexpression also <br />                  links TAE1 protein synthesis. <br />                  Conclusion: We previously uncharacterized ORF, YBR261C, affects process of <br />                  protein synthesis reaffirm large-scale genetic profile analysis useful tool study <br />                  novel gene function(s). <br />  <br />  <br />  <br />  <br /> Background                                                                     tion containing thousands genes. Many genes <br /> The number available sequenced genomes pro-                             <span id='am-3' about='Thesaurus:code' typeof='owl:Thing'>code</span> proteins multiple functions, which <br /> vided biologists wealth sequence informa-                        known. Others <span id='am-86' about='Thesaurus:code' typeof='owl:Thing'>code</span> proteins completely <br />  <br />                                                                                                                                        Page 1 13 <br />                                                                                                                  (page number citation purposes) <br />  BMC Genomics 2008, 9:583                                                    http://www.biomedcentral.com/1471-2164/9/583 <br />  <br />  <br />  <br /> unknown function(s). <span id='am-6' about='protege:TO' typeof='owl:Thing'>To</span> tackle challenge,          Even underlying principles translation <br /> large-scale methodologies, term functional            machinery subject vigorous investiga- <br /> genomics, developed aim revealing            tions decades, details translation <br /> putative gene functions [1-3]. Due simple genetics,      related proteins, protein complexes pathways, well <br /> ease manipulation, conserved pathways, yeast         communications cross-talks cellu- <br /> Saccharomyces cerevisiae, emerged model organism        lar processes, fully elucidated. Recently, <br /> choice functional genomics [4]. While significant           large-scale genomic investigations uncovered <br /> knowledge gained various large-scale              numerous novel proteins thought functionally <br /> investigations, experiments needed uncover          related protein synthesis S. cerevisiae [14-16], sug- <br /> the details functions genes involved different     gesting remain undiscovered translation <br /> cellular processes. Exploring function individual        proteins. <br /> proteins greatly advance understanding the <br /> biology cell system.                                  Here, applied large-scale chemical-genetic profile <br />                                                                 analysis identify yeast deletion strains show <br /> Genes, involved similar pathways             increased sensitivity aminoglycoside antibiotic paro- <br /> genetically, interact other. Therefore, way    momycin. This compound exerts activity targeting <br /> study gene functions investigate interactions    process protein synthesis. Focused follow-up exper- <br /> make [3]. This based assumption       iments provided evidence YBR261C, previously <br /> that eukaryotic pathways functionally redun-           uncharacterized open reading frame (ORF) iden- <br /> dant. Thus, deletion gene tolerated         tified screen, affects process protein synthe- <br /> phenotypic consequences. Inactivation second func-         sis yeast. <br /> tionally related gene however, cause sickness even <br /> lethality. Therefore sickness double mutants &quot;syn-        Results <br /> thetic lethality&quot; used reveal novel gene func-      The Initial screening identification TAE1 <br /> tions. In simple terms, synthetic genetic array (SGA)           In order identify genes affect protein synthesis, we <br /> analysis refers large-scale investigation aimed exam-     screened entire set yGDA (~4700) increased sen- <br /> ining gene functions using double gene knockouts [3].           sitivity aminoglycoside paromomycin. Paromomy- <br />                                                                 cin binds small ribosomal subunit eukaryotic <br /> In addition role functional genomics, availability    cells compromises translation fidelity [17]. Previ- <br /> of yeast non-essential gene deletion array (yGDA,           ously, shown deletion certain translation <br /> approximately 4700 strains) provided opportu-          related genes caused increased sensitivity paromomy- <br /> nity investigate cellular target sites inhibitory     cin [18,19]. Therefore, employed hypersensitivity to <br /> compounds [5-7]. In way, compounds unknown            drug way detect novel gene candidates <br /> cellular target sites examined inhibitory         involved translation. It noted dele- <br /> effects yGDA. The hypersensitive strains genes      tion certain translation related genes cause <br /> known functions used form genetic profile      increased resistance paromomycin. This however, has <br /> activity target compound. This provides fast       investigated analysis. <br /> effective way investigate cellular target sites inhibi- <br /> tory compounds.                                                 We used yeast colony size reduction (CSR) tool to <br />                                                                 detect sensitivity drug treatments. We previously <br /> Similarly, inhibitory compounds known modes             shown, based parameters used (see <br /> activity used detect novel gene functions. This     Materials Methods), CSR analysis detect approxi- <br /> is novel concept various small-scale studies,      mately 63% sensitive strains detected a <br /> numerous gene functions examined based             large-scale spot test (ST) analysis repeats [20]. <br /> the increased sensitivity gene deletion strains     Hence, number sensitive strains would <br /> different compounds [8,9].                                      missed analysis, be <br />                                                                 detected ST. Similarly, large-scale ST analysis <br /> As final step gene expression pathway, regula-     failed detect 59% strains detected CSR, which <br /> tion protein synthesis (translation) used control      represent novel/false-positives associated CSR. <br /> the expression variety genes different physi- <br /> ological conditions. For example, cell division       Here, use sensitivity paromomycin tool to <br /> the early steps embryonic development [10,11], dur-       detect protein synthesis related genes. A draw for <br /> ing cellular transformation cancer development [12],        this, similar drug-based screening tests, is <br /> and well, stress conditions apoptosis [13].           detection sensitive strains genes no <br />                                                                 direct relation activity target drugs. A major <br />  <br />  <br />                                                                                                                     Page 2 13 <br />                                                                                               (page number citation purposes) <br />  BMC Genomics 2008, 9:583                                                    http://www.biomedcentral.com/1471-2164/9/583 <br />  <br />  <br />  <br /> source false-positives experiments           tion Associated Element 1, subjected further <br /> stem genes play role general stress        analysis potential involvement protein synthesis. <br /> conditions. Multi-drug resistant genes represent        ST analysis used confirm large-scale observa- <br /> these examples. For instance, shown        tions (Figure 2). When growth media supple- <br /> deletion QDR1, transporter gene member            mented sub-inhibitory concentration of <br /> efflux pumps, confers sensitivity unrelated          paromomycin (13 mg/ml), deletion strain TAE1 <br /> drugs [21]. To increase specificity selection pro-   (tae1&#206;&#8221;) showed reduction growth <span id='am-87' about='xsp:pattern' typeof='owl:Thing'>pattern</span> (Figure <br /> cedure, coupled initial screen secondary          2). <br /> search (based parameters) increased sen- <br /> sitivity second drug, reported activity    We examined tae1&#206;&#8221; strain increased sensitivity <br /> the process translation. For purpose, selected       3-amino-1,2,4-triazole (3-AT) cycloheximide. 3-AT <br /> calcofluor white (CW), known inhibit cell wall      affect translation altering pool amino acids <br /> function binding chitin [22]. In way,         cell [24]. Cycloheximide binds large ribosomal <br /> those gene deletions conferred sensitivity paromo-      subunit [25], inhibits translation elongation inter- <br /> mycin represent meaningful positives.                 fering tRNA translocation [26]. It previously <br />                                                                 reported deletion certain genes affect <br /> As expected, large-scale approach identified numer-         translation confer sensitivity multiple drugs that <br /> ous translation genes TEF4 (translation elonga-         target translation. For example, deletion Sfp1, which <br /> tion factor EF-1 gamma), HCR1 (a component                   regulates ribosome biogenesis, confers increased sensitiv- <br /> translation initiation factor 3), RPS18B (a ribosomal pro-      ity cycloheximide paromomycin [27]. Shown <br /> tein small subunit), etc, sensitive paromo-     Figure 2, spot test analysis indicated addi- <br /> mycin CW. The complete list genes        tion paromomycin, tae1&#206;&#8221; showed increased sensi- <br /> found Additional file 1. The list genes     tivity 3-AT. Also sensitivity tae1&#206;&#8221; to <br /> sensitive paromomycin CW Addi-          cycloheximide occasionally observed. However, due <br /> tional file 2. Of 325 gene deletions sensitive paro-     irreproducibility observations, sensitivity <br /> momycin alone, 42 genes appeared         presumed artifact. <br /> our similar drug screenings using different bioactive com- <br /> pounds. These represent false-positives          During preparation manuscript, new study <br /> be treated caution. From 325 total reported            reported sensitivity yeast gene deletion <br /> genes, 134 previously linked overall           mutants paromomycin investigated using heter- <br /> process protein biosynthesis. 191 however,           ozygous diploid yeast gene deletion mutant array [28]. <br /> have connected way process,       This different (haploid based), its <br /> therefore, represent novel/false-positive genes.       diploid mutant strains carry copy wild type <br /> These genes classified 5 smaller catego-    genes consequently obvious growth defects <br /> ries based cellular processes partici-     [29]. Of 51 mutant strains identified the <br /> pate, plus unknown. As indicated             authors sensitive paromomycin, 39 shared <br /> Figure 1, minor categories are: metabolism 45          systems. Of these, 16 mutants also <br /> genes, cellular component biogenesis organization           identified haploid system. It worth mentioning <br /> with 28, DNA maintenance 21, transport 20             tae1&#206;&#8221; detected increased sensitivity to <br /> and 38 genes. There 39 genes,       paromomycin heterozygous system, high- <br /> are unknown. An explanation smaller categories        lighting difference systems [28]. <br /> is represent minor cellular target sites (side- <br /> effects) paromomycin. Alternatively, simply        The effect TAE1 gene deletion protein synthesis <br /> represent false-positives (see above). It possible      Translation genes involved different aspects of <br /> that genes novel roles          translation. To examine involvement novel genes in <br /> translation. In case, represent communica-        protein synthesis, divided translation gen- <br /> tion bridges different cellular processes pro-      eral categories. Group includes genes are <br /> tein synthesis.                                                 associated ribosome biogenesis, group contains <br />                                                                 genes alter translation efficiency, group is <br /> One previously uncharacterized ORF identified         composed genes affect translation fidelity. <br /> this screen YBR261C. There reported information        Depending molecular function(s), transla- <br /> about ORF, computationally pre-          tion genes fall [30], [27,31] more <br /> dicted methyltransferases domain [23]. We             [32] categories. If TAE1 involved the <br /> therefore hypothesized novel transla-      process translation, fall of <br /> tion related gene. We termed ORF, TAE1, Transla-       categories. <br />  <br />  <br />                                                                                                                    Page 3 13 <br />                                                                                              (page number citation purposes) <br />  BMC Genomics 2008, 9:583                                                  http://www.biomedcentral.com/1471-2164/9/583 <br />  <br />  <br />  <br />  <br /> Figure 1 gene deletion strains increased sensitivity paromomycin <br /> Categories <br /> Categories gene deletion strains increased sensitivity paromomycin. Distribution paromomycin hyper- <br /> sensitive yeast deletion strains percentages according cellular functions. <br />  <br />  <br /> a) Involvement TAE1 ribosome biogenesis                 polysomes corresponding increases 80S mono- <br /> Ribosome biogenesis assembly           somes 60S subunits (Figure 3). These alterations in <br /> important processes protein synthesis pathway          ribosomal profile indicate involvement of <br /> [33]. This process explained overall step     TAE1 protein synthesis. <br /> that leads formation assembly functional <br /> ribosomes includes regulation pre-rRNA transcrip-      b) Involvement TAE1 translation efficiency <br /> tion, corresponding mechanisms, pre-rRNA              If TAE1 involved protein synthesis, based its <br /> processing, rRNA transport, rRNA maturation, ribosome         molecular function(s), deletion alter cell&apos;s effi- <br /> assembly, etc. [34]. We reasoned depending     ciency s synthesize proteins. To investigate possi- <br /> on molecular function(s) TAE1, gene         bility, used [35S] methionine incorporations to <br /> involved ribosome biogenesis assembly, deletion     measure rate total protein synthesis different <br /> might result alteration profile ribosomal    strains. As indicated Figure 4A, observed on <br /> subunits. To examine possibility, ribosome profile       average, tae1&#206;&#8221; reduced level [35S] methionine <br /> analysis performed. As expected, distinct 40S 60S     incorporation (approximately 22%) compared con- <br /> subunit peaks, 80S monosomes polysome          trol strain. To confirm observation, used induc- <br /> peaks (Figure 3) detected wild type      ible &#206;&#178;-galactosidase reporter construct (Figure 4B). It was <br /> tae1&#206;&#8221; strains. In addition, tae1&#206;&#8221; strain showed reduction   observed deletion TAE1 reduced level &#206;&#178;- <br />  <br />  <br />                                                                                                                   Page 4 13 <br />                                                                                             (page number citation purposes) <br />  BMC Genomics 2008, 9:583                                                       http://www.biomedcentral.com/1471-2164/9/583 <br />  <br />  <br />  <br />                                                                    efficiency protein synthesis cell, provide fur- <br />                                                                    ther support TAE1 affects protein synthesis. <br />  <br />                                                                    c) TAE1 Translation fidelity <br />                                                                    To make functional protein, fidelity protein syn- <br />                                                                    thesis maintained stages translation. <br />                                                                    This fidelity controlled start site selection <br />                                                                    elongation mis-incorporation wrong <br />                                                                    amino acid alter integrity final product, as <br />                                                                    termination, stop codon be <br />                                                                    read through. If TAE1 involved translation, it <br />                                                                    expected based molecular func- <br />                                                                    tion(s), deletion gene alter fidelity of <br />                                                                    translation. Translation fidelity studied using spe- <br />                                                                    cialized expression systems contain &#206;&#178;- <br />                                                                    galactosidase expression cassettes, premature stop <br />                                                                    codons [35]. In investigation, used plasmids <br />                                                                    pUKC817 pUKC818 contain lacZ genes carrying <br />                                                                    nonsense codons UAA UAG, respectively [35]. It <br />                                                                    observed deletion TAE1 resulted an <br />                                                                    increased level read nonsense codons <br />                                                                    investigated, suggesting deletion strain, the <br />                                                                    translation fidelity compromised (Figure 5A). <br />                                                                    For unknown reasons, average values &#206;&#178;-galactosi- <br />                                                                    dase activities wild type strain, systematically <br />                                                                    higher expected. <br />  <br />                                                                    Since differential levels &#206;&#178;-galactosidase activity above <br />                                                                    experiments stem altered levels mRNAs, <br />                                                                    content &#206;&#178;-galactosidase mRNAs WT tae1&#206;&#8221; <br />                                                                    investigated using Q-RT-PCR. We observed signif- <br />                                                                    icant variation amounts mRNAs (Figure <br />                                                                    5B), explain observed difference the <br /> Figure <br /> Increased <br /> tory  drugs <br />          2 sensitivity tae1&#206;&#8221; different translation inhibi-   &#206;&#178;-galactosidase activities. It concluded that <br /> Increased sensitivity tae1&#206;&#8221; different translation            observed differences likely stem translation read <br /> inhibitory drugs. Deletion TAE1 confers increased sensi-        through. Altogether, results indicate deletion <br /> tivity different drugs target translation. Decreasing      TAE1 compromise translation fidelity, provid- <br /> numbers wild type mutant (tae1&#206;&#8221;) yeast cells           ing evidence TAE1 affects translation. <br /> spotted solid media. The media supplemented with <br /> sub-inhibitory concentrations paromomycin (13 mg/ml), 3- <br /> AT (22 mg/ml), cycloheximide (45 ng/ml), (used        TAE1 genetically interacts translation related genes <br /> control shown panel). Standard organic              The genes functionally related involved in <br /> (YEPD) media used cycloheximide synthetic              similar pathways, genetically interact each <br /> complete (SC) media used paromomycin 3-AT              other. Consequently, studying genetic interactions of <br /> plates, control plate shown here. Yeast cells     novel gene used method infer func- <br /> grown mid-log phase diluted 10-3 10-6 folds.             tion gene [36,37]. If TAE1 true translation gene, <br /> Twenty microliters dilution (gradually decreasing)         expected TAE1 genetically <br /> was spotted media grown 30&#194;&#176;C 1&#226;&#8364;&#8220;2 days.         interact certain known translation associated genes. <br /> Deletion TAE1 conferred increased sensitivity paromo-        To investigate possibility, examined genetic <br /> mycin 3-AT. Occasional sensitivity cycloheximide        interactions TAE1 set 384 genes, are <br /> assumed artifact. <br />                                                                    known thought involved translation. As indi- <br />                                                                    cated Figure 6, observed TAE1 genetically <br />                                                                    interacted numerous translation related genes pro- <br /> galactosidase synthesis inducible expression plas-         duce sick phenotypes, classified i) sick, <br /> mid approximately ten-folds (Figure 4B). These obser-           ii) sick, iii) moderate. Lethal interactions not <br /> vations suggest deletion TAE1 reduce               considered. The interactions TAE1 transla- <br />  <br />  <br />                                                                                                                        Page 5 13 <br />                                                                                                  (page number citation purposes) <br />  BMC Genomics 2008, 9:583                                                         http://www.biomedcentral.com/1471-2164/9/583 <br />  <br />  <br />  <br />  <br /> Figure 3 profile analysis yeast strains <br /> Ribosome <br /> Ribosome profile analysis yeast strains. Deletion TAE1 results overall decrease polysomes, corre- <br /> sponding increase 80S monosome formations. There increase 60S subunit tae1&#206;&#8221; cells. The large polysomes <br /> were collected lower half gradients (x-axis). The values peaks represent areas curve for <br /> each peak, percentages. Each experiment repeated times similar outcomes. <br />  <br />  <br />  <br /> tion related genes divided catego-           rRNA synthesis (2 genes). Descriptions genes are <br /> ries ribosomal proteins forming largest cluster             listed Additional file 3. The fact TAE1 genetically <br /> (16 genes), followed involved amino acids                interacts different translation associated genes pro- <br /> and protein production (7 genes), involved              vides evidence involvement TAE1 the <br />                                                                      process translation. <br />  <br />                                                                      Phenotypic suppression overexpression TAE1 <br />                                                                      To examine cellular activity Tae1 protein, we <br />                                                                      employed high throughput approach, based phe- <br />                                                                      notypic suppression gene deletion mutants that <br />                                                                      known functions. Deletion genes, are <br />                                                                      involved specific pathway, cause increased sensi- <br />                                                                      tivity treatments target process. Such <br />                                                                      hypersensitivities compensated overexpres- <br /> Figure synthesis <br /> Protein  4          profile presence absence Tae1      sion genes similar cellular functions. For <br /> Protein synthesis profile presence absence                example, previously reported absence <br /> of Tae1. In (A) total protein synthesis measured using            Yku80, involved telomere maintenance, causes <br /> [35S] methionine incorporation. The average count [35S]          increased sensitivity elevated temperature. Overexpres- <br /> methionine incorporation wild type 11,356,073 (&#194;&#177;              sion Est2, catalytic subunit telomerase, or <br /> 1,300,000) counts, set 100%. In absence           Tlc1, RNA template component telomerase, com- <br /> Tae1 protein, total protein production reduced approx- <br />                                                                      pensated absence Yku80 reversed heat <br /> imately 22%. t-test analysis indicates observed differ- <br /> ence statistically significant p-value 0.05. In (B)   hypersensitivity yku80&#206;&#8221; [38]. <br /> the efficiency protein synthesis measured using induc- <br /> ible &#206;&#178;-galactosidase reporter gene. The average &#206;&#178;-galactosi-         Here, investigated activity Tae1 protein exam- <br /> dase activity wild type 7.51 (&#194;&#177; 0.6) units, set   ining effect overexpression suppressing hyper- <br /> 100%. In absence Tae1 protein, approximately ten-          sensitivities antibiotics neomycin streptomycin for <br /> fold reduction &#206;&#178;-galactosidase production observed.            384 translation related gene deletion yeast <br />                                                                      strains. Like paromomycin, antibiotics neomycin and <br />  <br />  <br />                                                                                                                         Page 6 13 <br />                                                                                                   (page number citation purposes) <br />  BMC Genomics 2008, 9:583                                                     http://www.biomedcentral.com/1471-2164/9/583 <br />  <br />  <br />  <br />  <br /> Figure <br /> Effect of5TAE1 deletion translation fidelity <br /> Effect TAE1 deletion translation fidelity. (A) Deletion TAE1 resulted increased levels &#206;&#178;-gal activity the <br /> lacZ reporters, pUKC817 pUKC818, containing premature stop codons, UAA UAG, respectively. The relative &#206;&#178;-gal <br /> activity determined normalizing activity mutant constructs (pUKC817 pUKC818) control construct <br /> (pUKC815), related obtained wild type strain (for example, value pUKC817/pUKC815 tae1&#206;&#8221; cells <br /> is related pUKC817/pUKC815 wild type cells). The average &#206;&#178;-gal activity wild type strain transformed <br /> with pUKC815 19.05 (&#194;&#177; 1.1) units. (B) Q-RT-PCR analysis indicates alterations relative contents LacZ mRNAs <br /> do explain difference &#206;&#178;-gal activities observed (A). The Ct control experimental samples calcu- <br /> lated threshold cycles. pUKC815 background construct premature stop codon used control. <br />  <br />  <br /> streptomycin, belong aminoglycoside family,               One approach study gene function make gene <br /> which binds ribosomes disrupts translation [30].          knockouts, observe mutant cells&apos; behavior to <br /> We observed overexpression Tae1 protein, sup-            internal and/or external stimuli. Here, screened set <br /> pressed drug sensitivity phenotypes 28 deletion          yGDA sensitivity paromomycin, known <br /> mutants known translation genes (Figure 7). These 28          target protein synthesis machinery. Due increased <br /> mutants showed sensitivity treatment neomycin            sensitivity, hypothesized yeast gene deletion <br /> and/or streptomycin. Tae1 overexpression however,                strain TAE1 affect process protein synthe- <br /> reversed observed drug sensitivities. These 28 gene          sis. We observed deletion TAE1 reduced trans- <br /> deletion mutants categorized main groups         lation fidelity. This observation expected, since <br /> of gene deletions ribosomal proteins (17 genes),         paromomycin known decrease fidelity transla- <br /> those translation control proteins (8 genes). Descrip-       tion. We observed deletion TAE1 reduced the <br /> tions deleted genes listed Additional file 4.      efficiency translation, necessarily coupled <br /> The fact overexpression Tae1 suppresses phe-         translation fidelity, suggesting wide-range effect for <br /> notypes deletion mutants translation genes,       TAE1 translation. <br /> confirms involvement TAE1 protein synthesis. <br />                                                                  Tae1 protein cytoplasm bioinfor- <br /> Discussion                                                       matically predicted contain S-adenosylmethionine- <br /> Annotating gene functions important aspect           dependent methyltransferase activity. Certain members of <br /> of post-genomic era. Identifying &quot;which gene does what&quot;          family proteins shown methylate dif- <br /> is fundamental tasks systems biology,          ferent components translation machinery. For <br /> sets basis understanding biology cell. There    example, DIM1 SPB1, nucleolar proteins <br /> are currently numerous uncharacterized genes             involved rRNA methylation; TRM proteins (such as <br /> known functions [4]. Moreover, numerous genes          TRM1 TRM2), tRNA methyltransferases <br /> with multiple functions, eluci-        cytoplasm nucleus; Mtq2, which <br /> dated.                                                           methylates translation release factor SUP45 found <br />  <br />  <br />                                                                                                                      Page 7 13 <br />                                                                                                (page number citation purposes) <br />  BMC Genomics 2008, 9:583                                                       http://www.biomedcentral.com/1471-2164/9/583 <br />  <br />  <br />  <br />  <br /> Figure 6interaction TAE1 translation genes <br /> Genetic <br /> Genetic interaction TAE1 translation genes. TAE1 genetically interacts numerous translations related genes <br /> to produce sick phenotypes. These interacting genes divided major categories ribosomal subunits <br /> (16 genes, blue circles), amino acid protein production (7 genes, black circles). The interacting genes shown red cir- <br /> cles involved ribosome biogenesis. Strong, moderate weak interactions shown red, green blue lines, <br /> respectively. <br />  <br />  <br /> in cytoplasm nucleus. Presuming Tae1            co-regulated number ribosome processing fac- <br /> a methyltransferase activity, assume Tae1             tors different microarray analyses [40]. <br /> affects translation methylating component transla- <br /> tion machinery.                                                   There accumulation 80S monosomes tae1&#206;&#8221; <br />                                                                   cells, indicate defect translation initiation. <br /> In agreement observed reduction translation           Formation defective ribosomes absence TAE1 <br /> efficiency tae1&#206;&#8221;, ribosomal profile analysis indi-        readily start elongation, explain this <br /> cated overall decrease polysomes TAE1              accumulation 80S monosomes. Alternatively, pos- <br /> deleted. In contrast, 60S free subunits specifically     sible TAE1 affect translation initiation mod- <br /> accumulated tae1&#206;&#8221; cells. Since 40S 60S subunits            ifying translation initiation protein. The latter <br /> are equilibrium 80S monosomes, increase            explanation however, explain accumulation of <br /> 60S subunit indicate defect 40S biogenesis [39].         60S subunits. Regardless, experiments required <br /> Assuming TAE1 methyltransferase, possible             investigate details molecular activity TAE1, <br /> explanation TAE1 effect 40S biogenesis             identify potential substrate(s). <br /> either methylating 18S rRNA directly, methylating a <br /> factor affects 40S biogenesis. In agreement          Our genetic analysis revealed TAE1 genetically inter- <br /> role TAE1 ribosome biogenesis, TAE1         acts number translation genes. In accord the <br />                                                                   above-suggested function TAE1 ribosome subunit <br />  <br />  <br />                                                                                                                        Page 8 13 <br />                                                                                                  (page number citation purposes) <br />  BMC Genomics 2008, 9:583                                                      http://www.biomedcentral.com/1471-2164/9/583 <br />  <br />  <br />  <br />  <br /> Figure 7 <br /> Overexpression    TAE1 phenotypically suppresses hypersensitivity numerous translation genes drug treatments <br /> Overexpression TAE1 phenotypically suppresses hypersensitivity numerous translation genes against <br /> drug treatments. Overexpression TAE1 suppressed inhibitory effects neomycin and/or streptomycin 28 yeast <br /> gene deletion strains. Among them, 17 genes belong ribosomal proteins (blue circles) 8 genes involved translation <br /> control (black circles). Red circles represent genes. Complete partial suppressions shown green blue <br /> lines, respectively. <br />  <br />  <br /> biogenesis, majority observed interactions       tion. One mutant strains identified screen <br /> found ribosomal protein genes. Similarly,         deletion strain previously uncharacterized <br /> phenotypic suppression analysis indicated functional           ORF, YBR261C, termed TAE1, Translation <br /> compensation TAE1 overexpression, absence          Associated Element 1. Our follow-up experiments indi- <br /> 17 different ribosomal proteins drug treatments. It      cated deletion TAE1 caused reduction trans- <br /> should noted genetic phenotypic suppres-         lation efficiency fidelity. Deletion mutant strain for <br /> sion analysis resulted different sets proteins. This   TAE1 altered ribosome profile. Our genetic <br /> is expected, genetic interaction analyses generally target    analyses confirmed involvement gene <br /> the genes involved different pathways process        translation. Identification new gene process <br /> (redundant pathways), phenotypic suppression             translation suggests exist novel <br /> analyses generally target genes pathway.         translation genes, discovered. <br /> This data used study detailed mecha- <br /> nism TAE1 activity.                                           In addition, large-scale phenotypic suppression analy- <br />                                                                  sis used set path similar approaches to <br /> Conclusion                                                       investigate gene functions. Furthermore, data <br /> In conclusion, investigated sensitivity yGDA        reaffirms large-scale chemical-genetic profile <br /> paromomycin, drug targets process transla-         analysis successfully used functional genomics. <br />  <br />  <br />                                                                                                                       Page 9 13 <br />                                                                                                 (page number citation purposes) <br />  BMC Genomics 2008, 9:583                                                      http://www.biomedcentral.com/1471-2164/9/583 <br />  <br />  <br />  <br />  <br /> Materials methods                                             Gene expression analysis <br /> Drug resistance screening yeast gene deletion array           Constitutively    expressed    &#206;&#178;-galactosidase     (using <br /> Sub-inhibitory concentrations drugs estimated             pUKC815, pUKC817 pUKC818) assayed as <br /> from minimum inhibitory concentration (MIC) measure-              described [43]. The units enzyme activity calcu- <br /> ments based 96 varied concentration format                lated nanomoles O-nitrophenyl-&#206;&#177;-D-galactopyrano- <br /> described [41].                                                (OPNG) hydrolyzed microgram total protein <br />                                                                   [44]. All assays conducted triplicate. Induced &#206;&#178;- <br /> Approximately 4700 MATa haploid yeast, S. cerevisiae              galactosidase (using p416) assayed [45]. <br /> strains (BY4741, MATa ura3&#206;&#8221;0 leu2&#206;&#8221;0 his3&#206;&#8221;1 met15&#206;&#8221;0) <br /> from non-essential Gene Deletion Array (yGDA)                 In vivo [35S] methionine incorporation performed as <br /> described [3,5] manually arrayed agar plates          previously described Schwartz Parker [46] with <br /> as previously described [36] 384 array format,          modifications. Briefly, yeast strains grown mid-log <br /> sub-inhibitory concentrations paromomycin (13 mg/              phase 30&#194;&#176;C YEPD. The cells harvested, resus- <br /> ml), CW (40 &#206;&#188;g/ml) drug (control). All plates        pended pre-warmed minimal medium lacking methio- <br /> were incubated 30&#194;&#176;C 1&#226;&#8364;&#8220;2 days. The sensitivity       nine, supplemented 10 &#206;&#188;Ci/ml [35S] <br /> gene deletion array different drugs investigated        methionine. The cells incubated 1 h 30&#194;&#176;C and <br /> before [36]. In brief, different strains pinned        harvested centrifugation. The samples then <br /> plates, sub-inhibitory concentration target       washed distilled water times collected (1 &#206;&#188;l <br /> drug (used control) incubated            aliquot) Whatman paper. The paper air dried and <br /> at 30&#194;&#176;C 1&#226;&#8364;&#8220;2 days. Digital images plates used          exposed storage phosphor screen 1 h. The counts <br /> to analyze growth individual colonies. For           normalized final cell totals. Each experiment <br /> plate average size colonies (white pixels) calcu-      repeated times. <br /> lated equation (1). <br />                                                                   Total RNA isolated using Bio-Rad RNA isolation kit. <br />                                                                   cDNA constructed 0.5 &#206;&#188;g total RNA each <br />                                      &#226;&#710;&#8216; <br />                                          N <br />                      S ave = 1 / N              Si         (1)    strain using iScript cDNA synthesis kit SYBR green <br />                                          =1 <br />                                                                   supermix (Bio-Rad) according instructions the <br /> where N total number colonies present given        manufacturer. The quantification mRNA per- <br /> plate Si area object i. The deviation area   formed using real-time RT-PCR (Q-RT-PCR) Rotor- <br /> each colony plate&apos;s average area used fur-       Gene RG-300 Corbett research. The PCR quantifica- <br /> ther analysis (relative growth) calculated sub-        tion melting curves generated using Rotor <br /> tracting scalar Save plate&apos;s ordered area array      gene 6 software. The amplification performed: initial <br /> explained equation (2).                                        denaturation 95&#194;&#176;C 10 min followed 40 cycles at <br />                                                                   95&#194;&#176;C 30 s, 55&#194;&#176;C 20 s 72&#194;&#176;C 20 s with <br />         &#206;&#8221;Si = Si - Save; = 1, ..., 384 (16 &#195;&#8212; 24 = 384)   (2)    using 50 nmoles forward (5&apos;-ACTATCCCGAC- <br />                                                                   CGCCTTACT) reverse (5&apos;-TAGCGGCTGATGTT- <br /> Relative colony size reduction 30%               GAACTG) primers. The fluorescence read end <br /> counted &quot;hit&quot;. Each experiment repeated            round amplification. All standard dilutions and <br /> times. The gene deletion strains, &quot;hits&quot;        samples run triplicate. Quantification mRNA <br /> or experiments, counted posi-           achieved comparing threshold cycle (Ct) value <br /> tives. Colonies highest average reductions        sample RNA deletion strain Ct value <br /> 61% 30&#226;&#8364;&#8220;60% defined supersensitives               WT strain&apos;s standard RNA [47]. <br /> and sensitives, respectively. <br />                                                                   Ribosome profile analysis <br /> ST test analysis performed growing yeast cell cul-         Ribosome profiling performed described Foani <br /> tures YEPD media mid-log, following 10-3 10-6            et al [48] following modifications. Wild type and <br /> folds dilutions. Twenty microliters dilution (grad-       mutant yeast cells grown YEPD 30&#194;&#176;C den- <br /> ually decreasing), spotted media contain-           sity 2 &#195;&#8212; 107 cells/ml. Cycloheximide (50 &#206;&#188;g/ml) was <br /> ing sub-inhibitory concentrations drugs (13            added culture cultures quickly <br /> mg/ml paromomycin, 22 mg/ml 3-AT, 45 ng/ml                    chilled ice water bath. Cells centrifuged at <br /> cycloheximide), (control). YEPD media             4000 rpm 4 min 4&#194;&#176;C sorvall SLA-1500 rotor. Cell <br /> used cycloheximide SC media used paro-            pellets resuspended 10 ml ice-cold YA buffer <br /> momycin 3-AT plates. The growth patterns com-            (breaking buffer A: 10 mM Tris-HCl [pH 7.4], 100 mM <br /> pared 1&#226;&#8364;&#8220;2 days 30&#194;&#176;C [42].                          NaCl, 30 mM MgCl2, cycloheximide 50 &#206;&#188;g/ml, heparin <br />                                                                   200 &#206;&#188;g/ml) centrifuged 4000 rpm, 4 min, 4&#194;&#176;C <br />  <br />  <br />                                                                                                                     Page 10 13 <br />                                                                                                (page number citation purposes) <br />  BMC Genomics 2008, 9:583                                                        http://www.biomedcentral.com/1471-2164/9/583 <br />  <br />  <br />  <br /> (Sorvall SS34 rotor), twice. The pellets resus-     ments. MA, MJ AG wrote manuscript. All authors <br /> pended 0.5 ml YA buffer vortex stored -       read approved final manuscript. <br /> 80&#194;&#176;C till day. Cells thawed ice water <br /> bath. Glass beads added vortex 20 sec max-    Additional material <br /> imum speed, 10 times 30 sec interval. The superna- <br /> tants centrifuged 8,000 &#195;&#8212; g 10,000 &#195;&#8212; g 10 <br /> min 30 min, respectively. Twenty OD260 units each <br />                                                                    Additional file 1 <br />                                                                    Supplemental Table 1. List genes deletions, sensitive to <br /> supernatant fractionated 8&#226;&#8364;&#8220;48% sucrose gradi-              paromomycin. <br /> ents containing 50 mM Tris-acetate (pH 7.0), 50 mM                 Click file <br /> NH4Cl, 12 mM MgCl2, 1 mM dithiothreitol. The                   [http://www.biomedcentral.com/content/supplementary/1471- <br /> extracts centrifuged 2.5 hrs 39,000 rpm           2164-9-583-S1.xls] <br /> using SW40-Ti rotor Beckman LE-80K 4&#194;&#176;C. The <br /> ribosome profiles analyzed collected gradi-          Additional file 2 <br /> ent solutions monitoring absorbance 254 nm.              Supplemental Table 2. List genes deletions, sensitive to <br />                                                                    paromomycin calcofluor white (CW). <br /> Each experiment repeated times similar              Click file <br /> results.                                                           [http://www.biomedcentral.com/content/supplementary/1471- <br />                                                                    2164-9-583-S2.xls] <br /> Genetic interaction phenotypic suppression analysis <br /> Genetic interactions TAE1 set yeast gene          Additional file 3 <br /> deletion strains 384 genes known thought              Supplemental Table 3. Descriptions translation related genes that <br /> involved translation assessed synthetic genetic         genetically interact TAE1. <br />                                                                    Click file <br /> miniarray profiling discussed [49] colony <br />                                                                    [http://www.biomedcentral.com/content/supplementary/1471- <br /> growth differences assessed [20]. Possible syn-         2164-9-583-S3.doc] <br /> thetic sick interactions confirmed spotting assay <br /> explained [50]. We divided interactions              Additional file 4 <br /> strong, moderate weak interactions based              Supplemental Table 4. Descriptions translation related genes that <br /> double mutant phenotypes sick, sick slightly           phenotypically suppressed overexpression TAE1, treat- <br /> sick.                                                              ment neomycin and/or streptomycin. <br />                                                                    Click file <br />                                                                    [http://www.biomedcentral.com/content/supplementary/1471- <br /> The TAE1 overexpression construct, pGAL1/10-GST-                   2164-9-583-S4.doc] <br /> 6xHis-YBR261C obtained yeast gene overex- <br /> pression array explained [51]. A compatible MAT&#206;&#177; <br /> strain (Y7092, MAT&#206;&#177; can1&#206;&#8221;::STE2pr-Sp_his5lyp1&#206;&#8221;his3&#206;&#8221;1 <br /> leu2&#206;&#8221;0 ura3&#206;&#8221;0 met15&#206;&#8221;0) transformed          Acknowledgements <br /> construct crossed set yeast gene deletion   We like thanks Julie Guzzo Helena Friesen <br /> strains 384 translation genes (see above) [49].     Dr. Brenda Andrews&apos;s lab University Toronto discus- <br /> Sensitivity yeast strains TAE1 overex-     sions assistance initial steps SGA experiments for <br /> pression, neomycin streptomycin per-          preliminary analysis. The authors like thank Dr. Mick <br /> formed using colony size measurements discussed         Tuite (University Kent Canterbury) Dr. Terri Goss Kinzy (Univer- <br />                                                               sity Medicine Dentistry New Jersey) generous gifts of <br /> [20]. Phenotypic complementations divided into <br />                                                               pUKC-based plasmids. Many thanks Chris Kiani excellent techni- <br /> two categories complete partial suppressions. Par- <br />                                                               cal assistance Joanna Freedman excellent editorial work. This work <br /> tial complementation drugs assigned par-      supported Natural Sciences Engineering Research Council <br /> tial. Complete complementation drug,              Canada (NSERC). This work dedicated loving memory Abbas <br /> regardless other, assigned complete.                Golshani. <br />  <br /> Abbreviations                                                 References <br /> yGDA: yeast non-essential Gene Deletion Array; ORF:           1.      Ito T, Chiba T, Ozawa R, Yoshida M, Hattori M, Sakaki Y: A compre- <br />                                                                       hensive two-hybrid analysis explore yeast protein <br /> open reading frame; Tae1: Translation Associated Element              interactome. Proc Natl Acad Sci USA 2001, 98(8):4569-4574. <br /> 1; 3-AT: 3-Amino-1, 2, 4-triazole.                            2.      Gavin AC, Bosche M, Krause R, Grandi P, Marzioch M, Bauer A, <br />                                                                       Schhultz J, Rick JM, Michon AM, Cruciat CM, et al.: Functional <br />                                                                       organization yeast proteome systematic analysis of <br /> Authors&apos; contributions                                                protein complexes. Nature 2002, 415(6868):141-147. <br /> All authors contributed conceptual development      3.      Tong AHY, Evangelista M, Parsons AB, Xu H, Bader GD, Pag&#195;&#169; N, Rob- <br />                                                                       inson M, Raghibizadeh S, Hogue CWV, Bussey H, Andrews B, Tyers <br /> the project. MA VE conducted experiments                  M, Boone C: Systematic genetic analysis ordered arrays <br /> analyzed results. JX AG administered experi-              yeast deletion mutants. Science 2001, 294:2364-2368. <br />                                                               4.      Pe&#195;&#177;a-Castillo L, Hughes TR: Why 1000 <br />                                                                       uncharacterized yeast genes? Genetics 2007, 176(1):7-14. <br />  <br />  <br />                                                                                                                            Page 11 13 <br />                                                                                                       (page number citation purposes) <br />  BMC Genomics 2008, 9:583                                                                        http://www.biomedcentral.com/1471-2164/9/583 <br />  <br />  <br />  <br /> 5.    Parsons AB, Lopez A, Givoni IE, Williams DE, Gray CA, Porter J,            24.   Zhou W, Edelman GM, Mauro VP: Isolation identification of <br />       Chua G, Sopko R, Brost RL, Ho CH, Wang J, Ketela T, Brenner C,                   short nucleotide sequences affect translation initiation <br />       Brill JA, Fernandez GE, Lorenz TC, Payne GS, Ishihara S, Ohya Y,                 Saccharomyces cerevisiae. Proc Natl Acad Sci USA 2003, <br />       Andrews B, Hughes TR, Frey BJ, Graham TR, Andersen RJ, Boone C:                  100(8):4457-4462. <br />       Exploring mode-of-action bioactive compounds                     25.   Leeds P, Wood JM, Lee BS, Culbertson MR: Gene products that <br />       chemical-genetic profiling yeast. Cell 2006, 126(3):611-625.                  promote mRNA turnover Saccharomyces cerevisiae. Mol <br /> 6.    Lopez A, Parsons AB, Nislow C, Giaever G, Boone C: Chemical-                     Cell Biol 1992, 12(5):2165-2177. <br />       genetic approaches exploring mode action nat-                26.   Obrig TG, Culp WJ, McKeehan WL, Hardesty B: The mechanism <br />       ural products. Prog Drug Res 2008, 66:237-271.                                   cycloheximide related glutarimide antibiotics <br /> 7.    Parsons AB, Brost RL, Ding H, Li Z, Zhang C, Sheikh B, Brown GW,                 inhibit peptide synthesis reticulocyte ribosomes. J Biol <br />       Kane PM, Hughes TR, Boone C: Integration chemical-genetic                     Chem 1971, 246(1):174-181. <br />       genetic interaction data links bioactive compounds                  27.   Fingerman I, Nagaraj V, Norris D, Vershon AK: Sfp1 plays key <br />       cellular target pathways. Nat Biotechnol 2004, 22(1):62-69.                      role yeast ribosome biogenesis.             Eukaryot Cell 2003, <br /> 8.    Lesage G, Sdicu AM, M&#195;&#169;nard P, Shapiro J, Hussein S, Bussey H: Anal-              2(5):1061-1068. <br />       ysis beta-1,3-glucan assembly Saccharomyces cerevisiae               28.   Hillenmeyer ME, Fung E, Wildenhain J, Pierce SE, Hoon S, Lee W, <br />       using synthetic interaction network altered sensitivity                    Proctor M, St Onge RP, Tyers M, Koller D, Altman RB, Davis RW, <br />       caspofungin. Genetics 2004, 167(1):35-49.                                     Nislow C, Giaever G: The chemical genomic portrait yeast: <br /> 9.    Chloupkov&#195;&#161; M, LeBard LS, Koeller DM: MDL1 high copy sup-                    uncovering <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span>phenotype</span> genes.                 Science 2008, <br />       pressor ATM1: evidence role resistance oxida-                     320(5874):362-365. <br />       tive stress. J Mol Biol 2003, 331(1):155-165.                              29.   Deutschbauer AM, Jaramillo DF, Proctor M, Kumm J, Hillenmeyer <br /> 10.   Johnstone O, Lasko P: Translational Regulation RNA Local-                    ME, Davis RW, Nislow C, Giaever G: Mechanisms haploinsuf- <br />       ization Drosophila Oocytes Embryos. Annu Rev Genet                        ficiency revealed genome-wide profiling yeast. Genetics <br />       2001, 35:365-406.                                                                2005, 169(4):1915-1925. <br /> 11.   Kuersten S, Goodwin EB: The Power 3&apos;UTR: Transla-                   30.   Walter F, Vicens Q, Westhof E: Aminoglycoside-RNA interac- <br />       tional Control Development. Nat Rev Genet 2003,                              tions. Curr Opin Chem Biol 1999, 3(6):694-704. <br />       4(8):626-637.                                                              31.   Iouk TL, Aitchison JD, Maguire S, Wozniak RW: Rrb1p, yeast <br /> 12.   Ruggero D, Pandolfi PP: Does ribosome translate cancer?                      nuclear WD-repeat protein involved regulation of <br />       Nat Rev Cancer 2003, 3(3):179-192.                                               ribosome biosynthesis. Mol Cell Biol 2001, 21(4):1260-1271. <br /> 13.   Holcik M, Sonenberg N: Translation control stress apop-             32.   Dinman JD, Kinzy TG: Translational misreading: mutations in <br />       tosis. Nat Rev Mol Cell Biol 2005, 6(4):318-327.                                 translation elongation factor 1alpha differentially affect pro- <br /> 14.   Peng WT, Robinson MD, Mnaimneh S, Krogan NJ, Cagney G, Morris                    grammed ribosomal frameshifting drug sensitivity. RNA <br />       Q, Davierwala AP, Grigull J, Yang X, Zhang W, Mitsakakis N, Ryan                 1997, 3(8):870-881. <br />       OW, Datta N, Jojic V, Pal C, Canadien V, Richards D, Beattie B, Wu         33.   Venema J, Tollervey D: Ribosome synthesis Saccharomyces <br />       LF, Altschuler SJ, Roweis S, Frey BJ, Emili A, Greenblatt JF, Hughes TR:         cerevisiae. Annu Rev Genet 1999, 33:261-311. <br />       A panoramic view yeast noncoding RNA processing. Cell                   34.   Rosado IV, Kressler D, la Cruz J: Functional analysis Saccha- <br />       2003, 113(7):919-933.                                                            romyces cerevisiae ribosomal protein Rpl3p ribosome syn- <br /> 15.   Krogan NJ, Peng WT, Cagney G, Robinson MD, Haw R, Zhong G,                       thesis. Nucleic Acids Res 2007, 35(12):4203-4213. <br />       Guo X, Zhang X, Canadien V, Richards DP, Beattie BK, Lalev A,              35.   Stansfield I, Akhmaloka , Tuite MF: A mutant allele SUP45 <br />       Zhang W, Davierwala AP, Mnaimneh S, Starostine A, Tikuisis AP,                   (SAL4) gene Saccharomyces cerevisiae shows temperature- <br />       Grigull J, Datta N, Bray JE, Hughes TR, Emili A, Greenblatt JF: High-            dependent allosuppressor omnipotent suppressor phe- <br />       <span id='am-4' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span> macromolecular composition yeast RNA-                              notypes. Curr Genet 1995, 27(5):417-426. <br />       processing complexes. Mol Cell 2004, 13(2):225-239.                        36.   Ideker T, Sharan R: Protein networks disease. Genome Res <br /> 16.   Fleischer TC, Weaver CM, McAfee KJ, Jennings JL, Link AJ: System-                2008, 18:644-652. <br />       atic identification functional screens uncharacterized              37.   Boone C, Bussey H, Andrews BJ: Exploring genetic interactions <br />       proteins associated eukaryotic ribosomal complexes.                         networks yeast. Nat Rev Genet 2007, 8(6):437-49. <br />       Genes Dev 2006, 20(10):1294-1307.                                          38.   Teo SH, Jackson SP: Telomerase subunit overexpression sup- <br /> 17.   Fan-Minogue H, Bedwell DM: Eukaryotic ribosomal RNA deter-                       presses telomere-specific checkpoint activation yeast <br />       minants aminoglycoside resistance role trans-                    yku80 mutant. EMBO Rep 2001, 2(3):197-202. <br />       lational fidelity. RNA 2008, 14:148-157.                                   39.   Vl&#195;&#161;sek L, Phan L, Schoenfeld LW, Val&#195;&#161;skov&#195;&#161; V, Hinnebusch AG: <br /> 18.   Kressler D, la Cruz J, Rojo M, Linder P: Fal1p essential                Related eIF3 subunits TIF32 HCR1 interact RNA <br />       DEAD-box protein involved 40S-ribosomal-subunit bio-                          recognition motif PRT1 required eIF3 integrity and <br />       genesis Saccharomyces cerevisiae.              Mol Cell Biol 1997,            ribosome binding. EMBO J 2001, 20:891-904. <br />       17(12):7283-7294.                                                          40.   Saccharomyces Genome Database                   [http://www.yeastge <br /> 19.   Frigieri MC, Jo&#195;&#163;o Luiz MVS, Apponi LH, Zanelli CF, Valentini SR: Syn-            nome.org/] <br />       thetic lethality eIF5A Ypt1 reveals connec-                  41.   Galv&#195;&#161;n IJ, Mir-Rashed N, Jessulat M, Atanya M, Golshani A, Durst T, <br />       tion translation secretory pathway yeast.                     Petit P, Amiquet VT, Boekhout T, Summerbell R, Cruz I, Arnason JT, <br />       Mol Genet Genomics 2008, 280:211-221.                                            Smith ML: Antifungal antioxidant activities phyto- <br /> 20.   Memarian N, Jessulat M, Alirezaie J, Mir-Rashed N, Xu J, Zareie M,               medicine pipsissewa, Chimaphila umbellata. Phytochemistry <br />       Smith M, Golshani A: Colony size measurement yeast                        2008, 69(3):738-746. <br />       gene deletion strains functional genomics. BMC Bioinfor-               42.   Anderson JB, Sirjusingh C, Parsons AB, Boone C, Wickens C, Cowen <br />       matics 2007, 8:117.                                                              LE, Kohn LM: Mode selection experimental evolution of <br /> 21.   Nunes PA, Tenreiro S, S&#195;&#161;-Correia I: Resistance adaptation                 antifungal drug resistance Saccharomyces cerevisiae. Genet- <br />       quinidine Saccharomyces cerevisiae: role QDR1                              ics 2003, 163(4):1287-1298. <br />       (YIL120w), encoding plasma membrane transporter                   43.   Lucchini G, Hinnebusch AG, Chen C, Fink GR: Positive regulatory <br />       major facilitator superfamily required multidrug resist-                     interactions HIS4 gene Saccharomyces cerevisiae. <br />       ance. Antimicrob Agents Chemother 2001, 45(5):1528-1534. <br /> </body></html>