<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br />     Cwc23, an Essential J Protein Critical for Pre-mRNA Splicing with <br />                         a Dispensable J Domain <br />   <br />  <br />              J proteins are structurally diverse, obligatory cochaperones of Hsp70s, each with a highly conserved J <br />           domain that plays a critical role in the stimulation of Hsp70&#226;&#8364;&#8482;s ATPase activity. The essential protein, Cwc23, <br />           is one of 13 J proteins found in the cytosol and/or nucleus of Saccharomyces cerevisiae. We report that a partial <br />           loss-of-function CWC23 mutant has severe, global defects in pre-mRNA splicing. This <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span>mutation</span> leads <span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span> <br />           accumulation of the excised, lariat form of the intron, as well as unspliced pre-mRNA, suggesting a role for <br />           Cwc23 in spliceosome disassembly. Such a role is further supported by the observation that this mutation <br />           results in reduced interaction between Cwc23 and Ntr1 (SPP382), a known component of the disassembly <br />           pathway. However, Cwc23 is a very atypical J protein. Its J domain, although functional, is dispensable for both <br />           cell viability and pre-mRNA splicing. Nevertheless, strong genetic interactions were uncovered between point <br />           mutations encoding alterations in Cwc23&#226;&#8364;&#8482;s J domain and either Ntr1 or Prp43, a DExD/H-box helicase <br />           essential for spliceosome disassembly. These genetic interactions suggest that Hsp70-based chaperone ma- <br />           chinery does play a role in the disassembly process. Cwc23 provides a unique example of a J protein; its <br />           partnership with Hsp70 plays an auxiliary, rather than a central, role in its essential cellular function. <br />  <br />  <br />    Hsp70-based machineries constitute a key component of the                  other 12 J proteins (31), indicating that Cwc23 carries out a <br /> cell&#226;&#8364;&#8482;s chaperone network, playing a central role in many pro-                 specialized cellular function. Cwc23 has been linked to pre- <br /> cesses, including de novo protein folding, protein translocation              mRNA splicing via both genome-wide and spliceosomal com- <br /> across membranes, and remodeling of protein complexes (6,                     ponent-directed physical interaction studies (7, 25). <br /> 13). By their ability to bind to short, exposed, hydrophobic                     Pre-mRNA splicing is a highly precise and stepwise process <br /> stretches of polypeptide, Hsp70s serve as the core of this pro-               whereby noncoding introns are removed <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> nascent tran- <br /> tein folding machinery. However, Hsp70s cannot function                       scripts. The splicing reaction is catalyzed by the spliceosome, <br /> alone. J proteins (often referred to as Hsp40s) are their oblig-              a large ribonucleoprotein (RNP) complex composed of five <br /> atory partners, serving to stimulate Hsp70&#226;&#8364;&#8482;s ATPase activity                  small nuclear RNAs and at least 100 different proteins (5, 24, <br /> and thereby stabilizing interaction with client proteins (19). J              38). The widely held model of the splicing pathway posits that <br /> proteins are very diverse but, by <span id='am-2' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span>, contain an &#226;&#172;&#402;70-                 the spliceosome is not a monolithic, stable complex but rather <br /> amino-acid &#226;&#8364;&#339;J domain&#226;&#8364;&#65533; that interacts directly with the Hsp70                  a series of complexes that must be assembled de novo for each <br /> ATPase domain (12, 17). It is well established that the J do-                 splicing event and subsequently disassembled upon completion. <br /> main is critical for function of the Hsp70-based chaperone                    Both the assembly and the disassembly steps of pre-mRNA <br /> machinery, since single amino acid alterations in its highly                  splicing require a series of macromolecular rearrangements <br /> conserved HPD motif disrupt function, both in vitro and in                    involving changes in RNA-RNA, RNA-protein, and protein- <br /> vivo, without affecting domain structure (9, 14). The Saccha- <br />                                                                               protein interactions. A positive &#226;&#8364;&#339;two-hybrid&#226;&#8364;&#65533; interaction be- <br /> romyces cerevisiae genome encodes 22 J proteins. Of these, 13 <br />                                                                               tween Cwc23 and Ntr1, an essential protein required for effi- <br /> are found in the cytosol and/or nucleus. One, Cwc23, is the <br />                                                                               cient spliceosomal disassembly, was recently reported (26). As <br /> focus of this report. Deletion of CWC23 causes inviability in <br />                                                                               part of the &#226;&#8364;&#339;NTR complex,&#226;&#8364;&#65533; Ntr1 recruits the DExD/H-box <br /> some strain backgrounds, although adaptation or suppression <br />                                                                               ATPase Prp43, which is critical for disassembly, to the spliceo- <br /> allows slow growth in others (31, 34). The growth defect of <br />                                                                               some (4, 35). The interaction surface between Prp43 and Ntr1 <br /> cwc23&#226;&#338;&#172; cells cannot be rescued by overexpression of any of the <br />                                                                               involves a region of Ntr1 termed the G patch, a motif found <br />                                                                               in many proteins and known to be involved in both protein- <br />    * Corresponding author. Mailing address: Department of Biochem-            protein and protein-RNA interactions (1). Given the links <br /> istry, 433 Babcock Dr., University of Wisconsin&#226;&#8364;&#8221;Madison, Madison,             between Cwc23 and RNA splicing, we further investigated <br /> WI 53706. Phone: (608) 263-7105. Fax: (608) 262-3453. E-mail: ecraig <br />                                                                               whether Cwc23 plays a role in this process. Indeed, a partial <br /> @wisc.edu. <br />    &#226;&#8364;&#160; Supplemental material for this article may be found at http://mcb        loss-of-function mutation in CWC23 causes profound global <br /> .asm.org/.                                                                    effects on splicing, demonstrating a role of Cwc23 in this pro- <br />    &#226;&#8364;&#161; Present address: Department of Chemistry and Biochemistry, Uni-          cess. However, the J domain of Cwc23 is not normally required <br /> versity of Colorado at Boulder, Boulder, CO 80309-0215. <br />                                                                               for either cell viability or pre-mRNA splicing. However, when <br />    &#194;&#167; Present address: Biology Department, Ithaca College, Ithaca, NY <br /> 14850.                                                                        the interaction between Ntr1 and Prp43 is compromised, <br />    &#228;&#338;&#164; <br />      Published ahead of print on 12 October 2009.                             Cwc23&#226;&#8364;&#8482;s J domain becomes essential, indicating a role for it, <br />  <br />                                                                          33 <br />  34       SAHI ET AL.                                                                                                                             MOL. CELL. BIOL. <br />  <br />  <br />             TABLE 1. Yeast strains generated in this study                           CWC23 plasmid. Subsequently, the diploids were sporulated, and tetrads were <br />                                                                                      dissected to obtain the haploid strains. Y2241 and Y2242 strains were made to <br /> Strain                                    Description <br />                                                                                      study the genetic interaction of cwc23&#226;&#338;&#172;J with the essential SIS1 gene by allowing <br /> Y1818...........&#226;&#338;&#172;cwc23:KanMx &#229;&#8230;&#179;316-Cwc23&#229;&#8230;&#180;                                             repression of Sis1 synthesis by the addition of doxycycline. sis1&#226;&#338;&#172;:LEU2 [Tetr- <br /> Y2178...........&#226;&#338;&#172;cwc23:ADE2 &#229;&#8230;&#179;316-Cwc23&#229;&#8230;&#180;                                              Sis1] (3) was crossed with cwc23&#226;&#338;&#172;:KanMx expressing either wt Cwc23 or Cwc23&#226;&#338;&#172;J <br /> Y2236...........&#226;&#338;&#172;cwc23:KanMx &#229;&#8230;&#179;314-Cwc23&#226;&#338;&#172;J&#229;&#8230;&#180; &#226;&#338;&#172;apj1:HIS3                                from a HIS3 CEN-based plasmid (pRS313), followed by tetrad dissection to <br /> Y2237...........&#226;&#338;&#172;cwc23:KanMx &#229;&#8230;&#179;314-Cwc23&#226;&#338;&#172;J&#229;&#8230;&#180; &#226;&#338;&#172;ydj1:HIS3                                obtain the haploid strain. <br /> Y2238...........&#226;&#338;&#172;cwc23:KanMx &#229;&#8230;&#179;314-Cwc23&#226;&#338;&#172;J&#229;&#8230;&#180; &#226;&#338;&#172;jjj1:URA3                                    ntr1&#226;&#338;&#172; and prp43&#226;&#338;&#172; strains were obtained from Beate Schwer (33). Y2244, Y2245, <br /> Y2239...........&#226;&#338;&#172;cwc23:KanMx &#229;&#8230;&#179;314-Cwc23&#226;&#338;&#172;J&#229;&#8230;&#180; &#226;&#338;&#172;xdj1:LEU2                                and Y2246, used to assess synthetic growth defects with NTR1 mutants, were made <br /> Y2240...........&#226;&#338;&#172;cwc23:KanMx &#229;&#8230;&#179;314-Cwc23&#226;&#338;&#172;J&#229;&#8230;&#180; &#226;&#338;&#172;caj1:LEU2                                by crossing cwc23&#226;&#338;&#172; cells expressing wt Cwc23, Cwc23&#226;&#338;&#172;J, or Cwc23H50Q from a <br /> Y2241...........&#226;&#338;&#172;cwc23:KanMx &#229;&#8230;&#179;313-Cwc23&#229;&#8230;&#180; &#226;&#338;&#172;sis1:LEU2 &#229;&#8230;&#179;Tetr-Sis1&#229;&#8230;&#180;                      pRS314 plasmid (obtained as described above) with ntr1&#226;&#338;&#172; carrying wt NTR1 on <br /> Y2242...........&#226;&#338;&#172;cwc23:KanMx &#229;&#8230;&#179;313-Cwc23&#226;&#338;&#172;J&#229;&#8230;&#180; &#226;&#338;&#172;sis1:LEU2 &#229;&#8230;&#179;Tetr-Sis1&#229;&#8230;&#180;                    a URA3 CEN plasmid (p360-Ntr1). Haploid strains were obtained by tetrad <br /> Y2244...........&#226;&#338;&#172;cwc23:KanMx &#229;&#8230;&#179;314-Cwc23&#229;&#8230;&#180; &#226;&#338;&#172;ntr1:natR &#229;&#8230;&#179;p360-Ntr1&#229;&#8230;&#180;                      dissection. Genetic interactions between NTR1 and CWC23 mutants were inves- <br /> Y2245...........&#226;&#338;&#172;cwc23:KanMx &#229;&#8230;&#179;314-Cwc23 H50Q&#229;&#8230;&#180; &#226;&#338;&#172;ntr1:natR                             tigated by transforming pRS413-based plasmids (HIS3 CEN) with wt or G-patch <br />                   &#229;&#8230;&#179;p360-Ntr1&#229;&#8230;&#180;                                                        mutant NTR1 genes (encoding the alterations L68A, Y74A, or L80A, which were <br /> Y2246...........&#226;&#338;&#172;cwc23:KanMx &#229;&#8230;&#179;314-Cwc23&#226;&#338;&#172;J&#229;&#8230;&#180; &#226;&#338;&#172;ntr1:natR &#229;&#8230;&#179;p360-Ntr1&#229;&#8230;&#180;                    described previously [33]) into Y2244, Y2245, and Y2246. To investigate whether <br /> Y2251...........&#226;&#338;&#172;cwc23:ADE2 &#229;&#8230;&#179;316-Cwc23&#229;&#8230;&#180; &#226;&#338;&#172;prp43:KanMx &#229;&#8230;&#179;p358-Prp43&#229;&#8230;&#180;                    the expression of a J domain fragment could complement the synthetic growth <br /> Y2253...........&#226;&#338;&#172;cwc23:ADE2 &#229;&#8230;&#179;316-Cwc23&#229;&#8230;&#180; &#226;&#338;&#172;prp43:KanMx                                 defect between J domain mutants of CWC23 and G-patch NTR1 mutants in trans, <br />                   &#229;&#8230;&#179;p358-Prp43-Y402A&#229;&#8230;&#180;                                                 J domain-encoding fragments of CWC23 (Cwc231-225 [CEN ADH] or Cwc23-J <br /> Y2305...........&#226;&#338;&#172;cwc23:ADE2 &#229;&#8230;&#179;316-Cwc23&#229;&#8230;&#180; &#226;&#338;&#172;ntc20:KanMx                                 [2&#226;&#65533;&#174;m GPD]) or another cytosolic J protein gene JJJ1 (Jjj11-128 [2&#226;&#65533;&#174;m GPD]) were <br /> Y2306...........&#226;&#338;&#172;cwc23:ADE2 &#229;&#8230;&#179;316-Cwc23&#229;&#8230;&#180; &#226;&#338;&#172;isy1:KanMx                                  transformed into Y2244, Y2245, and Y2246. The Y2178 strain (cwc23&#226;&#338;&#172;:ADE2 <br /> Y2307...........&#226;&#338;&#172;cwc23:ADE2 &#229;&#8230;&#179;316-Cwc23&#229;&#8230;&#180; &#226;&#338;&#172;lea1:KanMx                                  [pRS316-Cwc23]), was constructed by swapping the KanMx cassette in the het- <br /> Y2308...........&#226;&#338;&#172;cwc23:ADE2 &#229;&#8230;&#179;316-Cwc23&#229;&#8230;&#180; &#226;&#338;&#172;msl1:KanMx                                  erozygous diploid cwc23&#226;&#338;&#172;:KanMx strain with the NotI-linearized ADE2 disrup- <br /> Y2309...........&#226;&#338;&#172;cwc23:ADE2 &#229;&#8230;&#179;316-Cwc23&#229;&#8230;&#180; &#226;&#338;&#172;mud1:KanMx                                  tion cassette plasmid (37). The resulting strain was transformed with the <br /> Y2310...........&#226;&#338;&#172;cwc23:ADE2 &#229;&#8230;&#179;316-Cwc23&#229;&#8230;&#180; &#226;&#338;&#172;mud2:KanMx                                  pRS316-Cwc23 plasmid and sporulated, and tetrads were dissected to yield <br />                                                                                      Y2178. To test a synthetic genetic interaction with PRP43, yeast strains Y2251 <br />                                                                                      and Y2253 were constructed by crossing Y2178 with prp43&#226;&#338;&#172; cells harboring <br />                                                                                      either p358-Prp43 or p358-Prp43Y402A (TRP CEN) plasmids (33). Haploids <br />                                                                                      having the deletion alleles of both CWC23 and PRP43 were obtained by tetrad <br /> and thus Hsp70-based chaperone machinery, in pre-mRNA                                dissection. Finally, Y2251 and Y2253 were transformed with pRS313 (HIS3 <br /> splicing.                                                                            CEN) plasmids expressing wt Cwc23, Cwc23&#226;&#338;&#172;J, or Cwc23H50Q. Transformants <br />                                                                                      were selected on dropout media, and synthetic genetic interactions between <br />                                                                                      CWC23 and PRP43 mutants were scored after plating cells on 5-FOA medium. <br />                         MATERIALS AND METHODS                                        The effect of expression of a J domain containing fragment in trans was inves- <br />    Construction of plasmids. A DNA fragment containing 758 bp upstream and           tigated by transforming Cwc231-225 (CEN ADH), Cwc23-J (2&#226;&#65533;&#174;m GPD), or <br /> 545 bp downstream of start and stop codon of CWC23 was PCR amplified from            Jjj11-128 (2&#226;&#65533;&#174;m GPD) into Y2251 and Y2253 expressing Cwc23&#226;&#338;&#172;J. <br /> genomic DNA and cloned in pRS316 (URA3 CEN), pRS314 (TRP CEN) and                        Strains Y2305, Y2306, Y2307, Y2308, Y2309, and Y2310 (see Table 1) were <br /> pRS313 (HIS3 CEN) vectors as a BamHI/SacII fragment. To obtain approxi-              made to study the genetic interaction between mutations in CWC23 and other <br /> mately normal expression levels of Cwc231-225, the corresponding DNA fragment        spliceosomal genes. Homozygous diploid strains from the yeast <span id='am-162' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-163' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-239' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span>knockout</span> collec- <br /> was cloned under the control of the alcohol dehydrogenase (ADH) promoter in          tion (40) for ntc20, isy1, lea1, msl1, mud1, or mud2 were sporulated. Haploids <br /> the CEN-based vectors pRS413, pRS414, and pRS415 (23). All mutations, trun-          obtained by tetrad dissection were mated with Y2178. Double deletions of <br /> cations, and internal deletions were made by quick change or sewing PCR using        CWC23 and one of the six genes, obtained by tetrad dissection, were transformed <br /> appropriate primer pairs and standard methods. For overexpression, the coding        with pRS314 or pRS314 harboring wt Cwc23 or Cwc23&#226;&#338;&#172;J. <br /> sequence for full-<span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span> Cwc23 was PCR amplified and cloned into the SpeI/               Protein expression and purification. His-tagged Ssa1 and Ssb1 were purified <br /> XhoI sites of the 2&#226;&#65533;&#174;m plasmid, pRS424, which contains the GPD promoter (23).         as described previously (15, 28). Cwc231-225-His6 was expressed from pET3a <br /> The Cwc231-225 coding sequence was also cloned into the same sites in CEN            plasmid in C41(DE3) cells (22). Cells were induced with 0.2 mM IPTG (isopro- <br /> TEF- and CEN GPD-based vectors (23). Jjj1-J domain (amino acids 1 to 128)            pyl-&#226;&#65533;&#164;-D-thiogalactopyranoside) at 20&#194;&#176;C for 8 h. The cell pellet was resuspended <br /> overexpression was carried out essentially as described previously (31), except      and lysed in the Ni-NTA binding buffer (25 mM HEPES [pH 7.5], 300 mM KCl, <br /> the coding fragment was subcloned from pRS424 into the analogous vector              25 mM imidazole, 5% glycerol) supplemented with protease inhibitor cocktail <br /> having a LEU2 marker, pRS425. By PCR sewing using overlapping primers,               (Roche). The supernatant was loaded onto a Ni-NTA agarose column (Nova- <br /> DNA encoding the J domain of Cwc23 was fused to a random stretch of 32               gen), washed with several volumes of Ni-NTA buffer, and eluted with 300 mM <br /> codons that preceded a 3&#226;&#171;&#187;HA tag and then cloned into the SpeI/XhoI sites of          imidazole in Ni-NTA binding buffer. Eluted protein was further purified by using <br /> pRS425. For the two-hybrid experiment, the complete Ntr1 coding sequence was         HiPrep 16/60 Sephacryl S-200 HR (GE Healthcare) equilibrated with the Gel <br /> PCR amplified and cloned in the GAL4 binding domain plasmid, pGBDU-C1,               filtration buffer (25 mM HEPES [pH 7.5], 100 mM KCl, 10 mM magnesium <br /> whereas CWC23 fragments were cloned into the BamHI/EcoRI sites of a GAL4             acetate, 5% glycerol). The purified protein was concentrated by using a Cen- <br /> activation domain plasmid pGAD-C1 (16). Constructs for bacterial expression          triprep YM-10 (Millipore), divided into aliquots, snap-frozen in liquid nitrogen, <br /> were made by PCR amplifying CWC23 fragments with appropriate primers such            and stored at &#226;&#171;&#186;80&#194;&#176;C. For antibody production, pET3a-Cwc2381-171-His6 was <br /> that all contained His6 tags at their C termini. The fragments were cloned into      used. Eluted protein was dialyzed overnight against 10% glycerol at 4&#194;&#176;C. Poly- <br /> the NdeI/BamHI sites in pET3a (Novagen).                                             clonal antibodies to this protein were commercially generated at Harlan (Mad- <br />    Genetic methods. Most yeast strains used have the W303 genetic background         ison, WI). <br /> (10). Those generated here are listed in Table 1. The starting strain for many of        ATPase assay. Single-turnover ATPase assays of Ssa1 and Ssb1 were carried <br /> these is Y1818 cwc23&#226;&#338;&#172;:KanMx [pRS316-Cwc23]. Plasmids carrying wild-type (wt)         out as described previously (15, 21). Isolated Hsp70-[&#226;&#65533;&#163;-32P]ATP complexes were <br /> or mutant CWC23 alleles were transformed into Y1818, and transformants were          incubated with equimolar amounts of Cwc231-225 or Cwc231-225/H50Q at 25&#194;&#176;C, <br /> selected on appropriate dropout media. The functionality of different CWC23          and the samples were removed for detection of ATP and ADP. The fractions of <br /> alleles was tested by checking the ability of the cells to lose the URA3-based       ATP hydrolyzed to ADP over the time courses were quantified by <br /> pRS316-Cwc23 plasmid by spotting on 5-fluoroorotic acid (5-FOA) plates and           PhosphorImager analysis (Molecular Dynamics) and fit to one-phase exponential <br /> incubation at 30&#194;&#176;C for 3 days.                                                       decay by using Prism 4.0 (GraphPad Software). <br />    Yeast strains Y2236, Y2237, Y2238, Y2239, and Y2240 were constructed to               Splicing assays. To examine pre-mRNA splicing defects in the cwc23&#226;&#338;&#172; strain <br /> investigate the synthetic genetic interaction between cwc23&#226;&#338;&#172;J and deletions of       by Northern blot analysis, yeast cells from wt and cwc23&#226;&#338;&#172; microcolonies were <br /> genes encoding nonessential cytosolic/nuclear J proteins (Apj1, Ydj1, Jjj1, Xdj1,    subcultured in rich media at 30&#194;&#176;C for 2 days. Total cellular RNA was isolated by <br /> and Caj1) by mating single J protein deletion strains as described previously (31)   using acid phenol (32). Equivalent cells (optical density at 600 nm &#226;&#171;&#189; 1.0) were <br /> with a cwc23&#226;&#338;&#172;:KanMx strain expressing Cwc23&#226;&#338;&#172;J from pRS314 plasmid (con-              spun down at 2,000 &#226;&#171;&#187; g for 5 min at room temperature in a 2-ml microcentrifuge <br /> structed as described in the previous paragraph), after losing the URA-based wt      tube. The cells were washed once with chilled distilled water. The cells were <br />  VOL. 30, 2010                                                          Cwc23, ESSENTIAL J PROTEIN WITH NONESSENTIAL J DOMAIN                          35 <br />  <br /> resuspended in 400 &#226;&#65533;&#174;l of TES buffer (10 mM Tris [pH 7.5], 10 mM EDTA, 0.5%                                       RESULTS <br /> sodium dodecyl sulfate [SDS]) and 400 &#226;&#65533;&#174;l of acidic phenol-chloroform-isoamyl <br /> alcohol (25:24:1 [pH 4.2]). The cells were lysed by adding 400-&#226;&#65533;&#174;l equivalents of        Cwc23&#226;&#8364;&#8482;s C terminus, but not its N-terminal J domain, is <br /> 0.5-mm glass beads, followed by incubation at 65&#194;&#176;C for 1 h with intermittent         essential for cell viability. The N-terminal 80-amino-acid seg- <br /> vortexing. The tubes were incubated on ice for 5 min and then centrifuged at <br />                                                                                      ment of S. cerevisiae Cwc23 (Fig. 1A), as well as orthologs <br /> 16,000 &#226;&#171;&#187; g at 4&#194;&#176;C. The upper aqueous phase containing the total RNA was <br /> precipitated by addition of 1/10 volume of sodium acetate (pH 5.2) and 2.5 <br />                                                                                      identified in other fungal species and in higher eukaryotes (see <br /> volumes of chilled ethanol. Total RNA was spun down for 10 min at 16,000 &#226;&#171;&#187; g         Fig. S1 in the supplemental material), are highly similar in <br /> at 4&#194;&#176;C. Finally, the RNA pellet was washed with 70% ethanol, air dried, and          sequence to known J domains. To determine whether the J <br /> dissolved in 50 &#226;&#65533;&#174;l of RNase-free distilled water. For use with a full-length actin   domain of Cwc23 is essential for its in vivo function, a mutation <br /> probe, 5 &#226;&#65533;&#174;g of total RNA was resolved on a denaturing morpholinepropanesul- <br />                                                                                      encoding a histidine-to-glutamic-acid substitution in the invari- <br /> fonic acid-formaldehyde 1% agarose gel and transferred by capillary to a nylon <br /> membrane (GE Healthcare). For use with actin intron probe, 20 &#226;&#65533;&#174;g of RNA was          ant HPD motif was made. This particular mutation was se- <br /> resolved on a denaturing morpholinepropanesulfonic acid-formaldehyde 1.5%            lected because it has been demonstrated to severely affect the <br /> agarose gel. Radiolabeled PCR fragment corresponding to the full-length or           ability of several J proteins to stimulate the ATPase activity of <br /> ACT1 intron DNA was used as a probe. Radiolabeling was done by using the             their partner Hsp70 and consequently their in vivo function <br /> Prime-A gene labeling system (Promega, Madison, WI). Hybridization was car- <br />                                                                                      (9). Surprisingly, not only was the cwc23H50Q strain viable (Fig. <br /> ried out in ULTRAhyb solution (Ambion) at 42&#194;&#176;C, overnight. After the wash- <br /> ings, the bands were visualized by using a PhosphorImager.                           1B, panels 1 and 2), it grew indistinguishably from wild-type <br />    To monitor pre-mRNA splicing defects on a global scale, splicing-sensitive        (wt) (Fig. 1B, panels 3 to 5). To address the concern that the <br /> whole genome microarrays were used. Cultures were grown as previously de-            H50Q mutation might not completely abolish J domain func- <br /> scribed (29). Total RNA was isolated as described previously, but for each           tion, we constructed cwc23&#226;&#338;&#172;J, with a deletion of the 80 codons <br /> microarray cDNA was prepared from 20 &#226;&#65533;&#174;g of total RNA in a 50-&#226;&#65533;&#174;l reaction <br /> mixture containing 50 mM Tris-HCl (pH 8.3), 75 mM KCl, 3 mM MgCl2, 10 mM <br />                                                                                      of the N-terminal J domain. Remarkably, cwc23&#226;&#338;&#172;J cells also <br /> dithiothreitol, 0.5 mM ATP, 0.5 mM CTP, 0.5 mM GTP, 0.5 mM TTP, 0.01 mM              grew indistinguishably from wt cells (Fig. 1B), establishing that <br /> 5-(3-aminoallyl)-dUTP, 12.5 &#226;&#65533;&#174;g of dN9 primer, and 5 ng of M-MLV RT. Labeled          the J domain region is not required for Cwc23&#226;&#8364;&#8482;s essential func- <br /> cDNA samples were hybridized on custom designed Agilent 8x15K microarrays.           tion. <br /> These microarrays contain probes targeting over 6,000 yeast genes. For intron- <br />                                                                                         The dispensability of the J domain region of Cwc23 raises <br /> containing genes, the microarrays also contain probes that target regions of the <br /> introns, as well as the junction between neighboring exons (29). Microarray <br />                                                                                      the question of whether it is actually a functional J domain. To <br /> design details can be found at NCBI&#226;&#8364;&#8482;s Gene Expression Omnibus (GEO; http:            address this question, we carried out two experiments. First, we <br /> //www.ncbi.nlm.nih.gov/geo/) under accession number GPL8154. Samples were            made use of our previous observation that Cwc23, when ex- <br /> hybridized according to manufacturer&#226;&#8364;&#8482;s protocols at 60&#194;&#176;C for 17 h. Microarrays       pressed from the strong GPD promoter, substantially rescued <br /> were washed for one minute in 6&#226;&#171;&#187; SSPE (0.9 M NaCl, 60 mM NaH2PO4, 6 mM <br />                                                                                      the severe growth defects of cells lacking Ydj1, the most abun- <br /> EDTA), 0.005&#226;&#171;&#187; Sarkosyl, and then briefly washed in 0.06&#226;&#171;&#187; SSPE&#226;&#8364;&#8220;0.005&#226;&#171;&#187; Sar- <br /> kosyl prior to scanning. Images were obtained by using an Axon 4000B scanner,        dant cytosolic/nuclear J protein (31). However, Cwc23H50Q was <br /> and data were extracted from the images by using GenePix 6.0 software. Raw           completely ineffective in rescuing the growth defect of ydj1&#226;&#338;&#172; <br /> data were processed by using Bioconductor to implement the Loess normaliza-          cells (Fig. 1C), even though it was expressed at levels similar to <br /> tion function. After normalization, replicate values were averaged to determine      that of wt Cwc23 (Fig. 1C, right panel). Second, we tested the <br /> the behavior of each feature on the array. Each experiment was performed as a <br /> dye-flipped replicate, with the composite behavior being presented in the figures <br />                                                                                      effect of the H50Q alteration on the ability of Cwc23 to stim- <br /> and text. Both raw and processed data are available from GEO.                        ulate the ATPase activity of Hsp70. Since this was the first time <br />    Other methods. Total protein for immunoblot analysis was isolated by treating     Cwc23 had been purified, we first tested whether Cwc23 was <br /> cells with 0.1 N NaOH for 7 min, followed by resuspension in 1&#226;&#171;&#187; SDS sample           competent to stimulate the ATPase activity of either of the two <br /> buffer (62.5 mM Tris-HCl [pH 6.8], 5% glycerol, 2% SDS, 2% &#226;&#65533;&#164;-mercaptoeth-            classes of cytosolic/nuclear Hsp70s, Ssa and Ssb. Ssa1 and Ssb1 <br /> anol, 0.01% bromophenol blue) and incubation for 5 min in a boiling water bath. <br /> Anti-Cwc23 polyclonal antibody was affinity purified using Cwc2381-171 as de- <br />                                                                                      were loaded with [32P]ATP, and the ability of a 225-amino-acid <br /> scribed previously (20). Then, 5 mg of affinity-purified Cwc2381-171 was resolved    fragment of Cwc23 to stimulate the basal ATPase activities was <br /> by SDS&#226;&#8364;&#8220;12.5% polyacrylamide gel electrophoresis (PAGE). Protein was trans-           monitored. Although little effect on Ssb1&#226;&#8364;&#8482;s ATPase activity was <br /> ferred to a nitrocellulose membrane by electroblotting. The membrane was             observed (Fig. 1D, right), the ATPase activity of Ssa1 was stim- <br /> stained with Ponceau S for 5 min and destained with 5% acetic acid. The section <br />                                                                                      ulated &#226;&#172;&#402;7-fold when present in a stoichiometric amount (Fig. 1D, <br /> of the membrane containing the protein band was excised, washed three times <br /> (for 5 min each time) with TBS buffer (10 mM Tris-HCl [pH 7.5], 0.9% NaCl),          left). This level of stimulation was similar to that of Ydj1, an <br /> and then incubated with 10 ml of anti-Cwc23 polyclonal antibody for 1 h at room      established J protein partner of Ssa (15). The H50Q alteration <br /> temperature. The membrane was then washed four times, for 5 min each time,           virtually eliminated this stimulatory ability (Fig. 1D, left). These <br /> with TBS buffer and then briefly in distilled water. Purified antibody was eluted    results support the idea that Cwc23 is a functional J protein, <br /> with 3 ml of cold 100 mM glycine-HCl (pH 2.5) for 10 min. Eluted antibody was <br />                                                                                      which, like Ydj1 and many other cytosolic J proteins, can partner <br /> transferred to a fresh tube and neutralized with 1 M Tris-HCl (pH 8.5). Cwc23 <br /> expressed from either its native promoter or an ADH promoter on a CEN-based          with Ssa. Nevertheless, inactivation of the J domain does not <br /> plasmid was detected by the more sensitive ECL Advance Western detection kit         result in a loss of Cwc23&#226;&#8364;&#8482;s essential cellular function. <br /> (GE Healthcare). Otherwise, a Western Lightening Plus ECL kit (Perkin-Elmer)            Truncation of Cwc23&#226;&#8364;&#8482;s C terminus, but not its J domain, <br /> was used.                                                                            causes global defects in pre-mRNA splicing. Since Cwc23 was <br />    The fungal orthologs of Cwc23 were identified at http://www.broad.mit.edu <br /> /regev/orthogroups/html/6/6/OG3966.html. Higher eukaryotic homologs were <br />                                                                                      found physically associated with the spliceosome, we sought to <br /> identified by using amino acids 58 to 289 of Cwf23 (Cwc23 ortholog in S. pombe),     determine whether Cwc23 is indeed involved in pre-mRNA <br /> which lacks the J domain, as a query sequence in BLASTp at the National Center       splicing. Earlier, we reported CWC23 to be an essential gene, <br /> for Biotechnology Information. The amino acids corresponding to the J domains        since viable colonies could not be obtained using the standard <br /> of selected Cwc23 orthologs were aligned by using the CLUSTAL W mode of              selection on 5-FOA containing medium for mutants whose <br /> MegAlign module of DNASTAR (Lasergene 8). RNA recognition motif in <br /> Cwc23 orthologs was identified by searching the respective protein sequences at <br />                                                                                      viability is maintained by a wt gene in a URA3-based plasmid <br /> Conserved Domain database (http://www.ncbi.nlm.nih.gov/Structure/cdd/cdd             (31) (Fig. 1B). However, as previously reported (34), dissection <br /> .shtml) and Pfam (http://pfam.sanger.ac.uk/search).                                  of a cwc23&#226;&#338;&#172; heterozygous diploid strain occasionally yielded <br />  36     SAHI ET AL.                                                                                                              MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />   FIG. 1. The J domain of Cwc23 is dispensable. (A) Cwc23 has a conserved J domain. A sequence alignment of the predicted J domains of the <br /> indicated J proteins of S. cerevisiae is shown. The conserved histidine-proline-aspartic acid (HPD) tripeptide motif is marked (***). (B) The J <br /> domain is not critical for growth. In panels 1 and 2, equal numbers of cwc23&#226;&#338;&#172; cells harboring two plasmids&#226;&#8364;&#8221;URA3 CEN-Cwc23 and a vector <br /> containing no insert (&#226;&#171;&#186;), CWC23 (WT), cwc23H50Q (H50Q), or cwc23&#226;&#338;&#172;J (&#226;&#338;&#172;J)&#226;&#8364;&#8221;were spotted on medium without (&#226;&#171;&#186;) or with (&#226;&#171;&#185;) 5-FOA to select <br /> for cells having lost the URA3 CEN-Cwc23 plasmid, as indicated. In panels 3 to 5, 10-fold serial dilutions of equivalent numbers of cwc23&#226;&#338;&#172; cells <br /> expressing either wt Cwc23 (WT), Cwc23H50Q (H50Q), or Cwc23&#226;&#338;&#172;J (&#226;&#338;&#172;J) were spotted on selective medium and incubated at the indicated <br /> temperatures for 3 days. (C) Cwc23&#226;&#8364;&#8482;s J domain is functional. For the left panel, 10-fold serial dilutions of an equivalent number of ydj1&#226;&#338;&#172; cells <br /> expressing either wt Ydj1 from its own promoter (Ydj1), or Cwc23 (WT), or Cwc23H50Q (H50Q) from the GPD promoter or harboring only an <br /> empty vector (&#226;&#171;&#186;) were spotted on selective medium and incubated at 30&#194;&#176;C for 3 days. For the right panel, total protein was isolated from strains <br /> indicated in the left panel and subjected to SDS-PAGE, electroblotted, and probed with antibodies specific for Cwc23 or, as a loading control, Ssc1 <br /> (row C). (D) Cwc23 stimulates the ATPase activity of the Hsp70, Ssa1. Affinity-purified Cwc231-225-His6 (WT) or the J domain mutant (H50Q) <br /> was incubated with equimolar amounts of Ssa1-[&#226;&#65533;&#163;-32P]ATP (left panel) or Ssb1-[&#226;&#65533;&#163;-32P]ATP (right panel). Reactions were stopped after the <br /> indicated times and subjected to chromatography. ATP hydrolysis was quantified, and the data were fit to a one-phase exponential decay equation. <br />  <br />  <br />  <br /> cwc23&#226;&#338;&#172; microcolonies (Fig. 2A, left panel). Total RNA from                  nearly normal levels of the mutant protein, supporting our <br /> primary cultures of the cwc23&#226;&#338;&#172; microcolonies and from wt                    conclusion that Cwc231-225 is not simply a hypomorphic allele <br /> cells was used for hybridization analysis using radiolabeled                but rather is functionally defective (Fig. 2C). <br /> full-length actin (ACT1) genomic DNA probe. Consistent with                    To determine whether pre-mRNA splicing was globally af- <br /> a role for Cwc23 in splicing of actin pre-mRNA, two bands                   fected when Cwc23 function was compromised, whole-genome <br /> were observed in the cwc23&#226;&#338;&#172; sample: one at the position of the              splicing microarrays were used. For each intron-containing gene a <br /> mature mRNA found in the wt sample and the other at the                     minimum of three primers, targeting three regions of each <br /> position expected for the larger unspliced pre-mRNA species                 pre-mRNA, were present on the microarray: the intron to <br /> (Fig. 2A, right panel).                                                     probe pre-mRNA levels, the exon-exon junction to probe ma- <br />   Because the haploid cwc23&#226;&#338;&#172; strains commonly accumulated                   ture mRNA levels, and an exonic region to assess changes in <br /> suppressing mutations that could complicate analysis of Cwc23               total mRNA levels. By simultaneously considering the pre- <br /> function, we chose to continue our investigation by searching               mRNA, mature mRNA, and total RNA levels in mutant rela- <br /> for partial loss-of-function mutants of CWC23 by truncating                 tive to wt cells, differences in splicing efficiency can be assessed. <br /> the 3&#226;&#172;&#732; end of the gene. Although cwc231-200 was inviable (data              As seen in Fig. 3A, significant levels of unspliced pre-mRNA <br /> not shown), cwc231-225, which encodes a protein lacking the                 were detected for most intron-containing genes in cells ex- <br /> C-terminal 58 amino acids, was viable, growing more poorly                  pressing cwc231-225 whether grown at the permissive tempera- <br /> than wt cells between 23 and 37&#194;&#176;C but not forming colonies at               ture of 30&#194;&#176;C or after a shift to the nonpermissive temperature <br /> 18&#194;&#176;C (Fig. 2B). We noted that cells expressing 5- to 10-fold                of 16&#194;&#176;C for 30 min. When we consider both the number of <br /> higher levels of Cwc231-225 grew no better than cells expressing            transcripts affected and the magnitude of pre-mRNA accumu- <br />  VOL. 30, 2010                                                 Cwc23, ESSENTIAL J PROTEIN WITH NONESSENTIAL J DOMAIN                              37 <br />  <br />  <br />  <br />  <br />    FIG. 2. Cwc23 plays a role in pre-mRNA splicing. (A) Cwc23 is <br /> critical for the splicing of actin pre-mRNA. For the left panel, spores <br /> from individual asci of cwc23&#226;&#338;&#172;:KanMx heterozygous diploid cells were <br /> separated on solid rich media and then incubated at 30&#194;&#176;C for 5 days. <br /> Two sample tetrads are shown. The presence (&#226;&#171;&#185;) or absence (&#226;&#171;&#186;) of wt <br /> CWC23 is indicated. For the right panel, equal amounts of total RNA <br /> from wt and cwc23&#226;&#338;&#172; cells were resolved in a denaturing agarose gel, <br /> transferred to nylon membrane, and probed with radiolabeled DNA <br /> encompassing the full-length actin gene (ACT1). Bands were visual- <br /> ized utilizing a PhosphorImager (top panel). As a loading control, the <br /> membrane was stained with methylene blue to visualize rRNA (bottom <br /> panel). (B) The C-terminal region of Cwc23 is functionally important. <br /> Tenfold serial dilutions of equal numbers of cwc23&#226;&#338;&#172; cells expressing <br /> either Cwc23 (WT) or Cwc231-225 (225) were spotted on minimal <br /> medium plates and incubated at the indicated temperatures. (C) Over- <br /> expression of Cwc231-225 does not significantly affect the growth phe- <br /> notype. For the left panel, equivalent numbers of cwc23&#226;&#338;&#172; cells express- <br /> ing Cwc23 from its own promoter (WT-Cwc23) or Cwc231-225 from the            FIG. 3. cwc231-225 is defective in pre-mRNA splicing, but cwc23&#226;&#338;&#172;J is <br /> ADH, TEF, or GPD promoter were spotted on solid minimal medium            not. (A) Global analyses of pre-mRNA splicing. For each of 301 intron- <br /> and incubated for 3 days. For the right panel, total protein extracted    containing genes, the behaviors of the total mRNA, pre-mRNA, and <br /> from the samples on the left was subjected to SDS-PAGE and elec-          mature mRNA were determined (indicated as T, P, and M, respectively, <br /> troblotted. Membranes were probed with anti-Cwc23 antibodies and,         above each lane; see the diagram at top). Each horizontal line describes <br /> as a loading control, Ssc1 (row C).                                       the behavior of a single intron-containing gene. Ratio values were calcu- <br />                                                                           lated for each of the strain comparisons. The data are presented as <br />                                                                           false-colored representations of the log2 value for each feature. For the <br />                                                                           cwc231-225 experiments, samples were first collected from both mutant <br /> lation, the splicing defects seen in the cwc231-225 mutant are            and wt strains while growing at 30&#194;&#176;C. An additional sample was collected <br /> similar to those seen in a strain containing a temperature-sensitive      after both strains had been shifted to 16&#194;&#176;C for 30 min. For the prp43Y402A <br />                                                                           experiment, both mutant and wt strains were grown at 30&#194;&#176;C then shifted <br /> allele of the essential spliceosomal protein Prp43, prp43-Y402A           to 37&#194;&#176;C for 30 min prior to sample collection. For the Cwc23&#226;&#338;&#172;J experi- <br /> (33) (Fig. 3A). Although no determination can be made regarding           ment, samples were collected from both mutant and wt strains growing at <br /> the mechanistic role of a protein based solely on the identity of         30&#194;&#176;C. (B) cwc231-225 accumulates actin lariat-introns. Equal amounts of <br /> the pre-mRNAs whose splicing is impacted in an experiment such            total RNA isolated from cwc23&#226;&#338;&#172; cells expressing wt Cwc23 (WT), <br />                                                                           Cwc23&#226;&#338;&#172;J (&#226;&#338;&#172;J), Cwc231-225 (1-225) were resolved on denaturing agarose <br /> as this (29), these data nevertheless make it clear that Cwc23 plays      gels, transferred to nylon membrane, and probed with either full-length <br /> a prominent, global role in pre-mRNA splicing based on both the           (left) or intronic (right) actin DNA. Bands were visualized utilizing a <br /> widespread defect and the level of pre-mRNA accumulation seen             PhosphorImager (top panel). p, precursor; m, mature; i, intron lariat. As <br /> in the cwc231-225 strain.                                                 a loading control, the membrane was stained with ethidium bromide to <br />    Cwc23 interacts with Ntr1, an essential spliceosome disas-             visualize rRNA (bottom panel). <br /> sembly factor (4, 35), suggesting that it might also be important <br /> for the disassembly process. A hallmark of mutants in this                tial spliceosomal complexes, indirectly leading to a global de- <br /> pathway is that the excised lariat introns, which are normally            fect in pre-mRNA splicing. To directly test this idea, we <br /> rapidly degraded, become significantly stabilized (2). The ac-            probed RNA isolated from cwc231-225, cells with an intron- <br /> cumulation of these lariat-bound complexes sequesters essen-              specific probe, as well as the more standard probe encompass- <br />  38     SAHI ET AL.                                                                                                    MOL. CELL. BIOL. <br />  <br />  <br /> ing the entire actin gene. cwc231-225 cells had high levels of       Likewise, none of the Ssa or Ssb Hsp70 genes showed any <br /> both actin pre-mRNA and intron lariat compared to wt cells           significant change in expression. In sum, we obtained no evi- <br /> (Fig. 3B), indicating a critical involvement of Cwc23 in the         dence in either our genetic or our biochemical analyses that <br /> splicing disassembly pathway.                                        any other J domain is functionally substituting in trans when <br />    Having obtained evidence that Cwc23 plays a role in disas-        Cwc23&#226;&#8364;&#8482;s J domain is absent or nonfunctional. Rather, our data <br /> sembly of the spliceosome, we addressed whether the J domain         support the idea that under typical laboratory conditions the J <br /> was important for splicing. We observed virtually no pre-            domain of Cwc23, and thus Hsp70 machinery function, is dis- <br /> mRNA splicing defects in the cwc23&#226;&#338;&#172;J strain using microarray         pensable. <br /> analysis (Fig. 3A). In addition, the level of the intron lariat in      Cwc23&#226;&#8364;&#8482;s J domain is critical when interaction between <br /> cwc23&#226;&#338;&#172;J cells was indistinguishable from that present in wt cells    Cwc23 and Ntr1 is compromised. Since the Hsp70:J protein <br /> (Fig. 3B). Together, our results confirm that although the C         chaperone machinery is known to modulate protein-protein <br /> terminus of Cwc23 is required for both cell viability and effi-      interactions, we wanted to determine whether Cwc23&#226;&#8364;&#8482;s J do- <br /> cient pre-mRNA splicing, the J domain is dispensable for both.       main is important when the function of a protein with which <br />    No compensation by other J proteins in the absence of             Cwc23 interacts, Ntr1 (26), is compromised. First, we carried <br /> Cwc23&#226;&#8364;&#8482;s J domain. Since the J domain is critical for the func-       out a yeast two-hybrid experiment to exclude the possibility <br /> tion of all yeast J proteins tested thus far, we wanted to deter-    that the J domain of Cwc23 is important for this interaction. <br /> mine whether the lack of an observable effect upon deletion of       Full-length CWC23 and fragments encoding amino acids 1 to <br /> Cwc23&#226;&#8364;&#8482;s J domain was attributable to compensation, in trans,         83 (J domain), 1 to 225, 81 to 283 (Cwc23&#226;&#338;&#172;J), and 226 to 283 <br /> by other J proteins. Besides Cwc23, six other J proteins (Apj1,      (the C-terminal 58 amino acids) were cloned into a GAL4 <br /> Xdj1, Caj1, Sis1, Jjj1, and Ydj1) are known to be at least           activation domain plasmid, and the fusion proteins were coex- <br /> partially localized to the nucleus or involved in nuclear func-      pressed with a fusion of the GAL4 DNA-binding domain and <br /> tion (www.yeastgenome.org). Thus, we constructed a set of            Ntr1. Since the HIS3 gene was under the control of GAL4 in <br /> double mutants, each having cwc23&#226;&#338;&#172;J and a complete deletion          the tester strain, growth in the absence of histidine is depen- <br /> of one of the six J protein genes listed above. Four of the          dent on interaction between Cwc23 and Ntr1. As expected <br /> double mutant strains (the apj1&#226;&#338;&#172; cwc23&#226;&#338;&#172;, xdj1&#226;&#338;&#172; cwc23&#226;&#338;&#172;, caj1&#226;&#338;&#172;         from previous work (26), the full-length Cwc23 fusion allowed <br /> cwc23&#226;&#338;&#172;, and jjj1&#226;&#338;&#172; cwc23&#226;&#338;&#172; strains) expressing cwc23&#226;&#338;&#172;J grew            robust cell growth (Fig. 4A). Cells expressing the Cwc23&#226;&#338;&#172;J <br /> indistinguishably from the single J protein gene deletions. In       fusion grew as well as those expressing full-length protein, <br /> addition, no defects in the splicing of actin pre-mRNA were          while those expressing a fusion of the N-terminal 81 amino <br /> observed (data not shown). In the case of cells lacking Ydj1, a      acids did not, indicating that the J domain is not important for <br /> very subtle but reproducible synthetic growth defect was ob-         interaction with Ntr1. On the other hand, cells expressing the <br /> served at 18&#194;&#176;C (see Fig. S2A in the supplemental material).          Cwc231-225 fusion grew extremely poorly in the absence of <br /> However, accumulation of actin pre-mRNA was not detected             histidine (Fig. 4A), indicating the importance of the C-termi- <br /> at 23&#194;&#176;C or after a shift to 18, 37, or 42&#194;&#176;C (see Fig. S2B in the     nal 58 amino acids of Cwc23 for interaction with Ntr1. How- <br /> supplemental material). We conclude that the slight synthetic        ever, a fusion of the C-terminal 58 amino acids did not support <br /> growth defect at 18&#194;&#176;C is not related to pre-mRNA splicing.           growth. These results are consistent with the idea that the <br /> Rather, we think that Cwc23, at its normal low levels, is slightly   C-terminal 58 amino acids of Cwc23 is important, but not <br /> compensating for the absence of Ydj1, a finding consistent with      sufficient, for interaction with Ntr1, and that the J domain does <br /> the robust compensation observed when Cwc23 is expressed at          not play a role in the interaction. <br /> higher levels (Fig. 1C). Since Sis1 is essential, we exploited the      Since the absence of the J domain did not compromise the <br /> fact that levels of Sis1 can be expressed at ca. 10% of normal       interaction between Cwc23 and Ntr1, we proceeded to inves- <br /> levels by placing Sis1 under the control of the Tetr repressible     tigate whether mutations in NTR1 have synthetic genetic in- <br /> promoter to test for a genetic interaction between SIS1 and          teractions with CWC23 J domain mutations. We tested three <br /> cwc23&#226;&#338;&#172;J (3). Cells with lowered levels of Sis1 and either wt         NTR1 point mutations, L68A, Y74A, and L80A, each of which <br /> Cwc23 or Cwc23&#226;&#338;&#172;J grew indistinguishably. Likewise, no defect         alters the G patch, causing significant defects in interaction <br /> in the splicing of actin pre-mRNA was detected (data not             with Prp43 in vitro but no obvious defect in vivo (33, 35). We <br /> shown).                                                              used the plasmid shuffling technique, plating strains with dif- <br />    As a complementary approach, we queried our microarray            ferent combinations of NTR1 and CWC23 alleles, as well as a <br /> data to determine whether cytosolic HSP70s and other J pro-          wt NTR1 gene on a URA3-based plasmid. Cells were plated on <br /> tein genes might be upregulated in CWC23 mutant strains (8,          5-FOA-containing plates to select for those having lost the <br /> 9, 39). We plotted the geometric mean intensity versus the log       URA3-based plasmid and thus the wt NTR1 gene it contained. <br /> ratio value for each of the &#226;&#172;&#402;6,000 unique mRNA probes in-            As expected, the control strains expressing either the wt NTR1 <br /> cluded on our microarray analyses of cwc231-225 and cwc23&#226;&#338;&#172;J          or CWC23 genes grew well on the 5-FOA plates. However, we <br /> strains compared to a wt strain (see Fig. S2C in the supple-         were unable to recover yeast strains expressing any of the Ntr1 <br /> mental material). CWC23 mRNA was significantly increased in          variants in combination with Cwc23&#226;&#338;&#172;J (Fig. 4B). We reasoned <br /> cwc231-225, presumably because it is expressed from the ADH          that this lack of function could be the result of indirect effects <br /> promoter. Even though it is under the control of its endoge-         on protein conformation, rather than loss of J domain func- <br /> nous promoter, a modest increase in Cwc23&#226;&#338;&#172;J mRNA was                 tion. Therefore, we combined the H-to-Q substitution muta- <br /> observed, perhaps because it is expressed from a plasmid.            tion in CWC23 described above that renders J domains non- <br /> Strikingly, none of the other 21 J protein genes showed any          functional with G-patch NTR1 mutations. No double mutants <br /> significant difference in expression compared to the wt strain.      containing both cwc23H50Q and ntr1 mutants were recovered <br />  VOL. 30, 2010                                                     Cwc23, ESSENTIAL J PROTEIN WITH NONESSENTIAL J DOMAIN                      39 <br />  <br />  <br />                                                                                on 5-FOA plates, suggesting that cells cannot tolerate defects <br />                                                                                in both the J domain of Cwc23 and the G patch of Ntr1. <br />                                                                                   Since we demonstrated that the C terminus, but not the J <br />                                                                                domain, is important for Cwc23&#226;&#8364;&#8482;s interaction with Ntr1, we <br />                                                                                performed the analogous intragenic synthetic genetic interac- <br />                                                                                tion test, combining the N-terminal and C-terminal trunca- <br />                                                                                tions, generating cwc2381-225. Cwc2381-225 was stably expressed <br />                                                                                (Fig. 4D). However, no viable cwc2381-225 cells were obtained <br />                                                                                (Fig. 4C). Consistent with this result, cwc231-225, H50Q was also <br />                                                                                unable to rescue the viability of a cwc23&#226;&#338;&#172; strain (Fig. 4C). We <br />                                                                                conclude that J domain function becomes critical when the <br />                                                                                function of the C-terminal region, which is important for in- <br />                                                                                teraction with Ntr1, is compromised. <br />                                                                                   Specificity of synthetic genetic interaction of CWC23 J do- <br />                                                                                main mutations. Both circumstances in which Cwc23&#226;&#8364;&#8482;s J do- <br />                                                                                main was found to be important involved Ntr1: (i) when <br />                                                                                Cwc23&#226;&#8364;&#8482;s interaction with Ntr1 was compromised by the alter- <br />                                                                                ation of Cwc23 itself and (ii) when the Ntr1-Prp43 interaction <br />                                                                                was compromised by the alteration of Ntr1. We pursued the <br />                                                                                idea that J domain function was important when interaction <br />                                                                                among the components of the NTR complex was compromised <br />                                                                                by testing other synthetic genetic interactions. First, we tested <br />                                                                                a PRP43 mutation, prp43Y402A, an allele of PRP43 known to <br />                                                                                affect interaction with Ntr1 (33). Again, we used the plasmid <br />                                                                                shuffling technique, plating strains on 5-FOA-containing <br />                                                                                plates. As expected, prp43Y402A cells expressing full-length <br />                                                                                Cwc23 grew indistinguishably from wt cells. However, yeast <br />                                                                                strains expressing prp43Y402A and cwc23&#226;&#338;&#172;J or cwc23H50Q could <br />                                                                                not be recovered at 18 and 23&#194;&#176;C. Double mutants containing <br />                                                                                prp43Y402A and either cwc23&#226;&#338;&#172;J or cwc23H50Q were recovered at <br />                                                                                30&#194;&#176;C but grew slowly at that temperature (Fig. 5A and data not <br />                                                                                shown). <br />                                                                                   To more broadly explore the relationship between Cwc23 <br />                                                                                and splicing, we next looked for synthetic genetic interactions <br />                                                                                with components of the spliceosomal machinery that are <br />                                                                                known to function at steps other than disassembly. We exam- <br />                                                                                ined interactions with deletions of six nonessential compo- <br />    FIG. 4. The J domain of Cwc23 is critical when the G patch of Ntr1 is       nents, including the early-acting U1 component MUD1, the <br /> altered. (A) The C-terminal region is critical for interaction with Ntr1. A <br />                                                                                commitment complex factor MUD2, the U2-associated com- <br /> two-hybrid analysis was performed. Ntr1 fused to the GAL4 binding <br /> domain (BD) and/or Cwc23 proteins fused to the GAL4 activation do-             ponents LEA1 and MSL1, and finally ISY1 and NTC20, which <br /> main (AD) was expressed in a two-hybrid tester strain (see Materials and       are components of the NTC (for Prp nineteen complex) that is <br /> Methods). Equal numbers of cells were spotted on nonselective or selec-        added during spliceosomal activation (5). Notably, the NTR <br /> tive medium. Plates were incubated at 30&#194;&#176;C for 2 days. Growth on selec-        (for nineteen-complex related) components Ntr1 and Ntr2 <br /> tive medium is indicative of a positive two-hybrid interaction. (B) Equal <br /> numbers of ntr1&#226;&#338;&#172; cwc23&#226;&#338;&#172; URA3-Ntr1 cells harboring CWC23 (WT),                  have been shown to weakly interact with the NTC, suggesting <br /> cwc23&#226;&#338;&#172;J (&#226;&#338;&#172;J), or cwc23H50Q (H50Q) on a TRP-based plasmid and either            that they could interact with the spliceosome well prior to the <br /> a HIS3-based empty plasmid (&#226;&#171;&#186;) or the same plasmid carrying NTR1               disassembly step (35). Nevertheless, none of the genes we <br /> (WT), ntr1L68A (L68A), ntr1Y74A (Y74A), or ntr1L80A (L80A) were spotted        tested showed synthetic growth defects with cwc23&#226;&#338;&#172;J (see Fig. <br /> either on a plate with (&#226;&#171;&#185;) or without (&#226;&#171;&#186;) 5-FOA and incubated at 30&#194;&#176;C <br />                                                                                S3 in the supplemental material), indicating that the severe <br /> for 3 days. The white space indicates removal of strains not relevant to the <br /> present study. (C) The J domain is critical in the absence of C-terminal 58    synthetic growth defects observed with the NTR1 and PRP43 <br /> amino acids. Equal numbers of cwc23&#226;&#338;&#172; cells harboring two plasmids              mutations are specific. <br /> (URA3-Cwc23 and a vector containing no insert [&#226;&#171;&#186;], CWC23, or the                  The J domain and C-terminal regions of Cwc23 cannot <br /> indicated cwc23 mutant genes) were spotted on medium with (&#226;&#171;&#185;) or               function in trans. The results described above suggest the pos- <br /> without (&#226;&#171;&#186;) 5-FOA. Plates were incubated at 30&#194;&#176;C for 3 days. Cwc23H50Q <br /> (H50Q), Cwc23&#226;&#338;&#172;J (&#226;&#338;&#172;J), Cwc231-225 (1-225), Cwc2381-225 (81-225), and            sibility that Cwc23 carries out two independent functions: the <br /> Cwc231-225/H50Q (1-225/H50Q) are indicated. (D) Total protein extracted        C-terminal region being critical for an interaction with Ntr1 <br /> from strains, indicated in panel C was subjected to SDS-PAGE, electro-         and the N-terminal J domain for an interaction with Hsp70. <br /> blotted to a membrane, and probed with antibody specific for Cwc23 and,        We therefore examined, again using the 5-FOA selection tech- <br /> as a loading control, Ssc1 (row C, bottom panel). Cwc2381-225 (81-225), <br />                                                                                nique, whether the expression of Cwc231-225 could suppress the <br /> and Cwc231-225/H50Q (1-225 H50Q) were assessed prior to transfer to <br /> 5-FOA, since they are inviable in the absence of wt Cwc23. A dotted line       synthetic genetic defects seen in double mutant containing <br /> denotes the position of a segment of the gel which contained samples that      prp43Y402A and either cwc23H50Q or cwc23&#226;&#338;&#172;J. The prp43Y402A <br /> were irrelevant to the present study and thus removed.                         mutation was chosen because the synthetic growth defects with <br />  40     SAHI ET AL.                                                                                                      MOL. CELL. BIOL. <br />  <br />  <br />                                                                       was revealed to be a J protein with unusual, if not unique, <br />                                                                       characteristics. Although it plays a critical role in pre-mRNA <br />                                                                       splicing, its J domain is not normally required for this function. <br />                                                                       Unlike the situation with many J proteins, the J domain plays <br />                                                                       only an auxiliary role; its involvement in splicing is revealed <br />                                                                       when the function of other proteins (i.e., Ntr1 or Prp43) in- <br />                                                                       volved in spliceosomal disassembly are compromised. <br />                                                                          Cwc23&#226;&#8364;&#8482;s role in pre-mRNA splicing: disassembly of the spli- <br />                                                                       ceosome. Several lines of evidence point to an essential role for <br />                                                                       Cwc23 in the pre-mRNA splicing pathway. First, a partial loss- <br />                                                                       of-function CWC23 mutant affects both cell fitness and global <br />                                                                       pre-mRNA splicing. Second, this mutation, a truncation of the <br />                                                                       codons encoding the 58 C-terminal amino acids, affects Cwc23&#226;&#8364;&#8482;s <br />                                                                       physical interaction with Ntr1, a known essential splicing factor <br />                                                                       (35). More specifically, available data point to a critical role for <br />                                                                       Cwc23 in the spliceosome disassembly step of the pathway. <br />                                                                       CWC23 mutants accumulate a product of the splicing reaction, <br />                                                                       the lariat-intron, which is known to be stabilized upon failure <br />                                                                       to disassemble the spliceosome (2). Indeed, extensive evidence <br />                                                                       accumulated over a decade supports a role of Prp43 in cata- <br />                                                                       lyzing the final step of splicing, the dissociation of the lariat- <br />    FIG. 5. The J domain of Cwc23 is important specifically when in- <br />                                                                       intron, and the disassembly of the spliceosome (38). Thus, the <br /> teractions between NTR complex components are compromised. <br /> (A) The J domain of Cwc23 is important in the presence of             synthetic genetic interactions between CWC23 mutations and <br /> Prp43Y402A. Equal numbers of prp43&#226;&#338;&#172; cwc23&#226;&#338;&#172; URA3-Cwc23 cells har-      NTR1 or PRP43 mutations, but not mutations in genes in- <br /> boring a PRP43 (WT) or prp43Y402A plasmid and either a HIS3-based     volved in other steps in the pathway, also serve to place Cwc23 <br /> empty plasmid (&#226;&#171;&#186;), CWC23 (WT), cwc23&#226;&#338;&#172;J (&#226;&#338;&#172;J), or cwc23H50Q (H50Q)      at the disassembly step. <br /> were spotted onto media with (&#226;&#171;&#185;) or without (&#226;&#171;&#186;) 5-FOA and incu- <br /> bated at 30&#194;&#176;C for 2 days or 23&#194;&#176;C for 3 days. (B) Expression of J         Cwc23: an essential J protein with a nonessential J domain. <br /> domain-containing fragments in trans did not restore growth of        The most surprising result of the present study is the lack of a <br /> prp43Y402A cells expressing J domain mutant genes. prp43&#226;&#338;&#172; cwc23&#226;&#338;&#172;      <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span>phenotype</span> upon deletion of the J domain of Cwc23; there is no <br /> URA3-Cwc23 cells harboring either PRP43 (WT) or prp43Y402A plas-      obvious defect in either cell fitness or pre-mRNA splicing. To <br /> mids were transformed with an empty HIS3-based vectors (&#226;&#171;&#186;) or the <br />                                                                       our knowledge, Cwc23 is the only example of an essential J <br /> same vector containing CWC23 (WT), cwc23H50Q (H50Q), or cwc23&#226;&#338;&#172;J <br /> (&#226;&#338;&#172;J) in combination with either an empty LEU2-based (&#226;&#171;&#186;) vector or     protein with a nonessential J domain. Usually, alteration of the <br /> the same vector expressing one of three J domain containing frag-     J domain leads to the same phenotypic effect as deletion of the <br /> ments: Cwc231-225 (1-225) from the ADH promoter, or the J domain      entire protein. Indeed, in some cases much of a J protein can <br /> fragments of Jjj1 or Cwc23 driven by the GPD promoter (Jjj1-J and     be deleted with little or no affect, as long as the J domain itself <br /> Cwc23-J, respectively). Transformants were spotted onto media with <br /> or without 5-FOA and incubated at 23&#194;&#176;C for 3 days.                    is left intact (18, 41). An extreme example of this is provided by <br />                                                                       Ydj1, a 409-amino-acid J protein with a well-characterized <br />                                                                       client protein-binding domain. Although a complete deletion <br /> J domain mutations were the least severe of the PRP43 and             of Ydj1 has severe effects on cell growth, expression of the J <br /> NTR1 mutants discussed above. As seen in Fig. 5B, the inclu-          domain alone, at the level that Ydj1 is normally present, per- <br /> sion of cwc231-225 was insufficient for rescue. Similar results       mits quite robust growth (31). On the other hand, when dra- <br /> were obtained when examining the synthetic interaction seen           matic effects of mutations outside the J domain have been found, <br /> with strains containing ntr1L80A, a mutation in the G patch of        the consequences are typically no more severe than those that <br /> Ntr1 (see Fig. S4 in the supplemental material).                      only affect J domain mutation. For example, deletion of the re- <br />    Since Cwc23 is of low abundance in cells, we also wanted to        gion necessary for binding of the J protein Zuo1 to the ribosome <br /> test whether higher levels of expression of a J domain could          results in the same phenotype as disruption of only the J domain <br /> overcome the synthetic growth defect when in trans with the           (43). <br /> CWC23 J domain mutants. Plasmids were constructed that                   To our knowledge, only two examples in which the J domain <br /> allowed for the overexpression of J domain fragments of either        of a protein appears to be of less biological importance than <br /> Cwc23 or the J protein Jjj1 from the GPD promoter. Overex-            other domains have been reported previously: P58IPK and <br /> pression of the J domains also failed to rescue the synthetic         RSP16. Neither case is as striking as that of Cwc23. In the case <br /> growth defects of strains combining J domain mutations in             of P58IPK, its J domain-independent function appears to be in <br /> CWC23 with prp43Y402A (Fig. 5B). In sum, we conclude that the         a different cellular compartment from its J domain-dependent <br /> two domains of Cwc23 must be present as a single polypeptide          function. In the cytosol, where it is present in low abundance, <br /> to perform their function(s).                                         it acts as an inhibitor of interferon-induced protein kinase <br />                                                                       (PKR), a J domain-independent activity important for produc- <br />                                                                       tive infection by influenza virus (42). However, P58IPK is pre- <br />                          DISCUSSION <br />                                                                       dominantly found in the endoplasmic reticulum lumen, where <br />  Here we report evidence supporting an important role for             it plays an important role as a bona fide J protein cochaperone <br /> Cwc23 in the disassembly of the spliceosome. In addition, Cwc23       of Hsp70 (27). RSP16 plays a role in flagellar stroke movement <br />  VOL. 30, 2010                                             Cwc23, ESSENTIAL J PROTEIN WITH NONESSENTIAL J DOMAIN                                        41 <br />  <br />  <br /> in Chlamydomonas. A fragment lacking the J domain appears             systems. Regardless of the exact role of chaperone system in <br /> to function as well as full-length protein in regulation of flagel-   spliceosome disassembly, Cwc23 may provide an extreme ex- <br /> lar beating (44), although its exact function in this process is      ample of an &#226;&#8364;&#339;underappreciated class&#226;&#8364;&#65533; of J proteins whose <br /> not known. The functionality of the J domain of RSP16 has yet         biological role is defined more by their non-J domain regions <br /> to be demonstrated. RSP16 does have the defining HPD motif,           than by their J domains, that is, their traditional Hsp70-related <br /> but its orthologs from human, zebrafish, mouse, and mosquito          functions. Understanding how such J proteins evolved may <br /> do not (44).                                                          afford important insight into the diversity of J protein function <br />    Function of Cwc23&#226;&#8364;&#8482;s J domain. Both our in vitro and in vivo        and how chaperone systems may be recruited to fine-tune <br /> results establish that S. cerevisiae Cwc23 has a functional J         protein complex remodeling. <br /> domain. It has the capacity to stimulate the weak ATPase <br /> activity of the Hsp70, Ssa1, and substantially rescue the severe                                 <br /> growth defect of ydj1&#226;&#338;&#172; cells. Both abilities are eliminated by <br />                                                                         <br /> the same single amino acid alteration in the J domain. How-            <br /> ever, the J domain of Cwc23 is clearly dispensable for cell           <br /> viability and efficient pre-mRNA splicing. Nevertheless, the         <br /> severe synthetic genetic interactions between mutations that         <br /> alter the J domain of Cwc23 in ways known to disrupt func- <br /> tional interaction with Hsp70 and NTR1 or PRP43 mutations                                              <br /> that affect the Ntr1:Prp43 interaction suggests that the J do-    <br /> main function of Cwc23 becomes critical when the interaction             <br /> between Ntr1 and Prp43 is compromised. We suspect that the              <br /> J domain is important under some environmental condition                  <br /> that we did not reproduce in the laboratory.                          <br />    The questionable functionality of the J domain of RSP16               <br /> orthologs in higher eukaryotes raises the issue of the conser-             <br /> vation of Cwc23 as a J protein. However, unlike RSP16 dis-         <br /> cussed above, Cwc23 orthologs in higher eukaryotes do contain           <br /> an HPD motif, suggesting a conservation of J domain function.         <br /> Interestingly, S. cerevisiae, compared to most eukaryotes, has     <br /> evolved a simplified splicing machinery, since it lacks alterna-          <br /> tive splicing and overall has few intron-containing genes (5,          <br /> 30). Perhaps the J domains of Cwc23 orthologs play a more                 <br /> prominent role in organisms with more complex splicing ma-                <br /> chineries. That Cwc23 orthologs identified in higher eukaryotes        <br /> have an RNA recognition motif (i.e., RRM; data not shown),         <br /> a motif known to mediate protein-RNA interactions (11) that           <br /> is absent from the fungal proteins, is in keeping with this line       <br /> of logic.                                                             <br />    What is the relationship between the role of Cwc23&#226;&#8364;&#8482;s J do-          <br /> main, and thus Hsp70, and that of the C-terminal region of             <br /> Cwc23? Are their functions totally independent or at least              <br /> partially inter-related? Although we cannot answer these ques-        <br /> tions with certainty, the results reported here provide some            <br /> insight. The function of the two domains cannot be completely         <br /> independent, since the J domain cannot carry out its roles when       <br /> expressed as an independent polypeptide. Interaction of                <br /> Cwc23 with Ntr1 may serve to tether the J domain to a par-           <br /> ticular location, enormously increasing its local concentration,       <br /> and efficiently allowing recruitment of Hsp70. However, on the            <br /> other hand, the C-terminal domain must be playing a role           <br /> beyond simply tethering the J domain since this domain is        <br /> essential and Cwc23&#226;&#8364;&#8482;s J domain is not.                                 <br />    Recent biochemical results indicate that the interaction be-         <br /> tween Prp43 and Ntr1 is critical in the disassembly step, with        <br /> Ntr1 not only tethering Prp43 to the spliceosome but also able       <br /> to activate its helicase activity (4, 33, 36). An involvement of         <br /> Cwc23 in modulating the interaction between Prp43 and the               <br /> NTR complex, increasing the efficiency of spliceosome disas-       <br /> sembly, is an intriguing possibility. Participation in the remod-          <br /> eling of protein complexes is a known role for Hsp70-J protein       <br /> </body></html>