<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL OF BACTERIOLOGY, Sept. 1985, p. 918-924                                                                       Vol. 163, No. 3 <br /> 0021-9193/85/090918-07$02.00/0 <br /> Copyright &#194;&#169; 1985, American Society Microbiology <br />  <br />  <br />  Isolation Characterization Transposon Tn5-Induced Mutants <br />         Pseudomonas perfectomarina Defective Nitrous <br />                           Oxide Respiration <br />                               WALTER G. ZUMFT,* KARL DOHLER, AND HEINZ KORNER <br />             Lehrstuhl fur Mikrobiologie, Universitat Karlsruhe, D-7500 Karlsruhe 1, Federal Republic Germany <br />                                            Received 1 April 1985/Accepted 5 June 1985 <br />  <br />              Transposon (TnS) mutagenesis Pseudomonas perfectomarina plasmid pSUP2021 [(pBR325- <br />            Mob(RP4))::TnS] chromosomaily integrated RP4 plasmid Escherichia coli donor, produced <br />            distinct groups mutants defective nitrous oxide respiration. One group mutants lacked <br />            structural protein N20 reductase, second synthesized copper-free apoprotein; group <br />            expressed low level intact enzyme. The mutants provided evidence N20 immediate precursor <br />            dinitrogen denitrification documented essentiality copper enzyme. Synthesis N20 <br />            reductase depended strongly growth conditions, N20-grown cells expressing lowest level of <br />            enzyme. Regulatory responses mutants elicited nitrate oxygen unaltered compared with <br />            wild-type behavior. <br />  <br />   Respiration nitrous oxide (N20) common trait               derivative E. coli C600 integrated RP4-2-Tc::Mu <br /> denitrifying bacteria. Interestingly, certain denitrifiers          chromosome carries addition plasmid <br /> unable utilize N20 (14), Wolinella sp., non-               pSUP2021 [(pBR325-Mob(RP4))::TnS]. The recipient a <br /> denitrifying species, does respire substrate (27, 35). N20         spontaneous streptomycin-resistant mutant P. perfecto- <br /> respiration strict requirement copper (15, 23),               marina ATCC 14405. <br /> which attributed novel type metalloprotein                 Media growth conditions. The donor recipient <br /> denitrifying pseudomonads (25). The protein Pseu-                  grown LB medium nitrate (LBN) containing the <br /> domonas perfectomarina Baumann et al. 1983 (formerly P.                 following (grams liter): tryptone, 10; yeast extract, 5; <br /> perfectomarinus) identified terminal oxidoreduc-             NaCl, 10; NaNO3, 1. The pH adjusted 7.0 1 <br /> tase, multicopper enzyme dimeric structure containing              N NaOH. An Erlenmeyer flask 20 ml Luria broth was <br /> an unusual chromophore (37, 38).                                        inoculated agar slant. Cultures incubated 24 <br />   Denitrification received scant investigation physi-            h gyratory shaker (180 rpm) 30&#194;&#176;C. <span id='am-2' about='protege:TO' typeof='owl:Thing'>To</span> screen Nos- <br /> ological genetics, genes assimilatory respi-           mutants, exconjugants grown N20 agar plates <br /> ratory nitrate reductase successfully discriminated <br /> with approach (17). Here,                  containing synthetic, nitrogenous oxide-free medium (23). <br /> transposon TnS mutagenesis P. perfectomarina                The medium supplemented 1 ,uM CuCl2, strepto- <br /> suitable dissection partial reactions, components,           mycin (200 ,ug/ml), kanamycin (50 ,ug/ml). <br /> regulatory functions denitrification. TnS chosen be-               For chromatographic immunochemical analysis, mu- <br /> cause predominantly random insertion (4). We                tants grown 30&#194;&#176;C synthetic medium containing <br /> used Escherichia coli vector plasmid pSUP2021,                kanamycin (50 ,ug/ml). Cells grown anaerobically with <br /> was derived plasmid pBR325, carries TnS,                   0.1% NaNO3 (completely filled culture vessels) anaerobi- <br /> made transmissible insertion origin transfer          cally N20 sparging 2-liter round flask approx- <br /> the broad-host-range plasmid RP4 (30). A special donor                  imately 50 ml gas min. Oxygen-limited growth, or <br /> strain E. coli, carrying plasmid, transfer         0.1% nitrate, obtained incubating 1 liter of <br /> functions RP4 integrated chromosome. This                   medium 2-liter Erlenmeyer flask 120 rpm a <br /> ensured mobilization pSUP2021 P. perfectomarina                 gyratory shaker. Flasks covered aluminum foil. <br /> without cotransfer RP4. The transposition event                  The oxygen transfer rate, KLaC* value, determined as <br /> conveniently selected transposon-coded kanamy-               described previously (9), 0.02 mmol 02 liter per <br /> cin resistance. This strategy mutagenesis originally             min conditions. Cells harvested centrifu- <br /> developed random TnS insertion Rhizobium sp. (31).               gation 8,000 x g 10 min. The cell paste suspended <br /> We usefulness isolation P.                     centrifugation 1 volume (wt/vol) 20 mM Tris- <br /> perfectomarina mutants defective N20 respira-               hydrochloride (pH 7.5) used activity <br /> tion. Mutants isolated fall groups:              measurements stored -26&#194;&#176;C. <br /> without N20 reductase, synthesizing copper-free apo-                 Conjugation. One milliliter stationary-phase recipient <br /> enzyme, exhibiting low level enzyme.             mixed 0.4 ml exponential-phase donor, centri- <br />                MATERIALS AND METHODS                                    fuged, suspended 60 ,ll LBN. The cell mixture was <br />   Bacterial strains plasmids. The donor strain TnS              spotted nitrocellulose filters (SM 113; Sartorius), which <br />                                                                         placed LBN plates incubated overnight 30&#194;&#176;C. <br /> mutagenesis E. coli SM10-2021 (30). This strain                The filters immersed test tubes 2 ml of <br />                                                                         0.85% (wt/vol) saline mixed thoroughly Vortex <br />   * <br />       Corresponding author.                                             mixer. Samples 0.1 ml suspension plated on <br />                                                                   918 <br />  VOL. 163, 1985                                        NITROUS OXIDE REDUCTASE MUTANTS OF PSEUDOMONAS                        919 <br />  <br />  LBN agar streptomycin (200 ,ug/ml) kanamycin (25        adjusted 1 mg protein ml 0.15 M NaCl 20 mM <br />  p.g/ml) incubated aerobically 48 h 30&#194;&#176;C.              4-morpholinepropanesulfonate (pH 7.0). One milligram of <br />    Screening Nos- mutants. Exconjugants transferred       enzyme emulsified 1 ml complete Freund adju- <br /> by replica plating LBN plate plate         vant injected subcutaneously the <br />  synthetic medium nitrogenous oxides (see above).         animal. Another sample 1 mg protein Freund <br />  Replica plates placed anaerobic jars fitted vents   incomplete adjuvant administered 3 weeks intra- <br />  (GasPak 150; BBL Microbiology Systems, Cockeysville,             muscularly cervical region. Additional samples <br />  Md.). The atmosphere changed N20 evacuating            1 0.5 mg enzyme saline injected 1 and <br /> the jars times refilling N20. The jars        2 weeks, respectively. Blood samples taken <br /> incubated 48 h 30&#194;&#176;C. Replica plates examined      weeks 5, 6, 7, animal sacrificed. <br />  colonies grown incubation N20.      The antibody titer followed double immunodiffusion <br />  Loops corresponding colonies master plates           (8) high week 5. Serum drawn immu- <br /> were transferred activity measurements vials con-        nization showed precipitin reaction crude extract of <br /> taining 3 ml synthetic medium asparagine, kanamy-         wild type purified reductase. Antiserum was <br /> cin (50 ,ug/ml), 0.1% NaNO3. The vials evacuated         immunoabsorbed batches (8) crude extract <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> once, filled helium, incubated 30&#194;&#176;C overnight.        mutant KD8 grown nitrate. <br /> The products denitrification assayed gas chroma-          N20 reductase quantitated electroimmunoassay <br /> tography (10). Colonies produced quantitatively          (19). A 1-mm gel agarose NA (sulfate content, 0.02%; <br /> N20 assay formed traces N2 inocu-         Pharmacia) cast vertically 9- 8.5-cm mold that <br /> lated NB-glycerol medium (33) stored -26&#194;&#176;C.           lined polyester foil (GelBond; FMC <br />    Preparation fractionation cell-free extracts en-    Corp.). Of 8.5-cm height, approximately 5 cm taken <br /> zymes. Thawed cell paste mutants wild type          gel containing 1% immunoabsorbed antiserum; the <br /> usually disrupted pulsed cavitation 75 W 10 min         remainder bottotn top, contact paper <br /> under continuous cooling ice bath. For vitro analysis    wicks electrode buffer, cast gel <br /> of N20 reductase activity, suspension freshly harvested      containing antiserum. Gel electrode buffers 75 <br /> cells broken passages French press        mM barbital (pH 8.6), 1.3 mM calcium lactate. During <br /> 110 MPa. The press anaerobic filling       sample application hour voltage 5 V <br /> times rapid filling device anaerobic buffer.     cm- applied; 3 h, 10 V cm- used. The <br /> The homogenate collected argon-filled, capped serum        gel cooled 10&#194;&#176;C. <br /> bottles centrifuged anaerobically 60 min 47,500 x         Crossed immunoelectrophoresis (2) used con- <br /> g 111,000 x g. For gel permeation chromatography,            ditions electroimmunoassay. A gel antiserum <br /> sample supernatant approximately 100 mg          cast glass plate; electrophoresis 60 min at <br /> protein applied anaerobic 2.5- 90-cm column       10 V cm-1, strip 3 cm transferred of <br /> Sephacryl S-200 (superfine; Pharmacia Fine Chemicals, Upp-        above-specified mold. The gel overlaid antiser- <br /> sala, Sweden) eluted 0.1 M KCl 20 mM Tris-            um-containing agarose; gelling, electrophoresis was <br /> hydrochloride (pH 7.5). Fractions 4.8 ml collected        continued 16 h 3 V cm-. <br /> under argon analyzed N20-reducing activity,                  Gels washed overnight 0.9% NaCl with <br /> cytochrome content absorbance 420 nm,             changes deionized water dried 30&#194;&#176;C air <br /> copper content. After run column washed         stream. The dry sheets stained Coomassie brilliant <br /> 1 liter 0.5% Triton X-100 Tris buffer         blue R-250 destained desired contrast (2). <br /> equilibrated buffered KCl. Fraction no. 48              Analytical methods. Sodium dodecyl sulfate-polyacryl- <br /> protein fractionation dialyzed extensively 5 mM       amide gel electrophoresis performed 0.5-mm horizon- <br /> Tris buffer (pH 7.5) remove KCl. The dialysate stored      tal slab gels linear gradient acrylamide 9.5 to <br /> frozen immunochemical electrophoretic analysis.         22.6% (11). The starting voltage 200 V 15 min; the <br /> N20 reductase nitrite reductase (cytochrome cdl)         separation voltage 500 V approximately 150 min. <br /> prepared published methods (37, 39).                           The gels fixed silver stained minor adaptations <br />    Enzymatic assays. Reduction nitrite, NO, N20         polychromatic procedure (29). Five commercial brands <br /> intact cells lactate electron donor assayed    mercaptoethanol (from Bio-Rad Laboratories, LKB In- <br /> gas chromatography (10). Substrate coitcentrations 17        struments Inc., E. Merck AG, Roth, Sigma Chemical <br /> mM sodium nitrite solution 117 kPa NO N20      Co.) caused appearance prominent, striated bands at <br /> gas phase. N20 reductase activity monitored vitro          Mr 66,000 56,000 weaker bands Mr 135,000 and <br /> spectrophotometrically photoreduced 5-deazaflavin        48,000 stained gels (21). These bands observed <br /> benzyl viologen mediator (37). The reaction mixture        samples applied. Substitution mercap- <br /> was prepared 12-ml cylindrical, stoppered vials           toethanol dithioerythritol molar concentra- <br /> contained (in 2.65 ml final volume) 250 ,umol phosphate        tion eliminated artifacts. Background staining from <br /> buffer (pH 7.1), 100 ,umol neutralized triethanolamine, 40     electrode wicks (paper no. 1; Whatman Ltd.) avoided by <br /> nmol 5-deazaflavin, 0.25 ,umol benzyl viologen.         boiling 1% sodium dodecyl sulfate 1 h followed <br /> The atmosphere vials N20. The assay mixture            electrophoresis running conditions overnight. <br /> was reduced exposure daylight 3 min. Benzyl                Conditions registration difference spectra were <br /> viologen oxidation, addition appropriate          similar previous specifications (23). Protein deter- <br /> amount enzymes, followed 600 nm Perkin-            mined cell-free extracts biuret method (12) or <br /> Elmer 557 spectrophotometer.                                      Folin method (20). Whole cells di- <br />    Immunochemical techniques. Antibodies purple       gested overnight 1 N NaOH assayed protein. <br /> form N20 reductase raised New Zealand white          The copper content chromatographic fractions deter- <br /> rabbit. A sample electrophoretically homogeneous en-           mined atomic absorption spectroscopy 324.7 nm. <br /> zyme extensively dialyzed water, lyophilized,            Chemicals. Antibiotics benzyl viologen from <br />  920     ZUMFT, DOHLER, AND KORNER                                                                                             J. BACTERIOL. <br />  <br /> Sigma, Munchen. N20 99.5% purity Hoechst <br /> AG, Frankfurt. Bacto-agar Difco Laboratories, <br /> Detroit, Mich. Deazaflavin gift late P. <br /> Hemmerich. Chemicals electrophoresis LKB,                                  0 <br />                                                                                        C.) <br />                                                                                          0 <br /> Grafelfing. Dithioerythritol chemicals from <br /> Merck, Darmstadt, analytical-grade quality.                                       1 23 <br />                            RESULTS                                         FIG. 2. Quadruplicated crossed immunoelectrophoresis N20 <br />    Selection mutants. P. perfectomarina suscep-             reductase extracts mutant KD2 wild type with <br /> tible kanamycin synthetic asparagine-citrate me-              immunoabsorbed antiserum enzyme. Samples: 1 4, <br />                                                                         1.2 ,ug purified N20 reductase; 2, 45 ,ug KD2 crude extract; 3, <br /> dium LBN. Selection Tn5 mutants                45 ,ug wild-type extract. Other conditions described the <br /> done LBN agar containing 200 jig streptomycin 25              text. <br /> ptg kanamycin ml. Under conditions exconjug- <br /> ants observed frequency 2 x 10-6 2 days of <br /> incubation air. TnS-coded streptomycin resistance                 nitrosoguanidine, mutants obtained unable to <br /> (26) did interfere counterselection used spon-            grow N20, nitrate-grown cells reduce this <br /> taneous streptomycin-resistant mutant P. perfectomarina.             substrate. This <span id='am-3' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-4' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-5' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-6' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span>phenotype</span> observed Tn5- <br /> Auxotrophic mutants frequency approx-                mutagenized cells. <br /> imately 1%. Markers included leu, ile, trp, met, his, phe,             Biochemical characterization. Cell-free extracts mutants <br /> tyr. The stability TnS-induced mutants high;                 grown nitrate oxygen limitation tested <br /> frequency revertants lower 10-8,                  spectrophotometrically N20 reduction following the <br /> mutants subcultured medium kanamycin.                   oxidation photochemically reduced benzyl viologen the <br />    Selection 19 mutants defective N20 respiration,            electron donor (see above). The mutants did exhibit a <br /> of thousands kanamycin-resistant clones,                 clearly discernible benzyl viologen-oxidizing activity vitro <br /> based inability grow agar plates N20            low background bleaching. Figure 1 shows mutant <br /> atmosphere (Nos- phenotype). The mutants                      KD2 representative example, contrasting the <br /> grown anaerobically liquid medium nitrate induce               activity wild type. Since reasonable assume <br /> the denitrification (25, 38) tested vivo          that, assay, benzyl viologen couples directly the <br /> gas chromatography lack N20 reduction                   enzyme mediation component, the <br /> asparagine electron donor. Thirteen strains, desig-              absence activity indicated reductase function was <br /> nated KD2, KD3, KD7, KD8, KD10, KD12 KD19                        affected. A possible loss structural protein was <br /> were chosen 11 independent mutageneses analyzed                checked immunochemically. <br /> in detail.                                                                 An antiserum raised purified N20 reductase was <br />    During selection process isolated mutants             assured monospecific immunoabsorption cell- <br /> defective cytochrome synthesis visually recog-             free extract mutant KD8, lacked structural <br /> nized agar plates greenish color. Selection             protein N20 reductase (see below). Crossed immunoelec- <br /> respiratory nitrate nitrite utilization yielded mutants         trophoresis N20 reductase wild-type extract pro- <br /> TnS insertions, exhibited Nar- Nir- phenotype.              duced single precipitin line antiserum. There was <br /> By chemical mutagenesis N-methyl-N&apos;-nitro-N-                       immunochemical identity purified enzyme and <br />                                                                         reactive component wild-type mutant extracts <br />                                                                         (Fig. 2). Cross-reacting material N20 reductase <br />                                                                         antiserum identified mutants electroim- <br />                    0.6   - <br />                                                                         munoassay (Fig. 3). The cross-reacting material in <br />                                                                         mutants KD2, -7, -12, -16, -17 (group I) comparable <br />                                                                         wild type, i.e., N20 reductase content 2 <br />               E                                                         3% total protein, depending growth conditions. <br />                                                                         Mutants KD3, -8, -13, -15, -18, -19 negative (group <br />               c <br />                                                                         II); mutants KD10 -14 (group IIT) showed small <br />                                                                         cross-reacting material (Fig. 3). <br />               Cu0 0.3    - <br />  <br />  <br />                                                                            Information exact nature cross-reacting ma- <br />               0                                                         terial sought fractionating cell-free extracts mo- <br />               .0                                                        lecular sieve gel (Fig. 4), previously shown to <br />               0 <br />               .0                                                        yield characteristic profile copper chromophore <br />                                                                         N20 reductase (25, 37). None mutants group II <br />                                                                         showed copper peak retention volume (230 ml) for <br />                                                                         N20 reductase, accordance absence im- <br />                          0          3             6                     munochemically cross-reacting material. However, elu- <br />                                Time (min)                               tion profiles group I mutants devoid this <br />                                                                         copper peak (Fig. 4B), copper associated with <br />   FIG. 1. Comparison benzyl viologen-dependent reduction          particles eluting exclusion volume low Mr <br /> nitrous oxide crude extracts wild type (      ) mutant    copper protein unknown function (25). The copper con- <br /> KD2 (- -). The assay started injection 10 ,ul crude <br /> extract (arrow). Decreasing absorbance 600 nm corresponds         tent crude extracts mutants 30 40 ng mg of <br /> the oxidation reduced benzyl viologen. The wild-type assay           protein, wild type 140 ng <br /> contained 28 ,ug protein, mutant contained 33 jig.   mg   1. <br /> Controls N20 shown wild type ( 0)             Fraction no. 48, corresponded correct elution <br /> mutant (-0-).                                                           position N20 reductase Sephacryl column, was <br />  VOL. 163, 1985 <br />  <br />  <br />         ST 2 <br />  <br />  <br />  <br />  <br />         ST 3 <br />                    ~ <br />                    A. <br />                        7 12 16 17 10 14 <br />  <br />  <br />  <br />  <br />                        8     13 15 18 19 WT <br />                                             ,       t0 <br />                                                            NITROUS OXIDE REDUCTASE MUTANTS OF PSEUDOMONAS <br />  <br />                                                                         strate, wild type showed accumulation N20, <br />                                                                         NO inhibits N20 reduction (23). <br />                                                                            The inability mutants utilize N20 oxidant was <br />                                                                         documented following cytochrome absor- <br />                                                                         bance intact cells addition nitrogenous oxides <br />                                                                         (Fig. 6). The wild type mutant KD2 responded alike to <br />                                                                         nitrite NO strong transient oxidation their <br />                                                                         cytochromes lactate-reduced cells. In manner, <br />                                                                         cytochromes wild-type cells oxidized N20, <br />                                                                         response demonstrable mutants. The <br />                                                                         addition N20 wild-type cells produced spectral changes <br />                                                                         long-wavelength region indicative cytochrome cdl. <br />                                                                         The <span id='am-81' about='xsp:pattern' typeof='owl:Thing'>pattern</span> soluble c-type cytochromes evidenced by <br />                                                                         gel permeation chromatography unchanged, irrespec- <br />                                                                                                                                       921 <br />  <br />  <br />  <br />  <br />                                                                         tive group mutants belonged.4Examples <br />                                                                         given Fig. 4 strains KD2 -10 rtpresenta- <br />                                                                         tive mutants. <br />                                                                            The quantitative immunoassay allowed check the <br />                                                                         expression N20 reductase different growtI condi- <br />                                                                         tions probe mutants group I, although <br />                                                                         enzymatically inactive, identical re- <br />    FIG. 3. Electroimmunoassay N20 reductase structural pro-          sponse respect protein synthesis. Comparable levels <br /> tein wild type Nos mutants. The numbers refer         N20 reductase protein synthesized mutant KD2 <br /> corresponding mutant strains. Each contained 30 ,ug crude       wild-type cells (Fig. 7). The level enzyme expression <br /> extract 3p.1.Other samples: ST, 0.6 ,ug purified N20 reductase;   N20-grown wild-type cells surprisingly low. Nitrate <br /> WT, 30 ,ug crude extract wild type. Cells grown     stimulated synthesis N20 reductase, diminished <br /> nitrate 02 limitation. Further experimental details    02 pressure sufficed expression enzyme; complete <br /> text.                                                                   anaerobicity required. These regulatory responses <br />                                                                         unaltered mutants KD2 -7 (Table 2). The relation <br /> checked double immunodiffusion presence                   vivo activity total N20 re4uctase was <br /> material cross-reacting N20 reductase antiserum.                   determined immunoassay. We consistently high <br /> Group I mutants positive assay. More detailed              specific N20 reductase activity cells grown N2O or <br /> molecular size information obtained sodium dode-               02 limitation absence nitrate significantly <br /> cyl sulfate gradient polyacrylamide gel electrophoresis (Fig.           lower activity nitrate was, present. The low N20 <br /> 5). The main components fraction no. 48, evident             reductase content N20-grown cells largely compen- <br /> the wild-type trace Fig. 5, peptides Mr                sated high catalytic activity (Table 2). <br /> around 60,000 70,000. They represent monomeric <br /> forms nitrite reductase (cytochrome cdl) N20 <br /> reductase, respectively. Mutants group I, devoid                                      A~ ?A      rCu <br /> of copper peak gel filtration, showed presence a <br /> peptide electrophoretic mobility                      0.4 <br />                                                                                    O                         llpg/mI. 1.0        I <br /> N20 reductase monomer. Group II mutants lacked this <br /> peptide. A weak band Mr 120,000, seen samples                            0.2   [                                    osc <br /> except N20 reductase preparation, residual undis- <br /> sociated dimer cytochrome cd1, caused use                         Eo :I                        : :                  &apos;E <br /> dithioerythritol reductant. The pattern low Mr                       c 0                                        o      X <br /> proteins (30,000 40,000) showed variation repet-                    R <br /> itive runs, depending sample concentration                      %.O0.4 <br />                                                                                   OA                                      1.0 <br /> reflect non-stoichiometric silver-staining (28); relation                 0 <br /> TnS insertion remains open.                                                        2         Br                            0.5 <br />                                                                                                                                  0 <br />  <br />  <br />  <br />  <br />    Strain KD10 group III, low level                                              I Al                          z <br />  <br /> immunochemically cross-reacting material crude extracts <br /> (Fig. 3), showed weak N20-reducing activity vivo. The gel                   lO            :Cn <br /> permeation profile revealed small copper peak (Fig. 4C),                                             (i <br /> and electrophoretic pattern faint band                            OA <br /> position enzyme monomer (Fig. 5). This mutant, thus, <br /> expressed low level intact N20 reductase.                                     o2F <br />    Physiology mutants. In Table 1 activities partial <br /> denitrification reactions representative Nos- mu-                           0                                     0 <br /> tants compared wild type. Except                                     40          60      80 <br /> the loss N20 reduction, nitrogenous oxide-                                             Fraction number <br /> reducing activity affected mutants. The rates               FIG. 4. N20 reductase activity, copper protein, cytochrome <br /> nitrite nitric oxide utilization similar wild           elution profiles wild type (A), mutant KD2 (B), mutant <br /> type mutants. The wild type liberated denitrification            KD10 (C) gel permeation Sephacryl S-200. Symbols: <br /> intermediates nitrite. The mutants, contrast, trans-                 ) copper content; (--- -) absorbance 420 nm; (@-* Y N20 <br /> formed nitrite quantitatively N20. With NO sub-               reductase activity. See text details. <br />  922         ZUMFT, DOHLER, AND KORNER                                                                                                            J. BACTERIOL. <br />  <br />                          STWT2 7 12 16 17 10 14 A B 3 8 13 15 1819WTST <br />  <br />  <br />  <br />                                                                                                                                         -94 <br />                                                                                                                                           67 <br />  <br />                                                                                                                                           43 <br />                                                                                                                                           -30 <br />  <br />                                                                                                                                       - <br />                                                                                                                                           20.1 <br />  <br />                                                                                                                                       - <br />  <br />  <br />  <br />  <br />   FIG.         5.   Sodium dodecyl sulfate        gradient   polyacrylamiide gel electropho&apos;resi&apos;s wild type Nos- mutants. The numbers denote <br /> specific       mutant strains. Each slot contained 100           nig protein fraction no. 48 gel permeation chromatography (see Fig. 4). Other <br /> lanes: WT,          idenitically   treated wild type; A, 50 ng    purified cytochrome cdl; B,    12.5 ng     purified N20 reductase; ST, standard proteins <br /> (Pharmacia), molecular               masses   given kilodaltons. Proteins     stained     polychromatic silver stain. <br />  <br />  <br />  <br />                                         DISCUSSION                                         strains affected energy transduction (e.g., proton <br />    Transposon TnS preferred element mutagenesis <br />                                                                                          translocation) resembled chemically induced mutants of <br /> because low insertional specificity organisms                              P. perfectomarina. <br /> (4). Among pseudomonads, genomic insertion TnS                                    The mutants obtained work provide evidence P. <br /> been shown Pseudomonas aeruginosa (31a) Pseudomo-                                      perfectomarina N20 precursor dinitrogen at <br /> nas putida (26, 34) Pseudomonas solanacearum (6). The <br />                                                                                            +1 oxidation level nitrogen. The role N20 an <br /> TnS mutagenesis adapted P.                                              intermediate nitrite respiration deduced, among <br /> perfectomarina proven effective isolation                                   others, physiological (18) biochemical (24) studies, <br /> respiratory mutants. TnS transposition based suicide <br /> plasmid pJB4JI, however, did generate respiratory mu- <br /> tants Pseudomonas stutzeri (7). Besides mutants impaired <br /> in N20 reduction, mutants defective nitrate <br /> and nitrite respiration slightly lower frequency. The                                                                         AA.01 <br /> isolation auxotrophic mutants indicated absence <br /> of preferential insertion sites. <br />    The Nos phenotype described work character-                                                                                        A <br /> ized inability grow N20 terminal <br /> electron acceptor, lack N20-reducing activity anaerobi- <br /> cally grown nitrate-induced cells, inability of <br /> N20 act cytochrome oxidant intact cells. Tn7- <br /> induced mutants Nos- phenotype obtained P. <br />                                                                                                                            ABa <br /> stutzeri incompetent synthesize cell mass the <br /> expense N20 respiration, possessed terminal <br /> reductase, cells reduced N20 (7). Such                                                                &lt;W1~~~~~ <br />     TABLE 1. Respiratory activities endproducts partial <br />  denitrification reactions wild type mutant strains grown <br />                     nitrate 02 limitation <br />                                                                                                           450          550            650 <br />                             Results following respiratory substrate:                                   Wavelength (nm) <br />       Strain               Nitrite             Nitric oxide          Nitrous oxide           FIG. 6. Comparison oxidative response cellular <br />                     Activitya Product&quot; Activity       Product      Activity Product        cytochromes nitrogenous oxides wild type mutant <br />                                                                                            KD2. Cells grown reduced oxygen tension nitrate. <br /> Wild type             2.2          N2       1.4     6.5% N20,         10.3     N2          Reaction mnixtures 2 ml He-filled cuvettes contained cell <br />                                                       93.5% N2{                            suspensions equivalent 2.2 mg protein, 150 ,umol sodium <br />  KD2                   2.8         N201.0               N20      0                         lactate, 85 ,umol potassium phosphate buffer (pH 7.0). The <br />  KD3                   3.0         N201.0               N20      0                         difference spectrum recorded injectiQn 20 ,umol of <br />                                                                                            anaerobic NaNO2 0.3 ml NO N20 reference cuvette. <br />    a <br />      Activities given uptake respiratory substrate micromoles             Curves: A, lactate-reduced-minus-N20-oxidized spectrum wild- <br />  hour milligram cell protein.                                                   type cells: B, lactate-reduced-minus-N20-oxidized spectrum of <br />    bUnless specified, product consisted 100lo indicated gas <br />  species.                                                                                  KD2; C, lactate-reduced-minus-NO-oxidized spectrum KD2; D, <br />    c   Distribution products determined 24 min reaction time.              lactate-reduced-minus-nitrite-oxidized spectrum KD2. <br />  VOL. 163, 1985                                                 NITROUS OXIDE REDUCTASE MUTANTS OF PSEUDOMONAS                                923 <br />  <br />            A       B    C       D ST                  b      C <br />                                                                             lost ability synthesize structural protein of <br />                                                                             N20 reductase affected central regulatory gene <br />                                                                             controlling expression enzyme. Group III mutants <br />                                                                             low levels enzyme comprise <br />                                                                             regulatory mutants. <br />                                                                                Our data document TnS mutagenesis essentiality of <br />                                                                             copper protein N20 respiration. Several independent <br />                                                                             criteria attributed N20 reductase function this <br />                                                                             metalloprotein (23, 25, 37, 38). The mutational loss the <br />                                                                             structural protein metal group, concomitant with <br />   FIG. 7. Electroimmunoassay N20 reductase wild type              loss enzymatic activity cell-free extracts, supports <br /> (A D) mutant KD2 (a c), grown different           previous conclusions. Mutants Nos- <br /> conditions. Samples A a, 02 limited, nitrate, KLaC* 0.02        phenotype, N20 reductase activity vitro an <br /> mmol 02 liter min; B b, 02 limited, nitrate; C          intact copper protein, exhibit enzymatic activity without <br /> and c, anaerobic, nitrate; D, anaerobic, N20. Each           protein observed. <br /> was filled 3 IlJ crude extract containing 45 FLg protein. Well      Within qualitative limits, major soluble <br /> ST contained 3 ,ul 0.9 ,ug purified N20 reductase.                  c-type cytochromes lost mutants, other <br />                                                                             partial denitrification reactions proceeded rates shown by <br /> from inhibitors (3, 36), isotope effects (32).The                  wild type. Cytochromes b c type been <br /> physiologically induced accumulation intermediate,                  proposed electron donation N20 reductase (1, 5, 22). <br /> however, restricted implementation                      If cytochromes electron transport <br /> copper deficiency growth media, caused transi-                   chain common respiratory processes, selection <br /> tory evolution N20 (16, 23). TnS-induced mutants P.                   procedure relying unaltered aerobic growth growth <br /> perfectomarina termination denitrification                 depending nitrate-to-N20 reduction suitable <br /> the level N20, exhibited phenotype identical strains             identify mutants. Alternatively, N20 reductase <br /> Pseudomonasfluorescens, biotypes D E, consti-                     able react type electron <br /> tutively express trait (13, 25). The molecular basis               donor, single transposition event suffice to <br /> the truncated denitrification P. fluorescens unknown.                 produce electron donor mutant. <br />   Mutants expected belong categories affected                   The N20 reductase protein synthesized by <br /> the structural gene enzyme electron donor,                   wild type mutants depended strongly the <br /> property related prosthetic group(s), regulatory             growth conditions. Especially, nitrate-grown cells, when <br /> function. Immunochemical identification, mo-                  compared grown N20, overproduced struc- <br /> lecular size information gel permeation detergent                  tural protein. The wild type mutants group I re- <br /> electrophoresis, indicated group I Nos mutants                  sponded alike synthesis N20 reductase protein to <br /> continued synthesize dimeric apoprotein N20                       conditions influence expression this <br /> reductase. The mutants significantly lower copper                     enzyme. Specific enzyme activity intact cells, calculated <br /> content wild type, chromatographic                        basis immunochemically determined reductase <br /> separation metal associated apoprotein.                content, revealed large differences cells grown in <br /> Mutants group I defective chromophore syn-                     presence nitrate cells grown N20 a <br /> thesis processing, or, likely low Mr copper               low-oxygen tension. The cellular concentration en- <br /> protein present, copper uptake. Mutants group II                  zyme high, catalytic activity low, whenever <br />                                                                             nitrate present. In cell types opposite was <br />                                                                             found. One possibility account observation a <br />     TABLE 2. Relation growth mode, N20 reductase                    cellular mechanism regulating activity en- <br />  content, enzymatic activity wild-type Nos- strains           zyme. <br />                        P. perfectomarina <br />                           N20 reductase                                                        ACKNOWLEDGMENTS <br />                        contenta (% total     Whole cell         N20 <br />                              protein)                          reductase      We thank H. Bohme preparation antiserum. A Piihler, <br /> Growth conditions                               activity     sp act   R. Simon, W. Klipp kindly provided strains helpful advice. <br />                                                   wildtpeb,c <br />                        KD2    KD71 typeb <br />                                    Wild         wild   type&quot;   wild typed   We acknowledge valuable technical assistance A. <br />                                                                             Schwickerath. <br /> 02 limited,       2.1     2.8      2.3        12.8             9.3       This work supported Deutsche Forschungsgemein- <br />   N03-                                                                      schaft. <br /> 02 limited,            1.2     1.1      1.4        20.2            24.1 <br />   NO3                                                                                      LITERATURE CITED <br />                                                                              1. Alefounder, P. R., A. J. Greenfield, J. E. G. McCarthy, S. J. <br /> Anaerobic,        2.4     2.4      2.3        19.0            13.2         Ferguson. 1983. Selection organisation denitrifying elec- <br />   N03-                                                                          tron-transfer pathways Paracoccus denitrificans. Biochim. <br />                                                                                 Biophys. Acta 724:20-39. <br /> Anaerobic,                         0.2            4.5         37.5      2. Axelsen, N. H., J. Kr0ol, B. Weeke. 1973. A manual of <br />   N20                                                                           quantitative immunoelectrophoresis. Methods applications. <br />   a <br />     Apoprotein case mutants.                                          Scand. J. Immunol. 2(Suppl 1):47-59. <br />   b Average determinations.                                         3. Balderston, <br /> </body></html>