<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The EMBO Journal (2007) 26, 5083&#226;&#8364;&#8220;5092   |&amp;   2007 European Molecular Biology Organization | All Rights Reserved 0261-4189/07 <br />                                                                                                                                                   THE <br /> www.embojournal.org <br />                                                                                                                                              EMBO <br />                                                                                                                                               JOURNAL <br /> RAP-1 RAL-1/exocyst pathway coordinate <br /> hypodermal cell organization Caenorhabditis <br /> elegans <br />  <br />  <br /> Ester W Frische1, Wendy Pellis-van Berkel1,                                             cells stimulating formation cell&#226;&#8364;&#8220;cell contacts via <br /> Gijs van Haaften2, Edwin Cuppen2,                                                       E-cadherin adherens junctions (Hogan et al, 2004; Price <br /> Ronald HA Plasterk2, Marcel Tijsterman2,                                                et al, 2004). Also VE-cadherin, present endothelial cells, is <br /> Johannes L Bos1 Fried JT                                                            control Rap1 (Fukuhara et al, 2006). The current <br /> Zwartkruis1,*                                                                           view Rap1 acts locally different effectors mediate <br /> 1 <br />                                                                                         effect integrins cadherins. For example, Rap1- <br />   Department Physiological Chemistry, Centre Biomedical                          binding proteins RIAM, RAPL, TIAM VAV been <br /> Genetics, UMC Utrecht, Utrecht, The Netherlands 2Hubrecht <br /> Laboratory, Netherlands Institute Developmental Biology, Utrecht,                   invoked Rap1 effectors functioning enhancement of <br /> The Netherlands                                                                         cell&#226;&#8364;&#8220;matrix contacts, AF-6 Cdc42 prime <br />                                                                                         candidates regulators cadherin adherens junctions <br /> The small Ras-like GTPase Rap1 identified                                 (for review, Bos, 2005; Kooistra et al, 2007)). However, <br /> regulator integrin activation cadherin-mediated                                  Rap1 induce cell polarity suspension cells (Shimonaka <br /> cell&#226;&#8364;&#8220;cell contacts. Surprisingly, <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-6' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span>null mutants</span> RAP-1                              et al, 2003) polarizing activity Rap1 also <br /> Caenorhabditis elegans viable fertile. In synthetic                           contribute effects seen integrins cadherins. A <br /> lethal RNAi screen C. elegans rap-1 mutants,                                   second Ras-like GTPase involved cell migration Ral <br /> Ras-like GTPase ral-1 emerged seven genes                                     (Suzuki et al, 2000; Rosse et al, 2006). In active, GTP- <br /> specifically required viability. Depletion exoc-8                            bound form, Ral binds various effectors play specific <br /> sec-5, encoding putative RAL-1 effectors members                                roles migration. For example, Ral induce filopodia <br /> of exocyst complex, caused lethality rap-1                                  formation interacting actin-bundling protein <br /> mutants, did affect wild-type worms. The RAP-1                                  filamin (Ohta et al, 1999). Filopodia the <br /> and RAL-1/exocyst pathway appear coordinate                                      leading edge migrating cells outgrowing neurites. <br /> hypodermal cell movement elongation em-                                      Furthermore, active Ral bind Sec5 (Brymora et al, <br /> bryonic development. They mediate effect                                  2001) Exo84 (Moskalenko et al, 2003; Zhang et al, 2005), <br /> through targeting a-catenin homologue HMP-1                                  proteins present exocyst complex mediates <br /> lateral membrane. Genetic interactions RAP-                               targeting E-cadherin basolateral plasma membrane. <br /> 1 RAL-1/exocyst pathway act parallel                                 Binding Ral Sec5 Exo84, likely <br /> larval stages. Together data provide vivo evidence                             present distinct subcomplexes exocyst, mutually <br /> for exocyst complex downstream RAL-1 effector                               exclusive. Thus, Ral enhance formation a <br /> cell migration.                                                                         functional exocyst complex specific subcellular site by <br /> The EMBO Journal (2007) 26, 5083&#226;&#8364;&#8220;5092. doi:10.1038/                                     bringing different subcomplexes exocyst, <br /> sj.emboj.7601922; Published online 8 November 2007                                      containing Sec5 Exo84 (Jin et al, 2005). The exocyst <br /> Subject Categories: cell &amp; tissue architecture; development                             complex originally discovered yeast protein <br /> Keywords: adherens junction; embryogenesis; Ral; Rap1;                                  complex required polarized secretion tip newly <br /> synthetic lethality                                                                     formed buds (TerBush et al, 1996). In vertebrate cells, the <br />                                                                                         exocyst complex mediates polarized delivery membranes <br />                                                                                         Golgi apparatus recycling endosomes the <br />                                                                                         basolateral membrane, apical membrane <br />                                                                                         (Lipschutz Mostov, 2002; Prigent et al, 2003; Yeaman <br /> Introduction                                                                            et al, 2004). Among proteins targeted basolateral <br /> Cell migration important aspect various biological                             membranes exocyst complex LDL receptor <br /> processes like development immune responses. Several                                E-cadherin (Grindstaff et al, 1998; Langevin et al, 2005; <br /> members Ras-like family GTPases involved                           Shipitsin Feig, 2004). Targeting E-cadherin the <br /> control cell migration. For example, Rap1 affects migration                          basolateral membrane special importance epithelial <br /> by enhancing integrin-mediated cell&#226;&#8364;&#8220;matrix attachment                                cells establish cell&#226;&#8364;&#8220;cell contacts other <br /> many different cell types vitro (Caron et al, 2000;                                  epithelial cells. Once E-cadherin stabilized the <br /> Katagiri et al, 2000). In addition, Rap1 inhibit migration                          membrane homotypic interactions E-cadherin on <br />                                                                                         neighboring cells, help targeting exocyst <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> <br /> *Corresponding author. Department Physiological Chemistry, Centre                    lateral membrane, enhancing formation of <br /> for Biomedical Genetics, UMC Utrecht, Universiteitsweg 100, Utrecht                     cell&#226;&#8364;&#8220;cell contacts elaborating cell polarity. <br /> 3584 CG, The Netherlands. Tel.: &#195;&#190; 31 30 2533992; Fax &#195;&#190; 31 30 2539035;                       In vivo, requirement Ras-like GTPases cell <br /> E-mail: G.J.T.zwartkruis@umcutrecht.nl <br />                                                                                         migration clearly illustrated mutant <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> of <br /> Received: 27 April 2007; accepted: 19 October 2007; published                           Rap1 mutant flies. Embryos, devoid Rap1 protein, display <br /> online: 8 November 2007                                                                 defects ventral furrow closure, head involution and <br />  <br /> &amp; 2007 European Molecular Biology Organization                                                                       The EMBO Journal   VOL 26 | NO 24 | 2007 5083 <br />        RAP-1 RAL-1 hypodermal cell migration <br />       EW Frische et al <br />  <br />  <br />  <br />       migration mesodermal presumptive primordial germ            identified various genes involved the <br />       cells (Asha et al, 1999). Thus, morphogenetic events       RAL-1/exocyst pathway required viabi- <br />       complete cell layers individual cell migration      lity rap-1 mutants. <br />       disturbed. Also migration group specialized follicle <br />       cells, called border cells, control Ras-like <br />       GTPases Drosophila. Migration border cells represents        Results <br />       cadherin-dependent mode cell movement requires            RAP-1 RAP-2 required C. elegans <br />       signaling Ras Ral (Lee et al, 1996). It presently   embryonic development <br />       unclear Ral effector required process.            In addition previously described rap-1(pk2082) allele, <br />          A major migration event Caenorhabditis elegans       second rap-1 allele (tm861) isolated, a <br />       hypodermal cells embryonic development.                  549 bp deletion removes exon 3 exon 4, leaving <br />       Hypodermal cells epidermal cells C. elegans.            42 amino acids RAP-1 intact (Figure 1). Both <br />       They arise series rows dorsal       homozygous mutant strains viable fertile, although <br />       embryo. The dorsal-most arrays cells undergo              progression larval stages delayed (Pellis-van Berkel <br />       process called dorsal intercalation, organizes          et al, 2005; data shown). Like rap-1(pk2082);rap- <br />       single dorsal row. After process begun,        2(gk11) animals, majority rap-1(tm861);rap-2(gk11) ani- <br />       ventral-most cells start migrate ventrally      mals die late larval stages. The small fraction double <br />       envelope embryo. In process ventral enclosure,     homozygous adults, derived rap-1(pk2082/ &#195;&#190; ); <br />       anteriorly located cells, named leading cells,        rap-2(gk11) rap-1(tm861/ &#195;&#190; );rap-2(gk11) animals, egg- <br />       crucial establish cell&#226;&#8364;&#8220;cell contacts   laying defective. Surprisingly, inside animals viable off- <br />       contralateral neighbors (Williams-Masson et al, 1998;        spring present. We, therefore, isolated embryos N2 or <br />       review Simske Hardin, 2001). In dorsal         rap-1(tm861);rap-2(gk11) animals determined percen- <br />       ventral cells, row lateral epidermal cells, called seam       tage non-hatched embryos. Although percentage non- <br />       cells, present. After ventral enclosure completed,       hatched rap-1(tm861);rap-2(gk11) embryos higher (19%, <br />       ovoid embryo elongates fourfold. This process relies      n &#194;&#188; 340) compared wild-type embryos (3%, n &#194;&#188; 365), <br />       circumferentially oriented actin bundles linked        majority double mutants hatched normally. Of the <br />       apical site adherens junctions hypodermal cells.         embryos did hatch, majority (73%) elongated <br />       In embryos mutant a-catenin homologue HMP-1             twofold. This showed RAP-1 RAP-2 <br />       b-catenin homologue HMP-2, circumferential actin           required normal viability late larval stages, no <br />       bundles dorsal hypodermal cells detach adherens            strict requirement RAP-1 RAP-2 embryogenesis. <br />       junctions, probably explains elongation phenotype <br />       seen animals (Costa et al, 1998). <br />                                                                          Synthetic lethal screen rap-1 mutants <br />          So far, relatively gene products identified, <br />                                                                          The normal development rap-1 mutants C. elegans <br />       play crucial role ventral migration hypodermal <br />                                                                          contrasts situation Drosophila suggests that <br />       cells. Among HMP-1, HMP-2 cadherin HMR- <br />                                                                          signaling pathways functionally compensate the <br />       1, physically interacting cadherin&#226;&#8364;&#8220;catenin <br />       complex (CCC). In HMR-1 homozygous embryos, embryos <br />       devoid maternal HMP-1 -2 protein, leading cells             1                  5                           4 <br />                                                                                      2                6                   3 <br />       migrate completely ventral (Costa et al, 1998). <br />       Other proteins involved APC-related protein APR-1 <br />       present CCC, inositol 1,4,5- <br />       triphosphate receptor ITR-1 ephrin receptors ligands <br />       (George et al, 1998; Hoier et al, 2000; Thomas-Virnig et al, <br />                                                                                                       tm861        * <br />                                                                                                                 pk2082 <br />       2004). These proteins required neuroblast                  C27B7 <br />                                                                                               29 000                   28 000 <br />       cells, hypodermal cells migrate. Also the <br />       cytoskeletal regulatory WAVE/WASP proteins required <br />       normal hypodermal cell migration (Withee et al, 2004). <br />       The interconnectivity different genes been <br />       established. Remarkably, role C. elegans <br />       integrins established hypodermal cell migration <br />       (Cox Hardin, 2004). <br />          In present study, performed synthetic lethal      Figure 1 Schematic representation genomic organization of <br />       RNAi screen C. elegans rap-1 mutant, revealed         rap-1 locus detection mutant alleles. Filled boxes indicate <br />       rap-1 mutants highly sensitive diminished signal-      exons line underneath graphic representation indicates <br />       ing RAL-1/exocyst pathway. During embryogenesis,           position deletion rap-1(tm861) animals star <br />                                                                          indicates amber codon (amino acid 130) pk2082 allele. <br />       Ras-like GTPases RAP-1 RAL-1 act concert             Numbers indicate position rap-1 locus C27B7 <br />       orchestrate hypodermal cell migration sorting.                 cosmid. Below PCR reactions shown, performed single-worm <br />       Interfering Ral-1/exocyst pathway rap-1 mutant         lysates demonstrate presence wild-type allele in <br />       background leads loss CCC adherens junctions.          &#195;&#190; / &#195;&#190; &#195;&#190; /  worms absence wild-type allele in <br />                                                                           /  worms. An arrowhead indicates 170 bp Sau3A product that <br />       Interestingly, observed phenotype severe               characteristic pk2082 allele. Arrows indicate position <br />       CCC mutants alone, indicating additional roles        PCR primers used detection deletion wild-type <br />       Ral-1/exocyst pathway. Furthermore, screen          alleles. <br />  <br />  <br /> 5084 The EMBO Journal VOL 26 | NO 24 | 2007                                                   &amp; 2007 European Molecular Biology Organization <br />                                                                                                              RAP-1 RAL-1 hypodermal cell migration <br />                                                                                                                                        EW Frische et al <br />  <br />  <br />  <br /> rap-1 pathway. To investigate option, genome-wide                  exoc-7. Although synthetic lethal <br /> synthetic lethal RNAi screen performed using                      screen, quantification showed progeny sec- <br /> Ahringer library (Kamath et al, 2003; van Haaften et al,                  10(RNAi). Finally, sec-5(RNAi) resulted specific <br /> 2004). Apart wild type rap-1(pk2082) mutants,                    synthetic lethality rap-1 alleles. <br /> rap-2(gk11) epac-1(pk1313) mutants included                      In vertebrates, Sec5 identified member the <br /> this screen. epac-1(pk1313) mutants carry deletion               exocyst complex direct downstream effector <br /> homologue cAMP-dependent Rap-specific guanine                      Ral. Other Ral effectors include RalBP, involved endo- <br /> nucleotide exchange factor (GEF) Epac (T20G5.5; unpub-                    exocytosis, actin-bundling protein filamin. For <br /> lished results) (de Rooij et al, 1998). Foods caused                 effectors, single homologue the <br /> specific lethality mutant strains, re-               C. elegans genome (rlbp-1 (T23G11.5), flna-1 (C23F12.1)). In <br /> screened using mutants case rap-2                  addition, single homologue family RALGEFs is <br /> epac-1 rap-1(pk2082) rap-1(tm861) (see                   found, RGL-1 (F28B4.2). We tested effect RNAi feeding <br /> Supplementary data). In rescreen, genes emerged                   L1 larvae plate genes. ral-1 and <br /> that caused clear reproducible synthetic lethality               sec-5(RNAi) prevented rap-1(pk2082) producing viable <br /> rap-2 epac-1. In contrast, seven genes caused specific                 offspring, rlbp-1(RNAi) effect. flna-1(RNAi) <br /> synthetic lethality rap-1 mutants (Table I). These include             resulted high number sick animals, also <br /> the MKP7 homologue vhp-1 (F08B1.1)(Mizuno et al, 2004)                    seen wild-type animals (Supplementary Figure 1). <br /> and gene unknown function C01B7.1. Interestingly,                  Together, results suggest functionality the <br /> ral-1 exoc-8 encode homologues                     ral-1/sec-5/exoc-8 pathway required rap-1 mutants, but <br /> vertebrate exocyst complex members. Finally, genes                    wild-type worms. Further evidence comes <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> encoding phosphatase sur-6 (Sieburth et al, 1999),                observation rgl-1(RNAi) reduces number <br /> vesicle-sorting protein phi-24 (Howard et al, 2001) him-3, <br /> known involved meiosis, gave synthetic lethality, but <br /> with variable results rap-1(tm861). To exclude possi-             Table II rap-1(tm861); dpy-20 animals carrying transgenic <br /> bility additional <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span>mutation</span> present rap-1 strains          rap-1 (rescue 1 2) sensitive exoc-8(RNAi) than <br />                                                                           rap-1(tm861); dpy-20 animals <br /> was responsible observed synthetic lethality, two <br /> independent rap-1(tm861) strains, rap-1 control                                      Control RNAi            exoc-8         Control RNAi <br /> of general let-858 promoter, generated. Although                                                          (RNAi)-1            (%) <br /> level RAP-1 low detection blot, observed             Wild type                    62                 70                  113 <br /> a partial rescue phenotype worms                   rap-1(tm861);                57                 10                   17 <br /> subjected exoc-8(RNAi) food L1-stage animals (Table II).            dpy-20 <br />    We focused RAL-1 exocyst member EXOC-8,                  Rescue 1                     61                 36                   59 <br />                                                                           Rescue 2                     65                 86                  132 <br /> their vertebrate homologues RalA Exo84 previously <br /> been reported directly interact: active GTP-bound               Numbers second column absolute numbers of <br /> form, RalA binds Exo84. Therefore, retested effects             viable progeny 16-h egg lay. Numbers mean value of <br />                                                                           progeny hermaphrodites (n &#194;&#188; 15 rescue strain 1 2, n &#194;&#188; 10 <br /> of RNAi exocyst complex members viability              wild-type rap-1(tm861); dpy-20). The column shows <br /> both rap-1 mutants (Supplementary Table I). No               percentage progeny exoc-8(RNAi) in <br /> limited effects scored sec-6, sec-8, sec-15, sec-3               comparison animals subjected control RNAi. <br />  <br />  <br /> Table I Overview genes synthetic lethal screen rap-1(pk2082) <br /> Localization      Gene              Mammalian homologue                  N2         rap-1 (pk2082)              rap-2 (gk11)         epac-1 (pk1313) <br />  <br /> 90G7              ral-1             RalA, RalB                      21 (549)               0   (0)                36   (702)             19   (344) <br />                   Y53G8AR.3                                         45 (1068)              0   (0)                23   (408)             36   (640) <br /> 25G8              exoc-8            EXOC8                           38 (984)               2   (26)               55   (1087)            21   (377) <br />                   Y105E8B.2                                         55 (1314)             14   (189)              69   (1204)            97   (1721) <br /> 2C2               phi-24            CHMP1B                         146 (2198)             22   (433)              93   (2300)           139   (2699) <br />                   F23C8.6                                           88 (1038)             30   (256)              83   (1141)            50   (813) <br />                                                                     26 (826)               9   (158)              68   (1200)                 &#226;&#8364;&#8221; <br /> 16E8              sur-6             PR55/B                          59 (892)               4   (75)               44   (1101)            47   (912) <br />                   F26E4.1                                           47 (550)               0   (0)                51   (696)             53   (614) <br />                                                                     47 (1533)              1   (13)               55   (973)             46   (740) <br /> 44D3              vhp-1             MKP7                            91 (2165)              0   (0)                38   (935)            163   (2350) <br />                   F08B1.1                                          166 (3150)              8   (72)               24   (327)             95   (2282) <br /> 101G7             him-3             HORMAD1                         54 (819)              14   (276)              58   (1424)            51   (984) <br />                   ZK381.1                                          108 (1259)             35   (296)              68   (1203)           126   (1472) <br />                                                                    107 (3479)              7   (121)                   &#226;&#8364;&#8221;                 55   (884) <br /> 104B8                               ARP10                          137 (2068)             27   (529)              79   (1934)           109   (2129) <br />                   C49H3.8                                           74 (2393)             80   (681)             100   (1378)           123   (1437) <br />                                                                        &#226;&#8364;&#8221;                  35   (583)              71   (1257)           111   (1785) <br /> 146B2                               Zn-finger                      117 (1760)              9   (168)              62   (1545)            99   (1928) <br />                   C01B7.1           Protein                        102 (1208)              7   (64)               96   (1319)            90   (1054) <br />                                                                     43 (1373)                  &#226;&#8364;&#8221;                  17   (299)             20   (330) <br /> Numbers represent percentage progeny relative control RNAi strain determined rescreening experiments. <br /> Numbers brackets total numbers worms day 7. Successive lines represent independent experiments. <br />  <br /> &amp; 2007 European Molecular Biology Organization                                                         The EMBO Journal         VOL 26 | NO 24 | 2007 5085 <br />        RAP-1 RAL-1 hypodermal cell migration <br />       EW Frische et al <br />  <br />  <br />  <br />       rap-1 offspring, effect seen wild-type                                       junctions characterized consist <br />       worms (Supplementary Figure 1A; data shown). A draw-                                          single electron-dense junction, apical and <br />       RNAi studies efficacy RNAi                                          basal domain discerned immunofluorescence. The <br />       variable. However, set RNAi constructs                                       apical domain junction harbors CCC, the <br />       tested RNAi-sensitive background using rrf-3                                        basal domain contains membrane-associated protein <br />       strain, similar results observed (Supplementary Figure                                       DLG-1 binding partner AJM-1. We used DLG-1 and <br />       1B, C). Strikingly, reduction seen number                                    AJM-1 localization visualize adherens junctions the <br />       rap-1;rrf-3 mutants rgl-1(RNAi).                                                            hypodermis developing gut, allows simulta- <br />                                                                                                         neous evaluation development tissues relative <br />       Comparative analysis ral-1/sec-5/exoc-8(RNAi)                                                  (Segbert et al, 2004). The <span id='am-394' about='xsp:pattern' typeof='owl:Thing'>pattern</span> organiza- <br />       rap-1 mutants                                                                                  tion hypodermal cells rap-1 mutant worms control <br />       To obtain definitive proof ral-1, sec-5 exoc-8                                      RNAi identical wild-type worms dorsal, <br />       operate pathway C. elegans, compared                                        seam ventral cells form regular arrays cells (Figure 2A; <br />       phenotypes rap-1 mutants RNAi feeding plate.                                          reviewed Simske Hardin, 2001). In contrast, of <br />       When rap-1 L1 larvae subjected RNAi                                       class I embryos obtained ral-1, sec-5 exoc-8(RNAi) <br />       genes, animals make adulthood sluggish                                         characterized failure cells align stereo- <br />       largely sterile. In contrast, L4 larvae used                                     typed pattern dorsal, seam ventral cells (Figure 2B&#226;&#8364;&#8220;D; <br />       start RNAi, resulting adults appeared healthy.                                         Supplementary Figure 2). 3D-animation indicates this <br />       Initially, produce viable offspring start                                        results defects dorsal intercalation ventral <br />       shed embryos hatch. Inspection                                          migration (data shown). In class I embryos, migra- <br />       embryos showed phenotypic series roughly                                            tion appears stopped reaching ventral <br />       classified types (Figure 2). In class I embryos,                                     midline. Analysis pharyngeal intestinal cells these <br />       hypodermis covers entire embryo, little morpho-                                     embryos showed tissues affected (Figure <br />       genesis seen following comma stage. Class II embryos                                       2F, G), rare cases abnormalities seen <br />       appeared disorganized hypodermal cells clus-                                         (Figure 2H). Overall, phenotypes ral-1, sec-5 exoc-8 <br />       tered envelope embryo. Hypodermal cell                                             (RNAi) similar. In class II embryos, DLG-1 and <br />  <br />  <br />  <br />                                                                L4440                  ral-1(RNAi)-1          sec-5(RNAi)-1             exoc-8(RNAi)-1 <br />                                  Hypodermal cells <br />  <br />  <br />  <br />  <br />                                                            A           DLG-1    B               DLG-1   C               DLG-1     D               DLG-1 <br />                             Class I <br />  <br />  <br />  <br />  <br />                                                            E           DLG-1    F               DLG-1   G               DLG-1     H               DLG-1 <br />                                         Intestinal cells <br />  <br />  <br />  <br />  <br />                                                                        Hypodermal cells                                      Intestinal cells <br />  <br />                                                            I           DLG-1    J               AJM-1    K              DLG-1      L               AJM-1 <br />                             ral-1(RNAi)-1 <br />                               Class II <br />  <br />  <br />  <br />  <br />       Figure 2 Phenotype rap-1 mutant embryos carrying dlg-1HGFP gene subjected control, ral-1, sec-5 exoc-8(RNAi). Phenotypes of <br />       rap-1 mutant embryos carrying dlg-1HGFP marker (FZ271) derived animals subjected RNAi ral-1, sec-5 exoc-8. DLG- <br />       1 indicates dlg-1HGFP expression, AJM-1 indicates immunofluorescence staining using MH27 antibody. Genes used RNAi indicated <br />       panel. L4440 vector control RNAi. (A&#226;&#8364;&#8220;H) class I embryos, (I&#226;&#8364;&#8220;L) class II embryos. (A&#226;&#8364;&#8220;D) (I&#226;&#8364;&#8220;J) focal <br />       planes visualize hypodermal cells, (E&#226;&#8364;&#8220;H) (K&#226;&#8364;&#8220;L) focal planes level gut. The anterior posterior visible seam cells <br />       marked *. In cases, AJM staining performed (only shown class II, subjected ral-1 (RNAi)), demonstrated clear <br />       colocalization DLG-1HGFP). Class I embryos characterized halted migration hypodermal cells examplified ral-1(RNAi) <br />       embryo (arrow B) misalignment hypodermal cells shown sec-5 exoc-8(RNAi) embryo (C, D). Halted migration be <br />       discerned early stages migration comparison developmental stage gut. Note normal hypodermal cell <br />       migration (A, E), gut extended entire <span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span> embryos, contrast gut ral-1(RNAi) embryo, which <br />       established clear adherens junctions shows mature pharynx (B, F). In cases, gap intestinal pharyngeal <br />       cells observed (arrowhead H). In class II embryos, hypodermal cells envelope entire embryo (arrow I, J) clearly recognizable <br />       gut structure outside embryo (arrowhead K). All embryos oriented pharynx left side. <br />       Scale bar: 10 mm. <br />  <br />  <br /> 5086 The EMBO Journal VOL 26 | NO 24 | 2007                                                                                  &amp; 2007 European Molecular Biology Organization <br />                                                                                                   RAP-1 RAL-1 hypodermal cell migration <br />                                                                                                                             EW Frische et al <br />  <br />  <br />  <br /> AJM-1 staining showed presence disorganized patches            EEA-1 (early endosome) RAB-11 (recycling endosome) <br /> of hypodermal cells various sites, proteins         staining (Andrews Ahringer, 2007; Poteryaev et al, 2007; <br /> colocalized (Figure 2I&#226;&#8364;&#8220;L, sec-5 exoc-8(RNAi); data            Supplementary Figure 3). Genetic disruption CCC mem- <br /> shown). The intestine embryos frequently                 bers shown result elongation defects. We, <br /> recognizable exterior, indicating despite             therefore, performed limited time lapse analysis rap-1 <br /> severe morphogenetic defect, cell specification tissue        mutants ral-1(RNAi). Elongation embryos is <br /> had occurred normally (arrowhead Figure 2K). To study cell            completely blocked, development the <br /> specification hypodermal cells class II embryos, used        intestine pharynx disturbed (Figure 5). Loss of <br /> the seam cell marker SCMHGFP, specifically ex-               HMR-1 maternal embryonic HMP-1 -2 causes <br /> pressed 10 hypodermal seam cells lateral         migration defects hypodermal cells, been <br /> the 1.5-fold stage onwards (Koh Rothman, 2001). GFP-              reported cause dorsal intercalation defects result in <br /> positive cells detected aberrantly localized hypo-        abnormal alignment dorsal, lateral ventral cells <br /> dermal cell clusters following ral-1, sec-5 exoc-8(RNAi)           (Costa et al, 1998). In respect, defects seen rap-1 <br /> feeding rap-1(tm861) animals, showing specification           mutants ral-1, sec-5 exoc-8(RNAi) appear more <br /> of subset hypodermal cell fates occurred         severe. Possibly, deletion rap-1 enhances phenotype <br /> these embryos (Figure 3, showing single focal plane).               hmr-1 mutants. Alternatively, RAL-1/exocyst pathway <br /> However, number SCMHGFP-positive cells re-                 targets proteins CCC membrane, <br /> duced cases. Whether consequence               involved cell migration. To investigate this, <br /> altered cell division, defects specification loss cell       constructed hmr-1;rap-1 double mutants. Double homo- <br /> death presently clear. In conclusion, phenotype         zygous embryos, derived rap-1(tm861);hmr-1(zu389/ <br /> rap-1 mutants ral-1, sec-5 exoc-8(RNAi) consistent           &#195;&#190; ) hermaphrodites did differ derived from <br /> with fact genes operate single       hmr-1(zu389/ &#195;&#190; ) hermaphrodites (Supplementary Figure 4). <br /> pathway involved hypodermal cell organization.                     Due fragility hmr-1 embryos, able to <br />                                                                       stain sufficient numbers embryos adherens junction <br /> Localization HMP-1                                                 markers. As loss maternal HMP-1 protein hmp-1 <br /> Certain aspects RNAi phenotypes like arrested              homozygous embryos strongly enhances phenotype to <br /> migration hypodermal cells consistent                resemble hmr-1 embryos, investigated the <br /> defect CCC function. We, therefore, investigated           effect loss RAP-1 hmp-1(zu278) embryos. Again, <br /> HMP-1 normally located adherens junctions. HMP-1               double homozygous embryos discerned from <br /> colocalized DLG-1 wild type control, ral-1, sec-5          hmp-1 mutants (Supplementary Figure 4). Together, these <br /> or exoc-8 (RNAi) rap-1 mutants control RNAi.                   results indicate aberrant organization hypodermal <br /> In contrast, rap-1 mutant embryos ral-1,             cells rap-1 embryos seen ral-1, exoc-8 sec-5(RNAi) <br /> sec-5 exoc-8(RNAi), diffuse distribution HMP-1            simple compound phenotype rap-1 members <br /> observed (Figure 4). So gene products required          CCC. <br /> targeting HMP-1 adherens junctions stabilizing it <br /> there. This diffuse staining clearly distinct of <br />                                                                       Genetic interactions RAP-1 RAL-1/ <br />                                                                       exocyst pathway <br />                                                                       To obtain additional proof genetic interactions between <br />                L4440                     ral-1(RNAi)-1 <br />                                                                       RAP-1 RAL-1 pathways C. elegans, studied <br />       A AJM-1 Seam nuclei         B AJM-1 Seam nuclei                 independent mutants. sec-5(pk2357) sec-5(pk2358) <br />                                                                       carry amber mutation amino acid position 389 <br />                                                                       obtained target selected mutagenesis (Cuppen <br />                                                                       et al, 2007; Figure 6). Animals homozygous sec-5(pk2357) <br />                                                                       sec-5(pk2358) adult produce embryos <br />                                                                       elongate normally progress L2 stage. <br />                                                                       These sec-5 mutations hypomorphic based fact that <br />            sec-5(RNAi)-1                 exoc-8(RNAi)-1               placed deficiency (mnDf67) adults not <br />                                                                       produce elongated embryos. The sec-5(tm1443) allele most <br />       C AJM-1 Seam nuclei          D AJM-1 Seam nuclei                likely null, 385 bp deletion removing two <br />                                                                       exons (Figure 6). Indeed, sec-5(tm1443) <br />                                                                       mutants severe phenotype: homozygotes never <br />                                                                       produce progeny die late L4-stage larvae. <br />                                                                          When introduced rap-1(tm861) mutant background, <br />                                                                       phenotype sec-5 alleles clearly enhanced, <br />                                                                       showing RAP-1 SEC-5 act different pathways. sec- <br /> Figure 3 Phenotype rap-1 mutant embryos carrying               5(pk2357);rap-1(tm861) double homozygotes arrested late <br /> SCMHGFP gene subjected control, ral-1, sec-5 exoc-8(RNAi)       L3 early L4 larvae (n &#194;&#188; 23, confirmed PCR). Importantly, <br /> (A&#226;&#8364;&#8220;D respectively). Expression seam cell marker SCMHGFP        sec-5(tm1443);rap-1(tm861) die L2 (25%) L3 (72%) <br /> in nuclei disorganized hypodermis class II embryos. <br /> Hypodermal cell borders visualized using AJM-1 antibody <br />                                                                       stage. This phenotype severe rap-1(tm861) <br /> (red). Only focal plane shown condition, seam nuclei   animals sec-5(RNAi). This likely maternal <br /> also present slices (data shown). Scale bar: 10 mm.      input SEC-5, sec-5(tm1443) homozygous embryos shed <br />  <br /> &amp; 2007 European Molecular Biology Organization                                               The EMBO Journal      VOL 26 | NO 24 | 2007 5087 <br />        RAP-1 RAL-1 hypodermal cell migration <br />       EW Frische et al <br />  <br />  <br />  <br />                                                                        Wild type                             rap-1(tm861) <br />                                                       A             DLG-1      B       HMP-1   C          DLG-1      D         HMP-1 <br />  <br />  <br />  <br />  <br />                                      L4440 <br />                                                           *            * <br />  <br />  <br />                                                       E             DLG-1      F       HMP-1   G             DLG-1 H           HMP-1 <br />                                      ral-1(RNAi)-1 <br />  <br />  <br />  <br />                                                           * <br />                                                                            * <br />  <br />  <br />  <br />                                                       I             DLG-1      J       HMP-1   K             DLG-1 L           HMP-1 <br />                                      sec-5(RNAi)-1 <br />  <br />  <br />  <br />  <br />                                                               *        * <br />                                      exoc-8(RNAi)-1 <br />  <br />  <br />  <br />  <br />                                                       M             DLG-1      N       HMP-1   O             DLG-1 P           HMP-1 <br />  <br />  <br />                                                               *        * <br />  <br />  <br />       Figure 4 Localization HMP-1 wild-type rap-1 mutant embryos subjected control, ral-1, sec-5 exoc-8(RNAi). dlg-1HGFP HMP-1 <br />       staining lateral view wild type carrying dlg-1HGFP (FZ224) (A, B, E, F, I, J, M, N) rap-1 mutant embryos (FZ271) (C, D, G, H, K, L, <br />       O, P), derived animals subjected control L4440 (A&#226;&#8364;&#8220;D), ral-1(RNAi) (E&#226;&#8364;&#8220;H), sec-5(RNAi) (I&#226;&#8364;&#8220;L) exoc-8(RNAi) (M&#226;&#8364;&#8220;P). Pictures <br />       hypodermal cells DLG-1 indicates dlg-1HGFP expression HMP-1 indicates immunofluorescence staining using P1E11 <br />       antibody. The anterior posterior visible seam cells marked *. Scale bar: 10 mm. <br />  <br />  <br />  <br />                                  0 min                            102 min          193 min         261 min                     376 min <br />  <br />                                                                                                                                                 DLG-1 <br />                  L4440 <br />                  ral-1(RNAi)-1 <br />  <br />  <br />  <br />  <br />                                                                                                                                                 DLG-1 <br />  <br />  <br />  <br />  <br />       Figure 5 Time-lapse analysis rap-1 mutant embryos subjected control ral-1(RNAi). Time lapse recordings embryos derived from <br />       dlg-1HGFP;rap-1(tm861) mutant (FZ271) embryos control L4440 (upper panels) ral-1(RNAi) (lower panels). Last timepoint shows <br />       DLG-1HGFP expression gut embryo. <br />  <br />  <br />  <br />       heterozygous animals sec-5(RNAi) die embryo (data                               (Moskalenko et al, 2002; Feig, 2003). Expression either <br />       shown).                                                                              mutant protein (Figure 7A) did severely affect these <br />          For ral-1, did identify mutants stop codon.                             transgenic animals, took slightly longer reach <br />       As alternative, generated transgenic animals,                             adulthood compared heat shocked, wild-type worms. <br />       dominant negative RAL-1(S31N) constitutively                                  In contrast, expression RAL-1 mutant proteins rap-1 <br />       GTP-bound RAL-1(G26V) control heat-                                   mutant background induced clear larval arrest, was <br />       shock promoter. Despite constitutively GTP-bound,                                  strongest RAL-1(G26V) (Figure 7B, C). Notably, whereas <br />       mutations corresponding RAL-1(G26V) (like Ral G23V                                 expression HA-RAL-1(G26V) delays development in <br />       mammals), described interfering mutants                                     wild-type worms, rap-1 worms expressing protein <br />  <br /> 5088 The EMBO Journal VOL 26 | NO 24 | 2007                                                                       &amp; 2007 European Molecular Biology Organization <br />                                                                                                                         RAP-1 RAL-1 hypodermal cell migration <br />                                                                                                                                                   EW Frische et al <br />  <br />  <br />  <br />        A <br />               B <br />                               E                     D                                        Discussion <br />                                F                   C <br />                                                                                              Synthetic lethal screens efficient approach towards <br />                                                                                              identification signaling pathways function redun- <br />                                                                                              dant fashion mediate critical functions development <br />                               tm1443           * <br />                                              pk2357                                          stage organism&#226;&#8364;&#8482;s life cycle (Tong et al, <br />                                              pk2358 <br />                                                                                              2001; van Haaften et al, 2004; Withee et al, 2004). Here, we <br />       T23G7                                                                                  used RNAi feeding-based method screen for <br />                                12 000                   10 000                   8 000 <br />                                                                                              signaling routes function conjunction RAP- <br /> Figure 6 Schematic representation genomic organization of <br />                                                                                              1 pathway C. elegans. Seven genes identified, whose <br /> the sec-5 locus. Filled boxes indicate exons line underneath <br /> the graphic representation indicates position deletion                         function required distinct rap-1 mutants to <br /> sec-5(tm1443) animals star indicates stop codon (amino                           produce viable offspring. These genes synthetic lethal <br /> acid 389) pk2357 pk2358 allele. Numbers indicate                              independent rap-1 mutations, strongly suggesting <br /> position rap-1 locus T23G7 cosmid. Arrows indicate <br />                                                                                              mutations rap-1 closely linked mutation <br /> the position PCR primers used detection deletion <br /> and wild-type alleles.                                                                       responsible observed effect. Indeed, obtained a <br />                                                                                              partial rescue phenotype exoc-8(RNAi) lines <br />                                                                                              expressing rap-1 heterologous promoter. Synthetic <br />                                                                                              lethality ral-1(RNAi) generally robust, was <br />    A                                   N2    hsp::ralG26V hsp::ralS31N                       rescued, contributed low levels RAP-1 <br />                               &#226;&#8364;&#8220;     0 6 18 &#226;&#8364;&#8220; 0 6 18 &#226;&#8364;&#8220; 0 6 18 Time (h)                        expression transgene (data shown). High levels <br />                                                                                              RAP-1 expression appear tolerated C. elegans <br />                                                                                              seen Rap1 tissue culture cells (data not <br />    B                    100                                                                  shown). <br />                                                                                                 Not genes function pathway (see <br />    % counted worms <br />  <br />  <br />  <br />  <br />                         80 <br />                                                                                              below), based literature, likely ral-1, <br />         strain <br />  <br />  <br />  <br />  <br />                         60                                                                   sec-5 exoc-8 so; vertebrate cells, active Ral directly <br />                                                                      N2 <br />                                                                                              binds exocyst members Sec5 Exo84 (Brymora et al, <br />                         40 <br />                                                                      rap-1(tm861) <br />                                                                                              2001; Moskalenko et al, 2003), results targeting of <br />                         20 <br />                                                                                              proteins, including E-cadherin, basolateral membrane <br />                                                                      hsp::HA-ral-1(G26V) <br />                                                                                              (Moskalenko et al, 2002; Shipitsin Feig, 2004). The fact <br />                                                                      rap-1(tm861) <br />                          0 <br />                                                                      [hsp::HA-ral-1(G26V)] <br />                                                                                              Ral binds Sec5 Exo84 mutually exclusive <br />                                   L1     L2    L3     L4   Adult <br />                                        Developmental stage                                   manner hint distinct pathways. In addition, Ral <br />                                                                                              collaborates Sec5 formation filopodia (Sugihara <br />    C                    100                                                                  et al, 2002). Analysis embryos derived rap-1 mutant <br />                                                                                              worms, grown ral-1, sec-5 exoc-8(RNAi), showed that <br />    % counted worms <br />  <br />  <br />  <br />  <br />                          80 <br />                                                                                              displayed virtually identical phenotypes. This strongly <br />         strain <br />  <br />  <br />  <br />  <br />                          60                                                                  suggests C. elegans genes function together <br />                                                                     N2                       hypodermal cell migration. <br />                          40 <br />                                                                     rap-1(pk2082)               Rap1 claimed activate Ral pathway in <br />                          20                                                                  Drosophila direct interaction RalGEF RGL (Mirey <br />                                                                     hsp::HA-ral-1(S31N) <br />                                                                                              et al, 2003). Also C. elegans RAP-1 RGL-1 interact a <br />                           0                                         rap-1(pk2082) <br />                                L1       L2     L3     L4            [hsp::HA-ral-1(S31N)]    yeast two-hybrid assay (J Riedl F Zwartkruis, unpub- <br />                                                            Adult <br />                                        Developmental stage                                   lished results). Although studies did exclude the <br />                                                                                              option RAP-1 contributes activation RAL-1 C. <br /> Figure 7 Effect heat-shock-driven expression RAL-1 proteins <br /> in wild-type rap-1 mutant backgrounds. (A) Western blot                               elegans, current hypothesis RAP-1 distinct <br /> total lysates L1 larvae. Larvae lysed heat shock                           function parallel RAL-1. This based fact <br /> ( ) 0 h (0), 6 h (6) 18 h (18) 2-h heat shock. Probing                      RAP-1 null mutants mild phenotype <br /> was 12CA5 antibody, recognizes HA-tag                             normal conditions, indicating RAP-1 major <br /> the N-terminus overexpressed proteins. (B, C) Embryos subjected <br /> to heat shock allowed develop 5 days determination                            activator RAL-1/exocyst pathway. Furthermore, inter- <br /> of developmental stage. Percentage animals different                               fering level RAL-1 expression mutant RAL-1 <br /> developmental stages total number hatched animals                               proteins strong effect rap-1 mutants wild- <br /> shown overexpression RAL-1(G26V) RAL-1(S31N) (B)                                  type animals. It possible RAP-1 converges RAL-1 <br /> and (C), respectively. <br />                                                                                              pathway level exocyst complex below. As no <br />                                                                                              interactions RAP-1 exocyst documen- <br />                                                                                              ted, favor option. In agreement this, we <br /> never reach adulthood. The effects expression Ral-                                     RAP-1 clearly enhances phenotype sec-5 <br /> 1(G26V) rap-2(gk11) background significantly                                   mutations. <br /> different observed wild-type worms, showing                                       The defects observed rap-1 embryos following ral-1, <br /> the specificity effect RAL-1(G26V) rap-1 animals                                sec-5 exoc-8(RNAi) prominent hypodermal cell <br /> (data shown). Together data RAP-1                                    migration ventral enclosure. It most <br /> and RAL-1/exocyst pathway interact stages                                  severely affected class II embryos gastrulation move- <br /> of life cycle.                                                                           ments abnormal, investigated any <br />  <br /> &amp; 2007 European Molecular Biology Organization                                                                     The EMBO Journal      VOL 26 | NO 24 | 2007 5089 <br />        RAP-1 RAL-1 hypodermal cell migration <br />       EW Frische et al <br />  <br />  <br />  <br />       detail. Various causes underlie improper alignment         HMP-1 result effects RAP-1 endocytosis, <br />       defective migration hypodermal cells. For example,          sorting events recycling endosomes exit from <br />       hypodermal cells need specified correctly accommo-        endosomes. The situation complicated the <br />       date correct position. Abnormal cell specification,       observation exocyst member Sec5 functions only <br />       leading defects dorsal intercalation, ventral enclosure      exocytosis, endocytosis (Sommer et al, 2005; <br />       elongation, proposed underlie phenotype        Sonnichsen et al, 2005). Indeed, currently unclear and <br />       apr-1 mutants (Hoier et al, 2000). However, cell specifica-     exocyst complex interacts recycling endo- <br />       tion unlikely cause migration defects ral-      somes. In respect, phi-24 interesting hit from <br />       1(RNAi), rap-1 embryos. First, class II embryos,           screen, protein homologous endosomal <br />       expression seam cell marker detected,          protein CHMP1 (Howard et al, 2001), involved in <br />       demonstrates specification           vesicular sorting. <br />       hypodermis occurred. Second, did observe               In contrast HMP-1, DLG-1 AJM-1 localized normally <br />       abnormal patterning hypodermal cells onset        adherens junctions consistent finding <br />       migration, contrast reported apr-1 mutants      localization proteins occurs indepen- <br />       (data shown). Finally, Ral-1/exocyst pathway               dently CCC (Costa et al, 1998; Koppen et al, 2001). In <br />       regulate migration tissue culture                  addition, shows RAL-1 RAP-1 pathways affect <br />       cells, unlikely depend cell fate specification      transport subclass proteins present the <br />       (Rosse et al, 2006).                                               basolateral membrane. Differential effects inhibition the <br />          A likely explanation observed phenotype         exocyst complex transport basolateral membrane <br />       inability hypodermal cells change adhesive         previously shown Drosophila, DE-cadher- <br />       properties support extensive reorganization        transport blocked delivery septate junction <br />       cell&#226;&#8364;&#8220;cell contacts required dorsal intercalation ventral   protein Coracle normal (Langevin et al, 2005). Importantly, <br />       migration. Cell&#226;&#8364;&#8220;cell adhesion fundamental importance         class I embryos differ hmr-1 embryos shape <br />       cell sorting, migration cell shape changes. This         alignment hypodermal cells clearly abnormal. <br />       exemplified Drosophila, loss DE-cadherin            Therefore, synthetic <span id='am-316' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-317' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-318' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-319' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-320' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-321' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-322' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-323' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-324' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-325' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-326' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-327' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-328' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-330' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-331' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-332' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-333' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-334' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-335' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-337' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-338' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-341' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-342' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-343' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-344' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-345' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-347' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-348' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-349' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-350' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-351' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-352' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-353' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-354' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-355' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-356' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-357' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-358' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-359' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-362' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-363' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-364' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-365' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-366' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-367' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-368' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-369' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-370' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-372' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-373' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-374' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-375' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-376' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-378' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-380' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-381' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-382' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-383' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-384' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-385' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-386' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-389' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-390' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-391' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-392' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-393' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span>lethal phenotype</span> observed be <br />       border cells blocks migration follicle cells dramatic   simply attributed loss CCC adherens junction. <br />       effects cell sorting (Lee et al, 1996; Niewiadomska et al,      Double mutant analysis demonstrates phenotype of <br />       1999; Pacquelet Rorth, 2005). Even loss DE-cadherin         hmr-1 hmp-1 embryos does severe rap-1 <br />       cell circumference, seen clones Rap1     simultaneously absent. Consequently, RAL-1/exocyst <br />       mutant cells following cell division wing, changes    pathway required targeting CCC, has <br />       sorting behavior (Knox Brown, 2002). Indeed, ana-          additional functions. <br />       lysis proteins present adherens junctions revealed          It interesting learn genes <br />       absence HMP-1 likely complete CCC             uncovered synthetic lethal screen fit RAL-1/ <br />       (Costa et al, 1998). This indicates cell adhesive proper-     exocyst pathway. At gene, vhp-1, indirectly <br />       ties changed. Loss HMP-1 adherens junctions           linked Ral signaling. VHP-1 phosphatase stress- <br />       interfering RAL-1/exocyst route line        induced kinases KGB-1 PMK-1, homologous to <br />       recent studies vertebrate tissue culture cells Drosophila   Jnk p38 MAPK, respectively (Mizuno et al, 2004). <br />       (Shipitsin Feig, 2004; Classen et al, 2005; Langevin et al,    Previously, Ral activate Jnk-pathway <br />       2005). It noted, however, C. elegans, loss    tissue culture cells (de Ruiter et al, 2000), in <br />       HMP-1 adherens junctions ral-1(RNAi)            Drosophila act negative regulator Jnk <br />       seen rap-1 worms wild-type worms. Thus,         (Sawamoto et al, 1999). Therefore, vhp-1 act together <br />       ral-1 depletion, RAP-1 crucial targeting CCC     RAL-1 processes, possibly different RAL-1 <br />       lateral membrane stabilizing it. In mammalian cells, Rap1       effectors. <br />       shown promote E-cadherin interaction newly             In conclusion, synthetic lethal screen used a <br />       forming adherens junctions tissue culture cells,     starting point learn genes, function a <br />       required maintenance mature junctions         redundant fashion RAP-1. Our data demonstrate <br />       (Hogan et al, 2004; Price et al, 2004). Possibly, Rap1 acts     functional overlap Ras-like GTPases RAP-1 and <br />       inhibiting endocytosis E-cadherin proteins,       RAL-1 various phases life cycle. Using RNAi <br />       ligated E-cadherin cells. To so, Rap1        time interference RAL-1/exocyst pathway, we <br />       suggested interact actin binding protein AF-6      identify GTPases novel elements C elegans hypo- <br />       p120 catenin, established stabilizer adherens junctions      dermal cell migration. <br />       (Hoshino et al, 2005). Although attractive explain the <br />       lack HMP-1 adherens junctions terms combined <br />       effect diminished targeting RAL-1/exocyst complex       Materials methods <br />       enhanced internalization, lack RAP-1, the <br />                                                                          Worms <br />       situation complex. It kept mind <br />                                                                          General methods culturing manipulating worms used were <br />       reorganization cell&#226;&#8364;&#8220;cell contacts, cadherin      described previously (Lewis Fleming, 1995). Worms were <br />       recycled endosomes (Bryant Stow, 2004). Indeed,            cultured NGM plates 201C. Strains, constructs detection of <br />       disturbing function endosomes Drosophila       mutants described Supplementary data. <br />       wing period cells organize            Transgenic animals obtained injection plasmid DNA <br />                                                                          gonads dpy-20(e1362) animals, rap-1(tm861);dpy- <br />       hexagonal arrays, results loss DE-cadherin selective    20(e1362) (Mello et al, 1991). FZ311 FZ312 with <br />       cell contact sites (Classen et al, 2005). Consequently, loss    pPD103.5 containing untagged cDNA sequence rap-1 (C27B7.8). <br />  <br /> 5090 The EMBO Journal VOL 26 | NO 24 | 2007                                                  &amp; 2007 European Molecular Biology Organization <br />                                                                                                      RAP-1 RAL-1 hypodermal cell migration <br />                                                                                                                                EW Frische et al <br />  <br />  <br />  <br /> Transgenic arrays integrated irradiating animals 40 Gy    2-h egg lay. Embryos mounted 3% agarose pad M9 &#195;&#190; <br /> of gamma radiation 137 Cs source (Way et al, 1991). Target-     buffer, covered coverslip sealed Vaseline. <br /> selected mutagenesis obtaining sec-5 ral-1 mutants was <br /> performed described previously (Cuppen et al, 2007). Mutant         Heat shock protocol Western blotting <br /> animals outcrossed times N2 animals       For heat shock experiments, hermaphrodites allowed lay <br /> phenotypic analysis. Determination viability N2              eggs 2 h (day 0). At 1 h removal adults, plates <br /> rap-1(tm861);rap-2(gk11) performed cutting gravid adults        incubated 331C 30 min. On days 1, 2 3 2-h heat <br /> in water collecting embryos drawn-out pipette.          shock given. Detection HA-tagged RAL-1 lysates of <br /> Embryos counted non-hatched embryos scored               heat-shocked animals immunoblotting using the <br /> after 24 h.                                                            12CA5mAb HA (Pellis-van Berkel et al, 2005). <br />    The synthetic lethal screen performed essentially as <br /> described previously (van Haaften et al, 2004). However, used       Supplementary data <br /> 20&#226;&#8364;&#8220;25 L1 larvae 100 ml M9 &#195;&#190; buffer, 50 ml   Supplementary data available The EMBO Journal Online <br /> induced bacterial suspension added. Scoring 7       (http://www.embojournal.org). <br /> days eye genome scale screen counting total <br /> amount progeny comparing of <br /> progeny control RNAi (L4440).                                       Acknowledgements <br />                                                                        We thank R Vulders technical assistance, Dr S Mitani rap- <br /> Antibody staining time-lapse recording                             1(tm861) sec-5(tm1443) mutant, Dr M Labouesse DLG- <br /> Antibody staining embryos performed described                1HGFP strain, Dr J Ahringer, Dr A Spang Dr B Grant for <br /> previously (Bossinger et al, 2001). Antibodies used are: a-HMP-1       antibodies. Some nematode strains used provided the <br /> (clone P1E11, Chemicon), a-AJM-1 (provided O Bossinger),            Caenorhabditis Genetics Center, funded NIH <br /> mabMH27, mouse, hybridoma supernatant), EEA-1 (provided B           National Center Research Resources (NCRR) thank <br /> Grant), RAB-11 (provided A Spang) Alexa anti-mouse          T Stiernagle sending these. We thank Dr O Bossinger for <br /> 568 secondary antibody (Jackson ImmRes Lab). Embryos              valuable discussions teaching antibody labeling embryos, <br /> visualized Zeiss Axiokop 2. For time-lapse recordings, L4      Dr A Fire making available heat shock constructs J Mudde <br /> animals RNAi feeding plate day 1. On day 3,          technical assistance target-selected mutagenesis. This work <br /> adults moved fresh plate embryos collected    supported CBG (EF, WP-vB, FZ). <br />  <br />  <br />  <br />  <br /> References <br /> Andrews R, Ahringer J (2007) Asymmetry early endosome               Fukuhara S, Sakurai A, Yamagishi A, Sako K, Mochizuki N (2006) <br />   distribution C. elegans embryos. PLoS ONE 2: e493                    Vascular endothelial cadherin-mediated cell&#226;&#8364;&#8220;cell adhesion regu- <br /> Asha H, Ruiter ND, Wang MG, Hariharan IK (1999) The Rap1               lated small GTPase, Rap1. J Biochem Mol Biol 39: 132&#226;&#8364;&#8220;139 <br />   GTPase functions regulator morphogenesis vivo. EMBO J     George SE, Simokat K, Hardin J, Chisholm AD (1998) The VAB-1 <br />   18: 605&#226;&#8364;&#8220;615                                                             Eph receptor tyrosine kinase functions neural epithelial <br /> Bos JL (2005) Linking Rap cell adhesion. Curr Opin Cell Biol 17:       morphogenesis C. elegans. Cell 92: 633&#226;&#8364;&#8220;643 <br />   123&#226;&#8364;&#8220;128                                                              Grindstaff KK, Yeaman C, Anandasabapathy N, Hsu SC, Rodriguez- <br /> Bossinger O, Klebes A, Segbert C, Theres C, Knust E (2001) Zonula         Boulan E, Scheller RH, Nelson WJ (1998) Sec6/8 complex is <br />   adherens formation Caenorhabditis elegans requires dlg-1,            recruited cell&#226;&#8364;&#8220;cell contacts specifies transport vesicle <br />   homologue Drosophila gene discs large. Dev Biol 230:         delivery basal-lateral membrane epithelial cells. Cell <br />   29&#226;&#8364;&#8220;42                                                                   93: 731&#226;&#8364;&#8220;740 <br /> Bryant DM, Stow JL (2004) The ins outs E-cadherin traffick-     Hogan C, Serpente N, Cogram P, Hosking CR, Bialucha CU, Feller <br />   ing. Trends Cell Biol 14: 427&#226;&#8364;&#8220;434                                       SM, Braga VM, Birchmeier W, Fujita Y (2004) Rap1 regulates the <br /> Brymora A, Valova VA, Larsen MR, Roufogalis BD, Robinson PJ               formation E-cadherin-based cell&#226;&#8364;&#8220;cell contacts. Mol Cell Biol 24: <br />   (2001) The brain exocyst complex interacts RalA               6690&#226;&#8364;&#8220;6700 <br />   GTP-dependent manner: identification novel mamma-               Hoier EF, Mohler WA, Kim SK, Hajnal A (2000) The Caenorhabditis <br />   lian Sec3 gene second Sec15 gene. J Biol Chem 276:                elegans APC-related gene apr-1 required epithelial cell <br />   29792&#226;&#8364;&#8220;29797                                                             migration Hox gene expression. Genes Dev 14: 874&#226;&#8364;&#8220;886 <br /> Caron E, Self AJ, Hall A (2000) The GTPase Rap1 controls functional    Hoshino T, Sakisaka T, Baba T, Yamada T, Kimura T, Takai Y (2005) <br />   activation macrophage integrin alphaMbeta2 LPS          Regulation E-cadherin endocytosis nectin afadin, <br />   inflammatory mediators. Curr Biol 10: 974&#226;&#8364;&#8220;978                           Rap1, p120ctn. J Biol Chem 280: 24095&#226;&#8364;&#8220;24103 <br /> Classen AK, Anderson KI, Marois E, Eaton S (2005) Hexagonal            Howard TL, Stauffer DR, Degnin CR, Hollenberg SM (2001) CHMP1 <br />   packing Drosophila wing epithelial cells planar cell          functions member newly defined family vesicle <br />   polarity pathway. Dev Cell 9: 805&#226;&#8364;&#8220;817                                   trafficking proteins. J Cell Sci 114: 2395&#226;&#8364;&#8220;2404 <br /> Costa M, Raich W, Agbunag C, Leung B, Hardin J, Priess JR (1998) A     Jin R, Junutula JR, Matern HT, Ervin KE, Scheller RH, Brunger AT <br />   putative catenin-cadherin mediates morphogenesis          (2005) Exo84 Sec5 competitive regulatory Sec6/8 effec- <br />   Caenorhabditis elegans embryo. J Cell Biol 141: 297&#226;&#8364;&#8220;308                 tors RalA GTPase. EMBO J 24: 2064&#226;&#8364;&#8220;2074 <br /> Cox EA, Hardin J (2004) Sticky worms: adhesion complexes C.         Kamath RS, Fraser AG, Dong Y, Poulin G, Durbin R, Gotta M, <br />   elegans. J Cell Sci 117: 1885&#226;&#8364;&#8220;1897                                      Kanapin A, Le Bot N, Moreno S, Sohrmann M, Welchman DP, <br /> Cuppen E, Gort E, Hazendonk E, Mudde J, van Belt J, Nijman IJ,         Zipperlen P, Ahringer J (2003) Systematic functional analysis <br />   Guryev V, Plasterk RH (2007) Efficient target-selected muta-            Caenorhabditis elegans genome using RNAi. Nature 421: <br />   genesis Caenorhabditis elegans: <span id='am-238' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-239' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-244' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-251' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-254' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-256' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-267' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-270' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-273' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-290' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span>knockout</span>          231&#226;&#8364;&#8220;237 <br />   gene. Genome Res 17: 649&#226;&#8364;&#8220;658                                         Katagiri K, Hattori M, Minato N, Irie S, Takatsu K, Kinashi T (2000) <br /> de Rooij J, Zwartkruis FJ, Verheijen MH, Cool RH, Nijman SM,              Rap1 potent activation signal leukocyte function-asso- <br />   Wittinghofer A, Bos JL (1998) Epac Rap1 guanine-nucleotide-        ciated antigen 1 distinct protein kinase C phosphatidy- <br />   exchange factor directly activated cyclic AMP. Nature 396:           linositol-3-OH kinase. Mol Cell Biol 20: 1956&#226;&#8364;&#8220;1969 <br />   474&#226;&#8364;&#8220;477                                                              Knox AL, Brown NH (2002) Rap1 GTPase regulation adherens <br /> de Ruiter ND, Wolthuis RM, van Dam H, Burgering BM, Bos JL                junction positioning cell adhesion. Science 295: 1285&#226;&#8364;&#8220;1288 <br />   (2000) Ras-dependent regulation c-Jun phosphorylation          Koh K, Rothman JH (2001) ELT-5 ELT-6 required continu- <br />   mediated Ral guanine nucleotide exchange factor-Ral              ously regulate epidermal seam cell differentiation cell <br />   pathway. Mol Cell Biol 20: 8480&#226;&#8364;&#8220;8488                                    fusion C. elegans. Development 128: 2867&#226;&#8364;&#8220;2880 <br /> Feig LA (2003) Ral-GTPases: approaching 15 minutes fame.      Kooistra MR, Dube N, Bos JL (2007) Rap1: key regulator in <br />   Trends Cell Biol 13: 419&#226;&#8364;&#8220;425                                            cell&#226;&#8364;&#8220;cell junction formation. J Cell Sci 120: 17&#226;&#8364;&#8220;22 <br />  <br /> &amp; 2007 European Molecular Biology Organization                                                  The EMBO Journal      VOL 26 | NO 24 | 2007 5091 <br />        RAP-1 RAL-1 hypodermal cell migration <br />       EW Frische et al <br />  <br />  <br />  <br />       Koppen M, Simske JS, Sims PA, Firestein BL, Hall DH, Radice AD,         Segbert C, Johnson K, Theres C, van Furden D, Bossinger O (2004) <br />         Rongo C, Hardin JD (2001) Cooperative regulation AJM-1               Molecular functional analysis apical junction formation in <br />         controls junctional integrity Caenorhabditis elegans epithelia.      gut epithelium Caenorhabditis elegans. Dev Biol 266: 17&#226;&#8364;&#8220;26 <br />         Nat Cell Biol 3: 983&#226;&#8364;&#8220;991                                              Shimonaka M, Katagiri K, Nakayama T, Fujita N, Tsuruo T, Yoshie <br />       Langevin J, Morgan MJ, Sibarita JB, Aresta S, Murthy M, Schwarz T,        O, Kinashi T (2003) Rap1 translates chemokine signals integrin <br />         Camonis J, Bellaiche Y (2005) Drosophila exocyst compo-                 activation, cell polarization, motility vascular endo- <br />         nents Sec5, Sec6, Sec15 regulate DE-Cadherin trafficking            thelium flow. J Cell Biol 161: 417&#226;&#8364;&#8220;427 <br />         recycling endosomes plasma membrane. Dev Cell 9:          Shipitsin M, Feig LA (2004) RalA RalB enhances polarized <br />         355&#226;&#8364;&#8220;376                                                                 delivery membrane proteins basolateral surface of <br />       Lee T, Feig L, Montell DJ (1996) Two distinct roles Ras          epithelial cells. Mol Cell Biol 24: 5746&#226;&#8364;&#8220;5756 <br />         developmentally regulated cell migration. Development 122: 409&#226;&#8364;&#8220;418    Sieburth DS, Sundaram M, Howard RM, Han M (1999) A PP2A <br />       Lewis JA, Fleming JT (1995) Basic culture methods. Methods Cell           regulatory subunit positively regulates Ras-mediated signaling <br />         Biol 48: 3&#226;&#8364;&#8220;29                                                           Caenorhabditis elegans vulval induction. Genes Dev 13: <br />       Lipschutz JH, Mostov KE (2002) Exocytosis: masters            2562&#226;&#8364;&#8220;2569 <br />         exocyst. Curr Biol 12: R212&#226;&#8364;&#8220;R214                                  Simske JS, Hardin J (2001) Getting shape: epidermal morpho- <br />       Mello CC, Kramer JM, Stinchcomb D, Ambros V (1991) Efficient              genesis Caenorhabditis elegans embryos. BioEssays 23: 12&#226;&#8364;&#8220;23 <br />         gene transfer C. elegans: extrachromosomal maintenance         Sommer B, Oprins A, Rabouille C, Munro S (2005) The exocyst <br />         integration transforming sequences. Embo J 10: 3959&#226;&#8364;&#8220;3970             component Sec5 present endocytic vesicles oocyte of <br />       Mirey G, Balakireva M, L&#226;&#8364;&#8482;Hoste S, Rosse C, Voegeling S, Camonis J         Drosophila melanogaster. J Cell Biol 169: 953&#226;&#8364;&#8220;963 <br />         (2003) A Ral guanine exchange factor-Ral pathway con-              Sonnichsen B, Koski LB, Walsh A, Marschall P, Neumann B, Brehm <br />         served Drosophila melanogaster sheds new light            M, Alleaume AM, Artelt J, Bettencourt P, Cassin E, Hewitson M, <br />         connectivity Ral, Ras, Rap pathways. Mol Cell Biol 23:       Holz C, Khan M, Lazik S, Martin C, Nitzsche B, Ruer M, Stamford <br />         1112&#226;&#8364;&#8220;1124                                                               J, Winzi M, Heinkel R et al (2005) Full-genome RNAi profiling <br />       Mizuno T, Hisamoto N, Terada T, Kondo T, Adachi M, Nishida E,             early embryogenesis Caenorhabditis elegans. Nature 434: <br />         Kim DH, Ausubel FM, Matsumoto K (2004) The Caenorhabditis               462&#226;&#8364;&#8220;469 <br />         elegans MAPK phosphatase VHP-1 mediates novel JNK-like              Sugihara K, Asano S, Tanaka K, Iwamatsu A, Okawa K, Ohta Y <br />         signaling pathway stress response. EMBO J 23: 2226&#226;&#8364;&#8220;2234              (2002) The exocyst complex binds small GTPase RalA to <br />       Moskalenko S, Henry DO, Rosse C, Mirey G, Camonis JH, White MA            mediate filopodia formation. Nat Cell Biol 4: 73&#226;&#8364;&#8220;78 <br />         (2002) The exocyst Ral effector complex. Nat Cell Biol 4:        Suzuki J, Yamazaki Y, Li G, Kaziro Y, Koide H (2000) Involvement of <br />         66&#226;&#8364;&#8220;72                                                                   Ras Ral chemotactic migration skeletal myoblasts. Mol <br />       Moskalenko S, Tong C, Rosse C, Mirey G, Formstecher E, Daviet L,          Cell Biol 20: 4658&#226;&#8364;&#8220;4665 <br />         Camonis J, White MA (2003) Ral GTPases regulate exocyst               TerBush DR, Maurice T, Roth D, Novick P (1996) The Exocyst a <br />         assembly dual subunit interactions. J Biol Chem 278:            multiprotein complex required exocytosis Saccharomyces <br />         51743&#226;&#8364;&#8220;51748                                                             cerevisiae. EMBO J 15: 6483&#226;&#8364;&#8220;6494 <br />       Niewiadomska P, Godt D, Tepass U (1999) DE-Cadherin required         Thomas-Virnig CL, Sims PA, Simske JS, Hardin J (2004) The inositol <br />         intercellular motility Drosophila oogenesis. J Cell Biol     1,4,5-trisphosphate receptor regulates epidermal cell migration in <br />         144: 533&#226;&#8364;&#8220;547                                                            Caenorhabditis elegans. Curr Biol 14: 1882&#226;&#8364;&#8220;1887 <br />       Ohta Y, Suzuki N, Nakamura S, Hartwig JH, Stossel TP (1999) The         Tong AH, Evangelista M, Parsons AB, Xu H, Bader GD, Page N, <br />         small GTPase RalA targets filamin induce filopodia. Proc Natl        Robinson M, Raghibizadeh S, Hogue CW, Bussey H, Andrews B, <br />         Acad Sci USA 96: 2122&#226;&#8364;&#8220;2128                                              Tyers M, Boone C (2001) Systematic genetic analysis ordered <br />       Pacquelet A, Rorth P (2005) Regulatory mechanisms required            arrays yeast deletion mutants. Science 294: 2364&#226;&#8364;&#8220;2368 <br />         DE-cadherin function cell migration types adhe-       van Haaften G, Vastenhouw NL, Nollen EA, Plasterk RH, Tijsterman <br />         sion. J Cell Biol 170: 803&#226;&#8364;&#8220;812                                          M (2004) Gene interactions DNA damage-response path- <br />       Pellis-van Berkel W, Verheijen MH, Cuppen E, Asahina M, Rooij          way identified genome-wide RNA-interference analysis of <br />         J, Jansen G, Plasterk RH, Bos JL, Zwartkruis FJ (2005)                  synthetic lethality. Proc Natl Acad Sci USA 101: 12992&#226;&#8364;&#8220;12996 <br />         Requirement Caenorhabditis elegans RapGEF pxf-1            Way JC, Wang L, Run JQ, Wang A (1991) The mec-3 gene contains <br />         rap-1 epithelial integrity. Mol Biol Cell 16: 106&#226;&#8364;&#8220;116               cis-acting elements mediating positive negative regulation in <br />       Poteryaev D, Fares H, Bowerman B, Spang A (2007) Caenorhabditis           cells produced asymmetric cell division Caenorhabditis <br />         elegans SAND-1 essential RAB-7 function endosomal             elegans. Genes Dev 5: 2199&#226;&#8364;&#8220;2211 <br />         traffic. EMBO J 26: 301&#226;&#8364;&#8220;312                                           Williams-Masson EM, Heid PJ, Lavin CA, Hardin J (1998) The <br />       Price LS, Hajdo-Milasinovic A, Zhao J, Zwartkruis FJ, Collard JG,         cellular mechanism epithelial rearrangement morpho- <br />         Bos JL (2004) Rap1 regulates E-cadherin-mediated cell&#226;&#8364;&#8220;cell adhe-        genesis Caenorhabditis elegans dorsal hypodermis. Dev <br />         sion. J Biol Chem 279: 35127&#226;&#8364;&#8220;35132                                      Biol 204: 263&#226;&#8364;&#8220;276 <br />       Prigent M, Dubois T, Raposo G, Derrien V, Tenza D, Rosse C,             Withee J, Galligan B, Hawkins N, Garriga G (2004) Caenorhabditis <br />         Camonis J, Chavrier P (2003) ARF6 controls post-endocytic               elegans WASP Ena/VASP proteins play compensatory roles <br />         recycling downstream exocyst complex effector.              morphogenesis neuronal cell migration. Genetics 167: <br />         J Cell Biol 163: 1111&#226;&#8364;&#8220;1121                <br /> </body></html>