<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                               NIH Public Access <br />                                Author Manuscript <br />                                Pac Symp Biocomput. Author manuscript; available PMC 2009 December 17. <br />                            Published final edited form as: <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             Pac Symp Biocomput. 2009 ; : 391&#226;&#8364;&#8220;402. <br />  <br />  <br />  <br />                            HIGH THROUGHPUT INTERACTION DATA REVEALS DEGREE <br />                            CONSERVATION OF HUB PROTEINS <br />                            A. Fox1,*, D. Taylor1, D. K. Slonim1,2 <br />                            1Department Computer Science, Tufts University, 161 College Avenue, Medford, MA 02155, <br />  <br />                            USA <br />                            2Department         Pathology, Tufts University School Medicine, 145 Harrison Ave., Boston, MA <br />                            02111, USA <br />  <br />                            <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />                                    Research model organisms relies unspoken assumptions conservation protein- <br />                                    protein interactions species, analyses suggest conservation limited. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                    Fortunately, purposes crucial issue global conservation interactions, but <br />                                    preferential conservation functionally important ones. An observed bias essentiality in <br />                                    highly-connected proteins implies functional importance &#226;&#8364;&#339;hubs&#226;&#8364;?. We define <br />                                    notion degree-conservation demonstrate hubs preferentially degree-conserved. <br />                                    We protein likely hub high-degree ortholog, a <br />                                    protein hub, tends remain so. We identify positive correlation the <br />                                    average degree protein conservation interaction partners, the <br />                                    conservation individual hub interactions surprisingly high. Our work important <br />                                    implications prediction protein function, computational inference PPIs, interpretation <br />                                    data model organisms. <br />  <br />  <br />                            1 Introduction <br />                                                 The power comparative genomic research grown steadily availability of <br />                                                 genomic sequence annotation increasing numbers organisms. A variety of <br />                                                 techniques solving diverse problems motif discovery[1], gene expression <br />                                                 analysis[2], regulatory network inference[3] interactome discovery[4] rely the <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 assumption functionally important protein-DNA protein-protein interactions (PPIs) <br />                                                 preferentially conserved species. Yet remains <span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span> fully understand <br />                                                 conservation protein interaction modules functions[5]. Although studies of <br />                                                 &#226;&#8364;&#732;interologs&#226;&#8364;&#8482; (orthologous pairs proteins interactions conserved)[6] report <br />                                                 preferential conservation interactions highly similar orthologous protein pairs[7], <br />                                                 Mika Rost[8] point inference PPIs based sequence homology across <br />                                                 species continues quite inaccurate, high levels sequence identity. <br />  <br />                                                 For interpretation data model organisms, low interolog conservation might <br />                                                 acceptable, provided functionally important interactions which <br />                                                 orthologs exist species conserved acceptably high rate. Yet this <br />                                                 remains shown. Recent work[9] shows functional modules better conserved <br />                                                 yeast fly expected chance. Furthermore, previous work in <br />                                                 studying transcriptional regulation shows despite preferential sequence conservation <br />  <br />  <br />  <br />                            * <br />                                afox01@cs.tufts.edu. <br />                             Fox et al.                                                                                              Page 2 <br />  <br />  <br />                                          functionally important proteins, functional regulatory modules especially well <br />                                          conserved[10]. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Here, focus conservation interactions proteins, most <br />                                          highly-connected proteins, &#226;&#8364;&#339;hubs&#226;&#8364;? species&#226;&#8364;&#8482; interaction network. We select hub <br />                                          proteins precisely high connectivity serve indication functional <br />                                          importance. Indeed, proteins interactions likely essential[11], and <br />                                          these, hubs connected hubs essential still[12]. If hub proteins <br />                                          likely functionally important, roles hubs preferentially <br />                                          conserved, individual interactions not. We investigate <br />                                          property hub protein conserved organisms, we <br />                                          determine level PPI conservation hubs neighbors. We that <br />                                          interactions high degree proteins preferentially conserved when <br />                                          specific interactions hubs lost gained, high degree hub protein is <br />                                          nonetheless conserved. Thus, greater hope the functional importance of <br />                                          hubs conserved well. <br />  <br />                                          We point ample reason expect priori high degree hub <br />                                          proteins preferentially conserved evolution[13]. This based the <br />                                          argument protein evolved functional interactions proteins, <br />                                          dramatic change affect partners, likely be <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          evolutionarily disadvantageous. However, given recent work[9, 10] casts doubt upon <br />                                          modular functional conservation, hypothesis requires proof. <br />  <br />                                          There reasons skeptical quality quantity available PPI data. <br />                                          Estimates false-positive rates high throughput interaction data sets range <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br />                                          45% 90%[14, 15], coverage estimates human yeast interactomes are <br />                                          10% 50%, respectively[14, 16]. <br />  <br />                                          Thus, attempts identify high degree proteins affected noise, coverage, and <br />                                          testing bias[16]. Methods estimating true degree interactions[17] available, but <br />                                          directly applicable data sets studied here, cases clear <br />                                          interactions tested. Nonetheless, modest data filtering, obtain clear <br />                                          evidence important trends. <br />  <br />                                          Our results provide additional evidence identifying biologically relevant high-degree <br />                                          proteins model organisms shed light human response. Our work has <br />                                          important implications protein function prediction, comparative genomics, network <br />                                          inference analysis. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               2 Background Definitions <br />                               2.1 Data Sources <br />                                          All PPIs Homo sapiens, Drosophila melanogaster, Caenorhabditis elegans, and <br />                                          Saccharomyces cerevisiae downloaded BioGRID[18], IntAct[19] and <br />                                          MINT[20] online databases July, 2008 combined form single large data set. <br />  <br />                                          To map orthologous proteins species, used InParanoid database[21] because <br />                                          discriminates true orthologs &#226;&#8364;&#732;in&#226;&#8364;&#8482; paralogs differentiated relevant <br />                                          speciation event &#226;&#8364;&#732;out&#226;&#8364;&#8482; paralogs arose speciation. <br />  <br />  <br />  <br />  <br />                                          Pac Symp Biocomput. Author manuscript; available PMC 2009 December 17. <br />                             Fox et al.                                                                                                  Page 3 <br />  <br />  <br />                               2.2 Filtering Out PPI Noise <br />                                           To improve reliability data[17], built PPI graph assay method, with <br />                                           directed edges (b, p) indicating roles protein (b)ait (p)rey where <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           appropriate. We define baitrank protein p species s assay type the <br />                                           fraction proteins species s having (non-zero) out-degree p&#226;&#8364;&#8482;s out-degree for <br />                                           assay a. The preyrank defined analogously in-degree. If given out-degree in- <br />                                           degree zero corresponding baitrank preyrank undefined. <br />  <br />                                           We concluded assay type inconsistent protein p p&#226;&#8364;&#8482;s baitrank and <br />                                           preyrank defined |baitrank(p) &#226;&#710;&#8217; preyrank(p)| &gt; 0.1. Under conditions we <br />                                           removed data assay type p. After filtering, degrees estimated using <br />                                           naive method[17] remaining data. Overall, approach removed 36% the <br />                                           data, led stronger results. Full datasets descriptive statistics (Supplemental File 1) <br />                                           available http://bcb.cs.tufts.edu/hubsPSB09. <br />  <br />                                2.3 Ranking Protein Degrees <br />                                           Degrees proteins different species directly compared species <br />                                           different degree distribution. To address issue, ranked proteins each <br />                                           species degree filtered dataset species, worked these <br />                                           ranks instead raw degree. We define r, rank protein p degree d species s, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           fraction proteins species s having non-zero degree d. <br />  <br />                                           Individual proteins labeled hubs using straightforward threshold rank. The <br />                                           hub threshold t defined interval [0.5,1) intuitively, makes little sense talk <br />                                           protein rank 0.5 hub. <br />  <br />                               2.4 Degree Conservation <br />                                           A protein considered evolutionarily conserved more <br />                                           orthologous proteins species consideration. Most the <br />                                           literature conservation hub proteins focused type sequence <br />                                           conservation[22, 23]. At end spectrum, consider specific <br />                                           PPIs conserved, given pairs proteins known orthologs species. <br />                                           (Such conserved interactions frequently referred interologs[6].) <br />  <br />                                           In paper, focus middle ground - looking interaction degree a <br />                                           protein conserved. This conservation occur specific interologs are <br />                                           maintained. We define pair orthologous proteins degree-conserved hub threshold t <br />                                           proteins hubs hub threshold t respective species. This <span id='am-3' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span> be <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           naturally extended species. <br />  <br />                               3 Having Orthologous Hub Increases Hub Likelihood <br />                               3.1 Degree Conservation Ortholog Pairs <br />                                           We analyzed pairs orthologous proteins species. We created a <br />                                           dataset D containing proteins filtered data species in <br />                                           proteins orthologous. We interested determining proteins that <br />                                           hub ortholog significantly higher probability hub protein. <br />  <br />                                           To this, compare observed posterior probability protein hub, given that <br />                                           ortholog species hub, prior probability protein a <br />                                           hub data set. A simple way define prior probability hub just (1 &#226;&#710;&#8217; t), <br />                                           t hub threshold. We uninformed prior, U(t). However, are <br />  <br />  <br />                                           Pac Symp Biocomput. Author manuscript; available PMC 2009 December 17. <br />                             Fox et al.                                                                                                     Page 4 <br />  <br />  <br />                                          considering proteins orthologs 1 3 species consideration, and <br />                                          shown proteins distant orthologs higher average degree[24], <br />                                          true prior higher. Thus, compute informed prior, I(t). Let hubt(p) indicate <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          p hub threshold t let orth(x, y) indicate x y orthologs. Then I(t) = <br />                                          Pr[hubt(p) = 1], represents observed fraction proteins data set are <br />                                          hubs. We compute I(t) dividing number proteins D rank &#226;&#8240;&#165; t total size <br />                                          D. <br />  <br />                                          We define posterior probability P(t) = Pr[hubt(pi) = 1| &#226;&#710;&#402; pj s pecies, s.t. <br />                                          orth(pi, pj) &#226;&#710;&#167; hubt (pj) = 1]. To compute P(t), create set St containing proteins <br />                                          ortholog species hub threshold t. Then P(t) = |{pi &#226;&#710;&#710; St | <br />                                          hubt (pi) = 1}|/|St|. Intuitively, P(t) fraction proteins St hubs threshold t. <br />                                          Figure 1a shows P(t), I(t) U(t). <br />  <br />                                          We compare observed distribution P informed prior I using Kolmogorov- <br />                                          Smirnov (KS) Test[25]. We used 1-sided KS Test determine [P &gt; I] the <br />                                          intervals t &#226;&#710;&#710; [x, 1) x &#226;&#710;&#710; {0.5, 0.55,&#226;&#8364;&#166;, 0.95}. Tests significant p &lt; 10&#226;&#710;&#8217;6 all <br />                                          intervals. These results support hypothesis probability protein hub is <br />                                          higher ortholog hub, implying hub proteins degree-conserved often <br />                                          expected chance. In way, determined I significantly greater than <br />                                          U, expected reason described above. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          To determine increase hub probability given existence orthologous hub, we <br />                                          plot fold-change probabilities F(t) = P(t)/I(t) Figure 1b. From graph F(t) we <br />                                          probability given protein hub greater hub ortholog the <br />                                          magnitude effect increases higher hub threshold. Indeed, F(t) peaks at <br />                                          approximately t = &#206;&#179; = 0.99. When t &gt; &#206;&#179; expected number conserved hubs quickly <br />                                          approaches zero, diminishing expectation number degree-conserved <br />                                          pairs data set (the numerator P(t)) t &gt; &#206;&#179;. For hub thresholds t &#226;&#8240;&#710; 0.93 we <br />                                          two-fold increase hub likelihood. <br />  <br />                               3.2 Degree Conservation Across Four Species <br />                                          We extend pairwise ortholog analysis look ortholog groups four <br />                                          species. We create 4-component rank vectors xi = [rank(yi), rank(wi), rank(fi), rank(hi)] <br />                                          yi, wi, fi hi proteins form ortholog group yeast, worm, fly and <br />                                          human. Let nT total number rank vectors, n(t) number rank vectors that <br />                                          exhibit degree-conservation hub threshold t, O(t) = n(t)/nT proportion of <br />                                          ortholog groups exhibit hub conservation hub threshold t. We compute the <br />                                          expected rate conservation degree conserved orthology E(t). Then <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          E(t) = &#206;&#160;c&#226;&#710;&#352;M Pr[r &gt; t|r &#226;&#710;&#710; c], M matrix rank 4-vectors generated the <br />                                          orthology groups c represents column ranks specific particular species. If O(t) <br />                                          significantly greater E(t) conclude degree fact conserved by <br />                                          orthology. <br />  <br />                                          We compare observed rate 4-species degree-conservation O(t) expected rate <br />                                          E(t) using series KS tests. We tested interval t &#226;&#710;&#710; [a, b] &#226;&#710;&#710; {0.5, 0.51,&#226;&#8364;&#166;, <br />                                          0.95} b &#226;&#710;&#710; {0.55, 0.56,&#226;&#8364;&#166;, 1.0} &#226;&#8240;&#164; (b &#226;&#710;&#8217; 0.05). The KS test results showed [O &gt; <br />                                          E] t-intervals fully subsumed interval t &#226;&#710;&#710; [0.66, 0.95] (p &lt; 10&#226;&#710;&#8217;6) we <br />                                          conclude high degree conserved considered species. <br />  <br />                                          The lack 4-species degree-conservation t &gt; 0.95 attributed incomplete <br />                                          interaction data, low overlap interactome coverage species[26], combination <br />                                          effects. <br />  <br />  <br />                                          Pac Symp Biocomput. Author manuscript; available PMC 2009 December 17. <br />                             Fox et al.                                                                                                Page 5 <br />  <br />  <br />                                          To gain understanding functional classes proteins degree-conserved, we <br />                                          performed functional enrichment analysis (DAVID[27]) yeast proteins degree- <br />                                          conserved (t = 0.8) 1 3 species. Enriched functions included primary <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          metabolism, protein synthesis, splicing, DNA repair regulation cell-cycle (FDR &lt; <br />                                          0.05). Enrichment basic cellular processes expected given hubs required <br />                                          degree-conserved yeast higher eukaryotes. In contrast, degree-conserved <br />                                          (t = 0.8) proteins human fly enriched processes cell signalling <br />                                          (the p53, Wnt, ErbB, Notch TGF-&#206;&#178; pathways), cell differentiation, cellular <br />                                          developmental processes, post-translational protein modification regulation of <br />                                          protein kinase activity (FDR &lt; 0.05). Full details functional enrichment results <br />                                          (Supplemental File 2) available online http://bcb.cs.tufts.edu/hubsPSB09. <br />  <br />                               4 Once Hub, Always Hub <br />                                          Our hypothesis hubs exhibit degree-conservation relies intuition a <br />                                          protein evolved connections, difficult protein lose importance <br />                                          neighbors affected change. However, analyses far <br />                                          fail account orthologs diverged proteins evolved its <br />                                          observed &#226;&#8364;&#339;hub&#226;&#8364;? role. A simple evolutionary model expects hubs remain highly <br />                                          connected account larger fraction data. We this <br />                                          &#226;&#8364;&#339;once hub, hub&#226;&#8364;? hypothesis. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          To test hypothesis, simple phylogenetic tree (Figure 2a) relating four <br />                                          species data set. We seek determine closely data matches this <br />                                          phylogenetic model. For ortholog group spanning species, form rank <br />                                          vector Section 3.2. We create 4-bit bitstring describing hub status its <br />                                          component protein species. (If given protein&#226;&#8364;&#8482;s rank greater hub <br />                                          threshold t, species&#226;&#8364;&#8482; component bitstring 1; 0.) An ortholog <br />                                          group considered consistent model (with assumption parsimony) bitstring <br />                                          explained evolution hub property at point tree (Figure <br />                                          2b). <br />  <br />                                          We consider possible ortholog groups species. Figure 3 shows the <br />                                          fraction ortholog groups consistent model grows hub threshold increases. <br />                                          For t &gt; 0.73, majority data explained straightforward phylogenetic <br />                                          model hub property selective pressure retained has <br />                                          evolved. <br />  <br />                                          One caveat sufficiently high t, data all-zero bitstring is <br />                                          consistent model. Thus, potentially growth fraction of <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          consistent ortholog groups category. However, the <br />                                          case repeating analysis excluding all-zero data (Figure 3). Full details the <br />                                          consistency analysis (Supplemental File 3) provided http://bcb.cs.tufts.edu/ <br />                                          hubsPSB09. <br />  <br />                                5 Hub Interactions Are Preferentially Conserved <br />                                          Finally, examine retention hub property retention of <br />                                          individual PPIs, simply general importance high degree proteins. We consider <br />                                          models. In one, interologs conserved sufficiently - high degree <br />                                          proteins - &#226;&#8364;&#339;module&#226;&#8364;? structure hub neighbors retained. This suggests <br />                                          function protein partners likely conserved well. The <br />                                          alternative hypothesis assumes protein evolved large number partners, it <br />                                          pressure evolve slowly[24]. Although protein&#226;&#8364;&#8482;s interaction <br />  <br />                                          Pac Symp Biocomput. Author manuscript; available PMC 2009 December 17. <br />                             Fox et al.                                                                                             Page 6 <br />  <br />  <br />                                         partners lost time, high sequence stability hubs allow new <br />                                         interactions arise retained easily. Note hypothesis is <br />                                         somewhat odds growing body work suggesting interologs conserved at <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         moderate level[7, 26], functional modules generally conserved intact[9]. <br />  <br />                                         To address question, analyzed conservation individual interactions a <br />                                         function protein interaction degree. To measure conservation interactions a <br />                                         given protein p species s1, designated second species s2, rely following <br />                                         definitions. First, let G1 protein-protein interaction graph species s1: G1 = <br />                                         &#226;&#338;&#169;V1,E1&#226;&#338;&#170;, let G2 similarly defined species s2. Furthermore, let relation orth, <br />                                         defined V1 &#195;&#8212; V2, denote orthologous pairs proteins species. Let the <br />                                         potential number conserved interactions p, Poten(p) number protein <br />                                         interaction partners q p s1 orthologs p q exist s2. Also let the <br />                                         actual number conserved interactions p, Conserv(p) number protein <br />                                         interactions p s1 conserved s2. Formally, protein p &#226;&#710;&#710; V1, define <br />                                         Poten(p) = |{q &#226;&#710;&#710; V1|&#226;&#338;&#169;p, q&#226;&#338;&#170; &#226;&#710;&#710; E1 &#226;&#710;&#167; (&#226;&#710;&#402;p&#226;&#8364;&#178;, q&#226;&#8364;&#178; &#226;&#710;&#710; V2|orth(p, p&#226;&#8364;&#178;) &#226;&#710;&#167; orth(q, q&#226;&#8364;&#178;))}| Conserv(p) = <br />                                         |{q &#226;&#710;&#710; V1|&#226;&#338;&#169;p, q&#226;&#338;&#170; &#226;&#710;&#710; E1 &#226;&#710;&#167; (&#226;&#710;&#402;p&#226;&#8364;&#178;, q&#226;&#8364;&#178; &#226;&#710;&#710; V2|orth(p, p&#226;&#8364;&#178;) &#226;&#710;&#167;orth(q, q&#226;&#8364;&#178;) &#226;&#710;&#167; &#226;&#338;&#169;p&#226;&#8364;&#178;, q&#226;&#8364;&#178; &#226;&#338;&#170; &#226;&#710;&#710; E2)}|. Then we <br />                                         compute conservation rate R(p) = Conserv(p)/Poten(p) proportion of <br />                                         interactions conserved interactions possibly conserved. <br />  <br />                                         We measured conservation rate R(p) protein pair species compared <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         rank(p). Results fly proteins shown Figure 4. The plot reveals the <br />                                         proportion protein&#226;&#8364;&#8482;s interactions conserved increases protein rank (or degree) <br />                                         increases. In words, hubs preferential conservation individual protein <br />                                         interactions non-hubs. <br />  <br />                                         For species pairs, graphs similar shape (Supplemental File 4), the <br />                                         overall conservation highest degree proteins lower (65%&#226;&#8364;&#8220;85%). The <br />                                         especially high conservation fly human interologs reflect bias in <br />                                         protein interactions chosen test assays particular species. Even the <br />                                         similar species pairs, however, high hub thresholds, half of <br />                                         interologs conserved. This contrasts sharply general analysis Mika and <br />                                         Rost[8], shows interolog conservation hovering 25% highest levels <br />                                         sequence similarity, using data overlaps significantly ours. We conclude the <br />                                         interolog conservation hub proteins simply fact that <br />                                         highly-connected proteins higher sequence conservation, interactions are <br />                                         truly preferentially conserved. <br />  <br />                                         We note number species pairs, significant fraction (up 40%) the <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         interactions conserved high hub thresholds. This suggests degree- <br />                                         conservation hubs rely solely conservation individual pairwise <br />                                         interactions, functional roles hub proteins themselves. <br />  <br />                               6 Discussion <br />                                         We demonstrated important novel facts conservation protein <br />                                         interactions function. First, likelihood protein hub increases an <br />                                         ortholog hub. Second, protein evolves hub, tends remain so, <br />                                         specific interaction partners gained lost. Finally, rate interolog <br />                                         conservation hub proteins higher low-degree proteins. <br />  <br />                                         These results imply that, said functional conservation modules <br />                                         defined various clustering methods (e.g., Wang Zhang[9]), functional modules <br />                                         network hubs likely persist species. This result crucial implications <br />  <br />                                         Pac Symp Biocomput. Author manuscript; available PMC 2009 December 17. <br />                             Fox et al.                                                                                                              Page 7 <br />  <br />  <br />                                         unspoken assumption underlying research model organisms: the <br />                                         functional relationships key proteins conserved humans model <br />                                         organisms studied. Furthermore, &#226;&#8364;&#339;once hub, hub&#226;&#8364;? property implies that <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         possible infer identity hub proteins organisms sufficient <br />                                         protein interaction data sets available. For example, orthologous proteins <br />                                         worm fly PPI hubs, reasonable assume evolution the <br />                                         hub property occurred ancestor worm fly; therefore, orthologous <br />                                         protein bees likely hub well. <br />  <br />                                         In addition, stronger-than-expected conservation individual interactions hub <br />                                         proteins important implications computational prediction protein-protein <br />                                         interactions. Although growth experimentally-generated protein interaction data is <br />                                         substantial, interactomes organisms likely remain largely incomplete for <br />                                         time, computational inference interactions continue play important <br />                                         role. Furthermore, suggested Matthews et al.[4], computational predictions be <br />                                         used guide selection experimental assays, higher predictive accuracy reduce <br />                                         cost screening, allowing generation data. <br />  <br />                                         Despite results, issues surrounding data require caution in <br />                                         interpreting results based existing protein interaction networks. Most importantly, the <br />                                         data noisy extraordinarily incomplete. Since different subsets the <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         interactomes species investigated, possible bias species <br />                                         coverage affect conclusions based data currently available. In addition, some <br />                                         methods identifying protein interactions (such coimmunoprecipitation affinity <br />                                         precipitation[9]) identify protein complexes pairwise interactions. Many pairwise <br />                                         interactions inferred participants complexes[28], potentially <br />                                         creating high-degree nodes erroneously. Thus, informative revisit these <br />                                         conclusions quality quantity interactome data grow. <br />  <br />                               Supplementary Material <br />                                         Refer Web version PubMed Central supplementary material. <br />  <br />  <br />                               Acknowledgments <br />                                         This research supported award number R21LM009411 National Library Medicine. The content <br />                                         solely responsibility authors does necessarily represent official views National <br />                                         Library Medicine National Institutes Health. We thank members Tufts BCB research group <br />                                         helpful discussions comments. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               References <br />                                         1. Blanchette M, Tompa M. Genome Res. 2002; 12(5):739. [PubMed: 11997340] <br />                                         2. Eisen M, Spellman P, Brown P, Botstein D. PNAS. 1998; 95:14863. [PubMed: 9843981] <br />                                         3. Yu J, Smith V, Wang P, Hartemink A, Jarvis E. Bioinformatics. 2001; 20:3594. [PubMed: <br />                                            15284094] <br />                                         4. Matthews L, Vaglio P, Reboul J, Ge H, Davis B, Garrels J, Vincent S, Vidal M. Genome Res. 2001; <br />                                            11:2120. [PubMed: 11731503] <br />                                         5. Han J, Bertin N, Hao T, Goldberg D, Berriz G, Dupuy D, Walhout A, Cusick M, Roth F, Vidal M. <br />                                            Nature. 2004; 88:6995. <br />                                         6. Walhout A, Sordella R, Lu X, Hartley J, Temple G, Brasch M, Thierry-Mieg N, Vidal M. Science. <br />                                            2000; 287:116. [PubMed: 10615043] <br />                                         7. Yu H, Luscombe N, Lu H, Zhu X, Xia Y, Han J. Genome Res. 2004; 14 <br />                                         8. Mika S, Rost B. PLoS Comput. Biol. 2006; 2:e79. [PubMed: 16854211] <br />  <br />  <br />                                         Pac Symp Biocomput. Author manuscript; available PMC 2009 December 17. <br />                             Fox et al.                                                                                                Page 8 <br />  <br />  <br />                                         9. Wang Z, Zhang J. PLoS Comp Biol. 2007; 3 <br />                                         10. Babu M, Luscombe N, Aravind L, Gerstein M. Curr Opin Cell Biol. 2004; 14:283. <br />                                         11. Jeong H, Mason S, Barabasi A, Oltvai Z. Nature. 2001; 411:41. [PubMed: 11333967] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         12. Batada N, Reguly T, Breitkreutz A, Boucher L, Breitkreutz B, Hurst L, Tyers M. PLoS Biol. 2006; <br />                                             4:e317. [PubMed: 16984220] <br />                                         13. Wuchty S, Barabasi A, Ferdig M. BMC Evol Biol. 2006; 6:1471. <br />                                         14. Hart G, Ramani A, Marcotte E. Genome Biol. 2006; 7:120. [PubMed: 17147767] <br />                                         15. D&#226;&#8364;&#8482;haeseleer P, Church G. Proc IEEE Comp Syst Bioinformatics. 2004 Aug.216 <br />                                         16. Chiang T, Scholtens D, Sarkar D, et al. Genome Biol. 2007; 8:186. <br />                                         17. Scholtens D, Chiang T, Huber W, Gentleman R. Bioinformatics. 2008; 24:218. [PubMed: <br />                                             18025006] <br />                                         18. Stark C, Breitkreutz B, Reguly T, Boucher L, Breitkreutz A, Tyers M. Nucleic Acids Res. 2006 <br />                                             Jan.34:D535. [PubMed: 16381927] <br />                                         19. Hermjakob H, Montecci-Palazzi L, Lewington C, Mudali S, Kerrien S, et al. Nucleic Acids Res. <br />                                             2004; 32:D452. [PubMed: 14681455] <br />                                         20. Zanzoni A, Montecci-Palazzi L, Quondam M, Ausiello G, Helmer-Citterich M, et al. FEBS <br />                                             Letters. 2002; 513:135. [PubMed: 11911893] <br />                                         21. Remm M, Storm C, Sonnhammer E. JMB. 2001; 314:1041. <br />                                         22. Fraser H, Hirsh A, Steinmetz L, Scharfe C, Felman M. Science. 2002; 296:750. [PubMed: <br />                                             11976460] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         23. Jordan I, Wolf I, Koonin E. BMC Evol Biol. 2003; 3:1. [PubMed: 12515583] <br />                                         24. Brown K, Jurisica I. Genome Biol. 2007; 8 <br />                                         25. Press, W.; Teukolsky, S.; Vetterling, W.; Flannery, B. Numerical Recipes C, Art of <br />                                             Scientific Computing. Cambridge Univeristy Press; 1992. <br />                                         26. Gandhi T, Zhong J, Mathivanan S, et al. Nature Gen. 2006:38. <br />                                         27. D G Jr, Sherman B, Hosack D, et al. Genome Biol. 2003; 4:3. <br />                                         28. Bader G, Hogue C. Nature Biotechnol. 2002; 20:991. [PubMed: 12355115] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         Pac Symp Biocomput. Author manuscript; available PMC 2009 December 17. <br />                             Fox et al.                                                                                            Page 9 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         Figure 1. <br />                                         Hub likelihood plots. a) Conditional probability hub given ortholog a <br />                                         hub, compared prior probability hub. The solid line represents P(t), the <br />                                         posterior (conditional) hub probability; dot-dash line U(t) uninformed prior <br />                                         probability, dashed line I(t) informed prior based observed data. b) Fold <br />                                         Increase F hub likelihood, given orthologous hub protein, function hub <br />                                         threshold t. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         Pac Symp Biocomput. Author manuscript; available PMC 2009 December 17. <br />                             Fox et al.                                                                                         Page 10 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         Figure 2. <br />                                         a) Simple phylogenetic model species data set, based data the <br />                                         Tree Life database (www.tol.org). b) Consistency hub-status bitstrings <br />                                         (yeast,worm,fly,human) &#226;&#8364;&#732;Once hub, hub&#226;&#8364;&#8482; hypothesis. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         Pac Symp Biocomput. Author manuscript; available PMC 2009 December 17. <br />                             Fox et al.                                                                                         Page 11 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         Figure 3. <br />                                         Consistency &#226;&#8364;&#339;once hub, hub&#226;&#8364;? hypothesis, function hub threshold. <br />                                         The y-axis shows percentage ortholog groups consistent hypothesis, <br />                                         given phylogenetic model Figure 2. The solid line represents consistent ortholog <br />                                         groups; dashed line excludes all-zero vectors (proteins hubs any <br />                                         species) computation. <br />  <br />  <br />  <br />  <br />                                         Pac Symp Biocomput. Author manuscript; available PMC 2009 December 17. <br />                             Fox et al.                                                                                          Page 12 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         Figure 4. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         Interolog conservation function protein interaction degree rank. The data shown are <br />                                         fly proteins orthologs species. Similar-rank <br />                                         proteins binned conservation rates averaged trend seen clearly. <br />                                         Conservation interologs low low-degree proteins, increases function of <br />              <br /> </body></html>