<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Resource <br />  <br /> Comparative genome analysis kingdom <br /> of eukaryotic organisms: Specialization <br /> and diversification Fungi <br /> Michael J. Cornell,1,2 Intikhab Alam,1 Darren M. Soanes,3 Han Min Wong,3 <br /> Cornelia Hedeler,1 Norman W. Paton,1 Magnus Rattray,1 Simon J. Hubbard,2 <br /> Nicholas J. Talbot,3 Stephen G. Oliver2,4,5,6 <br /> 1 <br />  School Computer Science, University Manchester, Manchester M13 9PL, United Kingdom; 2Faculty Life Sciences, <br /> University Manchester, Manchester M13 9PL, United Kingdom; 3Department Biosciences, University Exeter, Exeter EX4 <br /> 4QD, United Kingdom; 4Department Biochemistry, University Cambridge, Cambridge CB2 1 GA, United Kingdom <br />  <br />  <br />         The recent proliferation genome sequencing diverse fungal species provided opportunity for <br />         comparative genome analysis eukaryotic kingdom. Here, report comparative study 34 complete <br />         fungal genome sequences, representing broad diversity Ascomycete, Basidiomycete, Zygomycete species. We <br />         clustered predicted protein-encoding gene sequences species provide means investigating <br />         gene innovations, gene family expansions, protein family diversification, conservation essential gene <br />         functions&#226;&#8364;&#8221;empirically determined Saccharomyces cerevisiae&#226;&#8364;&#8221;among fungi. The results presented with <br />         reference phylogeny 34 fungal species, based 29 universally conserved protein-encoding gene <br />         sequences. We contrast phylogeny based gene presence absence that, two <br />         phylogenies largely agreement, differences positioning species. We investigated <br />         levels gene duplication demonstrate varies greatly fungal species, are <br />         instances coduplication distantly related fungi. We investigated extent orthology protein <br />         families demonstrate unexpectedly high levels diversity genes involved lipid metabolism. These <br />         analyses collated e-Fungi data warehouse, providing online resource comparative genomic <br />         analysis fungi. <br />         [Supplemental material available online www.genome.org.] <br />  <br /> The Fungi represent single eukaryotic kingdom, characterized                             It striking 431 published studies de- <br /> by osmotrophic growth habitat extracellular en-                          scribing completed genome sequences (October 2006), 361 de- <br /> zymes secreted break complex substrates, result-                     scribe bacterial genomes, 28 Archaeal genomes, 42 are <br /> ing simple sugars amino acids taken growing                      genomes eukaryotic organisms (for compendium, see <br /> fungus. Fungi exist distinct morphological growth forms,                      http://www.tigr.org/tdb/ genomes online database <br /> the unicellular yeasts (which grow budding simple fission)                     http://www.genomesonline.org). Of eukaryotic genome se- <br /> and filamentous fungi (which produce polarized hyphal                            quences currently available, half come king- <br /> strands aggregate form network called mycelium). The                     dom Fungi, making credible candidates king- <br /> osmotrophic growth habit fungi extremely effective colo-                   dom-wide exploration genomic relatedness. In addition <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> <br /> nizing diverse habitats fungi principal de-                     sampling vast evolutionary diversity, complete ge- <br /> graders biomass terrestrial ecosystems (de Boer et al.                     nome sequences available closely related fungal <br /> 2005) important pathogens plants animals.                       species&#226;&#8364;&#8221;for instance, Saccharomyces &#226;&#8364;&#339;sensu stricto&#226;&#8364;? species <br />       The yeasts filamentous fungi cover huge evolutionary                     group members closely related mate to <br /> range. The Pezizomycotina (filamentous ascomycetes)                          produce viable hybrids (Naumov et al. 1997, 2000; Marinoni et <br /> Saccharomycotina (budding yeasts), example, diverged                        al. 1999). The diverse biology morphology fungi, coupled <br /> one 900&#226;&#8364;&#8220;1000 million years ago (Mya) (Hedges et                         availability substantial genomic sequence informa- <br /> al. 2004), Saccharomycotina evolution-                        tion, means Fungi unique group to <br /> arily diverged Chordate phylum animal kingdom                        study eukaryotic genome evolution, methods are <br /> (Goffeau 2004).                                                                      developed analyses, results obtained, are <br />                                                                                      likely help comparative genome analyses eu- <br /> 5 <br />  Present address: Department Biochemistry, University Cam- <br />                                                                                      karyotic kingdoms. To end, assembled genome se- <br /> bridge, Cambridge CB2 1 GA, UK.                                                      quences filamentous fungi yeast database, which <br /> 6 <br />  Corresponding author.                                                               e-Fungi. In study, report results com- <br /> E-mail Steve.oliver@mole.bio.cam.ac.uk; fax 44-1223-766-002. <br />                                                                                      prehensive comparative genomic analysis carried using the <br /> Article published online print. Article publication date http:// <br /> www.genome.org/cgi/doi/10.1101/gr.6531807. Freely available online                   e-Fungi resource. <br /> through Genome Research Open Access option.                                            The majority publicly available fully sequenced fun- <br />  <br />  <br /> 17:1809&#226;&#8364;&#8220;1822 &#194;&#169;2007 Cold Spring Harbor Laboratory Press; ISSN 1088-9051/07; www.genome.org                           Genome Research          1809 <br />                                                                                                                           www.genome.org <br />  Cornell et al. <br />  <br />  <br /> gal genomes come Ascomycete species,              2003) resulted different numbers predicted ORFs, <br /> investigated sequenced Basidiomycete fungi, Ustilago maydis         despite similar degrees sequence coverage achieved. <br /> and Phanerochaete chrysosporium, plus Zygomycete Rhizopus           ORFs real clearly likely conserved across <br /> oryzae Microsporidian Encephalitozoon cuniculi. In addi-        species. Second, occurrence orphans lack <br /> tion, included non-fungal species, Oomycetes            sequence data closely related fungal species. For example, <br /> Phytophthora sojae Phytophthora ramorum. Oomycetes het-         Saccharomycotina yeast species analysis <br /> erokonts diverged Fungi early evolution       and, consequence, &gt;97% predicted protein products of <br /> of eukaryotes. Nevertheless, terms biological activity,     Eremothecium (formerly Ashbya) gossypii genes members of <br /> they demonstrate convergent evolution fungal plant patho-          clusters. In comparison, Schizosaccharomyces pombe only <br /> gens. For example, oomycetes share filamentous, osmotrophic           Archaeascomycete yeast data set, 20% ORFs <br /> growth habit capable invasive growth coloniza-           encode proteins included cluster. <br /> tion living plant tissue (Money et al. 2004). <br />      We used features e-Fungi ask fundamen- <br />                                                                         Phylogenies fungi based concatenated protein <br /> tal questions fungal evolutionary biology, including pat- <br />                                                                         sequences Dollo parsimony <br /> terns gene loss gene duplication, extent protein family <br /> conservation kingdom Fungi, distinctive ge-           A common approach molecular phylogenetics use se- <br /> nomic features separate yeasts filamentous species. Our        quence comparisons single genes proteins multiple <br /> study represents comprehensive analysis king-           species order construct phylogenies. However, availabil- <br /> dom Fungi carried date highlighted surpris-         ity complete genome sequences allows construction of <br /> ing levels hitherto unrecognized functional biochemical          accurate phylogenies using multiple concatenated pro- <br /> diversity fungal species.                                         tein sequence alignments. We used maximum likelihood ap- <br />                                                                         proach order construct phylogenetic trees based concat- <br />                                                                         enated sequence alignments multiple protein families. <br />                                                                         Maximum likelihood methods thought seriously <br /> Results                                                                 influenced saturation, resulting long-branch artifacts, in <br />                                                                         comparison parsimony distance-based methods (Felsen- <br /> Comparative genomics Fungi <br />                                                                         stein 1978; Guindon Gascuel 2003). <br /> Our aim organize available genomic data               Protein families selected constructing sequence <br /> form readily used assess evolutionary relat-       similarity trees using protein clusters spanned ge- <br /> edness fungal species generate clusters re-        nomes showed minimal duplication genomes. The <br /> lated sequences. The 36 genomes used analysis listed        proteins obtained (see Supplemental Table 1) generally in- <br /> in Table 1. There clearly large differences size fungal   volved fundamental processes translation, transcrip- <br /> genomes number predicted protein sequences. They             tion, DNA replication recombination. Because the <br /> range size 2.5 Mb, encoding 1996 predicted open reading         great divergence species analysis, small <br /> frames (ORFs), microsporidian E. cuniculi, &gt;46 Mb, en-       size E. cuniculi genome, sets protein families were <br /> coding 17,467 ORFs, R. oryzae. Among Ascomycete            used resolve phylogeny. A well-resolved broader species <br /> species, Pezizomycotina larger genomes Sac-           tree based concatenated sequences 12 protein families, <br /> charomycotina: general, Saccharomycotina genomes            included Homo sapiens model plant species Arabi- <br /> sizes 8 12 Mb, Pezizomycotina be-             dopsis thaliana, clearly separates fungal, animal, plant taxa <br /> tween 29 39 Mb. The exception Yarrowia lipolytica,         maximum bootstrap support, shown Figure 1A. The <br /> has 20-Mb genome&#226;&#8364;&#8221;nearly twice large            oomycete species shown diverged prior diver- <br /> largest Saccharomycotina. However, despite larger,         gence fungi animals. This agreement previous <br /> the Y. lipolytica genome does appear encode significantly        phylogenetic analysis based small subunit rRNA sequences <br /> more protein sequences Saccharomycotina ge-              (Baldauf et al. 2000). E. cuniculi, appearing re- <br /> nomes. There 6544 ORFs predicted Y. lipolytica compared         solved, long branch, indicative rapidly evolving <br /> to 6317 Debaromyces hansenii.                                        genes. Again, consistent previous phylogenetic analy- <br />      In order generate clusters proteins conserved       sis (Thomarat et al. 2004). <br /> among particular fungal species, translated amino acid sequences              A second set 29 protein families used generate a <br /> were compared using BLASTP (Altschul et al. 1990). Se-            tree resolved deeper nodes, shown Figure 1B. Using <br /> quences clustered basis similarity using      larger protein set, high levels support achieved <br /> an E-value cutoff 10 &#226;&#179;&#175; 10&#226;&#179;&#174;5 (for details, Methods).         majority branches. The tree shows divergence <br /> This method generates clusters solely basis similarity,       Zygomycetes followed Basidiomycetes mark <br /> and distinction orthologs paralogs.              earliest bifurcations fungal tree, followed split of <br /> However, using information stored e-Fungi database,          S. pombe (Taphrinomycotina) rest Ascomycetes. <br /> we able distinguish orthologs paralogs          The Ascomycetes divide Pezizomycotina the <br /> running analyses. Orphan proteins, i.e.,          Saccharomycotina. There support tree previous <br /> cluster, classified sufficiently       sequence-based phylogenetic studies. These results con- <br /> similar proteins genomes.         sistent recent phylogenies fungi (Fitzpatrick et <br /> This reasons. First, orphans arise        al. 2006; Galagan et al. 2006; James et al. 2006), confirm <br /> due errors genome annotation. For example, sequenc-           well-supported aspects species tree. <br /> ing members Saccharomyces &#226;&#8364;&#339;sensu stricto&#226;&#8364;? complex                Phylogenetic profiling, uses presence absence <br /> two different research groups (Cliften et al. 2003; Kellis et al.       derived character states genes gene families, is <br />  <br />  <br /> 1810      Genome Research <br />             www.genome.org <br />                                                                                    Comparative genome analysis fungal kingdom <br />  <br />  <br />                    Table 1.   Overview MCL clusters <br />  <br />                                                    Genome size      Total    ORFs found <br />                    Genome                             (Mb)a         ORFs     clusters                Data source <br />  <br />                    Saccharomyces cerevisiaeb           12.2         5869         5529        NCBIc <br />                    Saccharomyces paradoxusd            11.9         8955         6630        Saccharomyces Genome Databasee <br />                    Saccharomyces mikataed              11.5         9057         6495        Saccharomyces Genome Databasee <br />                    Saccharomyces mikatae f             10.8         3100         2975        Saccharomyces Genome Databasee <br />                    Saccharomyces kudriavzevii f        11.2         3768         3727 <br />                    Saccharomyces bayanusd              11.5         9424         6218        Saccharomyces Genome Databasee <br />                    Saccharomyces bayanus f             11.9         4966         4943        Saccharomyces Genome Databasee <br />                    Saccharomyces castellii f           11.4         4677         4549        Saccharomyces Genome Databasee <br />                    Saccharomyces kluyveri f            11.0         2968         2938        Saccharomyces Genome Databasee <br />                    Candida glabratab                   12.3         5180         4848        NCBIc <br />                    Kluyveromyces waltii                10.6         5230         5020        Kellis et al. 2004g <br />                    Eremothecium gossypii b              8.8         4718         4582        NCBIc <br />                    Kluyveromyces lactisb               10.7         5326         4989        NCBIc <br />                    Candida albicansh                   27.6        14213        13647        NCBIc <br />                    Candida lusitaniae                  12.1         5941         5243        Broad Institutei <br />                    Debaryomyces hansenii b             12.2         6310         5679        NCBIc <br />                    Yarrowia lipolyticab                20.6         6544         5536        NCBIc <br />                    Aspergillus nidulans                30.1        10701         9367        Broad Institutei <br />                    Aspergillus niger                   37.2        11200        10072        DOE Joint Genome Institutej <br />                    Aspergillus fumigatus               29.4         9923         8841        CADRE Databasek <br />                    Aspergillus oryzae                  37.2        12062        10199        DOGAN Databasel <br />                    Aspergillus terreus                 29.3        10406         9253        Broad Institutei <br />                    Coccidioides immitis                28.9        10457         7949        Broad Institutei <br />                    Stagonospora nodorum                37.2        16597        10461        Broad Institutei <br />                    Botrytis cinerea                    42.7        16448        10238        Broad Institutei <br />                    Sclerotinia sclerotiorum            38.3        14522         9947        Broad Institutei <br />                    Gibberella zeae                     36.1        11640        10048        Broad Institutei <br />                    Trichoderma reesei                  34.5         9783         8302        DOE Joint Genome Institutej <br />                    Magnaporthe grisea                  38.8        12841         9816        Broad Institutei <br />                    Neurospora crassa                   38.0         9826         7633        Broad Institutei <br />                    Chaetomium globosum                 34.9        11124         9121        Broad Institutei <br />                    Schizosaccharomyces pombeb          12.5         4416         3212        NCBIc <br />                    Ustilago maydis                     19.7         6522         5211        Broad Institutei <br />                    Phanerochaete chrysosporium         30.0        11776         6927        DOE Joint Genome Institutej <br />                    Rhizopus oryzae                     46.1        17467        14092        Broad Institutei <br />                    Encephalitozoon cuniculi b           2.5         1996         1079        NCBIc <br />                    Phytophthora sojae                  86.0        19071        15578        DOE Joint Genome Institutej <br />                    Phytophthora ramorum                66.7        15890        13848        DOE Joint Genome Institutej <br />  <br />                    The numbers ORFs listed Table 1 Y. lipolytica D. hansenii differ previously published <br />                    (6703 6906) (see Table 2; Dujon et al. 2004). This appears to removal pseudogenes <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> <br />                    analysis. When loading data e-Fungi database, identified 151 Y. lipolytica 579 D. hansenii <br />                    CDS sequences annotated pseudogenes. <br />                    a <br />                      Genome sizes estimated total size chromosomes contigs. <br />                    b <br />                      Genomes complete chromosome sequences available. <br />                    c <br />                      NCBI (http://www.ncbi.nlm.nih.gov/Genomes/). <br />                    d <br />                      Genome sequenced Kellis et al. 2003. <br />                    e <br />                      Saccharomyces Genome Database (ftp://genome.ftp.stanford.edu/pub/yeast/data_download/sequence/ <br />                    fungal_genomes/). <br />                    f <br />                      Genome sequenced Cliften et al. 2003. <br />                    g <br />                      Kellis et al. 2004, Supplemental data (http://www.nature.com/nature/journal/v428/n6983/extref/ <br />                    nature02424-s1.htm). <br />                    h <br />                      Genome sequence C. albicans diploid form (Jones et al. 2004). <br />                    i <br />                     Broad Institute (http://www.broad.mit.edu/annotation/fgi/). <br />                    j <br />                     DOE Joint Genome Institute (http://www.jgi.doe.gov). <br />                    k <br />                      CADRE Database (http://www.cadre-genomes.org.uk). <br />                    l <br />                     DOGAN Database (http://www.bio.nite.go.jp/dogan/MicroTop?GENOME_ID=ao). <br />  <br />  <br /> commonly used tree construction (Fitz-Gibbon House                  absence), constructed 23,230 protein clusters present and <br /> 1999; Wolf et al. 2002; Huson Steel 2004). The Dollo parsi-             absent Ascomycetes, Basidiomycetes, Zygomycetes, <br /> mony method (Farris 1977), based model derived                  subjected Dollo parsimony method. The resulting <br /> character states lost subsequently regained,          phylogenetic tree shown Figure 1C. In general, Dollo <br /> applicable phylogenetic profiles reconstruction          parsimony tree supports sequenced-based tree. There dif- <br /> the parsimonious evolutionary paths. Huson Steel                   ferences positions members Saccharomyces <br /> (2004) showed approach perform favorably compari-                &#226;&#8364;&#339;sensu stricto&#226;&#8364;? complex, attributed dif- <br /> son distance-based methods constructing gene content                 ferences reporting ORFs different sequencing <br /> trees. Since presence/absence data character states natu-        groups, clearly affect analysis based gene <br /> rally treated binary characters, matrix 1/0 (presence/           presence absence. The major differences trees <br />  <br />  <br />                                                                                                                    Genome Research     1811 <br />                                                                                                                       www.genome.org <br />  Cornell et al. <br />  <br />  <br />  <br />  <br /> Figure 1. Species trees. (A) Broad species tree based concatenated sequences 30 universal protein clusters using maximum likelihood <br /> approach. (B) Basidiomycete Ascomycete species tree based 60 universal fungal proteins. (C) Fungal phylogenetic tree based Dollo parsimony. <br /> Strongly supported branches agree sequence-based fungal species tree shown B. <br />  <br />  <br /> are, firstly, Dollo parsimony analysis places S. pombe              ing Neurospora crassa genome shown RIP activity <br /> next Y. lipolytica, division Ascomycetes        resulted presence duplicated genes (Gala- <br /> Pezizomycotina Saccharomycotina. Therefore, ba-               gan Selker 2004). <br /> sis gene presence/absence, S. pombe closer Saccharomyces                  A previous study (Dujon et al. 2004) analyzed genome re- <br /> cerevisiae expected basis sequence con-              dundancy Saccharomycotina yeast genomes (S. cerevisiae, <br /> servation. Secondly, Dollo parsimony analysis does po-               C. glabrata, Kluyveromyces lactis, D. hansenii, Y. lipolytica). We <br /> sition Coccidioides immitis Aspergillus species,             expanded analysis include 14 Pezizomycotina spe- <br /> places diverged member Pezizomycotina.                 cies, Taphrinomycotina S. pombe, Basidiomycetes, Zy- <br />                                                                              gomycete, Microsporidian. The Candida albicans genome <br />                                                                              excluded analysis sequence used <br /> Gene duplication Fungi                                                refers normal diploid form fungus and, therefore, we <br /> Gene duplication major force evolution, allowing ac-             expect minimum copies protein- <br /> quisition new functions. Duplicated genes retained              encoding sequence. Protein clusters analyzed determine <br /> they provide advantage, increased gene               contained protein given <br /> dosage effect increased functional diversification.            fungal species. The result analysis shown Supplemen- <br /> Alternatively, duplicated gene lost. In evolution             tal Figure 1A. For Saccharomycotina genomes, results are <br /> of S. cerevisiae Candida globrata genomes, following               consistent previously published study (Dujon et <br /> whole-genome duplication (WGD), majority dupli-                   al. 2004); level duplication D. hansenii <br /> cated genes lost. In Pezizomycotina species, repeat-             genome appears lower analysis. S. cerevisiae (438 duplica- <br /> induced <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span>point mutation</span> (RIP; mechanism active re-                  tion-containing clusters) twice duplica- <br /> moval duplicated sequences) identified. The sequenc-             tions K. lactis Kluyveromyces waltii (206 181 clusters, <br />  <br />  <br /> 1812      Genome Research <br />             www.genome.org <br />                                                                               Comparative genome analysis fungal kingdom <br />  <br />  <br /> respectively), diverged prior WGD event. C. glabrata       ronment. It proposed duplication dif- <br /> appears fewer duplication-containing clusters S. cer-      ferent sets genes promote speciation (Lynch Force <br /> evisiae. Only 325 identified, indicating greater gene loss         2000) and, conversely, parallel duplication sets <br /> post-WGD. Y. lipolytica possesses greatest number highly         genes allow distantly related species adapt the <br /> duplicated clusters. Forty-seven clusters containing          niche (Hughes Friedman 2003). An analysis S. <br /> five proteins identified, compared 10 S. cer-        cerevisiae S. pombe genomes shown independent <br /> evisiae. Dujon et al. (2004) K. lactis lowest        gene duplication gene families occurred far <br /> level duplication yeast genomes analyzed,           greater extent expected chance (Hughes Friedman <br /> analysis E. gossypii genome indicates less,     2003). <br /> possibly reflection possessing smallest genome             We investigated parallel duplication pairs of <br /> Saccharomycotina analyzed.                                              fungal species identifying instances coduplication pro- <br />       Gene duplication Pezizomycotina, general,            tein clusters. The frequency clusters occurred <br /> appears slightly higher Saccharomycotina.          compared expected frequency based duplications <br /> The exceptions N. crassa C. immitis (372 374 clusters,      occurring independently species using &#226;?&#185;2 analysis. <br /> respectively), possess fewer duplication-containing          Some results analysis shown Table 2, the <br /> clusters S. cerevisiae. Among Aspergillus genomes, Asper-      remainder Supplemental Table 2. Our analysis does not <br /> gillus niger Aspergillus oryzae (which positioned       distinguish independently occurring duplications and <br /> each phylogenetic tree) possess duplica-          occurred ancestral species. Possibly result of <br /> tion-containing clusters. In Basidiomycetes, duplication         this, obtain significant chi-squared results pairs of <br /> the P. chrysosporium genome (885 clusters) appears higher          fungi. However, able identify instances &#226;?&#185;2 <br /> than U. maydis (300 clusters). There little duplication   score agreement phylogenetic tree. For ex- <br /> in E. cuniculi genome, likely reflection small        ample, S. cerevisiae, highest chi-squared score obtained <br /> genome size. In contrast, Zygomycete R. oryzae possesses         C. glabrata, scores decrease Saccharomy- <br /> far duplication clusters (2481) fungi analyzed,     cotina species, Y. lipolytica having score similar of <br /> almost times highest, P. chrysosporium.       Pezizomycotina. The score S. pombe higher than <br /> In order ensure results presence      Y. lipolytica, agreement parallel duplication ob- <br /> of transposons, analysis repeated protein clusters         served Hughes Friedman (2003). Both Basidiomycete spe- <br /> containing likely transposons removed. This did greatly alter       cies low scores, score R. oryzae unexpectedly <br /> the results analysis (see Supplemental Fig. 1B);        high, possibly indicating adaptation similar ecological niche. <br /> some fungal genomes appear contain large numbers trans-           Both S. cerevisiae R. oryzae environments <br /> posons, sequence similarity elements means            associated rotting fruit, true many <br /> they occur small number clusters.                          fungal species (Wheeler et al. 2003; Tournas Katsoudas <br />       As discussed above, low number duplications         2005). Alternatively, high level codupli- <br /> N. crassa genome result RIP activity. Orthologs          cation reflects fact WGD event the <br /> enzyme responsible, DNA methyltransferase encoded rid <br /> gene, identified Pezizomycotina (Galagan         Table 2.   Chi-squared scores coduplication OrthoMCL <br /> Selker 2004), evidence RIP activity Tan1 trans-      clusters <br /> posable element sequences genome A. oryzae (Montiel et <br />                                                                         Clusters          S. cerevisiae         S. pombe            R. oryzae <br /> al. 2006). Our analysis shows N. crassa fewer duplication- <br /> containing clusters Pezizomycotina C. immi-         Scer                  &#226;&#8364;&#8221;                   300.1              116.4 <br /> tis, suggesting RIP process active N. crassa   Cgla                1244.1                 94.6               25.2 <br /> other Pezizomycotina species. This agreement experi-         Kwal                 566.4                100.1               14.0 <br /> mental evidence shows RIP active Magnaporthe       Klac                 476.7                 94.8               17.8 <br />                                                                         Egos                 528.2                 89.1               21.6 <br /> grisea N. crassa (Ikeda et al. 2002). To investigate    Dhan                 350.2                 99.7                6.1 <br /> this, repeated analysis Galagan Selker (2004)         Clus                 393.0                 54.5               24.9 <br /> analyzed percentage identity best BlastP hits        Ylip                  88.2                 15.1                1.0 <br /> species. The results (shown Supplemental Fig. 2) demonstrate         Cimm                  21.0                  8.6                2.5 <br />                                                                         Anid                  57.3                 17.2                0.2 <br /> that great variability Pezizomycotina terms      Afum                  63.5                 16.8                2.0 <br /> of number proteins high percentage identity. For M.         Ater                  51.2                 14.3                0.0 <br /> grisea, 911 proteins BlastP hits identity 90%          Anig                  35.9                 12.9                8.0 <br /> and 100%, Sclerotinia sclerotiorum 529 Chaetomium         Aory                  54.7                 11.0                1.8 <br />                                                                         Snod                  23.0                  0.5               31.3 <br /> globosum 419. In contrast, Gibberella zeae 21, A. oryzae 34, A. <br />                                                                         Bcin                  84.9                 22.8                0.1 <br /> niger 68, N. crassa 61. Again, result supports sugges-     Sscl                  66.5                 22.2                0.4 <br /> tion differing levels RIP activity Pezizomy-       Gzea                  30.9                  6.5               17.4 <br /> cotina species. Among non-Pezizomycotina species analyzed,          Tree                  71.1                 17.2                0.9 <br /> the intriguing result large number R. oryzae pro-        Mgri                  16.4                  3.4               16.1 <br />                                                                         Ncra                  56.9                 16.0                0.8 <br /> teins identity scores 90% 100%. A total 4231        Cglo                  24.5                  2.6               16.5 <br /> proteins identified, 10 times S. cerevi-      Spom                 300.1                 &#226;&#8364;&#8221;                  19.8 <br /> siae (435). This result indicates recent large-   Pchr                  12.9                  2.8               29.1 <br /> scale duplication R. oryzae genes.                                   Umay                   9.5                 22.9                2.8 <br />                                                                         Rory                 116.4                 19.8               &#226;&#8364;&#8221; <br />       Gene duplication lead organism possessing genes         Ecun                   3.3                  0                  4.9 <br /> of novel function, potentially improving adaptation envi- <br />  <br />  <br />                                                                                                             Genome Research           1813 <br />                                                                                                                www.genome.org <br />  Cornell et al. <br />  <br />  <br /> evolution S. cerevisiae and, discussed above, large-scale du-      twice proteins containing motifs Saccha- <br /> plication R. oryzae genome. These examples         romycotina species. This expansion includes frequently <br /> parallel duplication specific pairs, scores C.     identified Pfam motifs&#226;&#8364;&#8221;PF00172, PF04082, PF00096, PF00098, <br /> glabrata S. pombe, C. glabrata R. oryzae        PF00170. <br /> much lower, despite S. cerevisiae C. glabrata closely                Protein clusters expanded Saccharomy- <br /> related. It interesting note appears little       cotina compared Pezizomycotina rarer, likely reflec- <br /> parallel duplication S. pombe R. oryzae.                     tion fact Saccharomycotina fewer predicted <br />       To investigate genetic basis yeast filamentous      ORFs. The clusters identified associated <br /> growth habits fungi, analyzed Pfam motifs associated          cell wall structure biosynthesis. There large expan- <br /> proteins Saccharomycotina Pezizomycotina species,            sion number proteins containing PIR motifs (PF00399). <br /> identify motifs expansion protein            These motifs directly link 1,3-beta-glucan impor- <br /> numbers Pezizomycotina compared Saccharomy-               tant cell wall anchoring (Kapteyn et al. 1999; Castillo et al. <br /> cotina vice versa. Table 3 lists examples motifs iden-        2003). S. cerevisiae Pir2p (currently Hsp150p) shown to <br /> tified. Many motifs expanded Pezizomycotina indi-          locate cell wall G. zeae, suggesting incorpora- <br /> cate increased metabolic flexibility compared Saccharomy-         tion mechanism ASL-CWPs exists Saccharomycotina <br /> cotina. For example, expansions protein families            Pezizomycotina (Narasimhan et al. 2003). Our analysis iden- <br /> involved transport cell. Major facilitator        tified proteins containing PIR domains Pezizomycotina <br /> superfamily transporters (PF07690), involved transport         species. However, failed identify PIR proteins of <br /> small solutes, including sugar uptake drug efflux (Maiden et         Aspergillus species C. immitis, appear be <br /> al. 1987; Griffith et al. 1992), sugar transporter (PF00083) pro-        absent C. globosum. We unable identify PIR <br /> teins, ABC transporters (PF00005), identi-           motifs non-Ascomycete species. There ex- <br /> fied multidrug resistance proteins (Jungwirth Kuchler             pansions motifs associated cell wall synthesis. <br /> 2006), expansion. There expansion motifs          These include glucan synthases (PF02364), catalyze the <br /> associated utilization different carbon sources, e.g.,       formation beta-1,3-glucan, PMT (PF02366) MIR <br /> the alcohol dehydrogenase domains PF08240 PF00106.                   (PF02815) domains associated mannosyltransferase activity. <br /> Analysis cytochrome P450 proteins (PF00067) shows                These differences probably consequence difference in <br /> there huge expansion number P450 proteins           chemical composition cell wall Saccharo- <br /> in Pezizomycotina compared Saccharomycotina. The              mycotina Pezizomycotina. The cell walls filamentous <br /> numbers P450 proteins identified Pezizomycotina species            ascomycetes, Neurospora Aspergillus species, contain <br /> are broad agreement previous analysis (Deng et al.             far greater quantities chitin, major structural component <br /> 2007). The P450 family highly diverged, P450 proteins             hyphal wall, yeasts, chitin confined <br /> have range roles, including degradation toxins       bud scars (for review, Bowman Free 2006). In line <br /> production secondary metabolites, important fun-           this, proteins Pfam domains associated chitin are <br /> gal adaptation (Deng et al. 2007). As expansion       frequently Pezizomycotina (see Supple- <br /> Pezizomycotina, analysis indicates P. chrysosporium and,        mental Fig. 3). <br /> to lesser extent, R. oryzae possess large numbers P450s. <br /> The low number cytochrome P450 proteins Sac- <br />                                                                          Protein family conservation diversification among <br /> charomycotina shown S. pombe, possesses only <br />                                                                          Fungi <br /> two. <br />       In addition expansion Pfam motifs associated        To investigate gene orthology different species, ana- <br /> responses environmental stresses resources,           lyzed co-occurrence genomes particular protein clusters. <br /> expansion motifs associated regulation gene ex-           This provides indication particular orthologous <br /> pression. Analysis 84 Pfam motifs associated DNA bind-           gene gene family conserved particular fungal species <br /> ing shows that, average, Pezizomycotina species possess al-           provides powerful means analyzing conservation fun- <br />  <br />  <br /> Table 3. Expansion Pfam motifs Pezizomycotina Saccharomycotina <br /> Pfam <br /> motifs    Scer Cgla Kwal Klac Egos Dhan Clus Ylip Afum Anid Anig Aory Ater Bcin Cglo Cimm Gzea Mgri Ncra Sscl Snod Tree <br />  <br /> PF07690   45    34    63    56    31    88    61   123   220   267     379   415   303 135   168   127   251    176   111 181    299   166 <br /> PF00083   33    18    26    23    13    56    26    27    86   108      98   121   115  42    49    24   106     64    36 55      89    61 <br /> PF00005   31    25    28    23    20    31    34    32    52    49      71    92    58  47    55    49    59     47    35 57      61    57 <br /> PF00067    3     3     3     5     3     9     8    17    71   117     149   154   115  77    89    43   107    132    39 90     146    71 <br /> PF00106   14    13    17    18    16    35    22    28   113   157     196   170   148  87    98    61   149    121    61 92     157   123 <br /> PF08240   20    12    18    16     9    34    16    20    59    73     118   105    82  30    52    29    71     55    28 53      78    49 <br /> PF00172   52    39    47    58    40    88    69    50   188   239     278   202   184  75   132    95   260    111    90 106    155   188 <br /> PF00096   38    44    33    30    32    37    45    34    52    61      61    61    66  40    79    63    65     56    55 72     106    44 <br /> PF00399   10    11     6     5     3     2     2     8     0     0       0     0     0   1     0     0     1      1     1   1      2     1 <br /> PF02364    3     3     1     3     3     2     6     2     1     1       1     1     1   1     1     1     1      1     1   1      1     1 <br /> PF02366    6     6     5     4     4     4     5     4     3     3       3     3     3   2     3     3     3      3     3   3      3     3 <br /> PF02815    7     7     5     5     4     4     5     4     3     3       3     3     3   2     3     3     3      3     3   4      3     3 <br />  <br /> Pfam motifs: PF07690 (MFS_1), PF00083 (Sugar_tr), PF00005 (ABC_tran), PF00067 (p450), PF00106 (adh_short), PF08240 (ADH_N), PF00172 <br /> (Zn_clus), PF00096 (zf-C2H2), PF00399 (PIR), PF02364 (Glucan_synthase), PF02366 (PMT), PF02815 (MIR). <br />  <br />  <br /> 1814      Genome Research <br />            www.genome.org <br />                                                                                  Comparative genome analysis fungal kingdom <br />  <br />  <br /> gal species exhibiting distinct growth habits lifestyles. Figure       conserved 4&#226;&#8364;&#8220;6, 9&#226;&#8364;&#8220;11, 13&#226;&#8364;&#8220;15 species deter- <br /> 2A shows relative frequency genome numbers protein              mined. For clusters peaks, genome scored as <br /> clusters species: S. cerevisiae, C. glabrata, S. pombe.     present absent, percentage clusters a <br /> The plots similar S. cerevisiae C. glabrata, reflecting   given genome present calculated. The results dis- <br /> fact positioned close phylogenetic          played Figure 2B. The clusters containing 4&#226;&#8364;&#8220;6 genomes are <br /> tree. There extra peak S. cerevisiae genomes, re-       clearly composed proteins closely related mem- <br /> flecting close relationship members Sac-        bers Saccharomyces &#226;&#8364;&#339;sensu stricto&#226;&#8364;? complex&#226;&#8364;&#8221;Saccharomyces <br /> charomyces &#226;&#8364;&#339;sensu stricto&#226;&#8364;? clade, which, indeed, in-          paradoxus, Saccharomyces mikatae, Saccharomyces bayanus, and <br /> terbreed. We expand relationships below.                         Saccharomyces kudriavzevii. It apparent S. kudriavzevii a <br />       The composition protein clusters encoded S. cerevisiae        lower score &#226;&#8364;&#339;sensu stricto&#226;&#8364;? genomes, reflecting the <br />  <br />  <br />  <br />  <br />                                                       Figure 2.   (Continued page) <br />  <br />  <br />                                                                                                               Genome Research           1815 <br />                                                                                                                  www.genome.org <br />  Cornell et al. <br />  <br />  <br />  <br />  <br /> Figure 2. Protein family conservation fungal species. (A) The graph shows relative frequency numbers species represented clusters <br /> containing S. cerevisiae (black), C. glabrata (gray), S. pombe (dashed). (B,C) The graphs fraction protein clusters containing a <br /> representative fungal genome. Protein clusters selected basis following criteria: (B) Protein clusters containing S. cerevisiae and <br /> a total genomes (black), 11 genomes (gray), 13 to17 genomes (white). (C) Protein clusters containing S. pombe total <br /> of 13 24 genomes including S. cerevisiae (black) G. zeae (white). <br />  <br />  <br /> lower number ORFs reported species (Cliften et al.                  clusters containing 13&#226;&#8364;&#8220;15 genomes shows C. albicans, D. <br /> 2003). This finding consistent phylogenetic analysis,               hansenii, Y. lipolytica occur majority clus- <br /> which showed species closely related. Two                 ters, fraction reduces sharply Pezizomycotina. <br /> other Saccharomyces species, Saccharomyces castellii Saccharo-              It interesting note despite S. pombe, P. chrysosporium, U. <br /> myces kluyveri, members Saccharomyces &#226;&#8364;&#339;sensu lato&#226;&#8364;?                   maydis, R. oryzae diverged S. cerevisiae than <br /> group, (unlike &#226;&#8364;&#339;sensu stricto&#226;&#8364;? clade) unable                   filamentous fungi, occur greater fraction these <br /> mate S. cerevisiae (Marinoni et al. 1999). Our phylogenetic                protein clusters. This result agreement positioning <br /> tree confirms distantly related S. cerevisiae,            S. pombe closer expected S. cerevisiae Dollo parsi- <br /> and protein products genes far fewer               mony tree higher expected coduplication results for <br /> these clusters (&#226;&#710;&#188;11% 1.4%, respectively). It appears, there-                S. cerevisiae, S. pombe, R. oryzae. It suggests targeted gene <br /> fore, peak 4&#226;&#8364;&#8220;6 genomes represents group proteins               loss occurred, indicating non-Ascomycete <br /> specific members Saccharomyces &#226;&#8364;&#339;sensu stricto&#226;&#8364;? com-                   species retained sequences Saccharomy- <br /> plex, largely absent budding yeasts.                       cotina lost Pezizomycotina. <br />       We analyzed set 977 clusters containing essential S.                        Gene Ontology annotation proteins used identify <br /> cerevisiae proteins. The distribution cluster sizes obtained using           GO terms over-represented clusters associated with <br /> this set different obtained using S. cerevisiae pro-           Saccharomycotina (those containing 1&#226;&#8364;&#8220;15 species) compared to <br /> teins, peak 4&#226;&#8364;&#8220;6 proteins absent (see Supplemental Fig. 4).            associated fungal species (containing 30&#226;&#8364;&#8220;36 spe- <br /> Only essential genes peculiar Saccharomyces               cies). There inherent bias analysis, proteins that <br /> &#226;&#8364;&#339;sensu stricto&#226;&#8364;? clade identified: KAR1 (YNL188W), SPC29                    possess orthologs Saccharomycotina less <br /> (YPL124W), TEN1 (YLR010C). Kar1p essential protein                    likely GO annotations. For Saccharomycotina-specific <br /> involved nuclear fusion mating spindle pole                    clusters, GO terms associated fungal cell wall <br /> body duplication mitosis (Pereira et al. 1999). Spc29p                (GO:0009277), Saccharomycotina-specific transcription fac- <br /> also component spindle pole body, linking central                  tors (GO:0003704, GO:0016455), components spindle <br /> plaque component Spc42p inner plaque component                           pole body (GO:0005816) appear over-represented. <br /> Spc110p (Elliott et al. 1999). Ten1p involved regulation                     As Figure 2A shows, frequency distribution protein <br /> of telomere <span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span> nuclear telomere cap complex                  clusters obtained S. pombe quite different ob- <br /> (Grandin et al. 2001).                                                          tained budding yeasts. There broad peak the <br />       Analysis protein clusters containing 9&#226;&#8364;&#8220;11 genomes shows                righthand side, representing proteins common many <br /> that proteins S. castellii S. kluyveri occur                fungal genomes, sharp peaks representing <br /> majority clusters, C. glabrata, E.                 branches phylogenetic tree. Instead, broad peak <br /> gossypii, K. lactis. The fraction clusters genome             15 genomes. Further analysis shows is <br /> is reduces sharply C. albicans D. hansenii. Again,            actually overlapping peaks. Figure 2C shows species <br /> is consistent phylogenetic relationships. Analysis                  represented clusters containing S. pombe S. cerevisiae <br />  <br />  <br /> 1816      Genome Research <br />              www.genome.org <br />                                                                               Comparative genome analysis fungal kingdom <br />  <br />  <br /> proteins, proteins S. pombe filamentous fungus         glyoxylate cycle). In pathway, initial reaction each <br /> G. zeae. The results clearly clusters divided   round &#226;?&#164;-oxidation catalyzed acyl-CoA dehydrogenase <br /> into subgroups budding-yeast-specific proteins filamen-         (Thieringer Kunau 1991). Recent studies suggest N. <br /> tous-fungi-type proteins. This implies that, S. pombe         crassa lack peroxisomes (Schliebs et al. 2006). <br /> split rest fungi prior divergence              By surveying available fungal genomes presence <br /> budding yeasts filamentous fungi, retained set         absence acyl-CoA dehydrogenases acyl-CoA oxidases, it <br /> budding-yeast-type proteins (which subsequently lost         established organism peroxisomal or <br /> the filamentous fungi) set filamentous-fungi-type pro-        non-peroxisomal pathways &#226;?&#164;-oxidation fatty acids (Fig. <br /> teins (which lost budding yeasts). The GO           3). The results suggest members class Saccharo- <br /> annotations associated proteins clusters       mycetes, Y. lipolytica, possess peroxisomal <br /> analyzed. There 102 clusters specific S. pombe Sac-     &#226;?&#164;-oxidation pathway. The S. cerevisiae genome contains one <br /> charomycotina compared 464 specific S. pombe             acyl-CoA oxidase gene, encodes enzyme ac- <br /> Pezizomycotina. Among S. pombe Saccharomycotina                cept fatty acid chains differing lengths (Hiltunen et al. 2003). <br /> clusters, identified containing nuclear pore proteins          This case C. glabrata, K. waltii, K. lactis, E. gossypii, <br /> (GO:0006913). There S. pombe/Pezizomycotina clusters           Saccharomyces species. Like N. crassa, species <br /> with annotation. Among S. pombe/Pezizomycotina clus-          filamentous ascomycete (M. grisea Trichoderma reesei) lack <br /> ters, identified 16 clusters associated cellular transport     predicted peroxisomal &#226;?&#164;-oxidation pathway. All species <br /> (GO:0006810), 16 DNA repair (GO:0006281),               filamentous ascomycetes, dimorphic species Y. lipolytica, <br /> meiosis, seven associated progression S phase         Basidiomycetes, Zygomycetes, Oomycetes have <br /> in mitotic cell cycle (GO:0000084). Analysis clusters con-      peroxisomal non-peroxisomal &#226;?&#164;-oxidation pathways. <br /> taining Y. lipolytica proteins shows contains set          It striking, however, fission yeast S. pombe the <br /> Pezizomycotina-associated proteins lost            obligate intracellular parasite E. cuniculi possess <br /> other Saccharomycotina species (see Supplemental Fig. 5).              &#226;?&#164;-oxidation pathway. This observation con- <br />                                                                        firmed lack homolog S. cerevisiae FOX2 gene <br />                                                                        (which encodes multifunctional &#226;?&#164;-oxidation protein) the <br /> Analysis fatty acid &#226;?&#164;-oxidation pathways Fungi <br />                                                                        genomes of fungi. The lack &#226;?&#164;-oxidation has <br /> To investigate diversity conservation key metabolic         noted case E. cuniculi surprising, <br /> pathways Fungi, protein cluster data combined           unicellular eukaryote does possess mitochondria or <br /> with metabolic pathway data LIGAND database (Goto             peroxisomes (Katinka et al. 2001), observation the <br /> et al. 2002). Protein clusters contained components meta-      well-studied fission yeast S. pombe lacks pathway a <br /> bolic pathways analyzed determine            novel finding emerged comparative genomic <br /> genomes represented. These subsequently mapped               analysis. In order search evidence S. pombe <br /> onto known metabolic pathways investigate multiple          &#226;?&#164;-oxidation, identified 72 protein clusters containing Pfam <br /> pathway components missing species.                      motifs associated acyl-CoA dehydrogenases (PF00441, <br />       Fatty acid &#226;?&#164;-oxidation process degrades fatty acids    PF08028, PF02770, PF02771). None clusters contains <br /> into acetyl-CoA, fed TCA cycle produce       S. pombe protein. <br /> ATP (in plants fungi) used synthesize carbohy-              The genome dimorphic yeast Y. lipolytica contains six <br /> drates glyoxylate cycle gluconeogenesis. Fatty acids       acyl-CoA oxidase-encoding genes. Five cloned and <br /> are activated addition coenzyme A, reaction cata-       products shown different substrate chain-length <br /> lyzed fatty acyl-CoA ligases concomitant         preferences (Wang et al. 1999). This fact unreasonable in <br /> transport. Subsequent steps bring complete oxidation         light variety hydrocarbons fatty acids yeast can <br /> of &#226;?&#164;-carbon release acetyl-CoA. The resulting acyl-     use source carbon. The species Phytophthora also <br /> CoA, shortened carbon units, undergo additional             contain large numbers genes encoding acyl-CoA oxidases; <br /> &#226;?&#164;-oxidation cycles. In mammals, &#226;?&#164;-oxidation occurs         likely different substrate specificities. Likewise <br /> mitochondria peroxisomes (Kunau et al. 1995). One           genomes filamentous Ascomycetes, Basidiomycetes, <br /> ways pathways differ enzyme catalyzes         Zygomycetes, yeasts C. albicans, Candida lus- <br /> initial reaction round acyl-CoA oxidation. In           itaniae, D. hansenii, possess multiple genes en- <br /> mitochondrial pathway, catalyzed acyl-CoA dehydro-          zymes. <br /> genase (EC 1.3.99.3), electrons produced reaction             The fact mammals fungi possess per- <br /> being passed electron transport chain produce ATP. In      oxisomal non-peroxisomal &#226;?&#164;-oxidation pathways suggests <br /> the peroxisomal pathway, initial reaction catalyzed         that, evolution species fungi, both <br /> acyl-CoA oxidase (EC 1.3.3.6), electrons passed             pathways lost. Relating fungal <br /> directly molecular oxygen producing hydrogen peroxide,              phylogeny generated study (Fig. 1), evidence suggests <br /> which detoxified catalase. In yeasts, S. cerevisiae,     non-peroxisomal &#226;?&#164;-oxidation lost divergence <br /> &#226;?&#164;-oxidation occurs solely peroxisomes (Hiltunen et al. 2003).       ancestor Y. lipolytica rest Saccharomy- <br /> A genetic study A. nidulans suggested that, filamentous     cetes. Strictly peroxisomal &#226;?&#164;-oxidation lost <br /> Ascomycete, &#226;?&#164;-oxidation occurs peroxisomes mito-           divergence ancestor N. crassa M. grisea from <br /> chondria, specific short-chain fatty acids        ancestor G. zeae, lineage T. reesei, and <br /> (Maggio-Hall Keller 2004). A type &#226;?&#164;-oxidation path-       replaced glyoxysomal derivative this <br /> way discovered filamentous Ascomycete, N.          metabolic pathway. Interestingly, recent evidence shown <br /> crassa. This pathway occurs glyoxysome (a microbody        M. grisea requires peroxisomal biogenesis machinery <br /> is distinct peroxisomes, contains enzymes        carry fatty acid &#226;?&#164;-oxidation; therefore, compartments <br />  <br />  <br />                                                                                                              Genome Research              1817 <br />                                                                                                                 www.genome.org <br />  Cornell et al. <br />  <br />  <br />  <br />  <br /> Figure 3. The occurrence peroxisomal/non-peroxisomal pathways beta-oxidation fatty acids fungal oomycete species. The step <br /> in peroxisomal beta-oxidation catalyzed acyl-CoA oxidase. The step non-peroxisomal beta-oxidation catalyzed acyl-CoA dehydroge- <br /> nase. The occurrence pathway fungal oomycete species defined based identification acyl-CoA oxidase acyl-CoA <br /> dehydrogenase encoding enzymes genome species. Acyl-CoA oxidases identified homology Pox1 S. cerevisiae and <br /> acyl-CoA dehydrogenases homology acyl-CoA dehydrogenase N. crassa (NCBI accession no. CAD70984.1). <br />  <br />  <br /> are highly conserved (Bhambra et al. 2006; Ramos-Pamplona               genomic data available large number of <br /> Naqvi 2006). Indeed, evidence N. crassa shows or-                 species. <br /> thologs peroxisomal biogenesis proteins necessary for-                 We chosen construct bioinformatics platform for <br /> mation glyoxysomes (Managadze et al. 2007).                              carrying comparative functional genomics Fungi for <br />                                                                             reasons. The Fungi group organisms great sci- <br />                                                                             entific, medical, agricultural, industrial importance. There is <br />                                                                             wealth genome sequence EST data Fungi, includ- <br /> Discussion <br />                                                                             ing important model organisms (e.g., S. cerevisiae, S. pombe, <br /> Comparative genomics powerful technique identifying                N. crassa), human (e.g., C. albicans, A. fumigatus), plant <br /> orthologous genes different species (Tatusov et al. 1997),           (e.g., M. grisea, U. maydis) pathogens, major process <br /> inferring presence metabolic pathways organisms                organisms industry (e.g., A. niger), play major <br /> which discovered previously (Giles et al.                ecological roles recycling plant materials (e.g., P. chryso- <br /> 2003), defining transcription-factor-binding sites              sporium). Moreover, techniques genetic manipulation of <br /> other regulatory motifs DNA sequences (Cliften et al.                fungi developed rapidly im- <br /> 2001; Kellis et al. 2003). The availability complete genome              proving efficiency scope. Finally, the <br /> sequences increasingly large number organisms gen-            Fungi possible, moment, perform comparative <br /> erated need techniques enable analysis gene               genomic analyses entire eukaryotic kingdom. The fungal <br /> action interaction comprehensive ge-             kingdom shows great diversity, comparisons both <br /> nome sequencing efforts preceded them. This led             short long evolutionary distances made. This holds <br /> development different levels functional genomic analy-            prospect results analyses, tools we <br /> sis&#226;&#8364;&#8221;the transcriptome, proteome, metabolome (Oliver                     develop perform them, general biological in- <br /> 2000), high-throughput approaches determining                terest utility eukaryotic genome sequences gen- <br /> phenotypes (Oliver 1996, 1997; Winzeler et al. 1999; Giaever et             erated. <br /> al. 2002). While range functional genomic technolo-                   The e-Fungi data warehouse object-oriented data ware- <br /> gies currently applied model organisms,                house based Genome Information Management System <br /> which fungus S. cerevisiae vanguard, situation        (GIMS) architecture (Cornell et al. 2003) developed pre- <br /> changing rapidly. For instance, DNA microarrays starting             viously enable integrative analyses genome sequence and <br /> be fabricated well-studied organisms (e.g., A. nidu-          functional genomic data Saccharomyces cerevisiae. It avail- <br /> lans; Sims et al. 2005). In situation, need carry        able online http://www.e-fungi.org.uk/database.html pro- <br /> out comparative genomic analyses exploit just ge-              vides &gt;90 detailed querying forms (so-called &#226;&#8364;&#339;canned&#226;&#8364;? queries) to <br /> nome sequences, burgeoning functional                enable users perform investigations. This multispe- <br />  <br />  <br /> 1818      Genome Research <br />             www.genome.org <br />                                                                                 Comparative genome analysis fungal kingdom <br />  <br />  <br /> cies warehouse work progress, currently (July 2007)        examples involved cell wall structure. <br /> populated sequence functional data 34 fungal spe-            This reflect different chemical composition of <br /> cies oomycetes.                                                   cell walls yeasts fungal hyphae. This point empha- <br />       The Fungi evolutionarily diverse group,         sized fact genomes filamentous Ascomy- <br /> also genomes span size range 18-fold             cetes Basidiomycetes expanded families chito- <br /> smallest (that microsporidian E. cuniculi) largest (the     sanases&#226;&#8364;&#8221;chitin principal polysaccharide hyphal cell <br /> Zygomycete, R. oryzae). Our whole-genome-based phylogenetic               walls, chitosan discrete stages development <br /> analysis demonstrates Oomycetes distant            fungi, minor components yeast cell <br /> relatives fungi, previously predicted SSU rDNA             walls. <br /> analysis. Oomycetes diverged recent common an-                    The representation different fungal species the <br /> cestor groups earlier split             various protein clusters defined instructive. The <br /> fungi plants. Moreover, interesting note that,           protein products essential genes S. cerevisiae (Giaever et al. <br /> despite mycology traditional province departments            2002), instance, belong clusters contain representa- <br /> of botany, animals closely related true fungi        tives species range fungal kingdom. <br /> plants.                                                                   Very essential genes peculiar Saccharomyces &#226;&#8364;&#339;sensu <br />       For this, influence overall size construction     stricto&#226;&#8364;? clade. However, examples proteins essen- <br /> of whole-genome phylogenies overlooked. Differ-             tial functions clusters members ex- <br /> ences total gene numbers complicate construction                clusive Saccharomyces &#226;&#8364;&#339;sensu stricto&#226;&#8364;? involved in <br /> whole-genome phylogenies. E. cuniculi placed             chromosome segregation nuclear fusion. This clade-specific <br /> the coarse-grained tree, based 12 protein families (Fig. 1A),          set proteins contribute reproductive iso- <br /> since sufficient representatives genome         lation Saccharomyces &#226;&#8364;&#339;sensu stricto&#226;&#8364;? mem- <br /> of 29 families used higher-resolution tree (Fig. 1B). Yet      bers genus (the Saccharomyces &#226;&#8364;&#339;sensu lato&#226;&#8364;?) whom <br /> other problems associated total gene numbers arise              mate. It interesting similar clade- <br /> the gene presence/absence criterion Dollo parsimony ap-            specific gene set revealed genome sequences inter- <br /> proach used. Here, difficulties resolution        breeding species Saccharomyces &#226;&#8364;&#339;sensu lato&#226;&#8364;? (Marinoni <br /> the Saccharomyces &#226;&#8364;&#339;sensu stricto&#226;&#8364;? clade, conse-            et al. 1999) available. <br /> quence major differences gene annotation                  The study highlighted metabolic diversity of <br /> different sequencing centers, despite similar sequence         fungi reflected gene inventories associated particular <br /> coverage. This emphasizes genomes chosen model            metabolic processes fatty acid &#226;?&#164;-oxidation, appears <br /> for construction microarrays sequence comparisons              absent process Taphrinomycotina S. pombe, and <br /> using CGH approach (Edwards-Ingram et al. 2004)               compartmentalized distinctly budding yeasts filamen- <br /> comprehensively sequenced, accurately uniformly an-               tous fungi. This demonstrates comparative genomic analysis <br /> notated.                                                                  reveal new fundamental characteristics well-studied <br />       The relative sizes genomes reflects, part, differ-   fungal species S. pombe. <br /> ences levels gene duplication species&#226;&#8364;&#8221;a con-                  In summary, e-Fungi data warehouse designed <br /> tention confirmed fact E. gossypii       provide e-science framework analysis diverse <br /> smallest genome lowest level gene duplication           functional genomic information currently generated for <br /> of Saccharomycotina. In general, Ascomycetes,               Fungi. Using transcriptional profiling, proteomic, and <br /> the filamentous Pezizomycotina greater levels gene du-            deletion-mutant data extensively compiled S. <br /> plication yeast-like Saccharomycotina. However,            cerevisiae reference set, aim build resource which <br /> appears levels duplication genome R. oryzae         newly acquired data filamentous fungi, including patho- <br /> far exceed fungal species study. As      genic species industrially relevant fungi, integrated. <br /> as differences amounts gene duplication appear         The ability carry extensive, multifaceted querying key <br /> be differences loss duplicates. Analysis sequence simi-      feature data warehouse, implemented using <br /> larities Pezizomycotina species suggests differences        e-science protocols harness data stored remotely distinct <br /> amount RIP activity, appear differences gene        databases, allowing interrogated single site. This <br /> loss following WGD genomes S. cerevisiae C.             study provided evidence utility a <br /> glabrata. We demonstrated evidence             resource investigating fundamental features fungi and <br /> coevolution, measured coduplication S. cerevisiae S.        exploring genetic genomic relatedness. <br /> pombe and, unexpectedly, S. cerevisiae R. oryzae. <br />       Protein clusters expanded Pezizomycotina, in <br /> comparison Saccharomycotina, include various families           Methods <br /> transport proteins, proteins involved utilization differ- <br /> ent carbon sources, transcription factors. All reflects       Orthology assignments <br /> the greater metabolic versatility broader substrate range          Predicted protein sequences 36 genomes downloaded <br /> the filamentous Ascomycetes compared yeasts. An excep-             respective sequencing project repositories (see Table 1). All- <br /> tion general rule dimorphic Saccharomycotina Y.            against-all BLAST searches (Altschul et al. 1990) performed <br /> lipolytica, shares number gene families Pe-           reveal similarities 348,995 predicted protein se- <br /> zizomycotina lost members                quences, identifying 47,342,483 similarities maximum <br /> Saccharomycotina.                                                         E-value 1 &#226;&#179;&#175; 10&#226;&#179;&#174;5. OrthoMCL clustering (Li et al. 2003) ap- <br />       Protein clusters expanded Saccharomycotina,         plied default parameters, generating 30,084 clusters, includ- <br /> as compared Pezizomycotina, (predictably) rare,            ing 5406 containing paralogs. <br />  <br />  <br />                                                                                                              Genome Research           1819 <br />                                                                                                                 www.genome.org <br />  Cornell et al. <br />  <br />  <br /> Formulating data sets fungal species tree                           database (http://pfam.sanger.ac.uk) allowed identify 48 <br />                                                                         domains (see Supplemental Table 3). <br /> Additional homology searches performed using CHASE <br /> (Alam et al. 2004) select universal (i.e., present species) <br /> protein clusters. Two data sets formulated: (1) To better    Identification over-represented GO terms <br /> rooting fungal phylogenetic tree, H. sapiens A. thaliana     The GO terms associated proteins clusters counted, <br /> sequences collected 12 universal fungal protein clusters       term counted cluster remove <br /> from HomoloGene orthologs databases National Cen-            bias clusters containing large numbers paralogs. <br /> ter Biotechnology Information (http://www.ncbi.nlm.nih.             For sets clusters, GO terms compared using chi- <br /> gov/HomoloGene) using S. cerevisiae intermediate. (2) To          squared analysis, (for sake brevity) sig- <br /> resolve deep branches tree, additional 17 protein clus-       nificant clusters discussed paper. <br /> ters selected lowest number duplicates. <br />  <br /> Phylogenetic analysis procedure generate fungal species              Acknowledgments <br /> trees                                                                   We thank Broad Institute DOE Joint Genome Insti- <br /> Multiple sequence alignments generated universal pro-          tute releasing data ahead publication. We acknowledge the <br /> tein families using ClustalW (Thompson et al. 1994). Protein            financial support Biotechnology Biological Sciences <br /> sequences concatenated, columns &gt;50% gaps            Research Council (BBSRC). The development e-Fungi data- <br /> excluded concatenated alignment. The alignments             <span id='am-160' about='xsp:base' typeof='owl:Thing'>base</span> funded BBSRC Bioinformatics e-Science <br /> concatenated sequences used phylogenetic tree con-             programme II. <br /> struction using PhyML (Guindon Gascuel 2003), a <br /> fast heuristic algorithm estimating large phylogenies maxi- <br /> mum likelihood. We used 100 bootstrap replicates assess con-         References <br /> fidence inferred relationships assign bootstrap support <br />                                                                         Alam, I., Fuellen, G., Rehmsmeier, M., Dress, A. 2004. Comparative <br /> levels clade maximum likelihood tree. We used              homology agreement search: An effective combination of <br /> JTT model amino acid evolution (Jones et al. 1992)                homology-search methods. Proc. Natl. Acad. Sci. 101: 13814&#226;&#8364;&#8220;13819. <br /> GTR+&#226;&#338;&#171;+I model evolution discrete-Gamma rate cat-           Altschul, S.F., Gish, W., Miller, W., Myers, E.W., Lipman, D.J. 1990. <br /> egories (Yang 1994). The resulting trees visualized using              Basic local alignment search tool. J. Mol. Biol. 215: 403&#226;&#8364;&#8220;410. <br />                                                                         Baldauf, S.L., Roger, A.J., Wenk-Siefert, I., Doolittle, W.F. 2000. A <br /> MEGA (Kumar et al. 2004). We initially used PhyML evaluate               kingdom-level phylogeny eukaryotes based combined protein <br /> bootstrap support, observed maximum like-              data. Science 290: 972&#226;&#8364;&#8220;977. <br /> lihood Bayesian methods prone local maxima prob-            Bhambra, G.K., Wang, Z.-Y., Soanes, D.M., Wakley, G.E., Talbot, <br /> lems. We selected trees obtained PhyML              N.J. 2006. Peroxisomal carnitine acetyl transferase required for <br />                                                                             elaboration penetration hyphae plant infection by <br /> all bootstrap samples exhaustively computed tree&#226;&#8364;&#8482;s like-           Magnaporthe grisea. Mol. Microbiol. 61: 46&#226;&#8364;&#8220;60. <br /> lihood bootstrap sample, order avoid problems            Bowman, S.M. Free, S.J. 2006. The structure synthesis the <br /> with PhyML finding maximum likelihood tree                  fungal cell wall. Bioessays 28: 799&#226;&#8364;&#8220;808. <br /> case.                                                                   Castillo, L., Martinez, A.I., Garcera, A., Elorza, M.V., Valentin, E., and <br />                                                                             Sentandreu, R. 2003. Functional analysis cysteine residues and <br />                                                                             repetitive sequence Saccharomyces cerevisiae Pir4/Cis3: The <br /> Dollo parsimony analyses                                                    repetitive sequence needed binding cell wall <br />                                                                             beta-1,3-glucan. Yeast 20: 973&#226;&#8364;&#8220;983. <br /> The presence absence 18,656 protein clusters containing <br />                                                                         Cliften, P.F., Hillier, L.W., Fulton, L., Graves, T., Miner, T., Gish, W.R., <br /> proteins Ascomycete, Basidiomycete, Zygomycete                Waterston, R.H., Johnston, M. 2001. Surveying Saccharomyces <br /> converted 1/0 matrix Dollo parsimony analysis. The               genomes identify functional elements comparative DNA <br /> seqboot program Phylip package (Felsenstein 1989)              sequence analysis. Genome Res. 11: 1175&#226;&#8364;&#8220;1186. <br />                                                                         Cliften, P., Sudarsanam, P., Desikan, A., Fulton, L., Fulton, B., Majors, J., <br /> used generate 100 Bootstrap data sets based phyletic pro- <br />                                                                             Waterston, R., Cohen, B.A., Johnston, M. 2003. Finding <br /> files 23,230 protein clusters. This data set subjected            functional features Saccharomyces genomes phylogenetic <br /> Phylip&#226;&#8364;&#8482;s dollop program. Phylip&#226;&#8364;&#8482;s consense program used              footprinting. Science 301: 71&#226;&#8364;&#8220;76. <br /> compute consensus tree.                                             Cornell, M., Paton, N.W., Hedeler, C., Kirby, P., Delneri, D., Hayes, A., <br />                                                                             Oliver, S.G. 2003. GIMS: An integrated data storage analysis <br />                                                                             environment genomic functional data. Yeast 20: 1291&#226;&#8364;&#8220;1306. <br /> Pfam domain assignments                                                 Boer, W., Folman, W.B., Summerbell, R.C., Boddy, L. 2005. <br /> Protein sequences scanned using Pfam database release 18               Living fungal world: The impact fungi soil bacterial niche <br />                                                                             development. FEMS Microbiol. Rev. 29: 795&#226;&#8364;&#8220;811. <br /> (Finn et al. 2006) hmmpfam (Eddy 1998) E-value              Deng, J., Carbone, I., Dean, R.A. 2007. The evolutionary history of <br /> cutoff 1 &#226;&#179;&#175; 10&#226;&#179;&#174;1.                                                         cytochrome P450 genes filamentous Ascomycetes. BMC Evol. <br />                                                                             Biol. 7: 30. doi: 10.1186/1471-2148-7-30. <br /> Identification Pfam domains expanded                                 Dujon, B., Sherman, D., Fischer, G., Durrens, P., Casaregola, S., <br />                                                                             Lafontaine, I., De Montigny, J., Marck, C., Neuveglise, C., Talla, E., <br /> in Pezizomycotina Saccharomycotina                                  et al. 2004. Genome evolution yeasts. Nature 430: 35&#226;&#8364;&#8220;44. <br /> Pfam domains identified using chi-squared analysis, assum-         Eddy, S.R. 1998. Profile hidden Markov models. Bioinformatics <br />                                                                             14: 755&#226;&#8364;&#8220;763. <br /> ing domains equally distributed Saccha-           Edwards-Ingram, L.C., Gent, M.E., Hoyle, D.C., Hayes, A., Stateva, L.I., <br /> romycotina Pezizomycotina. For domains significant &#226;?&#185;2              Oliver, S.G. 2004. Comparative genomic hybridization provides <br /> scores, checked ensure expansion limited             new insights molecular taxonomy Saccharomyces <br /> to single species.                                                        &#226;&#8364;&#339;sensu stricto&#226;&#8364;? complex. Genome Res. 14: 1043&#226;&#8364;&#8220;1051. <br />                                                                         Elliott, S., Knop, M., Schlenstedt, G., Schiebel, E. 1999. Spc29p a <br />                                                                             component Spc110p subcomplex essential spindle <br /> Identification transposon sequences fungal species                    pole body duplication. Proc. Natl. Acad. Sci. 96: 6205&#226;&#8364;&#8220;6210. <br />                                                                         Farris, J.S. 1977. Phylogenetic analysis Dollo&#226;&#8364;&#8482;s Law. Syst. Zool. <br /> Because genomes analyzed study not <br />                                                                             26: 77&#226;&#8364;&#8220;88. <br /> been fully annotated, identified transposon sequences         Felsenstein, J. 1978. Cases parsimony compatibility <br /> presence characteristic Pfam motifs. A search Pfam          methods positively misleading. Syst. Zool. 27: 401&#226;&#8364;&#8220;410. <br />  <br />  <br />  <br /> 1820     Genome Research <br />             www.genome.org <br />                                                                                          Comparative genome analysis fungal kingdom <br />  <br />  <br /> Felsenstein, J., 1989. PHYLIP&#226;&#8364;&#8221;Phylogeny inference package (version 3.2).          Kellis, M., Birren, B.W., Lander, E.S. 2004. Proof evolutionary <br />     Cladistics 5: 164&#226;&#8364;&#8220;166.                                                            analysis ancient genome duplication yeast Saccharomyces <br /> Finn, R.D., Mistry, J., Schuster-Bockler, B., Griffiths-Jones, S., Hollich, V.,       cerevisiae. Nature 428: 617&#226;&#8364;&#8220;624. <br />     Lassmann, T., Moxon, S., Marshall, M., Khanna, A., Durbin, R., et al.         Kumar, S., Tamura, K., Nei, M. 2004. MEGA3: Integrated software <br />     2006. Pfam: Clans, web tools services. Nucleic Acids Res.                     molecular evolutionary genetics analysis sequence <br />     34: D247&#226;&#8364;&#8220;D251. doi: 10.1093/nar/gkj149.                                           alignment. Brief. Bioinform. 5: 150&#226;&#8364;&#8220;163. <br /> Fitz-Gibbon, S.T. House, C.H. 1999. Whole genome-based                        Kunau, W.H., Dommes, V., Schulz, H. 1995. Beta-oxidation fatty <br />     phylogenetic analysis free-living microorganisms. Nucleic Acids                acids mitochondria, peroxisomes, bacteria: A century of <br />     Res. 27: 4218&#226;&#8364;&#8220;4222.                                                               continued progress. Prog. Lipid Res. 34: 267&#226;&#8364;&#8220;342. <br /> Fitzpatrick, D.A., Logue, M.E., Stajich, J.E., Butler, G. 2006. A fungal      Li, L., Stoeckert, C.J., Roos, D.S. 2003. OrthoMCL: Identification of <br />     phylogeny based 42 complete genomes derived supertree                     ortholog groups eukaryotic genomes. Genome Res. <br />     combined gene analysis. BMC Evol. Biol. 6: 99. doi:                           13: 2178&#226;&#8364;&#8220;2189. <br />     10.1186/1471-2148-6-99.                                                       Lynch, M. Force, A. 2000. The probability duplicate gene <br /> Galagan, J.E. Selker, E.U. 2004. RIP: The evolutionary cost                    preservation subfunctionalization. Genetics 154: 459&#226;&#8364;&#8220;473. <br />     genome defense. Trends Genet. 20: 417&#226;&#8364;&#8220;423.                                    Maggio-Hall, L.A. Keller, N.P. 2004. Mitochondrial beta-oxidation in <br /> Galagan, J.E., Henn, M.R., Ma, L.J., Cuomo, C.A., Birren, B. 2006.                Aspergillus nidulans. Mol. Microbiol. 54: 1173&#226;&#8364;&#8220;1185. <br />     Genomics fungal kingdom: Insights eukaryotic biology.             Maiden, M.C., Davis, E.O., Baldwin, S.A., Moore, D.C., Henderson, <br />     Genome Res. 15: 1620&#226;&#8364;&#8220;1631.                                                        P.J. 1987. Mammalian bacterial sugar transport proteins are <br /> Giaever, G., Chu, A.M., Ni, L., Connelly, C., Riles, L., Veronneau, S.,               homologous. Nature 325: 641&#226;&#8364;&#8220;643. <br />     Dow, S., Lucau-Danila, A., Anderson, K., Andre, B., et al. 2002.              Managadze, D., W&#195;&#188;rtz, C., Sichting, M., Niehaus, G., Veenhuis, M., and <br />     Functional profiling Saccharomyces cerevisiae genome. Nature               Rottensteiner, H. 2007. The peroxin PEX14 Neurospora crassa is <br />     418: 387&#226;&#8364;&#8220;391.                                                                     essential biogenesis glyoxysomes Woronin <br /> Giles, P.F., Soanes, D.M., Talbot, N.J. 2003. A relational database           bodies. Traffic 8: 687&#226;&#8364;&#8220;701. <br />     discovery genes encoding amino acid biosynthetic enzymes            Marinoni, G., Manuel, M., Petersen, R.F., Hvidtfeldt, J., Sulo, P., and <br />     pathogenic fungi. Comp. Funct. Genomics 4: 4&#226;&#8364;&#8220;15.                                  Piskur, J. 1999. Horizontal transfer genetic material among <br /> Goffeau, A. 2004. Evolutionary genomics: Seeing double. Nature                        Saccharomyces yeasts. J. Bacteriol. 181: 6488&#226;&#8364;&#8220;6496. <br />     430: 25&#226;&#8364;&#8220;26.                                                                   Money, N.P., Davis, C.M., Ravishankar, J.P. 2004. Biomechanical <br /> Goto, S., Okuno, Y., Hattori, M., Nishioka, T., Kanehisa, M. 2002.                evidence convergent evolution invasive growth process <br />     LIGAND: Database chemical compounds reactions                           fungi oomycete water molds. Fungal Genet. Biol. <br />     biological pathways. Nucleic Acids Res. 30: 402&#226;&#8364;&#8220;404.                              41: 872&#226;&#8364;&#8220;876. <br /> Grandin, N., Damon, C., Charbonneau, M. 2001. Ten1 functions               Montiel, M.D., Lee, H.A., Archer, D.B. 2006. Evidence RIP <br />     telomere end protection length regulation association                 (repeat-induced point <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span>mutation</span>) transposase sequences of <br />     Stn1 Cdc13. EMBO J. 20: 1173&#226;&#8364;&#8220;1183.                                            Aspergillus oryzae. Fungal Genet. Biol. 43: 439&#226;&#8364;&#8220;445. <br /> Griffith, J.K., Baker, M.E., Rouch, D.A., Page, M.G.P., Skurray, R.A.,            Narasimhan, M.L., Lee, H., Damsz, B., Singh, N.K., Ibeas, J.I., <br />     Paulsen, I.T., Chater, K.F., Baldwin, S.A., Henderson, P.J.F. 1992.           Matsumoto, T.K., Woloshuk, C.P., Bressan, R.A. 2003. <br />     Membrane transport proteins: Implications sequence                             Overexpression cell wall glycoprotein Fusarium oxysporum <br />     comparisons. Curr. Opin. Cell Biol. 4: 684&#226;&#8364;&#8220;695.                                   increases virulence resistance plant PR-5 protein. Plant J. <br /> Guindon, S. Gascuel, O. 2003. A simple, fast, accurate                        36: 390&#226;&#8364;&#8220;400. <br />     algorithm estimate large phylogenies maximum likelihood.                Naumov, G.I., Naumova, E.S., Sniegowski, P.D. 1997. <br />     Syst. Biol. 52: 696&#226;&#8364;&#8220;704.                                                          Differentiation European Far East Asian populations of <br /> Hedges, S.B., Blair, J.E., Venturi, M.L., Shoe, J.L. 2004. A molecular            Saccharomyces paradoxus allozyme analysis. Int. J. Syst. Bacteriol. <br />     timescale eukaryote evolution rise complex                          47: 341&#226;&#8364;&#8220;344. <br />     multicellular life. BMC Evol. Biol. 4: 2. doi: 10.1186/1471-2148-4-2.         Naumov, G.I., Naumova, E.S., Masneuf, I., Aigle, M., Kondratieva, V.I., <br /> Hiltunen, J.K., Mursula, A.M., Rottensteiner, H., Wierenga, R.K.,                     Dubourdieu, D. 2000. Natural polyploidization cultured <br />     Kastaniotis, A.J., Gurvitz, A. 2003. The biochemistry                      yeast Saccharomyces sensu stricto: Auto- allotetraploidy. Syst. Appl. <br />     peroxisomal beta-oxidation yeast Saccharomyces cerevisiae.                 Microbiol. 23: 442&#226;&#8364;&#8220;449. <br />     FEMS Microbiol. Rev. 27: 35&#226;&#8364;&#8220;64.                                               Oliver, S.G. 1996. From DNA sequence biological function. Nature <br /> Hughes, A.L. Friedman, R. 2003. Parallel evolution gene                        379: 597&#226;&#8364;&#8220;600. <br />     duplication genomes unicellular fungi. Genome Res.              Oliver, S.G. 1997. From gene screen yeast. Curr. Opin. Genet. Dev. <br />     13: 1259&#226;&#8364;&#8220;1264.                                                                    7: 405&#226;&#8364;&#8220;409. <br /> Huson, D.H. Steel, M. 2004. Phylogenetic trees based gene                  Oliver, S. 2000. Guilt-by-association goes global. Nature 403: 601&#226;&#8364;&#8220;603. <br />     content. Bioinformatics 20: 2044&#226;&#8364;&#8220;2049.                                        Pereira, G., Grueneberg, U., Knop, M., Schiebel, E. 1999. Interaction <br /> Ikeda, K., Nakayashiki, H., Kataoka, T., Tamba, H., Hashimoto, Y., Tosa,              yeast gamma-tubulin complex-binding protein Spc72p with <br />     Y., Mayama, S. 2002. Repeat-induced point mutation (RIP)                   Kar1p essential microtubule function karyogamy. EMBO <br />     Magnaporthe grisea: implications sexual cycle natural              J. 18: 4180&#226;&#8364;&#8220;4195. <br />     field context. Mol. Microbiol. 45: 1355&#226;&#8364;&#8220;1364.                                 Ramos-Pamplona, M. Naqvi, N.I. 2006. Host invasion during <br /> James, T.Y., Kaulf, F., Schoch, C.L., Matheny, P.B., Hofstetter, V., Cox,             rice-blast disease requires carnitine-dependent transport of <br />     C.J., Celio, G., Guedin, C., Fraker, E., Miadlikowska, J. 2006.               peroxisomal acetyl-CoA. Mol. Microbiol. 61: 61&#226;&#8364;&#8220;75. <br />     Reconstructing early evolution Fungi using six-gene                  Schliebs, W., W&#195;&#188;rtz, C., Kunau, W.H., Veenhuis, M., Rottensteiner, <br />     phylogeny. Nature 443: 818&#226;&#8364;&#8220;822.                                                   H. 2006. A eukaryote catalase-containing microbodies: <br /> Jones, D.T., Taylor, W.R., Thornton, J.M. 1992. The rapid                         Neurospora crassa exhibits unique cellular distribution four <br />     generation mutation data matrices protein sequences.                      catalases. Eukaryot. Cell 5: 1490&#226;&#8364;&#8220;1502. <br />     Comput. Appl. Biosci. 8: 275&#226;&#8364;&#8220;282.                                             Sims, A.H., Gent, M.E., Lanthaler, K., Dunn-Coleman, N.S., Oliver, S.G., <br /> Jones, T., Federspiel, N.A., Chibana, H., Dungan, J., Kalman, S., Magee,              Robson, G.D. 2005. Transcriptome analysis recombinant <br />     B.B., Newport, G., Thorstenson, Y.R., Agabian, N., Magee, P.T., et al.            protein secretion Aspergillus nidulans unfolded protein <br />     2004. The diploid genome sequence Candida albicans. Proc. Natl.                response vivo. Appl. Environ. Microbiol. 71: 2737&#226;&#8364;&#8220;2747. <br />     Acad. Sci. 101: 7329&#226;&#8364;&#8220;7334.                                                    Tatusov, R.L., Koonin, E.V., Lipman, D.J. 1997. A genomic <br /> Jungwirth, H. Kuchler, K. 2006. Yeast ABC transporters&#226;&#8364;&#8221;A tale                  perspective protein families. Science 278: 631&#226;&#8364;&#8220;637. <br />     sex, stress, drugs aging. FEBS Lett. 580: 1131&#226;&#8364;&#8220;1138.                      Thieringer, R. Kunau, W.H. 1991. The beta-oxidation in <br /> Kapteyn, J.C., Van Egmond, P., Sievi, E., Van Den Ende, H., Makarow,                  catalase-free microbodies filamentous fungus Neurospora <br />     M., Klis, F.M. 1999. The contribution O-glycosylated                   crassa. Purification multifunctional protein possessing <br />     protein Pir2p/Hsp150 construction yeast cell wall                2-enoyl-CoA hydratase, L-3-hydroxyacyl-CoA dehydrogenase, and <br />     wild-type cells beta 1,6-glucan-deficient mutants. Mol. Microbiol.            3-hydroxyacyl-CoA epimerase activities. J. Biol. Chem. <br />     31: 1835&#226;&#8364;&#8220;1844.                                                                    266: 13110&#226;&#8364;&#8220;13117. <br /> Katinka, M.D., Duprat, S., Cornillot, E., Metenier, G., Thomarat, F.,             Thomarat, F., Vivares, C.P., Gouy, M. 2004. Phylogenetic analysis of <br />     Prensier, G., Barbe, V., Peyretaillade, E., Brottier, P., Wincker, P., et         complete genome sequence Encephalitozoon cuniculi supports <br />     al. 2001. Genome sequence gene compaction eukaryote                    fungal origin microsporidia reveals high frequency of <br />     parasite Encephalitozoon cuniculi. Nature 414: 450&#226;&#8364;&#8220;453.                           fast-evolving genes. J. Mol. Evol. 59: 780&#226;&#8364;&#8220;791. <br /> Kellis, M., Patterson, N., Endrizzi, M., Birren, B., Lander, E.S. 2003.       Thompson, J.D., Higgins, D.G., Gibson, T.J. 1994. CLUSTAL W: <br />     Sequencing comparison yeast species identify genes                  Improving sensitivity progressive multiple sequence <br />     regulatory elements. Nature 423: 241&#226;&#8364;&#8220;254.                                         alignment sequence weighting, position-specific gap <br />  <br />  <br />  <br />                    <br /> </body></html>