<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                            NIH Public Access <br />                             Author Manuscript <br />                             Science. Author manuscript; available PMC 2009 September 18. <br />                            Published final edited form as: <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             Science. 2008 October 3; 322(5898): 104&#226;&#8364;&#8220;110. doi:10.1126/science.1158684. <br />  <br />  <br />  <br />                            High Quality Binary Protein Interaction Map Yeast <br />                            Interactome Network <br />  <br />                            Haiyuan Yu1,2,*, Pascal Braun1,2,*, Muhammed A. Yildirim1,2,3,*, Irma Lemmens4, Kavitha <br />                            Venkatesan1,2, Julie Sahalie1,2, Tomoko Hirozane-Kishikawa1,2, Fana Gebreab1,2, Na <br />                            Li1,2, Nicolas Simonis1,2, Tong Hao1,2, Jean-Fran&#196;&#8225;ois Rual1,2, Am&#195;&#169;lie Dricot1,2, Alexei <br />                            Vazquez5, Ryan R. Murray1,2, Christophe Simon1,2, Leah Tardivo1,2, Stanley Tam1,2, Nenad <br />                            Svrzikapa1,2, Changyu Fan1,2, Anne-Sophie Smet4, Adriana Motyl6, Michael E. <br />                            Hudson6, Juyong Park1,7, Xiaofeng Xin8, Michael E. Cusick1,2, Troy Moore9, Charlie <br />                            Boone8, Michael Snyder6, Frederick P. Roth1,10, Albert-L&#195;&#161;szl&#195;&#179; Barab&#195;&#161;si1,7, Jan <br />                            Tavernier4, David E. Hill1,2, Marc Vidal1,2,+ <br />                            1Center Cancer Systems Biology (CCSB), Dana-Farber Cancer Institute, Boston, MA 02115, <br />  <br />                            USA <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            2Department Cancer Biology, Dana-Farber Cancer Institute Department Genetics, Harvard <br />  <br />                            Medical School, Boston, MA 02115, USA <br />                            3School   Engineering Applied Sciences, Harvard University, Cambridge, MA 02138, USA <br />                            4Department Medical Protein Research, VIB, Department Biochemistry, Faculty of <br />                            Medicine Health Sciences, Ghent University, 9000 Ghent, Belgium <br />                            5The Simons Center Systems Biology, Institute Advanced Studies, Princeton, NJ 08540, USA <br />  <br />                            6Department     Molecular, Cellular, Developmental Biology, Yale University, New Haven, CT <br />                            06620, USA <br />                            7Centerfor Complex Network Research Departments Physics, Biology Computer <br />                            Science, Northeastern University, Boston, MA 02115, USA <br />                            8Banting Best Department Medical Research Department Medical Genetics and <br />                            Microbiology, Terrence Donnelly Centre Cellular Biomolecular Research, University of <br />                            Toronto, Toronto, Canada M5S 3E1 <br />                            9Open <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                    Biosystems, Huntsville, AL 35806, USA <br />                            10Departmentof Biological Chemistry Molecular Pharmacology, Harvard Medical School, <br />                            Boston, MA 02115, USA <br />  <br />                            <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />                                 Current yeast interactome network maps contain molecular complexes limited <br />                                 somewhat controversial representation direct binary interactions. We carried comparative <br />                                 quality assessment current yeast interactome datasets, demonstrating high-throughput yeast <br />                                 two-hybrid (Y2H) provides high-quality binary interaction information. As large fraction the <br />                                 yeast binary interactome remains mapped, developed empirically-controlled mapping <br />                                 framework produce &#226;&#8364;&#339;second-generation&#226;&#8364;? high-quality high-throughput Y2H dataset covering <br />                                 ~20% yeast binary interactions. Both Y2H affinity-purification followed mass <br />  <br />  <br />                            +<span id='am-3' about='protege:TO' typeof='owl:Thing'>To</span> correspondence addressed. Email: marc_vidal@dfci.harvard.edu. <br />                            *These authors contributed equally work. <br />                            Supporting Online Material SOM Text Figs. S1-35 Tables S1-5 References <br />                             Yu et al.                                                                                                          Page 2 <br />  <br />  <br />                                        spectrometry (AP/MS) data equally high quality fundamentally different and <br />                                        complementary nature resulting networks different topological biological properties. <br />                                        Compared co-complex interactome models, binary map enriched transient signaling <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        interactions inter-complex connections highly significant clustering essential <br />                                        proteins. Rather correlating essentiality, protein connectivity correlates genetic <br />                                        pleiotropy. <br />  <br />                                                  A crucial step understanding cellular systems properties mapping networks of <br />                                                  physical DNA-, RNA- protein-protein interactions, &#226;&#8364;&#339;interactome network&#226;&#8364;?, an <br />                                                  organism completely accurately possible. One approach consists in <br />                                                  systematically testing pairwise combinations predicted proteins derive &#226;&#8364;&#339;binary&#226;&#8364;? <br />                                                  interactome. Early attempts binary interactome mapping used high-throughput yeast two- <br />                                                  hybrid (Y2H), protein interaction reconstitutes transcription factor activates <br />                                                  expression reporter genes. High-throughput Y2H maps generated for <br />                                                  Saccharomyces cerevisiae (1-3), Caenorhabditis elegans (4-6), Drosophila melanogaster (7), <br />                                                  human (8-10). An alternative approach consists generating &#226;&#8364;&#339;co-complex&#226;&#8364;? interactome <br />                                                  maps, achievable high-throughput co-affinity purification followed mass spectrometry <br />                                                  (AP/MS) identify proteins bound tagged baits, Escherichia coli (11,12), S. <br />                                                  cerevisiae (13-16), human (17). <br />  <br />                                                  To investigate fundamental questions interactome network structure function, is <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  necessary understand size quality currently available maps, including <br />                                                  thorough evaluation differences binary co-complex maps, affected <br />                                                  conclusions global local properties interactome networks (18,19). Here, we <br />                                                  address issues using yeast S. cerevisiae model system. <br />  <br />                                                  First, compared quality existing high-throughput binary co-complex datasets to <br />                                                  information obtained curating low-throughput experiments described literature <br />                                                  (Fig. 1A). For binary interactions examined: (i) subset Uetz et al. proteome- <br />                                                  scale all-by-all screen (&#226;&#8364;&#339;Uetz-screen&#226;&#8364;?), excluding pairs focused, potentially <br />                                                  biased experiment involving 193 baits (&#226;&#8364;&#339;Uetz-array&#226;&#8364;?) (2); (ii) Ito et al. interactions <br />                                                  times (&#226;&#8364;&#339;Ito-core&#226;&#8364;?), independently times (&#226;&#8364;&#339;Ito- <br />                                                  noncore&#226;&#8364;?), distinction recommended authors seldom applied literature (3). <br />                                                  For co-complex associations, investigated high-throughput AP/MS datasets referred <br />                                                  &#226;&#8364;&#339;Gavin&#226;&#8364;? (15) &#226;&#8364;&#339;Krogan&#226;&#8364;? (16). For literature-curated interactions, considered <br />                                                  curated publications (&#226;&#8364;&#339;LC-multiple&#226;&#8364;?) (20), considered of <br />                                                  higher quality curated single publication. <br />  <br />                                                  To experimentally compare quality datasets, selected representative sample <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  ~200 protein interaction pairs tested means independent <br />                                                  interaction assays, Y2H yellow fluorescent protein complementation assay (PCA) (21) <br />                                                  [Supporting Online Material (SOM) I]. In PCA, bait prey proteins fused non- <br />                                                  fluorescent fragments yellow fluorescent protein that, brought close proximity by <br />                                                  interacting proteins, reconstitute fluorescent protein mammalian cells. In contrast, <br />                                                  reconstitution transcription factor Y2H experiments takes place nucleus yeast <br />                                                  cells. In terms assay designs, Y2H PCA considered orthogonal assays can <br />                                                  used validate other&apos;s results. <br />  <br />                                                  No single assay expected detect 100% genuine interactions, actual fraction of <br />                                                  positives detected inherently linked stringency assay implemented. To <br />                                                  identify optimal scoring condition assay selected set ~100 well-documented <br />                                                  yeast protein-protein interaction pairs [&#226;&#8364;&#339;positive reference set&#226;&#8364;? (PRS)] set ~100 random <br />                                                  pairs [&#226;&#8364;&#339;random reference set&#226;&#8364;? (RRS)] (Fig. 1B; SOM II). Because RRS pairs picked <br />  <br />  <br />                                                       Science. Author manuscript; available PMC 2009 September 18. <br />                             Yu et al.                                                                                                Page 3 <br />  <br />  <br />                                        uniformly 14&#195;&#8212;106 possible pairings proteins yeast ORFeome collection <br />                                        (22) (excluding reported interacting), pairs extremely unlikely be <br />                                        interacting. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        Sampled pairs binary Uetz-screen Ito-core datasets tested positive levels high <br />                                        positive control PRS, demonstrating high quality (Fig. 1C). A sample literature- <br />                                        curated LC-multiple interactions tested slightly lower Y2H, indistinguishable <br />                                        PCA (Fig. 1C), demonstrating high-throughput Y2H datasets comparable in <br />                                        quality literature-curated information. In striking contrast, sampled pairs Ito-noncore <br />                                        tested levels similar negative control RRS, confirming low quality particular <br />                                        dataset (Fig. 1C). <br />  <br />                                        Sampled pairs Gavin Krogan high-throughput AP/MS datasets tested poorly our <br />                                        binary interaction assays (Fig. 1C), albeit levels similar Munich Information Center <br />                                        Protein Sequences (MIPS) complexes, widely-used gold standard (23). This observation <br />                                        demonstrates that, detecting binary interactions, Y2H performs better AP/MS <br />  <br />                                        Our experimental data quality assessment shows binary Uetz-screen, Ito-core, LC- <br />                                        multiple datasets high quality, Ito-noncore used. AP/MS datasets, <br />                                        intrinsically good quality (15,16), used caution binary <br />                                        interaction information needed. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        Our experimental results contrast strikingly computational analyses suggested that <br />                                        high-throughput Y2H datasets contain false positives literature-curated high- <br />                                        throughput AP/MS datasets (24,25). In computational analyses, quality dataset often <br />                                        determined fraction interactions present pre-defined gold standard set (24). <br />                                        Generally, MIPS complexes considered gold standard proteins <br />                                        constituting given complex modeled interacting other. Such modeling results in <br />                                        limited biased sampling issues binary interactions proteins a <br />                                        complex contact directly (fig. S1), direct physical interactions occur <br />                                        complexes (fig. S2; SOM III). Hence, MIPS complexes appropriate for <br />                                        benchmarking co-complex membership datasets, binary interaction datasets. <br />                                        This distinction corroborated poor experimental confirmation rate pairs MIPS <br />                                        complexes using binary assays (Fig. 1C). <br />  <br />                                        To computationally re-examine quality existing yeast interactome datasets assembled <br />                                        binary gold standard set (&#226;&#8364;&#339;Binary-GS&#226;&#8364;?) 1,318 high-confidence physical binary interactions <br />                                        (Fig. 1B; SOM III). Binary-GS includes direct physical interactions well-established <br />                                        complexes conditional interactions (e.g., dependent posttranslational <br />                                        modifications) represents well-documented direct physical interactions yeast <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        interactome (26). When measured Binary-GS, quality high-throughput Y2H <br />                                        datasets (with exception Ito-noncore) substantially better (SOM IV V) that <br />                                        high-throughput AP/MS datasets (Fig. 1D). Our results demonstrate distinct nature of <br />                                        binary co-complex data. Generally, Y2H datasets contain high quality direct binary <br />                                        interactions, AP/MS co-complex datasets composed direct interactions mixed <br />                                        preponderant indirect associations (SOM VI). <br />  <br />                                        The proteome-wide binary datasets, Uetz-screen Ito-core, contain 682 843 interactions, <br />                                        respectively (2,3). The overlap datasets appears low (3,24): 19% Uetz- <br />                                        screen 15% Ito-core interactions detected dataset. Given our <br />                                        demonstration high quality datasets (Fig. 1C, D), conclude small overlap <br />                                        stems primarily low sensitivity (i.e., false negatives) low specificity <br />                                        (i.e., false positives previously suggested). <br />  <br />  <br />                                             Science. Author manuscript; available PMC 2009 September 18. <br />                             Yu et al.                                                                                               Page 4 <br />  <br />  <br />                                        Several factors affect sensitivity. First, space pair-wise protein combinations <br />                                        actually tested dataset considerably different. We refer fraction <br />                                        possible pairs tested given screen &#226;&#8364;&#339;completeness&#226;&#8364;?. For example, missing 10% <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        ORFs mapping project reduce common tested space 66% <br />                                        [(0.9&#195;&#8212;0.9) &#195;&#8212; (0.9&#195;&#8212;0.9)] possible pair-wise combinations. Second, different protein <br />                                        interaction assays different versions assay detect different subsets pairs <br />                                        possible interactions, explaining partly limited overlap datasets obtained <br />                                        different Y2H versions. For assay, &#226;&#8364;&#339;assay-sensitivity&#226;&#8364;? estimated fraction <br />                                        PRS interactions detected, Y2H assay determined empirically ~20% <br />                                        (Fig. 1C). Finally, screening tens hundreds millions protein pairs tested <br />                                        space, search space need sampled multiple times report nearly all <br />                                        interactions detectable assay used. The fraction theoretically detectable <br />                                        interactions particular assay given experiment &#226;&#8364;&#339;sampling-sensitivity&#226;&#8364;?. <br />                                        These parameters fully account seemingly small overlap Ito-core and <br />                                        Uetz-screen (SOM VII), demonstrating large fraction S. cerevisiae binary <br />                                        interactome remains mapped. Therefore, carried new proteome-scale yeast high- <br />                                        throughput Y2H screen (fig. S3). <br />  <br />                                        We used 5,796 Gateway-cloned ORFs available yeast MORF collection (22). After <br />                                        subcloning ORFs Y2H vectors removing auto-activators (27,28), search <br />                                        space 3,917 DB-Xs 5,246 AD-Ys, representing completeness 77% (Fig. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        2A; SOM VI), comparable recent AP/MS datasets (15) (~78%; SOM VI). <br />  <br />                                        To address sampling-sensitivity, determined fraction detectable interactions is <br />                                        pass trials search space 658 DB-X 1,249 AD-Y ORFs. A <br />                                        single trial identified 60% possible interactions detected high- <br />                                        throughput Y2H, repeats required obtain 80-90% (Fig. 2B; SOM <br />                                        VI). We decided screen search space times independently yield an <br />                                        estimated sampling-sensitivity 85% (Fig. 2B). In total ~88,000 colonies picked, of <br />                                        21,432 scored positive detailed phenotyping (SOM I). After identifying all <br />                                        putative interaction pairs sequencing, phenotypically retesting using fresh cultures <br />                                        archival stocks, eliminating novo auto-activators (28), obtained final dataset, <br />                                        &#226;&#8364;&#339;CCSBYI1&#226;&#8364;?, 1,809 interactions 1,278 proteins. <br />  <br />                                        To validate overall quality CCSB-YI1, tested 94 randomly-chosen interactions by <br />                                        PCA mammalian protein-protein interaction trap (MAPPIT; SOM I) (21,29). MAPPIT <br />                                        takes place mammalian cell membrane measures interactions activation of <br />                                        STAT3-dependent reporter expression. Using PCA MAPPIT confirmation rate <br />                                        CCSB-YI1 similar Ito-core Uetz-screen (Fig. 1C). The precision [i.e., <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        fraction true positives dataset (30)] CCSB-YI1 estimated 94-100% (Fig. 2C; <br />                                        fig. S4; SOM VI). Additionally, performance high-throughput Y2H approach was <br />                                        confirmed larger RRS 1,000 random pairs (30) (Fig. 1B), tested positive <br />                                        (SOM II). <br />  <br />                                        The overlaps Uetz-screen (27%) Ito-core (35%) CCSB-YI1 (Fig. 2D) be <br />                                        explained completeness, assay- sampling-sensitivity experiments (SOM <br />                                        VII) agree results pairwise confirmation datasets (Fig. <br />                                        1C). Similar principles apply large-scale experiments AP/MS, likely <br />                                        accounting low overlap Krogan Gavin (~25%; fig. S5B). <br />  <br />                                        Factoring completeness, precision, assay-, sampling-sensitivity, estimated the <br />                                        yeast binary interactome consists ~18,000&#194;&#177;4,500 interactions (SOM VI), experimentally <br />                                        validating previous computational estimates 17,000 25,000 interactions (31,32). To obtain <br />  <br />  <br />                                             Science. Author manuscript; available PMC 2009 September 18. <br />                             Yu et al.                                                                                                Page 5 <br />  <br />  <br />                                        comprehensive map binary yeast interactome combined available <br />                                        high-quality proteome-scale Y2H datasets (SOM VII). The union Uetz-screen, Ito-core, and <br />                                        CCSB-YI1, &#226;&#8364;&#339;Y2H-union&#226;&#8364;?, contains 2,930 binary interactions 2,018 proteins, which, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        according empirical estimate interactome size, represents ~20% yeast <br />                                        binary interactome (Fig 3A). <br />  <br />                                        We re-examined global topological features new yeast interactome network, facing <br />                                        lower risk over-interpreting properties limited sampling various biases data <br />                                        (18). To contrast topological properties binary Y2H-union network co- <br />                                        complex network, used integrated AP/MS dataset (33), generated by <br />                                        combining raw high-throughput AP/MS data (15,16). This &#226;&#8364;&#339;Combined-AP/MS&#226;&#8364;? dataset, <br />                                        composed 9,070 co-complex membership associations 1,622 proteins, attempts to <br />                                        model binary interactions co-complex data (Fig. 3A). <br />  <br />                                        As previously macromolecular networks, connectivity &#226;&#8364;&#339;degree&#226;&#8364;? <br />                                        distribution datasets best approximated power-law (34) (fig. S6; SOM VIII). <br />                                        Highly connected proteins, &#226;&#8364;&#339;hubs&#226;&#8364;?, reportedly likely encoded essential genes <br />                                        connected proteins (35). Surprisingly, Y2H-union lacked correlation between <br />                                        degree essentiality (Fig. 3B). This discrepancy stem biases datasets <br />                                        available time original observation: interactions reported Uetz et al. (Uetz-array <br />                                        Uetz-screen) literature-curated interactions. Although Uetz-array high quality <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        (fig. S7), experimental design negatively influence network analyses. Most hub <br />                                        proteins Uetz-array baits (fig. S8) percentage essential proteins in <br />                                        193 bait proteins times higher (34.7%) protein-encoding ORFs the <br />                                        yeast genome (18.4%), explaining high correlation degree essentiality (Fig. <br />                                        3C). Likewise, literature-curated interactions prone sociological inspection <br />                                        biases (SOM VII). Thus, refrain heretofore using LC-multiple further <br />                                        topological biological analyses. No significant correlation degree of <br />                                        connectedness essentiality observed proteome-wide high- <br />                                        throughput binary datasets available today (i.e., Ito-core, Uetz-screen, CCSB-YI1; Fig. <br />                                        3C), new versions C. elegans human interactome maps (fig. S9; SOM <br />                                        IX). <br />  <br />                                        Hub proteins instead relate pleiotropy, number phenotypes observed consequence <br />                                        gene <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span>knock-out</span> (SOM I). There significant correlation Y2H-union between <br />                                        connectivity number phenotypes observed global phenotypic profiling analyses <br />                                        yeast genes (36) (Fig. 3D). Thus number binary physical interactions mediated a <br />                                        protein better correlate number cellular processes participates <br />                                        essentiality. The correlation degree number phenotypes observed <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        Combined-AP/MS, likely co-complex associations reflect size protein <br />                                        complexes number processes involved in. <br />  <br />                                        We confirmed concept modularity yeast interactome network, date hubs <br />                                        dynamically interact partners appear particularly central global connectivity <br />                                        static party hubs appear function locally specific biological modules (37). The <br />                                        proportion date party hubs strikingly different Y2H-union Combined- <br />                                        AP/MS (Fig. 3E). There significantly date hubs binary network, party <br />                                        hubs prevalent co-complex network. In binary network, date hubs crucial to <br />                                        topological integrity network, party hubs minimal effects. However, in <br />                                        co-complex network, date party hubs affect topological integrity network <br />                                        equally, likely hubs Combined-AP/MS reside large stable complexes, while <br />                                        hubs Y2H-union preferentially connect diverse cellular processes. <br />  <br />  <br />  <br />                                             Science. Author manuscript; available PMC 2009 September 18. <br />                             Yu et al.                                                                                                              Page 6 <br />  <br />  <br />                                        Surprisingly, essential proteins strongly tended interact (Fig. 4A; SOM IX). <br />                                        Concentrating subnetwork formed interactions mediated essential <br />                                        proteins (fig. S10), giant component size larger expected by <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        chance (Fig. 4B). To better understand clustering essential proteins, examined the <br />                                        interacting essential protein pairs reported complex, finding 106 <br />                                        interacting essential protein pairs, greater number expected chance (Fig. 4C; SOM <br />                                        IX). <br />  <br />                                        We investigated overall relationships Y2H-union Gene Ontology (GO) <br />                                        attributes (38), phenotypic expression profiling similarities (39), transcriptional <br />                                        regulatory networks (40). Both Y2H-union Combined-AP/MS significant <br />                                        enrichment (all P&lt;10-10) functionally similar pairs GO branches (Fig. 5A) <br />                                        (41). There significant enrichment positive correlations phenotypic profiles (36) <br />                                        interacting pairs datasets (Fig. 5B; fig. S11). Such interactions supported by <br />                                        strong phenotypic information constitute likely possibilities functional relationships. Lastly, <br />                                        datasets significantly enriched pairs co-expressed conditions (fig. <br />                                        S12), Combined-AP/MS shows higher enrichment (Fig. 5C), agreeing the <br />                                        different nature assays: AP/MS aims detecting stable complexes Y2H <br />                                        tends detect transient condition specific protein interactions. This observation is <br />                                        supported enrichment kinase-substrate pairs Y2H-union (SOM X; fig. S13). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        To explore mechanisms co-expression interacting protein pairs combined <br />                                        transcriptional regulatory networks interactome network information (40). Interacting <br />                                        proteins networks showed tendency co-regulated common transcription <br />                                        factors (TFs; Fig. 5D). Similarly observed co-expression correlation analysis <br />                                        (Fig. 5C), enrichment interacting pairs Combined-AP/MS significantly higher <br />                                        Y2H-union. Strikingly, observed significant enrichment protein-protein <br />                                        interactions TFs involved common &#226;&#8364;&#339;multi-input motif&#226;&#8364;? (42,43) (MIM, where <br />                                        multiple TFs co-regulate given set genes; Fig. 5D; SOM &#195;&#8212;). The fraction co-regulating <br />                                        TF pairs higher binary interactome co-complex network, suggesting <br />                                        various TFs function form transient complexes differentially regulate <br />                                        transcriptional targets (44). <br />  <br />                                        These observations suggest binary interactome dataset enriched transient or <br />                                        condition-specific interactions linking different subcellular processes molecular machines. <br />                                        To explore possibility calculated &#226;&#8364;&#339;edge-betweenness&#226;&#8364;? interaction a <br />                                        merged network available interactions (SOM XI), measuring number shortest paths <br />                                        protein pairs traverse given edge. The higher edge-betweenness of <br />                                        interactions Y2H-union shows tendency Y2H detect key interactions outside of <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        complexes, significantly AP/MS (Fig. 5D). Several examples complex- <br />                                        to-complex connectivity evident complete map MIPS complexes connected Y2H <br />                                        interactions (fig. S14). <br />  <br />                                        Overall, infer Y2H interrogates different subspace interactome than <br />                                        AP/MS, Y2H interactions represent key connections different complexes and <br />                                        pathways. Y2H AP/MS provide orthogonal information interactome both <br />                                        vital obtain complete picture cellular protein-protein interaction networks. <br />  <br />                               Acknowledgments <br />                                        Supported funds W.M. Keck Foundation awarded M.V. F.P.R.; Institute Sponsored Research <br />                                        funds Dana-Farber Cancer Institute Strategic Initiative awarded M.V. CCSB; National Institutes <br />                                        Health (NIH) grant (R01-HG001715) awarded M.V. F.P.R.; National Institutes Health (NIH) grants <br />                                        (U01-A1070499-01 U56-CA113004) awarded A.-L.B.; grants University Ghent <br />  <br />  <br />  <br />                                              Science. Author manuscript; available PMC 2009 September 18. <br />                             Yu et al.                                                                                                             Page 7 <br />  <br />                                        (GOA12051401) Fund Scientific Research Flanders (FWO-V G.0031.06) awarded J.T.; grant <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br />                                        National Cancer Institute Canada awarded C.B.; NIH grant (HG003224) awarded F.P.R. I.L. is <br />                                        postdoctoral fellow FWO-V. M.V. &#226;&#8364;&#339;Chercheur Qualifi&#195;&#169; Honoraire&#226;&#8364;? Fonds la Recherche <br />                                        Scientifique (FRS-FNRS, French Community Belgium). We thank members laboratories helpful <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        discussions, Agencourt Biosciences sequencing assistance. <br />  <br />  <br />                               References Notes <br />                                        1. Fromont-Racine M, Rain JC, Legrain P. Nat. Genet 1997;16:277. [PubMed: 9207794] <br />                                        2. Uetz P, et al. Nature 2000;403:623. [PubMed: 10688190] <br />                                        3. Ito T, et al. Proc. Natl. Acad. Sci. USA 2001;98:4569. [PubMed: 11283351] <br />                                        4. Walhout AJ, et al. Science 2000;287:116. [PubMed: 10615043] <br />                                        5. Reboul J, et al. Nat. Genet 2003;34:35. [PubMed: 12679813] <br />                                        6. Li S, et al. Science 2004;303:540. [PubMed: 14704431] <br />                                        7. Giot L, et al. Science 2003;302:1727. [PubMed: 14605208] <br />                                        8. Colland F, et al. Genome Res 2004;14:1324. [PubMed: 15231748] <br />                                        9. Rual JF, et al. Nature 2005;437:1173. [PubMed: 16189514] <br />                                        10. Stelzl U, et al. Cell 2005;122:957. [PubMed: 16169070] <br />                                        11. Butland G, et al. Nature 2005;433:531. [PubMed: 15690043] <br />                                        12. Arifuzzaman M, et al. Genome Res 2006;16:686. [PubMed: 16606699] <br />                                        13. Gavin AC, et al. Nature 2002;415:141. [PubMed: 11805826] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        14. Ho Y, et al. Nature 2002;415:180. [PubMed: 11805837] <br />                                        15. Gavin AC, et al. Nature 2006;440:631. [PubMed: 16429126] <br />                                        16. Krogan NJ, et al. Nature 2006;440:637. [PubMed: 16554755] <br />                                        17. Ewing RM, et al. Mol. Syst. Biol 2007;3:89. [PubMed: 17353931] <br />                                        18. Han JD, Dupuy D, Bertin N, Cusick ME, Vidal M. Nat. Biotechnol 2005;23:839. [PubMed: 16003372] <br />                                        19. Scholtens D, Vidal M, Gentleman R. Bioinformatics 2005;21:3548. [PubMed: 15998662] <br />                                        20. Reguly T, et al. J. Biol 2006;5:11. [PubMed: 16762047] <br />                                        21. Remy I, Michnick SW. Methods Mol. Biol 2004;261:411. [PubMed: 15064473] <br />                                        22. Gelperin DM, et al. Genes Dev 2005;19:2816. [PubMed: 16322557] <br />                                        23. Mewes HW, et al. Nucleic Acids Res 2006;34:D169. [PubMed: 16381839] <br />                                        24. von Mering C, et al. Nature 2002;417:399. [PubMed: 12000970] <br />                                        25. Bader JS, Chaudhuri A, Rothberg JM, Chant J. Nat. Biotechnol 2004;22:78. [PubMed: 14704708] <br />                                        26. Yu H, et al. Genome Res 2004;14:1107. [PubMed: 15173116] <br />                                        27. Vidalain PO, Boxem M, Ge H, Li S, Vidal M. Methods 2004;32:363. [PubMed: 15003598] <br />                                        28. Walhout AJ, Vidal M. Genome Res 1999;9:1128. [PubMed: 10568752] <br />                                        29. Eyckerman S, et al. Nat. Cell Biol 2001;3:1114. [PubMed: 11781573] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        30. Jansen R, Gerstein M. Curr. Opin. Microbiol 2004;7:535. [PubMed: 15451510] <br />                                        31. Grigoriev A. Nucleic Acids Res 2003;31:4157. [PubMed: 12853633] <br />                                        32. Jansen R, et al. Science 2003;302:449. [PubMed: 14564010] <br />                                        33. Collins SR, et al. Mol. Cell. Proteomics 2007;6:439. [PubMed: 17200106] <br />                                        34. Barab&#195;&#161;si AL, Albert R. Science 1999;286:509. [PubMed: 10521342] <br />                                        35. Jeong H, Mason SP, Barabasi AL, Oltvai ZN. Nature 2001;411:41. [PubMed: 11333967] <br />                                        36. Dudley AM, Janse DM, Tanay A, Shamir R, Church GM. Mol. Syst. Biol 2005;1:0001. [PubMed: <br />                                             16729036] <br />                                        37. Han JD, et al. Nature 2004;430:88. [PubMed: 15190252] <br />                                        38. Gene Ontology Consortium. Nucleic Acids Res 2008;36:D440. [PubMed: 17984083] <br />                                        39. Vidal M. Cell 2001;104:333. [PubMed: 11239391] <br />                                        40. Yu H, Gerstein M. Proc. Natl. Acad. Sci. USA 2006;103:14724. [PubMed: 17003135] <br />                                        41. Yu H, Jansen R, Stolovitzky G, Gerstein M. Bioinformatics 2007;23:2163. [PubMed: 17540677] <br />  <br />  <br />  <br />                                              Science. Author manuscript; available PMC 2009 September 18. <br />                             Yu et al.                                                                                           Page 8 <br />  <br />  <br />                                        42. Lee TI, et al. Science 2002;298:799. [PubMed: 12399584] <br />                                        43. Milo R, et al. Science 2002;298:824. [PubMed: 12399590] <br />                                        44. Luscombe NM, et al. Nature 2004;431:308. [PubMed: 15372033] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        45. Yu H, Kim PM, Sprecher E, Trifonov V, Gerstein M. PLoS Comput. Biol 2007;3:e59. [PubMed: <br />                                            17447836] <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Science. Author manuscript; available PMC 2009 September 18. <br />                             Yu et al.                                                                     Page 9 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        Science. Author manuscript; available PMC 2009 September 18. <br />                             Yu et al.                                                                                          Page 10 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        Fig. 1. <br />                                        Evaluation S. cerevisiae protein-protein interaction datasets. (A) Number interactions <br />                                        reported various large-scale S. cerevisiae protein-protein interaction datasets. (B) Schema <br />                                        pipeline used assemble binary positive random reference sets. (C) Fraction a <br />                                        random sample interactions dataset confirmed Y2H PCA. (D) Fraction of <br />                                        positives dataset calculated using MIPS Binary-GS. (Error bars indicate standard <br />                                        error). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              Science. Author manuscript; available PMC 2009 September 18. <br />                             Yu et al.                                                                                    Page 11 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        Fig. 2. <br />                                        Large-scale Y2H interactome screen. (A) Completeness Y2H screen. (B) Sampling- <br />                                        sensitivity CCSB Y2H screens measured screening subspace multiple times. (C) <br />                                        Fraction protein pairs PRS, RRS, CCSBYI1 test positive PCA, MAPPIT or <br />                                        Y2H. (D) Overlaps high-quality large-scale S. cerevisiae Y2H datasets <br />                    <br /> </body></html>