<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                                                                                                                                                     JCB: ARTICLE <br />  <br />  <br />  <br />  <br />                               Excess Mcm2&#226;&#8364;&#8220;7 license dormant origins <br />                               replication used conditions <br />                               replicative stress <br />                               Anna M. Woodward,1 Thomas G&#195;&#182;hler,1 M. Gloria Luciani,1 Maren Oehlmann,1 Xinquan Ge,1 Anton Gartner,1 <br />                               Dean A. Jackson,2 J. Julian Blow1 <br />                               1 <br />                                Wellcome Trust Biocentre, University Dundee, Dundee DD1 5EH, Scotland, UK <br />                               2 <br />                                Department Biomolecular Sciences, University Manchester Institute Science Technology, Manchester, M60 1QD, England, UK <br />  <br />  <br />  <br />  <br />                               I <br />                                  n late mitosis early G1, replication origins                             allow complete genome replication presence of <br />                                  licensed subsequent use loading complexes                          replication inhibitors. These results suggest metazoan <br /> THE JOURNAL OF CELL BIOLOGY <br />  <br />  <br />  <br />  <br />                                  minichromosome maintenance proteins 2&#226;&#8364;&#8220;7 (Mcm2&#226;&#8364;&#8220;7).                                replication origins actually comprised can- <br />                               The number Mcm2&#226;&#8364;&#8220;7 complexes loaded DNA                                      didate origins, normally remain dormant <br />                               greatly exceeds number replication origins used                              unless cells experience replicative stress. Consistent with <br />                               S phase, function excess Mcm2&#226;&#8364;&#8220;7                               model, using Caenorhabditis elegans, that <br />                               unknown. Using Xenopus laevis egg extracts,                              partial RNAi-based knockdown MCMs no <br />                               excess Mcm2&#226;&#8364;&#8220;7 complexes license additional                               observable effect normal conditions causes lethality <br />                               dormant origins &#239;&#172;?re unperturbed                                  treatment low, nontoxic, levels the <br />                               S phases suppression caffeine-sensitive                             replication inhibitor hydroxyurea. <br />                               checkpoint pathway. Use additional origins can <br />  <br />  <br />  <br />  <br />                               Introduction <br />                               In eukaryotes, precise duplication genome S phase                     new origins prevented late G1 midmitosis <br />                               achieved initiation replication numerous ori-                  (Diffley, 2004; Blow Dutta, 2005). This means what- <br />                               gins distributed DNA. In late mitosis early                      problems encountered replication forks during <br />                               G1, origins licensed replication loading complexes                       S phase, previously licensed sites used. If con- <br />                               minichromosome maintenance proteins 2&#226;&#8364;&#8220;7 (Mcm2&#226;&#8364;&#8220;7),                            verging replication forks stall irreversibly S phase, the <br />                               thus, forming &#226;&#8364;&#339;prereplicative complex&#226;&#8364;? (Diffley, 2004; Blow                       DNA probably remain unreplicated, poten- <br />                               Dutta, 2005). Licensing involves coordinated action                      tially leading cell death chromosome rearrangement. <br />                               origin recognition complex (ORC) Cdc6 Cdt1 proteins                         Therefore, crucial cells organize replication <br />                               (Gillespie et al., 2001), probably act clamp Mcm2&#226;&#8364;&#8220;7                        machinery minimize risk happening. <br />                               DNA. Mcm2&#226;&#8364;&#8220;7 displaced replication ori-                                 Previous work range eukaryotes, including <br />                               gins initiate, suggested provide                    Saccharomyces cerevisiae, humans, Xenopus laevis, has <br />                               helicase activity unwind DNA ahead replication fork                       demonstrated Mcm2&#226;&#8364;&#8220;7 complexes loaded DNA in <br />                               (Labib Diffley, 2001). <span id='am-4' about='protege:TO' typeof='owl:Thing'>To</span> prevent replication origins                       20-fold excess number DNA-bound ORC mol- <br />                               firing single S phase, ability license                   ecules number replication origins (Burkhart et al., <br />                                                                                                                   1995; Lei et al., 1996; Rowles et al., 1996; Donovan et al., <br />                               Correspondence J. Julian Blow: j.j.blow@dundee.ac.uk                             1997; Mahbubani et al., 1997; Edwards et al., 2002). However, <br />                               M.G. Luciani&#226;&#8364;&#8482;s present address Dept. Medicine, Vienna General Hospital,       normal replication rates maintained number of <br />                               1090 Vienna, Austria.                                                               Mcm2&#226;&#8364;&#8220;7 molecules reduced 1&#226;&#8364;&#8220;2 origin (Mahbubani <br />                               M. Oehlmann&#226;&#8364;&#8482;s present address Dept. Biochemistry, National University         et al., 1997; Edwards et al., 2002; Cortez et al., 2004; Oehlmann <br />                               Ireland, Galway, Ireland. <br />                               Abbreviations used paper: HU, hydroxyurea; MCM, minichromosome <br />                                                                                                                   et al., 2004; Tsao et al., 2004). This observation, termed the <br />                               maintenance; ODP, origin decision point; ORC, origin recognition complex.           &#226;&#8364;&#339;MCM paradox&#226;&#8364;? (Hyrien et al., 2003), led speculation on <br />                               The online version article contains supplemental material.                  possible function extra complexes, particularly as <br />  <br />                               &#194;&#169; The Rockefeller University Press $8.00 <br />                               The Journal Cell Biology, Vol. 173, No. 5, June 5, 2006 673&#226;&#8364;&#8220;683 <br />                               http://www.jcb.org/cgi/doi/10.1083/jcb.200602108 <br />                                                                                                                                                                                   JCB   673 <br />        MCM proteins exclusively colocalize sites DNA <br />       synthesis S phase (Madine et al., 1995; Krude et al., 1996; <br />       Dimitrova et al., 1999). <br />              There evidence Mcm2&#226;&#8364;&#8220;7 complexes on <br />       DNA significant distances ORC bound (Ritzi <br />       et al., 1998; Edwards et al., 2002; Harvey Newport, 2003; <br />       Danis et al., 2004). It suggested distant <br />       multiple MCM complexes cooperatively pump double- <br />       stranded DNA and, thus, unwind (Laskey Madine, <br />       2003). More recently, role excess MCM proteins in <br />       checkpoint activation proposed, based obser- <br />       vation cells partially depleted Mcm7 display reduc- <br />       tion replication checkpoint signaling (Cortez et al., 2004; <br />       Tsao et al., 2004). A possibility suggested <br />       excess Mcm2&#226;&#8364;&#8220;7 provide replication origins use <br />       certain contingencies, incomplete DNA replica- <br />       tion (Lucas et al., 2000; Edwards et al., 2002; Hyrien et al., <br />       2003; Blow Dutta, 2005). These different proposals are <br />       mutually exclusive. <br />              In study, demonstrate role excess Mcm2&#226;&#8364;&#8220;7 <br />       complexes licensing &#226;&#8364;&#339;dormant&#226;&#8364;? replication origins. These <br />       dormant origins effectively suppressed unperturbed         Figure 1. Replication characteristics minimally licensed DNA. (A) Sperm <br />       DNA replication, support initiation replication          nuclei incubated X. laevis egg extract supplemented biotin-16- <br />                                                                             dUTP. To produce minimally licensed DNA, extract supplemented <br />       forks stalled response range replicative stresses,     geminin shortly sperm addition. After 30 min, DNA isolated <br />       including replication inhibition aphidicolin, mitomycin C,         spread glass slides, biotin-labeled tracks detected <br />       etoposide, actinomycin D. In support model, dem-        &#239;&#172;&#8218;uorescent antibodies. The distance center points of <br />                                                                             adjacent tracks labeled DNA measured. (B) Sperm nuclei were <br />       onstrate Caenorhabditis elegans, partial knockdown            incubated X. laevis egg extract containing &#206;&#177;-[32P]dATP, plus minus <br />       MCMs induces hypersensitivity nontoxic levels         aphidicolin and/or caffeine. To produce minimally licensed DNA, extract <br />       hydroxyurea (HU).                                                  supplemented geminin shortly sperm addition. At indi- <br />                                                                             cated times, total DNA synthesized measured. <br />  <br />  <br />       Results <br />                                                                             mum licensing; 17.1 kb minimum licensing). Under normal <br />       Minimally licensed chromatin replicates                               circumstances, clusters 3&#226;&#8364;&#8220;7 adjacent origins initiate simi- <br />       poorly presence aphidicolin                                 lar times, different clusters activated different <br />       caffeine                                                          stages S phase (Blow et al., 2001). We observed major <br />       When demembranated sperm nuclei incubated X. laevis            difference minimally maximally licensed DNA <br />       egg extracts, average 10&#226;&#8364;&#8220;20 Mcm2&#226;&#8364;&#8220;7 complexes                clustering replication origins, average 6.1 <br />       loaded replication origin entry S phase         origins cluster maximally licensed DNA 4.8 for <br />       (Mahbubani et al., 1997; Edwards et al., 2002; Oehlmann et al.,       minimally licensed DNA. Therefore, conclude minimum <br />       2004); DNA &#226;&#8364;&#339;maximally licensed.&#226;&#8364;? If Cdt1             licensing does significantly change replication origin usage <br />       inhibitor geminin added extract shortly sperm     normal conditions. <br />       DNA, number DNA-bound MCM complexes                            Next, examined excess Mcm2&#226;&#8364;&#8220;7 complexes <br />       limited minimum required support approximately              important replication forks stress by <br />       normal replication kinetics (Oehlmann et al., 2004);     supplementing extracts low concentrations DNA <br />       DNA &#226;&#8364;&#339;minimally licensed.&#226;&#8364;? To examine role excess               polymerase inhibitor aphidicolin. Fig. 1 B (crosses closed <br />       Mcm2&#226;&#8364;&#8220;7 complexes, investigated minimum               circles) shows replication minimally maximally li- <br />       licensing causes alteration spacing adjacent        censed DNA inhibited similar extents 10 &#206;&#188;M aphidicolin. <br />       replication origins. Nascent DNA labeled biotin-16-          We previously shown that, concentration, aphidico- <br />       dUTP early S phase, DNA isolated,          lin slows replication forks approximately threefold, that <br />       spread microscope slides, analyzed fluorescence             major effect replication induce ATR-dependent <br />       microscopy. Short fluorescent tracks seen,            checkpoint suppresses initiation events (Luciani <br />       caused bidirectional replication origins center     et al., 2004). Therefore, supplemented extracts with <br />       track, spacing replication origins           caffeine, ATR kinase inhibitor (Fig. 1 B, crossed <br />       determined measuring distance centers           boxes). Consistent previous study (Luciani et al., 2004), <br />       adjacent tracks (Herrick et al., 2000; Blow et al., 2001). Fig. 1 A   addition caffeine completely rescued aphidicolin- <br />       shows significant difference average         induced inhibition replication maximally licensed <br />       interorigin distance samples (15.8 kb maxi-       DNA. In contrast, caffeine partially rescued replication <br />  <br />  <br /> 674   JCB &#226;&#8364;&#162; VOLUME 173 &#226;&#8364;&#162; NUMBER 5 &#226;&#8364;&#162; 2006 <br />  when minimally licensed DNA replicated presence <br /> of aphidicolin. This suggests excess Mcm2&#226;&#8364;&#8220;7 complexes <br /> are required way allow rescue DNA replication <br /> after inhibition replication fork progression. <br />  <br /> Maximally licensed chromatin use <br /> extra origins replication <br /> Replication forks X. laevis normally progress  10 bp/s <br /> (Callan, 1972; Mahbubani et al., 1992; Strausfeld et al., 1994; <br /> Walter Newport, 1997), slowed approximately <br /> threefold 10 &#206;&#188;M aphidicolin (Luciani et al., 2004). Fork rate <br /> is significantly affected presence caffeine (Luciani <br /> et al., 2004). To replicate normal rates presence of <br /> aphidicolin caffeine, maximally licensed DNA use <br /> more replication forks normal. Minimally licensed DNA, <br /> in contrast, does appear capable using replication <br /> forks. Therefore, assessed number active forks by <br /> measuring quantity Cdc45 loaded chromatin. Cdc45 <br /> loads replication forks (probably binding Mcm2&#226;&#8364;&#8220;7) just <br /> before initiation moves forks elongate <br /> (Hopwood Dalton, 1996; Aparicio et al., 1997, 1999; Zou <br /> and Stillman, 1998; Mimura Takisawa, 1998). Fig. 2 A <br /> shows maximally minimally licensed chromatin <br /> contained similar quantities Cdc45 undisturbed <br /> S phase, expected similar replication rates. Con- <br /> sistent previous study (Edwards et al., 2002), amount <br /> of Cdc45 maximally licensed chromatin increased  20-fold           Figure 2. Caffeine allows initiation extra forks maximally <br /> when extract treated combination aphidicolin            licensed DNA. (A) Sperm nuclei incubated X. laevis egg extract, <br /> and caffeine. In contrast, minimally licensed chromatin showed        plus minus aphidicolin and/or caffeine. To produce minimally licensed <br />                                                                       DNA, extract supplemented geminin shortly sperm addition. <br /> only two- threefold increase similar conditions.           At 40 min, chromatin isolated immunoblotted Cdc45. <br /> A slight increase chromatin-bound Cdc45 seen         Coomassie-stained histones shown loading control. (B) Sperm <br /> both maximally minimally licensed chromatin treated          nuclei (400 ng DNA) incubated different volumes egg extract <br />                                                                       supplemented 40 &#206;&#188;M aphidicolin plus caffeine. To produce minimally <br /> with caffeine, consistent acceleration       licensed DNA, extract supplemented geminin shortly sperm <br /> origin firing caffeine causes (Marheineke Hyrien, 2004;      addition. At 40 min, chromatin isolated immunoblotted for <br /> Shechter et al., 2004).                                               Cdc45. (C) Sperm nuclei incubated egg extract supplemented with <br />                                                                       biotin-16-dUTP caffeine. To produce minimally licensed DNA, extract <br />        Next, performed quantitative immunoblotting esti-        supplemented geminin shortly sperm addition. At 30 min, <br /> mate Cdc45 loaded chromatin            DNA isolated spread glass slides, biotin-labeled tracks <br /> conditions (Fig. 2 B). A fixed quantity X. laevis sperm nuclei     detected &#239;&#172;&#8218;uorescent antibodies. The distance center <br />                                                                       points adjacent tracks labeled DNA measured. <br /> was incubated increasing quantities egg extract supple- <br /> mented aphidicolin caffeine. Extracts optionally <br /> supplemented geminin shortly sperm addition                   These results suggest maximally licensed DNA should <br /> create minimally licensed chromatin. In absence added          contain increased density nascent strands conditions <br /> geminin (maximum licensing), quantity chromatin-bound          increased number forks. Because recent <br /> Cdc45 increased increasing extract volume (Fig. 2 B,             studies shown caffeine accelerate origin firing <br /> shaded bars), caused, part, increased quantities Mcm2&#226;&#8364;&#8220;7      (Marheineke Hyrien, 2004; Shechter et al., 2004), re- <br /> being loaded DNA (Mahbubani et al., 1997). On            peated fluorescent fiber-labeling experiments using extracts <br /> minimally licensed chromatin, however, Cdc45 remained              supplemented caffeine. Fig. 2 C (left) shows when <br /> a fairly constant level 3&#226;&#8364;&#8220;5 ng (Fig. 2 B, open bars). This level   maximally licensed chromatin replicated extract contain- <br /> of Cdc45 minimally licensed chromatin corresponds 1&#226;&#8364;&#8220;2           ing caffeine average origin spacing declined dramatically <br /> molecules Cdc45 10-kb DNA, 1&#226;&#8364;&#8220;2 molecules Cdc45           (compare Fig. 1 A), observed spacing na- <br /> per molecule ORC (Rowles et al., 1996). If single       scent strands falling near resolution limit technique <br /> molecule Cdc45 replication fork, results sug-        (1&#226;&#8364;&#8220;2 kb). When fiber-labeling experiments repeated <br /> gest normal conditions single molecule ORC            minimally licensed chromatin, reduction spacing was <br /> allows, average, single pair replication forks initiate;   pronounced maximally licensed DNA (Fig. <br /> the number forks increased certain circum-            2 C, right). These data suggest extra Mcm2&#226;&#8364;&#8220;7 molecules <br /> stances, excess Mcm2&#226;&#8364;&#8220;7 molecules present        maximally licensed DNA license dormant replication origins <br /> on DNA.                                                           used presence caffeine. <br />  <br />  <br />                                                           EXCESS MCM2&#226;&#8364;&#8220;7 LICENSE DORMANT ORIGINS &#226;&#8364;&#162; WOODWARD ET AL.                                  675 <br />                                                                                                If correct, nascent strands addi- <br />                                                                                        tional origins rapidly fuse high molecular weight <br />                                                                                        products observable alkaline agarose gels (Fig. 3 A, <br />                                                                                        left). On minimally licensed DNA, interorigin distances are <br />                                                                                        larger, nascent strand fusion occur later (Fig. 3 A, <br />                                                                                        right). To test this, maximally minimally licensed DNA sam- <br />                                                                                        ples pulsed &#206;&#177;-[32P]dATP beginning S phase, <br />                                                                                        chased unlabeled dATP different periods. <br />                                                                                        Fig. 3 B (i iv) shows maximally minimally licensed <br />                                                                                        DNA gave similar profiles absence drug <br />                                                                                        treatment. Although long smear nascent strands makes an <br />                                                                                        accurate rate determination impossible, modal size na- <br />                                                                                        scent strands increased rate  25 nt/s. Within expected <br />                                                                                        error, consistent strand elongated two <br />                                                                                        forks, moving 10 nt/s, previously reported (Callan, <br />                                                                                        1972; Mahbubani et al., 1992; Strausfeld et al., 1994; Walter <br />                                                                                        Newport, 1997; Luciani et al., 2004). <br />                                                                                               In contrast, behavior maximally minimally <br />                                                                                        licensed DNA presence aphidicolin caffeine (Fig. 3 B, <br />                                                                                        ii v) showed dramatic difference consistent with <br />                                                                                        model shown Fig. 3 A. This highlighted Fig. 3 C, <br />                                                                                        relative abundance nascent strands sample <br />                                                                                        lane corresponding alkaline gel plotted, the <br />                                                                                        earliest sample lane plotted nearest top. Both maximally and <br />                                                                                        minimally licensed nascent strands started uni- <br />                                                                                        form size untreated sample aphidicolin had <br />                                                                                        limited rate elongation. During minutes, both <br />                                                                                        maximally minimally licensed nascent strands increased in <br />                                                                                        size  6 nt/s ( 3 nt/s fork). For minimally licensed <br />                                                                                        sample (Fig. 3, B [v] C [v]), strands maintained this <br />                                                                                        rate time course. In later samples, however, some <br />                                                                                        strands doubled size, consistent fusion between <br />                                                                                        adjacent strands (Fig. 3 B, v, arrows). The larger strands first <br />                                                                                        visible  10 kb, consistent being <br />                                                                                        fusions smallest replicons  5 kb seen Fig. 1 A. <br />                                                                                        A different <span id='am-83' about='xsp:pattern' typeof='owl:Thing'>pattern</span> seen maximally licensed DNA <br />                                                                                        (Fig. 3, B [ii] C [ii]). A smear higher molecular weight <br />                                                                                        strands built rapidly did minimally <br />                                                                                        licensed sample, middle time series ma- <br />                                                                                        jority strands shifted higher molecular weight <br />                                                                                        form. By end time series, majority strands <br />                                                                                        exclusion limit gel. Unlike case the <br />                                                                                        minimally licensed DNA, obvious lower size limit <br />                                                                                        strand fusion events occurred. These results suggest <br />  <br />  <br />       Figure 3. Maximally licensed DNA undergo premature nascent <br />       strand fusion presence caffeine. (A) Model use addi-        supplemented &#206;&#177;-[32P]dATP, chased unlabeled <br />       tional dormant origins extract treated aphidicolin caffeine.         dATP 2 min (i, iii, iv, vi) 5 min (ii v). At different <br />       A small segment ( 30 kb) chromosomal DNA shown, normally             times DNA isolated, separated alkaline agarose gel <br />       supports initiation origins. Candidate replication origins       electrophoresis, autoradiographed. The times minute <br />       bound Mcm2&#226;&#8364;&#8220;7 shown gray circles. Nascent strands shown             27&#226;&#8364;&#8220;36 min. The times ii 31, 33, minute 35&#226;&#8364;&#8220;45, <br />       double-headed arrows. On maximally licensed DNA (left), nascent strands          48, 51, 54 min. The times iii minute 23&#226;&#8364;&#8220;32 min. <br />       initiate dormant origins rapidly fuse. On minimally licensed chro-      The times iv minute 23&#226;&#8364;&#8220;30, 32, 34, 36, 40 min. <br />       matin (right), dormant origins &#239;&#172;?rst strand fusion events     The times v 2 min 32&#226;&#8364;&#8220;50, 55, 60 min. The times <br />       occur closely spaced origins. (B C) Sperm nuclei            vi min 23&#226;&#8364;&#8220;30, 32, 34, 36, 40 min. Molecular <br />       incubated X. laevis egg extract, plus minus aphidicolin and/or caffeine.   weight markers (&#206;&#187;-HindIII) shown left (sizes kilobases). The <br />       To produce minimally licensed DNA, extract supplemented                 autoradiographs shown B. The x-ray &#239;&#172;?lm scanned the <br />       geminin shortly sperm addition. i&#226;&#8364;&#8220;iii, maximally licensed DNA; iv&#226;&#8364;&#8220;vi,      density <span id='am-2' about='rdfs:label' typeof='owl:Thing'>label</span> lane quanti&#239;&#172;?ed. The scans lane shown <br />       minimally licensed DNA. iv, additions; ii v, plus caffeine      C (earliest times top, slowest migration right). The migration <br />       aphidicolin; iii vi, plus caffeine. At start S phase, extract     &#206;&#187;-HindIII shown ticks bottom. <br />  <br />  <br /> 676   JCB &#226;&#8364;&#162; VOLUME 173 &#226;&#8364;&#162; NUMBER 5 &#226;&#8364;&#162; 2006 <br />                                                                                 presence aphidicolin caffeine, maximally li- <br />                                                                                censed DNA displays dramatic reduction replicon size, an <br />                                                                                effect seen minimally licensed DNA. Note <br />                                                                                that, consistent previous work (Luciani et al., 2004), <br />                                                                                sign caffeine reducing fork stability, did <br />                                                                                observe buildup low molecular weight products. <br />                                                                                       When extracts supplemented caffeine alone, <br />                                                                                obvious difference maximally mini- <br />                                                                                mally licensed DNA, samples yielding fully repli- <br />                                                                                cated product (Fig. 3 B, iii vi). Our fiber-labeling experiments <br />                                                                                caffeine (Fig. 2 C) predict caffeine increase <br />                                                                                number active forks DNA maximally li- <br />                                                                                censed. However, fork-slowing effect aphidico- <br />                                                                                lin, rapid fusion nascent strands arising <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> <br />                                                                                additional origins dormant absence caf- <br />                                                                                feine. Once initial fusion events taken place the <br />                                                                                maximally licensed sample, maximally minimally li- <br />                                                                                censed samples behave similarly. This effect explains why <br />                                                                                high levels Cdc45 loading maximally <br />                                                                                licensed DNA treated caffeine (Fig. 2 A). <br />  <br />                                                                                The extra origins normally &#239;&#172;?re <br />                                                                                Our results far suggest excess Mcm2&#226;&#8364;&#8220;7 complexes li- <br />                                                                                cense dormant origins, used normal condi- <br />                                                                                tions, rescue DNA synthesis response <br />                                                                                replication fork inhibition. It known Cdks required <br />                                                                                initiation replication forks X. laevis system, but <br />                                                                                fork elongation (Strausfeld et al., 1994; Luciani et al., <br />                                                                                2004). It known Cdks act individual origins very <br />                                                                                shortly origin fires (Strausfeld et al., 1994; Luciani <br />                                                                                et al., 2004). Therefore, predicted Cdks inhib- <br />                                                                                ited start S phase, origins had <br />                                                                                fired, dormant origin firing prevented rescue <br />                                                                                DNA replication place. To inhibit Cdks, used <br />                                                                                roscovitine, purine analogue inhibits initiation <br />                                                                                X. laevis (Meijer et al., 1997; Luciani et al., <br />                                                                                2004). A model predicted behavior shown Fig. 4 A. <br />                                                                                The model left shows initiation occurring dormant <br />                                                                                origins replication occurs presence aphidicolin <br />                                                                                caffeine. The model right shows roscovitine <br />                                                                                added shortly origins fired, initiation <br />                                                                                occur dormant origins, replication profile <br />                                                                                resembles minimally licensed DNA treated aphidi- <br />                                                                                colin caffeine. <br />                                                                                       Fig. 4 (B C) experimental results support <br /> Figure 4. Roscovitine blocks premature nascent strand fusion. (A) Model        model. Fig. 4 B (left) shows effect DNA replication <br /> for effect roscovitine dormant origin &#239;&#172;?ring. The model same <br /> as maximally licensed DNA shown Fig. 3 A, roscovitine       different combinations caffeine, aphidicolin, and <br /> is added shortly &#239;&#172;?rst origins &#239;&#172;?re (right), preventing    roscovitine added X. laevis egg extract start of <br /> dormant origins &#239;&#172;?ring. (B) Sperm nuclei incubated X. laevis       S phase. Consistent previous work (Luciani et al., 2004), <br /> egg extract supplemented &#206;&#177;-[32P]dATP. At 25 min (left), 30 min <br /> (right), extract optionally supplemented aphidicolin, caffeine,       addition roscovitine early S phase allowed existing <br /> and/or roscovitine. At indicated times, total DNA synthesis        forks elongate, replication rate subsequently trailed <br /> measured. (C) Sperm nuclei incubated X. laevis egg extract supple-     clustered nascent strands fused another. <br /> mented aphidicolin caffeine. At 25 min, extract supple- <br /> mented &#206;&#177;-[32P]dATP minus (i iii) plus (ii iv) roscovitine. <br /> At 30 min, extract supplemented unlabeled dATP. At indicated <br /> times, DNA isolated, separated alkaline agarose gel electrophoresis, <br /> and autoradiographed. End-labeled &#206;&#187;-HindIII run molecular               The scans lane shown iii iv (earliest times top; <br /> weight standards. The autoradiographs shown ii. The x-ray &#239;&#172;?lm     slowest migration right). The migration &#206;&#187;-HindIII shown ticks <br /> was scanned density label lane quanti&#239;&#172;?ed.      bottom. <br />  <br />  <br />                                                                   EXCESS MCM2&#226;&#8364;&#8220;7 LICENSE DORMANT ORIGINS &#226;&#8364;&#162; WOODWARD ET AL.                                      677 <br />        Figure 5. Minimally licensed DNA sensitive range <br />       replication inhibitors. Sperm nuclei incubated in <br />       X. laevis egg extract supplemented &#206;&#177;-[32P]dATP plus <br />       minus caffeine. To produce minimally licensed DNA, <br />       extract supplemented geminin shortly sperm <br />       addition. At 5 min, extract optionally supplemented <br />       500 &#206;&#188;M mitomycin C, 200 &#206;&#188;M etoposide, 4 ng/&#206;&#188;l <br />       actinomycin D. At indicated times, total of <br />       DNA synthesized measured. <br />  <br />  <br />  <br />  <br />       As expected maximally licensed DNA, aphidicolin inhib-             consistent fusion replicons 5&#226;&#8364;&#8220;10 kb size <br />       ited replication inhibition completely res-          (Fig. 4 C, ii, arrows). This profile resembles minimally <br />       cued addition caffeine. However, roscovitine            licensed DNA treated aphidicolin plus caffeine (Fig. 3, <br />       added aphidicolin caffeine,         B [v] C [v]). As predicted, inhibiting Cdk-dependent initi- <br />       rescue replication. A related experiment shown Fig.           ation dormant origins maximally licensed DNA, pattern <br />       4 B (right), difference inhibitors added       nascent strand labeling resemble minimally <br />       later S phase. At time, addition roscovitine      licensed DNA, dormant origins exist. <br />       significant effect replication kinetics, suggesting that <br />       virtually normal replication origins fired         Dormant origins activated response <br />       time inhibitors added. As before, inhibition           range replication stresses <br />       replication aphidicolin rescued coaddition           We investigated activation dormant <br />       caffeine, rescue substantially dependent fur-       origins maximally licensed DNA useful recovering <br />       ther Cdk activity. Some roscovitine-resistant rescue caf-           replication-induced stresses aphidicolin <br />       feine observed, possibly reflecting additional recovery            (Fig. 5). Both mitomycin C (a cross-linking agent) etopo- <br />       pathways, restart previously stalled forks. The         (a topoisomerase II inhibitor) induced block replica- <br />       significance version experiment require-      tion efficiently rescued caffeine when <br />       ment Cdks seen late stage S phase ori-        excess Mcm2&#226;&#8364;&#8220;7 present DNA. This similar to <br />       gins initiated, suggesting complete replication      results aphidicolin (Fig. 1 A) suggests general <br />       dependent use origins normally           requirement dormant origins rescue DNA synthesis <br />       fired (dormant origins).                                          replication block. Actinomycin D, acts as <br />              To confirm interpretation results, per-         DNA intercalator inhibitor primase compo- <br />       formed alkaline gel analysis nascent strands. Nascent           nent DNA polymerase &#206;&#177; showed dramatic <br />       DNA labeled &#206;&#177;-[32P]dATP beginning S phase,          effect. Replication slightly inhibited maxi- <br />       presence aphidicolin caffeine. The 32P          mally licensed DNA treated 4 ng/&#206;&#188;l actinomycin D, <br />       chased unlabeled dATP, while, time, extract           inhibition severe minimally <br />       supplemented minus (Fig. 4 C, iii) plus roscovitine       licensed DNA. Although small inhibition of <br />       (Fig. 4 C, ii iv). At different times thereafter, labeled DNA      maximally licensed DNA fully reversed caffeine, <br />       analyzed alkaline gels. The sample treated aphidi-         virtually rescue caffeine observed mini- <br />       colin caffeine (Fig. 4 C, i) closely resembled corre-          mally licensed DNA. This result explained inabil- <br />       sponding sample shown Fig. 3 B (ii), nascent DNA               ity actinomycin D activate strong ATR-dependent <br />       increasing rapidly size migrated exclusion      checkpoint response, despite strong inhibition replication <br />       limit gel end time series. Addition rosco-     (Hekmat-Nejad et al., 2000). Dormant origins cases <br />       vitine (Fig. 4 C, ii iv) completely inhibited in-       promote significant recovery replication <br />       crease modal strand <span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span>, strands increased       inhibition need checkpoint alleviation by <br />       size constant rate  6 nt/s ( 3 nt/s fork). At later time   caffeine. Consistent interpretation, treatment of <br />       points, small number strands migrated higher molecu-         X. laevis egg extracts actinomycin D greatly enhances <br />       lar weight smear approximately twice size majority,      Cdc45 loading (Edwards et al., 2002). <br />  <br />  <br /> 678   JCB &#226;&#8364;&#162; VOLUME 173 &#226;&#8364;&#162; NUMBER 5 &#226;&#8364;&#162; 2006 <br />  Figure 6. Knockdown Mcm7 C. elegans causes hypersensitivity HU. C. elegans grown plates plus minus bacteria expressing anti-Mcm7 <br /> siRNA (2%) &#194;&#177;9.5 mM HU. After 7 d, plates examined worm growth. (A) The number F1 (&#239;&#172;?rst) F2 (second) generation worms in <br /> standardized areas counted, &#194;&#177;SD. (B) Representative picture plate, case showing adult F1 worm. Closed arrowhead, F1 adult; open <br /> arrowheads, F2 larvae. Note absence F2 worms sample containing HU anti-Mcm7 siRNA. Bar, 0.2 mm. <br />  <br />  <br /> Excess Mcm2&#226;&#8364;&#8220;7 required C. elegans <br />                                                                             Discussion <br /> to survive HU <br /> The large amounts replicative stress need use          The results presented work provide solution MCM <br /> X. laevis cell-free activate global checkpoint pathways              paradox showing excess Mcm2&#226;&#8364;&#8220;7 license dormant <br /> that prevent replication completing. To test physiologi-           replication origins normally used sup- <br /> cal significance findings, turned vivo system,           pression caffeine-sensitive pathway. In X. laevis, are <br /> where effect lower levels replicative stress              10 dormant origins active origin, dormant <br /> be reliably assessed. We took advantage ability lower             origin activation permitted treatment caffeine, av- <br /> specific gene expression C. elegans providing siRNA-                  erage origin spacing drops 5- 10-fold. The caffeine sensitivity <br /> expressing bacteria food source. We mixed different              suggests involvement ATR-dependent checkpoint in <br /> ratios bacteria expressing MCM7 siRNA bacteria                       suppressing activation dormant origins. This conclusion <br /> expressing vector determine maximum quantity                consistent results mammalian cells, that <br /> MCM7 siRNA observable effect C. elegans                  suppression ATM&#226;&#8364;&#8220;ATR Chk1 checkpoint pathways leads <br /> life cycle (Table S1, available http://www.jcb.org/cgi/content/          dramatic reduction average spacing adjacent <br /> full/jcb.200602108/DC1). This creates state function-             replication origins (unpublished data). We these <br /> ally equivalent minimally licensed state X. laevis.               dormant origins used allow complete replication un- <br /> We determined low concentration HU (9.5 mM),                der conditions replicative stress, propose is <br /> had observable effect C. elegans life cycle (Table S1).           reason existence. <br /> HU lowers dNTP pools inhibiting ribonucleotide reductase                        Detailed analysis effect performed in <br /> and inhibits DNA synthesis. Fig. 6 A shows treat-              X. laevis egg extracts, tested key consequence <br /> ment combination MCM7 siRNA 9.5 mM                     model lowering MCM levels C. elegans opera- <br /> HU completely abrogated C. elegans proliferation. There               tionally mimic minimally licensed state. Reduction MCM <br /> dramatic reduction number adult worms                levels causes dramatic hypersensitivity HU, showing <br /> generation fed siRNA. The adult worms                obvious defect absence replicative stress. There is <br /> this generation appeared sterile (not depicted), pro-             published evidence effect operate mammals. <br /> duced virtually eggs larvae (Fig. 6 B). Similar results          DNA fiber autoradiography studies shown increase in <br /> obtained using knockdown MCM5 MCM6 (Tables S2                        density replication origins replication inhib- <br /> and S3). These experiments reduction Mcm2&#226;&#8364;&#8220;7                  ited FdUrd, thymidylate synthase inhibitor (Ockey <br /> proteins causes dramatic hypersensitivity HU, show-              Saffhill, 1976; Taylor, 1977), treatment ultraviolet <br /> ing obvious defect absence replicative stress.                 light (Griffiths Ling, 1985, 1987, 1989; Painter, 1985). Simi- <br /> Although possible explanations                   larly, reduced nucleotide availability shown increase <br /> happening C. elegans experiments, results fully            frequency initiation origins normally used <br /> consistent results obtained X. laevis idea         (Anglana et al., 2003). Therefore, believe dormant ori- <br /> excess Mcm2&#226;&#8364;&#8220;7 license dormant origins used dur-                gins present metazoans, potentially <br /> ing undisturbed S phases.                                                   provide role responding replicative stresses. <br />  <br />  <br />                                                              EXCESS MCM2&#226;&#8364;&#8220;7 LICENSE DORMANT ORIGINS &#226;&#8364;&#162; WOODWARD ET AL.                                   679 <br />        Figure 7. Model use dormant origins chromosomal DNA. (A) A small segment chromosomal DNA shown single molecule of <br />       ORC bound it. Distributed ORC Mcm2&#226;&#8364;&#8220;7 double hexamers (hexagons), represent candidate origin. The collec- <br />       tion Mcm2&#226;&#8364;&#8220;7 ORC designated single license group. (Bi) Once initiation occurs given license group generates weak <br />       caffeine-sensitive checkpoint signal inhibits initiation Mcm2&#226;&#8364;&#8220;7 license group. (Ci) As fork replicates DNA, uniniti- <br />       ated Mcm2&#226;&#8364;&#8220;7 dormant origins displaced DNA. (Bii) The rightward fork stalls, potentially generating strong global checkpoint signal. <br />       (Cii) One dormant origins license group escapes inhibition initiates, thus, allowing DNA right stalled fork be <br />       replicated. The escape inhibition stochastic, occur result attenuation checkpoint signal DNA repair. (D) Completion <br />       DNA replication. <br />  <br />  <br />       A model behavior                                                        prolonged, forks rapidly replicate surrounding DNA, <br />       dormant origins                                                              displacing bound Mcm2&#226;&#8364;&#8220;7 molecules elimi- <br />       Our results suggest classically defined replication origins            nating dormant origins (Fig. 7, Ci D). Abolishing this <br />       actually comprised candidate origins,                  inhibitory signal caffeine absence replication <br />       normally used, outlined Fig. 7. It known                  stress causes decrease origin spacing, does signifi- <br />       work X. laevis replication forks initiate               cantly increase overall replication rates. In contrast Shechter <br />       sites Mcm2&#226;&#8364;&#8220;7, ORC, bound (Hua Newport,                    et al. (2004), we restrict analysis extracts <br />       1998; Rowles et al., 1999; Edwards et al., 2002; Harvey                     replicate DNA efficiently (as case extracts <br />       Newport, 2003; Danis et al., 2004). Based previous studies,                  recapitulate behavior intact eggs), inhibition ATM and <br />       Mcm2&#226;&#8364;&#8220;7 spread kilobases ORC                      ATR kinases does greatly accelerate replication-timing <br />       loaded them, form term &#226;&#8364;&#339;license group&#226;&#8364;? (Ritzi et al.,              program (Luciani et al., 2004; study). Dormant origins are <br />       1998; Edwards et al., 2002; Harvey Newport, 2003; Danis                     just late-firing origins, confirmed experi- <br />       et al., 2004). Each site containing bound Mcm2&#226;&#8364;&#8220;7 candidate                 ments roscovitine (Fig. 4), replication-timing pro- <br />       replication origin potentially initiate replication (Fig.              gram appears operate independently checkpoint signals <br />       7 A). The wide dispersal Mcm2&#226;&#8364;&#8220;7 increases usefulness                  (unpublished data). Our results rule role Mcm2&#226;&#8364;&#8220;7 <br />       candidate origins dealing replicative stress.                     checkpoint activation; indeed, recently reported phos- <br />               Fig. 7 (Bi&#226;&#8364;&#8220;Ci) shows events occurring absence                    phorylation Mcm2&#226;&#8364;&#8220;7 proteins ATR (Cortez et al., 2004; <br />       replicative stress. Initiation occurs candidate                Tsao et al., 2004) contribute inhibition dormant <br />       origins (the primary origin). Consistent previous studies                  origins propose. <br />       (Edwards et al., 2002; Marheineke Hyrien, 2004; Shechter                           Fig. 7 (Bii&#226;&#8364;&#8220;Cii) shows events occurring origin where <br />       et al., 2004), results caffeine suggest role ATM&#226;&#8364;&#8220;ATR             forks experiences replicative stress stalled or <br />       preventing origin license group initi-               slowed. In Fig. 7 Bii, rightward fork stalls dor- <br />       ating replication. We speculate initiation taken                  mant origins license group eliminated. If enough <br />       place primary origin, ATM&#226;&#8364;&#8220;ATR activated, per-                    forks stall (as occurred experiments using replication <br />       haps locally, inhibit initiation remaining candidate ori-             inhibitors X. laevis), global checkpoint signal activated <br />       gins license group (Fig. 7 Bi). To prevent initiation               represses initiation replication origins the <br />       dormant origins, inhibition need absolute               genome. However, use dormant origins more <br />  <br />  <br /> 680   JCB &#226;&#8364;&#162; VOLUME 173 &#226;&#8364;&#162; NUMBER 5 &#226;&#8364;&#162; 2006 <br />  physiologically important levels fork stalling        tightly localized origins multiple Mcm2&#226;&#8364;&#8220;7 complexes <br /> strongly activate global checkpoint response. Fig. 7 Cii           associated them, mechanism reproducibly <br /> shows fork stalled, nearby dormant          selects initiate form primary origin. <br /> origins undergo initiation, ultimately allowing              It remains unclear certain candidate origins might <br /> DNA region replicated (Fig. 7 D). There        selected preferential initiation, difference <br /> potential explanations dormant origin escape          underlying chromatin attractive possibility <br /> the inhibitory signal emitted stalled fork. One possibility,   (Machida et al., 2005). In case X. laevis, initiation can <br /> mentioned previous paragraph, inhibitory             occur variable distance away ORC is <br /> signal weak (e.g., forks stalled         bound (Harvey Newport, 2003) directed to- <br /> and global suppression initiation), sto-    ward specific DNA sequences DNA-bound transcription <br /> chastic fashion extended lifespan dormant origin pro-        factors (Danis et al., 2004). Because occurs without <br /> vides sufficient length time undergo initiation.         changing localization ORC, presumably happens <br /> Another possibility DNA repair processes operating           increasing initiation probability certain candidate <br /> the stalled fork attenuate checkpoint signal. A possi-        origins. Interestingly, transcription factor binding induced his- <br /> bility certain types replication inhibition (such    tone acetylation favored origins (Danis et al., 2004), <br /> induced actinomycin D) robustly activate check-          supporting case chromatin modification involved <br /> point system, leaving dormant origins free fire. These       candidate origin selection. Somatic cells undergo transi- <br /> controls allow dormant origins used               tion called origin decision point (ODP) middle of <br /> nearby forks encounter problems, thus, lowering probability         G1, initiation directed occur specific preferred <br /> that converging forks stall irreversibly leave         sequences randomly distributed sites (Wu <br /> DNA unable replicate.                                   Gilbert, 1996). Because ODP occurs Mcm2&#226;&#8364;&#8220;7 <br />        Our model emphasizes counterintuitive nature          bound DNA origins functionally <br /> intra&#226;&#8364;&#8220;S phase checkpoint globally blocks origin fir-       licensed (Dimitrova et al., 2002), possible ODP <br /> ing response replication stress con-          represents stage certain candidate origins selected <br /> ditions better cells respond using         preferentially support initiation. <br /> origins, fewer. One possible explanation the <br /> global intra&#226;&#8364;&#8220;S phase checkpoint provides strategy respond- <br />                                                                         Materials methods <br /> ing severe replicative stresses affect replication <br /> forks transient.                                           Preparation use X. laevis egg extracts <br />                                                                         X. laevis egg extracts prepared previously described (Chong <br />                                                                         et al., 1997) supplemented 250 &#206;&#188;g/ml cycloheximide, 25 mM <br /> What replication origin?                                           phosphocreatine, 15 &#206;&#188;g/ml creatine phosphokinase, 300 &#206;&#188;M CaCl2 <br /> Our model suggests normal conditions firing           use. Demembranated X. laevis sperm nuclei (Chong et al., 1997) <br />                                                                         added extract &#239;&#172;?nal concentration 10&#226;&#8364;&#8220;15 ng DNA/&#206;&#188;l and <br /> a primary origin represses origins license group. <br />                                                                         incubated 23&#194;&#176;C. For replication assays, extracts supplemented <br /> Thus, maximally minimally licensed DNA                &#206;&#177;-[32P]dATP, DNA synthesis measured trichloroacetic <br /> overall replication kinetics average origin spacing.       acid precipitation, previously described (Chong et al., 1997). To obtain <br />                                                                         minimally licensed DNA samples, recombinant geminin lacking the <br /> Origin spacing largely independent number of <br />                                                                         destruction box (McGarry Kirschner, 1998; Tada et al., 2001) was <br /> Mcm2&#226;&#8364;&#8220;7 complexes bound DNA determined                  added &#239;&#172;?nal concentration 10 ng/&#206;&#188;l. The appropriate geminin <br /> location ORC binding. In X. laevis, regular ORC spacing (Blow        addition time determined extract produce minimal licens- <br />                                                                         ing, previously described (Oehlmann et al., 2004). Chromatin was <br /> et al., 2001; Oehlmann et al., 2004), coupled repression <br />                                                                         isolated subjected Western blot analysis, previously described <br /> of excess Mcm2&#226;&#8364;&#8220;7 license groups, reliably support            (Oehlmann et al., 2004). <br /> complete genome replication, dormant origins used a <br /> backup problems encountered forks originating            Reagents antibodies <br />                                                                         Aphidicolin (Sigma-Aldrich) dissolved DMSO 18 mM used <br /> from primary origins. This reminiscent &#226;&#8364;&#339;Jesuit model&#226;&#8364;?         &#239;&#172;?nal concentration 10 &#206;&#188;M extract. Caffeine (Sigma-Aldrich) was <br /> for selection replication initiation sites, suggests      dissolved H20 100 mM used &#239;&#172;?nal concentration of <br /> DNA contains potential initiation sites, sites        5 mM. Roscovitine (Calbiochem) dissolved DMSO 400 mM and <br />                                                                         used &#239;&#172;?nal concentration 500 &#206;&#188;M. Mitomycin C dissolved in <br /> repressed, activated (DePamphilis, 1999).              H20 5 mM used &#239;&#172;?nal concentration 500 &#206;&#188;M. Etoposide was <br />         Replication origins metazoans different         dissolved DMSO 25 mM used &#239;&#172;?nal concentration of <br /> forms (DePamphilis, 1999; Gilbert, 2001; Machida et al., 2005).         200 &#206;&#188;M. Actinomycin D dissolved DMSO 1 &#206;&#188;g/&#206;&#188;l used <br />                                                                         &#239;&#172;?nal concentration 4 ng/&#206;&#188;l. Anti-Cdc45 antibody (Mimura and <br /> At replication origins, origin near lamin          Takisawa, 1998) gift H. Takisawa (Osaka University, Osaka, <br /> B2 gene, replication initiates reproducibly tightly defined        Japan). Geminin gift A. Ferenbach (University Dundee, <br /> locus; origins, origin near dihydrofolate      Dundee, Scotland). <br /> reductase gene, replication initiate potential sites <br />                                                                         DNA &#239;&#172;?ber labeling <br /> distributed initiation zone. In case origins       X. laevis sperm DNA incubated extract supplemented 50 &#206;&#188;M <br /> with large initiation zones, dihydrofolate reductase        biotin-16-dUTP (Roche) desired conditions. Reactions (20 &#206;&#188;l) <br /> origin, zone consist multiple sites containing bound        stopped resuspension 350 &#206;&#188;l ice-cold NIB (50 mM <br />                                                                         Hepes-KOH, pH 7.6, 50 mM KCl, 5 mM MgCl2, 2 mM DTT, 0.5 mM <br /> Mcm2&#226;&#8364;&#8220;7 (Alexandrow et al., 2002), can-           spermidine-3HCl, 0.15 mM spermine-4HCl, 100 &#206;&#188;M PMSF, 1 &#206;&#188;g/ml <br /> didate origins initiating S phase. It possible       aprotinin, leupeptin, pepstatin) containing 30 &#206;&#188;M aphidicolin. <br />  <br />  <br />                                                            EXCESS MCM2&#226;&#8364;&#8220;7 LICENSE DORMANT ORIGINS &#226;&#8364;&#162; WOODWARD ET AL.                                    681 <br />        The resuspended extract underlayered 100 &#206;&#188;l NIB containing 30 &#206;&#188;M            This work supported Cancer Research UK (CR-UK) grants <br />       aphidicolin 10% (wt/vol) sucrose spun 2,000 g swinging        C303/A3135 STU063/001 J.J. Blow CR-UK grant CDA A5991 <br />       bucket rotor 5 min 4&#194;&#176;C. The supernatant removed leave      A. Gartner. <br />       sucrose cushion, cushion washed twice                The authors &#239;&#172;?nancial con&#239;&#172;&#8218;icts work. <br />       100 &#206;&#188;l NIB plus 30 &#206;&#188;M aphidicolin. The cushion was <br />       carefully removed leave 40 &#206;&#188;l, resuspended 400 &#206;&#188;l             Submitted: 17 February 2006 <br />       ice-cold PBS plus 0.1% (vol/vol) Triton X-100. This underlayered with <br />                                                                                     Accepted: 3 May 2006 <br />       PBS plus 10% sucrose spun like resuspended extract. The super- <br />       natant removed cushion washed twice with <br />       100 &#206;&#188;l PBS. The cushion carefully removed leave pelleted <br />       nuclei, gently resuspended 50 &#206;&#188;l PBS freezing         References <br />       dry ice. <br />              DNA spread glass slides according previously de-         Alexandrow, M.G., M. Ritzi, A. Pemov, J.L. Hamlin. 2002. A potential <br />       scribed standard conditions (Blow et al., 2001). Slides rehydrated               role mini-chromosome maintenance (MCM) proteins initia- <br />       HPLC pure water incubated 2 h blocking solution contain-              tion dihydrofolate reductase replication origin. J. Biol. Chem. <br />       ing PBS, 1% (wt/vol) BSA, 0.05% (vol/vol) TWEEN 20. Slides                   277:2702&#226;&#8364;&#8220;2708. <br />       incubated anti-biotin (B7653; Sigma-Aldrich) 20 &#206;&#188;g/ml         Anglana, M., F. Apiou, A. Bensimon, M. Debatisse. 2003. Dynamics of <br />       buffer 2 h, washed extensively (10 changes buffer 2 h),               DNA replication mammalian somatic cells: nucleotide pool modulates <br />                                                                                             origin choice interorigin spacing. Cell. 114:385&#226;&#8364;&#8220;394. <br />       labeled 1 &#206;&#188;g/ml cy3-conjugated anti&#226;&#8364;&#8220;mouse antibody (715-165-150; <br />       Jackson ImmunoResearch Laboratories) buffer 2 h. Samples      Aparicio, O.M., D.M. Weinstein, S.P. Bell. 1997. Components dynamics <br />       washed extensively (10 changes buffer 2 h),                     DNA replication complexes S. cerevisiae: redistribution MCM <br />                                                                                             proteins Cdc45p S phase. Cell. 91:59&#226;&#8364;&#8220;69. <br />       washed twice PBS DNA stained YOYO (Y-3601 diluted at <br />       1:10,000 1-mM stock; Invitrogen) 10 min. Samples              Aparicio, O.M., A.M. Stout, S.P. Bell. 1999. Differential assembly Cdc45p <br />       rinsed &#239;&#172;?ve times PBS mounted Vectashield. Images cap-                  DNA polymerases early late origins DNA replication. Proc. <br />                                                                                             Natl. Acad. Sci. USA. 96:9130&#226;&#8364;&#8220;9135. <br />       tured using microscope (LSM510-META; Carl Zeiss MicroImaging, Inc.) <br />       measurements using associated LSM software.                 Blow, J.J., A. Dutta. 2005. Preventing re-replication chromosomal DNA. <br />       Random &#239;&#172;?elds selected using YOYO staining ensure                    Nat. Rev. Mol. Cell Biol. 6:476&#226;&#8364;&#8220;486. <br />       single DNA &#239;&#172;?bers, &#239;&#172;?ber bundles, scored. Fields rich           Blow, J.J., P.J. Gillespie, D. Francis, D.A. Jackson. 2001. Replication origins <br />       unambiguous single &#239;&#172;?bers recorded using 100&#195;&#8212;, NA 1.4, lens                 Xenopus egg extract 5&#226;&#8364;&#8220;15 kilobases apart activated clus- <br />       optimal resolution sensitivity. For sample minimum 200                  ters different times. J. Cell Biol. 152:15&#226;&#8364;&#8220;25. <br />       measurements made.                                                        Burkhart, R., D. Schulte, D. Hu, C. Musahl, F. Gohring, R. Knippers. 1995. <br />                                                                                             Interactions human nuclear proteins P1Mcm3 P1Cdc46. Eur. J. <br />       Pulse-chase experiments                                                               Biochem. 228:431&#226;&#8364;&#8220;438. <br />       Pulse-chase experiments performed incubating X. laevis sperm          Callan, H.G. 1972. Replication DNA chromosomes eukaryotes. Proc. <br />       DNA interphase egg extract, inhibitors added appropriate.              R. Soc. Lond. B. Biol. Sci. 181:19&#226;&#8364;&#8220;41. <br />       At beginning S phase (the exact timing determined indepen-       Chong, J.P., P. Th&#195;&#182;mmes, A. Rowles, H.M. Mahbubani, J.J. Blow. 1997. <br />       dently extract), extract supplemented 0.2 mCi/ml                Characterization Xenopus replication licensing system. Methods <br />       &#206;&#177;-[32P]dATP (from stock 10 mCi/ml 3,000 Ci/mmol; GE Health-                  Enzymol. 283:549&#226;&#8364;&#8220;564. <br />       care). After 2-min pulse (5 min samples containing aphidicolin),    Cortez, D., G. Glick, S.J. Elledge. 2004. Minichromosome maintenance pro- <br />       radioactive nucleotide chased addition 2.5 mM unlabeled                     teins direct targets ATM ATR checkpoint kinases. Proc. <br />       Mg-dATP. Aliquots (10 &#206;&#188;l) subsequently removed sample            Natl. Acad. Sci. USA. 101:10078&#226;&#8364;&#8220;10083. <br />       desired time points, reactions stopped, DNA processed        Danis, E., K. Brodolin, S. Menut, D. Maiorano, C. Girard-Reydet, M. <br />       electrophoresis 0.7% agarose gels, previously described (Luciani            Mechali. 2004. Specification DNA replication origin transcrip- <br />       et al., 2004). HindIII-digested &#206;&#187;-phage DNA (New England Biolabs)                 tion complex. Nat. Cell Biol. 6:721&#226;&#8364;&#8220;730. <br />       end-labeled &#206;&#177;-[32P]dATP loaded comparison (12.5 ng           DePamphilis, M.L. 1999. Replication origins metazoan chromosomes: fact or <br />       lane). Alkaline agarose gels exposed autoradiography &#239;&#172;?lm                      fiction? Bioessays. 21:5&#226;&#8364;&#8220;16. <br />       (Kodak), scanned using scanner (CanoScan 9900F; Canon)            Diffley, J.F. 2004. Regulation early events chromosome replication. Curr. <br />       Photoshop software (Adobe). Lane densities analyzed using                    Biol. 14:R778&#226;&#8364;&#8220;R786. <br />       GelEval software (FrogDance Software).                                        Dimitrova, D.S., I.T. Todorov, T. Melendy, D.M. Gilbert. 1999. Mcm2, but <br />                                                                                             RPA, component mammalian early G1-phase prereplication <br />       C. elegans work                                                                       complex. J. Cell Biol. 146:709&#226;&#8364;&#8220;722. <br />       C. elegans RNAi feeding performed previously described, using          Dimitrova, D.S., T.A. Prokhorova, J.J. Blow, I.T. Todorov, D.M. Gilbert. <br />       MCM5 (R10E4.4), MCM6 (ZK632.1), MCM7 (F32D1.10)                          2002. Mammalian nuclei licensed DNA replication during <br />       genome-wide RNAi library (Fraser et al., 2000; Kamath et al., 2003).                  late telophase. J. Cell Sci. 115:51&#226;&#8364;&#8220;59. <br />       MCM RNAi-expressing bacteria titrated bacteria expressing           Donovan, S., J. Harwood, L.S. Drury, J.F. Diffley. 1997. Cdc6p-dependent <br />       RNAi GFP. Three P0 worms placed RNAi plates containing                loading Mcm proteins pre-replicative chromatin budding yeast. <br />       100 &#206;&#188;g/ml ampicillin, 5 mM IPTG, indicated concentration HU                Proc. Natl. Acad. Sci. USA. 94:5611&#226;&#8364;&#8220;5616. <br />       L4 larval stage, resulting adult worms taken    Edwards, M.C., A.V. Tutter, C. Cvetic, C.H. Gilbert, T.A. Prokhorova, J.C. <br />       plates 40 h later. After 4 d, number adult F1 worms scored,            Walter. 2002. MCM2-7 complexes bind chromatin distributed pat- <br />       sterile worms determined absence eggs body. The               tern surrounding origin recognition complex Xenopus egg extracts. <br />       number viability F2 animals scored 2&#226;&#8364;&#8220;3 d later. At stage,             J. Biol. Chem. 277:33049&#226;&#8364;&#8220;33057. <br />       wild-type conditions, plates starved exces-     Fraser, A.G., R.S. Kamath, P. Zipperlen, M. Martinez-Campos, M. Sohrmann, <br />       sive F2 larvae. Plates viewed microscope (Stemi SV11; Carl                J. Ahringer. 2000. Functional genomic analysis C. elegans chro- <br />       Zeiss MicroImaging, Inc.) using 20&#195;&#8212; objective. Representative images                mosome I systematic <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span>RNA interference</span>. Nature. 408:325&#226;&#8364;&#8220;330. <br />       captured using Openlab 3.1.2 software (Improvision) exported         Gilbert, D.M. 2001. Making sense eukaryotic DNA replication origins. <br />       tiff &#239;&#172;?les.                                                                         Science. 294:96&#226;&#8364;&#8220;100. <br />                                                                                     Gillespie, P.J., A. Li, J.J. Blow. 2001. Reconstitution licensed replication <br />       Online supplemental material                                                          origins Xenopus sperm nuclei using purified proteins. BMC Biochem. <br />       Tables S1&#226;&#8364;&#8220;S3 number C. elegans adult F1 worms observed                 2:15. <br />       different HU concentrations different amounts siRNA directed   Griffiths, T.D., S.Y. Ling. 1985. Effect ultraviolet light thymidine <br />       MCM7 (Table S1), MCM5 (Table S2), MCM6 (Table S3). Online                         incorporation, DNA chain elongation replicon initiation wild-type <br />       supplemental material available http://www.jcb.org/cgi/content/                 excision-deficient Chinese hamster ovary cells. Biochim. Biophys. <br />       full/jcb.200602108/DC1.                                                               Acta. 826:121&#226;&#8364;&#8220;128. <br />                                                                                     Griffiths, T.D., S.Y. Ling. 1987. Activation alternative sites replicon <br />       Thanks Andrew Ferenbach recombinant geminin, Neil Perkins,                 initiation Chinese hamster cells exposed ultraviolet light. Mutat. <br />       Anatoliy Li, Margret Michalski-Blow comments manuscript.               Res. 184:39&#226;&#8364;&#8220;46. <br />  <br />  <br /> 682   JCB &#226;&#8364;&#162; VOLUME 173 &#226;&#8364;&#162; NUMBER 5 &#226;&#8364;&#162; 2006 <br />  Griffiths, T.D., S.Y. Ling. 1989. Effects UV light DNA chain growth       Ritzi, M., M. Baack, C. Musahl, P. Romanowski, R.A. Laskey, R. Knippers. <br />         replicon initiation human cells. Mutat. Res. 218:87&#226;&#8364;&#8220;94.                     1998. Human minichromosome maintenance proteins human <br /> Harvey, K.J., J. Newport. 2003. CpG methylation DNA restricts pre-                  origin recognition complex 2 protein chromatin. J. Biol. Chem. <br />         replication complex assembly Xenopus egg extracts. Mol. Cell. Biol.             273:24543&#226;&#8364;&#8220;24549. <br />         23:6769&#226;&#8364;&#8220;6779.                                                               Rowles, A., J.P. Chong, L. Brown, M. Howell, G.I. Evan, J.J. Blow. 1996. <br /> Hekmat-Nejad, M., Z. You, M.C. Yee, J.W. Newport, K.A. Cimprich. 2000.                 Interaction origin recognition complex replication <br />         Xenopus ATR replication-dependent chromatin-binding protein re-               licensing Xenopus. Cell. 87:287&#226;&#8364;&#8220;296. <br />         quired DNA replication checkpoint. Curr. Biol. 10:1565&#226;&#8364;&#8220;1573.        Rowles, A., S. Tada, J.J. Blow. 1999. Changes association Xenopus <br /> Herrick, J., P. Stanislawski, O. Hyrien, A. Bensimon. 2000. Replication fork           origin recognition complex chromatin licensing replication <br />         density increases DNA synthesis X. laevis egg extracts. J. Mol.          origins. J. Cell Sci. 112:2011&#226;&#8364;&#8220;2018. <br />         Biol. 300:1133&#226;&#8364;&#8220;1142.                                                        Shechter, D., V. Costanzo, J. Gautier. 2004. ATR ATM regulate the <br /> Hopwood, B., S. Dalton. 1996. Cdc45p assembles complex                     timing DNA replication origin firing. Nat. Cell Biol. 6:648&#226;&#8364;&#8220;655. <br />         Cdc46p/Mcm5p, required minichromosome maintenance,            Strausfeld, U.P., M. Howell, R. Rempel, J.L. Maller, T. Hunt, J.J. Blow. <br />         essential chromosomal DNA replication. Proc. Natl. Acad. Sci. USA.             1994. Cip1 blocks initiation DNA replication Xenopus extracts <br />         93:12309&#226;&#8364;&#8220;12314.                                                                    inhibition cyclin-dependent kinases. Curr. Biol. 4:876&#226;&#8364;&#8220;883. <br /> Hua, X.H., J. Newport. 1998. Identification preinitiation step DNA      Tada, S., A. Li, D. Maiorano, M. Mechali, J.J. Blow. 2001. Repression of <br />         replication independent origin recognition complex cdc6,            origin assembly metaphase depends inhibition RLF-B/Cdt1 by <br />         dependent cdk2. J. Cell Biol. 140:271&#226;&#8364;&#8220;281 <br /> </body></html>