<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Molecular Biology Cell <br /> Vol. 14, 4230 &#226;&#8364;&#8220; 4237, October 2003 <br />  <br />  <br /> Nuclear Pore Protein gp210 Is Essential Viability in <br /> HeLa Cells Caenorhabditis elegans <br /> Merav Cohen,* Naomi Feinstein,* Katherine L. Wilson,&#226;&#8364;&#160; and <br /> Yosef Gruenbaum*&#226;&#8364;&#161; <br />  <br /> *Department Genetics, The Institute Life Sciences, The Hebrew University Jerusalem, <br /> Jerusalem, 91904 Israel; &#226;&#8364;&#160;Department Cell Biology, The Johns Hopkins University School of <br /> Medicine, Baltimore Maryland 21205 <br />  <br /> Submitted April 27, 2003; Revised June 9, 2003; Accepted June 9, 2003 <br /> Monitoring Editor: Joseph Gall <br />  <br />  <br />  <br /> Gp210 evolutionarily conserved membrane protein nuclear pore complex (NPC). We studied phenotypes <br /> produced RNAi-induced downregulation gp210 human (HeLa) cells Caenorhabditis elegans embryos. <br /> HeLa cell viability requires Gp210 activity. The dying cells accumulated clustered NPCs aberrant membrane <br /> structures nuclear envelope, suggesting gp210 required directly indirectly nuclear pore formation and <br /> dilation anchoring structural integrity mature NPCs. Essential roles gp210 confirmed C. <br /> elegans, RNAi-induced reduction gp210 caused embryonic lethality. The nuclear envelopes embryos with <br /> reduced gp210 aberrant nuclear membrane structures clustered NPCs, confirming gp210 plays critical <br /> roles nuclear membrane mechanisms conserved nematodes humans. <br />  <br />  <br /> INTRODUCTION                                                            handed polyproline type II (PII) helical structure, is <br />                                                                         implicated protein-protein interactions (Pilpel et al., 2003). <br /> In eukaryotes, nuclear membranes separate the <br />                                                                         Several previous studies addressed ultrastructural <br /> chromosomes cytoplasm. Communication de- <br />                                                                         location gp210 mature NPCs, targeting nuclear <br /> pends nuclear pore complexes (NPCs), po- <br />                                                                         pore membranes, cell cycle dynamics (reviewed by <br /> sitioned sites fusion inner outer <br />                                                                         Bodoor et al., 1999). The hypothesis gp210 be <br /> membranes. NPCs protein complexes esti- <br />                                                                         required form nuclear pores recently tested using <br /> mated mass 60 MDa yeast 125 MDa verte- <br />                                                                         cell-free extracts Xenopus eggs (Drummond Wilson, <br /> brates (reviewed Pante Aebi, 1996; Vasu and <br /> Forbes, 2001). Each NPC consists multiple copies                  2002). These experiments focused function the <br /> nearly 30 distinct proteins (Rout et al., 2000; Cronshaw et             exposed tail gp210, accessible reagents. This <br /> al., 2002) termed nucleoporins, collectively main-                study showed addition competing gp210 tail <br /> tain pore structure mediate molecular transport be-                 polypeptides blocked growth nascent nuclei, <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> <br /> tween nucleoplasm cytoplasm (reviewed Gorlich                    dominant inhibition pore formation. In arrested <br /> and Kutay, 1999; Macara, 2001; Lei Silver, 2002).                   nuclei, apparently fused undilated &#226;&#8364;&#339;mini-pores&#226;&#8364;? accu- <br />    Of &#226;&#172;&#402;30 distinct nucleoporins identified compre-             mulated nuclear envelope. Antibodies the <br /> hensive proteomic analysis vertebrate NPCs,                 exposed gp210 tail inhibited pore formation; in <br /> (gp210 Pom121) integral membrane proteins (Cron-                case, arrested intermediates electron-dense <br /> shaw et al., 2002). Of two, gp210 detectable           determine membranes actually fused. Under <br /> both animals plants, suggesting function               conditions, arrested nuclei accumulated <br /> been conserved evolution (Cohen et al., 2001). Very little           &#226;&#8364;&#339;twinned&#226;&#8364;? membrane structures, characterized short re- <br /> of gp210 exposed cytoplasm. There short                  gions close apposition inner outer membranes <br /> (&#226;&#172;&#402;60-residue) C-terminal tail, exposed faces               (Drummond Wilson, 2002). These results suggested that <br /> the NPC; gp210 crosses membrane once,                 gp210 tail mediates dilation nascent pores and <br /> of mass (&#226;&#172;&#402;1750 N-terminal residues) positioned            hinted gp210 directly mediate &#226;&#8364;&#339;porogenic&#226;&#8364;? <br /> nuclear envelope lumen (Greber et al., 1990). Gp210 forms               membrane fusion. These experiments did address the <br /> stable dimers, possibly higher-order oligomers (Favreau             function lumenal domain gp210. <br /> et al., 2001). The lumenal domain rat gp210 includes                To understand function gp210, determined its <br /> amphiphilic helix, potentially similar &#226;&#8364;&#339;fusion&#226;&#8364;?             downregulation phenotypes human cultured cells <br /> peptides viral proteins stimulate membrane fusion               Caenorhabditis elegans embryos. This strategy elimi- <br /> (Greber et al., 1990) C-terminal domain left-             nates entire gp210 protein vivo, domi- <br />                                                                         nantly inhibiting exposed domain vitro. We report <br />  Article published online ahead print. Mol. Biol. Cell 10.1091/ <br />                                                                         gp210 essential protein human and <br /> mbc.E03&#226;&#8364;&#8220; 04 &#226;&#8364;&#8220; 0260. Article publication date available       C. elegans cells, appears required directly or <br /> www.molbiolcell.org/cgi/doi/10.1091/mbc.E03&#226;&#8364;&#8220; 04 &#226;&#8364;&#8220; 0260.                 indirectly pore formation stability mature <br />  &#226;&#8364;&#160; <br />    Corresponding author. E-mail address: gru@vms.huji.ac.il.            NPCs. <br />  <br />  <br /> 4230                                                                                     &#194;&#169; 2003 The American Society Cell Biology <br />                                                                                                                                  gp210 Is Essential Viability <br />  <br />  <br /> MATERIALS AND METHODS                                                                1 h Cy3-conjugated secondary antibodies diluted 1:200 PBS, and <br />                                                                                      washed PBS times (10 min each). Slides incubated 1 h with <br /> Strains, Antibodies, Materials                                                   antibodies lamin A/C, lamin B2, emerin, FG-repeat nucleoporins <br />                                                                                      diluted 1:200, 1:200, 1:100, 1:50, respectively, PBSM. Slides were <br /> C. elegans Bristol N2 animals maintained standard conditions as <br />                                                                                      washed described incubated 1 h appropriate FITC-conjugated <br /> described (Brenner, 1974). Rabbit polyclonal antibodies rod and <br />                                                                                      secondary antibody diluted 1:200 PBS, washed PBS 10 min, <br /> tail domains Ce-lamin rabbit polyclonal antibodies human <br />                                                                                      incubated 10 min 10 &#226;?&#174;g/ml DAPI, washed PBS. Slides were <br /> emerin described previously (Liu et al., 2000; Lee et al., 2001). To generate <br />                                                                                      mounted glycerol containing 2% N-propyl-gallate. <br /> polyclonal antibodies Ce-Gp210, rats immunized 3-week <br /> intervals antigens. Rat 3783 immunized syn- <br /> thetic peptide conjugated keyhole limpet hemocyanin (KLH). This peptide,          SDS-PAGE Immunoblotting <br /> NH2-CDKTGSFGDNTLNNTTH-COOH (Gp210-C peptide), comprised Ce-                          Mixed stage N2 animals recovered plates washing 1 ml PBS, <br /> Gp210 residues 1777&#226;&#8364;&#8220;1792 plus N-terminal cysteine. Peptide synthesis,             transferred 1.5-ml Eppendorf tube, mixed 1 ml 2&#226;&#171;&#187; sample loading <br /> purification reverse-phase HPLC use C18 analytical column,          buffer (25 mM Tris-HCl, pH 6.8, 20% glycerol, 0.2 M &#226;?&#164;-mercaptoethanol, 4% <br /> and conjugation KLH Boston Biomolecules (Woburn, MA).                SDS, 0.001% bromophenol blue), frozen &#226;&#171;&#186;70&#194;&#176;C 30 min then <br /> Rat 3586 immunized gel-purified recombinant polypeptide corre-            boiled 5 min loading SDS-PAGE. Proteins separated on <br /> sponding exon 3 Ce-gp210 fused GST. Immunizations serum                 5&#226;&#8364;&#8220;17% SDS-PAGE gradient gels transferred nitrocellulose membranes <br /> production Covance Research Products (Denver, PA). mAb 414,             buffer lacking SDS facilitate transfer high-molecular-weight pro- <br /> which recognizes FG-repeat nucleoporins (nup358, nup214/CAN,                    teins. Nonspecific binding blocked PBST containing 5% nonfat dry <br /> nup153, nup98, p62), purchased BABCO (Richmond, CA).                    milk overnight probed antibodies Ce-Gp210 (sera 3783 or <br /> Anti-human gp210 antibodies kindly provided Dr. J.C. Courvalin               3586, diluted 1:500), PBST-milk 2 h 22&#226;&#8364;&#8220;24&#194;&#176;C. After washing three <br /> (Courvalin et al., 1990). Rabbit polyclonal antibodies human lamins          times PBST, blots incubated 1 h 22&#226;&#8364;&#8220;24&#194;&#176;C horseradish peroxi- <br /> A/C kindly provided Dr. K. Weber (Harborth et al., 2001). Polyclonal         dase&#226;&#8364;&#8220; conjugated secondary antibodies. Blots washed described <br /> antibodies specific lamin B2 kindly provided Dr. G. Krohne               visualized enhanced chemiluminescence (Amersham Pharma- <br /> (Wurzburg). All secondary antibodies purchased Jackson Labora-             cia Biotech). <br /> tories (West Grove, PA). Epon purchased Agar Scientific (Stansted, <br /> Essex, UK). <br />                                                                                      Sample Preparation TEM Analysis <br /> dsRNA-mediated Interference Templates C. elegans                              C. elegans embryo samples prepared exactly described (Cohen et al., <br /> HeLa Cells                                                                           2002). HeLa cells grown sterile chamber slide cover (Nunc, <br />                                                                                      Naperville, IL) 3 d siRNA transfection fixed 2.5% glutar- <br /> Full-<span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span> Ce-lamin cDNA (Liu et al., 2000) DNA fragments correspond-           aldehyde plus 3% formaldehyde 0.1 M sodium cacodylate (pH 7.4) 1 h <br /> ing Ce-gp210 exons 3&#226;&#8364;&#8220;5 (including introns) Ce-gp210 exons 8 &#226;&#8364;&#8220;15 (with-         room temperature. Samples washed times (10 min each) 0.1 M <br /> out introns; kindly provided Dr. Y. Kohara, Japan) subcloned            sodium cacodylate (pH 7.4) incubated 1 h room temperature in <br /> feeding vector L4440 (Timmons Fire, 1998) used RNAi feeding           osmium tetroxide solution (1% OsO4, 1.5% K3Fe(CN)6, 0.1 M sodium caco- <br /> experiments. The Ce-gp210 fragments subcloned pBlueScript             dylate, pH 7.4). Samples washed times (10 min each) in <br /> KS used templates prepare dsRNA gp210(RNAi) soaking exper-             sodium cacodylate pH 7.4, dehydrated sequential incubation in <br /> iments. The dsRNA oligonucleotides used inhibit human gp210 gene              30%, 50%, 70%, 90%, 95% ethanol (10 min each), followed washes <br /> (5&#226;&#172;&#732;-AAUGCGUGCGAAUCCTCACUdTdT-3&#226;&#172;&#732;) corresponded nucleotides                         (30 min each) 100% ethanol. These dehydrated samples (a) <br /> 523&#226;&#8364;&#8220;543 human gp210 coding sequence (accession number: BC020573).             incubated 2 h room temperature 25% Epon ethanol, (b) incubated <br /> These oligos designed described (Elbashir et al., 2002) purchased        overnight 50% Epon ethanol, (c) incubated 75% Epon ethanol for <br /> <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> Dharmacon (Lafayette, CO) deprotected desalted form. As                8 h, (d) incubated overnight 100% Epon, finally incubated 8 h in <br /> control used nonspecific duplex firefly luciferase Photinus pyralis     fresh 100% Epon. Epon block polymerized 2 d 60&#194;&#176;C dry oven <br /> (GL2) final concentrations gp210 RNA duplex.                      sectioned 80 &#226;&#8364;&#8220;90-nm sections, using Diatome diamond <br />                                                                                      knife. Sections picked 200-mesh bar copper grids, stained <br /> RNA-mediated Inhibition (RNAi) C. elegans                                         uranyl acetate lead citrate, viewed electron microscope <br /> Feeding experiments using optimal protocol (Kamath et al.,              (Philips Tecnai 12; www.feic.com/tecnai/advanc.htm) equipped Mega- <br /> 2001), 1 mM isopropyl-beta-d-thiogalactoside. Residual protein               View II CCD camera, analySIS version 3.0 software (Soft Imaging System <br /> gp210(RNAi) embryos detected indirect immunofluorescence de-               GmbH, Mu&#194;&#168;nster, Germany). <br /> scribed (Liu et al., 2000). For soaking experiments, single-strand RNA was <br /> transcribed vitro using RiboMAX (Promega, Madison WI), according           FACS Analysis Cell Sorting <br /> manufacturer protocol. RNA phenol-extracted twice, ethanol-precipi- <br />                                                                                      HeLa cells transfected siRNA targeting gp210 harvested after <br /> tated, resuspended soaking buffer (10.9 mM Na2HPO4, 5.5 mM <br />                                                                                      5 d. Harvested cells washed sterile PBS, fixed 100% methanol <br /> KH2PO4, 2.1 mM NaCl, 4.7 mM NH4Cl, 3 mM spermidine, 0.05% gelatin). <br /> The complementary strands RNA annealed incubating 15 min              overnight 4&#194;&#176;C, washed PBS, incubated 100 &#226;?&#174;l 100 &#226;?&#174;g/ml RNAse <br />                                                                                      50 &#226;?&#174;g/ml propidium iodide (Sigma, Dorset, United Kingdom), incubated <br /> at 68&#194;&#176;C, followed 30 min 37&#194;&#176;C, transferring worms the <br />                                                                                      ice 10 min, stored 4&#194;&#176;C FACS analysis. FACS analysis <br /> solution. Soaking described (Maeda et al., 2001). <br />                                                                                      performed FACSCalibur (Becton-Dickinson, San Jose, CA). <br />                                                                                      The left edge control cell profile taken border separating <br /> Cell Culture siRNA Transfection                                                  normal, diploid cells (to right) apoptotic, hypodiploid cells (to the <br /> Cultured HeLa cells transfected siRNA described (Elbashir et al.,       left). The percentage apoptotic cells calculated ratio events on <br /> 2002) minor changes. In short, human HeLa SS6 cells grown            left events population described (Benhar et al., <br /> at 37&#194;&#176;C DMEM supplemented 10% FCS, penicillin, streptomycin.             2001). <br /> Cells passed regularly maintain exponential growth. The day before <br /> transfection, cells trypsinized, diluted fresh medium an-          Nuclear Import Assays <br /> tibiotics, transferred 24-well plates (532 &#226;?&#174;l well). Transient trans- <br /> fection siRNAs using TransIT-TKO Transfection Reagent (Mirus             HeLa SS6 cells cultivated coverslips 3 d siRNA transfection <br /> Corporation, Madison, WI). For well, preincubated 50 &#226;?&#174;l OPTI-          treated 33 &#226;?&#174;g/ml digitonin 4.5 min. As transport <br /> MEM 1 medium (Gibco, Grant Island, NY) plus 3 &#226;?&#174;l TransIT-TKO 5 min               substrate, assayed BSA conjugated rhodamine nuclear <br /> at room temperature. Separately, 12 &#226;?&#174;l OPTIMEM 1 medium mixed               localization signal peptide (NLS-BSA) described (Broder et al., 1997). Trans- <br /> 3 &#226;?&#174;l siRNA duplex. The mixtures combined incubated 15 min               location fluorescent NLS-BSA conjugates nuclei digitonin-perme- <br /> at room temperature allow complex formation final concentration          abilized HeLa cells assayed confocal microscopy. <br /> 100 nM siRNA. <br />                                                                                      Cell Growth <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span>Survival</span> <br /> Indirect Immunofluorescence                                                          Two days siRNA treatment, cells harvested, counted replated <br /> Unless stated, steps room temperature. Staining        equal numbers triplicates 24-well plates, left grow described <br /> C. elegans indirect immunofluorescence described previously (Gruen-           above. Each day, triplicates fixed 10 min room temper- <br /> baum et al., 2002). Transfected HeLa SS6 cells grown glass coverslips          ature 0.5% glutaraldehyde (Merck, Frankfurt, Germany), washed three <br /> 24-well plates fixed methanol 6 min &#226;&#171;&#186;20&#194;&#176;C, washed PBS,           times, 10 s each, water, 1 min 0.1 M borate (pH 8.5) then <br /> blocked 10% milk PBS 1 h. Cells incubated 1 h gp210              incubated 1 h 1% methylene blue (Sigma). Wells washed in <br /> antibodies (Courvalin et al., 1990) diluted 1:100 PBS containing 10% milk         running distilled water till washing water ran clear completely air-dried. <br /> (PBSM). Slides washed times (10 min each) PBS, incubated          The stain extracted 0.1 N HCl 1 h 37&#194;&#176;C. Absorbance read in <br />  <br />  <br />  <br /> Vol. 14, October 2003                                                                                                                                           4231 <br />  M. Cohen et al. <br />  <br />  <br />  <br />  <br /> Figure 1. Phenotypes HeLa cells reduced gp210. (A) Transfection HeLa cells dsRNA specific human gp210 gene <br /> (gp210-siRNA) caused significantly reduced expression gp210 relative control cells transfected nonspecific dsRNA (GL2-siRNA). <br /> Cells double-stained gp210 (Courvalin et al., 1990) indirect immunofluorescence DAPI visualize DNA imaged by <br /> fluorescence microscopy. (B) The growth rates gp210-siRNA untreated HeLa cells quantified colorimetric assay (see <br /> MATERIALS AND METHODS). The graph shows ratio OD(630) measurements gp210-siRNA cells vs. untreated cells different <br /> time points. The exponential regression monotonously descending (with negative exponential coefficient) showing growth rate <br /> of gp210-siRNA cells lower untreated cells times. (C) Downregulation gp210 disrupts FG-repeat nucleoporins. <br /> Gp210-siRNA cells control GL2-siRNA cells double-stained human gp210 FG-repeat nucleoporins (mAb414) visualized <br /> by laser confocal microscopy. The exposure times shown immunostaining identical set panels A C. (D) <br /> Thin-section transmission electron micrographs GL2-siRNA control cells (a b) gp210-siRNA cells (c&#226;&#8364;&#8220;f), 3 d siRNA treatment. <br /> Cells reduced gp210 condensed chromatin (c), clustering mature NPCs (d), enlarged nuclear envelope lumenal space (e), and <br /> clusters pore-related membrane structures (f). Arrows D depict mature NPCs (b d), membrane blebs (e), various arrested pore <br /> structures (f), named &#226;&#8364;&#339;twinned membranes&#226;&#8364;? (t) &#226;&#8364;&#339;fusion-arrested&#226;&#8364;? membranes (f), described text. Bars: 30 &#226;?&#174;m A, 20 <br /> &#226;?&#174;m C, 200 nm D (b, d&#226;&#8364;&#8220;f), 2000 nm D (a c). <br />  <br />  <br />  <br /> an enzyme-linked immunosorbent assay (ELISA) reader 630 nm de-   gp210 (Figure 1A). Survival assays showed gp210 <br /> scribed (Shir Levitzki, 2002).                                     siRNA treatment reduced cell proliferation lethal to <br />                                                                        cells 5 d (Figure 1B). Likewise, propidium io- <br /> RESULTS                                                                dide staining FACS analysis performed 5 d trans- <br />                                                                        fection showed 60% gp210 siRNA- <br /> Gp210 Is Essential HeLa Cells                                       treated cells dead, compared 6% untreated cells <br /> The term &#226;&#8364;&#339;pore&#226;&#8364;? refer specifically membrane                (unpublished data). The dying cells sub-G1 suggest- <br /> structure (hole), proteinaceous NPC assembled                  ing apoptosis, remaining cells divided nor- <br /> therein. To determine downregulation <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span>phenotype</span>                 mally G1 G2 phases. We analyzed the <br /> gp210 vivo, used siRNA technique (Elbashir et          cultures double staining gp210 antibodies and <br /> al., 2002) reduce gp210 expression human tissue culture          trypan blue dye, stains dead cells. This analysis <br /> (HeLa) cells. Cells transfected 21-<span id='am-238' about='xsp:base' typeof='owl:Thing'>base</span> pair               showed 91% (n &#226;&#171;&#189; 96) trypan blue&#226;&#8364;&#8220;positive cells the <br /> dsRNA corresponding nucleotides 523&#226;&#8364;&#8220;543 human                gp210-siRNA culture negative gp210, re- <br /> gp210 coding sequence, 99.7% (n &#226;&#171;&#189; 1564)                   maining 9% significantly reduced levels gp210 stain- <br /> &#226;&#8364;&#339;gp210-siRNA&#226;&#8364;? cells severely reduced gp210 immu-                 ing. In contrast, trypan blue&#226;&#8364;&#8220;positive cells GL2- <br /> nofluorescence signal (Figure 1A). Control cells transfected           siRNA culture normal levels gp210 (unpublished <br /> with dsRNA corresponding luciferase gene (GL2-                  data). We concluded gp210 essential for <br /> siRNA cells) normal immunofluorescence signals                 viability dividing cells. <br />  <br />  <br /> 4232                                                                                                         Molecular Biology Cell <br />                                                                                                    gp210 Is Essential Viability <br />  <br />  <br />  <br />  <br /> Figure 2. Quantitation pore-related nu- <br /> clear membrane structures gp210-siRNA <br /> HeLa cells. We identified (A) quantified <br /> (B C) distinct morphological fea- <br /> tures including mature NPCs (m), fusion- <br /> arrested structures (f), twinned mem- <br /> branes (t) TEM micrographs gp210- <br /> siRNA cells control GL2-siRNA HeLa <br /> cells. The number counted structures for <br /> each category shown B, graphed in <br /> C percentage total structures <br /> counted gp210-siRNA control <br /> GL2-siRNA cells. Bars A, 50 nm. <br />  <br />  <br />  <br /> Gp210 Is Required Assembly Maintenance (or           subset rounded membrane profiles connected a <br /> Both) Nuclear Pore Complexes                                 dense &#226;&#8364;&#339;bridge&#226;&#8364;? &#226;&#172;&#402;25-nm diameter (unpublished data). We <br /> Indirect immunofluorescence staining showed nuclear        quantified structures mature NPCs TEM micro- <br /> envelopes reduced levels gp210 stained weakly      graphs gp210-siRNA control GL2-siRNA cells. Com- <br /> for FG repeat nucleoporins, detected using mAb 414              pared controls, gp210-siRNA cells fewer ma- <br /> (mAb414; Figure 1C). These results suggested loss       ture NPCs (50 vs. 85%) significantly fusion- <br /> gp210 direct downstream effects     arrested structures (32 vs. 5%) twinned membranes (18 <br /> assembly, recruitment stability FG-repeat nucleopor-      vs. 10%). The change distribution mature NPCs and <br /> ins, positioned NPC.                   pore-related structures highly significant (chi-square <br />    To visualize nuclear envelope morphology gp210-           test; p &#226;&#172;?&#226;&#172;? 0.00001). It interesting 50% NPCs <br /> siRNA cells, used thin-section transmission electron mi-     appeared normal 3 d gp210-siRNA treatment. We <br /> croscopy (TEM). Other cells samples          hypothesize cells depleted gp210, cooper- <br /> verified gp210-deficient immunofluorescence staining,     ative interactions remaining gp210 proteins might <br /> using gp210 antibodies (unpublished data). GL2-siRNA cells      allow subset NPCs survive. We speculate the <br /> served positive controls. Five days siRNA transfec-    aberrant nuclear membrane structures, comprising 50% of <br /> tion cells died, exhibiting typical apoptotic mor-     total pore-related structures, represent 1) mature <br /> phology including condensed chromatin, blebbed mem-             NPCs degenerated insufficient gp210 sub- <br /> branes, disrupted mitochondrial structure (unpublished      units 2) arrested intermediates formation new <br /> data). We analyzed gp210-siRNA cell cultures 3 d      pores. <br /> after transfection, cell growth viability re-        Three days treatment, gp210-siRNA cells a <br /> duced cells alive (Figure 1B) at-       significantly lower immunofluorescence signal lamins <br /> tached culture wells. The control (GL2-siRNA) nuclei     A/C nuclear envelope, similar higher lamin <br /> typical nuclear envelopes widely spaced mature NPC          A/C signal nucleoplasm (Figure 3Ab). The nuclear <br /> structures (Figure 1D, b). In contrast, gp210-siRNA   envelope signal lamin B2 slightly reduced, com- <br /> nuclei reproducibly aberrant phenotypes including ab-       pared matched GL2-siRNA control cells (Figure 3Aa). <br /> normally condensed abnormally distributed chromatin         No difference detected emerin, integral mem- <br /> (Figure 1Dc) clustered NPCs; 43% analyzed nuclei         brane protein localizes diffusion plane of <br /> (n &#226;&#171;&#189; 81) clusters NPCs          ER/nuclear membrane subsequently retained at <br /> located 250 nm nearest neighbor (Figure         inner nuclear membrane binding stable structures <br /> 1Dd), compared 0% (n &#226;&#171;&#189; 38) control GL2-siRNA            (Holmer Worman, 2001; Figure 3Ac). These results <br /> cells (Figure 1Db). Additional phenotypes included en-      hinted gp210 downregulation affect the <br /> largement lumenal space inner outer      efficiency lamin import nuclei lamin assembly. We <br /> membranes (Figure 1De) accumulation aberrant         tested nuclear import gp210-siRNA cells. <br /> pore-related membrane structures (Figures 1Df 2A).          Three days transfection, gp210- GL2-siRNA cells <br /> These aberrant structures, termed &#226;&#8364;&#339;twinned membranes&#226;&#8364;?           permeabilized digitonin incubated a <br /> and &#226;&#8364;&#339;fusion-arrested,&#226;&#8364;? similar caused dom-     fluorescence import-competent substrate (rhodamine-nuclear <br /> inant inhibition gp210 tail (Drummond Wilson,        localization signal peptide-BSA, referred NLS-BSA) and <br /> 2002), adapted similar nomenclature. The arrested         fixed analyzed confocal microscopy. All cells <br /> membrane structures shown higher magnification        remaining attached coverslips considered live cells. <br /> Figure 2A. Twinned membranes consisted short (&#226;&#172;&#402;40 nm)        In control GL2-siRNA cells, NLS-BSA substrate effi- <br /> regions inner outer membranes unusu-         ciently imported nuclei, seen enriched rhoda- <br /> ally close (Figure 2A, t). Most fusion-arrested membranes       signal nuclear envelope inside nucleus <br /> formed &#226;&#171;&#187; structure determine       (Figure 3B, middle). Other controls showed nuclear signal <br /> membrane fusion occurred (Figure 2A, f), minor        unpermeabilized GL2-siRNA cells, expected (Figure 3B, <br />  <br />  <br /> Vol. 14, October 2003                                                                                                        4233 <br />  M. Cohen et al. <br />  <br />  <br />  <br />  <br />                                                                                         Figure 3. Localization nuclear lamins <br />                                                                                         emerin gp210-siRNA cells. (A) Im- <br />                                                                                         munofluorescence GL2-siRNA and <br />                                                                                         gp210-siRNA cells double-stained gp210 <br />                                                                                         plus lamin B (a), lamin A/C (b), or <br />                                                                                         emerin (c). Images obtained laser <br />                                                                                         confocal microscopy. (B) Assay vitro <br />                                                                                         import rhodamine-labeled NLS-BSA into <br />                                                                                         nuclei digitonin-permeabilized gp210- <br />                                                                                         siRNA cells (left), GL2-siRNA cells (mid- <br />                                                                                         dle), unpermeabilized cells (right). Each <br />                                                                                         arrow indicates nucleus. Bars: 20 &#226;?&#174;m in <br />                                                                                         A 10 &#226;?&#174;m B. <br />  <br />  <br /> right), import assayed 4&#194;&#176;C blocked             Gp210). We previously identified gene encoding Ce-gp210 <br /> treatment wheat germ agglutinin (unpublished data). We         C. elegans chromosome I, position 1.02 (Cohen et al., <br /> then examined import gp210-siRNA cells, some-           2001). This gene, previously named npp-12 WormBase <br /> what surprised answer. The fluorescent substrate         (http://www.wormbase.org) encodes predicted protein of <br /> found predominantly cytoplasm nuclear envelope,          1847 residues predicted mass 209 kDa one <br /> with weak signal nucleoplasm (Figure 3B, left). The        predicted membrane-spanning region near C termini <br /> fluorescent signal strongest nuclear envelope, sug-      (Figure 4A), similar vertebrate gp210. Ce-Gp210 nine <br /> gesting fluorescent substrate bind NPCs,         potential sites N-linked glycosylation, plus predicted <br /> failed translocate efficiently. This result suggested re-   bacterial Ig-like Type 2 (Big2) domain. Interestingly, Big2 <br /> duced levels gp210 compromise import ac-          domains characteristic Escherichia coli intimins and <br /> tivity mature-appearing NPCs.                                    bacterial bacteriophage surface proteins (Kelly et <br />                                                                     al., 1999). To knowledge gp210 eukaryotic <br /> C. elegans gp210 Is Associated NPCs                            protein shares domain (Cohen et al., 2001). <br /> To determine essential roles gp210 con-            We raised rat polyclonal antibodies C-terminal <br /> served evolution, studied gp210 C. elegans (Ce-            peptide Ce-gp210 (serum 3783) recombinant <br />  <br />  <br />  <br />  <br />                                                                                         Figure 4. Gp210 C. elegans. (A) A sche- <br />                                                                                         matic diagram full-length Ce-Gp210 (res- <br />                                                                                         idues 1&#226;&#8364;&#8220;1847), drawn approximately to <br />                                                                                         scale, showing exons (numbered boxes), <br />                                                                                         single predicted transmembrane domain <br />                                                                                         (TM), bacterial Ig-like type 2 (Big2) <br />                                                                                         domain. Regions used antigen marked <br />                                                                                         orange lines labeled serum num- <br />                                                                                         ber (3586 3783). (B) Western blot of <br />                                                                                         crude protein extract mixed stage C. el- <br />                                                                                         egans probed preimmune (P) im- <br />                                                                                         mune (I) serum 3586. (C) Confocal images of <br />                                                                                         adult C. elegans gonad double-stained by <br />                                                                                         indirect immunofluorescence using Ce- <br />                                                                                         Gp210 serum 3783 (left panel), FG-repeat <br />                                                                                         nucleoporins (mAb414; NUPs; middle pan- <br />                                                                                         el), merged images (right panel). <br />  <br />  <br />  <br /> 4234                                                                                                  Molecular Biology Cell <br />                                                                                                       gp210 Is Essential Viability <br />  <br />  <br />  <br />  <br /> Figure 5. RNAi-induced downregulation <br /> phenotype Ce-Gp210. (A) Double immu- <br /> nostaining control embryos (left) Ce- <br /> gp210(RNAi) embryos (right; exons 8 &#226;&#8364;&#8220;15 were <br /> targeted), imaged fluorescence micros- <br /> copy. Embryos stained indirect im- <br /> munofluorescence using antibodies against <br /> Ce-Gp210 (serum 3783), Ce-lamin (Liu et al., <br /> 2000), FG-repeat nucleoporins (Nups; <br /> mAb414). Arrows indicate NPC clusters. (B) <br /> Thin-section TEMs control untreated em- <br /> bryos (a) Ce-gp210(RNAi) embryos in <br /> which exons 3&#226;&#8364;&#8220;5 targeted (b c). Ce- <br /> gp210(RNAi) embryos condensed <br /> chromatin (b), enlarged nuclear envelope <br /> lumen (b), clustered NPCs (c). NU, nu- <br /> cleus; Cyt, cytoplasm. Bars: 10 &#226;?&#174;m (A), 200 <br /> nm (B, c), 750 nm (B, b), 1500 nm (B, a). <br />  <br />  <br />  <br /> polypeptide comprising exon 3 (residues 102&#226;&#8364;&#8220;223) Ce-             ever, studies lacked controls specific loss Ce- <br /> Gp210 (serum 3586; MATERIALS AND METHODS). On                   Gp210 protein. To determine definitively RNAi-induced <br /> immunoblots C. elegans total protein lysates, sera          downregulation phenotype Gp210, fed, injected, or <br /> specifically recognized major band migrated 185           soaked nematodes dsRNA corresponding different <br /> kDa plus minor band 188 kDa (Figure 4B shows results           combinations Ce-Gp210 exons. By indirect immunofluo- <br /> for serum 3586). We know bands                   rescence, Ce-Gp210 protein reduced F1 embryos <br /> present, speculate differential glycosylation, alter-      treated dsRNA included exons 3&#226;&#8364;&#8220;5 (un- <br /> native splicing, proteolysis produce them. Recog-          published data) exons 8 &#226;&#8364;&#8220;15 (Figure 5A). Feeding and <br /> nition proteins specifically competed pre-           injection methods efficient soaking, <br /> treating antibodies peptide antigen (unpublished           produced 15% embryonic lethality comma <br /> data). The antipeptide serum 3783 gave stronger signals          stage. The remaining embryos hatched developed nor- <br /> immunofluorescence assays used following             mally fertile indistinguishable wild-type <br /> indirect immunofluorescence experiments. Ce-Gp210 local-            animals. All arrested gp210(RNAi) embryos little no <br /> ized nuclear rim C. elegans gonads punctate          detectable Ce-Gp210, animals escaped embry- <br /> <span id='am-239' about='xsp:pattern' typeof='owl:Thing'>pattern</span> characteristic NPCs, colocalized            onic lethality stained positive Ce-Gp210 (unpublished <br /> family FG-repeat nucleoporins recognized mAb                  data). We concluded RNAi method effective <br /> mAb414 (Davis Blobel, 1986; Figure 4C). C. elegans fed          embryos. Nevertheless, early lethality associated <br /> with lmn-1(RNAi) construct produced lamin-depleted              reduced Ce-Gp210 protein showed conclusively that <br /> embryos Ce-gp210 signal clustered               Ce-gp210 essential C. elegans. <br /> (unpublished data), consistent effect FG-repeat            To determine nucleoporins affected loss of <br /> nucleoporins (Liu et al., 2000).We concluded anti-         Ce-Gp210, Ce-gp210(RNAi) embryos double-stained by <br /> bodies specifically recognized endogenous Ce-Gp210 pro-             indirect immunofluorescence using antibodies en- <br /> teins NPC used analyze down-             dogenous Ce-Gp210 plus endogenous FG-repeat <br /> regulation phenotype Gp210 C. elegans.                        nucleoporins Ce-lamin. Ce-lamin localized normally in <br />                                                                     absence Ce-Gp210 (Figure 5A, gp210(RNAi)). Thus, <br /> Gp210 Is Essential Viability C. elegans Embryos,             subtle effect lamin B localization seen HeLa cells <br /> and Its Loss Produces Aberrant Nuclear Membrane                     detectable C. elegans. The main difference between <br /> Structures NPC Spacing Phenotypes                               C. elegans human cell phenotypes FG-repeat <br /> In previous studies, wild-type C. elegans fed       nucleoporins remained relatively abundant nuclear <br /> or soaked dsRNA corresponding various exons                envelopes gp210-reduced C. elegans cells. However, these <br /> Ce-gp210 (Fraser et al., 2000; Maeda et al., 2001). Fraser et al.   nucleoporins partially clustered seen indirect <br /> (2000) fed C. elegans dsRNA corresponding exons 5&#226;&#8364;&#8220;9         immunofluorescence staining (Figure 5A, arrows in <br /> and phenotypes. In contrast, Maeda et al. (2001)           gp210(RNAi)) higher resolution TEM (Figure 5Bc). <br /> used dsRNA soaking strategy targeted exons 8 &#226;&#8364;&#8220;15          Thus, NPC clustering consistent feature gp210- <br /> Ce-gp210; approach lethal 36% embryos            depletion worm human cells (Figure 1D). These <br /> and caused remaining animals arrest larvae. How-          results collectively suggested Gp210 directly indi- <br />  <br />  <br /> Vol. 14, October 2003                                                                                                           4235 <br />  M. Cohen et al. <br />  <br />  <br /> rectly required spacing, anchoring, structural         ruptions, accumulation aberrant nuclear membrane <br /> stability NPCs.                                                 structures NPC clustering. These similarities collec- <br />    To analyze nuclear ultrastructure C. elegans cells lacking   tively suggest loss gp210 cause direct and <br /> Ce-Gp210, collected F1 embryos produced               downstream effects, loss NPCs. <br /> after feeding dsRNA Ce-Gp210 (targeting ex- <br /> ons 3&#226;&#8364;&#8220;5). These F1 embryos fixed visualized            Aberrant Membrane Structures gp210-depleted Cells: <br /> thin-section TEM (see MATERIALS AND METHODS). Con-                 Parallels HA-mediated Fusion <br /> trol embryos obtained animals fed &#226;&#8364;&#339;empty&#226;&#8364;?           Our findings distinguish degenerating <br /> L4440 vector. Nuclei Ce-gp210(RNAi) embryos showed          preexisting NPCs arrested intermediates. However, the <br /> several aberrant phenotypes including abnormally con-              formation abnormal membrane structures support <br /> densed chromatin (Figure 5Bb), NPC clustering (Figure 5Bc)         models gp210 involved nuclear membrane <br /> and subtle enlargement nuclear envelope lumen (Fig-         fusion events. It interesting consider recent <br /> ure 5Bb). These phenotypes seen control em-            studies HA (hemagglutinin), fusogenic protein en- <br /> bryos (Figure 5Ba). The lethality associated nuclear       coded influenza virus, fusogenic membrane <br /> NPC phenotypes seen gp210-depleted C. elegans embryos           proteins (Skehel Wiley, 2000). Kozlov Cherno- <br /> were consistent phenotypes gp210-depleted          mordik (2002) proposed model account for <br /> HeLa cells, strongly suggesting gp210 functions          (hundreds thousands) molecules HA <br /> been conserved evolution.                                       required efficient membrane fusion pore <br />                                                                    dilation biologically significant diameters. In model, <br />                                                                    large numbers HA molecules aggregate laterally form <br /> DISCUSSION <br />                                                                    &#226;&#8364;&#339;coat&#226;&#8364;? cooperatively undergoes conformational <br /> The results study gp210 essential     change curved surface, pushes membranes <br /> viability dividing cells. Gp210 required nuclear        provides driving force membrane fu- <br /> pore assembly dilation anchoring             sion dilation. Fusion occur fewer HA <br /> mature NPCs. The downregulation phenotypes gp210               molecules, resulting products arrested either <br /> were similar C. elegans embryos human cell             states. One state &#226;&#8364;&#339;hemi-fused&#226;&#8364;?: here, inner <br /> cultures, suggesting gp210 function conserved        leaflets fuse, forming membrane &#226;&#8364;&#339;bridge&#226;&#8364;? blocks con- <br /> in evolution. The amino acid sequences gp210 C. elegans      tent mixing compartments. A second ar- <br /> and mammals (rat) 25% identical (44% similar)             rested state involves complete fusion leaflets, the <br /> (Cohen et al., 2001). These identical residues likely    resulting pore fails dilate (Kozlov Chernomordik, <br /> essential structure function gp210 serve    2002). <br /> as starting points future molecular analysis.                     Our results cells running gp210 are <br />                                                                    consistent models. We speculate twinned <br /> Gp210 Is Involved NPC Spacing                                   membranes represent earliest detectable stage of <br /> In human cells C. elegans, nuclei reduced            arrest, fewest gp210 molecules, inter- <br /> gp210 failed space NPCs evenly nuclear en-           mediates pronounced curvature membrane (al- <br /> velope, evidenced NPC clustering. Given large            outer membrane) close contact the <br /> volume nucleus, distribution NPCs             second membrane, lacked &#226;&#8364;&#339;focal&#226;&#8364;? point fusion. We <br /> probably important efficient nuclear import export.        propose fusion-arrested pores gp210 to <br /> The mechanisms NPC spacing understood. Nu-              initiate, complete, fusion reaction. <br /> clear lamins critical spacing (Lenz-Bohme et al., 1997;    Note twinned fusion-arrested membrane structures <br /> Liu et al., 2000; Schirmer et al., 2001), NPCs attach   produced assembling nuclei, treatment with <br /> lamin filaments unclear. One possibility suggested        competing gp210 tail polypeptides antibodies directed <br /> the interaction Nup153 lamins demon-             Xenopus gp210 tail domain, respectively (Drum- <br /> strated role Nup153 NPC anchoring (Smythe et al.,           mond Wilson, 2002). Overall, results strongly sup- <br /> 2000; Walther et al., 2001). We determine          port direct role gp210 nuclear membrane fusion and <br /> experiments gp210 directly involved anchoring             consistent current models multiple copies <br /> NPCs. However, given mass primarily lumenal,           fusogenic protein (e.g., gp210) required effi- <br /> we suggest loss gp210 disrupts nucleopor-        ciently fuse flattened membranes dilate result- <br /> ins direct roles NPC anchoring. Indeed, NPC clus-          ing pore. Interestingly, results suggest additional roles <br /> tering phenotypes caused mutations yeast            gp210 maintaining spacing structural integrity <br /> nucleoporins (Zabel et al., 1996; Bucci Wente, 1998;           mature NPCs. Further studies test models will <br /> Siniossoglou et al., 2000; Ryan Wente, 2002),          shed light fusion mechanism fundamental nucle- <br /> which vertebrate homologues (reviewed Vasu             ocytoplasmic communication eukaryotes. <br /> Forbes, 2001). <br />                                                                    ACKNOWLEDGMENTS <br /> Similarities gp210 Downregulation Phenotypes in <br /> Human C. elegans Cells                                         We thank J.C. Courvalin anti-human gp210 antibodies K. Weber <br />                                                                    G. Krohne antibodies human lamin. We thank A. Loyter <br /> Knockdown gp210 gene human HeLa cells               help performing import assay. This work funded grants <br /> and C. elegans embryos caused nuclear membranes accu-           Israel Science Foundation (ISF) Austrian Bank (to Y.G.), the <br /> mulate aberrant structures termed twinned fusion-ar-           National Institutes Health (GM64535 K.L.W.), Journal Cell Science <br />                                                                    travel award Boeringer Ingelheim travel allowance (to M.C.). <br /> rested membranes. These aberrant structures repre- <br /> sent mixture arrested intermediates porogenic <br /> nuclear membrane fusion degenerating mature NPCs.              REFERENCES <br /> There similarities C. elegans            Benhar, M., Dalyot, I., Engelberg, D., Levitzki, A. (2001). Enhanced ROS <br /> human cell phenotypes, including cell death, chromatin dis-        production oncogenically transformed cells potentiates c-Jun N-terminal <br />  <br />  <br />  <br /> 4236                                                                                                           Molecular Biology Cell <br />                                                                                                                                 gp210 Is Essential Viability <br />  <br /> kinase p38 mitogen-activated protein kinase activation sensitization       Kelly, G., Prasannan, S., Daniell, S., Fleming, K., Frankel, G., Dougan, G., <br /> to genotoxic stress. Mol. Cell. Biol. 21, 6913&#226;&#8364;&#8220; 6926.                              Connerton, I., Matthews, S. (1999). Structure cell-adhesion frag- <br />                                                                                    ment intimin enteropathogenic Escherichia coli. Nat. Struct. Biol. 6, <br /> Bodoor, K., Shaikh, S., Enarson, P., Chowdhury, S., Salina, D., Raharjo, W.H.,     313&#226;&#8364;&#8220;318. <br /> and Burke, B. (1999). Function assembly nuclear pore complex pro- <br /> teins. Biochem. Cell Biol. 77, 321&#226;&#8364;&#8220;329.                                            Kozlov, M.M., Chernomordik, L.V. (2002). The protein coat membrane <br />                                                                                    fusion: lessons fission. Traffic 3, 256 &#226;&#8364;&#8220;267. <br /> Broder, Y.C., Stanhill, A., Zakai, N., Friedler, A., Gilon, C., Loyter, A. <br />                                                                                    Lee, K.K., Haraguchi, T., Lee, R.S., Koujin, T., Hiraoka, Y., Wilson, K.L. <br /> (1997). Translocation NLS-BSA conjugates nuclei permeabilized <br />                                                                                    (2001). Distinct functional domains emerin bind lamin A DNA-bridg- <br /> mammalian cells supported protoplast extract. An experimental <br />                                                                                    ing protein BAF. J. Cell Sci. 114, 4567&#226;&#8364;&#8220; 4573. <br /> system studying plant cytosolic factors involved nuclear import. FEBS <br /> Lett. 412, 535&#226;&#8364;&#8220;539.                                                                Lei, E.P., Silver, P.A. (2002). Protein RNA export nucleus. <br />                                                                                    Dev. Cell 2, 261&#226;&#8364;&#8220;272. <br /> Brenner, S. (1974). The genetics Caenorhabditis elegans. Genetics 77, 71&#226;&#8364;&#8220;94. <br />                                                                                    Lenz-Bohme, B., Wismar, J., Fuchs, S., Reifegerste, R., Buchner, E., Betz, H., <br /> Bucci, M., Wente, S.R. (1998). A novel fluorescence-based genetic strategy     Schmitt, B. (1997). Insertional <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span>mutation</span> Drosophila nuclear lamin <br /> identifies mutants Saccharomyces cerevisiae defective nuclear pore com-     dm(0) gene results defective nuclear envelopes, clustering nuclear pore <br /> plex assembly. Mol. Biol. Cell 9, 2439 &#226;&#8364;&#8220;2461.                                      complexes, accumulation annulate lamellae. J. Cell Biol. 137, 1001&#226;&#8364;&#8220; <br />                                                                                    1016. <br /> Cohen, M., Tzur, Y.B., Neufeld, E., Feinstein, N., Delannoy, M.R., Wilson, <br /> K.L., Gruenbaum, Y. (2002). Transmission electron microscope studies        Liu, J., Rolef-Ben Shahar, T., Riemer, D., Spann, P., Treinin, M., Weber, K., <br /> the nuclear envelope C. elegans embryos. J. Struct. Biol. 140, 232&#226;&#8364;&#8220;240.         Fire, A., Gruenbaum, Y. (2000). Essential roles Caenorhabditis elegans <br />                                                                                    lamin gene nuclear organization, cell cycle progression, spatial orga- <br /> Cohen, M., Wilson, K.L., Gruenbaum, Y. (2001). Membrane proteins            nization nuclear pore complexes. Mol. Biol. Cell 11, 3937&#226;&#8364;&#8220;3947. <br /> the nuclear pore complex: Gp210 conserved vertebrates, Drosophila <br /> melanogaster, C. elegans Arabidopsis. Palo Alto, CA: Gene Teraphy              Macara, I.G. (2001). Transport nucleus. Microbiol. Mol. <br /> Press.                                                                             Biol. Rev. 65, 570 &#226;&#8364;&#8220;594. <br />                                                                                    Maeda, I., Kohara, Y., Yamamoto, M., Sugimoto, A. (2001). Large-scale <br /> Courvalin, J.C., Lassoued, K., Bartnik, E., Blobel, G., Wozniak, R.W.          analysis gene function Caenorhabditis elegans high-throughput <br /> (1990). The 210-kD nuclear envelope polypeptide recognized human au-            RNAi. Curr. Biol. 6, 171&#226;&#8364;&#8220;176. <br /> toantibodies primary biliary cirrhosis major glycoprotein the <br /> nuclear pore. J. Clin. Invest. 86, 279 &#226;&#8364;&#8220;285.                                       Pante, N., Aebi, U. (1996). Molecular dissection nuclear pore <br />                                                                                    complex. Crit. Rev. Biochem. Mol. Biol. 31, 153&#226;&#8364;&#8220;199. <br /> Cronshaw, J.M., Krutchinsky, A.N., Zhang, W., Chait, B.T., Matunis, M.J. <br />                                                                                    Pilpel, Y., Bogin, O., Brumfeld, V., Reich, Z. (2003). Polyproline type II <br /> (2002). Proteomic analysis mammalian nuclear pore complex. J. Cell <br />                                                                                    conformation C-terminal domain nuclear pore complex protein <br /> Biol. 158, 915&#226;&#8364;&#8220;927. <br />                                                                                    gp210. Biochemistry 42, 3519 &#226;&#8364;&#8220;3526. <br /> Davis, L.I., Blobel, G. (1986). Identification characterization       Rout, M.P., Aitchison, J.D., Suprapto, A., Hjertaas, K., Zhao, Y., Chait, B.T. <br /> nuclear pore complex protein. Cell 45, 699 &#226;&#8364;&#8220;709.                                   (2000). The yeast nuclear pore complex: composition, architecture, trans- <br /> Drummond, S.P., Wilson, K.L. (2002). Interference cytoplasmic         port mechanism. J. Cell Biol. 148, 635&#226;&#8364;&#8220; 651. <br /> tail gp210 disrupts &#226;&#8364;&#339;close apposition&#226;&#8364;? nuclear membranes blocks          Ryan, K.J., Wente, S.R. (2002). Isolation characterization new <br /> nuclear pore dilation. J. Cell Biol. 158, 53&#226;&#8364;&#8220; 62.                                  Saccharomyces cerevisiae mutants perturbed nuclear pore complex assem- <br />                                                                                    bly. BMC Genet. 3, 17. Available www.biomedcentral.com/1471-2156/3/ <br /> Elbashir, S.M., Harborth, J., Weber, K., Tuschl, T. (2002). Analysis gene <br />                                                                       <br /> </body></html>