<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR AND CELLULAR BIOLOGY, Oct. 1995, p. 5618&#226;&#8364;&#8220;5626                                                                                      Vol. 15, No. 10 <br /> 0270-7306/95/$04.0010 <br /> Copyright q 1995, American Society Microbiology <br />  <br />  <br />  <br />              SCS1, Multicopy Suppressor hsp60-ts Mutant Alleles, <br />               Does Not Encode Mitochondrially Targeted Protein <br />                                          YOUMIN SHU        AND   RICHARD L. HALLBERG* <br />                                 Department Biology, Syracuse University, Syracuse, New York 13244 <br />                             Received 2 May 1995/Returned modification 27 June 1995/Accepted 17 July 1995 <br />  <br />             We identified isolated Saccharomyces cerevisiae gene which, overexpressed, suppressed the <br />           temperature-sensitive <span id='am-238' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-239' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-240' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-250' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-253' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-256' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-267' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-284' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-290' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-296' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-299' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span>phenotype</span> cells expressing mutant allele gene encoding mitochondrial <br />           chaperonin, Hsp60. This gene, SCS1 (suppressor chaperonin sixty-1), encodes 757-amino-acid protein of <br />           unknown function which, nonetheless, human, rice, Caenorhabditis elegans homologs high <br />           degrees (ca. 60%) amino acid sequence identity. SCS1 essential gene, SCS1-null strains not <br />           grow 37&amp;C growth-related defects 30&amp;C well. This gene expressed 30 and <br />           38&amp;C, producing little differences mRNA levels temperatures. Overexpression SCS1 <br />           complement HSP60-<span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span>null allele</span>, indicating suppression bypassing Hsp60 <br />           activity. Of 10 hsp60-ts alleles tested, suppressed SCS1 overexpression. There were <br />           common mutant phenotypes strains expressing alleles clue were <br />           suppressible not. An epitope (influenza virus hemagglutinin)-tagged form SCS1 single <br />           copy complemented SCS1-null allele. The Scs1-hemagglutinin protein comparable levels <br />           similar multiply modified forms cells growing 30 38&amp;C. Surprisingly, localized either <br />           cell fractionation procedures immunocytochemistry, proteins mitochondria <br />           cytosol. The overexpression SCS1 significant effects cellular levels mRNAs encoding <br />           proteins Cpn10 Mge1, mitochondrial protein cochaperones, mRNAs encoding a <br />           number mitochondrial cytosolic proteins analyzed. The implications findings discussed. <br />  <br />  <br />    The Saccharomyces cerevisiae chaperonin, Hsp60, es-                 allele specific suppressing capabilities (ii) iden- <br /> sential component protein refolding machinery located                 tification novel proteins functional relationship <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> <br /> in mitochondrial matrix (4, 22, 23, 25). While Hsp60, like               Hsp60 heretofore unrecognized. This report de- <br /> its Escherichia coli counterpart, GroEL (8, 13), appears play             scribes identification example latter. We have <br /> a major role refolding process, does act (16).          identified gene, SCS1, encoding protein unknown func- <br /> Genetic (10) biochemical (20) studies indicate            tion which, overproduced, suppresses ts phenotype of <br /> E. coli, protein folding pathway involving sequen-            some, all, hsp60 mutant alleles. However, quite <br /> tial activities interactions proteins DnaK, DnaJ,                 unexpectedly, apparently does extramitochon- <br /> GrpE, GroEL, GroES. Homologs of proteins                    drial location. We present evidence suggests Scs1 <br /> have identified mitochondria S. cerevisiae                involved regulation expression genes en- <br /> (2, 5, 19, 25, 28, 30), recent evidence (15, 19, 21) suggests            coding mitochondrial heat shock proteins discuss this <br /> that similar protein folding pathway probably exists                account <span id='am-473' about='xsp:pattern' typeof='owl:Thing'>pattern</span> suppression. <br /> the mitochondrial matrix. It remains, however, proven <br /> whether case and, importantly, all <br /> matrix-targeted proteins follow pathway.                                                MATERIALS AND METHODS <br />    One approach used identify functionally                     Yeast strains growth media. The S. cerevisiae strains used study are <br /> interacting proteins overproduction                   derived strains: diploid a/aW303 (ura3/ura3 leu2/leu2 <br /> protein ameliorates, i.e., suppresses, deleterious effects            his3/his3 trp1/trp1 ade3/ade3) aW303. All hsp60-ts strains chromo- <br /> a mutant protein overproduced protein                      somal copy HSP60 inactivated insertion HIS3 gene (25). The <br />                                                                              mutant form HSP60 strains (generated hydroxyl- <br /> thought interact. Causing overexpression particular                  amine treatment [26, 32] site-directed mutagenesis [1, 18]) carried on <br /> gene introducing extra copies genes condition-              plasmid pFL39 (a pUC19-derived plasmid containing yeast CEN4 TRP1 <br /> ally lethal cells (high-copy suppression) proven successful              sequences; obtained Patrick Linder). <br /> in identifying and/or confirming physical interaction                    Cells grown YPD (1% yeast extract, 2% dextrose, 2% peptone), <br />                                                                              YPEG (1% yeast extract, 3% ethanol, 3% glycerol, 2% peptone), synthetic <br /> proteins E. coli S. cerevisiae.                             medium (0.67% yeast nitrogen <span id='am-472' about='xsp:base' typeof='owl:Thing'>base</span>, 2% dextrose) supplemented ap- <br />    As generated array 30 different temper-                 propriate amino acids (0.002%) required. <br /> ature-sensitive (ts) alleles S. cerevisiae HSP60 gene (32a),             Isolation multicopy suppressor SCS1. The ts strain expressing hsp60- <br /> we decided use high-copy suppression identify genes                    G432D (12) transformed S. cerevisiae genomic library cloned into <br />                                                                              plasmid YEp351 (LEU2) (9). Transformants allowed grow selective <br /> whose products functionally interact Hsp60. Two possible                plates 308C 3 days replicated new selective plates and <br /> outcomes approach (i) identification other,              incubated 388C temperature-resistant colonies appeared (5 days). Plas- <br /> already characterized mitochondrial chaperones                 mid DNAs recovered standard methods (31) colonies that <br />                                                                              grew restriction enzyme mapped order distinguish between <br />                                                                              YEp351 plasmids pFL39 plasmids containing mutant HSP60 gene. In <br />                                                                              case, YEp351 plasmids containing DNA inserts identified, isolated, <br />   * Corresponding author. Mailing address: Department Biology,            used transform original ts strain. In case did second <br /> Lyman Hall, Syracuse University, Syracuse, NY 13244. Phone: (315)            transformation result transformants non-ts. The YEp351 plasmid <br /> 443-1104. Fax: (315) 443-2156. Electronic mail address: hallberg@            possessing suppressing activity contained 7-kb chromosomal insert. It was <br /> mailbox.syr.edu.                                                             designated pSH1. <br />  <br />                                                                       5618 <br />  VOL. 15, 1995                                                                                                    CHARACTERIZATION OF SCS1                          5619 <br />  <br />                                                                                        112-bp BglII fragment encoding repeats hemagglutinin (HA) peptide <br />                                                                                        (YPYDVPDYA) inserted internally C terminally SCS1 gene. In <br />                                                                                        case, 112-bp fragment directly cloned BclI site. Such a <br />                                                                                        construct introduced 27 amino acids reading frame Scs1. The <br />                                                                                        gene encoding protein designated SCS1::HA3(1). For C-terminal <br />                                                                                        epitope tagging, BglII site introduced carboxyl terminus the <br />                                                                                        coding sequence. To generate single-stranded DNA site-directed mutagen- <br />                                                                                        esis, 3.3-kb BglII-PvuII fragment containing entire SCS1 gene sub- <br />                                                                                        cloned shuttle vector pRS315 (33), phagemid. A synthetic <br />                                                                                        oligonucleotide (GACAGCGAGATACAAGATCTATGAAACAGCAATTCAT) <br />                                                                                        designed BglII site created immediately the <br />                                                                                        termination codon. Site-directed mutagenesis carried method of <br />                                                                                        Kunkel et al. (18), correctly mutagenized clones identified by <br />                                                                                        digesting isolated DNAs BglII. A 112-bp BglII fragment containing HA <br />                                                                                        epitopes cloned newly generated BglII site of plasmids. <br />                                                                                        Restriction analysis used identify plasmids single insertion <br />                                                                                        occurred proper orientation. The gene encoding terminally tagged <br />                                                                                        protein designated SCS1::HA3(2). <br />                                                                                           The fusion gene expressed Scs1 terminal tail histidine <br />                                                                                        residues (SCS1::his6) created site-directed mutagenesis. A synthetic oli- <br />                                                                                        gonucleotide (GACAGCGAGATACAGCACCACCACCACCACCACTGAA <br />                                                                                        ACAGGAATTCATTAC) used introduce sequence encoding histi- <br />                                                                                        dine residues carboxyl terminus Scs1 methods described above. <br />                                                                                        To purify Scs1-His6, cells expressing fusion protein grown syn- <br />                                                                                        thetic medium harvested late log phase. The cells resuspended in <br />                                                                                        lysis buffer (8 M urea, 0.1 M NaH2PO4, 0.01 Tris-HCl [pH 8.0]) broken by <br />                                                                                        using glass beads. Scs1-His6 purified Ni column supplied by <br />                                                                                        Qiagen, using procedure suggested company. <br />                                                                                           RNA isolation analysis. Mid-log-phase cells growing synthetic medium <br />                                                                                        308C divided equal parts. One shifted 388C, the <br />                                                                                        kept 308C. Following 30-min incubation, cells harvested and <br />                                                                                        total RNA isolated hot phenol method (17). RNA treated glyoxal <br />                                                                                        (as described reference 25) electrophoretically separated 1% agarose <br />                                                                                        gels transferred Biotrans membranes. Cloned copies specific <br />    FIG. 1. Identification mapping SCS1 gene. (a) Restriction map         genes, liberated respective plasmids nuclease digestion, iso- <br /> the 7-kb insert YEp351 library that, high copy number, suppressed        lated labeled [32P]dATP random priming (1). Northern (RNA) <br /> ts phenotype ts3 cells. Bam, BamHI; RI, EcoRI; Bgl, BglII; Pvu, PvuII; Hin,         analyses carried Biotrans membranes radioactive <br /> HindIII; Bcl, BclI; Sac, SacI. The flanking sites shown plasmid         probes. <br /> insert itself. There approximately 100 additional bases chromosomal DNA            Western blot (immunoblot) analyses cell fractionation. To identify HA <br /> between sites multiple cloning site plasmid. (b) Determina-       epitope-tagged forms Scs1 cell extracts, pelleted yeast cells were <br /> tion partial fragments 7-kb insert possess suppressing activity. The    dissolved 1.4 M NaOH&#226;&#8364;&#8220;5% b-mercaptoethanol, solubilized proteins <br /> last DNA shown 1.7-kb fragment containing HIS3 gene inserted          processed previously described (12). These proteins separated by <br /> BclI site. In case, DNA fragment inserted YEp352, trans-             sodium dodecyl sulfate (SDS)-polyacrylamide gel electrophoresis (PAGE) (10% <br /> formed strain expressing hsp60-G432D, tested ability           gels), transferred nitrocellulose membranes, immunodecorated a <br /> endow cells ability grow 388C. (c) Growth 30 388C ts3 cells   monoclonal antibody directed HA epitope (12CA5; Boehringer <br /> containing different multicopy plasmids: YEp352, YEpSCS1 (YEp352 contain-              Mannheim) followed alkaline phosphatase-conjugated goat anti-mouse <br /> ing 3.5-kb BglII-PvuII fragment shown panel b), YEpHSP60 (YEp352            immunoglobulin G antibody (Bethesda Research Laboratories). <br /> containing HSP60) (25).                                                                   For cellular fractionation, late-log-phase cells growing synthetic medium <br />                                                                                        harvested, spheroplasts generated, cell homogenization car- <br />                                                                                        ried described Daum et al. (6). Initial homogenates centrifuged at <br />                                                                                        3,500 3 g 5 min remove partially unbroken cells. The resulting <br />    Plasmid constructions DNA manipulations. The suppressing gene carried           supernatant considered essentially unfractionated cell ho- <br /> by pSH1 localized subcloning restriction fragments yeast chromo-         mogenate. For fractionation purposes, cell homogenate spun at <br /> somal insert (Fig. 1) high-copy-number vector YEp352 (URA3) (14).             20,000 3 g 15 min pellet nuclei mitochondria. The supernatant <br /> These subcloned fragments introduced original strain (ts3) express-      generated designated postmitochondrial fraction. After nuclear/ <br /> ing hsp60-G432D, non-ts transformants identified. For sequencing              mitochondrial pellet redissolved initial homogenization buffer, aliquots of <br /> purposes, 4.0-kb BglII-EcoRI fragment (Fig. 1) subcloned              solutions added 0.5 volume 33 SDS-gel sample buffer and <br /> pBS(1) pBS(2) phagemids. Nested deletions generated using                  subjected SDS-PAGE, separated proteins transferred nitro- <br /> exonuclease III digestion. Both strands insert sequenced standard       cellulose membranes immunodecorated described above. <br /> methods.                                                                                  Indirect immunofluorescence. To visualize cells immunocytochemically, the <br />    Two different protocols used create SCS1-null strains. Initially        procedure described Davis Fink (7) followed zymolyase <br /> both, 5.5-kb SacI-PstI (the multiple cloning site pSH1)           used generate spheroplasts protease inhibitors omitted. The <br /> fragment subcloned vector pBS(1) plasmid pSH2. Then,           primary antibody used monoclonal antibody described directed <br /> the case, 1.7-kb BamHI fragment carrying HIS3 gene cloned       HA epitope (final concentration, 1 mg/ml), secondary anti- <br /> BclI site new plasmid create insertion coding region       body fluorescein isothiocyanate-conjugated goat anti-mouse immunoglobu- <br /> SCS1. In second case, 1.7-kb HIS3 fragment cloned              lin G (final concentration, 5 mg/ml; Bethesda Research Laboratories). <br /> BglII-BclI-cut pSH2 create insertion construct approximately <br /> one-half SCS1 coding region 610 bases 59 AUG codon were <br /> deleted. Subsequently, EcoRI digestion used liberate linear 3.9-kb <br />                                                                                                                         RESULTS <br /> or linear 5.3-kb disrupted SCS1 gene, used the <br /> one-step gene disruption procedure (29) transform wild-type a/aW303                Identification high-copy-number suppressor con- <br /> diploid strain. To obtain haploid SCS1-null cells, diploid transformants          ditional hsp60 mutant. To identify genes which, when <br /> induced sporulate 258C, His1 colonies isolated. To confirm         overexpressed, suppress ts phenotype hsp60 mutants, we <br /> disruption SCS1 gene DNA construct occurred, Southern <br /> analyses carried EcoRI-digested DNA strains, using the <br />                                                                                        transformed strain ts3, expressing hsp60-G432D, strong ts <br /> 2.2-kb EcoRI-EcoRI fragment probe. In case, DNA fragments            allele (12), high-copy-number plasmid (YEp351) yeast <br /> the predicted sizes disrupted nondisrupted forms SCS1            genomic DNA library. The resulting transformants, initially <br /> (data shown). The gene containing just HIS3 insertion designated           selected 308C, screened 388C. Among approx- <br /> scs1::HIS3(1). The gene partial deletion plus HIS3 insertion desig- <br /> nated scs1::HIS3(2). Strains expressing gene designated SCS1- <br />                                                                                        imately 30,000 transformants, colonies grew 388C after <br /> nulla; strains expressing designated SCS1-nullb.                       4 days incubation. To determine suppression ts <br />    To generate SCS1 genes expressed proteins carrying foreign epitope,        phenotype ts3 cells plasmid-borne gene, plas- <br />  5620       SHU AND HALLBERG                                                                                                                         MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />    FIG. 2. Sequence analysis SCS1. (a) Sequence suppressing BglII-PvuII DNA fragment. The single long open reading frame shown. Putative TATA boxes <br /> at 2120 270 bases 59 AUG start codon indicated. The underlined BclI sequence site SCS1 coding sequence interrupted <br /> by insertion HIS3 gene (see Fig. 1). (b) Sequence comparisons Scs1 protein encoded partial cDNA isolated human myeloblasts (GenBank <br /> entry D26445). The upper sequence yeast protein (S.c.) amino acid residues 276 712 (<span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> total 757); lower human protein (H.s.) <br /> from amino acid residues 26 445 (from total 484). The middle sequence indicates identities letters conservative changes plus signs. <br /> Dashes indicate gaps. Not considering sequence yeast protein counterpart human protein, sequences protein fragments exhibit <br /> 60% identity 76% similarity conservative changes counted. <br />  <br />  <br />  <br /> mid DNAs isolated non-ts cell lines                          concluded extragenic suppressor isolated, <br /> retransformed original host. Of different                            named SCS1 (suppressor chaperonin sixty-1). <br /> plasmids, conferred new ts3 transformants                           Different-size fragments 7-kb chromosomal insert in <br /> ability grow 388C, indicating suppression                      YEp351 subcloned high-copy-number plas- <br /> result plasmid-borne gene. This suppressing plasmid, con-                         mid (YEp352), abilities suppress determined <br /> taining 7-kb chromosomal insert (Fig. 1a), did contain                           (Fig. 1b). The suppressing activity localized 3.5-kb <br /> the HSP60 gene (which suppressed complemen-                              fragment. An insertion 1.7-kb HIS3 gene BclI site <br /> tation), restriction map plasmid insert differed                      3.5-kb segment abolished suppressing ac- <br /> from bona fide HSP60 gene (25). From finding,                         tivity, indicating BclI site open reading frame <br />  VOL. 15, 1995                                                                             CHARACTERIZATION OF SCS1                5621 <br />  <br />  <br />  <br />  <br />                                                            FIG. 2&#226;&#8364;&#8221;Continued. <br />  <br />  <br />  <br /> of SCS1 control region. While overexpression            SCS1 suppress hsp60-ts alleles bypass <br /> SCS1 allowed ts3 cells grow 388C, effective      suppressor. An important question SCS1 sup- <br /> as HSP60 itself ameliorating temperature sensitivity         pressed allele-specific manner. If SCS1 allele spe- <br /> (Fig. 1c).                                                           cific suppression, suppresses group hsp60-ts <br />    SCS1 encodes evolutionarily conserved protein          alleles elicit similar mutant phenotypes, able <br /> as unknown function. The entire sequence 3.5-kb           predict sort functional defect overproduction Scs1 <br /> BglII-PvuII DNA fragment determined (Fig. 2a). It con-           corrects. Therefore, 10 hsp60-ts mutant strains trans- <br /> tained single long open reading frame 2,271 bases poten-        formed YEpSCS1, transformants tested for <br /> tially encoding 86-kDa protein 757 amino acid residues.        growth 388C (Fig. 3). Among 10 strains transformed, 5 <br /> The position reading frame encoding SCS1              ts phenotypes suppressed. Thus, SCS1 allele <br /> consistent DNA fragments did did sup-           specific. Comparing ts phenotypes strains, as <br /> press (Fig. 1b). Two putative TATA boxes 120          induction solubility Hsp60 (11, 12) high temper- <br /> 70 bases upstream ATG codon. As determined            ature, reversibility high-temperature treatment via- <br /> by search Prosite database, translated protein          bility, buildup precursor forms mitochondrial pro- <br /> sequence contained obvious functional motifs. One notable         teins nonpermissive temperature, two <br /> feature protein sequence abundance serines         strains, expressing hsp60-Y510D hsp60-ts8, have <br /> and threonines 200 amino acids, account      phenotypes similar ts3 cells. Thus, high-copy sup- <br /> for 20% total amino acid residues.                      pression of strains did occurring by <br />    Using BLASTP TBLASTN search Na-               alleviation particular Hsp60 misfunction. A possible <br /> tional Center Biotechnology Information database,             explanation finding discussed below. <br /> identified homologous protein sequences. The          As mutant strains suppressed overproduc- <br /> an S. cerevisiae sequence essentially identical (two amino acid      tion Scs1, unlikely Scs1 replaces function of <br /> differences) sequence Scs1. The information accom-         heat-inactivated Hsp60; is, does act bypass <br /> panying sequence (not published) database (Gen-          suppressor. However, test possibility directly, intro- <br /> Bank entry U06330) indicated encoded multicopy             duced SCS1 gene YEp351 strain chromo- <br /> suppressor mutant allele ROX3 (27), gene encoding        somal copy HSP60 disrupted viable <br /> predominantly nuclear protein regulates expression       carried copy HSP60 YEp352, noncentro- <br /> the iso-2-cytochrome c gene, CYC7. Another database se-              meric, URA3-containing plasmid. Such double transformants <br /> quence SCS1 related partial cDNA          grown media selected retention YEp351 <br /> created human myeloblast mRNA (GenBank entry                    (and SCS1) applied selection YEp352 (and HSP60). <br /> D26445). In case, 484-amino-acid human sequence             After days growth, cells plated YPD agar, <br /> encoded partial cDNA homologous               colonies formed, replicated onto <br /> almost entire <span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span> C-terminal two-thirds Scs1    5-fluoro-orotic acid-containing YPD plates cells <br /> sequence (Fig. 2b). The overall identity 55%, 78%         lost YEp352 plasmid survive with- <br /> similarity conservative substitutions considered. In       copy functional HSP60 gene. No cells grew on <br /> addition, amino acid sequences 26 200 residues       5-fluoro-orotic acid 25 378C (data shown), <br /> short partial cDNA sequences mice, rice, C. elegans         indicating overexpression SCS1 does allow cells to <br /> genes second human gene gave identities Scs1 ranging        dispense Hsp60 showing suppression take <br /> from 63 69% (data shown). No function ascribed         place ameliorating defect Hsp60 function high <br /> any homologs.                                               temperatures bypassing defect. <br />  5622       SHU AND HALLBERG                                                                                                                     MOL. CELL. BIOL. <br />  <br />  <br />                                                                                               TABLE 1. Growth characteristics cells expressing <br />                                                                                                         nonfunctional SCS1 gene <br />                                                                                                         Doubling timeb (h)           Relative growth ethanolc <br />                                                                                       Cell typea <br />                                                                                                      YPD      YPGal     YPEG         0%         3%      6%      9% <br />  <br />                                                                                      W303             2.0       2.8       2.5      1 (13.6)    0.71    0.11     0.02 <br />                                                                                      SCS1-nulla       3.1       2.9       4.9      1 (9.7)     0.07    0.04     0.02 <br />                                                                                         a <br />                                                                                           W303 cells (see Materials Methods) parent strain in <br />                                                                                      SCS1 disrupted insertion functional HIS3 gene. These <br />                                                                                      cells contain gene scs1::HIS3(1) designated SCS1-nulla. <br />                                                                                         b <br />                                                                                           The compositions media cells grown given in <br />                                                                                      Materials Methods. All growth measurements 308C. <br />                                                                                         c <br />                                                                                           Cells growing YPD 308C diluted optical density 600 nm <br />                                                                                      (OD600) 0.1 fresh YPD medium containing ethanol various concen- <br />                                                                                      trations. The cultures incubated 308C, OD600 culture was <br />                                                                                      measured 16 h later. The relative growth particular concentration <br />                                                                                      ethanol determined dividing OD600 culture OD600 of <br />                                                                                      comparable culture containing ethanol. Each number parentheses is <br />                                                                                      OD600 control culture 16 h. <br />  <br />  <br />  <br />                                                                                      rupted SCS1 gene, wild-type SCS1 gene centromere- <br />    FIG. 3. SCS1 suppression strains expressing hsp60-ts alleles. Cells      containing plasmid pRS315 (33) reintroduced SCS1- <br /> expressing alleles indicated diagram left transformed    nulla cells. Such transformants longer ts (Fig. 4). <br /> plasmid YEp351 containing 7-kb chromosomal insert. (Mutant strains are <br /> designated [i] altered amino acid HSP60 [e.g., Y510D], [ii] absence       As initial disruption SCS1 generated simply <br /> of designated number amino acids carboxyl terminus Hsp60            inserting HIS3 gene it, conceivable ts <br /> [D26], [iii] hsp60-ts alleles alterations determined     phenotype SCS1-nulla cells expression a <br /> [ts8 ts6]. In strains tested, chromosomal copy HSP60 hsp60::HIS3   truncated form Scs1 simply function ele- <br /> and mutant HSP60 allele carried pFL39 [12].) Such transformants were <br /> grown 258C selective plates order maintain plasmid YEp351,      vated temperatures. To rule possibility, generated <br /> they replicated new selective plates. One plates main-      second SCS1 disruption strain 610 bases 59 the <br /> tained 258C incubated 378C.                                      AUG codon two-thirds SCS1 open reading <br />                                                                                      frame replaced HIS3 gene [scs1::HIS3(2); see <br />                                                                                      Materials Methods]. These SCS1-nullb cells exhibited the <br />                                                                                      phenotype SCS1-nulla cells, indicating ts <br />    If mutant Hsp60s totally inactivated 388C                        phenotype absence Scs1, presence of <br /> and consequently insufficient activity maintain normal                        ts form. <br /> mitochondrial biogenesis, deficiency conceivably                          One possible reason SCS1-null cells ts (or ethanol <br /> overcome increasing levels Hsp60 mitochon-                          sensitive) Scs1 stress-inducible protein whose <br /> dria. Western analyses suppressed nonsuppressed                          function primarily solely required high temperatures or <br /> mutant strains, wild-type controls, 30                        stressful conditions expressed then. <br /> 388C revealed (data shown) obvious differences                             The experiments described expression the <br /> Hsp60 amounts normal increases Hsp60                          SCS1 gene markedly affected heat stress the <br /> occur higher temperatures (22, 25). This lack effect                        form Scs1 similarly unaffected. <br /> SCS1 overexpression expression HSP60 indepen-                             Determining metabolism cellular location Scs1. <br /> dently confirmed Northern analyses (see below).                                   To study metabolism Scs1 ascertain cellular <br />    SCS1 essential 30&amp;C required cell <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span>survival</span>                   location, created gene encoding epitope-tagged form <br /> at elevated temperatures. The chromosomal copy SCS1                           Scs1 (see Materials Methods details). This gene, <br /> inactivated haploid diploid cells one-step gene                       SCS1::HA3(2), introduced single-copy plasmid <br /> disruption (29) using linear copy SCS1 HIS3                        pRS315 SCS1-nulla cells. In cases, transformants <br /> gene inserted [scs1::HIS3(1); Materials Methods].                        longer ts (Fig. 5a), indicating gene encoding <br /> In cases, viable His1 cells grew 308C glucose- <br /> containing medium obtained. DNA hap- <br /> loid diploid transformants analyzed, analysis <br /> confirmed copy SCS1 gene disrupted <br /> with HIS3 gene (data shown). Thus, appeared that <br /> SCS1 essential growth 308C. Haploid cells ex- <br /> pressing scs1::HIS3(1) (SCS1-nulla cells) grew nonfer- <br /> mentable carbon sources, regardless growth me- <br /> dium, SCS1-nulla cells grew slowly wild-type cells at <br /> 308C (Table 1). SCS1-nulla cells failed grow 388C irre- <br /> spective growth medium (Fig. 4). In examining number <br /> of growth-related properties SCS1-nulla cells, we <br /> found addition ts, hypersensitive <br /> to ethanol (Table 1). Diploid cells [SCS1/scs1::HIS3(1)] were <br /> indistinguishable wild-type cells regard                             FIG. 4. Growth properties SCS1-nulla cells. Wild-type cells, SCS1-nulla <br /> growth properties (data shown).                                                  cells containing plasmid, SCS1-nulla cells transformed pRSCS1 <br />    To confirm ts phenotype result dis-                       (pRS315 containing SCS1) grown YPD 30 388C. <br />  VOL. 15, 1995                                                                                              CHARACTERIZATION OF SCS1                 5623 <br />  <br />  <br />                                                                                      SCS1::HA3(2). This analysis showed (Fig. 5b, lane 1) a <br />                                                                                      group immunologically reactive protein bands <br />                                                                                      ranging apparent mass 95 115 kDa was <br />                                                                                      present cells expressing SCS1::HA3(2) absent control <br />                                                                                      cells (Fig. 5b, lane 3). No forms immunoreactive pro- <br />                                                                                      teins predicted size (ca. 89 kDa) fusion protein. <br />                                                                                      This finding suggests Scs1 modified man- <br />                                                                                      ner differing degrees modification occurred. Alter- <br />                                                                                      natively, band lowest apparent mass an <br />                                                                                      unmodified form Scs1 anomalous electrophoretic <br />                                                                                      mobility, bands modified isoforms. <br />                                                                                      The amounts forms immunoreactive species were <br />                                                                                      cells growing 30 378C (Fig. 5b, lane 2), <br />                                                                                      showing Scs1 levels change response elevated <br />                                                                                      temperature. The level Hsp60 increased two- to <br />                                                                                      threefold cells shifted 378C, indicating normal <br />                                                                                      stress response occurred (22). <br />                                                                                         To determine different electrophoretic forms <br />                                                                                      produced result presence C-terminal <br />                                                                                      HA epitope, products SCS1 fusion genes were <br />                                                                                      examined. The produced gene [SCS1::HA3(1)] <br />                                                                                      HA3 sequence introduced internal BclI <br />                                                                                      site (see Materials Methods). The second produced <br />                                                                                      gene (SCS1::his6), histidine residues were <br />                                                                                      added C terminus Scs1. The gene did com- <br />                                                                                      plement single copy did expressed multicopy <br />                                                                                      plasmid (data shown). The complemented single <br />                                                                                      copy (Fig. 5a). The protein internal HA epitope was <br />                                                                                      electrophoretically indistinguishable termi- <br />                                                                                      nal HA epitope (data shown), Scs1-His6 protein <br />                                                                                      purified affinity chromatography multiple forms ap- <br />                                                                                      proximately sizes HA epitope fusion proteins <br />                                                                                      (Fig. 5b). It likely multiplicity forms is <br />                                                                                      inherent property Scs1, confirmed <br />                                                                                      antiserum Scs1 itself. <br />                                                                                         The ability SCS1 suppress variety hsp60 mutant <br />                                                                                      alleles indicates direct indirect role Scs1 the <br />    FIG. 5. Analysis Scs1 fusion proteins expressed SCS1-nulla cells. SCS1      process protein folding mitochondria. If Scs1 di- <br /> genes containing C-terminal HA3 epitope [SCS1::HA3(2)] C-terminal             rectly involved Hsp60 function, expect be <br /> hexahistidine sequence (SCS1::his6) (see Materials Methods) intro-          localized mitochondrial matrix inner surface of <br /> duced SCS1-nulla cells CEN plasmid pRS315. (a) Transformed and <br /> nontransformed SCS1-nulla cells grown 388C. (b) Electrophoretic analysis of <br />                                                                                      mitochondrial inner membrane. To test this, cells express- <br /> the fusion proteins. SCS1-nulla cells expressing SCS1::HA3(2) grown          ing single copy SCS1::HA3(2) subjected subcellular <br /> 308C harvested (lane 1) transferred 388C 90 min collected     fractionation. It clear analysis (Fig. 5c) Scs1 <br /> (lane 2). Total proteins directly extracted unbroken cells, separated   did copurify mitochondria recovered almost <br /> SDS-PAGE, transferred filters, immunodecorated anti-HA or <br /> anti-Hsp60 serum described Materials Methods. As control (lane 3), <br />                                                                                      entirely postmitochondrial supernatant fraction. The <br /> proteins nontransformed SCS1-nulla cells grown 308C ana-           small Scs1 copurifying mitochondria could <br /> lyzed. Proteins extracted SCS1-nulla cells expressing SCS1::his6,      indicate small fraction Scs1 normally targeted there. <br /> the Scs1 containing C-terminal histidines purified described Ma-       However, size cytosolic protein, indi- <br /> terials Methods. This protein (lane 4) separated SDS-PAGE and <br /> stained silver. As control, proteins SCS1-nulla cells containing no <br />                                                                                      cating proteolytic processing typical matrix-targeted pro- <br /> plasmid purified, subjected SDS-PAGE, analyzed fash-         teins, consider alternative explanation unlikely cer- <br /> ion (lane 5). (c) Localization Scs1 cellular fractionation. SCS1::HA3(2)-      tainly disproven. The fact Scs1 did copurify with <br /> expressing cells grown overnight 308C. Spheroplasts generated       Hsp60 makes unlikely directly involved in <br /> cellular fractionation carried described Materials Methods. <br /> Electrophoretically separated proteins analyzed presence Scs1 <br />                                                                                      processes mitochondria. <br /> and Hsp60 panel b. Lane 1, total proteins extracted cells      While multiple forms Scs1 evident these <br /> converted spheroplasts; lane 2, initial homogenate disrupted sphero-       fractionation experiments carried out, reproducibly, we <br /> plasts prior centrifugation; lane 3, purified mitochondria; lane 4, postmi-   relative amounts various forms differed <br /> tochondrial supernatant. All lanes contain identical cellular equivalents of <br /> protein. <br />                                                                                      seen proteins directly solubilized from <br />                                                                                      unbroken cells. Until know posttranslational modifi- <br />                                                                                      cations Scs1, explanation these <br />                                                                                      findings. <br /> the epitope-tagged Scs1 fully complemented null allele.                             The subcellular fractionation studies carried out <br /> This finding suggests chimeric protein tar-                         cells grown 308C. At shown proteins <br /> geted normal cellular location fully functional.                       redistributed cell heat shock temperatures <br />    Using monoclonal antibody recognizes HA                               (35), examined case Scs1. <br /> epitope, immunodecorated Western blots total proteins                          Using indirect immunofluorescence, determined cellu- <br /> isolated control cells expressing single copy                    lar location epitope-tagged Scs1 cells grown 30 and <br />  5624       SHU AND HALLBERG                                                                                                                   MOL. CELL. BIOL. <br />  <br />  <br />                                                                                   cline. Thus, normal regulation mitochondrial <br />                                                                                   chaperones apparently requires functional Scs1. By contrast, <br />                                                                                   synthesis major heat shock proteins SCS1-nulla <br />                                                                                   cells 378C qualitatively quantitatively indistinguish- <br />                                                                                   able seen wild-type cells (data shown). Scs1 is <br />                                                                                   clearly required general heat shock response. <br />                                                                                      While effect overproducing Scs1 seen these <br />                                                                                   mRNAs wild-type cells, conceivable the <br />                                                                                   effects overexpression SCS1 different hsp60-ts <br />                                                                                   mutants. Examining expression genes in <br />                                                                                   ts3 cells containing multiple copies SCS1, found <br />                                                                                   case (Fig. 7b). Overexpression SCS1 caused <br />                                                                                   Mge1 Cpn10 mRNA levels higher in <br />                                                                                   wild-type cells 30 388C, appar- <br />                                                                                   ent differential effect relative controls Hsp60 mRNA <br />                                                                                   levels temperature. The level mRNA mitochon- <br />                                                                                   drial Hsp70 (Scs1 [5]), mitochondrial protein chaper- <br />                                                                                   one, appeared slightly elevated cells overproducing <br />                                                                                   Scs1 308C, differences relative controls seen <br />                                                                                   388C. Overexpression SCS1 ts3 cells discernible <br />                                                                                   effects, compared wild-type cells, normal levels of <br />                                                                                   mRNAs large subunit coenzyme QH2-cyto- <br />                                                                                   chrome c reductase (Cor1 [34]), mitochondrial pro- <br />                                                                                   tein, levels cytoplasmic protein actin the <br />                                                                                   large subunits vacuolar ATPase (VAT2 [36]). Although <br />                                                                                   testing admittedly limited array genes, nonetheless, we <br />                                                                                   encoding mitochondrial protein chaper- <br />                                                                                   ones affected Scs1 overproduction. <br />  <br />                                                                                                                  DISCUSSION <br />    FIG. 6. Localization Scs1 immunocytochemistry. The cells ex- <br /> pressing SCS1::HA3(2) analyzed Fig. 5 grown 308C (a b) <br />                                                                                     We identified S. cerevisiae gene, SCS1, which, in <br /> or 388C (c d) prepared immunocytochemical analysis described    high copy number, suppresses <span id='am-394' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-395' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-396' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-397' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-398' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-399' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-400' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-401' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-402' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-403' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-404' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-405' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-406' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-407' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-408' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-409' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-410' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-411' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-412' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-413' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-414' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-415' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-416' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-417' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-418' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-419' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-420' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-421' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-422' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-423' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-424' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-425' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-426' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-427' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-428' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-429' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-430' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-431' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-432' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-433' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-434' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-435' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-436' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-437' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-438' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-439' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-440' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-441' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-442' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-443' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-444' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-445' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-446' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-447' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-448' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-449' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-450' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-451' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-452' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-453' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-454' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-455' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-456' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-457' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-458' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-459' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-460' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-461' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-462' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-463' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-464' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-465' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-466' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-467' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-468' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-469' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-470' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-471' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span>lethal phenotype</span> num- <br /> Materials Methods. Fixed cells treated 49,6-diamidino-2-       ber hsp60-ts alleles. As essentially previous studies <br /> phenylindole (a, c, e) fluorescein isothiocyanate-conjugated goat anti-   place Hsp60 solely mitochondrial matrix (11, 12, 22), <br /> mouse antibody (b, d, f). Control cells (e f), nontransformed SCS1- <br /> nulla strain, grown 308C. The arrowheads indicate positions of <br /> nuclei comparable panels. <br />  <br />  <br />  <br />  <br /> 388C (Fig. 6). At temperature, Scs1 located the <br /> cytosol excluded nucleus. If changes Scs1 <br /> cellular location occur result increased tempera- <br /> tures, detectable level analysis. <br />    Effects deletion overexpression SCS1 ex- <br /> pression genes. Given SCS1-null cells ts for <br /> growth Scs1 levels unaffected temperature, as <br /> Scs1 indirectly affect Hsp60 function, wondered <br /> whether Scs1 involved regulating levels other <br /> mitochondrial chaperones. We examined effect the <br /> absence Scs1 levels mRNAs mito- <br /> chondrial chaperones, Hsp60, Cpn10 (28), mitochon- <br /> drial GrpE homolog, Mge1 (2), compared mRNA <br /> levels wild-type cells wild-type cells overexpressing <br /> SCS1. From results shown Fig. 7a, concluded several <br /> things. First, agreement Western analysis, levels <br /> of expression SCS1 significantly different wild- <br /> type cells 30 388C. Second, overexpression SCS1                         FIG. 7. Northern analyses cells expressing different numbers SCS1 <br /> control cells (i.e., non-Hsp60 mutant strains) discern-                    genes. RNA isolated cells grown 308C grown 308C <br />                                                                                   shifted 388C 30 min. Northern analyses carried as <br /> ible effect levels mRNAs produced                     described Materials Methods, using DNA probes encoding portions the <br /> genes temperature, HSP60 CPN10 showing                         genes SCS1, HSP60, CPN10, MGE1, SSC1, COR1, VAT2, actin gene. <br /> their normal temperature inducibility (22, 28). However,                      (a) Sources RNA: SCS1-nulla cells (lanes 1 4), W303 cells (lanes 2 5), <br /> absence Scs1 brought depression mRNA level                      W303 cells transformed YEpSCS1 (lane 3 6). Lanes 1 3, cells <br />                                                                                   grown 308C; lanes 4 6, cells grown 308C shifted 388C. (b) <br /> encoding Mge1 308C. More dramatically, genes,                    Sources RNA: ts3 cells (lanes 1 3) ts3 cells transformed the <br /> little elevation mRNA levels 388C                    plasmid YEpSCS1. Lanes 1 2, cells grown 308C; lanes 3 4, cells grown <br /> SCS1-nulla cells. For Hsp60 mRNA, actually de-                        308C shifted 388C. <br />  VOL. 15, 1995                                                                                  CHARACTERIZATION OF SCS1                         5625 <br />  <br />  <br /> finding Scs1 mitochondrial protein totally        null cells permissive nonpermissive temperatures. <br /> unexpected. However, observation consistent        Thus, stress-inducible gene encoding mitochondri- <br /> fact overexpressed SCS1 complement                 on-targeted protein requires Scs1 correct expression. From <br /> HSP60-null allele did suppress replacing        data available far, predict Scs1 in- <br /> (i.e., bypassing) Hsp60 function. That is, overexpressed Scs1        volved signaling pathway involving expression of <br /> must, manner, alleviate misfunctioning func-          mitochondrion-targeted proteins stress conditions. <br /> tionally abnormal Hsp60s. We considered              While SCS1 essential gene required for <br /> achieved high levels Scs1 elevated Hsp60 levels,       nonfermentative growth, absolutely required growth <br /> increasing overall activity Hsp60s par-     survival carbon sources higher temperatures. It is <br /> tially functional nonpermissive temperatures. A direct test       necessary normal resistance elevated levels eth- <br /> of hypothesis proved untenable. As       anol 308C. The levels Scs1 normally elevated at <br /> apparent quantitative qualitative differences Hsp60s       higher temperatures, indication tran- <br /> in suppressed (or unsuppressed) mutant strains containing        scription SCS1 altered. As SCS1-null strain grows <br /> plasmid YEpSCS1, restoration Hsp60 function            slowly media 308C, Scs1 function both <br /> likely occur functional physical interaction    normal elevated temperatures. This conclusion sup- <br /> with mitochondrial proteins levels           ported finding levels Cpn10 Mge1 <br /> and/or activities affected Scs1 overproduction.              mRNAs considerably reduced SCS1-null cells 308C. <br />    The mutations alleles suppressed Scs1 overproduc-       While proven preventing normal tem- <br /> tion scattered carboxyl-terminal one-half      perature-induced elevation molecular chaperones <br /> Hsp60. It be, however, domains containing         Hsp60, Cpn10, Mge1 necessarily lethal, certainly <br /> affected amino acids Hsp60 monomer share common             likely so. If case, account ts <br /> site normally folded native complex. As altered       lethality SCS1-null cells. <br /> amino acid residues mutant Hsp60s conserved           We know accounts anomalous elec- <br /> E. coli chaperonin GroEL (13, 25), located homologous         trophoretic patterns Scs1 immunodeco- <br /> sites recently published X-ray crystal structure GroEL     rating proteins expressed SCS1-HA3 gene. While <br /> (3). The mutated residues localized all, occurring       SCS1 fusion genes expressed produce heteroge- <br /> in different regions well-separated domains            neous arrays proteins similar sizes patterns, we <br /> GroEL monomer. These observations suggest in-           conclude normal situation have <br /> directly affecting Hsp60 function, Scs1 act num-       produced antibody normal Scs1. We did attempt <br /> ber different intermediaries, single one.                   ascertain various electrophoretic forms <br />    The likely known candidates direct indirect         phosphorylated, glycosylated, ubiquitinated. While <br /> functional interaction Hsp60 Cpn10 Mge1 (10,            analyses exhaustive carried on <br /> 20). Cpn10 physically interacts Hsp60 functions        impure proteins, obtained evidence these <br /> cochaperone (10, 15). Similarly, Mge1 functions protein         posttranslational modifications accounted way the <br /> chaperone upstream Hsp60 protein folding pathway           various electrophoretic isoforms. We noted (data not <br /> of imported mitochondrial matrix proteins (2, 19, 21). Scs1-         shown) HA epitope-tagged Scs1 highly unstable in <br /> induced overexpression other, both,        vivo. Determination rapid turnover intrinsic <br /> genes reasonably account suppressibility          Scs1 itself induced presence HA epitope <br /> number different mutant strains. One prediction        require availability proper antibodies. <br /> there hsp60-ts mutants suppressible overproduction a <br /> single molecular chaperone, identifiable.                                ACKNOWLEDGMENTS <br /> We begun analyses using gene encoding the <br /> cochaperonin Cpn10. Thus far, analyzing mutant                  We especially thank Rich Zitomer discussing unpublished <br />                                                                      data regarding ROX3 suppressor. We thank Liz Hallberg for <br /> strains, suppressible Scs1          helping produce characterize strains used this <br /> overproduction suppressible Cpn10 overexpres-            study. Haifeng Yang carried growth experiments with <br /> sion. Another prediction similar effects    SCS1-null cells. The individuals named parentheses kindly sup- <br /> overexpressing Scs1 MGE1 CPN10 expression            plied cloned copies following yeast genes: actin (Bob West), <br /> hsp60-ts mutant strains. We examined          COR1 (Mary Crivellone), VAT2 (Patty Kane), SSC1 (Betty Craig), and <br /> strains true.                                 MGE1 CPN10 (Sabine Rospert). <br />    While evolutionary conservation SCS1 reasonably high,          This work supported NIH grant GM-46302 R.L.H. <br /> no data regarding functions homologs higher <br /> plants animals published. The comparison                                          ADDENDUM <br /> available present SCS1 previously identified a <br /> multicopy suppressor <span id='am-316' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-317' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-318' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-319' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-320' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-321' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-322' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-323' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-324' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-325' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-326' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-327' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-328' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-330' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-331' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-332' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-333' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-334' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-335' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-337' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-338' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-339' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-341' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-342' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-344' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-347' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-348' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-349' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-350' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-351' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-352' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-353' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-354' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-355' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-356' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-357' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-358' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-359' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-362' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-363' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-364' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-365' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-366' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-367' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-368' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-369' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-370' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-372' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-373' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-374' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-375' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-376' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-378' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-380' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-381' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-382' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-383' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-384' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-385' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-386' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-388' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-389' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-390' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-391' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-392' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-393' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span>mutation</span> essential gene ROX3          Three different complete human cDNAs homologs <br /> (27). ROX3 encodes nuclear regulatory factor          SCS1 cloned. They shown encode <br /> way controls, things, production mito-        regulatory subunits (B) protein phosphatase 2A (21a). <br /> chondrial protein iso-2-cytochrome c. It noteworthy      While direct evidence Scs1 same <br /> gene encoding protein, CYC7, normally up-regulated        function S. cerevisiae, observation consistent the <br /> heat shock temperatures degree similar seen         fact overproduction regulatory subunit of <br /> other heat-inducible genes (24). Whether ROX3 involved         yeast Cdc28 kinase (Clb2) suppresses ts phenotype of <br /> regulating genes encoding mitochondrion-targeted pro-          SCS1-null strain. <br /> teins normal heat shock temperatures been <br />                                                                                                        REFERENCES <br /> reported. We predict might. How Scs1 overpro- <br />                                                                       1. Ausubel, F. M., R. Brent, R. E. Kingston, D. D. Moore, J. A. Smith, J. G. <br /> duction suppresses ROX3 mutant                 Seidman, K. Struhl (ed.). 1989. Current protocols molecular biology. <br /> direct indirect effect known. Preliminary data (37)       John Wiley &amp; Sons, New York. <br /> indicate expression CYC7 altered SCS1-         2. Bolliger, L., O. Deloche, B. S. Glick, C. Georgopoulos, P. Jeno, N. Kronidou, <br />  5626         SHU AND HALLBERG                                                                                                                       MOL. CELL. BIOL. <br />  <br />  <br />       M. Horst, N. Morishima, G. Schatz. 1994. A mitochondrial homolog              1992. Successive action DnaK, DnaJ, GroEL pathway of <br />       bacterial GrpE interacts mitochondrial hsp70 essential via-          chaperone-mediated protein folding. Nature (London) 356:683&#226;&#8364;&#8220;689. <br />       bility. EMBO J. 13:1998&#226;&#8364;&#8220;2006.                                                    21. Manning-Krieg, U., P. E. Scherer, G. Schatz. 1991. Sequential action of <br />  3.   Braig, K., Z. Otwinowski, R. Hegde, D. C. Boisvert, A. Joachimiak, A. L.             mitochondrial chaperonins protein import matrix. EMBO J. 10: <br />       Horwich, P. B. Sigler. 1994. The crystal structure bacterial chap-        3273&#226;&#8364;&#8220;3280. <br />       eronin GroEL 2.8 &#195;&#8230;. Nature (London) 371:578&#226;&#8364;&#8220;586.                              21a.McCright, B., D. M. Virshup. Submitted publication. <br />  4.   Cheng, M. Y., F.-U. Hartl, J. Martin, R. A. Pollock, F. Kalousek, W. Neupert,    22. McMullin, T. W., R. L. Hallberg. 1988. A highly evolutionarily con- <br />       E. M. Hallberg, R. L. Hallberg, A. L. Horwich. 1989. Mitochondrial heat          served mitochondrial protein structurally related protein encoded by <br />       shock protein hsp60 essential assembly proteins imported yeast        Escherichia coli groEL gene. Mol. Cell. Biol. 8:371&#226;&#8364;&#8220;380. <br />       mitochondria. Nature (London) 337:620&#226;&#8364;&#8220;625.                                       23. Ostermann, J., A. L. Horwich, W. Neupert, F.-U. Hartl. 1989. Protein <br />  5.   Craig, E. A., J. Kramern, J. Shilling, M. Werner-Washburne, S. Holmes, J.            folding mitochondria requires complex formation hsp60 ATP <br />       Kosic-Smithers, C. M. Nicolet. 1989. SSC1, essential member            hydrolysis. Nature (London) 341:125&#226;&#8364;&#8220;130. <br />       yeast HSP70 multigene family, encodes mitochondrial protein. Mol. Cell.        24. Pillar, T. M., R. E. Bradshaw. 1991. Heat shock stationary phase <br />       Biol. 9:3000&#226;&#8364;&#8220;3008.                                                                   induce trans <br /> </body></html>