<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br />    A Lesion in the DNA Replication Initiation Factor Mcm10 Induces <br />           Pausing of Elongation Forks through Chromosomal <br />              Replication Origins in Saccharomyces cerevisiae <br />  <br />  <br />            We describe a new minichromosome maintenance factor, Mcm10, and show that this essential protein is <br />          involved in the initiation of DNA replication in Saccharomyces cerevisiae. The mcm10 mutant has an autono- <br />          mously replicating sequence-specific minichromosome maintenance defect and arrests at the nonpermissive <br />          temperature with dumbbell morphology and 2C DNA content. Mcm10 is a nuclear protein that physically <br />          interacts with several members of the MCM2-7 family of DNA replication initiation factors. Cloning and <br />          sequencing of the MCM10 gene show that it is identical <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> DNA43, a gene identified independently for its <br />          putative role in replicating DNA. Two-dimensional DNA gel analysis reveals that the mcm10-1 lesion causes a <br />          dramatic reduction in DNA replication initiation at chromosomal origins, including ORI1 and ORI121. <br />          Interestingly, the mcm10-1 lesion also causes replication forks to pause during elongation through these same <br />          loci. This novel <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span>phenotype</span> suggests a unique role for the Mcm10 protein in the initiation of DNA synthesis at <br />          replication origins. <br />  <br />  <br />    DNA replication is a fundamental process of all dividing                large complexes (35). Despite their structural similarity, each <br /> cells. During the eukaryotic cell cycle, DNA replication occurs            of these proteins is indispensable for cell viability in S. cerevi- <br /> exactly once and is initiated only upon the completion of mi-              siae (26, 28, 54), consistent with the idea that each of these six <br /> tosis. Strict regulation of timing appears to be mediated                  proteins is a subunit of a larger complex that is essential for the <br /> through the coordinated activities of numerous proteins. Due               initiation of DNA synthesis. Although there is no evidence for <br /> in large part to studies of viral DNA replication, the enzymatic           the direct interaction between the MCM2-7 protein complexes <br /> activities at the replication fork were elucidated many years              and replication origins, these proteins have been shown to bind <br /> ago (33). In contrast, trans-acting factors which regulate the             chromatin prior to the onset of DNA synthesis and to be <br /> initiation of DNA replication have been described only re-                 released <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> chromatin after the onset of DNA synthesis <br /> cently. In an effort to gain a comprehensive understanding of              (56). Furthermore, a defect in any one of these proteins results <br /> the factors involved in this essential process, we have sought to          in a dramatic reduction in the frequency of initiation events at <br /> identify gene products that influence the initiation of DNA                replication origins (55). <br /> replication.                                                                  The mcm mutants affect the maintenance of minichromo- <br />    Saccharomyces cerevisiae provides an excellent eukaryotic               somes in a manner that is dependent on the exact nature of the <br /> model for identifying proteins involved in DNA replication.                ARS: the activities of different ARSs, as measured by plasmid <br /> Many replication initiation factors currently under investiga-             stability, are affected to different extents in a given mcm mu- <br /> tion are conserved in mammalian cells and were found initially             tant. This mutant phenotype is referred to as the ARS-specific <br /> in yeast. Members of the origin recognition complex (ORC)                  Mcm2 defect and appears to be unique to DNA replication <br /> (20, 22) were originally identified biochemically through their            mutants defective in the initiation step of DNA synthesis. All <br /> binding to the consensus sequence of autonomously replicating              mcm mutants are characterized by an ARS-specific Mcm2 <br /> sequences (ARSs), which function as DNA replication origins                defect (21, 39, 54). Strains mutated in Abf1 or in the essential <br /> (2). Our screen for yeast minichromosome maintenance                       subunits of the ORC also display an ARS-specific Mcm2 phe- <br /> (mcm) mutants (39) has also been fruitful in identifying repli-            notype (19, 47). Interestingly, ARSs that are exquisitely sensi- <br /> cation initiation factors, such as those of the MCM2-7 family              tive to the mcm mutations appear to be relatively insensitive to <br /> (9, 29, 52). The MCM2-7 proteins are a family of six conserved             the abf1 mutations and vice versa, suggesting that individual <br /> proteins that are ubiquitous in eukaryotes. Their essential role           origins have customized regulatory mechanisms for origin us- <br /> in the initiation of DNA synthesis not only has been demon-                age (39, 47). These observations also suggest that the defined <br /> strated by in vivo studies in a number of organisms, including             patterns of ARS specificity exhibited by different replication <br /> S. cerevisiae (55), Schizosaccharomyces pombe (40), and Dro-               initiation mutants reflect their involvement in independent <br /> sophila melanogaster (51), but also is supported by in vitro               mechanisms that regulate the initiation of DNA synthesis at <br /> studies in Xenopus laevis (8, 34, 38). The MCM2-7 proteins                 replication origins. <br /> interact with one another, and possibly other proteins, to form               The screen for minichromosome maintenance-defective mu- <br />                                                                            tants has also uncovered other genes that may play a role in the <br />                                                                            regulation of DNA replication (39). For example, MCM1 en- <br />   * Corresponding author. <br />                                                                            codes a global transcription factor that regulates the expression <br />   &#226;&#8364;&#160; Present address: Department of Molecular Oncology, Genentech,          of diverse genes (45, 46). While it is likely that Mcm1 regulates <br /> Inc., South San Francisco, CA 94080.                                       DNA synthesis by modulating the expression of DNA replica- <br />   &#226;&#8364;&#161; Present address: Howard Hughes Medical Research Institute,             tion genes, as suggested by the putative Mcm1 binding <br /> Stanford University Medical School, Stanford, CA 94305.                    sites found upstream of MCM3, CDC46/MCM5, and CDC47/ <br />  <br />                                                                     3261 <br />  3262       MERCHANT ET AL.                                                                                                                      MOL. CELL. BIOL. <br />  <br />  <br />                  TABLE 1. Plasmids used in this study                               tion bubbles during extraction from cells, comparisons between ORI1 and <br />                                                                                     ORI121 were made by using a single DNA preparation. DNA from each prep- <br />                                                                      Source or      aration was analyzed in at least two separate 2D gels to assess the consistency of <br />      Plasmid                        Description <br />                                                                      reference      results. For analysis of ORI1, genomic DNA was digested to completion with <br />                                                                                     NcoI or EcoRI. For ORI121, BamHI and EcoRI were used. Typically, between <br /> pYCMCM10             Original clone containing MCM10               6                30 and 50 mg of DNA was loaded in each gel. Probes for detecting ORI1 or <br /> pBTM116              2mm TRP1-lexA(1-202)                          S. Fields        ORI121 were prepared by isolating the NcoI or EcoRI fragment of ARS1 or the <br /> pGAD2F               2mm LEU2 GAL4(768-881)                        S. Fields        BamHI-EcoRI fragment of ARS121, followed by radiolabelling (16). Radiola- <br /> pBTM.MCM2            pBTM with LEXA-MCM2 gene fusion               35               belled ARS1 or ARS121 DNA probe (12.5 3 106 cpm) was added per 10 ml of <br /> pBTM.MCM3            pBTM with LEXA-MCM3 gene fusion               35               hybridization solution and allowed to hybridize for 24 h at 42&#194;&#176;C. For better <br />                                                                                     separation of Y-form and linear DNA of smaller-molecular-size fragments (,1.5 <br /> pBTM.CDC46           pBTM with LEXA-CDC46 gene fusion              35 <br />                                                                                     kb), the second-dimension gel was run at an angle of 95 to 100&#194;&#176; to the direction <br /> pBTM.MCM6            pBTM with LEXA-MCM6 gene fusion               Y. Kawasaki      of electrophoresis at a voltage of 5 V/cm for 6 h with a 2% agarose gel (31). <br /> pBTM.CDC47           pBTM with LEXA-CDC47 gene fusion              44                  Preparation of anti-Mcm10 antiserum. Mcm10 was purified for antibody pro- <br /> pBTM.CDC54           pBTM with LEXA-CDC54 gene fusion              Y. Kawasaki      duction by expression of plasmid pMCM10.His in E. coli. In this plasmid, six <br /> pGAD.MCM10           pGAD2F with GAL4-MCM10 gene                   This study       histidine residues replace the first 54 amino acids of Mcm10. Expression from the <br />                        fusion                                                       lacZ promoter produces a histidine-tagged protein, designated Mcm10.His. <br /> YCp101               ARS1 LEU2 CEN5                                54               Mcm10.His was expressed at high levels in E. coli JM109 and purified by means <br /> YCp120               ARS120 URA3 LEU2 CEN5                         54               of immobilized metal affinity chromatography over an Ni21-nitrilotriacetic acid <br />                                                                                     resin column (Qiagen, Inc., Chatsworth, Calif.). Anti-Mcm10 antiserum was <br /> YCp121               ARS121 LEU2 CEN5                              54 <br />                                                                                     raised in two rabbits injected three times at 2-week intervals with 0.2 mg of <br /> YCp131               ARS131 LEU2 CEN5                              54               Mcm10.His. Antisera were collected and stored at 220&#194;&#176;C. Affinity purification of <br /> YCpH2B               ARSH2B URA3 LEU2 CEN5                         54               the anti-Mcm10 antisera was achieved by incubating crude antisera with <br /> YCpHML               ARSHMLa URA3 LEU2 CEN5                        54               Mcm10.His immobilized on an Immobilon-P membrane (Millipore Corp., Bed- <br /> YCpHO                ARSHO URA3 CEN5                               54               ford, Mass.). The resulting membrane was incubated for 16 h at 4&#194;&#176;C with a 20% <br /> pQE31                ampR; parent vector for pMCM10.His            Qiagen           solution of crude antiserum in a mixture of 150 mM NaCl, 10 mM Tris (pH 8.0), <br /> pMCM10.His           pQE with Mcm10.His coding sequence            This study       1 mM EDTA, 1% lowfat milk, and 0.1% Tween 20. The Mcm10-specific anti- <br /> pEG(KT)              2mm URA3 GAL1-GST                             D. Mitchell      bodies were eluted with 100 mM glycine, pH 2.5, followed by neutralization to <br />                                                                                     pH 8.0 with Tris buffer. This purification procedure was repeated three times in <br /> pEG(KT)MCM2          2mm URA3 GAL1-GST-MCM2                        35 <br />                                                                                     succession to produce the purified anti-Mcm10 antibodies used in subsequent <br /> pEG(KT)MCM3          2mm URA3 GAL1-GST-MCM3                        35               studies. <br /> pEG(KT)MCM4          2mm URA3 GAL1-GST-CDC54                       Y. Kawasaki         Western blot analysis. Proteins from strain BJ2168 were extracted by glass <br /> pEG(KT)MCM5          2mm URA3 GAL1-GST-CDC46                       Y. Kawasaki      bead lysis in the presence of protease inhibitors and separated by molecular <br /> pEG(KT)MCM6          2mm URA3 GAL1-GST-MCM6                        Y. Kawasaki      weight on polyacrylamide gels. Proteins were transferred to Immobilon-P <br /> pEG(KT)MCM7          2mm URA3 GAL1-GST-CDC47                       Y. Kawasaki      membranes and incubated with purified anti-Mcm10 antibodies as described <br />                                                                                     elsewhere (25). Affinity-purified goat anti-rabbit immunoglobulin G (IgG)&#226;&#8364;&#8220;horse- <br />                                                                                     radish peroxidase conjugate was obtained from Bio-Rad Laboratories (Rich- <br />                                                                                     mond, Calif.) and visualized with Renaissance chemiluminescence reagents (Du- <br />                                                                                     Pont, Boston, Mass.). <br /> MCM7, direct evidence for Mcm1 playing such a role is elusive                          Flow cytometry. Yeast cells were grown at 23&#194;&#176;C and shifted to 37&#194;&#176;C for 4 h. <br /> (1). In the current study, we have begun to elucidate the func-                     The final concentration of cells was 1.5 3 108 (wild type) and 8 3 107 (mcm10-1 <br /> tion of a new minichromosome maintenance factor, Mcm10.                             mutant). Cells were stained with propidium iodide as published previously (27). <br />                                                                                        Indirect immunofluorescence. Indirect immunofluorescence was carried out <br /> We have characterized the mcm10 mutant phenotype and com-                           by a modification of the Kilmartin and Adams method (30). Primary antibodies <br /> pared it to that of other replication initiation mutants. We have                   used were affinity-purified rabbit anti-Mcm10 antibodies and rat anti-yeast tu- <br /> cloned and analyzed the MCM10 gene sequence and studied                             bulin antibodies (Yol1/34; John Kilmartin, Cambridge, United Kingdom). Sec- <br /> the interaction of Mcm10 with members of the MCM2-7 pro-                            ondary antibodies used were fluorescein isothiocyanate-conjugated goat anti-rat <br />                                                                                     IgG antibodies (Cappel, Durham, N.C.) and rhodamine-conjugated goat anti- <br /> tein family. Finally, we have examined chromosomal replica-                         rabbit IgG antibodies (Boehringer Mannheim, Indianapolis, Ind.). <br /> tion origin usage to find direct evidence of a DNA replication                         Two-hybrid analysis. PCR-amplified MCM10 gene was cloned into pGAD2F <br /> defect in the mcm10 mutant.                                                         (18) to allow production of the Gal4-Mcm10 fusion protein for two-hybrid <br />                                                                                     analysis. pGAD.MCM10 complements the temperature sensitivity of the mcm10- <br />                                                                                     1 mutant. Other two-hybrid system plasmids have been described elsewhere (35). <br />                         MATERIALS AND METHODS                                       Plasmids were transformed into yeast strain EGY40 and selected on medium <br />    Strains and plasmids. Escherichia coli JM109 [recA1 supE44 endA1 hsdR17          lacking tryptophan, uracil, and leucine. Colonies thus isolated were patched onto <br /> gyrA96 relA1 thiD(lac-proAB) F9 (traD36 proAB1 lacIq lacZDM15)] was used to         medium lacking the same amino acids and containing X-Gal (5-bromo-4-chloro- <br /> propagate plasmids and was used to express the Mcm10.His protein. E. coli           3-indolyl-b-D-galactopyranoside). <br /> JM101 [supE thiD(lac-proAB) F9 (traD36 proAB1 lacIq lacZDM15)] was used to             GST fusion protein affinity chromatography. Glutathione S-transferase (GST) <br /> propagate minichromosomes. Yeast strains used in this study include 8534-8C         and GST-Mcm fusion proteins expressed in yeast were conjugated to glutathio- <br /> (MATa ura3-52 leu2-3,112 his4-34), mcm10-2C (MATa his3 his4 leu2-3,112              ne-Sepharose 4B (Pharmacia) as described elsewhere (35). The protein-conju- <br /> ura3-52 mcm10-1), mcm10-1D (MATa leu2-3,112 his mcm10-1), mcm10-14                  gated beads were incubated with yeast protein extracts at 4&#194;&#176;C for 2 h with gentle <br /> (MATa lys2-801 his mcm10-1), EGY40 (MATa ura3-52 trp1 his3 leu2), and               shaking. The beads were then washed four times with 10 ml of ice-cold buffer <br /> BJ2168 (MATa pep4-3 prc1-407 prb-1122 ura3-52 trp1 leu2). The mcm10-1 mu-           containing 0.1 M NaCl. Bound proteins were eluted from the beads with 500 ml <br /> tation was isolated by Richard Surosky as described by Gibson et al. (21).          of ice-cold elution buffer containing 0.5 M NaCl. Eluates were analyzed on <br /> Plasmids used in this study are described in Table 1.                               Western blots with anti-Mcm10 antibodies. <br />    Media, chemicals, and enzymes. Rich (yeast extract-peptone-dextrose) and <br /> complete media were prepared as described elsewhere (24). Plates containing <br /> 5-fluoroorotic acid were made according to the method of Boeke et al. (3). DNA <br />                                                                                                                       RESULTS <br /> sequencing was performed with the sequencing kit from United States Biochem- <br /> ical Corp. (Cleveland, Ohio). Restriction enzymes were purchased from New <br />                                                                                        Mcm10 is required for stable plasmid maintenance. We <br /> England Biolabs (Beverly, Mass.) or Bethesda Research Laboratories Life Tech-       have isolated several DNA replication initiation mutants which <br /> nologies, Inc. (Gaithersburg, Md.). DNA primers were purchased from Ransom          lose minichromosomes in an ARS-specific manner. In a given <br /> Hill Bioscience, Inc. (Ramona, Calif.). Random hexanucleotides and deoxyribo-       mcm mutant, plasmids with certain ARSs have high loss rates <br /> nucleotides were purchased from Pharmacia (Uppsala, Sweden). <br />    Plasmid loss rates. Yeast strains were transformed with the appropriate plas- <br />                                                                                     while other ARSs allow maintenance of the minichromosome <br /> mids (15). Mitotic plasmid stability assays were performed as described previ-      at rates close to that of an isogenic wild-type strain (39, 54). To <br /> ously (21) except that growth in nonselective media occurred over 10 to 15          elucidate the <span id='am-160' about='xsp:pattern' typeof='owl:Thing'>pattern</span> of minichromosome maintenance in the <br /> generations. Loss rates per generation were determined by the formula 1 2           mcm10 mutant, we determined loss rates in the mutant strain <br /> (F/I)1/N, where I is the initial percentage of plasmid-bearing cells and F is the <br /> percentage of plasmid-bearing cells following growth over N generations. <br />                                                                                     mcm10-2C at room temperature (approximately 23&#194;&#176;C) and at <br />    2D DNA gels. Two-dimensional (2D) DNA gel electrophoresis was performed          30&#194;&#176;C. The results in Fig. 1 indicate that at room temperature <br /> according to the neutral-neutral method (4). To control for shearing of replica-    the minichromosome maintenance phenotype of the mcm10 <br />  VOL. 17, 1997                                                        Mcm10 AND ELONGATION FORK PAUSING IN S. CEREVISIAE                                    3263 <br />  <br />  <br />  <br />  <br />   FIG. 1. Minichromosome loss rates in wild-type (8534-8C) and mcm10-1 (mcm10-2C) strains at 23 and 30&#194;&#176;C. The maximum theoretical loss rate is 0.5 per <br /> generation. The values are averages derived with two independent transformants and were obtained in three or four independent experiments. Measurements that have <br /> negligible variations from experiment to experiment are shown without error bars. <br />  <br />  <br />  <br /> mutant strain resembles that of the wild type. While the loss                      mcm10 mutant, in addition to cells containing either 1C or 2C <br /> rates appear to be slightly elevated relative to wild-type rates,                  DNA, a significant portion of the cell population has a DNA <br /> there is little variation in loss rates of plasmids with different                 content between 1C and 2C and is therefore undergoing DNA <br /> ARSs. At 30&#194;&#176;C, however, a striking, ARS-specific pattern of                        replication (Fig. 3). This result suggests that at 30&#194;&#176;C, mcm10 <br /> minichromosome maintenance emerges. Loss rates of all mini-                        cells have a prolonged S phase which presumably contributes <br /> chromosomes increase in the mcm10 mutant. Moreover, plas-                          to the observed 60% increase in the <span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span> of the cell cycle. <br /> mids containing some ARSs, such as ARS1, are lost at greater                       After 4 h at 37&#194;&#176;C, the wild-type strain shows a normal distri- <br /> rates than plasmids containing other ARSs, such as ARS121.                         bution of cells in different stages of the cell cycle (Fig. 3). In <br /> The ARS-specific Mcm2 phenotype suggests that the mcm10-1                          contrast, few of the mcm10 mutant cells contain 1C DNA; <br /> <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span>mutation</span> affects DNA replication initiation at ARSs on the                         instead, the vast majority of cells show a nuclear DNA content <br /> minichromosomes.                                                                   of approximately 2C DNA (Fig. 3). A significant portion of the <br />    mcm10 mutant cells have a cell division cycle arrest pheno-                     cell population has a DNA content of slightly greater than 2C. <br /> type at the restrictive temperature. Growth of the mcm10                           Such a population of cells is expected if mitochondrial DNA <br /> mutant strain is normal at 12 and 23&#194;&#176;C. At 30&#194;&#176;C, the mutant                        continues to replicate after cessation of chromosomal DNA <br /> strain shows a slight growth defect, with a doubling time of 120                   replication at the nonpermissive temperature. <br /> min compared to 75 min in the wild-type strain. At 37&#194;&#176;C, while                        The mcm10-1 mutant is allelic to another DNA replication <br /> the wild-type strain forms colonies in 2 days, the mcm10 mu-                       mutant, the dna43-1 mutant. MCM10 was cloned by comple- <br /> tant fails to grow (data not shown).                                               mentation of the temperature-sensitive phenotype of mcm10-1. A <br />    To investigate a possible cell division cycle arrest phenotype                  YCp50-based yeast genomic library (48) was used to transform <br /> of the mcm10 strain, we examined the morphology and the                            the mcm10-1 strain, and viable transformants were selected on <br /> position of 49,6-diamidino-2-phenylindole (DAPI)-stained nu-                       chloramphenicol-uracil medium at 37&#194;&#176;C. A dozen viable trans- <br /> clei of mutant and wild-type cells microscopically. At 23&#194;&#176;C,                       formants grew up among approximately 17,400 transformants <br /> mcm10 mutant cells in all stages of the cell cycle were observed                   after 5 days of growth at 37&#194;&#176;C. The plasmid DNA that comple- <br /> (data not shown), consistent with the normal growth rate ob-                       mented the temperature-sensitive phenotype of the mcm10-1 <br /> served in the mutant at the permissive temperature. After 4 h                      strain was recovered from four of the viable transformants. All <br /> at 37&#194;&#176;C, however, the mcm10 mutant strain has arrested pre-                        four plasmids contained an identical 7.6-kb insert in the YCp50 <br /> mitotically; over 95% of cells contain a large bud and a single                    vector. Subcloning of the 7.6-kb insert indicated that a 2.2-kb <br /> nucleus (Fig. 2C and D). The isogenic wild-type strain grows                       KpnI-XbaI fragment complemented the temperature-sensitive <br /> well at 37&#194;&#176;C, as revealed by the presence of cells in all stages of                phenotype of mcm10-1. <br /> the cell cycle (Fig. 2A and B).                                                       Sequencing of the 2.2-kb KpnI-XbaI fragment indicated that <br />    We also assessed the status of nuclear DNA replication in                       it contains an open reading frame that had already been iden- <br /> arrested mutant cells. Nuclear DNA was stained with pro-                           tified as DNA43 in an independent screen for DNA replication <br /> pidium iodide. Fluorescence-activated cell sorting analysis of                     mutants (49). DNA43 is located on chromosome IX, 32 centi- <br /> stained cells shows that at the permissive growing temperature                     morgans distal to the centromere from his5. Significantly, So- <br /> of 23&#194;&#176;C, both wild-type (8534-8C) and mcm10 mutant cells are                       lomon and colleagues (49) showed that at the restrictive tem- <br /> present in two main populations (Fig. 3). One population of                        perature, while RNA and protein synthesis continues in the <br /> cells contains 1C DNA and is therefore in the G1 phase of the                      dna43-1 mutant cells, DNA synthesis ceases after one genera- <br /> cell cycle. The other has 2C DNA content and is therefore                          tion. Additionally, they showed that DNA43 encodes an essen- <br /> completing S phase or undergoing mitosis. At 30&#194;&#176;C, the wild-                       tial gene product. Both dna43-1 and mcm10-1 are recessive <br /> type cells contain mostly either 1C or 2C DNA (Fig. 3). In the                     conditional lethal mutations. Genetic analysis indicates that <br />  3264       MERCHANT ET AL.                                                                                                                    MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />    FIG. 2. The mcm10-1 mutant arrests premitotically at 37&#194;&#176;C. Wild-type (8534-8C [A and B]) and mcm10-1 mutant (mcm10-2C [C and D]) strains were grown at 23&#194;&#176;C <br /> for several hours until early log phase and then shifted to 37&#194;&#176;C for 4 h. (A and C) Nomarski microscopy reveals cell morphology. (B and D) DAPI staining of the DNA <br /> reveals the morphology and location of the nucleus. <br />  <br />  <br />  <br />  <br /> mcm10-1 and dna43-1 are noncomplementing alleles of the                             oride filter. These purified antibodies react predominantly with <br /> same gene. Tetrads of mcm10/dna43 diploids comprise four                            a single yeast protein of approximately 70 kDa, close to the <br /> heat-sensitive spores. A search in the SGD database found that                      predicted molecular mass of 65.8 kDa of Mcm10 (Fig. 4, lane <br /> MCM10/DNA43 encodes a conceptual protein of 571 amino                               2). To verify that the single band is indeed Mcm10, Mcm10 was <br /> acids (open reading frame product YIL150c) with a predicted                         overproduced by expression from the Gal1-10 promoter in <br /> molecular mass of 65.8 kDa and an isoelectric point of 9.7, in                      yeast cells. Immunoblotting analysis shows that an overpro- <br /> contrast to the 59.6-kDa molecular mass reported by Solomon                         duced 70-kDa Mcm10 protein was indeed recognized by the <br /> et al. No significant homologies were detected between Mcm10                        purified antibodies (data not shown). <br /> and any other sequences in the database.                                              The location of the Mcm10 protein in different stages of the <br />    Mcm10 is localized in the nucleus throughout the cell cycle.                     cell cycle was determined by indirect immunofluorescence of <br /> Antiserum from rabbits injected with the histidine-tagged                           Mcm10 in a population of asynchronously growing wild-type <br /> Mcm10 truncation protein cross-reacts with numerous proteins                        cells (Fig. 5). Light microscopy of cells in various stages of the <br /> in yeast extract (Fig. 4, lane 1). Antibodies were affinity puri-                   cell cycle is shown in Fig. 5A. Specific stages of the cell cycle <br /> fied against Mcm10.His immobilized on a polyvinylidene diflu-                       can be determined by the position of the DAPI-stained nucleus <br />  VOL. 17, 1997                                                          Mcm10 AND ELONGATION FORK PAUSING IN S. CEREVISIAE                                       3265 <br />  <br />  <br />  <br />  <br />   FIG. 3. Flow cytometry reveals a nuclear DNA content of approximately 2C in mcm10-1 cells arrested at 37&#194;&#176;C. Wild-type (8534-8C) or mcm10-1 (mcm10-2C) cells <br /> were grown at 23&#194;&#176;C until early log phase and then incubated for 4 h at 23, 30, or 37&#194;&#176;C. Cells were prepared for flow cytometric analysis as described in Materials and <br /> Methods. <br />  <br />  <br />  <br /> of each cell (Fig. 5B) and the morphology of the spindles                            homolog of S. pombe Mis5 (28, 50). MCM6 is an essential gene <br /> revealed by immunostaining of tubulin (Fig. 5D). As seen in                          that encodes a conceptual protein of 1,017 amino acids (open <br /> Fig. 5C, the anti-Mcm10 antibodies detect a protein that is                          reading frame product YGL201c) (28). It contains all three <br /> present in the nucleus at all stages of the cell cycle.                              conserved domains characteristic of the MCM2-7 family (32). <br />    Mcm10 interacts with several members of the MCM2-7 fam-                              In the yeast two-hybrid system, if two proteins interact, they <br /> ily. The mcm10-1 arrest phenotype shares a notable similarity                        reconstitute transcriptional activity detected by expression of <br /> with the arrest phenotype of established replication initiation                      the lacZ reporter gene. Cells expressing the reporter gene form <br /> mutants, including mcm2-1 and mcm3-1 strains isolated in this                        blue colonies on X-Gal culture plates. A chimeric construct <br /> lab. Additionally, the mcm mutants, including mcm10-1, exert                         encoding the Gal4 transcriptional activation domain fused N <br /> their most dramatic effects on a similar subset of minichromo-                       terminally to Mcm10 was cloned in vector pGAD2F. Recom- <br /> somes. Furthermore, previous work in our lab has demon-                              binants of the coding regions of members of the MCM2-7 <br /> strated that many MCM gene products in yeast extracts interact                       family fused to the DNA binding domain of LexA were cloned <br /> with each other to form complexes that appear larger than 700                        in vector pBTM116 (Fig. 6A). Except for pBTM.MCM6, which <br /> kDa as determined by sedimentation through glycerol gradi-                           was not tested, each of the constructs (pBTM.MCM2, pBTM. <br /> ents (35). We speculated that since Mcm10 is involved in                             MCM3, pBTM.CDC46, pBTM.CDC47, pBTM.CDC54, and <br /> initiating DNA replication, it might interact with other initia-                     pGAD.MCM10) complements a null or a temperature-sensi- <br /> tion factors. Therefore, we used the two-hybrid system (18)                          tive allele of the respective genes; hence, the fusion proteins <br /> and GST&#226;&#8364;&#8220;Mcm2-7 affinity chromatography (35) to examine the                           are functional. <br /> interaction of Mcm10 with members of the MCM2-7 family.                                 As shown in Fig. 6A, plasmids encoding LexA-Mcm2, LexA- <br /> The MCM2-7 family includes Mcm2 and Mcm3, identified by <br />                                                                                      Mcm6, or LexA-Cdc47 produce a strong blue color in cells <br /> the mcm screen; Cdc46/Mcm5, Cdc47/Mcm7, and Cdc54/ <br />                                                                                      which also produce the Gal4-Mcm10 fusion protein. However, <br /> Mcm4, identified by a screen for cdc mutants that fail to enter <br />                                                                                      no such blue color could be detected in cells expressing the <br /> S phase (43); and Mcm6, which was cloned as the S. cerevisiae <br />                                                                                      Gal4-Mcm10 protein together with any of the other LexA- <br />                                                                                      Mcm fusions. Because negative results are not proof of an ab- <br />                                                                                      sence of interactions, these observations suggest that Mcm10 <br />                                                                                      interacts with at least three members of the MCM2-7 family of <br />                                                                                      DNA replication initiation factors: Mcm2, Mcm6, and Cdc47/ <br />                                                                                      Mcm7. <br />                                                                                         As an independent biochemical method for detection of the <br />                                                                                      physical interaction between Mcm10 and members of the <br />                                                                                      MCM2-7 family, we constructed GST fusions of each of the <br />                                                                                      MCM2-7 proteins for affinity chromatography. Each of these <br />                                                                                      GST-MCM fusions complemented the corresponding mcm <br />                                                                                      mutant for function except GST-Mcm6, which was not tested. <br />                                                                                      Each of the GST-Mcm fusion proteins expressed in yeast was <br />                                                                                      bound to glutathione-Sepharose resin. Yeast protein extracts <br />                                                                                      (Fig. 6B, lane 1) were incubated with the resin before it was <br />    FIG. 4. Preparation of monospecific anti-Mcm10 antibodies. Western blots <br />                                                                                      extensively washed with buffer containing 0.1 M NaCl. Proteins <br /> of yeast extracts were probed with antiserum from rabbit (lane 1) or affinity-       were eluted from each column coupled with GST (lane 2) <br /> purified Mcm10-specific antibodies (lane 2).                                         GST-Mcm2 (lane 3), GST-Mcm3 (lane 4), GST-Mcm4 (lane <br />  3266       MERCHANT ET AL.                                                                                                                   MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />   FIG. 5. Indirect immunofluorescence of Mcm10 protein shows that the Mcm10 protein is constitutively nuclear. Wild-type (8534-8C) cells in log phase at 30&#194;&#176;C were <br /> prepared. (A) Nomarski microscopy; (B) DAPI staining; (C) affinity-purified anti-Mcm10 antibodies; (D) Yol1/34 antitubulin antibodies. <br />  <br />  <br />  <br /> 5), GST-Mcm5 (lane 6), GST-Mcm6 (lane 7), and GST-Mcm7                             ORI121 were visualized by hybridization with probes corre- <br /> (lane 8) with 0.5 M NaCl. Western blots of each of these                           sponding to the entire length of the relevant restriction frag- <br /> protein fractions were probed with Mcm10-specific antibodies.                      ment. <br /> As shown in Fig. 6B, Mcm10 was eluted from columns coupled                            The results of 2D gel analysis are shown in Fig. 7. As antic- <br /> with GST-Mcm3, GST-Mcm4, and GST-Mcm6. Thus, interac-                              ipated from the minichromosome maintenance data, ORI121 <br /> tions between Mcm10 and five of the six MCM2-7 proteins,                           (Fig. 7A) is more active than ORI1 (Fig. 7C). In the wild-type <br /> Mcm2, Mcm3, Mcm4, Mcm6, and Mcm7, could be detected by                             strain, a strong bubble arc is visible for both origins. The <br /> either two-hybrid analysis or GST fusion affinity chromatogra-                     bubble arc indicates that DNA replication has initiated from <br /> phy or both.                                                                       within the restriction fragment. Both ORI1 and ORI121 are <br />    The mcm10-1 mutation disrupts DNA replication initiation                        used as initiation origins in the vast majority of cells, in agree- <br /> at chromosomal origins. The data presented above are consis-                       ment with results obtained by others (17, 53). In the mcm10 <br /> tent with the hypothesis that the MCM10 gene product is                            mutant, in contrast, replication initiation at the same sites is <br /> important for initiating DNA replication. We anticipated that                      severely reduced (Fig. 7B and D). Examination of restriction <br /> if Mcm10 is a DNA replication initiation factor, then the                          fragments containing ORI121 (Fig. 7B) reveals a weak bubble <br /> mcm10 mutant should show a decrease in the frequency of                            arc and a strong Y arc. This pattern indicates that ORI121 is <br /> chromosomal initiation events in vivo. Therefore, we examined                      functioning as an active origin of DNA replication in only a <br /> chromosomal replication origin usage in the mcm10 strain by                        minor portion of the cells in S phase. DNA replication is pre- <br /> using 2D DNA gels (4). We chose to analyze cells growing at                        sumably initiated at sites outside of the ORI121 DNA frag- <br /> 30&#194;&#176;C since plasmid stability assays had shown that at this tem-                    ment, and the ORI121 fragment is replicated by elongation <br /> perature minichromosome loss rates vary in an ARS-specific                         from these neighboring sites. The ORI1 restriction fragments <br /> manner (Fig. 1). Moreover, if ARS usage on the minichromo-                         give rise to an even weaker bubble signal (Fig. 7D). The pre- <br /> some reflects respective origin usage on the natural chromo-                       dominant Y arc indicates that ORI1 is used very infrequently <br /> some, then ORI121 should be used more frequently than                              to initiate DNA replication at 30&#194;&#176;C in the mcm10 strain. <br /> ORI1 to initiate DNA replication.                                                     The mcm10-1 mutation retards replication elongation in the <br />    Wild-type (8534-8C) and mcm10 mutant (mcm10-2C) strains                         region of the ARS. Shorter exposures of the 2D gel autoradio- <br /> were grown at 30&#194;&#176;C for several generations. Chromosomal                            graphs consistently reveal an accumulation of signal in discrete <br /> DNA was harvested from cultures in early log phase. The DNA                        locations within the Y arcs at or near replication origins in the <br /> was digested with restriction enzymes to derive 3-kb linear                        mutant. Each elongation arc, formed by ORI1 or ORI121 <br /> ORI121 fragments, with a centrally located origin (53), or to                      fragments, contains two spots of increased intensity (Fig. 8B <br /> derive 5-kb linear ORI1 fragments, with an off-center origin                       and D) not observed in the wild-type strain (Fig. 8A and C). <br /> (17) (see Fig. 9A). Restriction-digested genomic DNA was                           The greater intensity reflects the accumulation of replication <br /> then analyzed by electrophoresis in two dimensions, followed                       intermediates of a particular size, as has been documented in <br /> by Southern blotting; DNA fragments containing ORI1 or                             several other studies (5, 12, 23, 37). The presence of a com- <br />  VOL. 17, 1997                                                         Mcm10 AND ELONGATION FORK PAUSING IN S. CEREVISIAE                                        3267 <br />  <br />  <br />  <br />  <br />                                                                                      FIG. 7. 2D DNA gel analysis of replication origins reveals decreased origin <br />                                                                                   usage at ORI121 and ORI1 in the mcm10-1 mutant. DNA was prepared from <br />                                                                                   wild-type (8534-8C) or mcm10-1 mutant (mcm10-2C) cells grown to early log <br />                                                                                   phase at 30&#194;&#176;C. (A and B) Replication origin usage at ORI121; (C and D) <br />                                                                                   replication origin usage at ORI1. In all panels, the first dimension was run left to <br />                                                                                   right and the second dimension was run top to bottom. Open arrows, bubble arcs; <br />                                                                                   closed arrows, Y arcs. <br />  <br />  <br />  <br />                                                                                   direction at two independent sites or from both directions at a <br />                                                                                   single site, we prefer a single-pause-site hypothesis for the <br />                                                                                   reason presented below. <br />                                                                                      Two pause signals representing two populations of pause <br />                                                                                   intermediates (PI) are expected if replication forks elongating <br />                                                                                   from either direction pause at a single site that is asymmetri- <br />                                                                                   cally located within the restriction fragment (Fig. 8E and F). In <br />    FIG. 6. Physical interactions between Mcm10 and members of the MCM2-7          one population (PI1), the fork enters from the right and pauses <br /> family of proteins. (A) Two-hybrid analysis. BTM116 and BTM.MCM denote            at or near the ARS. In the second population (PI2), the fork <br /> plasmids that express LexA and LexA&#226;&#8364;&#8220;Mcm2-7 proteins, respectively. GAD2F <br /> and GAD.MCM10 denote plasmids that express Gal4 and Gal4-Mcm10 pro-               enters from the left and, again, pauses at or near the ARS. The <br /> teins, respectively. Yeast transformants of EGY40 carrying each pair of the       spacing of the pause signals in both ORI1 (Fig. 8F) and <br /> plasmids and the lacZ reporter gene were assayed for b-galactosidase expression   ORI121 (Fig. 8E) is consistent with this hypothesis. The ap- <br /> by colony color on X-Gal selective plates. (B) GST&#226;&#8364;&#8220;Mcm2-7 fusion affinity         proximate masses of the observed PI support the idea that <br /> column chromatography. The Western blot contains input yeast extracts and <br /> proteins eluted in buffer B&#226;&#8364;&#8220;0.5 M NaCl from glutathione-Sepharose 4B columns      pausing occurs at or near the ARS. In the case of ORI121, the <br /> coupled with GST or GST-Mcm2 through GST-Mcm7 as indicated. The Western           arc of elongation intermediates begins at 3 kb and ends, upon <br /> blot was probed with Mcm10-specific antibodies.                                   complete replication, at 6 kb (Fig. 8B and E). A high percent- <br />                                                                                   age of replicating molecules has accumulated with sizes of <br />                                                                                   roughly 4.5 and 5 kb (open triangles in Fig. 8B; PI1 and PI2 in <br />                                                                                   Fig. 8E). For ORI1, the mass of the Y arc begins at approxi- <br /> plete Y arc and the absence of a double-Y signal reflects a                       mately 5 kb and ends at about 10 kb (Fig. 8D). Replicating <br /> pause, not a complete halt, of replication fork progression (23).                 molecules have accumulated with sizes of roughly 7 and 9 kb <br /> In contrast to the diffuse elongation pause signal observed in a                  (open triangles in Fig. 8D; PI1 and PI2 in Fig. 8F). While the <br /> region near tRNA genes as a result of tRNA transcription (12),                    exact position(s) of pausing cannot be determined from these <br /> the pause signals that we observe near the chromosomal rep-                       gels, the data suggest that in both cases, pausing occurs within <br /> lication origins in the mcm10 mutant are quite compact, indi-                     a 1-kb region that overlaps with the autonomously replicating <br /> cating that the elongation fork may be pausing within a small                     sequence at ORI1 or ORI121. <br /> segment of the DNA. Furthermore, pausing of elongation due                           Pausing of elongation forks at ORI1 occurs within a region <br /> to transcriptional interference generates a single spot in 2D gel                 of 300 bp that includes the minimal functional ARS1 sequence. <br /> analysis, since forks stall only when they oppose the direction                   The functional ARS1 is a 117-bp sequence consisting of an <br /> of gene transcription. In contrast, we observe two pause signals                  essential A domain and three important elements (B1, B2, and <br /> in each Y arc formed by ORI1 and ORI121 DNA fragments in                          B3) in the B domain (41). To improve resolution of the loca- <br /> the mcm10 mutant. Although two pause signals could result                         tion of the pause site(s), genomic ORI1 DNA was cleaved into <br /> from the pausing of elongation forks emanating from one                           two fragments (I and II) with EcoRI (Fig. 9A). Fragment I, <br />  3268      MERCHANT ET AL.                                                                                                                MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />    FIG. 8. DNA replication forks pause during elongation through two chromosomal regions containing replication origins in the mcm10 mutant. (A to D) Short <br /> exposures of identical gels shown in Fig. 7. Southern blots were probed with 32P-labelled BamHI-EcoRI ARS121 fragment (A and B) or 32P-labelled NcoI ARS1 <br /> fragment (C and D). Triangles indicate pause signals. (E and F) Cartoons depicting the approximate locations of ARS121 in the BamHI-EcoRI fragment containing <br /> ORI121 (E) and ARS1 in the NcoI fragment containing ORI1 (F). <br />  <br />  <br />  <br /> 1,453 bp, contains a centrally located ARS1 such that the B3                     ical isolation of the ORC has added to the repertoire of factors <br /> element is positioned at 751 bp from one end. Fragment II,                       known to play a part in the initiation process. However, our <br /> 3,089 bp, does not contain an ARS. If pause signals are the                      understanding of how DNA replication initiates remains in- <br /> results of the stalling of replication forks emanating from both                 complete and must be broadened by identifying additional <br /> directions at ARS1, then two predictions can be made. First,                     gene products required for the process. To this end, we have <br /> no pause signals should be detected within fragment II. Sec-                     characterized the mcm10 mutant. The data presented in this <br /> ond, two clustered pause signals, which may or may not be                        paper implicate the MCM10 gene product in initiating DNA <br /> resolved by 2D gel electrophoresis, should be detected within                    replication, possibly in conjunction with the MCM2-7 proteins <br /> fragment I.                                                                      or their protein complexes. <br />    2D gels of EcoRI-restricted genomic yeast DNA were                               The mcm10 mutant arrest phenotype is remarkably similar <br /> probed with the 1.4-kb (Fig. 9B) and 3.1-kb (Fig. 9C) EcoRI <br />                                                                                  to that of several known initiation mutants. While the mcm10 <br /> fragments to detect replication intermediates in regions I and <br />                                                                                  mutant grows normally at 23&#194;&#176;C, it arrests following 4 h of <br /> II, respectively. A single pause signal is detected within region <br /> I, as expected if a single pause site is located near the center of              incubation at 37&#194;&#176;C with a distinct, uniform morphology (Fig. <br /> the fragment (Fig. 9B). In corroboration, no pause signals                       2). Cells arrest premitotically, with a large bud and a single <br /> could be detected in region II, which is devoid of an ARS.                       nucleus. The significance of the elongation of the bud is not <br /> These results support the hypothesis that the mcm10 mutation                     clear. Temperature-sensitive strains containing lesions in ei- <br /> not only prevents initiation of DNA synthesis at replication                     ther Mcm3, Cdc46/Mcm5, or Cdc47/Mcm7 arrest with a dumb- <br /> origins but also impedes the progression of elongation forks                     bell morphology similar to that of the arrested mcm10 strain <br /> through these replication origins.                                               (21, 42), as does a strain depleted of Mcm2 (54). Examination <br />                                                                                  of the nuclear DNA content in the arrested mcm10 strain <br />                                                                                  shows approximately 2C DNA and indicates that prior to ar- <br />                              DISCUSSION <br />                                                                                  resting in the cell cycle, the mcm10 mutant may replicate <br />   The phenotype of the mcm10 mutant suggests a DNA repli-                        nearly all of its DNA at the restrictive temperature. This nearly <br /> cation initiation defect. In eukaryotic cells, the initiation of                 2C DNA content is again similar to the results obtained with <br /> DNA replication is orchestrated by a vast network of gene                        other initiation mutants. In these mutants, DNA replication <br /> products. Many of the relevant genes have been identified                        initiating from a subset of the chromosomal origins is appar- <br /> through screens for cell division cycle (cdc) and minichromo-                    ently sufficient to replicate the majority of the genomic DNA <br /> some maintenance (mcm) mutants. Additionally, the biochem-                       prior to arrest. Other studies have also suggested that the <br />  VOL. 17, 1997                                                       Mcm10 AND ELONGATION FORK PAUSING IN S. CEREVISIAE                                    3269 <br />  <br />  <br />  <br />  <br />    FIG. 9. Mapping of pause sites in the region near ARS1 on the chromosome. (A) Map of relevant restriction sites in the region near ARS1. Arrows indicate the <br /> restriction fragments that were analyzed by 2D gels in Fig. 8C and D (NcoI fragment) and in panels B (1.4-kb EcoRI fragment I) and C (3.1-kb EcoRI fragment II). <br /> Boxes A, B1, B2, and B3 represent the elements of a functional ARS1. (B and C) 2D gels of EcoRI-restricted genomic DNA probed with fragments I and II, <br /> respectively. Replication intermediates were enriched by benzoylated naphthoylated DEAE-cellulose chromatography (14). <br />  <br />  <br />  <br />  <br /> entire yeast genome is replicated by initiation at only a fraction                with multiple members of the MCM2-7 family in the two- <br /> of all origins (10).                                                              hybrid system and in affinity chromatography (Fig. 6). Indirect <br />    The minichromosome maintenance assay reveals that at                           immunofluorescence shows that Mcm10 is a nuclear protein <br /> 30&#194;&#176;C the mcm10 mutant perturbs minichromosome mainte-                             (Fig. 5). Thus, it is likely that Mcm10 interacts with members <br /> nance in an ARS-specific manner (Fig. 1). The plasmid con-                        or complexes of the MCM2-7 proteins in the nucleus to regu- <br /> taining ARS121 is the most stable, followed by that containing                    late the initiation of DNA synthesis at replication origins. <br /> ARS120. The other plasmids tested, containing ARS1, ARS131,                          Mcm10 is required for the initiation of DNA replication at <br /> ARSHO, ARSH2B, or ARSHML, are consistently less stable.                           chromosomal replication origins. The results described above <br /> Based upon the idea that plasmid ARS activity reflects chro-                      provide strong evidence that Mcm10 has a role in initiating <br /> mosomal origin activity (11, 55) and on 2D gel data (discussed                    DNA replication. We therefore used 2D gel analysis to deter- <br /> below), we infer that the mcm10 mutant uses only a subset of                      mine directly whether the mcm10 mutant is defective in the ini- <br /> chromosomal replication origins in vivo, resulting in a pro-                      tiation of chromosomal DNA replication. Indeed, at ORI121 <br /> longed S phase and thus an increase in the length of the cell                     and at ORI1, the frequency of initiation events is reduced <br /> cycle in the mcm10 mutant at 30&#194;&#176;C.                                                significantly in the mutant compared to that in the wild-type <br />    The Mcm10 protein interacts with several members of the                        (Fig. 7). Furthermore, the mcm10-1 mutation exerts a more <br /> MCM2-7 family. Initiation of DNA synthesis involves the in-                       dramatic effect on initiation at ORI1 than on ORI121. Given <br /> teraction of a large number of proteins, many of which exist as                   the dramatic reduction in initiation frequencies at the two <br /> subunits of large complexes. The ORC, a protein complex of                        origins examined in the mcm10 mutant, we conclude that ini- <br /> six subunits, is believed to be constitutively bound to replica-                  tiation of DNA synthesis at replication origins requires the <br /> tion origins in S. cerevisiae (2), and the MCM2-7 proteins are                    MCM10 gene product. In sharp contrast to the abf1 mutants <br /> believed to mediate the transition of replication origins from                    (47), mcm10 affects the activities of the same set of ARSs in a <br /> an inactive prereplicative state to an active, replicative state                  hierarchy similar to that observed in the mcm2, mcm3 (55), <br /> (13). While the MCM2-7 proteins can form large complexes, it                      and mcm5 (7) mutants or when Mcm2 is limiting (35). The <br /> is unclear whether these large complexes are functional and                       similar ARS specificities exhibited by the mcm mutants and the <br /> whether they contain all six members of the MCM2-7 proteins                       physical interactions between Mcm10 and members of the <br /> and/or other proteins (35). Additional proteins may interact                      MCM2-7 proteins support the hypothesis that Mcm2-7 and <br /> with the MCM2-7 proteins to induce the replicative state at                       Mcm10 function in the same pathway in regulating origin us- <br /> replication origins. As reported in this paper, Mcm10 interacts                   age in the initiation of DNA replication. Finally, because the <br />  3270      MERCHANT ET AL.                                                                                                                         MOL. CELL. BIOL. <br />  <br />  <br /> MCM10 gene product is essential (49), we speculate that the                                <br /> essential function of this protein lies in initiating DNA repli-                           <br />                                                                                     <br />    The mcm10-1 mutation causes pausing of DNA replication                                <br /> at or near the ARS. In vivo analysis of DNA replication initi-                      <br /> ation in the mcm10 mutant revealed a decrease in the fre-                              <br /> quency of initiation events at the two origins examined, similar                   <br /> to that observed for other initiation mutants. However, we                             <br /> observed a striking difference in the 2D gel patterns derived                    <br /> from the mcm10 mutant strain. Unlike other mutants exam-                                 <br /> ined in other studies (19, 36, 55), the mcm10-1 lesion induces                    <br /> distinct pause signals in the elongation arcs of both chromo-                            <br /> some segments examined (Fig. 8). Several lines of evidence                          <br /> suggest that these pause signals are the result of the pausing of                        <br /> replication forks from opposite directions at or near the ARS                       <br /> during elongation through ORI1 and ORI121. The presence of                            <br /> two pause signals and the spacing of the pair of signals in each                         <br /> of the two origin fragments studied support this interpretation.                     <br /> More convincingly, we have shown that separation of the two                               <br /> pause signals near ORI1 on 2D gels can be manipulated by                            <br /> changing the relative position of ARS1 within the DNA frag-                            <br /> ment (Fig. 9). Furthermore, we have mapped the pause signals                           <br /> in the ORI1 region within a 300-bp sequence that includes                               <br /> ARS1. Assuming that the recessive mcm10-1 mutation results                         <br /> in a loss of Mcm10 function, the pausing suggests that Mcm10                             <br /> must be present to allow unimpeded progression of DNA                             <br />                                                                                           <br /> replication forks through the ARS-containing regions of the                               <br /> chromosomes.                                                                       <br />    What might be the role of Mcm10 in the initiation of DNA                              <br /> replication? We envision that Mcm10 may mediate the initia-                          <br /> tion of DNA synthesis at replication origins through interac-                             <br /> tion with the MCM2-7 proteins. The loss of Mcm10 function                            <br /> not only prevents replication initiation at chromosomal origins                          <br /> but also presents a block to replication fork movement through                         <br /> the same regions. This block could result from the prolonged                        <br /> binding of the replication initiation complex which is normally                  <br /> discharged from replication origins upon activation of initia-                            <br /> tion. In this hypothesis, the failure to initiate DNA synthesis at                       <br /> replication origins in the mcm10 mutant could also be the                         <br /> reason for the blockage of elongation fork movement in those                        <br /> regions. Elucidation of the function of Mcm10 awaits identifi-                            <br /> cation of the DNA sequences where the replication fork                              <br /> pauses and the gene products associated with the pause sites in                           <br />                                                                                            <br /> the mcm10 mutant.                                                                 <br /> </body></html>