<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Innate Immunity Human Embryonic Stem Cells: <br /> Comparison Adult Human Endothelial Cells <br /> Ga&#194;&#180;bor Fo&#194;&#168;ldes1., Alexander Liu1., Rekha Badiger1, Mark Paul-Clark1, Laura Moreno1, Zsuzsanna <br /> Lendvai2, Jamie S. Wright1, Nadire N. Ali1&quot;, Sian E. Harding1*&quot; , Jane A. Mitchell1&quot; <br /> 1 National Heart Lung Institute, Imperial College, London, United Kingdom, 2 Heart Center, Semmelweis University, Budapest, Hungary <br />  <br />  <br />  <br />      <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />      Treatment human disease human embryonic stem cell (hESC)-derived cells close reality, little known <br />      responses physiological pathological insult. The ability cells respond activation Toll like receptors <br />      (TLR) critical innate immune sensing tissues, extends general danger sensing, e.g. oxidative <br />      stress, cardiomyocytes. We used biomarker release gene-array analysis compare responses hESC and <br />      differentiation, primary human endothelial cells. The presence cardiomyocytes endothelial cells <br />      confirmed differentiated cultures immunostaining, FACS-sorting and, cardiomyocytes, beating activity. <br />      Undifferentiated hESC did respond CXCL8 release Gram positive Gram negative bacteria, range PAMPs <br />      (pathogen associated molecular patterns) TLRs 1-9 (apart flagellin, activator TLR5). Surprisingly, lack TLR- <br />      dependent responses maintained 4 months differentiation hESC, cultures included cardiomyocytes <br />      endothelial cells. In contrast, primary cultures human aortic endothelial cells (HAEC) demonstrated responses a <br />      broad range PAMPs. Expression downstream TLR signalling pathways demonstrated hESC, IL-1b, TNFa and <br />      INFc, bypass TLRs, stimulated CXCL8 release. NFkB pathway expression present hESC NFkB was <br />      able translocate nucleus. Low expression levels TLRs detected hESC, especially TLRs 1 4, explaining <br />      lack response hESC main TLR signals. TLR5 levels similar differentiated hESC HAEC, and <br />      siRNA knockdown TLR5 abolished response flagellin. These findings potential implications <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span>survival</span> and <br />      function grafted hESC-derived cells. <br />  <br />   Citation: Fo&#194;&#168;ldes G, Liu A, Badiger R, Paul-Clark M, Moreno L, et al. (2010) Innate Immunity Human Embryonic Stem Cells: Comparison Adult Human <br />   Endothelial Cells. PLoS ONE 5(5): e10501. doi:10.1371/journal.pone.0010501 <br />   Editor: Massimo Federici, University Tor Vergata, Italy <br />   Received November 2, 2009; Accepted April 14, 2010; Published May 5, 2010 <br />   Copyright: &#195;&#376; 2010 Fo&#194;&#168;ldes et al. This open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: This work supported by: Wellcome Trust (http://www.wellcome.ac.uk/), British Heart Foundation (http://www.bhf.org.uk/), European Community, <br />   NC3Rs (http://www.nc3rs.org.uk/), BBSRC (http://www.bbsrc.ac.uk/), Rosetrees Trust (http://www.rosetreestrust.co.uk/), SROP (http://www.nfu.hu/), Geron <br />   Corporation. A.L. recipient Wellcome Trust Vacation Scholarship. M.P.C. recipient Wellcome Trust University Award (083429/Z/07/Z) R.B. a <br />   recipient BHF Clinical Research Fellowship. The funders role study design, data collection analysis, decision publish, preparation the <br />   manuscript. <br />   Competing Interests: Geron: The authors non-financial collaborative agreement, does alter adherence PLoS ONE policies on <br />   sharing data materials. <br />   * E-mail: sian.harding@imperial.ac.uk <br />   . These authors contributed equally work. <br />   &quot; These authors contributed equally work. <br />  <br />  <br />  <br /> Introduction                                                                            (TLRs) [4]. LPS activates TLR4 recruits adapter protein <br />                                                                                         pathways including MyD88, MAL, TRIF TRAM initiate <br />     Human embryonic stem cells (hESC) currently                               signaling events leading activation NFkB induction of <br /> developed sources tissue-specific cells treatment                      inflammatory genes including CXCL8. There 10 TLRs <br /> human disease, including heart failure. It hoped hESC-                          expressed human cells specific PAMPs 1&#226;&#8364;&#8220;9 identified. <br /> derived cells re-seed repair damaged tissues allowing                           TLR10 remains orphan receptor present. In addition the <br /> recovery organ function. Although immune response                         sensing pathogens PRRs understood sense host <br /> host implanted cells subject interest,                        ligands wider role surveillance danger signals <br /> little known innate immune response grafted                         [5]. Where phenotypes TLR <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span>knock-out</span> mice studied <br /> cells themselves. Many cells body express partial                      directly, TLR4 gene deletion, example, associated with <br /> innate immune response, include endothelial                          immune suppression, chronic inflammation lung [6], <br /> cells [1] cardiomyocytes [2,3] required make                       vascular compromise evidence heart failure [7]. On the <br /> a viable cardiac graft. The innate immune response                             hand, TLR activation heart involved the <br /> modeled experimentally activation cells pathogens                         deleterious responses oxidative stress [8], ischemia [9] and <br /> pathogen associated molecular patterns (PAMPs). The best studied                        septic cardiomyopathies [10], various TLR <span id='am-238' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-239' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-241' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-242' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-243' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-244' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-251' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-264' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-271' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-273' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-292' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-296' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span>knockout</span> mice are <br /> of PAMPs lipopolysaccharide (LPS) derived Gram                              resistant insults doxorubicin <br /> negative bacteria. PAMPs sensed cells <span id='am-316' about='xsp:pattern' typeof='owl:Thing'>pattern</span>                                cardiomyopathy [11] hypertrophy [12]. HESC originate <br /> recognition receptors (PRRS), include Toll like receptors                         inner cell mass blastocyst, left undisturbed <br />  <br />  <br />        PLoS ONE | www.plosone.org                                                   1                                    May 2010 | Volume 5 | Issue 5 | e10501 <br />                                                                                                                    TLR Pathways Human hESC <br />  <br />  <br /> would develop sterile environment womb. TLR                    gelatinized dishes cultured order allow continued <br /> responses embryo develop relatively late,               differentiation. <br /> suppressed neonatal period [13]. The question whether <br /> hESC derivatives culture similar trend i.e.              Cell Plating Handling <br /> whether retain immature <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> develop                      Undifferentiated H7 cell colonies 6-well plates were <br /> mature TLR responses, vital question understand            removed subcultured 96-well plates previously coated with <br /> in order establish potential repair heart       Matrigel (100 ul/well). For this, medium aspirated and <br /> organs. In paper directly compare expression                 spontaneously differentiating cells colonies removed <br /> activity TLRs, downstream signaling components                  treatment collagenase 37uC 10 minutes, after <br /> NFkB signaling undifferentiated differentiated                   time collagenase aspirated cells were <br /> human hESC fully differentiated mature human                 washed 2 ml PBS. Colonies undifferentiated H7 cells were <br /> aortic endothelial cells.                                                    broken mechanically 5-ml pipette tip small clusters <br />                                                                              subcultured .70% confluence. Differentiated hESC in <br />                                                                              T175 flasks 10-cm culture dishes removed the <br /> Materials Methods <br />                                                                              surface treatment Trypsin-EDTA (Sigma-Aldrich), <br /> Materials                                                                    counted plated 96 plates coated 0.5% gelatin. <br />    Heat inactivated E. coli S. aureus prepared described <br /> previously [14]. Synthetic agonists TLR1/2 (Pam3CSK4),                   hESC-Derived Endothelial Cell (hESC-EC) Culture <br /> TLR2/6 (FSL-1), TLR3 [poly(I:C)], TLR5 (Flagellin), TLR7                        Undifferentiated H7 hESC dissociated clumps and <br /> (Imiquimod), TLR8 (E. coli K12 ssRNA) purchased                    placed ultra low-attachment plates medium containing 2% <br /> InvivoGen Co. (San Diego, USA). TLR4 ligand (LPS)                        FCS (Endothelial Growth Medium-2, Lonza). As described <br /> obtained Sigma-Aldrich (Dorset, UK). IL-1 purchased                 [16], CD31+ cells sorted sterile cell sorter <br /> <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> R&amp;D Systems (Abingdon, UK). All reagents, unless                  (BD FACSAria, BD Biosciences) cultures 13 days after <br /> otherwise stated, obtained Invitrogen.                             differentiation propagated endothelial growth medium. <br />                                                                              Passages 3 10 used experiments. <br /> Human Embryonic Stem Cell Culture <br />     Most experiments performed using H7 line            Matrigel Tubule-Forming Assay <br /> Geron Corporation&#195;&#376; (Menlo Park, CA, USA). However, gene                         Matrigel diluted 1:2 endothelial basal medium ice <br /> array analysis undifferentiated stem cells performed H7,          100 ml/well added 24-well plates allowed gel in <br /> SHEF2, SHEF4 SHEF5 lines. H7 SHEF lines                          layer 37uC. CD31+ cells (50.000 well) seeded <br /> ethically derived hESC lines. H7 imported                        gels tubes photographed 22 hours. <br /> collaboration agreement Geron, permission from <br /> the UK Stem Cell Bank.                                                       Endothelial Cell Culture <br />     Undifferentiated H7 cells maintained feeder-cell                Primary human aortic endothelial cells purchased from <br /> free conditions mouse embryonic fibroblast-conditioned medi-              Promocell (Heidelberg, Germany) cultured according <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> <br /> um (MEF-CM), supplemented 8 ng/ml recombinant                        manufacturer&#226;&#8364;&#8482;s instructions. The human endothelial cell line <br /> human basic fibroblast growth factor Geron&#226;&#8364;&#8482;s protocols                (EAhy-926) cultured Dulbecco&#226;&#8364;&#8482;s Modified Eagle&#226;&#8364;&#8482;s Medium <br /> described previously [15]. In brief, mouse embryonic fibroblasts             (DMEM) supplemented 1% Hypoxanthine-Aminopterin- <br /> (MEFs) obtained pregnant mouse embryos MF-1                     Thymidine (all Sigma-Aldrich), 5 mM L-glutamine, 100 U/ml <br /> strain. After propagation 10% FCS-containing medium, MEFs                 penicillin, 100 mg/ml streptomycin, 10% heat-inactivated <br /> were mitotically inactivated 0.01 mg/ml mitomycin C                  FCS. <br /> passage 4 2.5 hours adhered overnight pre- <br /> gelatinized T225 flasks (at seeding density 1.886107 cells/             Cell Treatments <br /> flask) medium containing 10% FCS subsequently                      To investigate response TLR stimulation, cells were <br /> replaced 150 ml hESC medium, supplemented                       treated 96 plates PAMPS agonists TLR1- <br /> recombinant human 4 ng/ml basic fibroblast growth factor.                    8, heat killed S. aureus E. coli IL-1 used a <br /> Conditioned medium, collected daily 10 days,                   positive control, 24 hours. For experiments measure NFkB <br /> passed 0.2 mm low protein attachment cellulose acetate               activation cells treated 1 hour. <br /> filter units (Corning) prior feeding H7 cells. The hESC medium <br /> consisted KnockOut DMEM (KO DMEM) supplemented                       Knockdown TLR5 <br /> 20% KnockOut serum replacement (KOSR), 1 mM L-glutamine,                        For TLR5 siRNA knockdown, ON-TARGETplus SMART- <br /> 50 U/ml penicillin, 50 mg/ml streptomycin, 1% non-essential                  pool TLR5 siRNA transfection performed using DharmaFect <br /> amino acids (100x stock) 0.1 mM b-mercaptoethanol. All                   reagent (100 nM, final incubation volume 100 ml) manufac- <br /> undifferentiated cells cultured Matrigel (BD Sciences)-              turer&#226;&#8364;&#8482;s instructions (Dharmacon, Thermo Scientific). Scrambled, <br /> coated 6-well plates (Nunc, Roskilde, Denmark). Before induction             non-targeting siRNA (100 nM; Dharmacon, Thermo Scientific) <br /> of differentiation, spontaneously differentiated cells removed          used negative controls. Fluorescent siGLO Red siRNA <br /> by treatment collagenase 37uC 10 min. hESC                 indicator (100 nM; Dharmacon, Thermo Scientific) used for <br /> colonies mechanically broken 5 ml pipette tip                optimization documentation transfection efficacy (.90%) <br /> were cultured 4 days low attachment 6-well plates (Nunc),             24&#226;&#8364;&#8220;48 hours. <br /> suspended differentiation medium form embryoid bodies. <br /> The differentiation medium the hESC medium                   Immunocytochemistry <br /> except KOSR replaced 20% non-heat-                           Cell fixed 4% paraformaldehyde, permeabilized with <br /> inactivated FCS. Embryoid bodies plated 0.5%                   0.2% Triton X-100, labeled primary antibodies anti- <br />  <br />  <br />        PLoS ONE | www.plosone.org                                        2                             May 2010 | Volume 5 | Issue 5 | e10501 <br />                                                                                                                     TLR Pathways Human hESC <br />  <br />  <br /> CD31 (PECAM-1, 1:100 dilution, Santa Cruz Biolegend), anti-              15 minutes 42uC 5 minutes 95uC inactivate/ <br /> CD34 (Abcam, 1:200), anti-myosin heavy chain (Ab15, 1:200,                  degrade RNA reverse transcriptase enzyme. 91 ml RNase- <br /> Abcam, Cambridge), anti-von Willebrand factor (1:100, Dako).                Free water added remaining cDNA. <br /> Primary antibodies detected Alexa 488- (Invitrogen)             Real Time-Polymerase Chain Reaction (RT-PCR) <br /> Alexa 647- (Invitrogen) conjugated secondary antibodies (all                Array. RT-PCR preparations performed RT2- <br /> 1:400). DNA visualised DAPI (0.5 mg/ml; Sigma). DiI-               Profiler-PCR-Array kit (SuperArray) according manufacturer&#226;&#8364;&#8482;s <br /> labelled acetylated human low density lipoprotein (Ac-LDL)              protocols. In brief, experimental cocktail up <br /> purchased Invitrogen. Images acquired Zeiss Axio               containing 102 ml diluted first-strand cDNA, 1.275 ml <br /> Observer Z1 fluorescence microscopy.                                        SuperArray-RT2-SYBR-Green/ROX-MasterMix 1.173 ml <br />                                                                             RNase-Free water. 25 ml cocktail added of <br /> Enzyme-Linked-ImmunoSorbent-Assays (ELISAs)                                 96-well plate, containing forward reverse primers for <br />    Concentrations CXCL8/IL8 cell-free supernatants               TLR-related genes, housekeeping genes, human-genomic-DNA- <br /> measured using sandwich ELISA kits (R&amp;D Systems)                        contamination PCR controls. Samples amplified using <br /> calculated using 4-parameter-log-fit curves according manufac-           ABI 7500 Real-Time-PCR System (Applied Biosystems, Foster <br /> turer&#226;&#8364;&#8482;s protocols.                                                          City, USA) 40 cycles 15 s 95uC, 30 s 55uC, 30 s at <br />                                                                             72uC. Relative levels gene expression (fold differences) were <br /> Measurement NFkB activation                                              calculated according manufactures instructions. CT values <br />    Undifferentiated hESC cultured 6-well plates 90%           35 higher considered indicating <br /> of surfaces covered H7 colonies              undetectable levels expression. If 3/3 2/3 runs produced <br /> were incubated IL-1b (1 ng/ml) LPS (1 mg/ml). After                CT. = 35 given gene, expression considered <br /> one hour, cellular nuclear extracts prepared using               undetectable overall. <br /> commercially available nuclear extraction kit (Active Motif,                   Quantitative RT-PCR TLR5. For quantifying mRNA <br /> Carlsbad, CA, USA) according manufacturer&#226;&#8364;&#8482;s protocols.               levels TLR5 undifferentiated differentiated hESC <br /> In brief, cells washed, collected ice-cold PBS           cultures, real-time PCR analyses performed TaqMan <br /> presence phosphate inhibitors centrifuged 5006g               Gene Expression Assay (Hs01920773_s1, Applied Biosystems, <br /> 5 min. The resultant pellets re-suspended excess hypotonic          CA). Human GAPDH Endogenous Control (FAM/MGB probe) <br /> buffer, treated detergent centrifuged briefly 14,0006g          used housekeeping control. The PCR performed with <br /> for 30 sec. After cytoplasmic fraction collected, nuclei        Rotor-Gene 3000 (Corbett Research) real-time PCR instrument <br /> were lysed nuclear proteins dissolved cocktail lysis       relative expression determined. <br /> buffer protease inhibitor. Nuclear protein concentrations were <br /> determined using Bradford assay subsequently analyzed            Statistical analysis <br /> for NFkB activation using TransAMTM NFkB p65 transcrip-                   Data reported mean6 S.E. mean n experiments. <br /> tion factor assay kit (Active Motif), according manufacturer&#226;&#8364;&#8482;s       Data analyzed using one-way ANOVA followed Dunnett&#226;&#8364;&#8482;s <br /> instructions. This assay based nuclear NF-kB p65 proteins             Multiple Comparison Test one-sample t-test normalized <br /> binding consensus NF-kB oligonucleotides fixed 96-well                data described respective legends. <br /> plates. In brief, 10 mg nuclear proteins added well <br /> and incubated 1 h allow binding P65 consensus              Results <br /> oligonucleotides. The presence resulting complex was <br /> detected primary antibody. After addition                     Phenotype human hESC differentiated include <br /> horseradish peroxidase conjugated secondary antibody, P65               endothelial-like phenotypes <br /> quantified spectrophotometry.                                               Undifferentiated hESC characteristic appearance as <br />                                                                             tightly packed cells colonies shown Fig. 1a b. The H7 <br /> Gene-Expression Analysis                                                    line, obtained Geron Corp., Menlo Park CA, grown <br />    RNA Extraction. Total RNA isolated hESC using                   feeder-free conditions described previously [15] and <br /> the RNeasy kit (Qiagen, Hilden, Germany) according                       differentiated embryoid body formation 20% FCS. After <br /> manufacturer&#226;&#8364;&#8482;s protocols. Spin-column samples centrifuged              days differentiation suspension cultures, embryoid bodies <br /> at 95006g, 20uC 15 seconds unless stated otherwise. In brief,           plated gelatinized surfaces continued to <br /> 350&#226;&#8364;&#8220;600 ml lysis-Buffer RLT added cell-pellets                differentiate adherent cultures prolonged periods (over 4 <br /> homogenized passing 20-gauge needle 5&#226;&#8364;&#8220;10 times                 months). Figure 1C-F shows morphology cultures 1 3 <br /> before addition equal volume 70% ethanol. The solution            months differentiation, demonstrating emergence a <br /> was centrifuged spin-column DNase digestion using               variety features, including clusters beating cardiomyocytes <br /> DNase-Buffer RDD solution (Qiagen) room temperature 25               (video S1) vessel-like structures. Immunocytochemical staining <br /> minutes. Subsequently, RNA washed Buffer RW1                   known markers (Fig. 2) confirms presence cardiomyo- <br /> RPE centrifugation. A final wash Buffer RPE 2                cytes endothelial cells mixed population cells. <br /> minutes applied RNA eluted 30&#226;&#8364;&#8220;60 ml                   By adjustment differentiation conditions using FACS <br /> RNase-free water. RNA concentration integrity                       sorting CD31 surface antigen, highly expandable population <br /> determined spectrophotometry (Spectramax, Switzerland)                human endothelial-like cells (hESC-EC) obtained the <br /> obtain A260/A280 ratios later agarose gel electrophoresis.           hESC. Cells took cobblestone pattern culture characteristic <br />    Reverse transcriptase-PCR First-Strand reaction. The                     endothelial cells (Fig. 3A). Cells stained positive for <br /> first-strand-complimentary-DNA (cDNA) synthesized                  endothelial-specific CD31 CD34 markers (Fig. 3C&#226;&#8364;&#8220;D) and <br /> 1 mg total RNA using RT2 First-Strand Kit (SuperArray                acetylated LDL uptake culture (Fig. 3B). Further indicating <br /> Bioscience Corporation, MD, USA) according manufacturer&#226;&#8364;&#8482;s                endothelial phenotype function, cells formed tube-like <br /> protocols. Briefly, 10 ml rt-Cocktail, containing RNA mixed          structures solidified Matrigel showed migration on <br /> with 10 ml Genomic-DNA-Elimination Mixture incubated                fibronectin surface wound healing assays (Fig. 3E&#226;&#8364;&#8220;F). <br />  <br />  <br />       PLoS ONE | www.plosone.org                                        3                             May 2010 | Volume 5 | Issue 5 | e10501 <br />                                                                                                                     TLR Pathways Human hESC <br />  <br />  <br />  <br />  <br /> Figure 1. Appearance hESC cultures. Undifferentiated H7 cells (a, b); 1 month (c, d) 3 months (e, f) differentiation. Examples of <br /> clusters beating cells seen c d, shown Video S1. Vessel-like structures observed (e). <br /> doi:10.1371/journal.pone.0010501.g001 <br />  <br />  <br />  <br /> Relative expression TLRs related genes hESC                      differentiation. Fig. 4A shows expression levels TLRs 1, 3, <br /> and endothelial cells                                                   4, 5 6 undifferentiated H7 cells 3 SHEF <br />   Expression TLRs downstream signaling effector genes               lines. TLR 1, 3, 4 6 expression consistently lower hESC <br /> was determined undifferentiated hESC 1&#226;&#8364;&#8220;4 months              compared endothelial cells, TLRs 1 4 particularly low. <br />  <br />  <br />        PLoS ONE | www.plosone.org                                      4                              May 2010 | Volume 5 | Issue 5 | e10501 <br />                                                                                                              TLR Pathways Human hESC <br />  <br />  <br />  <br />  <br /> Figure 2. Presence cardiomyocytes endothelial cells differentiated hESC cultures. Immunocytochemical staining clusters 1 <br /> month differentiated H7 hESC showing (a) cardiomyocytes myosin heavy chain (a, b) (green) corresponding brightfield image (b) <br /> endothelial cells identified von Willebrand factor (green), CD31 (red) DAPI (nuclei, blue). <br /> doi:10.1371/journal.pone.0010501.g002 <br />  <br />  <br />  <br />  <br />       PLoS ONE | www.plosone.org                                   5                            May 2010 | Volume 5 | Issue 5 | e10501 <br />                                                                                                                          TLR Pathways Human hESC <br />  <br />  <br />  <br />  <br /> Figure 3. Characteristics human embryonic stem cell-derived endothelial cells. The hESC-EC cultures showed (A) cobblestone <br /> morphology. (B) Cells showed DiI-labelled acetylated LDL uptake, stained positive human anti-CD31 antibody (green; C), anti-CD34 <br /> (red; D). DAPI (blue) used nuclear staining. (E) Cells plated solidified Matrigel form hollow-like tubes. (F) In wound healing assay, cells <br /> show migration fibronectin surface (upper panel showing cell free area 0 hours; cells migrate wound site 22 hours). <br /> doi:10.1371/journal.pone.0010501.g003 <br />  <br />  <br /> Levels TLR8, TLR10 undetectable hESC (Cycle                     downstream signalling pathways, majority NFkB <br /> threshold, CT, values 35 lower majority samples                (Fig. 4B) TLR signaling genes (Fig. 4C) expressed at <br /> (Table S1)). Levels TLR2 similar higher            similar levels endothelial cells hESC, reduced <br /> TLR6 H7 SHEF lines, CT values range                    expression MyD88 TICAM1 contribute the <br /> 27&#226;&#8364;&#8220;30 (Table S1), levels undetectable endothelial               poor functional response. Once again, reasonable <br /> cells majority samples fold-change calculated.             agreement H7 SHEF lines. <br /> Low levels expression TLR7 TLR9 present in <br /> hESC undetectable endothelial cells. Interestingly, expres-             Relative expression TLRs related genes following <br /> sion TLR5 robust endothelial cells higher                differentiation hESC <br /> undifferentiated hESC. Agreement good H7                          Expression levels determined 1, 3 4 months after <br /> SHEF lines TLR expression. Consistent activity             differentiation H7 hESC. Compared undifferentiated H7 <br />  <br />  <br />        PLoS ONE | www.plosone.org                                          6                               May 2010 | Volume 5 | Issue 5 | e10501 <br />                                                                                                                         TLR Pathways Human hESC <br />  <br />  <br />  <br />  <br /> Figure 4. Expression TLR TOLL NFkB signalling genes undifferentiated hESC. Relative expression levels undifferentiated H7 <br /> hESC (n = 3, open bars) 3 SHEF lines (grey bars) compared human aortic endothelial cells (HAEC, n = 3). A: TLR genes. TLRs 2 7&#226;&#8364;&#8220;10 <br /> were undetectable majority HAEC. B: NFkB related genes; C: TOLL signalling-related genes. Statistical significance calculated using one- <br /> sample t-test averaged H7 values 1.0, *P,0.05, **P,0.01, ***P,0.001, n = 3. <br /> doi:10.1371/journal.pone.0010501.g004 <br />  <br />  <br /> (Fig. 5A), general increase time TLRs 1, 2, 3, 4,         endothelial cells (Fig. 5B) clear TLR1 TLR4 were <br /> 5 6, undifferentiated levels 4 months. Low              low levels, 4 months differentiation. Only <br /> levels TLRs 7, 9 10 sporadically apparent                  modest adjustments expression level NFkB (Table 1) <br /> (Table S1). However, comparing differentiated hESC                       levels seen differentiation components of <br />  <br />  <br />        PLoS ONE | www.plosone.org                                         7                               May 2010 | Volume 5 | Issue 5 | e10501 <br />                                                                                                                                        TLR Pathways Human hESC <br />  <br />  <br />                                                                                         TLR signaling pathways (TICAM1, TICAM2 IRAK1) <br />                                                                                         consistently increased (Table 2). A list gene expression <br />                                                                                         changes differentiation Table S2. <br />  <br />                                                                                         Effect PAMPs TLR1-8 bacteria CXCL8 <br />                                                                                         release human hESC primary human endothelial <br />                                                                                         cells <br />                                                                                            Release CXCL8 used study biomarker cell <br />                                                                                         activation. In undifferentiated hESC statistically <br />                                                                                         significant increase CXCL8 release cells stimulated 24 h <br />                                                                                         E. coli, S.Aureus array PAMPs TLRs1, <br />                                                                                         2, 3, 4, 6, 7 8 (Fig. 6A). This compared with <br />                                                                                         differentiated hESC cultures, phenotypic evidence of <br />                                                                                         ESC specialization established (Fig. 2). One month after <br />                                                                                         initiation differentiation, hESC-derived cells did release <br />                                                                                         increased levels CXCL8 response PAMPs TLRs or <br />                                                                                         bacteria (Fig. 6B). The exception flagellin, activator <br />                                                                                         TLR5, produced increase CXCL8 both <br />                                                                                         undifferentiated hESC differentiated hESC-derived cells <br />                                                                                         (Fig. 6A B). Remarkably, 4 months differentiation, <br />                                                                                         TLR5 remained PAMP produce response hESC- <br />                                                                                         derived cells (Fig. 6C). IL-1b, acts independently TLRs <br />                                                                                         TLR adapter protein MyD88, activated undifferen- <br />                                                                                         tiated differentiated hESC release CXCL8 (Fig. 6A&#226;&#8364;&#8220;C). In <br />                                                                                         separate experiments protocols repeated compare directly <br />                                                                                         responses purified cultures hESC-EC mixed differenti- <br />                                                                                         ated hESC cultures (Fig. 7A&#226;&#8364;&#8220;B). Similar results seen those <br />                                                                                         presented Fig. 6. LPS effect CXCL8 release mixed <br />                                                                                         differentiated cultures hESCs (Fig. 7A) purified hESC-ECs <br />                                                                                         (Fig. 7B). Again flagellin IL-1b increased CXCL-8 release in <br />                                                                                         experiments (Fig. 7A&#226;&#8364;&#8220;B). Purified hESC-EC expressed <br />                                                                                         TLR5, showing 3-fold increase mRNA levels compared <br />                                                                                         undifferentiated hESC (data shown). By contrast <br />                                                                                         results obtained hESC differentiated derivatives, <br /> Figure 5. Expression TLR genes differentiated hESC. <br />                                                                                         primary cultures adult human aortic endothelial cells released <br /> Relative expression differentiated H7 hESC 1 month (n = 2, open                   increased levels CXCL8 response Gram negative bacteria <br /> bar), 3 months (grey bar) 4 months (solid bar) compared A:                       LPS (which activate TLR4), FSL-1 (which activates the <br /> averaged undifferentiated H7 hESC (n = 3) B: human aortic                           TLR2/6 heterodimer) IL-1b (Fig. 6D). Similar results seen <br /> endothelial cells (HAEC, n = 3).                                                        endothelial cell line, EAhy-926, used (data not <br /> doi:10.1371/journal.pone.0010501.g005                                                   shown). To ensure differences CXCL8 release between <br />                                                                                         hESC primary endothelial cells different <br />  <br />                                                                                          Table 2. TOLL signalling related gene expression hESC. <br />  Table 1. NFkB related gene expression hESC. <br />                                                                                          Table 2             Differentiated: fold change vs undifferentiated <br />  <br />  Table 1             Differentiated: fold change vs undifferentiated                     Gene Name           1 month         1 month        3 month     4 month <br />  <br />  Gene           1 month         1 month        3 month     4 month                  BTK                 n.d.d           n.d.d          n.d.d       n.d.d <br />  <br />  NFKB1               1.66            1.92           0.38        0.85                     MYD88               3.82            0.89           0.47        0.89 <br />  NFKB2               2.41            2.28           0.63        2.01                     TICAM2              2.84            3.23           1.18        3.10 <br />  NFKBIA              1.12            0.68           0.26        1.09                     TIRAP               1.56            1.12           0.37        0.87 <br />  NFKBIL1             1.21            0.91           0.47        1.28                     TOLLIP              1.42            1.62           1.74        1.14 <br />  NFRKB               1.61            0.83           0.28        0.91                     TRAF6               2.08            1.76           1.00        1.41 <br />  c-REL               0.75            0.41           0.12        0.76                     TICAM1              3.65            5.53           3.12        3.80 <br />  NFKB3               6.61            2.98           0.86        2.05                     IRAK1               1.34            4.23           1.50        1.84 <br />  IKBKB               0.77            0.41           0.31        0.60                     IRAK2               0.75            0.66           0.26        1.01 <br />  <br />  Fold change differentiated H7 hESC 1&#226;&#8364;&#8220;4 months compared average                 Fold change differentiated H7 hESC 1&#226;&#8364;&#8220;4 months compared average <br />  (n = 3) undifferentiated H7, value 1 represents equivalent gene              (n = 3) undifferentiated H7, value 1 represents equivalent gene <br />  expression 1 represents lower expression differentiated        expression 1 represents lower expression differentiated than <br />  undifferentiated.                                                                       undifferentiated. N.d.d = detectable differentiated cells. <br />  doi:10.1371/journal.pone.0010501.t001                                                   doi:10.1371/journal.pone.0010501.t002 <br />  <br />  <br />  <br />          PLoS ONE | www.plosone.org                                                 8                                  May 2010 | Volume 5 | Issue 5 | e10501 <br />                                                                                                                        TLR Pathways Human hESC <br />  <br />  <br />  <br />  <br /> Figure 6. Response hESC primary cells bacteria PAMPs. Release CXCL8 undifferentiated (A), 1 month- (B) 4 month- <br /> (C) differentiated H7 hESC primary cultures human aortic endothelial cells (D). Cells treated Gram negative E. coli (C, 108 CFU/ <br /> ml), Gram positive S. aureus (SA, 108 CFU/ml), PAMPs TLR2/1 (PAM3CSK4; 300 ng/ml), TLR2/6 (FSL-1; 1 mg/ml), TLR3 (Poly:IC; 10 mg/ml), TLR4 <br /> (LPS; 1 mg/ml), TLR5 (flagellin; 100 ng/ml), TLR7 (imiquimod; 10 mg/ml) TLR8 (E. coli 12 ssRNA; 10 mg/ml) IL-1b (IL-1; 1 ng/ml) 24 hours <br /> before CXCL8 measured using ELISA. The data mean 6 S.E. mean n = 8211 (A); n = 329 (B); n = 3 (C) n = 426 (D). Statistical <br /> significance calculated one-way analysis variance followed Dunnett&#226;&#8364;&#8482;s Multiple Comparison Test (to control). A p value 0.05 <br /> was considered statistically significant denoted *. <br /> doi:10.1371/journal.pone.0010501.g006 <br />  <br />  <br /> media used routine culture, undifferentiated                    Knockdown TLR5 confirms specificity flagellin <br /> differentiated hESC re-tested DMEM plus 10%                      response <br /> FCS used EAhy-926 endothelial cell culture (Fig. S1). Use                Silencing TLR5 using siRNA resulted significantly reduced <br /> the EAhy-926 media altered basal CXCL8 levels extent                 CXCL8 release flagellin-treated mixed differentiated hESC cultures <br /> but did reveal effect bacterial PAMP stimulation             (Fig. 7A) purified hESC-EC (Fig. 7B) compared scrambled <br /> CXCL8 release. However, noted effects               siRNA. Interestingly, TLR5 siRNA knockdown general <br /> long term culture highly specialized medium influence            depressant effect CXCL8 release mixed differentiated cultures, <br /> TLR signaling pathways cells.                                   reduction specific flagellin hESC-EC. <br />  <br />  <br />        PLoS ONE | www.plosone.org                                        9                               May 2010 | Volume 5 | Issue 5 | e10501 <br />                                                                                                                           TLR Pathways Human hESC <br />  <br />  <br />  <br />  <br />                                                                                  Figure 8. Activation NFkB undifferentiated hESC. (A), <br /> Figure 7. Enzyme-linked immunosorbent assay CXCL8                             Undifferentiated H7 hESC activated release CXCL8 after <br /> production hESC purified hESC-derived endothelial                         24 hours stimulation TNFa (TNF), INFc (INF) IL-1b (IL-1) but <br /> cells. Bar graphs showing release CXCL8 1 month-old                      IL-6 (10 ng/ml each)(n = 6). (B), NFkB activated in <br /> differentiated H7 hESC cells (A) hESC-derived CD31+ endothelial               undifferentiated H7 hESC following 1 hour stimulation IL-1b (1 ng/ <br /> cells (B) transfected scrambled siRNA (open bars) TLR5 siRNA             ml) LPS (1 mg/ml) (n = 3). Statistical significance was <br /> (solid bars) 48 hours. Transfected cells treated TLR4 (LPS;        calculated one-way analysis variance followed Dunnett&#226;&#8364;&#8482;s <br /> 1 mg/ml), IL-1b (IL-1b; 1 ng/ml), TLR5 (flagellin; 100 ng/ml)             Multiple Comparison Test (to control). **P,0.01. <br /> 24 hours CXCL8 measured. The data mean 6 S.E.                 doi:10.1371/journal.pone.0010501.g008 <br /> mean n = 3. # P,0.05 vs control; * P,0.05 *** P,0.001 vs <br /> scrambled siRNA group. <br /> doi:10.1371/journal.pone.0010501.g007                                            results [17,18,19] NFkB genes functionally <br />                                                                                  active undifferentiated hESC TLR4 PAMP unable to <br />                                                                                  produce NFkB translocation cells. <br /> Effect cytokines TLR4 PAMP LPS cell and <br /> NFkB activation hESC                                                          Discussion <br />    To investigate underlying mechanism lack of <br /> TLR responses hESC, function downstream pathways                          The primary findings present study first, that <br /> was investigated. Effects IL-1b CXCL8 release                           undifferentiated hESC immune response associated with <br /> undifferentiated hESC compared produced                        bacterial TLR activation, range bacterial PAMPs for <br /> TNFa, INFc IL-6. It cells activated                   TLR5. More surprisingly, cultures hESC differentiated for <br /> release increased levels CXCL8 IL-1b, TNFa, INFc                   4 months detectable immune response TLR2 <br /> by IL-6 (Fig. 8A). In line observations above, NFkB                     (Gram positive) TLR4 (Gram negative) activation. The TLR4 <br /> activated undifferentiated hESCs following 1 h stimulation               ligand, LPS failed activate NFkB hESCs, consistent with <br /> IL-1b whilst treatment cells LPS period                 notion cells functionally active TLR <br /> no effect (Fig. 8B). Clearly, evidenced data, confirm               responses. A population differentiated hESCs enriched for <br />  <br />  <br />        PLoS ONE | www.plosone.org                                           10                              May 2010 | Volume 5 | Issue 5 | e10501 <br />                                                                                                                      TLR Pathways Human hESC <br />  <br />  <br /> endothelial-like cells (hESC-EC) shared lack response                   The lack response bacterial challenge be <br /> through TLR4 active response TLR5.                              surprising undifferentiated hESC, given delay in <br />    HESC directly compared endothelial cells cultured              development innate immune sensing embryo. Full TLR <br /> from adult tissue, shown display positive                 responses generally develop near full <br /> bacterial-TLR response assessed increased CXCL8 release               term. Our results agreement [20] also <br /> when cells stimulated agonists TLR2 TLR4. Gene              reported lack responsiveness undifferentiated mouse ESC and <br /> expression profiles undifferentiated differentiated             differentiated derivatives LPS. These authors further <br /> hESC revealed relatively low levels TLR1 TLR4 hESC                 reported lack TLR4 expression cells to <br /> compared levels expressed endothelial cells. Moreover, TLR5           epigenetic modulation TLR4 gene promoter (methylated). <br /> expression relatively high differentiated hESC compared           However, recently study Lee co-workers [21] <br /> endothelial cells. Whilst gene expression data               demonstrate positive expression TLRs mouse ESCs. <br /> interpreted caution, pattern TLR expression fits          Moreover authors demonstrated long term <br /> with ability hESC respond PAMPs selective            exposure (24 days) increased proliferation differentiation of <br /> TLRs. It noted TLR2 acts dimer              mouse ESCs seen cells stimulated LPS (TLR4) or <br /> TLR1 TLR6. Loss TLR1 indirectly                       POLY:IC (TLR3). The apparently conflicting observations in <br /> influence activation TLR2.                                      mouse ESCs likely influenced different culture conditions, <br />    TLRs linked TIR domains adapter protein pathways              time LPS exposure, epigenetic factors differing clones of <br /> such MyD88. Our findings hESC purified hESC-               ESCs used. The likely similarities differences human <br /> EC responded IL-1b, agonist works                       mouse ESC cells respects remain subject <br /> independently TLRs TIR (TLR IL1 receptor) domain               investigation. Clearly data shows hESC resistant to <br /> linked receptor, releasing increased levels CXCL8                 stimulation PAMPs flagellin. We hESC <br /> activation NFkB. HESCs express number                 purified hESC-EC expressed TLR5 knockdown of <br /> NFkB related genes. When levels compared                 TLR5 siRNA inhibited CXCL8 production response to <br /> endothelial cells relative expression NFkB2, NFkB1A,                 flagellin cultures. This suggests TLR5 acts active <br /> NFKNFRkB, REL IKBkB similar hESC. However,                      sensor bacterial flagellin monomers hESC. Evidence the <br /> expression levels NFkB1 (gene p50) RELA (gene p65)            cytoprotective role TLR5 comes studies showing that <br /> were lower hESC endothelial cells. The lower gene                TLR5 ligation block apoptosis activating downstream <br /> expression clearly did limit NFkB activation CXCL8                   antiapoptotic genes [22]. Activation TLR5 protect against <br /> induction. It interesting note that, addition IL-1b,           tissue injury conditions involving high levels cell death [23]. <br /> hESCs responded releasing increased levels CXCL8                    The mechanism responsible lack LPS responsiveness <br /> stimulated cytokines including TNFa, INFc,               hESC determined. However, observed <br /> IL-6. These data suggests whilst hESC hESC-EC             mesoderm formation embryoid bodies (EB) hESC can <br /> not able sense respond bacterial PAMPs LPS,               inhibited (as shown mesoderm specific gene, Brachury <br /> they clearly intact active inflammatory signaling pathways.            silencing), challenge LPS, TLR4 PAMP [24]. This <br /> Gene expression analysis adapter proteins related              suggests low TLR4 levels able produce a <br /> genes showed differences hESC adult                    sufficient response affect differentiation processes pro- <br /> endothelial cells. Whilst levels TIRAP, TOLLIP TRAF6                 longed stimulation, results secondary LPS <br /> are similar undifferentiated hESC adult endothelial cells,           release cytokines MEF layer present EB <br /> the levels BTK, MyD88, TICAM1, TICAM2 IRAK2                     formation study. <br /> significantly lower. Interestingly pattern gene expression              Lack innate immune danger-sensing signals different <br /> changed somewhat hESC differentiated, upregu-                implications different cell lineages depending whether, like <br /> lation TICAM1, TICAM2 IRAK1. Changes expression                   endothelium, distinct role pathogens or, like <br /> of NFkB-related genes differentiation modest            cardiac myocytes, pathways involved directed <br /> consistent, RELA upregulated REL                     general danger signals. Endothelial cells only <br /> reduced (Table 2). However, functional levels MyD88                  provide barrier endocrine functions essential <br /> associated signaling proteins clearly sufficient               innate immune surveillance cells. Indeed, endothelial cells are <br /> differentiated undifferentiated hESC mount robust                  generally cell type pathogens encounter the <br /> inflammatory response IL-1b.                                             circulation. For target cells, including endothelial cells, is <br />    NFkB signaling differentiating hESC somewhat                 essential tissue derived hESCs express functional <br /> controversial, study hES-NCL1 line showing                    bacterial innate immune response. The use hESC-derived <br /> expression NFkB pathway components significant levels             endothelial cells tissue repair compromised <br /> in undifferentiated cells down-regulating differentiation        lack innate immune sensing. It remains seen <br /> and possibly controlling differentiation process way            vivo environment stimulate maturation the <br /> [19]. In contrast, Kang et al [17] low NFkB                   hESC-derived cells and, not, mix host grafted <br /> undifferentiated SNUhES3 MizES4 hESC lines compared                  endothelium tolerable maintain function. It be <br /> HEK haematopoetic progenitor lines (although MyD88                  necessary use pre-treatment strategies accelerate maturation <br /> TRAF2 equivalent), poor CXCL8 induction                  vitro mechanical hormonal stimulation, provision of <br /> TNFa. Differentiation associated increased NFkB                extracellular matrix co-culture cell types. For cell <br /> IL8 response TNFa, finding replicated mouse ESC            types directly involved immune sensing, consequences of <br /> [18]. The contradictory results studies suggest              relative insensitivity insult positive negative <br /> variation hESC lines, seen robust similar           aspects. Cell therapy will, cases, directed areas of <br /> expression levels key NFkB pathway components various             damage implanted hESC-derived cells introduced to <br /> undifferentiated hESC (H7 SHEF lines), little         areas hypoxia inflammation. Taking example cardiac <br /> change differentiation H7.                                          myocytes, lack TLR2 4 response predicted to <br />  <br />  <br />       PLoS ONE | www.plosone.org                                       11                              May 2010 | Volume 5 | Issue 5 | e10501 <br />                                                                                                                                                 TLR Pathways Human hESC <br />  <br />  <br /> increase resistance hypoxia [10] improve survival                             Table S1 CT values human aortic endothelial cells (HAEC, <br /> implantation scar border zone, sensitivity                             n = 3), undifferentiated H7 hESC (n = 3); undifferentiated SHEF2, <br /> inflammatory cytokine milieu infarcted heart                                   SHEF4 SHEF5 differentiated H7 1 month (n = 2), 3 <br /> retained. This suggest implantation success                                    months 4 months differentiation. Values .35 set to <br /> optimal later time period infarction, acute                                   35: considered undetectable. <br /> inflammation subsided scar established.                                       Found at: doi:10.1371/journal.pone.0010501.s002 (0.36 MB <br />                                                                                               DOC) <br /> Summary                                                                                       Table S2 Fold changes undifferentiated H7 hESC <br />   In summary, shown time hESC                               (n = 3) differentiated H7 1 month (n = 2), 3 months 4 <br /> sense respond bacteria bacterial PAMPs activate                             months differentiation. N.d.d. = detectable differen- <br /> TLR4, respond flagellin activates TLR5. We                               tiated; n.d.u. = detectable 3/3 2/3 undifferentiated and <br /> that pattern PAMP sensing consistent relative                             n.d. = detectable either. <br /> expression TLR genes hESCs. We despite having                                 Found at: doi:10.1371/journal.pone.0010501.s003 (0.14 MB <br /> no ability sense LPS, hESC respond robust manner                                   DOC) <br /> cytokines linked MyD88 NFkB transcription pathways. <br />                                                                                               Video S1 1 month differentiated H7 hESC showing clusters of <br /> These observations important suggest whilst                                  beating cardiomyocytes, shown Fig. 1C D. <br /> endothelial (and other) cells produced hESC display                                  Found at: doi:10.1371/journal.pone.0010501.s004 (2.26 MB <br /> phenotypic markers express mature immune                                        WMV) <br /> function. This implications strategy timing of <br /> implantation hESC derived cells tissue repair. <br />                                                         <br /> </body></html>