<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The Saccharomyces cerevisiae SEC14 Gene Encodes a <br /> Cytosolic Factor That Is Required Transport of <br /> Secretory Proteins Yeast Golgi Complex <br /> Vytas A. B n k t s , * D v d E. M l e h o r n , * Scott D. Emr,* n d R o b e r t Greene* <br /> * Department Microbiology, University Illinois Urbana-Champaign, Urbana, Illinois 61801; and <br />  Division Biology, California Institute Technology, Pasadena, California 91125 <br />  <br />  <br />  <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span>. We obtained characterized                         mass 37 kD, behaved predominantly a <br /> genomic clone SEC14, Saccharomyces cerevisiae                     cytosolic protein subceUular fractionation experi- <br /> gene product required export yeast                     ments. These data consistent notion that <br /> secretory proteins Golgi complex. Gene dis-                   SEC14p cytosolic factor promotes protein ex- <br /> ruption experiments indicated SEC14 essen-                  port yeast Golgi. Additional radiolabeling experi- <br /> tial gene yeast vegetative growth. Nucleotide se-                  ments revealed presence SEC14p-related <br /> quence analysis revealed presence intron                     polypeptides extracts prepared yeasts <br /> within SEC14 structural gene, predicted                    Kluyveromyces lactis Schizosaccharomyces pombe. <br /> synthesis hydrophilic polypeptide 35 kD mo-                 Furthermore, K. lactis SEC14p able func- <br /> lecular mass. In confirmation, immunoprecipitation ex-                 tionally complement S. cerevisiae sec14~ defects. These <br /> periments demonstrated SEC14p un-                             data suggested degree conservation SEC14p <br /> glycosylated polypeptide, apparent molecular                   structure function yeasts species. <br />  <br />  <br />  <br />        HE Golgi complex common eukaryotic cells.             established isolation characterization of <br />  <br /> T      This organelle fundamental importance the <br />        regulation intracellular protein traffic transits <br /> the later stages secretory pathway, recycling of <br />                                                                        yeast sec mutants exhibit conditional defects specific <br />                                                                        stages secretory pathway (23). Second, Dunphy et al. <br />                                                                        (6) obtained evidence suggesting measure func- <br /> membrane cell surface, communication                 tional conservation yeast mammals respect <br /> aspects endocytic pathway (14). Morphological stud-             cytosolic factors participate driving Golgi trans- <br /> ies indicated conservation basic Golgi structure             port functions vitro. The yeast sec mutant collection iden- <br /> throughout eukaryotic kingdom. The Golgi complex dis-              tifies nuclear genes, SEC7 SEC14, products <br /> plays segmented structure consisting set four,            appear play crucial roles protein transport the <br /> more, membraneous cisternae flattened centers di-             Golgi (23). <br /> lated rims (9). This structural motif coincides com-               Here report invertase secretion rapidly blocked <br /> partmentalized biochemical organization apparent            shift sec14&apos;s strain nonpermissive tempera- <br /> several experimental levels. In effort identify Golgi        ture, suggesting direct role SEC14p facilitating pro- <br /> components mediate protein transport or-              tein transport yeast Golgi identifying SEC14 as <br /> ganelle, Rothman colleagues (1, 2, 26, 27) devised            reasonable candidate detailed molecular analysis. As an <br /> an assay vesicular stomatitis virus G protein               initial step goal, present characterization of <br /> transport Golgi followed vivo             SEC14 gene. We SEC14 essential gene that <br /> vitro. These studies indicated intercisternal Golgi          belongs category rare S. cerevisiae genes con- <br /> transport mediated vesicles cytosolic proteins          tain introns, SEC14p abundant yeast poly- <br /> and ATP required. There indications fatty            peptide exhibits glycosyl modification. Moreover, <br /> acylation reactions GTP-binding proteins in-                subcellular fraetionation experiments demonstrated SEC14p <br /> volved process (12, 21).                               behave cytoplasmic species, associa- <br />    We chosen approach problem protein exit              tion particulate fractions observed. These data <br /> <span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span> Golgi genetic perspective using yeast              consistent SEC14p cytosolic factor pro- <br /> Saccharomyces cerevisiae model system. The reasons                motes protein transport yeast Golgi complex. Fi- <br /> for twofold. First, basic scheme yeast              nally, present evidence indicate structural func- <br /> secretory pathway mammals. This            tional conservation SEC14p yeast species. <br />  <br />  <br />  <br /> &#194;&#169; The Rockefeller University Press, 0021-9525/89/04/1271/11 $2.00 <br /> The Journal Cell Biology, Volume 108, April 1989 1271-1281           1271 <br />  We propose elucidation SEC14p function S. cere-                   terization experiments involved subcloning deletion analyses coupled <br /> visiae contribute general understanding pro-                    eomplementation tests. The subcloning analyses performed in- <br />                                                                                  troducing defined restriction fragments complementing insert of <br /> teins exit Golgi complex.                                               YCpS0 (SEC14) CEN4-URA3 vector pSEYC58 (7). Deletion analy- <br />                                                                                  ses involved removal defined fragments YCp50 (SEC14) re- <br />                                                                                  striction enzymes cleaved insert DNA (or cases <br /> Materials n d M e t h o d s                                                    cleaved YCp50 sequences immediately surround insert; e.g., <br />                                                                                  Eco RI, Sal I, Sma I), followed religation. All manipulations were <br /> Bacterial YeastStrains                                                       performed plasmid DNA purified E. coil relevant con- <br />                                                                                  structs confirmed correct reintroduction yeast pur- <br /> Escherichia coil strains MC1061 (5) KK2186, F&apos; traA36 laciqZAMl5             poses complementation. Complementation analyses involved transforma- <br /> proA+B+/A (lac-pro) supE thi rpsL endA sbcB15 hsdR4, used plas-         tion desired plasmids CTYI-IA selection Urn+ 30&quot;C. <br /> mid maintenance. Yeast haploid strains included YP54 (MATa ode2-101              Purified transformants screened ability form single colonies <br /> urn3-52 Ahis3-200 lys2-801 Atrpl ) isogenic MATa derivative, YP52,       nonpermissive temperature 36&#194;&#176;C. <br /> from R. Davis, SF292-2A (MATa sec14-3 &apos;s gal2 mal) R. Schek- <br /> man (University California). The prototypic sects strain used the <br /> studies CTYI-tA (MATa ura3-52 Ahis 3-200 lys2-801 secl4-Y~),               DNA Sequencing <br /> meiotic segregant SF292-2A x YP54 (this study). Strain CTY182            The nucleotide sequence analysis carried using dideoxy chain <br /> the isogenic SEC14 derivative CTYI-IA. Yeast diploid strains CTYD1        termination method (33). The inferred SEC14 protein sequence com- <br /> CTYD38 generated crossing CTYI-IA x YP54 YP52 &#195;&#8212;                      pared Protein Information Resource Library FASTP program <br /> YP54, respectively. Diploid strain CTYD2 isogenic CTYDI            ktup = 1 ktup = 2 modes (19). <br /> generated recombining sec 14Ah:HIS3 (see text) SEC14 allele <br /> of CTYD1. All strains constructed standard genetic techniques <br /> (35). Gene disruption technology described principle Roth-        SEC14 Antibodies <br /> stein (28). Schizosaccharomyces pombe Kluyveromyces lactis strains <br />                                                                                  A phage T4-1ysozyme-SECl4p protein fusion constructed subelon- <br /> SP255 (h- ode 6-210) MV&#194;&#162;98-8C (MAToe arg lys ura) obtained <br />                                                                                  ing 550-bp Ssp l-Hind III fragment SECI4 deletion construct into <br /> from Jo Anne Wise (University Illinois) Stephen Johnston (Duke <br />                                                                                  Sna BI-Hind III half-sites pCKR101. The resultant plasmid (pRE43) <br /> University), respectively. <br />                                                                                  programs synthesis 182-residue polypeptide (Mr = 21 kD) re- <br />                                                                                  sponse addition Isopropyl/3-D-thiogalactopyranoside host <br /> Media Reagents                                                               growth medium (the T4 lysozyme gene pTAC control vector <br />                                                                                  system). Residues I 7 polypeptide represent amino termi- <br /> YP, yeast minimal, defined minimal media described (35). <br />                                                                                  nus T4 lysozyme, codons 8 182 represent fusion codons generated <br /> Minimal media K. lactis S. pombe described (22, <br />                                                                                  construction, codons 9-181 represent amino acids 111-284 of <br /> 25). Standard E. coil media employed (20). All reagents invertase <br />                                                                                  SECI4p. Codon 183 amber codon. Thus, 173 182 total residues in <br /> assays, concanavalin A-Sepharose, protein A-Sepharose, protease <br />                                                                                  engineered polypeptide represent SECI4p primary sequence. E. coil <br /> inhibitors obtained Sigma Chemical Co. (St. Louis, MO). Re- <br />                                                                                  strain MCI061 (pRFA3) produce ,~10 nag fusion peptide per <br /> striction enzymes nucleic acid modifying enzymes Bethesda <br />                                                                                  liter cells 3-h induction 2.5 mM IPTG. The hybrid protein <br /> Research Laboratories (Gaithersburg, MD) Pharmacia Fine Chemicals <br />                                                                                  purified general procedure reported TrpE fusion proteins (16) <br /> (Piscataway, NJ). Reagents SDS-PAGE, agarose gel electrophoresis, and <br />                                                                                  reside pellet 0.1% SDS, 50 mM Tris (pH 7.5), 50 <br /> electrophoresis DNA sequencing reactions Bio-Rad Laborato-          mM EDTA wash insoluble fraction total cellular lysate. Anti- <br /> ries (Indianapolis, IN). Trans-<span id='am-5' about='rdfs:label' typeof='owl:Thing'>Label</span> (&gt;1,000 Ci/mmol) ICN Ra-           bodies raised primary injection 500/Lg hybrid protein pol- <br /> diochemicals (Irvine, CA). a-nP-dATP (800 Ci/mmol) obtained from <br />                                                                                  piteal lymph nodes New Zealand White female rabbits. After 6-wk rest, <br /> Amersham Corp. (Arlington Heights, IL). <br />                                                                                  500 ~.g boost injections administered intramuscularly 2 wk until <br />                                                                                  satisfactory response obtained. <br /> Invertase Secretion <br /> Strain CTYI-IA grown mid-logarithmic phase YP (2%) glucose             Immunoprecipitation SEC14 CPY <br /> medium 30&#194;&#176;C shaking. Ceils pelleted (2,000 g, 1 rain), washed <br /> with 2 vol water, resuspended 2 vol glucose (0.1%) YP medium.       Strain CTY182 grown early logarithmic growth stage (OD~o = <br /> The culture split incubated 30&#194;&#176;C 30 rain. After    0.5) 2 % glucose yeast minimal medium lacking methionine cysteine. <br /> time, culture shifted 36&#194;&#176;C; kept 30&#194;&#176;C. Samples       A 2-ml aliquot culture radiolabeled Trans-Label (150 t~Ci/ <br /> (500 pl) taken culture 0, i5, 30, 45, 60 rain postshift,   ml) 30 min 30&#194;&#176;C. Incorporation label terminated addi- <br /> immediately adjusted 10 mM NAN3, kept ice. The samples            tion NaN3 10 mM culture split equal aliquots. <br /> washed twice 1 ml cold 10 mM NaN3 resuspended 500 p.l             Protein TCA precipitated samples, followed cell lysis with <br /> of same. Each sample subsequently split. For each, nonpermea-        glass beads, boiling 1% SDS buffer, clarification extract centrifuga- <br /> bilized sample adjusted 500 #1 250 tzl 10 mM NAN3. The            tion, dilution extract immunoprecipitation buffer 0.5% <br /> samples permeabilized similarly adjusted 10 mM NAN3,             Tween 20 substituted 2 % Triton X-100 (34). Four yeast extracts <br /> 0.2 % Triton X-100 subjected cycle freeze-thaw. These samples      treated CPY antiserum (a) preimmune SECI4p serum, <br /> were used measure external total invertase activities, respectively,      (b) immune SEC14p antiserum, (c) SECI4p antiserum plus 50/zl cell- <br /> using assay Goldstein Lampen (13). The optical densities       free extract prepared E. coil MCI061 (pRFA3) induced T4 <br /> nonpermeabilized samples measured 600 nm Gilson spectro-          lysozyme-SEC14p fusion protein synthesis, (d) SECI4p antiserum plus <br /> photometer served normalize data. Invertase units expressed      50 td extract derived MCI061 (pRFA3) cells induced lysozyme- <br /> as nanomoles glucose produced min 30&#194;&#176;C.                                   SEC14p fusion protein synthesis. These E. coil extracts represented protein <br />                                                                                  obtained OD60o units cells. The remaining yeast extract was <br />                                                                                  treated 30 tA packed beads concanavalin A-Sepharose separated <br /> Transformations                                                                  bound soluble fractions (34). Each fraction treated CPY <br /> E. Coil transformed ampicillin resistance calcium chloride        SEC14p antisera, resolution immunoprecipitates achieved <br /> method (20). Yeast transformed Ura+ His + lithium acetate      SDS-PAGE aotoradiography described (34). CPY SEC14p <br /> procedure (15).                                                                  antibodies added 1:I,000 final dilution preimmune se- <br />                                                                                  rum added 1:500 final dilution. <br /> Cloning Characterization SEC14 <br />                                                                                  Detection SEC14p Cross-Reacting Species <br /> YCp50 (SEC14) plasmids identified transformation CTYI-IA <span id='am-7' about='protege:TO' typeof='owl:Thing'>to</span> <br /> Urn + and, 24 h incubation 30&#194;&#176;C, challenge selection       S. cerevisiae CTY182, K. lactis MW98-8C, S. pombe SP255, S. cere- <br /> plates nonpermissive growth conditions (36*C). More detailed de-            visiae CTYI-IA/pKLI2 (see text) grown O D ~ 0.5 their <br /> scriptions YCp50 (17, 32). Ura+ Ts+ transformants         respective glucose minimal media, appropriate nutritional supple- <br /> were screened proper behavior described text. Physical charac-     ments added (22, 25, 35) 30&#194;&#176;C shaking. One O1)60o <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> <br />  <br />  <br />  <br /> The Journal Cell Biology, Volume 108, 1989                                    1272 <br />  of cells yeast culture radiolabeled Tram-Label (150 ~tCi)          600 r- <br /> for 30 rain 30&quot;C. Recovery immunoprecipitates clarified extracts               | 30* C <br /> of TCA precipitates SEC14p antiserum preimmune serum were <br /> performed exactly described above.                                                 400 <br />  <br /> Fractionation SEC14p <br /> Strain CTY182 grown OD60o 0.5 glucose defined minimal                 200 <br /> medium shaking 22&#194;&#176;C. Some ODt0o units cells radiola- <br /> beled 50 #Ci Trans-Label ODt0o cells 20 min 22&#194;&#176;C. In- <br /> corporation label terminated addition NaN3 10 mM.            - ~ tO0 <br /> The ceils washed 1 ml 10 mM NaN3 converted sphero-           26_~    , <br /> plasts resuspension spheroplast buffer (50 mM KPO4 [pH 7.5], 1.4 M <br /> sorbitol, 50 mM B-mereaptoethanol, 10 mM NaN3), addition 2.5                                      I             I       I <br /> t~g/OD6oo cells oxalyticase (Enzogenetics, Corvallis, OR), incuba-        ~    00            15      30      45      60 <br /> tion 30&#194;&#176;C 15 min. This treatment resulted virtually quantitative        ~ 800 <br /> conversion yeast cells spheroplasts. Spheroplasts pelleted a <br />                                                                                                  36* C <br /> 2-min spin 500 g, resuspended 1.5 ml lysis buffer (10 mM Mops       ~&quot; <br /> [Ph 7.5], 0.3 M sorbitol, 2/zg/ml chymostatin, leupeptin pepstatin A, <br />  1 mM PMSF, 0.1 mM EGTA, 3 mM benzamidine, 30 U/ml aprotinin,                ~    600 <br />  100 t~g/ml BSA). 500 v.l lysate removed saved whole <br /> cell extract. The remaining 1.0 ml centrifuged 3 min 500 g <br /> to remove intact cells large debris, yielding low speed pellet,         400 <br /> and resulting supernatant spun 12,000 g 15 min 4&#194;&#176;C. The <br /> low speed 12,000 g pellets solubilized 150 td boiling buffer <br /> (50 mM &quot;Iris, pH 7.5, 1% SDS, 1 mM EDTA). The 12,000 g supernatant                    200 <br /> fraction centrifuged 1 h 100,000 g rotor (model SW 50.1; <br /> Beckman Instruments, Inc., Palo Alto, CA) ultracentrifuge (model                             O        O <br /> L8-70 M; Beckman Instruments, Inc.) generate high speed superna- <br /> rant high speed pellet fractions. The high speed pellet solubilized              0           I       I       I       I <br /> in 150 #1 boiling buffer. TCA used precipitate protein                    0      15      30      45      60 <br /> high speed supernatant cell fractions precipitates                            Time Minutes <br /> also solubilized 150 #1 boiling buffer. Samples (15 #1) clarified ex- <br /> tracts derived cell, low speed pellet, 12,000 g pellet, high      Figure 1. Invertase secretion rapidly blocked imposition of <br /> speed supernatant, 100,000 g pellet fractions treated CPY,         secl4&quot; defect. Strain CTYI-IA (secl4~, SUC2) cultured in <br /> SECI4p, invertase antibodies. Precipitates analyzed SDS-PAGE          YP medium containing 2% glucose 30&#194;&#176;C. The cells shifted <br /> and autoradiography described above.                                          YP (0.1% glucose) medium 30 min 30&#194;&#176;C, culture was <br />                                                                                  split, portion incubated 30&#194;&#176;C 36&#194;&#176;C. <br />                                                                                  At indicated times temperature shift, aliquots re- <br /> Results                                                                          moved total extracellular invertase determined for <br />                                                                                  each. o, extracellular invertase levels; o, total invertase activities. <br /> Invertase Secretion Responds Rapidly Loss                                  All points represent averages determinations all <br /> SEC14p Function                                                                  cases deviated &lt;10%. This figure depicts rep- <br />                                                                                  resentative experiment consistent results obtained other <br /> The classification secl&#194;&#162;&quot; mutants conditionally                      experimental trials. <br /> defective Golgi transport processes based ob- <br /> servation yeast exhibit marked exaggeration of <br /> Golgi-related structures (Berkeley bodies) nonpermis- <br /> sive conditions, intracellular invertase pool                  dent data shown Fig. 1, CTYI-IA cells incubated <br /> accumulates restrictive conditions consists en-                    permissive temperature (30&#194;&#176;C) exhibited extracel- <br /> zyme exhibits fully matured outer glycosyl chains (8,                       lular distribution essentially invertase was <br /> 23). Previous analyses secretion competence secl4~                     synthesized shift (i.e., t = 0). This point indicated <br /> yeast incubated nonpermissive conditions involved                      observation total invertase activity, measured <br /> measurement intracellular invertase acid phosphatase                      Triton X-IO0 permeabilized yeast, entirely accounted <br /> accumulation long exposure cells                    cell surface invertase measured parallel yeast <br /> restrictive temperature (23). To assess rate                    samples permeabilized. Different results <br /> secl4~ secretory block imposed, adopted experi-                         obtained CTYI-IA cultures incubated 36&#194;&#176;C. Even <br /> mental approach described Salminen Novick (32)                       early 15-min time point essentially invertase <br /> allowed compare efficiency invertase secretion                   synthesized postshift detected latent form. This re- <br /> secl&#194;&#162;&quot; yeast differential rate invertase synthesis.                    sult observed 30-, 45-, 60-min time <br /> CTYI-IA (secl4&quot;, SUC2) cells grown 30&#194;&#176;C YP                            points. Parallel experiments isogenic SEC14 strain, <br /> medium containing final glucose concentration 2 %;                        CTY 182, indicated absence significant intracellu- <br /> condition represses secretory invertase synthesis. The                      lar accumulation invertase 30 36&#194;&#176;C (not shown). <br /> cells shifted 0.1% glucose YP medium incubated                       These data indicated invertase secretion rapidly <br /> at 30&#194;&#176;C 30 min allow induction secretory inver-                    blocked shift sec14&quot; strain restrictive tem- <br /> tase synthesis. The culture subsequently split                      perature. <br /> equal portions, portion incubated 30&#194;&#176;C the <br /> other 36&#194;&#176;C. Samples collected various times post-                     Isolation SEC14 Gene <br /> shift each, total extracellular invertase activi-                   The recessive nature secl4&quot; allowed recovery SEC14 <br /> ties determined (see Materials Methods). As evi-                     clones existing DNA libraries complementation. <br />  <br />  <br />  <br /> Bankaitis et al. Identification SEC14 Protein                                  1273 <br />  Strain CTYI-IA (MATa, ura3-52, Ahis3-200 lys2-801, secl4-                                    pCTY14 unique Mlu I site insert subse- <br /> 3&quot;) transformed yeast genomic DNA library                                         quent transformation CTYI-IA (sec14&quot;) Ura &#195;&#183;. The re- <br /> maintained YCp50, shuttle plasmid carrying yeast cen-                                 sultant haploid strain (CTY175) verified experi- <br /> tromere URA3 gene. The desired transformants                                    enced expected integration event judged meiotic <br /> obtained simultaneous selection uracil prototrophy                                    segregation analysis URA3 Southern analysis of <br /> and temperature resistance (see Materials Methods).                                      genomic DNA prepared (data shown). CTYI75 <br /> From estimated 2 &#195;&#8212; 104 potential Ura + transformants,                                     mated YP54 (MATer, ura3-52, SEC14), resultant <br /> 20 recovered Ts&#195;&#183;. Linkage Ts&#195;&#183;                                     diploid induced sporulate, meiotic progeny <br /> character plasmid tested curing can-                               examined standard tetrad analysis. For total of <br /> didate transformants resident YCp50 plasmids                                     30 asci dissected, tetrads yield <br /> selecting fluoroorotic acid-resistant derivatives (3).                              viable spores exhibit absolute cosegregation <br /> Six candidate transformants yielded ts- derivatives                                   sec14&quot; YIp5-borne URA3 marker. These data in- <br /> upon curing events and, criterion, original                                 dicated complementing gene pCTY3 originated <br /> six Ura + Ts&#195;&#183; transformants considered identify pu-                                  region yeast genome homologous the <br /> tative SEC14 clones. The cognate plasmids recovered                                     SEC14 locus. The second criterion demanded recombi- <br /> from candidates purified. Retransformation                                nation vitro-generated disruption comple- <br /> purified plasmids CTYI-IA, selecting Ura +,                                    menting gene yeast genome result disruption of <br /> confirmed acquisition Ts&#195;&#183; unselected marker                                  SEC14. To construct disruption, 5.5-kb Eco RI-Xba <br /> all cases. Restriction mapping experiments demonstrated                                      I restriction fragment subcloned pCTY3 (Fig. 2) <br /> that complementing plasmids contained overlapping                                    corresponding sites pUC18 (38). The yeast HIS3 <br /> segments yeast genome. A detailed physical character-                                 gene used replace 168-bp Barn HI fragment ly- <br /> ization plasmid insert (pCTY3) performed                                     ing Eco RI-Xba I insert, resulting disruption <br /> (Fig. 2). By combination deletion mapping subelon-                                  complementing gene deletion insertion muta- <br /> ing analyses, complementing activity pCTY3 in-                                    genesis. The purified Eco RI-Xba I fragment used to <br /> sert localized 2.0-kb Mlu I-Eco RV restriction frag-                                transform diploid strain, CTYD1 (MATa/MATa, Ahis3- <br /> ment. Such experiments indicated Kpn I, Hpa                                    200/Ahis3-200, sec 14&quot;/SEC14), His +. Of 200 His + trans- <br /> I, Pvu II sites internal sec14&apos;s comple-                                 formants examined, 97 exhibit recessive <br /> menting gene; Bam HI sites (Fig. 2). Various                                 sec14&apos;~ <span id='am-86' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-87' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span>phenotype</span>. The simplest explanation data <br /> internal regions presumed SEC14 gene employed                                    SEC14 disruption recombined the <br /> as templates radiolabeled RNAs known polar-                                    wild-type SEC14 allele diploid test strain. Southern <br /> ity generated run-off transcription vitro, used                               analysis genomic DNA prepared representa- <br /> to probe total polyadenylated yeast RNA fractions. These                                 tive His+ts - transformants verified expected recom- <br /> Northern analyses revealed unique hybridizing RNA spe-                                     bination events occurred. These results, coupled <br /> cies 1.2 kb <span id='am-6' about='xsp:length' typeof='owl:Thing'>length</span> observed total                                integrative mapping data, proved authenticity of <br /> and poly A &#195;&#183; RNA fractions, transcribed di-                                   cloned SEC14 gene. The particular disruption allele that <br /> rection Mlu I Bam HI. This mRNA                                      described designated sec14A1::HIS3. As de- <br /> be approximately abundant TRP1 transcript                                   scribed below, sec14A1::HIS3 represents deletion se- <br /> steady state level (Cleves, A., V. Bankaitis, unpublished                                quences encoding carboxy-terminal end SEC14 <br /> data).                                                                                       protein. <br />    To prove complementing activity pCTY3 repre- <br /> sented authentic SEC14 gene, independent genetic <br /> criteria employed. In case, 3.6-kb Barn HI                                  SEC14 Encodes Essential Function <br /> fragment lies adjacent SEC14 (see Fig. 2) sub-                                   The conditional phenotype sec]4~&quot; yeast result from <br /> cloned integrating vector YIp5 (4). Integration                              synthesis temperature-sensitive gene product or, alterna- <br /> recombinant plasmid (pCTY14) genomic homolog                                        tively, sec14 null phenotype. To distinguish <br /> of 3.6-kb Bam HI fragment directed linearization                                  possibilities secl4Al::HlS3 recombined the <br />  <br />  <br />                                                                                                                       Figure 2. Physical map yeast <br />                                                                                                                       SECI4 gene. The ~13-kb genomic insert <br />                                                                                                                       sec14~s complementing plasmid <br />                                                                                                                       pCTY3 (see text) given. The SECI4 <br />                                                                                                                       coding region defined heavy ar- <br /> YC-P-sg-r-#     , <br />                               t         )It.          I I <br />                                                               Y//////////////////////A <br />  <br />  <br />                                                               31t               I        J Jt` <br />                                                                                                          V_C_p_5_O_   row, indicates direction <br />                                                                                                                       SEC14 transcription (see text). Re- <br />         E       E    Bg        Bo         BI~ B         MXb B XbXb              Xb       M P B           s            striction site abbreviations fol- <br />                                                                                                                       lows: Eco RI (E), Bgl II (Bg), Bam HI <br />                                                                                                                       (B), Pvu II (Pv), Mlu I (M), Xba I (Xb), <br />                                                                                                                       Eco RV (R), Hpa I (lip), Kpn I (K), and <br />                                                                                                                       Sal I (S). The 3.5-kb Barn HI fragment <br />                                   iii           i                  i <br />                                                                                                                       used integrative genetic <br />                           R         BPB          Hpg                   M                                              mapping experiments indicawxlby the <br />                                      !                                                                                hatched bar (see text). <br />  <br />  <br />  <br /> The Journalof Cell Biology,Volume 108, 1989                                                  1274 <br />  SEC14 gene yeast diploid strain CTYD1 (see above).         SEC14 mRNA encodes extreme amino terminus of <br /> The resultant diploid (CTYD2) subjected tetrad        gene product. As illustrated Fig. 3, consensus 5&apos; <br /> analysis. Dissection total 113 tetrads revealed     splice site (5&apos;CTATGT3&apos;; reference 24) immediately followed <br /> classes asci. The predominant class consisted 109 tet-      indicated SEC14 codon. This fol- <br /> rads uniformly yielded viable spores invia-      lowed second canonical splicing sequence (5&quot;I&apos;ACTAA- <br /> ble spores. The classes represented        CA3&apos;) observed 105 nucleotides downstream from <br /> asci yielded viable spores inviable spore      consensus 5&apos; splice site. The TACTAAC box known to <br /> per ascus, viable inviable spores ascus,    participate formation lariat splicing intermediate <br /> respectively. All 226 viable haploid progeny pheno-          (37). Normally, 3&apos; splice site defined AG <br /> typically His-ts- and, therefore, received sec14&quot;         encountered nucleotides downstream <br /> sec14A1::HIS3. No His + progeny obtained, suggesting         TACTAAC. For SEC14, AG encountered <br /> that SEC14 essential yeast gene. The minority tetrad        nucleotides downstream TACTAAC, a <br /> classes provided strong support conclusion      second AG observed 36 nucleotides downstream from <br /> presumably identified gene conversion events SEC14         TACTAAC. If AG marked 3&apos; end intron, <br /> locus strengthened correlation secl4Al::          predicted splicing event preserve reading frame <br /> HIS3 lethality. The parental diploid (CTYD1) exhibited        defined indicated initiator codon (Fig. 3). This reading <br /> excellent spore viabilities tetrad analysis.                 frame extend restriction sites that <br />    To verify lethal consequences receiving    genetically defined reside SEC14 (e.g., <br /> sec14A1::HIS3 trivial aspect          Kpn I, Hpa I, Pvu II, Bam HI sites). Sequence <br /> specific construction, second disruption SEC14 con-      analysis SEC14 cDNA clone confirmed authenticity <br /> structed. This accomplished inserting HIS3        intron, second AG defined 3&apos;-splice <br /> unique Hpa I site resides SEC14 structural        junction (Malehorn, D. E., V. A. Bankaitis, unpublished <br /> gene, resulting preservation amino-terminal        data). We noted open reading frame <br /> one-third SEC14 protein (see below). The new disrup-       significant length revealed sequence analy- <br /> tion recombined SEC14 alleles diploid    sis, available in-frame AUG corresponded <br /> yeast strain (CTYD38) homozygous Ahis3-200            MET75. We noted Barn HI sites lay ex- <br /> and SEC14. Analysis 30 tetrads SEC14::HIS3/           treme 3&apos;-end SEC14 coding region Hpa I <br /> SEC14 derivative showed 30 asci yielded viable       site sat one-third way SEC14 coding se- <br /> progeny apiece 60 surviving haploid         quence. As described above, interruption SEC14 sequence <br /> segregants His-. These results agreement        regions resulted lethal dysfunction of <br /> those obtained cross, confirmed essen-     SEC14p. Comparisons SEC14 nucleotide sequence, <br /> tial nature SEC14.                                             inferred primary sequence GenBank Protein <br />    Microscopic visualization inviable spores obtained      Information Resource Bank data bases failed reveal ho- <br /> from cross indicated spores germi-     mologies obvious significance. <br /> nated. In cases, <span id='am-8' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-9' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span>null mutants</span> completed two <br /> rounds cell division cessation growth. Thus,      Identification SEC14p <br /> concluded SEC14 encodes function essential     Our interpretation SEC14 nucleotide sequence data <br /> yeast vegetative growth.                                          predicted molecular mass 35 kD gene product. <br />                                                                   Furthermore, data indicated SEC14p ex- <br /> SEC14 Nucleotide Sequence                                         hibit asparagine-linked glycosyl modifications. To subject <br /> The localization SEC14 2.0-kb Mlu I-Eco RV restric-       predictions direct test, 20-kD fragment <br /> tion fragment, coupled identification marker re-      SEC14p engineered high-level production E. <br /> striction sites SEC14 gene (Fig. 2), facilitated de-   coli, partially purified, used source antigen <br /> termination SEC14 nucleotide sequence (Fig. 3). We         polyclonal antiserum raised rabbits <br /> interpreted sequence data indicate SEC14 encodes      (see Materials Methods). This antiserum used <br /> a polypeptide 304 residues (Mr = 35,032) termi-        specifically precipitate SEC14p radiolabeled yeast <br /> nated single UGA codon. This polypeptide inferred        extracts analysis. To visualize SEC14p, strain CTY182 <br /> to hydrophilic nature (27 % charged amino acids)      (SEC14) radiolabeled 30 min psS]methionine <br /> to exhibit net negative charge neutral pH (pI --- 5.3).      [35S]cysteine. Incorporation label terminated with <br /> Hydropathy analysis suggested regions        NaN3 TCA, SEC14p precipitated clari- <br /> inferred primary sequence exhibited net hydrophobic char-         fied extracts Con A-Sepharose bound unbound frac- <br /> acter. These regions bounded residues 165 194         tions derived extracts. CPY precipitated <br /> (inclusive), residues 212 245 (inclusive), respec-        experiments purposes. First, served a <br /> tively (Fig. 4). The longest uninterrupted stretch uncharged   reference SEC14p levels normalized. Sec- <br /> residues 14 extended GLY181 VAL194         ond, served control glycoprotein Con A binding <br /> SEC14p primary sequence. As obvious structure resem-           experiments. The immunoprecipitates evaluated by <br /> bling classical signal peptide discerned, un-     SDS-PAGE autoradiography. <br /> likely SEC14p itself passenger secretory           As data lane B Fig. 5 indicate, treatment radio- <br /> pathway. We noted SEC14p did display            labeled CTY182 extracts CPY SEC14p an- <br /> potential sites N-linked glycosyl modification.               tisera precipitated major radiolabeled species; 61-kD <br />    Our interpretation SEC14p sequence demanded         mCPY polypeptide 37 kD molecular mass. <br /> excision 156 nucleotide intron portion      Trace amounts CPY precursor forms ob- <br />  <br />  <br />  <br /> Bankaitis et al. Identification SECI4 Protein                  1275 <br />    ~. ~   -970        -geq       -999       .990       -98              -09                                               .91         .goq       -99~ <br /> kCCa~CTGAATAT CTTCCTCATT AATACCAATT GCATTATTA~ CAGGGGCTT~ T T C T A T A T ~                                 C C A G T A T C ~ A~AGGGAACT TCTCCCTAGT GAACCAATGT <br />  <br />  <br />   ATGGTGCG~           ATCAACCAT~ ATCCCTGAC~ TTACATCCA~ CTTTTGGGT~ AAATAGTGTA CACATT&amp;GGA ATAAAATATA TGAAATCATG TTTATTTACG <br />  <br />              -170             -19p              -~59               .1,~           -199              -,a.o           -~1.o                -1o9               -99 <br />   TGTTAGATGT TCCGTACAAC GCGACTGGCT AGTTCGTTTT TTCCACATTT TTTTTTTCAA CGTTGATCAA CAATAAAATG GTACATCTTC CTTTGTTTGT <br />  <br />                 .?o               &quot;*P            &quot;W                 .4p               -s~            -2p             .19                        ~                         ~9 <br />   GTCTTAAGT~ TCCTTTCCTG GTATATCGCT TGACCAGCGC CTGTAGCATC AAACCAAACC TTCATCGCCC TAAAACACA                                                        ..TG GTT ACA       GTATG~ <br />                                                                                                                                                 MET Val Thr /      Intron <br />  <br />  <br />   TGTTGCTTT~ ATTTACTTTT TCTTTTTTTG ACATTCATTG TGACAATA                                      CACATTCTTC AGATAGTTC            GTCTATATG~ AGCAAAAATG ATATATCAAT <br />  <br />  <br />   RAGTTTAe      IT ACARACACAA <br />                            &apos;&quot;                    &apos; &quot; TATGACTT <br />                                         GTGGTATTA~                  2     CTTTAAAT    2                             . <br />                                                                                                   C~.~ C~A GAA AAG GA~  . <br />                                                                                                                        TTT  . GAA <br />                                                                                                                            TTA . TCC TAC9~ CCT C~A AAC <br />   ......................................................                                      /   GIn Gln GIu Lys Glu Phe Leu Glu Set Tyr Pro Gin A~n <br />          91.o                             9e~ <br />   TGT CCT CCA GAT GCC TTG CCT GGT ACT CCA GGA AAT TTA GAC AGC GeT <br />                                                                           94o                         ~,~ <br />                                                                       GAG AAG GCA TTG GC~ GAA eTA AGA AAA CT~ TTG <br />                                                                                                                                          270                        999 <br />  <br />   Cys Pro Pro ASp Ala Leu Pro G1y Thr Pro Gly ASh Leu ASp Ser Ala Gin G1u Lys Ala Leu Ala GIu Leu Arg Lys Leu Leu <br />  <br />                        ~9.                             51~                             .p                           o49                               9~ <br />   GAA GAC aCT GGT TTC ATT GAA CGT TTA GAC GAT TCA ACT TTA CTA CGT TTT TTG AG~ GCC AGA AAA TTT GAT GTT CAA TTG GCT <br />   GIU ASp Ala Gly Phe Ile Glu Arg Leu ASp Asp SeE Thr Leu Leu Arg Phe Leu Arg Ala Arg Lys Phe Asp Val Gln Leu Ala <br />    975                   5e 9                4op             v.pe t 420              400                 450 <br />   A~   G ~ ATG TTT GAA AAC TGC GAA AAA TaG AGG AAG GAT TAT ~ T ~C~ GAC ACT ATC TTG CAJ~ GAT TTT CAT TAT GA~ GAA AAA <br />   LyS GIu Me~ P h e GIu ASh Cys GI~ Lys Trp Arg Lys ASp Tyr aly Thr Asp Thr Ile Leu Gln ASp Phe His Tyr Asp Glu Lys <br />  <br />                  4*~                            490                             49~                           51o                               S2S                       549 <br />   CCA TTG ATT GCC AAA TTC TAC CCA CAA TAT TAT CAT AAA Ace GAT AAA GAT GGC CGC CCA GTA TAT TTT GAA GAA TTA GGT GeT <br />   Pro Leu Ile Ala Lys Phe Tyr Pro GI~ Tyr Tyr His Lys Thr Asp Lys Asp Gly Arg Pro Val Tyr Phe Glu G1u Leu Gly Ala <br />  <br />                              5~                              s?p                             ~9                             909                              91p <br />   GTT AAC TTA CAT GAA ATG AAC AAG GTT ACC TCT GAA GAG AGG ATG TTG AAA AAC TTG GTT TGG GAA TAC GAA TCT GTC GTT CAA <br />   Vll ASh Leu His Glu M@~ Ash Lys Val Thr Set Glu GI~ Arg Met Leu Lys ASh Leu Val Trp GIu Tyr Glu Set Val Val Gln <br />  <br />        ..o                 94~                 9~                  u~                  99~                 ?os <br />   TAC AGA TTA CCT GCC TGT TCA AGA GCT aCT GGT CAC CTA GTG GAA ACT TCA TGT ACA ATT ATG GAT TTG AAA GGT ATC TCC ATA <br />   Tyr Arg Leu Pro Ala Cys Ser Arg A1a Ala Gly His Le~ Val GIU ThE Set Cys Thr Ile Met Asp Leu Lys Gly Ile Set lle <br />  <br />                  ?~p                  7~S                  750                 ?~0                 75o <br />   TCT AGT GCA TAC AGT GTT ATG TCA TAT GTT AGG GAA GCC TCC TAC ATA AGT CAA AAC TAT TAC CCC GAA CGT ATG GGT AAA TTT <br />   Ser S e t Ala Tyr Ser Val Met Set T y ~ Val Arg GIU Ala Set Tyr Ile Set Gin Asn Tyr Tyr Pro GlU Arg Met G1y Lys Phe <br />    ?|S                 119                  1~6                  145               bmHI                  9TO <br />   TAC ATe ATe AAC GCG CCA TTC GGT TTC TCT A C C GCA TTT AGG CTA TT~ AAA CCT TTC TT~ G~T CC&amp; GTC ACT GTT TeA AAG ATT <br />   Tyr Ile Ile ASh Ala Pro Phe G1y Phe Set Thr Ala Phe Arg Leu Phe Lys Pro Phe Leu Asp Pro Val Tbr Val Set Lys Ile <br />  <br />            US                                                                              54&#194;&#167;                 290 <br />   TTT ATC TTG GGT TCT TCT TAC CA~ AAG GAA TTA TTA AAG CAA ATT C   ~       AAC TTA CCA GTC AAA TTT GGC GGT AAG TCT <br />   Phe Ile Leu Gly SeE Ser Tyr Gln Lys Glu Leu Leu Lys Gin Ile Pro Ala Glu Asn Leu Pro Val LyS Phe G1y Gly Lys Set <br />  <br />                    97~                 ~o           E~oRV I ~    k~.l     BW. HI I~0                 loss <br />   GAA GTT GAT GAA TCC AAG GGT GaG TTA TAC CTA TCC G.%T ATe GG~ CCA TaG A G ~ _ ~     AAG TAT ATT GGA CCG GAA GGT GAA <br />   Glu Val Asp Glu Set Lys Gly Gly Leu Tyr Leu Set Asp Ile aly Pro Trp Ar~ Asp Pro Lya Tyr Ile Gly Pro GIU Gly Glu <br />  <br />       1050                                  1oep       I~        11~        111o       Ilqo      llSO        11%0 <br />   GCT CCG GAA GCC TTT TCG ATG AAA TGA TATTATTTTTAAAGACATTTTTTTTTCG~ GAGAGAAAA~ AGGAGTTCTA AGTAACGAT~ TTTTACAACT <br />   Ala Pro G1u Ala Phe Ser Met Lys *** <br />  <br />           11S.0      1150      1170       1180       1199       1200        1210      1220       1900        1240 <br />   GAGATAAGTA CAAGTATGC~ TATTTACTCT ATTCTACTTT TTTGAATAGG CATAGTTCAC ACTTTTATCT CCTTAGTTT~ TACTACTCT~ TTTTTATTTT <br />  <br />          1250        1290      1270       12|p       1290       1300        1310      1329       I~9         194o <br />   TTTTTTTTC~ TCCACTCAG~ TATTAGATTA TTTATTATTT TATTATCTGT GGTATCATTT TCTACAGCCA TACCTTGATT TTAGTTTATT TTGTTGGCA~                                                                 Figure 3. Nucleotide sequence of <br />             1000              1090              1570               1590           1990             1400 <br />   TTATATACA~ TTATCTTATA GAATACATA~ GCACGTATTA AACAATAAAA AGAACATAA~ CAAAAATGAA ACCTCGCTA~ AATAAAACT~ TATTACGCA~ <br />                                                                                                                     1410             1420                  1409       1449      SEC14. The DNA sequence the <br />                                                                                                                                                                                 SEC14coding strand pre- <br />         1400        1490      1470        1490      1499        1500       |5~0      1020       IS~ E~o~    1540 <br />  TGCAGTGCG~ CGTAAAGAGT ATTCATTAGA AGTACAATGG TAGCCCTATT TGGTTGCAA~ ATGTTTTAGC AAAGAGATTA GTAGGCTAG~ ATATC~AGTC                                                                  dicted primary sequence its <br />         155o        I~4~      I$7~        15~I0     199~       1800        1510      1520       1290        1945 <br />                                                                                                                                                                                 product shown. The intron is <br />  CATGCCCCTT TATCACATA~ TGTAATGATA GATGCACTCC GGAAAAGTAA AAAAAGTTTT TCAGGTTACA ATTGTGCAAA TTGCGAGTCT ATCTCGAAT~ <br />                                                                                                                                                                                 bounded hash marks is <br />                     11mo      197o        11~o      158o       I?O~        171o      172o        1730      1740 <br />  ATTTTGCAGC GCTCATTTCA CGGTTGAGCC GTTTTGTAT~ ACAAGCCAA~ TTTAACATTT TATTTTAACA GGGTTCACCG ~.TTATATTC CC~ATTGCT <br />                                                                                                                                                                                 indicated dotted underline. <br />             I ?Sp      1~        177o               1796 <br />                                                                                                                                                                                 Hallmark restriction sites are <br />  .I~I,TCTTTTAC AATTCATGCT TTTGTTCTTC TCTCTGTAACTCTCCTG                                                                                                                          referred text shown. <br />  <br />  <br /> served. The 37-kD species apparent preim-                                                                            CTY182 extracts yielded CPY immunoreactive spe- <br /> mune serum substituted SEC14p antiserum                                                                            cies. No SEC14p recovered fraction. The 37- <br /> precipitation reaction (lane A). Furthermore, precipitation                                                                    kD SEC14p quantitatively recovered Con A su- <br /> this polypeptide competed inclusion unla-                                                                       pernatant fraction (lane D) and, expected, CPY species <br /> beled extract E. coli cells induced syn-                                                                   recovered soluble fraction. Taken together, <br /> thesis 21-kD SEC14p antigen (lane E), effec-                                                                       data demonstrated SEC14p abundant and <br /> tively competed inclusion extract E. coli cells                                                                   unglycosylated polypeptide 37 kD apparent molecular <br /> were induced antigen synthesis (lane F). This com-                                                                       mass. These data provided strong confirmation our <br /> petition experiment, coupled data obtained                                                                          interpretation sequence data; especially respect <br /> with preimmune serum, identified 37-kD polypeptide                                                                        assignment initiator codon (Fig. 3). <br /> as yeast SEC14p. Pulse-chase experiments indi- <br /> cated SEC14p stable mCPY yeast cultures un- <br /> dergoing logarithmic growth (manuscript preparation).                                                                          SEC14p Resides Primarily Cytosolic Fraction <br /> Thus, densitometric estimations relative levels                                                                         The nucleotide sequence data indicated SEC14p hy- <br /> SEC14p mCPY suggested species pres-                                                                           drophilic character suggested cytoplasmic location for <br /> ent similar intracellular concentrations (within factor                                                                   polypeptide. To test directly, employed standard <br /> two; data shown).                                                                                                             differential centrifugation techniques fractionate radiola- <br />    As shown Fig. 5, lane C, Con A bound fraction                                                                      beled yeast lysates various membrane enclosed cyto- <br />  <br />  <br />  <br /> The Journal o f Cell Biology, V o l u m e                    108, 1989                                                            1276 <br />   1.599 <br />  <br />  <br />  <br />  <br />                                                                                                      F~gure4. Hydropathy analysis SEC- <br />                                                                                                      14p. The inferred SEC14p amino <br />                                                                                                      acid sequence used generate a <br />                                                                                                      hydropathy profile method of <br />                                                                                                      Kyte Doolittle (18). A scanning <br />                                                                                                      window seven residues used. <br />                                                                                                      Amino acid residues numbered <br />                                                                                                      horizontal axis net <br />                                                                                                      hydrophobicity values indicated <br />                                                                                                      vertical axis. H)xlrophilic values <br />                                                                                                      lie horizontal axis while <br /> -2.6i8                                                                                               hydrophobic values lie above. <br />  <br />  <br /> plasmic compartments (see Materials Methods). The                  organelles remain intact cell lysis. The condi- <br /> presence SEC14p various fractions deter-               tions radiolabeling designed reveal CPY the <br /> mined quantitative immunoprecipitation analysis             pl p2 forms enzyme. These precursor forms of <br /> fraction SEC14p antiserum. The osmotic lysis proce-               CPY employed lumenal markers ER early <br /> dure employed similar previously re-              Golgi complex, late Golgi complex, respectively (36). <br /> ported (10, 29), expected membrane-enclosed            The constitutively expressed form invertase employed <br />                                                                        cytoplasmic marker. <br />                                                                           From data shown Fig. 6, clear pl and <br />                                                                        p2 CPY sedimented quantitatively 12,000 g and <br />  <br />  <br />  <br />  <br /> Figure 5. Identification SEC14p. Strain CTY182 (SEC14) was <br /> grown glucose (2 %) yeast minimal medium allowed incor-      Figure 6. Fractionation SEC14p yeast cytosol. Strain <br /> porate radiolabel 30 min 30&#194;&#176;C. Clarified extracts recov-   CTY182 grown minimal defined glucose (2%) medium, ra- <br /> ered immunoprecipitations performed CPY antiserum         diolaheled Trans-Label, osmotically lysed, lysate sub- <br /> variable SEC14p sera: lane A, CPY antiserum preimmune              jected rounds differential centrifugation (see Materials <br /> SEC14p serum; lane B, CPY antiserum immune SEC14p se-              Methods). The cell (WC), 500 g pellet (LSP), 12,000 g <br /> rum. Lanes C D represent precipitations CPY SECI4p          pellet (P12), 100,000 g supernatant (S/00), 100,000 g pellet <br /> from Con A bound Con A superuatant fractions derived          (PIO0) fractions probed presence precursor forms <br /> the clarified extract, respectively. Lanes E F indicate pro-   CPY, cytoplasmic form invertase, SEC14p. The pl and <br /> files obtained clarified extract treated CPY    p2 CPY, indicated left, served lumenal markers ER and <br /> and SEC14p antibodies presence E. coli extract de-    early Golgi complex, late Golgi complex, respectively. Cyto- <br /> rived cells induced SEC14p antigen synthesis (lane E),    plasmic invertase served cytoplasmic marker. All im- <br /> or presence ofE. coli extract obtained cells induced   munoprecipitates displayed recovered aliquots of <br /> high-level synthesis SEC14p antigen poly-     clarified lysates. Note WC fraction derived <br /> clonal SEC14p antiserum raised (lane F). The precursor         one-half material used generate other <br /> mature forms glycoprotein CPY indicated left,      fractions (see Materials Methods). We estimate recoveries <br /> the SECI4p.                                                            antigen fractionation exceed 80 %. <br />  <br />  <br />  <br />  <br /> Bankaitis et al. Identification SEC14 Protein                        1277 <br />  100,000 g pellet fractions exhibited ,o60 40% dis-    tated S. pombe extracts. The K. lactis S. pombe <br /> tribution material pellet fractions,        SEC14p immunoreactive polypeptides specifically rec- <br /> respectively. The virtual absence soluble pl p2 CPY       ognized SEC14p antibodies since, cases, this <br /> indicated integrity ER Golgi complex         binding quantitatively competed 21-kD T4 lyso- <br /> been satisfactorily maintained lysis procedure. In con-   zyme-SEC14p fusion protein SEC14p an- <br /> trast, radiolabeled invertase recovered exclusively     tiserum raised (data shown). We noted K. lactis <br /> the 100,000 g supernatant fraction, expected soluble    S. pombe incorporated one-third one-seventh <br /> and membrane enclosed species. An identical fraction-        total radiolabel incorporated S. cerevisiae <br /> ation profile observed phosphoglycerate kinase, an-      strain CTY182, respectively. Sample loadings normal- <br /> other cytoplasmic species (data shown). We noted        ized basis incorporation values. We emphasize <br /> the significant absence cytoplasmic invertase lumenal     that, criterion, S. pombe extract underrep- <br /> proCPY forms low speed pellet fraction, suggesting        resented factor respect S. cerevisiae <br /> that quantitative lysis spheroplasts achieved.       K. lactis extracts used immunoprecipitation ex- <br />    SEC14p exhibited fractionation profile        periment shown Fig. 7. Consequently, SEC14p antise- <br /> similar cytoplasmic invertase. In experiment      rum appeared recognize K. iactis S. pombe poly- <br /> shown, 65 % radiolabeled SEC14p recovered        peptides approximately equal efficiencies. These data <br /> from 100,000 g supernatant fraction. However,           demonstrated K. lactis S. pombe exhibited <br /> 35 % total, radiolabeled SEC14p cosedimented         polypeptides authentically related S. cere- <br /> material pelleted 500, 12,000, 100,000 g (Fig. 6).   visiae SEC14. Since yeasts span evolutionary distances <br /> The proportion SEC14p observed sediment        great S. cerevisiae S. pombe those <br /> the pellet fractions varied somewhat method cell     yeasts mammals (30), clear <br /> lysis experiment experiment (not shown). We          SEC14p epitopes conserved wide phylogenet- <br /> have reproducibly observed range 20--40%. The sig-          ic boundaries. <br /> nificance associations pellet presently un-       Do elements structural conservation SEC14p un- <br /> clear exclude possibility sediment-    derly functional conservation well? In initial effort <br /> able SEC14p artifactually retained fractions. We     address question, attempts isolate geno- <br /> note essentially invertase phosphoglycerate kinase    mic clones K. lactis S. pombe genes com- <br /> was observed pellet conditions. The     plement, suppress, sec14-Y~ <span id='am-164' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-165' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span>mutation</span> CTYMA. <br /> conclusion derived data predominant       We immediately successful isolating such <br /> fraction SEC14p localized yeast cytosol.           clones K. lactis genomic library propagated the <br />                                                                  yeast shuttle vector YEp24 (generously provided J. Sal- <br />                                                                  meron S. Johnston; references 28, 31). The details will <br /> SEC14 Epitopes Are Conserved                                  described elsewhere. Radiolabeling CTYI-IA bearing <br /> Vefdely Divergent Yeasts                                         representative complementing K. lactis clone (pKL12), <br /> Dunphy et al. (6) obtained compelling evidence indicat-     followed precipitation SEC14p immunoreactive mate- <br /> ing S. cerevisiae mammals employ functionally in-       rials cell-free extracts resolution products by <br /> terchangeable cytosolic factors drive intercisternal Golgi    SDS-PAGE autoradiography, yielded profile shown <br /> transport vitro. These data suggest elements struc-   Fig. 7. Clearly, major SEC14p species minor <br /> tural conservation exist eukaryotic compo-           SEC14p species observed. The SEC14p lowest ap- <br /> nents involved essential Golgi functions. To test     parent molecular mass, major species, migrated fash- <br /> SEC14p exhibits structural conservation widely         ion identified S. cerevisiae SEC14p. The doublet <br /> divergent organisms, polyclonal SEC14p antiserum         migrating 39-40 kD apparent molecular mass repre- <br /> employed specific probe presence SEC14p          sented SEC14p antigenic homolog K. lactis. We do <br /> homologs extracts prepared yeast species.          know doublet observed K. lactis <br /> These included budding yeast Kluyveromyces lactis        SEC14p expressed S. cerevisiae but, apparently, not <br /> the fission yeast Schizosaccharomyces pombe. S. cerevisiae       K. lactis. We noted synthesis K. lactis SEC14p <br /> CTY182, K. lactis MW98-8C, S. pombe SP255 ra-           doublet common independent sec14&apos;~ <br /> diolabeled incorporation label terminated        complementing clones obtained (not shown). <br /> TCA. Clarified extracts yeast strain prepared     These results suggested that, case K. lactis, <br /> and split equal aliquots. One aliquot treated       structural conservation SEC14p epitopes underlies <br /> with preimmune serum mixed              conservation protein function well, the <br /> SEC14p antiserum. The antibody complexes washed             K. lactis SEC14p substitute S. cerevisiae. <br /> under stringent conditions evaluated SDS-PAGE            A trivial explanation functional rescue sec14&apos;~ <br /> autoradiography.                                                 defect imagined situation K. lactis <br />    As shown Fig. 7, treatment yeast extracts      SEC14p S. cerevisiae Secl4p ~sinteract. Such in- <br /> preimmune serum failed precipitate radiolabeled spe-      teraction make defective Secl4p ~ effi- <br /> cies. Although relevant portion autoradio-       cient. If case, expect K. lactis <br /> gram shown, preimmune serum lanes blank           SEC14 overcome thehaploid <span id='am-242' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-243' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-249' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-261' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-267' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-271' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-284' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-289' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-309' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-318' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-319' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span>lethal phenotype</span> S. cere- <br /> all cases. However, SEC14p antiserum did precipi-      visiae sec14 null mutation. To test directly, pKL12 was <br /> tate single protein yeast extracts. Not    transformed CTYD2 (MATa/MATc~, ura3-52/ura3-52, <br /> was expected 37-kD polypeptide recovered S. cere-       Ahis3-2OO/Ahis3-200, sec14&quot;/secl4A 1::HIS3 ) linear trans- <br /> visiae extracts, 39-kD immunoreactive species de-      formation Ura &#195;&#183;. The resultant diploid subjected <br /> tected K. lactis extracts, 35-kD protein precipi-   tetrad analysis. Of I0 tetrads analyzed, seven <br />  <br />  <br /> The Journal Cell Biology, Volume 108, 1989                    1278 <br />                                                                                  Figure 7. SEC14p homologs K. lactis S. <br />                                                                                 pombe. S. cerevisiae CTY182, K. lactis MW98-8C, S. <br />                                                                                 pombe SP255, S. cerevisiae CTYI-IA (see14~) <br />                                                                                 carrying K. lactis genomic DNA clone (pKL12) that <br />                                                                                 complements see14~ lesion (see text), cul- <br />                                                                                 tured respective glucose minimal media at <br />                                                                                 30&#194;&#176;C radiolabeled Trans-Label 30 min. <br />                                                                                 Clarified extracts prepared culture <br />                                                                                 split. One aliquot treated preimmune <br />                                                                                 SEC14p serum (P), treated with <br />                                                                                 SEC14p antiserum (I). Immunoprecipitates re- <br />                                                                                 solved SDS-PAGE autoradiography. Sample <br />                                                                                 loadings normalized function total incor- <br />                                                                                 poration radiolabel TCA-precipitable counts. <br />                                                                                 By virtue poor incorporation radio- <br />                                                                                 label, S. pombe extract underrepresented, rel- <br />                                                                                 ative extracts, factor im- <br />                                                                                 munoprecipitation reactions. <br />  <br />  <br />  <br />  <br /> to yield viable Ura+ haploid progeny. Each           authenticity clone obtained integrative <br /> seven tetrads exhibited 2 His+/2 His- segregation pat-      mapping gene disruption experiments. The gene disrup- <br /> tern (data shown). As established sec14-          tion experiments indicated SEC14 essential gene. <br /> AI::HIS3 haploid lethal mutation (see above), data      Nucleotide sequence analysis SEC14 identified 1,071-bp <br /> demonstrated pKL12 suppressed see14 disruption            structural gene potential encoding 304-amino <br /> allele. Further confirmation conclusion obtained      acid polypeptide mol wt = 35,032 (Fig. 3). This in <br /> from experiments haploid S. cerevisiae strains carry-        good agreement Northern blot analyses sized the <br /> ing pKL12 shown successfully receive sec14A1::HIS3         SEC14 mRNA ,x,l.2 kb. An interesting feature SEC14 <br /> or SEC14::HIS3 (see above) linear transformation,           sequence clear presence 156-bp intron ex- <br /> execute proper replacement SEC14 null alleles.       treme 5&apos;-end structural gene. Very S. cerevisiae <br /> Taken together, data demonstrated directly          genes nucleotide sequences determined ex- <br /> K. lactis SEC14p functionally homologous S. cere-       hibit introns. Those do, single exception, exhibit <br /> visiae SEC14p.                                                     intron extreme 5&apos;-end the <br />                                                                    gene (11). In respect, SEC14 intron typical. Ex- <br />                                                                    amination inferred SEC14p primary sequence revealed <br /> Discussion <br />                                                                    acidic polypeptide hydrophilic character lacked any <br /> The classification secl4 &apos;s mutants defective       obvious signals membrane insertion entry yeast <br /> Golgi transport function based cytological bio-     secretory pathway (Fig. 4). Comparisons SEC14 nucleo- <br /> chemical data (8, 23). These include observation          tide sequence inferred SEC14p sequence GenBank <br /> secl4 &apos;s yeast exhibit marked exaggeration Golgi-derived      PIR data bases failed reveal homologies obvi- <br /> structures nonpermissive conditions, in-        ous significance provided clue the <br /> tracellular pool secretory glycoprotein accumulates        biochemical nature SEC14p function. <br /> under conditions consists polypeptides exhibit           Some general insight possible functions SEC14p <br /> fully matured outer chain glycosyl modifications. Although         provided fractionation experiments. We identified <br /> such data constitute persuasive evidence involvement        S. cerevisiae SEC14p unglycosylated polypep- <br /> of SEC14p yeast Golgi function, data allow         tide 37 kD apparent molecular mass (Fig. 5). These <br /> one distinguish involvement direct       determinations excellent agreement predict- <br /> an indirect nature.                                                ed nucleotide sequence. Furthermore, S. cerevisiae <br />    We obtained evidence support direct participa-        SEC14p quite abundant intracellular spe- <br /> tion SEC14p yeast secretory processes. Invertase secre-      cies; approximately abundant vacuolar CPY. Frac- <br /> tion demonstrated dramatically blocked       tionation radiolabeled yeast lysates indicated SEC14p to <br /> shift see14 &apos;~ yeast restrictive conditions. Furthermore,    primarily reside 100,000 g supernatant fraction (Fig. <br /> this secretory block:was imposed rapid kinetics (Fig.         6). As result, models requiring lumenal integral mem- <br />  1). A rapid marked inhibition invertase secretion    brane disposition SEC14p rejected. The present data <br /> more likely represent primary defect yeast secretory       suggest SEC14p represents cytosolic factor in- <br /> function, manifestation secondary          volved promoting intercisternal transport secre- <br /> effect resulting general metabolic difficulty. Sec-    tory polypeptides Golgi Complex or, alternatively, <br /> ondary effects reasonably expected exhibit      facilitating packaging secretory protein ap- <br /> a longer course onset, specific phenotype terms   propriate vesicle carrier final delivery. The requirement <br /> of organelle exaggeration.                                         cytosolic factors driving vesicle-mediated transport of <br />    To gain insight role SEC14p yeast secre-    protein Golgi complex well-documented (1, <br /> tory metabolism, isolated characterized SEC14      2, 6, 26, 27). The significance pelletable forms of <br /> gene S. cerevisiae genomic library. Genetic proof       SEC14p remains determined. <br />  <br />  <br /> Bankaitis et al. Identification SEC14 Protein                   1279 <br />     Although precise biochemical mechanism SEC14p                              ments Golgi measured coupled incorporation N-acetylglu- <br />                                                                                       cosamine. Cell. 39:405-416. <br /> function remains elusive, obtained evidence sug-                     2. Balch, W. E., B. S. Glick, andJ. E. Rothman. 1984. Sequential intermedi- <br /> gest conservation SEC14p structure. Such evidence                            ates pathway intercompartmental transport cell-free system. <br /> derived experiments polyclonal antisera directed                           Cell. 39:525-536. <br />                                                                                 3. Boecke, J. D., F. LaCroute, G. R. Fink. 1984. A positive selection <br /> against S. cerevisiae SEC14p employed probes                              mutants lacking orotidine-5&apos;-pbosphate decarboxylase activity in <br /> homologs widely divergent yeast species. Immunoprecipi-                           yeast: 5&apos;-flaoro-orotic acid resistance. Mol. &amp; Gen. Genet. 197:345-346. <br /> tations SEC14p cross-reactive materials K. lactis                   4. Botstein, D., S. C. Falco, S. Stewart, M. Brennan, S. Scherer, D. I. Stinch- <br />                                                                                      comb, K. Struhl, R. Davis. 1979. Sterile host yeast (SHY): eukary- <br /> S. pombe extracts identified polypeptides 39 35                          otic biological containment recombinant DNA experiments. <br /> kD apparent molecular mass, respectively, behaved                             Gene. 8:17-24. <br />                                                                                 5. Casadahan, M. J., S. N. Cohen. 1980. Analysis ofgene control signals <br /> as authentic homologs S. cerevisiae SEC14p (Fig. 7).                           DNA fusion cloning Escherichia coll. J. Mol. Biol. 138: <br /> This remarkable result considering wide diver-                        179-207. <br /> gence S. cerevisiae S. pombe (30). At                      6. Dunphy, W. G., S. R. Pfeffer, D. O. Clary, B. W. Wattenberg, B. S. Glick, <br />                                                                                      L E. Rothman. 1986. Yeast mammals utilize similar cytosolic <br /> the case K. lactis, observed antigenic homology un-                           components drive protein transport Golgi complex. Proc. <br /> derlay functional homology well. We demonstrated                           Natl. Acad. Sci. USA. 83:1622-1626. <br /> genomic clones K. lactis DNA complemented S.                        7. Emr, S. D., A. Vassarotti, J. Garrett, B. L. Geller, M. Takeda, M. G. <br />                                                                                       Douglas. 1986. The amino terminus yeast F~-ATPase ~-subunit <br /> cerevisiae secl4 &apos;s defect directed synthesis K. lac-                     precursor functions mitochondrial import signal. Z Cell Biol. 102: <br /> tis SEC14p (Fig. 7). Since S. pombe SEC14p did ap-                            523-533. <br />                                                                                 8. Esmon, B., P. Novick, R. Schekman. 1981. Compartmentalized as- <br /> pear cross-reactive S. cerevisiae SEC14p                          sembly oligosaccharides exported glycoproteins yeast. Cell. <br /> than K. lactis homolog, predict S. pombe                          25:451-460. <br /> SEC14p prove functionally homologous. Ex-                       9. Farquhar, M. G. 1981. Membrane recycling secretory cells: implications <br />                                                                                       traffic products specialized membranes Golgi com- <br /> periments currently underway test hypothesis.                            plex. Methods Cell Biol. 23:399-427. <br />    Is measure structural functional conservation                   10. Feldman, R. I., M. Bemstein, R. Schekman. 1987. Product SEC53 <br /> SEC14p likely observed divergent yeasts?                             required folding glycosylation secretory proteins lu- <br />                                                                                      men yeast endoplasmic reticalum. J. Biol. Chem. 262:9332-9339. <br /> Preliminary Southern blot analyses mammalian fish                       II. Fink, G. 1987. Psendogenes yeast? Cell. 49:5-6. <br /> genomic DNAs identified potential SEC14-related nu-                       12. Glick, B. S., J. E. Rothman. 1987. Possible role fatty acylcoenzyme <br />                                                                                      A intracellular protein transport. Nature (Lond.). 326:309-312. <br /> cleic acid sequences organisms S. pombe                 13. Goldstein, A., andJ. O. Lampen. 1979. B-D-fructofuranoside fructohydro- <br /> (not shown), yeast demonstrated                           lase yeast. Methods Enzymol. 42:504-511. <br /> the existence antigenic homolog (Fig. 7). Although                    14. Griffiths, G., K. Simons. 1986. The trans Golgi network: sorting at <br />                                                                                      exit site Golgi complex. Science (Wash. DC). 234:438-443. <br /> have, point, failed unequivocally demonstrate                     15. Ito, H., Y. Fukuda, K. Murata, A. Kimura. 1983. Transformation of <br /> SEC14p cross-reactive polypeptide mammalian Vero cell                              intact yeast cells treated alkali cations. J. Bacteriol. 153:163-168. <br /> extracts (data shown), data intriguing light                  16. Kleid, D. G., D. Yansura, B. Small, D. Dowbenko, D. M. Moore, M. 3. <br />                                                                                      Grubman, P. D. McKercher, D. O. Morgan, P. H. Robertson, H. L. <br /> of work Dunphy et al. (6) showed functional                             Bachrach. 1981. Clones viral protein vaccine foot mouth disease: <br /> interchangeability S. cerevisiae mammals                            responses cattle swine. Science (Wash. DC). 214:1125-1129. <br />                                                                                17. Kuo, C., J. Campbell, 1983. Cloning Saccharomyces cerevisiae <br /> respect cytosolic factors required drive in-                       DNA replication genes: Isolation CDC8 gene genes that <br /> tercisternal Golgi transport vitro. We generated con-                        compensate cdcS-I mutation. Mol. Cell Biol. 3:1730-1737. <br /> ditions possible remove functional SEC14p ac-                   18. Kyte, J., R. F. Doolittle. 1982. A method displaying hydropho- <br />                                                                                       bic character protein. J. Mol. Biol. 157:105-132. <br /> tivity yeast lysates. With vitro transport assays                  19. Lipman, D., W. Pearson. 1985. Rapid sensitive protein similarity <br /> firmly place, SECl4p-depleted cytosolic fractions                             searches. Science (Wash. DC). 227:1435-1441. <br /> can tested Golgi transport-promoting activity,                      20. Maniatis, T., E. F. Fritsch, J. Sambrook. 1982. Molecular Cloning: <br />                                                                                       A Laboratory Manual. Cold Spring Harbor Laboratory, Cold Spring Har- <br /> with hope obtaining direct evidence SEC14p in-                             bor, New York. 545 pp. <br /> volvement cytosolic, Golgi transport-promoting factor.                    21. Melancon, P., B. S. Glick, V. Malbotra, P. J. Weidman, T. Serafini, M. L. <br />                                                                                       Gleason, L. Orei, 3. E. Rothman. 1987. Involvement GTP-binding <br /> We believe continued genetic biochemical analysis                            &quot;G&quot; proteins transport Golgi stack. Cell. 51:1053-1062. <br /> of SEC14p function reveal important insights                    22. Mitchison, J. M. 1970. Physiological cytological methods Schizo- <br /> intracellular protein traffic proceeds later stages                       saceharomyces pombe. Methods Cell Physiol. 4:131-156. <br />                                                                        <br /> </body></html>