<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Molecular Microbiology (2009) 73(5), 826&#226;&#8364;&#8220;843 &#228;&#352;?                                               doi:10.1111/j.1365-2958.2009.06764.x <br />                                                                                               First published online 30 June 2009 <br />  <br />  <br />  <br /> The ethanolamine branch Kennedy pathway is <br /> essential bloodstream form Trypanosoma brucei                                                                mmi_6764 826..843 <br />  <br />  <br />  <br />  <br /> Federica Gibellini,1 William N. Hunter1                        Introducton <br /> Terry K. Smith2* <br /> 1 <br />  Division Biological Chemistry Drug Discovery,              The unicellular eukaryote Trypanosoma brucei the <br /> School Life Sciences, University Dundee, Dundee              cause sleeping sickness humans Nagana in <br /> DD1 5EH, Scotland, UK.                                             livestock, Sub-Saharan Africa. Current treatment <br /> 2 <br />  Centre Biomolecular Sciences, The North Haugh,                limitations efforts directed the <br /> The University, St Andrews KY16 9ST, Scotland, UK.                 assessment drug targets development new <br />                                                                    drugs. Studies T. brucei metabolism unravel <br />                                                                    novel unique aspects parasite&#226;&#8364;&#8482;s biology that <br /> Summary                                                            intrinsic directly appli- <br />                                                                    cable understanding disease assist the <br /> Phosphatidylethanolamine (GPEtn), major phospho- <br />                                                                    development new therapies (van Hellemond and <br /> lipid component trypanosome membranes, syn- <br />                                                                    Tielens, 2006). <br /> thesized novo ethanolamine the <br />                                                                       One area potential drug target novo <br /> Kennedy pathway. Here composition GPEtn <br />                                                                    biosynthesis phospholipids (Ancelin Vial, 1986; <br /> molecular species bloodstream form Trypa- <br />                                                                    Hern&#195;&#161;ndez-Alcoceba et al., 1997; Urbina, 2006; Choubey <br /> nosoma brucei determined, new insights <br />                                                                    et al., 2007; Jez, 2007). Phospholipids contribute an <br /> into phospholipid metabolism, vitro vivo <br />                                                                    important structural role membrane prop- <br /> characterization key enzyme Kennedy <br />                                                                    erties determine membrane fluidity cell surface <br /> pathway, cytosolic ethanolamine-phosphate cyti- <br />                                                                    charge. They implicated wide variety cellular <br /> dylyltransferase (TbECT) . Gene <span id='am-240' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-241' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-242' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-244' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-249' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-251' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-254' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-259' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-261' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-277' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-284' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-299' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-300' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-317' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span>knockout</span> indicates <br />                                                                    processes, including cell division cell signalling <br /> that TbECT essential growth <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span>survival</span>, thus <br />                                                                    (Dowhan, 1997). Ethanolamine derived phospholipids <br /> highlighting importance Kennedy pathway <br />                                                                    (GPEtn, phosphatidylethanolamine), include dia- <br /> for pathogenic stage African trypanosome. <br />                                                                    cylGPEtn, alkyl-acylGPEtn alkenyl-acylGPEtn (plas- <br /> Phosphatiylserine decarboxylation, potential <br />                                                                    malogen), increase tendency membranes <span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span> <br /> salvage pathway, does appear active cul- <br />                                                                    form non-bilayer structures, influencing membrane <br /> tured bloodstream form T. brucei, upregu- <br />                                                                    fusion trafficking (Dowhan, 1997; Bakovic et al., <br /> lated Kennedy pathway disrupted. In <br />                                                                    2007). These lipids affect folding, stabilization and <br /> vivo metabolic labelling phospholipid composi- <br />                                                                    activity integral membrane-bound proteins in <br /> tion analysis ESI-MS/MS knockout cells con- <br />                                                                    certain cases GPEtn GPEtn-derived ethanolamine- <br /> firmed significant decrease GPEtn species, well <br />                                                                    phosphoglycerol attached protein residues a <br /> as changes relative abundance phos- <br />                                                                    post-translational modification (Signorell et al., 2008a). <br /> pholipid species. Reduction GPEtn levels a <br />                                                                    GPEtn donor ethanolamine-phosphate <br /> profound influence morphology mutants <br />                                                                    capping glycosylphosphatidylinositol (GPI) anchor <br /> and compromised mitochondrial structure and <br />                                                                    required attachment proteins outer <br /> function, glycosylphosphatidylinositol <br />                                                                    leaflet cell membrane (Menon Stevens, 1992; <br /> anchor biosynthesis. TbECT genetically <br />                                                                    Menon et al., 1993; Imhof et al., 2000). This particularly <br /> validated potential drug target African <br />                                                                    important bloodstream form T. brucei, which <br /> trypanosome. <br />                                                                    relies dense coat GPI-anchored variant surface <br />                                                                    glycoprotein (VSG) circumvent attack host <br />                                                                    immune (Nagamune et al., 2000). Disruption of <br /> Accepted 1 June, 2009. *For correspondence. E-mail tks1@st-        GPEtn biosynthetic pathways T. brucei likely to <br /> andrews.ac.uk; Tel. (+44) 1334 463412; Fax (+44) 1334 462595.      severely impair parasite homeostasis thus, the <br /> Re-use article permitted accordance Terms <br /> and Conditions set http://www3.interscience.wiley.com/      constituent enzymes represent novel targets for <br /> authorresources/onlineopen.html                                    chemotherapy. <br /> &#194;&#169; 2009 The Authors <br /> Journal compilation &#194;&#169; 2009 Blackwell Publishing Ltd <br />                                                                                               Ethanolamine metabolism Trypanosoma brucei 827 <br />  <br />    The major pathways biosynthesis GPEtn             Results discussion <br /> are CDP-ethanolamine (CDP-Etn) pathway, also <br /> called Kennedy pathway, phosphatidylserine              Contributions Kennedy pathway GPSer <br /> (GPSer) decarboxylation pathway. The CDP-Etn pathway                decarboxylation pathway GPEtn biosynthesis in <br /> consists enzymatic steps. Initially, ethanolamine          bloodstream form T. brucei <br /> kinase (EK, EC 2.7.1.82) catalyses ATP-depen-                   The relative contribution GPSer decarboxylation <br /> dent phosphorylation ethanolamine (Etn), forming                 pathway GPEtn biosynthesis assessed stable <br /> ethanolamine-phosphate (Etn-P), by-product                  isotope labelling d3-serine normal culturing media <br /> ADP. In stage two, CTP:ethanolamine-phosphate cyti-             overnight, order allow adequate time phospholipid <br /> dylyltransferase (ECT, EC 2.7.7.14), subject            synthesis dynamic equilibrium phospho- <br /> study, utilizes Etn-P CTP form high-energy               lipids pools reached. Subsequent analysis the <br /> donor CDP-Etn release pyrophosphate.                    phospholipid molecular species electrospray ionization <br /> This reaction considered rate-limiting                 tandem mass spectrometry (ESI-MS/MS) allowed the <br /> step Kennedy pathway (Sundler Akesson,                   overall contribution GPSer decarboxylation <br /> 1975). Diacylglycerol: CDP-ethanolamine ethanolamine-               pathway GPEtn biosynthesis assessed. Upon <br /> phosphotransferase (EPT, EC 2.7.8.1) catalyses final            labelling d3-serine readily incorporated newly <br /> reaction pathway, utilizing CDP-Etn diacylglyc-          synthesized d3-GPSer, virtually super imposable <br /> erol alkyl-acylglycerol form diacylGPEtn plasmalo-         pre-existing unlabelled GPSer visualized by <br /> gen, respectively, CMP by-product.                          neutral loss scans m/z 89 87 respectively (compare <br />    An alternative route synthesis GPEtn           Fig. 1A B). <br /> decarboxylation GPSer phosphatidylserine                       Similarly, newly synthesized d3-GPEtn formed by <br /> decarboxylase (PSD). This pathway actually sole              GPSer decarboxylation newly synthesized d3-GPSer <br /> route GPEtn biosynthesis Escherichia coli            detected parent ion scan analysis lipids that <br /> major Saccharomyces cerevisiae,                  produce collison induced 199 m/z fragment nega- <br /> yeast Kennedy pathway active (Dowhan,                   tive ion mode, opposed bulk GPEtn visualized <br /> 1997). In mammalian cells relative contributions             collision induced 196 m/z fragment ion (compare <br /> these pathways GPEtn formation cell-type depen-               Fig. 1C D). Table S1 shows annotation the <br /> dent, GPSer decarboxylation prevailing BHK21                GPEtn molecular species identified bloodstream form <br /> and CHO cells Kennedy pathway prevailing                 T. brucei. <br /> most mammalian tissues (hamster heart, rat heart,                      Although previous work (Rifkin et al., 1995) to <br /> kidney liver), cultured glioma cells (Vance,                indicate bloodstream form T. brucei able syn- <br /> 2008).                                                              thesize GPEtn GPSer decarboxylation, our <br />    Metabolic labelling pulse-chase experiments pro-             experiment clearly shows trace amounts d3-GPEtn <br /> vided evidence T. brucei GPEtn                (Fig. 1D), differ significantly novo syn- <br /> made ethanolamine serine,            thesized GPEtn Kennedy pathway (Fig. 1C). This <br /> Kennedy pathway used GPEtn biosyn-               suggests GPSer decarboxylation contributes little to <br /> thesis (Rifkin et al., 1995). This recently confirmed        biosynthesis GPEtn conditions it <br /> <span id='am-396' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-397' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-398' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-399' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-400' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-401' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-402' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-403' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-404' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-405' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-406' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-407' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-408' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-409' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-410' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-411' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-412' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-413' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-414' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-415' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-416' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-417' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-418' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-419' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-420' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-421' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-422' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-423' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-424' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-425' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-426' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-427' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-428' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-429' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-430' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-431' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-432' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-433' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-434' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-435' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-436' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-437' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-438' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-439' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-440' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-441' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-442' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-443' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-444' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-445' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-446' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-447' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-448' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-449' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-450' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-451' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-452' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-453' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-454' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-455' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-456' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-457' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-458' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-459' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-460' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-461' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-462' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-463' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-464' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-465' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-466' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-467' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-468' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-469' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-470' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-471' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-472' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-473' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span>RNA interference</span> insect form parasite (Signo-         confirms importance Kennedy pathway the <br /> rell et al., 2008a,b), now, detailed studies      biosynthesis GPEtn bloodstream T. brucei. <br /> performed bloodstream T. brucei. We show, first <br /> time, putative T. brucei CTP:ethanolamine- <br />                                                                     Identification, cloning sequencing TbECT <br /> phosphate cytidylyltransferase (TbECT ) gene (Tb11. <br /> 01.5730) encodes functional enzyme, charac-                The genome sequence T. brucei (Berriman et al., 2005) <br /> terize enzyme activity substrate requirements.              invaluable resource understanding the <br /> Formation bloodstream form T. brucei TbECT condi-              molecular cellular biology important parasite. A <br /> tional double knockout (cKO) allowed demonstrate              putative TbECT (Tb11.01.5730) identified the <br /> that TbECT essential non-permissive condi-             T. brucei genome database (http://www.genedb.org); the <br /> tions synthesis GPEtn GPI-anchors severely            putative open reading frame (ORF) PCR-amplified <br /> compromised. We GPSer decarboxylation                genomic DNA (Lister 427), cloned sequence <br /> makes minor contribution bulk GPEtn biosynthe-            submitted GenBank Nucleotide Sequence Database <br /> sis compensate loss Kennedy            Accession number FM992871. The complete ORF <br /> pathway TbECT cKO. These findings suggest              encodes protein 384 residues calculated <br /> may therapeutic opportunities targeting Kennedy           molecular mass 43.4 kDa. Although TbECT shows con- <br /> pathway.                                                            siderable similarity ECTs mammalian organisms <br />  <br /> &#194;&#169; 2009 The Authors <br /> Journal compilation &#194;&#169; 2009 Blackwell Publishing Ltd, Molecular Microbiology, 73, 826&#226;&#8364;&#8220;843 <br />  828 F. Gibellini, W. N. Hunter T. K. Smith &#228;&#352;? <br />  <br />  <br />  <br />  <br /> Fig. 1. ESI-MS/MS spectra following molecular species: GPSer (A); (d3)-GPSer (B); GPEtn (C) (d3)-GPEtn (D) bloodstream <br /> T. brucei labelled (d3)-serine overnight. Data normalized largest peak display vertical axes linked order directly <br /> compare intensities non-deuterated versus deuterated phospholipid species. <br />  <br />  <br />  <br />  <br /> and shares 40% sequence identity human ECT,                      expression vector pET20bTEV. This vector encodes <br /> unsurprisingly closely related putative ECTs                   hexa-histidine tag C-terminal protein, <br /> other kinetoplastids Leishmania major                           removed proteolytic cleavage with <br /> T. cruzi, 59% 62% sequence identity respectively                   Tobacco Etch Virus (TEV) protease (Fig. S1A). The <br /> (Fig. 2A B). Like previously characterized ECTs,                        protein expressed purified described in <br /> TbECT contains large tandem repeated sequences,                         Experimental procedures. Typical yields 20&#226;&#8364;&#8220;25 mg <br /> probably result gene duplication (Nakashima et al.,                    litre culture. TbECT stored -80&#194;&#176;C in <br /> 1997; Bladergroen et al., 1999). The N-terminal repeat                   buffer containing 5% glycerol loss activity over <br /> strongly conserved ECTs members                        period months. <br /> the cytidylyltransferase family: glycerol-3-phosphate cyti-                    Matrix-assisted laser desorption/ionization (MALDI) <br /> dylyltransferases (GCT) N-terminal domain                        mass spectrum cleaved recombinant TbECT <br /> choline-phosphate cytidylyltransferases (CCT).                              returned molecular mass 44 623 Da (theoretical <br />    All signature motifs characterize cytidylyl-                value predicted recombinant TbECT: 44 622 Da). <br /> transferase family present TbECT sequence:                          Analytical ultracentrifugation performed on <br /> two conserved GX(Y/F)DXXHXGH sequences, contain-                            untagged TbECT. The sample displayed single species <br /> ing HXGH motive (47HFGH 227HAGH, Fig. 2A,                           mass ~52 kDa, agrees theoretical mass <br /> boxed) adjacent Asp44 Asp224 (Fig. 2A,                         monomer (Fig. S1B). <br /> asterisks); RTX(G/C/S)ISTT motif (152RTECISTT)                             Interestingly, rat human ECTs, which <br /> (Fig. 2A, underlined); regions 96KWVDAVI                        TbECT shares high sequence identity, form dimers or <br /> and 278RYVDDVV (Fig. 2A, double underlined). The struc-                     higher aggregates solution (Sundler, 1975; Tie and <br /> ture Bacillus subtilis GCT (Weber et al., 1999; Pattridge                Bakovic, 2007). The reported monomeric ECT <br /> et al., 2003) revealed HXGH motif                          S. cerevisiae (Ohtsuka et al., 2006), shares <br /> RTX(G/C/S)ISTT motif involved formation                   34% sequence identity TbECT (Fig. 2), despite both <br /> CTP binding site, motif involved                homologues having sequence motif predicted <br /> formation dimer interface.                                             play role dimer interface. <br />                                                                                TbECT catalysed formation CDP-ethanolamine <br />                                                                             ethanolamine-phosphate, CTP magnesium- <br /> Recombinant expression characterization TbECT <br />                                                                             dependent manner (Fig. S2). At pH optimum 7.5 <br /> To establish TbECT active ethanolamine-                            (Fig. 3A) TbECT displays Km 12.09 &#226;&#171;&#190; 0.87 mM <br /> phosphate cytidylyltransferase obtain material                   (Fig. 3B) ethanolamine-phosphate Km of <br /> the characterization properties, TbECT cloned                    12.67 &#226;&#171;&#190; 0.78 mM (Fig. 3C) CTP. These values <br />  <br />                                                                                                                         &#194;&#169; 2009 The Authors <br />                                                    Journal compilation &#194;&#169; 2009 Blackwell Publishing Ltd, Molecular Microbiology, 73, 826&#226;&#8364;&#8220;843 <br />                                                                                                         Ethanolamine metabolism Trypanosoma brucei 829 <br />  <br />  <br />  <br />  <br /> Fig. 2. A. ClustalW alignment predicted amino acid sequences Trypanosoma brucei ECT (Q382C3-1) ethanolamine <br /> cytidylyltransferases glycerol-3-phosphate cytidylyltransferases eukaryotes: Leishmania major (Q4Q5J3), Trypanosoma cruzi <br /> (Q922E4), Homo sapiens (Q99447), Saccharomyces cerevisiae (P33412) ECTs Bacillus subtilis GCT. Residues shaded according to <br /> percentage identity using software Jalview 2.08.1. Boxed underlined regions characterisitic signature motif cytidylyltransferases. <br /> B. Phylogenetic analysis ECTs GCTs Fig. 2A plus ECTs Mus musculus (Q922E4), Rattus norvegicus (O88637), <br /> Chlamydomonas reinhardtii (Q84JV7) Arabidopsis thaliana (Q9ZVI9). <br />  <br />  <br />  <br />  <br /> compare favourably Km values previously charac-                     The enzyme, however, able utilize dCTP, although <br /> terized ECTs rat (KmCTP = 53 mM; KmETN-P = 65 mM                       reduced rate, implying enzyme able to <br /> (Sundler, 1975) order magnitude                        discriminate deoxyribonucleotide ribo- <br /> lower corresponding Kms mouse (Tie                          nucleotide moiety. This consistent trace amounts of <br /> Bakovic, 2007). Particularly striking unusually                      dCDP-ethanolamine bloodstream form <br /> low Km cofactor CTP; reflect relatively                T. brucei (Rifkin et al., 1995). <br /> low intracellular concentration nucleotide triphos-                    Interestingly, TbECT unable accept 2-aminoethyl <br /> phate T. brucei (Hofer et al., 1998) compared                       phosphonate (AEP) substrate. AEP essential <br /> the pool present mammalian cells (Stryer, 1995;                          metabolite formation phosphonolipids, which <br /> Lazarowski Boucher, 2001) The Vmax TbECT                          carry covalent bond phosphorous the <br /> 1.55 &#226;&#171;&#190; 0.03 mmol min-1 mg-1, corresponding kcat                     carbon nitrogenous <span id='am-474' about='xsp:base' typeof='owl:Thing'>base</span> (Baer Stanacev, <br /> 1.15 s-1.                                                                   1964). However, previous studies (Ferguson et al., 1982) <br />   TbECT activity selective CTP,                   failed detect phosphonolipids T. brucei, unlike <br /> no activity detected ATP, GTP UTP (Table S2).                   closely related trypanosomatid, T.cruzi, is <br />  <br /> &#194;&#169; 2009 The Authors <br /> Journal compilation &#194;&#169; 2009 Blackwell Publishing Ltd, Molecular Microbiology, 73, 826&#226;&#8364;&#8220;843 <br />  830 F. Gibellini, W. N. Hunter T. K. Smith &#228;&#352;? <br />  <br />                                                                             reflected absence enzymes biosyn- <br />                                                                             thesis AEP T. brucei (Sarkar et al., 2003) and, as <br />                                                                             shown here, substrate specificity TbECT. <br />  <br />  <br />                                                                             Construction TbECT conditional null mutant <br />  <br />                                                                             To assess importance TbECT the <br />                                                                             entire Kennedy pathway survival bloodstream <br />                                                                             form T. brucei, conditional null knockout generated <br />                                                                             characterized. <br />                                                                                Preliminary analysis showed TbECT gene was <br />                                                                             present single copy haploid genome and <br />                                                                             expressed bloodstream form insect life cycle <br />                                                                             stages (Fig. S3). A conventional mutagenesis approach <br />                                                                             (Martin Smith, 2006a) followed: allele was <br />                                                                             replaced puromycin resistance gene (PAC) gener- <br />                                                                             ating DECT::PAC cell line; haemagglutinin (HA)- <br />                                                                             tagged tetracycline-inducible ectopic copy gene was <br />                                                                             introduced ribosomal DNA, yielding cell line <br />                                                                             ECT-HA Ti DECT::PAC; maintaining expression of <br />                                                                             ectopic copy addition tetracycline, remaining <br />                                                                             allele replaced hygromycin resistance gene <br />                                                                             resulting conditional double knockout cell line ECT- <br />                                                                             HA Ti DECT::PAC/DECT::HYG. The genotype condi- <br />                                                                             tional null mutant verified Southern blot analysis, <br />                                                                             ectopic copy present (Fig. 4A, lane 3). <br />  <br />  <br />                                                                             TbECT essential T. brucei bloodstream form <br />                                                                             To establish ECT essential gene, cell <br />                                                                             line ECT-HA Ti DECT::PAC/DECT::HYG cultured the <br />                                                                             presence absence tetracycline growth rates <br />                                                                             compared wild-type. Triplicate cultures cell <br />                                                                             type inoculated 1 &#194;&#165; 105 cells ml-1 subcultured <br />                                                                             days necessary. In presence tetra- <br />                                                                             cycline, is, permissive conditions allow expres- <br />                                                                             sion ECT-HA Ti, cells showed normal growth kinetics, <br />                                                                             comparable wild-type cells (Fig. 4B). However, when <br />                                                                             cultured tetracycline-free media (non-permissive condi- <br />                                                                             tions), cells immediately displayed reduced growth rate <br />                                                                             48 h attained approximately half <br />                                                                             cell density wild-type (Fig. 4B). After 72 h an <br />                                                                             increasing cell debris observed by <br />                                                                             day 4 cell numbers fell limit detection light <br />                                                                             microscopy. RT-PCR confirmed TbECT mRNA disappear- <br /> Fig. 3. Analysis CDP-ethanolamine formation recombinant <br /> TbECT.                                                                      ance 24 h non-permissive conditions (Fig. 4C). <br /> A. Ethanolamine-phosphate cytidylyltransferase activity                 However, 7&#226;&#8364;&#8220;8 days tetracycline removal live cells <br /> determined end-point assay function pH described in <br />                                                                             visible normal growth rates were <br /> Experimental procedures. <br /> B C. Determination TbECT Michaelis-Menten constants              resumed. This resumption growth revertant <br /> ethanolamine-phosphate CTP (inserts Lineweaver&#226;&#8364;&#8220;Burk                cells overcoming tetracycline control, demon- <br /> plots). (B) CTP concentration held constant (0.1 mM) while <br />                                                                             strated reappearance TbECT mRNA these <br /> ethanolamine-phosphate concentration varied. (C) <br /> Ethanolamine-phosphate concentration held constant (0.1 mM)             cells (Fig. 4C). This phenomenon, described times <br /> while CTP concentration varied.                                         T. brucei conditional <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span>null mutants</span> essential <br />                                                                             genes (Martin Smith, 2006a), ascribed to <br />  <br />                                                                                                                         &#194;&#169; 2009 The Authors <br />                                                    Journal compilation &#194;&#169; 2009 Blackwell Publishing Ltd, Molecular Microbiology, 73, 826&#226;&#8364;&#8220;843 <br />                                                                                              Ethanolamine metabolism Trypanosoma brucei 831 <br />  <br />                                                                                        Fig. 4. TbECT essential survival of <br />                                                                                        bloodstream form Trypanosoma brucei in <br />                                                                                        culture. <br />                                                                                        A. Confirmation genotype T. brucei ECT <br />                                                                                        conditional double knockout cell line. <br />                                                                                        Southern-blot analysis NcoI-digested <br />                                                                                        genomic DNA (5 mg); ECT ORF probe <br />                                                                                        shows allelic TbECT 4.4 kb ectopic <br />                                                                                        copy TbECT-HATi ~5 kb 1; parental cells <br />                                                                                        (SM) 2; DECT::PAC/ECT-HA Ti 3; <br />                                                                                        DECT::PAC/DECT::HYG/ECT-HA Ti 4; <br />                                                                                        Undigested DNA. <br />                                                                                        B. Growth curves T. brucei parental cells (1 <br />                                                                                        &#226;&#8364;&#8220; filled circles) TbECT conditional <br />                                                                                        knockout cells grown presence (2 &#226;&#8364;&#8220; <br />                                                                                        circles) absence (3 &#226;&#8364;&#8220; filled triangles) <br />                                                                                        tetracycline. <br />                                                                                        C. RT-PCR amplification TbECT RNA <br />                                                                                        transcripts total RNA extracted <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> <br />                                                                                        TbECT conditional null mutants grown the <br />                                                                                        absence tetracycline 0, 12, 24, 36 and <br />                                                                                        48 h. The upper panel shows RT-PCR <br />                                                                                        products using primers specific TbECT; <br />                                                                                        middle panel shows PCR-negative <br />                                                                                        control, reverse transcriptase; the <br />                                                                                        lower panel shows loading control using <br />                                                                                        tubulin tyrosine-ligase (TbTTL) primers. <br />  <br />  <br />  <br />  <br /> deletion tetracycline repressor protein gene (Roper          overall cellular dimensions (almost &#226;&#8364;&#732;stumpy-like&#226;&#8364;&#8482;) when <br /> et al., 2002).                                                      compared control cells (Fig. S4A, compare panels 1 <br />                                                                     2 3 4). Determination average cell <br />                                                                     volume (CASY cell counter, SEDNA Scientific) confirmed <br /> TbECT deprivation affects cellular dimensions and <br />                                                                     average reduction 30% cell volume after <br /> subcellular architecture <br />                                                                     48 h growth tetracycline free media (Fig. S4B). <br /> Scanning electron micrographs TbECT cKO cells                      Transmission electron microscopy used investi- <br /> deprived tetracycline 36 h suggested reduction          gate subcellular architecture TbECT cKO cells at <br />  <br /> &#194;&#169; 2009 The Authors <br /> Journal compilation &#194;&#169; 2009 Blackwell Publishing Ltd, Molecular Microbiology, 73, 826&#226;&#8364;&#8220;843 <br />  832 F. Gibellini, W. N. Hunter T. K. Smith &#228;&#352;? <br />  <br /> various time points tetracycline removal. As shown                 lane 3); downregulation of <br /> Fig. 5A (panels b&#226;&#8364;&#8220;e) main morphological defect                          EK1 channelling excess Etn-P other <br /> appears abnormally enlarged mitochondria,                          catabolic metabolic pathways. This means the <br /> which usually narrow tubular structure                 phenotypical effects ECT knockout the <br /> SM parental cells (Fig. 5A, panel a). This abnormal                         disappearance CDP-Etn downstream metabo- <br /> enlargement mitochondria quantified 60                   lites, i.e. GPEtn, accumulation of <br /> suitable transmission electron micrographs relevant                      toxic levels Etn-P. <br /> time points. At 36 h tetracycline ~70%                          Since GPEtn donor terminal phospho- <br /> examined mitochondria significantly enlarged,                    ethanolamine group GPI anchors (Menon et al., 1993; <br /> increased ~90% 42 h (Fig. 5A, panel f).                               Imhof et al., 2000), biosynthesis previ- <br />    To investigate enlargement reflects loss                  ously validated drug target African trypano- <br /> of function organelle, trypanosomes stained                     some, tested possibility reduction cellular <br /> with membrane potential-dependent fluorescent dye                       GPEtn produce knock-on effect GPI <br /> Mitotracker Red (Fig. 5B) observed fluorescence                      pathway. <br /> microscopy. TbECT cKO trypanosomes grown non-                            An [3H]Man labelling parental cells TbECT <br /> permissive conditions 36 h accumulation                   cKO cells grown permissive conditions highlighted the <br /> of dye mitochondria, volume                   formation expected mature GPI anchor glycolipids <br /> the organelle appears increased (Fig. 5B, lane c),                    A C (Fig. 6A, lanes 7 8) previously reported <br /> suggesting enlarged, mitochondria                 (Martin Smith, 2006b). However, conditional <br /> able maintain membrane electrostatic potential.                      double knockout cells grown non-permissive conditions <br /> However, 42 h tetracycline removal, dye                      36 h displayed marked decrease of <br /> signal completely disappears, suggesting loss                     labelled species (Fig. 6A, lane 9), meaning the <br /> function (Fig. 5B, lane d). Changes mitochondrial mor-                   decrease GPEtn synthesis, deletion TbECT, <br /> phology function observed mam-                      detrimental effect biosynthesis mature GPI <br /> malian PSD mutants (Steenbergen et al., 2005)                        anchors. <br /> T. brucei procyclic ACP mutants result decrease                      In addition, [3H]Myr labelling shows a <br /> of GPEtn organelle (Guler et al., 2008).                            slight decrease phospholipids synthe- <br />                                                                             sized conditional double null cells grown non- <br />                                                                             permissive conditions 36 h (Fig. 6A, compare lanes <br /> <span id='am-162' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-163' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span>Phenotype</span> analysis TbECT conditional null mutants <br />                                                                             10&#226;&#8364;&#8220;12). <br /> by vivo labelling <br />                                                                                To confirm cells viable time of <br /> The biochemical phenotype conditional double                         labelling, ability cells perform normal rates of <br /> knockout cells investigated vivo labelling                   protein synthesis confirmed. The wild-type cells and <br /> either [3H]Etn, [3H]Ser, [3H]Man [3H]Myr (Fig. 6)                     conditional double null cells grown presence or <br /> ascertain effect TbECT disappearance GPEtn                        absence tetracycline 36 h showed similar amounts <br /> biosynthesis potential knock-on effects GPI anchor                   [35S]methionine incorporation newly synthesized <br /> biosynthesis general phospholipid metabolism. The                       proteins (Fig. 6B), meaning biochemical changes <br /> results clear decrease GPEtn formed                       detected vivo labelling experiments con- <br /> the Kennedy pathway cKO non-permissive                         sequence TbECT deprivation general <br /> conditions (Fig. 6A, compare lanes 1&#226;&#8364;&#8220;3). This GPEtn                         loss viability. We note 48 h, protein synthesis has <br /> decrease does prompt upregulation GPSer                       partially compromised (data shown), 36 h <br /> decarboxylation pathway, GPEtn observed                         chosen biochemical phenotyping. <br /> from newly synthesized GPSer (Fig. 6A, lanes 4&#226;&#8364;&#8220;6). <br /> This represents fundamental difference behav- <br />                                                                             Lipid profile TbECT conditional null mutants by <br /> iour mammalian cells culture: Kennedy pathway <br />                                                                             ESI-MS/MS <br /> is favoured pathway biosynthesis GPEtn <br /> when ethanolamine readily available culture                       To ascertain decrease GPEtn decrease <br /> media, GPSer decarboxylation pathway com-                       particular GPEtn species global decrease of <br /> pensate conditions ethanolamine deprivation                        GPEtn species, affected composition <br /> (Voelker, 1984).                                                            phospholipid classes, whole-cell extracts the <br />    As expected, blocking Kennedy pathway ECT                     TbECT cKO cell line grown presence absence of <br /> level results disappearance CDP-Etn product                   tetracycline 36 h semi-quantitatively analysed by <br /> (Fig. 6C, lane 3). Surprisingly though, does trans-                ESI-MS/MS compared wild-type (Figs 7 8 <br /> late accumulation substrate Etn-P (Fig. 6C,                  Figs S5&#226;&#8364;&#8220;8). The negative positive ion survey <br />  <br />                                                                                                                         &#194;&#169; 2009 The Authors <br />                                                    Journal compilation &#194;&#169; 2009 Blackwell Publishing Ltd, Molecular Microbiology, 73, 826&#226;&#8364;&#8220;843 <br />                                                                                                        Ethanolamine metabolism Trypanosoma brucei 833 <br />  <br />               A <br />  <br />  <br />  <br />  <br />               B <br />  <br />  <br />  <br />  <br /> Fig. 5. Loss ECT activity affects mitochondrial morphology function. <br /> A. Transmission electron micrographs TbECT cKO cells taken (a) 36 (b) 42 h (c&#226;&#8364;&#8220;e) tetracycline removal the <br /> media. (f) Quantification sections displaying abnormal enlargement mitochondria wild-type TbECT cKO cells different <br /> time points tetracycline removal (SM parental cell, n = 43; TbECT cKO + Tet, n = 83; TbECT cKO + Tet, n = 65; TbECT cKO -Tet42 h, <br /> n = 68). Scale bar = 2 mm. <br /> B. Mitotracker red CMXRos used stain live cells prior fixation (3); cells counterstained DAPI (2); corresponding <br /> phase contrast images shown (1), merged fluorescence-phase images shown (4). Mitotracker fluorescence images were <br /> collected acquisition time 542 ms allow data examined differences fluorescence intensity. (a), SM parental <br /> cells; (b) TbECT cKO cells grown presence tetracycline; TbECT cKO cells grown absence tetracycline 36 h (c) or <br /> 42 h (d). <br />  <br />  <br /> &#194;&#169; 2009 The Authors <br /> Journal compilation &#194;&#169; 2009 Blackwell Publishing Ltd, Molecular Microbiology, 73, 826&#226;&#8364;&#8220;843 <br />  834 F. Gibellini, W. N. Hunter T. K. Smith &#228;&#352;? <br />  <br />  <br />  <br />  <br /> Fig. 6. Phenotype analysis TbECT conditional knockout vivo labelling. <br /> A. SM parental cells (lanes 1, 4, 7 10); TbECT cKO cells grown presence (lanes 2, 5, 8 11) absence tetracycline 36 h <br /> (lanes 3, 6, 9 12) labelled [3H]ethanolamine ([3H]Etn, lanes 1&#226;&#8364;&#8220;3), [3H]serine ([3H]Ser, lanes 4&#226;&#8364;&#8220;6), [3H]mannose ([3H]Man, <br /> lanes 7&#226;&#8364;&#8220;9) [3H]myristate ([3H]Myr, 10&#226;&#8364;&#8220;12), lipids extracted, desalted run HPTLC described Experimental procedures. <br /> Incorporation radiolabel lipids detected fluorography. Lipid standards GPEtn, lysoGPEtn GPSer indicated left. <br /> Glycolipids A C indicated right. <br /> B. Cells labelled [35S]methionine proteins extracted separated 4&#226;&#8364;&#8220;10% SDS-PAGE gel. Proteins detected <br /> by Coomassie Blue staining (lanes 1&#226;&#8364;&#8220;3) fluorography (lanes 4&#226;&#8364;&#8220;6). Lanes 1 4, SM parental cells; lanes 2 5, TbECT cKO cells <br /> grown presence tetracycline; lanes 3 6, TbECT cKO cells grown absence tetracycline 36 h. <br /> C. The water-soluble metabolites Kennedy pathway extracted cells labelled [3H]Etn run HPTLC. The metabolites <br /> Etn, Etn-P CDP-Etn indicated right. <br />  <br />  <br /> scans TbECT cKO grown non-permissive                           GPEtn investigated. Figure 7 shows major <br /> conditions contain substantial differences control                 GPEtn peak encompassing molecular species mass <br /> ones, highlighted asterisks (Fig. S5A B).                     range m/z 724&#226;&#8364;&#8220;732 (e-36:3 &#226;&#8364;&#8220; a-36:0, Table S1) con- <br />    A detailed investigation individual phospho-                 siderably reduced TbECT cKO grown absence <br /> lipid classes conducted parent ion neutral loss                  tetracycline 36 h, compared parental <br /> scanning specific collision induced fragment ions char-                  cells TbECT cKO grown presence tetracy- <br /> acteristic phospholipids class. Initially effect            cline (Fig. 7, compare A B C). The principal <br />  <br />                                                                                                                         &#194;&#169; 2009 The Authors <br />                                                    Journal compilation &#194;&#169; 2009 Blackwell Publishing Ltd, Molecular Microbiology, 73, 826&#226;&#8364;&#8220;843 <br />                                                                                                Ethanolamine metabolism Trypanosoma brucei 835 <br />  <br />  <br />  <br />  <br /> Fig. 7. GPEtn phospholipid analysis ESI-MS/MS. Ethanolamine-containing phospholipids analysed ESI-MS/MS negative ion <br /> mode using parent-ion scanning collision induced fragment m/z 196. The arrows indicate peaks internal standard GPEtn <br /> (15:0/15:0), subjected ninefold magnification clarity. <br /> A. SM parental cells. <br /> B. TbECT cKO cells grown presence tetracycline. <br /> C. TbECT cKO cells grown absence tetracycline 36 h. <br /> D. Daughter ion spectrum m/z 726 [M-H]- ion. <br />  <br />  <br />  <br /> component peak identified plasmalogen                 18:2) (Fig. 8D). This phenomenon attributed the <br /> GPEtn (e-18:0/18:2) daughter ion scanning m/z 726              alkyl-acyl-glycerol utilized EPT last <br /> (Fig. 7D). The relative ratio intensity GPEtn          step Kennedy pathway lack CDP-Etn, and <br /> (e-18:0/18:2) peak m/z 726 internal              excess lipid/substrate channelled the <br /> standard GPEtn (15:0/15:0), used compare                  formation ether GPCho phospholipids the <br /> amount GPEtn different samples showed             CDP-choline branch Kennedy pathway. Whereas <br /> the plasmalogen GPEtn 36 h ECT               GPEtn (a-18:0/18:2) species normally proceed <br /> deprivation reduced            formation plasmalogen form GPEtn (e-18:0/ <br /> contained parental cells TbECT cKO grown              18:2), GPCho (a-18:0/18:2) remains alkyl-acyl <br /> in presence tetracycline.                                     form. <br />    Parallel GPEtn reduction correspond-              In addition, cKO cells grown absence of <br /> ing increase phosphatidylcholine (GPCho) peak              tetracycline 36 h increase phospha- <br /> m/z 768&#226;&#8364;&#8220;776 includes phospholipids carry               tidic acid (GPA) phosphatidylglycerol (GPGro) peaks <br /> same kind fatty acyl constituents (e-36:3 &#226;&#8364;&#8220; a-36:0)               m/z 749 773 respectively (Fig. S6), assessed by <br /> (Fig. 8, compare A B C), major species             parent ion scanning negative ion mode glycerol- <br /> identified daughter ion scanning GPCho (a-18:0/                cyclic phosphate collision induced ion m/z 153. These <br />  <br /> &#194;&#169; 2009 The Authors <br /> Journal compilation &#194;&#169; 2009 Blackwell Publishing Ltd, Molecular Microbiology, 73, 826&#226;&#8364;&#8220;843 <br />  836 F. Gibellini, W. N. Hunter T. K. Smith &#228;&#352;? <br />  <br />  <br />  <br />  <br /> Fig. 8. GPCho sphingomylein phospholipid analysis ESI-MS/MS. <br /> A&#226;&#8364;&#8220;C. Choline-containing phospholipids analysed ESI-MS/MS positive ion mode using parent-ion scanning collision induced <br /> fragment phosphoryl-choline m/z 184. The arrows indicate peaks internal standard GPCho (21:0/21:0). (A) SM parental cells; (B) <br /> TbECT cKO cells grown presence tetracycline; (C) TbECT cKO cells grown absence tetracycline 36 h. <br /> D. Daughter ion spectrum m/z 773 [M + H]+ ion. <br />  <br />  <br />  <br /> peaks identified GPA (18:0/22:5) GPGro                          stream form T. brucei GPSer synthesized base- <br /> (18:0/18:2) daughter ion ES/MS-MS (Fig. S6D E).                      exchange reaction GPEtn, pathway to <br /> Since increase GPGro (18:0/18:2, m/z 773) mir-                    present procyclic form parasite (Signorell <br /> rored cKO cells grown presence tetracy-                    et al., 2008b). <br /> cline, hypothesize increase these <br /> phospholipid species result trypanosome <br />                                                                             ECT depletion knock-on effect GPI anchor <br /> metabolism attempting compensate suboptimal <br />                                                                             biosynthetic pathway <br /> amount GPEtn and/or overall unused lipid accep- <br /> tor GPEtn EPT.                                                      The effect GPEtn deprivation GPI-anchor bio- <br />    Interestingly, total composition profile               synthetic pathway studied using a <br /> both GPSer phosphatidylinositol (GPIno)                         trypanosomal cell-free system, vitro assay which <br /> altered significantly cKO absence tetracy-                 throughput GPI pathway maximized <br /> cline (Figs S7 S8 respectively). This correlates                   (Smith et al., 1996). Washed membranes incubated <br /> the observation GPSer biosynthesis margin-                     GDP-[3H]Man UDP-GlcNAc presence of <br /> ally affected reduction GPEtn levels (Fig. 6, lanes                 dithiothreitol (DTT) (Fig. 9, lane 2), leading forma- <br /> 4&#226;&#8364;&#8220;6) exclude possibility blood-                    tion dolichol-phosphate-[3H]mannose (DPM) and <br />  <br />                                                                                                                         &#194;&#169; 2009 The Authors <br />                                                    Journal compilation &#194;&#169; 2009 Blackwell Publishing Ltd, Molecular Microbiology, 73, 826&#226;&#8364;&#8220;843 <br />                                                                                                Ethanolamine metabolism Trypanosoma brucei 837 <br />  <br />                                                                       ary antibody and, expected, cytosolic localization <br />                                                                       observed (Fig. 10A). This confirmed Western <br />                                                                       blot analysis subcellular fractions probed the <br />                                                                       anti-HA antibody (Fig. 10B). <br />                                                                         Cytosolic localization ECT T. brucei shared with <br />                                                                       mammalian cells (Vermeulen et al., 1993; Bleijerveld et al., <br />                                                                       2004); however, differs considerably localization <br />                                                                       enzyme plants algae, studies on <br />                                                                       Ricinus communis endosperm, Arabidopsis thaliana and <br />                                                                       Chlamydomonas reinhardtii reveal mitochondrial (or peri- <br />                                                                       mitochondrial) ER localization (Mizoi et al., 2006; <br />                                                                       Wang Moore, 1991; Yang et al., 2004). <br />  <br />  <br />                                                                       Conclusions <br />  <br />                                                                       We demonstrated Kennedy pathway is <br />                                                                       sole relevant route GPEtn biosynthesis the <br /> Fig. 9. Effects GPEtn deprivation GPI intermediates             bloodstream form T. brucei required its <br /> formation. T. brucei cell-free SM parental cells (lanes 1 <br /> and 2), TbECT cKO cells grown presence (lane 3)             survival. <br /> absence tetracycline 36 (lane 4) 42 h (lane 5)           We highlighted differences behaviour mam- <br /> labelled GDP-[3H]mannose presence absence           malian cells culture alternative pathway, <br /> phenylmethanesulphonyl fluoride, indicated. <br />                                                                       GPSer decarboxylation, compensate loss of <br />                                                                       Kennedy pathway deprivation precursor <br /> [3H]mannosylated       GPI      intermediates,      including         ethanolamine culture media (Voelker, 1984); in <br /> Man1GlcN-PI (M1), Man2GlcN-PI (M2), Man3GlcN-PI                       bloodstream T. brucei GPSer decarboxylation route <br /> (M3), Man3GlcN-(acyl)-PI(aM3), EtN-P-Man3GlcN-PI (A&#226;&#8364;&#178;)                 appears insignificant biosynthesis GPEtn is <br /> and sn-2 lyso species A&#226;&#8364;&#178; (Q).                                  unable compensate loss Kennedy <br />    Upon prolonged absence tetracycline, i.e. 42 h,             pathway disrupted TbECT gene <br /> formation GPI intermediates downstream                  knockout. <br /> Etn-P addition (glycolipid A&#226;&#8364;&#178; Q) greatly reduced,             This work suggested intriguing difference the <br /> it paralleled increase GPI intermediate               phospholipid metabolism main life cycle stages <br /> upstream reaction (aM3; Fig. 9, lane 4). This dem-             African trypanosome. In procyclic form GPSer is <br /> onstrates GPEtn deprivation specific effect           generated base-exchange reaction GPEtn and <br /> on GPI pathway, causing arrest level Etn-P          downregulation Kennedy pathway completely <br /> addition.                                                             blocked GPSer synthesis (Signorell et al., 2008a). <br />    As positive control GPI pathway stalled              However, Kennedy pathway was <br /> level Etn-P addition, cell-free pre-                disrupted bloodstream form T. brucei, GPSer <br /> incubated phenylmethanesulphonyl fluoride (PMSF),                biosynthesis occurred levels comparable the <br /> which inhibits acylation GPI intermediates (Fig. 9,            wild-type. Therefore, life cycle stage GPSer biosyn- <br /> lane 1), preventing addition Etn-P (Guther et al.,         thesis likely proceed coupling serine to <br /> 1994).                                                                CDP-DAG represents major difference the <br />                                                                       phospholipid metabolism life cycle stages. <br />                                                                          The lethality TbECT cKO non-permissive <br /> TbECT localized cytosol <br />                                                                       conditions shows corresponding cytidylyl- <br /> Since optimal TbECT activity occurs neutral pH                     transferase involved choline branch Kennedy <br /> (Fig. 4A) signal peptides identified             pathway unable compensate loss etha- <br /> amino acid sequence, cytosolic localization                  nolamine branch. The generation TbECT cKO cell <br /> enzyme hypothesized. The intracellular location               line allowed assessment phenotypic changes <br /> assessed taking advantage HA tag present                 GPEtn deprivation. We showed morphol- <br /> the C-terminus ectopic copy cell line ECT-              ogy mutants mitochondrion affected <br /> HA Ti DECT::PAC/DECT::HYG cultured presence                 early stage later mitochondrial function was <br /> tetracycline. Detection achieved immunofluores-                disrupted. GPI anchor biosynthesis affected and <br /> cence using primary antibody HA-tag               blocked level Etn-P addition, GPEtn <br /> fluorescein 5&#226;&#8364;&#178;-isothiocyanate (FITC)-conjugated second-               donor substrate. <br />  <br /> &#194;&#169; 2009 The Authors <br /> Journal compilation &#194;&#169; 2009 Blackwell Publishing Ltd, Molecular Microbiology, 73, 826&#226;&#8364;&#8220;843 <br />  838 F. Gibellini, W. N. Hunter T. K. Smith &#228;&#352;? <br />  <br />  <br />  <br />  <br /> Fig. 10. Subcellular localization ECT-HA bloodstream form T. brucei cells. <br /> A. Cells expressing ECT-HA costained HA epitope nuclear marker DAPI. (1) Phase contrast image. (2) DAPI staining. <br /> (3) HA epitope staining FITC detection. (4) Merged image. <br /> B. Western blot analysis subcellular fractions cells expressing ECT-HA differential centrifugation. Lane 1, microsomal fraction; lane <br /> 2, large granular fraction; lane 3, cytosolic fraction. <br />  <br />  <br />    Overall, perturbation lipid homeo-                   GPEtn(15:0/15:0);      1,2-dimyristoyl-sn-glycero-3-[phospho- <br /> stasis potential interfering numerous cel-                  rac-(1-glycerol)], GPGro(14:0/14:0); 1,2-diheneicosanoyl-sn- <br /> lular processes structures, explains relative                 glycero-3-phosphocholine,        GPCho(21:0/21:0);       1,2- <br />                                                                             dimyristoyl-sn- glycerol-3-[phospho-L-serine], GPSer(14:0/ <br /> rapidity cell death makes TbECT attractive drug <br />                                                                             14:0); 1-dodecanoyl-2-tridecanoyl- sn-glycero-3- <br /> target.                                                                     phospho-(1&#226;&#8364;&#178;-myo-inositol), GPIno(12:0/13:0) purchased <br />    Furthermore, characterization enzymatic                       Avanti Polar Lipids. <br /> properties ECT provided invaluable information for <br /> future experiments aimed developing inhibitors and <br /> new therapeutic agents African trypanosome.                     Cloning sequencing T. brucei ECT <br />  <br />                                                                             The putative TbECT gene amplified T. brucei strain <br /> Experimental procedures                                                     427 genomic DNA 5&#226;&#8364;&#178;-and 3&#226;&#8364;&#178; untranslated <br />                                                                             regions (UTRs) 278 402 bp, respectively, using Pfu <br /> Reagents                                                                    DNA polymerase forward reverse primers <br />                                                                             5&#226;&#8364;&#178; - ATAAGTAAgcggccgcGCTAAAGGTGTTGGTGAAACTAG <br /> Chemicals purchased Sigma-Aldrich Fluka.                       CGC-3&#226;&#8364;&#178; (F1) 5&#226;&#8364;&#178;-ATAAGTAAgcggccgcTGGTGAAAC <br /> Restriction endonucleases DNA modifying enzymes                         AAAACGTTAGTACA-3&#226;&#8364;&#178; (R2) containing NotI restric- <br /> were New England Biolabs Promega. [2-3H] ethano-                    tion site (lower case). The resulting 1.8 kb (ECT UTRs) <br /> lamine (50 Ci mmol-1) D-[2-3H] mannose (15 Ci mmol-1)                   fragment cloned pCR-Blunt-II TOPO vector (Invitro- <br /> were purchased Amersham. [9,10&#226;&#8364;&#8220;3H(N)]- tetrade-                        gen) yielding pCR-Blunt-II-flECT construct. Clones were <br /> canoic acid (myristic acid) (47 Ci mmol-1) Perkin                  sequenced compared annotated Gene Data <br /> Elmer, L-[3&#226;&#8364;&#8220;3H]-serine (20 Ci mmol-1) GDP-                        Bank sequences. <br /> [3H]mannose (20 Ci mmol-1) ARC. L-[35S] methion- <br /> ine (1175 Ci mmol-1) MP Biochemicals. L-[2,2,3&#226;&#8364;&#8220; <br /> 2 <br />   H]-serine (d3-Ser) CDN Isotopes.                                 T. brucei cell culture <br />    The     non-natural     phospholipids  standards   1,2- <br /> diheptadecanoyl-sn-glycero-3-phosphate, GPA(17:0/17:0);                     Bloodstream form T. brucei brucei strain Lister 427, previ- <br /> 1,2-dipentadecanoyl-sn-glycerol-3-phosphoethanolamine                       ously genetically modified express T7 polymerase the <br />  <br />                                                                                                                         &#194;&#169; 2009 The Authors <br />                                                    Journal compilation &#194;&#169; 2009 Blackwell Publishing Ltd, Molecular Microbiology, 73, 826&#226;&#8364;&#8220;843 <br />                                                                                               Ethanolamine metabolism Trypanosoma brucei 839 <br />  <br /> tetracycline repressor protein (Wirtz et al., 1999), referred       These constructs purified using QIAprep Miniprep <br /> to wild-type. This cell line allows inducible expression    Plasmid Kit (Qiagen), linearized NotI, precipitated with <br /> of ectopic genes control T7 promoter           sodium acetate/ethanol dissolved sterile water a <br /> tetracycline operator. Cells cultured 37&#194;&#176;C 5%           final concentration 2 mg ml-1. The DNA elec- <br /> CO2 HMI-9 medium supplemented 2.5 mg ml-1 G418           troporated T. brucei bloodstream form cells: 3 &#194;&#165; 107 cells <br /> to maintain neomycin drug pressure. TbECT conditional           re-suspended T-cell Nucleofector solution trans- <br /> null mutant culturing media supplemented puromy-           fected using program X-001 AMAXA Biosystems <br /> cin, phleomycin, hygromycyn tetracycline concen-         nucleofector. Transfected cells left recover HMI-9 <br /> trations section below. Cells counted daily     medium overnight 6 h; 10-fold dilution cells was <br /> average cell volume recorded CASY Cell Counter           applied stage. After recovery, drug selection was <br /> Analyser System Model TT using lower upper cell dimen-          achieved presence final concentration of <br /> sion limits 2.40 5.70 mm.                                    0.1 mg ml-1 puromycin cells containing PAC gene; <br />                                                                     2.5 mg ml-1 phleomycin cells transfected the <br />                                                                     pLew100 construct; 4 mg ml-1 hygromycyn cells <br /> Stable isotope labelling bloodstream form T. brucei              containing HYG gene. Before deletion second <br />                                                                     TbECT allele expression ectopic TbECT ensured by <br /> 2.5 &#194;&#165; 107T. brucei bloodstream form cells density           addition 1 mg ml-1 tetracycline. This concentration was <br /> 0.5 &#194;&#165; 106 cells ml-1 incubated overnight 37&#194;&#176;C            maintained culturing TbECT conditional null mutant, <br /> HMI-9 media supplemented 1 mM d3-Ser. Total lipids             arrest ECT gene expression achieved by <br /> were extracted method Bligh Dyer (1959),          washing cells times tetracycline-free HMI-9 prior to <br /> samples analysed Micromass Quattro Ultima               culturing tetracycline-free HMI-9. <br /> triple quadrupole mass spectrometer equipped nano- <br /> electrospray source. [M-H] adducts unlabelled GPSer (d3)- <br /> labelled GPSer, unlabelled GPEtn (d3)-labelled GPEtn            RNA isolation cDNA synthesis <br /> were monitored neutral loss scanning m/z 96 99 <br /> and parent ion scanning m/z 196 199 respectively.        Total RNA isolated bloodstream form T. brucei <br /> Each spectrum encompasses 50 repetitive scans.             using RNeasy mini kit (Qiagen). TbECT specific cDNA <br />                                                                     generated amplified using specific forward <br />                                                                     5&#226;&#8364;&#178;-GAGATATACATATGAAACGGTCGGTGTCGAAGG-3&#226;&#8364;&#178; and <br /> Construction T. brucei ECT conditional <span id='am-318' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-319' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-320' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-321' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-322' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-323' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-324' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-325' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-326' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-327' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-328' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-329' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-330' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-331' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-332' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-333' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-334' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-335' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-336' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-337' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-338' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-341' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-342' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-344' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-347' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-348' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-349' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-350' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-351' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-352' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-353' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-354' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-355' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-356' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-357' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-358' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-359' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-362' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-363' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-364' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-365' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-366' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-367' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-368' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-369' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-370' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-372' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-373' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-374' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-375' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-376' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-377' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-378' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-380' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-381' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-382' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-383' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-384' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-385' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-386' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-387' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-388' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-389' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-390' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-391' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-392' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-393' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-394' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-395' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span>gene knockout</span>             reverse 5&#226;&#8364;&#178;-CCGGATCCTCACACCTCTCTGACATTTCTGTA <br />                                                                     C-3&#226;&#8364;&#178; primers using SuperScript III One step RT-PCR kit <br /> To construct T. brucei gene replacement cassettes, 5&#226;&#8364;&#178;       Platinum Taq (Invitrogen). As negative control to <br /> and 3&#226;&#8364;&#178; UTRs adjacent TbECT ORF amplified           exclude DNA contamination RNA sample, reverse tran- <br /> the pCR-Blunt-II-flECT construct using primers F1           scriptase omitted reaction replaced with <br /> 5&#226;&#8364;&#178; - GGATCCGTTTAAACTTACGGACCGTCAAGCTTCAAAG                          GoTaq polymerase (Promega). Tubulin tyrosine-ligase <br /> GGGTAAAACCCACAGATAC-3&#226;&#8364;&#178; (R1) 5&#226;&#8364;&#178;-UTR;                 (TbTTL) cDNA amplified using forward and <br /> primers 5&#226;&#8364;&#178;-AAGCTTGACGGTCCGTAAGTTTAAACGGATCC                         reverse primers 5&#226;&#8364;&#178;-GGAATTCCATATGATGAGTGAGTTCC <br /> ACTTTTATCCTTCGTGAGGAAT-3&#226;&#8364;&#178; (F2) R2 3&#226;&#8364;&#178;                    CGTTGGTGTC-3&#226;&#8364;&#178; 5&#226;&#8364;&#178;-CGCGGATCCGCGTCACGTGC <br /> UTR. The amplified products used knitting            TCCCGATAGGCAATTC-3&#226;&#8364;&#178; equal RNA input. The <br /> PCR, utilized short linker region (underlined) containing    PCR products run 1% agarose gel. <br /> the restriction sites BamHI, HindIII PmeI order to <br /> anneal 5&#226;&#8364;&#178;-to 3&#226;&#8364;&#178; UTR. <br />    A HindIII restriction site internal sequence       Southern Northern blotting <br /> 3&#226;&#8364;&#178;-UTR silenced site directed mutagenesis using the <br /> forward primer 5&#226;&#8364;&#178;-CCATCAAAAAAGAAGGAGAAGCCTTGC                       Southern Northern blotting performed essentially <br /> ATGGATAAGTAGAG-3&#226;&#8364;&#178;, reverse primer 5&#226;&#8364;&#178;-CTCTACTT                   described (Martin Smith, 2006a), using TbECT <br /> ATCCATGCAAGGCTTCTCCTTCTTTTTTGATGG-3&#226;&#8364;&#178;                        specific probes generated PCR using primers previ- <br /> QuickChange Site Directed Mutagenesis Kit (Stratagene).             ously described amplification ECT ORF for <br />    The stitched UTRs ligated pGEM-5Zf(+)             ligation pLew100. <br /> (Promega) NotI sites antibiotic resistance <br /> markers hygromycin phosphotransferase (HYG) puromy- <br /> cin acetyltransferase (PAC) ligated BamHI          Subcellular localization studies <br /> and HindIII restriction sites.                                      Immunofluorescence, subcellular fractionation differen- <br />    To generate tetracycline inducible ectopic copy       tial centrifugation carried TbECT cKO cells <br /> TbECT gene, ORF amplified PCR pCR-              grown presence tetracycline described before <br /> Blunt-II-flECT construct using Pfu polymerase               (Martin Smith, 2006b). <br /> forward reverse primers 5&#226;&#8364;&#178;-CCCAAGCTTGGGATG <br /> AAACGGTCGGTGTCGAAGGT-3&#226;&#8364;&#178; 5&#226;&#8364;&#178;-CCTTAATTAAGG <br /> CACCTCTCTGACATTTCTGTACA-3&#226;&#8364;&#178;. The PCR product                     Mitotracker staining <br /> then cloned vector pLew100, contained a <br /> phleomycin resistance cassette (Wirtz et al., 1999), using      Mid-log bloodstream form T. brucei parental SM cells and <br /> HindIII PacI sites (underlined).                                TbECT cKO cells grown presence absence tet- <br />  <br /> &#194;&#169; 2009 The Authors <br /> Journal compilation &#194;&#169; 2009 Blackwell Publishing Ltd, Molecular Microbiology, 73, 826&#226;&#8364;&#8220;843 <br />  840 F. Gibellini, W. N. Hunter T. K. Smith &#228;&#352;? <br />  <br /> racycline 36 42 h incubated 10 min 37&#194;&#176;C                 twice 0.08 M sodium cacodylate pH 7.2 postfixed/ <br /> in HMI-9 medium containing 50 nM Mitotracker Red CMXRos                     stained 1% osmium tetraoxide (aq) 1 h. Specimens <br /> (Molecular Probes). Cells harvested, washed                    rinsed twice distilled water fixation/ <br /> HMI-9 medium incubated 30 min                        staining 3% uranyl acetate (aq) 16&#226;&#8364;&#8220;18 h. Samples were <br /> absence Mitotracker. Cells collected centrifugation              rinsed twice distilled water, dehydrated graded <br /> (800 g, 10 min), washed PBS fixed 4%(w/v)                       ethanol series washed twice propylene oxide. <br /> paraformaldehyde PBS. Cells counterstained                     Samples infiltrated 50% (v/v) mixture of <br /> 4,6-diamidino-2-phenylindole (DAPI, 2 mg ml-1), washed                   Durcopan resin propylene oxide 24 h rotary <br /> PBS, let adhere polylysine slides slides mounted.                mixer room temperature. After infiltration 24 h <br />                                                                             100% Durcopan resin, samples polymerized 60&#194;&#176;C <br />                                                                             24 h. Sections cut using Leica Ultracut UCT micro- <br /> In vivo T. brucei metabolic labelling                                       tome, mounted pioloform coated copper grids, post <br />                                                                             stained 3% uranyl acetate (aq) Reynold&#226;&#8364;&#8482;s lead <br /> For metabolic labelling 2 &#194;&#165; 107 mid-log cells centrifuged              citrate examined using JEOL-1200 EX transmission <br /> (800 g, 10 min) washed in: serine free Minimal Essential                electron microscope operating accelerating voltage of <br /> Media (MEM), [3H]serine [3H]ethanolamine labelling;                 80 Kv. <br /> methionine free MEM, [35S]methionine labelling; glucose <br /> free MEM, [3H]mannose labelling MEM supplemented <br /> with defatted BSA precoupled [3H]myristate                         Electrospray tandem mass spectrometry <br /> [3H]myristate labelling. The cells re-suspended in <br />                                                                             Total lipids 1 &#194;&#165; 108 trypanosomes extracted the <br /> the appropriate media final concentration 1 &#194;&#165; 107 <br />                                                                             method Bligh Dyer (1959) un-natural lipid stan- <br /> cells ml-1. Cells labelled 1 h 37&#194;&#176;C 50 mCi ml-1 <br />                                                                             dards added prior lipid extraction [1 nmol standard per <br /> of relevant radiolabelled species. Two aliquots containing <br />                                                                             sample, exception GPIno(12:0 13:0) only <br /> each 1 &#194;&#165; 107 cells collected labelling <br />                                                                             0.2 nmol added sample]. Samples analysed <br /> experiment centrifugation (800 g, 10 min) used for <br />                                                                             Micromass Quattro Ultima triple quadrupole mass <br /> lipid protein analysis. <br />                                                                             spectrometer equipped nano-electrospray source, as <br />    Lipids extracted chloroform:methanol:water <br />                                                                             described previously (Guler et al., 2008). Each spectrum <br /> (10:10:3) 1 h, supernatant removed pellet <br />                                                                             encompasses 50 repetitive scans. <br /> extracted fresh 10:10:3 1 h. The supernatants were <br /> pooled dried stream nitrogen prior desalting <br /> using butanol/water partitioning. Lipids separated                  TbECT recombinant protein expression purification <br /> HPTLC using silica 60 HPTLC plates chloroform : <br /> methanol : water (10:10:3 v/v) solvent. The aqueous                  The ECT ORF PCR amplified pCR-Blunt-II- <br /> phases containing ethanolamine metabolites dried                   flECT construct Pfu polymerase using forward <br /> under nitrogen stream separated HPTLC using                        primer 5&#226;&#8364;&#178;-GGAATTCCATATGATGAAACGGTCGGTGTCGA <br /> silica 60 HPTLC plates solvent composed                      AG-3&#226;&#8364;&#178; containing NdeI restriction site (underlined) the <br /> 100% ethanol : 0.5% sodium chloride : 25% ammonium                          reverse primer 5&#226;&#8364;&#178;-GGATCCCCTTGAAAATACAGGTTTTCG <br /> hydroxide (10:10:1). Cold ethanolamine, ethanolamine-                       CCGCCGGTACCCACCTCTCTGACATTTCTGTACACATCT <br /> phosphate CDP-ethanolamine standards run                        GGC-3&#226;&#8364;&#178;, contains BamHI restriction site (underlined) <br /> parallel detected ninhydrin. Radiolabelled species                 encodes product carrying TEV protease cleavage site <br /> were detected fluorography -80&#194;&#176;C, spraying                 (bold). The amplicon purified (QIAquick PCR purification <br /> En3hanceTM using Kodak Biomax MS film inten-                    kit, Qiagen), subcloned pCR-Blunt II TOPO (Invitrogen) <br /> sifying screen. Proteins separated 4&#226;&#8364;&#8220;12% SDS-                     sequenced. Using NdeI BamHI restriction sites <br /> PAGE gel visualized Coomassie blue staining.                         putative TbECT ligated expression vector <br /> Destained gel soaked En3hanceTM (NEN) 30 min,                    pET-20b (Novagen), generating construct pET20b- <br /> washed water twice, soaked 10% glycerol dried.                  TbECT-TEVP-His6, incorporates C-terminal TEV <br /> The dried gel exposed Kodak Biomax MS film                  cleavable hexa-histidine tag expressed. <br /> 7 days -80&#194;&#176;C.                                                               The pET20b-TbECT-TEVP-His6 construct transformed <br />                                                                             BL21(DE3) Codon plus RIL cells clones selected on <br />                                                                             LB-agar plates containing carbenicillin (100 mg ml-1) and <br /> Electron microscopy                                                         chloramphenicol (25 mg ml-1). Starter cultures grown at <br />                                                                             37&#194;&#176;C single colonies 10 ml LB medium containing <br /> For scanning electron microscopy samples fixed                     carbenicillin (50 mg ml-1) chloramphenicol (12.5 mg ml-1), <br /> directly HMI-9 media adding glutaraldehyde final                 5 h. The starter culture used inoculate <br /> concentration 2.5% (v/v) processed described                      1 l autoinduction media (Studier, 2005) cells were <br /> before (Urbaniak et al., 2006). Samples examined                  grown room temperature 24 h. Cells harvested by <br /> using Philips XL 30 environmental scanning electron micro-                centrifugation (3500 g, 20 min, 4&#194;&#176;C) re-suspended in <br /> scope operating accelerating voltage 15 Kv.                        buffer A (50 mM HEPES pH 7.5, 300 mM NaCl, 10 mM <br />    For transmission electron microscopy cell pellets                   imidazole, 2 mM b-mercaptoethanol 5% glycerol). Cells <br /> fixed 24 h Peter&#226;&#8364;&#8482;s fixative (1.25% glutatalde-               lysed presence Dnase I using French <br /> hyde, 1% paraformaldehyde 0.08 M sodium cacodylate                       press lysate cleared centrifugation (35 000 g, <br /> pH 7.2 0.02% calcium chloride). Samples rinsed                    30 min, 4&#194;&#176;C). <br />  <br />                                                                                                                         &#194;&#169; 2009 The Authors <br />                                                    Journal compilation &#194;&#169; 2009 Blackwell Publishing Ltd, Molecular Microbiology, 73, 826&#226;&#8364;&#8220;843 <br />                                                                                              Ethanolamine metabolism Trypanosoma brucei 841 <br />  <br />    The cleared lysate applied 5 ml HisTrapTM column        boiling 100&#194;&#176;C 5 min. Substrates products were <br /> (GE-Healthcare) preloaded Ni2+. Unbound proteins          separated HPTLC using silica 60 plates 100% <br /> removed washing column 15 column volumes             ethanol : 0.5% sodium chloride : 25% ammonium hydroxide <br /> buffer A, TbECT eluted imidazole gradient         (10:10:1, v/v) solvent. Substrates products visu- <br /> in buffer.                                                 alized spraying ninhydrin (0.2% water-saturated <br />    Fractions containing TbECT pooled His-tag           butan-1-ol). <br /> was cleaved TEV protease (1 mg protease per <br /> 10&#226;&#8364;&#8220;20 mg protein substrate) containing uncleavable <br /> His-tag, dialysing buffer A. The cleaved TbECT        Cell-free assay GPI biosynthesis <br /> was purified remaining His-tagged version of <br />                                                                     Membranes T. brucei wild-type TbECT conditional null <br /> TbECT, tag His-tagged TEV protease a <br /> second round nickel ion affinity chromatography,              mutants grown presence absence tetracycline for <br /> described above. Fractions containing cleaved TbECT            30, 36 42 h isolated prepared described <br /> pooled, purity sample checked SDS-PAGE,           previously (without addition tunicamycin prior lysis) <br /> and aliquots snap frozen liquid nitrogen stored      (Smith et al., 1997), snap frozen liquid nitrogen stored <br /> -80&#194;&#176;C.                                                              -80&#194;&#176;C required. The cell-free assay was <br />                                                                     carried visualized described (Smith et al., <br />    In order assess molecular weight oligomeric <br />                                                                     1997) using 1 &#194;&#165; 107 cell equivalents assay, GDP- <br /> state solution TbECT, protein purified above <br />                                                                     [3H]Mannose (0.3 mCi 107 cell equivalents) without <br /> but glycerol, interfere analyses, was <br />                                                                     1 mM PMSF. <br /> omitted purification buffers. Exact molecular weight <br /> was assessed MALDI time-of-flight mass spectrometry. <br /> Sedimentation velocity experiments performed (wave- <br /> <span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span> 280 nm, rotor AN50-TI 45 000 r.p.m. 20&#194;&#176;C), <br />                                                                     Acknowledgements <br /> using Beckman Coulter XL-1 analytical ultracentrifuge. The        The authors like thank Martin Kierans John <br /> samples run 25 mM HEPES pH 7.5, 50 mM NaCl,                 James (CHIPs facility, University Dundee) assistance <br /> 2 mM DTT, 2 mM MgCl2 concentrations 0.25, 0.5             electron microscopy. The authors like to <br /> 0.75 mg ml-1. Samples centrifuged simultaneously           thank Mark Agacan (Dundee) running AUC. This work <br /> A280 measurements taken five-minute intervals 16 h.          supported Wellcome Trust Senior Research <br /> The resultant data analysed using programs                 Fellowship 067441(T.K.S.), Wellcome Trust grants 082596 <br /> SEDFIT SEDNTERP (Schuck, 2000; Lebowitz et al.,                 083481 (WNH) Wellcome Trust PhD studentship <br /> 2002).                                                              (F.G.). <br />  <br /> Assay TbECT activity <br />                                                                     Note added proof <br /> Ethanolamine-phosphate cytidylyltransferase activity was <br />                                                                     The nucleotide sequence reported paper been <br /> measured malachite green colorimetric assay 96-well <br />                                                                     submitted GenBankTM/EBI Data Bank accession <br /> plate format using SpectraMAX 340PC plate reader <br />                                                                     number FM992871. <br /> (Molecular Devices). The assay contained 25 mM Bis-tris <br /> propane pH 7.5, 5 mM MgCl2, 1.425 mg ml-1 recombinant <br /> TbECT, 0.1 <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> inorganic pyrophosphatase (Sigma)            References <br /> varying concentrations CTP Etn-P, total volume of <br /> 50 ml. The reaction carried 5 min room tem-          Ancelin, M.L., Vial, J.H. (1986) Quaternary ammonium <br /> perature stopped addition 100 ml malachite           compounds efficiently inhibit Plasmodium falciparum <br /> green reagent (Biomol Green), absorbance recorded             growth vitro impairment choline transport. Anti- <br /> at 620 nm. Controls conducted make sure enzyme                microb Agents Chemother 29: 814&#226;&#8364;&#8220;820. <br /> activity product formation displayed linear relationship      Baer, E., Stanacev, N.Z. (1964) Phosphonolipids. J Biol <br /> in time span assay. All enzyme assays con-            Chem 239: 3209&#226;&#8364;&#8220;3214. <br /> ducted triplicate. Enzyme Kinetics, SigmaPlot           Bakovic, M., Fullerton, M.D., Michel, V. (2007) Metabolic <br /> (SPSS), used analyse fit kinetic data. Sub-            molecular aspects ethanolamine phospholipid bio- <br /> strate recognition investigated substituting            synthesis: role CTP:phosphoethanolamine cytidylyl- <br /> Etn-P CTP substrate analogues alternative nucle-           transferase (Pcyt2). Biochem Cell Biol 85: 283&#226;&#8364;&#8220;300. <br /> otide phosphates.                                                   Berriman, M., Ghedin, E., Hertz-Fowler, C., Blandin, G., <br />    The pH optimum reaction assessed measur-             Renauld, H., Bartholomeu, D.C., et al. (2005) The genome <br /> ing TbECT activity various pHs: Bis-Tris methane (pH 6.5,          African trypanosome Trypanosoma brucei. Science <br /> 7.0), Bis-Tris propane (pH 7.5, 8.0, 8.5) Glycine (pH 9.0).        309: 416&#226;&#8364;&#8220;422. <br />    The production CDP-ethanolamine ethanolamine-          Bladergroen, B.A., Houweling, M., Geelen, M.J.H., and <br /> phosphate cytidylyltransferase assay assessed using             van Golde, L.M.G. (1999) Cloning expression of <br /> modified method Tijburg et al. (1992). Briefly, 2 mg puri-      CTP:phosphoethanolamine cytidylyltransferase cDNA from <br /> fied protein incubated reaction mixture (total             rat liver. Biochem J 143: 107&#226;&#8364;&#8220;114. <br /> volume 50 ml) 100 mM Tris pH 8, 10 mM MgCl2, 5 mM                Bleijerveld, O.B., Klein, W., Vaandrager, A.B., Helms, J.B., <br /> CTP, 4 mM Etn-P 37&#194;&#176;C 20 min quenched                Houweling, M. (2004) Control CDPethanolamine <br />  <br /> &#194;&#169; 2009 The Authors <br /> Journal compilation &#194;&#169; 2009 Blackwell Publishing Ltd, Molecular Microbiology, 73, 826&#226;&#8364;&#8220;843 <br />  842 F. Gibellini, W. N. Hunter T. K. Smith &#228;&#352;? <br />  <br />    pathway mammalian cells: effect CTP:phosphoetha-                   Menon, A.K., Eppingerl, M., Mayor, S., Schwarz, R. <br />    nolamine cytidylyltransferase overexpression                       (1993) Phosphatidylethanolamine donor the <br />    intracellular diacylglycerol. Biochem J 379: 711&#226;&#8364;&#8220;                phosphoethanolamine group trypanosome glycosyl- <br />    719.                                                                       phosphatidylinositol. EMBO J 12: 1907&#226;&#8364;&#8220;1914. <br /> Bligh, E.G., Dyer, W.J. (1959) A rapid method total                  Mizoi, J., Nakamura, M., Nishida, I. (2006) Defects in <br />    lipid extraction purification. Can J Biochem Physiol 37:               CTP:phosphorylethanolamine cytidylyltransferase affect <br />    911&#226;&#8364;&#8220;917.                                                                   embryonic postembryonic development Arabidopsis. <br /> Choubey, V., Maity, P., Guha, M., Kumar, S., Srivastava, K.,                  Plant Cell 18: 3370&#226;&#8364;&#8220;3385. <br />    Puri, S.K., Bandyopadhyay, U. (2007) Inhibition                   Nagamune, K., Nozaki, T., Maeda, Y., Ohishi, K., Fukuma, <br />    Plasmodium falciparum choline kinase hexadecyl-                         T., Hara, T., et al. (2000) Critical roles glycosylphos- <br />    trimethylammonium bromide: possible antimalarial                         phatidylinositol Trypanosoma brucei. Proc Natl Acad Sci <br />    mechanism. Antimicrob Agents Chemother 51: 696&#226;&#8364;&#8220;706.                        USA 97: 10336&#226;&#8364;&#8220;10341. <br /> Dowhan, W. (1997) Molecular basis membrane phospho-                     Nakashima, A., Hosaka, K., Nikawa, J. (1997) Cloning of <br />    lipid diversity: Why lipids? Annu Rev                    human cDNA CTP-phosphoethanolamine cytidylyl- <br />    Biochem 66: 199&#226;&#8364;&#8220;232.                                                       transferase complementation vivo yeast mutant. <br /> Ferguson, M.A.J., Allen, A.K., Snary, D. (1982) The                       J Biol Chem 272: 9567&#226;&#8364;&#8220;9572. <br />    detection phosphonolipids protozoan Trypano-                   Ohtsuka, J., Nagata, K., Lee, W.C., Ono, Y., Fukuda, R., <br />    soma cruzi. Biochem J 207: 171&#226;&#8364;&#8220;174.                                        Ohtab, A., Tanokura, M. (2006) Crystallization and <br /> Guler, J.L., Kriegova, E., Smith, T.K., Lukes, J., Englund,               preliminary X-ray analysis CTP:phosphoethanolamine <br />    P.T. (2008) Mitochondrial fatty acid synthesis required                 cytidylyltransferase (ECT) Saccharomyces cerevisiae. <br />    normal mitochondrial morphology function                        Acta Cryst F62: 1003&#226;&#8364;&#8220;1005. <br />    Trypanosoma brucei. Mol Microbiol 67: 1125&#226;&#8364;&#8220;1142.                         Pattridge, K.A., Weber, C.H., Friesen, J.A., Sanker, S., Kent, <br /> Guther, M.L.S., Masterson, W.J., Ferguson, M.A.J.                         C., Ludwig, M.L. (2003) Glycerol-3-phosphate <br />    (1994) The effects phenylmethylsulfonyl fluoride                     cytidylyltransferase. J Biol Chem 278: 51863&#226;&#8364;&#8220;51871. <br />    inositol-acylation fatty acid remodeling African                  Rifkin, M.R., Strobos, C.A.M., Fairlamb, A.H. (1995) <br />    trypanosomes. J Biol Chem 269: 18694&#226;&#8364;&#8220;18701.                                Specificity ethanolamine transport further <br /> van Hellemond, J.J., Tielens, A.G.M. (2006) Adaptations                   metabolism Trypanosoma brucei. J Biol Chem 270: <br />    lipid metabolism protozoan parasite Trypano-                 16160&#226;&#8364;&#8220;16166. <br />    soma brucei. FEBS Lett 580: 5552&#226;&#8364;&#8220;5558.                                   Roper, J.R., Guther, M.L.S., Milne, K.G., Ferguson, <br /> Hern&#195;&#161;ndez-Alcoceba, R., Saniger, L., Campos, J., N&#195;&#186;&#195;&#177;ez.                       M.A.J. (2002) Galactose metabolism essential the <br />    M.C., Khaless, F., Gallo, M.A., et al. (1997) Choline kinase               African sleeping sickness parasite Trypanosoma brucei. <br />    inhibitors novel approach antiproliferative drug                  Proc Natl Acad Sci USA 99: 5884&#226;&#8364;&#8220;5889. <br />    design. Oncogene 15: 2289&#226;&#8364;&#8220;2301.                                          Sarkar, M., Hamilton, C.J., Fairlamb, A.H. (2003) Prop- <br /> Hofer, A., Ekanem, J.T., Thelander, L. (1998) Allosteric                  erties phosphoenolpyruvate mutase, enzyme in <br />    regulation Trypanosoma brucei ribonucleotide reductase                  aminoethylphosphonate biosynthetic pathway Trypa- <br />    studied vitro vivo. J Biol Chem 273: 34098&#226;&#8364;&#8220;                      nosoma cruzi. J Biol Chem 278: 22703&#226;&#8364;&#8220;22708. <br />    34104.                                                                   Schuck, P. (2000) Size distribution analysis macro- <br /> Imhof, I., Canivenc-Gansel, E., Meyer, U., Conzelmann,                    molecules sedimentation velocity ultracentrifugation <br />    A. (2000) Phosphatidylethanolamine donor                         Lamm equation modeling. Biophys J 78: 1606&#226;&#8364;&#8220;1619. <br />    phosphorylethanolamine linked a-1,4-linked                    Signorell, A., Jelk, J., Rauch, M., B&#195;&#188;tikofer, P. (2008a) <br />    mannose yeast GPI structure. Glycobiology 10: 1271&#226;&#8364;&#8220;                     Phosphatidylethanolamine precursor ethano- <br />    1275.                                                                      lamine phosphoglycerol moiety bound eukaryotic elon- <br /> Jez, J.M. (2007) Phosphatidylcholine biosynthesis                        gation factor 1A. J Biol Chem 283: 20320&#226;&#8364;&#8220;20329. <br />    potential target inhibition metabolism parasitic               Signorell, A., Rauch, M., Jelk, J., Ferguson, M.A.J., and <br />    nematodes. Current Enzyme Inhibition 3: 133&#226;&#8364;&#8220;142.                           B&#195;&#188;tikofer, P. (2008b) Phosphatidylethanolamine Trypa- <br /> Lazarowski, E.R., Boucher, R.C. (2001) UTP extra-                   nosoma brucei organized separate pools is <br />    cellular signalling molecule. News Physiol Sci 16: 1&#226;&#8364;&#8220;5.                    synthesized exclusively Kennedy pathway. J Biol <br /> Lebowitz, J., Lewis, M.S., Schuck, P. (2002) Modern                       Chem 283: 23636&#226;&#8364;&#8220;23644. <br />    analytical ultracentrifugation protein science: tutorial            Smith, T.K., Cottaz. S., Brimacombe, J.S., Ferguson, <br />    review. Protein Sci 11: 2067&#226;&#8364;&#8220;2079.                                         M.A.J. (1996) Substrate specificity Dol-P- <br /> Martin, K.L., Smith, T.K. (2006a) Phosphatidylinositol                    Man:GlcN-PI a1&#226;&#8364;&#8220;4 mannosyltransferase glyco- <br />    synthesis essential bloodstream form Trypanosoma                     sylphosphatidylinositol biosynthetic pathway African <br />    brucei. Biochem J 396: 287&#226;&#8364;&#8220;295.                                            Trypanosomes. J Biol Chem 271: 6476&#226;&#8364;&#8220;6482. <br /> Martin, K.L., Smith, T.K. (2006b) The glycosylphosphati-                Smith, T.K., Sharma, D.K., Crossman, A., Dix, A., Brima- <br />    dylinositol (GPI) biosynthetic pathway bloodstream form                 combe, J.S., Ferguson, M.A.J. (1997) Parasite and <br />    Trypanosoma brucei dependent novo synthe-                     mammalian GPI biosynthetic pathways distin- <br />    sis inositol. Mol Microbiol 61: 89&#226;&#8364;&#8220;105.                                 guished using synthetic substrate analogues. EMBO J 16: <br /> Menon, A.K., Stevens, V.L. (1992) Phosphatidylethano-                     6667&#226;&#8364;&#8220;6675. <br />    lamine donor ethanolamine residue linking                Steenbergen, R., Nanowski, T.S., Beigneux, A., Kulinski, A., <br />    glycosylphosphatidylinositol anchor protein. J Biol Chem                Young, S.G., Vance, J.E. (2005) Disruption the <br />    267: 15277&#226;&#8364;&#8220;15280.                                                          phosphatidylserine decarboxylase gene mice causes <br />  <br />                                                                                                                         &#194;&#169; 2009 The Authors <br />                                                    Journal compilation &#194;&#169; 2009 Blackwell Publishing Ltd, Molecular Microbiology, 73, 826&#226;&#8364;&#8220;843 <br />                                                                                               Ethanolamine metabolism Trypanosoma brucei 843 <br />  <br />    embryonic lethality mitochondrial defects. J Biol Chem         ethanolamine cytidylyltransferase purified rat liver. <br />    280: 40032&#226;&#8364;&#8220;40040.                                 <br /> </body></html>