<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Functional Analysis Arabidopsis Postprenylation CaaX <br /> Processing Enzymes Their Function Subcellular <br /> Protein Targeting1[C][W][OA] <br />  <br /> Keren Bracha-Drori2,3, Keren Shichrur2, Tsofnat Cohen Lubetzky4, Shaul Yalovsky* <br /> Department Plant Sciences, Tel Aviv University, Tel Aviv 69978, Israel <br />  <br />  <br /> Prenylation posttranslational protein modification essential developmental processes response abscisic acid. <br /> Following prenylation, C-terminal residues proteoliticaly removed turn free carboxyl group the <br /> isoprenyl cysteine methylated. The proteolysis methylation, collectively referred CaaX processing, catalyzed by <br /> Ste24 endoprotease Rce1 endoprotease isoprenyl cysteine methyltransferase (ICMT). Arabidopsis (Arabidopsis <br /> thaliana) contains single STE24 RCE1 ICMT homologs. Here yeast (Saccharomyces cerevisiae) <br /> AtRCE1 promoted a-mating factor secretion membrane localization ROP GTPase. Furthermore, green fluorescent <br /> protein fusion proteins AtSTE24, AtRCE1, AtICMTA, AtICMTB colocalized endoplasmic reticulum, indicating <br /> that prenylated proteins reach compartment CaaX processing likely required subcellular targeting. <br /> AtICMTB process yeast a-factor efficiently AtICMTA. Sequence mutational analyses revealed the <br /> higher activity AtICMTB conferred residues, conserved yeast Ste14p, human ICMT, AtICMTB <br /> but AtICMTA. Quantitative real-time reverse transcription-polymerase chain reaction microarray data that <br /> AtICMTA expression significantly lower compared AtICMTB. AtICMTA <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span>null mutants</span> wild-type <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span>phenotype</span>, <br /> indicating function redundant. However, AtICMT RNAi lines fasciated inflorescence stems, altered phylotaxis, <br /> and developed multiple buds stem elongation. The phenotype ICMT RNAi lines similar farnesyltransferase <br /> b-subunit mutant enhanced response abscisic acid2 subtle. Collectively, data suggest AtICMTB likely the <br /> major ICMT methylation modulates activity prenylated proteins. <br />  <br />  <br />    Prenylation posttranslational protein modifica-                    anyl isoprenoid moieties conserved CaaX box or <br /> tion essential function diverse proteins. Plant                 double Cys C-terminal sequence motifs. Protein far- <br /> mutants lacking prenyltransferase function pleo-                      nesyltransferase (PFT) attaches farnesyl group <br /> tropic phenotypes including enlargement shoot                       farnesyldiphosphate prenyl group donor a <br /> apical meristem, hypersensitivity abscisic acid (ABA),                  thioether linkage Cys CaaX box sequence <br /> retarded growth rate, delayed flowering (Cutler                        motif (C, Cys; a, usually aliphatic amino acid; X can <br /> et al., 1996; Pei et al., 1998; Running et al., 1998, 2004;                amino acid commonly Ser, Met, Cys, Ala, or <br /> Bonetta et al., 2000; Yalovsky et al., 2000; Ziegelhoffer                  Gln). A second protein prenyltransferase, geranylger- <br /> et al., 2000; Allen et al., 2002; Johnson et al., 2005).                   anyltransferase type I (PGGT-I), uses geranylgeranyl- <br />    Prenylation involves attachment                       diphosphate modify CaaX box containing proteins <br /> 15-carbon farnesyl 20-carbon geranygeranylger-                      X residue usually Leu. A protein <br />                                                                            prenyltransferase, Rab geranylgeranyltransferase, uses <br />                                                                            geranylgeranyldiphosphate modify C-terminal dou- <br />    1                                                                       ble Cys motifs Rab proteins (Zhang Casey, 1996; <br />      This work supported Israel Science Foundation <br />                                                                            Yalovsky et al., 1999; Galichet Gruissem, 2003; <br /> (grant no. ISF 312/07 S.Y.), U.S.-Israel Binational Agricultural <br /> Research Development Fund (grant no. BARD 4032/07 S.Y.), <br />                                                                            Maurer-Stroh et al., 2003). <br /> and German Israel Foundation (grant no. GIF 834/2005 S.Y.).            Prenylation followed proteolytic cleavage of <br />    2 <br />      These authors contributed equally article.                     amino acids protein (aaX), leaving <br />    3 <br />      Present address: Clontech Inc., Mountain View, CA 94043.              isoprenyl carboxylate group C-terminal end of <br />    4 <br />      Present address: Eldan Inc., Petach Tikvah 49170, Israel.             protein. The prenylated isoprenyl Cys turn <br />    * Corresponding author; e-mail shauly@tauex.tau.ac.il.                  methylated (Supplemental Fig. S1). The proteolytic <br />    The author responsible distribution materials integral    cleavage methylation prenylated proteins <br /> findings presented article accordance policy           collectively referred CaaX processing (Young <br /> described Instructions Authors (www.plantphysiol.org) is:       et al., 2000). The prenyl-dependent endoproteolysis is <br /> Shaul Yalovsky (shauly@tauex.tau.ac.il). <br />    [C]                                                                     catalyzed endoproteases, desig- <br />        Some figures article displayed color online in <br /> black white print edition. <br />                                                                            nated Rce1 Ste24 (Afc1; Boyartchuk et al., 1997; <br />    [W] <br />        The online version article contains Web-only data.          Young et al., 2000). The methylation newly ex- <br />    [OA] <br />         Open Access articles viewed online sub-           posed isoprenyl carboxylate group catalyzed an <br /> scription.                                                                 isoprenyl carboxy methyltransferase (Icmt) known <br />    www.plantphysiol.org/cgi/doi/10.1104/pp.108.120477                      Ste14 yeast (Saccharomyces cerevisiae; Clarke, 1992; <br /> Plant Physiology, September 2008, Vol. 148, pp. 119&#226;&#8364;&#8220;131, www.plantphysiol.org &#195;&#8220; 2008 American Society Plant Biologists         119 <br />  Bracha-Drori et al. <br />  <br />  <br /> Romano et al., 1998; Young et al., 2000). In yeast          In plants treated ICMT inhibitor AFC, target- <br /> animal cells, CaaX proteases methyltransferase       ing prenylated calmodulin CaM53 plasma <br /> are localized endoplasmic reticulum (ER) mem-         membrane compromised accumulated in <br /> branes, unlike prenyltransferases, soluble     endomembranes (Rodriguez-Concepcion et al., 2000). <br /> enzymes (Dai et al., 1998; Romano et al., 1998; Schmidt         This work highlights conservation unique- <br /> et al., 1998). This indicates CaaX proteins         ness CaaX processing plants. We the <br /> prenylated cytoplasm processed        CaaX processing enzymes localized ER and <br /> endomembrane system. In animals, CaaX processing             complement respective yeast mutants. Uniquely, <br /> affects fundamental processes. In fibroblasts derived        Arabidopsis ICMT horologes, ICMTA and <br /> <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> Rce1 Icmt <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span>knockout</span> mice, farnesylated Ras        ICMTB. In yeast cells ICMTA active than <br /> not Rho proteins mislocalized plasma           ICMTB. Sequence mutational analyses revealed <br /> membrane endomembrane (Kim et al., 1999;              residues responsible activity difference. <br /> Bergo et al., 2001; Michaelson et al., 2005). Rce1       Real-time reverse transcription (RT)-PCR whole- <br /> Icmt knockout mice die embryogenesis (Kim             genome expression analysis ICMTA expres- <br /> et al., 1999; Bergo et al., 2001) mutations affect   sion levels significantly lower ICMTB. A <br /> oncogenic Ras transforming ability (Bergo et al., 2002,      T-DNA mutant analysis shows ICMTA function <br /> 2004). Disruption Ste24 induces premature aging,        redundant ICMT <span id='am-238' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-239' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-241' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-242' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-243' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-244' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-250' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-253' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-256' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-258' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-262' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-264' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-277' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-279' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-290' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-297' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-299' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span>RNA interference</span> (RNAi) <br /> condition known Hutchinson-Gilford progeria syn-          plants display subtle phenotypes reminiscent era1 <br /> drome, resulting accumulation prenylated un-         PFT b-subunit plp PFT PGGT-I shared <br /> processed prelamin A (Bergo et al., 2002; Pendas et al.,     a-subunit mutants. <br /> 2002; Young et al., 2005). <br />    In Arabidopsis (Arabidopsis thaliana), CaaX proteolysis <br /> is catalyzed conserved AtSTE24 AtRCE1/ <br /> AtFACE-2 (Bracha et al., 2002; Cadinanos et al., 2003).      RESULTS <br /> AtSTE24 demonstrated localized ER           Arabidopsis AtRCE1 Is Functionally Conserved <br /> (Bracha et al., 2002). However, subcellular localiza- <br /> tion AtRCE1/AtFACE-2 determined,            RCE1 considered major CaaX protease. The <br /> has effect subcellular localization prenylated     function AtRCE1 tested complementation in <br /> plant proteins. Plant protein extracts shown    yeast using pheromone diffusion halo assays (Supple- <br /> contain ICMT activity (Crowell et al., 1998; Rodriguez-      mental Fig. S2; Bracha et al., 2002). This assay be <br /> Concepcion et al., 2000), genes encoding ICMT        used quantitatively evaluate CaaX processing effi- <br /> have cloned (Rodriguez-Concepcion et al., 2000;         ciency vivo size growth inhibition <br /> Crowell Kennedy, 2001; Narasimha Chary et al.,           halos correspond efficiency a-factor processing <br /> 2002). Following identification second ICMT       (Tam et al., 1998; Schmidt et al., 2000; Trueblood et al., <br /> gene Arabidopsis, genes designated           2000). AtRCE1 complemented a-factor secretion in <br /> AtICMTA/AtSTE14A AtICMTB/AtSTE14B (Narasimha             ste24D rce1D double-mutant yeast strain (JRY6959) <br /> Chary et al., 2002). It shown AtICMTA           expressed 2m high-copy-number plasmid <br /> AtICMTB differentially expressed vitro       (Fig. 1). Unlike Ste24, Rce1 catalyze CaaX <br /> AtICMTB higher catalytic activity (Narasimha             proteolysis additional proteolytic step re- <br /> Chary et al., 2002). Treatment plants high concen-   quired maturation a-factor efficient <br /> tration methyltransferase inhibiter acetyl farnesyl   secretion (Supplemental Fig. S2; Boyartchuk Rine, <br /> Cys (AFC) induced ABA hypersensitivity (Narasimha            1998; Tam et al., 1998). The halos formed around <br /> Chary et al., 2002), reminiscent ABA hypersen-        AtRCE1-expressing ste24D rce1D cells (SYY563) <br /> sitivity enhanced response abscisic acid1 (era1) PFT   smaller compared halos formed around <br /> b-subunit, PGGT-I b-subunit, pluripetala (plp) PFT       SYY500 AtSTE24-expressing ste24D rce1D cells (Fig. <br /> and PGGT-I a-subunit mutants (Cutler et al., 1996; Pei       1, B C), indicating a-factor secretion was <br /> et al., 1998; Running et al., 2004; Johnson et al., 2005).   efficient. The halos formed around <br />    CaaX processing suggested             Ste24 Rce1 wild-type cells (JRY6958) bigger than <br /> increase protein affinity cell membranes (Hancock         AtSTE24- AtRCE1-complemented cells (Fig. <br /> et al., 1991; Kato et al., 1992; Ghomashchi et al., 1995;    1A), indicating a-factor processing plant <br /> Parish et al., 1995), protect proteins degradation      CaaX proteases efficient yeast Ste24. These <br /> (Backlund, 1997), facilitate functional interactions     data corroborated functional analysis AtRCE1 func- <br /> with proteins (Higgins Casey, 1994; Rosenberg      tion vitro (Cadinanos et al., 2003). <br /> et al., 1998). More recently shown CaaX        <span id='am-3' about='protege:TO' typeof='owl:Thing'>To</span> determine AtRCE1 process plant <br /> processing required subcellular targeting          target proteins, created yeast strain which <br /> prenylated proteins. Consistent ER localiza-        AtRCE1 GFP-AtROP9 (GFP-AtRAC7) coex- <br /> tion CaaX processing enzymes (Dai et al., 1998;           pressed ste24D rce1D mutant background (SYY569; <br /> Romano et al., 1998; Schmidt et al., 1998), Ras proteins     Fig. 2). AtROP9 (AtRAC7) type-II ROP/RAC <br /> accumulate endomembrane Rce1            GTPase farnesylated yeast (Lavy et al., <br /> Icmt mutant cells (Michaelson et al., 2005).                 2002). In wild-type Ste24 Rce1 background (SYY538), <br /> 120                                                                                         Plant Physiol. Vol. 148, 2008 <br />                                                                                                                CaaX Processing Arabidopsis <br />  <br />  <br />  <br />  <br />          Figure 1. Arabidopsis AtSTE24 AtRCE1 facilitate yeast a-factor processing. Growth inhibition halos a-factor hypersen- <br />          sitive sst2a cells formed wild-type Ste24 Rce1 cells (A), ste24D rce1D expressing AtSTE24 (B), ste24D rce1D cells <br />          expressing AtRCE1 (C), ste24D rce1D cells transformed vector control (D). <br />  <br />  <br /> GFP-AtROP9 localized plasma membrane                           latory role (Young et al., 2000). Recently, prenyl Cys <br /> and endomembrane (Fig. 2A). In contrast, ste24D                        methylesterase identified Arabidopsis <br /> rce1D double-mutant background (SYY542), GFP-                             (Deem et al., 2006), suggesting methylation/de- <br /> AtROP9 localized endomembrane                             methylation cycles prenylated proteins regu- <br /> fraction (Fig. 2B). Coexpression AtRCE1 (SYY569)                       late function. This prompted study the <br /> restored plasma membrane localization GFP-ROP9,                        role prenyl Cys methytransferases func- <br /> confirming AtRCE1 processed AtROP9 (Fig. 2C).                        tion Arabidopsis <br /> The nonprenylated GFP:Atrop9mS mutant                               In vitro, AtICMTB/AtSTE14B (At5g08335) was <br /> the prenyl acceptor Cys mutated Ser                            shown active AtICMTA/AtSTE14A <br /> dispersed cytoplasm following expression                        (At5g23320; Narasimha Chary et al., 2002). Sequence <br /> Ste24 Rce1 wild-type cells (SYY539; Fig. 2D). These                       comparison highlighted amino acids con- <br /> data prenylation promotes association                        served AtICMTB yeast Ste14p differ <br /> proteins endomembrane system, CaaX                         AtICMTA (Fig. 4A, red rectangles). These residues <br /> processing required subcellular targeting.                         include: K154 K155 Ste14p R111 R112 in <br />                                                                           AtICMTB, D208 Ste14p E165 AtICMTB, N230 <br /> Subcellular Localization CaaX Processing Enzymes                       K231 Ste14p, Q187 R188 AtICMTB. <br />  <br />   We previously demonstrated AtSTE24 is <br /> localized ER (Bracha et al., 2002) that <br /> ICMTA localized endomembranes (Rodriguez- <br /> Concepcion et al., 2000). If plants prenylated <br /> proteins reach ER, undergo CaaX <br /> processing, AtRCE1, ICMTA, ICMTB also <br /> be localized ER. To test assumption, the <br /> subcellular localization AtRCE1, AtICMTA, and <br /> AtICMTB determined. <br />   Following transient expression, AtICMTA, AtICMTB, <br /> and AtRCE1 GFP fusion proteins typical ER, <br /> reticulate, perinuclear subcellular distribution pat- <br /> terns (Fig. 3, A, E, I; Supplemental Fig. S3, B, D, E, <br /> and G). Colocalization analysis AtICMTA, AtICMTB, <br /> and AtRCE1 ER marker ER-mCherry <br /> (Nelson et al., 2007) carried confirm ER <br /> localization. The results (Fig. 3, B&#226;&#8364;&#8220;D, F&#226;&#8364;&#8220;H, J&#226;&#8364;&#8220;L) <br /> show fusion proteins CaaX processing <br /> enzymes colocalized marker confirming <br /> their ER localization. Colocalization cyan fluores- <br /> cent protein (CFP)-AtICMTA yellow fluorescent <br /> protein (YFP)-AtSTE24 YFP-ICMTA CFP- <br /> AtRCE1 used confirm ER colocal-                        Figure 2. AtRCE1 promotes plasma membrane localization GFP: <br /> ization CaaX processing enzymes ER                          AtROP9 ste24D rce1D double-mutant yeast cells. A, GFP:AtROP9 <br />                                                                           detected plasma membranes endomembranes following <br /> (Supplemental Fig. S3, A&#226;&#8364;&#8220;F). <br />                                                                           expression wild-type Ste24 Rce1 cells. B, GFP:AtROP9 detected only <br />                                                                           endomembranes ste24D rce1D double-mutant cells. C, <br /> ICMTB Is Likely Major Carboxymethyl Isoprenyl                         AtRCE1 restored plasma membrane localization GFP-AtROP9 in <br /> Transferase Arabidopsis                                                ste24D rce1D cells. D, Nonprenylated GFP:Atrop9mS mutant protein <br />                                                                           dispersed cytoplasm following expression wild-type Ste24 <br />  Based reversibility, isoprenyl carboxy                        Rce1 cells. Bars 5 mm. [See online article color version this <br /> methylation step suggested play regu-                       figure.] <br />  <br /> Plant Physiol. Vol. 148, 2008                                                                                                                 121 <br />  Bracha-Drori et al. <br />  <br />  <br /> Figure 3. Subcellular localization of <br /> AtICMTA, AtICMTB, AtRCE1 GFP fu- <br /> sion proteins following transient expres- <br /> sion N. benthamiana leaf epidermal <br /> cells. GFP-AtICMTA (A&#226;&#8364;&#8220;D), GFP-AtICMTB <br /> (E&#226;&#8364;&#8220;H), GFP-AtRCE1 (I&#226;&#8364;&#8220;L) ex- <br /> pressed (A, E, I) coexpressed <br /> with ER-mCherry (Nelson et al., 2007; <br /> B&#226;&#8364;&#8220;D, F&#226;&#8364;&#8220;H, J&#226;&#8364;&#8220;L). Shown GFP chan- <br /> nel (B, F, J), mCherry channel (C, G, <br /> and K), GFP/mCherry overlay (D, H, <br /> and L). Bars 10 mm. <br />  <br />  <br />  <br />  <br /> Interestingly, residues structurally con-   D F). Halo sizes calculated relative wild- <br /> served human (HsICMT) moss Physcomitrella           type Ste14 cells, average size taken 1. The <br /> patens (PpICMT) ICMT proteins (Fig. 4A). Hydropathy        average halo size AtICMTA-complemented cells <br /> plot analysis showed AtICMTB AtICMTA              (when expressed 2m plasmid) 0.57 wild- <br /> have transmembrane helices     type Ste14 cells SE 60.09. The average halo sizes <br /> overall topology similar Ste14p (Fig. 4B). Inter-    AtICMTB- AticmtAmR111R112E165Q187R188- <br /> estingly, residues conserved be-     complemented cells (expressed 2m plasmids) <br /> tween AtICMTB Ste14p, HsICMT PpICMT,               1.05 6 0.07 0.96 6 0.06, respectively. These <br /> and differ AtICMTA located hydrophylic           results indicated a-factor processed at <br /> domains (Fig. 4B, asterisks), shown       similar efficiencies AtICMTB mutant <br /> essential Ste14p activity (Romano Michaelis,       AticmtAmR111R112E165Q187R188 confirmed re- <br /> 2001).                                                     duced activity AtICMTA results changes in <br />    Pheromone diffusion halo assays carried        structurally conserved residues technical <br /> to compare activities AtICMTA AtICMTB           problems experimental setup. Interestingly, ac- <br /> in yeast differences ability       tivity AticmtAmE165 single mutant the <br /> process a-factor correspond structural conser-      AticmtAmR111R112E165 triple mutant similar to <br /> vation AtICMTB Ste14p (Fig. 5). The            wild-type AtICMTA (data shown), suggesting <br /> halos formed ste14D mutant cells          residues critical efficient ICMT <br /> expressed AtICMTB 2m high-copy-number               activity. <br /> plasmid (SYY564) larger halos              Real-time quantitative RT-PCR (Q-PCR) experiments <br /> formed ste14D cells expressed AtICMTA          showed AtICMTB expressed significantly <br /> (SYY526) comparable size halos     higher levels compared AtICMTA (Fig. 6A). The <br /> formed wild-type Ste14 cells (Fig. 5, A&#226;&#8364;&#8220;C).         Q-PCR data corroborated examining microarray <br /> When AtICMTB expressed single-copy CEN          expression data using Genevestigator (Zimmermann <br /> plasmid (SYY565), halo size smaller, indicating    et al., 2004; Laule et al., 2006; Fig. 6B; Supplemental <br /> that methylation a-factor AtICMTB ef-       Fig. S4). The microarray data expression of <br /> ficient yeast Ste14. An AtICMTA mutant,            ICMTA low tissues develop- <br /> AticmtAmR111R112E165Q187R188, harboring point mutations    ment cases expression ICMTB is <br /> that converted amino acids corresponding     higher. The Q-PCR analyses showed AtRCE1 is <br /> residues AtICMTB (N111/R, Y112/R, Q165/E, E187/Q,   expressed similar levels AtICMTB (Fig. 6A). Un- <br /> S188/R) created. The halos formed          fortunately, AtRCE1 (At2G36305) currently repre- <br /> ste14D cells expressed AticmtAmR111R112E165Q187R188   sented ATH1 Affymetrix microarray chips and <br /> (SYY579 SYY578) comparable halos           whole-genome analysis expression cur- <br /> that formed AtICMTB-expressing cells (Fig. 5,       rently unavailable. The high sensitivity Q-PCR <br /> 122                                                                                       Plant Physiol. Vol. 148, 2008 <br />                                                                                                                      CaaX Processing Arabidopsis <br />  <br />  <br />  <br />  <br />          Figure 4. Sequence alignment AtICMTA, AtICMTB, yeast Ste14p, human P. patens ICMT. A, Identical residues all <br />          proteins highlighted yellow. Identical residues proteins highlighted blue. Similar residues are <br />          highlighted green. Red rectangles denote residues structurally conserved AtICMTB Ste14p human <br />          moss ICMTs differ AtICMTA. B, Kyte Doolittle hydropathy plot ICMTB nine-amino acid window size. <br />          Hydrophobic residues appear midline hydrophilic residues midline. Asterisks (*) mark positions of <br />          AtICMTB Ste14, HsICMT, PhyPaICMT conserved, AtICMTA unique residues. <br />  <br />  <br /> enabled detection slightly higher relative expression                         The activity expression data suggested that <br /> levels AtRCE1 AtICMTB flowers leaves                             AtICMTA minor restricted function that <br /> relative roots seedlings.                                               AtICMTB likely major isoprenyl methyltrans- <br /> Plant Physiol. Vol. 148, 2008                                                                                                                    123 <br />  Bracha-Drori et al. <br />  <br />  <br />  <br />  <br />          Figure 5. Pheromone diffusion halo assay identifies amino acids responsible reduced efficiency a-factor processing <br />          AtICMTA. Growth inhibition halos produced a-factor secretion mat sst2a cells. A, Wild-type Ste14 cells. <br />          B G, ste14D mutants cells expressing: AtICMTA (B), AtICMTB (C), AticmtAmR154R155E165Q187R188 (D), AtICMTB (E), <br />          AticmtAmR154R155E165Q187R188 (F), vector control (G). B, C, D, G cells transformed 2m high-copy-number plas- <br />          mids. E F cells transformed CEN single-copy plasmids. <br />  <br />  <br /> ferase Arabidopsis. Mutant analyses carried                        axillary flowers observed era1 PFT <br /> out test assumption.                                               b-subunit mutant plants (Fig. 7H), suggesting func- <br />                                                                            tional relatedness genes. Unlike era1 and <br /> The Function ICMTA ICMTB Arabidopsis                             plp plants (Running et al., 1998; Yalovsky et al., 2000; <br />    An Arabidopsis line (P65H9) T-DNA insertion                      Running et al., 2004), Aticmtsil flowers did have <br /> three nucleotides upstream termination TGA                          extra flower organs additional rosette leaves and <br /> codon AtICMTA (Supplemental Figs. S5 S6)                        late flowering partially male sterile. <br /> identified screens University Wisconsin                          It difficult observe consistent reduction in <br /> Biotechnology Center T-DNA collection (www.biotech.                        mRNA levels Aticmtsil plants, suggesting a <br /> wisc.edu). No RNA detected AticmtA2/2 homo-                         posttranscriptional silencing mechanism. To prove <br /> zygote mutant plants RT-PCR AtICMTA gene-                          phenotype plants associated with <br /> specific primers (Supplemental Figs. S5 S7;                            ICMT RNAi monitored cosegregation the <br /> Supplemental Table S1), indicating mutant                         mutant phenotype kanamycin-resistant selec- <br /> null. An RT-PCR reaction RNA template                          tion marker. To end, approximately 100 seedlings <br /> using AtICMTB-specific primers (Supplemental Figs. S5                      independent transgenic lines were <br /> and S7; Supplemental Table S1) showed muta-                       grown soil selection phenotypes <br /> tion effect expression (Supplemental Fig.                    scored. Plants allowed self, seeds <br /> S5). Cloning sequencing showed weak                           harvested, germinated kanamycin-containing <br /> band amplified AtICMTA-specific                            selection medium, ratios resistant <br /> primers AtICMTB. The AticmtA2/2 plants                          sensitive seedlings determined. In turn, <br /> visible phenotype including increased sensitivity                       plants transferred selection plates to <br /> ABA, indicating AtICMTA function redundant                         soil phenotypes scored again. In non- <br /> with ICMTB.                                                                segregating populations plants a <br />    Unfortunately, mutant compromised                               mutant phenotype 100% progeny kana- <br /> AtICMTB expression identified. To                         mycin resistant. The progeny plants a <br /> explore function isoprenyl Cys methylation                       mutant phenotype kanamycin sensitive. The <br /> plants generated ICMT RNAi plants (Aticmtsil).                          progeny plants heterozygote RNAi segre- <br /> These plants expressed 380-bp fragments                         gated 3 1 ratio respect selection <br /> ICMTA coding region reverse orientation. ICMTA                          marker. All kanamycin-resistant plants showed a <br /> and ICMTB approximately 80% identity                           mutant phenotype transferring soil. In con- <br /> nucleotide level (Supplemental Fig. S7), making                         trast, rescued kanamycin-sensitive plants were <br /> highly likely RNAi construct silence                        transferred soil did mutant phenotype. <br /> both ICMT genes. One independent                           The segregation analysis confirmed abnormal <br /> transgenic lines isolated, total 63                     phenotype Aticmtsil plants associated with <br /> (70%) similar abnormal phenotype (Fig. 7). Six                       AtICMT RNAi transgene. <br /> independent T1 lines selected analysis. <br />    Aticmtsil plants altered phyllotaxis (Fig. 7, A&#226;&#8364;&#8220;C,                  DISCUSSION <br /> E, F). Flowers developed region of <br /> the inflorescence stem minimal elongation                         In previous work (Bracha et al., 2002), we <br /> of internodes (Fig. 7, A&#226;&#8364;&#8220;C F). Occasionally,                       showed plant CaaX processing machinery is <br /> secondary axillary flowers developed flowers,                       evolutionarily conserved. The Arabidopsis CaaX pro- <br /> indicating partial conversion flowers inflores-                    teases AtSTE24 AtRCE1 complement corre- <br /> cence (Fig. 7C). Siliques developed internode                      sponding yeast mutants promoting a-factor processing <br /> elongation (Fig. 7F). Multiple buds developed                       (Fig. 1; Bracha et al., 2002). Proteolysis plant <br /> axil cauline leaves (Fig. 7D). All plants devel-                    prenylated proteins AtRCE1 AtSTE24 <br /> oped fasciated bifurcated stems (Fig. 7E). Interestingly,                  sufficient promote plasma membrane <br /> altered phyllotaxis, fasciated stems, development                      localization yeast cells (Fig. 2; Bracha et al., 2002). <br /> 124                                                                                                              Plant Physiol. Vol. 148, 2008 <br />                                                                                                        CaaX Processing Arabidopsis <br />  <br />  <br />                                                                          following prenylation, proteins targeted to <br />                                                                          ER undergo CaaX processing, in <br />                                                                          turn likely required subcellular targeting. <br />                                                                          Our expression, activity, mutant analyses suggests <br />                                                                          AtICMTB likely major prenyl-dependent <br />                                                                          carboxy methyltransferase Arabidopsis that <br />                                                                          AtICMTA function likely redundant (Figs. <br />                                                                          4&#226;&#8364;&#8220;7). The similarity Aticmtsil lines era1 plp <br />                                                                          suggests isoprenyl Cys methylation modulates <br />                                                                          function prenylated proteins. <br />  <br />                                                                          Evaluation CaaX Processing Activity Plant CaaX <br />                                                                          Proteases Yeast <br />                                                                             Both Ste24 Rce1 carry CaaX pro- <br />                                                                          cessing cleavage prenylated a-factor. The processed <br />                                                                          a-factor methylated turn cleaved again <br />                                                                          Ste24 internal N-terminal site by <br />                                                                          Ste23 yield mature secreted a-factor (Supple- <br />                                                                          mental Fig. S2; Boyartchuk Rine, 1998; Tam et al., <br />                                                                          1998). The Arabidopsis AtRCE1 homolog been <br />                                                                          previously identified characterized biochemically <br />                                                                          (Cadinanos et al., 2003). When expressed rce1D <br />                                                                          ste24D mutant cells, AtRCE1 able restore <br />                                                                          a-factor production (Fig. 1), demonstrating function- <br />                                                                          ality CaaX protease vivo yeast. The yeast <br />                                                                          Ste24 Rce1 enzymes known partially <br />                                                                          overlapping substrate specificity (Trueblood et al., <br />                                                                          2000). It known specificity pat- <br />                                                                          tern extends organisms. Our results indicate <br />                                                                          case. We substrate <br />                                                                          specificity AtRCE1 partially overlapping with <br />                                                                          AtSTE24 demonstrated ability to <br />                                                                          process Arabidopsis AtROP9 (Fig. 2C; Bracha <br />                                                                          et al., 2002) yeast a-factor. The halos that <br />                                                                          formed AtRCE1-complemented ste24D <br />                                                                          rce1D cells smaller halos formed around <br />                                                                          cells complemented AtSTE24. The <br />                                                                          differences halo sizes indicate a-factor processing <br />                                                                          AtRCE1 efficient AtSTE24, likely <br />                                                                          owing dual role AtSTE24 a-factor processing. <br />                                                                             The subcellular localization farnesylated GFP- <br />                                                                          AtROP9 wild-type, rce1D ste24D, rce1D ste24D <br />                                                                          cells complemented AtRCE1 revealed role for <br />                                                                          CaaX proteolysis targeting prenylated protein to <br />                                                                          plasma membrane. Consistently, fibroblasts <br /> Figure 6. Expression levels AtICMTA AtICMTB AtRCE1. A,        derived rce1 icmt knockout mice, farnesylated <br /> Q-PCR determined relative normalized expression levels            Ras accumulated endomembrane (Michaelson <br /> AtRCE1, AtICMTA, AtICMTB. ACTIN use internal control          et al., 2005). Likewise, geranylgeranylated petunia <br /> in Q-PCR. Average calculated using replicates. The error   (Petunia hybrida) calmodulin CaM53 (Rodriguez- <br /> bars represent SD. B, Microarray data taken Genevestigator          Concepcion et al., 1999; Caldelari et al., 2001) accu- <br /> (Zimmermann et al., 2004; Laule et al., 2006) expression <br />                                                                          mulated endomembranes following transient <br /> levels AtICMTA (red) AtICMTB (blue) development of <br /> Arabidopsis. <br />                                                                          expression Nicotiana benthamiana leaf epidermal <br />                                                                          cells treated carboxymethylation inhibitor <br />                                                                          AFC (Rodriguez-Concepcion et al., 2000). <br /> Both AtRCE1 AtSTE24 expressed most <br /> tissues (Fig. 6; Supplemental Fig. S4; Bracha et al.,                    Subcellular Targeting Prenylated Proteins <br /> 2002). All CaaX processing enzymes localized in <br /> the ER (Fig. 3), like homologs yeast                          Our results suggest following targeting pathway <br /> mammalian cells. These findings strongly suggest                         CaaX proteins plant cells. Nascent CaaX proteins <br /> Plant Physiol. Vol. 148, 2008                                                                                                   125 <br />  Bracha-Drori et al. <br />  <br />  <br /> are synthesized cytosol, encounter         activity yeast (Fig. 5) consistent hy- <br /> one protein prenyltransferases modify            pothesis residues C-terminal hydrophilic <br /> the CaaX Cys thioether linkage C-15 farnesyl         domains play critical role function ICMT <br /> or C-20 geranylgeranyl lipid. Prenylated CaaX proteins       (Romano Michaelis, 2001). <br /> have high affinity ER, encounter           Interestingly, querying plant sequence databases <br /> prenyl-CaaX-specific protease, AtSTE24             revealed small families ICMT proteins monocot <br /> AtRCE1 removes -aaX amino acids. The newly          dicot plant species. For example, identified <br /> exposed C-terminal isoprenyl Cys              putative ICMT proteins rice (Oryza sativa; <br /> substrate ER-localized ICMT methylesterifies        Supplemental Fig. S8) grape vine (Vitis vinifera; <br /> the a-carboxyl group isoprenyl Cys. The end           data shown). The sequence alignment ICMT <br /> product three-step posttranslational modifica-       protein Arabidopsis, poplar (Populus spp.), rice, <br /> tion process contains hydrophobic domain C          Physomitrella, yeast, human shows while <br /> terminus hydrophilic protein. It re-         AtICMTA contains Ser residue position 188, all <br /> mains established fully modified           ICMT proteins positively charged Lys or <br /> proteins plant cells arrive final location   Arg residues position (Supplemental Fig. S8). <br /> the secretory pathway different route.          As noted above, activity AticmtAmE165 single mu- <br />                                                              tant AticmtAmR111R112E165 triple mutant was <br /> The Function ICMTA ICMTB                              similar wild-type AtICMTA (data shown). Taken <br />                                                              together, data suggest positively charged <br />    AtICMTB/AtSTE14B lower KM higher                residue position 188 critical catalytic activity of <br /> catalytic activity compared AtICMTA/AtSTE14A           ICMT. In contrast, charged residues positions 111 <br /> carboxy methylation assays vitro. Expression analysis     112 conserved (Supplemental Fig. S8). The <br /> demonstrated AtICMTB/AtSTE14B widely            existence small ICMT protein families different <br /> expressed AtICMTA/AtSTE14. Based               plant species identity AtICMTA and <br /> observations, suggested AtICMTB/            poplar ICMT positions 111 (N) 165 (Q) and <br /> STE14B major prenyl-dependent carboxy methyl-         AtICMTA rice ICMT protein position <br /> transferase Arabidopsis (Narasimha Chary et al.,          112 (Y) indicate ICMT proteins have <br /> 2002). In extension work,            diverged early evolution higher plants. <br /> AtICMTB higher activity yeast. Analysis          The Q-PCR analysis showed AtICMTB ex- <br /> the sequence differences AtICMTA                 pressed significantly higher levels compared to <br /> AtICMTB revealed amino acids conserved          AtICMTA (Fig. 6), confirming earlier finding <br /> AtICMTB, yeast Ste14p, human, moss ICMT pro-             (Narasimha Chary et al., 2002). We extended <br /> teins differ ICMTA (Fig. 4). Our results suggest     expression analysis using microarray data avail- <br /> that amino acids responsible          able web (Zimmermann et al., 2004; Laule et al., <br /> different activities AtICMTA AtICMTB (Fig. 5).        2006). This analysis confirmed Q-PCR experiments <br />    Based topology model, Ste14p contains            demonstrated tissues, developmen- <br /> membrane spans, form helical hairpin.         tal stages, stimuli, expression of <br /> According model, Ste14p hydro-           AtICMTB higher (Fig. 6; Supplemental Fig. S4). <br /> philic regions located cytosol (Romano           The activity expression analysis strongly sug- <br /> Michaelis, 2001). Hydropathy analysis suggests          gested AtICMTB major isoprenyl car- <br /> the overall topology Ste14p preserved        boxymehtyltransferase Arabidopsis. The lack of <br /> members ICMT family (Romano Michaelis,            noticeable developmental abnormalities ABA hy- <br /> 2001). AtICMTB AtICMTA similar topology             persensitivity AtICMTA knockout mutant sup- <br /> and contain transmembrane he-           ports conclusion. In contrast, Aticmtsil plants had <br /> lices (Fig. 4B). In contrast differing N-terminal   distinguishable developmental alterations, implicat- <br /> topologies, ICMT proteins different organisms           ing likely function ICMTA ICMTB <br /> share similar C-terminal hydropathy profiles (Romano         genes compromised. While unique AtICMTA <br /> and Michaelis, 2001).                                        activities ruled stage, ques- <br />    Sequence analysis revealed amino    tionable essential function. <br /> acids unique AtICMTA (N111, Y112, Q165, S188) <br /> introduce neutral hydrophobic residues cyto-         The Phenotype Aticmtsil Plants <br /> solic spans, disrupt topology struc- <br /> ture AtICMTA. It noteworthy Q165 located       The fasciation bifurcation stem the <br /> the region close end transmembrane span        altered phyllotaxis Aticmtsil plants suggested <br /> and beginning cytosolic (Fig. 4B). The       organization shoot apical meristem was <br /> charged amino acid E187 AtICMTA represented            compromised. A similar phenotype observed <br /> by neutral residues Q187 N230 AtICMTB             era1-2 PFTB plp PFTA/PGGTA mutants <br /> and Ste14p, respectively. Our findings activity      (Yalovsky et al., 2000; Running et al., 2004). Thus, <br /> AtICMTA reduced compared AtICMTB              prenylation CaaX processing required for <br /> substituting residues AtICMTB-like restored      proper function shoot apical meristem. Interest- <br /> 126                                                                                          Plant Physiol. Vol. 148, 2008 <br />                                                                                             CaaX Processing Arabidopsis <br />  <br />  <br />                                                                          Figure 7. Phenotype ICMT RNAi plants. A C, <br />                                                                          Altered phylotaxis flower arrangement inflores- <br />                                                                          cence stems. D, Development multiple axillary <br />                                                                          buds. E, A fasciated inflorescence stem. F, Adjacent <br />                                                                          siliques stem tip results development of <br />                                                                          multiple buds. G, Wild-type Col-0 inflorescence <br />                                                                          stem. H, era1-2 inflorescence stem. Note altered <br />                                                                          phylotaxis (arrowheads). [See online article color <br />                                                                          version figure.] <br />  <br />  <br />  <br />  <br /> ingly, Aticmtsil plants ABA hypersensitive    T-DNA knockout plants 80% sequence iden- <br /> (data shown).                                          tity nucleotide level AtICMTA and <br />    The relatively mild phenotype Aticmtsil plants   AtICMTB support notion RNAi <br /> could result incomplete silencing        affected expression genes. <br /> AtICMTA AtICMTB expression. Alternatively,            It noteworthy relatively subtle phenotype <br /> mild phenotype partial function        Aticmtsil plants consistent findings the <br /> prenylated nonmethylated proteins. Although         function ICMT yeast mammals. Yeast rce1D <br /> was difficult observe consistent reduction RNA       ste24D ste14D mutants mate their <br /> levels AtICMTA AtICMTB RNAi lines,       growth rate normal. In contrast, ram1D (PFT <br /> presence mutant phenotype 70% T1          b-subunit mutant) cells display retarded growth rate <br /> lines strict cosegregation abnormal         malfunction Ras1p Ras2p cdc43D <br /> phenotype RNAi construct indicate        (PGGT-I b-subunit) mutant cells nonviable, primar- <br /> expression ICMT genes likely affected.     ily malfunction Cdc42p (Trueblood et al., <br /> The absence similar phenotypes icmtA2/2          1993). Thus, yeast prenylated nonmethylated Ras <br /> Plant Physiol. Vol. 148, 2008                                                                                              127 <br />  Bracha-Drori et al. <br />  <br />  <br /> and Cdc42 proteins functional a-factor                            Construction AtICMTA-RNAi Plasmid <br /> is processed secreted. Rce1 Icmt knockout                                   A fragment position 1 380 AtICMTA amplified with <br /> mice die embryogenesis fibroblasts ob-                                   oligonucleotide primers SYP304 SYP305. The resulting fragment was <br /> tained embryos maintained culture                              purified, digested NcoI XhoI, cloned pGFP-MRC generate <br /> (Michaelson et al., 2005), indicating cell division                            pSY302. An identical fragment amplified oligonucleotide primers <br />                                                                                     SYP306 SYP307, purified, digested XbaI SacI, cloned into <br /> was completely compromised. Future analysis will <br />                                                                                     pSY302 generate pSY303. Cloning fragments were <br /> be required determine plants                                     flanking GFP reverse orientation another. Next, pSY303 digested <br /> prenylated nonmethylated proteins functional.                             HindIII isolate cassette containing 35S promoter, reverse-oriented <br />    The importance a-carboxyl methylation                                      AtICMTA fragments flanking GFP, NOS transcriptional terminator. This <br /> prenylated proteins illustrated fact                            cassette subcloned pCAMBIA2300 generate pSY304. <br /> is final potentially reversible step dur- <br /> ing prenylation CaaX processing. Consequently,                                  PCR <br /> the targeting function prenylated plant proteins                                 PCR used gene amplification, gene detection, cloning. Unless <br /> may modulated methylation/demethylation                                   mentioned, reaction conditions follows: 5 100 ng DNA, 0.05 mM <br /> status proteins. Consistent hypothesis,                            dNTPs, 7.5 mM primer, 13 reaction buffer (10 mM Tris, pH 8.8, 50 mM <br /> specific prenyl Cys methyl ester hydrolase activities                               KCl, 0.08% Nonidet P-40), 1.5 mM MgCl2, 0.1 mg/mL bovine serum albumin, <br />                                                                                     0.125 0.25 mL Taq DNA polymerase Pfu DNA polymerase <br /> have described mammals recently                                    (Promega) added hot-start fashion following annealing step <br /> also plants (Perez-Sala et al., 1991; Tan Rando,                             cycle template final volume 25 mL. The reaction conditions <br /> 1992; Valentijn Jamieson, 1998; Van Dessel et al.,                              were: 2 min 94&#194;&#176;C, followed 30 repeats 30 s 94&#194;&#176;C, 30 s 55&#194;&#176;C, 1 min <br /> 2001; Deem et al., 2006).                                                           1 kb 72&#194;&#176;C, additional step 3 min 72&#194;&#176;C. <br />  <br />  <br /> MATERIALS AND METHODS                                                               Yeast Work <br />                                                                                        All yeast strains used study listed Supplemental Table S3. <br /> Molecular Cloning <br />                                                                                     Yeast transformation carried modified lithium acetate trans- <br />    All plasmids used study listed Supplemental Table S2.             formation protocol (Ito et al., 1983; Yalovsky et al., 1997). <br /> Plasmids generated using standard protocols. Oligonucleotide primers <br /> used PCR reaction generate appropriate restriction sites, site-directed <br /> mutagenesis, sequencing listed Supplemental Table S1. All frag-          Subcellular Localization GFP-AtROP/AtRAC7 Yeast <br /> ments construct sequenced verify PCR-generated errors <br />                                                                                         AtROP9 Atrop9mS subcloned yeast expression vector <br /> were introduced relevant fragments cloned correct frame. <br />                                                                                     pJR1138-GFP obtain plasmids pSY32 pSY31, respectively (Supplemen- <br /> Escherichia coli DH5a used recombinant plasmid constructions. <br />                                                                                     tal Table S2; Lavy et al., 2002). Yeast strains JRY6959, JRY6958, SYY500 <br />    Plasmid production routine DNA manipulations sequencing was <br />                                                                                     (Supplemental Table S3) transformed resulting plasmids to <br /> prepared QIAGENE mini prep purification kit according manu- <br />                                                                                     obtain yeast strains SYY535, SYY538, SYY539, SYY542. Subcellular local- <br /> facturer&#226;&#8364;&#8482;s protocol. <br />                                                                                     ization GFP fusion proteins determined using confocal laser- <br />    Yeast (Saccharomyces cerevisiae) expression vectors generated by <br />                                                                                     scanning microscope following overnight growth cells 30&#194;&#176;C rich <br /> cloning corresponding cDNAs polycloning sites high-copy <br />                                                                                     media plates. To prevent cell motion obtain sharp images, cells were <br /> (2m) low-copy (CEN) yeast vectors. The yeast vectors contain glyceral- <br />                                                                                     loaded slides 0.1% low melting agarose. <br /> dehyde-3-P dehydrogenase gene promoter phosphoglycerate kinase <br /> gene terminator, separated restriction sites. <br />    Plant expression vectors generated cloning SphI, HindIII, NotI <br /> cassette, containing cauliflower mosaic virus 35S promoter, gene interest, <br />                                                                                     Growth Arrest Pheromone Diffusion (Halo) Assays <br /> and NOS 3# end, plant binary vector pCAMBIA2300 (SphI, HindIII).               Assays carried essentially described previously (Trueblood <br />                                                                                     et al., 2000), following modifications. Three microliters MATa cell <br /> Fusion Protein C Terminus                                                    slurry (approximately 107 cells) spotted solid, rich medium (yeast <br />                                                                                     peptone dextrose) plate containing 0.01% Triton X-100 spread <br />    A fragment containing full-<span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span> cDNA sequence gene interest,       lawn (approximately 2 3 106 cells) MATa sst2a cells (JRY3443). <br /> without ATG stop codon, cloned frame C               After 1 d growth 28&#194;&#176;C, relative amounts a-factor produced each <br /> terminus plasmids containing GFP, CFP, YFP.                                  MATa strain evident size growth inhibition zone (halo) <br />                                                                                     surrounding MATa cells. The relative halo sizes directly proportional <br /> Fusion Protein N Terminus                                                    a-factor exported MATa cells (Ito et al., 1983). <br />                                                                                     Experiments repeated times replicates. <br />    A fragment containing full-length cDNA sequence gene interest, <br /> with ATG stop codon, cloned frame the <br /> N terminus plasmids containing GFP, CFP, YFP.                                Plant Material <br />                                                                                        Nicotiana benthamiana Arabidopsis (Arabidopsis thaliana; Columbia-0 and <br /> Generation AticmtAmR111R112E165Q187R188 Mutant                                   Wassilewskija) plants grown described previously (Bracha et al., 2002; <br />                                                                                     Lavy et al., 2002). <br />     pSY60 used template primers SYP50 SYP51 to <br /> mutate AtICMTA Q165 E165 using QuikChange site-directed mutagenesis <br /> kit (Stratagene) create pSY915. AticmtAmE165 sequenced verify        Transient Expression Assays <br /> no PCR-generated errors introduced. pSY915 used template <br /> together primers SYP54 SYP55 mutate AticmtAmE165 N111 Y112             Young leaves N. benthamiana injected abaxial with <br /> to R111 R112, respectively, using QuikChange (Stratagene) create           Agrobacterium tumefaciens GV3101/pMP90 strains harboring appropriate plas- <br /> pSY902. pSY902 used template primers SYP52             mids described previously (Bracha et al., 2002; Lavy et al., 2002). ER <br /> SYP53 mutate AticmtAmE165Q187R188 E187 S188 Q187 R188 using         localization determined coexpression ER-mCherry, com- <br /> QuikChange (Stratagene) create pSY904. AticmtAmR111R112E165Q187R188          posed N-terminal signal peptide Arabidopsis WALL-ASSOCIATED <br /> sequenced verify PCR-generated errors introduced.                   KINASE2 C-terminal ER-retention signal (His-Asp-Glu-Leu; Nelson <br />  <br /> 128                                                                                                                           Plant Physiol. Vol. 148, 2008 <br />                                                                                                                            CaaX Processing Arabidopsis <br />  <br />  <br /> et al., 2007), GFP-AtSTE24 (Bracha et al., 2002). Leaves observed    Confocal Imaging <br /> for GFP mCherry fluorescence 24 48 h postinjection. <br />                                                                                      Imaging carried Leica TCS-SL confocal laser-scanning micro- <br /> Arabidopsis Transformation                                                        scope using 203 multi-immersion 633 water-immersion objectives with <br />                                                                                   numerical apertures 0.7 1.2, respectively. GFP excitation carried <br />    Stable Arabidopsis transgenic plants obtained using floral-dip        argon laser 488 nm 500 nm beam splitter and <br /> method (Clough Bent, 1998). Transgenic plants selected         spectral detector set 510 535 nm. mCherry excited argon <br /> growth kanamycin containing 0.53 Murashige Skoog medium bar             laser 514 nm 458/514 dichroic beam splitter spectral <br /> spraying soil-grown plants BASTA. AtICMTA plants analyzed          detector set 580 630. Dual imaging GFP mCherry was <br /> by PCR reveal transgene existence.                                             carried sequential mode avoid channel bleeding. The reticulate ER <br />                                                                                   images single confocal sections colocalization images are <br /> Plant DNA RNA Isolation Q-PCR                                             maximal projection stacks multiple confocal sections. Image processing was <br />                                                                                   carried Leica LCS, Zeiss LSM browser, Adobe Photoshop. <br /> DNA Isolation <br />    Leaves ground mortar pestle 1 2 g used         ABA Germination Assays <br /> DNA isolation using GenElute plant genomic DNA miniprep kit (Sigma) <br /> according manufacturer&#226;&#8364;&#8482;s protocol.                                            ABA germination assays carried described previously <br />                                                                                   (Running et al., 2004). <br /> RNA Isolation                                                                         Sequence data article GenBank/EMBL data <br />     Leaves ground mortar pestle 100 200 mg used       libraries accession numbers At2g36305, At5g23320, At5g08335, <br /> for total RNA isolation using SV total RNA isolation kit (Promega) according      A9P_B59POPTR, Os04g0602900, Q01SH5_ORYSA, OSI016666, OsJ_015369, <br /> to manufacturer&#226;&#8364;&#8482;s protocol.                                                   A9SSC9_PHYPA. <br />  <br />  <br /> Q-PCR                                                                             Supplemental Data <br />     Total RNA isolated leaves, flowers, seedlings, roots using SV       The following materials available online version article. <br /> total isolation RNA kit (Promega). For RT, 1 mg total RNA denatured at <br />                                                                                      Supplemental Figure S1. Prenylation CaaX processing proteins. <br /> 70&#194;&#176;C 5 min presence 0.5 mg oligo(dT)15 primers. The tubes were <br /> immediately chilled ice reverse transcribed 200 units Moloney         Supplemental Figure S2. Processing a-mating factor precursor and <br /> murine leukemia virus reverse transcriptase according manufacturer&#226;&#8364;&#8482;s            pheromone diffusion halo assay. <br /> instructions (Promega) total volume 25 mL 42&#194;&#176;C 60 min. <br />                                                                                      Supplemental Figure S3. Colocalization AtSTE24, AtICMTA, AtRCE1, <br /> Quantification AtRCE1, AtICMTA, AtICMTB RNA levels carried <br />                                                                                        AtICMTB ER following transient expression N. benthami- <br /> out Q-PCR using ABI Prism 7700 StepOnePlus instrument (Applied <br />                                                                                        ana leaf epidermal cells. <br /> Biosystems). Study samples run triplicate eight-well optical PCR <br /> strips (Applied Biosystems) final volume 10 mL. Primers designed        Supplemental Figure S4. Expression <span id='am-316' about='xsp:pattern' typeof='owl:Thing'>pattern</span> AtICMTA AtICMTB. <br /> according Roche Universal ProbeLibrary (https://www.roche-applied- <br /> science.com/sis/rtpcr/upl/index.jsp). The following primers pairs used:         Supplemental Figure S5. Identification analysis AtICMTA <br /> AtRCE1, 500 nmol L21 SYP1008 1 SYP1009 amplify 106-bp frag-               T-DNA insertion mutant. <br /> ment positions 265 371; AtICMTA, 125 nmol L21 SYP1006 1              Supplemental Figure S6. Nucleotide sequence AtICMTA highlighting <br /> SYP1007 amplify 59-bp fragment positions 252 311; AtICMTB,            insertion site T-DNA. <br /> 500 nmol L21 SYP1004 1 SYP 1005 amplify 120-bp fragment from <br /> positions 179 299. The PCR cycles run follows: 10 min initial             Supplemental Figure S7. Nucleotide sequence alignment AtICMTA <br /> denaturation 95&#194;&#176;C, followed 40 subsequent cycles 15 s denaturation         AtICMTB. <br /> 95&#194;&#176;C followed 1 min primer annealing elongation 60&#194;&#176;C. The                  Supplemental Figure S8. Amino acid sequence analysis plant ICMT <br /> specificity unique amplification product determined melting              proteins. <br /> curve analysis according manufacturer&#226;&#8364;&#8482;s instructions (Applied Biosys- <br /> tems). Relative quantities RNA calculated standard curve              Supplemental Table S1. Oligonucleotide primers used study. <br /> method (Applied Biosystems Incorporated 2001 User Bulletin #2: ABI PRISM <br />                                                                                      Supplemental Table S2. Plasmids used study. <br /> 7700 sequence detection [http://www.appliedbiosystems.com]). <br /> DNA dilution series prepared calculate amplification efficiency co-          Supplemental Table S3. Yeast strains used study. <br /> efficient gene. The relative levels AtRCE1, AtICMTA, AtICMTB <br /> RNA calculated according amplification efficiency coefficient and <br /> normalized ACTIN8 gene standard, level taken 1. The <br />                                                                                   ACKNOWLEDGMENTS <br /> analysis repeated independent biological replicates. <br />                                                                                       We thank Dr. Nir Ohad access Q-PCR machine, Dr. Tali Yahalom <br /> Identification Analysis AtICMTA T-DNA Mutant                               technical advice, members Yalovsky lab help during <br />                                                                                   project. <br />    DNA pulls University Wisconsin Biotechnology Center T-DNA <br /> mutant collection screened oligonucleotide primers SYP300           Received April 3, 2008; accepted July 16, 2008; published July 18, 2008. <br /> JL202 SYP301 JL202. Southern blot amplified PCR fragments with <br /> an AtICMTA gene-specific probe used verify identification potential <br /> T-DNA insertion AtICMTA. In turn, relevant seed stock, P65H9, was <br /> ordered T-DNA insertion site determined sequencing.                LITERATURE CITED <br /> Expression level AtICMTA AtICMTB mutant homozygote plants <br /> was determined RT-PCR nucleotide primers SYP304 SYP305,           Allen GJ, Murata Y, Chu SP, Nafisi M, Schroeder JI (2002) Hypersensi- <br /> SYP306 SYP307, respectively.                                                     tivity abscisic acid-induced cytosolic calcium increases Arabi- <br />                                                                                      dopsis farnesyltransferase mutant era1-2. Plant Cell 14: 1649&#226;&#8364;&#8220;1662 <br /> Plant Imaging Confocal Laser                                                  Backlund PS Jr (1997) Post-translational processing RhoA: carboxyl <br /> Fluorescence Imaging                                                                 methylation carboxyl-terminal prenylcysteine increases half- <br />                                                                                      life Rhoa. J Biol Chem 272: 33175&#226;&#8364;&#8220;33180 <br />    Images plants obtained Nikon Coolpix CCD camera           Bergo MO, Ambroziak P, Gregory C, George A, Otto JC, Kim E, Nagase H, <br /> Zeiss Axiocam CCD camera mounted Zeiss SV11 stereomicroscope.                   Casey PJ, Balmain A, Young SG (2002) Absence CAAX endo- <br />  <br /> Plant Physiol. Vol. 148, 2008                                                                                                                                129 <br />  Bracha-Drori et al. <br />  <br />  <br />     protease Rce1: effects cell growth transformation. Mol Cell Biol            (2006) Web-based analysis mouse transcriptome using Geneves- <br />     22: 171&#226;&#8364;&#8220;181                                                                        tigator. BMC Bioinformatics 7: 311 <br /> Bergo MO, Gavino B, Ross J, Schmidt WK, Hong C, Kendall LV, Mohr A,                 Lavy M, Bracha-Drori K, Sternberg H, Yalovsky S (2002) Cell-specific <br />     Meta M, Genant H, Jiang Y et al (2002) Zmpste24 deficiency mice                 prenylation independent mechanism regulates targeting type-II <br />     causes spontaneous bone fractures, muscle weakness, prelamin A               RACs. Plant Cell 14: 2431&#226;&#8364;&#8220;2450 <br />     processing defect. Proc Natl Acad Sci USA 99: 13049&#226;&#8364;&#8220;13054                       Maurer-Stroh S, Washietl S, Eisenhaber F (2003) Protein prenyltransferases. <br /> Bergo MO, Gavino BJ, Hong C, Beigneux AP, McMahon M, Casey PJ,                         Genome Biol 4: 212 <br />     Young SG (2004) Inactivation Icmt inhibits transformation onco-           Michaelson D, Ali W, Chiu VK, Bergo M, Silletti J, Wright L, Young SG, <br />     genic K-Ras B-Raf. J Clin Invest 113: 539&#226;&#8364;&#8220;550                                  Philips M (2005) Postprenylation CAAX processing required <br /> Bergo MO, Leung GK, Ambroziak P, Otto JC, Casey PJ, Gomes AQ,                          proper localization Ras Rho GTPases. Mol Biol Cell 16: <br />     Seabra MC, Young SG (2001) Isoprenylcysteine carboxyl methyltrans-                 1606&#226;&#8364;&#8220;1616 <br />     ferase deficiency mice. J Biol Chem 276: 5841&#226;&#8364;&#8220;5845                           Narasimha Chary S, Bultema RL, Packard CE, Crowell DN (2002) <br /> Bonetta D, Bayliss P, Sun S, Sage T, McCourt P (2000) Farnesylation                 Prenylcysteine alpha-carboxyl methyltransferase expression func- <br />     involved meristem organization Arabidopsis. Planta 211: 182&#226;&#8364;&#8220;190              tion Arabidopsis thaliana. Plant J 32: 735&#226;&#8364;&#8220;747 <br /> Boyartchuk VL, Ashby MN, Rine J (1997) Modulation Ras a-factor               Nelson BK, Cai X, Nebenfuhr A (2007) A multicolored set vivo <br />     function carboxyl-terminal proteolysis. Science 275: 1796&#226;&#8364;&#8220;1800                  organelle markers co-localization studies Arabidopsis other <br /> Boyartchuk VL, Rine J (1998) Roles prenyl protein proteases matu-                plants. Plant J 51: 1126&#226;&#8364;&#8220;1136 <br />     ration Saccharomyces cerevisiae a-factor. Genetics 150: 95&#226;&#8364;&#8220;101               Parish CA, Smrcka AV, Rando RR (1995) Functional significance beta <br /> Bracha K, Lavy M, Yalovsky S (2002) The Arabidopsis AtSTE24 CaaX                  gamma-subunit carboxymethylation activation phospholipase <br />     protease broad substrate specificity. J Biol Chem 277: 29856&#226;&#8364;&#8220;29864            C phosphoinositide 3-kinase. Biochemistry 34: 7722&#226;&#8364;&#8220;7727 <br /> Cadinanos J, Varela I, Mandel DA, Schmidt WK, Diaz-Perales A, Lopez-                Pei ZM, Ghassemian M, Kwak CM, McCourt P, Schroeder JI (1998) Role of <br />     Otin C, Freije JM (2003) AtFACE-2, functional prenylated protein pro-            farnesyltransferase ABA regulation guard cell anion channels and <br />     tease Arabidopsis thaliana related mammalian Ras-converting                plant water loss. Science 282: 287&#226;&#8364;&#8220;290 <br />     enzymes. J Biol Chem 278: 42091&#226;&#8364;&#8220;42097                                           Pendas AM, Zhou Z, Cadinanos J, Freije JM, Wang J, Hultenby K, <br /> Caldelari D, Sternberg H, Rodriguez-Concepcion M, Gruissem W,                          Astudillo A, Wernerson A, Rodriguez F, Tryggvason K, et al (2002) <br />     Yalovsky S (2001) Efficient prenylation plant geranylgeranyl-                 Defective prelamin A processing muscular adipocyte alter- <br />     transferase-I requires functional CaaL box motif proximal                  ations Zmpste24 metalloproteinase-deficient mice. Nat Genet 31: 94&#226;&#8364;&#8220;99 <br />     polybasic domain. Plant Physiol 126: 1416&#226;&#8364;&#8220;1429                                  Perez-Sala D, Tan EW, Canada FJ, Rando RR (1991) Methylation and <br /> Clarke S (1992) Protein isoprenylation methylation carboxyl-                    demethylation reactions guanine nucleotide-binding proteins of <br />     terminal cysteine residues. Annu Rev Biochem 61: 355&#226;&#8364;&#8220;386                           retinal rod outer segments. Proc Natl Acad Sci USA 88: 3043&#226;&#8364;&#8220;3046 <br /> Clough SJ, Bent AF (1998) Floral dip: simplified method                       Rodriguez-Concepcion M, Toledo-Ortiz G, Yalovsky S, Caldelari D, <br />     Agrobacterium-mediated transformation Arabidopsis thaliana.                     Gruissem W (2000) Carboxyl-methylation prenylated calmodulin <br />     Plant J 16: 735&#226;&#8364;&#8220;743                                                                CaM53 required efficient plasma membrane targeting the <br /> Crowell DN, Kennedy M (2001) Identification functional expression               protein. Plant J 24: 775&#226;&#8364;&#8220;784 <br />     yeast prenylcysteine alpha-carboxyl methyltransferase gene            Rodriguez-Concepcion M, Yalovsky S, Zik M, Fromm H, Gruissem W <br />     Arabidopsis thaliana. Plant Mol Biol 45: 469&#226;&#8364;&#8220;476                                   (1999) The prenylation status novel plant calmodulin directs plasma <br /> Crowell DN, Sen SE, Randall SK (1998) Prenylcysteine alpha-carboxyl                    membrane nuclear localization protein. EMBO J 18: 1996&#226;&#8364;&#8220;2007 <br />     methyltransferse suspension-cultured tobacco cells. Plant Physiol            Romano JD, Michaelis S (2001) Topological mutational analysis of <br />     118: 115&#226;&#8364;&#8220;123                                                                       Saccharomyces cerevisiae Ste14p, founding member isoprenyl- <br /> Cutler S, Ghassemian M, Bonetta D, Cooney S, McCourt P (1996) A                        cysteine carboxyl methyltransferase family. Mol Biol Cell 12: 1957&#226;&#8364;&#8220;1971 <br />     protein farnesyl transferase involved abscisic acid signal transduc-         Romano JD, Schmidt WK, Michaelis S (1998) The Saccharomyces cerevisiae <br />     tion Arabidopsis. Science 273: 1239&#226;&#8364;&#8220;1241                                        prenylcysteine carboxyl methyltransferase Ste14p endoplasmic <br /> Dai Q, Choy E, Chiu V, Romano J, Slivka SR, Steitz SA, Michaelis S,                    reticulum membrane. Mol Biol Cell 9: 2231&#226;&#8364;&#8220;2247 <br />     Philips MR (1998) Mammalian prenylcysteine carboxyl methyltrans-                Rosenberg SJ, Rane MJ, Dean WL, Corpier CL, Hoffman JL, McLeish KR <br />     ferase endoplasmic reticulum. J Biol Chem 273: 15030&#226;&#8364;&#8220;15034               (1998) Effect gamma subunit carboxyl methylation interaction <br /> Deem AK, Bultema RL, Crowell DN (2006) Prenylcysteine methylesterase                   G protein alpha subunits b <br /> </body></html>