<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Research <br />  <br /> Natural selection cis trans regulation yeasts <br /> J.J. Emerson,1,10 Li-Ching Hsieh,1,2,10 Huang-Mo Sung,3,10 Tzi-Yuan Wang,1,4,10 <br /> Chih-Jen Huang,4,5,6 Henry Horng-Shing Lu,7 Mei-Yeh Jade Lu,1,4 Shu-Hsing Wu,8 <br /> and Wen-Hsiung Li1,4,9,11 <br /> 1 <br />   Genomics Research Center, Academia Sinica, Taipei 115, Taiwan; 2Institute Information Science, Academia Sinica, Taipei 115, <br /> Taiwan; 3Department Life Sciences, National Cheng Kung University, Tainan 701, Taiwan; 4Biodiversity Research Center, Academia <br /> Sinica, Taipei 115, Taiwan; 5Molecular Biological Agricultural Sciences Program, Taiwan International Graduate Program, <br /> Academia Sinica, Taipei 115, Taiwan; 6Graduate Institute Biotechnology, National Chung Hsing University, Taichung 402, Taiwan; <br /> 7 <br />   Institute Statistics, National Chiao Tung University, Hsinchu 30010, Taiwan; 8Institute Plant Microbial Biology, Academia <br /> Sinica, Taipei 115, Taiwan; 9Department Ecology Evolution, University Chicago, Chicago, Illinois 60637, USA <br />  <br />  <br />        Gene expression regulated cis elements, DNA segments closely linked genes regulate, and <br />        trans factors, usually proteins capable diffusing unlinked genes. Understanding patterns sources <br />        regulatory variation crucial understanding phenotypic genome evolution. Here, measure genome-wide <br />        allele-specific expression deep sequencing investigate patterns cis trans expression variation two <br />        strains Saccharomyces cerevisiae. We propose statistical modeling framework based binomial distribution that <br />        simultaneously addresses normalization read counts derived different parents estimating cis trans <br />        expression variation parameters. We expression polymorphism yeast common cis trans, though <br />        trans variation common. Constraint expression evolution correlated hallmarks constraint, <br />        including gene essentiality, number protein interaction partners, constraint amino acid substitution, indicating <br />        cis trans polymorphism clearly purifying selection, trans variation appears more <br />        sensitive selective constraint. Comparing interspecific expression divergence S. cerevisiae S. paradoxus our <br />        intraspecific variation suggests significant departure neutral model molecular evolution. A examination <br />        correlation polymorphism divergence category suggests cis divergence frequently <br />        mediated positive Darwinian selection trans divergence. <br />        [Supplemental material available online http://www.genome.org. The sequencing data study been <br />        submitted NCBI Gene Expression Omnibus (http://www.ncbi.nlm.nih.gov/geo) accession no. GSE20749.] <br />  <br /> Untangling genetic basis phenotypic variation              ation different strains S. cerevisiae (Kvitek et al. <br /> between species central topic evolutionary biology. It long      2008), concluding expression polymorphism the <br /> been argued evolution protein sequences insufficient           influence diversifying selection adaptation different envi- <br /> explain morphological diversity present nature (King              ronments. It argued cis expression level polymor- <br /> Wilson 1975). Consequently, evolution gene expression           phism purifying selection, trans expression poly- <br /> been invoked alternative explanation phenotypic inno-              morphism positive selection (Ronald Akey 2007). <br /> vation (Halder et al. 1995; Carroll 2008). As result, effort      Studies Drosophila took advantage comparing allele-specific <br /> been devoted understanding expression evolution eukaryotes,            expression (ASE) patterns parental strains their <br /> especially model systems like budding yeast Saccharomyces cer-           hybrid offspring (i.e., F1 hybrids) investigate cis trans expres- <br /> evisiae (Rockman Kruglyak 2006). Previous genome-wide studies            sion evolution (Wittkopp et al. 2004, 2008). This experimental de- <br /> of expression variation yeast taken advantage full-genome         sign measures combined expression variation (both cis trans <br /> microarrays determine linkage relationship genes              effects) locus measurement expression differences <br /> with variable expression causative mutations. Generally,       parental strains. The combined effects categories <br /> assumed sets genes exhibiting local linkage QTL maps          genetic variation influencing gene regulation explain expres- <br /> enriched cis variation distant linkage regu-        sion differences measured genetically distinct strains. <br /> lated trans. These QTL studies demonstrated gene regulation          However, expression differences measured F1 hybrids be- <br /> polymorphisms yeast common dominated distant               tween strains longer attributed trans- <br /> linkages (Brem et al. 2002; Yvert et al. 2003). Furthermore,          factors, genomes share cell trans- <br /> shown transcripts linked &#226;&#8364;&#732;&#226;&#8364;&#732;hotspot&#226;&#8364;&#8482;&#226;&#8364;&#8482; reg-        factors. Consequently, hybrid experiment measures cis <br /> ulators. For example, Yvert et al. (2003) reported 1265 variable tran-       variation. These experiments Drosophila 78 genes showed that <br /> scripts regulated 13 distant QTLs. Another study variation        cis differences dominate species species <br /> in transcript levels corroborated ubiquity transcript vari-       (Wittkopp et al. 2008). ASE polymorphism studies yeast have <br />                                                                              expression level variation usually numerically domi- <br /> 10 <br />   These authors contributed equally work.                            nated trans variants (Wang et al. 2007; Sung et al. 2009), even <br /> 11 <br />   Corresponding author.                                                      single-input module genes chosen minimize im- <br /> E-mail whli@uchicago.edu; fax (773) 702-9740. <br /> Article published online print. Article publication date   pact trans variation (Wang et al. 2007). Additionally, yeast only <br /> http://www.genome.org/cgi/doi/10.1101/gr.101576.109.                         52%&#226;&#8364;&#8220;78% expression QTLs mapped region gene <br />  <br />  <br /> 826     Genome Research                                20:826&#226;&#8364;&#8220;836 &#195;&#8220; 2010 Cold Spring Harbor Laboratory Press; ISSN 1088-9051/10; www.genome.org <br />            www.genome.org <br />                                                                                          Selection cis trans regulation yeasts <br />  <br />  <br /> they regulate confirmed cis (Ronald et al. 2005), implying      Intraspecific genomic DNA sequencing <br /> that 22%&#226;&#8364;&#8220;48% genes local linkages be <br />                                                                            The cell density ratio strains co-culture ex- <br /> regulated trans. Indeed, Ronald et al. (2005) reported specific <br />                                                                            periment (dCo) need 1, introduces systematic bias in <br /> instances trans regulation stemming local linkage. In these <br />                                                                            mRNA transcript counts strains. <span id='am-3' about='protege:TO' typeof='owl:Thing'>To</span> correct this <br /> ASE studies, predominance trans linkage generally consis- <br />                                                                            bias, sequenced genomic DNA (gDNA) samples <br /> tent results QTL mapping (Brem et al. 2002; Yvert <br />                                                                            provided mRNA estimate dCo (co-culture) dHy <br /> et al. 2003), demonstrated majority linkages <br />                                                                            (F1 hybrid). We estimated dCo = 1.30 that, expected, dHy = <br /> are distant likely driven &#226;&#8364;&#732;&#226;&#8364;&#732;hotspot&#226;&#8364;&#8482;&#226;&#8364;&#8482; <br />                                                                            1.00. That dHy indistinguishable 1, expected, indicates that <br /> trans-activating factors regulate thousands transcripts spread <br />                                                                            method effective way estimate d (Supplemental Fig. S1). <br /> throughout genome. In contrast, recent genome-wide com- <br />                                                                            We used ratios estimation allelic expression ratios in <br /> parison cis trans variation S. cerevisiae S. paradoxus <br />                                                                            terms RM/BY (eCo eHy), accounting effect cell density <br /> has shown cis variation common trans variation, <br />                                                                            (see Methods). We estimated means confidence in- <br /> though intraspecific data reported (Tirosh et al. 2009). <br />                                                                            tervals cis trans contributions expression change terms <br />       Counting sequencing reads long used measure the <br />                                                                            deviations null hypotheses: log2(eHy) = log2(eCis) and <br /> relative copy numbers sequences. Bailey et al. (2002) used <br />                                                                            log2(eCo) = log2(eCis) + log2(eTrans) (see Methods). Our results agree <br /> an increase read counts identify increases copy number due <br />                                                                            closely data set 227 genes collected study using <br /> to segmental duplication. More recently, Seoighe et al. (2006) used <br />                                                                            pyrosequencing (Fig. 1; Sung et al. 2009). The correlation between <br /> representation EST libraries identify ASE indicative both <br />                                                                            genes ranges 0.74 0.81, regression lines esti- <br /> genomic imprinting genetically caused allelic differences. With <br />                                                                            mated comparisons indistinguishable di- <br /> the advent high-throughput sequencing technologies like 454 <br />                                                                            agonal running origin slope equal 1. <br /> Life Sciences (Roche) sequencing, perspective more <br /> quantitative (Springer Stupar 2007). The application even <br /> higher capacity deep sequencing technologies quantitative mea-          Intraspecific transcriptome sequencing <br /> surement nucleotide frequencies obvious benefits mea-           expression estimation <br /> suring transcriptomes (Nagalakshmi et al. 2008). It enabled       Among 4442 genes passed bioinformatics filters, 4282 <br /> more accurate measurements allelic differences genetically           sequence reads alleles experiments (Sup- <br /> variable nucleotide pools single nucleotide polymorphism (SNP)         plemental Tables S1&#226;&#8364;&#8220;S3). <span id='am-1' about='protege:FROM' typeof='owl:Thing'>From</span> 12 channels IGA sequencing of <br /> discovery (Van Tassell et al. 2008) ASE (Wang et al. 2008; Bloom       cDNA samples (24 channels total), we <br /> et al. 2009).                                                              mapped 1.202 1.188 million sequence reads hybrid <br />       Here, investigate relative contributions cis trans     sample BY RM SNPs, respectively; co-culture sample <br /> regulatory differences overall expression variation entire      mapped 1.096 1.330 million reads. <br /> genome S. cerevisiae, using Illumina Genome Analyzer (IGA) se-                In total, 1180 genes (28%) exhibit expression polymorphism <br /> quencing mRNA measure genome-wide ASE co-culture                P-value threshold 1% (false discovery rate [FDR] &lt; 5%; <br /> experiment composed strains S. cerevisiae (denoted BY         Supplemental Fig. S2; correlated estimates, Methods). Next, we <br /> and RM) F1 hybrid. We report relative impact           classified genes examining relationship cis and <br /> selective constraint various modes gene regulation            trans polymorphisms genome (Fig. 2D; see <br /> species genome time, demonstrating          Methods section Independent Estimates). The data indicate that <br /> that large proportion genes exhibit expression polymorphism,          trans variation frequent, observed previously (Brem et al. <br /> with trans variation dominating cis, removing genes        2002; <br />                                                                                    Yvert  et  al. 2003),    greater magnitudes change <br /> influenced trans &#226;&#8364;&#732;&#226;&#8364;&#732;hotspots&#226;&#8364;&#8482;&#226;&#8364;&#8482; identified Yvert et al. (2003).        &#195;&#176; log2 &#195;&#176;etrans &#195;&#382; &gt; log2 &#195;&#176;ecis &#195;&#382;  (Wilcoxon rank sum test, P-values &lt; <br /> Furthermore, date, genome-wide comparison cis trans            0.01) cis variation. Nearly times genes exhibit <br /> variation species reported. In       trans change genes cis change (123 vs. 33). <br /> study, compare polymorphism data divergence data             For 178 genes significant |log2(ecis)| 6&#194;&#188; |log2(etrans)|, 79% of <br /> from recent study (Tirosh et al. 2009), showing cis        (140) genes |log2(ecis)| &lt; |log2(etrans)|. Interestingly, <br /> differences common species species,           116 genes unambiguous expression variation cis trans <br /> but trans variation compatible neutral       variation simultaneously (Fig. 2C,D, &#226;&#8364;&#732;&#226;&#8364;&#732;dominant&#226;&#8364;&#8482;&#226;&#8364;&#8482; &#226;&#8364;&#732;&#226;&#8364;&#732;both&#226;&#8364;&#8482;&#226;&#8364;&#8482; cat- <br /> model selection, cis variations appear driven           egories; use &#226;&#8364;&#732;&#226;&#8364;&#732;dominant&#226;&#8364;&#8482;&#226;&#8364;&#8482; confused with <br /> fixation positive Darwinian selection.                             genetic dominance; merely indicates variation type <br />                                                                            regulation greater other), 589 genes <br />                                                                            clear evidence variation category ambiguous <br /> Results                                                                    evidence (Fig. 2A,C,D, &#226;&#8364;&#732;&#226;&#8364;&#732;major&#226;&#8364;&#8482;&#226;&#8364;&#8482; categories). <br />                                                                            Thus, trans differences clearly dominate, influencing 64% (558/ <br /> Orthology SNP identification                                           863) differentially expressed genes, half genes <br /> We designed bioinformatics filters based primarily unambigu-            showing expression polymorphism indicate clear significant cis <br /> ous orthology, unique sequence, presence SNPs se-              effect (49% 421/863). (See Supplemental Fig. S6 de- <br /> lected 4442 genes analysis initial pool 6604 ORFs           tailed description classifications above.) <br /> with untranslated region (UTR) information (Nagalakshmi et al.                   Previous eQTL studies showed large &#226;&#8364;&#732;&#226;&#8364;&#732;hotspot&#226;&#8364;&#8482;&#226;&#8364;&#8482; effect (Brem <br /> 2008). (A detailed description filters available            et al. 2002; Yvert et al. 2003 showed 1265/1716 distant <br /> Methods Supplemental Tables S1&#226;&#8364;&#8220;S3.) We identified 893 SNPs             eQTLs 1265/2294 eQTLs). To investigate ASE variation <br /> caused putative genome reference sequence errors cor-               genes likely influenced hotspots, dis- <br /> rected them. In total, analysis incorporated 35,225 SNP sites          carded 1265 genes identified Yvert et al. (2003) (Fig. 2D). As <br /> distributed 4442 orthologous gene pairs.                             expected, 76% differentially expressed genes discarded fell <br />  <br />  <br />                                                                                                                    Genome Research             827 <br />                                                                                                                       www.genome.org <br />  Emerson et al. <br />  <br />  <br />  <br />  <br /> Figure 1. Comparison methods estimating ASE polymorphism. (A&#226;&#8364;&#8220;C ) Y-axis, log2(eIGA) (IGA transcriptome data), versus x-axis, log2(ePyro) <br /> (pyrosequencing). (A,B) comparison co-culture hybrid results methods, respectively; (C ) comparison IGA sequencing and <br /> pyrosequencing value log2(eCo)   log2(eHy), log2(etrans). To test regressions differ equality, tested estimated <br /> regression coefficients null hypothesis H0: m = 1; b = 0. All P-values hypothesis tests regression parameters significant (all <br /> P-values &gt; 0.05), indicating IGA pyrosequencing similar results. <br />  <br />  <br /> into various trans categories (68%) &#226;&#8364;&#732;&#226;&#8364;&#732;both&#226;&#8364;&#8482;&#226;&#8364;&#8482; cate-                ation. To reconcile difference, performed additional <br /> gory (8%). In contrast, &lt;2% hotspot genes fell cis-                 tests: (1) comparing v genes showing significant P-value <br /> only category, remaining 22% genes falling cis                     (P &lt; 0.01) null hypothesis log2(e) = 0 not <br /> categories putatively minor trans component. Despite dis-                    significant (P &gt; 0.05) using Wilcoxon rank sum test; (2) <br /> carding genes, trans variation remains prevalent form                  correlation tests |log2(e)| v, using Pearson, Kendall, <br /> variation (Fig. 2D).                                                                Spearman methods. For tests (one <br />                                                                                     Wilcoxon test correlation tests cis trans), we <br />                                                                                     low v associated low expression variation (all <br /> Evolutionary constraint expression polymorphism <br />                                                                                     P-values &lt; 10 5). <br /> To test ASE data evolutionary constraint, compared                         To investigate cis trans variation sensitive <br /> magnitudes expression ratio estimates categories                     purifying selection, compared quantity |log2(etrans)|   <br /> of genes predicted strongly constrained versus predicted                |log2(ecis)| constrained unconstrained categories. For <br /> to weakly constrained (Fig. 3). For protein&#226;&#8364;&#8220;protein interaction                  constraint based protein&#226;&#8364;&#8220;protein interactions essen- <br /> networks (Stark et al. 2006; Collins et al. 2007), genes             tial/nonessential genes, quantity |log2(etrans)|   |log2(ecis)| is <br /> most interactions significantly lower expression variation                 significantly lower strongly constrained categories in <br /> those genes absent networks, cis trans poly-                  weakly constrained categories (Fig. 3A,B). Comparing genes <br /> morphism (Wilcoxon rank sum tests, P-values &lt; 10 6). Essential                  lowest v highest shows positive rela- <br /> genes (genes cause lethality deleted; Deutschbauer et al.                 tionship, difference significant (all correlation <br /> 2005) showed significantly expression variation non-                 P-values &gt; 0.05). <br /> essential genes cis trans (Wilcoxon rank sum tests, all <br /> P-values &lt; 10 15), corroborating previous study (Ronald Akey <br /> 2007). One important measure constraint based sequence                        Comparing expression polymorphism expression divergence <br /> evolution v (Yang 1997, 2007), ratio rate nonsyn-                  We compared relative contributions cis trans regulatory <br /> onymous substitution rate synonymous substitution (i.e.,                  polymorphism data set cis trans regulatory divergence <br /> Ka/Ks). Our expression estimates value v signifi-               S. cerevisiae S. paradoxus recent publication <br /> cantly correlated |log2(ecis)| |log2(etrans)| (all P-values &lt;         (Tirosh et al. 2009). We adopted following perspectives: (1) <br /> 0.0001, Pearson, Kendall, Spearman correlation coefficients).               comparing relative cis/trans contributions between <br /> Comparing 50th percentile genes lowest v                   species; (2) comparing correlations polymorphism <br /> with highest v (Fig. 3C) demonstrates magnitude                divergence cis trans. In comparison, test <br /> expression variation strongly constrained genes lower               hypothesis proposed recent study Drosophila sister <br /> in constrained genes.                                                          species (Fig. 4; Wittkopp et al. 2008). We compared relationship <br />       This correlation v expression surprising,                   hybrid experiment versus co-culture/parental ex- <br /> it contradicts previous report relationship                    periment species (Fig. 4A,B, respectively), using <br /> (Ronald Akey 2007). One possible source discrepancy                     major axis (MA) regression standardized major axis (SMA) <br /> might compared expression level difference                        regression (Warton et al. 2006). Regression estimates closer the <br /> between set genes low v high v,                 diagonal indicate cis predominates, estimate near <br /> Ronald Akey study did opposite, comparing v                         horizontal axis indicates trans predominates. We that <br /> genes expression variation expression vari-                   interspecific slope significantly greater intraspecific <br />  <br />  <br /> 828      Genome Research <br />             www.genome.org <br />                                                                                                       Selection cis trans regulation yeasts <br />  <br />  <br />  <br />  <br /> Figure 2. Genome-wide ASE polymorphism S. cerevisiae: (A&#226;&#8364;&#8220;C ) y-axis, log2(ecis), versus x-axis, log2(etrans). (A) Scatter plot cis trans estimators <br /> with shading points indicating category individual genes fall into, determined two-dimensional 99% confidence intervals. These <br /> classifications respect relationship confidence intervals point lines running origin, are <br /> the axes diagonals. Trans-only (t only) cis-only (c only) indicate confidence intervals genes, respectively, overlap <br /> the log2(ecis) = 0 log2(etrans) = 0 axis line only. &#226;&#8364;&#732;&#226;&#8364;&#732;Both&#226;&#8364;&#8482;&#226;&#8364;&#8482; refers genes overlap log2(ecis) = log2(etrans) line log2(ecis) = <br />  log2(etrans) line (the positive negative diagonals). Trans-dominant (t dom) cis-dominant (c dom) genes overlap lines fall the <br /> quadrants nearest log2(ecis) = 0 log2(etrans) = 0 lines, respectively. Trans-major (t maj) overlaps log2(ecis) = 0 line the <br /> diagonals, cis-major (c maj) overlaps log2(etrans) = 0 line diagonals. &#226;&#8364;&#732;&#226;&#8364;&#732;ns&#226;&#8364;&#8482;&#226;&#8364;&#8482; indicates nonsignificant genes: them <br /> overlaps log2(ecis) = 0 log2(etrans) = 0 lines. A detailed explanation genes classified Supplemental Figure <br /> S6. (B) A contour plot two-dimenstional probability density function data (from two-dimensional kernel density estimator MASS <br /> library R) indicating genes fall cis/trans space using independent estimates cis trans. The &#226;&#8364;&#732;&#226;&#8364;&#732;elevation&#226;&#8364;&#8482;&#226;&#8364;&#8482; indicated contours <br /> expresses probability point falling region. The total volume beneath surface sums unity. (C ) A summary classifications based the <br /> results Figure 1A. Dark green, trans-only; light green, trans-dominant; dark orange, cis-only; light orange, cis-dominant; hashed regions, &#226;&#8364;&#732;&#226;&#8364;&#732;cis/ <br /> trans-major&#226;&#8364;&#8482;&#226;&#8364;&#8482; classifications; white, both; gray, nonsignificant genes. (D) The histogram indicates significant trans changes dominate comparison to <br /> significant cis changes. The left bar pair discarding 1265 &#226;&#8364;&#732;&#226;&#8364;&#732;hotspot&#226;&#8364;&#8482;&#226;&#8364;&#8482; genes Yvert et al. (2003), right bar discarding <br /> them. Importantly, trans-dominant cis-dominant confused genetic dominance; instead, meant convey the <br /> magnitude trans variation greater cis variation reverse, respectively. <br />  <br />  <br /> (P &lt; 2.2 3 10 16 MA SMA regressions), in-                        tion proportion variation measured co-culture ex- <br /> dicating magnitude cis regulatory variation relative                   periment, |log2(ecis)|/|log2(epar)|; magnitude cis variation <br /> trans greater species species.                                 proportion total variation cis trans, <br />       Next, investigate measures cis variation:                       |log2(ecis)|/[|log2(ecis)| + |log2(etrans)|]. We plotted values <br /> magnitude cis variation, |log2(ecis)|; magnitude cis varia-                 quantile (Fig. 4C&#226;&#8364;&#8220;E) interspecific cis <br />  <br />  <br />                                                                                                                                    Genome Research               829 <br />                                                                                                                                       www.genome.org <br />  Emerson et al. <br />  <br />  <br />  <br />  <br /> Figure 3. Constraint expression polymorphism. (A) Genes presently detected protein&#226;&#8364;&#8220;protein interaction (ppi) partners current data (light <br /> bars) versus genes upper 50th percentile showing interactions (dark bars). (B) Nonessential genes (light bars) genes homozygous <br /> knockouts fitness greater 0.85. Essential genes (light bars) genes homozygous knockouts lethal. (C ) Genes lower <br /> 50th percentile v (dark bars) versus genes upper 50th percentile v (light bars). Pairs bars labeled &#226;&#8364;&#732;&#226;&#8364;&#732;cis&#226;&#8364;&#8482;&#226;&#8364;&#8482; &#226;&#8364;&#732;&#226;&#8364;&#732;trans&#226;&#8364;&#8482;&#226;&#8364;&#8482; compare mean <br /> expression divergence |log2(e)| putatively strongly constrained weakly constrained categories. The category &#226;&#8364;&#732;&#226;&#8364;&#732;trans   cis&#226;&#8364;&#8482;&#226;&#8364;&#8482; compares the <br /> quantity |log2(etrans)|   |log2(ecis)| putatively strongly constrained weakly constrained categories. Significance indicated follows: ns, P &gt; <br /> 0.05; *, 0.01 &lt; P &lt; 0.05; **, 0.01 &lt; P &lt; 0.001; ***, P &lt; 0.001. Both cis trans mutations subject purifying selection comparisons. trans <br /> mutations sensitive changes constraint cis changes constraint determined number protein interactions essentiality, <br /> but measured protein coding sequence (v). <br />  <br />  <br />  <br /> values consistently higher intraspecific cis values,                 cis trans parameter estimates highly concordant py- <br /> regardless cis values scaled (Wilcoxon rank sum test,                  rosequencing experiments (Fig. 1). <br /> P-value &lt; 2.2 3 10 16 comparisons). Interestingly,                     Our expression parameter estimates (Fig. 2) demonstrate that, <br /> show cis share total cis + trans variation (Fig. 4E) sig-          expression polymorphism common types re- <br /> nificantly higher divergence polymorphism. Next,                  gulatory variation, trans differences dominate magnitude <br /> conduct formal test neutral <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span>mutation</span> hypothesis               number. Interestingly, short evolutionary <br /> genome comparing polymorphism data divergence                       time divergence BY RM strains, many <br /> data Tirosh et al. (2009). We follow framework Kreitman                 genes exhibit cis trans differences simultaneously, indi- <br /> and Aguade (1986) dividing significant expression differences                  cating sustained mutations affecting the <br /> into 2 3 2 contingency table. The test decisively rejects                   expression <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span>phenotype</span>. This prevalence trans variation persists <br /> predictions neutral theory (Table 1A; P &lt; 10 10, Fisher&#226;&#8364;&#8482;s exact            trans-regulated genes controlled hotspots (Brem <br /> test). Clearly, far cis expression differences             et al. 2002; Yvert et al. 2003) discarded (Fig. 2D). One central <br /> the species expected within-species polymor-                    result molecular evolution nonsyn- <br /> phism data, suggesting natural selection plays important                  onymous mutations genes deleterious. To investigate this <br /> role shaping expression variation.                                             perspective context expression polymorphism, com- <br />       To trace source <span id='am-161' about='xsp:pattern' typeof='owl:Thing'>pattern</span>, examined cis                pared expression constraint constraint functional <br /> trans data separately. Interestingly, significant trans differences,          categories. We predict genes interaction partners, <br /> genes showing expression polymorphism tend differential                   essential genes, genes low v strongly con- <br /> expression species; correspondence signifi-                   strained. Comparing magnitude expression variation be- <br /> cantly polymorphic significantly divergent genes                      tween pairs category shows constraint ex- <br /> greater expected chance (Fig. 5A; Table 1B; Fisher&#226;&#8364;&#8482;s exact                pression corresponds constraint measures <br /> test, P &lt; 1 3 10 6). In contrast, corresponding association                 constraint (Fig. 3, cis trans columns). Interestingly, our <br /> observed cis variation (Fig. 5B; Table 1B; Fisher&#226;&#8364;&#8482;s exact test, P &gt;           results regarding essential genes agree report ex- <br /> 0.25). Under neutral theory, neutral variation correlated              pression variation (Ronald Akey 2007), conflicting <br /> polymorphism divergence, nonneutral categories                     results regard v; correlation v cis <br /> exhibit weaker correlations absence correlation alto-               variation (Fig. 3; Results), Ronald Akey not, per- <br /> gether. This suggests number significant differences              haps result different means measuring expression or <br /> cis-regulatory change subject nonneutral forces. Unlike                 different subsets comprising respective comparisons. <br /> for qualitative measures discussed above, pattern does ex-                     To determine category gene regulation more <br /> tend quantitative measures expression ratios (Fig. 5C,D; Sup-               strongly influenced purifying selection, compared dif- <br /> plemental Fig. S3C,D).                                                            ference trans cis various categories (i.e., <br />                                                                                   |log2(etrans)|   |log2(ecis)|, Fig. 3, trans &#226;&#8364;&#8220; cis column). For all <br />                                                                                   functional categories, trans variation higher uncon- <br /> Discussion                                                                        strained category constrained category, differ- <br /> Our results demonstrate inferring genetic architecture                ence statistically significant protein&#226;&#8364;&#8220;protein interactions <br /> expression level evolution performed straightforward,                 essential genes. This stronger filtering trans polymorphisms <br /> comprehensive, rigorous manner. Through novel applica-                      essential genes genes interaction partners sug- <br /> tion simple binomial models, infer confidence intervals                 gests expression differences species and <br /> for cis trans effects correcting experimental sampling              strongly influenced cis variants compared trans variants <br /> biases (via d parameter, Methods; Supplemental Fig. S1). These                considering constrained constrained <br />  <br />  <br /> 830      Genome Research <br />             www.genome.org <br />                                                                                               Selection cis trans regulation yeasts <br />  <br />  <br />  <br />  <br /> Figure 4. Between-species regulatory differences dominated cis-regulatory changes within-species differences. A B follow Figure 1 <br /> in Wittkopp et al. (2008), comparing regression hybrid co-culture lines intraspecific interspecific variation, respectively. The <br /> slope interspecific comparison (A) larger intraspecific comparison (B), regardless line-fitting method used (SMA MA <br /> regressions conducted using &#226;&#8364;&#732;&#226;&#8364;&#732;smatr&#226;&#8364;&#8482;&#226;&#8364;&#8482; package R (Warton et al. 2006; R Development Core Team 2009). C&#226;&#8364;&#8220;E follow Figure 2 Wittkopp et al. <br /> (2008). Measurements cis species consistently dominate, regardless measured. All comparisons significant, P &lt; 2.2 3 <br /> 10 16, indicating cis-regulatory changes dominate species species. <br />  <br />  <br /> categories. Indeed, established long timescales,              A recent study compared cis trans expression evolution <br /> important &#226;&#8364;&#732;&#226;&#8364;&#732;toolbox&#226;&#8364;&#8482;&#226;&#8364;&#8482; genes provide phenotypic inno-                   species 78 genes species Dro- <br /> vation despite highly constrained peptide sequences,                  sophila (Wittkopp et al. 2008). Plots hybrid ASE differences <br /> such innovation strongly influenced evolution cis elements             versus parental differences showed interspecific ASE variation <br /> (Carroll 2008). However, opportunity adaptation                 fits closely &#226;&#8364;&#732;&#226;&#8364;&#732;all cis&#226;&#8364;&#8482;&#226;&#8364;&#8482; line (the diagonal hybrid = <br /> reconfiguring trans-regulators remains intriguing possibility            co-culture) does intraspecific ASE variation (Fig. 1; Wittkopp <br /> genes constrained categories. Future investigations ASE            et al. 2008), results consistently significant <br /> in mutation accumulation lines (MAL) compared natural lines                  different partitions data (for three-fourths the <br /> (NL) (Denver et al. 2005) enable examination dis-               partitions, slopes estimate 95% confi- <br /> tribution fitness effects wider range deleterious variants. If      dence interval [CI] [Wittkopp et al. 2008; Supple- <br /> our hypothesis correct regarding trans variants, expres-           mental Table 5], assuming 95% CI = slope, mean 6 1.96 3 SE). <br /> sion ratio NL versus MAL lower constrained cate-                Similarly, relative contributions ASE measured cis/(cis + <br /> gories higher relaxed categories.                                        trans), polymorphism divergence significantly dif- <br />       In order extend inferences longer timescales,                ferent (Fig. 2, column 3; Wittkopp et al. 2008). Interestingly, one <br /> compared polymorphism data recent data set expres-                 comparison authors inferred polymorphism indirectly <br /> sion S. cerevisiae S. paradoxus. One key prediction              divergence data, cis effect observed (Fig. 3; Wittkopp <br /> the neutral theory molecular evolution degree                 et al. 2008). <br /> polymorphism rate fixation increasing func-                       To test hypothesis cis variation subject natural <br /> tions mutation rate (Kimura 1968; Kimura Ota 1971),                  selection genome-wide scale, compared expression <br /> leading recognition comparing intraspecific variation               polymorphism data recent expression divergence data set <br /> (polymorphism) interspecific variation (divergence) power-              (Tirosh et al. 2009). Our results strongly suggest comparison <br /> ful strategy testing hypotheses concerning natural selection                intraspecific expression variation, interspecific expression <br /> (Kreitman Aguade 1986; Hudson et al. 1987; McDonald                     variation strongly shaped cis evolution. Impor- <br /> Kreitman 1991; Bustamante et al. 2002). Importantly, con-                  tantly, regressions co-culture hybrid experiments fall <br /> trasts distinguish variation resulting higher                  significantly closer &#226;&#8364;&#732;&#226;&#8364;&#732;all cis&#226;&#8364;&#8482;&#226;&#8364;&#8482; (i.e., hybrid = co-culture) line <br /> mutation rates (Fisher 1922; Haldane 1927) variation                 interspecific comparisons intraspecific comparisons <br /> the action natural selection.                                                (Fig. 4A.B). Moreover, cumulative cis expression divergence is <br />  <br />  <br />                                                                                                                          Genome Research              831 <br />                                                                                                                             www.genome.org <br />  Emerson et al. <br />  <br />  <br /> Table 1.   Comparison polymorphism divergence cis trans mutations                        gence cis trans individually. By <br />                                                                                                             doing this, hope adduce evidence <br /> (A) Comparison significant genes cis trans categories our <br />    polymorphism data divergence data Tirosh et al. (2009)a                                     bearing categories is <br />                                                                                                             likely neutral. Under neutrality <br />                                          Polymorphism                              Divergence               predict levels polymorphism <br />                                                                                                             divergence mutation driven. If the <br /> Cis                                            396                                    1270                  rate mutation varies genes, it <br /> Trans                                          412                                      541                 vary manner between <br /> (B) Significant nonsignificant genes comparisons polymorphism divergence          b    polymorphism divergence the <br />                                                                                                             gene. For example, genes are <br />                                      Significant polymorphism         Nonsignificant polymorphism           significantly variable species should <br />                                                                                                             likely variable spe- <br /> Trans                                                                                                       cies, genes variable within <br />    Significant divergence                    124 (94.8)                         417 (446.2)                 species likely variable <br />    Nonsignificant divergence                 288 (317.2)                      1523 (1493.8) <br /> Cis                                                                                                         species. Indeed, pattern ex- <br />    Significant divergence                    222 (213.8)                      1048 (1056.2)                 actly observe trans mutations <br />    Nonsignificant divergence                 174 (182.2)                        908 (899.8)                 (Table 1B; Fig. 5A), suggesting trans <br />                                                                                                             differences conform prediction of <br /> a <br />   Significant nonhomogeneity evidence violation neutral theory, described Kreitman     neutral theory. On hand, cis dif- <br />                                           16 <br /> and Aguade (1986). P-value &lt; 2.2 3 10 . <br /> b <br />   This enables test homogeneity polymorphism (columns) divergence (rows),             ferences fit prediction poorly <br /> the categories divided significant nonsignificant genes. The P-values trans cis     (Table 1B; Fig. 5B). The number signifi- <br /> are 2.33 3 10 4 0.377, respectively. This result unchanged independent estimates used     cis differences common poly- <br /> (Supplemental Table S4).                                                                                    morphism divergence greater <br /> c <br />   Numbers indicate observations numbers parentheses indicate expectations. The numbers in <br /> boldface correspond Figure 5, A B, respectively. <br />                                                                                                             expected chance, indicating that <br />                                                                                                             polymorphism divergence correlate <br />                                                                                                             weakly, arguing neutrality <br /> significantly consistently cis polymorphism (Fig.             cis variation. Given relatively low level cis polymorphism <br /> 4C), cis divergence scaled expression variation                 versus high level cis divergence combined evidence <br /> between unhybridized parental strains (Fig. 4D). These results              Table 1B, suggest cis-regulatory differences under <br /> strongly suggest cis variation plays greater role                positive selection alternative trans polymorphism <br /> species species. Most interestingly, compare                balancing selection. We examined correlation be- <br /> the relative contributions cis variation total cis + trans ex-            tween magnitude expression divergence polymorphism <br /> pression variation polymorphism divergence,                      (Fig. 5C,D). Unlike count data described above, pooled ex- <br /> find cis divergence plays larger role predicted            pression variation estimates, polymorphism predicts divergence <br /> from polymorphism data (Fig. 4E), strongly implicating action               equally trans variation (r2 = 23%, Fig. 5C) cis vari- <br /> of natural selection. We think possible larger sample size         ation (r2 = 26%, Fig. 5D; Supplemental Fig. S3). While for <br /> the Drosophila genome corroborate results.                            trans variants, gene polymorphic good pre- <br />        We presented formal test neutral mutation hy-              dictor differs species, magnitude of <br /> pothesis genome comparing significant poly-                   expression level polymorphism weakly related magni- <br /> morphic genes significant divergent genes data               tude expression level divergence. This weak relationship nearly <br /> Tirosh et al. (2009). We divide significant expression differences         disappears genes pooled polymorphism expression <br /> a 2 3 2 contingency table (Table 1A), described Results. If           level (data shown), likely magnitude change in <br /> data follow predictions neutral theory (or alternatively             expression level vary greatly different mutations the <br /> if categories experience similar selection regimes),                  gene. <br /> divergence/polymorphism ratios similar cis                     These observations indicate trans variation conforms more <br /> trans categories. A violation homogeneity table ev-            closely predictions neutral theory cis variation. <br /> idence category violates neutrality, failure           Thus, comparisons polymorphism divergence data for <br /> to reject weak indication neutrality, possible       expression levels strongly suggest cis evolution strongly shapes <br /> both categories similar nonneutral selective regimes.            differences species variation strongly sha- <br /> The test unequivocally rejects predictions neutral theory            ped positive natural selection. Taken together, analyses paint <br /> (Table 1A; P &lt; 1 3 10 10, Fisher&#226;&#8364;&#8482;s exact test), strongly suggesting        comprehensive picture selective forces shaping cis trans <br /> either excess divergent cis differences species          evolution reinforce idea cis expression differences play <br /> as occur positive selection excess          dominant role adaptive expression divergence species. <br /> polymorphism trans regulatory variation species as <br /> might observed balancing selection (Kreitman Aguade                Methods <br /> 1986; Hudson et al. 1987). Importantly, test capable of <br /> determining table highly heterogeneous. It de-               Yeast strains growth conditions <br /> termine source departure homogeneity.                           Two culture types prepared: co-culture hybrid. The co- <br />        The test relies correlation polymorphism            culture experiment prepared approximately equal <br /> and divergence neutral variants different categories.           amounts MATa strains called BY RM. The hybrid strain <br /> To interrogate predicted relationship detail, mea-              derived mating BY (MATa) 3 RM (MATa) grown <br /> sured association significant polymorphism diver-               standard YPAD medium. <br />  <br />  <br /> 832     Genome Research <br />            www.genome.org <br />                                                                                                   Selection cis trans regulation yeasts <br />  <br />  <br />                                                                                                                 double-stranded cDNA syn- <br />                                                                                                                 thesized, fragmented, subjected to <br />                                                                                                                 Illumina Genome Analyzer (IGA) se- <br />                                                                                                                 quencing protocol. <br />                                                                                                                      For expression measurements, over- <br />                                                                                                                 night subcultures used prepare four <br />                                                                                                                 technical replicates hybrid <br />                                                                                                                 (WL201) cultures BY + RM co-cultures <br />                                                                                                                 starting OD600 = 0.1 harvested <br />                                                                                                                 cell density reached OD600 = 1.0. <br />                                                                                                                 Total RNA extracted hot <br />                                                                                                                 acid phenol method (Kohrer Domdey <br />                                                                                                                 1991). Total RNA concentration within <br />                                                                                                                 replicate quantified Nano- <br />                                                                                                                 Drop 1000 spectrophotometer (Thermo <br />                                                                                                                 Scientific). Equal amounts RNA were <br />                                                                                                                 pooled sample group (i.e., hy- <br />                                                                                                                 brid co-culture) combined RNA <br />                                                                                                                 sample. The quality combined RNA was <br />                                                                                                                 assessed BioAnalyzer (Agilent). <br />                                                                                                                      The hybrid co-culture mRNA was <br />                                                                                                                 purified using oligo(dT) Dynabeads (Invi- <br />                                                                                                                 trogen) according manufacturer&#226;&#8364;&#8482;s <br />                                                                                                                 protocol. Subsequent reverse transcription <br />                                                                                                                 carried oligo(dT) primers and <br />                                                                                                                 Superscript II kit (Invitrogen) follow- <br />                                                                                                                 ing manufacturer&#226;&#8364;&#8482;s instructions. Tran- <br />                                                                                                                 scriptome sequencing steps per- <br />                                                                                                                 formed Fasteris SA, Switzerland. Sam- <br />                                                                                                                 ples containing cDNAs fragments 200&#226;&#8364;&#8220; <br />                                                                                                                 400 nucleotides (nt) sequenced <br />                                                                                                                 24 lanes (12 hybrid 12 co- <br />                                                                                                                 culture) flow cell IGA sequencer <br />                                                                                                                 using Illumina&#226;&#8364;&#8482;s genomic shotgun pro- <br /> Figure 5. Greater correlation trans polymorphism divergence cis poly- <br /> morphism divergence. (A,B) The number significant genes manifested intraspecific <br />                                                                                                                 tocol, yielding 35-nt-long reads. <br /> measurements interspecific measurements trans (A) cis (B). The histograms indicate null <br /> hypothesis homogeneous association significant polymorphisms significant divergence. <br /> Let P number genes significant expression differences species D number         Yeast genome sequencing <br /> of differences species. If P gene names randomly drawn replacement the <br /> gene list, histogram represents probability independent draw D genes results x       To estimate relative cell densities the <br /> genes common lists. Each histogram hypergeometric distribution representing upper       samples confirm SNP assignments, <br />                                                                                                            gDNA hybrid co- <br /> lefthand cell 2 3 2 table Fisher&#226;&#8364;&#8482;s exact test (Table 1B). For trans genes (A), cell indicates <br /> overlap significant expression polymorphisms significant expression divergence       culture samples used cDNA se- <br /> significantly higher expected chance, cis genes (B), overlap  quencing extracted, fragmented, and <br /> range expected simply randomly shuffling data. (C,D) The relationship polymorphism <br />                                                                                                            subjected IGA DNA sequencing <br /> and divergence estimates shown genes significant polymorphism estimates. The regression <br /> estimate trans polymorphism divergence describes 23% variance, be-           protocol. The genomic DNA extracted <br /> tween cis polymorphism divergence describes 26%. Each point composed genes grouped            using Qiagen Q100 genomic purification <br /> bins according polymorphism estimates. The values obtained taking median bin     kit (Qiagen). Within sample group, <br /> divergence polymorphism. Each bin contains 11 12 genes.                                             equal amounts DNA pooled into <br />                                                                                                            combined DNA sample, analogous the <br />      The laboratory strain designated &#226;&#8364;&#732;&#226;&#8364;&#732;BY&#226;&#8364;&#8482;&#226;&#8364;&#8482; officially named                   pooling strategy employed transcriptome sequencing above. <br /> BY4741 (MATa his3D1 leu2D0 met15D0 ura3D0) descendant                     The combined DNA sample fragmented sonication, <br /> of S288C. The strain designated &#226;&#8364;&#732;&#226;&#8364;&#732;RM&#226;&#8364;&#8482;&#226;&#8364;&#8482; (a gift Dr. Leland                     shotgun libraries prepared according Illumina&#226;&#8364;&#8482;s <br /> Hartwell, Fred Hutchinson Cancer Research Center) officially                    gDNA protocol. The genomic DNA sequencing carried on <br /> either RM11-1a (MATa lys2D0 ura3D0 hoTKAN) RM11-1a (MATa                        Illumina GA-II (IGA-II) sequencer High Throughput Se- <br /> lys2D0 ura3D0 hoTKAN). Both haploid strains derived                       quencing Core Facility Academia Sinica, yielding reads 40 nt in <br /> Bb32(3), natural isolate described previously (Mortimer et al.                   <span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span>. <br /> 1994). We designated hybrid BY4741 3 RM11-1a <br /> cross constructed lab WL201. The co-culture sample is <br />                                                                                    Mapping IGA reads reference genomes <br /> simply mixture BY4741 RM11-1a. All yeast strains were <br /> grown YPAD media 30&#194;&#176;C 250 rpm shaking.                                  The BY reference genome downloaded SGD project on <br />                                                                                    April 3, 2008 (ftp://ftp.yeastgenome.org/yeast/). The RM reference <br />                                                                                    genome downloaded Saccharomyces cerevisiae RM11-1a <br /> Total RNA extraction sequencing                                                Sequencing Project, Broad Institute (http://www.broad.mit.edu). <br /> To estimate expression ratios hybrid co-culture exper-                 Every cDNA sequence read used query reference <br /> iments, total RNA extracted purified mRNA,                        yeast genome using MEGABLAST &#226;&#8364;&#732;&#226;&#8364;&#732;wordsize&#226;&#8364;&#8482;&#226;&#8364;&#8482; parameter set <br />  <br />  <br />                                                                                                                             Genome Research           833 <br />                                                                                                                               www.genome.org <br />  Emerson et al. <br />  <br />  <br /> to 8 (Zhang et al. 2000), yielding homology search datasets,        ward allele cDNA data. By comparing SNP count data <br /> for BY genome RM genome. We recorded           obtained cDNA obtained gDNA, classified 893 <br /> hits nucleotide mismatches. A mismatch            sites true errors reference genomes, 540 true <br /> to sequencing error sequence read obtained, sequence          differential expression, failed classify 57 sites. Using in- <br /> error reference genome(s), SNP site ref-        formation, corrected 893 &#226;&#8364;&#732;&#226;&#8364;&#732;true error&#226;&#8364;&#8482;&#226;&#8364;&#8482; SNPs reference <br /> erence genomes. For set homology search results, classi-         genomes discarded data set 57 SNPs we <br /> fied read uninformative (perfectly matching genomes);          failed classify. We repeated mapping computation to <br /> informative (matching SNPs genome); in-             obtain final read counts. For details procedure, <br /> formative error (matching SNP containing IGA                consult Supplemental Materials. <br /> error); error (containing IGA errors). See Supple- <br /> mental material details classifications.                 Modeling gene expression discrete sampling process <br />       From 12 channels cDNA IGA sequencing data each <br />                                                                             To estimate expression parameters IGA read data, formu- <br /> of samples (total 24 channels), obtained 71,309,740 <br />                                                                             lated question terms binomial distribution. A nor- <br /> and 71,549,168 raw reads hybrid sample co- <br />                                                                             malization parameter required differences total reads <br /> culture sample, respectively, subjected classification <br />                                                                             samples occur sampling effort evenly <br /> described (Table S1). Most mapped reads matched one <br />                                                                             distributed samples, study design or <br /> place genome, uniquely identifying expression sin- <br />                                                                             experimental error. Though authors recommend employing <br /> gle transcript (Table S2). <br />                                                                             methods related standard quantile normalization (Bolstad et al. <br />       In order determine read mapping strategy resulted in <br />                                                                             2003) count data (Balwierz et al. 2009), consider rescaling <br /> spurious differential expression (Degner et al. 2009), examined <br />                                                                             counts (which contain important information regarding sam- <br /> each combination experiment (hybrid co-culture) strain <br />                                                                             pling variance) ideal. Moreover, apply noise <br /> (BY RM). By dividing data combination six <br />                                                                             correction. There reasonable physical rationales noise correc- <br /> channels each, compared independent partitions the <br />                                                                             tion array studies (Tu et al. 2002). In case deep sequencing, <br /> same biological material, determining null hypoth- <br />                                                                             however, clear mapped sequence reads comprise the <br /> esis differential expression rejected differential ex- <br />                                                                             &#226;&#8364;&#732;&#226;&#8364;&#732;signal&#226;&#8364;&#8482;&#226;&#8364;&#8482; comprise &#226;&#8364;&#732;&#226;&#8364;&#732;noise.&#226;&#8364;&#8482;&#226;&#8364;&#8482; <br /> pression absent. Our results indicate typical rejection rates (Sup- <br /> plemental Fig. S4). <br />       In order investigate potential sources sequencing            Cis trans parameter estimation <br /> amplification bias data, examined number times             Let measurement total informative expression read counts for <br /> a unique read represented channel sequencing data.           gene N read counts RM allele X the <br /> The histograms expression read-counts compare favorably               BY allele N-X. The data binomial distribution pro- <br /> a priori estimates based discrete stochastic models (Supple-             portion parameter p. Let j represent single experiment (co-culture <br /> mental Fig. S5).                                                            hybrid), e expression ratio parameter RM BY <br />                                                                             d normalization ratio parameter RM BY. <br /> Identifying orthologous pairs polymorphic sites                         The proportion parameter p expressed terms d e: <br /> between BY RM genomes                                                                                           d j ej <br />                                                                                                               pj =              : <br /> Each gene refined set BY gene transcripts UTR in-                                                     d j ej + 1 <br /> formation (Nagalakshmi et al. 2008) aligned RM ge- <br />                                                                             The assumptions hybrid experiment exhibits cis <br /> nome using BLAT axtChain (Kent 2002; Kent et al. 2003) to <br />                                                                             variation co-culture exhibits combination cis trans <br /> identify ortholog RM genome. These alignments were <br />                                                                             variation expressed as <br /> then used identify polymorphic sites orthol- <br /> ogous genes. We restricted attention high scoring pairs                                                  eHy = ecis <br /> (HSPs) derived chaining procedure, neglecting reads                                               eCo = ecis etrans : <br /> mapped nonhomologous regions. <br />                                                                             Consequently, binomial proportion parameters re- <br /> Exclusion overlapping gene regions                                       written follows: <br /> Transcript sequencing data derived double-stranded                                                 d Hy eHy      d Hy ecis <br /> cDNA; therefore, unable determine strand reads                                    pHy =               = <br />                                                                                                            d Hy eHy + 1 d Hy ecis + 1 <br /> are mapped regions transcripts opposite                                                                                          &#195;&#176;1&#195;&#382; <br /> strands overlap (Nagalakshmi et al. 2008). To avoid misattribution,                                          d Co eCo     d Co ecis etrans : <br />                                                                                                    pCo   =             = <br /> we temporarily excluded overlapping regions. Genes                                             d Co eCo + 1 d Co ecis etrans + 1 <br /> average read count <span id='am-160' about='xsp:base' typeof='owl:Thing'>base</span> nonoverlapping regions gene           Thus, likelihood functions hybrid co-culture experi- <br /> were 0.025 read/nt discarded data set. Next,        ments expressed as: <br /> we reintegrated reads mapping regions longer over-                                                                          <br /> lapped expressed genes opposite strand, yielding 4566               L eHy  d Hy ; XHy ; N Hy =  L ecis jd <br />                                                                                                                      Hy ; X Hy ; N Hy   <br /> SNP containing gene pairs possessing unique orthologous regions.                                             N Hy XHy                  N Hy  XHy &#195;&#176;2&#195;&#382; <br />                                                                                                           =           pHy 1   pHy <br /> These genes subjected study (Table S3).                                                      XHy <br />  <br />                                                                                    L&#195;&#176;eCo jd Co ; XCo ; N Co &#195;&#382; = L&#195;&#176;ecis ; etrans jd Co ; XCo ; N Co &#195;&#382; <br /> Detecting errors genomic sequences strains                                                               <br />                                                                                                                   N Co XCo                    N  X     &#195;&#176;3&#195;&#382; <br /> If expression level SNP exhibits particularly strong bias                                         =             pCo 1   pCo Co Co : <br />                                                                                                                   XCo <br /> toward allele, indicates strong pattern differential <br /> expression error reference genome aligned reads         Because Equation 3 prevents estimating etrans independently <br /> to. Our data contained 1490 SNPs exhibiting strong bias to-          ecis, instead examine product Equations 3 4 <br />  <br />  <br /> 834     Genome Research <br />            www.genome.org <br />                                                                                                  Selection cis trans regulation yeasts <br />  <br />  <br /> to obtain likelihood function etrans estimated in-                 Bloom JS, Khan Z, Kruglyak L, Singh M, Caudy AA. 2009. Measuring <br /> dependently ecis:                                                                differential gene expression short read sequencing: Quantitative <br />                                                                                       comparison 2-channel gene expression microarrays. BMC Genomics <br />                                                                                       10: 221. doi: 10.1186/1471-2164-10-221. <br />                                                                   N Hy XHy <br />        L ecis ; etrans jd Hy ; XHy ; N Hy ; d Co ; XCo ; N Co =         pHy       Bolstad BM, Irizarry RA, Astrand M, Speed TP. 2003. A comparison of <br />                                                                   XHy                 normalization methods high density oligonucleotide array data <br />                          N Hy  XHy N                                       &#195;&#176;4&#195;&#382;       based variance bias. Bioinformatics 19: 185&#226;&#8364;&#8220;193. <br />                                           Co                 N  X <br />          3 1   pHy                                 1   pCo Co Co :                Brem RB, Yvert G, Clinton R, Kruglyak L. 2002. Genetic dissection of <br />                                        X <br />                                        Co                                             transcriptional regulation budding yeast. Science 296: 752&#226;&#8364;&#8220;755. <br />                                                                                   Bustamante CD, Nielsen R, Sawyer SA, Olsen KM, Purugganan MD, Hartl <br /> We obtain expression parameter estimates ecis                       DL. 2002. The cost inbreeding Arabidopsis. Nature 416: 531&#226;&#8364;&#8220;534. <br /> Equations 2 4 etrans Equation 4 using standard likeli-            Carroll SB. 2008. Evo-devo expanding evolutionary synthesis: A <br /> hood maximization methods R (R Development Core Team 2009).                        genetic theory morphological evolution. Cell 134: 25&#226;&#8364;&#8220;36. <br />                                                                                   Collins SR, Kemmeren P, Zhao X-C, Greenblatt JF, Spencer F, Holstege FCP, <br />                                                                                       Weissman JS, Krogan NJ. 2007. Toward comprehensive atlas the <br /> Independent estimate cis trans parameters                                      physical interactome Saccharomyces cerevisiae. Mol Cell Proteomics 6: <br />                                                                                       439&#226;&#8364;&#8220;450. <br /> When cis trans estimated described above, estimates                Degner JF, Marioni JC, Pai AA, Pickrell JK, Nkadori E, Gilad Y, Pritchard JK. <br /> are negatively correlated (cf. Fig. 2B Supplemental Fig. S2B;                 2009. Effect read-mapping biases detecting allele-specific <br /> Supplemental material details). In order estimate ecis                    expression RNA-sequencing data. Bioinformatics 25: 3207&#226;&#8364;&#8220;3212. <br />                                                                                   Denver DR, Morris K, Streelman JT, Kim SK, Lynch M, Thomas WK. 2005. <br /> and etrans independently, divide hybrid data par-                     The transcriptional consequences mutation natural selection in <br /> titions channels each. We estimate ecis Equation 2                   Caenorhabditis elegans. Nat Genet 37: 544&#226;&#8364;&#8220;548. <br /> using partition hybrid data estimate etrans                   Deutschbauer AM, Jaramillo DF, Proctor M, Kumm J, Hillenmeyer ME, Davis <br /> Equation 4 using partition hybrid data                    RW, Nislow C, Giaever G. 2005. Mechanisms haploinsufficiency <br />                                                                                       revealed genome-wide profiling yeast. Genetics 169: 1915&#226;&#8364;&#8220;1925. <br /> the co-culture data. For rationale correlated estimates             Edgar RC. 2004. MUSCLE: A multiple sequence alignment method with <br /> preferred independent estimates preferred,                       reduced time space complexity. BMC Bioinformatics 5: 113. doi: <br /> Supplemental material.                                                                10.1186/1471-2105-5-113. <br />                                                                                   Emerson JJ, Kaessmann H, Betran E, Long M. 2004. Extensive gene traffic on <br />                                                                                       mammalian X chromosome. Science 303: 537&#226;&#8364;&#8220;540. <br /> Estimation normalization parameter                                         Fisher R. 1922. On dominance ratio. Proc R Soc Edinb 42: 321&#226;&#8364;&#8220;341. <br />                                                                                   Haldane J. 1927. A mathematical theory natural artificial selection, <br /> We estimate normalization parameter d gDNA data                          Part V: Selection mutation. Proc Camb Philol Soc 23: 838&#226;&#8364;&#8220;844. <br /> (which independent cDNA data) follows:                             Halder G, Callaerts P, Gehring WJ. 1995. Induction ectopic eyes by <br />                                                                                       targeted expression eyeless gene Drosophila. Science 267: <br />                                          G                                            1788&#226;&#8364;&#8220;1792. <br />                                          + Xi;j                                   Hudson RR, Kreitman M, Aguade M. 1987. A test neutral molecular <br />                             ^j =         i=1                                          evolution based nucleotide data. Genetics 116: 153&#226;&#8364;&#8220;159. <br />                             d      G <br />                                                     ; <br />                                                                                   Kent WJ. 2002. BLAT&#226;&#8364;&#8221;the BLAST-like alignment tool. Genome Res 12: 656&#226;&#8364;&#8220; <br />                                    + N i;j   Xi;j                                     664. <br />                                    i=1 <br />                                                                                   Kent WJ, Baertsch R, Hinrichs A, Miller W, Haussler D. 2003. Evolution&#226;&#8364;&#8482;s <br />                                                                                       cauldron: Duplication, deletion, rearrangement mouse and <br /> where indexes gene j represents experiment (hybrid <br />                                                                                       human genomes. Proc Natl Acad Sci USA 100: 11484&#226;&#8364;&#8220;11489. <br /> or co-culture).                                                                   Kimura M. 1968. Evolutionary rate molecular level. Nature 217: 624&#226;&#8364;&#8220; <br />                                                                                       626. <br />                                                                                   Kimura M, Ota T. 1971. Protein polymorphism phase molecular <br /> Sequence divergence statistics                                                        evolution. Nature 229: 467&#226;&#8364;&#8220;469. <br />                                                                                   King MC, Wilson AC. 1975. Evolution levels humans and <br /> Orthologous genes S. cerevisiae S. paradoxus de-                     chimpanzees. Science 188: 107&#226;&#8364;&#8220;116. <br /> termined Fungal Orthogroups Repository (Wapinski et al.                  Kohrer K, Domdey H. 1991. Preparation high molecular weight RNA. <br /> 2007). The coding sequences frame-aligned pairwise                       Methods Enzymol 194: 398&#226;&#8364;&#8220;405. <br /> codon divergence statistics Ka Ks calculated using               Kreitman ME, Aguade M. 1986. Excess polymorphism Adh locus in <br />                                                                                       Drosophila melanogaster. Genetics 114: 93&#226;&#8364;&#8220;110. <br /> PAML package (Yang 1997, 2007). All methods alignment                      Kvitek DJ, Will JL, Gasch AP. 2008. Variations stress sensitivity and <br /> statistical calculation follow used Emerson et al. (2004),              genomic expression diverse S. cerevisiae isolates. PLoS Genet 4: <br /> the exception MUSCLE (Edgar 2004) use alignment.                         e1000223. doi: 10.1371/journal.pgen.1000223. <br />                                                                                   McDonald JH, Kreitman M. 1991. Adaptive protein evolution Adh <br />                                                                                       locus Drosophila. Nature 351: 652&#226;&#8364;&#8220;654. <br /> Acknowledgments                                                                   Mortimer RK, Romano P, Suzzi G, Polsinelli M. 1994. Genome renewal: A <br />                                                                                       new phenomenon revealed genetic study 43 strains of <br /> W.-H.L. supported Academia Sinica, Taiwan; In-                             Saccharomyces cerevisiae derived natural fermentation grape <br /> ternational E. Balzan Prize Foundation; NIH grants GM30998                        musts. Yeast 10: 1543&#226;&#8364;&#8220;1552. <br />                                                                                   Nagalakshmi U, Wang Z, Waern K, Shou C, Raha D, Gerstein M, Snyder M. <br /> and GM081724. J.J.E. supported Academia Sinica Dis-                         2008. The transcriptional landscape yeast genome defined by <br /> tinguished Postdoctoral Fellowship. We thank Michael McDonald,                        RNA sequencing. Science 320: 1344&#226;&#8364;&#8220;1349. <br /> Jun-Yi Leu, Bin He thoughtful discussion key issues                 R Development Core Team. 2009. R: A language environment statistical <br /> early versions manuscript. We thank anonymous                       computing. R Foundation Statistical Computing, Vienna, Austria. <br />                                                                                   Rockman MV, Kruglyak L. 2006. Genetics global gene expression. Nat Rev <br /> reviewers helpful suggestions constructive criticisms.                        Genet 7: 862&#226;&#8364;&#8220;872. <br />                                                                                   Ronald J, Akey JM. 2007. The evolution gene expression QTL Saccharomyces <br />                                                                                       cerevisiae. PLoS One 2: e678. doi: 10.1371/journal.pone.0000678. <br /> References                                                                        Ronald J, Brem RB, Whittle J, Kruglyak L. 2005. Local regulatory variation in <br />                                                                                       Saccharomyces cerevisiae. PLoS Genet 1: e25. doi: 10.1371/journal.pgen. <br /> Bailey JA, Gu Z, Clark RA, Reinert K, Samonte RV, Schwartz S, Adams MD,               0010025. <br />     Myers EW, Li PW, Eichler EE. 2002. Recent segmental duplications       Seoighe C, Nembaware V, Scheffler K. 2006. Maximum likelihood inference <br />     human genome. Science 297: 1003&#226;&#8364;&#8220;1007.                                             imprinting allele-specific expression EST data. <br /> Balwierz PJ, Carninci P, Daub CO, Kawai J, Hayashizaki Y, Belle WV, Beisel C,         Bioinformatics 22: 3032&#226;&#8364;&#8220;3039. <br />     van Nimwegen E. 2009. Methods analyzing deep sequencing                   Springer NM, Stupar RM. 2007. Allele-specific expression patterns reveal <br />     expression data: Constructing human mouse promoterome                biases embryo-specific parent-of-origin effects hybrid maize. <br />     deepCAGE data. Genome Biol 10: R79. doi: 10.1186/gb-2009-10-7-r79.                Plant Cell 19: 2391&#226;&#8364;&#8220;2402. <br />  <br />  <br />  <br />                                                                                                                              Genome Research              835 <br />                                                                                                                                 www.genome.org <br />  Emerson et al. <br />  <br />  <br /> Stark C, Breitkreutz B-J, Reguly T, Boucher L, Breitkreutz A, Tyers M. 2006.    Wapinski I, Pfeffer A, Friedman N, Regev A. 2007. Natural history and <br />     BioGRID: A general repository interaction datasets. Nucleic Acids Res      evolutionary principles gene duplication fungi. Nature 449: 54&#226;&#8364;&#8220;61. <br />     34: D535&#226;&#8364;&#8220;D539.                                                              Warton DI, Wright IJ, Falster DS, Westoby M. 2006. Bivariate line-fitting <br /> Sung H-M, Wang T-Y, Wang D, Huang Y-S, Wu J-P, Tsai H-K, Tzeng J, Huang            methods allometry. Biol Rev Camb Philos Soc 81: 259&#226;&#8364;&#8220;291. <br />     C-J, Lee Y-C, Yang P, et al. 2009. Roles trans cis variation      Wittkopp PJ, Haerum BK, Clark AG. 2004. Evolutionary changes cis and <br />     yeast intraspecies evolution gene expression. Mol Biol Evol 26:             trans gene regulation. Nature 430: 85&#226;&#8364;&#8220;88. <br />     2533&#226;&#8364;&#8220;2538.                                                                  Wittkopp PJ, Haerum BK, Clark AG. 2008. Regulatory changes underlying <br /> Tirosh I, Reikhav S, Levy AA, Barkai N. 2009. A yeast hybrid provides insight      expression differences Drosophila species. Nat Genet <br />     evolution gene expression regulation. Science 324: 659&#226;&#8364;&#8220;662.        40: 346&#226;&#8364;&#8220;350. <br /> Tu Y, Stolovitzky G, Klein U. 2002. Quantitative noise analysis gene        Yang Z. 1997. PAML: A program package phylogenetic analysis by <br />     expression microarray experiments. Proc Natl Acad Sci 99: 14031&#226;&#8364;&#8220;14036.         maximum likelihood. Comput Appl Biosci 13: 555&#226;&#8364;&#8220;556. <br /> Van Tassell CP, Smith TPL, Matukumalli LK, Taylor JF, Schnabel RD, Lawley       Yang Z. 2007. PAML 4: Phylogenetic analysis maximum likelihood. Mol <br />     CT, Haudenschild CD, Moore SS, Warren WC, Sonstegard TS. 2008. <br /> </body></html>