<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Extreme polyploidy large bacterium <br /> Jennifer E. Mendell*, Kendall D. Clements&#226;&#8364;&#160;, J. Howard Choat&#226;&#8364;&#161;, Esther R. Angert*&#194;&#167; <br /> *Department Microbiology, Cornell University, Ithaca, NY 14853; &#226;&#8364;&#160;School Biological Sciences, University Auckland, Auckland 1142, New Zealand; <br /> and &#226;&#8364;&#161;School Marine Tropical Biology, James Cook University, Townsville, Queensland 4811, Australia <br />  <br /> Edited James M. Tiedje, Michigan State University, East Lansing, MI, approved March 11, 2008 (received review August 10, 2007) <br />  <br /> Cells rely diffusion metabolites biomolecules.                  lengths 200&#226;&#8364;&#8220;300 &#226;?&#174;m widths 50&#226;&#8364;&#8220;60 &#226;?&#174;m (15) and <br /> Diffusion highly efficient short distances. Although          reproduce solely formation multiple internal offspring <br /> eukaryotic cells broken free diffusion-dictated constraints            (Fig. 1). This reproductive strategy likely evolved endo- <br /> on cell size, bacteria archaea forced remain small.            spore formation (16). A large Epulopiscium contains substan- <br /> Exceptions rule bacterial symbionts             tial DNA arranged periphery the <br /> surgeonfish; Epulopiscium spp. cigar-shaped cells reach               cytoplasm (17). This unusual feature key ability of <br /> lengths excess 600 &#226;?&#174;m. A large Epulopiscium contains thou-               large cells maintain active metabolism despite having <br /> sands times DNA bacterium Escherichia coli,             low surface-<span id='am-8' about='protege:TO' typeof='owl:Thing'>to</span>-volume ratio. To characterize size and <br /> but composition DNA understood. Here,               conformation Epulopiscium sp. type B genome, used <br /> present evidence Epulopiscium contains tens thousands               quantitative PCR enumerate copy number genes in <br /> copies genome. Using quantitative, single-cell PCR assays               individuals DNA extracted populations cells. The <br /> targeting single-copy genes, determined copy num-                 results surveys suggest Epulopiscium highly <br /> ber positively correlated Epulopiscium cell size. Although             polyploid life cycle, individual contains <br /> other bacteria known possess multiple genomes, polyploidy               tens thousands copies genome. <br /> of magnitude observed Epulopiscium unprecedented. The <br /> arrangement genomes cell periphery permit                    Results Discussion <br /> regional responses local stimuli, allowing Epulopiscium             DNA Content Large Small Epulopiscium Cells. Currently, no <br /> maintain unusually large size. Surveys sequences                 Epulopiscium sp. available culture, prevents use <br /> single-copy genes (dnaA, recA, ftsZ) revealed genetic homo-                standard methods (18) assessing composition or <br /> geneity cell consistent small (&#226;&#172;&#8225;1%)             conformation genome. Based 16S rRNA gene se- <br /> the parental DNA transferred generation. The                 quence surveys, Epulopiscium type B populations most <br /> results suggest abundance genome copies                    homogeneous characterized Epulopiscium morphotypes <br /> Epulopiscium allow unstable genetic feature, long                 (13), suited gene-based <br /> mononucleotide tract, essential gene. With evolution              studies presented here. By quantifying DNA <br /> extreme polyploidy large cell size, Epulopiscium acquired <br />                                                                                extracted 5,000 purified large cells mature offspring <br /> some advantages eukaryotic cells. <br />                                                                                (Fig. 1f ), 5,000 cells small, immature offspring (Fig. 1e), <br />                                                                                large cells contain &#226;&#172;&#8225;250 pg DNA, small <br /> dnaA &#229;&#8230;&#169; Epulopiscium &#229;&#8230;&#169; large bacteria &#229;&#8230;&#169; mononucleotide repeat &#229;&#8230;&#169; polyploid <br />                                                                                cells contain &#226;&#172;&#8225;85 pg DNA. In contrast, human diploid cell <br />                                                                                contains 6 pg DNA. The tremendous DNA in <br />  <br /> I t appreciated eukaryotes orders of <br />   magnitude larger known members Bacterial and <br /> Archaeal domains. Eukaryotes broken free constraints <br />                                                                                Epulopiscium likely conformations: (i) a <br />                                                                                copies enormous genome (19), (ii) thousands copies <br />                                                                                small genome (17, 20), (iii) copies complete <br /> on cell size development sophisticated <span id='am-3' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> uptake               genome portions highly amplified. Because only <br /> systems, subcellular compartmentalization, use                    small proportion mother-cell DNA partitioned into <br /> cytoskeleton motor proteins transport vesicles. With                newly formed offspring, unlikely Epulopiscium cells <br /> further advance multicellularity, cell tissue specialization            possess copies enormous genome (15). Instead, <br /> have allowed eukaryotes attain tremendous dimensions (1, 2).                hypothesized Epulopiscium genome comparable to <br /> Until recently (3, 4), bacterial (and archaeal) cells consid-             size bacterial genomes (21), present in <br /> ered simple, displaying little subcellular organization. Although              high numbers. <br /> we know bacterial cells highly organized, <br /> possessing motor cytoskeletal proteins, extensive                 Ploidy Large Small Epulopiscium Cells. The composition of <br /> intracellular membranes instances (5), cells                 DNA individual Epulopiscium cells assessed using <br /> believed rely diffusion access nutrients                    real-time quantitative PCR. Four genes assayed: ftsZ, <br /> metabolically important chemicals. Diffusion coefficients                   dnaA, recA, 16S rRNA gene. The are <br /> small molecules impose time constraints metabolite flux                 generally unlinked, single-copy genes (22&#226;&#8364;&#8220;24), they <br /> (6) require bacterial cells maintain high surface-to-             used represent <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> genome Epulopiscium. Large <br /> volume ratios. As result, cytoplasm far <br /> <span id='am-5' about='protege:FROM' typeof='owl:Thing'>from</span> external environment, exchange rapid enough <br /> to sustain metabolic processes. Most large bacteria fit                   Author contributions: J.E.M. E.R.A. designed research; J.E.M., K.D.C., J.H.C. per- <br /> paradigm maintain cytoplasm; extremely                formed research; J.E.M., K.D.C., J.H.C., E.R.A. wrote paper. <br /> long slender, spherical contain intracellular                The authors declare conflict interest. <br /> vacuole press cytoplasm layer just                This article PNAS Direct Submission. <br /> cytoplasmic membrane (7&#226;&#8364;&#8220;10). In addition, large bacteria                  Data deposition: The sequences reported paper deposited GenBank <br /> contain intracellular mineral inclusions, reduce                 database (accession nos. EF127641, EF127642, EU500234, EU500235, EU500236, and <br /> the volume active cytoplasm (11). Possible exceptions               EU500237). <br /> standard Firmicutes (12).                                 &#194;&#167;To   correspondence addressed. E-mail: era23@cornell.edu. <br />    Our model studying cell biology large bacteria                This article contains supporting information online www.pnas.org/cgi/content/full/ <br /> Epulopiscium sp. type B, occurs intestinal tract               0707522105/DCSupplemental. <br /> the unicornfish Naso tonganus (13, 14). These type B cells attain              &#194;&#169; 2008 The National Academy Sciences USA <br />  <br />  <br />  <br /> 6730 &#226;&#8364;&#8220; 6734 &#229;&#8230;&#169; PNAS &#229;&#8230;&#169; May 6, 2008 &#229;&#8230;&#169; vol. 105 &#229;&#8230;&#169; no. 18                                                              www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.0707522105 <br />                                                                                       Table 2. Gene copy number 156 pg Epulopiscium <br />                              mother cell <br />                                                                                      genomic DNA <br />                                         offspring                             e <br />                                                                                      Gene       Large-cell DNA*       Small-cell DNA*       Control DNA&#226;&#8364;&#160; <br />   d                                                                                  ftsZ       42,600 &#226;&#171;&#190; 5,820         31,900 &#226;&#171;&#190; 1,640            290 <br />                                              b                                       dnaA       38,200 &#226;&#171;&#190; 7,940         36,500 &#226;&#171;&#190; 7,000            120 <br />                                                                                      recA       41,900 &#226;&#171;&#190; 8,330         26,300 &#226;&#171;&#190; 4,300             30 <br />                                                                               f      16S       285,000 &#226;&#171;&#190; 36,000       177,000 &#226;&#171;&#190; 11,200            40 <br />  <br />                                                                                      *Mean &#226;&#171;&#190; SD, n &#226;&#171;&#189; 10. <br />           c                                                                          &#226;&#8364;&#160;Mean negative controls, 150 pg DNA extracted surgeonfish gut <br />  <br />                                                                                       contents Epulopiscium cells, n &#226;&#171;&#189; 3. <br />  <br />  <br />  <br />                                                                                      Epulopiscium cells contain tens thousands copies fully <br /> Fig. 1. Epulopiscium sp. type B life cycle. Offspring production follows           replicated, &#226;&#172;&#8225;3.8 Mb genome. However, marker numbers <br /> circadian cycle. (a) Early day, mother cell possesses small, internal <br />                                                                                      obtained small-cell genomic DNA varied (F &#226;&#171;&#189; 11.82, P &#226;&#171;&#189; <br /> offspring. (b c) Offspring size increases day (b) they <br /> fill mother-cell cytoplasm (c). (d) Finally, &#226;&#8364;&#732;&#226;&#8364;&#732;mature&#226;&#8364;&#8482;&#226;&#8364;&#8482; offspring cells emerge   0.000). These small cells taken early stage their <br /> from mother-cell envelope. Note emergence, cells begin         growth cycle, presumably time active DNA repli- <br /> to develop generation offspring. (e f ) Images DAPI-stained       cation. This supposition supported observation the <br /> cells representing populations small (e) large ( f) Epulopiscium cells    replication origin-linked marker dnaA (30) numerous <br /> used studies. (Scale bar: 50 &#226;?&#174;m.)                                           ftsZ recA. <br />  <br />                                                                                      Cell Volume Genome. Bacillus subtilis (also member the <br /> Epulopiscium cells average possess 50,000 120,000 copies <br />  <br />  <br />  <br />  <br />                                                                                                                                                              MICROBIOLOGY <br />                                                                                      Firmicutes) maintains fairly constant cell-volume-to-genomic- <br /> of markers (Table 1). The genomes many <br />                                                                                      DNA ratio variety exponential growth rates (31). We <br /> Firmicutes (low G&#226;&#171;&#185;C Gram-positive bacteria) multiple (as <br /> many 15) rRNA operons (25). The ribosomal RNA operon                           took advantage natural variation cell size Epulopi- <br /> preferentially amplified eukaryotes (26&#226;&#8364;&#8220;28). Ribosomal                       scium populations investigate genome copy number <br /> RNA synthesis rate-limiting step assembly ribosomes                     varied respect cytoplasmic volume, ratio <br /> in bacteria (29). For reasons, considered                          similar B. subtilis. The genome copy number proxy <br /> rRNA operon good candidate gene amplification                               ftsZ assayed individuals taken natural Epulo- <br /> Epulopiscium. Real-time PCR assays 16S rRNA gene                              piscium type B populations represent extremes of <br /> showed large Epulopiscium cells 240,000 740,000                         offspring development cell size (examples shown Fig. 1 e <br /> copies gene (Table 1). Single-cell PCR amplification                      f ). A linear relationship cell size copy number <br /> the internal transcribed spacer (ITS) 16S 23S                        observed populations (Fig. 2). Cells maintained <br /> rRNA genes showed cells unique                         average ratio genome 1.9 &#226;?&#174;m3 cytoplasm <br /> ITSs, indicative multiple rRNA operons [supporting infor-                         range volumes 28,200 436,000 &#226;?&#174;m3. When <br /> mation (SI) Fig. S1]. These results suggest Epulopiscium                        analyzed separately, means ratios cells two <br /> type B multiple rRNA operons genome; however,                           Epulopiscium populations significantly different (t test, <br /> the rRNA gene copy number individuals large enough,                        P &#226;&#171;&#189; 0.481). In comparison, B. subtilis exponential growth in <br /> relative single-copy markers, indicate substantial                             rich media harbors chromosome 0.7 &#226;?&#174;m3 cytoplasm <br /> amplification operon.                                                        (based data Sharpe et al. ref. 31). Epulopiscium cells <br />    It difficult determine single-cell data                    appear follow rules bacteria link cell <br /> single-copy markers equally present Epulopiscium type B                       growth DNA replication (31&#226;&#8364;&#8220;33); however, the <br /> because cell size gene copy number varied greatly                             populations compared, Epulopiscium maintains larger <br /> individuals given population. Additionally, using cells                       cytoplasmic-volume-to-genome ratio B. subtilis. Further <br /> instead purified genomic DNA real-time PCR assays                        studies needed determine cytoplasmic regions <br /> could introduce factors alter amplification efficiency.                     large bacterial cells functionally equivalent. <br /> We assayed relative gene copy numbers using <br /> purified Epulopiscium genomic DNA. This approach al-                            Factors Leading Extreme Polyploidy. The significance the <br /> lowed rough estimation genome size. Gene copy numbers                       substantial polyploidy genetic expansion recorded Epu- <br /> of single-copy markers statistically similar                       lopiscium considered respect dominant <br /> genomic DNA extracted large cells (one-way ANOVA, F &#226;&#171;&#189;                           attributes bacterium: nutritional biology symbiont <br /> 1.25, P &#226;&#171;&#189; 0.301), markers averaging 40,900 copies 156 pg <br />                                                                                      extraordinarily large size. Genetic amplification seen in <br /> of DNA (Table 2). These results support idea large <br />                                                                                      diverse organisms (bacteria, archaea, protists, fungi, plants, and <br />                                                                                      animals). Amplification provides resources support rapid <br /> Table 1. Gene copy number individual Epulopiscium                                 growth division, cell specialization adaptation, may <br />                                                                                      enhance repair genetic lesions (34&#226;&#8364;&#8220;40). For bacteria, <br /> Gene             Copy number*                       Range&#226;&#8364;&#160;              Control&#226;&#8364;&#161; <br />                                                                                      polyploidy allied metabolic adaptation symbiont <br /> ftsZ                  80,600                  35,800&#226;&#8364;&#8220;198,000              240        differentiation. Buchnera aphidicola, proteobacterial symbi- <br /> dnaA                  48,700                  29,800&#226;&#8364;&#8220;153,000              240        ont aphids, contains average 120 copies chromosome, <br /> recA                 120,000                  60,300&#226;&#8364;&#8220;205,000               70        expansion genetic resources correlates host devel- <br /> 16S                  368,000                 241,000&#226;&#8364;&#8220;737,000              310        opment presumably increased host demand essential <br /> *Median gene copy number, n &#226;&#171;&#189; 10. <br />                                                                                      biomolecules provided Buchnera (41). During differentiation <br /> &#226;&#8364;&#160;Copy number range (minimum &#226;&#8364;&#8220; maximum).                                              symbiotic bacteroids, species rhizobia undergo mod- <br /> &#226;&#8364;&#161;Mean negative controls, surgeonfish gut contents Epulopiscium               est genome proliferation (42). Therefore, factor may <br />  cells, n &#226;&#171;&#189; 3.                                                                       led polyploidy Epulopiscium selection pressure for <br />  <br /> Mendell et al.                                                                                             PNAS &#229;&#8230;&#169; May 6, 2008 &#229;&#8230;&#169; vol. 105 &#229;&#8230;&#169; no. 18 &#229;&#8230;&#169; 6731 <br />                                 500000 <br />  <br />  <br />                                400000 <br />  <br />  <br />                                300000 <br />  <br />  <br />                                200000 <br />  <br />  <br />                                100000 <br />  <br />  <br />                                      0 <br />                                          0          100000           200000           300000          400000           500000 <br />                                                                                           3 <br />                                                                        Cell Volume (&#194;&#181;m ) <br />  <br /> Fig. 2. The relationship gene copy number cytoplasmic volume Epulopiscium sp. type B cells. The pink circles represent data individuals <br /> of small-cell population (n &#226;&#171;&#189; 26), blue diamonds represent data large-cell population (n &#226;&#171;&#189; 31). <br />  <br />  <br />  <br /> the evolution symbiotic relationship contributes host             remains seen mechanisms large <br /> metabolism.                                                                    Epulopiscium cells enhance movement molecules throughout <br />    Alternatively, extreme polyploidy Epulopiscium                bulk central cytoplasm. <br /> more tightly linked cell size. In insect plant cells, an <br /> expansion genomic resources accompanied increase                 Polyploidy Intracellular Genetic Diversity. It coming light <br /> in size (42&#226;&#8364;&#8220;45), size proportionate                  genome duplication subsequent divergence orthologs <br /> ploidy (46). Although advantage large cell size Epulo-               important driving force genome evolution the <br /> piscium spp. determined, features                     generation morphological complexity (44, 52). Polyploidy in <br /> symbiont&#226;&#8364;&#8482;s environment considered. The host feeding                  Epulopiscium allow diversification genome copies <br /> behavior varies substantially daily basis, concentrations            supporting increase cell size. Such simple cellular <br /> of food occurring different points relatively long                   modification important advance the <br /> alimentary tract. Populations large Epulopiscium cells migrate              development contemporary eukaryotic cell. In previous <br /> to distinct regions intestinal tract different times day          study, estimated &#226;&#172;&#8225;1% DNA Epulopiscium <br /> (47), presumably response host digestive processes. We                   sp. type B cell inherited (15). For average type B cell, this <br /> speculate large size allows motile cells better control             DNA comprise 230 genome equivalents. Al- <br /> over position <span id='am-6' about='xsp:length' typeof='owl:Thing'>length</span> intestinal tract            reproduction imposes significant genomic population <br /> host. In addition, alimentary microbial community                   bottleneck, genetic diversity passed the <br /> host complex supports high numbers ciliate                      offspring. <br /> bacterial predators (48). Epulopiscium type B cells appear                     To evaluate gene diversity Epulopiscium, followed an <br /> avoid predation largest ciliates inhabit N.            approach used Mark Welch Meselson (53, 54) study <br /> tonganus intestinal tract (49).                                                bdelloid rotifers. There mounting evidence functional <br />    The biased distribution DNA cytoplasm                      divergence genes alleles asexual, but <br /> Epulopiscium permits functional compartmentalization re-                   highly successful, rotifers (55). For Epulopiscium gene surveys, <br /> gional specialization large cells. A similar peripheral           cloned PCR products single-cell amplification <br /> arrangement nucleoids reported large                    reactions targeting single-copy genes ftsZ, dnaA, recA (SI <br /> bacterium, Thiomargarita namibiensis (50), com-                   Text). As mentioned above, multiple ITS sequences been <br /> position nucleoids determined. Single                   recovered Epulopiscium single-cell PCR amplification <br /> spherical cells Thiomargarita generally 100&#226;&#8364;&#8220;300 &#226;?&#174;m                   (Fig. S1), method allow recovery of <br /> diameter cells large 800 &#226;?&#174;m occur. Each Thiomargarita                abundant variants single-copy genes, present. Alignments of <br /> cell contains large, fluid-filled vacuole, takes &#226;&#172;&#8225;98%               single-copy gene clones revealed consensus gene sequence <br /> of cell volume. This central vacuole confines active                   predominant library; however, did detect se- <br /> cytoplasm shallow, 0.5&#226;&#8364;&#8220; 2-&#226;?&#174;m layer just                      quence variants (SI Text Tables S1, S2, S3). For ftsZ and <br /> cytoplasmic membrane. As bacteria, close associa-                 recA, clones varied consensus 1 2 nt, all <br /> tion DNA cell membrane accommodates transertion                    variants unique (Tables S1 S2). Single-nucleotide <br /> (51), linked processes transcription, translation,                  changes appeared transitions vast majority coded <br /> insertion proteins cytoplasmic membrane. In large                  nonsynonymous amino acid substitutions. The overall fre- <br /> bacteria, genomic copies arrayed cellular periphery                 quency nucleotide differences comparable published <br /> would permit transcription gene disparate locations                  TaqDNA polymerase PCR error rates (56&#226;&#8364;&#8220;58). For dnaA, we <br /> within cell, reducing transit time proteins                    observed similar trends (Table S3). During analysis dnaA, <br /> metabolites site synthesis entry site action. In         however, surprised common, single-nucleotide <br /> way, large bacterium function like microcolony,                 deletion 12 30 dnaA clones (SI Text Table S3). The <br /> different regions cell independently responding local                deletion occurred mononucleotide tract 10 adenines. <br /> stimuli, alleviate pressure remain                  Further investigations (SI Text Fig. S2) indicate least <br /> small sake rapid intracellular diffusive transport. In              deletion variants produced amplifica- <br /> stark contrast Thiomargarita, central cytoplasm Epu-                 tion thermostable polymerases. A high frequency slip- <br /> lopiscium relatively free DNA apparently active. This                page mononucleotide tracts length PCR am- <br /> subcellular arrangement allows rapid growth internal                plification observed (59, 60). Long, <br /> offspring, seen related, smaller (12&#226;&#8364;&#8220;35 &#226;?&#174;m long),                mononucleotide repeats common eukaryotic genomes, <br /> endospore-forming bacterium Metabacterium polyspora (16). It                   rare bacterial archaeal genomes (61&#226;&#8364;&#8220;63). In bacteria, <br />  <br /> 6732 &#229;&#8230;&#169; www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.0707522105                                                                                           Mendell et al. <br />  these highly mutable motifs tend genes                             using Perkin&#226;&#8364;&#8220;Elmer LS50B fluorometer. Genomic DNA quantified <br /> encode variable surface proteins. We unique, long                             using ethidium bromide spot test. For spot test, lambda DNA was <br /> (10 bp) mononucleotide repeat essential bacterial                                used generate serial dilution ranging 50 1 &#226;?&#174;g/ml. Equal volumes <br />                                                                                            DNA (unknowns standards) 2 &#226;?&#174;g/ml solution ethidium bromide <br /> gene. The functional significance dnaA adenine-deletion                            mixed, 10 &#226;?&#174;l mixture spotted Petri dish. The spots <br /> variant frequency expression deter-                              illuminated UV light photographed. The fluorescence intensity <br /> mined.                                                                                     unknown compared fluorescence intensities DNA <br />                                                                                            standards. <br /> Evolutionary Implications Extreme Polyploidy Bacteria. To- <br /> gether, findings support idea Epulopiscium sp. type                           Primer Probe Design Quantitative PCR. Epulopiscium type B dnaA <br /> B cells highly polyploid life cycle. Gene                             (GenBank accession no. EF127641) recA (GenBank accession no. EF127642) <br /> sequence surveys suggest bottleneck imposed Epu-                               genes cloned using approach previously used clone ftsZ (67). <br />                                                                                            TaqMan probes primer sequences Epulopiscium dnaA (probe: 5&#226;&#172;&#732;- <br /> lopiscium genomes reproduction restrict diversifica- <br />                                                                                            TTCTTTCTTTTCCGGCGATAAATTGAATATCATCTATTAG; forward primer: 5&#226;&#172;&#732;- <br /> tion orthologs individual. Nevertheless, extreme                                  GACCAACCTCCTGCCTTCAGAAATAAA; reverse primer: 5&#226;&#172;&#732;-GTGTTAAATGTAT- <br /> polyploidy allow Epulopiscium harbor unstable genetic                               GGAAAAATTCTTCTTGT), ftsZ (probe: 5&#226;&#172;&#732;-CACAGGTGACTCGACACTTGCAATC; <br /> features, mononucleotide tracts, essential genes,                           forward primer: 5&#226;&#172;&#732;-ATTAAAGGTGCAGGTGGCGTACT; reverse primer: 5&#226;&#172;&#732;- <br /> without detriment.                                                                         GCTAGCTCCCGCAT TCAACT), recA (probe: 5&#226;&#172;&#732;-TTAAGAATAAAATTGCTCCTC- <br />    The functional dichotomy &#226;&#8364;&#732;&#226;&#8364;&#732;somatic&#226;&#8364;&#8482;&#226;&#8364;&#8482; &#226;&#8364;&#732;&#226;&#8364;&#732;germ-line&#226;&#8364;&#8482;&#226;&#8364;&#8482;                               CATTTAAACAAGCAG; forward primer: 5&#226;&#172;&#732;-TGCGACAGAAATAATCGGCAG- <br /> genomes enormous highly polyploid bacterium                                  CAAAA; reverse primer: 5&#226;&#172;&#732;-AGAAGAAATTCCTTCGCCATAGATAA), the <br /> represents unique intermediate &#226;&#8364;&#732;&#226;&#8364;&#732;typical&#226;&#8364;&#8482;&#226;&#8364;&#8482; asex-                             16S rRNA gene (probe: 5&#226;&#172;&#732;-CCATGCCGCCTACACACCCTTTACA; forward primer: <br />                                                                                            5&#226;&#172;&#732;-ATTAAAGGTGCAGGTGGCGTACT; reverse primer: 5&#226;&#172;&#732;-GCTAGCTCCCGCAT- <br /> ual, single-celled microorganism multicellular organism. <br />                                                                                            TCAACT) used real-time PCR designed using Primer Express (Ap- <br /> The genetic material successful, solitary bacterium                                plied Biosystems). TaqMan probes used real-time PCR assays 5&#226;&#172;&#732; <br /> replicated faithfully passed offspring. For mul-                             end-labeled reporter dye 6-carboxyfluorescein (FAM) 3&#226;&#172;&#732;-labeled <br /> ticellular organisms, germ line inherited;                             quencher dye carboxytetramethylrhodamine (TAMRA). <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                  MICROBIOLOGY <br /> waste genetic resources somatic cells offset the <br /> diversification cellular function, commonly leads                              TaqMan Quantitative PCR Assays. Primer sets tested standard PCR <br /> increased size, enhanced access resources, improved                                 amplifications using Epulopiscium cells, microbial community DNA extracted <br /> metabolic capacity. Compared solitary existence, co-                                surgeonfish intestinal contents, genomic DNA other <br /> lonial microbes (e.g., actinobacteria) populations                              Firmicutes. A proteinase K solution (as above) irradiated 3 min using <br />                                                                                            UV transilluminator (FisherBiotech) aliquoted tube a <br /> cooperative microorganisms (e.g., cellular slime molds, myx- <br />                                                                                            96-well PCR plate. One washed Epulopiscium added tube and <br /> obacteria) benefit improved metabolic potential                                   incubated 50&#194;&#176;C 1 h. The plate heated, 95&#194;&#176;C 15 min, inactivate <br /> perhaps better dispersal cost genomes cells                          proteinase K. TaqMan assay reaction mixtures prepared ice and <br /> play supporting role (64, 65). The enormous, polyploid                                   contained 1&#226;&#171;&#187; TaqMan Universal Master Mix (ABI), 900 nM appropriate <br /> Epulopiscium cell converged advantages social                                forward reverse primers, 200 nM appropriate fluorogenic <br /> microbes additional benefits (exceptional motility,                               probe. To determine copy number genes Epulopiscium genomic DNA, <br /> enhanced resistance predation) normally large                                  DNA extracts large- small-cell populations diluted 1:10 and <br /> eukaryotic microbes multicellular organisms.                                            1:100 TE. The concentrations dilutions determined using the <br />                                                                                            PicoGreen assay described above, 1 &#226;?&#174;l diluted Epulopiscium DNA was <br />                                                                                            used assays. The small-cell DNA concentration adjusted 156 pg/&#226;?&#174;l <br /> Methods <br />                                                                                            match concentration large-cell DNA sample. For gene quantifi- <br /> Epulopiscium Collection. Epulopiscium sp. morphotype B cells obtained                 cation, 1 &#226;?&#174;l DNA added PCR plate containing TaqMan <br /> from N. tonganus, collected spearfishing reefs Lizard Island,                 reaction mixture. Each genomic DNA sample run triplicate each <br /> Australia. Sections gut removed, intestinal contents                  gene assay. Quantification standards generated serial dilutions of <br /> fixed 80% ethanol stored &#226;&#171;&#186;20&#194;&#176;C. Individual Epulopiscium cells                    plasmid DNA (2.0 &#226;&#171;&#187; 107 copies 2.0 &#226;&#171;&#187; 102 copies &#226;?&#174;l) containing dnaA, ftsZ, <br /> were collected gut contents using standard Gilson micropipettor                  recA, 16S rRNA gene cloned Epulopiscium type B. Standards were <br /> and dissecting microscope (Nikon SMZ-U). Cells transferred                     run duplicate, template controls run triplicate assay. <br /> times sterile ethanol wash buffer [80% ethanol, 145 mM NaCl, 50                    Thermal cycling conditions follows: 2 min 50&#194;&#176;C, 10 min 95&#194;&#176;C <br /> mM Tris&#228;&#161;&#160;HCl (pH 8.0), 0.05% Igepal] rinsed sterile deionized water.                 followed 40 cycles 15 s 95&#194;&#176;C 1 min 60&#194;&#176;C. Data compiled and <br />                                                                                            analyzed using Sequence Detection Software, version 1.3 (ABI). <br /> DNA Extraction Quantification. DNA extracted 5,000 handpicked, <br /> washed cells. DNA extraction quantification protocols based                    Cell Volume Gene Copy Number Comparisons. Images Epulopiscium cells <br /> standard procedures (66). Epulopiscium cells lysed incubation                   acquired Cooke SensiCam CCD camera Olympus BX61 <br /> proteinase K (100 &#226;?&#174;g/ml 10 mM Tris, pH 8.0) 50&#194;&#176;C 1 h followed             microscope equipped LCPlanF1 &#226;&#171;&#187;40 objective. Using Slidebook soft- <br /> rounds freeze&#226;&#8364;&#8220;thaw. Cell lysate extracted twice phenol/                        ware (calibrated stage micrometer), cell length width were <br /> chloroform/isoamyl alcohol (25:24:1), nucleic acids precipitated              determined. The cell volume calculated using formula prolate <br /> with 0.3 M sodium acetate ethanol. The pellet rinsed 70%                      ellipsoid. After image acquisition, cell transferred a <br /> ethanol air-dried. The pellet dissolved sterile water treated               96-well plate processed quantitative PCR described above. Statis- <br /> with RNase A (10 &#226;?&#174;g/ml) 37&#194;&#176;C 30 min.                                                tical analyses (t test ANOVA) performed SAS software, version <br />    PicoGreen assays quantify genomic DNA performed follows.                     9.1 SAS Windows. <br /> DNA bacteriophage lambda diluted TE [10 mM Tris (pH 8.0), 1 mM <br /> EDTA] generate dilution series ranging 500 10 ng/ml. Genomic <br /> DNA Epulopiscium diluted 1:5,000 TE. Aliquots (50 &#226;?&#174;l)                   ACKNOWLEDGMENTS. We thank staff directors Lizard Island <br />                                                                                            Research Station support; Will Robbins David Raubenheimer help <br /> standards genomic DNA dispensed triplicate wells                     field; Rebekah Ward, David Miller, John Helmann, anonymous <br /> microtiter plate. PicoGreen (Molecular Probes) prepared according               reviewers comments manuscript; Simona Despa statistical <br /> manufacturer&#226;&#8364;&#8482;s instructions, 50-&#226;?&#174;l aliquots mixed DNA                    consultation. Funding provided National Science Foundation Molec- <br /> solutions microtiter plate well. Relative fluorescence determined              ular Cellular Biology Program Grant MCB 0721583. <br />  <br />  <br />  1. Bonner JT (2004) Perspective: The size-complexity rule. Evol Int J Org Evol 58:1883&#226;&#8364;&#8220;    3. Losick R, Shapiro L (1999) Changing views nature bacterial cell: From <br />     1890.                                                                                      biochemistry cytology. J Bacteriol 181:4143&#226;&#8364;&#8220; 4145. <br />  2. West GB, Brown JH (2005) The origin allometric scaling laws biology          4. Shapiro L, Losick R (1997) Protein localization cell fate bacteria. Science 276:712&#226;&#8364;&#8220;718. <br />     genomes ecosystems: Toward quantitative unifying theory biological structure    5. Shively, JM (2006) Prokaryote complex intracellular structures: Descriptions dis- <br />     organization. J Exp Biol 208:1575&#226;&#8364;&#8220;1592.                                                coveries. Microbiology Monographs, ed Shively JM (Springer, Berlin), Vol 2, pp 3&#226;&#8364;&#8220;22. <br />  <br />  <br />  <br /> Mendell et al.                                                                                                            PNAS &#229;&#8230;&#169; May 6, 2008 &#229;&#8230;&#169; vol. 105 &#229;&#8230;&#169; no. 18 &#229;&#8230;&#169; 6733 <br />   6. Berg HC (1993) Random Walks Biology (Princeton Univ Press, Princeton).                   39. Romero D, Palacios R (1997) Gene amplification genomic plasticity prokaryotes. <br />  7. Blakemore RP, Canale-Parola E (1973) Morphological ecological characteristics            Annu Rev Genet 31:91&#226;&#8364;&#8220;111. <br />     Spirochaeta plicatilis. Arch Microbiol 89:273&#226;&#8364;&#8220;289.                                          40. Rosenberg SM, Hastings PJ (2004) Adaptive <span id='am-9' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-10' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span>point mutation</span> adaptive amplification <br />  8. Delaporta B (1964) Descriptive study bacteria large size. Ann Institut Pasteur       pathways Escherichia coli Lac system: Stress responses producing genetic change. <br />     107:845&#226;&#8364;&#8220; 862.                                                                                   J Bacteriol 186:4838 &#226;&#8364;&#8220; 4843. <br />  9. Gundersen JK, Jorgensen BB, Larsen E, Jannasch HW (1992) Mats giant sulphur              41. Komaki K, Ishikawa H (2000) Genomic copy number intracellular bacterial symbionts <br />     bacteria deep-sea sediments fluctuating hydrothermal flow. Nature                     aphids varies response developmental stage morph host. Insect <br />     360:454 &#226;&#8364;&#8220; 456. <br />                                                                                                     Biochem Mol Biol 30:253&#226;&#8364;&#8220;258. <br /> 10. Schulz HN, et al. (1999) Dense populations giant sulfur bacterium Namibian shelf <br />                                                                                                 42. Mergaert P, et al. (2006) Eukaryotic control bacterial cell cycle differentiation <br />     sediments. Science 284:493&#226;&#8364;&#8220; 495. <br />                                                                                                     Rhizobium-legume symbiosis. Proc Natl Acad Sci USA 103:5230 &#226;&#8364;&#8220;5235. <br /> 11. Head IM, Gray ND, Babenzien H, Glo&#194;&#168;ckner FO (2000) Uncultured giant sulfur bacteria <br />                                                                                                 43. Hanson KK, Kelley AC, Bienz M (2005) Loss Drosophila borealin causes polyploidy, <br />     genus Achromatium. FEMS Microbiol Ecol 33:171&#226;&#8364;&#8220;180. <br /> 12. Angert ER (2006) The enigmatic cytoarchitechture Epulopiscium spp. Microbiology              delayed apoptosis, abnormal tissue development. Development 132:4777&#226;&#8364;&#8220; 4787. <br />     Monographs, ed Shively JM (Springer, Berlin), Vol 2, pp 285&#226;&#8364;&#8220;301.                            44. Maere S, et al. (2005) Modeling gene genome duplications eukaryotes. Proc Natl <br /> 13. Angert ER, Clements KD, Pace NR (1993) The largest bacterium. Nature 362:239 &#226;&#8364;&#8220;241.              Acad Sci USA 102:5454 &#226;&#8364;&#8220;5459. <br /> 14. Clements KD, Sutton DC, Choat JH (1989) Occurrence characteristics unusual           45. Saucedo LJ, Edgar BA (2002) Why size matters: Altering cell size. Curr Opin Genet Dev <br />     protistan symbionts surgeonfishes Acanthuridae Great Barrier Reef Aus-              12:565&#226;&#8364;&#8220;571. <br />     tralia. Marine Biol 102:403&#226;&#8364;&#8220; 412.                                                           46. Sugimoto-Shirasu K, Roberts K (2003) Big up: Endoreduplication cell-size control <br /> 15. Angert ER, Clements KD (2004) Initiation intracellular offspring Epulopiscium. Mol        plants. Curr Opin Plant Biol 6:544 &#226;&#8364;&#8220;553. <br />     Microbiol 51:827&#226;&#8364;&#8220; 835.                                                                      47. Fishelson L, Montgomery WL, Myrberg AA (1985) A unique symbiosis gut a <br /> 16. Angert ER (2005) Alternatives binary fission bacteria. Nat Rev Microbiol 3:214 &#226;&#8364;&#8220;          tropical herbivorous surgeonfish (Acanthuridae: Teleostei) Red Sea. Science <br />     224.                                                                                            229:49 &#226;&#8364;&#8220;51. <br /> 17. Robinow C, Angert ER (1998) Nucleoids coated vesicles &#226;&#8364;&#732;&#226;&#8364;&#732;Epulopiscium&#226;&#8364;&#8482;&#226;&#8364;&#8482; spp.          48. Grim JN (1993) Description somatic kineties vestibular organization Balan- <br />     Arch Microbiol 170:227&#226;&#8364;&#8220;235.                                                                     tidium jocularum sp. n., possible implications class Litostomatea genus <br /> 18. Herschleb J, Ananiev G, Schwartz DC (2007) Pulsed-field gel electrophoresis. Nat Protoc         Balantidium. Acta Protozool 32:37&#226;&#8364;&#8220; 45. <br />     2:677&#226;&#8364;&#8220; 684. <br />                                                                                                 49. Grim JN (2006) Food vacuole contents ciliate, Balantidium jocularum (Balanti- <br /> 19. Bresler V, Montgomery WL, Fishelson L, Pollak PE (1998) Gigantism bacterium, <br />                                                                                                     didae), symbiont intestine surgeonfish, Naso tonganus (Acanthuridae). <br />     Epulopiscium fishelsoni, correlates complex patterns arrangement, quantity, <br />                                                                                                     J Eukaryotic Microbiol 53:269 &#226;&#8364;&#8220;274. <br />     segregation DNA. J Bacteriol 180:5601&#226;&#8364;&#8220;5611. <br />                                                                                                 50. Schulz HN (2006) The genus Thiomargarita. The Prokaryotes, eds Dworkin, M, Falkow <br /> 20. Bresler V, Fishelson L (2003) Polyploidy polyteny gigantic eubacterium <br />     Epulopiscium fishelsoni. Mar Biol 143:17&#226;&#8364;&#8220;21.                                                    S, Rosenberg E, Schleifer KH, Stackebrandt E (Springer, New York), Vol 6, pp 1156 &#226;&#8364;&#8220; <br /> 21. Liolios K, Tavernarakis N, Hugenholtz P, Kyrpides NC (2006) The Genomes On Line                 1163. <br />     Database (GOLD) version 2: A monitor genome projects worldwide. Nucleic Acids            51. Norris V, Turnock G, Sigee D (1996) The Escherichia coli enzoskeleton. Mol Microbiol <br />     Res 34:332&#226;&#8364;&#8220;334.                                                                                 19:197&#226;&#8364;&#8220;204. <br /> 22. Baldo L, Bordenstein S, Wernegreen JJ, Werren JH (2006) Widespread recombination            52. Infante JJ, Dombek KM, Rebordinos L, Cantoral JM, Young ET (2003) Genomewide <br />     Wolbachia genomes. Mol Biol Evol 23:437&#226;&#8364;&#8220; 449.                                        amplifications caused chromosomal rearrangements play major role adap- <br /> 23. Eisen JA (1995) The RecA protein model molecule molecular systematic studies           tive evolution natural yeast. Genetics 165:1745&#226;&#8364;&#8220;1759. <br />     bacteria: Comparison trees RecAs 16S rRNAs species. J Mol        53. Mark Welch D, Meselson M (2000) Evidence evolution bdelloid rotifers <br />     Evol 41:1105&#226;&#8364;&#8220;1123.                                                                              sexual reproduction genetic exchange. Science 288:1211&#226;&#8364;&#8220;1215. <br /> 24. Karlin S, Weinstock GM, Brendel V (1995) Bacterial classifications derived recA        54. Mark Welch D, Meselson MS (2001) Rates nucleotide substitution sexual and <br />     protein sequence comparisons. J Bacteriol 177:6881&#226;&#8364;&#8220; 6893.                                       anciently asexual rotifers. Proc Natl Acad Sci USA 98:6720 &#226;&#8364;&#8220; 6724. <br /> 25. Rainey FA, Ward-Rainey NL, Janssen PH, Hippe H, Stackebrandt E (1996) Clost <br /> </body></html>