<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The Histone H3K36 Methyltransferase MES-4 Acts <br /> Epigenetically Transmit Memory Germline Gene <br /> Expression Progeny <br /> Andreas Rechtsteiner1., Sevinc Ercan2., Teruaki Takasaki1., Taryn M. Phippen1, Thea A. Egelhofer1, <br /> Wenchao Wang1&#194;&#164;, Hiroshi Kimura3, Jason D. Lieb2*, Susan Strome1* <br /> 1 Department Molecular, Cell, Developmental Biology, University California Santa Cruz, Santa Cruz, California, United States America, 2 Department Biology, <br /> Carolina Center Genome Sciences Lineberger Comprehensive Cancer Center, University North Carolina Chapel Hill, Chapel Hill, North Carolina, United States <br /> of America, 3 Graduate School Frontier Biosciences, Osaka University, Suita, Osaka, Japan <br />  <br />  <br />  <br />      <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />      Methylation histone H3K36 higher eukaryotes mediated multiple methyltransferases. Set2-related H3K36 <br />      methyltransferases targeted genes association RNA Polymerase II involved preventing aberrant <br />      transcription initiation body genes. The targeting roles NSD family mammalian H3K36 <br />      methyltransferases, known involved human developmental disorders oncogenesis, known. We used <br />      genome-wide chromatin immunoprecipitation (ChIP) investigate targeting roles Caenorhabditis elegans <br />      NSD homolog MES-4, maternally provided progeny required <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span>survival</span> nascent germ cells. ChIP <br />      analysis early C. elegans embryos revealed that, consistent immunostaining results, MES-4 binding sites are <br />      concentrated autosomes leftmost ,2% (300 kb) X chromosome. MES-4 overlies coding regions of <br />      approximately 5,000 genes, modest elevation 59 regions gene bodies. Although MES-4 generally found <br />      Pol II-bound genes, analysis gene sets different temporal-spatial patterns expression revealed Pol II <br />      association genes necessary sufficient recruit MES-4. In early embryos, MES-4 associates genes <br />      previously expressed maternal germ line, interaction does require continued association Pol II <br />      loci. Conversely, Pol II association genes newly expressed embryos does lead recruitment MES-4 <br />      genes. These findings suggest MES-4, related mammalian NSD proteins, provide an <br />      epigenetic function H3K36 methylation novel likely unrelated ongoing transcription. We propose <br />      MES-4 transmits memory gene expression parental germ line offspring memory role is <br />      critical PGCs execute proper germline program. <br />  <br />   Citation: Rechtsteiner A, Ercan S, Takasaki T, Phippen TM, Egelhofer TA, et al. (2010) The Histone H3K36 Methyltransferase MES-4 Acts Epigenetically Transmit <br />   Memory Germline Gene Expression Progeny. PLoS Genet 6(9): e1001091. doi:10.1371/journal.pgen.1001091 <br />   Editor: Wolf Reik, The Babraham Institute, United Kingdom <br />   Received March 9, 2010; Accepted July 26, 2010; Published September 2, 2010 <br />   Copyright: &#195;&#376; 2010 Rechtsteiner et al. This open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: This research funded NIH grant GM34059 SS NIH-NHGRI modENCODE grant U01 HG004270 JDL. The funders role study <br />   design, data collection analysis, decision publish, preparation manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: strome@biology.ucsc.edu (SS); jlieb@bio.unc.edu (JDL) <br />   . These authors contributed equally work. <br />   &#194;&#164; Current address: Department Pediatric Oncology, Dana Farber Cancer Institute, Boston, Massachusetts, United States America <br />  <br />  <br />  <br />  <br /> Introduction                                                                            H3K36 methylation carried SET domain-containing <br />                                                                                         protein Set2 [9]. In complex eukaryotes, H3K36 methyla- <br />    Regulation gene expression DNA packaging                             tion carried groups enzymes. The group <br /> chromatin emerged important theme development                              includes Set2-related proteins, called MET-1 C. elegans, <br /> (reviewed [1]). Chromatin consists nucleosomal units                           Hypb/Set2 fruit flies, HYPB/Setd2 mammals, [10&#226;&#8364;&#8220;13]. <br /> 147 bp segments DNA wrapped histone octamers                                  The second group includes MES-4-related proteins called MES-4 <br /> composed H2A, H2B, H3, H4 [1&#226;&#8364;&#8220;3]. Post-translational                              C. elegans, dMES-4 fruit flies, NSD1 (for nuclear receptor- <br /> modification histones methylation, acetylation addition                      binding SET domain protein 1), NSD2/WHSC1/MMSET, and <br /> of covalent marks constitutes important mechanism                           NSD3/WHSC1L1 mammals [11,14&#226;&#8364;&#8220;16]. Mutations the <br /> regulate chromatin structure, DNA accessibility, recruitment                        human MES-4/NSD genes underlie developmental disorders and <br /> of regulatory factors transcription [4,5]. Some modifica-                        various human malignancies [15&#226;&#8364;&#8220;17]. <br /> tions, H3K27 methylation, additionally serve epigenetic                            In C. elegans, MES-4 histone methyltransferase (HMT) is <br /> roles propagating particular chromatin states gene                               essential germ cell viability. The genetics mes-4 demonstrate <br /> expression patterns mother daughter cells [e.g. 6,7].                           MES-4 serves transgenerational (mother offspring) role. <br />    This paper focuses MES-4, C. elegans enzyme                                Maternal mes-4(+) enables homozygous mes-4/mes-4 progeny <br /> methylates histone H3 lysine 36 (H3K36me) [8]. In yeast,                         form fully functional germ line, absence maternal mes- <br />  <br />  <br />         PLoS Genetics | www.plosgenetics.org                                        1                         September 2010 | Volume 6 | Issue 9 | e1001091 <br />                                                                                                           Epigenetic Role H3K36 Methylation <br />  <br />  <br />  <br />   Author Summary                                                           recruitment different Set2. MES-4 associates and <br />                                                                            maintains methylation germline-expressed genes cell <br />   Germ cells transmit genome generation                    division transcription-independent manner, establishing a <br />   next. The identity immortality germ cells                 novel epigenetic role H3K36 methylation. <br />   crucial perpetuation species, mechanisms <br />   regulate properties remain elusive. In C.elegans,             Results <br />   histone methyltransferase MES-4 required survival <br />   primordial germ cells. MES-4 methylates histone H3                MES-4, MET-1, required early-embryo <br />   lysine 36 (H3K36), modification previously linked                maintenance H3K36 trimethylation absence of <br />   transcription elongation involved preventing <br />   aberrant transcription initiation body genes.              RNA Polymerase II <br />   Surprisingly, genome-wide analysis MES-4 binding                     Previous studies demonstrated MES-4 generates <br />   sites C. elegans embryos revealed MES-4 capable               H3K36me2 adult germ lines embryos [8], Set2 <br />   associating genes expressed germ                ortholog MET-1 generates H3K36me3 embryos [10]. Our <br />   line parent worms longer actively                immunostaining experiments demonstrate MES-4 contrib- <br />   transcribed embryos. <span id='am-4' about='protege:TO' typeof='owl:Thing'>To</span> knowledge,              utes significantly H3K36me3 well, H3K36me3 signal <br />   example transcription-uncoupled H3K36 methylation.                    abolished double mutant germ lines embryos <br />   We suggest MES-4-generated H3K36 methylation                        lacking MES-4 MET-1 (Figure 1A&#226;&#8364;&#8220;1D; germline data <br />   serves &#226;&#8364;&#732;&#226;&#8364;&#732;epigenetic role,&#226;&#8364;&#8482;&#226;&#8364;&#8482; marking germline-ex-                   shown). To investigate roles served two <br />   pressed genes carrying memory gene                         H3K36me3 HMTs, tested dependence HMT <br />   expression generation germ cells next.                RNA Pol II. <br />                                                                               Previous studies suggested Pol II-independent component of <br />                                                                            MES-4 function. MES-4-generated H3K36me2 immunofluores- <br /> 4(+) severely impairs ability primordial germ cells             cence signal embryos persists RNAi depletion AMA-1, <br /> (PGCs) progeny proliferate causes deteriorate            large subunit Pol II [8]. In study tested the <br /> early [18]. Based immunostaining germ lines embryos,             transcription dependence MES-4 MET-1 embryos <br /> MES-4 H3K36me2 marks generated MES-4                        treating wild-type, mes-4 mutant, met-1 mutant worms with <br /> dramatically concentrated autosomes absent             RNAi ama-1 analyzing 80&#226;&#8364;&#8220;110-cell embryos for <br /> but left tip X chromosomes [8,19]. Surprisingly,                H3K36me3. The persistence H3K36me3 depletion <br /> microarray analysis adult germ lines fertile mes-4/mes-4           AMA-1 wild-type suggests H3K36me3 least <br /> mothers revealed up-regulation numerous (345) X-linked genes,           partly Pol II-independent (Figure 1E). H3K36me3 staining was <br /> along mis-regulation autosomal genes (155 up- 115              undetectable simultaneous loss MES-4 AMA-1, but <br /> down-regulated) (data [8]; unpublished results) suggest-          easily detected simultaneous loss MET-1 AMA-1 <br /> ing MES-4 participates silencing X chromosomes              (Figure 1F 1G). These results consistent MET-1 <br /> wild-type adult germ lines [8,20,21]. Learning MES-4               mediating Pol II-dependent H3K36me3, indicate MES-4 <br /> H3K36 methylation promote PGC survival hinges identifying <br />                                                                            mediates Pol II-independent H3K36me3. There are <br /> the targets MES-4 binding gene level elucidating the <br />                                                                            precedents Pol II-uncoupled H3K36me3. To further <br /> role MES-4-generated H3K36 methyl marks. <br />                                                                            investigate Pol II-independent binding methylation of <br />    Studies yeast Set2 established paradigm binding         targets MES-4, help understand activity <br /> and function H3K36 HMTs. Yeast Set2 associates &#226;&#8364;&#732;&#226;&#8364;&#732;SRI            promotes viability PGCs, analyzed MES-4 binding <br /> domain&#226;&#8364;&#8482;&#226;&#8364;&#8482; C-terminal domain RNA Polymerase II (Pol              genome high resolution using ChIP-chip. <br /> II) elongation phase transcription methylates <br /> H3K36 body genes, predominantly 39 coding <br /> region [22&#226;&#8364;&#8220;26]. Set2-catalyzed H3K36me marks recognized                Sites MES-4 binding distributed five <br /> by chromodomain Eaf3 plant homeo domain                     autosomes under-represented X <br /> (PHD) Rco1, complex Rpd3 enzyme                 chromosome <br /> deacetylates histones. Delivery Rpd3 body genes                  To determine distribution MES-4 genome by <br /> suppresses spurious transcription initiation coding             ChIP-chip, used rabbit anti-MES-4 antibody immunopre- <br /> region [27&#226;&#8364;&#8220;29]. Studies X-chromosome dosage compensation                cipitate endogenous MES-4 (Figure 2A Figure S2A), an <br /> in Drosophila revealed role H3K36 methylation          alternative strategy used mouse anti-FLAG antibody to <br /> mediating spreading dosage compensation complex                immunoprecipitate FLAG-tagged MES-4 (Figure S2C). Tagged <br /> sites initial recruitment [12,30].                                      MES-4 judged functional criteria: autosomal <br />    To investigate targeting roles MES-4 C. elegans,       concentration MES-4::GFP::FLAG resembles endoge- <br /> employed ChIP-chip determine genome-wide distribution            nous MES-4 (Figure S2), transgene rescues mes-4(bn73) <br /> high resolution MES-4, H3K36me2, H3K36me3, Pol II                mutant worms (see Text S1). ChIP performed extracts <br /> extracts early-stage embryos. Our analysis revealed              early embryos reasons. First, MES-4 levels appear <br /> MES-4 accumulates bodies ,20% genes,               relatively low maternal germ line dramatically increase <br /> the majority MES-4-bound genes autosomes            fertilization (Y. Fong S. Strome, unpublished result), <br /> the left tip X. Comparing MES-4 Pol II association         suggesting early embryogenesis period important for <br /> classes genes different temporal-spatial patterns               MES-4 function ChIP signals robust <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> expression revealed MES-4 resides genes expressed               early embryos isolated germline tissue. Second, MES-4 <br /> the maternal germ line genes newly expressed early              binding concentrated autosomes stages analyzed, <br /> embryos. These findings, especially observation genes             including early embryos, suggesting mechanisms that <br /> specifically expressed adult germ line bound MES-4           control selective recruitment autosomes operational in <br /> but Pol II early embryos, point mode MES-4                  early embryos [19]. Third, early embryos harvested in <br />  <br />  <br />       PLoS Genetics | www.plosgenetics.org                             2                    September 2010 | Volume 6 | Issue 9 | e1001091 <br />                                                                                                                   Epigenetic Role H3K36 Methylation <br />  <br />  <br />                                                                                    As observed immunofluorescence staining chromo- <br />                                                                                 somes, MES-4 ChIP peaks concentrated autosomes and <br />                                                                                 left tip X. The peak-finding algorithm ChIPOTLe [31] <br />                                                                                 (http://sourceforge.net/projects/chipotle-2/) used identify <br />                                                                                 5391 genomic regions significant MES-4 enrichment. 5291 of <br />                                                                                 5391 MES-4 binding sites distributed five <br />                                                                                 autosomes. Only 100 5391 MES-4 sites (1.9%) located on <br />                                                                                 X chromosome, contrast 953 (17.7%) expected the <br />                                                                                 X MES-4 distributed uniformly chromosomes (p- <br />                                                                                 value,102131) (Figure 2B Figure S2D). 25% MES-4 binding <br />                                                                                 sites X located leftmost 300 kb (1.7%) X. <br />                                                                                 The density binding sites left tip X (0.08 sites/kb) is <br />                                                                                 similar autosomes (0.06 sites/kb). The remaining <br />                                                                                 ,17.4 Mb X chromosome extremely low density of <br />                                                                                 MES-4 sites (0.004 sites/kb), apparently low detect <br />                                                                                 immunofluorescence. The far left tip X, resembles the <br />                                                                                 autosomes MES-4 binding density, marked dot of <br />                                                                                 MES-4 immunostaining [8] (Figure 2). <br />  <br />                                                                                 MES-4 associates coding regions expressed <br />                                                                                 genes <br />                                                                                    MES-4-bound regions generally overlie coding regions of <br />                                                                                 single genes sets adjacent genes (Figure 3). Of 5391 <br />                                                                                 MES-4 binding sites, 5217 overlap coding region 1 <br />                                                                                 gene, 143 overlap 1 kb upstream (64) downstream (79) at <br />                                                                                 1 gene, 31 reside intergenic regions (Table S1A). <br />                                                                                 Conversely, 22,241 annotated genes WormBase, 4400 <br />                                                                                 overlap MES-4-bound site additional 98 overlap their <br />                                                                                 1 kb upstream downstream region. Thus, large majority <br />                                                                                 (.96%) MES-4 overlies genic regions, approximately 1 5 <br />                                                                                 genes associated MES-4. <br />                                                                                    To investigate MES-4 generally accumulates on <br />                                                                                 expressed silent genes, compared distribution MES- <br />                                                                                 4 early embryos Pol II genome. ChIP <br />                                                                                 analysis Pol II performed using 8WG16 monoclonal <br />                                                                                 antibody, detects unphosphorylated phos- <br />                                                                                 phoryated forms Pol II [32]. Most genes bound MES-4 <br />                                                                                 associated Pol II. This illustrated viewing <br />                                                                                 individual genes (Figure 3) plotting MES-4 ChIP signal <br />                                                                                 versus Pol II ChIP signal genes (Figure S4C). However, as <br />                                                                                 discussed below, certain gene classes specific temporal or <br />                                                                                 spatial expression patterns high MES-4 Pol II high <br />                                                                                 Pol II MES-4, indicating Pol II necessary <br />                                                                                 sufficient recruit MES-4. <br />                                                                                    Viewing distribution MES-4 Pol II gene bodies <br /> Figure 1. Two C. elegans HMTs, MES-4 MET-1, perform H3K36                   stratified transcript accumulation early embryos revealed that <br /> trimethylation display different dependence Pol II.                      MES-4 levels highest highly transcribed genes and <br /> H3K36me3 levels reduced relative wild type (A) mes-4 mutant           MES-4 overlies gene bodies (Figure 4). Genes aligned their <br /> (B), met-1 mutant (C), ama-1(RNAi) (E) ,80&#226;&#8364;&#8220;110-cell embryos.                transcript start sites (TSS) end sites (TES) binned 5 <br /> H3K36me3 undetectable mes-4; met-1 (D) mes-4; ama-1(RNAi)             equally sized groups based microarray measurements of <br /> (F), present met-1; ama-1(RNAi) (G) embryos. Embryos were <br />                                                                                 transcript levels extracts used ChIP. Average MES-4 <br /> stained parallel images acquired processed identically. See <br /> Figure S1 specificity H3K36me3 antibody.                                 Pol II profiles genes bin plotted starting <br /> doi:10.1371/journal.pgen.1001091.g001                                           1 kb upstream 1.5 kb downstream TSS from <br />                                                                                 1.5 kb upstream 1 kb downstream TES (Figure 4). Genes <br /> quantities sufficient ChIP, pure germline tissue currently            highest RNA level highest MES-4 binding, followed <br /> cannot. ChIP performed triplicate sheared chromatin                 progressively lower levels MES-4 genes lower RNA <br /> prepared formaldehyde-fixed early embryos (see Materials                   levels. In highly expressed genes, MES-4 levels rise across <br /> and Methods). As controls, performed ChIP using Protein A                    TSS, reaching maximum value ,500 bp gene body, <br /> and Protein G beads beads coated antibodies                  gradually decline drop TES. Pol II shows a <br /> were specific C. elegans proteins. Endogenous MES-4                  different profile MES-4. Pol II levels peak TSS again <br /> tagged MES-4 displayed remarkably concordant ChIP distribu-                     39 TES (Figure 4). Although MES-4 Pol II accumulate on <br /> tions, differed significantly controls (Figure 3            genes, 39 peak Pol II compared 39 drop <br /> and Figure S3). Here, focus ChIP analysis endogenous                   MES-4 suggests MES-4 does track Pol II it <br /> MES-4.                                                                          tracks subpopulation Pol II. <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                                 3                      September 2010 | Volume 6 | Issue 9 | e1001091 <br />                                                                                                               Epigenetic Role H3K36 Methylation <br />  <br />  <br />  <br />  <br /> Figure 2. MES-4 concentrated autosomes left tip X chromosome. (A) The anti-MES-4 antibody used ChIP stains <br /> the 10 autosomes left tip X (arrow; X chromosome marked arrowhead), similar previously characterized <br /> antibodies [8,19]. The MES-4 signal over-exposed dot MES-4 left end X absence detectable MES-4 the <br /> rest X. Additional staining, including absence MES-4 signal mes-4 mutant, shown Figure S2B. (B) 5291 5391 MES-4 binding <br /> sites distributed autosomes. Of 100 MES-4 peaks X chromosome, 25 leftmost 300 kb. The expected and <br /> observed numbers MES-4 peaks chromosome Figure S2D. All ChIPs shown Figure 2, Figure 3, Figure 4, Figure 5, and <br /> Figure 6 performed early embryo extracts. <br /> doi:10.1371/journal.pgen.1001091.g002 <br />  <br />  <br /> The methylation H3K36 generally mirrors MES-4                             performed using highly specific monoclonal antibodies [33] <br /> accumulation                                                                 (Figure S1). <br />    To compare distribution MES-4 distributions                 The H3K36me2 H3K36me3 ChIP profiles generally <br /> H3K36 methyl marks early embryos, performed ChIP-chip                  mirror MES-4. MES-4 H3K36me2 me3 ChIP <br /> using antibodies H3K36me2 H3K36me3. We                           signals genes highly correlated (Figure 3 Figure <br /> analyzed H3K4me3 H3K9me3. All histone mark ChIPs                         S3B). In bodies actively expressed genes, H3K36me2 and <br />  <br />  <br />  <br />  <br /> Figure 3. MES-4 concentrated Pol II&#226;&#8364;&#8220;bound genes. For representative ,180 kb autosomal (ChrIV) region genome, <br /> annotated genes (arrows; genes encoding multiple transcript variants, longest ones shown) ChIP z-scores (standardized log2 ratios of <br /> ChIP/Input signals) Pol II, MES-4, MES-4::FLAG, H3K36me2, H3K36me3, H3K4me3, H3K9me3, H3, control ChIP (Protein A beads + rabbit IgG) <br /> are shown. Pol II, MES-4, H3K36me2 H3K36me3 generally colocalize bodies transcribed genes. H3K4me3 mainly promoter <br /> and 59 coding region transcribed genes. H3K9me3 absent expressed genes. See Figure S3 analysis reproducibility concordance <br /> among ChIPs. <br /> doi:10.1371/journal.pgen.1001091.g003 <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                              4                     September 2010 | Volume 6 | Issue 9 | e1001091 <br />                                                                                                                   Epigenetic Role H3K36 Methylation <br />  <br />  <br />  <br />  <br /> Figure 4. MES-4 H3K36 methyl marks concentrated gene bodies highly expressed genes. Genes .2 kb long (9932 <br /> genes) grouped 5 equal sized bins based expression early embryos. Colors display bins highest (green) lowest (orange) <br /> transcript level early embryo extracts generated parallel ChIP extracts. Average z-score profiles ChIP target shown 50 bp <br /> steps 2.5 kb transcript start site (TSS) transcript end site (TES). Error bars indicate 95% confidence intervals means. <br /> The row shows different control ChIPs: Protein A beads, Protein G beads, Protein A beads coated rabbit IgG, sheep anti-mouse <br /> beads coated mouse anti-FLAG. GC content variation shown. <br /> doi:10.1371/journal.pgen.1001091.g004 <br />  <br /> me3 levels rise TSS drop TES (Figure 4).             understanding mechanism action. Gene Ontology (GO) <br /> The relatively flat profiles H3K36me2 me3 gene                   analysis MES-4-bound genes revealed MES-4 <br /> bodies contrast 39 enriched profiles H3K36                         preferentially associates genes participate reproduc- <br /> methylation generated Set2 homologs yeast                      tion, growth development (Table S1C). To charac- <br /> systems [e.g. 11,25]. The distinctive gene-body profiles                    terize MES-4 target genes, analyzed RNA levels conjunction <br /> H3K36me2 me3 shared histone methyl                        MES-4, Pol II, H3K36me2 me3 ChIP signals on <br /> marks. In keeping H3K4me3 hallmark active                      genes exhibit different temporal-spatial expression patterns <br /> chromatin specifically marking transcription initiation,                   (Figure 5A). We generated gene categories, based previous <br /> H3K4me3 distributed TSS, declines sharply                    studies (Table S2): (1) 4693 &#226;&#8364;&#732;&#226;&#8364;&#732;germline-expressed&#226;&#8364;&#8482;&#226;&#8364;&#8482; genes whose <br /> within gene bodies (Figure 4). H3K9me3,                        transcripts present dissected adult hermaphrodite germ lines <br /> associated transcriptionally silent chromatin, shows               according SAGE analysis [34]; (2) 169 &#226;&#8364;&#732;&#226;&#8364;&#732;germline-specific&#226;&#8364;&#8482;&#226;&#8364;&#8482; <br /> an inverse relationship gene expression (Figure 4). The histone             genes transcripts expressed exclusively maternal <br /> mark patterns described compared histone H3,                   germ line (transcripts detected dissected hermaphrodite <br /> which reflects nucleosome occupancy. We H3 modestly               germ lines SAGE, enriched germ line based on <br /> elevated gene bodies shows pronounced dip                      whole-worm microarrays, accumulate embryos strictly by <br /> promoter region highly expressed genes, reported             maternal contribution determined microarray analysis of <br /> other organisms [e.g. 25].                                                     staged embryos, represented muscle, gut, neuron <br />                                                                                SAGE sets) [21,34&#226;&#8364;&#8220;36]; (3) 797 &#226;&#8364;&#732;&#226;&#8364;&#732;embryo-expressed&#226;&#8364;&#8482;&#226;&#8364;&#8482; genes whose <br /> Genes bound MES-4 early embryos                           transcripts maternally provided increase level <br /> had expressed previously maternal germ line                        embryogenesis [35]; (4) 323 &#226;&#8364;&#732;&#226;&#8364;&#732;soma-specific&#226;&#8364;&#8482;&#226;&#8364;&#8482; genes whose <br />   MES-4 essential germline development survival,                transcripts detected muscle, gut, neuron SAGE sets but <br /> knowing genes bound MES-4 critical                          adult germ line SAGE microarray analysis <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                                5                      September 2010 | Volume 6 | Issue 9 | e1001091 <br />                                                                                                               Epigenetic Role H3K36 Methylation <br />  <br />  <br />  <br />  <br /> Figure 5. Distribution RNA, Pol II, MES-4, H3K36me2, H3K36me3 levels various gene sets. (A) Gene classes defined as <br /> described Results. The number genes set shown parentheses. For gene set, boxplots distribution RNA level <br /> (standardized z-score calculated microarray analysis) Pol II, MES-4, H3K36me2 H3K36me3 average z-scores early embryo <br /> ChIP-chip analysis. Each box extends 25th 75th percentile z-scores set, whiskers extending box indicate the <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                              6                     September 2010 | Volume 6 | Issue 9 | e1001091 <br />                                                                                                              Epigenetic Role H3K36 Methylation <br />  <br />  <br />  <br /> 2.5th 97.5th percentile. Wedges median indicate 95% confidence interval medians. See Figure S4C scatter plots the <br /> ubiquitous, germline-expressed, germline-specific embryo-expressed categories. (B) Examples Pol II- MES-4+ H3K36me+ Pol II+ MES-42 <br /> H3K36me2 genes. The germline-expressed gene csr-1 (left dashed box) contains high levels MES-4, H3K36me2 H3K36me3 lacks Pol II and <br /> H3K4me3. The soma-expressed gene ceh-13 (right dashed box) contains Pol II H3K4me3 lacks MES-4, H3K36me2 H3K36me3. Both csr-1 <br /> and ceh-13 flanked genes typical high concordance Pol II, MES-4 H3K36me marks. See Table S2 lists genes in <br /> different temporal-spatial classes Figure 6C examples individual genes. <br /> doi:10.1371/journal.pgen.1001091.g005 <br />  <br /> [21,34,36]; (5) 2580 &#226;&#8364;&#732;&#226;&#8364;&#732;ubiquitous&#226;&#8364;&#8482;&#226;&#8364;&#8482; housekeeping genes              MES-4 Pol II-bound genes (the two <br /> transcripts shared muscle, gut, neuron adult germline             large classes discussed above) (Figure 3, Figure S3B, Figure <br /> SAGE sets [34,36]; (6) 415 &#226;&#8364;&#732;&#226;&#8364;&#732;silent&#226;&#8364;&#8482;&#226;&#8364;&#8482; genes, serpentine receptor         S4C), Pol II accumulation sufficient recruit MES-4. <br /> genes expressed mature neurons                    In addition clues MES-4, members the <br /> expected expressed cells early embryos [37]. The                 embryonically-expressed soma-specific genes provide first <br /> distinctive patterns MES-4, Pol II, H3K36 methylation              natural examples transcription elongation absence of <br /> genes different temporal-spatial classes provided key insights            detectable H3K36 methylation (Figure 5B Figure S4C), a <br /> into MES-4 recruitment function.                                         phenomenon observed yeast set2 mutants previously [9]. <br />    First, largest classes genes, germline-expressed genes         The C. elegans Set2 ortholog MET-1, participates in <br /> and ubiquitously-expressed genes, share <span id='am-240' about='xsp:pattern' typeof='owl:Thing'>pattern</span> high RNA                transcription-dependent H3K36 methylation embryos <br /> accumulation high Pol II, MES-4 H3K36me2/me3                         (Figure 1), expected methylate H3K36 genes newly <br /> association (Figure 5A Figure S4C). The common theme                     expressed embryos. By immunofluorescence, MET-1-mediated <br /> among genes expressed maternal germ            H3K36me3 detected 2- 40-cell embryos gradually <br /> line. Thus, early embryo extracts, genes expressed                 rises 40-cell stage onward (Furuhashi et al., unpublished <br /> maternal germ line typically remain bound Pol II, MES-4,              unpublished results). In ChIP experiments, genes <br /> H3K36me2/me3. As interesting sidelight, clustering Pol             bound Pol II lacking MES-4 display little no <br /> II MES-4-bound genes displayed Figure 3 observed                  H3K36me2/me3 (Figure S4C). Thus, early embryo <br /> across autosomes consistent finding ,40%             samples, MET-1 appears relatively minor contributor to <br /> genes germline-enriched expression organized operons,            H3K36 methylation. <br /> and operons cluster monocistronic <br /> germline-expressed genes [38].                                               MES-4 responsible H3K36 methylation germline- <br />    Second, silent genes exhibit low RNA accumulation low Pol             specific genes <br /> II, MES-4, H3K36me2/me3 (Figure 5A). This result fits                  The results strongly suggest MES-4 mediates the <br /> seeing lowest levels Pol II, MES-4, H3K36me2/me3                      majority H3K36 methylation early embryos. To investigate <br /> association genes quantiles expression                 MES-4&#226;&#8364;&#8482;s contribution, depleted MES-4 RNAi and <br /> (Figure 4).                                                                  performed ChIP-chip analysis MES-4 H3K36me3. <br />    Third importantly, germline-specific class shows           Evidence RNAi effective provided following <br /> distinct pattern: high RNA accumulation, high MES-4                      observations: mes-4(RNAi) embryos used ChIP lacked <br /> H3K36me2/me3 signals, low Pol II (Figure 5A Figure                   detectable MES-4 immunostaining, MES-4 H3K36me3 <br /> S4C). To explore deviation usual high correlation              ChIP signal autosomes, MES-4 overwhelmingly resides <br /> between MES-4 Pol II, analyzed individual germline-                   wild type, reduced mes-4(RNAi) extract low level <br /> specific genes UCSC genome browser. Numerous genes                    observed X chromosome (Figure 6A). Importantly, <br /> displayed robust MES-4 H3K36me2/me3 ChIP signal                   H3K36me3 ChIP signal mes-4(RNAi) completely lost from <br /> detectable Pol II ChIP signal (Figure 5B Figure 6C). We                  germline-specific genes (Figure 6B 6C), supporting our <br /> identified 214 genes having high MES-4 high H3K36me3                  hypothesis genes H3K36 methylated solely by <br /> (z-scores.1) low Pol II (z-score,0) (Table S3). Notably, 180             MES-4 absence active transcription. H3K36me3 <br /> of 214 genes (84%) evidence germline expression                  signal ubiquitously-expressed germline-expressed genes, <br /> based microarray and/or SAGE analysis [21,34],                majority actively transcribed early embryos, <br /> those genes known involved germline-specific                    reduced eliminated (Figure 6B 6C). The <br /> processes (e.g. meiosis genes him-8, htp-1, htp-2, htp-3, rec-8,         remaining H3K36 methylation genes probably to <br /> syp-2, zim-2, zim-3, P-granule genes pgl-1, pgl-3, glh-2, glh-       (low) MET-1 activity possibly incomplete knock-down of <br /> 4). The germline-specific class genes indicates MES-4            MES-4. Embryo-expressed soma-specific genes showed little <br /> persist genes previously expressed maternal              change H3K36me3 levels mes-4(RNAi), retaining the <br /> germ line longer actively transcribed early              low levels H3K36me3 seen wild type (Figure 6B 6C). <br /> embryos, revealing MES-4 associate genes                       This suggests detection H3K36me3 low <br /> independently Pol II. This class probably contributes              MET-1 activity set actively transcribed genes. Taken <br /> MES-4 H3K36me2 me3 staining observed depletion                 together, RNAi results demonstrate embryogenesis <br /> of Pol II (Figure 1E) [8].                                                   MES-4 responsible H3K36 trimethylation germline- <br />    Fourth, embryonically-expressed genes soma-specific genes             specific genes contributes majority H3K36 trimethyla- <br /> are silent maternal germ line activated            tion germline-expressed genes well. <br /> embryogenesis. These genes display unusual pattern high <br /> RNA accumulation high Pol II low MES-4                           The RNA polymerase-independent epigenetic function of <br /> H3K36me2/me3 (Figure 5A Figure S4C). In fact, inspection                 MES-4 reflected distribution H3K36me3 on <br /> of individual genes class identified examples robust Pol II       target genes <br /> ChIP signal low detectable MES-4 H3K36me2/me3                     The results reveal embryos MES-4 associate <br /> (Figure 5B, Figure 6C, Figure S4C). These findings suggest          genes maintain H3K36 methylation Pol II- <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                              7                    September 2010 | Volume 6 | Issue 9 | e1001091 <br />                                                                                                              Epigenetic Role H3K36 Methylation <br />  <br />  <br />  <br />  <br /> Figure 6. RNAi depletion MES-4 substantially reduces H3K36me3 germline-expressed genes. (A) MES-4 H3K36me3 ChIP-chip <br /> signal autosomes (gray) X chromosome (red), specifically genes highest expression wild type (top 20% dark color) <br /> and genes lowest expression wild type (lowest 20% light color). MES-4 dramatically reduced genome-wide mes-4(RNAi). H3K36me3 <br /> on autosomes reduced lower level observed X, MES-4 largely absent wild type. The remaining low level H3K36me3 <br /> on transcribed genes mes-4(RNAi) likely transcription-coupled methylation MET-1. (B) MES-4 H3K36me3 ChIP-chip signal on <br /> ubiquitously-expressed (orange), germline-expressed (green), germline-specific (blue), embryo-expressed (red) gene sets wild type mes- <br /> 4(RNAi). In mes-4(RNAi), H3K36me3 dramatically reduced genes expressed germline (ubiquitous germline-expressed) and <br /> undetectable germline-specific genes. The embryo-expressed class (and soma-specific class, shown) lack MES-4 wild type little <br /> change mes-4(RNAi). (C) Examples individual genes illustrating patterns observed (B). <br /> doi:10.1371/journal.pgen.1001091.g006 <br />  <br />  <br /> independent manner. If MET-1 resembles Set2 recruited              H3K36me3 generated known Set2 homologs <br /> to genes elongating Pol II, MET-1-generated H3K36me3                [11,23,25,39]. Indeed, soma-specific genes, generally lack <br /> should enriched 39 coding region genes, similar             MES-4 binding presumed acquire H3K36 methylation <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                             8                     September 2010 | Volume 6 | Issue 9 | e1001091 <br />                                                                                                                Epigenetic Role H3K36 Methylation <br />  <br />  <br /> from MET-1, display enrichment 39 gene body                     memory gene expression adult germ line cells the <br /> (Figure 7A red). This pattern relatively low-level early              embryo (see Discussion). <br /> embryos MET-1 activity low (see discussion previous <br /> section) pronounced later stages (L3 larvae; data             The H3K36 methyltransferase activity maternal MES-4 <br /> [37]) somatic cells actively engaged transcription             crucial germline development progeny <br /> and MET-1 level activity expected elevated. In                      Embryos mes-4/mes-4 mothers develop sterile adults, <br /> contrast, germline-specific genes display distribution                    embryonic expression MES-4(+) sufficient restore <br /> H3K36me3 slightly 59 enriched, similar distribution             fertility [18]. Based findings embryos MES-4 persists <br /> of MES-4 (Figure 7A blue). The difference H3K36me3                       germline-expressed genes absence ongoing <br /> profiles germline- versus soma-specific genes strengthens               transcription, reasoned failure embryonic expression <br /> notion MES-4 MET-1 recruited genes different                MES-4(+) rescue fertility likely loss H3K36 <br /> mechanisms serve different roles targets. We                      methylation generated maternal MES-4(+) inability of <br /> hypothesize MES-4 serves epigenetic role transmits                 embryonically expressed MES-4(+) newly generate patterns <br />                                                                                H3K36 methylation needed normal PGC development. <br />                                                                                Previous tests rescue fertility embryonic expression of <br />                                                                                MES-4(+) performed using mes-4/mes-4 mutant mothers that <br />                                                                                lack detectable H3K36 HMT activity (bn23, bn50, bn67 alleles <br />                                                                                mes-4 [8,18]). We tested additional alleles goal <br />                                                                                assessing mes-4/mes-4 mutant mothers weak HMT <br />                                                                                activity provide sufficient H3K36 marking enable <br />                                                                                embryonic expression MES-4(+) rescue fertility. The bn73 <br />                                                                                allele does produce detectable MES-4 protein, bn85 allele <br />                                                                                produces MES-4 lacking catalytic SET domain, and <br />                                                                                bn58 allele produces MES-4 weak HMT activity [8]. <br />                                                                                Embryonic expression MES-4(+) did restore fertility to <br />                                                                                animals MES-4(2) mothers (bn73 allele) mothers that <br />                                                                                produce MES-4 lacking HMT activity (bn85 allele) (Figure 7B). <br />                                                                                Importantly, embryonic expression MES-4(+) restored fertility <br />                                                                                animals mothers produced mutant MES-4 weak <br />                                                                                HMT activity (bn58 allele) (Figure 7B). This demonstrates the <br />                                                                                HMT activity MES-4 critical maternal MES-4 function, <br />                                                                                strengthens view MES-4 acts epigenetically and <br />                                                                                transmits developmentally important H3K36 marks the <br />                                                                                parents&#226;&#8364;&#8482; germ line embryos, ensure normal development of <br />                                                                                PGCs. <br />  <br />                                                                                Discussion <br />                                                                                   MES-4 emerged critical germline regulator several <br />                                                                                studies. Its maternal-effect sterile (mes) <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> demonstrates <br />                                                                                action MES-4(+) maternal germ line and/or a <br />                                                                                maternal supply MES-4(+) required PGCs offspring <br />                                                                                thrive survive [18]. Germline functions require MES-4 <br />                                                                                repression genes X chromosome, transgene silencing, <br />                                                                                cosuppression, RNAi [8,40&#226;&#8364;&#8220;42]. In addition these <br /> Figure 7. The distribution requirement MES-4-                       repressive roles, MES-4 required promote ectopic expression <br /> catalyzed H3K36 methyl marks. (A) Germline-specific soma-                  germline genes somatic cells synMuv B mutant larvae <br /> specific genes display different profiles H3K36me3, MES-4, Pol II       [43,44]. The current study identified MES-4 binding targets as <br /> in early embryos (emb) H3K36me3 L3 larvae (L3 data from <br />                                                                                genes expressed adult germ line. The targeting and <br /> [37]). Gene profiles described Figure 4, using germline- <br /> and soma-specific genes described Results. This figure highlights       function MES-4 appear distinct well-studied Set2 <br /> reciprocal relationship Pol II levels MES-4/H3K36me3               family H3K36 HMTs. We speculate MES-4 transmits the <br /> levels germline- soma-specific genes early embryos (also         memory germline gene expression patterns parental <br /> Figure 5). On germline-specific genes, H3K36me3, like MES-4, slightly       germ line PGCs offspring, loss function <br /> more elevated 59 39 gene bodies, soma-specific              causes PGCs degenerate young larvae. <br /> genes, H3K36me3 elevated 39 59 gene bodies, especially <br /> in L3s, probably transcription-dependent methylation MET-1.             By immunofluorescence ChIP-chip analysis, MES-4 is <br /> (B) Embryonic expression MES-4(+) rescues fertility mes-4 mutants        concentrated autosomes left tip X <br /> only maternal MES-4 retains HMT activity. Analysis                ([8,19] study). The strong autosomal bias MES-4 ChIP <br /> mutant MES-4 proteins [8]. M-Z2 (lacking maternal             signal X chromosome structure recalcitrant <br /> zygotic expression mes-4+) worms mes-4/mes-4 hermaphrodites            ChIP, ChIP analysis dosage compensation complex <br /> produced mes-4/mes-4 mothers. M-Z+ worms mes-4/+                       preferentially targets X [45,46]. Instead, autosomal <br /> hermaphrodites produced mating mes-4/mes-4 mothers wild- <br /> type males. At 24 M-Z+ hermaphrodites assessed fertility.       enrichment MES-4 likely explained association of <br /> 75% bn58/+ hermaphrodites produced viable progeny; 8% produced              MES-4 germline-expressed genes, significantly <br /> embryos failed hatch.                                                  under-represented X [21]. The left end X several <br /> doi:10.1371/journal.pgen.1001091.g007                                          distinquishing properties, including relatively high concentration <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                                9                    September 2010 | Volume 6 | Issue 9 | e1001091 <br />                                                                                                                Epigenetic Role H3K36 Methylation <br />  <br />  <br /> of H3K9me3 low density periodic AA/TT clusters                         methylate chromatin pre-existing H3K36 methyl marks. <br /> compared rest X [47] presence X                  Consistent this, embryonic expression MES-4(+) rescue <br /> chromosome &#226;&#8364;&#732;&#226;&#8364;&#732;pairing center&#226;&#8364;&#8482;&#226;&#8364;&#8482; meiosis [48]. The autosome-like             fertility embryos mothers produce mutant MES-4 <br /> higher density MES-4 binding sites left end X                 weak HMT activity, embryos mothers lack <br /> consistent &#226;&#8364;&#732;&#226;&#8364;&#732;pseudo-autosomal&#226;&#8364;&#8482;&#226;&#8364;&#8482; nature. Perhaps          MES-4 produce MES-4 lacks HMT activity. The <br /> arose translocation autosomal segment end              maintenance HMT activity MES-4 enables MES-4 serve a <br /> X. In fact, comparison genomes C. elegans Pristionchus          truly epigenetic role, propagation particular chromatin state <br /> pacificus suggests evolved common ancestor                meiosis mitosis. Recent studies demonstrate the <br /> which chromosomes X V fused ([49] R. Sommer,                     Polycomb Repressive Complex 2 initiates maintains a <br /> personnal communication). The autosomal concentration MES-                 repressed chromatin state, binding chromatin <br /> 4 regulated MES proteins, MES-2, MES-3,                 marks generates [6]. <br /> and MES-6, form C. elegans Polycomb Repressive                         How MES-4 initially recruited germline-expressed genes? <br /> Complex 2 like PRC2 catalyze methylation H3K27 [50&#226;&#8364;&#8220;                    Our findings MES-4 associate genes generate <br /> 52]. Loss component MES-2/3/6 complex leads                  H3K36 methylation independently Pol II rule the <br /> loss silencing X chromosomes germ line                   possibility MES-4 Pol II-dependent mode well. In the <br /> significant MES-4 immunofluorescence signal <span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span>             adult germ line, MES-4 initially recruited expressed <br /> of X oocytes embryos [8,19]. Our model predicts               genes Pol II-dependent manner. Another possibility that <br /> in mes-2, mes-3, mes-6 mutant embryos, elevated MES-4 signal              MES-4, PHD fingers, binds H3K36 methyl marks <br /> will observed X-linked genes expression up-                   generated MET-1 and/or MES-4. Yet possibility that <br /> regulated maternal germ line.                                          chromodomain protein MRG-1, like counterparts other <br />    Our studies demonstrate early embryos MES-4                        systems [12,27,28,30,54], associates methylated H3K36 and <br /> associates genes expressed maternal germ                helps recruit MES-4. In fact, MRG-1, like MES-4, displays <br /> line. Genes expressed exclusively maternal germ line              autosomal enrichment immunofluorescence, mrg-1 mutants <br /> high MES-4 low Pol II. Some individual genes MES-4+                   display suite mes-4-like defects, including maternal-effect death <br /> Pol II2. These genes strongly argue Pol II required            PGCs [55]. MES-4 association autosomes lost mrg- <br /> maintain MES-4 association genes. Conversely, genes newly                1 mutants vice versa [55]. Determining MRG-1 <br /> expressed embryos genes expressed specifically somatic              participates recruiting MES-4 and/or downstream effector <br /> cells high Pol II low MES-4. Indeed, numerous                        MES-4-mediated H3K36 methylation progress. <br /> individual genes Pol II+ MES-42. This consistent                     An important question future investigation PGCs <br /> Pol II sufficient recruit MES-4 expressed genes.              embryos mes-4 mutant mothers die. An attractive scenario <br />    Additional evidence supports notion MES-4 capable              MES-4 marking genes expressed maternal germ <br /> of associating gene bodies methylating H3K36                         line identifies genes expressed progeny&#226;&#8364;&#8482;s germ line. A <br /> independently Pol II. First, RNAi depletion large subunit           related scenario, temporally precede first, that <br /> of RNA Pol II does impair MES-4 binding chromosomes                 MES-4 marking genes helps genes repressed the <br /> H3K36 di2 trimethylation, detected immunofluorescence                PGCs embryogenesis. Indeed, wild-type PGCs generally do <br /> ([8] study). Second, slightly 59 enriched distribution        acquire marks active transcription embryos hatch <br /> MES-4 genes bodies quite different 39 enriched             L1s, mes-4 mutant PGCs acquire marks <br /> distribution Set2 homologs, driven association            prematurely, mid-embryogenesis ([56] Furuhashi et <br /> Set2 elongating Pol II [11,23,25,39]. Recent studies yeast            al., unpublished). In addition potential role up- down- <br /> have revealed Set2 recruited genes independently            regulating transcription PGCs, MES-4 influence gene <br /> Pol II, impaired H3K36 di2 trimethylation [53]. In              expression level regulation splicing. Recent papers <br /> contrast, MES-4 appears capable di2 trimethylating                     report H3K36me3 enriched expressed exons relative to <br /> H3K36 absence Pol II. Taken together, findings                  introns [37] level H3K36me3 influences alternative <br /> suggest MES-4 related mammalian NSD                      splicing [57]. Like H3K36me3, MES-4 appears exon- <br /> proteins provide layer function H3K36 methylation              enriched (Table S1B Figure S5), raising exciting possibility <br /> that novel likely unrelated ongoing transcription.            MES-4 preferentially associates exons methylates <br />    An attractive model MES-4 serves maintenance                  manner facilitates regulates splicing. Experiments <br /> HMT mark germline-expressed genes pass memory                   planned isolate PGCs wild-type mes-4 mutant <br /> gene expression generation germ cells next. In             embryos compare RNA accumulation splicing <br /> this model, H3K36 HMT MET-1 serves Set2-like role                   patterns. In meantime, analyzed overlap MES- <br /> tracks Pol II methylate H3K36 gene expression               4-bound genes embryos genes mis-regulated mes-4 <br /> the adult germ line; embryogenesis, MES-4 maintains                    mutant adult germ lines, small (Figure S6). The <br /> H3K36 methylation genes independently Pol II. We                  small overlap stage difference (i.e. embryo <br /> think MES-4 serve maintenance role                      versus adult germ line) technical biological effects that <br /> transcriptionally repressed cells (e.g. germline blastomeres)          cause documented transcription regulators display a <br /> and potentially generations (e.g. met-1 mutant worms). In              low overlap factor-bound genes genes mis-regulated <br /> support model, MES-4 does appear capable              factor absent [e.g. 58,59]. Despite absence gene <br /> novo H3K36 methylation soma embryonic PGCs:                        expression data PGCs, strong dependence early PGC <br /> embryonic expression MES-4 embryos lack maternal                   development maternal MES-4 demonstrates functional <br /> H3K36me3 (mes-4; met-1 double mutant mothers) does                        importance MES-4. <br /> generate detectable H3K36me3 signal (Furuhashi et al., unpub-                    MES-4 HMT activity detected cells early <br /> lished). In contrast, embryonic expression MET-1 generates                 embryos restricted PGCs mid late <br /> robust H3K36me3. Thus, model posits MES-4                       embryogenesis ([19] unpublished results). Our model that <br /> specialized maintenance HMT embryos                          MES-4 propagates memory germline gene expression <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                              10                     September 2010 | Volume 6 | Issue 9 | e1001091 <br />                                                                                                             Epigenetic Role H3K36 Methylation <br />  <br />  <br /> through embryogenesis raises question somatic cells           reserved input. Rabbit antibodies used IP anti-histone <br /> embryo deal signal. Our current view synMuv            H3 serum (Active Motif AR-0144) affinity-purified anti-MES- <br /> B chromatin regulators antagonize germline fate somatic cells.            4 (SDI). Mouse monoclonal antibodies used IP M2 anti- <br /> Loss synMuv B proteins causes somatic cells express                    FLAG (Sigma F3165), 8WG16 anti-Pol II (Abcam ab817), anti- <br /> germline-specific genes, concomitant loss maternal MES-               H3K4me3 (Wako 305-34819), anti-H3K9me3 (from Hiroshi <br /> 4 suppresses germline potential somatic cells [43,44].                Kimura), anti-H3K36me2 (from Hiroshi Kimura), anti- <br /> Further tests model interplay MES-4,               H3K36me3 (from Hiroshi Kimura). <br /> MET-1, MRG-1 synMuv B chromatin regulators shed                    MES-4 Pol II ChIPed using 3 mg embryo protein and <br /> light germline identity passed generation                  5 mg antibody. For ChIP reaction, antibody incubated <br /> generation germline gene expression patterns                     overnight 4uC embryo extract. Protein A- Protein G- <br /> controlled.                                                                  coupled Dynabeads (Invitrogen) equilibrated 50 ml FA buffer <br />                                                                              added, nutated 2 hr 4uC, concentrated using Dynal <br /> Materials Methods                                                        Magnetic Particle Concentrator (MPC) (Invitrogen) washed at <br />                                                                              room temperature 5 min 1 ml of following <br /> RNAi depletion RNA polymerase II                                          buffers: FA buffer, 2 washes; FA buffer 1M NaCl, 1 wash. <br />    We depleted AMA-1 feeding N2 worms bacteria expressing                 Beads transferred new tube, washed 10 min at <br /> dsRNA ama-1 (from Ahringer RNAi feeding library                  room temperature 1 ml FA buffer 500 mM NaCl, once <br /> [60]). In brief, L3-L4 hermaphrodites fed 24 hours               TEL buffer (0.25 M LiCl, 1% NP-40, 1% sodium <br /> 24uC, transferred new plates score embryonic                  deoxycholate, 1 mM EDTA, 10 mM Tris-HCl, pH 8.0) and <br /> lethality F1 progeny (.98%) dissected obtain embryos            twice 5 min room temperature 1 ml TE. To elute <br /> staining.                                                                    complexes, beads incubated twice 65uC 150 ml of <br />                                                                              elution buffer (1% SDS TE 250 mM NaCl) 15 min. <br /> Generation MES-4 ChIP reagents                                            Input samples brought volume 300 ml incubated <br />    Affinity purified rabbit antibodies directed amino acids          20 mg RNAse A 30 minutes 37uC. Both input IP <br /> 729&#226;&#8364;&#8220;828 MES-4 generated Strategic Diagnostic Inc.                 samples incubated 20 mg proteinase K 1 hr at <br /> (SDI, Newark, DE). mes-4 animals expressing MES-4::GFP::FLAG               55uC. Crosslinks reversed overnight 65uC. DNA was <br /> transgene generated described Supplement.                         purified using Zymo DNA purification column. <br />                                                                                 Histone H3 methyl marks ChIPed using 1 mg embryo <br />                                                                              protein 2 mg antibody method above, except <br /> Preparation N2 mes-4(RNAi) early embryo extracts <br />                                                                              IP beads Dynabeads coupled sheep anti-mouse IgG <br /> and chromatin                                                                (Invitrogen) equilibrated 50 ml 5 mg/ml IgG-free BSA <br />    For preparation embryo extracts, N2 adult worms grown             (Sigma) FA buffer, input IP samples were <br /> from synchronized L1s standard S-basal medium shaking             incubated RNAse A proteinase K. <br /> 230 rpm. N2 worms fed HB101 58&#226;&#8364;&#8220;60 hr 20uC, fed <br /> bacteria expressing dsRNA mes-4 (from Ahringer RNAi              Amplification LM&#226;&#8364;&#8220;PCR <br /> feeding library [60]) 48 hr 24uC. To ensure majority of <br />                                                                                 Input IPed DNAs blunted T4 DNA polymerase, <br /> embryos used extract preparation early stages, worms           purified using Zymo DNA purification column, ligated a <br /> processed extract majority worms contained 10          unidirectional linker prepared following HPLC-purified <br /> fewer embryos. Embryos obtained dissolving adult worms               oligos: Long: 59-AGAAGC TTGAATTCGAGCAGTCAG-39 <br /> with bleach analyzed DAPI staining. Embryo preparations               Short: 59-CTGCTCGAATTCAAGCTTCT-39. DNAs were <br /> contained average 37% ,28-cell embryos, 30% 28&#226;&#8364;&#8220;100-cell                purified using Zymo DNA purification column amplified <br /> embryos, 33% ,100&#226;&#8364;&#8220;300-cell embryos. 50 ml packed                      Short oligo 80 ml volume using following cycling <br /> embryos set aside RNA extraction expression profiling.          conditions: 55uC 4 min, 72uC 5 min, 94uC 5 min <br /> The remaining embryos washed cross-linked 1.85%                followed 25 cycles of: 94uC 1 min, 55uC 1 min, 72uC for <br /> formaldehyde M9 buffer 30 min room temperature,               1 min. Samples incubated additional 5 min 72uC. <br /> washed 5 min following buffers: twice M9,            DNA purified using Qiagen PCR purification kit eluted <br /> once 100 mM Tris-HCl pH7.5, 10 ml FA buffer                   30 50 ml dH2O. <br /> (50 mM HEPES/KOH pH7.5, 1 mM EDTA, 1% Triton X-100, <br /> 0.1% sodium deoxycholate, 150 mM NaCl). Embryos frozen <br />                                                                              Microarray hybridizations data analysis display <br /> at 280uC. 2 ml embryos, thawed resuspended 4 ml FA <br />                                                                                 NimbleGen 2.1M probe tiling arrays, 50 bp probes, <br /> buffer containing protease inhibitors (Roche protease inhibitor <br />                                                                              designed WS170 (ce4) used. 2&#226;&#8364;&#8220;3 independent ChIPs <br /> cocktail tablet), dounce-homogenized 30 times using tight <br />                                                                              performed antibody antibody <br /> pestle 4uC. Chromatin sheared approximate size range <br />                                                                              control (NoAb). Amplified samples labeled hybridized by <br /> of 200&#226;&#8364;&#8220;800 bp using Branson sonicator Diagenode bioruptor. <br />                                                                              Roche NimbleGen Service Laboratory. Most ChIP samples <br /> To remove cellular debris, samples centrifuged 13,000 rpm, <br />                                                                              labeled Cy5 input reference Cy3; for <br /> 4uC 15 min. To remove lipids, samples filtered              targets, ChIP antibody dye swapped. For each <br /> Millipore Ultrafree-MC 0.45 mm filter units 13,000 rpm, 4uC           probe, intensity sample channel divided the <br /> 1 min. Protein concentration determined samples                 reference channel transformed log2 space. The enrichment <br /> stored 280uC.                                                             scores replicate calculated standardizing log <br />                                                                              ratios mean zero standard deviation (z-score). The mes- <br /> Chromatin immunoprecipitation (ChIP) early                              4(RNAi) ChIP samples normalized respect X <br /> embryos                                                                      chromosome wild type. MES-4 largely absent <br />   ChIPs performed using 1&#226;&#8364;&#8220;6 mg worm embryo protein                   X wild type, expected affected <br /> and 2&#226;&#8364;&#8220;5 mg antibody IP. 5&#226;&#8364;&#8220;10% starting material                   RNAi. The log2 ratios MES-4 H3K36me3 in <br />  <br />  <br />       PLoS Genetics | www.plosgenetics.org                              11                    September 2010 | Volume 6 | Issue 9 | e1001091 <br />                                                                                                              Epigenetic Role H3K36 Methylation <br />  <br />  <br /> mes-4(RNAi) normalized X chromosome                 temperature. 20 mg RNA hybridized single <br /> same mean standard deviation X chromosome                  color 4-plex NimbleGen expression array 72,000 probes <br /> respective wild-type ChIP samples (after wild-type ChIPs             (three 60-mer oligo probes gene). Quantile normalization <br /> been z-score normalized). Genome-wide Pearson correlations              [62] Robust Multichip Average (RMA) algorithm [63] <br /> calculated ChIP targets replicates using probes&#226;&#8364;&#8482;         used normalize summarize multiple probe values <br /> z-scores smoothing 250 bp. The average z-score             gene obtain expression value gene sample. <br /> replicates used analyses.                                        The expression values gene averaged four <br />    Scatter plots boxplots generated obtaining           samples. <br /> average z-score gene. Z-scores probes located completely <br /> within transcript start site (TSS) end site (TES)               Assessment rescue embryonic expression MES- <br /> averaged gene.                                                      4(+) <br />    Gene body profile plots various ChIP targets                    mes-4(bn58, bn73, bn85) dpy-11(e224)/DnT1[unc(n754)let] <br /> generated aligning genes length greater 2 kb             (IV;V) mothers allowed produce mes-4 dpy-11 (Dpy) <br /> TSS TES. The genomic regions 1.5 kb upstream 1 kb                     hermaphrodites. These mes-4 dpy-11 (Dpy) hermaphrodites were <br /> downstream TSS 1.5 kb upstream 1 kb                          mated wild-type males. The Dpy progeny, result self- <br /> downstream TES divided 50 50-bp bins each.                fertilization, M-Z2 (lacking maternal zygotic mes- <br /> Probes genomic regions assigned nearest bin. <br />                                                                              4(+) product). The non-Dpy progeny, result cross- <br /> A profile group genes generated averaging probes&#226;&#8364;&#8482; <br />                                                                              fertilization, M-Z+. At 24 category progeny <br /> z-scores bin genes group. Error bars <br />                                                                              scored fertility (the production embryos). <br /> indicate 95% confidence intervals mean. <br />                                                                              Supporting Information <br /> ChiPOTle peak finding analysis <br />    ChiPOTle 2.0 (http://sourceforge.net/projects/chipotle-2) [31]            Figure S1    Specificity newly developed monoclonal antibodies <br /> was run using p-value cut-off E-20 average z-score           directed H3K36me2 H3K36me3. Specificity was <br /> MES-4 Protein A NoAb control (window size 500 bp, step               analyzed ELISA using histone H3 peptides containing different <br /> size 100 bp, Bonferroni p-value correction). 5408 peaks                 modifications. Microtiter plates coated indicated peptides <br /> identified MES-4 38 NoAb control. 17 MES-4                   (full sequences Table 1 [33]) incubated increasingly <br /> peaks 1 step (100 bp) NoAb peak                   higher dilutions antibody (starting 1:100 dilution a <br /> eliminated, producing final set 5391 MES-4 peaks. Expected            hybridoma culture supernatant). After incubation peroxidase- <br /> values number peaks different chromosomes                     conjugated secondary antibody washing, colorimetric <br /> calculated dividing total number peaks chromosome               signal tetramethylbenzidine detected measuring the <br /> length (Figure S2D).                                                         absorbance 405 nm (Abs) using plate reader. <br />                                                                              Found at: doi:10.1371/journal.pgen.1001091.s001 (0.36 MB TIF) <br /> Peak-gene annotation analysis                                                Figure S2 Specificity MES-4 ChIP reagents distribution <br />    The list gene coordinates (transcript start-end) down- <br />                                                                              MES-4 peaks chromosomes. (A&#226;&#8364;&#8220;C) One-cell embryos at <br /> loaded WormBase (http://www.wormbase.org/). We ana- <br />                                                                              pronuclear meeting doubly stained DAPI SDI anti- <br /> lyzed MES-4 peaks respect genomic annotations three <br />                                                                              MES-4 antibody (A, B) triply stained DAPI, anti-FLAG, <br /> ways. First, MES-4 peaks overlap gene 1 bp <br />                                                                              anti-MRG-1 (C). Arrows point X chromosomes. (A) Wild- <br /> in gene body 1 kb gene&#226;&#8364;&#8482;s 59 39 were <br />                                                                              type embryo. (B) mes-4(bn85) embryo. The middle panel represents <br /> identified (Table S1A). Second, MES-4 peak could <br />                                                                              longer exposure shown A demonstrate complete <br /> overlap multiple genes, genes overlap MES-4 peak <br />                                                                              lack detectable MES-4. (C) mes-4(bn73) embryo carrying the <br /> by 1 bp gene body identified, producing a <br />                                                                              MES-4::GFP::FLAG transgene. MRG-1 marks autosomes. <br /> set &#226;&#8364;&#732;&#226;&#8364;&#732;MES-4-bound&#226;&#8364;&#8482;&#226;&#8364;&#8482; genes. Third, determined the <br />                                                                              Scale bar: 5 mm. (D) For chromosome number of <br /> distribution MES-4 peaks different genomic annotations <br />                                                                              observed MES-4 peaks given. The numbers expected peaks <br /> using Cis-regulatory Element Annotation System (CEAS) [61]. In <br />                                                                              calculated assuming uniform distribution MES-4 peaks <br /> CEAS, peak assigned single coordinate using <br />                                                                              chromosomes based length. The number observed <br /> WormBase version WS170 location coordinate with <br />                                                                              peaks X highlighted asterisk. <br /> respect annotation classes exon, intron, 59, 39, (in <br />                                                                              Found at: doi:10.1371/journal.pgen.1001091.s002 (2.95 MB TIF) <br /> our case .1 kb away) determined. We chose probe the <br /> maximum enrichment value peak&#226;&#8364;&#8482;s coordinate. One                         Figure S3 High concordance MES-4, MES-4::FLAG, <br /> coordinate assigned multiple genomic annotations.                  Pol II, H3K36me2/me3 ChIPs. (A) Genome browser view <br /> Therefore, assign peak single region, imposed               showing reproducibility MES-4 MES-4::FLAG ChIPs <br /> following hierarchy: exon, intron, 39, 59, 1 kb away. For                    similarity MES-4 MES-4::FLAG ChIP <br /> comparison, probes microarray run                distributions. The z-scores biological replicates average <br /> CEAS parameters. Table S1B reports results              z-scores shown {similar, tilde operator } <br /> MES-4 peak assignments exons introns.                                 180 kb region ChrIV shown Figure 3. (B) Heatmap of <br />                                                                              Pearson correlation coefficients ChIP biological replicates and <br /> Transcription profiling early embryos                                   ChIP performed different targets. Correlations were <br />   RNA isolated early embryo preparations                calculated based z-scores probes microarrays, <br /> used ChIP using Trizol purified using RNeasy kit                 median smoothing 250 bp. Green indicates positive <br /> (Qiagen, catalog 74104). Samples analyzed Agilent                 correlation, red indicates anti-correlation, white indicates no <br /> Bioanalyzer ensure rRNAs degraded                  correlation. Numbers cells indicate rounded correlation <br /> RNA free protein DNA contamination. Two                    values. <br /> samples treated DNase 30 min room                           Found at: doi:10.1371/journal.pgen.1001091.s003 (2.82 MB TIF) <br />  <br />  <br />       PLoS Genetics | www.plosgenetics.org                              12                    September 2010 | Volume 6 | Issue 9 | e1001091 <br />                                                                                                               Epigenetic Role H3K36 Methylation <br />  <br />  <br /> Figure S4 MES-4 bound genes largely overlap germline-                         and/or preferentially recovered ChIP steps depend on <br /> expressed genes. (A) Venn diagram showing overlap MES-4-                   strength <span id='am-239' about='xsp:base' typeof='owl:Thing'>base</span> pairing [e.g. Quail MA, Kozarewa I, Smith F, <br /> bound genes genes germline-expressed gene set. The                Scally A, Stephens PJ, et al. (2008) A large genome center&#226;&#8364;&#8482;s <br /> overlap enriched ,3-fold overlap expected chance               improvements Illumina sequencing system. Nat Methods 5: <br /> (p-value,102300). Notably, average SAGE tag count                  1005&#226;&#8364;&#8220;1010]. <br /> germline SAGE library [34] 1906 germline-expressed genes               Found at: doi:10.1371/journal.pgen.1001091.s005 (0.99 MB <br /> not MES-4-bound 1.6, average tag count                      TIF) <br /> germline-expressed genes MES-4-bound 14.6, indicat- <br /> ing germline-expressed genes MES-4-bound                  Figure S6 Overlap genes bound MES-4 embryos <br /> very weakly expressed germline possibly incorrectly                 genes mis-regulated mes-4 mutant germ lines. Based on <br /> classified germline-expressed. (B) Venn diagram showing                    transcript profiling dissected adult germ lines wild type and <br /> overlap MES-4-bound genes embryo-expressed gene                   mes-4 mutants, 352 X-linked genes 270 autosomal genes are <br /> set. The overlap , 1/2 expected chance (p-value ,               significantly mis-regulated (False Discovery Rate [FDR],0.05) in <br /> 10213). Of 90 genes overlapping, 50 actually                   mes-4 mutants compared wild type ([8] unpublished <br /> germline-expressed based SAGE [34] germline-enriched                    results). The overlap mis-regulated genes genes <br /> based microarray analysis [21], strengthening observed                 bound MES-4 small, greater expected chance for <br /> depletion MES-4 embryo-expressed gene set. (C)                    categories genes: X-linked genes up-regulated mes-4 <br /> Scatter plots MES-4 vs Pol II (top row), H3K36me3 vs Pol II                mutants, autosomal genes down-regulated mes-4 mutants. <br /> (middle row), MES-4 vs H3K36me3 (bottom row) genes                There numerous possible explanations relatively low <br /> (gray) ubiquitously-expressed (orange), germline-expressed               overlap MES-4-bound genes genes mis-regulated in <br /> (green), germline-specific (blue), embryo-expressed (red) genes           mes-4 mutants. MES-4 affect expression of the <br /> highlighted various columns. For gene, mean z-score             genes resides. MES-4 directly influence small <br /> was calculated averaging ChIP signal probes                 number genes, turn control genes&#226;&#8364;&#8482; expression. <br /> located transcript start end site. Most striking          MES-4 directly influence expression of target <br /> the low Pol II, high MES-4, high H3K36me3 pattern                     genes, 1) effect loss MES-4 requires analysis of <br /> germline-specific genes, medium-high Pol II, low MES-4,               PGCs instead adult germ cells, 2) MES-4 transiently <br /> low H3K36me3 pattern embryo-expressed genes.                          associates regulates target genes, failed to <br /> Found at: doi:10.1371/journal.pgen.1001091.s004 (2.07 MB TIF)                 capture transient associations, 3) factors mask the <br />                                                                               effect loss MES-4. Even documented transcription <br /> Figure S5      MES-4 H3K36 methyl marks exonic <br />                                                                               regulators generally display low overlap genes bound <br /> enrichment. (A) Pol II, MES-4, H3K36me2, H3K36me3 are <br />                                                                               genes mis-regulated mutants, variety biological as <br /> elevated exons compared introns. This especially apparent <br />                                                                               technical reasons [e.g. 58,59]. <br /> on genes long introns. (B) Several ChIP targets, including <br />                                                                               Found at: doi:10.1371/journal.pgen.1001091.s006 (0.60 MB <br /> MES-4 H3K36me marks, increase exon starts and <br /> decrease exon ends. 2767 intron-exon-intron triplets              TIF) <br /> exons introns 300 bp identified binned            Table S1 Analysis MES-4 binding sites. (A) Distribution of <br /> transcript level Figure 4. Profiles MES-4 ChIPs           MES-4 peaks respect underlying genes. The number of <br /> are shown 300 bp upstream 200 bp downstream                      MES-4 peaks overlap gene body, 1 kb 59 39 <br /> exon start sites (St) 200 bp upstream 300 bp downstream                gene, .1 kb away gene given. (B) Number of <br /> from exon end sites (End). MES-4, Pol II, H3K36me2                        MES-4 peaks maximum coordinate maps exon or <br /> H3K36me3 levels elevated exons relative introns                  intron. The middle coordinate probe maximum <br /> expressed genes. The recently published [37]. (C)                  ChIP value MES-4 peak mapped. P-values were <br /> Analysis separated IP input signals. Profiles log2 scores        calculated chi-square test observed expected <br /> IP input 2767 intron-exon-intron triplets             distributions. Expected distributions determined all <br /> (B) shown. To better exonic enrichment respect               probes microarray map exon intron. (C) Gene <br /> intronic signal, triplet average log2 score intronic          Ontology terms associate MES-4-bound genes. Seven <br /> probes subtracted. MES-4, H3K36me3 H3 IPs clearly                     representative terms significantly associate MES-4- <br /> show exonic enrichment; input shows significant exonic               bound genes shown. <br /> enrichment IP. H3K9me3 IP shows exonic enrichment;                  Found at: doi:10.1371/journal.pgen.1001091.s007 (0.46 MB TIF) <br /> input shows exonic enrichment, leading exonic <br /> depletion log2 ratios IP Input H3K9me3. NoAb Prot              Table S2 Lists genes different temporal-spatial classes. See <br /> A beads IgG exonic enrichment IP                   Results explanation ubiquitous, germline-expressed, <br /> input. Recent findings suggest codon composition and/or                  germline-specific, embryo-expressed, soma-specific gene sets <br /> higher GC content exons bias occupied                 derived published microarray SAGE data [21,34&#226;&#8364;&#8220; <br /> nucleosomes [Schwartz S, Meshorer E, Ast G (2009) Chromatin                   36]. Silent genes [37]. <br /> organization marks exon-intron structure. Nat Struct Mol Biol 16:             Found at: doi:10.1371/journal.pgen.1001091.s008 (0.16 MB <br /> 990&#226;&#8364;&#8220;995; Tilgner H, Nikolaou C, Althammer S, Sammeth M,                       XLS) <br /> Beato M, et al. (2009) Nucleosome positioning determinant             Table S3 List genes high MES-4 H3K36me3 (z- <br /> exon recognition. Nat Struct Mol Biol 16: 996&#226;&#8364;&#8220;1001]. Increased                scores.1) low Pol II (z score,0). <br /> nucleosome occupancy turn lead higher levels histone <br />                                                                               Found at: doi:10.1371/journal.pgen.1001091.s009 (0.04 MB <br /> modifiers histone modifications exons. The fact of <br />                                                                               XLS) <br /> the ChIP signals analyzed, including H3 naked Protein G <br /> beads input chromatin, display degree exonic                 Text S1 Supplemental methods. <br /> enrichment suggests exonic DNA generally GC-rich                 Found at: doi:10.1371/journal.pgen.1001091.s010 (0.04 MB <br /> DNA preferentially solubilized chromatin extraction                 DOC) <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                              13                     September 2010 | Volume 6 | Issue 9 | e1001091 <br />                                                                                                                                      Epigenetic Role H3K36 Methylation <br />  <br />  <br />  <br /> Acknowledgments                                                                             Author Contributions <br /> We thank Jinkyo Suh, Hirofumi Furuhashi, Bill Kelly sharing                         Conceived designed experiments: AR SE TT TMP JDL SS. <br /> unpublished results Bill Kelly, Hirofumi Furuhashi, members                      Performed experiments: TT TMP TAE. Analyzed data: AR SE <br /> Strome Lieb labs, members modENCODE                                    TT. Contributed reagents/materials/analysis tools: WW HK. Wrote the <br /> consortium helpful discussions.                                                         paper: AR SE TT JDL SS. <br />  <br /> References <br />  1. Allis CD, Jenuwein T, Reinberg D (2007) Epigenetics - Overview concepts.            27. Carrozza MJ, Li B, Florens L, Suganuma T, Swanson SK, et al. (2005) Histone <br />     Epigenetics Cold Spring Harbor Laboratory Press. Allis, Jenuwein, Reinberg,                 H3 methylation Set2 directs deacetylation coding regions Rpd3S to <br />     eds. pp 23&#226;&#8364;&#8220;61.                                                                              suppress spurious intragenic transcription. Cell 123: 581&#226;&#8364;&#8220;592. <br />  2. Felsenfeld G, Groudine M (2003) Controlling double helix. Nature 421:               28. Joshi AA, Struhl K (2005) Eaf3 chromodomain interaction methylated H3- <br />     448&#226;&#8364;&#8220;453.                                                                                    K36 links histone deacetylation Pol II elongation. Mol Cell 20: 971&#226;&#8364;&#8220;978. <br />  3. Luger K, Mader AW, Richmond RK, Sargent DF, Richmond TJ (1997) Crystal                  29. Li B, Gogol M, Carey M, Lee D, Seidel C, et al. (2007) Combined action of <br />     structure nucleosome core particle 2.8 A resolution. Nature 389:                  PHD chromo domains directs Rpd3S HDAC transcribed chromatin. <br />     251&#226;&#8364;&#8220;260.                                                                                    Science 316: 1050&#226;&#8364;&#8220;1054. <br />  4. Kouzarides T (2007) Chromatin modifications function. Cell 128:               30. Sural TH, Peng S, Li B, Workman JL, Park PJ, et al. (2008) The MSL3 <br />     693&#226;&#8364;&#8220;705.                                                                                    chromodomain directs key targeting step dosage compensation the <br />  5. Strahl BD, Allis CD (2000) The language covalent histone modifications.                  Drosophila melanogaster X chromosome. Nat Struct Mol Biol 15: 1318&#226;&#8364;&#8220;1325. <br />     Nature 403: 41&#226;&#8364;&#8220;45.                                                                      31. Buck MJ, Nobel AB, Lieb JD (2005) ChIPOTle: user-friendly tool the <br />  6. Margueron R, Justin N, Ohno K, Sharpe ML, Son J, et al. (2009) Role                  analysis ChIP-chip data. Genome Biol 6: R97. <br />     polycomb protein EED propagation repressive histone marks. Nature             32. Cho EJ, Kobor MS, Kim M, Greenblatt J, Buratowski S (2001) Opposing effects <br />     461: 762&#226;&#8364;&#8220;767.                                                                               Ctk1 kinase Fcp1 phosphatase Ser 2 RNA polymerase II C- <br />  7. Muller J, Verrijzer P (2009) Biochemical mechanisms gene regulation                   terminal domain. Genes Dev 15: 3319&#226;&#8364;&#8220;3329. <br />     polycomb group protein complexes. Curr Opin Gen Dev 19: 1&#226;&#8364;&#8220;9.                            33. Kimura H, Hayashi-Takanaka Y, Goto Y, Takizawa N, Nozaki N (2008) The <br />  8. Bender LB, Suh J, Carroll CR, Fong Y, Fingerman IM, et al. (2006) MES-4:                 organization histone H3 modifications revealed panel specific <br />     autosome-associated histone methyltransferase participates silencing            monoclonal antibodies. Cell Struct Funct 33: 61&#226;&#8364;&#8220;73. <br />     X chromosomes C. elegans germ line. Development 133: 3907&#226;&#8364;&#8220;3917.                  34. Wang X, Zhao Y, Wong K, Ehlers P, Kohara Y, et al. (2009) Identification of <br />  9. Strahl BD, Grant PA, Briggs SD, Sun ZW, Bone JR, et al. (2002) Set2                    genes expressed hermaphrodite germ line C. elegans using SAGE. BMC <br />     nucleosomal histone H3-selective methyltransferase mediates transcriptional            Genomics 10: 213. <br />     repression. Mol Cell Biol 22: 1298&#226;&#8364;&#8220;1306.                                                35. Baugh LR, Hill AA, Slonim DK, Brown EL, Hunter CP (2003) Composition <br /> 10. Andersen EC, Horvitz HR (2007) Two C. elegans histone methyltransferases                    dynamics Caenorhabditis elegans early embryonic transcriptome. <br />     repress lin-3 EGF transcription inhibit vulval development. Development 134:             Development 130: 889&#226;&#8364;&#8220;900. <br />     2991&#226;&#8364;&#8220;2999.                                                                              36. Meissner B, Warner A, Wong K, Dube N, Lorch A, et al. (2009) An integrated <br /> 11. Bell O, Wirbelauer C, Hild M, Scharf AN, Schwaiger M, et al. (2007) Localized               strategy study muscle development myofilament structure Caenorhabditis <br />     H3K36 methylation states define histone H4K16 acetylation transcrip-                 elegans. PLoS Genet 5: e1000537. doi:10.1371/journal.pgen.1000537. <br />     tional elongation Drosophila. EMBO J 26: 4974&#226;&#8364;&#8220;4984.                                  37. Kolasinska-Zwierz P, Down T, Latorre I, Liu T, Liu XS, et al. (2009) <br />                                                                                                 Differential chromatin marking introns expressed exons H3K36me3. <br /> 12. Larschan E, Alekseyenko AA, Gortchakov AA, Peng S, Li B, et al. (2007) MSL <br />                                                                                                 Nat Genet 41: 376&#226;&#8364;&#8220;381. <br />     complex attracted genes marked H3K36 trimethylation using a <br />                                                                                             38. Reinke V, Cutter AD (2009) Germline expression influences operon organiza- <br />     sequence-independent mechanism. Mol Cell 28: 121&#226;&#8364;&#8220;133. <br />                                                                                                 tion Caenorhabditis elegans genome. Genetics 181: 1219&#226;&#8364;&#8220;1228. <br /> 13. Sun XJ, Wei J, Wu XY, Hu M, Wang L, et al. (2005) Identification and <br />                                                                                             39. Bannister AJ, Schneider R, Myers FA, Thorne AW, Crane-Robinson C, et al. <br />     characterization novel human histone H3 lysine 36-specific methyltrans- <br />                                                                                                 (2005) Spatial distribution di- tri-methyl lysine 36 histone H3 active <br />     ferase. J Biol Chem 280: 35261&#226;&#8364;&#8220;35271. <br />                                                                                                 genes. J Biol Chem 280: 17732&#226;&#8364;&#8220;17736. <br /> 14. Li Y, Trojer P, Xu CF, Cheung P, Kuo A, et al. (2009) The target NSD <br />                                                                                             40. Dudley NR, Labbe JC, Goldstein B (2002) Using <span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span>RNA interference</span> identify <br />     family histone lysine methyltransferases depends nature the <br />                                                                                                 genes required RNA interference. Proc Natl Acad Sci U S A 99: 4191&#226;&#8364;&#8220;4196. <br />     substrate. J Biol Chem 284: 34283&#226;&#8364;&#8220;34295. <br />                                                                                             41. Kelly WG, Fire A (1998) Chromatin silencing maintenance a <br /> 15. Nimura K, Ura K, Shiratori H, Ikawa M, Okabe M, et al. (2009) A histone H3                  functional germline Caenorhabditis elegans. Development 125: 2451&#226;&#8364;&#8220;2456. <br />     lysine 36 trimethyltransferase links Nkx2-5 Wolf-Hirschhorn syndrome.                42. Robert VJ, Sijen T, van Wolfswinkel J, Plasterk RH (2005) Chromatin and <br />     Nature 460: 287&#226;&#8364;&#8220;291.                                                                        RNAi factors protect C. elegans germline repetitive sequences. Genes <br /> 16. Rayasam GV, Wendling O, Angrand PO, Mark M, Niederreither K, et al.                         Dev 19: 782&#226;&#8364;&#8220;787. <br />     (2003) NSD1 essential early post-implantation development              43. Unhavaithaya Y, Shin TH, Miliaras N, Lee J, Oyama T, et al. (2002) MEP-1 <br />     catalytically active SET domain. EMBO J 22: 3153&#226;&#8364;&#8220;3163.                                      homolog NURD complex component Mi-2 act maintain <br /> 17. Wang GG, Cai L, Pasillas MP, Kamps MP (2007) NUP98-NSD1 links H3K36                         germline-soma distinctions C. elegans. Cell 111: 991&#226;&#8364;&#8220;1002. <br />     methylation Hox-A gene activation leukaemogenesis. Nat Cell Biol 9:              44. Wang D, Kennedy S, Conte D, Jr., Kim JK, Gabel HW, et al. (2005) Somatic <br />     804&#226;&#8364;&#8220;812.                                                                                    misexpression germline P granules enhanced RNA interference in <br /> 18. Capowski EE, Martin P, Garvin C, Strome S (1991) Identification                          retinoblastoma pathway mutants. Nature 436: 593&#226;&#8364;&#8220;597. <br />     grandchildless loci products required normal germline                     45. Ercan S, Giresi PG, Whittle CM, Zhang X, Green RD, et al. (2007) X <br />     development nematode Caenorhabditis elegans. Genetics 129: 1061&#226;&#8364;&#8220;1072.                chromosome repression localization C. elegans dosage compensation <br /> 19. Fong Y, Bender L, Wang W, Strome S (2002) Regulation different                       machinery sites transcription initiation. Nat Genet 39: 403&#226;&#8364;&#8220;408. <br />     chromatin states autosomes X chromosomes germ line C. elegans.         46. Jans J, Gladden JM, Ralston EJ, Pickle CS, Michel AH, et al. (2009) A <br />     Science 296: 2235&#226;&#8364;&#8220;2238.                                                                     condensin-like dosage compensation complex acts distance control <br /> 20. Kelly WG, Schaner CE, Dernburg AF, Lee MH, Kim SK, et al. (2002) X-                         expression genome. Genes Dev 23: 602&#226;&#8364;&#8220;618. <br />     chromosome silencing germline C. elegans. Development 129: 479&#226;&#8364;&#8220;492.           47. Gu SG, Fire A (2010) Partitioning C. elegans genome nucleosome <br /> 21. Reinke V, Gil IS, Ward S, Kazmer K (2004) Genome-wide germline-enriched                     modification, occupancy, positioning. Chromosoma 119: 73&#226;&#8364;&#8220;87. <br />     sex-biased expression profiles Caenorhabditis elegans. Development 131:          48. Phillips CM, Meng X, Zhang L, Chretien JH, Urnov FD, et al. (2009) <br />     311&#226;&#8364;&#8220;323.                                                                                    Identification chromosome sequence motifs mediate meiotic pairing and <br /> 22. Kizer KO, Phatnani HP, Shibata Y, Hall H, Greenleaf AL, et al. (2005) A novel               synapsis C. elegans. Nat Cell Biol 11: 934&#226;&#8364;&#8220;942. <br />     domain Set2 mediates RNA polymerase II interaction couples histone H3            49. Hong RL, Sommer RJ (2006) Pristionchus pacificus: well-rounded nematode. <br />     K36 methylation transcript elongation. Mol Cell Biol 25: 3305&#226;&#8364;&#8220;3316.                    Bioessays 28: 651&#226;&#8364;&#8220;659. <br /> 23. Krogan NJ, Kim M, Tong A, Golshani A, Cagney G, et al. (2003) Methylation               50. Bender LB, Cao R, Zhang Y, Strome S (2004) The MES-2/MES-3/MES-6 <br />     histone H3 Set2 Saccharomyces cerevisiae linked transcriptional              complex regulation histone H3 methylation C. elegans. Curr Biol 14: <br />     elongation RNA polymerase II. Mol Cell Biol 23: 4207&#226;&#8364;&#8220;4218.                               1639&#226;&#8364;&#8220;1643. <br /> 24. Li B, Howe L, Anderson S, Yates JR, 3rd, Workman JL (2003) The Set2 histone             51. Ketel CS, Andersen EF, Vargas ML, Suh J, Strome S, et al. (2005) Subunit <br />     methyltransferase functions phosphorylated carboxyl-terminal                    contributions histone methyltransferase activities fly worm polycomb <br />     domain RNA polymerase II. J Biol Chem 278: 8897&#226;&#8364;&#8220;8903.                                    group complexes. Mol Cell Biol 25: 6857&#226;&#8364;&#8220;6868. <br /> 25. Pokholok DK, Harbison CT, Levine S, Cole M, Hannett NM, et al. (2005)                   52. Xu L, Fong Y, Strome S (2001) The Caenorhabditis elegans maternal-effect sterile <br />     Genome-wide map nucleosome acetylation methylation yeast. Cell                    proteins, MES-2, MES-3, MES-6, associated complex embryos. <br />     122: 517&#226;&#8364;&#8220;527.                                                                               Proc Natl Acad Sci U S A 98: 5061&#226;&#8364;&#8220;5066. <br /> 26. Rao B, Shibata Y, Strahl BD, Lieb JD (2005) Dimethylation histone H3              53. Du HN, Fingerman IM, Briggs SD (2008) Histone H3 K36 methylation is <br />     lysine 36 demarcates regulatory nonregulatory chromatin genome-wide.                    mediated trans-histone methylation pathway involving interaction <br />     Mol Cell Biol 25: 9447&#226;&#8364;&#8220;9459.                                                                Set2 histone H4. Genes Dev 22: 2786&#226;&#8364;&#8220;2798. <br />  <br />  <br />  <br />         PLoS Genetics | www.plosgenetics.org                                           14                          Septem <br /> </body></html>