<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>An Integrative Multi-Network Multi-Classifier <br /> Approach Predict Genetic Interactions <br /> Gaurav Pandey1.&#194;&#164;a, Bin Zhang2,3.*, Aaron N. Chang2&#194;&#164;b, Chad L. Myers1, Jun Zhu2,3, Vipin Kumar1, <br /> Eric E. Schadt2&#194;&#164;c* <br /> 1 Department Computer Science Engineering, University Minnesota, Twin Cities, Minneapolis, Minnesota, United States America, 2 Rosetta Inpharmatics, LLC, <br /> Seattle, Washington, United States America, 3 Sage Bionetworks, Seattle, Washington, United States America <br />  <br />  <br />  <br />      <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />      Genetic interactions occur combination mutations results surprising <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span>phenotype</span>. These interactions capture <br />      functional redundancy, important predicting function, dissecting protein complexes functional <br />      pathways, exploring mechanistic underpinnings common human diseases. Synthetic sickness lethality are <br />      studied types genetic interactions yeast. However, yeast, small proportion gene pairs have <br />      tested genetic interactions large number possible combinations gene pairs. <span id='am-3' about='protege:TO' typeof='owl:Thing'>To</span> expand set of <br />      known synthetic lethal (SL) interactions, devised integrative, multi-network approach predicting these <br />      interactions significantly improves existing approaches. First, defined large number features for <br />      characterizing relationships pairs genes various data sources. In particular, features are <br />      independent known SL interactions, contrast previous approaches. Using features, developed a <br />      non-parametric multi-classifier predicting SL interactions enabled simultaneous use multiple <br />      classification procedures. Several comprehensive experiments demonstrated SL-independent features in <br />      conjunction advanced classification scheme led improved performance compared current <br />      state art method. Using approach, derived yeast transcription factor genetic interaction network, part <br />      supported literature. We used approach predict SL interactions non-essential <br />      gene pairs yeast (http://sage.fhcrc.org/downloads/downloads/predicted_yeast_genetic_interactions.zip). This integrative <br />      approach expected effective robust uncovering new genetic interactions tens millions of <br />      unknown gene pairs yeast hundreds millions gene pairs higher organisms like mouse human, in <br />      genetic interactions identified date. <br />  <br />   Citation: Pandey G, Zhang B, Chang AN, Myers CL, Zhu J, et al. (2010) An Integrative Multi-Network Multi-Classifier Approach Predict Genetic <br />   Interactions. PLoS Comput Biol 6(9): e1000928. doi:10.1371/journal.pcbi.1000928 <br />   Editor: Gary D. Bader, University Toronto, Canada <br />   Received September 18, 2009; Accepted August 10, 2010; Published September 9, 2010 <br />   Copyright: &#195;&#376; 2010 Pandey et al. This open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: This work supported Merck &amp; Co. The funders role study design, data collection analysis, decision publish, or <br />   preparation manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: bin.zhang@sagebase.org (BZ); eschadt@pacificbiosciences.com (EES) <br />   &#194;&#164;a Current address: Department Plant Microbial Biology, University California, Berkeley, Berkeley, California, United States America. <br />   &#194;&#164;b Current address: Regulus Therapeutics, Carlsbad, California, United States America. <br />   &#194;&#164;c Current address: Pacific Biosciences, Menlo Park, California, United States America. <br />   . These authors contributed equally. <br />  <br />  <br />  <br />  <br /> Introduction                                                                           SSL interactions uncovered limited contexts <br />                                                                                        interactions assessed [1]. <br />     Genetic interactions occur combination mutations                            To expand set known SSL interactions, efforts <br /> results surprising phenotype. These interactions capture                          undertaken build models predict genetic <br /> functional redundancy, important predicting                           interactions, particularly SSL ones, yeast organisms <br /> function [1,2], dissecting protein complexes functional                           [4,5,6,7,8,9,10,11]. The multiple network decision tree (MNDT) <br /> pathways [3] exploring sources underlying complex                              approach [8] represents comprehensive work predict <br /> inherited human diseases [1].                                                          SSL interactions high precision. MNDT extracted both <br />     In yeast, systematic deletion genes (,6000)                    SSL-dependent features (referred 2-hop features least <br /> instrumental delineating non-essential genes                            networks known SSL interaction <br /> combination gene mutations lead loss                            network), SSL-independent features (the known SSL network <br /> viability. However, testing pair-wise combinations                        involved) train decision tree-based classifier. Given two <br /> genes genetic interactions different conditions                      networks characterizing relationships genes, 2-hop <br /> still prohibitive terms time materials. Synthetic sickness                   feature genes A B used represent is <br /> and lethality (SSL) studied types genetic                              2-step path A B gene C two <br /> interactions yeast. However, small portion possible                   links (A&#226;&#8364;&#8220;C B&#226;&#8364;&#8220;C) different networks [8]. For example, the <br />  <br />  <br />         PLoS Computational Biology | www.ploscompbiol.org                          1                        September 2010 | Volume 6 | Issue 9 | e1000928 <br />                                                                                                                     Predicting Genetic Interactions <br />  <br />  <br />  <br />   Author Summary                                                             interaction networks, used SVM classifier to <br />                                                                              predict SSL interactions [11]. Zhong Sternberg [9] predicted <br />   Genetic interactions occur combination                           genetic interactions C. elegans genes using machine <br />   mutations results surprising phenotype. These                         learning approach. However, limited of <br />   interactions capture functional redundancy,                   genome-wide data available C. elegans, integrated <br />   important predicting function, dissecting protein                      features orthologous gene pairs yeast fly. Although they <br />   complexes functional pathways, exploring                      predictions C. elegans genes, provided <br />   mechanistic underpinnings common human diseases.                        estimates accuracy predictor using examples of <br />   Due large number possible combinations                        specific pathways biological systems, making compre- <br />   genes, small portion gene pairs yeast                hensive estimation utility biological features C. <br />   pairs higher organisms like mouse human <br />                                                                              elegans itself (organism-specific features) difficult. Overall, MNDT <br />   tested. Therefore, predicting genetic interac- <br />                                                                              [8] remains extensive effective organism- <br />   tions received significant attention past several <br />   years. The existing methods primarily rely known                    specific approach literature extracting integrating a <br />   genetic interactions, far effective                   wide variety biological features predicting genetic interac- <br />   classifying gene pairs connected                        tions basis validating new algorithms [9]. <br />   known genetic interactions. Here developed non-                          In paper, build existing approaches by <br />   parametric multi-classifier predicting genetic                  developing Multi-Network Multi-Classifier (MNMC) <br />   interactions based large number novel features                     framework predicts SL interactions yeast effectively. <br />   independent known genetic interactions. This                        The enhanced accuracy predictions achieved by <br />   approach led improved performance com-                          incorporating comprehensive set SL-independent <br />   pared current state-of-the-art method. Using                   features capture relationships genes, and <br />   approach, derived yeast transcription factor                  simultaneously employing multiple classification procedures, thus <br />   genetic interaction network,                        leveraging strengths reducing effects their <br />   supported literature. This integrative approach                      respective weaknesses. Since method based SL- <br />   expected effective robust uncovering                     independent features, appropriate settings where <br />   new genetic interactions yeast species.                       gene pairs tested, including higher organisms <br />                                                                              large-scale genetic interaction screens feasible. <br /> physical-SSL relationship, link protein                We applied approach predict genetic <br /> interaction network the SSL interaction                   interactions known transcription factors (TFs) S. <br /> network, SSL-dependent feature involves known             cerevisiae uncovered number novel SL interactions between <br /> SSL interactions. The results MNDT showed                   TFs, supported available knowledge about <br /> effective features differentiating SSL non-SSL interactions              TFs. We expanded effort predicting genetic <br /> were 2-hop features derived overlay multiple             interactions approximately 7.5 million pairs non- <br /> networks, particularly networks known                essential S. cerevisiae genes, results available at <br /> SSL network. However, use SSL-independent features               http://sage.fhcrc.org/downloads/downloads/predicted_yeast_- <br /> in study led true positive rate 40%        genetic_interactions.zip. The details results the <br /> false positive rate 20%. Given small fraction          materials methods used subsequent <br /> total set gene pairs various organisms tested            sections. However, discussion, like note <br /> SSL interactions, SSL-dependent features available           focused prediction SL interactions, we <br /> for remaining gene pairs, thus, prediction            demonstrate approach capable predicting other <br /> those new pairs expected low accuracy. This             categories genetic interactions, like synthetic sickness. <br /> problem exacerbated higher organisms very <br /> few genetic interactions identified date. Chipman               Results <br /> and Singh extended MNDT approach utilizing existing <br /> SL data, features gene expression, protein              The step building classifier predict SL pairs the <br /> interaction GO functional annotation data, predict SSL                identification set features treat variables the <br /> interactions S. cerevisiae C. elegans genes [4]. However,          prediction procedure. The ideal features case that <br /> the primary focus work demonstrate random                capture information relationships genes. <br /> walks networks produced effective features 2-hop                Towards end, extracted 152 SL-independent features <br /> features prediction problem, did help               (no known SL interaction involved) number sources, <br /> address problem dependence SSL data making novel               including multiple gene expression studies [12,13,14,15], protein- <br /> SSL predictions. Particular emphasis placed              protein interaction databases (www.yeastgenome.org Sept <br /> to best use features addressing problem.                  2007), transcription factor binding databases [16], functional <br />    In specialized approach, Qi et al. focused network          annotations defined Gene Ontology (www.yeastgenome. <br /> of SSL interactions genes S. cerevisiae, used diffusion       org May 2008), gene network modules clique <br /> kernels defined network SVM classifier predict          communities [16]. Among 152 features identified, 62 were <br /> several novel SSL interactions [7]. They predicted              intended capture likelihood genes directly <br /> pairs functionally associated genes high likelihood        related (e.g., co-regulated gene expression studies, <br /> belonging complexes, pathways GO functional                  protein/DNA sequence similarity, direct physical interactions <br /> classes. However, approach faces challenge Wong             PPI network). The 90 features derived by <br /> et al.&#226;&#8364;&#8482;s approach effective predicting genetic        overlaying pairs networks (individual features) using a <br /> (SSL) interactions genes represented            methodology similar used deriving binary 2-hop <br /> in known SSL network. Among efforts, Paladugu et al.               features previously described MNDT approach [8]. In <br /> focused extracting multiple features protein                    MNDT, overlay (2-hop) features essentially capture binary <br />  <br />  <br />        PLoS Computational Biology | www.ploscompbiol.org                 2                     September 2010 | Volume 6 | Issue 9 | e1000928 <br />                                                                                                                            Predicting Genetic Interactions <br />  <br />  <br /> transitive relationships gene pairs. We extended                   Text S1 presents complete ranking 152 SL- <br /> approach computing weights edges overlaid                     independent features 15 SL-dependent ones (the known <br /> networks exhaustive search strongest transitive link             SL interactions involved) considered study. Not <br /> (maximum product weights input edges)                surprisingly, features derived physical protein interactions <br /> a set weighted networks, shown Figure 1. Therefore,                functional annotations discriminative, <br /> &#226;&#8364;&#732;&#226;&#8364;&#732;overlay&#226;&#8364;&#8482;&#226;&#8364;&#8482; feature generalization 2-hop one,              consistent previous findings [8,18]. <br /> involves finding strongest link connecting genes,                       Using set features, developed integrative <br /> opposed finding &#226;&#8364;&#732;&#226;&#8364;&#732;any&#226;&#8364;&#8482;&#226;&#8364;&#8482; link 2-hop feature. Such                    classification predicting given gene pair is <br /> generalization makes use information weighted                    synthetic lethal not. As step, negative (majority) class <br /> networks overlaid, overlay features expected              (non-SL, 92.9% set) randomly under-sampled produce <br /> be effective predicting genetic interactions. For                 set negative examples size positive <br /> details compute overlay features, Materials                (minority) class (SL, 7.1% set) handling rare class <br /> and Methods section. A complete list features used                problem [19] data set. This balanced combination of <br /> study Table S1 Text S1.                                      sets used train non-parametric multi-classifier <br />                                                                                 enabled simultaneous use multiple classification <br /> Datasets classification method                                              procedures, SVM, neural networks decision trees. <br />    To characterize extent features                     Such multi-classifier combination (henceforth referred as <br /> differentiate SL non-SL classes, collected 9,994 SL                  ensemble MNMC) desirable complex problems like SL <br /> interactions 125,509 non-SL interactions SGD                       prediction involving noisy inputs, precise solutions a <br /> database (www.yeastgenome.org, May 2008). Note                  high coverage difficult achieve single classification <br /> of interactions SGD labeled SL actually synthetic              procedure [20] (see Materials Methods details). Note that <br /> sick (SS) interactions, stemming SS interactions               under-sampling applied training set, the <br /> having referred &#226;&#8364;&#732;synthetic lethal&#226;&#8364;&#8482; original                   true ratio number positive negative examples is <br /> publications [2,17]. However, (SS) interactions                 maintained test set. Thus, results presented are <br /> expected strong effects, exhibit similar                unbiased comparable methods. <br /> characteristics synthetic lethal interactions learning and <br /> classification. For simplicity, refer interactions        Validation <br /> SL paper. The SL network prepared comprised                        We tested individual classifiers ensemble <br /> 9994 SL interactions covering 2502 genes average,                    (MNMC) SGD-SL dataset. Figure 3(A) shows receiver <br /> each gene 8 connections, overall data set, referred            operating characteristic (ROC) curves seven classifiers based <br /> as SGD-SL, consisted 135,503 interactions.                                   10-fold cross-validation. As observe figure, <br />    We employed Kolmogorov-Smirnov (KS) test                        ensemble consistently outperformed individual classifiers, <br /> capture difference distributions feature sets            SVM classifier performing best individual <br /> of positive (SL) negative (non-SL) examples [11]. The D-                    classifiers k-NN performing worst. At false positive rate <br /> statistic KS test used measure                   20%, true positive rate ensemble roughly 55%, <br /> discriminative power feature. Figure 2 shows                        2% higher best individual classifier (SVM). In addition, <br /> distribution 6 discriminative features                  prediction precision (fraction number true SL <br /> classes, D-statistic values, Figure S1 Text             predictions size complete set SL predictions) the <br /> S1 shows ratio frequencies features. Table S1            ensemble high 49% vis-a-vis recall (fraction the <br />  <br />  <br />  <br />  <br /> Figure 1. Computing overlay features. One feature overlaid feature generate overlay feature. Each feature treated an <br /> undirected network genes nodes, value feature gene pair weight edge connecting them. <br /> Given weighted networks N1 N2 , corresponding features respectively, value overlay feature nodes A B, the <br /> strongest (max product weights edges) transitive link, i.e., max (w(A,Ci )   w(B,Ci )), (A,Ci ) [ N1 (Ci ,B) [ N2 vice versa. <br />                                                                                i <br /> doi:10.1371/journal.pcbi.1000928.g001 <br />  <br />  <br />        PLoS Computational Biology | www.ploscompbiol.org                    3                       September 2010 | Volume 6 | Issue 9 | e1000928 <br />                                                                                                                         Predicting Genetic Interactions <br />  <br />  <br />  <br />  <br /> Figure 2. The 6 discriminative SL-independent features used MNMC prediction approach. The Kolmogorov-Smirnov (KS) <br /> test used capture difference distributions feature SL non-SL classes. The D-statistics p-values KS test are <br /> shown here. The features left right are: (A) Pathway Comembership &#226;&#8364;&#8221; number pathways genes <br /> belong to, (B) SemanSim BP &#226;&#8364;&#8221; similarity genes using annotations GO Biological Process terms semantic similarity the <br /> terms, (C) PPI Community Comembership &#226;&#8364;&#8221; number PPI communities genes belong to, (D) O(PPI, SemanSim BP) &#226;&#8364;&#8221; overlay <br /> feature PPI SemanSim BP based networks, (E) Common Functions &#226;&#8364;&#8221; number common functions genes belong to, (F) <br /> SemanSim CC &#226;&#8364;&#8221; similarity genes using annotations GO Cellular Component terms semantic similarity terms. A <br /> description features used study Table S1 Text S1. <br /> doi:10.1371/journal.pcbi.1000928.g002 <br />  <br />  <br />        PLoS Computational Biology | www.ploscompbiol.org                  4                      September 2010 | Volume 6 | Issue 9 | e1000928 <br />                                                                                                                       Predicting Genetic Interactions <br />  <br />  <br />  <br />  <br /> Figure 3. Classification SL dataset. (A) Receiver operating characteristic (ROC) curves individual classifiers ensemble <br /> (MNMC) using SL-independent features. (B) ROC curves ensemble classifier data set using features (MNMC.all; AUC = 0.837) SL- <br /> independent features (MNMC.slif; AUC = 0.741). (C) Precision-recall curves using features (MNMC.all) SL-independent features (MNMC.slif). The <br /> corresponding ROC precision-recall curves random classifier shown. <br /> doi:10.1371/journal.pcbi.1000928.g003 <br />  <br />  <br /> number true SL predictions size complete set known              To evaluate importance SL-independent <br /> SL examples) 16.6% classification score threshold 0.2.            feature set predicting SL pairs, constructed positive test set <br /> This type high precision important predicting new SL               337 least-connected SL pairs network 9,994 <br /> interactions high confidence. We tested performance            known SL interactions, positive training set 9,129 <br /> of ensemble based expanded feature set including               positive SL examples did share gene test set. <br /> 152 SL-independent features additional set 15 SL-                  The connectivity SL pair SL network defined as <br /> dependent features (for details, Materials Methods). As              minimum degrees genes comprising pair. <br /> shown Figure 3(B), observed 15% increase true                 Therefore, 337 least-connected pairs, SL-dependent <br /> positive rate false positive rate 20% using expanded             features based network overlay missing less <br /> feature set (MNMC.all), SL-independent set                    effective well-connected pairs. The 337 SL gene <br /> (MNMC.slif), demonstrating ensemble able             pairs covered 283 unique genes, giving rise 199 pairs were <br /> to make effective use information provided features.           included non-SL interactions original data set used <br /> A similar advantage SL-dependent features reflected           negative test set. The negative training set comprised <br /> in precision-recall results experiment (Figure 3(C)).            125,310 non-SL interactions obtained removing negative <br />  <br />  <br />        PLoS Computational Biology | www.ploscompbiol.org                 5                      September 2010 | Volume 6 | Issue 9 | e1000928 <br />                                                                                                                       Predicting Genetic Interactions <br />  <br />  <br /> test set original 125,509 non-SL interactions data           discriminative 730 unseen samples. On contrary, the <br /> set. Predictions carried test sets using         discriminative features dataset become <br /> ensemble classifier trained using training sets. The overall             highly discriminative unseen samples. For example, top <br /> performance SL-independent SL-dependent features                 discriminative features, O(SemanSim CC, Brem Abs <br /> was evaluated terms respective ROC precision-recall            TOM), O(SemanSim CC, SemanSim BP) O(SemanSim MF, <br /> curves. As shown Figure 4(A), performance predictor            SemanSim BP) larger D-statistics (0.92, 0.88 0.85 <br /> based SL-independent feature set (MNMC.slif)                       respectively) unseen data set (0.03, 0.09 0.07 <br /> consistently better SL-dependent feature set                respectively) SGD-SL data set. These interesting <br /> (MNMC.all). For example, false positive rate 20%, SL-            observations actually point future research direction for <br /> independent features lead true positive rate 75%, 5%                 predicting genetic interactions: partition samples <br /> higher obtained SL-dependent features. Note               distinct groups based discriminative utility features <br /> that difference performance sets features        available train individual classifiers group. <br /> as large expected, allow Weka (our implementation <br /> platform) impute missing values, SL-dependent            Comparison MNDT approaches <br /> features absent training test set examples.                To evaluate effectiveness overall prediction approach, <br /> However, strictest case imputation allowed,         i.e., set features multi-classifier predictive model, we <br /> the gap expected larger. This difference                    performed direct comparison approach current <br /> performance likely overfitting results          state art algorithm MNDT [8], using SSL dataset used <br /> classifiers using feature set. In conclusion,                study. This dataset comprised 3,866 SSL <br /> experiment showed SL-independent features                     examples 688,045 non-SSL examples [8]. This number is <br /> effective predicting new SL interactions weakly                 slightly different Wong et al.&#226;&#8364;&#8482;s data set use <br /> connected known SL interaction network. Given             ORF names instead SGD IDs deletion duplicates <br /> small fraction gene pairs experimentally tested         version data set. Figure 4(C) shows ROC <br /> for SL interactions them, given majority             curves result four-fold cross-validation procedure, <br /> the untested pairs expected weakly connected             corresponding MNMC based features (MNMC.all), <br /> the known SL network, SL-independent features conjunc-                MNMC based SL-independent features (MNMC.slif), <br /> tion multi-classifier approach expected lead          MNDT based features (MNMC.all), MNDT based <br /> robust accurate predictions, largely reduce             SL-independent features (MNDT.slif), addition a <br /> burden experiments.                                                       curve corresponding random classifier. Note feature <br />    Another test performance approach based              sets used MNDT [8] different used MNMC. <br /> an independent test set constructed 730 new SL interactions             Not surprisingly, methods better performance all <br /> added SGD interaction database May                        features, including SL-dependent &#226;&#8364;&#8220;independent features, <br /> November 2008. These interactions formed positive test set               used. It seen precision-recall ROC <br /> for experiment, negative test set 5163 non-SL                curves MNMC.all higher MNDT.all most <br /> interactions genes constituting positive test set       range score threshold, reflected the <br /> extracted non-SL interactions original data set.             higher AUC score (0.897 vs 0.862). Although this <br /> The ensemble classifier trained using 9994 positive             difference agreeably high, indicates advantage of <br /> and 120346 ( = 125509-5163) negative examples original                under-sampling multi-classifier prediction technique. <br /> data set. This trained classifier used make predictions          On hand, MNMC.slif (AUC = 0.805) outperforms <br /> for new test set, results evaluated using ROC              MNDT.slif (AUC = 0.598) substantially, shows our <br /> curves. Differences performance different classifiers          approach able make better use SL-independent <br /> (shown Figure 4(E)) similar discussed above. Here,           features SL prediction, performance MNDT largely <br /> we false positive rate 10%, SL-independent features         comes SL-dependent features. For example, FPR of <br /> produce 5% higher true positive rate SL-dependent                 20%, MNMC.slif leads TPR 65%, 28% higher that <br /> features.                                                                    produced MNDT.slif using SL-independent features, and, <br />    The advantage SL-independent features                     FPR 30%, gap TPRs even <br /> clearer corresponding precision-recall curves, shown            larger (31%). Similar observations form the <br /> Figure 4(B) 4(F). Take result unseen data               precision-recall curves experiment (Figure 4(D)). These <br /> example (Figure 4(F)), precision MNMC.slif 10%               results demonstrates advantage approach existing <br /> higher MNMC.all recall 20%                      approaches, arises facts 1) employ an <br /> difference precision larger (30%) recall             extended set features characterize gene pairs, 2) we <br /> 10%. This provides additional evidence currently                employ under-sampling multi-classifier ensemble carry <br /> unscreened gene pairs, SL-independent features provide              training predictions. <br /> accurate predictions lower dependence                       We compared results produced PPI- <br /> currently known SL pairs.                                                    SVM method proposed Paradugu et al [11]. When SL- <br />    Although MNMC.slif outperforms MNMC.all recall               independent features used, MNMC.slif outperforms PPI-SVM <br /> 45% unseen samples, precision MNMC.slif          dataset. For example, FPR 18%, highest TPR <br /> high, shown Figure 4(F). The low performance results          PPI-SVM w/o 2Hop 52.4% MNMC.slif leads TPR <br /> fact discriminative features based data         62.4%. <br /> are available 730 SL pairs. For example, the <br /> Pathway Comembership Common Functions features                       Discovery novel SL interactions transcription <br /> available 3% 30% 730 pairs respectively,           factors <br /> the numbers 5% 40% 9994 SL pairs. Moreover,                Given accuracy SL predictions provided our <br /> most discriminative features based data                  approach, applied study functional redundancy within <br />  <br />  <br />        PLoS Computational Biology | www.ploscompbiol.org                 6                      September 2010 | Volume 6 | Issue 9 | e1000928 <br />                                                                            Predicting Genetic Interactions <br />  <br />  <br />  <br />  <br /> PLoS Computational Biology | www.ploscompbiol.org   7   September 2010 | Volume 6 | Issue 9 | e1000928 <br />                                                                                                                       Predicting Genetic Interactions <br />  <br />  <br />  <br /> Figure 4. Receiver operating characteristic (ROC) curves ensemble based SL-dependent (MNMC.all) &#226;&#8364;&#8220;independent <br /> features (MNMC.slif). (A) ROC curves (AUC MNMC.all = 0.819 MNMC.slif = 0.851) (B) Precision-recall curves classification 337 <br /> least connected SL 199 corresponding non-SL interactions using SL-independent SL-dependent features; (C) ROC curves (AUC of <br /> MNDT.all = 0.862, MNMC.all = 0.897, MNDT.slif = 0.598 MNMC.slif = 0.805) (D) Precision-recall curves classification Wong et al.&#226;&#8364;&#8482;s SSL <br /> dataset [8] using MNMC MNDT based on features (all) SL-independent features (slif); (E) ROC curves (AUC of <br /> MNMC.all = 0.616 MNMC.slif = 0.633) (F) Precision-recall curves classification independent test set constructed SGD <br /> interaction database using MNMC.all MNMC.slif. The corresponding ROC precision-recall curves random classifier applied these <br /> prediction problems shown. <br /> doi:10.1371/journal.pcbi.1000928.g004 <br />  <br /> the yeast transcriptional regulatory network. <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span>Knock-out</span> studies           synthetic interactions YAP TFs, namely <br /> yeast revealed surprising robustness single deletions,             YAP1:YAP2, YAP1:YAP3, YAP1:YAP5 YAP4:YAP6. Previous <br /> particularly transcription factors, rate gene             clustering analyses YAP protein sequences [23] YAP DNA <br /> essentiality 8% compared genome background rate                 binding sequences [26] revealed YAP TFs be <br /> 17% [21]. Expression profiling experiments revealed                grouped related subfamilies: 1) YAP1 YAP2, 2) YAP4 <br /> putative targets change relatively little expression           YAP6, 3) YAP5 YAP7. Here, predict genes <br /> upon deletion corresponding regulators, provides                subfamilies synthetic lethal interactions <br /> further evidence robustness transcriptional network           them. In particular, YAP1 predicted SL <br /> or suggests limitations current understanding         interactions YAP2, YAP3 YAP5. YAP1 plays central role <br /> transcriptional network [22]. We hypothesized predicted             response oxidative stress regulates response to <br /> SL interactions transcription factors (TFs) provide            H2O2-induced stress, cadmium, drug stress, YAP2 <br /> insights genetic relationships underlying redundancy.          responds cadmium stress [27]. Thus, SL interaction <br /> Towards end, generated test set 6,903 TF pairs             YAP1 YAP2 implies loss ability respond to <br /> the currently available 118 TFs yeast (except MATA1) [16],             cadmium stress TFs deleted, consistent the <br /> which 8 known SL interactions 5 known             previous finding double mutant yap1yap2 more <br /> have SL interactions. We used ensemble classifier trained             sensitive cadmium [27]. As example, consider the <br /> our SGD-SL data set make predictions remaining 6,890              predicted SL interaction YAP4 YAP6. Although YAP4 <br /> TF gene pairs using SL-independent feature set. Among              YAP6 regulate osmotic stress, yap4 null mutant <br /> 6,890 pairs, predicted 467 SL interactions based                     shows impaired growth exposed hyperosmolarity [28]. <br /> classification score threshold 0.2, achieved precision            However, double mutant yap4yap6 strain displays further <br /> of 49% recall 14% determined 10-fold cross-                    reduction glycerol metabolism accumulation, is <br /> validation collected SGD-SL dataset. Since threshold           crucial osmo-tolerance [27]. All together, analyses imply <br /> of 0.2, precision MNMC.slif 41% recall 14%             condition-specific SL interactions YAP1 YAP2, and <br /> the 730 new SL interactions (unseen data set), estimate          YAP4 YAP6. Finally, known YAP <br /> precision predicted TF SL network lie 41%                proteins, class basic leucine zipper (bZIP) proteins, <br /> and 49% recall 14%. Fourteen TF pairs                     DNA-binding domains similar true yeast AP-1 factor <br /> predicted synthetic lethal classification scores 0.4.       GCN4 [23,27]. GCN4 MET28 group 14 <br /> The TF SL interactions terms classification             known bZIP proteins [23]. The predicted TF SL interaction <br /> scores GZF3:DAL80, NRG1:AZF1, HAP3:HAP5, SWI5:ACE                       network includes SL interactions YAP1 GCN4, and <br /> and YHP1:YOX1. Figure 5(A) shows network 467                      YAP5 MET28. <br /> predicted 8 known SL interactions 106 transcription                 We surveyed predicted SL interactions HAP TFs, <br /> factors. Table S2 Text S1 lists 475 TF SL interactions            shown network Figure 5(C). Interestingly, HAPs (2, <br /> Table S3 Text S1 shows degree transcription factor <br />                                                                              3, 4 5) form fully connected clique missing link <br /> in network. On average, 106 TFs participates <br />                                                                              HAP2 HAP4, HAP1 does interact any <br /> in 9 SL interactions, slightly observed known SL <br />                                                                              HAP TFs. Not surprisingly, HAP2, HAP3, HAP4, and <br /> interaction network (8 interactions gene). Six TFs (FKH1, <br />                                                                              HAP5 share CCAAT-binding factor (CBF) form protein- <br /> AZF1, GZF3, STP1, REB1 CHA4) involved 25 <br />                                                                              protein protein-DNA interaction complex [29]. The fact that <br /> synthetic lethal interactions each. <br />                                                                              assembly Hap2p, Hap3p, Hap5p requires three <br />    Some predicted SL interactions transcription <br />                                                                              subunits simultaneously suggests condition-specific SL interactions <br /> factors actually supported literature. Recent studies <br />                                                                              TFs. Furthermore, previously identified <br /> have revealed functions YAP family transcription <br />                                                                              interaction Hap4p Hap2p/Hap3p/Hap5p-DNA <br /> factors (YAP1, YAP2 (CAD1), YAP3, YAP4 (CIN5), YAP6-8) a <br />                                                                              complex [29] supported predictions. <br /> response stress induced drug treatments, oxidative stress, <br />                                                                                 In summary, exploration small predicted TF <br /> metal detoxification, DNA damage, others <br />                                                                              SL interaction network leads interesting findings. <br /> [23,24,25,26]. These studies suggested YAP <br /> TFs overlapping distinct functions,                    Thus, predicted TF SL interactions expected useful <br /> relationships understood. In                   studying specific functions TFs for <br /> particular, systematic study genetic                    understanding general mechanisms underlying robustness in <br /> interactions YAP transcription factors date.                    regulatory networks. <br /> Figure 5(B) shows sub-network predicted SL interactions <br /> involving YAP transcription factors. This network comprised           Prediction genetic interactions non- <br /> of 56 links 36 TFs, including 7 YAP TFs (YAP1-7). YAP5                 essential yeast genes <br /> has highest number interactions (14), followed YAP6 (12),            In order expand utility genetic interactions, used <br /> YAP2 (11), YAP1 (9), YAP3 (7) YAP4 (6), YAP7            approach make predictions synthetic lethal interactions <br /> single SL interaction REB1. As shown Figure 5(B),              pairs non-essential genes S. cerevisiae. For this <br />  <br />  <br />        PLoS Computational Biology | www.ploscompbiol.org                 8                     September 2010 | Volume 6 | Issue 9 | e1000928 <br />                                                                                                                       Predicting Genetic Interactions <br />  <br />  <br />  <br />  <br /> Figure 5. A network predicted synthetic lethal interactions yeast transcription factors (TFs). (A) A global TF SL <br /> interaction network including 467 predicted 8 experimentally verified interactions SGD (highlighted red). (B) YAP-TF related synthetic <br /> lethal interactions; C) HAP-TF related synthetic lethal interactions. <br /> doi:10.1371/journal.pcbi.1000928.g005 <br />  <br /> endeavor, adopted list 3885 non-essential genes used           perfectly balanced training set (1:1 ratio number of <br /> the construction SGA arrays [30] derived possible            examples positive negative classes), did observe a <br /> gene pairs set. After excluding 135,503 gene pairs         dependence results sampling ratio. Table 1 lists the <br /> our SGD-SL data set, obtained 7,471,681 gene pairs could <br /> potentially encode genetic interactions. Next, MNMC.slif, trained <br />                                                                              Table 1. Dependence AUC scores ensemble <br /> using SGD-SL data set, employed compute the <br />                                                                              classifiers trained using SSL-dependent SSL-independent <br /> classification scores denoting likelihood 7.5 million <br />                                                                              features sampling ratio used generated training <br /> unseen gene pairs encode genetic interactions. Applying                 set Wong et al&#226;&#8364;&#8482;s SSL data set [8]. <br /> threshold 0.2 scores, results above, <br /> wepredicted 50,210 SL interactions 3477 genes, thus <br /> demonstrating wide gene coverage predictions. The                 Sampling Ratio         SSL-dependent         SSL-independent <br /> prediction scores predicted classes (SL non-SL) a <br />                                                                              1                      0.8972                0.8053 <br /> threshold 0.2 7.5 million pairs available <br /> at http://sage.fhcrc.org/downloads/downloads/predicted_yeast                 1.5                    0.9024                0.8083 <br /> _genetic_interactions.zip. We expect valuable resource             2                      0.8978                0.8120 <br /> will useful computational experimental biologists                 3                      0.8928                0.8045 <br /> aiming understand utilize synthetic lethal interactions            5                      0.8827                0.7831 <br /> yeast. <br />                                                                              7                      0.8745                0.7711 <br />                                                                              10                     0.8627                0.7607 <br /> Effect under-sampling                                                     20                     0.7977                0.6268 <br />   An important characteristic approach under-                  50                     0.6370                0.5542 <br /> sampling non-SL class construct training set. <br /> Although results presented paper generated using            doi:10.1371/journal.pcbi.1000928.t001 <br />  <br />  <br />  <br />        PLoS Computational Biology | www.ploscompbiol.org                9                      September 2010 | Volume 6 | Issue 9 | e1000928 <br />                                                                                                                                   Predicting Genetic Interactions <br />  <br />  <br />  <br />  Table 2. Dependence prediction performance                                  Effect size positive training samples <br />  size positive training sample set.                                                        We tested available SL interactions <br />                                                                                            (positive examples) training affected combined classifier&#226;&#8364;&#8482;s <br />                                                                                            performance. Different portions (10%, 20%, 30%,&#226;&#8364;&#166;, 100%) the <br />  Percentage (%) positive training examples             AUC MNMC.slif                 9994 SL examples SGD-SL data set used 10-fold <br />                                                                                            cross-validation procedure test efficacy resultant <br />  100                                                      0.7298 <br />                                                                                            predictor. Table 2 shows AUCs obtained these <br />  90                                                       0.7332 <br />                                                                                            prediction experiments. It seen table the <br />  80                                                       0.7289                           performance classifier quite robust of <br />  70                                                       0.7328                           training SL examples available, AUC varying narrow <br />  60                                                       0.7261                           range ,0.7&#226;&#8364;&#8220;0.74. <br />  50                                                       0.7211 <br />  40                                                       0.7187                           Comparing PPI-only non-PPI features predicting <br />  30                                                       0.7051                           genetic interactions <br />                                                                                               We investigated performance PPI-only non- <br />  20                                                       0.7125 <br />                                                                                            PPI features predicting genetic interactions. In experiment, <br />  10                                                       0.7019 <br />                                                                                            split 152 SL-independent features sets: (1) set <br />  Different portions (10% 100%) 9994 SL examples data set             PPI-only features derived PPI network <br />  used 10-fold cross-validation procedure test efficacy resultant        (2) set non-PPI features don&#226;&#8364;&#8482;t involve PPI <br />  predictor.                                                                                network. Note PPI-related overlay features, involve <br />  doi:10.1371/journal.pcbi.1000928.t002 <br />                                                                                            PPI network networks, excluded the <br />                                                                                            feature sets. The feature sets used train test <br /> AUC scores ensemble classifiers constructed using SL-                               multi-classifier SGD-SL data set. Figure 6 shows their <br /> dependent SL-independent features previously described                            performance terms ROC precision-recall curves. As <br /> SSL data set [8] varying ratio number                             expected, non-PPI feature set substantially outperforms the <br /> positive negative examples training set. As expected,                       PPI-only one. At FPR 20%, TPR non-PPI features <br /> performance classifiers deteriorates imbalance                          16.6% higher PPI-only features (53.4% versus <br /> between classes increases, performance                             36.8%). At recall 20%, precision non-PPI features is <br /> quite stable ratio 10. Thus, results                        18.9% higher PPI-only features (36.7% versus <br /> not sensitive reasonable skewing sizes                          17.8%) recall 30%, precision non-PPI <br /> classes, determination optimal sampling ratio given                       features 15.7% higher PPI-only features is <br /> data set difficult.                                                                 (29.9% versus 14.2%). <br />  <br />  <br />  <br />  <br /> Figure 6. The performance PPI-only non-PPI features predicting genetic interactions. In experiment, split 152 SL- <br /> independent features sets: (1) set PPI-only features derived PPI network (2) set non-PPI features <br /> which don&#226;&#8364;&#8482;t involve PPI network. The feature sets used train test multi-classifier SGD-SL data set. (A) ROC curves <br /> of ensemble based PPI-only (AUC = 0.605) non-PPI (AUC = 0.731) feature sets. (B) Precision-recall curves ensemble based the <br /> PPI-only non-PPI feature sets. The corresponding ROC precision-recall curves random classifier shown. <br /> doi:10.1371/journal.pcbi.1000928.g006 <br />  <br />  <br />        PLoS Computational Biology | www.ploscompbiol.org                              10                     September 2010 | Volume 6 | Issue 9 | e1000928 <br />                                                                                                                        Predicting Genetic Interactions <br />  <br />  <br />     This difference performance results factors: (1)         classifier. More generally, partition GI interactions into <br /> non-PPI features, GO annotations, microrarray data etc.,             different groups based features characterizing gene pairs and <br /> form richer source information just physical                train classifier group. Such ideas worth <br /> interaction proteins corresponding genes              exploring future work important problem. <br /> measuring strength association them, (2) the <br /> PPI-only features generally smaller coverage. We                 Materials Methods <br /> believe performance PPI-only features be <br /> potentially improved including PPI-based features,             Data sources <br /> as Paladugu et al.&#226;&#8364;&#8482;s study [11] (some            For purpose feature extraction, compiled yeast <br /> included PPI-only features), non-PPI features            microarray datasets [12,13,14,15], protein-protein interaction <br /> still valuable predicting genetic interactions, shown         databases (www.yeastgenome.org Sept 2007), transcription <br /> by [8].                                                               factor binding databases [16], functional annotations defined in <br />     In summary, results presented section demonstrated           Gene Ontology (www.geneontology.org May 2008), <br /> the utility MNMC approach predicting novel SL                     mutant phenotype data (www.yeastgenome.org May 2008), <br /> interactions, particularly using SL-independent features.               phylogenetic profiles proteins [31], KEGG pathway member- <br />                                                                              ships genes [32], BLAST sequence similarity scores yeast <br />                                                                              genes proteins [33] gene network modules clique <br /> Discussion <br />                                                                              communities [16]. <br />    We devised integrative, multi-network approach                        In addition, prepared dataset 9994 SL 125,509 non- <br /> predicting synthetic lethal (SL) interactions, extends             SL interactions SGD interaction database (www. <br /> previously proposed MNDT approach [8]. In approach,                  yeastgenome.org May 2008). The SL interactions were <br /> first defined large number features characterizing              directly extracted database. To maintain consistency, <br /> relationships pairs genes, developed multi-            non-SL interactions identified the <br /> classifier predicting given gene pair belongs           corresponding bait prey proteins determined not <br /> to SL non-SL class. Comprehensive experiments              SL interactions corresponding studies. This dataset, <br /> data sets demonstrated features, conjunction          named SGD-SL, minor variants, used our <br /> advanced classification scheme, led improved performance               cross-validation experiments, training set making <br /> when compared current state art method. In                     novel predictions. <br /> particular, large number features identified were <br /> independent known SL interactions (in contrast MNDT),              Feature extraction <br /> and shown effective making predictions                In order build classifier predicting SL pairs, step <br /> for gene pairs connected known SL                 construct set features various characteristics <br /> interactions.                                                                gene pairs. We used types features study, namely <br />    Application approach known transcription factor            features derived individual data sets features derived by <br /> pairs led TF SL interaction network. Several             overlaying pairs data sets. Details features follow. <br /> predicted synthetic lethal interactions transcription <br /> factors supported literature. It note         Features derived individual data sets <br /> SL data used paper obtained rich media,                Here, used types sources data derive the <br /> most predicted SL interactions expected refer rich          likelihood genes related different <br /> media conditions. However, showed condition-               forms. These relationships captured using various measures, <br /> specific SL interactions predicted approach.              degree co-expression different microarray <br /> This reflect fact features functional              data sets, direct indirect links protein interaction other <br /> annotation, gene expression signatures, sequence similarity              types networks, similarity evolution patterns using mutual <br /> can help identify condition-specific interactions.                      information phylogenetic profiles genes, <br />    Our approach expected effective uncovering new               similarity functional labels assigned genes Gene <br /> genetic interactions millions gene pairs yeast               Ontology, others. We included measures <br /> hundreds millions gene pairs higher organisms like mouse            importance gene pair itself computing betweenness <br /> and human tested interactions. In               corresponding interaction protein interaction and <br /> addition, type predictor utility               Bayesian networks. This computation gave set 62 features, <br /> comprehensive empirical screens carried out, given            details provided Table S1 Text S1. Below, <br /> the effects leading genetic interactions context-        discuss novel features used study <br /> dependent feasible experimentally assess            discriminative the <br /> interactions contexts.                                             nonSL SL classes. <br />    Given difficulty problem predicting genetic <br /> interactions, best classification methods suffer low                Number shared GO biological process functions <br /> precision low coverage. On hand, opens                  A straightforward way measure functional similarity of <br /> the door exploring new methods. The fact features               genes count functions shared. <br /> independent known genetic interaction (GI) network                However, case assignments functional classes the <br /> better predict gene pairs connected GI network             GO ontologies, count biased general classes to <br /> the features including GI dependent independent               genes assigned. Thus, used 138 <br /> features lead better performance pairs connected           populated GO biological process terms Myers et al. have <br /> the GI network, suggests new avenue improve                 suggested useful functional analyses prediction <br /> prediction performance: use connectivity candidate               studies [34]. We use number shared annotations these <br /> gene pair known GI network select appropriate                   functions features data set. However, this <br />  <br />  <br />       PLoS Computational Biology | www.ploscompbiol.org                 11                      September 2010 | Volume 6 | Issue 9 | e1000928 <br />                                                                                                                       Predicting Genetic Interactions <br />  <br />  <br /> computation carried GO                      input features undirected network, genes as <br /> ontologies, list functions available. Thus,            nodes, value feature gene pair weight <br /> we used extensive method calculating functional             edge connecting them. Then value feature <br /> similarity genes, described below.                                   obtained overlaying networks N1 N2 computed <br />                                                                                max&#194;&#189;w(g1 ,c)|w(c,g2 ) , (g1 ,c) [ N1 (c,g2 ) [ N2 or <br /> Functional similarity using semantic similarity GO                          vice versa. A missing value placed edges does not <br /> functional classes                                                             exist networks. An illustration computation is <br />    We computed similarity functions genes            shown Figure 1. <br /> the basis entire hierarchical structures ontologies           We used approach derive overlay features gene pairs <br /> in GO. More specifically, compute similarity               using fourteen input networks maximum coverage <br /> functional classes GO ontologies using Lin&#226;&#8364;&#8482;s semantic            gene pairs SGD-SL data set, computing an <br /> similarity measure [35], defined                                   overlay feature using pair networks (no self-overlays, which <br />                                                                                captured individual features). These input <br />                                                                                networks included correlation topological overlap measure- <br />                                     2|&#194;&#189;log pms (c1 ,c2 )                       ments microarray data sets, number common <br />                 linsim(c1 ,c2 )~ <br />                                    log p(c1 )z log p(c2 )                      mutant phenotypes, direct links protein interaction network, <br />                                                                                semantic similarity-based relationships GO <br /> Here, c1 c2 classes (or nodes) similarity            ontologies, sequence similarity scores using BLAST e-value <br /> is calculated, p(c) denotes probability                   gene protein sequences, co-causality <br /> protein annotated class c, estimated                    measure Bayesian network. These input networks were <br /> the available set GO annotations organism. Also,                     chosen covered substantial fraction the <br /> pms (c1 ,c2 )~ min p(c), S(c1 ,c2 ) set common                 non-essential yeast genes reasonably high individual <br />              c[S(c1 ,c2 ) <br /> ancestors c1 c2 . Thus, pms (c1 ,c2 ) denotes probability        discriminative power. Also, networks be <br /> occurrence minimum subsumer c1 c2 . Intuitively,                 sparsified using pre-specified thresholds (list networks the <br /> Lin&#226;&#8364;&#8482;s measure measures semantic similarity c1 c2                 corresponding thresholds provided Table S2 Text S1) in <br />                                                                                order make computation feasible. In particular, the <br /> terms contents minimum subsumer node the <br />                                                                                edges (gene pairs) carrying weights higher positive <br /> ontology, used extensively quantifying relation- <br />                                                                                threshold lower negative threshold (if any) were <br /> ships functional classes GO ontologies [36,37,38]. <br />                                                                                retained sparsified network. Also, used flexible <br />    Now, given set annotations genes entire <br />                                                                                formulation overlay features, maximum value <br /> ontology, groups A B, functional similarity <br />                                                                                product scores constituting overlay assigned <br /> between genes computed using Tao et al.&#226;&#8364;&#8482;s <br />                                                                                value gene pair, compared just trying any <br /> approach [37] follows. For annotation group A, the <br />                                                                                path, Wong et al. [8]. Thus, study, were <br /> most similar annotation group B, vice versa. Next, <br />                                                                                able expand set features overlaid, use more <br /> the set P mutually similar pairs annotations are <br />                                                                                flexible formulation, leading good results. Note overlay <br /> found groups A B, functional similarity the <br />                                                                                feature constructed using semsim_mf semsim_cc features <br /> two genes computed as <br />                                                                                used data sets prohibitive time <br />                                              P                                 requirements computing feature thousands gene <br />                                      2|                 linsim(ai ,bi )        pairs. Also, generated 14 additional overlay features, using the <br />                                           (ai ,bi )[P                          known SL interaction network input networks, and <br />          FunctionalSim(A,B)~ <br />                                                DADzDBD                         mentioned fourteen networks, 15th by <br />                                                                                overlaying SL network itself. Note additional <br />                                                                                features computed fair manner, positive SL <br />    Such similarity measure takes specificity relative                examples training set used. <br /> positioning annotations account robustly                  We differentiate sets features rest the <br /> simple count common functional annotations,                      study, depending set features include features <br /> complexity annotations GO ontologies.                 computation depends known SL interactions not <br /> Tao et al. [37] demonstrated measure computes                    (referred SL-dependent SL-independent features <br /> similarity set annotations genes accurately              respectively). In total, feature sets included 152 SL- <br /> than measures, all-pair average similarity used           independent features, 15 SL-dependent features. Descriptions <br /> other studies [36].                                                            features Table S1 Text S1. Note that <br />    Using measure, created features functional            combination 167 SL-dependent SL-independent <br /> similarity gene pair, corresponding           features referred &#226;&#8364;&#732;&#226;&#8364;&#732;all&#226;&#8364;&#8482;&#226;&#8364;&#8482; features discussion the <br /> GO ontologies, Biological Process, Molecular Function                   results. <br /> and Cellular Component. We used features for <br /> computing additional features using network overlays, described             Under-sampling <br /> below.                                                                            As mentioned before, data set significantly skewed, with <br />                                                                                small fraction examples belonging positive (SL) <br /> Features derived overlaying pairs data sets                              class, rest negative (non-SL) class. It well-known <br /> (Network Overlay Features)                                                     standard classification algorithms ideally designed for <br />    Previous work suggested interesting set 2-hop features            balanced classes [39,40]. Thus, integral overall <br /> gene pairs [8]. There, individual features                  methodology under-sampling majority (negative) class, <br /> &#226;&#8364;&#732;&#226;&#8364;&#732;overlaid&#226;&#8364;&#8482;&#226;&#8364;&#8482; individual features generate transitive           randomly under-sample negative examples the <br /> feature. In formulation overlay features, treat            complete set number equal positive <br />  <br />  <br />        PLoS Computational Biology | www.ploscompbiol.org                  12                     September 2010 | Volume 6 | Issue 9 | e1000928 <br />                                                                                                                                                  Predicting Genetic Interactions <br />  <br />  <br /> ones [19]. All classifiers trained balanced                         obtain balanced training set. The ensemble classifier <br /> set. Note under-sampling carried test set,                        trained revised training set, predictions <br /> thus ensuring evaluation results unbiased                                    test examples remaining fold using the <br /> comparable methods.                                                                score discussed above. Repeating procedure n times each <br />                                                                                               fold treated test set produces score example in <br /> Training classification                                                                   data set. This collection evaluated using ROC curve <br />    Once features gene pairs consideration                                corresponding AUC score. In addition, conducted <br /> been computed, adopted multi-classifier predicting                            experiments independent test constructed SGD <br /> whether given gene pair synthetic lethal not. A balanced                              interaction database (www.yeastgenome.org), data set is <br /> combination positive negative training sets (described                             treated training set classifiers trained. In a <br /> above) used train non-parametric multi-classifier                             similar setting, used dataset training set make <br /> that enabled simultaneous use multiple classification                                  predictions SL interactions 118 transcription factors <br /> procedures, SVM, neural network, RIPPER (rule-based                                    pairs non-essential genes yeast. <br /> classifier), random forest, k-nearest neighbor decision tree. <br /> The combination strategy based noisy-AND function,                                 Supporting Information <br /> denoted following formula: <br />                                                                                               Text S1 Supplementary figures tables. <br />                                N              N <br />                                                                                               Found at: doi:10.1371/journal.pcbi.1000928.s001 (0.22 MB PDF) <br />                      p(x)~ P pi (x){ P (1{pi (x)),                               &#195;&#176;1&#195;&#382; <br />                               i~1            i~1 <br />                                                                                               Acknowledgments <br /> where x given gene pair pi(x) represents probability                             We thank Dr. Sharyl Wong Prof. Frederick Roth providing the <br /> that x predicted SL classifier i. Thus, score simply                            details SSL data set used study. Thanks to <br /> computes difference products probabilities                            Weka team promptly answering questions use <br /> the example belong SL non-SL classes                              Weka efficiently tasks. Finally, deepest gratitude to <br /> classifiers, higher score, likely test                         Prof. Vipin Chaudhary Center Computational Research the <br /> example denotes SL interaction constituent genes.                              State University New York Buffalo providing their <br /> The probabilities pi(x) obtained individual classifier                           computing resources run large-scale experiments. <br /> <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> Weka machine learning suite [41] using entire <br /> classification methodology implemented.                                                   Author Contributions <br />    This classification methodology used n-fold cross-                            Conceived designed experiments: GP BZ ANC CLM JZ VK EES. <br /> validation framework SGD-SL Wong et al.&#226;&#8364;&#8482;s datasets.                                Performed experiments: GP. Analyzed data: GP BZ JZ. <br /> n21 randomly constructed n folds treated                                    Contributed reagents/materials/analysis tools: ANC. Wrote paper: <br /> training set, under-sampling procedure executed                             GP BZ CLM EES. <br />  <br /> References <br />  1. Boone C, Bussey H, Andrews BJ (2007) Exploring genetic interactions                         Saccharomyces cerevisiae microarray hybridization. Mol Biol Cell 9: <br />     networks yeast. Nat Rev Genet 8: 437&#226;&#8364;&#8220;449.                                                  3273&#226;&#8364;&#8220;3297. <br />  2. Tong AH, Lesage G, Bader GD, Ding H, Xu H, et al. (2004) Global mapping                16.   Zhu J, Zhang B, Smith EN, Drees B, Brem RB, et al. (2008) Integrating large- <br />     yeast genetic interaction network. Science 303: 808&#226;&#8364;&#8220;813.                                    scale functional genomic data dissect complexity yeast regulatory <br />  3. Collins SR, Miller KM, Maas NL, Roguev A, Fillingham J, et al. (2007)                           networks. Nat Genet 40: 854&#226;&#8364;&#8220;861. <br />     Functional dissection protein complexes involved yeast chromosome                   17.   Tong AH, Evangel <br /> </body></html>