<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>EUKARYOTIC CELL, Nov. 2008, p. 1880&#226;&#8364;&#8220;1887                                                                                        Vol. 7, No. 11 <br /> 1535-9778/08/$08.00&#226;&#171;&#185;0 doi:10.1128/EC.00222-08 <br /> Copyright &#194;&#169; 2008, American Society Microbiology. All Rights Reserved. <br />  <br />  <br />  <br />         The Synthetic Genetic Network PKC1 Identifies Novel <br />          Modulators Components Protein Kinase C Signaling in <br />                          Saccharomyces cerevisiae&#228;&#338;&#164; <br />                                    Sue A. Krause,1 Hong Xu,2&#226;&#8364;&#160; Joseph V. Gray1* <br />       Molecular Genetics Integrative &amp; Systems Biology, Faculty Biomedical Life Sciences, University Glasgow, <br />            Anderson College Complex, 56 Dumbarton Road, Glasgow G11 6NU, United Kingdom,1 Banting and <br />                    Best Department Medical Research, University Toronto, 112 College Street, Toronto, <br />                                                  Ontario M5G 1L6, Canada2 <br />                                              Received 7 July 2008/Accepted 9 September 2008 <br />  <br />              Budding yeast Saccharomyces cerevisiae contains protein kinase C (PKC) isozyme encoded the <br />           essential gene PKC1. Pkc1 activated small GTPase Rho1 plays central role cell wall <br />           integrity (CWI) signaling pathway. This pathway acts primarily remodel cell surface the <br />           normal life cycle various environmental stresses. The pathway heavily branched, multiple <br />           nonessential branches feeding central essential Rho1-Pkc1 module. In attempt identify <br />           novel components modifiers CWI signaling, determined synthetic lethal genetic network around <br />           PKC1 using dominant-negative synthetic genetic array analysis. The resulting mutants hypersensitive <span id='am-8' about='protege:TO' typeof='owl:Thing'>to</span> <br />           lowered Pkc1 activity. The corresponding 21 nonessential genes closely related CWI function: 14 behave <br />           chemical-genetic epistasis test acting pathway, 6 genes encode known components. <br />           Twelve 21 <span id='am-9' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-10' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span>null mutants</span> display elevated CWI reporter activity, consistent idea pathway <br />           activated compensates loss gene products. Four 21 mutants display low CWI reporter <br />           activity, consistent idea pathway compromised mutants. One group <br />           mutants lacks Ack1(Ydl203c), uncharacterized SEL-1 domain-containing protein modulates <br />           pathway activity. Epistasis analysis places Ack1 upstream Pkc1 CWI pathway dependent the <br />           upstream Rho1 GTP exchange factors Rom2 Tus1. Overall, synthetic genetic network PKC1 <br />           directly efficiently identifies known novel components PKC signaling yeast. <br />  <br />  <br />    The cell wall integrity (CWI) pathway acts primarily main-             perturbations, detected effect cell surface <br /> tain cell integrity life cycle (reviewed refer-            sensed multiple members WSC family cell <br /> ence 22). The pathway responds variety external                 surface sensors (11, 15, 38). These putative mechano-sensors <br /> and internal stresses, heat shock, hypo-osmotic shock,               bind activate Rho guanine nucleotide exchange <br /> cytoskeletal defects, cell wall defects, secretion blocks. The           factors (GEFs) Rom2 Tus1 activate Rho1 (28, 32). <br /> pathway acts regulate activity expression key cell wall            However, upstream signaling complex, mul- <br /> synthesis enzymes control processes actin po-                 tiple additional branches feeding information path- <br /> larization polarized secretion ensure appropriate re-                 way. Some signals transduced Rho1 regulators but <br /> modeling cell surface.                                                independently WSC sensors, e.g., TOR signaling mod- <br />    The small GTPase Rho1 central CWI pathway.                      ulates Rho1 GEF Rom2 (33), CDK1/Cdc28 directly phos- <br /> Rho1-GTP directly binds activates multiple effectors,                 phorylates Rho1 GEF Tus1 (19), mRNA binding <br /> including Fks1/2 glucan synthases, Bni1/Bnr1 formins,                protein Mpt5 inhibits expression Rho1 GTPase-activat- <br /> and exocyst component Sec3 (22). One key direct                      ing protein Lrg1 (34). Finally, actin cytoskeletal defects appear <br /> target Pkc1, Saccharomyces cerevisiae homolog                 sensed unknown mechanism independent both <br /> mammalian protein kinase C essential protein required                 WSC sensors known Rho1 regulators (12). <br /> for maintenance cell integrity (21).                                     Here, focus Pkc1 hub CWI pathway. Pkc1 <br />    The CWI pathway stimulated variety intrinsic               controls multiple downstream branches, best understood of <br /> extrinsic signals transduced variety upstream branches              mitogen-activated protein (MAP) kinase cascade <br /> (22). Many signals, e.g., heat shock cell wall                  terminating Slt2 MAP kinase. This cascade essen- <br />                                                                              tial normal conditions required cells survive <br />                                                                              stresses high temperature. Slt2 known directly <br />    * Corresponding author. Mailing address: Molecular Genetics and <br /> Integrative &amp; Systems Biology, Faculty Biomedical Life Sci- <br />                                                                              activate, others, Rlm1 Swi4 transcription fac- <br /> ences, University Glasgow, Anderson College Complex, 54-56                tors, important expression genes involved in <br /> Dumbarton Road, Glasgow G11 6NU, United Kingdom. Phone: (141)                cell surface construction (1, 41). <br /> 330-6135. Fax: (141) 330-4878. E-mail: J.Gray@bio.gla.ac.uk.                    We set identify new regulators components the <br />    &#226;&#8364;&#160; Present address: Molecular Oncology Breast Cancer Program, <br /> BC Cancer Research Centre, 675 West 10th Avenue, Vancouver, Brit- <br />                                                                              CWI/Pkc1 pathway. We chose genetic strategy. Synthetic <br /> ish Columbia V5Z 1L3, Canada.                                                lethal genetic interactions, type <span id='am-87' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-88' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span>phenotype</span> enhancement, <br />    &#228;&#338;&#164; <br />      Published ahead print 19 September 2008.                          occur loss-of-function mutations different genes <br />  <br />                                                                       1880 <br />  VOL. 7, 2008                                                                                            GENETIC NETWORK AROUND PKC1                               1881 <br />  <br />  <br /> cause lethality present together. Such interac-                       2&#226;?&#174;PKC1 (DL289) plasmids (40); pROM2-HA pSLT2-HA (33); pROM2-GFP <br /> tions systematically uncovered yeast                          pTUS1-GFP (44); pMKK1 (14). <br />                                                                                        pACK1 pTUS1 generated inserting respective genes the <br /> given &#226;&#8364;&#339;query&#226;&#8364;? <span id='am-165' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-166' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-242' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span>mutation</span> using synthetic genetic array (SGA)                       vector, pGREG535 (Euroscarf), homologous recombination. This vector con- <br /> analysis (36) diploid-based synthetic lethality analysis                    tains GAL1,10 promoter hemagglutinin tag. The vector digested with <br /> microarrays (27). Although synthetic genetic interactions tend                      SalI remove HIS&#226;&#171;&#185; marker. PCR amplified genes (using Ack1F Ack1R <br /> to occur genes parallel &#226;&#8364;&#339;buffering&#226;&#8364;? pathways                          Tus1F Tus1R) cotransformed SalI-digested pGREG535 into <br />                                                                                     BY4741 yeast cells. Transformants selected SD-Leu plates. Plasmid <br /> processes, mutations essential genes thought prefer-                      DNA independent yeast colonies transformed XL1-Blue <br /> entially sensitize organism second mutation affects                    Escherichia coli cells (Stratagene). Plasmid DNA purified, presence <br /> the parallel process (26).                                       orientation insert confirmed restriction digestion. The function <br />    Here, determined set mutations nonessential                         plasmids confirmed vivo complementation relevant <br />                                                                                     null-mutant phenotypes. <br /> genes (null mutations) confer hypersensitivity lowered <br />                                                                                        DN-SGA analysis. DN-SGA analysis performed previously described <br /> Pkc1 activity novel modification SGA analysis, domi-                       SGA analysis (36, 37), following modifications. The chromo- <br /> nant-negative SGA (DN-SGA) analysis. We                           somal copy PKC1 left unaltered query strain (Y4115). Instead, the <br /> the corresponding 21 genes closely functionally linked                      wild-type &#226;&#8364;&#339;query&#226;&#8364;? strain transformed URA3-marked plasmid [pGAL- <br />                                                                                     PKC1(dead)] expressing dominant-negative allele PKC1 encoding catalyt- <br /> to CWI pathway, including uncharacterized YDL203c/ <br />                                                                                     ically inactive full-<span id='am-6' about='xsp:length' typeof='owl:Thing'>length</span> protein [Pkc1(dead)] expressed control the <br /> ACK1 gene, encodes new upstream modulator basal                          inducible GAL1,10 promoter. The transformed query strain crossed the <br /> CWI activity.                                                                       arrayed haploid deletion collection (&#226;&#172;&#402;4,700 viable null mutants) glucose- <br />                                                                                     containing medium lacking uracil. Subsequent sporulation selection for <br />                                                                                     transformed haploid progeny yielded arrayed collection haploid deletion <br />                         MATERIALS AND METHODS                                       mutants, containing pGAL-PKC1(dead). This arrayed collection sub- <br />                                                                                     sequently pinned SGalactose-URA medium, transformants prolifer- <br />    Media, chemicals, transformation. The rich yeast medium (YPD) con-           ated poorly, all, 1 day inducing medium identified. The <br /> sisted 1% Bacto yeast extract (Difco), 2% peptone (Difco), 2% glucose.       analysis repeated twice, mutants grew poorly, all, upon <br /> Rich YPGalactose medium described YPD, 2% galactose          induction Pkc1(dead) independent runs were <br /> the sole carbon source. Synthetic derived medium (SD, SGal, Suc, SRaffinose)     considered positives. <br /> contained 0.67% yeast nitrogen <span id='am-321' about='xsp:base' typeof='owl:Thing'>base</span> amino acids (Difco); 2% glucose,           Determination staurosporine sensitivity. The sensitivity strain the <br /> 2% galactose, 0.005% sucrose, 2% raffinose, respectively; appropriate        Pkc1 inhibitor staurosporine determined follows. Cells cultured <br /> <span id='am-1' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> supplements drug additions appropriate. Solid media      overnight YPD (untransformed cells) room temperature supplemented <br /> described above, addition 2% Bacto agar (Difco). Chemicals         sorbitol (1 M) appropriate. Cultures diluted fresh medium <br /> <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> Sigma-Aldrich unless stated otherwise. Restriction enzymes New       incubated agitation 6 h room temperature. Culture density was <br /> England Biolabs. All transformations performed using conventional          adjusted optical density 600 nm (OD600) 0.05. Aliquots (100 &#226;?&#174;l) were <br /> lithium acetate method (10).                                                        placed wells sterile 96-well microtiter plate, staurosporine (or <br />    Oligonucleotides. The following primers used (Sigma-Aldrich): Ack1F         vehicle control) added. The plates incubated room <br /> (5&#226;&#172;&#732;-GAATTCGATATCAAGCTTATCGATACCGTCGACAATGGTTAATC                                    temperature overnight gentle agitation. The final OD600 culture was <br /> AGGGCCAACC-3&#226;&#172;&#732;), Ack1R (5&#226;&#172;&#732;-GCGTAGCATAACTAATTACATGACTCG                               determined using plate reader. The analysis performed times. <br /> AGGTCGACTTAATTTGTCAAACTTCGAAGAGC-3&#226;&#172;&#732;), Tus1F (5&#226;&#172;&#732;-GAATT                                  Determination sensitivity tunicamycin, calcofluor white, chlorpromazine, <br /> CGATATCAAGCTTATCGATACCGACAATGTATCGATATAATTAGAA                                      benomyl. Mutants congenic wild-type cells grown YPD or <br /> GTTCCCCT-3&#226;&#172;&#732;), Tus1R (5&#226;&#172;&#732;-GCGTAGCATAACTAATTACATGACTCG                             synthetic medium mid-logarithmic phase. The cultures diluted an <br /> AGGTCGACTTATTGATGATCTGCCATTGCT-3&#226;&#172;&#732;). The identities                          OD600 0.02, serial dilutions fresh medium made. Samples (5 &#226;?&#174;l) <br /> haploid deletion strains confirmed PCR amplification replacement     dilution spotted fresh plates tunicamycin (1.5 <br /> locus, using KanB primer appropriate gene-specific &#226;&#8364;&#339;Primer A&#226;&#8364;?           &#226;?&#174;g/ml), calcofluor white (150 &#226;?&#174;g/ml), chlorpromazine (200 &#226;?&#174;g/ml) (Calbiochem), <br /> flanking primer according Yeast <span id='am-243' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-244' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-250' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-261' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-267' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-271' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-277' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-289' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-290' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-292' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-296' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-300' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-318' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-319' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-320' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span>Knockout</span> Consortium (http://www              benomyl (20 &#226;?&#174;g/ml). The plates incubated 2 3 days 25&#194;&#176;C. <br /> -sequence.stanford.edu/group/yeast_deletion_project/confirmation.html).                Quantifying Rlm1-dependent reporter activity. Cells transformed a <br />    Strains, genetic crosses, plasmids. Most S. cerevisiae strains used   LacZ reporter construct, pRLM1-LacZ (17). Transformed cells cultured to <br /> are derived BY4741 BY4742 yeast strain S288c strain back-        mid-logarithmic phase SD-URA room temperature, &#226;?&#164;-galactosidase <br /> ground. For DN-SGA screen, query strain MAT&#226;?&#163; Mfa1&#226;&#338;&#172;::MFA1pr-             activity quantified Miller Units standard protocol according to <br /> HIS3 can1&#226;&#338;&#172; his3&#226;&#338;&#172; leu2&#226;&#338;&#172; ura3&#226;&#338;&#172; MET15&#226;&#171;&#185; lys2&#226;&#338;&#172; transformed pGAL1-                   Stewart et al. (34). During estimation stress induction CWI signaling, <br /> PKC1(dead). We obtained wild-type BY4741 strain, genotype              transformed cells treated calcofluor white (150 &#226;?&#174;g/ml), chlorproma- <br /> MATa his3&#226;&#338;&#172;1 leu2&#226;&#338;&#172; met15&#226;&#338;&#172; ura3&#226;&#338;&#172; (Euroscarf), derivative single        zine (200 &#226;?&#174;g/ml), latruculin B (40 &#226;?&#174;g/ml) 3 h 25&#194;&#176;C prior the <br /> null mutations api2&#226;&#338;&#172;::KanMX4, atg15&#226;&#338;&#172;::KanMX4, bck1&#226;&#338;&#172;::KanMX4, bni1&#226;&#338;&#172;::                &#226;?&#164;-galactosidase assay. All assays performed triplicate separate <br /> KanMX4, chl1&#226;&#338;&#172;::KanMX4, cik1&#226;&#338;&#172;::KanMX4, csf1&#226;&#338;&#172;::KanMX4, ctf19&#226;&#338;&#172;::KanMX4,                days. The relative reporter activity mutant calculated relative the <br /> egd1&#226;&#338;&#172;::KanMX4, jnm1&#226;&#338;&#172;::KanMX4, las21&#226;&#338;&#172;::KanMX4, mid1&#226;&#338;&#172;::KanMX4, mig1&#226;&#338;&#172;::                level untreated congenic wild-type cells assayed day. <br /> KanMX4, opi3&#226;&#338;&#172;::KanMX4, ost3&#226;&#338;&#172;::KanMX4, per1&#226;&#338;&#172;::KanMX4, rom2&#226;&#338;&#172;::KanMX4, <br /> sec28&#226;&#338;&#172;::KanMX4, spa2&#226;&#338;&#172;::KanMX4, swi4&#226;&#338;&#172;::KanMX4, tus1&#226;&#338;&#172;::KanMX4, ump1&#226;&#338;&#172;:: <br /> KanMX4, ydl203c&#226;&#338;&#172;::KanMX4. We obtained wild-type strain                                                   RESULTS <br /> BY4742, genotype MAT&#226;?&#163; his3&#226;&#338;&#172;1 leu2&#226;&#338;&#172; lys2&#226;&#338;&#172; ura3&#226;&#338;&#172;, homozy- <br /> gous wsc1&#226;&#338;&#172;/&#226;&#338;&#172; diploid mutant MATa/MAT&#226;?&#163; his3&#226;&#338;&#172;1/his3&#226;&#338;&#172;1 leu2&#226;&#338;&#172;/leu2&#226;&#338;&#172; met15&#226;&#338;&#172;/                We previously shown allele PKC1 encoding a <br /> MET15 LYS2/lys2&#226;&#338;&#172; ura3&#226;&#338;&#172;/ura3&#226;&#338;&#172; wsc1&#226;&#338;&#172;::KanMX4/wsc1&#226;&#338;&#172;::KanMX4                catalytically inactive full-length protein behaves dominant- <br /> strain background (Euroscarf). A deletion, marked NatMX, TUS1           negative allele (40), i.e., expression allele trans lowers <br /> gene generated query strain described Tong et al. (36). The <br /> temperature-sensitive mutant PKC1, stt1-1, W303 strain back- <br />                                                                                     Pkc1 activity presence wild-type chromosomal <br /> ground (MATa ura3&#226;&#338;&#172; leu2&#226;&#338;&#172; trp1&#226;&#338;&#172; his3&#226;&#338;&#172; ade2&#226;&#338;&#172; can1-100) (43). The homozygous           copy gene (11). We exploited allele identify the <br /> slt2&#226;&#338;&#172;/&#226;&#338;&#172; strain EG123 strain background (MATa/MAT&#226;?&#163; trp1&#226;&#338;&#172;/trp1&#226;&#338;&#172;            synthetic genetic network PKC1. <br /> leu2&#226;&#338;&#172;/leu2&#226;&#338;&#172; ura3&#226;&#338;&#172;/ura3&#226;&#338;&#172; his4&#226;&#338;&#172;/his4&#226;&#338;&#172; can1&#226;&#338;&#172;/can1&#226;&#338;&#172;) (11).                                 DN-SGA analysis. We transformed wild-type query <br />    All genetic backcrosses congenic wild-type cells opposite    strain marked plasmid expressing catalytically inac- <br /> mating type. Haploid rom2&#226;&#338;&#172; tus1&#226;&#338;&#172; cells generated standard genetic <br /> manipulations cross tus1::NatMX4 rom2::KanMX4 haploid <br />                                                                                     tive allele PKC1 [pkc1(dead)] control ga- <br /> strains.                                                                            lactose-inducible GAL1,10 promoter (40). The transformed <br />    The plasmids used study include pGAL-PKC1, pGAL-PKC1(dead),          query strain crossed arrayed haploid deletion <br />  1882      KRAUSE ET AL.                                                                                                             EUKARYOT. CELL <br />  <br />  <br />                                                                            sensitivity cosegregated marked deletion 10 <br />                                                                            tetrads 21 28 cases genetic backcrosses, consistent <br />                                                                            idea mutations cause sensitivity lowered <br />                                                                            Pkc1 activity. The identity knockout strain was <br />                                                                            successfully confirmed PCR. These 21 mutants constitute <br />                                                                            PKC1 interaction network (Table 1). <br />                                                                               Chemical sensitivity links interacting genes CWI <br />                                                                            pathway. Pkc1-Slt2 activity stimulated multiple different <br />                                                                            stresses cell. Many stresses caused by <br />                                                                            chemical treatment: chlorpromazine deforms plasma mem- <br />                                                                            brane (5), calcofluor white disrupts chitin structure cell <br />                                                                            wall (29), tunicamycin indirectly causes secretion block <br />    FIG. 1. Chemical-genetic analysis indicates two-thirds      interfering protein glycosylation (20). Mutants defec- <br /> PKC1 genetic interaction network comprises &#226;&#8364;&#339;within-pathway&#226;&#8364;? inter-         tive Pkc1 pathway closely allied processes tend to <br /> actions. (A) Example sorbitol suppression hypersensitivity    hypersensitive compounds. We <br /> a null mutant staurosporine. Wild-type api2&#226;&#338;&#172; cells grown <br />                                                                            21 mutants PKC1 network also <br /> in rich medium diluted wells 96-well plate containing <br /> fresh medium supplemented staurosporine (range, 0 &#226;?&#174;g/ml 50         sensitive chemicals, 12 (57%) sen- <br /> &#226;?&#174;g/ml). The cells incubated presence (&#226;&#171;&#185;)      sitive more, 6 (29%) sensitive three <br /> absence (&#226;&#171;&#186;) sorbitol (1 M). (B) Scheme illustrating functional      (Table 2). We conclude mutants PKC1- <br /> classification PKC1-interacting genes. Fourteen genes        interacting network display phenotypes associated <br /> behave encoding proteins act Pkc1 pathway (staurospor- <br /> ine sensitivity suppressed osmotic stabilization),      CWI dysfunction. <br /> encode known CWI pathway components (see text). The remaining                 The PKC1 network rich known components. We expect <br /> seven genes behave encoding proteins function, part,   PKC1-interacting genes encode components of <br /> parallel Pkc1 (staurosporine sensitivity suppressed osmotic   branched CWI pathway (26). Indeed, 6 the <br /> stabilization). <br />                                                                            21 interacting genes (29%) encode known components of <br />                                                                            Pkc1 pathway (Table 1) (22): Bck1, direct target the <br />                                                                            kinase; Swi4 Mig1 downstream transcription factors; <br /> collection, resulting arrayed, transformed haploid mutant               Bni1 formin Mid1 calcium channel, act <br /> progeny generated standard SGA methodology (36). The <br /> &#226;&#172;&#402;5,000 arrayed transformants transferred me- <br /> dium containing galactose sole carbon source (inducing <br />                                                                                      TABLE 1. Confirmed genes members the <br /> condition), growth spot noted. The <br />                                                                                                      PKC1 network <br /> sixty-six transformants grew poorly galac- <br />                                                                              Genea                        Protein function processb <br /> tose-containing medium three independent <br /> runs taken constitute set primary mutants iden-             API2 ............Protein unknown function. <br /> tified analysis.                                                    ATG15 ........Lipase required intravacuolar lysis of <br />                                                                                               autophagosomes. <br />    Mutants eliminated analysis of <br />                                                                            BCK1 ...........MAP kinase kinase downstream Pkc1. <br /> the following applied: (i) untransformed mutants                  BNI1............Formin nucleates formation linear actin <br /> unable proliferate galactose sole carbon source,                                 filaments. <br /> thereby eliminating gal mutants; (ii) sensitivity expression         CHL1 ..........Protein required establishing sister chromatid <br /> Pkc1(dead) reproduced direct transformation;                                  pairing S phase. <br />                                                                            CIK1............Kinesin-associated protein required karyogamy and <br /> (iii) mutants sensitive expression wild-type Pkc1                                  mitotic spindle orientation. <br /> driven equivalent pGAL-PKC1(WT) construct,                 CSF1 ...........Protein required low-temp fermentation. <br /> eliminating sensitive overexpression protein               CTF19 .........Outer kinetochore protein required mitotic <br /> rather lowered activity. In all, 29 mutants survived                              chromosome segregation. <br />                                                                            EGD1..........Protein involved novo cotranslational protein <br /> triage behaved reproducibly sensitive induc- <br />                                                                                               folding. <br /> tion Pkc1(dead) expression.                                             JNM1...........Part dynactin complex required proper nuclear <br />    Confirmation genetic interactions: staurosporine sensi-                                 migration spindle partitioning. <br /> tivity. We wished independently confirm genetic inter-              LAS21 .........Protein involved synthesis GPI anchors. <br /> actions PKC1. Staurosporine broad-spectrum protein               MID1...........Calcium channel. <br />                                                                            MIG1...........Transcription factor involved glucose repression. <br /> kinase inhibitor preferentially inhibits isozymes protein          OPI3............Phospholipid methyltransferase. <br /> kinase C (35). Pkc1 appears limiting target stau-             OST3 ...........Subunit oligosaccharyltransferase ER lumen. <br /> rosporine yeast (40, 43). Indeed, sensitivity wild-type          PER1 ...........Protein required GPI-phospholipase A2 activity. <br /> cells staurosporine, like inviability pkc1&#226;&#338;&#172; mutants,          SEC28 .........Protein required ER-Golgi retrograde transport. <br />                                                                            SPA2............Component polarisome acts scaffold Mkk1 <br /> suppressed osmotic stabilization (Fig. 1A data not <br />                                                                                               Slt2. <br /> shown). Furthermore, pretreatment staurosporine abol-                 SWI4............Transcription factor involved late G1 gene expression. <br /> ishes activation CWI signaling challenge stress               UMP1..........Chaperone required maturation 20S proteasome. <br /> (S. A. Krause J. V. Gray, unpublished observation). We                 YDL203c ....Protein unknown function. <br /> found 28 29 robust mutants identified DN-SGA                  a <br />                                                                                  Genes encoding known components CWI pathway bold. <br /> analysis hypersensitive staurosporine. This hyper-               b <br />                                                                                  GPI, glycosylphosphatidylinositol; ER, endoplasmic reticulum. <br />  VOL. 7, 2008                                                                                         GENETIC NETWORK AROUND PKC1                        1883 <br />  <br />  <br />    TABLE 2. Chemical sensitivities null mutants PKC1                        The majority interacting genes encode inhibitors CWI <br />                     interaction network                                            signaling. What functional relationships link interacting <br />                                      Level sensitivitya to:                     gene products Pkc1 function? Some gene products may <br /> Mutant                                                                             function inhibitors CWI signaling. Mutants lacking such <br />                                           Calcofluor <br />                    Tunicamycin                                   Chlorpromazine    proteins defective processes CWI <br />                                             white <br />                                                                                    pathway responds, e.g., cell wall construction, result- <br /> atg15                   &#226;&#171;&#185;                     &#226;&#171;&#185;                        &#226;&#171;&#185; <br /> chl1                    &#226;&#171;&#185;                     &#226;&#171;&#185;                        &#226;&#171;&#185;           ing elevation pathway activity acting ameliorate con- <br /> cik1                    &#226;&#171;&#185;                     &#226;&#171;&#185;                        &#226;&#171;&#185;           sequences mutations. <br /> csf1                    &#226;&#171;&#185;                     &#226;&#171;&#185;                        &#226;&#171;&#185;              We estimated CWI pathway activity 21 null mutants <br /> ctf19                   &#226;&#171;&#185;                     &#226;&#171;&#185;                        &#226;&#171;&#185;           using established transcriptional reporter ex- <br /> egd1                    &#226;&#171;&#185;                     &#226;&#171;&#185;                        &#226;&#171;&#185; <br /> bck1                    &#226;&#171;&#185;                     &#226;&#171;&#185;                        &#226;&#171;&#186; <br />                                                                                    pression bacterial LacZ gene dependent Rlm1 <br /> jnm1                    &#226;&#171;&#185;                     &#226;&#171;&#185;                        &#226;&#171;&#186;           transcription factor (17). This reporter faithfully reflects the <br /> las21                   &#226;&#171;&#185;                     &#226;&#171;&#185;                        &#226;&#171;&#186;           activity state Pkc1-MAP kinase pathway (17). We found <br /> ost3                    &#226;&#171;&#185;                     &#226;&#171;&#185;                        &#226;&#171;&#186;           majority mutants (12/21) display high basal Rlm1- <br /> sec28                   &#226;&#171;&#185;                     &#226;&#171;&#185;                        &#226;&#171;&#186; <br />                                                                                    LacZ reporter activity (Fig. 2). The mutants highest <br /> opi3                    &#226;&#171;&#185;                     &#226;&#171;&#186;                        &#226;&#171;&#185; <br /> mid1                    &#226;&#171;&#185;                     &#226;&#171;&#186;                        &#226;&#171;&#186;           basal reporter activity (&#226;&#172;&#381;3 times wild-type cells) point <br /> per1                    &#226;&#171;&#185;                     &#226;&#171;&#186;                        &#226;&#171;&#186;           best-known triggers roles CWI signaling: <br /> spa2                    &#226;&#171;&#185;                     &#226;&#171;&#186;                        &#226;&#171;&#186;           las21&#226;&#338;&#172;, per1&#226;&#338;&#172;, ost3&#226;&#338;&#172; defective cell wall/surface syn- <br /> ump1                    &#226;&#171;&#185;                     &#226;&#171;&#186;                        &#226;&#171;&#186;           thesis (2, 7, 18), bni1&#226;&#338;&#172; (and api2&#226;&#338;&#172; thought be) <br /> ydl203c                 &#226;&#171;&#185;                     &#226;&#171;&#186;                        &#226;&#171;&#186; <br /> api2                    &#226;&#171;&#186;                     &#226;&#171;&#185;                        &#226;&#171;&#186;           defective actin organization/cell morphogenesis (3, 6). A <br /> bni1                    &#226;&#171;&#186;                     &#226;&#171;&#185;                        &#226;&#171;&#186;           novel trigger suggested high reporter activity seen for <br /> swi4                    &#226;&#171;&#186;                     &#226;&#171;&#185;                        &#226;&#171;&#186;           opi3&#226;&#338;&#172; mutants. This mutant lacks phospholipid methyltrans- <br /> mig1                    &#226;&#171;&#186;                     &#226;&#171;&#186;                        &#226;&#171;&#185;           ferase required phosphatidylcholine (PC) synthesis (23). <br />   a <br />     &#226;&#171;&#185; indicates hypersensitivity compound relative sensitivity    The CWI pathway respond changes PC <br /> congenic wild-type control cells. &#226;&#171;&#186; indicates sensitivity indistinguishable   levels, complementing established role Pkc1 directly <br /> that congenic wild-type cells. <br />                                                                                    phosphorylating enzymes involved PC synthesis (4, 42). <br />                                                                                       To date, feedback control CWI signaling estab- <br />                                                                                    lished MAP kinase downstream branch (22). Impor- <br />                                                                                    tantly, mutants PKC1 network displaying <br /> in distinct downstream branches; Spa2, component                          elevated Rlm1-LacZ reporter activity lack known component <br /> the polarisome complex that, Bni1, properly localizes                     pathway. One these, Swi4, acts downstream the <br /> downstream MAP kinase module cell periphery.                                MAP kinase branch: others, Bni1 Mig1, (22). <br />    Chemical-genetic epistasis test. Are 15 genes                         These observations support feedback control CWI path- <br /> closely linked CWI signaling? To begin address ques- <br /> tion, exploited unique property pkc1&#226;&#338;&#172; mutants: invia- <br /> bility suppressed high-osmolarity medium, condition <br /> that thought physically stabilize fragile mutant cells <br /> (22). Pathway activity required cell viability <br /> under condition. High osmolarity does completely <br /> bypass requirement kinase: Pkc1 required for <br /> growth high temperature, condition. <br />    We reasoned mutations affect Pkc1 pathway/ <br /> process enhance phenotype caused loss of <br /> Pkc1 activity osmotically stabilized medium room <br /> temperature. Pathway activity required con- <br /> dition, hence, mutations affecting pathway com- <br /> ponent exacerbate phenotypes mutants <br /> with lowered Pkc1 activity. Osmotic stabilization should <br /> thus suppress synthetic genetic interaction PKC1 <br /> noted normal media. In contrast, mutations affect                          FIG. 2. The basal activity Rlm1-LacZ reporter construct is <br /> parallel process enhance pkc1 defects ab-                       altered majority Pkc1 genetic network. Each 21 <br /> sence presence osmotic stabilization, previ-                     network mutants, congenic wild-type strain, trans- <br /> ously noted BRO1 gene (25).                                           formed plasmid containing LacZ expressed control of <br />                                                                                    Slt2-dependent transcription factor Rlm1. The transformed mu- <br />    We tested 21 mutants. We 1 M sorbitol sup-                       tants assayed &#226;?&#164;-galactosidase activity. The relative activity was <br /> presses staurosporine hypersensitivity two-thirds                    determined comparison activity wild-type transformed <br /> mutants (Fig. 1A B), functionally linking cor-                     cells. The experiment performed multiple different days in <br /> responding gene products Pkc1. This chemical-genetic test                       triplicate day. Data day scaled average <br />                                                                                    data wild-type controls day. The average relative activ- <br /> of functional relatedness highly specific: mutants                 ities mutants days standard deviations shown <br /> lack known component pathway fall &#226;&#8364;&#339;within-                      mutant. The relative activity wild-type cells 1.0 by <br /> pathway&#226;&#8364;? class (Fig. 1B).                                                          <span id='am-4' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span>. <br />  1884     KRAUSE ET AL.                                                                                                    EUKARYOT. CELL <br />  <br />  <br /> way indicate feedback extensive <br /> than previously recognized. <br />    Some interacting genes encode activators CWI signaling. <br /> Some remaining genes PKC1 network encode <br /> activators CWI signaling. Mutants lacking proteins <br /> would low pathway activity, rendering hypersensi- <br /> tive reduction. In total, mutants displayed low <br /> basal Rlm-LacZ reporter activity, consistent compro- <br /> mised CWI signaling: bck1&#226;&#338;&#172;, egd1&#226;&#338;&#172;, ctf19&#226;&#338;&#172;, ydl203c&#226;&#338;&#172; (Fig. <br /> 2). BCK1 encodes known MAP kinase kinase kinase is <br /> absolutely required activity Slt2 Rlm1. Rlm1-LacZ <br />                                                                       FIG. 3. Ydl203c/Ack1 activates Rlm1-LacZ reporter activity is <br /> reporter activity minimal mutant (&#226;&#172;?2% wild-type <br />                                                                     required stress induction reporter. (A) Wild-type cells <br /> levels). EGD1 encodes subunit nascent polypeptide-         cotransformed Rlm1-LacZ reporter plasmid either <br /> associated complex (30). CTF19, gene regulated CWI         vector control plasmid expressing YDL203c GAL1,10 <br /> pathway (31), encodes protein involved accurate mitotic        promoter. The transformed cells grown synthetic medium <br /> chromosome segregation (13). Both egd1&#226;&#338;&#172; ctf19&#226;&#338;&#172; mutants          containing galactose sucrose induce expression YDL203c. <br />                                                                     The transformed cells assayed &#226;?&#164;-galactosidase activity. The <br /> display modest reductions reporter activity (&#226;&#172;&#402;81%            experiment performed triplicate. The relative activity de- <br /> &#226;&#172;&#402;66% wild-type levels, respectively). Egd1 Ctf19         termined relative average activity vector controls analyzed <br /> be novel activators CWI signaling, roles remain     day. Standard deviations replicates shown for <br /> be clarified. Finally, mutants lacking uncharacterized          strain. (B) Wild-type yld203c&#226;&#338;&#172; cells transformed the <br />                                                                     Rlm1-LacZ reporter construct examined ability acti- <br /> YDL203c gene display substantial reductions reporter gene <br />                                                                     vate expression reporter construct. Overnight cell cultures <br /> activity (down &#226;&#172;&#402;25% wild-type levels).                        diluted treated 150 &#226;?&#174;g/ml calcofluor white <br />    Ydl203c/Ack1 novel upstream activator Pkc1. Loss      (CW), 200 &#226;?&#174;g/ml chlorpromazine (CPZ), 40 &#226;?&#174;g/ml latrunculin B <br /> YDL203c results low Rlm1-LacZ reporter activity. The stau-       (LatB) 3 hours prior assay &#226;?&#164;-galactosidase activity. The <br /> rosporine hypersensitivity ydl203c&#226;&#338;&#172; suppressed sorbi-      activity determined relative untreated, wild-type cells, <br />                                                                     analyses performed triplicate day. Stan- <br /> tol, supporting role Ydl203c CWI/Pkc1 pathway          dard deviations replicates shown strain. <br /> (Fig. 1B data shown). If Ydl203c important <br /> for CWI signaling, mutants lacking protein should <br /> display appropriate phenotype. The mutants tem- <br /> perature sensitive growth hypersensitive tunica-     (Fig. 4A B). Second, examined overexpression of <br /> mycin treatment (Table 2 data shown).                       YDL203c suppress defects mutants defective at <br />    If Ydl203c bona fide regulator CWI signaling,    various levels pathway. We overexpression of <br /> expect overproduction activation stimulate reporter       gene efficiently suppresses defects rom2&#226;&#338;&#172; tus1&#226;&#338;&#172; <br /> activity. Indeed, overexpression YDL203c         mutants wsc1&#226;&#338;&#172;/wsc1&#226;&#338;&#172; mutants lacking major upstream <br /> wild-type cells elevates Rlm1-LacZ reporter activity approx-     sensor defects pkc1ts slt2&#226;&#338;&#172;/slt2&#226;&#338;&#172; mutants <br /> imately twofold (Fig. 3A).                                          lacking key downstream components. These data point the <br />    Is activation CWI signaling compromised ydl203c&#226;&#338;&#172;         possibility Ydl203c acts Pkc1 pathway or <br /> mutant? We assayed activation reporter activity wild-type     upstream Pkc1 (Fig. 4C D). We rename gene ACK1 <br /> and ydl203c&#226;&#338;&#172; mutants cell surface stresses (calcofluor         (activator C kinase 1). <br /> white chlorpromazine treatment) actin depolymer-            No clear molecular mode action indicated se- <br /> ization (latrunculin B treatment). We reporter     quence Ack1 save presence SEL-1 domain, <br /> efficiently activated mutant treat-     suggesting possibility protein-protein interactions (24). <br /> ments tested (Fig. 3B). Ydl203c required       Two recent proteomic studies link Ack1 Rho1 GEFs: <br /> the transduction cell surface actin cytoskeleton defects   Ack1 complexes Rom2 (8, 9) Tus1 (8), <br /> the CWI pathway. Activation reporter activity           consistent role upstream Pkc1 CWI pathway. <br /> latrunculin B treatment is, however, efficient          Does Ack1 act Rho1 GEFs? Our genetic analysis <br /> upon treatment cell surface stressors, indicating     indicates Rom2 activity wholly dependent Ack1 <br /> Ydl203c preferentially enhance input. Overall,         vice versa: overexpression suppresses defect <br /> protein appears selectively affect CWI sig-        caused absence other. The holds Tus1 <br /> naling basal induced conditions.                     Ack1. There possible explanations. First, Ack1 <br />    If Ydl203c acts Pkc1 pathway, epistatic       act independently main Rho1 GEFs stimu- <br /> analysis specify point action protein. We       late Pkc1 activity. Second, Ack1 act Rho1 GEFs, <br /> undertook complementary analyses. First, examined         activity wholly dependent Ack1. If <br /> activation overexpression known pathway components,           model applies, ACK1 affect CWI sig- <br /> ranging upstream Rom2 Tus1 Rho1 GEFs            naling absence Rom2 Tus1; latter <br /> downstream Slt2 MAP kinase, suppress tunicamycin          applies, ACK1 function abolished absence <br /> sensitivity ydl203c&#226;&#338;&#172; mutants. We activation        Rho1 GEFs. We constructed rom2&#226;&#338;&#172; tus1&#226;&#338;&#172; double <br /> overexpression corresponding genes efficiently       mutant haploids standard genetic manipulations. We found <br /> suppress mutant defect, consistent idea           double mutant viable strain background but <br /> Ydl203c acts upstream Rho1 GEFs pathway         temperature sensitive hypersensitive benomyl. We <br />  VOL. 7, 2008                                                                                GENETIC NETWORK AROUND PKC1                      1885 <br />  <br />  <br />                                                                                                        &#226;&#710;&#8224; <br />                                                                     &#226;&#710;&#8224; <br />                                      &#194;&#181; <br />                                      &#194;&#181; <br />                                      &#194;&#181; <br />                                      &#194;&#181; <br />                                      &#194;&#181; <br />  <br />                                                                                                 &#226;&#710;&#8224; <br />                                                                                                            &#226;&#710;&#8224; <br />  <br />                                  &#226;&#710;&#8224; <br />                                  &#226;&#710;&#8224; <br />                                 &#226;&#710;&#8224; <br />                                                                                                            &#226;&#710;&#8224; <br />                                &#226;&#710;&#8224;&#226;&#710;&#8224; <br />                                 &#226;&#710;&#8224;      &#226;&#710;&#8224;                                                        &#226;&#710;&#8224; <br />                                 &#226;&#710;&#8224;      &#226;&#710;&#8224; <br />    FIG. 4. Epistasis analysis places Ydl203c/Ack1 Pkc1 pathway upstream Pkc1. (A) ydl203c&#226;&#338;&#172; cells transformed plasmids <br /> overexpressing ROM2, TUS1, PKC1, MKK1, SLT2 vector control. Transformed cells grown synthetic medium, diluted OD600 <br /> of 0.02, serially diluted fresh medium. The dilutions spotted synthetic medium plates containing tunicamycin (Tm; 1.5 <br /> &#226;?&#174;g/ml). The plates incubated 25&#194;&#176;C 2 days, growth transformants compared vector control. &#226;&#8364;&#339;&#226;&#171;&#185;&#226;&#8364;? indicates <br /> growth better vector control. (B) Example suppression tunicamycin hypersensitivity ydl203c&#226;&#338;&#172; cells overexpression <br /> of ROM2. (C) wsc1&#226;&#338;&#172;/wsc1&#226;&#338;&#172;, rom2&#226;&#338;&#172;, tus1&#226;&#338;&#172;, pkc1ts, slt2&#226;&#338;&#172;/&#226;&#338;&#172;, rom2&#226;&#338;&#172; tus1&#226;&#338;&#172; cells congenic wild-type cells transformed vector <br /> control plasmid expressing YDL203c GAL1,10 promoter. The transformed cells streaked synthetic medium plates <br /> containing galactose sucrose induce expression YDL203c. Plates incubated 25&#194;&#176;C, duplicate set incubated 37&#194;&#176;C <br /> to examine ability Ydl203c suppress temperature sensitivity pkc1ts, slt2&#226;&#338;&#172;/&#226;&#338;&#172;, rom2&#226;&#338;&#172; tus1&#226;&#338;&#172; cells. Plates containing benomyl (20 <br /> &#226;?&#174;g/ml) used assess ability YDL203c overexpression suppress benomyl hypersensitivity rom2&#226;&#338;&#172; rom2&#226;&#338;&#172; tus1&#226;&#338;&#172; cells. Plates <br /> containing chlorpromazine (200 &#226;?&#174;g/ml) used examine ability YDL203c overexpression suppress chlorpromazine hypersen- <br /> sitivity tus1&#226;&#338;&#172; cells. Plates containing calcofluor white (25 &#226;?&#174;g/ml) used examine ability YDL203c overexpression suppress the <br /> calcofluor white hypersensitivity wsc1&#226;&#338;&#172;/wsc1&#226;&#338;&#172; cells. All plates incubated 25&#194;&#176;C 2 days growth transformants was <br /> assessed. &#226;&#8364;&#339;&#226;&#171;&#185;&#226;&#8364;? indicates growth better vector control transformants; &#226;&#8364;&#339;&#226;&#171;&#186;&#226;&#8364;? indicates growth equal vector control. <br /> (D) Example ability YDL203c overexpression suppress benomyl hypersensitivity rom2&#226;&#338;&#172; cells. Wt, wild type. <br />  <br />  <br />  <br /> found overexpression ACK1 unable suppress             genetic analysis useful addition genetic <br /> the defects tested rom2&#226;&#338;&#172; tus1&#226;&#338;&#172; double mutants (Fig. 4              arsenal. <br /> data shown). Whatever role, Ack1 novel upstream                  The genetic interaction network PKC1. The synthetic <br /> activator CWI-Pkc1 signaling functionally dependent              lethal network PKC1 rich genes functionally and <br /> the Rho1 GEFs.                                                             closely linked Pkc1. All 21 null mutants display the <br />                                                                            appropriate chemical sensitivities (Table 2). An independent <br />                           DISCUSSION                                       chemical-genetic epistasis test places two-thirds genes <br />                                                                            functional envelope Pkc1 (Fig. 1), six <br />    DN-SGA analysis. Here, report adaptation SGA                   encode known components CWI pathway. We confirm <br /> methodology, DN-SGA, &#226;&#8364;&#339;query&#226;&#8364;? mutation dom-                    gene, YDL203c/ACK1, encodes novel modula- <br /> inant negative expressed trans plasmid              tor pathway (see below). Our findings consistent with <br /> the control regulatable promoter. In case PKC1 <br />                                                                            long-held expectation genetic enhancements hypo- <br /> query, adapted method identified true positives high <br />                                                                            morphic alleles efficiently enrich mutations affecting the <br /> degree specificity: &#226;&#172;&#381;95% primary strains identified as <br />                                                                            pathway biological process (26). <br /> robustly sensitive expression dominant-negative allele <br />                                                                               If synthetic lethal network PKC1 preferentially <br /> were sensitive protein kinase C inhibitor <br />                                                                            identifies components CWI pathway, the <br /> staurosporine. <br />                                                                            known components pathway represented the <br />    DN-SGA analysis strengths. Because query al- <br /> lele plasmid borne extrachromosomal,                network? There multiple possible reasons. First, some <br /> genetically linked chromosomal locus (16). The entire               components, like Rho1, essential covered this <br /> genome surveyed approach. Second,                   analysis. Second, components redundant, e.g., Mkk1 <br /> lethality dependent conditional expression                Mkk2, mutation gene causes little or <br /> dominant-negative query allele, terminal phenotype asso-               phenotypic change. Third, components in- <br /> ciated synthetic lethal interaction easily de-            volved branches processes contribute to <br /> termined. For example, synthetic lethal interac-               essential role(s) Pkc1. Fourth, components may <br /> tions PKC1 reported accompanied increased                 contribute preferentially activated the <br /> cell lysis, consistent idea cell integrity defect          basal state pathway. Only mutations affecting basal <br /> underlies majority interactions (S. A. Krause             activity identified nonstressed conditions <br /> J. V. Gray, unpublished observations). DN-SGA analysis ex-                 examined here. Finally, SGA analysis de- <br /> tends range alleles amenable systematic                 rivatives systematic genome-wide, suffer high <br />  1886     KRAUSE ET AL.                                                                                                             EUKARYOT. CELL <br />  <br />  <br /> rate false negatives (36). Such genetic screens              Ack1 new upstream component Pkc1 pathway. <br /> fully saturating.                                                    Multiple lines evidence place uncharacterized protein <br />    Functional relationships underlying genetic network.          Ydl203c/Ack1 Pkc1 pathway. First, ack1&#226;&#338;&#172; mutants are <br /> Here, probed likely functional relationships        hypersensitive lowered Pkc1 activity. Second, hypersen- <br /> underlie genetic interactions PKC1. We estimated            sitivity efficiently suppressed osmotic stabilization (Fig. 1), <br /> Pkc1 pathway activity mutant using Rlm1-LacZ           placing Ack1 function envelope Pkc1 vivo <br /> reporter, faithful indicator Pkc1-Slt2 activity state (17).     activity. Third, CWI pathway compromised ack1&#226;&#338;&#172; <br /> We identified distinct subgroups            mutants: mutants display low Rlm1-LacZ reporter activity <br /> PKC1 network. We consider turn.                              (Fig. 2) hypersensitive tunicamycin (Table 2), and <br />    We majority (12/21) mutants PKC1         hypersensitivity suppressed overexpression many <br /> network display high basal reporter activity, consistent    pathway components (Fig. 4A). Fourth, overexpression of <br /> idea mutants cause defects Pkc1 path-         ACK1 sufficient stimulate CWI pathway activity: overex- <br /> way responds. These mutants particularly dependent          pression sufficient suppress defects mutants <br /> on Pkc1 activity high activity required ameliorate   defective upstream signaling (Fig. 4C) elevate basal <br /> the phenotypic consequences mutations. Such behavior          Rlm1-LacZ reporter activity (Fig. 3A). Finally, standard ep- <br /> is expected signaling pathways affect cell viability    istasis analysis places Ack1 pathway indicates <br /> that negative-feedback control. Both conditions ap-        acts upstream Pkc1 (Fig. 4). <br /> ply Pkc1/CWI pathway: overall pathway essential           We fully understand Ack1 modulates CWI <br /> and known feedback control activity state      signaling. The protein complex the <br /> of nonessential, downstream MAP kinase module (22).              Rho1 GEFs Rom2 (8, 9) Tus1 (8). Indeed, genetic <br /> Our data point possibility feedback control      analysis indicates Rom2 Tus1 required for <br /> extensive embraces Bni1 Mig1 branches.              Ack1 affect CWI signaling: overexpression ACK1 can <br />    Six mutants display normal basal reporter activity,        suppress defects rom2&#226;&#338;&#172; tus1&#226;&#338;&#172; mutants those <br /> suggestive normal Pkc1 pathway activity, assessed       double mutant (Fig. 4C). Ack1 modulator <br /> activity Slt2-Rlm1-dependent reporter. Such mutants            scaffold Rho1 GEF complexes. Whatever role, it <br /> could deficient branch pathway       does appear grossly alter stability proteins or <br /> MAP kinase branch controls Rlm1-LacZ reporter. In           subcellular localization (S. A. Krause J. V. Gray, <br /> addition, CWI pathway activity (as assessed reporter          unpublished observations). Our investigations ongoing. <br /> gene activity) feedback control response           Ack1 clearly important setting activity level the <br /> to loss gene products: activity remains normal. Alter-      pathway does appear required gross activation <br /> natively, mutants deficient process          pathway response cell surface cytoskeletal de- <br /> pathway acts parallel CWI pathway           fects (Fig. 3B). Whatever mode action, Ack1 joins ever <br /> absence surveyed pathway. Examples            increasing number components, including PUF protein <br /> both types appear represented mutant class:        Mpt5 (34) ubiquitination enzyme Ubp3 (39), point <br /> chemical-genetic epistasis test (Fig. 1) places     unanticipated complexity setting modulating pathway <br /> proteins envelope Pkc1 function. Two      activity. <br /> known components pathway: Spa2 Mid1. Spa2 is <br />                                                                                                ACKNOWLEDGMENTS <br /> important proper localization Slt2 MAP kinase, but <br /> this localization important activation Rlm1          We thank David Levin, David Pellman, Mike Hall plasmids; <br /> transcription factor Slt2. Mid1 putative calcium channel     Charlie Boone query strains; Josephine McGhie technical <br />                                                                      support. We thank laboratory members colleagues helpful <br /> lying separate branch relative MAP kinase module, <br />                                                                      discussions. <br /> leaving Slt2-Rlm1 communication fully intact.                          This work supported research grants C20144 BB/ <br />    Finally, null mutants display compromised reporter ac-       E011632 Biotechnology Biological Sciences Research <br /> tivity, suggestive low Pkc1-Slt2-Rlm1 activity. Such mutants      Council (J.V.G.). <br /> could hypersensitive reduction Pkc1 activity                                         REFERENCES <br /> the activity Pkc1 (as assessed Slt2-Rlm1-dependent           1. Baetz, K., J. Moffat, J. Haynes, M. Chang, B. Andrews. 2001. Transcrip- <br /> reporter) unstressed mutants closer minimal thresh-         tional coregulation cell integrity mitogen-activated protein kinase Slt2 <br />                                                                          cell cycle regulator Swi4. Mol. Cell. Biol. 21:6515&#226;&#8364;&#8220;6528. <br /> old activity kinase required support cell viability.        2. Benachour, A., G. Sipos, I. Flury, F. Reggiori, E. Canivenc-Gansel, C. Vion- <br /> These mutants lack positively acting components            net, A. Conzelmann, M. Benghezal. 1999. Deletion GPI7, yeast gene <br /> pathway. Indeed, genes behave chemical-genetic             required addition chain glycosylphosphatidylinositol <br />                                                                          (GPI) core structure, affects GPI protein transport, remodeling, cell wall <br /> epistasis test Pkc1 pathway (Fig. 1). The mutant          integrity. J. Biol. Chem. 274:15251&#226;&#8364;&#8220;15261. <br /> with lowest reporter activity lacks Bck1, kinase known       3. Bidlingmaier, S., M. Snyder. 2002. Large-scale identification genes <br /> be required communication Pkc1 Slt2. In                  important apical growth Saccharomyces cerevisiae directed allele <br />                                                                          replacement technology (DART) screening. Funct. Integr. Genomics 1:345&#226;&#8364;&#8220; <br /> contrast, mutants lacking EGD1 CTF19 display                  356. <br /> modest reporter defect. The relationships Edg1            4. Choi, M. G., V. Kurnov, M. C. Kersting, A. Sreenivas, G. M. Carman. <br />                                                                          2005. Phosphorylation yeast choline kinase protein kinase C. <br /> Ctf19 CWI signaling established,             Identification Ser25 Ser30 major sites phosphorylation. J. Biol. <br /> CTF19 transcriptional target pathway (31). The               Chem. 280:26105&#226;&#8364;&#8220;26112. <br /> fourth mutant, lacking YDL203c/ACK1, displays interme-             5. De Filippi, L., M. Fournier, E. Cameroni, P. Linder, C. DeVirgilio, M. Foti, <br />                                                                          O. Deloche. 2007. Membrane stress coupled rapid translational <br /> diate reporter defect. We gene encodes           control gene expression chlorpromazine-treated cells. Curr. Genet. <br /> novel activator CWI signaling (see below).                            52:171&#226;&#8364;&#8220;185. <br />  VOL. 7, 2008                                                                                              GENETIC NETWORK AROUND PKC1                                1887 <br />  <br />  6. Evangelista, M., S. Zigmond, C. Boone. 2003. Formins: signaling effec-        26. Ooi, S. L., X. Pan, B. D. Peyser, P. Ye, P. B. Meluh, D. S. Yuan, R. A. Irizarry, <br />     tors assembly polarization actin filaments. J. Cell Sci. 116:2603&#226;&#8364;&#8220;         J. S. Bader, F. A. Spencer, J. D. Boeke. 2006. Global synthetic-lethality <br />     2611.                                                                                 analysis yeast functional profiling. Trends Genet. 22:56&#226;&#8364;&#8220;66. <br />  7. Fujita, M., M. Umemura, T. Yoko-o, Y. Jigami. 2006. PER1 required          27. Pan, X., D. S. Yuan, D. Xiang, X. Wang, S. Sookhai-Mahadeo, J. S. Bader, <br />     GPI-phospholipase A2 activity involved lipid remodeling GPI-            P. Heiter, F. Spencer, J. D. Boeke. 2004. A robust toolkit functional <br />     anchored proteins. Mol. Biol. Cell 17:5253&#226;&#8364;&#8220;5264.                                      profiling yeast genome. Mol. Cell 16:487&#226;&#8364;&#8220;496. <br />  8. Gavin, A. C., M. Bosche, R. Krause, P. Grandi, M. Marzioch. 2002.             28. Philip, B., D. E. Levin. 2001. Wsc1 Mid2 cell surface sensors for <br />     Functional organization yeast proteome systematic analysis               cell wall integrity signaling act Rom2, guanine nucleotide <br />     protein complexes. Nature 415:141&#226;&#8364;&#8220;147.                                                exchange factor Rho1. Mol. Cell. Biol. 21:271&#226;&#8364;&#8220;280. <br />  9. Gavin, A. C., P. Aloy, P. Grandi, R. Krause, M. Boesche, M. Marzioch, C.          29. Pringle, J. R. 1991. Staining bud scars cell wall chitin with <br />     Rau, L. J. Jensen, S. Bastuck, B. Dumpelfeld, A. Edelmann, M. A. Heurtier,            calcofluor. Methods Enzymol. 194:732&#226;&#8364;&#8220;735. <br />     V. Hoffman, C. Hoefert, K. Klein, M. Hudak, A. M. Michon, M. Schelder, M.         30. Reimann, B., J. Bradsher, J. Franke, E. Hartmann, M. Wiedmann, S. Prehn, <br />     Schirle, M. Remor, T. Rudi, S. Hooper, A. Bauer, T. Bouwmeester, G.                   B. Wiedmann. 1999. Initial characterization nascent polypeptide- <br />     Casari, G. Drewes, G. Neubauer, J. M. Rick, B. Kuster, P. Bork, R. B.                 associated complex yeast. Yeast 15:397&#226;&#8364;&#8220;407. <br />     Russell, G. Superti-Furga. 2006. Proteome survey reveals modularity        31. Roberts, C. J., B. Nelson, M. J. Marton, R. Stoughton, M. R. Meyer, H. A. <br />     yeast cell machinery. Nature 440:631&#226;&#8364;&#8220;636.                                         Bennett, Y. D. He, H. Dai, W. L. Walker, T. R. Hughes, M. Tyers, C. Boone, <br /> 10. Gietz, R. D., R. H. Scheistl, A. R. Wellems, R. A. Woods. 1995. Studies           S. H. Friend. 2000. Signaling circuitry multiple MAPK pathways <br />     transformation intact yeast cells LiAc/SS-DNA/PEG pro-               revealed matrix global gene expression profiles. Science 287:873&#226;&#8364;&#8220;880. <br />     cedure. Yeast 11:355&#226;&#8364;&#8220;360.                                                         32. Schmelzle, T., S. B. Helliwell, M. N. Hall. 2002. Yeast protein kinases <br /> 11. Gray, J. V., J. P. Ogas, Y. Kamada, M. Stone, D. E. Levin, I. Herskowitz.         Rho1 exchange factor Tus1 novel components cell integ- <br />     1997. A role Pkc1 MAP kinase pathway Saccharyomces cerevisiae              rity pathway yeast. Mol. Cell. Biol. 22:1329&#226;&#8364;&#8220;1339. <br />     bud emergence identification putative upstream regulator.             33. Schmidt, A., M. Bickle, T. Beck, M. N. Hall. 1997. The yeast phospha- <br />     EMBO J. 16:4924&#226;&#8364;&#8220;4937.                                                                 tidylinositol kinase homolog Tor2 activates Rho1 Rho2 exchange <br /> 12. Harrison, J. C., E. S. Bardes, Y. Ohya, D. J. Lew. 2001. A role           factor Rom2. Cell 88:531&#226;&#8364;&#8220;542. <br />     Pkc1p/Mpk1p kinase cascade morphogenesis checkpoint. Nat. Cell <br />                                                                                       34. Stewart, M. S., S. A. Krause, J. McGhie, J. V. Gray. 2007. Mpt5p, stress <br />     Biol. 3:417&#226;&#8364;&#8220;420. <br />                                                                                           tolerance- lifespan-promoting PUF protein Saccharomyces cerevisiae, <br /> 13. Hyland, K. M., J. Kingsbury, D. Koshland, P. Hieter. 1999. Ctf19p: a <br />                                                                                           acts upstream cell wall integrity pathway. Eukaryot. Cell 6:262&#226;&#8364;&#8220;270. <br />     novel kinetochore protein Saccharomyces cerevisiae potential link <br />                                                                                       35. Tamaoki, T., H. Nomoto, I. Takahashi, Y. Kato, M. Morimoto, F. <br />     kinetochore mitotic spindle. J. Cell Biol. 145:15&#226;&#8364;&#8220;28. <br />                                                                                           Tomita. 1986. Staurosporine, potent inhibitor phospholipid/Ca2&#226;&#171;&#185; de- <br /> 14. Irie, K., M. Takase, S. Leek, D. E. Levin, H. Araki, K. Matsumoto, Y. <br />                                                                                           pendent protein kinase. Biochem. Biophys. Res. Commun. 135:397&#226;&#8364;&#8220;4 <br /> </body></html>