<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Published online 4 November 2008                                                     Nucleic Acids Research, 2008, Vol. 36, No. 22 e151 <br />                                                                                                                   doi:10.1093/nar/gkn811 <br />  <br />  <br /> A novel method generate unmarked <br /> gene deletions intracellular pathogen <br /> Rhodococcus equi using 5-fluorocytosine <br /> conditional lethality <br /> R. van der Geize1,*, W. Jong1, G. I. Hessels1, A. W. F. Grommen2, A. A. C. Jacobs2 <br /> and L. Dijkhuizen1 <br /> 1 <br />  Groningen Biomolecular Sciences Biotechnology Institute (GBB), Department Microbiology, University <br /> of Groningen, Kerklaan 30, 9751 NN Haren 2Intervet Schering-Plough Animal Health, Bacteriological R&amp;D, <br /> Wim Ko&#194;&#168;rverstraat 35, Postbus 31, 5830 AA Boxmeer, The Netherlands <br />  <br /> Received September 16, 2008; Revised Accepted October 10, 2008 <br />  <br />  <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span>                                                                        INTRODUCTION <br /> A novel method efficiently generate unmarked                                 An important molecular tool functional genomics stu- <br /> in-frame gene deletions Rhodococcus equi                                 dies targeted inactivation gene interest. <br /> developed, exploiting cytotoxic effect                                   Ideally, unmarked gene deletions constructed using a <br /> 5-fluorocytosine (5-FC) action cytosine                               positive selection step rare second recombination <br /> deaminase (CD) uracil phosphoribosyltransfer-                               event. The sacB gene Bacillus subtilis most <br />                                                                                 widely used suicide genes, conferring sucrose sensitivity <br /> ase (UPRT) enzymes. The opportunistic, intracellular <br />                                                                                 Gram-negative bacteria (1,2). Previously, we <br /> pathogen R. equi resistant high concentrations <br />                                                                                 reported use sacB counter-selection e&#239;&#172;&#402;ciently <br /> of 5-FC. Introduction Escherichia coli genes                                 generate unmarked gene deletions Rhodococcus erythro- <br /> encoding CD UPRT conferred conditional                                      polis (3). This method subsequently applied in <br /> lethality R. equi cells incubated 5-FC.                                 Rhodococcus species (4&#226;&#8364;&#8220;6). Sucrose sensitivity by <br /> <span id='am-6' about='protege:TO' typeof='owl:Thing'>To</span> exemplify use codA::upp cassette                               sacB reported mycolic acid con- <br /> counter-selectable marker, unmarked in-frame                                 taining actinobacteria, i.e. Corynebacterium glutamicum <br /> gene deletion mutant R. equi constructed.                                Mycobacterium sp. (7). Despite considerable e&#239;&#172;&#8364;orts, <br /> The supA supB genes, putative cho-                                unable apply sacB counter-selectable <br /> lesterol catabolic gene cluster, efficiently                               marker Rhodococcus equi, lack sucrose sen- <br /> deleted R. equi wild-type genome. Pheno-                               sitivity. Similar observations reported for <br /> typic analysis generated &quot;supAB mutant                                   Streptomyces lividans (7). The sacB counter-selection <br /> confirmed supAB essential growth                                generally applicable Gram-positive <br /> R. equi cholesterol. Macrophage <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span>survival</span> assays                              bacteria, like B. subtilis. In B. subtilis, mazF, encoding <br /> revealed &quot;supAB mutant able                                      toxin, used suicide marker generate <br />                                                                                 unmarked gene deletion mutants (8). This method relies <br /> survive proliferate macrophages comparable <br />                                                                                 availability tightly regulated, inducible pro- <br /> to wild type. Thus, cholesterol metabolism does                                 moter, limiting applicability bacteria devel- <br /> not appear essential macrophage                                       oped molecular toolboxes. The use alternative suicide <br /> survival R. equi. The CD-UPRT based 5-FC                                     genes reported well, e.g. glkA, pyrF, upp and <br /> counter-selection useful asset                                  rpsL (9&#226;&#8364;&#8220;12). A major drawback markers, how- <br /> the generation unmarked in-frame gene deletions                              ever, function glkA, pyrF, upp or <br /> in actinobacteria well, actinobacteria                              strA <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span>null mutants</span>, respectively, necessitating construc- <br /> generally appear 5-FC resistant 5-FU                                  tion, availability, null mutant strain be <br /> sensitive.                                                                      mutated. A PCR-targeted gene replacement strategy has <br />  <br />  <br /> *To correspondence addressed. Tel: +31 50 363 2257; Fax: +31 50 363 2254; Email: r.van.der.geize@rug.nl <br />  <br /> The authors wish known that, opinion, authors regarded joint First Authors. <br />  <br /> &#195;&#376; 2008 The Author(s) <br /> This Open Access article distributed terms Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/ <br /> by-nc/2.0/uk/) permits unrestricted non-commercial use, distribution, reproduction medium, provided original work properly cited. <br />  e151 Nucleic Acids Research, 2008, Vol. 36, No. 22                                                             PAGE 2 OF 10 <br />  <br />  <br /> been reported streptomycetes relies avail-       M. tuberculosis H37Rv, mce4 cluster in <br /> ability ordered genomic libraries e&#239;&#172;&#402;ciently generate       H37Rv essential growth cholesterol. <br /> mutants (13). This method, however, requires labo-                The R. equi strain 103S genome sequence recently has <br /> rious extra round mutagenesis remove resist-           available (http://www.sanger.ac.uk/Projects/ <br /> ance marker integrated genome generate               R_equi/). Proper genome annotation identi&#239;&#172;?cation <br /> unmarked mutants. Recently, simpli&#239;&#172;?ed method               pathogenicity genes requires simple methods gene <br /> marker removal actinomycetes using Flp recombinase            deletion mutagenesis. A method isolation gene <br /> has described (14).                                         deletion mutants R. equi double homologous <br />    Cytosine deaminase (CD, EC 3.5.4.5) uracil phos-          recombination strategy &#239;&#172;?rst reported Navas et al. <br /> phoribosyltransferase (UPRT, EC 2.4.2.9) enzymes             (27). An improved method, using lacZ counter- <br /> involved pyrimidine salvage pathway, converting           selectable marker, subsequently reported Jain <br /> cytosine uracil dUMP. CD activity              et al. (33). These methods, however, involve screen- <br /> found certain prokaryotes lower eukaryotes. The           ing handling large numbers colonies select for <br /> genes encoding activities Escherichia coli, codA        rare second recombination event (34,35). Here we <br /> and upp, respectively, cloned characterized        introduction codA::upp cassette confers <br /> (15,16). Interestingly, microorganisms expressing CD con-        5-FC sensitivity R. equi allowing positive selection of <br /> vert innocuous cytosine analog 5-&#239;&#172;&#8218;uorocytosine               targeted gene deletion mutants. A simple e&#239;&#172;&#402;cient <br /> (5-FC) 5-&#239;&#172;&#8218;uorouracil (5-FU), highly toxic com-            procedure generate unmarked in-frame gene deletions <br /> pound lethal living cells. The cytotoxicity largely        R. equi reported, exempli&#239;&#172;?ed construction and <br /> exerted following UPRT mediated conversion 5-FU               characterization  supAB mutant. <br /> into 5-&#239;&#172;&#8218;uoro-dUMP, irreversibly inactivates thymi- <br /> dylate synthase inhibiting RNA DNA synthesis <br /> (17). Heterologous expression E. coli codA shown          MATERIALS AND METHODS <br /> to confer 5-FC sensitivity mammalian cells, ordinarily        Culture media growth conditions R. equi strain RE1 <br /> not producing CD (18). Concomitant expression E. coli <br />                                                                  Virulent R. equi wild type strain RE1 isolated a <br /> CD UPRT fusion protein, encoded codA::upp,           foal pyogranulomatous pneumonia September <br /> was shown enhance 5-FC cytotoxicity (19).         2007 The Netherlands. R. equi strains grown at <br /> A recent study pyrimidine salvage Streptomyces spe-        308C (200 r.p.m.) Luria-Bertani (LB) medium or <br /> cies indicated lack CD activity sensitivity     mineral acetate medium (MM-Ac). MM-Ac contained <br /> 5-FU (20). Some actinobacteria, like Rhodococcus           K2HPO4 (4.65 g/l), NaH2PO4   H2O (1.5 g/l), sodium- <br /> species, shown 5-FU sensitive (21,22).           acetate (2 g/l), NH4Cl (3 g/l), MgSO4   7H2O (1 g/l), thia- <br /> Most Nocardia species certain Mycobacterium strains,         (40 mg/l, &#239;&#172;?lter sterile) Vishniac stock solution <br /> however, 5-FU resistant, mycobacteria           (1 ml/l). Vishniac stock solution prepared follows <br /> strains highly 5-FU sensitive (21).                         [modi&#239;&#172;?ed (36)]: EDTA (10 g/l) ZnSO4   7H2O <br />    R. equi facultative intracellular pathogen causes   (4.4 g/l) dissolved distilled water (pH 8 using <br /> fatal pyogranulomatous bronchopneumonia young                 2 M KOH). Then, CaCl2 2H2O (1.47 g/l), MnCl2 7 <br /> foals. It emerging opportunistic pathogen          H2O (1 g/l), FeSO4 7H2O (1 g/l), (NH4)6 Mo7O24 4 H2O <br /> immuno-compromised humans, particularly HIV infected             (0.22 g/l), CuSO4 5H2O (0.315 g/l) CoCl2 6 H2O <br /> patients (23,24). In addition pathogenic life-style, R.   (0.32 g/l) added order pH 6 &#239;&#172;?nally <br /> equi common soil-dwelling microorganism capable          stored pH 4. For growth solid media Bacto-agar <br /> rapid growth soil manure, using plant animal          (15 g/l) added. 5-Fluorocytosine (Sigma-Aldrich <br /> sterols sole carbon energy sources (25,26).               Chemie, Zwijndrecht, The Netherlands) 5-&#239;&#172;&#8218;uorouracil <br /> Knowledge sterol metabolism R. equi extremely           (Sigma-Aldrich) stock solutions (10 mg/ml) prepared <br /> limited. So far, gene encoding proposed &#239;&#172;?rst        distilled water, dissolved heating 508C, &#239;&#172;?lter- <br /> step cholesterol degradation, cholesterol oxidase             sterilized added autoclaved media. <br /> (choE), identi&#239;&#172;?ed inactivated R. equi               For growth cholesterol, R. equi strains inocu- <br /> (27,28). A cholesterol catabolic gene cluster           lated 25 ml MM-Ac liquid medium grown 24 h <br /> identi&#239;&#172;?ed closely related Rhodococcus jostii RHA1         200 r.p.m. 308C. The pre-culture (0.5 ml) used <br /> (6,29). This cluster encodes putative cholesterol         inoculate 50 ml MM medium containing 0.5 g/l cho- <br /> uptake system, designated mce4 operon. &#226;&#8364;&#732;Mammalian                lesterol (Sigma-Aldrich) sole carbon energy source <br /> cell entry&#226;&#8364;&#8482; (mce) genes critical virulence factors    &#239;&#172;?nely dispersed sonication. Regular <br /> intracellular pathogen Mycobacterium tuberculosis (30)           turbidity (OD600nm) measurements cholesterol grown <br /> and, interestingly, mce1 mce4 strains M. tuber-           cultures possible high background of <br /> culosis H37Rv showed attenuated survival mice (31).           cholesterol suspension. Protein content culture <br /> The supAB genes mce4 operon              used measure biomass formation was <br /> encode permease subunits cholesterol uptake           determined follows. A sample (0.5 ml) culture <br /> system (6). The supAB mce4 genes shown               pelleted centrifugation (5 min 12 000g) thor- <br /> be essential growth R. jostii RHA1 cholesterol         oughly resuspended 0.1 ml Bacterial Protein Extraction <br /> (6). Pandey Sassetti (32) recently con&#239;&#172;?rmed             Reagent (B-PER, Pierce, PerBio Science Nederland B.V., <br /> cholesterol used carbon energy source             Etten-Leur, The Netherlands). Then, 0.4 ml distilled water <br />  PAGE 3 OF 10                                                           Nucleic Acids Research, 2008, Vol. 36, No. 22 e151 <br />  <br />  <br /> was added suspension vortexed incubated          Construction plasmids pSET-Pkan-codA, <br /> at room temperature 5&#226;&#8364;&#8220;10 min. An aliquot 160 ml           pSET-Pkan-codAupp pSelAct <br /> was mixed 640 ml distilled water 200 ml           The aphII promoter region ampli&#239;&#172;?ed pRESQ <br /> protein assay reagent (BioRad) added. Protein con-           (37) using PCR primers Pkan-F Pkan-E5-R <br /> tent sample determined using bovine serum             (Table 1). The obtained PCR product 367 bp was <br /> albumin (BSA) standard described                  blunt-ligated EcoRV digested pBluescript(II)KS <br /> manufacturer.                                                    (Stratagene), resulting pBs-Pkan. A SalI/NotI restric- <br />                                                                  tion released 431 bp fragment comprising aphII pro- <br />                                                                  moter cloned SalI/NotI digested <br /> Cloning, PCR chromosomal DNA isolation <br />                                                                  pORF-codA::upp (InvivoGen, San Diego, USA), yielding <br /> E. coli DH5a used host cloning procedures.        plasmid pORF-Pkan-codAupp. The Pkan-codA::upp <br /> Restriction enzymes obtained Fermentas                 cassette subsequently isolated pORF-Pkan- <br /> GmbH (St Leon-Rot, Germany). PCR performed                codAupp 2.4 kb SmaI/NheI fragment ligated <br /> a reaction mixture (25 ml) consisting Tris&#226;&#8364;&#8220;HCl (10 mM,        EcoRV/XbaI digested pSET152 resulting plasmid <br /> pH 8), 10   High-Fidelity polymerase bu&#239;&#172;&#8364;er (Fermentas),          pSET-Pkan-codAupp (Figure 2). The Pkan-codA cassette <br /> dNTPs (0.2 mM), DMSO (2%), PCR primers (10 ng/ml                 (1733 bp) ampli&#239;&#172;?ed pSET-Pkan-codAupp using <br /> each, Table 1) High-Fidelity polymerase enzyme               primers Pkan-F codA-R2 (Table 1) ligated into <br /> (1&#226;&#8364;&#8220;2 U, Fermentas). For colony PCR, cell material            EcoRV digested pSET152, resulting pSET-Pkan-codA <br /> mixed 100 ml chloroform 100 ml 10 mM              (Figure 2). Suicide plasmid pSelAct (Figure 3) con- <br /> Tris&#226;&#8364;&#8220;HCl pH 8, vortexed vigorously centrifuged               structed ligating 2.4 kb Klenow-treated EcoRI/NheI <br /> (2 min 14 000g). A sample upper water phase            fragment pORF-Pkan-codAupp SspI digested <br /> (1 ml) subsequently used template PCR. Chro-          pBs-Apra-ori dephosphoryated alkaline phospha- <br /> mosomal DNA R. equi cell cultures isolated using          tase. Plasmid pBs-Apra-ori constructed <span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> the GenElute Bacterial Genomics DNA Kit (Sigma-                  pBluescript(II)KS bla cassette removed <br /> Aldrich) according instructions manufacturer.      BspHI, followed Klenow treatment replaced <br />                                                                  apramycin-oriT cassette obtained 1.3 kb XbaI <br />                                                                  fragment (Klenow treated) pIJ773 (13). <br /> Electrotransformation R. equi strain RE1 <br />                                                                  Macrophage survival test <br /> R. equi strain RE1 cells transformed electropora- <br /> tion essentially described (27). Brie&#239;&#172;&#8218;y, cell cultures   The human monocyte cell line U937 (38) used test <br /> grown 50 ml LB 308C OD600 reached 0.8&#226;&#8364;&#8220;1.0.           survival R. equi strains. The monocytes grown <br /> The cells pelleted (20 min 4500g) washed             RPMI 1640 (Invitrogen) + NaHCO3 (1 g/l) + sodium <br /> twice 10% ice-cold glycerol. Pelleted cells            pyruvate (0.11 g/l) + glucose medium (4.5 g/l) (RPMI <br /> re-suspended 0.5&#226;&#8364;&#8220;1 ml ice-cold 10% glycerol               1640 medium), bu&#239;&#172;&#8364;ered 10 mM HEPES (Hopax <br /> 200 ml aliquots ice. MilliQ-eluted plasmid           &#239;&#172;?ne chemicals, Taiwan) supplemented penicillin <br /> DNA (5&#226;&#8364;&#8220;10 ml; GenElute Plasmid Miniprep Kit, Sigma-              (200 IU/ml), streptomycin (200 IU/ml) 10% fetal <br /> Aldrich) added 200 ml cells 2 mm gapped cuv-           bovine serum (FBS). The cells grown suspension <br /> ettes. Electroporation performed single pulse      378C 5% CO2. For macrophage survival assay, <br /> 12.5 kV/cm, 1000  <br />  25 mF. Electroporated cells          monocytes grown days described <br /> gently mixed 1 ml LB medium allowed recover          above. The culture medium replaced fresh cul- <br /> for 2 h 378C 200 r.p.m. Aliquots (200 ml)            ture medium cells activated overnight with <br /> plated LB agar medium containing apramycin                  phorbol 12-myristate 13-acetate (60 ng/ml, PMA, Sigma- <br /> (50 mg/ml; Duchefa Biochemie, Haarlem, The Nether-               Aldrich) induce di&#239;&#172;&#8364;erentiation macrophages. <br />                                                                  The di&#239;&#172;&#8364;erentiated cells spun (5 min 200g) <br /> lands). Transformants appeared 2&#226;&#8364;&#8220;3 days incuba- <br />                                                                  pellet re-suspended fresh, antibiotic free <br /> tion 308C. The transformation e&#239;&#172;&#402;ciency for <br />                                                                  RPMI 1640 medium 10% FBS. For strain to <br /> non-replicative plasmids integrating homologous <br />                                                                  tested, tube containing 10 ml cell suspension <br /> recombination   10 transformants/mg plasmid DNA. <br />                                                                  ( 106 cells/ml) inoculated R. equi, pre-grown in <br />                                                                  <span id='am-2' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> broth (Difco, Detroit, MI., USA) 378C, a <br /> 5-Fluorocytosine positive selection R. equi                   multiplicity infection (MOI) approximately 10 bac- <br />                                                                  teria macrophage. The bacteria incubated with <br /> R. equi transformants inoculated LB liquid               macrophages 1 h 378C 5% CO2. The <br /> medium (25 ml) grown overnight (20&#226;&#8364;&#8220;24 h) 308C             medium replaced 10 ml RPMI 1640 medium sup- <br /> and 200 r.p.m. 5-FC selection performed plating           plemented 10% FBS 100 mg/ml gentamycin and <br /> 100 ml aliquots dilution series (10 1 10 3 MM-Ac      incubated 1 h kill extra-cellular bacteria. <br /> medium) grown culture MM-Ac agar plates              The macrophages (with internalized R. equi) spun <br /> supplemented 5-FC (100 mg/ml). Dilution cul-         (5 min 200g) pellet re-suspended in <br /> ture prior plating crucial e&#239;&#172;&#8364;ective 5-FC selec-       40 ml RPMI 1640 medium, bu&#239;&#172;&#8364;ered 10 mM HEPES <br /> tion. 5-FC resistant colonies appeared 2&#226;&#8364;&#8220;3 days         supplemented 10% FBS gentamycin <br /> incubation 308C.                                              (10 mg/ml). This suspension divided culture <br />  e151 Nucleic Acids Research, 2008, Vol. 36, No. 22                                                                   PAGE 4 OF 10 <br />  <br />  <br /> bottles (10 ml each) incubated 378C 5% CO2. <br /> After 4, 28, 52 76 h macrophages (one culture <br /> bottle strain) spun (5 min 200g) and <br /> the pellet washed twice 1 ml antibiotic free RPMI <br /> 1640 medium. Finally pellet lysed 1% <br /> Triton X-100 (Sigma-Aldrich) 0.01 M phosphate buf- <br /> fered saline, followed live count determination (plate <br /> counting). <br />  <br /> Culture media growth conditions 5-FC selection <br /> in actinobacteria <br /> Actinobacterial strains (Table 2) grown late <br /> exponential phase shaken liquid cultures complex <br /> medium 308C, Amycolatopsis methanolica <br /> and Mycobacterium smegmatis grown at <br /> 378C. Tryptic Soy Broth (TSB) used Corynebacter- <br /> ium glutamicum, Arthrobacter globiformis, Amycolatopsis <br /> orientalis, A. methanolica M. smegmatis. TSB supple- <br /> mented 2.5% NaCl used Salinospora tropica, <br /> and YEME:TSB (1:1) (39) Streptomyces strains <br /> and Saccharopolyspora erythrea. LB used R. equi <br /> and Rhodococcus rhodochrous LBP (3) used           Figure 1. Introduction E. coli codA::upp cassette, encoding CD and <br /> R. erythropolis R. jostii. Minimal regeneration medium     UPRT, confers 5-FC sensitivity R. equi strain RE1. Panels wild <br />                                                                type strain RE1, recombinant strain RE1 containing pSET152, recom- <br /> (MRM) consisted K2SO4 (0.25 g/l), (NH4)2SO4 (2 g/l),        binant strain RE1 containing plasmid pSET-codA::upp streaked on <br /> MgCl2 6H20 (0.6 g/l), 2-[(hydroxyl-1, 1-bis (hydroxy-          mineral acetate agar (MM-Ac) containing (A) addition, (B) 5-FC <br /> methyl)ethyl)amino] ethanesulfonic acid; TES (5.73 g/l)        (100 mg/ml) (C) 5-FU (50 mg/ml) (D) LB agar containing <br /> and trace elements (39). Sucrose (10.3% w/v) L-proline     5-FC (100 mg/ml). <br /> (0.3% w/v) added MRM medium Streptomyces <br /> strains, omitted strains. Glucose (20 mM) <br /> was used carbon energy source, all <br /> Rhodococcus strains grown presence of <br /> acetate (2 g/l). Autoclaved MRM supplemented with <br /> KH2PO4 (0.1 g/l), CaCl2 2H20 (3 g/l) NaOH (0.28 g/l). <br /> For agar plates 2% granulated agar added. Filter- <br /> sterilized thiamine (40 mg/ml) added autoclaved <br /> MRM medium growth Rhodococcus strains. 5-FC <br /> and 5-FU freshly prepared 10 mg/ml stocks in <br /> distilled water, dissolved heating 508C added <br /> to autoclaved medium. <br />                                                                Figure 2. The pSET152 derived integrative plasmids used introduce <br /> RESULTS AND DISCUSSION                                         (A) codA (B) codA::upp cassette R. equi RE1, expressed <br />                                                                control aphII kanamycin resistance cassette promoter <br /> CD-UPRT mediated 5-fluorocytosine sensitivity                  (PaphII). The apramycin resistance cassette (aac(3)IV), the <br /> of R. equi strain RE1                                          Streptomyces PhiC31 integrase gene (int) RP4 origin transfer <br />                                                                (oriT) indicated. <br /> An obvious prerequisite applicability 5-FC <br /> based conditionally lethal positive selection nat- <br /> ural resistance R. equi strain RE1 5-FC. To test <br /> this, R. equi RE1 streaked acetate mineral <br /> (MM-Ac) agar plates supplemented 5-FC 5-FU <br /> and incubated 3 days 308C. Examination the <br /> plates revealed R. equi resistant high concen- <br /> trations 5-FC (100 mg/ml), highly sensitive a <br /> lower concentration 5-FU (50 mg/ml), indicating the <br /> feasibility developing 5-FC based positive selection <br /> system R. equi (Figure 1). Next, examined whether <br /> expression E. coli genes codA upp R. equi <br /> would confer sensitivity 5-FC act a <br /> suicide marker. The integrative E. coli-Streptomyces shut-     Figure 3. Schematic representation non-replicative plasmid <br /> tle vector pSET152 provided stable convenient vehi-      pSelAct used generate unmarked in-frame gene deletions R. equi <br /> cle introduce codA functional codA::upp fusion    RE1. Restriction sites indicated unique. <br />  PAGE 5 OF 10                                                         Nucleic Acids Research, 2008, Vol. 36, No. 22 e151 <br />  <br />  <br /> the R. equi genome (40,41). The aphII promoter region       SupA 54 amino acids SupB, separated by <br /> the kanamycin resistance cassette (Pkan) used           6 bp SmaI restriction site. The &#239;&#172;?rst supA <br /> drive expression codA codA::upp, respectively. The       supB, respectively, left intact ensure proper <br /> two resulting integrative plasmids, pSET-Pkan-codA             expression downstream genes mce4 operon. <br /> and pSET-Pkan-codAupp (Figure 2), mobilized sepa-         Introduction plasmid pSelAct- supAB R. equi <br /> rately electroporation R. equi RE1. Transformants        RE1, selecting apramycin resistance, resulted 29 <br /> harboring pSET-Pkan-codA pSET-Pkan-                  transformants selected grown non- <br /> codAupp streaked MM-Ac agar plates                   selectively overnight 20 h LB medium (i.e. lacking <br /> (Figure 1A), MM-Ac agar plates supplemented               apramycin 5-FC). Aliquots (100 ml) overnight <br /> 100 mg/ml 5-FC (Figure 1B), MM-Ac agar plates sup-          cultures plated 10 1 10 3 dilutions MM- <br /> plemented 50 mg/ml 5-FU (Figure 1C) incubated         Ac agar plates supplemented 100 mg/ml 5-FC and <br /> for 3 days 308C. The presence codA::upp cassette     incubated 3 days 308C. Con&#239;&#172;&#8218;uent growth 5-FC <br /> rendered R. equi sensitive 5-FC, expression      selection plates obtained transformants. <br /> codA did result 5-FC sensitivity (Figure 1B).     Conceivably, codA::upp suicide cassette inac- <br /> Growth R. equi strain carrying codA::upp cassette   tivated transformant spontaneous <span id='am-241' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-242' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-243' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-250' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-256' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-257' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-264' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-270' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-273' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-279' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-292' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-295' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-302' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-308' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-318' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span>mutation</span> or <br /> under nonselective conditions similar wild      transposon insertion, reported occur during <br /> type (Figure 1A), indicating expression condition-     sacB counter-selection (3,42). Typically, &gt;102 colonies <br /> ally lethal detrimental cells 5-FC          grew 10 2 plate 50 colonies replica <br /> omitted medium. Importantly, 5-FC selection        picked LB agar apramycin select <br /> was observed complex agar media (LB) used             5-FCR/ApraS colonies. The frequency 5-FCR/ApraS <br /> (Figure 1D). This repression pyrimi-      colonies transformants varied <br /> dine salvage pathway a&#239;&#172;&#8364;ecting 5-FC uptake.                     70&#226;&#8364;&#8220;90%, indicating suicide cassette had <br />    To enable generation unmarked in-frame gene          inactivated 10&#226;&#8364;&#8220;30% 5-FCR/ApraS colonies. <br /> deletions R. equi using CD-UPRT based 5-FC condi-           For transformant, eighteen 5-FCR/ApraS colonies <br /> tional lethality, developed non-replicative sui-     checked colony PCR presence the <br /> cide vector designated pSelAct (Figure 3; Materials        mutant  supAB genotype using PCR oligonucleo- <br /> and Methods section). Vector pSelAct small (5.6 kb)       tides 6 7 amplifying supAB genes (Table 1, <br /> conjugative plasmid based pBlueScript(II)KS, harbor-        Figure 4). Two eighteen FCR/ApraS colonies gave <br /> ing aac(3)IV cassette apramycin resistance    PCR product expected size (231 bp) were <br /> be used E. coli R. equi. The vector contains     selected characterization. Genomic DNA was <br /> MCS unique restriction sites cloning          isolated  supAB mutants subjected to <br /> &#239;&#172;&#8218;anking regions gene using lacZ depen-      PCR analysis supAB locus up- down- <br /> dent blue-white screening E. coli.                          stream &#239;&#172;&#8218;anking regions supAB (Figure 5, Table 1 for <br />                                                                oligonucleotides used). This con&#239;&#172;?rmed presence a <br /> Unmarked in-frame gene deletion supAB R. equi            genuine supAB gene deletion cases revealed no <br />                                                                aberrant genomic rearrangements supAB locus <br /> A BLAST search supA (ro04696), supB (ro04697)         (Figure 5). One mutant strain chosen, designated R. <br /> and mce4 (ro04698-ro04703) genes R. jostii RHA1,            equi RE1 supAB, used further <br /> predicted encode cholesterol uptake machinery           characterization. <br /> (6), revealed genes apparent operonic <br /> organization highly conserved R. equi 103S <br />                                                                Phenotypical analysis functional complementation <br /> genome (Figure 4; http://www.sanger.ac.uk/Projects/ <br />                                                                R. equi RE1&quot;supAB mutant <br /> R_equi/). Highest amino acid identities observed <br /> for SupA (94%) SupB (89%), mce4                R. equi RE1 wild type RE1 supAB mutant strain <br /> encoded proteins Mce4A-Mce4F showed lower identities           grown MM-Ac liquid medium used inocu- <br /> (57&#226;&#8364;&#8220;71%) (Figure 4).                                           late MM-cholesterol liquid medium. The RE1 supAB <br />    The supA supB genes &#239;&#172;?rst genes       mutant completely blocked growth cholesterol <br /> mce4 operon, encode cholesterol permease           sole carbon energy source (Figure 6). Growth on <br /> subunits (Figure 4). To demonstrate speci&#239;&#172;?c role          acetate steroid substrate 4-androstene-3,17-dione <br /> supAB cholesterol catabolism, aimed selec-     (AD) una&#239;&#172;&#8364;ected comparable wild type <br /> tive inactivation, a&#239;&#172;&#8364;ecting expression          strain (data shown). This indicated supAB are <br /> downstream genes. This prompted develop              essential cholesterol catabolism, probably acting as <br /> unmarked in-frame gene deletion R. equi             permease subunits cholesterol ABC transporter. <br /> based counter-selection 5-FC. The upstream         These results fully consistent <span id='am-163' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-164' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span>phenotype</span> the <br /> downstream &#239;&#172;&#8218;anking regions ( 1.5 kb) supAB genes        supAB mutant strain RHA1 (6). <br /> were ampli&#239;&#172;?ed PCR ligated pSelAct generat-            To ensure growth de&#239;&#172;?ciency the <br /> ing pSelAct- supAB (Figure 4; Table 1 primers). This       RE1 supAB mutant solely deletion the <br /> construct used generate  supAB mutant,             supAB genes, 3163-bp DNA fragment carrying <br /> reducing wild type supA (765 nt) supB (852 nt)         wild-type supAB genes introduced electropora- <br /> genes, encompassing 1637 bp, single in-frame open-        tion RE1 supAB mutant. The DNA fragment <br /> reading-frame 231 bp, encoding &#239;&#172;?rst 20 amino acid       obtained PCR strain RE1 wild type DNA <br />  e151 Nucleic Acids Research, 2008, Vol. 36, No. 22                                                                               PAGE 6 OF 10 <br />  <br />  <br />  <br />  <br /> Figure 4. (A) Schematic overview molecular organization mce4 gene clusters R. jostii RHA1 R. equi 103S. Percentages indicate <br /> amino acid sequence identities Mce4 proteins R. jostii RHA1 R. equi RE1. (B) Construction mutagenic plasmid pSelAct- <br />  supAB used generate unmarked supAB gene deletion mutant R. equi RE1. Small black arrows numbers indicate PCR oligonu- <br /> cleotides used study (Table 1) site annealing. (C) Molecular organization mce4 locus R. equi RE1 following <br /> integration pSelAct- supAB single homologous recombination, second homologous recombinant event 5-FC counter-selection, <br /> resulting supAB gene deletion. <br />  PAGE 7 OF 10                                                                        Nucleic Acids Research, 2008, Vol. 36, No. 22 e151 <br />  <br />  <br /> Table 1. Oligonucleotides used study <br />  <br /> No   PCR amplicon                                             Size (bp)         Oligonucleotide sequence (nr.) <br />  <br /> 1    Upstream region supAB (deletion construct)               1592              supABequiUP-F ATCGCGAGGTCAGCTTGGAG (1) <br />                                                                                 supABequiUP-R CCCGGGCGCCGACATCGCGAAGAATC (2) <br /> 2    Downstream region supAB (deletion construct)             1470              supABequiDOWN-F CCCGGGCTCATCCACACCTACTACGG (3) <br />                                                                                 supABequiDOWN-R ACTAGTGAGCTGCTGAATCTGAACTGG (4) <br /> 3    Upstream region supAB (con&#239;&#172;?rmation deletion mutant)      1786 (wt: 3192)   supABequiContrUP-F CGGGAGTGCGTAGATGAGTGCA (5) <br />                                                                                 supABequiContr-R TTATCCCGAAAGGTTGAAGTTG (6) <br /> 4    Downstream region supAB (con&#239;&#172;?rmation deletion mutant)    1570 (wt: 2976)   supABequiContr-F GTGGTCGACCTCCTCGAGGTAC (7) <br />                                                                                 supABequiContrDOWN-R GGACTTGAGCCCGGAGGCATCG (8) <br /> 5    supAB genes (con&#239;&#172;?rmation deletion mutant)                 231 (wt: 1637)   supABequiContr-F GTGGTCGACCTCCTCGAGGTAC (7) <br />                                                                                 supABequiContr-R TTATCCCGAAAGGTTGAAGTTG (6) <br /> 6    supAB genes (complementation  supAB mutant)              3163              supABequiUP-F ATCGCGAGGTCAGCTTGGAG (1) <br />                                                                                 supABequiContr-R TTATCCCGAAAGGTTGAAGTTG (6) <br /> 7    aphII promoter region                                     367              Pkan-F AGCTTCACGCTGCCGCAAGCACT <br />                                                                                 Pkan-E5-R GATATCATGCGAAACGATCCTCATCCTG <br /> 8    Pkan-codA cassette                                       1733              Pkan-F AGCTTCACGCTGCCGCAAGCACT <br />                                                                                 codA-R2 GTCAACGTTTGTAATCGATGGCTTCT <br /> 9    vapA                                                      408              vapA-F GCAGCAGTGCGATTCTCAATAG (9) <br />                                                                                 vapA-R TAACTCCACCGGACTGGATATG (10) <br />  <br /> Primer numbering used text shown brackets following nucleotide sequence. <br />  <br />  <br />  <br />  <br /> Figure 5. PCR analysis wild type R. equi strain RE1 (wt)  supAB gene deletion mutants R. equi strain RE1 (1 2). Panels show <br /> ampli&#239;&#172;?cation (I) supAB locus [6 + 7], (II) upstream region supAB [5 + 6], (III) downstream region supAB [7 + 8], (IV) vapA <br /> virulence gene [9 + 10]. The oligonucleotides used mentioned brackets described Table 1 and/or Figure 4. <br />  <br />  <br /> using oligonucleotides 1 6 (Table 1, Figure 4) and <br /> cloned integrative plasmid SET152, resulting in <br /> pSET-supAB. The cholesterol growth negative phenotype <br /> of RE1 supAB mutant harboring pSET-supAB was <br /> fully complemented, restoring growth cholesterol to <br /> levels comparable wild type (Figure 6). We conclude <br /> that supAB deletion did exert polar e&#239;&#172;&#8364;ects on <br /> the expression genes mce4 operon. <br />    These data developed novel, simple <br /> and e&#239;&#172;&#402;cient method generate unmarked in-frame gene <br /> deletions R. equi, based 5-FC lethality pre- <br /> sence E. coli codA::upp fusion gene. <br />  <br /> Deletion supAB does affect R. equi RE1 survival <br /> in macrophages <br /> Intracellular survival proliferation R. equi <br /> RE1 supAB mutant human monocyte cell line <br /> U937 compared wild type strain RE1 <br /> (Figure 7). The avirulent, plasmid free strain R. equi <br /> 103 (43) included negative control macro-                      Figure 6. Growth curves mineral medium supplemented choles- <br /> phage survival (Figure 7). The results revealed                    terol (0.5 g/l) R. equi RE1 wild-type (&#239;&#172;?lled diamond), RE1  supAB <br /> RE1 supAB mutant able survive proliferate                      mutant strain (&#239;&#172;?lled square), RE1  supAB + pSET-supAB complemen- <br />                                                                             ted strain (&#239;&#172;?lled triangle), RE1  supAB + pSET152 control strain <br /> macrophages comparable wild-type parent strain                       (x). Protein content (mg/l) culture used measure for <br /> RE1. By contrast, avirulent strain 103 failed pro-                   biomass formation. The data represent averages indepen- <br /> liferate, resulting reduced numbers intracellular                     dent experiments. <br />  e151 Nucleic Acids Research, 2008, Vol. 36, No. 22                                                                       PAGE 8 OF 10 <br />  <br />  <br /> bacteria time (Figure 7). These results indicate               Actinobacteria generally 5-FC resistant <br /> cholesterol metabolism essential macrophage                 The CD-UPRT based 5-FC selection a <br /> survival R. equi RE1 suggest cholesterol                   useful asset generation unmarked in-frame gene <br /> metabolism important virulence R. equi                   deletions actinobacteria. To examine whether <br /> RE1 vivo. These observations consistent                CD-UPRT selection potentially e&#239;&#172;&#8364;ective counter- <br /> &#239;&#172;?nding ChoE important virulence                  selectable marker family microorganisms, we <br /> R. equi (28).                                                          tested sensitivity actinobacteria towards <br />                                                                        5-FC 5-FU. A wide selection actinobacterial strains <br />                                                                        grown complex medium late exponential/ <br />                                                                        early stationary phase plated 10-fold dilutions <br />                                                                        mineral selection media containing increasing con- <br />                                                                        centrations 5-FC 5-FU (Table 2). The tested strains <br />                                                                        generally resistant high concentrations 5-FC <br />                                                                        (100 mg/ml), highly sensitive lower concentrations <br />                                                                        5-FU (20&#226;&#8364;&#8220;50 mg/ml), indicating feasibility devel- <br />                                                                        oping 5-FC based positive selection other <br />                                                                        actinobacteria. <br />  <br />                                                                        ACKNOWLEDGEMENTS <br />                                                                        The sequence data R. equi 103S produced the <br />                                                                        R. equi Sequencing Group Sanger Institute were <br />                                                                        obtained ftp://ftp.sanger.ac.uk/pub/pathogens/re. <br />                                                                        We thank J.A. Salas (Oviedo, Spain), E.M.H. <br />                                                                        Wellington (Warwick, UK) M. Fukuda (Nagaoka, <br />                                                                        Japan) providing strains. <br />  <br /> Figure 7. Survival proliferation R. equi strains human <br /> monocyte cell line U937. Macrophage cell suspensions infected <br />                                                                        FUNDING <br /> with wild type virulent strain R. equi RE1 (&#239;&#172;?lled diamond), mutant     Intervet Schering-Plough Animal Health, Boxmeer, <br /> strain R. equi RE1 supAB (&#239;&#172;?lled square) non-virulent (control) <br /> strain R. equi 103 (&#239;&#172;?lled triangle). Following 1-h incubation to <br />                                                                        The Netherlands. Funding open access charge: <br /> allow phagocytosis, cells washed treated gentamycin   Groningen Biomolecular Sciences Biotechnology <br /> kill remaining extra-cellular bacteria. The numbers intracellular   Institute (GBB), University Groningen, The <br /> bacteria determined plate counts following macrophage lysis.   Netherlands. <br /> The data represent averages independent experiments. <br /> Plate counts carried duplicate.                            Con&#239;&#172;&#8218;ict statement. None declared. <br />  <br />  <br /> Table 2. Growth actinobacteria MRM mineral agar media supplemented di&#239;&#172;&#8364;erent concentrations 5-FC 5-FU <br />  <br /> Strain                                                  5-FC (mg/ml)                 5-FU (mg/ml)          Reference/origin <br />  <br />                                              0         20       50     100      20       50         100 <br />  <br /> Amycolatopsis methanolica NCIB11946          ++        +                        &#226;&#8364;&#8220;        &#226;&#8364;&#8220;          &#226;&#8364;&#8220;      (44) <br /> Amycolatopsis orientalis ATCC 19795          ++        ++       +      +                                   ATCC <br /> Arthrobacter globiformis DSM 20124           ++        ++       +      +        &#226;&#8364;&#8220;        &#226;&#8364;&#8220;          &#226;&#8364;&#8220;      DSMZ <br /> Corynebacterium glutamicum ATCC 13032        ++        +                                 &#226;&#8364;&#8220;          &#226;&#8364;&#8220;      ATCC <br /> Mycobacterium smegmatis mc2155               ++        +                        &#226;&#8364;&#8220;        &#226;&#8364;&#8220;          &#226;&#8364;&#8220;      (45) <br /> Rhodococcus rhodochrous DSM 43269            ++        ++       ++     ++       +        &#226;&#8364;&#8220;          &#226;&#8364;&#8220;      DSMZ <br /> Rhodococcus erythropolis SQ1                 ++        ++       ++     ++       &#226;&#8364;&#8220;        &#226;&#8364;&#8220;          &#226;&#8364;&#8220;      (46) <br /> Rhodococcus jostii RHA1                      ++        ++       ++     ++       +                   &#226;&#8364;&#8220;      M. Fukuda, Nagaoka (Japan) <br /> Rhodococcus equi RE1                         ++        ++       ++     ++       +        &#226;&#8364;&#8220;          &#226;&#8364;&#8220;      Schering-Plough, Netherlands <br /> Salinospora tropica                          ++        ++       ++     ++                                  DSMZ <br /> Saccharopolyspora erythraea                  ++        ++       ++     ++       &#226;&#8364;&#8220;        &#226;&#8364;&#8220;          &#226;&#8364;&#8220;      DSMZ <br /> Streptomyces albus J1074                     ++        ++       ++     ++       &#226;&#8364;&#8220;        &#226;&#8364;&#8220;          &#226;&#8364;&#8220;      (47) <br /> Streptomyces avermitilis MA-4680             ++        ++       ++     ++       ++       ++         ++     ATCC (ATCC 31267) <br /> Streptomyces coelicolor M145                 ++        ++       ++     ++       &#226;&#8364;&#8220;        &#226;&#8364;&#8220;          &#226;&#8364;&#8220;      (39) <br /> Streptomyces griseus DSM40236                ++        ++       ++     ++       &#226;&#8364;&#8220;        &#226;&#8364;&#8220;          &#226;&#8364;&#8220;      DSMZ <br /> Streptomyces lividans TK23                   ++        ++       ++     ++       &#226;&#8364;&#8220;        &#226;&#8364;&#8220;          &#226;&#8364;&#8220;      (39) <br /> Streptomyces scabies ISP5078                 ++        ++       ++     +        &#226;&#8364;&#8220;        &#226;&#8364;&#8220;          &#226;&#8364;&#8220;      Wellington, Warwick (UK) <br /> Streptomyces tendae Tu&#194;&#168; 901/8c               ++        ++       ++     +        +        &#226;&#8364;&#8220;          &#226;&#8364;&#8220;      (48) <br />  <br /> Symbols indicate growth (++), moderate growth (+), slight/minor growth ( ) growth (&#226;&#8364;&#8220;). <br /> ATCC, American Type Culture Collection; DSMZ, Deutsche Sammlung von Mikroorganismen und Zellkulturen. <br />  PAGE 9 OF 10                                                                      Nucleic Acids Research, 2008, Vol. 36, No. 22 e151 <br />  <br />  <br /> REFERENCES                                                                     cytotoxicity 5-&#239;&#172;&#8218;uorocytosine. . FEMS Microbiol. Lett., <br />                                                                                167, 41&#226;&#8364;&#8220;49. <br />  1. Ried,J.L. Collmer,A. (1987) An nptI-sacB-sacR cartridge        20. Hughes,L.E., Beck,D.A. O&#226;&#8364;&#8482;Donovan,G.A. (2005) <br />     constructing directed, unmarked mutations gram-negative                 Pathways pyrimidine salvage Streptomyces. Curr. Microbiol., <br />     bacteria marker exchange-eviction mutagenesis. Gene,                    50, 8&#226;&#8364;&#8220;10. <br />     57, 239&#226;&#8364;&#8220;246.                                                           21. Tsukamura,M. (1981) Di&#239;&#172;&#8364;erentiation genera <br />  2. Reyrat,J.M., Pelicic,V., Gicquel,B. Rappuoli,R. (1998)                 Mycobacterium, Rhodococcus Nocardia susceptibility to <br />     Counterselectable markers: untapped tools bacterial genetics           5-&#239;&#172;&#8218;uorouracil. J. Gen. Microbiol., 125, 205&#226;&#8364;&#8220;208. <br />     pathogenesis. Infect. Immun., 66, 4011&#226;&#8364;&#8220;4017.                       22. Tsukamura,M., Komatsuzaki,C., Sakai,R., Kaneda,K., Kudo,T. <br />  3. Van der Geize,R., Hessels,G.I., Van Gerwen,R., Van der Meijden,P.          Seino,A. (1988) Mesenteric lymphadenitis swine caused by <br />     Dijkhuizen,L. (2001) Unmarked gene deletion mutagenesis             Rhodococcus sputi. J. Clin. Microbiol., 26, 155&#226;&#8364;&#8220;157. <br />     kstD, encoding 3-ketosteroid  1-dehydrogenase, Rhodococcus          23. Prescott,J.F. (1991) Rhodococcus equi: animal human <br />     erythropolis SQ1 using sacB counter-selectable marker. FEMS             pathogen. Clin. Microbiol. Rev., 4, 20&#226;&#8364;&#8220;34. <br />     Microbiol. Lett., 205, 197&#226;&#8364;&#8220;202.                                        24. Meijer,W.G. Prescott,J.F. (2004) Rhodococcus equi. Vet. Res., <br />  4. Nga,D.P., Altenbuchner,J. Heiss,G.S. (2004) NpdR, repressor          35, 383&#226;&#8364;&#8220;396. <br />     involved 2,4,6-trinitrophenol degradation Rhodococcus opacus     25. Ahmad,S., Roy,P.K., Basu,S.K. Johri,B.N. (1993) Cholesterol <br />     HL PM-1. J. Bacteriol., 186, 98&#226;&#8364;&#8220;103.                                       side-chain cleavage immobilized cells Rhodococcus equi <br />  5. Sharp,J.O., Sales,C.M., LeBlanc,J.C., Liu,J., Wood,T.K.,                   DSM89-133. Indian J. Exp. Biol., 31, 319&#226;&#8364;&#8220;322. <br />     Eltis,L.D., Mohn,W.W. Alvarez-Cohen,L. (2007) An inducible         26. Murohisa,T. Iida,M. (1993) Some new intermediates micro- <br />     propane monooxygenase responsible N-nitrosodimethylamine            bial chain degradation b-sitosterol. J. Ferment. Bioeng., <br />     degradation Rhodococcus sp. strain RHAI. Appl. Environ.                 76, 174&#226;&#8364;&#8220;177. <br />     Microbiol., 73, 6930&#226;&#8364;&#8220;6938.                                             27. Navas,J., Gonza&#194;&#180;lez-Zorn,B., Ladro&#194;&#180;n,N., Garrido,P. Va&#194;&#180;zquez- <br />  6. Van der Geize,R., Yam,K., Heuser,T., Wilbrink,M.H., Hara,H.,               Boland,J.A. (2001) Identi&#239;&#172;?cation mutagenesis allelic <br />     Anderton,M.C., Sim,E., Dijkhuizen,L., Davies,J.E., Mohn,W.W.               exchange choE, encoding cholesterol oxidase the <br />     et al. (2007) A gene cluster encoding cholesterol catabolism          intracellular pathogen Rhodococcus equi. J. Bacteriol., <br />     soil actinomycete provides insight Mycobacterium tuberculosis         183, 4796&#226;&#8364;&#8220;4805. <br />     survival macrophages. Proc. Natl Acad. Sci. USA,                    28. Pei,Y., Dupont,C., Sydor,T., Haas,A. Prescott,J.F. (2006) <br />     104, 1947&#226;&#8364;&#8220;1952.                                                            Cholesterol oxidase (ChoE) important virulence of <br />  7. Ja&#194;&#168;ger,W., Scha&#194;&#168;fer,A., Pu&#194;&#168;hler,A., Labes,G. Wohlleben,W. (1992)       Rhodococcus equi. Vet. Microbiol., 118, 240&#226;&#8364;&#8220;246. <br />     Expression Bacillus subtilis sacB gene leads sucrose         29. McLeod,M.P., Warren,R.L., Hsiao,W.W., Araki,N., Myhre,M., <br />     sensitivity gram-positive bacterium Corynebacterium                 Fernandes,C., Miyazawa,D., Wong,W., Lillquist,A.L., Wang,D. <br />     glutamicum Streptomyces lividans. J. Bacteriol.,                et al. (2006) The complete genome Rhodococcus sp. RHA1 <br />     174, 5462&#226;&#8364;&#8220;5465.                                                            provides insights catabolic powerhouse. Proc. Natl Acad. Sci. <br />  8. Zhang,X.Z., Yan,X., Cui,Z.L., Hong,Q. Li,S.P. (2006) mazF,           USA, 103, 15582&#226;&#8364;&#8220;15587. <br />     novel counter-selectable marker unmarked chromosomal               30. Rengarajan,J., Bloom,B.R. Rubin,E.J. (2005) Genome-wide <br />     manipulation Bacillus subtilis. Nucleic Acids Res., 34, e71.            requirements Mycobacterium tuberculosis adaptation and <br />  9. Van Wezel,G.P. Bibb,M.J. (1996) A novel plasmid vector            survival macrophages. Proc. Natl Acad. Sci. USA, <br />     uses glucose kinase gene (glkA) positive selection          102, 8327&#226;&#8364;&#8220;8332. <br />     stable gene disruptants Streptomyces. Gene, 182, 229&#226;&#8364;&#8220;230.           31. Joshi,S.M., Pandey,A.K., Capite,N., Fortune,S.M., Rubin,E.J. and <br /> 10. Hosted,T.J. Baltz,R.H. (1997) Use rpsL dominance                Sassetti,C.M. (2006) Characterization mycobacterial virulence <br />     selection gene replacement Streptomyces roseosporus. J.             genes genetic interaction mapping. Proc. Natl Acad. Sci. <br />     Bacteriol., 179, 180&#226;&#8364;&#8220;186.                                                  USA, 103, 11760&#226;&#8364;&#8220;11765. <br /> 11. Knipfer,N., Seth,A. Shrader,T.E. (1997) Unmarked gene inte-        32. Pandey,A.K. Sassetti,C.M. (2008) Mycobacterial persistence <br />     gration chromosome Mycobacterium smegmatis                 requires utilization host cholesterol. Proc. Natl Acad. Sci. <br />     precise replacement pyrF gene. Plasmid, 37, 129&#226;&#8364;&#8220;140.                USA, 105, 4376&#226;&#8364;&#8220;4380. <br /> 12. Fabret,C., Ehrlich,S.D. Noirot,P. (2002) A new mutation            33. Jain,S., Bloom,B.R. Hondalus,M.K. (2003) Deletion of <br />     delivery genome-scale approaches Bacillus subtilis.          vapA encoding Virulence Associated Protein A attenuates the <br />     Mol. Microbiol., 46, 25&#226;&#8364;&#8220;36.                                                intracellular actinomycete Rhodococcus equi. Mol. Microbiol., <br /> 13. Gust,B., Challis,G.L., Fowler,K., Kieser,T. Chater,K.F. (2003)         50, 115&#226;&#8364;&#8220;128. <br />     PCR-targeted Streptomyces gene replacement identi&#239;&#172;?es protein         34. Pei,Y., Nicholson,V., Woods,K. Prescott,J.F. (2007) <br />     domain needed biosynthesis sesquiterpene soil odor              Immunization intrabronchial administration 1-week-old foals <br />     geosmin. Proc. Natl Acad. Sci. USA, 100, 1541&#226;&#8364;&#8220;1546.                        unmarked double gene disruption strain Rhodococcus equi <br /> 14. Fedoryshyn,M., Petzke,L., Welle,E., Bechthold,A.                       strain 103+. Vet. Microbiol., 125, 100&#226;&#8364;&#8220;110. <br />     Luzhetskyy,A. (2008) Marker removal actinomycetes genome          35. Pei,Y., Parreira,V., Nicholson,V.M. Prescott,J.F. (2007) <br />     using Flp recombinase. Gene, 419, 43&#226;&#8364;&#8220;47.                                   Mutation virulence assessment chromosomal genes of <br /> 15. Andersen,L., Kilstrup,M. Neuhard,J. (1989) Pyrimidine, purine          Rhodococcus equi 103. Can. J. Vet. Res., 71, 1&#226;&#8364;&#8220;7. <br />     nitrogen control cytosine deaminase synthesis Escherichia    36. Vishniac,W. Santer,M. (1957) The thiobacilli. Bacteriol. Rev., <br />     coli K 12. Involvement glnLG purR genes regu-            21, 195&#226;&#8364;&#8220;213. <br />     lation codA expression. Arch. Microbiol., 152, 115&#226;&#8364;&#8220;118.             37. Van der Geize,R., Hessels,G.I., Van Gerwen,R., Van der Meijden,P. <br /> 16. Danielsen,S., Kilstrup,M., Barilla,K., Jochimsen,B. Neuhard,J.         Dijkhuizen,L. (2002) Molecular functional characterization <br />     (1992) Characterization Escherichia coli codBA operon               kshA kshB, encoding components 3-ketosteroid <br />     encoding cytosine permease cytosine deaminase. Mol.                    9a-hydroxylase, class IA monooxygenase, Rhodococcus <br />     Microbiol., 6, 1335&#226;&#8364;&#8220;1344.                                                  erythropolis SQ1. Mol. Microbiol., 45, 1007&#226;&#8364;&#8220;1018. <br /> 17. Heidelberger,C., Danenberg,P.V. Moran,R.G. (1983)                  38. Sundstrom,C. Nilsson,K. (1976) Establishment character- <br />     Fluorinated pyrimidines nucleosides. Adv. Enzymol. Relat.        ization human histiocytic lymphoma cell line (U937). Int. J. <br />     Areas Mol. Biol., 54, 58&#226;&#8364;&#8220;119.                                              Cancer, 17, 565&#226;&#8364;&#8220;577. <br /> 18. Mullen,C., Kilstrup,M. Blaese,R.M. (1992) Transfer          39. Kieser,T., Bibb,M.J., Buttner,M.J., Chater,K.F. and <br />     bacterial gene cytosine deaminase mammalian cells confers           Hopwood,D.A. (2000) Practical Streptomyces Genetics, John Innes <br />     lethal sensitivity 5-&#239;&#172;&#8218;uorocytosine: A negative selection system.        Foundation, Norwich, UK. <br />     Proc. Natl Acad. Sci. USA, 89, 33&#226;&#8364;&#8220;37.                                  40. Bierman,M., Logan,R., O&#226;&#8364;&#8482;Brien,K., Seno,E.T., Rao,R.N. and <br /> 19. Tiraby,M., Cazaux,C., Baron,M., Drocourt,D., Reynes,J.P.               Schoner,B.E. (1992) Plasmid cloning vectors conjugal <br />     Tiraby,G. (1998) Concomitant expression E. coli cytosine                transfer DNA Escherichia coli Streptomyces spp. Gene, <br />     deaminase uracil phosphoribosyltransferase improves                116, 43&#226;&#8364;&#8220;4 <br /> </body></html>