<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Combined analysis reveals a core set of cycling genes <br />  <br />  <br />  <br />  <br />  <br />  <br />                                                                                                                                                              deposited research <br />                  <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />  <br />                  Background: Global transcript levels throughout the cell cycle have been characterized using <br />                  microarrays in several species. Early analysis of these experiments focused on individual species. <br />                  More recently, a number of studies have concluded that a surprisingly small number of genes <br />                  conserved in two or more species are periodically transcribed in these species. Combining and <br />                  comparing data <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> multiple species is challenging because of noise in expression data, the <br />                  different synchronization and scoring methods used, and the need <span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span> determine an accurate set of <br />  <br />  <br />  <br />  <br />                                                                                                                                                              refereed research <br />                  homologs. <br />  <br />                  Results: To solve these problems, we developed and applied a new algorithm to analyze <br />                  expression data from multiple species simultaneously. Unlike previous studies, we find that more <br />                  than 20% of cycling genes in budding yeast have cycling homologs in fission yeast and 5% to 7% of <br />                  cycling genes in each of four species have cycling homologs in all other species. These conserved <br />                  cycling genes display much stronger cell cycle characteristics in several complementary high <br />                  throughput datasets. <br />  <br />                  Essentiality analysis for yeast and human genes confirms these findings. Motif analysis indicates                                           interactions <br />                  conservation in the corresponding regulatory mechanisms. Gene Ontology analysis and analysis of <br />                  the genes in the conserved sets sheds light on the evolution of specific subfunctions within the cell <br />                  cycle. <br />  <br />                  Conclusion: Our results indicate that the conservation in cyclic expression patterns is much <br />                  greater than was previously thought. These genes are highly enriched for most cell cycle categories, <br />                                                                                                                                                              information <br />  <br />  <br />  <br />  <br />                  and a large percentage of them are essential, supporting our claim that cross-species analysis can <br />                  identify the core set of cycling genes. <br />  <br />  <br />  <br />  <br />                                                               Genome Biology 2007, 8:R146 <br />  R146.2 Genome Biology 2007,      Volume 8, Issue 7, Article R146      Lu et al.                                          http://genomebiology.com/2007/8/7/R146 <br />  <br />  <br />  <br />  <br />         Background                                                                      tion of the above may bias the analysis and prevent the iden- <br />         The cell cycle is a series of linked, fundamentally conserved                   tification of an accurate set of conserved cycling genes. <br />         processes that result in high-fidelity cell duplication. Global <br />         transcript levels throughout the cell cycle have been charac-                   Here we use an algorithm that analyzes data from all species <br />         terized using microarray expression data in several species.                    concurrently. This differs from previous methods that per- <br />         These include humans [1], budding and fission yeast [2-6],                      formed this analysis separately for each species and then <br />         plants [7], and bacteria [8]. Early analysis of these experi-                   looked at the overlap. Our method overcomes many of the <br />         ments focused on individual species. Hundreds of genes have                     obstacles discussed above. We use the same scoring method <br />         been identified whose transcripts oscillate during the cell                     for all species, and include parameters that allow a gene in <br />         cycle, and in budding yeast it is estimated that 15% of all genes               one species to influence the score of a <span id='am-84' about='Thesaurus:Homologous_Gene' typeof='owl:Thing'>homologous gene</span> (in <br />         are subject to this type of control. Despite this large cross-spe-              either the same or in another species). These parameters are <br />         cies effort, a number of studies have concluded that a surpris-                 continuous and depend on the similarity between the genes. <br />         ingly small number of genes conserved in two or more species                    They allow for one to many and for many to many mappings <br />         are periodically transcribed in these species. Rustici and cow-                 between genes; they also allow higher quality expression data <br />         orkers [4] compared fission and budding yeast expression                        in one species to improve the quality of the data for other <br />         data. Dyczkowski and Vingron [9] compared three lists of                        species. <br />         cycling genes (budding and fission yeast and human), and <br />         Jensen and colleagues [10] added a fourth species (Arabidop-                    We analyze expression data from four species: budding [2] <br />         sis). All three studies concluded that periodicity at the tran-                 and fission yeast [4-6], human [1], and plants [7]. Our pri- <br />         script level was conserved across species in only a small                       mary goal is to determine sets of genes that are conserved in <br />         number of cases.                                                                sequence and at the transcript level between all and subsets of <br />                                                                                         these species. Our findings indicate that the set of conserved <br />         When comparing cyclic expression patterns across species,                       cycling genes is much larger than was previously thought. <br />         researchers face several challenges. In some cases the lists                    These findings are validated and explained using a large <br />         derived for each species were generated using different                         number of complementary high throughput datasets. <br />         expression analysis methods. For example, the scoring meth- <br />         ods used by Spellman [2] and Rustici [4] and their colleagues <br />         are different, which makes direct comparison problematic.                       Results and discussion <br />         Another challenge arises when determining the set of                            Combined analysis of cell cycle expression data <br />         homologs between the species being analyzed. Although                           We developed an algorithm for combining sequence and <br />         using curated databases results in a more accurate set of con-                  expression data in order to identify cycling genes [12]. The <br />         served pairs, this analysis is limited to a small (and some-                    algorithm uses probabilistic graphical models, and in partic- <br />         times biased) set of genes. In addition, the binary assignment                  ular Markov random fields, to combine these data sources. <br />         (ortholog or not) in databases cannot account for more com-                     Genes are represented as nodes in the graph and are con- <br />         plex similarity measures, which are often represented using a                   nected by edges to other genes (in the same species and all <br />         more continuous value (for example, BLAST e-value). Relying                     other species), based on their sequence similarity as deter- <br />         on the actual strength of homology may help when looking for                    mined by a BLAST score (Figure 1a). Each node (gene) is <br />         conserved sets. Finally, expression data are noisy. Repeated                    assigned an initial cycling score that is determined from <br />         experiments, even within the same species, often result in rel-                 expression data using a method from de Lichtenberg and <br />         atively low agreement [5], and differences between species                      coworkers [13]. Starting with this score, we propagate infor- <br />         may be even more problematic because radically different                        mation along the edges of the graph until convergence. Thus, <br />         synchronization procedures must be used [11]. Any combina-                      if a node with a medium to high score is connected to a set of <br />  <br />  <br />         Figure 1overview <br />         Method      (see following page) <br />         Method overview. (a) Genes (nodes in the graph) are connected to other genes based on sequence similarity. Species identity is indicated by shape of <br />         nodes. Genes are also connected to a &apos;score node&apos;, which represents cycling expression score. Information is propagated along the edges until <br />         convergence. Genes are assigned a posterior score and a cut-off is applied to select the top genes for each species. (b) The subgraph containing the <br />         selected genes is further analyzed by identifying multidomain homology cliques. Examples of identified cliques of conserved genes are presented in panels c <br />         to f. (c) Cyclins. Fission yeast Cig2 promotes the onset of S phase [45]. Human Ccna2 is part of the G2 checkpoint [46]. (d) Cdc6/Cdc18 is a conserved <br />         and essential component of pre-replication complexes (pre-RCs). Orc1 is the largest subunit of the origin recognition complex (ORC), which binds <br />         specifically to replication origins and triggers the assembly of pre-RCs [47]. (e) TOG related proteins, a family of microtubule-associated proteins (MAPs). <br />         Proteins in this group localize to the plus-end tips of microtubules and are essential for spindle pole organization. Alp14 is a component of the Mad2- <br />         dependent spindle checkpoint cascade sharing redundant functions with Dis1. Mutants with both genes knocked out are nonviable [48]. (f,g) Microtubule <br />         component clique and expression profiles for fission yeast Nda3 in eight experiments [4-6]. Nda3, a known cell division gene [49], obtains a high cycling <br />         score but is not one of the 600 top cycling fission genes based on expression analysis. Using our method, its score is correctly elevated because its <br />         sequence similarity to high scoring genes. <br />  <br />  <br />  <br />         nodes with high scores, then the information from the neigh-                    (Figure 2b). Similarly, both our list and the original list [1] of <br />         boring nodes can be used to elevate our belief in the assign-                   cycling human genes are enriched for binding of known cell <br />         ment of this node, and vice versa. This method allows us to                     cycle factors (Nrf1 and E2f2; Additional data file 1 [Support- <br />         identify several cycling genes that can be missed in an analy-                  ing Figure 4d]). Genes on both lists exhibit lower expression <br />         sis focused on a single species as a result of expression noise                 levels in nonproliferating tissues (Figure 2c) and higher <br />         (Figure 1 and Additional data file 1 [Supporting Figures 1 to                   expression levels in cancer cells (Additional data file 1 [Sup- <br />         3]). Similarly, genes with marginal scores that are only con-                   porting Figure 4c]). Expression data for fission yeast and <br />         nected to low scoring genes can be filtered out of the cycling                  Arabidopsis support our list for these species as well (Figure <br />         gene lists. Once the algorithm converges each gene is                           2d-e). <br />         assigned a posterior cycling score between 0 and 1. For com- <br />         parison reasons, we select for each species a set of genes with                 Combined, these results indicate that the species-specific lists <br />         roughly equal size to those used in the original reports                        derived using our method are comparable in quality to those <br />         (although the identity of these genes is different), remove all                 of previously reported cell cycle gene lists. Additional data file <br />         other genes from the graph, and consider only the subgraph                      2 (Supporting Tables 1 to 3) presents the percentage overlap <br />         induced by the selected genes. This graph is analyzed to iden-                  between the lists of cycling genes identified using our method <br />         tify multidomain homology cliques [14] (Figure 1b-e). Each of                   and previously reported cycling gene lists for the four species. <br />         these cliques is then analyzed to determine the set of species <br />         included. These findings are reported as three cell cycle con-                  Conserved cycling genes <br />         servation (CCC) sets with conservation across two (budding                      Figure 3a presents the number of conserved genes for the dif- <br />         and fission yeast), three (yeasts and human cells), or all four                 ferent evolutionary distances represented in our datasets. <br />         species.                                                                        About 21% of the budding and fission yeast cycling genes <br />                                                                                         reside in cliques containing genes from these two species <br />         See Materials and methods and Additional data file 3 for fur-                   (CCC2). When adding human genes, roughly 10% of cycling <br />         ther details on our graph-based algorithm and on clique anal-                   yeast genes and 8% of cycling human genes are included in <br />         ysis. Also see our supporting website [15] for a complete list of               such cliques (CCC3). Finally, between 5% and 7% of cycling <br />         genes identified using our algorithm.                                           genes in all four species are conserved in sequence and <br />                                                                                         expression (CCC4). Additional data file 2 (Supporting Tables <br />         Analysis of identified cycling genes                                            4 to 10) presents the list of genes assigned to CCC4 and CCC3 <br />         Our method combines expression and sequence data. This                          for each of the species. We note that although our original <br />         raises an obvious question; is the quality of our lists compara-                sequence similarity criterion was based on BLAST e-values, <br />         ble to the quality of previous lists that relied on expression                  following the clique analysis the resulting sets are in very <br />         data alone? In other words, does our method sacrifice the                       good agreement with curated homology databases [17]. For <br />         accuracy with respect to the set of cycling genes in each spe-                  example, 82% of budding yeast genes in CCC2 have a curated <br />         cies in order to obtain a larger set of conserved genes?                        fission yeast homolog in CCC2. Similarly, 82% of fission yeast <br />                                                                                         genes in CCC2 have a curated budding yeast homolog in <br />         A possible way to assess the quality of such lists is by compar-                CCC2. See our supporting website [15] for complete hom- <br />         ing them with other high-throughput data sources [13]. For                      ology references. <br />         example, protein-DNA binding data are available for nine <br />         budding yeast transcription factors that are known to be                        To test the agreement of our conserved lists with complemen- <br />         involved in cell cycle specific transcription [16]. It is expected              tary high-throughput datasets, we have repeated and <br />         that many cycling genes would be bound by these factors.                        extended our analysis discussed above but focusing only on <br />         When comparing the genes in our list with the original list [2],                genes included in CCC3 and CCC4. As Figures 2 and 3 and <br />         we find that both exhibit a threefold enrichment for these                      Additional data file 1 (Supporting Figure 4) show, CCC3 and <br />         interactions compared with a random gene list (Figure 2a).                      CCC4 genes exhibit much stronger cell cycle characteristics <br />         Stationary phase expression experiments yield similar results                   when compared with the original set of cycling genes for each <br />  <br />  <br />         Figure 2of(see <br />         Analysis    cycling <br />                        following <br />                             genespage) <br />                                   using complementary high throughput datasets <br />         Analysis of cycling genes using complementary high throughput datasets. (a) Number of interactions between cycling genes and nine cell cycle <br />         transcription factors. (b) Average expression level of sets of budding yeast genes in stationary phase (data from Gasch and coworkers [50]). (c) <br />         Expression levels of human genes in normal tissues, using data presented by Shyamsundar and colleagues [51] (also see Additional data file 1 [Supporting <br />         Figure 4]). Genes in the conserved set have lower expression levels for most nonproliferating normal tissues when compared with the full list and the list <br />         presented by Whitfield and coworkers [1]. For 26 out of 36 normal tissues this difference is significant with a P value &lt; 0.05. (d) Arabidopsis cells in <br />         developmental arrest experiments [52]. Flower cells in the mutants stop growing after stage 11, whereas cells in the stem grow normally. Again, the <br />         conserved set is expressed at lower levels in developmental arrest (P = 0.027 at stages 11 and 12; P = 0.003 at stages 13 and 14). (e) Expression data from <br />         studying sexual differentiation and mating in fission yeast [53]. <br />  <br />  <br />  <br />         species. For example, in a protein-protein interaction dataset                 motif finders were run on each dataset; SOMBRERO [24,25], <br />         for budding yeast [18,19], genes in CCC3 are involved in ten                   BioProspector [26], Consensus [27], and AlignACE [28] (see <br />         times more pair-wise interactions when compared with a ran-                    Materials and methods, below, for details). All four motif <br />         dom set of similar size from the full set of cycling genes (Fig-               finding algorithms were able to identify the Swi6 motif in BY6 <br />         ure 3c). This indicates that these genes have long been                        and FY6C, indicating that this motif is conserved between the <br />         involved in the same function. Similarly, the percentages of                   two species, at least for some of the conserved cycling genes <br />         human genes bound by two cell cycle transcription factors are                  (Additional data file 1 [Supporting Figures 6 and 7]). In sharp <br />         much higher for the CCC3 and CCC4 sets (Nrf1 and E2f2 [20];                    contrast, none of these motif finders was able to identify the <br />         Additional data file 1 [Supporting Figure 4d]). Also, for                      Swi6 motif in the upstream regions of genes in FY6NC. <br />         humans the CCC3 and CCC4 sets are much more repressed in <br />         several nonproliferating tissues when compared with the full                   Mechanistic similarities and differences between cell <br />         set of cycling human genes (Figure 2c). Similarly, CCC4 genes                  cycle regulation in budding and fission yeast <br />         exhibit stronger cell cycle characteristics in Arabidopsis and                 We have extended the motif analysis discussed above to study <br />         fission yeast expression experiments (Figure 2d-e).                            ten additional transcription factors that were determined to <br />                                                                                        play a key role in regulating cycling genes in budding yeast <br />         We have also repeated our analysis by comparing our lists                      [3,16]. For each of these factors we extracted all cycling bud- <br />         with subsets of cycling genes with high amplitude in each of                   ding yeast genes determined to be bound by this factor [23] <br />         the four species. As shown in Additional data file 1 (Support-                 and their fission yeast homologs. As we did for Swi6, we fur- <br />         ing Figure 5), high amplitude genes exhibit similar cell cycle                 ther divided the fission yeast genes into two sets; the first con- <br />         characteristics to the CCC3 and CCC4 sets for human and                        tains fission yeast genes in CCC2 and the second (a negative <br />         plants. However, for the two yeasts these high amplitude                       control list) contains noncycling fission yeast homologs of <br />         genes are more similar to the full set of cycling genes. This                  cycling budding yeast genes. Next, we ran the four motif find- <br />         indicates that expression analysis alone cannot be used to                     ers on each dataset. <br />         identify this core set of genes. <br />                                                                                        The results are presented in Table 1 and Additional data file 1 <br />         Motif analysis for budding and fission yeast genes                             (Supporting Figures 6 to 17). In Table 1 we report on the <br />         To further validate our findings of a large overlap between the                number of motif finders that identified the correct motif for <br />         cycling genes in the two yeast species, we turned to motif                     each factor and on the percentage of genes in the set that con- <br />         analysis. Several transcription factors are conserved between                  tained this motif. Similar to the results obtained for Swi6, the <br />         budding and fission yeast [21]. A possible explanation for                     other two G1/S factors, namely Swi4 and Mbp1, exhibit the <br />         expression conservation (or lack thereof) is in the conserva-                  optimal motif conservation <span id='am-87' about='xsp:pattern' typeof='owl:Thing'>pattern</span>; the expected motifs are <br />         tion (or lack of conservation) of a binding motif for these                    found in both the fission yeast cell cycle genes and the positive <br />         cycling genes.                                                                 control of conserved budding yeast cell cycle genes, but are <br />                                                                                        not found in the negative control set of noncycling fission <br />         We started by looking at genes bound by the budding yeast                      yeast genes. Motif scan analysis (Additional data file 2 [Sup- <br />         factor Swi6, which regulates transcription at the G1/S transi-                 porting Table 11]) confirms the results for these factors. For <br />         tion [22]. We extracted three lists for this factor. The first,                G2/M, the Fkh2 sets display similar, although less significant, <br />         denoted BY6, contained cycling budding yeast genes in CCC2                     pattern (two of four motif finders identified the correct motif <br />         determined to be bound by Swi6 [23]. The second list,                          for the cycling set). However, Fkh1 and Fkh2 motifs also <br />         denoted FY6C, contained fission yeast genes that both were in                  appear, although less strongly, in the negative control sets. In <br />         CCC2 and had homologs in BY6. These genes were deter-                          total, FKH-like motifs are present in eight of the 11 negative <br />         mined to be cycling and conserved by our method. The third                     control datasets. The M/G1 phase analysis is complicated by <br />         list (FY6NC) contained noncycling fission yeast genes with                     small dataset size. This may result from the lack of conserva- <br />         cycling budding yeast homologs bound by Swi6. This latter                      tion between the two species for this phase [21]. As a result, <br />         list serves as a negative control because it contains genes that               motif match for this set is either weak (Swi5) or nonexistent <br />         have lost their cycling status between the two species. Four                   (Mcm1 and Yox1). <br />  <br />  <br />         Figure 3 (see of <br />         Conservation   following <br />                           cyclingpage) <br />                                   genes <br />         Conservation of cycling genes. (a) Percentage of conserved cycling genes in the four species. (b) Enrichment of cell cycle related Gene Ontology GO <br />         terms between all cycling genes and the CCC3 set in budding yeast, fission yeast, and humans. (c) Yeast protein-protein interactions [18]. We counted the <br />         number of interactions within a random set of 80 cycling yeast genes. In all, 1,000 sets were sampled. The histogram on the left plots the number of <br />         interactions observed for these sets. X represents internal interactions with the CCC3 set, which has significantly more internal interactions. <br />  <br />  <br />         The biologic importance of the core set of cycling                              most enriched lists are obtained when using the CCC3 and <br />         genes                                                                           CCC4 sets. For these, the percentage climbs to 16% (CCC3) <br />         To further validate that genes in CCC3 and CCC4 are core                        and 17% (CCC4). These findings highlight the importance of <br />         cycling genes, we studied their importance using deletion                       the conserved set and support our conclusion that it contains <br />         data. Surprisingly, only 15% of cycling yeast genes are essen-                  key cycling genes. <br />         tial in rich media conditions [29], which is roughly equal to <br />         the overall percentage of essential yeast genes (18%). How-                     Conserved protein complexes regulated by the cell cycle <br />         ever, as Figure 4 shows, 35% of budding yeast genes in the                      To determine cell cycle regulated protein complexes con- <br />         CCC3 list and 46% of the genes in the CCC4 lists are essential.                 served between these species, we searched for protein com- <br />         To test whether similar result could be obtained using only                     plexes with one or more subunits in the CCC3 set using high- <br />         sequence data (without expression data for the other species),                  throughput protein-protein interaction data. This type of data <br />         we extracted from the full list of cycling budding yeast genes                  is thus far only available in budding yeast [18,19]. Additional <br />         those with homologs in all other species, without taking into                   data file 2 (Supporting Table 12) and Additional data file 1 <br />         account their cycling status in these other species. Although                   (Supporting Figure 18) present some of the protein com- <br />         this increased the percentage of essential genes (to 27%),                      plexes that we identified. Some of these complexes are known <br />         these percentages remained well below those achieved for                        to regulate important events in the cell cycle. For example, the <br />         CCC4, which uses the expression data.                                           origin recognition complex (ORC) is a well conserved com- <br />                                                                                         plex that is involved in the initiation of DNA synthesis [31]. <br />         We have also carried out similar analyses for human genes                       Other examples are the cohesin complex, which is responsible <br />         using data from <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span>RNA interference</span> (RNAi) experiments [30].                       for binding the sister chromatids during mitosis after S phase <br />         In these experiments 24,373 genes were knocked down using                       [32], and the ribonucleoside-diphosphate reductase (RNR) <br />         RNAi and assessed for phenotypic influence on cell growth.                      complex, which is involved in the maintenance of the cellular <br />         For 1,152 (4.7%) of the genes, the resulting knockdown cells                    pool of dNTPs [33]. <br />         presented phenotypic growth defects. As Mukherji and cow- <br />         orkers [30] note in their report, roughly 6% of cycling human                   Gene Ontology analysis of conserved cycling genes <br />         genes reported by Whitfield and colleagues [1] are included in                  The CCC3 list gives us our first look at the conserved core of <br />         this list. Similar to the process we conducted in yeast, we con-                periodically transcribed genes across evolution. Even though <br />         sidered sequence data only and extracted from the Whitfield                     CCC3 contains relatively few genes (0.4% to 1.3% of the total <br />         list those genes with homologs in the other three species. For                  number of genes for each species), many of these genes play a <br />         this list, the percentage of genes increases to 10%. Again, the                 role in key processes required for growth. Using Gene Ontol- <br />  <br />         Table 1 <br />  <br />         Summary of motif-finding results <br />  <br />         Budding yeast   Transcription     Fission yeast cell cycle     Negative control                Positive control              Extended positive control <br />         phase           factor            genes                        (fission yeast non-cell-cycle   (conserved budding yeast      (all budding yeast CC <br />                                                                        genes)                          cell-cycle genes)             genes) <br />  <br />         G1/S                  Swi4            <br />                               Swi6             <br />                              Mbp1         <br />  <br />         G2/M                  Fkh1           <br />                               Fkh2          <br />                              Ndd1            <br />  <br />         M/G1                Mcm1a         <br />                              Ace2           <br />                               Swi5             <br />                              Yox1       <br />                              Yhp1          <br />  <br />         Motif analysis of the conserved cycling genes in budding and fission yeast. For each set and each factor we list the number of motif finders (up to four) <br />         that identified the correct motif. Each motif finder often recovers multiple correct motifs, and each motif is associated with a list of predicted <br />         instances in promoter regions. We report the percentage of promoters that contain instances predicted by at least one-third of the correct motifs. <br />         The first and third columns are the CCC2 genes in budding and fission yeast, respectively. The second column is non-cycling fission yeast genes with <br />         homolog cycling budding yeast genes. See Additional data file 3 for further details. aMcm1 regulates genes in G2/M and M/G1. bThese datasets contain <br />         ten genes or fewer. ~, weak matches to the known motif. <br />  <br />  <br />                                                                      Genome Biology 2007, 8:R146 <br />  http://genomebiology.com/2007/8/7/R146                                                                                  Genome Biology 2007,   Volume 8, Issue 7, Article R146   Lu et al. R146.9 <br />  <br />  <br />  <br />  <br />                                                                                                                     centage of cycling genes. This indicates that many of the genes <br />    (a)                                                Percentage of essential budding yeast genes                   associated with these functions have been conserved in cyclic <br />                                                                                                                     expression between the species. These include categories <br />                                          50 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                     <span id='am-3' about='rdfs:comment' typeof='owl:Thing'>comment</span> <br />                                                    Spellman (15.3%)                           All genes (17.9%) <br />                                                                                                                     related to DNA metabolism (P = 5.7 &#195;&#8212; 10-12 for CCC3 and P = <br />          Percentage of essential genes <br />  <br />  <br />  <br />  <br />                                                    Spellman w/ homologs (27.0%) <br />                                                    Our list (15.2%) <br />                                                                                                                     1.1 &#195;&#8212; 10-6 for the full list) and chromatin assembly (P = 3.7 &#195;&#8212; <br />                                          40 <br />  <br />  <br />  <br />  <br />                                                    CCC3 (34.7%) <br />                                                    CCC4 (45.9%) <br />                                                                                                                     10-5 versus P = 0.01). In contrast, there are a number of cate- <br />                                                                                                                     gories that are much more enriched in the full list, indicating <br />                                          30 <br />  <br />  <br />  <br />  <br />                                                                                                                     that they have probably evolved, or at least greatly expanded, <br />                                                                                                                     in the individual species. These include categories such as <br />                                          20 <br />  <br />  <br />  <br />  <br />                                                                                                                     mitosis for fission yeast (P = 1.6 &#195;&#8212; 10-4 versus P = 3.6 &#195;&#8212; 10-7) <br />                                                                                                                     and the cell wall category, which exhibits a great deal of spe- <br />                                          10 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                     reviews <br />                                                                                                                     cies-specific variation between the budding yeast, the fission <br />                                                                                                                     yeast, and metazoans [35]. For the human list, DNA repair <br />                                          0 <br />  <br />  <br />  <br />  <br />                                                                                                                     and chromosome segregation were more significantly <br />                                                                                                                     enriched in the full set (P = 5.9 &#195;&#8212; 10-4 versus P = 7.3 &#195;&#8212; 10-5, and <br />                                                                                                                     P &gt; 0.1 versus P = 9.0 &#195;&#8212; 10-8, respectively). Although these <br />                                                                                                                     functions are conserved across organisms, our analysis <br />    (b)                                    Percentage of human genes strongly effecting cell cycle progression <br />                                                                                                                     indicates that many of these genes are cycling only in human <br />                                                                                                                     cells, perhaps indicating that these functions have been <br />                                          20 <br />  <br />  <br />  <br />  <br />                                                    Whitfield (5.6%)                           All genes (4.7%) <br />          Percentage of essential genes <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                     reports <br />                                                    Whitfield w/ homologs (9.7%) <br />                                                    Our list (7.4%)                                                  adapted to accommodate the longer cell cycle. <br />                                                    CCC3 (15.7%) <br />                                                    CCC4 (17.3%) <br />                                          15 <br />  <br />  <br />  <br />  <br />                                                                                                                     Analysis of specific CCC3 genes <br />                                                                                                                     Partial functional knowledge is available for all but one <br />                                          10 <br />  <br />  <br />  <br />  <br />                                                                                                                     (YPL247C) of the 72 budding yeast genes on CCC3. Sixteen of <br />                                                                                                                     these genes encode products that are involved in DNA repli- <br />                                                                                                                     cation and another 23 are involved in chromosome organiza- <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                     deposited research <br />                                          5 <br />  <br />  <br />  <br />  <br />                                                                                                                     tion and biogenesis. These include structural components <br />                                                                                                                     (Mcms, tubulins, and histones) as well as regulatory proteins <br />                                          0 <br />  <br />  <br />  <br />  <br />                                                                                                                     (cyclins, Cdc20, and Cin8). The mcm2 (cdc19) and mcm6 <br />                                                                                                                     genes were previously known to be cyclic subunits of the <br />                                                                                                                     highly conserved Mcm pre-replication complex in fission <br />                                                                                                                     yeast [5,36]. Our combined analysis indicates that two other <br /> The <br /> Figure <br />     importance <br />         4        of the core cycling genes <br /> The importance of the core cycling genes. (a) Percentage of essential                                               genes (mcm3 and mcm5) may also be periodic, similar to the <br /> genes in different sets of budding yeast genes [29]. Although 18% of                                                budding yeast and human Mcm subunits. Another large class <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                     refereed research <br /> budding yeast genes are essential, only 15% of cycling genes are essential.                                         of conserved cyclic genes is involved in chromosome segrega- <br /> Our analysis resolves this apparent contradiction by showing that the                                               tion (ASE1, KIP1, NUM1, and STU2) and cytokinesis (MOB1, <br /> conserved cycling genes lists contain a much higher percentage of essential <br /> genes (35% and 46% for CCC3 and CCC4). Sequence alone cannot                                                        HOF1, KEL2, and IQG1). In addition, the list includes factors <br /> account for this high percentage (27%), indicating the importance of the                                            that affect transcription globally (ARP7 and TUP1) and spe- <br /> combined analysis. (b) Similar analysis for the human lists using data from                                         cifically (ACE2, FKH2, and HCM1). Interestingly, the S phase <br /> RNA interference knockdown experiments [30].                                                                        specific transcription factor Hcm1 has a conserved cyclic <br />                                                                                                                     transcript, as do 22 of its predicted targets [3]. The fact that <br /> ogy (GO) analysis [34], we identified categories that were                                                          nearly 30% of the budding yeast CCC3 genes are potential tar- <br /> enriched in this set. For budding yeast these categories                                                            gets of Hcm1 is consistent with the known role of Hcm1 in reg- <br /> include cell cycle (P = 3 &#195;&#8212; 10-15), DNA replication (P = 2 &#195;&#8212; 10-                                                    ulating genes involved in chromosome dynamics [3,37].                           interactions <br /> 13), and mitosis (P = 1 &#195;&#8212; 10-7). Similar enrichments were found <br />  <br /> for human conserved cycling genes and for fission yeast. For                                                        There is only a small number of genes in CCC3 that are not <br /> example, cell cycle (P = 5 &#195;&#8212; 10-17), DNA replication (P = 7 &#195;&#8212; 10-                                                   obviously involved in cell cycle specific processes. These <br /> 14) and cell division (P = 2 &#195;&#8212; 10-9) are enriched in humans, and                                                    genes include three involved in metal homeostasis (SMF2, <br /> cell cycle (P = 10-9) and chromatin assembly/disassembly (P                                                         SMF3, and CTH2), some cell wall proteins (FIG2, AGA1, and <br /> = 10-9) are enriched in fission yeast. Figure 3 and Additional                                                      SED1) and alkaline phosphatase (PHO8). These gene prod- <br /> data file 2 (Supporting Tables 13 to 21) present P values for                                                       ucts could be involved in unknown aspects of the cell division <br />                                                                                                                                                                                                     information <br />  <br />  <br />  <br />  <br /> the various GO categories.                                                                                          cycle, or they could be evolutionarily related to other cell cycle <br />                                                                                                                     proteins. <br /> Some categories were more enriched in the CCC3 set than in <br /> the full list. For these categories, CCC3 contains a dispropor- <br /> tionate number of genes when compared with the overall per- <br />  <br />                                                                                                      Genome Biology 2007, 8:R146 <br />  R146.10 Genome Biology 2007,   Volume 8, Issue 7, Article R146   Lu et al.                                   http://genomebiology.com/2007/8/7/R146 <br />  <br />  <br />  <br />  <br />          Conclusion                                                              using expression experiments in multiple species, including <br />          By applying a combined analysis, coupled with an unbiased               immune response and circadian rhythm. <br />          homology metric, we were able to identify a large set of genes <br />          as conserved in sequence and cycling status between four dif- <br />          ferent species: budding and fission yeast, human, and Arabi-            Materials and methods <br />          dopsis.                                                                 Assigning cyclic status to genes <br />                                                                                  We applied a probabilistic graphical model to combine micro- <br />          A number of previous efforts to compare cycling gene lists              array expression data and sequence data for identification of <br />          derived independently for each species concluded that only a            cycling genes, as described in Lu and coworkers [12]. We used <br />          small number of genes are conserved between these species.              microarray expression data reported by Spellman [2], Rustici <br />          For example, Rustici and coworkers [4] concluded that only              [4], Oliva [5], Peng [6], Whitfield [1], and Menges [7] and <br />          5% to 10% of cycling budding yeast genes have a cycling                 their coworkers. We downloaded protein sequences from the <br />          homolog in fission yeast. Jensen and colleagues [10] identi-            National Center for Biotechnology Information website [41]. <br />          fied only five orthologous groups as conserved between the              The method starts by using gene specific expression data to <br />          four species (about 1% of the cycling genes) and only eight             compute a cycling score based on both the amplitude and <br />          groups (2%) between the three species of CCC3. The differ-              periodicity [13]. We run BLASTALL [42] to calculate bit <br />          ences between these conclusions can be attributed to differ-            scores between all pairs of sequences, as was done by Sharan <br />          ences in the analysis of the expression and sequence data, as           and coworkers [43]. We use a Markov random field to model <br />          mentioned in the Introduction (above). We note, however,                the joint likelihood of the data. The (hidden) cycling status of <br />          that the results presented by Oliva and coworkers [5] provide           each gene is represented by a node in the graph, and two <br />          partial support to our conclusions. Although they did not               nodes are connected by an edge if the bit score for the two <br />          carry out a complete conservation analysis, they found that 72          genes is above a threshold. We define potential functions on <br />          of their top 200 cycling fission yeast genes (36%) had a                nodes to capture information from the cycling scores, where <br />          cycling homolog in budding yeast. Earlier work that used                we assume the scores of cycling and the noncycling genes fol- <br />          clustering methods to look at global expression similarities            low a mixture of extreme value distributions, and define <br />          between species also supports our findings regarding the                potential functions on edges to capture the correlation of <br />          extent of expression conservation [38,39]. Although our anal-           cycling statuses between similar genes. The posterior beliefs <br />          ysis identifies a larger fraction of conserved cycling tran-            of the cycling status of the genes are estimated using loopy <br />          scripts than does that conducted by Jensen and colleagues               belief propagation algorithm. Finally, we rank the genes by <br />          [10], we find the same striking co-occurrence of cell cycle spe-        their posterior and use the name number as were used in the <br />          cific phosphorylation of the gene products they encode. As              original papers (500 for Arabidopsis, 800 for budding yeast, <br />          Additional data file 2 (Supporting Table 22) shows, when                600 for fission yeast genes, and 1,000 human genes). See <br />          using data on Cdk1 phosphorylation [40] we find that 65% of             Additional data file 3 for complete details. <br />          tested CCC3 gene products are phosphorylated by Cdk1. This <br />          percentage is twice the percentage of phosphorylated gene               Identifying conserved sets <br />          products from the full set of tested cycling genes (33%) and            Genes identified as cycling in each species were used to iden- <br />          eight times higher than the percentage of tested random                 tify conserved sets of cycling genes. This is done using the <br />          genes (8%). These finding reinforces the view that there is a           Markov clustering algorithm (MCL) [14] as follows. First, we <br />          conserved core of genes that are regulated at multiple levels           start with the graph of all cycling genes. Edges in the graph <br />          during the cell cycle in most eukaryotic cells.                         are defined based on the bit score cut-off, as mentioned <br />                                                                                  above. Second, for any connected subgraph in this graph, we <br />          Our results are strongly supported by the fact that genes con-          use MCL to break it into smaller subgraphs if it has more than <br />          served in two or more species display much stronger cell cycle          30 nodes. Third, repeat the previous step until all connected <br />          characteristics than the full list for each species. They also          subgraphs have at most 30 nodes. <br />          show extensive interactions within the set, and almost half of <br />          the CCC4 yeast genes are essential. These observations and              Next, we assign genes to different conserved sets based on the <br />          GO analysis indicates that these genes are crucial compo-               other species represented in the subgraph to which they <br />          nents of the cell cycle system. Combined, these findings sup-           belong. The numbers of genes in the conserved sets are shown <br />          port our claim that the lists we derive contain a core                  in Figure 3, in which the sets are organized as a tree reflecting <br />          conserved set of cycling genes.                                         the evolutionary relation between the four species. <br />  <br />          Our findings indicate that combined analysis of expression              Motif discovery <br />          and sequence data leads to refined lists containing a core set          For each gene in the lists, the appropriate intergenic region <br />          of system specific genes. Although we have focused here on              was extracted from the budding or fission yeast genome. Four <br />          the cell cycle, such an analysis can be carried out to study a          motif finders were run on each dataset: SOMBRERO [24,25], <br />          number of other biologic systems that have been profiled                Consensus [27], BioProspector [26], and AlignACE [28]. Both <br />  <br />   <br />  <br /> SOMBRERO and BioProspector require a background model,                           <br /> and the background was constructed from all intergenic                     <br /> regions (in the appropriate genome) for both cases. SOM-                                                                                                              comment <br /> BRERO was run using default settings, and simultaneously                   <br /> using all known yeast motifs as an appropriate source of prior                <br /> knowledge. Consensus and BioProspector were run using                             <br /> default settings requiring the top ten motifs to be reported.           <br /> AlignACE was run using default settings (using a seed motif                        <br /> <span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span> of 10), and provided with the background intergenic                  <br /> GC content (31.45% for fission and 35.3% for budding). See                 <br /> Additional data file 3 for further details.                               <br />  <br /> P value analysis                                                                  <br /> GO enrichment P values were computed using STEM [44],                               <br /> which relies on hyper-geometric distribution. Corrected P                      <br />                                                                      <br /> values were computed by permutation analysis using STEM.   <br /> </body></html>