<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>RNase Y, a novel endoribonuclease, initiates <br /> riboswitch turnover in Bacillus subtilis <br />  <br />                                              the pyrophosphatase RppH has been shown <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> be the rate <br />                                                                  limiting step in the 50 end dependent action of RNase E on many <br />                                                                                         mRNAs (Deana et al, 2008). Inactivation of RNase E increases <br />  <br />                                     bulk mRNA stability (Babitzke and Kushner, 1991) and the <br />                                                                                         abundance of 440% of all E. coli mRNAs (Lee et al, 2002). <br /> In contrast to Escherichia coli, initiation of mRNA decay in                                B. subtilis has no ortholog of RNase E but the essential <br /> Gram-positive organisms is poorly understood. We studied                                RNase J1 (rnjA) and its paralog RNase J2 (rnjB) have endo- <br /> the fate of the highly structured RNAs generated by pre-                                nucleolytic cleavage specificity similar to that of RNase E <br /> mature transcription termination of S-adenosylmethionine                                (Even et al, 2005). They also have a 50 &#226;&#8364;&#8220;30 exonucleolytic <br /> (SAM)-dependent riboswitches in Bacillus subtilis. An                                   activity (Mathy et al, 2007) and are sensitive to the phos- <br /> essential protein of earlier unknown function, YmdA,                                    phorylation state of the 50 end, with monophosphorylated or <br /> was identified as a novel endoribonuclease (now called                                  hydroxylated 50 ends being the preferred substrates (Even et al, <br /> RNase Y) that was capable of preferential cleaving in vitro                             2005; Mathy et al, 2007; Li de la Sierra-Gallay et al, 2008). The <br /> of the 50 monophosphorylated yitJ riboswitch upstream                                   X-ray crystallographic structure of RNase J1 showed the pre- <br /> of the SAM-binding aptamer domain. Antiterminated full-                                 sence of a mononucleotide-binding pocket at one nucleotide <br /> <span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span> yitJ mRNA was not a substrate for RNase Y                                        distance <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> the catalytic site that cannot accommodate a <br /> in vivo and in vitro, transcripts capable of forming the                                triphosphate group (Li de la Sierra-Gallay et al, 2008). <br /> antiterminator were only cleaved in the presence of SAM.                                    RNases J1 and J2 have overlapping substrate specificities <br /> Turnover of 10 other SAM-dependent riboswitches was                                     but together affect the expression levels of hundreds of genes <br /> also initiated by RNase Y. Depletion of this ribonuclease                               (Ma&#194;&#168;der et al, 2008). Individual case studies have shown that <br /> increased the half-life of bulk mRNA more than two-fold.                                RNases J1/J2 are effectively involved in the degradation of <br /> This indicates that RNase Y might be not only important                                 specific structural mRNAs and in fine-tuning the expression <br /> for riboswitch RNA turnover but also as a key player in the                             of a polycistronic transcript through a processing event that <br /> initiation of mRNA decay in B. subtilis. About 40% of the                               generates matured transcripts with differential stabilities <br /> sequenced eubacterial species have an RNase Y orthologue.                               (Ma&#194;&#168;der et al, 2008). In theory, the polyvalence of RNases <br /> The EMBO Journal (2009) 28, 3523&#226;&#8364;&#8220;3533. doi:10.1038/                                     J1/J2 could explain all characteristics described for mRNA <br /> emboj.2009.283; Published online 24 September 2009                                      degradation in B. subtilis, including a decay-initiating clea- <br /> Subject Categories: RNA; cellular metabolism                                            vage followed by 50 exonuclease degradation. However, in an <br /> Keywords: Bacillus subtilis; endoribonuclease; mRNA degra-                              rnjA/rnjB double mutant (RNase J1 was depleted) the half- <br /> dation; riboswitch; RNase Y                                                             life of global mRNA was only slightly increased (Even et al, <br />                                                                                         2005) an observation that does not support the idea of an <br />                                                                                         important function for RNases J1/J2 in initiating global <br />                                                                                         mRNA decay. More generally, direct evidence that endonu- <br />                                                                                         cleolytic cleavage has a major function in mRNA decay in <br /> Introduction                                                                            B. subtilis is still elusive. <br /> The degradation of mRNA is a vital process in all organisms.                                Secondary structures in the untranslated leader are a <br /> By regulating this process, cells can control gene expression                           common feature of many B. subtilis mRNAs. To gain more <br /> and react rapidly to changing growth conditions. Bacterial                              insight into the events initiating decay of structured mRNA <br /> mRNA degradation has been best studied in the two-model                                 we have studied the fate of S-adenosylmethionine (SAM)- <br /> organisms Escherichia coli and Bacillus subtilis and has                                dependent riboswitches. This conserved genetic system <br /> revealed surprising differences in enzymes and pathways                                 (S-box) controls the expression of 11 transcriptional units <br /> between Gram-negative and Gram-positive bacteria.                                       (about 26 genes) involved in biosynthesis and transport of <br /> In E. coli, mRNA decay is initiated by endonucleolytic                                  methionine and SAM in B. subtilis (Grundy and Henkin, <br /> cleavage followed by 30 &#226;&#8364;&#8220;50 exonucleolytic degradation.                                 1998, 2003; Winkler and Breaker, 2005; Henkin, 2008). <br /> The major player in this process is RNase E, an essential                               Regulation occurs at the level of premature transcription <br /> endoribonuclease with a significant preference for 50 mono-                             termination (Grundy and Henkin, 1998). In the presence of <br /> phosphorylated substrates (Carpousis et al, 2009). The                                  methionine (when SAM levels are high) SAM binds directly to <br /> removal of the 50 triphosphate from primary transcripts by                              the leader RNA. This causes a structural rearrangement that <br />                                                                                         prevents the formation of the antiterminator (which forms <br /> *Corresponding author. IBPC, CNRS UPR9073, Institut de Biologie                         preferentially in the absence of the effector) by stabilizing an <br /> Physico-Chimique, 13, rue Pierre et Marie Curie, Paris 75005, France.                   antiantiterminator structure, called the SAM aptamer (posi- <br /> Tel.: &#195;&#190; 33 1 58415127; Fax: &#195;&#190; 33 1 58415020;                                            tions &#195;&#190; 58 to &#195;&#190; 177, Figure 1A). This favours the formation <br /> E-mail: putzer@ibpc.fr                                                                  of the terminator structure and provokes premature termina- <br /> Received: 19 March 2009; accepted: 24 August 2009; published                            tion of transcription (Epshtein et al, 2003; McDaniel et al, <br /> online: 24 September 2009                                                               2003, 2005; Winkler et al, 2003). Terminator read-through is <br />  <br /> &amp; 2009 European Molecular Biology Organization                                                                       The EMBO Journal   VOL 28 | NO 22 | 2009 3523 <br />        RNase Y initiates mRNA decay in B. subtilis <br />       K Shahbabian et al <br />  <br />  <br />  <br />  <br />       Figure 1 Model for regulation of S-box gene expression in response to SAM. (A) In the presence of methionine (high SAM level), binding of <br />       SAM (represented by an encircled S) stabilizes the antiantiterminator structure favouring the formation of the terminator structure, which <br />       causes premature termination of transcription. (B) When SAM levels are low, the default antiterminator structure forms by <span id='am-238' about='xsp:base' typeof='owl:Thing'>base</span> pairing of <br />       sequences of the antiantiterminator (grey, named I) and the terminator (hatched, named II), allowing expression of the downstream coding <br />       region. The positions of oligonucleotides are shown. Relevant positions on the transcript are numbered from the transcription start point <br />       (determined in Figure 2C). The yitJ AUG start codon is indicated. <br />  <br />  <br />  <br />       induced in response to starvation for methionine when SAM               Initiation of riboswitch degradation requires an <br />       levels are low (Figure 1B). Under noninducing conditions                essential gene of unknown function <br />       (SAM excess), this transcription antitermination system pro-            The most straight-forward hypothesis for the apparently <br />       duces large amounts of prematurely terminated mRNA from a               efficient turnover of the yitJ riboswitch RNA was an initial <br />       constitutive promoter. Efficient degradation of these highly            cleavage preparing the structured RNA for exonucleolytic <br />       structured RNAs with bound SAM (Tomsic et al, 2008) is not              digestion. Obvious candidates for a decay-initiating cleavage <br />       only important for nucleotide recovery but also for recycling           were RNase III and RNases J1/J2. The former is a double <br />       of SAM, a costly metabolite.                                            strand-specific endoribonuclease that could cleave within the <br />          In this work, we examined the degradation of these com-              long structured 50 noncoding region. RNases J1/J2 have been <br />       plex structures. We have identified a novel endoribonuclease,           shown to have a function in the expression of various <br />       encoded by an essential gene of earlier unknown function,               structured leader mRNAs (Even et al, 2005; Choonee et al, <br />       ymdA that is responsible for initiating the decay of all SAM-           2007; Deikus et al, 2008; Yao et al, 2008). We carried out a <br />       dependent riboswitches in B. subtilis. We provide evidence              Northern blot analysis using an oligonucleotide probe <br />       that this enzyme, called RNase Y, is likely to be a key player          (HP1400) hybridizing close to the putative 50 end of the yitJ <br />       in initiating mRNA decay in this organism and we discuss                mRNA (positions 28&#226;&#8364;&#8220;58, Figure 1) (Grundy and Henkin, <br />       how RNases Y and J might act together.                                  1998). In a rnjA, rnjB double mutant, which expressed no <br />                                                                               RNase J2 (gene deletion) and showed a 20-fold reduced <br />                                                                               level of RNase J1 from the Pxyl-rnjA gene in the absence <br />       Results                                                                 of the inducer (Ma&#194;&#168;der et al, 2008), we were unable to detect <br />                                                                               any prematurely terminated transcripts (Figure 2B, lanes 2 <br />       Turnover of the yitJ riboswitch is very efficient <br />                                                                               and 3). A similar result was obtained in an rnc null mutant <br />       Expression of the S-box gene yitJ was reported to be specifi- <br />       cally induced by starvation for methionine (Grundy and                  (Figure 2B, lane 6), suggesting that neither RNase III <br />       Henkin, 1998). Northern blot analysis was carried out                   nor RNases J1 and J2 are involved in the initial event <br />       to monitor the presence of terminated and read-through                  leading to the degradation of the riboswitch. We then <br />       transcripts in a methionine auxotroph grown in the presence             turned our attention to the ymdA gene, one of the few <br />       or absence of methionine. Total RNA was gel separated and               remaining essential genes (Kobayashi et al, 2003) whose <br />       hybridized with an RNA probe complementary to the 30 end                function remained unknown. We constructed a strain <br />       of the yitJ leader and the beginning of the structural gene             (SSB447) in which the ymdA chromosomal copy was under <br />       (positions 152&#226;&#8364;&#8220;346, Figure 1, see Materials and methods).               control of the IPTG-inducible Pspac promoter. Total RNA was <br />       Care was taken to exclude yitJ leader sequences prone to                extracted from this strain grown in the presence of 2 mM <br />       cross-hybridize with other S-box gene leaders. As expected,             IPTG or after resuspension in the same medium without IPTG <br />       the full-length transcript of 2.1 kb was only detected when the         to deplete YmdA. The RNA was separated on a polyacryla- <br />       cells were starved for methionine (Figure 2A). The yitJ                 mide gel and probed with the same oligonucleotide probe <br />       promoter is constitutively active and is not required for               (HP1400) described above. In the absence of IPTG, we <br />       methionine regulation of yitJ expression (Grundy and                    detected a strong signal of 220 nucleotides. This likely <br />       Henkin, 1998). In the presence of methionine one would                  corresponded to the yitJ leader mRNA prematurely termi- <br />       thus have expected to detect the prematurely terminated                 nated at the leader terminator by the interaction of SAM with <br />       leader transcript of about 220 nt. However, we failed to detect         the riboswitch. In the presence of IPTG, the riboswitch RNA <br />       any transcript of this size (Figure 2A and B, lane 1). This             was undetectable, suggesting that the ymdA gene product <br />       observation indicated that an efficient mechanism must oper-            could encode a ribonuclease initiating the decay of the <br />       ate to initiate the decay of this highly structured transcript.         riboswitch (Figure 2B, lanes 4 and 5). <br />  <br /> 3524 The EMBO Journal VOL 28 | NO 22 | 2009                                                       &amp; 2009 European Molecular Biology Organization <br />                                                                                                      RNase Y initiates mRNA decay in B. subtilis <br />                                                                                                                             K Shahbabian et al <br />  <br />  <br />  <br />                                                                          respect to the yitJ start codon and originates from a Sigma A <br />                                                                          type promoter located upstream (Figure 2C, lane 3 and <br />                                                                          Figure 2E). This transcription start site is 30 nucleotides <br />                                                                          upstream with respect to the earlier predicted 50 end of the <br />                                                                          yitJ mRNA (Grundy and Henkin, 1998; Tomsic et al, 2008). <br />                                                                          The expected length of the leader transcript, originating at <br />                                                                          this position and terminating immediately downstream of the <br />                                                                          terminator is about 220 nucleotides. This corresponds to the <br />                                                                          prematurely terminated leader transcript stabilized in the <br />                                                                          YmdA depleted strain (Figure 2B, lane 5). <br />                                                                             As the yitJ riboswitch is efficiently degraded in the pre- <br />                                                                          sence of YmdA, we wanted to know whether read-through <br />                                                                          transcripts still contain the complete leader sequence. For <br />                                                                          that purpose, total RNA was extracted from a methionine <br />                                                                          auxotrophic strain (but wild type for ymdA) grown in the <br />                                                                          presence or after withdrawal of methionine. The RNA was <br />                                                                          used for primer extension with an oligonucleotide comple- <br />                                                                          mentary to sequences within the yitJ ORF (HP1056, <br />                                                                          Figure 1B). We detected two major signals corresponding to <br />                                                                          the full-length transcript (346 nt) and a transcript of about <br />                                                                          125 nt, which corresponds to a premature arrest of the reverse <br />                                                                          transcriptase at the terminator structure (Figure 2D, lane 2). <br />                                                                          An arrest at the leader terminator might seem at odds with <br />                                                                          the notion that the read-through transcript is not cleaved <br />                                                                          because it exists in the antiterminator conformation. <br />                                                                          However, the RNA used for extending the primer in vitro is <br />                                                                          denatured before the reaction and it is reasonable to assume <br />                                                                          that under these conditions some of the transcript will refold <br />                                                                          into the terminator conformation. No signal was detected at <br />                                                                          a position potentially corresponding to a cleavage site. This <br />                                                                          suggested that the riboswitch is in its antiterminator confor- <br />                                                                          mation, which folds first and is more stable than the termi- <br />                                                                          nator conformation (McDaniel et al, 2003; Winkler et al, <br /> Figure 2 Analysis of yitJ leader RNA in vivo. (A) Northern blot          2003), and that in this form it is not an efficient substrate <br /> analysis of yitJ mRNA. Total RNA from the methionine auxotrophic         for YmdA. This in turn implies that YmdA should not have <br /> strain SSB441, isolated from cultures grown in the presence or           a significant impact on the expression level of yitJ (see <br /> absence of methionine were separated on a denaturing 1.2%                Discussion). <br /> agarose gel. The RNA was probed with a uniformly labelled anti- <br /> sense 195 nt RNA covering 30 sequences of the leader transcript and <br /> extending into the yitJ coding region (positions 152&#226;&#8364;&#8220;346 Figure 1B       YmdA is an endoribonuclease <br /> delimited by oligonucleotides HP1231 and HP1056). An RNA se-             A truncated form of YmdA lacking the 25 N-terminal amino <br /> quence ladder was run in parallel as a size marker. Migration of 23S     acids comprising a membrane-binding domain was purified <br /> and 16S ribosomal RNAs is indicated. (B) High resolution Northern <br /> analysis of the yitJ leader. Total RNA from an RNase J1/J2 double        by affinity chromatography (see Materials and methods and <br /> mutant (rnjA inducible), an rnc mutant and a ymdA mutant                 Discussion). To characterize the enzymatic activity of YmdA, <br /> (inducible), was isolated in the presence or absence of inducer,         we performed in vitro assays on riboswitch RNAs synthe- <br /> separated on an 8% polyacrylamide gel and probed with the leader-        sized with T7 RNA polymerase. The first substrate contained <br /> specific oligonucleotide HP1400 (Figure 1A). A [32P]-labelled DNA <br /> ladder was transferred to the membrane together with the RNA.            leader sequences from position 1 up to 196 (30 end defined <br /> (C) Identification of the yitJ transcription start. Oligonucleotide      by primer HP1238, Figure 1A), which corresponds to the <br /> HP1238 (Figure 1A) was hybridized to and extended on total RNA           nucleotide immediately upstream of the terminator stem <br /> isolated from a wild type and Pspac-ymdA mutant strain (&#194;&#177;IPTG).          (Figure 3C). This RNA cannot fold into the antiterminator <br /> The same primer was used for generating the sequence ladder. <br />  &#195;&#190; 1 indicates the band corresponding to the yitJ transcription start;   conformation (Figure 1B), thus increasing the potential to <br /> the first transcribed nucleotide of the yitJ mRNA is encircled.          form the SAM-binding aptamer (Figure 1A). This configura- <br /> (D) Primer extension analysis of total RNA isolated from a methio-       tion corresponds to that of the terminated leader transcript, <br /> nine auxotrophic strain grown in the presence or absence of              which is the most likely substrate for YmdA in vivo. <br /> methionine. T indicates the region corresponding to the terminator <br /> stem. (E) Sequence of the yitJ sigma A type promoter preceding the       Incubation of the 50 -labelled monophosphorylated transcript <br /> new transcription start point identified in (C).                         with YmdA produced a major fragment resulting from en- <br />                                                                          donucleolytic cleavage after an A residue at position 52 <br />                                                                          (Figure 3A and C). Two minor but not reproducible cleavages <br />                                                                          were also observed after positions 124 (in the apical loop of <br />    We used the strain depleted for YmdA to determine                     the aptamer) and 185 (between the aptamer and the termi- <br /> experimentally the transcription start point of yitJ by primer           nator, Figure 3A). <br /> extension using oligonucleotide HP1238 (Figure 1).                          Binding of SAM to the riboswitch aptamer precludes for- <br /> Transcription starts with a G residue at position&#226;&#8364;&#8221;265 with               mation of the default antiterminator conformation (Epshtein <br />  <br /> &amp; 2009 European Molecular Biology Organization                                                 The EMBO Journal     VOL 28 | NO 22 | 2009 3525 <br />        RNase Y initiates mRNA decay in B. subtilis <br />       K Shahbabian et al <br />  <br />  <br />  <br />                                                                             to the structural context of the cleavage site and provides a <br />                                                                             rationale for the fact that the leader transcript, when part of <br />                                                                             the antiterminated full-length transcript, is not a good sub- <br />                                                                             strate for YmdA. <br />                                                                                We therefore considered YmdA to be an endoribonuclease <br />                                                                             and renamed it RNase Y (rny). Cleavage by RNase Y required <br />                                                                             the presence of Mg &#195;&#190; 2 ions, which could be replaced by Mn &#195;&#190; 2 <br />                                                                             or Zn &#195;&#190; 2 (data not shown). <br />  <br />                                                                             RNase Y activity is sensitive to the 5 0 phosphorylation <br />                                                                             state of the substrate <br />                                                                             The endoribonucleolytic activity of RNase Y had been de- <br />                                                                             tected on a 50 monophosphorylated substrate (Figure 3). We <br />                                                                             wanted to determine whether cleavage by RNase Y was <br />                                                                             altered by the nature of the substrate 50 end. The same <br />                                                                             transcript as described above (30 end defined by primer <br />                                                                             1238, Figure 3C) carrying either a tri- or monophosphorylated <br />                                                                             50 end was uniformly labelled with a-[32P]UTP and tested <br />                                                                             under our standard conditions. As expected, the 50 monopho- <br />                                                                             sphorylated substrate was cleaved producing two fragments <br />                                                                             of 145 and 52 nt (Figure 4A, lanes 3 and 4) with identical <br />                                                                             yield, with respect to the number of U residues present in <br />                                                                             each fragment (data not shown), plus many additional pro- <br />                                                                             ducts, some in significant yields. <br />                                                                                In contrast, the transcript carrying a 50 PPP group was <br />                                                                             resistant to cleavage under the conditions used (Figure 4A, <br />                                                                             lanes 1 and 2). This indicated that RNase Y is sensitive to the <br />                                                                             50 phosphorylation state of its substrate and requires a 50 P for <br />                                                                             significant cleavage to occur. A kinetic analysis using 30 end- <br />                                                                             labelled mono- and triphosphorylated substrates confirmed <br />                                                                             the preference of RNase Y for transcripts carrying a mono- <br />                                                                             phosphate group at the 50 end (Figure 4B). The initial reaction <br />       Figure 3 Endonucleolytic cleavage of yitJ leader transcripts by <br />       YmdA. (A) Cleavage of a 50 end-labelled monophosphorylated            rate observed for the 50 P substrate was about 20-fold faster <br />       transcript (30 end defined by primer HP1238). The major cleavage      than that observed with the 50 PPP substrate but after 90 min <br />       site was mapped on a sequence ladder generated with a primer          a similar amount of cleavage product was produced with both <br />       whose 50 end corresponded to the transcription start point.           substrates (Figure 4C). <br />       (B) Effect of increasing SAM concentrations (25 and 80 mM) on <br />       the cleavage of a 50 end-labelled monophosphorylated transcript (30 <br />       end defined by primer HP1356). (C) Cartoon of the riboswitch          Degradation of the yitJ riboswitch in vivo requires <br />       leader. The vertical arrow indicates the cleavage site for YmdA in    RNase Y, RNase J1, PNPase and RNase R <br />       the leader mRNA as identified in panel A. The position of the         We have shown that the turnover of the yitJ riboswitch <br />       primers delimiting the transcript 30 ends on the PCR templates are <br />       shown. The encircled S stands for SAM.                                depends on the presence of RNase Y in vivo (Figure 2B, <br />                                                                             lanes 4 and 5). The discovery that RNase Y is an endoribo- <br />                                                                             nuclease capable of cleaving in vitro the riboswitch leader in <br />                                                                             its terminator conformation provided a rationale for how this <br />       et al, 2003; McDaniel et al, 2003; Winkler et al, 2003). We           nuclease might initiate the decay of the riboswitch in vivo. <br />       therefore tried to mimic the in vivo situation by using a             Therefore, we performed a primer extension analysis using <br />       substrate 12 nt longer, which contained the sequences re-             primer HP1134 (Figure 5D) on total RNA isolated from a <br />       quired for the formation of the antiterminator (up to position        number of endo- and exoribonuclease mutants. As expected, <br />       208, oligonucleotide HP1356, Figure 3D). Such a transcript            the full-length leader transcript was greatly stabilized when <br />       should a priori not be a good substrate for RNase Y because,          RNase Y expression was reduced (Figure 5A, lanes 6 and 7). <br />       in the absence of SAM, it was shown to fold preferentially            In a strain depleted for RNase J1, we detected a weak signal at <br />       into the antiterminator conformation (Epshtein et al, 2003;           position &#195;&#190; 50 (indicated by a scissors symbol labelled 1 in <br />       McDaniel et al, 2003). However, the RNA should become                 Figure 5A, lane 3), which corresponds quite well to the RNase <br />       accessible to cleavage on addition of SAM, which shifts the           Y cleavage site identified in vitro (position 52, Figure 3C), and <br />       RNA structure towards the terminator conformation. As                 a number of smaller degradation intermediates (Figure 5A, <br />       shown in Figure 3B, cleavage by YmdA at its major cleavage            lane 3), implying that the RNase Y cleaved mRNA is less <br />       site (position &#195;&#190; 52) was barely detectable in the absence of          efficiently degraded from its 50 end under these conditions. <br />       SAM but was stimulated significantly by increasing over the           However, we observed a more significant accumulation of <br />       physiological range the concentration of the metabolite,              degradation intermediates in strains devoid of the major 30 &#226;&#8364;&#8220;50 <br />       which in vivo reaches B300 mM in the presence of methio-              exoribonucleases, polynucleotide phosphorylase and ribonu- <br />       nine (Tomsic et al, 2008). A very weak cleavage was also              clease R (Figure 5A, lanes 4 and 5). The two accumulating <br />       observed at position 54. This indicates that YmdA is sensitive        transcripts have 50 ends corresponding either to the original <br />  <br /> 3526 The EMBO Journal VOL 28 | NO 22 | 2009                                                    &amp; 2009 European Molecular Biology Organization <br />                                                                                                      RNase Y initiates mRNA decay in B. subtilis <br />                                                                                                                             K Shahbabian et al <br />  <br />  <br />  <br />                                                                          transcription start or to position &#195;&#190; 58 (Figure 5D). The quite <br />                                                                          dramatic effect of the 30 &#226;&#8364;&#8220;50 exonuclease mutations on the <br />                                                                          stabilization of the riboswitch degradation intermediates <br />                                                                          suggested that the leader transcript must be cleaved some- <br />                                                                          where else to create an entry site for these nucleases. <br />                                                                          Extension of primer HP1238 complementary to sequences <br />                                                                          immediately upstream of the terminator (Figure 1A) failed to <br />                                                                          detect any degradation intermediates on total RNA isolated <br />                                                                          from the pnp/rnr and rnjA mutants (data not shown). This <br />                                                                          indicated that the riboswitch RNA was cleaved at a second <br />                                                                          site somewhere between primers HP1134 and HP1238. We <br />                                                                          performed a 50 /30 RACE experiment by first ligating total RNA <br />                                                                          from the wild type, rnjA/B and pnp/rnr double-mutant <br />                                                                          strains. RT&#226;&#8364;&#8220;PCR across the junction of circularized mRNA <br />                                                                          using primers HP1394 (positions &#195;&#190; 146 to &#195;&#190; 121) and <br />                                                                          HP1395 (positions &#195;&#190; 148 to &#195;&#190; 166, Figure 5D) specifically <br />                                                                          produced fragments of B130 bp for the mutant strains <br />                                                                          (Figure 5B, lanes 1 and 2). This was the size expected for a <br />                                                                          transcript cleaved on either side of the aptamer domain <br />                                                                          (Figure 5D). The RT&#226;&#8364;&#8220;PCR products were purified cloned <br />                                                                          and 20 recombinant plasmids were sequenced. This con- <br />                                                                          firmed the site of RNase Y cleavage upstream of the aptamer <br />                                                                          (Figure 5D, cleavage 1) and is in agreement with the primer <br />                                                                          extension product observed in the RNase J1/J2 mutant <br />                                                                          (Figure 5A, lane 3, position &#195;&#190; 50). The second cleavage by <br />                                                                          RNase Y (see Discussion) occurred after the A residue &#195;&#190; 182, <br />                                                                          also in a single-stranded region between the aptamer and the <br />                                                                          terminator structures (Figure 5D). Further confirmation of the <br />                                                                          site 2 cleavage was obtained through an S1 mapping analysis <br />                                                                          of the downstream cleavage product containing the transcrip- <br />                                                                          tion terminator. A fragment of expected size (25 nucleotides, <br />                                                                          see legend to Figure 5C) accumulated in cells depleted for <br />                                                                          RNase J1 presumably because 50 &#226;&#8364;&#8220;30 exonucleolytic degrada- <br />                                                                          tion of this fragment was inefficient under these conditions <br />                                                                          (Figure 5C, lane 3). On the basis of our data, turnover of the <br />                                                                          yitJ riboswitch in its terminator conformation likely proceeds <br />                                                                          as illustrated schematically in Figure 5D (see Discussion). <br />  <br />                                                                          RNase Y acts on all riboswitch controlled S-box gene <br />                                                                          family members <br /> Figure 4 RNase Y cleavage of the yitJ riboswitch is sensitive to the <br /> 50 phosphorylation state of the substrate. (A) Equal amounts of <br />                                                                          We next investigated whether RNase Y initiated decay of the <br /> [32P] uniformly labelled tri- and monophosphorylated yitJ tran-          SAM-dependent riboswitches of the other S-box genes. For <br /> scripts (30 end defined by primer HP1238) were used as substrates        that purpose, we performed Northern blot analysis of the <br /> in post-transcriptional cleavage assays with RNase Y as described in     other 10 SAM riboswitches present in B. subtilis. mRNA <br /> Material and methods. Reaction products were separated on an 8% <br /> polyacrylamide gel. The major cleavage site is indicated by scissors <br />                                                                          was prepared from strain SSB447, which is wild type for <br /> and the cleavage products are identified by schematic drawings           methionine biosynthesis and contains the IPTG-inducible <br /> on the right. A [32P]-labelled 50 bp DNA ladder was used as a size       rny gene. Specific oligonucleotide probes (Supplementary <br /> marker. (B) Kinetic analysis of RNase Y cleavage of 30 end-labelled      Table I) for each riboswitch were used to avoid potential <br /> tri- and monophosphorylated yitJ transcripts (30 end defined by <br />                                                                          cross-hybridization. With the exception of cysH, a pre- <br /> primer HP1238). Samples from scaled-up cleavage assays were <br /> taken at the indicated time points. The reaction products were           maturely terminated riboswitch transcript was detected for <br /> separated on a 5% polyacrylamide gel. The RNase Y-dependent              all the genes in the culture depleted for RNase Y (Figure 6). <br /> cleavage product of 150 bases is indicated by an arrow. The band         The cysH operon has been shown earlier to be regulated <br /> below corresponds to a spontaneous cleavage of the transcript after <br />                                                                          primarily at the level of transcription initiation in response <br /> its purification and is also observed without addition of the enzyme. <br /> (C) Graph of kinetic analysis of RNase Y cleavage of yitJ transcripts    to O-acetyl-serine (Mansilla et al, 2000) and no prematurely <br /> bearing mono- and triphosphorylated 50 ends. The reactions were          terminated leader transcript was thus expected to be present. <br /> carried out as in (B) but incubated up to 90 min to detect cleavage of   The 11 S-box genes were also analysed in the RNase J1/J2 <br /> 50 tri-phosphorylated transcripts. The part of the gels containing the   double mutant. Using gene-specific probes, we observed no <br /> cleavage products are shown on the top (50 -P) and bottom of the <br /> graph (50 -PPP), respectively. The numbers on the Y axis represent       effect of the absence/depletion of RNases J1/J2 on the <br /> arbitrary units generated by quantification with the Image J             expected full-length leader RNA levels of any of the S-box <br /> program.                                                                 genes (data not shown). Taken together, this indicates that <br />                                                                          RNase Y is the key enzyme for the turnover of SAM- <br />                                                                          dependent riboswitches in B. subtilis. <br />  <br /> &amp; 2009 European Molecular Biology Organization                                                 The EMBO Journal     VOL 28 | NO 22 | 2009 3527 <br />        RNase Y initiates mRNA decay in B. subtilis <br />       K Shahbabian et al <br />  <br />  <br />  <br />       Bulk mRNA stability is increased in an RNase Y depleted    the IPTG-inducible rny copy) grown in the presence or absence <br />       strain                                                     of IPTG inducer. The measured mRNA half-lives were <br />       We next investigated whether RNase Y has an impact on      6.1&#194;&#177;1.3 min ( IPTG) and 3.5&#194;&#177;0.4 min ( &#195;&#190; IPTG), respectively, <br />       global mRNA stability. The chemical decay rate of pulse- <br />       labelled RNA was determined in strain SSB447 (containing <br />  <br />  <br />  <br />  <br />                                                                  Figure 6 Turnover of all functional S-box riboswitches in B. subtilis <br />                                                                  is initiated by RNase Y. Northern blot analysis of the leader tran- <br />                                                                  scripts of 10 S-box genes. Total RNA isolated from the rny (ymdA) <br />                                                                  depletion strain SSB447 grown in the presence and absence of IPTG <br />                                                                  was separated on 8% acrylamide gels and probed with leader- <br />                                                                  specific oligonucleotides (Supplementary Table I). The putative full- <br />                                                                  length leader transcripts are marked by an arrow head. The shorter <br />                                                                  transcripts are likely degradation intermediates. A [32P]-labelled <br />                                                                  DNA ladder was transferred to the membrane along with the RNA. <br />  <br />  <br />                                                                  Figure 5 Degradation of the yitJ riboswitch in vivo requires RNase <br />                                                                  Y, RNase J1, PNPase and RNase R. (A) Extension of primer HP1134 <br />                                                                  (see panel D) on total RNA isolated from wild-type and different <br />                                                                  RNase mutant strains (pnpA and rnr encode polynucleotide phos- <br />                                                                  phorylase and RNase R, respectively). The scissors symbol J        1 <br />                                                                  indicates the RNase Y in vivo cleavage site upstream of the aptamer <br />                                                                  (position &#195;&#190; 50, panel D, see text). Extension products correspond- <br />                                                                  ing to the transcription start ( &#195;&#190; 1) and a degradation intermediate <br />                                                                  ( &#195;&#190; 58) stabilized in the pnpA and rnr mutants (lanes 4 and 5) are <br />                                                                  indicated. (B) Mapping of two RNase Y cleavage sites in vivo by <br />                                                                  50 /30 RACE. Total RNA isolated from the wild-type, pnpA/rnr and <br />                                                                  rnjB/rnjA double-mutant strains, was ligated to create circular RNA <br />                                                                  from potential degradation intermediates. RT&#226;&#8364;&#8220;PCR was used to <br />                                                                  amplify the sequences across the 50 /30 junction. The amplified <br />                                                                  products with an expected size of B130 bp (not present in the <br />                                                                  wild type) were gel purified, cloned and sequenced. The cleavage <br />                                                                  sites were deduced from sequences representing the majority of <br />                                                                  clones from the two mutants (20 independent clones). All <br />                                                                  sequences from the remaining clones identified ligated junctions, <br />                                                                  which, probably due to partial degradation, were located within the <br />                                                                  aptamer domain (data not shown). The products with a size of <br />                                                                  B100 bp present in all lanes were generated by PCR amplification <br />                                                                  from tandemly ligated aptamer RNAs not relevant here (data not <br />                                                                  shown). (C) Detection of the 30 terminal fragment of the yitJ leader <br />                                                                  created by RNase Y cleavage 2. Total RNA from a wild-type and <br />                                                                  rnjB/rnjA double mutant was subjected to S1 mapping analysis <br />                                                                  using oligonucleotide HP1442 (complementary to positions &#195;&#190; 207 <br />                                                                  to &#195;&#190; 131 on the mRNA). The band corresponding in size to the <br />                                                                  expected 30 terminal fragment is indicated. (D) Scheme of the yitJ <br />                                                                  riboswitch RNA on which the two RNase Y cleavage sites (as <br />                                                                  determined by 50 &#226;&#8364;&#8220;30 RACE) on either side of the aptamer domain <br />                                                                  are indicated. The degradation pathway of the riboswitch following <br />                                                                  initial cleavage by RNase Y is illustrated based on the data shown in <br />                                                                  panels A and C (see Discussion). <br />  <br /> 3528 The EMBO Journal VOL 28 | NO 22 | 2009                                          &amp; 2009 European Molecular Biology Organization <br />                                                                                                    RNase Y initiates mRNA decay in B. subtilis <br />                                                                                                                           K Shahbabian et al <br />  <br />  <br />  <br />                                                                       as we show here, the turnover of this transcript is dependent <br />                                                                       on the product of the ymdA gene (Figure 2B). In vitro <br />                                                                       experiments characterized the YmdA protein as an endoribo- <br />                                                                       nuclease (named RNase Y) capable of cleaving the yitJ <br />                                                                       riboswitch six nucleotides upstream of the aptamer structure <br />                                                                       (Figure 3A and C). Interestingly, RNase Y appeared to be very <br />                                                                       sensitive to the RNA conformation downstream of its clea- <br />                                                                       vage site. Consequently, an in vitro transcript including 30 <br />                                                                       sequences, allowing the formation of the antiterminator but <br />                                                                       not the terminator (Figure 3C, primer HP1356), was a poor <br />                                                                       substrate for RNase Y. However, more efficient cleavage was <br /> Figure 7 RNase Y domain structure and effect of point mutations       obtained by adding increasing amounts of SAM (Figure 3B), <br /> on RNase Y activity. (A) The KH and HD domains are indicated          which is known to favour the terminator conformation by <br /> together with the HisAsp doublet conserved in HD domain <br /> proteins. TMD indicates the N-terminal transmembrane domain.          binding and stabilizing the mutually exclusive aptamer do- <br /> (B) Cleavage of the yitJ leader transcript by wild-type RNase Y and   main (McDaniel et al, 2005). This suggested that the principal <br /> H368A and D369A mutant proteins. Cleavage products were sepa-         substrate for RNase Y is the riboswitch complexed with SAM. <br /> rated on an 8% polyacrylamide gel (similar to as shown in Figure      Corroborating this hypothesis, extension of a primer specific <br /> 3A and B). Only the portion of the gel corresponding to the major <br /> cleavage product (52 nt) is shown.                                    for read-through (antiterminated) transcripts present in RNA <br />                                                                       isolated under methionine starvation conditions, detected <br />                                                                       essentially transcripts, which contained the complete leader <br /> compared with 2.8&#194;&#177;0.2 min for the wild-type strain. In the            sequences (Figure 2D). In this case, the more stable anti- <br /> RNase Y depletion strain, global mRNA decay is thus slowed            terminator conformation probably remains predominant and <br /> more than two-fold compared with the wild type.                       inhibits cleavage of the read-through transcripts. Such a <br />                                                                       cleavage would actually be counterproductive as this should <br /> RNase Y is a widely conserved HD domain protein                       destabilize the structural mRNA encoding the YitJ protein <br /> On the basis of the presence of the conserved HD domain, the          and reduce the effect of transcriptional antitermination. In <br /> YmdA protein had been assigned to the superfamily of metal-           support of this notion, depletion of RNase Y had no signifi- <br /> dependent phosphohydrolases (Aravind and Koonin, 1998).               cant impact on the induced expression of a yitJ-lacZ fusion <br /> The presence of an RNA-binding KH domain (Grishin, 2001)              after methionine starvation (data not shown). <br /> together with the HD domain (Figure 7A) suggested that the               Initiation of the yitJ riboswitch decay in vivo turned out to <br /> primary function of this protein is in nucleic acid metabolism.       be more complex than anticipated from the in vitro data. We <br /> As a putative metalloenzyme RNase Y possesses a distinctive           could confirm that RNase Y cleaves the riboswitch upstream <br /> combination of metal-chelating residues, essentially histi-           of the aptamer domain in vivo, two nucleotides upstream of <br /> dines and aspartates (Aravind and Koonin, 1998). To assess            the cleavage site identified in vitro (Figures 3C and 5D). Small <br /> the importance of the HD domain for RNase Y activity,                 differences in cleavage site specificity when comparing in <br /> we mutated His368 and Asp369 that constitute the highly               vivo and in vitro conditions are not uncommon and probably <br /> conserved HD motif. As shown in Figure 7B, the mutated                reflect to some extent, conformational differences of the <br /> proteins were severely impaired in cleavage of the yitJ               structured RNA when presented to the nuclease. More sur- <br /> riboswitch, indicating that the HD domain is important for            prising was the fact that in vivo a second very efficient <br /> the endonucleolytic activity of RNase Y.                              cleavage occurred downstream of the aptamer domain <br />                                                                       (14 nt upstream of the terminator structure, Figure 5D). <br />                                                                       Although cleavage 1 (also observed in vitro) was about <br /> Discussion                                                            50% efficient (as estimated by the primer extension signals <br /> The expression of many genes in B. subtilis and Gram-                 obtained in the pnpA-rnr mutants, Figure 5A, lanes 4 and 5), <br /> positive organisms in general is controlled by transcription          cleavage between the aptamer and the terminator was vir- <br /> antitermination mechanisms (Merino and Yanofsky, 2005;                tually 100% efficient on the prematurely terminated <br /> Grundy and Henkin, 2006), which produce large amounts                 transcripts. The degradation intermediates observed in the <br /> of prematurely terminated transcripts from strong constitu-           pnpA-rnr mutant strains accumulated to even higher levels <br /> tive promoters. The SAM-dependent riboswitch controlling              than the intact leader transcript in the RNase Y depleted <br /> the yitJ gene belongs to this class (Tomsic et al, 2008) and          strain (Figure 5A, lanes 4, 5 and 7). This difference is <br /> forms a highly structured mRNA, especially in its terminator          probably due to the fact that PNPase and RNase R are <br /> conformation, that is when complexed with SAM. It was                 completely absent in the pnpA-rnr <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span>knock-out</span> mutants, <br /> surprising to find that this presumably very stable transcript        whereas RNase Y has only been depleted, a procedure in <br /> was virtually undetectable in a wild-type strain (Figure 2B,          which some residual enzyme remains in the cell. The apta- <br /> lane 1). This meant that there must be a very efficient               mer domain is thus predominantly degraded by the 30 &#226;&#8364;&#8220;50 <br /> mechanism to initiate its decay, providing, a priori, a perfect       exoribonucleases PNPase and RNase R (Figure 5A, lanes 4 <br /> role for RNases J1/J2. These enzymes, particularly RNase J1,          and 5). However, the presence of degradation intermediates <br /> with their dual endo- and 50 &#226;&#8364;&#8220;30 exoribonucleolytic activities        in the RNase J1 depleted strain (Figure 5A, lane 3) suggests <br /> are thought to constitute the heart of the RNA degradation            that some degradation of the aptamer domain also occurs <br /> machinery in Gram-positive organisms. However, they were              through the RNase J1 50 &#226;&#8364;&#8220;30 exonucleolytic activity attacking <br /> not found to be involved in the rate-determining initiating           the riboswitch RNA cleaved at site 1. One of the major <br /> events of SAM riboswitch decay (Figure 2B, lane 3). Instead,          degradation intermediates identified in the pnpA-rnr mutants <br />  <br /> &amp; 2009 European Molecular Biology Organization                                              The EMBO Journal      VOL 28 | NO 22 | 2009 3529 <br />        RNase Y initiates mRNA decay in B. subtilis <br />       K Shahbabian et al <br />  <br />  <br />  <br />       had a 50 end at position &#195;&#190; 58 (Figure 5A, lanes 4 and 5). This          RNase Y appeared to cleave in single-stranded A or AU rich <br />       corresponds to the nucleotide immediately upstream of the            sequences upstream of a secondary structure, a context <br />       highly structured aptamer domain (Figure 5D) and presum-             resembling the cleavage specificities of RNases J1/J2 <br />       ably constitutes a stall site for the 50 &#226;&#8364;&#8220;30 exoribonucleolytic      (Deikus et al, 2008; Even et al, 2005; Yao et al, 2007) and <br />       progression of RNase J1.                                             also E. coli RNase E (Callaghan et al, 2005). The requirement <br />          The riboswitch leader in its terminator conformation              for a downstream secondary structure would explain why <br />       strongly accumulated in an RNase Y depleted strain                   cleavage at site 1 only occurs in the presence of SAM: the <br />       (Figure 2B, lane 5), indicating that the terminator efficiently      lower stem of the aptamer domain (Figure 3C) only forms on <br />       protects the leader riboswitch from 30 to 50 exonucleolytic          binding of the metabolite (McDaniel et al, 2005). This sug- <br />       attack. Degradation of the fragment containing the terminator        gested that cleavage site selection by RNase Y is extremely <br />       thus had to occur principally by the 50 &#226;&#8364;&#8220;30 exonucleolytic           specific. However, SAM might only be needed to form a <br />       activity of RNase J1 using the cleavage at site 2 as an entry        (unspecific) secondary structure 30 to the cleavage site, a <br />       site. This could be confirmed by S1 nuclease mapping, which          role played by the terminator structure downstream of <br />       showed an accumulation of the 25 nt terminal fragment in an          cleavage site 2. In this case, the cleavage specificity of <br />       RNase J1 depleted strain (Figure 5C, lane 3). Taken together,        RNase Y would be compatible with a more general function <br />       our data suggest that SAM riboswitch turnover initiates with         in mRNA metabolism. <br />       rate determining endonucleolytic cleavages by RNase Y,                  RNase Y preferred a 50 monophosphorylated substrate and <br />       which are likely dependent on the intracellular SAM concen-          its depletion significantly increased bulk mRNA stability, two <br />       tration determining RNA folding. The cleavage products then          characteristics it shares with RNase E. Reduced cleavage of <br />       become substrates for the 30 &#226;&#8364;&#8220;50 exonucleases PNPase and             50 triphosphorylated RNA avoids attack of nascent RNA <br />       RNase R (the aptamer domain and probably the 50 nt 50                transcripts and is a priori a useful property for an enzyme <br />       terminal leader fragment) and the 50 &#226;&#8364;&#8220;30 exoribonucleolytic          responsible for setting the decay rate of many mRNAs. <br />       activity of RNase J1. The latter is required for the degradation     However, RNase Y seems to be able to discriminate also <br />       of the 30 terminal fragment and accessory for that of the            between distinct RNA conformations located downstream of <br />       aptamer domain.                                                      its cleavage site. The rather poor cleavage efficiency observed <br />          We have not yet been able to show reproducibly in vitro           in vitro might reflect this sensitivity. It thus remains to be <br />       that RNase Y is responsible for the site 2 cleavage on the           seen to what extent the nature of the 50 end and RNA <br />       riboswitch RNA. This might be due to the unfavourable struc-         structure, respectively, contribute to substrate specificity <br />       tural conformation obtained with the in vitro transcripts chosen     and enzyme activity. <br />       to optimize cleavage at site 1. As the full-length terminated           A recent study using a bacterial two hybrid system showed <br />       riboswitch leader can only be detected in an RNase Y mutant          that PNPase, RNase J1 and the glycolytic enzymes 6-phos- <br />       (Figure 2B, lane 5), it is nevertheless very likely that RNase Y     phofructokinase and enolase can interact with the ymdA gene <br />       cleaves at both sites. As an alternative, cleavage at site 1 could   product and potentially form a degradosome-like complex <br />       be obligatory to create a structural context recognized by           in vivo (Commichau et al, 2009). It will be interesting to see if <br />       another RNase. However, if this was the case then cleavage at        such a complex can be purified and how the addition of <br />       site 2 would be expected to be less or as efficient as cleavage      glycolytic enzymes might improve or alter RNase Y activity in <br />       at site 1. We have observed the contrary (see above).                vitro. The same authors also showed that an earlier reported <br />          RNase Y initiated the turnover of all SAM-dependent               processing of the B. subtilis gapA operon (Meinken et al, <br />       riboswitches in B. subtilis (Figure 6) with the exception of         2003) is diminished in vivo when ymdA expression is reduced <br />       cysH, which was shown earlier to be regulated at the level of        (Commichau et al, 2009). Testing already known RNA clea- <br />       transcription initiation (Mansilla et al, 2000). The sizes of the    vages in an RNase Y depleted strain might thus rapidly lead to <br />       stabilized full-length leader transcripts detected in the RNase      the discovery of new potential substrates for RNase Y. An <br />       Y depleted strain correlated quite well with the transcription       earlier systematic inactivation analysis of unknown but es- <br />       initiation sites inferred from a sequence-based search for           sential genes had shown that depletion of YmdA resulted in <br />       promoters (Grundy and Henkin, 1998 #1760). However, it               several effects on cell and chromosome morphology (Hunt <br />       remains to be analysed whether the degradation pathway of            et al, 2006). Interestingly, a H368A <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span>mutation</span> (that caused a <br />       the yitJ riboswitch elucidated here is also applicable to the        strong decrease in endonucleolytic activity, Figure 7B) <br />       other S-box riboswitches. As the RNA&#226;&#8364;&#8220;SAM complexes are               showed a <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span>phenotype</span> similar to the depleted strain, suggesting <br />       very stable with half-lives ranging from 4 to 11 min (Tomsic         that it is the impaired endonucleolytic activity of RNase Y <br />       et al, 2008), an efficient turnover of the large amounts             that is responsible for the observed effects. <br />       of riboswitch RNA produced is not only important for                    Gfp-fusion experiments localized the YmdA protein to <br />       nucleotide recycling but also for liberating bound SAM,              the cell periphery (Hunt et al, 2006), consistent with the <br />       a costly metabolite.                                                 presence of a predicted N-terminal transmembrane domain <br />          The chemical decay rate of bulk mRNA was slowed more              (Figure 7A). In our hands, the affinity purified, full-length <br />       than two-fold in the RNase Y depletion strain compared with          His-tagged protein was not suitable for in vitro assays due to <br />       the wild-type strain (6.1 versus 2.8 min, respectively). The         considerable contamination by a large number of proteins <br />       effect of RNase Y on global mRNA stability is thus significant       presumably associated with membrane fragments. We re- <br />       and distinctly greater than the 30% increase in mRNA                 solved this problem by deleting the 25 N-terminal amino <br />       stability observed in a RNase J1/J2 double-mutant strain             acids comprising the transmembrane domain and using in- <br />       (Even et al, 2005). This suggests that mRNA cleavage by              tein technology for purification. <br />       RNase Y might have a major function in the initiation of                Localization of RNase Y (alone or in complex with other <br />       mRNA degradation in B. subtilis.                                     proteins) at the membrane, further extends the analogy to <br />  <br /> 3530 The EMBO Journal VOL 28 | NO 22 | 2009                                                   &amp; 2009 European Molecular Biology Organization <br />                                                                                                   RNase Y initiates mRNA decay in B. subtilis <br />                                                                                                                          K Shahbabian et al <br />  <br />  <br />  <br /> E. coli RNase E. In fact, RNase E contains a short domain with      Table I Occurrence of RNase Y in prokaryotes <br /> the propensity to form an amphipathic a-helix that is neces- <br />                                                                                                               RNase <br /> sary and sufficient to bind to membranes (Khemici et al, <br /> 2008), providing a rationale for the cytoskeletal-like structure                                  E/G              J              Y <br /> of the degradosomal proteins in the cell (Taghbalout and            Archaea <br /> Rothfield, 2007). However, the consequence of this localiza-           Crenarchaeota              NO             NO              NO <br /> tion on RNA processing and degradation is not clear, neither <br />                                                                        Euryarchaeota              NO            30/36            NO <br /> for RNase E nor for RNase Y. It was suggested that the <br /> membrane might serve as a storehouse for RNase E and                   Nanoarchaeota           <br /> that the active form of the enzyme is the small proportion of       Bacteria <br /> molecules that are cytoplasmic (Khemici et al, 2008). The <br />                                                                        Actinobacteria          <br /> conserved pseudo-compartmentalization of RNase Y and <br /> RNase E in two very different organisms (B. subtilis and               Bacteroidetes         <br /> E. coli, respectively) clearly suggests an evolutionary advantage      Chlamydiae               <br /> for such a configuration. The membrane bound nucleases                 Cyanobacteria          <br /> might provide an anchor for assembling potentially transient <br />                                                                        Firmicutes <br /> degradosome-type complexes thereby fine tuning the degrada- <br /> tion machinery under various physiological conditions.                     Bacillales         <br />     RNase Y is the first member of a new class of ribonucleases            Clostridia        <br /> that constitute a subgroup within the superfamily of HD <br />                                                                            Lactobacillales       <br /> proteins by combining the metal-chelating HD domain with <br /> the RNA-binding KH domain (Aravind and Koonin, 1998).                  Mollicutes              <br /> Single amino acid substitutions of the highly conserved HD             Proteobacteria <br /> doublet greatly reduced the ribonucleolytic activity of RNase <br />                                                                            Alpha                <br /> Y, indicating that the catalytic activity resides within the HD <br /> domain. Some cleavage by the D369A mutant also occurred                    Beta                <br /> at a second site 2&#226;&#8364;&#8220;3 nucleotides upstream corresponding                    Delta               <br /> closely to the in vivo cleavage site (Figure 7, lane 4).                   Epsilon              <br />     Orthologs of RNase Y occur in about 40% of the currently <br />                                                                            Gamma               <br /> sequenced eubacteria (Table I), with similarities ranging <br /> from 56 to 99%. RNase Y is absent from the archea and                  Spirochaetales          <br /> eukaryotic organisms with a single exception, Drosophila <br /> willistoni. Organisms containing RNase Y also have either <br /> an RNase E/G like enzyme or RNase J. A few bacteria                 direction. This property is especially useful for degrading <br /> actually have all three enzymes, like the Frankia and               transcripts whose 30 ends are protected by a secondary <br /> Salinispora species of Actinobacteria and 25&#226;&#8364;&#8220;30% of the             structure. With this in mind, the 50 &#226;&#8364;&#8220;30 exonucleolytic activity <br /> Bacillales (e.g. Listeria) and Clostridia (Table I). This raises    of RNase J may have evolved in organisms like B. subtilis <br /> interesting questions as to the sharing of functions for all        that, in contrast to E. coli, does not appear to have an efficient <br /> three enzymes and whether one of these usually essential            polyadenylation pathway to allow 30 &#226;&#8364;&#8220;50 degradation of struc- <br /> RNases becomes dispensable in this context. On the other            tured RNAs. In B. subtilis, 50 road blocks (structural or <br /> hand, we only found a single eubacterial species (Sulcia            formed by bound ribosomes, etc.) can substantially stabilize <br /> muelleri in the Bacteroidetes) that has none of the three           even nontranslated downstream sequences, implying that an <br /> enzymes. This is also the case for the Crenarcheota and the         endoribonuclease like RNase Y should not efficiently cleave <br /> majority of the Euryarcheota in which novel key ribonu-             the body of an mRNA. We are still ignorant of the exact <br /> cleases probably remain to be identified.                           parameters that determine substrate specificity for RNase Y <br />     How does RNase Y fit into our current understanding of          and, for that matter, also for the endonucleolytic activity of <br /> mRNA processing and degradation in bacteria? It is intriguing       RNase J. Further characterisation of RNase Y should include <br /> that two completely unrelated enzymes like RNase Y and              an analysis of the relative importance of the preference of <br /> RNase E, from a Gram-positive (B. subtilis) and Gram-nega-          RNase Y for the 50 phosphorylation state and conformation, <br /> tive organism (E. coli) that are thought to follow very             respectively, of the RNA substrate. Interestingly, each of these <br /> different strategies concerning mRNA metabolism, share so           two characteristics has the potential to control access of <br /> many features including large effects on global mRNA stabi-         RNase Y to nascent transcripts. Together, they could easily <br /> lity and a 50 end preference for mono phosphorylated sub-           provide the specificity necessary to reconcile the activity <br /> strates. This raises the possibility that mRNA processing and       of RNase Y with our current understanding of mRNA <br /> degradation might be more similar between B. subtilis and           metabolism in Gram-positive bacteria. <br /> E. coli than presently assumed, with an endonucleolytic clea- <br /> vage being the crucial step in initiating mRNA decay in both <br /> organisms. RNases J1/J2 have smaller effects on bulk mRNA           Materials and methods <br /> than RNase Y (Even et al, 2005), suggesting that these <br /> enzymes are not initiating decay on a large scale (be it            Bacterial strains and growth conditions <br />                                                                     The prototrophic B. subtilis 168 strain 1A2 (BGSC) was the parent <br /> endo- or exonucleolytically) but are mostly involved in             strain used for the construction of mutants. The RNase J1/J2 <br /> degrading cleavage products exonucleolytically in a 50 &#226;&#8364;&#8220;30          double-mutant strain SSB357 (rnjBHspc, PxylHrnjA) was described <br />  <br /> &amp; 2009 European Molecular Biology Organization                                             The EMBO Journal      VOL 28 | NO 22 | 2009 3531 <br />        RNase Y initiates mRNA decay in B. subtilis <br />       K Shahbabian et al <br />  <br />  <br />  <br />       earlier (Even et al, 2005) and the RNase III mutant strain BG322        S1 nuclease mapping <br />       (Herskowitz and Bechhofer, 2000) was provided by D Bechhofer.           A measure of 30 mg of total RNA was mixed with 50 end-labelled <br />       For strain SSB447, the ymdA gene was put under control of the Pspac     oligonucleotide HP1442 (105 cpm, B10 ng), precipitated, and <br />       promoter by Campbell-type integration of plasmid pHMD1. Induc-          resuspended in 40 mM Pipes pH 6.4, 1 mM EDTA pH8, 0.4 M NaCl <br />       tion of yitJ expression by methionine starvation was carried out in     and 80% deionized formamide. Hybridization was carried out by <br />       strain SSB441 (metC2, lys1) provided by T Leighton. E. coli strain      incubating the mixture at 481C overnight and S1 nuclease <br />       SSC91, used for the overexpression of YmdA, was constructed by          (Fermentas) digestion (200 U) was carried out according to the <br />       lysogenisation of strain IBPC852 (Drnb201Htet, rph1, thyA715,           manufacturer&#226;&#8364;&#8482;s instructions at 371C for 30 min. The reaction <br />       rne1) kindly provided by P Regnier with lDE3. B. subtilis was           products were resolved on a 5% polyacrylamide gel. <br />       grown in SMS minimal medium (Spizizen, 1958) supplemented <br />       with 0.1% L-glutamine. Methionine was added to a final concentra-       5 0 /3 0 RACE <br />       tion of 2 mM when required. Antibiotics were added, at the              Total RNA (5 mg) was circularized with RNA ligase (Biolabs) and <br />       following concentrations: ampicillin (200 mg/ml), kanamycin             subjected to RT&#226;&#8364;&#8220;PCR across the 50 /30 junction, using oligonucleo- <br />       (5 mg/ml), chloramphenicol (4 mg/ml). For depletion of the essential    tides HP1134 (RT) and HP1394 and HP1395 (PCR). The PCR <br />       RNases J1 and Y, strains bearing inducible copies of these genes        fragments of B130 bp were isolated from agarose gels, digested <br />       (Pxyl and Pspac, respectively) were grown overnight in the              with EcoRI and BamHI and cloned into plasmid pJRD184 <br />       presence of inducer. The cells were centrifuged and resuspended         (Heusterspreute and Thi, 1985) for sequencing. Ten clones of each <br />       in medium without inducer. Growth was monitored every 30 min            mutant were sequenced. <br />       and the first slowing of growth in the absence of inducer, compared <br />       with a culture grown with inducer, was detectable at an OD600 of        Assay of RNase Y activity <br />       B0.4. Cells were harvested immediately after the first detectable       Reactions were carried out using about 10 nM RNA substrate and <br />       change in growth rate.                                                  0.25 mM RNase Y in a reaction volume of 10 ml. The buffer contained <br />                                                                               20 mM Hepes pH 8, 8 mM MgCl2, 0.48 U/ml RNAsin (Promega), <br />       Construction of the YmdA depletion strain                               100 mM NaCl and the reactions were incubated for 10 min at 301C. <br />       Plasmid pHMD1 was used to make the ymdA depletion strain: a             For time course experiments, samples of scaled-up reactions (50 ml) <br />                                                                               were taken at the indicated times. Reactions were stopped by the <br />       500 bp fragment was PCR amplified using oligonucleotides HP1113/ <br />                                                                               addition of 5 ml stop buffer (87.5% formamide, 0.05% xylene <br />       HP1114, digested SalI and SphI and ligated into the respective sites <br />       of plasmid pDG648 (kindly provided by P Stragier) downstream            cyanol, 0.05% bromophenol blue, 5 mM EDTA) and samples were <br />       of Pspac promoter. Campbell-type integration of pHMD1 on the            loaded on 5 or 8% denaturing polyacrylmide gels. The RNA <br />       B. subtilis chromosome rendered ymdA expression IPTG dependent.         substrates were synthesized with T7 RNA polymerase from PCR <br />                                                                               products (primer pairs HP1198/HP1238 and HP1198/HP1356). In <br />                                                                               both cases, the mRNA transcripts contained an additional 50 G <br />       Purification of YmdA                                                    residue when compared with the wild-type mRNA. The in vitro <br />       Plasmid pHMD3 was used to overexpress YmdA: the coding                  transcripts were either 50 end-labelled with g-[32P] ATP using T4 <br />       sequence of the ymdA gene excluding the first 25 residues               polynucleotide kinase following dephosphorylation, continuously <br />       comprising a transmembrane domain was PCR amplified using               labelled with a-[32P] UTP or 30 end-labelled with [32P]-pCp using T4 <br />       oligonucleotides HP1182/HP754 and cloned in phase with the intein       RNA ligase. 50 monophosphorylated transcripts were synthesized <br />       domain in plasmid pKYB1 (New England Biolabs) to give plasmid           by including a 50-fold excess of GMP over GTP in the reaction. <br />       pHMD3. YmdA was isolated as intein fusion protein using the                To test the effect of SAM on mRNA cleavage, SAM (25 and <br />       IMPACT system (New England Biolabs). Overexpression of the              80 mM) was added to the RNA substrate and the mixture was <br />       recombinant proteins was carried out in strain SSC91. Purified          incubated at 701C for 10 min and slowly cooled to 371C, before the <br />       YmdA was heated 10 min at 451C before use to inactivate the             addition of RNase Y. <br />       thermosensitive RNase E that might potentially contaminate the             A 50 end-labelled 50 bp DNA ladder was used as a size marker in <br />       preparation.                                                            most experiments. <br />  <br />       Site-directed mutagenesis                                               Chemical decay of bulk mRNA <br />       The H368A and D369A mutations in the YmdA HD domain were                Total mRNA decay was measured as described earlier (Wang and <br />       introduced on plasmid pHMD3 using the Quikchange (Stratagene)           Bechhofer, 1996; Even et al, 2005). <br />       strategy (oligonucleotides HP1298, HP1299, HP1300, HP1301) and <br />       KOD DNA polymerase (Novagen).                                            <br />       Northern blot analysis <br />       Northern blot analysis was carried out as described (Putzer et al, <br />       1992) using 6 mg of total RNA separated either on a 1.2% agarose or    <br />       8% acrylamide gel. A continuously a-[32P] UTP-labelled riboprobe        <br />       was generated by T7 in vitro transcription of a PCR fragment made       <br />       with the primers HP1231 (50 end corresponds to position &#195;&#190; 152)         <br />       and HP1058 (corresponds to HP1056 plus the upstream T7 promoter          <br />       sequence) was used to detect both the terminated yitJ leader and        <br />       full-length transcript. 50 [32P]-labelled oligonucleotides (listed in   <br />       Supplementary Table I) were used as probes to specifically detect      <br />       the leader transcripts of all 11 S-box genes.                        <br />  <br />       Primer extension analysis <br />       A measure of 5 mg of total RNA was subjected to reverse               <br />       transcription analysis by extending the 50 [32P]-labelled primers <br />       with Superscript III reverse transcriptase (Invitrogen).               <br />  <br />  <br />  <br /> </body></html>