<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Sigurdsson et al. BMC Systems Biology 2010, 4:140 <br /> http://www.biomedcentral.com/1752-0509/4/140 <br />  <br />  <br />  <br />  <br />  RESEARCH ARTICLE                                                                                                                               Open Access <br />  <br /> A detailed genome-wide reconstruction mouse <br /> metabolism based human Recon 1 <br /> Martin I Sigurdsson1,2,3, Neema Jamshidi4, Eirikur Steingrimsson1,3, Ines Thiele3,5*, Bernhard &#195;&#732; Palsson3,4* <br />  <br />  <br />   <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />   Background: Well-curated validated network reconstructions extremely valuable tools systems biology. <br />   Detailed metabolic reconstructions mammals recently emerged, including human reconstructions. They <br />   raise question various successful applications microbial reconstructions replicated complex <br />   organisms. <br />   Results: We mapped published, detailed reconstruction human metabolism (Recon 1) mammals. By <br />   searching genes homologous Recon 1 genes mammalian genomes, able create draft <br />   metabolic reconstructions mammals, including mouse. Each draft reconstruction created in <br />   compartmentalized non-compartmentalized version different approaches. Using gap-filling algorithms, <br />   able produce cellular components versions mouse metabolic <br />   reconstruction. We finalized functional model iterative testing passed predefined set 260 validation <br />   tests. The reconstruction largest, comprehensive mouse reconstruction <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span>-date, accounting 1,415 <br />   genes coding 2,212 gene-associated reactions 1,514 non-gene-associated reactions. <br />   We tested mouse model <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> prediction capabilities. The majority predicted essential genes were <br />   essential vivo. However, non-tissue specific model unable predict gene essentiality of <br />   metabolic genes shown essential vivo. Our <span id='am-239' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-240' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-241' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-258' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-259' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-272' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-278' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-296' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-305' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span>knockout</span> simulation lipoprotein lipase gene corre- <br />   lated experimental results, suggesting softer phenotypes simulated. <br />   Conclusions: We created high-quality mouse genome-scale metabolic reconstruction, iMM1415 (Mus <br />   Musculus, 1415 genes). We demonstrate mouse model used perform phenotype simulations, <br />   similar models microbe metabolism. Since mouse important experimental organism, model <br />   essential tool studying metabolic phenotypes mice, including outcomes drug <br />   screening. <br />  <br />  <br /> Background                                                                            [5]. The resulting model condition-specific be <br /> The genome-scale reconstruction metabolic net-                               used phenotype simulations using various con- <br /> works emerged years ago [1], years                              straint-based reconstruction analysis (COBRA) <br /> first genome sequencing entire organism                                   methods [5,6]. This approach proven successful for <br /> was published [2]. To date, 29 bacteria, 2 archaea 5                              various microorganisms eukaryotes addressing <br /> eukaryotes reconstructed organ-                                various biological biotechnological questions, such <br /> isms, 5 updates published [3]. The                                    analysis knowledge gaps [7], simulation phe- <br /> reconstruction process established metabolic                              notype traits [8], analysis evolution metabolic net- <br /> networks [4]. Once assembled, reconstruction                                  works [9,10] metabolic engineering applications <br /> be readily converted mathematical format add-                               [11]. The numerous applications recently been <br /> ing balances (e.g., mass -balance constraints), steady-                               reviewed [3,12]. <br /> state assumptions bounds (e.g. physical constraints)                                The release human genome [13] annota- <br />                                                                                       tion provided appropriate foundation human <br /> * Correspondence: ithiele@hi.is; bpalsson@bioeng.ucsd.edu                             metabolic reconstructions. Three approaches been <br /> 3 <br />  Center Systems Biology, University Iceland, Reykjavik, Iceland                published date. Two those, HumanCyc [14] <br /> Full list author information available end article <br />  <br />                                          &#194;&#169; 2010 Sigurdsson et al; licensee BioMed Central Ltd. This Open Access article distributed terms Creative <br />                                          Commons Attribution License (http://creativecommons.org/licenses/by/2.0), permits unrestricted use, distribution, and <br />                                          reproduction medium, provided original work properly cited. <br />  Sigurdsson et al. BMC Systems Biology 2010, 4:140                                                          Page 2 13 <br /> http://www.biomedcentral.com/1752-0509/4/140 <br />  <br />  <br />  <br />  <br /> and Edinburgh Human Metabolic Network [15,16]            reconstruction manually completed vali- <br /> were created largely automated top-down               dated. We present simulation results using func- <br /> approach, Recon 1 [17] created exten-        tional model mouse metabolism test phenotype <br /> sive bottom-up manual curation automated draft         prediction capabilities. <br /> map, including extensive gap-assessment filling <br /> using bibliomic data. Recon 1 compartments         Results <br /> (cytoplasm, extracellular space, mitochondria, Golgi         Identification homologous genes <br /> apparatus, endoplasmic reticulum, lysosome, peroxisome       The HomoloGene database contains information on <br /> and nucleus) accounts 1496 genes coding          homologous genes 5 mammals 15 non-mammals <br /> 2004 proteins [17]. Overall, Recon 1 covers major        [28]. We searched HomoloGene database all <br /> metabolic pathways occurring human cell            1496 genes Recon 1 human match for <br /> accounting 2,233 biochemical transformations         1,464 genes (97.8%). The mammalian organism the <br /> 1,510 transport exchange reactions. The Recon 1          highest number genes homologous Recon 1 genes <br /> can converted functional silico model       mouse (Mus musculus) (1,415 genes, 97%). The <br /> be converted functional model human metabo-        non-mammalian organism highest number of <br /> lism, validated applying 288 flux balance       genes homologous Recon 1 genes zebra-fish <br /> analysis (FBA) based tests core functionality          (Danio rerio) (1,200 genes, 82%) (Additional file 1). <br /> human metabolism [17]. A recent use Recon 1 <br /> includes study topology human disease find-      Creation draft metabolic network reconstructions <br /> ing co-morbid diseases functional         For mammal, draft reconstruction created <br /> relationship Recon 1 [18]. Also, expression profil-   different approaches (Figure 1a). In approach A <br /> ing data mapped Recon 1 used create cel-       (modelA), reactions linked Recon 1 genes did <br /> lular network human tissue types [19].               <span id='am-473' about='Thesaurus:Homologous_Gene' typeof='owl:Thing'>homologous gene</span> removed the <br /> Furthermore systems based analysis effects       reconstruction corresponding species. Therefore, <br /> imprinted genes metabolic functions recently          modelA included reactions linked genes homolo- <br /> been published [20].                                         gous Recon 1 genes addition non-gene asso- <br />    Metabolic networks reconstructed        ciated reactions (1,514). Of non-gene-associated <br /> further mammals, central metabolic reconstruction       reactions, 676 transporter reactions, 452 were <br /> cattle (Bos taurus) [21] mouse (Mus musculus)       demand exchange reactions, leaving 385 reactions <br /> reconstructions [22-25].Two mouse reconstruc-       metabolic pathway gene-associa- <br /> tions constructed manually [24,25],           tion. In approach B (modelB), included genes <br /> remaining generated semi-automatically              reactions homologues Recon 1 genes as <br /> [22,23]. The genome-scale manual curated recon-        transporter demand reactions. All other <br /> struction represents metabolism mouse hybri-          reactions, including non-gene associated reactions, were <br /> doma cell lines [25] recently expanded       removed reconstruction. <br /> represent 1494 reactions coded 724 genes           The ratio Recon 1 genes <span id='am-474' about='Thesaurus:Homologous_Gene' typeof='owl:Thing'>homologous gene</span> <br /> compartments [24]. In addition, pre-genome sequen-         ratio reactions Recon 1 included <br /> cing reaction network intermediate metabolism          draft reconstructions mammal shown <br /> mouse hybridoma cell lines built analyzed            Figure 1b. The mammal highest ratio of <br /> using linear optimization [26].                              mapped genes (97%) reactions (98%) mouse <br />    The mouse serves fundamental experimental ani-       chimpanzee (Pan troglodytes) lowest <br /> mal human biomedical applications. Furthermore,          ratio mapped genes (84%) reactions (91%). <br /> the availability phenotyped inbred knockout mouse         Furthermore, created mammalian <br /> strains [27] make ideal examining phenotype       modelA modelB compartmentalized non- <br /> prediction capabilities mammalian reconstructions.        compartmentalized version. The complex, com- <br /> Therefore, detailed reconstruction mouse metabo-        partmentalized versions metabolic dead ends <br /> lism converted highly functional model      (321 vs. 150) reactions zero <br /> is great importance.                                    fluxes. The number reactions non-zero fluxes <br />    Given high sequence homology             compartmentalized non-compartmentalized <br /> mammalian genomes, obvious starting point             versions draft mammalian models created using <br /> metabolic reconstruction mammals comprehen-        approach A B summarized Table 1. We per- <br /> sive human metabolic reconstruction, Recon 1. We             formed flux variability analysis (FVA) models these <br /> therefore sought create draft mammalian metabolic         reconstructions, allowing uptake extracellular <br /> reconstructions based Recon 1. The mouse                  metabolites models order maximize <br />  Sigurdsson et al. BMC Systems Biology 2010, 4:140                                                                                           Page 3 13 <br /> http://www.biomedcentral.com/1752-0509/4/140 <br />  <br />  <br />  <br />  <br />  Figure 1 Creation draft mammalian reconstructions. a) A schematic figure showing approaches used generate draft <br />  mammalian reconstructions using Recon 1. Approach A removes gene-associated reactions Recon 1 <span id='am-475' about='Thesaurus:Homologous_Gene' typeof='owl:Thing'>homologous gene</span> the <br />  reconstructed animal, keeping non-gene-associated reactions. Approach B removes gene-associated reactions Recon 1 without <br />  <span id='am-476' about='Thesaurus:Homologous_Gene' typeof='owl:Thing'>homologous gene</span> reconstructed non-gene-associated reactions (excluding transporters demand reactions). GAR - gene- <br />  associated-reactions; nGAR - non-gene-associated reaction; transp/dem - transporters demand reactions. b) Ratio reactions (black bar) and <br />  genes (gray bar) successfully mapped Recon 1 indicated mammalian draft reconstruction. c) A phylogenetic tree based on <br />  transcripts protein domain sequences SuperFamily database [64] reconstructed mammals. d) A phylogenetic tree based on <br />  flux variability analysis (FVA) reactions mammals reconstructed approach A. e) A phylogenetic tree based flux variability analysis <br />  (FVA) reactions mammals reconstructed approach B. <br />  Sigurdsson et al. BMC Systems Biology 2010, 4:140                                                                             Page 4 13 <br /> http://www.biomedcentral.com/1752-0509/4/140 <br />  <br />  <br />  <br />  <br /> Table 1 Results flux variability analysis draft                     reactions existed humans 1 (1%) reactions <br /> mammalian models compared human model (H.                                existed mouse (Additional file 2). For 25 <br /> sapiens, Recon 1)                                                               remaining Recon 1 subsystems (excluding transporters), <br />                 Approach                       Approach                         did extensive literature search differences <br />                 A                              B                                species. Of those, cytochrome P <br />                 Compartm. Non-                 Compartm. Non-                   450 metabolism reported differ significantly <br />                           Compartm.                      Compartm. <br />                                                                                 mouse human [32]. These defined differ- <br /> Organism        N       %      N       %       N      %       N        % <br />                                                                                 ences guide subsequent gap filling process. <br /> H. sapiens      3189           2037            957            1280              However, metabolic functions seemed <br /> P. troglodytes 1566     49%    1074    53%     394    41%     688      54%      present human mouse, employ <br /> R. norvegius    2113    66%    1503    74%     726    76%     911      71%      validation tests designed evaluate the <br /> M. musculus     2753    86%    1720    84%     851    89%     1127     88%      predictive potential Recon 1 [17]. <br /> B. taurus       1716    54%    1129    55%     556    58%     787      61% <br /> C. lupus fam.   1971    62%    1249    61%     627    66%     842      66%      Gap filling mouse metabolic reconstruction <br /> Absolute number (N) percentage (%) active reactions (with   First, decided test mouse meta- <br /> non-zero flux) compartmentalized non-compartmentalized <br /> models created using approach A B. <br />                                                                                 bolic models able produce biomass optimiz- <br />                                                                                 ing corresponding reaction accounting all <br /> likelihood non-zero fluxes. Compared humans,                          known biomass precursors required cell replication. <br /> mouse model highest percentage (84-89%)                             The compartmentalized non-compartmentalized <br /> the chimpanzee model lowest percentage (41-                             draft mouse models approach A produced biomass <br /> 54%) active reactions non-zero fluxes                           modifications. In contrast, versions of <br /> model versions (Table 1). We constructed phylo-                          modelB unable produce biomass. Therefore, we <br /> genetic tree reconstructed mammals based                          sought gap draft non-compartmentalized <br /> FVA results (Figure 1d,e). The results indicated                       modelB identify reactions needed added to <br /> mouse draft model highest degree similarity                          model production biomass components. <br /> with human model modelA (Figure 1d)                            The SMILEY gap filling algorithm identifies reactions <br /> modelB (Figure 1e). These findings surprising                        database need added model to <br /> chimpanzee closest relative humans                               fulfill optimality condition (e.g. production bio- <br /> approximately 99% overall genome similarity [29]. It                         mass) [7]. Using SMILEY algorithm, searched for <br /> also closest ancestor humans mammals                              minimal number reactions entire <br /> reconstructed protein coding sequence com-                              KEGG database necessary add non-compart- <br /> pared (Figure 1c).                                                              mentalized mouse modelB order enable pro- <br />                                                                                 duction biomass. <br /> Identification unique metabolic functions mouse                         The results 40 iterations algorithm were <br /> Given high number reactions mapped draft                          checked manually criteria: i) result did not <br /> mouse reconstruction, high number reactions                          suggest reversible reaction known irreversible <br /> with non-zero fluxes resulting corresponding                             reaction ii) added reaction(s) known to <br /> metabolic modelA modelB, sought finalize                              occur mouse. Of 40 iterations, 30 suggested add- <br /> the mouse reconstruction gap analysis/filling. First,                        ing reaction addition to <br /> we aimed understand metabolic differences                                metabolic reactions. This reaction (KEGG <br /> between human mouse ensure mouse                                ID R06522) exists mouse humans (according to <br /> reconstruction merely modified version                              KEGG Entrez gene [33]) included in <br /> human metabolism. This implies genes                              Recon1. It phosphohydrolase reaction involved in <br /> pathways unique mouse metabolism needed                                sphingolipid metabolism. Adding reaction resulted <br /> identified. Therefore, employed Comparative                              ability refined non-compartmentalized <br /> Pathway Analyzer 1.0 [30] relying KEGG data-                             modelB model produce biomass similar rate as <br /> <span id='am-477' about='xsp:base' typeof='owl:Thing'>base</span> [31] extract metabolic maps displaying exis-                        non-compartmentalized modelA. The addition did, <br /> tence enzymes mouse human 66                             however, result biomass production com- <br /> of 99 Recon 1 subsystems (Additional file 2). We                           partmentalized modelB. Subsequently, decided to <br /> used maps perform manual search poten-                            focus remainder study modelA, cap- <br /> tial gaps human mouse metabolism. Out                            tures known metabolic capabilities the <br /> 1550 reactions present subsystems, 1492                            mouse. ModelB developed is <br /> (96%) reactions existed species, 46 (3%)                                missing significant fraction metabolic reactions and <br />                                                                                 does function. <br />  Sigurdsson et al. BMC Systems Biology 2010, 4:140                                                            Page 5 13 <br /> http://www.biomedcentral.com/1752-0509/4/140 <br />  <br />  <br />  <br />  <br /> Validation mouse metabolic reconstruction             included reactions mouse reconstruction were <br /> To validate mouse reconstruction ensure           steroid metabolism (16 46 reactions) and <br /> mouse-characteristic metabolic properties result-     blood group biosynthesis (14 46 reactions). How- <br /> ing models, manually tested 288 FBA           ever, largest percentage non-included reactions <br /> validation tests developed Recon 1 [17]        subsystems stilbene, coumarine lig- <br /> using versions modelA. Our literature survey      nin biosynthesis (50%, 1 2 reactions) limonene <br /> mouse metabolism, focusing mouse essenti-        pinene degradation (50%, 3 6 reactions). This <br /> ality 288 human tests, revealed tests    data suggests simulation results metabo- <br /> essential human metabolism. Furthermore          lism subsystems likely inaccurate and <br /> we evidence mouse-specific additional tests      points areas future improvement mouse <br /> of essentiality added validation pro-   model. In remainder paper, use models <br /> cess. Therefore, evaluated manually required       derived iMM1415 computations. <br /> reactions test present mouse metabo- <br /> lism using Comparative Pathway Analyzer 1.0. A total      Comparison iMM1415 published mouse metabolic <br /> 260 tests passed requirements         networks <br /> used validation compartmentalized mouse           Table 2 provides comparison reconstruc- <br /> modelA (Additional file 3). For non-compartmenta-        tion published reconstructions, most <br /> lized version modelA, additional tests           detailed date recently published mouse <br /> removed compartment specific, resulting      metabolic network Selvarasu et al [24]. Our model <br /> in 254 validation tests.                                     functions cellular compartments compared to <br />    We ran validation tests compartmentalized      compartments reconstructions. It the <br /> mouse draft modelA iterative manner evalu-         detailed description mitochondria metabolism <br /> ated particularly failed tests. This process revealed    date fewer reactions cytosol. Furthermore <br /> that 131 reactions initially removed       iMM1415 contains attempt metabo- <br /> missing homologous genes HomoloGene data-             lism golgi apparatus, lysosome, ribosome, peroxi- <br /> base, 36 sequence physiological evidence          nucleus. Due compartmentalization, a <br /> existence mouse (according KEGG Entrez-        greater number transport reactions was <br /> Gene databases) (Additional file 4). Those 36 reactions      required reconstruction compared previous <br /> were added model,             ones. As biochemical data exists transport reac- <br /> remaining 95 reactions left final model      tions larger number non-gene associated reactions <br /> due missing physiologic sequence evidence. This        needed model models. <br /> addition resulted finalized mouse modelA           Albeit difficult compare directly different defi- <br /> passed 260 validation tests. Also, added      nition minimum growth medium biomass, our <br /> reaction discovered gap filling modelB (KEGG       model predicts fewer essential genes higher per- <br /> ID R06522). Furthermore, unique mouse reactions              centage experimentally verified than <br /> which lead metabolic dead ends model        models published date (Table 2). <br /> were added (KEGG IDs R03184, R00647 R01465). <br /> The non-compartmentalized modelA similarly passed            Essentiality mouse metabolic genes <br /> 100% 254 validation tests addition       Given high degree functionality iMM1415 we <br /> these reactions. We determined functionality     decided employ phenotype simulations to <br /> modelB, compartmentalized version            compare silico results published experimental <br /> cannot produce biomass. The compartmentalized            results knockout mice mice mutations in <br /> non-compartmentalized models created approach B          metabolic genes. First, performed simulation single <br /> passed 50% 85% validation tests respectively.     gene knockouts genes iMM1415 order to <br />                                                              determine silico gene essentiality. A minimal growth <br /> Properties iMM1415                                        medium supplemented glucose used this <br /> The resulting compartmentalized mouse metabolic              simulation (as defined additional file 7). A total 53 <br /> reconstruction (Additional file 5), created curation   genes essential deletion resulted <br /> modelA, termed iMM1415, contains total 3,724 reac-      zero biomass production (Additional file 8). We found <br /> tions 1,415 metabolic genes (Additional file 6 con-      information homozygous knockout phenotype 17 <br /> tains detailed description reactions iMM1415,      genes literature (Table 3). Of those, 14 (82%) <br /> including biomass reaction). Analysis reactions   genes confirmed <span id='am-395' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-396' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-397' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-398' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-399' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-400' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-401' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-402' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-403' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-404' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-405' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-406' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-407' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-408' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-409' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-410' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-411' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-412' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-413' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-414' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-415' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-416' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-417' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-418' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-419' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-420' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-421' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-422' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-423' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-424' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-425' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-426' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-427' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-428' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-429' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-430' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-431' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-432' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-433' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-434' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-435' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-436' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-437' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-438' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-439' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-440' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-441' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-442' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-443' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-444' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-445' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-446' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-447' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-448' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-449' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-450' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-451' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-452' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-453' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-454' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-455' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-456' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-457' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-458' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-459' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-460' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-461' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-462' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-463' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-464' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-465' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-466' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-467' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-468' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-469' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-470' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-471' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-472' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span>lethal phenotype</span>, the <br /> removed Recon 1 creation iMM1415          remaining (18%) non-lethal phenotypes (Table <br /> revealed greatest absolute number non-           3). The majority genes predicted and <br />  Sigurdsson et al. BMC Systems Biology 2010, 4:140                                                                                      Page 6 13 <br /> http://www.biomedcentral.com/1752-0509/4/140 <br />  <br />  <br />  <br />  <br /> Table 2 Properties iMM1415 comparison existing models <br />                                              iMM1415   Selvarasu et al (2010).    Evsikov et al (2009).   Quek et al (2008).    Sheikh et al (2005). <br /> Dominant reconstruction method               Manual             Manual                  Automatic              Automatic              Manual <br /> Genes                                         1,415              724                      1,060                  1,399                  473 <br /> Reactions                                     3,724              715                      2,018                  1,757                   0 <br /> Compounds                                     2,774              1,285                    1,377                  2,104                  872 <br /> Compartments                                    8                  3                        1                      3                     3 <br /> Reactions                                     3,726              1,494                    2,018                  2,037                 1,220 <br />     Demand&amp; Exchange&amp; Biomass reactions        455                 0                        0                      0                     0 <br />                                  Cytosol       931               1,085                     NA                    1,650                  618 <br />                            Mitochondrial       409                161                      NA                     387                    12 <br />                              Extracellular     473                NA                       NA                     NA                    NA <br />                                     Golgi      250                NA                       NA                     NA                    NA <br />                                 Lysosome       194                NA                       NA                     NA                    NA <br />                                 Ribosome       184                NA                       NA                     NA                    NA <br />                              Peroxisome        95                 NA                       NA                     NA                    NA <br />                                  Nucleus       88                 NA                       NA                     NA                    NA <br />                                 Transport     1,101              248                        0                      64                   267 <br /> Non-gene associated reactions                 1,514              291                        0                     148                   324 <br /> Essential genes according modela            53a               109a                      NA                     NA                    72 <br /> Of experimental data available           17                 20                       NA                     NA                    NA <br /> % match experimental data                   82%               70%                       NA                     NA                    NA <br /> Produces biomass                                1                  1                        0                      1                     1 <br /> a <br />  &#226;&#8364;&#339;minimum growth&#226;&#8364;? medium, differences exist biomass minimum growth medium definitions publications. <br /> NA - available. <br />  <br />  <br /> confirmed essentiality status cholesterol                    Prediction normal phenotypes <br /> metabolism. This result indicates cholesterol metabo-                   To evaluate predictive potential iMM1415, <br /> lism especially vulnerable mutations environmen-                   identified genes Mouse Genome Infor- <br /> tal insults, inability cholesterol                 matics database [27], null <span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span>mutation</span> resulted <br /> metabolism overcome disruption metabolic reactions                     normal phenotype. A total genes were <br /> via alternative pathways.                                                    present mouse reconstruc- <br /> Second, searched Mouse Genome Informatics                             tion (OCRL, ACO1, PAFAH1B3, PGM1, FUT9, RHBG, <br /> database [27] mouse reconstruction genes                         ITPKC, SORD). For those, gene deletion no <br /> phenotype described word &#226;&#8364;&#339;lethality&#226;&#8364;?.                      effect model, isozymes existed the <br /> Furthermore, required genes homozy-                       reconstruction. For remaining genes (PGM1, <br /> gous <span id='am-317' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-318' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-319' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-320' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-321' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-322' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-323' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-324' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-325' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-326' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-327' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-328' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-329' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-330' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-331' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-332' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-333' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-334' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-335' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-337' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-338' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-341' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-342' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-343' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-344' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-347' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-348' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-349' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-350' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-351' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-352' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-353' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-354' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-355' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-356' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-357' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-358' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-359' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-362' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-363' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-364' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-365' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-366' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-367' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-368' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-369' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-370' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-372' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-373' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-374' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-375' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-376' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-378' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-380' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-381' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-382' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-383' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-384' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-385' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-386' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-387' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-389' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-390' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-391' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-392' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-393' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-394' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span>gene knockout</span> embryonic                      FUT9, SORD), deletion led elimination corre- <br /> prenatal lethal phenotype. A total 88 genes iden-                    sponding reactions. The effect deletions on <br /> tified way. Five genes overlapped silico                    growth depends i) presence alternative reac- <br /> essential genes (FDFT1, SPTLC1, PHGDH, HMGCR,                                tions/pathways ii) importance reactions bio- <br /> CBS, SPTLC2). The observed discrepancy lethality                      mass precursor synthesis. We investigated effect of <br /> be explained either, i) growth environment simula-                        gene deletions overall network by <br /> tions, ii) incomplete biomass reaction, iii) missing regula-                 performing FVA. We compared FVA results <br /> tion, iv) wrongly included reactions, v)                              essential gene knockout (DHCR7) FVA results <br /> combination aforementioned possibilities. Wrongly                     wild type (Table 4) considering major sec- <br /> included reactions identified systematically                     tions metabolism (Table 4). The largest perturbation <br /> eliminating non-gene-associated reactions                               resulted knockout simulation essential <br /> iMM1415 (e.g., using GrowMatch algorithm [34])                            gene DHCR7, seven major subsections metabo- <br /> improve prediction lethality. In contrast,                    lism changed significantly. The knockout simulation of <br /> genes essential silico vivo non-essen-                   PGM1 led significant changes major sub- <br /> tial growth suggest missing functions metabolic                   sections metabolism. The knockout simulation the <br /> network. Using SMILEY, related algorithms,                       FUT9 gene led significant changes major <br /> be possible identify missing candidate genes.                             subsections metabolism knockout simulation <br />  Sigurdsson et al. BMC Systems Biology 2010, 4:140                                                                                       Page 7 13 <br /> http://www.biomedcentral.com/1752-0509/4/140 <br />  <br />  <br />  <br />  <br /> Table 3 Results gene essentiality predictions finalized mouse model <br />  Gene                          Reactions                               Subgroup           Mutation             <span id='am-3' about='rdfs:comment' typeof='owl:Thing'>Comment</span>                   Reference <br />  Name                                                                                      lethal <br />   EBP       3-beta-hydroxysteroid-delta(8),delta(7)-isomerase          Cholesterol          yes      X-linked genes, homozygous         Means et al. <br />                                                                        Metabolism                    males non-viable (prenatal <br />                                                                                                                lethality) <br /> DHCR7                7-dehydrocholesterol reductase                    Cholesterol          yes           Prenatal lethality           Yu et al. <br />                                                                        Metabolism                            homozygotes <br /> DHCR24              24-dehydrocholesterol reductase                    Cholesterol          yes           Prenatal lethality          Mirza et al. <br />                                                                        Metabolism                            homozygotes <br />  FDFT1                     Squalene synthase                           Cholesterol          yes           Prenatal lethality         Tozawa et al. <br />                                                                        Metabolism                            homozygotes <br /> HSD17B4     C-3 sterol keto reductase, Beta oxidation long          Cholesterol          yes      Pre/Peri/Postnathal lethality   Huyghe et al. <br />            chain fatty acid, 3-hydroxyacyl-CoA dehydrogenase,          Metabolism                             homozygotes <br />                hydroxysteroid (17-beta) dehydrogenase 4, <br />            peroxisomal lumped long chain fatty acid oxidation <br /> NSDHL     C-3 sterol dehydrogenase, C-4 methyl sterol oxidase          Cholesterol          yes        X-linked gene, males         Cunningham <br />                                                                        Metabolism                    homozygous females non-           et al <br />                                                                                                        viable (prenatal lethality) <br />  SC5DL                     Lathosterol oxidase                         Cholesterol          yes           Perinatal lethality        Krakowiak et <br />                                                                       Metabolism                             homozygotes                     al. <br /> SPTLC1    serine palmitoyltransferase, long chain base subunit 1      Sphingolipid          yes          Embryonic lethality         Hojjati et al. <br />                                                                       Metabolism                             homozygotes <br />  DHFR           dihydrofolate reductase, folate reductase          Folate Metabolism        yes          Embryonic lethality          Di Pietro et <br />                                                                                                             homozygotes                       al. <br />   PISD      phosphatidylserine decarboxylase, mitochondrial        Glycerophospholipid      yes          Embryonic lethality         Steenbergen <br />                                                                        Metabolism                           homozygotes                     et al. <br /> PHGDH              phosphoglycerate dehydrogenase                  Glycine, Serine,     yes          Embryonic lethality         Yoshida et al. <br />                                                                         Threonine                           homozygotes <br />                                                                        Metabolism <br /> HMGCR           Hydroxymethylglutaryl CoA reductase (ir)               Cholesterol          yes          Embryonic lethality         Tanaka et al. <br />                                                                        Metabolism                           homozygotes <br />   CBS       cystathionine beta-synthase, selanocystathionine           Methionine           yes      Homozygous mice die 5       Watanabe et <br />                              beta-synthase                             Metabolism                         weeks birth                 al. <br /> SPTLC2                serine C-palmitoyltransferase                   Sphingolipid          yes          Embryonic lethality         Hojjati et al. <br />                                                                       Metabolism                            homozygotes <br />   PAH          L-Phenylalanine,tetrahydrobiopterin:oxygen             Tyr, Phe, Trp                  Homozygous mice            http://www. <br />                              oxidoreductase                           Biosynthesis                    dysruptions gene       informatics. <br />                                                                                                                  viable                    jax.org <br /> TM7SF2                    C-14 sterol reductase                        Cholesterol                   Although mixture           http://www. <br />                                                                        Metabolism                       mutations lethal,        informatics. <br />                                                                                                      homozygotes mutations         jax.org <br />                                                                                                                 viable <br />  Gpam             glycerol-3-phosphate acyltransferase                Triacylglycerol              Homozygous mice viable        Howerton et <br />                                                                          Synthesis                                                          al. <br />  <br />  <br /> of SORD led significant changes major                       free fatty acids. Individuals born lipoprotein <br /> subsections metabolism. Therefore, suggest                        lipase deficiency elevated levels triglycerides and <br /> six genes likely notice-                    VLDL suffer recurrent episodes abdominal <br /> able metabolic phenotypes minimal medium                           pain pancreatitis eruptive xanthomas of <br /> condition presence isozymes alternative                         skin [35]. Mutations associated with <br /> reactions/pathways.                                                          increased risk ischemic heart disease man [36]. <br />                                                                              Mice lipoprotein lipase born greatly <br /> Analysis lipoprotein lipase deficiency silico                       elevated levels triglycerides VLDL, nur- <br /> phenotype                                                                    sing, triglyceride levels soon extremely high. <br /> Finally, sought simulate knockouts genes                       Heterozygotes null mutation LPL survive until <br /> softer phenotypes. The LPL gene encodes lipoprotein                          adulthood elevated triglyceride levels [37]. <br /> lipase (EC 3.1.1.34) [35], enzyme hydrolyzes chy-                       We simulated knockout LPL gene per- <br /> lomicrons low-density lipoproteins (VLDL)                           formed FVA determine effect gene deletion <br />  Sigurdsson et al. BMC Systems Biology 2010, 4:140                                                                                                 Page 8 13 <br /> http://www.biomedcentral.com/1752-0509/4/140 <br />  <br />  <br />  <br />  <br /> Table 4 Results Flux Variability Analysis (FVA) 3 knockout models genes confirmed normal <br /> phenotype (PGM1, FUT9, SORD) essential gene (DHCR7) <br />                                                    PGM1                          FUT9                          SORD                         DHCR7 <br />                                                   p-value                p-value              p-value                p-value <br /> Amino Acid Metabolism                      61      160    &lt;0.00001       85      133     0.00115      122       98     0.10560      68      152      &lt;0.00001 <br /> Carbohydrate Metabolism                    35      55      0.03501       48      48      1.00000       40       55     0.12380      22       75      &lt;0.00001 <br /> Cofactor Vitamin Metabolism            20      75     &lt;0.00001       53      45      0.41900       60       37     0.01953      25       71      &lt;0.00001 <br /> Energy Metabolism                          12      34     0.001180       19      27      0.23820       27       19     0.23820      13       32      0.00462 <br /> Glycan Metabolism                          60      283    &lt;0.00001      118      229    &lt;0.00001      166      182     0.39110      49      294      &lt;0.00001 <br /> Lipid Metabolism                          109      243    &lt;0.00001      165      189     0.20210      196      162     0.07234      128     225      &lt;0.00001 <br /> Nucleotide Metabolism                      64      103     0.00255       80      91      0.40020       87       90     0.82160      45      122      &lt;0.00001 <br /> Other Amino Acids Metabolism               6       19      0.00932       11      14      0.54850       12       13     0.84150       7       18      0.02781 <br /> Secondary Metabolites Metabolism           0        0      1.00000       0        0      1.00000        0       0      1.00000       0       0       1.00000 <br /> Transporters                              368      437     0.01502      397      422     0.38240      393      443     0.08376      337     479      &lt;0.00001 <br /> For reaction model, FVA determines minimum maximum flux resulting optimal solution objective function (here: biomass <br /> production). Only solutions resulted maximum biomass yield wild type model used analysis. By comparing flux <br /> range (the span vmax,i vmin,i reaction i) knockout model wild type model, reaction assigned status of <br /> decreased (down), non-changing increased (up) flux capacity (representing metabolic activity). For metabolism subsystem, number reactions with <br /> decreased increased flux capacity shown. Significance testing performing single value Chi-Square test 1 degree freedom <br /> comparing probability increased decreased flux capacity. Significant p-values (p &lt; 0.05/10) typeset bold. <br />  <br />  <br /> to network properties. The largest perturbation                             Initial analysis sequenced mammalian genomes <br /> observed i) glycan metabolism, 31                               revealed highest number genes orthologous <br /> reactions decreased 312 reactions increased flux                         Recon 1 genes mouse the <br /> capacity (p &lt; 10-16 probability decreased                        lowest number present chimpanzee (Table 1). <br /> and increased flux capacity); ii) lipid metabolism,                          This result surprising sequence homology <br /> where 100 reactions decreased 253 reactions                             human chimpanzee ~99% [29] the <br /> with increased flux capacity (p &lt; 10 -15                            sequence homology human mouse in <br /> probability decreased increased flux capacity).                           range 85-92% [52,53]. This observation reflects <br /> Additionally, flux capacity changed significantly                            fact enzymes involved human metabo- <br /> within triacylglycerol metabolism (Figure 2). The                            lism discovered analyzing mouse ortholo- <br /> results suggest increased flux capacity                            gues, leading high likelihood mouse metabolic <br /> towards triglyceride synthesis decreased                          enzymes known human present in <br /> flux capacity triglyceride degradation. The                              Recon 1. <br /> simulation results consistent hypertri-                          The initial search homologous genes indicated that <br /> glyceridemia resulting LPL mutations, pre-                               functional mouse reconstruction produced <br /> viously demonstrated mouse knockout model                               based Recon 1. Using different approaches, we <br /> [37-51].                                                                         created versions draft mouse model (Figure 1). <br />                                                                                  These draft models completed automated <br /> Discussion                                                                       manual gap filling analysis validated 260 <br /> Can success metabolic systems biology                                  FBA-based tests. This effort resulted iMM1415, <br /> microbes repeated complex organisms? A                               accounts genes reactions, distribu- <br /> prerequisite success existence high                           ted cellular compartments, previously <br /> quality metabolic network reconstructions. Here,                              published reconstructions (Table 2) addition its <br /> examined detailed validated network recon-                            representation cellular compartments. Thus, the <br /> struction human metabolism, Recon 1, used                            mouse metabolic reconstruction presented the <br /> as basis reconstructing metabolic networks                              comprehensive reconstruction available date. <br /> other mammals. We focused mouse create                              However, usage human Recon 1 reactions <br /> manual curated, comprehensive reconstruction mouse                            information (including cellular location reversibility) <br /> metabolism subsequently employed deter-                              risks reactions included skewed towards <br /> mine normal lethal growth phenotypes. Overall,                            human metabolism reactions unique the <br /> found good agreement model&#226;&#8364;&#8482;s prediction                              mouse missed. <br /> and reported mouse phenotypes, suggesting                                  A surprisingly high number non-gene-associated <br /> mouse reconstruction similar quality properties                           reactions included Recon 1. Many are <br /> Recon 1.                                                                         intra-cellular transport reactions, demand exchange <br />  Sigurdsson et al. BMC Systems Biology 2010, 4:140                                                                                     Page 9 13 <br /> http://www.biomedcentral.com/1752-0509/4/140 <br />  <br />  <br />  <br />  <br />  Figure 2 Results comparison flux variability analysis (FVA) LPL knockout model wild type model. Results <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br />  comparison flux variability analysis (FVA) LPL knockout model wild type model. a) A triacylglycerol metabolism is <br />  shown. Reactions increased flux capacity knockout model shown red reactions decreased flux capacity shown in <br />  green. b) The distribution fluxes FVA wild type model (dark gray) compared knockout model (black) indicates increased flux <br />  diacylglycerol acetyltransferase (DGAT) monoacylglycerol acetyltransferase (MOGAT). These results suggest that <br />  hypertriglyceridemia result knockout LPL gene. Reactions: AGPAT1 - 1-acylglycerol-3-phosphate O-acyltransferase 1; DGAT - <br />  diacylglycerol acyltransferase; GPAM - glycerol-3-phosphate acyltransferase; LPS - lipoprotein lipase LPS2 - lipoprotein lipase 2; LPS3 - lipase; <br />  MOGAT - monoacylglycerol acyltransferase; PPAP - phosphatidic acid phosphatase. Metabolites: a-lysophosphatidic acid; DAG - diacylglycerol <br />  (Homo sapiens); G - glycerol; G3p - Glycerol 3-phosphate; MAG - monoacylglycerol 2; PA - phosphatidic acid; R - R groups (total); TAG - <br />  triacylglycerol. <br />  <br />  <br /> reactions sparse literature supporting exis-                    biochemical activity characterized the <br /> tence. They included provide functional                          encoding gene unknown organism [54]. Many <br /> model human metabolism cellular                        enzymes (13-37%) major metabolism <br /> compartments added                              pathways [55] suggesting metabolic reconstructions <br /> iMM1415. We feel gain highly functional                     well-annotated organisms set non- <br /> model inclusion supersedes risk arti-                 gene associated reactions order ensure cor- <br /> ficially inflating properties mouse reconstruc-                 rect functionality. This observation argues against <br /> tion. Furthermore, 30-40% known enzymes                             reconstructing analyzing metabolic networks based <br /> with EC number orphan meaning                            solely genomic data likely have <br />  Sigurdsson et al. BMC Systems Biology 2010, 4:140                                                            Page 10 13 <br /> http://www.biomedcentral.com/1752-0509/4/140 <br />  <br />  <br />  <br />  <br /> numerous missing metabolic functions. Also, non-            simulations resulted little perturbation the <br /> gene associated reactions extremely interesting          mouse metabolic network. Recently, shown for <br /> area future research suggest pathways            yeast deletion ~80% yeast genes <br /> enzymes require experimental exploration           resulted apparent phenotype grown rich <br /> [55]. Furthermore, reactions leading metabolic       medium, measurable growth phenotype observed <br /> dead ends model included Recon 1             ~97% yeast genes medium either <br /> subsequently mouse reconstruction. They high-            depleted certain ingredients biologically active <br /> light missing links knowledge form testable              compounds added medium [58]. Furthermore, <br /> hypotheses investigated. Any             virtually genes essential growth <br /> effort consensus metabolic reconstruction           conditions [58]. It important subject future <br /> mouse focus reviewing non-gene-asso-              research mammalian models suggest similar <br /> ciated reactions leading dead-ends          findings, analysis understanding of <br /> model detail.                                                complex genotypes effects environmen- <br />    Our approach theory used con-            tal factors disease pathogenesis. <br /> struct high quality metabolic reconstructions other <br /> mammals. Given results (Table 1), likely       Conclusions <br /> rat reconstruction obtained using similar            Here, created reconstruction mouse <br /> approach presented (Figure 1). However,             metabolism based sequence homology, using the <br /> resulting network expected complete, limit-       highly detailed Recon 1 human metabolism a <br /> ing application requiring manual curation.         basis reconstruction. The model rig- <br /> Metabolic reconstructions mammals           orously validated using 260 flux balance analysis based <br /> insufficient quality predictive potential using         tests. The resulting reconstruction, iMM1415, to <br /> presented approach low number homolo-             date comprehensive reconstruction mouse <br /> gous genes Recon1 genes.                                   metabolism. Our phenotype simulation results suggest <br />    The phenotype prediction properties metabolic mod-        current quality human Recon 1 <br /> els extensively applied microorganisms,           derived iMM1415 models sufficient for <br /> however, limited studies exists mammals                phenotype predictions. The existence two <br /> [20,56,57]. The mouse ideal model organism test-      reconstructions encourage creation of <br /> ing validating phenotypic properties multiple         detailed metabolic network reconstructions other <br /> inbred strains, thousand gene knockout strains      mammals. With ongoing international knockout <br /> various cell lines exist [27] including knockouts       mouse project http://www.knockoutmouse.org aiming <br /> metabolic genes. We majority predicted        produce knockout protein coding genes within <br /> essential genes lethal phenotype vivo. Inter-     mouse genome Collaborative Cross project <br /> estingly, majority silico essential genes    aiming crossbreed inbred strains mice [59], <br /> found cholesterol metabolism, indicating      validation mammalian metabolic models should <br /> the importance mutation vulnerability metabo-       extensive. Also, sequencing inbred <br /> lism pathway. This observation concordance           strains experimental mice provide new opportu- <br /> results based earlier mouse metabolic model,        nities studying effects genetic variability on <br /> the majority essential genes cholesterol           metabolism, utilizing mouse reconstruction a <br /> metabolism [24]. These results guide             data analysis platform. The existent emerging <br /> research helpful understanding human            reconstructions joined collaborative <br /> disease based lipid abnormalities, atherosclero-     effort reach consensus reconstruction the <br /> sis [24]. However, genes identified           mouse metabolism maximize accuracy utili- <br /> mouse knockout databases [27] result lethal          zation properties. Reconstruction jamboree meetings <br /> phenotype vivo non-lethal phenotype silico.     held various organisms obtain con- <br /> The disagreement result non-tissue specifi-      sensus metabolic reconstructions increase their <br /> city iMM1415 incomplete biomass reaction              content community driven approach [4,60]. Thor- <br /> highlights starting points future research   oughly validated genome-scale reconstructions should <br /> insight mouse metabolism.                                  provide broad platform studying mammalian <br />    The majority genes vivo experimen-        metabolism form basis developing <br /> tal knockout strain developed non-              therapeutic interventions related pathological meta- <br /> lethal phenotype. Similarly, silico knockout                 bolic states [38,42-44,48,51,61]. <br />  Sigurdsson et al. BMC Systems Biology 2010, 4:140                                                           Page 11 13 <br /> http://www.biomedcentral.com/1752-0509/4/140 <br />  <br />  <br />  <br />  <br /> Methods                                                      databases used determine candidate reac- <br /> All model calculations MATLAB version           tions gaps non-compartmentalized draft <br /> 2009a (The Mathworks Inc.) using COBRA toolbox           model. The gap filling algorithm set run 40 <br /> [6] Mosek linear solver (Mosek ApS, Denmark).        iterations. Each solution manually checked <br /> Statistical tests figure preparation R,     applicability (such feasibility sug- <br /> version 2.9.1 (The R foundation, Austria). In-house          gested directionality) choosing proposed <br /> scripts data processing written Java     solution. <br /> available request authors. <br />                                                              Validation <br /> Flux balance analysis flux variability analysis          For validation mouse reconstruction, used a <br /> Given reconstructed metabolic network mathe-          modification validation process used crea- <br /> matical form appropriate constraints reac-      tion human Recon 1 network. Each 288 <br /> tion (representing reversibility, maximum minimum        validation tests used Recon 1 manually checked <br /> flux etc.) biological objective Z (such biomass),   applicability mouse. This manu- <br /> flux balance analysis (FBA) calculates set network      ally reviewing validation test using Comparative <br /> fluxes maximize Z. It derived applica-      Pathway Analyzer 1.0 [30]. If enzymes participating <br /> tions used analyze flow metabolites        validation test choice existed human and <br /> through metabolic model [62]. One application      mouse, test used mouse validation. This <br /> flux variability analysis (FVA) [63]. FVA uses linear pro-   resulted 260 validation tests mouse model. <br /> gramming methods minimal maximal             The model improved manually iterative <br /> flux values achieved reaction      manner checking reactions involved failed <br /> contributing maximal production objective          tests passed validation tests. <br /> reaction. <br />                                                              Simulations <br /> Creation draft reconstructions                            Information mouse knockout phenotypes were <br /> The entire HomoloGene build 52 downloaded             gained Mouse Genome Informatics site (http:// <br /> 11/17/2008 [28]. We extracted data dataset      www.informatics.jax.org, accessed 1/12/2010) [27]. <br /> all 1,496 genes Recon 1 [17]. For 20 spe-     For gene knockout simulated, model <br /> cies HomoloGene database, checked         corresponding reaction bounds set zero was <br /> the availability homologous gene Recon 1       created. Both wild type knockout simulation <br /> gene. Next created draft reconstructions         type models allowed cellular uptake vital amino acids, <br /> species using different approaches described       vital fatty acids, glucose, oxygen, hydrogen, sulfur oxide, <br /> the result section. The draft reconstructions con-      phosphate ions optimizing biomass <br /> verted condition-specific draft models [4]. All draft   (detailed exchange constraints listing Additional <br /> models checked functionality optimizing      file 7). As computational methods allow direct <br /> biomass allowing unrestricted uptake metabo-        sampling flux solution space large networks <br /> lites (v min, metabolite &#226;&#8240;&#165; -1000 mmol/gdw hr). FVA       [65,66], performed FVA[63] estimate range of <br /> performed draft models determine number         flux values reaction resulting optimal <br /> reactions non-zero flux. Following FVA        solution. By comparing FVA wild type model <br /> draft model, drew phylogenetic trees results       knockout models, reaction assigned a <br /> and compared phylogenetic tree based tran-         status decreased flux capacity (i.e., reduced vmax,i), no <br /> scripts protein domain sequence SuperFam-        change increased flux capacity. For unbiased selec- <br /> ily database [64].                                           tion interesting metabolic subsystems, counted <br />                                                              number reactions increased decreased flux <br /> Gap filling                                                  ranges calculated single value Chi-Square test <br /> Gap filling using SMILEY algorithm,          1 degree freedom comparing observation <br /> described [7]. We downloaded used          probability increased decreased flux <br /> entire KEGG database (accessed 9/15/2009) [31]         capacity. Since independent tests of <br /> universal reaction database. Additionally, created      10 major subsystems, p-value 0.05/10 con- <br /> database transport exchange reaction          sidered statistically significant correct multiple <br /> metabolites Recon1 KEGG database. These           testing. <br />  Sigurdsson et al. BMC Systems Biology 2010, 4:140                                                                                                    Page 12 13 <br /> http://www.biomedcentral.com/1752-0509/4/140 <br />  <br />  <br />  <br />  <br />                                                                                    7.    Reed JL, Patel TR, Chen KH, Joyce AR, Applebee MK, Herring CD, Bui OT, <br /> Additional material                                                                      Knight EM, Fong SS, Palsson BO: Systems approach refining genome <br />                                                                                          annotation. Proc Natl Acad Sci USA 2006, 103:17480-17484. <br />  Additional file 1: Supplemental file S1: Absolute number                      8.    Nogales J, Palsson B&#195;&#732;, Thiele I: A genome-scale metabolic reconstruction <br />  percentage Recon 1 genes species                             Pseudomonas putida KT2440: iJN746 cell factory. BMC Syst Biol <br />  HomoloGene database.                                                                    2008, 2:79. <br />                                                                                    9.    P&#195;&#161;l C, Papp B, Lercher MJ, Csermely P, Oliver SG, Hurst LD: Chance and <br />  Additional file 2: Supplemental file S2: Details KEGG search                     necessity evolution minimal metabolic networks. Nature 2006, <br />  unique reactions mouse human using Comparative Pathway                           440:667-670. <br />  Analyzer 1.0.                                                                     10.   Zhang Y, Thiele I, Weekes D, Li Z, Jaroszewski L, Ginalski K, Deacon AM, <br />  Additional file 3: Supplemental file S3: List 260 flux balance                       Wooley J, Lesley SA, Wilson IA, Palsson B, Osterman A, Godzik A: Three- <br />  analysis tests used validation mouse reconstruction.                         dimensional structural view central metabolic network of <br />  Additional file 4: Supplemental file S4: List genes added                      Thermotoga maritima. Science (80-) 2009, 325:1544-1549. <br />  validation process corresponding metabolic                     11.   Park JH, Lee KH, Kim TY, Lee SY: Metabolic engineering Escherichia coli <br />  subgroups.                                                                              production L-valine based transcriptome analysis in <br />                                                                                          silico gene knockout simulation. Proc Natl Acad Sci USA 2007, <br />  Additional file 5: Supplemental file S5: The iMM1415 model                           104:7797-7802. <br />  mouse metabolism SMBL format.                                                12.   Oberhardt MA, Palsson B&#195;&#732;, Papin JA: Applications genome-scale <br />  Additional file 6: Supplemental file S6: A detailed reaction list                metabolic reconstructions. Mol Syst Biol 2009, 5:320. <br />  reactions iMM1415, including biomass reaction.                         13.   Lander ES, Linton LM, Birren B, Nusbaum C, Zody MC, Baldwin J, Devon K, <br />  Additional file 7: Supplemental file S7: Uptake rates exchange                      Dewar K, Doyle M, FitzHugh W, Funke R, Gage D, Harris K, Heaford A, <br />  reactions mouse reconstruction minimal medium                              Howland J, Kann L, Lehoczky J, LeVine R, McEwan P, McKernan K, <br />  conditions.                                                                             Meldrim J, Mesirov JP, Miranda C, Morris W, Naylor J, Raymond C, Rosetti M, <br />                                                                                          Santos R, Sheridan A, Sougnez C: Initial sequencing analysis the <br />  Additional file 8: Supplemental file S8: A list genes predicted                   human genome. Nature 2001, 409:860-921. <br />  essential mouse metabolic network,                           14.   Romero P, Wagg J, Green ML, Kaiser D, Krummenacker M, Karp PD: <br />  bibliographic information available.                                                 Computational prediction human metabolic pathways the <br />                                                                                          complete human genome. Genome Biol 2005, 6:R2. <br />                                                                                    15.   Hao T, Ma H, Zhao X, Goryanin I: Compartmentalization Edinburgh <br />                                                                                          Human Metabolic Network. BMC Bioinformatics 2010, 11:393. <br /> Acknowledgements                                                                   16.   Ma H, Sorokin A, Mazein A, Selkov A, Selkov E, Demin O, Goryanin I: The <br /> This work supported ERC grant agreement No. 232816 - SYSTEM_Us                    Edinburgh human metabolic network reconstruction functional <br /> between European Research Council Bernhard &#195;&#732;. Palsson                      analysis. Mol Syst Biol 2007, 3:135. <br /> Marie Curie International Reintegration Grant 7th European              17.   Duarte NC, Becker SA, Jamshidi N, Thiele I, Mo ML, Vo TD, Srivas R, <br /> Community Framework Programme.                                                           Palsson B&#195;&#732;: Global reconstruction human metabolic network <br />                                                                                          based genomic bibliomic data. Proc Natl Acad Sci USA 2007, <br /> Author details                                                                           104:1777-1782. <br /> 1 <br />  Department Biochemistry Molecular Biology, Faculty Medicine,            18.   Lee D, Park J, Kay KA, Christakis NA, Oltvai ZN, Barab&#195;&#161;si A: The implications <br /> University Iceland, Reykjavik, Iceland. 2Department Genetics                   human metabolic network topology disease comorbidity. Proc Natl <br /> Molecular Medicine, Landspitali-University Hospital, Reykjavik, Iceland. 3Center         Acad Sci USA 2008, 105:9880-9885. <br /> for Systems Biology, University Iceland, Reykjavik, Iceland. 4Department     19.   Shlomi T, Cabili MN, Herrg&#195;&#165;rd MJ, Palsson B&#195;&#732;, Ruppin E: Network-based <br /> Bioengineering, University California, San Diego, La Jolla, California 92093-         prediction human tissue-specific metabolism. Nat Biotechnol 2008, <br /> 0412, USA. 5Faculty Industrial Engineering, Mechanical Engineering &amp;                  26:1003-1010. <br /> Computer Science, University Iceland, Reykjavik, Iceland.                       20.   Sigurdsson MI, Jamshidi N, Jonsson JJ, Palsson BO: Genome-scale network <br />                                                                                          analysis imprinted human metabolic genes. Epigenetics 2009, 4:43-46. <br /> Authors&#226;&#8364;&#8482; contributions                                                             21.   Seo S, Lewin HA: Reconstruction metabolic pathways cattle <br /> MIS, NJ, ES, IT BOP conceived study. MIS, NJ IT created               genome. BMC Syst Biol 2009, 3:33. <br /> validated models. MIS performed phenotype simulations. MIS drafted             22.   Evsikov AV, Dolan ME, Genrich MP, Patek E, Bult CJ: MouseCyc: curated <br /> the manuscript. MIS, ES, IT BOP revised manuscript. All authors read             biochemical pathways database laboratory mouse. Genome Biol <br /> and approved final manuscript.                                                       2009, 10:R84. <br />                                                                                    23.   Quek L, Nielsen LK: On reconstruction Mus musculus genome- <br /> Received: 17 May 2010 Accepted: 19 October 2010                                          scale metabolic network model. Genome Inform 2008, 21:89-100. <br /> Published: 19 October 2010                                                         24.   Selvarasu S, Karimi IA, Ghim G, Lee D: Genome-scale modeling in <br />                                                                                          silico analysis mouse cell metabolic network. Mol Biosyst 2010, <br /> References                                                                               6:152-161. <br /> 1. Edwards JS, Palsson BO: Systems properties Haemophilus                   25.   Sheikh K, F&#195;&#182;rster J, Nielsen LK: Modeling hybridoma cell metabolism <br />     influenzae Rd metabolic genotype. J Biol Chem 1999, 274:17410-17416.                 using generic genome-scale metabolic model Mus musculus. <br /> 2. Palsson B: Metabolic systems biology. FEBS Lett 2009, 583:3900-3904.                  Biotechnol Prog 2005, 21:112-121. <br /> 3. Feist AM, Palsson B&#195;&#732;: The growing scope applications genome-scale         26.   Savinell JM, Palsson BO: Network analysis intermediary metabolism <br />     metabolic reconstructions using Escherichia coli. Nat Biotechnol 2008,               using linear optimization. II. Interpretation hybridoma cell <br />     26:659-667.                                                                          metabolism. J Theor Biol 1992, 154:455-473. <br /> 4. Thiele I, Palsson B&#195;&#732;: A protocol generating high-quality genome-          27.   Blake JA, Bult CJ, Eppig JT, Kadin JA, Richardson JE: The Mouse Genome <br />     scale metabolic reconstruction. Nat Protoc 2010, 5:93-121.                           Database genotypes::phenotypes. Nucleic Acids Res 2009, 37:D712-9. <br /> 5. Price ND, Reed JL, Palsson B&#195;&#732;: Genome-scale models microbial cells:          28.   Sayers EW, Barrett T, Benson DA, Bryant SH, Canese K, Chetvernin V, <br />     evaluating consequences constraints. Nat Rev Microbiol 2004,                  Church DM, DiCuccio M, Edgar R, Federhen S, Feolo M, Geer LY, <br />     2:886-897.                                                                           Helmberg W, Kapustin Y, Landsman D, Lipman DJ, Madden TL, Maglott DR, <br /> 6. Becker SA, Feist AM, Mo ML, Hannum G, Palsson B&#195;&#732;, Herrgard MJ:                        Miller V, Mizrachi I, Ostell J, Pruitt KD, Schuler GD, Sequeira E, Sherry ST, <br />     Quantitative prediction cellular metabolism constraint-based                 Shumway M, Sirotkin K, Souvorov A, Starchenko G, Tatusova TA: Database <br />     models: COBRA Toolbox. Nat Protoc 2007, 2:727-738.                               resources National Center Biotechnology Information. Nucleic <br />                                                                                          Acids Res 2009, 37:D5-15. <br />  Sigurdsson et al. BMC Systems Biology 2010, 4:140                                                                                                 Page 13 13 <br /> http://www.biomedcentral.com/1752-0509/4/140 <br />  <br />  <br />  <br />  <br /> 29. Winckler W, Myers SR, Richter DJ, Onofrio RC, McDonald GJ, Bontrop RE,         49. Watanabe M, Osada J, Aratani Y, Kluckman K, Reddick R, Malinow MR, <br />     McVean GAT, Gabriel SB, Reich D, Donnelly P, Altshuler D: Comparison            Maeda N: Mice deficient cystathionine beta-synthase: animal models <br />     fine-scale recombination rates humans chimpanzees. Science (80-)            mild severe homocyst(e)inemia. Proc Natl Acad Sci USA 1995, <br />     2005, 308:107-111.                                                                 92:1585-1589. <br /> 30. Oehm S, Gilbert D, Tauch A, Stoye J, Goesmann A: Comparative Pathway           50. Yoshida K, Furuya S, Osuka S, Mitoma J, Shinoda Y, Watanabe M, Azuma N, <br />     Analyzer&#226;&#8364;&#8220;a web server comparative analysis, clustering                     Tanaka H, Hashikawa T, Itohara S, Hirabayashi Y: Targeted disruption the <br />     visualization metabolic networks multiple organisms. Nucleic Acids           mouse 3-phosphoglycerate dehydrogenase gene causes severe <br />     Res 2008, 36:W433-7.                                                               neurodevelopmental defects results embryonic lethality. J Biol <br /> 31. Kanehisa M, Goto S: KEGG: kyoto encyclopedia genes genomes.                 Chem 2004, 279:3573-3577. <br />     Nucleic Acids Res 2000, 28:27-30.                                              51. Yu H, Wessels A, Tint GS, Patel SB: Partial rescue neonatal lethality of <br /> 32. Nelson DR, Zeldin DC, Hoffman SMG, Maltais LJ, Wain HM, Nebert DW:                 Dhcr7 null mice nestin promoter-driven DHCR7 transgene <br />     Comparison cytochrome P450 (CYP) genes mouse                       expression. Brain Res Dev Brain Res 2005, 156:46-60. <br />     human genomes, including nomenclature recommendations genes,               52. Blatt C, DePamphilis ML: Striking homology mouse human <br />     pseudogenes alternative-splice variants. Pharmacogenetics 2004,                transcription enhancer factor-1 (TEF-1). Nucleic Acids Res 1993, 21:747-748. <br />     14:1-18.                                                                       53. Zhu L, Swergold GD, Seldin MF: Examination sequence homology <br /> 33. Maglott D, Ostell J, Pruitt KD, Tatusova T: Entrez Gene: gene-centered             human chromosome 20 mouse genome: intense <br />     information NCBI. Nucleic Acids Res 2007, 35:D26-31.                            conservation genomic elements. Hum Genet 2003, 113:60-70. <br /> 34. Kumar VS, Maranas CD: GrowMatch: automated method                       54. Chen L, Vitkup D: Distribution orphan metabolic activities. Trends <br />     reconciling silico/in vivo growth predictions. PLoS Comput Biol 2009, 5:        Biotechnol 2007, 25:343-348. <br />     e1000308.                                                                      55. Lespinet O, Labedan B: Orphan enzymes unexplored <br /> 35. Emi M, Wilson DE, Iverius PH, Wu L, Hata A, Hegele R, Williams RR,                 reservoir new drug targets. Drug Discov Today 2006, 11:300-305. <br />     Lalouel JM: <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span>Missense mutation</span> (Gly&#226;&#8364;&#8221;&#226;&#8364;&#8221;Glu188) human lipoprotein               56. Jamshidi N, Palsson B&#195;&#732;: Systems biology SNPs. Mol Syst Biol 2006, 2:38. <br />     lipase imparting functional deficiency. J Biol Chem 1990, 265:5910-5916.       57. Shlomi T, Cabili MN, Ruppin E: Predicting metabolic biomarkers human <br /> 36. Wittrup HH, Tybjaerg-Hansen A, Steffensen R, Deeb SS, Brunzell JD,                 inborn errors metabolism. Mol Syst Biol 2009, 5:263. <br />     Jensen G, Nordestgaard BG: Mutations lipoprotein lipase gene            58. Hillenmeyer ME, Fung E, Wildenhain J, Pierce SE, Hoon S, Lee W, Proctor M, <br />     associated ischemic heart disease men. The Copenhagen city                 St Onge RP, Tyers M, Koller D, Altman RB, Davis RW, Nislow C, Giaever G: <br />     heart study. Arterioscler Thromb Vasc Biol 1999, 19:1535-1540.                     The chemical genomic portrait yeast: uncovering phenotype all <br /> 37. Weinstock PH, Bisgaier CL, Aalto-Set&#195;&#164;l&#195;&#164; K, Radner H, Ramakrishnan R, Levak-        genes. Science (80-) 2008, 320:362-365. <br />     Frank S, Essenburg AD, Zechner R, Breslow JL: Severe hypertriglyceridemia,     59. Threadgill DW, Hunter KW, Williams RW: Genetic dissection complex <br />     reduced high density lipoprotein, neonatal death lipoprotein                quantitative traits: fantasy reality community effort. <br />     lipase knockout mice. Mild hypertriglyceridemia impaired low             Mamm Genome 2002, 13:175-178. <br />     density lipoprotein clearance heterozygotes. J Clin Invest 1995,            60. Herrg&#195;&#165;rd MJ, Swainston N, Dobson P, Dunn WB, Arga KY, Arvas M, <br />     96:2555-2568.                                                                      Bl&#195;&#188;thgen N, Borger S, Costenoble R, Heinemann M, Hucka M, Le Nov&#195;&#168;re N, <br /> 38. Cunningham D, Spychala K, McLarren KW, Garza LA, Boerkoel CF,                      Li P, Liebermeister W, Mo ML, Oliveira AP, Petranovic D, Pettifer S, <br />     Herman GE: Developmental expression <span id='am-478' about='xsp:pattern' typeof='owl:Thing'>pattern</span> cholesterogenic                 Simeonidis E, Smallbone K, Spasi&#196;&#8225; I, Weichart D, Brent R, Broomhead DS, <br />     enzyme NSDHL negative selection NSDHL-deficient cells                Westerhoff HV, Kirdar B, Penttil&#195;&#164; M, Klipp E, Palsson B&#195;&#732;, Sauer U: A <br />     heterozygous Bpa(1H)/+ mouse. Mol Genet Metab 2009, 98:356-366.                    consensus yeast metabolic network reconstruction obtained a <br /> 39. Di Pietro E, Sirois J, Tremblay ML, MacKenzie RE: Mitochondrial NAD-               community approach systems biology. Nat Biotechnol 2008, <br />     dependent methylenetetrahydrofolate dehydrogenase-                                 26:1155-1160. <br />     methenyltetrahydrofolate cyclohydrolase essential embryonic             61. Mirza R, Hayasaka S, Kambe F, Maki K, Kaji T, Murata Y, Seo H: Increased <br />     development. Mol Cell Biol 2002, 22:4158-4166.                                     expression aquaporin-3 epidermis DHCR24 knockout mice. <br /> 40. Hojjati MR, Li Z, Jiang X: Serine palmitoyl-CoA transferase (SPT) deficiency       Br J Dermatol 2008, 158:679-684. <br />     sphingolipid levels mice. Biochim Biophys Acta 2005, 1737:44-51.        62. Orth JD, Thiele I, Palsson B&#195;&#732;: What flux balance analysis? Nat Biotechnol <br /> 41. Howerton SB, Nagpal A, Williams LD: Surprising roles electrostatic              2010, 28:245-248. <br />     interactions DNA-ligand complexes. Biopolymers 2003, 69:87-99.              63. Mahadevan R, Schilling CH: The effects alternate optimal solutions in <br /> 42. Huyghe S, Mannaerts GP, Baes M, Van Veldhoven PP: Peroxisomal                      constraint-based genome-scale metabolic models. Metab Eng 2003, <br />     multifunctional protein-2: enzyme, patients knockout               5:264-276. <br />     mouse model. Biochim Biophys Acta 2006, 1761:973-994.                          64. Gough J, Karplus K, Hughey R, Chothia C: Assignment homology to <br /> 43. Krakowiak PA, Wassif CA, Kratz L, Cozma D, Kov&#195;&#161;rov&#195;&#161; M, Harris G,                   genome sequences using library hidden Markov models that <br />     Grinberg A, Yang Y, Hunter AGW, Tsokos M, Kelley RI, Porter FD:                    represent proteins known structure. J Mol Biol 2001, 313:903-919. <br />     Lathosterolosis: inborn error human murine cholesterol               65. Schellenberger J, Palsson B&#195;&#732;: Use randomized sampling analysis of <br />     synthesis lathosterol 5-desaturase deficiency. Hum Mol Genet                metabolic networks. J Biol Chem 2009, 284:5457-5461. <br />     2003, 12:1631-1641.                                                            66. Thiele I, Price ND, Vo TD, Palsson B&#195;&#732;: Candidate metabolic network states <br /> 44. Means GD, Boyd Y, Willis CR, Derry JM: Transgenic rescue tattered           human mitochondria. Impact diabetes, ischemia, diet. J Biol <br />     phenotype using BAC encoding Ebp. Mamm Genome 2001,                           Chem 2005, 280:11683-11695. <br />     12:323-325. <br /> 45. Mirza R, Hayasaka S, Takagishi Y, Kambe F, Ohmori S, Maki K, Yamamoto M,        doi:10.1186/1752-0509-4-140 <br />     Murakami K, Kaji T, Zadworny D, Murata Y, Seo H: DHCR24 gene knockout           Cite article as: Sigurdsson et al.: A detailed genome-wide <br />     mice demonstrate lethal dermopathy differentiation                     reconstruction mouse metabolism based human Recon 1. BMC <br />     maturation defects epidermis. J Invest Dermatol 2006, 126:638-647.       Systems Biology 2010 4:140. <br /> 46. Steenbergen R, Nanowski TS, Beigneux A, Kulinski A, Young SG, Vance JE: <br />     Disruption phosphatidylserine decarboxylase gene mice causes <br />     embryonic lethality mitochondrial defects. J Biol Chem 2005, <br />     280:40032 <br /> </body></html>