<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL OF BACTERIOLOGY, June 1986, p. 746-750                                                                           Vol. 166, No. 3 <br /> 0021-9193/86/060746-05$02.00/0 <br /> Copyright &#194;&#169; 1986, American Society Microbiology <br />  <br />  <br />          Deletion spf (Spot 42 RNA) Gene Escherichia coli <br />                              GRAHAM F. HATFULL AND CATHERINE M. JOYCE* <br /> Department Molecular Biophysics Biochemistry, Yale University Medical School, New Haven, Connecticut 06510 <br />                                          Received 1 November 1985/Accepted 10 March 1986 <br />  <br />             <span id='am-3' about='protege:TO' typeof='owl:Thing'>To</span> investigate function spot 42 RNA, small RNA Escherichia coli, constructed strain which <br />          spf, structural gene RNA, deleted. We achieved using Aatt phage k carrying DNA <br />          fragment spanning spf region precise deletion spf. By integration phage spf locus <br />          subsequent excision, able cross spf deletion bacterial chromosome. The fact <br />          deletion obtained indicated spf essential gene. We did observe major <br />          defect Aspf cells, strain background deletion caused slight growth impairment. <br />  <br />  <br />    Spot 42 RNA Escherichia coli moderately abundant                             MATERIALS AND METHODS <br /> (100 200 copies cell) RNA, 109 nucleotides long (14, <br /> 15). It encoded single gene (spf) located immediately                Materials. Restriction endonucleases obtained <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> downstream polA 86.6 min E. coli genetic map               New England Biolabs Boehringer Mannheim Biochemi- <br /> (2, 7, 13). The function spf gene unknown,             cals. T4 DNA ligase Collaborative Research, Inc., <br /> the abundance RNA suggested gene                    Boehringer Mannheim Biochemicals. Oligonucleotide <br /> product play important, essential, role                 linkers Collaborative Research, Inc., New <br /> vivo. The cellular distribution spot 42 RNA gives little               England Biolabs. Bacterial strains used work listed <br /> clue function; RNA associated                    Table 1. Plasmids general techniques relevant this <br /> membranes, ribosomes, nucleoid (15).                    work described previously (8). The <br />    Currently know ultimate gene                     plasmids derived vector pNG16, deletion <br /> product spf RNA protein. The RNA itself               derivative pBR322. <br /> structural catalytic component ribonucleoprotein                     Construction genetically marked spf deletion. This <br /> particle, like Ml RNA component RNase P (5). On                construction outlined Fig. 1. pGH78 contains 2.1- <br /> other hand, spot 42 RNA sequence features suggesting                  kilobase-pair (kb) SacI-HindIII fragment carrying end of <br /> that mRNA: ribosome-binding site followed                  polA coding region spf gene, BamHI linker <br /> by initiation codon open reading frame                 inserted AluI site 46 <span id='am-238' about='xsp:base' typeof='owl:Thing'>base</span> pairs (bp) the <br /> coding capacity 15-residue peptide (14). The potential              termination codon polA (8). This BamHI site defined the <br /> translated product rich hydrophobic amino acids,                    upstream endpoint deletion spf. The downstream <br /> particularly leucine, contains total charged           endpoint defined using plasmid, AN3058, carrying an <br /> residues (including N- C-terminal amino acids). The               vitro-constructed deletion polA (6). This plasmid con- <br /> latter alternating <span id='am-239' about='xsp:pattern' typeof='owl:Thing'>pattern</span> positively negatively           tains DNA Sau3A site position 3058 our <br /> charged groups adjacent pairs separated approxi-                  published sequence (7), HindIII site position 3822, <br /> mately turn a-helix. Consideration hydro-                BamHI site position original Sau3A site. <br /> phobic nature putative peptide gene product led             As genetic marker used kanamycin resistance (Kmr) <br /> and suggest inserted mem-                   gene Tn903; plasmid pGH54 contains gene 950-bp <br /> branes, serving structural role forming channels             BamHI fragment inserted BamHI site pNG16 (8). <br /> to allow passage certain ions small molecules (7,               The 800-bp BamHI-EcoRI fragment AN3058 the <br /> 13). Sahagan Dahlberg shown expression                   950-bp BamHI fragment pGH54 purified ligated <br /> spf negatively regulated cyclic AMP (15). This             large EcoRI-BamHI fragment pGH78, a <br /> taken circumstantial evidence linking spf gene             structure Kmr gene (transcribed same <br /> product membrane structure function, cyclic                 direction polA spf) replaces spf. The Sall site the <br /> AMP negative regulator membrane                  vector DNA replaced EcoRI linker the <br /> envelope proteins (1, 10, 12) cell division (17).           resulting 3.2-kb EcoRI fragment recovered cloning <br />    The problem assigning function gene locus             EcoRI site pNG16, pGH84. The Kmr Aspf <br /> approached asking gene                   construction inserted EcoRI vector, Xgtl- <br /> (which available cloned fragment) essential               XB (4). The structure resulting phage, XGH84, was <br /> for viability E. coli. This determining                 verified Southern blot analysis, using pCJ1 probe. <br /> whether gene replaced chromosome                  A control phage, XGH86, carries Kmr gene <br /> in vitro-constructed deletion, using strategy             deletion spf, constructed analogous way <br /> described previously (8). If target gene essential         insertion Kmr fragment BamHI site of <br /> <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-6' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span>survival</span>, obtain strain gene             pGH78. <br /> deleted. This null <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span>mutation</span> used test                     Isolation analysis lysogens. Lysogens XGH84 E. <br /> functions gene.                                                    coli CM4722 selected 30&#194;&#176;C L-broth plates seeded <br />                                                                           Xch80del9 (11) subsequently checked X <br />   * Corresponding author.                                                 immunity Kmr. Chromosomal DNA candidate <br />                                                                     746 <br />  VOL. 166, 1986                                                                                                 DELETION OF THE spf GENE             747 <br />  <br />           TABLE 1. Bacterial strains used work                                desired transductants selected plates containing 40 <br />  Strain                   Genotype                         Source                     ,ug kanamycin ml. <br /> CM4722&apos;   F+ A(gal-bio) thi-I relAl spoTI     W. S. Kelley <br />                                                                                          Growth CJ200 CJ205 liquid medium com- <br /> CJ200     CM4722 spf&apos; Kmr                     This work                               pared using competition experiment. Each strain was <br /> CJ205     CM4722 Aspf Kmr                     This work                               mixed equal number cells parent strain, <br /> S165      AgalS165 rpsL                   Our strain                              CM4722 (Kms). After period growth, mixture was <br />                                                 collection                            plated L broth, proportion Kmr cells (CJ200 or <br /> LE392  lacY galK galT metB trpR supE supF     Our strain                              CJ205) determined. <br />          hsdR                                   collection                               The sensitivity spf&apos; Aspfstrains salt, detergents, <br /> AW330 lac gal ara xyl thr leu thi fhuA rpsL   R. Macnab                               drugs compared ways. In method, a <br /> 20SOK- galK galP                              R. Macnab                               heavy inoculum strain tested streaked out <br /> JC1552 leuB trp hisG argG metB lacY gal       B. Bachmann                             sector L-broth plate. The substance be <br />          malA xyl mtl rpsL fhuA tsx supE        (CGSC)b                               tested placed center plate. After <br /> AB2557 purF dsdA aroC ilvD lac malA xyl mtl B. Bachmann                               incubation, face-upwards, position inhibition zone <br />          rpsL fhuA tsx supE                     (CGSC) <br /> KL188 pyrD trp thyA thi galK malA xyl mtl B. Bachmann                             isogenic Aspf spf&apos; pairs compared. Alterna- <br />          rpsL                                   (CGSC)                                tively, strains plated agar substance <br />   CM4722 F+ derivative HfrH strain KS302 (16). <br />                                                                                       tested placed filter paper disk lawn of <br />  b <br />     CGSC, E. coli Genetic Stock Center, Yale University, New Haven,                   cells, described Benson DeCloux (3). Similar <br /> Conn.                                                                                 results obtained methods. <br />                                                                                          Swarming behavior motile strains examined on <br />                                                                                       tryptone plates containing 0.38% agar. <br /> lysogens analyzed Southern blots determine                                                         RESULTS <br /> structure spf locus.                                                             Our method determining particular gene is <br />    Curing lysogens. Lysogens grown overnight                               essential survival (Fig. 2) described detail <br /> 30&#194;&#176;C L broth. Cured cells selected plating                                 previously (8). Briefly, involves insertion Kmr <br /> appropriate dilutions 42&#194;&#176;C L-broth plates containing 1                          marker sequences immediately flanking the <br /> mM EDTA. The proportion survivors retained Kmr                                gene interest, using standard vitro manipulations. The <br /> was determined replica plating. To obtain independent                           genetically marked deletion cloned phage A vector <br /> estimate relative frequencies recombination                             Aatt thermoinducible (cI857). At permissive <br /> events giving rise Kmr kanamycin-sensitive (Kms)                               temperature, phage lysogenize homologous re- <br /> cells, sample lysogen culture induced                         combination cloned bacterial sequences, giving a <br /> phage lysate. The composition lysate determined                            structure integrated prophage flanked a <br /> by forming lysogens strain LE392 determining                              wild-type deleted (Kmr) copy target gene. <br /> the relative numbers Kmr Kms lysogens. (For                            Spontaneous curing lysogen regenerate <br /> details, reference 8.)                                                            wild-type configuration leave Kmr-marked dele- <br />    Hybridization probes. Southern blots probed                            tion chromosome, depending position the <br /> with nick-translated pCJ1, carries 5-kb HindIll                               recombinational crossover. Failure obtain Kmr cells <br /> fragment containing polA spf genes (9), spot                          temperature-resistant cured survivors indi- <br /> 42 RNA labeled vitro transcription described                                 cate deletion target gene lethal the <br /> previously (7).                                                                       necessary recombinational events did place a <br />    Other methods. Transduction spf Kmr E. coli                                high frequency. The possibility ad- <br /> CJ200, Aspf Kmr E. coli CJ205, carried                               dressed independent estimate fraction dele- <br /> using P1 kc following standard procedures (11). The                               tion-producing events, examining product <br />  <br />  <br />                                          500 bp <br />  <br />                                   S    Sc                            Bspf                          H.,R <br />                                                                              P.II <br />                                                                                                                        pGH78 <br />                                                                              B <br />                                                                                                                        AN3058 <br />                             R    Sc                                 KmR                                   H,R <br />                                                       za <br />                                                            B <br />                                                                     I%--H1-               B <br />                                                                                                           II <br />                                                                                                                        pGH84 <br />                             R    Sc                                 Km           IH       B                     H,R <br />                                                                                                                  ,r <br />                             I                                       . .... <br />                                                                                               -W <br />                                                                                                                       * oGH36 <br />   FIG. 1. Construction plasmids used work. For details text. The line represents DNA derived E. coli <br /> chromosome. Within region indicated polA coding sequence (hatched area), spf gene (black arrow), functional <br /> open reading frame 22-kilodalton protein (open arrow) (7). The DNA fragment carrying Kmr gene represented wavy line. <br /> DNA derived plasmid vectors drawn line shown entirety. B, BamHI; H, Hindlll; R, EcoRI; S, Sall; Sc, <br /> Sacl. <br />  748     HATFULL AND JOYCE                                                                                                        J. BACTERIOL. <br />  <br /> of excision process, phage produced induction                                            XGH84 <br /> the lysogen, repeating curing addi- <br /> tional copy target gene trans remove                                            R KmR R <br /> selective pressure deletion. <br />    Phage lysogen structure. The construction XGH84, <br /> in spf gene replaced Kmr while <br /> retaining surrounding sequences, described                                 S                .                S <br /> Materials Methods section Fig. 1. The deletion <br /> endpoints carefully chosen remove completely                                               13.3 - <br /> the spf-transcribed region encroaching of <br /> the extremely close neighboring genes (7). Lysogens of <br /> XGH84 CM4722 selected, chromosomal                                                      I Integration <br /> DNA analyzed Southern blots identify in <br /> which integration taken place spflocus. Digestion <br /> with EcoRI SalI gives pair fragments (of sizes 8.2                     SR                               R KMR               S <br /> and 8.3 kb, 7.5 9.0 kb, depending position            A     -4      _                  X A            C              | <br /> the recombihational crossover) integration taken place <br />                                                                                  8.3   *                                   8.2 <br /> at spf (Fig. 3). Integration chromosome               OR <br /> results 13.3-kb fragnient diagnostic unrearranged                    S              KmR R <br /> spf locus 3.2-kb fragment derived XGHS4. We                                                            R.   *S <br /> chose independent lysogens, type A of <br />                                                                             B <br />                                                                                         9.0        b    *x-m               7.5 <br /> type B, curing experiment. <br />    Curing lysogens shows spf deleted. Cured             FIG. 3. Integration XGH84 spf locus. Two alternative <br /> cells selected overnight cultures chosen            structures, designated A B, result integration of <br /> lysogens, proportion Kmr colonies deter-              XGH84 homologous recombination spf. Symbols: *, position <br />                                                                      spf gene; mm, upstream flanking DNA carried XGH84; <br /> mined (Table 2). In case, substantial proportion           zi, downstream flanking DNA: Chroniosomal DNA external to <br /> Kmr, spf-deleted, cells obtained. For         region, homology XGH84 shown wavy line. The <br /> lysogen tested, ratio Kmr Kms cured cells              position relevant EcoRI (R) Sall (S) sites indicated, along <br /> approximately reciprocal ratio Kmr Kms                  sizes (in kilobases) fragments hybridize a <br /> transducing phage obtained induction lysogen,             pCJ1 probe. <br /> indicating selective bias spf-deleted <br /> cells. There consistent three- fourfold bias favor <br />                                                                      recombination upstream flanking region, that <br />                                                                      lysogens structure A gave tnore Kmr cells, lysogens of <br />                                                                      structure B gave Kms cells (Table 2 Fig. 3). This is <br />                                                                      probably greater <span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span> upstream region of <br />                            ., X        _                             flanking homology (1,150 bp opposed 770 bp). <br />                                                                         Southern blot analysis chromosomal DNA cured <br />                                                                      cells obtained way showed Kms cells had <br />                                                                      regained 5-kb HindIII fragment diagnostic unrear- <br />                                                                      ranged spf locus (data shown). Kmr cells, other <br />                                    Integration                       hand, new HindIII fragments 4.7 1.1 kb (Fig. <br />                                                                      4B, lanes 1 2). Moreover, DNA cells did not <br />                                                                      hybridize spot 42 RNA (Fig. 4B, lanes 5 6). This <br />                   KmR.                     x                         observation confirms hybridization data Rice and <br />                                                                      Dahlberg (13) indicated wild-type E. coli <br />                                                                      chromosome contains single locus homologous spot 42 <br />                                        Curing                        RNA. <br />                                                                         The curing experiment provided cells lacking the <br />                                                                      spf gene, serve tool examline the <br />                                                                      <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span>phenotype</span> spf null mutation probe the <br />               x                                 KmR                  function spf gene. To sure phenotype was <br />                                                       =P__           attributable Aspf mutation, generated isogenic <br />                                   OR                                 spf&apos; Kmr strain, using control phage XGH86, which <br />                                                                      Kmr inserted removing spf (Fig. 1). The <br />                      )                           X                   strains CJ200 (spfJ Kmr) CJ205 (Aspf Kmr) identical <br />                                                                      presence CJ200 230 bp spanning spf <br />                                                                      gene (Fig. 4). Southern blot analysis DNA spf+ <br />                                                                      Kmr cells confirmed smnaller HindIII fragment was <br />    FIG. 2. Method determining gene deleted.     1.3 kb, compared 1.1 kb Aspf cells (Fig. 4B, lanes 3 <br /> Phage integration excision used cross deletion   4), fragment hybridized spot 42 RNA <br /> target gene (X) bacterial chromosome. For details the <br /> text. The tegions homology phage chromo-      (lanes 7 8). <br /> some boxed: X-i, upstream flanking DNA; =., downstream             Phenotype Aspf cells. We compared growth CJ205 <br /> flanking DNA. Chromosomal DNA external regions           (Aspf) CJ200 (spf) various solid media: L broth or <br /> homology shown wavy lines. (Note      MacConkey agar, supplemented glucose, lactose, <br /> possible integrated structures shown.)                            galactose, maltose, M9 minimal medium (11) supple- <br />  VOL. 166, 1986                                                                                      DELETION OF THE spf GENE              749 <br />  <br />  <br />                    A                        I kb <br />  <br />                                 H                                               H <br />                                                                                                            spf <br />  <br />  <br />                                 H                                  KmR ,                   H <br />                                                     4.7                   &apos;O        1.                     Aspf KmR <br />  <br />  <br />                                 H                                  KmR I,H                     H <br />                                  &apos;*                 4.7                 -O..*        1.3       w <br />                                                                                                            spf KmR <br />  <br />    B             1 2 3 4              5 6 7 8 <br />                                                                   FIG. 4. Restriction analysis cured cells. (A) The diagram <br />                                                                 compares   structure spf locus wild-type cells (spfJ), the <br />   4.7- <br />                                                                 /spf Kmr cells obtained curing XGH84 lysogens, spf&apos; <br />                                                                 Kmr cells derived XGH86. The spf gene represented the <br />                                                                 black arrow, Kmr insert represented boxed region. <br />                                                                 HindlIl sites (H) shown, sizes (in kilobases) of <br />                                                                 Hindlll fragments hybridize pCJ1 probe. (B) Southern blot <br />                                                                 analysis Aspf Kmr cells (lanes 1, 2, 5, 6) spfF Kmr cells <br />                                                                 (lanes 3, 4, 7, 8). Chromosomal DNA isolates each <br />                                                                 genotype digested HindIII. Each digest divided <br />                                                                 tracks 1% agarose gel lanes 1 4 5 to <br />                                                                 8 contain identical sets DNA samples. After transfer nitrocel- <br />   1.3-                  -..k   ~.. <br />                                                                 lulose, lanes 1 4 hybridized nick-translated pCJ1, and <br />                                                                 lanes 5 8 hybridized spot 42 RNA transcribed vitro. <br />   1.1-                                                          The sizes fragments (in kilobases) indicated. <br />  <br />  <br />  <br />  <br /> mented glucose glycerol. We difference      generations temperature, proportion CJ205 <br /> colony size morphology 37&#194;&#176;C 42&#194;&#176;C. However,        cells decreased (typically 50% start the <br /> the plates incubated 23&#194;&#176;C, CJ205 (Aspf) gave smaller    experiment 25% 10 generations). We suspect the <br /> colonies CJ200. The difference noticeable      growth difference liquid medium 42&#194;&#176;C be <br /> L-broth plates, amounted twofold difference      peculiar CM4722 background, effect was <br /> in colony diameter 2 days growth. We believe      seen S165 strain background. Moreover, effect in <br /> the slower growth 23&#194;&#176;C attributable spf deletion   liquid medium 42&#194;&#176;C accompanied observed <br /> and secondary mutation, observed          difference plating efficiency colony morphology the <br /> same result independent isolates CJ205         temperature. <br /> genotype obtained curing transduction.              We tested number properties diagnostic alter- <br /> However, effect dependent CM4722 back-           ation membrane structure function. The Aspf mutation <br /> ground. No difference growth 23&#194;&#176;C seen        did affect sensitivity cells NaCl, EDTA, sodium <br /> compared Aspf Kmr spf&apos; Kmr derivatives         dodecyl sulfate, various antibiotics (carbenicillin, tetracy- <br /> strains listed Table 1. Since CM4722 strain   cline, nalidixic acid, rifampin), 37&#194;&#176;C 23&#194;&#176;C. <br /> this group F+, transferred F factor         (Increased sensitivity compounds characteristic of <br /> CM4722 Aspf derivative strain S165. The resulting <br /> strain indistinguishable F- parent plates at <br /> 23&#194;&#176;C, ruling effect F fact <br /> </body></html>