<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Mechanisms suppress multipolar <br /> divisions cancer cells extra <br /> centrosomes <br /> Mijung Kwon,1,5 Susana A. Godinho,1,5 Namrata S. Chandhok,1 Neil J. Ganem,1 Ammar Azioune,2 <br /> Manuel Thery,3 David Pellman1,4,6 <br /> 1 <br />  Department Pediatric Oncology, Dana-Farber Cancer Institute, Harvard Medical School, Boston, Massachusetts 02115, <br /> USA; 2Biologie du Cycle Cellulaire et la Motilite, Institut Curie, CNRS, UMR144, Paris 75005, France; 3CEA, DSV, <br /> iRTSV, Laboratoire Biopuces, Grenoble F38054, France; 4Department Pediatric Hematology/Oncology, Children&#226;&#8364;&#8482;s <br /> Hospital, Harvard Medical School, Boston, Massachusetts 02115, USA <br />  <br />  <br />  <br />  <br /> Multiple centrosomes tumor cells create potential multipolar divisions lead aneuploidy <br /> and cell death. Nevertheless, cancer cells successfully divide mechanisms suppress <br /> multipolar mitoses. A genome-wide RNAi screen Drosophila S2 cells secondary analysis cancer <br /> cells defined mechanisms suppress multipolar mitoses. In addition proteins organize microtubules <br /> at spindle poles, identified novel roles spindle assembly checkpoint, cortical actin cytoskeleton, <br /> and cell adhesion. Using live cell imaging fibronectin micropatterns, interphase cell shape <br /> and adhesion <span id='am-315' about='xsp:pattern' typeof='owl:Thing'>pattern</span> determine success subsequent mitosis cells extra centrosomes. <br /> These findings identify cancer-selective therapeutic targets: HSET, normally nonessential kinesin <br /> motor, essential viability certain extra centrosome-containing cancer cells. Thus, morphological <br /> features cancer cells linked unique genetic requirements <span id='am-3' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-4' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-5' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span>survival</span>. <br /> [Keywords: Centrosomes; mitosis; actin; adhesion; cancer; cell cycle] <br /> Supplemental material available http://www.genesdev.org. <br /> Received May 30, 2008; revised version accepted June 24, 2008. <br />  <br />  <br />  <br /> Centrosomes play crucial role equal segregation                    (Fukasawa 2007). Centrosome amplification can, prin- <br /> of chromosomes contributing bipolar spindle as-                       ciple, arise types cell division errors: cen- <br /> sembly mitosis (Doxsey 2001). The tight control                      trosome overduplication, novo synthesis centro- <br /> of centrosome duplication, limiting cell                     somes, cell fusion, cytokinesis failure (Boveri 1929; <br /> cycle, ensures normal cells enter mitosis                     Nigg 2002; Ganem et al. 2007). <br /> centrosomes microtubule organizing centers (MTOCs).                        The role supernumerary centrosomes tumor bi- <br /> Failure properly control centrosome number func-                     ology likely multifaceted. Whereas multiple cen- <br /> tion lead multipolar spindles, aneuploidy, disruption                trosomes facilitate tumorigenesis promoting <br /> of cell polarity, failure asymmetric cell divisions                  aneuploidy and/or disrupting cell polarity, also <br /> (Heneen 1970; Nigg 2002).                                                   impose fitness cost growth mature cancers <br />    Increased centrosome number, termed centro-                        potential multipolar mitoses. <span id='am-2' about='protege:TO' typeof='owl:Thing'>To</span> cir- <br /> some amplification, common characteristic solid                     cumvent problem, cancer cells appear have <br /> and hematological cancers. Centrosome amplification                         mechanisms suppress multipolar mitoses, best <br /> correlates aneuploidy malignant behavior                        studied clustering supernumerary centrosomes <br /> tumor cell lines, mouse tumor models, human tu-                         groups enabling bipolar mitosis (Ring et al. <br /> mors (Levine et al. 1991; Lingle et al. 1998; D&#226;&#8364;&#8482;Assoro                      1982; Brinkley 2001; Nigg 2002). Centrosome clustering <br /> et al. 2002; Pihan et al. 2003; Giehl et al. 2005). <span id='am-159' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-160' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span>Mutation</span>                tumor cells incompletely understood; however, is <br /> or misregulation variety tumor suppressors                       expected rely significant degree microtubule- <br /> oncogenes correlated centrosome amplification                      associated proteins (MAPs) motors organize the <br />                                                                             spindle poles (Karsenti Vernos 2001; Nigg 2002). For <br />                                                                             example, recent work uncovered requirement cyto- <br /> 5 <br />  These authors contributed equally work.                            plasmic dynein, minus end-directed microtubule (MT) <br /> 6 <br />  Corresponding author.                                                      motor, NuMA, spindle associated MAP, centro- <br /> E-MAIL david_pellman@dfci.harvard.edu; FAX (617) 632-6845. <br /> Article published online ahead print. Article publication date   clustering (Quintyne et al. 2005). The existence of <br /> online http://www.genesdev.org/cgi/doi/10.1101/gad.1700908.              mechanisms suppress multipolar mitoses raises the <br />  <br />  <br />          GENES &amp; DEVELOPMENT 22:2189&#226;&#8364;&#8220;2203 &#194;&#169; 2008 Cold Spring Harbor Laboratory Press ISSN 0890-9369/08; www.genesdev.org         2189 <br />  Kwon et al. <br />  <br />  <br />  <br /> possibility novel therapeutic strategy cancer:         bona fide role centrosome clustering (Fig. 1C; <br /> Drugs interfere centrosome clustering mecha-          Supplemental Table S1). Among validated genes, 62% <br /> nisms lethal tumor cells containing multiple        genes list (82 133 genes) mam- <br /> centrosomes, potentially spare normal cells. Al-            malian homologs (Fig. 1C), 33% genes (43) do <br /> though drugs, including Taxol, promote mul-         known function (Fig. 1D). Centrosome cluster- <br /> tipolar mitosis, specific cells multiple       ing occur varying efficiency. Knockdown caused a <br /> centrosomes (Chen Horwitz 2002; Rebacz et al. 2007).        variety centrosome clustering defects, clas- <br />    To comprehensively define genes required sup-         sified groups: bipolar scattered (spindles with <br /> press multipolar mitoses, performed genome-wide            multiple centrosomes scattered spindle), <br /> RNAi screen near tetraploid Drosophila S2 cells. The         multipolar spindles, multiaster spindles (small and <br /> classes genes discovered screen enabled iden-    compact multipolar spindles) (Fig. 1E). <br /> tification processes suppress multipolar mitoses.          We identified genes involved diverse range cel- <br /> The spindle assembly checkpoint (SAC) ensures adequate          lular processes, suggesting unappreciated complexity in <br /> time clustering centrosomes prior anaphase. Cen-      mechanisms controlling organization supernu- <br /> trosome clustering achieved combination              merary centrosomes (Fig. 1D). We number of <br /> spindle-intrinsic MT-binding forces actin-regulating        genes promote bundling spindle MTs, includ- <br /> forces cell cortex. The distribution cortical   ing minus end-directed kinesin Ncd (human HSET). <br /> forces determined interphase adhesion pattern.        The screen unexpectedly identified genes involved <br /> Thus, tumor microenvironment determine              SAC, actin regulation, cell polarity, cell adhe- <br /> success mitosis cells extra centrosomes. Finally,    sion. Below present experiments define over- <br /> we validate idea tumors uniquely         lapping mechanisms suppress multipolar mitoses: a <br /> dependent centrosome clustering survival.                timing mechanism employing SAC, intrinsic pole clus- <br />                                                                 tering mechanisms relying MT regulators, novel <br /> Results                                                         mechanism requiring actin cell adhesion. <br /> RNAi screen genes required clustering <br /> supernumerary centrosomes                                       The spindle assembly checkpoint (SAC) prevents <br />                                                                 multipolar mitoses <br /> We used RNAi screen comprehensively define the <br /> molecular pathways required clustering supernumer-          The SAC components Mad2, BubR1 (human Bub1), and <br /> ary centrosomes (Fig. 1). Of Drosophila cell lines        CENP-Meta (human CENP-E) required centrosome <br /> characterized, near-tetraploid S2 cells suit-     clustering (Fig. 2A; Supplemental Table S1), suggesting a <br /> able screen &gt;50% cells contain extra         role SAC process. This requirement was <br /> centrosomes efficiently clustered poles       evident cells treated with <br /> during mitoses (&gt;95%) (Supplemental Figs. S1, S2).              MG132 (Fig. 2A), indicating short treatment with <br />    We screened 23,172 dsRNAs targeting &#226;&#710;&#188;99%              MG132 employed screen partially masked effect <br /> Drosophila genome (&#226;&#710;&#188;14,000 genes) identify genes             SAC gene RNAi spindle multipolarity. This finding <br /> whose knockdown leads multipolar spindles (centro-           somewhat surprising, given previous work PtK1 <br /> some declustering) S2 cells (Fig. 1A,B). S2 cells       cells suggesting SAC activated multipolar <br /> exposed dsRNA 4 d mitotic figures en-           spindles multiple centrosomes (Sluder et al. 1997). <br /> riched treatment proteasome inhibitor MG132            Time-lapse imaging supported role SAC in <br /> during 9 h RNAi treatment (Supplemental         preventing multipolar mitoses. In S2 cells, centri- <br /> Fig. S2). Cells stained DNA, MTs, centro-          oles MTs labeled GFP-SAS-6 mCherry <br /> somes, images acquired 20&#195;&#8212; objective,           &#226;?&#163;-tubulin, clear correlation in- <br /> using high-throughput automated microscope (Fig. 1B;          creased number centrosomes prolonged time re- <br /> see Supplemental Material detail).                      quired form bipolar spindle (2.7-fold) (Fig. 2B,C; <br />    By visual inspection &#226;&#710;&#188;96,000 images, scored        Supplemental Table S3; Supplemental Movie S1). We <br /> percentage multipolar spindles RNAi condi-          measured interval NEBD ana- <br /> tion. Using 95% confidence interval, primary screen       phase onset (visualized GFP-Cid, Drosophila <br /> identified 701 candidates associated multipolar          CENP-A) comparing cells two cen- <br /> spindle <span id='am-81' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-82' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span>phenotype</span> (Fig. 1C; Supplemental Fig. S3). Two-         trosomes. Relative cells centrosomes, cells <br /> hundred-ninety-two genes selected initial cohort        multiple centrosomes exhibited marked delay in <br /> for study based strength phenotype,       anaphase onset (1.8-fold) (Fig. 2D; Supplemental Table S3; <br /> the existence readily identifiable mammalian ho-             Supplemental Movies S2, S3). Moreover, delay in <br /> mologs, predicted off-target effects. Addi-       anaphase onset abolished Mad2 RNAi, cells <br /> tionally, eliminated genes previously         entered anaphase declustered centrosomes mis- <br /> determined required cytokinesis Drosophila         aligned kinetochores (Fig. 2D,E; Supplemental Table S3). <br /> cells (Echard et al. 2004; Eggert et al. 2004)          Further suggesting SAC activation, multipolar spindles <br /> spindle multipolarity secondary effect cyto-        strong increase number BubR1 foci rela- <br /> kinesis failure (Goshima et al. 2007). Of 292 genes         tive bipolar metaphase spindles (Supplemental Fig. <br /> selected secondary screen, 133 confirmed        S4). Finally, requirement SAC prevent mul- <br />  <br />  <br /> 2190      GENES &amp; DEVELOPMENT <br />                                                                                                          Suppression multipolar mitosis <br />  <br />  <br />  <br />  <br /> Figure 1. Genome-wide screen genes required clustering supernumerary centrosomes S2 cells. (A) Sample images <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> screen. Normal bipolar spindles control EGFP RNAi (top panel) multipolar spindles Ncd RNAi (bottom panel) shown. <br /> Cells stained MTs (&#226;?&#163;-tubulin, green), centrosomes (&#226;?&#165;-tubulin, red), DNA (blue). (Insets) Images high magnification. (B) <br /> Scheme genome-wide screen S2 cells. Procedure primary secondary screens. Addition MG132, proteasome <br /> inhibitor, end RNAi treatment used increase mitotic index. Immediate transfer cells Con-A used <br /> to facilitate cell attachment. (C) Table summarizing screen results. (D) Gene Ontology (GO) annotations 133 genes from <br /> the screen. (E) Images showing classes centrosome clustering defects S2 cells. Cells stained MTs (&#226;?&#163;-tubulin, green), <br /> centrosomes (&#226;?&#165;-tubulin, red), DNA (blue). Bar, 10 &#194;&#181;m. <br />  <br />  <br /> tipolar mitoses partially suppressed artifi-             Spindle-intrinsic pole clustering forces prevent <br /> cial metaphase delay imposed treatment MG132                  multipolar mitoses <br /> (Fig. 2A). This suggests SAC does monitor <br /> multipolar mitosis se, SAC activa-                Previous work S2 cells demonstrated critical role <br /> tion, likely triggered abnormal kinetochore attach-                MT motors MAPs spindle pole focusing (Go- <br /> ment tension, provides sufficient time compensa-               shima et al. 2005; Morales-Mulia Scholey 2005). <br /> tory mechanisms organize multiple centrosomes.                     We identified Ncd, Kinesin-14 family member, the <br />  <br />  <br />                                                                                                    GENES &amp; DEVELOPMENT              2191 <br />  Kwon et al. <br />  <br />  <br />  <br />  <br /> Figure 2. The SAC prevents multipolar mitoses S2 cells. (A) Mad2 requirement centrosome clustering. Centrosome clustering <br /> defects scored S2 cells RNAi EGFP, Mad2 alone, EGFP Mad2 plus 7 h MG132 treatment. Graph shows the <br /> average independent experiments; mean &#194;&#177; SD. (*) P &lt; 0.05; (***) P &lt; 0.001, Student&#226;&#8364;&#8482;s t-test. (B) Cell cycle delay associated with <br /> multipolar mitoses. Comparison time (minutes) taken NEBD bipolar spindle formation S2 cells two <br /> centrosomes (2c) extra centrosomes (&gt;2c) mitosis. Dots bars indicate individual cells average, respectively, in <br /> minutes. (**) P &lt; 0.002, Student&#226;&#8364;&#8482;s t-test. (C) Examples time-lapse images mitotic S2 cells (top panels) than <br /> two (bottom panels; Supplemental Movie S1) centrosomes. Centrosomes MTs labeled GFP-SAS-6 (green) mCherry-&#226;?&#163;- <br /> tubulin (red). (D) Delay induced extra centrosomes requires Mad2. Time taken NEBD anaphase onset obtained from <br /> S2 cells expressing GFP-Cid mCherry-&#226;?&#163;-tubulin RNAi EGFP control Mad2 (average times shown red green <br /> bars cells two centrosomes, respectively. (*) P &lt; 0.02; (**) P &lt; 0.002, Student&#226;&#8364;&#8482;s t-test. Anaphase delay is <br /> not cell undergoing mitotic slippage observed cells normal number kinetochores: data obtained <br /> from S2 cells containing 20&#226;&#8364;&#8220;26 Cid dots cell. (E) Examples time lapse images (see Supplemental Movies S3, S4). (Bottom panels) <br /> Premature anaphase onset occurs centrosome clustering Mad2 depleted cells. Arrows indicate centrosomes. Bar, 10 &#194;&#181;m. <br />  <br />  <br />  <br /> strongest hit primary screen. Ncd minus end-                identified screen. This expected be- <br /> directed motor bundles MTs spindle poles                    cause S2 cells loss dynein does significantly in- <br /> (Karabay Walker 1999). By GFP-SAS-6 labeling,                    duce multipolar mitoses, does compromise <br /> demonstrated Ncd required cluster multiple                   centrosome attachment tight focusing spindle <br /> centrosomes (Supplemental Movie S6). Drosophila dynein                  poles (Goshima et al. 2005). Further validating screen, <br />  <br />  <br /> 2192      GENES &amp; DEVELOPMENT <br />                                                                                                 Suppression multipolar mitosis <br />  <br />  <br />  <br /> we confirmed role MAP Asp pole focusing          actin-based contractility promotes bipolar spindle assembly. <br /> (Wakefield et al. 2001; Morales-Mulia Scholey 2005).          Time-lapse imaging enabled visualization actin- <br /> Additionally, screen identified factors      dependent movement centrosomes cell <br /> that contribute intrinsic cohesion spindle MTs       cortex. Spinning disc microscopy used define the <br /> (Supplemental Table S1). We identified requirement         trajectory centrosome movement Ncd-depleted <br /> of Bj1/RCC1 (RanGEF) centrosome clustering, con-           S2 cells. Centrosomes Ncd-depleted cells exhibited <br /> sistent role preventing multipolar mitosis      striking increase mobility; speed extent <br /> mammalian cells (Chen et al. 2007). We identified         movement increased (Fig. 4A, panel b vs. panel a; <br /> the ADP-ribosylation factors Tankyrase CG15925,          Supplemental Movie S6). Moreover, bulk cen- <br /> putative human PARP-16 homolog. ADP-ribosylation            trosome movement directed away spin- <br /> tankyrase thought contribute spindle bipolarity       dle cell cortex. By contrast, transient <br /> by providing static matrix anchor MT motors         LatA treatment cells reduced centrosome mobility <br /> and spindle proteins (Chang et al. 2005). A role     (Fig. 4A, panel c; Supplemental Movie S5). Moreover, <br /> PARP-16 mitosis previously described.          dramatic mobility centrosomes Ncd-depleted <br />                                                                cells suppressed exposure cells LatA (Fig. <br />                                                                4A, panel e; Supplemental Movie S7). Together, these <br />                                                                results reveal actin-dependent forces centrosomes <br /> Actin-dependent forces spindle intrinsic forces <br />                                                                that, properly organized, suppress spindle mul- <br /> cooperatively prevent multipolar mitoses <br />                                                                tipolarity. <br /> In addition genes likely contribute bun-           Our results provided insight nature the <br /> dling organization spindle MTs, unexpectedly         cortical force generators regulate spindle multipo- <br /> identified genes involved organization regula-      larity. We MT + tip CLIP-190 the <br /> tion actin cytoskeleton, formin Form3/      myosin Myo10A important centrosome cluster- <br /> INF2 (Fig. 1D; Supplemental Table S1; Chhabra              ing (Supplemental Table S1). Drosophila Myo10A a <br /> Higgs 2006). Knockdown of genes does         human Myo15 homolog bind MTs unique <br /> induce cytokinesis failure (Echard et al. 2004; Eggert et      MyTH4-FERM domain. Myo10, member mamma- <br /> al. 2004). Further supporting direct role actin       lian MyTH4-FERM-containing myosin, known be <br /> preventing multipolar mitoses, brief (2 h) treatment         required spindle positioning (Sousa Cheney 2005; <br /> with actin inhibitors similarly induced multipolar mito-       Toyoshima Nishida 2007). RNAi Myo10A, but <br /> ses S2 cells (Fig. 3A). Live cell imaging S2 cells       Drosophila MyTH4-FERM-containing <br /> revealed actin required initial clus-   myosin Myo7, induced twofold increase centrosome <br /> tering multiple centrosomes. Relative controls           clustering defects (Fig. 4B) cytokinesis failure (data <br /> (14.7 &#194;&#177; 6.4 min), 1.5-fold delay centrosome     shown). Moreover, knockdown Myo10A did not <br /> clustering 13 15 LatA-treated cells (22.1 &#194;&#177; 12.3 min).   additive effect spindle multipolarity cells <br /> The remaining 15 cells completely failed cluster     concomitantly treated LatA (Fig. 4B). Finally, <br /> extra centrosomes (Fig. 3B; Supplemental Table S3;             centrosome tracking Myo10A-depleted cells revealed a <br /> Supplemental Movie S5). The cell cycle delay induced        similar effect centrosome movement LatA treatment <br /> LatA treatment likely activation SAC,      (Figs. 4A, panels c,d; Supplemental Movie S8); cells de- <br /> evidenced prominent labeling kinetochores           pleted Myo10, nondirected greatly reduced move- <br /> BubR1 LatA-treated cells (Supplemental Fig. S4). The        ments centrosomes detected contrast ex- <br /> fact LatA treatment synergistic Ncd       tensive cortically directed movement shown Ncd-de- <br /> or Bj1/RCC1-depletion suggested actin cytoskel-       pleted cells (Fig. 4A, panels b,d). Together, data suggest <br /> eton intrinsic spindle forces cooperate suppress        multiple centrosomes organized combinatorially <br /> spindle multipolarity (Fig. 3C).                               spindle-intrinsic forces actin-dependent cortical <br />    Next, determined cortical contraction required     forces acting astral MTs. <br /> for centrosome clustering. Cells exposed soluble <br /> tetravalent lectin concanavalin A (Con-A), cross- <br />                                                                Cell shape, cell polarity, adhesion effects <br /> links plasma membrane globally blocks cor- <br />                                                                spindle multipolarity <br /> tical contraction (Canman Bement 1997). This treat- <br /> ment induced centrosome clustering defects (Fig. 3A). Fur-     Our screen identified requirement genes implicated <br /> thermore, enhancing myosin II-based              cell adhesion centrosome clustering: Turtle, Echi- <br /> contractility suppress spindle multipolarity. Low con-     noid, Cad96Ca, CG33171, Fit1 (Supplemental Table <br /> centrations calyculin A (CA) inhibit myosin light-      S1). The Drosophila FERM domain containing protein <br /> chain phosphatase (MLCP) promote myosin II activa-         Fit1 appears highly conserved function regu- <br /> tion altering distribution (Gupton Water-      lating cell&#226;&#8364;&#8220;matrix adhesion higher eukaryotes (Tu <br /> man-Storer 2006). Wild-type S2 cells treated CA     et al. 2003). The mammalian Fit1 homolog, Mig-2/hu- <br /> modest decrease centrosome clustering defects (15%&#226;&#8364;&#8220;         man PLEKHC1, localizes focal adhesions (FAs) is <br /> 9%). Moreover, CA treatment partially rescued cen-         important integrin-mediated cell adhesion modu- <br /> trosome clustering defect induced Ncd RNAi (Fig.            lation cell shape linking integrins actin cytoskel- <br /> 3D). Thus, cells extra centrosomes, normal actin   eton (Tu et al. 2003). The uncharacterized CG33171 pro- <br />  <br />  <br />                                                                                            GENES &amp; DEVELOPMENT             2193 <br />  Kwon et al. <br />  <br />  <br />  <br />  <br /> Figure 3. Actin-dependent forces cooperate spindle intrinsic forces cluster supernumerary centrosomes. (A) Actin require- <br /> ment centrosome clustering S2 cells. Cells treated Latrunculin (40 &#194;&#181;M LatA), Cytochalasin D (20 &#194;&#181;M), Con-A (0.25 <br /> mg/mL) 2 h percentage centrosome clustering defects determined. Graph shows average independent <br /> experiments (mean &#194;&#177; SD, [**] P &lt; 0.005, Student&#226;&#8364;&#8482;s t-test). (B) Live cell imaging used measure time NEBD centrosome <br /> clustering S2 cells expressing GFP-SAS-6 mCherry-&#226;?&#163;-tubulin presence absence LatA. There delay centrosome <br /> clustering LatA-treated cells (P &lt; 0.02, Student&#226;&#8364;&#8482;s t-test), asterisks indicate cells failed cluster centrosomes (Supple- <br /> mental Movie S5). (C) Percentage cells centrosome clustering defects RNAi Ncd Bj1 (RCC1) conjunction <br /> with LatA treatment (2 h). (D) Calyculin A (0.75 nM 2 h) partially rescues centrosome clustering defect Ncd RNAi-treated <br /> cells. Graph shows average independent experiments; mean &#194;&#177; SD. (*) P &lt; 0.05, Student&#226;&#8364;&#8482;s t-test. Bar, 10 &#194;&#181;m. <br />  <br />  <br /> tein homology mammalian Col18A, previously                     polarity genes Crumbs Cornetto, impor- <br /> implicated regulation cell matrix adhesion (Dix-              tant astral MT function, asymmetric cell division and <br /> elius et al. 2002). Turtle Echinoid containing fibro-               epithelial polarity (Bulgheresi et al. 2001; Tepass et al. <br /> nectin (FN) type III domains involved cell&#226;&#8364;&#8220;cell                  2001; Munro 2006). A number genes been <br /> adhesion (Bodily et al. 2001; Wei et al. 2005). In addition             identified previously requirement to <br /> we identified posterior/lateral polarity gene PAR-1                 maintain normal interphase cell shape adhesion <br /> (PAR-1/MARK/KIN1 family member) apical                          (Kiger et al. 2003; Bakal et al. 2007). <br />  <br />  <br /> 2194      GENES &amp; DEVELOPMENT <br />                                                                                                           Suppression multipolar mitosis <br />  <br />  <br />  <br />  <br /> Figure 4. Genes involved cell adhesion cell polarity affect spindle multipolarity actin cytoskeleton. (A) Visualization <br /> of actin-dependent cortical forces centrosomes. Centrosome tracking S2 cells expressing GFP-SAS-6 mCherry-&#226;?&#163;-tubulin <br /> (inset, t = 0 min, NEBD) treated control RNAi (panel a), Ncd RNAi (panel b) (Supplemental Movie S6), LatA (panel c) (Supple- <br /> mental Movie S5), Myo10A RNAi (panel d) (Supplemental Movie S8), Ncd RNAi/LatA (panel e) (Supplemental Movie S7). Images <br /> were taken indicated time (minutes) NEBD. Centrosome trajectories shown blue (start path) red (end path). <br /> The direction movements indicated white arrows. (B) Requirement actin, cell polarity, adhesion genes clustering <br /> supernumerary centrosomes S2 cells. Centrosome clustering defects RNAi LatA treatment (2 h). (C) Percent- <br /> age centrosome clustering defects RNAi genes involved cell adhesion, polarity, actin conjunction with <br /> Myo10A RNAi. Graph shows average independent experiments (mean &#194;&#177; SD). LatA treatment single gene <br /> knockdowns (&#226;&#710;&#8217;LatA) induce centrosome clustering defects significantly control RNAi levels. P &lt; 0.05, Student&#226;&#8364;&#8482;s t-test. Bar, 10 &#194;&#181;m. <br />  <br />  <br />   How genes controlling interphase cell shape                Conservation mechanisms prevent <br /> adhesion affect spindle morphology? One appealing hy-                  multipolar mitoses <br /> pothesis genes control organization of <br /> cortical actin-based forces act astral MTs. Indeed,            We determined mammalian cancer cells use similar <br /> LatA treatment Myo10 depletion showed enhance-                   mechanisms cluster multiple centrosomes examine <br /> ment spindle multipolarity combined                       relevance screen cancer. Furthermore, tech- <br /> depletion CG33171, Fit1, Crumbs, Cornetto, PAR-1                 niques available manipulating cell adhesion mam- <br /> proteins (Fig. 4B,C), suggesting genes influ-               malian cells enabled directly characterize the <br /> ence centrosome clustering actin cytoskeleton.                 adhesion pattern influences spindle organization. Al- <br />  <br />  <br />                                                                                                     GENES &amp; DEVELOPMENT              2195 <br />  Kwon et al. <br />  <br />  <br />  <br /> though variability efficiency, clus-      ond, tetraploid BSC-1 cells, RFs readily <br /> tering extra centrosomes commonly observed            visualized DIC imaging, noted strong correlation <br /> mammalian cells (Supplemental Table S4 describes           positioning RFs cells <br /> cells characterized below).                                    underwent bipolar (bipolar distribution RFs) multi- <br />    Transient actin disruption led significant increase    polar division (isotrophic distribution RFs) (Fig. 6C; <br /> in frequency multipolar spindles cell lines     Supplemental Movies S9, S10). Third, RFs accumulate <br /> contained multiple centrosomes cells           specific proteins, ERM protein ezrin, which <br /> normal centrosome number (Fig. 5A; Supplemental Table          implicated cortical heterogeneity local <br /> S4). These multipolar spindles result declustering     force generation astral MTs. Disruption corti- <br /> extra centrosomes splitting/frag-      cal heterogeneity src kinase inhibitor PP2 (Thery <br /> mentation centrioles (Fig. 5A). Actin presumably influ-     et al. 2005), induced multipolar spindles MDA- <br /> ences centrosome positioning forces astral MTs.         231 MCF-7 cells (Supplemental Fig. S7A). <br /> Consistent idea, low-dose nocodazole treat-          Fourth, evaluate role cell&#226;&#8364;&#8220;matrix adhesion for <br /> ment, selectively disassembling astral MTs, increased          efficiency mitosis, examined cells plated on <br /> the frequency multipolar spindles specifically cells     different concentrations FN vary strength of <br /> with extra centrosomes (Supplemental Fig. S5).                 cell&#226;&#8364;&#8220;matrix attachment. We observed concentra- <br />    The parallel Drosophila mammalian               tions FN inhibit FA turnover (30 &#194;&#181;g/mL) in- <br /> cells extended genetic requirements centro-         creased frequency multipolar spindles MDA- <br /> some clustering. siRNA Ncd homolog HSET (a              231 cells MCF-7 cells (Supplemental Fig. S7B). <br /> Kinesin-14 member) Myo10 increased frequency           Moreover, effect reversed CA, which <br /> of multipolarity, specifically cells harboring multiple     promotes FA turnover increasing cortical contractil- <br /> centrosomes (Fig. 5B&#226;&#8364;&#8220;E). As S2 cells, Myo10-induced         ity (Supplemental Fig. S7C; Gupton Waterman- <br /> multipolarity consequence cytokinesis failure      Storer 2006). <br /> (data shown). Finally, determined              To directly test role cell adhesion pattern <br /> actin cytoskeleton influences centrosome organization          RF positioning centrosome clustering, used <br /> in parallel intrinsic spindle pole clustering forces.     FN micropatterns manipulate cell adhesion patterns <br /> Disruption actin HSET combinatorial          (Thery et al. 2005). We observed MDA-231 cells <br /> effect. By contrast, LatA treatment did increase       plated Y-shaped O-shaped micropatterns caused <br /> frequency spindle multipolarity cells Myo10        significant (three- fourfold) increase multipolar <br /> was depleted (Fig. 5F). Thus, similar overlapping mecha-       spindles compared controls (Fig. 6D,E). In con- <br /> nisms prevent multipolar mitoses mammalian cancer           trast, plating cells H-shaped micropattern suppressed <br /> cells Drosophila S2 cells.                                 frequency multipolar spindles relative control <br />                                                                cells (2%, half control). Thus, O Y arrange- <br />                                                                ments adhesive contacts bias cells multipolar mi- <br /> Interphase cell shape, adhesion, multipolar <br />                                                                toses, bipolar arrangements adhesive contacts <br /> mitoses <br />                                                                (H-shape) promote bipolar mitoses. Our findings demon- <br /> Although cells round mitosis, preserve            strate interphase cell adhesion pattern cell <br /> memory interphase shape retaining actin-           shape remarkable influence fidelity of <br /> containing retraction fibers (RFs) linked sites strong   mitosis, specifically cancer cells containing extra cen- <br /> cell&#226;&#8364;&#8220;matrix adhesion (Mitchison 1992; Thery Born-          trosomes. <br /> ens 2006). The interphase adhesion pattern dis- <br /> tribution actin-containing RFs known strongly <br />                                                                Disruption centrosome clustering selectively <br /> influence spindle orientation mitosis (Thery et al. <br />                                                                kill cancer cells <br /> 2005). The finding screen preventing multi- <br /> polar mitoses requires cell&#226;&#8364;&#8220;matrix adhesion genes     In principle, disruption centrosome clustering could <br /> actin regulators, suggested appealing hypothesis: These     selective effect viability cancer cells <br /> gene products act cooperatively organize extra        containing multiple centrosomes somatic <br /> centrosomes affecting distribution and/or com-      cells centrosomes mitosis. As first <br /> position RFs cortical force generators.            step evaluating potential therapeutic strat- <br />    Several lines evidence support hypothesis. First,   egy characterized sensitivity different cancer <br /> we used live-cell imaging correlate interphase cell         cell lines knockdown HSET. <br /> shape pattern cell division mitosis.          HSET particularly interesting therapeutic target <br /> MDA-231 (breast cancer cells containing extra centro-          nonessential cell division normal <br /> somes) assumed elongated polarized shape         cells, kinesins amenable small molecule inhi- <br /> interphase, uniformly underwent bipolar divi-           bition (Mayer et al. 1999; Mountain et al. 1999). We <br /> sions (Fig. 6A,B; Supplemental Fig. S6). By contrast,          depletion HSET siRNA leads in- <br /> MDA-231 cells assumed round shape interphase         crease multipolar spindles human cancer cells con- <br /> had increased frequency multipolar divisions. Note       taining multiple centrosomes (Fig. 5D). To determine <br /> that fixed MDA-231 cells multipolar anaphases           consequences centrosome declustering, moni- <br /> contain extra centrosomes (Supplemental Fig. S6). Sec          tored cell division multiple cell lines contain ex- <br />  <br />  <br /> 2196      GENES &amp; DEVELOPMENT <br />                                                                                                               Suppression multipolar mitosis <br />  <br />  <br />  <br />  <br /> Figure 5. Mechanisms centrosome clustering conserved mammalian cells. (A) Actin requirement clustering centrosomes <br /> in cancer cells. Quantitation multipolar spindles treatment DMSO, LatA (5 &#194;&#181;M) DCB (10 &#194;&#181;M) 2 h cancer cell <br /> lines. N1E-115 cells stained MTs (&#226;?&#163;-tubulin, green), centrioles (centrin2, red) DNA (blue). (B,C) Western blots showing <br /> the depletion HSET (B) Myo10 (C) 3 d siRNA MDA-231 MCF-7 cells (ctr, control). (D) HSET Myo10 <br /> requirement centrosome clustering. Quantitation mitotic cells multipolar spindles treatment HSET (&#226;&#710;&#8217;HSET) or <br /> Myo10 (&#226;&#710;&#8217;Myo10) siRNA 3 d MDA-231 MCF-7. (E) Representative images D (&#226;?&#163;-tubulin, red; centrin2, green; DNA, blue). <br /> (F) The percentage mitotic MDA-231 cells multipolar spindles treatment siRNA HSET Myo10, (+) and <br /> without (&#226;&#710;&#8217;) LatA. Note percentage multipolar spindles cells extra centrosomes graphics figure. <br /> Graph shows average independent experiments; mean &#194;&#177; SD. (*) P &lt; 0.05; (**) P &lt; 0.005; (***) P &lt; 0.001, Student&#226;&#8364;&#8482;s t-test. Bar, 10 <br /> &#194;&#181;m. <br />  <br /> tra centrosomes using DIC microscopy (Fig. 7A; Supple-                   dramatic increase multipolar anaphases (88%) in <br /> mental Fig. S8). We depletion human HSET                   N1E-115 cells nearly 100% cells contain extra <br /> confirmed independent siRNAs induced                        centrosomes (Fig. 7A; Supplemental Movies S11, S12; <br />  <br />  <br />                                                                                                         GENES &amp; DEVELOPMENT              2197 <br />  Kwon et al. <br />  <br />  <br />  <br />  <br /> Figure 6. Interphase adhesion pattern controls bipolar/multipolar decision. (A) Bright-field images representing different <br /> interphase cell shapes MDA-231 cells: elongated, polarized, round. (B) Elongated cell morphology correlated bipolar <br /> division. Quantitation bipolar multipolar divisions live-cell imaging MDA-231 cells different interphase shapes <br /> (n = 279; 45% elongated, 46% polarized, 9% round). (C) RF position correlates bipolar multipolar divisions. Representative <br /> images DIC movies 4N BSC-1 cells interphase (left) mitosis (right). Yellow asterisks indicate RFs. Sixty-six <br /> binucleated cells imaged independent experiments. (D) Interphase cell shape controls spindle morphology cells with <br /> extra centrosomes. MDA-231 cells plated FN micropatterns (H, O, Y) stained MTs (&#226;?&#163;-tubulin, green), centrosomes <br /> (pericentrin, red), DNA (blue). (E) Percentage cells multipolar spindles MDA-231 cells plating different patterns <br /> (H, O, Y). Note percentage multipolar spindles quantified metaphase cells. Controls obtained from <br /> cells attached unconstrained FN micropatterns. Graph represents independent experiments. Bar, 10 &#194;&#181;m. <br />  <br />  <br /> Spiegelman et al. 1979). A similar result obtained                 HSET knockdown effect cell division a <br /> with MDA-231 cells &#226;&#710;&#188;50% cells contain extra                   variety diploid control cells (Supplemental Fig. S8). <br /> centrosomes (24% multipolar anaphases HSET                         Strikingly, depletion HSET N1E-115 cells for <br /> depletion) (Supplemental Fig. S8) tetraploid BJ               6 d reduced cell viability &gt;90%. By DIC imaging, most <br /> and NIH-3T3 cells extra centrosomes. By contrast,                 cells appeared undergo apoptosis, the <br />  <br />  <br /> 2198      GENES &amp; DEVELOPMENT <br />                                                                                                Suppression multipolar mitosis <br />  <br />  <br />  <br />  <br />                                                                     Figure 7. Forced declustering centrosomes kills can- <br />                                                                     cer cells extra-centrosomes. (A) Multipolar ana- <br />                                                                     phases induced HSET depletion N1E-115 cells. <br />                                                                     Representative images DIC movies N1E- <br />                                                                     115 cells HSET (n = 435) control (n = 265) <br />                                                                     siRNA. Colored arrowheads indicate mitotic cells im- <br />                                                                     mediately onset anaphase. Bar, 10 <br />                                                                     &#194;&#181;m. (B) Loss cell viability inhibition colony <br />                                                                     formation N1E-115 cells 6 d HSET siRNA. <br />                                                                     (Left panel) Relative cell number control HSET- <br />                                                                     depleted (&#226;&#710;&#8217;HSET) cells independent experi- <br />                                                                     ments 6 d post-transfection. Bright-field images of <br />                                                                     control HSET-depleted N1E-115 cells. (Inset) A sur- <br />                                                                     viving N1E-115 cell HSET depletion stained for <br />                                                                     senescence-associated &#226;?&#164;-galactosidase activity (blue). <br />                                                                     (Right panel) Graph represents average colony num- <br />                                                                     ber independent experiments different <br />                                                                     areas (area, 10 mm2). The corresponding images are <br />                                                                     shown below. (C) HSET RNAi-induced cell death in <br />                                                                     various cancer cell lines proportion fraction of <br />                                                                     cells extra centrosomes. Relative cell number in <br />                                                                     control (CTR) HSET-depleted (&#226;&#710;&#8217;HSET) cells 6 <br />                                                                     d post-transfection. HSET efficiently depleted <br />                                                                     different cell lines shown Supplemental <br />                                                                     Figure S9. Percentage cells cen- <br />                                                                     trosomes (&gt;2c) indicated graph. Graph <br />                                                                     shows average independent experiments. <br />                                                                     All graphics represent mean &#194;&#177; SD. (**) P &lt; 0.005; (***) <br />                                                                     P &lt; 0.001, Student&#226;&#8364;&#8482;s t-test. <br />  <br />  <br />  <br /> surviving cells senescent (Fig. 7B).        Discussion <br /> HSET depletion induced cell death other <br /> cancer cells lines (MDA-231, BT-549, NHO2A)           The association multiple centrosomes can- <br /> rough proportion fraction cells containing extra   cer known nearly 100 years (Boveri 1929). <br /> centrosomes (Fig. 7C; Supplemental Table S4). In con-        Nevertheless, understanding multiple centro- <br /> trast, viability cells possess cen-   somes affect division survival tumor cells re- <br /> trosomes (MCF-7 NIH-3T3) slightly re-           mains poorly understood. Here used genome-wide <br /> duced absence HSET (Fig. 7C) HSET      RNAi approach define mechanisms cells <br /> efficiently knocked RNAi (Fig. 5B; Supplemen-        suppress multipolar mitoses. Our experiments lead to <br /> tal Fig. S9). Thus, centrosome declustering induce       novel conclusions. First, minus <br /> cell death selectively cells supernumerary cen-      end-directed motor HSET/Ncd essential clustering <br /> trosomes.                                                    extra centrosomes. Second, uncovered previously <br />  <br />  <br />                                                                                          GENES &amp; DEVELOPMENT              2199 <br />  Kwon et al. <br />  <br />  <br />  <br /> unappreciated role SAC suppressing spindle        extra centrosomes. The actin cytoskeleton affect <br /> multipolarity. Third, demonstrate interphase ad-     distribution centrosomes mecha- <br /> hesion pattern actin organization profoundly in-     nisms: controlling cell shape, controlling con- <br /> fluence success mitosis cells contain multiple     tractility stiffness cell cortex, influenc- <br /> centrosomes. Finally, findings suggest new thera-      ing cell adhesion pattern organization and <br /> peutic strategy: Blocking centrosome clustering pro-     integrity RFs. Our experiments support key role for <br /> moting multipolar mitoses selectively induce death       interphase adhesion pattern distribu- <br /> in tumors high proportion cells containing         tion RFs. <br /> multiple centrosomes.                                           The effect adhesive geometry centrosome <br />    Our findings illustrate fact centrosome clus-    clustering dramatically demonstrated cells <br /> tering uses processes present normal cells    plated FN micropatterns. Interphase cell adhesion ap- <br /> being unique feature cancer cells. Consistent      pears translated cortical forces pulling spindle <br /> this idea, nontransformed cell types cluster extra   poles mitosis. This possible accu- <br /> centrosomes cytokinesis failure (Supplemental          mulation actin-related cues specific cortical regions <br /> Table S4; Sluder Nordberg 2004). Furthermore, clus-      facing cell adhesion areas/RFs (Thery et al. 2005). Corti- <br /> tering extra centrosomes tumors inferred     cal cues activate force generators pull astral MTs <br /> comparing frequency extra centrioles fre-    tension induce spindle pole movement (Grill <br /> quency bipolar mitoses (Supplemental Table S4;            et al. 2001). Based theoretical considerations, numeri- <br /> Levine et al. 1991; Lingle Salisbury 1999). Most         cal simulations, experiments, proposed <br /> strikingly, flies containing extra centrosomes viable    recently spatial distribution cues induce <br /> because robust centrosome clustering (Basto et al.        torque astral MTs, guiding spindle positioning (Thery <br /> 2008). Nevertheless, efficiency centrosome clus-      et al. 2007). According view, spindle pole separa- <br /> tering varies cancer cell lines; remains in-     tion spindle positioning depend pole&#226;&#8364;&#8220;pole <br /> teresting possibility that, selective pressure,    pole&#226;&#8364;&#8220;cortex mechanical interactions (Supplemental <br /> strength pathways modulated different tu-     Fig. S10). Thus, cells containing multiple centrosomes, <br /> mor types.                                                   specific localization cortical cues corre- <br />                                                              sponding distribution forces promote (O and <br />                                                              Y patterns) suppress (H pattern) multipolar mitoses <br /> The SAC multipolar mitoses <br />                                                              (Fig. 7E). An important direction future work be <br /> A mitotic timing mechanism important suppress-        define various factors identified ex- <br /> ing multipolar mitoses. S2 cells multiple centro-       periments (e.g., PAR-1, PI-3 kinase, Myo10, myosin <br /> somes spend time mitosis cells         II) coordinately influence forces exerted centro- <br /> centrosomes. This delay mediated SAC activation.       somes. <br /> Therefore, SAC activation suppresses multipolar mito- <br /> ses providing additional time centrosome cluster- <br />                                                              Multipolar mitoses cancer therapeutics <br /> ing mechanisms facilitate bipolar spindle assembly. <br /> Thus, data demonstrate multipolar mitoses       Our experiments potential translational implica- <br /> activate SAC S2 cells, recently similar    tions: Inhibiting centrosome clustering tumors a <br /> checkpoint-mediated mitotic delay reported            high proportion cells supernumerary centro- <br /> mammalian tissue culture cells (Yang et al. 2008). In-       somes selectively kill tumor cells. Indeed, found <br /> deed, SAC essential flies containing extra    HSET, nonessential mitosis normal <br /> centrosomes (Basto et al. 2008).                             cells, essential survival cancer cells with <br />   Our findings contrast previous study reporting      extra centrosomes. Thus, like ploidy-specific lethal <br /> that multipolar mitoses activate SAC PtK1      mutations described previously (Storchova et al. <br /> cells (Sluder et al. 1997). Differences cell type   2006), HSET important characteristics appeal- <br /> explain contrasting results. We note Mad2      ing chemotherapeutic target: It essential normal <br /> RNAi abrogated cell cycle delay spindle         cells, essential survival certain <br /> was grossly abnormal, little effect extra cen-    tumor cells. <br /> trosomes scattered bipolar spindle.           There evidence cell context-specific re- <br /> Thus, agreement main conclusions Sluder       quirements HSET/Ncd. Although required <br /> et al. (1997), SAC activated spindles de-    pole organization normal somatic cells, Ncd is <br /> fective produce abnormal kinetochore attach-       essential female meiosis fly cells lack <br /> ment tension, SAC does appear recog-       centrosomes (Endow Komma 1998). Centrosomes <br /> nize abnormal spindles se.                               impose strong bundling force spindle poles that <br />                                                              somatic cells override requirement HSET/ <br />                                                              Ncd. The force generated HSET/Ncd necessary in <br /> The influence actin matrix adhesion <br />                                                              meiotic divisions centrosomes absent (Moun- <br /> on frequency multipolar mitoses <br />                                                              tain et al. 1999). Thus, speculate forces hold- <br /> Our screen identified critical role actin cyto-    ing centrosomes cancer cells in <br /> skeleton suppressing multipolar mitoses cells     common forces bundle minus ends of <br />  <br />  <br /> 2200      GENES &amp; DEVELOPMENT <br />                                                                                                           Suppression multipolar mitosis <br />  <br />  <br />  <br /> MTs acentrosomal spindles (Karsenti Vernos 2001;                Cell culture drug treatment <br /> Nigg 2002). Although HSET necessarily mu-                 Cell lines, drug treatments, fluorescent protein constructs, and <br /> tated, amplified, over- underexpressed tumors,             transfection conditions described Supplemental Ma- <br /> interesting note wide variation HSET             terial. <br /> expression tumors (http://cgap.nci.nih.gov/SAGE/Ana- <br /> tomicViewer). Perhaps tumors glioblasto- <br />                                                                        Immunofluorescence microscopy live cell imaging <br /> mas HSET expression increased adaptive response <br /> to promote centrosome clustering. HSET therefore, illus-               Protocols immunofluorescence, live-cell imaging, anti- <br /> trates class cancer-dependency factors                 bodies used study described Supplemental <br /> only identified functional screens directed ex-               Material. Briefly, 3D time-lapse (4D) imaging performed in <br />                                                                        S2 cells using spinning-disk confocal microscopy (Yokogawa). <br /> periments. <br />                                                                        Long-term DIC imaging mammalian cells performed on <br />    In addition multiple centrosomes, screen iden-               Nikon TE2000E inverted microscope temperature- and <br /> tified features cells influence extent            CO2-controlled environment. <br /> which cells require HSET/Ncd. The strength orga- <br /> nization interphase adhesion sites strikingly influ- <br />                                                                        Microcontact printing <br /> ences cells multiple centrosomes undergo <br /> bipolar multipolar mitoses. These findings suggest               FN micropatterns glass coverslips using micro- <br /> interesting mechanism tumor microenvi-                    contact printing technique adapted work Fink et al. <br /> ronment influence genetic stability. N1E-115 neu-                (2007) important modifications allow better <br /> roblastoma cells classic &#226;&#8364;&#339;small round blue-cell&#226;&#8364;?                 stamping FN patterns glass (Supplemental Material). <br />                                                                        MDA-231 cells plated micropatterns 4&#226;&#8364;&#8220;5 h prior to <br /> tumor. These cells low integrin &#226;?&#164;-1 expression and <br />                                                                        fixation. <br /> can attach laminin FN vitro (Sarner et al. <br /> 2000; Amendola et al. 2001). The round shape inef- <br /> ficient substrate attachment cells con-              Acknowledgments <br /> tribute dramatic requirement HSET. Like- <br /> wise, results screen suggest inhibition             We thank B. Mathey-Prevot HMS Drosophila RNAi <br /> HSET synergistic PI-3 kinase inhibitor,                Screening Center (DRSC) Institute Chemistry Cell <br />                                                                        Biology (ICCB) Longwood help screen; D. Glover, <br /> which important cancer therapeutic target (Ward et <br />                                                                        M. Bettencourt-Dias, J. Raff, R. Basto, G. Goshima, C. Sunkel, <br /> al. 2003). Finally, data suggest cancer cells            G. Rogers, S. Rogers, M. Bornens, S. Henikoff, C. Walczak, and <br /> compromised SAC signaling depen-                    S. Cai reagents; R. Basto J. Raff communicating un- <br /> dent HSET.                                                          published results; M. Bettencourt-Dias, A. Kung, R. King, <br />    Despite exciting development oncogene-specific               S. Rogers, S. Yoshida comments manuscript. This <br /> targeted therapies, present therapies avail-              work supported Susan Komen grant M.K., FCT <br /> able small number tumor types (Bild et al. 2006).             grant S.A.G., fellow award LLS N.J.G., and <br /> Conventional cytotoxic chemotherapies remain                       NIH grant GM083299 DFCI/Novartis Drug Discovery <br /> mainstay clinical practice, strong need              Grant D.P. <br /> to develop rational, personalized utilization these <br /> drugs. Recently, significant progress us- <br /> ing genetic information individually tailor use              References <br /> DNA-damaging agents (O&#226;&#8364;&#8482;Connor et al. 2007). Similarly,                 Amendola, R., Martinez, R., Negroni, A., Venturelli, D., Tanno, <br /> the work described suggests ways morpho-                    B., Calabretta, B., Raschella, G. 2001. DR-nm23 expres- <br /> logical features cells used individualize                  sion affects neuroblastoma cell differentiation, integrin ex- <br /> the use anti-mitotic drugs.                                            pression, adhesion characteristics. Med. Pediatr. Oncol. <br />                                                                           36: 93&#226;&#8364;&#8220;96. <br />                                                                        Bakal, C., Aach, J., Church, G., Perrimon, N. 2007. Quan- <br />                                                                           titative morphological signatures define local signaling net- <br /> Materials methods                                                     works regulating cell morphology. Science 316: 1753&#226;&#8364;&#8220;1756. <br />                                                                        Basto, R., Brunk, K., Vinadogrova, T., Peel, N., Franz, A., Khod- <br />                                                                           jakov, A., Raff, J.W. 2008. Centrosome amplification can <br /> RNAi screen data analysis <br />                                                                           initiate tumorigenesis flies. Cell 133: 1032&#226;&#8364;&#8220;1042. <br /> Detailed descriptions RNAi screen data analyses         Bild, A.H., Yao, G., Chang, J.T., Wang, Q., Potti, A., Chasse, D., <br /> be Supplemental Material. S2 cells treated              Joshi, M.B., Harpole, D., Lancaster, J.M., Berchuck, A., et al. <br /> with dsRNA 3.5 d followed 9 h MG132. Cells                 2006. Oncogenic pathway signatures human cancers a <br /> imaged automated microscope using 20&#195;&#8212; air objective.            guide targeted therapies. Nature 439: 353&#226;&#8364;&#8220;357. <br /> In secondary screen, high-resolution 3D images taken          Bodily, K.D., Morrison, C.M., Renden, R.B., Broadie, K. <br /> with Zeiss Axiovert microscope using 40&#195;&#8212; air ELWD objec-              2001. A novel member Ig superfamily, turtle, a <br /> tive (Zeiss). Multipolar spindles scored &#226;&#710;&#188;200 mitotic           CNS-specific protein required coordinated motor control. <br /> spindles RNAi condition. The follow-up experiments              J. Neurosci. 21: 3113&#226;&#8364;&#8220;3125. <br /> RNAi screen repeated MG132, data presented            Boveri, T. 1929. The origin malignant tumors. Williams and <br /> in figures unperturbed cycling cells, unless other-          Wilkins, Baltimore, MD. <br /> wise specified. Statistical analysis data described   Brinkley, B.R. 2001. Managing centrosome numbers game: <br /> Supplemental Material.                                                    From chaos stability cancer cell division. Trends Cell <br />  <br />  <br />                                                                                                     GENES &amp; DEVELOPMENT              2201 <br />  Kwon et al. <br />  <br />  <br />    Biol. 11: 18&#226;&#8364;&#8220;21.                                                       tioning Caenorhabditis elegans embryo. Nature 409: <br /> Bulgheresi, S., Kleiner, E., Knoblich, J.A. 2001. Inscuteable-        630&#226;&#8364;&#8220;633. <br />    dependent apical localization microtubule-binding            Gupton, S.L. Waterman-Storer, C.M. 2006. Spatiotemporal <br />    protein Cornetto suggests role asymmetric cell division.          feedback actomyosin focal-adhesion systems <br />    J. Cell Sci. 114: 3655&#226;&#8364;&#8220;3662.                                           optimizes rapid cell migration. Cell 125: 1361&#226;&#8364;&#8220;1374. <br /> Canman, J.C. Bement, W.M. 1997. Microtubules suppress              Heneen, W.K. 1970. In situ analysis normal abnormal <br />    actomyosin-based cortical flow Xenopus oocytes. J. Cell             patterns mitotic apparatus cultured rat-kangaroo <br />    Sci. 110: 1907&#226;&#8364;&#8220;1917.                                                   cells. Chromosoma 29: 88&#226;&#8364;&#8220;117. <br /> Chang, P., Coughlin, M., Mitchison, T.J. 2005. Tankyrase-1         Karabay, A. Walker, R.A. 1999. Identification microtu- <br />    polymerization poly(ADP-ribose) required spindle             bule binding sites Ncd tail domain. Biochemistry 38: <br />    structure function. Nat. Cell Biol. 7: 1133&#226;&#8364;&#8220;1139.                  1838&#226;&#8364;&#8220;1849. <br /> Chen, J.G. Horwitz, S.B. 2002. Differential mitotic re-            Karsenti, E. Vernos, I. 2001. The mitotic spindle: A self- <br />    sponses microtubule-stabilizing -destabilizing drugs.           machine. Science 294: 543&#226;&#8364;&#8220;547. <br />    Cancer Res. 62: 1935&#226;&#8364;&#8220;1938.                                          Kiger, A.A., Baum, B., Jones, S., Jones, M.R., Coulson, A., Ech- <br /> Chen, T., Muratore, T.L., Schaner-Tooley, C.E., Shabanowitz, J.,          everri, C., Perrimon, N. 2003. A functional genomic <br />    Hunt, D.F., Macara, I.G. 2007. N-terminal &#226;?&#163;-methyla-               analysis cell morphology using <span id='am-237' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-238' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-240' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-246' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-249' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-261' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-296' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-302' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span>RNA interference</span>. J. Biol. <br />    tion RCC1 necessary stable chromatin association             2: 27. <br />    normal mitosis. Nat. Cell Biol. 9: 596&#226;&#8364;&#8220;603.                     Levine, D.S., Sanchez, C.A., Rabinovitch, P.S., Reid, B.J. <br /> Chhabra, E.S. Higgs, H.N. 2006. INF2 Is WASP homology 2             1991. Formation tetraploid intermediate associated <br />    motif-containing formin severs actin filaments accel-         development cells centrioles <br />    erates polymerization depolymerization. J. Biol.              elastase-simian virus 40 tumor antigen transgenic <br />    Chem. 281: 26754&#226;&#8364;&#8220;26767.                                                mouse model pancreatic cancer. Proc. Natl. Acad. Sci. 88: <br /> D&#226;&#8364;&#8482;Assoro, A.B., Barrett, S.L., Folk, C., Negron, V.C., Boeneman,          6427&#226;&#8364;&#8220;6431. <br />    K., Busby, R., Whitehead, C., Stivala, F., Lingle, W.L.,        Lingle, W.L. Salisbury, J.L. 1999. Altered centrosome struc- <br />    Salisbury, J.L. 2002. Amplified centrosomes breast cancer:          ture associated abnormal mitoses human breast <br />    A potential indicator tumor aggressiveness. Breast Cancer           tumors. Am. J. Pathol. 155: 1941&#226;&#8364;&#8220;1951. <br />    Res. Treat. 75: 25&#226;&#8364;&#8220;34.                                              Lingle, W.L., Lutz, W.H., Ingle, J.N., Maihle, N.J., Salisbury, <br /> Dixelius, J., Cross, M., Matsumoto, T., Sasaki, T., Timpl, R.,            J.L. 1998. Centrosome hypertrophy human breast tumors: <br />    Claesson-Welsh, L. 2002. Endostatin regulates endothe-             Implications genomic stability cell polarity. Proc. <br />    lial cell adhesion cytoskeletal organization. Cancer Res.          Natl. Acad. Sci. 95: 2950&#226;&#8364;&#8220;2955. <br />    62: 1944&#226;&#8364;&#8220;1947.                                                      Mayer, T.U., Kapoor, T.M., Haggarty, S.J., King, R.W., <br /> Doxsey, S. 2001. Re-evaluating centrosome function. Nat. Rev.             Schreiber, S.L., Mitchison, T.J. 1999. Small molecule <br />    Mol. Cell Biol. 2: 688&#226;&#8364;&#8220;698.                                            inhibitor mitotic spindle bipolarity identified pheno- <br /> Echard, A., Hickson, G.R., Foley, E., O&#226;&#8364;&#8482;Farrell, P.H. 2004.           type-based screen. Science 286: 971&#226;&#8364;&#8220;974. <br />    Terminal cytokinesis events uncovered RNAi screen.         Mitchison, T.J. 1992. Actin based motility retraction fibers <br />    Curr. Biol. 14: 1685&#226;&#8364;&#8220;1693.                                             mitotic PtK2 cells. Cell Motil. Cytoskeleton 22: 135&#226;&#8364;&#8220;151. <br /> Eggert, U.S., Kiger, A.A., Richter, C., Perlman, Z.E., Perrimon, N.,   Morales-Mulia, S. Scholey, J.M. 2005. Spindle pole organi- <br />    Mitchison, T.J., Field, C.M. 2004. Parallel chemical genetic       zation Drosophila S2 cells dynein, abnormal spindle <br />    genome-wide RNAi screens identify cytokinesis inhibitors           protein (Asp), KLP10A. Mol. Biol. Cell 16: 3176&#226;&#8364;&#8220;3186. <br />    targets. PLoS Biol. 2: e379. doi: 10.1371/journal.pbio.         Mountain, V., Simerly, C., Howard, L., Ando, A., Schatten, G., <br />    0020379.                                                               Compton, D.A. 1999. The kinesin-related protein, <br /> Endow, S.A. Komma, D.J. 1998. Assembly dynamics                HSET, opposes activity Eg5 cross-links microtu- <br />    anastral:astral spindle: The meiosi <br /> </body></html>