<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br />                Herpes Simplex Virus Type 1 DNA Cleavage and <br />             Encapsidation Require the Product of the UL28 Gene: <br />         Isolation and Characterization of Two UL28 Deletion Mutants <br />  <br />  <br />               The herpes simplex virus type 1 UL28 gene contains a 785-amino-acid open reading frame that codes for an <br />            essential protein. Studies with temperature-sensitive mutants which map <span id='am-6' about='protege:TO' typeof='owl:Thing'>to</span> the UL28 gene indicate that the <br />            UL28 gene product (ICP18.5) is required for packaging of viral DNA and for expression of viral glycoproteins <br />            on the surface of infected cells (C. Addison, F. J. Rixon, and V. G. Preston, J. Gen. Virol. 71:2377-2384, 1990; <br />            B. A. Pancake, D. P. Aschman, and P. A. Schaffer, J. Virol. 47:568-585, 1983). In this study, we describe the <br />            isolation of two UL28 deletion mutants that were constructed and propagated in Vero cells transformed with <br />            the UL28 gene. The mutants, gCB and gCA7B, contained deletions of 1,881 and 537 bp, respectively, in the <br />            UL28 gene. Although the mutants synthesize viral DNA, they fail to form plaques or produce infectious virus <br />            in cells that do not express the UL28 gene. Transmission electron microscopy and Southern blot analysis <br />            demonstrated that both mutants are defective in cleavage and encapsidation of viral DNA. Analysis by cell <br />            surface immunofluorescence showed that the UL28 gene is not required for expression of viral glycoproteins on <br />            the surface of infected cells. A rabbit polyclonal antiserum was made against an Escherichia coli-expressed <br />            Cro-UL28 fusion protein. This antibody reacted with an infected-cell protein having an apparent molecular <br />            mass of 87 kDa. The 87-kDa protein was first detected at 6 h postinfection and was expressed as late as 24 h <br />            postinfection. No detectable UL28 protein was synthesized in gCB- or gCA7B-infected Vero cells. <br />  <br />    The herpes simplex virus type 1 (HSV-1) genome is                   have shown that the products of the UL6, UL25, UL28, <br /> 152,260 bp in size and contains at least 72 open reading               UL32, and UL33 genes are required for formation of full <br /> frames (27). Genetic and biochemical evidence suggests that            capsids (1-3, 45, 46). At the nonpermissive temperature <br /> at least 30 of the proteins encoded by these open reading              these mutants synthesize near-wild-type levels of viral DNA <br /> frames are components of the HSV-1 virion (6, 17, 26, 38,              but are defective in nucleocapsid maturation, accumulating <br /> 48). The architecture of the HSV-1 virion consists of a                only type B capsids, and the viral DNA remains in a <br /> trilaminar lipid envelope, an amorphous layer known as the             high-molecular-weight (concatemeric) form. The DNA syn- <br /> tegument, an icosohedral capsid, and a DNA-containing                  thesized at the nonpermissive temperature by these ts mu- <br /> core (11, 42, 47, 51). The HSV-1 capsid is assembled in the            tants can be cleaved and packaged after shifting to the <br /> nuclei of virus-infected cells, and seven polypeptides have            permissive temperature but this is dependent on the synthe- <br /> been identified as capsid components (7, 11, 13, 16, 17). The          sis of new proteins and capsids following the temperature <br /> seven capsid proteins are the products of six HSV-1 genes.             shift. These results indicate that the cleavage of viral DNA <br /> VP5, VP19C, VP22a, VP23, VP24, and VP26 are the prod-                  and its encapsidation are tightly coupled events. In addition, <br /> ucts of genes UL19, UL38, UL26.5, UL18, UL26, and                      these results are consistent with a model of nucleocapsid <br /> UL35, respectively (8, 10, 24, 28, 29, 37, 39-41, 43, 50).             assembly in which empty capsids are filled with concate- <br /> VP21 is probably the product of the UL26 gene, but this                meric DNA which is cleaved when a genome equivalent is <br /> assignment has not been directly confirmed (8). Three dif-             packaged into the capsid. <br /> ferent capsid types, designated A, B, and C or empty,                     The UL28 gene is a highly conserved gene within the <br /> intermediate, and full, respectively, have been identified in          herpesvirus family. Homologs have been found in pseudo- <br /> wild-type virus-infected cells (4, 13, 34-36, 44). Type A              rabies virus, HSV-2, Epstein-Barr virus, human cytomega- <br /> capsids are intranuclear capsids devoid of DNA and internal <br /> structure. Type B capsids are also intranuclear capsids that           lovirus, varicella-zoster virus, equine herpesvirus, Marek&apos;s <br /> lack DNA but exhibit internal structural features. Type C              disease virus, bovine herpesvirus, and herpesvirus saimiri <br /> capsids contain DNA and a densely staining core and are the            (reviewed in reference 32). A 95-kDa protein has been <br /> capsids which are enveloped and released <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> infected                 identified in HSV-1-infected cells by using rabbit antiserum <br /> cells.                                                                 made against a synthetic peptide that was synthesized from <br />    In addition to the six HSV-1 genes required for capsid              the predicted sequence of the UL28 protein (33). In pseu- <br /> assembly, at least five HSV-1 genes that are required for              dorabies virus, the UL28 homolog has been shown to be a <br /> cleavage and encapsidation of viral DNA have been identi-              79-kDa protein that plays an essential role in the cleavage <br /> fied. Studies with HSV-1 temperature-sensitive (ts) mutants            and encapsidation of viral DNA (30, 33). Two different ts <br />                                                                        mutants have been isolated, which map within the UL28 <br />                                                                        gene of HSV-1 (2, 31), and two very different phenotypes <br />                                                                        have been described for these two mutants. tsZ47 fails to <br />   * <br />       Corresponding author.                                            express viral glycoproteins on the surface of infected cells, <br />                                                                 3470 <br />  VOL. 67, 1993                                         CHARACTERIZATION OF HSV-1 UL28 DELETION MUTANTS                      3471 <br />  <br /> whereas ts1203 is defective in cleavage and encapsidation of      control of the isopropyl-,-D-thiogalactopyranoside (IPTG)- <br /> viral DNA. To further investigate the role of the UL6, UL25,      inducible Lac promoter, and translation terminates at the <br /> UL28, UL32, and UL33 genes in cleavage and encapsidation          UL28 TAG stop codon. pECH85 was constructed in the <br /> of viral DNA, we have set out to construct additional             following manner. A 1.47-kb SmaI fragment containing the <br /> mutants with mutations in these genes. In this paper, we          5&apos; end of the HSV-1 UL28 gene was isolated from pKBG (9) <br /> describe the isolation and characterization of two UL28 null      and ligated into the HincIl site of pBlueScript SK(-) (Strat- <br /> mutants. The mutants were propagated on cell lines express-       agene, La Jolla, Calif.) to generate pECH76. The 3&apos; end of <br /> ing the wild-type UL28 gene, and the <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> of the UL28        the UL28 gene was added as a 2.77-kb BstEII-FspI fragment <br /> mutants was characterized. In this report, we identify the        to the unique BstEII and EcoRV sites of pECH76 to generate <br /> UL28 gene product and demonstrate that it is required for         pECH82. A 3.1-kb BamHI-ClaI fragment isolated from <br /> cleavage and encapsidation of viral DNA but that it is not        pECH82 was then cloned into the unique BamHI and ClaI <br /> needed for cell surface expression of viral glycoproteins.        sites of pHK412 (32) to generate pECH85. <br />    (These results were presented at the 17th International           Transformation of Vero cells. At 1 day prior to transfec- <br /> Herpesvirus Workshop, Edinburgh, Scotland, August 1992.)          tion, approximately 2 x 106 Vero cells were seeded in <br />                                                                   100-mm-diameter culture dishes. Cells were cotransformed <br />                 MATERIALS AND METHODS                             with calcium phosphate-precipitated DNA (6 ,ug of pSG18- <br />                                                                   B50 or pSG18-BA50, 0.5 ,ug of pRSV-neo, and 20 ,ug of <br />    Cells, viruses, and monoclonal antibodies. African green       salmon testis DNA) by the method of Graham and van der <br /> monkey kidney cells (Vero; ATCC CCL 81) were propa-               Eb (15). After a 4-h incubation at 37&#194;&#176;C, the medium was <br /> gated and maintained as previously described (21). Vero-Al        removed and the cells were glycerol shocked (15% glycerol). <br /> (Al), Vero-Cl (Cl), and Vero-D6 (D6) cells were propagated        The transfected cells were left in normal growth medium for <br /> and maintained in Dulbecco modified Eagle medium plus             48 h, trypsinized, and split 1:10 into additional 100-mm- <br /> 10% fetal bovine serum and 0.5 mg of G418 (Geneticin;             diameter culture dishes containing medium with 1 mg of <br /> GIBCO-BRL) per ml. HSV-1 KOS was used as the wild-type            G418 per ml. After 14 to 21 days, individual G418-resistant <br /> strain and HSV-1 A2 (22) was used as the parental virus for       colonies were isolated with cloning cylinders. Ten clonal <br /> the isolation of the UL27 and UL28 host range mutants.            isolates from cells transfected with pSG18-B50 (cell lines Al <br /> HSV-1 gB deletion mutant KAT and the gB-complementing             to A10) or pSG18-BA50 (cell lines Cl to C10) were amplified, <br /> cell line D6 were provided by Stan Person, University of          and aliquots were frozen. <br /> Pittsburgh Medical School, Pittsburgh, Pa. (5). Pools of gB-,        Isolation of recombinant viruses. Marker transfer of mam- <br /> gC-, and gD-specific monoclonal antibodies were obtained          malian cells with HSV-1 genomic DNA and cloned DNA <br /> from Joseph Glorioso, University of Pittsburgh Medical            fragments was done as described previously (22). Infectious <br /> School. The gB pool contained antibodies B4, B7, and B8           A2 viral DNA was mixed with a fivefold molar excess of a <br /> (25). The gC antibodies (pool 2) have been described previ-       cloned DNA fragment containing a disrupted UL28 gene and <br /> ously (20). The gD pool contained antibodies D6, D8, and D9       used to transfect Al or Cl cells. The resulting virus stocks <br /> (18).                                                             were screened for the ability to hybridize with a restriction <br />     Plasmids. pRSV-neo contains the bacterial gene for neo-       fragment specific for the region deleted in the A2 virus by an <br /> mycin resistance under control of the Rous sarcoma virus          in situ hybridization procedure (22). The A2 virus contains a <br /> promoter (14). Plasmid pSG18-B50 (the plasmid used to             deletion that removes sequences of the gC gene between <br /> isolate cell line Al; see below and Fig. 1) containing se-        bases -569 and +124 relative to the 5&apos; terminus of the gC <br /> quences of the HSV-1 genome between bases 52588 and               mRNA. The two plasmids used in the cotransfections, <br /> 60362 was described previously (19). Plasmid pSG18-BA50           pSG18-gCB and pSG18-gCA7B, contain a portion of the <br /> (the plasmid used to isolate cell line Cl, see below and Fig.     sequences that are deleted from the A2 virus. Hybridizing <br /> 1) is identical to pSG18-B50 with the exception that a 969-bp     plaques were purified three times on Cl cells before stocks <br /> BstEII fragment (nucleotides 53164 to 54133) was deleted by       were made. <br /> partially digesting pSG18-B50 with BstEII and then religat-          DNA isolation. Total-infected-cell DNA and DNase <br /> ing it with T4 DNA ligase.                                        I-treated total-infected-cell DNA were harvested by the <br />     Plasmids pSG18-gCA7B and pSG18-gCB were constructed           method of Stow et al. (49). Cells were infected at a multi- <br /> from pSG18-B50 in the following manner. pSG18-B50 was             plicity of infection (MOI) of 5. At approximately 3 days <br /> digested with XhoI, and the ends were blunted with T4 DNA         postinfection, cells were removed into a 50-ml conical cen- <br /> polymerase. Following ligation of BglII linkers, the sample       trifuge tube and spun at 2,000 rpm (Sorvall SS34 rotor) for 15 <br /> was digested with BglII and XhoI; the vector fragment             min. The cell pellet was frozen and thawed once, resus- <br /> generated by this digest was gel purified and ligated with a      pended in 1 ml of lysis buffer (0.6% sodium dodecyl sulfate <br /> 268-bp BglII-XhoI fragment of the HSV-1 gC gene (-144 to           [SDS], 10 mM EDTA, 10 mM Tris-HCl [pH 7.5], 500 ,ug of <br />  + 124 region of the gC promoter) isolated from a BglII-XhoI       proteinase K per ml), and held at 37&#194;&#176;C for 4 h. NaCl was <br /> digest of plasmid pGCA7 (22) to yield pSG18-gCA7B. pSG18-          added to a final concentration of 200 mM, and the sample <br /> gCB was constructed by digesting pSG18-gCA7B with BglII           was extracted once with phenol and once with phenol- <br /> and subjecting the product to a partial digest with BstEII; the    chloroform, and the DNA was ethanol precipitated. The final <br /> appropriate vector fragment generated by this digest was           DNA pellet was resuspended in TE buffer (10 mM Tris-HCl <br /> isolated and ligated with a 693-bp BglII-BstEII fragment of        [pH 7], 1 mM EDTA) containing 10 ,g of RNase A per ml. <br /> the HSV-1 gC gene (the -569 to +124 region of the gC               For preparation of DNaseI-treated DNA, cells were infected <br /> promoter) isolated from a BstEII-BglII digest of plasmid           and harvested as described above and the cell pellet was <br /> pGC (22).                                                          resuspended in 1 ml of rSb buffer (10 mM Tris-HCl [pH 7.4], <br />     The Escherichia coli expression plasmid pECH85 contains        10 mM KCl, 1.5 mM MgCl2) and sonicated, 50 ,ug of RQ1 <br />  a gene fusion consisting of 27 codons of the Xcro protein, 30     RNase-free DNase (Promega) was added, and the sample <br /> codons from the 5&apos; noncoding region of the UL28 gene, and          was incubated at 37&#194;&#176;C for 2 h. SDS, EDTA, and proteinase <br />  all 786 codons of the UL28 gene. Transcription is under the       K were then added as described above, and the sample was <br />  3472    TENGELSEN ET AL.                                                                                             J. VIROL. <br />  <br /> subjected to phenol-chloroform extraction and precipitation     viously described (32). The Western-Light chemilumines- <br /> as described above.                                             cent detection system (Tropix, Inc., Bedford, Mass.) was <br />    HSV-1 DNA encapsidation. Cells were infected with virus      used to identify proteins reactive with the H85 antisera. The <br /> at an MOI of 5 and harvested 3 days postinfection. Total-       filters were blocked overnight at 4&#194;&#176;C in PBS containing 5% <br /> infected-cell DNA was prepared from half of the sample, and     powdered milk. Then H85 antiserum (diluted 1:200) was <br /> DNase I-treated total-infected-cell DNA was prepared from       added to the blocking solution and incubated for 2 h at room <br /> the other half (see above). All of the DNA samples were then    temperature. The filters were subsequently incubated with <br /> digested with BamHI, and Southern blots were prepared as        alkaline phosphatase-conjugated immunoglobulin G in PBS <br /> previously described (22). The blots were probed with the       with 5% powdered milk, washed, and incubated with the <br /> 32P-labeled BamHI K fragment of HSV-1 DNA.                      dioxetane substrate CSPD, and chemiluminescence was <br />    Analysis of viral DNA replication. Viral DNA synthesis       detected by exposure of the membrane to Kodak X-Omat <br /> was determined by the method of Gadler (12) as modified by      XAR5 film (Kodak, Rochester, N.Y.). The H85 antiserum <br /> Zhu and Weller (52). Cells in 75-cm2 flasks were infected       was raised against a Cro-UL28 fusion protein expressed <br /> with virus at an MOI of 5 for 24 h at 37&#194;&#176;C. The flasks were     from E. coli by using plasmid pECH85 (see above). The <br /> washed once with phosphate-buffered saline (PBS), and the       Cro-UL28 protein accumulated as insoluble aggregates in E. <br /> cell monolayer scraped into 1 ml of PBS. The cells were         coli NFPU2 when induced with 1 mM IPTG. The aggregates <br /> pelleted and resuspended in 100 ,ll of PBS and freeze-thawed    were purified as previously described (32), emulsified in <br /> once. A series of fivefold dilutions in PBS were spotted onto   complete Freund&apos;s adjuvant, and injected intramuscularly <br /> GeneScreen Plus (NEN/Dupont) by using a dot blot appara-        into New Zealand White rabbits. Subsequent boosts were <br /> tus (Schleicher &amp; Schuell). The membrane was treated twice      injected intramuscularly in Freund&apos;s incomplete adjuvant. <br /> with 0.4 M NaOH and twice with 1 M Tris (pH 8.0). The blot         Immunoprecipitation. Immunoprecipitation was done as <br /> was probed with a 32P-labeled HSV-specific probe (Sall          described previously (20), with a mixture of the gB and gD <br /> fragment, nucleotides 54826 to 58701, isolated from a SalI      monoclonal antibodies. Antigen-antibody complexes were <br /> digest of pSG18-B50 [see Fig. 1]) as previously described       collected with protein A-Sepharose beads (Pharmacia Fine <br /> (22).                                                           Chemicals, Piscataway, N.J.). <br />    Transmission electron microscopy. Monolayers of either <br /> Vero or Cl cells were grown to 90% confluency and infected                                 RESULTS <br /> with either KOS, gCB, or gCA7B at an MOI of 5. At 12 h <br /> postinfection, the cells were fixed for 30 min with 1%             Isolation of two UL28 deletion mutants. According to the <br /> glutaraldehyde-0.1 M sodium cacodylate buffer (Nacac) and       DNA sequence analyses of McGeoch et al. (27), the 785- <br /> rinsed three times for 10 min with 0.1 M Nacac-7% sucrose.      amino-acid open reading frame for the UL28 gene extends <br /> Saniples were held in this buffer overnight at 4&#194;&#176;C and then     from nucleotide 58159 (ATG) to nucleotide 55802 (TAG) in <br /> postfixed with 1% osmium tetroxide for 1 h at room temper-      the HSV-1 genome (Fig. 1). The 5.6-kb UL28 mRNA and the <br /> ature. The samples were rinsed again in 0.1 M Nacac-7%          3.3-kb UL27 mRNA are expressed as 3&apos;-coterminal tran- <br /> sucrose and embedded in epon resin. Sections 1 ,um thick        scripts, and the promoter for the UL27 gene is contained <br /> were prepared, stained with toluidine blue, and observed by     within the C-terminal coding region of the UL28 gene (the <br /> light microscopy for areas appropriate for thin sectioning.     transcription start site of UL27 gene is at nucleotide 56080). <br /> Samples were then thin sectioned and prepared by contrast       Two plasmids containing deletions in UL28 gene coding <br /> staining with lead citrate and uranyl acetate. Thin sections    sequences were constructed starting from plasmid pSG18- <br /> were visualized on a JEOL 1200ES transmission electron          B50 (Fig. 1). pSG18-gCB contains an 1,881-bp BstEII-XhoI <br /> microscope and subsequently photographed. A minimum of          deletion (nucleotides 57923 to 56042) that removes the cod- <br /> 10 fields were visualized for each sample.                      ing region for amino acids 78 to 706 of the UL28 gene, and <br />    Immunofluorescence. Vero and Cl cells were grown to          pSGl8-gCA7B contains a 537-bp XhoI-XhoI deletion (nucle- <br /> confluency on 35-mm dishes. The cells were infected at an       otides 56579 to 56042) that removes the coding region for <br /> MOI of 2, and at 8 h postinfection they were rinsed with PBS    amino acids 528 to 706 of the UL28 gene. The deletions <br /> three times. They were briefly fixed with freshly prepared      introduced into pSG18-gCB and pSG18-gCA7B remove es- <br /> 3% paraformaldehyde for 10 min at room temperature and          sential promoter sequences from the UL27 gene. To ensure <br /> rinsed three times with PBS and once with PBS-BLOTTO            that these two deletions did not alter expression of the UL27 <br /> (BLOTTO is 2% nonfat dry milk plus 1% bovine serum              gene, the deleted sequences were replaced with sequences of <br /> albumin. Primary antibody (a pool of either gC or gD            the HSV-1 gC promoter (Fig. 1). The gC promoter fragment <br /> monoclonal antibodies) was applied at a 1:1,000 dilution in     inserted into pSG18-gCB consisted of a 693-bp BstEII-BglII <br /> PBS-BLOTTO for 45 min at 37&#194;&#176;C. Monolayers were then             fragment that includes sequences located 569 bp upstream <br /> gently rinsed twice with lx BLOTTO, twice with 0.5 x            and 124 bp downstream of the gC gene transcription start <br /> BLOTTO, and then once with PBS alone. Fluorescein               site, whereas the gC promoter fragment inserted into pSG18- <br /> isothiocyanate-conjugated sheep anti-mouse immunoglobu-         gCA7B consisted of a 268-bp XhoI-BglII fragment that <br /> lin G (Sigma) was applied at a concentration of 1:128 in lx     includes sequences located 144 bp upstream and 124 bp <br /> PfBS-BLOTTO and the cells were incubated for 1 h at room        downstream of the gC gene transcription start site. Both of <br /> temperature (in the dark) and then rinsed three times with      the gC promoter fragments have been previously shown to <br /> PBS. Just prior to observation, the samples were rinsed with    contain complete elements of the gC promoter (22). <br /> a freshly prepared antiquench reagent, a 1-mg/ml p-phe-            The UL28 gene of HSV-1 is essential for viral replication <br /> nylenediamine solution made in PBS-20% glycerol (23).           (2). To propagate viral mutants with deletions in the UL28 <br /> Coverslips were applied, and the fluorescence was observed      gene, we constructed cell lines which express the wild-type <br /> and recorded by fluorescence photomicroscopy.                   UL28 gene. To select for cell lines that carry the UL28 gene, <br />    Western immunoblots and UL28-specific antisera. Proteins     we relied on the fact that the UL28 gene overlaps at its 3&apos; end <br /> were electroblotted onto 0.2-R,m polyvinylidene difluoride      with the essential UL27 (gB) gene (Fig. 1). Vero cells were <br /> memibranes (Transblot; Bio-Rad, Richmond, Calif.) as pre-       cotransformed with plasmids pRSV-neo, which contains the <br />  VOL. 67, 1993                                                                    CHARACTERIZATION OF HSV-1 UL28 DELETION MUTANTS                                   3473 <br />  <br />                                            UL                                    Ls        TABLE 1. Plaque-forming efficiencies of HSV-1 strains on Vero, <br />                                                                                                              Al, Cl, and D6 ceilsa <br />       52,588 53,164 54,133 54,826 56,042 56,579 57,923 58,701            60,362                                                                             Titer <br />            I                           I                                II                  Virus                        Cell line <br />                                                                                                                                                           (PFU/ml) <br />            B    Bs         Bs          S         X   X       Bs         S             B <br />                                                                                            KOS                            Vero                           2.4 x 109 <br />                  OsUL28_______              -i <br />                                                                    -C_ :U~L29 <br />                                                                        5.6 kb mRNA <br />                                                                                       g~                                  Al                             3.0 x 109 <br />                   &lt;rUL28       &gt;                                       ICP18.5                                            Cl                             1.9 x 109 <br />                                                                                                                           D6                             1.6 x 109 <br />                                                  3.3kb mRNA <br />                                                                                            A2                             Vero                           1.3   x 109 <br /> Cell lines                                                                                                                Al                             1.5   x 109 <br />            B                                                                          B                                   Cl                             0.9   x 109 <br /> Al            4 <br />            L- -L.              UL277                 UTL28                                                                D6                             1.0   x 109 <br />                                                                                       B    KAT                            Vero                           &lt;5    x 102 <br /> C1l <br />                     969 <br />                           ip     AUL27               UlL28                                                                Al                             8.8   x   105 <br />            B                                         x                                                                    Cl                             &lt;5    x 102 <br /> D6                             UL27                                                                                       D6                             6.2   x 105 <br />                                                                                            gCB                            Vero                           &lt;5    x   102 <br /> Viruses                                                                                                                   Al                               4   x   107 <br />  B    Bg X                                                                            B                                   Cl                             6.5   x   107 <br /> gCA7BAU2                                                                                                                  D6                             &lt;5    x   102 <br />                                                                                            gCA7B                          Vero                           &lt;5 x      102 <br />                                                                                                                           Al                              1x       107 <br />                                                                                                                           Cl                              1x       107 <br />                                                                                                                           D6                             &lt;5 x      102 <br /> gCB                            UL27 <br />                                                  g <br />                                                      AUL28    Bs                             a Virus stocks were prepared in Vero cells (KOS and A2), D6 cells (KAT), <br />                                                                                            and Cl cells (gCB and gCA7B). <br />            B         969   Xp <br />                                                                    B <br /> KAT            Lfceltetin AUL27                  H~ <br />    FIG. 1. Regions of the HSV-1 genome that were used to isolate                           then plated on Al cells and screened by a previously <br /> transformed cells and recombinant viruses. The HSV-1 genome is                             described, in situ viral plaque hybridization procedure (22) <br /> shown at the top of the figure, with UL and US referring to the long                       for the ability to hybridize with a restriction fragment <br /> and short unique region sequences, respectively. The BamHI region                          specific for the region deleted in the A2 virus. The A2 virus <br /> located between nucleotides 52588 and 60362 of the HSV-1 genome                            contained a deletion that removed bases -569 to + 124 of the <br /> is expanded to demonstrate the genomic location of the UL27 (gB) <br /> and UL28 (ICP18.5) genes (19, 27). The location of the UL27 and                            gC gene (22). Because both pSG18-gCB and pSG18-gCA7B <br /> UL28 mRNA coding sequences are shown with the open boxed                                   contain all or a portion of the -569 to +124 gC promoter <br /> regions representing the open reading frames for these two genes.                          sequences, homologous recombination between the plasmid <br /> For the cell lines, regions of the HSV-1 contained in recombinant                          and viral sequences resulted in the introduction of the gC <br /> plasmids pSG18B50, pSG18BA50, and pKBXX are shown. These                                   sequences at the UL28 locus. The inserted sequences could <br /> plasmids were used to isolate transformed Vero cell lines Al                               then serve as a marker to screen for recombinant products. <br /> (pSG18B50) Cl (pSG18BA50), or D6 (isolated by Cai et al. [5] by                            Positively hybridizing plaques were selected from the two <br /> using pKBXX). For the viruses, plasmids pSG18-gCA7B and                                    transfections, plaque purified three times, and propagated on <br /> pSG18-gCB were constructed and used to isolate the two UL28                                Al cells. The recombinant viruses isolated from transfec- <br /> deletion viruses gCA7B and gCB as described in Materials and                               tions with pSG18-gCB and pSG18-gCA7B were designated <br /> Methods. pSG18-gCB contains an 1,881-bp deletion that removes <br /> nearly all of the coding region of the UL28 gene while pSG18-gCA7B                         gCB and gCA7B, respectively. <br /> contains a 537-bp deletion in the 3&apos; end of the UL28 gene. The                                To confirm that the gCB and gCA7B viruses harbored the <br /> deleted sequences were replaced with the indicated regions of the                          UL28 mutations present in pSG18-gCB and pSGl8-gCA7B, <br /> HSV-1 gC gene. In both recombinant viruses, expression of the                              we tested the growth properties of these two mutants by <br /> UL27 gene is from the gC promoter sequences that were inserted.                            measuring their plaque-forming efficiencies on Al, Vero, and <br /> Abbreviations: B, BamHI; Bg, BglII; Bs, BstEII; H, HindIII; S,                             D6 cells (Table 1). Both mutants were incapable of forming <br /> SalI; X, XhoI.                                                                             plaques in Vero or D6 cells, but both formed plaques equally <br />                                                                                            well in Al cells. KOS and A2 formed plaques on all three cell <br />                                                                                            lines, whereas the gB deletion virus, KAT, formed plaques in <br /> G418 resistance gene, and pSG18-B50, which contains both                                   D6 and Al cells but not in Vero cells. Because Al cells are <br /> the UL27 (gB) and UL28 genes of HSV-1 (Fig. 1). After 2                                    capable of complementing the gB deletion mutant, KAT, we <br /> weeks, 10 G418-resistant colonies were isolated, amplified,                                could not rule out that the gCB and gCA7B viruses harbored <br /> and screened for their ability to complement replication of                                mutations which affected both the UL27 and UL28 genes. <br /> the gB deletion mutant KAT. One of these cell lines,                                       To address this question, we isolated a second UL28- <br /> designated Al, yielded the highest level of complementation                                expressing cell line, designated Cl (Fig. 1). The Cl cell line <br /> and was chosen for further study.                                                          was isolated by cotransfecting Vero cells with plasmids <br />   The mutations in pSG18-gCB and pSG18-gCA7B were                                          pSG18-BA50 and pRSV-neo, and G418 resistant cell lines <br /> transferred into the viral genome by marker transfer in which                              were isolated and tested for the ability to complement the <br /> Al cells were cotransfected with each of the plasmids and                                  growth of mutants gCB and gCA7B. Plasmid pSG18-BA50 is <br /> HSV A2 viral DNA. The progeny from the transfection were                                   identical to pSG18-B50, except that a 969-bp BstEII frag- <br />  <br />                      IGCB <br />                                               A                          FIG. 3. Southern blots of DNA from UL28 deletion mutants. <br />                     7. <br />                                                                       Autoradiographic exposure of a Southern blot of plasmid DNA or <br />                           .oA                                         viral DNA separated on 0.8% agarose gels. Total-infected-cell DNA <br />                     aI <br />                                                                       isolated from Vero cells infected with KOS (lanes 6 to 8), gCB (lanes <br />                                                                       3 to 5), or gCA7B (lanes 9 to 11) and plasmid DNA from pSG18-gCB <br />                                                                       (lanes 1 and 2) and pSG18-gCA7B (lane 12) were digested with either <br />                                                                       BamHI (lanes 1, 3, 6, and 9), BamHI-BglII (lanes 2, 4, 7, and 10), or <br />                      o   i      10   l    X       X                   BamHI-BgiII-SalI (lanes 5, 8, 11, and 12). The blot was hybridized <br />                                                                       with a UL28-specific probe (SalI fragment, nucleotides 54826 to <br />                    TIME POSTINFECTION (h)                             58701, gel purified from a SalI digest of pSG18B50 [Fig. 1]). <br />                                                                       Markers, HindIll digest of lambda DNA. <br />    FIG. 2. Single-step growth of KOS, gCB, and gCA7B. Replicate <br /> cultures of Vero (0), Al (A), and Cl (0) cells were infected with <br /> KOS, gCB, or gCA7B at an MOI of 5. The cultures were harvested <br /> at the indicated times postinfection and freeze-thawed three times,      To demonstrate that the two UL28 mutants contain the <br /> and the yield of virus at each time point was determined by plaque    deletion/insertion mutations at the appropriate position in <br /> titer determination on Al cells.                                      the viral genome, we isolated total-cell DNA from Vero cells <br />                                                                       infected with KOS, gCB, or gCA7B. The DNA samples were <br />                                                                       digested with either BamHI, BamHI-EcoRI, or BamHI- <br /> ment (nucleotides 53164 to 54133) has been deleted (Fig. 1).          EcoRI-Sall, and subjected to gel electrophoresis and South- <br /> The BstEII deletion removes sequences that <span id='am-4' about='Thesaurus:code' typeof='owl:Thing'>code</span> for the               ern blot hybridization (Fig. 3). Plasmids pSG18-gCB and <br /> membrane-spanning domain of the gB protein (the same                  pSG18-gCA7B were also digested with some of the same <br /> deletion as in the KAT virus) and results in the expression of        enzymes and subjected to electrophoresis in parallel with the <br /> a secreted form of the gB protein. When we tested the                 viral DNA. The filter was probed with a 3,839-bp Sall <br /> plaque-forming efficiencies of KOS, A2, gCB, gCA7B, and               fragment (nucleotides 54826 to 58701 [Fig. 1]) that contains <br /> KAT on Cl cells, we found that KAT was the only virus                 the entire UL28 gene and a portion of the UL27 gene. The <br /> which failed to form plaques in these cells (Table 1). The            BamHI digests demonstrate that both gCB and gCA7B <br /> nearly 100-fold difference in titers of KOS and A2 compared           contain BamHI fragments (Fig. 3, lanes 3 and 9) that are <br /> with the mutant viruses is probably the result of low levels of       smaller than the KOS fragment (lane 6) and that the gCB <br /> complementation by the complementing cell lines. Taken                BamHI fragment is identical in size to the BamHI insert of <br /> together, the data from the plaque assays with Al (comple-            pSG18-gCB (compare lanes 1 and 3). The BamHI-BglII <br /> ments both UL27 and UL28 mutants), Cl (complements                    digests demonstrate that the two mutants contain a BglII site <br /> UL28 mutants), and D6 (complements UL27 mutants) cells                near the center of the BamHI fragment, which is not present <br /> demonstrate that both gCB and gCA7B fail to plaque on Vero            in the same BamHI fragment from KOS DNA (lanes 4, 7 and <br /> cells because of a UL28 <span id='am-163' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-164' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-239' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-240' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span>mutation</span>. Intracellular replication of        10) and that identical-sized BamHI-BgiII fragments are <br /> the two UL28 deletion mutants was examined by establish-              detected in pSG18-gCB (lane 2) and gCB (lane 4). Finally, <br /> ing single-step growth curves for KOS, gCB, and gCA7B in              the BamHI-BglII-SalI digests clearly shows that the gCA7B <br /> Vero, Al, and Cl cells. Samples from each infected culture            (lane 11) virus contains the same hybridizing fragments as <br /> were harvested at various times postinfection and assayed             pSG18-gCA7B (lane 12). We concluded from these Southern <br /> for infectious virus by determination of virus titer on Al            blot data that the desired UL28 deletion mutants had been <br /> cells. All three viruses replicated in Al and Cl cells, and the       generated. <br /> time course of replication was the same for each virus (Fig.             Replication, cleavage, and packaging of HSV-1 DNA. Pre- <br /> 2). Only KOS replicated on Vero cells, indicating that the            vious studies with a ts mutant (ts1203) whose mutation maps <br /> UL28 deletion mutants fail to replicate (produce infectious           to the UL28 gene showed that the UL28 gene was needed for <br /> virus particles) in cells that do not express the UL28 protein.       cleavage and encapsidation of viral DNA (2). At the nonper- <br /> The parent virus A2 showed similar growth to that of KOS              missive temperature, ts1203 was found to synthesize con- <br /> on all three cell lines (data not shown).                             catemeric viral DNA, which was not processed into <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> <br />  VOL. 67, 1993                                                CHARACTERIZATION OF HSV-1 UL28 DELETION MUTANTS                            3475 <br />  <br />                                                                                    A                          B <br />                                                                                   I: .S7 <br />                                                                                                                   &apos;   : <br />  <br />  <br />            MOCK Cl <br />                         Vero <br />            KOS             c <br />  <br />  <br />  <br />  <br />                         Vero                                                           0 <br />                                                                                        ....... <br />                                                                                         -il&apos;&quot; 4 <br />                                                                                                   .0          .,iaa                 K <br />  <br />                                                                                                                           v* <br />            gCB             ci4                                                                                                      Q <br />                                                                                                                                     S <br />  <br />  <br />  <br />                         Vero <br />            gCA7B                    **-* <br />   FIG. 4. Analysis of viral DNA synthesis. Vero cells or Cl cells <br /> were  infected with the indicated virus at 37&#194;&#176;C for 24 h. A series of       FIG. 5. Processing of virus DNA. (Top) Autoradiograph of <br /> fivefold dilutions were spotted on a nylon membrane, and the cells       Southern blot analysis of BamHI-digested total-infected-cell DNA <br /> were lysed on the membrane. The 32P-labeled Sall fragment de-            and encapsidated (DNase-resistant) DNA hybridized to the 32p_ <br /> scribed in the legend to Fig. 2 was used as a hybridization probe, and   labeled BamHI K fragment of HSV-1 DNA. (A) DNA isolated from <br /> the blots were visualized by autoradiography.                            Vero cells mock infected (lane 1) or infected with KOS (lanes 2 and <br />                                                                          5), gCB (lanes 3 and 6), or gCA7B (lanes 4 and 7). Lanes: 1 to 4, <br />                                                                          total-virus-infected-cell DNA; 5 to 7; encapsidated DNA. (B) DNA <br /> <span id='am-5' about='xsp:length' typeof='owl:Thing'>length</span> molecules. The ability of gCB and gCA7B to induce                 isolated from Cl cells infected with KOS (lanes 1 and 4), gCB (lanes <br />                                                                          2 and 5), or gCA7B (lanes 3 and 6). Lanes: 1 to 3, total-virus- <br /> viral DNA synthesis was assessed by dot blot hybridization               infected-cell DNA, 4 to 6, encapsidated DNA. (Bottom) HSV-1 <br /> of total DNA from infected cells to a virus-specific probe               genome, showing the locations of BamHI-K, -Q, and -S fragments. <br /> (see Materials and Methods). The results shown in Fig. 4 <br /> clearly demonstrate that gCB and gCA7B synthesize viral <br /> DNA in both Cl and Vero cells. These results indicate that <br /> the UL28 gene is not required for HSV DNA synthesis. To                    Transmission electron microscopy of virus infected cells. <br /> determine whether the newly synthesized viral DNA was                    Thin sections of virus-infected Vero and Cl cells were <br /> cleaved and encapsidated, viral DNA processing was inves-                examined by transmission electron microscopy to determine <br /> tigated by using Southern blot analysis. During viral DNA                the presence of full, intermediate, and empty capsids. The <br /> replication, concatemeric DNA is cleaved into <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span>-length                nuclei of Vero cells infected with either gCB or gCA7B <br /> molecules and packaged into virions. The encapsidated viral              contained capsids which had electron-translucent cores (in- <br /> DNA contains free chromosomal termini, whereas the non-                  termediate capsids) (Fig. 6C and E). No dense-cored (full <br /> packaged DNA does not. The presence of chromosomal                       capsids) or empty capsids were observed in mutant-infected <br /> ends can easily be monitored by Southern blot analysis of                Vero cells. By contrast, the predominant capsid form found <br /> total-infected-cell DNA digested with BamHI and probed                   in mutant-infected Cl cells or KOS-infected Vero and Cl <br /> with the HSV-1 BamHI K fragment. Only encapsidated viral                 cells were capsids with dense cores (Fig. 6A, B, D, and F). <br /> DNA with free chromosomal ends will give rise to the                     These results corroborate the Southern blot data described <br /> terminal BamHI Q and S fragments, whereas concatemeric                   above, showing that the UL28 gene product is required for <br /> DNA gives rise only to the joint-spanning BamHI K frag-                  assembly of full capsids. <br /> ment (Fig. 5). Terminal fragments Q and S were absent from                  Large numbers of virus particles were found in both the <br /> viral DNA isolated from Vero cells infected with the two                 nucleus and cytoplasm of Vero and Cl separately cells <br /> UL28 mutants (Fig. 5A, lanes 3 and 4). When the infections               infected with all three viruses. The intermediate capsids <br /> were done with Cl cells, the terminal Q and S fragments                  present in the cytoplasm of Vero cells infected with the two <br /> were readily detected (Fig. SB, lanes 2 and 3), indicating that          mutants appeared to be enveloped, although the envelope <br /> the UL28 gene was required for efficient cleavage of con-                was distinctly different from the envelopes found in wild- <br /> catemeric DNA to <span id='am-241' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span>-length molecules. To examine the                   type-virus-infected Vero or Cl cells or the envelopes found <br /> packaging of viral DNA, we treated infected-cell lysates with            in Cl cells infected with either of the two mutants. The <br /> DNase I and then analyzed them by Southern blot analysis                 particles found in the cytoplasm of Vero cells infected with <br /> following BamHI digestion (see Materials and Methods).                   gCB and gCA7B appeared to contain a very defined envelope <br /> Protection of encapsidated DNA from DNase I digestion is                 around the capsid, whereas wild-type virus and the mutants <br /> evidenced by the presence of the BamHI K, Q, and S                       in Cl cells contained capsids that had a surrounding mem- <br /> fragments. Vero cells infected with gCB or gCA7B do not                  brane but also contained an electron-opaque amorphous <br /> contain DNase-resistant DNA (Fig. 5A, lanes 6 and 7),                    region beneath the envelope. The significance of the quali- <br /> whereas the identical infections in Cl cells contain DNase-              tative difference in the envelope found with virions made by <br /> resistant DNA (Fig. 5B, lanes 5 and 6). These results clearly            the two mutants in Vero cells is unclear. However, the two <br /> demonstrate that viral DNA synthesized in Vero cells in-                 UL28 <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span>null mutants</span> differ in this regard from the UL28 ts <br /> fected with the two UL28 deletion mutants is endless and is              mutants, since cytoplasmic, enveloped particles were not <br /> not packaged into capsids.                                               found at the nonpermissive temperature with the ts mutants. <br />  <br />  <br />                 A.                                                         proteins of approximately 50 and 80 kDa were detected with <br />                            M 1   2   3   4    6   8 12 16 22               the H85 antiserum at early times postinfection. The 50-kDa <br />                  205                                                       protein was a cell-expressed protein, since it was detected in <br />                                                                            mock-infected cells. The significance of the 80-kDa protein is <br />                                                                            not known. <br />                      116                                                     To characterize expression of the UL28 protein from the <br />                                                                            two mutant viruses, we isolated extracts of infected Vero <br />                  80 <br />                                                                            and Cl cells at 12 and 22 h postinfection. Neither gCB or <br />                                                                            gCA7B expressed an immunoreactive protein in Vero cells <br />                            I~~~~~~~~~~~I <br />                                                                            (Fig. 7B). This is expected for the gCB mutant, since the <br />                  49.5                                                      deletion removes nearly all of the UL28 coding sequences. <br />                                                                            gCA7B is predicted to encode a UL28 peptide of at least 60 <br />                                                                            kDa; however, a band of this size was not observed (Fig. <br />                                                                            7B), indicating that either this protein is not recognized by <br />                                                                            the H85 antiserum or the truncated protein may not be <br />                 B. <br />                                Kos            GCB         A7b              stable. In gCB- and gCA7B-infected Cl cells, a protein <br />                            M 1 2 3           1 2 3      1 2 3              identical in size to the UL28 protein expressed from KOS- <br />                  205                                                       infected Vero or Cl cells was detected (Fig. 7B). This <br />                                                                            protein was not present in mock-infected Cl cells (Fig. 7B, <br />                                                                            lane M). These results demonstrate that although the two <br />                  116                                                       mutants do not make a UL28 protein in infected Vero cells, <br />                                                                            they are capable of inducing expression of the UL28 gene <br />                                                                            resident in Cl cells. Therefore, this experiment confirms that <br />                  80 <br />                                                                            the ability of Cl cells to support growth of the two UL28 null <br />                                                                            mutants is due to its ability to provide the missing UL28 <br />                                                      1. W .   t.           protein. <br />                  49.5                                                         Is the UL28 gene required for expression of viral glycopro- <br />                                                                            teins on the surface of infected cells? Studies with a ts mutant, <br />                                                                            tsZ47, indicated that the UL28 gene may be required for <br />                                                                            efficient transport of viral glycoproteins in infected cells (31). <br />    FIG. 7. Detection of the UL28 gene product during viral infec-          Complementation analysis with HSV-1 and HSV-2 ts mu- <br /> tion by immunoblotting with a chemiluminescence detection system           tants indicated that the ts lesion was within the HSV-1 UL28 <br /> (see Materials and Methods). (A) Time course of expression from            gene, but marker rescue experiments showed that two <br /> KOS-infected Vero cells. Cells were harvested at the indicated             different regions of the HSV-1 genome could rescue the ts <br /> times postinfection (time in hours given above each lane), and cell        phenotype. To address the role of the UL28 protein in <br /> extracts were prepared as described in Materials and Methods and <br /> subjected to electrophoresis and immunoblotting. Lane M, mock-             expression of viral glycoproteins, we infected Vero cells <br /> infected Vero cells. (B) Expression of the UL28 protein from               with either KOS or gCB and added [35S]methionine to the <br /> mutant-infected cells. Vero cells (lanes 1 and 2) or Cl cells (lanes 3)    cells at 5 h postinfection. At 24 h postinfection, the cells <br /> infected with the indicated virus were harvested at 12 h (lanes 1) or      were harvested and lysed and expression of gB and gD were <br /> 22 h (lanes 2 and 3) postinfection and treated as described above.         analyzed by immunoprecipitation with gB- and gD-specific <br /> Filters were probed with polyclonal antisera raised against an E.          monoclonal antibodies (Fig. 8). No difference in the relative <br /> coli-expressed Cro-UL28 fusion protein. Lane M, mock-infected Cl           amount of gB or gD expressed from gCB as compared with <br /> cells. The position of the UL28 protein is indicated with an arrow in      KOS was found. In addition, the sizes of the two viral <br /> both panels.                                                               glycoproteins were identical to that found in KOS-infected <br />                                                                            cells, indicating that mature glycoproteins were being ex- <br />                                                                            pressed in gCB-infected Vero cells. To determine whether <br />    Detection of UL28 protein in KOS-, gCB-, and gCA7B-                     the glycoproteins were being expressed on the surface of <br /> infected Vero and Cl cells. To identify the protein product of             infected cells, we performed cell surface immunofluores- <br /> the HSV-1 UL28 gene, we raised a polyclonal antiserum                      cence (Fig. 9). Vero cells infected with KOS (Fig. 9A and B), <br /> (H85) against an E. coli-expressed Cro-UL28 fusion protein                 gCB (Fig. 9C and D), or gCA7B (Fig. 9E and F) were fixed at <br /> (see Materials and Methods). The H85 antiserum was used                    8 h postinfection and treated with gD-specific (Fig. 9A, C, <br /> for immunoblot analysis of KOS-infected Vero cell extracts.                and E) or gC-specific (Fig. 9B, D, and F) monoclonal <br /> In mock-infected cells and in infected cells harvested at 1, 2,            antibodies, followed by fluorescein isothiocyanate-conju- <br /> 3, and 4 h postinfection, there was no specific reactivity by              gated sheep anti-mouse immunoglobulin G. Fluorescence <br /> the antiserum (Fig. 7A). An immunoreactive band of 87 kDa                  microscopy demonstrated cell surface immunofluorescence <br /> was detected at 6 h postinfection, and the intensity of this               for KOS-, gCB-, and gCA7B-infected Vero cells when the gD <br /> band increased at later times of infection. The open reading               antibody was used. The level of fluorescence was similar for <br /> frame of the UL28 gene predicts a protein of approximately                 the three viruses, suggesting that the amount of gD ex- <br /> 85 kDa, which is in good agreement with the size of the                    pressed on the surface of cells infected with the two UL28 <br /> protein detected with the H85 antiserum. Two additional                    mutants is comparable to that in a wild-type infection. The <br />  <br />  <br />  <br />   FIG. 6. Transmission electron micrographs of thin-section preparations of virus-infected cells. Vero cells infected with KOS (A), gCB (C), <br /> or gCA7B (E), Cl cells infected with KOS (B), gCB (D), or gCA7B (F). The arrowheads point to virus particles with or without an <br /> envelope. Abbreviations: N, nucleus; C, cytoplasm; E, extracellular. <br />  <br />   FIG. 8. Immunoprecipitation of gB and gD from Vero cells <br /> infected with KOS or gCB. Vero cells infected with the indicated <br /> virus were labeled with [35S]methionine from 5 to 24 h postinfection, <br /> and gB and gD were immunoprecipitated with a mixture of gB- and <br /> gD-specific antibodies. Immunoprecipitated material was resolved <br /> by SDS-PAGE on 10% polyacrylamide gels and visualized by <br /> autoradiography. <br />  <br />  <br /> parent virus used to construct the gCB and gCA7B mutants                  FIG. 9. Expression of viral glycoproteins on the surface of <br /> contains a deletion in the gC promoter, and so neither UL28             infected cells. Vero cells infected with KOS (A and B), gCB (C and <br />                                                                         D) or gCA7B (E and F) were fixed at 8 h postinfection with 3% <br /> mutant expresses gC. When infected cells were treated with              paraformaldehyde and treated with gD-specific (panels A, C, and E) <br /> the gC antibody, cell surface immunofluorescence was ob-                or gC-specific (panels B, D, and F) monoclonal antibodies followed <br /> served only with wild type-infected cells. These data show              by fluorescein isothiocyanate-conjugated sheep anti-mouse immu- <br /> that the UL28 gene product is not required for efficient                noglobulin G. Cell surface fluorescence was observed by fluores- <br /> expression of viral glycoproteins on the surface of infected            cence microscopy. The photographs in panels D and F were <br /> cells.                                                                  overexposed relative to the other panels to demonstrate the pres- <br />                                                                         ence of cells. <br />                            DISCUSSION <br />    Studies with HSV-1 ts mutants have demonstrated that                 the UL28 gene product has an apparent molecular weight of <br /> cleavage of concatemeric HSV-1 DNA to <span id='am-242' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span>-length mole-                 87,000, which is close to the molecular weight of 85,573 for <br /> cules does not occur in the absence of DNA packaging (1-3,              the UL28 protein predicted by DNA sequence analysis (27), <br /> 39, 45, 46). In addition to the seven capsid proteins, at least         and that this protein is expressed at late times during <br /> five other viral gene (UL6, UL25, UL28, UL32, and UL33)                 infection. Electron microscopy and Southern blot analysis <br /> products are required for the cleavage and packaging event              demonstrated that the UL28 null mutants are defective in <br /> (1-4, 46). Understanding the role of these genes in cleavage            cleavage and packaging of viral DNA. <br /> and packaging of viral DNA is important to our understand-                 A number of HSV ts mutants thought to have lesions in <br /> ing of the mechanism of virus maturation. In this study we              the UL28 gene have been isolated. The mutation in ts1203 <br /> isolated Vero cell lines transformed with sequences that                has been mapped, by marker rescue, to the coding region of <br /> express the UL28 gene and used these cell lines to isolate              the UL28 gene. At the nonpermissive temperature, the <br /> two UL28 null mutants. Both of the UL28 null mutants                    phenotype of ts1203 is similar to that of the two UL28 null <br /> isolated fail to form plaques on or replicate in Vero cells,            mutants described in this study, since this mutant produces <br /> confirming that this is an essential HSV-1 gene. Although we            only B capsids and fails to cleave high-molecular-weight <br /> did not marker rescue the UL28 mutations, the fact that two             viral DNA into unit-length molecules (2). In contrast to the <br /> different UL28 deletion mutants grow only on cell lines that            UL28-null mutants, though, no cytoplasmic enveloped par- <br /> express the UL28 gene strongly suggests that the pheno-                 ticles were detected in ts1203-infected cells at the nonper- <br /> types of gCB and gCA7B are due to the introduced mutations              missive temperature. The ts1203 mutant produces large <br /> in the UL28 gene. The absence of expression of the UL28                 numbers of intermediate capsids that are found only in the <br /> gene product in Vero cells infected with the two mutants was            nucleus of infected cells, whereas Vero cells infected with <br /> demonstrated by immunoblot analysis with antiserum spe-                 either of the two UL28 null mutants contain nearly equal <br /> cific for this protein. The antiserum was also used in immu-            numbers of intermediate capsids in both the nucleus and the <br /> noblot experiments to show that in KOS-infected Vero cells,             cytoplasm. Some of the particles found in the cytoplasm of <br />  <br />  <br /> Vero cells infected with the UL28 null mutants contain a             <br /> membrane envelope which, when examined by transmission                  <br /> electron microscopy, appears different from the envelope                <br /> found in Cl cells infected with wild-type virus or with either    <br /> of the two mutants. The membrane is very defined and                  <br /> appears to be missing the electron-opaque amorphous region            <br /> beneath the envelope that is seen with wild-type virus                  <br /> particles (Fig. 6). This region may represent the tegument            <br /> portion of the virus particle. To examine whether this              <br /> portion of the virus particle is missing from the two null         <br /> mutants, we plan to isolate enveloped virions from Vero                  <br /> cells infected with the two UL28 null mutants and examine                <br /> their protein make-up by SDS-polyacrylamide gel electro-    <br /> phoresis (PAGE). There are several proteins associated with                <br /> the tegument region, such as VP16, that are easily identified          <br /> by SDS-PAGE, and these proteins can be used as markers to              <br /> determine whether the tegument is present in UL28 mutant                 <br /> particles.                                                                <br />    Studies with a second ts mutant, tsZ47, suggest that the            <br /> UL28 gene product is needed for the efficient transport of             <br /> viral glycoproteins to the cell membrane (31). These results       <br /> are complicated by the fact that marker rescue studies have          <br /> shown that tsZ47 contains two ts mutations (31), one that               <br /> maps to the left end of the HSV-1 genome (map units 0.058          <br /> to 0.074) and one that maps to a region near the UL28 gene          <br /> (map units 0.312 to 0.415). Furthermore, Addison et al. (2)          <br /> have shown that in complementation tests ts1203 efficiently            <br /> complements tsZ47, indicating that tsZ47 probably does not            <br /> contain a lesion in UL28. To study the role of the UL28 gene        <br /> product in glycoprotein transport, we examined the expres-          <br /> sion of gD on the surface of Vero cells infected with a UL28         <br /> null mutant. Immunofluorescence clearly demonstrated that            <br /> a UL28 null mutation does not alter the expression of the gD        <br /> gene on the surface of infected cells. We conclude from                  <br /> these studies that the UL28 gene product is not involved in        <br /> the transport of viral glycoproteins to cell membranes.                <br /> Studies with a pseudorabies virus mutant that contains a             <br /> deletion in the UL28 gene homolog corroborate these results           <br /> (30).                                                                <br />    In summary, we have described a system for studying the      <br /> role of the UL28 gene product in HSV-1 morphogenesis.                <br /> UL28-transformed Vero cell lines which allow for the isola-          <br /> tion and preparation of mutant virus stocks have been           <br /> established. The availability of the complementing cell lines          <br /> will allow for the isolation of additional UL28 mutants                  <br /> containing both deletion and nonsense mutations. Charac-   <br /> terization of these mutants should ultimately lead to an            <br /> understanding of the role of the UL28 protein in the cleavage        <br /> and packaging of viral DNA. The fact that the UL28 gene is     <br /> conserved among almost all herpesviruses indicates that this              <br /> protein has a common function in all these viruses. Under-                <br /> standing how this protein functions during HSV-1 infections               <br /> may therefore tell us how cleavage/packaging occurs in other     <br /> herpesviruses.                                     <br /> </body></html>