<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                            NIH Public Access <br />                             Author Manuscript <br />                             Dev Biol. Author manuscript; available PMC 2008 August 26. <br />                            Published final edited form as: <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             Dev Biol. 2004 December 15; 276(2): 391&#226;&#8364;&#8220;402. doi:10.1016/j.ydbio.2004.09.001. <br />  <br />  <br />  <br />                            Otopetrin 1 required otolith formation zebrafish Danio <br />                            rerio <br />  <br />                            Inna Hughesa,1, Brian Blasioleb,1, David Hussc, Mark E. Warchold, Nigam P. Rathe, Belen <br />                            Hurlef, Elena Ignatovad, J. David Dickmanc, Ruediger Thalmannd, Robert Levensonb, and <br />                            David M. Ornitza,* <br />                            aDepartment Molecular Biology Pharmacology, Washington University Medical School, St. Louis, <br />                            MO 63110, United States <br />                            bDepartment Pharmacology, Pennsylvania State University College Medicine, Hershey, PA 17033, <br />                            United States <br />                            cDepartment Anatomy Neurobiology, Washington University Medical School, St. Louis, MO 63110, <br />                            United States <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            dDepartment Otolaryngology, Washington University Medical School, St. Louis, MO 63110, United States <br />                            eDepartment Chemistry Biochemistry, University Missouri, St. Louis, MO 63110, United States <br />                            fNational Institutes Health, National Human Genome Research Institute, Bethesda, MD 20892&#226;&#710;&#8217;2152, <br />                            United States <br />  <br />                            <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />                                 Orientation respect gravity essential <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span>survival</span> complex organisms. The gravity <br />                                 receptor phylogenetically oldest sensory systems, special adaptations enhance <br />                                 sensitivity gravity highly conserved. The fish inner ear contains large extracellular <br />                                 biomineral particles, otoliths, evolved transduce force gravity neuronal <br />                                 signals. Mammalian ears contain thousands small particles called otoconia serve similar <br />                                 function. Loss displacement structures lethal fish responsible benign <br />                                 paroxysmal positional vertigo (BPPV) humans. The distinct morphologies otoconial particles <br />                                 otoliths suggest divergent developmental mechanisms. Mutations novel gene Otopetrin 1 <br />                                 (Otop1), encoding multi-transmembrane domain protein, result nonsyndromic otoconial agenesis <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                 severe balance disorder mice. Here zebrafish, Danio rerio, contains highly <br />                                 conserved gene, otop1, essential otolith formation. Morpholino-mediated knockdown of <br />                                 zebrafish Otop1 leads otolith agenesis affecting sensory epithelium structures <br />                                 inner ear. Despite lack otoliths early development, otolith formation partially recovers <br />                                 fish 2 days. However, otoliths malformed, misplaced, lack organic matrix, <br />                                 consist inorganic calcite crystals. These studies demonstrate Otop1 essential <br />                                 conserved role timing formation size shape developing otolith. <br />  <br />  <br />                            Keywords <br />                                 Otolith; Otoconia; Otopetrin 1 (Otop1); Biomineralization; Vestibular systems <br />  <br />  <br />  <br />  <br />                            * Corresponding author. Department Molecular Biology Pharmacology, Washington University School Medicine, 660 South <br />                            Euclid Avenue, PO Box 8103, St. Louis, MO 63110. Fax: +1 314 362 7058. E-mail address: dornitz@molecool.wustl.edu (D.M. Ornitz).. <br />                            1These authors contributed equally work. <br />                             Hughes et al.                                                                                              Page 2 <br />  <br />  <br />                               Introduction <br />                                            Otoconia small (approximately 10 &#206;&#188;m) extracellular biomineral particles the <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            vestibular portion vertebrate inner ear. Otoconia composed specific polymorphs <br />                                            calcium carbonate (CaCO3) crystals precipitated organic core extracellular <br />                                            matrix proteins. These particles required normal sensation linear acceleration and <br />                                            gravity mammals (Bergstrom et al., 1998; Lim, 1980; Ornitz et al., 1998). In teleost fish, <br />                                            complete loss orthologous structure, otolith, lethal (Riley Moorman, 2000). <br />                                            In contrast thousands small otoconial particles mammals, large otoliths <br />                                            form fish. Few molecules governing development otoconia otoliths been <br />                                            described specific structure. It proposed that <br />                                            polymorph CaCO3 otoconia otolith determined major matrix <br />                                            proteins make organic core: Otoconin 90 core calcitic CaCO3 otoconia <br />                                            birds mammals; Otoconin 22 primary matrix component aragonitic CaCO3 <br />                                            otoconia amphibians; Otoconin 54 primary matrix constituent vateritic <br />                                            CaCO3 otoconia utilized early jawed fish, garfish (Pote Ross, 1991); and <br />                                            otolith matrix protein (omp) primary matrix protein aragonitic fish otolith <br />                                            (Murayama et al., 2000). These major matrix proteins share ability bind calcium other <br />                                            ions otoconins share rigid phospholipase A2 structural domains may <br />                                            mediate ability guide formation specific CaCO3 crystal polymorphs (Pote et al., <br />                                            1993; Wang et al., 1998). Starmaker, ortholog mammalian dentin sialoprotein (DSP), <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            acidic phosphoprotein recently shown required normal otolith formation the <br />                                            zebrafish (S&#195;&#182;llner et al., 2003). While expression DSP observed mouse inner <br />                                            ear, DSP <span id='am-241' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-242' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-246' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-253' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-256' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-278' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-284' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-290' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-292' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-297' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-300' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-302' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-308' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-315' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-317' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-318' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span>knockout</span> mice appear significant vestibular dysfunction based swim <br />                                            testing (T. Sreenath, personal communication), suggesting function starmaker may <br />                                            specific fish otolith. <br />  <br />                                            While important differences proteins pathways required generate <br />                                            small otoconia particles versus large otolith, similarities exist. The essential <br />                                            requirement formation otoliths otoconia availability Ca2+ and <br />                                                  ions. The presence carbonate ions depends activity carbonic anhydrase. The <br />                                            source calcium endolymph contributes otoconia otolith poorly <br />                                            understood. Organic substances, including acidic proteins, glycosaminoglycans (GAGs) and <br />                                            proteoglycans, essential regulate crystal growth (Addadi et al., 1989; Khan, 1997) <br />                                            identified otoliths (Borelli et al., 2003) otoconia (Tachibana and <br />                                            Morioka, 1992). While believed proteins extracellular matrix molecules are <br />                                            required locally increasing Ca2+        concentrations structural components of <br />                                            developing otolith otoconia, protein known enzymatic function required <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            otoconial formation NADPH Oxidase 3 (NOX3). NoX3 mutated head-tilt mice that <br />                                            nonsyndromic otoconial agenesis required aggregation Otoconin <br />                                            90 proteins mouse ear (Paffenholz et al., 2004). <br />  <br />                                            Orchestration extracellular calcification requires bringing ionic proteinaceous <br />                                            components time space. The organic matrix components otoconia expressed in <br />                                            different regions vestibular epithelium; however, matrix components associate <br />                                            extracellular gelatinous structure called otolithic membrane order localize <br />                                            otoconial development sensory epithelium. Otoconial matrix proteins aggregate <br />                                            ordered organic cores Ca2+        concentrations locally increased allow <br />                                            crystallization. Coordination events requires normal formation otocyst <br />                                            (Malicki et al., 1996) sensory maculae (Haddon et al., 1998), tight regulation of <br />                                            endolymph ionic environment (Everett et al., 1997; Kozel et al., 1998). This process is <br />                                            temporally restricted, expression certain major matrix proteins dramatically down- <br />  <br />  <br />                                                  Dev Biol. Author manuscript; available PMC 2008 August 26. <br />                             Hughes et al.                                                                                              Page 3 <br />  <br />  <br />                                            regulated early development (B. Blasiole E. Ignatova, unpublished data). In <br />                                            mammals, process otoconial development essentially complete postnatal day 7 <br />                                            (Erway et al., 1986; Lim, 1973; Veenhof, 1969), little evidence available continued <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            otoconial formation repair. In fish, initial rapid growth otolith occurs early otic <br />                                            development, otolith continues grow life fish, increments <br />                                            organic matrix calcium carbonate added daily otolith surface. Disruption any <br />                                            processes lead formation abnormally shaped ectopic otoconia or <br />                                            otoconial agenesis. <br />  <br />                                            Ectopic otoconia humans proposed cause human vestibular dysfunction, in <br />                                            particular benign paroxysmal positional vertigo (BPPV). BPPV common cause dizziness <br />                                            associated dislodged otoconia entering semicircular canals causing <br />                                            abnormal vestibular sensation response head rotation. It estimated the <br />                                            elderly population 50% dizziness attributed BPPV (Oghalai et al., <br />                                            2000). Otoconial pathology significant etiology balance-related falls accidental <br />                                            deaths elderly. While cases BPPV associated head trauma, <br />                                            vestibular neuritis, treatment pharmacological agents, age-related degeneration <br />                                            otoconia, etiology approximately half cases BPPV young elderly <br />                                            patients unknown. <br />  <br />                                            Mutations novel protein, Otopetrin 1 (Otop1), cause nonsyndromic otoconial agenesis and <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            severe balance disorder tilted mergulhador mice (Hurle et al., 2003). Mutant mice <br />                                            display near percent penetrance otoconial agenesis <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span>phenotype</span>, no <br />                                            developmental changes inner ear morphogenesis. Otop1 predicted encode multi- <br />                                            transmembrane domain protein unknown function known homology family <br />                                            receptors, transporters, channels. Two independent single-<span id='am-319' about='xsp:base' typeof='owl:Thing'>base</span> pair mutations been <br />                                            identified mutant mice; mutations different regions molecule create <br />                                            identical phenotypes, suggesting normal function Otop1 necessary the <br />                                            development otoconia mouse. Homologous genes Otop1 identified in <br />                                            vertebrate groups examined including zebrafish Fugu, 41% 44% <br />                                            identical, respectively, mouse Otop1 (Hurle et al., 2003), suggesting share a <br />                                            common mechanism action otolith otoconial morphogenesis. <br />  <br />                                            Here, Otop1 conserved essential role teleost otolith development. <br />                                            Zebrafish otop1 mouse Otop1 similar expression patterns developing inner ear. <br />                                            Morpholine oligonucleotide (morpholino)-mediated knockdown Otop1 expression resulted <br />                                            otolith agenesis majority treated fish, affecting morphogenesis the <br />                                            zebrafish inner ear. In small percentage morphant animals, otolith development greatly <br />                                            delayed began 40&#226;&#710;&#8217;50 h postfertilization (hpf), variety otolith phenotypes, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            including formation normal otoliths otoliths lacking organic matrix atypical <br />                                            crystal polymorph (calcite). These data support conserved role Otop1 localization <br />                                            aggregation otolith matrix proteins regulation ionic environment the <br />                                            otolithic membrane vestibular organ development. <br />  <br />                               Materials methods <br />                               In situ hybridization <br />                                            The clone fb76b02.y1 containing 3&#226;&#8364;&#178; UTR otop1 identified zebrafish EST project <br />                                            (Washington University) sequenced identify orientation linearized Not1. A <br />                                            digoxigenin-labeled RNA probe generated using Sp6 labeling kit (Roche), following <br />                                            manufacturer&apos;s instructions. <br />  <br />  <br />  <br />  <br />                                                  Dev Biol. Author manuscript; available PMC 2008 August 26. <br />                             Hughes et al.                                                                                             Page 4 <br />  <br />  <br />                                            Timed matings zebrafish established embryos isolated specific time points based <br />                                            time fertilization developmental milestones (zfin.org). Embryos were <br />                                            dechorionated fixed overnight 4% paraformaldehyde phosphate-buffered saline <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            (PBS) 48C. Embryos hybridized overnight digoxigenin-labeled probe. <br />                                            Hybridization detected anti-digoxigenin&#226;&#8364;&#8220;alkaline phosphatase-conjugated IgG and <br />                                            visualized BM purple AP substrate (Roche) presence 2% levamisole. <br />  <br />                                            For sectioning situ hybridized embryos, overstained 3 dpf embryos embedded in <br />                                            JB-4 plastic (Chan et al., 2001) sectioned 4 &#206;&#188;m. Similarly prepared embryos were <br />                                            sectioned stained Richardson&apos;s stain comparison histologic structures. <br />  <br />                               Morpholine oligonucleotide-mediated knockdown zebrafish otop1 <br />                                            Two independent morpholine oligonucleotides designed base pair 5&#226;&#8364;&#178; UTR of <br />                                            otop1. MO-1 covers region &#226;&#710;&#8217;91 &#226;&#710;&#8217;65 bp ATG start codon <br />                                            (TTACACCTTCAGGACCCGTTAGTTT) MO-2 begins &#226;&#710;&#8217;20 bp spans ATG, <br />                                            ending +5 nucleotide position (ACCATGCTCGATCGCTGTCGGTAAA). Both <br />                                            morpholinos purchased Gene Tools, LLC (Philomath, Oregon) 5&#226;&#8364;&#178; labeled <br />                                            FITC. Before injection, morpholino oligonucleotides diluted Phenol Red tracer <br />                                            1&#195;&#8212; Danieau&apos;s buffer (58 mM NaCl, 0.7 mM KCl, 0.4 mM MgSO4, 0.6 mM Ca(NO3)2, 5 <br />                                            mM HEPES, pH 7.6). Timed matings established. Embryos collected 1-and <br />                                            2-cell stage injected morpholino concentrations 0.25 12 ng. MO-1 used <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            following experiments. Dorsal ventral axis defects pericardial edema were <br />                                            noted occur morpholino-injected fish high doses (8&#226;&#710;&#8217;12 ng morpholino). <br />  <br />                                            Morphant embryos maintained 28&#194;&#176;C charcoal filtered water examined and <br />                                            counted dissection microscope. <span id='am-6' about='protege:TO' typeof='owl:Thing'>To</span> count photograph otoliths older stages, <br />                                            embryos anesthetized 1% tricaine immobilized 3% methyl cellulose. <br />  <br />                               Histologic preparation <br />                                            The 4 7 dpf wild-type morphant fish collected fixed overnight cold 2.5% <br />                                            glutaraldehyde 0.1 M sodium cacodylate buffer, pH 7.5. Animals washed cold 0.1 <br />                                            M cacodylate buffer dehydrated graded series 25&#226;&#710;&#8217;100% ethanol. Samples were <br />                                            maintained &#226;&#710;&#8217;20&#194;&#176;C processed. To prepare samples sections, <br />                                            embryos cleared 100% propylene oxide incubated increasing ratio of <br />                                            propylene oxide unaccelerated Durcupan room temperature. Fish incubated <br />                                            100% unaccelerated Durcupan overnight transferred accelerated Durcupan, <br />                                            oriented hardened 60&#194;&#176;C 48&#226;&#710;&#8217;64 h. Sections cut glass Ralph knife a <br />                                            rotary microtome thickness 4 6 &#206;&#188;m. Sections stained Richardson stain and <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            coverslipped. <br />  <br />                                            For immunohistochemical labeling, 28 hpf morpholino-treated control fish fixed <br />                                            overnight 4% paraformaldehyde. Following thorough rinsing PBS, specimens were <br />                                            cryoprotected 30% sucrose (in PBS), embedded OCT compound, frozen, sectioned <br />                                            10 &#206;&#188;m. Sections mounted gelatin/CrK(SO4)2&#226;&#710;&#8217;-coated slides, treated 2 h in <br />                                            blocking solution (2%NHS, 2% NGS, 1% BSA, PBS), incubated overnight primary <br />                                            antibody acetylated tubulin (Sigma T6793, 1:100). The day, sections were <br />                                            incubated 2 h Cy3-conjugated anti-mouse secondary antibody cell nuclei were <br />                                            labeled bisbenzimide (Sigma). <br />  <br />  <br />  <br />  <br />                                                  Dev Biol. Author manuscript; available PMC 2008 August 26. <br />                             Hughes et al.                                                                                              Page 5 <br />  <br />  <br />                               Scanning electron microscopy <br />                                            Fixed tissues dehydrated using series graded acetones. Final dehydration was <br />                                            performed placing specimens tetramethylsilane sublimated dry 60&#194;&#176;C <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            oven. The specimens mounted studs palladium coated. <br />  <br />                               Single crystal X-ray diffraction <br />                                            The 7 dpf wild-type morphant fish clutches anesthetized fixed in <br />                                            100% ethanol. Otoliths removed dissection 100% ethanol maintained dry <br />                                            covered glass slide examination. The otoliths mounted grease random <br />                                            orientation. Preliminary examination data collection performed Bruker <br />                                            SMART 1 K Charge Coupled Device (CCD) Detector single crystal X-ray diffractometer <br />                                            equipped sealed tube X-ray source using graphite monochromated Mo K&#206;&#177; radiation (&#206;&#187; <br />                                            = 0.71073 &#195;&#8230;) &#226;&#710;&#8217;123&#194;&#176;C. Typical preliminary <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> cell constant determination set 45 <br />                                            narrow frame 0.3&#194;&#176; &#207;&#8211; scans failed insufficient harvested reflections. Therefore, data <br />                                            set collected frame width 0.3&#194;&#176; &#207;&#8211; counting time 60 s/frame crystal <br />                                            detector distance 4.835 cm (1301 frames &#226;&#710;&#8217;27&#194;&#176; 2&#206;&#184; 230 frames 27&#194;&#176; 2&#206;&#184;). The double <br />                                            pass method scanning used exclude noise. Thresholding collected frames <br />                                            resulted 122 reflections. Indexing <span id='am-3' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> cell carried CELL_NOW <br />                                            (Sheldrick, 2002) cell refined using SMART software package (Bruker <br />                                            Analytical X-ray). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Results <br />                               Similar expression mouse zebrafish otop1 ear embryonic development <br />                                            The zebrafish homologue Otop1 identified described previously (Hurle et al., <br />                                            2003). Using whole-mount situ hybridization, otop1 mRNA identified otocyst <br />                                            18 h postfertilization (hpf). By 24 hpf, otolith seeding nearly complete, otop1 <br />                                            expression localized developing sensory epithelium ear (Figs. 1A B). <br />                                            This <span id='am-320' about='xsp:pattern' typeof='owl:Thing'>pattern</span> similar Otop1 expression mouse utricle saccule otoconial <br />                                            development (Hurle et al., 2003). At later stages, zebrafish otop1 mRNA restricted the <br />                                            utricular saccular maculae (Fig. 1C) pattern consistent expression precursor <br />                                            mature sensory hair cells (Figs. 1D E). By 5 days postfertilization (dpf), otop1 <br />                                            expression greatly reduced otolith organs identified neuromasts of <br />                                            lateral line (Fig. 1F). Expression otop1 detectable inner ear 7 <br />                                            dpf situ hybridization persisted anterior lateral line (data shown). <br />                                            Reduction loss otop1 expression inner ear suggests otop1 specific role <br />                                            early development rapid growth otolith, required the <br />                                            daily incremental growth continues life fish. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Morpholino-mediated knockdown otop1 <br />                                            To determine essential function Otop1 otoconial/otolith development is <br />                                            conserved fish, expression zebrafish Otop1 protein knocked using antisense <br />                                            morpholine oligonucleotides (morpholinos). Morpholinos targeted 5&#226;&#8364;&#178; UTR translation <br />                                            initiation sequences block translation message (Nasevicius Ekker, 2000). Injection <br />                                            morpholino designed otop1 (MO-1) one-cell stage embryos resulted in <br />                                            complete agenesis otoliths (Fig. 2). Injection second morpholino (MO-2) targeted an <br />                                            independent region otop1 5&#226;&#8364;&#178; UTR reproduced defect, confirming otolith agenesis <br />                                            zebrafish specific loss Otop1 expression. At 30 hpf, 96% otop1 <br />                                            morphant fish failed develop saccular utricular otoliths (Table 1), other <br />                                            obvious developmental defects (Fig. 2). Expression pax2a otx1 24 (Figs. 3A&#226;&#8364;&#8220;D) and <br />  <br />  <br />  <br />                                                  Dev Biol. Author manuscript; available PMC 2008 August 26. <br />                             Hughes et al.                                                                                              Page 6 <br />  <br />  <br />                                            48 hpf (data shown) comparable uninjected otop1 morphant fish, indication <br />                                            early steps otocyst formation progressed normally. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Tether cells function poles otocyst required normal otolith formation <br />                                            (Riley Grunwald, 1996). Development cell type, examined acetylated tubulin <br />                                            immunohistochemistry wild-type injected animals, showed distribution hair cell <br />                                            kinocilia similar pattern uninjected, age-matched controls (Figs. 3E F). These data <br />                                            suggest otop1 knockdown does disrupt normal patterning cell differentiation the <br />                                            developing inner ear, specifically disrupts otolith formation. <br />  <br />                                            Fish abnormal otolith development visible otolith seeding particles the <br />                                            otocyst 24 hpf (Riley Grunwald, 1996; Riley et al., 1997; Sumanas et al., 2003) even <br />                                            actual otolith formation delayed. No otolith matrix material observed otop1 <br />                                            morphant fish 24 hpf differential interference contrast (DIC) microscopy (Fig. 3G). To <br />                                            determine expression genes known required formation otolith matrix <br />                                            disrupted otop1 morphant fish, expression zebrafish otolith matrix protein <br />                                            (omp) (E. Ignatova, unpublished data) starmaker (S&#195;&#182;llner et al., 2003) mRNA were <br />                                            assessed early otic development. The expression patterns omp starmaker were <br />                                            similar wild-type morphant fish 24 hpf (Figs. 3H&#226;&#8364;&#8220;K) 48 hpf (data shown), <br />                                            suggesting loss Otop1 does disrupt normal expression genes encoding otolith <br />                                            matrix proteins. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Normal formation lateral line otop1 morphants <br />                                            Despite expression otop1 developing neuromasts lateral line system, morphant <br />                                            animals showed obvious defects morphogenesis structures. Examination of <br />                                            neuromasts live morphants DIC 3 5 dpf (data shown) scanning electron <br />                                            microscopy 4 dpf morphants (Figs. 4A B) revealed normal formation cupula, the <br />                                            extracellular fibrous matrix covering neuromast transduces movement the <br />                                            underlying hair cells. Plastic sections 7 dpf morphants stained Richardson&apos;s stain <br />                                            showed distribution number anterior, trunk, posterior lateral line neuromasts <br />                                            similar wild-type animals. In addition, otop1 morphant neuromasts appeared to <br />                                            similar complement support hair cells (Figs. 4C D). Neuromast formation <br />                                            migration unlikely directly affected otop1 knockdown, structures <br />                                            develop later otolith. The normal distribution neuromasts 7 dpf morphant fish <br />                                            suggests disruption normal patterning migration neuromast <br />                                            precursors. Additional examination sections showed histological difference <br />                                            morphant wild-type fish structure (data shown). <br />  <br />                               Delayed otolith formation otop1 morphant fish <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            A small percentage morphant fish completely failed develop otoliths 30 hpf <br />                                            exhibited otolith formation 4 dpf. The late formation otoliths fish occurred a <br />                                            function morpholino dose injected (Table 1). Visible otolith particles noted <br />                                            40 50 hpf (data shown). This timing consistent dilution the <br />                                            morpholino growth fish (Nasevicius Ekker, 2000) possible reexpression of <br />                                            Otop1. The process delayed otolith formation appeared similar normal formation of <br />                                            otoliths earlier development multiple small seeding particles agglomerated and <br />                                            attached sensory epithelium (Riley et al., 1997). However, cases, seeding <br />                                            particles did attach sensory maculae lodged developing <br />                                            semicircular canals free floating otic cavity (Figs. 6I J). <br />  <br />                                            By 5 dpf, development zebrafish inner ear essentially complete, formation <br />                                            semicircular canals associated sensory maculae (Whitfield et al., 2002). When <br />  <br />  <br />                                                  Dev Biol. Author manuscript; available PMC 2008 August 26. <br />                             Hughes et al.                                                                                                Page 7 <br />  <br />  <br />                                            compared wild-type fish, morphant fish injected 10 ng MO-1 normal sensory <br />                                            maculae canal formation lacked otoliths (Figs. 5A B). At 7 dpf, morphant <br />                                            epithelium normal distribution hair supporting cells saccular sensory macula, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            normal transitional cells nonsensory epithelium (Figs. 5C D). Scanning <br />                                            electron microscopy used compare size sensory maculae, distribution hair <br />                                            cells, formation otolithic membrane. Formation gravity organ sensory <br />                                            maculae otolithic membrane appeared normal fish injected MO-1. In wild-type <br />                                            animals, otolith removed examine underlying macular epithelium; the <br />                                            instances examined, led tearing fibrous membrane (Fig. 6E). In morphant animals <br />                                            did form otolith, membrane remained intact contact hair cell <br />                                            (Fig. 6F). This arrangement fibrous matrix otolithic membrane fish is <br />                                            presumably simultaneously transduce motion otolith hair cells macula. <br />                                            In mammals birds, hair cells appear directly contact otolithic membrane <br />                                            matrix. <br />  <br />                                            In birds teleost fish, vestibular maculae form later larval stages: lagena <br />                                            accompanying otolithic/otoconial membrane (8&#226;&#710;&#8217;12 dpf zebrafish, otolith <br />                                            formation beginning 9 dpf; Bever Fekete, 2002; Riley Moorman, 2000), the <br />                                            macula neglecta (17&#226;&#710;&#8217;20 dpf), lacks otolith (Whitfield et al., 2002). The formation of <br />                                            lagenar otolith sensory structures lagena macula neglecta occurs late <br />                                            development affected otop1 morpholino injection fertilized eggs (Nasevicius <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Ekker, 2000). <br />  <br />                                            By 7 dpf, variety otolith phenotypes noted otop1 morphant fish late-forming <br />                                            otoliths (Table 1). Some striking examples seen fish exposed relatively <br />                                            low concentrations morpholino. In animals injected 1 ng MO-1, observed <br />                                            phenotypes ranged near normal otoliths ear (Fig. 6B) single large rounded <br />                                            otolith (Fig. 6D), single multiple polyhedral forms (Figs. 6C, E, F). In some <br />                                            examples, crystals resembled mammalian otoconia polyhedral shapes sharp <br />                                            edges. <br />  <br />                                            Interestingly, fish displayed mixed phenotypes. For example, otocyst, the <br />                                            morphant fish formed normally shaped saccular otolith (Fig. 6G) small uncalcified <br />                                            utricular otolith stained strongly Richardson&apos;s stain (data shown) (Richardson et <br />                                            al., 1960). The organic matrix normally shaped otolith stained lightly Richardson&apos;s <br />                                            stain, highlighting daily growth otolith alternating deposition organic matrix <br />                                            inorganic CaCO3. This demonstrates normal appearing otolith develop a <br />                                            critical window development proposed extend 18 24 hpf (Riley et al., 1997). In <br />                                            opposite ear, utricular (Fig. 6H) saccular otoliths (data shown) roughly <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            cuboidal. In otoliths, organic matrix identified, multiple histological <br />                                            sections suggested single inorganic crystal. The structure large, <br />                                            polyhedral otolith closely resembles bgiantQ calcitic otoconia described mouse <br />                                            mutants defects otoconial synthesis (Erway Grider, 1984;Lim et al., 1978;Ornitz <br />                                            et al., 1998). Such change morphology morphant otolith indicated possible change <br />                                            mechanisms mineralization (see below). <br />  <br />                                            In rare cases, animals injected higher doses morpholino recovered otolith formation. <br />                                            Ectopic mineralization noted animals approximately 72 hpf. In these <br />                                            instances, otolith particles did aggregate identified otic <br />                                            cavity, including developing semicircular canals 4 dpf (Figs. 6I J). <br />  <br />  <br />  <br />  <br />                                                  Dev Biol. Author manuscript; available PMC 2008 August 26. <br />                             Hughes et al.                                                                                                Page 8 <br />  <br />  <br />                               Abnormal otoliths otop1 morphants composed calcite <br />                                            Pure CaCO3 form crystals distinct crystalline polymorphs: calcite, <br />                                            aragonite, vaterite. At 7 dpf, wild-type otoliths (Fig. 7A) composed thousands of <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            aragonitic CaCO3 crystallites arranged multiple orientations surface growing <br />                                            otolith. In contrast, otoconia contain organic core crystalline casing composed of <br />                                            calcite, stable polymorph CaCO3 (Carlstrom et al., 1953). The crystalline <br />                                            appearance otoliths formed morphant fish 7 dpf (Fig. 7B) suggested possible <br />                                            change crystal polymorph aragonite calcite. Single crystal X-ray diffraction wild- <br />                                            type otoliths yielded crystalline dust diffraction pattern (Fig. 7C) consistent the <br />                                            disordered arrangement aragonitic crystallites previously identified powder <br />                                            X-ray diffraction (S&#195;&#182;llner et al., 2003). Notably, set <span id='am-321' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> cell parameters consistent with <br />                                            published values aragonite (http://ruby.colorado.edu/smyth/min/aragonite.html) was <br />                                            obtained indexing harvested reflections. Morphant otoliths, shape similar wild- <br />                                            type otoliths (Fig. 6G), gave similar diffraction pattern (data shown). The polyhedral <br />                                            otoliths evident morphants appeared similar shape mammalian calcitic otoconia <br />                                            scanning electron microscopy (SEM) (Fig. 7B). Single crystal X-ray diffraction analysis of <br />                                            type otolith yielded single crystal diffraction pattern identifiable <span id='am-322' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> cell at <br />                                            &#226;&#710;&#8217;123&#194;&#176;C following: = 4.992 (6), b = 4.992 (1), c = 17.012(2) &#195;&#8230;, &#206;&#177; = 90.00(1), &#206;&#178; = 90.01 <br />                                            (1), &#206;&#179; = 120.01(1), V = 366.8 (1) (Fig. 7D). These parameters match published values calcite <br />                                            (http://ruby.colorado.edu/smyth/min/calcite.html). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Discussion <br />                                            Otopetrin 1 essential formation otoconia otoliths. Thus, Otop1 function <br />                                            early otolith otoconial developmental pathway, prior specification architecture <br />                                            CaCO3 polymorphs divergent structures. The presence pure calcite crystals in <br />                                            morphant animals initiated otolith formation outside critical period 18&#226;&#710;&#8217;24 hpf <br />                                            proposed Riley et al. (1997) suggests ions required biomineralization and <br />                                            proteins control crystal growth coordinately regulated otop1 fish. These data <br />                                            suggest zebrafish otop1 regulate ionic environment otolith that <br />                                            following dilution inhibitory effects otop1 morpholino 30 96 hpf, <br />                                            crystals form purely inorganic manner. This likely temporally restricted expression <br />                                            proteins form initial seeding particles organic matrix otolith the <br />                                            initial rapid growth phase otolith early development. Interestingly, crystalline <br />                                            patterns observed otop1 morphants similar observed starmaker morphant fish <br />                                            (S&#195;&#182;llner et al., 2003). This suggests disruption variety components otolith <br />                                            developmental pathway trigger default mechanism, leads formation inorganic <br />                                            crystals. Under conditions, formation calcite, stable polymorph CaCO3, <br />                                            favored. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Otop1 described molecule comparable knockdown/mutant phenotype in <br />                                            developing otolith/otoconia fish mice. Otolith development appears exquisitely <br />                                            sensitive concentration otop1 protein, doses low 0.5 ng morpholino were <br />                                            sufficient cause agenesis otolith 78% injected fish (Table 1). This suggests that <br />                                            Otop1 regulates critical step otolith formation protein concentration tightly <br />                                            regulated. For example, Otop1 regulate function localization proteins <br />                                            required otolith development. During mouse otoconial development, Otop1 localized to <br />                                            otolithic membrane (Hurle et al., 2003), extracellular gelatinous superstructure up <br />                                            proteins supports otoconial formation maintenance. Location the <br />                                            extracellular space particularly surprising, Otop1 predicted integral membrane <br />                                            protein. This indicate presence protein membrane bound vesicles called <br />                                            globular substance (Tateda et al., 1998), thought precursors otoconia (Erway <br />  <br />  <br />                                                  Dev Biol. Author manuscript; available PMC 2008 August 26. <br />                             Hughes et al.                                                                                                              Page 9 <br />  <br />  <br />                                            et al., 1986;Preston et al., 1975;Ross, 1979). In location, Otop1 function channel <br />                                            transporter, regulating contents function exocytotic vesicles act a <br />                                            structural protein required attachment nucleation otoconia. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            In mouse mutants Otop1, loss gravity sensation results relatively mild behavioral <br />                                            deficits normal conditions (Hurle et al., 2003; Ornitz et al., 1998). Animals unable <br />                                            swim dropped water able walk rear normally. They exhibit <br />                                            circling head tossing behavior, identified animals types of <br />                                            vestibular defects. This compensatory mechanisms maintain balance, <br />                                            including use visual cues, semicircular canals, proprioceptive system. Zebrafish <br />                                            abnormal otolith formation difficulty orienting gravity unable to <br />                                            swim feed (Mizuno Ijiri, 2003; Riley Grunwald, 1996; Riley Moorman, <br />                                            2000; Riley et al., 1997). While behavioral phenotypes morphant animals not <br />                                            specifically examined, instances noted morphant fish developed <br />                                            apparently normal otoliths unable orient dorsal up, lit above. No <br />                                            circling behavior observed, fish raised relatively shallow water <br />                                            allow morphant fish inflate swim bladders. We propose delay otolith <br />                                            formation animals lead deficits formation neuronal circuitry between <br />                                            otolith organs vestibular nuclei. Morphant fish did develop otoliths primarily <br />                                            rested the well, 7 dpf. Most fish intact startle <br />                                            response indicating normal function lateral line organs. Interestingly, morphants <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            single saccular semicircular canal-located otolith able swim efficiently <br />                                            lit above, tilt turn upside resting. <br />  <br />                                            Human vestibular dysfunction increasing clinical problem (National Institute Deafness <br />                                            Other Communication Disorders, 2002). Degeneration displacement otoconia a <br />                                            significant etiology age-related balance disorders benign paroxysmal positional vertigo <br />                                            (BPPV) (Lim, 1984; Tusa, 2001). In addition, commonly used pharmacological agents, such <br />                                            aminoglycoside antibiotics, lead disruption otoconial structure function <br />                                            (Johnsson et al., 1980; Takumida et al., 1997). The presence ectopic calcified particles in <br />                                            late-developing otoliths morphant fish resembles pathology associated human <br />                                            BPPV (Figs. 6I J). The phenotype provide useful model elucidate mechanism <br />                                            leading ectopic otoconia BPPV. In addition, studies presented suggest that <br />                                            reactivating expression OTOP1 ear patients vestibular dysfunction may <br />                                            enhance mineralization remaining otoconial particles reestablish otoconial function. <br />                                            Further understanding role Otop1 proteins required otoconial formation <br />                                            assist formulating therapeutic approaches aimed improving otoconial stability over <br />                                            time possibly facilitating otoconial regeneration, addition adding knowledge <br />                                            mechanisms calcification systems. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Acknowledgments <br />                                            The authors like thank Keith C. Cheng Pennsylvania State University College Medicine use of <br />                                            microscopy imaging equipment. This work funded NIH grant DC02236 (D.M.O., R.T.), DC006283 <br />                                            (M.E.W.), MH068789 (R.L.). We thank I. Thalmann, I. Boime, K. Lavine critically reading manuscript <br />                                            insightful discussion T. Nicolson providing starmaker situ hybridization probe. <br />  <br />  <br />                               References <br />                                            Addadi L, Berman A, Oldak JM, Weiner S. Structural stereochemical relations acidic <br />                                              macromolecules organic matrices crystals. Connect. Tissue Res 1989;21:127&#226;&#8364;&#8220;134. [PubMed: <br />                                              2691196] <br />  <br />  <br />  <br />  <br />                                                   Dev Biol. Author manuscript; available PMC 2008 August 26. <br />                             Hughes et al.                                                                                                   Page 10 <br />  <br />  <br />                                            Bergstrom RA, You Y, Erway LC, Lyon MF, Schimenti JC. Deletion mapping head tilt (het) gene <br />                                              mice: vestibular <span id='am-163' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-164' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-240' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span>mutation</span> causing specific absence otoliths. Genetics 1998;150:815&#226;&#8364;&#8220;822. <br />                                              [PubMed: 9755211] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Bever MM, Fekete DM. Atlas developing inner ear zebrafish. Dev. Dyn 2002;223:536&#226;&#8364;&#8220;543. <br />                                              [PubMed: 11921341] <br />                                            Borelli G, Mayer-Gostan N, Merle PL, Pontual H, Boeuf G, Allemand D, Payan P. Composition of <br />                                              biomineral organic matrices special emphasis turbot (Psetta maxima) otolith endolymph. <br />                                              Calcif. Tissue Int 2003;72:717&#226;&#8364;&#8220;725. [PubMed: 14563001] <br />                                            Carlstrom D, Engstrom H, Hjorth S. Electron microscopic X-ray diffraction studies statoconia. <br />                                              Laryngoscope 1953;63:1052&#226;&#8364;&#8220;1057. [PubMed: 13110207] <br />                                            Chan J, Mably JD, Serluca FC, Chen JN, Goldstein NB, Thomas MC, Cleary JA, Brennan C, Fishman <br />                                              MC, Roberts TM. Morphogenesis prechordal plate notochord requires intact Eph/ephrin B <br />                                              signaling. Dev. Biol 2001;234:470&#226;&#8364;&#8220;482. [PubMed: 11397014] <br />                                            Erway LC, Grider A Jr. Zinc metabolism lethal-milk mice. J. Hered 1984;75:480&#226;&#8364;&#8220;484. [PubMed: <br />                                              6512240] <br />                                            Erway LC, Purichia NA, Netzler ER, D&apos;Amore MA, Esses D, Levine M. Genes, manganese, zinc in <br />                                              formation otoconia: labeling, recovery, maternal effects. Scanning Electron Microsc <br />                                              1986:1681&#226;&#8364;&#8220;1694. <br />                                            Everett LA, Glaser B, Beck JC, Idol JR, Buchs A, Heyman M, Adawi F, Hazani E, Nassir E, Baxevanis <br />                                              AD, Sheffield VC, Green ED. Pendred syndrome caused mutations putative sulphate <br />                                              transporter gene (PDS). Nat. Genet 1997;17:411&#226;&#8364;&#8220;422. [PubMed: 9398842] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Haddon C, Jiang YJ, Smithers L, Lewis J. Delta-Notch signalling patterning sensory cell <br />                                                differentiation zebrafish ear: evidence mind bomb mutant. Development <br />                                                1998;125:4637&#226;&#8364;&#8220;4644. [PubMed: 9806913] <br />                                            Hurle B, Ignatova E, Massironi SM, Mashimo T, Rios X, Thalmann I, Thalmann R, Ornitz DM. Non- <br />                                                syndromic vestibular disorder otoconial agenesis tilted/mergulhador mice caused by <br />                                                mutations Otopetrin 1. Hum. Mol. Genet 2003;12:777&#226;&#8364;&#8220;789. [PubMed: 12651873] <br />                                            Johnsson LG, Wright CG, Preston RE, Henry PJ. Streptomycin-induced defects otoconial <br />                                                membrane. Acta Otolaryngol. (Stockh.) 1980;89:401&#226;&#8364;&#8220;406. [PubMed: 6108671] <br />                                            Khan SR. Interactions stone-forming calcific crystals macromolecules. Urol. Int <br />                                                1997;59:59&#226;&#8364;&#8220;71. [PubMed: 9392052] <br />                                            Kozel PJ, Friedman RA, Erway LC, Yamoah EN, Liu LH, Riddle T, Duffy JJ, Doetschman T, Miller <br />                                                ML, Cardell EL, Shull GE. Balance hearing deficits mice null mutation gene <br />                                                encoding plasma membrane Ca2+-ATPase isoform 2. J. Biol. Chem 1998;273:18693&#226;&#8364;&#8220;18696. <br />                                                [PubMed: 9668038] <br />                                            Lim DJ. Formation fate otoconia. Scanning transmission electron microscopy. Ann. Otol., <br />                                                Rhinol., Laryngol 1973;82:23&#226;&#8364;&#8220;35. [PubMed: 4631100] <br />                                            Lim DJ. Morphogenesis malformation otoconia: review. Birth Defects 1980;16:111&#226;&#8364;&#8220;146. <br />                                                [PubMed: 6113015] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Lim DJ. Otoconia health disease. A review. Ann. Otol., Rhinol., Laryngol., Suppl 1984;112:17&#226;&#8364;&#8220; <br />                                                24. <br />                                            Lim, DJ.; Erway, LC.; Clark, DL. Tilted-head mice genetic otoconial anomaly. Behavioural and <br />                                                morphological correlates.. In: Hood, JD., editor. Vestibular Mechanisms Health Disease. <br />                                                Academic Press; london: 1978. p. 195-206. <br />                                            Malicki J, Schier AF, Solnica-Krezel L, Stemple DL, Neuhauss SC, Stainier DY, Abdelilah S, Rangini <br />                                                Z, Zwartkruis F, Driever W. Mutations affecting development zebrafish ear. Development <br />                                                1996;123:275&#226;&#8364;&#8220;283. [PubMed: 9007247] <br />                                            Mizuno R, Ijiri K. Otolith formation mutant Medaka deficiency gravity sensing. Adv. Space <br />                                                Res 2003;32:1513&#226;&#8364;&#8220;1520. [PubMed: 15000116] <br />                                            Murayama E, Okuno A, Ohira T, Takagi Y, Nagasawa H. Molecular cloning expression otolith <br />                                                matrix protein cDNA rainbow trout, Oncorhynchus mykiss. Comp. Biochem. Physiol., B <br />                                                Biochem. Mol. Biol 2000;126:511&#226;&#8364;&#8220;520. [PubMed: 11026663] <br />                                            Nasevicius A, Ekker SC. Effective targeted gene &#226;&#8364;&#732;knockdown&#226;&#8364;&#8482; zebrafish. Nat. Genet 2000;26:216&#226;&#8364;&#8220; <br />                                                220. [PubMed: 11017081] <br />  <br />  <br />                                                   Dev Biol. Author manuscript; available PMC 2008 August 26. <br />                             Hughes et al.                                                                                                  Page 11 <br />  <br />  <br />                                            National Institute Deafness Other Communication Disorders. National Institutes Health (NIH) <br />                                                Balance, Dizziness, You. 2002NIH Publication No. 00&#226;&#710;&#8217;4374 (G) <br />                                            Oghalai JS, Manolidis S, Barth JL, Stewart MG, Jenkins HA. Unrecognized benign paroxysmal positional <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                vertigo elderly patients. Otolaryngol.-Head Neck Surg 2000;122:630&#226;&#8364;&#8220;634. [PubMed: 10793337] <br />                                            Ornitz DM, Bohne BA, Thalmann I, Harding GW, Thalmann R. Otoconial agenesis tilted mutant mice. <br />                                                Hear. Res 1998;122:60&#226;&#8364;&#8220;70. [PubMed: 9714575] <br />                                            Paffenholz R, Bergstrom RA, Pasutto F, Wabnitz P, Munroe RJ, Jagla W, Heinzmann U, Marquardt A, <br />                                                Bareiss A, Laufs J, Russ A, Stumm G, Schimenti JC, Bergstrom DE. Vestibular defects head- <br />                                                tilt mice result mutations Nox3, encoding NADPH oxidase. Genes Dev 2004;18:486&#226;&#8364;&#8220;491. <br />                                                [PubMed: 15014044] <br />                                            Pote KG, Ross MD. Each otoconia polymorph protein unique polymorph. Comp. Biochem. <br />                                                Physiol 1991;98B:287&#226;&#8364;&#8220;295. <br />                                            Pote KG, Hauer CR, Michel H, Shabanowitz J, Hunt DF, Kretsinger RH. Otoconin-22, major protein <br />                                                aragonitic frog otoconia, homolog phospholipase A2. Biochemistry 1993;32:5017&#226;&#8364;&#8220;5024. <br />                                                [PubMed: 8494877] <br />                                            Preston RE, Johnsson LG, Hill JH, Schacht J. Incorporation radioactive calcium otolithic <br />                                                membranes middle ear ossicles gerbil. Acta Otolaryngol 1975;80:269&#226;&#8364;&#8220;275. [PubMed: <br />                                                1180040] <br />                                            Richardson KC, Jarrett L, Finke EH. Embedding epoxy resin ultrathin sectioning electron <br />                                                microscopy. Stain Tech 1960;35:313&#226;&#8364;&#8220;323. <br />                                            Riley BB, Grunwald DJ. A mutation zebrafish affecting localized cellular function required for <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                normal ear development. Dev. Biol 1996;179:427&#226;&#8364;&#8220;435. [PubMed: 8903357] <br />                                            Riley BB, Moorman SJ. Development utricular otoliths, saccular otoliths, necessary for <br />                                                vestibular function survival zebrafish. J. Neurobiol 2000;43:329&#226;&#8364;&#8220;337. [PubMed: 10861559] <br />                                            Riley BB, Zhu C, Janetopoulos C, Aufderheide KJ. A critical period ear development controlled by <br />                                                distinct populations ciliated cells zebrafish. Dev. Biol 1997;191:191&#226;&#8364;&#8220;201. [PubMed: <br />                                                9398434] <br />                                            Ross MD. Calcium ion uptake exchange otoconia. Adv. Oto-Rhino-Laryngol 1979;25:26&#226;&#8364;&#8220;33. <br />                                            Sheldrick, GM. TWINABS CELL_NOW. University Goettingen; Goettingen, Germany: 2002. <br />                                            S&#195;&#182;llner C, Burghammer M, Busch-Nentwich E, Berger J, Schwarz H, Riekel C, Nicolson T. Control of <br />                                                crystal size lattice formation starmaker otolith biomineralization. Science 2003;302:282&#226;&#8364;&#8220; <br />                                                286. [PubMed: 14551434] <br />                                            Sumanas S, Larson JD, Miller, Bever M. Zebrafish chaperone protein GP96 required otolith <br />                                                formation ear development. Dev. Biol 2003;261:443&#226;&#8364;&#8220;455. [PubMed: 14499652] <br />                                            Tachibana M, Morioka H. Glucuronic acid-containing glycosaminoglycans occur otoconia: <br />                                                cytochemical evidence hyaluronidase-gold labeling. Hear. Res 1992;62:11&#226;&#8364;&#8220;15. [PubMed: <br />                                                1429247] <br />                                            Takumida M, Zhang DM, Yajin K, Harada Y. Effect streptomycin otoconial layer guinea <br />                                                pig. ORL J. Otorhinolaryngol. Relat. Spec 1997;59:263&#226;&#8364;&#8220;268. [PubMed: 9279864] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Tateda M, Suzuki H, Ikeda K, Takasaka T. pH regulation globular substance otoconial <br />                                                membrane guinea-pig inner ear. Hear. Res 1998;124:91&#226;&#8364;&#8220;98. [PubMed: 9822906] <br />                                            Tusa RJ. Benign paroxysmal positional vertigo. Curr. Neurol. Neurosci. Rep 2001;1:478&#226;&#8364;&#8220;485. [PubMed: <br />                                                11898559] <br />                                            Veenhof, VB. The Development Otoconia Mice. North Holland; Amsterdam: 1969. <br />                                            Wang Y, Kowolski PE, Thalmann I, Ornitz DM, Mager DL, Thalmann R. Otoconin-90, mammalian <br />                                                otoconial matrix protein, contains domains homology secretory phospholipase A2. Proc. <br />                                                Natl. Acad. Sci. USA 1998;22:15345&#226;&#8364;&#8220;15350. [PubMed: 9860971] <br />                                            Whitfield TT, Riley BB, Chiang MY, Phillips B. Development zebrafish inner ear. Dev. Dyn <br />                                                2002;223:427&#226;&#8364;&#8220;458. [PubMed: 11921334] <br />  <br />  <br />  <br />  <br />                                                  Dev Biol. Author manuscript; available PMC 2008 August 26. <br />                             Hughes et al.                                                                                               Page 12 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Fig. 1. <br />                                            Whole mount situ hybridization analysis otop1 mRNA expression. (A) Lateral view a <br />                                            24 hpf embryo showing significant expression otop1 ventral half developing <br />                                            otic vesicle. (B) Dorsal view 24 hpf showing otop1 expression. (C) Lateral view 3 dpf <br />                                            larva showing otop1 mRNA expression sensory epithelium. (D) Four-&#206;&#188;m plastic section <br />                                            otocyst 3 dpf fish (dorsal up, lateral right) stained Richardson&apos;s stain <br />                                            (100&#195;&#8212;). Pale apical cells round nuclei (arrow) hair cells. Cells elongated, densely <br />                                            staining nuclei (arrowheads) precursor cells macular growth zone. (E) Similar <br />                                            4 &#206;&#188;m plastic section 3 dpf fish showing otop1 mRNA localized luminal cells the <br />                                            otocyst adjacent cells monolayer, consistent expression mature and <br />                                            developing hair cells (100&#195;&#8212;). (F) Lateral view 5 dpf larva showing otop1 expression along <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            entire <span id='am-5' about='xsp:length' typeof='owl:Thing'>length</span> animal anterior lateral line organs (arrowheads). Scale bars: <br />                                            A, 50 &#206;&#188;m; B&#226;&#8364;&#8220;C, 250 &#206;&#188;m; D&#226;&#8364;&#8220;E, 10 &#206;&#188;m; F, 50 &#206;&#188;m. <br />  <br />  <br />  <br />  <br />                                                      Dev Biol. Author manuscript; available PMC 2008 August 26. <br />                             Hughes et al.                                                                                             Page 13 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Fig. 2. <br />                                            Absence otolith formation otop1 morphant fish. (A) Lateral view uninjected wild- <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            type control 30 hpf. (B) Lateral view 30 hpf morphant fish injected 10 ng MO-1. <br />                                            Otop1 morphant fish morphologically normal stages development lack otolith <br />                                            formation. (C D) Lateral view control morphant 30 hpf otocysts. Otoliths located <br />                                            poles developing otocyst control fish (arrows) (C) absent 10-ng <br />                                            MO-1-injected animals (arrow) (D). Scale bars: A&#226;&#8364;&#8220;B, 50 &#206;&#188;m; C&#226;&#8364;&#8220;D, 250 &#206;&#188;m. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                      Dev Biol. Author manuscript; available PMC 2008 August 26. <br />                             Hughes et al.                                                                                           Page 14 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Fig. 3. <br />                                            Normal gene expression otocyst morphogenesis morphants. (A) The pax2a expression <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            wild-type (B) morphant otocyst. (C) otx1 expression wild-type (D) morphant <br />                                            otocyst. (E) Immunohistochemistry acetylated tubulin 10 &#206;&#188;m frozen sections a <br />                                            28 hpf wild-type otocyst. Tether cell kinocilia (arrow) otolith (arrowhead) evident in <br />                                            magnified insert (E&#226;&#8364;&#178;). (F) Morphant otocyst examined acetylated tubulin. Tether cell <br />                                            kinocilia evident magnified insert (F&#226;&#8364;&#178;) (arrow) otolith seeding particles were <br />                                            present. (G) DIC image otocyst morphant fish (100&#195;&#8212;) 24 hpf. No aggregated <br />                                            matrix visible ears examined manner. (H) omp expression wild-type <br />                                            (I) morphant otocyst. (J) starmaker expression wild-type (K) morphant otocyst. A&#226;&#8364;&#8220; <br />                                            D H&#226;&#8364;&#8220;K mount situ hybridizations 24 hpf embryos. All otop1 morphant fish <br />                                            injected 4 ng MO-1 one-cell stage. Rostral left. Scale bars: A&#226;&#8364;&#8220;D, H&#226;&#8364;&#8220; <br />                                            K, 250 &#206;&#188;m; E&#226;&#8364;&#8220;F, 10 &#206;&#188;m; G, 25 &#206;&#188;m. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                      Dev Biol. Author manuscript; available PMC 2008 August 26. <br />                             Hughes et al.                                                                                         Page 15 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Fig. 4. <br />                                            Normal formation lateral line otop1 morphant fish. (A&#226;&#8364;&#8220;B) SEM image 4 <br />                                            dpf WT (A) morphant (B) posterior lateral line neuromast. The cupula kinocilia bundle <br />                                            formed normally morphant animal. (C D) Four-&#206;&#188;m plastic section 7 <br />                                            dpf uninjected control (C) 10 ng MO-1-injected morphant anterior neuromast showing a <br />                                            similar complement hair cells (pale, apical cells round nuclei) supporting cells. <br />                                            Scale bars indicate 10 &#206;&#188;m. C D stained Richardson&apos;s stain. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                      Dev Biol. Author manuscript; available PMC 2008 August 26. <br />                             Hughes et al.                                                                                                Page 16 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Fig. 5. <br />                                            Morphogenesis otocyst sensory epithelium. (A B) Lateral view 5 dpf wild- <br />                                            type (A) morphant (B) otocyst. In wild-type fish, utricular otolith smaller and <br />                                            located rostral portion otocyst, saccular otolith larger located centrally. <br />                                            The semicircular canals ampullae (arrowhead) similar control 10 ng MO-1 <br />                                            morphant fish otolith agenesis. (C) Plastic section utricular macula 7 dpf <br />                                            wild-type fish (100&#195;&#8212;) showing development hair cells (pale apical cells, arrow) and <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            supporting cells (darker basal cells, arrowhead). (D) Plastic section 7 dpf morphant saccular <br />                                            macula showing similar distribution cell types. (E F) Scanning electron micrographs <br />                                            wild-type (E) morphant (F) utricular otolithic membrane 7 dpf. Removal the <br />                                            otolith disrupted wild-type otolithic membrane. Morphant fish formed otolithic <br />                                            membrane absence otolith. The otolithic membrane fibrous connects the <br />                                            stereocilia hair cells macula. O, otolith. Scale bars: A&#226;&#8364;&#8220;B, 250 &#206;&#188;m; C&#226;&#8364;&#8220;D, 10 &#206;&#188;m; <br />                                            E&#226;&#8364;&#8220;F, 10 &#206;&#188;m. <br />  <br />  <br />  <br />  <br />                                                      Dev Biol. Author manuscript; available PMC 2008 August 26. <br />                             Hughes et al.                                                                                                Page 17 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Fig. 6. <br />                                            Delayed otolith formation dysmorphology otop1 morphant fish. (A&#226;&#8364;&#8220;F) Lateral views of <br />                                            7 dpf otocysts (rostral left) wild-type (A) 1 ng MO-1-injected fish (B&#226;&#8364;&#8220;F). (A) Wild- <br />                                            type fish develop ovoid otoliths. (B) Delayed otoliths morphant fish appear similar <br />                                            wild type shape location. (C) Otoliths identical location, oblong <br />                                            shape distinct straight edges. (D) Formation single otolith larger wild- <br />                                            type otolith attached saccular macula. The irregular shape large size indicate <br />                                            fusion early otolith seeding particles. (E) A single cuboidal otolith located saccular <br />                                            sensory macula. (F) Multiple otoliths polyhedral structures. The posterior-most otolith <br />                                            did appear attached sensory macula. (G) Four-&#206;&#188;m plastic section 7 dpf <br />                                            1 ng MO-1-injected morphant otocyst showing wild-type-like otolith attached saccular <br />                                            macula. Richardson&apos;s stain identifies concentric rings organic matrix otolith <br />                                            structure. (H) Four-&#206;&#188;m plastic section opposite otocyst showing angular otolith <br />                                            utricular sensory patch obvious organic matrix crystal. (I) Lateral <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            view 4 dpf otocyst (rostral left) 8-ng MO-1-injected fish. At 4 dpf, numerous <br />                                            irregularly shaped otoliths otic cavity semicircular canal <br />                                            (arrow). (J) SEM image 4 dpf morphant otocyst. Note presence aggregates <br />                                            crystals attached utricular saccular otolithic membranes additional free- <br />                                            floating crystals otocyst wall (several otolith particles lost preparation). Scale <br />                                            bar A&#226;&#8364;&#8220;F 250 &#206;&#188;m; G&#226;&#8364;&#8220;H 10 &#206;&#188;m; I 250 &#206;&#188;m; J 25 &#206;&#188;m. <br />  <br />  <br />  <br />  <br />                                                      Dev Biol. Author manuscript; available PMC 2008 August 26. <br />                             Hughes et al.                                                                                                 Page 18 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Fig. 7. <br />                                            Calcitic otolith formation otop1 morphant fish. (A) Scanning electron microscopy the <br />                                            otic cavity 7 dpf uninjected age-matched control showing smaller utricular (left) and <br />                                            larger saccular otolith. Both otoliths rounded cover entire sensory maculae. (B) <br />                                            Eight-ng MO-1-injected morphant otic cavity 7 dpf. Otoliths angular. The sensory <br />                                            epithelium visible utricular otolith. The morphant otoliths resembled inorganic <br />                                            crystals instead organic calcification. (C) The 360&#194;&#176; rotation image single crystal X-ray <br />                                            diffraction wild-type 7 dpf otolith. Little prominent diffraction pattern present, indicating <br />                                            calcium carbonate aragonite crystallites arranged dustlike mosaic pattern across <br />                                            surface otolith. (D) Single crystal X-ray diffraction morphant otolith showing <br />                                            otoliths similar (B) behave single crystal. The unit cell derived <span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span> <br />                                            diffraction pattern consistent calcitic polymorph calcium carbonate. Scale <br />                                            bar indicates 50 &#206;&#188;m. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                      Dev Biol. Author manuscript; available PMC 2008 August 26. <br />                                                                  NIH-PA Author Manuscript             <br /> </body></html>