<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Identifying essential genes Escherichia coli <br /> <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> metabolic optimization principle <br /> Carlotta Martellia, Andrea De Martinob, Enzo Marinaric,1, Matteo Marsilid, Isaac Pe&#194;&#180;rez Castilloe <br /> aDipartimento di Fisica, bConsiglio Nazionale delle Ricerche (Istituto Sistemi Complessi)/Istituto Nazionale la Fisica della Materia (Centre for <br />  <br /> Statistical Mechanics Complexity) Dipartimento di Fisica, cDipartimento di Fisica, Consiglio Nazionale delle Ricerche/Istituto Nazionale la <br /> Fisica della Materia Istituto Nazionale di Fisica Nucleare, Universita` di Roma &#226;&#8364;&#732;&#226;&#8364;&#732;La Sapienza&#226;&#8364;&#8482;&#226;&#8364;&#8482;, Piazzale A. Moro 2, 00185 Rome, Italy; dThe Abdus Salam <br /> International Center Theoretical Physics, Strada Costiera 11, 34014 Trieste, Italy; eDepartment Mathematics, King&#226;&#8364;&#8482;s College London, Strand, <br /> London WC2R 2LS, United Kingdom <br />  <br /> Communicated Nicola Cabibbo, University Rome, Rome, Italy, December 29, 2008 (received review April 3, 2008) <br />  <br /> Understanding organization reaction fluxes cellular me-                 or, matrix notation, (A &#226;&#171;&#186; B)&#226;?&#175; &#226;&#171;&#189; 0, &#226;?&#175; &#226;&#171;&#189; {&#226;?&#175;1,&#226;&#8364;&#166;,&#226;?&#175;N} a <br /> tabolism stoichiometry topology under-                    flux vector. The problem finding flux vector satisfying <br /> lying biochemical network central issue systems biology. In               set P linear equations (1). For real metabolic networks, the <br /> this task, important devise reasonable approximation                     usually underdetermined N &#226;&#172;&#381; P [e.g., <br /> schemes rely stoichiometric data only, full-                  Escherichia coli &#226;&#172;&#8225;1,100 reactions &#226;&#172;?800 metabolites (1)], <br /> scale kinetic approaches computationally affordable                  multiple solutions exist, specify of <br /> small networks (e.g., red blood cells, &#226;&#172;&#8225;50 reactions). Methods                    working, physical states network. <br /> commonly used based finding stationary flux configu-                      In organisms high functional specificity main <br /> rations satisfy mass-balance conditions metabolites,               objective produce certain specified metabolites, like, e.g., <br /> coupling local optimization rules (e.g., maximization                  red blood cells, physical states taken consistent <br /> biomass production) reduce size solution space                 mass-balance conditions. In cases, important <br /> single point. Such methods widely applied                      able explore solution space Eq. 1 uniformly. <br /> proven able reproduce experimental findings relatively <br />                                                                                   Uniform sampling achieved small networks Monte <br /> simple organisms specific conditions. Here, define study <br />                                                                                   Carlo methods (7), and, recently, message-passing algo- <br /> a constraint-based model cellular metabolism neither <br />                                                                                   rithms used (8, 9). However, computational consid- <br /> mass balance flux stationarity postulated the <br />                                                                                   erations prevent application techniques <span id='am-7' about='protege:TO' typeof='owl:Thing'>to</span> <br /> relevant flux configurations optimize global growth the <br />                                                                                   explore general aspects larger metabolic networks. <br /> system. In case Escherichia coli, steady flux states are <br />                                                                                      For complex organisms, normally complements Eq. <br /> recovered solutions, mass-balance conditions vi- <br /> olated metabolites, implying nonzero net production of <br />                                                                                   1 assumption physical state network <br /> the latter. Such solutions furthermore turn provide                    obeys specific optimization principle. In flux-balance analysis <br /> correct statistics fluxes bacterium E. coli different               (FBA) (5), choice usually falls maximization of <br /> environments compare available experimental                     biomass production, useful proxy growth capabilities of <br /> evidence individual fluxes. Conserved metabolic pools play                   organism. FBA widely applied different organ- <br /> key role determining growth rate flux variability. Finally,             isms (mostly bacteria) investigate general aspects metab- <br /> are able connect phenomenological gene essentiality                       olism, like flux distributions different environments (10), <br /> &#226;&#8364;&#732;&#226;&#8364;&#732;frozen&#226;&#8364;&#8482;&#226;&#8364;&#8482; fluxes (i.e., fluxes smaller allowed variability) E.           evolutionary dispensability enzymes (11), plasticity of <br /> coli metabolism.                                                                  reaction network (12). Minimization metabolic adjust- <br />                                                                                   ment (MOMA) (6) instead able predict reorganization <br /> fluxomics &#229;&#8230;&#169; growth &#229;&#8230;&#169; stoichiometry &#229;&#8230;&#169; conserved moieties &#229;&#8230;&#169; gene essentiality       fluxes reaction <span id='am-8' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-9' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span>knockout</span> minimizing distance <br />                                                                                   FBA solution obtained Eq. 1 the <br />                                                                                   knockout. <br /> C    ellular metabolism involves complex network interac- <br />      tions cross-regulations metabolites, proteins, and <br /> genes. Although knowledge regulatory functions the <br />                                                                                      Here, consider constraint-based model metabolic <br />                                                                                   activity aim characterizing flux states corresponding <br /> genetic level level protein&#226;&#8364;&#8220;protein interactions                 optimal net metabolic production. We assume con- <br /> still infancy, biochemical network reactions                   stancy-in-time fluxes impose mass-balance con- <br /> describes metabolism characterized                    straints metabolites. Rather, allow recov- <br /> organisms (1&#226;&#8364;&#8220;3). Most information metabolic network                     ered properties solutions. This assumption turns to <br /> is contained matrices B &#226;&#171;&#189; {bi&#226;?&#174;} A &#226;&#171;&#189; {ai&#226;?&#174;} describing,                  able reproduce empirical statistics fluxes the <br /> respectively, input output stoichiometric coefficients                 bacterium E. coli different environments remarkable <br /> each metabolite &#226;?&#174; (ranging 1 P) metabolic                  accuracy, physical biochemical origin robust- <br /> reactions (1 N). Their knowledge allows <span id='am-4' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span>                ness emerging picture completely understood. <br /> constraint-based models prediction metabolic                      Further biological insight gained comparing the <br /> fluxes possible (4&#226;&#8364;&#8220;6). These models typically rely                        variability individual fluxes data phenomenological <br /> steady-state assumption reflects time scale separation                   gene essentiality. <br /> between (faster) equilibration metabolic processes the <br /> (slower) dynamics genetic regulation. Under condition, <br /> the concentration X&#226;?&#174; metabolite &#226;?&#174; flux &#226;?&#175;i &#226;&#177;&#8211; 0                       Author contributions: C.M., A.D.M., E.M., M.M., I.P.C. performed research. <br /> reaction constant time globally linked set                   The authors declare conflict interest. <br />                                                                                                                                                                           BIOPHYSICS <br />  <br />  <br />  <br />  <br /> mass-balance conditions:                                                          Freely available online PNAS open access option. <br />                                                                                   1To   correspondence addressed. E-mail: enzo.marinari@roma1.infn.it. <br />  <br />  <br />                   &#229;&#8224;&#732; <br />              &#226;?&#174;     N <br />          dX                                                                       This article contains supporting information online www.pnas.org/cgi/content/full/ <br />              &#226;&#171;&#189;          &#229;&#8230;&#177;ai&#226;?&#174; &#226;&#171;&#186; bi&#226;?&#174;&#229;&#8230;&#178;vi &#226;&#172;&#8230; 0,     @&#226;?&#174; &#226;&#171;&#189; 1, . . . , P         [1]      0813229106/DCSupplemental. <br />           dt <br />                   i&#226;&#171;&#189;1                                                             &#194;&#169; 2009 The National Academy Sciences USA <br />  <br />  <br />  <br /> www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.0813229106                                                        PNAS &#229;&#8230;&#169; February 24, 2009 &#229;&#8230;&#169; vol. 106 &#229;&#8230;&#169; no. 8 &#229;&#8230;&#169; 2607&#226;&#8364;&#8220;2611 <br />  Model Definitions <br /> The <span id='am-1' about='protege:abstract' typeof='owl:Thing'>abstract</span> setup consider originally introduced J. <br /> Von Neumann (13) model growth production economies as <br /> an autocatalytic process. We consider N chemical <br /> reactions P reagents, fluxes concentrations <br /> evolving discrete time t &#226;&#171;&#189; 0,1,&#226;&#8364;&#166; (we leave time scale <br /> unspecified). For now, assumed purely <br /> autocatalytic, total input given metabolite at <br /> a certain time step come (all of) output <br /> at previous time step. Let Si(t) denote scale which <br /> reaction operates time t, total input output <br /> of metabolite &#226;?&#174; given I&#226;?&#174;(t) &#226;&#171;&#189; &#194;&#165;iSi(t)bi&#226;?&#174; O&#226;?&#174;(t) &#226;&#171;&#189; <br /> &#194;&#165;iSi(t)ai&#226;?&#174;, respectively. Stability requires C&#226;?&#174;(t) :&#226;&#171;&#189; O&#226;?&#174;(t) &#226;&#171;&#186; <br /> I&#226;?&#174;(t &#226;&#171;&#185; 1) &#226;&#177;&#8211; 0 times, need an <br /> outside metabolic source survive. We focus attention on <br /> the feasibility dynamical paths constant growth rate &#226;?&#179; &#226;&#172;&#381;      Fig. 1. Flux distributions &#226;?&#179;* E. coli minimal environment 3 <br /> 0, I&#226;?&#174;(t &#226;&#171;&#185; 1) &#226;&#171;&#189; &#226;?&#179;I&#226;?&#174;(t). It easy that, paths,   different carbon sources: glutamate (glu), succinate (suc), glucose (glc). <br /> reaction rates behave Si(t) &#226;&#171;&#189; si&#226;?&#179;t, constants si &#226;&#177;&#8211; 0     Here, N &#226;&#171;&#189; 1,053 P &#226;&#171;&#189; 630 (glu); N &#226;&#171;&#189; 1,054 P &#226;&#171;&#189; 630 (suc); N &#226;&#171;&#189; 1,057 and <br /> satisfying P linear constraints                                   P &#226;&#171;&#189; 631 (glc). (Inset) The distributions plotted binning the <br />                                                                       abscissa. <br />  <br />  <br />                &#229;&#8224;&#732; <br />                N <br />  <br />         c&#226;?&#174; &#226;&#172;&#8230;         &#229;&#8230;&#177;ai&#226;?&#174; &#226;&#171;&#186; &#226;?&#179;bi&#226;?&#174;&#229;&#8230;&#178;si &#226;&#177;&#8211; 0,   @&#226;?&#174; &#226;&#171;&#189; 1, . . . , P.   [2] <br />                                                                       &#226;&#8364;&#732;&#226;&#8364;&#732;Leaf Removal.&#226;&#8364;&#8482;&#226;&#8364;&#8482; Once network built, remove <br />                i&#226;&#171;&#189;1 <br />                                                                       internal metabolites produced, <br /> The main assumption network&#226;&#8364;&#8482;s physical state          corresponding constraints satisfied by <br /> s* &#226;&#171;&#189; {si&#226;&#180;&#177;} corresponds growth rate      taking &#226;?&#179; &#226;&#171;&#189; 0 applying null flux corresponding <br /> &#226;?&#179; attains maximum possible value &#226;?&#179;* compatible           reactions (as actually FBA). <br /> constraints (Eq. 2). The rationale choice lies fact <br /> that, general conditions, proven paths      Accounting Reversibility. We disposed reversible reac- <br /> the optimal use resources coincide, apart initial          tions introducing 2 separate fluxes taking absolute <br /> transient, maximal expansion (14). Note        difference positive net flux. <br /> sis essentially discrete-time version continuous          The size resulting network, i.e., N P, ultimately <br /> time fluxes &#226;?&#175;i Eq. 1.                                              different different environments. <br />    It intuitive solution space Eq. 2 shrinks &#226;?&#179;      We calculate maximum growth rate &#226;?&#179;* numerically a <br /> increased starting &#226;?&#179; &#226;&#171;&#189; 0, flux configuration      generalized MinOver algorithm (18), detailed ref. 17, which <br />                                                                       solutions fixed &#226;?&#179;, &#226;?&#179; gradually increased. <br /> viable state. Quite importantly, then, fixed set input&#226;&#8364;&#8220; <br />                                                                       It shown rigorously (17) algorithm converges <br /> output relations, solutions Eq. 2 bound coincide <br />                                                                       solution fixed &#226;?&#179; 1 solution exists. Moreover, <br /> with Eq. 1 &#226;?&#179;* &#226;&#171;&#189; 1 c&#226;?&#174; &#226;&#171;&#189; 0 @&#226;?&#174;. <br />                                                                       multiple solutions occur (in case form a <br />    The typical behavior Von Neumann&#226;&#8364;&#8482;s model fully <br />                                                                       connected set), algorithm provides uniform sampling the <br /> appreciated analytically case random graphs. Depend- <br />                                                                       solution space [see supporting information (SI) test this <br /> ing size topology, autocatalytic networks with <br />                                                                       property low dimensions]. Anticipating flux <br /> random stoichiometry rise different optimal states.      vectors satisfy Eq. 2 &#226;?&#179;* E. coli, particularly <br /> As constrained optimization problems (15), key           important advantage uniform sampling required to <br /> control parameter ratio N/P &#226;&#171;&#189; n variables-to-               characterize solution space. <br /> constraints. In random topologies (16, 17), n increases, the <br /> system crosses contracting phase &#226;?&#179;* &#226;&#172;? 1        Characterization Solutions <br /> expanding &#226;?&#179;* &#226;&#172;&#381; 1, passing stationary regime        For metabolic network E. coli, &#226;?&#179;* &#226;&#171;&#189; 0.999 &#226;&#171;&#190; <br /> with &#226;?&#179;* &#226;&#171;&#189; 1 reaction fluxes constant time       0.001 independently environmental conditions have <br /> FBA (although values fluxes settle different).    tested, state optimal growth compatible one <br />                                                                       constant fluxes. The distribution reaction fluxes &#226;?&#179;* <br /> Application E. coli <br />                                                                       (see Fig. 1) displays regime (&#226;&#172;?2 decades) scaling form <br /> A natural reaction network modeled autocatalytic         P(s) &#226;&#172;&#8225; s&#226;&#171;&#186;&#226;?&#165; exponent close &#226;&#171;&#186;1 [in reasonable agreement <br /> system uptake reactions provide resources           experimental evidence based limited set of <br /> account exchanges metabolites surrounding             measured fluxes (19, 10)], followed cutoff. Note flux <br /> environment included. We applied Von Neumann             histogram remarkably stable different solutions. This <br /> scheme cellular metabolism bacterium E. coli,        scenario partially reproduced FBA (10), the <br /> reconstructed ref. 1. To set stage, 3 different operations     solution obtained optimizing biomass production systemat- <br /> have performed.                                                 ically overestimates &#226;?&#165;. To compare individual fluxes exper- <br />                                                                       iments, aimed studying solution obtained conditions <br /> Environment Selection. To fix environment cell lives,   similar described ref. 19, focusing attention 17 <br /> we defined set external metabolites             fluxes central metabolism, ref. 6. Unfortunately, <br /> applied uptake fluxes. We considered 3 distinct environ-         unable reproduce M9 medium used ref. <br /> mental conditions: (i) isolated cell, uptakes; (ii) minimal   19, fix uptakes 3 metabolites, glucose, <br /> environment, uptakes restricted set metabolites (10),    CO2, O2 identically ref. 19. For remaining the <br /> namely CO2, K, NH4, PI, O2, SO4, glu-L, succ, glc;      environment, chose simulate minimal medium fixed <br /> (iii) rich environment, uptakes external metabolites.     4 remaining external uptakes arbitrary values (we did not <br />  <br /> 2608 &#229;&#8230;&#169; www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.0813229106                                                                                    Martelli et al. <br />  Fig. 2. Comparison reaction fluxes predicted Von Neumann scheme <br /> with 17 fluxes measured ref. 19 analyzed ref. 6. The different <br /> reactions (in specific order) reported horizontal axis (labeled as <br /> in ref. 6), corresponding fluxes (relative glucose uptake) the <br /> vertical axis. Red markers denote experimental values, blue ones theoretical <br /> predictions (see, however, Characterization Solutions details the <br /> environmental conditions considered). Note flux vectors satisfy Von <br /> Neumann&#226;&#8364;&#8482;s conditions optimal growth. Three different solutions <br /> are reported here. <br />                                                                                         Fig. 4. Overlap-based comparison solution spaces E. coli and <br />                                                                                         random counterpart. (A B) Mean overlap 500 different <br /> observe strong dependency results parameters).                        solutions function &#226;?&#179; E. coli random metabolic network; last <br /> Results shown Fig. 2. We stress medium                               point abscissa corresponds value &#226;?&#179;* estimated simulation. <br /> consider strictly identical used experiments. It                   (C D) Overlap histogram P(q) &#226;?&#179;* E. coli random network. <br />                                                                                         Note different scales y axes lower images. <br /> worth noting (see ref. 20) experiments employ 13C- <br /> labeled carbon sources substrates growing bacterial <br /> culture kept constant density, relative abundance <br /> of metabolites captured nuclear magnetic resonance                         This quantity equals 1 s&#226;?&#163; s&#226;?&#164; coincide becomes <br /> mass spectroscopy. Reaction fluxes different metabolic path-                         smaller smaller distance s&#226;?&#163; s&#226;?&#164; the <br /> ways inferred, assuming model reaction                                space flux vectors increases. In computing it, should <br /> network.                                                                                consider flux zero value threshold <br />    With stationarity recovered &#226;?&#179;* &#226;&#171;&#189; 1, discussed above,                       &#226;?&#167;, account fact loss information <br /> difference Von Neumann solution FBA                             relative fluctuations different solutions fluxes <br /> arises fact c&#226;?&#174;s attain lowest allowed                       smaller &#226;?&#167;. We chosen &#226;?&#167; &#226;&#171;&#189; 10&#226;&#171;&#186;5 ensure overlaps <br /> value. This clearly seen Fig. 3. In solution,                           overestimated. Results obtained &#226;?&#167; &#226;&#171;&#189; 10&#226;&#171;&#186;5, 10&#226;&#171;&#186;6, and <br /> metabolites exist nonzero c&#226;?&#174;, implying steady                        10&#226;&#171;&#186;7 identical. We furthermore taken account the <br /> state, net production metabolites occurs. This                       fact overlap null fluxes defined by <br /> signal incompleteness stoichiometric data.                                    consistency equal 1. In Fig. 4, report behavior of <br />    To insight existence multiple flux states                      mean overlap &#229;&#8230;&#183;q&#226;?&#163;&#226;?&#164;&#229;&#8230;&#184; (the angular brackets representing an <br /> compatible Eq. 2 &#226;?&#179;* shape solution                            average pairs solutions) E. coli a <br /> space, rough effective way consists monitoring mean                        network topology N/P random Gaussian <br /> overlap different solutions. Given 2 solutions &#226;?&#163; &#226;?&#164; <br />                                                                                         stoichiometry. Random networks provide important bench- <br /> at fixed &#226;?&#179;, define overlap q&#226;?&#163;&#226;?&#164; as: <br />                                                                                         mark behavior real networks tested, <br />  <br />                                     &#229;&#8224;&#732;                       &#229;&#8224;&#732; <br />                                     N                       N                           highlight extent observations specific the <br />                                 2            s i&#226;?&#163;s i&#226;?&#164;   1                               particular organism studied. In particular, role <br />                         q&#226;?&#163;&#226;?&#164; &#226;&#171;&#189;                         &#226;&#171;&#189;           q&#229;&#8230;&#177;&#226;?&#163;&#226;?&#164; <br />                                                                      i&#229;&#8230;&#178; <br />                                                                         .         [3] <br />                                 N         si2&#226;?&#163; &#226;&#171;&#185; si2&#226;?&#164; N                                 topology stoichiometry fully analyzed. For in- <br />                                     i&#226;&#171;&#189;1                     i&#226;&#171;&#189;1 <br />                                                                                         stance, ref. 17, shown power-law behavior the <br />                                                                                         flux distribution ascribed specific network <br />                                                                                         topology. <br />                                                                                   12       One sees artificial metabolic network, &#229;&#8230;&#183;q&#226;?&#163;&#226;?&#164;&#229;&#8230;&#184; an <br /> 100 <br />                                                                                   10    increasing function &#226;?&#179; approaches 1 &#226;?&#179;3 &#226;?&#179;* (Fig. 4B). <br /> 200                                                                               8 <br />                                                                                         Moreover, overlap histogram marked &#226;?&#166;-peak q &#226;&#171;&#189; 1 <br />                                                                                         mass increases &#226;?&#179; 3 &#226;?&#179;*. These results indicate the <br />                                                                                   6     optimal solution unique. On hand, behavior of <br /> 300 <br />                                                                                   4     &#229;&#8230;&#183;q&#226;?&#163;&#226;?&#164;&#229;&#8230;&#184; E. coli suggests volume solutions stops <br /> 400 <br />                                                                                   2 <br />                                                                                         contracting &#226;?&#179; reaches &#226;&#172;&#8225;0.8 below. In particular, &#226;?&#179;*, <br />                                                                                         multiple solutions survive. From corresponding histogram, <br /> 500 <br />                   100      200            300       400             500     600 <br />                                                                                   0     Fig. 4C, &#226;&#172;&#8225;30% reactions overlap close <br />                                     metabolites                                         1. In different jargon, say &#226;&#172;&#8225;30% the <br />                                                                                         variables frozen (i.e., assume value solutions <br /> Fig. 3. Values c&#226;?&#174; different metabolites (horizontal axis) 500 different       constrained optimization problem), remain- <br />                                                                                                                                                                             BIOPHYSICS <br />  <br />  <br />  <br />  <br /> solutions. The color <span id='am-3' about='Thesaurus:code' typeof='owl:Thing'>code</span> reported right. The white background <br />                                                                                         ing free. The existence frozen variables characterizes many <br /> corresponds c&#226;?&#174; &#226;&#171;&#189; 0. Colored marks indicate net production the <br /> corresponding metabolite solution mass balance violation. One              random constraint satisfaction problems (21), normally <br /> sees mass balance holds metabolites, con-               hard identify topological motifs variables more <br /> sistently unbalanced, be, depending                likely frozen. In present case, reasonable expect <br /> solution.                                                                               purely structural reasons, reaction chains entirely <br />  <br /> Martelli et al.                                                                                              PNAS &#229;&#8230;&#169; February 24, 2009 &#229;&#8230;&#169; vol. 106 &#229;&#8230;&#169; no. 8 &#229;&#8230;&#169; 2609 <br />  Fig. 5. E. coli&#226;&#8364;&#8482;s central metabolism: Nodes represent metabolites; arrow <br /> joining 2 nodes present reaction exists converting other. <br /> Red (green) links denote frozen (free) reactions, overlap larger (smaller) <br /> than 0.9. <br />  <br />  <br />  <br /> frozen reaction chain frozen. This is <br /> indeed confirmed map frozen/free fluxes E. coli&#226;&#8364;&#8482;s <br /> central metabolism, displayed Fig. 5. The chain-like of <br /> glycolysis appears frozen. <br />    The existence frozen fluxes raises obvious question of <br /> their biological significance. To address point, have <br />                                    (i)                                             Fig. 6. Essential genes (vertical axis) versus overlap corresponding <br /> correlated reaction overlaps q&#226;?&#163;&#226;?&#164;       essentiality                reactions reconstructed metabolic network E. coli (horizontal axis). <br /> corresponding genes according notion universal essen- <br /> tiality used ref. 22, combines phenomenological rele- <br /> vance (a gene essential knocking causes cell die)              pool coupled common functional group. For <br /> with evolutionary retention (the presence gene different                 example 3 metabolites ATP, AMP, ADP belong a <br /> species). In ref. 22, 55 essential genes E. coli involved                    pool cofactors preserve adenylate moiety (24). This <br /> metabolism identified present 80%                    implies change concentration given metabolite <br /> 32 different bacterial genomes. We able link 52 of <br />                                                                                    accomplished considering entire pool to <br /> such genes reactions reconstructed network. It turns <br />                                                                                    metabolite belongs. <br /> out, Fig. 6, 43 genes correspond reactions with <br />                                                                                      A conserved pool g formally defined P-dimensional <br /> overlap &#226;&#172;&#381;0.8 7 genes relate reactions an <br /> overlap significantly &#226;&#172;?80%. Whether gene &#226;&#8364;&#732;&#226;&#8364;&#732;essential&#226;&#8364;&#8482;&#226;&#8364;&#8482;                 <span id='am-6' about='xsd:boolean' typeof='owl:Thing'>Boolean</span> vector elements zg&#226;?&#174; zg&#226;?&#174; &#226;&#171;&#189; 1 &#226;?&#174; &#229;&#402;&#8224; g zg&#226;?&#174; &#226;&#171;&#189; <br /> (according <span id='am-5' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span> ref. 22) depends choice                  0 satisfying <br /> the environment. In particular, ref. 22 considers rich medium <br /> as environment. For comparison, considered the- <br /> oretical values overlaps fluxes calculated assuming a <br /> rich environment. Note that, principle, gene be <br /> linked variable flux different environments. <br /> However, results presented qualitatively preserved the <br /> minimal environment different carbon sources. This sug- <br /> gests frozen fluxes, Von Neumann&#226;&#8364;&#8482;s framework are <br /> allowed limited variability state optimal growth is <br /> to kept, carry evolutionary significance. <br />  <br /> Role Conserved Moieties <br /> To trace physical origin results presented above, <br /> we studied rank matrix A &#226;&#171;&#186; &#226;?&#179;B associated the <br /> P linear constraints (Eq. 2) function parameter &#226;?&#179;, see <br /> Fig. 7. <br />    The singularity occurring &#226;?&#179; &#226;&#171;&#189; 1 related presence                   Fig. 7. The rank matrix associated inequalities (2) for <br /> of conserved pools metabolites, groups reagents                        E. coli (isolated cell) function &#226;?&#179;. Around singularity, rank equals <br /> total concentration constant time (23). Their existence                   number metabolites; conserved pools metabolites present only <br /> due fact concentrations metabolites certain                exactly &#226;?&#179; &#226;&#171;&#189; 1. <br />  <br />  <br /> 2610 &#229;&#8230;&#169; www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.0813229106                                                                                                     Martelli et al. <br />                    &#229;&#8224;&#732; <br />                    P                                                                         belong conserved pool (within stoichiometric descrip- <br />                                                                                              tion used). <br />                         zg&#226;?&#174;&#229;&#8230;&#177;ai&#226;?&#174; &#226;&#171;&#186; bi&#226;?&#174;&#229;&#8230;&#178; &#226;&#171;&#189; 0,       @i &#226;&#171;&#189; 1, . . . , N.                 [4] <br />                                                                                                 We conclude state optimal growth &#226;?&#179;* &#226;&#171;&#189; 1, <br />                   &#226;?&#174;&#226;&#171;&#189;1 <br />                                                                                              addition stationarity reaction rates, spontaneous <br /> Such conservation laws manifest precisely &#226;?&#179; &#226;&#171;&#189; 1,                               condition mass balance holds most, all, metabolites. <br />                                                                                                 In summary, Von Neumann model relies assump- <br /> By virtue Eq. 2, relation &#194;&#165;&#226;?&#174; zg&#226;?&#174;c&#226;?&#174; &#226;&#177;&#8211; 0 hold g. <br />                                                                                              tion arrangement metabolic fluxes follows principle <br /> It follows &#226;?&#179;* &#226;&#177;&#8226; 1, fluxes connected a <br />                                                                                              growth maximization. It does imply priori the <br /> metabolite belonging conserved pool equal zero.                              mass balance stationarity fluxes, 2 conditions <br /> Because null solution s &#226;&#171;&#189; 0 discarded obvious                                 essentially recovered maximum growth rate. This is <br /> physical grounds, fluxes connected metabolites                            presence conserved metabolic pools. The singu- <br /> conserved pool necessarily &#226;?&#179;* &#226;&#177;&#8226; 1. This consistent                              larity rank A &#226;&#171;&#186; &#226;?&#179;B &#226;?&#179; &#226;&#171;&#189; 1 implies physically <br /> the results obtained different environmental conditions                               relevant solutions tends &#226;?&#179; 3 1. <br /> implies scenario stable small pertur-                              In limit, mass balance recovered (i.e., c&#226;?&#174; &#226;&#171;&#189; 0) most, but <br /> bations network topology.                                                             all, metabolites. This provides nontrivial correction the <br />    Moreover, Eqs. 4 2, easy that, &#226;?&#179; &#226;&#171;&#189; 1,                           picture extracted FBA reproduces limited experi- <br /> &#226;?&#174; &#229;&#402;&#8224; g implies c&#226;?&#174; &#226;&#171;&#189; 0, i.e., metabolite belonging                                  mental evidence available. <br /> conserved pool mass-balance condition strictly valid. <br /> We shown instead (Fig. 3) values constraints                            ACKNOWLEDGMENTS. We gratefully acknowledge important discussions <br />                                                                                              G. Bianconi, R. Monasson, A. Pagnani, F. Ricci Tersenghi a <br /> are zero: For metabolites, mass balance                                  number useful suggestions. M.M. supported Information Society <br /> condition reached, assert                                 Technologies STREP GENNETEC contract no. 034952. <br />  <br />  <br />  1. Reed JL, Vo TD, Schilling CH, Palsson B&#195;&#732; (2003) An expanded genome-scale model        13. Von Neumann J (1937) U&#194;&#168;ber ein o&#194;&#168;konomisches Gleichungssystem und eine Verallge- <br />     Escherichia coli K-12 (iJR904 GSM/GPR). Genome Biol 4:R54.                                   meinerung des Brouwerschen Fixpunktsatzes. Ergebn Math Kolloq 8:73&#226;&#8364;&#8220; 83. English <br />  2. Wiback SJ, Palsson B&#195;&#732; (2002) Extreme pathway analysis human red blood cell                translation Rev Econ Studies 13:1 (1945). <br />     metabolism. Biophys J 83:808 &#226;&#8364;&#8220; 818.                                                      14. McKenzie LW (1986) Optimal economic growth, turnpike theorems comparative <br />  3. Duarte NC, Herrg&#195;&#165;rd MJ, Palsson B&#195;&#732; (2004) Reconstruction validation Saccha-           dynamics. Handbook Mathematical Economics, eds Arrow KJ, Intriligator MD (North <br />     romyces cerevisiae iND750, fully compartmentalized genome-scale metabolic model.           Holland, Amsterdam), Vol 3, 1281&#226;&#8364;&#8220;1355. <br />     Genome Res 14:1298 &#226;&#8364;&#8220;1309.                                                                15. Monasson R (2007) Optimization problems statistical physics. Complex Systems <br />  4. Edwards JS, Ibarra RU, Palsson B&#195;&#732; (2001) In silico predictions Escherichia coli           LXXXV, Lecture Notes Les Houches Summer School 2006 (Elsevier, Amsterdam) <br />     metabolic capabilities consistent experimental data. Nat Biotech 19:125&#226;&#8364;&#8220;130.        1&#226;&#8364;&#8220; 65. <br />  5. Kauffman KJ, Prakash P, Edwards JS (2003) Advances flux balance analysis. Curr Opin   16. De Martino A, Marsili M (2005) Typical properties optimal growth Von <br />     Biotech 14:491&#226;&#8364;&#8220; 496.                                                                         Neumann expanding model large random economies. J Stat L09003. <br />  6. Segre` D, Vitkup D, Church GM (2002) Analysis optimality natural perturbed     17. De Martino A, Martelli C, Monasson R, Perez Castillo I (2007) Von Neumann&#226;&#8364;&#8482;s expand- <br />     metabolic networks. Proc Natl Acad Sci USA 99:15112&#226;&#8364;&#8220;15117.                                   ing model random graphs. J Stat P05012. <br />  7. Wiback SJ, Famili I, Greenberg HJ, Palsson B&#195;&#732; (2004) Monte Carlo sampling used    18. Krauth W, Me&#194;&#180;zard M (1987) Learning algorithms optimal stability neural <br />     determine size shape steady-state flux space. J Theor Biol 228:437&#226;&#8364;&#8220;        networks. J Phys A 20:L745&#226;&#8364;&#8220;L752. <br />     447.                                                                                     19. Emmerling M, et al. (2002) Metabolic flux responses pyruvate kinase knockout in <br />  8. Bianconi G, Zecchina R (2008) Viable flux distribution metabolic networks. Networks       Escherichia coli. J Bacteriol 184:152&#226;&#8364;&#8220;164. <br />     Heterogeneous Media 3:361&#226;&#8364;&#8220;369.                                                           20. Sauer U (2006) Metabolic networks motion: 13C-based flux analysis. Mol Syst Biol 2:62. <br />  9. Braunstein A, Mulet R, Pagnani A (2008) Estimating size solution space     21. Semerjian G (2008) On freezing variables random constraint satisfaction <br />     metabolic networks. BMC Bioinformatics 9:240.                                                problems. J Stat Phys 130:251&#226;&#8364;&#8220;293. <br /> 10. Almaas E, Kovacs B, Vicsek T, Oltvai ZN, Baraba&#194;&#180;si AL (2004) Global organization      22. Gerdes SY, et al. (2003) Experimental determination level analysis of <br />     metabolic fluxes bacterium Escherichia coli. Nature 427:839 &#226;&#8364;&#8220; 843.                    essential genes Esche <br /> </body></html>