<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Characterization PMT Gene Family Cryptococcus <br /> neoformans <br /> Sven D. Willger1,2*, Joachim F. Ernst1, J. Andrew Alspaugh3, Klaus B. Lengeler1 <br /> 1 Institut fu&#194;&#168;r Mikrobiologie, Molekulare Mykologie, Heinrich-Heine-Universita&#194;&#168;t, Du&#194;&#168;sseldorf, Germany, 2 Department Veterinary Molecular Biology, Montana State <br /> University, Bozeman, Montana, United States America, 3 Departments Medicine Molecular Genetics/Microbiology, Duke University Medical Center, Durham, <br /> North Carolina, United States America <br />  <br />  <br />  <br />      <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />      Background: Protein-O-mannosyltransferases (Pmt&#226;&#8364;&#8482;s) catalyze initial step protein-O-glycosylation, addition of <br />      mannose residues serine threonine residues target proteins. <br />  <br />      Methodology/Principal Findings: Based protein similarities, highly conserved protein family divided into <br />      subfamilies: Pmt1 sub-family, Pmt2 sub-family Pmt4 sub-family. In contrast Saccharomyces <br />      cerevisiae Candida albicans, similar filamentous fungi, putative PMT genes (PMT1, PMT2, PMT4) were <br />      identified genome human fungal pathogen Cryptococcus neoformans. Similar Schizosaccharomyces pombe <br />      C. albicans, C. neoformans PMT2 essential gene. In contrast, pmt1 pmt4 single mutants viable; however, <br />      pmt1/pmt4 deletions synthetically lethal. <span id='am-241' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-242' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-244' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-246' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-277' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-279' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-284' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-289' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-290' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-317' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-318' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span>Mutation</span> PMT1 PMT4 resulted distinct defects cell morphology <br />      cell integrity. The pmt1 mutant susceptible SDS medium wild-type strains mutant cells were <br />      enlarged. The pmt4 mutant grew poorly high salt medium demonstrated abnormal septum formation defects in <br />      cell separation. Interestingly, pmt1 pmt4 mutants demonstrated variety-specific differences levels of <br />      susceptibility osmotic cell wall stress. Delayed melanin production pmt4 mutant alteration of <br />      classical virulence-associated phenotypes. However, pmt1 pmt4 mutants showed attenuated virulence murine <br />      inhalation model cryptococcosis. <br />  <br />      Conclusion/Significance: These findings suggest C. neoformans protein-O-mannosyltransferases play crucial role in <br />      maintaining cell morphology, reduced protein-O-glycosylation leads alterations stress resistance, cell wall <br />      composition, cell integrity, <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span>survival</span> host. <br />  <br />   Citation: Willger SD, Ernst JF, Alspaugh JA, Lengeler KB (2009) Characterization PMT Gene Family Cryptococcus neoformans. PLoS ONE 4(7): e6321. <br />   doi:10.1371/journal.pone.0006321 <br />   Editor: Xiaorong Lin, Texas A &amp; M University, United States America <br />   Received May 8, 2009; Accepted June 23, 2009; Published July 27, 2009 <br />   Copyright: &#195;&#376; 2009 Willger et al. This open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: This study supported DAAD-short-term-scholarship (Deutscher Akademischer Austausch Dienst, http://www.daad.de/en/index.html) <br />   Deutsche Forschungsgemeinschaft (SPP1160)(http://www.dfg.de). The funders role study design, data collection analysis, decision <span id='am-6' about='protege:TO' typeof='owl:Thing'>to</span> <br />   publish, preparation manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: sven.willger@montana.edu <br />  <br />  <br />  <br /> Introduction                                                                        Broader phylogenetic analyses reveal protein-O-mannosyltrans- <br />                                                                                     ferases grouped major subfamilies, corresponding to <br />     Protein-O-glycosylation essential, evolutionary conserved                 S. cerevisiae Pmt1p, Pmt2p Pmt4p. Many fungal species contain <br /> protein modification studied extensively yeasts                PMT genes genome, subfamily. <br /> Saccharomyces cerevisiae Candida albicans. This process                   In S. cerevisiae, O-glycosylation affects stability, localization, <br /> been identified fungal species [1&#226;&#8364;&#8220;3], higher eukaryotes                 function proteins, preventing exportation misfolded <br /> [4&#226;&#8364;&#8220;6] certain bacterial genera [7,8]. In yeasts fungi,                   proteins ER [12&#226;&#8364;&#8220;14]. Similarly, human pathogenic <br /> protein-O-glycosylation initiated luminal                     fungus C. albicans, correct O-mannosylation important for <br /> endoplasmic reticulum (ER) addition mannosyl residue                    morphogenesis, adherence host cells, virulence [15]. C. <br /> to specific serine/threonine residues proteins entering                      albicans proteins modified Pmt&#226;&#8364;&#8482;s include chitinases, <br /> secretory pathway [9,10]. This modification derived                   proteases, proteins involved glucan synthesis, heat-shock proteins, <br /> the polyisoprenoid carrier lipid dolichyl phosphate-activated                       cell-surface antigens important virulence (reviewed [2,3]). <br /> mannose (Dol-P-Man), followed addition short, linear,                     In addition secreted proteins, proper function various <br /> mannosyl-rich glycans. Maturation modification                   receptors requires intact protein-O-glycosylation activity [14,16]. <br /> glycosyl chains occur Golgi apparatus.                                          Although protein-O-mannosylation abundant in <br />     The initial step O-glycosylation catalysed highly conserved          higher eukaryotes, defects process result human disease, <br /> family integral ER membrane proteins, protein-O-mannosyl-                    muscle-eye-brain disease (MEB) Walker&#226;&#8364;&#8220;Warburg <br /> transferases (Pmt&#226;&#8364;&#8482;s) [11]. The Pmt family initially identified S.            Syndrome (WWS) [17]. Furthermore, targeted deletion of <br /> cerevisiae, seven PMT genes identified [3,11].                   POMT1, causes WWS mice resulted embryonic <br />  <br />  <br />        PLoS ONE | www.plosone.org                                               1                                    July 2009 | Volume 4 | Issue 7 | e6321 <br />                                                                                                                       C. neoformans PMT Gene Family <br />  <br />  <br /> lethality [18]. Mutation Drosophila PMT homologs alters muscle              growth conditions. The wild-type strains JEC21 (serotype D) and <br /> structures alignment adult cuticle [19,20]. Therefore,              H99 (serotype A) incubated mid-logarithmic phase rich <br /> the analysis PMT-gene family different species              medium (YPD) 30uC 37uC, salt stress (YPD+1 M NaCl), <br /> last years revealed protein-O-mannosylation activity               capsule-inducing conditions (DMEM). Northern blots and <br /> involved central developmental growth processes                 quantitative real-time PCR demonstrated little variation the <br /> uni- multicellular eukaryotes.                                             expression of PMT genes strain variety <br />     C. neoformans opportunistic human fungal pathogen causing            conditions (data shown). Therefore, C. <br /> life-threatening meningoencephalitis. This fungus produces             neoformans PMT genes constitutively expressed most <br /> extracellular factors important virulence, including           growth conditions. Our data support confirm recent <br /> extensive polysaccharide capsule, secreted hydrolytic                  observation PMT4A gene induced changes in <br /> enzymes, cell wall-targeted pigment melanin (reviewed                  temperature <span id='am-2' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> availability [22]. <br /> in [21]). Since protein-O-glycosylation predominantly affects <br /> extracellular proteins, defect biological process              Protein-O-glycosylation essential C. neoformans <br /> affect interface pathogenic microorganism host.                    We used targeted gene disruption characterise the <br />     PMT4, putative Pmt orthologs basidiomycete             biological functions C. neoformans PMT genes both <br /> C. neoformans, recently identified. Disruption C.              serotype backgrounds. Since shown organisms <br /> neoformans PMT4 gene results dramatic effects virulence [22].            pmt mutant strains highly sensitive cell wall <br /> In addition, pmt4 mutant strains morphological defects                destabilizing drugs drugs targeting protein synthesis, including <br /> alterations cell wall, possibly changes glycan                used C. neoformans dominant selectable markers (such as <br /> composition/synthesis. However, relative roles C. neoformans            hygromycin B), used ADE2 URA5 genes selectable <br /> PMT1 PMT2 elucidated. In paper                   markers perform experiments. The selectable marker <br /> report continuing analysis complete C. neoformans PMT                 cassettes inserted loop region PMT genes, <br /> gene family biologically distinct varieties, var. grubii var.       predicted essential enzyme function [27]. <br /> neoformans. We identified C. neoformans PMT genes,                   While pmt1 pmt4 single mutant strains easily be <br /> have begun define overlapping distinct functions               isolated serotypes, able isolate pmt2 mutant <br /> stress response, cell wall integrity, survival host.                strains serotype, rounds of <br />                                                                                transformation (.300 transformants). This finding suggested that <br /> Results                                                                        PMT2 essential gene C. neoformans, S. pombe <br />                                                                                C. albicans [15,28]. To confirm hypothesis, we <br /> Three PMT genes present C. neoformans                                   isolated homozygous ade2/ade2 diploid strain cross of <br />    Fungal genomes typically contain multiple genes encoding                    strains JEC156 (MATa ade2 ura5) JEC157 (MATalpha ade2 ura5 <br /> protein-O-mannosyltransferases. For example, S. cerevisiae contains            lys1) according methods Sia et al. [29]. By transforming a <br /> seven PMT genes (PMT1-7), PMT genes present              pmt2::ADE2 disruption construct diploid strain, were <br /> genome human pathogenic fungus C. albicans. Three putative              able isolate heterozygous PMT2/pmt2::ADE2 strains similar <br /> PMT genes recently identified basidiomycetous yeast C.             frequencies (,10%) compared pmt1 pmt4 single <br /> neoformans variety neoformans (serotype D), major sub-            mutants. We allowed independent heterozygous diploid <br /> family: PMT1 (CND06150 chromosome 4), PMT2 (CNJ01930                        mutants sporulate. <span id='am-4' about='protege:FROM' typeof='owl:Thing'>From</span> sporulation, isolated 75 <br /> on chromosome 10), PMT4 (CND01240 chromosome 4)                         ADE+haploid progeny, wild-type PMT2 locus. In <br /> [23]. C. neoformans Pmt1 shares 41% amino acid identity 59%                contrast, genetic markers (mating type, LYS1) demon- <br /> similarity S. cerevisiae Pmt1; CnPmt2 47% identity 65%              strated expected levels recombination. We assume the <br /> similarity ScPmt2; CnPmt4 42% identity 61%                      ADE+progeny pmt2, ectopic integration <br /> similarity ScPmt4. Beyond direct amino acid sequence compar-                pmt2::ADE recombination restores ade2 allele <br /> ison, detailed phylogenetic sequence analysis places            wild-type. To determine isolated 75 ADE+haploid <br /> the C. neoformans PMT genes major PMT              progeny wild-type PMT2 locus performed Southern <br /> gene families. Hydrophobicity analysis predicted Pmt protein            Blot analyses diploid strains sporulated showing <br /> sequences revealed seven-transmembrane helical structure                     PMT2 pmt2::ADE2 alleles present (data not <br /> commonly predicted class proteins [9].                             shown). The progeny subsequently analyzed colony <br />    Although closely related, C. neoformans var. neoformans var.            PCR nature PMT2 allele present strains <br /> grubii strains (serotypes D A, respectively) distinct                 amplifying short fragment spanning region ADE2 <br /> characteristics terms intracellular signaling cellular               marker integrated (data shown). The failure isolate <br /> physiology [24&#226;&#8364;&#8220;26]. As scientific models, var. grubii strains             haploid pmt2 mutants sporulation PMT2/pmt2 <br /> been used extensively pathogenesis experiments, var.               diploid strongly suggests PMT2 gene essential. <br /> neoformans strains tractable mating genetic systems.                To verify insertion selectable marker cassette <br /> The PMT genes present var. grubii genome,                resulted loss transcript pmt1 pmt 4 mutants <br /> encoding proteins 97&#226;&#8364;&#8220;98% identical amino acid                  performed real time PCRs PMT1, PMT2, PMT4 <br /> level var. neoformans. For clarity, chosen            specific primers cDNA derived serotype D mutants <br /> refer var. grubii (serotype A) PMT genes PMT1A, PMT2A,               grown 30uC. As control used cDNA derived the <br /> and PMT4A; var. neoformans (serotype D) genes PMT1D,                serotype D wild-type strain JEC21. Since couldn&#226;&#8364;&#8482;t generate a <br /> PMT2D, PMT4D.                                                              pmt2 mutant normalized data strain PMT2 <br />                                                                                expression. The results confirm PMT genes are <br /> PMT genes C. neoformans constitutively expressed                        constitutively expressed little variation the <br /> under various growth conditions                                                wild-type strain JEC21 (Figure 1). In insertion mutants the <br />    We characterized transcriptional regulation entire C.            transcript PMT1 PMT4 completely abolished the <br /> neoformans PMT gene family various physiologically relevant              respective mutant. The real time PCR data clearly that <br />  <br />  <br />        PLoS ONE | www.plosone.org                                          2                                July 2009 | Volume 4 | Issue 7 | e6321 <br />                                                                                                                     C. neoformans PMT Gene Family <br />  <br />  <br />  <br />  <br /> Figure 1. PMTs differentially expressed loss pmt gene compensated overexpression PMT <br /> gene. The wild-type (JEC21), Dpmt1 Dpmt4 mutant strains incubated mid-logarithmic phase YPD medium total RNA isolated. <br /> Quantitative real-time PCR performed corresponding cDNA samples assess relative PMT1, PMT2, PMT4 transcript abundance <br /> compared wild-type strain6one standard deviation. The expression values strains normalized PMT2 expression. <br /> doi:10.1371/journal.pone.0006321.g001 <br />  <br /> none PMT genes differentially expressed               properly mother cells, unusually large amounts cell <br /> mutants compensate loss PMT gene (Figure 1).                wall chitin area (Figure 4, arrow). These results confirmed <br />                                                                               recently published data pmt4A mutant cell <br /> The &#226;&#8364;&#732;&#226;&#8364;&#732;Pmt holoenzyme&#226;&#8364;&#8482;&#226;&#8364;&#8482; functions dimer multimer                       aggregates connected un-degraded cell wall material <br /> of individual Pmt proteins                                                demonstrated transmission electron microscopy [22]. <br />    It shown yeasts Pmt2 forms heterodimer           Moreover, single cell retained single nucleus shown <br /> with Pmt1 proteins dimers high protein-O-               DAPI staining (Figure 4) obvious effect of <br /> mannosylation activity [28,30]. Based finding                     pmt4 mutation cell cycle. <br /> interaction studies Pmts higher eukaryotes, i.e. human,                 In contrast pmt4 mutant strains, pmt1A pmt1D <br /> where POMT1 (member PMT4 family) POMT2                             mutants did prominent cell aggregation grown <br /> (member PMT2 family) form heterodimers [31], predicted              conditions (Figure 3A), calcofluor white <br /> that Cryptococcus Pmt2 forms heterodimers Pmts.            staining revealed normal <span id='am-320' about='xsp:pattern' typeof='owl:Thing'>pattern</span> chitin deposition cell <br /> This model protein function predict pmt1 pmt4               separation (Figure 4). When incubated 37uC, pmt1 pmt4 <br /> double mutation synthetically lethal            mutants demonstrated prominent morphological changes com- <br />                                                                               pared wild type. Both mutants grew large, dysmorphic cells <br /> no Pmt protein interact Pmt2. To test hypothesis, <br />                                                                               subject spontaneous lysis. Although DAPI staining revealed <br /> we crossed MATalpha pmt1D strain MATa pmt4D strain <br />                                                                               single nuclei strains, nuclei pmt1 pmt4 mutants <br /> and isolated individual basidiospores microdissection. We tested <br />                                                                               displaced cell periphery large central <br /> the resulting progeny strains following genetic markers: <br />                                                                               structure. This structure demonstrated enlarged <br /> mating type, PMT1, PMT4, ADE2, URA5. The marker <br />                                                                               vacuole staining FM4-64 (Figure 4). Although large <br /> segregation pattern demonstrated isolated spores the <br />                                                                               vacuole present cells 30uC, prominent in <br /> result meiotic recombination (Figure 2). More importantly, <br />                                                                               cells incubated 37uC. In contrast, wild-type cells displayed <br /> among 15 dissected spores able identify pmt1 <br />                                                                               multiple, small vacuoles incubated similar conditions. <br /> pmt4 double mutant, strongly suggesting pmt1 pmt4 <br /> mutations synthetically lethal. These results suggest protein- <br /> O-glycosylation essential process C. neoformans,            The pmt mutants sensitive different cell stresses <br /> either Pmt1 Pmt4 interact Pmt2 provide cell                 Pmt proteins play important roles cell wall architecture, and <br /> with optimal protein-O-mannosylation activity Pmt1                mutations proteins result increased sensitivity to <br /> Pmt4 common essential target independent Pmt2.                      various cell stresses elevated growth temperature, osmotic <br />                                                                               stress cell wall destabilising agents. Consistent the <br />                                                                               temperature-dependent cell morphology changes pmt <br /> pmt1 pmt4 mutant strains altered cell                                mutants, pmt1A pmt4A strains displayed marked defect <br /> morphology                                                                    growth elevated temperatures (Figure 5). When incubated on <br />    For yeasts filamentous fungi, loss certain                YPD medium 30uC, mutant strains grew similar wild <br /> protein-O-mannosyltransferases results cell morphology defects.            type. However, 37uC, pmt mutants displayed reduced <br /> In contrast wild-type cells grow single yeasts simple         colony size, consistent profound temperature-sensitive <br /> buds, C. neoformans pmt4A pmt4D mutant strains                   growth defect (Figure 5). Formal growth curves, measuring <br /> exhibited cell aggregation <span id='am-163' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-164' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-239' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-240' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> cells grown              changes optical density log-phase cultures liquid <br /> 24 h 48 h YPD SD medium 30uC. The cells               YPD, confirmed observations solid media (data shown). <br /> often associated multi-cell aggregates (Figure 3A). In addition,           In contrast wild type reconstituted strains, neither <br /> the cell aggregates resolved vortexing sonication          mutant strain grew 39uC (Figure 5). The pmt1D pmt4D <br /> suggesting Dpmt4 strain defect cell                    mutant strains displayed similar severe temperature- <br /> separation, altered cell adherence (Figure 3B).                   dependent growth defect. <br /> Calcofluor white staining cell wall chitin pmt4 mutant                 The pmt mutant strains serotypes displayed <br /> strains demonstrated failure daughter cells separate                  increased susceptibility salt stress (Figure 6 A). When incubated <br />  <br />  <br />        PLoS ONE | www.plosone.org                                         3                               July 2009 | Volume 4 | Issue 7 | e6321 <br />                                                                                                                      C. neoformans PMT Gene Family <br />  <br />  <br />  <br />  <br /> Figure 2. A pmt1 pmt4 double deletion lethal serotype D. Serotype D strains pmt1D (MATa pmt1D::URA5 ade2-27) pmt4D (MATa <br /> pmt4D::ADE2 ura5) crossed standard V8 mating media individual spores isolated days micromanipulation. A: 15 <br /> individual progeny analyzed colony PCR presence PMT1 (upper panel) PMT4 alleles (lower panel). Position respective <br /> wild-type disruption alleles indicated right. Wild-type strain JEC21 used control. B: The 15 strains A spotted the <br /> indicated plates, grown 2&#226;&#8364;&#8220;3 days 30uC subsequently analyzed auxotrophic marker distribution. Strains spotted left (#1) <br /> to right (wild-type control). C: Summary genotypes identified A B respect PMT alleles mating-type. Mating-type of <br /> the individual spores determined standard mating reactions using wild-type strains JEC20 (MATa) JEC21 (MATa) tester strains. <br /> doi:10.1371/journal.pone.0006321.g002 <br />  <br /> in presence 0.5 M KCl, pmt4A pmt4D strains grew               contrast, pmt1D mutant grows poorly presence of <br /> slower wild-type/reconstituted strains. Both pmt4 mutant                sorbitol (Figure 6B), growth unaffected SDS <br /> strains demonstrated striking growth inhibition                  (Figure 6C). The pmt4D strain inhibited high sorbitol <br /> addition 0.7 M 1 M NaCl medium (Figure 6A). The                 concentrations, contrast corresponding pmt4A strain; <br /> pmt1A pmt1D strains inhibited NaCl,                however, growth pmt4D inhibited SDS. Therefore, <br /> lesser extent pmt4 strains. The cell wall destabilising             sorbitol SDS different cell surface destabilizing <br /> agents Congo red, caffeine, calcofluor white significant          effects pmt mutants C. neoformans varieties, <br /> effect growth pmt mutants variety (data          suggesting Pmt proteins play distinct roles related <br /> shown).                                                                      divergent strain backgrounds. <br />    In contrast concordant effects salt temperature on <br /> the various pmt mutants different varieties, pmt1         Virulence factors C. neoformans differentially <br /> pmt4 mutants demonstrated variety-specific differences levels         affected pmt mutations <br /> of susceptibility osmotic cell wall stress. The pmt4A              Besides ability grow 37uC, virulence fungal <br /> mutant susceptible corresponding pmt1A strain            pathogen C. neoformans determined set specific virulence <br /> sorbitol (2 M 2.5 M) (Figure 6B), pmt1A mutant                factors, including secretion various hydrolytic enzymes, the <br /> more susceptible effects 0.1% SDS (Figure 6C). In                  ability produce polysaccharide capsule, expression of <br />  <br />  <br />        PLoS ONE | www.plosone.org                                        4                                July 2009 | Volume 4 | Issue 7 | e6321 <br />                                                                                                                 C. neoformans PMT Gene Family <br />  <br />  <br />  <br />  <br /> Figure 3. The pmt1 pmt4 disruption strains altered cell morphology. A: Serotype A wild-type strain H99 mutant strains <br /> pmt1A (pmt1A::URA5) pmt4A (pmt4A::URA5) incubated YPD 30uC 37uC OD600 1, cells subsequently analyzed by <br /> light microscopy (DIC). B: Serotype A pmt4A strain incubated YPD 30uC OD600 1, cell suspension analyzed light <br /> microscopy (DIC) treatment 1 min vortexing sonication. <br /> doi:10.1371/journal.pone.0006321.g003 <br />  <br /> the antioxidant melanin pigment. A common feature                 diameter cell itself [32]. The average capsule ratio 100 <br /> factors involvement extracellular components. Since              wild-type cells 1.8 (+/20.02), ratio similar the <br /> protein glycosylation mainly affects extracellular surface-             pmt1 mutant 1.75+/20.05). In contrast, capsule ratio the <br /> exposed proteins, hypothesized virulence             pmt4 mutant 1.45+/20.03, indicative slightly reduced <br /> factors affected mutations genes affecting protein-         capsule size (data shown). Results identical serotype A <br /> O-mannosylation. Compared wild type, pmt1 pmt4                  serotype D strains, reduction capsule size the <br /> mutants demonstrated changes activity secreted            pmt4 strains complemented wild-type PMT4 allele. <br /> enzymes urease phospholipase B (data shown), of <br /> which linked virulence C. neoformans.                      Protein-O-mannosylation mating <br />    Another secreted protein essential virulence C.              The effect Pmt&#226;&#8364;&#8482;s mating analyzed serotype D <br /> neoformans enzyme laccase, phenoloxidase catalyses       strains crosses vigorous serotype A, <br /> rate-limiting step melanin production. While pmt1 mutant                mating type associated virulence this <br /> strains did defect melanin production, pmt4            strain background [33,34]. The pmt1 mutant strains did show <br /> mutants (both serotype A serotype D) delayed melanin           obvious defect unilateral (wild-type x pmt1) bilateral <br /> production. The melanin production delay complemented               (pmt1 x pmt1) crosses. In contrast, pmt4 mutation showed <br /> the pmt4+PMT4 reconstituted strain (Figure 7). This finding                notable effects mating. Unilateral matings pmt4 strains to <br /> contrasts prior observation melanin production            wild-type testers behaved similar wild-type control crosses (data <br /> serotype A pmt4 mutant [22]. This difference small           shown). On hand, bilateral cross pmt4 strains <br /> differences disruption constructs used studies (see         revealed delayed mating reaction, reduced filament <br /> Discussion).                                                               formation 48 h comparison wild-type controls <br />    Another virulence-associated phenotype C. neoformans          (Figure 8). In addition, aerial hyphae produced during <br /> ability produce large polysaccharide capsule. Using standard        cross, indicated lack white mycelial rim <br /> India ink counter-stain, assessed capsule formation          surrounding mating patch (Figure 8; small picture). Micro- <br /> our pmt mutant strains. Microscopic analyses various                scopic analysis hyphal structures produced bilateral <br /> serotype A D mutant strains revealed strains             pmt4 mating reaction showed irregularly shaped thickened <br /> capsule incubated DMEM capsule-inducing medium. To                 filaments swollen distal tips, phenotypes reminiscent of <br /> control cell size variation mutant wild-type           mutations swoA locus Aspergillus nidulans, gene that <br /> cells, quantified relative capsule size determining total        interestingly recently encode PMT <br /> cell diameter cells surrounding capsule compared        genes present A. nidulans [35]. <br />  <br />  <br />       PLoS ONE | www.plosone.org                                       5                               July 2009 | Volume 4 | Issue 7 | e6321 <br />                                                                                                                      C. neoformans PMT Gene Family <br />  <br />  <br />  <br />  <br /> Figure 4. pmt1 pmt4 mutant strains altered cell wall phenotypes vacuole distribution. Wild-type serotype A strain H99 <br /> and pmt mutant strains pmt1A (pmt1A::URA5) pmt4A (pmt4A::URA5) incubated YPD 30uC OD600 1. Cells fixed 30 min in <br /> 10% formaldehyde subsequently stained described chitin staining dye calcofluor white, DNA dye DAPI vacuolar dye FM4-64. <br /> Pictures taken 400x using Zeiss Axioskop 2 Plus Fluorescence Microscope AxioCam MRM digital camera. <br /> doi:10.1371/journal.pone.0006321.g004 <br />  <br /> pmt mutant strains severely attenuated virulence                       modify proteins targeted secretion, process essential <br />    Considering effect different pmt mutations growth         fungi [3,28,36]. Its role fungal pathogenesis also <br /> virulence-associated phenotypes, likely mutations         explored C. albicans C. neoformans [22,37]. In this <br /> would negative effect virulence C. neoformans. We            report, extend prior studies Pmt function human <br /> therefore tested pathogenicity serotype A strains isolated       pathogenic yeast C. neoformans. First, demonstrated C. <br /> macrophages animals. When co-incubated                       neoformans contains PMT genes, gene encodes <br /> activated J774A.1 macrophages, pmt1A pmt4A strains                     Pmt enzyme major classes proteins. In <br /> demonstrated reduced survival 24 hours compared wild-type                contrast, fungi S. cerevisiae undergone <br /> (6-fold 40-fold reduction, respectively) (Figure 9). These relative        paralogous duplication PMT genes classes, perhaps <br /> differences macrophage survival likely differences            developing novel functions various Pmt proteins. <br /> high-temperature growth virulence factor expression.                          In addition conservation sequence similarity PMT <br />    In murine model inhaled cryptococcosis, mutant strains            genes, demonstrated basic function C. neoformans <br /> were significantly reduced virulence. In model, 105 C.            Pmt enzymes likely similarly conserved. For example, current <br /> neoformans colony-forming units intranasally inoculated               models Pmt enzyme activity suggest Pmt holoenzymes <br /> complement-defective A/Jcr mice, simulating natural respiratory            function optimally heterodimers [28,30,38], activity of <br /> route infection humans. Animals infected wild-type        Pmt2 class proteins required component protein <br /> reconstituted strains median survival 18&#226;&#8364;&#8220;19 days,              complexes [15,28]. Consistent working model developed <br /> mice survived past 21 days infection. The median survival             fungi, demonstrated PMT2 gene essential <br /> mice infected pmt4A mutant 40 days (p,0.001 compared              viability C. neoformans. Additionally, simultaneous <br /> to wild-type), mice infected pmt1A strain died            mutation C. neoformans PMT1 PMT4 genes to <br /> the infection course 45-day experiment (Figure 9).           synthetically lethal, predicted model Pmt2 <br />                                                                                function concert Pmt proteins. Such functional <br />                                                                                conservation suggests strategies block fungal Pmt function <br /> Discussion <br />                                                                                inhibit growth diverse fungal species. <br />    Over past years, Pmt-mediated process protein-O-                Besides essential process, demonstrated that <br /> glycosylation defined species. Pmt&#226;&#8364;&#8482;s primarily             protein-O-glycosylation significant effect virulence of <br />  <br />  <br />        PLoS ONE | www.plosone.org                                          6                               July 2009 | Volume 4 | Issue 7 | e6321 <br />                                                                                                                    C. neoformans PMT Gene Family <br />  <br />  <br />  <br />  <br /> Figure 5. pmt mutant strains growth defects high growth temperatures. Over-night cultures wild-type, pmt mutant and <br /> corresponding reconstituted strains serotype A (upper panel) serotype D (lower panel) adjusted OD600 0.1 diluted 10- <br /> fold serial dilutions 1024 dilution. 5 ml dilution step spotted YPD plates, plates incubated indicated <br /> temperature 2&#226;&#8364;&#8220;3 days. Strains used serotype A strains H99 (wild-type), pmt1A (pmt1A::URA5), pmt4A (pmt4A::URA5), pmt1A+PMT1A <br /> (pmt1A::URA5 PMT1A-NeoR) pmt4A+PMT4A (pmt4A::URA5 PMT4A-NeoR), respective serotype D strains JEC21 (wild-type), pmt1D <br /> (pmt1D::URA5), pmt4D (pmt4D::URA5), pmt1D+PMT1D (pmt1D::URA5 PMT1D-NeoR) pmt4D+PMT4D (pmt4D::URA5 PMT4D-NeoR). <br /> doi:10.1371/journal.pone.0006321.g005 <br />  <br /> C. neoformans, instances required             dramatic reduction melanin production. Our observation of <br /> viability. We demonstrated mutations PMT1              reduced melanin serotype A D pmt4 mutants different <br /> PMT4 genes result dramatic attenuation virulence             prior report melanin defect observed a <br /> macrophage killing assay mouse inhalation model                     serotype A pmt4 mutant [22]. One difference pmt4 <br /> cryptococcosis, models assess different aspects                  mutants method <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span>gene mutation</span>. Between 5th 6th <br /> cryptococcal pathogenesis. Our studies role Pmt4 C.             transmembrane domains, Pmt proteins characterized an <br /> neoformans pathogenesis consistent prior reports [22].              extensive loop 5 essential mannosyl-transferase activity <br /> However, experiments demonstrate pmt1 mutant               [27]. Loop 5 contains conserved motifs (A&#226;&#8364;&#8220;C) are <br /> even attenuated survival vivo compared pmt4                   important enzyme activity, domain C C- <br /> mutants, despite similar vitro temperature- cell wall-sensitive       terminal domain region protein. Olson et al. disrupted <br /> phenotypes. Interestingly, pmt1 pmt4 mutants high              Pmt4 downstream motif C; contrast, pmt4 mutations <br /> temperature sensitivity 39uC, great impact            resulted deletion domain C loop 5, supposedly <br /> infectivity mouse model. To investigate role                   resulting complete inactivation Pmt4 enzyme. <br /> temperature sensitivity virulence attenuation test               In addition melanin delay, pmt1 pmt4 mutant cells <br /> the virulence mutants heterologous host model,            display aberrant cell morphology pronounced cell aggrega- <br /> as wax moths (Galleria mellonella), does require high              tion. These morphological changes linked cell wall- <br /> temperature infection.                                                temperature-sensitive phenotypes identified vitro, resulting <br />    Virulence basidiomycete C. neoformans linked           cell lysis stress. Similarly, hyperflocculant dysmorphic <br /> several well-defined phenotypes, including production                C. neoformans cells demonstrated reduced virulence mouse <br /> export extracellular factors melanin, capsule,                model, increased susceptibility complement-activated <br /> various lytic enzymes [39&#226;&#8364;&#8220;43]. Of classic virulence-                   phagocytosis macrophages [45]. <br /> associated phenotypes, melanin production altered                   The temperature sensitivity pmt1 pmt4 mutants likely <br /> one strains, pmt4 mutant. C. neoformans strains        explains altered virulence strains. Other C. <br /> defective melanin production attenuated virulence,                neoformans strains altered growth mammalian physiological <br /> including strains mutations laccase gene LAC1,                   temperatures display similar, predictable virulence defects [46]. <br /> encoding rate limiting enzyme melanin production                      Interestingly, serotype A strains, serotype D, pmt1 <br /> [39,40]. Therefore, delay melanin production likely plays             mutant showed significant growth defect 37uC liquid <br /> a role reduced virulence pmt4 mutant strain.                   medium solid medium. The differential growth effect in <br />    The C. neoformans laccase Lac1 direct target             liquid medium caused higher shearing forces are <br /> Pmt4. Lac1 N-glycosylated, cell wall associated protein [44],          present growth solid medium. <br /> and N-glycosylation precedes O-glycosylation. Altered Pmt4                Moreover, previous reports demonstrated pmt4 deficient <br /> activity lead mislocalization Lac1             strains showed dramatic differences overall pattern of <br />  <br />  <br />        PLoS ONE | www.plosone.org                                        7                               July 2009 | Volume 4 | Issue 7 | e6321 <br />                                                                                                                        C. neoformans PMT Gene Family <br />  <br />  <br />  <br />  <br /> Figure 6. pmt mutant strains sensitive hyper-osmotic stresses. Over-night cultures wild-type, pmt mutant corresponding <br /> reconstituted strains serotype A (upper panel) serotype D (lower panel) adjusted OD600 0.1 diluted 10-fold serial <br /> dilutions 1024 dilution. 5 ml dilution step spotted YPD plates containing stress agents, plates incubated the <br /> indicated temperature 2&#226;&#8364;&#8220;3 days. Strains used serotype A strains H99 (wild-type), pmt1A (pmt1A::URA5), pmt4A (pmt4A::URA5), <br /> pmt1A+PMT1A (pmt1A::URA5 PMT1A-NeoR) pmt4A+PMT4A (pmt4A::URA5 PMT4A-NeoR), respective serotype D strains JEC21 (wild-type), <br /> pmt1D (pmt1D::URA5), pmt4D (pmt4D::URA5), pmt1D+PMT1D (pmt1D::URA5 PMT1D-NeoR) pmt4D+PMT4D (pmt4D::URA5 PMT4D-NeoR). Strains were <br /> spotted YPD plates supplemented various salts (A), sorbitol (B) SDS (C) indicated concentrations. To test G418 resistance (20 mg/ml) <br /> (C), night cultures initially diluted OD600 1.0 versus 0.1 used experiments. Plates incubated 30uC 2&#226;&#8364;&#8220;3 days. <br /> doi:10.1371/journal.pone.0006321.g006 <br />  <br />  <br />  <br />        PLoS ONE | www.plosone.org                                          8                                 July 2009 | Volume 4 | Issue 7 | e6321 <br />                                                                                                                  C. neoformans PMT Gene Family <br />  <br />  <br />  <br />  <br /> Figure 7. pmt4 pmt1 mutant strains delayed melanin synthesis. A: 10 ml over-night cultures serotype A strains H99 <br /> (wild-type), pmt1A (pmt1A::URA5), pmt4A (pmt4A::URA5) pmt4A+PMT4A (pmt4A::URA5 PMT4A-NeoR) spotted standard Niger seed- <br /> plates, plates incubated days 30uC. B: Cells 2 ml over-night cultures A harvested resuspended in <br /> 2 ml glucose-free asparagine-medium supplemented L-DOPA, cultures shaken 30uC night. Subsequently cells pelleted, <br /> and cell pellet supernatant photographed. C: OD480 supernatants cultures B determined graphically displayed <br /> using Prism 5 (GraphPad, San Diego, Calif.). Graph shows overall result independent experiments. <br /> doi:10.1371/journal.pone.0006321.g007 <br />  <br /> mannosylated proteins 2D gel electrophoresis [22]. Extracel-              composition proteins specifically affected the <br /> lular mannoproteins important regarding immunological                    pmt1 pmt4 mutation. The identification immediate targets <br /> aspects C. neoformans-host interactions including T-cell activation       individual cryptococcal Pmt proteins provide further <br /> [47&#226;&#8364;&#8220;49]. Therefore, defects protein-O-glycosylation              explanations defects pathogenicity pmt1 <br /> impair virulence C. neoformans altering immunological              pmt4 mutants C. neoformans. <br /> response microbial cells affect pathogenesis. It             Targets Pmt enzyme complex defined in <br /> therefore interesting analyze pmt1               various fungal species. In fact, Fks1 protein modified <br /> mutation results similar differences cell wall mannoprotein            C. neoformans Pmt4 [22]. Since Fks1 plays major role the <br />  <br />  <br />        PLoS ONE | www.plosone.org                                        9                             July 2009 | Volume 4 | Issue 7 | e6321 <br />                                                                                                                       C. neoformans PMT Gene Family <br />  <br />  <br />  <br />  <br /> Figure 8. pmt4D mutant strains serotype-dependent filamentation defects. To perform matings wild-type pmt mutant strains, <br /> the respective strains grown over-night YPD 30uC. Cells harvested, washed twice sterile 0.9% saline, incubated saline <br /> on shaker hour 30uC. 10 ml respective strains mixed, cells spotted standard solid V8 agar plates, were <br /> subsequently incubated RT dark. Edges colonies colonies (small pictures) photographed 48 h. Shown <br /> are bilateral matings serotype D strains JEC20 JEC21 (wild-type), SW5 (MATa) SW6 (MATa) (pmt1D mutants), SW8 (MATa) SW9 <br /> (MATa) (pmt4D mutants), respectively. <br /> doi:10.1371/journal.pone.0006321.g008 <br />  <br /> synthesis glucans, important components fungal                 proteins candidates involved in <br /> cell walls, observation offers potential mechanism cell         remodelling cell wall including chitin-deacetylases (Cda1- <br /> wall defects described pmt4 mutant strain [22]. Given              3) potential endoglucanases (AAW45003, AAW46063 <br /> similar distinct phenotypes pmt1 pmt4 mutants,            AAW46065). It interesting these <br /> likely Pmt proteins possess different affinities            proteins identified direct target Pmt-dependent O- <br /> Pmt target proteins.                                                           glycosylation deletions of corresponding genes <br />    Given role protein-O-glycosylation plays microbial              result morphological phenotypes similar pmt mutants. <br /> pathogenesis, investigators explored Pmt proteins potential               In conclusion, PMT gene encodes group proteins <br /> targets therapeutic intervention. Derivatives rhodanine-3-              critical biology virulence C. neoformans. Given high <br /> acetic acid recently identified specifically inhibit        degree conservation gene family pathogenic <br /> Pmt1 isoform C. albicans. Treating wild-type cells               fungi, gene family ideal antifungal drug target. Future <br /> inhibitors resulted phenotypic transcriptional changes                  studies explore possibility define the <br /> reminiscent C. albicans pmt1 deletion strains [15,50,51]. Since             mechanisms PMT family mutant defects. <br /> Pmt2 Pmt4 orthologs present mammals, possibility <br /> of developing Pmt isoform-specific inhibitors renders Pmt proteins             Materials Methods <br /> an attractive target antifungal drug development. Additionally, <br /> targeting specific Pmt isoforms inhibitory compounds                  Strains media <br /> also useful killing phytopathogenic fungi, Pmt proteins               Reference strains used study congenic serotype <br /> have identified plants [3].                                       D strains JEC20 (MATa) JEC21 (MATa), serotype A <br />    The cell wall defect pmt1 pmt4 mutants striking.              strain H99. pmt mutant strains constructed transforming the <br /> One mechanism proposed Pmt4 control C. neoformans cell              JEC20/21 derivatives JEC155 (MATa, ade2 ura5), JEC156 (MATa, <br /> wall integrity recently provided. The stress related induction             ade2 ura5), F99, respectively, 5-FOA resistant derivative of <br /> of Fks1 gene, encoding catalytic subunit b-1,3-glucan               H99. All strains used study listed Table 1. Yeast- <br /> synthase, dependent Pmt4 [22]. Since b-1,3-glucan major             peptone-dextrose (YPD) yeast nitrogen <span id='am-319' about='xsp:base' typeof='owl:Thing'>base</span> media, synthetic <br /> component fungal cell walls, altered glucan synthesis                 (SD) medium, V8 agar mating, filament agar, Niger seed for <br /> likely result altered cell integrity stress conditions [52].          melanin production, serum-free Dulbecco&#226;&#8364;&#8482;s modified Eagle&#226;&#8364;&#8482;s <br /> Moreover, defects C. neoformans PKC/cell integrity signalling               medium capsule induction prepared previously <br /> resulted phenotypes similar pmt mutant strains              reported [32,53,55,56]. <br /> [53]. These phenotypes included morphological defects, defects in <br /> vacuolar biogenesis, growth defects elevated growth tempera-                Plasmid construction <br /> tures low SDS concentrations, higher sensitivity                      PMT homologs identified tblastn searches using S. <br /> osmotic stresses. Even interesting shown                 cerevisiae Pmt1 C. albicans Pmt1 protein sequences the <br /> proper PKC/cell integrity signalling necessary melanin                  National Center Biotechnology Information (http://www.ncbi. <br /> production; pkc1 mutants improper laccase localization                nlm.nih.gov). For generating PMT disruption constructs PMT <br /> reduced melanin production. The cell wall abnormalities,                   open reading frames (ORF) amplified PCR from <br /> resulting changes PKC signalling, explain melanin              reference strains H99 JEC21, fragments obtained were <br /> production delay pmt4 mutants.                                          cloned standard cloning vectors. Except PMT1A BglII sites <br />    Levitz Specht identified 55 potential GPI anchored                      integrated PMT ORFs using oligonucleotide based <br /> membrane proteins serine/threonine rich regions                  site directed mutagenesis (Stratagene Quick Change site directed <br /> thought potential O-glycosylation targets [54]. Within             mutagenesis kit). Subsequently, ADE2 URA5 containing BamHI <br />  <br />  <br />        PLoS ONE | www.plosone.org                                         10                                July 2009 | Volume 4 | Issue 7 | e6321 <br />                                                                                                                 C. neoformans PMT Gene Family <br />  <br />  <br />  <br />  <br /> Figure 9. pmt mutant strains serotype A attenuated virulence. A: A macrophage killing assay performed serotype A <br /> wild-type strain H99 pmt mutant strains SW13 (pmt1A::URA5) SW14 (pmt4A::URA5) previously described. CFU independent <br /> experiments repetitions strain shown. B: A murine inhalation model performed serotype A strains H99 (wild-type), <br /> pmt1A (pmt1A::URA5), pmt4A (pmt4A::URA5), pmt1A+PMT1A (pmt1A::URA5 PMT1A-NeoR) pmt4A+PMT4A (pmt4A::URA5 PMT4A-NeoR) previously <br /> described. Briefly, strain 10 A/Jcr mice infected intranasally 105 CFU, survival time post infection determined. <br /> doi:10.1371/journal.pone.0006321.g009 <br />  <br /> fragments cloned BglII sites disrupt respective        amplified PCR primers containing BamHI restriction <br /> ORFs amino acid E468 (2A), K461 (4A), W315 (1D), N508                 sites. Corresponding BamHI fragments subsequently cloned <br /> (2D) Q412 (4D), respectively. For PMT1A existing EcoRV               BamHI restriction site vector pJAF1 containing Tn5 <br /> restriction site used generate disruption constructs. In         derived NeoR resistance marker [59]. <br /> this case ADE2 URA5 containing SmaI fragments used                The primers utilized plasmid construction presented in <br /> disrupting PMT1A ORF position D439. These constructs                 Table S1. <br /> were PCR amplified generate linear disruption allele and <br /> gel purified use biolistic transformation appropriate           Strain construction <br /> auxotroph strain [57]. Biolistic transformations conducted                To generate homozygous serotype D diploid ade2 strain, <br /> using established methods [58]. For reconstituting generated            strains JEC156 JEC157 crossed according Sia et al. <br /> disruption strains PMT ORFs plus additional 500&#226;&#8364;&#8220;700 bp up-              (2000) random spores isolated SD solid media <br /> and downstream respective START STOP codons                      supplemented 20 mg/ml adenine. Plates grown for <br />  <br />  <br />       PLoS ONE | www.plosone.org                                       11                              July 2009 | Volume 4 | Issue 7 | e6321 <br />                                                                                                                    C. neoformans PMT Gene Family <br />  <br />  <br />  <br />  Table 1. Strains used study. <br />  <br />  <br />  Serotype A strains                             Genotype                                                             Source/reference <br />  H99                                            wild-type MATa                                                       [67] <br />  F99                                            MATa ura5 (5-FOA resistent isolate H99)                           Ping Wang (Heitman-laboratory) <br />  SW13                                           MATa pmt1::URA5                                                      study <br />  SW14                                           MATa pmt4::URA5                                                      study <br />  SW15                                           MATa pmt1::URA5 PMT1-NeoR                                            study <br />                                                                             R <br />  SW16                                           MATa pmt4::URA5 PMT4-Neo                                             study <br />  Serotype D strain                              Genotype                                                             Source/reference <br />  JEC20                                          wild-type MATa                                                       [68] <br />  JEC21                                          wild-type MATa                                                       [68] <br />  JEC155                                         MATa ade2-27 ura5                                                    Jeff Edman <br />  JEC156                                         MATa ade2-27 ura5                                                    Jeff Edman <br />  JEC157                                         MATa ade2 ura5 lys1                                                  Jeff Edman <br />  JEC34                                          MATa ura5                                                            Jeff Edman <br />  JEC43                                          MATa ura5                                                            Jeff Edman <br />  JEC50                                          MATa ade2-27                                                         Jeff Edman <br />  SW1                                            MATa pmt1::ADE2                                                      study <br />  SW2                                            MATa pmt1::ADE2                                                      study <br />  SW4                                            MATa pmt1::URA5 ade2-27                                              study <br />  SW5                                            MATa pmt1::URA5                                                      study <br />  SW6                                            MATa pmt1::URA5                                                      study <br />  SW7                                            MATa pmt4::ADE2 ura5                                                 study <br />  SW8                                            MATa pmt4::ADE2                                                      study <br />  SW9                                            MATa pmt4::ADE2                                                      study <br />  SW10                                           MATa pmt4::URA5 ade2-27                                              study <br />  SW11                                           MATa pmt4::URA5                                                      study <br />  SW12                                           MATa pmt4::URA5                                                      study <br />  SW17                                           MATa pmt1::URA5 PMT1-NeoR                                            study <br />  SW18                                           MATa pmt4::URA5 PMT4-NeoR                                            study <br />  SW19                                           MATa pmt4::ADE2 ura5                                                 study <br />  <br />  doi:10.1371/journal.pone.0006321.t001 <br />  <br />  <br /> several days 37uC prevent filament formation usually seen             strand synthesis reverse transcription kit (Invitrogen). The resulting <br /> with diploid strains. In second step, reddish Ade2 strains           cDNA used template quantitative real-time PCR using <br /> isolated re-streaked plates, incubated RT          iQ SYBR Green Supermix (Bio-Rad) according manufac- <br /> to induce filament basidiospore formation typical diploid            turer&#226;&#8364;&#8482;s specifications. The iCycler iQ multicolor real-time <br /> Cryptococcus strains conditions. To generate                  detection used fluorescence detector the <br /> heterozygous pmt2/PMT2 serotype D strain, resulting diploid               following PCR conditions: initial denaturing cycle 95uC for <br /> ade2/ade2 diploid strain transformed ADE2 based                 3 min 40 cycles denaturation 95uC 10 s and <br /> pmt2D-disruption construct using biolistic transformation. Hetero-          annealing/extension 58uC 330 s. These cycles followed <br /> zygous mutants initially identified colony PCR                  standard melting curve 53uC 93uC fluorescent <br /> subsequently confirmed Southern analysis.                                monitoring 0.5uC. These data confirmed amplification of <br />                                                                             single product primer pair lack primer <br /> RNA preparation                                                             dimerization. Reactions performed triplicate, data <br />    Strains incubated mid-logarithmic phase 30uC               expressed average cycle threshold, 6 standard <br /> YPD medium. Cells collected centrifugation flash                deviation. Standard PCRs run fivefold dilutions the <br /> frozen dry ice. Total RNA extracted lyophilized cells           cDNA template determine optimal template and <br /> using TRIzol reagent (Invitrogen Life Technologies, Carlsbad,           optimal annealing temperature experimental reference <br /> CA).                                                                        reactions, using 500 nM primer. <br />                                                                                Gene amplification strain condition normalized <br /> Real-time PCR                                                               constitutively expressed GPD gene [60]. Induction (n- <br />   Total RNA isolated relevant strains described             fold) calculated relative wild-type strain JEC21 using the <br /> above. The RNA treated RNase-free DNase, cDNA                  Bio-Rad iCycler software system, utilizes comparative <br /> was synthesized using oligo(dT) primers SuperScript first-         cycle threshold statistical methods previously described [61] <br />  <br />  <br />          PLoS ONE | www.plosone.org                                    12                                July 2009 | Volume 4 | Issue 7 | e6321 <br />                                                                                                                   C. neoformans PMT Gene Family <br />  <br />  <br />  <br /> Mating analyses haploid filamentation                                   In vitro virulence assays <br />    To determine defects mating, various pmt mutant                  To determine capsule production, strains grown <br /> tester strains pre-grown YPD plates 30uC 2&#226;&#8364;&#8220;3 days,           night Dulbecco&#226;&#8364;&#8482;s modified Eagles medium 3 days at <br /> a small growth removed using sterile tooth pick             30uC [32]. Subsequently, 5 ml culture mixed an <br /> and spotted solid V8 plates (pH 5.0 7.0),           equal volume India ink, cells analyzed light <br /> mixed respective tester strain. Plates incubated             microscopy. Melanin production determined different <br /> room temperature (RT) dark 3 days mating            techniques. First, 10 ml over-night cultures spotted onto <br /> and haploid filamentation assessed light microscopy.                 standard Niger seed plates, plates incubated 30uC for <br />                                                                             2&#226;&#8364;&#8220;4 days controls showed brownish colour indicative of <br />                                                                             melanin production. Second, cells 25 ml YPD over-night <br /> Basidiospore isolation <br />                                                                             cultures harvested resuspended 2 ml glucose-free <br />    To isolate basidiospores genetic crosses, strains were <br />                                                                             asparagine-medium supplemented L-DOPA, cell sus- <br /> crossed solid V8 plates described the <br />                                                                             pensions incubated 24 h 30uC rotary shaker. <br /> mating analyses incubated RT dark until <br />                                                                             Subsequently cells pelleted, melanin production was <br /> basidiospore formation noticeable. Areas showing basidio- <br />                                                                             documented photographing cell pellet, determining the <br /> spores excised single basidiospores transferred to <br />                                                                             OD480 supernatants [64]. Urease activity tests wild-type <br /> fresh YPD plates using micromanipulator (single spore isolation).         pmt mutant strains performed following way. One <br /> Alternatively, excised filaments transferred 1.5 ml               colony strain added sterile deionized H2O a <br /> reaction tube containing 500 ml water, vortexed vigorously,            microcentrifuge tube vortexed vigorously. One BBL TAX- <br /> and spores subsequently plated fresh YPD plates                  OTM urease differentiation disk (Becton, Dickinson, &amp; Co., Sparks, <br /> respective selective media (random spore isolation). To isolate             MD) added tube using sterile techniques, then <br /> haploid Ade2+ strains heterozygote serotype D pmt2/                incubated 30uC checked 10 min 30 min for <br /> PMT2 strain random spore analysis spreading spores               urease activity according manufacturer&#226;&#8364;&#8482;s instructions. <br /> on SD plates containing 20 mg/ml adenine, plates were <br /> incubated 37uC prevent filamentation. Subsequently, only <br />                                                                             In vivo virulence analyses <br /> white colonies re-streaked SD plates were <br />                                                                                Overall virulence C. neoformans strains tested by <br /> incubated RT. Strains filament these <br />                                                                             macrophage killing assays murine inhalation model. <br /> conditions discarded filamentation indicate that <br />                                                                             For macrophage killing assay 50 ml freshly grown J77A.1 <br /> these strains diploid nature. <br />                                                                             macrophages (,16105 J774A.1 macrophages DMEM) were <br />                                                                             pipetted 96 microtiter plates activated adding <br /> Serial dilution patch tests                                                 50 ml DMEM supplemented INF-c (100 U/ml) LPS <br />    Standard serial dilution patch tests performed follows:          (0.6 mg/ml). Negative controls &#226;&#8364;&#732;&#226;&#8364;&#732;activated&#226;&#8364;&#8482;&#226;&#8364;&#8482; DMEM <br /> strains tested pre-grown night 5 ml liquid YPD           containing additional supplements. Microtiter plates then <br /> medium 30uC stationary phase. OD600 determined              incubated 12&#226;&#8364;&#8220;18 h 37uC 5% CO2 atmosphere to <br /> next day, cultures diluted OD600 1, tenfold serial        generate macrophage cell-layer. C. neoformans strains tested <br /> dilutions YPD dilution 1024. Finally,             grown night liquid YPD medium, cells were <br /> 5 ml various dilutions spotted respective solid        harvested washed times PBS. Cells were <br /> media incubated indicated temperatures.                          resuspended DMEM containing 1 mg/ml monoclonal antibody <br />                                                                             mAb18B7 cell titer 16106 living cells ml, determined by <br />                                                                             Trypan Blue staining, incubated 1 h 37uC 5% CO2. <br /> Microscopy                                                                  Subsequently, 100 ml C. neoformans cells pipetted the <br />    Light, DIC fluorescence microscopy pictures taken               macrophage cell layers, microtiter plates incubated at <br /> using Zeiss Axioskop 2 Plus Fluorescence Microscope mounted               37uC 5% CO2 hour. Non incorporated cryptococcal <br /> with AxioCam MRM digital camera, alternatively                  cells removed carefully washing macrophage cell <br /> Nikon Eclipse E400 microscope Nikon CoolPix 990 digital               layers times PBS. Finally cell layers covered with <br /> camera. Cell wall material genomic DNA stained using               150 ml fresh DMEM culture medium incubated another <br /> the Fluorescent Brightener 28 (calcofluor white; Sigma Aldrich)             24 h 37uC 5% CO2. To determine survival rates of <br /> or 4,6-diamidinO-2-phenylindole (DAPI; Molecular Probes).                   cryptococcal cells covering culture medium removed and <br /> First, C. neoformans strains grown OD600 ,1, cells            transferred fresh reaction tube. Macrophages titer plates <br /> were harvested fixed 30 min 10% formaldehyde.                    incubated 100 ml 0.5% SDS solution ,5 min RT, <br /> Subsequently, cells washed times PBS               cells lysed pipetting cells times, and <br /> permeabilized 1% Triton X-100 10 min. Finally, cells               cell lysate combined respective culture <br /> were washed times PBS stained                 supernatant. Finally, wells washed 200 ml PBS <br /> the indicated dyes. To visualize vacuoles, cells stained          solution, subsequently added respective cell <br /> the lipophilic dye N-(3-triethylammoniumpropyl)-4-(6-(4-                    lysate. Cell lysates diluted 300 fold, cryptococcal CFU <br /> (diethylamino) phenyl) hexatrienyl) pyridinium dibromide (FM                determined standard techniques solid YPD plates, <br /> 4&#226;&#8364;&#8220;64; T-3166; Invitrogen Corporation, Carlsbad, CA)                      incubated 30uC. Macrophage killing assays repeated at <br /> described previously [62,63]. Briefly, logarithmically growing              times [65]. The murine inhalation model systemic <br /> C. neoformans cells corresponding 1 ml OD600 ,0.1             Cryptococcosis performed described [66]. Briefly, <br /> harvested resuspended 100 ml YPD. Subsequently,                  groups female A/Jcr mice (,20 g body weight) were <br /> vacuolar stain FM4-64 added final concentration                 infected intra-nasally 105 cryptococcal CFU resuspended in <br /> 40 mM, cells incubated rotary shaker 45 min               PBS. To determine survival rates, mice inspected vitality <br /> at 30uC. Before microscopy cells washed times             twice day, individuals showing ,15% weight loss, <br /> YPD.                                                                        neurological abnormalities extreme anorexia sacrificed <br />  <br />  <br />       PLoS ONE | www.plosone.org                                       13                               July 2009 | Volume 4 | Issue 7 | e6321 <br />                                                                                                                                              C. neoformans PMT Gene Family <br />  <br />  <br /> immediately according animal protection regulations                            Found at: doi:10.1371/journal.pone.0006321.s001 (0.02 MB <br /> Duke University Medical Center Animal Care &amp; Use Program.                                    DOC) <br /> Mice maintained Research Institute Children <br /> Animal Facility accordance American Association                               Acknowledgments <br /> Accreditation Laboratory Animal Care guidelines. <br />                                                                                              We wish thank J. Heitman providing C. neoformans strains and <br />                                                                                              plasmids. A special thanks members Ernst/Lengeler, <br /> Statistical analysis                                                                         Alspaugh laboratories Robert Cramer technical assistance and <br />   The software program Prism 5 (GraphPad, San Diego, Calif.)                                 helpful discussion. <br /> was used statistical tests. Log-rank tests utilized to <br /> determine significance survival animal studies. <br />                                                                                              Author Contributions <br /> Supporting Information                                                                       Conceived designed experiments: SDW AA KL. Performed the <br />                                                                                              experiments: SDW. Analyzed data: SDW. Contributed reagents/ <br /> Table S1 Plasmids used study                                                         materials/analysis tools: JE AA KL. Wrote paper: SDW AA KL. <br />  <br /> References <br />  1. Tanner W, Lehle L (1987) Protein glycosylation yeast. Biochim Biophys Acta            21. Perfect JR, Casadevall A (2002) Cryptococcosis. Infect Dis Clin North Am 16: <br />     906: 81&#226;&#8364;&#8220;99.                                                                                  837&#226;&#8364;&#8220;874, v&#226;&#8364;&#8220;vi. <br />  2. Strahl-Bolsinger S, Gentzsch M, Tanner W (1999) Protein O-mannosylation.                 22. Olson GM, Fox DS, Wang P, Alspaugh JA, Buchanan KL (2007) Role of <br />     Biochim Biophys Acta 1426: 297&#226;&#8364;&#8220;307.                                                          protein O-mannosyltransferase Pmt4 morphogenesis virulence of <br />  3. Ernst JF, Prill SK (2001) O-glycosylation. Med Mycol 39 Suppl 1: 67&#226;&#8364;&#8220;74.                      Cryptococcus neoformans. Eukaryot Cell 6: 222&#226;&#8364;&#8220;234. <br />  4. Jurado LA, Coloma A, Cruces J (1999) Identification human homolog            23. Loftus BJ, Fung E, Roncaglia P, Rowley D, Amedeo P, et al. (2005) The genome <br />     Drosophila rotated abdomen gene (POMT1) encoding putative protein                          basidiomycetous yeast human pathogen Cryptococcus neoformans. <br />     O-mannosyl-transferase, assignment human chromosome 9q34.1.                           Science 307: 1321&#226;&#8364;&#8220;1324. <br />     Genomics 58: 171&#226;&#8364;&#8220;180.                                                                    24. Franzot SP, Fries BC, Cleare W, Casadevall A (1998) Genetic relationship <br />  5. Willer T, Amselgruber W, Deutzmann R, Strahl S (2002) Characterization                    Cryptococcus neoformans var. neoformans strains serotypes A D. J Clin <br />     POMT2, novel member PMT protein O-mannosyltransferase family                        Microbiol 36: 2200&#226;&#8364;&#8220;2204. <br />     specifically localized acrosome mammalian spermatids. Glycobiology             25. Tanaka R, Imanishi Y, Nishimura K (2003) Difference FKS1 gene sequences <br />     12: 771&#226;&#8364;&#8220;783.                                                                                 serotypes A D Cryptococcus neoformans. J Clin Microbiol 41: <br />  6. Lyalin D, Koles K, Roosendaal SD, Repnikova E, Van Wechel L, et al. (2006)                   4457&#226;&#8364;&#8220;4459. <br />     The twisted gene encodes Drosophila protein O-mannosyltransferase 2                  26. Sugita T, Ikeda R, Shinoda T (2001) Diversity strains Cryptococcus <br />     genetically interacts rotated abdomen gene encoding Drosophila                      neoformans var. gattii revealed sequence analysis multiple genes a <br />     protein O-mannosyltransferase 1. Genetics 172: 343&#226;&#8364;&#8220;353.                                      chemotype analysis capsular polysaccharide. Microbiol Immunol 45: <br />  7. VanderVen BC, Harder JD, Crick DC, Belisle JT (2005) Export-mediated                         757&#226;&#8364;&#8220;768. <br />     assembly mycobacterial glycoproteins parallels eukaryotic pathways. Science           27. Girrbach V, Zeller T, Priesmeier M, Strahl-Bolsinger S (2000) Structure- <br />     309: 941&#226;&#8364;&#8220;943.                                                                                function analysis dolichyl phosphate-mannose: protein O-mannosyltrans- <br />  8. Mahne M, Tauch A, Puhler A, Kalinowski J (2006) The Corynebacterium                          ferase ScPmt1p. J Biol Chem 275: 19288&#226;&#8364;&#8220;19296. <br />     glutamicum gene pmt encoding glycosyltransferase related eukaryotic protein-        28. Willer T, Brandl M, Sipiczki M, Strahl S (2005) Protein O-mannosylation is <br />     O-mannosyltransferases essential glycosylation resuscitation                   crucial cell wall integrity, septation viability fission yeast. Mol <br />     promoting factor (Rpf2) secreted proteins. FEMS Microbiol Lett                     Microbiol 57: 156&#226;&#8364;&#8220;170. <br />     259: 226&#226;&#8364;&#8220;233.                                                                            29. Sia RA, Lengeler KB, Heitman J (2000) Diploid strains pathogenic <br />  9. Strahl-Bolsinger S, Scheinost A (1999) Transmembrane topology pmt1p,                    basidiomycete Cryptococcus neoformans thermally dimorphic. Fungal Genet Biol <br />     member evolutionarily conserved family protein O-mannosyltrans-                     29: 153&#226;&#8364;&#8220;163. <br />     ferases. J Biol Chem 274: 9068&#226;&#8364;&#8220;9075.                                                     30. Girrbach V, Strahl S (2003) Members evolutionarily conserved PMT <br /> 10. Willer T, Valero MC, Tanner W, Cruces J, Strahl S (2003) O-mannosyl glycans:                 family protein O-mannosyltransferases form distinct protein complexes among <br />     yeast novel associations human disease. Curr Opin Struct Biol 13:               themselves. J Biol Chem 278: 12554&#226;&#8364;&#8220;12562. <br />     621&#226;&#8364;&#8220;630.                                                                                 31. Akasaka-Manya K, Manya H, Nakajima A, Kawakita M, Endo T (2006) <br /> 11. Gentzsch M, Tanner W (1997) Protein-O-glycosylation yeast: protein-specific               Physical functional association human protein O-mannosyltransferases 1 <br />     mannosyltransferases. Glycobiology 7: 481&#226;&#8364;&#8220;486.                                               2. J Biol Chem 281: 19339&#226;&#8364;&#8220;19345. <br /> 12. Bourdineaud JP, van der Vaart JM, Donzeau M, Sampaio G, Verrips CT, et                32. Granger DL, Perfect JR, Durack DT (1985) Virulence Cryptococcus neoformans: <br />     al. (1998) Pmt1 mannosyl transferase involved cell wall incorporation               regulation capsule synthesis carbon dioxide. J Clin Invest 76: 508&#226;&#8364;&#8220;516. <br />     proteins Saccharomyces cerevisiae. Mol Microbiol 27: 85&#226;&#8364;&#8220;98.                   33. Kwon-Chung KJ, Edman JC, Wickes BL (1992) Genetic association mating <br /> 13. Harty C, Strahl S, Romisch K (2001) O-mannosylation protects mutant alpha-                   types virulence Cryptococcus neoformans. Infect Immun 60: 602&#226;&#8364;&#8220;605. <br />     factor precursor endoplasmic reticulum-associated degradation. Mol Biol             34. Nielsen K, Cox GM, Wang P, Toffaletti DL, Perfect JR, et al. (2003) Sexual <br />     Cell 12: 1093&#226;&#8364;&#8220;1101.                                                                          cycle Cryptococcus neoformans var. grubii virulence congenic alpha <br /> 14. Proszynski TJ, Simons K, Bagnat M (2004) O-glycosylation sorting                        isolates. Infect Immun 71: 4831&#226;&#8364;&#8220;4841. <br />     determinant cell surface delivery yeast. Mol Biol Cell 15: 1533&#226;&#8364;&#8220;1543.             35. Shaw BD, Momany M (2002) Aspergillus nidulans polarity mutant swoA is <br /> 15. Prill SK, Klinkert B, Timpel C, Gale CA, Schroppel K, et al. (2005) PMT family               complemented protein O-mannosyltransferase pmtA. Fungal Genet Biol 37: <br />     Candida albicans: protein mannosyltransferase isoforms affect growth,                263&#226;&#8364;&#8220;270. <br />     morphogenesis antifungal resistance. Mol Microbiol 55: 546&#226;&#8364;&#8220;560.                      36. Gentzsch M, Tanner W (1996) The PMT gene family: protein O-glycosylation in <br /> 16. Lommel M, Bagnat M, Strahl S (2004) Aberrant processing WSC family                    Saccharomyces cerevisiae vital. Embo J 15: 5752&#226;&#8364;&#8220;5759. <br />     Mid2p cell surface sensors results cell death Saccharomyces cerevisiae         37. Rouabhia M, Schaller M, Corbucci C, Vecchiarelli A, Prill SK, et al. (2005) <br />     O-mannosylation mutants. Mol Cell Biol 24: 46&#226;&#8364;&#8220;57.                                            Virulence fungal pathogen Candida albicans requires isoforms of <br /> 17. Beltran-Valero Bernabe D, Currier S, Steinbrecher A, Celli J, van                         protein mannosyltransferases. Infect Immun 73: 4571&#226;&#8364;&#8220;4580. <br />     Beusekom E, et al. (2002) Mutations O-mannosyltransferase gene                    38. Gentzsch M, Immervoll T, Tanner W (1995) Protein O-glycosylation in <br />     POMT1 rise severe neuronal migration disorder Walker-Warburg                     Saccharomyces cerevisiae: protein O-mannosyltransferases Pmt1p Pmt2p <br />     syndrome. Am J Hum Genet 71: 1033&#226;&#8364;&#8220;1043.                                                      function heterodimer. FEBS Lett 377: 128&#226;&#8364;&#8220;130. <br /> 18. Willer T, Prados B, Falcon-Perez JM, Renner-Muller I, Przemeck GK, et al.                39. Rhodes JC, Polacheck I, Kwon-Chung KJ (1982) Phenoloxidase activity and <br />     (2004) Targeted disruption Walker-Warburg syndrome gene Pomt1                      virulence isogenic strains Cryptococcus neoformans. Infect Immun 36: <br />     mouse results embryonic lethality. Proc Natl Acad Sci U S A 101:                          1175&#226;&#8364;&#8220;1184. <br />     14126&#226;&#8364;&#8220;14131.                                                                             40. Kwon-Chung KJ, Polacheck I, Popkin TJ (1982) Melanin-lacking mutants of <br /> 19. Martin-Blanco E, Garcia-Bellido A (1996) Mutations rotated abdomen                    Cryptococcus neoformans virulence mice. J Bacteriol 150: 1414&#226;&#8364;&#8220;1421. <br />     locus affect muscle development reveal intrinsic asymmetry                     41. Chang YC, Kwon-Chung KJ (1994) Complementation capsule-deficient <br />     Drosophila. Proc Natl Acad Sci U S A 93: 6048&#226;&#8364;&#8220;6052.                                          mutation Cryptococcus neoformans restores virulence. Mol Cell Biol 14: <br /> 20. Ichimiya T, Manya H, Ohmae Y, Yoshida H, Takahashi K, et al. (2004) The                      4912&#226;&#8364;&#8220;4919. <br />     twisted abdomen phenotype Drosophila POMT1 POMT2 mutants                          42. Chen SC, Muller M, Zhou JZ, Wright LC, Sorrell TC (1997) Phospholipase <br />     coincides heterophilic protein O-mannosyltransferase activity. J Biol             activity Cryptococcus neoformans: new virulence factor? J Infect Dis 175: <br />     Chem 279: 42638&#226;&#8364;&#8220;42647.                                                                       414&#226;&#8364;&#8220;420. <br />  <br />  <br />  <br />         PLoS ONE | www.plosone.org                                                      14                                       July 2009 | Volume 4 | Issue 7 | e6321 <br />                                                                                                                                                  C. neoformans PMT Gene Family <br />  <br />  <br /> 43. Chen SC, Wright LC, Santangelo RT, Muller M, Moran VR, et al. (1997)                         56. Vartivarian SE, Anaissie EJ, Cowart RE, Sprigg HA, Tingler MJ, et al. (1993) <br />     Identification extracellular phospholipase B, lysophospholipase, acyl-                    Regulation cryptococcal capsular polysaccharide iron. J Infect Dis 167: <br />     transferase produced Cryptococcus neoformans. Infect Immun 65: 405&#226;&#8364;&#8220;411.                       186&#226;&#8364;&#8220;190. <br /> 44. Zhu X, Gibbons J, Garcia-Rivera J, Casadevall A, Williamson PR (2001)                        57. Davidson RC, Blankenship JR, Kraus PR, Jesus Berrios M, Hull CM, et al. <br />     Laccase Cryptococcus neoformans cell wall-associated virulence factor. Infect            (2002) A PCR-based strategy generate integrative targeting alleles large <br />     Immun 69: 5589&#226;&#8364;&#8220;5596.                                                                             regions homology. Microbiology 148: 2607&#226;&#8364;&#8220;261 <br /> </body></html>