<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>KREPA6 RNA-binding protein essential editosome <br /> integrity <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-6' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span>survival</span> Trypanosoma brucei <br />  <br />  <br /> SALVADOR ZIPAGAN TARUN JR.,1 ACHIM SCHNAUFER,1 NANCY LEWIS ERNST,1 ROSEMARY PROFF,1 <br /> JUNPENG DENG,2,4 WIM HOL,2 KENNETH STUART1,3 <br /> 1 <br />  Seattle Biomedical Research Institute, Seattle, Washington 98109, USA <br /> 2 <br />  Department Biochemistry, University Washington, Seattle, Washington 98195, USA <br /> 3 <br />  Department Pathobiology, University Washington, Seattle, Washington 98195, USA <br />  <br />  <br />  <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> <br /> Most mitochondrial mRNAs kinetoplastid protozoa require post-transcriptional RNA editing inserts deletes <br /> uridylates, process catalyzed multiprotein editosomes. KREPA6 smallest editosome proteins have <br /> predicted oligonucleotide-binding (OB) folds. Inactivation KREPA6 expression results disruption ultimate loss ;20S <br /> editosomes inhibition procyclic form cell growth. Gel shift studies recombinant KREPA6 binds RNA, not <br /> DNA, preference oligo-(U) 39 end gRNA (UU)12 homopolymer. Thus, KREPA6 essential for <br /> the structural integrity presence ;20S editosomes cell viability. It functions RNA binding primarily <br /> through gRNA 39 oligo(U) tail. The significance findings key steps editing discussed. <br /> Keywords: Trypanosoma brucei; trypanosomatids; RNA editing; editosome; OB-fold; RNA-binding protein <br />  <br />  <br />  <br />  <br /> INTRODUCTION                                                                         RNA editing carried 20S editosomes, likely <br />                                                                                   coordination multiprotein complexes. The <br /> The protozoan parasite Trypanosoma brucei responsible <br />                                                                                   editosomes contain catalysts cleave pre-mRNA, <br /> for African sleeping sickness humans nagana in <br />                                                                                   add delete Us, ligate processed products <br /> cattle. RNA editing T. brucei related kinetoplastids is <br />                                                                                   (Rusche et al. 1997; Madison-Antenucci et al. 1998; <br /> a unique, essential, extensive form post-transcrip- <br />                                                                                   Panigrahi et al. 2001a). They accurately edit single sites <br /> tional RNA processing uridylates (Us) deleted <br />                                                                                   model pre-RNAs vitro (Seiwert Stuart 1994; Kable <br /> and inserted majority mitochondrial pre- <br />                                                                                   et al. 1996; Igo Jr. et al. 2000, 2002). The current model <br /> mRNAs specified guide RNAs (gRNAs). The edited <br />                                                                                   RNA editing 20S editosomes envisions series of <br /> mRNAs encode protein components respiratory chain <br />                                                                                   dynamic interactions pre-mRNA cog- <br /> complexes I, III, IV, V, RPS12 MURF2, <br />                                                                                   nate gRNA RNAs RNA-binding <br /> a protein unknown function (for reviews, Madison- <br />                                                                                   proteins catalysts editosomes (Madison- <br /> Antenucci et al. 2002; Simpson et al. 2004; Stuart et al. <br />                                                                                   Antenucci et al. 2002; Stuart Panigrahi 2002; Stuart <br /> 2005). It functions stage-specific fashion resulting in <br />                                                                                   et al. 2005 references therein). Editing generally pro- <br /> differentially expressed proteins, accommodate <br />                                                                                   ceeds 39 59 direction commences gRNA <br /> the alternate energy requirements insect (pro- <br />                                                                                   selection step complementary <span id='am-160' about='xsp:base' typeof='owl:Thing'>base</span>-pairing the <br /> cyclic forms, PF) animal hosts (bloodstream forms, BF) <br />                                                                                   59 &#226;&#8364;&#732;&#226;&#8364;&#732;anchor&#226;&#8364;&#8482;&#226;&#8364;&#8482; region gRNA 8&#226;&#8364;&#8220;10-base pair <br /> (Benne et al. 1986; Feagin et al. 1987, 1988; Koslowsky et al. <br />                                                                                   (bp) region mRNA sequence adjacent block be <br /> 1990). <br />                                                                                   edited, followed specific endonucleolytic cleavage of <br />                                                                                   pre-mRNA unpaired nucleotide immediately up- <br />     4                                                                             stream anchor duplex. As guided base comple- <br />     Present address: Department Biochemistry Molecular Biology, <br /> Oklahoma State University, Stillwater, OK 74078, USA.                             mentarity central informational region the <br />    Reprint requests <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span>: Kenneth Stuart, Seattle Biomedical Research               gRNA, U removal addition 39 end pre- <br /> Institute, 307 Westlake Avenue North, Suite 500, Seattle, WA 98109,               mRNA 59 cleavage fragment catalyzed U- <br /> USA; e-mail: kenneth.stuart@sbri.org; fax: (206) 256-7229. <br />    Article published online ahead print. Article publication date   specific 39-exoribonuclease (exoUase) terminal <br /> http://www.rnajournal.org/cgi/doi/10.1261/rna.763308.                             uridylyl transferase (TUTase), respectively, followed by <br />  <br /> RNA (2008), 14:347&#226;&#8364;&#8220;358. Published Cold Spring Harbor Laboratory Press. Copyright &#195;&#8220; 2008 RNA Society.                                    347 <br />  Tarun et al. <br />  <br />  <br />  <br /> joining processed 59 fragment 39 mRNA              (Matthews Sunde 2002; Theobald et al. 2003; <br /> fragment RNA ligases. Multiple cycles catalytic        Bochkarev Bochkareva 2004). The OB-fold KREPA4 <br /> steps required complete editing region specified    resembles S1 ribosomal protein, others <br /> by single gRNA, mRNAs require sequential use         closely resemble Escherichia coli DNA <br /> multiple gRNAs. This complex process entail numer-            single-strand binding protein (SSB) (Schnaufer et al. 2003; <br /> ous protein&#226;&#8364;&#8220;RNA RNA&#226;&#8364;&#8220;RNA interactions, identifi-            Worthey et al. 2003; Salavati et al. 2006). KREPA1 and <br /> cation infancy (Sacharidou et al. 2006;   KREPA2 proteins interact KREL2 KREL1, re- <br /> Salavati et al. 2006; Yu Koslowsky 2006).                      spectively, and, case KREPA1, enhance catalysis <br />    Consistent expected complexity inter-           (Schnaufer et al. 2003). These proteins suggested to <br /> actions 20S editosomes RNA editing               provide OB-fold trans function substrate <br /> substrates, sequences 20S editosome proteins            binding (Schnaufer et al. 2003). KREPA3 binds duplex <br /> have predicted catalytic and/or RNA interaction motifs             RNA DNA zinc-dependent fashion (Brecht et al. <br /> (Schnaufer et al. 2003; Worthey et al. 2003; Stuart et al.         2005), KREPA4 specifically binds 39 oligo(U) tail <br /> 2005), experimentally confirmed.           gRNA (Salavati et al. 2006). Knockdown expression <br /> The kinetoplastid RNA editing endonuclease1 (KREN1)                of proteins inhibits cell growth and, except <br /> that cleaves deletion sites, KREN2 cleaves insertion          KREPA3, results partial disruption of <br /> sites, KREN3 editing endonuclease specifically        20S editosomes (Drozdz et al. 2002; Huang et al. 2002; <br /> cleaves COII insertion site RNase III, U1-like zinc       O&#226;&#8364;&#8482;Hearn et al. 2003; Brecht et al. 2005; Salavati et al. 2006). <br /> finger-like, double-stranded RNA-binding (dsRBM)               Thus, KREPA family members critical functions in <br /> motifs (Carnes et al. 2005, 2007; Trotter et al. 2005).            RNA editing associated RNA substrate binding and <br /> Similarly, U-deletion editing exoUases KREX1               20S complex integrity. <br /> KREX2 U-insertion editing TUTase KRET2                   We examined KREPA6, smallest (18 kDa) KREPA <br /> 39&#226;&#8364;&#8220;59 exonuclease poly(A) polymerase domains,                  family member present different 20S edito- <br /> respectively (Aphasizhev et al. 2003; Ernst et al. 2003; Kang      somes (Panigrahi et al. 2006), interacts several <br /> et al. 2005; N.L. Ernst, unpubl.). The editing RNA Ligases         editosome proteins (Schnaufer et al. 2003; A. <br /> KREL1 KREL2 motifs characteristic           Schnaufer, unpubl.) loss affects complex <br /> activities (McManus et al. 2001; Schnaufer et al. 2001;            structure U-deletion U-insertion endonuclease <br /> Rusche&#194;&#180; et al. 2001). All members KREPA family          activities vitro (Law et al. 2007). Thus, a <br /> of 20S editosome proteins oligonucleotide-binding             critical member 20S editosomes, specific roles RNA <br /> (OB) fold motifs (Schnaufer et al. 2003; Panigrahi et al.          editing unknown. We confirm KREPA6 is <br /> 2006; Salavati et al. 2006), KREPB4&#226;&#8364;&#8220;8 family mem-          critical normal growth PF Trypanosome brucei and <br /> bers U1 zinc finger-like motifs (Worthey et al. 2003).        structural integrity 20S editosomes. Importantly, we <br /> The 20S editosomes common set proteins          loss KREPA6 strongly affects RNA <br /> physically functionally arranged subcomplexes          editing vivo protein binds RNA not <br /> can catalyze U removal ligation steps deletion          DNA vitro, displaying high cooperativity, similar <br /> editing U addition ligation steps insertion          affinity pre-mRNA gRNA, weaker affinity <br /> editing (Schnaufer et al. 2003). However, KREN1, KREN2,        partially duplexed form. Also, KREPA6 preferen- <br /> and KREN3 endonucleases different 20S editosomes            tially binds gRNA 39 oligo(U) tail. Thus, KREPA6 is <br /> along common set proteins           RNA-binding component 20S editosome is <br /> or specific proteins (Panigrahi et al. 2006). Thus,            essential structural integrity. <br /> endonuclease activity appears associated different 20S <br /> editosomes, catalyze subsequent U <br />                                                                    RESULTS <br /> deletion U insertion ligation steps editing. <br />    While components 20S editosomes account for <br />                                                                    KREPA6 important growth vivo <br /> the key catalytic steps editing identified, <br />                                                                    RNA editing <br /> the functions editosomal proteins not <br /> been determined include RNA-binding pro-              Conditional repression KREPA6 expression RNAi in <br /> tein interaction. Some KREPA family members             PF T. brucei inhibited cell growth accumulation of <br /> have shown function RNA-binding and/or                  edited RNAs (Fig. 1). Induction dsRNA spans the <br /> protein interaction (Schnaufer et al. 2003; Brecht et al.          entire KREPA6 coding sequence, using opposing <br /> 2005; Salavati et al. 2006). In addition aforemen-          promoter Wang et al. (2000), resulted a <br /> tioned OB-fold motifs, KREPA1, KREPA2, KREPA3                  40% knockdown KREPA6 mRNA 4 d compared <br /> also contain zinc-finger motifs (Panigrahi et al. 2001b).      uninduced cells (Fig. 1B). Growth inhibition became <br /> As shown organisms, OB-fold zinc-finger               apparent 4 d, cells exhibiting reduced motility, <br /> domains function nucleic acid protein binding                persisted 5 d (Fig. 1A). During period, <br />  <br /> 348      RNA, Vol. 14, No. 2 <br />                                                                                            Essential RNA-binding protein RNA editing <br />  <br />  <br />  <br />                                                                            loss KREPA6 repression observed (for <br />                                                                            example, O&#226;&#8364;&#8482;Hearn et al. 2003; Luo et al. 2004). <br />                                                                            Quantitative real-time RT-PCR (qPCR) analysis indicated <br />                                                                            level KREPA6 mRNA reduced substantially <br />                                                                            (Fig. 1B, light gray bar), levels KREPA1&#226;&#8364;&#8220;5 <br />                                                                            mRNAs, encode related editosome proteins, were <br />                                                                            unchanged slightly increased (Fig. 1B, gray bars), indi- <br />                                                                            cating knockdown specific KREPA6 mRNA. <br />                                                                            Western analysis fractions glycerol gradient purified <br />                                                                            total cell lysates KREPA6 RNAi cells anti- <br />                                                                            KREPA6 polyclonal antibodies revealed KREPA6 pro- <br />                                                                            tein dramatically reduced 2 d RNAi knock- <br />                                                                            essentially undetectable 4 d, level <br />                                                                            control mitochondrial protein, 2-OGDCp, compo- <br />                                                                            nent 2-oxoglutarate dehydrogenase complex, was <br />                                                                            essentially unaffected (Fig. 2). The apparent greater re- <br />                                                                            duction KREPA6 protein compared mRNA may <br />                                                                            reflect use qPCR primers KREPA6 39 UTR lying <br />                                                                            outside region targeted RNAi, effect dsRNA <br />                                                                            expression translation KREPA6 mRNA rather <br />                                                                            stability. Overall, RNAi knockdown dramati- <br />                                                                            cally reduced cellular level KREPA6 protein and <br />                                                                            proliferation PF T. brucei cells. <br />                                                                               KREPA6 repression resulted 90% reduction edited <br />                                                                            A6 RPS12 mRNAs 60% reduction edited COII <br />                                                                            MURF2 mRNAs 4 d RNAi induction deter- <br />                                                                            mined qPCR (Fig. 1B, white bars). In contrast, relative <br />                                                                            levels corresponding pre-edited mRNAs (Fig. 1B, black <br />                                                                            bars) increased. In addition, levels ND4 COI <br />                                                                            mitochondrial mRNAs (Fig. 1B, slashed bars) not <br />                                                                            undergo editing essentially unaffected KREPA6 <br />                                                                            knockdown, similar nuclearly encoded KREPA1&#226;&#8364;&#8220;5 <br />                                                                            mRNAs. Hence, KREPA6 repression resulted loss of <br />                                                                            KREPA6 mRNA protein, RNA editing vivo. <br /> FIGURE 1. KREPA6 repression RNAi inhibits cell growth and <br /> reduces RNA editing PF cells. (A) Growth RNAi-A6 cells the <br /> absence (uninduced, open squares) presence tetracycline (tet)        KREPA6 essential integrity <br /> (induced, solid squares) monitored 12 d. For comparison, <br /> growth uninduced (open triangles) induced (solid triangles)         20S editosomes <br /> 29.13 parental strain cells determined. The cumulative cell <br />                                                                            Repression KREPA6 expression RNAi caused loss <br /> number calculated multiplying cell densities dilution <br /> factor. (B) Quantitative real-time PCR analysis total RNA          20S editosomes (Fig. 2A&#226;&#8364;&#8220;C, upper panels). RNAi-A6 PF <br /> RNAi-A6 cells KREPA6 expressed repressed        cells induced 2 4 d, cell lysates were <br /> 4 d. The relative abundance RNA comparison           fractionated glycerol gradients examined West- <br /> KREPA6 expressed, line 1 indicating change, and <br /> bars line indicating increase decrease, <br />                                                                            ern analysis using monoclonal antibodies specific the <br /> respectively. The levels 18S rRNA used normalization. The      KREPA1, KREPA2, KREPA3, KREL1 editosome com- <br /> error bars indicate standard deviation replicates. Relative   ponents. Uninduced cells analyzed parallel. Each <br /> mRNA levels (i) KREPA editosome proteins (KREPA6, light             editosome proteins dramatically reduced with <br /> gray bar; KREPA1 KREPA5, dark gray bars), (ii) selected mito- <br /> chondrial pre-edited (black bars) edited (white bars) mRNAs,       virtually detectable signal remaining 20S region <br /> never-edited mRNAs (slashed bars).                                         (fractions 11&#226;&#8364;&#8220;15) 4 d KREPA6 repression, closely <br />                                                                            paralleling loss KREPA6 protein (Fig. 2C). Approx- <br />                                                                            imately 90% strongest Western signal, corresponding <br /> most cells reduced motility, formation cell                     KREPA1 20S region, eliminated day 2 <br /> debris observed, days 6&#226;&#8364;&#8220;8, suggesting cell                   RNAi (Fig. 2B). A small Western signal was <br /> death (data shown). Growth resumed z9 d                      detected fractions 5&#226;&#8364;&#8220;9, especially KREPA1, possibly <br /> was accompanied reappearance KREPA6 protein                      indicating slight accumulation retention editosome <br /> based Western analysis (data shown), suggesting                     subcomplexes seen previously knockdown <br />  <br />                                                                                                             www.rnajournal.org      349 <br />  Tarun et al. <br />  <br />  <br />  <br />  <br /> FIGURE 2. KREPA6 repression leads loss 20S editosomes. RNAi-A6 cell lines grown (A) absence presence tet (B) 2 d or <br /> (C) 4 d. Equal numbers cells lysed fractionated glycerol gradients. Odd-numbered fractions analyzed Western blot with <br /> a mixture monoclonal antibodies specific editosome proteins KREPA1, KREPA2, KREL1, KREPA3, polyclonal antibody <br /> to KREPA6. Blots probed monoclonal antibody recognizing component (2-OGDCp) mitochondrial 2-oxoglutarate <br /> dehydrogenase complex control. <br />  <br />  <br />  <br />  <br /> of mRNAs editosome components (Huang et al.                     function substrate RNA binding during <br /> 2002; O&#226;&#8364;&#8482;Hearn et al. 2003; Wang et al. 2003; Salavati et al.              editing, RNA-binding ability assessed electro- <br /> 2006; Babbarwal et al. 2007). This suggests KREPA6                phoretic mobility-shift assay (EMSA). Recombinant His- <br /> critical overall editosome integrity assembly              tagged KREPA6 expressed E. coli (HisKREPA6) <br /> since KREPA1, KREPA2, KREPA3 mRNA levels,                         purified near homogeneity (Fig. 4B) incubated <br /> and presumably translation mitochondrial                        59 end-labeled synthetic RNAs comparable lengths <br /> import, remained virtually unchanged day 4 RNAi                     representing pre-edited ATPase subunit 6 mRNA (A6 pre- <br /> induction (see Fig. 1B). The absence proteins                    mRNA), cognate gRNA gA6[14] (gRNA), gA6[14] <br /> probably reflects degradation result perturbed              minus oligo(U) tail (gRNADU-tail), annealed pre- <br /> 20S editosome structural integrity assembly.                           mRNA gRNA (pre-mRNATgRNA), pBluescript <br />    Repression KREPA6 resulted loss vitro                 RNA (pBS), resolved native gels, described in <br /> RNA editing activities (Fig. 3). Glycerol gradient fractions              Materials Methods. HisKREPA6 bound RNAs in <br /> of crude mitochondrial lysates RNAi-A6 cells                  protein concentration-dependent manner, resulting in <br /> KREPA6 expressed (uninduced) repressed 2                   shifted bands (Fig. 4C) accumulation radio- <br /> 4 d analyzed pre-cleaved insertion deletion                   active signal wells gel. The majority the <br /> editing assays. These assays bypass step editing,            shifted bands occurred higher concentrations of <br /> the endonucleolytic cleavage mRNA, ef-                    HisKREPA6, did material did leave well, <br /> ficiently measure subsequent TUTase ligase activities             especially case pre-mRNA pBS. The bands <br /> associated insertion editing (Fig. 3A), exoUase              particularly distinct gRNAs pre-mRNA, <br /> ligase activities associated deletion editing (Fig. 3B),             suggesting relatively stable well-defined protein&#226;&#8364;&#8220;RNA <br /> respectively. KREPA6 repression resulted loss TUTase,               complexes. This partial duplex pre- <br /> exoUase, ligase activities vitro 20S region          mRNATgRNA pBS. Essentially RNA was <br /> gradient, consistent substantial loss 20S edito-              bound z7 mM HisKREP6, pBS RNA, was <br /> some proteins (Fig. 3, fractions 13&#226;&#8364;&#8220;17). Residual TUTase,                 completely bound z20 mM HisKREPA6. <br /> exoUase, ligase activities detected fractions,             The complex partially overlapping <span id='am-161' about='xsp:pattern' typeof='owl:Thing'>pattern</span> gel- <br /> and small increase activities observed                  shifted bands did permit rigorous determination of <br /> fractions 7&#226;&#8364;&#8220;9 RNAi induction, suggesting                  apparent equilibrium dissociation constants (KD app) for <br /> complete editosomes remained functional sub-                     individual HisKREPA6/RNA complex. Thus, iso- <br /> complexes generated KREPA6 loss, consistent                     therm overall binding HisKREPA6 each <br /> with results Western analysis (see Fig. 2). Taken            RNA, referring retarded bands including the <br /> together, data demonstrate KREPA6 essential             wells gel, approximated best statistical <br /> the structural functional integrity 20S editosomes.                fit binding curve various equilibrium binding <br />                                                                           models (Fig. 4D). Each resulting best fit conformed the <br />                                                                           Hill equation coefficients greater 1, indicating <br /> KREPA6 binds pre-edited mRNA gRNA <br />                                                                           cooperative binding HisKREPA6 RNA (Fig. 4D, <br /> cooperatively similar affinities vitro <br />                                                                           inset), estimated overall KD app low <br /> Since presence OB-fold motif KREPA6 (Fig.                    micromolar range, indicating moderate affinity of <br /> 4A; Schnaufer et al. 2003; Worthey et al. 2003) suggested                 HisKREPA6 RNA tested. F-test analysis the <br />  <br /> 350      RNA, Vol. 14, No. 2 <br />                                                                                           Essential RNA-binding protein RNA editing <br />  <br />  <br />  <br />                                                                            KD app values RNA species underestimated <br />                                                                            likely &lt;100% recombinant protein in <br />                                                                            preparation active. Determination KD app <br />                                                                            HisKREPA6/duplex pre-mRNATgRNA complex <br />                                                                            (Fig. 4C, panel 5) performed accurately due <br />                                                                            overlap complex HisKREPA6/gRNA <br />                                                                            (Fig. 4C, lane 2, bound gRNA) duplexed pre- <br />                                                                            RNATgRNA protein (Fig. 4C, lane 3, free duplex), <br />                                                                            included here. <br />  <br />  <br />                                                                            Recombinant KREPA6 protein preferentially binds <br />                                                                            gRNA 39 oligo(U) tail <br />                                                                            Gel-shift competition assays revealed general correlation <br />                                                                            affinity HisKREPA6 gRNA based <br />                                                                            overall KD app measured (see Fig. 4D) and <br />                                                                            binding specificities associated 39 oligo(U) tail <br />                                                                            (Fig. 5A,B). A 500&#226;&#8364;&#8220;1000-fold molar excess unlabeled <br />                                                                            gRNA essentially abolished gel-shifted radiolabeled <br />                                                                            gRNA band similar excesses unlabeled gRNADU- <br />                                                                            tail pBS resulted reduction &lt;15% (Fig. 5A,B). <br />                                                                            Incubation pre-mRNA resulted additional bands <br />                                                                            include partial gRNA:mRNA duplex (see Fig. 4E) <br />                                                                            and, presumably, additional RNA/protein complexes, based <br />                                                                            sensitivity higher-shifted bands protease <br /> FIGURE 3. KREPA6 repression results substantial reduction          digestion (data shown). The observed competition <br /> RNA editing activities vitro. Crude mitochondrial extracts           consistent higher KD app values HisKREPA6 <br /> RNAi-A6 cells induced tet indicated number days            gRNA-DU tail pBS relative gRNA (see Fig. 4C, <br /> were fractionated glycerol gradients. Odd-numbered gradient <br />                                                                            panel). Thus, oligo(U) tail important <br /> fractions assayed (A) pre-cleaved insertion (B) pre- <br /> cleaved deletion editing activities vitro. The radiolabeled input       element binding HisKREPA6 gRNA, either <br /> RNAs, input RNA Us added (+2U), Us removed                acting direct binding target and/or stabilizing a <br /> ( 4U), ligated products unprocessed 59 39 input RNAs                structure favorable binding. This paralleled by <br /> (ligated), edited products indicated. Positive control reactions <br /> (+) performed using fraction z20S editosomes contains <br />                                                                            complementary gel-shift competition assays homo- <br /> peak editing activity. The region corresponding 20S editosomes       polymeric poly(A), poly(G), poly(C), poly(U) RNAs of <br /> indicated, based Western blot analyses gradients            heterogeneous lengths (10&#226;&#8364;&#8220;70 nucleotides [nt]) (Fig. <br /> antibodies specific known editosome proteins Figure 2 (data      5C,D). Poly(U) effective <br /> not shown). <br />                                                                            competitor HisKREPA6 binding gA6[14] gRNA, <br />                                                                            essentially eliminating gel shift 1000-fold molar <br />                                                                            excess, homopolymers reduced by <br /> overall KD app values Hill plots indicated                z40% (Fig. 5C,D). <br /> differences RNA species statistically <br /> significant (P-values &lt; 0.0001) (data shown). A6 pre- <br />                                                                            Recombinant KREPA6 protein poly(U) <br /> mRNA, gRNA, gRNADU-tail RNAs signifi- <br />                                                                            RNA-binding protein <br /> cantly higher Hill coefficients lower KD app values for <br /> HisKREPA6 binding compared pBS RNA, indicating                          HisKREPA6 preference oligo(U) gel-shift <br /> both higher cooperativity affinity HisKREPA6                        assays using synthetic 24-mer (UU)12, (UG)12, (UC)12, <br /> binding editing-related RNAs compared                       (AG)12 RNAs (Fig. 6). When limiting concentrations <br /> pBS. The complex gel-shift patterns binding models                     (0.5&#226;&#8364;&#8220;2 nM) 59-end-labeled RNA incubated room <br /> together suggest multiple HisKREPA6                  temperature increasing excess amounts purified <br /> editing RNA-binding interactions influence other,                HisKREPA6 (30 nM&#226;&#8364;&#8220;15 mM), (UU)12 bound by <br /> or concentration dependent HisKREPA6 oligomerization                       HisKREPA6 high affinity (Fig. 6A,B). HisKREPA6 <br /> (which influenced RNA binding), both.                       bound 50% (UU)12 z200 nM protein, <br /> Absence oligo(U) tail (i.e., gRNADU tail) signifi-                  indicating KD app value. Synthetic (AG)12 <br /> cantly reduced cooperativity affinity, indicating                 (UG)12 resulted gel-shifted bands higher <br /> that tail contributes binding energy. The overall              HisKREPA6 concentrations highest <br />  <br />                                                                                                            www.rnajournal.org    351 <br />  Tarun et al. <br />  <br />  <br />  <br />  <br /> FIGURE 4. Recombinant KREPA6 binds gRNA pre-mRNA cooperatively similar affinities. (A) Schematic representation native <br /> pre-processed KREPA6 recombinant TEV cleavable N-terminally histidine-tagged version, HisKREPA6, used study, highlighting <br /> the single central OB-fold domain. The mitochondrial import signal (MIS) KREPA6 replaced hexa-histidine tag. (B) Coomassie- <br /> stained SDS-PAGE gel affinity-purified HisKREPA6 protein used EMSA. Due affinity tag, migrates z22 kDa. (C) EMSA gel <br /> analysis HisKREPA6 binding (1) ATPase 6 pre-mRNA (pre-mRNA), (2) cognate gRNA, gA6[14] 39-oligo(U) tail (gRNA), (3) <br /> gA6[14] 39-oligo(U) tail (gRNADU-tail), (4) control pBluescript RNA, (5) partial duplexes A6 pre-mRNA gA6[14] (pre- <br /> mRNATgRNA). [32P]- radiolabeled RNA incubated various concentrations HisKREPA6 resolved native gels. The unbound <br /> radiolabeled RNA (free) HisKREPA6 bound RNA complexes (bound) indicated. Note range HisKREPA6 concentrations used in <br /> partial duplex RNA binding (panel 5) different (0&#226;&#8364;&#8220;12 mM) panels (0&#226;&#8364;&#8220;20 mM), additional concentrations (asterisks). <br /> (D) HisKREPA6/RNA-binding curves indicating relative RNA bound different HisKREPA6 concentrations. In case, the <br /> amount RNA bound includes shifted bands gel including signals wells (C) average replicates. <br /> Binding data fit described Materials Methods. Quantitation HisKREPA6 bound duplex pre-mRNATgRNA <br /> could accurately overlap gel shifts HisKREPA6 bound gRNA pre-mRNATgRNA. <br />  <br />  <br />  <br /> concentration HisKREPA6 (z15 mM) 50%                      DISCUSSION <br /> the input RNA shifted. Essentially (UC)12 RNA was <br /> bound HisKREPA6. The RNA degradation observed                       We conclude results reported KREPA6 <br /> high concentrations purified HisKREPA6 likely                critical role structural integrity z20S <br /> residual nucleases preparation. HisKREPA6                     editosomes appears function RNA <br /> did shift 24-mer deoxyribonucleotide-analogs,                     binding. Repression RNAi KREPA6 expression resulted <br /> d(UU)12 d(AG)12, significant extent (data                  essentially complete loss z20S editosomes little <br /> shown), indicating specificity HisKREPA6 RNA.                  accumulation free editosome proteins subcomplexes. <br />  <br /> 352      RNA, Vol. 14, No. 2 <br />                                                                                           Essential RNA-binding protein RNA editing <br />  <br />  <br />  <br />  <br /> FIGURE 5. Recombinant KREPA6 preferentially binds poly(U) gRNA oligo(U) tail. (A) EMSA gel analysis HisKREPA6 binding <br /> to radiolabeled gA6[14] gRNA oligo(U) tail absence presence excess unlabeled competitor RNAs. (Lanes 8,9) The multiple <br /> bands observed presence excess unlabeled pre-mRNA partially duplexed gRNATpre-mRNA and/or HisKREPA6 binding to <br /> the duplex RNA. (B) Quantitation gRNA binding A, indicating percent gRNA bound presence competitor <br /> RNAs relative absence. (A, lane 2) The HisKREPA6 bound gRNA absence competitor set 100%. The <br /> competitor RNAs gRNA U tail (black bars), gRNADU tail (striped bars), pre-mRNA (gray bars), pBlueScript control RNA (white <br /> bars). The apparent increase HisKREPA6 gRNA binding presence excess pre-mRNA &gt;100% formation of <br /> partial gRNATpre-mRNA duplexes. (C) EMSA analysis HisKREPA6 binding radiolabeled gRNA U tail absence presence of <br /> excess unlabeled homopolymeric competitor RNAs. Molar excess based nucleotide gram weight estimate heterogeneous <br /> polymer sizes (10&#226;&#8364;&#8220;70 nt) RNAs. (D) Quantitation gRNA binding C, indicating percent gRNA bound the <br /> presence competitor RNAs relative absence. (C, lane 2) As B, HisKREPA6 bound gRNA absence of <br /> competitor set 100%. The competitors poly(U) (black bars), poly(G) (striped bars), poly(A) (gray bars), poly(C) (white bars). <br />  <br />  <br />  <br /> Not surprisingly, resulted loss editing          editosome structural integrity. A recent study reported a <br /> the viability PF T. brucei, reliant process.        similar finding (Law et al. 2007). The concomitant loss of <br /> Recombinant KREPA6 gel retardation studies                  editosome proteins indicates integration of <br /> to moderate affinity RNA general greater             proteins editosome essential their <br /> affinity oligo(U) affinity DNA. These results             stability limiting amounts KREPA6 appears to <br /> indicate KREPA6 integral component z20S                    compromise assembly and/or stability 20S <br /> editosomes suggest functions RNA binding                  editosomes, leading rapid turnover editosome proteins. <br /> that entail binding 39 oligo(U) tail gRNAs.                These structural effects appear severe relative similar <br />                                                                          inactivation expression editosome proteins KREPA1 or <br />                                                                          KREPA3 RNAi PF T. brucei, caused slight <br /> Effects KREPA6 repression 20S editosome <br />                                                                          changes editosome sedimentation (Drozdz et al. 2002; <br /> structure cell growth <br />                                                                          O&#226;&#8364;&#8482;Hearn et al. 2003; Brecht et al. 2005). However, results <br /> The extensive loss 20S editosomes knockdown                   reported similar consequences inactivation <br /> KREPA6 expression (Fig. 2) indicates essential             expression KREPA2, KREPA4, KREPB4 PFs <br />  <br />                                                                                                               www.rnajournal.org        353 <br />  Tarun et al. <br />  <br />  <br />  <br />                                                                                                 editosome structural stability, its <br />                                                                                                 ability bind RNA apparent <br />                                                                                                 preference oligo(U) RNA. This <br />                                                                                                 RNA-binding ability consistent <br />                                                                                                 predicted OB-fold structure <br />                                                                                                 (Schnaufer et al. 2003), domain asso- <br />                                                                                                 ciated nucleic acid recognition/ <br />                                                                                                 binding various proteins (Theobald <br />                                                                                                 et al. 2003; Bochkarev Bochkareva <br />                                                                                                 2004). The KREPA family members <br />                                                                                                 predicted OB-fold domains <br />                                                                                                 near C termini, previously <br />                                                                                                 suggested KREPA1 KREPA2 <br />                                                                                                 provide substrate recognition/ <br />                                                                                                 binding function trans cata- <br />                                                                                                 lytic proteins directly <br />                                                                                                 interact editosome (Schnaufer <br />                                                                                                 et al. 2003). RNA binding also <br />                                                                                                 expected function associat- <br />                                                                                                 ing pre-mRNA gRNA the <br />                                                                                                 editosome relocating RNAs <br />                                                                                                 multiple coordinated steps <br />                                                                                                 editing. <br />                                                                                                    KREPA6 exhibited similarly com- <br />                                                                                                 plex pattern binding various syn- <br /> FIGURE 6. Recombinant KREPA6 higher affinity poly(U) RNA. (A) EMSA analysis <br /> of HisKREPA6 binding synthetic 24-mer RNAs. (B) Quantitation RNA binding A thetic RNAs, generating multiple but <br /> indicating percent RNA bound increasing HisKREPA6 concentrations. The RNA discrete, concentration-dependent gel- <br /> ligands 59-end-labeled (UC)12 (white bars), (AG)12 (striped bars), (UG)12 (gray bars), shifted products, including material <br /> (UU)12 (black bars). Owing apparent RNA degradation lanes containing high <br />                                                                                                 did enter gel (Fig. 4C). <br /> concentrations HisKREPA6, radioactive free RNA absence of <br /> HisKREPA6 (lane 11 gel) used estimate percent binding detailed Hence, KREPA6 mul- <br /> Materials Methods. Also RNA degradation, apparent KD estimations tiple alternate binding sites the <br /> not accurately determined.                                                                   RNAs, bind multiple RNA ligands <br />                                                                                                 simultaneously, oligomeric con- <br />                                                                                                 formations influence binding. The <br /> (Huang et al. 2002; Salavati et al. 2006; Babbarwal et al. 2007)           strong positive cooperativity KREPA6 binding RNA <br /> and KREPB5 (TbMP44) BFs (Wang et al. 2003). KREPA6                      (Fig. 4D) consistent possibilities. Understand- <br /> has shown directly interact KREPA1                     ing functional significance behavior KREPA6 <br /> partially associate heterotrimeric insertion                  require determining stoichiometry KREPA6 <br /> deletion subcomplexes editosomes (Schnaufer et al. 2003)                editosome. <br /> as directly interact various editosome                  Remarkably, KREPA6 binds RNAs <br /> proteins (A. Schnaufer, unpubl.). Therefore, KREPA6 appears                tested similar, moderate affinities (range overall <br /> to central role establishing and/or maintaining                  KD app&#226;&#8364;&#8482;s = 1.3&#226;&#8364;&#8220;5.1 mM), preference KREPA6 for <br /> key protein&#226;&#8364;&#8220;protein interactions 20S editosomes,                 oligo(U) binding clearly evident competition <br /> hypothesized previously (Stuart et al. 2005). KREPA4,                      direct binding studies (Figs. 5, 6). Therefore, the <br /> KREPB4, KREPB5 similarly critical importance,                   kinetics substrate binding, addressed in <br /> while KREPA1 KREPA2 appear central                       studies, significantly contribute relative <br /> overall editosome stability, given formation stable                 specificity observed. The preference KREPA6 for <br /> subcomplexes inactivation. Nevertheless               oligo(U), recombinant protein vitro, implies <br /> noncatalytic proteins, editosome,                interaction gRNA 39 oligo(U) tail, although <br /> are critical 20S editosome assembly and/or stability.                  interaction stretches U deleted are <br />                                                                            inserted specific sites editing be <br />                                                                            excluded. For example, presence oligo(U) tail <br /> RNA binding KREPA6 <br />                                                                            result stable binding KREPA6 gRNA <br /> Another important characteristic KREPA6 uncovered                    (i.e., result reduced OFF rate constant, kOFF). This <br /> this study, addition role protein interaction             interaction significantly contribute binding of <br />  <br /> 354      RNA, Vol. 14, No. 2 <br />                                                                                Essential RNA-binding protein RNA editing <br />  <br />  <br />  <br /> gRNA editosome complex, analogous           MATERIALS AND METHODS <br /> the kinetic contribution mRNA 39-poly(A) tail to <br /> ribosome binding translation eukaryotes     Plasmids <br /> poly(A)-binding protein (Tarun Jr. Sachs 1995; Sachs <br />                                                               An RNAi construct inducible knockdown KREPA6 expres- <br /> and Buratowski 1997). Detailed kinetic studies be <br />                                                               sion prepared pZJM vector, harbors opposing <br /> necessary uncover underlying mechanism         tetracycline (tet)-inducible T7 promoters flanking cloning site <br /> observed specificity.                                         (Wang et al. 2000; Morris et al. 2001). A 563-bp XhoI/HindIII <br />    Despite preference KREPA6 protein           fragment encompassing entire KREPA6 open reading frame <br /> oligo(U), moderate affinity binding RNA, essentially   plus 48 17 nt 59 39 UTR, respectively, generated <br /> regardless sequence, reflect general RNA affinity    PCR amplification PF T. brucei genomic DNA (strain 29.13) <br /> that important process RNA editing. Hence,       using 59-GACCTCGAGAAGGGGGTTCGTATTCTGTT-39 the <br /> KREPA6 utilize general RNA-binding prop-         forward primer 59-GCCAAGCTTGCGGAAACACAAGGGC <br /> erties higher affinity relative preference    ATTA-39 reverse primer contain relevant restriction <br /> oligo(U) distinct functionally related purposes.      sites (underlined italics). The PCR product initially cloned <br /> The characteristic provide low-affinity        pGEM-T Easy (Promega) released digestion with <br />                                                               HindIII XhoI subcloned pZJM digested <br /> interaction platform gRNA and/or pre-mRNA during <br />                                                               enzymes, generating RNAi plasmid pZJMA6&#226;&#8364;&#8220; <br /> editing. On hand, inconceivable    1. The HisKREPA6 expression plasmid pProEX-HisKREPA6&#226;&#8364;&#8220;1 <br /> each 1200 different gRNAs uniquely       kind gift Junpeng Deng Wim Hol (J. Deng W. <br /> recognized 20S editosome, preferential recognition     Hol, unpubl.). It expresses TEV-cleavable N-terminally His- <br /> of common element gRNA oligo(U) tail        tagged KREPA6 mitochondrial targeting signal a <br /> KREPA6 facilitate positioning anchor duplex      pProEX HTa vector (Invitrogen). <br /> (which common structural element RNA <br /> editing) editing sites respective RNA-binding    Transfection, RNAi induction growth <br /> and catalytic domains editosome proteins. Further- <br /> more, potential functions integrated         PF T. brucei strain 29&#226;&#8364;&#8220;13 cells (Wirtz et al. 1999) transfected <br /> those members KREPA family           described previously (Schnaufer et al. 2001), using 15 mg of <br />                                                               pZJMA6&#226;&#8364;&#8220;1 plasmid DNA linearized digestion with <br /> RNA-binding components editosome. For example, <br />                                                               NotI. The resultant stable cell lines selected growth in <br /> like KREPA6, KREPA4 binds gRNA preference <br />                                                               SDM-79 medium containing 10% FBS supplemented 15 <br /> for oligo(U) tail essential editing    mg/mL G418, 25 mg/mL hygromycin, 2.5 mg/mL phleomycin, <br /> editosome structural integrity, appears      designated RNAi-A6 PF. Correct integration 18S <br /> stably associated deletion insertion subcom-     rRNA locus confirmed PCR using vector-specific and <br /> plexes KREPA6 (Salavati et al. 2006). Therefore,   genome-specific primers. RNAi induced half culture <br /> conceivable KREPA6 KREPA4 involved          growth medium density 2.0 3 107 cells/ <br /> in &#226;&#8364;&#732;&#226;&#8364;&#732;hand-off&#226;&#8364;&#8482;&#226;&#8364;&#8482; process respective OB-fold do-    mL, dilution 1.2 3 106 cells/mL, addition 1 mg/mL tet. <br /> mains coordinate forward progression RNA            The induced uninduced cultures maintained between <br /> editing steps. This subunit switching recurring      1.2 3 106 2.0 3 107 cells/mL, cell density monitored <br /> unifying theme emerged mechanistic studies      daily using particle counter (Beckman). <br /> in DNA replication, repair, recombination (Stauffer <br /> and Chazin 2004). In transactions, OB-fold          Quantitative real-time PCR (qPCR) <br /> domains various DNA-binding proteins act coopera-          Oligonucleotide primers qPCR designed using Primer <br /> tively handing DNA substrate, direct     Express software (Applied Biosystems) purchased <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> competition binding sites allosteric structural re-    Invitrogen. Primers corresponding sequences specific <br /> arrangements facilitate dynamic restructuring DNA       KREPA6 mRNA, outside OB-fold motif, its <br /> processing complexes course DNA processing.         39 UTR, used distinguish KREPA family <br /> KREPA6 KREPA4, probably pre-             members (Worthey et al. 2003). Primers specific pre-edited <br /> dicted OB-fold proteins editosome,       edited mRNAs (A6, RPS12, COII, MURF2), never-edited <br /> play analogous roles editosome complexes        mRNAs (COI ND4), control 18S rRNA previously <br /> RNA editing. Such RNA-binding functions KREPA-         reported (Carnes et al. 2005; Trotter et al. 2005). Briefly, total <br />                                                               cellular RNA extracted using Trizol essentially described <br /> family proteins integrated pre- <br />                                                               manufacturer&#226;&#8364;&#8482;s protocol (GIBCO BRL) z1 3 108 <br /> dicted RNA-binding motifs editosome proteins, <br />                                                               KREPA6 RNAi PF cells RNAi induced 4 d of <br /> such zinc fingers, dsRBMs, Pumilio motifs. Clearly,    growth 1 mg/mL tet non-induced cells. Ten micro- <br /> both structural information protein/RNA com-             grams RNA DNase-treated using DNA-free kit <br /> plexes detailed studies dynamic events        (Ambion), following manufacturer&#226;&#8364;&#8482;s instructions, the <br /> occur RNA editing needed validate       quality resultant RNA assessed using Bioanalyzer <br /> hypothesis.                                                   (Agilent Technologies). cDNA generated intact total <br />  <br />                                                                                                    www.rnajournal.org        355 <br />  Tarun et al. <br />  <br />  <br />  <br /> cellular RNA using random hexamer primers TaqMan                HisKREPA6 protein expression purification <br /> Reverse Transcription Reagent Kit (Applied Biosystems) accord- <br /> ing manufacturer&#226;&#8364;&#8482;s instructions. Control reactions       The pProEX-HisKREPA6&#226;&#8364;&#8220;1 plasmid encoding recombinant <br /> reverse transcriptase included. Amplification reactions (25        KREPA6 protein TEV cleavable N-terminal His6 tag was <br /> mL total) set triplicate 96-well plate format,      transformed expressed E. coli BL-21(DE3). Cells were <br /> each reaction contained 2.5 mL appropriate dilutions cDNA         grown OD600 0.5&#226;&#8364;&#8220;0.7 37&#194;&#176;C LB broth containing 100 <br /> (typically 1:7 1:50) 5 mL 1.5 mM stock       mg/mL ampicillin induced 1 mM isopropyl b-D-1- <br /> the forward reverse primer pairs, 12.5 mL SYBR Green         thiogalactopyranoside 20 h 18&#194;&#176;C. The protein purified <br /> PCR Master Mix (Applied Biosystems). qPCR reactions                follows steps 4&#194;&#176;C. Two grams fresh quick- <br /> run ABI Prism 700 Sequence Detection System (Applied             frozen, washed cells resuspended 6 mL wash buffer A <br /> Biosystems) following amplification conditions: 10 min        (50 mM Tris-HCl pH 8.0) contained Complete Mini- <br /> at 95&#194;&#176;C followed 40 cycles 15 sec 95&#194;&#176;C 1 min 60&#194;&#176;C       EDTA free Protease Inhibitor Cocktail (Roche). They lysed <br /> at cycle. PCR efficiencies calculated linear regression    30-min incubation 4&#194;&#176;C 100 mg/mL lysozyme (Sigma), and <br /> using LinRegPCR (Ramakers et al. 2003), amplification           lysate cleared 30 min centrifugation 10,000g after <br /> analyzed using Pfaffl method (Pfaffl 2001). Data               adding NaCl 1.0 M sonicating reduce viscosity. In order <br /> normalized 18S rRNA, relative changes mRNA abun-              remove nucleic acids, cleared lysate mixed 5 mL of <br /> dance RNAi induction expressed fold-changes               Q-Sepharose beads (Amersham Biosciences) pre-equil- <br /> relative uninduced control cells.                                    ibrated buffer A containing 1.0 M NaCl incubated for <br />                                                                         60 min gentle shaking poured glass column, and <br />                                                                         flowthrough collected. HisKREPA6 protein purified <br /> Glycerol gradient fractionations Western analysis <br />                                                                         flowthrough binding 0.5 mL column nickel- <br /> RNAi-A6 cells induced 1 mg/mL tet 0, 2, 4 d (i.e.,     nitrilotriacetic acid (QIAGEN) equilibrated buffer <br /> until prior growth inhibition), lysates prepared       washing stepwise 5 mL buffer B (20 mM Tris-HCl at <br /> 1 3 108 total cells mitochondria isolated 1.25 3 1010      pH 8.0, 100 mM KCl, 10% glycerol) containing 2 20 mM <br /> cells induction described previously (Harris et al. 1990),   imidazole, eluting 1.5 mL buffer B containing <br /> using lysis buffer protease inhibitors [20 mM HEPES             250 mM imidazole. Each step HisKREPA6 purification was <br /> pH 7.9, 10 mM Mg(OAc)2, 50 mM KCl, 1 mM EDTA, 1 mg/mL                   monitored SDS-PAGE Western analysis using rabbit anti- <br /> pepstain, 2 mg/mL leupeptin, 1 mM pefabloc]. One milliliter         KREPA6 anti-His antibodies. A single protein band an <br /> of cleared lysates layered 11-mL 10%&#226;&#8364;&#8220;30% glycerol          apparent size 22 kDa seen Western stained gels. The <br /> gradient contained 10 mM Tris-Cl (pH 7.2), 10 mM MgCl2,            purified protein dialyzed changes buffer C (10 <br /> 100 mM KCl, 1 mM DTT, 1 mg/mL pepstatin, 2 mg/mL leupeptin,             mM Tris-HCl pH 7.2, 100 mM KCl, 10% glycerol, 1 mM <br /> and 1 mM pefabloc. Total cell lysates centrifuged 9 h      DTT) resulted z1 mL 4&#226;&#8364;&#8220;5 mg/mL protein based on <br /> mitochondrial extracts 12 h 38,000 rpm SW40            comparison serial dilutions protein known amounts <br /> rotor (Beckman) 4&#194;&#176;C. Half-milliliter fractions collected        BSA determined Bradford analysis using Coomassie <br /> from top, aliquoted, quick-frozen liquid nitrogen        Plus Kit (Pierce) comparison serial dilutions protein <br /> stored 80&#194;&#176;C Western vitro RNA editing assays.             known amounts BSA stained SDS-PAGE gels. Aliquots <br />    For Western analysis, 30 mL fractions electrophoresed           stored 80&#194;&#176;C. <br /> in pre-cast 10% Tris-HCl SDS-PAGE gels (Bio-Rad) trans- <br /> ferred PVDF membrane described previously (Trotter                RNA-binding assays <br /> et al. 2005). The membranes probed monoclonal <br /> antibodies specific KREPA1, KREPA2, KREPA3, KREL1               Electrophoretic mobility shift assays (EMSA) performed as <br /> (Panigrahi et al. 2001b), rabbit polyclonal antibodies specific         previously described (Salavati et al. 2006) indicated. <br /> for KREPA6 (Schnaufer et al. 2003), monoclonal antibody            Briefly, single-strand RNA binding, 2&#226;&#8364;&#8220;8 fmol [g-32P] 59- <br /> recognizing protein, designated 2-OGDCp,               end-labeled RNA incubated increasing concentrations <br /> mitochondrial 2-oxoglutarate dehydrogenase complex (A. Panigrahi,       purified HisKREPA6 protein 20-mL reactions RBB-1 <br /> pers. comm.).                                                           buffer (20 mM Tris-HCl pH 7.4, 150 mM KCl, 5 mM Mg2Cl, <br />                                                                         1 mM DTT, 0.1 mg/mL BSA, 2 units/mL RNAsin) 30 min at <br /> RNA synthesis vitro RNA editing                                  room temperature. Reactions incubated, mixed gel- <br />                                                                         loading dye (0.05% bromophenol blue, 0.05% xylene cyanol, 8% <br /> Synthetic 24-mer RNAs (UU12, UG12, UC12, AG12) DNAs             sucrose), loaded directly native 10% TBE gel (Bio-Rad), <br /> (dAG12 dUU12) purchased de-protected according             pre-run 110 V 30 min 0.5% TBE buffer 4&#194;&#176;C. The gel <br /> to manufacturer&#226;&#8364;&#8482;s protocol (Dharmacon). The A6 short/               electrophoresed 1.5 h pre-running conditions, fixed <br /> TAG.1 ATP synthase subunit 6 pre-mRNA cognate                   10% isopropanol plus 7% acetic acid 30 min, air-dried <br /> gA6[14] gRNA oligo(U) tail transcribed         cellophane sheets overnight. For duplex RNA-binding <br /> in vitro PCR products described previously (Seiwert et al.      assay, equimolar (250 nM) amounts internally labeled <br /> 1996) used gel-shift analyses (Salavati et al. 2006). In vitro   [32P]UTP gRNA unlabeled A6 short/TAG.1 pre-mRNA were <br /> pre-cleaved insertion assayed using 59-labeled 59-CL18         annealed 20 mL 50 mM Tris (pH 7.5), 0.1 mM EDTA, 100 <br /> 39-CL13pp gPCA6&#226;&#8364;&#8220;2A RNAs, pre-cleaved deletion used            mM KCl, 3% glycerol, 2 mM MgCl2, 1 mL RNA Secure <br /> 59-labeled U5-59, U5-39, gA6[14]PC-del RNAs previously           (Ambion) GeneAmp System 9700 PCR machine (Applied <br /> described (Igo Jr. et al. 2000, 2002).                                  Biosystems) 10 min 60&#194;&#176;C, cooled 27&#194;&#176;C 2&#194;&#176;/min, and <br />  <br /> 356      RNA, Vol. 14, No. 2 <br />                                                                                              Essential RNA-binding protein RNA editing <br />  <br />  <br />  <br /> held temperature 20 h (D. Koslowsky, unpubl.).                Carnes, J., Trotter, J.R., Peltan, A., Fleck, M., Stuart, K. 2007. <br /> Annealing efficiency determined electrophoresis pre-               RNA editing Trypanosoma brucei requires different <br /> cast 10% TBE native gels (Bio-Rad). One-tenth (25 nM)                 editosomes. Mol. Cell. Biol. doi: 10.1128/MCB.01374-07. <br />                                                                           Drozdz, M., Palazzo, S.S., Salavati, R., O&#226;&#8364;&#8482;Rear, J., Clayton, C., and <br /> annealed RNA used reaction gel-shift analyses as <br />                                                                              Stuart, K. 2002. TbMP81 required RNA editing Trypa- <br /> described single-strand RNA binding.                               nosoma brucei. EMBO J. 21: 1791&#226;&#8364;&#8220;1799. <br />    HisKREPA6-bound unbound RNAs visualized using                 Ernst, N.L., Panicucci, B., Igo Jr., R.P., Panigrahi, A.K., Salavati, R., <br /> a Molecular Dynamics PhosphorImager, scaled-up reactions                 Stuart, K. 2003. TbMP57 39 terminal uridylyl transferase <br /> performed triplicate quantified using ImageQuant                 (TUTase) Trypanosoma brucei editosome. Mol. Cell 11: <br /> software. The overall apparent equilibrium dissociation constant             1525&#226;&#8364;&#8220;1536. <br /> (overall KD app), i.e., HisKREPA6 protein concentration            Feagin, J.E., Jasmer, D.P., Stuart, K. 1987. Developmentally <br />                                                                              regulated addition nucleotides apocytochrome b tran- <br /> which half RNA bound, estimated sum               scripts Trypanosoma brucei. Cell 49: 337&#226;&#8364;&#8220;345. <br /> all shifted radiolabel lane, including (which    Feagin, J.E., Abraham, J.M., Stuart, K. 1988. Extensive editing <br /> may aggregated material), RNA/protein stoichiometry             cytochrome c oxidase III transcript Trypanosoma brucei. <br /> in band uncertain. The data analyzed using GraphPad             Cell 53: 413&#226;&#8364;&#8220;422. <br /> Prism Software (GraphPad Software, Inc.) determine overall          Harris, M.E., Moore, D.R., Hajduk, S.L. 1990. Addition <br /> binding model RNA. The EC50 values RNA,                    uridines edited RNAs trypanosome mitochondria <br />                                                                              occurs independently transcription. J. Biol. Chem. 265: <br /> defined concentration HisKREPA6 evoking 50% RNA <br />                                                                              11368&#226;&#8364;&#8220;11376. <br /> binding, taken overall KD app determined using           Huang, C.E., O&#226;&#8364;&#8482;Hearn, S.F., Sollner-Webb, B. 2002. Assembly and <br /> the Hill equation floating Hill coefficient P-values          function RNA editing complex Trypanosoma brucei <br /> &lt;0.0001 F-test.                                                       requires band III protein. Mol. Cell. Biol. 22: 3194&#226;&#8364;&#8220;3203. <br />    RNA-gel-shift assays competitor RNAs carried          Igo Jr., R.P., Palazzo, S.S., Burgess, M.L.K., Panigrahi, A.K., and <br /> described using fixed (6 pmol) HisKREPA6                   Stuart, K. 2000. Uridylate addition RNA ligation contribute to <br /> protein. The radiolabeled RNA mixed various molar                   specificity kinteoplastid insertion RNA editing. Mol. Cell. <br />                                                                              Biol. 20: 8447&#226;&#8364;&#8220;8457. <br /> excess amounts unlabeled competitor RNA buffer prior             Igo Jr., R.P., Weston, D.S., Ernst, N.L., Panigrahi, A.K., Salavati, R., <br /> the addition HisKREPA6 protein. The competition                 Stuart, K. 2002. Role uridylate-specific exoribonuclease activ- <br /> was estimated ratio bound RNA presence cold              ity Trypanosoma brucei RNA editing. Eukaryot. Cell 1: 112&#226;&#8364;&#8220;118. <br /> competitor bound RNA competitor.                              Kable, M.L., Seiwert, S.D., Heidmann, S., Stuart, K. 1996. RNA <br />                                                                              editing: A mechanism gRNA-specified uridylate insertion into <br />                                                                              precursor mRNA. Science 273: 1189&#226;&#8364;&#8220;1195. <br /> ACKNOWLEDGMENTS                                                           Kang, X., Rogers, K., Gao, G., Falick, A.M., Zhou, S., Simpson, L. <br />                                                                              2005. Reconstitution uridine-deletion precleaved RNA editing <br /> We thank G. Cross E. Wirtz providing T. brucei strain                recombinant enzymes. Proc. Natl. Acad. Sci. 102: 1017&#226;&#8364;&#8220; <br /> 29.13, P. Englund pZJM vector, members Stuart             1022. <br /> laboratory insightful comments. Funding work             Koslowsky, D.J., Bhat, G.J., Perrollaz, A.L., Feagin, J.E., Stuart, K. <br /> provided NIH grant GM042188 K.S., initial joint support             1990. The MURF3 gene T. brucei contains multiple domains <br /> AI014102 K.S., NIH grant GM077418 W.H.,                    extensive editing homologous subunit NADH <br />                                                                              dehydrogenase. Cell 62: 901&#226;&#8364;&#8220;911. <br /> NIH F-32 Ruth L. Kirschstein National Research Service Awards             Law, J.A., O&#226;&#8364;&#8482;Hearn, S., Sollner-Webb, B. 2007. In Trypanosoma <br /> for Individual Post-doctoral Fellows grant AI68303 S.Z.T.                 brucei RNA editing, TbMP18 (band VII) critical editosome <br />                                                                              integrity insertional deletional cleavages. Mol. <br /> Received August 6, 2007; accepted October 31, 2007.                          Cell. Biol. 27: 777&#226;&#8364;&#8220;787. <br />                                                                           Luo, S., Rohloff, P., Cox, J., Uyemura, S.A., Docampo, R. 2004. <br />                                                                              Trypanosoma brucei plasma membrane-type Ca2+-ATPase 1 <br /> REFERENCES                                                                   (TbPMC1) 2 (TbPMC2) genes encode functional Ca2+- <br />                                                                              ATPases localized acidocalcisomes plasma membrane, <br /> Aphasizhev, R., Aphasizheva, I., Simpson, L. 2003. A tale         essential Ca2+ homeostasis growth. J. Biol. Chem. 279: <br />    TUTases. Proc. Natl. Acad. Sci. 100: 10617&#226;&#8364;&#8220;10622.                         14427&#226;&#8364;&#8220;14439. <br /> Babbarwal, V.K., Fleck, M., Ernst, N.L., Schnaufer, A.,               Madison-Antenucci, S., Sabatini, R.S., Pollard, V.W., Hajduk, S.L. <br />    Stuart, K.D. 2007. An essential role KREPB4 RNA editing             1998. Kinetoplastid RNA-editing-associated protein 1 (REAP-1): <br />    structural integrity editosome Trypanosoma brucei.          A novel editing complex protein repetitive domains. EMBO J. <br />    RNA 13: 737&#226;&#8364;&#8220;744.                                                          17: 6368&#226;&#8364;&#8220;6376. <br /> Benne, R., Van den Burg, J., Brakenhoff, J.P.J., Sloof, P., Van           Madison-Antenucci, S., Grams, J., Hajduk, S.L. 2002. Editing <br />    Boom, J.H., Tromp, M.C. 1986. Major transcript                 machines: The complexities trypanosome RNA editing. Cell <br />    frameshifted coxII gene trypanosome mitochondria contain             108: 435&#226;&#8364;&#8220;438. <br />    nucleotides encoded DNA. Cell 46: 819&#226;&#8364;&#8220;826.    Matthews, J.M. Sunde, M. 2002. Zinc fingers&#226;&#8364;&#8221;Folds many <br /> Bochkarev, A. Bochkareva, E. 2004. From RPA BRCA2: Lessons            occasions. IUBMB Life 54: 351&#226;&#8364;&#8220;355. <br />    single-stranded DNA binding OB-fold. Curr. Opin.           McManus, M.T., Shimamura, M., Grams, J., Hajduk, S.L. 2001. <br />    Struct. Biol. 14: 36&#226;&#8364;&#8220;42.                                                  Identification candidate mitochondrial RNA editing ligases <br /> Brecht, M., Niemann, M., Schlu&#194;&#168;ter, E., Mu&#194;&#168;ller, U.F., Stuart, K.,       Trypanosoma brucei. RNA 7: 167&#226;&#8364;&#8220;175. <br />    Go&#194;&#168;ringer, H.U. 2005. TbMP42, protein component RNA           Morris, J.C., Wang, Z., Drew, M.E., Paul, K.S., Englund, P.T. <br />    editing complex African trypanosomes endo-exoribonu-               2001. Inhibition bloodstream form Trypanosoma brucei gene <br />    clease activity. Mol. Cell 17: 621&#226;&#8364;&#8220;630.                                   expression <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span>RNA interference</span> using pZJM dual T7 vector. <br /> Carnes, J., Trotter, J.R., Ernst, N.L., Steinberg, A.G., Stuart, K.      Mol. Biochem. Parasitol. 117: 111&#226;&#8364;&#8220;113. <br />    2005. An essential RNase III insertion editing endonuclease         O&#226;&#8364;&#8482;Hearn, S., Huang, C.E., Hemann, M., Zhelonkina, A., Sollner- <br />    Trypanosoma brucei. Proc. Natl. Acad. Sci. 102: 16614&#226;&#8364;&#8220;16619.              Webb, B. 2003. Trypanosoma brucei RNA editing complex: Band II <br />  <br />  <br />                                                                                                                   www.rnajournal.org          357 <br />  Tarun et al. <br />  <br />  <br />  <br />     structurally critical maintains band V ligase,            Seiwert, S.D. Stuart, K. 1994. RNA editing: Transfer genetic <br />     nonessential. Mol. Cell. Biol. 23: 7909&#226;&#8364;&#8220;7919.                                information gRNA precursor mRNA vitro. Science 266: <br /> Panigrahi, A.K., Gygi, S., Ernst, N., Igo Jr., R.P., Palazzo, S.S.,              114&#226;&#8364;&#8220;117. <br />     Schnaufer, A., Weston, D., Carmean, N., Salavati, R.,                     Seiwert, S.D., Heidmann, S., Stuart, K. 1996. Direct visualization <br />     Aebersold, R., et al. 2001a. Association novel proteins,              uridylate deletion vitro suggests mechanism kineto- <br />     TbMP52 TbMP48, Trypanosoma brucei RNA editing                   plastid RNA editing. Cell 84: 831&#226;&#8364;&#8220;841. <br />     complex. Mol. Cell. Biol. 21: 380&#226;&#8364;&#8220;389.                                    Simpson, L., Aphasizhev, R., Gao, G., Kang, X. 2004. Mitochon- <br /> Panigrahi, A.K., Schnaufer, A., Carmean, N., Igo Jr., R.P., Gygi, S.P.,          drial proteins complexes Leishmania Trypanosoma <br />     Ernst, N.L., Palazzo, S.S., Weston, D.S., Aebersold, R., Salavati, R.,       involved U-insertion/deletion RNA editing. RNA 10: 159&#226;&#8364;&#8220;170. <br />     et al. 2001b. Four related proteins Trypanosoma brucei RNA         Stauffer, M.E. Chazin, W.J. 2004. Structural mechanisms DNA <br />     editing complex. Mol. Cell. Biol. 21: 6833&#226;&#8364;&#8220;6840.                             replication, repair, recombination. J. Biol. Chem. 279: 30915&#226;&#8364;&#8220; <br /> Panigrahi, A.K., Ernst, N.L., Domingo, G.J., Fleck, M., Salavati, R.,            30918. <br />     Stuart, K.D. 2006. Compositionally functionally distinct          Stuart, K. Panigrahi, A.K. 2002. RNA editing: Complexity and <br />     editosomes Trypanosoma brucei. RNA 12: 1038&#226;&#8364;&#8220;1049.                         complications. Mol. Microbiol. 45: 591&#226;&#8364;&#8220;596. <br /> Pfaffl, M.W. 2001. A new mathematical model relative quantifi-            Stuart, K.D., Schnaufer, A., Ernst, N.L., Panigrahi, A.K. 2005. <br />     cation real-time RT-PCR. Nucleic Acids Res. 29: e45. doi:                 Complex management: RNA editing trypanosomes. Trends <br />     10.1093/nar/29.9.e45.                                                        Biochem. Sci. 30: 97&#226;&#8364;&#8220;105. <br /> Ramakers, C., Ruijter, J.M., Deprez, R.H.L., Moorman, A.F.M.              Tarun Jr., S.Z. Sa <br /> </body></html>