<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Physcomitrella patens mutants affected heat <br /> dissipation clarify evolution photoprotection <br /> mechanisms land colonization <br /> Alessandro Alboresia, Caterina Gerottob, Giorgio M. Giacomettib, Roberto Bassia,1, Tomas Morosinottob <br /> a <br />     Dipartimento di Biotecnologie, Universit&#195;&#160; di Verona, 37134 Verona, Italy; bDipartimento di Biologia, Universit&#195;&#160; di Padova, 35121 Padua, Italy <br />  <br /> Edited* Diter von Wettstein, Washington State University, Pullman, WA, approved April 23, 2010 (received review March 5, 2010) <br />  <br /> Light source energy photosynthetic organisms;                  particularly important &#239;&#172;?eld conditions, plants <br /> in excess, however, drives formation reactive oxygen               lacking PSBS rapidly counterselected (8). <br /> species and, consequently, photoinhibition. Plants algae                    The green alga Chlamydomonas reinhardtii shows different <br /> evolved mechanisms regulate light harvesting ef&#239;&#172;?ciency                      requirements NPQ activation, process induced by <br /> response variable light intensity avoid oxidative dam-                acclimation high light, constitutive plants. <br /> age. Nonphotochemical quenching (NPQ) consists rapid                       Also, green alga does need PSBS (15&#226;&#8364;&#8220;17) distinct <br /> dissipation excess excitation energy heat. Although wide-                   Lhc-like protein, called LHCSR (or Li818), implying important <br /> spread oxygenic photosynthetic organisms, NPQ shows                         differences mechanism activation heat dissipation <br /> important differences machinery. In land plants, Ara-              (17). LHCSR taxa, brown and <br /> bidopsis thaliana, NPQ depends presence PSBS, whereas <br />                                                                                   green algae (18), vascular plants. Conversely, although <br /> in green alga Chlamydomonas reinhardtii requires different <br />                                                                                   genes encoding PSBS identi&#239;&#172;?ed genome (19), <br /> protein called LHCSR. In work, proteins are <br /> present moss Physcomitrella patens. By generating KO <br />                                                                                   protein itself detected algae date (16), <br /> mutants lacking PSBS and/or LHCSR, demonstrate                       consistent NPQ dependent LHCSR. Further <br /> both gene products active NPQ. Plants lacking proteins                evidence NPQ activation different algae vascular <br /> are susceptible high light stress WT, implying                  plants (20) provided lack Lhcb6, an <br /> they active photoprotection. These results suggest                    antenna component involved PSBS-dependent NPQ <br /> NPQ fundamental mechanism <span id='am-3' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-4' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-5' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span>survival</span> excess light                   activation plants (12, 21). <br /> that land colonization, photosynthetic organisms evolved                        The organization photosystems similar vascular <br /> a unique mechanism excess energy dissipation losing                    plants green algae absence Lhcb6 <br /> the ancestral algae.                                                 Lhcb3 (18, 20). Grana stacks developed ancient <br />                                                                                   taxa Streptophyta line (Coleochaetales Charales) (22, <br /> LHCSR     | nonphotochemical quenching | photosystem | plants evolution |         23), algae, most, stacking two <br /> PSBS                                                                              thylakoid membranes. <br />                                                                                      Studying organisms diverged green lineage <br />  <br /> S   unlight provides energy supporting life photosynthetic <br />     organisms leads formation reactive oxygen <br /> species excess (1, 2). During early Devonian, when <br />                                                                                   leading vascular plants early land colonization helps us <br />                                                                                   understand photosynthetic organisms adapted chal- <br />                                                                                   lenges different environmental conditions. The moss Phys- <br /> plants &#239;&#172;?rst colonized terrestrial habitats, underwent                     comitrella patens valuable choice model organism be- <br /> competition organisms. However, adapt                     cause genome completely sequenced tools are <br /> harsher physicochemical conditions original water                     available genetic manipulation (3, 24). Early studies the <br /> ecosystem (3) atmosphere concentration oxy-                     organization photosynthetic genes showed P. patens the <br /> gen, inhibitor photosynthesis, higher (4) concentra-                 example plants LHCSR PSBS <br /> tion carbon dioxide, &#239;&#172;?nal acceptor electrons extracted                  genes present (3, 18, 20). In work, of <br /> <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> water photosystems, lower. Moreover, sessile form                  genes expressed corresponding polypeptides <br /> of life acquired land prevented escape rapid changes                   accumulated thylakoid membranes. By generating <br /> light intensity swimming deeper, behavior typical algae <br />                                                                                   speci&#239;&#172;?c KO mutants LHCSR genes PSBS <br /> (5). The combination conditions makes likely <br /> that light harvested excess respect maximal rate                Physcomitrella genome, proteins are <br /> of photochemical reactions, fast ef&#239;&#172;?cient photopro-                     active promoting NPQ contribute photoprotection <br /> tection response essential survival.                                       excess light conditions. These results suggest the <br />    The fastest response high light stress provided non-                  PSBS-dependent NPQ plants evolved LHCSR- <br /> photochemical quenching (NPQ), consists thermal                      based mechanism typical algae lost. The dis- <br /> dissipation chlorophyll excited singlet states (1Chl*) (6&#226;&#8364;&#8220;8).              appeared later stages plant evolution, newly <br /> NPQ major components: energy quenching (qE),                        evolved PSBS-dependent mechanism ensured suf&#239;&#172;?cient level <br /> is activated seconds increase light intensity,                photoprotection. <br /> inhibitory quenching, slower relaxes 1&#226;&#8364;&#8220;2 h in <br /> the dark (7, 9). In vascular plants, qE activation requires PSBS, <br /> a protein homologous light harvesting antenna subunits                      Author contributions: A.A., R.B., T.M. designed research; A.A. C.G. performed <br /> photosystems (Lhc) (10), activated accumulation                   research; A.A., C.G., T.M. analyzed data; G.M.G., R.B., T.M. wrote paper. <br /> of protons chloroplast lumen protonation                    The authors declare con&#239;&#172;&#8218;ict interest. <br /> glutamate residues (11). Activated PSBS induces decrease                     *This Direct Submission article prearranged editor. <br /> excited state lifetime pigment-binding subunits                     1 <br />                                                                                    <span id='am-2' about='protege:TO' typeof='owl:Thing'>To</span> correspondence addressed. E-mail: roberto.bassi@univr.it. <br /> antenna consequent reorganization photosystem                      This article contains supporting information online www.pnas.org/lookup/suppl/doi:10. <br /> subunits thylakoid membrane (12&#226;&#8364;&#8220;14). NPQ proven                     1073/pnas.1002873107/-/DCSupplemental. <br />  <br />  <br />  <br /> 11128&#226;&#8364;&#8220;11133 | PNAS | June 15, 2010 | vol. 107 | no. 24                                                                 www.pnas.org/cgi/doi/10.1073/pnas.1002873107 <br />  Results                                                                                              mydomonas extracts yielded signal PSBS LHCSR only, <br /> P. patens Expresses Both LHCSR PSBS Shows High NPQ. NPQ                                      respectively. In Chlamydomonas, LHCSR detected in <br /> in green alga C. reinhardtii relies presence LHCSR.                                    extracts high light-acclimated cells, consistent recent <br /> This group Lhc-like proteins algae                                  report high light acclimation needed accumulation <br /> not land plants, remarkable exception P. patens,                                      induction NPQ activity (17) variance P. patens, <br /> a bryophyte diverged vascular plants 400 million                                    constitutively accumulated. <br /> years ago, early land colonization (3, 25). A search The                                       Fig. 1B shows NPQ activation response illumination <br /> Institute Genomic Research (TIGR) sequence databases                                             strong actinic light dark-adapted plants higher P. patens <br /> allowed identi&#239;&#172;?cation ESTs encoding LHCSR                                             respect Arabidopsis. Consistently, decline maximal <br /> PSBS proteins bryophyte, Tortula ruralis,                                         &#239;&#172;&#8218;uorescence yield (Fm&#226;&#8364;&#178;), parameter closely re&#239;&#172;&#8218;ecting life- <br /> (TA794_38588 CN203629, respectively), suggesting                                       time chlorophyll (Chl) singlet excited states antenna <br /> is unique property P. patens.                                                               compartment, faster &#239;&#172;?nal intensity lower moss <br />    Finding genome sequences does imply corre-                                           respect vascular plant (Fig. 1C). It noted that <br /> sponding polypeptides expressed play expected                                   actinic light intensity measurement chosen be <br /> function, shown case PSBS green algae (16).                                     just suf&#239;&#172;?cient saturate photosynthesis ensure maximal NPQ <br /> We searched LHCSR PSBS polypeptides                                             amplitude organisms. Despite using lower actinic light in- <br /> P. patens thylakoids using speci&#239;&#172;?c antibody probes. We detected                                   tensity (800 vs. 1,200 &#206;&#188;E&#194;&#183;m&#226;&#710;&#8217;2&#194;&#183;s&#226;&#710;&#8217;1), capacity heat dissipation was <br /> speci&#239;&#172;?c signals expected molecular mass P. patens                                         higher P. patens Arabidopsis thaliana. Consistently, the <br /> LHCSR (around 23 kDa) (20) PSBS (22 kDa), showing                                           stronger NPQ mosses observed large range actinic <br /> both proteins actively expressed accumulate mem-                                      lights (Fig. S1), con&#239;&#172;?rming capacity dissipating excita- <br /> brane thylakoids (Fig. 1A). A search Arabidopsis Chla-                                     tion energy heat higher moss respect plant. <br />  <br />                                                                                                      Targeted KO PSBS, LHCSR1, LHCSR2. It tempting hy- <br />   A                                                    At                   Pp          Cr           pothesize faster stronger photoprotection found <br />                                                                                                      mosses relies action PSBS-dependent and <br />                                                                                                      LHCSR-dependent mechanisms energy dissipation. To assess <br />       &#206;&#177; - PSBS                                                                                       case, produced speci&#239;&#172;?c KO mosses <br />                                                                                                      depleted genes encoding putative NPQ triggering <br />       &#206;&#177; - LHCSR                                                                                      polypeptides. P. patens valuable organism gene KO <br />                                                                                                      performs homologous recombination high ef&#239;&#172;?- <br />                                                                                                      ciency (24). Disruption constructs genes cre- <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                          PLANT BIOLOGY <br />                                              Light                            Light o&#239;&#172;&#8364;            ated substitute entire coding sequences selection <br />   B                                                                                                  cassettes, schematized Fig. 2 A&#226;&#8364;&#8220;C. PEG-mediated pro- <br />                               4 <br />  <br />  <br />  <br />  <br />                               3 <br />                                                                                                      A                     Pp PSBS <br />                                                                                                                                                    D                    Pp psbs KO <br />                                                                                                                                                                WT   1    2   3   4 <br />                   NPQ <br />  <br />  <br />  <br />  <br />                               2 <br />                                                                                                                                                          LB <br />                                                                                                                                                          RB <br />                                                                                                                           ZEOCIN res                 Control <br />                               1 <br />  <br />                                                                                                                                                                           KO lines <br />                                                                                                      B               Pp LHCSR1                     E           WT psbs lhcsr1 lhcsr2 <br />                               0 <br />  <br />   C                          1.0       0         200        400         600       800         1000 <br />                                                                                                                                                      Pp PSBS <br />                                                               Time (seconds) <br />                                                                                                                                                    Pp LHCSR1 <br />                              0.8                                                                                     HYGROMYCIN res <br />                                                                                                                                                    Pp LHCSR2 <br />        Fluorescence (a.u.) <br />  <br />  <br />  <br />  <br />                              0.6 <br />  <br />                                                                                                      C                 Pp LHCSR2 <br />                                                                                                                                                    F                      KO lines <br />                                                                                                                                                                WT   psbs lhcsr1 lhcsr2 <br />                                                                                                                                                      Pp PSBS <br />                              0.4 <br />  <br />  <br />                                                                                                                                                    Pp LHCSR1 <br />                              0.2                                                                                      G418 res <br />                                                                                                                                                    Pp LHCSR2 <br />                                                                                                           500 bp <br />                                    0       200          400           600        800         1000 <br />  <br />                                                               Time (seconds) <br />                                                                                                      Fig. 2. KO mutant generation characterization. Scheme constructs <br />                                                                                                      used KO generation. Genomic region PSBS (A) LHCSR (B C) <br /> Fig. 1. Comparison NPQ A. thaliana, P. patens, C. reinhardtii. (A)                            genes schematized, exons shown black. Gray boxes represent the <br /> Western blotting NPQ involved polypeptides, PSBS, LHCSR                               genomic regions exploited homologous recombination. Below shown <br /> Arabidopsis (At), Physcomitrella (Pp), Chlamydomonas (Cr). Thylakoids                            constructs homologous recombination: genes antibiotic re- <br /> (1 &#206;&#188;g Chl) loaded cases. In case Cr, thylakoids                           sistance located regions homologous genome (all pri- <br /> isolated high light-grown cells; here, LHCSR bands recognized                           mers used mutagenesis reported Table S1). (D) Example of <br /> as study Peers et al. (17). Bands squared LHCSR                            veri&#239;&#172;?cation DNA insertion genome ampli&#239;&#172;?cation right <br /> a larger molecular mass Cr respect Pp (29 vs. 23 kDa). Anti-LHCSR                         left borders. An example independent lines Psbs KO shown. <br /> antibody recognizes proteins lower speci&#239;&#172;?city (likely Lhc proteins)                       (E) Ampli&#239;&#172;?cation genomic DNA using primers external target re- <br /> at different molecular masses At Pp. NPQ (B) &#239;&#172;&#8218;uorescence                             combination region. Mutants carrying single insertion identi&#239;&#172;?ed the <br /> (C) kinetics measured Pp (black) compared At (red). Actinic                            different size ampli&#239;&#172;?ed band respect WT. Ampli&#239;&#172;?cates were <br /> light intensities shown saturate photosynthetic capacity                             sequenced verify insertion correct position. (F) Evaluation <br /> organisms, shown fact pulses induce small peak                              PSBS LHCSR gene expression assessed RT-PCR WT different <br /> &#239;&#172;&#8218;uorescence (Fm&#226;&#8364;&#178;), activate maximal NPQ.                                                    mutant lines. Elongation Factor-1 alpha (EF1&#206;&#177;) reported control. <br />  <br />  <br /> Alboresi et al.                                                                                                                  PNAS | June 15, 2010 | vol. 107 | no. 24 | 11129 <br />  toplast transformation led isolation stably resistant                    content (Table 1), indicates mutations affect <br /> colonies characterized PCR verify presence                  composition photosynthetic apparatus, thus <br /> the insertion expected target site. A &#239;&#172;?rst set PCR assays                comparable mutants. The maintenance photosystem <br /> on genomic DNA performed amplify left right                        organization con&#239;&#172;?rmed measuring functional <br /> borders inserted cassette (an example ampli&#239;&#172;?cates                    antenna size, determined vivo measuring &#239;&#172;&#8218;uorescence <br /> selected lines psbs mutants reported Fig. 2D). At                   kinetics plants treated dichloromethylurea (DCMU) <br /> four independent clones insertion correct position                 (Fig. S2). Also, quantum yield photosystem II (PSII) in <br /> were isolated retained characterization.                           dark-adapted samples unaffected: All plants showed vari- <br />    During transformation Physcomitrella, multiple insertions                 able maximal &#239;&#172;&#8218;uorescence (Fv/Fm) value 0.80 &#194;&#177; 0.03 <br /> the site occur (24, 26, 27). To select single copy                      10 days growth control light conditions. <br /> insertions, used PCR assay primers annealing                         When NPQ kinetics recorded instead, KO plants showed <br /> genomic sequences insertion region. PCR                      large differences respect WT (Fig. 3B). The stronger <br /> ampli&#239;&#172;?cations possible case single in-                   <span id='am-81' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-82' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> observed case lhcsr1 KO, followed by <br /> sertion, multiple insertions, fragments large                  psbs KO lhcsr2 KO. This <span id='am-315' about='xsp:pattern' typeof='owl:Thing'>pattern</span> consistent the <br /> to ef&#239;&#172;?ciently ampli&#239;&#172;?ed (Fig. 2E). All PCR fragments                        Western blotting analysis shown Fig. 3A: LHCSR protein <br /> sequenced control, &#239;&#172;?nally retained                 immunodetected WT synthesized PpLHCSR1 gene, <br /> independent single insertion lines transformation. The                    isoform expressed lower levels, explaining the <br /> suppression gene expression con&#239;&#172;?rmed RT-PCR                         small phenotype lhcsr2 KO. It pointed by <br /> speci&#239;&#172;?c primers: cDNAs present WT lost                      overloading lhcsr1 KO thylakoids, able detect <br /> corresponding mutants (Fig. 2F).                                                   LHCSR2 Western blotting, content de- <br />    Single-insertion mutants grown, 10-day-old plants                      tection limit Fig. 3, shown Fig. S3. Although antibody <br /> were harvested used thylakoid puri&#239;&#172;?cation. When ana-                       bind isoforms different af&#239;&#172;?nity, these <br />                                                                                    results strongly suggest major compensatory in- <br /> lyzed Western blotting, signal PSBS detected in <br />                                                                                    crease LHCSR2 absence LHCSR1. <br /> psbs KO (Fig. 3A). KO LHCSR showed the <br />                                                                                       It interesting observe KO mutants com- <br /> immunoblotting signal lost lhcsr1 KO, protein <br />                                                                                    petent NPQ, activity comparable NPQ <br /> level substantially unaffected lhcsr2 KO. Red Ponceau <br />                                                                                    levels measured WT Arabidopsis (Fig. S4). Furthermore, the <br /> staining anti-CP43 antibody routinely used verify                      largest fraction NPQ attributed qE, component <br /> equal loading samples (Fig. 3A). In mutants, ob-                    dependent low luminal pH, fast kinetics relaxation <br /> served signi&#239;&#172;?cant alterations Chl a/b Chl/carotenoid                     sensitivity nigericin. This different case of <br />                                                                                    Arabidopsis, residual NPQ PSBS-less mutant npq4 <br />                                                                                    small slow rise relaxation (10). <br />     A                                           <span id='am-237' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-238' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-240' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-242' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-244' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-261' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-264' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-267' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-271' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-277' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-279' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-289' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-303' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span>Knock-out</span>                             We report data line genotype, but <br />                              WT         psbs        lhcsr1 lhcsr2                  results replicated selected lines. Also, we <br />                                                                                    did observe signi&#239;&#172;?cant difference NPQ amplitude or <br />            -PSBS                                                                   PSBS/LHCSR protein accumulation mutants having <br />                                                                                    single multiple insertions, suggesting effect inserting <br />                                                                                    multiple copies DNA relevant observed pheno- <br />            -LHCSR                                                                  type (an example shown Fig. S5). In cases, was <br />                                                                                    strict correlation recombination event shown by <br />                                                                                    PCR assay, absence protein Western blotting, and <br />            -CP43                                                                   amplitude NPQ phenotype, implying low NPQ <br />                                                                                    phenotype attributable KO mutations (Fig. S5). More- <br />                                                                                    over, transformations using different plasmids carrying either <br />                        Light                     Light o&#239;&#172;&#8364;                       zeocin neomycin resistance tested case PSBS. <br />     B                                                                              In cases, phenotype same, con&#239;&#172;?rming effect <br />                                                                                    presence/absence PSBS NPQ stronger than <br />                                                                                    possible secondary effects (Fig. S6). <br />  <br />                                                                                    Double Triple KO Mutants. Once role gene NPQ <br />                                                                                    assessed, performed transformation mutant <br />                                                                                    backgrounds obtain double triple KO mutants. The pres- <br />                                                                                    ence <span id='am-159' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-160' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span>mutation</span> veri&#239;&#172;?ed PCR Western <br />  <br />                                                                                    Table 1. Pigment-binding properties isolated mutants <br />                                                                                                        WT           psbs KO          lhcsr1 KO          lhcsr2 KO <br />  <br />                                                                                    Chl a/Chl b         2.56           2.58              2.52               2.60 <br />                                                                                    Chl/Car             4.33           4.25              4.60               4.43 <br />  <br />                                                                                                        WT          psbs lhcsr1 KO          psbs lhcsr1 lhcsr2 KO <br /> Fig. 3. P. patens psbs lhcsr KO phenotypes. (A) Western blotting using <br /> antibodies LHCSR PSBS. Western blotting CP43           Chl a/Chl b        2.56               2.42                       2.57 <br /> shown loading control. Thylakoids loaded cases, 0.5 &#206;&#188;g Chl     Chl/Car            4.33               4.52                       4.29 <br /> for anti-PSBS 0.3 &#206;&#188;g Chl others. (B) NPQ kinetics selected <br /> lines. Curves WT psbs KO, lhcsr1 KO, lhcsr2 KO shown               Pigment-binding properties single multiple mutants depleted in <br /> black, blue, green, red, respectively. Averages SD calculated          PSBS, LHCSR1, LHCSR2, respectively. Chl a/b Chl/carotenoids (Car) <br /> from &#239;&#172;?ve independent measures.                                            ratios reported. SD 0.1 case Chl a/b 0.3 Chl/Car. <br />  <br />  <br /> 11130 | www.pnas.org/cgi/doi/10.1073/pnas.1002873107                                                                                                 Alboresi et al. <br />        A                                       Knock-out <br />                                                            psbs <br />                                             psbs <br />                                WT                         lhcsr1 <br />                                            lhcsr1 <br />                                                           lhcsr2 <br />              -PSBS <br />  <br />             -LHCSR <br />  <br />             -CP43 <br />                     Light                     Light off <br />      B <br />  <br />                                                                                 Fig. 5. Photoprotection capacity NPQ-affected mutants. The photo- <br />                                                                                 protection capacity previously described mutants compared <br />                                                                                 WT treating 5-day-old plants additional 3 days high light. <br />                                                                                 PSII quantum yield, expressed Fv/Fm, monitored daily. WT, psbs KO, <br />                                                                                 lhcsr1 KO shown black, red, blue, respectively, psbs <br />                                                                                 lhcsr1 psbs lhcsr1 lhcsr2 KO double triple mutants shown in <br />                                                                                 green cyan. Values WT culture kept control light <br />                                                                                 conditions reported black dots. Data mutants grown in <br />                                                                                 control conditions omitted clarity indistin- <br />                                                                                 guishable data WT culture. <br />  <br />  <br />  <br />  <br />                                                                                                                                                                 PLANT BIOLOGY <br />                                                                                 plants partial recovery following days, dem- <br />                                                                                 onstrating acclimating mechanisms acting slower <br />                                                                                 time courses active mosses independent NPQ, pre- <br /> Fig. 4. Phenotypes P. patens psbs lhcsr KO double triple             viously reported plants algae (29&#226;&#8364;&#8220;31). In single KO <br /> mutants. (A) Western blotting using antibody LHCSR PSBS. (B) <br />                                                                                 mutants, phenotype photoprotection small, any, suggest- <br /> NPQ kinetics selected lines. Curves WT, lhcsr1 KO, psbs lhcsr1 KO, and <br /> psbs lhcsr1 lhcsr2 KO shown black, red, green, blue, respectively. <br />                                                                                 ing PSBS LHCSR able provide photoprotection <br /> Averages SD calculated &#239;&#172;?ve independent measures.          capacity similar WT, absence subunit, <br />                                                                                 conditions. <br />  <br /> blotting, described (Fig. 4A). The depletion LHCSR1                 Discussion <br /> in psbs KO background causes reduction NPQ.                        PSBS LHCSR Are Active Independently P. patens NPQ. Arabi- <br /> However, double mutant retains residual NPQ having                 dopsis NPQ known depend presence the <br /> qE properties, attributed residual presence              PSBS subunit photosystem II (10, 11). Although PSBS <br /> LHCSR2. This hypothesis veri&#239;&#172;?ed isolating triple mutants,                gene conserved genome green algae (19), was <br /> where LHCSR2 missing. NPQ triple mutant                     detected polypeptide (16). On contrary, green <br /> very low completely depleted fast component, qE (Fig.                alga C. reinhardtii, shown recently NPQ requires <br /> 4B). In fact, small residual NPQ stable 10              LHCSR, different Lhc-like polypeptide previously called Li818. <br /> min dark recovery following illumination, observed                 Genes encoding LHCSR polypeptide algae <br /> PSBS-less npq4 mutant Arabidopsis (10).                                      groups, including diatoms (32). <br />                                                                                    PSBS gene polypeptide identi&#239;&#172;?ed several <br /> Role Individual LHCSR PSBS Gene Products Protection              plant species comparable NPQ kinetics, all <br /> High Light Stress. NPQ mechanism known contribute                    data available suggest NPQ depends PSBS vascular <br /> resistance light stress higher plants algae (8, 17,              plants. Algae highly diversi&#239;&#172;?ed characterized group <br /> 28). To prove case P. patens, treated WT             organisms; thus, cautious generalizing. <br /> mutant 5-day-old plants high light intensities. Fv/Fm                  Nevertheless, far, PSBS polypeptide detected in <br /> measured day monitor PSII photodamage. As shown                     green alga belonging Viridiplantae, despite presence of <br /> Fig. 5, WT plants, shift high light conditions, showed                corresponding gene genome (16). Thus, present knowl- <br /> a drastic decrease PSII ef&#239;&#172;?ciency, clear sign photo-                    edge consistent hypothesis NPQ activation relies <br /> inhibition. PSII ef&#239;&#172;?ciency recovers following days, sug-               different mechanisms. This proven true Chla- <br /> gesting plants start acclimating light conditions. Single           mydomonas Arabidopsis, model systems green <br /> KO mutants similarly affected light treatment, where-               algae plants, respectively, subject pre- <br /> as double psbs lhcsr1 triple psbs lhcsr1 lhcsr2 KO                  sent study. Although molecular mechanisms LHCSR <br /> mutants showed stronger degree PSII photoinhibition. This               action unclear, available data Arabidopsis suggest that <br /> a clear indication shift high light conditions affects              PSBS acts modulator antenna proteins (12, 21, 33, 34). <br /> plants depleted PSBS LHCSR drastically.                        Among different Lhc proteins, Lhcb3 Lhcb6 shown to <br /> Despite experiencing stronger photoinhibition, however,                  speci&#239;&#172;?cally involved PSBS-dependent NPQ activation (12, <br />  <br /> Alboresi et al.                                                                                        PNAS | June 15, 2010 | vol. 107 | no. 24 | 11131 <br />  21) present plants. So far, evidence                               Materials Methods <br /> obtained presence algae (20), suggesting                                 Plant Growth. Protonemal tissue P. patens, Gransden WT strain, grown <br /> function PSBS algae, any, substantially different                             minimum PpNO3 medium (35) solidi&#239;&#172;?ed 0.8% Plant Agar (Duchefa <br /> that plants.                                                                                Biochemie). Plants propagated sterile conditions 9-cm Petri <br />   P. patens plant PSBS LHCSR                                     dishes overlaid cellophane disk (A.A. Packaging Limited), previously <br /> gene sequences (20). The report PSBS                                    described (20). Plates placed growth chamber controlled <br /> LHCSR EST sequences T. ruralis suggests                                       conditions: 24 &#194;&#176;C, 16-h light/8-h dark photoperiod, light intensity of <br />                                                                                                40 &#206;&#188;E&#194;&#183;m&#226;&#710;&#8217;2&#194;&#183;s&#226;&#710;&#8217;1 (control conditions). For high light tests, 5-day-old plants <br /> present mosses well. There indication present <br />                                                                                                moved control 350 &#206;&#188;E&#194;&#183;m&#226;&#710;&#8217;2&#194;&#183;s&#226;&#710;&#8217;1, maintaining temperature and <br /> that vascular plants carry LHCSR-encoding sequences                                     photoperiod. <br /> their genomes, suggests change mechanisms of <br /> photoprotection Viridiplantae evolution. In work,                               NPQ Measurements. In vivo chlorophyll &#239;&#172;&#8218;uorescence P. patens mea- <br /> we analyzed role LHCSR PSBS polypeptides                                    sured room temperature Dual PAM-100 &#239;&#172;&#8218;uorometer (Heinz Walz <br /> P. patens generating speci&#239;&#172;?c KO plants: mutants showed                                  GmbH, Germany), saturating light 6,000 &#206;&#188;E&#194;&#183;m&#226;&#710;&#8217;2&#194;&#183;s&#226;&#710;&#8217;1 actinic light <br /> a decreased NPQ phenotype, implying LHCSR                                        830 &#206;&#188;E&#194;&#183;m&#226;&#710;&#8217;2&#194;&#183;s&#226;&#710;&#8217;1. In case Arabidopsis, plants grown 100 <br /> PSBS active triggering photoprotection mechanism.                                  &#206;&#188;E&#194;&#183;m&#226;&#710;&#8217;2&#194;&#183;s&#226;&#710;&#8217;1 8-h light/16-h dark photoperiod measured with <br />   LHCSR PSBS NPQ activities P. patens largely in-                                   1,200 &#206;&#188;E&#194;&#183;m&#226;&#710;&#8217;2&#194;&#183;s&#226;&#710;&#8217;1 actinic light. Before measurements, plates dark- <br /> dependent; study, shown mutants lacking PSBS                              adapted 40 min room temperature. The parameters Fv/Fm NPQ <br /> or LHCSR protein (psbs lhcsr1 KO, respectively)                                calculated (Fm&#226;&#710;&#8217;Fo)/Fm (Fm&#226;&#710;&#8217;Fm&#226;&#8364;&#178;)/Fm&#226;&#8364;&#178; (36). Antenna size was <br /> are capable signi&#239;&#172;?cant NPQ. Although mutual interactions                              determined rising time PSII &#239;&#172;&#8218;uorescence DCMU-treated plants. <br /> cannot excluded, effect appears additive, consistent <br />                                                                                                Protoplast Transformation KO Generation. Genomic P. patens pro- <br /> with observation conditions maximal NPQ, &#239;&#172;&#8218;uores- <br />                                                                                                tonemal DNA extracted (37) used starting template mo- <br /> cence quenching P. patens faster stronger Arabi-                             lecular cloning. All regions downstream target coding sequences <br /> dopsis (Fig. 1). Consistently, lhcsr1 KO plants, missing                           ampli&#239;&#172;?ed PCR subcloned pGEM-T Vector (catalog no. <br /> LHCSR protein, NPQ amplitude comparable                                     A3600; Promega). Regions PSBS (locus XM_001778511), LHCSR1 (locus <br /> Arabidopsis plants (Fig. S4).                                                                  XM_001776900), LHCSR2 (locus XM_001768019) genes succes- <br />   The mutant phenotype high light stress experiments                                   sively cloned BZRf/BNRr, BHRf, BNRf plasmids (kindly provided by <br /> supports hypothesis; observed strong light sensitivity                            F. Nogue, Institut National la Recherche Agronomique, Versailles, France), <br /> when PSBS LHCSR absent (or largely depleted                                   respectively. P. patens transformation performed study by <br /> in case psbs lhcsr1 double KO mutant). When                                    Schaefer Zr&#195;&#191;d (24) minor modi&#239;&#172;?cations. Brie&#239;&#172;&#8218;y, 5- 6-day-old pro- <br /> protein present, suf&#239;&#172;?cient activate NPQ                                tonemal tissue collected protoplast generation PEG-mediated <br />                                                                                                transformation. Resistant colony selection started 6 days transformation <br /> provide protection levels equivalent WT experimental <br />                                                                                                transferring regenerated plants culture media supplemented the <br /> conditions tested. <br />                                                                                                appropriate antibiotic [50 &#206;&#188;g&#194;&#183;L&#226;&#710;&#8217;1 G418 (Sigma-Aldrich); 30 &#206;&#188;g&#194;&#183;L&#226;&#710;&#8217;1 Hygromicyn <br />                                                                                                B (Sigma-Aldrich), 50 &#206;&#188;g&#194;&#183;L&#226;&#710;&#8217;1 zeocin (Duchefa Biochemie)]. Resistant colo- <br /> Evolution Heat Dissipation Mechanisms. Presented results allow                              nies transferred 10 days nonselective media in <br /> the proposal P. patens retains LHCSR-dependent NPQ                                    antibiotic media isolate stable transformants. Con&#239;&#172;?rmation DNA in- <br /> mechanism Chlamydomonas, additionally                                        sertion performed PCR assay, detailed Table S1. <br /> showing newly evolved PSBS-dependent mechanism typical <br /> of land plants. Together, mechanisms provide optimal                                     Thylakoid Extraction, SDS/PAGE, Western Blotting Analyses. Thylakoids <br /> photoprotection Physcomitrella high light conditions. Our                                protonemal tissue (10- 12-day-old plants) prepared using an <br /> results suggest PSBS-dependent mechanism excess                                    Arabidopsis protocol minor modi&#239;&#172;?cations (20). After SDS/PAGE, pro- <br /> energy dissipation appeared plant evolution                                  teins transferred nitrocellulose membrane (Sartorious AG) using <br /> ancestral LHCSR-dependent function lost. This &#239;&#172;?nding                                       blot Biorad detected speci&#239;&#172;?c polyclonal anti- <br /> highlights importance photoprotection newly colo-                                bodies produced laboratory. <br /> nized land environment. In fact, PSBS-dependent <br />                                                                                                ACKNOWLEDGMENTS. We grateful Giulia Bonente (Universit&#195;&#160; di Ver- <br /> NPQ mechanism evolved later, established                                   ona, Verona, Italy) providing high light- low light-adapted Chlamy- <br /> being superimposed ancestral mechanism ensure                                  domonas cells Fabien Nogu&#195;&#169; [Institut Jean-Pierre Bourgin (IJPB), <br /> a suf&#239;&#172;?cient level continual protection. In absence NPQ,                              Institut National la Recherche Agronomique (INRA), Versailles, France] <br /> in fact, strong decrease &#239;&#172;?tness fast counterselection                                 suggestions moss transformation. A doctoral grant awarded to <br />                                                                                                C.G. Fondazione Cassa di Risparmio di Padova e Rovigo (CaRiPaRo). A.A. <br /> would expected, shown case psbs-less Arabidopsis                               R.B. received &#239;&#172;?nancial support Italian Ministry Research <br /> mutants (8) consistent effect high light                                   (FIRB PAREALLELOMICS, Grant RBIP06CTBR). T.M. received &#239;&#172;?nancial support <br /> treatment triple mutant (Fig. 5).                                                       Universit&#195;&#160; di Padova (Grant CPDA089403). <br />  <br />  <br /> 1. Barber J, Andersson B (1992) Too good thing: Light bad                 10. Li XP, et al. (2000) A pigment-binding protein essential regulation photo- <br />    photosynthesis. Trends Biochem Sci 17:61&#226;&#8364;&#8220;66.                                                    synthetic light harvesting. Nature 403:391&#226;&#8364;&#8220;395. <br /> 2. Li Z, Wakao S, Fischer BB, Niyogi KK (2009) Sensing responding excess light.         11. Li XP, Phippard A, Pasari J, Niyogi KK (2002) Structure-function analysis of <br />    Annu Rev Plant Biol 60:239&#226;&#8364;&#8220;260.                                                                 photosystem II subunit S (PsbS) vivo. Funct Plant Biol 29:1131&#226;&#8364;&#8220;1139. <br /> 3. Rensing SA, et al. (2008) The Physcomitrella genome reveals evolutionary insights      12. Betterle N, et al. (2009) Light-induced dissociation antenna hetero-oligomer is <br />    conquest land plants. Science 319:64&#226;&#8364;&#8220;69.                                              needed non-photochemical quenching induction. J Biol Chem 284:15255&#226;&#8364;&#8220;15266. <br /> 4. Scott AC, Glasspool IJ (2006) The diversi&#239;&#172;?cation Paleozoic &#239;&#172;?re systems &#239;&#172;&#8218;uctuations   13. Bonente G, Howes BD, Caffarri S, Smulevich G, Bassi R (2008) Interactions the <br />    atmospheric oxygen concentration. Proc Natl Acad Sci USA 103:10861&#226;&#8364;&#8220;10865.                    photosystem II subunit PsbS xanthophylls studied vivo vitro. J Biol Chem <br /> 5. Waters ER (2003) Molecular adaptation origin land plants. Mol Phylogenet <br />                                                                                                    283:8434&#226;&#8364;&#8220;8445. <br />    Evol 29:456&#226;&#8364;&#8220;463. <br />                                                                                                14. Kiss AZ, Ruban AV, Horton P (2008) The PsbS protein controls organization the <br /> 6. Genty B, Briantais J-M, Baker NR (1989) The relationship quantum yield <br />                                                                                                    photosystem II antenna higher plant thylakoid membranes. J Biol Chem 283:3972&#226;&#8364;&#8220;3978. <br />    photosynthetic electron transport quenching chlorophyll &#239;&#172;&#8218;uorescence. <br />                                                                                                15. Finazzi G, et al. (2006) Nonphotochemical quenching chlorophyll &#239;&#172;&#8218;uorescence in <br />    Biochim Biophys Acta 990:87&#226;&#8364;&#8220;92. <br />                                                                                                    Chlamydomonas reinhardtii. Biochemistry 45:1490&#226;&#8364;&#8220;1498. <br /> 7. Niyogi KK (2000) Safety valves photosynthesis. Curr Opin Plant Biol 3:455&#226;&#8364;&#8220;460. <br />                                                                                                16. Bonente G, et al. (2008) The occurrence psbS gene product Chlamydomonas <br /> 8. K&#195;&#188;lheim C, Agren J, Jansson S (2002) Rapid regulation light harvesting plant <br />    &#239;&#172;?tness &#239;&#172;?eld. Science 297:91&#226;&#8364;&#8220;93.                                                          reinhardtii photosynthetic organisms correlation energy <br /> 9. Nilkens M, et al. (2010) Identi&#239;&#172;?cation slowly inducible zeaxanthin-dependent               quenching. Photochem Photobiol 84:1359&#226;&#8364;&#8220;1370. <br />    component non-photochemical quenching chlorophyll &#239;&#172;&#8218;uorescence generated               17. Peers G, et al. (2009) An ancient light-harvesting protein critical regulation <br />    steady-state conditions Arabidopsis. Biochim Biophys Acta 1797:466&#226;&#8364;&#8220;475.                algal photosynthesis. Nature 462:518&#226;&#8364;&#8220;521. <br />  <br />  <br />  <br /> 11132 | www.pnas.org/cgi/doi/10.1073/pnas.1002873107                                                                                                                        Alboresi et al. <br />  18. Koziol AG, et al. (2007) Tracing evolution light-harvesting antennae         28. Li XP, Muller-Moule P, Gilmore AM, Niyogi KK (2002) PsbS-dependent enhancement <br />     chlorophyll a/b-containing organisms. Plant Physiol 143:1802&#226;&#8364;&#8220;1816.                             feedback de-excitation protects photosystem II photoinhibition. Proc Natl <br /> 19. Anwaruzzaman M, et al. (2004) Genomic analysis mutants affecting xanthophyll                Acad Sci USA 99:15222&#226;&#8364;&#8220;15227. <br />     biosynthesis regulation photosynthetic light harvesting Chlamydomonas            29. Ballottari M, Dall&#226;&#8364;&#8482;Osto L, Morosinotto T, Bassi R (2007) Contrasting behavior higher <br />     reinhardtii. Photosynth Res 82:265&#226;&#8364;&#8220;276.                                                        plant photosystem I II antenna systems acclimation. J Biol Chem 282: <br /> 20. Alboresi A, Caffarri S, Nogue F, Bassi R, Morosinotto T (2008) In silico biochemical       8947&#226;&#8364;&#8220;8958. <br />     analysis Physcomitrella patens photosynthetic antenna: Identi&#239;&#172;?cation subunits        30. Sirikhachornkit A, Shin JW, Baroli I, Niyogi KK (2009) Replacement alpha-tocopherol <br />     evolved land adaptation. PLoS ONE 3:e2033.                                          beta-tocopherol enhances resistance photooxidative stress xanthophyll- <br /> 21. Kov&#195;&#161;cs L, et al. (2006) Lack light-harvesting complex CP24 affects structure        de&#239;&#172;?cient strain Chlamydomonas reinhardtii. Eukaryot Cell 8:1648&#226;&#8364;&#8220;1657. <br />     function grana membranes higher plant chloroplasts. Plant Cell 18:           31. Maeda H, DellaPenna D (2007) Tocopherol functions photosynthetic organisms. <br />     3106&#226;&#8364;&#8220;3120.                                                                                     Curr Opin Plant Biol 10:260&#226;&#8364;&#8220;265. <br /> 22. Larkum AWD, Vesk M (2003) Photosynthesis Algae, eds Larkum AWD, Douglas SE,             32. Coesel S, et al. (2009) Diatom PtCPF1 new cryptochrome/photolyase family <br />     Raven JA (Kluwer, The Netherlands), pp 11&#226;&#8364;&#8220;28.                                                  member DNA repair transcription regulation activity. EMBO Rep 10: <br /> 23. Gunning BES, Schwartz OM (1999) Confocal microscopy thylakoid auto&#239;&#172;&#8218;uo-                      655&#226;&#8364;&#8220;661. <br />     rescence relation origin grana phylogeny green algae. Aust J Plant     33. Ruban AV, et al. (2007) Identi&#239;&#172;?cation mechanism photoprotective energy <br />     Physiol 26:695&#226;&#8364;&#8220;708.                                                                            dissipation higher plants. Nature 450:575&#226;&#8364;&#8220;578. <br /> 24. Schaefer DG, Zr&#195;&#191;d JP (1997) Ef&#239;&#172;?cient gene targeting moss Physcomitrella             34. Teardo E, et al. (2007) Evidences interaction PsbS photosynthetic <br />     patens. Plant J 11:1195&#226;&#8364;&#8220;1206.                                                                  complexes maize thylakoids. Biochim Biophys Acta 1767:703&#226;&#8364;&#8220;711. <br /> 25. Nishiyama T, et al. (2003) Comparative genomics Physcomitrella patens game-             35. Ashton NW, Grimsley NH, Cove DJ (1979) Analysis gametophytic development in <br />     tophytic transcriptome Arabidopsis thaliana: Implication land plant evolution.         moss, Physcomitrella patens, using auxin cytokinin resistant mutants. Planta <br />     Proc Natl Acad Sci USA 100:8007&#226;&#8364;&#8220;8012.                                                          144:427&#226;&#8364;&#8220;435. <br /> 26. Mur&#195;&#169;n E, Nilsson A, Ulfstedt M, Johansson M, Ronne H (2009) Rescue                     36. Demmig-Adams B, et al. (1996) Using chlorophyll &#239;&#172;&#8218;uorescence assess fraction <br />     characterization episomally replicating DNA moss Physcomitrella. Proc              absorbed light allocated thermal dissipation excess excitation. Physiol Plant <br />     Natl Acad Sci USA 106:19444&#226;&#8364;&#8220;19449.                                                             98:253&#226;&#8364;&#8220;264. <br /> 27. Kamisugi Y, et al. (2006) The mechanism gene targeting Physcomitrella patens:        37. Allen GC, Flores-Vergara MA, Krasynanski S, Kumar S, Thompson WF (2006) A <br />     Homologous recombination, concatenation multiple integration. Nucleic Acids                modi&#239;&#172;?ed protocol rapid DNA isolation plant tissues using cetyltri- <br />     Res 34:6205&#226;&#8364;&#8220;6214.                                                                              methylammonium bromide. Nat Protoc 1:2320&#226;&#8364;&#8220;2325. <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                              PLANT BIOLOGY <br />  <br />  <br />  <br />  <br /> Alboresi et al.                                                                                        <br /> </body></html>