<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Copyright &#195;&#8220; 2008 Genetics Society America <br /> DOI: 10.1534/genetics.107.085779 <br />  <br />  <br />  <br />            Schizosaccharomyces pombe Histone Acetyltransferase Mst1 (KAT5) <br />                 Is Essential Protein Required Damage Response <br />                              Chromosome Segregation <br />  <br />         Eliana B. Go&#194;&#180;mez,*,&#226;&#8364;&#160; Rebecca L. Nugent,* Sebastia&#194;&#180;n Laria&#226;&#8364;&#160; Susan L. Forsburg*,&#226;&#8364;&#160;,1 <br />            *Molecular Computational Biology Section, University Southern California, Los Angeles, California 90089-2910 <br />             &#226;&#8364;&#160;Molecular Cell Biology Laboratory, Salk Institute Biological Studies, La Jolla, California 90089-2910 <br />                                                       Manuscript received December 12, 2007 <br />                                                       Accepted publication April 3, 2008 <br />  <br />  <br />                                                                   <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> <br />                 Schizosaccharomyces pombe Mst1 member MYST family histone acetyltransferases the <br />              likely ortholog Saccharomyces cerevisiae Esa1 human Tip60 (KAT5). We isolated temperature- <br />              sensitive allele essential gene. mst1 cells pleiotropic <span id='am-164' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-165' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-240' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> restrictive temperature. <br />              They sensitive variety DNA-damaging agents spindle poison thiabendazole. mst1 an <br />              increased frequency Rad22 repair foci, suggesting endogenous damage. Two-hybrid results that <br />              Mst1 interacts number proteins involved chromosome integrity centromere function, <br />              including methyltransferase Skb1, recombination mediator Rad22 (Sc Rad52), chromatin <br />              assembly factor Hip1 (Sc Hir1), Msc1 protein related family histone demethylases. mst1 <br />              mutant sensitivity hydroxyurea suggests defect recovery following HU arrest. We conclude Mst1 <br />              plays essential roles maintenance genome stability recovery DNA damage. <br />  <br />  <br />  <br />  <br /> M      ANY events DNA metabolism, including repli- <br />          cation repair, modulated <span id='am-321' about='xsp:pattern' typeof='owl:Thing'>pattern</span> <br /> of different histone modifications chromatin, <br />                                                                                Work experimental systems shows that <br />                                                                             ScEsa1 histone acetyltransferase catalytic subunit of <br />                                                                             large protein complex called NuA4 smaller <br /> leading speculation &#226;&#8364;&#732;&#226;&#8364;&#732;histone <span id='am-5' about='Thesaurus:code' typeof='owl:Thing'>code</span>&#226;&#8364;&#8482;&#226;&#8364;&#8482; pro-                     subcomplex, Piccolo, NuA4 (Allard et al. <br /> vides epigenetic information facilitates numerous                      1999; Boudreault et al. 2003; Eberharter et al. 2005; <br /> chromatin functions (reviewed Jenuwein Allis                         Selleck et al. 2005). The NuA4 complex acetylates <br /> 2001; Kouzarides 2007). Histone acetylation associ-                      histone H4 H2A (Smith et al. 1998; Allard et al. <br /> ated diverse chromatin functions (reviewed                          1999; Galarneau et al. 2000). It associated <br /> Kouzarides 2007; Lee Workman 2007). Histone                             acetylation histone variant H2AZ (Keogh <br /> acetylation changes association DNA under-                      et al. 2006; Millar et al. 2006). Esa1 family HATs contain <br /> lying nucleosomes (Shogren-Knaak et al. 2006)                           chromodomain, motif associated binding <span id='am-7' about='protege:TO' typeof='owl:Thing'>to</span> <br /> also creates specific binding sites proteins involved                   methylated histones (reviewed la Cruz et al. <br /> in variety DNA transactions (reviewed Kouzarides                    2005). While suggests aspect Esa1 <br /> 2007; Lee Workman 2007). Histone acetyltrans-                           function mediated binding methylated histone(s), <br /> ferase (HAT) enzymes separated                          target binding remains elusive. The mam- <br /> sequence-specific subfamilies differ sub-                     malian homolog Tip60 (reviewed Squatrito et al. <br /> strates composition associated com-                     2006). Under new initiative regularize nomencla- <br /> plexes. The MYST family HATs (reviewed Utley                          ture chromatin field, Mst1, Esa1, Tip60 are <br /> and Cote 2003) extensively characterized                        defined KAT5 family histone acetyltransferases <br /> budding yeast, members (reviewed                         (Allis et al. 2007). <br /> Lafon et al. 2007). We reported MYST family                           ScESA1 essential gene, loss function <br /> proteins fission yeast (Go&#194;&#180;mez et al. 2005): Mst2,                   results cell-cycle-specific arrest replication but <br /> likely ortholog Saccharomyces cerevisiae Sas2,                      mitosis (Smith et al. 1998; Clarke et al. 1999), <br /> nonessential gene required normal telomere silenc-                      recent report suggests be <br /> ing (Go&#194;&#180;mez et al. 2005). Mst1, likely ortholog S.                   required cell cycle entry (Early et al. 2004). The <br /> cerevisiae Esa1, essential gene subject                 roles ScEsa1 diverse, recent studies also <br /> this report.                                                                linked spindle assembly checkpoint (Le <br />                                                                             Masson et al. 2003) morphogenesis checkpoint <br />   1 <br />                                                                             (Le Masson et al. 2003; Ruault Pillus 2006). <br />    Corresponding author: Molecular Computational Biology Section, <br /> 1050 Childs Way, RRI 201, University Southern California, Los Angeles,   Genome-level analyses budding yeast indicate that <br /> CA 90089-2910. E-mail: forsburg@usc.edu                                     ScEsa1 activity associated active gene expression <br />  <br /> Genetics 179: 757&#226;&#8364;&#8220;771 ( June 2008) <br />  758                                                   E. B. Go&#194;&#180;mez et al. <br />  <br /> (Galarneau et al. 2000; Vogelauer et al. 2002). Para-             tractable fission yeast system. In report, present <br /> doxically, however, H4 acetylation associated        initial genetic characterization gene. Mst1 is <br /> decreased gene expression (Deckert Struhl 2001).              essential viability, mutants sensitive a <br /> Recent studies suggest ScEsa1 repress activity      variety DNA-damaging agents, including hydroxy- <br /> of specific gene regions, particularly heterochromatic        urea (HU), ultraviolet (UV) irradiation, methyl meth- <br /> domains telomeres rDNA (Clarke et al.              anesulfonate (MMS), bleomycin. Mutants also <br /> 2006).                                                            sensitive spindle poison thiabendazole. Using a <br />    Histone acetylation NuA4 complex role          two-hybrid screen, identify number potential <br /> DNA double-strand break (DSB) repair (reviewed                 interaction proteins involved DNA dynamics, includ- <br /> van Attikum Gasser 2005b; Squatrito et al.                    ing chromatin proteins, transcription factors, the <br /> 2006). Increased histone acetylation near damage              recombination protein Rad22 (ScRad52). Analysis of <br /> site occurs (Yuet al. 2005) acetylation H4 linked       HU response suggests Mst1 required for <br /> to activation homologous recombination non-                normal recruitment Rad22 HU. Thus, Mst1 has <br /> homologous end-joining (Bird et al. 2002; Tamburini               multiple roles maintenance genome integrity in <br /> and Tyler 2005). ScEsa1 recruited DNA breaks                S. pombe. <br /> (Bird et al. 2002; Tamburini Tyler 2005). One of <br /> the major responses DNA double-strand breaks is <br /> phosphorylation histone variant H2AX                              MATERIALS AND METHODS <br /> damage checkpoint kinases ATM ATR (reviewed in <br />                                                                      Strains, media, manipulations: Strains used study <br /> Foster Downs 2005; Morrison Shen 2005).                   listed Table 1. Strains grown maintained on <br /> H2AX phosphorylation recruitment NuA4 occur                yeast extract plus supplements (YES) Edinburgh minimal <br /> with similar timing (Downs et al. 2004). Interestingly,           media appropriate supplements, using standard techni- <br /> two chromatin-remodeling complexes, INO80                     ques (Forsburg Rhind 2006). Matings performed <br /> SWR1, recruited H2AX,                synthetic sporulation agar plates 2&#226;&#8364;&#8220;3 days 25&#194;&#176;. Tem- <br />                                                                   perature-sensitive (ts) cells grown 25&#194;&#176; non-ts cells <br /> through common Arp4 subunit share              32&#194;&#176;. Transformations carried electroporation. <br /> NuA4 (Downs et al. 2004; Robert et al. 2006). NuA4,               Double mutants constructed standard tetrad analysis <br /> INO80, SWR1 proposed facilitate exchange            random spore analysis. G418 plates YES supplemented <br /> phospho-H2AX, ultimately downregulating damage                100 mg/ml G418 (Sigma, St. Louis). <br /> signal (Morrison et al. 2004; Tsukuda et al. 2005; van               Plasmids constructions: Disruption tagging of <br />                                                                   mst11 described Go&#194;&#180;mez et al. (2005). Site-directed mu- <br /> Attikum Gasser 2005b; Papamichos-Chronakis                    tants constructed using Quick-Change mutagenesis <br /> et al. 2006). Recent studies implicated NuA4             kit (Stratagene, La Jolla, CA). <br /> complex regulation modification additional              Primers used L-S/F (CATACAATTTTCTTATGAATC <br /> histone H2A variant called H2AZ (Babiarz et al. 2006;             GACCAAGCGTGAGCACAAAC) L-S/R GTTTGTGCTCA <br /> Keogh et al. 2006; Millar Grunstein 2006). It             CGCTTGGTCGATTCATAAGAAAATTGTATG L-P/F GG <br />                                                                   TGTAGGAATATTTGTCTTCCGTCCAACCTTTTTCTGGATC <br /> been suggested downregulation H2AX                AC L-P/R GTGATCCAGAAAAAGTTTGGACGGAAGA <br /> phosphorylation signal reflects exchange phospho-              CAAATATTCCTACACC. Complementation temperature <br /> H2AX H2AZ (reviewed van Attikum                       sensitivity assessed transforming expression plas- <br /> Gasser 2005a).                                                    mid heterozygous mst11/DmstTura41 diploid FY1754 <br />    Tip60, human ortholog Mst1 ScEsa1,                  isolating haploids contain Dmst1Tura41 and <br />                                                                   plasmid markers. An integrant constructed moving <br /> acetylates nonhistone proteins independently               Pnmt-mst1-L344S cassette plasmid pEBG99 plas- <br /> NuA4 complex substrates including Notch (Kim                 mid pJK148 create plasmid pEBG98 integrating this <br /> et al. 2007), ATM (Sun et al. 2005; Jiang et al. 2006), p53       construct leu1-32 locus diploid. Haploid temper- <br /> (Sykes et al. 2006; Tang et al. 2006), DNA-PKcs               ature-sensitive segregants containing Dmst1Tura41 <br /> ( Jiang et al. 2006). Tip60-deficient cells exposed            leu11Tnmt-mst1-L344S isolated random spore <br />                                                                   analysis. <br /> ionizing radiation defects chromosome repair                 Silencing assessed examining growth absence <br /> and reduced apoptosis (Ikura et al. 2000). This leads          uracil low adenine strain ura41 integrated into <br /> the important conclusion HATs limited             centromere ade61 integrated telomere a <br /> histones substrates. These data suggest KAT5              minichromosome. <br /> family HATs contribute DNA damage response                 FACS: Samples collected 5 10 hr, ethanol <br />                                                                   fixed, analyzed flow cytometry described previously <br /> ways: first, affecting DNA repair recovery (e.g.,          (Go&#194;&#180;mez Forsburg 2003). <br /> through histone modifications recruit specific DNA              Immunoprecipitation: Samples prepared described <br /> damage response proteins) and, second, affecting               Go&#194;&#180;mez et al. (2002). For co-immunoprecipitation MCM <br /> the checkpoint response (e.g., direct modifica-           proteins, used anti-HA antibody 12CA5 (Abcam) and <br /> tion p53 ATM).                                              antiV5 (Invitrogen, San Diego). For histone acetylation assays, <br />                                                                   dried beads treated 20 ml histone acetylation <br />    To gain insight diverse function       mix containing 5 mm C14-Acetyl CoA (ICN), 0.7 mg core <br /> KAT5 protein family DNA metabolism, investi-                histones, 50 mm HEPES (pH 7.9), 10% glycerol, 1 mm <br /> gated role Schizosaccharomyces pombe Mst1           DTT. The reaction incubated 1 hr 30 min 30&#194;&#176;. Beads <br />                                               Mst1 Required Damage Response                                                   759 <br />  <br />                                                             TABLE 1 <br />                                                     Strains used study <br />  <br /> Strain                                           Genotype                                                          Source <br /> 11           h  ade6-M210                                                                                Our stock <br /> 261          h  ura4-D18 leu1-32 ade6-M210 can1-1                                                        Our stock <br /> 865          h  Dcds1Tura4 ura4-D18 leu1-32                                                              T. Wang <br /> 1104         h  Drad3Tura41 ura4 leu1-32 ade6-M210 (?)                                                   Our stock <br /> 1107         h  Drad3Tura41 ura4-D18 leu1-32 ade6-M216                                                   Our stock <br /> 1256         h  Dmad2Tura41 ade6M210 leu1-32 ura4D18                                                     S. Sazer <br /> 1552         h  Dbub1Tura41 ade6M210 leu1-32 ura4D18                                                     K. Hardwick <br /> 1754         h1/h  Dmst1Tura41/1 ura4-D18/ura4-D18 leu1-32/leu1-32 ade6-M216/ade6-M210                   Go&#194;&#180;mez et al. (2005) <br /> 2180         h1 cbh1TGFP-pBSIIKS-ura41-chh1C leu1-32 ura4-D18                                            Baum Clarke (2000) <br /> 2192         h  Dchk1Tura41 ura4-D18 ade6-M216                                                           Our stock <br /> 2217         h  Dcbh1TLEU2 ura4-D18 ade6-M216 leu1-32                                                    Y. Murakami <br /> 2339         h1 Dmst1Tura41 leu11Tmst1-L344S ura4-D18 ade6-M216 (#7)                                     This study <br /> 2396         h  Dmst1Tura41 leu11Tmst1-L344S ura4-D18 ade6-M210                                          This study <br /> 2449         h1 Dmst1TkanMX6 leu11Tmst1-L344S1 ura4-D18 ade6-M210                                        This study <br /> 2450         h  Dmst1TkanMX6 leu11Tmst1-L344S1 ura4-D18 ade6-M210                                        This study <br /> 2495         h1 Dtaz1Tura41 Dmst1TkanMX6 leu11Tmst1-L344Sura4-D18 leu1-32 ade6-M210                      This study <br /> 2498         h  Drhp51Tura41 Dmst1TkanMX6 leu11Tmst1-L344Sura4-D18 leu1-32 ade6-M210                     This study <br /> 2499         h1 Drad50TkanMX6 Dmst1Tura41 leu11Tmst1-L344S1ura4-D18 leu1-32 ade6-M21?                    This study <br /> 2501         h1 Drad32Tura41 mst1TkanMX6 leu11Tmst1-L344S1 ura4-D18 leu1-32 ade6-M21?                    This study <br /> 2505         h1 mst2Tura41 mst1TkanMX6 leu11Tmst1-L344S1 ura4-D18 leu1-32 ade6-M210                      This study <br /> 2506         h  mst2Tura41 mst1TkanMX6 leu11Tmst1-L344S1 ura4-D18 leu1-32 ade6-M210                      This study <br /> 2508         h1 swi6Tura41 mst1TkanMX6 leu11Tmst1-L344S1 ura4-D18 leu1-32 ade6-M210                      This study <br /> 2525         h  hsk1-1312 mst1TkanMX6 leu11Tmst1-L344S1 leu1-32 ade6-M21?                                This study <br /> 2535         h1 Dmst1TkanMX6 leu11Tmst1-L344S1 ade6-M210 ura4-D18 <br /> 2562         h1 Drad12Tura41 Dmst1TkanMX6 leu11Tmst1-L344S1 ade6-M210 ura4-D18                           This study <br /> 2566         h  Dclr4Tura41 Dmst1TkanMX6 leu11Tmst1-L344S1                                               This study <br />                ade6-M210 ura4-D18 (otrR(SphI)Tade61) ? <br /> 2569         h1 orp1-4 Dmst1Tura41 leu11Tmst1-L344S1 ade6-M210 ura4-D18 (can1-1)?                        This study <br /> 2570         h1 Dclr3TkanMX6 Dmst1Tura41 leu11Tmst1-L344S1 ade6-M210 ura4-D18 (can1-1)?                  This study <br /> 2572         h1 mcm7-98 Dmst1Tura41 leu11Tmst1-L344S1 ade6-M216 ura4-D18 (can1-1)?                       This study <br /> 2574         h1 cdc19-P1 Dmst1Tura41 leu11Tmst1-L344S1 ade6-M210 ura4-D18                                This study <br /> 2576         h1 pol1-1 Dmst1Tura41 leu11Tmst1-L344S1 ade6-M210 ura4-D18                                  This study <br /> 2578         h1 cdc21-M68 Dmst1Tura41 leu11Tmst1-L344S1 ade6-M216 ura4-D18 (can1-1)?                     This study <br /> 2601         h1 Dskb1Tura41 mst1TkanMX6 leu11Tmst1-L344S1 ura4-D18 ade6-M210                             This study <br /> 2535         h1 Dmst1TkanMX6 leu11Tmst1-L344S ade6-M210 ura4-D18                                         This study <br /> 3135         h  ade6-M210 leu1-32 ura4-D1 DhipTura41                                                     S. Whitehall <br /> 3286         h  Dcds1Tura41 rad22YFPTKanMX6 ura4-D18 leu1-32 ade6?                                       This study <br /> 3369         h1 Dmst1Tura41 leu11Tmst1-L344S ura4-D18 ade6-M21? rad22-YFP:kanMX4                         This study <br /> 3436         h  rad22-YFP:kanMX4 ade6-M210 ura4-D18 leu1-32                                              This study <br /> 3479         h1 Dmsc1TkanMX6 leu1-32                                                                     N. Walworth <br /> 3480         h1 msc1-HA-kanMX6 leu1-32                                                                   N. Walworth <br /> 3553         h? Dmst1TkanMX6 leu11Tmst1-L344S1 otr1L(dh/HindIII)Tura41                                   This study <br />                leu1-32 ura4DS/E ade6DN/N [ch16 M23TLEU21 telTade61] <br /> 3563         h? Dcbh1TLEU2 Dmst1TkanMX6 leu11Tmst1-L344S1 ade6-M21? ura4-D18                             This study <br /> 3562         h? Dmst1TkanMX6 leu11Tmst1-L344S1 ade6-M210 ura4-D18 DhipTura41                             This study <br />  <br />  <br />  <br /> were spun down, supernatant recovered boiled         SDS&#226;&#8364;&#8220;PAGE, followed immunoblotting anti-V5 (Invi- <br /> SDS&#226;&#8364;&#8220;Laemmli loading buffer electrophoresis 15%            trogen) 1:2500 dilution anti-GFP (living colors, <br /> SDS&#226;&#8364;&#8220;polyacrylamide gel electrophoresis (PAGE). The gel          Clontech) 1:1000 dilution. Crosslinking immunoprecipi- <br /> fixed treated fluorography-enhancing solution (NEN         tations previously described (Auger et al. 2008). <br /> Life Science) drying autoradiography.                    Briefly, cells crosslinked 1% formaldehyde (Sigma) <br />    For co-immunoprecipitation Rad22-YFP Cbh1-GFP            50 min ice. Harvested cells vortexed glass <br /> (strains 3436, 3480), prepared soluble protein lysates      beads IP buffer (50 mm HEPES&#226;&#8364;&#8220;KOH, pH 1.6, 500 mm <br /> fission yeast cells B88 buffer Moreno et al. (1991) A      NaCL, 1 mm EDTA, 1% Triton X-100, 0.1% sodium deoxy- <br /> total 1.0 mg protein used immunoprecipitation.           cholate). Extracts treated DNase I reverse cross- <br /> Lysates precleared sepharose A beads (Repligen)       link. A total 1.0 mg protein used immunoprecipitation. <br /> 1 hr. One microliter anti-GFP (Abcam 290) added          Whole-cell extracts immunoprecipitates incubated with <br /> left rotate overnight 4&#194;&#176;. Proteins analyzed 8%        SDS sample buffer 1 hr 90&#194;&#176; Western blot analysis. <br />  760                                                      E. B. Go&#194;&#180;mez et al. <br />  <br />                                                                                                         Figure 1.&#226;&#8364;&#8221;Spore germi- <br />                                                                                                      nation. (A) Wild-type diploid <br />                                                                                                      strain (ura41/ura4-D18) and <br />                                                                                                      (B) heterozygous disrup- <br />                                                                                                      tion mutant (mst11/Dmst1T <br />                                                                                                      ura41) sporulated and <br />                                                                                                      processed random spore <br />                                                                                                      analysis. Spores inocu- <br />                                                                                                      lated media lacking ura- <br />                                                                                                      cil sampled flow <br />                                                                                                      cytometry monitor entry <br />                                                                                                      S phase process <br />                                                                                                      DNA replication. Cell <br />                                                                                                      morphology visualized <br />                                                                                                      staining DNA with <br />                                                                                                      DAPI. <br />  <br />  <br />    Microscopy: Whole fixed cells stained described       metazoans suggest Hbo1 histone acetyltransfer- <br /> DAPI Calofluor (Go&#194;&#180;mez Forsburg 2003). Cells            ase, MYST family protein, function in <br /> visualized Leica DMR microscope. Objectives used were <br />                                                                      DNA replication (Iizuka Stillman 1999; Burke <br /> Leica 100X/1.30, PLFL, NA &#194;&#188; 1.30, Leica 63X/1.32, <br /> PLApo, NA &#194;&#188; 1.32. Images captured Hamamatsu              et al. 2001; Iizuka et al. 2006). To investigate whether <br /> digital camera Improvision Openlab software (Improvi-            S. pombe Mst1 contributes DNA replication, ex- <br /> sion, Lexington, MA). Images assembled using Canvas             amined phenotype cells containing Dmst1 <br /> (ACD/Deneba) adjusted contrast. Nuclear spreads              allele. Mutant diploids heterozygous Dmst1Tura41 <br /> were performed Pankratz Forsburg (2005) with <br />                                                                      wild-type diploids heterozygous ura41 were <br /> the following modifications. Rad22YFP detected using <br /> Abcam 290-50 rabbit anti-GFP, 1:2000 5% BSA (Sigma),          induced sporulate. We inoculated purified spores <br /> secondary donkey anti-rabbit cy3 1:500 DNA stained          diploids medium lacking uracil carry <br /> 13 Hoechst (Invitrogen). The objective used Leica 63X/           bulk spore germination. Only spores containing the <br /> 1.32, PLApo, NA &#194;&#188; 1.32. Images captured assembled           ura41 gene able germinate conditions, <br /> as above. <br />                                                                      DNA content determined flow cytom- <br />    Damage assays: Cells grown A595 &#194;&#188; 1, serially diluted <br /> fivefold, spotted YES plates YES plates containing       etry. Spores containing Dmst1Tura41 synthesized DNA <br /> 10 mg/ml thiabendazole (TBZ), 5 mm HU, 0.005, 0.007,           approximately 2C DNA content, timing sim- <br /> 0.01% MMS, incubated 3&#226;&#8364;&#8220;5 days 25&#194;&#176;, 29&#194;&#176;, 32&#194;&#176;,        ilar wild type (Figure 1). Half popula- <br /> indicated. For UV irradiation assay, 1000 cells plated       tion contains nongerminating ura4-D18 spores, which <br /> duplicate YES immediately treated indicated <br />                                                                      persist 1C population. Upon microscopic examina- <br /> doses UV irradiation using Stratalinker (Stratagene). <br /> Control plates left untreated. Plates incubated 32&#194;&#176;     tion, observed Dmst1Tura41 cells <br /> for 2&#226;&#8364;&#8220;3 days. Colonies counted percentage               showed highly disordered mitosis, including high <br /> <span id='am-8' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-9' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span>survival</span> obtained relative number colonies grown       fraction cut phenotypes. This suggests defects chro- <br /> on untreated control plates.                                     mosome segregation, result mitotic <br />    Two-hybrid strains: The mst11 open reading frame was <br />                                                                      defects replication defects checkpoint <br /> isolated PCR plasmid pEBG72 primers HAT-3 <br /> forward (AGATCTCGCCAAAAACTTGCTCAATCTTCTTCC)                          disruption. <br /> and HAT-3 reverse (GGTACCCGACATAGACTGGCACATTA                           Isolation characterization temperature-sensi- <br /> CATCC) cloned two-hybrid vector pDBLeu cleaved          tive allele: To better characterize Mst1, constructed a <br /> with NcoI NotI create plasmid pEBG91. Budding-yeast           temperature-sensitive allele based temperature-sensi- <br /> two-hybrid strain AH109 (Clontech) HIS3, ADE2, lacZ <br />                                                                      tive alleles S. cerevisiae ESA1 (Clarke et al. 1999). We <br /> reporters downstream heterologous GAL4-responsive pro- <br /> moter elements transformed S. pombe Matchmaker           tested mutations: L271P (corresponding L254P in <br /> cDNA library (Clontech) screened interactions             ScEsa1) L344S (corresponding L267S ScEsa1). <br /> described Leverson et al. (2002). Plasmids positive          Constructs containing placed LEU2 <br /> clones isolated verified retransformation,       plasmid control nmt1 promoter trans- <br /> identity interacting open reading frame determined <br />                                                                      formed Dmst1Tura41 diploid. Following sporu- <br /> by sequence. <br />                                                                      lation, assessed recovery spores 25&#194;&#176; containing <br />                                                                      ura41 marker (marking Dmst1) LEU2 <br />                           RESULTS <br />                                                                      marker (marking plasmid). Both mutants able <br />   Dmst1 cells undergo disordered mitosis: Previously,                complement disruption 25&#194;&#176;. However, upon <br /> we identified members MYST family his-                 shift 36&#194;&#176;, mst1-L344S strain temperature <br /> tone acetyltransferases fission yeast. Our analysis indi-         sensitive. <br /> cated Mst1 likely ortholog S. cerevisiae Esa1,           We attempted replace endogenous mst11 allele <br /> and like ScEsa1, chromatin-bound protein                L344S allele, success. We reasoned that, <br /> essential viability (Go&#194;&#180;mez et al. 2005). Data              integrated native promoter, single <br />                                                Mst1 Required Damage Response                                                761 <br />  <br />  <br />  <br />  <br />    Figure 2.&#226;&#8364;&#8221;Characterization temperature-sensitive allele. (A&#226;&#8364;&#8220;D) Comparison growth rate FACS wild-type (FY11) <br /> and mst1-L344S cells (FY 2396) grown presence thiamine 25&#194;&#176; 36&#194;&#176;. (Top) Colony formation plates, colony <br /> size proportional growth rate. (Bottom) FACS analysis following shift 36&#194;&#176;. (E) mst1-L344S cells 25&#194;&#176; (left) 36&#194;&#176; 6 <br /> (middle) 8 (right) hr shift. Cells stained DAPI Calcofluor. (F) Fraction cells indicated phenotypes <br /> during temperature-shift experiment. <br />  <br />  <br /> copy mst1-L344S insufficient viability.             fied Figure 2F). Strikingly, number cells <br /> Therefore, integrated mst1-L344S control                staining Calcofluor significantly increased the <br /> the nmt11 promoter leu1-32 locus het-               mst1-L344S strain restrictive temperature, nearly <br /> erozygous diploid sporulated create haploid              50% cells arrested septa 6 hr. Some these <br /> strain Dmst1Tura41 leu11TPnmt1-mst1-L344S. Upon shift               mononucleate, showing asymmetric nuclear seg- <br /> to 36&#194;&#176; addition thiamine shut nmt1                regation. Additionally, increase over- <br /> promoter, strain unable grow. However, cells            number binucleate cells relative wild type. <br /> were able grow presence thiamine 25&#194;&#176;             S. pombe unusual cell cycle cytokinesis is <br /> 32&#194;&#176;, indicating protein functional lower             completed subsequent nuclear cell cycle. <br /> temperatures. There sufficient background expression             Thus, cells 2C DNA content FACS be <br /> of nmt11 promoter presence thiamine                 mononucleate cells G2 M phase binucleated <br /> for stable proteins maintained (Figure 2, A&#226;&#8364;&#8220;D). An            cells nuclei G1 S phase cyto- <br /> equivalent strain kanMX marker Dmst1 behaved              kinesis complete. Therefore, arrest binucleate <br /> similarly (data shown).                                         septated cell represent defect entering DNA rep- <br />    We examined phenotype cells containing mst1-              lication ensuing cell cycle. Alternatively, could <br /> L344S shifting asynchronous cells restrictive             represent defect cytokinesis delays entry into <br /> temperature. The increase cell number blocked                cell cycle. <br /> within 2 hr. However, cell mass measured A595 con-                  This phenotype different disordered <br /> tinued increase  8 hr. DNA content, measured              mitosis seen Dmst1 mutant spores. Because ger- <br /> flow cytometry, showed change 2C peak                   minating spores arise stationary G1 cells, while <br /> characteristic exponentially growing wild-type fission           exponentially growing S. pombe predominantly G2 <br /> yeast, later time points showed flattening            2C DNA content, reasoned different <br /> the profile increase cells higher DNA                phenotypes represent effect inactivating <br /> content. This common heterogeneous, elongated                 Mst1 different cell cycle stages. Alternatively, they <br /> cells typical cdc mutants.                           suggest mst1-L344S allele creates hypo- <br />    We examined cells nuclear septum morphol-                morph complete inactivation gene. <br /> ogy staining DAPI Calcofluor, respectively,             We attempted resolve <span id='am-242' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-243' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-251' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-267' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-270' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-273' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-277' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-292' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-299' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-302' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-315' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-318' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-319' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span>mutation</span> affects <br /> and observing fluorescence (Figure 2E, quanti-                S phase synchronizing mst1-L344S cells G1, <br />  762                                                    E. B. Go&#194;&#180;mez et al. <br />  <br />                            TABLE 2                                 perature sensitivity. We defined genetic interaction as <br /> Genetic interactions mst1-L344S mutants          occurring double-mutant strain unable to <br />                                                                    grow temperatures parents growth <br />                                             Double-mutant          proficient (&#226;&#8364;&#732;&#226;&#8364;&#732;reduced growth temperature&#226;&#8364;&#8482;&#226;&#8364;&#8482;). There <br /> Mutation          Protein affected           phenotype             modest genetic interactions replication <br />                                                                    initiation mutants. Additionally, observed the <br />                    DNA replication <br /> mcm2ts       MCM helicase                  Reduced growth          maximum growth temperature reduced mst1 <br />                                              temperature           combined mutations Drhp51 (ScRad51), <br /> mcm4ts       MCM helicase                  Reduced growth          Drad50, Drad22, affect homologous DNA <br />                                              temperature           recombination. The Drad22 mst1-L344S strain grew <br /> mcm7ts       MCM helicase                  Reduced growth          extremely slowly 25&#194;&#176;; however, the <br />                                              temperature           high frequency spontaneous suppressors Drad22 <br /> orp1ts       ORC subunit                   Reduced growth <br />                                                                    backgrounds (Osman et al. 2005), confi- <br />                                              temperature <br /> pol1ts       DNA polymerase              Reduced growth          dent strain suppressor free. <br />                                              temperature              Because mst1-L344S cells defects chromosome <br /> hsk1ts       Cdc7 kinase                   Reduced growth          segregation sensitive thiabendazole (see <br />                                              temperature           Figure 5), examined interactions genes required <br /> Drad2        FEN endonuclease              No effect               centromere heterochromatin assembly (Table 2, <br />                         DSB repair                                 centromere). We observed maximum-growth tem- <br /> Drad32       MRE11                         No effect               perature strikingly lower parents <br /> Drad50       RAD50                         Reduced growth          mst1-L344S combined Dswi6, Dclr3, and <br />                                              temperature           Dclr4, define genes required establish and <br /> Drhp51       RAD51                         Reduced growth          maintain centromeric heterochromatin (reviewed in <br />                                              temperature <br />                                                                    Ekwall 2007). However, observed defects in <br /> Drqh1        SGS1/Blm helicase             No effect <br /> Drti1        Rad52 homolog                 No effect               centromere telomere silencing mst1-L344S cells <br />                                                                    (data shown). This suggests heterochroma- <br />               Centromere/heterochromatin                           tin itself intact. We examined additional chromatin- <br /> Dcbh1        CENP-B homolog            No effect <br />                                                                    related proteins. No effect observed mst1-L344S <br /> Dclr3        Histone deacetylase       Reduced growth <br />                                          temperature               combined mutation histone acetyltransferase <br /> Dclr4        Histone methyltransferase Reduced growth              Dmst2 telomere-binding protein Dtaz1. <br />                                          temperature                  Identification potential Mst1 interaction partners <br /> Dhip1        HIRA homolog              No effect                   two-hybrid screening: SpMst1 interacts the <br /> Dmsc1        Putative demethylase      No effect                   actin-related protein SpAlp5/ScArp4, known subunit <br /> Dmst2        MYST acetyltransferase    No effect                   NuA4 complex (Minoda et al. 2005). Because <br /> Dswi6        HP1 homolog               Reduced growth <br />                                                                    evidence suggests Tip60 function indepen- <br />                                          temperature <br />                                                                    dently NuA4 complex (see Introduction), we <br />                           Other                                    sought identify additional proteins interact with <br /> Dskb1        Arginine methyltransferase    No effect               Mst1, using two-hybrid screen. We isolated number <br /> Dtaz1        Telomere-binding protein      No effect <br />                                                                    interacting gene products, summarized Table 3. <br />   Double mutants constructed growth rate relative            We isolated Rad22, S. pombe ortholog homol- <br /> to single mutants assessed streaking different       ogous DNA recombination protein Rad52. Rad22 <br /> temperatures.                                                      promotes strand-invasion step homologous re- <br />                                                                    combination. Fission yeast additional Rad22 <br /> using nitrogen starvation (supplemental Figure 1).                 family member, called Rti1. We constructed plasmids to <br /> When released nitrogen starvation complete               test Mst1 interacted two-hybrid system <br /> medium, mst-L344S cells completed replication                 Rti1 Rhp51. Although Rhp51 Rad22 <br /> similar timing wild type, suggesting in-              interact two-hybrid screens (e.g., Catlett <br /> activation mst1 mst1-L344S allele              Forsburg 2003), observed evidence for <br /> Dmst1 allele sufficient cells block DNA              interaction Mst1 Rhp51 Mst1 <br /> replication cell cycle.                               Rti1, suggesting interaction specific <br />   mst1-L344S genetically interacts replication,               (Figure 4A). Immunoprecipitation Rad22YFP using a <br /> recombination, heterochromatin mutants: We ex-                 cross-reacting anti-GFP antibody precipitates Mst1-V5 <br /> amined phenotypes double mutants,                     (Figure 4B); however, immunoprecipitation Mst1-V5 <br /> contained mst1-L344S mutations genes                 does precipitate Rad22-YFP (data shown). <br /> required DNA replication, double-strand break                     We isolated chromatin-related proteins po- <br /> repair, heterochromatin assembly (Table 2, Figure              tential Mst1 partners. The first, Hip1, putative histone <br /> 3). We tested double-mutant strains tem-             chaperone closely related ScHir1 human <br />                                               Mst1 Required Damage Response                                             763 <br />  <br />  <br />  <br />  <br />                                                                                      Figure 3.&#226;&#8364;&#8221;Genetic interactions. Double <br />                                                                                   mutants constructed serial dilu- <br />                                                                                   tions performed different tem- <br />                                                                                   peratures characterize synthetic <br />                                                                                   interactions. Equivalent numbers expo- <br />                                                                                   nentially growing cells diluted fivefold <br />                                                                                   incubated indicated temperatures. <br />  <br />  <br />  <br />  <br /> HIRA (Blackwell et al. 2004). The second, Cbh1,              budding yeast hsl7 esa1 mutants been <br /> one CenpB homologs fission yeast                shown synthetic interactions (Ruault Pillus <br /> associated centromere assembly (Baum                   2006). Two putative transcription factors iso- <br /> Clarke 2000; Nakagawa et al. 2002). We tested               lated. The res2/pct11 gene studied extensively <br /> three CenpB homologs interaction Mst1. Only            required G1/S-specific transcription (Tanaka <br /> Cbh1 showed specific interaction (Figure 4). We               et al. 1992; Zhu et al. 1994). taf1111 (also known taf11) is <br /> observed co-immunoprecipitation Cbh1                 putative TFIID subunit required induction genes <br /> and Mst1 soluble crosslinked lysates (data          nitrogen starvation sexual development <br /> shown).The gene isolated Msc1. Msc1           (Ueno et al. 2001). The remaining two-hybrid isolates in- <br /> was identified high-copy suppressor check-          cluded bud61, associated actin function <br /> <span id='am-86' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-87' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span>point mutation</span> Dchk1; E3 ligase activity             (Glynn et al. 2001), SEC18 homolog likely involved <br /> homology family demethylases, mutation              vesicular transport (SPAC1834.11c). The pre- <br /> disrupts histone acetylation (Ahmed et al. 2004; Dul        viously described fission yeast Mst1 interactor, Alp5 <br /> Walworth 2007). We constructed double mutants                   (Minoda et al. 2005), isolated screen, nor <br /> containing mst1-L344S Dhip1, Dcbh1, Dmsc1           subunits NuA4 complex. <br /> tested growth different temperatures. There              We observed little Mst1 (or Mst2) protein the <br /> difference growth characteristics double          soluble fraction (supplemental Figure 2a). This may <br /> mutants compared mst1-L344S cells alone. We             explain attempts confirm two-hybrid <br /> unable detect Msc1 soluble extracts               interactions using co-immunoprecipitation fission <br /> crosslinking protocol unsuccessful protein          yeast unsuccessful. Upon over- <br /> (data shown).                                               production Mst1, possible observe some <br />    The frequently isolated two-hybrid clone            interactions proteins; example, found <br /> skb11, encodes arginine methyltransferase,          Mcm2 interact Mst1 Mst2 when <br /> ortholog S. cerevisiae Hsl7 (Fujita et al. 1999; Shulewitz   partners overproduced (supplemental Figure <br /> et al. 1999; Lee et al. 2000). We observed obvious genetic   2b). However, observed interactions endog- <br /> interactions Dskb1 mst1-L344S, contrast       enous levels MCM proteins (data shown), even <br />  764                                                    E. B. Go&#194;&#180;mez et al. <br />  <br />                                                             TABLE 3 <br />                                              Genes isolated two-hybrid screening <br />  <br />         b-Galactosidase          Gene                                         Function <br />         1                     bud6/fat11      Actin-interacting protein (ScBUD6/AIP3 homolog) (Glynn et al. 2001) <br />         1                     cbh11           One fission yeast CENP-B proteins. Required for <br />                                                 heterochromatin function. No S. cerevisiae homolog <br />                                                 (Baum Clarke 2000; Nakagawa et al. 2002). <br />         1                     hip11           HIRA histone chaperone (ScHIR1 homolog) (Blackwell et al. 2004) <br />         11                    msc11           High-copy suppressor Dchk1 E3 ubiquitin ligase and <br />                                                 putative demethylase (Ahmed et al. 2004; Dul Walworth 2007). <br />                                                 No S. cerevisiae homolog. <br />         11                    rad221          Homologous recombination protein (ScRAD52 homolog) <br />                                                 (Ostermann et al. 1993) <br />         1                     res2/pct11      Transcription factor; forms complex Cdc10 (ScMBP1 homolog) <br />                                                 (Miyamoto et al. 1994). <br />         1                     sec181          Putative vesicle membrane fusion protein; uncharacterized ScSec18 <br />                                                 homolog. ORF: SPAC1834.11c. <br />         11                    skb11           Arginine methyltransferase (ScHSL7 homolog) (Gilbreth et al. 1996) <br />         11                    taf1111         Transcription factor; putative TFIID subunit required sexual <br />                                                 development SPAC2G11.14 (Ueno et al. 2001). <br />             b-Galactosidase qualitative measure degree activation two-hybrid system. <br />  <br />  <br /> though MCMs coprecipitated                  centromere kinetochore function TBZ <br /> those conditions (e.g., Sherman et al. 1998; Go&#194;&#180;mez et al.          sensitive; relate synthetic interactions <br /> 2002). We completed initial characterization                 observed heterochromatin mutants <br /> Mst1 protein showing acetylation core           (Table 2) physical interactions Cbh1 and <br /> histones observed IP&#226;&#8364;&#8220;acetylation assay                 Msc1, affect centromere function (Table 3). <br /> (supplemental Figure 3).                                               Mst1 required recovery HU arrest: HU <br />    mst1-L344S cells damage sensitive: Data suggest              causes nucleotide depletion, results replica- <br /> that NuA4 complex species recruited              tion fork stalling early S phase. Defects HU <br /> double-strand breaks association phos-                 response reflect failure arrest cell cycle <br /> phorylated histone H2AX (Downs et al. 2004; Robert                  acute treatment ability restore <br /> et al. 2006). Our data showing interaction                  replication recover HU removed (reviewed <br /> genetically physically Rad22 suggested                Branzei Foiani 2007). As shown Figure 6, A, <br /> Mst1 contribute repair DSBs influencing                B, D, mst1-L344S mutants arrest 1C DNA <br /> homologous recombination. Rad22 im-                   content modestly elongated morphology, do <br /> plicated recovery HU (Meister et al. 2005;                  wild-type Dcds1 cells, indicating checkpoint- <br /> Bailis et al. 2008). Therefore, examined mst1-L344S              mediated cell cycle arrest intact. As seen time <br /> mutants evidence role damage response.              course Figure 6, A B, wild-type cells complete S <br />    We examined growth mst1-L344S response               phase 30 min removing HU, Dcds1 <br /> to chronic exposure low levels DNA-damaging                   mutants severely delayed, consistent their <br /> agents (Figure 5A). We examined MMS (alkylating                     recovery defect. Wild-type cells proceed division <br /> agent delays replication), HU (inhibits ribonucleotide              enter cell cycle, seen increase of <br /> reductase causes fork stalling), camptothecin               4C population (reflecting binucleate cells com- <br /> (CPT; blocks topoisomerase causes DSBs). mst1-                  plete S phase completing previous cytokine- <br /> L344S cells sensitive HU MMS,                sis). This population seen Dcds1 mutants, <br /> CPT 29&#194;&#176;. mst1-L344S sensitive UV irradi-             defective recovery broad, <br /> ation (200 J/m2), g-irradiation (400 Gy) 25&#194;&#176;          heterogeneous peak DNA content later time <br /> (not shown). We observed sensitivity agents           points. The mst1 cells resemble wild type, suggesting <br /> increased temperature (data shown). The mst1-              mutants maintain restart replication <br /> L344S mutant extremely sensitive bleomycin                fork normally. This suggests sensitivity we <br /> 32&#194;&#176; (radio-mimetic; Figure 5B). These results con-              observed mst1 cells HU reflects defects later <br /> sistent known roles Mst1 family acetyltrans-               stage recovery. Similar results seen for <br /> ferases forms DSB repair. We observed               Drad22 Drhp51 (Meister et al. 2005; Bailis et al. <br /> substantial sensitivity thiabendazole, microtubule-            2008). Interestingly, mst1-L344S cells were <br /> destabilizing drug (Figure 5C). Mutations affecting                 sensitive HU chronic treatment (Figure 5), <br />                                                 Mst1 Required Damage Response                                            765 <br />  <br />                                                                       One measure DNA damage provided re- <br />                                                                    cruitment recombination proteins repair centers <br />                                                                    (Caspari et al. 2002; Du et al. 2003; Meister et al. 2005). <br />                                                                    During HU treatment, Rad22 focus formation can <br />                                                                    distinguish functions. In wild-type cells, Rad22 is <br />                                                                    recruited burst recovery phase cells <br />                                                                    released HU. In contrast, checkpoint <br />                                                                    mutants fail respond properly HU, collapsed <br />                                                                    replication forks cause DNA breaks Rad22 is <br />                                                                    recruited HU treatment (Caspari et al. 2002; <br />                                                                    Du et al. 2003; Meister et al. 2005; Bailis et al. 2008). To <br />                                                                    determine reduced Mst1 function causes DNA <br />                                                                    damage, examined Rad22YFP focus formation in <br />                                                                    wild-type mst1-L344S cells 32&#194;&#176; (Figure 6, E F). <br />                                                                    Due background fluorescence observed mst1- <br />                                                                    L344S mutant cells, detect Rad22YFP using <br />                                                                    indirect immunofluorescence nuclear spreads. We <br />                                                                    observed modest increase Rad22 focus formation in <br />                                                                    mst1-L344S cells HU treatment, relative wild- <br />                                                                    type cells (compare 0-hr time point mst1 wild type <br />                                                                    Figure 6F). This suggests cells suffer some <br />                                                                    endogenous damage. During acute HU <br />                                                                    treatment, Dcds1 cells dramatic increase in <br />                                                                    Rad22YFP focus formation, consistent replication <br />                                                                    fork collapse double-strand breaks (Meister et al. <br />   Figure 4.&#226;&#8364;&#8221;Two-hybrid interactions. (A) A prey construct          2005; Bailis et al. 2008). There change the <br /> containing Mst1 tested interaction full-<span id='am-6' about='xsp:length' typeof='owl:Thing'>length</span>        Rad22YFP foci mst1-L344S, consistent the <br /> Rad22, Rti1, Rhp51, Cbh1, Cbh2, Abp1 (see materials <br /> and methods). S. cerevisiae cells containing indicated plas-   model replication forks intact. <br /> mids tested growth SD media lacking histidine <br /> and adenine (right), compared control media containing <br /> both supplements. Growth selective media indicates                              DISCUSSION <br /> successful two-hybrid interaction. (B) Physical interaction be- <br /> tween Mst1-V5 Rad22-GFP. One milligram protein             The MYST family histone acetyltransferases is <br /> immunoprecipitated using 1 ml anti-GFP antibody. Input          associated diverse functions (reviewed Utley <br /> lanes: 50 mg total protein extract. GFP detected by <br /> ECL1 incubation 30 min. After 30 min, V5 blots ex-        Cote 2003). Previously, showed S. pombe <br /> posed film ECL1 (Pierce) incubation 30 sec.           mst11 essential gene encodes protein <br />                                                                    constitutively bound chromatin (Go&#194;&#180;mez et al. <br /> viability relatively unaffected acute treatment          2005). In report, characterized pheno- <br /> in HU block-and-release protocol (Figure 6C). This             types associated deletion mst11 tempera- <br /> suggests prolonged exposure HU leads                 ture-sensitive allele. <br /> accumulation growth defects, transient                     S. pombe Mst1 ortholog S. cerevisiae Esa1 and <br /> exposure single generation causes relatively little          human Tip60, catalytic subunit NuA4 family <br /> problem.                                                           histone acetyltransferases, referred as <br />  <br />  <br />                                                                                                 Figure 5.&#226;&#8364;&#8221;Damage sensitivity <br />                                                                                              mst1-L344S. (A) mst1 sensitivity <br />                                                                                              relative DNA-damaging agents. <br />                                                                                              Exponentially growing cells were <br />                                                                                              diluted fivefold YES plates with <br />                                                                                              indicated drugs grown 4 <br />                                                                                              days 29&#194;&#176;. (B) mst1 sensitive <br />                                                                                              bleomycin. Exponentially <br />                                                                                              growing cells diluted fivefold <br />                                                                                              YES plates grown 3 days at <br />                                                                                              32&#194;&#176;. (C) mst1 sensitive the <br />                                                                                              spindle poison TBZ. Exponen- <br />                                                                                              tially growing cells diluted <br />                                                                                              fivefold YES plates grown <br />                                                                                              3 days 32&#194;&#176;. <br />  766                                                     E. B. Go&#194;&#180;mez et al. <br />  <br />  <br />  <br />  <br />   Figure 6.&#226;&#8364;&#8221;mst1-L344S HU recovery defect. (A B) Time course DNA content following release HU 32&#194;&#176; (A) <br /> and 36&#194;&#176; (B) cells YES. Strains: wild type (FY11), mst1-L344S (FY2396), Dcds1 (FY865). (C) Viability cells following treat- <br /> ment 12 mm HU 32&#194;&#176;. Circles, wild type; triangles, Dcds1; squares, mst1. Open symbols, HU; solid symbols, 12 mm HU. <br /> (D) Morphology cells C released ( HU), (4 hr HU), release (2.5 hr rel). Cells fixed ethanol <br /> and stained DAPI. Bar, 10 mm. (E) Rad22YFP foci wild-type, Dcds1, mst1 cells 2 hr HU. Rad22YFP detected by <br /> immunofluorescence anti-GFP antibody, recognizes YFP variant. (F) Fraction cells Rad22YFP foci 0 or <br /> 2 hr incubation HU. -, negative control Rad22YFP. Strains: wild type (FY261), wild type Rad22YFP (FY3436), mst1 <br /> Rad22YFP (FY3369), Dcds1 Rad22YFP (FY3286). <br />  <br /> the KAT5 family (reviewed Squatrito et al. 2006;                  direct acetylation ATM p53 (Sun et al. 2005; Jiang <br /> Allis et al. 2007). In addition presumed effects           et al. 2006; Sykes et al. 2006; Tang et al. 2006), as <br /> transcriptional regulation (Smith et al. 1998; Allard                repair direct recruitment phosphorylated <br /> et al. 1999; Clarke et al. 1999, 2006; Durant Pugh               histone H2AX double-strand breaks (Bird et al. <br /> 2006), family implicated checkpoint control            2002; Downs et al. 2004; Tamburini Tyler 2005; <br />                                              Mst1 Required Damage Response                                              767 <br />  <br />                                                           TABLE 4 <br />                                     Summary processes associated mst11 function <br />  <br />                                                                                               mst1ts genetic    Mst1 physical <br /> Associated genes                                      Function                                 interactions     interactions <br />                                         Process: DNA double-strand break repair <br />                    Phenotypes: mst1ts mutant sensitivity DNA-damaging agents MMS, bleomycin, CPT <br /> rhp511                   Homologous recombination                                             Yes              None (Y2H) <br /> rad221                   Homologous recombination                                             Yes              Yes (Y2H, IP) <br />                                        Process: replication fork stability/recovery <br />     Evidence: mst1ts mutant sensitive chronic HU exposure. Increased Rad22 foci permissive conditions <br /> mcm21, mcm41, mcm71     MCM helicase, required replication fork assembly                   Yes (all)     Mcm2 (coIP) <br />                           elongation, recovery HU <br />  <br /> hsk11                    MCM kinase, required replication                                    Yes           ND <br />                           initiation HU recovery <br /> pol11                    DNA polymerase                                                        Yes           ND <br />                                               Process: centromere function <br />             Evidence: Dmst1, mst1ts mutants chromosome segregation defects. mst1ts mutant TBZ sensitive. <br />                           No silencing defect suggests heterochromatin intact mst1ts. <br /> clr31                   Pericentromere heterochromatin assembly, silencing,                       Yes (all)   ND <br /> clr41                      centromere cohesion, assembly Cnp1 histone central core <br /> swi61 <br /> cbh11                    Pericentromere heterochromatin assembly, assembly Cnp1               None          Yes (Y2H) <br />                            histone central core <br /> msc11                    Assembly Cnp1 histone central core, kinetochore assembly          None          Yes (Y2H) <br /> hir11                    Pericentromere heterochromatin silencing, kinetochore assembly          None          Yes (Y2H) <br />   IP, co-immunoprecipitation. Y2H, yeast two-hybrid interaction. ND, done. <br />  <br />  <br /> Robert et al. 2006). Data suggest family       data systems; centromere function; and <br /> HAT proteins acetylates histone H2AZ (Babiarz et al.             control mitotic entry. <br /> 2006; Keogh et al. 2006; Millar et al. 2006); histone         Even permissive temperatures, mst1-L344S cells are <br /> variant paradoxically associated regions         sensitive variety agents disrupt cell cycle, <br /> active gene transcription, centromeres           including bleomycin MMS, cause DNA <br /> and silent heterochromatin (reviewed Guillemette              damage double-strand breaks; hydroxyurea, which <br /> and Gaudreau 2006).                                              causes replication fork arrest; thiabendazole, which <br />    We characterized mutations affecting mst11 func-          causes spindle depolarization. The damage sensitivity <br /> tion. As reported previously, Dmst1 mutation              expected data numerous sources link <br /> causes lethality (Go&#194;&#180;mez et al. 2005). In report,        members family double-strand break <br /> show spores bearing disruption able              response. The recruitment KAT5 orthologs dou- <br /> germinate complete DNA replication. However,                 ble-strand breaks thought modulate break <br /> they highly disordered mitosis. We con-              response downregulate signal associ- <br /> structed temperature-sensitive mutant mst1-L344S,            ated H2AX phosphorylation recruiting the <br /> which contains lesion temperature-                 remodelers INO80 SWR1 alternative <br /> sensitive budding-yeast mutant. At restrictive tem-          histone H2AZ (Morrison et al. 2004; Tsukuda et al. <br /> perature, strain proficient replication,        2005; van Attikum Gasser 2005b; Papamichos- <br /> whether shifted asynchronously (Figure 2), released        Chronakis et al. 2006). <br /> <span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span> nitrogen starvation G1 (supplemental Figure 1),            When mst1-L344S combined mutations <br /> or released hydroxyurea S phase (Figure 6). We           Drhp51 (Rad51), Drad50, Drad22, cells severe <br /> observed pleiotropic phenotypes, including disordered            synthetic growth phenotype grow more <br /> mitosis, irregular segregation, high fraction cell      poorly parent. This particularly apparent <br /> cycle arrest septa. Cells arrest cell     Drad22 mst1-L344S; however, Drad22 is <br /> cycles mixed phenotype.                                prone accumulate suppressors fbh11 gene <br />    Our genetic analysis subsequent two-hybrid                (Osman et al. 2005), sure double <br /> screen suggest Mst1 impacts pathways (Table           mutant carrying suppressor. It possible <br /> 4): damage response, consistent                Drad22 mst1-L344S synthetically lethal we <br /> physiological analysis mst1-L344S mutant              able test convincingly. These <br />  768                                                    E. B. Go&#194;&#180;mez et al. <br />  <br /> genetic data suggest mst1-L344S requires func-              tromere function, leading synthetic effect the <br /> tional recombination pathway viability. To assess              double mutant. We loss silencing centro- <br /> whether mst1-L344S causes damage itself, examined               meres telomeres mst1 mutant, suggests that <br /> focus formation Rad22YFP mst1-L344S cells. There             Swi6-formed heterochromatin largely intact, and <br /> is increase foci mst1 compared wild type            aspect centromere assembly function <br /> 32&#194;&#176;, absence damaging treatment                 disrupted. A good candidate central core. Three <br /> (Figure 6F). This suggests attenuation Mst1                proteins identified two-hybrid tests <br /> function results form damage requires              linked central core kinetochore function: <br /> Rad22 repair, suggests Rad22-YFP can <br />                                                                    1. We isolated Msc1, identified high- <br /> be recruited sites damage absence Mst1. <br />                                                                       copy suppressor strain lacking checkpoint <br />    We observed physical association Mst1 and <br />                                                                       kinase Chk1 (Ahmed et al. 2004). A recent study shows <br /> Rad22 two-hybrid analysis co-immunoprecip- <br />                                                                       overexpression Msc1 rescues mutations of <br /> itation. This interaction specific Rad22, neither <br />                                                                       cnp1 (the specialized H3-Cen/CenpA centro- <br /> the Rad22 homolog Rti1 Rad51 homolog Rhp51 <br />                                                                       mere central core) addition Dchk1 this <br /> associate Mst1 two-hybrid tests (Figure 4). This <br />                                                                       suppression requires H2AZ histone variant en- <br /> could reflect role recruitment. Another possibility is <br />                                                                       coded pht11 (Ahmed et al. 2007). H2AZ linked <br /> that Mst1 acetylates Rad22 itself, Tip60 does ATM <br />                                                                       heterochromatin assembly centromere func- <br /> or p53, modulate behavior, investigate <br />                                                                       tion (reviewed Guillemette Gaudreau <br /> this future. <br />                                                                       2006) H2AZ known substrate KAT5 family <br />    Unexpectedly, mst1-L344S cells sensitive to <br />                                                                       HATs (Babiarz et al. 2006; Keogh et al. 2006; Millar <br /> hydroxyurea, causes replication fork arrest. The <br />                                                                       et al. 2006; Auger et al. 2008). <br /> mutant cells arrest properly 1C content without <br />                                                                    2. We isolated Cbh1, CenpB <br /> Rad22-YFP foci appear complete S phase follow- <br />                                                                       homologs fission yeast (Baum Clarke 2000; <br /> ing release, indicating checkpoint functions <br />                                                                       Nakagawa et al. 2002). The CenpB homologs are <br /> and replication fork intact; instead, have <br />                                                                       proposed affect centromere structure (Nakagawa <br /> modestly reduced viability reminiscent Drhp51 or <br />                                                                       et al. 2002) and, metazoans, CenpB involved in <br /> Drad22 mutants (Meister et al. 2005; Bailis et al. 2008). <br />                                                                       recruitment H3-Cen/CenpA (reviewed Mel- <br /> We observe HU sensitivity mst1-L344S HU <br />                                                                       lone Allshire 2003). <br /> differs cells exposed low levels prolonged <br />                                                                    3. We isolated Hip1, HIRA/Hir1 homolog. HIR <br /> time, causes cell death (chronic exposure, shown <br />                                                                       proteins histone chaperones required assem- <br /> in Figure 5), exposed higher levels <br />                                                                       bly chromatin outside S phase (reviewed in <br /> over single generation (acute exposure, shown in <br />                                                                       Gunjan et al. 2005). In fission yeast, deletion Hip1 <br /> Figure 6). Similar results observed fork <br />                                                                       causes chromosome loss, TBZ sensitivity, disrup- <br /> protection mutant Dswi1 (ScTof1), survives acute <br />                                                                       tion centromere silencing (Blackwell et al. <br /> exposure dies chronic exposure (Sommariva <br />                                                                       2004). <br /> et al. 2005). It possible HU sensitivity is <br /> related interaction mst1 mutations           HIR proteins S. cerevisiae implicated normal <br /> recombination genes; however, Rad22-YFP recruitment                kinetochore function (Sharp et al. 2002). In contrast to <br /> appears largely normal mst1 cells treated HU.              heterochromatin mutants, synthetic <br />    A second broad functional area defined two-              growth phenotypes mst1 combined mutants <br /> hybrid genetic interactions centromere assembly.            Dcbh1, Dmsc1, Dhip1. This consistent with <br /> Fission yeast centromeres large complex ele-               mst11 acting pathway genes. Together <br /> ments, central core surrounded flanking                  segregation defects mst1-L344S, two <br /> repeats form peri-centromeric heterochromatin                 hybrid interactions suggest Mst1 affect the <br /> (reviewed Ekwall 2007; Grewal Jia 2007).                    centromere central core. We examining mitotic <br /> Mutations disrupting heterochromatin, central              phenotypes mst1 determine this <br /> core, kinetochore typically sensitivity             occur role H2AZ process. <br /> spindle poisons TBZ, high fraction                   There indirect evidence Mst1 affect <br /> chromosome loss disordered mitoses, ob-                  mitotic entry. We isolated Skb1 methyltransferase <br /> served (Figures 1, 2, 5). Our genetic analysis                 two-hybrid screen. Budding yeast Hsl7, the <br /> suggests mutations disrupt normal pericentro-            ortholog Skb1, identified mutations of <br /> meric heterochromatin function Dclr3 (histone              hsl7 synthetically lethal histone tail mutants <br /> deacetylase), Dclr4 (methyltransferase), Dswi6 (het-            (Ma et al. 1996); genetic interactions ScDhsl7 <br /> erochromatin protein 1) reduce growth tempera-                 ScEsa1 suggest histone tail modifications are <br /> ture mst1-L344S quite substantially. The simplest               important normal mitosis Swe1 kinase is <br /> interpretation epistasis experiment mst11          activated (Ruault Pillus 2006). While ob- <br /> affects separate pathway Clr4 Swi6 cen-              served genetic interaction SpDskb1 our <br />                                                        Mst1 Required Damage Response                                                          769 <br />  <br /> temperature-sensitive allele mst1, physical in-                      San Diego, hospitality E.B.G. work. <br /> teraction observed, genetic                       R.L.N. supported Department Defense W81XWH-05-1-0391 <br />                                                                             (X. Chen). This work supported National Institutes Health <br /> data budding yeast, suggests role for <br />                                                                             grant R01 GM059321 S.L.F. <br /> histone tail modification proper regulation of <br /> mitotic entry species. Another link CDK <br /> activation provided isolation Hip1 a <br />                                                                                                   LITERATURE CITED <br /> putative interactor. The Dhip1 mutation synthetically <br /> lethality cdc25-22, temperature-sensitive                   Ahmed, S., C. Palermo, S. Wan N. C. Walworth, 2004 A novel <br />                                                                                 protein similarities Rb binding protein 2 compensates for <br /> mutation mitotic inducer Cdc25 opposes                              loss Chk1 function affects histone modification fission <br /> Wee1 (ScSwe1) (Blackwell et al. 2004; reviewed                               yeast. Mol. Cell. Biol. 24: 3660&#226;&#8364;&#8220;3669. <br /> Karlsson-Rosenthal Millar 2006).                                        Ahmed, S., B. E. Dul, X. Qiu N. C. Walworth, 2007 Msc1 acts <br />                                                                                 histone H2A.Z promote chromosome stability in <br />    We did isolate NuA4 complex proteins                          Schizosaccharomyces pombe. Genetics 177: 1487&#226;&#8364;&#8220;1497. <br /> in two-hybrid screen, previously                           Allard, S., R. T. Utley, J. Savard, A. Clarke, P. Grant et al., <br /> characterized SpAlp5/ScArp4 known interact                           1999 NuA4, essential transcription adaptor/histone H4 ace- <br />                                                                                 tyltransferase complex containing Esa1p ATM-related co- <br /> Mst1 co-immunoprecipitation (Minoda et al. 2005). <br />                                                                                 factor Tra1p. EMBO J. 18: 5108&#226;&#8364;&#8220;5119. <br /> This surprising two-hybrid screen identifies                  Allis, C. D., S. L. Berger, J. Cote, S. Dent, T. Jenuwien et al., <br /> binary associations, interactions                        2007 New nomenclature chromatin modifying enzymes. <br /> large NuA4 complex likely indirect (Auger                             Cell 131: 633&#226;&#8364;&#8220;636. <br />                                                                             Auger, A., L. Galarneau, M. Altaf, A. Nourani, Y. Doyon et al., <br /> et al. 2008). We speculate structure two-                       2008 Eaf1 platform NuA4 molecular assembly that <br /> hybrid bait construct, amino-terminus                           evolutionarily links chromatin acetylation ATP-dependent ex- <br /> Mst1 fused DNA-binding domain Gal4,                            change histone H2A variants. Mol. Cell. Biol. 28: 2257&#226;&#8364;&#8220;2270. <br />                                                                             Babiarz, J. E., J. E. Halley J. Rine, 2006 Telomeric heterochro- <br /> interfere assembly NuA4 subunits Mst1.                         matin boundaries require NuA4-dependent acetylation his- <br /> Conversely, did observe co-immunoprecipitation                           tone variant H2A.Z Saccharomyces cerevisiae. Genes Dev. <br /> of epitope-tagged Mst1 two-hybrid                              20: 700&#226;&#8364;&#8220;710. <br />                                                                             Bailis, J. M., D. D. Luche, T. Hunter S. L. Forsburg, <br /> clones. This reflect chromatin association                         2008 MCM proteins interact checkpoint recombina- <br /> proteins involved, temporally spatially limiting asso-                       tion proteins promote S phase genome stability. Mol. Cell. <br /> ciation, indicate C-terminal epitope                          Biol. 28: 1724&#226;&#8364;&#8220;1738. <br />                                                                             Baum, M., L. Clarke, 2000 Fission yeast homologs human <br /> tag Mst1 available protein a <br />                                                                                 CENP-B redundant functions affecting cell growth and <br /> complex factors. A detailed structure&#226;&#8364;&#8220;                          chromosome segregation. Mol. Cell. Biol. 20: 2852&#226;&#8364;&#8220;2864. <br /> function analysis localization studies progress                  Bird, A. W., D. Y. Yu, M. G. Pray-Grant, Q. Qiu, K. E. Harmon et al., <br /> to resolve observations.                                                  2002 Acetylation histone H4 Esa1 required DNA <br />                                                                                 double-strand break repair. Nature 419: 411&#226;&#8364;&#8220;415. <br />    The interactions observed, addition                    Blackwell, C., K. A. Martin, A. Greenall, A. Pidoux, R. C. <br /> phenotypes associated mst1 mutants, suggest                            Allshire et al., 2004 The Schizosaccharomyces pombe HIRA-like <br /> that Mst1 affects variety cell functions. Importantly,                     protein Hip1 required periodic expression histone <br />                                                                                 genes contributes function complex centromeres. <br /> the recent discovery Tip60 acetylate nonhis-                           Mol. Cell. Biol. 24: 4309&#226;&#8364;&#8220;4320. <br /> tone proteins absence NuA4 complex (e.g.,                     Boudreault, A. A., D. Cronier, W. Selleck, N. Lacoste, R. T. <br /> Sun et al. 2005; Sykes et al. 2006; Tang et al. 2006; Kim                       Utley et al., 2003 Yeast enhancer polycomb defines global <br />                                                                                 Esa1-dependent acetylation chromatin. Genes Dev. 17: 1415&#226;&#8364;&#8220; <br /> et al. 2007) indicates identification Mst1 a <br />                                                                                 1428. <br /> histone acetyltransferase limiting defini-                     Branzei, D., M. Foiani, 2007 Interplay replication check- <br /> tion. We speculate pleiotropic effects mst1                         points repair proteins stalled replication forks. DNA <br /> mutants indicate variety functions                             Rep. 6: 994&#226;&#8364;&#8220;1003. <br />                                                                             Burke, T. W., J. Cooks-Gowen, M. Asano J. R. Nevins, <br /> directly modulated Mst1 catalytic activity, neces-                       2001 Replication factors MCM2 ORC1 interact his- <br /> sarily linked histone acetylation. Our study suggests                        tone acetyltransferase HBO1. J. Biol. Chem. 276: 15397&#226;&#8364;&#8220;15408. <br /> that Mst1 functions replication fork stabil-                    Caspari, T., J. M. Murray A. M. Carr, 2002 Cdc2-cyclin B ki- <br />                                                                                 nase activity links Crb2 amd Rqh1-topoisomerase III. Genes Dev. <br /> ity centromere assembly, experiments                             16: 1195&#226;&#8364;&#8220;1208. <br /> underway determine effects medi-                       Catlett, M. G., S. L. Forsburg, 2003 Schizosaccharomyces <br /> ated histone modifications aspects Mst1                       pombe Rdh54 (TID1) acts Rhp54 (RAD54) repair meiotic <br />                                                                                 double-strand breaks. Mol. Biol. Cell 14: 4707&#226;&#8364;&#8220;4720. <br /> function.                                                                   Clarke, A. S., J. E. Lowell, S. J. Jacobson L. Pillus, <br />   We thank Louise Clarke, Greg Freyer, Stevan Marcus, Paul Russell,             1999 Esa1p essential histone acetyltransferase required <br /> Nancy Walworth, Simon Whitehall strains. We thank Vanessa               cell cycle progression. Mol. Cell. Biol. 19: 2515&#226;&#8364;&#8220;2526. <br /> Angeles Paula Tran assistance genetics growth assays;      Clarke, A. S., E. Samal L. Pillus, 2006 Distinct roles the <br />                                                                                 essential MYST family HAT Esa1p transcriptional silencing. <br /> Su Lam help two-hybrid analysis; Marc Green for <br />                                                                                 Mol. Biol. Cell 17: 1744&#226;&#8364;&#8220;1757. <br /> microscopy assistance; Jeff Hodson, John Marlett, Ji-Ping <br />                                                                             Deckert, J., K. Struhl, 2001 Histone acetylation promoters <br /> Yuan lab support. We thank Joaquin Espinosa assistance             differentially affected specific activators repressors. Mol. <br /> histone acetyltransferase assays. We grateful Ruben Petreaca         Cell. Biol. 21: 2726&#226;&#8364;&#8220;2735. <br /> careful reading manuscript members Forsburg lab        la Cruz, X., S. Lois, S. Sanchez-Molina M. A. Martinez- <br /> for helpful comments course study. We                Balbas, 2005 Do protein motifs read histone <span id='am-320' about='Thesaurus:code' typeof='owl:Thing'>code</span>? Bio- <br /> thank Lorraine Pillus Pillus lab University California,       Essays 27: 164&#226;&#8364;&#8220;175. <br />  770                                                                  E. B. Go&#194;&#180;mez et al. <br />  <br /> Downs, J. A., S. Allard, O. Jobin-Robitaille, A. Javaheri, A. Auger              Keogh, M. C., T. A. Mennella, C. Sawa, S. Berthelet, N. J. Krogan <br />     et al., 2004 Binding chromatin-modifying activities phos-                  et al., 2006 The Saccharomyces cerevisiae histone H2A variant <br />     phorylated histone H2A DNA damage sites. Mol. Cell 16:                        Htz1 acetylated NuA4. Genes Dev. 20: 660&#226;&#8364;&#8220;665. <br />     979&#226;&#8364;&#8220;990.                                                                     Kim, M. Y., E. J. Ann, J. Y. Kim, J. S. Mo, J. H. Park et al., 2007 Tip60 <br /> Du, L. L., T. M. Nakamura, B. A. Moser P. Russell, 2003 Retention                histone acetyltransferase acts negative regulator Notch1 <br />     recruitment Crb2 double-strand breaks requires Rad1                signaling means acetylation. Mol. Cell. Biol. 27: 6506&#226;&#8364;&#8220;6519. <br />     Rad3 complexes. Mol. Cell. Biol. 23: 6150&#226;&#8364;&#8220;6158.                          Kouzarides, T., 2007 Chromatin modifications function. <br /> Dul, B. E., N. C. Walworth, 2007 The plant homeodomain                           Cell 128: 693&#226;&#8364;&#8220;705. <br />     fingers fission yeast Msc1 exhibit E3 ubiquitin ligase activity.          Lafon, A., C. S. Chang, E. M. Scott, S. J. Jacobson L. Pillus, <br />     J. Biol. Chem. 282: 18397&#226;&#8364;&#8220;18406.                                                 2007 MYST opportunities growth control: yeast genes illu- <br /> Durant, M., B. F. Pugh, 2006 Genome-wide relationships be-                       minate human cancer gene functions. Oncogene 26: 5373&#226;&#8364;&#8220;5384. <br />     tween TAF1 histone acetyltransferases Saccharomyces cer-              Lee, J. H., J. R. Cook, B. P. Pollack, T. G. Kinzy, D. Norris et al., <br />     evisiae. Mol. Cell. Biol. 26: 2791&#226;&#8364;&#8220;2802.                                         2000 Hsl7p, yeast homologue human JBP1, protein <br /> Early, A., L. S. Drury J. F. Diffley, 2004 Mechanisms involved                   methyltransferase. Biochem. Biophys. Res. Commun. 274: 105&#226;&#8364;&#8220; <br />     regulating DNA replication origins cell cycle               111. <br />     response DNA damage. Philos. Trans. R. Soc. Lond. B Biol. Sci.            Lee, K. K., J. L. Workman, 2007 Histone acetyltransferase com- <br />     359: 31&#226;&#8364;&#8220;38.                                                                      plexes: size doesn&#226;&#8364;&#8482;t fit all. Nat. Rev. Mol. Cell Biol. 8: 284&#226;&#8364;&#8220;295. <br /> Eberharter, A., R. Ferreira P. Becker, 2005 Dynamic chro-                    Le Masson, I., D. Y. Yu, K. Jensen, A. Chevalier, R. Courbeyrette <br />     matin: concerted nucleosome remodelling acetylation. Biol.                   et al., 2003 Yaf9, novel NuA4 histone acetyltransferase sub- <br />     Chem. 386: 745&#226;&#8364;&#8220;751.                                                              <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span>, required cellular response spindle stress in <br /> Ekwall, K., 2007 Epigenetic control centromere behavior. Annu.                    yeast. Mol. Cell. Biol. 23: 6086&#226;&#8364;&#8220;6102. <br />     Rev. Genet. 41: 63&#226;&#8364;&#8220;81.                                                       Leverson, J. D., H. K. Huang, S. L. Forsburg T. Hunter, <br /> Forsburg, S. L., N. Rhind, 2006 Basic methods fission yeast.                 2002 The Schizosaccharomyces pombe aurora-related kinase <br />     Yeast 23: 173&#226;&#8364;&#8220;183.                                                               Ark1 interacts inner centromere protein Pic1 medi- <br /> Foster, E. R., J. A. Downs, 2005 Histone H2A phosphorylation                     ates chromosome segregation cytokinesis. Mol. Biol. Cell 13: <br />     DNA double-strand break repair. FEBS J. 272: 3231&#226;&#8364;&#8220;3240.                       1132&#226;&#8364;&#8220;1143. <br /> Fujita, A., A. Tonouchi, T. Hiroko, F. Inose, T. Nagashima et al.,               Ma, X. J., Q. Lu M. Grunstein, 1996 A search proteins that <br />     1999 Hsl7p, negative regulator Ste20p protein kinase                     interact genetically histone H3 H4 amino termini un- <br />     Saccharomyces cerevisiae filamentous growth-signaling path-                  covers novel regulators Swe1 kinase Saccharomyces cerevi- <br />     way. Proc. Natl. Acad. Sci. USA 96: 8522&#226;&#8364;&#8220;8527.                                   siae. Genes Dev. 10: 1327&#226;&#8364;&#8220;1340. <br /> Galarneau, L., A. Nourani, A. A. Boudreault, Y. Zhang, L. Heliot et al.,         Meister, P., A. Taddei, L. Vernis, M. Poidevin, S. M. Gasser et al., <br />     2000 Multiple links NuA4 histone acetyltransferase                   2005 Temporal separation replication recombination re- <br />     complex epigenetic control transcription. Mol. Cell 5: 927&#226;&#8364;&#8220;937.           quires intra-S checkpoint. J. Cell Biol. 168: 537&#226;&#8364;&#8220;544. <br /> Gilbreth, M., P. Yang, D. Wang, J. Frost, A. Polverino et al.,                   Mellone, B. G., R. C. Allshire, 2003 Stretching it: putting the <br />     1996 The highly conserved skb1 gene encodes protein                       CEN(P-A) centromere. Curr. Opin. Genet. Dev. 13: 191&#226;&#8364;&#8220;198. <br />     interacts Shk1, fission yeast Ste20/PAK homolog. Proc.                Millar, C. B., M. Grunstein, 2006 Genome-wide patterns of <br />     Natl. Acad. Sci. USA 93: 13802&#226;&#8364;&#8220;13807.                                            histone modifications yeast. Nat. Rev. Mol. Cell Biol. 7: 657&#226;&#8364;&#8220;666. <br /> Glynn, J. M., R. J. Lustig, A. Berlin F. Chang, 2001 Role                 Millar, C. B., F. Xu, K. Zhang M. Grunstein, 2006 Acetylation <br />     bud6p tea1p interaction actin microtu-                    H2AZ Lys 14 associated genome-wide gene activity in <br />     bules establishment cell polarity fission yeast. Curr.             yeast. Genes Dev. 20: 711&#226;&#8364;&#8220;722. <br />     Biol. 11: 836&#226;&#8364;&#8220;845.                                                           Minoda, A., S. Saitoh, K. Takahashi T. Toda, 2005 BAF53/ <br /> Go&#194;&#180;mez, E. G., S. L. Forsburg, 2003 Analysis Schizosaccharomy-                Arp4 homolog Alp5 fission yeast required histone H4 <br />     ces pombe cell cycle, pp. 93&#226;&#8364;&#8220;111 Cell Cycle Checkpoint Protocols, edi-        acetylation, kinetochore-spindle attachment, gene silencing <br />     ted H. G. Lieberman. Humana Press, Tatawa, NJ.                                centromere. Mol. Biol. Cell 16: 316&#226;&#8364;&#8220;327. <br /> Go&#194;&#180;mez, E. G., M. G. Catlett S. L. Forsburg, 2002 Different                  Miyamoto, M., K. Tanaka H. Okayama, 1994 res21, new <br />     phenotypes vivo associated ATPase motif mutations                    member cdc101/SWI4 family, controls &#226;&#8364;&#732;start&#226;&#8364;&#8482; mitotic <br />     Schizosaccharomyces pombe minichromosome maintenance pro-                     meiotic cycles fission yeast. EMBO J. 13: 1873&#226;&#8364;&#8220;1880. <br />     teins. Genetics 160: 1305&#226;&#8364;&#8220;1318.                                              Moreno, S., A. Klar P. Nurse, 1991 Molecular genetic analysis <br /> Go&#194;&#180;mez, E. B., J. Espinosa S. L. Forsburg, 2005 S. pombe mst21                   fission yeast Schizosaccharomyces pombe. Methods Enzymol. <br />     encodes MYST-family histone acetyltransferase required                     194: 795&#226;&#8364;&#8220;823. <br />     telomere silencing. Mol. Cell. Biol. 25: 8887&#226;&#8364;&#8220;8903.                          Morrison, A. J., X. Shen, 2005 DNA repair context of <br /> Grewal, S. I., S. Jia, 2007 Heterochromatin revisited. Nat. Rev.                 chromatin. Cell Cycle 4: 568&#226;&#8364;&#8220;571. <br />     Genet. 8: 35&#226;&#8364;&#8220;46.                                                             Morrison, A. J., J. Highland, N. J. Krogan, A. Arbel-Eden, J. F. <br /> Guillemette, B., L. Gaudreau, 2006 Reuniting contrast-                       Greenblatt et al., 2004 INO80 gamma-H2AX interaction <br />     ing functions H2A.Z. Biochem. Cell Biol. 84: 528&#226;&#8364;&#8220;535.                         links ATP-dependent chromatin remodeling DNA damage re- <br /> Gunjan, A., J. Paik A. Verreault, 2005 Regulation histone                     pair. Cell 119: 767&#226;&#8364;&#8220;775. <br />     synthesis nucleosome assembly. Biochimie 87: 625&#226;&#8364;&#8220;635.                    Nakagawa, H., J. K. Lee, J. Hurwitz, R. C. Allshire, J. Nakayama <br /> Iizuka, M., B. Stillman, 1999 Histone acetyltransferase HBO1                     et al., 2002 Fission yeast CENP-B homologs nucleate centro- <br />     interacts ORC1 subunit human initiator protein.                  meric heterochromatin promoting heterochromatin-specific <br />     J. Biol. Chem. 274: 23027&#226;&#8364;&#8220;23034.                                                 histone tail modifications. Genes Dev. 16: 1766&#226;&#8364;&#8220;1778. <br /> Iizuka, M., T. Matsui, H. Takisawa M. M. Smith, 2006 Reg-                    Osman, F., J. Dixon, A. R. Barr M. C. Whitby, 2005 The F-Box <br />     ulation replication licensing acetyltransferase Hbo1. Mol.                 DNA helicase Fbh1 prevents Rhp51-dependent recombination <br />     Cell. Biol. 26: 1098&#226;&#8364;&#8220;1108.                                                       mediator proteins. Mol. Cell. Biol. 25: 8084&#226;&#8364;&#8220;8096. <br /> Ikura, T., V. V. Ogryzko, M. Grigoriev, R. Groisman, J. Wang et al.,             Ostermann, K., A. Lorentz H. Schmidt, 1993 The fission <br />     2000 Involvement TIP60 histone acetylase complex                       yeast rad22 gene, having function mating-type switching <br />     DNA repair apoptosis. Cell 102: 463&#226;&#8364;&#8220;473.                                     repair DNA damages, encodes protein homolog to <br /> Jenuwein, T., C. D. Allis, 2001 Translating histone code.                    rad52 Saccharomyces cerevisiae. Nucleic Acids Res. 21: <br />     Science 293: 1074&#226;&#8364;&#8220;1080.                                                          5940&#226;&#8364;&#8220;5944. <br /> Jiang, X., Y. Sun, S. Chen, K. Roy B. D. Price, 2006 The FATC                Pankratz, D. G., S. L. Forsburg, 2005 Meiotic S-phase damage <br />     domains PIKK proteins functionally equivalent partic-                 activates recombination checkpoint arrest. Mol. Biol. <br />     ipate Tip60-dependent activation DNA-PKcs ATM.                     Cell 16: 1651&#226;&#8364;&#8220;1660. <br />     J. Biol. Chem. 281: 15741&#226;&#8364;&#8220;15746.                                             Papamichos-Chronakis, M., J. E. Krebs C. L. Peterson, <br /> Karlsson-Rosenthal, C., J. B. Millar, 2006 Cdc25: mecha-                         2006 Interplay Ino80 Swr1 chromatin remodeling <br />     nisms checkpoint inhibition recovery. Trends Cell Biol.                   enzymes regulates cell cycle checkpoint adaptation response <br />     16: 285&#226;&#8364;&#8220;292.                                                                     DNA damage. Genes Dev. 20: 2437&#226;&#8364;&#8220;2449. <br />                                                       Mst1 Required Damage Response                                                     771 <br />  <br /> Robert, F., S. Hardy, Z. Nagy, C. Baldeyron, R. Murr et al.,               Sykes, S. M., H. S. Mellert, M. A. Holbert, K. Li, R. Marmorstein <br />     2006 The transcriptional histone acetyltransferase cofactor                 et al., 2006 Acetylation p53 DNA-binding domain regu- <br />     TRRAP associates MRN repair complex plays                    lates apoptosis induction. Mol. Cell 24: 841&#226;&#8364;&#8220;851. <br />     role DNA double-strand break repair. Mol. Cell. Biol. 26:           Tamburini, B. A., J. K. Tyler, 2005 Localized histone acetyla- <br />     402&#226;&#8364;&#8220;412.                                                                    tion deacetylation triggered homologous recombina- <br /> Ruault, M., L. Pillus, 2006 Chromatin-modifiying enzymes                    tion pathway double-strand DNA repair. Mol. Cell. Biol. 25: <br />     essential Saccharomyces cerevisiae morphogenesis               4903&#226;&#8364;&#8220;4913. <br />     checkpoint constitutively activated. Genetics 174: 1135&#226;&#8364;&#8220;            Tanaka, K., K. Okazaki, N. Okazaki, T. Ueda, A. Sugiyama et al., <br />     1149.                                                                       1992 A new cdc gene required S phase entry Schizosacchar- <br /> Selleck, W., I. Fortin, D. Sermwittayawong, J. Cote S. Tan,                 omyces pombe encodes protein similar cdc101 SWI4 <br />     2005 The Saccharomyces cerevisiae Piccolo NuA4 histone ace-                 gene products. EMBO J. 11: 4923&#226;&#8364;&#8220;4932. <br />     tyltransferase complex requires Enhancer Polycomb A do-         Tang, Y., J. Luo, W. Zhang W. Gu, 2006 Tip60-dependent acet- <br />     main chromodomain acetylate nucleosomes. Mol. Cell.                  ylation p53 modulates decision cell-cycle arrest <br />     Biol. 25: 5535&#226;&#8364;&#8220;5542.                                                        apoptosis. Mol. Cell 24: 827&#226;&#8364;&#8220;839. <br /> Sharp, J. A., A. A. Franco, M. A. Osley P. D. Kaufman,                 Tsukuda, T., A. B. Fleming, J. A. Nickoloff M. A. Osley, <br />     2002 Chromatin assembly factor I Hir proteins contribute                2005 Chromatin remodelling DNA double-strand break <br />     building functional kinetochores S. cerevisiae. Genes Dev. 16:        site Saccharomyces cerevisiae. Nature 438: 379&#226;&#8364;&#8220;383. <br />     85&#226;&#8364;&#8220;100.                                                                Ueno, M., R. Kurokawa, H. Renauld, K. Watanabe, T. Ushimaru <br /> Sherman, D. A., S. G. Pasion S. L. Forsburg, 1998 Multiple                  et al., 2001 Schizosaccharomyces pombe taf11 required for <br />     domains fission yeast Cdc19p (MCM2) required as-             nitrogen starvation-induced sexual development enter- <br />     sociation core MCM complex. Mol. Biol. Cell 9: 1833&#226;&#8364;&#8220;               ing dormant GO state. Curr. Genet. 38: 307&#226;&#8364;&#8220;313. <br />     1845.                                                                  Utley, R. T., J. Cote, 2003 The MYST family histone acetyl- <br /> Shogren-Knaak, M., H. Ishii, J. M. Sun, M. J. Pazin, J. R. Davie et al.,        transferases. Curr. Top. Microbiol. Immunol. 274: 203&#226;&#8364;&#8220;236. <br />     2006 Histone H4&#226;&#8364;&#8220;K16 acetylation controls chromatin structure           van Attikum, H., S. M. Gasser, 2005a ATP-dependent chroma- <br />     protein interactions. Science 311: 844&#226;&#8364;&#8220;847.                             tin remodeling DNA double-strand break repair. Cell Cycle <br /> Shulewitz, M. J., C. J. Inouye J. Thorner, 1999 Hsl7 localizes              4: 1011&#226;&#8364;&#8220;1014. <br />     septin ring serves adapter regulatory              van Attikum, H., S. M. Gasser, 2005b The histone code at <br />     pathway relieves tyrosine phosphorylation Cdc28 protein             DNA breaks: A guide repair? Nat. Rev. Mol. Cell Biol. 6: <br />     kinase Saccharomyces cerevisiae. Mol. Cell. Biol. 19: 7123&#226;&#8364;&#8220;              757&#226;&#8364;&#8220;765. <br />     7137.                                                                  Vogelauer, M., L. Rubbi, I. Lucas, B. J. Brewer M. Grunstein, <br /> Smith, E. R., A. Eisen, W. Gu, M. Sattah, A. Pannuti et al.,                    2002 Histone acetylation regulates time replication ori- <br />     1998 ESA1 histone acetyltransferase essential              gin firing. Mol. Cell 10: 1223&#226;&#8364;&#8220;1233. <br />     growth yeast. Proc. Natl. Acad. Sci. USA 95: 3561&#226;&#8364;&#8220;3565.             Yu, Y., Y. Teng, H. Liu, S. H. Reed R. Waters, 2005 UV irradi- <br /> Sommariva, E., T. K. Pellny, N. Karahan, S. Kumar, J. A. Huberman               ation stimulates histone acetylation chromatin remodeling <br />     et al., 2005 Schizosaccharomyces pombe Swi1, Swi3, Hsk1             repressed yeast locus. Proc. Natl. Acad. Sci. USA 102: <br />     components novel S-phase response pathway alkylation                8650&#226;&#8364;&#8220;8655. <br />     damage. Mol. Cell. Biol. 25: 2770&#226;&#8364;&#8220;2784.                                Zhu, Y., T. Takeda, K. Nasmyth N. Jones, 1994 pct11, which <br /> Squatrito, M., C. Gorrini B. Amati, 2006 Tip60 DNA dam-                  encodes new DNA-binding partner p85cdc10, required for <br />     age response growth control: tricks HAT.                    meiosis fission yeast Schizosaccharomyces pombe. Genes <br />     Trends Cell Biol. 16: 433&#226;&#8364;&#8220;442.                                              Dev. 8: 885&#226;&#8364;&#8220;898. <br /> Sun, Y., X. Jiang, S. Chen, N. Fernandes B. D. Price, 2005 A <br />     role Tip60 histone acetyltransferase  <br /> </body></html>