<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>An insect symbiosis influenced by <br /> bacterium-specific polymorphisms in <br /> outer-membrane protein A <br /> Brian L. Weiss*, Yineng Wu*, Jonathon J. Schwank*, Nicholas S. Tolwinski&#226;&#8364;&#160;, Serap Aksoy*&#226;&#8364;&#161; <br /> *Department Epidemiology Public Health, Section Vector Biology, Yale University School Medicine, LEPH 606, 60 College Street, <br /> New Haven, CT 06520; &#226;&#8364;&#160;Program Developmental Biology, Memorial Sloan&#226;&#8364;&#8220;Kettering Cancer Center, New York, NY 10065 <br />  <br /> Edited David L. Denlinger, Ohio State University, Columbus, OH, approved August 12, 2008 (received review June 13, 2008) <br />  <br /> Beneficial bacterial symbioses ubiquitous nature. However,          species undergone minimal coevolution hosts, <br /> functional molecular basis host tolerance resident symbiotic         signifying relatively recent establishment sym- <br /> microbes, contrast resistance closely related bacteria       biosis (8). Sodalis cultivated vitro, genetic <br /> recognized foreign, remain largely unknown. We used tsetse              transformation developed introduce and <br /> fly (Glossina morsitans), depends symbiotic flora fecun-          express foreign gene products vitro vivo (9). <br /> dity limited exposure foreign microbes, investigate          Furthermore, distant tsetse species reconstituted with <br /> tolerance phenomenon exhibited symbiosis. We examined                   different Sodalis strains subsequently mimic natural infec- <br /> the potential role bacterium-specific polymorphisms present          tions (8). <br /> major bacterial surface protein, outer-membrane protein A (OmpA), <br />                                                                                   Sodalis proliferates despite residing tsetse&#226;&#8364;&#8482;s hostile gut and <br /> on host infection outcomes. Tsetse successfully superinfected <br />                                                                                hemolymph environment activate and/or ex- <br /> with mutualistic facultative symbiont, Sodalis glossinidius, <br />                                                                                posed immune effectors. Analysis symbiont&#226;&#8364;&#8482;s genome <br /> whereas infections Escherichia coli K12 lethal. In contrast, <br />                                                                                sequence indicates immunogenic components its <br /> tsetse resistant E. coli OmpA mutant strain, whereas <br /> recombinant Sodalis expressing E. coli OmpA pathogenic. <br />                                                                                cell membrane altered, including truncated lipopolysac- <br /> Profiling tsetse immunity-related gene expression incriminated              charide (LPS), missing O-antigen modified outer-membrane <br /> peptidoglycan recognition protein (pgrp)-lb determinant            protein A (OmpA) (10). Here, focus OmpA, which <br /> infection outcomes observed. RNAi-induced knockdown tsetse               comprises major component outer membrane the <br /> pgrp-lb significantly reduced host mortality infection              Enterobacteriaceae. This protein functions evasin and <br /> otherwise lethal E. coli K12. Our results polymorphisms           target mammalian immune system, modulator biofilm <br /> the exposed loop domains OmpA represent microbial adaptation              formation, bacteriophage receptor (11). Additionally, <br /> that mediates host tolerance endogenous symbiotic bacteria.                 functional studies pathogenic bacteria, including <br />                                                                                Leptospira interrogans (12), Neisseria gonorrhoeae (13), Salmo- <br /> peptidoglycan recognition protein LB &#229;&#8230;&#169; Sodalis glossinidius &#229;&#8230;&#169; tsetse           nella enterica (14), E. coli (15) demonstrated direct <br />                                                                                role OmpA virulence phenotypes host infection. No <br />                                                                                information exists function bacterial OmpA relates <br /> B    acteria highly adaptive diverse group organisms <br />      characteristics necessary inhabit virtually every <br /> ecosystem, including habitation organisms. Al- <br />                                                                                host-symbiont homeostasis. In present study, use the <br />                                                                                bacteria Sodalis E. coli (K12) Dipteran insects <br /> though pathogenic microorganisms detrimental effects                 Glossina morsitans morsitans Drosophila melanogaster <span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span> <br /> on animals plants extensively investigated,                  understand OmpA-mediated mechanisms influence <br /> greater majority relationships nature represent beneficial               symbiotic versus pathogenic infection outcomes. We report <br /> and sustained &#226;&#8364;&#732;&#226;&#8364;&#732;symbiotic&#226;&#8364;&#8482;&#226;&#8364;&#8482; associations (1). For example,                 tsetse immunity-related gene expression response to <br /> human intestinal microbiome representative enormous                   infection symbiotic versus foreign bacteria discuss how <br /> yet phylogenetically restricted population bacteria pro-               results contribute phenomenon host toler- <br /> vides immunological, metabolic, genetic constituents               ance endogenous flora. <br /> that consequently required evolve (2). To <br /> date experimental studies performed deter-                    Results Discussion <br /> mine functional molecular basis host tolerance                   E. coli K12 Infections Are Lethal Tsetse, Whereas Sodalis Proliferates. <br /> resident microbial flora.                                                      Sodalis E. coli genetically transformed construct <br />    Insects represent taxon documented symbiotic                    expresses luciferase reporter gene [pIL; supporting <br /> interactions allowed hosts exploit unusually                information (SI) Fig. S1 Materials Methods]. Relative <br /> restricted nutritional resources. Different insect symbioses re- <br />                                                                                light units (RLU) recombinant (rec) cell vitro were <br /> flect varying levels host&#226;&#8364;&#8220;microbe integration. Among these <br /> are heritable obligate mutualists. These symbionts been <br /> associated insect hosts millions years                   Author contributions: B.L.W., J.J.S., N.S.T., S.A. designed research; B.L.W., Y.W., J.J.S., <br /> provide metabolic supplementation host&#226;&#8364;&#8482;s <span id='am-1' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span>-                    N.S.T. performed research; B.L.W., J.J.S., S.A. analyzed data; B.L.W. S.A. <br /> restricted diets (3). In addition highly integrated asso-              wrote paper. <br /> ciations, insects harbor facultative mutualists. Grouped                  The authors declare conflict interest. <br /> within designation Sodalis glossinidius, tsetse                This article PNAS Direct Submission. <br /> endosymbiont closely related insect symbionts                 Data deposition: The sequence reported paper deposited GenBank <br /> <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> Hippoboscid flies (4), grain weevils (5), lice (6)            database (accession no. EU684475). <br /> as important enteric human pathogens including Escherichia                  &#226;&#8364;&#161;To   correspondence addressed. E-mail: serap.aksoy@yale.edu. <br /> coli, Salmonella, Yersinia. Selective elimination Sodalis               This article contains supporting information online www.pnas.org/cgi/content/full/ <br /> results reduced tsetse longevity (7). Phylogenetic reconstruc-              0805666105/DCSupplemental. <br /> tion analysis using ftsZ indicated Sodalis distant tsetse            &#194;&#169; 2008 The National Academy Sciences USA <br />  <br />  <br />  <br /> 15088 &#226;&#8364;&#8220;15093 &#229;&#8230;&#169; PNAS &#229;&#8230;&#169; September 30, 2008 &#229;&#8230;&#169; vol. 105 &#229;&#8230;&#169; no. 39                                                       www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.0805666105 <br />                                                                                          OmpA. However, Western blot analysis using E. coli anti- <br />                                                                                         OmpA polyclonal antibody confirmed Sodalis E. coli <br />                                                                                         express comparable amounts OmpA (Fig. 2A). To determine <br />                                                                                         E. coli OmpA (OmpAE.coli) responsible this <br />                                                                                         bacterium&#226;&#8364;&#8482;s lethal effect tsetse, flies infected a <br />                                                                                         transposon-generated mutant strain E. coli K12 (MG1655; <br />                                                                                         K12&#226;&#338;&#172;OmpA::Tn5Kan-2) (18) transformed express <br />                                                                                         luciferase (recE. coli&#226;&#338;&#172;OmpA-pIL). Infection outcome the <br />                                                                                         OmpA mutant strain different WT E. coli. <br />                                                                                         Whereas majority recE. coli&#226;&#338;&#172;OmpA-infected flies survived, <br />                                                                                         treated WT parental strain perished 12 dpi <br />                                                                                         (Fig. 2B). We used transcomplementation express Om- <br />                                                                                         pAE.coli OmpASodalis E. coli&#226;&#338;&#172;OmpA mutant background <br />                                                                                         (Fig. 2 A Fig. S1 B C). The OmpAE.coli-complemented <br />                                                                                         strain (recE. coli&#226;&#338;&#172;OmpA-pIOE.coli) exhibited lethal <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span>phenotype</span> in <br />                                                                                         tsetse similar WT E. coli. In contrast, flies infected the <br />                                                                                         OmpASodalis-complemented strain (recE. coli&#226;&#338;&#172;OmpA-pIOSodalis) <br />                                                                                         survived similarly infected OmpA mutants (Fig. <br />                                                                                         2C). Temporal analysis luciferase expression the <br />                                                                                         infection process indicated tsetse able clear infec- <br />                                                                                         tions recE. coli&#226;&#338;&#172;OmpA-pIL mutants transcomplemented <br />                                                                                         express OmpASodalis luciferase (Fig. 2D; using construct <br />                                                                                         pIOSodalis-Luc, Fig. S1D). These results suggest bacterium- <br /> Fig. 1. Host infection outcomes septic treatment rec bacteria. (A)           specific OmpA molecules responsible different infection <br /> Number cells fly time injecting 1 &#226;&#171;&#187; 103 CFU recSodalispIL <br />                                                                                         outcomes tsetse. <br /> (black circles) recE. colipIL (red squares) tsetse&#226;&#8364;&#8482;s hemocoel. The exper- <br /> iment repeated independent times (n &#226;&#171;&#189; 25 flies experiment) for <br /> both species rec bacteria. Rec cell number fly determined                 Sodalis That Express E. coli OmpA Pathogenic Tsetse. To <br /> dividing RLU expression vivo average number RLU produced               evaluate role OmpA host infection outcomes, <br /> cell vitro. All recE. colipIL-infected flies perished 10 dpi. (B C) Percent   Sodalis transformed express E. coli OmpA (using construct <br /> <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span>survival</span> female Drosophila (red squares) (B), female (red squares)           pIOE.coli, Fig. S1B). This gene engineered recOm- <br /> male (red circles) (C) tsetse systemic infection E. coli K12 grown        pAE.coli displays T7 epitope fourth outer loop (19). When <br /> MM media. Systemic infections tsetse performed microinjection                anti-T7 antibody used, expression OmpAE.coli in <br /> with glass needle, Drosophila pricked tungsten needle             Sodalis (recSodalis-pIOE.coli) confirmed Western blot anal- <br /> dipped concentrated bacterial solution. Control B C MM media <br />                                                                                         ysis cultured recSodalis-pIOE.coli hemolymph from <br /> (black squares); WT Sodalis (black circles); heat-killed E. coli K12 (black trian- <br /> gles). n &#226;&#171;&#189; 50 flies treatment. (D) Number recSodalispIL recE. colipIL <br />                                                                                         recSodalis-pIOE.coli-infected tsetse (Fig. 2 A). Furthermore, local- <br /> per tsetse fly time os infection 1,000 CFU ach bacteria.         ization recOmpAE.coli Sodalis &#226;&#8364;&#8482; outer membrane confirmed <br /> The experiment repeated duplicate (n &#226;&#171;&#189; 25 flies experiment)              immunofluorescence (Fig. S3). Tsetse Drosophila then <br /> both species bacteria. For experiments, d stands days infection.       systemically infected OmpAE.coli-expressing Sodalis, their <br />                                                                                         mortality rates subsequently monitored daily. Virtually all <br />                                                                                         tsetse infected OmpAE.coli-expressing Sodalis perished 8 dpi, <br /> determined recSodalispIL recE. colipIL different                        similar flies infected WT E. coli. In <br /> time points log growth phases. recSodalispIL                           contrast, Drosophila mortality unaffected treatment with <br /> produced 8.0 RLU cell, recE. colipIL produced 2.2 RLU                           OmpAE.coli-expressing Sodalis (Fig. 2E). Given Sodalis &#226;&#8364;&#8482; slower <br /> per cell (Fig. S2 A B, respectively). We injected 1 &#226;&#171;&#187; 103                      growth rate comparison E. coli, conclude host death <br /> colony-forming units (CFU) recSodalispIL recE. colipIL                           simply consequence septic shock. Rather, results <br /> directly tsetse&#226;&#8364;&#8482;s hemocoel measured luciferase activity                        expression OmpA commensal foreign <br /> as indicator cell growth time. Bacteria infected                     organisms differentially influences host infection outcomes. We <br /> fly groups exhibited virtually log-phase growth. recSodalispIL                          hypothesize presence variations structure of <br /> reached average peak density 8 &#226;&#171;&#187; 104 cells 14 days                       OmpA symbiotic pathogenic bacteria determinants of <br /> infection (dpi) detrimental effect host mortality.                        infection phenotype. <br /> In contrast, recE. colipIL grew rapidly infection resulted <br /> in death flies 10 dpi (Fig. 1A). To validate                          Residues Displayed External Loops OmpA Exhibit Polymor- <br /> experimental procedure comparative context, similarly                           phisms. We aligned OmpA putative transmembrane domains <br /> infected Drosophila recE. colipIL. Our results, like                         pathogenic symbiotic eubacteria. Our alignment in- <br /> from previous studies (16), demonstrate Drosophila                             cluded pathogens E. coli, Shigella flexneri, Salmonella typhi- <br />                                                                                                                                                                 MICROBIOLOGY <br />  <br />  <br />  <br /> survive infection E. coli K12 (Fig. 1B), tsetse                       murium, Yersinia pestis. The symbionts included So- <br /> again susceptible (Fig. 1C). Interestingly, lethal effect                        dalis; CMS, new lineage bloodsucking fly Craterina <br /> recE. colipIL tsetse restricted systemic infections,                          melbae (4); SOPE, principle endosymbiont associated with <br /> because number live recE. colipIL                                 weevil, Sitophilus oryzae (5); Candidatus Hamiltonella de- <br /> provided os blood meal, flies able clear                         fensa, facultative symbiont aphid species (20); <br /> infections (Fig. 1D). This finding confirms tsetse, like                     Photorhabdus luminescens, mutualistic symbiont of <br /> insects (17), exhibit different immune mechanisms gut                          heterorhabditid nematodes (21). Previous phylogenetic analysis <br /> and hemocoel compartments.                                                              indicated included symbionts form distinct lineage <br />    E. coli&#226;&#8364;&#8482;s virulence tsetse depends bacterium-specific                          related enteric pathogens (8). The 171 aa Om- <br /> OmpA expression. We speculated different tsetse                                pAE.coli form membrane-traversing &#226;?&#164;-barrels connected by <br /> survival outcomes observed infection closely related                          loops (L1&#226;&#8364;&#8220;L4) exposed external environment <br /> members family Enterobacteriaceae result                                (22). A previous study using peptide binding assay revealed that <br /> evolutionary adaptations surface-exposed molecules                           highly-pathogenic domains located exposed residues <br />  <br /> Weiss et al.                                                                                          PNAS &#229;&#8230;&#169; September 30, 2008 &#229;&#8230;&#169; vol. 105 &#229;&#8230;&#169; no. 39 &#229;&#8230;&#169; 15089 <br />  Fig. 2. OmpA expression bacterial virulence tsetse. (A) Western blot analysis OmpA expression 1 &#226;&#171;&#187; 104 CFU WT Sodalis (Sg) E. coli K12 (Ec) <br /> (Left); OmpA expression vitro mutant MG1655 (lane 1), recE. coli&#226;&#338;&#172;OmpA-pIOE.coli (lane 2) recE. coli&#226;&#338;&#172;OmpA-pIOSodalis (lane 3) (Center); OmpAE.coli expression <br /> in recSodalis-pIOE.coli (pIO) vitro hemolymph (in vivo) systemically-infected flies (Right). recSodalispIL (pIL) included negative control. The <br /> antibody used probe blot indicated blot. (B) Systemic infection tsetse 1 &#226;&#171;&#187; 103 CFU recE. coli&#226;&#338;&#172;OmpA-pIL (black squares) WT <br /> parental MG1655 cells (red squares). (C) Survival tsetse infected E. coli K12 MG1655&#226;&#338;&#172;OmpA::Tn5Kan-2 transcomplemented express OmpASodalis <br /> (recE.coli&#226;&#338;&#172;OmpA-pIOSodalis; black squares) OmpAE.coli (recE.coli&#226;&#338;&#172;OmpA-pIOE.coli; red squares). (D) The number recE. coli&#226;&#338;&#172;OmpA-pIL (black squares) recE. <br /> coli&#226;&#338;&#172;OmpA-pIOSodais-Luc (black circles) fly measured described Fig. 1 legend. (E) Systemic infection tsetse Drosophila recSodalis-pIOE <br /> coli. In B&#226;&#8364;&#8220;E, experiments performed duplicate, 25 tsetse experiment. In E, Drosophila (n &#226;&#171;&#189; 50) used control. For infections, tsetse <br /> received 1 &#226;&#171;&#187; 103 CFU bacteria, Drosophila pricked tungsten needle dipped concentrated bacterial solution. d, days infection. <br />  <br />  <br />  <br /> that comprise L1 L2 OmpA E. coli K1 (15). We                                Tsetse&#226;&#8364;&#8482;s Immune Response Infection Virulent Avirulent <br /> observed high conservation &#226;?&#164;-barrel structures                       Microbes. To address role tsetse&#226;&#8364;&#8482;s immune response in <br /> representative bacteria. In contrast, exposed loop regions,                       relation infection outcome, evaluated expression <br /> especially loop 1, divergent symbiont proteins                       profile representative sample tsetse immunity genes. We <br /> contained amino acid insertions substitutions                           included antimicrobial peptides (AMPs) (attacin, cecropin, and <br /> absent pathogens (Fig. 3). The signature motif loop                       defensin) associated regulator (peptidoglycan recognition <br /> 1 conserved closely related pathogens E.                          protein (pgrp)-lb), parasite-binding protein (annexin), thio- <br /> coli, Salmonella, Shigella present Yersinia.                          ester-containing proteins (tep2 tep4), prophenoloxidase- <br />    Based genome comparison studies phylogenetic anal-                          activating enzyme (PoAE). In insects TEPs function patho- <br /> yses, Sodalis SOPE form sister taxa (8). Interestingly,                       gen-specific opsonins bind bacteria parasites and <br /> results multiple PCR-amplification experiments indicate                          promote phagocytosis/encapsulation (26), PoAE <br /> that OmpASOPE highly homologous OmpASodalis                             induces melanization (27). RNA harvested 4 dpi flies <br /> been rendered nonfunctional pseudogene. Given obligate                          treated avirulent (WT Sodalis E. coli&#226;&#338;&#172;OmpA) and <br /> nature SOPE&#226;&#8364;&#8482;s association Sitophilus residence                        virulent (WT E. coli recSodalis-pIOE.coli) bacteria, host <br /> within host cells (5), loss OmpA function                            gene expression levels measured using real-time quan- <br /> symbiont likely reflective reductive genome processes.                          titative PCR. <br /> Many intracellular symbionts, including tsetse&#226;&#8364;&#8482;s obligate mutu-                          Our relative expression results indicate Imd Toll <br /> alist Wigglesworthia glossinidius (23), have, fact, lost                      signal transduction pathways, control expression of <br /> protein possibly extreme adaptation highly integrated                        AMPs (28), activated response avirulent virulent <br /> symbioses.                                                                            infection types, evidenced induced expression of <br />    Our alignment indicates modifications present                     attacin, cecropin, defensin (Table 1). However, expres- <br /> the exposed loop domains OmpA H. defensa                             sion levels AMPs compared avirulent and <br /> P. luminescens, microbes exhibit distinctly                         virulent infections, cecropin transcripts significantly less <br /> pathogenic characteristics insect secondary hosts. The                       abundant flies received virulent bacteria. Insufficient <br /> virulent phenotypes exhibited bacteria result                       expression cecropin explain lethal effects WT E. <br /> production phage-borne toxins, type III secretion                           coli recSodalis-pIOE.coli tsetse, Cecropin exhibits <br /> effectors, immunosuppressive antibiotics (24, 25). However,                       potent Gram-negative bactericidal activity (29). Infection with <br /> little known mechanisms allow H. defensa                        virulent bacteria resulted significantly higher expression <br /> P. luminescens reside symbionts aphids heter-                        host pgrp-lb avirulent counterparts. Given role <br /> orhabditid nematodes, respectively. We speculate                             PGRP-LB Drosophila negative regulator Imd <br /> OmpA polymorphisms bacteria exhibit underlie                            pathway (30), induction pgrp-lb prevented <br /> symbiotic component lifecycles. Our OmpA sequence                            activation innate immune response flies infected with <br /> alignment, coupled results functional expression                        virulent bacteria. To test hypothesis, used RNA inter- <br /> studies, suggest bacterium-specific variations host-                          ference knock expression tsetse pgrp-lb. Using this <br /> exposed OmpA domains responsible differential                          strategy, able repress transcription gene by <br /> infection outcomes observed infection tsetse                          &#226;&#172;&#8225;70% (Fig. 4A) subsequently reverse lethal effect E. <br /> Sodalis E. coli.                                                                  coli K12 infection tsetse (Fig. 4B). Finally, observed <br />  <br /> 15090 &#229;&#8230;&#169; www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.0805666105                                                                                                        Weiss et al. <br />  Fig. 3. Multiple sequence alignment 205 N-terminal putative amino acids OmpA Sodalis, SOPE (S. oryzae principal endosymbiont), CMS (C. <br /> melbae symbiont), H. defensa, P. luminescens, Y. pestis, E. coli 536 (UPEC), S. typhimurium, S. flexneri. For SOPE, N-terminal 118 aa included, <br /> because rest ORF interrupted multiple insertions generate stop codons frame. External loop structures underlined designated <br /> L1&#226;&#8364;&#8220;L4. L1 L2 previously proven encode virulent domains (15). Conserved residues outlined black, substitutions gray. <br />  <br />  <br />  <br /> significantly lower tep 2 PoAE expression levels flies                       persist, evidenced continuous expression luciferase. <br /> infected E. coli. The lack phagocytosis, encapsulation,                     Sodalis &#226;&#8364;&#8482; survival result demonstrated resistance to <br /> and melanization responses enable bacterium                     host AMPs (e.g., Diptericin Attacin) (31, 32). Alternatively, <br /> undergo unimpeded replication, lethal tsetse.                      Sodalis protected elimination encapsulation or <br />    We infections virulent bacteria did activate                  melanization processes entering host hemocytes through <br /> host immune responses, infections avirulent strains                    use type III secretion (TTSS) (9, 33). In contrast, <br /> induced expression immunity-related genes. Although                          E. coli likely cleared hemolymph its <br /> activation tsetse&#226;&#8364;&#8482;s immune arsenal eliminated E. coli&#226;&#338;&#172;OmpA                       inability invade persist host cells. Further reflective <br /> and E. coli&#226;&#338;&#172;OmpA express OmpASodalis, WT Sodalis able                      homeostatic relationship tsetse Sodalis the <br />  <br />  <br /> Table 1. Fold change expression immunity-related genes tsetse infected avirulent versus virulent bacteria <br />                                                                                                             Mean fold change &#226;&#171;&#190; SEM <br />  <br />                                                                                               avirulent                                      virulent <br />                                                                                                                                                                        MICROBIOLOGY <br />  <br />  <br />  <br /> Gene                 Gene family              GenBank accession no.                Sgm-WT                  Ec-&#226;&#338;&#172;omp               Sgm-pIO                 Ec-WT <br />  <br /> annexin           Parasite-binding                   NP&#227;&#8250;&#173;523370                   14.8 &#226;&#180;&#382; 3.8                6.7 &#226;&#171;&#190; 2.12           1.0 &#226;&#171;&#190; 0.4              3.2 &#226;&#171;&#190; 1.0 <br /> attacin           AMP                                 AF368909                    2.4 &#226;&#171;&#190; 0.3                1.5 &#226;&#171;&#190; 0.79           2.1 &#226;&#171;&#190; 1.1             &#226;&#171;&#186;1.6 &#226;&#171;&#190; 0.07 <br /> cecropin          AMP                                   P83403                   18.8 &#226;&#180;&#382; 4.6               14.3 &#226;&#180;&#382; 2.0           &#226;&#171;&#186;2.9 &#226;&#171;&#190; 0.1              1.3 &#226;&#171;&#190; 0.09 <br /> defensin          AMP                                AAL34112                     8.3 &#226;&#171;&#190; 1.2                9.5 &#226;&#171;&#190; 2.51           3.0 &#226;&#171;&#190; 1.5              7.3 &#226;&#171;&#190; 0.68 <br /> PGRP-LB           AMP regulator                      DQ307160                    &#226;&#171;&#186;3.1 &#226;&#180;&#382; 0.06               1.1 &#226;&#180;&#382; 0.11           2.6 &#226;&#171;&#190; 1.1              2.5 &#226;&#171;&#190; 0.13 <br /> PoAE              Melanization                       ABC84592                    77.0 &#226;&#180;&#382; 2.7               13.8 &#226;&#180;&#382; 8.6            1.3 &#226;&#171;&#190; 0.08             4.1 &#226;&#171;&#190; 1.6 <br /> tep2              Phagocytosis                       ABC84589                    19.6 &#226;&#180;&#382; 3.8               11.5 &#226;&#180;&#382; 1.24           2.7 &#226;&#171;&#190; 0.9              6.0 &#226;&#171;&#190; 0.92 <br /> tep4              Phagocytosis                       ABC84590                     4.6 &#226;&#180;&#382; 0.39               6.2 &#226;&#171;&#190; 1.1            2.9 &#226;&#171;&#190; 0.5              4.5 &#226;&#171;&#190; 0.75 <br />  <br />    Ec-WT, WT E. coli-infected flies; Sgm-pIO, OmpA E. coli-expressing Sodalis, Sgm-WT, WT Sodalis-infected flies; Ec-&#226;&#338;&#172;OmpA, OmpA-deficient E. coli-infected flies. <br /> Bolded values represent significantly different levels gene expression avirulent virulent strains bacteria. Complete statistical analysis <br /> of data shown Table S1. <br />  <br />  <br /> Weiss et al.                                                                                         PNAS &#229;&#8230;&#169; September 30, 2008 &#229;&#8230;&#169; vol. 105 &#229;&#8230;&#169; no. 39 &#229;&#8230;&#169; 15091 <br />                                                                                       supplemented 5% heat-inactivated FBS (8). E. coli K12 strains (including <br />                                                                                      mutant WT MG1655) grown LB media. <br />  <br />                                                                                      Plasmid Construction Bacterial Transformation. Constructs designated pIL <br />                                                                                      (4,793 bp), pIOE.coli (4,211 bp), pIOSodalis (4,214 bp), pIOSodalis-Luc (5,867 bp) <br />                                                                                      encode firefly luciferase gene, E. coli OmpA gene, Sodalis OmpA <br />                                                                                      gene, Sodalis OmpA luciferase genes, respectively, tran- <br />                                                                                      scriptional control Sodalis&#226;&#8364;&#8482; insulinase promoter (Fig. S1). Details the <br />                                                                                      engineering constructs described SI Text. <br />                                                                                         All E. coli transformations performed electroporation according to <br />                                                                                      standardized protocol (38). Sodalis genetically transformed via <br />                                                                                      electroporation using published protocol (8). After transformation, recSo- <br />                                                                                      dalis recE. coli cells plated MM LB agar plates, respectively <br /> Fig. 4. Effect pgrp-lb knockdown E. coli infection tsetse. (A) pgrp-lb      (supplemented antibiotics) (39). Single colonies bacterial strain <br /> expression reduced &#226;&#172;&#8225;70% flies injected PGRP-LB dsRNA com-             placed liquid culture. <br /> pared controls received GFP dsRNA. (B) Systemic infection tsetse <br /> with 1 &#226;&#171;&#187; 103 WT E. coli 4 days injection GFP (black circles)          Luciferase Reporter Assays. All luciferase reporter assays performed <br /> PGRP-LB dsRNA (black squares). pgrp-lb expression standardized host           according manufacturer&#226;&#8364;&#8482;s protocol (Promega) using Sirius single- <br /> tubulin.                                                                             tube luminometer (Berthold Detection Systems). Overnight cultures both <br />                                                                                      bacterial species subcultured OD600 0.05. Subsequently, OD600 <br />                                                                                      readings taken recSodalispIL day (Sodalis &#226;&#8364;&#8482; maximum OD600 &#226;&#171;&#189; <br /> fact expression immunity-related genes Sodalis-                           1.0) recE. colipIL 2 h (E. coli&#226;&#8364;&#8482;s maximum OD600 &#226;&#171;&#189; 1.8) strains <br /> infected flies returned similar levels age-                       grown approximately midlog phase. For vitro assays, 5 &#226;&#171;&#187; 105 rec cells <br /> matched uninfected tsetse 14 dpi (Table S2). Our findings                         time point subjected luciferase quantification. This proce- <br /> suggest fatal outcome WT E. coli OmpAE.coli-                         dure repeated twice clonal populations bacterial strain. <br /> expressing Sodalis infections tsetse result direct                   RLU cell determined dividing RLU expression cell number. <br /> inhibitory effects E. coli OmpA host immune activation.                           RLU fly determined homogenizing individuals 100 &#226;?&#174;l PBS <br /> Alternatively, bacteria escape host immune recogni-                        pelleting cellular debris. Supernatant (90 &#226;?&#174;l) sample <br />                                                                                      assayed luciferase expression described (n &#226;&#171;&#189; 3 individ- <br /> tion indirect OmpAE.coli-mediated process as <br />                                                                                      uals time point, value representing average RLU expression <br /> differential regulation biofilm formation alteration                        3). Based assumption luciferase expression remains constant <br /> bacterial gene expression associated cellular adhesion                      vitro vivo, determined rec bacterial cell number each <br /> invasion processes.                                                                  species designated time points dividing RLU values fly average <br />                                                                                      number RLU produced cell vitro (8.0 recSodalispIL 2.2 recE. <br /> Conclusions                                                                          colipIL). For vivo experiments, background luciferase expression (from WT <br /> Recent studies demonstrate insects exhibit                          tsetse) subtracted experimental values. <br /> adaptive immunity respond specifically, a <br /> lasting manner, prior exposure specific pathogens (34,                      Infection Tsetse Drosophila rec Mutant Bacteria. Septic <br /> 35). A comparable mechanism account tsetse&#226;&#8364;&#8482;s initial                         infection tsetse Drosophila achieved anesthetizing individuals <br /> hyperimmune response superinfection native symbi-                        CO2. Subsequently, tsetse injected 1 &#226;&#171;&#187; 103 CFU live bacterial <br />                                                                                      cells using glass needles Narashige IM300 microinjector, whereas <br /> ont Sodalis versus lack robust immune response E. coli. <br />                                                                                      Drosophila pricked tungsten needle dipped concentrated <br /> Our results demonstrate tsetse&#226;&#8364;&#8482;s susceptibility infection                    culture appropriate bacterial strain (SI Text, note 1). Control groups and <br /> with normally nonpathogenic E. coli K12 reflect unique                       sample sizes infection experiments indicated corresponding <br /> ecological aspects fly&#226;&#8364;&#8482;s lifestyle. Tsetse feeds exclusively                 figure legends. Heat-killed cells incubated 90 min 80&#194;&#176;C. <br /> on sterile vertebrate blood exhibits unusual viviparous                          For tsetse feeding experiments, blood meal supplemented 3.3 &#226;&#171;&#187; <br /> reproductive strategy intrauterine larvae nourished                        104 CFU live bacterial cells milliliter blood (which equaled 1 &#226;&#171;&#187; 103 cells <br /> with secretions female milk glands. This mode repro-                         30 &#226;?&#174;l, quantity consumed tsetse fly feeding, determined by <br /> duction restricts exposure immature stages range                      weight gain). <br /> environmental microbes crucial period immune <br /> system development. In contrast Drosophila, resistant                    Outer-Membrane Protein A Alignments. A 690-bp OmpA fragment SOPE <br /> E. coli infection, reproduces consumes organic material                       (S. oryzae principle endosymbiont; GenBank accession no. EU426969) C. <br />                                                                                      melbae secondary symbiont (CMS; GenBank accession no. EU684475) was <br /> undergoing decomposition wide variety bacteria. <br />                                                                                      amplified insect total genomic DNA using following primer pair: <br /> Throughout evolution, tsetse&#226;&#8364;&#8482;s limited contact microbial                        OmpA-F, 5&#226;&#172;&#732;-ATGAAAAAGACAGCTATC-3&#226;&#172;&#732;; OmpA-R, 5&#226;&#172;&#732;-GTCAGACTTCAGGGT- <br /> fauna compromised development robust                               GAA-3&#226;&#172;&#732;. Putative OmpA protein sequences above-mentioned mi- <br /> systemic immune capable eliminating wide range                        crobes, Sodalis (BAE74305), H. defensa, (EU682308), P. luminescens <br /> foreign microbes, including E. coli. It remains seen                   (NP&#227;&#8250;&#173;929054), Y. pestis (NP670036), E. coli 536 (UPEC; CP000247), S. typhi- <br /> other insect taxa reside &#226;&#8364;&#732;&#226;&#8364;&#732;sterile&#226;&#8364;&#8482;&#226;&#8364;&#8482; ecological niches                   murium (X02006) S. flexneri (AF234271), aligned using ClustalX <br /> rely symbiotic bacteria survival exhibit com-                      1.83 (http://bips.u-strasbg.fr/fr/ClustalX/) BioEdit (www.mbio.ncsu.edu/ <br /> promised immunity challenged foreign microbes.                             BioEdit/bioedit.html) software packages. <br />  <br /> Materials Methods                                                                Immunological Assays. Western blot situ antibody staining protocols are <br /> Insects Bacterial Cultures. G. m. morsitans maintained Yale&#226;&#8364;&#8482;s            described in SI Text. For Western blots, 1 &#226;&#171;&#187; 104 CFU cultured WT <br /> insectary 24&#194;&#176;C 50 &#226;&#8364;&#8220;55% relative humidity. These flies received defi-         Sodalis, WT E. coli, recSodalispIL, recSodalis-pIOE.coli, recE. coli&#226;&#338;&#172;OmpA-pIOE.coli <br /> brinated bovine blood 48 h artificial membrane feeding              recE. coli&#226;&#338;&#172;OmpA-pIOSodalis pelleted, reconstituted protein loading <br /> system (36). D. melanogaster (strain OregonR) treated maintained         buffer, boiled, placed ice. Hemolymph (20 &#226;?&#174;l) infected fly <br /> 25&#194;&#176;C 40 &#226;&#8364;&#8220;50% relative humidity.                                                 groups similarly treated. Anti-E. coli OmpA anti-T7 epitope primary <br />    S. glossinidius isolated surface-sterilized G. m. morsitans pupae       antibodies diluted 1:20,000. For situ antibody staining, smears were <br /> and cultured Aedes albopictus C6/36 cells described (37). Sodalis,       prepared using 1 &#226;&#171;&#187; 108 CFU recSodalis-pIOE.coli recSodalispIL, the <br /> has doubling time &#226;&#172;&#8225;24 h, subsequently maintained cell-free vitro        anti-T7 epitope primary antibody diluted 1:5,000. <br /> at 25&#194;&#176;C Mitsuhashi&#226;&#8364;&#8220;Maramorosch (MM) medium (1 mM CaCl2, 0.2 mM <br /> MgCl2, 2.7 mM KCl, 120 mM NaCl, 1.4 mM NaHCO3, 1.3 mM NaH2PO4, 22 mM                 Analysis Immunity-Related Gene Expression. Sample preparation quan- <br /> D (&#226;&#171;&#185;) glucose, 6.5 g/liter lactalbumin hydrolysate, 5.0 g/liter yeast extract)   titative real-time PCR (qRT-PCR) performed described (8). Amplifica- <br />  <br />  <br /> 15092 &#229;&#8230;&#169; www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.0805666105                                                                                                           Weiss et al. <br />  tion primers listed Table S3. Quantitative measurements per-                       qRT-PCR performed described above. pgrp-lb expression treatment <br /> formed biological samples duplicate, results normalized                         control flies normalized relative tsetse&#226;&#8364;&#8482;s constitutively expressed <br /> relative tsetse&#226;&#8364;&#8482;s constitutively expressed &#226;?&#164;-tubulin gene (determined                  &#226;?&#164;-tubulin gene. <br /> each corresponding sample). Fold-change data represented fraction <br /> of average normalized gene expression levels bacteria-infected flies relative               ACKNOWLEDGMENTS. We grateful Claudia Lohs technical assis- <br /> to expression levels PBS-injected controls. Values represented                   tance situ antibody staining experiments, Dr. Nancy Moran (University <br /> mean (&#226;&#171;&#190;SEM), statistical significance determined using Student&#226;&#8364;&#8482;s                  Arizona, Tucson, AZ) providing H. defensa OmpA sequence, Dr. <br /> t test Microsoft Excel software.                                                           Abdelaziz Heddi (Institut National la Recherche Agronomique, Villeur- <br />                                                                                                banne, France) critical review manuscript S. oryzae genomic <br /> Tsetse <span id='am-240' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-241' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-243' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-256' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-258' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-271' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-295' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-303' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span>RNA Interference</span> Subsequent Bacterial Infection. PGRP-LB GFP                    DNA, Dr. Va&#194;&#180;clav Hypsa (Institute Parsitology, Cesk&#195;&#169; Budejovice, Czech <br /> double-stranded (ds) RNA prepared according manufacturer&#226;&#8364;&#8482;s                          Republic) C. melbae genomic DNA, Dr. Thomas Woods (Texas A&amp;M Uni- <br />                                                                                                versity, College Station, TX) E. coli K12 MG1655 rec WT strains, Dr. <br /> protocol (Ambion). Gene-specific PCR primers, encoding 5&#226;&#172;&#732; T7 RNA <br />                                                                                                Patrick Daugherty (University California, Santa Barbara, CA) plasmid <br /> polymerase binding site, listed Table S3. Treatment control flies <br />                                                                                                pB33OT4, Dr. Nemani V. Prasadarao (The Saban Research Institute, Los <br /> were injected 4 &#226;?&#174;g dsPGRP-LB dsGFP RNA, respectively. Four days                    Angeles, CA) E. coli anti-OmpA polyclonal antibody. This work was <br /> after injection, groups flies infected 1 &#226;&#171;&#187; 103 CFU E. coli                supported National Institute Allergy Infectious Diseases Grant <br /> K12 described previously. RNAi performed separate experiments                    AI51584, National Institute General Medical Sciences Grant 069449, an <br /> for treatment control flies (n &#226;&#171;&#189; 25 flies experiment).                            Ambrose Monell Foundation grant (all S.A.). B.L.W. recipient of <br />    For quantification pgrp-lb expression dsRNA injection, RNA                     Kirschstein&#226;&#8364;&#8220;National Research Service Award training fellowship <br /> harvested treatment control flies 4 days treatment.                       F32AI062680. <br />  <br />  <br />  1. Moran NA (2006) Symbiosis. Curr Biol 16:R866 &#226;&#8364;&#8220;R871.                                        20. Moran NA, Russell JA, Koga R, Fukatsu T (2005) Evolutionary relationships new <br />  2. Ley RE, Peterson DA, Gordon  <br /> </body></html>