<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                            NIH Public Access <br />                             Author Manuscript <br />                             J Biol Chem. Author manuscript; available PMC 2008 March 3. <br />                            Published final edited form as: <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             J Biol Chem. 2006 October 13; 281(41): 30725&#226;&#8364;&#8220;30735. <br />  <br />  <br />                            Mutant Analysis Shal (Kv4) Voltage-gated Fast Transient <br />                            K+ Channel Caenorhabditis elegans*,S <br />  <br />                            Gloria L. Fawcett&#226;&#8364;&#161;, Celia M. Santi&#226;&#8364;&#161;, Alice Butler&#226;&#8364;&#161;, Thanawath Harris&#194;&#167;, Manuel <br />                            Covarrubias&#194;&#167;, Lawrence Salkoff <br />                            &#226;&#8364;&#161;Department Anatomy Neurobiology, Washington University School Medicine, St. Louis, Missouri <br />                            63110 <br />                            &#194;&#182;Department Genetics, Washington University School Medicine, St. Louis, Missouri 63110 <br />                            &#194;&#167;Department Pathology, Anatomy, Cell Biology, Jefferson Medical College, Thomas Jefferson <br />                            University, Philadelphia, Pennsylvania 19107 <br />  <br />                            <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                 Shal (Kv4) &#206;&#177;-subunits conserved family voltage-gated potassium channels. <br />                                 Previous work shown Shal potassium channel subfamily underlies predominant fast <br />                                 transient outward current Drosophila neurons (Tsunoda, S., Salkoff, L. (1995) J. Neurosci. <br />                                 15, 1741&#226;&#8364;&#8220;1754) fast transient outward current mouse heart muscle (Guo, W., Jung, W. E., <br />                                 Marionneau, C., Aimond, F., Xu, H., Yamada, K. A., Schwarz, T. L., Demolombe, S., Nerbonne, <br />                                 J. M. (2005) Circ. Res. 97, 1342&#226;&#8364;&#8220;1350). We Shal channels play role the <br />                                 predominant transient outward current Caenorhabditis elegans muscle. Green fluorescent protein <br />                                 promoter experiments revealed SHL-1 expression subset neurons C. <br />                                 elegans body wall muscle male-specific diagonal muscles. The shl-1 (ok1168) null mutant <br />                                 removed fast transient outward current muscle cells. SHL-1 currents strongly resembled <br />                                 Shal currents species active depolarized voltage range. We <br />                                 determined remaining delayed-rectifier current cultured myocytes carried the <br />                                 Shaker ortholog SHK-1. In shl-1 (ok1168) mutants significant compensatory increase in <br />                                 SHK-1 current. Male shl-1 (ok1168) animals exhibited reduced mating efficiency resulting <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br />                                 apparent difficulty locating hermaphrodite vulva. SHL-1 channels apparently important <br />                                 fine-tuning complex behaviors, mating, play crucial role <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span>survival</span> and <br />                                 propagation species. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Voltage-dependent potassium (K+) (Kv)2 channels key regulators membrane <br />                                               excitability. Based inactivation kinetics, Kv currents loosely divided two <br />                                               categories follows: noninactivating slowly inactivating K+ currents, rapidly <br />                                               inactivating transient currents known &#226;&#8364;&#339;A-type&#226;&#8364;? currents. A-type currents first <br />                                               described molluscan neurons Hagiwara et al. (3) later Connor Stevens (4), <br />                                               Neher (5), Thompson (6). Classical A-type currents (IA) voltage-dependent, Ca2+- <br />                                               independent K+ currents undergo rapid activation inactivation. The classical A-type <br />  <br />  <br />                            *This work supported National Institutes Health Grants R24 RR017342-01, R01 GM067154-01A1 (<span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> L. S.), R01 NS032337 <br />                            (to M. C.), T32 AA007463 (to T. H.). <br />                            SThe on-line version article (available http://www.jbc.org) contains supplemental Figs. 1 2. <br />                            1To correspondence addressed: Dept. Anatomy Neurobiology, Washington University School Medicine, 660 <br />                            S. Euclid Ave., St. Louis, MO 63110. Tel.: 314-362-3644; Fax: 314-362-3446; E-mail: salkoffl@pcg.wustl.edu. <br />                            2The abbreviations used are: Kv, mammalian voltage-gated K+ channel family; unc, uncoordinated mutant; exp, expulsion-defective in <br />                            defecation mutant; GFP, green fluorescent protein; RNAi, <span id='am-473' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-474' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-475' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-476' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-477' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-478' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-479' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-480' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-481' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-482' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-483' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-484' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-485' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-486' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-487' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-488' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-489' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-490' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-491' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-492' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-493' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-494' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-495' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-496' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-497' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-498' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-499' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-500' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-501' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-502' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-503' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-504' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-505' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-506' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-507' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-508' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-509' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-510' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-511' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-512' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-513' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-514' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-515' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-516' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-517' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-518' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-519' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-520' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-521' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-522' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-523' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-524' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-525' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-526' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-527' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-528' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-529' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-530' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-531' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-532' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-533' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-534' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-535' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-536' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-537' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-538' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-539' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-540' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-541' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-542' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-543' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-544' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-545' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-546' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-547' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-548' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-549' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-550' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span>RNA interference</span>; dsRNA, double-stranded RNA; DIDS, 4,4&#226;&#8364;&#178;- <br />                            diisothiocyanostilbene-2,2&#226;&#8364;&#178;-disulfonic acid; NLS, nuclear localization sequence; ACh, acetylcholine; EST, expressed sequence tag. <br />                             Fawcett et al.                                                                                               Page 2 <br />  <br />  <br />                                             currents activate subthreshold voltages recover inactivation quickly compared with <br />                                             Kv currents. They characterized strong steady-state voltage-dependent <br />                                             inactivation. Repolarization potentials negative &#226;&#710;&#8217;50 mV typically required restore <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             channel availability. A-type currents widespread distribution abundantly <br />                                             expressed neurons cardiac smooth muscle cells, thought play many <br />                                             important physiological roles (2,7-11). <br />  <br />                                             Connor Stevens (12) proposed IA determine interspike interval in <br />                                             repetitively firing neurons. Studies cerebellar granule cells demonstrated role IA in <br />                                             determining latency spike (13). IA demonstrated limit back <br />                                             propagation action potentials dendritic arbor (14), impact long term potentiation <br />                                             CA1 hippocampal neurons (15), affect neuronal excitability visual cortex (16), and <br />                                             modulate compartmentalization membrane excitability distal dendritic spines (17). Fast <br />                                             transient K+ currents cardiac muscles contribute myocyte excitability (18) the <br />                                             fast repolarization phase cardiac action potential (2,8,19,20). Finally, the <br />                                             pharmacological block channels carry fast transient currents gastrointestinal smooth <br />                                             muscle mice supports conclusion window currents carried channels <br />                                             contribute resting membrane potential cellular excitability smooth muscle (21, <br />                                             22). <br />  <br />                                             The molecular identity A-type currents different tissues subject of <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             extensive investigation. Experiments transgenic animals carrying dominant negative Shal <br />                                             &#206;&#177;-subunit constructs indicate Shal family channels express cardiac fast transient <br />                                             outward K+ currents cardiac myocytes (23). However, molecular composition IA <br />                                             currents neurons remains uncertain. In Drosophila, synthetic deletion allowed the <br />                                             physiological analysis embryonic neurons lacking shal gene, showed vast <br />                                             majority neurons express Shal currents (1). It noteworthy Kv channels, like the <br />                                             Kv1 family, capable forming A-type currents, add complexity trying to <br />                                             establish specific contribution particular type KV channel particular cell type. <br />  <br />                                             Here Caenorhabditis elegans shal gene (shl-1) encodes unique transient <br />                                             current muscle cells. We fast transient current carried Shal channels <br />                                             contrasts Shaker channels (shk-1) C. elegans, express currents only <br />                                             slowly inactivating. Both Shal Shaker channels expressed body wall muscle cells <br />                                             variety neurons. Shal currents recorded body wall muscle cells culture <br />                                             heterologously expressed Xenopus oocytes properties similar Shal channels from <br />                                             species regard rapid activation inactivation, differ they <br />                                             active positive voltage range. The <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span>phenotype</span> shl-1 deletion <span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span>mutation</span> was <br />                                             manifested increase muscle excitability reflected abnormal aldicarb sensitivity, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             abnormal thrashing behavior, mating deficiencies. <br />  <br />                               MATERIALS AND METHODS <br />                               Molecular Biology <br />                                             EST yk327e11 (Kohara) encodes full-<span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> shl-1 cDNA obtained cloned <br />                                             pBSc II KS+ plasmid. Three shk-1 isoform (a, c, d2) expression constructs were <br />                                             created using EST yk442g4, encodes tetramerization domain core protein, <br />                                             includes entire 3&#226;&#8364;&#178; end gene. yk442g4 inserted Oocyte Express pOX <br />                                             vector. The alternative splice isoform expression constructs created inserting EST <br />                                             encoding variable 5&#226;&#8364;&#178; end gene yk442g4&#226;&#710;&#183;pOX using BbvC1. The ESTs used <br />                                             yk813e07, yk753e1, yk1164g03, yk1168a01 yk736e9 shk-1 isoforms a, <br />                                             c, d2. All ESTs obtained laboratory Yuji Kohara sequenced. <br />                                             Intron/exon boundaries determined comparing verified EST genomic sequence. <br />  <br />  <br />                                                  J Biol Chem. Author manuscript; available PMC 2008 March 3. <br />                             Fawcett et al.                                                                                              Page 3 <br />  <br />  <br />                                             The W363F dominant negative SHL-1 (GenBank&#226;&#8222;&#162; accession mumber NM_068574) construct <br />                                             created site-directed mutagenesis full-length shl-1 cDNA, changing the <br />                                             phenylalanine position 363 tryptophan. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Xenopus Oocytes cRNA Injection <br />                                             Female Xenopus laevis obtained Nasco (Fort Atkinson, WI). Oocytes collected <br />                                             (according guidelines Institutional Animal Care Use Committee Thomas <br />                                             Jefferson University) anesthesia (immersion 0.2% 3-aminobenzoic acid ethyl ester <br />                                             (Sigma) 30 min) frogs humanely killed final oocyte collection. <br />                                             Before injection, oocytes defolliculated digestion collagenase (2 mg ml&#226;&#710;&#8217;1; Sigma) <br />                                             calcium-free external solution (see below). The shl-1 cRNA injected (total RNA injected <br />                                             ~2&#226;&#8364;&#8220;100 ng oocyte) defolliculated oocytes using Nanoject microinjector (Drummond, <br />                                             Broomall, PA). Currents recorded 1&#226;&#8364;&#8220;5 days post-injection. <br />  <br />                               Electrophysiology; Two-electrode Voltage Clamp <br />                                             Macroscopic whole-oocyte currents recorded ND96 plus 1 mM DIDS using two- <br />                                             electrode voltage clamp technique Yuan et al. (24). Conductance-voltage relationship <br />                                             data obtained using standard voltage-step protocol stepping holding potential of <br />                                             &#226;&#710;&#8217;100 mV voltages &#226;&#710;&#8217;80 +50 mV steps 10 mV period 450 ms (for <br />                                             shl-1 currents, n =7) 5 s (for shk-1 currents, isoform n = 5&#226;&#8364;&#8220;6). Steady-state <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             inactivation data SHL-1 currents (n = 7) obtained using protocol stepping a <br />                                             holding potential &#226;&#710;&#8217;100 mV range &#226;&#710;&#8217;120 +10 mV 10-mV steps 9 s a <br />                                             depolarizing step +40 mV 250 ms. Steady-state inactivation data SHK-1 currents <br />                                             (n = 8) obtained using protocol stepping holding potential &#226;&#710;&#8217;100 mV range <br />                                             &#226;&#710;&#8217;120 +10 mV 10-mV steps 38 s depolarizing step +40 mV 1 s. The <br />                                             dominant negative SHL-1 construct cRNA examined 1:1 ratio co-injection the <br />                                             SHL-1 cRNA using protocols described SHL-1 cRNA oocyte expression. All <br />                                             protocols utilized 5-s intersweep interval &#226;&#710;&#8217;100 mV allow inactivated channel <br />                                             recovery. Current records filtered 1 kHz, acquired digitally using Clampex 9.0 (Axon <br />                                             Laboratories), analyzed using Clampfit 9.2 (Axon Laboratories). G-V curves were <br />                                             obtained converting peak current values I-V relationships conductances by <br />                                             using equation: G = I (V &#226;&#710;&#8217; EK), G conductance; I peak current; V the <br />                                             command pulse potential; EK K+ reversal potential. Conductance values were <br />                                             normalized fitted Boltzmann equation: G/Gmax = {1 + exp(&#226;&#710;&#8217;(V &#226;&#710;&#8217; V0.5a)/ka)}&#226;&#710;&#8217;1, <br />                                             G peak conductance; Gmax maximal peak conductance; V V0.5a the <br />                                             command potential midpoint activation, respectively; ka activation slope <br />                                             factor. Steady-state inactivation analysis performed using normalized current during <br />                                             test pulse plotted function prepulse potential. The data fitted the <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Boltzmann equation: I/Imax = {1 + exp(V &#226;&#710;&#8217; V0.5i/ki)}&#226;&#710;&#8217;1, I peak current; Imax the <br />                                             peak current prepulse potential &#226;&#710;&#8217;80 mV; V V0.5i prepulse potential <br />                                             half-inactivation potential, respectively; ki inactivation slope factor. <br />  <br />                               Electrophysiology; Patch Recordings Xenopus Oocytes <br />                                             Macropatch unitary recordings conducted described previously (25) using an <br />                                             Axopatch 200A 200B amplifier (Axon Instruments, Foster City, CA). Patch pipettes were <br />                                             constructed Corning Glass 7052 7056 (Warner Instrument Corp., Hamden, CT). For <br />                                             recording fast currents (e.g. tail current relaxations) single channel currents, the <br />                                             pipettes (~0.5&#226;&#8364;&#8220;1 5&#226;&#8364;&#8220;10 megohms bath solution, respectively) coated with <br />                                             Sylgard elastomer (Dow Corning Co., Midland, MI). Passive leak current capacitive <br />                                             transients subtracted online using P/4 procedure. The recordings filtered 0.5&#226;&#8364;&#8220; <br />  <br />  <br />  <br />  <br />                                                  J Biol Chem. Author manuscript; available PMC 2008 March 3. <br />                             Fawcett et al.                                                                                            Page 4 <br />  <br />  <br />                                             8 kHz (&#226;&#710;&#8217;3 dB, 8-pole Bessel filter; Frequency Devices, Haverhill, MA) digitized 2&#226;&#8364;&#8220;40 <br />                                             kHz. All experiments recorded room temperature (23 &#194;&#177; 1 &#194;&#176;C). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Data analysis conducted using Clampfit 8&#226;&#8364;&#8220;9 (Axon Instruments), SigmaPlot 8&#226;&#8364;&#8220;9 (Systat <br />                                             Software Inc., Point Richmond, CA), Origin 7.0 (OriginLab Inc., Northhampton, MA). <br />                                             The voltage dependence peak chord conductance (Gp-V relationship), steady-state <br />                                             inactivation, time-dependent current relaxations analyzed described elsewhere <br />                                             (25,26). The unitary conductance estimated applying voltage ramp protocols evoke <br />                                             single channel currents (&#226;&#710;&#8217;100 +100 mV; 0.9 ms/mV). In case, passive leak current <br />                                             capacitive transients subtracted using blank sweeps (no unitary currents). <br />                                             Results expressed means &#194;&#177; S.E. <br />  <br />                               Constructs Transgenic Animals <br />                                             The expression construct shl-1 created using overlapping PCR products <br />                                             encompassing entire 10-kb upstream region gene. We designed ~500-bp overlap <br />                                             PCR pieces (5&#226;&#8364;&#178;-PCR 6.4 kb, 3&#226;&#8364;&#178;-PCR 7.3 kb), similar overlap <br />                                             &#226;&#8364;&#339;core&#226;&#8364;? protein coding genomic DNA fragment (genomic DNA positions 22316&#226;&#8364;&#8220;24990 <br />                                             YAC Y73B6BL) fused in-frame GFP pPD95 GFP expression plasmid. The <br />                                             construct originally created pPD95 plasmid containing nuclear localization sequence <br />                                             (NLS), lines created NLS removed same <br />                                             construct. The expression construct shk-1 created using long 11-kb PCR <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             fragment encompassing upstream noncoding sequence ~800 bp the <br />                                             open reading frame. The remainder shk-1&#226;&#710;&#183;GFP construct complete genomic <br />                                             shk-1 gene open reading frame fused in-frame GFP pPD95 GFP expression plasmid. <br />                                             No NLS present construct. <br />  <br />                                             Animals injected described Salkoff et al. (27). We utilized ~90 ng/&#206;&#188;l XhoI- <br />                                             linearized plasmid DNA shl-1 100 ng/&#206;&#188;l SphI-linearized shk-1 transgene core <br />                                             constructs. For shl-1 expression construct, 75 ng/&#206;&#188;l Shal 5&#226;&#8364;&#178;-PCR 91 ng/&#206;&#188;l the <br />                                             Shal 3&#226;&#8364;&#178;-PCR added. For shk-1 expression construct, 100 ng/&#206;&#188;l Shaker PCR was <br />                                             added linearized plasmid marker plasmid. At lines examined <br />                                             shl-1 shk-1 expression analysis. <br />  <br />                               Construction Dominant Negative Transgenic Lines <br />                                             The pshl-1&#226;&#710;&#183;W363F&#226;&#710;&#183;GFP dominant negative transgenic construct created using site- <br />                                             directed mutagenesis alter tryptophan position 363 phenylalanine the <br />                                             expression construct pshl-1&#226;&#710;&#183;shl-1&#226;&#710;&#183;GFP. The dominant negative transgenic plasmid <br />                                             pshl-1&#226;&#710;&#183;W363F&#226;&#710;&#183;GFP (40 ng/&#206;&#188;l) combined PCR products encompassing <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             shl-1 promoter (40 ng/&#206;&#188;l each) lin-15+ phenotypic marker plasmid (150 ng/&#206;&#188;l) and <br />                                             injected adult hermaphrodite N2 animals. Each lines integrated exposing <br />                                             ~20 P0 L4 animals line &#206;&#179;-ray irradiation. Two integrated lines, designated 320 and <br />                                             625, obtained &gt;50 L2 animals exhibited 100% GFP expression non-Muv. <br />                                             Integration verified crossing potential integrated line TY1657 marker strain <br />                                             (dpy-5; rol-6; lon-1; bli-6; unc-23). Line 320 integrated chromosome X line 625 <br />                                             integrated chromosome IV. <br />  <br />                                             A nonintegrated pmyo-3&#226;&#710;&#183;W363F&#226;&#710;&#183;GFP transgenic line constructed. We subcloned <br />                                             GFP pmyo-3 containing plasmid pPD95.52. We subcloned W363F dominant <br />                                             negative shl-1 cDNA pPD95.52 plasmid, creating pmyo-3&#226;&#710;&#183;W363F plasmid, co- <br />                                             injected N2 animals pRF4 (rol-6). <br />  <br />  <br />  <br />  <br />                                                  J Biol Chem. Author manuscript; available PMC 2008 March 3. <br />                             Fawcett et al.                                                                                            Page 5 <br />  <br />  <br />                               RNAi <br />                                             Double-stranded RNA (dsRNA) synthesized using standard methods described Fire <br />                                             et al. (28) Christensen et al. (29). The size integrity dsRNA assayed Tris <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             boric acid/EDTA-agarose gels. Cells plated L-15 control medium L-15 medium <br />                                             containing 15 &#206;&#188;g/ml dsRNA final volume. One hour plating, dsRNA diluted a <br />                                             final concentration 5 &#206;&#188;g/ml. Media containing dsRNA replaced day. <br />                                             Electrophysiological experiments performed 2&#226;&#8364;&#8220;4 days plating cells. <br />  <br />                               C. elegans Cell Culture <br />                                             Embryonic cells isolated cultured described (29) following modifications. <br />                                             Nematode eggs separated adult carcasses sucrose gradient. Cellular debris <br />                                             carcasses removed filtration. Muscle cells identified based their <br />                                             distinctive morphology cell culture. An integrated myo-3&#226;&#710;&#183;GFP-transformed strain, which <br />                                             labeled body wall muscle cells GFP, verified method identification (29). <br />                                             Recordings performed 4&#226;&#8364;&#8220;6 days plating. <br />  <br />                               C. elegans Myocyte Electrophysiology <br />                                             Whole-cell recordings obtained using patch clamp technique (30). Whole-cell <br />                                             currents external solution contained following (in mM): 140 NaCl, 5 KCl, 5 CaCl2, 5, <br />                                             MgCl2, 11 dextrose, 5 Hepes, pH 7.2, NaOH. The 10 nM Ca2+ 4 mM Cl&#226;&#710;&#8217; internal <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             solution contained 120 K+ gluconate, 20 KOH, 2 MgCl2, 4 magnesium gluconate, 5 Tris, 0.25 <br />                                             CaCl2, 36 sucrose, 5 EGTA, 4 Na2ATP. The program EGTA (Ed McClesky, Oregon Health <br />                                             Sciences University, Vollum Institute, Portland) used calculate free Ca2+. Whole-cell <br />                                             current traces obtained applying voltage steps &#226;&#710;&#8217;70 +60 mV 10-mV <br />                                             increments holding potential &#226;&#710;&#8217;70 mV. Prepulse inactivation curves shk-1 and <br />                                             shl-1 obtained eliciting K+ currents test potential +50 mV applied a <br />                                             prepulse &#226;&#710;&#8217;70 +25 mV, 5-mV steps. The normalized current test pulse <br />                                             plotted function prepulse potential. Analysis performed above. <br />  <br />                               Strains <br />                                             We utilized slo-2 (nf100) strain isolated Salkoff laboratory DA2056 shl-1 <br />                                             (ok1168) strain provided International C. elegans Gene <span id='am-239' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-240' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-241' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-242' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-249' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-253' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-257' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-261' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-262' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-267' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-272' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-278' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-289' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-292' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-297' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-299' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-300' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-305' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-309' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span>Knockout</span> Consortium the <br />                                             Oklahoma Medical Research Foundation laboratory Dr. Leon Avery. The following <br />                                             alleles utilized studies control strains: unc-51 (e369), unc-10 (md1117), ttx-3 <br />                                             (k55), tax-6 (p675), unc-64 (e246), unc-64 (OxIs34). <br />  <br />                               Characterization shl-1 (ok1168) Breakpoints <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             The RB1144 shl-1 (ok1168) mutant obtained C. elegans Knockout Consortium <br />                                             laboratory Dr. Robert Barstead. The Knockout Consortium utilized PCR-based <br />                                             screen identify mutant, region screened encompassing ~3 kb. We utilized a <br />                                             series primers just outside region region (primer sequences available <br />                                             request) sequence screened 3-kb region identify deletion breakpoints. <br />                                             The 10&#195;&#8212; outcrossed DA2056 shl-1 (ok1168) strain obtained Dr. Leon Avery. <br />  <br />                               Aldicarb Assay <br />                                             25 age-matched hermaphrodite worms strains (N2, unc-64 (OxIs34), integrated <br />                                             pshl-1&#226;&#710;&#183;W363F&#226;&#710;&#183;GFP line 320, shl-1 (ok1168), integrated pshl-1&#226;&#710;&#183;W363F&#226;&#710;&#183;GFP line 625, and <br />                                             pmyo-3&#226;&#710;&#183;W363F&#226;&#710;&#183;GFP line 22) placed NGM-0.25 mM aldicarb plates small (1 <br />                                             cm diameter) spot OP50 bacteria (31,32). Animals examined paralysis 30 <br />                                             min 5 h, tap response lightly prodding worm tail platinum <br />  <br />  <br />                                                  J Biol Chem. Author manuscript; available PMC 2008 March 3. <br />                             Fawcett et al.                                                                                              Page 6 <br />  <br />  <br />                                             wire pick. We assayed plates strain assay. All assays performed blind <br />                                             times. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Thrashing Assay <br />                                             Age-matched hermaphrodite worms N2, shl-1 (ok1168), unc-10 (md1117) grown at <br />                                             15 &#194;&#176;C examined individually glass staining dish 60 &#206;&#188;l M9 saline (33) for~1 min. <br />                                             Assays recorded using Nikon SMZ 1500 dissecting scope PixelLink A6XX <br />                                             camera, magnification approximately &#195;&#8212;20. The Streampix program used capture all <br />                                             video data. Body bends, defined change direction bending middle the <br />                                             body (33), counted data analyzed using Sigmaplot (Jandel, San Rafael, CA). At least <br />                                             20 animals strain assay blind assays examined. <br />  <br />                               Mating Efficiency Method <br />                                             Mating efficiency scored described Hodgkin (34) following modifications. <br />                                             Two N2 DA2056 shl-1 (ok1168) males N2 hermaphrodites placed 3.5- <br />                                             cm NGM plate seeded small (20 &#206;&#188;l actively growing culture) OP50. <br />                                             Efficiency scored counting ratio male progeny total progeny, a <br />                                             successful mating plate possessing 10% male progeny. 42 mating plates strain were <br />                                             analyzed. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Turning behavior vulva location analyzed using procedure defined Loer and <br />                                             Kenyon (35) Crowder et al. (36). 10 males N2 DA2056 shl-1 (ok1168) were <br />                                             mated N2 hermaphrodites unc-51 (e369) hermaphrodites analyzed for <br />                                             turning behavior, followed location vulva. 9 10 N2 males capable exhibiting <br />                                             good turns (g) followed immediately slow search vulva using spicules (s). The <br />                                             N2 male passed vulva 10% length hermaphrodite and <br />                                             proceeded quickly swim search vulva, alternatively, turn <br />                                             hermaphrodite new approach. None DA2056 shl-1 (ok1168) males <br />                                             exhibited change turning behavior compared N2 males. DA2056 shl-1 (ok1168) <br />                                             males demonstrated variability ability locate vulva. These animals frequently <br />                                             exhibited process vulval location exhibited following <span id='am-551' about='xsp:pattern' typeof='owl:Thing'>pattern</span>: g/f/g/f/g/p/g/o, where <br />                                             &#226;&#8364;&#339;g&#226;&#8364;? good turn; &#226;&#8364;&#339;f&#226;&#8364;? fast pass vulva slowing; &#226;&#8364;&#339;p&#226;&#8364;? slow pass over <br />                                             vulva temporary pause relatively quick (&lt;10 s) continuation the <br />                                             hermaphrodite again; &#226;&#8364;&#339;o&#226;&#8364;? indicates falling hermaphrodite mating. Animals <br />                                             mated exhibited pattern similar g/f/g/f/g/s/si, &#226;&#8364;&#339;s&#226;&#8364;? slowed approach the <br />                                             vulva; &#226;&#8364;&#339;si&#226;&#8364;? indicates spicule insertion. We examined 60 males strain N2 DA2056 <br />                                             shl-1 (ok1168). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               RESULTS <br />                               Tissue Expression Patterns <br />                                             Shal channels highly conserved &#206;&#177;-subunits voltage-gated potassium <br />                                             channels. From initiator methionine amino acid residue 430, encompasses all <br />                                             membrane spanning domains, Shal &#206;&#177;-subunits 75% identity Drosophila 69% <br />                                             identity human orthologs (37). To examine native Shal (SHL-1) Shaker (SHK-1) <br />                                             currents, determined tissue expression patterns genes. We constructed a <br />                                             full-length cDNA translational fusion shl-1 ~5 kb upstream promoter region <br />                                             tagged GFP, pshl-1&#226;&#710;&#183;shl-1&#226;&#710;&#183;GFP. Two versions expression plasmid constructed, <br />                                             nuclear localization signal. Expression SHL-1 observed in <br />                                             posterior intestine, body wall muscle, vulval muscle, male-specific diagonal muscles, a <br />                                             variety motor neurons, interneurons, sensory neurons (Fig. 1, A C; Table 1). Several <br />                                             neurons muscle cells expressing SHL-1&#226;&#710;&#183;GFP known associated variety of <br />  <br />  <br />                                                  J Biol Chem. Author manuscript; available PMC 2008 March 3. <br />                             Fawcett et al.                                                                                              Page 7 <br />  <br />  <br />                                             behaviors, including thermosensation, chemosensation, dauer formation, egg-laying, male <br />                                             mating behavior, locomotion. Expression posterior intestine suggests potential role <br />                                             shl-1 defecation. We examined tissue expression Shaker gene (shk-1) using <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             translational GFP transgene (see &#226;&#8364;&#339;Materials Methods&#226;&#8364;?). The SHK-1&#226;&#710;&#183;GFP fusion protein <br />                                             expressed variety interneurons sensory neurons, body wall muscle <br />                                             (Fig. 1B). <br />  <br />                               Electrophysiological Properties SHL-1 Expressed Xenopus Ooctyes <br />                                             Prior experiments mammals, Drosophila, lobster, jellyfish showed Shal potassium <br />                                             channels carry fast, transient outward current (37). Our results that, general, the <br />                                             properties Shal channels C. elegans similar systems(supplemental <br />                                             Fig. 1 Fig. 2, A C). However, current-voltage steady-state inactivation <br />                                             relationships C. elegans Shal currents depolarized heterologously <br />                                             expressed Shal currents species (38-41). Although prepulse inactivation kinetics of <br />                                             Shal-Kv4 channels described exponential time course, results have <br />                                             hinted presence biexponential prepulse inactivation kinetics (25). In C. elegans SHL-1 <br />                                             prepulse inactivation clearly biexponential time constants decreasing with <br />                                             membrane depolarization (supplemental Fig. 1). SHL-1 channels exhibit fast deactivation <br />                                             modest voltage dependence (supplemental Fig. 2). <br />  <br />                                             In contrast shl-1 appears produce protein product, cloned expressed <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             splice variants shk-1 locus termed a, c, d2 (GenBank&#226;&#8222;&#162; accession numbers, <br />                                             respectively, CAA88477.2, CAD57716.1, CAD57717.1). Of three, isoforms a <br />                                             d2 expressed currents cRNA forms injected Xenopus oocytes. <br />                                             Ion currents basic channel properties SHK-1 isoform shown Fig. 2, B, left, and <br />                                             C, left. The SHK-1 isoform expresses currents depolarized voltage range than <br />                                             isoform d2. The V0.5a isoform 2.4 &#194;&#177; 0.8 mV ka 7.4 &#194;&#177; 0.7 mV, V0.5i is <br />                                             &#226;&#710;&#8217;19.5 &#194;&#177; 2.8 mV ki = 3.7 &#194;&#177; 0.3 mV. For isoform d2 V0.5a &#226;&#710;&#8217;31.9 &#194;&#177; 1.2 mV; kaa is <br />                                             7.4 &#194;&#177; 1.1 mV; V0.5i &#226;&#710;&#8217;62.5 &#194;&#177; 5.9 mV, ki 3.4 &#194;&#177; 0.5 mV. Currents splice variants <br />                                             inactivate slowly characterized delayed rectifiers (Fig. 2B, left). <br />  <br />                               Electrophysiology Molecular Dissection Native SHL-1 SHK-1 Cultured <br />                               Myocytes <br />                                             Because GFP promoter experiments indicated SHL-1 SHK-1 expressed body <br />                                             wall muscle cells, recorded cells culture analyzed voltage-dependent <br />                                             current components. Unlike adult body wall muscle cells, Ca2+-dependent inward currents <br />                                             seen C. elegans myocytes culture (29). Cultured myocytes easy to <br />                                             identify characteristic teardrop shape significantly <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             larger (three times) cultured neurons. In addition unequivocally identified by <br />                                             expressing GFP control body wall muscle-specific myosin heavy chain promoter <br />                                             pmyo-3 (42). We sought examine native properties SHL-1 SHK-1 isolating <br />                                             current different approaches. <br />  <br />                                             By using cultured myocytes treated RNAi, able selectively suppress either <br />                                             shl-1 shk-1 mRNA translation (28) record currents remaining treatment <br />                                             (7). Previously, shown SLO-2 K+ channels abundantly expressed cultured <br />                                             myocytes required intracellular Ca2+ Cl&#226;&#710;&#8217; relatively high levels active <br />                                             (43). We took advantage special requirements SLO-2 channels eliminate <br />                                             large SLO-2 current component, performed recordings pipette recording <br />                                             solutions containing low physiological concentrations Cl&#226;&#710;&#8217; Ca2+ (see &#226;&#8364;&#339;Materials and <br />                                             Methods&#226;&#8364;?). Patch clamp recordings wild type (N2) myocytes slo-2 (nf100) <br />                                             myocytes demonstrated voltage-dependent current components present both <br />  <br />  <br />                                                  J Biol Chem. Author manuscript; available PMC 2008 March 3. <br />                             Fawcett et al.                                                                                                Page 8 <br />  <br />  <br />                                             genotypes, fast transient component slowly inactivating component (7). Those <br />                                             experiments showed ion currents N2 cells low intracellular Cl&#226;&#710;&#8217; Ca2+ are <br />                                             indistinguishable slo-2 (nf100) myocytes. The properties observed SHL-1 and <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             SHK-1 currents expressed Xenopus oocytes (Fig. 2, A-C, left) combined GFP tissue <br />                                             expression data showing shl-1 shk-1 genes expressed body wall muscle <br />                                             (Fig. 1) led hypothesize macroscopic voltage-dependent outward K+ currents <br />                                             expressed C. elegans cultured myocytes SHL-1 SHK-1 currents. To investigate <br />                                             hypothesis, treated slo-2 (nf100) myocytes shk-1 RNAi. This resulted the <br />                                             removal slowly inactivating current component, leaving fast transient current (Fig. <br />                                             2A, right). Notably, native fast transient current strongly resembles SHL-1 current <br />                                             observed shl-1 cRNA heterologously expressed Xenopus oocytes. Conversely, <br />                                             treatment myocytes shl-1 RNAi removed fast transient current component leaving <br />                                             slowly inactivating current (7). This slowly inactivating current virtually identical to <br />                                             seen shl-1 (ok1169) deletion mutant (Fig. 2B, right). This slowly inactivating current <br />                                             resembled heterologously expressed SHK-1 current. Our data support conclusion that <br />                                             normally expressed macroscopic voltage-dependent K+ currents C. <br />                                             elegans myocytes culture SHL-1 SHK-1 channels carry currents. <br />  <br />                                             We characterized conductance/voltage steady-state inactivation relationships the <br />                                             SHL-1 SHK-1 currents recorded vivo myocytes (Fig. 2C, right). Typical Shal <br />                                             currents systems, V0.5 activation fast transient SHL-1 current more <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             hyperpolarized observed slowly inactivating SHK-1 current (11.2 20.4 <br />                                             mV, respectively). The SHL-1 current voltage-sensitive, slope factor k of <br />                                             14.1 mV versus 7.7 mV SHK-1. As expected, V0.5 steady-state inactivation also <br />                                             hyperpolarized SHL-1 currents SHK-1 currents (&#226;&#710;&#8217;33 &#226;&#710;&#8217;6.9 mV, <br />                                             respectively). Therefore, native SHL-1 current significantly hyperpolarized both <br />                                             activation steady-state inactivation SHK-1 current, agreement with <br />                                             literature relative voltage ranges currents channels (37,44). The <br />                                             conductance/voltage steady-state inactivation relationships SHL-1 SHK-1 <br />                                             currents recorded vivo myocytes (Fig. 2C, right) similar two <br />                                             currents expressed heterologously Xenopus oocytes (Fig. 2C, left) (7). <br />  <br />                                             A second way separating analyzing currents present myocytes cell culture was <br />                                             creating transgenic animal carrying dominant negative form shl-1, removed or <br />                                             greatly diminished SHL-1 current. We constructed transgenes translationally tagged <br />                                             GFP fused shl-1 dominant negative construct site-directed alteration in <br />                                             pore channel (W363F, &#226;&#8364;&#339;Materials Methods&#226;&#8364;?). One transgene under <br />                                             control native shl-1 promoter (pshl-1), control body wall <br />                                             muscle-specific promoter pmyo-3. The pshl-1&#226;&#710;&#183;W363F&#226;&#710;&#183;GFP transgenic line then <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             integrated obtain stable expression line. The effect of transgenes native <br />                                             SHL-1 current analyzed using patch clamp technique cultured myocytes. Cells <br />                                             carrying dominant negative transgene identified GFP fluorescence. In these <br />                                             cells observed current consisted primarily slowly inactivating currents. We <br />                                             compared ratio normalized peak current sustained current gauge the <br />                                             effectiveness removing SHL-1 component (Fig. 3). This analysis showed significant <br />                                             reduction peak transient current animals carrying dominant negative construct <br />                                             RNAi-treated animals. In cells small transient current remained, <br />                                             represent residual component SHL-1 current larger component of <br />                                             slower inactivating SHK-1 current. In 1 9 1 7 cells, pshl-1&#226;&#710;&#183;W363F&#226;&#710;&#183;GFP and <br />                                             pmyo-3&#226;&#710;&#183;W363F&#226;&#710;&#183;GFP, respectively, observed hyperpolarized slowly inactivating <br />                                             current, represent expression variable SHK-1 isoforms (data shown). Control <br />                                             slo-2 (nf100) cells carrying transgene clearly slowly inactivating SHK-1 <br />                                             fast transient SHL-1 currents. However, cells carrying dominant negative <br />  <br />  <br />                                                   J Biol Chem. Author manuscript; available PMC 2008 March 3. <br />                             Fawcett et al.                                                                                                    Page 9 <br />  <br />  <br />                                             transgenes showed significant reduction the fast transient outward current <br />                                             (Fig. 3). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             A deletion mutant Y73B6BL.19 shl-1 locus (International C. elegans Knockout <br />                                             Consortium) method characterized native physiological K+ currents <br />                                             cultured myocytes. DNA sequence analysis showed 5&#226;&#8364;&#178; 3&#226;&#8364;&#178; breakpoints were <br />                                             intronic sequence. The deletion removed exons containing terminal amino <br />                                             acids fourth transmembrane domain entire fifth transmembrane domain (Fig. <br />                                             4A). Furthermore, deletion resulted frameshift adding 16 ectopic amino acid residues <br />                                             early stop codon. This early termination resulted elimination membrane- <br />                                             spanning domains 5 6, residues encompassing pore channel, <br />                                             downstream sequence encompassing carboxyl cytoplasmic portions channel <br />                                             (Fig. 4B). Clearly, removal protein domains required ion conduction channel <br />                                             resulted loss-of-function. We cultured C. elegans myocytes shl-1 (ok1168) <br />                                             deletion mutant strain examined currents remaining cells using single-electrode <br />                                             whole-cell patch clamp. Our results showed fast transient current completely <br />                                             absent cells (Fig. 2B, right, Fig. 4C, bottom). The currents remaining these <br />                                             shl-1 deletion cells appeared highly similar observed majority wild-type <br />                                             slo-2 (nf100) myocytes treatment shl-1 RNAi (Fig. 4C, middle). This result is <br />                                             consistent molecular analysis shl-1 (ok1168) deletion, showing cannot <br />                                             form functional channel functions effective null homozygous. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Definitive proof SHL-1 SHK-1 constitute voltage-dependent components in <br />                                             C. elegans myocytes results demonstration selective elimination SHL-1 <br />                                             SHK-1 currents produced cells having macroscopic voltage-dependent outward current <br />                                             (43). To remove SHL-1 component, utilized myocytes transgenic dominant <br />                                             negative line (Fig. 5B) shl-1 deletion mutant (Fig. 5C). To remove SHK-1 current, <br />                                             treated myocyte genotypes shk-1 RNAi (Fig. 5, B C). In cells we <br />                                             observed complete removal macroscopic outward current. Further examination such <br />                                             cells high gain revealed single channel openings (Fig. 5, B C, right traces). These <br />                                             single channel openings exhibited detectable voltage dependence open <br />                                             probability noticeably different different holding potentials (data shown). We <br />                                             hypothesize single channel currents observed removal macroscopic <br />                                             currents genes encoding nonvoltage-dependent potassium <br />                                             channels expressed C. elegans muscle (27). We conclude three <br />                                             K+ channels produce macroscopic outward K+ currents cultured C. elegans myocytes, <br />                                             SHL-1, SHK-1, previously studied SLO-2 current (24). <br />  <br />                                             Prior results suggested certain instances altered expression Shal channels are <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             accompanied altered expression ion channel types (43). Our results C. elegans <br />                                             bear (Fig. 6). In studies using shl-1 deletion mutant, the <br />                                             remaining sustained current, likely SHK-1, up-regulated result the <br />                                             complete removal SHL-1. To test hypothesis, compared sustained current <br />                                             amplitudes slo-2 (nf100) myocytes (12.5 pA/pF &#194;&#177; 1.9), N2 myocytes (12.6 pA/pF &#194;&#177; 1.5), <br />                                             shl-1 (ok1168) myocytes (25.5 pA/pF &#194;&#177; 8.5). All currents normalized account <br />                                             variable cell size difference (45). There significant difference (p=0.66 p=0.34 <br />                                             respectively) amplitudes normalized sustained currents N2 myocytes (12.6 <br />                                             pA/pF &#194;&#177; 1.5) shl-1 RNAi-treated myocytes (14.9 pA/pF &#194;&#177; 6.8) the <br />                                             pshl-1&#226;&#710;&#183;W363F&#226;&#710;&#183;GFP (16.2 pA/pF &#194;&#177; 7.4) transgenic myocytes. However, significant <br />                                             (p &lt; 0.05) increase sustained current amplitude shl-1 (ok1168) myocytes, most <br />                                             likely up-regulation SHK-1 current. Fig. 6 summarizes findings shows <br />                                             relatively larger sustained current component shl-1 mutant myocytes control animals <br />                                             (Fig. 6A) statistical analysis data (Fig. 6B). These results parallel studies lobster <br />  <br />  <br />                                                   J Biol Chem. Author manuscript; available PMC 2008 March 3. <br />                             Fawcett et al.                                                                                                Page 10 <br />  <br />  <br />                                             demonstrated changes Shal (Kv4) channel expression levels result in <br />                                             changes whole-cell currents up-regulation channel types (46). <br />                                             Notably, studies up-regulation compensatory currents depends complete <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             absence channel protein product, ion conduction pathway protein product <br />                                             blocked. Therefore Shal protein functional role regulation, distinct <br />                                             ion conduction. <br />  <br />                               Physiological Behavioral Analysis SHL-1 <br />                                             A distinctive feature C. elegans body wall muscle resting potentials cells are <br />                                             known significantly positive resting potentials muscle cells in <br />                                             mammalian species (47). This raises possibility SHL-1 channels C. elegans, even <br />                                             functioning depolarized voltage range mammalian (or <br />                                             Drosophila) orthologs, active near muscle cell resting potential. A significant <br />                                             Shal &#226;&#8364;&#339;window current&#226;&#8364;? contribute substantially cell resting conductance thus <br />                                             basal muscle excitability muscle responsiveness neurotransmitter. If case, <br />                                             eliminating Shal current deletion mutant dominant negative construct <br />                                             increase muscle responsiveness neurotransmitter manifest increase <br />                                             sensitivity aldicarb. Aldicarb acetylcholinesterase inhibitor results delayed <br />                                             clearance acetylcholine (ACh) synaptic cleft neuromuscular junction. In C. <br />                                             elegans, accumulated ACh results rigid paralysis animals severity of <br />                                             phenotype depending ACh released cleft responsiveness of <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             postsynaptic muscle membrane ACh (48). Thus, explore possibility heightened <br />                                             responsiveness postsynaptic membrane ACh SHL-1 current removed, <br />                                             exposed shl-1 (ok1168) deletion mutant dominant negative transgenic line <br />                                             pshl-1&#226;&#710;&#183;W363F&#226;&#710;&#183;GFP aldicarb (0.25 mM) (Fig. 7A). (Note expression dominant <br />                                             negative transgene control native shl-1 promoter.) We observed rigid paralysis <br />                                             animals paralysis midpoints 125 min dominant negative transgenic <br />                                             animals 132 min shl-1 (ok1168) animals (p &lt; 0.0028 p &lt; 0.0025, <br />                                             respectively). These values significantly different observed wild-type N2 <br />                                             animals ~155 min. We observed significant (p &lt; 0.05) aldicarb hypersensitivity <br />                                             dominant negative transgenic lines native shl-1 promoter (Fig. 7B). <br />                                             We conclude SHL-1 current contribute postsynaptic muscle response <br />                                             released ACh. <br />  <br />                                             An alternative hypothesis explain results hypersensitivity a <br />                                             presynaptic effect SHL-1 current controls release ACh-containing vesicles <br />                                             presynaptic terminal. To differentiate pre- postsynaptic causes aldicarb <br />                                             hypersensitivity, expressed dominant negative transgene control body wall <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             muscle-specific myo-3 promoter (49) limit expression dominant negative construct to <br />                                             muscle. The demonstration pmyo-3&#226;&#710;&#183;W363F&#226;&#710;&#183;GFP transgenic line aldicarb- <br />                                             hypersensitive prove absence SHL-1 current muscle was <br />                                             sufficient confer aldicarb hypersensitivity. Indeed, pmyo-3&#226;&#710;&#183;W363F&#226;&#710;&#183;GFP transgenic line <br />                                             exhibited identical aldicarb hypersensitivity exhibited transgenic line <br />                                             pshl-1&#226;&#710;&#183;W363F&#226;&#710;&#183;GFP line 320 (Fig. 7C). Heterozygous shl-1(ok1168)/+ animals exhibited an <br />                                             intermediate level aldicarb sensitivity (Fig. 7D). This result appears validate the <br />                                             hypothesis increased sensitivity accumulated ACh neuromuscular junction is <br />                                             postsynaptic effect. <br />  <br />                                             We examined variety behaviors shl-1 (ok1168) mutant animals effort to <br />                                             understand shl-1 deletion mutant phenotype. shl-1 (ok1168) animals exhibited defect in <br />                                             aldicarb sensitivity slight significant alteration thrashing behavior (data shown). <br />                                             However, observed mating behavior presents rigorous behavioral assay reveal <br />  <br />  <br />                                                   J Biol Chem. Author manuscript; available PMC 2008 March 3. <br />                             Fawcett et al.                                                                                             Page 11 <br />  <br />  <br />                                             subtle defects sensory processing; mating requires complex coordination sensory <br />                                             information intricate motor functions. Mating efficiency determined observing the <br />                                             number mating plates possessing male progeny. Hermaphrodite C. elegans produce male <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             progeny low penetrance unless mated males (50). Two N2 shl-1 (ok1168) males <br />                                             N2 hermaphrodites placed small spot (~1 cm diameter) mating plates. <br />                                             The number successful mating events experiments, indicated number of <br />                                             mating plates male progeny, significantly lower plates shl-1 (ok1168) male <br />                                             animals N2 males (48.8 85.4%, respectively) (Table 2). One possible <br />                                             complication interpreting results phenomenon feminization worms <br />                                             possessing XO sex chromosome genotype exhibit female gonads sexual behavior, <br />                                             instead male. Conceivably, SHL-1 protein involved sex <br />                                             determination turning males phenotypic hermaphrodites (50,51). To eliminate this <br />                                             possibility, examined proportion males total progeny individual mating <br />                                             plates. The proportion male progeny successfully mated N2 hermaphrodite &#195;&#8212; N2 male <br />                                             plates statistically different proportion male progeny successfully mated <br />                                             N2 hermaphrodite &#195;&#8212; shl-1 (ok1168) male plates. This eliminated possibility feminization <br />                                             producing observed difference wild type mutant mating efficiency. We <br />                                             conclude shl-1 (ok1168) strain exhibits decreased mating efficiency compared <br />                                             N2 animals. <br />  <br />                                             As follow-up studies sought determine defective steps shl-1 mutant <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             mating behavior carefully observing detailed behavior shl-1 (ok1168) males and <br />                                             comparing wild-type N2 males (35,36). These observations showed mutant <br />                                             males effective locating vulva hermaphrodite. On average, mutant males <br />                                             missed vulva location passing 58.1% time compared wild-type animals, <br />                                             unable locate vulva 42.1% time (p &lt; 0.05). Furthermore, mutant <br />                                             animals unable locate vulva pass moved past vulva 25% of <br />                                             length hermaphrodite. N2 males typically ~10% length <br />                                             hermaphrodite passing vulva. We conclude shl-1 (ok1168) males are <br />                                             capable fine motor control mating behavior N2 animals. Some measure of <br />                                             deficit fine motor control mating involve fact SHL-1 expressed in <br />                                             male-specific diagonal muscles tail region. <br />  <br />                               DISCUSSION <br />                                             SHL-1 SHK-1 C. elegans voltage-gated potassium channels exhibit fast transient and <br />                                             delayed rectifier currents, respectively. In species, Shal channels typically described <br />                                             operating hyperpolarized voltage range relative voltage-gated K+ channels <br />                                             (37,52). However, C. elegans, SHL-1 activates depolarized voltage range is <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             typically observed Shal channels species (37). Recent work demonstrated that <br />                                             pyramidal neurons rat visual cortex activate depolarized voltages are <br />                                             typically observed mammals (16). Nevertheless, SHK-1 currents operate more <br />                                             depolarized voltage range maintain relative functional relationship to <br />                                             SHL-1 channels, consistent species (25,38-41,53,54). Furthermore, <br />                                             evidence C. elegans body wall muscle rests positive value relative to <br />                                             muscle cells species (approximately &#226;&#710;&#8217;20 &#226;&#710;&#8217;30 mV (47,55)) suggests that <br />                                             SHL-1 channels evolved serve specialized purpose species. A second gene <br />                                             encoding transient K+ channel reported C. elegans, kvs-1 (56). KVS-1 channels <br />                                             exhibit depolarized activation inactivation range observed for <br />                                             SHL-1 channels; exhibits significantly slower inactivation kinetics. kvs-1 closer to <br />                                             Shab (Kv2) family voltage-gated K+ channels Shal family (57), gene <br />                                             expressed body wall muscle. <br />  <br />  <br />  <br />                                                  J Biol Chem. Author manuscript; available PMC 2008 March 3. <br />                             Fawcett et al.                                                                                               Page 12 <br />  <br />  <br />                                             Native SHL-1 channels properties similar heterologously expressed cloned <br />                                             channels carry fast transient currents rapid activation inactivation. <br />                                             However, native channels activate somewhat hyperpolarized voltage range than <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             cloned channels (approximately &#226;&#710;&#8217;14 mV). Such differences conceivably to <br />                                             accessory proteins post-translational modification occurring native cells. In mammals, <br />                                             modulatory subunits post-translational modification affects Shal channel voltage sensitivity, <br />                                             similar mechanisms known affect Shaker currents well. In C. elegans one <br />                                             protein similar mammalian modulatory subunit identified (58). <br />  <br />                                             We showed major component voltage-dependent current remaining shl-1 <br />                                             (ok1168) deletion mutant background encoded Shaker gene, shk-1, the <br />                                             properties delayed rectifier current. We characterized isoforms produced the <br />                                             shk-1 locus, isoform isoform d2. Both express slowly inactivating currents, d2 <br />                                             isoform SHK-1 functions hyperpolarized voltage range SHK-1 isoform and <br />                                             inactivate slightly faster. When Shal component removed RNAi cell <br />                                             culture cells strain carrying dominant negative form Shal, remaining <br />                                             current properties SHK-1 isoform d2. However, current remaining <br />                                             shl-1 deletion strain properties similar SHK-1 isoform a. In latter <br />                                             case, SHK-1 currents appeared larger. Precedents exist literature, which <br />                                             second current component up-regulated compensate removal of <br />                                             component. The compensating component differ radically properties the <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             original component, removed (59,60). For example, changes HCN ion channel <br />                                             expression response increased Shal subunit expression observed previously in <br />                                             lobster stomatogastric ganglion cells (46,61). In case, up-regulation SHK-1 <br />                                             current observed shl-1 RNAi-treated myocytes dominant negative <br />                                             transgenic line pshl-1&#226;&#710;&#183;W363F. Because deletion mutant exhibited increased <br />                                             SHK-1 current, compensatory expression current responsive lack of <br />                                             SHL-1 protein, functional not. <br />  <br />                                             Although complete knockout SHL-1 SHK-1 slo-2 (nf100) background <br />                                             eliminated macroscopic outward currents, unidentified unitary currents remained. We <br />                                             hypothesize single-channel currents likely two-pore twk channels <br />                                             expressed body wall muscle (27). However, macroscopic outward currents C. <br />                                             elegans cultured myocytes entirely carried SLO-2, SHK-1, SHL-1 channels. <br />  <br />                                             Detailed analysis shl-1-GFP cell-type expression patterns showed channel expression in <br />                                             cells involved variety behaviors. However, mutant phenotypes observed for <br />                                             chemosensation, thermosensation, defecation, pharyngeal pumping. On hand, we <br />                                             observed significant shl-1 mutant phenotype regard response aldicarb. The total <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             elimination SHL-1 current tissues <span id='am-317' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-318' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-319' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-320' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-321' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-322' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-323' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-324' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-325' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-326' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-327' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-328' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-330' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-331' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-332' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-333' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-334' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-335' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-337' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-338' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-341' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-342' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-344' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-347' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-348' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-349' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-350' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-351' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-352' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-353' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-354' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-355' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-356' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-357' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-358' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-359' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-362' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-363' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-364' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-365' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-366' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-367' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-368' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-369' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-370' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-371' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-372' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-373' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-374' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-375' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-376' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-377' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-378' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-380' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-381' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-382' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-383' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-384' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-385' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-386' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-389' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-390' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-391' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-392' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-393' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-394' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span>deletion mutation</span>, specific <br />                                             inhibition SHL-1 current body wall muscle selective muscle-specific expression of <br />                                             dominant negative construct, resulted aldicarb hypersensitivity. This result strongly <br />                                             suggests muscle membrane itself site confers aldicarb hypersensitivity. An <br />                                             attractive hypothesis takes account unique properties SHL-1 current that <br />                                             mutant membrane susceptible depolarization acetylcholine wild-type <br />                                             membrane, SHL-dependent component membrane resting conductance been <br />                                             removed. This come SHL-1 window current, contribute the <br />                                             resting membrane K+ conductance controlling basal resting membrane excitability. The fact <br />                                             SHL-1 current contributes K+ conductance rest does necessarily mean that <br />                                             significantly changes resting membrane potential. Indeed, membrane resting <br />                                             potential close EK, driving force K+ low small significant change <br />                                             K+ conductance measurable effect resting potential. However, <br />                                             current components active rest need large major impact the <br />  <br />  <br />                                                   J Biol Chem. Author manuscript; available PMC 2008 March 3. <br />                             Fawcett et al.                                                                                                                  Page 13 <br />  <br />  <br />                                             overall excitability cell, significantly change resting membrane <br />                                             potential. This total cell resting conductance usually small represents a <br />                                             tiny fraction overall cell membrane conductance peak electrical activity. A crucial <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             factor contribution critical balance inward outward currents threshold <br />                                             active responses. Many models change ~10% resting cell conductance <br />                                             major effect excitable properties cell. <br />  <br />                                             We demonstrated significant reduction mating efficiency shl-1 (ok1168) deletion mutant <br />                                             males, partially involves reduced ability locate vulva the <br />                                             hermaphrodite. Male copulatory behavior includes input neurons common both <br />                                             hermaphrodite male, male-specific neurons (note did specifically <br />                                             examine male-specific neurons SHL-1 channel expression). The difficulty shl-1 <br />                                             (ok1168) animals effectively mate appears to deficit fine motor control or <br />                                             spicule insertion (62,63), result mutant alteration membrane excitability <br />                                             muscle neurons both. The ether-ago-go-related-like potassium channel, UNC-103, is <br />                                             known impact male-specific muscle contraction leading failure spicule insertion into <br />                                             vulva hermaphrodite reduces mating efficiency (62). Conceivably, SHL-1 <br />                                             functions similar manner, removal SHL-1 channels results hypersensitivity of <br />                                             muscle ACh, lowering efficiency spicule insertion impeding sensory perception <br />                                             vulva. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Loss shl-1 gene product produces cell organismal lethal. Nevertheless, <br />                                             role fine-tuning complicated behavior mating offer essential survival benefits <br />                                             factor contributing extraordinarily high conservation Shal <br />                                             potassium channel. <br />  <br />                               Supplementary Material <br />                                             Refer Web version PubMed Central supplementary material. <br />  <br />                               Acknowledgements <br />                                             We thank Dr. Cornelia Bargmann detailed cell expression analysis SHL-1 protein. We thank Dr. Michael <br />                                             Nonet cell expression studies, helpful suggestions, sharing equipment, mutant strains behavioral assays. <br />                                             We thank Dr. Crowder help mating, chemosensation, thermosensation behavioral assays. We thank <br />                                             Dr. Leon Avery providing backcrossed shl-1 (ok1168) strain. We thank Drs. Gonzalo Ferreira, Michael <br />                                             Crowder, Aaron DiAntonio, Jeanne Nerbonne, Michael Nonet comments discussion course <br />                                             work. We thank Dr. Tim Schedl suggesting mating phenotype direction inquiry. We also <br />                                             thank Dr. Aguan Wei Dr. Alex Yuan critical comments suggestions. <br />  <br />  <br />                               References <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             1. Tsunoda S, Salkoff L. J Neurosci 1995;15:1741&#226;&#8364;&#8220;1754. [PubMed: 7891132] <br />                                             2. Guo W, Jung WE, Marionneau C, Aimond F, Xu H, Yamada KA, Schwarz TL, Demolombe S, <br />                                                Nerbonne JM. Circ Res 2005;97:1342&#226;&#8364;&#8220;1350. [PubMed: 16293790] <br />                                             3. Hagiwara S, Kusano K, Saito N. J Physiol (Lond) 1961;155:471&#226;&#8364;&#8220;489. <br />                                             4. Connor JA, Stevens CF. J Physiol (Lond) 1971;213:21&#226;&#8364;&#8220;30. [PubMed: 5575340] <br />                                             5. Neher E. J Gen Physiol 1971;58:36&#226;&#8364;&#8220;53. [PubMed: 5564761] <br />                                             6. Thompson SH. J Physiol (Lond) 1977;265:465&#226;&#8364;&#8220;488. [PubMed: 850203] <br />                                             7. Santi CM, Yuan A, Fawcett G, Wang Z-W, Butler A, Nonet ML, Wei A, Rojas P, Salkoff L. Proc Natl <br />                                                Acad Sci USA 2003;100:14391&#226;&#8364;&#8220;14396. [PubMed: 14612577] <br />                                             8. Xu H, Li H, Nerbonne JM. J Physiol (Lond) 1999;519:11&#226;&#8364;&#8220;21. [PubMed: 10432335] <br />                                             9. Hahn J, Tse TE, Levitan ES. J Neurosci 2003;23:10859&#226;&#8364;&#8220;10866. [PubMed: 14645479] <br />                                             10. Ohya S, Tanaka M, Oku T, Asai Y, Watanabe M, Giles WR, Imaizumi Y. FEBS Lett 1997;420:47&#226;&#8364;&#8220; <br />                                                 53. [PubMed: 9450548] <br />  <br />  <br />                                                    J Biol Chem. Author manuscript; available PMC 2008 March 3. <br />                             Fawcett et al.                                                                                              Page 14 <br />  <br />  <br />                                             11. Birnbaum SG, Varga AW, Yuan L-L, Anderson AE, Sweatt JD, Schrader LA. Physiol Rev <br />                                                  2004;84:803&#226;&#8364;&#8220;833. [PubMed: 15269337] <br />                                             12. Connor JA, Stevens CF. J Physiol (Lond) 1971;213:31&#226;&#8364;&#8220;53. [PubMed: 5575343] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             13. Shibata R, Nakahira K, Shibasaki K, Wakazono Y, Imoto K, Ikenaka K. J Neurosci 2000;20:4145&#226;&#8364;&#8220; <br />                                                  4155. [PubMed: 10818150] <br />                                             14. Yuan L-L, Adams JP, Swank M, Sweatt JD, Johnston D. J Neurosci 2002;22:4860&#226;&#8364;&#8220;4868. [PubMed: <br />                                                  12077183] <br />                                             15. Ramakers GMJ, Storm JF. Proc Natl Acad Sci USA 2002;99:10144&#226;&#8364;&#8220;10149. [PubMed: 12114547] <br />                                             16. Yuan W, Burkhalter A, Nerbonne JM. J Neurosci 2005;25:9185&#226;&#8364;&#8220;9194. [PubMed: 16207878] <br />                                             17. Cai X, Lian CW, Muralidharan S, Kao JPY, Tang C-M, Thompson SM. Neuron 2004;44:351&#226;&#8364;&#8220;364. <br />                                                  [PubMed: 15473972] <br />                                             18. Sah T, Ramirez RJ, Oudit GY, Gidrewicz D, Trivieri MG, Zobel C, Backx PH. J Physiol (Lond) <br />                                                  2003;546:5&#226;&#8364;&#8220;18. [PubMed: 12509475] <br />                                             19. Guo W, Xu H, London B, Nerbonne JM. J Physiol (Lond) 1999;521:587&#226;&#8364;&#8220;599. [PubMed: 10601491] <br />                                             20. Nerbonne JM. J Physiol (Lond) 2000;525:285&#226;&#8364;&#8220;298. [PubMed: 10835033] <br />                                             21. Amberg GC, Koh SD, Hatton WJ, Murray KJ, Monaghan K, Horowitz B, Sanders KM. J Physiol <br />                                                  (Lond) 2002;544:403&#226;&#8364;&#8220;415. [PubMed: 12381814] <br />                                             22. Amberg GC, Koh SD, Imaizumi Y, Ohya S, Sanders KM. Am J Physiol 2003;284:C583&#226;&#8364;&#8220;C595. <br />                                             23. Barry DM, Xu H, Schuessler RB, Nerbonne JM. Circ Res 1998;83:560&#226;&#8364;&#8220;567. [PubMed: 9734479] <br />                                             24. Yuan A, Dourado M, Butler A, Walton N, Wei A, Salkoff L. Nat Neurosci 2000;3:771&#226;&#8364;&#8220;779. [PubMed: <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  10903569] <br />                                             25. Beck EJ, Covarrubias M. Biophys J 2001;81:867&#226;&#8364;&#8220;883. [PubMed: 11463631] <br />                                             26. Jerng HH, Covarrubias M. Biophys J 1997;72:163&#226;&#8364;&#8220;174. [PubMed: 8994601] <br />                                             27. Salkoff L, Butler A, Fawcett G, Kunkel M, McArdle C, Paz-Y-Mino G, Nonet M, Walton N, Wang <br />                                                  Z-W, Yuan A, Wei A. Neuroscience 2001;103:853&#226;&#8364;&#8220;859. [PubMed: 11301195] <br />                                             28. Fire A, Xu S, Montgomery MK, Kostas SA, Driver SE, Mello CC. Nature 1998;391:806&#226;&#8364;&#8220;811. <br />                                                  [PubMed: 9486653] <br />                                             29. Christensen M, Estevez A, Yin X, Fox R, Morrison R, McDonnell M, Gleason C, Miller DM III, <br />                                                  Strange K. Neuron 2002;33:503&#226;&#8364;&#8220;514. [PubMed: 11856526] <br />                                             30. Hamill OP, Marty A, Neher E, Sakmann B, Sigworth FJ. Pfluegers Arch Eur J Physiol 1981;391:85&#226;&#8364;&#8220; <br />                                                  100. [PubMed: 6270629] <br />                                             31. Jorgensen EM, Hartwieg E, Schuske K, Nonet ML, Jin Y, Horvitz HR. Nature 1995;378:196&#226;&#8364;&#8220;199. <br />                                                  [PubMed: 7477324] <br />                                             32. Saifee O, Wei L, Nonet ML. Mol Biol Cell 1998;9:1235&#226;&#8364;&#8220;1252. [PubMed: 9614171] <br />                                             33. Sulston, J.; Hodgkin, J. The Nematode Caenorhabditis elegans. Wood, WB., editor. Cold Spring <br />                                                  Harbor Laboratory Press; Cold Spring Harbor, NY: 1988. p. 587-607. <br />                                             34. Hodgkin J. Nature 1983;304:267&#226;&#8364;&#8220;268. [PubMed: 6866126] <br />                                             35. Loer CM, Kenyon CJ. J Neurosci 1993;13:5407&#226;&#8364;&#8220;5417. [PubMed: 8254383] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             36. Crowder CM, Shebester LD, Schedl T. Anesthesiology 1996;85:901&#226;&#8364;&#8220;912. [PubMed: 8873562] <br />                                             37. Jerng HH, Pfaffinger PJ, Covarrubias M. Mol Cell Neurosci 2004;27:343&#226;&#8364;&#8220;369. [PubMed: 15555915] <br />                                             38. Pak MD, Baker K, Covarrubias M, Butler A, Ratcliffe A, Salkoff L. Proc Natl Acad Sci USA <br />                                                  1991;88:4386&#226;&#8364;&#8220;4390. [PubMed: 2034678] <br />                                             39. Baro DJ, Coniglio LM, Cole CL, Rodrigues HE, Lubell JK, Kim MT, Harris-Warrick RM. J Neurosci <br />                                                  1996;16:1689&#226;&#8364;&#8220;1701. [PubMed: 8774437] <br />                                             40. Holmqvist MH, Cao J, Hernandez-Pineda R, Jacobson MD, Carroll KI, Sung MA, Betty M, Ge P, <br />                                                  Gilbride KJ, Brown ME, Jurman ME, Lawson D, Silos-Santiago I, Xie Y, Covarrubias M, Rhodes <br />                                                  KJ, Distefano PS, An WF. Proc Natl Acad Sci USA 2002;99:1035&#226;&#8364;&#8220;1040. [PubMed: 11805342] <br />                                             41. B&#195;&#164;hring R, Boland LM, Varghese A, Gebauer M, Pongs O. J Physiol (Lond) 2001;535:65&#226;&#8364;&#8220;81. <br />                                                  [PubMed: 11507158] <br />                                             42. Okkema PG, Harrison SW, Plunger V, Aryana A, Fire A. Genetics 1993;135:385&#226;&#8364;&#8220;404. [PubMed: <br />                                                  8244003] <br />  <br />  <br />  <br />                                                  J Biol Chem. Author manuscript; available PMC 2008 March 3. <br />                             Fawcett et al.                                                                                               Page 15 <br />  <br />  <br />                                             43. Yuan A, Santi CM, Wei A, Wang Z-W, Pollak K, Nonet M, Kaczmarek L, Crowder CM, Salkoff L. <br />                                                 Neuron 2003;37:765&#226;&#8364;&#8220;773. [PubMed: 12628167] <br />                                             44. Covarrubias M, Wei A, Salkoff L. Neuron 1991;7:763&#226;&#8364;&#8220;773. [PubMed: 1742024] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             45. Kovac M, Davis WJ, Matera E, Gillette R. J Neurophysiol 1982;47:909&#226;&#8364;&#8220;927. [PubMed: 7086475] <br />                                             46. MacLean JN, Zhang Y, Johnson BR, Harris-Warrick RM. Neuron 2003;37:109&#226;&#8364;&#8220;120. [PubMed: <br />                                                 12526777] <br />                                             47. Jospin M, Mariol M-C, Segalat L, Allard B. J Physiol (Lond) 2002;544:373&#226;&#8364;&#8220;384. [PubMed: <br />                                                 12381812] <br />                                             48. Nonet ML, Grundahl K, Meyer BJ, Rand JB. Cell 1993;73:1291&#226;&#8364;&#8220;1305. [PubMed: 8391930] <br />                                             49. Waterston, RH. The Nematode Caenorhabditis elegans. Wood, WB., editor. Cold Spring Harbor <br />                                                 Laboratory Press; Cold Spring Harbor, NY: 1988. p. 281-336. <br />                                             50. Hodgkin J. J Embryol Exp Morphol 1984;83:103&#226;&#8364;&#8220;117. [PubMed: 6533237] <br />                                             51. Hodgkin J. Development (Camb) 1987;101:5&#226;&#8364;&#8220;16. <br />                                             52. Coetzee WA, Amarillo Y, Chiu J, Chow A, Lau D, McCormack T, Moreno H, Nadal MS, Ozaita A, <br />                                                 Pountney D, Saganich M, Vega-Saenz Miera E, Rudy B. Ann N Y Acad Sci 1999;868:233&#226;&#8364;&#8220;285. <br />                                                 [PubMed: 10414301] <br />                                             53. Wei A, Covarrubias M, Butler A, Baker K, Pak M, Salkoff L. Science 1990;248:599&#226;&#8364;&#8220;603. [PubMed: <br />                                                 2333511] <br />                                             54. Dilks D, Ling HP, Cockett M, Sokol P, Numann R. J Neurophysiol 1999;81:1974&#226;&#8364;&#8220;1977. [PubMed: <br />                                                 10200233] <br />                                             55. Hille, B. Ionic Channels Excitable Membranes. Sinauer Associates Inc.; Sunderland, MA: 1992. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 p. 76-78.p. 115-139.p. 507 <br />                                             56. Bianchi L, Kwok SM, Driscoll M, Sesti F. J Biol Chem 2003;278:12415&#226;&#8364;&#8220;12424. [PubMed: 12533541] <br />                                             57. Wei A, Jegla T, Salkoff L. Neuropharmacology 1996;35:805&#226;&#8364;&#8220;829. [PubMed: 8938713] <br />                                             58. Price M, Thompson RJ, Eshcol JO, Wemmie JA, Benson CJ. J Biol Chem 2004;279:53886&#226;&#8364;&#8220;53891. <br />                                                 [PubMed: 15471860] <br />                                             59. Takahashi E, Ino M, Miyamoto N, Nagasu T. Neurosci Lett 2004;359:37&#226;&#8364;&#8220;40. [PubMed: 15050706] <br />                                             60. Guo W, Li H, London B, Nerbonne JM. Circ Res 2000;87:73&#226;&#8364;&#8220;79. [PubMed: 10884375] <br />                                             61. Baro DJ, Ayali A, French L, Scholz NL, Labenia J, Lanning CC, Graubard K, Harris-Warrick RM. <br />                                                 J Neurosci 2000;20:6619&#226;&#8364;&#8220;6630. [PubMed: 10964967] <br />                                             62. Garcia LR, Sternberg PW. J Neurosci 2003;23:2696&#226;&#8364;&#8220;2705. [PubMed: 12684455] <br />                                             63. Garcia LR, Mehta P, Sternberg PW. Cell 2001;107:777&#226;&#8364;&#8220;788. [PubMed: 11747813] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   J Biol Chem. Author manuscript; available PMC 2008 March 3. <br />                             Fawcett et al.                                                                                       Page 16 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             FIGURE 1. Cell-type expression patterns shl-1 shk-1 <br />                                             A, GFP expression transgene pshl-1&#226;&#710;&#183;shl-1&#226;&#710;&#183;GFP N2 animals. Expression observed <br />                                             variety neurons, including pharyngeal neurons (top left, white arrow), interneurons <br />                                             (bottom left, arrow), sensory neurons (bottom left, curved arrow), phasmid neurons <br />                                             (bottom right, white arrow). In addition, observed expression muscle types, body <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             wall muscle (bottom left, arrow) vulval muscle (top right, white arrow). B, GFP <br />                                             expression transgene pshk-1&#226;&#710;&#183;shk-1&#226;&#710;&#183;GFP N2 animals. Expression observed a <br />                                             variety neurons ganglia head (top). Expression body wall muscle <br />                                             cells visible (bottom) striated pattern. C, top, GFP expression transgene <br />                                             pshl-1&#226;&#710;&#183;shl-1&#226;&#710;&#183;GFP male N2 animals. Expression observed male-specific <br />                                             diagonal muscle cells. C, bottom, Nomarski picture GFP view above. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  J Biol Chem. Author manuscript; available PMC 2008 March 3. <br />                             Fawcett et al.                                                                                               Page 17 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             FIGURE 2. Comparison conductance voltage steady-state inactivation properties SHK-1 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             SHL-1 C. elegans slo-2 (nf100) myocytes heterologously expressed Xenopus oocytes <br />                                             A, left, currents SHL-1 heterologously expressed Xenopus oocytes obtained under <br />                                             voltage clamp holding potential &#226;&#710;&#8217;100 mV 10-mV steps &#226;&#710;&#8217;80 +50 mV for <br />                                             450 ms 5-s interpulse intervals. A, right, whole-cell currents shk-1 RNAi treated <br />                                             slo-2(nf100) myocytes obtained voltage clamp holding potential &#226;&#710;&#8217;70 mV <br />                                             10-mV steps &#226;&#710;&#8217;70 +60 mV 5-s interpulse intervals. B, left, currents from <br />                                             SHK-1 isoform heterologously expressed Xenopus oocytes obtained voltage clamp <br />                                             holding potential &#226;&#710;&#8217;100 mV 10-mV steps &#226;&#710;&#8217;80 +60 mV. B, right, currents <br />                                             shl-1(ok1168) myocytes obtained voltage clamp holding potential &#226;&#710;&#8217;70 <br />                                             mV 10-mV steps &#226;&#710;&#8217;70 mV +60 mV 5-s interpulse intervals. C, left, steady- <br />                                             state inactivation activation data plotted SHL-1 SHK-1 heterologously expressed <br />                                             Xenopus oocytes. &#226;&#8211;&#161;, SHL-1; &#226;&#8211;&#160;, SHK-1. Best fits Boltzmann equation shown lines <br />                                             data points. V0.5a = 25.0&#194;&#177;1.6 mV (n = 7), ka = 14. 0&#194;&#177;1.2 mV;V0.5i = &#226;&#710;&#8217;36.7 &#194;&#177; 0.7 mV <br />                                             (n = 7), ki = 7. 3 &#194;&#177; 0.6 mV. V0.5a = 2. 4 &#194;&#177; 0.8 mV (n = 8), ka = 7. 4 &#194;&#177; 0.7 mV; V0.5i = &#226;&#710;&#8217;19. 5 &#194;&#177; <br />                                             2.8 mV (n = 8), ki = 3. 7 &#194;&#177; 0.3 mV. Calculated conductance used equation G = I/(E &#226;&#710;&#8217; EK), <br />                                             G conductance; I measured current; E test voltage, EK estimated <br />                                             &#226;&#710;&#8217;96 mV. Steady-state inactivation data obtained using voltage step protocol stepping <br />                                             &#226;&#710;&#8217;120 +20 mV 10-mV steps, holding potential &#226;&#710;&#8217;100 mV. Normalization <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             obtained following ratio: I/Imax, Imax maximum observed current, and <br />                                             I current given voltage. C, right, steady-state inactivation activation data were <br />                                             plotted SHL-1 SHK-1 currents observed cultured C. elegans myocytes. The <br />                                             activation parameter values V0. 5a = 11.2 &#194;&#177; 1.5 mV, ka = 14.1 &#194;&#177; 1.04 (n = 5); V0.5a = 20. <br />                                             4 &#194;&#177; 2, ka = 7. 7 &#194;&#177; 1.1 SHL-1 (&#226;&#8211;&#161;)(n = 3) SHK-1 (&#226;&#8211;&#160;) (n = 7) currents, respectively. SHL-1 <br />                                             currents inactivated values V0.5i = &#226;&#710;&#8217;33.1 mV &#194;&#177; 1.2 ki = 8.3 &#194;&#177; 0.7 (n = 2), whereas <br />                                             SHK-1 inactivation parameter values V0. 5i = &#226;&#710;&#8217;6.95&#194;&#177;1.7, ki = 5.8&#194;&#177;0.5 (n =3). Internal <br />                                             concentrations Cl&#226;&#710;&#8217; Ca2+ 120 mM 10 nM, respectively. Previously published in <br />                                             Santi et al. (7). <br />  <br />  <br />  <br />  <br />                                                   J Biol Chem. Author manuscript; available PMC 2008 March 3. <br />                             Fawcett et al.                                                                                         Page 18 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             FIGURE 3. Removal SHL-1 current cultured C. elegans myocytes <br />                                             The ratio peak current (pA/pF) minimum sustained current (pA/pF) plotted for <br />                                             wild type N2 strain (3.0831 &#194;&#177; 0.1973) (n = 7), dominant negative strains <br />                                             pshl-1&#226;&#710;&#183;W363F (1. 5623&#194;&#177;0. 1101) (n=5) pmyo-3&#226;&#710;&#183;W363F (2.2910&#194;&#177;0.0020) (n=5), <br />                                             shl-1RNAi-treated myocytes (1.5723&#194;&#177;0.1502) (n=3), deletion mutant shl-1(ok1168) <br />                                             (1.8088&#194;&#177;0.1324) (n=6). * signifies p &lt; 0.05. ** signifies p &lt; 0.0005. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  J Biol Chem. Author manuscript; available PMC 2008 March 3. <br />                             Fawcett et al.                                                                                              Page 19 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             FIGURE 4. Characterization shl-1 (ok1168) deletion mutant breakpoints <br />                                             A, schematic shl-1 genomic locus C. elegans showing 10 exons gene. The black <br />                                             box represents genomic area deleted shl-1(ok1168) deletion mutant. B, pictorial <br />                                             representation truncated protein produced shl-1(ok1168) mutant. <br />                                             The deletion removes terminal 3 amino acids fourth transmembrane domain, a <br />                                             frameshift resulting splicing exons 3 6 results missense coding 16 amino <br />                                             acids ending stop codon. The portion channel removed designated <br />                                             dashed line. C, slo-2(nf100) myocytes display voltage-sensitive macroscopic outward <br />                                             currents presence SHL-1 SHK-1 channels (n = 7) (top). RNAi removal <br />                                             SHL-1 current results retention slowly inactivating voltage-sensitive current, SHK-1 <br />                                             (n = 3) (middle). The shl-1(ok1168) deletion mutant (n= 7) results removal fast transient <br />                                             current (bottom). Cells held &#226;&#710;&#8217;70 mV stepped &#226;&#710;&#8217;70 +60 mV 10-mV <br />                                             increments. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  J Biol Chem. Author manuscript; available PMC 2008 March 3. <br />                             Fawcett et al.                                                                                          Page 20 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             FIGURE 5. Removal macroscopic outward currents C. elegans myocytes <br />                                             A, currents slo-2(nf100) cells shown control SHL-1 SHK-1 <br />                                             currents present. Whole-cell recordings obtained holding potential &#226;&#710;&#8217;70 mV, <br />                                             stepped &#226;&#710;&#8217;70 +60 mV 10-mV intervals. B, dominant negative <br />                                             pshl-1&#226;&#710;&#183;shl-1&#226;&#710;&#183;W363F cells (n = 3) treated shk-1 RNAi remove SHL-1 SHK-1 <br />                                             macroscopic currents. SLO-2 currents inhibited limiting intracellular Ca2+ and <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Cl&#226;&#710;&#8217;. There complete removal macroscopic SHL-1 SHK-1 currents these <br />                                             myocytes. Unidentified single channel openings seen records (right traces). C, <br />                                             shl-1(ok1168) deletion mutant cells (n = 3) treated shk-1 RNAi B. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  J Biol Chem. Author manuscript; available PMC 2008 March 3. <br />                             Fawcett et al.                                                                                      Page 21 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             FIGURE 6. Up-regulation SHK-1 current shl-1 (ok1168) mutant myocytes <br />                                             A, current traces shl-1 (ok1168) myocytes (top), N2 wild-type myocytes (middle), and <br />                                             slo-2 (nf100) myocytes (bottom) shown scale normalized cell capacitance. <br />                                             B, mean normalized sustained current amplitudes standard errors slo-2 (nf100) <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             myocytes (12.5 pA/pF&#194;&#177;1.9), N2 myocytes (12.6 pA/pF&#194;&#177;1.5), shl-1 (ok1168) myocytes <br />                                             (25.5 pA/pF &#194;&#177; 8.5). * indicates p &lt; 0.05 compared shl-1 (ok1168) myocytes. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  J Biol Chem. Author manuscript; available PMC 2008 March 3. <br />                             Fawcett et al.                                                                                             Page 22 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             FIGURE 7. Removal SHL-1 current body wall muscle sufficient confer aldicarb <br />                                             hypersensitivity 0.25 mM aldicarb <br />                                             A, shl-1(ok1168) deletion mutant pshl-1&#226;&#710;&#183;W363F&#226;&#710;&#183;GFP line 320 carrying the <br />                                             dominant negative construct displayed aldicarb hypersensitivity compared N2 animals. <br />                                             B, tested lines carrying dominant negative constructs integrated different <br />                                             chromosomes ensure observed aldicarb hypersensitivity phenotype because <br />                                             position effects transgenic animals. Both lines carrying integrated constructs (320 <br />                                             chromosome X 625 chromosome IV) displayed aldicarb hypersensitivity. The <br />                                             aldicarb hypersensitivity dominant negative transgenic line influenced by <br />                                             position effects. C, expression dominant negative transgene control pmyo-3 (a <br />                                             muscle-specific promoter) resulted aldicarb hypersensitivity similar observed the <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             dominant negative native shl-1 promoter. D, heterozygous shl-1(ok1168)/+ <br />                                             animals intermediate phenotype fell N2 shl-1(ok1168) <br />                                             homozygotes. The positive control hypersensitivity aldicarb assays strain <br />                                             unc-64(OxIs34), gain-of-function integrated transgenic syntaxin line. <br />  <br />  <br />  <br />  <br />                                                  J Biol Chem. Author manuscript; available PMC 2008 March 3. <br />                                Fawcett et al.                                                                                                     Page 23 <br />  <br />                                                                                      TABLE 1 <br />                               Cell identification SHL-1 expression pattern <br />  <br />                            Non-neuronal cells <br />                            &#226;&#8364;&#402;Muscle (body wall) <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            &#226;&#8364;&#402;Muscle (vulval) <br />                            &#226;&#8364;&#402;Intestine <br />                            &#226;&#8364;&#402;Diagonal muscles (male) <br />  <br />                            Neurons <br />                            &#226;&#8364;&#402;Sensory neurons                                      AWC, ASE, ASJ, ADL, ASK, ADF, AFD strongly, ASI, URX, AWB weaker more <br />                                                                                  var <br /> </body></html>