<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL OF BACTERIOLOGY, Jan. 1989, p. 230-237                                                                       Vol. 171, No. 1 <br /> 0021-9193/89/010230-08$02.00/0 <br />  <br />  <br />  <br />  The REVJ Gene Saccharomyces cerevisiae: Isolation, Sequence, <br />                     Functional Analysis <br />                         FRANK W. LARIMER,l* JACK R. PERRY,2t AND ALICE A. HARDIGREE&apos; <br />       Biology Division, Oak Ridge National Laboratory,1 The University Tennessee-Oak Ridge Graduate School of <br />                                      Biomedical Sciences,2 Oak Ridge, Tennessee 37831 <br />                                          Received 29 March 1988/Accepted 27 September 1988 <br />  <br />              The REV) gene Saccharomyces cerevisiae required nora         induction mutatio    pysical ad <br />            chemical agents. We determined sequence      3,485-<span id='am-241' about='xsp:base' typeof='owl:Thing'>base</span>-pair segient DNA              the <br />            rev)-) mutant. Gene disruption used confirm DNA contained REV) gene. The sequenced <br />            segment contAins singe ong open reading frame, encode polypeptide 986 amino acid residues <br />            The REVI transcript 3.1 kilose pirs <span id='am-5' about='xsp:length' typeof='owl:Thing'>length</span>. Frameshift mutations introduced open readin <br />            frame yielded RevC phepotype. A base substitution, encoding Gly-193 Arg-193, fond open <br />            reading frame rev)-I. Deltion mutants, lacking segments 5&apos; region REVI, had. Intermediate <br />            mutability relative REV) rev)-); complete deletion exhibited lower mutability rev)-) . REV) not <br />            essential gene. An in-frame fusion 5&apos; eMd REV) open reading frame lacZ gene produced <br />             3-galactosidase activity constitutively. The predied REV1 protein hydrophilic, pedkted pl 9.82. <br />            No homologies RAD1, RAD2, RAD3, RAD7, RAD10 proteins noted. A 152-residue internal segment <br />            displayed 25% identity UMUC protein. <br />  <br />   ,The Saccharomyces cerevisiae REV] gene product func-                   In paper isolation REV) gene <br /> tions cellular process required mutagenesis caused             mutant allele rev)-) report nucleotide and <br /> by UV radiation chemical mutagens (14, 20, 25).                deduced amino acid sequences. Deletion analysis used <br /> The REV) gene small group yeast genes,                   determine functional limits REVI coding <br /> including REV3, REV7, CDC7, NGM2, mutant                      sequence 5&apos;- 3&apos;-flanking sequences. A prelimi- <br /> alleles severe general effect mutability (21-24,          nary characterization response REVl::lacZ fusions <br /> 34, 35). Mutants REVI exhibit little mutability         heat shock DNA-damaging agents presented. <br /> by UV light chemicals slightly                  Similarity analysis suggests segment REVI <br /> sensitive lethal effects agents (23, 24).               protein homologous UMUC protein. <br />    The REV) gene product thought constitute a <br /> DNA repair process &quot;error prone&quot; &quot;&apos;mutagenic&quot;                             MATERIALS AND METHODS <br /> its function (14, 20, 25). Just SOS processing umuDC                 Media. LB medium (29) (with antibiotic supplements as <br /> is distinguished Rec functions uvrABC-              appropriate) used propagate E. coli strains and <br /> controlled excision repair Escherichia coli (56), mutagenic          plasmid-bearing derivatives. YEPD synthetic complete <br /> repair distinct major repair pathways              (SC) yeast media described (24). SC-URA <br /> yeasts (14, 20, 25). Excision repair controlled genes          SC uracil, SC-LYS SC lysine, and <br /> the RAD3 epistasis group, recombinational strand break              SC-LEU SC leucine. <br /> repair controlled genes RADS2 epistasis group. <br /> The REV) gene RAD6 epistasis group;                     Yeast strains. S. cerevisiae S288C gal2 mel mal SUC2 <br /> RAD6 gene product recently.been identified ubiqui-             CUP) used source wild-type DNA library <br /> tin transferase required process histones             construction. Strains XL68-3D rev)-) lys)-) leu2-3 his4-38 <br /> chromatin initiate DNA repair (18).                                  trpl-289 ura3-52, XL69-5A REVI lys)-) leu2-3 trpl-289 <br />    As initial step understanding regulation           his3M, XL7O-1B RPV) lysl-l leu2-3 trp)-289 ura3-52 <br />                                                                         constructed laboratory, using standard tech- <br /> function mutagenic repair, sought clone char-              niques mating, sporulation, tetrad analysis (52). <br /> acterize REV) gene. We hope use clone                     Bacterial strains. E. coli BNN45 (F metB thi hsdR lacY <br /> overproduce REV) gene product facilitate purifica-               supE44 supF), derived ED8654 (31), used routinely <br /> tion, characterization, vitro reconstruction              cloning host. E. coli DH1 recAl (F- Alac-pro, endA) <br /> mutagetiic repair complex. In regard, cloning           gyrA96 thi-) hsdR) 7 supE44 relAI) (13) used cosmid <br /> REV3 gene reported (J. F. Lemontt, S. V. Lair,                 host. E. coli JM107 (Alac-pro endAl gyrA96 thi-) hsdR)7 <br /> A. K. Beck, E. G. Bermsteine, abstr., p. 33, 11th Int.              supE44 relA) (F&apos; traD36 proAB+ lacIZAM)5]) (57) was <br /> Conf. Yeast Genet. Mol. Biol., Montpellier, France, 1982),            used host M13 vectors. E. coli GM215 (F- dam-3 <br /> fragment complements rev2 identified (53),               endAl rna-) thi-) supE44) (30) Was used produce Dam- <br /> and CDC7, poteritiates mutability present                 plasmid DNA BclI subcloning. <br /> high copy number (51), shown encode protein                  Plasmids, enzymes, biochemicals. Plasmids YCp5O, <br /> homologous protein kinases (38).                                     YIp5, YEp13, pHC79 described previously <br />                                                                         (6, 15, 19, 50). Restriction endonucleases, T4 DNA ligase, <br />   * <br />     Corresponding author.                                               Klenow fragment DNA polymerase I purchased <br />   t Present address: Department Microbiology, University          New England BioLabs Bethesda Research Labora- <br /> Tennessee, Knoxville, TN 37919.                                         tories used according specifications manu- <br />                                                                   230 <br />  VOL. 171, 1989                                                                   REV] GENE OF S. CEREVISIAE               231 <br />  <br /> facturer. Plasmid cosmid DNAs prepared E.         start points transcript performed S1 <br /> coli alkaline lysis technique (29). High-molecular-      nuclease protection assay described previously (12). <br /> weight yeast DNA prepared described previously              DNA sequence analysis. The dideoxy chain termination <br /> (36). Cosmid plasmid DNA recovered yeast           method Sanger et al. (49) used determine the <br /> cells alkaline lysis (10). Oligonucleotides used adapt-   nucleotide sequence strands REV] gene. <br /> ers, linkers, sequencing primers synthesized       Computer-assisted analyses nucleic acid sequence data <br /> Systec 1410 synthesizer, using phosphoramidite chemistry.       prediction protein hydropathy secondary struc- <br /> Restriction fragments DNA sequencing cloned       ture performed using set programs developed <br /> M13-mpl8 M13-mpl9. Radioisotopes sequencing          Stephens (54), IBI-Pustell Sequence Analysis Pro- <br /> labeling obtained Dupont, NEN Research Prod-          grams (International Biotechnologies, Inc.), Na- <br /> ucts. Nick translation kits purchased Amersham        tional Biomedical Research Foundation Protein Identifica- <br /> Corp. o-Nitrophenyl-o-D-galactopyranoside 5-bromo-4-        tion Resource Data Base software. <br /> chloro-3-indolyl-p-D-galactopyranoside (X-Gal) ob-            ,I-Galactosidase assays. Plate assays replica <br /> tained Sigma Chemical Co.                                  plating X-Gal plates. For quantitative 3-galactosidase <br />    Cosmid library. The shuttle vector YCp5O (URA3 ARSJ          assays (48), cells grown optical density 600 nm <br /> CENIV Ampr Tetr) modified inserting 1.65-kilobase      approximately 1.0 YEPD medium. For induction ex- <br /> pair (kbp) BglII fragment bearing lambda cos site (<span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span>     periments, cultures split desired treatment was <br /> pHC79) BamHI site tet gene (pFL17). A new       applied sample. At various time points, 5 ml sample <br /> BamHI site introduced inserting BamHI oligonu-         removed assayed B-galactosidase activity. A <br /> cleotide linker unique PvuII site cos segment     1-ml portion used measure optical density 600 nm, <br /> (pFL18). Total yeast DNA S288C (&gt;100 kbp)              remaining cells pelleted centrifugation and <br /> partially digested Sau3A separated neutral          suspended 1 ml Z buffer (48). The assay o-nitro- <br /> sucrose density gradient centrifugation. DNA fragments       phenyl-p-D-galactopyranoside-hydrolyzing activity been <br />                                                                 described (48). One <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> = OD420 x 1,000/[t(h) x <br /> 35 40 kbp length ligated BamHI-digested           V(ml) x OD6.], V volume sample <br /> cosmid pFL18 packaged phage particles (29). Ap-        assayed OD optical density. <br /> proximately 12,000 clones derived packaging, <br /> or 32 genome copies. An amplified DNA library made <br /> from 2,500 pooled clones DH1.                                                         RESULTS <br />    Transformation. Bacterial transformations per- <br /> formed method Hanahan (13), yeast trans-           Isolation REV) complementation. Cosmid library <br /> formations lithium acetate method Ito et al.     DNA used transform XL68-3D uracil indepen- <br /> (17).                                                           dence. Some 440 Ura+ transformants screened for <br />    Rev activity screen. Rev activity cosmid-transformed      complementation revl-I. Two Rev+ transformants were <br /> isolates monitored using replica plate papillation     identified. Cosmids recovered transformant, <br /> assay. Master plates prepared transferring         preliminary restriction maps obtained (Fig. 1). The <br /> isolate SC-URA plate. Twenty independent isolates         cosmid inserts shared 15.5-kbp overlapping segment. <br /> per plate applied &quot;patches&quot; (1 1 cm) form       Subcloning HindlIl BglII fragments showed the <br /> master plate. Master plates incubated 30&#194;&#176;C 24 h.    Rev+ segment contained 8.0-kbp BglII frag- <br /> Two replicas master plate SC-LYS.          ment (pFL41; Fig. 1), HindIII subclones <br /> One SC-LYS replicas exposed 40 J 254-nm        Rev-. Both cosmid BgIII clones fully <br /> UV light m2. The replicas incubated 30&#194;&#176;C 3   complemented revl-l (Fig. 2). Deletions PvuII <br /> 5 days, patches scored Lys&apos; papillae. Rev-     ClaI fragments pFL41 Rev- (not shown), extend- <br /> patches normally exhibit 0 2 papillae, Rev&apos; patches    ing limits Rev+ segment 1,100 bp. <br /> have 10 30 papillae. Quantitative mutagenesis assays         Disruption REV) gene. The subcloning results indi- <br /> Lys&apos; reversion performed described Lemontt           cated 736-bp HindIII-ClaI fragment wholly con- <br /> (24).                                                           tained complementing region. Therefore, this <br />    Gene disruptions integrations. Integrating plasmids      fragment internal gene. <span id='am-6' about='protege:TO' typeof='owl:Thing'>To</span> test predic- <br /> were targeted REV] restriction endonuclease cleavage      tion, HindIII-ClaI fragment cloned integrat- <br /> within REV] segment (37). One-step gene disruption      ing plasmid YIp5 (50). Integration plasmid containing an <br /> performed method Rothstein (47). The structures    internal fragment gene homologous chromo- <br /> all integrations disruptions confirmed Southern     somal copy results disruption chromosomal <br /> transfer hybridization analysis (29).                           sequence, yielding gene interrupted vector <br />    REV) transcript analysis. Total RNA extracted       sequences. Rev+ strain XL70-1B transformed with <br /> XL70-1B transformed multicopy YEp13-REVJ plas-         plasmid pFL44 (Fig. 1), linearized partial <br /> mid pFL190. Cells grown SC-LEU titer 107,       digestion restriction endonuclease EcoRI, creating <br /> and RNA extracted vortexing phenol glass        recombinogenic free ends direct integration the <br /> beads, followed extraction phenol-chloroform-           region homology. The transformants obtained displayed <br /> isoamyl alcohol precipitation LiCl (52). Dot hybrid-   Rev- <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span>phenotype</span> (Table 1). A disruption strain was <br /> izations applied Schleicher &amp; Schuell BA 85 nitro-      crossed rev]-l mutant, resulting diploid was <br /> cellulose performed according specifications.      shown Rev-. After sporulating diploid, spores <br /> Glyoxylated total RNA subjected agarose gel elec-        Rev- (11 tetrads). The disruption crossed a <br /> trophoresis, transferred GeneScreen (Dupont, NEN Re-         Rev+ strain, resulting diploid sporulated <br /> search Products), hybridized using protocol   tetrads dissected, disruption (Rev- Ura+) <br /> manufacturer. RNA ladder Bethesda Research Labo-           segregated 2:2 (nine tetrads). These results indicate the <br /> ratories used provide size standards. Mapping     HindIII-ClaI fragment internal REVI gene. <br />  232     LARIMER ET AL.                                                                                                                                      J. BACTERIOL. <br />  <br />                                                                      10 kbp <br />                   A <br />                       Plasm id                                                                              co. <br />                                                                                                                                            Vector <br />                       pFL21 B/SXHGXHG GH         H             G X        HGG GXHB/S X G H B/S                                          YCp5O cos <br />                                            Cos <br />                       pFL22            B/S IH  GX BG1 XI <br />                                             I I* I---      -              HI GGt GXHHH <br />                                                                                    I~ H                 GG                H GXG B/S     YCp5O cos <br />  <br />                   B <br />                                                                                  RE&apos;Vf                      ~I <br />  <br />  <br />  <br />  <br />                                      8/4         Ss   X              LCP        CB EH VE              EVP ..B/G <br />                                                                      No         No                  ....I <br />                       pFL41            I                                           I                   I <br />                                                                                                                                         YCp5O <br />                       pFL 1 57                                                                                                          YIp5 <br />                       pFLl46                                                     I                                                      YIp5 <br />                       pFLl48                                                                                                            YIp5 <br />                       pFL44                                                                                                             YIp5 <br />                       pFL240                     I------         /S3                                                                    YCp5O <br />                       pFL180                                                                                                            YCp5O <br />                       pFLl62                 i8I                                                                                        YCp5O <br />                                                +8 <br />                       pFL 137          1       IV                                                                 I                     YCp5O <br />                       pFLl52                                         I    kbp <br />                                                                                                 /OcZ                  I                 YCpSO <br />                                                                      I <br />  <br />  <br />   FIG. 1. Plasmid structures. Restriction sites indicated letters. B, BamHI; B/G, BamHI-Bglll junction; B/S, BamHI-Sau3A junction; <br /> C, ClaI; E, EcoRI; G, BglII; H, HindIII; L, BclI, P, PstI; Ss, BssHII; V, PvuII; X, XhoI; cos, lambda cohesive end site. (A) Cosmids <br /> complementing rev)-l. Heavy line indicates cosmid CEN vector pFL18 sequences; light line indicates yeast DNA insert. (B) Subclones of <br /> REV) fragments derivative plasmid constructions. The lacZ fragment pFL152 pMC1871 (7). Not shown: pFL190, the <br /> 8-kbp BglII REV) fragment vector YEp13. <br />  <br />  <br />    A deletion REV] constructed one-step gene                                  BAL31 deletions used localize 3&apos; end REV). <br /> disruption (47). A segment REV) fragment                                  Plasmid pFL41 linearized Sall site tet gene, <br /> deleted replaced segment bearing HIS3 gene                              timed BAL31 digestions carried degrade the <br /> (pFL240; Fig. 1). The Rev insert (plus HIS3) excised                             3&apos; end REV) subclone. Both Rev- Rev&apos; deletions <br /> from pFL240 used transform haploid XL69-4A.                               (pFL162 pFL180, Fig. 1, Table 1) whose <br /> The resulting His&apos; strains Rev- (Table 1),                                 endpoints fell 640-bp EcoRI fragment. <br /> consequence replacement chromosomal sequences                                 Nucleotide sequence REV) gene. The dideoxynucleo- <br /> with rev)A::HIS3 segment. Because deletion                                tide chain termination method (49) used determine the <br /> viable haploid, REV) essential gene.                                  nucleotide sequence REVI DNA fragments cloned both <br />    Direction transcription size transcript. The ClaI- <br /> EcoRI REV) internal fragment pFL44was cloned                               TABLE 1. Mutability Rev constructions reference alleles <br /> M13 phage mpl9. Both orientations fragment with <br /> respect phage obtained. Radioactively labeled                                                                      Lys&apos;    revertants 108 survivors (% <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span>survival</span>) <br /> single-stranded DNA probe (16) phage used                                        Allele [free plasmid]                           40J                80 J <br /> in separate dot hybridizations total RNA. Hybridiza-                                                                    No UV          UV              UV <br /> tion obtained phage priming Rev                                                                                         m2            M2 <br /> fragment 5&apos;-EcoRI-ClaI-3&apos;, indicating direction                          Haploids <br /> transcription REV) ClaI EcoRI. Northern                              REV)a                           8.3                 6,014 (42)        17,321 (8) <br /> transfer hybridization analysis total RNA, using REV)                           rev): PFL44a                   13                     161 (17)           153 (3) <br /> probe, revealed 3.1-kilobase transcript (Fig. 3A).                                         revlA::HIS3b              4.7                    11 (17)            21 (6) <br />    Functional limits REV] gene. To map 5&apos; end                                  rev]-Jc                  16                      83 (14)           234 (2) <br /> the minimum complementing segment REV), deletion                                          REVMA3355C [pFL180]       8.9                 3,529 (59)         8,547 (14) <br /> subclones (pFL157, pFL146, pFL148) constructed                                      rev1A2933C [pFL162]       4.0                    83 (12)           123 (6) <br /> in integrating vector YIp5 (Fig. 1). Integration                                     rev:: 1104(+8)c [pFL137] 5.0                    208 (14)           214 (2) <br /> targeted REV) linearizing plasmids HpaI                            Diploids <br /> site REV) segment. These integrations produce                             REVI/REVi                                 6.5         790 (63)         2,398 (28) <br /> sequential series interruptions 5&apos; end REV). The                                rev)::pFL44/revl:: pFL44 3.3                     24 (22)            38 (12) <br /> mutability phenotypes interruptions shown                                    revl-llrevl-)            2.0                      7.5 (26)           8.7 (11) <br /> Fig. 4. The interruption terminating BcIl site (pFL157)                               revl::pFL44/revl-1                  4.1          42 (22)            14 (11) <br /> shows Rev&apos; mutability, ClaI (pFL146)                                XL70-1B. <br /> PstI (pFL148) interruptions intermediate mutability                               b XL69-5A. <br /> with respect REV) rev)-l.                                                         c XL68-3D. <br />                                                                                                                   ;&apos;rde^o.4!:09-6S7*f.FbMM <br /> VOL. 171, 1989                                                                          REVI GENE OF S. CEREVISIAE                                                             233 <br />  <br />                                                                        Isolation sequence rev]-l. Targeted integration of <br />                                                                      plasmid pFL157 Rev- strain XL68-3D yielded Rev&apos; <br />                                                                      transformants (not shown), mapping rev1-1 <span id='am-163' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-164' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span>mutation</span> <br />                                                                      Bcll site HpaI site. To recover revl-l, <br />           &lt;                \                                         plasmid pFL41 digested PvuIl BssHII, the <br />                                                                      vector fragment sequences flanking REV) used to <br />           &gt;       10                0                                transform XL68-3D. The plasmid reconstituted vivo <br />           cr_                                                        gapped-plasmid repair, transfers missing chro- <br />                                                                      mosomal sequence (rev1-)) plasmid gene conver- <br />                                                                      sion (37). The repaired rev1-] plasmid (pFL73; shown) <br />                                                                      recovered yeast cells rev1-) insert was <br />                  . 1           .            .       .A               sequenced. The rev)-) allele consists G-to-A substitu- <br />                                                                      tion nt 937, encoding Gly-193 Arg-193 amino acid <br />           &gt; 10 o4                                                    substitution (Fig. 6). <br />                 0~~~~~~~ <br />                                                                        5&apos;- 3&apos;-flanking sequences REVI gene. The 5&apos;- <br />                                                                      flanking region REVI 67% AT rich poly(dA-dT) <br />                                                                      sequences. There striking dA-dT regions: 10 T&apos;s <br />           JD <br />           U) <br />                                0                                     nt 74 83; 18 T&apos;s, 3 A&apos;s, 1 C nt 41 62; 25 <br />                                                                      A&apos;s, 5 T&apos;s, 6 G&apos;s, 3 C&apos;s nt 100 138; 15 T&apos;s, <br />                 0~~~~~                                               14 A&apos;s, 3 C&apos;s, 1 G nt 254 286. The T-rich <br />                                                                      regions resemble elements required constitutive tran- <br />           z <br />                                                                      scription HIS3, PET56, DEDI genes (55). The <br />                                                                      sequence TATATA occurs nt 185 190 upstream the <br />                                                                      coding sequence. This sequence resembles canonical <br />                                                                      TATA(T/A)A(T/A) usually 26 34 nt upstream <br />           &gt;10 <br />           w                                                          point initiation transcription higher eucaryotes <br />                 1 -IB-                                                       A                      B <br />                                                                                     1 2                     l     2        A G C T <br />                                                                                                                                            ..     ..         ..        .. <br />  <br />                                                                                                                                        .up$ .eb, #-: <br />  <br />  <br />                        0    20   40 60      80                                                                                                   , <br />                            UV FLUENCE, J/m2                                                                              s:;:i.&apos;:Ss <br />                                                                                                                                                         ,;,S,&gt;_ &apos;S: <br />  <br />                                                                                                                                                         # sih <br />                                                                                                                                            :. :t l&apos; .                         ...   :.: <br />  <br />  <br />   FIG. 2. Survival mutability Rev strains plasmid- <br /> transformed strains following UV irradiation. (A) UV survival; (B) <br /> UV mutability. Symbols: 0, XL70-1A REVI; 0, XL68-3D rev)-);                                                                                                           &gt;N <br />  <br />  <br />  <br />  <br /> O, XL68-3D rev)-) (pFL22 REVI); A, XL68-3D rev]-] (pFL41                                                         _- <br />                                                                                                                  _ <br />                                                                                                                                    * <br />  <br />                                                                                                                                        C.         &apos;t <br />                                                                                                                                                         j; <br />  <br />  <br />  <br /> REV)).                                                                                        279- <br />                                                                       7.46-                                     w                                       ,gS#j: <br />                                                                                                                          bh <br />  <br />  <br /> orientations M13 phage vectors. The sequencing strategy            4.40                                                                 ..    ,.b)                               ..... <br />  <br />  <br />  <br />  <br /> is shown Fig. 5. The sequence DNA strand                                                                  Se:                            . . . . ;s- <br />                                                                                                                                                                   ....... <br />                                                                                                                                                                             .Si.&gt;.. i: H <br />  <br />  <br /> independently determined. Figure 6 shows sequence              2.37- <br /> 3,485 bp corresponding mRNA regions flanking                                       255                 -                        AW     :&apos;4                       ._},:.:4:t <br />  <br />  <br /> the gene. The overall base composition REV) frag-               1.35 -                                                                       :: S <br />                                                                                                                                                          *, Sm4 :5 <br />                                                                                                                                                               St <br />                                                                                                                                                                                          , <br />  <br />                                                                                                                                                                                           .: <br /> ment slightly AT rich, 64% A+T, respect                                                                               ,_ &apos;                              ....: &apos;&apos;. <br />  <br /> bulk yeast genomic DNA. The REV) coding sequence                                                                                        *:           4g:&apos;* <br />  <br />  <br />  <br />  <br /> somewhat purine rich 55% A+G strand correspond- <br /> ing mRNA.                                                                                                              o <br />  <br />                                                                                                                           ..: :: <br />                                                                                                                                       &apos;&apos; <br />  <br />                                                                                                                                                          -* <br />                                                                                                                                                                              .2 7A&quot; <br />  <br />                                                                                                                                                                                     .: <br />  <br />    The 5&apos; end REV) transcript mapped Si <br /> nuclease protection. Two transcription start points                                                                 4oW 02                          _|&#194;&#163;- <br /> observed (Fig. 3B), nucleotide (nt) 318 nt                                                                                    W -J <br />  <br />  <br />  <br /> 342. Each start point short T run. The down-                                                                                      &gt;rb ; <br />  <br />  <br /> stream ATG (nt 361 363) defines open reading frame                                                                       &amp;                ,lenjb <br />  <br />  <br />  <br />  <br /> 2,955 bp length, encode protein 985 amino            FIG. 3. Northern (RNA) hybridization analysis (A) S1 nu- <br /> acids predicted molecular weight 112,239. A lacZ           clease mapping 5&apos; termini (B) REV) transcript. (A) <br /> translational fusion (pFL152; Fig. 1) BamHI site         Lane 1, 10 ,ug total yeast RNA; lane 2, 20 p.g RNA. Indicated <br /> nt 1518 reading frame produces ,-galactosidase activ-        margin locations sizes (kilobases) marker <br /> ity, demonstrating open reading frame transcribed       RNAs. Complementary EcoRI-Clal probe labeled 32P by <br /> and translated. Furthermore, dGGTCGACC oligonucleo-                primer extension appropriate M13 clone. (B) Lane 1, 5 ng of <br /> tide linker inserted HpaI site nt 1104 (pFL137;          REVI probe alone; lane 2, 5 ng REV) probe plus 200 tLg total <br /> Fig. 1) yielded Rev- frameshift mutant (Table 1). All        yeast RNA. Indicated margin sizes (nt) the <br />                                                                      protected fragments. Complementary BclI-Pstl probe labeled <br /> reading frames extensively blocked termination co-            32P PstI end, fragment hybridized or <br /> dons. In REV) gene 61 possible codons used,              RNA 80% formamide 45&#194;&#176;C 12 h. The hybrids were <br /> unlike codon bias characteristic highly expressed             digested S1 nuclease (500 U/ml) 30 min 37&#194;&#176;C. The <br /> yeast genes (3).                                                     sequencing ladder shown M13mpl8, using -52 primer. <br />  234     LARIMER ET AL.                                                                                                       J. BACTERIUOL. <br />  <br />                                                                         TABLE 2. P-Galactosidase activity REV) ::lacZ fusion in <br />                                                                                              S. cerevisiae <br />              _- <br />                                                                             Treatment               ~Assay time       P-Gaiactosidase <br />              -J <br />                                                                               Treatment                    Assayi(me)   activity (U) <br />  <br />                                                                       None (220C)                               0                  2.7 <br />                                                                                                                60                  2.7 <br />              (0                                                                                               120                  2.4 <br />                                                                       Heat shock (37&#194;&#176;C)                        60                  1.3 <br />                                                                                                               120                  1.3 <br />                                                                       UV, 80 J/M2 (220C)                       60                  2.2 <br />                                                                                                               120                  2.4 <br />              0     &gt;~~~~~~                                            4-Nitroquinoline-N-oxide, <br />                                                                         0.5 jig/ml (22&#194;&#176;C) <br />                                                                                                                60 <br />                                                                                                               120 <br />                                                                                                                                    2.5 <br />                                                                                                                                    2.3 <br />  <br />  <br />              C,,                                                                              DISCUSSION <br />              z     1o-0-                                                Cosmid cloning. Since Rev phenotype did easily <br />                                                                       lend itself positive selection scheme, chose to <br />              w                                                        construct cosmid library minimize number trans- <br />              w                                                        formed clones individually screened. This choice was <br />                                                                       quickly justified obtaining independent clones of <br />              +           7                                            REV), slightly genome equivalent of <br />                    o-J                                                clones. Our library derives advantages the <br />                                                                       choice CEN vector: large plasmids stabilized, and <br />                                                                       genes lethal high copy number main- <br />                                                                       tained single copy, REV) gene did prove <br />                  io 8B                                                deleterious present high copy number. <br />                       0    20 40 60             80                      REV1 protein. The translation REV) open reading <br />                           UV FLUENCE, J/m2                            frame indicates 112,239-dalton protein containing 37.2% <br />    FIG. 4. Survival mutability transformed strains having 5&apos;   nonpolar, 35.0% polar, 11.8% acidic, 16.0%o basic amino <br /> interruptions REV) region. (A) UV survival; (B) UV muta-       acids. The REV1 protein substantially basic, a <br /> bility. Symbols: 0, XL7O-1B REV); 0, XL68-3D rev)-); A, XL70-         predicted pl 9.82. The translation REV) open <br /> 1B REV)::pFL157; 0, XL70-1B REVl::pFL146; 0, XL70-1B                  reading frame does exhibit significant sequence <br /> REVI::pFL148.                                                         similarity S. cerevisiae RADM (45), RAD2 (28), <br />                                                                       RAD3 (43), RAD7 (39), RAD10 (44) proteins. A protein <br />                                                                       similarity search (26) National Biomedical Research <br /> (5, 11). The base 3 nt upstream ATG G           Foundation Protein Identification Resource Gen- <br /> REV), A usually eucaryotic mRNAs             Bank data banks did reveal strong similarities. The <br /> (8).                                                                  highest score returned indicated weak similarity (z score of <br />   The sequence TAAATAAA, yeast                 4.65) alignment E. coli UMUC protein (40). <br /> genes 28 33 nt upstream 3&apos; end mRNA (2), <br /> appears nt 3436 3443. This sequence includes <br /> the possible polyadenylation signal AATAAA (42). The                     L IC IP A     H          CB S         E D          VTSE               E <br /> Rev&apos; 3&apos; deletion Del3355 (pFL180) removes sequence,                 I M <br />                                                                             ATG <br />                                                                                 II   II)-I <br />                                                                                      .II <br />                                                                                    ATG <br />                                                                                        I II <br />                                                                                                                                          TGA <br /> however, indicating sequence essential for <br /> Rev function. An inverted repeat, AAAAAaTAAgGaCtT <br /> TAaTTTTT, occurs downstream nt 3453 3475. <br /> This structure function transcription termination                  -X     -     -~     -Z       -*        -.     -4- <br /> signal.                                                                       4-4 <br />                                                                            4-.;~~ 4.-.~                4                    46-    a-     4 <br />    Response REV) heat shock DNA-damaging agents. <br />                                                                                                                         - <br />  <br />  <br />  <br /> The Rev&apos; strain XL70-1B carrying REVJ::lacZ fusion <br /> CEN plasmid pFL152 showed J-galactosidase activity on <br /> X-gal plates. Low basal activity observed direct                                                     I kbp <br /> assay (Table 2). As DNA repair genes yeasts are <br /> known inducible DNA-damaging agents UV                 FIG. 5. DNA ,sequencing strategy. Plain arrows denote sub- <br /> (1, 9, 27, 28, 41, 46, 48) 4-nitroquinolone-N-oxide (27, 48),      cloned segments primed M13 sequencing primer; arrows with <br /> we monitored p-galactosidase activity following treatment             open circles denote segments primed internally oligonucleo- <br /> with agents. In addition, response heat shock            tides synthesized using deduced sequence. The positions the <br />                                                                       translational initiator (ATG) terminator (TGA) codon indi- <br /> was examined. The REV)::lacZ fusion did respond                cated sequenced fragment; restriction endonuclease <br /> such treatments (Table 2). Similar results obtained          cleavage sites used generate subclones shown above. A, AccI; <br /> treatments 30&#194;&#176;C, integrated REV): :lacZ         B, BamHI; C, ClaI; D, HindIlI, E, EcoRI, H, HpaI; I, HincII; L, <br /> fusion (data shown).                                              BclI; P, PstI; S, ScaI; T, SstI; V, PvuII. <br />  VOL.    171, 1989                                                                                    REV] GENE OF S. CEREVISIAE                            235 <br />  <br />  <br />  <br />               *        *         *         *        *         *         *        *         *        *         *         *        *         *    BBclls <br />    I OCAATACICATCACATTATGACTACGAMCTGCTIGATGTCCTTTTTTTATTATATTTTTCTTCAGTGAGMACaTsmmiiTACACAGACCAAGACaGGAAAAGTAGCTAAGAAGAAACAAATCATGAAAAMTGTGAAGTGATICA <br />               *        *         *        *         *        *        *        *        *         *        *        *         *        *         t <br />  151 TIGcAcATcoATcACTTMAcATCAcTTSAGaATATATAGAOTaGGTTTACGGCAc CCTTTAACACCAATACCTTTTGOCATAGTCTAAAGAmTGGTTCTTAATTTTAAAAATTTAACTAAAGATTTCCCTACAAAGAAG <br />             HincIi      *               ClaI <br />                                          C         *     *       *       *         *       *         *         *         *       *         *        * <br />  301   TAACGCMACAAmTCTCTT^C7CMMA&amp;          AAATTTC                          GTCTTGTAGA TTTATTGGACAWGATTTGGATACTCTATAAATA&amp; GAAACACCT;ATAAAACAATT&amp;CCTT <br />                                               TAGGCATATCCAGCGATWGTG AACATGGT&amp; <br />                                                                    M G B H d G L V D L L D S D L E Y S I N R 9                     P D K N N C L <br />             *        *        *        t   *         *        *         *            *         * HincII*          *        *        *     PstI* <br />                                                  CTTGACAGCTTirCTATCTACGTTAAGCGATGATTCCTTAArTATATGTCAACCAA7CTCCCAAACTAATAAGAATAATTCTAATCCAACrCA <br /> 451 AWCCAGCAAAGtTCJATGATTCACATTTACAOCAAAGAaCCOTCMOO <br />      S Q Q S V N D S H L T A K T G G L N A R S F L S T L S D D S L I E Y V N Q L S Q T N K N N S N P T A <br />               *        *         *       *        *        *        *         *        *        *        *        t         *        *     AccI <br />   601~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~kc <br />  601 OGTtMACTTAATACTACTAAAATATT^ACTG TGTGAAGTTACAt&amp;CTGATCTTGGTGCGAAGATTCACCTATAGCTCGTAGCGTCATCGAGATCCAGAAAGTATAGCATGGCTGATOATGATTAAAAATACTGTG= <br />       G T L R F T T K[ N I S C D E L H A D L G G G L D S P I A R S V I B I Q E S D S N G D D V K K N T V Y <br />               *        *         *         *       *         *        *         *        *         *        *         *        *       *         $ <br />  751 ACTAzGAassTTATCACOAMGAoAGCACOGACAAACCCTWAAGATCAAATATTAAAGATCAATATAAAGATCAATTTCTAGTCAAACA(ITAAATATT&apos;CAAAATTOCG7CATCTATATMMAAA TACACCAAACCTOGA <br />       T R 8 A Y F H 9 11 A H G Q T L Q D Q I L K D Q Y K D Q I S S Q S S K I F K N C V I Y I N G Y T K P a <br />                                     revl-l <br />               *,       *         *     A *         *         *        *         *        *         *        *         *        *       *        * <br />  901   AGACTsGCATTACAcriGAtGAATAGTTTTSACAMMAATTTTTACACTATTTd7TCTTCAAAGAAACGCTTACTCACATAGTGGCTTCCAATTACCATTAAGAAAGATTGAATTTOCAAATTACAAAGTAGTCAGTCCOGAT <br />        R L Q L H L M I          V   L H G K F L H Y L S S K K T V T H                  I   V A S N L P L K K R       I   E       F A N Y K V     V   S   P D <br />                *            *       *          *         *   HNpaI *       *        *          *        *        *             *                   *             * <br /> 1051 T0AuTAGT70ATAaCGTCAAAAGCAGATTATTGCMMGCAATTACTCGTTAACATCTAAACTTGATGAACAGCAAAAAAACTAGATATMTAAAACCGTAAATTCAATTCCAT&apos;r CCTCAGAGACTA&amp;;CCTGCACMAAAG <br />       N I V D S V K F A R L L P W Q N Y S L T S K L D 9 Q Q K K L DN C K T V N S I P L P S B T S L H K G <br />              *         *        *        *.        *        *        *         *        *       *         t        *        *         *        * <br /> 1201 TCTAAAMTA=aTCaaCAnTaTGT:CACTAGMAACAGTCACCAGTAAACTCMCATAATTTWAGGCGMAAAATAGTCGCTTGTGATACTGAT ATTTCCTTACTTCCTATT   TCATTGAGATTACATCATCTCTCCGCA <br />       S K[ C V G S A L L P V 9 Q Q S P V N L N N L B A K R I V A C D D P D F L T S Y F A H S R L H H L S A <br />              *      *      *     *          *        *        *        *        * ClaI   *        *       t         *                   * <br /> 1351 7MAGATCT      AATaMMACATCCAC AAGTACAGC TCCAAAATTACOAAGATACCTACATTATCTTTC ATATCGAI AGTTTTTTsCaCAAoC <br />            GAATAAATTCTG                                                                             CTGTl  OCGTATCTATGTAGMAATTCTAGTTTCCA <br />       12 K A N L K D K[ F L N F N I H K Y T K I T D K D T Y I I F H I D F D C F F A T V A Y L C R S S S F S <br />                *        B_lJ      *       *        *        *        *        *        t        *    *       *        *       *        * <br /> 1501   CAG TTAAOTCTAT^aAoCTa AGTTM ACTCCA7TCAACCGATATSCCAACT^AGTOAATAGACAGTCATATuaATTAAAMToGMTaTaaG TGTCTC AAGCT&amp;AAAAATGTTOCCAATWAxTCAAA <br />        A C D F K[ R D P I V V C H G T K[ N S D I A S C N Y V A                   R S Y G I K N G M W V S Q A E K M L P N G I K <br />               *          *         *         *        *       ScaI      *        *        *         *        *        *        *         * <br /> 1661 CTAATATC;ATTAiCCAATATCCTTTAaCAATTTCAATTAAAAT TOaAACTCTCMAchlgTAAAAarmCATATTCMTAATTTGATTTTATCTATA7TAT&amp;CTGATGGTTaTTCG7TAGATAA7CCCTGATAATATT,CAT <br />       L I S L P Y T f B Q F Q L K S E A F Y S T L K R L N I F N L I L P I S I D 9 A V C V R I I P D N I H <br />                *            *       *           *        *         *       *        *          *        *        *             *           *       *             * <br /> 1901 AACACTAATACTTAAMAAATTOCMAAGATACaC <br />                                      CAAiATAATTTTaTCAATGACOAA TOCATsCAG9OATTOaTGAT=ATTCTCTTOTGTTACAAGACTAOCTCTCAAATOAAACCAAT GTTACAAT <br />        N T N T L N A R L C B                  I R Q I    I F Q G T N G C T V S           I G C S D S L V L A R L A LK M A K P N G Y N <br />             *       *       FcoRI     *      *       *      *      *        *        *        * <br /> 1951 ATCC_ _aAGATAACTAICTAA       MTCTACMAACTAGA7AICTlTOAOT7 CTICTACTCMaTCMAAGTTAATCAACGTTATAGATCACATTCmMTaTTAAGACMAGAAAGATATACTTA <br />       I T F K S N L S B B F W S S F K[ L D D L P V G H S T L S R L E S T F D 8 P H S L N D L R K R Yr T L <br />              *         *         *       *        *        *        *        HindII     *         *        *        *       *     BcoRV        * <br /> 2101 (;ATMOCTT:AGTCAMOTTaaTATMAGTTCAGTA7AGACTCaCaTAMaaCMACtATAMAAUGAT tGATAA AA CTTAATATGTaAtCOACMM AGAAGCTTAC AAAGAAAATCA?TTATCAT7AT&amp;CAATTWGA <br />       D A L K[ A S V G S       R   L G M   9 I H L A L Q     G  Q D D B L S L K I         L  Y  D   P X R  V  L Q R   K S L   S I  D  I N W  G <br />               5 *-      *   Accl*       *        *        *       *       * <br />                                                                             CMAAACAATCMACAAATTMCATTAAAATGAGMMAAGA7T AAAaCOCTCC ATTGAACCCCCA <br /> 2251 ATCAGATTTAAAATATCAiCTCA TAIGACTMAT^AGAAAAGTCATA TCTTTTAGAAAATTaAATBTAAtM <br />       I R F K N I T Q V D L F I L R S C Q Y L L R K L N L I N K T T S Q I T L K L M R R C K D A P I K P P <br />         ,      *            *       *           *        *         *       *        *          *        *        *             *           *       *             * <br /> 2401 AAATATAl cTsOaaAJlAoTocA?cAaTCcGn&#194;&#162;MCAJCAATTFAGTATTCCAACAAAMATTTGAATTATTOCTACCGAAATGMMAAAG TaTATCGAAC l= TCCMAA7dATTAAGAaGlCTMCTCTA <br />       K[ Y M M G R C D S F S R S S R L G I P T N L F G I I A T 9 M K S L Y R T L G C P P M L L R. G L A L <br />                    *        *       *           *PvuII   *         *       *        *          *        *        *             *           *       *             * <br /> 2551   CAATCAACAAMTTWT7ATTT AATM CACTQAACTGGMMACCTTAAAACAAATMCACAG,ACCTTTCAACTACCOGAAGTAAAAATGACATTAACAACGAGTTCGAGAAGAATTAT <br />         Q F N K[ L V D V P D N N L K L R L P F K T I V T N R A F E A L P 8 D V K N D I N N 8 F 8 K R N Y <br />            *     flincII *    SacI*     *        *        *       *        *ScaI   *             *      *       *        *        * <br /> 2701 AGGAAATCGr&amp;MACtTGIG <br />   2701 -ShMaMAMTCATT^aCT _AAMAT TMCAATTAAGOATTTAGCATMTCCGATTAGATCCAAATATQTACTAT  OGAAGACACTTTATaM GAAMCTACCAACC CAATTCG OCAGAAGAAGG <br />      K R R 8 S 0 L T S N S L S S K K K G F A I S R L I V N D L P S T M 8 B H F M N B L P T Q I R A 8 V R <br />                *IcoRI       *       *           *        *         *       *        *4         *        *        *             *           *       *        * <br /> 2851 cAcoAcTTraAGuMAMAAATCcAAcACM)GAAAsMACTACTACAAGAMAAAGATAAGAAGAAMaMSC                TACAAMcGAaATAAAATCATTTCA7MT^AATATATATTCCAOCC       ATCAAATTCCAAATCTG <br />         R D L R I Q K K I Q Q T K L G N L Q 8 K I K R R E E S L Q N E K N H F M G Q N S                                            I   F Q P I K F Q N L <br />  <br /> 3001 ACAMTM hUAAATTTqMATTAMAACTAATGTTCCM    AAACTT;TAGMATOGAGOOCCaATGAMAAAATGTTAAATTATTCGTGAAATATTOAsTTAAACTTTOCGATTCTAAT^AGATCCATTAGTTCTTCATTA <br />       T R F * K K I C* 0Q L V * K Q W * V A K T* L G D * Q G P * H B K D* V K L * F V K *&apos;Y L I K* L C D * S N R * V H L V* L H L * <br />  <br /> 3151 TCAACCTAATATsCAAOMTTMAAAIOCTt&apos; TMTTTAATAGTCATTCAaCT CGWTT&lt;CAG2ACTTWAAATTATTC7TCAAGTAS TAT7CCACTTTTAAACAGAAATAAACATACTTACCJAGACTGTCGTAAACTTGAC <br />       S N L I S R 8 L N L C A F L N Q D ZI S G F Q T W B R I L L N D I I P L L N R N Kt H T Y Q T V R K L D <br />                *            *       *           *         *  4   *       *       *       *      *                *         *      *       *       * <br /> 3301 ATOGCTSTAA   ATTTAAAAAAAAAACAT                 AC   TAAACACAGTTTGCGACATTACCTGTATA7CTOCATTTTAlCAAG               AATACTACACATGACATTAATAAAATTAT <br />         N D F K V * <br />                *            *       *BcoRI <br /> 3451 AAAAAAAATAA40ACTTTAATTTTTACTA0AA?TC 3485 <br />   FIG. 6. DNA sequence REV] gene flanking regions. Restriction endonuclease cleavage sites underlined. Nucleotide <br /> numbers shown margin. The deduced amino acid sequence (in single-letter <span id='am-4' about='Thesaurus:code' typeof='owl:Thing'>code</span>) given nucleotide sequence. Symbols: <br /> V, transcript start points; A, endpoints 5&apos; deletions; A, endpoints 3&apos; deletions. <br />  236     LARIMER ET AL.                                                                                                      J. BACTERIUOL. <br />  <br />              380       390      400           410           420       additional domains generate novel mutability <br /> REV1     CRSSSFSACDFKR     CHGTKNSDIA          SYGIKCNG-NWVSIAEK      phenotypes. <br /> uMUc      !-ETVFRP-LGINILSIPJGC <br />                 20           30 <br />                                         L&amp;AA- <br />                                         40  60      50 <br />                                                                                           ACKNOWLEDGMENTS <br />              430      440      450      460                470          We pleased acknowledge Bruce Suttle technical assis- <br /> REVI     NLPNGIKLISLPYTFEOOLLEAFYI&amp;RL4IFNLILPI             ViCVR      tance DNA sequencing. We indebted Richard Mural and <br /> UMUC      URR-CG6I4CS      AAlN!         EESPRIEI-T         FF-D      Thomas Soper discussion manuscript. <br />                      70      80         90           100                This research sponsored Office Health Envi- <br />                                                                       ronmental Research, U.S. Department Energy, contract <br />                480        490       SO          510        520        DE-AC05-84OR21400 Martin Marietta Energy Systems, Inc. <br /> REVI       IIPDNIHNTNTLNARLCEEIRbEIF SITN6CISICSDSLV        ULKA <br /> UMUC       -LTG-JdICRD1-TDJGREIVATUJL <br />               110         120       130 <br />                                          YILWIA&amp;TKT!H <br />                                                  140 <br />                                                                                               LITERATURE CITED <br />                                                             150 <br />                                                                        1. Angulo, J. F., J. Schwenke, P. L. Moreau, E. Mouustacchi, R. <br />   FIG. 7. Similarity REV1 protein UMUC protein.               Devoret. 1985. A yeast protein analogous Escherichia coli <br /> Identities indicated solid blocks; similarities scoring &gt;+1        RecA protein cellular level enhanced UV irradi- <br /> (using Mutation Data Matrix) indicated vertical lines. A       ation. Mol. Gen. Genet. 201:20-24. <br /> FASTP similarity search (26) performed (k-tuple = 2), using    2. Bennetzen, J. L., B. D. Hall. 1982. The primary structure of <br /> translation REV) translations 16,749 sequences           Saccharomyces cerevisiae gene alcohol dehydrogenase <br /> GenBank. The average score 24.4, standard deviation         I. J. Biol. Chem. 257:3018-3025. <br /> 22.26. The optimized score REVI versus UMUC 128. The z         3. Bennetzen, J. L., B. D. Hall. 1982. Codon selection yeast. <br /> score alignment 4.65.                                        J. Biol. Chem. 257:3026-3031. <br />                                                                        4. Braus, G., G. Panvivini, R. Hitter. 1988. A consensus <br />                                                                           transcription termination sequence promoter region is <br /> Figure 7 gives alignment 152 residues REV1                  necessary efficient gene expression TRPI gene of <br /> protein segment UMUC protein. There 38                  Saccharomyces cerevisiae. Mol. Gen. Genet. 212:495-504. <br /> matches (25% identity) alignment. When residues                5. Breathnach, R., P. Chambon. 1981. Organization and <br /> with similarity scores &gt; + 1 added identities,          expression eucaryotic split genes coding proteins. Annu. <br />                                                                           Rev. Biochem. 50:349-383. <br /> two sequences 42% similarity. The notable block              6. Broach, J. R., J. N. Strathern, J. B. Hcks. 1979. Transfor- <br /> of identities, SIDEA-C, partly conserved (SIDE)                mation yeast: development hybrid cloning vector and <br /> the umuC homolog mucB (40). Both REV) umuC                        isolation CAN) gene. Gene 8:121-133. <br /> required error-prone repair, functionally supporting           7. C          n, M. J., A. Martinez-Arias, S. K. Shapira, J. <br /> suggestion peptide segments homologous.                    Chou. 1983. (3-Galactosidase gene fusions analyzing gene <br /> This alignment prove useful basis targeting               expression Escherichia coli yeast. Methods Enzymol. <br /> mutations identify functionally important segments              100:293-308. <br /> REV1 protein.                                                          8. Cian, A. M., T. F. Donahue. 1987. Sequence structural <br />                                                                           features associated translational initiator regions yeast- <br />   Expression REV). The REV)::lacZ fusion results sug-                  review. Gene 59:1-18. <br /> gest REV) constitutively expressed, number            9. Cole, G. M., D. Schild, S. T. Lovett, R. K. Mortimer. 1987. <br /> of yeast repair genes (32, 33). The occurrence                   Regulation RAD54- RAD52-lacZ gene fusions Saccha- <br /> poly(dA-dT) tracts consistent constitutivity (55),                romyces cerevisiae response DNA damage. Mol. Cell. <br /> although BclI d9letion (pFL157) indicates               Biol. 7:1078-1084. <br /> prominent T-rich segment required mutability.         10. Filetici, P., N. Junakovic, P. Ballario. 1985. Rapid alkaline <br />   Both ClaI (pFL146) PstI (pFL148) deletions lack                     preparation yeast circular covalently closed DNA mole- <br /> upstream sequences presumed necessary transcription,                  cules. Curr. Genet. 9:123-126. <br /> yet interruptions exhibit significant residual mutability.       11. Gannon, F., K. O&apos;Hare, F. Perrin, J. P. LePennec, C. Benoist, <br />                                                                           M. Cochet, R. Brentnach, A. Royal, A. Garapin, B. CanM, and <br /> Both constructs immediately downstream                  P. Chambon. 1979. Organization sequences 5&apos; end of <br /> a vector sequence known weak promoter activity                 cloned complete ovalbumin gene. Nature (London) 278:428- <br /> S. cerevisiae (4). The ClaI deletion produce hybrid               434. <br /> transcript initiate downstream start point         12. Hames, B. D., S. J. Higgins. 1985. Nucleic acid hybridiza- <br /> translated normally. The PstI deletion, however, re-                  tion. IRL Press, Oxford. <br /> moved N-terminal coding sequence. The se-                 13. Hanan, D. 1983. Studies transformation Escherichia coli <br /> quence PstI site vector shows in-frame             plasmids. J. Mol. Biol. 155:557-580. <br /> ATG codon 34 bp upstream. A transcript truncated              14. Haynes, R. H., B. A. Kunz. 1981. DNA repair muta- <br /> gene initiate translation ATG, elsewhere,            genesis yeast, p. 371-414. In J. N. Strathern, E. W. Jones, <br />                                                                           J. R. Broach (ed.), The molecular biology yeast <br /> yield truncated REV1 protein. This suggests                     Saccharomyces. Life cycle inheritance. Cold Spring Harbor <br /> N-terminal segment REV1 protein dispensable.                    Laboratory, Cold Spring Harbor, N.Y. <br />   Residual mutability rev)-l. The extreme Rev- pheno-              15. Hohn, B., knd J. Coins. 1980. A small cosmid efficient <br /> type revlA: :HIS3 gene disruption, compared               cloning large DNA fragments. Gene 11:291-298. <br /> reference allele rev)-) (Table 1), implies Arg-193           16. Hu, N. T., J. Messing. 1982. The making strand-specific <br /> missense protein encoded rev)-),                  M13 probes. Gene 17:271-277. <br /> truncated gene product revl::pFL44, retains residual Rev           17. Ito, H., Y. Fukuda, K. Murata, A. Kinura. 1983. Transfor- <br /> activity. Consequently, precise description rev)-)                 mation intact yeast cells treated alkali cations. J. <br /> mutability (14, 20, 25) need reinterpreted light          Bacteriol. 153:163-168. <br /> rev)-) leakiness. The location rev)-) substitution <br />                                                                       18. Jentsch, S., J. P. McGrath, A. Va <br /> </body></html>