<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>OsMADS51 Is Short-Day Flowering Promoter That <br /> Functions Upstream Ehd1, OsMADS14, Hd3a1[W][OA] <br /> Song Lim Kim2, Shinyoung Lee2, Hyo Jung Kim, Hong Gil Nam, Gynheung An* <br /> Division Molecular Life Sciences Biotechnology Research Center, Pohang University Science <br /> and Technology, Pohang 790&#226;&#8364;&#8220;784, Republic Korea <br />  <br />  <br /> Although flowering regulatory mechanisms extensively studied Arabidopsis (Arabidopsis thaliana), other <br /> species elucidated. Here, investigated role OsMADS51, type I MADS-box gene short-day <br /> (SD) promotion pathway rice (Oryza sativa). In SDs OsMADS51 <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-6' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span>null mutants</span> flowered 2 weeks later normal, in <br /> long days loss OsMADS51 little effect flowering. Transcript levels flowering regulators&#226;&#8364;&#8221;Ehd1, OsMADS14, <br /> and Hd3a&#226;&#8364;&#8221;were decreased mutants, OsGI Hd1 unchanged. Ectopic expression of <br /> OsMADS51 caused flowering occur 7 d earlier SDs. In ectopic expression lines, transcript levels Ehd1, <br /> OsMADS14, Hd3a increased, OsGI Hd1 remained same. These results indicate OsMADS51 is <br /> a flowering promoter, particularly SDs, gene functions upstream Ehd1, OsMADS14, Hd3a. <span id='am-3' about='protege:TO' typeof='owl:Thing'>To</span> further <br /> investigate relationship flowering promoters, generated transgenic plants expression Ehd1 or <br /> OsGI suppressed. In Ehd1 <span id='am-316' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-317' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-318' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-319' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-320' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-321' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-322' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-323' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-324' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-325' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-326' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-327' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-328' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-330' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-331' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-332' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-333' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-334' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-335' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-337' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-338' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-341' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-342' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-344' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-345' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-347' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-348' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-349' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-350' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-351' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-352' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-353' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-354' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-355' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-356' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-357' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-358' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-359' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-362' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-363' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-364' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-365' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-366' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-367' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-368' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-369' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-370' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-372' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-373' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-374' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-375' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-376' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-377' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-378' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-379' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-380' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-381' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-382' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-383' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-384' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-385' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-386' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-388' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-389' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-390' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-391' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-392' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-393' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span>RNA interference</span> plants, OsMADS51 expression affected, supporting conclusion <br /> that MADS-box gene functions upstream Ehd1. However, OsGI antisense plants, OsMADS51 transcript level was <br /> reduced. In addition, circadian expression <span id='am-397' about='xsp:pattern' typeof='owl:Thing'>pattern</span> MADS-box gene similar OsGI. These results <br /> demonstrate OsMADS51 functions downstream OsGI. In summary, OsMADS51 novel flowering promoter that <br /> transmits SD promotion signal OsGI Ehd1. <br />  <br />  <br />  <br />    The ability adjust timing flowering                    little effect short days (SDs). GI expression is <br /> important factor trying adapt plants re-                       regulated circadian clock, peak tran- <br /> gional environments control harvest time.                       script levels 8 10 h dawn. Timing, height, <br /> Regulatory mechanisms control flowering                         duration peak influenced daylength <br /> been studied extensively Arabidopsis (Arabidopsis                      (Fowler et al., 1999). The GI protein acts the <br /> thaliana). In long-day (LD) species, flowering                    circadian oscillator CO promote flowering by <br /> promoted regulatory genes, including                           increasing CO FT mRNA abundance (Mizoguchi <br /> GIGANTEA (GI), CONSTANS (CO), FLOWERING                                   et al., 2005). CO putative transcription factor con- <br /> LOCUS T (FT), SUPPRESSOR OF OVEREXPRESSION                                taining B-box zinc fingers C-terminal CCT <br /> OF CO1 (SOC1), APETALA1 (AP1).                                        domain (Putterill et al., 1995). CO promotes flowering <br />    GI encodes nuclear protein defines proper                   response LDs SDs (Putterill et al., 1995). <br /> amplitude period <span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span> circadian rhythms                          CO protein activates expression early <br /> (Park et al., 1999; Mizoguchi et al., 2005). In Arabidop-                 target genes (e.g. SOC1 FT [Samach et al., 2000; <br /> sis, mutations GI delay flowering LDs,                     Suarez-Lopez et al., 2001]). FT, encodes pro- <br />                                                                           tein similar phosphatidylethanolamine-binding pro- <br />    1                                                                      teins Raf kinase inhibitor proteins, promotes <br />      This work supported Crop Functional <br />                                                                           flowering LDs (Kardailsky et al., 1999; Kobayashi <br /> Genomic Center, 21st Century Frontier Program (grant no. <br /> CG1111); Biogreen 21 Program, Rural Development Administra- <br />                                                                           et al., 1999). FT primarily expressed vascular tis- <br /> tion; Korea Science Engineering Foundation (KOSEF)                sues, interactor, FD, predominantly ex- <br /> through National Research Laboratory Program funded            pressed shoot apex (Abe et al., 2005; Wigge et al., <br /> Ministry Science Technology (grant no. M10600000270&#226;&#8364;&#8220;               2005). SOC1 integrates vernalization GA signals <br /> 06J0000&#226;&#8364;&#8220;27010); Korean government (KOSEF grant no.                flowering (Moon et al., 2005). Its expression af- <br /> R15&#226;&#8364;&#8220;2004&#226;&#8364;&#8220;033&#226;&#8364;&#8220;05002&#226;&#8364;&#8220;0).                                                    fected loss FT, activation-tagged FT <br />    2 <br />      These authors contributed equally article.                    mutants shown induce SOC1 up-regulation <br />    * Corresponding author; e-mail genean@postech.ac.kr.                   (Schmid et al., 2003; Moon et al., 2005). In contrast, FT <br />    The author responsible distribution materials integral   expression remains unaffected loss gain of <br /> findings presented article accordance policy <br />                                                                           SOC1 expression. In response FT-mediated floral <br /> described Instructions Authors (www.plantphysiol.org) is: <br /> Gynheung An (genean@postech.ac.kr). <br />                                                                           induction, AP1 contributes inflorescence meri- <br />    [W] <br />        The online version article contains Web-only data.         stem committed flowering (Huang et al., <br />    [OA] <br />         Open Access articles viewed online sub-          2005). <br /> scription.                                                                   In contrast Arabidopsis, rice (Oryza sativa) SD <br />    www.plantphysiol.org/cgi/doi/10.1104/pp.107.103291                     plant flowering inhibited early <br /> 1484       Plant Physiology, December 2007, Vol. 145, pp. 1484&#226;&#8364;&#8220;1494, www.plantphysiol.org &#195;&#8220; 2007 American Society Plant Biologists <br />                                                                                        OsMADS51, Short-Day Flowering Promoter Rice <br />  <br />  <br />                                                                           high night (Izawa et al., 2002; <br />                                                                           Kojima et al., 2002; Hayama et al., 2003). In contrast to <br />                                                                           CO, Hd1 double role regulating downstream <br />                                                                           flowering activators&#226;&#8364;&#8221;repression LDs promo- <br />                                                                           tion SDs (Yano et al., 2000). One downstream <br />                                                                           targets Hd3a, FT ortholog, acts preferentially <br />                                                                           inductive SDs (Kojima et al., 2002). In addition to <br />                                                                           photoperiod-controlled pathway, rice contains an <br />                                                                           alternative inductive pathway appears function <br />                                                                           independently Hd1. Ehd1, encoding B-type re- <br />                                                                           sponse regulator, promotes flowering inducing the <br />                                                                           expressions Hd3a, FTL1, OsMADS14 (Doi et al., <br />                                                                           2004). These results indicate different signal- <br />                                                                           ing pathways intrinsically involved photo- <br />                                                                           periodic flowering pathway rice. <br />                                                                              The relationship AP1 group MADS-box <br />                                                                           genes differ rice Arabidopsis. <br />                                                                           For example, AP1 related genes determine <br />                                                                           floral meristem identity downstream FT. However, <br />                                                                           rice, OsMASD14, member AP1 group, <br />                                                                           apparently functions upstream Hd3a flowering <br />                                                                           activator. Its ectopic expression causes significant early <br />                                                                           flowering tissue culture (Jeon et al., 2000), and <br />                                                                           transcription Hd3a up-regulated OsMADS14- <br />                                                                           overexpressing cultured cells (S. Lee G. An, un- <br />                                                                           published data). <br />                                                                              Most plants eventually flower, nonper- <br />                                                                           missive conditions. This autonomous pathway reg- <br /> Figure 1. OsMADS51 genomic structure, positions T-DNA inser-           ulated mainly SOC1 Arabidopsis. The gene acts <br /> tional mutant alleles, osmads51 mutant phenotypes. A, Schematic       flowering inducer, result converging photo- <br /> diagram T-DNA insertional position OsMADS51. This gene              period, autonomous, vernalization, GA pathways <br /> consists exons (black boxes) introns (lines      (Araki, 2001; Mouradov et al., 2002; Simpson Dean, <br /> black boxes). T-DNA located 9,606 bp (osmads51-1) 15,927 <br />                                                                           2002). OsMASD50, Arabidopsis SOC1 homolog, <br /> bp (osmads51-2) translation start site. F1 R1 primers used <br /> for RT-PCR real-time PCR analyses. F2, R2, Gus2R primers <br />                                                                           acts flowering activator similar SOC1. Ectopic <br /> for genotyping osmads51-1. F3, R2, LB2R primers genotyp-      expression causes dramatic early flowering re- <br /> ing osmads51-2. B, Semiquantitative RT-PCR OsMADS51 transcript         generating callus stage, OsMASD50 mutants flower <br /> in osmads51 mutants (HO) wild-type segregants (WT). Actin1        month late (Lee et al., 2004; Lee An, 2007). <br /> used normalization cDNA quantity. PCR cycles numbered 28           Most MADS-box genes regulate flowering time <br /> OsMADS51 25 Actin1. C, <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span>Phenotype</span> OsMADS51 mutants               belong type II, group contains conserved <br /> (HO) compared wild-type segregants (WT). Plants cultivated      MEF2-like MADS-box K regions. In addition, in- <br /> in growth room SD conditions (10 h light/14 h dark). Photo-       termediate (I) C-terminal (C) regions present. <br /> graph taken 75 DAG. Arrows indicate panicles emerging           Therefore, termed MIKC-type genes (Munster <br /> heading stage. <br />                                                                           et al., 1997). The possible roles group, type <br />                                                                           I MADS-box genes flowering time, well <br /> developmental stages, induced day-                           understood. Type I group proteins contain conserved <br /> length shortens fall. Although roles genes                  SRF-like MADS domain, C-terminal region is <br /> involved photoperiod pathway generally                         clearly defined. Furthermore, type I proteins do <br /> conserved rice Arabidopsis, differ-                      carry conserved K-box region (Alvarez-Buylla <br /> ences exist. OsGI, GI ortholog, regulates flowering <br /> time response photoperiodic conditions. Muta- <br /> tions gene delay flowering inductive SDs, <br />                                                                           Table I. Days heading osmads51 <br /> whereas flowering marginally induced LDs <br /> (Hayama et al., 2003). In comparison, overexpression                       Days heading scored panicle appeared. WT, <br />                                                                           Wild type; HO, osmads51/osmads51 homozygotes. <br /> of OsGI transgenic rice causes late flowering both <br /> SDs LDs. Expression Hd1, ortholog CO,                                                  Days Heading    Days Heading <br />                                                                                 Alleles     Genotypes <br />                                                                                                            SD           LD <br /> affected plants, indicates OsGI <br /> functions regulate Hd1, gene shows                            osmads51-1       WT           75    6   2.2      96   6   3.8 <br /> circadian rhythm (Hayama et al., 2003). However,                                          HO           89    6   5.2     100   6   3.2 <br /> expression pattern differs OsGI. Hd1                            osmads51-2       WT           75    6   2.2      97   6   2.5 <br />                                                                                               HO           90    6   3.8     101   6   4.0 <br /> transcript levels lowest midday and <br /> Plant Physiol. Vol. 145, 2007                                                                                                                1485 <br />  Kim et al. <br />  <br />  <br />                                                                           RESULTS <br />                                                                           Isolation T-DNA Insertional Mutants OsMADS51 <br />                                                                              We previously reported generation a <br />                                                                           T-DNA insertional mutant population japonica rice <br />                                                                           (Jeon An, 2001; Jeong et al., 2002, 2006; An et al., <br />                                                                           2003, 2005b; Ryu et al., 2004). The sequence database is <br />                                                                           open public (www.postech.ac.kr/life/pfg/risd) <br />                                                                           seeds available scientific community. <br />                                                                           <span id='am-1' about='protege:FROM' typeof='owl:Thing'>From</span> database, identified indepen- <br />                                                                           dent T-DNA insertional mutant alleles, osmads51-1 and <br />                                                                           osmads51-2, T-DNAs inserted first <br />                                                                           intron alleles, 9,606 15,927 bp the <br />                                                                           start codon osmads51-1 osmads51-2, respectively <br />                                                                           (Fig. 1A). Homozygous (osmads51/osmads51) plants <br />                                                                           did accumulate OsMADS51 transcript (Fig. 1B). <br />                                                                           Under SDs, flowering homozygous progeny of <br />                                                                           alleles 2 weeks later wild-type <br />                                                                           segregants (Fig. 1C; Table I). In contrast, little change in <br />                                                                           flowering time noted LDs (Table I). These <br />                                                                           results indicate OsMADS51 flowering pro- <br />                                                                           moter functions preferentially SDs. <br />  <br />  <br />  <br />                                                                           Ectopic Expression OsMADS51 Causes <br />                                                                           Early Flowering <br />  <br /> Figure 2. OsMADS51 overexpression. A, Schematic diagram UbiT              To elucidate functional roles OsMADS51, <br /> OsMADS51 construct. OsMADS51 full-length cDNA clone                   constructed chimeric binary vector (Ubi: <br /> placed Ubi promoter (Pubi) nos terminator (Tnos). B,          OsMADS51) placing OsMADS51 full-length <br /> Semiquantitative RT-PCR analyses UbiTOsMADS51 transgenic (OX)          cDNA maize ubiquitin (Ubi) promoter (Fig. <br /> lines 3 4 segregating wild-type controls (WT). Actin1   2A). This construct introduced Agrobacterium <br /> used normalization cDNA quantity. RT-PCR performed        transferred plant cells cocultivation (An <br /> 28 cycles OsMADS51 25 cycles Actin1. C, Phenotype <br />                                                                           et al., 1985; Hiei et al., 1997). Seven primary (T1) trans- <br /> UbiTOsMADS51 transgenic plants (OX) segregating wild- <br /> type controls (WT) cultivated 62 d SD conditions. Arrows <br />                                                                           genic plants obtained planted paddy <br /> indicate heading panicles.                                                field. All showed early flowering, albeit varying de- <br />                                                                           grees. For example, lines flowered 2 weeks early, <br />                                                                           lines 10 d sooner, lines just <br /> et al., 2000). Although overexpression type I                        3 d compared control transgenic plants. <br /> MADS-box gene, AGAMOUS-LIKE28 (AGL28), pro-                               The extent early flowering correlated with <br /> motes flowering autonomous pathway                          OsMADS51 expression levels confirmed by <br /> Arabidopsis (Yoo et al., 2006), roles MADS-                   northern-blot analyses (data shown).We then <br /> box genes type remain unknown.                                    chose lines (3 4) manifested early flower- <br />    Here, elucidated functional role                      ing 10 d selected homozygous plants and <br /> type I MADS-box gene, OsMADS51, rice. This gene                        segregating wild type using hygromycin selection. <br /> was originally identified large-scale cDNA                        T3 progeny reared growth room either <br /> analysis rice genome research program                          SDs (10 h light) LDs (14 h light) elucidate the <br /> (Shinozuka et al., 1999). OsMADS51 unique rice; a <br /> <span id='am-394' about='Thesaurus:Homologous_Gene' typeof='owl:Thing'>homologous gene</span> lacking Arabidopsis. However, <br /> homologous genes occur, 74% identity in <br /> wheat (Triticum aestivum; accession no. ABF57941) and <br />                                                                           Table II. Days heading OsMADS51-overexpressing plants <br /> maize (Zea mays; accession no. CAD23412). Nevertheless, <br />                                                                            Days heading scored panicle appeared. WT, <br /> their roles elucidated. OsMADS51 is <br />                                                                           Wild type; OX, homozygous UbiTOsMADS51. <br /> composed 164 amino acids contains SRF-like <br /> MADS domain. However, like type I MADS-box                                                             Days Heading   Days Heading <br />                                                                              Transgenic Plants   Genotypes <br />                                                                                                                 SD          LD <br /> genes, K-box region conserved the <br /> C-terminal region short. Expression ubiquitous                          Line 3            WT          72   6   3.8      93   6   3.1 <br /> all organs (Shinozuka et al., 1999). In study,                                             OX          63   6   4.0      91   6   2.4 <br /> have investigated function OsMADS51                                  Line 4            WT          73   6   1.7      94   6   2.4 <br />                                                                                                    OX          66   6   2.3      93   6   3.3 <br /> regard flowering time. <br /> 1486                                                                                                            Plant Physiol. Vol. 145, 2007 <br />                                                                     OsMADS51, Short-Day Flowering Promoter Rice <br />  <br />  <br /> daylength-dependent roles OsMADS51. Homozy-            investigated expression OsGI, Hd1, Ehd1, <br /> gous plants exhibited early flowering 9 d (line 3)     OsMADS14, Hd3a osmads51 mutants and <br /> and 7 d (line 4) SDs. However, significant pheno-   wild-type segregants grown continuous SDs <br /> typic difference observed homozygous          LDs. At 55 d germination (DAG), leaf blades <br /> transgenic lines wild-type plants grown LDs        harvested transcript levels measured <br /> (Table II). These results support conclusion     reverse transcription (RT)-PCR quantitative real- <br /> OsMADS51 SD flowering activator.                     time PCR (Fig. 3; Supplemental Fig. S1). Expression <br />                                                           OsGI Hd1 unchanged OsMADS51 <br /> Relationship OsMADS51 Various                     mutants lighting conditions (Fig. 3, A, B, D, <br /> Flowering-Time Regulators                                 E). However, transcript levels Ehd1, OsMADS14, <br />                                                           Hd3a greatly reduced osmads51 mutants <br />   To evaluate relationship OsMADS51           SDs LDs (Fig. 3, A, B, F, G, H). These results <br /> and previously identified flowering regulators,        indicate Ehd1, Hd3a, OsMADS14 probably <br />  <br />                                                                         Figure 3. Expression profiles various flowering <br />                                                                         regulators OsMADS51 mutants. Shown semi- <br />                                                                         quantitative RT-PCR analyses OsMADS51, OsGI, <br />                                                                         Hd1, Ehd1, OsMADS14, Hd3a wild-type (WT) <br />                                                                         OsMADS51 mutant plants (HO) grown 55 d <br />                                                                         SD (A) LD (B) conditions. Actin1 used <br />                                                                         normalization cDNA quantity. RT-PCR cycles <br />                                                                         numbered 28 flowering regulators 25 for <br />                                                                         Actin1. From independent experiments, <br />                                                                         reproducible data presented. Real-time <br />                                                                         PCR analyses OsMADS51 (C), OsGI (D), Hd1 (E), <br />                                                                         Ehd1 (F), OsMADS14 (G), Hd3a (H) WT and <br />                                                                         HO plants grown 55 d SD LD conditions. <br />                                                                         RNA prepared leaf blades harvested 0 h <br />                                                                         light turned on. Each data point the <br />                                                                         average independent experiments. <br />                                                                         y axis, Relative values transcript levels of <br />                                                                         regulatory genes Ubi; vertical bars, standard <br />                                                                         deviation. <br />  <br />  <br />  <br />  <br /> Plant Physiol. Vol. 145, 2007                                                                                          1487 <br />  Kim et al. <br />  <br />  <br /> function downstream OsMADS51. As expected,         (Fig. 4; Supplemental Fig. S2). We used 35-DAG plants <br /> transcript levels OsMADS51, OsGI, Ehd1, OsMADS14,      flowered early and, therefore, their <br /> and Hd3a higher SD-grown plants. Interest-        regulatory genes expressed earlier. In wild-type <br /> ingly, expression difference SDs LDs     control plants, transcript levels Ehd1, OsMADS14, <br /> was greatest OsMADS14 Hd3a. Therefore,         Hd3a low 35 DAG LDs (Fig. 4), <br /> appears LD-specific inhibitors coordinately regu-    transcripts detected 55 DAG (Fig. 3), <br /> late downstream genes. Hd1 candidate         showing age-dependent up-regulation patterns. As <br /> because proposed negative regulator   expected <span id='am-238' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-239' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-240' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-241' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-242' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-249' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-253' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-258' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-261' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-264' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-277' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-289' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-292' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-294' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-308' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-315' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span>knockout</span> experiment, Ehd1, Hd3a, <br /> in LDs (Yano et al., 2000), transcript    OsMADS14 transcript levels increased (Fig. 4, A, <br /> level higher LDs compared SDs.                 B, F, G, H) SD conditions, of <br />    Transcript levels examined   OsGI Hd1 significantly changed the <br /> genes overexpressing OsMADS51 transgenic plants        UbiTOsMADS51 transgenic plants daylength <br />  <br />  <br />  <br /> Figure 4. Expression profiles various flowering <br /> regulators OsMADS51-overexpressing lines. <br /> Shown semiquantitative RT-PCR analyses of <br /> OsMADS51, OsGI, Hd1, Ehd1, OsMADS14, and <br /> Hd3a WT UbiTOsMADS51 plants (OX) grown <br /> for 35 d SD (A) LD (B) conditions. Actin1 <br /> was used normalization cDNA quantity. RT- <br /> PCR cycles numbered 28 flowering regulators <br /> under SD; 32 Ehd1, OsMADS14, Hd3a; and <br /> 28 OsMADS51, OsGI, Hd1 LD. For <br /> Actin1, 25 cycles performed. From three <br /> independent experiments, reproducible data <br /> are presented. Real-time PCR analyses OsMADS51 <br /> (C), OsGI (D), Hd1 (E), Ehd1 (F), OsMADS14 (G), and <br /> Hd3a (H) WT OsMADS51 mutant plants (HO) <br /> grown 35 d SD LD conditions. RNA was <br /> prepared leaf blades harvested 0 h light <br /> was turned on. Each data point average two <br /> or independent experiments. y axis, Relative <br /> values transcript levels regulatory genes <br /> and Ubi; vertical bars, standard deviation. <br />  <br />  <br />  <br />  <br /> 1488                                                                                  Plant Physiol. Vol. 145, 2007 <br />                                                                      OsMADS51, Short-Day Flowering Promoter Rice <br />  <br />  <br /> (Fig. 4, A, B, D, E). These results support       Hd3a&#226;&#8364;&#8221;that demonstrated function down- <br /> conclusion OsMADS51 functions upstream            stream OsGI (Hayama et al., 2003; Doi et al., 2004). <br /> Ehd1, Hd3a, OsMADS14.                                 As expected, transcript levels reduced in <br />                                                           OsGI antisense plants (Fig. 5, E&#226;&#8364;&#8220;G). These results <br /> Expression Analyses OsMADS51 OsGI                   indicate OsGI does function pathway <br /> Antisense Plants                                          OsMADS51, OsGI occurring upstream. <br />  <br />    In osmads51 mutants UbiTOsMADS51 trans-            Expression Analyses OsMADS51 Ehd1-RNAi Plants <br /> genic plants, transcript levels OsGI unchanged <br /> (Figs. 3 4). Therefore, believe OsGI      In osmads51 plants, Ehd1 transcript level was <br /> functions upstream OsMADS51 inde-     significantly reduced (Fig. 3, A F), was <br /> pendent pathway. Hayama et al. (2003) previ-         notably increased UbiTOsMADS51 plants (Fig. <br /> ously reported OsGI RNAi plants delayed         4, A F). Therefore, likely Ehd1 located <br /> flowering SDs, observed late-          downstream OsMADS51. To confirm hypothe- <br /> flowering phenotype OsGI antisense plants.         sis, generated transgenic plants carrying Ehd1- <br /> Therefore, SD-specific effect suggests OsGI     RNA interference (RNAi) construct (Fig. 6B). In all, 30 <br /> and OsMADS51 function path-           independently transformed plants obtained, <br /> way. To investigate relationship    20 flowered later control. We <br /> genes, generated OsGI antisense plants       observed Ehd1 transcript levels reduced in <br /> that OsMADS51 transcript level reduced         transgenics delayed. Two RNAi plants <br /> plants grown SDs LDs (Fig. 5A). As        selected showed significant change <br /> controls, examined expression levels         flowering time. Under SD conditions, they <br /> other flowering regulators&#226;&#8364;&#8221;Ehd1, OsMADS14,            flowered 7 d later controls. Ehd1 transcript <br />  <br />                                                                         Figure 5. Expression level analyses flowering reg- <br />                                                                         ulators OsGi antisense plants. A, Schematic dia- <br />                                                                         gram OsGi. F1, F2, R1, R2 primers used for <br />                                                                         RT-PCR real-time PCR analyses. B, Schematic <br />                                                                         diagram OsGi antisense construct. C, RT-PCR <br />                                                                         analysis OsGi antisense (OsGi-as) plants their <br />                                                                         wild-type segregants (WT). OsGi-as plants were <br />                                                                         grown 55 d SDs LDs. RNA prepared <br />                                                                         leaf blades harvested 0 h light was <br />                                                                         turned on. Endogenous OsGI transcript ampli- <br />                                                                         fied F1 R1 primer sets, OsGI-as <br />                                                                         endogenous transcripts amplified with <br />                                                                         F2 R2 primers. PCR cycles numbered 28 OsGI <br />                                                                         25 Actin1. Actin1 used normalization <br />                                                                         cDNA quantity. Transcript amounts OsMADS51 <br />                                                                         (D), Ehd1 (E), OsMADS14 (F), Hd3a (G) were <br />                                                                         analyzed quantitative real-time PCR. Each data <br />                                                                         point average independent <br />                                                                         experiments. y axis, Relative values tran- <br />                                                                         script levels regulatory genes Ubi; vertical <br />                                                                         bars, standard deviation. <br />  <br />  <br />  <br />  <br /> Plant Physiol. Vol. 145, 2007                                                                                           1489 <br />  Kim et al. <br />  <br />  <br /> levels reduced late-flowering transgenic   level low beginning day (Fig. 7A), but <br /> plants significant RNAi tran-       increased light reached maximum the <br /> script accumulated (Fig. 6C). Because primers used   end day. In darkness, level continu- <br /> for detecting RNAi transcript amplified en-      ously reduced minimal amount. A similar rhythm <br /> dogenous Ehd1 transcript well, PCR    observed LDs (data shown). Under SDs, a <br /> band wild type.                                   similar induction curve observed OsMADS51 <br />    In Ehd1-RNAi plants, OsGI OsMADS51 tran-      (Fig. 7B), increasing light decreasing the <br /> script levels changed (Fig. 6, D E), con-   dark. However, change transcript amounts was <br /> firming function upstream Ehd1. As          great OsGI. Under LD, such <br /> previously reported (Doi et al., 2004), expression    rhythm (data shown). <br /> Hd3a significantly reduced Ehd1-RNAi <br /> plants (Fig. 6G). The OsMADS14 transcript level      Relationship OsMADS51 OsMADS50 <br /> also reduced RNAi plants, confirming pos- <br /> sibility flowering regulators down-          The relationship OsMADS51 OsMADS50 <br /> stream OsGI (Fig. 6F).                                investigated. We previously reported <br />                                                          osmads50 plants late-flowering phenotype <br /> Diurnal Rhythms OsMADS51 OsGI                     (delay 1 month), overexpressors of <br />                                                          OsMADS50 display extreme early-flowering phe- <br />   Because OsMAD51 functions OsGI             notype (Lee et al., 2004). Here, mutations down- <br /> Ehd1, transcript level MADS-box gene        regulated expression OsMADS14 Hd3a, <br /> oscillate OsGI Ehd1. Therefore, measured    suggesting OsMADS50 functions upstream of <br /> OsMADS51 transcription leaf blades 55-DAG        OsMADS14. To evaluate OsMADS50 another <br /> plants grown SDs LDs (Fig. 7). As reported      flowering activator influence OsMADS51, we <br /> previously (Hayama et al., 2003), OsGI transcript    examined OsMADS51 expression altered <br />  <br /> Figure 6. Expression level analysis flowering reg- <br /> ulators Ehd1-RNAi plants. A, Schematic diagram of <br /> Ehd1. F1, F2, R1, R2 primers used RT-PCR <br /> and real-time PCR analyses. B, Schematic diagram of <br /> Ehd1-RNAi construct. C, RT-PCR analysis Ehd1- <br /> RNAi plants wild-type segregants (WT). <br /> Endogenous Ehd1 transcript amplified F1 <br /> and R1 primer sets Ehd1-RNAi endog- <br /> enous transcripts amplified F2 R2 <br /> primers. PCR cycles numbered 28 Ehd1 25 for <br /> Actin1. Actin1 used normalization cDNA <br /> quantity. D G, Real-time PCR analysis OsMADS51 <br /> (D), OsGI (E), OsMADS14 (F), Hd3a (G) Ehd1- <br /> RNAi plants. Samples RT-PCR real-time PCR <br /> were harvested 0 h light turned on. Each <br /> data point average independent <br /> experiments. y axis, Relative values tran- <br /> script levels regulatory genes Ubi; vertical <br /> bars, standard deviation. <br />  <br />  <br />  <br />  <br /> 1490                                                                                   Plant Physiol. Vol. 145, 2007 <br />                                                                                     OsMADS51, Short-Day Flowering Promoter Rice <br />  <br />  <br />                                                                        rice Arabidopsis (Izawa et al., 2003). <br />                                                                        However, regulatory factors just one <br />                                                                        model systems. For example, FLC present <br />                                                                        Arabidopsis, Ehd1 exclusive rice <br />                                                                        (Michaels Amasino, 1999; Doi et al., 2004). There- <br />                                                                        fore, understanding mechanisms regulate <br />                                                                        flowering involves focusing species-specific <br />                                                                        genes gain essential information elucidate <br />                                                                        details diversification species. In <br />                                                                        fact, existence FLC family Arabidopsis <br />                                                                        makes possible explain vernalization pathway <br />                                                                        crucifers, does exist rice (Michaels and <br />                                                                        Amasino, 1999). <br />                                                                           We, therefore, speculated MADS-box <br />                                                                        genes diversified control gene-specific <br />                                                                        flowering mechanisms. OsMADS51 selected for <br />                                                                        investigation regard MADS-box genes <br />                                                                        group grass species. <br />                                                                        OsMADS51 identified differential analy- <br />                                                                        sis transcripts mature spikelets (Ciaffi et al., <br />                                                                        2005). Although expression pattern OsMADS51 <br />                                                                        studied (Shinozuka et al., 1999), our <br />                                                                        report concerns functional roles gene. <br />                                                                           Our transgenic analyses showed OsMADS51 is <br />                                                                        SD-dependent flowering activator. Under SDs, <br />                                                                        OsMADS51 mutants flower 2 weeks later com- <br /> Figure 7. Diurnal rhythms OsMADS51 (A) OsGI (B) SD        pared control, transformants that <br /> conditions. Plants grown SDs 10 h light (0&#226;&#8364;&#8220;10 h, white   overexpress OsMADS51 flower 1 2 weeks early. <br /> box) 14 h dark (10&#226;&#8364;&#8220;24 h, black box). RNA prepared leaf <br />                                                                        Under LDs, effects milder, OsMADS51 <br /> blades 50-DAG plants 0, 4, 8, 12, 16, 20 h light was <br /> turned on. Each data point average independent <br />                                                                        mutants flowering 3 4 d later overexpressors <br /> experiments. y axis, Relative values transcript levels      flowering 1 2 d early. In rice, SD <br /> regulatory genes Ubi; vertical bars, standard deviation.           promotion pathways proposed&#226;&#8364;&#8221;one medi- <br />                                                                        ated Hd1 Ehd1 (Yano et al., 2000; <br />                                                                        Doi et al., 2004). These pathways converge induce <br /> in osmads50 plants. However, proved be <br />                                                                        Hd3a resultant protein product moved the <br /> the case (Supplemental Fig. S3). We that <br />                                                                        shoot apical meristem induce flowering (Tamaki <br /> levels OsMADS50 unchanged osmads51 <br /> plants (Supplemental Fig. S3). Our results indicate that <br /> these MADS-box genes function independently to <br /> control flowering time. <br /> Analysis Transcript Levels OsMADS51 According to <br /> Developmental Stage <br />    Because OsMADS51 early-acting activator, we <br /> measured transcript levels various developmen- <br /> tal stages remain constant <br /> or change plants matured. At 0 h (immediately after <br /> the light turned on), sampled leaf blades from <br /> plants grown SDs. Real-time PCR analyses showed <br /> that transcription high early develop- <br /> mental stages, day 43, declined and <br /> remained low remaining period (Fig. 8). This <br /> indicates OsMADS51 functions early 2 <br /> months growth rice. <br />                                                                        Figure 8. Changes OsMADS51 transcript levels growth <br />                                                                        phase. Plants reared SD conditions (10 h light/14 h dark). <br /> DISCUSSION                                                             RNA prepared leaf blades 13-, 28-, 43-, 54-, 69-, 81-, and <br />                                                                        97-DAG plants 0 h light turned on. Each data point the <br />   Intensive molecular genetics studies regula-                  average independent experiments. y axis, Relative <br /> tory mechanisms control flowering time                       values transcript levels regulatory genes Ubi; vertical <br /> revealed major players conserved                    bars, standard deviation. <br />  <br /> Plant Physiol. Vol. 145, 2007                                                                                                           1491 <br />  Kim et al. <br />  <br />  <br />                                                                          target OsGI, levels OsMADS51 and <br />                                                                          Ehd1 down-regulated OsGI antisense plants. <br />                                                                          These results suggest OsMADS51 acts down- <br />                                                                          stream OsGI. The daily rhythm observed with <br />                                                                          OsMADS51 supports hypothesis because <br />                                                                          rhythm similar OsGI, showing a <br />                                                                          peak middle day (Fig. 7). Therefore, the <br />                                                                          SD promotion pathway mediated OsGI be <br />                                                                          divided routes. One well-known Hd1- <br />                                                                          mediated pathway, the OsMADS51- <br />                                                                          Ehd1-mediated pathway. Our finding consistent <br />                                                                          previous report Hd1 works regardless of <br />                                                                          function Ehd1 (Doi et al., 2004). Furthermore, <br />                                                                          study confirms Hd1 Ehd1 operate inde- <br />                                                                          pendently provides additional evidence that <br /> Figure 9. Model flowering-signal pathways rice photo-     OsGI common upstream regulator Hd1 Ehd1 <br /> period signal translated OsGI, divided inde-   (Fig. 9). <br /> pendent pathways. One mediated Hd1; OsMADS51             Both Ehd1 OsMADS51 occur orphan genes <br /> and Ehd1. OsMADS50 generates independent signal turns on <br />                                                                          family rice genome putative <br /> OsMADS14. All signals finally activate Hd3a, produces a <br /> putative florigen. <br />                                                                          orthologs Arabidopsis genome. <br />                                                                          Therefore, unique genes probably function dif- <br />                                                                          ferentiate SD-inducible rice LD-inducible Arabi- <br />                                                                          dopsis. Thus, conclude OsMADS51 plays <br /> et al., 2007). To investigate relationship                  important role transmitting flowering signal <br /> regulators OsMADS51, studied expres-                       OsGI Ehd1 SD promotion pathway and <br /> sion patterns OsMADS51 mutants. Transcript levels                     route independent previously repor- <br /> from OsMADS14 examined                            ted pathways mediated Hd1 OsMADS50 (Fig. 9). <br /> gene down-regulated Ehd1 mutant back- <br /> ground (Fig. 3) ectopic expression causes <br /> significant early flowering (Fig. 4). <br />    RT-PCR quantitative real-time PCR analyses                        MATERIALS AND METHODS <br /> demonstrated Ehd1 expression down-regulated                      Plant Materials Growth <br /> in OsMADS51 mutants up-regulated Ubi: <br /> OsMADS51 plants. This suggests OsMADS51                                T-DNA-tagging lines generated rice (Oryza sativa) var. japonica <br />                                                                          &#226;&#8364;&#732;Dongjin&#226;&#8364;&#8482; &#226;&#8364;&#732;Hwayoung&#226;&#8364;&#8482; (Jeon et al., 2000; Jeong et al., 2002). Insertion sites <br /> acts upstream Ehd1. To explore relation-                 determined inverse PCR (An et al., 2003; Ryu et al., 2004; Jeong et al., <br /> ship OsMADS51 Ehd1, generated                             2006). T-DNA-tagged mutants searched flanking sequence <br /> Ehd1 RNAi plants measured OsMADS51 mRNA                              database (www.postech.ac.kr/life/pfg/risd), linked Salk <br /> levels RNAi plants. The previously identified Ehd1                    Institute Genomic Analysis Laboratory (http://signal.salk.edu/cgi-bin/RiceGE; <br />                                                                          An et al., 2005a, 2005b). Plants reared growth rooms either <br /> mutant allele amino acid change reduces <br />                                                                          continuous SDs (10 h light/14 h dark) LDs (14 h light/10 h dark). <br /> DNA-binding ability reduces Ehd1 function (Doi <br /> et al., 2004). Our RNAi results support role <br /> of Ehd1 flowering activator. In Ehd1 RNAi                       Screening OsMADS51 Mutants from <br /> plants, OsMADS51 mRNA levels similar                         T-DNA-Tagging Lines <br /> RNAi control plants, OsMADS14                                The osmads51 homozygous progeny screened T1 seeds mu- <br /> Hd3a expression affected <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span>mutation</span>. There-                     tants. Genotyping performed 35 cycles 95&#194;&#176;C 15 s, 55&#194;&#176;C 30 s, <br /> fore, combined results OsMADS51                        72&#194;&#176;C 40 s, using combination specific T-DNA border primer sets <br />                                                                          follows. The primer sets osmads51-1 included F2 primer (5#-CAACAACT- <br /> mutants, conclude OsMADS51 transmits                           GACGAGGAGCAC-3#), R2 primer (5#-TAGATCATGACCTCCAGTAC-3#), and <br /> flowering signal Ehd1, turns                       Gus2R primer T-DNA (5#-TTGGGGTTTCTACAGGACGTAAC-3#). The <br /> OsMADS14 Hd3a. OsMADS51 regulates Ehd1,                              osmads51-2 primer sets F3 primer (5#-CCTCCCATGTTGTGTTCCTT-3#), <br /> OsMADS14, Hd3a SDs                           R3 primer (5#-AAATGGCAAACCACTCGAAC-3#), LB2R primer T-DNA <br />                                                                          (5#-ACGTCCGCAATGTGTTATTAAG-3#). The Gus2R LB2R primers were <br /> LDs. However, absolute amounts flower- <br />                                                                          used obtain T-DNA flanking sequences inverse PCR (An et al., 2003; <br /> ing activators quite low LDs, probably                   Jeong et al., 2006). <br /> not sufficient inducing flowering signal. <br />    In contrast, levels Hd1 OsGI notably <br />                                                                          Vector Construction Transformation <br /> changed OsMADS51 mutants overexpressing <br /> plants, implying OsMADS51 acts                           The full-length cDNA clone OsMADS51 (GenBank accession no. <br /> downstream genes independently. To dis-                      AB003327; Shinozuka et al., 1999) amplified PCR using forward <br />                                                                          primer (5#-aagcttggggagctagggtttttcg-3#) reverse primer (5#-ggtaccaaact- <br /> sect possibilities, applied antisense technol-                  tatgcacaggacacc-3#). These primers contained restriction enzyme <br /> ogy evaluate transgenic plants suppress OsGI.                    sites, HindIII Asp-718 (underlined primer sequences), subsequent <br /> Not level Hd3a, known downstream                           cloning. The PCR product cloned pBluescript SK2 (Stratagene). <br />  <br /> 1492                                                                                                                Plant Physiol. Vol. 145, 2007 <br />                                                                                                 OsMADS51, Short-Day Flowering Promoter Rice <br />  <br />  <br /> Afterward, cDNA subcloned pGA1611 binary vector (Lee et al.,         ACKNOWLEDGMENTS <br /> 1999; Kim et al., 2003) maize (Zea mays) Ubi promoter the <br /> nopaline synthase (nos) terminator generate UbiTOsMADS51 construct.           We express thanks Ko Shimamoto providing pANDA <br /> To create Ehd1-RNAi construct, PCR amplified 346-bp Ehd1 cDNA          (p2K-1) binary vector, Chung-Hwan Ryu Dong-Hoon Jung valu- <br /> fragment (724&#226;&#8364;&#8220;1,069 bp translation initiation site Ehd1 cDNA      able discussions. We acknowledge Priscilla Licht critical reading of <br /> clone; GenBank accession no.AB092506), using forward primer (5#-gtt-            manuscript. <br /> gccagtcatctgcag-3#) reverse primer (5#-ggatgtggatcatgagacat-3#). The <br />                                                                                   Received June 4, 2007; accepted October 9, 2007; published October 19, 2007. <br /> PCR product cloned pGA2960, Gateway vector modified from <br /> pENTR 1A (Invitrogen). Afterward, product subcloned the <br /> pANDA (p2K-1) binary vector clonase reaction (Miki Shimamoto, <br /> 2004). Our OsGI antisense construct generated inserting partial <br /> cDNA clone binary vector pGA1611 antisense orientation. The <br />                                                                                   LITERATURE CITED <br /> cDNA clone OsGI (GenBank accession no. AJ133787mRNA) amplified             Abe M, Kobayashi Y, Yamamoto S, Daimon Y, Yamaguchi A, Ikeda Y, <br /> by PCR using forward primer (5#-agacagcaaatttgactgcggcag-3#)                 Ichinoki H, Notaguchi M, Goto K, Araki T (2005) FD, bZIP protein <br /> reverse primer (5#-gacatgctgagtgcacggacatgcg-3#). The PCR product          mediating signals floral pathway integrator FT shoot <br /> cloned pGEM-T vector (Promega). This OsGI cDNA product               apex. Science 309: 1052&#226;&#8364;&#8220;1056 <br /> digested restriction enzymes XhoI (991 bp start codon) XbaI         Alvarez-Buylla ER, Pelaz S, Liljegren SJ, Gold SE, Burgeff C, Ditta GS, <br /> (2,374 bp start codon). The digested product cloned pBluescript        Ribas Pouplana L, Martinez-Castilla L, Yanofsky MF (2000) An <br /> SK2 subcloned pGA1611 binary vector using KpnI SacI. Rice           ancestral MADS-box gene duplication occurred divergence of <br /> transformation performed according Agrobacterium-mediated coculti-            plants animals. Proc Natl Acad Sci USA 97: 5328&#226;&#8364;&#8220;5333 <br /> vation methods (Lee et al., 1999). All transgenic rice plants grown   An G, Jeong DH, Jung KH, Lee S (2005a) Reverse genetic approaches for <br /> greenhouse transferred confined paddy field.                           functional genomics rice. Plant Mol Biol 59: 111&#226;&#8364;&#8220;123 <br />                                                                                   An G, Lee S, Kim SH, Kim SR (2005b) Molecular genetics using T-DNA in <br />                                                                                      rice. Plant Cell Physiol 46: 14&#226;&#8364;&#8220;22 <br /> RNA Isolation Semiquantitative RT-PCR                                         An G, Watson BD, Stachel S, Gordon MP, Nester EW (1985) New cloning <br />                                                                                      vehicles transformation higher plants. EMBO J 4: 277&#226;&#8364;&#8220;284 <br />     Total RNA isolated Tri Reagent (MRC). First-strand cDNA          An S, Park S, Jeong DH, Lee DY, Kang HG, Yu JH, Hur J, Kim SR, Kim <br /> synthesized 2 mg total RNA, using Moloney murine leukemia virus              YH, Lee M, et al (2003) Generation analysis end sequence <br /> reverse transcriptase (Promega), 10 ng oligo(dT)15 primer            database T-DNA tagging lines rice. Plant Physiol 133: 2040&#226;&#8364;&#8220;2047 <br /> 2.5 mM deoxyriboneucleotide triphosphate, described Han et al. (2006).      Araki T (2001) Transition vegetative reproductive phase. Curr Opin <br /> Actin1 used normalization cDNA quantity. Primers included             Plant Biol 4: 63&#226;&#8364;&#8220;68 <br /> 5#-GTTTGCTCTGCTCCTACTC-3# 5#-ACTCCTCCTCCAGCATTGAA-3#                          Ciaffi M, Paolacci AR, D&#226;&#8364;&#8482;Aloisio E, Tanzarella OA, Porceddu E (2005) <br /> for OsMADS51; 5#-TGGAGAAAGGTTGTGGATGC-3# 5#-CATAGACGG-                           Identification characterization gene sequences expressed in <br /> CACTTCAGCAGAT-3# OsGI; 5#-GTTGCCAGTCATCTGCAGAA-3#                            wheat spikelets heading stage. Gene 346: 221&#226;&#8364;&#8220;230 <br /> 5#-GGATGTGGATCATGAGACAT-3# Ehd1; 5#-TTCTCCTCTCCAAA-                           Doi K, Izawa T, Fuse T, Yamanouchi U, Kubo T, Shimatani Z, Yano M, <br /> GATTCC-3# 5#-CATACGCCTTTCTTGTTTCA-3# Hd1; 5#-TCCTATG-                        Yoshimura A (2004) Ehd1, B-type response regulator rice, confers <br /> CAGAAAAGGTCCTT-3# 5#-GGACGAAGCCAAAATATACAC-3#                                short-day promotion flowering controls FT-like gene expression <br /> OsMADS14; 5#-ATGGCCGGAAGTGGCAGGGAC-3# 5#-ATCGATCGG-                              independently Hd1. Genes Dev 18: 926&#226;&#8364;&#8220;936 <br /> GATCATCGTTAG-3# Hd3a; 5#-GTATCCATGAGACTACATACATA-                         Fowler S, Lee K, Onouchi H, Samach A, Richardson K, Morris B, <br /> CAACT-3# 5#-TACTCAGCCTTGGCAATCCACA-3# Actin1. The                            Coupland G, Putterill J (1999) GIGANTEA: circadian clock-controlled <br /> reactions included initial 5 min denaturation 95&#194;&#176;C, followed             gene regulates photoperiodic flowering Arabidopsis encodes <br /> 95&#194;&#176;C 15 s, 55&#194;&#176;C 30 s, 72&#194;&#176;C 40 s. PCR cycles numbered 28      protein possible membrane-spanning domains. EMBO J <br /> flowering-regulator genes 25 Actin1.                                         18: 4679&#226;&#8364;&#8220;4688 <br />                                                                                   Han MJ, Jung KH, Yi G, Lee DY, An G (2006) Rice Immature Pollen 1 (RIP1) a <br />                                                                                      regulator late pollen development. Plant Cell Physiol 47: 1457&#226;&#8364;&#8220;1472 <br />                                                                                   Hayama R, Yokoi S, Tamaki S, Yano M, Shimamoto K (2003) Adaptation of <br /> Quantitative Real-Time RT-PCR <br />                                                                                      photoperiodic control pathways produces short-day flowering rice. <br />     Real-time PCR performed using Roche LightCycler II previously           Nature 422: 719&#226;&#8364;&#8220;722 <br /> described (Han et al., 2006). cDNA prepared leaf blades plants    Hiei Y, Komari T, Kubo T (1997) Transformation rice mediated by <br /> at 35 55 DAG. The Ubi mRNA level used normalize expression             Agrobacterium tumefaciens. Plant Mol Biol 35: 205&#226;&#8364;&#8220;218 <br /> ratio gene. Primers included 5#-AACCAAGATCGGCAGTATGG-3#              Huang T, Bohlenius H, Eriksson S, Parcy F, Nilsson O (2005) The mRNA <br /> 5#-GATTGATTGCTCCAGCAGGT-3# Hd1; 5#-GCCAACTAATGCTCGA-                             Arabidopsis gene FT moves leaf shoot apex induces <br /> GTCC-3# 5#-CAGCTGCTGCAGGGTAGTTA-3# OsMADS14; 5#-AGC-                         flowering. Science 309: 1694&#226;&#8364;&#8220;1696 <br /> CCAAGTGACCCTAACCT-3# 5#-GTTGTAGAGCTCGGCGAAGT-3#                           Izawa T, Oikawa T, Sugiyama N, Tanisaka T, Yano M, Shimamoto K <br /> Hd3a; 5#-CACGGTTCAACAACATCCAG-3# 5#-TGAAGACCCTGA-                            (2002) Phytochrome mediates external light signal repress FT <br /> CTGGGAAG-3# Ubi. Changes gene expression calculated              orthologs photoperiodic flowering rice. Genes Dev 16: 2006&#226;&#8364;&#8220;2020 <br /> DDCt method.                                                                      Izawa T, Takahashi Y, Yano M (2003) Comparative biology comes into <br />                                                                                      bloom: genomic genetic comparison flowering pathways rice <br />     Sequence data article GenBank/EMBL data            Arabidopsis. Curr Opin Plant Biol 6: 113&#226;&#8364;&#8220;120 <br /> libraries accession numbers AB003327, AB092506, AJ133787.               Jeon J, An G (2001) Gene tagging rice: high throughput for <br />                                                                                      functional genomics. Plant Sci 161: 211&#226;&#8364;&#8220;219 <br />                                                                                   Jeon JS, Lee S, Jung KH, Jun SH, Jeong DH, Lee J, Kim C, Jang S, Yang K, <br />                                                                                      Nam J, et al (2000) T-DNA insertional mutagenesis functional <br /> Supplemental Data                                                                    genomics rice. Plant J 22: 561&#226;&#8364;&#8220;570 <br />                                                                                   Jeon JS, Lee S, Jung KH, Yang WS, Yi GH, Oh BG, An GH (2000) Production of <br />    The following materials available online version article. <br />                                                                                      transgenic rice plants showing reduced heading date plant height by <br />    Supplemental Figure S1. Expression profiles various flowering reg-            ectopic expression rice MADS-box genes. Mol Breed 6: 581&#226;&#8364;&#8220;592 <br />      ulators 55-DAG osmads51-2 plants.                                         Jeong DH, An S, Kang HG, Moon S, Han JJ, Park S, Lee HS, An K, An G <br />                                                                                      (2002) T-DNA insertional mutagenesis activation tagging rice. <br />    Supplemental Figure S2. Expression profiles various flowering reg- <br />                                                                                      Plant Physiol 130: 1636&#226;&#8364;&#8220;1644 <br />      ulators 35-DAG UbiTosmads51 plants line 4. <br />                                                                                   Jeong DH, An S, Park S, Kang HG, Park GG, Kim SR, Sim J, Kim YO, Kim <br />    Supplemental Figure S3. Analyses relationship OsMADS51                 MK, Kim SR, et al (2006) Generation flanking sequence-tag <br />      OsMADS50.                                                                   database activation-tagging lines japonica rice. Plant J 45: 123&#226;&#8364;&#8220;132 <br />  <br /> Plant Physiol. Vol. 145, 2007                                                                                                                             1493 <br />  Kim et al. <br />  <br />  <br /> Kardailsky I, Shukla VK, Ahn JH, Dagenais N, Christensen SK, Nguyen          Park DH, Somers DE, Kim YS, Choy YH, Lim HK, Soh MS, Kim HJ, Kay <br />    JT, Chory J, Harrison MJ, Weigel D (1999) Activation tagging          SA, Nam HG (1999) Control circadian rhythms photoperiodic <br />    floral inducer FT. Science 286: 1962&#226;&#8364;&#8220;1965                                    flowering Arabidopsis GIGANTEA gene. Science 285: 1579&#226;&#8364;&#8220;1582 <br /> Kim SR, Lee S, Kang HG, Jeon JS, Kim KM, An G (2003) A complete              Putterill J, Robson F, Lee K, Simon R, Coupland G (1995) The CONSTANS <br />    sequence pGA1611 binary vector. J PIant Biol 16: 211&#226;&#8364;&#8220;214              gene Arabidopsis promotes flowering encodes protein showing <br /> Kobayashi Y, Kaya H, Goto K, Iwabuchi M, Araki T (1999) A pair               similarities zinc finger transcription factors. Cell 80: 847&#226;&#8364;&#8220;857 <br />    related genes antagonistic roles mediating flowering signals.     Ryu CH, You JH, Kang HG, Hur J, Kim YH, Ha <br /> </body></html>