<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>BMC Systems Biology                                                                                                                      BioMed Central <br />  <br />  <br />  <br /> Research article                                                                                                                       Open Access <br /> The effects of protein interactions, gene essentiality and regulatory <br /> regions on expression variation <br /> Linqi Zhou, Xiaotu Ma and Fengzhu Sun* <br />  <br /> Address: Molecular and Computational Biology Program, Department of Biological Sciences, University of Southern California, Los Angeles, CA <br /> 90089-2913, USA <br /> Email: Linqi Zhou - linqizho@usc.edu; Xiaotu Ma - xma@usc.edu; Fengzhu Sun* - fsun@usc.edu <br /> * Corresponding author <br />  <br />  <br />  <br />  <br /> Published: 26 June 2008                                                       Received: 5 February 2008 <br />                                                                               Accepted: 26 June 2008 <br /> BMC Systems Biology 2008, 2:54   doi:10.1186/1752-0509-2-54 <br /> This article is available <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span>: http://www.biomedcentral.com/1752-0509/2/54 <br /> &#194;&#169; 2008 Zhou et al; licensee BioMed Central Ltd. <br /> This is an Open Access article distributed under the terms of the Creative Commons Attribution License (http://creativecommons.org/licenses/by/2.0), <br /> which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited. <br />  <br />  <br />  <br />  <br />                  <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />                  Background: Identifying factors affecting gene expression variation is a challenging problem in <br />                  genetics. Previous studies have shown that the presence of TATA box, the number of cis-regulatory <br />                  elements, gene essentiality, and protein interactions significantly affect gene expression variation. <br />                  Nonetheless, the need <span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span> obtain a more complete understanding of such factors and how their <br />                  interactions influence gene expression variation remains a challenge. The growth rates of yeast cells <br />                  under several DNA-damaging conditions have been studied and a gene&apos;s toxicity degree is defined <br />                  as the number of such conditions that the growth rate of the yeast deletion strain is significantly <br />                  affected. Since toxicity degree reflects a gene&apos;s importance to cell <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span>survival</span> under DNA-damaging <br />                  conditions, we expect that it is negatively associated with gene expression variation. Mutations in <br />                  both cis-regulatory elements and transcription factors (TF) regulating a gene affect the gene&apos;s <br />                  expression and thus we study the relationship between gene expression variation and the number <br />                  of TFs regulating a gene. Most importantly we study how these factors interact with each other <br />                  influencing gene expression variation. <br />                  Results: Using yeast as a model system, we evaluated the effects of four separate factors and their <br />                  interactions on gene expression variation: protein interaction degree, toxicity degree, number of <br />                  TFs, and the presence of TATA box. Results showed that 1) gene expression variation is negatively <br />                  correlated with the protein interaction degree in the protein interaction network, 2) essential <br />                  genes tend to have less expression variation than non-essential genes and gene expression variation <br />                  decreases with toxicity degree, and 3) the number of TFs regulating a gene is the most important <br />                  factor influencing gene expression variation (R2 = 8&#226;&#8364;&#8220;14%). In addition, the number of TFs regulating <br />                  a gene was found to be an important factor influencing gene expression variation for both TATA- <br />                  containing and non-TATA-containing genes, but with different association strength. Moreover, <br />                  gene expression variation was significantly negatively correlated with toxicity degree only for <br />                  TATA-containing genes. <br />                  Conclusion: The finding that distinct mechanisms may influence gene expression variation in <br />                  TATA-containing and non-TATA-containing genes, provides new insights into the mechanisms that <br />                  underlie the evolution of gene expression. <br />  <br />  <br />  <br />  <br />                                                                                                                                        Page 1 of 12 <br />                                                                                                                  (page number not for citation purposes) <br />  BMC Systems Biology 2008, 2:54                                                http://www.biomedcentral.com/1752-0509/2/54 <br />  <br />  <br />  <br />  <br /> Background                                                      studied the effect of protein-protein interactions and pro- <br /> Gene expression variation has been studied on three dif-        tein <span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span> on evolutionary variation (variation among <br /> ferent levels: single cells across a common environment         strains in a species). They found that evolutionary varia- <br /> [1], within one species across a variety of different envi-     tion is negatively correlated with protein-protein interac- <br /> ronments [2,3], and across different species/strains, which     tions in Saccharomyces cerevisiae or Drosophila melanogaster <br /> is often referred to as evolutionary variation [4-8]. In this   [4] and negatively correlated with protein length in Dro- <br /> paper, we study genetic factors affecting gene expression       sophila melanogaster [5]. These studies highlighted the <br /> variation within one species across many different envi-        importance of protein interactions and gene regulatory <br /> ronmental conditions. Broadly, the genetic factors affect-      regions on gene expression variation. <br /> ing gene expression primarily include the binding of <br /> regulatory proteins to cis-elements in the upstream of the      Only a few studies, however, have integrated such differ- <br /> gene, as well as physical and genetic interactions with         ent data sources in a way that collectively identifies and <br /> other genes. With the availability of many gene expression      interprets the key factors affecting gene expression varia- <br /> profiles, protein interaction networks, and gene regula-        tion. Therefore, we conducted studies of proteomic and <br /> tory networks, it is now possible to study how gene             genomic factors marginally and collectively influencing <br /> expression variation is associated with both network fea-       gene expression variation across different perturbation <br /> tures and genomic factors. In the case of protein interac-      conditions within one species: yeast. <br /> tion networks, interaction degree, i.e., the number of <br /> interacting partners of a given protein, is one of many fac-    Protein interactions play an important role in gene expres- <br /> tors. The presence or absence of TATA box and the               sion variation. Protein-protein interactions are key bio- <br /> number of transcription factors (TF) regulating a gene          logical events in a living cell, and proteins in a cell interact <br /> provide examples of genomic factors influencing gene            with each other to perform certain functions. High <br /> expression variation.                                           throughput technologies, including yeast two-hybrid sys- <br />                                                                 tems and mass spectrometry, have generated a large <br /> Many studies have focused on individual factors affecting       amount of protein interactions in yeast. Computational <br /> gene expression variation. For instance, Newman et al. [1]      methods have also been developed to study the reliability <br /> developed an experimental technique to study protein            of the observed interactions [10,11] and to build reliable <br /> expression noise in single cells and showed that chromo-        protein interaction networks. These efforts have resulted <br /> somal distance to other genes and mRNA-half life are            in the development of several protein interaction data- <br /> associated with expression noise. However, they did not         bases, albeit with differing degrees of reliability, including <br /> find a relationship between protein expression noise and        MIPS [12], DIP [13] and BioGrid [14]. From an evolution- <br /> protein-protein interactions. Recently, using a more com-       ary point of view, the expression profiles of neighboring <br /> plete interaction dataset, Batada et al. [9] found that pro-    genes of a target gene in a protein interaction network <br /> tein expression variation is negatively correlated with         may put some constraints on the target gene&apos;s expression. <br /> interaction degree when protein abundance was control-          Thus, in a protein interaction network, the interacting <br /> led using the data in Newman et al. [1]. This relationship      partners of a specific protein can affect the corresponding <br /> continues to hold within the viable genes. Several groups       gene&apos;s expression. Therefore, protein physical interaction <br /> investigated mRNA expression variation within species.          degree, i.e., the number of interacting partners of a given <br /> For example, Nelson et al. [2] studied the relationship         protein, can significantly affect gene expression variation. <br /> between the number of tissues or body parts (expression         In the present study, we show that gene expression varia- <br /> variation), where the gene is expressed, and gene spacing       tion decreases with protein interaction degree and that <br /> in C. elegans and D. melanogaster. They found that gene         protein interaction degree accounts for 1&#226;&#8364;&#8220;2% of the <br /> expression variation increases in relation to the intergenic    expression variation in model organism yeast, a result <br /> distances between genes. Walther et al. [3] found a posi-       consistent with previous studies [4,5]. <br /> tive correlation between the frequency of a gene&apos;s differen- <br /> tial expression and the number of cis-regulatory elements       Another key factor affecting gene expression variation is <br /> of that gene in A. thaliana. Furthermore, several groups        gene essentiality. Genes can be classified into essential <br /> have studied gene expression variation, also known as           and non-essential genes based on the fitness <span id='am-162' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-163' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span>phenotype</span> of <br /> evolutionary variation, across different species/strains.       the yeast cell when the gene is deleted under normal <br /> Using gene expression data from several yeast species [7],      growth conditions [15]. Essential genes are those that, <br /> as well as from different strains derived from <span id='am-240' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-241' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-242' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-249' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-258' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-264' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-273' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-277' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-296' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-297' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-307' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-317' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span>mutation</span>-        when deleted, will render the yeast cell non-viable. Non- <br /> accumulation experiments [6], it was found, for instance,       essential genes can be further classified into no-phenotype <br /> that the interspecies/interstrain variation of gene expres-     and toxicity-modulating genes based on the fitness phe- <br /> sion is significantly correlated with the presence/absence      notype of yeast cell when the gene is deleted under the <br /> of the TATA box in the promoter region. Lemos et al. [4,5]      conditions of four DNA-damaging treatments [16]. Spe- <br />  <br />  <br />                                                                                                                      Page 2 of 12 <br />                                                                                                (page number not for citation purposes) <br />  BMC Systems Biology 2008, 2:54                                                  http://www.biomedcentral.com/1752-0509/2/54 <br />  <br />  <br />  <br /> cifically, we define a gene&apos;s toxicity modulation degree as        influencing TATA-containing gene expression variation is <br /> the number of DNA-damaging treatments significantly                much more complicated than that in non-TATA-contain- <br /> affecting the deletion strain&apos;s fitness (toxicity modulation       ing genes. For example, TATA-containing genes were <br /> degree = 0 (no phenotype), 1, 2, 3, and 4). The higher the         found more likely to be epigenetic regulated [17,18]. <br /> toxicity modulation degree, the more important the gene            Thus, this study gives a more complete analysis of factors <br /> is in relation to <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span>cell survival</span>. Therefore, toxicity degree        and their interaction affecting gene expression variation <br /> gives a quantitative measurement of a gene&apos;s importance            than may be found in previous studies. <br /> to yeast cell survival. We measure a gene&apos;s functional <br /> importance in relation to cell survival by the essentiality        Results and Discussion <br /> of the essential genes and the toxicity modulation degree          We present our results based on the MIPS protein physical <br /> of non-essential genes. Since the expression of genes              interaction data [12] and the yeast gene expression pro- <br /> important for cell survival are generally stable under many        files under 40 Ca and Na exposure conditions [19]. The <br /> different stimuli and cannot fluctuate extensively, we             results based on three interaction datasets (MIPS [12], DIP <br /> hypothesize and show that expression variation of essen-           [13], and BioGrid [14]) and four other gene expression <br /> tial genes is lower than that of non-essential genes and           datasets (chemostat (nutritional stress) [20], environmen- <br /> decreases with toxicity degree within non-essential genes.         tal stress [21], oxidative stress [22], and a combined gene <br />                                                                    expression dataset over more than 1,500 conditions [7]) <br /> The number of cis-elements has been shown to be posi-              are given in the Additional Files 1 and 2. We study the <br /> tively associated with gene expression variation [6]. The          expression data individually in order to minimize the var- <br /> number of cis-elements is usually approximated using               iation among different laboratories. By doing so, we can <br /> computational approaches and many contain false posi-              also confirm whether the results based on different gene <br /> tive and negative predictions. Theoretically, a given gene&apos;s       expression data are consistent. Consistency of results <br /> expression <span id='am-318' about='xsp:pattern' typeof='owl:Thing'>pattern</span> can become increasingly complex with            using a variety of different datasets adds confidence to the <br /> the increasing number of transcription factors that regu-          conclusions. In this manuscript, we use genes and pro- <br /> late this gene, either directly or indirectly. In this study, we   teins interchangeably. We declare statistical significance if <br /> hypothesize that the number of TFs is a significant predic-        a p-value is less than 0.05 without adjusting for multiple <br /> tor of expression variation and show that the number of            comparisons. In this study, we conducted an exploratory <br /> TFs regulating a gene (hereinafter referred to as &apos;number of       study of factors affecting gene expression variation. As in <br /> TFs&apos;) accounts for 8&#226;&#8364;&#8220;14% of its expression variation,              many epidemiological studies, we did not adjust p-values <br /> much higher than that can be explained by the number of            for multiple comparisons. Therefore, some of our findings <br /> cis-elements (0.3&#226;&#8364;&#8220;1.7%). This implies the importance of            need to be further tested in other datasets. <br /> indirect trans-effect on expression variation. <br />                                                                    Gene expression variation versus protein interaction <br /> The TATA box is a conserved element in the eukaryotic              degree <br /> promoter region and is usually bound by TATA-binding               We measured gene expression variation by the logarithm <br /> proteins. The presence of TATA box has been shown to be            of the variance of the expression levels of each gene across <br /> one of the most important factors contributing to gene             the 40 Ca and Na exposure conditions [19]. We then stud- <br /> expression variation [6,7]. Further analysis of the individ-       ied the relationship between the expression variation and <br /> ual genomic and proteomic factors affecting gene expres-           protein interaction degree using the LOWESS function in <br /> sion indicates that there might be two distinct                    R [23] to fit the data. On these bases, it was obvious that <br /> mechanisms that specifically influence gene expression             gene expression variation decreases with the degree of <br /> variation of TATA-containing and non-TATA-containing               protein physical interaction (Figure 1A). The decreasing <br /> genes. Most importantly, we show that significant nega-            trend is especially significant when the interaction degree <br /> tive correlation between expression variation and toxicity         is relatively low (&#226;&#8240;&#164; 20). In contrast, when the interaction <br /> degree is only present for TATA-containing genes and that          degree is greater than 40, the decreasing trend is not as <br /> toxicity degree accounts for 1.3&#226;&#8364;&#8220;2.6% of the expression            obvious. Biologically, this may be explained by the fact <br /> variation. In contrast, the relationship between expression        that gene expression variation stabilizes when the interac- <br /> variation and toxicity degree is absent for non-TATA-con-          tion degree is above a given threshold. Another potential <br /> taining genes. The fact that TATA-containing genes are             explanation is that interactions between proteins of high <br /> enriched in stress-related genes [17] may explain this dif-        degrees are simply less reliable [24]. The large number of <br /> ference. Although the number of TFs is significantly posi-         less reliable interactions for proteins with high degrees <br /> tively correlated with expression variation for both TATA-         can skew the true relationship between gene expression <br /> and non-TATA-containing genes, the association strength            variation and interaction degrees. Since the true underly- <br /> is higher for non-TATA containing genes than for TATA-             ing mechanism of this phenomenon is not clear, we lim- <br /> containing genes. These results imply that the mechanism           ited our further analysis to proteins with an interaction <br />  <br />  <br />                                                                                                                        Page 3 of 12 <br />                                                                                                  (page number not for citation purposes) <br />  BMC Systems Biology 2008, 2:54                                                    http://www.biomedcentral.com/1752-0509/2/54 <br />  <br />  <br />  <br />  <br /> Figure <br /> Gene expression <br />          1         variation is negatively correlated with protein interaction degree <br /> Gene expression variation is negatively correlated with protein interaction degree. The x-axis represents protein <br /> physical interaction degree, and the y-axis represents gene expression variation. A) The LOWESS fit to the gene expression <br /> variation. B) Bar-plot of the expression variation of all the genes with a given protein interaction degree together with the lin- <br /> ear regression fit to the gene expression variation in relation to the interaction degree. The linear coefficient &#206;&#178; = -0.0302, R2 = <br /> 1.41%, and p-value = 9.704e-14. The red dots are the mean expression variation of the genes given the protein physical inter- <br /> action (PPI) degree. The bar represents the standard deviation of the gene expression variation given PPI degree. To keep the <br /> same scale for gene expression variation across the figures, the range of the y-axis is -2.5 to 0.5. <br />  <br />  <br />  <br />  <br /> degree of no more than 20. Protein physical interaction             Gene expression variation versus essentiality, toxicity <br /> degree has been found to be negatively associated with              modulation, and interaction degrees <br /> gene expression variation for single cells [9] and evolu-           As noted above, we divided genes into two classes: essen- <br /> tionary expression variation [4,5] between different                tial and non-essential genes. Essential genes are less likely <br /> strains/species. Our result on the relationship between             to be perturbed than non-essential genes, as significant <br /> gene expression variation and interacting degree within S.          perturbations of essential genes will, for example, render <br /> cerevisiae is consistent with their findings.                       the yeast cell non-viable. We further classified the non- <br />                                                                     essential genes into five groups (no phenotype, 0, and tox- <br /> Accordingly, we then used linear regression to fit the              icity-modulating proteins with degrees 1, 2, 3, and 4, <br /> expression variation for proteins with a maximal physical           respectively) according to the cell&apos;s fitness phenotype <br /> interaction degree of 20:                                           changes under four DNA-damaging agents (methylating <br />                                                                     agent methyl methanesulfonate (MMS), the bulky alkylat- <br />                           v = &#206;&#177; + &#206;&#178;d                                ing agent 4-nitroquinoline-N-oxide (4NQO), the oxidiz- <br />                                                                     ing agent tert-butyl hydroperoxide (t-BuOOH), and 254- <br /> where v is the gene expression variation and d is the inter-        nm UV radiation) when the non-essential genes are <br /> action degree. &#206;&#177; and &#206;&#178; are parameters. The fitted line and          knocked out [16]. Since toxicity degree reflects the func- <br /> the corresponding bar-plot for the expression variation             tional importance of genes in relation to cell survival <br /> are shown in Figure 1B. The gene expression variation is            under several DNA-damaging perturbations, we expected <br /> significantly negatively correlated with the protein inter-         that gene expression variation would decrease as the tox- <br /> action degree (&#226;&#8240;&#164; 20) (R2 = 1.41%, &#206;&#178; = -0.0302, p-value =            icity degree increases. Since the number of genes with tox- <br /> 9.704e-14). The negative correlation between expression             icity degree 4 was small (n = 32), we combined them with <br /> variation and interaction degree implies that protein with          the group having toxicity degree 3. We referred to the <br /> high interaction degrees do not tolerate extensive expres-          essential genes as the group with toxicity degree 4. Figure <br /> sion variation and such protein need more precise control           2A shows the bar-plot and the linear regression fit of the <br /> on gene expression for an organism to function normally.            gene expression variation with respect to toxicity degree. <br />  <br />  <br />                                                                                                                          Page 4 of 12 <br />                                                                                                    (page number not for citation purposes) <br />  BMC Systems Biology 2008, 2:54                                                   http://www.biomedcentral.com/1752-0509/2/54 <br />  <br />  <br />  <br />  <br /> Figure <br /> The  effect <br />          2 of essentiality, toxicity degree, and protein interaction degree on gene expression variation <br /> The effect of essentiality, toxicity degree, and protein interaction degree on gene expression variation. A) Bar- <br /> plot of the expression variation of all the genes with a given toxicity degree together with the linear regression fit to the <br /> expression variation of the genes in relation to the toxicity degree. The linear coefficient &#206;&#178; = -0.0629, R2 = 0.75%, and the p- <br /> value = 4.73e-08. B) The mean expression variation and the linear regression fit to the expression variation with respect to PPI <br /> degree for non-essential genes stratified according to toxicity degree and for the essential genes. The &#206;&#178; values are -0.0172, - <br /> 0.0230, -0.0304, -0.0164 and -0.0460 for toxicity degree 0, 1, 2, and 3, and the essential genes, respectively. The corresponding <br /> p-values are 0.0313, 0.0141, 0.0011, 0.2248 and 0.0013, respectively. R2 is 0.31%, 0.75%, 2.75%, 0.8% and 4.41%, respectively. <br /> The labels are the same as those in Figure 1. <br />  <br />  <br /> Indeed, a significant negative association between gene            is, conversely, caused by the positive correlation between <br /> expression variation and toxicity degree was observed (R2          interaction degree and toxicity degree. We consequently <br /> = 0.75%, &#206;&#178; = -0.0629, p-value = 4.73e-08). In the study of         studied the relationship between gene expression varia- <br /> the relationship between gene essentiality and evolution-          tion and protein interaction degree within gene groups <br /> ary expression variation conducted by Tirosh and Barkai            stratified according to their toxicity degrees (Figure 2B). <br /> [8] and Choi et al. [25], they found that essential genes          Using Ca and Na exposure gene expression data [19], we <br /> tend to have lower variation than non-essential genes.             found a significant decreasing trend of gene expression <br /> This is consistent with our result which demonstrates that         variation with respect to interaction degree in all the strata <br /> the variation in essential genes (toxicity degree = 4) is          except for the one with toxicity degree 3. The correspond- <br /> lower than that of non-essential genes (combine the pro-           ing (R2, &#206;&#178;, p-value) are (0.31%, -0.017, 0.03), (0.75%, - <br /> teins with toxicity degree = 0, 1, 2, 3) (p-value = 0.027).        0.023, 0.014), (2.75%, -0.030, 0.001), (0.8%, -0.016, <br /> However, our study differs from these two studies in that          0.2248), and (4.41%, -0.046, 0.001) for toxicity degrees 0, <br /> we further classified the genes according to their toxicity        1, 2, 3 and 4, respectively. The fraction of expression vari- <br /> degree and found negative association between gene                 ation explained by the protein interaction degree seems to <br /> expression and toxicity degree.                                    increase as the toxicity degree increases. <br />  <br /> We observed a positive correlation between interaction             In our analyses, both toxicity and protein interaction <br /> degree and toxicity degree (data not shown) and, there-            degrees are negatively associated with gene expression var- <br /> fore, asked whether the observed negative correlation              iation. Hence, the more important a gene is to the survival <br /> between gene expression variation and interaction degree           of the yeast cell, the less variation there is in its expression <br />  <br />  <br />                                                                                                                         Page 5 of 12 <br />                                                                                                   (page number not for citation purposes) <br />  BMC Systems Biology 2008, 2:54                                                 http://www.biomedcentral.com/1752-0509/2/54 <br />  <br />  <br />  <br /> levels across many different conditions. Similarly, the          variation and toxicity degree was observed for the TATA- <br /> higher the interaction degree of a gene, the more stability      containing group (Figure 3A) (R2 = 2.59%, &#206;&#178; = -0.1674, p- <br /> is observed in its expression levels. Biologically, a gene is    value = 7.174e-06). However, the association between <br /> important to the cell&apos;s survival since it participates in        expression variation and toxicity degree for the non-TATA- <br /> many important biological processes. Any perturbation of         containing group is only marginally significant (R2 = <br /> this gene&apos;s expression will likely cause deleterious effect to   0.13%, &#206;&#178; = -0.02338, p-value = 0.0413). Consistent with <br /> the corresponding biological process and thus renders the        the other two gene expression datasets, chemostat [20] <br /> cell non-viable. An evolutionary consequence of this             and environmental stress [21], as well as the combined <br /> hypothesis is that genes important to cell survival appear       gene expression data from [7] (see Additional File 2: Sup- <br /> to have robust expression levels.                                plementary Table 7), such a highly significant negative <br />                                                                  relationship between gene expression variation and toxic- <br /> Expression variation versus gene regulatory regions: TATA        ity degree could be observed in TATA-containing genes, <br /> box, number of TFs, and toxicity degree                          but not for the non-TATA-containing genes. Thus, the <br /> Previous studies have established the relationship               effect of toxicity degree on gene expression variation is dif- <br /> between gene expression variation and the regulatory             ferent for TATA-containing genes versus non-TATA-con- <br /> regions, including the presence/absence of TATA box              taining genes. The relative small R2 value between gene <br /> [6,7], the length of intergenic regions [2], and the number      expression variation and toxicity degree alone may be <br /> of cis-regulatory elements [3]. We therefore asked if the        explained by the absence of association between them <br /> observed relationship between gene expression variation          within the non-TATA-containing genes. <br /> and toxicity degree are the same for TATA-containing <br /> genes and non-TATA-containing genes. To accomplish               Previous studies identified the number of cis-element <br /> this goal, we first stratified the yeast genes based on the      motifs contributing to gene expression variation in A. thal- <br /> presence/absence of TATA boxes and reanalyzed the rela-          iana [3], but Landry et al. [6] found that the number of cis- <br /> tionship between gene expression variation and toxicity          elements only marginally affects gene expression varia- <br /> degree. Consistent with previous findings [6,7], it is clear     tion in yeast. Here we studied these two factors (the <br /> that the gene expression variation of TATA-containing            number of cis-elements and the number of TFs) in relation <br /> genes is much higher than that of non-TATA-containing            to gene expression variation. We first analyzed the rela- <br /> genes for each fixed toxicity degree (p-value &lt; 2.2e-16).        tionship between gene expression variation and number <br /> Significant negative association between gene expression         of cis-elements [26] using the linear model. Similar to <br />  <br />  <br />  <br />  <br /> Figure <br /> The effect <br />         3 of TATA box, number of TFs, and toxicity degree on gene expression variation <br /> The effect of TATA box, number of TFs, and toxicity degree on gene expression variation. A) The relationship <br /> between expression variation and toxicity degree stratified by the presence/absence of the TATA box (R2 = 2.59%, &#206;&#178; = - <br /> 0.1674, p-value = 7.174e-06 for the TATA-containing gene set; R2 = 0.13%, &#206;&#178; = -0.0234, p-value = 0.0413 for the non-TATA- <br /> containing gene set). B) The relationship between expression variation and the number of TFs up to 25 (R2 = 8.28%, &#206;&#178; = <br /> 0.0654, p-value &lt; 2.2e-16). The labels are the same as those in Figure 1. <br />  <br />  <br />  <br />  <br />                                                                                                                      Page 6 of 12 <br />                                                                                                (page number not for citation purposes) <br />  BMC Systems Biology 2008, 2:54                                                                       http://www.biomedcentral.com/1752-0509/2/54 <br />  <br />  <br />  <br /> MacIsaac et al. [26], we defined cis-elements according to                          Overall analysis of factors affecting gene expression <br /> binding probability (p &lt; 0.0001) and classified them                                variation <br /> according to their conservation in two other yeast species.                         As enumerated above, we have identified several factors <br /> The results are presented in Table 1 for the different gene                         influencing gene expression variation in S. cerevisiae. In <br /> expression data. For the Ca and Na exposure data [19], the                          addition to the presence/absence of TATA box identified <br /> highest R2 is 1.33% (p-value = 4.45e-08) when cis-ele-                              in previous studies [6,7], we found that gene expression <br /> ments that are conserved in at least one species are used.                          variation decreases as both the protein interaction and <br /> We then studied the relationship between gene expression                            toxicity modulation degrees increase. These findings are <br /> variation and the number of TFs that influence target gene                          consistent with other studies for expression variation of <br /> expression based on the gene regulatory network devel-                              single cells [9] and evolutionary expression variation <br /> oped in Hu et al. [27]. A highly significant positive corre-                        across different strains/species [4,5,25]. We also found <br /> lation between gene expression variation and the number                             that gene expression variation increases as the number of <br /> of TFs was observed (R2 = 8.28%, &#206;&#178; = 0.0654, p-value &lt;                              TFs or the number of cis-elements increases and that the <br /> 2.2e-16) (Figure 3B). The fraction of variation explained                           number of TFs regulating a gene is a much better predictor <br /> by the number of TFs (R2 = 8.28%) is much higher than                               of expression variation than the number of cis-elements. <br /> that by the number of cis-elements indicating that the                              We therefore studied the contribution of each factor and <br /> number of TFs is a better predictor of gene expression var-                         their interactions on gene expression variation by taking <br /> iation than the number of cis-elements. This result implies                         the other factors into consideration using the Akaike <br /> the importance of trans-effect for gene expression varia-                           Information Criterion (AIC) [28]. We retained the model <br /> tion.                                                                               with the smallest AIC. Table 2 gives the results of factors <br /> Table 1: The relationship between gene expression variation and the number of cis-elements. <br />  <br />  Cis-elements are identified with binding p &lt; 0.0001 and conservation in at least 2 other yeast species <br />  <br />  Gene expression data-set                                                                                          Linear regression              R2 <br />  <br />                                                                                                                    &#206;&#178;              p value <br />  <br />  Ca_Na exposure                                                                                                    0.0589         1.17e-06        1.25% <br />  Chemostat                                                                                                         0.0188         0.0067          0.39% <br />  Environmental Stress                                                                                              0.0507         5.48e-07        1.33% <br />  Oxidative Stress                                                                                                  0.0025         0.725           0.01% <br />  <br />  Cis-elements are identified with binding p &lt; 0.0001 and conservation in at least 1 other yeast <br />  <br />  Gene expression data-set                                                                                          Linear regression              R2 <br />  <br />                                                                                                                    &#206;&#178;              p value <br />  <br />  Ca_Na exposure                                                                                                    0.0539         4.45e-08        1.33% <br />  Chemostat                                                                                                         0.0161         0.0046          0.36% <br />  Environmental Stress                                                                                              0.0516         3.56e-10        1.74% <br />  Oxidative Stress                                                                                                  0.0013         0.826           0 <br />  <br />  Cis-elements are identified with binding p &lt; 0.0001 and no Conservation Criteria <br />  <br />  Gene expression data-set                                                                                          Linear regression              R2 <br />  <br />                                                                                                                    &#206;&#178;              p value <br />  <br />  Ca_Na exposure                                                                                                    0.0393         1.05e-07        1.02% <br />  Chemostat                                                                                                         0.0151         0.0004          0.45% <br />  Environmental Stress                                                                                              0.0346         1.68e-08        1.15% <br />  Oxidative Stress                                                                                                  0.0028         0.526           0.01% <br />  <br />  Cis-elements are identified with three different criteria according to their conservation in two other species. &#206;&#178; is the linear coefficient in the linear <br />  model, the p-value is related to the null hypothesis that &#206;&#178; &#226;&#8240;&#160; 0 versus &#206;&#178; = 0, and the R2 is the fraction of variation explained by the number of cis- <br />  elements. <br />  <br />  <br />                                                                                                                                              Page 7 of 12 <br />                                                                                                                        (page number not for citation purposes) <br />  BMC Systems Biology 2008, 2:54                                                                     http://www.biomedcentral.com/1752-0509/2/54 <br />  <br />  <br />  <br /> Table 2: Analysis of four factors and their interactions affecting expression variation using stepwise selection with AIC. <br />  <br />   variable            Ca_Na_exposure                        Chemostat                        Environmental Stress                 Oxidative Stress <br />  <br />                model       p value        R2       model       p value       R2       model        p value      R2        model        p value         R2 <br />  <br />     x1            &#226;&#710;&#353;        0.4252       0.05%        &#226;&#710;&#353;         0.1582      0.16%         &#226;&#710;&#353;        0.0647      0.28%          &#226;&#710;&#353;         0.1075        0.21% <br />     x2            &#226;&#710;&#353;        0.3721       0.06%        &#226;&#710;&#353;         0.0635      0.28%         &#226;&#710;&#353;        0.0861      0.24%                    0.8729        0.002% <br />     x3            &#226;&#710;&#353;       1.16E-20      6.76%        &#226;&#710;&#353;        4.59E-09     2.73%         &#226;&#710;&#353;        &lt; 2e-16     13.42%         &#226;&#710;&#353;        7.30E-09        2.65% <br />     x4            &#226;&#710;&#353;       3.22E-12      3.83%        &#226;&#710;&#353;        7.96E-10     3.00%         &#226;&#710;&#353;        &lt; 2e-16      6.92%         &#226;&#710;&#353;         0.8841        0.002% <br />    x1*x2 <br />    x1*x3 <br />    x1*x4          &#226;&#710;&#353;        0.0581       0.29%        &#226;&#710;&#353;        0.0100       0.53%                                             &#226;&#710;&#353;         0.0348        0.36% <br />    x2*x3                                             &#226;&#710;&#353;        0.0422       0.33% <br />    x2*x4          &#226;&#710;&#353;        0.0226       0.41%        &#226;&#710;&#353;        0.0036       0.68%         &#226;&#710;&#353;        0.0108      0.53% <br />    x3*x4          &#226;&#710;&#353;        0.0039       0.67%        &#226;&#710;&#353;        0.0185       0.45%         &#226;&#710;&#353;        6.2e-09     2.71% <br />   R2model                  16.36%                             12.73%                               22.39%                               4.43% <br />  <br />  The four main factors include protein interaction degree (x1), toxicity degree (x2: treat essential genes as ones with toxicity degree 4), number of <br />  TFs (x3), and the presence of TATA box (x4: 1-TATA containing genes, 0-non-TATA containing genes). The protein interaction data used in this <br />  analysis is based on the MIPS dataset. The column marked with &quot;&#226;&#710;&#353;&quot; indicates inclusion in the final linear model. The multiple linear regression is <br />  based on the final linear model, respectively. The p-value is related to the null hypothesis that &#206;&#178; &#226;&#8240;&#160; 0 versus &#206;&#178; = 0. R2 is the variation explained by the <br />  model and each independent variable, respectively. <br />  <br /> included in the final linear model using the MIPS interac-                        non-TATA-containing genes is about 1.5-fold higher than <br /> tion data [12] and the four expression profiles by stepwise                       the corresponding values for the TATA-containing genes. <br /> selection with AIC. We then studied the effect of the                             Accordingly, the p-values for the non-TATA-containing <br /> selected factors on expression variation using linear                             genes are about three orders of magnitude smaller than <br /> regression, and the corresponding p-values and R2 values                          the corresponding p-values for the TATA-containing gene <br /> are given in Table 2. The results showed that protein inter-                      set. We noted that TATA-containing genes tend to have <br /> action degree only explained less than 1% of variation                            higher number of TFs than non-TATA-containing genes <br /> when adjusted for the three other main factors and two                            (p-value = 1.184e-10), but this cannot explain our obser- <br /> interaction terms. Except for the oxidative stress gene                           vation that the association strength between expression <br /> expression dataset [22], the consistently selected model                          variation and the number of TFs within the TATA-contain- <br /> contained four main factors including protein interaction                         ing genes is lower than that within the non-TATA-contain- <br /> degree, toxicity degree, the number of TFs, and TATA box,                         ing genes. One possible explanation is that the presence of <br /> and two interaction terms, i.e., interaction between TATA                         TATA-box weakens the effect of the number of TFs on gene <br /> and toxicity degree and TATA and number of TFs. The two                           expression during evolution. Second, the toxicity degree is <br /> interaction terms are found to be statistically significant                       only a significant contributor for gene expression varia- <br /> across three expression datasets: Ca and Na exposure [19],                        tion within the TATA-containing genes in the Ca-Na expo- <br /> chemostat [20], and environmental stress [21] (p-value &lt;                          sure [19] and Chemostat [20] gene expression dataset, <br /> 0.05) and explained 0.4% &#226;&#8364;&#8220; 2.7% of the variation.                                 while it is not a significant contributor for gene expression <br />                                                                                   variation in the non-TATA-containing gene set. Within the <br /> Because we found that TATA box interact with toxicity                             environmental stress gene expression dataset [21], the <br /> degree and the number of TFs influencing gene expression                          effect of toxicity degree on gene expression variation was <br /> variation, we reanalyzed the contributions of toxicity                            not statistically significant. However, we did observe a <br /> degree and number of TFs on gene expression variation                             decreasing trend of gene expression variation with respect <br /> stratified by the presence/absence of TATA box using the                          to toxicity degree within the TATA-containing genes (see <br /> linear model. The protein physical interaction degree was                         Additional File 1, Supplemental Figure 5). <br /> not included in this analysis because the interaction <br /> between TATA box and physical interaction degree was                              The toxicity degree of a gene measures the tolerance of the <br /> not detected. Table 3 shows the different results for TATA                        yeast cell to different external stress conditions when the <br /> and non-TATA gene sets. First, the number of TFs is the                           gene is knocked out. Therefore, it might be expected that <br /> most significant factor for gene expression variation in                          the different relationship between gene expression varia- <br /> both TATA- and non-TATA-containing genes with p-val-                              tion and toxicity degree for TATA-containing genes and <br /> ues less than 5e-6 for all the datasets. However, the asso-                       non-TATA-containing genes is due to the enrichment of <br /> ciation strength between the number of TFs and gene                               stress-related genes in TATA-containing genes, as found in <br /> expression variation measured by the coefficient &#206;&#178; for the                        [17]. To test this hypothesis, we used a set of genes related <br />  <br />  <br />                                                                                                                                              Page 8 of 12 <br />                                                                                                                        (page number not for citation purposes) <br />  BMC Systems Biology 2008, 2:54                                                                       http://www.biomedcentral.com/1752-0509/2/54 <br />  <br />  <br />  <br /> Table 3: The effect of two factors on expression variation stratified by the presence/absence of TATA box. <br />  <br />  TATA dataset <br />  <br />    variable             Ca_Na_exposure                        Chemostat                       Environmental Stress                 Oxidative Stress <br />  <br />                     &#206;&#178;         p-value      R2          &#206;&#178;          p-value      R2         &#206;&#178;           p-value     R2           &#206;&#178;          p-value      R2 <br />  <br />       x2         -0.1436      0.0024     1.95%      -0.0764       0.0034     1.81%     -0.0510         0.156    0.43%       0.0081         0.778      0.02% <br />       x3         0.0312       5.4e-10    7.90%       0.0168       1.5e-09    7.50%      0.0283        2.6e-13   10.79%      0.0138        4.8e-06     4.37% <br />   R2 (model)                  9.47%                               8.97%                               11.09%                               4.39% <br />  <br />  Non-TATA dataset <br />  <br />    variable             Ca_Na_exposure                        Chemostat                       Environmental Stress                 Oxidative Stress <br />  <br />                     &#206;&#178;         p-value      R2          &#206;&#178;          p-value      R2         &#206;&#178;           p-value     R2           &#206;&#178;          p-value      R2 <br />  <br />       x2         -0.0167       0.402     0.06%      -0.0026       0.8189     0.005%     0.0258         0.086     0.26%      0.0038         0.747     0.009% <br />       x3         0.0502       &lt; 2e-16    8.70%       0.0300       &lt; 2e-16     9.56%     0.0468        &lt; 2e-16    12.6%      0.0181        4.4e-10    3.40% <br />   R2 (model)                  8.85%                                9.62%                              12.67%                               3.39% <br />  <br />  The linear model that includes the toxicity degree (x2) and the number of TFs (x3) is built for each of the four gene expression datasets, <br />  respectively. R2 is the variation explained by each independent factor and the model, respectively. &#206;&#178; is the linear coefficient in the linear model, and <br />  the p-value is related to the null hypothesis that &#206;&#178; &#226;&#8240;&#160; 0 versus &#206;&#178; = 0. <br /> to Environmental Stress Response (ESR) [21]. If the                              than non-TATA-containing genes [17]. Basehoar et al. [17] <br /> hypothesis is true, we would expect a higher association                         suggested that two distinct regulatory mechanisms may be <br /> between expression variation and toxicity degree within                          present at TATA- and TATA-less promoters. The results in <br /> the ESR genes than that within the non-ESR genes. How-                           Table 3 support their findings. <br /> ever, our data shows that the &#206;&#178; values are similar for the <br /> two groups of genes (Table 4). Thus we cannot explain the                        The results based on the oxidative stress gene expression <br /> interaction between TATA and toxicity degree by the                              dataset [22] are not consistent with the results based on <br /> enrichment of ESR genes in TATA-containing genes. The                            the other three gene expression datasets. This observation <br /> biological mechanisms underlying the observed interac-                           may be due to the relatively small gene expression varia- <br /> tion are not clear and need to be further studied.                               tion in this data. For example, the range of the variance of <br />                                                                                  the expression levels within the oxidative stress dataset, <br /> We also did the same analysis for the average gene expres-                       (0.07, 5.34), is much smaller than the corresponding <br /> sion variation across the four expression datasets (Ca and                       ranges, (0.02, 10.59), (0.17, 9.18), and (0.09, 11.07), for <br /> Na exposure [19], chemostat [20], environmental stress                           the Ca and Na exposure [19], chemostat [20], and envi- <br /> [21], and oxidative stress [22]) and the combined gene                           ronmental stress conditions [21], respectively. <br /> expression data of Landry et al. [7], and the results are pre- <br /> sented as Additional File 2. The same conclusions can be                         We also studied the contributing factors for gene expres- <br /> obtained indicating the robustness of our results. Previous                      sion variation using the DIP [13] and BioGrid interactions <br /> studies showed that TATA- and non-TATA-containing                                [14], and the results are given in Additional File 1 and File <br /> genes might recruit different coactivator complexes for                          2. Similar conclusions as those based on the MIPS interac- <br /> gene expression [17]. TATA-containing genes were also                            tion data [12] were obtained. The consistency of the <br /> found to be subject to greater nucleosomal regulation                            results using different combinations of protein interaction <br /> Table 4: The effect of toxicity degree on expression variation stratified by the set of environmental stress response (ESR). <br />  <br />  Gene group                   Ca_Na_exposure                       Chemostat                     Environmental Stress                Oxidative Stress <br />  <br />                              &#206;&#178;            p-value             &#206;&#178;             p-value              &#206;&#178;              p-value               &#206;&#178;             p-value <br />  <br />  ESR                       -0.0710        0.0053           -0.0314           0.0532           -0.0176            0.3640            0.0026           0.8708 <br />  Non-ESR                   -0.0848       2.09e-13          -0.0354          1.11e-07          -0.0579           1.28e-11           -0.0091          0.2086 <br />  <br />  The linear model is built for each of the four gene expression datasets, respectively. &#206;&#178; is the linear coefficient in the linear model and the p-value is <br />  related to the null hypothesis that &#206;&#178; &#226;&#8240;&#160; 0 versus &#206;&#178; = 0. <br />  <br />  <br />                                                                                                                                               Page 9 of 12 <br />                                                                                                                        (page number not for citation purposes) <br />  BMC Systems Biology 2008, 2:54                                              http://www.biomedcentral.com/1752-0509/2/54 <br />  <br />  <br />  <br /> data sets and gene expression profiles showed the robust-     least 40 conditions. These datasets include yeast gene <br /> ness of our conclusions. However, the fraction of gene        expression profiles under 40 Ca and Na exposure condi- <br /> expression variation explained by all factors is less than    tions [19], chemostat (i.e., nutritional stress) at 100 con- <br /> 25%. One possible explanation is that the measurement         ditions [20], environmental stress at 156 conditions [21] <br /> of gene expression changes and other factors, including       and oxidative stress at 70 conditions [22]. These data were <br /> the toxicity degree and interaction degree, are still very    analyzed separately to ensure that between-laboratory <br /> noisy. We expect that the true R2 would be higher than        variation was minimized. A combined gene expression <br /> that observed in this study.                                  profile under more than 1,500 conditions was collected <br />                                                               by [7]. The responsiveness for each gene across more than <br /> Conclusion                                                    1,500 conditions calculated by [7] was used in our analy- <br /> We implemented a system-wide analysis of proteomic            sis as expression variation. <br /> and genomic factors affecting gene expression variation. <br /> Among four different factors (protein interaction degree,     Protein interaction data <br /> toxicity degree, TATA box, the number of TFs), TATA-box       We downloaded yeast protein interaction data from three <br /> and the number of TFs are the most important factors          different data sources. The MIPS (Munich Information <br /> influencing gene expression variation. The influence of       Center for Protein Sequences) [12] dataset (version: <br /> TATA-box on evolutionary gene expression variation has        PPI_18052006.tab) contains 11,124 protein physical <br /> been extensively studied both computationally and exper-      interactions involving 4,404 proteins. The DIP core inter- <br /> imentally [6,8], and our results are consistent with their    action dataset [13] (version: ScereCR20070107) contains <br /> findings. Although it is intuitive that the number of TFs     5,738 protein interactions involving 2,161 proteins. The <br /> regulating a gene should have a significant effect on the     DIP core interactions were assessed by a number of qual- <br /> gene&apos;s expression variation, the magnitude of its influence   ity tests and are supposed to be highly reliable [11]. The <br /> has not been studied in large scale expression datasets to    BioGrid [14] dataset (version 2.0.34) contains 59,317 <br /> the best of our knowledge. Our findings demonstrated          protein physical interactions involving 5,054 proteins. <br /> that the gene regulation is a main factor affecting gene <br /> expression variation. Protein interaction degree and toxic-   Essential and toxicity modulating genes <br /> ity degree do not account for as much variation when          Large scale gene deletion studies have identified about <br /> compared to the influence of the number of TFs and the        17&#226;&#8364;&#8220;20% of the genes essential for yeast cell survival [15] <br /> TATA-box.                                                     under normal conditions. Even within the class of non- <br />                                                               essential genes, a gene&apos;s importance in relation to cell sur- <br /> In our overall analysis, we also found the interactions       vival is not the same. Further studies classified the non- <br /> between TATA-box and toxicity degree as well as the           essential genes based on the cell&apos;s fitness phenotypes <br /> number of TFs influence expression variation. The further     under four different DNA damage perturbations when a <br /> study stratified by TATA-box indicated that TATA-contain-     gene is knocked out [16]. The toxicity modulating genes <br /> ing genes and non-TATA containing genes behave differ-        were defined as those significantly affecting the cell&apos;s fit- <br /> ently in relation to the toxicity degree and the number of    ness phenotype when knocked out. We defined the toxic- <br /> TFs. The effect of the number of TFs on expression varia-     ity degree of a gene as the number of perturbations that <br /> tion within the TATA-containing genes is lower than that      significantly affected the deletion strain&apos;s fitness. Essential <br /> for the non-TATA-containing genes. On the other hand,         genes were downloaded from the SGD website [29], and <br /> toxicity degree is associated with expression variation       the toxicity degrees of non-essential genes were calculated <br /> within the TATA-containing genes only. These findings         from [16]. <br /> suggest that the regulatory mechanism might be more <br /> complicated for TATA-containing genes than non-TATA           Gene Regulatory Network <br /> containing genes.                                             Studies have shown that gene expression variation is pos- <br />                                                               itively correlated with the number of cis-regulatory ele- <br /> Methods                                                       ments and the length of intergenic region in several <br /> In order to study factors affecting gene expression varia-    organisms. Since cis-elements control the expression of <br /> tion, we collected data on gene expression profiles, pro-     genes through interaction with the TFs, it is interesting to <br /> tein physical interactions, gene regulatory networks,         study if the number of TFs regulating a gene has an effect <br /> essentiality and toxicity resistance. Details of these data   on gene expression variation. The mapping of cis-ele- <br /> are given below.                                              ments to genes was obtained using motif discovery algo- <br />                                                               rithms, PhyloCon and Converge, with binding p-value <br /> Gene expression profiles                                      less than 0.001 and conservation in at least 0, 1 or 2 other <br /> A large number of gene expression studies are available. In   yeast species [26]. The mapping of the TFs to genes is <br /> this study, we chose gene expression studies containing at    obtained from Hu et al. [27]. <br />  <br />  <br />                                                                                                                   Page 10 of 12 <br />                                                                                              (page number not for citation purposes) <br />  BMC Systems Biology 2008, 2:54                                                       http://www.biomedcentral.com/1752-0509/2/54 <br />  <br />  <br />  <br /> TATA-containing genes                                           not clear. Since these factors are not highly correlated, we <br /> A TATA box is a DNA sequence (cis-element) found in the         treat them as independent factors in our joint analysis. <br /> promoter region of most eukaryotic genes. The TATA con- <br /> sensus sequence was identified as TATA(A/T)A(A/T)(A/G)          In the overall analysis, we first used stepwise selection to <br /> [17]. The TATA box has been identified as a very impor-         find a model that gives the smallest AIC (Akaike informa- <br /> tant factor for gene expression variation. The relationship     tion criterion) = 2*K+n*ln(SSE/n), where K is the number <br /> between yeast genes and the TATA box was downloaded             of parameters in the model; n is the number of observa- <br /> from [17]. There are 1090 out of 6278 genes that were pre-      tions; and SSE is the residual sum of squares [30]. We then <br /> dicted to have a TATA box. Our analysis used these 1090         used linear regression to analyze the relationship between <br /> genes as TATA-containing genes and other genes as non-          gene expression variation and the retained factors and <br /> TATA-containing genes. (We note that 607 genes are not          interactions. The corresponding p-values and the R2 val- <br /> classified in [17], and the results are essentially the same    ues are reported in Table 2. <br /> when these genes are not considered (data not shown).) <br />                                                                 Authors&apos; contributions <br /> Statistical Analysis                                            All authors contributed to the design and coordination of <br /> Gene expression variation was measured by the logarithm         the study. LZ carried out the calculations and performed <br /> of the variance of the gene expression levels under various     the statistical analysis. FS drafted the original manuscript, <br /> conditions. The distribution of the variance was not nor-       which was revised by XM and LZ. All authors read and <br /> mal. In addition, the standard deviations of the resulting      approved the final manuscript. <br /> distributions conditional on the independent variables <br /> (protein physical interaction degree, toxicity degree, TATA     Additional material <br /> box, number of TFs) differed widely, making the linear <br /> model for the variance invalid. To avoid these problems, <br /> we measured the gene expression variation by the loga- <br />                                                                      Additional file 1 <br />                                                                      The statistical analysis of individual factors on gene expression variation <br /> rithm of the variance. The resulting distributions seem to           using four expression datasets, respectively. <br /> fit the conditions for the linear model. Hence, in our               Click here for file <br /> study, we used a linear model to study the relationship              [http://www.biomedcentral.com/content/supplementary/1752- <br /> between the expression variation and each factor. In the             0509-2-54-S1.pdf] <br /> study of the relationship between the gene expression var- <br /> iation and interaction degrees, we first used the LOWESS             Additional file 2 <br /> function in R [23] to fit the data. An approximate linear            Overall analysis of four main factors (protein interaction degree, toxicity <br />                                                                      degree, number of TFs and the presence of TATA box) and their interac- <br /> relationship between gene expression variation and inter-            tions. <br /> action degree was observed when the interaction degree               Click here for file <br /> was less than 20. We then proceeded to use linear regres-            [http://www.biomedcentral.com/content/supplementary/1752- <br /> sion to fit the data up to interaction degree 20.                    0509-2-54-S2.pdf] <br />  <br />  <br />                          v = &#206;&#177; + &#206;&#178;d <br />  <br /> where v is the gene expression variation and d is the inter-    Acknowledgements <br /> action degree. &#206;&#177; and &#206;&#178; are parameters. We tested the sta-       We thank Mr. David Martin for carefully reading the manuscript and for <br /> tistical significance for the relationship between gene         suggestions that significantly improved the presentation of the paper. This <br />                                                                 work is partly supported by the NIH/NSF Joint Mathematical Biology Initi- <br /> expression variation and interaction degree based on the <br />                                                                 ative DMS-0241102 and NIH P50 HG 002790. We sincerely thank the <br /> linear regression model.                                        anonymous reviewers for pointing out several important references that <br />                                                                 were missed in the original version. We also thank the reviewers for sug- <br /> Before we do the joint analysis of expression variation         gestions that significantly improved the presentation of the paper. <br /> with respect to the four factors (protein interaction degree, <br /> toxicity degree, number of TFs, and TATA box), we tested        References <br /> if the four factors are highly correlated. We calculated the    1.      Newman JR, Ghaemmaghami S, Ihmels J, Breslow DK, Noble M, <br />                                                                         DeRisi JL, Weissman JS: Single-cell proteomic analysis of S. cer- <br /> correlation matrix between them and it is given in Addi-                evisiae reveals the architecture of biological noise. Nature <br /> tional File 2 (Supplementary Table 9). All the correlation              2006, 441:840-6. <br /> coefficients are smaller than 0.3 indicating that they are      2.      Nelson CE, Hersh BM, Carroll SB: The regulatory content of <br />                                                                         intergenic DNA shapes genome architecture. Genome Biol <br /> not highly correlated. Although it might be more compu-                 2004, 5:R25. <br /> tationally reasonable to first find the principal compo-        3.      Walther D, Brunnemann R, Selbig J: The regulatory <span id='am-3' about='Thesaurus:code' typeof='owl:Thing'>code</span> for tran- <br /> nents of these factors and then analyze the data using                  scriptional response diversity and its relation to genome <br />                                                                         structural properties in A. thaliana. PLoS Genet 2007, 3:e11. <br /> linear regression, the interpretation of the final result is <br />  <br />  <br />                                                                                                                                Page 11 of 12 <br />                                                                                                           (page number not for citation purposes) <br />  BMC Systems Biology 2008, 2:54                                                                   http://www.biomedcentral.com/1752-0509/2/54 <br />  <br />  <br />  <br /> 4.    Lemos B, Meiklejohn CD, Hartl DL: Regulatory evolution across           24.   Bader JS, Chaudhuri A, Rothberg JM, Chant J: Gaining confidence <br />       the protein interaction network. Nat Genet 2004, 36:1059-60.                  in high-throughput protein interaction networks. Nat Biotech- <br /> 5.    Lemos B, Bettencourt BR, Meiklejohn CD, Hartl DL: Evolution of                nol 2004, 22:78-85. <br />       proteins and gene expression levels are coupled in Dro-                 25.   Choi JK, Kim SC, Seo J, Kim S, Bhak J: Impact of transcriptional <br />       sophila and are independently associated with mRNA abun-                      properties on essentiality and evolutionary rate. Genetics <br />       dance, protein length, and number of protein-protein                          2007, 175:199-206. <br />       interactions. Mol Biol Evol 2005, 22:1345-54.                           26.   MacIsaac KD, Wang T, Gordon DB, Gifford DK, Stormo GD, Fraen- <br /> 6.    Landry CR, Lemos B, Rifkin SA, Dickinson WJ, Hartl DL: Genetic                kel E: An improved map of conserved regulatory sites for Sac- <br />       properties influencing the evolvability of gene expression.                   charomyces cerevisiae. BMC Bioinformatics 2006, 7:113. <br />       Science 2007, 317:118-21.                                               27.   Hu Z, Killion PJ, Iyer VR: Genetic reconstruction of a functional <br /> 7.    Tirosh I, Weinberger A, Carmi M, Barkai N: A genetic signature of             transcriptional regulatory network.                 Nat Genet 2007, <br />       interspecies variations in gene expression. Nat Genet 2006,                   39(5):683-7. <br />       38:830-4.                                                               28.   Akaike H: Information theory and an extension of the maxi- <br /> 8.    Tirosh I, Barkai N: Evolution of gene sequence and gene                       mum likelihood principle. In Proceedings of Second International <br />       expression are not correlated in yeast. Trends Genet 2008,                    Symposium on Information Theory Edited by: Petrov BN, Csaki F. <br />       24:109-13.                                                                    Akademiai Kiado, Budapest; 1973:267-281. <br /> 9.    Batada NN, Reguly T, Breitkreutz A, Boucher L, Breitkreutz BJ, Hurst    29.   Winzeler EA, Shoemaker DD, Astromoff A, Liang H, Anderson K, <br />       LD, Tyers M: Stratus not altocumulus: a new view of the yeast                 Andre B, Bangham R, Benito R, Boeke JD, Bussey H, Chu AM, Con- <br />       protein interaction network. PLoS Biol 2006, 4(10):e317.                      nelly C, Davis K, Dietrich F, Dow SW, El Bakkoury M, Foury F, Friend <br /> 10.   Deng MH, Sun FZ, Chen T: Assessment of the reliability of pro-                SH, Gentalen E, Giaever G, Hegemann JH, Jones T, Laub M, Liao H, <br />       tein-protein interactions and protein function prediction.                    Liebundguth N, Lockhart DJ, Lucau-Danila A, Lussier M, M&apos;Rabet N, <br />       Pac Symp Biocomput 2003:140-151.                                              Menard P, Mittmann M, Pai C, Rebischung C, Revuelta JL, Riles L, Rob- <br /> 11.   Deane CM, Salwinski L, I Xenarios, Eisenberg D: Protein interac-              erts CJ, Ross-MacDonald P, Scherens B, Snyder M, Sookhai-Mahadeo <br />       tions: two methods for assessment of the reliability of high                  S, Storms RK, V&#195;&#169;ronneau S, Voet M, Volckaert G, Ward TR, Wysocki <br />       throughput observations. Mol Cell Proteomics 2002, 1:349-56.                  R, Yen GS, Yu K, Zimmermann K, Philippsen P, Johnston M, Davis <br /> 12.   Mewes HW, Amid C, Arnold R, Frishman D, G&#195;&#188;ldener U, Mannhaupt                 RW: Functional characterization of the S. cerevisiae genome <br />       G, M&#195;&#188;nsterk&#195;&#182;tter M, Pagel P, Strack N, St&#195;&#188;mpflen V, Warfsmann J,              by gene deletion and parallel analysis. Science 1999, 285:901-6. <br />       Ruepp A: MIPS: analysis and annotation of proteins from                 30.   Burnham KP, Anderson DR: Model selection and multimodel inference: <br />       whole genomes. Nucleic Acids Res 2004:D41-44.                                 a practical-theoretic approach 2nd edition. Springer-Verlag; 2002. <br /> 13.   Salwinski L, Miller CS, Smith AJ, Pettit FK, Bowie JU, Eisenberg D: <br />       The Database of Interacting Proteins: 2004 update. Nucleic <br />       Acids Res 2004:D449-451. <br /> 14.   Stark C, Breitkreutz BJ, Reguly T, Boucher L, Breitkreutz A, Tyers M: <br />       BioGRID: a general repository for interaction datasets. <br />       Nucleic Acids Res 2006, 34 (Database issue):D535-9. <br /> 15.   Giaever G, Chu AM, Ni L, Connelly C, Riles L, V&#195;&#169;ronneau S, Dow S, <br />       Lucau-Danila A, Anderson K, Andr&#195;&#169; B, Arkin AP, Astromoff A, El- <br />       Bakkoury M, Bangham R, Benito R, Brachat S, Campanaro S, Curtiss <br />       M, Davis K, Deutschbauer A, Entian KD, Flaherty P, Foury F, Garfinkel <br />       DJ, Gerstein M, Gotte D, G&#195;&#188;ldener U, Hegemann JH, Hempel S, Her- <br />       man Z, Jaramillo DF, Kelly DE, Kelly SL, K&#195;&#182;tter P, LaBonte D, Lamb <br />       DC, Lan N, Liang H, Liao H, Liu L, Luo C, Lussier M, Mao R, Menard <br />       P, Ooi SL, Revuelta JL, Roberts CJ, Rose M, Ross-Macdonald P, Sche- <br />       rens B, Schimmack G, Shafer B, Shoemaker DD, Sookhai-Mahadeo S, <br />       Storms RK, Strathern JN, Valle G, Voet M, Volckaert G, Wang CY, <br />       Ward TR, Wilhelmy J, Winzeler EA, Yang Y, Yen G, Youngman E, Yu <br />       K, Bussey H, Boeke JD, Snyder M, Philippsen P, Davis RW, Johnston <br />       M: Functional profiling of the Saccharomyces cerevisiae <br />       genome. Nature 2002, 418:387-91. <br /> 16.   Said MR, Begley TJ, Oppenheim AV, Lauffenburger DA, Samson LD: <br />       Global network analysis of phenotypic effects: protein net- <br />       works and toxicity modulation in Saccharomyces cerevisiae. <br />       Proc Natl Acad Sci USA 2004, 101:18006-11. <br /> 17.   Basehoar AD, Zanton SJ, Pugh BF: Identification and distinct reg- <br />       ulation of yeast TATA box-containing genes. Cell 2004, <br />       116:699-709. <br /> 18.   Choi JK, Kim YJ: Epigenetic regulation and the variability of <br />       gene expression. Nat Genet 2008, 40:141-7. <br /> 19.   Yoshimoto H, Saltsman K, Gasch AP, Li HX, Ogawa N, Botstein D, <br />       Brown PO, Cyert MS: Genome-wide analysis of gene expres- <br />       sion regulated by the calcineurin/Crz1p signaling pathway in                      Publish with Bio Med Central and every <br />       Saccharomyces cerevisiae. J Biol Chem 2002, 277:31079-88. <br /> 20.   Saldanha AJ, Brauer MJ, Botstein D: Nutritional homeostasis in                   scientist can read your work free of charge <br />       batch and steady-state culture of yeast. Mol Biol Cell 2004,                  &quot;BioMed Central will be the most significant development for <br />       15:4089-104.                                                                  disseminating the results of biomedical researc h in our lifetime.&quot; <br /> 21.   Gasch AP, Spellman PT, Kao CM, Carmel-Harel O, Eisen MB, Storz <br />       G, Botstein D, Brown PO: Genomic expression programs in the                       Sir Paul Nurse, Cancer Research UK <br />       response of yeast cells to environmental changes. Mol Biol Cell                 Your research papers will be: <br />       2000, 11:4241-57. <br /> 22.   Shapira M, Segal E, Botstein D: Disruption of yeast forkhead-                     available free of charge to the entire biomedical community <br />       associated cell cycle transcription by oxidative stress. Mol Biol                 peer reviewed and published immediately upon acceptance <br />       Cell 2004, 15:5659-69. <br /> 23.   The R Project for Statistical Computing                [http://www.r-             cited in PubMed and archived on PubMed Central <br />       project.org/]                                                                     yours &#226;&#8364;&#8221; you keep the copyright <br />  <br />                                                                                Submit your manuscript here:                                BioMedcentral <br />                                                                                http://www.biomedcentral.com/info/publishing_adv.asp <br />  <br />  <br />  <br />  <br />                                                                                                                                               Page 12 of 12 <br />                                                                                                                          (page number not for citation purposes) <br />   <br /> </body></html>