<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Molecular Microbiology (2010) 76(6), 1461&#226;&#8364;&#8220;1482 &#228;&#352;?                                                              doi:10.1111/j.1365-2958.2010.07151.x <br />                                                                                                                First published online 15 April 2010 <br />  <br />  <br />  <br /> The essential neutral sphingomyelinase involved the <br /> traf&#239;&#172;?cking variant surface glycoprotein the <br /> bloodstream form Trypanosoma brucei                                                  mmi_7151 1461..1482 <br />  <br />  <br />  <br />  <br /> Simon A. Young Terry K. Smith*                                    Introduction <br /> Biomolecular Science, The North Haugh, The University, <br /> St. Andrews, Fife Scotland KY16 9ST, UK.                              The medically important protozoan kinetoplastid Trypano- <br />                                                                       soma brucei, cause African sleeping sickness, has <br />                                                                       complex life cycle alternates insect <br /> Summary                                                               vector mammalian host (Matthews, 2005). Despite <br /> Sphingomyelin main sphingolipid Trypano-                    phospholipid content life cycle <br /> soma brucei, causative agent African sleeping                  stages remarkably similar (Patnaik et al., 1993; Rich- <br /> sickness. In vitro vivo characterization                mond et al., 2010). Phosphatidylcholine (PC) sphin- <br /> T. brucei neutral sphingomyelinase demonstrates                       gomyelin (SM) abundant phospholipid <br /> that directly involved sphingomyelin catabo-                 species eukaryotic cells accounts half of <br /> lism. Gene <span id='am-241' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-242' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-243' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-251' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-258' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-278' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-289' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-290' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-296' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-315' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-316' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-317' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-318' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span>knockout</span> studies bloodstream                        total phospholipids T. brucei membranes (Patnaik <br /> form parasite indicate neutral sphin-                 et al., 1993). <br /> gomyelinase essential growth <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span>survival</span>,                        Sphingolipids, common SM, are <br /> thus highlighting novo biosynthesis                    essential structural components eukaryotic mem- <br /> ceramide unable compensate loss                      branes, function diverse cellular processes (Cowart <br /> sphingomyelin catabolism. The <span id='am-163' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-164' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-240' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span>phenotype</span>                        et al., 2006; Lahiri Futerman, 2007). They abun- <br /> conditional knockout given new insights                      dant outer leaflet plasma membrane many <br /> the highly active endocytic exocytic pathways                  cells lysosomal membranes and <br /> the bloodstream form T. brucei. Hence, forma-                  endoplasmic reticulum (ER) Golgi, they <br /> tion ceramide endoplasmic reticulum                         play important role controlling membrane fluidity <br /> affects post-Golgi sorting rate deposition                  (Koynova Caffery, 1995; van Meer et al., 2008). <br /> newly synthesized GPI-anchored variant surface gly-                      Eukaryotic SM synthase activity observed in <br /> coprotein cell surface. This directly influ-                   plasma membrane, primarily localized the <br /> ences corresponding rate endocytosis,                  Golgi, transfers choline&#226;&#8364;&#8220;phosphate head- <br /> recycling endosomes, pre-existing cell surface                     group PC ceramide. Ceramide formed <br /> variant surface glycoprotein. The trypanosomes use                    novo biosynthesis, commences by <br /> this coupled endocytic exocytic mechanism                      condensation serine palmitoyl-CoA, via <br /> maintain cell density crucial variant                      hydrolysis (degradation) complex sphingolipids nor- <br /> surface glycoprotein protective coat. TbnSMase                     mally sphingomyelinases (SMases) (Futerman and <br /> therefore genetically validated drug target                      Riezman, 2005). <br /> against African trypanosomes, suggests                               The diversity sphingolipid synthesis developing <br /> that interfering endocytic transport                      area anti-fungals yeast synthesize significant <br /> variant surface glycoprotein highly desirable                    amounts inositol phosphorylceramide (IPC), the <br /> strategy drug development African                         transfer inositol-1-phosphate headgroup phos- <br /> trypanosomasis.                                                       phatidylinositol (PI) ceramide, contrast <span id='am-6' about='protege:TO' typeof='owl:Thing'>to</span> <br />                                                                       mammals, synthesize SM (Dickson et al., <br />                                                                       2006). Similarly various sphingolipid species their <br /> Accepted 22 March, 2010. *For correspondence. E-mail tks1@st- <br /> andrews.ac.uk; Tel. (+44) 1334 463412; Fax (+44) 1334 462593. The     biosynthesis kinetoplastid parasites drawn <br /> nucleotide sequence(s) reported paper submitted   attention potential novel chemotherapeutic <br /> the EBI Data Bank accession number(s) FM992873.                  targets (Suzuki et al., 2008). Trypanosoma cruzi and <br /> Re-use article permitted accordance Terms <br /> and Conditions set http://www3.interscience.wiley.com/         Leishmania synthesize significant amounts IPC <br /> authorresources/onlineopen.html                                       (Kaneshiro et al., 1986; Bertello et al., 1995; Uhrig et al., <br /> &#194;&#169; 2010 Blackwell Publishing Ltd <br />  1462 S. A. Young T. K. Smith &#228;&#352;? <br />  <br /> 1996; Zhang et al., 2007) orthologues yeast           GPI-dependent trafficking trypanosomes unclear <br /> AUR1 inositol phosphoceramide synthase (Denny et al.,            (Denny Smith, 2004). <br /> 2006). The viability Leishmania major promastigote             Seminal work Bangs&#226;&#8364;&#8482; group resulted the <br /> null serine palmitoyltransferase, enzyme     hypothesis post-Golgi sorting soluble GPI- <br /> the sphingoid <span id='am-475' about='xsp:base' typeof='owl:Thing'>base</span> pathway, suggests Leishmania          anchored cargo influenced endocytic rate each <br /> not require sphingolipids (Zhang et al., 2003). However,      particular life cycle stage T. brucei (Tazeh et al., 2009). <br /> was subsequently discovered use sphingosine          This stage-specific variation mirrored depen- <br /> lyase generate ethanolamine (Zhang et al., 2007).             dency clathrin endocytosis bloodstream stage <br />    Procyclic T. brucei contain SM IPC (G&#195;&#188;ther           (Allen et al., 2003), enlarged flagellar pocket, <br /> et al., 2006; Fridberg et al., 2008), bloodstream          known &#226;&#8364;&#732;Big-Eye&#226;&#8364;&#8482; phenotype observed mem- <br /> stage parasites contain IPC synthesizing SM          brane vesicles continue delivered flagellar <br /> minor amounts ethanolamine phosphorylceramide                 pocket, internalized endocytic <br /> (EPC) (Bagnat et al., 2000; Sutterwala et al., 2008).            pathway flagellar pocket; site endocy- <br /> Recently, discovered mammalian stumpy           tosis exocytosis parasites (reviewed in <br /> form T. brucei upregulate mitochondrial SM synthase            Overath Engstler, 2004). <br /> (SLS1, Tb09.211.1030) (Kabani et al., 2009), responsible            Intracellular degradation sphingolipids achieved by <br /> for making IPC (Mina et al., 2009), paralleled       SMases [EC. 3.1.4.12], catalyses hydrolysis to <br /> by formation IPC identified electrospray mass           form ceramide corresponding headgroup i.e. <br /> spectrometry (ES-MS) (Kabani et al., 2009).                      phosphorylcholine case SM. <br />    Together, observations clearly indicate sph-          Various SMases described, differ in <br /> ingolipid synthesis African trypanosomes develop-          subcellular localization, tissue specificity are <br /> mentally regulated. RNAi silencing TbSLS locus            normally activated growth factors, cytokines, chemo- <br /> (four tandem genes) leads growth arrest, suggesting           therapeutic agents, irradiation, <span id='am-1' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> removal other <br /> synthesis sphingolipids essential          related stresses (Tomiuk et al., 1998; Yabu et al., 2008). <br /> trypanosomes (Sutterwala et al., 2008).                          Some thought regulate intracel- <br />    In yeast, sorting glycosylphosphatidylinositol         lular ceramide concentration corresponding <br /> (GPI)-anchored proteins influenced sphingolipid-           stress-induced associated responses. <br /> and ergosterol-rich lipid rafts (Zanolari et al., 2000),        To date, main types SMases; acidic <br /> thus yeast dependent novo synthesis cera-           Zn2+-dependent (cytosolic/secreted) independent <br /> mide (Sutterlin et al., 1997). This ceramide synthesized      (lysosomal), neutral Mg2+-dependent (membrane) and <br /> in ER transferred Golgi, process     independent (cytosolic) lastly secreted alkaline <br /> is independent ATP vesicular trafficking (Funato          SMase (reviewed Goni Alonso, 2002). <br /> and Reizman, 2001). The resulting IPC synthesized            There growing body evidence suggests <br /> Golgi (Levine et al., 2000) essential stable mem-         Mg2+-dependent neutral SMases (nSMases) major <br /> brane association GPI-anchored proteins subse-            source stress-induced ceramide production (Yabu <br /> quently correct sorting trafficking plasma      et al., 2008). Ceramide associated metabolites, such <br /> membrane (Watanabe et al., 2002).                                sphingosine-1-phosphate, known function as <br />    Sphingolipid/cholesterol-rich membrane lipid rafts         second messengers, causing various biological activities <br /> polarized mammalian epithelial cells impli-       mammalian cells, including activation protein- <br /> cated associated GPI anchors, strongly          kinases and/or protein-phosphatases 2A (Huwiler et al., <br /> influences sorting proteins leaving Golgi (see        2000). Increased levels ceramides exert anti- <br /> reviews Muniz Reizman, 2000; Chatterjee               proliferation effects, induce apotosis play major roles <br /> Mayor, 2001).                                                    mitogenesis endocytosis. <br />    The abundant GPI-anchored variant surface glycopro-              Neutral SMases cloned Bacillus cereus <br /> tein (VSG) strongly influences trafficking       (Yamada et al., 1988), allowed identification <br /> secretory endocytic pathways bloodstream-stage            nSMases bacteria including Listeria <br /> African trypanosomes (Engstler et al., 2004; Schwartz            ivanovii (Gonzalez-Zorn et al., 1999). Through homology <br /> and Bangs, 2006). Inhibition initial step sph-     searches yeast homologues (Sawai et al., 1999) and <br /> ingolipid biosynthesis, essential viability,     subsequently mammalian homologues identified <br /> surprisingly detrimental effect VSG          (Tomiuk et al., 1998; Hofmann et al., 2000). Despite the <br /> trafficking cell surface (Sutterwala et al., 2007),       extensive literature nSMase activity compara- <br /> despite reports presence VSG detergent-             tively recently mammalian nSMases been <br /> resistant membranes (Nolan et al., 2000; Denny et al.,           cloned characterized. Neutral SMases ubiqui- <br /> 2001). Thus, extent lipid rafts influence     tously expressed mammalian cells, are <br />  <br />                                                          &#194;&#169; 2010 Blackwell Publishing Ltd, Molecular Microbiology, 76, 1461&#226;&#8364;&#8220;1482 <br />                                                                                           Neutral sphingomyelinase Trypanosoma brucei 1463 <br />  <br /> predominately membrane-bound outer leaflet               organisms, significant regions sequence, <br /> the plasma membrane (Tomiuk et al., 1998),              different kinetoplastid nSMase <br /> of SM located.                                              homologues (identified here) nSMases <br />    Plasmodium contains bacterial nSMase homo-               including human protein. Outside kinetoplasts <br /> logue, capable cleaving SM lyso-PC            closest homologue surprisingly honey-bee <br /> (Hanada et al., 2002). Intriguingly, compound scy-             (Apis mellifera). <br /> phostatin inhibits plasmodium enzyme activity, well <br /> as preventing maturation trophozoite schizont at <br />                                                                    Overexpression TbnSMase E. coli <br /> low mM concentrations (Hanada et al., 2000). <br />    The yeast nSMase homologue Isc1 shown               To enable biochemical characterization, TbnSMase was <br /> be associated ER, early growth,           recombinantly expressed E. coli using pGEX-6P-1 <br /> late logarithmic growth outer leaflet    vector. The N-terminal GST fusion protein (full-<span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span>, <br /> mitochondria regulate sphingolipid               membrane bound) expressed E. coli C43 (DE3) <br /> metabolism (Vaena Avalos et al., 1998; Kitagaki et al.,         cells, bacterial host allows synthesis poten- <br /> 2007). The Isc1 protein inositol sphingolipid phospho-         tially toxic membrane proteins (Miroux Walker, <br /> lipase C activity, i.e. capable cleaving IPC generate        1996). The apparent molecular weight GST&#226;&#8364;&#8220; <br /> ceramide, required normal mito-              TbnSMase fusion protein ~81 kDa Western blot <br /> chondrial function (Sawai et al., 2000). Further recent            analysis (Fig. 1A, lane 2), smaller predicted <br /> studies elaborated this, clearly showing           size 91 kDa (including GST tag). A significant pro- <br /> the knockout Isc1 essential diauxic shift, i.e.     portion protein expected the <br /> it plays crucial role reprogramming mitochon-          membrane fraction (Fig. 1A, lane 3). These membranes <br /> drial gene expression transition anaerobic         washed phosphate-buffered saline (PBS) and <br /> to aerobic metabolism, coupled change carbon             suspended 100 mM Tris (pH 7.4), 10 mM MgCl2 buffer <br /> source (Kitagaki et al., 2009).                                    containing 20% glycerol, aliquots stored at <br />    In study, begin investigate catabolism        -80&#194;&#176;C loss activity. Total protein content was <br /> sphingolipids T. brucei.                                        calculated 50.8 mg ml-1 membrane prepara- <br />    Here recombinant expression charac-             tion. Enzyme activity assessed incubation of <br /> terization T. brucei nSMase. We report         GST&#226;&#8364;&#8220;TbnSMase containing membranes (127 mg protein) <br /> the creation T. brucei conditional null mutant,         25 nCi/50.5 pmol [3H-choline-methyl] SM or <br /> demonstrates nSMase activity essential         30 nCi/50.5 pmol [3H-choline-methyl] PC, mixed <br /> bloodstream form parasite, causing VSG trafficking          micelles 0.1% Triton-X 100. Reactions incu- <br /> to impaired.                                                    bated 37&#194;&#176;C 120 min 100 mM Bis-Tris propane <br />                                                                    (pH 7.5), 5 mM MgCl2, 1% glycerol presence or <br />                                                                    absence 5 mM EDTA. Chloroform/methanol/water <br /> Results                                                            phase separation used distinguish radiola- <br />                                                                    belled lipid substrates aqueous radiolabelled <br /> Cloning sequencing T. brucei nSMase <br />                                                                    products, subjected scintillation counting <br /> An nSMase homologue putatively identified T.            thin-layer chromatography. The assay showed that <br /> b. brucei (strain 927) genome database (Sanger Insti-              membranes containing GST&#226;&#8364;&#8220;TbnSMase cata- <br /> tute) interrogating Saccharomyces cerevisiae           lytically active (Fig. 1B, compare lanes 1 2) was <br /> nSMase (Isc1). This open reading frame (ORF)                       divalent cation-dependent (Fig. 1B, compare lanes <br /> (Tb927.5.3710) PCR-amplified T. b. brucei                 2 3). No activity PC apparent (Fig. 1B, <br /> strain 427, subcloned sequenced. The sequence                  lanes 6&#226;&#8364;&#8220;8), SMase activity observed <br /> has submitted EMBL-EBI, Accession No.                      equivalent E. coli membranes containing <br /> FM992873. An alignment predicted translated                 GST&#226;&#8364;&#8220;TbnSMase (Fig. 1B, lanes 4 5). The optimum <br /> sequence nSMases organisms shown                pH GST&#226;&#8364;&#8220;TbnSMase activity pH 8, very <br /> in Fig. S1A unrooted phylogenetic tree                   little activity pH 5.5 (Fig. 1C), representative an <br /> Fig. S1B. The predicted 65 kDa T. brucei nSMase                    nSMase. <br /> (TbnSMase) putative transmembrane domains                      Sphingomyelinase activity similarly detected <br /> (Fig. S1A, underlined) C-terminus,           use multi-well plate, fluorescence- <br /> present eukaryotic nSMases, contains               based, coupled assay (Invitrogen), product <br /> conserved residues critical catalytic activity Mg2+        SMase activity, choline&#226;&#8364;&#8220;phosphate, hydrolysed by <br /> binding (Fig. S1A, asterisks). Although TbnSMase shows             alkaline phosphatase, released choline oxi- <br /> considerable similarity nSMases variety               dized betaine H2O2 (Fig. 2A). Production H2O2 <br />  <br /> &#194;&#169; 2010 Blackwell Publishing Ltd, Molecular Microbiology, 76, 1461&#226;&#8364;&#8220;1482 <br />  1464 S. A. Young T. K. Smith &#228;&#352;? <br />  <br />                                                                                     Fig. 1. Expression, purification activity <br />                                                                                     recombinant TbnSMase. <br />                                                                                     A. TbnSMase cloned expressed <br />                                                                                     N-terminal GST fusion protein C43 <br />                                                                                     E. coli. Protein samples cell disruption <br />                                                                                     separated 10% SDS-PAGE gel, <br />                                                                                     transferred membrane detected with <br />                                                                                     anti-GST antibody. Lane 1, soluble protein <br />                                                                                     (supernatant 100 000 g); lane 2, <br />                                                                                     membrane fraction (pellet 100 000 g); <br />                                                                                     lane 3, insoluble fraction (pellet <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> <br />                                                                                     45 000 g). <br />                                                                                     B. The substrate [3H]-sphingomyelin (SM) and <br />                                                                                     product [3H]-choline&#226;&#8364;&#8220;phosphate the <br />                                                                                     TbnSMase assay separated phase <br />                                                                                     partitioning (organic aqueous <br />                                                                                     respectively), analysed HPTLC and <br />                                                                                     detected autoradiography described in <br />                                                                                     Experimental procedures. Lane 1, negative <br />                                                                                     control, [3H]-SM membranes; lane 2, <br />                                                                                     [3H]-SM TbnSMase; lane 3, [3H]-SM with <br />                                                                                     TbnSMase presence EDTA (5 mM); lane <br />                                                                                     4, negative control, [3H]-SM non <br />                                                                                     expressing TbnSMase membranes; lane 5, <br />                                                                                     [3H]-SM non-expressing TbnSMase <br />                                                                                     membranes EDTA (5 mM).; lane 6, <br />                                                                                     [3H]-PC membranes; lane 7, [3H]-PC <br />                                                                                     TbnSMase; lane 8, [3H]-PC with <br />                                                                                     TbnSMase presence EDTA (5 mM). <br />                                                                                     Percentages substrate turnover shown, <br />                                                                                     determined densitometry (ImageJ <br />                                                                                     software). <br />                                                                                     C. TbnSMase activity determined by <br />                                                                                     end-point radioactive assay function of <br />                                                                                     pH, using MES buffer (filled triangles) or <br />                                                                                     Bis-Tris Propane (filled squares) described <br />                                                                                     Experimental procedures. <br />  <br />  <br />  <br />  <br /> in presence horseradish peroxidase drives             bovine nSMase (detergent extracted bovine brain <br /> conversion Amplex Red reagent red-                 membranes) apparent Km ~40 mM spe- <br /> fluorescent resorufin, detectable using fluorescence           cific activity 371 nmol h-1 mg-1 (Bernardo et al., 2000); <br /> microplate reader (Molecular Devices Gemini XPS). The            (iii) human nSMase1 (His-nSMase expressed <br /> TbnSMase activity showed saturation kinetics                HEK293 cell lysates) Km 26.2 &#226;&#171;&#190; 10.5 mM a <br /> observed, apparent Km 45.7 &#226;&#171;&#190; 6.5 mM               specific activity 126.8 &#226;&#171;&#190; 17.4 nmol h-1 mg-1) (Sawai <br /> Vmax 424.0 &#226;&#171;&#190; 32.6 nmol h-1 mg-1 membrane protein              et al., 1999). <br /> (Fig. 2B).                                                          Sphingomyelinase activity, slightly reduced rate, <br />    Other homologous divalent cation-dependent nSMases            detected using equivalent amounts E. coli <br /> assayed similar manner similar kinetic proper-         membranes expressing N-terminal His-tagged <br /> ties including following: (i) S. cerevisiae ISC1 (over-      version TbnSMase protein (Fig. S2), again <br /> expressed S. cerevisiae JK9-3da cell lysates) Km       activity detected PC (Fig. S2, lane 3). <br /> of 34.8 mM Vmax 9200 nmol h-1 mg-1 (Sawai et al.,       The fluorescence-based coupled assay amenable to <br /> 2000); (ii) Plasmodium falciparum nSMase (GST                high-throughput screening inhibitors recombi- <br /> fusion expressed E. coli membranes) apparent          nant TbnSMase. This allowed determination IC50 <br /> Km ~90 mM specific activity 5.69 &#226;&#171;&#190;                   values various compounds including known nSMase <br /> 0.29 nmol h-1 mg-1 (Hanada et al., 2002). Three mamma-           inhibitors dihydroimidazolo-amide GW4869 <br /> lian homologues kinetically characterized:        (Luberto et al., 2002) Manumycin A (Fig. 2C) (Arenz <br /> (i) rat nSMase (GST fusion partially purified E. coli )     et al., 2001), gave IC50 values 14.4 &#226;&#171;&#190; 2.3 mM <br /> with apparent Km 3.85 &#226;&#171;&#190; 0.73 mM Vmax              24.6 &#226;&#171;&#190; 4.6 mM respectively. Manumycin A has <br /> 239.1 &#226;&#171;&#190; 33.2 nmol h-1 mg-1 (Mizutani et al., 2000); (ii)         ED50 10.0 &#226;&#171;&#190; 6.5 mM; however, noted <br />  <br />                                                          &#194;&#169; 2010 Blackwell Publishing Ltd, Molecular Microbiology, 76, 1461&#226;&#8364;&#8220;1482 <br />                                                                                                     Neutral sphingomyelinase Trypanosoma brucei 1465 <br />  <br />  <br />  <br />  <br /> Fig. 2. Activity recombinant TbnSMase. <br /> A. Reaction catalysed TbnSMase coupled coupled Amplex red assay. AP, alkaline phosphatase; ChOx, choline oxidase; <br /> Cho-P, choline&#226;&#8364;&#8220;phosphate; HRP, horseradish peroxidase. <br /> B. Determination TbnSMase Michaelis-Menten constants SM (inserts Lineweaver&#226;&#8364;&#8220;Burk plot). <br /> C. Structure manumycin A, commercially available nSMase inhibitor. <br /> D. Enzyme activity TbnSMase washed bloodstream T. brucei membranes (lanes 1&#226;&#8364;&#8220;3) E. coli membranes expressing <br /> GST&#226;&#8364;&#8220;TbnSMase (lanes 4&#226;&#8364;&#8220;6), pre-incubation (lanes 1 4) miltefosine (lanes 2 5) edelfosine (lanes 3 and <br /> 6) presence SM substrate described Experimental procedures. Insert shows structure edelfosine. <br />  <br />  <br /> that previously shown trypanocidal,                     nSMase activity T. brucei membranes (Fig. 2D, <br /> with suggestion farnesyl transferase                    lanes 1&#226;&#8364;&#8220;3) recombinant GST&#226;&#8364;&#8220;TbnSMase activity <br /> inhibitor (Ali et al., 1999), light findings            (Fig. 2D, lanes 4&#226;&#8364;&#8220;6). Edelfosine potent that <br /> may Manumycin A multiple trypanocidal                         miltefosine, IC50 21.0 &#226;&#171;&#190; 3.2 mM <br /> targets. The lyso-PC analogues miltefosine edelfos-                   recombinant GST&#226;&#8364;&#8220;TbnSMase (Fig. 2D, <br /> ine (structure insert Fig. 2D) inhibitors                 lane 6). <br />  <br /> &#194;&#169; 2010 Blackwell Publishing Ltd, Molecular Microbiology, 76, 1461&#226;&#8364;&#8220;1482 <br />  1466 S. A. Young T. K. Smith &#228;&#352;? <br />  <br />                                                                          TbnSMase essential gene bloodstream form <br />                                                                          T. brucei <br />  <br />                                                                          Southern blot analysis T. brucei genomic DNA <br />                                                                          showed TbnSMase present single copy gene <br />                                                                          haploid genome (data shown), RT-PCR <br />                                                                          analysis clearly shows TbnSMase expressed <br />                                                                          bloodstream procyclic forms parasite <br />                                                                          (Fig. S4), making amenable genetic validation <br />                                                                          drug target. One allele TbnSMase replaced <br />                                                                          knockout cassette containing puromycin drug <br />                                                                          resistance gene, homologous recombination and <br />                                                                          selection puromycin, creating DTbnSMase::PAC <br />                                                                          cell line (Fig. 4A, Lane 1). Attempts create null <br />                                                                          mutant homologous replacement second allele <br />                                                                          hygromycin resistance gene unsuccessful. <br />                                                                          Therefore, &#226;&#8364;&#732;wild-type&#226;&#8364;&#8482; cell line used consti- <br />                                                                          tutively expresses T7 RNA polymerase tet- <br />                                                                          racycline repressor protein, tetracycline-inducible (Ti) <br />                                                                          HA-tagged ectopic copy TbnSMase inte- <br />                                                                          grated rDNA locus (Wirtz et al., 1999) modi- <br />                                                                          fied version pLew100 vector encoding the <br />                                                                          blasticidin drug resistance protein (previously con- <br />                                                                          structed Smith lab) forming TbnSMase-HA Ti <br />                                                                          DTbnSMase::PAC cell line (Fig. 4A, Lane 2). The second <br />                                                                          allele replaced hygromycin resistance <br />                                                                          gene presence tetracycline. A TbnSMase- <br /> Fig. 3. Localization TbnSMase-HA Ti bloodstream form               HA Ti DTbnSMase::PAC/DTbnSMase::HYG clone was <br /> T. brucei.                                                               obtained genotype confirmed Southern blot <br /> A. Fixed TbnSMase-HA Ti cells incubated rat anti-HA <br />                                                                          analysis (Fig. 4A, Lane 3). To establish essentiality <br /> antibody, rabbit anti-rat FITC conjugated antibody DNA <br /> stained DAPI.                                                       TbnSMase bloodstream T. brucei, growth the <br /> B. Anti-HA Western analysis fractions differential               TbnSMase-HA Ti DTbnSMase::PAC/ DTbnSMase::HYG <br /> centrifugation, 14 500 g pellet (1), 140 000 g pellet (2) and <br />                                                                          conditional knockout (cKO) cell line monitored the <br /> supernatant 140 000 g pellet (3) TbnSMase-HA Ti cells, <br /> prepared described Experimental procedures.                    absence presence tetracycline certified <br />                                                                          tetracycline-free HMI-9. In presence tetracycline <br />                                                                          cKO cell line showed normal growth rates when <br /> nSMase located endoplasmic reticulum the <br />                                                                          compared wild-type cells (Fig. 4B). However, the <br /> bloodstream form T. brucei <br />                                                                          absence tetracycline, cells grew normally ~24&#226;&#8364;&#8220; <br /> The cellular location TbnSMase investigated using                 28 h ceasing division steadily declined cell <br /> the tetracycline-inducible overexpression vector pLew82,                 numbers limits detection light micros- <br /> containing full-length TbnSMase C-terminal                      copy day 5. At day 10 however, live cells were <br /> HA-tag. This construct introduced bloodstream                   visualized having wild-type morphologies subse- <br /> form T. brucei cells, generating cell line TbnSMase-                 quently resumed normal growth kinetics. RT-PCR analy- <br /> HA Ti integration confirmed PCR using specific                sis showed transcript level ectopic <br /> vector primers (data shown). Tetracycline-induced                    TbnSMase-HA Ti similar endogenous <br /> TbnSMase-HA Ti cells fixed, adhered poly-lysine                  copy seen wild-type cells (Fig. 4C, compare lanes1 <br /> slides permeablized. The TbnSMase-HA Ti protein                      2). In absence tetracycline 24 h tran- <br /> was detected immunofluorescence perinuclear                      script TbnSMase-HA Ti undetectable (Fig. 4C, <br /> staining reticular cytoplasmic signal (Fig. 3A                 lane 3), revertant cells, transcript for <br /> and Fig. S3), suggesting TbnSMase localized                   TbnSMase-HA Ti clearly shown (Fig. 4C, lane 4), <br /> the ER. This confirmed subcellular fractionation                  suggesting revertant cells overcome tet- <br /> studies, membrane bound TbnSMase-HA                            racycline control, phenomena described <br /> protein detected microsomal fraction                  essential genes T. brucei (Chang et al., <br /> the cytosolic fraction (Fig. 3B).                                        2002; Roper et al., 2002; Martin Smith, 2006a). <br />  <br />                                                                  &#194;&#169; 2010 Blackwell Publishing Ltd, Molecular Microbiology, 76, 1461&#226;&#8364;&#8220;1482 <br />                                                                                           Neutral sphingomyelinase Trypanosoma brucei 1467 <br />  <br />                                                                                      Fig. 4. TbnSMase essential the <br />                                                                                      survival bloodstream form Trypanosoma <br />                                                                                      brucei culture. <br />                                                                                      A. Confirmation genotype T. brucei <br />                                                                                      TbnSMase conditional double knockout cell <br />                                                                                      line. Southern blot analysis Pst1-digested <br />                                                                                      genomic DNA (3 mg); TbnSMase ORF <br />                                                                                      probe shows allelic TbnSmase 4 kb and <br />                                                                                      ectopic copy &#226;&#8364;&#8220; TbnSMase-HATi ~2 kb; <br />                                                                                      parental cells (lane 1); <br />                                                                                      DnSMase:::PAC/TbnSMase-HATi (lane 2); <br />                                                                                      DnSMase:::PAC <br />                                                                                      DnSMase:::HYG/TbnSMase-HATi (lane 3). <br />                                                                                      B. Growth curves T. brucei parental cells (1 <br />                                                                                      &#226;&#8364;&#8220; filled squares) TbnSmase conditional <br />                                                                                      knockout cells grown presence (2 &#226;&#8364;&#8220; <br />                                                                                      circles) absence (3 &#226;&#8364;&#8220; filled triangles) <br />                                                                                      tetracycline. <br />                                                                                      C. RT-PCR amplification TbnSMase RNA <br />                                                                                      transcripts total RNA extracted from <br />                                                                                      wild-type cells (lane 1) TbnSMase <br />                                                                                      conditional <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span>null mutants</span> grown the <br />                                                                                      presence (lane 2) absence tetracycline <br />                                                                                      1 10 days (lanes 3 4 <br />                                                                                      respectively). The upper panel shows RT-PCR <br />                                                                                      products using primers specific for <br />                                                                                      TbnSMase; lower panel shows loading <br />                                                                                      control using (TbINO1) primers. <br />  <br />  <br />  <br />  <br /> Lipidomic analysis nSMase conditional knockout              parent species collision-induced daughter ion char- <br />                                                                    acteristic PC SM phospholipids (parents of <br /> The biochemical phenotype TbnSMase condi-                   184 m/z) observed wild-type cKO cells <br /> tional knockout cells investigated using mass            cultured absence tetracycline 42 h (TbnS- <br /> spectrometry examine differences phospholipid                Mase cKO (42h&#226;&#8364;&#8220;tet)) (Fig. S5). Also minor <br /> species wild-type nSMase conditional               changes detected corresponding negative ion <br /> knockout trypanosomes. Based growth curve                   survey scans (Fig. S6A). Parent ion neutral loss <br /> (Fig. 4B), 42 h tetracycline chosen             scanning individual phospholipid classes acquired <br /> optimum time point characterize TbnSMase cKO                PI, phosphatidylethanolamine (PE), phosphati- <br /> phenotype, cells viable metaboli-            dylserine (PS) (Fig. S6B&#226;&#8364;&#8220;D). The difference any <br /> cally active. Six hours later (48h&#226;&#8364;&#8220;tet) case,      note compared wild-type bloodstream form <br /> thus 42 h deemed extreme point               phospholipid spectra unexpected presence <br /> to investigate advanced phenotype studying             inositolphosphorylceramide (IPC) (34:1, 778m/z) <br /> dying cells.                                                       (Fig. S6B, arrow). <br />    Initially, ES-MS used examine phospholipid              Consequently, hypothesized potential <br /> content whole-cell extracts TbnSMase-depleted              differences phospholipid profiles par- <br /> cells, focussing initially positive ion mode detec-     ticular organelle ER. Thus, lipids were <br /> tion PC SM. However, differences             extracted microsomal fraction (100,000g pellet, <br />  <br /> &#194;&#169; 2010 Blackwell Publishing Ltd, Molecular Microbiology, 76, 1461&#226;&#8364;&#8220;1482 <br />  1468 S. A. Young T. K. Smith &#228;&#352;? <br />  <br />                                                                                      Fig. 5. Mass spectrometric analyses of <br />                                                                                      phospholipids. Choline-containing <br />                                                                                      phospholipids extracted P100 <br />                                                                                      fraction described Experimental <br />                                                                                      procedures analysed ESI-MS/MS in <br />                                                                                      positive ion mode using parent-ion scanning <br />                                                                                      collision induced fragment for <br />                                                                                      phosphorylcholine 184 m/z (A) parental <br />                                                                                      cells. (B) TbnSMase cKO cells grown the <br />                                                                                      absence tetracycline 42 h. ESI-MS <br />                                                                                      negative ion survey scan spectra P100 lipid <br />                                                                                      extracts T. brucei bloodstream form cells. <br />                                                                                      (C) Parental cells. (D) TbnSMase cKO cells <br />                                                                                      grown absence tetracycline 42 h. <br />                                                                                      Asterisks highlight ions, altered <br />                                                                                      significantly intensity, described the <br />                                                                                      text. <br />  <br />  <br />  <br />  <br /> P100), representing internal organelle membranes,            phosphatidylglycerol (PG) (diacyl, C36:3, mixture of <br /> both wild-type TbnSMase cKO (42h&#226;&#8364;&#8220;tet) trypano-                C18:1, C18:2 C18:0, C18:3). The corresponding <br /> somes analysed ES-MS ES-MS-MS. Upon                    negative ion survey scan TbnSMase cKO (42h&#226;&#8364;&#8220; <br /> collision-induced daughter ion scanning choline&#226;&#8364;&#8220;              tet) P100 fraction (Fig. 5D), showed PG (diacyl, <br /> phosphate phospholipids (parents 184 m/z), i.e. PC             C36:3) species major species; however, there <br /> and SM (Fig. 5A B), differences observed.                significant increase species 641 m/z and <br /> These included significant decrease PC species           669 m/z (Fig. 5D, marked asterick), identified by <br /> at 771 m/z (alkylacyl 36:2), minor decreases        ES-MS/MS daughter ion fragmentation (Fig. S7B), as <br /> PC species 807 m/z (diacyl C38:5) 835 m/z (diacyl          diacylglycerol (DAG) species C38:5 (C16:0, C22:5) and <br /> C40:5) (Fig. 5A B, species highlighted asterisks).       C40:5 (C18:0, C22:5) respectively. As whole-cell <br /> This decrease contrast significant increase     extracts, presence IPC (34:1, 778m/z) was <br /> the SM species 703 m/z (C16:0) minor increases          observed TbnSMase cKO (42h&#226;&#8364;&#8220;tet) P100 fraction <br /> the SM species 677 m/z (C14:0) 731 m/z (C18:0).            inositol-phosphate containing phospholipids (parents <br /> Fully characterization choline&#226;&#8364;&#8220;phosphate contain-          241 m/z) (Fig. S7C). IPC normally present in <br /> ing phospholipids T. brucei shown Table S1 (Rich-        bloodstream form T. brucei, procyclic and <br /> mond et al., 2010).                                               stumpy form parasites (G&#195;&#188;ler et al., 2008; Kabani et al., <br />   In contrast, negative ion survey scan P100           2009). <br /> subcellular membrane fraction wild-type cells (Fig. 5C)          Taken together, set data clearly shows an <br /> only showed major species 771 m/z, identified           increase SM species organelle level the <br /> ES-MS/MS daughter ion fragmentation (Fig. S7A)                 lack nSMase activity. The corresponding decrease in <br />  <br />                                                           &#194;&#169; 2010 Blackwell Publishing Ltd, Molecular Microbiology, 76, 1461&#226;&#8364;&#8220;1482 <br />                                                                                            Neutral sphingomyelinase Trypanosoma brucei 1469 <br />  <br /> PC explainable lack choline&#226;&#8364;&#8220;phosphate               phase-light structures (Fig. 6C, D N), larger <br /> product SM hydrolysis TbnSMase ER,             flagellar pocket confirmed original, smaller <br /> reducing flux Kennedy pathway                daughter flagellar pocket exocytically active prior to <br /> novo PC biosynthesis. Alternatively, likely concur-        cytokinesis (Allen et al., 2003). Associated &#226;&#8364;&#732;Big- <br /> rently, SM synthesized expense deplet-          Eye&#226;&#8364;&#8482; phenotype noticeable increase pre- <br /> ing pool PC SM synthases.                            cytokinesis 2K2N cells. This lack proper cell-division of <br />                                                                    TbnSMase cKO (-tet) confirmed FACS analy- <br />                                                                    sis, showed significant increase proportion <br /> Loss nSMase activity leads a <br />                                                                    G2/M, corresponding decrease G0/G1 cells <br /> &#226;&#8364;&#732;Big-Eye&#226;&#8364;&#8482;-like phenotype <br />                                                                    (Fig. 6P), implying cell cycle arrest G2/M. <br /> The TbnSMase cKO cell line grown absence                    To ensure enlargement flagellar pocket in <br /> tetracycline exhibited phase light structure poste-       cell line similar previously described &#226;&#8364;&#732;Big-Eye&#226;&#8364;&#8482; <br /> rior end, significantly increases volume time.       defect endocytosis phenotype, previously well- <br /> This enlarged structure eventually major               characterized proteins sorted endosomal <br /> proportion cell volume trypanosomes                 network investigated. <br /> become rounded time (Fig. 6B&#226;&#8364;&#8220;D) compared with <br /> wild-type cells (Fig. 6A). Concurrently, proportion of <br />                                                                    Does loss nSMase affect protein trafficking? <br /> these abnormal cells increased time at <br /> 42 h, ~25% nSMase cKO cells exhibit morphology,            As diagnostic marker membrane protein trafficking in <br /> increasing 50% cells 72 h, compared         bloodstream form T. brucei, processing p67 in <br /> &lt;1% wild-type cells. This morphology similar          TbnSMase cKO cells examined. It docu- <br /> &#226;&#8364;&#732;Big-Eye&#226;&#8364;&#8482; phenotype described previously RNA              mented terminal endosomal marker p67, type I <br /> interference (RNAi) clathrin heavy chain T. brucei       membrane protein, undergoes complex processing in <br /> bloodstream forms (Allen et al., 2003). Allen col-             bloodstream form trypanosomes (Brickman Balber, <br /> leagues used immunogold a-VSG electron microscopy               1994). From synthesis ER-associated 100 kDa <br /> confirm phase light structure flagellar           glycoprotein (gp100), p67 N-glycosylated a <br /> pocket, massively enlarged small flask-shaped             150 kDa species (gp150) Golgi, trans- <br /> structure usually seen wild-type cells.                         ported proteolytically cleavage lysosome, <br />    Electron microscopy TbnSMase cKO grown                   forming firstly gp75 subsequently gp28, gp32, gp42. <br /> under permissive conditions showed highly                 Using method Alexander et al. (2002), process- <br /> enlarged flagellar pocket (Fig. 6H&#226;&#8364;&#8220;J), compared               ing monitored. Thus, wild-type TbnSMase <br /> with wild-type cells (Fig. 6E&#226;&#8364;&#8220;F, EM micro-             cKO (42h&#226;&#8364;&#8220;tet) cells pulse labelled 1 h [35S]- <br /> graphs Fig. S8).                                                methionine immunoprecipitation 0 2 h post <br />    This &#226;&#8364;&#732;Big-Eye&#226;&#8364;&#8482; enlargement flagellar pocket          pulse, using monoclonal a-p67 (mAb280 &#226;&#8364;&#8220; gift J. <br /> be attributed defect membrane transport            Bangs). Analysis [35S]-methionine-labelled immuno- <br /> the balance exocytosis endocytosis disturbed.           precipitates SDS-PAGE autoradiography revealed <br />    The decrease fluid endocytosis TbnSMase cKO               proteolytic processing p67 gp75 <br /> (42h&#226;&#8364;&#8220;tet) cells (Fig. 6L&#226;&#8364;&#8220;N), compared wild-type               intermediate end products gp32 gp42 was <br /> cells (Fig. 6K), confirmed FITC-BSA uptake                unaffected lack TbnSMase (Fig. 8A, compare <br /> by immunofluorescence. The FITC-BSA signal                  lanes 1 2, 3 4, loading controls Fig. S9). <br /> observed frequently endosomal network occa-                Similarly, trafficking lumenal proteins in <br /> sionally flagellar pocket wild-type cells (Fig. 6K),     TbnSMase cKO cells investigated examining the <br /> but significantly frequently associated just      processing trypanopain, cathepsin L orthologue. In <br /> flagellar pocket TbnSMase cKO (42h&#226;&#8364;&#8220;tet) cells;         wild-type bloodstream form trypanosomes, trypanopain <br /> a fraction endocytosed 30 min (Fig. 6O, lanes             synthesized 53 kDa pro-protein rapidly <br /> 4&#226;&#8364;&#8220;6), compared significant portion FITC-BSA          fully processed acidic environment endoso- <br /> being endocytosed wild-type cells (Fig. 6O,                 mal network 44 kDa mature active enzyme <br /> lanes1&#226;&#8364;&#8220;3) (Fisher&#226;&#8364;&#8482;s exact test, P = 0.056, n = 40 fields).         (Caffrey et al., 2001), similarly observed ourselves <br /> Extreme morphological &#226;&#8364;&#732;Big-Eye&#226;&#8364;&#8482; trypanosomes typically             using Western blot analysis a-trypanopain Ab <br /> demonstrated specific FITC staining longer            (a gift J. Bangs) (Fig. 8B, lane 1). However, TbnS- <br /> camera exposures (Fig. 6H).                                        Mase cKO (42h&#226;&#8364;&#8220;tet) cells, proteolytic processing of <br />    As clathrin RNAi &#226;&#8364;&#732;Big-Eye&#226;&#8364;&#8482; trypanosomes,               trypanopain incomplete 2&#226;&#8364;&#8220;5% immature <br /> TbnSMase cKO (42h&#226;&#8364;&#8220;tet) cells seen                   53 kDa pro-enzyme detectable Western blot (Fig. 8B, <br /> undergoing mitosis presented                lane 2). <br />  <br /> &#194;&#169; 2010 Blackwell Publishing Ltd, Molecular Microbiology, 76, 1461&#226;&#8364;&#8220;1482 <br />  1470 S. A. Young T. K. Smith &#228;&#352;? <br />  <br />                                                                 Fig. 6. Morphological phenotype the <br />                                                                 TbnSMase conditional knockout. <br />                                                                 A&#226;&#8364;&#8220;D. Merged DAPI-fluorescence-DIC images <br />                                                                 wild-type (A), TbnSMase cKO cells <br />                                                                 grown absence tetracycline 42 h <br />                                                                 (B&#226;&#8364;&#8220;D). <br />                                                                 E&#226;&#8364;&#8220;J. Transmission electron micrographs taken <br />                                                                 wild-type bloodstream T. brucei (E&#226;&#8364;&#8220;G) and <br />                                                                 TbnSMase cKO cells grown absence of <br />                                                                 tetracycline 42 h (H&#226;&#8364;&#8220;J). Insert (I) shows <br />                                                                 close plasma membrane, <br />                                                                 microtubles fussy VSG coat. FL, <br />                                                                 flagellum; FP, flagellar pocket; K, kinetoplast; <br />                                                                 M, mitochondria; G, glycosome; The asterisk <br />                                                                 (J) indicates enlarged concentric <br />                                                                 membrane structure the <br />                                                                 endosomal network. <br />                                                                 K&#226;&#8364;&#8220;O. Wild-type (K) TbnSMase cKO cells <br />                                                                 grown absence tetracycline 42 h <br />                                                                 (L&#226;&#8364;&#8220;N) incubated FITC-BSA as <br />                                                                 described Experimental procedures; images <br />                                                                 merged FITC, DAPI-fluorescence and <br />                                                                 DIC. (O) Evaluation endocytic function <br />                                                                 wild-type (1&#226;&#8364;&#8220;3) nSMase cKO (Tet-42h) <br />                                                                 (4&#226;&#8364;&#8220;6) trypanosomes assessed the <br />                                                                 incidence FITC-BSA signal solely <br />                                                                 endosomal region (1 4), dual <br />                                                                 endosomal flagellar pocket staining (2 <br />                                                                 5) flagellar pocket staining (3 <br />                                                                 6). <br />                                                                 P. FACS analysis TbnSMase cKO cells <br />                                                                 grown absence tetracycline 0, 24 <br />                                                                 42 h. Cells fixed, stained with <br />                                                                 propidium iodide analysed flow <br />                                                                 cytometry described Experimental <br />                                                                 procedures. <br />  <br />  <br />  <br />  <br />                                      &#194;&#169; 2010 Blackwell Publishing Ltd, Molecular Microbiology, 76, 1461&#226;&#8364;&#8220;1482 <br />                                                                                           Neutral sphingomyelinase Trypanosoma brucei 1471 <br />  <br />                                                                                      Fig. 7. Investigation endosomal <br />                                                                                      network. Wild-type cells (A) TbnSMase <br />                                                                                      cKO cells grown absence of <br />                                                                                      tetracycline 42 h (B&#226;&#8364;&#8220;D) probed with <br />                                                                                      anti-trypanopain. Images TRITC, <br />                                                                                      DAPI-fluorescence DIC. <br />  <br />  <br />  <br />  <br />    Examination TbnSMase cKO cells (42h&#226;&#8364;&#8220;tet)              ~80 kDa glycoprotein species (Fig. S11), indicating no <br /> subcellular localization trypanopain, typically           drastic defect processing. <br /> colocalizes p67 lysosome (Caffrey et al., 2001), <br /> shows endosomal staining <span id='am-476' about='xsp:pattern' typeof='owl:Thing'>pattern</span> greatly <br />                                                                    Does loss nSMase affect VSG trafficking? <br /> enlarged (Fig. 7B&#226;&#8364;&#8220;D Fig. S10) compared the <br /> corresponding wild-type staining pattern (Fig. 7A). In             To investigate effect, any, exocytosis newly <br /> addition, extension endosomal staining congru-           synthesized VSG reaching flagellar pocket, cells were <br /> ent expansion late endosomal network           labelled [35S]-methionine 1 h, total <br /> flagellar pocket. This indicates endosomal                protein soluble VSG (sVSG) preparations (obtained <br /> network malfunctioning, probably result lack         GPI-PLC cleavage cell surface VSG mild <br /> flux endocytic pathways.                               hypotonic lysis) separated SDS-PAGE [35S]- <br />    This malfunctioning endosomal network                labelled proteins detected autoradiography. Note: <br /> account unusual and, far know, novel              equal cell equivalents loaded gel, to <br /> structures observed electron microscopy            &#226;&#8364;&#732;Big-Eye&#226;&#8364;&#8482; 2K2N pre-cytokinesis phenotype, actual <br /> sections TbnSMase cKO cells (42h&#226;&#8364;&#8220;tet) (Fig. 6I          total protein 1.4 times higher TbnS- <br /> J Fig. S8F G) consisting concentric &#226;&#8364;&#732;onion-like&#226;&#8364;&#8482;        Mase cKO (42h&#226;&#8364;&#8220;tet) determined Bradford and <br /> vesicular membranes.                                               shown Fig. 8C (lanes 1 2). The total of <br />    Another soluble protein                    [35S]-methionine incorporation protein (excluding <br /> endocytic pathway, protein disulphide isomerise 2 (PDI-2)          VSG) cell lines similar (Fig. 8C, lanes 1 <br /> (Rubotham et al., 2006), similarly examined            2). Thus 42 h time point, TbnSMase cKO cells <br /> aberrant processing. As 55 kDa polypeptide, PDI-2             essentially viable based ability synthesize <br /> subsequently heavily glycosylated 80 kDa glyco-            protein level comparable wild-type cells. <br /> protein Golgi, ultimately colocalizes p67          However, clear proportion newly synthe- <br /> the terminal endosomes. Western blot analysis wild-             sized ([35S]-methionine labelled) VSG exported cell <br /> type TbnSMase cKO (42h&#226;&#8364;&#8220;tet) cells a-PDI-2 (a              surface, i.e. radiolabelled sVSG, drastically reduced <br /> gift Derek Nolan) revealed equal amounts               TbnSMase cKO (42h&#226;&#8364;&#8220;tet) cells, compared <br />  <br /> &#194;&#169; 2010 Blackwell Publishing Ltd, Molecular Microbiology, 76, 1461&#226;&#8364;&#8220;1482 <br />  1472 S. A. Young T. K. Smith &#228;&#352;? <br />  <br />  <br />  <br />  <br /> Fig. 8. Phenotypic analysis protein trafficking nSMase conditional knockout. <br /> A. Wild-type TbnSMase cKO cells grown absence tetracycline 42 h pulse labelled [35S]-methionine 1 h (lanes 1 <br /> and 3 respectively) chased 2 h (lanes 2 4 respectively) prior IP anti-P67 described Experimental procedures. <br /> The resulting IP [35S]-proteins separated 10% SDS-PAGE gel, prior Coomassie blue staining (Fig. S6) fluorography. The <br /> various maturation sizes P67 highlighted. <br /> B. Anti-trypanopain Western blot analysis wild-type (lane 1) TbnSMase cKO cells grown absence tetracycline 42 h (lane 2) <br /> shows incomplete endosomal processing. The maturation sizes trypanopain highlighted. <br /> C. Wild-type (lanes 1 3) TbnSMase cKO cells grown absence tetracycline 42 h (lanes 2 4) labelled with <br /> [35S]methionine 1 h. Total [35S]-protein (lanes 1 2) [35S]-sVSG (soluble VSG-obtained cell surface parasite) (lanes 3 <br /> and 4) prepared described Experimental procedures separated 10% SDS-PAGE gel, visualized Coomassie blue <br /> or fluorography. The black arrow (lane 1) refers protein longer synthesized TbnSMase cKO (-tet) cells (lane 2). <br /> D. Wild-type (lanes 1 2) TbnSMase cKO cells grown absence tetracycline 42 h (lanes 3 4) labelled with <br /> [3H]myristate 1 h absence (lanes 1 4) presence cycloheximide (60 mg ml-1) (lanes 2 3) described Experimental <br /> procedures. Densitometry (ImageJ) [3H]myristate-mfVSG (membrane form VSG, processes intact GPI-anchor) signals normalized <br /> to 100% wild-type cells absence cycloheximide. Lower panel shows VSG portion gel stained Coomassie blue <br /> as loading control. <br />  <br />  <br />  <br />  <br />                                                                   &#194;&#169; 2010 Blackwell Publishing Ltd, Molecular Microbiology, 76, 1461&#226;&#8364;&#8220;1482 <br />                                                                                             Neutral sphingomyelinase Trypanosoma brucei 1473 <br />  <br /> with Coomassie stained sVSG (Fig. 8C, lanes 3              currently, rate SM synthesis continues unabated <br /> 4) a-VSG221 Western blot analysis (Fig. S9B),              resulting increased SM levels by-product DAG, <br /> thus conclude rate newly synthesized           utilized catabolized. <br /> VSG getting cell surface drastically                  The depletion PC mitochondrion comes <br /> reduced TbnSMase cKO (42h&#226;&#8364;&#8220;tet) cells. The                   cells starved choline <br /> protein bands sVSG similar amounts            and/or choline&#226;&#8364;&#8220;phosphate local pool choline is <br /> of [35S]-methionine incorporation, showing protein            tied SM. The absence nSMase ER <br /> synthesis taking place. The exception       means choline&#226;&#8364;&#8220;phosphate liberated, which <br /> ~80 kDa protein (Fig. 8C, lane 1, black arrow), corre-         utilized synthesize PC Kennedy <br /> sponding protein band identified mass finger-          pathway. As parasite auxotrophic choline, it <br /> printing possibly HSP-70 PDI-2, interestingly,        source choline containing phospholipids the <br /> both involved protein folding.                     host environment, probably primarily components <br />     To investigate defect VSG synthesis,           low-density-lipoprotein (LDL). However, the <br /> subsequent exocytosis, wild-type TbnSMase                 nSMase cKO grown permissive conditions, it <br /> cKO (42h&#226;&#8364;&#8220;tet) cell lines labelled [3H]-myristate.        greatly reduced endocytic flux, ultimately prevent- <br /> As expected wild-type cells mature membrane form            ing catabolism pre-exisiting endosomal SM <br /> GPI-anchored VSG (mfVSG) [3H]-myristate labelled               PC, any, freshly endocytosed extracellular SM <br /> (Fig. 8D, lane 1). The corresponding [3H]-myristate                PC. <br /> labelled mfVSG TbnSMase cKO (42h&#226;&#8364;&#8220;tet) cells                    A consequence shift choline usage the <br /> however significantly reduced (~12%) (Fig. 8D, lane            mitochondrial associated SM synthase ultimately runs out <br /> 4). This significant reduction prompted assessment           normal substrate, PC, starts utilize PI instead, <br /> the myristate exchange GPI anchor pre-existing           giving rise unusual IPC formation blood- <br /> VSG, takes place endocytosis. As previously           stream form T. brucei. This pool PI ER derived <br /> shown, presence cycloheximide (to inhibit protein        novo synthesized inositol (Martin Smith, <br /> synthesis), myristate exchange wild-type cells              2006b). This implies mitochondrial SM synthase <br /> shows residual ~30% (Fig. 8D, lane 2) [3H]-myristate             utilize PC PI substrates synthesis <br /> incorporation (Buxbaum et al., 1996). However,              SM IPC respectively. However, bloodstream <br /> TbnSMase cKO (42h&#226;&#8364;&#8220;tet) cells incorporation                  form parasite prefers utilize PC, uses <br /> reduced mere ~3% compared wild-type level.               PI PC available. This contrast procyclic cells <br /> Despite significantly reduced myristate             substantial amounts IPC observed in <br /> exchanging happening TbnSMase cKO (42h&#226;&#8364;&#8220;tet) cells,              purified mitochondria, despite ample PC avail- <br /> it approximately ratio myristate exchange        able (G&#195;&#188;ler et al., 2008). <br /> total myristylated mfVSG observed wild-type cells, i.e.            An alternative possibility observed IPC formation <br /> 3&#226;&#8364;&#8220;12% 30&#226;&#8364;&#8220;100% respectively. This indicates                novo synthesized ceramide translocated <br /> despite rate endocytoses exocytoses mfVSG            Golgi resident SM synthase (SLS4), <br /> being significantly reduced ratio (relationship)          known capable utilizing PC PI (Mina <br /> maintained.                                                        et al., 2009), starved PC utilizes different pool of <br />                                                                    PI, synthesized Golgi extracellular <br />                                                                    inositol (Martin Smith, 2006a). The transition PC to <br /> Discussion                                                         PI utilization substrates mitochondrial SM syn- <br /> The essentiality TbnSMase clearly shows            thase(s) occurs bloodstream long-slender to <br /> de novo synthesis ceramide compensate            stumpy form differentiation, IPC formation clearly <br /> loss ceramide formation SM catabolism ER.            observed dramatic upregulation mitochon- <br />    The nSMase cKO trypanosomes grown permis-                 drial associated SM synthase (SLS1) (Kabani et al., <br /> sive conditions shows significant increase intra-        2009). These observations imply stumpy and <br /> celluar SM levels, ER and/or mitochondria        procyclic form parasites significantly decreased <br /> where SM synthase present. In conjunction            rate ceramide formation ER, little, if <br /> excess SM significant decrease PC, corre-            any, SM nSMase catabolize, flux of <br /> sponding increase intracelluar DAG. The observed                post-Golgi sorting protein destined recycling <br /> increase DAG species diacyl chain length            endosomes flagellar pocket, dependent <br /> as depleted PC species (C38:5 C40:5) implies               ceramide ER, severely reduced. <br /> that novo synthesis PC Kennedy pathway              Collectively, results allow propose an <br /> has slowed, synthesis DAG not,                  updated model endocytic exocytic pathways in <br /> resulting excess DAG. Alternatively, likely con-        bloodstream form T. brucei (Fig. 9A), expla- <br />  <br /> &#194;&#169; 2010 Blackwell Publishing Ltd, Molecular Microbiology, 76, 1461&#226;&#8364;&#8220;1482 <br />  1474 S. A. Young T. K. Smith &#228;&#352;? <br />  <br />  <br />  <br />  <br /> Fig. 9. Schematic pathways involved endocytosis exocytosis bloodstream form T. brucei: wild-type (A) nSMase cKO <br /> grown permissive conditions (B). The difference width arrows (A) (B) depicts relative changes flux the <br /> same vesicular trafficking pathway wild-type cells compared nSMase cKO grown permissive conditions. Dashed arrows <br /> indicate putative pathways. The enlarged flagellar pocket late endosomes representative &#226;&#8364;&#732;Big-Eye&#226;&#8364;&#8482; phenotype observed the <br /> nSMase cKO grown permissive conditions (B). Annotated cellular structures: EE, early endosomes; ER, endoplasmic reticulum; FP, <br /> flagellar pocket; G, Golgi; L, lysosome; LE, late endosomes; M, mitochondria; MAM, mitochondrion associated membrane; PM, plasma <br /> membrane; RE, recycling endosomes; FL, flagellum. The various types vesicles numbered follows: 1. Endocytic clathrin-coated <br /> vesicles flagellar pocket early endosomes, carrying mixture endocytosed nutrients, including LDL particles transferrin bound <br /> to GPI-anchored transferrin receptor, recycling pre-existing cell surface GPI-anchored VSG, checked undergoing myristate <br /> exchange. 2. Clathrin-coated vesicles early endosomes recycling endosomes carrying recycling pre-existing cell surface GPI-anchored <br /> VSG. 3. Clathrin-coated vesicles early endosomes late endosomes, carrying mixture endocytosed nutrients, including LDL <br /> particles transferrin bound GPI-anchored transferrin receptor. 4. Clathrin-coated vesicles late endosomes recycling endosomes, <br /> carrying recycled GPI-anchored transferrin receptor, free transferrin, newly synthesized membrane-bound protein destined for <br /> the flagellar pocket, newly synthesized GPI-anchored proteins, i.e. VSG. 5. Non-clathrin-coated, sphingomylein-enriched recycling <br /> endosomes destined flagellar pocket, contains recycling GPI-anchored transferrin receptor, recycling VSG newly synthesized <br /> VSG. 6. Non-clathrin-coated, SM-enriched vesicles trans-Golgi cisterna recycling endosomes, containing newly synthesized <br /> VSG possibly GPI-anchored proteins, ultimately destined flagellar pocket cell surface. 7. Clathrin-coated vesicles <br /> from Golgi late endosomes, containing newly synthesized proteins targeted to lysosome, i.e. p67 and <br /> trypanopain, recycling endosomes flagellar pocket, i.e. cell surface proteins VSG. 8. Clathrin-coated vesicles <br /> from late endosomes lysosome, containing mixture newly synthesized lysosomal proteins, i.e. p67 trypanopain, and <br /> endocytosed nutrients, i.e. LDL particles transferrin. 9. CopII vesicles ER Golgi, carrying newly synthesized proteins <br /> destined endocytic pathway. 10. Putative non-CopII vesicles going ER Golgi, possibly carrying lipids, e.g. <br /> ceramide PC. 11. Putative SM-enriched vesicle trafficking VSG directly Golgi flagellar pocket. <br />  <br />  <br />  <br />  <br />                                                                   &#194;&#169; 2010 Blackwell Publishing Ltd, Molecular Microbiology, 76, 1461&#226;&#8364;&#8220;1482 <br />                                                                                           Neutral sphingomyelinase Trypanosoma brucei 1475 <br />  <br /> nation effect pathways caused                   subset Cop II vesicles selectively transporting GPI- <br /> the nSMase cKO grown permissive conditions                   anchored cargo, e.g. VSG, ER Golgi (Sevova <br /> (Fig. 9B).                                                         Bangs, 2009), vesicles contain the <br />    Let discuss model bloodstream form             ceramide PC SM synthesis. <br /> the parasite. PC exclusively novo synthesized              The model suggests post-Golgi sorting proteins <br /> Kennedy pathway (Gibellini et al., 2009) ceramide              takes place trans-Golgi network clathrin- <br /> are utilized mitochondrial resident SM synthase            coated vesicles late endosome (Fig. 9, route 7), or <br /> form SM. This SM translocated ER,           SM-enriched vesicles recycling endosomes (Fig. 9, <br /> mitochondria-associated membrane fraction (Bionda                  route 6). <br /> et al., 2004; Achleitner et al., 1999), SM                  Proteins p67 trypanopain targeted be <br /> encounters nSMase, forming ceramide choline&#226;&#8364;&#8220;               translocated lysosome (Fig. 9, route 8) unaf- <br /> phosphate (Vacaru et al., 2009). This ER pool cera-             fected lack SM formation Golgi the <br /> mide travels PC ER Golgi             nSMase cKO cells grown permissive conditions. <br /> (Fig. 9, route 9) (Testerink et al., 2009),           Note, processing sorting enlarged late endo- <br /> trans-Golgi cisterna, different SM synthase, SLS4                some proteins affected, i.e. trypanopain, thus <br /> (Tb09211.1000) (Sutterwala et al., 2008), utilizes           likely p67 processing trypanopain, normally <br /> lipid precursors reform SM, subsequently               lysosomal, partially happens enlarged late <br /> used post-Golgi processing GPI-anchored pro-                endosomes. <br /> teins, i.e VSG. This similar situation yeast,            In late endosomes lysomal targeted proteins are <br /> where IPC, formed Golgi ceramide novo               joined endocytosed nutrients, i.e. transferrin LDL, <br /> synthesies ER PI, influences GPI-anchored               previously sorted recycling VSG <br /> protein trafficking (Watanabe et al., 2002). This explains         early endosomes (reviewed Overath Eng- <br /> why inhibition ceramide novo pathway                  stler, 2004) (Fig. 9, route 3), translocated lyso- <br /> myrocin, lethal did affect transport GPI-          (Fig. 9, route 8). <br /> anchored VSG (Sutterwala et al., 2007).                               Other cell surface proteins, i.e. invariant surface glyco- <br />    In case nSMase cKO grown permissive            protein 65 recycling transferrin receptor, are <br /> conditions, decreasing levels nSMase activity directly          sorted recycling endosomes (Fig. 9, route 4) join <br /> cause lack ceramide ultimately lack PC               recycling VSG newly synthesized VSG on <br /> (discussed eariler) ER, subsequently reduces          route flagellar pocket (Engstler et al., 2004). <br /> the ceramide travelling Golgi (Fig. 9,               The alternative post-Golgi sorting pathway utilizes <br /> route 9). The subsequent lack SM synthesis Golgi         SM-enriched vesicles carry newly synthesized VSG, <br /> decreases rate (flux) SM-enriched vesicles            recycling endosomal network (Fig. 9, route <br /> containing newly synthesized VSG translocated            6) putatively suggested, directly the <br /> the recycling endosomes (Fig. 9, route 6),           flagellar pocket (Fig. 9, route 11). <br /> the flagellar pocket (Fig. 9, route 5). This subsequently             Here recycling endosomes likely <br /> decreases flux exiting late endosomes (Fig. 9,             place myristate exchange pre-existing newly <br /> route 4), causing enlarge, reduces                 synthesized GPI-anchored VSG potentially other <br /> endocytic rate pre-exisitng cell surface VSG (Fig. 9,           GPI-anchored proteins, i.e. transferrin receptor takes <br /> route 1), early endosomes recycling endo-          place (Buxbaum et al., 1996). <br /> somes (Fig. 9, route 2).                                              The high flux SM-enriched recycling endosomes <br />    As organisms general question arises,            delivers cargo flagellar pocket (Fig. 9, route 5), <br /> SM translocated ER Golgi? The               allowing SM recycled endocytosis (Fig. 9, <br /> possible answer transportation SM              route 1), early late endosomes lyso- <br /> the ER Golgi detrimental cis-Golgi            (Fig. 9, routes 3 8), catabolized <br /> cisterna and/or enzymes therein, i.e. glycosidases             putative acidic SMase (Tb 927.4.1120), whose <br /> and glycosyltransferases. Alternatively, SM suitable        activity shown work SM PC containing <br /> for Cop II vesicle formation (Fig. 9, route 9), used      LDL (Coppens et al., 1995). <br /> for protein trafficking ER Golgi (Hughes                  Based model results, flux <br /> Stephens, 2008). Another previously proposed possibility           endosomal pathway highly dependent <br /> is lipids ceramide PC transported          high rates flux avenues of <br /> the Golgi CopII independent vesicles (Funato               post-Golgi sorting, i.e. late endosome recy- <br /> Reizman, 2001) (Fig. 9, route 10), SM                cling endosomes. To extrapolate model further, if <br /> cannot translocated using vesicles unknown.            reduction flux post-Golgi <br /> Interestingly, T. brucei recently shown use          sorting recycling endosomes, ultimately the <br />  <br /> &#194;&#169; 2010 Blackwell Publishing Ltd, Molecular Microbiology, 76, 1461&#226;&#8364;&#8220;1482 <br />  1476 S. A. Young T. K. Smith &#228;&#352;? <br />  <br /> flagellar pocket (exocytosis) (Fig. 9B, routes 6 5),       The TbnSMase ORF excised using HindIII PacI, and <br /> in nSMase cKO VSG RNAi (Sheader et al.,                subsequently ligated tetracycline-inducible expres- <br /> 2005; Smith et al., 2009), flux late          sion vectors pLew82 pLew100 (Wirtz et al., 1999) the <br />                                                                   HindIII PacI restriction sites. Plasmid DNA prepared <br /> endosome (Fig. 9B, routes 3 4) affected; in <br />                                                                   linearization NotI subsequently precipitated with <br /> turn causing decrease rate protein trafficking        sodium acetate/ethanol redissolved sterile water a <br /> through exocytic pathway ultimately                   final concentration 2 mg ml-1, ready transfection. <br /> endocytic pathway. This lack protein trafficking the <br /> recycling endosomes causes enlargement late <br /> endosome. Thus, new material entering flagellar           Construction T. brucei gene replacement cassettes <br /> pocket processed leading enlargement              The 5&#226;&#8364;&#178; 3&#226;&#8364;&#178; untranslated regions (UTRs) immediately adja- <br /> of flagellar pocket, characteristically known        cent nSMase ORF amplified T. brucei <br /> &#226;&#8364;&#732;Big-Eye&#226;&#8364;&#8482; phenotype.                                              genomic DNA using KOD Hot Start DNA polymerase. The <br />    So, high endocytic exocytic rates blood-            primers 5&#226;&#8364;&#178;-ATAAGTAAGCGGCCGCGCGTTTGGGCTGTT <br /> stream form T. brucei dependent high flux             CTCGTT-3&#226;&#8364;&#178; 5&#226;&#8364;&#178;-CGTTTAAACTTACGGACCGTCAAGCTT <br />                                                                   TTTTGAGTTTAATTCAGTGGG-3&#226;&#8364;&#178; used 5&#226;&#8364;&#178; UTR, <br /> SM-enriched vesicles Golgi recycling <br />                                                                   amplifying expected 468 bp product, primers <br /> endosomes flagellar pocket/cell                 5&#226;&#8364;&#178;-tgacggtccgtaagtttaaacggATCCAGAGGATTGCTTCATCAT <br /> surface. This explains difference significantly        CATG-3&#226;&#8364;&#178; 5&#226;&#8364;&#178;-ATAAGTAAGCGGCCGCTCCCCTCTGCG <br /> higher endocytic exocytic fluxes mammalian             TAGTGGTAAAATA-3&#226;&#8364;&#178; resulted correct 298 bp product <br /> bloodstream form parasite compared            3&#226;&#8364;&#178; UTR. These amplified products used a <br /> the procyclic insect form parasites.                       subsequent knitting PCR, resulting 744 bp product in <br />    In summary, T. brucei nSMase recombi-             5&#226;&#8364;&#178; UTR joined 3&#226;&#8364;&#178; UTR short <br />                                                                   complementary BamHI-HindIII linker region contained within <br /> natly expressed shown catalytically active. A <br />                                                                   described primers (italics). This PCR product ligated <br /> high-throughput assay developed allowing future          pGEM-5Zf(+) (Promega) NotI sites (underlined in <br /> high-throughput screening. Creation conditional              original primers) followed insertion hygro- <br /> knockout bloodstream parasite allowed               mycin (HYG) puromycin (PAC) resistance genes between <br /> genetic validation drug target, phenotype         cut BamHI HindIII restriction sites. Plasmid DNA was <br /> which given new insight highly active endocytic      prepared using QIAprep Miniprep Plasmid Kit (Qiagen), <br /> and exocytic pathways bloodstream T. brucei.                   digested NotI precipitation sodium <br />                                                                   acetate/ethanol redissolved sterile water final <br />    This work clearly demonstrates decrease the <br />                                                                   concentration 2 mg ml-1, ready transfection. <br /> rate deposition newly synthesized VSG cell <br /> surface causes corresponding decrease recycling <br /> of VSG, allowing trypanosome maintain                   Cultivation genetic modification T. brucei <br /> optimum cell density crucial VSG protective coat. <br />                                                                   Bloodstream form T. brucei strain 427, pre- <br /> Thus, new possible strategy fight African        viously modified express T7 polymerase the <br /> trypanosomasis drugs selectively interfere        tetracycline repressor protein (Wirtz et al., 1999), referred <br /> with post-Golgi transport VSG. To end          wild-type cells convenience. Cells grown <br /> currently investigating nSMase inhibitors potential anti-      HMI-9 media supplemented G418 (2.5 mg ml-1), at <br /> trypanosomal drugs.                                               37&#194;&#176;C 5% CO2 described (Wirtz et al., 1999; <br />                                                                   Chang et al., 2002; Roper et al., 2002). Transformation con- <br />                                                                   ditions subsequent drug selection described <br />                                                                   (Burkard et al., 2007). Addition tetracycline to <br /> Experimental procedures                                           media required final concentration of <br />                                                                   1 mg ml-1. For tetracycline-free experiments, Tet system- <br /> Cloning T. brucei nSMase gene ligation            approved fetal calf serum (Clontech) used cells were <br /> pLew vectors                                                      washed times tetracycline-free HMI-9 resus- <br />                                                                   pended tetracycline-free media 5 &#194;&#165; 104 <br /> A putative nSMase gene identified T. brucei <br />                                                                   cells ml-1. Cells counted day passaged <br /> genome database (Tb927.5.3710 &#226;&#8364;&#8220; Sanger Institute). <br />                                                                   density greater 7 &#194;&#165; 105 cells ml-1 <br /> The ORF amplified T. brucei strain 427 <br />                                                                   (normally second day). <br /> genomic DNA using forward reverse primers <br /> 5&#226;&#8364;&#178;-GAGGAGAAGCTTATGGCTGCAGAGATAACTG-3&#226;&#8364;&#178; and <br /> 5&#226;&#8364;&#178;-TGCTTAATTAATTTACAACCATTACCTTTATGC-3&#226;&#8364;&#178; con-                      Expression T. brucei nSMase E. coli <br /> taining HindIII PacI restriction sites respectively <br /> (underlined). A band expected size c. 1.7 kb        The nSMase ORF amplified TbnSMase-TOPO <br /> amplified using KOD Hot Start DNA polymerase (Merck Bio-          plasmid KOD Hot Start DNA polymerase using the <br /> sciences Ltd), purified using QIAquick PCR purification kit     primers 5&#226;&#8364;&#178;-GAGGGATCCATGGCTGCAGAGATAACTGTTC- <br /> (Qiagen) cloned pCR-Blunt II TOPO (Invitrogen).          3&#226;&#8364;&#178; 5&#226;&#8364;&#178;-TGCCCCGGGTCATTTACAACCATTACCTTTATG- <br />  <br />                                                           &#194;&#169; 2010 Blackwell Publishing Ltd, Molecular Microbiology, 76, 1461&#226;&#8364;&#8220;1482 <br />                                                                                               Neutral sphingomyelinase Trypanosoma brucei 1477 <br />  <br /> 3&#226;&#8364;&#178;, resulting single band c. 1.7 kb. The amplified product   10 mM MgCl2, 0.5% glycerol varying amounts SM <br /> was purified using QIAquick PCR purification kit (Qiagen)          (presented mixed micelles 0.1% Triton-X 100). Pro- <br /> prior blunt-end cloning pCR-Blunt II-TOPO (Invitrogen)       duction choline phosphate SM detected inclu- <br /> for sequencing. The reading frame digested               sion standard Amplex red assay components final <br /> ligated pGEX-6P-1 vector (GE Healthcare) using          assay volume 200 ml. Reaction mixtures incubated at <br /> BamHI SmaI sites (underlined) sequence con-              30&#194;&#176;C 1 h fluorescence product generated mea- <br /> firmed. This construct transformed E. coli C43 (DE3)        sured kinetically (Excitation 56 0nm, Emission 587 nm). T. <br /> competent cells expression. A single colony used          brucei bloodstream form washed membranes prepared as <br /> inoculate 5 ml Luria&#226;&#8364;&#8220;Bertani media containing 100 mg ml-1         containing estimated 35 mg total protein <br /> ampicillin 0.5% glucose (w/v) grown 37&#194;&#176;C over-            assayed nSMase activity identical manner with <br /> night. The 5 ml added 500 ml overnight express             100 mM SM (presented mixed micelles 0.1% Triton-X <br /> instant TB medium (Merck Biosciences Ltd) containing                 100). Inhibitor studies trypanosomal membranes <br /> 100 mg ml-1 ampicillin grown 37&#194;&#176;C 4 h,        conducted pre-incubation 500 mM inhibitor <br /> the culture incubated 20 h 25&#194;&#176;C. Cells     15 min, prior addition 100 mM SM, Amplex <br /> harvested centrifugation (1800 g, 15 min) resus-              red 30 min 37&#194;&#176;C. <br /> pended lysis buffer [50 mM Tris pH 8.0, 300 mM NaCl, 10%             Inhibitor studies GST&#226;&#8364;&#8220;TbnSMase conducted by <br /> Glycerol (v/v), 5 mM MgCl2, 1 mM DTT] incubated 37&#194;&#176;C          pre-incubation 50 mM inhibitors 30 min prior addi- <br /> with 0.2 mg ml-1 lysozyme (Sigma) 250 U benzonase                tion 100 mM SM Amplex red 30 min 37&#194;&#176;C. <br /> nuclease (Merck) 30 min. Cells disrupted sonica- <br /> tion (4 &#194;&#165; 60 s) ice cell debris, cells and <br /> ghosts cleared slow-speed centrifugation (14 500 g,          Southern blots <br /> 10 min, 4&#194;&#176;C). Expression identification GST&#226;&#8364;&#8220; <br />                                                                      The nSMase ORF PCR-amplified using ORF <br /> TbnSMase confirmed Western blotting using anti-GST <br />                                                                      primers pLew vectors gel purified labelled <br /> antibodies (Covance) ECL Western detection reagents <br />                                                                      DIG- (Roche). Genomic T. brucei DNA (3 mg) was <br /> (Amersham). Subsequently, E. coli membranes containing <br />                                                                      digested various restriction enzymes. Digestion products <br /> GST&#226;&#8364;&#8220;TbnSMase collected high-speed centrifugation <br />                                                                      separated 0.8% agarose gel transferred to <br /> (100 000 g, 1 h), washed PBS suspended 100 mM <br />                                                                      Hybond-N+ membrane (Amersham), hybridized over- <br /> Tris (pH 7.4), 10 mM MgCl2 20% glycerol stored as <br />                                                                      night ULTRA-HYB (Ambion) 42&#194;&#176;C dig-labelled <br /> aliquots -80&#194;&#176;C. Protein extracted GST&#226;&#8364;&#8220; <br />                                                                      nSMase ORF probe. Stringency washes carried at <br /> TbnSMase membranes solubilizing 1% Triton X-100 prior <br />                                                                      42&#194;&#176;C, consisted washes low stringency for <br /> to determining total protein content using BCA protein <br />                                                                      15 min [2&#194;&#165; SSC (300 mM NaCl, 30 mM sodium citrate), <br /> assay kit (Thermo Scientific). <br />                                                                      0.1% SDS] washes high stringency for <br />                                                                      15 min [0.2&#194;&#165; SSC (30 mM NaCl, 3 mM sodium citrate), <br /> nSMase enzyme assays                                                 0.1% SDS]. Bound probe detected anti-Dig antibody <br />                                                                      following manufacturers instructions. <br /> Sphingomyelinase activity confirmed use radio- <br /> active [3H]-SM, washed membranes (containing 127 mg of <br /> total protein determined using BCA assay kit (Thermo         RNA isolation cDNA synthesis <br /> Scientific) encoding GST&#226;&#8364;&#8220;TbnSMase incubated 100 mM <br />                                                                      Total RNA isolated bloodstream form T. brucei using <br /> Bis-Tris propane (pH 7.5), differing concentrations MgCl2 <br />                                                                      RNeasy mini kit (Qiagen). TbnSMase-specific cDNA <br /> (as stated individual figures), 0.5% glycerol 25 nCi <br />                                                                      generated amplified using specific forward <br /> [choline-methyl-3H]SM (50 Ci mmol-1, ARC) 50 pmol cold <br />                                                                      5&#226;&#8364;&#178;-GAGGAGCATATGGCTGCAGAGATAACTGTTC-3&#226;&#8364;&#178;                        and <br /> SM presented mixed micelles 0.1% Triton-X 100. <br />                                                                      reverse 5&#226;&#8364;&#178;-TGCGGATCCTCAAGCTTCACATAGAGACGCC <br /> Reactions incubated 37&#194;&#176;C 120 min. Hydrolysis of <br />                                                                      TC-3&#226;&#8364;&#178; primers using SuperScript III One step RT-PCR kit <br /> PC assayed identical manner 30 nCi <br />                                                                      Platinum Taq (Invitrogen). As negative control exclude <br /> [3H-choline-methyl]PC (60 Ci mmol-1, ARC) 50 pmol cold <br />                                                                      DNA contamination RNA sample, reverse transcriptase <br /> PC presented mixed micelles 0.1% Triton-X 100. For <br />                                                                      omitted reaction replaced GoTaq poly- <br /> analysis optimum pH activity, pH reaction <br />                                                                      merase (Promega). Inositol-3-phosphate synthase (TbINO1) <br /> buffer altered accordingly. Reactions (50 ml) were <br />                                                                      cDNA amplified using forward reverse <br /> stopped addition 800 ml chloroform:methanol (2:1 <br />                                                                      primers 5&#226;&#8364;&#178;-AGCGGCAAGCTTCTATGCCAGCCGTCCGTA <br /> v/v) followed 250 ml H2O. After phase separation, aqueous <br />                                                                      CGAAAA-3&#226;&#8364;&#178; 5&#226;&#8364;&#178;-CGACCTCGAGGTCAACTTCCCACG <br /> and organic phases dried resuspended in <br />                                                                      CCGCGAAGGAAAGGCAG-3&#226;&#8364;&#178; confirm equal RNA loading. <br /> 50% propan-1-ol 100% butan-1-ol examined TLC <br />                                                                      The PCR products run 1% agarose gel. <br /> using solvent systems methanol:0.6% NaCl:25% NH3 <br /> (100:100:10) chloroform:methanol:water (10:10:3 v/v) <br /> respectively.                                                        Subcellular localization studies <br />    Neutral sphingomyelinase activity character- <br /> ized using Amplex Red sphingomyelinase assay kit (Invit-         Tetracycline-induced mid-log T. brucei nSMase-HA Ti blood- <br /> rogen Ltd, He et al., 2002). Briefly, washed membranes               stream form cells harvested centrifugation (800 g, <br /> (containing 254 mg total protein) encoding GST&#226;&#8364;&#8220;                   10 min) used immunofluorescence differential <br /> TbnSMase incubated 100 mM Tris-HCl (pH 7.4),                 centrifugation previously described (Martin Smith, <br />  <br /> &#194;&#169; 2010 Blackwell Publishing Ltd, Molecular Microbiology, 76, 1461&#226;&#8364;&#8220;1482 <br />  1478 S. A. Young T. K. Smith &#228;&#352;? <br />  <br /> 2006b). The blocked cells incubated rat a-HA        concentration lysis buffer (1% Nonidet P-40, 0.5% <br /> (Roche) rabbit a-trypanopain antibody (James Bangs, Uni-          deoxycholate, 0.1% SDS). Monoclonal anti-p67 (mAb280) <br /> versity Wisconsin-Madison) followed respective           antibody (a gift James Bangs, University Wisconsin- <br /> secondary; FITC conjugated rabbit anti-rat (DakoCytomation)          Madison), recognizes uncharacterized peptide <br /> and TRITC conjugated Goat anti-rabbit immunoglobulins                epitope p67 lumenal domain, preadsorbed to <br /> (Sigma). 4,6-diamidino-2-phenylindole (DAPI) staining            Protein G-Sepharose 4B Fast Flow resin (Sigma) prior to <br /> used reveal localization nuclear kDNA.             incubation solubilized protein 1 h 4&#194;&#176;C. After <br />   Differential centrifugation Western analysis               incubation, resin beads washed times lysis <br /> HA-tagged protein detected using primary rat a-HA           buffer TEN buffer. Following centrifugation the <br /> (Roche), secondary HRP-conjugated rabbit a-rat                   resin time point suspended equal volume <br /> (Jackson) antibodies ECL Western detection reagents              2&#194;&#165; SDS-PAGE sample buffer prior denaturing 100&#194;&#176;C for <br /> (Amersham) used described previously (Martin             10 min, release complexes protein G <br /> Smith, 2006a).                                                       sepharose. Proteins separated 10% SDS-PAGE gel <br />                                                                      visualized Coomassie blue staining. To detect [35S]- <br />                                                                      labelled proteins destained gel soaked En3hance <br /> In vivo T. brucei metabolic labelling                                (NEN) detected autoradiography described before. <br /> For metabolic labelling, mid-log trypanosomes were <br /> centrifuged (800 g, 10 min), washed minimal essential             FACS analysis <br /> media resuspended specific supplement            Cells stained propidium iodide described previ- <br /> free media final concentration 1 &#194;&#165; 107 cells ml-1. Cells     ously (Hammarton et al., 2003). Briefly, cells fixed in <br /> were labelled 2 h 37&#194;&#176;C [9,10-3H] myristic acid           70% methanol, 30% PBS (1&#194;&#165;106 cells ml-1) incubated at <br /> (47 Ci mmol-1, Perkin Elmer) complexed defatted-BSA               4&#194;&#176;C overnight. Cells washed cold PBS, resuspended <br /> (1 mol mol-1) shaking water bath. Cells collected       1 ml PBS containing 10 mg ml-1 propidium iodide and <br /> centrifugation (800 g, 10 min), washed twice trypanosome          10 mg ml-1 RNase A, incubated 37&#194;&#176;C 45 min. FACS <br /> dilution buffer (TDB) buffer processed lipid       performed using Becton Dickinson FACScan using <br /> protein analysis. Lipids extracted using chloroform:            detector FL2-A. <br /> methanol: water (10:10:3 v/v) 1 h, supernatant <br />                                                                         Duplicate data sets collected analysed using <br /> removed pellet re-extracted chloroform:methanol <br />                                                                      WinMDI software (http://facs.scripps.edu) utilizing 10 000 <br /> (2:1 v/v) 1 h. The pooled supernatants dried, prior to <br />                                                                      events collected sample. Cells a <br /> desalting using butanol/water partioning. Lipids sepa- <br />                                                                      particular stage cell cycle (G0/G1, S, G2/M &gt;G2/M) <br /> rated silica 60 high-performance layer chromatography <br />                                                                      indicated percentage total. The gates used for <br /> (HPTLC) plates, chloroform : methanol : ammonia <br />                                                                      quantification different stages Sheader et al. <br /> (13 M) : ammonium acetate (1 M) : water, (180 : 140 : 9 : 9 : <br />                                                                      (2005). <br /> 23 v/v) solvent. Radiolabelled lipids detected by <br /> fluorography -70&#194;&#176;C, spraying En3hance using <br />                                                                      Lipidomic analysis <br /> Kodak XAR-5 film intensifying screen. Protein samples <br /> (cell pellets) quenched 2&#194;&#165; SDS-PAGE sample buffer            Phospholipid analysis performed bloodstream wild- <br /> and heated 95&#194;&#176;C 5 min. Proteins separated           type TbnSMase cKO trypanosomes (1 &#194;&#165; 108) pres- <br /> 10% SDS-PAGE gel visualized Coomassie blue stain-             ence absence tetracycline 42 h. Cells harvested <br /> ing. To detect [3H]-labelled proteins destained gel          washed TDB extracted according method of <br /> soaked En3hance (NEN) 30 min, washed water               Bligh&#226;&#8364;&#8220;Dyer (Bligh Dyer, 1959), dried N2, and <br /> three times, soaked 10% glycerol dried exposed         stored 4&#194;&#176;C analysed electrospray mass spec- <br /> XAR-5 film -80&#194;&#176;C.                                                 trometry (ES-MS ES-MS/MS) described previously <br />    When labelling [35S]-methionine, mid-log cells          (Richmond Smith, 2007; Gibellini et al., 2009; Richmond <br /> collected centrifugation, washed methionine free minimal       et al., 2010). <br /> essential media resuspended media final <br /> concentration 1 &#194;&#165; 107cells ml-1. Cells labelled          Western blotting <br /> 1 h 20 mCi ml-1 [35S]-methionine (MP Biomedicals, <br /> 1175 Ci mmol-1) 37&#194;&#176;C. Small-scale sVSG purification           Specific T. brucei proteins detected Western blot <br /> as performed described previously (Jones et al., 2005).           analysis, using rabbit a-VSG221 rabbit <br />    For immunoprecipitation (IP) [35S]-p67 T. brucei,     a-trypanopain (gifts James Bangs, University of <br /> the protocol Alexander et al. (2002) followed       Wisconsin-Madison) rabbit a-PDI-2 (a gift Derek <br /> modifications. Briefly, 5 &#194;&#165; 106 [35S]-methionine pulse-labelled      Nolan, Trinity College Dublin) antibodies secondary rat <br /> (1 h) trypanosomes harvested time point (0          a-rabbit HRP-conjugated antibody (Jackson ImmunoRe- <br /> 2 h), washed PBS solubilized 500 ml lysis          search Ltd). ECL detection reagents (GE Healthcare) were <br /> buffer [TEN buffer, 1% Nonidet P-40, 0.5% deoxycholate,              utilized visualize signal. <br /> 0.1% SDS (TEN buffer = 50 mM Tris-HCl, pH 7.5, 150 mM <br /> NaCl, 5 mM EDTA)] containing protease inhibitor cocktail.          FITC-BSA uptake assay <br /> The extracts precleared centrifugation (14 000 rpm, <br /> 5 min) detergents (1% Nonidet P-40, 0.5% deoxycho-            Mid-log trypanosomes harvested centrifugation <br /> late, 0.1% SDS) added culture supernatants        (800 g, 10 min) resuspended 1 &#194;&#165; 106 ml-1 minimal <br />  <br />                                                              &#194;&#169; 2010 Blackwell Publishing Ltd, Molecular Microbiology, 76, 1461&#226;&#8364;&#8220;1482 <br />                                                                                             Neutral sphingomyelinase Trypanosoma brucei 1479 <br />  <br /> essential media. The cells preincubated 10 min             Simons, K. (2000) Lipid rafts function biosynthetic <br /> 37&#194;&#176;C FITC-BSA added final concentration             delivery proteins cell surface yeast. Proc Natl <br /> 5 mg ml-1. After incubation 30 min, cells har-           Acad Sci USA 97: 3254&#226;&#8364;&#8220;3259. <br /> vested centrifugation washed twice PBS remove         Bernardo, K., Krut, O., Wiegmann, K., Kreder, D., Micheli, M., <br /> unbound FITC-BSA, prior fixation 4% paraformaldehyde            Sch&#195;&#164;fer, R., et al. (2000) Purification characterization <br /> and processed immunofluorescence analysis                      magnesium-dependent neutral sphingomyelinase from <br /> described above.                                                      bovine brain. J Biol Chem 275: 7641&#226;&#8364;&#8220;7647. <br />                                                                    Bertello, L., Goncalvez, M.F., Colli, W., Lederkremer, <br /> Electron Microscopy                                                   R.M. (1995) Structural analysis inositol phospholipids <br />                                                                       Trypanosoma cruzi epimastigote forms. Biochem J <br /> For transmission electron microscopy cell pellets fixed          310: 255&#226;&#8364;&#8220;261. <br /> for 24 h Peter&#226;&#8364;&#8482;s fixative (1.25% glutataldehyde, 1%     Bionda, C., Portoukalian, J., Scmitt, D., Rodriguez-Lafrasse, <br /> paraformaldehyde 0.08 M sodium cacodylate pH 7.2              C., Ardail, D. (2004) Subcellular compartmentalization <br /> 0.02% calcium chloride). Samples rinsed twice 1%              ceramide metabolism: MAM (mitochondria-associated- <br /> tannic acid 10 min post-fixed/stained 0.5% osmium            membrane) and/or mitochondria? Biochem J 382: 527&#226;&#8364;&#8220; <br /> tetraoxide (aq) 1 h. Specimens rinsed twice          533. <br /> distilled water fixation/staining 1% uranyl acid fol-    Bligh, E.G., Dyer, W.J. (1959) A rapid method total lipid <br /> lowed infiltration/embedding Spurr resin. Sections         extraction purification. Can J Biochem Physiol 37: <br /> cut using Leica Ultracut UCT microtome, mounted piolo-           911&#226;&#8364;&#8220;917. <br /> form coated copper grids, post-stained 3% uranyl acid         Brickman, M.J., Balber, A.E. (1994) Transport lyso- <br /> (aq) Reynold&#226;&#8364;&#8482;s lead citrate examined using Tecnai           somal membrane glycoprotein Golgi endo- <br /> 12 BioTWIN FEI, operated 120 kV.                somes lysosomes cell surface African <br /> Images collected using SIS Megaview III, 2 &#194;&#165; 2             trypanosomes. J Cell Sci 107: 3191&#226;&#8364;&#8220;3200. <br /> montages.                                                          Burkard, G., Fragoso, C.M., Roditi, I. (2007) Highly effi- <br />                                                                       cient stable transformation bloodstream forms Trypa- <br /> Acknowledgements                                                      nosoma brucei. Mol Biochem Parasitol 153: 220&#226;&#8364;&#8220;223. <br />                                                                    Buxbaum, L.U., Milne, K.G., Werbovetz, K.A., Englund, <br /> This work supported TKS&#226;&#8364;&#8482;s Wellcome Trust               P.T. (1996) Myristate exchange Trypanosoma brucei <br /> Senior Research Fellowship 067441, Wellcome project             variant surface glycoprotein. Proc Natl Acad Sci USA 93: <br /> grant 086658. We wish thank Jay Bangs, University               1178&#226;&#8364;&#8220;1183. <br /> Wisconsin generous gifts monoclonal a-p67 (mAb280)          Caffrey, C.R., Hansell, E., Lucas, K.D., Brinen, L.S., Hernan- <br /> and a-trypanopain antibodies Derek Nolan, University           dez, A.A., Cheng, J., et al. (2001) Active site mapping, <br /> Dublin a-PDI-2 antibody. We wish acknowledge              biochemical properties subcellular localization of <br /> access mass spectrometry facilities St Andrews Uni-          rhodesain, major cysteine protease Trypanosoma <br /> versity University Dundee. We wish thank           brucei rhodesiense. Mol Biochem Parasitol 118: 61&#226;&#8364;&#8220;73. <br /> Meg Stark (York) doing EM.                                 Chang, T., Milne, K.G., Guther, M.L.G., Smith, T.K., and <br />                                                                       Ferguson, M.A.J. (2002) Cloning Trypanosoma <br /> References                                                            brucei Leishmania major genes encoding the <br />                                                                       GlcNAc-phosphatidylinositol          De-N-acetylase        of <br /> Achleitner, G., Gaigg, B., Krasser, A., Kainersdorfer, E., Kohl-      glycosylphosphatidyl-inositol biosynthesis essential <br />    wein, S.D., Perktold, A., et al. (1999) Association        African sleeping sickness parasite. J Biol Chem 277: <br />    endoplasmic reticulum mitochondria yeast facili-        50176&#226;&#8364;&#8220;50182. <br />    tates interorganelle transport phospholipids         Chatterjee, S., Mayor, S. (2001) The GPI-anchor and <br />    membrane contact. Eur J Biochem 264: 545&#226;&#8364;&#8220;553.                      protein sorting. Cell Mol Life Sci 58: 1969&#226;&#8364;&#8220;1987. <br /> Alexander, D.L., Schwartz, K.J., Balber, A.E., Bangs,          Coppens, I., Levade, T., Courtoy, P.J. (1995) Host <br />    J.D. (2002) Developmentally regulated trafficking           plasma low density lipoprotein particles essential <br />    lysosomal membrane protein p67 Trypanosoma brucei.              source lipids bloodstream forms Trypanosoma <br />    J Cell Sci 115: 3255&#226;&#8364;&#8220;3263.                                         brucei. J Biol Chem 270: 5736&#226;&#8364;&#8220;5741. <br /> Ali, B.R.S., Pal, A., Croft, S.L., Taylor, R.T., Field, M.C.   Cowart, L.A., Okamoto, Y., Lu, X., Hannun, Y.A. (2006) <br />    (1999) The farnesyltransferase inhibitor manumycin A          Distinct roles novo versus hydrolytic pathways of <br />    novel trypanocide complex mode action including          sphingolipid biosynthesis Saccharomyces cerevisiae. <br />    major effects mitochondria. Mol Biochem Para 104:               Biochem J 393: 733&#226;&#8364;&#8220;740. <br />   67&#226;&#8364;&#8220;80.                                                           Denny, P.W., Smith, D.F. (2004) Rafts sphingolipid <br /> Allen, C.L., Goulding, D., Field, M.C. (2003) Clathrin-           biosynthesis kinetoplastid parasitic protozoa. Mol <br />   mediated endocytosis essential Trypanosoma brucei.            Microbiol 53: 725&#226;&#8364;&#8220;733. <br />   EMBO J 22: 4991&#226;&#8364;&#8220;5002.                                            Denny, P.W., Field, M.C., Smith, D.F. (2001) GPI- <br /> Arenz, C., Thutewohl, M., Block, O., Waldmann, H.,                    anchored proteins glycoconjugates segregate lipid <br />   Altenbach, H.J., Giannis, A. (2001) Manumycin A                 rafts Kinetoplastida. FEBS Lett 491: 148&#226;&#8364;&#8220;153. <br />   analogues irreversible inhibitors neutral         Denny, P.W., Shams-Eldin, H., Price, H.P., Smith, D.F., and <br />   sphingomyelinase. Chem Biochem 2: 141&#226;&#8364;&#8220;143.                          Schwarz, R.T. (2006) The protozoan inositol phosphorylce- <br /> Bagnat, M., Ker&#195;&#164;nen, S., Shevchenko, A., Shevchenko, A.,              ramide synthase: novel drug target defines new <br />  <br /> &#194;&#169; 2010 Blackwell Publishing Ltd, Molecular Microbiology, 76, 1461&#226;&#8364;&#8220;1482 <br />  1480 S. A. Young T. K. Smith &#228;&#352;? <br />  <br />   class sphingolipid synthase. J Biol Chem 281: 28200&#226;&#8364;&#8220;            Hofmann, K., Tomiuk, S., Wolff, G., Stoffel, W. (2000) <br />   28209.                                                               Cloning characterization mammalian brain- <br /> Dickson, R.C., Sumanasekera, C., Lester, R.L. (2006)               specific Mg2+-dependent neutral sphingomyelinase. Proc <br />   Functions metabolism sphingolipids Saccharomy-             Natl Acad Sci USA 37: 5895&#226;&#8364;&#8220;5900. <br />   ces cerevisiae. Prog Lipid Res 45: 447&#226;&#8364;&#8220;465.                        Hughes, H., Stephens, D.J. (2008) Assembly, organiza- <br /> Engstler, M., Thilo, L., Weise, F., Gr&#195;&#188;nfelder, C.G., Schwarz,         tion, function COPII coat. Histochem Cell Biol <br />   H., Boshart, M., Overath, P. (2004) Kinetics endocy-          129: 129&#226;&#8364;&#8220;151. <br />   tosis recycling GPI-anchored variant surface            Huwiler, A., Kolter, T., Pfeilschifter, J., Sandhoff, K. <br />   glycoprotein Trypanosoma brucei. J Cell Sci 117: 1105&#226;&#8364;&#8220;            (2000) Physiology pathphysiology sphingolipid <br />   1115.                                                                metabolism signaling. Biochimica Biophysica Acta <br /> Fridberg, A., Olsen, C.L., Nakayasu, E.S., Tyler, K.M.,                1485: 63&#226;&#8364;&#8220;99. <br />   Almeida, I.C., Engman, D.M. (2008) Sphingolipid syn-           Jones, D.C., Mehlert, A.M., G&#195;&#188;ther, M.L., Ferguson, <br />   thesis necessary kinetoplast segregation cytoki-          M.A.J. (2005) Deletion glucosidase II gene Trypa- <br />   nesis Trypanosoma brucei. J Cell Sci 121: 522&#226;&#8364;&#8220;535.                nosoma brucei reveals novel N-glycosylation mechanisms <br /> Funato, K., Reizman, H. (2001) Vesicular nonvesicu-            biosynthesis variant surface glycoprotein. J Biol <br />   lar transport ceramide ER Golgi apparatus              Chem 280: 35929&#226;&#8364;&#8220;35942. <br />   yeast. J Cell Biol 155: 949&#226;&#8364;&#8220;959.                                   Kabani, S., Fenn, K., Ross, A., Ivens, A., Smith, T.K., Ghazal, <br /> Futerman, A.H., Riezman, H. (2005) The ins outs             P., Matthews, K. (2009) Genome-wide expression pro- <br />   sphingolipid synthesis. Trends Cell Biol 15: 312&#226;&#8364;&#8220;318.                filing vivo-derived bloodstream parasite stages and <br /> Gibellini, F., Hunter, W.N., Smith, T.K. (2009) The etha-          dynamic analysis mRNA alterations synchronous <br />   nolamine branch Kennedy pathway essential               differentiation Trypanosoma brucei. BMC Genomics 10: <br />   bloodstream form Trypanosoma brucei. Mol Micro-               427. <br />   biol 73: 826&#226;&#8364;&#8220;843.                                                  Kaneshiro, E.S., Jayasimhulu, K., Lester, R.L. (1986) <br /> Goni, F.M., Alonso, A. (2002) Sphingomyelinases:                   Characterization inositol lipids Leishmania <br />   enzymology membrane activity. FEBS letters 531:                  donovani promastigotes: identification inostiol <br />   38&#226;&#8364;&#8220;46.                                                               sphingophospholipid. J Lipid Res 27: 1294&#226;&#8364;&#8220;1303. <br /> Gonzalez-Zorn, B., Dominguez-Bernal, G., Suarez, M., Ripio,          Kitagaki, H., Cowart, L.A., Matmati, N., Montefusco, D., <br />   M.T., Vega, Y., Novella, S., Vazquez-Boland, J.A.                Vaena Avalos, S., Novgorodov, S.A., et al. (2007) Isc1 <br />   (1999) The smcL gene Listeria ivanovii encodes sph-             regulates sphingolipid metabolism yeast mitochondria. <br />   ingomyelinase C mediates bacterial escape              Biocimica Biophysica Acta 1768: 2849&#226;&#8364;&#8220;2861. <br />   phagocytic vacuole. Mol Microbiol 33: 510&#226;&#8364;&#8220;523.                     Kitagaki, H., Cowart, L.A., Matmati, N., Montefusco, D., <br /> G&#195;&#188;ler, J.L., Kriegov&#195;&#161;, E., Smith, T.K., Luke, J., Englund,         Gandy, J., Vaena Avalos, S., et al. (2009) ISC1- <br />   P.T. (2008) Mitochondrial fatty acid synthesis required       dependent metabolic adaptation reveals indispensable <br />   normal mitochondrial morphology function Trypano-             role mitochondria induction nuclear genes during <br />   soma brucei. Mol Microbiol 67: 1125&#226;&#8364;&#8220;1142.                            diauxic shift Saccharomyces cerevisiae. J Biol Chem <br /> G&#195;&#188;ther, M.L.S., Lee, S., Tetley, L., Acosta-Serrano, A.,           284: 10818&#226;&#8364;&#8220;10830. <br />   Ferguson, M.A.J. (2006) GPI anchored proteins free             Koynova, R., Caffery, M.C. (1995) Phases phase <br />   GPI glycolipids procyclic form Trypanosoma brucei             transitions sphingolipids. Biochim Biophys Acta <br />   non-essential growth, required colonization           1255: 213&#226;&#8364;&#8220;236. <br />   tsetse fly components              Lahiri, S., Futerman, A.H. (2007) The metabolism and <br />   surface coat. Mol Biol Cell 17: 5265&#226;&#8364;&#8220;5274.                           function sphingolipids glycosphingolipids. Cell Mol <br /> Hammarton, T.C., Clark, J., Douglas, F., Boshart, M.,              Life Sci 64: 2270&#226;&#8364;&#8220;2284. <br />   Mottram, J.C. (2003) Stage-specific differences cell            Levine, T.P., Wiggins, C.A., Munro, S. (2000) Inositol <br />   cycle control Trypanosoma brucei revealed RNA                  phosphorylceramide synthase located Golgi appa- <br />   interference mitotic cyclin. J Biol Chem 278: 22877&#226;&#8364;&#8220;            ratus Saccharomyces cerevisiae. Mol Biol Cell 11: 2267&#226;&#8364;&#8220; <br />   22886.                                                               2281. <br /> Hanada, K., Mitamura, T., Fukasawa, M., Magistrado, P.A.,            Luberto, C., Hassler, D.F., Signorelli, P., Okamoto, Y., <br />   Horii, T., Nishijima, M. (2000) Neutral sphingolmyeli-           Sawai, H., Boros, E., et al. (2002) Inhibition tumor <br />   nase activity dependent Mg2+ anionic phospholip-              necrosis factor-induced cell death MCF7 novel <br />   ids intrraerythrocytic malaria parasite Plasmodium            inhibitor neutral sphingomyelinase. J Biol Chem 277: <br />   falciparum. Biochem J 346: 671&#226;&#8364;&#8220;677.                                  41128&#226;&#8364;&#8220;41139. <br /> Hanada, K., Palacpac, N.M.Q., Magistrado, P.A., Kurokawa,            Martin, K.L., Smith, T.K. (2006a) The glycosylphosphati- <br />   K., Rai, G., Sakata, D., et al. (2002) Plasmodium falci-             dylinositol (GPI) biosynthetic pathway bloodstream-form <br />   parum phospholipase C hydrolyzing sphingomyelin                  Trypanosoma brucei dependent novo synthe- <br />   lysocholinephospholipids possible target malaria            sis inositol. Mol Microbiol 61: 89&#226;&#8364;&#8220;105. <br />   chemotherapy. J Exp Med 195: 23&#226;&#8364;&#8220;34.                                Martin, K.L., Smith, T.K. (2006b) Phosphatidylinositol <br /> He, X., Chen, F., McGovern, M.M., Schuchman, E.H.                  synthesis essential bloodstream form Trypanosoma <br />   (2002) A fluorescence-based, high-throughput sphingomy-              brucei. Biochem J 396: 287&#226;&#8364;&#8220;295. <br />   elin assay analysis Niemann-Pick disease            Matthews, K.R. (2005) The developmental cell biology of <br />   disorders sphingomyelin metabolism. Anal                    Trypanosoma brucei. J Cell Sci 118: 283&#226;&#8364;&#8220;290. <br />   Biochem 306: 115&#226;&#8364;&#8220;123.                                              van Meer, G., Voelker, D.R., Feigenson, G.W. (2008) <br />  <br />                                                              &#194;&#169; 2010 Blackwell Publishing Ltd, Molecular Microbiology, 76, 1461&#226;&#8364;&#8220;1482 <br />                                                                                            Neutral sphingomyelinase Trypanosoma brucei 1481 <br />  <br />   Membrane lipids: they behave.             Sheader, K., Vaughan, S., Minchin, J., Hughes, K., Gull, K., <br />   Nat Rev 9: 112&#226;&#8364;&#8220;124.                                                et al. (2005) Variant surface glycoprotein <span id='am-397' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-398' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-399' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-400' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-401' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-402' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-403' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-404' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-405' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-406' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-407' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-408' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-409' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-410' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-411' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-412' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-413' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-414' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-415' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-416' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-417' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-418' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-419' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-420' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-421' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-422' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-423' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-424' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-425' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-426' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-427' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-428' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-429' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-430' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-431' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-432' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-433' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-434' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-435' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-436' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-437' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-438' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-439' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-440' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-441' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-442' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-443' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-444' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-445' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-446' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-447' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-448' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-449' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-450' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-451' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-452' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-453' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-454' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-455' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-456' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-457' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-458' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-459' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-460' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-461' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-462' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-463' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-464' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-465' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-466' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-467' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-468' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-469' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-470' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-471' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-472' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-473' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-474' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span>RNA interference</span> <br /> Mina, J.G., Pan, S.Y., Wansadhipathi, N.K., Bruce, C.R.,             triggers precytokinesis cell cycle arrest African <br />   Shams-Eldin, H., Schwarz, R.T., et al. (2009) The Trypa-           trypanosomes. Proc Natl Acad Sci USA 102: 8716&#226;&#8364;&#8220;8721. <br />   nosoma brucei sphingolipid synthase, essential enzyme         Smith, T.K., Vasileva, N., Gluenz, E., Terry, S., Portman, N., <br />   drug target. Mol Biochem Parasitol 168: 16&#226;&#8364;&#8220;23.                 Kramer, S., et al. (2009) Blocking variant surface glycopro- <br /> Miroux, B., Walker, J.E. (1996) Over-production pro-          tein synthesis Trypanosoma brucei triggers general <br />   teins Escherichia coli: mutant hosts allow synthesis       arrest translation initiation. PLoS ONE 4: e7532. <br />   membrane proteins globular proteins high          Sutterlin, C., Doering, T., Schimmoller, F., Schroder, S., and <br />   levels. J Mol Biol 260: 289&#226;&#8364;&#8220;298.                                   Riezman, H. (1997) Specific requirements ER to <br /> Mizutani, Y., Tamiya-Koizumi, K., Irie, F., Hirabayashi, Y.,         Golgi transport GPI-anchored proteins yeast. J Cell <br />   Miwa, M., Yoshida, S. (2000) Cloning expression            Sci 110: 2703&#226;&#8364;&#8220;2714. <br />   rat neutral sphingomyelinase: enzymologica; character-        Sutterwala, S.S., Creswell, C.H., Sanyal, S., Menon, A.K., <br />   ization identification essential histidine residues.        Bangs, J.D. (2007) De novo sphingolipid synthesis <br />   Biochimica Biophysica Acta 1485: 236&#226;&#8364;&#8220;246.                          essential viability, transport of <br /> Muniz, M., Reizman, H. (2000) Intracellular transport         glycosylphosphatidylinositol-anchored proteins African <br />   GPI-anchored proteins. EMBO J 19: 10&#226;&#8364;&#8220;15.                           trypanosomes. Euk <br /> </body></html>