<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL OF BACrERIOLOGY, Nov. 1992, p. 7090-7097                                                                    Vol. 174, No. 22 <br /> 0021-9193/92/227090-08$02.00/0 <br /> Copyright X) 1992, American Society Microbiology <br />  <br />  <br />           Mutants Carrying Conditionally Lethal Mutations Outer <br />           Membrane Genes omsA firA (ssc) Are Phenotypically <br />                      Similar, omsA Allelic firA <br />                                        RIIYTA VUORIO* AND MARTTI VAARA <br />                Department Bacteriology Immunology, University Helsinki, 00290 Helsink4 Finland <br />                                         Received 10 August 1992/Accepted 8 September 1992 <br />  <br />             We previously identified gene (the ssc gene) defective thermosensitive antibiotic- <br />          supersusceptible outer membrane permeability mutant SS-C SalmoneUla typhimurium shown this <br />          gene analogous Escherichia coli genefirA (L. Hirvas, P. Koski, M. Vaara, EMBO J. 10:1017-1023, <br />          1991). Others tentatively implicatedfirA different function, mRNA synthesis. Here report the <br />          defect thermosensitive outer membrane omsA mutant E. coli (T. Tsuruoka, M. Ito, S. Tomioka, A. <br />          Hirata, M. Matsuhashi, J. Bacteriol. 170:5229-5235, 1988) <span id='am-159' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-160' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span>mutation</span> firA; mutation <br />          changed codon 271 serine asparagine. The omsA-induced <span id='am-81' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-82' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> completely reverted by <br />          plasmids containing wild-type firA ssc. Plasmids carrying omsA allele, identical mutant allele <br />          prepared localized mutagenesis, control lac elicited partial complementation. Transcomple- <br />          mentation studies plasmids carrying various mutant alleles S. typhimurium gene indicated the <br />          ability plasmids complement omnsA mutation similar ability complement ssc <br />          mutation. The antibiotic-supersusceptible phenotype omsA mutant closely resembled ssc <br />          mutant, i.e., omsA mutant supersusceptible hydrophobic antibiotics large-peptide antibiotics <br />          intact outer membrane effective permeability barrier. As previously demonstrated with <br />          omnsA mutant, outer membrane ssc mutant selectively ruptured incubation 1 h <br />          growth-nonpermitting temperature; 82% periplasmic j-lactamase 3% the <br />          cytoplasmic marker enzyme released medium. All findings consistent our <br />          concept thatfirA essential gene involved generation outer membrane. <br />  <br />    The outer membrane (OM) gram-negative bacteria               known essential OM genes and, hence, turn be <br /> one comprehensively characterized biological               invaluable elucidation remaining enigmas OM <br /> membranes (26, 31). However, lack complete               biogenesis. Such mutants include omsA (outer mem- <br /> picture biosynthesis. The translocation process OM          brane structure) mutant Escherichia coli (39) ssc-1 <br /> proteins partially known (49). Very little known            (supersensitive class C) mutant Salmonella typhimurium <br /> about export lipopolysaccharide (LPS) component                 (17, 38). Both shown extremely susceptible <br /> <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> synthesis site cytoplasmic membrane            wide set antibiotics, OM an <br /> outer leaflet OM (33). Furthermore, lipid          effective permeability barrier. Both unable grow at <br /> A essential LPS viability bacterial cells,      42&#194;&#176;C. The study ssc-1 mutation led discover and <br /> only 2 (pxA4 lpxB) approximately 8 10 genes             sequence novel gene S. typhimurium, ssc gene (16). <br /> involved biosynthesis discovered (33). The           The sequence analog E. coli, firA gene, also <br /> search remaining essential genes OM biogenesis             known (5). On hand, gene mutated omsA <br /> has hampered lack mutants conditionally                mutant identified Tsuruoka et al. (39), their <br /> defective genes.                                              preliminary evidence suggested omsA mutation <br />    A characteristic feature intact enterobacterial OM       maps approximately 4 min E. coli chromosome. <br /> its relative impermeability hydrophobic drugs (28-30).             Since firA gene located 4 min, felt it <br /> The impermeability LPS (29, 45); outer leaflet       necessary study possible relationship genes. <br /> the OM contains LPS phospholipid lacks                In report, omsA mutation allelic <span id='am-2' about='protege:TO' typeof='owl:Thing'>to</span> <br /> glycerophospholipid patches, effective channels hy-           firA. Furthermore, phenotypic character- <br /> drophobic diffusion (29-31). Agents damage OM                istics ssc-1 (irA) mutant S. typhimurium the <br /> sensitize bacteria drastically hydrophobic inhibitors          omsA mutant E. coli closely resemble other. <br /> (40). Also, mutants severe defects nones- <br /> sential inner carbohydrate core region LPS (chemotypes <br /> Re, Rd2, Rd,, RcP-) supersusceptible (14, 28, 30,                           MATERIALS AND METHODS <br /> 37). Therefore, supersusceptibility hydrophobic drugs <br /> may suggest defect OM.                                          Bacterial strains plasmids. The E. coli S. typhimu- <br />    Accordingly, novel mutants conditionally lethal          rium strains plasmids used described Table 1. The <br /> and, permissive conditions, supersusceptible hy-             E. coli cloning hosts JM105 (50) vector pUC19 (50), <br /> drophobic antibiotics mutated previously un-              TG1 (8) phage M13mp8 M13mp9 (27), XL1-Blue <br />                                                                       (2) Bluescript SK+ (1). Cultures grown L broth <br />                                                                       (17) 2x TY medium (34), supplemented ampicillin <br />                                                                       (100 ,ug/ml) ampicillin (50 ,ug/ml) tetracycline (12.5 <br />   * Corresponding author.                                             ,ug/ml) appropriate. <br />                                                                7090 <br />  VOL. 174, 1992                                                                           OUTER MEMBRANE GENES omsA AND firA                   7091 <br />  <br />                   TABLE 1. Bacterial strains plasmids                           sodium citrate], 2% blocking reagent [catalog no. 1096176; <br /> Strain plasmid             Genotype relevant property         Reference(s) <br />                                                                                     Boehringer], 0.1% N-lauroylsarcosine, 0.02% sodium dode- <br />                                                                                     cyl sulfate [SDS]) 1 h 42&#194;&#176;C. Hybridization per- <br /> E. coli                                                                             formed 2.5 ml solution 500 pmol the <br />   CDH23-210                HfrC metBl proA3 reLAl lac-3             39              30-mer firA-specific oligonucleotide probe CTGGCTGATT <br />                              maL436 tsx-76 K-12                                     TAGCGCAGCAGTTGGATGCA labeled digoxigenin- <br />      CDH23-213             omsA sister transductant              39              dUTP tailing (Boehringer) 18 h 42&#194;&#176;C. The filter was <br />                              CDH23-210                                              washed twice 5 min time 2x SSC-0.1% SDS at <br /> S.   typhimurium                                                                    room temperature twice 5 min time with <br />      SH5014                ilv-1178 thr-914 his-6116                36              0.1 x SSC-0.1% SDS 54&#194;&#176;C. Detection performed with <br />                              metA422 metESSI trpB2 xyl-                             DIG luminescence detection kit (Boehringer) instructed <br />                              404 Hi-b H2-e,n,xflaA66                                supplier. <br />                              rpsL120 rfaJ4041 LT2                                      Growth viability determinations. Growth mea- <br />      SH7622                ssc-1 (from SH5014)                      17, 38          sured A550, determining viable count <br />                                                                                     plating L agar. <br /> Plasmids                                                                               Antibiotic susceptibility tests. MICs determined by <br />      pUCHS16Aa             ompH+                                    17              broth microdilution method L broth. The inoculum <br />      pUCHS14-IIa           ssc+                                     17 <br />      pSSClOla              Encodes Ssc amino acid              19              104 L agar-grown cells ml. Incubation 18 h <br />                              substitution Val-291- Met                              28 39&#194;&#176;C. The antibiotics fusidic acid (Na salt), <br />      pSSC1Or               Encodes Ssc amino acid              19              rifampin, clindamycin hydrochloride, bacitracin (65,000 <br />                              substitution Met-288-*Leu                              U/mg; Sigma Chemical Co., St. Louis, Mo.), erythromycin <br />      pSSC103a              Encodes Ssc amino acid              19              ethyl succinate (Orion, Helsinki, Finland), vancomycin hy- <br />                              substitution Met-290-3Leu                              drochloride (Eli Lilly &amp; Co., Indianapolis, Ind.), mupi- <br />      pSSC104a              Encodes Ssc amino acid              19              rocin (SmithKline Beecham, Worthing, West Sussex, En- <br />                              substitution Met-292--+Leu                             gland). As indicated text, alternative <br />      pSSC1O0a              Encodes Ssc amino acid              19              MIC test (E Test; AB Biodisk, Solna, Sweden) (42) was <br />                              substitution Gly-27--Asp <br />      pSSC106a              Encodes Ssc amino acid              19              used. Bacteria (in 0.9% NaCl; turbidity suspension <br />                              substitution Met-31--Leu                               equal McFarland 0.5 standard) streaked with <br />      pSSC107a              Encodes Ssc amino acid              19              swab cover entire surface L agar plate. After <br />                              substitution Leu-130--*Met                             surface dried, E Test strip placed and <br />      pSSC108a              Encodes Ssc amino acid              19              plate incubated 18 h 37&#194;&#176;C. The MIC then <br />                              substitution Gly-131--Asp                              interpreted noting point intersection growth <br />      pSSC109&quot;              Encodes Ssc amino acid              19              margin MIC scale E Test strip. <br />                              substitution Met-288--Lys                                 Susceptibility determinations agar diffusion method <br />      pSSC11O&apos;              Encodes Ssc amino acid              19              performed using L agar plates antibiotic discs as <br />                              substitution Met-290--*Lys <br />      pSSCllla              Encodes Ssc amino acid              19              previously described (17). The antibiotic discs (A/S Rosco, <br />                              substitution Met-292--+Lys                             Copenhagen, Denmark) contained mupirocin (10 pg), van- <br />      pSSC112a              Encodes Ssc amino acid              19              comycin (70 tg), bacitracin (40 units), rifampin (30 p,g). <br />                              substitution Gly-289- Asp                                 Measurement enzyme activities. Detection 1-lacta- <br />      pGAH317               hlpA+ firA+                              20              mase modified method O&apos;Callaghan et al. <br />      pOMSA1O1b             oMSA                                     This work       (32). Nitrocefin (Glaxo Pharmaceuticals, Ltd., Greenford, <br />      pOMSA102b             omsA                                     This work       England) added enzyme samples 0.01 M <br />      pOMSA103b             firA +                                   This work       HEPES (N-2-hydroxyethylpiperazine-N-2-ethanesulfonic <br />        The lac promoter vector opposite orientation ompH   acid) (pH, 7.2) yield final concentration 100 p,g/ml, <br /> ssc gene.                                                                           rate nitrocefin hydrolysis monitored at <br />     The lac promoter vector upstream orientation as <br />      b <br />                                                                                     540 nm. The activity glucose 6-phosphate dehydrogenase <br /> the firA gene. The wild-type firA gene cloned pGAH317,             determined described Langdon (23). Cellular <br /> omsA allele cloned CDH23-213. Plasmid pOMSA102 contains the <br /> onsA allele site-directed mutagenesis.                                      enzyme activities determined cells sonicated <br />                                                                                     twice 15 s time amplitude 12 p,m the <br />                                                                                      sonication microtip Soniprep 150 (Measuring &amp; Scien- <br />                                                                                     tific Equipment, Ltd., London, England). <br />   DNA techniques. General DNA manipulations car- <br /> ried described Sambrook et al. (34). Isolation                                                   RESULTS <br /> chromosomal DNA modified method of <br /> Marmur described previously (22). Nucleotide sequences                              Antibiotic-susceptible phenotype oms4 mutant. We <br /> were determined chain termination method (35)                           verified extended finding Tsuruoka et al. (39) <br /> Sequenase (United States Biochemical, Cleveland, Ohio).                             omsA mutant strain CDH23-213 remarkably suscep- <br /> Site-directed mutagenesis using oligonucle-                         tible certain antibiotics. The MICs rifampin fusidic <br /> otide mutagenesis kit, version 2 (Amersham, Buckingham-                             acid approximately 250-fold lower omsA mutant <br /> shire, England), Bluescript SK+ (Stratagene, La Jolla,                          strain isogenic omsA + sister transductant <br /> Calif.) vector.                                                              (CDH23-210), erythromycin, mupirocin, and <br />    Southern hybridization. Southern blotting                         clindamycin 128-, 64-, 16-fold lower, respectively <br /> alkaline conditions positively charged nylon mem-                              (Table 2). The omsA mutant strain susceptible to <br /> branes (Boehringer, Mannheim, Germany). The filter                              bacitracin vancomycin (Table 2). Common these <br /> prehybridized 20 ml hybridization buffer (50% form-                         antibiotics fact penetrate entero- <br /> amide, Sx SSC [lx SSC 0.15 M NaCl plus 0.015 M                                   bacterial OM water-filled, narrow porin chan- <br />  7092         VUORIO AND VAARA                                                                                                                     J. BACTERIOL. <br />  <br />  TABLE 2. Complementation onsA mutant phenotype                            carries ssc+ inserted gene, introduced <br />   determined measurement MICs various antibioticsa                          CDH23-213. The transformant resistant all <br />                                         MIC (,ug/ml) for:                            antibiotic probes tested omsA + control <br />                                                                                      (Table 2). Also, ability grow 42&#194;&#176;C resumed. <br />       Antibiotic    CDH23-210      CDH23-213 CDH23-213/ CDH23-213/                   Accordingly, ssc fully complemented omssA mutant <br />                       (omsA+)        (omtsA) pUCHS14-II    pUCHS16A                  phenotype. Another pUC19-derived plasmid, pUCHS16A <br />                                              (omsA sscl) (omsA4 ompHl) <br />                                                                                      (17), lacks ssc contains 5&apos;-flanking gene <br /> Fusidic acid                 512     2                   512        2                ompH, did elicit complementation (Table 2). <br /> Rifampin                       4     0.015                 2        0.015               Complementation different ssc alleles. To study further <br /> Clindamycin                  128     8                   128        8                ability ssc gene reverse E. coli omsA <br /> Bacitracin            &gt; 1,024       32                .1,024       32 <br /> Erythromycin               32        0.25                 64        0.5              mutation, tested various ssc mutant alleles this <br /> Vancomycin                128       32                   128       32                property. Plasmids pSSC101 pSSC112 (Table 1) contain <br /> Mupirocin                  64        1                    64        0.5              12 ssc alleles previously site-directed mutagenesis <br />                                                                                      (19). pSSC101 corresponds ssc-1 mutant allele <br />   MICs determined 28&#194;&#176;C L broth (supplemented ampicillin            (originally S. typhimurium mutant strain SH7622), <br /> when appropriate).                                                                   rest alleles encode mutant Ssc proteins, <br />                                                                                      having single amino acid alteration. The ability of <br />                                                                                      plasmids complement omsA mutation E. coli <br /> nels, hydrophobic molecules (rifampin,                         (Table 3) appeared similar ability to <br /> fusidic acid, erythromycin, mupirocin, clindamycin)                           complement ssc mutation S. typhimunium (19). Ac- <br /> large, hydrophilic molecules (bacitracin vancomycin)                             cordingly, pSSC101, pSSC109, pSSC110, pSSC111, <br /> (28, 30, 37, 40). In contrast, sensitization slight                       encode mutant Ssc proteins substitutions of <br /> sensitization antibiotics cefuroxime benzyl-                          Val-291-*Met, Met-288--Lys, Met-290-Lys, Met- <br /> penicillin, use porin channels, observed (data                         292-*Lys, respectively, caused partial complementa- <br /> not shown; reference 39). Furthermore, like                           tion. The defective ssc allele (Gly-289- &apos;Asp; carried by <br /> enteric bacteria defective damaged OM (28,                           pSSC112), potentiates antibiotic-sensitive <br /> 37, 40), omsA strain showed increased susceptibility                          phenotype SH7622 (19), completely unable reverse <br /> to aminoglycoside antibiotics (data shown;                              omsA phenotype. The ssc allele pSSC102 (Met- <br /> reference 39).                                                                       288--Leu) omsA strain fully resistant (Table 3) but <br />   oms4 mutant phenotype complemented ssc. As de-                               ssc-1 mutant partially resistant (19). The <br /> tailed introduction, gene mapping data                           remaining pSSC plasmids omsA mutant non- <br /> comparison phenotypic properties prompted                               defective, exactly changed sscl mutant. Put <br /> search possible relationship omsA firA.                            together, results support idea that <br /> Since previously cloned sequenced firA                                omsA allelic tofirA, E. coli analog ssc gene S. <br /> analog S. typhimunum, ssc gene (16), tested                                typhimunium. <br /> whether complement omsA mutation. We em-                                  Cloning sequencing oms4 allele offirA. We then <br /> ployed pUC19-derived multicopy plasmid pUCHS14-II (17),                              cloned sequenced thefirA gene (i.e., omsA allele) of <br />  <br />  <br />                    TABLE 3. Ability plasmid-encoded ssc firA alleles complement omsA mutation CDH23-213 <br />  <br />             Strain/plasmid                   ssc firA allele      Mupirocin <br />                                                                                                 Sensitivity&apos; to:Bacitracin <br />                                                                                         Vancomycin                                Rifampin <br />                                                                                                                                                        Sensitivity <br />                                                                                                                                                      interpretationb <br />                                                 plasmid <br />  <br /> CDH23-213                                        None                 13 (24)              11 (18)              17 (27)            29 (20)             SS (SS) <br /> CDH23-213/pUCHS14-II                             ssc+                  0                    2                    0                 14                  R <br /> CDH23-213/pSSC101                                ssc-101               3                    9                   10                 21                  S <br /> CDH23-213/pSSC102                                ssc-102               0                    4                    0                 16                  R <br /> CDH23-213/pSSC103                                ssc-103               0                    4                    0                 16                  R <br /> CDH23-213/pSSC104                                ssc-104               0                    4                    0                 15                  R <br /> CDH23-213/pSSC105                                ssc-0os               0                    4                    0                 16                  R <br /> CDH23-213/pSSC106                                ssc-106               0                    3                    0                 15                  R <br /> CDH23-213/pSSC107                                ssc-107               0                    3                    0                 14                  R <br /> CDH23-213/pSSC108                                ssc-108               0                    2                    0                 14                  R <br /> CDH23-213/pSSC109                                ssc-109               2                    9                    8                 20                  S <br /> CDH23-213/pSSC110                                ssc-11O               0                   10                   11                 21                  S <br /> CDH23-213/pSSC111                                ssc-111               0                   10                   10                 20                  S <br /> CDH23-213/pSSC112                                ssc-112              18                   13                   20                 35                  SS <br /> CDH23-213/pOMSA101                              omsA                   0 (6)                2 (4)                0 (4)             12   (12)           R (S) <br /> CDH23-213/pOMSA102                              omnsAc                 0 (4)                2 (3)                0 (3)             13   (10)           R (S) <br /> CDH23-213/pOMSA103                              firA+                  0 (0)                1 (1)                0 (0)             11   (7)            R (R) <br /> CDH23-213/pGAH317                               firA+                  0                    1                    0                 13                  R <br /> CDH23-210 (omsA+ control)                       None                   0 (0)                1 (0)                0 (0)             13   (8)            R (R) <br />   a <br />     Sensitivity antibiotics measured inhibition zones (in millimeters) antibiotic discs. Determinations L agar plates (with ampicillin <br /> [100 pg/mlJ plasmids used) 28&apos;C. In parentheses corresponding values obtained 39&apos;C. The diameter antibiotic disc (9 mm) is <br /> subtracted. <br />   b R, resistant; S, sensitive; SS, sensitive. Interpretations parentheses based sensitivity determinations performed 39&apos;C. <br />   c Plasmid pOMSA102 contains firA mutant allele onsA site-directed mutagenesis. <br />  VOL. 174, 1992                                                                           OUTER MEMBRANE GENES omsA AND firA                    7093 <br />  <br /> TABLE 4. Complementation antibiotic-sensitive phenotype                          1000 <br />     CDH23-213 plasmids containing omsA allele <br />                   determined 37&#194;&#176;C&quot; <br />                        firA                   MIC (pg/ml) of: <br /> Strain/plasmid      aliele   Vanco-     Fusidic   Ram            Erythro-    0 <br />                                                                                 0- <br />                     plasmid     mycin        acid     R              mycin <br />                                                                                 0        100 <br /> CDH23-213          None              8          2       0.016         0.125     1-0 <br /> CDH23-213/         omsA             32         64       1.5           3 <br />   pOMSA101                                                                      E <br /> CDH23-213/         o                32         48         1           2         co <br />                                                                                 o <br />   pOMSA102 <br /> CDH23-213/         firA+         &gt;256       &gt;256          6          12 <br />   pOMSA103                                                                                10 <br /> CDH23-210           None         &gt;256       &gt;256          8         32          0 <br />   MICs determined using E Test (see Materials Methods) <br /> in L agar plates (with ampicillin [100 mg/ml] plasmid-containing strains <br /> were studied). <br />    b Prepared site-directed mutagenesis. <br />  <br />  <br />                                                                                                                                                   10 <br /> CDH23-213. The firA-containing chromosomal 1.4-kb PstI <br /> fragment ligated pUC19 identified Southern                                                              Time (min) <br /> hybridization. Also, wild-type firA gene (from plasmid                                      11 <br /> pGAH317) cloned pUC19. The recombinant plasmids                                   10 <br /> were tested ability complement omsA <br /> mutation CDH23-213. If firA gene proximal lac <br />                                                                                                          B <br />                                                                                                10 <br /> and opposite orientation, plasmids withfirA +                            10 <br /> elicited complementation (data shown). On other <br /> hand, wild-type firA gene orientation                     a <br />                                                                                                 9 <br /> as lac promoter distal (pOMSA103), fully                        c <br />  <br />                                                                                           10 <br /> complemented omsA phenotype (Table 3). The identical <br /> construction containing omsA allele (pOMSA101) also <br /> complemented phenotype. However, comple-                            E <br />                                                                                  0 <br />  <br />                                                                                          10 <br />                                                                                                a <br /> mentation tests performed 39&#194;&#176;C (instead 28&#194;&#176;C), <br /> pOMSA101 proved complement weakly                                      0 <br />  <br /> pOMSA103 (Table 3). Consistent difference ob-                                         7 <br /> served Table 3, MICs vancomycin, erythromycin,                        &apos;   0    10 <br /> rifampin, fusidic acid 37&#194;&#176;C four- eightfold                      0 <br />  <br />  <br />  <br /> lower CDH23-213 carried pOMSA101                                              6 <br /> carried pOMSA103 (Table 4).                                                               10 <br />    The omsA allele sequenced using oligonucleotide <br /> primers. Compared wild-type firA sequence (5), the <br /> sequence identical <span id='am-3' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-4' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-5' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span>point mutation</span>. There <br /> was G-to-A transition nucleotide 812 (the numbering                                          0       1   00   200      300   400        500 <br /> consistent Fig. 4 reference 5). This changed codon <br /> 271 serine asparagine. The mutation                                                     Time (min) <br /> made site-directed mutagenesis. This omsA allele be-                           FIG. 1. Thermosensitivity growth determined strain <br /> haved way original omsA allele                           pairs analogous mutant alleles. CDH23-213 (omsA) its <br /> complementation assays (Tables 3 4).                                        isogenic wild-type control, CDH23-210, SH7622 (ssc-1) its <br />    The thermosensitive phenotype ssc-l mutant com-                       wild-type parent, SH5014, used. Growth measured asA550 <br /> pared oms,4 mutant. As omsA ssc-1                          (A), viability determined CFU/milliliter (B). The cultures <br /> mutations allelic, compared thermosensitive                          incubated 28&#194;&#176;C reached optical density (OD) of <br /> phenotypes caused mutations. After tem-                        approximately 0.2 0.3 (arrows), temperature was <br /> perature shifted 28 42&#194;&#176;C, growth of                     shifted 42&#194;&#176;C. Symbols: *, CDH23-210; 0, CDH23-213; *, <br /> the mutants stopped 1.5 h (Fig. 1). The CFU/milliliter                    SH5014; 0, SH7622. <br /> fell 1 2 log units SH7622 nearly 3 log units in <br /> CDH23-213. <br />     One distinct property omsA mutant                         37&#194;&#176;C) did induce marked leakage periplasmic <br /> releases periplasmic proteins temperature shift                       enzyme   SH7622/pUC19. The activity medium as <br />  (39). To study characteristic ssc-1               percentage total activity 82% 42&#194;&#176;C 4% at <br /> mutant, introduced plasmid vector pUC19 (which en-                           37&#194;&#176;C (the values control plasmid 2 0.3%, <br />  codes P-lactamase) SH7622. The SH7622 derivative                          respectively). Accordingly, OM SH7622 func- <br /> which carries pUC19-derived pUCHS14-II (which encodes                           tionally defective 42&#194;&#176;C allowed leakage the <br /> both 3-lactamase wild-type Ssc) used control.                      periplasmic contents. In contrast, protein leakage through <br />  As shown Fig. 2, temperature shift 28 42&#194;&#176;C (but                   cytoplasmic membrane minimal 42&#194;&#176;C; less <br />  7094                VUORIO AND VAARA                                                                                                                  J. BACT1ERIOL. <br />  <br />            100000 <br />                                                                                                      They concluded FirA physical association the <br />                                                                                                      transcriptional machinery directly involved tran- <br />                                                                                                      scription (5). We, however, linkedfirA apparently very <br />                                                                                                      different cellular function finding point mutation in <br />                                                                                                      ssc gene, firA analog S. typhimurium, results in <br />             GM0                                        E                                             thermosensitivity and, permissive temperature, a <br />                                                        B                                             drastic increase susceptibility large set hydropho- <br />  -2w <br />                                                        -20                                           bic antibiotics, dyes, detergents (17). Since unselec- <br />  -C                                                    0 <br />                                                                                                      tive increase susceptibility hydrophobic agents indi- <br />                                                                                                      cates defect OM permeability barrier agents, <br />                                                                                                      suggested firA gene involved generation of <br />                                                                                                      OM. <br />                                                                                                         The present investigation studied conditionally lethal <br />                                                                                                      omsA mutant E. coli, shown Tsuruoka <br />                                                                                                      et al. (39) defective OM permeability barrier <br />                                  Time (min) <br />                                                                                                      function growth-permissive temperatures. We also <br />                                                                                  lime (min)          showed mutation <span id='am-237' about='xsp:base' typeof='owl:Thing'>base</span> pair substitution <br />                                                                                                      firA, i.e., gene mutated above-mentioned <br />                                                                                                      phenotypically similar ssc-1 mutant S. typhimurium. <br />                                                                                                      Furthermore, results revealed OM firA <br />                                                                                                      mutants ruptured nonpermissive temperatures, <br />                         c                                                  D                         indicated leakage periplasmic 3-lactamase into <br />                160- 37                         X                11so       42C <br />                                                                                                      medium. Even interestingly, able show <br />                                                                                                      rupturing took place conditions the <br />        E <br />                                                                                                      cytoplasmic membrane remained impermeable cytoplas- <br />    .3           120 -                                    iE                                          mic enzymes. These results strengthen concept <br />                                                                                                      firA gene essential function OM biosyn- <br />                                                            2% <br />  <br />                                                            &quot;I <br />                                                                                                      thesis. Furthermore, results strongly suggest that <br />    -C <br />                   80-                                    &apos;a                                          gene functions identically E. coli S. typhimurium, <br />                                                                                                      proposed previously (17). <br />                                                                                                         The firA genes S. typhimurium (16) E. coli (5) are <br />                 40-                                              40                                  88% identical. The corresponding proteins molecular <br />                                                                                                      mass 36 kDa 96% identical. Downstream firA, <br />                                                                                                      560 bp apart, lie lpxA4 (3) lpxB (3, 4). These the <br />                     0       30      60    90   120                     0    30      60    90   120   known genes lipid A biosynthesis be <br />                                                                                                      cotranscribed firA (see references reference 6). <br />                              Time (min)                                          Tlme (min)          When firA S. typhimurium compared the <br />    FIG. 2. Release periplasmic amp-encoded 1-lactamase                                        bacterial genes GenBank library, best sequence <br /> cytoplasmic glucose 6-phosphate dehydrogenase ssc-1 mutant                                      homology lpx4 (47). Consistently, all <br /> strain SH7622. SH7622 carrying pUC19 (amp, open symbols)                                      SWISS-PROT protein sequences, best match with <br /> or pUCHS14-II (amp ssc+, closed symbols) shifted 28                                      FirA E. coli UDP-acetylglucosamine O-acyltransferase <br /> 37&apos;C (A C) 42&#194;&#176;C (B D) time zero. 1-Lactamase (A                                     (the LpxA protein) (47). The FirA LpxA proteins share <br /> and B) glucose 6-phosphate dehydrogenase (C D) activities                                    long regions distinct homology (homology allowing <br /> in growth medium (0, 0) corresponding cellular <br /> activities (5, *) shown. Units nanomoles minute.                                         equivalent amino acids, 43%) peculiar hexapeptide <br />                                                                                                      repeat theme acetyltransferases (6, 41, <br />                                                                                                      47). This homology LpxA suggests FirA involved <br />                                                                                                      lipid A biosynthesis function which <br /> than 3% total activity cytoplasmic marker                                              resembles LpxA, i.e., acyltransferase function. <br /> enzyme (glucose 6-phosphate dehydrogenase)                                              At enzymes directly involved in <br /> the medium 2 h 42&#194;&#176;C (Fig. 2).                                                               lipid A synthesis await molecular characterization (33). <br />                                                                                                      These include UDP-3-0-(3-hydroxymyristoyl)-acetylglucos- <br />                                                DISCUSSION                                            amine deacetylase UDP-3-0-(3-hydroxymyristoyl)-glu- <br />                                                                                                      cosamine N-acyltransferase (which catalyze second and <br />    The firA mutant (firA200) E. coli described                                       steps synthesis, respectively), lipid A 4&apos; kinase, and <br /> 1970 (24). It rifampin-sensitive phenotypic revertant                                          late acyltransferases (i.e., lauroyl myristoyl trans- <br /> (hence, fir versus nij) E. coli rpoB mutant strain,                                     ferases). Work progress laboratory elucidate <br /> carries rifampin-resistant P subunit RNA polymerase.                                          possible functional role FirA biogenesis lipid <br /> The MIC rifampin firA rpoB strain approxi-                                            A. Indirect evidence favoring role comes our <br /> mately threefold lower MIC rpoB strain (25).                                        finding rate lipid A synthesis, relative of <br /> Furthermore, firA rpoB strain thermosensitive                                            phospholipid synthesis, drastically decreased firA <br /> its RNA synthesis halted 42&#194;&#176;C (25). On basis                                          mutant 42&#194;&#176;C (13). Furthermore, mature LPS molecules <br /> evidence, firA mutation believed affect RNA                                               produced firA mutant 42&#194;&#176;C modified contain <br /> transcription. Later, Dicker Seetharam (5) constructed                                           palmitic acid (12, 13). <br /> strains overproduce product offirA, 36-kDa FirA                                           The lpx42 mutant E. coli isolated characterized by <br /> protein, showed antiserum RNA polymerase                                              Galloway Raetz best-characterized mutant strain <br /> holoenzyme coimmunoprecipitates FirA strains.                                              defective lipid A biosynthesis (7). It synthesizes drasti- <br />  VOL. 174, 1992                                                           OUTER MEMBRANE GENES omsA AND firA                    7095 <br />  <br /> cally reduced amounts lipid A 42&#194;&#176;C rapidly dies       (see above) encodes protein total different <br /> that temperature. We recently shown (48)            amino acid alterations, Gln-165--*Arg, Gly-228--&gt;Asp, and <br /> strain extremely susceptible hydrophobic antibiotics         Gly-252--iSer (5). Its spontaneous mutation, firA201, not <br /> (including rifampin, fusidic acid, erythromycin, clinda-       rifampin sensitive encodes protein pre- <br /> mycin) growth-permitting temperatures. Their MICs           served Gly-228- Asp Gly-252-*Ser mutations and <br /> 32- &gt;128-fold lower strain parental       lacks Gln-165--&gt;Arg mutation Ile-172--Phe in- <br /> type strain, MICs antibiotics use      stead (5). On basis preliminary transcomplementation <br /> porin channels OM unchanged. The            studies, Dicker Seetharam suggested Gly- <br /> very low MIC values hydrophobic antibiotics deter-          228--Asp Gly-252--Ser responsible ther- <br /> mined lpxA2 strain reminiscent deter-         mosensitive rifampin-sensitive phenotypes the <br /> mined omsA mutant present study. The lpxA2          firA200 mutant (6). They noted these <br /> and omsA mutants antibiotic-supersusceptible          mutations located turn residues (i.e., residue b, c, d <br /> enterobacterial strains published. Accordingly,           imperfect hexapeptide tandem repeat units) favor <br /> lpxA4 firA mutants phenotypically similar.            formation hypothetical 3-turn structure protein <br />    The 5&apos;-flanking region firA occupied hlpA (also        (6). However, Ser-271--&gt;Asn Val-291--&gt;Met is <br /> known ompH), encodes cationic 17-kDa protein              located residues. Regarding 11 novel FirA <br /> HLP-I (histone-like protein I, OmpH protein). This             proteins site-directed mutagenesis firA, those <br /> protein conservative structure mem-           Met-288-*Lys Gly-289--&gt;Asp defective <br /> bers family Enterobacteriaceae (15, 18, 20, 22, 43, 46),    ones complementation assays, followed proteins <br /> carries signal sequence characteristic secreted       Met-290-.Lys Met-292--Lys. The plasmid that <br /> proteins precursor form (18, 20, 21, 43),      encodes Gly-289--Asp appears sensitize ssc-1 and <br /> in isolated OM (18, 22), unknown function. Only         omsA mutants (reference 19 study) has <br /> three nucleotides separate hlpA stop firA initiation       negligible effect sensitivity firA+ strain <br /> codons (5, 16). We previously shown using minicells        (19). This taken suggest mutant protein is <br /> and complementation assays firA gene S.             defective proteins Val-291--oMet <br /> typhimurium expressed absence hipA pro-           Ser-271--&gt;Asn competes binding <br /> moter, i.e., plasmid lacks 5&apos; terminus hlpA      hypothetical ligand. Also, Gly-289--Asp mutation <br /> and vector-derived lac promoter opposite           does map hypothetical turn residues. There <br /> orientation firA (17). In present report,      empirical data tertiary structure FirA <br /> this construction able complement omsA              protein. In absence crystallographic data, is <br /> mutant allele, analogous construction containing          difficult draw conclusions regarding effect of <br /> the firA gene E. coli inactive. Only firA inserts   identified amino acid alterations conformation of <br /> E. coli whichfirA lac promoter comple-        protein. <br /> menting activity. We previously suggested firA           The various firA mutant strains certainly expected <br /> S. typhimurium use promoter            differ extents functional lesions. The firA201 <br /> identified putative promoter region 159 nucleotides up-          mutant apparently carries defective allele the <br /> stream overlaps protein-encoding region hlpA           firA200 mutant, supersusceptible rifampin <br /> (16). In E. coli, region homologous            conditions firA200 strain (5, 6). Among <br /> canonical promoter sequence. Accordingly, sug-         phenotypically characterized mutants, onmsA mutant <br /> gested firA normally cotranscribed hipA      defective. Even phenotypes the <br /> the firA gene S. typhimurinum utilize,         omsA ssc-1 mutants similar, differences <br /> extent, additional promoter mentioned above.                    found. On basis MIC values (reference 17 <br />    When present multicopy plasmid lac              study), conclude omsA mutant even <br /> promoter, omsA mutant allele able partially com-          susceptible tested antibiotics the <br /> plement phenotype chromosomal omisA mutant.             ssc-1 mutant. Furthermore, leaks periplasmic enzymes <br /> This probably overexpression plasmid-borne         37&#194;&#176;C (39), ssc-1 mutant leakage was <br /> gene construction. An analogous construction              detected temperature (this study). Moreover, the <br /> containing firA200 mutant allele (see below) comple-            omsA mutant releases cytoplasmic constituents <br /> ments rifampin-sensitive phenotype firA200 mu-           starts lyse (39), ssc-1 strain releases <br /> tant (Table 4 reference 5). Apparently, overex-          exclusively periplasmic material. In elucidation the <br /> pression takes place plasmids S. typhimuinum            function FirA, defective firA mutant (i.e., the <br /> firA lac control uses additional          omsA strain) expected informative <br /> promoter (as suggested above), since, shown              one. <br /> report, defectivefirA alleles S. typhimurium (ssc-1         Since firA lpx42 mutants extremely supersus- <br /> the defective missense base pair substitution mutants          ceptible hydrophobic agents, previously sug- <br /> by site-directed mutagenesis) complement omsA pheno-            gested (19, 48) useful tools studies of <br /> type weakly complement (whereas        bacterial physiology, applied bacteriology and <br /> wild-type firA S. typhimurium nondefective            biotechnology, fields permeability barrier <br /> vitro mutant alleles complement fully).                          properties OM potentially pose problems. Further- <br />    The mutant FirA protein described report        more, phenotypic revertants types mutants could <br /> one amino acid change, Ser-271- Asn. Also, FirA pro-            help evaluate lipid A essential molecule. <br /> tein encoded ssc-1 allele single alteration,           The evidence assumption lipid A its <br /> Val-291-)Met (17). Accordingly, thermosensitive            attached ketodeoxyoctulosonates essential via- <br />  antibiotic-supersusceptible phenotypes conferred        bility bacteria comes findings lpx42 <br />  single mutation. On hand, rifampin-sensitive       mutants kds mutants (defective ketodeoxyoctu- <br />  thermosensitive phenotype-causing mutation firA200             losonate synthesis) conditionally lethal inhibi- <br />  7096      VUORIO AND VAARA                                                                                                    J. BACTERIOL. <br />  <br /> tors ketodeoxyoctulosonate synthesis antibacterial (7,                murium. EMBO J. 10:1017-1023. <br /> 9-11, 31, 33). Whether bacteria live lipid A              18. Hirvas, L., P. Koski, M. Vaara. 1991. The ompH gene of <br /> suitable bypass mutations present remains                     Yersinia enterocolitica: cloning, sequencing, expression, and <br /> seen. Since OM certainly essential pathogenic <br />                                                                              comparison known enterobacterial ompH sequences. J. <br />                                                                              Bacteriol. 173:1223-1229. <br /> bacteria (44), inhibitors lipid A biosynthesis             19. Hirvas, L., M. Vaara. 1992. Effect Ssc protein inutations <br /> very promising new class antibacterial agents. Molecular                  outer membrane permeability barrier function Salmo- <br /> characterization of enzymes involved lipid A                   nella typhimurium: study using ssc mutant alleles by <br /> biosynthesis greatly advance development                   site-directed mutagenesis. FEMS Microbiol. Lett. 90:289-294. <br /> agents.                                                                  20. Hoick, A., K. Kleppe. 1988. Cloning sequencing the <br />                                                                              gene DNA-binding 17 K protein Eschenichia coli. <br />                     ACKNOWLEDGMENTS                                          Gene 67:117-124. <br />                                                                          21. Koski, P., L. Hirvas, M. Vaara. 1990. Complete sequence of <br />   We thank B. Kuusela excellent technical assistance T.              ompH gene encoding 16-kDa cationic outer membrane <br /> Tsuruoka providing strains CDH23-210 CDH23-213.                  protein SalmoneUa typhimurium. Gene 88:117-120. <br />   This study supported grant 1011749 Academy          22. Kosld, P., M. Rhen, J. Kantele, M. Vaara. 1989. Isolation, <br /> Finland (to M.V.) Sigrid Juselius Foundation (M.V.            cloning, primary structure cationic 16-kDa outer <br /> R.V.).                                                                       membrane protein Salmonella typhimurium. J. Biol. Chem. <br />                                                                              264:18973-18980. <br />                             REFERENCES                                   23. Langdon, R. G. 1966. Glucose-6-phosphate dehydrogenase from <br />  1. Alting-Mees, M. A., J. M. Short. 1989. pBluescript II: gene          erythrocytes. Methods Enzymol. 9:126-131. <br />     mapping vectors. Nucleic Acids Res. 17:9494.                         24. Lathe, R. 1977. Fine-structure mapping thefirA gene, locus <br />  2. Bullock, W. O., J. M. Fernandez, J. M. Short. 1987.                  involved phenotypic expression rifampin resistance in <br />     XL1-Blue: high efficiency plasmid transforming recA Esche-             Escherichia coli. J. Bacteriol. 131:1033-1036. <br />     richia coli strain 0-galactosidase selection. BioTechniques     25. Lathe, R., H. Buc, J.-P. Lecocq, E. K F. Banz. 1980. <br />     4:376-379.                                                               Prokaryotic histone-like protein interacting RNA polymer- <br />  3. Coleman, J., C. R. H. Raetz. 1988. First committed step           ase. Proc. Natl. Acad. Sci. USA 77:3548-3552. <br />     lipid A biosynthesis Escherichia coli: sequence lpx4       26. Lugtenberg, B., L. van Alphen. 1983. Molecular architec- <br />     gene. J. Bacteriol. 170:1268-1274. <br />                                                                              ture functioning outer membrane Eschenichia coli <br />  4. Crowell, D. N., W. S. Reznikoff, C. R. H. Raetz. 1987.               gram-negative bacteria. Biochim. Biophys. Acta <br />     Nucleotide sequence Escherichia coli gene lipid A             737:51-115. <br />     disaccharide synthase. J. Bacteriol. 169:5727-5734.                  27. Messing, J., J. VMeira. 1982. A new pair M13 vectors for <br />  5. Dicker, I. B., S. Seetharam. 1991. Cloning nucleotide            selecting DNA strand double-digest restriction frag- <br />     sequence firA gene firA200(Ts) allele                ments. Gene 19:269-276. <br />     Escherichia coli. J. Bacteriol. 173:334-344.                         28. Nikaido, H. 1976. Outer membrane Salmonella typhimurium. <br />  6. Dicker, I. B., S. Seetharam. 1992. What known           Transmembrane diffusion hydrophobic substances. <br />     structure function Escherichia coli protein FirA?             Biochim. Biophys. Acta 433:118-132. <br />     Mol. Microbiol. 6:817-823.                                           29. Nikaido, H. 1990. Permeability lipid domains bacterial <br />  7. Gailoway, S. M., C. R. H. Raetz. 1990. A mutant                   membranes. Adv. Membr. Fluid. 4:165-190. <br />     Escherichia coli defective step endotoxin biosyn-    30. Nikaido, H., M. Vaara. 1985. Molecular basis bacterial <br />     thesis. J. Biol. Chem. 265:6394-6402.                                    outer membrane permeability. Microbiol. Rev. 49:1-32. <br />  8. Gibson, T. J. 1984. Studies Epstein-Barr virus genome.        31. Nikaido, H., M. Vaara. 1987. Outer membrane, p. 7-22. In <br />     Ph.D. thesis. Cambridge University, Cambridge.                           F. C. Neidhardt, J. L. Ingraham, K. B. Low, B. Magasanik, M. <br />  9. Goldman, R. C., C. C. Doran, J. 0. Capobianco. 1990.                 Schaechter, H. E. Umbarger (ed.), Escherichia coli and <br />     Antibacterial agents specifically inhibit lipopolysaccha-          Salmonella typhimunum: cellular molecular biology. Amer- <br />     ride synthesis, p. 157-167. In A. Nowotny, J. J. Spitzer,            ican Society Microbiology, Washington, D.C. <br />     J. R. Sokatch (ed.), Cellular molecular aspects endotoxin     32. O&apos;Callaghan, C. H., A. Morris, S. M. Kirby, A. H. Shigler. <br />     reactions. Elsevier Science Publishers B.V., Amsterdam.                  1972. Novel method detection 1-lactamases using a <br /> 10. Goldman, R. C., W. E. Kohlbrenner, P. Lartey, A. Pernet.             chromogenic cephalosporin substrate. Antimicrob. Agents Che- <br />     1987. Antibacterial agents specifically inhibiting lipopolysaccha-       mother. 1:283-288. <br />     ride synthesis. Nature (London) 329:162-164.                         33. Raetz, C. R. H. 1990. Biochemistry endotoxins. Annu. Rev. <br /> 11. Hammond, S. M., A. Claesson, A. M. Jansson, L. G. Larsson,               Biochem. 59:129-170. <br />     B. G. Pring, C. M. Town, B. Ekstr6m. 1987. A new class        34. Sambrook, J., E. F. Fritsch, T. Maniatis. 1989. Molecular <br />     synthetic antibacterials acting lipopolysaccharide biosynthe-         cloning: laboratory manual, 2nd ed. Cold Spring Harbor <br />     sis. Nature (London) 327:730-732.                                        Laboratory Press, Cold Spring Harbor, N.Y. <br /> 12. Helander, I. M., L. Hirvas, J. Tuominen, M. Vaara. 1992.         35. Sanger, F., S. Nicklen, A. R. Coulson. 1977. DNA sequenc- <br />     Preferential synthesis heptaacyl lipopolysaccharide            ing chain-terminating inhibitors. Proc. Natl. Acad. Sci. <br />     ssc permeability mutant Salmonella typhimunium. Eur. J.               USA 74:5463-5467. <br />     Biochem. 204:1101-1106.                                              36. Stocker, B. A. D., M. Nurminen, P. H. Mikeli. 1979. <br /> 13. Helander, I. M., B. Lindner, U. Seydel, M. Vaara. Submit-            Mutants defective 33K outer membrane protein Sal- <br />     ted publication.                                                     monella typhimurium. J. Bacteriol. 139:376-383. <br /> 14. Helander, I. M., M. Vaara, S. Sukupolvi, M. Rhen, S. Saarela,        37. Sukupoli, S., M. Vaara. 1989. Salmonella typhimurium and <br />     U. Zahringer, P. H. Mikela. 1989. rfaP mutants Salmo-             Escherichia coli mutants increased outer membrane per- <br />     nella typhimurium. Eur. J. Biochem. 185:541-546.                         meability hydrophobic compounds. Biochim. Biophys. Acta <br /> 15. Hirvas, L., J. Coleman, P. Koski, M. Vaara. 1990. Bacterial          988:377-387. <br />     &quot;histone-like protein I&quot; (HLP-I) outer membrane constit-       38. Sukupolvi, S., M. Vaara, I. M. Helander, P. VlUanen, P. H. <br />     uent? FEBS Lett. 262:123-126.                                            Mikela. 1984. New Salmonella typhimurium mutants al- <br /> 16. Hirvas, L., P. Koski, M. Vaara. 1990. Primary structure          tered outer membrane permeability. J. Bacteriol. 159:704-712. <br />     expression Ssc-protein Salmonella typhimurium. Bio-        39. Tsuruoka, T., M. Ito, S. Tomioka, A. Hirata, M. Matsuha- <br />     chem. Biophys. Res. Commun. 173:53-59.                                   shi. 1988. Thermosensitive onsA mutation Escherichia coli <br /> 17. Hirvas, L., P. Koski, M. Vaara. 1991. Identification             causes thermoregulated release periplasmic proteins. J. <br />     sequence analysis gene mutated conditionally lethal        Bacteriol. 170:5229-5235. <br />     outer membrane permeability mutant SS-C Salmonella typhi-         40. Vaara, M. 1992. Agents increase permeability the <br />  VOL. 174, 1992                                                              OUTER MEMBRANE GENES              omsA   AND firA        7097 <br />  <br />     outer membrane. Microbiol. Rev. 56:395-411.                       46. Vuorio, R., L. Hirvas, R. B. Raybourne, D. T. Y. Yu, M. <br /> 41. Vaara, M. Eight bacterial proteins, including UDP-N-acetylglu-        Vaara. 1991. The nucleotide deduced amino acid sequence <br />     cosamine acyltransferase (LpxA) transferases       cationic 16 kDa outer membrane protein OmpH of <br />     Eschenchia coli, consist six-residue periodicity theme.          Yersinia pseudotuberculosis. Biochim. Biophys. Acta 1129: <br />     FEMS Microbiol. Lett., press.                                      124-126. <br /> 42. Vaara, M. Quantitative antimicrobial susceptibility testing    47. Vuorio, R., L. Hirvas, M. Vaara. 1991. The Ssc protein of <br />     outer membrane-defective mutant bacteria using E test.         enteric bacteria significant homology acyltransferase <br />     J. Antimicrob. Chemother., press.                                  LpxA lipid A biosynthesis, acetyltransferases. <br /> 43. Vaara, M., L. Hirvas, P. Koski. 1990. The cationic 16-kDa         FEBS Lett. 292:90-94. <br />     outer membrane protein OmpH enteric bacteria, p. 197-204. <br />     In A. Nowotny, J. J. Spitzer, E. J. Ziegler (ed.), Cellular   48. Vuorio, R., M. Vaara. 1992. The lipid A biosynthesis <br />     molecular aspects endotoxin reactions. Elsevier Science        mutation lpx42 Escherichia coli results drastic antibiotic <br />     Publishers B.V., Amsterdam.                                           supersusceptibility. Antimicrob. Agents Chemother. 36:826- <br /> 44. Vaara, M., H. Nikaido. 1984. Molecular organization            829. <br />     bacterial outer membrane, p. 1-45. In E. T. Rietschel (ed.),      49. Wickner, W., A. J. Driessen, F. U. Hartl. 1991. The <br />     Chemistry endotoxin. Elsevier Science Publishers B.V.,             enzymology protein translocation Escherichia coli <br /> </body></html>