<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Divergent Mechanisms Controlling Hypoxic Sensitivity <br /> and Lifespan DAF-2/Insulin/IGF-Receptor Pathway <br /> Meghann E. Mabon1,3, Barbara A. Scott1, C. Michael Crowder1,2,3* <br /> 1 Department Anesthesiology, Washington University School Medicine, St. Louis, Missouri, United States America, 2 Department Developmental Biology, <br /> Washington University School Medicine, St. Louis, Missouri, United States America, 3 The Division Biology &amp; Biomedical Sciences, Program Developmental <br /> Biology, Washington University School Medicine, St. Louis, Missouri, United States America <br />  <br />  <br />  <br />      <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />      Organisms cells vary greatly tolerance low oxygen environments (hypoxia). A delineation the <br />      determinants hypoxia tolerance incomplete, despite intense implications diseases stroke and <br />      myocardial infarction. The insulin/IGF-1 receptor (IGFR) signaling pathway controls <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span>survival</span> Caenorhabditis elegans a <br />      variety stressors including aging, hyperthermia, hypoxia. daf-2 encodes C. elegans IGFR homolog primary <br />      signaling pathway modulates activity FOXO transcription factor DAF-16. DAF-16 regulates transcription a <br />      large number genes, shown control aging. <span id='am-3' about='protege:TO' typeof='owl:Thing'>To</span> identify genes selectively regulate <br />      hypoxic sensitivity, compared whole-organismal transcriptomes daf-2 reduction-of-function alleles, of <br />      hypoxia resistant, thermotolerant, long lived, differ rank severities phenotypes. The <br />      transcript levels 172 genes increased hypoxia resistant daf-2 allele, e1370, relative alleles <br />      transcripts 10 genes decreased abundance. RNAi knockdown 6 10 genes produced a <br />      significant increase organismal survival hypoxic exposure expected regulation genes <br />      e1370 <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span>mutation</span> responsible hypoxia resistance. However, RNAi knockdown genes did prolong <br />      lifespan. These genes definitively separate mechanisms hypoxic sensitivity lifespan identify biological <br />      strategies survive hypoxic injury. <br />  <br />   Citation: Mabon ME, Scott BA, Crowder CM (2009) Divergent Mechanisms Controlling Hypoxic Sensitivity Lifespan DAF-2/Insulin/IGF-Receptor <br />   Pathway. PLoS ONE 4(11): e7937. doi:10.1371/journal.pone.0007937 <br />   Editor: Immo A. Hansen, New Mexico State University, United States America <br />   Received August 25, 2009; Accepted October 22, 2009; Published November 20, 2009 <br />   Copyright: &#195;&#376; 2009 Mabon et al. This open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: This work supported National Institute Neurological Disorders Stroke - R01 NS045905 (http://www.ninds.nih.gov/), American Heart <br />   Association Established Investigator Award (http://www.americanheart.org/presenter.jhtml?identifier = 3004142), McKnight Endowment Fund for <br />   Neuroscience - Neuroscience Brain Disorders Award (http://www.mcknight.org/neuroscience/). The funders role study design, data collection and <br />   analysis, decision publish, preparation manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: crowderm@morpheus.wustl.edu <br />  <br />  <br />  <br /> Introduction                                                                        various methods used studies different <br />                                                                                     phenotypes screened produced modestly overlapping gene <br />    Low ambient oxygen concentrations induce cell death.                         sets. Thus, DAF-16 appears directly indirectly control the <br /> However, cells vary greatly level duration hypoxia                    expression large number diverse genes influence <br /> that required produce death. Even                       specific daf-2 <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span>phenotype</span>. <br /> organism hypoxic sensitivity range widely. In humans,                              Multiple daf-2 alleles isolated, primarily screens for <br /> neurons cardiac myocytes exquisitely hypoxia sensitive,                     mutants constitutively form dauers. Gems et al. measured <br /> which lead stroke myocardial infarction. Thus,                           phenotypes sixteen daf-2 alleles lack concordance <br /> identification determinants hypoxic sensitivity                  severity different phenotypes cases [13]. <br /> biological medical significance.                                                For example, daf-2(e1370) moderately long lived dauer <br />    We previously shown reduction-of-function mutations                    constitutive thermotolerant alleles tested. <br /> in daf-2 gene confer powerful protection hypoxic injury                 On hand, daf-2(e1368) considerably thermotol- <br /> the nematode Caenorhabditis elegans [1]. daf-2 encodes insulin/                  erant dauer constitutive e1370 a <br /> IGF-receptor homolog signals conserved PI-3                          median lifespan 50% longer e1370. We measured the <br /> kinase signaling cascade regulate negatively activity                   hypoxic sensitivity thirteen daf-2 alleles wide range <br /> FOXO transcription factor, DAF-16 [2,3]. daf-2 reduction-of-                        sensitivities [1]. For example, e1370 hypoxia <br /> function mutants hypoxia resistant                        resistant daf-2 allele tested e1368 weakly resistant. <br /> thermotolerant, long lived, propensity form dauer                     The mechanism underlying divergence phenotypic <br /> larvae, capable long periods hibernation                         severities alleles obscure. <br /> response environmental stress [4]. Because                              In current study utilizing cDNA microarrays double <br /> interesting biological processes controlled daf-2 &#226;&#8364;&#8220; daf-16                   stranded RNA-mediated interference (RNAi) expression of <br /> pathway, considerable effort identify                          candidate genes, advantage daf-2 allelic variations to <br /> downstream targets pathway [5,6,7,8,9,10,11,12]. The                         identify genes regulated daf-2 selectively alter hypoxic <br />  <br />  <br />        PLoS ONE | www.plosone.org                                               1                          November 2009 | Volume 4 | Issue 11 | e7937 <br />                                                                                                                                  daf-2 Hypoxic Death Targets <br />  <br />  <br />  <br />  <br /> Figure 1. daf-2 allelic series hypoxia resistance does correlate daf-2 phenotypes. (A) Survival following recovery <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> 20 hour hypoxia insult (plotted data mean6SD independent experiments). (B) Wild-type (N2) daf-2 mutants assayed for <br /> thermotolerance (mean trials plotted); e1370 (red line), m596 (purple line), e1368 (blue line) N2 (black line). (C) Lifespans N2 and <br /> three daf-2 alleles. Each survival curve generated minimum 77 animals. Animals bagged crawled plate censored. <br /> (D, E, F) Dauer arrest measured 20u (C), 23u (D), 25u N2 daf-2 mutants. Black bars represent adults white bars represent dauers. <br /> All strains statistically different hypoxic survival, thermotolerance, dauer formation (except N2 e1368 D,E and <br /> m596 e1370 F), p,0.01 (two-tailed t-test A, linear regression B, Mantel-Cox Log-rank test C, Fisher&#226;&#8364;&#8482;s exact x2 D, E, F. <br /> doi:10.1371/journal.pone.0007937.g001 <br />  <br /> sensitivity affecting lifespan. The identity genes           weaker allele (Figure 1D,E,F). The results generally consistent <br /> implicates novel pathways selective regulation hypoxic             previously reported Gems et al. [13]. These authors <br /> sensitivity.                                                                  reported e1370 thermotolerant, followed closely by <br />                                                                               m596, distantly e1368. At culture temperature of <br /> Results                                                                       22.5uC, m596 longest lifespan followed e1368 then <br />                                                                               e1370. For dauer constitutive phenotype, order severity <br /> daf-2 Alleles Reveal Divergent Outputs Hypoxic                            found: m596.e1370.e1368. <br /> Survival Lifespan <br />     We chose daf-2 alleles study e1370, m596, e1368.        Transcriptional Profiling daf-2 Alleles <br /> We previously reported recovery 20 hours                      The transcriptional output daf-2 - daf-16 pathway the <br /> hypoxic incubation, organism survival alleles             distinct daf-2 allelic series hypoxia versus lifespan dauer <br /> 96%, 53%, 23%, respectively, compared 4% wild <br /> type strain N2 [1]. Gems et al. scored dauer constitutive, <br /> thermotolerance, long lifespan phenotypes alleles [13].            Table 1. Lifespan analysis daf-2 strains. <br /> However, severities daf-2 phenotypes varied consider- <br /> ably labs, particularly lifespan. Thus, scored the <br /> lifespan, thermotolerance, dauer constitutive, hypoxia resistant            Strain               Mean (d)             Maximum (d)          n <br /> phenotypes daf-2 alleles contemporaneously confirm              N2                   14.9                 29                   87 <br /> their phenotypic divergence. We e1370 was <br />                                                                                 Daf-2(e1368)         18.8*                37*                  86 <br /> strongly hypoxia resistant, m596 moderately resistant, e1368 <br /> weakly (Figure 1A). The order severities alleles              Daf-2(m596)          31.3*                51*                  77 <br /> quite similar thermotolerance (Figure 1B). However, lifespan            Daf-2(e1370)         22.5*{               51*                  86 <br /> m596 strongest allele having mean lifespan 31.3 days <br />                                                                                 Day 1 begins day adulthood. <br /> compared 22.5 days e1370 18.8 e1368 (Figure 1C               * <br />                                                                                  p,0.01 vs N2; <br /> Table 1). For dauer constitutive phenotype, m596               { <br />                                                                                   - p,0.01 vs e1368 m596. <br /> severe allele; e1370 close second e1368                  doi:10.1371/journal.pone.0007937.t001 <br />  <br />  <br />  <br />        PLoS ONE | www.plosone.org                                         2                           November 2009 | Volume 4 | Issue 11 | e7937 <br />                                                                                                                            daf-2 Hypoxic Death Targets <br />  <br />  <br /> formation offer means identify genes daf-2 - daf-16                  expression relative m596 relative e1368. These results <br /> pathway specifically regulate hypoxia. We hybridized                  suggest majority genes regulate lifespan, a <br /> genome oligonucleotide microarrays cDNA derived                        trait e1370 diverges m596 e1368. The <br /> e1370, m596, e1368 L4 stage larval animals grown                        expression 182 genes followed hypoxic <br /> normal conditions. Comparisons transcriptome                       phenotypic allelic series, up- down-regulated the <br /> alleles revealed 182 genes statistically different                     order e1370, m596, e1368 (Table 3). Of these, one, <br /> (p,0.01) e1370 m596 and/or e1368 (Figure 2A);                        C17C3.12, encodes strong mammalian homolog, mitochron- <br /> Raw data deposited NCBI Gene Expression                         drial acyl-CoA dehydrogenase, functions catabolically in <br /> Omnibus database (http://www.ncbi.nlm.nih.gov/geo/), acces-                      mammals fatty acid oxidation. In general, genes involved in <br /> sion number GSE18601. 172 genes up-regulated e1370                       regulation metabolic processes highly represented the <br /> compared m596 and/or e1368 (Table S1), 10                            182 gene list (Figure 2C). Transcriptional/DNA/RNA processing <br /> relatively down-regulated e1370 (Table 2, Figure 2B). The vast                genes abundant. <br /> majority e1370-upregulated genes greater increase                    A number previous studies identified genes whose <br />                                                                                  expression regulated daf-2 [7,8,9,10,11,12,14,15]. These <br />                                                                                  studies implicated large number genes regulated <br />                                                                                  daf-2 - daf-16 pathway. However, overlap the <br />                                                                                  datasets relatively modest presumably distinct <br />                                                                                  methodologies employed study. Perhaps the <br />                                                                                  experimental design microarrays sought gene <br />                                                                                  expression differences daf-2 alleles, overlap between <br />                                                                                  182 genes identified studies was <br />                                                                                  modest seven genes common (Table 4). Four of <br />                                                                                  seven genes expression changes qualitatively <br />                                                                                  similar previously. For example, Y54G11A.13/ <br />                                                                                  ctl-3 gene encodes C. elegans catalases are <br />                                                                                  predicted reduce peroxide free radicals. The ctl-3 transcript is <br />                                                                                  increased e1370 compared e1368 m596. Likewise <br />                                                                                  using quantitative mass spectrometry methods, Dong et al. found <br />                                                                                  CTL-3 protein increased e1370 compared wild type <br />                                                                                  worms [14]. Similarly, data Murphy et al. <br />                                                                                  [9] increased expression e1370 mutants unc-38, which <br />                                                                                  encodes subunit nicotinic acetylcholine receptor. <br />                                                                                      Ten genes down-regulated e1370 (die) compared the <br />                                                                                  alleles (Table 2, Figure 2B). These genes include akt-2, a <br />                                                                                  homolog serine/threonine protein kinase Akt/PKB. akt-2 <br />                                                                                  paralog, akt-1, functions downstream daf-2 the <br />                                                                                  daf-2 - daf-16 pathway. Knockdown akt-1 akt-2 produce a <br />                                                                                  partially redundant dauer constitutive phenotype [16], akt- <br />                                                                                  1 gain-of-function mutant partially suppresses hypoxia <br />                                                                                  resistance daf-2(1370) [1]. The role AKT-1/2 downstream <br />                                                                                  DAF-2 thought phosphorylation DAF-16, <br />                                                                                  inhibiting nuclear localization transcription factor <br />                                                                                  activity [17,18]. These new data suggest positive feedback <br />                                                                                  loop exist enhances reduction-of-function phenotype <br />                                                                                  e1370 compared m596 e1368 reducing levels of <br />                                                                                  akt-2 mRNA. Besides akt-2, die genes predicted to <br />                                                                                  kinase phosphatase activity, substrates are <br />                                                                                  unknown. Other particularly interesting die genes include acdh-2, <br />                                                                                  noted expression differences alleles that <br />                                                                                  follow hypoxic sensitivity allelic series, F54G2.1 - isoform of <br />                                                                                  presynaptic vesicle fusion protein UNC-13, F36F2.3 - E3 <br />                                                                                  ubiquitin ligase, T04A11.6 - ATP-dependent DNA <br />                                                                                  helicase. We previously whole-genome RNAi screen <br />                                                                                  genes promote hypoxic death knockdown several <br />                                                                                  genes, predicted control chromatin structure, produce <br />                                                                                  significant hypoxia resistance [19]. The current data the <br />                                                                                  daf-2 pathway acts regulation DNA/chromatin <br />                                                                                  modifying enzymes control hypoxic sensitivity. However, is <br /> Figure 2. Expression 182 genes differentially regulated                   unclear protects organism hypoxic injury. <br /> across daf-2 alleles. (A) Heat map expression 182 genes, either <br /> up- down-regulated e1370 m596 and/or e1368, p,0.01, <br /> One-Way ANOVA. (B) Genes expression downregulated in <br />                                                                                  Increased Hypoxic Survival die Gene Knockdown <br /> e1370 (die) relative m596 and/or e1368, p,0.01. (C) Relative number              Genes expression increased e1370 relative the <br /> of daf-2 regulated genes fall various functional categories.       alleles candidates encode hypoxia protective <br /> doi:10.1371/journal.pone.0007937.g002                                            factors. We chose 10 genes highest normalized relative <br />  <br />  <br />        PLoS ONE | www.plosone.org                                            3                        November 2009 | Volume 4 | Issue 11 | e7937 <br />                                                                                                                                          daf-2 Hypoxic Death Targets <br />  <br />  <br />  <br />  Table 2. Genes transcriptionally down-regulated e1370 vs m596 e1368 &#226;&#8364;&#8220; die genes. <br />  <br />  <br />  Sequence Gene                                                                                                                          e1370vs     e1370vs   e1368vs <br />  Name     Name         Protein/Homology                                 Biological Process         Expression                 p-value   m596        e1368     m596 <br />  <br />  F36F2.3               Predicted E3 ubiquitin ligase                    Protein Degradation/       ND                         0.0019    26.0        22.6      22.3 <br />                                                                         Modification/Turnover <br />  ZK262.8               synthetic lethal Argonaute mutant           Unknown                    ND                         0.0021    23.6        21.9      21.9 <br />  T22C1.8               Protein tyrosine phosphatase                     Signaling                  ND                         0.0027    25.2        22.5      22.1 <br />  F54D11.3              C. elegans-specific                              Unknown                    ND                         0.0048    27.1        22.6      22.7 <br />  F58D2.1               ZYG-11-like serine/threonine protein kinases     Unknown                    ND                         0.0059    22.2        21.5      21.5 <br />  C17C3.12    acdh-2    Mitochondrial short-chain acyl-CoA               Metabolism                 ND                         0.0060    22.4        23.6      21.5 <br />                        dehydrogenase <br />  F28H6.1     akt-2     Homolog serine/threonine kinase           Signaling                  Intestine, Neuronal,       0.0061    22.7        21.3      22.1 <br />                        Akt/PKB                                                                     Hypodermal, Pharyngeal <br />  F54G2.1               Isoform 1 protein UNC-13 homolog D            Unknown                    Intestine, Neuronal        0.0072    22.5        21.6      21.6 <br />  T20D4.18    srab-21   Sra family integral membrane protein             Unknown                    Hypodermal, Neuronal       0.0077    22.1        21.4      21.5 <br />  T04A11.6    him-6     ATP-dependent DNA helicase                       Transcription/DNA/RNA      Primarily Germline         0.0086    22.9        21.5      21.9 <br />                                                                         Processing <br />  <br />  Annotations WormBase (www.wormbase.org); p-value one-way ANOVA e1370 vs m596. <br />  doi:10.1371/journal.pone.0007937.t002 <br />  <br />  <br /> fold change e1370 compared m596 test hypoxia                               e1368 background compared vector control <br /> protection (Table S1). We used feeding RNAi reduce                             (Figure 3B). These results reduction function any <br /> expression e1370 background prediction                         genes capable converting weakly hypoxia <br /> would suppress partially hypoxia resistant phenotype.                      resistant e1368 animals strongly resistant ones. Also, the <br /> However, did observe significant change hypoxia                             sensitized background e1368 revealed F58D2.1 does in <br /> resistance treatment RNAis (data shown).                       fact encode hypoxic sensitivity promoting activity not <br /> This negative result explained number ways.                            apparent wild type background. <br /> Obviously, inadequate knockdown gene expression way.                             Given expression die gene transcripts regulated daf- <br /> Given 172 genes up-regulated, likely                      2, hypothesized hypoxia resistance phenotype die <br /> possibility knockdown single gene sufficient                        RNAi depend DAF-16, primary target DAF-2. <br /> produce measurable phenotype. Alternatively, up-regulated                         Indeed, hypoxia resistance phenotypes die genes were <br /> genes tested mediate responses maladaptive                       suppressed null mutation daf-16 (Figure 3C). However, the <br /> or unrelated survival hypoxia; thus, knockdown                        suppression hypoxia resistance incomplete the <br /> suppress e1370&#226;&#8364;&#8482;s hypoxia resistance.                                                    RNAi knockdowns. Thus, genes regulate hypoxic <br />    The die genes candidates promote daf-2-regulated hypoxic                      sensitivity entirely DAF-16 dependent pathway. <br /> sensitivity; thus, die gene reduction-of-function produce <br /> hypoxia resistance. To test hypothesis, used RNAi to <br />                                                                                         die Genes Are Not Equally Required Thermotolerance <br /> knockdown die genes; tested, two, <br /> C17C3.12 F28H6.1, library, other,                               Lifespan Extension <br /> T22C1.8, did grow streaked, fourth, F36F2.3,                              The experimental design microarrays devised enrich <br /> produced sick animals absence hypoxia. Five RNAi                       genes selectively regulate hypoxic sensitivity. To test for <br /> treatments conferred significant hypoxia resistance                        selective regulation, measured thermotolerance lifespan <br /> wild type animals (Figure 3A). We tested effect of <br /> knockdown die genes e1368 background. <br /> Knockdown genes produced strong hypoxia resistance                            Table 4. Genes overlap datasets. <br />  <br />  <br />  Table 3. Genes regulated according allelic series.                               Sequence       Overlapping Expression e1370     Expression e1370 <br />                                                                                          Name           Study       relative m596, e1368 relative wild type* <br />  <br />                                                                                          C09G4.5        Murphy           Increased                Decreased <br />  Sequence Name         Common Name           e1370*     m596*         e1368* <br />                                                                                          C47E8.5        Dong             Increased                Decreased <br />  C14A4.8                                     2.19       0.90          0.85 <br />                                                                                          F15E11.1       Dong             Increased                Increased <br />  Y37H2A.6              fbxa-211              1.20       0.92          0.90 <br />                                                                                          F17C11.9       Dong             Increased                Decreased <br />  F45D11.4                                    3.64       0.62          0.34 <br />                                                                                          F21F3.5        Murphy           Increased                Increased <br />  F45D11.14                                   3.70       0.54          0.28 <br />                                                                                          F45D11.14      Dong             Increased                Increased <br />  C17C3.12              acdh-2                0.79       1.88          2.86 <br />                                                                                          Y54G11A.13 Dong                 Increased                Increased <br />  T26H5.1                                     0.83       1.17          1.20 <br />                                                                                          Murphy et al. Nature 424:277-83, 2003. Dong et al. Science 317:660-3, 2007. <br />  *                                                                                       * <br />   Mean normalized expression, p,0.01.                                                  - data overlapping study. <br />  doi:10.1371/journal.pone.0007937.t003                                                   doi:10.1371/journal.pone.0007937.t004 <br />  <br />  <br />  <br />       PLoS ONE | www.plosone.org                                                    4                            November 2009 | Volume 4 | Issue 11 | e7937 <br />                                                                                                                             daf-2 Hypoxic Death Targets <br />  <br />  <br />                                                                                  (Figure 5 Table S2). Again, results consistent the <br />                                                                                  poor correlation lifespan hypoxic sensitivity <br />                                                                                  phenotypes daf-2 alleles [1]. <br />                                                                                     One possibility strong correlation thermotoler- <br />                                                                                  ance hypoxia resistance 26.5uC temperature which <br />                                                                                  hypoxic incubations performed. As models hypoxic <br />                                                                                  injury, sensitivity C. elegans hypoxia temperature <br />                                                                                  dependent [1]; higher temperature severe the <br />                                                                                  hypoxic injury. However, number observations suggest that <br />                                                                                  hypoxic sensitivity differences daf-2 alleles simply <br />                                                                                  manifestation varied thermotolerance. First, daf-2 alleles, <br />                                                                                  e979 e1369 moderate hypoxia resistance similar to <br />                                                                                  m596, thermotolerances high e1370 [13]. <br />                                                                                  Second, hypoxic incubations performed 20uC, e1370 <br />                                                                                  m579, highly hypoxia resistant assay <br />                                                                                  conditions [1], highly hypoxia resistant, m596 is <br />                                                                                  [20]. Finally, asked high temperature incubation <br />                                                                                  buffer room air environment produced significant wild <br />                                                                                  type animal death account daf-2 allelic <br />                                                                                  differences hypoxic sensitivity higher temperature. However, <br />                                                                                  observed significant death wild type animals a <br />                                                                                  20 hour normoxic incubation 28uC (Figure S1). Even 71 hour <br />                                                                                  incubation killed 5% animals. <br />  <br />                                                                                  Discussion <br />                                                                                      Utilizing daf-2 alleles distinct orders phenotypic severity <br />                                                                                  hypoxic sensitivity versus phenotypes basis for <br />                                                                                  DNA microarray experiments, identified 182 genes were <br />                                                                                  differentially expressed animals varying levels hypoxia <br />                                                                                  resistance. The expression of genes increased in <br />                                                                                  hypoxia resistant daf-2(e1370). These genes candidates for <br />                                                                                  antagonizing hypoxic death. However, unable detect <br />                                                                                  hypoxic hypersensitivity RNAi knockdown subset these <br />                                                                                  genes. In general, hypoxic hypersensitivity is <br />                                                                                  difficult produce model; reflect genetic <br />                                                                                  redundancy reminiscent relatively uncommon <br />                                                                                  phenotype shortened lifespan compared prolonged <br /> Figure 3. Survival hypoxia RNAi knockdown die                         lifespan. However, genes decreased daf- <br /> genes. Genetic background strains indicated closed (wild type -         2(e1370), able test that <br /> N2) open bars (mutant &#226;&#8364;&#8220; daf-2(e1368) daf-16(mu86)). Note            knockdown of produced hypoxia resistance. These <br /> that survival data 22 hour hypoxic incubation upper <br /> and lower panels 24 hour hypoxic incubation the <br />                                                                                  genes fact control hypoxic <br /> middle panel. (A) Survival recovery 22 hour hypoxic                 sensitivity relatively specific manner by <br /> incubation N2 wild type animals treated bacteria transfected            having detectable influence lifespan. In regard, <br /> with L4440 vector RNAi plasmids die genes.                 ZK262.8 intriguing unlike die <br /> * p,0.01 compared L4440 (two-tailed t-test, 6 trials/RNAi .30            genes hypoxia resistance knockdown phenotype, ZK262.8 <br /> worms/trial). (B) Effect RNAi treatments survival 24 hour          knockdown does confer thermotolerance. ZK262.8 poorly <br /> hypoxic incubation wild type daf-2(e1368) backgrounds. All <br /> RNAi&#226;&#8364;&#8482;s resistance e1368 background compared L4440 &#226;&#8364;&#8220; p,0.01              characterized gene clear homology genes other <br /> (two-tailed t-test, 3 trials/RNAi .30 worms/trial) (C) Effect RNAi       organisms. It implicated microRNA processing a <br /> treatments survival 22 hour hypoxic incubation wild type           synthetic <span id='am-238' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-239' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-242' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-267' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-270' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-292' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-295' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-305' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-307' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-309' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-315' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span>lethal phenotype</span> mutation Argonaut <br /> and daf-16(mu86 null) backgrounds. Hypoxia resistance significantly              homolog gene, alg-2 [21]. ALG-2 expressed most, all, <br /> suppressed mu86 background RNAi&#226;&#8364;&#8482;s &#226;&#8364;&#8220; p,0.01 (paired t-test, 6          cells C. elegans embryonic adult stages is <br /> trials/RNAi .30 worms/trial). <br /> doi:10.1371/journal.pone.0007937.g003 <br />                                                                                  hypothesized facilitate loading pre-miRNAs Dicer <br />                                                                                  complex [21]. However, unclear role ZK262.8 <br /> of die genes knockdown produced hypoxia resistance                gene product miRNA processing. <br /> in wild type background. Knockdown of die genes                   Another hypoxic death promoting die gene identified srab-21, <br /> produced significant thermotolerance (Figure 4). Only ZK262.8                    encodes serpentine receptor likely function in <br /> RNAi did protect thermal stress (Figure 4B). These results              chemosensation [22]. In genome RNAi screen genes <br /> are consistent strong correlation thermotolerance               promoting hypoxic death, identified serpentine receptor <br /> and hypoxia resistance observed various daf-2 alleles [1].                 genes knockdown produced hypoxia resistance [19]. These <br /> However, mechanisms thermal injury                genes represented 5% genes identified screen. <br /> hypoxic injury C. elegans identical. Despite considerable           Given genes thought encode chemosensory <br /> increase mean maximum lifespan daf-2 RNAi,                   receptors, normal chemosensory function appears promote <br /> of die gene RNAi&#226;&#8364;&#8482;s resulted significant alteration lifespan          hypoxic sensitivity C. elegans. Likewise aging, mutations in <br />  <br />  <br />        PLoS ONE | www.plosone.org                                            5                        November 2009 | Volume 4 | Issue 11 | e7937 <br />                                                                                                                    daf-2 Hypoxic Death Targets <br />  <br />  <br />  <br />  <br /> Figure 4. Increased thermotolerance die gene RNAi knockdown. Survival recovery incubations 37u various times for <br /> worms raised L4440 vector denoted RNAi. RNAi&#226;&#8364;&#8482;s panel A C-F produce significantly increased thermotolerance the <br /> RNAi panel B produces significantly decreased thermotolerance &#226;&#8364;&#8220; p,0.01, F-test nonlinear regression simultaneous curve-fitting. A <br /> minimum 353 animals/curve 18 animals/data point scored. <br /> doi:10.1371/journal.pone.0007937.g004 <br />  <br /> genes required chemosensation prolong lifespan [23]. The             expression somatic cells possible. Germ cells him-6 mutant <br /> prolonged lifespan mutants suppressed               animals increased sensitivity ionizing radiation, but <br /> mutations daf-16, model proposed activation         surprisingly decreased germ cell apoptosis genotoxic <br /> of chemosensory neurons increases insulin-like hormone signaling         stress [25]. Thus, possible hypoxia resistance is <br /> and limits lifespan. However, fact srab-21              secondary defects apoptosis remains determined. <br /> knockdown does prolong lifespan suggests simply altering           We identified significant number genes that <br /> ligand activation DAF-2 unlikely mechanism.                     regulate hypoxic sensitivity [1,19,26,27,28,29]. In case, there <br />    The identification ATP-dependent DNA helicase gene him-        complete understanding mechanism(s) gene <br /> 6 die gene promotes hypoxic death consistent           protects organism cells hypoxic injury, in <br /> previous results. In genome RNAi screen, identified       cases, genes identified here, mechanistic <br /> large number DNA processing genes knockdown                     information all. Future studies necessarily focus the <br /> phenotype hypoxia resistance [19]. Indeed, nucleic acid binding       promising hypoxic sensitivity genes. Such genes have <br /> and processing enzyme genes represented largest functional           clear mammalian orthologs, preferably considerable <br /> category helicases them. him-6 expressed           understanding cellular function defined. Addi- <br /> primarily germline assessed microarray experiments          tionally, promising genes able modulated <br /> comparing animals germline [24]. Thus,           control hypoxic injury producing effects. <br />  <br />  <br />       PLoS ONE | www.plosone.org                                     6                        November 2009 | Volume 4 | Issue 11 | e7937 <br />                                                                                                                             daf-2 Hypoxic Death Targets <br />  <br />  <br /> Thus, studies described important                  worms placed NGM plates spotted OP50 bacteria <br /> order identify specific pathways target hypoxic protection.          using glass Pasteur pipets recovered 20u 16220 hours <br />                                                                                prior scoring. Animals scored dead pharyngeal <br /> Materials Methods                                                          pumping spontaneous evoked movement (touching a <br />                                                                                platinum wire) observed. A minimum triplicate plates for <br /> Nematode Maintenance                                                           trial trials/RNAi scored. <br />   For strain maintenance worms cultured 20u on <br /> nematode growth medium (NGM) agar plates seeded OP50                      Feeding RNAi <br /> bacteria [30]. The following strains utilized study: N2              One-generation feeding RNAi performed described <br /> (wild-type), CB1370 daf-2(e1370) III, DR1565 daf-2(m596) III,                  previously [19,31]. Briefly, gravid adult worms left on <br /> DR1572 daf-2(e1368) III, CF1038 daf-16(mu86) I. Strains               RNAi plates 2 day old bacterial lawns 3 hours obtain <br /> obtained Caenorhabditis Genetics Center funded                 30250 eggs plate. RNAi plates composed NGM agar <br /> NIH NCRR.                                                                      supplemented 50 mg/ml carbenicillin 1 mM IPTG and <br />                                                                                seeded appropriate RNAi bacterial strain cultured in <br /> Hypoxia Assay                                                                  2xYT 50 mg/ml carbenicillin, 10 mg/ml tetracycline and <br />   Worms subjected hypoxia described previously [1].                 0.8 mM IPTG. Animals grown 20uC young adulthood, <br /> Briefly, plate washed 1.5 mL tube 1 mL               day past L4 stage phenotyped. <br /> M9 buffer (22 mM KH2PO4, 22 mM Na2HPO4, 85 mM NaCl, <br /> 1 mM MgSO4) [30]. Worms allowed settle gravity,                 Lifespan Assays <br /> 800 mL M9 removed. The tubes placed                       Lifespan assays performed OP50 bacteria <br /> anaerobic chamber (Forma Scientific) 26.5u 20, 22,                   (Figure 1C) RNAi bacteria containing L4440 empty <br /> 24 hours, depending assay. Following hypoxic insult,                vector control plasmid particular RNAi plasmid (Figure 5). <br />  <br />  <br />  <br />  <br /> Figure 5. No effects lifespan die gene RNAi knockdown. Lifespan survival curves animals grown L4440 vector bacteria <br /> versus RNAi bacteria. (A) The effect daf-2 RNAi lifespan shown positive control significantly different L4440 lifespan &#226;&#8364;&#8220; p%0.01 <br /> Mantel-Cox Log-rank test. (B-F) None RNAis significantly lengthen lifespan p,0.01 level. Lifespan assays performed 120 <br /> animals/RNAi grown 20uC. <br /> doi:10.1371/journal.pone.0007937.g005 <br />  <br />  <br />        PLoS ONE | www.plosone.org                                          7                         November 2009 | Volume 4 | Issue 11 | e7937 <br />                                                                                                                                            daf-2 Hypoxic Death Targets <br />  <br />  <br /> Gravid adult worms allowed lay eggs hours                          ter, Washington University St. Louis). Slides scanned a <br /> OP50 RNAi bacteria picked off. The animals cultured                           ScanArray 3000 (Perkin Elmer, Waltham, MA) 10 micrometer <br /> at 20uC reaching L4 stage placed                             resolution. Three biological replicates (independent RNA isola- <br /> density 30 worms/plate. For daf-2 allele lifespan data, L4                     tions), technical replicate (dye swap) performed <br /> animals transferred fresh NGM plates seeded OP50.                            condition. Raw microarray fluorescent intensity data were <br /> For die gene RNAi lifespan experiments, L4 animals                               processed analyzed GeneSpring GX 7.3.1 software <br /> transferred new RNAi plates. Worms transferred daily                              (Agilent Technologies, Santa Clara, CA). Each time point was <br /> while egg-laying transferred days                         subjected Lowess normalization (per spot array) <br /> plates freshly seeded OP50 RNAi bacteria. Worms                              separately. A flag filter applied include genes that <br /> scored daily survival censored bagging, exploding,                        hybridized trials narrowed gene list to <br /> crawling plate.                                                                   14,317. One-way ANOVA (p,0.01) performed genes <br />                                                                                           statistically different e1370 two <br /> Thermotolerance Assay                                                                     daf-2 alleles, m596 e1368. This gene list included 182 genes. <br />    Thermotolerance assays performed 37u water bath.                             Raw data deposited NCBI Gene Expression <br /> Young adult worms, day post L4 stage, washed tubes                          Omnibus database (http://www.ncbi.nlm.nih.gov/geo/), acces- <br /> using M9 buffer allowed settle gravity. Excess M9                           sion number GSE18601. <br /> removed 100 mL. M9 buffer preheated 37u added each <br /> tube final volume 500 mL. The tubes incubated                         Supporting Information <br /> water bath according time course. Worms removed <br /> from tubes appropriate time using glass Pasteur pipette                      Table S1 List genes upregulated daf-2(e1370). <br /> and allowed recover 20uC 16220 hours NGM plates                              Found at: doi:10.1371/journal.pone.0007937.s001 (0.06 MB <br /> spotted OP50 bacteria scored survival.                                  XLS) <br />                                                                                           Table S2 Lifespan analysis die gene RNAi knockdown. <br /> Dauer Arrest Assay                                                                        Found at: doi:10.1371/journal.pone.0007937.s002 (0.03 MB <br />    Ten gravid adult animals allowed lay eggs hour                         XLS) <br /> and picked off. Plates containing 30250 eggs 20uC, <br /> 23uC 25uC. Animals scored dauer arrest                               Figure S1 Comparison normoxic hypoxic lethality. <br /> non-dauer animals reached adulthood, 72 h 96 h later. The                              Young adult N2 animals incubated 28 degrees M9 <br /> fraction non-adult animals partial dauers arrested                        buffer incubator room air atmosphere (normoxia) <br /> non-dauer larvae formally scored; however, large                              incubator ,0.3% oxygen (hypoxia). The % dead <br /> majority non-adult animals true dauers                          animals scored recovery various incubation times. <br /> strains previously reported [13].                                                      Each point represents mean +/2 sd minimum trials <br />                                                                                           16 animals/trial. <br /> cDNA Microarrays                                                                          Found at: doi:10.1371/journal.pone.0007937.s003 (2.48 MB TIF) <br />    Animals synchronous egg lay cultured 20uC on <br /> NGM plates seeded OP50, total RNA isolated                              Acknowledgments <br /> L4 staged daf-2(e1370), daf-2(m596) daf-2(e1368) animals using <br />                                                                                           We thank Elaine Mardis Genome Center Washington <br /> Trizol reagent (Invitrogen). Dauer animals separated from <br />                                                                                           University St. Louis help design performance of <br /> L4 larvae differential pelleting M9. RNA quality                                DNA microarray experiments. <br /> measured using Nanodrop-1000 spectrophotometer (Thermo- <br /> Scientific). High quality RNA samples (OD 260/280.1.9) were <br /> used inputs microarrays. Eight micrograms RNA was <br />                                                                                           Author Contributions <br /> subjected reverse transcription subsequent hybridization                           Conceived designed experiments: MEM CMC. Performed the <br /> using 3DNA Array 350 Kit (Genisphere, Inc, Hatfield, PA). Cy3-                            experiments: MEM BAS. Analyzed data: MEM BAS CMC. <br /> and Cy5-labeled samples hybridized C. elegans                               Contributed reagents/materials/analysis tools: CMC. Wrote paper: <br />                                                                                           MEM BAS CMC. <br /> genome oligonucleotide microarrays (Genome Sequencing Cen- <br />  <br /> References <br />  1. Scott BA, Avidan MS, Crowder CM (2002) Regulation hypoxic death C.               8. Lamitina ST, Strange K (2005) Transcriptional targets DAF-16 insulin <br />     elegans insulin/IGF receptor homolog DAF-2. Science 296: 2388&#226;&#8364;&#8220;2391.                signaling pathway protect C. elegans extreme hypertonic stress. <br />  2. Ogg S, Paradis S, Gottlieb S, Patterson GI, Lee L, et al. (1997) The Fork head            Am J Physiol Cell Physiol 288: C467&#226;&#8364;&#8220;474. <br />     transcription factor DAF-16 transduces insulin-like metabolic longevity            9. Murphy CT, McCarroll SA, Bargmann CI, Fraser A, Kamath RS, et al. (2003) <br />     signals C. elegans. Nature 389: 994&#226;&#8364;&#8220;999.                                               Genes act downstream DAF-16 influence lifespan of <br />  3. Lin K, Dorman JB, Rodan A, Kenyon C (1997) daf-16: An HNF-3/forkhead                      Caenorhabditis elegans. Nature 424: 277&#226;&#8364;&#8220;283. <br />     family member function double life-span Caenorhabditis             10. McElwee J, Bubb K, Thomas JH (2003) Transcriptional outputs the <br />     elegans. Science 278: 1319&#226;&#8364;&#8220;1322.                                                          Caenorhabditis elegans forkhead protein DAF-16. Aging Cell 2: 111&#226;&#8364;&#8220;121. <br />  4. Riddle DL, Albert PS (1997) Genetic environmental regulation dauer             11. Lee SS, Kennedy S, Tolonen AC, Ruvkun G (2003) DAF-16 target genes that <br />     larva development. In: Riddle DL, Blumenthal T, Meyer BJ, Priess JR, eds                  control C. elegans life-span metabolism. Science 300: 644&#226;&#8364;&#8220;647. <br />     (1997) C elegans II. Cold Spring Harbor: Cold Spring Harbor Laboratory Press.         12. Yu H, Larsen PL (2001) DAF-16-dependent independent expression targets <br />     pp 739&#226;&#8364;&#8220;768.                                                                               DAF-2 insulin receptor-like pathway Caenorhabditis elegans include <br />  5. Samuelson AV, Carr CE, Ruvkun G (2007) Gene activities mediate                       FKBPs. J Mol Biol 314: 1017&#226;&#8364;&#8220;1028. <br />     increased life span C. elegans insulin-like signaling mutants. Genes Dev 21:       13. Gems D, Sutton AJ, Sundermeyer ML, Albert PS, King KV, et al. (1998) Two <br />     2976&#226;&#8364;&#8220;2994.                                                                                pleiotropic classes daf-2 mutation affect larval arrest, adult behavior, <br />  6. Pinkston-Gosse J, Kenyon C (2007) DAF-16/FOXO targets genes regulate                 reproduction longevity Caenorhabditis elegans. Genetics 150: 129&#226;&#8364;&#8220; <br />     tumor growth Caenorhabditis elegans. Nat Genet advanced online publication.            155. <br />  7. Oh SW, Mukhopadhyay A, Dixit BL, Raha T, Green MR, et al. (2006)                      14. Dong M-Q, Venable JD, Au N, Xu T, Park SK, et al. (2007) Quantitative Mass <br />     Identification direct DAF-16 targets controlling longevity, metabolism             Spectrometry Identifies Insulin Signaling Targets C. elegans. Science 317: <br />     diapause chromatin immunoprecipitation. N <br /> </body></html>