<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The Mych Gene Is Required Neural Crest <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span>Survival</span> <br /> during Zebrafish Development <br /> Sung-Kook Hong1*, Michael Tsang1,2, Igor B. Dawid1 <br /> 1 Laboratory Molecular Genetics, National Institute Child Health Human Development, National Institutes Health, Bethesda, Maryland, United States of <br /> America, 2 Department Microbiology Molecular Genetics, University Pittsburgh, School Medicine, Pittsburgh, Pennsylvania, United States America <br />  <br />  <br />  <br />      <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />      Background: Among Myc family genes, c-Myc known role neural crest specification Xenopus in <br />      craniofacial development mouse. There information function Myc genes neural crest <br />      development, developmental role zebrafish Myc genes. <br />  <br />      Principal Findings: We isolated zebrafish mych (myc homologue) gene. Knockdown mych leads severe defects in <br />      craniofacial development certain tissues including eye. These phenotypes appear caused cell <br />      death neural crest eye field anterior brain. <br />  <br />      Significance: Mych novel factor required neural crest <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span>cell survival</span> zebrafish. <br />  <br />   Citation: Hong S-K, Tsang M, Dawid IB (2008) The Mych Gene Is Required Neural Crest Survival Zebrafish Development. PLoS ONE 3(4): e2029. <br />   doi:10.1371/journal.pone.0002029 <br />   Editor: Mark R. Cookson, National Institutes Health, United States America <br />   Received November 23, 2007; Accepted March 14, 2008; Published April 30, 2008 <br />   This open-access article distributed terms Creative Commons Public Domain declaration stipulates that, placed public <br />   domain, work freely reproduced, distributed, transmitted, modified, built upon, used lawful purpose. <br />   Funding: Intramural Research Program National Institute Child Health Human Development, National Institutes Health <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: hongsk@mail.nih.gov <br />  <br />  <br />  <br />  <br /> Introduction                                                                         depletion. Mych knockdown embryos experience excess apoptotic <br />                                                                                      cell death neural crest population anterior brain. <br />    Myc genes function cellular proliferation regulating cell                   Subsequently, pharyngeal arches failed develop properly and <br /> cycle progression, apoptosis, cell transformation. Myc factors                   eyes smaller normal lacked laminar organiza- <br /> are thought regulators gene transcription activate                  tion. Thus, mych function required neural crest eye cell <br /> repress multiple target genes [1,2,3,4,5]. Myc proteins                     survival development. <br /> recognized functional domains, N-terminal domain (NTD) <br /> and C-terminal domain (CTD). The CTD contains basic <br /> helix-loop-helix leucine zipper (bHLH-LZ) motif necessary                    Results <br /> for target DNA binding regulation gene expression. The <br />                                                                                      Isolation full-<span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> mych cDNA sequence analysis <br /> Myc NTD contributes control transcriptional activation <br /> or repression downstream target genes [6,7,8,9]. While role                   Mych protein <br /> of Myc family cell proliferation cancer received                         A partial mych cDNA identified previously in situ <br /> wide attention, comparatively known develop-                       hybridization-based gene expression screen [16]. We cloned full- <br /> mental functions.                                                                    length mych containing 1,975 <span id='am-239' about='xsp:base' typeof='owl:Thing'>base</span> pairs (bp) using 59- RACE <br />    Neural crest cells originate edge neural plate               method. The cDNA predicts 360-amino acid protein that <br /> the dorsal aspect neural tube, migrate locations                  contains bHLH-LZ domain high sequence identity the <br /> where differentiate great variety cell types [10]. In                 CTD vertebrate N-myc C-myc proteins (57%&#226;&#8364;&#8220;72%) <br /> Xenopus, c-Myc expressed early premigratory neural crest cells,                (Fig. 1D). However 39&#226;&#8364;&#8220;57% identity NTD of <br /> and inhibition expression results loss neural crest                   Mych compared Myc family proteins, amino <br /> precursor cells derivatives [11]. The function c-Myc                 acid sequences entire proteins 36&#226;&#8364;&#8220;38% identical. A <br /> the neural crest involves Id3, gene likewise expressed neural                   phylogenic tree Myc family based CTD shows that <br /> crest precursors [12]. In mouse, conditional inactivation c-                  Mych located c-Myc N-Myc clusters, closer <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> <br /> myc using Wnt1-Cre targeted inactivation, led skull, middle                   N-Myc (Fig. 1C). The mych gene located chromosome 6 <br /> ear, coat pigmentation defects [13]. These findings implicate                    according Sanger genome center, version Zv7 (http://www. <br /> c-Myc regulation neural crest formation mouse. In                   ensembl.org/Danio_rerio/index.html; Zv7 Scaffold638, contig <br /> zebrafish, Myc genes cloned expression                       BX649289.11), confirmed radiation hybrid mapping using <br /> been reported [14,15]. Here present isolation novel                          LN54 panel [17] (Fig. 1A). To address intracellular <br /> member family, mych (for nomenclature Zebrafish                           localization Mych protein, Flag-tagged mych construct was <br /> Information Network: http://zfin.org), report expression                     transfected NIH3T3 cells; protein localized nucleus, <br /> <span id='am-240' about='xsp:pattern' typeof='owl:Thing'>pattern</span> functional analysis using morpholino-based                             generally true Myc family proteins (Fig. 1B). <br />  <br />  <br />        PLoS ONE | www.plosone.org                                                1                                  April 2008 | Volume 3 | Issue 4 | e2029 <br />                                                                                                                         Mych Neural Crest Formation <br />  <br />  <br />  <br />  <br /> Figure 1. Characterization mych comparison verte- <br /> brates Myc proteins. A. Radiation hybrid mapping determined the <br /> location mych chromosome 6. B. Nuclear localization Flag- <br /> Mych protein (green) NIH3T3 cells; nuclei stained DAPI <br /> (blue). C, D. Evolutionary comparisons. H, human; M, mouse; X, <br /> Xenopus; Z, zebrafish. C. Phylogenic tree Myc family proteins. D. <br /> Amino acid sequence alignments C-terminal regions containing <br /> highly conserved basic helix-loop-helix domains. Amino acid identities <br /> are shown color, basic, helix-loop-helix, leucine-zipper <br /> domains marked green, red, blue, respectively. The GenBank <br /> Accession Number mych EU232118. <br /> doi:10.1371/journal.pone.0002029.g001 <br />  <br />  <br /> Dynamic specific expression mych early                           Figure 2. Detection mych transcripts situ hybridization. <br />                                                                                A. Dorsal view 8-cell stage embryo. B. Double staining mych (blue) <br /> development                                                                    gsc (red) shield stage; arrowhead points shield. C-C&#226;&#8364;&#8482;. <br />    The expression mych developing zebrafish embryos                  Lateral (C) anterior dorsal (C&#226;&#8364;&#8482;) views 80% epiboly stage. Red <br /> examined mount situ hybridization. Maternal                        asterisk points anterior-dorsal area shown C&#226;&#8364;&#8482;. D&#226;&#8364;&#8220;I. 3-somite <br /> transcripts broadly expressed appear persist               stage embryos. D. Dynamic expression mych anterior brain <br /> through blastula stages (Fig. 2A). After shield formation, mych                region. E-E&#226;&#8364;&#8482;. Double staining mab21l2 (red) marker eye and <br />                                                                                midbrain. F-F&#226;&#8364;&#8482;. Double staining dmbx1a (red) marker eye <br /> expression excluded dorsal marginal area                    field midbrain; red arrowhead indicates eye field. G,G&#226;&#8364;&#8482;. Co- <br /> retained future prechordal plate region (Fig. 2B). During               expression ctsl1b (red), marking prechordal plate. H-H&#226;&#8364;&#8482;. Dorsal <br /> gastrulation, mych transcripts distinct dorsal                view HuC-positive neuronal cells (red) mych staining (blue) in <br /> ventral expression domains embryo (Fig. 2C,C9). To                      wild type (H) mibta52b mutant (H&#226;&#8364;&#8482;). I. Rhombomeres 3 5 are <br /> interpret early brain expression pattern mych, carried           marked egr2b (red), mych stains anterior of <br />                                                                                rhombomere 4. J&#226;&#8364;&#8220;K. Dorsal views 10-somite stage embryo showing <br /> double staining eye, tectum, prechordal plate markers,                mych expression eye midbrain (J) trunk somites (K). L. <br /> using mab21l2, dmbx1a/mbx, ctsl1b/hgg1 [18,19,20]. At 3-               Lateral view mych expression brain pharyngeal at <br /> somite stage, mych expressed eye field (Fig. 2E,E9,F,F9),            24hpf; mandibular (m), hyoid (h), branchial (b) arches indicated. <br /> midbrain (Fig. 2F,F9), prechordal plate (Fig. 2G,G9). The                  M. Expression mych eye 36hpf. Arrowhead points to <br /> bilateral expression mych hindbrain analyzed                  presumptive photoreceptor cell layer. N. mych expression heart <br /> double staining egr2b/krox20, marker rhombomeres 3                  intestine (int; magnified O) 72hpf. P&#226;&#8364;&#8220;Q. Co-expression with <br />                                                                                foxd3 (red) 4-somite stage. The yellow open square area P is <br /> and 5. Mych expressed rhombomere 4 (Fig. 2I),                   shown section Q. R&#226;&#8364;&#8220;S. Pharyngeal arch marker dlx2a (red S) is <br /> expression strongly increased neurogenic mutant mibta52b             co-expressed mych 32hpf. b, branchial arch; e, eye; h, hyoid arch; <br /> [21], suggesting mych transcripts present neuronal cells           ht, heart; int, intestine; le, lens; m, mandibular arch; mi, midbrain; ov, <br /> in rhombomere 4 (Fig. 2H). Mych expression presumptive                  otic vesicle; pp, prechordal plate; r, rhombomere; s, somite. <br /> hindbrain begins 90% epiboly (data shown),                 doi:10.1371/journal.pone.0002029.g002 <br /> gene early marker region. During mid- <br /> segmentation stages mych mainly expressed eye, midbrain,             Mych knockdown phenotypes <br /> and somites (Fig. 2J,K). At 24hpf, mandibular, hyoid,                         To study function mych embryonic development, we <br /> branchial arch expression increases (Fig. 2L), eye expression              used morpholino oligonucleotides targeted 59- <br /> is detected primarily photoreceptor layer 36hpf (Fig. 2M).           untranslated region mRNA (UTR MO) the <br /> At 72hpf, brain, heart gut strong mych expression                     intron 1 splice donor site (SP MO). To determine MOs specificity, <br /> (Fig. 2N,O). To test mych expressed neural crest             UTR MO coinjected mRNA fusion protein of <br /> development carried two-color situ hybridization                Mych GFP, showing loss GFP expression (Figure S1A,B). <br /> early neural crest marker foxd3 [22] pharyngeal arch                   The SP MO specificity examined RT-PCR, using sets of <br /> marker dlx2a [23]. At 4-somite stage, foxd3 mych                   primers flanking intron; results effective suppression <br /> colocalized region premigratory neural crest                     splicing (Figure S1C,D,E). Mych UTR MO injected embryos <br /> (Fig. 2P,Q), 32 hpf colocalization dlx2a seen               showed widening future trunk region 3-somite stage <br /> the pharyngeal arches (Fig. 2R,S). These observations indicate               (75%, n = 109) (Fig. 3B Fig. 4L,N) compared control MO- <br /> wide differential expression pattern mych embryonic             injected embryos (5%, n = 77) (Fig. 3A Fig. 4K,M). At 24hpf, <br /> development zebrafish.                                                  UTR MO SP MO-injected embryos (80%, n = 72) a <br />  <br />  <br />        PLoS ONE | www.plosone.org                                          2                                April 2008 | Volume 3 | Issue 4 | e2029 <br />                                                                                                                         Mych Neural Crest Formation <br />  <br />  <br />  <br />  <br /> Figure 3. Mych morphant phenotypes. A&#226;&#8364;&#8220;B. Group image 3- <br /> somite stages. Mych MO-injected embryos (B) reduced anterior <br /> and caudal regions compared control MO-injected embryos (A). C&#226;&#8364;&#8220;F. <br /> Lateral view 24hpf MO-injected embryos; (C,D) Control, (E) mych UTR <br /> MO, (F) mych SP MO. G&#226;&#8364;&#8220;H, J&#226;&#8364;&#8220;K. Lateral (G&#226;&#8364;&#8220;H) ventral (J&#226;&#8364;&#8220;K) views of <br /> control (G,J) mych UTR MO injected embryos (H,K) 72hpf. <br /> Arrowhead G points eye. Methyl green stained sections the <br /> eye shown insets (J&#226;&#8364;&#8482;,K&#226;&#8364;&#8482;).Yellow arrowhead J&#226;&#8364;&#8482; indicates ganglion <br /> cell layer, red arrow indicates photoreceptor cell layer control <br /> embryos; mych UTR MO-injected embryos retinal layering. F,I. <br /> Lateral view 24hpf mych SP MO injected embryo (F), embryo <br /> rescued coinjection mych mRNA (I). L,O. Embryo injected with <br /> mych UTR MO (2ng) plus mych SP MO (2.5ng) 24hpf (L), rescued <br /> embryo coinjection mych mRNA (O). M&#226;&#8364;&#8220;N. The <span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> of <br /> mych UTR MO-injected embryos (M) rescued mych mRNA (N), as <br /> seen 24hpf. h, heart. <br /> doi:10.1371/journal.pone.0002029.g003 <br />  <br /> small head reduced trunk (Fig. 3C&#226;&#8364;&#8220;F). To confirm                   Figure 4. Mych MO affects multiple regions 3-somite <br /> two MOs affect process injected combination                    stage. A&#226;&#8364;&#8220;P. Lateral views (A&#226;&#8364;&#8220;H) dorsal views (I&#226;&#8364;&#8220;P) control MO- <br /> half-maximal doses MO embryo observed                    injected (A,C,E,G,I,K,M) mych MO-injected embryos (B,D,F,H,J,L,N). <br /> similar phenotype (Fig. 3L) (85%, n = 88). The phenotypes elicited              Mab21l2 (A&#226;&#8364;&#8220;B&#226;&#8364;&#8482;) eye anlage midbrain marker; dmbx1a (C&#226;&#8364;&#8220;D&#226;&#8364;&#8482;) as <br /> by MO combination MOs rescued                        eye territory tectum marker; barhl2 (E,F) diencephalon marker; <br /> coinjection mych mRNA coinjection (Fig. 3I: 73% rescued,                     rx3 eye field marker. (I&#226;&#8364;&#8220;J) double <span id='am-2' about='rdfs:label' typeof='owl:Thing'>label</span> pax2a (red) egr2b <br />                                                                                 (blue). (K&#226;&#8364;&#8220;P). Embryos examined using probes, ctsl1b (red), <br /> n = 65; Fig. 3M,N: 85%, n = 94; Fig. 3O: 70%, n = 68). Head                 dlx3b (blue), ntl (blue), pcdh8 (red). (O&#226;&#8364;&#8220;P) mych MO-induced defects <br /> eye defects seen later stages (Fig. 3G&#226;&#8364;&#8220;H,J&#226;&#8364;&#8220;K),                 rescued co-injection mych mRNA. lnp, lateral neural plate; <br /> layering retina failed develop normally UTR MO-                mi, midbrain; n, notochord; pp, prechordal plate; r, rhombomere; psm, <br /> injected embryos 48hpf (Fig. 3J9,K9).                                        pre somitic mesoderm. <br />                                                                                 doi:10.1371/journal.pone.0002029.g004 <br /> Suppression mych affects expression anterior brain <br /> markers                                                                         injected embryos seen 3-somite stage. Mab21l2 and <br />    To analyze mych UTR MO phenotype carried                      dmbx1a, marking eye midbrain anlagen, reduced in <br /> out situ hybridization using early brain markers. During                     size expression level mych MO-injected embryos (Fig. 4A&#226;&#8364;&#8220; <br /> gastrulation expression hesx1/anf, six3a, otx2, zic1/opl             D), eye-specific gene rx3 showed size intensity <br /> were dramatically reduced intensity size                 reduction (Fig. 4G,H). In contrast, diencephalon marker barhl2 <br /> expression domains (Fig. 5A&#226;&#8364;&#8220;H). The regions marked                     altered mych MO injection (Fig. 4E,F). <br /> genes contribute specification eye territory                 In addition genes mark anterior neural plate also <br /> telencephalon [24]. By comparison, reduction                         tested posterior brain trunk markers injection mych <br /> expression hoxb1b posterior hindbrain eve1                 MO. Double staining pax2a egr2b mid- and <br /> future trunk-tail slightly reduced mych MO-injected                hindbrain indicates areas greatly affected by <br /> embryos (Fig. 5I&#226;&#8364;&#8220;L). Reduced anterior development mych MO-                   mych MO, egr2b expression rhombomere 5 reduced <br />  <br />  <br />        PLoS ONE | www.plosone.org                                           3                                April 2008 | Volume 3 | Issue 4 | e2029 <br />                                                                                                                      Mych Neural Crest Formation <br />  <br />  <br />  <br />                                                                              Early neural crest specification requires Mych function <br />                                                                                 Mych expressed mid- hindbrain early neural plate <br />                                                                              stages including preplacodal regions (Fig. 2), specifically in <br />                                                                              early neural crest seen overlapping staining crest <br />                                                                              marker foxd3 (Fig. 2P,Q). This pattern suggests possible function <br />                                                                              neural crest specification proceeds stage the <br />                                                                              junction neural non-neural ectoderm [25]. We <br />                                                                              tested effect injection mych MOs expression early <br />                                                                              neural crest marker foxd3 (Fig. 6A&#226;&#8364;&#8220;F). The foxd3 expression is <br />                                                                              dramatically reduced mych UTR MO (Fig. 6B), SP MO (Fig. 6 <br />                                                                              C), combination half-maximal dose MOs (Fig. 6E). <br />                                                                              We tested early neural crest markers snail1a, <br />                                                                              sox9b, sox10 dramatically reduced midbrain <br />                                                                              region completely lost hindbrain trunk neural crest <br />                                                                              regions (Fig. 6G&#226;&#8364;&#8220;L). To visualize cellular context the <br />                                                                              reduction expression genes sectioned foxd3- <br />                                                                              stained embryos (Fig. 6M,N). The foxd3 positive cells mych <br />                                                                              MO embryo contain reduced number neural crest cells a <br />                                                                              reduced thickness region neural plate-to-epidermis <br />                                                                              boundary (Fig. 6M9,N9). We suggest inhibition Mych <br />  <br />  <br />  <br />  <br /> Figure 5. Reduced expression early anterior brain markers <br /> after mych MO injection. A&#226;&#8364;&#8220;J. Dorsal views (A&#226;&#8364;&#8220;J) lateral views <br /> (K&#226;&#8364;&#8220;L) control MO-injected (A,C,E,G,I,K) mych UTR MO-injected <br /> embryos (B,D,F,H,J,L) 80% epiboly stage. The probes are <br /> indicated. <br /> doi:10.1371/journal.pone.0002029.g005 <br />  <br /> (Fig. 4I,J). Further, markers suggest widening axis,       Figure 6. mych MO impairs early neural crest induction. A&#226;&#8364;&#8220;F. <br /> also seen unstained embryos (Fig. 3B). Such widening                  Dorsal view foxd3 expression 4-somite stage embryos. Embryos <br />                                                                              injected 4ng mych UTR MO (B), 5ng mych SP MO (C), 2ng of <br /> further illustrated using probes label polster (future <br />                                                                              mych UTR MO (B) plus 2.5ng mych SP MO (C) showed reduced foxd3 <br /> hatching gland; ctsl1b), neural plate boundary placodal anlage           expression. This phenotype rescued mych mRNA coinjection <br /> (dlx3b), notochord (ntl), early somites (pcdh8/papc). All         (D,F). G&#226;&#8364;&#8220;L. Dorsal view control MO-injected (G,I,K) mych UTR MO <br /> these markers expressed approximately normal patterns             injected embryos (H,J,L) hybridized snail1a (G,H), sox9b (I,J), and <br /> the experimental embryos widened axis seen              sox10 (K,L). M&#226;&#8364;&#8220;N. Sections foxd3 expression region cont MO <br /> 82% (n = 89) mych MO-injected embryos, 92% (n = 45)               (M,M&#226;&#8364;&#8482;) mych UTR MO (N,N&#226;&#8364;&#8482;) injected embryos. Red asterisks A and <br />                                                                              B indicate location transverse sections, asterisks M N <br /> of control MO-injected embryos normal (Fig. 4K&#226;&#8364;&#8220;N). This                 indicate regions magnified M&#226;&#8364;&#8482;, N&#226;&#8364;&#8482;. h; hindbrain neural crest, m; <br /> axis abnormality fully rescued coinjection mych mRNA               midbrain neural crest, tnc; trunk neural crest. <br /> (85% normal, n = 80) (Fig. 4O,P).                                            doi:10.1371/journal.pone.0002029.g006 <br />  <br />  <br />        PLoS ONE | www.plosone.org                                        4                                April 2008 | Volume 3 | Issue 4 | e2029 <br />                                                                                                                        Mych Neural Crest Formation <br />  <br />  <br /> expression led loss neural crest precursors embryo, as <br /> supported experiments below. <br />  <br /> Mych knockdown disrupts pharyngeal arch development <br />    At 24hpf, mych strongly expressed developing <br /> pharyngeal arches maintained area two <br /> days subsequent development (Fig. 2). We observed effects of <br /> mych depletion pharyngeal development approaches. <br /> Whole mount situ hybridization 26hpf showed dlx2a and <br /> tbx1 expression lost branchial arches, defects in <br /> the mandibular hyoid arches comparatively mild (Fig. 7A <br /> and B, data shown). At 36hpf, hand2/dhand expression in <br /> the branchial arches likewise inhibited mych MO <br /> (Fig. 7C,D). Mesodermal endodermal development was <br /> likewise strongly inhibited mych MO injection, visualized <br /> by tbx1 [26] 45hpf (Fig. 7E,F). We loss myod <br /> expression MO-injected embryos, indicating loss pharyn- <br /> geal muscle (data shown). To test defects lead to <br /> loss arch tissue, injected mych MO fli1-eGFP <br /> transgenic line cranial neural crest visualized by <br /> GFP fluorescence [27]. A loss branchial arches seen, <br /> whereas portions mandibular hyoid arch structures <br /> were maintained (Fig. 7G,H). These early pharyngeal defects lead <br /> to loss parts cranial cartilages the <br /> neurocranium maintained, seen Alcian blue staining at <br /> day 5 (Fig. 7I&#226;&#8364;&#8220;L). These data indicate Mych required for <br /> pharyngeal arch development zebrafish embryo. <br />  <br /> Loss mych leads increased cell death early <br /> neural plate <br />    Myc family genes implicated regulation cell <br /> proliferation. Therefore tested inhibition mych <br /> expression decreases cell division embryo. Using Phospho- <br /> Histone H3 antibody identify proliferating cells no <br /> substantial difference control mych MO-injected <br /> embryos (data shown). <br />    As phenotypes resulting mych depletion be <br /> caused cell death, tested cell death different times <br /> during gastrulation segmentation stages. The earliest cell death <br /> was observed 80% epiboly stage anterior dorsal <br /> region, moderate increase number TUNEL- <br /> positive cells (data shown). At bud stage, mych MO-injected <br /> embryos showed highly increased numbers apoptotic cells, <br /> especially lateral edge neural plate brain <br />                                                                               Figure 7. Multiple defects pharyngeal arch development <br /> (65%, n = 67), compared control MO injected embryos, 95%                caused mych MO. Control MO (A,C,E,G,I,K) mych UTR MO- <br /> of showed low levels TUNEL-positive cells (n = 45)              injected embryos (B,D,F,H,J,L). A&#226;&#8364;&#8220;B. Lateral view dlx2a expression in <br /> (Fig. 8A,B). The cell death phenotype rescued mych mRNA                mych UTR MO-injected embryos 26hpf (B). C&#226;&#8364;&#8220;D. Ventral view of <br /> (80% normal, n = 76) (Fig. 8C). In neural plate                   expression hand2 36hpf. E&#226;&#8364;&#8220;F. Ventral view tbx1 expression at <br /> embryos 3-somite stage, dramatic increase TUNEL-                  45hpf. G&#226;&#8364;&#8220;H. Lateral view fli1-eGFP transgenic line 40hpf. I&#226;&#8364;&#8220;L. <br /> positive cells observed 85% (n = 45) mych MO-injected               Lateral (I,J) ventral (K,L) views Alcian blue stained day 5 control <br />                                                                               (I,K) mych UTR MO-injected embryos (J,L). ch, ceratohyal, b, <br /> embryos (Fig. 8E,E9), compared 1% (n = 34) control MO-               branchial arch; e, ethmoid plate; g, gill arches; h, hyoid arch; hs, <br /> injected embryos (Fig. 8D,D9). Again phenotype rescued               hyosymplectic; ht, heart; m, mandibular arch; mc, Meckel&#226;&#8364;&#8482;s cartilage; me, <br /> by injection mych mRNA 80% (n = 65) embryos                      mesoderm; pec, pectoral fin; pq, palatoquadrate. <br /> (Fig. 8F,F9). For statistical analysis, counted number              doi:10.1371/journal.pone.0002029.g007 <br /> TUNEL-positive cells anterior neural plate 20 embryos <br /> for injection condition; changes highly significant              crest derived. Zebrafish Myc family genes, including c-myc, N-myc, <br /> (Fig. 6G). These data indicate mcyh important                 L-myc max isolated previously distribution of <br /> survival neural plate cells early embryo development.                   transcripts reported [14,15]. The Mych protein is <br />                                                                               related Myc proteins, showing low sequence similarity the <br /> Discussion                                                                    N-terminal Myc domain (NTD), high similarity bHLH- <br />                                                                               LZ domain. As Myc proteins, Mych localized nucleus <br />    We isolated mych gene novel member Myc                function transcriptional factor (Fig. 1). Mych <br /> family zebrafish, shown involved survival       mRNA expressed maternally, restricted dorsal ventral <br /> of cells neural plate including region neural       domain gastrulation, later shows dynamic expression <br />  <br />  <br />        PLoS ONE | www.plosone.org                                         5                                April 2008 | Volume 3 | Issue 4 | e2029 <br />                                                                                                                        Mych Neural Crest Formation <br />  <br />  <br />                                                                                  eyes. These phenotypes fact mych <br />                                                                                  knockdown greatly increases level cell death anterior <br />                                                                                  neural plate (Fig. 8). While tissues affected mych MO- <br />                                                                                  injected embryos, neural crest differentiation especially <br />                                                                                  sensitive loss Mych function. Early markers neural crest <br />                                                                                  differentiation strongly reduced mych knockdown (Fig. 6), <br />                                                                                  development branchial arches severely disrupted, <br />                                                                                  effect extensive mandibular hyoid <br />                                                                                  arches (Fig. 7). Neural crest cells arise border neural and <br />                                                                                  non-neural domains, subsequently migrate multiple target <br />                                                                                  organs [10]. After migration pharyngeal arches, neural <br />                                                                                  crest cells undergo condensation chondrogenic differentiation <br />                                                                                  form cartilage elements developing craniofacial <br />                                                                                  skeleton. Expression mych maintained regions during <br />                                                                                  differentiation. A function mych supported the <br />                                                                                  observation knockdown expression causes reduction of <br />                                                                                  known regulatory genes branchial arches, dlx2a, and <br />                                                                                  ensuing morphological malformations (Fig. 7). These effects may <br />                                                                                  consequence increased apoptosis mych MO-injected <br />                                                                                  embryos, disruption branchial arch development in <br />                                                                                  conjunction apoptosis result loss Ap2 transcription <br />                                                                                  factor observed previously zebrafish [28,29]. The <br />                                                                                  specific involvement Myc family genes neural crest <br />                                                                                  development studied Xenopus mouse. In <br />                                                                                  Xenopus, c-Myc involved early neural crest specification [11], <br />                                                                                  conditional deletion c-Myc mouse results neural <br />                                                                                  crest defects, including reduction skull size deficits coat <br />                                                                                  pigmentation hearing [13]. In Xenopus, c-Myc appears have <br />                                                                                  role maintaining neural crest stem cells acting its <br />                                                                                  target Id3 prevent premature differentiation [11.12]. Zebrafish <br />                                                                                  contains additional members gene families compared to <br />                                                                                  tetrapods, genome duplication evolution [30]. The <br />                                                                                  resulting paralogs partly overlapping, partly distinct <br />                                                                                  functions correspond functions single <br />                                                                                  ortholog vertebrates. Therefore possible in <br />                                                                                  zebrafish mych takes functions carried N-Myc <br />                                                                                  c-Myc Xenopus mouse. Such model explain <br />                                                                                  role mych neural crest development, not <br /> Figure 8. Mych depletion results apoptosis early neural                clear molecular function mych zebrafish similar <br /> plate. A&#226;&#8364;&#8220;F&#226;&#8364;&#8482;. Detection cell death TUNEL assay. Lateral views               c-Myc Xenopus. In spite uncertainty, the <br /> (D,E,F) dorsal views (A,B,C,D&#226;&#8364;&#8482;,E&#226;&#8364;&#8482;,F&#226;&#8364;&#8482;) control MO (A,D,D&#226;&#8364;&#8482;), mych UTR       requirement mych function zebrafish neural crest develop- <br /> MO (B,E,E&#226;&#8364;&#8482;), rescued embryos received mych UTR MO and <br />                                                                                  ment supports view Myc family members essential <br /> mRNA (C,F,F&#226;&#8364;&#8482;) bud (A&#226;&#8364;&#8220;C) 3-somite stage (D&#226;&#8364;&#8220;F&#226;&#8364;&#8482;). Arrows B <br /> point TUNEL positive cells lateral edge neural plate. An       regulators neural crest development vertebrates. <br /> arrowhead B indicates TUNEL positive cells neural plate. <br /> Bracket D D&#226;&#8364;&#8482; indicates anterior neural plate region          Materials Methods <br /> TUNEL-positive cells counted. The results shown G. Average <br /> numbers positive cells embryo obtained counting 20                Fish strains <br /> embryos group. For comparisons p,0.01: Cont MO vs.                    Wild type zebrafish strains AB* maintained according to <br /> mych MO: p = 5.27e-14; mych MO vs. rescue: p = 4.87e-17. <br /> doi:10.1371/journal.pone.0002029.g008                                            The zebrafish book: A Guide Laboratory Use Zebrafish <br />                                                                                  (Danio rerio) [31]. The mibta52b mutant line obtained Ajay <br /> pattern high level expression certain regions                Chitnis, homozygote fli1-eGFP y1 transgenic line Brant <br /> anterior brain. For comparison carried situ hybridization              M. Weinstein. <br /> with zebrafish c-myc binding partner max. Both present <br /> maternally later expressed broad pattern (data            Isolation full-length mych cDNA RH mapping <br /> shown). In contrast, nmyc1 expression zygotic, tissue              The original partial clone 5144 [16] contains 1.6 kb, was <br /> distribution endoderm, retina, midbrain, hindbrain                    extended 1.9 kb full-length mych cDNA using SMART RACE <br /> branchial arches similar mych [15]. While mych                 cDNA Amplification Kit (ClonTech). Radiation hybrid mapping <br /> expression pattern gastrulation suggests involvement               LN54 panel [17], using primer sets: (1) <br /> gene dorsal-ventral patterning substantial changes             Forward 59-GCCGCAAGGAGGATCTGCGGACTT-39, Reverse <br /> the expression bmp2, chordin, goosecoid mych MO-injected               59-AGATACTAAC TCCAGCTGGTCCAC-39; (2) Forward <br /> embryos (data shown).                                                        59-TCGCCGACGTTTTCCGTCTACTTT-39, Reverse: 59- <br />    Inhibition mych expression resulted reduced size                 CAGTTGGAGAAAGTCTGTGTCCTC-39. Amino acid se- <br /> anterior brain major changes patterning (Figs. 4 5).              quence comparisons phylogenic tree analysis carried out <br /> At later stages observed strongly reduced size head                DNASIS MAX version 2.0 (MiraiBio, Hitachi software). <br />  <br />  <br />        PLoS ONE | www.plosone.org                                            6                              April 2008 | Volume 3 | Issue 4 | e2029 <br />                                                                                                                                            Mych Neural Crest Formation <br />  <br />  <br />  <br /> RT-PCR analysis                                                                             mono clonal anti-flag antibody (Sigma), 1:2000 dilution of <br />    To isolate total RNA, 20 cont MO mych SP MO                               mouse Alexa 488 (Invitrogen) secondary antibody. DAPI was <br /> injected embryos collected isolated using TRIzol reagent                           used stain nuclei. The images scanned Zeiss LSM 510 <br /> (Invitrogen). The first-strand cDNA synthesis performed                                 confocal microscope. <br /> SuperScript III First-Strand System (Invitrogen). RT-PCR reactions <br /> accomplished using following sets primers: Forward                                   Alcian blue Methyl Green staining <br /> F1-CTGGACTGCCACAC CGCGGCGCTCGCCTG/Reverse                                                      Pharyngeal cartilage staining carried previously <br /> R1-GCTGGAGCGCCGCACCGTCACCACATC; Forward F2-                                                 described [32] using Alcian blue (Sigma). The 0.5% methyl green <br /> GACAGCCAGGAGCAGATCGAATCCAC/Reverse R2-GCA-                                                  (Sigma) solution prepared 0.1 M sodium acetate buffer <br /> CGCTGCT GCTCCCGCCGGCTGTCCTC. The amplified                                                  (pH 4.2). After 5 minute staining, samples rinsed water <br /> genomic DNA confirmed sequencing analysis.                                          dehydrated 95% ethanol. <br />  <br /> Whole mount situ hybridization                                                           Histology <br />   Whole mount situ hybridization two-color situ                                      For sections, dehydrated embryos embedded JB-4 <br /> hybridization performed previously described [32]. Both                                  plastic resin (Polyscience Inc.), 7 mm sections obtained <br /> digoxigenin- fluorescein-labeled antisense RNA probes                              using Leica RM2165 microtome. The two-color double staining <br /> generated using RNA labeling kit (Roche).                                                sample embedded 10% gelatin (Electron Microscopy <br />                                                                                             Sciences) 1XPBS section performed using Vibratome <br /> mych MO rescue mRNA injection                                                           3000 (Ted Pella Inc.) 10 mm thickness. <br />    Antisense oligonucleotide Morpholinos (MO) designed <br /> and obtained Gene Tools, LLC. The mych 59untranslated <br />                                                                                             Supporting Information <br /> MO (UTR MO) does contain AUG translation start site <br /> sequences. The sequence UTR MO 59- ACTGTGGT-                                      Figure S1 Mych MO specificity. A&#226;&#8364;&#8220;B. Mych:GFP signal <br /> GATAAAAGT AGACGGAA-39, The mych splicing MO                                             detection bud stage injection (B) (A) <br /> designed splicing donor region intron exon 1 exon                            mych UTR MO. C. Schematic drawing mych SP MO design <br /> 2 (Figure S1 C). The sequence SP MO 59-GCAAAAGA                                   different sets RT-PCR primers. D&#226;&#8364;&#8220;E. RT-PCR shows <br /> CTCACCAGAATCGCTAG-39, control MO 59-CCTC-                                            SP MO eliminates normal mature mRNA band. <br /> TTACCTCAGTTACAATT TATA-39. In experiments 10ng                                          Embryos collected 3-somite stage. <br /> of Control MO, 4ng mych UTR MO, 5ng SP MO                                    Found at: doi:10.1371/journal.pone.0002029.s001 (8.19 MB TIF) <br /> injected one-cell stage embryos. Full-length mych mRNA was <br /> subcloned pCS2+ pCS2+-eGFP1 vectors synthesized                                 Acknowledgments <br /> by mMessagemMachine SP6 Kit (Ambion), 30 pg mRNA per <br /> embryo injected rescue experiments.                                                 We thank Mark Rath zebrafish maintenance, Neil A. Hukriede for <br />                                                                                             set information LN54 RH PCR mapping methods. <br /> TUNEL assay immunocytochemistry <br />    TUNEL assay performed described previously [32]. The                              Author Contributions <br /> localization Mych protein determined transfection                           Conceived designed experiments: ID SH. Performed the <br /> flag-tagged mych DNA NIH3T3 cells using FuGENE 6                                       experiments: SH. Analyzed data: ID SH MT. Contributed reagents/ <br /> tranfection reagent (Roche). We used 1:1000 dilution mouse                           materials/analysis tools: MT. Wrote paper: ID SH. <br />  <br /> References <br />  1. Dang CV (1999) c-Myc target genes involved cell growth, apoptosis,               13. Wei K, Chen J, Akrami K, Galbraith GC, Lopez IA, et al. (2007) Neural crest <br />     metabolism. Mol Cell Biol 19: 1&#226;&#8364;&#8220;11.                                                         cell deficiency c-myc causes skull hearing defects. Genesis 45: 382&#226;&#8364;&#8220;390. <br />  2. Grandori C, Cowley SM, James LP, Eisenman RN (2000) The Myc/Max/Mad                     14. Schreiber-Agus N, Horner J, Torres R, Chiu FC, DePinho RA (1993) Zebrafish <br />     network transcriptional control cell behavior. Annu Rev Cell Dev Biol            myc family max genes: differential expression oncogenic activity <br />     16: 653&#226;&#8364;&#8220;699.                                                                                vertebrate evolution. Mol Cell Biol 13: 2765&#226;&#8364;&#8220;2775. <br />  3. Oster SK, Marhin WW, Asker C, Facchini LM, Dion PA, et al. (2000) Myc             15. Loeb-Hennard C, Kremmer E, Bally-Cuif L (2005) Prominent transcription of <br />     essential negative regulator platelet-derived growth factor beta receptor                zebrafish N-myc (nmyc1) tectal retinal growth zones embryonic <br />     expression. Mol Cell Biol 20: 6768&#226;&#8364;&#8220;6778.                                                    early larval development. Gene Expr Patterns 5: 341&#226;&#8364;&#8220;347. <br />  4. Secombe J, Pierce SB, Eisenman RN (2004) Myc: weapon mass destruction.             16. Kudoh T, Tsang M, Hukriede NA, Chen X, Dedekian M, et al. (2001) A gene <br />     Cell 16: 153&#226;&#8364;&#8220;156.                                                                           expression screen zebrafish embryogenesis. Genome Res 11: 1979&#226;&#8364;&#8220;1987. <br />  5. Meyer N, Kim SS, Penn LZ (2006) The Oscar-worthy role Myc apoptosis               17. Hukriede NA, Joly L, Tsang M, Miles J, Tellis P, et al. (1999) Radiation <br />     Semin Cancer Biol 16: 275&#226;&#8364;&#8220;287.                                                              hybrid mapping zebrafish genome. Proc Natl Acad Sci USA 96: <br />  6. Ryan KM, Birnie GD (1996) Myc oncogenes: enigmatic fam <br /> </body></html>