<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>J protein cochaperone mitochondrial inner <br /> membrane required protein import the <br /> mitochondrial matrix <br /> Patrick D. D&#226;&#8364;&#8482;Silva*, Brenda Schilke*, William Walter*, Amy Andrew*&#226;&#8364;&#160;, Elizabeth A. Craig*&#226;&#8364;&#161; <br /> *Department Biochemistry, 433 Babcock Drive, University Wisconsin, Madison WI 53706; &#226;&#8364;&#160;Graduate Program Biomolecular Chemistry, <br /> 1300 University Avenue, University Wisconsin, Madison, WI 53706 <br />  <br /> Contributed Elizabeth A. Craig, September 24, 2003 <br />  <br /> The major Hsp70 mitochondrial matrix (Ssc1 yeast)               associated J protein Sec63 stimulate ATPase activity the <br /> critically important translocation proteins              luminal Hsp70 Kar2 essential translocation proteins <br /> cytosol, mitochondrial inner membrane,               cytosol endoplasmic reticulum lumen (9&#226;&#8364;&#8220;11). <br /> matrix. Tim44, peripheral inner membrane protein limited                 No J protein mitochondria reported involved <br /> sequence similarity J domain J-type cochaperones, tethers          protein translocation. However, mitochondrial J pro- <br /> Ssc1 import channel. Here report that, unlike J protein,         teins studied yeast Saccharomyces cerevisiae. <br /> Tim44 does stimulate ATPase activity Ssc1, does            The soluble matrix J protein Mdj1 known function Ssc1 <br /> affect stimulation known mitochondrial J protein          folding imported mitochondrially encoded proteins but <br /> a peptide substrate. Thus, conclude Tim44 does                   does play significant role import (12, 13). The J protein <br /> function J protein cochaperone Ssc1; rather, tethers Ssc1         Jac1 functions minor Hsp70 matrix, Ssq1, the <br /> <span id='am-2' about='protege:TO' typeof='owl:Thing'>to</span> import channel interactions independent              biogenesis Fe&#229;&#8230;&#190;S clusters (14, 15). Mdj2 J protein the <br /> critical J protein function. However, previously unstudied             inner membrane. However, deletion MDJ2 little no <br /> essential gene, PAM18, encodes 18-kDa protein contains             phenotypic effect; thus, function remains obscure (16). <br /> J domain localized mitochondrial inner membrane.                  Tim44, mitochondrial inner membrane protein, been <br /> Pam18 stimulates ATPase activity Ssc1; depletion Pam18             considered possible J protein partner Ssc1. Tim44, which <br /> in vivo disrupts import proteins mitochondrial matrix.           serves tether Ssc1 import channel, shows weak <br /> We propose Pam18 J protein partner Ssc1               similarity J domain Sec63, deletion region <br /> import channel critical Ssc1&#226;&#8364;&#8482;s function protein import.        disrupts interaction Tim44 Ssc1 mitochondria (17). <br />                                                                              Although controversial, recent reports indicate Tim44 <br />                                                                              interacts ATPase domain Ssc1, does J domain <br /> M       ost proteins mitochondrial matrix synthesized on <br />         cytosolic ribosomes imported <br /> across outer inner mitochondrial membranes. Translo- <br />                                                                              known J proteins (18, 19). However, regardless it <br />                                                                              acts J protein partner Ssc1, Tim44 regulates inter- <br /> cation inner membrane occurs inner                    action Ssc1 import channel. Tim44 tethers ATP- <br /> membrane channel driven membrane potential                 bound Ssc1 import channel. When polypeptide substrates <br /> an import motor (1&#226;&#8364;&#8220;3). Three critical components motor               bind Ssc1[ATP], Ssc1 released Tim44 absence <br /> are major Hsp70 molecular chaperone mitochondria                      ATP hydrolysis, permitting binding another <br /> (mtHsp70; Ssc1 yeast); peripheral inner membrane pro-                 Ssc1[ATP] channel second cycle interaction (20). <br /> tein Tim44, tethers Ssc1 import channel;                   Because importance import proteins into <br /> nucleotide release factor Ssc1, Mge1. Multiple cycles Ssc1            mitochondria cell function, set test idea that <br /> binding release translocating polypeptide, driving               Tim44, analogous Sec63, functions J protein partner of <br /> the import process, required import proteins             Ssc1 protein import. We Tim44 does have <br /> mitochondrial matrix (1&#226;&#8364;&#8220;3).                                                  activities associated prototypical J protein. Rather, a <br />    Ssc1, like Hsp70s, contains C-terminal domain                previously unstudied essential gene, PAM18, encodes J protein <br /> binds short segments unfolded polypeptides rich hydro-                 inner membrane important protein transloca- <br /> phobic amino acids (4, 5). The N-terminal ATPase domain                      tion vivo J protein partner Ssc1. <br /> regulates binding interaction adenine nucleo- <br /> tides. In turn, binding peptide segment C-terminal             Materials Methods <br /> domain stimulates ATPase activity N terminus. In              Yeast Strains, Plasmids, Genetic Techniques. PAM18 ob- <br /> ATP-bound state, interaction peptide substrate unstable,             tained PCR-amplifying genomic DNA position &#226;&#171;&#186;303 to <br /> with fast rates. In ADP-bound state,                 &#226;&#171;&#185;702 using Pfu Turbo polymerase (Stratagene) cloned <br /> interaction relatively stable, slow rates.                pRS316 (21). To obtain <span id='am-3' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-4' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-5' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span>null allele</span> PAM18, coding <br />                                                                                                                                                        BIOCHEMISTRY <br />  <br />  <br />  <br /> Therefore, thought ATP-bound form Hsp70                    sequences &#226;&#171;&#185;71 &#226;&#171;&#185;542 replaced TRP1 gene. <br /> initiates interaction polypeptide substrate,              A heterozygous PAM18&#229;&#8230;&#190;pam18 diploid PJ53 (22) carrying <br /> stabilized hydrolysis ATP. Exchange ATP bound               PAM18 plasmid sporulated. &#226;&#338;&#172;pam18 haploids carrying <br /> ADP results release peptide, completing cycle.            PAM18 pRS316 isolated dissecting tetrads YP <br />    Hsp70s rarely, ever, function independently. Rather,              medium (1% yeast extract&#229;&#8230;&#190;2% peptone). Mutants PAM18 <br /> function cochaperones. J proteins, named                 constructed site-directed mutagenesis using <br /> presence signature J domain interacts                   QuikChange protocol (Stratagene) cloned pRS315. <br /> ATPase domain, stimulate Hsp70&#226;&#8364;&#8482;s ATPase activity, facil-                   Pam18 epitope-tagged C terminus tandem <br /> itating interaction substrate polypeptides (6, 7). This activ- <br /> ity critical feature J protein function, amino acid <br /> alterations J domain decrease ATPase stimulation             Abbreviation: HA, hemagglutinin. <br /> functionally defective vivo (8). For example, lumen             &#226;&#8364;&#161;To   correspondence addressed. E-mail: ecraig@wisc.edu. <br /> the endoplasmic reticulum, ability membrane-                      &#194;&#169; 2003 The National Academy Sciences USA <br />  <br />  <br />  <br /> www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.1936150100                                             PNAS &#229;&#8230;&#169; November 25, 2003 &#229;&#8230;&#169; vol. 100 &#229;&#8230;&#169; no. 24 &#229;&#8230;&#169; 13839 &#226;&#8364;&#8220;13844 <br />  copies hemagglutinin (HA) peptide introducing a <br /> NotI site termination codon subcloning HA <br /> tag NotI fragment pGTEP1 (23). The tagged protein <br /> was functional; cells carrying tagged protein only <br /> PAM18 protein grew wild-type cells variety of <br /> conditions. For purposes purification, entire ORF of <br /> PAM18 cloned glutathione S-transferase expressing <br /> vector pGEX-KG (24), forming fusion protein Pam18 <br /> at C terminus. To modulate expression Pam18, entire <br /> ORF (&#226;&#171;&#185;1 &#226;&#171;&#185;507) cloned pCM189 (25), placing <br /> PAM18 control tetracycline-regulatable promoter, <br /> resulting repression transcription addition           Fig. 1. Effect Tim44 ATPase activity Ssc1. Ssc1:ATP complex (&#226;&#172;&#8225;0.2 <br /> tetracycline derivative doxycycline.                                    &#226;?&#174;M) incubated 23&#194;&#176;C presence various concentrations Mdj1 <br />    Complete synthetic media lacking specific amino acids             peptide P5. The rate conversion ADP measured plotted the <br /> containing 5-fluoroorotic acid prepared described (26).         fold stimulation basal rate determined absence additional <br /> All chemicals obtained Sigma unless stated.         components. (A) Mdj1, peptide P5, Tim44, indicated. (B) Reactions with <br />                                                                         1 &#226;?&#174;M Mdj1 (Left) 5 &#226;?&#174;M peptide P5 (Right) carried presence <br />                                                                         indicated concentrations Tim44. <br /> Fractionation Mitochondria. Mitochondria fractionated <br /> essentially described ref. 27. Briefly, 25 &#226;?&#174;g mitochondria <br /> were swollen 500 &#226;?&#174;l EM buffer (10 mM Mops KOH, pH                 purified Ssc1 antibodies crosslinked protein A- <br /> 7.2&#229;&#8230;&#190;1 mM EDTA) incubated ice 15 min. Efficiency <br />                                                                         Sepharose beads. Six micrograms Ssc1 incubated with <br /> of disruption outer membrane assessed incubation <br />                                                                         Ssc1 antibody beads 4&#194;&#176;C 1 h. Under condition, &#226;&#172;&#8225; 90% <br /> with proteinase K 50 &#226;?&#174;g&#229;&#8230;&#190;ml ice 15 min. As control, <br />                                                                         Ssc1 typically immunoprecipitated. After washes, <br /> sodium deoxycholate added 1% disrupt mitochon- <br /> drial inner membrane. Mitochondria separated mem-             Tim44 (0.6 &#226;?&#174;g) added. The mixture incubated final <br /> brane soluble fractions described ref. 28. Mitochondrial      volume 150 &#226;?&#174;l 23&#194;&#176;C 30 min point 1.5 &#226;?&#174;l a <br /> fractions subjected immunoblot analysis using            1 mg&#229;&#8230;&#190;ml solution peptide P5 added reactions, and <br /> ECL (Amersham Pharmacia Biotech) according                incubation continued additional 30 min. Tim44:Ssc1 <br /> manufacturer&#226;&#8364;&#8482;s suggestion, using polyclonal antibodies spe-          complex formation robust, forming 0&#194;&#176;C a <br /> cific Tim44, cytochrome b2, Mge1 monoclonal               variety salt buffer conditions (data shown). The final <br /> antibody specific HA.                                               concentrations Ssc1, Tim44, peptide P5 0.6, 0.09, <br />                                                                         10 &#226;?&#174;M, respectively. After centrifugation, beads were <br /> Protein Purification. Pam18 purified bacterial strain C41      washed times mitochondrial lysis buffer (250 mM <br /> (29). Expression induced 3 h 37&#194;&#176;C addition 1 mM        sucrose&#229;&#8230;&#190;80 mM KCl&#229;&#8230;&#190;20 mM Mops-KOH, pH 7.2&#229;&#8230;&#190;0.2% Triton <br /> isopropyl-&#226;?&#164;-D-thiogalactopyranoside. After affinity chromatog-          X-100&#229;&#8230;&#190;1 mM PMSF) containing appropriate nucleotide. <br /> raphy using glutathione agarose, biotinylated thrombin              Tim44 Ssc1 bound beads detected SDS&#229;&#8230;&#190; <br /> added mixture incubated 3 h 20&#194;&#176;C. The cleaved        PAGE immunoblot analysis using polyclonal antibodies <br /> Pam18 product recovered removal glutathione               specific Tim44 Ssc1. <br /> agarose beads centrifugation removal thrombin by <br /> streptavidin&#229;&#8230;&#190;agarose (catalog no. 69022-3, Novagen).                    Peptide Binding Assay. Briefly, fluorescence anisotropy assays for <br />    Tim44His (20), Mdj1His (30), Ssc1His (31), Ssq1His (31)     binding peptide carried described refs. 20 and <br /> purified described. All His-tagged proteins previously          32. Peptide P5 labeled fluorescein generate F-P5 as <br /> shown able rescue absence normal protein vivo.   described (33). Various concentrations Ssc1 incubated <br />                                                                         F-P5 (10 nM) 23&#194;&#176;C buffer A (25 mM Hepes-KOH, pH <br /> ATPase Assay. Isolation Hsp70:ATP complexes ATPase               7.4&#229;&#8230;&#190;100 mM KCl&#229;&#8230;&#190;11 mM Mg(OAC)2&#229;&#8230;&#190;10% glycerol). After <br /> assays carried essentially described refs. 20 32,    binding reached equilibrium, anisotropy measurements were <br /> with changes. To obtain radiolabeled ATP:Ssc1 complexes,           Beacon 2000 Fluorescence Polarization system <br /> 50 &#226;?&#174;g Hsp70 incubated 100 &#226;?&#174;Ci [&#226;?&#163;-32P]ATP (3,000          (Panvera, Madison, WI) 23&#194;&#176;C excitation 490 nm and <br /> Ci&#229;&#8230;&#190;mmol, DuPont) buffer A (25 mM Hepes-KOH, pH                       emission 535 nm. <br /> 7.4&#229;&#8230;&#190;100 mM KCl&#229;&#8230;&#190;11 mM Mg(OAC)2&#229;&#8230;&#190;10% glycerol) containing <br /> 25 &#226;?&#174;M ATP 100 &#226;?&#174;l ice 30 sec. The complex isolated         Results <br /> immediately NICK column (Amersham Pharmacia Bio- <br />                                                                         Tim44 Does Not Stimulate ATPase Activity Ssc1. To address the <br /> tech) frozen aliquots &#226;&#171;&#186;70&#194;&#176;C. For ATPase assays, <br />                                                                         possibility Tim44 acts J protein cochaperone the <br /> aliquots thawed ice; time 0 time shift 23&#194;&#176;C, <br />                                                                         import channel, tested ability affect ATPase activity <br /> when additions peptide P5 (CALLLSAPRR), Tim44, Mdj1, <br /> and&#229;&#8230;&#190;or Pam18 samples. At indicated                Ssc1 using single-turnover conditions. A complex between <br /> times, 3-&#226;?&#174;l aliquots removed mixed 1 &#226;?&#174;l stop           Ssc1 radiolabeled ATP isolated, rate ADP <br /> solution containing 4 M formic acid, 2 M LiCl, 36 mM ATP.           formation determined incubation presence and <br /> The mixture spotted polyethyleneimine-cellulose              absence Tim44. The basal rate ATP hydrolysis Ssc1 was <br /> TLC plate developed 1 M formic acid 0.5 M LiCl.              0.07 min&#226;&#171;&#186;1, consistent previous results (32). No effect on <br /> After drying, TLC plate exposed PhosphorImager             ATPase activity Ssc1 observed addition of <br /> screen (Molecular Dynamics), ATP ADP              Tim44 range concentrations, 35 &#226;?&#174;M, 140 <br /> was quantified PhosphorImager (Molecular Dynam-             molar excess Ssc1 (Fig. 1A). As control, increasing <br /> ics). The percentage hydrolysis 0 time (&#226;&#172;&#8225;20%) sub-            concentrations soluble mitochondrial J protein <br /> tracted time points.                                           Mdj1 peptide substrate added. Both stimulated the <br />                                                                         hydrolysis ATP. For example, 10-fold molar excess (2 &#226;?&#174;M) <br /> Coimmunoprecipitation Assay. The Ssc1 coimmunoprecipitation             Mdj1, activity stimulated 7-fold; 100-fold excess (20 <br /> assay carried essentially described ref. 20. Affinity-    &#226;?&#174;M), peptide activity stimulated 6.5-fold. <br />  <br /> 13840 &#229;&#8230;&#169; www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.1936150100                                                                                      D&#226;&#8364;&#8482;Silva et al. <br />  Fig. 2. Functional defect Ssc1A503D. (A) ATPase activity. Ssc1:ATP Ssc1A503D:ATP complexes (&#226;&#172;&#8225;0.2 &#226;?&#174;M) incubated 23&#194;&#176;C presence Mdj1 <br /> or peptide P5. The basal rates hydrolysis mutant wild-type protein indistinguishable. (B) Peptide binding measured fluorescence anisotropy: <br /> Fluorescein-labeled F-P5 (10 nM) incubated presence indicated concentrations wild-type A503D mutant protein ADP. Anisotropy <br /> measurements taken 25&#194;&#176;C; raw polarization values fitted one-site binding (hyperbola) equation determine Kd. (C) Effect peptide on <br /> Ssc1:Tim44 interaction. Tim44 (0.09 &#226;?&#174;M) incubated Ssc1 (0.6 &#226;?&#174;M) presence ATP ADP, indicated, 30 min. In cases, 10 &#226;?&#174;M <br /> peptide P5 added, incubation continued additional 30 min. The mixtures immunoprecipitated using Ssc1-specific antibodies described <br /> in Materials Methods. The presence Tim44 Ssc1 precipitate assessed immunoblot analysis using antibodies specific respective <br /> proteins. <br />  <br />  <br />  <br /> Tim44 Interaction Ssc1 Is Distinct That J Protein. To              rate Ssc1 observed, vitro, presence high <br /> assess Tim44 interacting Ssc1 sites similar                concentration peptide substrate turnover minute. <br /> or overlapping contacting J protein, tested                   This rate hydrolysis clearly sufficient drive import in <br /> effect increasing concentrations Tim44 ability                    vivo, rate cycle interaction would <br /> Mdj1 stimulate ATPase activity Ssc1. No effect                       occur imported polypeptide. Assuming rate of <br /> stimulation seen Tim44 added concentrations                       ATP hydrolysis Ssc1 observe vitro assays is <br /> between 5 35 &#226;?&#174;M, Mdj1 held constant 1 &#226;?&#174;M                         reflective natural state protein, cofactor Ssc1 <br /> (Fig. 1B). Similar experiments carried presence                   stimulates ATP hydrolysis exist. <br /> of peptide substrates. Again, effect seen peptide                          Because Tim44 does stimulate ATPase activity Ssc1, <br /> stimulation ATPase activity addition Tim44. These                      searched yeast genome database (www.yeastgenome.org) <br /> results indicate Tim44 does act J protein partner                potential J protein partners Ssc1. The ORF YLR008C <br /> Ssc1, does interfere binding substrate peptide.                  (PAM18) essential gene that <br />    To probe differences Tim44 J protein                   encodes protein containing predicted internal membrane <br /> interactions, used mutant isolated search                   spanning domain (Fig. 3A) J domain 57% identical <br /> for temperature-sensitive point mutations SSC1. Ssc1A503D                      J protein mitochondrial inner membrane <br /> has single amino acid alteration peptide-binding domain.                 Mdj2 (16). We confirmed deletion PAM18 W303 <br /> The ATPase activity A503D mutant protein stimu-                        strain background resulted inviability, deletion MDJ2 <br /> lated Mdj1 (Fig. 2A). However, interactions Tim44                     little phenotypic effect (data shown). <br /> a peptide substrate normal. Ssc1A503D bound fluorescein-                        To determine J domain Pam18 important for <br /> tagged peptide affinity wild-type protein,                  function, constructed mutants causing alterations in <br /> a dissociation constant (Kd) 0.3 &#226;?&#174;M (20), measured                       J domain. Because hallmark J domain highly <br /> fluorescence anisotropy (32) (Fig. 2B). A peptide substrate                       conserved tripeptide, histidine, proline, aspartic acid <br /> stimulated ATPase activity Ssc1A503D nearly                     (HPD), constructed <span id='am-81' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-82' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span>mutation</span> results replace- <br /> wild-type protein (4.0- 4.2-fold, respectively) peptide                  ment HPD alanines (Pam18A3). In addition, we <br /> concentration 15 &#226;?&#174;M (Fig. 2 A). Tim44 bound                            constructed mutation results substitution glu- <br /> Ssc1[ATP] Ssc1A503D[ADP], and, wild-type pro-                         tamine place histidine (Pam18H141Q). To test whether <br /> tein, peptide able stimulate release Tim44                       mutants rescue inviability &#226;&#338;&#172;pam18 strain, <br /> Ssc1A503D[ATP] complex (ref. 20 Fig. 2C). Thus, interac-                      plasmids carrying PAM18 mutants transformed a <br /> tion Tim44 peptide substrate, J protein                      &#226;&#338;&#172;pam18 strain carrying wild-type copy PAM18 cen- <br />                                                                                                                                                                     BIOCHEMISTRY <br />  <br />  <br />  <br /> Mdj1, normal mutant Ssc1 protein. Because Tim44                      tromeric plasmid having URA3 gene. When resulting <br /> cannot compete binding Mdj1 peptide interacts                      strains plated media containing 5-fluoroorotic acid, which <br /> normally Ssc1 A503D, conclude Tim44 J                       selects cells having lost URA3-containing plasmid carrying <br /> protein does bind sites Ssc1 prevent                       wild-type PAM18 gene, viable cells recovered (Fig. 3B). <br /> interaction known J protein.                                               Thus, J domain essential PAM18 function. <br />                                                                                      For Pam18 J protein partner Ssc1 vivo, must <br /> Essential J Protein Mitochondrial Inner Membrane, Pam18.                   localized mitochondria J domain interact <br /> Multiple binding release cycles Ssc1 incoming                      Ssc1, matrix. To determine cellular <br /> polypeptide chain required import (1&#226;&#8364;&#8220;3). It                  location Pam18, mitochondria cells expressing Pam18 <br /> estimated that, average, vivo, polypeptide chains pass                HA tag C terminus (Pam18-HA) isolated. <br /> through import channel minute (34). Thus, turnover                   These mitochondria contained protein reacted the <br /> rate ATP hydrolysis Ssc1 sufficiently fast allow                 HA-specific antibody present mitochondria <br /> numerous interaction cycles time frame. The turnover                  expressing Pam18 lacking HA tag (data shown). Mito- <br />  <br /> D&#226;&#8364;&#8482;Silva et al.                                                                                      PNAS &#229;&#8230;&#169; November 25, 2003 &#229;&#8230;&#169; vol. 100 &#229;&#8230;&#169; no. 24 &#229;&#8230;&#169; 13841 <br />                                                                                     localize Pam18, mitochondria having intact inner <br />                                                                                    membrane, disrupted outer membrane, formed by <br />                                                                                    swelling mitochondria buffer low osmotic strength. The <br />                                                                                    swollen mitochondria treated protease, were <br />                                                                                    intact mitochondria (Fig. 3C Left). Like Tim44, intermem- <br />                                                                                    brane space protein cytochrome b2, matrix protein Mge1, <br />                                                                                    Pam18 substantially affected protease digestion of <br />                                                                                    intact mitochondria, suggesting Pam18 outer <br />                                                                                    membrane. However, treatment swollen mitochondria with <br />                                                                                    protease resulted production predominant cleavage <br />                                                                                    fragment containing HA tag. These results suggest that <br />                                                                                    Pam18 located inner membrane. In addition, because <br />                                                                                    HA tag C terminus resistant protease attack, <br />                                                                                    J domain C-terminal putative membrane- <br />                                                                                    spanning domain predicted matrix the <br />                                                                                    inner membrane, N terminus protruding the <br />                                                                                    intermembrane space. Such orientation make J <br />                                                                                    domain accessible Ssc1 matrix. <br />                                                                                       To assess Pam18 role mitochondrial protein <br />                                                                                    import vivo, PAM18 placed control the <br />                                                                                    tetracycline-repressible promoter. Drug added turn off <br />                                                                                    expression, accumulation precursor form nuclear- <br />                                                                                    encoded mitochondrial protein, Hsp60, monitored. As the <br />                                                                                    case nuclear-encoded matrix proteins, prese- <br />                                                                                    quence Hsp60 cleaved matrix processing protease <br />                                                                                    immediately traversing inner membrane (35); the <br />                                                                                    accumulation normally undetectable precursor form has <br />                                                                                    used previously vivo indicator function the <br />                                                                                    import process (28, 36). Cells harvested 24 h drug <br />                                                                                    addition, growth tester strain begun slow <br />                                                                                    compared control strain. The precursor form Hsp60 <br />                                                                                    accumulated strain Pam18 expression was <br />                                                                                    repressed, control strain. This accumulation is <br />                                                                                    indicative role Pam18 import proteins the <br />                                                                                    mitochondrial matrix vivo. <br />  <br />                                                                                    Pam18 Specifically Stimulates Ssc1 ATPase Activity. The results <br />                                                                                    described previous section suggest Pam18 acts J <br />                                                                                    protein mitochondrial import. We purified PAM18 <br />                                                                                    wild-type mutant proteins tested ability stim- <br /> Fig. 3. In vivo function J protein Pam18. (A) Predicted amino acid          ulate ATPase activity Ssc1 using single-turnover assay <br /> sequence Pam18. A box drawn putative membrane-spanning            described above. Wild-type Pam18 stimulated ATPase ac- <br /> region. The region having sequence identity J domain underlined;       tivity 7-fold present 7-fold excess (Fig. 4A). Pam18 and <br /> highly conserved HPD indicated ***. (B) Requirement vivo J domain      Mdj1 similar concentration dependences stimula- <br /> of Pam18. Strains carrying plasmids expressing protein (&#226;&#171;&#186;), wild-type           tion Ssc1 ATPase activity, suggesting proteins <br /> Pam18, mutant Pam18, Pam18AAA Pam18H141Q (Pam18HI/Q) spot-             interact similarly Ssc1. As case Mdj1, addition <br /> ted 1- fold dilution series media lacking (Left) containing (Right) <br />                                                                                    excess Tim44 did inhibit ability Pam18 stimulate <br /> 5-fluoroorotic acid. Plates incubated 3 days 30&#194;&#176;C. (C) Fractionation <br /> of mitochondria. Mitochondria purified cells expressing Pam18 <br />                                                                                    Ssc1 (Fig. 4B), indicating binding Tim44 Pam18 are <br /> having HA tag C terminus. Mitochondria separated               mutually exclusive. <br /> membrane soluble components (Right). Mitochondrial protein (T)                To test Pam18 stimulates Ssc1 ATPase activity a <br /> fractional equivalents membrane pellet (P) supernatant (S)             prototypical J protein, used J domain mutants described <br /> analyzed. Mitochondria (M) subjected swelling disrupt outer         previous section, Ssc1 mutant a <br /> membrane (SW) (Left). Equivalent amounts treated proteinase K            defective interaction known J protein Mdj1. Pam18 <br /> and cases treated detergent (sodium deoxycholate), indicated.      mutant proteins altered J domains unable to <br /> The fractions subjected immunoblot analysis using antibodies            function vivo failed stimulate Ssc1&#226;&#8364;&#8482;s ATPase activity <br /> specific HA tag Pam18, antibody specific known <br />                                                                                    (Fig. 4A). In addition, tested ability wild-type Pam18 <br /> proteins intermembrane space (cytochrome b2), inner membrane <br /> (Tim44), matrix (Mge1). (D) &#226;&#338;&#172;pam18 cells carrying Pam18 control of <br />                                                                                    stimulate ATPase activity A503D Ssc1 mutant (Fig. <br /> the tetracycline regulatable promoter grown overnight rich media,       4C). Like Mdj1 (Fig. 2D), Pam18 did stimulate mutant <br /> which point subcultured presence absence tetra-         protein. Also, stimulation Pam18 specific, it <br /> cycline analogue doxycycline. After 24 h whole-cell lysates prepared      failed stimulate ATPase activity Hsp70 the <br /> separated SDS&#229;&#8230;&#190;PAGE, followed immunoblot analysis using antibodies         mitochondrial matrix Ssq1 (Fig. 4D). Thus, conclude that <br /> specific Hsp60.                                                                Pam18 J protein partner Ssc1. <br />  <br />                                                                                    Discussion <br /> chondria containing Pam18-HA separated membrane                          The data presented allow better understanding the <br /> and soluble fractions, subjected immunoblot                     mechanism translocation proteins mitochondrial <br /> analysis using HA-specific antibodies. Pam18-HA present                     inner membrane. First, Tim44 tethers Ssc1 import <br /> in membrane fraction, Tim44 (Fig. 3C Right). To                         channel, does function J protein cochaperone for <br />  <br /> 13842 &#229;&#8230;&#169; www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.1936150100                                                                                       D&#226;&#8364;&#8482;Silva et al. <br />  Fig. 4. Stimulation Ssc1 ATPase activity Pam18. Hsp70:ATP complexes prepared, rate hydrolysis measured 23&#194;&#176;C presence <br /> of various concentrations Mdj1, wild-type Pam18, mutant Pam18, presence absence Tim44. The rate conversion ADP measured and <br /> is plotted fold stimulation basal rate. In panel, Hsp70 ATP hydrolysis measured indicated asterisk. (A) Ssc1:ATP <br /> (&#226;&#172;&#8225;0.2 &#226;?&#174;M) incubated presence various concentrations wild-type Pam18, Mdj1, Pam 18 mutant proteins Pam18AAA Pam18H141Q. (B) <br /> Reactions 1 &#226;?&#174;M Pam18 carried presence indicated concentrations Tim44. (C) Ssc1:ATP Ssc1A503D:ATP complexes (&#226;&#172;&#8225;0.2 &#226;?&#174;M) were <br /> incubated presence excess Pam18. (D) Ssq1:ATP complexes (0.1 &#226;?&#174;M) presence various concentrations Pam18. <br />  <br />  <br />  <br /> mitochondrial Hsp70 Ssc1, does bind site(s)                 induced interaction peptide substrate rapidly de- <br /> Hsp70 J protein. Second, identified J protein                      stabilizes Ssc1:Tim44 interaction (20). Likewise, binding of <br /> partner Ssc1 inner membrane stimulates Ssc1&#226;&#8364;&#8482;s                     Mge1 Ssc1[ADP] destabilizes interaction Tim44 (20, 37). <br /> ATPase activity. These results lead model Ssc1,                     Thus, Tim44 evolved conformational-sensitive switch <br /> tethered import channel, stimulated J protein                  governing association Ssc1 import channel. <br /> in close proximity, ensuring rapid binding incoming                      The fact Tim44 J protein identification of <br /> polypeptide chain.                                                                new J protein partner Ssc1 allow forward revised <br />    Determining mechanism action Tim44 protein                        model protein import. In contrast endoplasmic retic- <br /> translocation important, central initial                  ulum, tethering import channel stimulation of <br /> interaction translocating protein import motor. The                 ATPase activity executed protein, J protein Sec63 <br /> inability Tim44 stimulate ATPase activity Ssc1,              (38), mitochondria separate critical functions two, or <br /> concentrations 20-fold higher required 5-fold                     more, proteins. Tim44 regulates Ssc1&#226;&#8364;&#8482;s interaction with <br /> stimulation known J protein partner Ssc1, Mdj1, argues                  import channel, Pam18 stimulates Ssc1&#226;&#8364;&#8482;s ATPase activ- <br /> strongly Tim44 functioning J cochaperone Ssc1. The                ity. Occupancy biologically relevant form Ssc1, <br /> original idea Tim44 function J-type cochaperone               Ssc1[ATP], predominates import channel (20). Interaction <br /> Ssc1 based limited sequence homology J domains                  incoming polypeptide stimulates dissociation Ssc1 <br /> of J proteins (17). It possible J-like region evolved            Tim44 and, combination Pam18, stimulates ATP <br /> <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> true J domain facilitate interaction Ssc1                   hydrolysis, stabilizing Ssc1:polypeptide substrate interaction. <br /> Tim44. In fact, 18-aa deletion completely removing region                 Interestingly, Ssc1&#226;&#8364;&#8482;s release Tim44 substrate binding <br /> resulted synthesis stable mutant protein inserted                independent ATP hydrolysis. Because Pam18&#226;&#8364;&#8482;s ability to <br /> correctly membrane defective interaction                   stimulate ATPase activity Ssc1 affected Tim44, <br /> Ssc1. These results suggested region Tim44 interacts                 determining relative rates release Tim44 ATP <br /> directly Ssc1 (17). However, scenario Tim44 having                   hydrolysis needed definitive model of <br /> evolved J-like domain interacts Ssc1 J domain                  import forth. However, noted that <br /> binding site unlikely competition functional                    mechanistically important ATP hydrolysis temporally <br /> interaction Tim44 true J proteins Mdj1 Pam18                  coupled release, failure hydrolyze ATP result <br /> were observed. Thus, interaction sites J proteins Tim44                dissociation polypeptide chain Ssc1 the <br /> are likely distinct; detailed structural information                 extreme instability Ssc1:substrate interaction Ssc1 is <br /> required clearly delineate exact sites interactions               bound ATP. Pam18 surely plays critical role stimulating <br /> these proteins.                                                                   hydrolysis rate. It play role coordinating the <br />    Our results indicate Pam18 prime candidate                 timing hydrolysis, allowing efficient coupling release from <br /> J protein partner Ssc1 mitochondrial import. As case                Tim44 ATP hydrolysis. <br />                                                                                      Determination exact mechanistic details protein import <br /> for components import machinery, Tim44, <br />                                                                                   coordination action Ssc1 cofactors awaits further <br /> Ssc1, Mge1, Pam18 essential. It located inner <br />                                                                                   analysis. However, precedent additional proteins <br />                                                                                                                                                                    BIOCHEMISTRY <br />  <br />  <br />  <br /> membrane, J domain extending matrix, well <br />                                                                                   mediating interactions Hsp70s&#229;&#8230;&#190;J proteins&#229;&#8230;&#190;substrate <br /> positioned interact Ssc1 tethered import <br />                                                                                   polypeptides. In Thermus thermophilus stable complex between <br /> channel. As expected bona fide J protein partner, can <br />                                                                                   Hsp70 J protein mediated protein, DafA (39). <br /> stimulate ATPase activity Hsp70 Ssc1, it <br />                                                                                   Upon binding substrate polypeptide, DafA displaced from <br /> functions, does stimulate ATPase activity of <br />                                                                                   complex. Interactions adapter proteins Tim44 and <br /> another matrix Hsp70, Ssq1. This stimulation affected by <br />                                                                                   DafA common presently appreciated serve <br /> the presence Tim44. Thus, Pam18 able stimulate                     important roles regulating interactions Hsp70s, J proteins, <br /> the ATPase activity Ssc1 tethered import                    substrate polypeptides. <br /> channel Tim44. <br />    However, Tim44 considered simply membrane                      We thank Jarek Marszalek helpful discussions thoughtful <br /> anchor Ssc1. The interaction proteins                   comments manuscript. This work supported National <br /> to evolved exquisitely sensitive conformation                   Institutes Health Grants GM278709 (to E.A.C.) 5T326M08349 <br /> of Ssc1. For example, conformational change Ssc1[ATP]                        (to A.A.). <br />  <br />  <br /> D&#226;&#8364;&#8482;Silva et al.                                                                                     PNAS &#229;&#8230;&#169; November 25, 2003 &#229;&#8230;&#169; vol. 100 &#229;&#8230;&#169; no. 24 &#229;&#8230;&#169; 13843 <br />   1.   Neupert, W. &amp; Brunner, M. (2002) Nat. Rev. Mol. Cell Biol. 8, 555&#226;&#8364;&#8220;565.               22.   James, P., Pfund, C. &amp; Craig, E. (1997) Science 275, 387&#226;&#8364;&#8220;389. <br />  2.   Pfanner, N. &amp; Geissler, A. (2001) Nat. Rev. Mol. Cell Biol. 5, 339&#226;&#8364;&#8220;349.              23.   Roof, D. M., Meluh, P. B. &amp; Rose, M. D. (1992) J. Cell Biol. 118, 95&#226;&#8364;&#8220;108. <br />  3.   Matouschek, A., Pfanner, N. &amp; Voos, W. (2000) EMBO Rep. 5, 404&#226;&#8364;&#8220;410.                  24.   Guan, K. L. &amp; Dixon, J. E. (1991) Anal. Biochem. 192, 262&#226;&#8364;&#8220;267. <br />  4.   Bukau, B. &amp; Horwich, A. L. (1998) Cell 92, 351&#226;&#8364;&#8220;366.                                  25.   Gari, E., Piedrafita, L., Aldea, M. &amp; Herrero, E. (1997) Yeast 13, 837&#226;&#8364;&#8220;848. <br />  5.   Hartl, F. &amp; Hayer-Hartl, M. (2002) Science 295, 1852&#226;&#8364;&#8220;1858.                           26.   Sikorski, R. S. &amp; Boeke, J. D. (1991) Metholds Enzymology: Guide Yeast <br />  6.   Cheetham, M. E. &amp; Caplan, A. J. (1998) Cell Stress Chaperones 3, 28&#226;&#8364;&#8220;36.                    Genetics Molecular Biology, eds. Guthrie, C. &amp; Fink, G. R., Vol. 194, pp. <br />  7.   Kelley, W. (1999) Curr. Biol. 9, R305&#226;&#8364;&#8220;R308.                                                302&#226;&#8364;&#8220;318. <br />  8.   Genevaux, P., Schwager, F., Georgopoulos, C. &amp; Kelley, W. L. (2002) Genetics         27.   Ryan, M. T., Voos, W. &amp; Pfanner, N. (2001) Methods Cell Biol. 65, 189&#226;&#8364;&#8220;215. <br />       162, 1045&#226;&#8364;&#8220;1053.                                                                      28.   Kang, P. J., Ostermann, J., Shilling, J., Neupert, W., Craig, E. A. &amp; Pfanner, <br />  9.   Brodsky, J. L. &amp; Schekman, R. (1993) J. Cell Biol. 123, 1355&#226;&#8364;&#8220;1363.                         N. (1990) Nature 348, 137&#226;&#8364;&#8220;143. <br /> 10.   Matlack, K. E. S., Plath, K., Misselwitz, B. &amp; Rapoport, T. A. (1997) Science        29.   Miroux, B. &amp; Walker, J. E. (1996) J. Mol. Biol. 260, 289&#226;&#8364;&#8220;298. <br />       277, 938&#226;&#8364;&#8220;941.                                                                        30.   Horst, M., Oppliger, W., Rospert, S., Schonfeld, H.-J., Schatz, G. &amp; Azem, A. <br /> 11.   Corsi, A. K. &amp; Schekman, R. (1997) J. Cell Biol. 137, 1483&#226;&#8364;&#8220;1493.                           (1997) EMBO J. 16, 1842&#226;&#8364;&#8220;1849. <br /> 12.   Hermann, J., Stuart, R., Craig, E. &amp; Neupert, W. (1994) J. Cell Biol. 127,           31.   Dutkiewicz, R., Schilke, B., Knieszner, H., Walter, W., Craig, E. A. &amp; <br />       893&#226;&#8364;&#8220;902.                                                                                   Marszalek, J. (2003) J. Biol. Chem. 278, 29719&#226;&#8364;&#8220;29727. <br /> 13.   Rowley, N., Prip-Buus, C., Westermann, B., Brown, C., Schwarz, E., Barrell, B.       32.   Liu, Q., Krzewska, J., Liberek, K. &amp; Craig, E. A. (2001) J. Biol. Chem. 276, <br />       &amp; Neupert, W. (1994) Cell 77, 249&#226;&#8364;&#8220;259.                                                     6112&#226;&#8364;&#8220;6118. <br /> 14.   Lutz, T., Westermann, B., Neupert, W. &amp; Herrmann, J. (2001) J. Mol. Biol. 307,       33.   Montgomery, D., Morimoto, R. &amp; Gierasch, L. (1999) J. Mol. Biol. 286, <br />       815&#226;&#8364;&#8220;825.                                                                                   915&#226;&#8364;&#8220;932. <br /> 15.   Voisine, C., Cheng, Y. C., Ohlson, M., Schilke, B., Hoff, K., Beinert, H.,           34.   Lim, J. H., Martin, F., Guiard, B., Pfanner, N. &amp; Voos, W. (2001) EMBO J. 20, <br />       Marszalek, J. &amp; Craig, E. (2001) Proc. Natl. Acad. Sci. USA 98, 1483&#226;&#8364;&#8220;1488.                 941&#226;&#8364;&#8220;950. <br /> 16.   Westermann, B. &amp; Neupert, W. (1997) J. Mol. Biol. 272, 477&#226;&#8364;&#8220;483.                      35.   Cheng, M., Hartl, F., Martin, J., Pollock, R., Kalousek, F., Neupert, W., <br /> 17.   Merlin, A., Voos, W., Maarse, A., Meijer, M., Pfanner, N. &amp; Rassow, J. (1999)              Hallberg, E., Hallberg, R. &amp; Horwich, A. (1989) Nature 337, 620&#226;&#8364;&#8220;625. <br />       J. Cell Biol. 145, 961&#226;&#8364;&#8220;972.                                                          36.   Gambill, B. D., Voos, W., Kang, P. J., Miao, B., Langer, T., Craig, E. A. &amp; <br /> 18.   Krimmer, T., Rassow, J., Kunau, W. H., Voos, W. &amp; Pfanner, N. (2000) Mol.                  Pfanner, N. (1993) J. Cell Biol. 123, 109&#226;&#8364;&#8220;117. <br />       Cell. Biol. 20, 5879&#226;&#8364;&#8220;5887.                                                           37.   Schneider, H. C., Westermann, B., Neupert, W. &amp; Brunner, M. (1996) EMBO <br /> 19.   Strub, A., Rottgers, K. &amp; Voos, W. (2002) EMBO J. 21, 2626&#226;&#8364;&#8220;2635.                           J. 15, 5796&#226;&#8364;&#8220;5803. <br /> 20.   Liu, Q., D&#226;&#8364;&#8482;Silva, P., Walter, W., Marszalek, J. &amp; Craig, E. A. (2003) Science 300,   38.   Rapoport, T., Matlack, K., Plath, K., Misselwitz, B. &amp; Staeck, O. (1999) Biol. <br />       139&#226;&#8364;&#8220;141.                                                                                   Chem. 380, 1143&#226;&#8364;&#8220;1150. <br /> 21.   Sikorski, R. S. &amp; Hieter, P. (1989) Genetics 122, 19&#226;&#8364;&#8220;27.                             39.   Klostermeier, D., Seidel, R. &amp; Reinstein, J. (1999) J. Mol. Biol. 287, 511&#226;&#8364;&#8220;525. <br />  <br />  <br />  <br />  <br /> 13844 &#229;&#8230;&#169; www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.1936150100                                                             <br /> </body></html>