<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Species-specific positive selection of the male-specific <br /> lethal complex that participates in dosage <br /> compensation in Drosophila <br />  <br /> In many taxa, males and females have unequal ratios of sex                     msl2, and msl3, maleless (mle) and males absent on the first (mof ) <br /> chromosomes <span id='am-2' about='protege:TO' typeof='owl:Thing'>to</span> autosomes, which has resulted in the invention of               (Fig. 1A), as well as two noncoding RNAs (roX1 and roX2). <br /> diverse mechanisms to equilibrate gene expression between the                  MSL1 and MSL2 play a central role in the assembly of the MSL <br /> sexes (dosage compensation). Failure to compensate for sex chro-               complex and targeting to the X chromosome (Fig. 1B). It is <br /> mosome dosage results in male lethality in Drosophila. In Dro-                 believed that this targeting of the MSL complex enables MOF to <br /> sophila, a male-specific lethal (MSL) complex of proteins and                  specifically acetylate lysine-16 on histone H4 tails, a histone <br /> noncoding RNAs binds to hundreds of sites on the single male X                 modification correlated with active transcription (8&#226;&#8364;&#8220;10). High- <br /> chromosome and up-regulates gene expression. Here we use                       resolution mapping of MSL-binding sites has revealed a strong <br /> population genetics of two closely related Drosophila species to               bias for the middle and 3&#226;&#172;&#732; ends of coding sequences, suggesting <br /> show that adaptive evolution has occurred in all five protein-                 that any transcriptional up-regulation may involve increased <br /> coding genes of the MSL complex. This positive selection is asym-              elongation efficiency by RNA polymerase (11&#226;&#8364;&#8220;14). Indeed, <br /> metric between closely related species, with a very strong signa-              experiments in Saccharomyces cerevisiae have shown that MOF <br /> ture apparent in Drosophila melanogaster but not in Drosophila                 recruitment (using a Gal4 DNA-binding domain fusion) results <br /> simulans. In particular, the MSL1 and MSL2 proteins have under-                in transcriptional up-regulation (5, 10, 13). However, it has been <br /> gone dramatic positive selection in D. melanogaster, in domains                suggested that the absence of the rest of the MSL complex in <br /> previously shown to be responsible for their specific targeting to             these experiments complicates the exact implication of this result <br /> the X chromosome. This signature of positive selection at an                   regarding X chromosome up-regulation in Drosophila (15). <br /> essential protein&#226;&#8364;&#8220;DNA interface of the complex is unexpected and                  Which DNA sequences target the MSL complex specifically to <br /> suggests that X chromosomal MSL-binding DNA segments may                       the X chromosome? There are &#226;&#172;&#8225;35&#226;&#8364;&#8220;40 high-affinity sites on the <br /> themselves be changing rapidly. This highly asymmetric, rapid                  X chromosome that are bound by this complex (16, 17). In total, <br /> evolution of the MSL genes further suggests that misregulated                  there are estimated to be 700 separable regions where the MSL <br /> dosage compensation may represent one of the underlying causes                 complex is bound as shown by chromatin immunoprecipitation <br /> of male hybrid inviability in Drosophila, wherein the fate of hybrid           experiments. These regions cover roughly 25% of the X chro- <br /> males depends on which species&#226;&#8364;&#8482; X chromosome is inherited.                     mosome and, presumably, include sites ranging in affinity (11, <br />                                                                                12, 18). These regions occur mostly in coding sequences of genes <br /> genetic conflict &#229;&#8230;&#169; McDonald&#226;&#8364;&#8220;Kreitman test &#229;&#8230;&#169; X chromosome &#229;&#8230;&#169;                     (transposable elements were not included in the arrays used in <br /> spiroplasmal &#229;&#8230;&#169; retrotransposons                                                the analyses) and may be enriched in GAGA motifs (11, 12, 19). <br />                                                                                There is some debate about the role of active transcription in <br />  <br /> C    hromosomal aneuploidy is highly deleterious; deletions larger <br />      than 3% of the genome and duplications larger than 10% are <br /> not tolerated in Drosophila (1), presumably because an imbalance <br />                                                                                attracting or maintaining the MSL complex (for review, see ref. <br />                                                                                13), but active transcription alone cannot explain the strong bias <br />                                                                                for binding to X chromosomal DNA. Computational sequence <br /> of expression levels of many genes is hard to accommodate in                   analysis can identify some &#226;&#8364;&#732;&#226;&#8364;&#732;higher-order&#226;&#8364;&#8482;&#226;&#8364;&#8482; features on X chro- <br /> stoichiometric complexes involving many different proteins (2). In             mosomal sequences that distinguish them <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> autosomal DNA, <br /> organisms with highly diverged sex chromosomes, there is fre-                  yet extensive efforts at identifying common sequence predictors <br /> quently a difference in number of sex chromosomes versus auto-                 of MSL-binding sites have yielded limited prediction power at <br /> somes in the heterogametic sex (XY or ZW). This difference                     best (12, 20). These findings have led to the suggestion that <br /> requires &#226;&#8364;&#732;&#226;&#8364;&#732;dosage compensation&#226;&#8364;&#8482;&#226;&#8364;&#8482; strategies to equilibrate expres-             degenerate and multiple weak signals may contribute to target- <br /> sion levels in both sexes. Recent evidence suggests that these                 ing (12, 19, 21). Arguably, comparative genomics methodology <br /> strategies operate at two levels. A primary mechanism is to increase           has limited applicability to this problem because MSL-binding <br /> gene expression of the single X chromosome by 2-fold in the                    sites have not been mapped in divergent Drosophila species or <br /> heterogametic sex, a strategy that appears to be universally con- <br /> served in animals (3, 4). However, different animal lineages have <br />                                                                                Author contributions: M.A.R., D.V., and J.J.B. contributed equally to this work; D.V. and <br /> adopted diverse, secondary strategies to equilibrate gene expression           H.S.M. designed research M.A.R., D.V., J.J.B., and H.S.M. performed research; M.A.R., D.V., <br /> in the two sexes (5). In mammals, this secondary modification                  J.J.B., and H.S.M. analyzed data; and M.A.R., D.V., J.J.B., and H.S.M. wrote the paper. <br /> involves the inactivation of one of the two female X chromosomes,              The authors declare no conflict of interest. <br /> whereas in Caenorhabditis elegans, it is achieved by 2-fold lower              Abbreviations: D&#226;&#8364;&#8220;M model, Dobzhansky&#226;&#8364;&#8220;Muller model; HKA test, Hudson&#226;&#8364;&#8220;Kreitman&#226;&#8364;&#8220; <br /> transcriptional output from both X chromosomes in hermaphro-                   Aguade test; MK test, McDonald&#226;&#8364;&#8220;Kreitman test; MLE, maleless; MOF, males absent on the <br /> dites. Flies adopt a different strategy; they double the transcrip-            first; MSL, male-specific lethal. <br /> tional output of the single male X chromosome in somatic cells (6,             Data deposition: The sequences reported in this paper have been deposited in the GenBank <br /> 7), which requires the targeting of a male-specific lethal complex             database (accession nos. EF630368 &#226;&#8364;&#8220;EF630425, EF634160 &#226;&#8364;&#8220;EF634184, EF653847&#226;&#8364;&#8220;EF653867, <br /> (MSL) to the X chromosome but not to autosomes in Drosophila                   and EF682039 &#226;&#8364;&#8220;EF682060). <br />                                                                                &#226;&#8364;&#161;Present <br /> males (5).                                                                                address: Department of Genetics, Stanford University, Stanford, CA 94305. <br />    In Drosophila melanogaster, the MSL complex consists of                     &#194;&#182;To   whom correspondence should be addressed. E-mail: hsmalik@fhcrc.org. <br /> proteins encoded by five genes: male-specific lethal genes, msl1,              &#194;&#169; 2007 by The National Academy of Sciences of the USA <br />  <br />  <br />  <br /> 15412&#226;&#8364;&#8220;15417 &#229;&#8230;&#169; PNAS &#229;&#8230;&#169; September 25, 2007 &#229;&#8230;&#169; vol. 104 &#229;&#8230;&#169; no. 39                                                        www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.0707445104 <br />  Fig. 1. MSL1 and MSL2 play a key role in assembly and targeting of the dosage compensation complex to the male X chromosome. (A) Five known protein <br /> components and two known RNAs comprising the MSL complex. The five MSL proteins are drawn to scale with known domains highlighted. MSL1 serves as a <br /> scaffold for the entire MSL complex. MSL1 binds to MSL2, and together they bind to the X chromosome. Amino acids 85&#226;&#8364;&#8220;186 of MSL1 are necessary and sufficient <br /> for binding to amino acids 1&#226;&#8364;&#8220;190 of MSL2. Together, amino acids 1&#226;&#8364;&#8220;265 of MSL1 and amino acids 1&#226;&#8364;&#8220;190 of MSL2 are sufficient for targeting to high-affinity <br /> binding sites on the male X chromosome (30, 33, 34). Targeting of MSL1 is abolished by deletion of amino acids 1&#226;&#8364;&#8220;26 (32). MSL1 also binds to other components <br /> of the MSL complex, including MSL3 [which contains a chromobarrel domain (61) that binds RNA (62)] and MOF [which contains a chromobarrel domain (61), <br /> a zinc finger, and an acetyltransferase domain with specific activity for histone H4 (10, 63)]. MLE encodes ATPase and RNA/DNA helicase activities (64). (B) <br /> Schematic model of the assembly of the MSL complex onto the male X chromosome that highlights the central scaffolding role of MSL1 and MSL2. <br />  <br />  <br />  <br /> even methodically in different D. melanogaster strains. In addi-                  tion acting on all five genes encoding protein components of the <br /> tion, genetic experiments have determined that any substantial                    MSL complex. This finding is highly unexpected because MSL <br /> segment of the X chromosome sequence appears to possess the                       function is essential for male viability. We further found that the <br /> ability to attract the MSL complex autonomously (22, 23).                         signature for rapid evolution is strikingly asymmetric, affecting <br /> Although autosomal genes inserted onto the X chromosome will                      D. melanogaster but (largely) not Drosophila simulans. Some of <br /> also frequently undergo dosage compensation, X chromosomal                        the strongest signatures of positive selection can be localized to <br /> sequences are superior in their ability to recruit the MSL                        the MSL domains responsible for X chromosomal targeting, <br /> complex compared with autosomal sequences despite varying in                      suggesting that MSL-recruiting DNA segments may also have <br />  <br />  <br />  <br />  <br />                                                                                                                                                                 GENETICS <br /> their affinity for MSL recruitment (24). The questions remain:                    evolved rapidly in the D. melanogaster lineage, where all of the <br /> What is recruiting the MSL complex specifically to the male X,                    mapping studies have been done. Together, these findings also <br /> and why are these sequences so difficult to define?                               suggest that incompatibilities between MSL proteins and MSL- <br />    We hypothesized that a MSL-binding site consensus is hard to                   recruiting DNA elements on the D. melanogaster X chromosome <br /> define because these motifs might be evolutionarily labile. A                     may represent one of the underlying causes of male hybrid <br /> selective pressure that prevented the stable coevolution of MSL                   inviability in Drosophila. <br /> proteins and DNA might have prevented the fixation of an <br /> optimal DNA sequence that could recruit the MSL complex.                          Results <br /> Such a scenario would preclude the identification of a consensus                  We sequenced all five protein-coding MSL complex genes from <br /> MSL-binding signature in the DNA. We explored this possibility                    multiple strains of D. melanogaster and D. simulans, two species <br /> by investigating the selective pressures shaping genes encoding                   that diverged 2.5 million years ago. Summary statistics for <br /> MSL proteins as a &#226;&#8364;&#732;&#226;&#8364;&#732;surrogate&#226;&#8364;&#8482;&#226;&#8364;&#8482; to studying the MSL-binding sites                 polymorphisms seen in these genes are presented in Table 1. <br /> themselves directly. We found strong evidence of positive selec-                  From these statistics, there is no evidence for a <span id='am-81' about='xsp:pattern' typeof='owl:Thing'>pattern</span> of rare, <br />  <br />  <br />             Table 1. Summary statistics for polymorphisms in MSL complex genes <br />                                                            Codon usage         Nucleotide                        Fu and        No. of         No. of <br />             Gene                        Location              (ENC)             diversity       Tajima&#226;&#8364;&#8482;s D        Li F*        strains         bp <br />  <br />             D. melanogaster <br />   <br />  <br />  <br />                The Fu and Li F statistic was calculated by using either D. simulans or D. melanogaster as an outgroup species. &#226;&#8364;&#161;, P &#226;&#172;&#65533; 0.05. <br />             &#226;&#8364;&#160;Seventy-five nucleotides were excluded from analysis because of one region between D. melanogaster and D. simulans with ambiguity <br />  <br />              in the alignment. <br />             &#194;&#167;Eighteen   codons were excluded from analysis due to two polymorphic indels of 12 and 42 nucleotides within D. simulans. <br />  <br />                                  No. of                                     Interspecies, total <br />                               segregating      Total no.      Sample              no. of <br />                      Gene         sites         of sites       size&#226;&#8364;&#160;           differences*       msl2       msl3       mle      mof <br />  <br />  <br />                         P values for all pairwise comparisons between the D. melanogaster MSL complex genes are shown with <br />                      significant deviations from neutral expectations highlighted in bold. <br />                      *D. simulans was used for the interspecies comparison. <br />                      &#226;&#8364;&#160;No. of D. melanogaster strains sequenced. <br />  <br />  <br />  <br />  <br /> singleton polymorphisms that might suggest recovery of poly-                    MSL genes shows a significantly discordant polymorphism pat- <br /> morphisms after a recent adaptive sweep in most of the MSL                      tern by the HKA test in D. simulans. <br /> genes. However, we see a Tajima&#226;&#8364;&#8482;s D value of &#226;&#171;&#186;1.55 and a Fu and                   We also tested for positive selection by using the McDonald&#226;&#8364;&#8220; <br /> Li value of &#226;&#171;&#186;2.58 (P &#226;&#172;&#65533; 0.05) in the mle gene of D. melanogaster,                Kreitman (MK) test (Table 3). This test evaluates whether an <br /> strongly supporting the possibility that a recent sweep has                     excess number of replacement (amino acid altering) changes <br /> affected the polymorphism spectrum (Table 1) (25, 26). To                       versus synonymous changes had been fixed between the two <br /> investigate this possibility further, we compared all of the MSL                species compared with replacement and synonymous polymor- <br /> genes in D. melanogaster by using a Hudson&#226;&#8364;&#8220;Kreitman&#226;&#8364;&#8220;Aguade                      phisms within each species (28). Under this test, we find that four <br /> (HKA) test, which examines whether interspecies divergence and                  of five MSL genes, msl1, msl2, msl3, mof, but not mle, show <br /> intraspecies polymorphisms are correlated, as would be pre-                     robust signatures of positive selection when compared across the <br /> dicted under neutrality (27). We find a significant discordance in              whole gene. This finding is highly unusual for essential genes <br /> the polymorphism patterns between mle and three other MSL                       because a high degree of evolutionary constraint is expected to <br /> genes, msl2, msl3, and mof (Table 2). Thus, both the Fu and Li                  act to preserve function. Using Drosophila yakuba as an outgroup <br /> F* statistic and the HKA test results strongly implicate a recent               species, we can ascertain which lineage has been affected by <br /> adaptive sweep in the mle gene in D. melanogaster. None of the                  positive selection by assigning the fixed changes to either the D. <br />  <br />  <br />                      Table 3. MK test for positive selection on MSL complex genes <br />                      MSL complex                   Observed      Observed          Observed       Observed <br />                      protein-encoding gene            Sp           Rp                 Sf             Rf         G value        P value <br />  <br />                      msl1 <br />                       Pooled                           54              41             56            78               5.025     0.025 <br />                       D. melanogaster                  18              10             24            53               9.097     0.0026 <br />                       D. simulans                      36              32             23            22               0.036     0.85 <br />                      msl2* <br />                       Pooled                           83              40             51            63           12.47         0.00041 <br />                       D. melanogaster                  36              22             24            41            7.743        0.0054 <br />                       D. simulans                      47              18             22            18            3.201        0.074 <br />                      msl3 <br />                       Pooled                           49              10             22            19            9.913        0.00164 <br />                       D. melanogaster                  24               3             11            15           13.23         0.00028 <br />                       D. simulans                      24               6              7             3            0.379        0.54 <br />                      mof <br />                       Pooled                           84              19             48            37           13.97         0.00019 <br />                       D. melanogaster                  30               7             25            23            7.86         0.0051 <br />                       D. simulans                      55              12             19            13            5.531        0.012 <br />                      mle <br />                       Pooled                           50              35             75            48               0.096     0.76 <br />                       D. melanogaster                  14              16             37            24               1.562     0.21 <br />                       D. simulans                      36              19             29            21               0.608     0.44 <br />  <br />                         We compared the ratio of replacement and synonymous changes that were polymorphic within the species <br />                      (Rp:Sp) with the ratio of replacement and synonymous changes that were found fixed between the species (Rf:Sf). <br />                      If no alteration in selective regimes occurred during evolution, we expect these ratios to be statistically <br />                      indistinguishable. However, an excess of fixed replacement changes is a clear indication of positive selection. <br />                      Pooled changes refer to all the polymorphism and fixed changes using both lineages. However, by using an <br />                      outgroup species (D. yakuba), we can also make the same comparison, specific either to the D. melanogaster or <br />                      D. simulans lineage. <br />                      *A region of 75 nucleotides was excluded from all msl2 analysis because of ambiguity in the alignment between <br />                        D. simulans and D. melanogaster (the region corresponds to nucleotides 904 &#226;&#8364;&#8220;978 relative to D. melanogaster). <br />  <br />  <br /> 15414 &#229;&#8230;&#169; www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.0707445104                                                                                     Rodriguez et al. <br />  melanogaster or D. simulans lineages. We find that there is robust      through females. Recent studies have directly implicated the <br /> evidence for positive selection acting on the D. melanogaster           presence of a functional MSL complex as a requirement for this <br /> lineage for four of five MSL genes (all except mle) by using the        male-specific killing by S. poulsonii (35). Under such a &#226;&#8364;&#732;&#226;&#8364;&#732;genetic <br /> MK test. Interestingly, mle is the only gene that appears to have       conflict&#226;&#8364;&#8482;&#226;&#8364;&#8482; scenario, one could imagine bacterial proteins evolving <br /> undergone a recent sweep in the D. melanogaster species (Tables         to &#226;&#8364;&#732;&#226;&#8364;&#732;detect&#226;&#8364;&#8482;&#226;&#8364;&#8482; MSL components through direct binding, whereas <br /> 1 and 2) and has likely reduced our ability to detect positive          MSL components could be under strong selective pressure to <br /> selection by using the MK test on this gene. In contrast, we find       evolve away from this recognition. This &#226;&#8364;&#732;&#226;&#8364;&#732;arms race&#226;&#8364;&#8482;&#226;&#8364;&#8482; would result <br /> that only one of five genes (mof ) has been subjected to positive       in changes in one or all of the MSL components because fixation <br /> selection along the D. simulans lineage; even in this case, the         of slightly deleterious mutations in the MSL complex would be <br /> intensity of positive selection is weaker than in the D. melano-        preferred over bacteria-induced male lethality. <br /> gaster lineage. Also, of the five MSL genes analyzed here, mof is          A second possible source driving positive selection of the MSL <br /> the only one that is also expressed robustly in females (8),            complex could be genetic conflict with retrotransposable ele- <br /> although the functional significance of MOF function in females         ments. It has been suggested previously that LINE1 non-LTR <br /> is still unclear.                                                       retrotransposons may provide &#226;&#8364;&#732;&#226;&#8364;&#732;landing sites&#226;&#8364;&#8482;&#226;&#8364;&#8482; for dosage com- <br />    The msl1 and msl2 genes are key players in targeting the MSL         pensation (X inactivation) in mammalian X chromosomes (36). <br /> complex to the male X chromosome. Mutational analyses of each           Under this second possibility, MSL binding to retrotransposons <br /> MSL gene have shown that MSL1 and MSL2 are capable of                   may be an important defense against them (37). Repeated <br /> targeting &#226;&#8364;&#732;&#226;&#8364;&#732;high-affinity&#226;&#8364;&#8482;&#226;&#8364;&#8482; sites, independent of other known MSL       specialization of the MSL complex to recognize retroelements <br /> components (29&#226;&#8364;&#8220;31). Targeting requires an interaction between           may also result in some of these elements becoming preferred <br /> the N-terminal domains of MSL1 and MSL2 (Fig. 1) and is                 landing sites for the MSL complex, effectively altering the <br /> abolished by deletion of the first 26 amino acids of MSL1 (30,          landscape of MSL binding to the X chromosome. It is important <br /> 32&#226;&#8364;&#8220;34). Because these targeting domains have been roughly               to note, however, that retrotransposons primarily mobilize in the <br /> mapped, we next addressed whether positive selection had                germ line, whereas the MSL proteins discussed here are acting <br /> shaped these regions in particular, focusing our analyses only on       predominantly in somatic tissues. <br /> the D. melanogaster lineage. If we parse the fixed or polymorphic          Both of these conflict scenarios fit well with our finding of <br /> changes that have taken place in MSL1 in the D. melanogaster            highly asymmetric positive selection because either the male- <br /> lineage, we find that the N-terminal domain (amino acids 1&#226;&#8364;&#8220;265),        killing bacteria or retrotransposons may provide a lineage- <br /> which is necessary and sufficient for both X chromosomal                specific selective pressure, not affecting even closely related <br /> targeting and for interactions with MSL2 (30, 32&#226;&#8364;&#8220;34), bears all         species. It has been suggested (38) that because the relative <br /> of the hallmarks of positive selection (Rf:Sf::Rp:Sp &#226;&#171;&#189; 30:6::5:8,       stoichiometries of regulator proteins are so intricately linked to <br /> G value 8.406, P &#226;&#172;&#65533; 0.005). In contrast, the remainder of the            each other, rapid evolution of any one component driven by <br /> MSL1 protein [amino acids 266-1039, which includes interaction          genetic conflict under any model [by male-killing bacteria or <br />  <br />  <br />  <br />  <br />                                                                                                                                               GENETICS <br /> interfaces with both MOF and MSL3 (33)] shows no evidence for           retrotransposons or even by &#226;&#8364;&#732;&#226;&#8364;&#732;centromere drive&#226;&#8364;&#8482;&#226;&#8364;&#8482; (39)] could <br /> positive selection (Rf:Sf::Rp:Sp &#226;&#171;&#189; 23:18::5:10, G value 2.229,          inevitably trigger a &#226;&#8364;&#732;&#226;&#8364;&#732;ripple effect of adaptation&#226;&#8364;&#8482;&#226;&#8364;&#8482; in other MSL <br /> P &#226;&#172;&#381; 0.15).                                                              genes. Each such alteration would trigger a coevolutionary <br />    Similarly, when we parse our fixed and polymorphic changes           episode in which other target genes and regulator proteins would <br /> for the MSL2 protein, we find that the domain required for              adjust to the changed landscape to ensure optimal function (38). <br /> binding MSL1 and thereby, targeting to DNA (amino acids                 Although it is unlikely that a single ripple event can explain the <br /> 1&#226;&#8364;&#8220;190) (34) evolves under positive selection in D. melanogaster         pervasive positive selection we have seen in multiple domains of <br /> (Rf:Sf::Rp:Sp &#226;&#171;&#189; 9:9::1:10, P &#226;&#172;&#65533; 0.025). In contrast to MSL1,             all MSL proteins, selection on one member of a complex might <br /> however, even the remainder of the MSL2 protein, which                  bring along changes in other members at any number of domains <br /> consists of a central coiled-coil domain and a C-terminus of as         if that produces an eventual fitness advantage by restoring <br /> yet-undefined function (Fig. 1), shows a robust signature of            optimal function. <br /> positive selection (Rf:Sf::Rp:Sp &#226;&#171;&#189; 32:15::21:26, G value 5.203,            One possible consequence of such rapid evolution is that MSL <br /> P &#226;&#172;&#65533; 0.025).                                                             components may quickly become incompatible in different <br />    Thus, the N-terminal domains of both MSL1 and MSL2 are               species. Such incompatibilities are thought to occur under a <br /> hot spots for positive selection, despite the fact that these           Dobzhansky&#226;&#8364;&#8220;Muller (D&#226;&#8364;&#8220;M) model wherein independently oc- <br /> domains are essential for the MSL1&#226;&#8364;&#8220;MSL2 interaction both with           curring allelic changes in different interacting components could <br /> each other and to binding sites on the X chromosome. Our                manifest as negative epistasis in resulting hybrids (the simplest <br /> findings support the idea that the rules that guide X chromo-           two-locus form is schematized in Fig. 2A). The possibility of this <br /> somal sequence-directed DNA binding have been evolutionarily            negative epistasis is greatly increased with accelerated change; <br /> labile, at least in the D. melanogaster lineage.                        genes responsible for postzygotic isolation are frequently subject <br />                                                                         to positive selection (40&#226;&#8364;&#8220;42). Rapid evolution can also result in <br /> Discussion                                                              asymmetric postzygotic isolation (Fig. 2B). There are two formal <br /> Rapid evolution of the MSL complex is at odds with the                  possibilities for D&#226;&#8364;&#8220;M incompatibilities arising from the positive <br /> expectation that proteins so essential for male viability ought to      selection of the genes encoding MSL1 and MSL2. The first <br /> be highly constrained and under purifying selection. Moreover,          possibility is that the two components could represent the MSL1 <br /> interactions between MSL proteins and their cognate DNA-                and MSL2 proteins themselves (Fig. 2C) because it is the <br /> binding sites should be especially well constrained because any         protein&#226;&#8364;&#8220;protein interaction surface between these two proteins <br /> mutations in one MSL component would have to be accommo-                that is a hot spot for positive selection. However, recognizing <br /> dated in the other MSL proteins and in the DNA target sites to          that these domains also determine the exclusive targeting to the <br /> retain the essential function of the complex. Indeed, it is             X chromosome, the D&#226;&#8364;&#8220;M incompatibility could exist between <br /> reasonable to speculate that the selective force that drove the         the MSL1&#226;&#8364;&#8220;MSL2 proteins, and MSL-targeting sites on the X <br /> rapid evolution must have imposed a stringent selective cost,           chromosome (Fig. 2D). Either of these incompatibilities would <br /> which would drive changes in the whole MSL complex.                     lead to compromised MSL function and thereby male inviability <br />    Male-killing bacteria provide an example of just such a              in interspecies hybrids. <br /> selective cost. For instance, Spiroplasma poulsonii specifically kill      Intriguingly, male hybrids have different outcomes in a cross <br /> male D. melanogaster flies, as they are transmitted exclusively         between D. melanogaster and D. simulans, depending on which <br />  <br /> Rodriguez et al.                                                                      PNAS &#229;&#8230;&#169; September 25, 2007 &#229;&#8230;&#169; vol. 104 &#229;&#8230;&#169; no. 39 &#229;&#8230;&#169; 15415 <br />                                                                                      X chromosome is inherited in the hybrid males (Fig. 2E). Hybrid <br />                                                                                     males that inherit the D. simulans X chromosome are viable. <br />                                                                                     However, hybrid males that inherit the D. melanogaster X <br />                                                                                     chromosome suffer larval lethality, dying at a developmental <br />                                                                                     stage similar to that of pure-species D. melanogaster males that <br />                                                                                     have mutated MSL components (43, 44). Because the msl1 and <br />                                                                                     msl2 genes reside on autosomes (Table 1), both hybrids should <br />                                                                                     acquire both D. simulans and D. melanogaster versions of both <br />                                                                                     genes. The viability of the D. simulans X-bearing hybrid males <br />                                                                                     implies that an incompatibility between MSL1 and MSL2 pro- <br />                                                                                     teins (Fig. 2B) is not likely to be causal for hybrid male <br />                                                                                     inviability. Instead, our findings suggest that although the <br />                                                                                     MSL1&#226;&#8364;&#8220;MSL2 interaction is not severely affected in hybrids, <br />                                                                                     hybrid inviability may result from negative epistasis between the <br />                                                                                     D. melanogaster X chromosome and D. simulans MSL compo- <br />                                                                                     nents (Fig. 2C). Because most of the MSL positive selection has <br />                                                                                     occurred along the D. melanogaster lineage, D. simulans MSL <br />                                                                                     proteins may not have the requisite changes for the correct <br />                                                                                     targeting to the D. melanogaster X. In support of this idea, a <br />                                                                                     recent study has found that MSL components do not correctly <br />                                                                                     target in D. melanogaster X-bearing hybrid males (44). In con- <br />                                                                                     trast, MSL localization and function are known to be normal in <br />                                                                                     D. simulans X-bearing hybrid males (44, 45). <br />                                                                                        The genetic dissection of the determinants of postzygotic isola- <br />                                                                                     tion in Drosophila has been greatly aided by the discovery of hybrid <br />                                                                                     rescue genes, so called because mutations in these genes restore <br />                                                                                     hybrid viability. (MSL components are not expected to be hybrid <br />                                                                                     rescue genes because compromised MSL function would be <br />                                                                                     strongly deleterious.) Hybrid males that inherit a D. melanogaster X <br />                                                                                     chromosome and are otherwise inviable, can be rescued by a <br />                                                                                     naturally occurring <span id='am-3' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-4' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-5' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span>mutation</span> in the Lhr (lethal hybrid rescue) gene <br />                                                                                     (43). A recent study firmly establishes that the Lhr gene has a <br />                                                                                     heterochromatic localization (41). This study is especially notewor- <br />                                                                                     thy because several heterochromatin proteins and remodeling <br />                                                                                     factors have been directly implicated in transcriptional regulation of <br />                                                                                     the male X chromosome (46&#226;&#8364;&#8220;49). Indeed, it is possible that other <br />                                                                                     hybrid incompatibility factors mapped in this cross may arise from <br />                                                                                     defects in dosage compensation. For instance, the intriguing finding <br />                                                                                     that nuclear pore complex proteins cause hybrid male lethality (40) <br />                                                                                     could be viewed in light of recent findings connecting them to the <br />                                                                                     MSL proteins (50). It is likely that a balance of &#226;&#8364;&#732;&#226;&#8364;&#732;negative&#226;&#8364;&#8482;&#226;&#8364;&#8482; dosage <br />                                                                                     regulators and &#226;&#8364;&#732;&#226;&#8364;&#732;positive&#226;&#8364;&#8482;&#226;&#8364;&#8482; MSL proteins is required to achieve the <br />                                                                                     correct level of transcription in Drosophila hybrid males (51). Such <br />                                                                                     nonadditive expression phenotypes have been observed in hybrids <br />                                                                                     of D. melanogaster and D. simulans (52) with an apparent over- <br />                                                                                     abundance of misregulated genes on the X chromosome (53). <br />                                                                                        The generality of Haldane&#226;&#8364;&#8482;s rule, wherein it is more likely that <br />                                                                                     the heterogametic sex will be inviable, has led to several attrac- <br />                                                                                     tive theories about how hybrid inviability could represent a <br />                                                                                     breakdown in dosage compensation in hybrids (54, 55). Indeed, <br /> Fig. 2. Positive selection of the MSL complex and X chromosomal MSL-                it has been clear for quite some time that the X chromosome <br /> binding sites might result in hybrid incompatibility. (A) Two-locus D&#226;&#8364;&#8220;M model       plays a disproportionate role in hybrid incompatibilities, referred <br /> for hybrid incompatibility between closely related species. Loci A and B            to as the &#226;&#8364;&#732;&#226;&#8364;&#732;large-X&#226;&#8364;&#8482;&#226;&#8364;&#8482; effect (56). Until recently, it has not been <br /> interact in the ancestral species. During (reproductive or recombinational)         clear that D&#226;&#8364;&#8220;M incompatibilities could arise in such systems <br /> isolation, there is a neutral fixation of the a and b alleles in the two popula-    because they are so essential for function and thereby predicted <br /> tions, which is tolerated because the new alleles (a and b) are still compatible <br /> with the old alleles (B and A, respectively). However, this fixation results in <br /> hybrid incompatibility because of negative epistatic interactions between the <br /> new a and b alleles. This model can explain the onset of incompatibilities even     Positive selection (bold arrows) in D. melanogaster has resulted in rapid <br /> under neutral evolution (56). (B) In the case of positive selection (bold arrows)   evolution of the MSL1&#226;&#8364;&#8220;MSL2 genes and (we infer) the X chromosomal MSL- <br /> driving the interaction of the A and B loci, only one lineage may evolve to the     binding sites. In male hybrids, D. simulans MSL1 and MSL2 are unable to <br /> new a and b alleles, resulting in incompatibility with the other lineage, which     recognize &#226;&#8364;&#732;&#226;&#8364;&#732;newly evolved&#226;&#8364;&#8482;&#226;&#8364;&#8482; MSL-binding sites on the D. melanogaster X <br /> still preserves the ancestral A and B alleles. (C) MSL1 and MSL2 could represent    chromosome resulting in mislocalization of the MSL complex in hybrids with <br /> the A and B loci in the D&#226;&#8364;&#8220;M model, with the positive selection (bold arrows) at     a D. melanogaster X chromosome (44). However, hybrids with a D. simulans X <br /> their interaction interface resulting in hybrid inviability. Under this model,      chromosome localize the MSL complex normally (45) because the D. melano- <br /> male hybrids containing either the D. melanogaster or D. simulans X chro-           gaster MSL1 and MSL2 proteins retain an ancestral DNA-binding ability. (E) <br /> mosomes would be inviable because the protein composition is expected to be         Known male hybrid incompatibility in D. melanogaster crosses to D. simulans. <br /> the same in both cases. (D) Model for hybrid incompatibility with MSL1&#226;&#8364;&#8220;MSL2         Male inviability occurs when a D. melanogaster X chromosome is combined <br /> and the X chromosomal MSL-binding sites, as A and B loci, respectively.             with a hybrid autosomal background (43). <br />  <br />  <br />  <br />  to evolve under a high degree of constraint. However, our                         High Fidelity (Invitrogen, Carlsbad, CA) and primers based on <br /> present analysis on MSL complex genes suggests that such genes                    D. melanogaster genomic sequence. Most PCR products were <br /> can and do evolve rapidly, which implies that even genes that                     sequenced directly except in the case of mle for some D. simulans <br /> participate in essential chromatin functions such as dosage                       strains. When direct sequencing of PCR products was not <br /> compensation (as described here), chromosome segregation                          possible because of a low yield of PCR products, these products <br /> (57), and defining origins of DNA replication (58) are not                        were cloned by using Topo-TA vectors (Invitrogen), and se- <br /> immune from being called to participate in genetic conflict and                   quencing was done on at least three separate colonies. ClustalX <br /> adaptation. Indeed, D&#226;&#8364;&#8220;M incompatibilities arising because of                      (59) was used to obtain multiple alignments, which were subse- <br /> rapid evolution of these essential protein&#226;&#8364;&#8220;DNA interactions are                   quently hand-edited with the amino acid sequence as a guide. <br /> more likely to result in hybrid inviability and sterility rather than             The DNASP software package (60) was used to perform several <br /> incompatibilities between two proteins that carry out a nones-                    tests for positive selection, including the Tajima&#226;&#8364;&#8482;s D (25) and Fu <br /> sential role in either species.                                                   and Li tests (26), as well as the MK (28) and the HKA (27) tests. <br />  <br /> Methods                                                                         <br /> All Drosophila strains were obtained from the Species Stock <br />                                                                                <br /> Center (Tucson, AZ) except for the African isofemale lines that                   <br /> were a gift from Daven Presgraves (University of Rochester,                       <br /> Rochester, NY). Genomic DNA was prepared as described                              <br /> previously (57). Genes were amplified by using PCR Supermix                     <br /> </body></html>