<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>2894&#226;&#8364;&#8220;2909 Nucleic Acids Research, 2009, Vol. 37, No. 9                                                           Published online 13 March 2009 <br /> doi:10.1093/nar/gkp152 <br />  <br />  <br /> The universal YrdC/Sua5 family required the <br /> formation threonylcarbamoyladenosine tRNA <br /> Basma El Yacoubi1, Benjamin Lyons1, Yulien Cruz1, Robert Reddy2, Brian Nordin2, <br /> Fabio Agnelli2, James R. Williamson2, Paul Schimmel2, Manal A. Swairjo2 and <br /> Vale&#194;&#180;rie Cre&#194;&#180;cy-Lagard1,* <br /> 1 <br />  Department Microbiology Department Microbiology Cell Science, University Florida, P.O. Box <br /> 110700, Gainesville, FL 32611-0700 2Department Chemistry Department Molecular Biology, <br /> The Skaggs Institute Chemical Biology, The Scripps Research Institute, La Jolla, CA 92037, USA <br />  <br /> Received November 4, 2008; Revised February 3, 2009; Accepted February 24, 2009 <br />  <br />  <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span>                                                                        center proposed form nucleic-acid-binding <br />                                               6                                 site (2). Binding double-stranded RNA (including <br /> Threonylcarbamoyladenosine (t A) universal                                 tRNA) experimentally veri&#239;&#172;?ed E. coli YrdC <br /> modification position 37 ANN decoding                               protein (YrdCEc) (2). The study COG0009 encoding <br /> tRNAs, imparts unique structure                                  gene Saccharomyces cerevisiae, SUA5, preliminarily <br /> anticodon loop enhancing binding ribosomes                               addressed function gene family. A mutant <br /> in vitro. Using combination bioinformatic,                                 allele, sua5-1 (Ser107 ! Phe replacement) identi&#239;&#172;?ed <br /> genetic, structural biochemical approaches,                                 suppressor translation initiation defect the <br /> the universal protein family YrdC/Sua5 (COG0009)                                leader region cyc1-1019 allele iso-1- <br /> was shown involved biosynthesis                                 cytochrome c encoding gene CYC1. In strain carrying <br /> this hypermodified <span id='am-396' about='xsp:base' typeof='owl:Thing'>base</span>. Contradictory reports                               cyc1-1019 allele, initiation shown occur an <br /> the essentiality yrdC wild-type gene                             aberrant up-stream frame ATG (due single <br />                                                                                 base-pair substitution) resulted expression  2% <br /> Escherichia coli SUA5 wild-type gene of <br />                                                                                 normal Cyc1 protein amount. Interestingly, sup- <br /> Saccharomyces cerevisiae led reconstruct                                  pression did occur transcriptional level as <br /> null alleles genes prove yrdC                              observed sua suppressors identi&#239;&#172;?ed the <br /> essential E. coli, SUA5 dispensable                            screen (4). A strain carrying sua5 null <span id='am-162' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-163' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-239' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span>mutation</span> <br /> yeast results severe growth phenotypes.                                  sua5::LEU2 displayed pleiotropic phenotypes that <br /> Structural biochemical analyses revealed                               included; slow growth, cytochrome a.a3 de&#239;&#172;?ciency, <br /> the E. coli YrdC protein binds ATP preferentially                           decreased cytochrome b levels inability grow on <br /> binds RNAThr lacking t6A modification. This                            lactate glycerol (4). The Sua5 protein implicated <br /> work lays foundation elucidating function                           post-transcriptional modulation gene expression but <br /> of protein family sequenced                                    biochemical function remained uncharacterized (4). In <br /> genome date understanding role t6A                                humans, YrdC ortholog (YrdCH) identi&#239;&#172;?ed a <br />                                                                                 retinoblastoma binding protein 10 (RBBP10) interactor <br /> in vivo. <br />                                                                                 possible role translation proposed based on <br />                                                                                 high ubiquitous expression pro&#239;&#172;?le tissues <br />                                                                                 including high levels protein synthesis <br /> INTRODUCTION                                                                    occurs (5). The YrdCH ortholog identi&#239;&#172;?ed a <br /> Found sequenced genomes date, YrdC                                new ischemia/reperfusion-inducible protein (IRIP) that <br /> (COG0009) family ranked 10 con-                             indirectly regulates activity variety trans- <br /> served hypothetical proteins considered high pri-                      porters probably interaction RSC1A1, a <br /> ority targets experimental study (1). The crystal                           modulator membrane transport unclear mode of <br /> structures Escherichia coli COG0009 family mem-                          action (6). <br /> bers YrdC YciO (2,3) reveal a/b twisted open-                               In E. coli, small deletion 12 nucleotides including <br /> sheet structure positively charged cleft                          initiation codon yrdC ( yrdC) <span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span> <br />  <br /> *To correspondence addressed. Tel: +1 352 392 9416; Fax: +1 352 392 5922; Email: vcrecy@u&#239;&#172;&#8218;.edu <br /> Present addresses: <br /> Brian Nordin, ActivX Biosciences, Inc. 11025 North Torrey Pines Road, La Jolla, CA 92037, USA. <br /> Manal A. Swairjo, Burnham Institute Medical Research, 10901 North Torrey Pines Road, La Jolla, CA 92037, USA. <br />  <br /> &#195;&#376; 2009 The Author(s) <br /> This Open Access article distributed terms Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/ <br /> by-nc/2.0/uk/) permits unrestricted non-commercial use, distribution, reproduction medium, provided original work properly cited. <br />                                                                           Nucleic Acids Research, 2009, Vol. 37, No. 9 2895 <br />  <br />  <br /> suppress temperature-sensitive <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> caused           possible, building phylogenetic trees using the <br /> prfA1 (an allele prfA gene encoding release factor        ClustalW tool (25) integrated SEED deriving spe- <br /> 1). These mutations inseparable           ci&#239;&#172;?c protein motifs (http://espript.ibcp.fr/ESPript/ <br /> function ribosome maturation suggested yrdC           ESPript/). <br /> based observation double mutant dis- <br /> played slightly higher amounts 17S rRNA, immature          Strains growth conditions <br /> form 16S rRNA (7,8).                                          All E. coli S. cerevisiae strains used study are <br />    In summary, members COG009 family              listed Table 1. Bacteria routinely grown LB <br /> act cellular regulators, pleiotropic e&#239;&#172;&#8364;ects        medium (BD Diagnostics Systems) 37  C unless other- <br /> linked translation vary experimental            wise stated. Bacterial growth media solidi&#239;&#172;?ed 15 <br /> system. Overall role molecular level           g/l agar (BD Diagnostics Systems) preparation of <br /> YrdC/Sua5 proteins remained unclear. The essentiality            plates. Transformation P1 transductions per- <br /> of gene family remained un-resolved issue,          formed following standard procedures (26). The sensitivity <br /> as appeared vary organism considered          P1 phage recipient strains used veri&#239;&#172;?ed. <br /> the approach utilized construct mutations. In E. coli,     Ampicillin (Amp, 100 mg/ml), Kanamycin (Km, 50 mg/ml), <br /> yrdC deletion mutant obtained systematic              L-Arabinose (Ara, 0.02&#194;&#8212;0.2%), used appro- <br /> single-gene <span id='am-240' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-241' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-242' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-244' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-250' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-256' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-258' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-272' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-279' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-284' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-292' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-297' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-302' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-303' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-305' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-308' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span>knockout</span> mutant collection,              priate. Bacterial chromosomal DNA prepared <br /> E. coli homolog yciO successfully deleted             using Qiagen kit DNeasy&#195;&#8226; Tissue Kit yeast chro- <br /> same study (9). Also, yrdC mutant isolated         mosome DNA prepared using Blood &amp; Tissue Kit <br /> a prfA1 background, mutations dis-          (Qiagen, Valencia, CA). <br /> sociable, suggesting gene yrdC essen-           Yeast strains routinely grown YPD (DIFCO <br /> tial wild-type E. coli (8). However, con&#239;&#172;&#8218;icting results       Laboratories) 30  C unless stated. Synthetic <br /> have reported insertion Tn5-araPBAD            minimal, Agar, SD base SD base Gal/ <br /> construct &#239;&#172;?rst 15 bp yrdCEc did lead        Raf dropout (-uracil, -ura; -leucine, -leu; <br /> lethality rich medium supplemented glucose (10).         -histidine, -his) purchased Clontech (Palo Alto, <br />    In yeast, isolated suppressor,    CA) prepared recommended manufacturer. <br /> sua5::LEU2 allele dissected away original           Glucose (Glu, 2%), Glycerol (Gly, 4%), 5-&#239;&#172;&#8218;uoro-orotic <br /> mutation, deletion gene successfully              acid (5-FOA, 0.1%) G 418 (300 mg/ml) used <br /> achieved homozygous backgrounds suggesting               appropriate. Transformations carried as <br /> the gene essential (4). However, yeast sys-       described pYES-DEST52 manual (Invitrogen, <br /> tematic gene deletion collection, single SUA5 copy       Carlsbad, CA). <br /> be deleted diploid background BY4743, no <br /> deletion obtained haploid background                  Cloning strains constructions <br /> BY4741 (11). <br />    N6-threonylcarbamoyl adenosine (t6A) anticodon-         All plasmids constructed listed Table 1. Sequences <br /> loop modi&#239;&#172;?cation position 37 tRNAs decod-            primers used given Table 2. The E. coli wild- <br /> ing ANN codons. Although considerable biochemical            type genes yrdCEc (NP_417741) yciOEc (NP_415783) <br /> biophysical information exists function           ampli&#239;&#172;?ed PCR genomic DNA prepared <br /> hypermodi&#239;&#172;?ed base (12&#194;&#8212;21), t6A biosynthesis pathway          strain E. coli K12 MG1655. In similar fashion <br /> has partially biochemically characterized          wild-type gene ywlCBs (NP_391576) ampli&#239;&#172;?ed <br /> shown ATP-dependent process requiring threo-            Bacillus subtilis 168 genomic DNA wild- <br /> nine carbonate genes involved        type yeast gene SUA5 (NP_011346) ampli&#239;&#172;?ed <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> its biosynthesis identi&#239;&#172;?ed (22).                       strain BY4743 genomic DNA. The wild-type gene <br />    In present study combining compar-           yrdCMm (NP 987306) ampli&#239;&#172;?ed Methanococcus <br /> ative genomic approach experimental validation         maripaludis S2 genomic DNA. For subcloning in <br /> the YrdC/Sua5 family involved t6A biosynthesis. We         pBAD24, forward primers (ol1) contained NcoI <br /> also investigated essentiality gene family    sequence reverse primers (ol2) contained a <br /> biological systems, E. coli yeast. In light role   HindIII sequence, reverse primer amplify <br /> this gene family formation critically impor-       SUA5 contained PstI site, BADEcyrdCol1/ <br /> tant tRNA modi&#239;&#172;?cation t6A37, hypotheses explain          BADEcyrdCol2; BADEcyciOol1/BADEcyciOol2; BADB <br /> of pleiotropic e&#239;&#172;&#8364;ects observed previous studies       sywlCol1/BADBsywlCol2;           BADMmyrdCol1/BADM <br /> presented.                                                       myrdCol2; BADsua5ol1/BADsua5ol2. PCR fragments <br />                                                                  &#239;&#172;?rst inserted pCR2.1 TOPO (Invitrogen) <br />                                                                  transformed Top 10 cells (Invitrogen) <br /> MATERIALS AND METHODS                                            according manufacturer protocol. Sequences were <br />                                                                  veri&#239;&#172;?ed subcloning appropriate sites of <br /> Bioinformatics <br />                                                                  pBAD24 (27). <br /> Analysis COG009 family performed                  For subcloning pYES, PCR ampli&#239;&#172;?ed <br /> SEED database (23). We used Blast tools             products corresponding yrdCEc, yciOEc, yrdCMm, <br /> resources NCBI (24). Annotations paralog families         ywlCBs genes &#239;&#172;?rst cloned entry <br /> were using physical clustering chromosome            vector      pENTR/D-TOPO           (Invitrogen),      using <br />  2896 Nucleic Acids Research, 2009, Vol. 37, No. 9 <br />  <br />  <br /> Table 1. Strains plasmids used <br />  <br /> Strain       Relevant characteristics                                                                                   Reference <br />  <br /> Yeast strains <br /> BY4741            MATa his3D1 leu2D0 met15D0 ura3D0 (S288C)                                                                  (59) <br /> BY4743            MATa/MAT  his3-D1/his3-D1 leu2-D0/leu2-D0 lys2-D0/lys2-D0 ura3-0/ura3-0                                    (59) <br /> VDC5118           YGN63 transformed plasmid pYES-DEST52; Ura+, Leu+                                                     This   study <br /> VDC5119           YGN63 transformed plasmid pYES-DEST52; Ura+, Leu+                                                     This   study <br /> VDC5120           YGN63 transformed plasmid pBY135; Ura+, Leu+                                                          This   study <br /> VDC5121           YGN63 transformed plasmid pBY135; Ura+, Leu+                                                          This   study <br /> VDC5122           YGN63 transformed plasmid pBY136; Ura+, Leu+                                                          This   study <br /> VDC5123           YGN63 transformed plasmid pBY136; Ura+, Leu+                                                          This   study <br /> VDC5124           YGN63 transformed plasmid pBY137; Ura+, Leu+                                                          This   study <br /> VDC5125           YGN63 transformed plasmid pBY137; Ura+, Leu+                                                          This   study <br /> VDC5126           YGN63 transformed plasmid pBY138; Ura+, Leu+                                                          This   study <br /> VDC5127           YGN63 transformed plasmid pBY138; Ura+, Leu+                                                          This   study <br /> VDC5300           Derivative BY4743; SUA5/sua5-D1::LEU2                                                                   This   study <br /> VDC5309           Derivative BY4741 transformed pYES-DEST52::yrdCEc; Ura+                                            This   study <br /> VDC5416           Derivative VDC5309; sua5-D1::LEU2; Ura+ Leu+                                                            This   study <br /> VDC5417           Derivative VDC5309; SUA5; Ura+ Leu+                                                                     This   study <br /> VDC5440           Derivative VDC5416 FOA treatment; sua5-D1::LEU2;Ura  Leu+ (cured pYES-DEST52::yrdCEc),         This   study <br />                    clone 14 <br /> VDC5441           Derivative VDC5416 FOA treatment; sua5-D1::LEU2;Ura  Leu+ (cured pYES-DEST52::yrdCEc),         This study <br />                    clone 15 <br /> YJN63             MAT  CYC I cyc7-67 ura3-52 leu2-3,112 cyh2 sua5::LEU2                                                      (4) <br /> YMH13             MAT  cyc1&#194;&#8212;1019 cyc7-67 ura3&#194;&#8212;52 leu2&#194;&#8212;3,112 cyh2                                                             (4) <br /> E. coli strains <br /> BL21(DE3)         F  ompT hsdSB(rB&#194;&#8212;, mB&#194;&#8212;) gal dcm (DE3) <br /> E. coli TOP10     F  mcrA  (mrr hsdRMS mcrBC) j80lacZ M15  lacX74 recA1 araD139  (ara-leu) 7697 galU galK rpsL               Invitrogen <br />                     (StrR) endA1 nupG <br /> MG1655                                                                                                                       Coli Genetic <br />                                                                                                                                Stock Center <br />                                                               R <br /> NM1100            IN(rrnD rrnE)1, rph-1, att PLCI857 Red abgTet                                                              (60) <br /> VDC5173           Derivative NM1100  yrdCEc-1::KmR/pBAD24::yrdCEc; KmR/AmpR                                               This study <br /> VDC5174           Derivative NM1100  yrdCEc-1::KmR/pBAD24::yrdCEc; KmR/AmpR                                               This study <br /> VDC5217           MG1655 transformed plasmid pBY125; AmpR                                                               This study <br /> VDC5218           MG1655 transformed plasmid pBY125; AmpR                                                               This study <br /> VDC5219           MG1655 transformed plasmid pBY126; AmpR                                                               This study <br /> VDC5220           MG1655 transformed plasmid pBY126; AmpR                                                               This study <br /> VDC5221           MG1655 transformed plasmid pBY128; AmpR                                                               This study <br /> VDC5222           MG1655 transformed plasmid pBY128; AmpR                                                               This study <br /> VDC5223           MG1655 transformed plasmid pBY130; AmpR                                                               This study <br /> VDC5224           MG1655 transformed plasmid pBY130; AmpR                                                               This study <br /> VDC5227           MG1655 transformed plasmid pBAD24; AmpR                                                               This study <br /> VDC5228           MG1655 transformed plasmid pBAD24; AmpR                                                               This study <br /> VDC5241           MG1655 transformed plasmid pBY166; AmpR                                                               This study <br /> VDC5333           MG1655 transformed plasmid pYC101.1; AmpR                                                             This study <br /> VDC5333           MG1655 transformed plasmid pYC101.4; AmpR                                                             This study <br /> VDC5335           MG1655 transformed plasmid pYC102.3; AmpR                                                             This study <br /> VDC5336           MG1655 transformed plasmid pYC102.3; AmpR                                                             This study <br /> VDC5337           MG1655 transformed plasmid pYC103.1; AmpR                                                             This study <br /> VDC5337           MG1655 transformed plasmid pYC103.2; AmpR                                                             This study <br /> VDC5339           MG1655 transformed plasmid pYC104.2; AmpR                                                             This study <br /> VDC5340           MG1655 transformed plasmid pYC104.4; AmpR                                                             This study <br /> VDC5342           MG1655 transformed plasmid pBY167; AmpR <br /> VDC5350           Derivative VDC5217 obtain P1 transduction  yrdCEc-1::KmR allele VDC5173;   KmR,   AmpR   This   study <br /> VDC5351           Derivative VDC5219 obtain P1 transduction  yrdCEc-1::KmR allele VDC5173;   KmR,   AmpR   This   study <br /> VDC5354           Derivative VDC5217 obtain P1 transduction  yrdCEc-1::KmR allele VDC5173;   KmR,   AmpR   This   study <br /> VDC5355           Derivative VDC5219 obtain P1 transduction  yrdCEc-1::KmR allele VDC5173;   KmR,   AmpR   This   study <br /> VDC5358           Derivative VDC5223 obtain P1 transduction  yrdCEc-1::KmR allele VDC5173;   KmR,   AmpR   This   study <br /> VDC5359           Derivative VDC5223 obtain P1 transduction  yrdCEc-1::KmR allele VDC517;    KmR,   AmpR   This   study <br /> VDC5362           Derivative VDC5242 obtain P1 transduction  yrdCEc-1::KmR allele VDC5173;   KmR,   AmpR   This   study <br /> VDC5363           Derivative VDC5242 obtain P1 transduction  yrdCEc-1::KmR allele VDC5173;   KmR,   AmpR   This   study <br /> VDC5366           Derivative VDC5333 obtain P1 transduction  yrdCEc-1::KmR allele VDC5173;   KmR,   AmpR   This   study <br /> VDC5367           Derivative VDC5333 obtain P1 transduction  yrdCEc-1::KmR allele VDC5173;   KmR,   AmpR   This   study <br /> VDC5370           Derivative VDC5337 obtain P1 transduction  yrdCEc-1::KmR allele VDC5173;   KmR,   AmpR   This   study <br /> VDC5371           Derivative VDC5337 obtain P1 transduction  yrdCEc-1::KmR allele VDC5173;   KmR,   AmpR   This   study <br /> Plasmids <br /> pBN200            pET21a:: yrdCEc                                                                                            This   Study <br /> pBN204            pYES::SUA5; URA3                                                                                           This   study <br /> pBY125            pBAD24:: ywlCBs cloned NcoI/XhoI fragment; AmpR                                                         This   study <br /> pBY126            pBAD24:: yrdCEc cloned NcoI/XhoI fragment; AmpR                                                         This   study <br />                                                                                  Nucleic Acids Research, 2009, Vol. 37, No. 9 2897 <br />  <br />  <br /> Table 1. Continued <br />  <br /> Strain     Relevant characteristics                                                                                 Reference <br />                                                                    R <br /> pBY128          pBAD24:: yciOEc cloned NcoI/XhoI fragment; Amp                                                        This   study <br /> pBY130          pBAD24:: SUA5 cloned NcoI/PstI fragment; AmpR                                                         This   study <br /> pBY135.1        pYesDEST52:: yrdCEc; AmpR URA3                                                                           This   study <br /> pBY136.1        pYesDEST52:: yciOEc; AmpR URA3                                                                           This   study <br /> pBY137.1        pYesDEST52:: ywlCBs; AmpR URA3                                                                           This   study <br /> pBY138.1        pYesDEST52:: yrdCMm; AmpR URA3                                                                           This   study <br /> pBY140          pCR2.1::UpyrdCEc; AmpR (999 base pairs upstream region yrdCEc cloned pCR2.1)                    This   study <br /> pBY141.6        pCR2.1::UpyrdCEc::DnyrdCEc; AmpR (999 base pairs upstream downstream region yrdCEc cloned   This   study <br />                   pCR2.1) <br /> pBY146.1        pGEM-T easy:: UpyrdCEc::DnyrdCEc; AmpR                                                                   This study <br /> pBY152.1        pGEM-T easy::UpyrdCEc::KmR::DnyrdCEc (DyrdCEC-1::KmR cassette) <br /> pBY166          pBAD24:: ycdCMm; AmpR                                                                                    This   study <br /> pBY167          pBAD24:: ycdCMm; AmpR                                                                                    This   study <br /> pYC101.1        Derived site directed mutagenesis pBY126; yrdCEcArg110!Ala; AmpR                                 This   study <br /> pYC101.4        Derived site directed mutagenesis pBY126; yrdCEcArg110!Ala; AmpR                                 This   study <br /> pYC102.2        Derived site directed mutagenesis pBY126; yrdCEcArg52!Ala; AmpR                                  This   study <br /> pYC102.3        Derived site directed mutagenesis pBY126; yrdCEcArg52!Ala; AmpR                                  This   study <br /> pYC103.1        Derived site directed mutagenesis pBY126; yrdCEcLys50!Ala; AmpR                                  This   study <br /> pYC103.2        Derived site directed mutagenesis pBY126; yrdCEcLys53!Ala; AmpR                                  This   study <br /> pYC104.2        Derived site directed mutagenesis pBY126; yrdCEcLys50!Ala; AmpR                                  This   study <br /> pYC104.4        Derived site directed mutagenesis pBY126; yrdCEcLys50!Ala; AmpR                                  This   study <br />  <br />  <br />  <br />  <br /> YcEcyrdCol3/YcEcyrdCol2; YcEcyciOol1/YcEcyciOol2;                      pET21b (Novagen, Inc., Madison, WI) using NdeI and <br /> YcMmyrdcol1/YcMmyrdCol2;                    YcBsywlCol1/               XhoI. <br /> YcBsywlCol2; sequenced recombination into <br /> pYES-DEST52 (carrying URA3 gene selection                   Construction E. coli DyrdCEc-1::KmR allele <br /> &#194;&#8212;ura media) (Invitrogen) using Gateway LR Clonase                  Fragments upstream (908 bp) downstream (991bp) of <br /> Enzyme Mix according manufacturer protocol.                        yrdCEc PCR ampli&#239;&#172;?ed E. coli MG1655 genomic <br /> Plasmids transformed yeast strains according              DNA using US71_ol1 UP979_ol2, DS1561_ol1 <br /> the manufacturer protocol (Invitrogen, Carlsbad, CA).                  DS2552_ol2, respectively. For subcloning purposes, <br /> Transformants screened SD -ura plates                      DS1561_ol1 DS2552_ol2 contained NotI and <br /> checked presence orthologous gene trans                  XhoI restriction sites respectively. The yrdCEc upstream <br /> by PCR using gene-speci&#239;&#172;?c primers used cloning.                fragment (UpyrdCEc) cloned pCR2.1-TOPO <br />    For cloning pRS313 (28), primers CENupsua5.ol1                 (Invitrogen, Carlsbad, CA). The resulting plasmid <br /> and CENdnsua5.ol2 used amplify SUA5 gene                   pCR2.1::UpyrdCEc (pBY140) checked orientation. <br /> from yeast 272 bp upstream region 124                The yrdCEc downstream fragment (DnyrdCEc) was <br /> bp downstream region (UpSUA5Dn). The upstream                       digested NotI XhoI ligated pBY140 <br /> region designed based previous data showing                     digested enzymes plasmid <br /> that presence 190 bp fragment upstream                     pBY141.6. The Upstream::Downstream fragment from <br /> SUA5 su&#239;&#172;&#402;cient drive gene expression (4). The PCR                 pBY141.6 subcloned pGEM-T easy vector <br /> product cloned pCR-Blunt TOPO (Invitrogen,                    (Promega) using ApaI SacI sites, resulting <br /> Carlsbad, CA) yielding pBY168.6. After checking                     plasmid pBY146.1 digested EcoRV site <br /> sequencing, UpSUA5Dn cloned pRS313                        present upstream downstream frag- <br /> using EcoRI restriction site &#239;&#172;&#8218;anking insert                 ments used clone KmR cassette (containing <br /> pCR-Blunt EcoRI site pRS313. The resulting                  FRT sites) ampli&#239;&#172;?ed pKD4 (29) <br /> plasmid, pBY176.3 (pRS313::UpSUA5Dn), allows SUA5                      using FRTKn_ol1 FRTKn_ol2 primers. The <br /> to expressed native promoter used             resulting     plasmid      pGEM-T         easy::UpyrdCEc:: <br /> complementation purposes.                                              KmR::DnyrdCEc (pBY152.1) used template <br />    For construction pBN204, yeast SUA5 gene                 generate PCR UpyrdCEc::KmR::DnyrdCEc <br /> ampli&#239;&#172;?ed using forward primer BN1 reverse primer                   deletion cassette used replacement yrdCEc. <br /> BN2 plasmid pM249 (4). The forward primer                         Induction   recombination functions preparation <br /> included yeast initiation sequence AAUA                      NM1100/pBAD24::yrdCEc, NM1100/pBAD24::yciOEc, <br /> BamHI site. The reverse primer included XhoI site.                  NM1100/pBAD24 competent cells as <br /> These sites used ligate DNA fragment                 described (30). After electroporation, cells plated <br /> pYES2 (URA3) (Invitrogen) generate pBN204. For                      LB Km supplemented 0.02% arabinose. Km resis- <br /> construction pBN200, yrdCEc ampli&#239;&#172;?ed geno-                 tant colonies re-isolated twice checked gene <br /> mic DNA using primers BN3 BN4 cloning                  replacement using Km-speci&#239;&#172;?c primers [k1 k2, (29)] in <br />  2898 Nucleic Acids Research, 2009, Vol. 37, No. 9 <br />  <br />  <br /> Table 2. Primers used study <br />  <br /> Primer                                                  Primer sequence <br />  <br /> BADBsywlCol1                                                 CCATGGAAACGAAAAGATGGTTT <br /> BADBsywlCol2                                                 AAGCTTTCAGCGAATCACTCTTCCTCC <br /> BADEcyciOol1                                                 CCATGGGCCAGTTTTTTTATATTCAT <br /> BADEcyciOol2                                                 AAGCTTTTATAAGAAAGGCTTCACATCAC <br /> BADEcyrdCol1                                                 CCATGGATAATAACCTGCAAAGAGA <br /> BADEcyrdCol2                                                 AAGCTTTTACCCCTGTCGAAACAGTTC <br /> BADMmyrdCol1                                                 CCATGGAAACTTTCGAGATTTCAGAAA <br /> BADMmyrdCol2                                                 AAGCTTTTATTCATTTACTGCTTTTAAAATCTC <br /> BADsua5ol1                                                   CCATGGACCTTGGACGACATTTTTT <br /> BADsua5ol2                                                   CTGCAGTTAAAACTGTATACAATTATTTGCAG <br /> YcMmyrdcol1                                                  CACCATGAAAACTTTCGAGATTTCAGAAAGT <br /> YcMmyrdCol2                                                  TCATTATTCATTTACTGCTTTTAAAATCTC <br /> YcBsywlCol1                                                  CACCATGAAAACGAAAAGATGGTTTGTG <br /> YcBsywlCol2                                                  TCAGCGAATCACTCTTCCTCCGG <br /> YcEcyciOol1                                                  CACCATGAGCCAGTTTTTTTATATTCATC <br /> YcEcyciOol2                                                  TCATTATAAGAAAGGCTTCACATCAC <br /> YcEcyrdCol1                                                  CACCATGAATAATAACCTGCAAAGAGA <br /> YcEcyrdCol3                                                  TCATTACCCCTGTCGAAACAGTTCA <br /> CENupsau5.ol1                                                GGTACCTATTTCGTTGAAAAATTCAGGC <br /> CENupsau5.ol2                                                GGTACCGTATGGGCGACTTTTCGTAT <br /> BN1                                                          CGGCGCTCCAAUAATGTACCTTGGACGACATTTTTTGGCA <br /> BN2                                                          CGACCAATGCTCGAGTTAAAACTGTATACAATTATTTGCAGC <br /> BN3                                                          CAG CCG CAT ATG AAT AAT AAC CTG CAA AGA GAC GC <br /> BN4                                                          GCC TCG CTC GAG CCC CTG TCG AAA CAG TTC ACC CG <br /> US71_ol1                                                     ACA ATG CCC TGC TAT GGC TG <br /> US979_ol2                                                    GTT ATT ATT CCG CCG AAA CCG <br /> DS1561_ol1                                                   TTA GCG GCC GCC GAC AGG GGT AAC ATA ATG G <br /> DS2552_ol2                                                   TTA CTC GAG GCG ATG CTG ACG AAA ACT CG <br /> FRTKnol1                                                     GTGTAGGCTGGAGCTGCTTC <br /> FRTknol2                                                     CTTAGTTCCTATTCCGAAGTTC <br /> sua5::Leu.ol1                                                TTT CGT TGA AAA ATT CAG GC <br /> sua5::Leu.ol3                                                GGG CGA CTT TTC GTA TAT ACA <br /> Upleu2_ol1                                                   AGA TCT CAC ACA GGG GCG CTA TCG C <br /> Dsleu2_ol2                                                   AGA TCT TAT AAA GTT TAT GTA CAA ATA TCA <br /> Chksua5leuol4                                                TTG CCA ATA CAA CAT AAC CG <br /> Chksua5leuol5                                                GTT ATT GCT CAT CAG CAG TA <br /> EcyrdCLys50Ala.ol1                                           ACTGTTGGAGTTAGCGCAGCGTCCGGTTG <br /> EcyrdCLys50Ala.ol2                                           CAACCGGACGCTGCGCTAACTCCAACAGT <br /> EcyrdCLys56Ala.ol1                                           CAGCGTCCGGTTGATGCGGGGCTGATTTTAATC <br /> EcyrdCLys56Ala.ol2                                           GATTAAAATCAGCCCCGCATCAACCGGACGCTG <br /> EcyrdCArg52Ala.ol1                                           GAGTTAAAACAGGCGCCGGTTGATAAGGGGC <br /> EcyrdCArg52Ala.ol2                                           GCCCCTTATCAACCGGCGCCTGTTTTAACTC <br /> EcyrdCArg110Ala.ol1                                          CTGGTTGACGGGCGCGTTTGATTCGCTTGCTG <br /> EcyrdCArg110Ala.ol2                                          CAGCAAGCGAATCAAACGCGCCCGTCAACCAG <br /> Leucassol3                                                   AGT CAT CGA ATT TGA TTC TG <br />  <br />  <br />  <br /> combination upstream downstream primers             sua5leu.ol3 primers. The Up::sua5-D1::Dn::LEU2 PCR <br /> (UP71_ol1 DN2552_ol2) described text.          product used transform yeast cells. Transformants <br />                                                              plated SD Gal/Raf -leu veri&#239;&#172;?ed PCR. To <br /> Construction sua5-D1::LEU2 allele                     cure URA3 marker containing plasmids (plasmid <br />                                                              derived pYES-DEST52 pYES2), 10 ml serial <br /> Primers sua5leu.ol1 sua5leu.ol3 used amplify     dilutions yeast liquid cultures grown SD Gal/Raf -leu <br /> a 1671 bp fragment encompassing SUA5. This fragment          spotted di&#239;&#172;&#8364;erent types plates. Plates <br /> was cloned pGEMT easy (Promega, Madison, WI)            containing SD Gal/Raf -ura -leu medium check for <br /> yielding pBY154. A BglII site exists ampli&#239;&#172;?ed     presence expected markers 5-FOA medium <br /> fragment used deletes 1000 bp        (SD Gal/Raf supplemented 50 mg/ml uracil and <br /> ampli&#239;&#172;?ed fragment, retaining, 225 bp homology       0.1% 5-FOA) select yeast cells did retain the <br /> region 50 (56 56 &#239;&#172;?rst bp     URA3 carrying plasmid [in presence URA3 gene <br /> SUA5) 323 bp homology 30 side. A leucine       5-FOA toxic (31)]. <br /> cassette ampli&#239;&#172;?ed wild-type BY4743 genomic <br /> DNA using primers UPleu.ol1 DNleu.ol2 that <br />                                                              Construction point mutantions yrdCEc by <br /> contained BglII sites ligated pBY154 diges- <br />                                                              site-directed mutagenesis <br /> tion vector insert BglII. The resulting <br /> plasmid, pBY164 used template amplify            Plasmid pBY126 (pBAD24::yrdCEc) used tem- <br /> Up:sua5-D1::Dn::LEU2 cassette using sua5leu.ol1          plate mutagenesis. Primer pairs EcyrdCLys50Ala. <br />                                                                          Nucleic Acids Research, 2009, Vol. 37, No. 9 2899 <br />  <br />  <br /> ol1/EcyrdCLys50Ala.ol2, EcyrdCArg52Ala.ol1/EcyrdCA              B 5 min, 50&#194;&#8212;75% B 4 min stay 75% B for <br /> rg52Ala.ol2, EcyrdCLys53Ala.ol1/EcyrdCLys53Ala.ol2              3 min, 75&#194;&#8212;100% B 8 min. The column was <br /> and EcyrdCArg110Ala.ol1/EcyrdCArg110Ala.ol2,               washed 100% B 7 min re-equilibrated at <br /> used mutate Lysine position 50, Arginine position      100% A 10 min. The &#239;&#172;&#8218;ow rate used 1 ml/min. <br /> 52, Lysine position 53 and, Arginine position 110         Chromatography performed room temperature. <br /> into Alanine. Mutagenesis performed described         The elution nucleosides followed UV detection <br /> the QuickChangesTM Site Directed Mutagenesis Kit           254 nm. The HPLC coupled hybrid triple <br /> (Stratagene, La Jolla, CA). All plasmids checked        quadrupole-ion trap (4000 Q-TRAP, Applied Biosystems, <br /> sequencing.                                                     Foster City, CA, USA) mass spectrometer equipped with <br />                                                                 TurboIonSpray (TIS) interface operated positive <br /> Growth curves                                                   ion mode. The ESI parameters were: curtain gas (CUR) at <br /> Growth curves performed using Bioscreen C MBR            30 psi, ion source (IS) 5000V, nebulizer gas (GS1) at <br /> (Oy Growth Curves AB Ltd, Finland) 30  max-           45 psi, TIS gas (GS2) 45 psi TIS probe temper- <br /> imum shaking. 300 ml culture used well,      ature 300  C. The information-dependent acquisition <br /> 10 replicates used condition. Yeast cultures      (IDA) mode operation employed a <br /> were grown SD Gal/Raf -his saturation diluted 200         survey scan m/z 100&#194;&#8212;600 acquired followed by <br /> times SD Gal/Raf -his loading Bioscreen.       collision-induced dissociation (CID) most <br /> The growth curves presented representative 10            intense ions. Survey MS/MS spectra &#239;&#172;?ve <br /> replicates.                                                     IDA cycle accumulated 1 s 3 s, respectively. <br />                                                                 tRNA extractions analysis performed least <br /> Puri&#239;&#172;?cation bulk tRNA                                        twice independently. <br /> For E. coli derivatives, 1 liter cultures grown over-   HPLC analysis <br /> night 37  C LB media supplemented ampicillin <br /> (50 mg/ml) arabinose required, harvested,          After digestion, samples immediately used analy- <br /> then stored &#194;&#8212;20  C. The cell pellets defrosted      sis HPLC. For that, 100 ml digested sample was <br /> resuspended 20 ml bu&#239;&#172;&#8364;er (10 mM Tris, 10 mM                added glass HPLC micro-vial run using <br /> MgCl2, pH 7.4). For yeast cultures, 2 l cell cultures        250 mM ammonium acetate pH 6 (Organic Solvent A) <br /> were grown saturation 30  C, harvested pellets        40% Acetonitrile (Aqueous Solvent B). Samples run <br /> immediately resuspended 20 ml bu&#239;&#172;&#8364;er (10 mM Tris,          60 min &#239;&#172;&#8218;ow rate 1.5 ml/min Suppleco <br /> 10 mM MgCl2, pH 7.4). In cases, homogenate             Discovery C-18 Reverse Phase column (15 cm   4.6 mm <br /> was treated bu&#239;&#172;&#8364;er-saturated phenol, pH 7.4                   5 um) (Suppleco 504955) detected Waters 2487 <br /> (Sigma), aqueous layer collected phenol       Dual Wavelength UV/Vis detector. Samples injected <br /> extraction step, RNA precipitated etha-        Waters 717 plus Autosampler, pumped a <br /> nol. The bulk tRNA puri&#239;&#172;?ed Nucleobond               Waters 1525 Binary Pump Waters Inline <br /> AX-400 columns (Clontech Laboratories, Mountain                 Degasser. Analysis Waters Breeze Software <br /> View, CA) (according manufacturer&#226;&#8364;&#8482;s protocol)            v3.20. The method used linear gradient consisting of <br /> and precipitated isopropanol.                              98% solvent A 2% solvent B start &#239;&#172;&#8218;ow rate <br />                                                                 1.5 ml/min, decreasing 80% A 20% B 60 min <br /> HPLC separation coupled electrospray tandem mass           change &#239;&#172;&#8218;ow rate. <br /> spectrometry analysis digested bulk tRNA <br />                                                                 Preparation tRNAThr <br /> To compare nucleoside constituents bulk tRNA <br /> puri&#239;&#172;?ed di&#239;&#172;&#8364;erent strains, samples              The plasmid containing E. coli tRNAThrCGU gene <br /> puri&#239;&#172;?ed bulk tRNA preparation enzymatically                thrW kind gift Dr Waas (The Scripps Research <br /> hydrolyzed described Pomerantz et al. (32)                Institute). tRNAThrCGU overexpressed MRE600 <br /> Approximately 100 mg bulk tRNA digested             cells induced isopropyl-b-D-thiogalactoside (IPTG, <br /> 10 units Nuclease P1 (Sigma), 0.01 units                  0.5 mM, 37  C, 12 h) LB-Amp (100 mg/ml). The har- <br /> Phosphodiesterase I (Sigma) 3 ml E. coli Alkaline        vested cells lysed bu&#239;&#172;&#8364;er (0.3 M NaOAc, pH 5.2, <br /> Phosphatase (Sigma P4252) total volume 113 ml.          10 mM EDTA) subjected phenol extraction (pH <br /> The digested extracts lyophilized resuspended       4.5) ethanol precipitation. tRNAThr partially pur- <br /> 20 ml water. The resuspended tRNA degradation                i&#239;&#172;?ed total nucleic-acid extract Nucleobond <br /> extracts injected LC&#194;&#8212;MS/MS sepa-           column (Clontech Laboratories, Mountain View, CA) <br /> ration identi&#239;&#172;?cation nucleosides. A C18-RP           using manufacturer protocols, precipitated isopropy- <br /> (Gemini 5m, 250   4.6 mm, Phenomenex, Torrance, CA,             lalcohol (100%, room temperature), washed ethanol <br /> 14 USA) analytical column used, C18 cartridge        (80%, &#194;&#8212;20  C), dissolved water, point it <br /> to prolong lifetime. The mobile phase consisted      80% pure. tRNAThr puri&#239;&#172;?ed C4 <br /> solvents: 250 mM ammonium acetate pH6.0 solvent          reverse-phase preparative column pre-equilibrated with <br /> A 40% acetonitrile solvent B. The gradient used         bu&#239;&#172;&#8364;er A (10 mM sodium phosphate pH 5.5, 1 M sodium <br /> for nucleoside separation performed previously           formate, 8 mM MgCl2) eluted 60-min 0&#194;&#8212;40% <br /> described (32). It consisted elution gradient sev-     gradient bu&#239;&#172;&#8364;er B (10 mM sodium phosphate pH 5.5, <br /> eral steps: 0&#194;&#8212;5% B 7 min, 5&#194;&#8212;10% B 15 min, 25&#194;&#8212;50%          10% methanol). The tRNA eluted equal-sized <br />  2900 Nucleic Acids Research, 2009, Vol. 37, No. 9 <br />  <br />  <br /> peaks. Upon assaying fractions threonine accep-       &#239;&#172;&#8218;uorescence signal taken 3-min incu- <br /> tance E. coli threonyl-tRNA synthetase using stan-     bation period step allow complete binding. <br /> dard aminoacylation assay described (33),          To correct dilution e&#239;&#172;&#8364;ects, titration was <br /> peaks contain tRNAThr &gt;97% purity.           repeated addition incremental <br /> Treatment base (pH 8.5) pooled sample          volumes water. All readings corrected bu&#239;&#172;&#8364;er <br /> both peaks yielded separate peaks ana-        absorption need inner &#239;&#172;?lter correction (absor- <br /> lytical reverse-phase chromatography, indicating     bance 295 nm tRNA highest concentration was <br /> two species correspond di&#239;&#172;&#8364;erent aminoacylation      0.05). Theoretical curves &#239;&#172;?tted exper- <br /> states. Fractions peaks pooled separately    imental data dissociation constants determined <br /> and tRNA precipitated, washed twice cold etha-       according following equation (36): <br /> nol, re-dissolved water. MALDI-TOF mass spec-                          0                                     1 <br /> trometry analysis &#239;&#172;?nal tRNA peaks                               &#195;&#176;Kd &#195;&#190; P0 &#195;&#190; N0 &#195;&#382;                      <br /> revealed molecular masses 24 540 24 686 Da. The         I              B  aqrt &#195;&#176;Kd &#195;&#190; P0 &#195;&#190; N0 &#195;&#382;2   &#195;&#176;4P0 N0 &#195;&#382; C <br />                                                                  &#194;&#188; 1 &#195;&#190; &#195;&#176;f   1&#195;&#382;B <br />                                                                               @ <br />                                                                                                                     C <br />                                                                                                                     A <br /> mass di&#239;&#172;&#8364;erence accounted molecular mass      I0                                2P0 <br /> carbamoyl-threonine (145.12 Da). <br />    The tRNAThr transcript synthesized standard T7 <br /> polymerase transcription (34) linearized DNA tem-        I0 I corrected &#239;&#172;&#8218;uorescence intensities <br /> plate [pUC18; (35)] containing E. coli tRNAThrCGU         protein protein&#194;&#8212;tRNA complex, respectively. P0 <br /> gene thrW (kind gift Dr K. Beebe, The Scripps              N0 total concentrations protein and <br /> Research Institute). After phenol&#194;&#8212;chloroform extraction       tRNA, respectively, f &#239;&#172;&#8218;uorescence intensity ratio <br /> of reaction, RNA ethanol precipitated,            bound free protein, Kd dissociation <br /> re-dissolved water puri&#239;&#172;?ed anion-exchange           constant. <br /> high-performance liquid chromatography heated <br /> (90  C) preparative DNA-Pac column (Dionex,                   Detection ATP binding STD-NMR <br /> Sunnyvale, CA) according standard protocols (34).          NMR experiments performed 278K Varian <br /> The single tRNAThr eluant pooled, ethanol precipi-        INOVA 600 NMR spectrometer equipped 5 mm <br /> tated, re-suspended water assayed threonine        triple resonance inverse detection probe. Samples were <br /> acceptance. Final purity &gt;95% 10% urea PAGE            prepared 500 ml 99% D2O PBS bu&#239;&#172;&#8364;er pH 7.5 <br /> analysis. All &#239;&#172;?nal tRNA samples prepared             (uncorrected) containing 10 mM phosphate, 137 mM <br /> binding studies heating 90  C slow-cooling          NaCl, 2.7 mM KCl, 0.5 mM MgCl2 0.02% NaN3. <br /> after addition 5 mM MgCl2 ensure proper                 Concentration NMR sample 20 mM protein <br /> folding.                                                      1 mM ligand. The STD pulse sequence implemen- <br />                                                               ted suggested (37), minor variations. The exci- <br /> Overexpression puri&#239;&#172;?cation E. coli YrdC            tation train composed 50 ms G4 composite pulses <br /> protein (YrdCEc)                                              1 ms delay pulses. On-resonance o&#239;&#172;&#8364;- <br /> E. coli BL21(DE3) cells transformed pET21         resonance irradiations centered &#194;&#8212;1 ppm and <br /> derivative (Novagen) vector containing E. coli yrdC       30 ppm, respectively, total saturation time was <br /> gene (pBN200). The protein overexpressed induc-        3 s. Initial FIDs multiplied line broadening <br /> tion 1 mM IPTG LB medium containing 100 mg/ml         function 1 Hz zero &#239;&#172;?lled twice original <br /> Amp. Cells harvested lysed bu&#239;&#172;&#8364;er (20 mM           data size prior Fourier transformation. A total 512 <br /> Tris, 500 mM NaCl, 28 mM imidazole, 1 mM b-mercap-            scans 16 dummy scans acquired, recycle <br /> toethanol protease inhibitor cocktail). The protein       delay 2 s. <br /> was puri&#239;&#172;?ed Ni-NTA a&#239;&#172;&#402;nity chromatography, <br /> dialyzed phosphate-bu&#239;&#172;&#8364;ered saline concen- <br />                                                               RESULTS <br /> trated using YM-3 Centriprep &#239;&#172;?lter (Millipore Corp., <br /> Billerica, MA).                                               Involvement Sua5/YrdC family proteins <br />                                                               t6A formation <br /> Measurements YrdCEc binding tRNA Trp <br />                                                               Bioinformatic identi&#239;&#172;?cation. Analysis sequenced <br /> &#239;&#172;&#8218;uorescence quenching <br />                                                               tRNAs (38) suggested organisms a <br /> We used steady-state techniques monitor changes     t6A biosynthesis pathway. Using MGDB database <br /> intrinsic &#239;&#172;&#8218;uorescence tryptophan chains (at   (http://mbgd.genome.ad.jp/), list orthologous families <br /> positions 89 106) E. coli YrdC addition        present E. coli K12, S. cerevisiae, B. subtilis 168, <br /> tRNA previously described (2). Using Jobin Yvon          Buchnera aphidicola APS, Wigglesworthia glossinidia, <br /> FluoroMax-3 spectro&#239;&#172;&#8218;uorometer (Edison, NJ),              Wollbachia endosymbiot mel mollicutes the <br /> a cell compartment kept constant temperature             database (16 genomes) generated. This analysis <br /> of 21  C, &#239;&#172;&#8218;uorescence signal (lex = 295 nm;               revealed existence 95 orthologous protein families, <br /> lem = 345 nm) free YrdC (2 mM 20 mM HEPES,              related translation (ribosomal protein, tRNA <br /> 100 mM NaCl, pH 7.5) taken. De&#239;&#172;?ned volumes           synthetases tRNA modi&#239;&#172;?cation). Only these <br /> concentrated aqueous solution tRNA added         families did experimentally veri&#239;&#172;?ed function <br /> in steps &#239;&#172;?nal tRNA concentrations 0.05&#194;&#8212;3.50 mM          work initiated been <br />                                                                                       Nucleic Acids Research, 2009, Vol. 37, No. 9 2901 <br />  <br />  <br />  <br />  <br /> Figure 1. (A) Schematic representation YrdC/Sua5 family. YrdC E. coli 192 amino acids, yeast Sua5 420 amino acids the <br /> human IRIP 279 amino acids 1&#194;&#8212;55 mitochondrial signal peptide. The position KxR( 50)SxN conserved motif given the <br /> E. coli sequence. (B) Sequence alignment YrdC domain YrdC Sua5 family proteins The E. coli YrdC, S. tokodaii Sua5 E. coli YciO <br /> sequences aligned based 3D alignment crystal structures. Secondary structure elements crystal structure E. coli YrdC are <br /> shown sequences. Red boxes indicate conserved residues YrdC YciO families. Residues interact bound AMP the <br /> S. tokodaii Sua5 crystal structure indicated red black stars. K50 R52, mutated present study used distinguish between <br /> the YciO YrdC families, indicated red stars. S107, residue mutated yeast sua5-1 mutant indicated #. Ec: E. coli, <br /> Mm: M. maripaludis, Bs: B. subtilis, St: Sulfolobus tokodaii, Sc: S. cerevisiae. <br />  <br />  <br /> functionally characterized since. The YrdC/Sua5                            enzyme family HypF (2) [involved maturation <br /> (Figure 1A) family probable candidate                     metal center Ni-Fe hydrogenase HypE (39)] <br /> an enzyme involved t6A biosynthesis following                   catalyzes chemistries similar ones expected in <br /> reasons: (i) organisms sequenced date homo-                  t6A biosynthesis (22); (iii) YrdC YciO E. coli were <br /> log YrdC/Sua5 proteins, organisms,                     bind double-stranded RNA tRNA (2); <br /> E. coli, representatives (YrdC YciO),                     (iv) Sua5 yeast proposed involved transla- <br /> most one; (ii) YrdC domain                     tion (4). <br />  2902 Nucleic Acids Research, 2009, Vol. 37, No. 9 <br />  <br />  <br />  <br />  <br /> Figure 2. (A) LC&#194;&#8212;MS/MS analysis yeast tRNA extracted di&#239;&#172;&#8364;erent strains linking disappearance t6A peak deletion sua5. <br /> UV traces ion extraction chromatograms 413 m/z (small windows) shown WT (YMH13) upper panels, sua5::LEU2 (YJN63) middle <br /> panels YGN63 transformed pBN204 (SUA5 URA3) lower panels. (B) Complementation t6A minus phenotype followed HPLC <br /> analysis bulk yeast tRNA extracted various strains. The position t6A peak con&#239;&#172;?rmed running t6A standard spiking the <br /> WT sample (data shown). Strain YGN63 transformed URA3 plasmids carrying SUA5 (as control) carrying wild-type genes yrdCEc, <br /> yciOEc ywlCBs, respectively recovery t6A peak monitored. (C) LC&#194;&#8212;MS/MS analysis yeast tRNA extracted the <br /> sua5::LEU2 yrdCMm strain. UV traces ion extraction chromatograms 413 m/z (small windows) shown. Positive negative controls were <br /> run presented (A) (data shown). <br />  <br />  <br /> In vivo experimental validation. In order examine                  wild-type genes: yrdCEc (pBY135.1), yciOEc (pBY136.1), <br /> t6A phenotype (i.e. presence absence t6A modi&#239;&#172;?ca-                   ywlCBs (pBY137.1) yrdCMm (pBY138.1). Bulk tRNA <br /> tion RNA) strains lacking members SUA5/                      extracted analyzed check presence <br /> yrdC gene family existing strain, YJN63 (sua5::LEU2)                   t6A transformed strains. All genes excep- <br /> (4), carries deletion allele SUA5 analyzed.                tion yciOEc, able complement t6A&#194;&#8212; pheno- <br /> Bulk tRNA extracted YGN63 near                        type YJN63 (Figure 2B C). <br /> isogenic wild-type strain YMH13 (SUA5) (4) pro- <br /> cessed LC&#194;&#8212;MS/MS analysis described                          Essentiality yrdC orthologs <br /> &#226;&#8364;&#732;Materials Methods&#226;&#8364;&#8482; section. This analysis revealed                   Essentiality S. cerevisae SUA5 wild-type gene. In order <br /> that 25.5 min peak detected UV trace                    clarify contradictory data essentiality of <br /> parent strain, corresponding protonated molecular                  SUA5, replacement wild-type allele a <br /> weight t6A (MH+= 413m/z), disappeared strain                    sua5-D1::LEU2 cassette attempted di&#239;&#172;&#8364;erent <br /> carrying sua5::LEU2 allele (Figure 2A). The t6A peak                  backgrounds (see &#226;&#8364;&#732;Material Methods&#226;&#8364;&#8482; section con- <br /> was recovered strain transformed                     struction details): diploid, BY4743 (SUA5/SUA5 <br /> plasmid carrying wild-type SUA5 gene (plasmid                         ura3D0/ura3D0 leu2-D0/leu2-D0), haploid BY4741 <br /> pBN204) con&#239;&#172;?rming t6A&#194;&#8212; phenotype (absence                        (SUA5 ura3D0 leu2-D0) VDC5309 (SUA5 yrdCEc <br /> of t6A peak) loss SUA5 (Figure 2A).                     URA3 ura3D0) strain BY4741 transformed with <br />    To determine sua5 homologs actual func-                   yrdCEc URA3 carrying plasmid (pBY135.1). Leu+ <br /> tional orthologs, E. coli, B. subtilis M. maripaludis                 clones obtained backgrounds, however, <br /> yrdC-like genes tested ability complement               analyzed PCR, clones derived the <br /> the t6A&#194;&#8212; phenotype sua5::LEU2 strain. YGN63                    diploid BY4743 merodiploid VDC5309 carried <br /> transformed URA3 plasmids carrying following                     sua5-D1::LEU2 marker right locus, yielding <br />                                                                                       Nucleic Acids Research, 2009, Vol. 37, No. 9 2903 <br />  <br />  <br />  <br />  <br /> Figure 3. (A) PCR ampli&#239;&#172;?cation con&#239;&#172;?rm presence correct location sua5-D1::LEU2 allele various yeast strains. Lanes 1, 3 5: <br /> primers Chksua5leu.ol4 Chksua5leu.ol5 lanes 2, 4 6: primers Chksua5leu.ol4 Leucassol3. Strains VDC5417 (BY4741 derivative <br /> SUA5 LEU2 URA3 yrdCEc) lanes 1/2; VDC5300 (BY4743 derivative (SUA5/sua5-D1::LEU2) lanes 3/4; VDC5416 (BY4741 derivative sua5- <br /> D1::LEU2 yrdCEc) lanes 5/6. Strains tested primer combinations. In VDC5417 cassette inserted ectopically, i.e. Leu+ and <br /> negative PCR tests using Chksua5leu.ol4/Leucassol3 primer set. For VDC5033 VDC5416 insertion cassette correct <br /> location con&#239;&#172;?rmed. Expected sizes are: 750 bp sua5-D1::LEU2 allele right locus using Chksua5leu.ol4/Leucassol3 combination and <br /> 2300 bp wild-type allele SUA5 2800 bp sua5-D1::LEU2 allele using Chksua5leu.ol4/Chksua5leu.ol5 primer combination. <br /> Annealing positions primers used shown panel A. Only relevant genotypes shown, refer Table 1 completeness. (B) <br /> FOA curing pBY135.1 (URA3 yrdCEc) strains VDC5300 (SUA5/sua5-D1::LEU2), VDC5417 (SUA5 LEU2 URA3 yrdCEc) VDC5416 <br /> (sua5-D1::LEU2 yrdCEc). 20 ml serial dilutions saturated cultures VDC5416, VDC5417 VDC5300 replica-plated SD Gal/Raf -ura <br /> -leu FOA plates analyzed 5 days. Strains carrying URA3 marker unable grow 5-FOA. VDC5300 used control <br /> for auxotrophies. (C) Veri&#239;&#172;?cation cured strains PCR using plasmid speci&#239;&#172;?c primers DEST52.ol1 DEST52.ol2: lane 1, VDC5417; lane 2, <br /> VDC5416; lane 3 4 corresponding respective FOA treated derivatives. <br />  <br />  <br /> SUA5/sua5-D1::LEU2 sua5-D1::LEU2 yrdCEc deriva-                        background (Figure 3B). Both curing pBY135.1 <br /> tives, respectively (Figure 3A). In haploid background                 sua5-D1::LEU2 genotype con&#239;&#172;?rmed by <br /> BY4741, ectopic insertion cassette occurred yielding                PCR (Figure 3C) plating SD Gal/Raf agar <br /> SUA5 LEU2 strains, correct insertion dele-                   plates lacking uracil/leucine leucine (data not <br /> tion cassette obtained. Thus, SUA5                  shown). These results showed SUA5 gene not <br /> disrupted backgrounds functional copy                           essential yeast con&#239;&#172;?rming results previous study <br /> remained present genome, suggesting SUA5                       (4). Further analysis growth phenotype of <br /> is essential gene yeast, agreement                   sua5-D1::LEU2 strains con&#239;&#172;?rmed growth defect phe- <br /> result systematic yeast gene deletion study (11).                   notypes observed FOA plates. Growth experiments on <br />    In order demonstrate essentiality                     YPD dramatic decrease growth rate cured <br /> SUA5, attempts dissociate sua5-D1::                       strain VDC5440 compared haploid parent strain <br /> LEU2 allele URA3 yrdCEc plasmid pBY135.1.                         BY4741 (Figure 4A). A plasmid carrying SUA5 under <br /> If SUA5 essential plasmid cur-                   control endogenous promoter region <br /> able VDC5416 (sua5-D1::LEU2 pBY135.1),                       (pBY176.3) complemented t6A de&#239;&#172;?ciency of <br /> should readily lost VDC5417 (SUA5 LEU2                             VDC5440 slow growth phenotype (Figure 4B <br /> pBY135.1). Since URA3 allele carried plasmid                    C), con&#239;&#172;?rming secondary <br /> confers sensitivity 5-FOA, strains tested                mutation. These results gene nor <br /> ability grow presence compound. Culture                  modi&#239;&#172;?cation essential yeast. The failure to <br /> dilutions VDC5416 plated 5-FOA medium                       obtain haploid mutant direct transformation is <br /> the resulting growth phenotype compared                     likely reduced growth phenotype. <br /> control strains: VDC5417 carrying ectopic insertion of <br /> the deletion cassette heterozygote VDC5300                         Essentiality E. coli yrdC. To test yrdCEc <br /> (SUA5/sua5-D1::LEU2 ura3D0/ura3D0). Derivatives                            gene essential E. coli, similar approach as <br /> from control strains grew presence 5-FOA                     employed yeast followed. Homologous <br /> expected. After prolonged incubation, 5-FOAR derivatives                   replacement wild-type yrdC allele  yrdCEc- <br /> of sua5-D1::LEU2 pBY135.1 recovered suggest-                     1::KmR allele attempted direct transformation of <br /> ing pBY135.1 eliminated                            UpyrdCEc::KmR::DnyrdCEc PCR generated fragment <br />  2904 Nucleic Acids Research, 2009, Vol. 37, No. 9 <br />  <br />  <br />  <br />  <br /> Figure 4. (A) Growth phenotype VDC5440 (sua5-D1::LEU2 ura3D0) compared WT parent BY4741 (B) Growth phenotype of <br /> VDC5440 transformed plasmid pRS313 carrying SUA5 control endogenous promoter region (pBY176.3) plasmid <br /> control pRS313 SD Gal/Raf -his. All growth experiments performed 30  C. Shown representative plots replications. (C) <br /> LC&#194;&#8212;MS/MS analysis tRNA extracted yeast strains showing absence presence t6A peak UV trace (left side) and <br /> extraction chromatogram (right side) VDC5440 transformed pBY176.3 (upper panels) VDC5440 transformed pRS313 (lower <br /> panels). <br />  <br />  <br /> into di&#239;&#172;&#8364;erent backgrounds: E. coli NM1100,                            P1 transduction following strains used as <br /> NM1100/pBAD24               NM1100/pBAD24::yrdCEc                       recipients: MG1655 transformed pBAD24 deriva- <br /> (pBY126) (see &#226;&#8364;&#732;Material Methods&#226;&#8364;&#8482; section),                          tives carrying functional yrdC homologs, yrdCEc, <br /> latter strain contains functional copy yrdCEc                     ywlCBs, yrdCMm SUA5Sc (pBY126, pBY125, pBY167 <br /> plasmid. Km resistant clones insertion                          pBY130 respectively) carrying non-functional <br /> marker expected location (as veri&#239;&#172;?ed PCR)                         homolog yciOEc (pBY128). P1 transductions carried <br /> were obtained NM1100/pBAD24::yrdCEc                           &#239;&#172;?ve times independent MG1655 back- <br /> the backgrounds suggesting chromo-                       grounds ortholog, Km-resistant colonies were <br /> somal copy gene eliminated func-                  recovered backgrounds (with variation of <br /> tion insured gene trans (Figure 5A).                             120&#194;&#8212;500 colonies transduction) MG1655/ <br />    The essentiality yrdCEc demonstrated                   pBAD24 MG1655/pBAD24::yciOEc never <br /> P1 transduction experiments. Strain VDC5173 (NM1100                         yielded colonies (while non-essential markers were <br />  yrdCEc-1::KmR/pBAD24::yrdCEc) used donor                        readily transduced strains). These data con&#239;&#172;?rm <br />                                                                                    Nucleic Acids Research, 2009, Vol. 37, No. 9 2905 <br />  <br />  <br />  <br />  <br /> Figure 5. (A) PCR ampli&#239;&#172;?cation con&#239;&#172;?rm presence  yrdCEc-1::KmR allele kanamycin-resistant NM1100/pBAD24::yrdCEc derivatives. <br /> Two independent KmR strains (VDC5173, lanes 1, 4 7 VDC5174 lanes, 2, 5 8) obtained transformation NM1100/ <br /> pBAD24::yrdCEc PCR generated  yrdCEc-1::KmR cassette NM1100/pBAD24::yrdCEc background strain (lanes, 3, 6 9) were <br /> subjected PCR analysis using primer combinations UpyrdC/K2 (lanes 1, 2 3), DsyrdC/K1 (lanes 4, 5 6) UpyrdC/DnyrdC (lanes <br /> 7, 8 9). The expected sizes UpyrdC/DsyrdC amplicons WT mutant strains 2340 3113 bp respectively, DsyrdC/K1 and <br /> UpyrdC/K2 ampli&#239;&#172;?cation NM1100 control. (B) Growth phenotype MG1655  yrdCEc-1::KmR P1 transductants. Strains <br /> were streaked LB Amp plates supplemented 0.02% arabinose incubated 24 h 37  C. MG1655/pBAD24 used control. <br />  <br />  <br />  <br /> that yrdCEc essential gene E. coli strains                 apparently non-speci&#239;&#172;?c binding YrdC and <br /> MG1655 NM1100, supports result obtained                     unmodi&#239;&#172;?ed vitro transcript tRNAThr be <br /> in yeast, SUA5, yrdCEc, yrdCMm, ywlCBs                  detected. These results suggest YrdC involved in <br /> functional orthologs yciOEc.is not. The pBAD con-                   t6A biosynthesis interacting directly tRNAs, con- <br /> struct places genes control arabinose               sistent previously reported binding YrdC to <br /> inducible promoter orthologs, SUA5Sc                  total tRNA (2). <br /> the pro&#239;&#172;?cient replacing yrdCEc gene. In the <br /> absence inducer, E. coli strain carrying                     Binding YrdCEc ATP. t6A biosynthesis crude <br />  yrdCEc-1/pBY130 (pBAD24::SUA5) showed delayed                        enzyme preparations shown require ATP <br /> growth phenotype observed strains carrying                  (22) Sua5St ortholog Sulfolobus tokodaii <br /> three genes (Figure 5B).                                          recently bind hydrolyze ATP <br />    Analysis bulk tRNA extracted classes                (41). To investigate YrdCEc speci&#239;&#172;?cally bind <br /> of P1 transductants obtained showed contained                  ATP, screened various nucleotides potential <br /> t6A indicating orthologs complement                 bind YrdC saturation-transfer di&#239;&#172;&#8364;erence NMR <br /> essentiality tRNA modi&#239;&#172;?cation phenotypes (data                  (STD-NMR) spectroscopy. A strong STD-NMR signal <br /> shown).                                                                 seen ATP ADP CTP and <br />                                                                         UTP, indicating adenine-speci&#239;&#172;?c binding (Figure 6B), <br />                                                                         pointing presence ATP-binding <br /> Direct involvement YrdC t6A biosynthesis <br />                                                                         sites YrdCEc. <br /> Preferential binding YrdCEc apomodi&#239;&#172;?ed <br /> tRNAThr. To investigate members YrdC/                    Essentiality E. coli YrdC conserved residues Arg52 and <br /> Sua5 family act directly tRNA t6A biosynthesis,                   Lys50. Arg52 Lys50 residues strictly conserved <br /> YrdCEc binding tRNA species known contain                       YrdC proteins absent YciO pro- <br /> t6A E. coli, tRNAThr, tested. Three forms                     teins (Figure 1B). Both residues located center of <br /> E. coli tRNAThrCGU generated; vivo expressed                positively charged cleft putative nucleic-acid- <br /> forms vitro transcribed form lacked                 binding site, based crystal structure the <br /> post-transcriptional modi&#239;&#172;?cations. As seen previously                S. tokodaii protein Sua5St, chains (Arg59 and <br /> other investigators (40), overexpression tRNAThr pro-                Lys57 using S. tokodaii residue numbering) interact with <br /> duced forms di&#239;&#172;&#8364;er exact molecular mass               stabilize AMP derived phosphate group found <br /> carbamoyl-threonine suggesting di&#239;&#172;&#8364;er                   bound cleft. (41). This suggested nucleotide <br /> presence absence modi&#239;&#172;?cation nucleotide                   binding YrdC Sua5 (41) critical function. <br /> 37. Both forms aminoacylated ThrRS similar                 To test functional signi&#239;&#172;?cance residues <br /> degrees indicating properly folded, func-                vivo, site-directed mutagenesis used construct <br /> tional tRNAs. Fluorescence quenching data                     E. coli strain MG1655 carrying pBAD24 plasmids harbor- <br /> YrdC binds fully modi&#239;&#172;?ed tRNAThr partially                    ing yrdCEc Lys50 ! Ala Arg52!Ala, yrdCEc <br /> modi&#239;&#172;?ed tRNAThr (lacking threonyl-carbamoyl                    Arg110!Ala Lys53!Ala control (pYC102.3, <br /> modi&#239;&#172;?cation) estimated dissociation constants                   pYC104.2, pYC101.1 pYC103.1, respectively). These <br /> 0.62 &#194;&#177; 0.13 0.11 &#194;&#177; 0.05 mM, respectively, indicating                used P1 experiments recipient strains as <br /> high binding a&#239;&#172;&#402;nity (Figure 6A). Only weak                          described earlier. More clones were <br />  2906 Nucleic Acids Research, 2009, Vol. 37, No. 9 <br />  <br />  <br />  <br />  <br /> Figure 6. (A) YrdC binding tRNAThr measured intrinsic &#239;&#172;&#8218;uorescence quenching assay. Relative Trp &#239;&#172;&#8218;uorescence intensity YrdC a <br /> function partially modi&#239;&#172;?ed vivo expressed tRNAThr lacking t6A37 modi&#239;&#172;?cation (&#239;&#172;?lled square), fully modi&#239;&#172;?ed vivo expressed tRNAThr <br /> (&#239;&#172;?lled circle) unmodi&#239;&#172;?ed tRNAThr transcript (&#239;&#172;?lled triangle). Sample conditions: 2 mM protein, 20 mM HEPES, pH 7.5, 100 mM NaCl, 20  C. <br /> lex = 295 nm, lem = 345 nm. (B) STD-NMR spectra nucleotides presence E. coli YrdC. For nucleotides ATP, ADP, CTP UTP <br /> two traces shown, lower trace reference 1D NMR spectrum, upper trace STD spectrum obtained irradiation of <br /> the aliphatic region YrdC. Ligand binding indicated positive signals STD spectrum, particularly strong H8, H2 and <br /> H1 resonances ATP ADP 6&#194;&#8212;8 ppm region. Little binding observed CTP UTP. <br />  <br />  <br />  <br /> recovered P1 experiments control strain                      concave surface crystal structure YrdC are <br /> MG1655pyrdCEc, MG1655pyrdCEc Arg110!Ala                                 strictly conserved YrdC subfamily absent in <br /> MG1655pyrdCEc Lys53!Ala used recipients,                           YciO sequences (Figure 1B). These residues essential <br /> but transductants recovered MG1655/                       YrdC function shown site directed mutagenesis <br /> pYC104.2 MG1655/pYC102.3 used, implying                           complementation experiments. Therefore COG009 <br /> that Arg52 Lys56 essential YrdCEc function.                    split sub-families results pre- <br /> All receiver strains tested sensitivity P1. These              sented suggest distinct functions YciO YrdC. <br /> results suggest nucleotide binding essential                  Like m1G37 	55, t6A37 tRNA <br /> function proteins consistent                     modi&#239;&#172;?cations present organisms the <br /> known ATP requirement t6A biosynthesis (22).                           set tRNA sequences available (38,42). It is <br />                                                                            universal modi&#239;&#172;?cations believed have <br />                                                                            present common ancestor (43,44) and <br /> DISCUSSION                                                                 sequenced genome (&gt;600 date) YrdC <br /> Our results indicate universally conserved YrdC/                  ortholog. <br /> Sua5 family involved biosynthesis N6-threonyl-                   In study essentiality gene family been <br /> carbamoyl adenosine. Deletion SUA5 led loss                   resolved. In E. coli, yrdC appears essential, however <br /> t6A independent yeast backgrounds. E. coli YrdC                     observed lack viability absence <br /> preferentially bound tRNAThr containing modi&#239;&#172;?ca-                       t6A modi&#239;&#172;?cation remains established. In <br /> tions t6A compared vitro (unmodi&#239;&#172;?ed) tran-                    apparent contradiction results, insertion the <br /> scribed tRNA. Two invariant residues involved                           PBAD promoter &#239;&#172;?rst 15 nucleotides yrdC <br /> nucleotide binding characterized, strongly indicating                 reported lethal repressing conditions, <br /> that role YrdC/Sua5 t6A biosynthesis mediated                 minimal rich medium (10). The pheno- <br /> by ATP binding hydrolyzing functions,                        type minimal medium attributed polar repres- <br /> further suggesting direct role protein                   sion downstream aroE gene (aromatic amino-acid <br /> biosynthetic pathway, ATP required t6A                        biosynthetic gene), essential growth mini- <br /> formation (22).                                                            mal medium, LB aroE yrdC was <br />    The YrdC/Sua5 proteins assigned                            considered essential. However, expression the <br /> COG009 organisms member                          pBAD promoter completely repressed glucose <br /> this family. Exceptions include plants bacteria                (27,45), expression pBAD repression yield suf- <br /> that homologs. Expression second                           &#239;&#172;?cient YrdC protein ful&#239;&#172;?ll essential function(s) the <br /> COG009 encoding gene E. coli, yciO, failed comple-                   cell. <br /> ment t6A minus phenotype yeast strains lacking                       In yeast, gene modi&#239;&#172;?cation essen- <br /> functional SUA5 gene. The subfamilies                      tial, SUA5 deletion strains lacking t6A been <br /> distinguished sequence analysis residues K50                     constructed [this study (4)]. However, t6A&#194;&#8212; strains <br /> R52 positively charged                        display marked slow growth phenotype [this study and <br />                                                                          Nucleic Acids Research, 2009, Vol. 37, No. 9 2907 <br />  <br />  <br /> (4)] possess defective mitochondria (4). Since mito-        recognition AUG alone, prokaryotic counter- <br /> chondria considered prokaryotic origin, result      modi&#239;&#172;?ed restricted recognizing <br /> provides additional evidence modi&#239;&#172;?cation plays        AUG, suggesting modi&#239;&#172;?cation play an <br /> an essential role prokaryotic translation.                   important role selecting appropriate translational <br />    The t6A modi&#239;&#172;?cation shown determi-          start site. In absence t6A, scanning ribosome <br /> nant e&#239;&#172;&#402;cient aminoacylation bacterial Isoleucyl-         initiate natural ATG start codon, also <br /> tRNA synthetase (IleRS), yeast            promiscuously ATG non-ATG codons, as <br /> cytoplasmic IleRS (46,47) molecular       interactions initiator tRNAs codons <br /> basis requirement YrdC Sua5 bacteria          stringent. This account situa- <br /> and yeast mitochondria, respectively.                       tion prokaryotes, initiator tRNA mod- <br />    Four mitochondrial tRNAs contain t6A (Lys, Ile, Arg2         i&#239;&#172;?ed position 37 ribosome initiate either <br /> and Met) yrdC/SUA5 homolog                 ATG, GTG TTG (56). For yeast, lack stringent <br /> been identi&#239;&#172;?ed yeast, protein predicted       translational start site selection lead several <br /> be cytoplasmic (48). In absence mitochondrial          N-terminal variants, including active forms Cyc1 <br /> speci&#239;&#172;?c homolog, Sua5 targeted mito-        protein account similar levels Cyc1 <br /> chondria, possibly alternative translational start site      activity seen sua5::LEU2 strain the <br /> usage described proteins yeast          cyc1-1019 sua5-1 suppressor strain. <br /> Trm5, HTS1, VAS1 Arabidopsis thaliana alanyl-,                 Finally, pleiotropic phenotypes linked the <br /> threonyl- valyl-tRNA synthetases (49&#194;&#8212;52).                   absence YrdC/Sua5 members observed. In <br />    Strains lacking member YrdC/Sua5 family             E. coli, yrdC described possible ribosome <br /> resulting defective pool tRNAs, instrumen-      maturation factor (RimN) needed correct processing <br /> tal studying vivo role modi&#239;&#172;?cation.           16S ribosomal RNA, maintenance normal <br /> Accurate codon&#194;&#8212;anticodon recognition critical insur-      amounts translating ribosomes (7,8). The ribosome <br /> ing accuracy translation t6A37 modi&#239;&#172;?cation       maturation phenotype re&#239;&#172;&#8218;ect indirect consequence <br /> has shown a&#239;&#172;&#8364;ect process vitro, mainly        yrdC mutation tRNA defects loss of <br /> preventing formation U33-A37 across-the-loop base        modi&#239;&#172;?cations(s) cellular malfunctions been <br /> pairing interaction, allowing cross-strand stacking      shown cause ribosome maturation defects (57). <br /> A38 t6A37 &#239;&#172;?rst position codon (15,16,          In conclusion, propose diverse phenotypes <br /> 18&#194;&#8212;20). The t6A modi&#239;&#172;?cation seven E. coli       YrdC/Sua5 mutant strains central and <br /> 12 S. cerevisiae tRNAs, mitochondrial         primordial role t6A modi&#239;&#172;?cation includes: <br /> (38). As e&#239;&#172;&#8364;ects anticodon-surrounding modi&#239;&#172;?ca-           insuring accurate decoding, maintaining reading <br /> tions adjusting thermodynamic properties antic-       frame, selecting appropriate translation initiation <br /> odon/codon interactions cognate none-cognate            codon (mainly Eukaryotes) and, determining recogni- <br /> recognition ribosome context dependent,        tion aminoacylation tRNAs prokaryotic and <br /> expected absence t6A tRNAs a&#239;&#172;&#8364;ect      mitochondrial IleRSs. <br /> translation ways. This re&#239;&#172;&#8218;ected pleio-        Although biochemical function YrdC/Sua5 <br /> tropic phenotypes strains lacking functional YrdC       proteins (i.e. catalytic activity) aside tRNA and <br /> Sua5 (4&#194;&#8212;8). In yeast, example, sua5-1 mutation      ATP binding (this study) ATPase activity (41), has <br /> isolated suppressor initiation defect a&#239;&#172;&#8364;ecting   determined, appropriately modi&#239;&#172;?ed tRNA sub- <br /> CYC1 gene expression (4). CYC1 nuclear gene encod-         strates appear critical interacting proteins. <br /> ing iso-1 form electron carrier protein cyto-        We unable threonine binding YrdC, <br /> chrome c localized mitochondria (53).           ATP binding conditions, incorporation of <br /> The cycl-1019 mutant expresses 2% normal                 threonine tRNAThr transcript presence of <br /> amount iso-l-cytochrome c single base-pair          YrdC (data shown). Further experimentation using <br /> substitution creates upstream ATG codon         apomodi&#239;&#172;?ed tRNA substrate needed determine <br /> out frame normal CYCl translational start.          role YrdC t6A biosynthesis. It likely that <br /> The sua5-1 <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span>point mutation</span> suppressed translational         protein partners interact YrdC especially since <br /> defect, restoring Cyc1 levels 60% normal levels       biosynthesis t6A proposed require ATP- <br /> (4). In sua5 suppressor strain lacking t6A (this study),   dependent steps (58). <br /> Cyc1 protein expression possibly high levels of <br /> frameshift errors translation initiated the <br /> upstream frame ATG, restoring orig- <br />                                                                 ACKNOWLEDGEMENTS <br /> inal reading frame. The absence modi&#239;&#172;?cations at <br /> position 37 Wyeosine m1G shown         The authors like thank Michael Hampsey <br /> to drastically increase frame-shifting yeast (54,55).        strains plasmids, Sophie Alvares LC/MS/MS <br /> However, fact Cyc1 protein expressed       analysis, Iris Porat William Whitman M. maripa- <br /> at nearly wild-type levels wild-type CYC1               ludis genomic DNA, Darrell Davies t6A standard, <br /> sua5::LEU2 strain (4), i.e. absence aberrant      John Aris advice regarding yeast genetics, Henri <br /> ATG cyc1-1019 allele, suggests         Grosjean Paul Agris insightful discussions and <br /> hypothesis. In Eukaryotes, cytoplasmic initiator            Nemat Keyhani critical reading editing the <br /> tRNA contains t6A position 37, restricted        manuscript. <br />  2908 Nucleic Acids Research, 2009, Vol. 37, No. 9 <br />  <br />  <br /> FUNDING                                                                    16. Murphy,F.V., Ramakrishnan,V., Malkiewicz,A. Agris,P.F. <br />                                                                                (2004) The role modi&#239;&#172;?cations codon discrimination by <br /> National Science Foundation (grant MCB-05169448);                           tRNALysUUU. Nat. Struct. Mol. Biol., 11, 1186&#194;&#8212;1191. <br /> U.S. Department Energy (grant no. DE&#194;&#8212;FG02&#194;&#8212;                              17. Schweisguth,D.C. Moore,P.B. (1997) On conformation of <br /> 07ER64498) [to V.D.C.]; National Institutes Health                          anticodon loops initiator elongator methionine tRNAs. <br />                                                                                J. Mol. Biol., 267, 505&#194;&#8212;519. <br /> (GM-23562) [to P.S.]. Funding open access charge:                      18. Stuart,J.W., Gdaniec,Z., Guenther,R., Marszalek,M., Sochacka,E., <br /> U.S. Department Energy (grant no. DE&#194;&#8212;FG02&#194;&#8212;                                  Malkiewicz,A. Agris,P.F. (2000) Functional anticodon <br /> 07ER64498).                                                                    architecture human tRNALys3 includes disruption intraloop <br />                                                                                hydrogen bonding naturally occurring amino acid <br /> Con&#239;&#172;&#8218;ict statement. None declared.                                  modi&#239;&#172;?cation, t6A. Biochemistry, 39, 13396&#194;&#8212;13404. <br />                                                                            19. Weissenbach,J. Grosjean,H. (1981) E&#239;&#172;&#8364;ect threonylcarbamoyl <br />                                                                                modi&#239;&#172;?cation (t6A) yeast tRNA Arg III codon-anticodon and <br />                                                                                anticodon-anticodon interactions. A thermodynamic kinetic <br /> REFERENCES                                                                     evaluation. Eur. J. Biochem., 116, 207&#194;&#8212;213. <br />                                                                            20. Yarian,C., Marszalek,M., Sochacka,E., Malkiewicz,A., <br />  1. Galperin,M.Y. Koonin,E.V. (2004) &#226;&#8364;&#732;Conserved hypothetical&#226;&#8364;&#8482;              Guenther,R., Miskiewicz,A. Agris,P.F. (2000) Modi&#239;&#172;?ed <br />     proteins: prioritization targets experimental study. Nucleic        nucleoside dependent Watson-Crick wobble codon binding by <br />     Acids Res., 32, 5452&#194;&#8212;5463.                                                 tRNALysUUU species. Biochemistry, 39, 13390&#194;&#8212;13395. <br />  2. Teplova,M., Tereshko,V., Sanishvili,R., Joachimiak,A.,                 21. Yarian,C., Townsend,H., Czestkowski,W., Sochacka,E., <br />     Bushueva,T., Anderson,W.F. Egli,M. (2000) The structure                Malkiewicz,A.J., Guenther,R., Miskiewicz,A. Agris,P.F. (2002) <br />     yrdC gene product Escherichia coli reveals new               Accurate translation genetic <span id='am-3' about='Thesaurus:code' typeof='owl:Thing'>code</span> depends tRNA modi&#239;&#172;?ed <br />     fold suggests role RNA binding. Protein Sci., 9,                  nucleosides. J. Biol. Chem., 277, 16391&#194;&#8212;16395. <br />     2557&#194;&#8212;2566.                                                             22. Elkins,B.N. Keller,E.B. (1974) The enzymatic synthesis of <br />  3. Jia,J., Lunin,V.V., Sauve,V., Huang,L.W., Matte,A. Cygler,M.           N-(purin-6-ylcarbamoyl)threonine, anticodon-adjacent base in <br />     (2002) Crystal structure YciO protein Escherichia coli.        transfer ribonucleic acid. Biochemistry, 13, 4622&#194;&#8212;4628. <br />     Proteins, 49, 139&#194;&#8212;141.                                                 23. Overbeek,R., Begley,T., Butler,R.M., Choudhuri,J.V., <br />  4. Na,J.G., Pinto,I. Hampsey,M. (1992) Isolation                      Chuang,H.Y., Cohoon,M., Cre&#194;&#180;cy-Lagard,V., Diaz,N., Disz,T., <br />     characterization SUA5, novel gene required normal growth          Edwards,R. et al. (2005) The subsystems approach genome <br />     Saccharomyces cerevisiae. Genetics, 131, 791&#194;&#8212;801.                       annotation use project annotate 1000 genomes. <br />  5. Chen,J., Ji,C., Gu,S., Zhao,E., Dai,J., Huang,L., Qian,J., Ying,K.,        Nucleic Acids Res., 33, 5691&#194;&#8212;5702. <br />     Xie,Y. Mao,Y. (2003) Isolation identi&#239;&#172;?cation novel        24. Altschul,S.F., Madden,T.L., Scha&#239;&#172;&#8364;er,A.A., Zhang,J., Zhang,Z., <br />     cDNA encodes human YrdC protein. J. Hum. Genet., 48,                  Miller,W. Lipman,D.J. (1997) Gapped BLAST PSI- <br />     164&#194;&#8212;169.                                                                   BLAST: new generation protein database search programs. <br />  6. Jiang,W., Prokopenko,O., Wong,L., Inouye,M.                            Nucleic Acids Res., 25, 3389&#194;&#8212;3402. <br />     Mirochnitchenko,O. (2005) IRIP, new ischemia/reperfusion-            25. Chenna,R., Sugawara,H., Koike,T., Lopez,R., Gibson,T.J., <br />     inducible protein participates regulation transporter       Higgins,D.G. Thompson,J.D. (2003) Multiple sequence <br />     activity. Mol. Cell Biol., 25, 6496&#194;&#8212;6508.                                  alignment Clustal series programs. Nucleic Acids Res., <br />  7. Kaczanowska,M. Ryden-Aulin,M. (2005) The YrdC protein &#194;&#8212;                31, 3497&#194;&#8212;3500. <br />     putative ribosome maturation factor. Biochim. Biophys. Acta,         26. Miller,J.H. (1972) Experiments Molecular Genetics. Cold Spring <br />     1727, 87&#194;&#8212;96.                                                               Harbor Laboratory Press, Cold Spring Harbor, NY. <br />  8. Kaczanowska,M. Ryden-Aulin,M. (2004) Temperature                   27. Guzman,L.M., Belin,D., Carson,M.J. Beckwith,J. (1995) Tight <br />     sensitivity caused mutant release factor 1 suppressed             regulation, modulation, high-level expression vectors <br />     mutations a&#239;&#172;&#8364;ect 16S rRNA maturation. J. Bacteriol., 186,              containing arabinose PBAD promoter. J. Bacteriol., 177, <br />     3046&#194;&#8212;3055.                                                                 4121&#194;&#8212;4130. <br />  9. Baba,T., Ara,T., Hasegawa,M., Takai,Y., Okumura,Y., Baba,M.,           28. Sikorski,R.S. Hieter,P. (1989) A shuttle vectors and <br />     Datsenko,K.A., Tomita,M., Wanner,B.L. Mori,H. (2006)                   yeast host strains designed e&#239;&#172;&#402;cient manipulation DNA in <br />     Construction Escherichia coli K-12 in-frame, single-gene                Saccharomyces cerevisiae. Genetics, 122, 19&#194;&#8212;27. <br />     knockout mutants: Keio collection. Mol. Syst. Biol., 2,            29. Datsenko,K.A. Wanner,B.L. (2000) One-step inactivation of <br />     2006.0008.                                                                 chromosomal genes Escherichia coli K-12 using PCR products. <br /> 10. Serina,S., Nozza,F., Nicastro,G., Faggioni,F., Mottl,H., Deho,G.           Proc. Natl Acad. Sci. USA, 97, 6640&#194;&#8212;6645. <br />     Polissi,A. (2004) Scanning Escherichia coli chromosome      30. Yu,D., Ellis,H.M., Lee,E.C., Jenkins,N.A., Copeland,N.G. and <br />     random transposon mutagenesis multiple phenotypic screening.           Court,D.L. (2000) An e&#239;&#172;&#402;cient recombination for <br />     Res. Microbiol., 155, 692.                                                 chromosome engineering Escherichia coli. Proc. Natl Acad. Sci. <br /> 11. Giaever,G., Chu,A.M., Ni,L., Connelly,C., Riles,L., Veronneau,S.,          USA, 97, 5978&#194;&#8212;5983. <br />     Dow,S., Lucau-Danila,A., Anderson,K., Andre,B. et al. (2002)           31. Boeke,J.D., LaCroute,F. Fink,G.R. (1984) A positive selection <br />     Functional pro&#239;&#172;?ling Saccharomyces cerevisiae genome.                mutants lacking orotidine-50 -phosphate decarboxylase activity in <br />     Nature, 418, 387&#194;&#8212;391.                                                      yeast: 5-&#239;&#172;&#8218;uoro-orotic acid resistance. Mol. Gen. Genet., 197, <br /> 12. Ashraf,S.S., Guenther,R., Ye,W., Lee,Y., Malkiewicz,A.                 345&#194;&#8212;346. <br />     Agris,P.F. (1997) Ribosomal binding modi&#239;&#172;?ed tRNA anticodons         32. Pomerantz,S.C. McCloskey,J.A. (1990) Analysis RNA <br />     related thermal stability. Nucleic Acids Symp. Ser., 36, 58&#194;&#8212;60.         hydrolyzates liquid chromatography-mass spectrometry. <br /> 13. Basavappa,R. Sigler,P.B. (1991) The 3 A&#203;&#353; crystal structure          Methods Enzymol., 193, 796&#194;&#8212;824. <br />     yeast initiator tRNA: functional implications initiator/elongator   33. Waas,W.F. Schimmel,P. (2007) Evidence tRNA <br />     discrimination. EMBO J., 10, 3105&#194;&#8212;3111.                                    synthetase-directed proton transfer stops mistranslation. <br /> 14. Durant,P.C., Bajji,A.C., Sundaram,M., Kumar,R.K.                       Biochemistry, 46, 12062&#194;&#8212;12070. <br />     Davis,D.R. (2005) Structural e&#239;&#172;&#8364;ects hypermodi&#239;&#172;?ed nucleosides     34. Shields,T.P., Mollova,E., Ste Marie,L., Hansen,M.R. Pardi,A. <br />     Escherichia coli human tRNALys anticodon loop: e&#239;&#172;&#8364;ect           (1999) High-performance liquid chromatography puri&#239;&#172;?cation of <br />     nucleosides s2U, mcm5U, mcm5s2U, mnm5s2U, t6A, ms2t6A.              homogenous-<span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span> RNA produced trans cleavage a <br />     Biochemistry, 44, 8078&#194;&#8212;8089.                                               hammerhead ribozyme. RNA, 5, 1259&#194;&#8212;1267. <br /> 15. Lescrinier,E., Nauwelaerts,K., Zanier,K., Poesen,K., Sattler,M.    35. Vieira,J. Messing,J. (1982) The pUC plasmids, an <br />     Herdewijn,P. (2006) The naturally occurring N6-threonyl adenine         M13mp7-derived insertion mutagenesis sequencing <br />     anticodon loop Schizosaccharomyces pombe tRNAi causes                   synthetic universal primers. Gene, 19, 259&#194;&#8212;268. <br />     formation unique U-turn motif. Nucleic Acids Res., 34,            36. Kurganov,B.I., Sugrobova,N.P. Yakovlev,V.A. (1972) <br />     2878&#194;&#8212;2886.                                                                 Estimation dissociation constant enzyme-ligand complex <br />                                                                                       Nucleic Acids Research, 2009, Vol. 37, No. 9 2909 <br />  <br />  <br />     &#239;&#172;&#8218;uorometric data &#226;&#8364;&#732;&#226;&#8364;&#732;di&#239;&#172;&#8364;erence&#226;&#8364;&#8482;&#226;&#8364;&#8482; method. FEBS Lett., 19,         49. Chatton,B., Walter,P., Ebel,J.P., Lacroute,F. Fasiolo,F. (1988) <br />     308&#194;&#8212;310.                                                                   The yeast VAS1 gene encodes mitochondrial cytoplasmic <br /> 37. Mayer,M. Bernd,M. (1999) Characterization ligand binding            valyl-tRNA synthetases. J. Biol. Chem., 263, 52&#194;&#8212;57. <br />     saturation transfer di&#239;&#172;&#8364;erence NMR spectroscopy. Angew. Chem.,       50. Lee,C., Kramer,G., Graham,D.E. Appling,D.R. (2007) Yeast <br />     Int. Ed., 38, 1784&#194;&#8212;1788.                                                   mitochondrial initiator tRNA methylated guanosine 37 the <br /> 38. Sprinzl,M. Vassilenko,K.S. (2005) Compilation tRNA                  Trm5-encoded tRNA (Guanine-N1-)-methyltransferase. J. Biol. <br /> </body></html>