<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Published online 20 July 2009                                                Nucleic Acids Research, 2009, Vol. 37, No. 17 5757&#226;&#8364;&#8220;5767 <br />                                                                                                                 doi:10.1093/nar/gkp568 <br />  <br />  <br /> High-throughput sequencing reveals suppressors <br /> of Vibrio cholerae rpoE mutations: fewer <br /> porin enough <br /> Brigid M. Davis* Matthew K. Waldor <br /> Channing Laboratory, Brigham Women&#226;&#8364;&#8482;s Hospital, Harvard Medical School HHMI, 181 Longwood <br /> Avenue, Boston, MA, USA <br />  <br /> Received April 22, 2009; Revised Accepted June 19, 2009 <br />  <br />  <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span>                                                                        bacterial inner membrane antisigma factor <br />                                                                                 RseA, interact RNA polymerase <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> <br /> Analyses suppressor mutations                                      direct transcription sE-dependent promoters (2,3). <br /> extremely valuable understanding gene function.                              However, misfolded outer membrane proteins (OMPs) <br /> However, techniques mapping suppressor muta-                                accumulate periplasm, trigger degra- <br /> tions available bacterial species.                             dation RseA release sE, allowing <br /> Here, used high-throughput sequencing technol-                               increased expression sE regulon, includes <br /> ogy identify spontaneously arising suppressor                                genes encoding periplasmic proteases, foldases cha- <br /> mutations enabled disruption rpoE (which                                perones aid periplasmic folding, rpoE <br /> encodes pE) Vibrio cholerae, agent cholera.                           itself (4&#226;&#8364;&#8220;6). Activation sE induces transcription <br /> The alternative sigma factor pE, activated                             small non-coding RNAs (sRNAs) repress <br /> by envelope stress, promotes expression factors                              production OMPs (7). Heightened expression these <br /> that help preserve and/or restore cell envelope                                 factors thought aid elimination inducing <br /> integrity. In Escherichia coli, rpoE essential                            stimulus restoration envelope homeostasis. sE <br /> gene disrupted presence                              activity induced cells enter stationary phase. <br />                                                                                 In Escherichia coli, response shown be <br /> additional suppressor mutations. Among panel <br />                                                                                 dependent alarmone ppGpp, typically <br /> of independent V. cholerae rpoE mutants,                                   accumulates nutrients limited (8,9). <br /> than 75% contain suppressor mutations                                         sE classically thought regulating stress <br /> reduce production OmpU, V. cholerae&#226;&#8364;&#8482;s principal                              response; however, essential for <br /> outer membrane porin. OmpU appears key                                  viability E. coli Yersinia enterocolitica, suggesting <br /> determinant V. cholerae&#226;&#8364;&#8482;s requirement                                plays fundamental role bacterial physi- <br /> production pE. Such dependence single                                 ology (10,11). sE initially recognized to <br /> factor contrasts markedly regulation pE                              essential E. coli, suppressor mutations <br /> E. coli, numerous factors contribute                            mask <span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span>phenotype</span> growth (or <br /> activation dominant. We identified                             lack thereof) E. coli rpoE mutants highly dependent <br /> a suppressor <span id='am-317' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-318' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-319' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-320' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-321' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-322' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-323' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-324' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-325' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-326' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-327' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-328' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-330' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-331' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-332' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-333' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-334' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-335' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-337' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-338' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-341' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-342' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-344' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-347' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-348' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-349' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-350' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-351' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-352' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-353' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-354' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-355' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-356' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-357' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-358' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-359' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-362' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-363' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-364' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-365' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-366' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-367' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-368' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-369' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-370' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-371' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-372' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-373' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-374' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-375' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-376' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-378' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-379' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-380' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-381' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-382' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-383' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-384' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-385' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-386' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-389' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-390' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-391' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-392' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-393' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-394' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span>mutation</span> differs pre-                                selective criteria employed. However, com- <br /> viously described suppressors elevates,                              parisons independently derived E. coli rpoE mutants <br /> rather reduces, pE&#226;&#8364;&#8482;s activity. Finally, analyses                           suggested strains contain suppressor <br /> of panel rpoE mutants shed light                                    mutations, suppressors identi&#239;&#172;?ed <br />                                                                                 date (12,13). These include loss function mutation <br /> mechanisms suppressor mutations may <br />                                                                                 ydcQ, encodes putative DNA-binding protein, <br /> arise V. cholerae.                                                           multicopy plasmids containing ptsN yhbW. Little is <br />                                                                                 known mechanisms suppressors <br />                                                                                 function; however, observed the <br /> INTRODUCTION                                                                    suppressors reduce basal activity envelope stress <br /> The alternative sigma factor sE (also known RpoE)                         response pathways well. Somewhat surprisingly, <br /> one regulators enable bacteria respond                       suppressors counteract E. coli&#226;&#8364;&#8482;s requirement the <br /> to perturbation cell envelope (1). In unstressed                         proteases RseP DegS, needed degrada- <br /> growing cells, sE activity relatively low, tethered                 tion RseA activation sE response to <br />  <br /> *To correspondence addressed. Tel: 617 525 4655; Fax: 617 525 4660; Email: bdavis@rics.bwh.harvard.edu <br />  <br /> &#195;&#376; 2009 The Author(s) <br /> This Open Access article distributed terms Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/ <br /> by-nc/2.0/uk/) permits unrestricted non-commercial use, distribution, reproduction medium, provided original work properly cited. <br />  5758 Nucleic Acids Research, 2009, Vol. 37, No. 17 <br />  <br />  <br /> periplasmic stresses, did function suppressors       sequencing technology carrying genomics-based <br /> for rpoE mutations. For example, production              approach suppressor genetics. <br /> OmpA OmpC reduced, deletion both <br /> genes overexpression RseX, sRNA neg- <br /> atively regulates them, degS rseP longer essen-       MATERIALS AND METHODS <br /> tial; however, rpoE required (14). This re&#239;&#172;&#8218;ect      Bacterial strains culture conditions <br /> the fact pathways shown activate <br /> sE E. coli.                                                   All V. cholerae strains generated study are <br />    Studies rpoE V. cholerae, etiologic agent           derivatives sequenced clinical isolate N16961 and <br /> of cholera, directly addressed rpoE          resistant streptomycin. For analysis rpoE pro- <br /> essential rpoE mutants contain suppressor             duction V. cholerae lacking various OMPs, transposon <br /> mutations. As early studies rpoE E. coli,         insertion mutants derived C6706 used (20). <br /> ability generate rpoE mutants led             A related mutant insertion vca0199 also <br /> assumption gene required, consid-        used, KnR transposon present <br /> eration possibility suppressors. In V. cholerae,       (BD2034) majority transposon excised <br /> rpoE shown required intestinal coloni-        &#239;&#172;&#8218;ippase (BD2041). Transposons excised <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> zation animal model disease resistance       subset C6706 OMP mutants, in <br /> an antimicrobial peptide, likely members reg-         transposon &#226;&#8364;&#732;scar&#226;&#8364;&#8482; remaining excision con- <br /> ulon identi&#239;&#172;?ed (15&#226;&#8364;&#8220;17). Furthermore,           tained stop codon prevent production full <br /> found expression sE reduced absence            <span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> gene product. Removal transposon, which <br /> of OmpU, abundant outer membrane protein                   contains lacZ, necessary prior monitoring <br /> V. cholerae expression controlled key vir-       rpoE::lacZ activity. C6706  toxRS obtained from <br /> ulence regulator ToxR (18). However, aspects         JJ Mekalanos. Escherichia coli strains DH5a pir and <br /> biology rpoE mutants remain explored, phe-          SM10 pir used cloning conjugation sui- <br /> notypic comparisons independently derived mutants             cide vectors, respectively. Bacteria grown LB at <br /> have reported.                                          378C unless noted, using antibiotics fol- <br />    Identi&#239;&#172;?cation suppressor mutations hampered         lowing concentrations: streptomycin, 200 mg/ml; carbeni- <br /> by lack scorable/selectable phenotypes and/or tools      cillin, 50 mg/ml; kanamycin, 50 mg/ml. Negative selection <br /> for genetic mapping. In V. cholerae, currently      strains containing sacB performed plates <br /> robust genetic tools mapping spontaneous suppressor          containing 10% sucrose 308C. LB plates containing <br /> mutations. However, recent development massively          0.25% SDS used assess sensitivity rpoE <br /> parallel sequencing technology provides potential path         mutants detergents. <br /> around experimental barriers. This technology <br />                                                                  Strain plasmid construction <br /> enables generation high coverage genomic sequence <br /> data cost permits analysis multiple samples,        Strains insertion mutations rpoE generated <br /> and allows direct identi&#239;&#172;?cation di&#239;&#172;&#8364;erences          using pBD1574, derivative suicide vector pGP704 <br /> between DNA sequences related strains. The                contains 424 bp internal fragment rpoE (21). <br /> approach ideally suited unbiased identi&#239;&#172;?cation         Integration vector, single crossover event <br /> genetic changes, particularly multiple changes           rpoE-derived sequence, results gene disrup- <br /> interdependent e&#239;&#172;&#8364;ects present single strain         tion confers carbenicillin resistance. Exconjugates <br /> [e.g. observed (19)], e&#239;&#172;&#8364;ects           selected plates containing streptomycin and <br /> missed phenotypic assays individual changes.               carbenicillin. When primary exconjugates gave rise to <br />    We generated characterized set V. cholerae           colonies varied sizes restreaking, occurred <br /> rpoE mutants displayed range               90% exconjugants, large colonies were <br /> of phenotypes, consistent possibility         selected analysis, order minimize analyses <br /> might contain distinct suppressor mutations. Using high-         mixed populations. Strains insertion mutations in <br /> throughput sequencing, identi&#239;&#172;?ed di&#239;&#172;&#8364;erences           ompS ompT generated using pBD1664 and <br /> between wt rpoE strains subsequently con-          pBD1665, contain internal gene fragments 451 <br /> &#239;&#172;?rmed suppressor mutations. Remarkably,                382 bp, respectively. <br /> majority strains lacked sE contained additional             All allele replacements generated using vectors <br /> mutations reduced expression V. cholerae&#226;&#8364;&#8482;s major         derived suicide vector pCVD442, using standard <br /> OMP, OmpU. An ompU-independent suppression path-                 allele exchange procedures (22,23). In brief, targeting <br /> way identi&#239;&#172;?ed, which, unlike rpoE suppres-        vector, confers carbenicillin resistance, trans- <br /> sors, elevated basal activity envelope stress        ferred conjugation V. cholerae recipient, it <br /> response pathway. Overall, data suggest key           integrated. Primary exconjugants (selected plates con- <br /> role sE respond endogenously generated           taining streptomycin carbenicillin) sub- <br /> stimulus transmitted OmpU. They shed light          jected counterselection sucrose plates identify <br /> the mechanisms used V. cholerae maintain appropri-         strains longer contained plasmid backbone, <br /> ate OMP expression, mechanisms                   screened PCR identify strains contain- <br /> which suppressor mutations arise organism.           ing mutation interest. Strains containing deletions <br /> Finally, results illustrate utility high-throughput   rpoE generated using pBD1578, which <br />                                                                       Nucleic Acids Research, 2009, Vol. 37, No. 17 5759 <br />  <br />  <br /> introduces 342-bp deletion. Strains rpoE       mutants, con&#239;&#172;?rmed Sanger sequencing, <br /> replaced rpoE::aph allele generated using          presented Supplementary Table 2. <br /> pBD1999, rpoE interrupted insertion of <br /> an aph gene cassette pUC71K. Mutants selected       b-galactosidase assays <br /> on plates containing kanamycin sucrose.            Reporter plasmids transformed strains made <br /> (Primary selection plates containing sucrose alone,         lacZ using plasmid pJL1. Assays performed <br /> followed patching plates containing kanamycin,          basically described (30). <br /> led similar conclusions.) Point mutations vca0199, <br /> vc2413 ompU (vc0633) promoter 50 UTR              Western blots <br /> were introduced using plasmids pBD2038, pBD2036, <br /> pBD2049 pBD2088, respectively. Deletions ompU          Protein samples generated overnight cultures of <br /> and ompT introduced using previously described       V. cholerae unless indicated. Culture volumes <br /> vectors pKEK276 pKEK309 (24). Deletion vc0972          normalized based OD600, cells pelleted <br /> was performed using pBD1900.                                  lysed 1  NuPAGE LDS sample bu&#239;&#172;&#8364;er (Invitrogen) <br />    All ompU::lacZ transcription reporter fusions deri-    containing 5% b-mercaptoethanol 958C. Samples were <br /> vatives pCB192N, contains promoterless lacZ        run NuPAGE Bis&#226;&#8364;&#8220;Tris gels (Invitrogen) trans- <br /> cassette (H. Kimsey, manuscript preparation). Each         ferred nitrocellulose membranes according the <br /> contains ompU-linked sequences extending position        manufacturer&#226;&#8364;&#8482;s instructions. Membranes probed <br /> &#226;&#8364;&#8220;411 +227 relative ompU transcriptional start       polyclonal antisera sE, OmpU, OmpT, OmpA, <br /> site (25). The rpoE::lacZ transcription reporter fusion       commercially available anti 6-His antibody <br /> is derivative pCB192 (26) contains sequences         (Genetex). Horseradish peroxidase&#226;&#8364;&#8220;conjugated secondary <br /> from &#226;&#8364;&#8220;91 +112 relative sE dependent transcrip-      antibodies Supersignal West Pico chemiluminescent <br /> tional start site, de&#239;&#172;?ned (16).                         substrate kit (Pierce) used detection bound <br />    Additional details regarding plasmid strain con-       primary antibodies. <br /> struction available request. The sequences of <br /> plasmids chromosomal loci used           Northern blots <br /> alter con&#239;&#172;?rmed Sanger sequencing.                     RNA harvested log phase cultures unless other- <br />                                                               wise noted. RNA extracted cell pellets using <br /> Genome sequencing analysis                                Trizol (Invitrogen), treated DNase I (Qiagen). <br />                                                               It electrophoresed glyoxal gels (Ambion), then <br /> Genome sequencing performed Partners               transferred Bright Star Plus nylon membranes <br /> Healthcare Center Genetics Genomics using          (Ambion). RNA integrity con&#239;&#172;?rmed assessment <br /> Illumina Genome Analyzer. Sequence data analyzed          EtBr-stained rRNA bands. Blots hybridized to <br /> using MAQ (http://maq.sourceforge.net/index.shtml) (27)       32 <br />                                                                 P-labeled vitro transcribed probes ULTRAhyb <br /> and Edena (http://www.genomic.ch/edena.php, (28)).            (Ambion) 688C washed according manufac- <br /> MAQ primarily used identi&#239;&#172;?cation single            turer&#226;&#8364;&#8482;s instructions. <br /> nucleotide polymorphisms (SNPs), enabled detec- <br /> tion single-nucleotide insertions deletions in <br /> regions extended mononucleotide tracts. The consen-      RESULTS <br /> sus sequence obtained rpoE mutants compared <br /> to control sequence wt progenitor       Isolation V. cholerae rpoE mutants <br /> published V. cholerae N16961 sequence, enabling       We previously isolated rpoE mutants of <br /> identi&#239;&#172;?cation mutant-speci&#239;&#172;?c SNPs SNPs dis-       V. cholerae; thus, clear absol- <br /> tinguishing isolate published genome (29).       ute barrier obtaining rpoE mutants organism <br /> MAQ reports candidate SNP consensus qual-         (16,17). However, strategies generating <br /> ity score re&#239;&#172;&#8218;ects trustworthiness SNP pre-    rpoE mutants failed, have <br /> diction. Filtered SNPs consensus quality scores lower    low frequency success. Furthermore, noted <br /> than 50 generally deemed unreliable.                     published mutants did share pheno- <br />    De novo sequence assembly, using Edena, performed      types (e.g. EtOH sensitivity), suggesting they <br /> on pooled data sequenced samples, using default      acquired distinct suppressor mutations. <br /> parameters. Assembled contigs compared           Thus, V. cholerae rpoE previously <br /> the published N16961 genome using MUMmer                   reported essential, undertook systematic <br /> identify SNPs, insertions, deletions (http://mummer       characterization V. cholerae rpoE mutants their <br /> .sourceforge.net; Supplementary Table 1). Comparable          potential suppressor mutations. <br /> analyses using sequencing results individual samples        We &#239;&#172;?rst assessed frequency obtaining insertion <br /> did yield genome-wide results, depth sequence   mutants following conjugation suicide vectors contain- <br /> coverage individual samples su&#239;&#172;&#402;cient      ing internal fragment rpoE control gene <br /> to permit contig assembly; however, subset di&#239;&#172;&#8364;erences    (ompS ompT), selectable marker bla. <br /> could identi&#239;&#172;?ed.                                           Carbenicillin resistant (carbR) mutants rpoE, ompS <br />    The precise locations sequences di&#239;&#172;&#8364;er      ompT obtained approximately equal <br /> the published N16961 genome individual rpoE               frequencies, suggesting rpoE mutants could <br />  5760 Nucleic Acids Research, 2009, Vol. 37, No. 17 <br />  <br />  <br />                                                                           performed Solexa Genome Analyzer. This <br />                                                                           approach yielded average 2.8 million 36-nt sequence <br />                                                                           reads sample (range 1.2&#226;&#8364;&#8220;4.0 million). Sequences <br />                                                                           subsequently mapped published V. cholerae <br />                                                                           N16961 genome sequence (29) using freely available <br />                                                                           software MAQ [Mapping Assembly Qualities; <br />                                                                           http://maq.sourceforge.net/index.shtml (27)]. MAQ uses <br />                                                                           Solexa nucleotide sequences associated qual- <br />                                                                           ity scores (which re&#239;&#172;&#8218;ect con&#239;&#172;?dence each <br />                                                                           nucleotide 36-mer called Genome <br />                                                                           Analyzer) predict SNPs, positions sequenced <br />                                                                           sample di&#239;&#172;&#8364;ers reference genome. MAQ cal- <br />                                                                           culates consensus quality score potential <br />                                                                           SNPs. An average 88% Solexa sequences could <br />                                                                           mapped reference genome. Numerous potential <br />                                                                           SNPs identi&#239;&#172;?ed; however, identi&#239;&#172;?ed as <br />                                                                           candidate SNPs wt rpoE mutant samples <br />                                                                           (and correspond di&#239;&#172;&#8364;erences strain <br />                                                                           published sequence) relatively low consen- <br />                                                                           sus quality scores. Based visual inspection sequence <br />                                                                           alignments (using MAQ&#226;&#8364;&#8482;s companion program, <br /> Figure 1. An rpoE insertion mutant restreaked plates containing Sm     MAQview) Sanger sequencing candi- <br /> and Carb following primary selection. The mutant derivative   date loci, determined candidate SNPs con- <br /> wt V. cholerae N16961. Both large (boxed) small (circled) colonies    sensus quality scores 50 likely false. <br /> are detectable. <br />                                                                           Removing loci list mutant-speci&#239;&#172;?c can- <br />                                                                           didates left relatively small number candidates, suit- <br />                                                                           able con&#239;&#172;?rmation using Sanger sequencing and/or <br /> arise absence suppressor mutations.                         elimination visual inspection aligned reads. <br /> However, noticed initial selection rpoE                   Through process veri&#239;&#172;?ed presence &#239;&#172;?ve can- <br /> mutants routinely yielded smaller colonies                   didate suppressor mutations, located vca0199 and <br /> control genes, restreaking colonies                   vc2413-coding regions (one each, strains BD1965 and <br /> typically yielded mixture large small colonies                   BD1963 respectively) intergenic region between <br /> (Figure 1). Further characterization mutants                       vc0633 vc0634 (three distinct mutations, strains <br /> revealed grouped distinct classes.                BD1962, BD1956 BD1987). The e&#239;&#172;&#8364;ects muta- <br /> Mutants distinguished levels OMP                     tions described below. <br /> expression, colony size morphology, their <br /> ability grow plates containing detergents. Together, <br />                                                                           Comparison newly sequenced isolates published <br /> these results suggested carbR rpoE insertion                     V. cholerae genome <br /> mutants frequently acquired several <br /> additional mutations suppressed small colony                     Solexa reads used inputs novo <br /> phenotype rpoE mutant enabled rapid                        assembly program Edena [Exact DE Novo Assembler <br /> growth.                                                                   (28)], generated contigs searched <br />   Derivatives wt V. cholerae rpoE deleted,                di&#239;&#172;&#8364;erences relative published V. cholerae <br /> using targeting vector conferring carbR sucrose                     N16961 sequence using MUMmer. The advantage of <br /> sensitivity, obtained readily rpoE                    approach performs gapped as <br /> insertion mutants. Identi&#239;&#172;?cation deletion mutants                      ungapped alignments, used identify <br /> required biased screening sucrose resistant colonies,                small insertions deletions, generally be <br /> namely preferential testing small di&#239;&#172;&#8364;erentially col-                detected data set using MAQ. Disadvantages are <br /> ored colonies. Mutants rpoE replaced                      does utilize Solexa quality scores pro- <br /> rpoE::aph obtained. Thus, rpoE appeared                      vide means identify high quality versus low quality <br /> to essential V. cholerae growth selection                candidate SNPs, requires greater number of <br /> conditions rpoE::aph.                                                 sequences generate consensus, we <br />                                                                           obtained single sample. Edena con&#239;&#172;?rmed pres- <br />                                                                           ence of candidate suppressors identi&#239;&#172;?ed using <br /> Identification candidate suppressor mutations <br />                                                                           MAQ; however, principal utility identi&#239;&#172;?cation <br /> To identify putative suppressor mutations                          di&#239;&#172;&#8364;erences starting strain refer- <br /> V. cholerae rpoE mutants, took advantage newly                      ence strain, pooled Solexa sequences <br /> developed massively parallel sequencing technology.                       generate Edena input. With comprehensive set <br /> Whole genome sequencing DNA subset inser-                    sequences, contigs spanning 3.9 million bases, or <br /> tion mutants deletion mutants wt                      &gt;88% V. cholerae genome, assembled, <br /> V. cholerae N16961, mutants derived,                  145 di&#239;&#172;&#8364;erences strains reference <br />                                                                                         Nucleic Acids Research, 2009, Vol. 37, No. 17 5761 <br />  <br />  <br />           A <br />                                                    delete 1A A9: 1956, 1966, 1967, 1968, 1969, 1996               delete 1A A8: 1960, <br />                                                                             insert 1A A9: 1970, 1987                   1964, 1972, 1997 <br />  <br />  <br />                                                     #                                                # <br />         &lt;--- greA&#226;&#8364;&#8482; <br />         (vc0634) <br />  <br />  <br />                                                                    ompU tx start <br />               #                                             #                                                # <br />                                                                                                                                   ompU&#226;&#8364;&#8482; ---&gt; <br />             insert T: 1995         T&gt;A: 1971     insert A: 1973                 insert A: 1974    G&gt;T: 1962                        (vc0633) <br />  <br />  <br />             Suppressors rpoE mutants lacking ompU-linked mutations <br />             1957: potential suppressor vca0199              1961: unknown <br />             1965: confirmed suppressor vca0199              1963: unknown <br />  <br />  <br />  <br />             B                                                                         C <br />                      rpoE mutants ompU-     rpoE muts w/o ompU- <br />                      linked suppressor mutations linked mutations <br />                   &#206;&#8221;ompU <br />                     1956 <br />                     1960 <br />                     1962 <br />                     1966 <br />                     1970 <br />                     1971 <br />                     1973 <br />                     1974 <br />                     1995 <br />                     1957 <br />                     1961 <br />                     1963 <br />                     1965 <br />                       wt <br />  <br />  <br />  <br />  <br />                                                                      OmpU <br />  <br />  <br />                                                                      OmpA <br />  <br />  <br />                                                                      OmpT <br />  <br />  <br />  <br /> Figure 2. Locations rpoE suppressor mutations e&#239;&#172;&#8364;ects OMP levels promoter activity. (A) Schematic depiction locations of <br /> suppressor mutations rpoE mutants. The ompU promoter region drawn scale; hash marks denote 100-bp intervals. The 50 ends coding <br /> sequences &#239;&#172;&#8218;anking genes shown black boxes; ompU-coding sequence corresponds shown (25) published <br /> genome sequence. Binding sites ToxR shown grey rectangles de&#239;&#172;?ned (32). Circles represent sites suppressor mutations, and <br /> numbers (e.g. 1956) correspond strain numbers corresponding rpoE mutants. rpoE deletion mutants underlined; strains <br /> are insertion mutants. In cases, mutants independently derived, despite detection identical suppressors multiple strains. Only pairs <br /> 1966/1967 1968/1969 potentially consist siblings. The precise positions (relative published N16961 genome) SNPs <br /> linked suppression rpoE mutations listed Supplementary Table 2. (B) Western blots showing expression OmpU, OmpA OmpT <br /> levels cell lysates wt,  ompU, non-redundant subset rpoE mutants. (C) b-galactosidase activity ompU::lacZ transcrip- <br /> tion reporter fusions generated wt V. cholerae indicated rpoE mutants. Activities shown averages +SEM independent <br /> cultures generated overnight cultures; log phase cultures yielded similar relative activities. <br />  <br />  <br />  <br /> strain identi&#239;&#172;?ed (Supplementary Table 1). These                      Many rpoE mutants contain mutations upstream <br /> included 40 SNPs, predicted                       ompU reduce expression <br /> by MAQ 79 small insertions deletions. A subset                         The intergenic sequence vc0633 vc0634, <br /> of correspond loci identi&#239;&#172;?ed recent study                       contain independently derived <br /> as likely errors published genomic sequence                        candidate suppressor mutations, contains promoters <br /> (31). An approximate position inser-                          genes. Notably, vc0633 encodes OmpU, which <br /> tions/deletions obtained performing                          previously shown determinant sE levels <br /> BLAST analyses sequences                                 (18). Given frequency mutations obtained this <br /> mapped MAQ. This approach utilized                             region knowledge concerning gene&#226;&#8364;&#8482;s role, <br /> if sequence depth does meet stringent conditions                      resequenced region rpoE mutants. <br /> required Edena, used search                            Remarkably, analyses revealed 17 21 rpoE <br /> for insertions deletions sequence individ-                     mutants contained mutations region (Figure 2A). <br /> ual sample pooled samples. However,                                Identical mutations detected multiple strains, <br /> BLAST analyses did identify insertions dele-                       known arisen indepen- <br /> tions rpoE mutants present                       dently. Most mutations con&#239;&#172;?rmed <br /> our wt strain.                                                                binding sites ToxR, activates transcription <br />  5762 Nucleic Acids Research, 2009, Vol. 37, No. 17 <br />  <br />  <br /> of ompU; however, mutations       Table 1. Frequency obtaining rpoE::aph mutants sucrose/ <br /> promoter, instead lie ompU&#226;&#8364;&#8482;s 50 UTR                  kanamycin selection <br /> (21,25,32). OmpU-coding sequences examined <br />                                                                 Parental       Mutation                rpoE mutanta       Percentage <br /> for strains, mutations detected.        strain                                                    rpoE::aph (n) <br /> Western blots showed strains ompU- <br /> linked mutations contained reduced amounts OmpU,             N16961         NA                      &#226;&#8364;&#8220;                   0   (85) <br /> although levels varied widely (Figure 2B). No muta-         BD2046         vc2413                  BD1963              0   (49) <br /> tions region identi&#239;&#172;?ed strains     BD2048         vca0199 (pt. mut.)      BD1965             91   (102) <br />                                                                 BD2064         ompU 50 UTR             BD1962             83   (31) <br /> contain candidate suppressor mutations loci       BD2096         ompU promoter           BD1956             95   (68) <br /> ompU, OmpU levels rpoE strains lacking ompU              MKW156          ompU                   &#226;&#8364;&#8220;                  96   (46) <br /> mutations equal wt V. cholerae.                JM3-3           ompT                   &#226;&#8364;&#8220;                   0   (33) <br />    To con&#239;&#172;?rm ompU-linked mutations respon-         BD1902          vc0972                 &#226;&#8364;&#8220;                   0   (45) <br />                                                                 BD2139          vca0199                &#226;&#8364;&#8220;                  45   (65) <br /> sible reduced OmpU levels, generated ompU::lacZ          C6706          NA                      &#226;&#8364;&#8220;                   0   (48) <br /> transcription reporter fusions containing wt mutant         BD2041b        vca0199, vca0200                            0   (52) <br /> promoter/UTR sequences, assessed activity wt       BD2093b         toxRS                  &#226;&#8364;&#8220;                  94   (54) <br /> V. cholerae. All mutant promoter fusions yielded at <br /> least 2-fold activity, did reporter fusion contain-   NA, applicable; n, number colonies tested carbenicillin <br />                                                                 resistance. <br /> ing mutation ompU 50 UTR (Figure 2C),              a <br />                                                                   Strain candidate suppressor mutation initially identi&#239;&#172;?ed, <br /> suggesting mutations impair ompU expression,        applicable. <br />                                                                 b <br /> in cases presumably reduced ToxR binding.             Derived C6706; strains derived N16961. <br /> However, 50 UTR mutation (identi&#239;&#172;?ed strain <br /> BD1962) likely in&#239;&#172;&#8218;uence processes than,             Suppressor mutations presumably reduce cell&#226;&#8364;&#8482;s need <br /> or addition to, transcription, (e.g. mRNA translation        sE; consequently, expected production <br /> or stability), e&#239;&#172;&#8364;ect OmpU mRNA (data              activation sE reduced strains contain- <br /> shown) protein levels (Figure 2B) dramatic         ing suppressor mutations. Our previous studies OmpU <br /> than promoter mutations yielded similar            sE suggested OmpU key determinant basal <br /> reporter activity. Additional northern b-galactosidase      levels rpoE expression, strains lacking OmpU con- <br /> assays suggest UTR promoter mutations           tained markedly lower level sE (18). Mutations that <br /> not alter transcription vca0634 (data shown),            downregulate OmpU production (e.g. promoter 50 <br /> although intergenic region presumably contains          UTR mutations) reduced abundance sE <br /> promoter well.                                               (Figure 3). In addition, mutations cause decrease <br />                                                                 sE activity, indicated reduced b-galactosidase <br /> OmpU-linked vca0199 mutations reduce V. cholerae&#226;&#8364;&#8482;s          rpoE::lacZ transcription reporter fusion whose <br /> requirement pE distinct processes                   activity dependent sE (Figure 4A data not <br />                                                                 shown). In contrast, <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span>point mutation</span> vca0199 that <br /> To determine candidate rpoE suppressor <br />                                                                 suppresses rpoE essentiality did appear in&#239;&#172;&#8218;uence <br /> mutations did fact facilitate growth rpoE <br />                                                                 production sE, increased sE activity fold, <br /> mutants, regenerated candidate suppressor mutations <br />                                                                 suggesting in&#239;&#172;&#8218;uences <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span>survival</span> di&#239;&#172;&#8364;erent mech- <br /> in wt strain, assessed rpoE::aph mutants <br />                                                                 anism (Figures 3 4A). Additional attributes of <br /> could obtained new strains. Following chro- <br />                                                                 vca0199 discussed below. <br /> mosomal integration targeting vector, candidate <br /> rpoE::aph mutants selected plates containing            Activation pE growth phase dependent <br /> sucrose kanamycin, patched plates con- <br /> taining carbenicillin eliminate false positives     Although abundance OmpU fairly consistent <br /> sucrose selection. Using selection protocol, sucrose   phases V. cholerae growth, accumulation <br /> and kanamycin-resistant colonies isolated wt           activation sE largely limited late stationary <br /> strain resistant carbenicillin, i.e. rpoE       phase (Figure 4). Cells contain low amounts sE <br /> mutants obtained (Table 1). In contrast, rpoE              log phase, sE activity overnight cul- <br /> mutants readily obtained strains containing muta-       tures 10-fold higher log phase cultures <br /> tions ompU promoter 50 UTR, indicating            (Figure 4). Similar growth phase-dependent activation in <br /> that reduction OmpU levels obviates V. cholerae&#226;&#8364;&#8482;s          E. coli attributed production ppGpp during <br /> need sE selection conditions. A compara-        stationary phase (8), ppGpp account this <br /> ble result observed using strains deletions         activation V. cholerae well. It noted, how- <br /> ompU toxRS (Table 1). Similarly,               ever, activation OmpU limited stationary <br /> rpoE::aph mutants generated high frequency              phase; diminished ompU expression resulted reduced <br /> in presence vca0199 candidate suppressor             activity (relative wt strain) phases growth <br /> mutation. However, mutants obtained                (Figure 4A). <br /> strain containing vc2413 candidate suppressor muta- <br />                                                                 ompU mutations modulate OmpT levels <br /> tion, suggesting mutation does function a <br /> suppressor. The consequences vc2413 mutation             All V. cholerae rpoE mutants contained higher levels <br /> were pursued further.                                       OmpA OmpT detected wt V. cholerae <br />                                                                                                                        Nucleic Acids Research, 2009, Vol. 37, No. 17 5763 <br />  <br />  <br />                                                                                                                             A <br />                   &#207;&#402;E <br />  <br />  <br />  <br />               OmpA <br />  <br />  <br />  <br />               OmpT <br />  <br />  <br />  <br />  <br />               OmpU <br />  <br />  <br />  <br />  <br />                                                                                                                                                              vca0199 pt <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                   vca0199 pt <br />                                                                                                                                               ompU 5&#226;&#8364;&#8482; utr <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                     ompU 5&#226;&#8364;&#8482; utr <br />                                                                                                                                          wt <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                wt <br />                        wt <br />  <br />                              2048 <br />  <br />                                               1965 <br />  <br />                                                      2064 <br />  <br />                                                                          1962 <br />  <br />                                                                                 2096 <br />  <br />                rpoE    +     +                 &#226;&#710;&#8217;      +                    &#226;&#710;&#8217;     +                     1956 <br />                                                                                                         &#226;&#710;&#8217; <br />                                     vca0199 <br />  <br />  <br />  <br />                                                             ompU 5&#226;&#8364;&#8482;UTR <br />  <br />  <br />  <br />                                                                                        ompU promoter <br />  <br />  <br />  <br />                                                                                                                                               O/N                                   log <br />  <br />         suppressor <br />                                                                                                                             B           O/N       log --&gt; mid <br />                                                                                                                                       cultures stationary phase <br />  <br />  <br /> Figure 3. sE OMPs produced wt V. cholerae strains con-                                                                                           0.5                     1.8 OD600 <br /> taining rpoE and/or suppressor mutations. Western blots probed <br /> with polyclonal antisera sE, OmpA, OmpT OmpU. Each rpoE                                                                                                                                    &#207;&#402;E <br /> mutant (strains BD1965, BD1962 BD1956) paired strain <br /> containing suppressor mutation (e.g. vca0199 ompU-linked <br /> sequences) identi&#239;&#172;?ed rpoE mutant. Extracts electrophoresed                                                                                                                             OmpU <br /> on NuPAGE gels generated cells cultured overnight LB. <br /> Culture volumes used extracts adjusted based culture <br /> OD600 ensure equal loading. <br />                                                                                                                                       &#206;&#8221;ompU <br />                                                                                                                                               rpoE (1956) <br />                                                                                                                                          wt <br />                                                                                                                                                                           wt <br />  <br /> (Figure 2B). However, unlike case OmpA, <br /> the degree OmpT abundance increased not <br /> uniform mutants. OmpT levels                                                            Figure 4. Levels activity sE OmpU produced V. cholerae <br /> slightly increased rpoE mutants lacked ompU-                                                          various growth phases. (A) b-galactosidase activity produced from <br /> linked suppressor mutations, markedly                                                         rpoE-P2::lacZ transcription reporter fusion assessed lacZ <br /> higher rpoE mutants contained ompU-linked                                                        derivative wt V. cholerae (wt) strains containing suppressor <br />                                                                                                               mutations ompU (BD2064 lacZ; ompU 50 UTR) vca0199 <br /> mutations (Figures 2B 3). These data suggest                                                         (BD2048 lacZ; vca0199 pt). Activity assessed overnight (O/N) <br /> sE&#226;&#8364;&#8220;mediated control OmpT accumulation [potentially                                                         log phase (O.D.600   0.5&#226;&#8364;&#8220;0.65; log) cultures. The reporter yielded <br /> due sE&#226;&#8364;&#8220;dependent expression regulatory sRNA,                                                       activity strain lacking rpoE (data shown). Data shown the <br /> seen V. cholerae OmpA E. coli OMPs,                                                           mean SEM representative experiment. (B) Western blots of <br />                                                                                                               OmpU sE produced wt V. cholerae indicated mutants <br /> (33,34)] relatively minor. Furthermore, suggest                                                       log stationary phase growth. <br /> that ompU regulate accumulation OmpT a <br /> sE&#226;&#8364;&#8220;independent fashion. Notably, relatively slight <br /> reductions OmpU signi&#239;&#172;?cant e&#239;&#172;&#8364;ect on <br />                                                                                                               monitored production sE. Disruption ompU <br /> OmpT accumulation did disruption rpoE a <br /> strain normal OmpU production (Figure 3, compare                                                         toxRS, key activators ompU expression, con- <br /> strains BD2096 BD1965).                                                                                   sistently reduced abundance sE 2- 4-fold. <br />                                                                                                               In contrast, disruption genes encoding <br />                                                                                                               OMPs OmpU generally did in&#239;&#172;&#8218;uence sE <br /> OMPs OmpU major determinants                                                               abundance, disruption ompA vc1622 did <br /> of pE abundance activity V. cholerae                                                                   slightly reduce sE levels subset experiments <br /> Western blot rpoE::lacZ transcription reporter anal-                                                      (Figure 5 data shown). The cause inconsistent <br /> yses panel V. cholerae OMP mutants suggest                                                       results mutants known. We also <br /> OmpU plays privileged role respect sE levels                                                     monitored activity sE-dependent rpoE::lacZ tran- <br /> V. cholerae. Strains mutations variety OMP-                                                      scription reporter fusion subset OMP mutants <br /> encoding genes obtained comprehensive                                                             (those excision lacZ-encoding transposon <br /> transposon insertion library V. cholerae mutants (20)                                                      resulted production truncated, full <br />  5764 Nucleic Acids Research, 2009, Vol. 37, No. 17 <br />  <br />  <br />                                                                                                                                                                                                                                    vca0199 probe <br />               0.25x C6706 (wt) <br />  <br />  <br />  <br />  <br />                                                                                                    vca0867 (ompW) <br />  <br />  <br />  <br />  <br />                                                                                                                                                       vca1028 (ompS) <br />                                  0.5x C6706 (wt) <br />  <br />  <br />  <br />                                                                          vc0633 (ompU) <br />  <br />  <br />  <br />                                                                                                                     vc1318 (ompV) <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                        vc2213 (ompA) <br />                                                                                                                                                                        vc1854 (ompT) <br />                                                    C6706 (wt) <br />                                                                                                                                                                                                                  28S <br />  <br />  <br />  <br />  <br />                                                                                          vca1008 <br />                                                                 &#206;&#8221;toxRS <br />  <br />  <br />  <br />  <br />                                                                                                                                     vc1622 <br />                                                                                                                                              vc0972 <br />                                                                                                                                                                                                                  18S <br />           E <br />  <br />  <br />  <br />  <br /> Figure 5. Production sE V. cholerae C6706 transposon <br /> insertion mutants derived it. The western blot probed with <br /> polyclonal antisera sE. Extracts electrophoresed NuPAGE gels                                                                                                                                                                 vca0200 probe <br /> were generated cells cultured overnight LB. Culture volumes <br /> used extracts adjusted based culture OD600 ensure equal                                                                                                                                           28S <br /> loading. Disrupted genes encode known putative outer membrane <br /> proteins porins, plus positive regulator (ToxRS) OmpU <br /> production. The &#239;&#172;?rst lanes (0.25   C6706 0.5   C6706) were <br /> loaded reduced (1/4 1/2) volumes wt sample enable                                                                                                                                             18S <br /> approximate quantitation signal intensities. <br />  <br />  <br />  <br /> length, OMP). Among set&#226;&#8364;&#8221;which included strains <br /> with mutations ompA, vc1622, ompU addi-                                                                                                                                                                     1     2      3     4       5    6 <br /> tional OMP-encoding genes&#226;&#8364;&#8221;the activity transcrip- <br />                                                                                                                                                                                                        Figure 6. Northern blots probed transcripts vca0199 and <br /> tion reporter reduced 2                                                                                                                                                       vca0200. Lanes 1&#226;&#8364;&#8220;3 contain RNA strains derived N16961; <br /> the ompU mutant (data shown). Together,                                                                                                                                                      lanes 4&#226;&#8364;&#8220;6 contain RNA strains derived C6706. Lane 1, wt <br /> data suggest that, growth conditions                                                                                                                                                  N16961; lane 2, BD2048 (vca0199 point mutation); lane 3, BD1965 <br /> assayed, OmpU principal OMP underlying sE                                                                                                                                                       (vca0199 point mutation rpoE insertion); lane 4, wt C6706; <br /> induction (Figure 5). Deletion OMP-encoding                                                                                                                                                 lane 5, BD2034 (vca0199 insertion, KnR); lane 6, BD2041 (vca0199 <br />                                                                                                                                                                                                        insertion, KnS). <br /> genes (ompT, vc0972) did reduce V. cholerae&#226;&#8364;&#8482;s <br /> requirement sE; rpoE::aph mutants be <br /> obtained strains lacking genes (Table 1).                                                                                                                                                     homologs distantly related organisms, including <br />                                                                                                                                                                                                        Bacteriodes fragilis, Listeria innocua Trichodesmium <br /> Not vca0199 mutations suppress V. cholerae&#226;&#8364;&#8482;s                                                                                                                                                       erythraeum; homologs absent vibrios other <br /> requirement rpoE                                                                                                                                                                                   V. cholerae gamma-proteobac- <br /> As noted above, mutation vca0199 appears                                                                                                                                                   teria well, suggesting genes probably <br /> suppress V. cholerae&#226;&#8364;&#8482;s requirement sE di&#239;&#172;&#8364;erent                                                                                                                                                acquired relatively recently horizontal gene transfer. <br /> mechanism does inhibition OmpU production,                                                                                                                                                     The vca0199 mutation detected high-throughput <br /> since vca0199 mutation increases,                                                                                                                                                      sequencing predicted result substitution Y <br /> diminishes, sE activity (Figure 4A). This e&#239;&#172;&#8364;ect detect-                                                                                                                                            S amino-acid 594 638, region minimal <br /> able log stationary phase cultures. vca0199                                                                                                                                                conservation vca0199 homologs. Interestingly, <br /> does appear member sE regulon,                                                                                                                                                      Sanger sequencing vca0199 rpoE mutants <br /> based Northern (data shown) previously pub-                                                                                                                                                 lacking ompU-linked suppressors identi&#239;&#172;?ed additional <br /> lished microarray analyses (17). Northern blotting                                                                                                                                                vc0199 mutation, predicted change P616L (in strain <br /> revealed suppressor mutation vca0199                                                                                                                                                   BD1957). <br /> may cause small reduction transcript                                                                                                                                                 Since e&#239;&#172;&#8364;ects protein function vca0199 <br /> the nearby downstream gene, vca0200,                                                                                                                                                   point mutations described known, we <br /> reduction evident experiments (Figure 6).                                                                                                                                               assessed additional mutations vca0199 <br /> vca0199 vca0200 probably cotranscribed,                                                                                                                                               suppress V. cholerae&#226;&#8364;&#8482;s requirement sE. <br /> <span id='am-239' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-240' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-241' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-243' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-250' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-267' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-272' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-277' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-278' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-290' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-292' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-294' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-303' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-309' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span>insertion mutation</span> vca0199 eliminates transcripts                                                                                                                                               Unexpectedly, assays gave somewhat ambiguous <br /> of genes (Figure 6). We unable detect expres-                                                                                                                                             results. An insertion mutation vca0199, elimi- <br /> sion chromosome-encoded epitope-tagged Vca0199                                                                                                                                                      nates expression vca0199 vca0200, did <br /> and Vca0200, suggesting proteins probably                                                                                                                                               enable replacement rpoE rpoE::aph (Table 1; <br /> produced fairly low levels. However, cell fractionation                                                                                                                                             strain BD2041). In contrast, rpoE::aph mutants be <br /> studies performed using plasmid-encoded Vca0199                                                                                                                                                 obtained vca0199 contained internal deletion, <br /> C-terminal His6 tag indicate product vca0199                                                                                                                                            alter expression vca0200 (Table 1, <br /> probably cytoplasmic protein (data shown).                                                                                                                                                       strain BD2139). However, rpoE mutants generated <br />    No function described vca0199                                                                                                                                                   background grew slowly; colonies were <br /> vca0200, putative protein products strong                                                                                                                                               signi&#239;&#172;?cantly smaller obtained presence of <br /> homology hypothetical proteins. The closest                                                                                                                                                    vca0199 point mutation. Furthermore, mutants were <br />                                                                        Nucleic Acids Research, 2009, Vol. 37, No. 17 5765 <br />  <br />  <br /> obtained lower frequency vca0199 con-           quite low (data shown). The observation <br /> tained internal deletion point mutation.    raises possibility OmpU&#226;&#8364;&#8482;s activating stimulus is <br /> Together, data suggest loss Vca0199 function     transmitted previously synthesized, than <br /> may facilitate growth rpoE mutants,         newly generated, protein. Factors accumulate the <br /> presence Vca0200, consequences          culture medium late stationary phase trigger mem- <br /> point mutation simply loss protein function.      brane OMP disruption reveal activat- <br /> Whether SNP vca0199 partially preserves wt protein      ing motifs masked properly folded proteins <br /> function and/or results novel function remains      (4). Alternatively, exposure activating domains within <br /> determined.                                                    OmpU coupled peptidoglycan remodeling <br />                                                                stationary phase, OmpU reported <br />                                                                associated polymer (36). It theoret- <br /> DISCUSSION                                                     ically possible sE-activating stimulus to <br />                                                                OmpU misfolding instead result OmpU&#226;&#8364;&#8482;s porin <br /> We assessed V. cholerae&#226;&#8364;&#8482;s requirement sE          activity, enhance transmission signal <br /> found essential growth       culture media periplasmic compartment. <br /> all selective conditions. Under permissive conditions         Our study reveals signi&#239;&#172;?cant similarities and <br /> disruption rpoE, mutants readily obtained          di&#239;&#172;&#8364;erences role regulation sE in <br /> but initial growth slow. More rapidly growing        V. cholerae E. coli. For organisms, rpoE abso- <br /> variants, arose frequently initial isolates,    lutely essential subset conditions tested, and <br /> were contain additional mutations unlinked         mutants derived wt cells con- <br /> sites. Introduction unlinked mutations wt        ditions (10). Under stringent selection conditions, <br /> V. cholerae enabled selection rpoE mutants            mutants isolated; however, found <br /> otherwise non-permissive conditions, con&#239;&#172;?rming            contain additional suppressor mutations. Such suppres- <br /> these mutations suppress V. cholerae&#226;&#8364;&#8482;s need sE. The        sor mutations permit disruption rpoE the <br /> majority suppressor mutations reduced production         non-permissive selective conditions (14,38). <br /> OmpU, general di&#239;&#172;&#8364;usion porin constitutes signif-      Deletion OMP-encoding genes reduces activity of <br /> icant portion V. cholerae&#226;&#8364;&#8482;s outer membrane protein          sE V. cholerae E. coli. sE negative <br /> (35,36). The basal level sE declined strains        regulator OMP production organisms, although <br /> ompU-linked mutations, presumably re&#239;&#172;&#8218;ecting dimin-         noted sE does appear in&#239;&#172;&#8218;uence <br /> ished role rpoE rendered non-essential.            V. cholerae&#226;&#8364;&#8482;s production principal OMP, OmpU, <br /> However, mutations vca0199, encodes             rpoE mutant lacks ompU-linked suppres- <br /> gene unknown function, suppress          sor produces wt amounts protein (Figure 3), and <br /> V. cholerae&#226;&#8364;&#8482;s need rpoE. Unlike ompU-linked sup-       overexpression sE did alter OmpU production <br /> pressors suppressors rpoE essentiality identi&#239;&#172;?ed     (data shown). A key di&#239;&#172;&#8364;erence species <br /> E. coli (13,37), suppressor mutations vca0199 aug-      single deletion combination deletions of <br /> mented, reduced, sE activity, suggesting      OMP-encoding genes rendered rpoE non-essential <br /> they diminish requirement sE distinct        E. coli (14), absence single V. cholerae <br /> mechanism, in&#239;&#172;&#8218;uencing process downstream          OMP su&#239;&#172;&#402;cient so. Thus, appears path- <br /> of sE activation. High-throughput genomic sequencing           ways activation sE V. cholerae limited, <br /> was crucial tool identi&#239;&#172;?cation suppressor muta-    monitoring responding OmpU (mis)- <br /> tions, particularly class,     folding paramount importance. Given signi&#239;&#172;?cant <br /> been predicted based previous knowledge sE&#226;&#8364;&#8482;s role        activation sE detectable stationary phase cul- <br /> and regulation V. cholerae. The availability        tures strain diminished OmpU, quite striking <br /> technology, continues improve, greatly         rpoE longer essential cells. <br /> facilitate similar studies organisms.                    Our work provides new evidence bacteria have <br />    We previously observed deletion ompU reduced        developed numerous safeguards coordinate balance <br /> V. cholerae&#226;&#8364;&#8482;s production sE; however, importance        expression OMPs. Excessive OMP accumulation <br /> of &#239;&#172;?nding recognized time (18). Our        known trigger sE activity resulting <br /> new data highlight central role played OmpU          transcription sRNAs downregulate OMP transla- <br /> a regulator sE; indeed, suggest sE&#226;&#8364;&#8482;s       tion (33,38,39). In addition, noted OMP- <br /> most critical roles V. cholerae respond   encoding sequences adjacent genes encoding <br /> endogenously generated factor. Even relatively small           negative regulators OMPs, suggesting that <br /> reductions OmpU levels dramatic e&#239;&#172;&#8364;ect         new OMP-encoding genes readily acquired <br /> organism&#226;&#8364;&#8482;s need for, production of, sigma factor.     conjunction means downregulating that <br /> It appears threshold level OmpU present           present (40). In study, observed that <br /> to generate requirement sE; simply reducing ompU         expression OmpU signi&#239;&#172;?cant negative e&#239;&#172;&#8364;ect <br /> promoter activity  60% renders rpoE inessential,       production OmpT. It long recognized <br /> though OmpU readily detectable. It notable    genes encoding proteins respond opposite <br /> that OmpU signi&#239;&#172;?cant in&#239;&#172;&#8218;uence activity           fashions transcriptional regulator ToxR, thus <br /> during log phase, OmpU synthesized,          expression coordinated (21,32,41); however, <br /> and stationary phase, OmpU transcript levels       disruption ompU previously reported to <br />  5766 Nucleic Acids Research, 2009, Vol. 37, No. 17 <br />  <br />  <br /> augment OmpT production. It remains seen                      heat-shock transcription-factor activity RseA, RseB and <br /> this interdependence OmpU OmpT production                         RseC proteins. Mol. Microbiol., 24, 355&#226;&#8364;&#8220;371. <br />                                                                          3. De Las Penas,A., Connolly,L. Gross,C.A. (1997) The <br /> mediated OmpU&#226;&#8364;&#8482;s porin function, sRNA                          sE-mediated response extracytoplasmic stress Escherichia coli <br /> overlaps coordinated ompU transcript,                   transduced RseA RseB, negative regulators of <br /> or alternate mechanism.                                             sigmaE. Mol. Microbiol., 24, 373&#226;&#8364;&#8220;385. <br />    Finally, identi&#239;&#172;?cation panel suppressor                   4. Walsh,N.P., Alba,B.M., Bose,B., Gross,C.A. Sauer,R.T. (2003) <br />                                                                             OMP peptide signals initiate envelope-stress response by <br /> mutations provides clues mechanisms                            activating DegS protease relief inhibition mediated its <br /> which suppressors arise. Two aspects                      PDZ domain. Cell, 113, 61&#226;&#8364;&#8220;71. <br /> suppressor mutations particularly striking. First,               5. Dartigalongue,C., Missiakas,D. Raina,S. (2001) <br /> same suppressor mutations generated distinct                      Characterization Escherichia coli sE regulon. J. Biol. Chem., <br /> selection protocols (i.e. growth presence antibio-                276, 20866&#226;&#8364;&#8220;20875. <br />                                                                          6. Rhodius,V.A., Suh,W.C., Nonaka,G., West,J. Gross,C.A. <br /> tics versus sucrose), suggesting process sup-                  (2006) Conserved variable functions sE stress response <br /> pressor formation determined solely selection                 related genomes. PLoS Biol., 4, e2. <br /> used. Second, suppressor mutations (11 21)                  7. Valentin-Hansen,P., Johansen,J. Rasmussen,A.A. (2007) Small <br /> single nucleotide contractions mononucleotide (A/T)                      RNAs controlling outer membrane porins. Curr. Opin. Microbiol., <br />                                                                             10, 152&#226;&#8364;&#8220;155. <br /> repeats. In E. coli, mutations characteristic                8. Costanzo,A. Ades,S.E. (2006) Growth phase-dependent <br /> the error-prone polymerase DinB, induced                        regulation extracytoplasmic stress factor, sE, guanosine <br /> speci&#239;&#172;?c stresses associated temporary forma-                    30 ,50 -bispyrophosphate (ppGpp). J. Bacteriol., 188, 4627&#226;&#8364;&#8220;4634. <br /> tion hypermutable state (42). Although mutation                9. Costanzo,A., Nicolo&#239;&#172;&#8364;,H., Barchinger,S.E., Banta,A.B., Gourse,R.L. <br /> stress response investigated V. cholerae,                  Ades,S.E. (2008) ppGpp DksA likely regulate activity <br />                                                                             extracytoplasmic stress factor sE Escherichia coli both <br /> our data consistent possibility formation                 direct indirect mechanisms. Mol. Microbiol., 67, 619&#226;&#8364;&#8220;632. <br /> of rpoE suppressor mutations facilitated               10. De Las Penas,A., Connolly,L. Gross,C.A. (1997) sE an <br /> response. Transient emergence mutator phenotype                        essential sigma factor Escherichia coli. J. Bacteriol., 179, <br /> may underlie high frequency suppres-                    6862&#226;&#8364;&#8220;6864. <br />                                                                         11. Heusipp,G., Schmidt,M.A. Miller,V.L. (2003) Identi&#239;&#172;?cation <br /> sors arise (e.g. Figure 1). Such possibility                     rpoE nadB host responsive elements Yersinia <br /> explored assessing factors needed                      enterocolitica. FEMS Microbiol. Lett., 226, 291&#226;&#8364;&#8220;298. <br /> formation stress-linked mutations (e.g. DinB, RpoS                   12. Button,J.E., Silhavy,T.J. Ruiz,N. (2007) A suppressor cell <br /> and SOS response) required development                     death caused loss sigmaE downregulates extracytoplasmic <br /> of rpoE suppressor mutations. Additionally, blocking                        stress responses outer membrane vesicle production in <br />                                                                             Escherichia coli. J. Bacteriol., 189, 1523&#226;&#8364;&#8220;1530. <br /> these processes permit characterization true               13. Hayden,J.D. Ades,S.E. (2008) The extracytoplasmic stress <br /> consequences rpoE disruption, unmodulated addi-                       factor, sE, required maintain cell envelope integrity in <br /> tional genomic changes.                                                     Escherichia coli. PLoS ONE, 3, e1573. <br />                                                                         14. Douchin,V., Bohn,C. Bouloc,P. (2006) Down-regulation of <br />                                                                             porins small RNA bypasses essentiality regulated <br />                                                                             intramembrane proteolysis protease RseP Escherichia coli. J. Biol. <br /> SUPPLEMENTARY DATA                                                          Chem., 281, 12253&#226;&#8364;&#8220;12259. <br />                                                                         15. Mathur,J. Waldor,M.K. (2004) The Vibrio cholerae <br /> Supplementary Data available NAR Online.                             ToxR-regulated porin OmpU confers resistance antimicrobial <br />                                                                             peptides. Infect. Immun., 72, 3577&#226;&#8364;&#8220;3583. <br />                                                                         16. Kovacikova,G. Skorupski,K. (2002) The alternative sigma <br /> ACKNOWLEDGEMENTS                                                            factorsE plays important role intestinal survival virulence <br />                                                                             Vibrio cholerae. Infect. Immun., 70, 5355&#226;&#8364;&#8220;5362. <br /> The authors thank H. Li, M. Chase, J. Livny                         17. Ding,Y., Davis,B.M. Waldor,M.K. (2004) Hfq essential for <br /> for helpful discussions regarding sequence analysis,                        Vibrio cholerae virulence downregulates sE expression. Mol. <br /> A. C. Matin, S. Lynch, P. Nemani, J. Kaper pro-                     Microbiol., 53, 345&#226;&#8364;&#8220;354. <br />                                                                         18. Mathur,J., Davis,B.M. Waldor,M.K. (2007) Antimicrobial <br /> viding antisera, D. E. Cameron, J. J. Mekalanos                     peptides activate Vibrio cholerae sE regulon an <br /> K. Klose strains plasmids.                                          OmpU-dependent signalling pathway. Mol. Microbiol., 63, 848&#226;&#8364;&#8220;858. <br />                                                                         19. Srivatsan,A., Han,Y., Peng,J., Tehranchi,A.K., Gibbs,R., <br />                                                                             Wang,J.D. Chen,R. (2008) High-precision, whole-genome <br /> FUNDING                                                                     sequencing laboratory strains facilitates genetic studies. PLoS <br />                                                                             Genet., 4, e1000139. <br /> Howard Hughes Medical Institute; National Institutes                 20. Cameron,D.E., Urbach,J.M. Mekalanos,J.J. (2008) A de&#239;&#172;?ned <br /> Health [AI-42347 M.K.W.]. Funding open access                        transposon mutant library use identifying motility genes <br />                                                                             Vibrio cholerae. Proc. Natl Acad. Sci. USA, 105, 8736&#226;&#8364;&#8220;8741. <br /> charge: Howard Hughes Medical Institute.                                21. Miller,V.L. Mekalanos,J.J. (1988) A novel suicide vector its <br />                                                                             use construction insertion mutations: osmoregulation outer <br /> Con&#239;&#172;&#8218;ict statement. None declared.                               membrane proteins virulence determinants Vibrio cholerae <br />                                                                             requires toxR. J. Bacteriol., 170, 2575&#226;&#8364;&#8220;2583. <br />                                                                         22. Davis,B.M. Waldor,M.K. (2007) RNase E-dependent <br /> REFERENCES                                                                  processing stabilizes MicX, Vibrio cholerae sRNA. Mol. <br />                                                                             Microbiol., 65, 373&#226;&#8364;&#8220;385. <br /> 1. Ruiz,N. Silhavy,T.J. (2005) Sensing external stress: watchdogs   23. Donnenberg,M.S. Kaper,J.B. (1991) Construction eae <br />    Escherichia coli cell envelope. Curr. Opin. Microbiol., 8,        deletion mutant enteropathogenic Escherichia coli using a <br />    122&#226;&#8364;&#8220;126.                                                                 positive-selection suicide vector. Infect. Immun., 59, 4310&#226;&#8364;&#8220;4317. <br /> 2. Missiakas,D., Mayer,M.P., Lemaire,M., Georgopoulos,C.            24. Provenzano,D., Lauriano,C.M. Klose,K.E. (2001) <br />    Raina,S. (1997) Modulation Escherichia coli sE (RpoE)             Characterization role ToxR-modulated outer <br />                                                                                     Nucleic Acids Research, 2009, Vol. 37, No. 17 5767 <br />  <br />  <br />     membrane porins OmpU OmpT Vibrio cholerae virulence.                belong sE regulon: role down-regulation outer <br />     J. Bacteriol., 183, 3652&#226;&#8364;&#8220;3662.                                             membrane proteins. J. Mol. Biol., 364, 1&#226;&#8364;&#8220;8. <br /> 25. Sperandio,V., Bailey,C., Giron,J.A., DiRita,V.J., Silveira,W.D.,       34. Song,T., Mika,F., Lindmark,B., Liu,Z., Schild,S., Bishop,A., <br />     Vettore,A.L. Kaper,J.B. (1996) Cloning characterization         Zhu,J., Camilli,A., Johansson,J., Vogel,J. et al. (2008) A new Vibrio <br />     gene encoding OmpU outer membrane protein Vibrio                cholerae sRNA modulates colonization a&#239;&#172;&#8364;ects release outer <br />     cholerae. Infect. Immun., 64, 5406&#226;&#8364;&#8220;5409.                                   membrane vesicles. Mol. Microbiol., 70, 100&#226;&#8364;&#8220;111. <br /> 26. Schneider,K. Beck,C.F. (1986) Promoter-probe vectors       35. Simonet,V.C., Basle,A., Klose,K.E. Delcour,A.H. (2003) The <br />     analysis divergently arranged promoters. Gene, 42, 37&#226;&#8364;&#8220;48.               Vibrio cholerae porins OmpU OmpT distinct channel <br /> 27. Li,H., Ruan,J. Durbin,R. (2008) Mapping short DNA                      properties. J. Biol. Chem., 278, 17539&#226;&#8364;&#8220;17545. <br />     sequencing reads calling variants using mapping quality scores.    36. Chakrabarti,S.R., Chaudhuri,K., Sen,K. Das,J. (1996) Porins <br />     Genome Res., 18, 1851&#226;&#8364;&#8220;1858.                                                Vibrio cholerae: puri&#239;&#172;?cation characterization OmpU. <br /> 28. Hernandez,D., Francois,P., Farinelli,L., Osteras,M. Schrenzel,J.       J. Bacteriol., 178, 524&#226;&#8364;&#8220;530. <br />     (2008) De novo bacterial genome sequencing: millions           37. Button,J.E., Silhavy,T.J. Ruiz,N. (2007) A suppressor cell <br />     short reads assembled desktop computer. Genome Res., 18,              death caused loss sE downregulates extracytoplasmic <br />     802&#226;&#8364;&#8220;809.                                                                   stress responses outer membrane vesicle production in <br /> 29. Heidelberg,J.F., Eisen,J.A., Nelson,W.C., Clayton,R.A.,                    Escherichia coli. J. Bacteriol., 189, 1523&#226;&#8364;&#8220;1530. <br />     Gwinn,M.L., Dodson,R.J., Haft,D.H., Hickey,E.K., Peterson,J.D.,        38. Mecsas,J., Rouviere,P.E., Erickson,J.W., Donohue,T.J. and <br />     Umayam,L. et al. (2000) DNA sequence chromosomes                Gross,C.A. (1993) The activity sigma E, Escherichia coli <br />     cholera pathogen Vibrio cholerae. Nature, 406, 477&#226;&#8364;&#8220;483.                heat-inducible sigma-factor, modulated expression outer <br /> 30. Miller,J.H. (1992) A Short Course Bacterial Genetics: A                 membrane proteins. Genes Dev., 7, 2618&#226;&#8364;&#8220;2628. <br />     Laboratory Manual Handbook Escherichia Coli Related        39. Figueroa-Bossi,N., Lemire,S., Maloriol,D., Balbontin,R., <br />     Bacteria. Cold Spring Harbor Press, Cold Spring Harbor, NY.                Casadesus,J. Bossi,L <br /> </body></html>