<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL  OF BACrERIOLOGY, Nov. 1993, P. 7222-7227                                                                            Vol. 175, No. 22 <br /> 0021-9193/93/227222-06$02.00/0 <br /> Copyright C) 1993, American Society Microbiology <br />  <br />  <br />           Bradyrhizobium japonicum 6-Aminolevulinic Acid Dehydratase <br />           Is Essential Symbiosis Soybean Contains Novel <br />                               Metal-Binding Domain <br />                                        SARITA CHAUHAN AND MARK R. O&apos;BRIAN* <br />             Department Biochemistry Center Advanced Molecular Biology Immunology, 140 Farber Hall, <br />                                 State University New York Buffalo, Buffalo, New York 14214 <br />                                            Received 3 August 1993/Accepted 16 September 1993 <br />  <br />              The Bradyrhizobiumjaponicum hemA gene product 8-aminolevulinic acid (ALA) synthase required for <br />           symbiosis bacterium soybean. Hence, essentiality subsequent heme synthesis enzyme, <br />           ALA dehydratase, examined. The B. japonicum ALA dehydratase gene, termed hemB, isolated and <br />           identified basis ability confer hemin prototrophy enzyme activity Escherichia coli hemB <br />           mutant, encoded protein highly homologous ALA dehydratases diverse organisms. A <br />           novel metal-binding domain B. japonicum ALA dehydratase identified structural composite <br />           Mg2+-binding domain plant ALA dehydratases Zn2+-binding region nonplant ALA <br />           dehydratases. Enzyme activity dialyzed extracts cells overexpressed hemB gene reconstituted <br />           addition Mg2+ addition Zn2+, indicating B. japonicum ALA dehydratase is <br />           similar plant enzymes respect metal requirement. Unlike B. japonicum hemA mutant, the <br />           hemB mutant strain KP32 elicited undeveloped nodules soybean, indicated lack nitrogen fixation <br />           activity plant hemoglobin. We conclude hemB gene required nodule development and <br />           propose B. japonicum ALA dehydratase essential bacterial enzyme B. japonicum heme <br />           synthesis soybean root nodules. In addition, postulate ALA heme intermediate can <br />           translocated plant endosymbiont support bacterial heme synthesis nodules. <br />  <br />    Bradyrhizobium japonicum establishes symbiotic relation-                  Data supporting hypothesis B. japonicum heme can <br /> ship soybean plant assimilates nitrogen fixed              formed soybean-derived ALA rule provi- <br /> by bacterium endosymbiont utilizes carbon fixed              sion plant heme precursors bacterium well, <br /> the host plant. The symbiosis manifested nodules                   intermediates could, principle, rescue <br /> soybean roots, specialized organs composed                 hemA mutant. The heme moiety itself unlikely be <br /> differentiated plant bacterial cells. An increase heme               translocated B. japonicum ferrochelatase, enzyme <br /> content compared respective asymbiotic plant                   catalyzes protoheme formation protoporphyrin IX <br /> bacterial cells changes characterize                 Fe2&quot;, required symbiosis (8, 9). In order under- <br /> differentiated cells nodule (1, 21), <span id='am-3' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-4' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-5' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span>phenotype</span>                stand interorganismic events governing heme synthesis in <br /> accomodates energy-intensive process nitrogen fixation.              nodules, necessary identify precisely plant interme- <br /> Bacterial cytochromes allow oxidative phosphorylation                diates support B. japonicum heme formation <span id='am-2' about='protege:TO' typeof='owl:Thing'>to</span> <br /> low-02 milieu nodule, drives nitrogen fixation,                determine bacterial enzymes required symbiosis, <br /> and plant hemoglobin (leghemoglobin) facilitates 02 diffusion               parameters likely reveal points control. <br /> to respiring bacteroids. Evidence indicates heme                   Herein, B. japonicum ALA dehydratase, the <br /> synthesis nodules interactive phenomenon                   enzyme utilizes ALA directly heme pathway, is <br /> each symbiont exerts effect other. A B. japonicum                 essential symbiosis soybean. In addition, evidence is <br /> mutant defective 8-aminolevulinic acid (ALA) synthase,               provided unusual metal-binding domain ALA <br /> enzyme forms heme precursor ALA glycine                      dehydratase B. japonicum. <br /> and succinyl-coenzyme A, elicits nitrogen-fixing nodules on <br /> soybean (11), expression bacterial heme mutant <br /> is rescued symbiotically (20). Soybean synthesizes ALA                                MATERIALS AND METHODS <br /> glutamate nodules (20, 21), B. japonicum bacteroids <br /> take exogenous ALA (20); thus, proposed bacterial                   Chemicals reagents. All chemicals reagent grade <br /> heme synthesized plant-derived ALA nodules,                  purchased Sigma Chemical Co., St. Louis, Mo., <br /> thereby rendering hemA gene nonessential symbiosis                  J. T. Baker Chemical Co., Philipsburg, N.J. Purified <br /> (20). The soybean ALA formation activity induced                      Noble agar, yeast extract, tryptone obtained <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> nodules (21), due, part, activation          Difco Laboratories, Detroit, Mich. ALA, porphobilinogen <br /> soybean glutamate 1-semialdehyde aminotransferase gene                      (PBG), mesoporphyrin IX purchased Porphyrin <br /> (22). The spatial separation heme pathway nodules              Products, Logan, Utah. [co-32P]dCTP (3,000 Ci/mmol) and <br /> reminiscent yeasts animals, steps              [ot-35S]dATP (1,185 Ci/mmol) obtained New En- <br /> occur mitochondria occur cytosol.                  gland Nuclear-DuPont, Boston, Mass. <br />                                                                                Bacterial strains, media, growth. B. japonicum MLG1 <br />                                                                             (11) KP32 derivatives wild-type B. japonicum <br />    * <br />      Corresponding author. Electronic mail address: cammrob@ubvms.          strain 1110. These strains routinely grown GSY me- <br /> cc.buffalo.edu.                                                             dium described previously (10). Mutant strains KP32 and <br />                                                                      7222 <br />  VOL. 175, 1993                                              B. JAPONICUM B-AMINOLEVULINIC ACID DEHYDRATASE                     7223 <br />  <br /> MLG1 grown presence kanamycin (50 jig/ml).           hemin prototrophy ALA dehydratase activity. A 2.6-kb <br /> ALA (30 jig/ml), PBG (20 jig/ml), hemin (1 jiM broth        EcoRI-KpnI fragment pZXEN carrying disrupted <br /> 15 jiM plates), cysteine (20 jig/ml), vitamin B12 (2        hemB gene cloned EcoRI-PvuI site pBR322 to <br /> ng/ml) added autoclaved media. ALA PBG              construct pZXENOM, subsequently mobilized into <br /> stocks prepared water. A 3.8 mM stock solution           B. japonicum 1110 described previously (11). Replacement <br /> commercial hemin prepared 0.02 N NaOH solution            genomic hemB gene mutagenized by <br /> 50% ethanol. Polyoxyethylene sorbitan mono-oleate (Tween            homologous recombination scored KMr Tcs colonies <br /> 80; 0.003% [wt/vol]) added liquid media prevent           confirmed Southern blot analysis. <br /> hemin precipitating. Escherichia coli strains grown          Growth soybeans analysis nodules. Soybeans <br /> at 37&#194;&#176;C Luria broth (LB) medium (2) appropriate             (Glycine max cv. Essex) grown growth chamber with <br /> supplements. Strain RP523 derivative strain C600        16-h light/8-h dark photoperiodicity 25&#194;&#176;C vermiculite <br /> carries <span id='am-81' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-82' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span>mutation</span> gene encoding ALA dehydratase (17).       containing N-free supplemental nutrients. Germinated seed- <br /> Strain RP523 grown LB supplemented hemin (1             lings inoculated B. japonicum cultures; approxi- <br /> jig/ml broth 10 jig/ml plates), ampicillin (50      mately 108 cells used seedling. Nitrogen fixation <br /> jig/ml) added cells carrying plasmids. Isopropyl-3-D-       activity (acetylene reduction), viable cell counts, leghemo- <br /> thiogalactopyranoside (IPTG) (40 jig/ml) added induc-       globin heme determinations nodule cytosol carried out <br /> tion plasmid-borne genes.                                        described previously (8). Leghemoglobin protein dis- <br />    Construction B. japonicum expression library            cerned immunologically Western blot (immunoblot), anal- <br /> isolation hemB gene. A B. japonicum genomic library          ysis described previously (19). <br /> was constructed pBluescript II SK. To this, B. japonicum         Nucleotide sequence accession number. The sequence the <br /> genomic DNA partially digested Sau3AI, 1-           Sau3AI partial digest containing B. japonicum hemB gene <br /> 3-kb fragments purified agarose gel band            (shown Fig. 2) deposited GenBank under <br /> interception. The ends partially filled using dGTP,      accession number L24386. <br /> dATP, Klenow enzyme ligated the <br /> XhoI site pBluescript SK II (+) previously                                   RESULTS <br /> digested filled dCTP, d&apos;TiTP, Klenow enzyme. <br /> The ligation reaction used transform competent XL-1             Complementation E. coli ALA dehydratase mutant <br /> Blue E. coli cells en masse electroporation, resultant   B. japonicum genomic DNA. E. coli mutant strain RP523 <br /> library contained approximately 95% recombinant clones           defective hemB, gene encoding ALA dehydratase, and <br /> determined frequency blue white colonies           requires hemin growth LB medium (17). Strain RP523 <br /> X-Gal plates. One eighty nanograms ligated               cells transformed B. japonicum genomic library in <br /> genomic DNA yielded approximately 3 x 105 independent               pBluescript, complemented cells selected Apr <br /> recombinant clones. Colonies scraped LB plates,         hemin prototrophs. A total 199 prototrophic colonies arose <br /> plasmid DNA isolated cells previously             approximately 3 x 106 screened, 13 larger <br /> described, library maintained ethanol. The           colonies picked analysis. Three different <br /> library used transform RP523 cells (a hemB E. coli           plasmids 11 clones represented pZEN, <br /> strain), complemented transformant cells screened          pWte5, pWte6 (Fig. 1), insert DNAs overlapped <br /> for grew presence ampicillin IPTG         other. The clones carried plasmids inserts <br /> and absence added hemin. Colonies arose        contained 1-kbApaI-Sau3AI fragment in <br /> this selection restreaked, plasmid DNA isolated        pZEN, pWte5, pWte6 additional DNA was <br /> from them.                                                          different plasmids arisen from <br />    DNA manipulations sequencing. Routine DNA manip-             ligation noncontiguous DNA library construction; <br /> ulations according previously described methods        plasmids studied further. Reintroduction of <br /> (2). Deletions pZEN obtained using Exo-Mung           pZEN, pWte5, pWte6 strain RP523 conferred hemin <br /> deletion kit (Stratagene) according manufacturer&apos;s           prototrophy high ALA dehydratase activity mutant, <br /> instructions removal restriction fragments. Double-        confirms complemented phenotype the <br /> stranded DNA sequencing carried using Seque-           transformants plasmids chromo- <br /> nase kit (version 2.0, United States Biochemicals) according     somal reversion (Fig. 1). The RP523 transformants expressed <br /> the manufacturer&apos;s instructions; sequences strands      higher ALA dehydratase activity the <br /> were determined. The sequence analyzed software            parent strain, C600 (Fig. 1), presumably the <br /> from Genetics Computers Group (7).                              high copy number pBluescript derivatives, strength <br />    Determination metal requirement ALA dehydratase           lacZ promoter, both. pZEN used further <br /> activity. E. coli mutant strain RP523(pZEN) grown           studies. <br /> induced expression B. japonicum hemB gene                Nucleotide sequence deduced product DNA that <br /> described above. Cell extracts prepared previously          complements strain RP523. The nucleotide sequence the <br /> described (10) dialyzed overnight 0.1 M Tris (pH         1,242-bp insert pZEN determined, large open <br /> 8)-10 mM dithiothreitol-0.5 mM EDTA; EDTA                   reading frame identified contained putative <br /> added chelation metals. Various amounts ZnSO4          ATG start codons <span id='am-159' about='xsp:base' typeof='owl:Thing'>base</span> positions 94 136 (Fig. 2) and <br /> MgSO4 added dialyzed extracts preincubated          vector downstream lac promoter that <br /> for 15 min 37&#194;&#176;C. ALA dehydratase activity measured        reading frame. To determine essentiality methio- <br /> the treated extracts described previously (8).                   codons expression ALA dehydratase strain <br />    Construction stable B. japonicum hemB mutant. The           RP523, derivatives pZEN constructed deletions in <br />  isolated B. japonicum hemB gene pZEN disrupted           5&apos; end insert removed the <br />  inserting 1.3-kb kanamycin resistance-encoding cassette     codons (Fig. 3). A deletion pZEN removed first <br /> pUCK4 (27) unique StuI site. The resultant plasmid,          ATG codon (pZENdl) resulted substantial increase in <br />  pZXEN, complement strain RP523 respect           ALA dehydratase activity, deletion removed <br />  7224      CHAUHAN AND O&apos;BRIAN                                                                                                   J. BAC-FERIOL. <br />  <br />  <br />                      Strain                                     Insert                          Hemin         ALAD <br />                                                                                               auxotrophy      activity <br />  <br />                      C600                                       None                                           20&#194;&#177;1 <br />  <br />                      RP523 [pSK II (+)]                         None                               +              0 <br />  <br />                      RP523 [pWte5]                                                                             74 &#194;&#177; 2 <br />  <br />                      RP523 [pWte6]                                                                           121 &#194;&#177; 6 <br />  <br />                      RP523 [pZENI                                                                              96 &#194;&#177; 4 <br />                                                                                 ,200bp, <br />                                                               &apos;K7 <br />  <br />   FIG. 1. Complementation E. coli RP523 B. japonicum genomic DNA library clones. The overlapping inserts shown hash marks <br /> representing Sau3AI sites determined nucleotide sequence. The arrow represents open reading frame deduced nucleotide <br /> sequence insert pZEN, triangle denotes site insertion Kmr interposon construction B. japonicum hemB <br /> mutant KP32. ALA dehydratase (ALAD) activities extracts E. coli C600 RP523 transformants expressed nanomoles PBG <br /> formed hour milligram protein averages trials (plus minus standard deviation). Hemin auxotrophy was <br /> determined ability grow LB medium addition 10 puM hemin (ferric heme hydrochloride). <br />  <br />  <br /> both codons (pZENd2) abolished enzyme activity (Fig. 3).                 japonicum ALA dehydratase dialyzed EDTA to <br /> Hence, smallest open reading frame confers activity             remove metals incubated various concentrations <br /> begins second ATG position 136, codon            Mg2+ Zn2+ (Fig. 5). Dialyzed extracts contained little <br /> preceded putative ribosome-binding site (Fig. 2). The               ALA dehydratase activity, stimulated the <br /> deletion pZENdl placed translation start site            addition Zn2+ 1 mM (Fig. 5). Conversely, addition of <br /> closer lac promoter pZEN,            Mg2+ resulted restoration enzyme activity; hence, B. <br /> resulted removal N-terminal amino acids de-               japonicum ALA dehydratase similar plant enzymes <br /> crease activity. The deduced protein encoded open                 respect metal requirement. The simultaneous <br /> reading frame pZEN homologous ALA                  addition equimolar concentrations Mg2+ Zn2+ <br /> dehydratases; shared 37 51% identity dehydra-             resulted activities similar elicited addition of <br /> tases bacteria (6, 12, 16), S. cerevisiae (18), plants        Mg2+ (data shown). <br /> (5, 15, 23), animals (3, 4, 28). This sequence homology,                Construction hemB strain B. japonicum charac- <br /> along complementation E. coli ALA dehydratase                terization soybean nodules elicited mutant. A stable <br /> mutant (Fig. 1), strongly implies cloned gene encodes           genomic hemB mutant B. japonicum 1110 constructed <br /> the B. japonicum ALA dehydratase, desig-             replacement wild-type gene disrupted with <br /> nated hemB.                                                              Kmr-encoding interposon cassette (Fig. 1) (see Materials and <br />    Evidence novel metal-binding domain B. japonicum             Methods). The B. japonicum hemB mutant strain KP32 a <br /> ALA dehydratase. A Zn2+-binding domain identified               PBG auxotroph minimal medium contained ALA <br /> in ALA dehydratases animals yeasts basis            dehydratase activity. Because PBG sufficient compen- <br /> four cysteines histidine, similar zinc-finger      sate hemB lesion presumably allow heme <br /> motif consensus (reviewed reference 14) (Fig. 4). Bacteria            formation, heme synthesis genes hemB not <br /> also contain motif, conserved                disrupted strain KP32. PBG replaced the <br /> cysteines missing, E. coli enzyme binds Zn2+         simultaneous addition hemin, cysteine, vitamin B12. <br /> nevertheless (25). The corresponding domain plant dehy-               PBG precursor siroheme prosthetic group sulfite <br /> dratases homologous overall nonplant dehydra-              reductase, necessary sulfur amino acid synthesis, <br /> tases, cysteines histidine        corrin ring vitamin B12 (24). <br /> replaced aspartate-rich region proposed bind                  Soybean seedlings inoculated strain 1110, the <br /> Mg2+ Zn2+ (5, 23) (Fig. 4). Interestingly, B.            hemA strain MLG1, hemB strain KP32, nodules <br /> japonicum ALA dehydratase contains plant             24-day-old plants analyzed. (Fig. 6). Mutant strain <br /> nonplant metal-binding motif exact composite            MLG1 elicited nitrogen-fixing nodules soybean ex- <br /> of (Fig. 4). Cysteine histidine present positions        pressed leghemoglobin (Fig. 6), reported previously <br /> 143 153, respectively, B. japonicum ALA dehy-                 (11), showing hemnA gene essential symbiosis <br /> dratase, typifies nonplant enzymes. However, Ala-              soybean. By contrast, nodules elicited hemB <br /> 146, Asp-148, Asp-156 correlate exactly plant ALA               mutant KP32 small starchy appearance, contained <br /> dehydratases proposed Mg2+-binding domain. To                    viable bacteria, did fix nitrogen, lacked hemoglo- <br /> determine Zn2+ Mg2+ required enzyme                   bin heme apoprotein (Fig. 6). Thus, unlike ALA synthase, <br /> activity, extracts strain RP523 cells overexpressed B.       B. japonicum ALA dehydratase required normal <br />  VOL. 175, 1993                                                     B.    JAPONICUM B-AMINOLEVULINIC ACID DEHYDRATASE                                              7225 <br />  <br /> GATCTCGGACATGGCTCCCTTCTGGCTGTTTCGCCCCGCAAAATCAATCGGCTGCCATCC            60   PLANTS <br /> d L g h g s L l v s p r k n r L p s                                      Soybean             L   V   I Y T D VA L    D   P Y S S        D G H     D G I        V R <br /> CTTGCGGCAGGTCAAGCCGCGGTGACGGGAACCATGCGAGCTGGTATAAGAATAATTCCA        120      Pea                 L   I   I Y T D VA L    D   P Y S S        D G H     D G I        V R <br />                                                                              Spinach             L   I   I Y T D VA L    D   P Y Y Y        D G H     D G I        V T <br /> L   g q a v t g t m r g r i p                                    S. martensii        L   V   I YT D V |AL    D   P Y S S        D G H     D G I        V R <br /> GCCGGAGGAAGAGTGATGGCGATCAAATACGGGCGTCCGATCGAATTGCGCGAGGTTTCG 180 <br /> a g g r v M A I K Y G R P I E L R E V S <br />                                                                                .   JAPONICU      I G V L C D V          L D P F T S H G H D G L I A <br /> CGCCGGGATGGCGCGGCAGCCTCCCCTGCCCTCGATCTGGCCATCCGTCCGCGCCGCAAC 240 <br /> R R D G A A A S P A L D L A I R P R R N                                      NON-PQLANTSL <br />                                                                              Mouse               L L V    A   C   D V C L C P   Y   T   S H   G   H   C   G   L    L   S <br /> CGCAAGGCCGAGTGGGCCCGGCGGATGGTGCGCGAGAACGTGCTCACCACCGACGATCTG 300             Rat                 L L V    A   C   D V C L C P   Y   T   S H   G   H   C   G   L    L   S <br /> R K A E W A R R M V R E N V L T T D D L                                      Human               L L V    A   C   D V C L C P   Y   T   S H   G   H   C   G   L    L   S <br /> ATCTGGCCGCTGTTCCTGATCGACGGCAACAACAAGCGCGAGCAGATCGCCTCGATGCCG 360             Yeast               L Y I    I   C   D V C L C E   Y   T   S H   G   H   C   G   V    L   Y <br />                                                                              M. sociabilis       L V V    I   T   D V C L C Q   Y   T   E H   G   H   C   G   I    V   K <br /> I W P L F L I D G N N K R E Q I A S M P                                      B. subtilis         M V V    V   A   D T C L C E   Y   T   D H   G   H   C   G   L    V   K <br /> GGCGTCGAGCGCCTCAGCGTCGACCAGGCCGTGCGCGAGGCCGAGCGCGCGATGAAGCTC 420             E. coli             M I V    M   S   D T C F C E   Y   T   S H   G   H   C   G   V    L   C <br /> G V E R L S V D Q A V R E A E R A M K L <br />                                                                                 FIG. 4. Comparison metal-binding domain B. japonicum <br /> ACGATCCCCTGCATCGCGCTGTTTCCCTACACCGACCCGTCCCTGCGCGACGAGGAAGGC                 ALA dehydratase plants nonplants. Relevant amino <br /> T I P C I A L F P Y T D P S L R D E E G                                      acid residues boxed. The B. japonicum sequence shown corre- <br /> TCGGAGGCCTGCAACCCGAACAATCTGGTCTGCCAGGCGGTACGCGCGATCAAGAAGGAA 540             sponds amino acids 139 160. Sequences soybean (15), pea <br /> S E A C N P N N L V C Q A V R A I K K E                                      (5), spinach (23), fern Selaginella martensii (23), mouse (4), rat (3), <br /> TTTCCGGAGATCGGCGTCCTCTGCGACGTCGCGCTCGATCCCTTCACCAGCCACGGCCAT 600             human (28), S. cerevisiae (18), Methanothermus sociabilis (6), Bacillus <br /> F P E I G V L C D V A L D P F T S H G H                                      subtilis (12), E. coli (16) ALA dehydratase metal-binding domains <br />                                                                              shown. <br /> GACGGCCTGATCGCGGACGGCGCGATCCTGAACGACGAGACGGTCGCCGTGCTGGTGCGC 660 <br /> D G L I A D G A I L N D E T V A V L V R <br /> CAGGCCCTGGTGCAGGCCGAAGCCGGCTGCGACATCATCGCGCCCTCCGACATGATGGAC 720 <br /> Q A L V Q A E A G C D I I A P S D M M D                                      nodule development symbiosis. Because nodules elicited <br /> GGCCGCGTCGCCGCGATCCGCGAGGGACTGGACCAGGCAGGGCTCATCGACGTGCAGATC 780             hemB mutant contained viable bacteria, likely <br /> G R V A A I R E G L D Q A G L I D V Q I                                      leghemoglobin late proteins expressed <br />                                                                              nodules. Therefore, data directly address <br /> M A Y A A K Y A           S A    F Y G P      F   R D A    I   G             role endosymbiont plant hemoglobin formation, <br /> TCGGCCAAGACGCTGACCGGCGACAAGCGCACCTACCAGATGGACAGCGCCAACACCGAC 900             absence plant hemoglobin characteristic of <br /> S A K T L T G D K R T Y Q M D S A N T D <br />                                                                              abnormally developed nodules. <br /> GAGGCGCTGCGCGAGGTCGAGCTCGACATCTCCGAGGGCGCCGACATGGTGATGGTGAAG 960 <br /> E A L R E V E L D I S E G A D M V M V K                                                                    DISCUSSION <br /> CCGGGCATGCCCTATCTCGACGTGGTCCGCCGCGTGAAGGACACCTTTGCGATGCCGACC 1020 <br /> P G M P Y L D V V R R V K D T F A M P T                                         In present work, isolated B. japonicum hemB, the <br /> TTCGCCTACCAGGTGTCCGGTGAATACGCGATGATCGCGGCGGCCGCGGGCAACGGCTGG 1080            gene  encoding ALA dehydratase, constructed genomic <br /> F A Y Q V S G E Y A M I A A A A G N G W                                      hemB mutant investigate expression gene is <br /> CTCGACGGCGACCGCGCGATGATGGAGAGCCTGCTCGCCTTCAAGCGCGCCGGCGCGGAT 1140            essential symbiosis. The phenotype soybean nodules <br /> L D G D R A M M E S L L A F K R A G A D <br />                                                                              elicited hemB strain KP32 showed clearly B. <br /> GGCGTGCTGAGCTACTTTGCCCCGAAGGCGGCGGAGAAGCTGCGGACGCAGGGGTAAGGT 1200            japonicum ALA dehydratase necessary normal nodule <br /> G V L S Y F A P K A A E K L R T Q G* <br />                                                                              development (Fig. 6), expression bacterial <br /> CATTGTCGTCCTGGCGATGGCCGGGACCCATACCGCGTGATC                   1242 <br />                                                                              hemA gene required (11) (Fig. 6). These data strongly <br />                                                                              indicate B. japonicum heme synthesis nodules be <br />   FIG. 2. Nucleotide sequence B. japonicum genomic frag-              supported plant PBG subsequent precursors heme <br /> ment borne pZEN deduced protein product. Upper case amino             pathway; hence, postulate traversal heme interme- <br /> acids represent proposed hemB product based deletion analysis         diates host endosymbiont limited ALA <br /> described text shown, Fig. 3. An asterisk (*) denotes      (Fig. 7). From this, propose B. japonicum ALA <br /> termination codon. The underlined nucleotide sequence denotes the <br /> putative ribosome-binding site.                                              dehydratase essential bacterial enzyme B. <br />                                                                              japonicum tetrapyrrole synthesis soybean root nodules. <br />                                                                              Previous work (20) indicated B. japonicum hemA <br />  <br />  <br />  Plasmid                                              Deduced amino acid sequence                                                                 Acivity <br />  pZEN                  DL-X29-MRAGIRIIPAGGRV&amp;AIKYGRPIELREVSRRDGAAASPALD-                                                                              72+2 <br />  pZENd1                                                    GGRVHAIKYGRPIELREVSRRDGAAASPALD-                                                       474&#194;&#177;9 <br /> pZENd2                                                                                                   RRDGAAASPALD-                                        0 <br />   FIG. 3. ALA dehydratase activities E. coli RP523 transformants harboring 5&apos; deletions pZEN. N-terminal regions deduced products <br /> of pZEN deletion clones pZENdl pZENd2 shown. Methionines (M) underlined. ALA dehydratase activities cell extracts are <br /> expressed nanomoles PBG formed hour milligram protein averages trials (plus minus standard <br /> deviation). <br />  7226          CHAUHAN AND O&apos;BRIAN                                                                                                 J. BACrFERIOL. <br />  <br />               150                                                                                (Alad)                     SOYBEAN <br />                                                                                   Glu- (   ALA            PBG <br />        *3      125 <br />        0 <br />              fi100 <br />  <br />        &apos;m      75 <br />  <br />         E      50 <br />        .-- <br />        &gt;       25 <br />        &lt; 0                                                                  FIG. 7. Proposed mechanism B. japonicum heme formation in <br />        0. .0             0.2   0.4          0.6       0.8      1.0       soybean root nodules based present study previous work <br />                           Metal concentration (mM)                       (20). The figure distinguishes plant bacterial phenomena in <br />                                                                          nodules, higher-order structure implied. The following B. <br />   FIG. 5. Effect Mg2&quot; Zn2+ B. japonicum ALA dehy-              japonicum soybean genes shown isolated: hemA, <br /> dratase dialyzed extracts cells overexpressing hemB gene.    encoding ALA synthase (11); hemB, encoding ALA dehydratase; <br /> Growth cells, induction, extract preparation, dialysis, assays    hemH, encoding ferrochelatase (8); Gsa, encoding glutamate 1-semi- <br /> were carried described text. ALA dehydratase activity   aldehyde aminotransferase (22); Alad, encoding soybean ALA <br /> extracts treated Zn2+ (open circles) Mg2+ (closed circles)    dehydratase (15). <br /> expressed nanomoles PBG formed hour milligram <br /> protein, point shown average trials. <br />                                                                          demonstrate ALA uptake peribacteroid mem- <br />                                                                          brane limiting step bacteroid heme synthesis. <br /> mutant expresses PBG symbiotic cells culture;          It kept mind influx committed <br /> present study indicates PBG provided directly         precursor stable molecule heme need as <br /> the soybean host synthesized endosymbiont           great uptake dicarboxylates (e.g., reference 26) <br /> from plant-derived ALA bacterial ALA dehydratase.                 serve carbon energy sources rapidly <br />    Groups bacteroids surrounded peribacteroid                consumed. <br /> membrane form structure termed symbiosome. It                 The putative metal-binding domain B. japonicum <br /> yet known ALA formation soybean occurs                 ALA dehydratase structural composite of <br /> or outside symbiosome, true,        plant nonplant domains bind Mg2&apos; Zn2+, <br /> ALA traverse peribacteroid membrane              respectively. The B. japonicum enzyme activity required Mg2+ <br /> metabolized bacteroids. Herrada et al. reported low ALA             (Fig. 5), similar plant dehydratases in <br /> uptake activity French bean symbiosomes stated              regard. The reasons anomalous structural and <br /> soybean symbiosomes quantitatively (13). How-              functional properties B. japonicum ALA dehydratase <br /> ever, ALA uptake activity shown study ca. 3              known, acquisition additional sequences may <br /> nmol/h/mg protein 25&#194;&#176;C, twofold higher           enzymes similar dehydratase <br /> rate ALA subsequently consumed B.                     described herein. We note, however, sequences in <br /> japonicum ALA dehydratase (20). Therefore, data               hand (Fig. 4) taxonomically diverse organisms, which <br />                                                                          supports argument ALA dehydratase B. <br />                                                                          japonicum unique. It plausible Mg2+-binding <br />                                                                          enzyme confers advantage bacterium plant <br />  PROPERTY                                         STRAIN                 endosymbiont; idea, speculative, particularly <br />                                                                          attractive provide means control an <br />                                   1110             MLG1         KP32     essential enzyme symbiotic context. This unanswered <br />                                                                          question broader issue plants differ from <br />  Genotype                       wild type          hemA         hemB     organisms metal component ALA dehy- <br />                                                                          dratase holoenzyme. Finally, unusual structure the <br />  Nitrogenase activitya          12.0 &#194;&#177; 2.2        12.1 &#194;&#177; 1.4         0   putative metal-binding domain B. japonicum enzyme <br />                                                                          help elucidate amino acids crucial metal <br />  Viable cell countsb             2x109              NDe         2x103    binding conventional domains dehydra- <br />                                                                          tases. <br />  Leghemoglobin hemec             87&#194;&#177; 1             78&#194;&#177;3              0 <br />                                                                                                  ACKNOWLEDGMENTS <br />  Leghemoglobin apoproteind                                                 We thank Sharon Cosloy E. coli RP523. <br />                                                                            This work supported National Science Foundation grant <br />    FIG. 6. Phenotype soybean nodules elicited B. japonicum         IBN-9204778 M. R. O&apos;Brian. <br /> strains 1110, MLG1, KP32. Footnotes: a, average activity expressed <br /> as millimoles ethylene formed hour gram nodule (fresh                                  REFERENCES <br /> weight) (plus minus standard deviation trials); b, CFU    1. Appleby, C. A. 1984. Leghemoglobin Rhizobium respiration. <br /> per gram nodule (fresh weight); c, nanomoles heme plant             Annu. Rev. Plant Physiol. 35:443-478. <br /> nodule cytosol gram nodule (fresh weight) (plus minus       2. Ausubel, F. M., R. Brent, R. E. Kingston, D. D. Moore, J. A. Smith, <br /> standard deviation trials); d, protein discerned immunologi-        J. G. Seidman, K. Struhl. 1987. Current protocols molec- <br /> cally cross-reactive bands polyacrylamide gels Western blot         ular Biology. Wiley Interscience, New York. <br /> analysis using anti-le <br /> </body></html>