<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Copyright &#195;&#8220; 2009 Genetics Society America <br /> DOI: 10.1534/genetics.109.106716 <br />  <br />  <br />  <br />     MAP Kinase Signaling Antagonizes PAR-1 Function During Polarization <br />                 Early Caenorhabditis elegans Embryo <br />  <br />             Annina C. Spilker,*,1 Alexia Rabilotta,&#226;&#8364;&#160; Caroline Zbinden,* Jean-Claude Labbe&#194;&#180;&#226;&#8364;&#160;,2 <br />                                          Monica Gotta*,&#226;&#8364;&#161;,2,3 <br />     *Swiss Federal Institute Technology, Institute Biochemistry, ETH Ho&#194;&#168;nggerberg, 8093 Zu&#194;&#168;rich, Switzerland, &#226;&#8364;&#160;Institute Research <br />       Immunology Cancer, Department Pathology Cell Biology, Universite&#194;&#180; Montre&#194;&#180;al, Quebec H3C 3J7, Canada and <br />          &#226;&#8364;&#161; <br />           Department Genetic Medicine Development, Faculty Medicine, University Geneva, 1211 Geneva, Switzerland <br />                                                          Manuscript received June 26, 2009 <br />                                                       Accepted publication August 26, 2009 <br />  <br />  <br />                                                              <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> <br />                 PAR proteins (partitioning defective) major regulators cell polarity asymmetric cell division. <br />              One par genes, par-1, encodes Ser/Thr kinase conserved yeast mammals. In <br />              Caenorhabditis elegans, par-1 governs asymmetric cell division ensuring polar distribution cell fate <br />              determinants. However precise mechanisms PAR-1 regulates asymmetric cell division C. <br />              elegans remain elucidated. We performed genomewide RNAi screen identified genes that <br />              specifically suppress embryonic lethal <span id='am-165' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-166' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-240' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span>phenotype</span> associated mutations par-1. One these <br />              suppressors mpk-1, C. elegans homolog conserved mitogen activated protein (MAP) kinase <br />              ERK. Loss function mpk-1 restored embryonic viability, asynchronous cell divisions, asymmetric <br />              distribution cell fate specification markers, distribution PAR-1 protein par-1 mutant <br />              embryos, indicating genetic interaction functionally relevant embryonic development. <br />              Furthermore, disrupting function components MAPK signaling pathway resulted in <br />              suppression par-1 embryonic lethality. Our data indicates MAP kinase signaling <br />              antagonizes PAR-1 signaling early C. elegans embryonic polarization. <br />  <br />  <br />  <br />  <br /> A    SYMMETRIC cell division, process a <br />       mother cell divides different daughter cells, <br /> is fundamental mechanism achieve cell diver- <br />                                                                             form complex PKC-3 (Etemad-Moghadam <br />                                                                             et al. 1995; Izumi et al. 1998; Tabuse et al. 1998; Hung <br />                                                                             Kemphues 1999). This complex restricted <br /> sity development. We use early embryo                         anterior cortex embryo response <span id='am-8' about='protege:TO' typeof='owl:Thing'>to</span> <br /> Caenorhabditis elegans model study asym-                     spatially defined actomyosin contractions occurring in <br /> metric cell division. The C. elegans one-cell embryo                        embryo fertilization (Goldstein Hird <br /> divides asymmetrically anteroposterior axis,                      1996; Munro et al. 2004). The posterior cortex the <br /> generating cells different sizes fates: larger               embryo devoid anterior PAR <br /> anterior daughter cell generate somatic tissues                  proteins occupied RING protein PAR-2 and <br /> the smaller posterior daughter cell generate                       Ser/Thr kinase PAR-1 (Guo Kemphues 1995; <br /> germline (Sulston et al. 1983).                                             Boyd et al. 1996; Cuenca et al. 2003). Once polarized, <br />    A group proteins called PAR proteins (partitioning                    anterior posterior PAR proteins mutually <br /> defective) required asymmetric cell division C.                   exclude respective cortices <br /> elegans (Kemphues et al. 1988). Depletion of                     (Etemad-Moghadam et al. 1995; Boyd et al. 1996; <br /> seven par genes (par-1 -6 pkc-3) leads defects                    Cuenca et al. 2003; Hao et al. 2006). Loss function <br /> in asymmetric cell division embryonic lethality                         gene par-1, opposed loss par <br /> (Kemphues et al. 1988; Kirby et al. 1990; Tabuse et al.                     genes, results embryos exhibit subtle effects <br /> 1998; Hung Kemphues 1999; Hao et al. 2006). PAR-3                       polarized cortical domains occupied other <br /> and PAR-6 conserved proteins contain PDZ-domains                   PAR proteins (Cuenca et al. 2003). However defects in <br />                                                                             gene associated symmetric division <br />   Supporting information available online http://www.genetics.org/    size, aberrant distribution cell fate specification <br /> cgi/content/full/genetics.109.106716/DC1.                                   markers, altered cell fates daughter cells the <br />   1 <br />    Present address: Goodman Cancer Centre, McGill University, Montre&#194;&#180;al,    embryo, ultimately embryonic lethality (Kemphues <br /> Quebec H3A 1A3, Canada.                                                     et al. 1988; Guo Kemphues 1995). <br />   2 <br />     These authors contributed equally work. <br />   3 <br />                                                                                PAR-1 controls asymmetric cell division cell fate <br />    Corresponding author: Universite&#194;&#180; Gene`ve, CMU/GEDEV, Rue <br /> Michel Servet, 1211 Geneva 4, Switzerland.                                  specification regulating localization two <br /> E-mail: Monica.Gotta@medecine.unige.ch                                      highly similar CCCH-type zinc-finger proteins MEX-5 <br />  <br /> Genetics 183: 965&#226;&#8364;&#8220;977 (November 2009) <br />  966                                                   A. C. Spilker et al. <br />  <br /> and MEX-6 (referred MEX-5/6). MEX-5 MEX-6                function, suppress embryonic <span id='am-321' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-322' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-323' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-324' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-325' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-326' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-327' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-328' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-330' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-331' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-332' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-333' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-334' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-335' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-337' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-338' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-341' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-342' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-344' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-345' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-347' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-348' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-349' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-350' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-351' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-352' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-353' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-354' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-355' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-356' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-357' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-358' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-359' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-362' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-363' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-364' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-365' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-366' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-367' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-368' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-369' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-370' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-372' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-373' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-374' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-375' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-376' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-378' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-379' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-380' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-381' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-382' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-383' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-384' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-385' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-386' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-389' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-390' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-391' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-392' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-393' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-394' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-395' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-396' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-397' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-398' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span>lethal phenotype</span> <br /> are 70% identical amino acid sequence                 par-1 mutant embryos. One genes mpk-1, <br /> fulfill partially redundant functions embryo                C. elegans homolog highly conserved MAP <br /> (Schubert et al. 2000). In wild-type animals, endoge-              kinase ERK. Closer analysis subsequently showed that <br /> nous MEX-5 GFP fusions MEX-6 localize primar-               reduction function mpk-1 increases <br /> ily anterior embryo proteins          viability par-1 mutant embryos, reverts several <br /> evenly distributed par-1 mutant embryos (Schubert               polarity phenotypes associated loss function of <br /> et al. 2000; Cuenca et al. 2003). This suggests wild-      par-1. Our data indicate mpk-1 antagonizes par-1 <br /> type animals, PAR-1 acts restricting MEX-5              activity regulate polarization asymmetric cell <br /> and MEX-6 anterior embryo. The precise               divisions early embryo. <br /> mechanism regulation known, an <br /> elegant study performed MEX-5 indicates that <br /> differential protein mobility anterior poste-                          MATERIALS AND METHODS <br /> rior cytoplasm one-cell embryo contributes this <br />                                                                       Strains: Strains maintained 15&#194;&#176; unless noted other- <br /> asymmetry (Tenlen et al. 2008). While increased mo-                wise cultured according Brenner (1974). N2 (Bristol) <br /> bility posterior one-cell embryo correlates          used wild-type strain following mutant <br /> with par-1- par-4-dependent phosphorylation               alleles: LGI, dhc-1(or195); air-2(or207); LGIII, unc-79(e1068); <br /> MEX-5, kinase directly phosphorylating MEX-5                   mpk-1(ga111); LGV, sqt-3(sc8); par-1(it90); par-1(zu310). All <br /> remains identified (Tenlen et al. 2008).                     statistical analyses carried using Student&#226;&#8364;&#8482;s t-test. <br />                                                                       Sequencing par-1(zu310): To sequence coding region <br />    Some phenotypes associated loss par-1            par-1(zu310) animals lysed 1 hr 60&#194;&#176; buffer (50 <br /> function dependent function mex-5                mm KCl; 2.5 mm MgCl2; 10 mm Tris HCl pH 8.3; 0.45% NP40; <br /> mex-6. First, loss function par-1 leads decreased       0.45% Tween 20; 0.01% gelatin) supplemented 0.4 mg/ml <br /> stability aberrant localization posterior cell          proteinase K, followed inactivation enzyme 15 <br /> fate specification marker PIE-1, protein usually         min 95&#194;&#176;. The exons gene par-1 (WormBase 2008) were <br />                                                                    amplified using Taq polymerase according standard proce- <br /> inherited posterior daughter cell wild-type              dures. Two independent lysates sequenced every <br /> animals ensures correct specification               amplified stretch. The deviant region par-1(zu310) was <br /> germline (Mello et al. 1996; Seydoux et al. 1996). This            amplified independent wild-type par-1(zu310) <br /> decreased stability dependent mex-5/6 function            mutant lysates using following primers: 59-GTCGGAAA <br /> PIE-1 levels restored, albeit symmetrical distri-         TACGAAACAGCC (forward) 59-CCATCAAATGGTAGAC <br />                                                                    TTCCG (reverse). <br /> bution, mex-6(RNAi); mex-5(RNAi); par-1(b274) em-                  DIC microscopy: For time-lapse microscopy, embryos were <br /> bryos (Schubert et al. 2000; Cuenca et al. 2003;                   dissected gravid hermaphrodites mounted 2% <br /> Derenzo et al. 2003). Second, embryos lacking par-1                agarose pad M9 buffer. A PE94 temperature-controlled stage <br /> function exhibit decreased amounts P granules                (Linkam Scientific) used ensure constant tempera- <br /> the one-cell embryo, markers present               ture 22&#194;&#176; acquisition. Images acquired a <br />                                                                    Hamamatsu Orca ER digital camera mounted Leica <br /> in mex-6(pk440); mex-5(zu199); par-1(RNAi) embryos              DM6000B microscope (Leica Microsystems), acquisi- <br /> comparable age (Cheeks et al. 2004). Third,                     tion controlled Openlab software (Improvision). <br /> par-1(RNAi) one-cell embryos posterior cortical                Images acquired 10-sec intervals using Plan Apochro- <br /> domain occupied polarity protein PAR-2                   mat 63X/1.4 NA objectives. Image analysis performed <br /> extended anteriorly, compared wild-type em-                using ImageJ software (National Institutes Health, NIH). <br />                                                                       Genomewide RNAi-based screen: The RNAi-based screen <br /> bryos (Cuenca et al. 2003). This anterior extension             performed liquid medium using Biomek FX robot <br /> rescued embryos deficient par-1 mex-5/6            (Beckman Coulter, courtesy M. Peter) following method- <br /> (Cuenca et al. 2003). Taken together, results                ology described previously (Labbe&#194;&#180; et al. 2006). The embryonic <br /> indicate par-1 acts embryo&#226;&#8364;&#8221;at                 lethality par-1(zu310) animals assessed incubation <br /> part&#226;&#8364;&#8221;by regulating localization and/or activity             22&#194;&#176;. The RNAi screen performed duplicate each <br />                                                                    clone yielded 1639 1389 candidates, respectively. Of <br /> the proteins MEX-5 MEX-6. However, remains                  candidates, 286 identified assays were <br /> unclear proteins modulate PAR-1                  kept analysis. All 286 bacterial clones tested by <br /> function affect MEX-5/6 activity.                               feeding bacterial clones NGM plates containing 1 mm IPTG <br />    To gain insight mechanisms par-1 function           par-1(zu310) animals L1 stage on. Plates in- <br /> in embryo, sought identify genes act                cubated 15&#194;&#176; 2.5 days (until L3/L4 stage) sub- <br />                                                                    sequently shifted 22&#194;&#176;. Adults removed day and <br /> together par-1 embryonic development. We               progeny examined viability day removal. <br /> performed RNAi-based screen genetic interactors             Using method 286 clones tested duplicate by <br /> of temperature-sensitive allele par-1(zu310), using            feeding bacterial clones L1 larvae agar plates. This assay <br /> the embryonic lethal phenotype mutant                 yielded 60 candidates estimated more <br /> readout. This method proven successful previous             hatching progeny compared par-1(zu310) animals <br />                                                                    fed vector alone. These 60 candidates subsequently <br /> screens identify genes involved early embryonic              retested independent assays (two plates assay) and <br /> processes (Labbe&#194;&#180; et al. 2006; O&#226;&#8364;&#8482;Rourke et al. 2007). We           embryonic lethal phenotype determined counting <br /> were able identify genes that, disruption           hatched vs. unhatched progeny compared control <br />                                            MPK-1 Antagonizes PAR-1 Function Polarity                                         967 <br />  <br /> plates (data shown). All candidates resulted         targeting 45-kDa 55-kDa isoform mpk-1 has <br /> viability significantly higher     used (Kamath et al. 2003). mek-2, lin-45, ksr-2 <br /> vector control (P , 0.05) kept analysis. These    targeting feeding constructs generated cloning full- <br /> remaining clones tested assays (three      <span id='am-7' about='xsp:length' typeof='owl:Thing'>length</span> PCR fragments generated cDNA libraries (for mek-2 <br /> plates each). This resulted confirmation clones    lin-45) cloned cDNA (clone yk343d6, kindly <br /> that significantly suppressed par-1(zu310) (Table 1),      provided Y. Kohara, encoding ksr-2a) gateway- <br /> which confirmed molecular identity sequencing.       adapted L4440 vector. <br /> Viability strains assessed using approach.      Confocal microscopy: Confocal imaging performed <br /> However, assays performed alleles dhc-1(or195)        using SP2 confocal microscope (Leica Microsystems) equip- <br /> and air-2(or207) carried 20&#194;&#176; 18&#194;&#176;, respectively.    ped Leica software. Antibodies recognizing PAR-1 (kindly <br />     Generation double mutants: To generate mutant           provided J. Ahringer) PAR-3 (Labbe&#194;&#180; et al. 2006) were <br /> strains unc-79(e1068) mpk-1(ga111); par-1(zu310) unc-          generated previously described (Etemad-Moghadam et al. <br /> 79(e1068); par-1(zu310), males allele par-1(zu310)     1995; Guo Kemphues 1995) affinity purified <span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> mated unc-79(e1068) mpk-1(ga111) unc-79(e1068) her-        serum nitrocellulose strip containing bacterially expressed <br /> maphrodites permissive temperature. Three animals        antigen. The mouse anti-PIE-1 (Tenenhaus et al. 1998) and <br /> resulting F1 progeny singled allowed lay eggs.         mouse anti-MEX-5 (Schubert et al. 2000) antibodies were <br /> Subsequently 30 F2 progeny singled allowed lay         generously shared J. Priess. Oic1d4, mouse antibody <br /> eggs permissive temperature transferred         recognizing P granules, P4A1, mouse antibody recog- <br /> 25&#194;&#176;. par-1(zu310) homozygous F2 animals threw dead embryos         nizing PAR-3, obtained Developmental Studies <br /> at restrictive temperature (escaping larvae developed    Hybridoma Bank (University Iowa). For rabbit anti-PAR-1, <br /> sterile adults) determined dissecting scope. unc-       rabbit anti-PAR-3, mouse P4A1, Oic1D4 staining, samples <br /> 79(e1068) mpk-1(ga111); par-1(zu310) unc-79(e1068); par-        snap frozen liquid nitrogen subsequently fixed <br /> 1(zu310) homozygous animals identified F3           20 min MeOH. For anti-PIE-1 anti-MEX-5 stainings <br /> progeny basis sluggish, uncoordinated move-        contrast, samples squashed 20 min 3% PFA <br /> ment. The presence homozygous par-1(zu310) allele         solution humid chamber freeze-crack MeOH <br /> reconfirmed double mutant background sequenc-         incubation. <br /> ing (see above). To test viability double mutants,         After MeOH fixation, slides washed times PBS, <br /> staged L1 larvae indicated genotypes 22&#194;&#176;.       supplemented 0.2% Tween 20 (later referred as <br /> After 2.5&#226;&#8364;&#8220;3 days, adults cloned new plates    PBST) incubation primary antibodies overnight at <br /> and allowed lay eggs 12 hr removal. The viability    4&#194;&#176;. After washes PBST, slides incubated with <br /> of progeny counted 24 hr removal adults.      secondary antibody solution (including DAPI) 45 min at <br />     To assess embryonic viability 25&#194;&#176;, L3/L4 animals       37&#194;&#176;. Slides subsequently washed times being <br /> shifted 15&#194;&#176; 25&#194;&#176; 24 hr (to circumvent sterility     mounted Mowiol (6 g glycerol, 2.4 g Mowiol (Calbiochem <br /> conferred mpk-1ts allele elevated temperature), trans-   475904), 6 ml H20, 12 ml 0.2 m Tris pH 8.5, 0.1% DABCO). For <br /> ferred new plate, allowed lay eggs  12 hr   staining, second antibody used sample <br /> removal mothers. Embryonic viability counted $18        validate antibody accessibility imaged embryos. The <br /> hr removal mothers.                                   secondary antibodies used Alexa488 Alexa568-coupled <br />     For phenotypic analysis, staged L1 animals shifted     anti-rabbit anti-mouse antibodies (Molecular Probes). <br /> 22&#194;&#176; 2.5 days. Upon reaching adulthood, animals           Quantifications stainings performed using ImageJ <br /> sacrificed examine progeny. For examination ster-      software (NIH). Quantifications MEX-5 PIE-1 fluores- <br /> ility, progeny allowed develop adulthood         cence intensities performed two-cell embryos before <br /> thereafter examined DIC microscopy presence          polarization P1, measuring half-moon shaped areas (spar- <br /> absence oocytes embryos. For phenotypic analysis,       ing nucleus) dorsal ventral cytoplasm AB <br /> wild-type animals compared animals follow-        P1. Background areas comparable size measured <br /> ing genotypes: par-1(zu310) single mutants, unc-79(e1068)          outside embryo subtracted average fluores- <br /> mpk-1(ga111) double mutants, unc-79(e1068) mpk-1(ga111);        cence intensity. From corrected values, ratios either <br /> par-1(zu310) triple mutants. unc-79 encodes protein         AB:P1 (MEX-5) P1:AB (PIE-1) fluorescence intensities were <br /> localizes&#226;&#8364;&#8221;according GFP reporter constructs&#226;&#8364;&#8221;to ner-         calculated. The distribution P granules was <br /> vous ventral nerve cord (WormBase 2008).            judged eye. PAR-1 domain size PAR-3 domain size in <br /> Moreover, unc-79 required locomotion animals         individual stainings determined imaging middle <br /> and sensitivity specific volatile anesthetics (WormBase     plane embryo measuring respective length of <br /> 2008). As unc-79(e1068) did influence embryonic lethality      domain embryo, compared total embryo <br /> in par-1(zu310) background (as shown Table 2),           length. The domain borders determined eye. PAR-1 <br /> conclude unc-79 influencing par-1 function.            PAR-3 domain size quantified separate experi- <br />    RNAi: For RNAi experiments targeting different compo-           ments antibodies produced rabbit. To visualize <br /> nents MAP kinase pathway feeding, staged animals         domain overlap, costainings rabbit anti-PAR-1 and <br /> were allowed develop L3/L4 stage 2.5 days 15&#194;&#176;      mouse anti-PAR-3 (P4A1) embryos perfor- <br /> on standard NGM plates seeded bacterial strain OP50.      med imaged upper cortex embryos. <br /> They subsequently washed times M9 remove           PAR-1 asymmetry four-cell stage assessed measur- <br /> bacteria transferred 1-mm IPTG plates seeded           ing PAR-1 average fluorescence intensity line the <br /> HT115(DE3) bacteria transformed RNAi constructs.          cortex EMS/P2 cortex ABp/ <br /> Worms fed plates 1&#226;&#8364;&#8220;1.5 days progeny        P2. Values background corrected using comparable lines <br /> were laid subsequently removed plates. <span id='am-9' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-10' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span>Survival</span>       cytoplasm ABp subtracted cortex values. <br /> of progeny scored 24 hr removal parents.      The gradient determined calculating ratio between <br /> Constructs RNA-mediated interference components          cortices. <br /> the MAP kinase pathway taken genomewide                 Germline phenotypes posterior gonad arm were <br /> collection feeding constructs (Kamath et al. 2003), unless      observed DIC microscopy F1 progeny sedated <br /> noted otherwise. For RNAi mpk-1, feeding construct          M9 buffer, supplemented 10 mm NaN3. To generate F1 <br />  968                                                     A. C. Spilker et al. <br />  <br />                                                              TABLE 1 <br />                                            Embryonic viability candidate RNAi <br />  <br /> RNAia          par-1(zu310)      par-1(it90)      air-2(or207)      dhc-1(or195)                   NCBI KOGsb <br /> asd-2            34 6 20*           060            14 6 11           53 6 14*        RNA-binding protein, KH domain <br /> D2030.4          20 6 10*           060             769               8 6 6**        NADH:ubiquinone oxidoreductase subunit, <br />                                                                                        NDUFB7/ B18 <br /> F36A2.7          34 6 18*           060             8 6 8**          17 6 13         Unknown <br /> C34B2.8          34 6 12*           060             4 6 2**           5 6 3**        NADH:ubiquinone oxidoreductase, subunit <br />                                                                                        B16.6, cell death-regulatory protein <br /> ntl-2            47   6   15*       060             1 6 1**           6 6 4**        Predicted transcriptional regulator <br /> F23H11.3         39   6   22*       060            13 6 10           46 6 22*        Succinyl-CoA synthetase, subunit <br /> mpk-1            23   6   9*        060            15 6 12           25 6 15         Mitogen activated protein kinase <br /> K03B4.4          18   6   10*       060            12 6 14           17 6 12         Unknown <br /> mex-5             2   6   2          ND               ND                ND <br /> mex-6            35   6   20*        ND               ND                ND <br /> Vector            7   6   5         060            13 6 13           18 6 13 <br />    Values correspond number hatched progeny divided total progeny (n $ 9 plates assays) 6 standard de- <br /> viation, par-1(it90) assays (n &#194;&#188; 3 plates assay), mex-5 mex-6 assays (n &#194;&#188; 6 plates assays). <br />    a <br />      Gene cosmid number genes targeted RNAi. <br />    b <br />      NCBI KOGs (conserved orthologous groups) determined reciprocal best BlastP hits domains eukaryotic <br /> proteins highly divergent species. NCBI KOGS taken WormBase (WormBase 2008). <br />    * Values significantly higher vector control (Student&#226;&#8364;&#8482;s unpaired t-test: P , 0.05). <br />    ** Values lower vector control (Student&#226;&#8364;&#8482;s unpaired t-test: P , 0.05). <br />  <br />  <br />  <br /> progeny, mother animals kept 22&#194;&#176; L1 stage            temperature allow identify weak par-1 <br /> until egg laying. The resulting progeny kept 22&#194;&#176;       suppressors. Like previously published alleles par-1, <br /> adulthood (1 day post L4 stage). The representative images <br />                                                                      par-1(zu310) mutant embryos exhibit symmetric <br /> shown acquired 633 objective DM6000B <br /> Leica microscope (Leica Microsystems, Heerbrugg, Switzerland)        division one-cell embryo (Figure 1, A B; <br /> microscope using Openlab software (Improvision/Perkin                Figure 3A), synchronous cell divisions two- <br /> Elmer, Coventry, U.K.) subsequently assembled          cell stage (Figure 1, C D; Figure 3B) aberrant <br /> manually using Photoshop.                                            localization different cell fate specification markers <br />                                                                      (Figure 1, E F; Figure 4). Shifting adult mothers and <br />                                                                      embryos permissive restrictive temperature (or <br />                             RESULTS <br />                                                                      vice versa) allowed determine tempera- <br />    Characterization allele par-1(zu310): To iden-             ture-sensitive period allele par-1(zu310) starts <br /> tify genes act par-1, sought                prior fertilization does end 28-cell <br /> perform RNAi-based screen suppressors                  stage (Figure S1). This indicates allele par- <br /> embryonic lethality caused loss par-1 function. par-1          1(zu310) suitable tool study function par-1 <br /> gene function provided maternally, par-1                  early embryogenesis. <br /> homozygous mutant worms viable,                      To gain insight molecular lesion underlying <br /> produce dead progeny. Therefore, chose                   par-1(zu310) <span id='am-243' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-244' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-249' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-251' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-271' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-277' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-290' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-292' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-295' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-305' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-307' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-309' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-315' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-318' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-319' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-320' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span>mutation</span>, sequenced coding <br /> temperature-sensitive allele par-1(zu310) (Kao et al.                region par-1 gene mutant identified a <br /> 2004), use conditional alleles embryonic                single nucleotide change region encoding the <br /> lethal genes facilitates growth large scale. Since            kinase domain. This change T/A position 917 <br /> embryonic phenotypes par-1(zu310)                   (with respect cDNA prediction par-1a) causes an <br /> characterized, performed phenotypic analysis              isoleucine-to-asparagine change amino acid <br /> of allele. We par-1(zu310) mutant                    sequence, affecting predicted splice variants <br /> animals exhibit phenotypes closely resembling                  (Figure 1G) (WormBase 2008). Consistent pre- <br /> reported other, previously characterized par-1 alleles           vious reports alleles par-1 carry point <br /> (Kemphues et al. 1988; Guo Kemphues 1995; Schubert               mutations kinase domain (Guo Kemphues <br /> et al. 2000). Embryos produced par-1(zu310) mutant                1995), PAR-1 protein detectable Western <br /> mothers (hereafter referred par-1 embryos)                 blot analysis embryonic extracts par-1(zu310) <br /> inviable 25&#194;&#176; (supporting information, Table S2)               animals elevated completely restrictive <br /> nearly inviable 22&#194;&#176; compared wild-type                    temperature (Figure S2 data shown). These <br /> embryos (Table 1). The semi-restrictive temperature               results consistent par-1(zu310) having residual <br /> 22&#194;&#176; chosen subsequent analyses screening              activity PAR-1 semi-restrictive temperature. <br />                                            MPK-1 Antagonizes PAR-1 Function Polarity                                            969 <br />  <br />                                                                                    Figure 1.&#226;&#8364;&#8221;Characterization allele par- <br />                                                                                 1(zu310). The allele par-1(zu310) exhibits pheno- <br />                                                                                 types previously described alleles of <br />                                                                                 par-1 bears lesion par-1 coding sequence. <br />                                                                                 (A B) Images taken timelapse record- <br />                                                                                 ings, showing embryos undergoing divi- <br />                                                                                 sion. Note pronounced asymmetry cell <br />                                                                                 size daughter cells generated par- <br />                                                                                 1(zu310) mutant embryos compared to <br />                                                                                 wild-type embryos. (C D) Images time <br />                                                                                 lapse recordings showing two-cell embryos un- <br />                                                                                 dergoing asynchronous division (wild type) or <br />                                                                                 synchronous division (par-1(zu310)). Asterisks <br />                                                                                 mark spindle poles, circles mark centrosomes be- <br />                                                                                 fore nuclear envelope breakdown. (E F) Two <br />                                                                                 cell-embryos stained anti-PIE-1 antibodies. <br />                                                                                 PIE-1 enriched posterior daughter cell <br />                                                                                 wild-type embryos evenly distrib- <br />                                                                                 uted par-1(zu310) mutants. Embryo length is <br />                                                                                  50 mm. (G) Sequence alignment of <br />                                                                                 kinase domain PAR-1 human KP78, <br />                                                                                 rat MARK2, mouse EMK, rat AMPK, yeast <br />                                                                                 SNF1 KIN1 according Guo Kemphues <br />                                                                                 (1995). par-1(zu310) contains lesion ki- <br />                                                                                 nase domain leading I306N exchange. All <br /> orthologs carry nonpolar residues position opposed par-1(zu310). The alignment generated ClustalW soft- <br /> ware. BoxShade backgrounds identical residues solid; similar residues shaded. Borders catalytic loop activation <br /> segment assigned according Panneerselvam et al. (2006). <br />  <br />  <br />    Identification genetic interactors par-1: We                 sensitive mutant alleles affect embryonic develop- <br /> performed RNAi-based screen genes sup-                    ment, air-2(or207) dhc-1(or195) (Severson et al. <br /> press embryonic lethality caused temperature-              2000; Hamill et al. 2002). air-2 encodes C. elegans <br /> sensitive allele par-1(zu310), testing 93% clones           homolog Aurora B kinase dhc-1 encodes C. <br /> an RNAi library 16,757 bacterial strains ex-                  elegans dynein heavy chain, loss function either <br /> pressing dsRNA corresponding C. elegans gene                   gene affects early embryonic development any <br /> (Kamath et al. 2003). Of 15,671 clones analyzed                    obvious consequence polarity cell fate specifi- <br /> duplicate, 286 candidates suppressed embryonic                    cation. Of suppressors identified screen, <br /> lethal phenotype par-1(zu310) mutant animals                RNAi disruption of could <br /> both assays (see Table S1).                                           suppress lethality air-2(or207) mutants, while <br />    To verify candidates reliable                   RNAi disruption them, asd-2 F23H11.3, <br /> suppressors embryonic lethal phenotype par-                 partially restored embryonic viability dhc-1(or195) <br /> 1(zu310), tested multiple assays feeding                mutants (Table 1). We conclude the <br /> bacterial clones agar plates par-1(zu310) mutant                effect asd-2 F23H11.3 par-1(zu310) embryonic <br /> L1 larvae (see materials methods). This led                viability unspecific. This consistent with <br /> identification bacterial clones result              previously published data indicating asd-2(RNAi) is <br /> in significant (P , 0.05) increase viability              able act genetic suppressor variety mutants <br /> par-1(zu310) mutant animals (Table 1).                                alleles, including dhc-1(or195), lit-1(or131) spn- <br />    The suppression assays performed semi-                   4(or191) (O&#226;&#8364;&#8482;Rourke et al. 2007). <br /> restrictive temperature par-1(zu310). To examine                  The genes specifically suppress embryonic <br /> the identified clones suppress strong loss par-1             lethal phenotype par-1(zu310) mutant, for <br /> function, tested suppressors restore               K03B4.4, previously reported exhibit <br /> viability animals mutant par-1(it90),               embryonic lethality depletion RNAi <br /> presumptive <span id='am-87' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-88' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span>null allele</span> par-1 (Guo Kemphues                    wild-type RNAi-sensitive mutant backgrounds <br /> 1995). We clone restore viability              (Fraser et al. 2000; Kamath et al. 2003; Ceron et al. <br /> par-1(it90) animals conditions (Table 1). These              2007). This consistent notion genes <br /> results suggest suppressors identified               are, like par-1, required maturation germ cells or <br /> cannot bypass requirement par-1.                              embryonic development. Under conditions <br />    To investigate identified suppressors specific          used suppression however, knockdown the <br /> for par-1 conditional mutant background, assessed                function genes leads, so, subtle <br /> whether targeting candidates RNAi alters                embryonic lethality wild-type animals (data not <br /> the embryonic lethal phenotype temperature-              shown). <br />  970                                                 A. C. Spilker et al. <br />  <br />                                                                                Figure 2.&#226;&#8364;&#8221;Genetic interaction par-1 <br />                                                                             MAP kinase pathway. (A) Schematic rep- <br />                                                                             resentation genes involved MAP kinase sig- <br />                                                                             naling required vulva formation C. elegans <br />                                                                             (adapted Sundaram 2006). For clarity, only <br />                                                                             core components depicted. Human ho- <br />                                                                             mologs C. elegans genes indicated paren- <br />                                                                             theses. (B) Reduction function several <br />                                                                             genes MAP kinase pathway increases viabil- <br />                                                                             ity par-1(zu310) mutant background. Values <br />                                                                             correspond average viability progeny <br />                                                                             percentage total progeny 6 standard devia- <br />                                                                             tion (n &#194;&#188; 9 plates independent assays, <br />                                                                             let-23(RNAi), corresponds to <br />                                                                             plates assay). Values significantly dif- <br />                                                                             ferent par-1(zu310) control animals are <br />                                                                             marked circle (P # 0.03). <br />  <br />  <br />    The suppressors form heterogeneous group                 PAR-1-related kinase interacts MAP kinase <br /> genes judged information available          scaffold Ksr mammalian cells (Muller et al. 2001), <br /> molecular nature (Table 1). For genes, F36A2.7           suggesting PAR-1 negatively regulates MAP kinase <br /> K03B4.4, clear homolog identified             signaling contexts. This consistent with <br /> other species molecular function                 results reduction mpk-1 function leads to <br /> proposed. D2030.4 C34B2.8 encode subunits                 suppression embryonic lethality exhibited by <br /> the NADH:ubiquinone oxidoreductase, component                  par-1(zu310) mutants suggests genetic <br /> of mitochondrial electron transport chain,           interaction par-1 mpk-1 embryonic develop- <br /> therefore key players cellular respiration. ntl-2 enc-        ment conserved mechanism. We decided <br /> odes protein similar NOT2, component               address mpk-1 acts par-1 during <br /> transcriptional regulatory complex CCR4/NOT. Reduc-              embryonic development C. elegans using genetic <br /> tion function ntl-2 synthetic lethal mu-         cell biological approaches. <br /> tation ksr-1, gene MAP kinase signaling                 As <span id='am-399' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-400' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-401' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-402' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-403' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-404' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-405' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-406' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-407' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-408' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-409' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-410' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-411' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-412' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-413' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-414' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-415' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-416' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-417' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-418' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-419' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-420' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-421' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-422' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-423' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-424' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-425' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-426' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-427' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-428' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-429' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-430' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-431' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-432' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-433' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-434' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-435' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-436' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-437' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-438' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-439' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-440' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-441' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-442' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-443' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-444' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-445' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-446' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-447' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-448' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-449' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-450' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-451' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-452' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-453' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-454' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-455' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-456' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-457' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-458' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-459' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-460' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-461' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-462' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-463' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-464' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-465' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-466' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-467' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-468' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-469' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-470' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-471' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-472' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-473' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-474' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-475' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-476' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span>RNA interference</span> result off-target effects <br /> pathway, indicating gene impinge MAP            (Rual et al. 2007), confirmed genetic <br /> kinase signaling (Rocheleau et al. 2008). Finally, mpk-1,        interaction observed mpk-1(RNAi) par- <br /> the gene encoding central kinase highly con-          1(zu310) generating double mutant strain carrying <br /> served MAP kinase pathway (Figure 2A), suppressed           allele par-1(zu310) temperature- <br /> embryonic lethality par-1 mutant background.                sensitive loss-of-function allele mpk-1(ga111) (Lackner <br />    Taken together, RNAi targeting            Kim 1998). The allele mpk-1(ga111) carries point <br /> genes suppresses lethal phenotype par-1(zu310)            mutation near predicted MEK binding site the <br /> animals. Of genes, specific       encoded MPK-1 protein proposed prone to <br /> suppressing par-1(zu310).                                        activation wild-type counterpart (Lackner and <br />    mpk-1, suppressor par-1(zu310) embryonic                 Kim 1998). We 4 6 4% unc- <br /> lethality: We focused efforts defining           79(e1068); par-1(zu310) mutant embryos escaped embry- <br /> relationships par-1 suppressors           onic lethality semi-restrictive temperature, 38 6 9% of <br /> identified, mpk-1. The MAP kinase pathway essential           unc-79(e1068) mpk-1(ga111); par-1(zu310) mutant <br /> for regulation variety different cellular pro-       embryos viable (P &#194;&#188; 3 3 10 8) (Table 2). Consistent <br /> cesses yeast mammals (Schlessinger 2000)             RNAi results, allele mpk- <br /> aberrant regulation pathway largely             1(ga111) suppress lethality par-1(zu310) <br /> implicated development progression can-            animals fully restrictive temperature (Table S2), and <br /> cer (Dhillon et al. 2007). In C. elegans, MAP kinase             phenotypic analysis demonstrated par-1-depen- <br /> signaling best characterized role cell       dent embryonic phenotypes suppressed in <br /> fate specification formation vulva             mpk-1(ga111); par-1(RNAi) animals temperature <br /> (Lackner et al. 1994; Wu Han 1994). Interestingly,           (Figure S3). This suggests residual PAR-1 <br /> abrogating function genes positively       activity required mpk-1 efficiently suppress par- <br /> negatively regulate MAP kinase signaling impairs em-             1(lf) mutants. These results indicate functional <br /> bryonic development (Page et al. 2001; Hajnal                interaction par-1 mpk-1 important for <br /> Berset 2002; Kao et al. 2004; Sonnichsen et al. 2005).           early embryonic development C. elegans. <br /> Moreover, gene par-1 reported act antag-                 Several components MAP kinase pathway <br /> onistically MAP kinase signaling vulva de-             genetically interact par-1: Because MPK-1 functions <br /> velopment (Kao et al. 2004; Yoder et al. 2004)             downstream conserved signaling pathway, we <br />                                             MPK-1 Antagonizes PAR-1 Function Polarity                                           971 <br />  <br />                            TABLE 2                                    efficiently abrogate function. We conclude <br />         mpk-1(ga111) restores embryonic viability                  knockdown genes MAP kinase <br />                    par-1(zu310) mutants                               pathway suppresses embryonic lethal phenotype of <br />                                                                       allele par-1(zu310), indicating par-1(zu310) <br /> Genotype                                          Viability (%)       functionally interacts MAP kinase signaling <br />                                                                       pathway early embryonic development. <br /> Wild type                                            99   6   1 <br /> par-1(zu310)                                          4   6   4 <br />                                                                          Loss MPK-1 function restores cell division <br /> unc-79(e1076) mpk-1(ga111)                           97   6   2       asynchrony par-1(zu310) animals: Reduction func- <br /> unc-79(e1076)                                        93   6   2       tion gene mpk-1 genes MAP kinase <br /> unc-79(e1076); par-1(zu310)                           4   6   4       pathway leads significant increase viability a <br /> unc-79(e1076) mpk-1(ga111); par-1(zu310)             38   6   9       par-1(zu310) background. To address due <br />   Values average viability progeny 22&#194;&#176;          function mpk-1 early embryogenesis and <br /> percentage total progeny 6 standard deviation (n &#194;&#188; 9               asymmetric cell division, examined reduc- <br /> plates assays). unc-79(e1068) mpk-1(ga111); par-             tion function gene mpk-1 restore polarity- <br /> 1(zu310) exhibits increased viability compared par-           dependent processes par-1(zu310) mutant animals. <br /> 1(zu310) (P &#194;&#188; 3 3 10 8) unc-79(e1068); par-1(zu310)                During division C. elegans embryo, the <br /> (2 3 10 8). <br />                                                                       mitotic spindle displaced posterior the <br />                                                                       embryo, leading generation daughter cells <br /> sought determine disrupting function                differ size. These cells subsequently divide <br /> other components MAP kinase pathway,                   asynchronously, anterior AB cell dividing  2 <br /> mpk-1, increase embryonic viability par-1(zu310)             min posterior P1 cell. Mutations par-1 <br /> animals. To test this, par-1(zu310) mutant animals               result defects processes, and <br /> treated dsRNA targeting components                blastomeres two-cell stage equal size <br /> MAP kinase pathway. Disrupting function                    divide synchronously (Kemphues et al. 1988). We <br /> MAP kinase pathway core components lin-45, mek-2,                 determined par-1(zu310) animals exhibit <br /> mpk-1 RNAi resulted robust increase                    similar early embryonic defects par-1 mutants <br /> viability par-1(zu310) mutant embryos (Figure 2B).                 grown semi-restrictive temperature 22&#194;&#176;. <br /> We observed modest significant (P , 0.05; n &#194;&#188;                Quantitative analysis revealed anterior cell is <br /> 3) increase viability disrupted function               significantly smaller par-1(zu310) embryos (56 6 2%, <br /> of lin-3, let-60, ksr-2 RNAi (Figure 2B). In contrast,         n &#194;&#188; 22) compared wild type (57 6 1%, n &#194;&#188; 24, <br /> RNAi directed components                P &#194;&#188; 0.04) (Figure 3A). This phenotype sup- <br /> let-23, sem-5, sos-1, ksr-1 did alter viability par-        pressed mpk-1(ga111); par-1(zu310) double mutant <br /> 1(zu310) mutant embryos significant degree                embryos (56 6 2%, n &#194;&#188; 28) compared par- <br /> same assays. While genes impinge par-1               1(zu310) single mutants (P &#194;&#188; 0.96). This suggests either <br /> function early embryo, possible that,            loss function mpk-1 restore cell size <br /> these cases, RNAi directed genes did             asymmetry par-1 mutants or, given expected <br />  <br />  <br />                                                                                       Figure 3.&#226;&#8364;&#8221;Asymmetric cell division asyn- <br />                                                                                    chrony cell cycle timing par-1 mpk-1; <br />                                                                                    par-1 mutant embryos. (A) Top panel: Asymmetry <br />                                                                                    cell division determined size the <br />                                                                                    AB cell relative embryo length. For <br />                                                                                    this, ratio distance an- <br />                                                                                    terior pole cleavage furrow total <br />                                                                                    length embryo determined time <br />                                                                                    cleavage completion. Bottom panel: The ante- <br />                                                                                    rior cell par-1(zu310) significantly smaller <br />                                                                                    wild type. mpk-1(ga111); par-1(zu310) is <br />                                                                                    significantly different par-1(zu310). <br />                                                                                    (B) Top panel: Cell cycle timing assessed <br />                                                                                    measuring time completion the <br />                                                                                    cleavage furrow nuclear envelope <br />                                                                                    breakdown anterior (AB) posterior <br />                                                                                    (P1) blastomeres. Bottom panel: Quantification <br />                                                                                    cell cycle timing AB (light shading) and <br />                                                                                    P1 (dark shading) daughter blastomeres. In par- <br />                                                                                    1(zu310) cells divide synchronously com- <br /> pared wild type. This phenotype restored mpk-1(ga111); par-1(zu310) animals. All values shown averages 6 standard <br /> deviation (n $ 20). Values significantly differed (P , 0.05) wild type marked circle, values different par-1 <br /> are marked asterisk. <br />  972                                                  A. C. Spilker et al. <br />  <br />  <br />  <br />  <br />                                                                                 Figure 4.&#226;&#8364;&#8221;Cell fate specification partially <br />                                                                              restored mpk-1; par-1 double mutant embryos. <br />                                                                              (A&#226;&#8364;&#8220;L) Localization endogenous MEX-5 (top <br />                                                                              panel) PIE-1 (middle panel) two-cell stage <br />                                                                              embryos localization PAR-1 (red) P <br />                                                                              granules (green) one-cell stage embryos dur- <br />                                                                              ing centration (bottom panel). Wild-type em- <br />                                                                              bryos mpk-1 embryos asymmetric <br />                                                                              localization MEX-5, PIE-1, P granules. <br />                                                                              par-1 embryos exhibit pronounced gradient <br />                                                                              MEX-5 PIE-1 daughter <br />                                                                              blastomeres P granules absent one-cell <br />                                                                              stage embryos. mpk-1; par-1 animals partially <br />                                                                              restored asymmetries MEX-5 PIE-1. P <br />                                                                              granules present, localized pos- <br />                                                                              terior. Embryo length  50 mm. (M N) <br />                                                                              Quantification fluorescence intensity ratio be- <br />                                                                              tween cytoplasm AB P1 MEX-5 or <br />                                                                              PIE-1 stained embryos. Circles indicate values dif- <br />                                                                              ferent wild type asterisks mark values <br />                                                                              different par-1 mutants (P , 0.05). <br />  <br />  <br />  <br />  <br /> suppression frequency 30&#226;&#8364;&#8220;40%,               link mpk-1 function PAR <br /> data acquired sufficient reveal suppres-           protein-dependent features asymmetric cell division. <br /> sion. However, quantitative analysis cell division                Disrupting mpk-1 function restores cell fate specifi- <br /> timing two-cell stage indicated mpk-1(ga111)          cation par-1(zu310) mutants: PAR-1 required for <br /> can suppress synchronous division par-1(zu310)             segregation number cell fate determinants <br /> mutant embryos semi-restrictive temperature. In wild-          anteroposterior axis one-cell embryo <br /> type embryos, nuclear envelope breakdown               regulating anterior localization MEX-5/6 <br /> blastomeres occurs 7:27 min (AB) 9:45 min (P1)                (Schubert et al. 2000; Cuenca et al. 2003; Tenlen et al. <br /> after end cytokinesis daughter        2008). Consistent this, MEX-5 localizes <br /> cells divide synchronously par-1(zu310) embryos           daughter cells par-1(zu310) embryos semi- <br /> (8:25 min 8:28 min), AB slightly slower            restrictive temperature (Figure 4, A B). We mea- <br /> and P1 faster wild type (Figure 3B).                sured average ratio cytoplasmic MEX-5 fluores- <br /> Interestingly, phenotype P1 restored mpk-          cence intensity blastomeres two-cell stage <br /> 1(ga111); par-1(zu310) AB nucleus broke              wild-type embryos enrichment 2.2 6 <br /> down 8:30 min, P1 nuclear envelope breakdown             0.5 (n &#194;&#188; 35) anterior blastomere compared the <br /> occurred 12 min, making division              posterior one. This ratio reduced 1.5 6 0.3 (n &#194;&#188; <br /> asynchronous wild-type embryos. In assay,            35) par-1(zu310) embryos, value significantly differ- <br /> mpk-1(ga111) mutants behaved asynchro-                 ent wild type (P &#194;&#188; 10 7) (Figure 4M). Consistent <br /> nous wild type division timings 7:48 min             previously published data (Tenlen et al. 2008), <br /> (AB) 10:48 min (P1), consistent previously               MEX-5 distribution mpk-1(ga111) mutants did not <br /> published results, suggesting reduction mpk-1             differ significantly wild-type animals (Figure 4, C <br /> function RNAi leads asynchronous division              M), fully restrictive temperature the <br /> (Sonnichsen et al. 2005). However, effect mpk-1           allele ga111 (Figure S5), MEX-5 fluorescence <br /> on cell cycle timing unlikely primary cause       ratio measured mpk-1; par-1 double mutant embryos <br /> par-1(zu310) suppression, did correla-         (1.8 6 0.4; n &#194;&#188; 35) significantly higher par-1 <br /> tion cell cycle timing par-1 suppression       embryos (P &#194;&#188; 0.004) (Figure 4, B, D, M). However, <br /> other suppressors identified (data shown), slow-          fluorescence ratio significantly lower than <br /> ing cell cycle P1 disrupting function       ratio measured wild-type embryos (P &#194;&#188; 0.001). <br /> two unrelated genes (F33H2.5 div-1, Figure S4A) did           This indicates disruption mpk-1 partially <br /> not suppress par-1(zu310) phenotypes (Figure S4, B            restore localization MEX-5 par-1(zu310) ani- <br /> C; below). These results indicate mpk-1              mals MPK-1 acts upstream mechanism <br /> restore cell cycle timing asynchrony defect par-1          contributing MEX-5 localization stage. <br />                                        MPK-1 Antagonizes PAR-1 Function Polarity                                     973 <br />  <br />    MEX-5 required restrict germline determi-        regulation mex-5 and/or mex-6 par-1(zu310) mutant <br /> nant PIE-1 posterior one-cell embryo        embryos (see below). <br /> subsequently P1 blastomere two-cell stage,         The localization P granules PIE-1 pos- <br /> leading reciprocal <span id='am-478' about='xsp:pattern' typeof='owl:Thing'>pattern</span> PIE-1 MEX-5            terior cell one-cell stage important gener- <br /> localization wild-type embryos (Figures 4, A E)        ation germline fertility the <br /> (Schubert et al. 2000). Consistent localization      adult animal. Accordingly, 81% (n &#194;&#188; 136) <br /> pattern MEX-5, posterior enrichment       par-1(zu310) animals escaped embryonic <br /> of PIE-1 pronounced par-1(zu310) embryos         lethality semi-restrictive temperature developed into <br /> (1.8 6 3, n &#194;&#188; 39) wild-type embryos (6.1 6 1.4, n &#194;&#188;     sterile adults, having visible germline (21%) <br /> 39, P &#194;&#188; 7 3 10 31) (Figure 4, E, F, N). We observed       exhibiting cellular material body cavity <br /> that posterior enrichment PIE-1 partially re-          resembled oocytes germ cells (60%) <br /> stored mpk-1; par-1 double mutants (2.5 6 0.9) (n .        correspond somatic gonad (Figure S6). <br /> 38, P &#194;&#188; 4 3 10 5) (Figure 4, F, H, N). Therefore,         This data consistent agametic phenotype that <br /> reduction mpk-1 partially restore asymmetric       previously described alleles par-1 (Guo <br /> localization PIE-1 par-1 mutant embryos,       Kemphues 1995). As cell fate specification was <br /> consistent restored localization upstream     partially restored mpk-1(ga111); par-1(zu310) embryos <br /> localizing factor MEX-5.                                      compared par-1(zu310), asked whether <br />    We assessed localization P granules,     fertility likewise restored. In contrast par-1(zu310) <br /> consist ribonucleoprotein particles that, PIE-1,        animals, 31% mpk-1(ga111); <br /> localize posterior embryo segregate      par-1(zu310) double mutant animals escaped em- <br /> germline blastomeres (Seydoux Strome 1999;                bryonic lethality sterile (n &#194;&#188; 132), the <br /> Schisa et al. 2001). P granules proposed           animals genotype contained oocytes and <br /> destabilized anterior embryo MEX-5/6         embryos germline (Figure S6). These data sug- <br /> activity (Schubert et al. 2000) PAR-1 suggested       gest restoring asymmetric localization PIE-1 <br /> to mediate posterior stabilization restricting       early embryos enrichment P granules in <br /> MEX-5/6 anterior (Cheeks et al. 2004). Because P       germ cell precursors late mpk-1; par-1 double mutant <br /> granule distribution highly dynamic       embryos physiologically relevant, correlates with <br /> mitosis, scored localization prophase,     increased fertility progeny double mu- <br /> the time centration nucleo-centrosomal com-         tants compared progeny par-1 animals. <br /> plex. We par-1(zu310) one-cell embryos            Disrupting specifically MEX-6 function suppresses <br /> largely devoid P granules semi-restrictive temper-      par-1(zu310) lethality: Our finding mpk-1 restores <br /> ature, consistent patterns previously re-       mex-5/6-dependent phenotypes par-1(zu310) <br /> ported par-1 alleles: detected P granules     embryos prompted test disrupting mex-5 <br /> 24% (n &#194;&#188; 32) par-1(zu310) mutant embryos          mex-6 RNAi suppress par-1(zu310) mutants. <br /> observed 90% (n &#194;&#188; 38) wild-type controls       Because mex-5 mex-6 partially redundant and <br /> (Figure 4, I J). Interestingly, 69%         share sequence identity, specifically disrupted <br /> (n &#194;&#188; 34) mpk-1; par-1 embryos contained P granules,        function using RNAi targeting unique regions <br /> but particles restricted           genes, previously described (Schubert et al. <br /> posterior (20/24) double mutant one-           2000). Interestingly, disrupting mex-5 <br /> cell embryos (Figure 4L). This indicates mpk-1     did suppress par-1(zu310) phenotypes, disruption <br /> can restore P granule stability par-1 mutant one-cell      mex-6 suppressed embryonic lethality par- <br /> embryos, P granules exhibit apolar distribution       1(zu310) mutant embryos semi-restrictive temper- <br /> mpk-1(ga111); par-1(zu310) mutant background, phe-          ature 22&#194;&#176; (Table 1). It unclear effect is <br /> notype observed mpk-1(ga111)        specific mex-6. Since genes partially <br /> par-1(zu310) single mutant embryos (Figure 4, J, K,           redundant functions, possibility disrupting <br /> and L). This phenotype reminiscent one-cell      mex-6 partially compromise, abrogate MEX-5/ <br /> embryos lacking function par-1; mex-5; mex-6           6 activity, remaining activity sufficient to <br /> (Cheeks et al. 2004), ectopically             ensure normal embryogenesis downstream PAR-1. <br /> stabilized, symmetrically localized P granules. In-       Alternatively, exclude uncover a <br /> terestingly, P granules restricted      new, specific function MEX-6 indepen- <br /> one cells later stage mpk-1; par-1 double mutant    dent MEX-5. The finding RNAi disruption both <br /> embryos, indicating mpk-1 act asymmetrically      mpk-1 mex-6 results similar effects par-1(zu310) <br /> segregate P granules later step embryonic         animals consistent genes functioning the <br /> development (data shown). Taken together,             direction regulate par-1-dependent processes. <br /> data reduction function mpk-1 restores           mpk-1 regulates size cortical PAR protein <br /> MEX-5 localization mex-5/6-dependent       domains zygote: Previous studies reported <br /> phenotypes, indicating mpk-1 act         par-1 required asymmetric <br />  974                                                      A. C. Spilker et al. <br />  <br />                                                                                             Figure 5.&#226;&#8364;&#8221;PAR-1 localization one- <br />                                                                                          four-cell embryos (A) Quantification <br />                                                                                          PAR domain sizes one-cell embryos <br />                                                                                          stained PAR-3 PAR-1 ana- <br />                                                                                          phase telophase. PAR-3 domain size <br />                                                                                          (green bars) PAR-1 domain size (red <br />                                                                                          bars) determined independent ex- <br />                                                                                          periments plotted bar <br />                                                                                          diagram comparison. Zero percent cor- <br />                                                                                          responds anterior embryo, <br />                                                                                          100% corresponds posterior <br />                                                                                          pole. Values shown correspond domain <br />                                                                                          size averages 6 standard deviation. Values <br />                                                                                          significantly different wild <br />                                                                                          type marked circle values sig- <br />                                                                                          nificantly different par-1 aster- <br />                                                                                          isk. Both anterior posterior <br />                                                                                          domains extended par-1(zu310) ani- <br />                                                                                          mals. (B) Confocal images embryos dou- <br />                                                                                          ble stained PAR-3 PAR-1 to <br />                                                                                          visualize domain extension em- <br />                                                                                          bryos. As single-stained embryos <br />                                                                                          quantified A, domains PAR-3 <br />                                                                                          PAR-1 extend par-1 mutant <br />                                                                                          embryos compared wild type (n &#194;&#188; 6). <br />                                                                                          Quantifications using mid- <br />                                                                                          plane confocal sections (top panels). (C&#226;&#8364;&#8220; <br />                                                                                          G) Dorsoventral PAR-1 asymmetry not <br />                                                                                          restored mpk-1(ga111); par-1(zu310) mu- <br /> tant embryos. (C&#226;&#8364;&#8220;F) Confocal images anti-PAR-1 staining (red) late four-cell stage embryos. Anti-tubulin staining shown in <br /> green, DAPI staining shown blue. (C, E, G) In wild-type embryos mpk-1(ga111) mutants, PAR-1 localizes asymmet- <br /> rically cortex P2 (cell marked asterisk), protein localizing P2/EMS P2/ABp. <br /> (D, F, G) In par-1(zu310) mpk-1(ga111); par-1(zu310) animals, PAR-1 localizes symmetrically dorsoventral axis. <br /> Embryo length  50 mm. (G) Top panel: Schematic representation four-cell stage embryo. The cortices measured <br /> for quantifications shown graph depicted red. (G) Bottom panel: Quantification ratio fluores- <br /> cence intensities cortex P2/EMS cortex P2/ABp. Values shown correspond average intensity <br /> 6 standard deviation. Values significantly different (P , 0.05) wild type marked circle. <br />  <br />  <br /> localization anterior posterior PAR proteins             PAR-3 extends posterior par- <br /> one-cell stage embryo, disrupting function leads            1(zu310) embryos: size PAR-3 domain was <br /> expansion PAR-2 cortical domain                     58 6 5% (n &#194;&#188; 22) total embryo length par-1(zu310) <br /> anterior embryonic polarization (Cuenca et al.                 compared 52 6 7% (n &#194;&#188; 39) wild-type animals (P &#194;&#188; <br /> 2003). This phenotype depends activity MEX-                 0.004) (Figure 5, A B). However, contrary what <br /> 5/6, suggesting feedback loop functions                 observed PAR-1, localization PAR-3 the <br /> the polarization embryo (Cuenca et al. 2003). We               anterior cortex restored mpk-1(ga111); par- <br /> therefore determined localization PAR proteins                 1(zu310) double mutants compared par- <br /> during asymmetric divisions C. elegans               1(zu310) embryos (57 6 5%; n &#194;&#188; 19; P &#194;&#188; 0.9) (Figure <br /> embryo quantifying length cortical PAR pro-                 5A). In mpk-1(ga111) mutant embryos, domain size <br /> tein domains middle focal plane embryos fixed               PAR-1 (35 6 8%, n &#194;&#188; 13) PAR-3 (52 6 7%, n &#194;&#188; 19) <br /> at anaphase telophase stages (Figure 5, A B).              did significantly differ wild type (P . 0.5) <br /> In agreement previous observations local-                 (Figure 5A). Our results indicate the <br /> ization PAR-2 (Cuenca et al. 2003),                  localization PAR-1 PAR-3 perturbed par- <br /> PAR-1 extends anterior par-1(zu310)                     1(zu310) mutant embryos anterior exten- <br /> mutants: size PAR-1 domain average                  sion PAR-1, posterior extension PAR-3, <br /> 44 6 7% (n &#194;&#188; 22) total embryo length par-1(zu310)               depends mpk-1 activity, suggesting mpk-1 func- <br /> mutant embryos compared 32 6 4% wild-type                       tions upstream par-1 regulate localization of <br /> embryos (n &#194;&#188; 22) (P &#194;&#188; 3 3 10 8) (Figure 5, A B).                  posterior PAR proteins. <br /> Disrupting mpk-1 function partially suppress                  Following division, PAR proteins required <br /> phenotype size PAR-1 domain mpk-                     asymmetric division germline blastomeres and <br /> 1(ga111); par-1(zu310) embryos 39 6 5% (n &#194;&#188; 23),                  adopt polarized distributions cells (Guo and <br /> which significantly smaller par-1(zu310)                   Kemphues 1995). However, PAR polarity dif- <br /> embryos (P &#194;&#188; 0.01) (Figure 5A). Interestingly,                ferentially established one-cell embryo compared <br />                                         MPK-1 Antagonizes PAR-1 Function Polarity                                    975 <br />  <br />                                                                Taken together, results provide novel link between <br />                                                                PAR-1 MAP kinase signaling regulation of <br />                                                                asymmetric division C. elegans embryo. <br />                                                                   To understand loss mpk-1 function suppresses <br />                                                                par-1, analyzed par-1 phenotypes embryos <br />                                                                mutant par-1 mpk-1. Importantly, precise <br />                                                                regulation MAP kinase signaling required for <br />    Figure 6.&#226;&#8364;&#8221;Models depicting possible genetic interactions    proper oocyte maturation embryonic cell fate <br /> between mpk-1 polarity regulators. mpk-1 act up-     specification (Page et al. 2001; Kao et al. 2004). Loss <br /> stream (A), downstream (B), parallel (C) par-1     mpk-1 function embryo leads embryonic <br /> to antagonize function.                                    lethality defects cell cycle timing (Kamath et al. <br />                                                                2003; Simmer et al. 2003; Sonnichsen et al. 2005) <br /> to later cells P lineage, genes re-        indicating mpk-1 function required oogenesis <br /> ported required PAR polarity two-cell         embryogenesis. How mpk-1 fulfills functions <br /> stage, one-cell stage embryos (Basham           remains elucidated. Our analysis shows <br /> Rose 1999). We tested mpk-1 sup-             embryos mutant mpk-1 par-1, several <br /> presses par-1 phenotypes later embryonic stages          polarity phenotypes par-1 partially restored, <br /> determining restore proper PAR-1                indicating mpk-1 antagonizes par-1-dependent sig- <br /> localization P2 blastomere four-cell stage par-      naling embryonic polarity links mpk-1 <br /> 1(zu310) mutants. This assessed quantifying         function polarity. PAR-1 levels comparable to <br /> fluorescence intensity PAR-1 cortices        wild type mpk-1(ga111); par-1(zu310) embryos, in- <br /> P2 shares neighboring blastomeres (Figure 5G,         dicating MPK-1 does suppress par-1(zu310) by <br /> top). We levels PAR-1 wild-type four-     regulating protein levels (Figure S2). Notably, mpk-1 <br /> cell stage embryos 3.1 6 0.9 (n &#194;&#188; 42) higher        mutant embryos did exhibit apparent defects <br /> cortex P2 EMS cortex           cell fate specification exception cell cycle <br /> P2 ABp (Figure 5, C G). This difference lost       asynchrony. This suggests mpk-1 only <br /> in par-1(zu310) animals, displayed ratio 1.1 6      negative regulator par-1 activity polarity. <br /> 0.4 (n &#194;&#188; 42; P &#194;&#188; 3 3 10 10) (Figure 5, D G).                  Our data indicate mpk-1 impinges PAR-1 <br /> Interestingly, asymmetric enrichment PAR-1           localization one-cell stage par-1(zu310) mutant em- <br /> the P2/EMS boundary restored mpk-1(ga111);          bryos loss mpk-1 function bypass the <br /> par-1(zu310) double mutant embryos, exhibited          requirement PAR-1 activity, asymmetric distribu- <br /> fluorescence ratio 1.1 6 0.5 (n &#194;&#188; 38; P &#194;&#188; 0.9) mpk-     tion MEX-5 restored mpk-1(ga111); par- <br /> 1(ga111) mutants embryos did differ significantly          1(zu310) embryos fully restrictive temperature (data <br /> from wild-type embryos assays (2.8 6 1.8, n &#194;&#188; 35,     shown). Interestingly, loss function mpk-1 at <br /> P &#194;&#188; 0.5) (Figure 5, E&#226;&#8364;&#8220;G). We conclude           semi-restrictive temperature par-1(zu310) animals not <br /> while disruption mpk-1 restore PAR-1 protein            partially restored asymmetric distribution of <br /> localization one-cell stage par-1(zu310) embryos,        MEX-5, suppressed phenotypes that <br /> does modulate PAR-1 protein distribution later          reported depend genes mex-5 and <br /> stage embryos.                                                 mex-6, distribution PIE-1 precise <br />                                                                <span id='am-5' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span> cortical domains occupied the <br />                                                                posterior PAR proteins one-cell stage. Moreover, <br />                       DISCUSSION <br />                                                                mpk-1(ga111); par-1(zu310) mutant embryos exhibited <br />    We performed genomewide screen genetic in-            ectopic stabilization apolar distribution P gran- <br /> teractors par-1 mutant allele better define        ules, phenotype reminiscent mex-6(pk440); mex-5(zu199); <br /> function PAR-1 embryonic polarity. By ap-           par-1(RNAi) embryos (Cheeks et al. 2004). Taken to- <br /> proach, identified genes partially             gether, results support model mpk-1 <br /> specifically suppress conditional mutant allele       act upstream par-1 impinge distribu- <br /> gene par-1, suggesting act       tion and/or activity MEX-5/6 (Figure 6A). As loss of <br /> par-1 fulfilling functions. According mole-    mpk-1 function does suppress phenotypes <br /> cular nature, suppressors cluster            associated reduction PAR-1 function, this <br /> prominent pathway, suggesting act          model implies cellular processes more <br /> different mechanisms suppress par-1 mutant. One         sensitive reduction PAR-1 activity. However, our <br /> of genetic interactors par-1 identified mpk-    results compatible alternative models in <br /> 1, gene encoding MAP kinase. We par-1           mpk-1 act downstream parallel par-1 <br /> genetically linked MAP kinase signaling pathway         (Figure 6, B C), consistent with <br /> and interaction mpk-1 par-1               previous reports indicating MPK-1/Erk activation <br /> impinges aspects asymmetric cell division.       depends PAR-1/C-TAK1 activity (Muller et al. 2001; <br />  976                                                             A. C. Spilker et al. <br />  <br /> Yoder et al. 2004). As results mpk-1                           grateful Anne Pacquelet sharing initial mex-5/6 suppression data <br /> regulates PAR-1 localization, models entail                      insightful advice, Alexander Schultze technical help, as <br />                                                                              Marc Therrien Christian Rocheleau comments the <br /> that regulation occurs indirectly negative                        manuscript. We thank laboratories M. Gotta, J.-C. Labbe&#194;&#180;, A. <br /> feedback loop MEX-5/6,                               Hajnal, M. Hengartner, M. Peter stimulating discussions. Some <br /> reported control PAR protein localization (Schubert                       strains used study provided Caenorhabditis <br /> et al. 2000; Cuenca et al. 2003). Biochemical analysis                  Genetics Center, funded National Institutes Health. <br /> be required resolve details epistatic                         This work supported Boehringer Ingelheim Foundation <br />                                                                              studentship A.S., Cole Foundation studentship A.R., the <br /> relationship.                                                                Canada Research Chair Cell Division Differentiation and <br />    Interestingly, embryos mutant E2F DP                          Canadian Institutes Health Research grant no. 158715 J.-C.L., <br /> transcription factor-related genes efl-1 dpl-1,                    funds Swiss Federal Institute Technology (ETH) and <br /> lead sustained activation MPK-1 C. elegans                      Swiss National Science Foundation M.G. The Institute of <br /> germline, exhibit Mex (Muscle excess) phenotype re-                        Research Immunology Cancer supported the <br />                                                                              Canadian Center Excellence Commercialization Research, <br /> miniscent mutations mex-5 mex-6 (Page et al.                        Canada Foundation Innovation, Fonds Recherche <br /> 2001). The morphological defects efl-1/dpl-1                     en Sante&#194;&#180; du Que&#194;&#180;bec. <br /> mex-5 mutants partially suppressed reduction <br /> of function components MAP kinase pathway <br /> (Page et al. 2001). In contrast results, indi- <br />                                                                                                    LITERATURE CITED <br /> cates increase MAP kinase activity efl-1 <br /> phenocopies loss function mex-5/6, suggesting                        Basham, S. E., L. S. Rose, 1999 Mutations ooc-5 ooc-3 dis- <br />                                                                                  rupt oocyte formation reestablishment asymmetric <br /> that mpk-1 mex-5/6 act antagonistically em-                           PAR protein localization two-cell Caenorhabditis elegans em- <br /> bryonic development. One possible explanation                           bryos. Dev. Biol. 215: 253&#226;&#8364;&#8220;263. <br /> apparent discrepancy data                         Benton, R., D. St Johnston, 2003 Drosophila PAR-1 14&#226;&#8364;&#8220; <br />                                                                                  3-3 inhibit Bazooka/PAR-3 establish complementary cortical <br /> be gain loss function mpk-1                         domains polarized cells. Cell 115: 691&#226;&#8364;&#8220;704. <br /> same phenotypic outcome. Alternatively, recent study                       Boyd, L., S. Guo, D. Levitan, D. T. Stinchcomb K. J. Kemphues, <br /> using gene expression profiling proposed muta-                              1996 PAR-2 asymmetrically distributed promotes associ- <br />                                                                                  ation P granules PAR-1 cortex C. elegans em- <br /> tions mpk-1 efl-1/dpl-1 actually control dif-                         bryos. Development 122: 3075&#226;&#8364;&#8220;3084. <br /> ferent target genes C. elegans germline (Leacock                      Brenner, S., 1974 The genetics Caenorhabditis elegans. Genetics <br /> and Reinke 2006), suggesting mpk-1 efl-1 act                        77: 71&#226;&#8364;&#8220;94. <br />                                                                              Ceron, J., J. F. Rual, A. Chandra, D. Dupuy, M. Vidal et al., <br /> different mechanisms. Future identification addi-                             2007 Large-scale RNAi screens identify novel genes interact <br /> tional genes regulate mpk-1- and/or mex-5/6-depen-                          C. elegans retinoblastoma pathway splicing- <br /> dent phenotypes help address this. Taken                                 related components synMuv B activity. BMC Dev. Biol. 7: 30. <br />                                                                              Cheeks, R. J., J. C. Canman, W. N. Gabriel, N. Meyer, S. Strome <br /> together, indicates interplay                              et al., 2004 C. elegans PAR proteins function mobilizing <br /> MPK-1 activity MEX-5/6 function complex                            stabilizing asymmetrically localized protein complexes. Curr. <br /> and warrants analysis.                                                   Biol. 14: 851&#226;&#8364;&#8220;862. <br />                                                                              Cuenca, A. A., A. Schetter, D. Aceto, K. Kemphues G. Seydoux, <br />    par-1 previously shown function genetically                            2003 Polarization C. elegans zygote proceeds distinct <br /> antagonistic MAP kinase signaling de-                              establishment maintenance phases. Development 130: <br /> velopment C. elegans vulva (Kao et al. 2004; Yoder                        1255&#226;&#8364;&#8220;1265. <br />                                                                              DeRenzo, C., K. J. Reese G. Seydoux, 2003 Exclusion germ <br /> et al. 2004) homolog kinases species                           plasm proteins somatic lineages cullin-dependent degra- <br /> been reported target MAP kinase pathway compo-                                dation. Nature 424: 685&#226;&#8364;&#8220;689. <br /> nents (Muller et al. 2001; Benton St Johnston                            Dhillon, A. S., S. Hagan, O. Rath W. Kolch, 2007 MAP ki- <br />                                                                                  nase signalling pathways cancer. Oncogene 26: 3279&#226;&#8364;&#8220;3290. <br /> 2003). This antagonistic relationship consistent                     Etemad-Moghadam, B., S. Guo K. J. Kemphues, <br /> our results that reduction function mpk-1                        1995 Asymmetrically distributed PAR-3 protein contributes to <br /> can compensate loss function par-1                          cell polarity spindle alignment early C. elegans embryos. <br />                                                                                  Cell 83: 743&#226;&#8364;&#8220;752. <br /> embryonic development indicate                        Fraser, A. G., R. S. Kamath, P. Zipperlen, M. Martinez-Campos, M. <br /> that antagonistic behavior par-1 MAP                              Sohrmann et al., 2000 Functional genomic analysis C. ele- <br /> kinase pathway reflects conserved mechanism. The                               gans chromosome I systematic RNA interference. Nature <br />                                                                                  408: 325&#226;&#8364;&#8220;330. <br /> core components transmitting MAP kinase signaling                        Goldstein, B., S. N. Hird, 1996 Specification anteroposte- <br /> conserved different cell types develop-                              rior axis Caenorhabditis elegans. Development 122: 1467&#226;&#8364;&#8220;1474. <br /> mental processes C. elegans. While work high-                        Guo, S., K. J. Kemphues, 1995 par-1, gene required estab- <br />                                                                                  lishing polarity C. elegans embryos, encodes putative Ser/ <br /> lights contribution MAP kinase signaling                             Thr kinase asymmetrically distributed. Cell 81: 611&#226;&#8364;&#8220;620. <br /> asymmetric cell division, probe                    Hajnal, A., T. Berset, 2002 The C. elegans MAPK phosphatase <br /> the mechanistic relationships MAP kinase                                 LIP-1 required G(2)/M meiotic arrest developing oo- <br />                                                                                  cytes. EMBO J. 21: 4317&#226;&#8364;&#8220;4326. <br /> signaling pathways controlling cell polarity.                      Hamill, D. R., A. F. Severson, J. C. Carter B. Bowerman, <br />                                                                                  2002 Centrosome maturation mitotic spindle assembly in <br />    We thank Ken Kemphues sharing par-1 mutant strains; Jim Priess            C. elegans require SPD-5, protein multiple coiled-coil do- <br /> and Julie Ahringer providing anti-MEX-5, anti-PIE-1, anti-PAR-           mains. Dev. Cell 3: 673&#226;&#8364;&#8220;684. <br /> 1 antibodies; Yuji Kohara sharing cDNA encoding ksr-2;        Hao, Y., L. Boyd G. Seydoux, 2006 Stabilization cell polarity <br /> Matthias Peter generously providing access robot <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span>. We         C. elegans RING protein PAR-2. Dev. Cell 10: 199&#226;&#8364;&#8220;208. <br />                                                MPK-1 Antagonizes PAR-1 Function Polarity                                                977 <br />  <br /> Hung, T. J., K. J. Kemphues, 1999 PAR-6 conserved               Rual, J. F., N. Klitgord G. Achaz, 2007 Novel insights RNAi <br />     PDZ domain-containing protein colocalizes PAR-3                off-target effects using C. elegans paralogs. BMC Genomics 8: 106. <br />     Caenorhabditis elegans embryos. Development 126: 127&#226;&#8364;&#8220;135.         Schisa, J. A., J. N. Pitt J. R. Priess, 2001 Analysis RNA asso- <br /> Izumi, Y., T. Hirose, Y. Tamai, S. Hirai, Y. Nagashima et al.,               ciated P granules germ cells C. elegans adults. Devel- <br />     1998 An atypical PKC directly associates colocalizes              opment 128: 1287&#226;&#8364;&#8220;1298. <br />     epithelial tight junction ASIP, mammalian homologue       Schlessinger, J., 2000 Cell signaling receptor tyrosine kinases. <br />     Caenorhabditis elegans polarity protein PAR-3. J. Cell Biol.          Cell 103: 211&#226;&#8364;&#8220;225. <br />     143: 95&#226;&#8364;&#8220;106.                                                         Schubert, C. M., R. Lin, C. J. Vries, R. H. Plasterk J. R. <br /> Kamath, R. S., A. G. Fraser, Y. Dong, G. Poulin, R. Durbin et al.,           Priess, 2000 MEX-5 MEX-6 function establish soma/ <br />     2003 Systematic functional analysis Caenorhabditis ele-           germline asymmetry early C. elegans embryos. Mol. Cell 5: <br />     gans genome using RNAi. Nature 421: 231&#226;&#8364;&#8220;237.                             671&#226;&#8364;&#8220;682. <br /> Kao, G., S. Tuck, D. Baillie M. V. Sundaram, 2004 C. elegans         Severson, A. F., D. R. Hamill, J. C. Carter, J. Schumacher B. <br />     SUR-6/PR55 cooperates LET-92/protein phosphatase 2A                 Bowerman, 2000 The aurora-related kinase AIR-2 recruits <br />     promotes Raf activity independently inhibitory Akt phos-          ZEN-4/CeMKLP1 mitotic spindle metaphase re- <br />     phorylation sites. Development 131: 755&#226;&#8364;&#8220;765.                             quired cytokinesis. Curr. Biol. 10: 1162&#226;&#8364;&#8220;1171. <br /> Kemphues, K. J., J. R. Priess, D. G. Morton N. S. Cheng,             Seydoux, G., C. C. Mello, J. Pettitt, W. B. Wood, J. R. Priess et al., <br />     1988 Identification genes required cytoplasmic localiza-          1996 Repression gene expression embryonic germ lin- <br />     tion early C. elegans embryos. Cell 52: 311&#226;&#8364;&#8220;320.                      eage C. elegans. Nature 382: 713&#226;&#8364;&#8220;716. <br /> Kirby, C., M. Kusch K. Kemphues, 1990 Mutations par           Seydoux, G., S. Strome, 1999 Launching germline Cae- <br />     genes Caenorhabditis elegans affect cytoplasmic reorganiza-           norhabditis elegans: regulation gene expression early germ <br />     tion cell cycle. Dev. Biol. 142: 203&#226;&#8364;&#8220;215.               cells. Development 126: 3275&#226;&#8364;&#8220;3283. <br /> Labbe&#194;&#180;, J. C., A. Pacquelet, T. Marty M. Gotta, 2006 A ge-           Simmer, F., C. Moorman, A. M. van der Linden, E. Kuijk, P. V. van <br />     nomewide screen suppressors par-2 uncovers potential reg-         den Berghe et al., 2003 Genome-wide RNAi C. elegans using <br />     ulators PAR protein-dependent cell polarity Caenorhabditis         hypersensitive rrf-3 strain reveals novel gene functions. PLoS <br />     elegans. Genetics 174: 285&#226;&#8364;&#8220;295.                                          Biol. 1: E12. <br /> Lackner, M. R., S. K. Kim, 1998 Genetic analysis Caeno-       Sonnichsen, B., L. B. Koski, A. Walsh, P. Marschall, B. Neumann <br />     rhabditis elegans MAP kinase gene mpk-1. Genetics 150: 103&#226;&#8364;&#8220;117.          et al., 2005 Full-genome RNAi profiling early embryogenesis <br /> Lackner, M. R., K. Kornfeld, L. M. Miller, H. R. Horvitz S. K.           Caenorhabditis elegans. Nature 434: 462&#226;&#8364;&#8220;469. <br />     Kim, 1994 A MAP kinase homolog, mpk-1, involved ras-           Sulston, J. E., E. Schierenberg, J. G. White J. N. Thomson, <br />     mediated induction vulval cell fates Caenorhabditis elegans.       1983 The embryonic cell lineage nematode Caenorhab- <br />     Genes Dev. 8: 160&#226;&#8364;&#8220;173.                                                   ditis elegans. Dev. Biol. 100: 64&#226;&#8364;&#8220;119. <br /> Leacock, S. W., V. Reinke, 2006 Expression profiling MAP          Sundaram, M. V., 2006 RTK/Ras/MAPK signaling. WormBook, Ed. <br />     kinase-mediated meiotic progression Caenorhabditis elegans.           The C. elegans Research Community, WormBook, doi/10.1895/ <br />     PLoS Genet. 2: e174.                                                     wormbook.1.80.1, http://www.wormbook.org. <br /> Mello, C. C., C. Schubert, B. Draper, W. Zhang, R. Lobel et al.,         Tabuse, Y., Y. Izumi, F. Piano, K. J. Kemphues, J. Miwa et al., <br />     1996 The PIE-1 protein germline specification C.                  1998 Atypical protein kinase C cooperates PAR-3 estab- <br />     elegans embryos. Nature 382: 710&#226;&#8364;&#8220;712.                                    lish embryonic polarity Caenorhabditis elegans. Development <br /> Muller, J., S. Ory, T. Copeland, H. Piwnica-Worms D. K.                  125: 3607&#226;&#8364;&#8220;3614. <br />     Morrison, 2001 C-TAK1 regulates Ras signaling phosphor-           Tenenhaus, C., C. Schubert G. Seydoux, 1998 Genetic re- <br />     ylating MAPK scaffold, KSR1. Mol. Cell 8: 983&#226;&#8364;&#8220;993.                   quirements PIE-1 localization inhibition gene expres- <br /> Munro, E., J. Nance J. R. Priess, 2004 Cortical flows powered            sion embryonic germ lineage Caenorhabditis elegans. <br />     asymmetrical contraction transport PAR proteins establish          Dev. Biol. 200: 212&#226;&#8364;&#8220;224. <br />     maintain anterior-posterior polarity early C. elegans     Tenlen, J. R., J. N. Molk, N. London, B. D. Page J. R. Priess, <br />     embryo. Dev. Cell 7: 413&#226;&#8364;&#8220;424.                                            2008 MEX-5 asymmetry one-cell C. elegans embryos requires <br /> O&#226;&#8364;&#8482;Rourke, S. M., M. D. Dorfman, J. C. Carter B. Bowerman,                PAR-4- PAR-1-dependent phosphorylation. Development <br />     2007 Dynein modifiers C. elegans: light chains suppress con-          135: 3665&#226;&#8364;&#8220;3675. <br />     ditional heavy chain mutants. PLoS Genet. 3: e128.                   WormBase, 2008 Available http://www.wormbase.org; Release <br /> Page, B. D., S. Guedes, D. Waring J. R. Priess, 2001 The C.              WS200, May 16, 2008. <br />     elegans E2F- DP-related proteins required embryonic      Wu, Y., M. Han, 1994 Suppression activated Let-60 ras pro- <br />     asymmetry negatively regulate Ras/MAPK signaling. Mol.               tein defines role Caenorhabditis elegans Sur-1 MAP kinase <br />     Cell 7: 451&#226;&#8364;&#8220;460.                                                         vulval differentiation. Genes Dev. 8: 147&#226;&#8364;&#8220;159. <br /> Panneerselvam, S., A. Marx, E. M. Mandelkow E. Mandelkow,            Yoder, J. H., H. Chong, K. L. Guan M. Han, 2004 Modulation <br />     2006 Structure catalytic ubiquitin-associated domains         KSR activity Caenorhabditis elegans Zn ions, PAR-1 ki- <br />     protein kinase MARK/Par-1. Structure 14: 173&#226;&#8364;&#8220;183.                 nase PP2A phosphatase. EMBO J. 23: 111&#226;&#8364;&#8220;119. <br /> Rocheleau, C. E., K. Cullison, K. Huang, Y. Bernstein, A. C. <br />     Spilker et al., 2008 The Caenorhabditis elegans ekl (enhancer <br />     ksr-1 lethality) genes include putative components germ- <br />     line small RNA pathway. Genetics 178: 1431&#226;&#8364;&#8220;1443.                                                    Communicating editor: K. Kemphues <br />                       Supporting Information <br /> http://www.genetics.org/cgi/content/full/genetics.109.106716/DC1 <br />  <br />  <br />  MAP Kinase Signaling Antagonizes PAR-1 Function During <br />   Polarization Early Caenorhabditis elegans Embryo <br />  <br />  <br />  Annina C. Spilker, Alexia Rabilotta, Caroline Zbinden, Jean-Claude Labb&#195;&#169; <br />                              Monica Gotta <br />  <br />  <br />         Copyright &#194;&#169; 2009 Genetics Society America <br />                 DOI: 10.1534/genetics.109.106716 <br />                       Supporting Information <br /> http://www.genetics.org/cgi/content/full/genetics.109.106716/DC1 <br />  <br />  <br />  MAP Kinase Signaling Antagonizes PAR-1 Function During <br />   Polarization Early Caenorhabditis elegans Embryo <br />  <br />  <br />  Annina C. Spilker, Alexia Rabilotta, Caroline Zbinden, Jean-Claude Labb&#195;&#169; <br />                              Monica Gotta <br />  <br />  <br />         Copyright &#194;&#169; 2009 Genetics Society America <br />                 DOI: 10.1534/genetics.109.106716 <br />  2 SI                                                     A. C. Spilker et al. <br />  <br />  <br />  <br />  <br />    FIGURE S1.&#226;&#8364;&#8221;The temperature sensitive period par-1(zu310) starts one-cell stage extends 28-cell <br /> stage. Embryos dissected gravid hermaphrodites grown permissive temperature shifted restrictive temperature <br /> (or vice versa) various developmental stages. The shifted embryos left develop final temperature survival <br /> was assessed day. This analysis revealed par-1 requirement par-1(zu310) animals starts one-cell stage and <br /> ends embryonic stage 28 nuclei visible middle focal plane embryo. Upper panel: Schematic <br /> representation experiment. Lower panel: Bar graphs representing percentage hatching non-hatching embryos, <br /> shifted (or not) particular stage (n&gt;7 stage). Morphogenesis scored presence absence embryo twitching. <br />                                                         A. C. Spilker et al.                                                3 SI <br />  <br />  <br />  <br />  <br />    FIGURE S2.&#226;&#8364;&#8221;mpk-1 does regulate PAR-1 levels. Western blot analysis PAR-1 protein levels present embryonic <br /> extracts obtained animals indicated genotypes grown 22&#194;&#186;C. The graph shows relative intensity PAR-1 levels <br /> in various extracts &#194;&#177; SD 2 independent experiments reveals levels PAR-1 similar conditions tested. <br /> Tubulin used reference. <br />  4 SI                                                    A. C. Spilker et al. <br />  <br />  <br />  <br />  <br />    FIGURE S3.&#226;&#8364;&#8221;mpk-1(ga111) does suppress strong loss par-1 function. Quantification cell cycle timing AB (light grey) <br /> and P1 (dark grey) daughter blastomeres par-1(RNAi) (n=11) mpk-1(ga111); par-1(RNAi) (n=13) embryos. The synchronous cell <br /> division phenotype par-1(RNAi) 2-cell stage embryos suppressed mpk-1(ga111). <br />                                                         A. C. Spilker et al.                                                 5 SI <br />  <br />  <br />  <br />  <br />    FIGURE S4.&#226;&#8364;&#8221;Slowing cell cycle does result suppression par-1(zu310). (A) Quantification cell cycle timing <br /> in AB (light grey) P1 (dark grey) daughter blastomeres control (n=9), F33H2.5(RNAi) (n=10) div-1(RNAi) (n=10) embryos. <br /> Cell cycle timing AB P1 div-1(RNAi) embryos significantly different control (p&lt;0.05). The division of <br /> P1 F33H2.5(RNAi) embryos slightly slower control, significant. (B) Localization endogenous <br /> MEX-5 two-cell stage embryos 22&#194;&#186;C. While wild-type embryos asymmetric localization MEX-5, par-1 mutant <br /> embryos exhibit pronounced gradient MEX-5 daughter blastomeres. MEX-5 localization not <br /> restored RNAi depletion F33H2.5 div-1. Embryo length ~50 &#206;&#188;m. (C) Quantification fluorescence intensity ratio <br /> between cytoplasm AB P1 MEX-5 stained embryos. The number embryos analyzed ranges 5 7. <br />  6 SI                                                 A. C. Spilker et al. <br />  <br />  <br />  <br />  <br />    FIGURE S5.&#226;&#8364;&#8221;A strong loss mpk-1 function does affect MEX-5 levels. The images localization endogenous MEX- <br /> 5 two-cell stage embryos. Quantification revealed MEX-5 localized asymmetrically wild-type (n=4) mpk- <br /> 1(ga111) (n=7) embryos grown 25&#194;&#186;C. Embryo length ~50 &#206;&#188;m. <br />                                                          A. C. Spilker et al.                                                 7 SI <br />  <br />  <br />  <br />  <br />    FIGURE S6.&#226;&#8364;&#8221;Fertility mpk-1; par-1 double mutant F1 progeny partially restored. DIC images posterior F1 animals <br /> at 24hs post larval stage L4. Posterior right ventral images. In wild-type, mpk-1 mpk-1; par-1 animals <br /> embryos oocytes visible ventrally. In par-1 animals gut visible body cavity (lower image) gut and <br /> some cellular material present. The cellular material reminiscent somatic gonad. <br />  8 SI                                   A. C. Spilker et al. <br />  <br />  <br />  <br />  <br />                                           TABLE S1 <br />  <br />                              Candidate Suppressors par-1(zu310) <br />  <br />  <br />        M01D7.7         C16C4.1                     T21B4.1           T20G5.6 <br />        C32E8.4         R52.10                      Y54G9A.2          Y39A1C.2 <br />        T03F1.2         D2062.9                     Y54G9A.5          Y56A3A.29 <br />        C18E3.6         Y27F2A.g                    F26H11.3          Y49E10.25 <br />        T03A1.3         F14D2.3                     Y46G5.ee          F53A2.1 <br />        ZK973.f         Y49F6C.7                    Y46G5.kk          F53A2.6 <br />        F21F3.5         F12E12.i                    C29F9.2           C51G7.d <br />        C46H11.10       C17A2.8                     T24C4.1           Y119D3_457.b <br />        C32E12.4        B0286.4                     F23H11.3          Y119D3_465.y <br />        F48C1.3         T05C1.5                     R06B10.2          Y53G8A_9248.d <br />        F26B1.7         C34F11.3                    T20H9.4           Y55B1A_116.a <br />        C55B7.2         C34F11.5                 <br /> </body></html>