<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                                      23 June 2009 <br />  <br />  <br />  <br />                      Neural <br />                      Development <br />                                        www.neuraldevelopment.com <br />  <br />  <br />  <br />  <br />   Neuroendocrine transcriptional <br />  programs adapt dynamically the <br /> supply demand neuropeptides <br />            Deborah M Kurrasch et al. <br />  <br />  <br />                                        Neural Development 2009, 4:22 <br />                     http://www.neuraldevelopment.com/content/4/1/22 <br />  Neural Development                                                                                                                       BioMed Central <br />  <br />  <br />  <br /> Research article                                                                                                                       Open Access <br /> Neuroendocrine transcriptional programs adapt dynamically the <br /> supply demand neuropeptides revealed NSF mutant <br /> zebrafish <br /> Deborah M Kurrasch1, Linda M Nevin2, Jinny S Wong3, Herwig Baier2 and <br /> Holly A Ingraham*1 <br />  <br /> Address: 1Department Cellular Molecular Pharmacology, University California, San Francisco, California, USA, 2Department of <br /> Physiology, University California, San Francisco, California, USA 3Gladstone Institute Cardiovascular Disease, San Francisco, California <br /> 94158, USA <br /> Email: Deborah M Kurrasch - kurrasch@ucalgary.ca; Linda M Nevin - Linda.Nevin@ucsf.edu; Jinny S Wong - jwong@gladstone.ucsf.edu; <br /> Herwig Baier - herwig.baier@ucsf.edu; Holly A Ingraham* - holly.ingraham@ucsf.edu <br /> * Corresponding author <br />  <br />  <br />  <br />  <br /> Published: 23 June 2009                                                      Received: 2 March 2009 <br />                                                                              Accepted: 23 June 2009 <br /> Neural Development 2009, 4:22   doi:10.1186/1749-8104-4-22 <br /> This article available <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span>: http://www.neuraldevelopment.com/content/4/1/22 <br /> &#194;&#169; 2009 Kurrasch et al; licensee BioMed Central Ltd. <br /> This Open Access article distributed terms Creative Commons Attribution License (http://creativecommons.org/licenses/by/2.0), <br /> which permits unrestricted use, distribution, reproduction medium, provided original work properly cited. <br />  <br />  <br />  <br />  <br />             <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />             Background: Regulated secretion specialized neuropeptides vertebrate neuroendocrine system <br />             critical ensuring physiological homeostasis. Expression cell-specific peptide markers the <br />             differentiating hypothalamus commences prior birth, predating physiological demand for <br />             secreted neuropeptides. The conserved function spatial expression hypothalamic peptides in <br />             vertebrates prompted search critical neuroendocrine genes newly hatched zebrafish larvae. <br />             Results: We screened mutant 5 days post-fertilization zebrafish larvae fail undergo visually <br />             mediated background adaptation disruption hypothalamic pomc expression. <span id='am-4' about='protege:TO' typeof='owl:Thing'>To</span> surprise, the <br />             ATPase N-ethylmaleimide sensitive factor (nsf) identified essential gene maintenance of <br />             neuroendocrine transcriptional programs embryo-to-larva transition. Despite normal <br />             hypothalamic development nsfst53 mutants, neuropeptidergic cells exhibited dramatic loss cell- <br />             specific markers 5 days post-fertilization accompanied elevated intracellular neuropeptide <br />             protein. Consistent role NSF vesicle-membrane fusion events intracellular trafficking, <br />             cytoplasmic endoplasmic reticulum-like membranes accumulate nsf-/- hypothalamic neurons similar to <br />             observed SEC18 (nsf ortholog) yeast mutants. Our data support model unspent <br />             neuropeptide cargo feedbacks extinguish transcription neuropeptidergic cells just become <br />             functionally required. In support model gnrh3 transcripts remained unchanged pre- <br />             migratory, non-functional gonadotropin-releasing hormone (GnRH) neurons nsf-/- zebrafish. <br />             Furthermore, oxytocin-like (oxtl, intp) transcripts, osmoreceptive neurons persist in <br />             mutant zebrafish, drop precipitously mutant zebrafish acutely challenged high salt. <br />             Conclusion: Our analyses nsf mutant zebrafish reveal unexpected role NSF hypothalamic <br />             development, mutant 5 days post-fertilization larvae exhibiting stage-dependent loss of <br />             neuroendocrine transcripts corresponding accumulation neuropeptides soma. Based our <br />             collective findings, speculate neuroendocrine transcriptional programs adapt dynamically both <br />             supply demand neuropeptides ensure adequate homeostatic responses. <br />  <br />  <br />  <br />                                                                                                                                        Page 1 16 <br />                                                                                                                  (page number citation purposes) <br />  Neural Development 2009, 4:22                                           http://www.neuraldevelopment.com/content/4/1/22 <br />  <br />  <br />  <br />  <br /> Background                                                    neuropeptide mediates dispersion black pigment gran- <br /> The hypothalamus participates maintenance           ules dermal melanophores causes darkening of <br /> homeostasis synthesis release neu-         skin color fish. Conversely, placed light <br /> ropeptides. The neuroendocrine appears           background, drop &#206;&#177;-melanocyte-stimulating hor- <br /> highly conserved mammals teleosts, evi-        mone levels results aggregation melanin a <br /> denced shared programs pituitary develop-        loss &apos;dark&apos; <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> overall blanching. <br /> ment [1-4] spatial patterns functional roles   Because &#206;&#177;-melanocyte-stimulating hormone directly <br /> hypothalamic neuropeptides [5-11]. In teleosts,       regulated hypothalamic inputs, reasoned criti- <br /> the neuroendocrine functional          cal developmental neuroendocrine genes dis- <br /> 48 72 hours post-fertilization (hpf),            covered VBA-mutant zebrafish [27] altered <br /> embryo hatches chorion fully             proopiomelanocortin (pomc) expression. <br /> exposed external environment [12]. By 5 days post- <br /> fertilization (dpf) newly hatched embryo devel-      From screen, mutant line showed complete and <br /> oped free-swimming larva respond         specific loss pomc expression presumptive arcuate <br /> adapt external internal cues. Thus, embryo-to-    nucleus hypothalamus. To surprise, this <br /> larva transition (3 5 dpf) distinct developmental     mutated gene identified N-ethylmaleimide sensitive <br /> stage, somewhat analogous mammalian birth         factor (nsf, SEC18), ATPase functions in <br /> newborn integrates sensory input initiates regulated-     membrane fusion events acting structural <br /> secretion maintain homeostasis.                            chaperone (reviewed [28,29]). These functional roles of <br />                                                               NSF important regulated secretion [30-32] for <br /> While demand neuroendocrine signaling reg-        intracellular trafficking [33]. Our analysis nsf mutant <br /> ulated neuropeptide secretion increases dramatically          zebrafish led examine hypothalamic develop- <br /> when newborn exposed external environment,         mental programs respond major secretory defect. We <br /> cell-specific transcriptional programs appear initiate     report nsf-deficient hypothalamic neurons exhibit a <br /> much earlier [13-16]. Indeed, birthdating experiments      marked accumulation neuropeptide proteins a <br /> rodents hypothalamic neurons originate         corresponding stage-dependent loss mRNA, suggesting <br /> the ventricular neuroepithelium beginning            neuropeptide stores cytoplasm feedback reg- <br /> embryonic day (E)12 largely differentiated E16     ulate transcriptional programs nucleus. <br /> [17,18], judged expression translation cell <br /> type-specific neuropeptide transcripts protein, respec-   Results <br /> tively. That onset hypothalamic transcriptional pro-   Loss hypothalamic pomc expression nsf mutant <br /> grams predates function best illustrated    zebrafish <br /> ontogeny gonadotropin-releasing hormone (GnRH)             To identify genes critical neurosecretory cell function, <br /> neurons (also known luteinizing hormone-releasing          screened collection 50 VBA-deficient zebrafish <br /> hormone (LHRH) neurons). In mammals                  mutants disrupted pomc expression. These mutants <br /> zebrafish subpopulation neurons migrates         described previously forward genetic screen <br /> the olfactory placode hypothalamus eventu-       employing chemical mutagen ethyl-nitroso urea [27]. <br /> ally orchestrates germ cell maturation onset sex-   Five mutants range hypothalamic defects were <br /> ual maturity [6,19-23]. However, embryonic             identified screen, (s364) reported <br /> development prior migration, immature            here. s364 mutant zebrafish display dark appearance on <br /> GnRH neurons express signature neuropeptide             light background lack inflated swim bladder (Fig- <br /> [6,21,22]. These studies suggest neu-      ure 1A). Additionally, mutant zebrafish listless star- <br /> roendocrine develops early fetus            tle response, paralyzed, eventually die 8 <br /> remains primed secrete neuropeptides later time         dpf (data shown). The <span id='am-317' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-318' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-319' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-320' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-321' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-322' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-323' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-324' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-325' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-326' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-327' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-328' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-330' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-331' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-332' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-333' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-334' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-335' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-336' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-337' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-338' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-341' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-342' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-343' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-344' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-346' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-347' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-348' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-349' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-350' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-351' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-352' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-353' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-354' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-355' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-356' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-357' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-358' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-359' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-360' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-362' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-363' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-364' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-365' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-366' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-367' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-368' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-369' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-370' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-372' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-373' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-374' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-375' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-376' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-378' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-379' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-380' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-381' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-382' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-383' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-384' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-385' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-386' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-389' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-390' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-391' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-392' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-393' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-394' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span>mutation</span> mapped chro- <br /> points [24,25].                                               mosome 3 zebrafish genome using polymorphic <br />                                                               microsatellite markers (HB, unpublished data). This posi- <br /> To identify genes critical hypothalamic     tion matched highly conserved AAA ATPase nsf, <br /> development embryo-to-larva transition,        alleles (st25 st53) identified <br /> re-screened mutant 5 dpf zebrafish larvae fail        previously unrelated screen [34]. The st25 st53 <br /> undergo visually mediated background adaptation (VBA)         alleles harbor nonsense missense nsf mutations car- <br /> for disruption neuropeptide transcription. VBA   boxy-terminal ATPase domain (D1) oli- <br /> characterized neuroendocrine reflex occurs fish,      gomerization domain (D2), respectively [34] (Figure 1). <br /> reptiles, amphibians (reviewed [26]). When placed      The s364 previously reported nsf mutants [34] exhibit <br /> on dark, light-absorbing background, pituitary melano-      similar locomotor VBA phenotypes. <br /> tropes release &#206;&#177;-melanocyte-stimulating hormone; this <br />  <br />  <br />                                                                                                                   Page 2 16 <br />                                                                                             (page number citation purposes) <br />  Neural Development 2009, 4:22                                                         http://www.neuraldevelopment.com/content/4/1/22 <br />  <br />  <br />  <br />  <br />   A                                       C                         2.5 <br />                                                                           WT <br />                                                                                                D               4.0 <br />                                                                                                                       n=10 ea. <br />  <br />  <br />  <br />  <br />                                               Relative Expression <br />                                                                           st53                                                             *** <br />                                                                     2.0 <br />  <br />  <br />  <br />  <br />                                                                                                   <span id='am-3' about='xsp:length' typeof='owl:Thing'>Length</span>, mm <br />                                                                     1.5 <br />           s364              WT                                                                                 3.5 <br />                                                                     1.0 <br />   s364 <br />                                                                     0.5 <br />  <br />    WT                                                               0.0                                        3.0 <br />                                                                           nsf         nsf-b                                WT           st53 <br />  <br />                                                                                                                       st25 L        P <br />                                                                                          st53 Y            STOP <br />               NSF <br />                  N1            N2                                                D1                                    D2 <br />                   Interaction                                          ATPase                             Inactive ATPase <br />                  SNAPs, SNAREs                                                                                 (hexamerization) <br />  <br />  <br />  <br />  <br />  B                                   WT                                               s364                                                   st53 <br />  <br />  <br />  <br />  <br />           ARC          ap <br />                        pp <br />  <br />  <br />                               pomc <br />                                  st25                                        s364 x st53                                       s364 x st25 <br />  <br />  <br />  <br />  <br /> Figure 1 (see legend page) <br />  <br />  <br />  <br />                                                                                                                                         Page 3 16 <br />                                                                                                                  (page number citation purposes) <br />  Neural Development 2009, 4:22                                                http://www.neuraldevelopment.com/content/4/1/22 <br />  <br />  <br />  <br />  <br /> Figure <br /> The  s3641 phenotype <br />            (see previousis page) <br />                            mutation nsf <br /> The s364 phenotype mutation nsf. (A) Mutant wild-type (WT) zebrafish (5 dpf) shown white <br /> background. Mutant zebrafish display dark appearance white background dispersed melanin (red arrowhead). <br /> Wild-type zebrafish inflated swim bladder (black arrowhead) mutant zebrafish not. (B) Loss pomc expression <br /> in arcuate nucleus (ARC) hypothalamus (arrowheads) mutant alleles wild-type 5 dpf shown. pomc <br /> expression maintained anterior pituitary (ap) posterior pituitary (pp) wild-type mutant alleles. s364/ <br /> st53 s364/st25 transheterozygotes display pomc phenotype s364, st53, st25 homozygotes. Images repre- <br /> sentative independent crosses fish each. (C) Relative transcript levels shown nsf and <br /> nsfb wild-type st53 zebrafish heads 5 dpf (n = 8 each). Fold induction relative wild-type head primer <br /> set taken 1.0 (horizontal dashed line). All samples, including additional control experimental samples, are <br /> normalized wild-type sample variation shown error bars (standard error mean). (D) The linear <br /> growth 5 dpf wild-type st53 zebrafish shown (wild-type = 3.6 mm, st53 = 3.8 mm; n = 10 group). Length was <br /> measured nose tip tail. Wild-type fish temporarily paralyzed using ice. Results expressed mean &#194;&#177; stand- <br /> ard deviation, statistical analyses unpaired t-test. ***P &lt; 0.001. A schematic representation NSF protein <br /> motifs. The st53 mutation introduces <span id='am-239' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-240' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-243' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-277' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-290' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-292' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-307' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-309' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-316' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span>nonsense mutation</span> just prior D2, st25 introduces <span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span>missense mutation</span> in <br /> D2. Two distinct amino-terminal domains (N1, N2) mediate substrate recognition. <br />  <br />  <br /> Multiple complementation crosses revealed s364 rep-           ruption POMC-melanocortin signaling increases linear <br /> resents novel loss-of-function allele nsf. Similar         growth zebrafish mice [35,36]. <br /> s364, st53 s364/st53 s364/st25 tran- <br /> sheterozygotes showed disrupted pomc staining               Hypothalamic cell markers absent nsfst53 mutant <br /> arcuate nucleus hypothalamus mutant zebrafish,           larvae <br /> whereas expression maintained anterior              We determined loss pomc expression nsfst53 <br /> posterior pituitary (Figure 1B). Low, persistent pomc          zebrafish extended hypothalamic markers, <br /> expression observed arcuate nucleus mis-         including transcription factors neuropeptides are <br /> sense nsf allele st25 s364/st25 transheterozy-          highly conserved teleosts mammals (Figure <br /> gotes (Figure 1B). Sequencing exons exon-           2A). Indeed, nearly hypothalamic transcripts lost <br /> intron junctions s364 revealed obvious mutations,            5 dpf nsfst53 zebrafish (Figure 2B, C; Additional file 1 <br /> suggesting nsf allele contains non-coding muta-        Figure S5). Loss hypothalamic markers secre- <br /> tion (data shown). Because genetic lesion generat-         tory-defective zebrafish stage-dependent evidenced <br /> ing s364 allele remains unknown, used nsfst53               normal expression observed earlier developmen- <br /> mutant zebrafish subsequent analyses               tal time points (24 hpf; data shown) just prior to <br /> mutant embryos identified genotyping, prior            hatching (48 hpf; Figure 2B). A time course the <br /> to appearance &apos;dark&apos; phenotype.                         embryo-to-larva transition revealed expression of <br />                                                                    hypothalamic markers begins decrease 4 dpf, <br /> Disruption nsf partially compensated upregulation         completely absent 5 dpf nsfst53 zebrafish (Fig- <br /> of nsfb                                                            ure 2B, C; Additional file 1 Figure S1). These data suggest <br /> The zebrafish genome contains duplicates ancestral           neuroendocrine nsfst53 zebrafish under- <br /> nsf gene, annotated nsf nsfb [34]. nsf ubiquitously      goes normal development loses differentiated cell <br /> expressed central nervous devel-          markers 3 5 dpf. In contrast, expression of <br /> opment [34] early larval stages (Additional file 1          emx1 forebrain (Additional file 1 Figure S2), pomc <br /> Figure S3A). As predicted nonsense muta-           pituitary (Figure 1B), gnrh3 nasal region <br /> tion nsfst53, nsf transcript levels nearly absent        (Figure 3) maintained normal levels, suggesting <br /> nsfst53 zebrafish (Figure 1C). However, quantitative PCR           nsf mutation selectively reduced expression of <br /> revealed nsfb upregulated nearly two-fold           transcripts hypothalamus proper. <br /> heads nsf mutant zebrafish (Figure 1C). Thus, compen- <br /> sation nsfb account <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span>survival</span> nsfst53             Loss hypothalamic markers apoptosis <br /> zebrafish 8 dpf given combined nsf nsfb mor-           The comprehensive loss neuroendocrine markers 5 <br /> pholino injections result prominent early lethality         dpf nsfst53 zebrafish explained obvious <br /> (Additional file 1 Table S1). Interestingly, nsfst53 zebrafish     defects gross anatomy hypothalamus sur- <br /> were noted longer wild-type siblings (3.8 mm            rounding areas (Figure 4A, B, G-I). For example, DAPI <br /> versus 3.6 mm; Figure 1D), molecular basis           staining showed distinct well-formed hypothala- <br /> of phenotype unclear, dis-          mus nsfst53 zebrafish obvious evidence of <br />                                                                    nuclear condensation apoptosis (Figure 4A, B; Addi- <br />  <br />  <br />                                                                                                                         Page 4 16 <br />                                                                                                   (page number citation purposes) <br />  Neural Development 2009, 4:22                                                                                              http://www.neuraldevelopment.com/content/4/1/22 <br />  <br />  <br />  <br />  <br />                                A.                                            3V                Rostral <br />                                                                                                                                       nr5a <br />                                                      Eye                                                            3V <br />                                                                                                                                       gnrh <br />                                                                                                                                       pomc <br />                                                                                                                                       nkx2.1 <br />                                                                                                                                       hypocretin <br />                                                                                                Caudal        A                        isotocin/ <br />                                                                            Yolk Sac                                                   oxytocin <br />                                               zebrafish                                                     mouse <br />  <br />                                B. <br />                                                                                       48 hpf                                  5dpf <br />                                                                       WT                        nsfst53              WT               nsfst53 <br />                                  nkx2.1a <br />                                  nr5a <br />                                  hypocretin <br />                                  avpl <br />                                  crh <br />  <br />  <br />  <br />  <br />                                C.                                   1.00 <br />                                                                                                                                            WT <br />                                               Relative Expression <br />  <br />  <br />  <br />  <br />                                                                                                                                            nsfst53 <br />  <br />                                                                     0.75 <br />  <br />                                                                     0.50 <br />  <br />                                                                     0.25 <br />  <br />                                                                     0.00 <br />                                                                                       nr5a                hcrt      avpl             CRH <br />  <br /> Figure <br /> Loss hypothalamic <br />           2             markers occurs embryo-to-larva transition <br /> Loss hypothalamic markers occurs embryo-to-larva transition. (A) Cartoon depicting spatial <span id='am-395' about='xsp:pattern' typeof='owl:Thing'>pattern</span>- <br /> ing known hypothalamic markers zebrafish (48 hpf) mouse (P0) brain. A dorsal view, ventral shown <br /> for zebrafish horizontal plane shown mouse conserved hypothalamic transcription factors neuropep- <br /> tides. Other anatomical landmarks include eyes, yolk sac, ventricle (3V), mouse cartoon, amygdala (A), (B) <br /> Whole mount expression patterns shown nkx2.1a, nr5a (ff1d), hcrt, avpl, crh 48-hpf (n = 15) 5-dpf (n = <br /> 20) wild-type (WT) nsfst53 zebrafish. The expression hypothalamic markers lost 5 dpf (right panel). The 48-hpf nsfst53 <br /> zebrafish identified genotyping. (C) Relative transcript levels neuroendocrine markers shown wild-type and <br /> nsfst53 zebrafish heads 5 dpf (n = 8 each). Fold induction relative 1.0 results single wild-type head chosen for <br /> each primer set. Samples variation shown error bars (standard error mean). <br />  <br />  <br />  <br />  <br />                                                                                                                                                                            Page 5 16 <br />                                                                                                                                                      (page number citation purposes) <br />  Neural Development 2009, 4:22                                              http://www.neuraldevelopment.com/content/4/1/22 <br />  <br />  <br />  <br />  <br />                  WT                          nsfst53                nsfst53 + nsf-MO             nsfst53 + nsfb-MO <br /> gnrh3 <br />  <br />  <br />  <br />  <br />           OE                          OE <br />  Gnrh <br />  <br />  <br />  <br />  <br />         eye                         eye <br />  <br /> Figure  3 <br /> gnrh3 transcripts persist protein levels normal nsfst53 zebrafish <br /> gnrh3 transcripts persist protein levels normal nsfst53 zebrafish. Whole-mount situ hybridization for <br /> gnrh3 expression olfactory placode 5 dpf wild-type (WT) nsfst53 control morpholino-injected larva shown <br /> (top panels). Immunocytochemistry GnRH peptide levels shown (bottom panels). Images representative least <br /> four animals genotype treatment. Anatomical landmarks include olfactory epithelium (OE) eye. <br />  <br />  <br />  <br /> tional file 1 Figure S4A, B). In fact, TUNEL (terminal deox-    remain viable early larval stages, fail main- <br /> ynucleotidyltransferase-mediated         dUTP    nick    end    tain cell-specific transcriptional programs later stages. <br /> labeling) staining sections (Figure 4B) whole- <br /> mounted (Additional file 1 Figure S3B) nsfst53 zebrafish        Hypothalamic neuropeptides accumulate nsfst53 <br /> fixed 12 h intervals 48 hpf 5.5 dpf          mutants <br /> failed reveal hypothalamic cell death time points.    Having shown transcriptional programs damp- <br /> On hand, nsfst53 mutants easily observes       ened neuroendocrine cells nsfst53 zebrafish, next <br /> classic hallmarks cell death hindbrain (Addi-         determined protein levels diminished. Unex- <br /> tional file 1 Figure S4C, D) sensory neuro-      pectedly, 5 dpf nsfst53 zebrafish exhibited prominent accu- <br /> nal populations olfactory neurons (Figure 4B;       mulation hypocretin neuropeptide Y (NPY) <br /> Additional file 1 Figure S4F), otic neurons (Additional file    protein cell soma (Figure 5A&#226;&#8364;&#8220;D; Additional file 1 <br /> 1 Figure S4H), photoreceptors (Additional file 1 Fig-       Figure S5) concomitant decrease neuropeptide <br /> ure S4J). Moreover, nsfst53 hypothalamic          peripheral projections (Additional file 1 Figure S5). By <br /> neurons differentiated organized normally, judged        contrast, moderate (hypocretin) faint (NPY) stain- <br /> by staining HuC/D (Figure 4C, D) synaptic vesicle       ing observed wild-type cell bodies (Figure 5A, C; <br /> protein 2 (SV2; Figure 4E, F). The slight decrease SV2       Additional file 1 Figure S5) higher protein levels <br /> staining (Figure 4F) coupled DiI tracing studies (data     observed cell projections (Additional file 1 Figure S5; <br /> not shown) suggested nsfst53 neurons lowered          data shown). We noted levels medially <br /> vesicle pools, retain normal projections. More impor-       expressed protein, tyrosine hydroxylase, equivalent <br /> tantly, electron microscopy (EM) analysis confirmed        wild-type mutant hypothalami (Additional file 1 <br /> nsfst53 hypothalamic neurons exhibit relatively normal          Figure S2E, F). Correlating finding, EM analysis <br /> morphology evidence synapse formation (Figure           showed increased cytoplasmic endoplasmic reticulum <br /> 4G, H). Taken together, data suggest nsf mutant        (ER)-like membranes mutant hypothalamic neurons. <br /> neurosecretory cells able differentiate correctly    These morphological structures decorated ribos- <br />                                                                 omes, lack secretory organelles, the <br />  <br />  <br />                                                                                                                     Page 6 16 <br />                                                                                               (page number citation purposes) <br />  Neural Development 2009, 4:22                           http://www.neuraldevelopment.com/content/4/1/22 <br />  <br />  <br />  <br />  <br />                     TUNEL / DAPI             HuC/D / DAPI               SV2 / DAPI <br />                 A                        C                       E <br />        WT <br />  <br />  <br />  <br />  <br />                 B                        D                       F <br />       nsfst53 <br />  <br />  <br />  <br />  <br />                 G                                                H <br />                             N <br />  <br />  <br />                                              N <br />                         N <br />       nsfst53 <br />  <br />  <br />  <br />  <br />                                                     N             I <br />                                          N <br />                  N <br />  <br />  <br />  <br />                                      N <br />                 2 &#206;&#188;m <br />  <br />  <br /> Figure 4 (see legend page) <br />  <br />  <br />  <br />  <br />                                                                                                 Page 7 16 <br />                                                                           (page number citation purposes) <br />  Neural Development 2009, 4:22                                                 http://www.neuraldevelopment.com/content/4/1/22 <br />  <br />  <br />  <br />  <br /> nsfst53 hypothalamus <br /> Figure    4 (see previous <br />                        undergoes <br />                           page) normal development does appear initiate apoptosis <br /> nsfst53 hypothalamus undergoes normal development does appear initiate apoptosis. Apoptosis (A) <br /> wild-type (WT) (B) nsfst53 5 dpf zebrafish (12 &#206;&#188;m horizontal sections) shown. Cell death observed hypotha- <br /> lamus (B), observed olfactory terminals (B, white arrowhead). Photoreceptor auto-fluorescence wild-type zebrafish <br /> is lost mutant (B, white arrow). DAPI staining wild-type (A, C, E) nsfst53 zebrafish (B, D, F) 5 dpf shown. Immuno- <br /> cytochemistry (C, D) Hu family proteins (E, F) SV2 shown wild-type nsfst53 5 dpf zebrafish. All images are <br /> representative 15 animals assayed. (G-I) Electron microscopy images nsfst53 hypothalamic sections (n = 3). Cell <br /> bodies (G) vesicle-filled synapse (H) resemble wild-type neurons (data shown). Anatomical landmarks (I) pres- <br /> ence dark-cored secretory vesicles (red arrowhead) used confirm location hypothalamus. N = nucleus. <br /> Note medially located cell bodies (Figure 4C, D, G) framed lateral projections (Figure 4E, F, H) in <br /> both cryosection EM images. <br />  <br />  <br /> Golgi mature secretory granules (Figure 5E, F).                 MO knock nsf resulted slight noticeable <br /> Although exact identity ER-like membranes             reduction oxtl gnrh3 expression nsfst53 zebrafish <br /> remains determined, gross morphology rem-           (Figures 3 7), suggesting maternally deposited nsf <br /> iniscent ER accumulation observed mutant SEC18               mRNA and/or partially active truncated NSF protein <br /> yeast, suggesting initial steps ER-to-golgi traf-      accounts expression oxtl gnrh3 nsfst53 <br /> ficking blocked nsf mutants.                                allelic background. <br />  <br /> Transiently blocking secretion wild-type zebrafish              We hypothesized selective loss neuropeptide <br /> mimics nsfst53 phenotype                                       transcripts nsfst53 zebrafish reflect activation of <br /> We asked impairing general secretion wild-type          particular neurosecretory cells embryo- <br /> fish induce similar transcriptional dysregulation,         to-larval transition. This &apos;use-dependent&apos; transcriptional <br /> as observed nsf mutants. Injections botulinum           hypothesis predicts gnrh3 oxtl neurons func- <br /> toxin B, prevents membrane fusion presyn-             tionally silent 3 5 dpf. If true, activation either <br /> aptic plasma membrane [37] performed different             gnrh3 oxtl neurons developmental stage <br /> doses (0.1 1 &#206;&#188;g) wild-type embryos one-cell         diminish respective neuropeptide transcripts. <br /> stage. Zebrafish paralyzed 4 dpf, 5 dpf      Although activating gnrh3 neurons technically difficult, <br /> 7 dpf analyzed, stages noted marked           reasoned activating oxtl cells high-salt chal- <br /> reduction hypothalamic transcripts tested,           lenge feasible given neurons are <br /> oxytocin-like expression, remained unchanged                 osmoreceptive, arginine vasopressin- <br /> (Figure 6 data shown). These data bolster              like (avpl; known vsnp) neurons required to <br /> notion nsfst53 neuroendocrine phenotype arises            maintain electrolyte homeostasis [38,39]. Indeed, we <br /> from general secretory defect.                                   oxtl expression significantly diminished a <br />                                                                    hyperosmolar acute challenge nsfst53 zebrafish, not <br /> A physiological challenge silences persistent oxtl                 wild-type zebrafish (Figure 7A). This result con- <br /> transcripts nsfst53 mutants                                     firmed quantitative PCR showing nearly 50% reduc- <br /> Of hypothalamic markers tested, neuropep-             tion oxtl transcripts 24 h salt challenge (Figure <br /> tides consistently maintained 5 dpf nsfst53                7B). We observed TUNEL-positive cells the <br /> zebrafish &#226;&#8364;&#8220; zebrafish ortholog oxytocin, oxytocin-like      gills salt-challenged nsfst53 wild-type zebrafish <br /> (oxtl; known isotocin [10]) GnRH iso-          (Figure 7A). Based observations, suggest that <br /> form, gnrh3; transcripts remained unchanged (Fig-             demand oxytocin-like neuropeptide high <br /> ures 3 7). For gnrh3,                   salt challenge overloads defective secretory pathway in <br /> corresponding protein levels unaffected mutant             nsfst53 mutants, resulting downregulation oxtl tran- <br /> zebrafish (Figure 3). At developmental stage gnrh3-           scripts. <br /> expressing neurons immature not <br /> migrated olfactory placode medial                  Discussion <br /> hypothalamus fully functional [6]. Compensa-             Here, unexpectedly identified nsf essential main- <br /> tion nsfb fails account persistence oxtl      tenance cell type-specific programs zebrafish <br /> gnrh3, little reduction transcript            hypothalamus developmental transition when <br /> observed dose-dependent knockdown nsfb               demand neuroendocrine signaling rises. We show <br /> mutant embryos using sequence-specific morpholino oli-             despite normal development nsfst53 hypothalami, <br /> gonucleotides (MOs; Figures 3 7). Control MO injec-            neuropeptidergic cells exhibit dramatic silencing cell- <br /> tions effect (data shown). Interestingly,         specific markers 5 dpf, concomitant accumula- <br />  <br />  <br />                                                                                                                         Page 8 16 <br />                                                                                                   (page number citation purposes) <br />  Neural Development 2009, 4:22                                                 http://www.neuraldevelopment.com/content/4/1/22 <br />  <br />  <br />  <br />  <br />                                        Hcrt / DAPI                            Npy / DAPI <br />                                   A                                  C <br />  <br />                          WT <br />  <br />  <br />  <br />  <br />                                   B                                  D <br />                         nsfst53 <br />  <br />  <br />  <br />  <br />                                                              14 &#206;&#188;m                                8 &#206;&#188;m <br />  <br />  <br />  <br />  <br />                         E                                            F <br />                                   PM                                     PM <br />  <br />  <br />  <br />  <br />                                        Golgi <br />  <br />  <br />  <br />  <br />                                          Nucleus <br />                                                                               Nucleus <br />                                                       WT                                   nsfst53 <br /> Accumulated <br /> Figure 5       neuropeptide protein buildup endoplasmic reticulum-like membranes nsfst53 zebrafish <br /> Accumulated neuropeptide protein buildup endoplasmic reticulum-like membranes nsfst53 <br /> zebrafish. The levels (A, B) hypocretin (C, D) NPY protein wild-type (WT) nsfst53 zebrafish shown (n = 8 <br /> each). Protein levels increased soma (B, D). DAPI stained nuclei shown (A-D). Electron microscopy images <br /> from (E) wild-type (F) nsfst53 zebrafish (n = 3 each). For EM analysis, neuroanatomical landmarks (obtained [50]) and <br /> the presence dense-core vesicles (Figure 4I) used locate hypothalamus. The budding secretory vesicles from <br /> the trans-Golgi network highlighted (E, red arrowhead). A build ribosome-bound ER-like membranes nsfst53 zebrafish <br /> is noted (F, red arrowhead). PM = plasma membrane. <br />  <br />  <br />  <br />  <br />                                                                                                                       Page 9 16 <br />                                                                                                 (page number citation purposes) <br />  Neural Development 2009, 4:22                                                 http://www.neuraldevelopment.com/content/4/1/22 <br />  <br />  <br />  <br />  <br /> Injection6of botulinum toxin B wild-type zebrafish mimics nsfst53 phenotype <br /> Figure <br /> Injection botulinum toxin B wild-type zebrafish mimics nsfst53 phenotype. Whole-mount situ hybridi- <br /> zation wild-type (WT) WT + botox 5 dpf shown (n = 4 each). The expression neuroendocrine markers in <br /> botox-injected WT fish highlighted (black arrowheads). Relative transcript levels shown heads WT WT + <br /> botox 5 dpf (n = 8 each). Fold induction relative control sample taken 1.0. All samples, including <br /> additional control experimental samples, normalized wild-type sample. Sample variation shown error <br /> bars (standard error mean). Statistical analyses unpaired t-test. *P &lt; 0.05; **P &lt; 0.01. NS = significant. <br />  <br />  <br /> tion neuropeptides soma. Furthermore,              exaggerated secretion-defective nsf mutant, as <br /> EM images nsfst53 neuroendocrine cells appear            depicted Figure 8. <br /> normal cell density, axonal growth, synapse forma- <br /> tion, cells exhibit build-up cytoplasmic ER-like        Our results loss hypothalamic cell markers <br /> membranes consistent marked cytoplasmic accu-             occurs nsf alleles examined, neuroendo- <br /> mulation hypocretin NPY neuropeptides                    crine neurons initiate expression signature mark- <br /> mutant hypothalami. Taken together, propose                ers. Indeed, expression patterns assayed <br /> neuroendocrine transcription normally modulated              neuropeptides transcription factors indistin- <br /> cytoplasm-to-nucleus feedback signaling,             guishable wild-type nsfst53 zebrafish, beginning as <br />                                                                    early 24 hpf; similar phenotype noted the <br />  <br />  <br />                                                                                                                        Page 10 16 <br />                                                                                                   (page number citation purposes) <br />  Neural Development 2009, 4:22                                                                       http://www.neuraldevelopment.com/content/4/1/22 <br />  <br />  <br />  <br />  <br /> A                         Control                 nsfst53                  nsfst53 + nsf-MO                               nsfst53 + nsfb-MO <br />        Untreated (oxtl) <br />     + 10X Salt (oxtl) <br />  <br />  <br />  <br />  <br />                                                                            B <br />                                                                                                                                               * <br />  <br />  <br />  <br />  <br />                                                                          Relative oxtl Expression <br />                                                                                                     1.5       No Treatment <br />                                                                                                               + 10x Salt <br />  <br />  <br />                                                                                                     1.0 <br />     + 10X Salt (Tunel) <br />  <br />  <br />  <br />  <br />                                                                                                     0.5 <br />  <br />  <br />                                                                                                     0.0 <br />                                                                                                                  WT                       nsfst53 <br />  <br />  <br />  <br />  <br /> Figure   7 downregulated nsfst53 zebrafish following acute osmolar challenge <br /> oxytocin-like <br /> oxytocin-like downregulated nsfst53 zebrafish following acute osmolar challenge. (A) Whole-mount situ <br /> hybridization 5 dpf wild-type (WT) nsfst53 control morpholino-injected zebrafish (top panels) following 24 h 10&#195;&#8212; <br /> salt treatment (middle panel) shown. The decrease expression oxytocin-like (oxtl) nsfst53 zebrafish exposed 10&#195;&#8212; salt <br /> is noted (black arrow). TUNEL-positive cells gills nsfst53 zebrafish following 10&#195;&#8212; salt treatment shown (black <br /> arrowhead). Images representative animals genotype treatment. The salt challenge repeated <br /> twice. (B) Relative transcript levels shown oxtl wild-type nsfst53 zebrafish heads 5 dpf, 10&#195;&#8212; salt <br /> exposure (n = 8 each). Fold induction relative control sample taken 1.0. All samples, including addi- <br /> tional control experimental samples, normalized wild-type sample. Variation samples shown error <br /> bars (standard error mean). Statistical analyses unpaired t-test. *P &lt; 0.05. <br />  <br />  <br /> nsfs364 allele (data shown). This surprising consider-       vival evidenced fact co-injection nsf-MO <br /> ing general role NSF membrane fusion              nsfb-MO nsfst53 embryos lethal (Additional <br /> ubiquitous expression development,                   file 1 Table S1). The persistence gnrh3 immature, pre- <br /> given Drosophila dNSF1 mutants exhibit impaired                migratory nsfst53 neurons, coupled ability to <br /> neurogenesis [40]. It possible normal hypotha-              dampen oxtl transcripts following acute salt challenge <br /> lamic development nsfst53 result compensa-              wild-type zebrafish, implies sub-populations of <br /> tion nsfb, maternally deposited nsf transcripts               neuroendocrine cells escape consequences dis- <br /> residual activity NSF encoded nsfst53 allele.             rupted NSF function. Thus, speculate increased <br /> Clearly, functional NSF required sur-          physiological demand secretion neuropeptide <br />  <br />  <br />                                                                                                                                            Page 11 16 <br />                                                                                                                       (page number citation purposes) <br />  Neural Development 2009, 4:22                                              http://www.neuraldevelopment.com/content/4/1/22 <br />  <br />  <br />  <br />  <br />                WT NEUROSECRETORY CELL                          nsf MUTANT NEUROSECRETORY CELL <br />  <br />  <br />  <br />                                                                                                                  PM <br />           Secretory <br />            vesicle <br />  <br />  <br />  <br />  <br />              Golgi <br />  <br />  <br />  <br />  <br />           ER-like <br />         membranes <br />  <br />  <br />  <br />  <br />                ER <br />  <br />  <br />  <br />  <br />                                                                      TF, NP <br />            Nucleus                                                    OFF <br /> Working8 model nsfst53 neuroendocrine cells <br /> Figure <br /> Working model nsfst53 neuroendocrine cells. A schematic regulated neuropeptide secretion shown, adapted <br /> from [47,48]. In wild-type (WT) cells, newly translated neuropeptides endoplasmic reticulum (ER) sorted the <br /> trans-Golgi network, packaged immature secretory granules, finally released mature dense-core secretory vesi- <br /> cles plasma membrane (reviewed [49]). In secretion-defective nsfst53 hypothalamic neurons, buildup ER-like mem- <br /> branes soma accumulation neuropeptides observed. We hypothesize secretory-deficient nsf mutants, <br /> accumulation &apos;unspent neuropeptide consumables&apos; triggers cytoplasmic-to-nuclear feedback signal silences neuroendo- <br /> crine transcriptional programs. NP = neuropeptide, PM = plasma membrane, TF = transcription factor. <br />  <br />  <br />  <br />  <br />                                                                                                                     Page 12 16 <br />                                                                                                (page number citation purposes) <br />  Neural Development 2009, 4:22                                                http://www.neuraldevelopment.com/content/4/1/22 <br />  <br />  <br />  <br /> cargo embryo-to-larva transition accounts           cent of, different from, well-described secretory <br /> this late-stage loss neuroendocrine transcripts nsf          feedback signaling pathways. Regulation nuclear tar- <br /> mutants.                                                           gets ER-associated events established for <br />                                                                    unfolded protein responsive (UPR) genes [42] for <br /> The neuroendocrine phenotype exhibited nsfst53              sterol regulatory genes (reviewed [43]). Here, have <br /> zebrafish partially mimicked munc18-1 mutant mice,           identify precise molecular nature feed- <br /> which exhibit selective stage-dependent (E18)                signal; signals arise buildup <br /> decrease primarily neuropeptide transcript expression           ER-like membranes, accumulation neuropep- <br /> [41]. Munc18-1 essential regulated exocytosis. The          tides cytoplasm, inability secrete neu- <br /> authors study posited transcripts encoding            ropeptides plasma membrane remains be <br /> &apos;consumables&apos; secreted products selectively            determined. The build ER-like membrane the <br /> downregulated munc18-1 synaptically silenced neu-           cytoplasm nsf hypothalamic neurons suggests neu- <br /> rons. Although neuropeptide protein levels localiza-           ropeptidergic cells responsive loss secretion and <br /> tion assessed study, findings predict        general trafficking defect. Indeed, mimic this <br /> that neuropeptides accumulate munc18-1                    phenotype transient block secretion wild- <br /> hypothalami. We extend hypothesis sug-            type animals botox injections. Thus, possible <br /> gesting unspent neuropeptide consumables feed            ER-to-golgi transport nsfst53 neurons partially <br /> to silence neuroendocrine transcriptional programs,             intact, overloaded functional demands <br /> shown model figure (Figure 8).                              placed neurosecretory neurons the <br />                                                                    embryo-to-lava transition. <br /> Our study second identify nsf essential in <br /> a specific neuronal cell-type program using forward              This cytoplasmic-to-nuclear signaling hypothesis assumes <br /> genetic screen zebrafish. Previously, Talbot col-           phenotypes observed nsfst53 hypothalami cell- <br /> leagues [34] unexpectedly nsf essential            autonomous, documented previously [34]. In case <br /> organization myelinated axons, mutant zebrafish            blastomere transplantation studies hypothala- <br /> exhibiting decreased myelin basic protein expression           mus technically challenging transplanted <br /> altered sodium channel clustering. Here,         clone sizes brain region small, because <br /> ubiquitous protein essential maintenance            selection correct situ hybridization (ISH) probe, <br /> neuroendocrine transcriptional programs. These distinct            recognizing small number cells, difficult. <br /> phenotypes illustrate different neuronal populations           Despite inherent limitations, preliminary transplan- <br /> adjust deficits membrane fusion intracellular            tation data revealed expression neuroendocrine mark- <br /> trafficking. We suggest hypothalamic neurons mount            ers medial hypothalamus wild-type &#226;&#8224;&#8217; nsfst53 <br /> a stage-dependent adaptive response major secre-           chimeras (data shown), supporting notion that <br /> tory defect silencing transcriptional programs. For             similar neurons, NSF functions autonomously in <br /> other neuronal cell types, loss nsf results vastly differ-   specialized neuroendocrine neurons. <br /> ent cellular responses. Indeed, observed prominent <br /> cell death sensory organs, including photorecep-           Conclusion <br /> tors, olfactory sensory neurons, mechanosensory cells          The unexpected role nsf maintenance hypotha- <br /> in nsf mutants. By contrast, measurable cell death          lamic cell type markers late developmental stages <br /> observed nsf mutant hypothalami evidenced             implies hypothalamic neurons adapt survive a <br /> normal gross organization nsfst53 hypothalamus,             major trafficking defect modifying transcriptional <br /> normal HuC/D staining, marks early neuronal dif-             programs. We speculate unlike sensory neurons, <br /> ferentiation, normal staining SV2 pan-neu-            undergo apoptosis, hypothalamic neurons cope <br /> ronal vesicle marker. Targeted knockdown nsf nsfb         major cellular stress ancient and <br /> the mutant background ruled survival              evolutionarily conserved neurons inherently pro- <br /> hypothalamic cells nsfst53 compensation           grammed defend maintain homeostatic physiolog- <br /> the duplicate gene nsfb and/or maternally deposited nsf            ical responses confronted changing <br /> mRNA. Taken together, hypothalamic neurons, unlike                 environments. <br /> other neuronal populations, able survive main- <br /> tain normal cellular integrity face major           Materials methods <br /> secretory deficit.                                                 Zebrafish <br />                                                                    All research involving zebrafish approved carried <br /> The phenotypes observed nsfst53 hypothalamic neurons,           according guidelines UCSF IACUC commit- <br /> including complete loss neuroendocrine transcripts          tee. Zebrafish maintained 28&#194;&#176;C 14 h light, <br /> and accumulation ER-like membranes, reminis-            10 h dark described previously [27]. Nsfst25 nsfst53 <br />  <br />  <br />                                                                                                                       Page 13 16 <br />                                                                                                  (page number citation purposes) <br />  Neural Development 2009, 4:22                                               http://www.neuraldevelopment.com/content/4/1/22 <br />  <br />  <br />  <br /> zebrafish kind gifts W Talbot (Stanford Univer-         plasmids: pomc, Roger Cone (Vanderbilt University Medi- <br /> sity). Adult nsf+/- zebrafish fed mixture fish flakes   cal Center); hcrt, Alexander Schier (Harvard University); <br /> and brine shrimp twice day. The 5 dpf fish main-         nsf, William Talbot (Stanford); oxtl avpl, Eric Glasgow <br /> tained egg water, consisting 0.03% Instant                  (Georgetown University); npy, Peter Schoonheim (UCSF); <br /> Ocean&#226;&#8224;&#8217;&#226;&#710;&#8364; sea salts (Spectrum Brands, Atlanta, GA, USA)             emx1, Su Guo (UCSF); crh, Limor Ziv (UCSF); ff1d, Miyuki <br /> in distilled water supplemented food.            Suzawa (UCSF); nkx2.1a, Steve Wilson (University Col- <br /> Embryos obtained natural crosses identified            lege London); gnrh3, Yonathan Zohar (Cambridge Uni- <br /> parents zebrafish embryo larvae stages               versity). <br /> determined according [44]. The VBA phenotype was <br /> assayed placing 5 dpf zebrafish white background          Immunohistochemistry sectioned zebrafish <br /> for &gt;15 minutes. Fish remained dark color             Mutant wild-type larvae fixed 4% PFA/PBS <br /> failed inflated swim bladders selected            overnight 4&#194;&#176;C cryoprotected 30% sucrose/PBS <br /> mutant zebrafish. Zebrafish &lt;5 dpf genotyped pre-         overnight 4&#194;&#176;C. For horizontal sections, zebrafish were <br /> viously reported [34].                                         embedded OCT (Tissue-Tek, Sakura Finetek, Tokyo, <br />                                                                   Japan) sectioned 12 &#206;&#188;m (Leica Microsystems, Ban- <br /> Length measurements                                               nockburn, IL, USA). Sectioned zebrafish left dry <br /> Mutant wild-type zebrafish (n = 10 each) sorted          (overnight 25&#194;&#176;C), washed PBS-Tw, incubated in <br /> for dark phenotype 5 dpf placed ice          block (10% Normal sheep serum/PBS; 1 h 25&#194;&#176;C), and <br /> to paralyze kept alive. They              left incubate overnight &#206;&#177;-HuC/D (1:400; Invitro- <br /> transferred 3% agarose mold images            gen, Carlsbad, CA, USA), &#206;&#177;-SV2 (1:50; Developmental <br /> captured Olympus light microscope equipped               Studies Hybridoma Bank, University Iowa), &#206;&#177;-hypocre- <br /> with CCD camera (Center Valley, PA, USA). The                   tin (1:500; Dr Kohgo, Asahikawa Medical College, Japan), <br /> zebrafish image measured nose tip          &#206;&#177;-GnRH (1:1,500; Gunma University, Japan), &#206;&#177;-Tyrosine <br /> the tail nearest 0.1 cm using standard ruler     Hydroxylase (1:100; Chemicon, Temecula, CA, USA) and <br /> scale adjusted appropriately using scale bar              &#206;&#177;-NPY (1:1,000; Dr Jo Harrold, University Liverpool). <br /> present captured image. Statistical analyses           After washing PBS-Tw, sections incubated with <br /> were conducted unpaired t-test.                                Alexa-488 (1:200; Invitrogen) 4 h, 25&#194;&#176;C. Prior to <br />                                                                   mounting, sections stained using Hoechst 33258 <br /> Whole mount situ hybridization                                 (Invitrogen). Confocal images captured using Zeiss <br /> Mutant wild-type embryos/larvae fixed 4%              LSM 5 Pascal microscope software. Confocal stacks <br /> paraformaldehyde (PFA) phosphate-buffered saline               processed using ImageJ software. Z-projec- <br /> (PBS) overnight 4&#194;&#176;C transferred 100% MeOH               tions slices comparisons were <br /> and stored -20&#194;&#176;C use. Embryos rehydrated            images processed equivalently. <br /> into PBS-Tween (PBS-Tw; 0.1% Tween-20), bleached (72              Fluorescence images adjusted Adobe Photoshop <br /> hpf 5 dpf; 1% H2O2 + 5% formamide PBS-Tw; 1 h),             using brightness/contrast, levels, curves functions <br /> treated proteinase K (20 &#206;&#188;g/ml; 8 minutes), refixed          order best represent protein levels. <br /> (4% PFA; 20 minutes), prehybridized (65 70&#194;&#176;C; 1 h) <br /> and hybridized (65 70&#194;&#176;C; overnight) hypotha-              Real-time quantitative PCR <br /> lamic riboprobes. Embryos washed (50% forma-                 Real-time quantitative PCR performed described <br /> mide/50% 2&#195;&#8212; SSC; 2&#195;&#8212; SSC; 0.2&#195;&#8212; SSC), blocked (5% sheep             previously [45]. The head dissected tail just <br /> serum heat inactive) incubated overnight 4&#194;&#176;C          prior swim bladder 5 dpf mutant wild-type <br /> &#206;&#177;-digoxigenin (DIG) antibody followed washing            zebrafish RNA isolated head and <br /> second incubation p-nitroblue tetrazolium chloride/          tail using TrizolT. All data normalized wild-type <br /> 5-bromo-4-chloro-3-indolyl phosphate (NBT/BCIP)                &apos;head&apos; sample (taken 1) error bar represents <br /> staining buffer (100 mM NaCl, 50 mM MgCl2, 100 mM                 variation &apos;head&apos; samples (n = 4). Prior to <br /> Tris-HCl, 0.1% Tween 20). The color reaction moni-            routine use, primer sets (Primer Express v2.0, Applied <br /> tored hours stopped washing PBS-          Biosystems, Foster City, CA, USA) validated to <br /> Tw. The embryos re-fixed transferred 90%              ensure amplification single product appropriate <br /> glycerol clearing. Embryos captured using Axi-        efficiency. Data obtained PCR reaction ana- <br /> ocam camera (Carl Zeiss, Thornwood, NY, USA). TUNEL-              lyzed using comparative CT method (User Bulletin No. <br /> based cell death assays carried using proce-        2, PerkinElmer Life Sciences, Waltham, MA USA). <br /> dure minor revisions according manufacturer&apos;s             Sequences provided Additional file 1 (Table S2). <br /> protocol (ApopTag; Millipore Corporation, Billerica, MA, <br /> USA). DIG-cRNA probes synthesized (Roche, Indi- <br /> anapolis, IN, USA) following generously gifted <br />  <br />  <br />                                                                                                                     Page 14 16 <br />                                                                                                (page number citation purposes) <br />  Neural Development 2009, 4:22                                                 http://www.neuraldevelopment.com/content/4/1/22 <br />  <br />  <br />  <br /> Osmolar challenge                                              pressin-like; PBS: phosphate-buffered saline; PBS-Tw: PBS <br /> For high salt, osmolar challenge, 4-dpf larvae trans-     Tween; PFA: paraformaldehyde; pomc: proopiomelanocortin; <br /> ferred egg water composed 0.3% Instant            SV2: synaptic vesicle protein 2; TUNEL: Terminal deoxy- <br /> Ocean&#226;&#8224;&#8217;&#226;&#710;&#8364; (challenge) 0.03% Instant Ocean&#226;&#8224;&#8217;&#226;&#710;&#8364; (con-             nucleotidyltransferase-mediated dUTP nick end labeling; <br /> trol) maintained 28&#194;&#176;C. Larvae collected 24 h       VBA: visually mediated background adaptation. <br /> later fixed immediately using cold 4% PFA. <br />                                                                Competing interests <br /> Morpholino analyses                                            The authors declare competing interests. <br /> MO analyses conducted published previously [5]. <br /> Briefly, MOs designed bind sequences surround-         Authors&apos; contributions <br /> ing initiating methionine (Gene Tools, Philomath,          DMK conducted initial characterization nsf mutant <br /> OR, USA). MOs resuspended water working           zebrafish including analysis transcripts, proteins, <br /> solution 1 &#206;&#188;M injected yolk one-         TUNEL, morpholino studies, EM images, drafted <br /> four-cell stage embryos (n = 100 embryos/MO; repeated 3        manuscript. LMN injected botulinum toxin B and <br /> times). Effective doses determined separately         aided confocal microscopy. JSW carried EM <br /> each MO. Larvae (5 dpf) collected, fixed overnight        experiments. HB participated design study, <br /> and stored 100% MeOH 20&#194;&#176;C mount              manuscript preparation, provided invaluable assist- <br /> ISH performed. Sequences provided Table S2          ance project. HAI conceived partic- <br /> (Additional file 1).                                           ipated design coordination study, well <br />                                                                manuscript preparation. <br /> Electron microscopy <br /> Tissue fixed 2% glutaraldehyde, 1% paraformalde-        Additional material <br /> hyde 0.1 M sodium cacodylate buffer, pH 7.4, postfixed <br /> in 2% osmium tetroxide buffer, block <br /> stained 2% aqueous uranyl acetate, dehydrated, infil- <br />                                                                     Additional file 1 <br />                                                                     Sup1. Supplemental figures S1&#226;&#8364;&#8220;S5, supplemental tables 1 2. <br /> trated, embedded LX-112 resin (Ladd Research                 Click file <br /> Industries, Burlington, VT, USA). Toluidine blue stained            [http://www.biomedcentral.com/content/supplementary/1749- <br /> semi-thin sections locate area interest,        8104-4-22-S1.pdf] <br /> using eyes guideline. Samples ultrathin sec- <br /> tioned Reichert-Jung (Leica) Ultracut S ultramicro- <br /> tome (Leica Microsystems, Bannockburn, IL USA), and <br /> stained 0.8% lead citrate. Grids examined       Acknowledgements <br /> JEOL JEM-1230 transmission electron microscope (JEOL           We especially want thank Dr W Talbot generously sharing st25 <br /> USA, Inc., Peabody, MA, USA) photographed using            st53 nsf alleles helpful comments. The authors thank Dr J Kay <br /> the Gatan Ultrascan 1000 digital camera (Gatan Inc., War-      assistance screen, Dr N Gosse help transplant studies, <br />                                                                Drs M Suzawa, P Schoonheim, L Ziv sharing situ probes. We <br /> rendale, PA, USA). <br />                                                                thank Gunma University, Institute Molecular Cellular Regulation <br />                                                                Biosignal Research Center kindly sharing LHRH (GnRH) antibody. We <br /> Botulinum toxin analyses                                       thank Drs C Cheung F Lee careful reading manuscript. Sup- <br /> Botulinum toxin injections analyses conducted         port study includes Giannini Foundation Fellowship DMK, and <br /> as published previously [46]. Briefly, progeny in-        NIH-RO1 EY013855 HB, NIH-RO1 DK063592 HAI. <br /> crosses heterozygous nsf53 adults injected the <br /> one-cell stage approximately 5 nl bolus 0.1      References <br /> 1.0 ng/nl BtTxB (EMDBiosciences, Darmstadt, Germany).          1.      Nica G, Herzog W, Sonntag C, Hammerschmidt M: Zebrafish pit1 <br />                                                                        mutants lack pituitary cell types develop severe <br /> Larvae (5 dpf) lacked startle response placed              dwarfism. Mol Endocrinol 2004, 18:1196-1209. <br /> either fix (4% PFA, overnight, 16&#194;&#176;C) Trizol&#226;&#8222;&#162; (-        2.      Herzog W, Sonntag C, Walderich B, Odenthal J, Maischein HM, Ham- <br /> 80&#194;&#176;C) processed ISH quantitative PCR,                    merschmidt M: Genetic analysis adenohypophysis formation <br />                                                                        zebrafish. Mol Endocrinol 2004, 18:1185-1195. <br /> respectively.                                                  3.      Zhu X, Lin CR, Prefontaine GG, Tollkuhn J, Rosenfeld MG: Genetic <br />                                                                        control pituitary development hypopituitarism. Curr <br />                                                                        Opin Genet Dev 2005, 15:332-340. <br /> Abbreviations                                                  4.      Toro S, Wegner J, Muller M, Westerfield M, Varga ZM: Identifica- <br /> dpf: days post-fertilization; E: embryonic day; EM: elec-              tion differentially expressed genes zebrafish hypoth- <br /> tron microscopy; ER: endoplasmic reticulum; GnRH:                      alamic-pituitary axis. Gene Expr Patterns 2009, 9:200-208. <br />                                                                5.      Kurrasch DM, Cheung CC, Lee FY, Tran PV, Hata PV, Ingraham HA: <br /> gonadotropin-releasing hormone; hpf: hours post-fertili-               The neonatal ventromedial hypothalamus transcriptome <br /> zation; ISH: situ hybridization; MO: morpholino oligo-              reveals novel markers spatially distinct patterning. J <br />                                                                        Neurosci 2007, 27:13624-13634. <br /> nucleotide; NPY: neuropeptide Y; NSF: N-ethylmaleimide         6.      Palevitch O, Kight K, Abraham E, Wray S, Zohar Y, Gothilf Y: Ontog- <br /> sensitive factor; oxtl: oxytocin-like; avpl: arginine vaso-            eny GnRH systems zebrafish brain: situ hybridiza- <br />  <br />  <br />  <br />                                                                                                                           Page 15 16 <br />                                                                                                      (page number citation purposes) <br />  Neural Development 2009, 4:22                                                               http://www.neuraldevelopment.com/content/4/1/22 <br />  <br />  <br />  <br />       tion promoter-reporter expression analyses intact                  29.   Zhao C, Slevin JT, Whiteheart SW: Cellular functions NSF: not <br />       animals. Cell Tissue Res 2007, 327:313-322.                                     just SNAPs SNAREs. FEBS Lett 2007, 581:2140-2149. <br /> 7.    Prober DA, Rihel J, Onah AA, Sung RJ, Schier AF: Hypocretin/              30.   Beckers CJ, Block MR, Glick BS, Rothman JE, Balch WE: Vesicular <br />       orexin overexpression induces insomnia-like phenotype                        transport endoplasmic reticulum Golgi <br />       zebrafish. J Neurosci 2006, 26:13400-13410.                                  stack requires NEM-sensitive fusion protein. Nature 1989, <br /> 8.    Song Y, Golling G, Thacker TL, Cone RD: Agouti-related protein                  339:397-398. <br />       (AGRP) conserved regulated metabolic state               31.   Diaz R, Mayorga LS, Weidman PJ, Rothman JE, Stahl PD: Vesicle <br />       zebrafish, Danio rerio. Endocrine 2003, 22:257-265.                             fusion following receptor-mediated endocytosis requires a <br /> 9.    Tessmar-Raible K, Raible F, Christodoulou F, Guy K, Rembold M,                  protein active Golgi transport. Nature 1989, 339:398-400. <br />       Hausen H, Arendt D: Conserved sensory-neurosecretory cell                 32.   Urbe S, Page LJ, Tooze SA: Homotypic fusion immature <br />       types annelid fish forebrain: insights hypothala-                   secretory granules maturation cell-free assay. J <br />       mus evolution. Cell 2007, 129:1389-1400.                                        Cell Biol 1998, 143:1831-1844. <br /> 10.   Unger JL, Glasgow E: Expression isotocin-neurophysin                   33.   Whiteheart SW, Matveeva EA: Multiple binding proteins suggest <br />       mRNA developing zebrafish. Gene Expr Patterns 2003,                          diverse functions N-ethylmaleimide sensitive factor. <br />       3:105-108.                                                                      J Struct Biol 2004, 146:32-43. <br /> 11.   Blechman J, Borodovsky N, Eisenberg M, Nabel-Rosen H, Grimm J,            34.   Woods IG, Lyons DA, Voas MG, Pogoda HM, Talbot WS: nsf is <br />       Levkowitz G: Specification hypothalamic neurons dual                      essential organization myelinated axons zebrafish. <br />       regulation homeodomain protein Orthopedia. Devel-                        Curr Biol 2006, 16:636-648. <br />       opment 2007, 134:4417-4426.                                               35.   Huszar D, Lynch CA, Fairchild-Huntress V, Dunmore JH, Fang Q, <br /> 12.   Flik G, Klaren PH, Burg EH Van den, Metz JR, Huising MO: CRF                Berkemeier LR, Gu W, Kesterson RA, Boston BA, Cone RD, Smith <br />       stress fish. Gen Comp Endocrinol 2006, 146:36-44.                            FJ, Campfield LA, Burn P, Lee F: Targeted disruption the <br /> 13.   Giacobini P, Wray S: Prenatal expression cholecystokinin                     melanocortin-4 receptor results obesity mice. Cell 1997, <br />       (CCK) central nervous (CNS) mouse. Neu-                        88:131-141. <br />       rosci Lett 2008, 438:96-101.                                              36.   Song Y, Cone RD: Creation genetic model obesity a <br /> 14.   Goshu E, Jin H, Lovejoy J, Marion JF, Michaud JL, Fan CM: Sim2 con-             teleost. Faseb J 2007, 21:2042-2049. <br />       tributes neuroendocrine hormone gene expression                 37.   Schiavo G, Benfenati F, Poulain B, Rossetto O, Polverino Laureto <br />       anterior hypothalamus. Mol Endocrinol 2004, 18:1251-1262.                       P, DasGupta BR, Montecucco C: Tetanus botulinum-B neu- <br /> 15.   Marin O, Baker J, Puelles L, Rubenstein JL: Patterning basal             rotoxins block neurotransmitter release proteolytic <br />       telencephalon hypothalamus essential guidance                     cleavage synaptobrevin. Nature 1992, 359:832-835. <br />       cortical projections. Development 2002, 129:761-773.                      38.   Antunes-Rodrigues J, Castro M, Elias LL, Valenca MM, xMcCann <br /> 16.   Tran PV, Lee MB, Marin O, Xu B, Jones KR, Reichardt LF, Rubenstein              MM: Neuroendocrine control body fluid metabolism. Phys- <br />       JR, Ingraham HA: Requirement orphan nuclear receptor                     iol Rev 2004, 84:169-208. <br />       SF-1 terminal differe <br /> </body></html>