<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL OF BACTERIOLOGY, Sept. 1992, p. 5814-5819                                                                      Vol. 174, No. 18 <br /> 0021-9193/92/185814-06$02.00/0 <br /> Copyright &#194;&#169; 1992, American Society Microbiology <br />  <br />  <br />        Cloning Erwinia herbicola Gene Necessary Gluconic <br />             Acid Production Enhanced Mineral Phosphate <br />            Solubilization Escherichia coli HB101: Nucleotide <br />            Sequence Probable Involvement Biosynthesis <br />                 Coenzyme Pyrroloquinoline Quinone <br />          SHIH-TUNG LIU,&apos;* LAN-YING LEE,&apos; CHIN-YING TAI,1 CHIEN-HUI HUNG,&apos; YU-SUN CHANG,&apos; <br />                      JAMES H. WOLFRAM,2 ROBERT ROGERS,2 AND ALAN H. GOLDSTEIN3* <br />         Department Microbiology Immunology, Chang-Gung Medical College, Kwei-ShanTaoyuan, Taiwan&apos;; <br />               Biotechnology Group, Idaho National Engineering Laboratory, Idaho Falls, Idaho 834152; and <br />                            Department Biology, California State University Los Angeles, <br />                               5151 State University Drive, Los Angeles, California 900323 <br />                                            Received 28 April 1992/Accepted 16 July 1992 <br />  <br />            Escherichia coli capable synthesizing apo-glucose dehydrogenase enzyme (GDH) cofactor <br />          pyrroloquinoline quinone (PQQ), essential formation holoenzyme. Therefore, absence <br />          exogenous PQQ, E. coli does produce gluconic acid. Evidence presented expression <br />          Erwinia herbicola gene E. coli HB101(pMCG898) resulted production gluconic acid, which, <br />          turn, implied PQQ biosynthesis. Transposon mutagenesis showed essential gene locus within <br />          1.8-kb region 4.5-kb insert plasmid pMCG898. This 1.8-kb region contained apparent <br />          open reading frame. In paper, present nucleotide sequence open reading frame, 1,134-bp <br />          DNA fragment coding protein Mr 42,160. The deduced sequence protein high <br />          degree homology gene III (Mr, 43,600) PQQ synthase gene complex Acinetobacter <br />          calcoaceticus previously identified Goosen et al. (J. Bacteriol. 171:447-455, 1989). In minicell analysis, <br />          pMCG898 encoded protein Mr 41,000. These data indicate E. coli HB101(pMCG898) <br />          produced GDH-PQQ holoenzyme, which, turn, catalyzed oxidation glucose gluconic acid the <br />          periplasmic space. As result gluconic acid production, E. coli HB101(pMCG898) showed enhanced <br />          mineral phosphate-solubilizing <span id='am-3' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-4' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-5' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> acid dissolution hydroxyapatite substrate. <br />  <br />    Quinoproteins play major role regulation                  synthase genes E. coli resulted GDH activity the <br /> bioenergetic processes gram-negative bacteria, in-               absence exogenous PQQ. We used slight modifi- <br /> cluding Erwinia spp. (2). For species Erwinia,               cation identify PQQ synthase gene <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> nonphosphorylating oxidation pathway primary mech-                Erwinia herbicola EHO10 (18). Our experimental approach <br /> anism aldose sugar utilization (2). The quinoprotein                 included unique phenotypic screening based our <br /> glucose dehydrogenase (GDH) controls unique step                  elucidation metabolic basis mineral <br /> this metabolic pathway. As such, GDH plays direct role              phosphate solubilization (Mps+) phenotype gram-negative <br /> the generation transmembrane proton motive force              bacteria. <br /> the oxidation aldose sugars (5). It generally                     Poorly soluble mineral phosphates hydroxyapatite <br /> accepted that, gram-negative bacteria, membrane-                  (HAP) dissolved acidification (7, 8). The bacterial <br /> bound quinoprotein GDH present periplasmic                Mps+ phenotype assayed addition HAP or <br /> of cytoplasmic membrane. GDH member                      poorly soluble phosphates culture medium. On <br /> largest group quinoproteins, use cofactor              agar plates, solubilization visualized directly by <br /> 2,7,9-tricarboxyl-lH-pyrrolo[2,3-flquinoline-4,5-dione (PQQ)             clearing zones, liquid culture level Pi may <br /> (5).                                                                     determined analytically. As expected, bacteria pro- <br />   Escherichia coli does synthesize PQQ, does                  duce large amounts gluconic acid, E. herbicola, <br /> synthesize apo-GDH dependent uptake                  high levels mineral phosphate solubilization, <br /> of PQQ environment culture medium (9). Bind-                 E. coli strains usually poor solubilizers. Our <br /> ing cofactor presumably simplified location          strategy clone genes E. herbicola E. <br /> the apoenzyme outer face cytoplasmic mem-                  coli order study encoded gene products involved in <br /> brane. The GDH holoenzyme formed E. coli                  Mps+ phenotype. It presumed genes responsi- <br /> functional genes PQQ biosynthesis introduced. This               ble Mps+ phenotype genes controlling organic <br /> system elegantly exploited Goosen et al. (9)              acid production and/or secretion identical or <br /> order identify isolate PQQ synthase genes                    closely related (7, 8). <br /> Acinetobacter calcoaceticus basis expression                In paper, report construction plasmid <br /> on plasmids cloned E. coli. The expression PQQ                   containing functional mineral phosphate-solubilizing gene <br />                                                                          defined HAP-solubilizing phenotype. Data that <br />                                                                          strongly suggest metabolic function protein <br />   * Corresponding authors.                                               encoded gene biosynthesis PQQ, which, in <br />                                                                   5814 <br />  VOL. 174, 1992                                      CLONING OF AN ERWINLA HERBICOLA PQQ SYNTHASE GENE                                       5815 <br />  <br />                                                TABLE 1. Bacterial strains plasmids <br />       Strain plasmid                                  Characteristics                                       Source reference <br /> Strains <br />   Envinia herbicola EHO10        Wild type                                                           18 <br />   Escherichia coli <br />     HB101                        F- hsdS20 (rB- mB-) supE44 recA13 ara-14proA2                       4 <br />                                    rpsL20(Strr) xyl-S mtl-ql galK2 lacYl X <br />     MG1063                       recA F+::TnlOOO                                                     10 <br />     P678-54                      F- thr leu thi supE lacYfluA gal mal xyl ara mtl min                C. I. Kado laboratory, University <br />                                                                                                         California Davis <br />     Eco898                        HB101::TnS                                                         S.-T. Liu laboratory <br />  <br /> Plasmids <br />   pSL775                          A cloning vector using pigment genes E. herbicola          14 <br />                                     selection marker; 7.0 kb <br />   pMCG898                         A deletion derivative pIMPS1 containing complete              This study <br />                                     PQQ synthase gene <br />  <br />  <br /> turn, results assembly GDH holoenzyme,               lysed, proteins analyzed sodium dodecyl <br /> presented. We report complete nucleotide sequence               sulfate-polyacrylamide gel electrophoresis (SDS-PAGE) ac- <br /> this gene deduced amino acid sequence                cording method Laemmli (13). <br /> shows high degree homology gene III               DNA sequencing. Supercoiled plasmids sequenced <br /> four-gene PQQ synthase complex A. calcoaceticus <br /> identified Goosen et al. (9). <br />  <br />                 MATERIALS AND METHODS                                                         O- Hindlil <br />    Bacterial strains, plasmids, culture conditions. Bacte-                                     8amHI                  BamHI <br /> rial strains plasmids used study listed Table <br /> 1. E. coli cultured Luria-Bertani (LB) broth (16) at <br /> 37&#194;&#176;C. E. herbicola cultured medium 28&#194;&#176;C. <br /> The antibiotics used selection ampicillin (Ap) (100 <br /> ,ug/ml) kanamycin (50 ,ug/ml). <br />    Subcloning. Plasmids screened according                         Hindlil <br /> method Kado Liu (12). Restriction enzymes, T4 DNA <br /> ligase, E. coli DNA polymerase I (Klenow fragment)                     pIMPS1 (55kb) <br /> were purchased Boehringer Mannheim Biologicals and <br /> were used according methods described Maniatis et <br /> al. (16). HindIII fragments insert cosmid pIMPS1 <br /> (8) subcloned HindIII site pBR322- <br /> derived vector pSL775 (14). The clone pB2 (27.6 kb) con-                                                                              ,   Banit <br /> taining Mps+ activity selected M9-HAP agar. This <br /> plasmid deleted BamHI (pB8, 13.3 kb),                                                                                      7.0 <br /> digested BanII, religated, resulted                                                                                   &apos;BgIll <br /> pMCG898 (Fig. 1), retained ability confer the <br /> HAP-solubilizing phenotype recipient E. coli. <br />    TnlOOO mutagenesis. TnlO00 insertional mutagenesis                                                                  pB8(1 3.3kb) <br /> (gamma-delta) carried according method of <br /> Guyer (10). Plasmid pMCG898 transformed                                                                      Banil <br /> MG1063. pMCG898::Tnl000 insertional mutants trans- <br /> ferred Eco890 conjugation. The resulting transcon- <br /> jugants cultured LB broth kanamycin then <br /> plated LB agar containing Ap kanamycin. The                                                                        Banil <br /> locations orientations TnlOOO insertions were <br /> determined restriction mapping BamHI.                                                                        AP(      Bgll <br />    Preparation minicells analysis plasmid-encoded                                                5.0 vector  PMC0981 <br />                                                                                                                         9.5kb  gSU 4. 5 <br /> proteins. The plasmids pMCG898 pBR322 trans-                                                          NdeI <br /> formed minicell-producing strain P678-54 (17). Cul-                                                                              Kpnt <br /> tures grown overnight LB-Ap broth 37&#194;&#176;C. Minicells                                                      EooRV                   del <br /> were purified sucrose density-gradient centrifugation, and <br />                                                                                                                               BamHI <br /> plasmid-encoded proteins labelled 45 pCi                    FIG. 1. Subcloning strategy map pMCG898 carrying a <br /> L-[35S]methionine (1,020 Ci/mmol; Amersham Corp.) accord-              putative E. herbicola PQQ synthase gene. This gene originally <br /> ing method Meagher et al. (17). The minicells           cloned 55-kb insert cosmid vector pHC79 (8). <br />   5816     LIU ET AL.                                                                                                      J. BACTEkRIOL. <br />  <br />  according chain termination method Sanger et al. <br />  (19). DNA sequenced gamma delta                 BamHI                                               Banil <br />  primers according method Liu et al. (15). Gaps were <br />  filled primers synthesized results obtained               I <br />  gamma-delta sequencing (primer walking). DNA <br />  strands sequenced. The primers synthe- <br />  sized Cruachem PS250 DNA synthesizer. Sequence                   I          I            I         I          I <br />  analysis, translation, comparisons previously iden-                                                                     I <br />  tified PQQ synthase genes using Intellige-             0          1          2          3           4          5 kb <br />  netics Suite program (11).                                         FIG. 2. Transposon TnlOOO mutagenic analysis region of <br />     Changes medium pH bacterial growth. Bacteria        pMCG898 necessary gluconic acid production Mps+ <br />  cultured overnight LB-Ap. Cells harvested        phenotype. The mutants shown figure contained <br />  centrifugation washed times LB-Ap.           approximately 1.8-kb region. These insertions simulta- <br />  Cells (from 5-ml culture) inoculated 250 ml       neously eliminated gluconic acid production Mps+ pheno- <br />  M9WP culture medium. Media (20 ml) sampled 6          type. Insertions outside region did affect GDH+ or <br />  h, cells centrifuged, supernatants         Mps+ phenotype. Insertions gamma-delta orientation (0) <br />  passed Millipore filters. The pH values deter-      delta-gamma orientation (-) indicated. <br /> mined, 100 ,ul supernatants spotted on <br /> M9WP-HAP medium determination HAP solubi- <br /> lization. M9WP-HAP modified M9 (16) minimal medium,         tions covered approximately 1.8-kb segment, as <br /> with soluble Pi replaced HAP previously described (8),     shown Fig. 2. Data based size insert region, <br />  1% glucose carbon source, appropriate amino acids      restriction map, sequencing results strongly <br /> and antibiotics (8). M9WP medium             indicated Mps- insertions located gene. <br /> added HAP.                                                       We did Mps- mutants insertions <br />    Mineral phosphate solubilization determination or-     outside 1.8-kb region. <br /> ganic acids bacterial growth. Liquid medium (GMS            Data shown Fig. 3 indicate proteins <br /> medium) consisted (for 1 liter) 10 g glucose, 2 g       encoded genes carried pMCG898. Plasmid <br /> (NH4)2S04, 0.3 g MgCl2. 6H20, 0.001 g MnCl2- 4H20,         pSL775 originally derived pBR322 (14). In <br /> 0.0006 g FeSO4. 7H20, 0.0006 g NaMoO4. Amino           pMCG898, pSL775-specific sequence de- <br /> acids added 20 ,ug/ml, Ap added 50 ,ug/ml.    leted pBR322 used control minicell <br /> Tricalcium phosphate present 20 mg/ml. A 50-ml <br /> volume media inoculated 5 ml 48-h-old <br /> (stationary phase) liquid culture various bacteria.                                       A   B    C    D <br /> After inoculated, flasks placed shaker and <br /> grown 30&#194;&#176;C (E. herbicola) 37&#194;&#176;C (E. coli). One- <br /> milliliter samples taken daily. The concentration of <br /> soluble Pi plate counts determined these <br /> samples. Organic acids separated high-performance <br /> liquid chromatography Beckman Interaction Ion 300 <br /> column. Detection A220. The identification un- <br /> known organic acids based comparison elution <br /> times unknowns elution times multiple stan-                              41   - <br />  <br />  <br /> dards. The detection limit gluconic acid 10 ppm.                                33 <br />    Nucleotide sequence accession number. The GenBank ac-                                32   -       I <br /> cession number nucleotide sequence PQQ <br /> synthase gene E. herbicola EHO10 M94448. <br />  <br />                 RESULTS AND DISCUSSION <br />    The screening strategy isolation clones was <br /> based fact that, plate assays, HB101 visibly <br /> solubilize poorly soluble mineral phosphates HAP <br /> and tricalcium phosphate, E. herbicola (8). <br /> Plasmid pMCG898 (insert size, 4.5 kb) (Fig. 1) originally <br /> derived cosmid clone pIMPS1, contained a <br /> 55-kb insert E. herbicola DNA. This cosmid identified <br /> by Goldstein Liu ability confer HAP- <br /> solubilizing phenotype HB101 (8). <br />    As shown Fig. 1 2, combination subcloning and <br /> transposon mutagenesis data indicated 1.8-kb region         FIG. 3. SDS-PAGE [35SJmethionine-labelled proteins en- <br /> of E. herbicola DNA sufficient confer GDH+            coded pMCG898 minicells. Proteins encoded pBR322 (lane <br /> Mps+ phenotype E. coli. On basis selection        A) synthesis directed pMCG898 (lane B) shown. A <br /> M9-HAP agar appropriate supplements, screened          Klenow fragment repair-induced <span id='am-81' about='xsp:base' typeof='owl:Thing'>base</span> insertion BamHl site of <br /> total 113 insertion mutants copy           pMCG898 caused loss 33-kDa protein (lane C), a <br />                                                                  similar insertion BanlI site did affect synthesis this <br /> pECO898::Tn1O00. Twenty-five Mps-, rest             protein (lane D). The indicated proteins Mrs 41,000 and <br /> were Mps+. These 25 mutants insertions loca-         32,000 apparently encoded 4.5-kb insert. <br />  VOL. 174, 1992                                      CLONING OF AN ERWINIA HERBICOLA PQQ SYNTHASE GENE                                 5817 <br />  <br />                                   RTGRCTGNCC   RCGCGCCCRC    GGTGRRCCCR    CCGCTCTGGC TGCTGGCGGR GCTGRCCTRC <br />                            60    CGCTGCCCGC    TGCRGTGCCC    CTRCTGCTCC    RRCCCRCTCG RTTTCGCCGC CCRAGRRRRR <br />                            120   GRGCTGRGTR    CCGRGCRGTG    GRTCGRGGTR    TTTCGCCRGG CRCGCGCGRT GGGCRGCGTG <br />                            180   CRGTTGGGCT    TCTCGGCGGG    CGRGCCGCTG    GTSCGTRRRG RTCTCCCCGR GCTGRTCGCC <br />                            240   GCCGCGCGCG    RTCTCGGCTT    TTRCRCCRRC    CTGRTCRCCT CCGGCRTCGG CCTGRCCGRG <br />                            300   RRGRRGCTGC    RGRCCTTCGC    CGRTGCCGGG    CTGGRCCRTR TTCRGRTCRG CTTTCRRGCC <br />                            360   RGCGRCGRGR    CGTTRRRCGC    GGCTCTGGCT    GGCTCTGCCR RRGCRTTTCR GCRGRRGCTG <br />                            420   RCRRTGGCCR    RRGCGGTTRR    GGCGCTCGGC    TRTCCGRTGG TGCTCRRCTT CGTGCTGCRT <br />                            480    CSCCRTRRTR   TCGRTCRGRT    TGRCCGTRTC    RTCGRGCTGG CGRTCCRGCT GGRTGCCGRT <br />                            540    GRCGTRGRGC   TGGCRRCCTG    CCRGTTCTRC    GGCTGGGCGC RCCTGRRTCG CGRRGGCCTG <br />                            600    CTTCCCRCCC   GCGRRCRGRT    CGCCRRRGCR    GRGRGCGTGG TCCRGCGCTR CCGCGRRCGC <br />                            660   RTGGCCGTCG    RCGGCRRGCT    GGCTRRTCTG    CTGTTTGTCR CGCCGGRCTR CTRTGRGGRG <br />                            720   CGGCCRRRRG    GCTGTRTGGG     CGGTTGGGGR    GCCRTTTTCR TGRGCGTGRC GCCGGRRGGR <br />                            780   RTGGCCCTGC    CCTGCCRCRG     TGCCCGTCRG    CTGCCGGTGR RGTTTCCGTC GGTGCTGGRR <br />                             840 CRCGRTCTCG     RGCRTRTCTG     GTRTCRCTCC    TTCGGCTTTR RCCGTTRCCG CGGCRRTGCG <br />                             900  TGGRTGCCGG    RGCCGTGCCG     RTCCTGCCCG    GRGRRRGRGR RRGRCTTCGG CGGCTCRTGC <br />                             960  TGCCRGGCCT    TCRTGCTCRC     CGGCGRCGCT    GRTRRTGCCG RCCCGGTGTG TGCTRRRTCG <br />                             1020 CCCCRTCRCG    GCRCCRTTTT     GGCGGCGCGR    GRRGCGGCRR RCCGCRCCCR SGCTCGGCRTC <br />                             1080 GRTCRGCTGC     GTTTCCGCRR    TCRGGCCRRC    TCCCGGCTGT TTTRCRRRGG CTRR <br />  <br />   FIG. 4. DNA sequence open reading frame (starting +1) presumed coding mps gene contained 1.8-kb region <br /> identified transposon mutagenesis essential gluconic acid production E. coli HB101(pMCG898). <br />  <br />  <br /> analyses. Three proteins apparent Mrs 41,000,                solubilizing 24 32% tricalcium phosphate <br /> 33,000, 32,000 synthesized pMCG898            solubilized E. herbicola time period. <br /> pBR322. When base insertion generated BamHI             Each value represents average separate experi- <br /> site pMCG898 Klenow repair, protein Mr          ments. Numbers CFUs similar various <br /> 33,000 longer synthesized (Fig. 3, lane C). However,          bacterial strains (not shown). <br /> a base addition BanIl site did eliminate                From Table 2, seen gluconic acid was <br /> synthesis protein (Fig. 3, lane D), suggesting      detectable media E. herbicola the <br /> proteins Mrs 41,000 32,000 encoded           HB1O1(pMCG898) cells media HB101 or <br /> E. herbicola DNA insert, protein Mr             HB1O1(pBR322) cells. E. herbicola carried most <br /> 33,000 encoded fusion sequence vector          mineral phase solubilization day 3. Likewise, the <br /> the insert. As discussed below, open reading frame <br /> identified 1.8-kb region pMCG898 essential for <br /> gluconic acid production encoded protein deduced                  MTIHAPPPLWLLAELTYRCPLQCPYCSNPLDFAAQEKELSTEQWIE VF <br /> Mr 42,160, compares favorably protein                       MTEGVGLPLWLLAELTYRCPLQCPYCSNPLDYAQHKNELTTQEWFDVF <br /> with Mr 41,000 seen minicell analysis. <br />    We obtained 1,960 bp sequence covering entire                    POARAMGSVQLGFSAGEPLVRK DLPELI AAA RDL G FYTNUTSG    GLTEKKLQ <br /> 1.8-kb region identified TnlO00 mutagenesis essential                DAQMGQVLGFSGGEPLVPaXE-LVAHA HQQGFYTNLITSGMGLTEQRIA <br /> for GDH+ Mps+ phenotype. Figure 4 shows com- <br /> plete sequence significant open reading frame                  TFAD AGLDHIQ SFQASDETLNA ALAGSAK AFQQKLT MAKAVKALGYPMVLN <br /> contained fragment. This open reading frame                 DLKQ AGLDHIQVSFQASDPVVNDALAGSKH  AFEQKYEMCRLVKKYDYPMVLN <br /> 1,134-bp long codes protein deduced Mr of <br /> 42,160. Figure 5 compares deduced amino acid sequence                  FVLHRHN IDQ IDR I ELAJQLDADDVELATCOFYGWAHLNREGLL PTREQ IAKA <br /> of putative PQQ synthase gene gene III a <br /> multigene complex involved synthesis PQQ A.                   FV IHRHN IDO IEQ I I ELClENADTVELA ICQFYGWAF LNRQGLLPTQECLTRA <br /> calcoaceticus (9). It seen that, overall, 60% <br /> identity E. herbicola gene gene III A.                  ESWQRYRERMAVDGKLANL LFVrPDYYEERPKGCMGGWGA IFMSVTPEG <br /> calcoaceticus.                                                             ER TNEYREKL KAONHPCKLI FWPDY YEERPKCMNGWGK IFFT VAPDG <br />    At end 24 h liquid culture, E. herbicola had <br />  lowered pH media 3.5. HB1O1(pMCG898)                        MALPCHSARCLPVKFPSVL EHDLEHIWYHSFGFNRYRGNAWMPEPCRSCP <br />  lowered pH approximately 4.0 end                MALPCH/RtLP IS FPNVREHKLSDI WYKSTGFNHFRGDAWMPEGCRSCP <br />  time period. HB1O1(pMCG898-50) (a pMCG898:Tnl000 mu- <br />  tant insertion outside 1.8-kb Mps region) gave a <br />  similar phenotype. HBBO1(pBR322) capable low-                  EKEKDFGGSCCQAF MLTGDADNADPVCAKS PHHGT IL AAREAANRTOLGID <br />  ering pH media 5.2                        DKDRDFGGCRCQAYKLTGDAANADPVCGKS PYHQMIEQARAESOLVAPLON <br />  conditions. HBBO1(pMCG898-51) (TnlOOO insertion within <br />  1.8-kb Mps region) unable lower pH <br />  5.2 (pH data shown).                                            QLRFRNQANSRLFYKG <br />    Table 2 shows soluble Pi released                    LVFRNSRNSKS LSATQNIPVHTITDI <br />  tricalcium phosphate medium various bacteria. It            FIG. 5. Alignment deduced amino acid sequences E. <br />  seen that, day 7, HB1O1(pMCG898) solubilized               herbicola mps gene (upper) gene III multigene complex <br />  approximately 81% tricalcium phosphate E.                 involved synthesis PQQ A. calcoaceticus. Regions of <br />  herbicola. HB101 HB1O1(pBR322) capable                   identity indicated connecting dots. <br />  5818       LIU ET AL.                                                                                                                   J. BACTEDRIOL. <br />  <br />    TABLE 2. Production gluconic acid release Pi                On basis data, propose have <br />      tricalcium phosphate different bacterial strains             cloned characterized PQQ synthase gene E. <br />                          liquid culturea                                    herbicola gluconic acid produced GDH-PQQ <br />                                                            Gluconic acid    holoenzyme provides molecular basis Mps+ phe- <br />         Strain             Days culture   P1 released <br />                                                 (i~g/ml)    (integrated     notype shown E. herbicola recombinant E. coli. In <br />                                                             area   units)   gram-negative bacteria, gluconic acid undergo <br /> EHO10                            3              350            8.65         extracellular oxidation 2,5-diketogluconic acid (2). <br /> HB101(pMCG898)                   7              300            4.12         These pathways provide basis metabolic <br /> HB101                            7               88            NDb          engineering Mps+ phenotype accomplish effica- <br /> HB1O1(pBR322)                    7              115            ND           cious biotransformation rock phosphates. <br /> Blankc                           7                8            ND <br />   Data averages separate experiments.                                           ACKNOWLEDGMENTS <br />   bND, detected.                                                          This work supported biological research grant NSC78- <br />   c No bacteria present.                                                    0418-B169-03 S.T.L. National Science Council the <br />                                                                             Republic China research contract DEAC07-76ID01570 <span id='am-2' about='protege:TO' typeof='owl:Thing'>to</span> <br />                                                                             J.H.W., R.R., A.H.G. USDOE Office Industrial <br />  gluconic acid peak detected day 3. The appearance                Processes, Division Advanced Industrial Concepts. <br />  gluconic acid peak HB101(pMCG898) delayed <br />  later experiment peaked day 7, did                                              REFERENCES <br />  maximum mineral phosphate solubilization. HB101(pMCG                        1. Ameyama, M., K. Matsushita, Y. Ohno, E. Shinagawa, 0. <br />  898) produced approximately 48% gluconic acid                           Adachi. 1981. D-Glucose dehydrogenase  <br /> </body></html>