<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Srihari et al. BMC Bioinformatics 2010, 11:504 <br /> http://www.biomedcentral.com/1471-2105/11/504 <br />  <br />  <br />  <br />  <br />  RESEARCH ARTICLE                                                                                                                             Open Access <br />  <br /> MCL-CAw: refinement MCL detecting <br /> yeast complexes weighted PPI networks by <br /> incorporating core-attachment structure <br /> Sriganesh Srihari1*, Kang Ning2,3*, Hon Wai Leong1* <br />  <br />  <br />   <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />   Background: The reconstruction protein complexes physical interactome organisms serves a <br />   building block understanding higher level organization cell. Over past years, several <br />   independent high-throughput experiments helped catalogue enormous physical protein <br />   interaction data organisms yeast. However, individual datasets lack correlation with <br />   contain substantial number false positives (noise). Over years, affinity scoring <br />   schemes devised improve qualities datasets. Therefore, challenge <span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span> <br />   detect meaningful novel complexes protein interaction (PPI) networks derived combining <br />   datasets multiple sources making use affinity scoring schemes. In attempt towards <br />   tackling challenge, Markov Clustering algorithm (MCL) proved popular reasonably <br />   successful method, mainly scalability, robustness, ability work scored (weighted) networks. <br />   However, MCL produces noisy clusters, match known complexes additional <br />   proteins reduce accuracies correctly predicted complexes. <br />   Results: Inspired recent experimental observations Gavin colleagues modularity structure yeast <br />   complexes distinctive properties &#226;&#8364;&#339;core&#226;&#8364;? &#226;&#8364;&#339;attachment&#226;&#8364;? proteins, develop core-attachment based <br />   refinement method coupled MCL reconstruction yeast complexes scored (weighted) PPI networks. <br />   We combine physical interactions recent &#226;&#8364;&#339;pull-down&#226;&#8364;? experiments generate unscored PPI network. <br />   We score network using available affinity scoring schemes generate multiple scored PPI networks. The <br />   evaluation method (called MCL-CAw) networks shows that: (i) MCL-CAw derives larger number of <br />   yeast complexes better accuracies MCL, particularly presence natural noise; (ii) Affinity <br />   scoring effectively reduce impact noise MCL-CAw improve quality (precision and <br />   recall) predicted complexes; (iii) MCL-CAw responds available scoring schemes. We discuss <br />   instances MCL-CAw successful deriving meaningful complexes, missed few <br />   proteins complexes affinity scoring networks. We compare MCL-CAw recent <br />   complex detection algorithms unscored scored networks, assess relative performance the <br />   algorithms networks. Further, study impact augmenting physical datasets computationally <br />   inferred interactions complex detection. Finally, analyse essentiality proteins predicted <br />   complexes understand possible correlation protein essentiality ability form complexes. <br />   Conclusions: We demonstrate core-attachment based refinement MCL-CAw improves predictions of <br />   MCL yeast PPI networks. We affinity scoring improves performance MCL-CAw. <br />  <br />  <br />  <br /> * Correspondence: srigsri@comp.nus.edu.sg; albertnk@gmail.com; <br /> leonghw@comp.nus.edu.sg <br /> 1 <br />  Department Computer Science, National University Singapore, 117590, <br /> Singapore <br /> 2 <br />  Department Pathology, University Michigan, Ann Arbor, MI 48109, USA <br /> Full list author information available end article <br />  <br />                                       &#194;&#169; 2010 Srihari et al; licensee BioMed Central Ltd. This Open Access article distributed terms Creative Commons <br />                                       Attribution License (http://creativecommons.org/licenses/by/2.0), permits unrestricted use, distribution, reproduction in <br />                                       medium, provided original work properly cited. <br />  Srihari et al. BMC Bioinformatics 2010, 11:504                                                                 Page 2 25 <br /> http://www.biomedcentral.com/1471-2105/11/504 <br />  <br />  <br />  <br />  <br /> Background                                                     iterative clustering approach derive complexes. <br /> Most biological processes carried proteins          Each complex partitioned groups pro- <br /> that physically interact form stoichiometrically stable     teins called &#226;&#8364;&#339;core&#226;&#8364;?, &#226;&#8364;&#339;attachment&#226;&#8364;? &#226;&#8364;&#339;module&#226;&#8364;? (depicted in <br /> complexes. Even relatively simple model organism        Additional files 1, Figure S1). On hand, Krogan <br /> Saccharomyces cerevisiae (budding yeast), com-           et al. [7] used machine learning techniques (Bayesian <br /> plexes comprised subunits work           networks C4.5-based decision trees) define confi- <br /> coherent fashion. These complexes interact indivi-        dence scores interactions derived direct bait- <br /> dual proteins complexes form functional            prey observations (the spoke model). Subsequently, Kro- <br /> modules pathways drive cellular machinery.        gan et al. defined high-confidence &#226;&#8364;&#732;Core&#226;&#8364;&#8482; dataset of <br /> Therefore, faithful reconstruction entire set      interactions, used Markov Clustering algorithm <br /> complexes physical interactions pro-          (MCL) [18,19] derive complexes. Hart et al. [12] gen- <br /> teins essential understand complex forma-       erated Probabilistic Integrated Co-complex (PICO) net- <br /> tions, higher level organization cell.     work integrating matrix modeled relationships the <br />   These physical interactions proteins            Gavin et al., Krogan et al. Ho et al. datasets using a <br /> been extensively catalogued yeast using high-         measure similar socio-affinity scores, used a <br /> throughput methods like yeast two-hybrid [1,2]             MCL procedure derive complexes network. <br /> direct purification complexes using affinity tags fol-      Collins et al. [11] developed Purification Enrichment <br /> lowed mass spectrometry (MS) analyses [3]. In 2002,         (PE) scoring generate &#226;&#8364;&#732;Consolidated net- <br /> the direct purification strategy &#226;&#8364;&#339;pull-down&#226;&#8364;?      work&#226;&#8364;&#8482; matrix modeled relationships the <br /> applied yeast independent studies Gavin           Gavin et al., Krogan et al. datasets. Collins et al. <br /> et al. [4] Ho et al. [5]. More recently (2006),        used Bayes classifier generate PE scores the <br /> separate groups, Gavin et al. [6] Krogan et al. [7],       Consolidated network incorporating diverse evidence <br /> employed tandem affinity purification (TAP) followed        hand-curated co-complexed protein pairs, Gene <br /> MS analyses produce enormous new data,            Ontology (GO) annotations, mRNA expression patterns, <br /> allowing complete mapping yeast interac-         cellular co-localization co-expression profiles. <br /> tome. Although individual datasets high           This new network shown high quality - <br /> quality, surprising lack correlation         comparable PPIs derived small-scale <br /> each [8,9], bias high abundance         experiments stored Munich Information Center <br /> proteins [10] proteins certain cellular       Protein Sequences (MIPS). Zhang et al. [13] used <br /> compartments (like cell wall plasma membrane)              Dice coefficient (DC) assign affinities protein pairs, <br /> [11]. Also, dataset contains substantial num-     evaluated affinity measure socio-affinity <br /> ber false positives (noise) compromise         PE measures. They concluded DC PE <br /> utility datasets focused studies like        offered best representation protein affinity, and <br /> complex reconstruction [11]. In order reduce            subsequently used complex prediction. Pu et al. <br /> impact discrepancies, number data integra-        [20] used MCL combined cluster overlaps the <br /> tion affinity scoring schemes devised            Consolidated network reveal interesting insights into <br /> [6,7,11-17]. These affinity scores encode reliabilities    complex organization. Wang et al. [21] proposed HACO, <br /> (confidence) physical interactions pairs         hierarchical clustering overlap algorithm, recon- <br /> proteins. Therefore, challenge detect            struct complexes used build &#226;&#8364;&#732;Complex- <br /> meaningful novel complexes protein             Net&#226;&#8364;&#8482;, interaction network proteins complexes, <br /> interaction (PPI) networks derived combining multi-         order study higher-level organization com- <br /> ple high-throughput datasets making use              plexes. Chua et al. [14] Liu et al. [15] developed net- <br /> these affinity scoring schemes.                                work topology-based scoring schemes called Functional <br />   The interaction data produced high-through-         Similarity Weight (FS Weight) Iterative-Czeka- <br /> put TAP/MS experiments comprise tagged &#226;&#8364;&#339;bait&#226;&#8364;? pro-          nowski-Dice (Iterative-CD), respectively, assign relia- <br /> teins associated &#226;&#8364;&#339;prey&#226;&#8364;? proteins co-purify        bility scores interactions networks. <br /> with baits. Gavin et al. [6] considered direct bait-prey   Subsequently, Liu et al. [16] used maximal clique mer- <br /> as indirect prey-prey relationships (a combination     ging strategy (called CMC) derive complexes net- <br /> of spoke matrix models), followed socio-affinity      works scored using systems. Friedel et al. [17] <br /> scoring encode affinities pro-       developed bootstrapped scoring score TAP/ <br /> tein pairs. The socio-affinity score quantizes log-ratio   MS interactions Gavin et al. Krogan et al., and <br /> of number times proteins observed               subsequently derived complexes using variant MCL. <br /> together relative expected         Friedel et al. [22] developed minimum spanning <br /> frequency dataset. Subsequently, Gavin et al. used      tree-based method reconstruct topology of <br />  Srihari et al. BMC Bioinformatics 2010, 11:504                                                                               Page 3 25 <br /> http://www.biomedcentral.com/1471-2105/11/504 <br />  <br />  <br />  <br />  <br /> complexes co-purified proteins TAP/MS assays.       Table 1 Properties PPI networks used the <br /> Voevodski et al. [23] used PageRank, random walk-         evaluation MCL-CAw <br /> based method employed context-sensitive web search,                  PPI Network                      #           #             Avg <br /> to define affinities proteins PPI net-                                             Proteins Interactions        node <br />                                                                                                                                   degree <br /> works. Subsequently, Voevodski et al. used predict <br />                                                                              Gavin                      1430          7592         10.62 <br /> co-complexed proteins network. Approaches <br />                                                                         Krogan &#226;&#8364;&#732;Core&#226;&#8364;&#8482;                   2708          7123          5.26 <br /> like CORE [24] COACH [25] adopted local dense <br /> neighborhood search derive cores attachments                     Gavin+Krogan                    2964         13507          9.12 <br /> <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> unscored networks. Mitrofanova et al. [26] mea-                 ICD(Gavin+Krogan)                  1628          8707         10.69 <br /> sured connectivity proteins unweighted                FSW(Gavin+Krogan)                  1628          8688         10.67 <br /> PPI networks edge-disjoint paths instead edges                Consolidated3.19                 1622          9704         11.96 <br /> overcome noise, modeled paths network                    Bootstrap0.094                  2719         10290          7.56 <br /> flow represented Gomory-Hu trees. They subse-                     Inferred                     954         11892         24.93 <br /> quently isolated groups nodes trees shared          Gavin+Krogan+Inferred                3418         25352         14.83 <br /> edge-disjoint paths order identify complexes. Very         ICD(Gavin+Krogan+Inferred)             2034         12009         11.81 <br /> recently, Ozawa et al. [27] used domain-domain interac-         FSW(Gavin+Krogan+Inferred)              1892         11705         12.37 <br /> tions validate refine complexes predicted     The Gavin+Krogan network generated combining Gavin and <br />                                                             Krogan-Core datasets, obtained BioGrid [32]. The ICD(Gavin+Krogan) and <br /> MCL.                                                        FSW(Gavin+Krogan) networks generated scoring Gavin+Krogan <br />    In study, develop algorithm derive yeast   network using Iterative-CDk FS Weightk schemes [14,15] (with k = 2 <br /> complexes weighted (affinity-scored) PPI networks.     iterations). The Consolidated3.19 network refers high-confidence subset <br />                                                             (PE cut- off: 3.19) Consolidated network derived Collins et al. [11]. <br /> Inspired experimental findings Gavin et al. [6]   The Bootstrap0.094 network refers high-confidence subset (BT cut-off: <br /> on modularity structure yeast complexes,     0.094) Bootstrap network derived Friedel et al. [17]. The Inferred <br /> distinctive properties &#226;&#8364;&#339;core&#226;&#8364;? &#226;&#8364;&#339;attachment&#226;&#8364;? pro-      network comprised computationally inferred interactions the <br />                                                             Predictome database [40]. The Gavin+Krogan+Inferred network generated <br /> teins, develop novel core-attachment based refine-     augmenting Gavin+Krogan network inferred interactions. <br /> ment method coupled MCL reconstruction            The ICD(Gavin+Krogan+Inferred) FSW(Gavin+Krogan+Inferred) networks <br />                                                             generated scoring augmented network using Iterative-CD <br /> yeast complexes. We proposed idea core-          FS Weight systems, respectively. <br /> attachment based refinement preliminary work [28] <br /> and called MCL-CA. <br />    However, MCL-CA worked unscored net- <br /> works. Here, devise improved algorithm (called        unscored scored PPI networks. Finally, we <br /> MCL-CAw) provide natural extension work            perform in-depth analysis predicted complexes <br /> on scored (weighted) PPI networks. Even              MCL-CAw. <br /> most eukaryotic complexes hypothesized dis- <br /> play core-attachment modularity, design        Methods <br /> our algorithm specific yeast complexes        The MCL-CAw algorithm: Identifying complexes <br /> lack sufficient evidence, high-throughput datasets       embedded interaction network <br /> and reference complexes organisms. We            Our MCL-CAw algorithm broadly consists two <br /> combine TAP/MS physical datasets Gavin et al.          phases. In phase, partition PPI network <br /> [6] Krogan et al. [7] generate unscored PPI       multiple dense clusters using MCL. Following this <br /> network (Table 1). We score network using         (in second phase), post-process (refine) these <br /> two topology-based affinity scoring schemes, FS             clusters obtain meaningful complexes. The MCL- <br /> Weight [14] Iterative-CD [15], generate scored       CAw algorithm consists following steps: <br /> PPI networks. We gather additional readily-avail- <br /> able scored PPI networks Collins et al. [11]            1. Clustering PPI network using MCL hierarchically <br /> Friedel et al. [17]. The evaluation MCL-CAw                2. Categorizing proteins cores clusters <br /> these networks demonstrates that: (a) MCL-CAw                 3. Filtering noisy clusters <br /> able derive higher number yeast complexes              4. Recruiting proteins attachments clusters <br /> with better accuracies MCL; (b) Affinity scoring            5. Extracting complexes clusters <br /> effectively reduces impact noise MCL-CAw               6. Ranking predicted complexes <br /> and improves quality (precision and <br /> recall) predicted complexes; (c) MCL-CAw               We use following notations describing our <br /> responds available affinity scoring            algorithm. The PPI network represented G = (V, <br /> schemes PPI networks. We compare MCL-CAw                E), V set proteins, E set of <br /> with recent complex detection algorithms         interactions proteins. For e = (p, q) <br />  Srihari et al. BMC Bioinformatics 2010, 11:504                                                                                                               Page 4 25 <br /> http://www.biomedcentral.com/1471-2105/11/504 <br />  <br />  <br />  <br />  <br /> &#195;&#381; E, confidence score (weight) w(p, q) encod-          protein cluster Ci = (Vi, Ei), given p &#195;&#381; PCore(Ci), if: <br /> ing affinity proteins p q. These affi- <br /> nity scores depend scoring used.                        &#226;&#8364;&#162; The weighted in-connectivity p respect Ci <br /> Clustering PPI network using MCL hierarchically                   average weighted in-connectivity Ci, <br /> The step algorithm partition (cluster)             given by: din(p, Ci) &#226;&#8240;&#165; davg(Ci); and <br /> the PPI network using MCL [18], simulates ran-                  &#226;&#8364;&#162; The weighted in-connectivity p respect to <br /> dom walks (called flow) identify relatively dense                Ci greater weighted out-connectivity p <br /> regions network. The inflation coefficient para-               respect Ci, given by: din(p, Ci) &gt; dout(p, Ci). <br /> meter I MCL used regulate granularity of <br /> the clusters - higher value finer gener-           The weighted in-connectivity d (p, C ) p with <br /> ated clusters (how choose I practice discussed        respect Ci total weight (score) interactions p <br /> the &#226;&#8364;&#339;Results&#226;&#8364;? section). MCL tends produce              proteins Ci. Similarly, weighted out- <br /> large clusters (sizes &#226;&#8240;&#165; 30) amalgamate smaller clus-         connectivity dout(p, Ci) p respect Ci total <br /> ters [7,20]. On hand, size distributions         weight interactions p proteins outside C . <br /> hand-curated complexes Wodak lab [29], MIPS                  These given din(p, Ci) = &#226;&#710;&#8216;{w(p, q) : q &#195;&#381;Vi} dout <br /> [30] Aloy et al. [31] (Table 2) reveal com-         (p, Ci) = &#226;&#710;&#8216; {w (p, q) : q &#226;&#710;&#8240; Vi}, respectively. The average <br /> plexes sizes 10. Therefore, perform           weighted in-connectivity davg(Ci) cluster Ci therefore <br /> hierarchical clustering iteratively selecting clusters     average weighted in-connectivities all <br /> of sizes 30 re-clustering using MCL.                                                              d avg (C ) = 1 &#226;&#8249;&#8230; &#226;&#710;&#8216; d (q , C ) . <br />                                                                   proteins Ci, given                                    | C | q&#226;&#710;&#710;V <br />   After iterative rounds MCL-based hierarchical clus-                                                                                                 i <br />  <br /> tering protein network G = (V, E), obtain               We use preliminary cores &#226;&#8364;&#732;extended <br /> collection k disjoint (non-overlapping) clusters {Ci :         core&#226;&#8364;&#8482; proteins. We categorize protein p &#226;&#710;&#8240; PCore(Ci) to <br /> Ci = (Vi, Ei), 1 &#226;&#8240;&#164; &#226;&#8240;&#164; k}, Vi &#226;&#352;&#8224; V Ei &#226;&#352;&#8224; E.               extended core protein cluster Ci, given p &#195;&#381; <br /> Categorizing proteins cores clusters                    ECore(Ci), if: <br /> Microarray analysis Gavin et al. [6] predicted <br /> complex components showed large percentage                  &#226;&#8364;&#162; The weighted in-connectivity p respect to <br /> pairs proteins cores co-expressed               PCore(Ci) average weighted in- <br /> same time cell cycle sporulation, consis-              connectivities non-cores r &#226;&#710;&#8240; PCore(Ci) the <br /> tent view cores represent main functional               preliminary cores, given by: din(p, PCore(Ci)) &#226;&#8240;&#165; davg <br /> units complexes. Three-dimensional structural                  (r, PCore(Ci)); and <br /> and yeast two-hybrid analysis showed core                    &#226;&#8364;&#162; The weighted in-connectivity p respect to <br /> components likely direct physical                  PCore(Ci) greater weighted out-connectiv- <br /> contact other. To reflect findings             ity p respect PCore(C ), given by: d (p, <br /> post-processing steps, expect:                                     PCore(Ci)) &gt; dout(p, PCore(Ci)). <br />  <br />     &#226;&#8364;&#162; Every complex predict comprise non-                 Here, din(p, PCore(Ci)) total weight interac- <br />     set core proteins;                               tions p preliminary cores Ci, given by: <br />     &#226;&#8364;&#162; The proteins cores display rela-            din(p, PCore(Ci)) = &#226;&#710;&#8216;{w(p, q) : q &#195;&#381; PCore(Ci)}. Similarly, <br />     tively high degree physical interactivity            d (p, PCore(C )) total weight interactions <br />     themselves.                                                   p non-core proteins Ci, given by: <br />                                                                      din(p, PCore(Ci)) = &#226;&#710;&#8216;{w(p, r) : r &#195;&#381; PCore (Ci)}. Finally, <br />   We identify core proteins cluster           davg(r, PCore(Ci)) average weight interactions of <br /> stages: identify set preliminary cores        non-cores r preliminary cores, given by: <br /> subsequently extend form final set cores. We <br />                                                                                                                                        &#226;&#710;&#8216; <br />                                                                                                                  1 <br />                                                                      d avg ( r , PCore(C ) ) =                               &#226;&#8249;&#8230;                    d ( r , PCore(C ) ) . <br /> categorize protein p &#195;&#381;V &#226;&#8364;&#732;preliminary core&#226;&#8364;&#8482;                                             (| C | &#226;&#710;&#8217; | PCore(C ) |) <br />                                                                                                                                    r&#226;&#710;&#710; <br />                                                                                                                                     / PCore(C ) <br />  <br />  <br />  <br />  <br /> Table 2 Properties hand-curated yeast complexes Wodak lab [29], MIPS [30] Aloy [31] <br />                                                                             # Complexes size <br />    Benchmark           #Complexes           # Proteins      &lt;3             3-10                    11-25                  &gt; 25                          Avg density <br />       Wodak                408                   1627       172            204                       27                      5                                0.639 <br />        MIPS                313                   1225       106            138                       42                     27                                0.412 <br />        Aloy                101                   630        23              58                       19                      1                                0.747 <br />  Srihari et al. BMC Bioinformatics 2010, 11:504                                                                            Page 5 25 <br /> http://www.biomedcentral.com/1471-2105/11/504 <br />  <br />  <br />  <br />  <br />   Combining preliminary extended core pro-                                        &#226;&#710;&#8217;&#239;?&#167; <br />                                                                                   &#226;&#381;&#8250;S &#226;&#381;&#382; <br /> teins, form final set core proteins cluster           I p &#226;&#8240;&#165; &#239;?&#161; .I c . &#226;&#381;&#339; c &#226;&#381;&#376;        ,                                    (2) <br /> Ci, given by:                                                                     &#226;&#381;? 2 &#226;&#381;&#160; <br />  <br />     Core(C ) = {PCore(C ) &#226;&#710;&#170; ECore(C )}.            (1)      Ip = I(p, Core(Ci)) total weight interac- <br />                                                                tions p Core(Ci), given I(p, Core(Ci)) = &#226;&#710;&#8216;{w(p, <br /> Filtering noisy clusters                                       q): q &#195;&#381; Core(C )}, I c = I(Core(C )) total <br /> Consistent assumption complex              weight interactions core proteins C i, <br /> comprises set core proteins, consider clus-       given I(Core(C )) = 12 &#226;&#8249;&#8230; &#226;&#710;&#8216; {w(q, r ) : q, r &#226;&#710;&#710; Core(C )} , Sc = <br /> ter noisy does include core protein        |Core(Ci)|, is normalized yield 1 core sets <br /> per criteria. We discard noisy              size two. The parameters g used control <br /> clusters.                                                      effects I (Core(Ci)) |Core(Ci)|. For simple <br /> Recruiting proteins attachments clusters               illustration, let = 0.5 g = 1, consider inter- <br /> Microarray analysis Gavin et al. [6] predicted     actions equal weight 1. Therefore, p attached <br /> complex components showed attachment proteins             core set proteins, total weight its <br /> were closely associated core proteins com-         interactions core proteins 3, is <br /> plexes showed greater degree heterogeneity        possible p connected core proteins <br /> in expression levels, supporting notion attach-       (how choose values g practice dis- <br /> ments represent non-stoichiometric components.           cussed &#226;&#8364;&#339;Results&#226;&#8364;? section). This step ensures that <br /> Also, attachment proteins seen shared             non-core proteins having sufficiently strong interactions <br /> two complexes, consistent view           cores clusters recruited <br /> the protein participate multiple complexes         attachments clusters. <br /> [20,21]. On hand, application MCL          Extracting complexes clusters <br /> PPI networks yields clusters share proteins        For cluster group constituent core <br /> (non-overlapping clusters). Mapping clusters        attachment proteins define unique complex. We <br /> to original PPI network shows proteins having         expect remaining proteins cluster to <br /> similar connectivities multiple clusters assigned       weaker associations resultant complex, <br /> arbitrarily clusters. These proteins        categorize noisy proteins. In fact, <br /> might assigned multiple clusters. To reflect     experiments [28] shown MCL clusters tend to <br /> these findings algorithm, expect attach-         include noisy proteins leading reduction <br /> ment proteins proteins complexes            accuracies clusters. Therefore, step ensures <br /> that are:                                                      noisy proteins discarded order extract <br />                                                                accurate complexes. Additionally, these <br />     &#226;&#8364;&#162; Non-core proteins;                                       resulting complexes include attachment proteins that <br />     &#226;&#8364;&#162; Closely interacting core proteins;          potentially recruited multiple complexes, this <br />     &#226;&#8364;&#162; May shared multiple complexes.                 step ensures predicted complexes adhere the <br />                                                                protein-sharing phenomenon observed real complexes <br />   We consider following criteria assign non-core      [6,20,21]. We discard complexes size 4 <br /> protein p belonging cluster Cj (called donor cluster)     false positives. It difficult to <br /> as attachment acceptor cluster Ci (the donor      predict small real complexes solely based interaction <br /> acceptor clusters same), is, p &#195;&#381; Attach(Ci):   (topological) information (also noted [16,24]). <br />                                                                  For cluster C , define unique complex <br />     &#226;&#8364;&#162; Protein p sufficiently strong interactions      Cmplx(Ci) as: <br />     core proteins Core(Ci) cluster Ci; <br />     &#226;&#8364;&#162; The stronger interactions core pro-            Cmplx(C ) = {Core(C ) &#226;&#710;&#170; Attach(C )}.                       (3) <br />     teins, stronger interactions p <br />                                                                  Each interaction (p, q) constituent proteins <br />     core proteins; <br />                                                                p q complex carries weight w(p, q) <br />     &#226;&#8364;&#162; For large core sets, strong interactions required <br />                                                                observed PPI network. <br />     core proteins or, alternatively, <br />                                                                Ranking predicted complexes <br />     weaker interactions them. <br />                                                                As final step, output predicted complexes a <br />                                                                reasonably meaningful order biological significance. <br />   Combining criteria, assign non-core p an <br />                                                                For this, rank predicted complexes decreasing <br /> attachment acceptor cluster Ci, p &#195;&#381; Attach <br />                                                                order weighted densities. The weighted density <br /> (Ci), if: <br />  Srihari et al. BMC Bioinformatics 2010, 11:504                                                                                        Page 6 25 <br /> http://www.biomedcentral.com/1471-2105/11/504 <br />  <br />  <br />  <br />  <br /> WD(C &#226;&#8364;&#178; ) = <br />            &#226;&#710;&#8216;       p ,q&#226;&#710;&#710;C &#226;&#8364;&#178;i w(p, q) <br />                                      .   predicted complex <br />                                                                   catalogue [29], 313 complexes MIPS [30], 101 <br />                                                                   complexes curated Aloy et al. [31]. The properties of <br />                 &#226;&#8364;&#178; <br />               | C | &#226;&#8249;&#8230;(| C&#226;&#8364;&#178;i | &#226;&#710;&#8217;1)                               reference sets shown Table 2. We consid- <br />                                                                   ered reference sets independently the <br /> C&#226;&#8364;&#178;i given [16]: <br />                                                                   evaluation MCL-CAw. We did <span id='am-2' about='Thesaurus:Merge_Into' typeof='owl:Thing'>merge into</span> <br />                                                                   comprehensive list complexes indivi- <br />  <br />     WD(C &#226;&#8364;&#178; ) = <br />                    &#226;&#710;&#8216;    p ,q&#226;&#710;&#710;C &#226;&#8364;&#178;i w(p, q) <br />                                           .                 (4) <br />                                                                   dual complex compositions different the <br />                                                                   sources complexes double- <br />                        &#226;&#8364;&#178; <br />                    | C | &#226;&#8249;&#8230;(| C&#226;&#8364;&#178;i | &#226;&#710;&#8217;1) <br />                                                                   counted (because different names used same <br />   The unweighted density predicted complex                complex). An alternative strategy adopted Wang <br /> defined similar way setting weights            et al. [21] integrating complexes three <br /> constituent interactions 1. This blindly favors           sources (MIPS [30], SGD [33] in-house <br /> small complexes, complexes proteins having                curated complexes) using Jaccard score: com- <br /> large number interactions considering              plexes overlapping Jaccard score 0.7 <br /> reliability interactions. On hand,         merged - proteins included into <br /> weighted density considers reliability (by means           resultant complex chosen based voting <br /> affinity scores) interactions. If complexes           scheme. <br /> have unweighted density, complex                  To accurate (as fair) evaluating our <br /> higher weighted density ranked higher.                         method benchmark sets, considered only <br />                                                                   set derivable benchmark complexes of <br /> Results                                                           PPI networks: protein present PPI <br /> Preparation experimental data <br />                                                                   network, remove set benchmark com- <br /> We gathered high-confidence Gavin Krogan-Core                 plexes. By repeated removals, size benchmark <br /> interactions deposited BioGrid http://thebiogrid.org/          complex shrinks 3, remove complex from <br /> [32] (version July 2009). These assembled              benchmark set generate final set derivable <br /> from combination bait-prey prey-prey relation-           benchmark complexes PPI networks. <br /> ships (the spoke matrix models) observed Gavin               In order evaluate biological coherence our <br /> et al. [6], bait-prey relationships (the spoke            predicted complexes, downloaded list cellular <br /> model) observed Krogan et al. [7]. We combined                 localizations (GO terms &#226;&#8364;&#339;Cellular Component&#226;&#8364;?) of <br /> these interactions build unscored Gavin+Krogan             proteins Gene Ontology (GO) [34]. We selected <br /> network (all edge-weights set 1). We                 informative GO terms. A GO term informa- <br /> applied Iterative-CDk [15,16] FS Weightk [14] scor-           tive 30 proteins annotated this <br /> ing (with k = 2 iterations, recommended [16])           term descendant terms annotated to <br /> Gavin+Krogan network, selected interactions               30 proteins [35]. The list essential genes <br /> with non-zero scores. This resulted ICD(Gavin              obtained Saccharomyces Genome Deletion <br /> +Krogan) FSW(Gavin+Krogan) networks, respec-                  Project [36,37]: http://www-sequence.stanford.edu/ <br /> tively. In addition scored networks,              group/yeast_deletion_project/deletions3.html <br /> downloaded Consolidated3.19 network (with PE cut- <br />                                                                   Evaluation metrics matching predicted and <br /> off: 3.19, recommended Collins et al. [11]) from <br />                                                                   benchmark complexes <br /> http://interactome-cmp.ucsf.edu/, Bootstrap0.094 <br /> network [17] (with BT cut-off 0.094) http://www.             Let B = {B1,B2,...,Bm} C = {C1,C2,...,Cn} sets of <br /> bio.ifi.lmu.de/Complexes/ProCope/. The Consolidated               benchmark predicted complexes, respectively. We <br /> network derived matrix modeled relation-             use Jaccard coefficient J quantify overlap <br /> ships original Gavin Krogan datasets using             benchmark complex Bi predicted com- <br /> the PE [11]. Therefore, network comprised             plex Cj : <br /> of additional prey-prey interactions missed                                 | Bi &#195;&#8225; C j | <br /> the Krogan &#226;&#8364;&#732;Core&#226;&#8364;&#8482; dataset. The Bootstrap network                  J( B , C j ) =                  .                                         (5) <br /> derived matrix modeled relationships using                                  | Bi &#195;&#710; C j | <br /> bootstrapped scores [17]. Table 1 summarizes some <br /> properties networks.                                       We consider Bi covered Cj, J(Bi, Cj) &#226;&#8240;&#165; over- <br />   The benchmark (reference) set complexes built            lap threshold t. In experiments, set threshold <br /> from hand-curated complexes derived                                                                               | Bi | + | C j | <br />                                                                   t = 0.5, requires                   | Bi &#195;&#8225; C j | &#226;&#8240;&#165;                    .   For <br /> sources: 408 complexes Wodak lab CYC2008                                                                                     3 <br />  Srihari et al. BMC Bioinformatics 2010, 11:504                                                                               Page 7 25 <br /> http://www.biomedcentral.com/1471-2105/11/504 <br />  <br />  <br />  <br />  <br /> example, |Bi| = |Cj | = 8, overlap Bi                     parameter I MCL, and g core-attachment <br /> and Cj 6.                                                  refinement order determine good combination of <br />   We use previously reported [16] definitions recall                       parameters MCL-CAw practice. Only pre- <br /> Rc (coverage) precision Pr (sensitivity) set                    dicted complexes size &#226;&#8240;&#165; 4 MCL MCL-CAw <br /> predicted complexes:                                                          considered setting parameters for <br />                                                                               experiments. We used F1 (harmonic mean of <br />            |{Bi | Bi &#226;&#710;&#710; B &#226;&#710;&#167; &#226;&#710;&#402;C j &#226;&#710;&#710; C ; J(Bi , C j ) &#226;&#8240;&#165; t}|                      precision recall) measured Wodak lab <br />     Rc =                                                                (6) <br />                                    |B|                                        [29], MIPS [30] Aloy [31] benchmarks basis <br />                                                                               choosing best values parameters. <br />  Here, |{Bi|Bi &#195;&#381; B &#206;&#8250; &#226;&#710;&#402;Cj &#195;&#381; C; J(Bi, Cj) &#226;&#8240;&#165; t}| gives                         We adopted following four-step procedure for <br /> number derived benchmarks.                                                 PPI network: <br />  <br />            |{C j | C j &#226;&#710;&#710; C &#226;&#710;&#167; &#226;&#710;&#402;Bi &#226;&#710;&#710; B; J(Bi , C j ) &#226;&#8240;&#165; t}|                         1. Run MCL range I values choose I <br />     Pr =                                                                (7)      offers best F1 measure; <br />                                    |C | <br />                                                                                  2. Set I chosen value, set certain for <br />                                                                                  MCL-CAw, choose g range values that <br />    Here, |{Cj |Cj &#195;&#381; C &#206;&#8250; &#226;&#710;&#402;Bi &#195;&#381; B; J(Bi, Cj) &#226;&#8240;&#165; t}| gives the <br />                                                                                  offers best F1 measure; <br /> number matched predictions. <br />                                                                                  3. Set I g chosen values, choose for <br />    We evaluate performance method by <br />                                                                                  MCL-CAw range values offers the <br /> plotting precision versus recall curves pre- <br />                                                                                  best F1 measure; <br /> dicted complexes. These curves plotted tuning a <br />                                                                                  4. Set g MCL-CAw chosen values, <br /> threshold number predicted complexes con- <br />                                                                                  reconfirm value chosen I. <br /> sidered evaluation. The predicted complexes are <br />                                                                               Setting I MCL <br /> considered decreasing order weighted densi- <br />                                                                               Inflation I MCL determines granularity the <br /> ties (that is, increasing order complex ranks). <br />                                                                               clustering - higher value finer clus- <br />                                                                               ters produced. Typical values used clustering PPI <br /> Biological coherence predicted complexes <br />                                                                               networks I = 1.8 1.9 [16,19,38]. For PPI <br /> A complex formed proteins localized <br />                                                                               network, ran MCL range I , measured <br /> within compartment cell. So, use the <br />                                                                               F1 benchmark sets. We normal- <br /> localization coherence predicted complexes a <br />                                                                               ized F1 values best F1 obtained on <br /> measure quality. Let L = {L1, L2,..., Lk } set <br />                                                                               benchmark, summed normalized F1 <br /> of known localization groups, Li contains a <br />                                                                               values benchmarks, finally normalized these <br /> set proteins similar localization annotations. The <br />                                                                               sums obtain final ranking I values. The <br /> co-localization score LS(Cj) predicted complex Cj is <br />                                                                               detailed calculations presented Additional files 1, <br /> defined maximal fraction constituent pro- <br />                                                                               Tables S1 S2. In Figure 1, sample F1 ver- <br /> teins co-localized localization <br />                                                                               sus I plots unscored Gavin+Krogan scored <br /> group proteins annotations. This is <br />                                                                               ICD(Gavin+Krogan) networks range I = 1.25 <br /> given follows [16]: <br />                                                                               3.0. We noticed inflation I = 2.5 gave best F1 <br />                   max{| C j &#195;&#8225; Li |: = 1, 2, &#226;&#8364;&#166;, k}                           unscored scored networks. The F1 obtained <br />     LS (C j ) =                                          .              (8)   I = 1.8 1.9 marginally I <br />                   | p : p &#226;&#710;&#710; C j &#226;&#710;&#167; &#226;&#710;&#402;Li &#226;&#710;&#710; L, p &#226;&#710;&#710; Li | <br />                                                                               = 2.5. <br />                                                                               Setting g CA refinement <br />   Therefore, co-localization score LS(C) set <br />                                                                               For PPI network, set I chosen value, <br /> of predicted complexes C just weighted average <br />                                                                               fixed certain a, ran MCL-CAw range g. <br /> over complexes [16]: <br />                                                                               We adopted method choose the <br />                                                                               value g offering best F1 measure. Figure 2 shows <br />  <br />     LS(C) = <br />                &#226;&#710;&#8216;    C j &#226;&#710;&#710;C <br />                              max{| C j &#195;&#8225; L |: = 1, 2, &#226;&#8364;&#166; , k} <br />                                                                     .   (9) <br />                                                                               sample F1 versus g plots unscored Gavin+Krogan <br />                                                                               scored ICD(Gavin+Krogan) networks I = 2.5, a <br />                &#226;&#710;&#8216;     C j &#226;&#710;&#710;C <br />                               | p : p &#226;&#710;&#710; C j &#226;&#710;&#167; &#226;&#710;&#402;L &#226;&#710;&#710; L, p &#226;&#710;&#710; L | <br />                                                                               = 1.00 g = 0.15 1.50. The detailed calculations <br />                                                                               presented Additional files 1, Table S3. We noticed <br />                                                                               g = 0.75 gave best F1 unscored and <br /> Setting parameters I, g MCL-CAw                                 scored networks. <br /> Before evaluating performance MCL-CAw,                                Next, set I g chosen values, ran <br /> describe procedure used setting inflation                             MCL-CAw range a. Figure 3 shows sample F1 <br />  Srihari et al. BMC Bioinformatics 2010, 11:504                                                                               Page 8 25 <br /> http://www.biomedcentral.com/1471-2105/11/504 <br />  <br />  <br />  <br />  <br />  Figure 1 Setting inflation parameter I MCL: F1 versus I          Figure 2 Setting parameter g core-attachment <br />  plot. (a): Plot unscored Gavin+Krogan network; (b): Plot    refinement: F1 versus g plot. (a): Plot unscored Gavin <br />  scored ICD(Gavin+Krogan) network. I = 2.5 gave best F1      +Krogan network; (b): Plot scored ICD(Gavin+Krogan) <br />  unscored scored networks.                                      network. g = 0.75 gave best F1 unscored scored <br />                                                                          networks (I = 2.50 = 1.00). <br />  <br />  <br /> versus plots unscored Gavin+Krogan and <br /> scored ICD(Gavin+Krogan) networks I = 2.5, = g =                    g = 0.75 scored networks final <br /> 0.75 = 0.50 1.75. The detailed calculations                combination parameters MCL-CAw. <br /> presented Additional files 1, Table S4. We noticed <br /> that = 1.50 gave best F1 unscored network,                 Evaluating performance MCL-CAw <br /> while = 1.0 gave best F1 scored networks.                  Figure 5 shows workflow considered evalua- <br /> Reconfirming I chosen values g                         tion MCL-CAw. The predicted complexes were <br /> Finally, PPI network, ran core-attachment                   tapped successive stages: <br /> refinement chosen values g a <br /> range I MCL. Figure 4 compares F1 versus                        1. After clustering using MCL; <br /> I plots plain-MCL MCL followed CA                               2. After hierarchical clustering followed core- <br /> refinement unscored Gavin+Krogan                                attachment refinement using MCL-CAw. <br /> scored ICD(Gavin+Krogan) networks range I =                         The effect core-attachment refinement predictions <br /> 1.25 3.0. The plots reconfirmed chosen                      MCL <br /> values g gave best performance CA                     Compare topmost rows Table 3 MCL MCL- <br /> refinement I = 2.5 (except Aloy bench-                     CAw evaluated unscored Gavin+Krogan network. <br /> mark, smallest benchmark three,                       They MCL-CAw achieved significantly <br /> which F1 best I = 1.75 marginally                        higher recall compared MCL Gavin+Krogan - an <br /> lower I = 2.5). The detailed calculations pre-                  average 31% higher number complexes derived than <br /> sented Additional files 1, Tables S5 S6. We                      MCL. In fact referring Figure 4(a), MCL-CAw <br /> settled I = 2.5, = 1.50 g = 0.75                       achieved higher F1 compared MCL entire range <br /> unscored Gavin+Krogan network, I = 2.5, = 1.0                     I = 1.25 3.00. In order analyse this <br />  Srihari et al. BMC Bioinformatics 2010, 11:504                                                                      Page 9 25 <br /> http://www.biomedcentral.com/1471-2105/11/504 <br />  <br />  <br />  <br />  <br />                                                                     significantly higher number benchmark complexes than <br />                                                                     MCL. <br />                                                                     Impact noise MCL MCL-CAw role of <br />                                                                     affinity scoring reducing impact <br />                                                                     Table 3 compares different evaluation metrics MCL <br />                                                                     MCL-CAw unscored Gavin+Krogan the <br />                                                                     scored PPI networks. Very clearly, MCL and <br />                                                                     MCL-CAw showed considerable improvement preci- <br />                                                                     sion recall scored networks. For example, <br />                                                                     MCL achieved 127% higher precision 51.3% <br />                                                                     higher recall (on average), MCL-CAw achieved <br />                                                                     132% higher precision 26.6% higher recall (on <br />                                                                     average Wodak lab benchmark) scored <br />                                                                     networks unscored Gavin+Krogan network. <br />                                                                     The precision versus recall curves (Figure 6) Gavin <br />                                                                     +Krogan dropped sharply, three <br />                                                                     scored networks - ICD(Gavin+Krogan), FSW (Gavin <br />                                                                     +Krogan) Consolidated 3.19 - displayed more <br />                                                                     &#226;&#8364;&#339;graceful&#226;&#8364;? decline. The curve Bootstrap0.094 displayed <br />                                                                     sudden dip beginning, stabilized sub- <br />                                                                     sequently achieve higher (final) precision recall <br />                                                                     compared unscored Gavin+Krogan network. <br />                                                                       Among scored PPI networks, MCL and <br />                                                                     MCL-CAw showed best precision recall Con- <br />                                                                     solidated3.19 network, directly attributed to <br />                                                                     high quality network. However, high quality <br />                                                                     Consolidated3.19 came expense lower protein <br />  Figure 3 Setting parameter core-attachment                coverage (see Table 4; noted [20]), resulting in <br />  refinement: F1 versus plot. (a): Plot unscored Gavin     reduced number derivable complexes. In order coun- <br />  +Krogan network; (b): Plot scored ICD(Gavin+Krogan) <br />                                                                     ter this, gathered larger subset Consolidated <br />  network. = 1.50 gave best F1 unscored network (I = <br />  2.50 g = 0.75). = 1.00 gave best F1 scored       network PE cut-off 0.623 (the average PE score), <br />  networks (I = 2.50 g = 0.75)..                                 accounted higher protein coverage (Table 4). <br />                                                                     We noticed improvement MCL-CAw over <br />                                                                     MCL significantly higher Consolidated0.623, com- <br />                                                                     pared improvement seen Consolidated3.19. We <br /> improvement, considered sets complexes                    noticed ICD scoring Consolidated0.623 drasti- <br /> derived Gavin+Krogan. (a) Set A = MCL &#226;&#710;&#169; MCL-                   cally reduced size network, revealing this <br /> CAw, consisting complexes correctly predicted             larger subset fact included significant false <br /> both MCL MCL- CAw, different Jaccard                   positives (noise). These experiments indicate rea- <br /> accuracies; (b) Set B = MCL-CAw\MCL, consisting              sonably good algorithm like MCL perform on <br /> complexes correctly predicted MCL-CAw,                high quality networks. However, lack protein <br /> MCL. There complex correctly predicted MCL                coverage scarcity high quality networks, <br /> that missed MCL-CAw. We calculated increase              need consider larger networks complex detection <br /> (percentage) accuracies complexes A B.              (particularly able detect novel complexes). This in <br /> This increase A noticably high, average           turn exposes algorithms higher natural <br /> 7.53% Wodak set. The increase B signifi-             noise (even scored networks). Therefore, need to <br /> cantly high, average 62.26% Wodak set.             develop algorithms detect larger number com- <br /> This shows: (a) CA-refinement successful improv-             plexes presence noise. In scenario, our <br /> ing accuracies MCL clusters; (b) This improvement            results MCL-CAw able derive considerably <br /> was particularly high low quality clusters MCL (that         higher number complexes MCL. Taking this <br /> is, set B). MCL-CAw successful elevating                 further, introduced different levels random noise to <br /> accuracies threshold t = 0.5 clusters           study impact MCL MCL-CAw. We introduced <br /> that difficult matched known complexes                10% 75% random noise (2000 10000 random interac- <br /> using MCL alone. Consequently, MCL-CAw derived                      tions) Gavin+Krogan network. We noticed that <br />  Srihari et al. BMC Bioinformatics 2010, 11:504                                                                                  Page 10 25 <br /> http://www.biomedcentral.com/1471-2105/11/504 <br />  <br />  <br />  <br />  <br />  Figure 4 Reconfirming inflation I MCL-Caw. (a): Plot unscored Gavin+Krogan network = 1.50 g = 0.75. (b): Plot the <br />  scored ICD(Gavin+Krogan) network = 1.00 g = 0.75. I = 2.5 gave best F1 chosen values g (except Aloy <br />  benchmark, smallest benchmark three, I = 1.75 gave marginally better F1). <br />  <br />  <br /> MCL-CAw performed better MCL intro-                       range 10% - 40% noise, covers typical levels of <br /> ducing 50% random noise (Table 5). However, 75% ran-                  noise seen TAP/MS datasets [9] (we say keeping in <br /> dom noise, performance MCL-CAw marginally                         mind MCL shown robust 80% <br /> dropped MCL. Therefore, MCL-CAw                        random noise [38]). We scored noisy networks <br /> reasonably robust random noise - stable                 using ICD scheme. We performance of <br />  <br />  <br />  <br />  <br />  Figure 5 Workflow evaluation MCL-Caw. The predicted complexes MCL-CAw tapped stages: (i) Clustering using MCL; <br />  (ii) Hierarchical clustering followed core-attachment refinement using MCL-CAw. These predicted complexes evaluated matching <br />  set benchmark complexes. <br />  Srihari et al. BMC Bioinformatics 2010, 11:504                                                                                            Page 11 25 <br /> http://www.biomedcentral.com/1471-2105/11/504 <br />  <br />  <br />  <br />  <br /> Table 3 (i) Impact core-attachment refinement MCL; (ii) Role affinity scoring reducing impact natural <br /> noise MCL MCL-CAw <br /> Benchmark Method            PPI             #Predicted              #Matched          Precision         #Derivable              #Derived              Recall <br />                           Network           complexes               prediction                         benchmarks              benchmarks <br />   Wodak         MCL          G+K                242                     55              0.226               182                      62               0.338 <br />                           ICD(G+K)              136                     68              0.500               153                      76               0.497 <br />                           FSW(G+K)              120                     69              0.575               153                      78               0.510 <br />                           Consol3.19            116                     70              0.603               145                      79               0.545 <br />                           Boot0.094             203                     76              0.374               172                      85               0.494 <br />                 MCL-         G+K                310                     77              0.248               182                      77               0.423 <br />                 CAw <br />                           ICD(G+K)              129                     80              0.620               153                      80               0.523 <br />                           FSW(G+K)              117                     72              0.615               153                      83               0.542 <br />                           Consol3.19            122                     82              0.672               145                      82               0.566 <br />                            Boot0.094            199                     79              0.397               172                      88               0.512 <br />    MIPS         MCL          G+K                242                     35              0.143               177                      40               0.226 <br />                           ICD(G+K)              136                     47              0.346               151                      60               0.397 <br />                           FSW(G+K)              120                     46              0.383               151                      61               0.404 <br />                           Consol3.19            116                     48              0.414               157                      63               0.401 <br />                            Boot0.094            203                     44              0.271               168                      56               0.333 <br />                 MCL-         G+K                310                     53              0.171               177                      53               0.300 <br />                 CAw <br />                           ICD(G+K)              129                     63              0.488               151                      63               0.417 <br />                           FSW(G+K)              117                     48              0.410               151                      66               0.437 <br />                           Consol3.19            122                     68              0.557               157                      68               0.433 <br />                            Boot0.094            199                     47              0.236               168                      59               0.351 <br />    Aloy         MCL          G+K                242                     43              0.179               76                       42               0.556 <br />                           ICD(G+K)              136                     58              0.426               75                       56               0.747 <br />                           FSW(G+K)              120                     57              0.475               75                       57               0.760 <br />                           Consol3.19            116                     54              0.466               76                       55               0.724 <br />                            Boot0.094            203                     56              0.276               76                       55               0.724 <br />                 MCL-         G+K                310                     52              0.168               76                       52               0.684 <br />                 CAw <br />                           ICD(G+K)              129                     59              0.457               75                       59               0.787 <br />                           FSW(G+K)              117                     60              0.513               75                       60               0.800 <br />                           Consol3.19            122                     57              0.467               76                       57               0.750 <br />                            Boot0.094            199                     57              0.286               76                       58               0.763 <br /> Affinity scoring PPI networks improved performance MCL MCL-CAw. Affinity scoring followed CA refinement compounded effect in <br /> improving performance MCL. <br />  <br />  <br /> both MCL MCL-CAw improved considerably                                  MCL-CAw displayed higher scores compared com- <br /> these scored networks. MCL-CAw performed considerably                          plexes, indicating proteins cores highly <br /> better MCL 50% 75% random noise (Table                         localized; (c) The complexes MCL MCL- <br /> 5). Therefore, affinity scoring helped MCL-CAw main-                        CAw displayed higher scores scored net- <br /> tain performance gain MCL.                                            works compared Gavin+Krogan network. <br /> Biological coherence predicted components <br /> The co-localization scores various predicted                           Relative ranking complex prediction algorithms and <br /> components (cores complexes) MCL-CAw                              affinity-scored networks <br /> are shown Table 6. The table shows that: (a) The pre-                       In order gauge performance MCL-CAw rela- <br /> dicted complexes MCL-CAw showed high co-locali-                             tive existing techniques, selected following <br /> zation scores compared MCL unscored                             recent algorithms proposed complex detection: <br /> and scored PPI networks. MCL included noisy <br /> proteins predicted clusters, reducing                             &#226;&#8364;&#162; On unscored Gavin+Krogan network, com- <br /> their biological coherence; (b) The predicted cores                             pared MCL [18,19], preliminary work <br />  Srihari et al. BMC Bioinformatics 2010, 11:504                                                                       Page 12 25 <br /> http://www.biomedcentral.com/1471-2105/11/504 <br />  <br />  <br />  <br />  <br />                                                                          Tables 8, 9, 10, 11 12 detailed comparisons <br />                                                                       complex detection algorithms unscored <br />                                                                       scored networks. Figures 7 8 precision <br />                                                                       versus recall curves networks, Table 13 <br />                                                                       shows area-under-the-curve (AUC) values these <br />                                                                       curves. Considering &#194;&#177; 5% error AUC values, table <br />                                                                       shows CORE attained highest AUC followed by <br />                                                                       MCL-CAw CMC unscored network, while <br />                                                                       MCL-CAw CMC achieved overall highest AUC <br />                                                                       scored networks. In addition this, net- <br />                                                                       work ranked algorithms based normal- <br />                                                                       ized final F1 measures (with respect best <br />                                                                       performing algorithm network), shown in <br />                                                                       Table 14. We summed normalized F1 values for <br />                                                                       algorithm networks obtain an <br />                                                                       overall ranking algorithms shown Table 15. <br />                                                                       The detailed calculations presented Additional <br />                                                                       files 1, Table S7. On unscored network CMC <br />                                                                       showed best F1 value, scored networks <br />                                                                       MCL-CAw showed best overall F1 value. In particu- <br />                                                                       lar, MCL-CAw performed best ICD(Gavin+Kro- <br />                                                                       gan), FSW(Gavin+Krogan) Consolidated 3.19 <br />                                                                       networks, HACO performed best Boot- <br />                                                                       strap0.094. This agreed relative per- <br />                                                                       formance gathered AUC values (Table 13). <br />                                                                          The precision MCL-CAw (0.397) lower on <br />  Figure 6 Impact affinity scoring performance MCL        Bootstrap0.094 compared scored networks (ICD <br />  MCL-Caw. (a). (b): Precision versus recall curves Gavin   - 0.620, FSW - 0.615, Consolidated3.19 - 0.672). MCL- <br />  +Krogan scored networks (ICD(Gavin+Krogan), FSW         CAw produced redundant complexes this <br />  (Gavin+Krogan), Consolidated3.19 Bootstrap0.094) MCL     network compared scored networks, leading to <br />  MCL-CAw, respectively, evaluated Wodak benchmark t = <br />                                                                       drop precision. In fact observed variance <br />  0.5. Both methods showed significant improvement the <br />  scored networks compared unscored Gavin+Krogan.                   CMC HACO algorithms well. For example, <br />                                                                       CMC achieved best recall ICD network, but <br />                                                                       lowest Consolidated network. Also, CMC pro- <br />     MCL-CA (2009) [28], CORE Leung et al. (2009)                   duced significantly fewer complexes (#77) Conso- <br />     [24], COACH Wu Min et al. (2009) [25], CMC                     lidated network compared networks (ICD - 171, <br />     Liu et al. (2009) [16], HACO Wang et al.                FSW - 179, Bootstrap - 203). Further, algorithms dis- <br />     (2009) [21];                                                      played &#226;&#8364;&#339;sudden dips&#226;&#8364;? precision versus recall curves <br />     &#226;&#8364;&#162; On affinity-scored networks, compared                    beginning Bootstrap0.094 network <br />     MCL, MCL incorporated cluster over-                  (see Figure 8). All findings indicate choice <br />     laps Pu et al. (2007) [20] (our implementation              affinity scoring schemes affected performance of <br />     this, called MCLO), CMC HACO.                                 algorithms. In words, algorithm use of <br />                                                                       certain characteristics PPI networks, favored <br />   Table 7 summarizes properties                   scoring scheme magnified reinforced those <br /> parameters used methods. We consider                   characteristics. There single algorithm which <br /> complexes size 4 algorithms              performed relatively best scored networks. <br /> entire evaluation. We dropped MCL-CA, CORE                        Having said that, note MCL-CAw ranked among <br /> COACH comparisons affinity-scored net-                 algorithms scored networks, and <br /> works methods assume unweighted net-                    MCL-CAw responded reasonably the <br /> works inputs. Further, results                  considered affinity scoring schemes. <br /> older methods MCODE Bader Hogue                            We ranked different affinity-scored networks <br /> (2003) [8] RNSC King et al. (2004) [39], instead               based F1 measures offered complex detec- <br /> include MCL comparisons, MCL                     tion algorithms, shown Tables 16 17. The <br /> significantly outperforms methods [16,38].                      table shows Consolidated3.19 network offered <br />  Srihari et al. BMC Bioinformatics 2010, 11:504                                                                                                  Page 13 25 <br /> http://www.biomedcentral.com/1471-2105/11/504 <br />  <br />  <br />  <br />  <br /> Table 4 MCL-CAw performed considerably better MCL presence natural noise <br />     PPI Network                #Proteins         #Interactions          Avg node deg               #Derived MCL            complexes (Recall) MCL-CAw <br />       Consol3.19                 1622                   9704                 11.96                   79 (0.545)                       82 (0.566) <br />      Consol0.623                 5423               102393                   37.76                   74 (0.321)                       84 (0.375) <br />     ICD(Cons3.19)                1161                   8688                 14.96                   58 (0.408)                       63 (0.443) <br />     ICD(Cons0.623)               1273               19996                    31.41                   52 (0.353)                       56 (0.381) <br />     FSW(Cons3.19)                1123                   8694                 15.48                   59 (0.401)                       65 (0.442) <br />    FSW(Cons0.623)                1341               20696                    30.87                   54 (0.360)                       57 (0.380) <br /> The Consolidated3.19 Consolidated0.623 networks subsets Consolidated network [11] derived PE cut-offs 3.19 0.623, respectively. We ran <br /> ICD FSW schemes networks. Consolidated0.623 significant false positives (about 81%) discarded scoring. The <br /> performance MCL-CAw marginally better MCL Consolidated3.19, MCL-CAw performed considerably better MCL &#226;&#8364;&#339;more noisy&#226;&#8364;? <br /> Consolidated0.623. <br />  <br />  <br /> the best F1 measures algorithms, followed                           orthologous proteins organisms like fly, mouse <br /> FSW(Gavin+Krogan), ICD(Gavin+Krogan) Boot-                                    human), based genetic (gene fusion, <br /> strap0.094 networks (the detailed calculations pre-                           chromosomal proximity, gene co-evolution) func- <br /> sented Additional files 2, Table S8). This agreed                         tional (traits neighbors, neighbors neighbors, etc.) <br /> with fact Consolidated 3.19 network                              associations; downloaded Predictome database <br /> shown TP/FP ratio comparable small-scale                             [40]http://cagt.bu.edu/page/Predictome_about. These <br /> experiments MIPS, high                             used generate Inferred network (Table 1). <br /> quality [11].                                                                     We augmented Gavin+Krogan network with <br />                                                                                   interactions generate Gavin+Krogan <br /> Impact augmenting physical PPI networks                                   +Inferred network scored versions, ICD <br /> computationally inferred interactions                                             (Gavin+Krogan+Inferred) FSW(Gavin+Krogan <br /> In set experiments, studied augment-                           +Inferred) networks (Table 1). <br /> ing physical PPI networks inferred interactions                             We evaluated MCL, MCL-CAw, CMC HACO on <br /> improved performance complex detection algo-                               augmented networks (Table 18). All algo- <br /> rithms. We gathered interactions yeast comprising                           rithms displayed low precision recall values on <br /> inferred interlogs (inferred interactions                            Inferred network, indicating inferred <br />  <br /> Table 5 (i) Impact introducing different levels artificial noise MCL MCL-CAw (ii) Role affinity scoring in <br /> reducing impact noise <br />  Method              PPI                   #Predicted            #Matched             Precisions          #Derivable                 #Derived            Recall <br />                    Network                 complexes             predictions                             benchmarks                 benchmarks <br />    MCL                G+K                     242                      55                0.226                182                        62              0.338 <br />                G+K+Rand2k                     265                      56                0.215                182                        64              0.352 <br />                G+K+Rand5k                     274                      61                0.223                182                        68              0.379 <br />                G+K+Rand10k                    316                      64                0.202                182                        69              0.379 <br />                     ICD(G+K)                  119                      73                0.613                153                        73              0.477 <br />              ICD(G+K+Rand2k)                  104                      59                0.567                153                        66              0.431 <br />              ICD(G+K+Rand5k)                  108                      60                0.546                151                        65              0.430 <br />                     ICD(G+K                   112                      60                0.546                150                        65              0.433 <br />                    +Rand10k) <br /> MCL-CAw           G+K                         310                      77                0.248                182                        77              0.423 <br />                G+K+Rand2k                     140                      59                0.421                182                        68              0.374 <br />                G+K+Rand5k                     116                      62                0.534                182                        70              0.384 <br />                G+K+Rand10k                    176                      64                0.363                182                        68              0.373 <br />                     ICD(G+K)                  129                      80                0.620                153                        80              0.523 <br />              ICD(G+K+Rand2k)                  102                      62                0.608                153                        73              0.477 <br />              ICD(G+K+Rand5k)                  102                      64                0.627                151                        76              0.503 <br />                     ICD(G+K                   106                      64                0.603                150                        76              0.506 <br />                    +Rand10k) <br /> The Gavin+Krogan network introduced 2000 - 10000 (10% 75%) random interactions. Following this, noisy networks scored using ICD <br /> scheme. With aid scoring, MCL-CAw able perform better MCL 50% random noise. <br />  Srihari et al. BMC Bioinformatics 2010, 11:504                                                                                                   Page 14 25 <br /> http://www.biomedcentral.com/1471-2105/11/504 <br />  <br />  <br />  <br />  <br /> Table 6 Co-localization scores predicted components                                   Cytoscape visualization environment http://www. <br /> from MCL MCL-CAw                                                                      cytoscape.org/[41]. <br />                                           Co-localization scores                          Instances correctly predicted complexes MCL-CAw <br />   PPI Network               MCL              MCL-CAw              MCL-CAw                 The example attachment protein shared <br />                           clusters            cores              complexes                predicted complexes MCL-CAw. The <br />   Gavin+Krogan             0.730                0.890                 0.866               subunits predicted complexes (Id# 57 22) <br />     ICD(Gavin              0.830                0.936                 0.912               make Compass complex involved telomeric <br />      +Krogan) <br />                                                                                           silencing gene expression [42], mRNA clea- <br />    FSW(Gavin               0.830                0.931                 0.912 <br />     +Krogan) <br />                                                                                           vage polyadenylation specificity factor, complex <br />  Consolidated3.19          0.790                0.923                 0.908 <br />                                                                                           involved RNAP II transcription termination [43]. The <br />   Bootstrap0.094           0.788                0.895                 0.874 <br />                                                                                           shared attachment Swd2 (Ykl018w) formed high confi- <br />                                                                                           dence connections subunits predicted <br /> Findings: (i) The complexes produced CA-refinement showed higher <br /> scores MCL; (ii) The complexes predicted scored                    complexes. On basis, post-processing procedure <br /> networks showed higher scores Gavin+Krogan network; (iii) The               assigned Swd2 (Ykl018w) predicted complexes, <br /> cores MCL-CAw showed higher scores complexes. <br />                                                                                           agreement available evidence [44] Swd2 <br />                                                                                           (Ykl018w) belongs Compass mRNA clea- <br /> interactions sufficient predict mean-                                   vage complexes. The example illustrates case <br /> ingful complexes. Interestingly, algorithms dis-                                     new protein predicted subunit a <br /> played marginal dip performance Gavin                                         known complex. The attachment protein Ski7 (Yor076c) <br /> +Krogan+Inferred compared Gavin+Krogan. This dip                                       included predicted complex (Id# 28) that <br /> in performance explained analysis                                       matched Exosome complex involved RNA proces- <br /> two augmented-scored networks, ICD(Gavin+Krogan                                           sing degradation [45]. Additionally, Ski7 (Yor076c) <br /> +Inferred) FSW(Gavin+Krogan+Inferred). Most                                           included prediction (Id# 105) matching <br /> algorithms showed higher precision recall                                    Ski complex (Additional files 1, Figure S2). However, <br /> two augmented-scored networks compared Gavin                                           Ski complex Wodak lab catalogue [29] did <br /> +Krogan Gavin+Krogan+Inferred. This indicates                                         include new protein. Further literature survey <br /> that augmenting raw inferred interactions gave lit-                                  suggested Ski7 acts mediator Ski <br /> tle benefit presence false positives (noise),                               Exosome complexes 3&#226;&#8364;&#8482;-to-5&#226;&#8364;&#8482; mRNA decay in <br /> scoring augmented networks helped improve                                      yeast [46]. <br /> precision recall values algorithms.                                              The RNA polymerase I, II, III complexes (also <br />                                                                                           called Pol I, II, III, respectively) required the <br /> In-depth analysis individual predicted complexes                                       generation RNA chains [47]. As Wodak lab <br /> To facilitate analysis individual predicted                                    catalogue [29], complexes share subunits: <br /> complexes, mapped complexes corre-                                     Yor224c, Ybr154c, Yor210w Ypr187w, Pol I <br /> sponding PPI networks examined interactions                                       Pol III share Ynl113w Ypr110c. Due the <br /> between components complex,                                        extensive sharing subunits, corresponding predic- <br /> between components given complex pro-                                      tions grouped large cluster by <br /> teins network. We performed analysis using                                    MCL. On hand, MCL-CAw segregated the <br />  <br />  <br /> Table 7 Existing complex detection methods selected comparisons MCL-CAw <br />                                                                                            Method <br />   Property          MCL               MCL-CA                  MCLO                 CORE               COACH                   CMC                  HACO <br />   Principle        Flow           Core-attach            MCL cluster       Core-attach      Core-attach          Maximal clique           Hier agglo <br />                 simulation     refinement              overlaps            p-values          dense                 merging               cluster with <br />                                      MCL                                                         neighborhood                                      overlaps <br />   Scored            Yes                   No                    Yes                 No                   No                    Yes                   Yes <br />  Networks <br /> Unassigned          No                   Yes                    No                  Yes                 Yes                    Yes                   Yes <br />  Proteins <br /> Parameters      Inflation I           Inflation I       Inflation I , Overlap        /                  Filter t    Merge m, Overlap t, Min       UPGMA <br />  (default)       (I = 2.5)             (I = 2.5)         a, b (2.5, 1.0, 0.5)                        (t = 0.225)     clust size (0.5, 0.25, 4)   cutoff (0.2) <br /> References      Dongen               Srihari et al.          Pu et al.          Leung et al.        Wu Min et al.           Liu et al.           Wang et al. <br />                 2000 [18]             2009 [28]              2007 [20]           2009 [24]           2009 [25]              2009 [16]             2009 [21] <br /> CORE (2009), COACH (2009), MCL-CA (2009) compared MCL-CAw unscored Gavin+Krogan network, MCL (2000, 2002), MCLO <br /> (2007), CMC (2009) HACO (2009) evaluated scored networks. <br />  Srihari et al. BMC Bioinformatics 2010, 11:504                                                                                                       Page 15 25 <br /> http://www.biomedcentral.com/1471-2105/11/504 <br />  <br />  <br />  <br />  <br /> Table 8 Comparisons different methods                                   Table 10 Comparisons different methods on <br /> the unscored Gavin+Krogan network                                                      FSW(Gavin+Krogan) network <br />                                                  Method                                                                                Method <br />                         MCL MCL- MCL- COACH CORE CMC HACO                                                          MCL     MCLO       MCL-CAw       CMC      HACO <br />                              CA CAw <br />                                                                                                   #Predicted       120       108         117         176       99 <br />           #Predicted    242        219     310    447      386        113      278 <br />                                                                                        Wodak      #Matched          69       61           72         76        68 <br /> Wodak #Matched           55         49      77     62       83        60          78   (#153) <br /> (#182)                                                                                            Precision       0.575     0.564       0.615       0.432     0.687 <br />           Precision     0.226 0.224 0.248         0.139    0.215 0.531 0.281                      #Derived         78        72           83          84        77 <br />           #Derived       62    49    77            49       83    60     85                       Recall          0.510     0.471       0.542       0.549     0.503 <br />           Recall        0.338 0.269 0.423         0.269    0.456 0.330 0.467 <br />                                                                                          MIPS     #Matched          46       42           48         49        42 <br />  MIPS #Matched           35         42      53     45       59        41          45    (#151) <br /> (#177)                                                                                            Precision       0.383     0.388       0.410       0.278     0.424 <br />           Precision     0.143 0.192 0.171         0.101    0.153 0.363 0.162                      #Derived          61       55           66         65        56 <br />           #Derived       40         42      53     38       59        41          57              Recall          0.404     0.364       0.437       0.430     0.371 <br />           Recall        0.226 0.237 0.300         0.215    0.333 0.232 0.322 <br />                                                                                         Aloy      #Matched          57       56           60         59        53 <br />  Aloy     #Matched       43         41      52     54       59        43          59    (#75) <br />  (#76)                                                                                            Precision       0.475     0.518       0.513       0.335     0.535 <br />           Precision     0.179 0.187 0.168         0.121    0.153 0.381 0.212                      #Derived          57       56           60         57        53 <br />           #Derived       42         41      52     37       59        43          59              Recall          0.760     0.747       0.800       0.760     0.707 <br />           Recall        0.556 0.539 0.684         0.487    0.776 0.566 0.776 <br />                                                                                        Methods considered: MCL, MCLO, MCL-CAw, CMC HACO. HACO <br /> Methods considered: MCL, MCL-CA, MCL-CAw, COACH, CORE, CMC HACO.                   performed best terms precision, MCL-CAw CMC performed <br /> CMC performed best terms precision, HACO CORE                      best terms recall. MCL-CAw close second terms precision. <br /> performed best terms recall. MCL-CAw stood                #Matched: #Predictions matching benchmark complex(es). #Derived: <br /> seven algorithms precision recall. #Matched: #Predictions                  #Benchmark complexes derived predicted complex(es). <br /> matching benchmark complex(es). #Derived: #Benchmark complexes                    The FSW(Gavin+Krogan) network. <br /> derived predicted complex(es).                                                 #Proteins 1628; #Interactions 8688. <br /> The unscored Gavin+Krogan network. <br /> #Proteins 2964; #Interactions 13507. <br />  <br />  <br />  <br /> Table 9 Comparisons different methods on <br /> the ICD(Gavin+Krogan) network                                                          Table 11 Comparisons different methods on <br />                                                                                        Consolidated3.19 network <br />                                                  Method <br />                            MCL           MCLO    MCL-CAw     CMC           HACO                                                        Method <br />                                                                                                                    MCL     MCLO       MCL-CAw       CMC      HACO <br />            #Predicted         136        121       129        171           104 <br />                                                                                                   #Predicted       116       119         122         77       101 <br /> Wodak      #Matched           68          73        80           86          68 <br /> (#153)                                                                                 Wodak      #Matched          70       80           82         67        57 <br />            Precision       0.500         0.603    0.620      0.503          0.654      (#145) <br />            #Derived           76          73        80           86          76                   Precision       0.603     0.672       0.672       0.870     0.564 <br />            Recall          0.497         0.477    0.523      0.562          0.497                 #Derived          79       80           82         67        64 <br />                                                                                                   Recall          0.545     0.552       0.566       0.462     0.441 <br />   MIPS&#226;&#8364;&#8482;    #Matched           47          56        63           65          41 <br />  (#151)                                                                                  MIPS     #Matched          48       65           68         56        40 <br />            Precision       0.346         0.463    0.488      0.380          0.394       (#157) <br />            #Derived           60          56        63           65          55                   Precision       0.414     0.546       0.557       0.727     0.396 <br />            Recall          0.397         0.371    0.417      0.430          0.364                 #Derived          63       65           68         56        57 <br />                                                                                                   Recall          0.401     0.414       0.433       0.357     0.363 <br />  Aloy      #Matched           58          56        59           59          53 <br />  (#75)                                                                                  Aloy      #Matched          54       56           57         45        44 <br />            Precision       0.426         0.463    0.457      0.345          0.510       (#76) <br />            #Derived         56            56       59          59             53                  Precision       0.466     0.471       0.467       0.584     0.436 <br />            Recall          0.747         0.747    0.787      0.787          0.707                 #Derived          55       56           57         45        45 <br />                                                                                                   Recall          0.724     0.737       0.750       0.592     0.592 <br /> Methods considered: MCL, MCLO, MCL-CAw, CMC HACO. HACO <br /> performed best terms precision, CMC performed best              Methods considered: MCL, MCLO, MCL-CAw, CMC HACO. CMC performed <br /> terms recall. MCL-CAw close second precision recall.              best terms precision, MCL-CAw performed best recall. <br /> #Matched: #Predictions matching benchmark complex(es). #Derived:                  #Matched: #Predictions matching benchmark complex(es). #Derived: <br /> #Benchmark complexes derived predicted complex(es).                            #Benchmark complexes derived predicted complex(es). <br /> The ICD(Gavin+Krogan) network.                                                         The Consolidated3.19 network. <br /> #Proteins 1628; #Interactions 8707.                                                    #Proteins 1622; #Interactions 9704. <br />  Srihari et al. BMC Bioinformatics 2010, 11:504                                                                                Page 16 25 <br /> http://www.biomedcentral.com/1471-2105/11/504 <br />  <br />  <br />  <br />  <br /> Table 12 Comparisons different methods                          Instances depicting mistakes predictions MCL-CAw <br /> the Bootstrap0.094 network                                                     Here discuss interesting case sharing <br />                                                 Method                         subunits extensive web interactions <br />                             MCL     MCLO       MCL-CAw       CMC      HACO     dense separating smaller subsumed <br />            #Predicted       203       204         199         203      127     complexes purely basis interaction informa- <br /> Wodak      #Matched          76       76           79         110       80     tion harder. It amalgamation the <br /> (#172)                                                                         clusters matching SAGA, SAGA-like (SLIK), ADA <br />            Precision       0.374     0.372       0.397       0.542     0.630   TFIID complexes. Based Wodak lab catalogue <br />            #Derived         85        85          88          106        90    [29], 20 subunits making SAGA complex <br />            Recall          0.494     0.494       0.512       0.616     0.523   involved transcriptional regulation [50] include four <br />   MIPS     #Matched          44       45           47          67       49     subunits (Ygr252w, Ydr176w, Ydr448w, Ypl254w) that <br />  (#168)                                                                        members ADA complex [51] well. Sixteen <br />            Precision       0.271     0.220       0.236       0.330     0.386   components SAGA complex including four <br />            #Derived          56       57           59          69       63     shared ADA complex, components <br />            Recall          0.333     0.339       0.351       0.411     0.375   SLIK complex [52]. Additionally, subunits <br />  Aloy      #Matched          56       55           57          76       59     (Ybr198c, Ygl112c, Ymr236w, Ydr167w, Ydr145w) the <br />  (#76)                                                                         SAGA complex belong TFIID complex [50]. <br />            Precision       0.276     0.269       0.286       0.374     0.465   Because extensive sharing subunits involved in <br />            #Derived          55       55           58          63       60     dense web interactions (436 interactions 31 <br />            Recall          0.724     0.723       0.763       0.829     0.789   constituent proteins, seen ICD(Gavin+Krogan) <br /> Methods considered: MCL, MCLO, MCL-CAw, CMC HACO. HACO                     network), MCL-CAw able segregate two <br /> performed best terms precision, CMC performed best in <br /> terms recall. MCL-CAw positioned precision recall.    distinct complexes - SAGA (0.708) SLIK (0.625). The <br /> #Matched: #Predictions matching benchmark complex(es). #Derived:          clusters matching TFIID ADA remained amalga- <br /> #Benchmark complexes derived predicted complex(es). <br />                                                                                mated together. In set analysis, compared <br /> The Bootstrap0.094 network. <br /> #Proteins 2719; #Interactions 10290. <br />                                                                                derived complexes Gavin+Krogan the <br />                                                                                ICD(Gavin+Krogan) networks, identified cases where <br />                                                                                MCL-CAw missed proteins com- <br /> large cluster independent complexes,                          plexes affinity scoring. From Wodak, MIPS <br /> matched Pol I, Pol II Pol III complexes                           Aloy reference sets, 13, 18 16 com- <br /> accuracies 0.714, 0.734 0.824, respectively.                            plexes, respectively, derived better accura- <br />   In addition cases, good fraction                       cies Gavin+Krogan network ICD <br /> known core-attachment structures (reported                              (Gavin+Krogan) network. And, 6, 2 2 <br /> supplementary materials Gavin et al. [6]) con-                         complexes, respectively, derived the <br /> firmed, putative complexes identified (pre-                           Gavin+Krogan network, missed totally ICD <br /> paration compendium currently progress).                               (Gavin+Krogan) network. Table 19 shows sample of <br /> Some examples worth quoting here. Our predicted                            complexes Wodak reference set. For the <br /> complex id# 44 closely matched HOPS complex.                               complexes derived lower accuracies <br /> All cores {Ylr148w, Ylr396c, Ymr231w, Ypl045w,                            (upper half Table 19), MCL-CAw missed few <br /> Yal002w} attachments {Ydr080w, Ydl077c}                                proteins low scores assigned corresponding <br /> that covered matched reported Gavin et                           interactions. For example, predicted complex from <br /> al. Biological experiments cores                        ICD(Gavin+Krogan) network matching SWI/SNF <br /> function vacuole protein sorting, help                         complex, proteins (Ymr033w Ypr034w) of <br /> of attachments, complex perform homotypic                              missed ones absent weak con- <br /> vacuole fusion [48]. We identified ubiquitin ligase                        nections rest members; instead, pro- <br /> ERAD-L complex comprising Yos9(Ydr057w), Hrd3                               teins present prediction matching RSC <br /> (Ylr207w), Usa1 (Yml029w) Hrd1 (Yol013c)                           complex. In Gavin+Krogan network, pro- <br /> involved degradation ER proteins [49]. This                          teins shared complexes matching the <br /> matched Hrd1/Hrd3 purified Gavin et al. Four                            SWI/SNF RSC complexes, agreed with <br /> subunits {Oca4, Oca5, Siw14, Oca1} predicted                              Wodak catalogue [29]. <br /> novel complex (Id# 66) showed high similarity func-                           In cases MCL-CAw completely missed <br /> tions (oxidant-induced cell-cycle arrest) localiza-                        complexes scored network (lower half of <br /> tion (cytoplasmic) verified SGD [33]. This                             Table 19), interesting note MCL-CAw had <br /> complex exactly matched putative complex 490                            pulled-in additional (noisy) proteins attachments <br /> Gavin et al.                                                                   predicted complexes, caused the <br />  Srihari et al. BMC Bioinformatics 2010, 11:504                                                                                  Page 17 25 <br /> http://www.biomedcentral.com/1471-2105/11/504 <br />  <br />  <br />  <br />  <br />  Figure 7 Comparison complex detection algorithms unscored Gavin+Krogan network. (a): Precision versus recall curves area- <br />  under-the-curve (AUC) values complex detection algorithms unscored Gavin+Krogan network, evaluated Wodak reference t = <br />  0.5. AUC MCL = 0.225, COACH = 0.169, CORE = 0.361, MCL-CAw = 0.323, CMC = 0.271, MCL-CA = 0.238, HACO = 0.136. (b): Number of <br />  predicted complexes, proportion true positives (correctly matched benchmark(s)) false positives (not matched benchmark) for <br />  algorithms. <br />  <br />  <br /> accuracies drop 0.5. One case pre-               lower accuracy 0.4. All core proteins or <br /> dicted complex id#36 matching eIF3 complex                    similar GO annotations (involvement translation, loca- <br /> low Jaccard score 0.4. The eIF3 complex Wodak                    lized cytoplasm ribosomal subunit) [34]. Upon ana- <br /> lab consisted 7 proteins: Yor361c, Ylr192c, Ybr079c,                  lysing GO annotations 8 attachment proteins, <br /> Ymr309c, Ydr429c, Ymr012w Ymr146c. The pre-                          noticed (Ymr146c) anno- <br /> dicted complex id#66 Gavin+Krogan network                       tation core proteins. This eIF3 <br /> consisted 8 proteins (Figure 9): 5 cores (Yor361c,                    complex Wodak lab [29]. Out remaining 7 <br /> Ylr192c, Ybr079c, Ymr309c, Ydr429c) 3 attachments                    attachment proteins, (Ypr041w, Ynl244c, Yml063w, <br /> (Yor096w, Yal035w, Ydr091c). Therefore, 2                     Yjl190c, Ydr091c) related GO annotations (transla- <br /> missed 3 additional proteins prediction, lead-                tion initiation, GTPase activity, cytoplasmic, ribosomal <br /> ing accuracy 0.5. The predicted complex id#36                   subunit) core proteins. A literature search revealed <br /> from ICD(Gavin+Krogan) network consisted 14                       proteins belonged multi-eIF initiation <br /> proteins: 6 cores (Yor361c, Ylr192c, Ybr079c, Ymr309c,                   factor conglomerate (containing eIF1, eIF2, eIF3 and <br /> Ydr429c, Yor096w) 8 attachments (Yal035w,                            eIF5) 40 S ribosomal subunit involved transla- <br /> Ydr091c, Yjl190c, Yml063w, Ymr146c, Ynl244c,                             tion [29]. The remaining (Yal035w, Yor204w) were <br /> Yor204w, Ypr041w). Therefore, 1 missed                    involved translation activity, absent the <br /> 8 additional proteins prediction, leading              Wodak lab catalogue. These potentially new <br />  Srihari et al. BMC Bioinformatics 2010, 11:504                                                                               Page 18 25 <br /> http://www.biomedcentral.com/1471-2105/11/504 <br />  <br />  <br />  <br />  <br />   Figure 8 Comparison complex detection algorithms scored networks. Precision versus recall curves area-under-the-curve <br />   (AUC) values complex detection algorithms evaluated Wodak reference t = 0.5. (a) ICD(Gavin+Krogan): AUC values MCL = 0.436, <br />   CMC = 0.494, MCL-CAw = 0.472, MCLO = 0.435, HACO = 0.305. (b) FSW(Gavin+Krogan): AUC values MCL = 0.431, CMC = 0.481, MCL-CAw = <br />   0.487, MCLO = 0.430, HACO = 0.461. (c) Consolidated3.19: AUC values MCL = 0.469, CMC = 0.399, MCL-CAw = 0.488, MCLO = 0.463, HACO = <br />   0.367. (d) Bootstrap0.094: AUC values MCL = 0.349, CMC = 0.513, MCL-CAw = 0.389, MCLO = 0.353, HACO = 0.317. <br />  <br />  <br />  <br /> proteins belonging eIF3 related complexes,                  example illustrates carefully incorporating GO infor- <br /> need investigated. We analysed GO                  mation algorithm include filter pro- <br /> annotations level-1 neighbors predicted                     teins useful cases making decisions <br /> complex seen network, annota-                     solely based interaction information difficult. <br /> tions similar proteins network. This <br />                                                                           Correlation essentiality proteins their <br />                                                                           ability form complexes <br /> Table 13 Area curve (AUC) values precision <br />                                                                           Early works Jeong et al. [53] Han et al. [54] stu- <br /> versus recall curves complex detection methods on <br /> the unscored scored PPI networks                                      died essentialities proteins based pairwise inter- <br />                                                                           actions interaction network, concluded <br /> PPI           MCL MCLO         MCL-      CMC HACO COACH CORE <br /> network                        CAw                                        hub (high-degree) proteins likely be <br /> G+K           0.225            0.323     0.271   0.136    0.169   0.361   essential. This formed criteria &#226;&#8364;&#339;cen- <br />                                                                           trality-lethality&#226;&#8364;? rule [53]. However, deeper insight can <br /> ICD(G+K)      0.436 0.435      0.472     0.494 0.305 <br />                                                                           obtained studying essentialities cluster or <br /> FSW(G+K)      0.431 0.430      0.487     0.481   0.461 <br />                                                                           group level proteins pairwise interactions. <br /> Consol3.19    0.469 0.463      0.488     0.399   0.367 <br />                                                                           Recently, Zotenko et al. [55] argued essential pro- <br /> Boot0.094     0.349 0.353      0.389     0.513 0.317 <br />                                                                           teins group densely connected sets of <br /> Considering &#194;&#177; 5% error AUC values, CORE showed highest value on <br /> unscored network, MCL-CAw CMC showed overall highest on <br />                                                                           proteins performing essential functions, get <br /> the scored networks.                                                      involved higher number interactions resulting in <br />  Srihari et al. BMC Bioinformatics 2010, 11:504                                                                                              Page 19 25 <br /> http://www.biomedcentral.com/1471-2105/11/504 <br />  <br />  <br />  <br />  <br /> Table 14 Relative ranking complex detection algorithms unscored affinity-scored networks <br /> PPI network                                           Wodak                       MIPS                         Aloy <br />                       Method                   F1             Norm         F1            Norm           F1            Norm          Total       Norm <br /> G+K                   CMC                     0.407           1.000       0.283          1.000         0.455          1.000         3.000        1.000 <br />                       HACO                    0.351           0.862       0.216          0.761         0.333          0.731         2.355        0.785 <br />                       MCL-CAw                 0.313           0.768       0.218          0.770         0.270          0.592         2.130        0.710 <br />                       CORE                    0.292           0.718       0.210          0.741         0.256          0.561         2.020        0.673 <br />                       MCL                     0.271           0.665       0.175          0.619         0.271          0.595         1.879        0.626 <br />                       MCL-CA                  0.244           0.601       0.212          0.749         0.278          0.610         1.960        0.653 <br />                       COACH                   0.183           0.450       0.137          0.486         0.194          0.426         1.361        0.454 <br /> ICD(G+K)              MCL-CAw                 0.567           1.000       0.450          1.000         0.578          0.976         2.976        1.000 <br />                       HACO                    0.565           0.995       0.378          0.841         0.593          1.000         2.837        0.953 <br />                       MCLO                    0.533           0.939       0.412          0.916         0.572          0.965         2.820        0.947 <br />                       CMC                     0.531           0.936       0.403          0.897         0.480          0.810         2.642        0.888 <br />                       MCL                     0.498           0.879       0.370          0.822         0.543          0.916         2.616        0.879 <br /> FSW(G+K)              MCL-CAw                 0.576           0.992       0.423          1.000         0.625          1.000         2.992        1.000 <br />                       HACO                    0.581           1.000       0.396          0.935         0.609          0.974         2.910        0.972 <br />                       MCL                     0.541           0.931       0.393          0.929         0.585          0.935         2.795        0.934 <br />                       MCLO                    0.513           0.884       0.376          0.888         0.612          0.979         2.750        0.919 <br />                       CMC                     0.484           0.833       0.338          0.798         0.465          0.744         2.375        0.794 <br /> Cons3.19              MCL-CAw                 0.614           1.000       0.487          1.000         0.576          0.979         2.979        1.000 <br />                       MCLO                    0.606           0.986       0.471          0.967         0.575          0.977         2.930        0.984 <br />                       CMC                     0.604           0.982       0.479          0.983         0.588          1.000         2.965        0.995 <br />                       MCL                     0.573           0.932       0.407          0.836         0.567          0.964         2.732        0.917 <br />                       HACO                    0.475           0.774       0.379          0.777         0.502          0.854         2.405        0.807 <br /> Boot0.094             HACO                    0.572           0.991       0.380          1.000         0.585          1.000         2.991        1.000 <br />                       CMC                     0.577           1.000       0.367          0.965         0.515          0.881         2.846        0.952 <br />                       MCL-CAw                 0.447           0.776       0.282          0.742         0.416          0.711         2.229        0.745 <br />                       MCL                     0.426           0.738       0.299          0.785         0.400          0.683         2.207        0.738 <br />                       MCLO                    0.424           0.736       0.267          0.701         0.392          0.670         2.108        0.705 <br /> Ranking obtained normalized F1 values. The G+K network unscored, remaining scored using affinity scoring schemes. <br />  <br />  <br />  <br />  <br /> Table 15 Overall relative ranking complex detection                            hubness property. Therefore, hubness just an <br /> algorithms unscored affinity-scored networks                               indirect indicator protein essentiality. More recently, <br /> Category          Method         Relative score         Normalized score          Kang et al. [56] studied essentiality proteins gener- <br /> Unscored            CMC               3.000                     1.000             ating reverse neighbor (RNN) topology [57] of <br />                    HACO               2.355                     0.785             protein networks. This topology groups proteins <br />                  MCL-CAw              2.130                     0.710             reverse neighborhood a <br />                   CORE                2.020                     0.673             given protein. Kang et al. concluded centrality within <br />                     MCL               1.879                     0.626             RNN topology better estimator essentiality <br />                   MCL-CA              1.960                     0.653             hubness degree interaction network. Stu- <br />                   COACH               1.361                     0.454             dies Hart et al. [12] showed essential proteins are <br /> Scored           MCL-CAw              3.745                     1.000             concentrated certain complexes, resulting a <br />                    HACO               3.733                     0.997             dichotomy essential non-essential complexes. <br />                    CMC                3.628                     0.969             Wang et al. [21] concluded size (largest) <br />                    MCLO               3.555                     0.949             recruiting complex protein better indicator <br />                     MCL               3.468                     0.926             protein essentiality hubness. <br /> Ranking obtained normalized F1 values. <br />  Srihari et al. BMC Bioinformatics 2010, 11:504                                                                                    Page 20 25 <br /> http://www.biomedcentral.com/1471-2105/11/504 <br />  <br />  <br />  <br />  <br /> Table 16 Relative ranking affinity scoring schemes complex detection <br /> PPI network                                           Wodak                       MIPS                     Aloy <br />                      Method                    F1             Norm         F1            Norm       F1            Norm    Total       Norm <br /> MCL                  Cons3.19                 0.573           1.000       0.407          1.000     0.567          0.970   2.970        1.000 <br />                      FSW(G+K)                 0.541           0.944       0.393          0.965     0.585          1.000   2.909        0.980 <br />                      ICD(G+K)                 0.498           0.871       0.370          0.908     0.543          0.928   2.706        0.911 <br />                      Boot0.094                0.426           0.744       0.299          0.733     0.400          0.684   2.161        0.728 <br /> MCLO                 Cons3.19                 0.606           1.000       0.471          1.000     0.575          0.939   2.939        1.000 <br />                      ICD(G+K)                 0.533           0.879       0.412          0.875     0.572          0.934   2.688        0.914 <br />                      FSW(G+K)                 0.513           0.847       0.376          0.798     0.612          1.000   2.645        0.900 <br />                      Boot0.094                0.424           0.700       0.267          0.567     0.392          0.641   1.908        0.649 <br /> MCL-CAw              Cons3.19                 0.614           1.000       0.487          1.000     0.576          0.921   2.921        1.000 <br />                      FSW(G+K)                 0.576           0.938       0.423          0.868     0.625          1.000   2.806        0.961 <br />                      ICD(G+K)                 0.567           0.923       0.450          0.923     0.578          0.925   2.771        0.949 <br />                      Boot0.094                0.447           0.728       0.282          0.579     0.416          0.666   1.973        0.675 <br /> CMC                  Cons3.19                 0.604           1.000       0.479          1.000     0.588          1.000   3.000        1.000 <br />                      Boot0.094                0.577           0.955       0.366          0.764     0.515          0.877   2.597        0.866 <br />                      ICD(G+K)                 0.531           0.880       0.403          0.843     0.480          0.816   2.538        0.846 <br />                      FSW(G+K)                 0.484           0.801       0.338          0.705     0.465          0.791   2.297        0.766 <br /> HACO                 FSW(G+K)                 0.581           1.000       0.396          1.000     0.609          1.000   3.000        1.000 <br />                      Boot0.094                0.572           0.984       0.380          0.961     0.585          0.961   2.906        0.969 <br />                      ICD(G+K)                 0.565           0.972       0.378          0.956     0.593          0.973   2.902        0.967 <br />                      Cons3.19                 0.495           0.852       0.379          0.957     0.502          0.824   2.634        0.878 <br /> Ranking obtained normalized F1 values. <br />  <br />  <br />   In work, attempt understand relation-                             10(b) shows essential proteins present higher <br /> ship essentiality proteins ability                       proportions higher ranked complexes. Both these <br /> to form complexes. Table 20 shows high propor-                             figures hint finding: essential proteins come <br /> tion (77.65%, 78.03%, 81.34% 76.35% ICD                              large groups perform essential functions. <br /> (Gavin+Krogan), FSW (Gavin+Krogan), Consolidated3.19 <br /> and Bootstrap0.094 networks, respectively) essential                           Discussion <br /> proteins present affinity-scored networks                             In spite advances computational approaches to <br /> belonged correctly predicted complex.                            derive complexes, high-accuracy reconstruction com- <br /> This indicated essential proteins members                          plexes remained challenging task. In deriving <br /> of complexes co-clustered groups proteins.                                  protein complexes PPI networks, key assumption <br />   To analyse ability essential proteins                        computational approaches com- <br /> form complexes groups, binned correctly pre-                            plexes form densely connected regions net- <br /> dicted complexes based sizes calculated                          works. Therefore, approaches attempt cluster <br /> proportion essential proteins complexes                        networks based measures related connectiv- <br /> bin (like [21]). Figure 10(a) shows essential pro-                        ities proteins network. Some approaches <br /> teins present higher proportions larger                            like MCL simulate random walks (called flow) iden- <br /> complexes. We calculated proportion essen-                            tify dense regions, like CMC merge maxi- <br /> tial proteins K ranked complexes. Figure                           mal cliques larger dense clusters. Therefore, the <br />                                                                                   performance methods varies widely depending <br />                                                                                   network densities. A glance Tables 8 12 <br /> Table 17 Overall relative ranking affinity scoring                             reveals methods considered comparison <br /> schemes complex detection                                                     work achieve low recall MIPS set <br /> Scored network              Relative score             Normalized score           compared Wodak Aloy sets. Table 2 shows <br /> Cons3.19                         4.878                        1.000               average density complexes MIPS much <br /> FSW(G+K)                         4.606                        0.944               lower Wodak Aloy sets. Only 52 of <br /> ICD(G+K)                         4.588                        0.941               137 (37.95%) derivable MIPS complexes size &#226;&#8240;&#165; 5 <br /> Boot0.094                        3.886                        0.797               detected Gavin+Krogan network by <br /> Ranking obtained normalized F1 values.                                       methods together. We analysed remaining 85 <br />  Srihari et al. BMC Bioinformatics 2010, 11:504                                                                                                  Page 21 25 <br /> http://www.biomedcentral.com/1471-2105/11/504 <br />  <br />  <br />  <br />  <br /> Table 18 Impact augmenting inferred interactions performance MCL, MCL-CAw, CMC HACO <br /> Method        PPI Network #Predicted complexes #Matched predictions Precision #Derivable benchmarks #Derived benchmarks Recall <br />   MCL            G+K                   242                         55                   0.226               182                           62              0.338 <br />                    I                    50                          2                   0.040               31                              3             0.097 <br />                 G+K+I                  249                         55                   0.221               189                           58              0.307 <br />               ICD(G+K+I)               115                         53                   0.461               156                           58              0.372 <br />               FSW(G+K+I)                89                         54                   0.607               141                           61              0.433 <br /> MCL-Caw          G+K                   310                         77                   0.248               182                           77              0.423 <br />                    I                    42                          2                   0.048               31                              3             0.097 <br />                 G+K+I                  315                         78                   0.247               189                           78              0.412 <br />               ICD(G+K+I)               118                         82                   0.694               156                           82              0.525 <br />               FSW(G+K+I)                95                         84                   0.884               141                           84              0.596 <br />   CMC            G+K                   113                         60                   0.531               182                           60              0.330 <br />                    I                    10                          3                   0.300               31                              5             0.161 <br />                 G+K+I                  119                         60                   0.504               189                           63              0.333 <br />               ICD(G+K+I)               184                         77                   0.418               156                           83              0.532 <br />               FSW(G+K+I)               186                         74                   0.398               141                           80              0.567 <br />  HACO            G+K                   278                         78                   0.281               182                           85              0.467 <br />                   I                     12                          2                   0.167                31                            2              0.064 <br />                 G+K+I                  309                         78                   0.252               189                           84              0.444 <br />               ICD(G+K+I)               119                         66                   0.589               156                           75              0.481 <br />               FSW(G+K+I)                98                         61                   0.622               141                           70              0.496 <br /> Most algorithms showed marginal dip performance Gavin+Krogan+Inferred compared Gavin+Krogan. However, scoring augmented network, <br /> their performance better compared Gavin+Krogan. This indicated inferred interactions useful complex detection provided affinity scoring is <br /> employed reduce impact noise present them. <br />  <br />  <br /> MIPS complexes                                comprising 17491 interactions 4932 proteins <br /> low densities (average 0.217) Gavin+Krogan                           giving low average node degree 7.092. MCL-CAw <br /> network. For example, MIPS complex 440.30.10                                  able achieve marginal improvement (22.8% <br /> (involved mRNA splicing) went undetected                            higher precision 7.4% higher recall) MCL, due <br /> methods 40 42 proteins present                            low average node degree DIP network. <br /> in Gavin+Krogan network. There 144 interac-                              These experiments methods consid- <br /> tions 40 proteins, giving low density                            ered difficult uncover complexes are <br /> 0.184 complex network. Continuing                             sparse. This prompt rethink whether <br /> this analysis, tested MCL MCL-CAw PPI                                 importance given model complexes as <br /> dataset DIP http://dip.doe-mbi.ucla.edu,                                     dense regions PPI networks. <br />  <br />  <br /> Table 19 Complexes derived lesser accuracy missed MCL-CAw affinity scoring <br /> Matched benchmark complex            #Proteins complexes        #Incorrect proteins complexes ICD(G+K)           Accuracies (Jaccard scores) <br /> Name                   #Proteins        G+K            ICD(G+K)                  Missed                       Addnl                 G+K           ICD(G+K) <br /> Kornbergs SRB              25            24                23                       2                             0                 0.960           0.920 <br /> SWI/SNF                    12            11                8                        4                             0                 0.769           0.667 <br /> TRAPP                      10            10                9                        1                             0                 1.000           0.900 <br /> 19/22 S reg                22            20                27                       0                             5                 0.909           0.815 <br /> TRAMP                      3              4                7                        0                             4                 0.750           0.429 <br /> Alpha-1,6                  5              9                11                       0                             6                 0.556           0.455 <br /> eIF3                       7              8                14                       1                             8                 0.500           0.400 <br /> Protein phosp              3              5                9                        0                             4                 0.600           0.333 <br /> Cdc73p/Paf1p               7              7                18                       0                             11                0.556           0.388 <br /> Chs5p/Arf-1                6              8                10                       2                             6                 0.556           0.400 <br /> The upper half shows sample complexes Wodak lab derived lower accuracies ICD(Gavin+Krogan) network compared Gavin <br /> +Krogan network. The lower half shows missed ICD(Gavin+Krogan) network. The #Incorrect proteins ICD(Gavin+Krogan) network respect <br /> to benchmark complexes. <br />  Srihari et al. BMC Bioinformatics 2010, 11:504                                                                                                  Page 22 25 <br /> http://www.biomedcentral.com/1471-2105/11/504 <br />  <br />  <br />  <br />  <br />   Figure 9 Example complex missed MCL-CAw ICD(Gavin+Krogan) network, Gavin+Krogan <br />   network. The eIF3 complex Wodak lab consisted 7 proteins: Yor361c, Ylr192c, Ybr079c, Ymr309c, Ydr429c, Ymr012w Ymr146c. The <br />   predicted complex id#36 ICD(Gavin+Krogan) network consisted 14 proteins: 6 cores (Yor361c, Ylr192c, Ybr079c, Ymr309c, Ydr429c, <br />   Yor096w) 8 attachments (Yal035w, Ydr091c, Yjl190c, Yml063w, Ymr146c, Ynl244c, Yor204w, Ypr041w). Therefore, 1 missed 8 <br />   additional proteins prediction, leading low accuracy 0.4. Hexagonal (Orange): eIF3 complex Wodak lab. Circle (Orange, Yellow <br />   Pink): Predicted complex id#36. Rectangle (Turquoise): Level-1 neighbors predicted complex id#36. <br />  <br />  <br />   Apart limitations existing compu-                              temporal spatial variability complexes, <br /> tational methods, inherent difficulties                             repeated purifications using TAP/MS methods yield <br /> in accumulation interactome data                               somewhat different &#226;&#8364;&#339;complex forms&#226;&#8364;? [20]. The PPI net- <br /> make complex detection difficult. Complexes display                                works constructed purifications represent <br /> different lifetimes, compositions vary based                             probabilistic average picture yeast inter- <br /> on cellular localizations (compartments) condi-                                actome [20]. Therefore, complexes predicted of <br /> tions. The protein recruited different                              networks approximate actual complex <br /> complexes different times conditions. Due                                compositions. <br />  <br />  <br /> Table 20 Essential genes predicted complexes MCL-CAw <br />                                                                     Number (Proportion) Essential genes present in <br />         PPI Network                  Whole network                Predicted cores              Predicted complexes                 Matched predictions <br />      ICD(Gavin+Krogan)                  604 (0.537)                  510 (0.454)                      552 (0.491)                        469 (0.417) <br />     FSW(Gavin+Krogan)                   604 (0.537)                  510 (0.454)                      552 (0.491)                        470 (0.418) <br />       Consolidated3.19                  611 (0.544)                  568 (0.506)                      576 (0.513)                        497 (0.442) <br />        Bootstrap0.094                   757 (0.674)                  634 (0.564)                      676 (0.601)                        578 (0.514) <br /> The figures brackets represent proportion essential genes present corresponding group 1123 total essential genes obtained the <br /> Yeast Genome Deletion project [36,37]. <br /> # Essential genes YDP: 1123 <br />  Srihari et al. BMC Bioinformatics 2010, 11:504                                                                                        Page 23 25 <br /> http://www.biomedcentral.com/1471-2105/11/504 <br />  <br />  <br />  <br />  <br />                                                                           attachment based refinement method coupled MCL to <br />                                                                           identify yeast complexes weighted PPI networks. <br />                                                                           We demonstrated algorithm (MCL-CAw) per- <br />                                                                           formed better MCL deriving meaningful yeast <br />                                                                           complexes particularly presence natural noise. <br />                                                                           We showed MCL-CAw responded reasonably <br />                                                                           considered affinity scoring schemes. In the <br />                                                                           future work, intend improve prediction ability <br />                                                                           algorithm incorporating information gene <br />                                                                           annotations, gene expressions, literature mining well <br />                                                                           domain-domain interactions. We intend extend <br />                                                                           work predict complexes organisms than <br />                                                                           yeast. In context, intend use MCL-CAw <br />                                                                           model study existence (and extent) core-attach- <br />                                                                           ment modularity complexes organisms. <br />  <br />                                                                           Availability <br />                                                                           The MCL-CAw software developed using PL/SQL on <br />                                                                           Oracle 10 g, using framework [58]. The source <br />                                                                           <span id='am-4' about='Thesaurus:code' typeof='owl:Thing'>code</span>, yeast PPI datasets, benchmark predicted yeast <br />                                                                           complexes used work freely available the <br />                                                                           MCL-CAw project homepage hosted NUS server: <br />                                                                           http://www.comp.nus.edu.sg/~leonghw/MCL-CAw/. <br />  <br />                                                                           Additional material <br />  <br />  Figure 10 Correlation essentiality proteins                Additional files 1: Additional figures tables: Figures core- <br />  abilities form complexes. (a): Proportion essential           attachment modularity illustration predicted complex MCL- <br />  proteins complexes different sizes, predicted ICD          CAw. Tables setting MCL-CAw parameters, ranking complex <br />  (Gavin+Krogan). Proportion essential proteins complex =           detection algorithms affinity-scored networks. <br />  #essential proteins/total #proteins complex. (b): Proportion    Additional files 2: The MCL-CAw software package: The source <span id='am-6' about='Thesaurus:code' typeof='owl:Thing'>code</span> <br />  essential proteins K ranked complexes.                         installation details MCL-CAw software. <br />  <br />  <br />  <br />   Another limitation arises bias TAP/MS <br />                                                                           Acknowledgements <br /> purifications complexes certain kind (for exam-                We like thank editor reviewers valuable <br /> ple, membrane-bound complexes). Since TAP/MS data                         comments suggestions; Sean Collins (UCSF) Caroline Friedel (LMU) <br /> are acquired single condition (rich media),                     making available Consolidated [11] Bootstrap [17] networks, <br />                                                                           respectively; Guimei Liu (NUS) Limsoon Wong (NUS) Iterative-CD, <br /> complexes present cell condi-                   FS Weight CMC softwares [14-16]; Henry Leung (HKU) Wu Min <br /> tion [21]. Therefore, new experimental assays needed                  (NTU) CORE [24] COACH [25] softwares, respectively. This work <br /> before complexes reconstructed studied.                   supported National University Singapore ARF <br />                                                                           grant R252-000-361-112. <br />   Finally, S. cerevisiae used model <br /> organism eukaryotic interactome analysis, key                    Author details <br />                                                                           1 <br /> complexes specialized organisms (including                        Department Computer Science, National University Singapore, 117590, <br />                                                                           Singapore. 2Department Pathology, University Michigan, Ann Arbor, MI <br /> human) studied analysing interaction                   48109, USA. 3Qingdao Institute Bioenergy Bioprocess Technology, <br /> datasets specific organisms. However,                        Qingdao 266101, China. <br /> incompleteness interactome data organ- <br />                                                                           Authors&#226;&#8364;&#8482; contributions <br /> isms makes reconstruction complexes difficult.                     SS conceived initial ideas discussed HWL KN. SS <br />                                                                           devised algorithm, developed software, performed experiments <br /> Conclusion                                                                analysis, wrote revised manuscript. HWL supervised the <br />                                                                           project, advised SS, reviewed revised manuscript. KN took part <br /> The ultimate goal interactome analysis under-                    discussions helped reviewing manuscript. All authors have <br /> stand higher level organization cell. Recon-                   read approved manuscript. <br /> struction protein complexes serves building block <br />                                                                           Received: 3 May 2010 Accepted: 12 October 2010 <br /> towards achieving goal. In paper, inspired               Published: 12 October 2010 <br /> findings Gavin et al. [6], developed novel core- <br />  Srihari et al. BMC Bioinformatics 2010, 11:504                                                                                                       Page 24 25 <br /> http://www.biomedcentral.com/1471-2105/11/504 <br />  <br />  <br />  <br />  <br /> References                                                                         16. Liu G, Wong L, Chua HN: Complex discovery weighted PPI <br /> 1. Ito T, Chiba T, Ozawa R, Yoshida M, Hattori M, Sakaki Y: A comprehensive            networks. Bioinformatics 2009, 25:1891-1897. <br />     two-hybrid analysis explore yeast protein interactome. Proc Natl        17. Friedel C, Krumsiek J, Zimmer R: Bootstrapping interactome: <br />     Acad Sci 2001, 98:4569-4574.                                                       unsupervised identification protein complexes yeast. Research in <br /> 2. Uetz P, Giot L, Cagney G, Traci A, Judson R, Knight J, Lockshon D,                  Computational Molecular Biology (RECOMB) 2008, 3-16. <br />     Narayan V, Srinivasan M, Pochart P, Emil QA, Li Y, Godwin B, Conover D,        18. Dongen S: Graph clustering flow simulation. PhD thesis University of <br />     Kalbfleisch T, Vijayadamodar G, Yang M, Johnston M, Fields S, Rothberg M:          Utrecht 2000. <br />     A comprehensive analysis protein-protein interactions                    19. Enright AJ, Dongen S, Ouzounis CA: An efficient algorithm large-scale <br />     Saccharomyces cerevisiae. Nature 2000, 403:623-627.                                detection protein families. Nucleic Acids Res 2002, 30(7):1575-1584. <br /> 3. Rigaut G, Shevchenko A, Rutz B, Wilm M, Mann M, Seraphin B: A generic           20. Pu S, Vlasblom J, Emili A, Greenbalt J, Wodak S: Identifying functional <br />     protein purification method protein complex characterization               modules physical interactome Saccharomyces cerevisiae. <br />     proteome exploration. Nature Biotechnol 1999, 17:1030-1032.                        Proteomics 2007, 7:944-960. <br /> 4. Gavin AC, Bosche M, Krause R, Grandi P, Marzioch M, Bauer A, Schultz J,         21. Wang H, kakaradov B, Collins SR, Karotki L, Fiedler D, Shales M, Shokat KM, <br />     Rick JM, Michon AM, Cruciat CM, Remor M, Hofert C, Schelder M,                     Walter T, Krogan NJ, Koller D: A complex-based reconstruction the <br />     Brajenovic M, Ruffner H, Merino A, Klien K, Hudak M, Dickson D, Rudi T,            Saccharomyces cerevisiae interactome. Mol Cell Proteomics 2009, <br />     Gnau V, Bauch A, Bastuck S, Huhse B, Leutwin C, Heurtier MA, Copley RR,            8:1361-1377. <br />     Edelmann A, Rybin V, Drewes G, Raida M, Bouwmeester T, Bork P,                 22. Fridel C, Zimmer R: Identifying topology protein complexes from <br />     Sepharin B, Kuster B, Neubauer G, Furga GS: Functional organization         affinity purification assays. Systems Biology 2009, 25:2140-2146. <br />     yeast proteome systematic analysis protein complexes. Nature             23. Voevodski K, Yu X: Spectral affinity protein networks. BMC Systems <br />     2002, 415:141-147.                                                                 Biology 2009, 3:112. <br /> 5. Ho Y, Gruhler A, Heilbut A, Bader G, Moore L, Adams SL, Millar A, Taylor P,     24. Leung H, Xiang Q, Yiu SM, Chin FY: Predicting protein complexes from <br />     Bennet K, Boutlier K, Yang L, Wolting C, Donaldson I, Schandorff S,                PPI data: core-attachment approach. Journal Comp Biology 2009, <br />     Shewnarane J, Vo M, Taggart M, Gouderault M, Muskat B, Alfarano C,                 16:133-44. <br />     Dewar D, Lin Z, Michalickova K, Willems AR, Sassi H, Nielson P,                25. Wu M, Li X, Ng SK: A core-attachment based method detect protein <br />     Rasmussen K, Anderson J, Johansen L, Hansen L, Jesperson H,                        complexes PPI networks. BMC Bioinformatics 2009, 10:169. <br />     Podtelejnikov A, Nielson E, Crawford J, Poulsen V, Sorensen B, Matthiesen J,   26. Mitrofanova A, Farach-Colton M, Mishra B: Efficient robust prediction <br />     Hendrickson RC, Gleeson F, Pawson T, Moran MF, Durocher D, Mann M,                 algorithms protein complexes using Gomory-Hu trees. Pacific <br />     Hogue CWV, Figeys D, Tyers M: Systematic identification protein                 Symposium Biocomputing (PSB) 2009, 215-226. <br />     complexes Saccharomyces cerevisiae mass spectrometry. Nature             27. Ozawa Y, Saito R, Fujimori S, Kashima H, Ishizaka M, Yanagawa H, <br />     2002, 415:180-183.                                                                 Miyamoto-Sato E, Tomita M: Protein complex prediction verifying and <br /> 6. Gavin AC, Aloy P, Grandi P, Krause R, Boesche M, Marzioch M, Rau C,                 reconstructing topology domain-domain interactions. BMC <br />     Jensen LJ, Bastuck S, Dumpelfeld B, Edelmann A, Heurtier MA, Hoffman V,            Bioinformatics 2010, 11:350. <br />     Hoefert C, Klein K, Hudak M, Michon AM, Schelder M, Schirle M, Remor M,        28. Srihari S, Ning K, Leong HW: Refining Markov clustering protein <br />     Rudi T, Hooper S, Bauer A, Bouwmeester T, Casari G, Drewes G,                      complex detection incorporating core-attachment structure. Genome <br />     Neubauer G, Rick JM, Kuster B, Bork P, Russel PB, Superti FG: Proteome             Informatics 2009, 23:159-168. <br />     survey reveals modularity yeast cell machinery. Nature 2006,            29. Pu S, Wong J, Turner B, Cho E, Wodak S: Up-to-date catalogues yeast <br />     440:631-636.                                                                       protein complexes. Nucleic Acids Res 2009, 37(3):825-831. <br /> 7. Krogan NJ, Cagney G, Yu H, Zhong G, Guo X, Ignatchenko A, Li J, Pu S,           30. Mewes HW, Amid C, Arnold R, Frishman D, Guldener U, Mannhaupt G, <br />     Datta N, Tikuisis A, Punna T, Alverez JM, Shales M, Zhang X, Davey M,              Munsterkotter M, Pagel P, Strack N, Stumpflen V, Warfsmann J, Ruepp A: <br />     Robinson M, Paccanaro A, Bray JE, Sheung A, Beattie B, Richards D,                 MIPS: analysis annotation proteins genomes. Nucleic <br />     Canadien V, Lalev A, Mena F, Wong P, Sharostine A, Canette M, Vlasblom J,          Acids Res 2006, 34:D169-D172. <br />     Wu S, Orsi C, Collins SR, Chandran S, Haw R, Rilstone J, Gandi K,              31. Aloy P, Bottcher B, Ceulemans H, Mellwig C, Fischer S, Gavin AC, Bork P, <br />     Thompson NJ, Musso G, Onge PS, Ghanny S, Lam M, Butland G, Altaf-Ul A,             Superti-Furga G, Serrano L, Russell RB: Structure-based assembly protein <br />     Kanaya S, Shilatifard A, Weissman J, Ingles J, Hughes TR, Parkinson J,             complexes yeast. Science 2004, 303:2026-2029. <br />     Gerstein M, Wodak S, Emili A, Greenblatt JF: Global landscape protein       32. Breitkreutz B, Stark C, Tyers M: The GRID: The General Repository for <br />     complexes yeast Saccharomyces cerevisiae. Nature 2006,                      Interaction Datasets. Genome Biology 2003, 4(3):R23. <br />     440:637-643.                                                                   33. Cherry JM, Adler C, Chervitz SA, Dwight SS, Jia Y, Juvik G, Roe T, <br /> 8. Bader GD, Hogue CWV: Analyzing yeast protein-protein interaction data               Schroeder M, Weng S, Botstein D: SGD: Saccharomyces Genome <br />     obtained different sources. Nature Biotechnology 2002, 20:991-997.            Database. Nucleic Acids Res 1998, 26:73-79. <br /> 9. von Mering C, Krause R, Snel B, Cornell M, Oliver SG, Fields S, Bork P:         34. Ashburner M, Ball CA, Blake JA, Botstein D, Butler H, Cherry M, Davis AP, <br />     Comparative assessment large-scale datasets protein-protein                  Dolinski K, Dwight SS, Epigg J, Harris MA, Hill DP, Issel-Tarver L, Kasarkis A, <br />     interactions. Nature 2002, 417:399-403.                                            Lewis S, Matase JC, Richardson J, Ringwald M, Rubin GM, Sherlock G: Gene <br /> 10. Batada N, Hurst LD, Tyers M: Evolutionary physiological importance             ontology: tool unification biology. Nature Genetics 2000, <br />     hub proteins. PLoS Comp Bio 2006, 2:e88.                                        25:25-29. <br /> 11. Collins SR, Kemmeren P, Zhao XC, Greenbalt JF, Spencer F, Holstege F,          35. Zhou X, Kao MC, Wong WH: Transitive functional annotation shortest- <br />     Weissman J, Krogan NJ: Toward comprehensive atlas physical                path analysis gene expression data. Proc Natl Acad Sci 2002, <br />     interactome Saccharomyces cerevisiae. Mol Cell Proteomics 2007,                 99:12783-8. <br />     6:439-450.                                                                     36. Winzeler EA, Shoemaker DD, Astromoff A, Liang H, Anderson K, Andre B, <br /> 12. Hart G, Lee I, Marcotte ER: A high-accuracy consensus map yeast                 Bangham R, Benito R, Boeke JD, Bussey H, Chu AM, Connelly C, Davis K, <br />     protein complexes reveals modular nature gene essentiality. BMC                 Dietrich F, Dow SW, Bakkoury E, Foury F, Friend SH, Gentalen E, Giaever G, <br />     Bioinformatics 2007, 8:236-247.                                                    Hegemann JH, Jones T, Laub M, Liao H, Liebundguth N, Lockhart DJ, <br /> 13. Zhang B, Park BH, Karpinets T, Samatova N: From pull-down data                  Lucau-Danila A, Lussier M, Menard P, Mittmann M, Pai C, Rebischung C, <br />     protein interaction networks complexes biological relevance.              Revuelta JL, Riles L, Roberts CJ, Ross-MacDonald P, Scherens B, Snyder M, <br />     Systems Biology 2008, 24:979-986.                                                  Sookhai-Mahadeo S, Storms RK, Veronneau S, Voet M, Volckaert G, Ward TR, <br /> 14. Chua H, Ning K, Sung W, Leong H, Wong L: Using indirect protein-protein            Wysocki R, Yen GS, Yu K, Zimmermann K, Philippsen P, Johnston M, <br />     interactions protein complex prediction. J Bioinformatics                  Davis RW: Functional characterization S. cerevisiae genome by <br />     Computational Biology 2008, 6:435-466.                                             gene deletion parallel analysis. Science 1999, 285:901-906. <br /> 15. Liu G, Li J, Wong L: Assessing predicting protein interactions using       37. Giaever G, Chu AM, Ni L, Connelly C, Riles L, Veronneau S, Dow S, Lucau- <br />     local global network topological metrics. Genome Informatics              Danila A, Anderson K, Andre B, Arkin AP, Astromoff A, El-Bakkoury M, <br />     2008, 22:138-149.                                                                  Bangham R, Benito R, Brachat S, Campanaro S, Curtiss M, Davis K, <br />                                                                                        Deutschbauer A, Entian KD, Flaherty P, Foury F, Garfinkel DJ, Gerstein M, <br />  Sriha <br /> </body></html>