<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>EUKARYOTIC CELL, Oct. 2008, p. 1819&#226;&#8364;&#8220;1830                                                                                         Vol. 7, No. 10 <br /> 1535-9778/08/$08.00&#226;&#171;&#185;0 doi:10.1128/EC.00088-08 <br /> Copyright &#194;&#169; 2008, American Society Microbiology. All Rights Reserved. <br />  <br />  <br />  <br />              TOR1 TOR2 Have Distinct Locations Live Cells&#228;&#338;&#164;&#226;&#8364;&#160; <br />                 Thomas W. Sturgill,1* Adiel Cohen,2 Melanie Diefenbacher,2 Mark Trautwein,2 <br />                                  Dietmar E. Martin,2 Michael N. Hall2 <br />             Department Pharmacology, University Virginia Health Sciences Center, Charlottesville, Virginia 22908,1 <br />                                Biozentrum, University Basel, CH-4056 Basel, Switzerland2 <br />                                                 Received 11 March 2008/Accepted 9 July 2008 <br />  <br />              TOR structurally functionally conserved Ser/Thr kinase multiprotein complexes that <br />           regulate cellular processes control cell growth. Although extensively studied, localization TOR <br />           ambiguous, possibly endogenous TOR live cells examined. Here, examined <br />           localization green fluorescent protein (GFP) tagged, endogenous TOR1 TOR2 live S. cerevisiae <br />           cells. A DNA cassette encoding copies green fluorescent protein (3XGFP) inserted TOR1 <br />           gene (at codon D330) TOR2 gene (at codon N321). The TORs tagged internally TOR1 or <br />           TOR2 tagged N C terminus functional. The TOR1D330-3XGFP strain hypersensitive <span id='am-6' about='protege:TO' typeof='owl:Thing'>to</span> <br />           rapamycin, cold sensitive, resistant manganese toxicity caused loss Pmr1, all <br />           indications TOR1-3XGFP expressed functional. TOR2-3XGFP functional, TOR2 an <br />           essential gene TOR2N321-3XGFP haploid cells viable. Thus, TOR1 TOR2 retain function the <br />           insertion 748 amino acids variable region noncatalytic domain. The localization patterns of <br />           TOR1-3XGFP TOR2-3XGFP documented imaging live cells. TOR1-3XGFP diffusely <br />           cytoplasmic concentrated near vacuolar membrane. The TOR2-3XGFP signal cytoplasmic but <br />           predominately dots plasma membrane. Thus, TOR1 TOR2 distinct localization patterns, <br />           consistent regulation cellular processes different complexes. <br />  <br />  <br />    TOR (target rapamycin) large (&#226;&#172;&#402;2,500 amino acids),                  localized surface vacuolar membrane using <br /> highly conserved protein kinase controls cell growth                   antibody raised sequence TOR2 (5). Subsequently, <br /> response nutrients. In Saccharomyces cerevisiae,                  [35S]TOR1 [35S]TOR2 P13 P100 <br /> two highly related TOR proteins, TOR1 TOR2 (11). Both                      membrane fractions (21). The P13 fraction enriched the <br /> proteins associate membranes partially resistant                  plasma membrane (PM), endoplasmic reticulum (ER), vacu- <br /> to extraction Triton X-100 (3, 21). TOR                   oles, mitochondria, P100 fraction enriched <br /> structurally functionally distinct multiprotein complexes,                 Golgi bodies, endosomes, secretory vesicles. TOR1 and <br /> TORC1 TORC2 (41). TORC1 contains TOR1 TOR2                              TOR2 P13 fraction fractionated equilib- <br /> and Kog1, Tco89, Lst8. TORC2 contains TOR2, Avo1,                          rium sucrose gradients similarly Pma1, PM marker. The <br /> Avo2, Avo3, Bit61, Lst8. Both TORC1 TORC2                          TORs distinct, unidentified membrane <br /> essential, TORC1 inhibited rapamycin. TORC1                     pool. As revealed immunofluorescence, overexpressed hem- <br /> controls growth-related processes, including transcrip-                agglutinin-tagged TOR1 TOR2 localized discrete sites <br /> tion, translation, ribosome biogenesis, <span id='am-1' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> transport,                &#226;&#8364;&#339;dots&#226;&#8364;? just beneath PM (21). By immunogold <br /> autophagy. TORC2 controls different set processes, in-                    electron microscopy, Wedaman et al. identified hemagglutinin- <br /> cluding actin organization, endocytosis, lipid biosynthesis                tagged TOR2 adjacent PM intracellular mem- <br /> (4). The TORCs, components, key role cell                   branous tracks (40). In recent biochemical study, TORC2 <br /> growth conserved yeast human. A major                        components Avo3 TOR2 correlated best early <br /> unanswered question TOR regulates processes,                    endosome marker (Rsv5), TORC1 components over- <br /> directly indirectly.                                                        lapped diffusely trans-Golgi, ER, vacuolar markers <br />    One potential source diversity biologic processes                (3). Li et al. reported immunostaining study TOR1 <br /> under TOR regulation subcellular localization TOR.                   predominantly nuclear exported cytoplasm in <br /> TOR signaling depend colocalization TOR                     response rapamycin treatment <span id='am-2' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> starvation (24). <br /> key substrates involved different cellular functions differ-             Thus, localization TOR ambiguous, possibly because <br /> ent times define branches TOR signaling. The localization                previous studies relied fixed fractionated cells. <br /> of TOR1 TOR2 cells studied biochemical                      Here, visualize localization functional, internally <br /> fractionation immunostaining. In early study, TOR2                      green fluorescent protein (GFP)-tagged TOR1 TOR2 in <br />                                                                                live cells. We localization patterns TOR1-3XGFP <br />    * Corresponding author. Mailing address: Department Pharma-              (TOR1 tagged DNA cassette encoding copies of <br /> cology, Box 800735, Room 5045, Jordan Hall, University Virginia             GFP) TOR2-3XGFP (TOR2 DNA cassette encoding <br /> Health Sciences Center, Charlottesville, VA 22908. Phone: (434) 924-           copies GFP) distinct complex, possibly account- <br /> 1919. Fax: (434) 924-5207. E-mail: Thomas_Sturgill@virginia.edu. <br />    &#226;&#8364;&#160; Supplemental material article http://ec <br />                                                                                ing heretofore ambiguity TOR localization. Further- <br /> .asm.org/.                                                                     more, various localization patterns TORs underlie <br />    &#228;&#338;&#164; <br />      Published ahead print 22 August 2008.                               control different cellular processes. <br />  <br />                                                                         1819 <br />  1820         STURGILL ET AL.                                                                                                                           EUKARYOT. CELL <br />  <br />  <br />                                         TABLE 1. Primers 3XGFP tagging TOR1 TOR2 using pOM3 <br />          Primer                                 Target                                                            Sequence (5&#226;&#172;&#732; 3&#226;&#172;&#732;)a <br />  <br /> TOR1-N-Fwd                                  N terminus                           GGTAAAGTGAAACATACATCAACCGGCTAGCAGGTTTGCATTGAT <br />                                                                                   ATG TGC AGG TCG ACA ACC CTT AAT <br /> TOR1-N-Rev                                  N terminus                           CGCTTTCAAAAGTTTACTCTTCCAAATCTGCTCCTCATGCGGTTC <br />                                                                                   GCG GCC GCA TAG GCC ACT <br /> TOR1-D67-Fwd                                D67                                  TGACTTCTAGTAGGTTTGATGGAGTGGTGATTGGCAGTAATGGG <br />                                                                                   TGC AGG TCG ACA ACC CTT AAT <br /> TOR1-D67-Rev                                D67                                  GGTTAATTCGCGGAAAATTTTCTCCAAAATGGGCTTAAAATTTAC <br />                                                                                   GCG GCC GCA TAG GCC ACT <br /> TOR1-D330-Fwd                               D330                                 ATCCATGCAAGTTTGTTGGTTTATAAGGAAATCTTGTTTTTGAAG <br />                                                                                   TGC AGG TCG ACA ACC CTT AAT <br /> TOR1-D330-Rev                               D330                                 TATGCAATTTAGACACATTTGGTCGAACACTTGATTCAAAAAGGG <br />                                                                                   GCG GCC GCA TAG GCC ACT <br /> TOR2-N-Fwd                                  N terminus                           TCTTTCTCAAAGAGATTTCTGATCTTTACTTTCCCCATATGAAAA <br />                                                                                   ATG TGC AGG TCG ACA ACC CTT AAT <br /> TOR2-N-Rev                                  N terminus                           AGACAATAAGTTAGGTGGCGTGGTGTATTTGTTAATGTATTTATT <br />                                                                                   GCG GCC GCA TAG GCC ACT <br /> TOR2-N321-Fwd                               N321                                 CAAAGATTATTTCAAGGTTGTACACATGGCTTAAGTCTCAATACG <br />                                                                                   TGC AGG TCG ACA ACC CTT AAT <br /> TOR2-N321-Rev                               N321                                 GCTGAGTAATTCTCGAAATACCAACAGAGTAGCATGCACTGAATC <br />                                                                                   GCG GCC GCA TAG GCC ACT <br />   a <br />     The underlined portion sequence TOR1 TOR2. The italicized portion ATG codon added primer replace initiating ATG codon. <br /> Sequence italicized underlined sequence pOM3 template. <br />  <br />  <br />  <br />                          MATERIALS AND METHODS                                          nonrepeated kanMX6 ORF. With TOR1/Fwd/&#226;&#171;&#186;500, expected prod- <br />                                                                                         ucts observed 9/15 colonies tested N-terminal tagging (741 nucleo- <br />    Yeast media. Rich medium (YPD) synthetic medium (SD SC) was <br />                                                                                         tides [nt]), 8/15 tested D67 tagging (942 nt), 11/16 tested D330 tagging <br /> prepared described previously (35). Rapamycin added YPD medium <br /> <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> dilutions 1-mg/ml stock 90% ethanol-10% Tween 20 just               (1,731 nt), TOR2/Fwd/&#226;&#171;&#186;486, expected product (727 nt) ob- <br /> plates poured.                                                                     served 2/2 colonies. <br />    Strains 3XGFP inserted TOR1 TOR2. A sequence 3XGFP                Cells taken cultures liquid YPD transformed pSH47 with <br /> the S65G <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span>mutation</span> optimized expression yeast amplified              lithium acetate-polyethylene glycol lower-efficiency, simplified method, and <br /> pBS-3XGFP-TRP1 (22) primers MT101 (5&#226;&#172;&#732;-CCCGATATCGGAGGATCC                           transformants selected SD-uracil (SD-ura) plates (secondary colo- <br /> ATGTCTAAAGGT-3&#226;&#172;&#732;) MT102 (5&#226;&#172;&#732;-GGACTAGTTTTGTACAATT                                      nies). Plasmid pSH47 introduces Gal-inducible Cre 5-fluoroorotic acid <br /> CATCCATACCAT-3&#226;&#172;&#732;) cloned EcoRV SpeI sites pUG6                 removable plasmid. Tiny amounts single colonies inoculated 4 ml of <br /> restriction, creating pOM3. We constructed 3XGFP strains described previ-            SC-ura containing 1% galactose 1% raffinose 15-ml disposable tubes. <br /> ously using pOM3, minor modifications (15). The strategy uses kanMX6             After induction 5 h (30&#194;&#176;C), cells collected centrifugation, resus- <br /> resistance marker select recombinants later removed Cre re-           pended small residual volume, streaked obtain single colonies <br /> combinase, restoring open reading frame (ORF) gene containing           YPD plates. These tertiary colonies identified, circled labeled, and <br /> repeated tag. The specific TOR primers used amplify 3.8-kb cassette         streaked YPD plates YPD plates containing G418 test excision of <br /> pOM3 given Table 1. Portions PCRs used transform haploid          kanMX6 Gal-induced Cre. A convention, example, TOR1N-15-6-1, was <br /> TB50a (for TOR1) diploid TB50a/&#226;?&#163; (for TOR2) strains high-efficiency            helpful indicate gene site followed single colony numbers (for <br /> lithium acetate-polyethylene glycol method. Transformants allowed re-           primary [G418], secondary [SD-ura], tertiary [YPD] colonies in- <br /> cover 4 6 h YPD plated YPD plates containing G418.            troduction Cre). The tertiary colonies lost kanMX6 marker were <br /> Single colonies streaked YPD plates allowed grow 30&#194;&#176;C,            studied further. <br /> plates stored 4&#194;&#176;C primary isolates.                                         Validation TOR1-3XGFP strains. TOR2-3XGFP strains verified as <br />    The TOR1 ORF disrupted stop codon kanMX6 marker               described later text. TOR1-3XGFP strains verified follows. Re- <br /> removal Cre recombinase (15). The primers used characterization           combination integrate 3XGFP cassette confirmed colony PCR <br /> of strains listed Table 2. Colonies tested integration correct   primers TOR1 sequences outside cassette. PCR primers TOR1/ <br /> site colony PCR Cre step, using reverse primer (kanMX6/Rev)           Fwd/&#226;&#171;&#186;72 TOR1/Rev/&#226;&#171;&#185;280 (Table 2) gave 2,599-nt band diagnostic for <br />  <br />  <br />  <br />                                                       TABLE 2. Primers used characterization strains <br />          Primer                                                    Locationa                                                                Sequence <br />  <br /> TOR1/Fwd/&#226;&#171;&#186;500                                   Promoter, &#226;&#171;&#186;500 &#226;&#171;&#186;480                                                 5&#226;&#172;&#732;-AGACAGCAAACCACCTGTTGC-3&#226;&#172;&#732; <br /> TOR1/Fwd/&#226;&#171;&#186;72                                    Promoter, &#226;&#171;&#186;72 &#226;&#171;&#186;53                                                   5&#226;&#172;&#732;-CCATAGTAGCTTCACGAGAG-3&#226;&#172;&#732; <br /> TOR1/Fwd/&#226;&#171;&#185;801                                   ORF, 801 821                                                        5&#226;&#172;&#732;-GGTGATCAGAATTGATGCGTC-3&#226;&#172;&#732; <br /> TOR1/Rev/&#226;&#171;&#185;280                                   ORF, 258 280                                                        5&#226;&#172;&#732;-TACTGGCCAATTTTCGTTCCTCC-3&#226;&#172;&#732; <br /> TOR1/Rev/&#226;&#171;&#185;1101                                  ORF, 1080 1101                                                      5&#226;&#172;&#732;-TGCTAATAGGGGAACAATCTGG-3&#226;&#172;&#732; <br /> TOR2/Fwd/&#226;&#171;&#186;486                                   Promoter, &#226;&#171;&#186;486 &#226;&#171;&#186;464                                                 5&#226;&#172;&#732;-CGTCATTTTTTACATACTGCTGC-3&#226;&#172;&#732; <br /> TOR2/Fwd/&#226;&#171;&#185;711                                   ORF, 711 730                                                        5&#226;&#172;&#732;-ATACAGGAGACATGCTGCGC-3&#226;&#172;&#732; <br /> TOR2/Fwd/&#226;&#171;&#185;931                                   ORF, 931 951                                                        5&#226;&#172;&#732;-TTTCAAGGTTGTACACATGGC-3&#226;&#172;&#732; <br /> TOR2/Rev/&#226;&#171;&#185;990                                   ORF, 970 990                                                        5&#226;&#172;&#732;-CAGAGTAGCATGCACTGAATC-3&#226;&#172;&#732; <br /> kanMX6/Rev                                      3XGFP cassette                                                         5&#226;&#172;&#732;-CCGTGCGGCCATCAAAATG-3&#226;&#172;&#732; <br /> GFP/Rev                                         3XGFP cassette, 165 185 (after Cre)                                 5&#226;&#172;&#732;-GCATCACCTTCACCTTCACCG-3&#226;&#172;&#732; <br />   a <br />       Location nt ATG start codon specified. <br />  VOL. 7, 2008                                                                                                    TOR1-3XGFP AND TOR2-3XGFP                          1821 <br />  <br />  <br />                                                              TABLE 3. Yeast strains plasmids <br /> Strain plasmid                                                    Genotype description                                                      Reference source <br />  <br />  Strains <br />    TB50a                    MATa leu2-3,112 ura3-52 rme1 trp1 his3 HMLa                                                                             Hall collection <br />    TB50a/&#226;?&#163;                  MATa/&#226;?&#163; leu2-3,112/leu2-3,112 ura3-52/ura3-52 rme1/rme1 trp1/trp1 his3/his3 HMLa/HMLa                                    Hall collection <br />    JK9-3da/&#226;?&#163;                MATa/&#226;?&#163; leu2-3,112/leu2-3,112 ura3-52/ura3-52 rme1/rme1 trp1/trp1 his4/his4 HMLa/HMLa                                    Hall collection <br />    AN9-2a                   MATa leu2-3,112 ura3-52 rme1 trp1 his3 tor1::kanMX6 HMLa <br />    LJ25-1A                  MATa leu2-3,112 ura3-52 rme1 trp1 his3 pmr1&#226;&#338;&#172;::kanMX4 HMLa                                                               10 <br />    LJ25-2C                  MAT&#226;?&#163; leu2-3,112 ura3-52 rme1 trp1 his3 pmr1&#226;&#338;&#172;::kanMX4 HMLa                                                               10 <br />    VA19                     MATa leu2-3,112 ura3-52 rme1 trp1 his3 KOG18XGFP::URA3 HMLa                                                             This   study <br />    VA33                     MATa leu2-3,112 ura3-52 rme1 trp1 his3 TOR1D330-3XGFP HMLa/pSH47 &#229;&#8230;&#179;CEN URA3&#229;&#8230;&#180;                                             This   study <br />    VA34                     MATa leu2-3,112 ura3-52 rme1 trp1 his3 TOR1D330-3XGFP HMLa/pSH47 &#229;&#8230;&#179;CEN URA3&#229;&#8230;&#180;                                             This   study <br />    VA35                     MATa leu2-3,112 ura3-52 rme1 trp1 his3 TOR1D330-3XGFP HMLa/pSH47 &#229;&#8230;&#179;CEN URA3&#229;&#8230;&#180;                                             This   study <br />    VA36                     MATa leu2-3,112 ura3-52 rme1 trp1 his3 TOR1D67-3XGFP HMLa/pSH47 &#229;&#8230;&#179;CEN URA3&#229;&#8230;&#180;                                              This   study <br />    VA38                     MATa leu2-3,112 ura3-52 rme1 trp1 his3 TOR1D67-3XGFP HMLa/pSH47 &#229;&#8230;&#179;CEN URA3&#229;&#8230;&#180;                                              This   study <br />    VA41                     MATa leu2-3,112 ura3-52 rme1 trp1 his3 TOR1N-3XGFP HMLa/pSH47 &#229;&#8230;&#179;CEN URA3&#229;&#8230;&#180;                                                This   study <br />    VA43                     MATa/&#226;?&#163; leu2-3,112/leu2-3,112 ura3-52/ura3-52 rme1/rme1 trp1/trp1 his3/his3 TOR2N321-3XGFP/                              This   study <br />                              TOR2 HMLa/HMLa <br />     VA66                    MATa leu2-3,112 ura3-52 rme1 trp1 his3 TOR1D330-3XGFP HMLa                                                              This   study <br />     VA68-9c                 MATa leu2-3,112 ura3-52 rme1 trp1 his3 TOR1D330-3XGFP HMLa/pSH47 &#229;&#8230;&#179;CEN URA3&#229;&#8230;&#180;                                             This   study <br />     VA68-9c                 MATa leu2-3,112 ura3-52 rme1 trp1 his3 pmr1&#226;&#338;&#172;::kanMX4 TOR1D330-3XGFP HMLa                                                This   study <br />     YGD25                   MATa leu2-3,112 ura3-52 rme1 trp1 his3 tor1&#226;&#338;&#172;::LEU2-4 pmr1&#226;&#338;&#172;::kanMX4 HMLa                                                 9 <br />     VA86                    MATa leu2-3,112 ura3-52 rme1 trp1 his3 TOR2N321-3XGFP HMLa/pSH47 &#229;&#8230;&#179;CEN URA3&#229;&#8230;&#180;                                             This   study <br />     VA102                   MATa leu2-3,112 ura3-52 rme1 trp1 his3 TOR2N321-3XGFP HMLa                                                              This   study <br />     VA109                   MATa leu2-3,112 ura3-52 rme1 trp1 his3 OG18XGFP::URA3 HMLa                                                              This   study <br />     VA121                   MATa leu2-3,112 ura3-52 rme1 trp1 his3 TOR2N321-3XGFP KOG18XGFP::URA3 HMLa                                              This   study <br />     YGD25                   MATa leu2-3,112 ura3-52 rme1 trp1 his3 tor1&#226;&#338;&#172;::LEU2-4 pmr1&#226;&#338;&#172;::kanMX4 HMLa                                                 9 <br />  <br />  Plasmids <br />    pSH47                    pGAL-cre &#229;&#8230;&#179;CEN URA3&#229;&#8230;&#180;                                                                                                     16 <br />    pTA19                    Integrating plasmid KOG1 (nt 3000&#226;&#8364;&#8220;4671)-8XGFP:URA3                                                                      2 <br />    pTPQ128                  pADH1-Sec7-dsRed &#229;&#8230;&#179;CEN LEU2&#229;&#8230;&#180;                                                                                             31 <br />    pTPQ127                  pGPD1-FYVE-dsRed &#229;&#8230;&#179;CEN LEU2&#229;&#8230;&#180;                                                                                             31 <br />  <br />  <br />  <br />  <br /> 3XGFP colony TOR1/D67-1-1-1, strain VA38 (Table 3). PCR            TOR1-3XGFP, VA109 (constructed similarly VA19 TB50&#226;?&#163;) was <br /> with set gave expected 2,602-nt band expected 3XGFP            crossed VA66 desired segregant (VA121) chosen. <br /> colony TOR1N-15-6-1, strain VA41 (Table 3). PCR TOR1/Fwd/&#226;&#171;&#185;801                 Spottings growth assays. Strains grown overnight liquid YPD (5-ml <br /> TOR1/Rev/&#226;&#171;&#185;1101 gave expected 2,548-nt band diagnostic 3XGFP            culture), collected centrifugation, washed water, 0.1 ml was <br /> TOR1/D330-14-1-6 14-1-3, strains VA34 VA35, respectively,             used inoculate SD (5 ml) overnight. Cells adjusted concentration an <br /> larger &#226;&#172;&#402;3.4-kb band TOR1/D330-3-1-2, strain VA33.                               optical density 600 nm 1.0, serially diluted 1:10 96-well plate, and <br />    As second proof, repeated PCR primer specific GFP     transferred plates pinning device. <br /> pOM3. Using TOR1/Fwd/&#226;&#171;&#186;72 GFP/Rev, bands corresponding                 FM4-64 staining. Cells grown YPD A600 0.8. Cells (20 40 <br /> expected 456-, 1,176-, 1,896-nt products 3XGFP insertion clearly   A600 units ml&#226;&#171;&#186;1 YPD medium) incubated ice 30 min 30 &#226;?&#174;mol <br /> present strain VA38 (Table 3). VA34 confirmed observation        liter&#226;&#171;&#186;1 FM4-64 dye (Molecular Probes Inc.), washed YPD, incu- <br /> expected 372-, 1,092-, 1,896-products using TOR1/Fwd/&#226;&#171;&#185;801 GFP/           bated 60 min steady-state experiments described previously (38). <br /> Rev. A PCR VA38 TOR1/Fwd/&#226;&#171;&#186;72 GFP/Rev gave expected                    Microscopy. Cells imaged log phase, immobilization on <br /> 260- 980-nt products 1,700-nt product VA38, prefer-       slides coated concanavalin A. Microscopy 3XGFP strains was <br /> ential amplification smaller products.                                           performed Zeiss Axioplan 2 microscope equipped MRm camera <br />    As final proof, correct integration confirmed sequenc-   (Carl Zeiss, Aalen Oberkochen, Germany) Plan Apochromat 63&#226;&#171;&#187;/1.40- <br /> ing. Genomic DNA purified cell breakage phenol-chloroform            numerical-aperture objective oil. Zeiss Axiovision 3.1 software used to <br /> method amplified PCR TOR1/Fwd/&#226;&#171;&#185;801 (for VA34 VA35),                 control filters acquire images field chosen differential <br /> TOR1/Fwd/&#226;&#171;&#186;72 (for VA34 VA41), GFP/Rev. The band priming                interference contrast (DIC). Exposure settings GFP different figures varied <br /> to GFP purified sequenced. VA33 VA34/VA35            800 2,000 ms noted; comparisons intensity are <br /> independent primary colonies.                                                       valid figures figures. Images analyzed Axiovi- <br />    Isolation proteins Western blot analysis. A crude supernatant protein     sion software (Zeiss), equal adjustments images (control 3XGFP <br /> (1,500 &#226;&#171;&#187; g 1 min) isolated method using glass beads breakage   strains) taken experiment. Axiovision files exported TIFF format <br /> of cells. The lysis buffer ice-cold phosphate-buffered saline, 10% glycerol,    cropped Photoshop modification brightness contrast. <br /> and 0.5% Tween 20 inhibitors (1.25 &#226;?&#174;g/ml leupeptin, 0.75 &#226;?&#174;g/ml antipain,       For confocal imaging, used Olympus 1X81 spinning-disk microscope and <br /> 0.25 &#226;?&#174;g/ml chymotrypsin, 0.25 &#226;?&#174;g/ml elastinol, 5 &#226;?&#174;g/ml pepstatin, 1 mM phen-        Plan Apochromat 60&#226;&#171;&#187;/1.42-numerical-aperture oil objective, used IQ <br /> ylmethylsulfonyl fluoride, 1 mM EDTA). Protein (40 &#226;?&#174;g), sodium dodecyl           Andor software collect deconvolute Z-stack images. Deconvoluted images <br /> sulfate sample buffer, loaded 7.5% sodium dodecyl sulfate-polyacryl-       exported ImageJ Photoshop. <br /> amide gel electrophoresis gel used Western blot analysis (6). A mixture <br /> of mouse monoclonal antibodies (clones 7.1 13.1) GFP (Roche <br /> Applied Science) used dilution 1:1,000. The antibody detection system <br /> (ECL kit) Amersham.                                                                                          RESULTS <br />    Kog1-8XGFP strains. Plasmid pTA19 contains portion (3,000 4,671 nt) of <br /> the KOG1/LAS24 ORF HindIII XbaI sites p8XGFPIU (2). TB50a <br />                                                                                       Identification region TOR1 internal 3XGFP tag- <br /> was transformed SalI-treated pTA19, generating strain VA19 colony        ging. Visualization low-abundance proteins, TORs, <br /> selection SD-ura (Table 3). To generate strains Kog1-GFP           requires incorporation multiple tags. The incor- <br />  1822     STURGILL ET AL.                                                                                                     EUKARYOT. CELL <br />  <br />  <br />  <br />  <br />    FIG. 1. The alignment evolutionary divergent species identifies region TOR insertion 3XGFP. Proteins aligned by <br /> ClustalW (7). Only relevant portion shown. Residues D67 D330 TOR1 N321 TOR2 indicated asterisks. Arrows indicate <br /> sites 3XGFP inserted D330 (TOR1) N321 (TOR2) retention function. Humans, Caenorhabditis elegans, Cryptococcus <br /> neoformans TOR. Default colors alignment parameters implementation ClustalW (7) http://npsa-pbil.ibcp.fr/. <br />  <br />  <br />  <br />  <br /> poration multiple copies N terminus           TOR2N-3XGFP, respectively) nonfunctional (see Fig. 5 and <br /> at C terminus protein question. However,            8). TOR contains multiple subdomains, have <br /> experience (data shown), TOR1 TOR2                   fully characterized important interactions <br /> tolerate fusion tag N C terminus.              conserved components TOR complexes. We reasoned <br /> Consistent previous observations, TOR1 TOR2                internal region permissive insertion 3XGFP <br /> with 3XGFP N terminus (encoded TOR1N-3XGFP                  revealed comparison evolutionary divergent <br />  VOL. 7, 2008                                                                                             TOR1-3XGFP AND TOR2-3XGFP                   1823 <br />  <br />  <br />  <br />  <br />    FIG. 2. TOR2-3XGFP expressed, TOR1D330-3XGFP strains normal sensitivity rapamycin. (A) Western blot anti-GFP antibodies 40 &#226;?&#174;g <br /> of total protein (see Materials Methods) control (Ctl), TB50a, VA34 strains. The indicated strains streaked YPD (B), YPD plus 1 nM <br /> rapamycin (Rap) (C), YPD plus 2 nM rapamycin (D). Cells grown 2 days 30&#194;&#176;C streaking scanned. <br />  <br />  <br />  <br /> TOR proteins. To identify candidate region insertion <br /> of 3XGFP, aligned TOR proteins (viz., human mTOR <br /> [mammalian TOR], Saccharomyces cerevisiae TOR1 TOR2, <br /> Schizosaccharomyces pombe Tor2, Caenorhabditis elegans TOR, <br /> and Cryptococcus neoformans TOR) (Fig. 1). We variable <br /> region near D330 noncatalytic domain S. cerevisiae <br /> TOR1. This region TORs contains natural inser- <br /> tions, D330 TOR1 viewed potentially permis- <br /> sive insertion 3XGFP. To test hypothesis, gen- <br /> erated independent strains (VA33, VA34, VA35), of <br /> which contain in-frame cassette encoding 3XGFP replacing <br /> D330 TOR1 (see Materials Methods Table 3). The <br /> 3XGFP insertion shown alignments Fig. S1 the <br /> supplemental material. We used different phenotypic tests <br /> to determine functionality new TOR1D330-3XGFP allele <br /> encoding TOR1-3XGFP, described directly below. <br />    TOR1-3XGFP encoded TOR1D330-3XGFP functional. <br /> TOR1-3XGFP expressed intact protein an <br /> apparent mass greater 250 kDa (Fig. 2A). A <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> <br /> for loss TOR1 rapamycin hypersensitivity. Rapa- <br /> mycin hypersensitivity tor1&#226;&#338;&#172; cells specific loss <br /> of TOR1 function point mutations inactivate <br /> TOR1 kinase activity cause hypersensitivity low con- <br /> centration (1 nM) rapamycin (33). The independent <br /> strains TOR1D330-3XGFP hypersensitive 1 nM <br /> or 2 nM rapamycin (Fig. 2B C) grew equivalently in <br /> the absence rapamycin (Fig. 2A), indicating TOR1- <br /> 3XGFP expressed functional. A higher concentra- <br />                                                                                    FIG. 3. TOR1D330-3XGFP strains adapt cold stress. The indicated <br /> tion rapamycin (5 nM) nearly completely inhibited                        strains streaked YPD (A) YPD plus 2 nM rapamycin <br /> growth TOR1D330-3XGFP TOR1 strains ex-                            (Rap) (B) Cells grown 4 days 15&#194;&#176;C (15 deg C) after <br /> pected (data shown).                                                        streaking scanned. <br />  1824     STURGILL ET AL.                                                                                               EUKARYOT. CELL <br />  <br />  <br />         A. YPD <br />                                    TOR1, <br />                                    PMR1 <br />  <br />  <br />         TOR1D330-3XGFP,                           tor1&#226;&#710;&#8224;, <br />         PMR1                                      PMR1 <br />  <br />  <br />  <br />         TOR1D330-3XGFP,                           tor1&#226;&#710;&#8224;, <br />         pmr1&#226;&#710;&#8224;                                     pmr1&#226;&#710;&#8224; <br />  <br />  <br />  <br />                                    TOR1, <br />                                    pmr1&#226;&#710;&#8224; <br />  <br />         B. YPD + 2 mM Mn2+ <br />                                 TOR1, <br />                                 PMR1 <br />  <br />  <br />         TOR1D330-3XGFP,                           tor1&#226;&#710;&#8224;, <br />         PMR1                                      PMR1 <br />  <br />  <br />                                                                        FIG. 5. TOR1D67-3XGFP partially functional. (A) Growth YPD <br />         TOR1D330-3XGFP,                           tor1&#226;&#710;&#8224;,            containing 1 nM rapamycin. The indicated strains (Table 3) were <br />         pmr1&#226;&#710;&#8224;                                     pmr1&#226;&#710;&#8224;             streaked, 3 days growth, plate scanned. Growth <br />                                                                     patterns YPD streaks experiment similar <br />                                                                     (data shown). (B) Growth 15&#194;&#176;C (15 deg) YPD YPD plus <br />                                 TOR1,                               1 nM rapamycin (Rap). The indicated strains assayed 10-fold <br />                                 pmr1&#226;&#710;&#8224;                               dilution pinning; TOR1D67-3XGFP (VA38) used. The plates <br />                                                                     scanned 4 days. <br />   FIG. 4. TOR1D330-3XGFP, like TOR1, sensitive Mn2&#226;&#171;&#185; a <br /> pmr1&#226;&#338;&#172; strain. The indicated strains streaked YPD (A) or <br /> YPD containing 2 mM Mn2&#226;&#171;&#185; (final concentration) (B). Cells were <br /> grown 2 days 30&#194;&#176;C, plates scanned. The TOR1    N terminus TOR1, significantly more <br /> PMR1 (TB50a), tor1&#226;&#338;&#172; PMR1 (AN9-2a), tor1&#226;&#338;&#172; pmr1&#226;&#338;&#172; (YGD25),             divergence sequence D330 (Fig. 1). Notably, TOR1 <br /> TOR1 pmr1&#226;&#338;&#172; (LJ25-1A), TOR1D330-3XGFP pmr1&#226;&#338;&#172; (VA68-9c), and <br />                                                                     functional 2XMyc higher numbers Myc pro- <br /> TOR1D330-3XGFP PMR1 (VA34) strains used (Table 3). <br />                                                                     teins, placed residues 86 87 N-terminal <br />                                                                     segment (27; A. Lorberg, personal communication). We found <br />                                                                     TOR1D67-3XGFP partially functional it <br />    The loss TOR1 causes cold-sensitive growth defect (Fig.     conferred partial resistance 1 nM rapamycin (Fig. 5A). <br /> 3). Adaptation cold stress rescued kinase-defec-     The function TOR1D67-3XGFP greater of <br /> tive TOR1 (data shown). TOR1D330-3XGFP strains         TOR1N-3XGFP, reflected rapamycin cold sensitivity <br /> cold sensitive (Fig. 3A). In fact, TOR1D330-3XGFP strains grew      corresponding strains. The TOR1D330-3XGFP strain was <br /> somewhat better 15&#194;&#176;C wild type. This enhanced           functional growth presence rapamycin at <br /> growth potential TOR1D330-3XGFP strains 15&#194;&#176;C         15&#194;&#176;C TOR1N-3XGFP TOR1D67-3XGFP strain <br /> observed presence 2 nM rapamycin (Fig. 3B). This          (Fig. 5B). <br /> result shows Tor1D330-3XGFP expressed func-        Kog1 essential protein binds TOR1 TOR2 in <br /> tional adaptation cold stress requires active TOR1       TORC1 (27). Strains containing Kog1 3XGFP 8XGFP <br /> kinase activity.                                                    incorporated C terminus genomic tags viable (2, <br />    Pmr1 P-type ATPase localizes predominantly          37). We confirmed KOG1C-8XGFP cells (VA19) not <br /> Golgi bodies transports Ca2&#226;&#171;&#185; Mn2&#226;&#171;&#185; lumen             hypersensitive 1 nM rapamycin (Fig. 5B). <br /> from cytoplasm (13). Pmr1 function important Ca2&#226;&#171;&#185;           TOR1-3XGFP diffusely cytoplasmic concentrated at <br /> regulation secretory pathway removal          discrete sites near vacuolar membranes. TOR1-3XGFP en- <br /> toxic levels Mn2&#226;&#171;&#185;. TOR1 genetic interaction           coded TOR1D330-3XGFP visualized live cells. In cells <br /> pmr1&#226;&#338;&#172; (9). A pmr1&#226;&#338;&#172; strain does grow presence 2        grown rich medium (YPD), TOR1-3XGFP dispersed <br /> mM Mn2&#226;&#171;&#185;, growth rescued loss TOR1. We        cytoplasm concentrated near vac- <br /> that TOR1D330-3XGFP pmr1&#226;&#338;&#172; strain sensitive Mn2&#226;&#171;&#185;     uolar membrane, dot (Fig. 6). TOR1-3XGFP <br /> TOR1 pmr1&#226;&#338;&#172; strain (Fig. 4). Thus, assayed separate      observed, rarely, dots near plasma <br /> phenotypic tests, TOR1-3XGFP encoded TOR1D330-3XGFP           membrane (data shown). The vacuole (Fig. 6A) iden- <br /> functional like wild-type TOR1.                                     tified autofluorescence contents red wavelengths <br />    For comparison insertion D330, con-            morphology DIC. Vacuolar autofluorescence was <br /> structed 3XGFP strains targeted residue D67, nearer         prominent cells grown rich medium <br />  VOL. 7, 2008                                                                                  TOR1-3XGFP AND TOR2-3XGFP               1825 <br />  <br />  <br />  <br />  <br />   FIG. 6. TOR1D330-3XGFP predominantly cytoplasmic concentrated dots near vacuolar membrane. (A) Localization of <br /> TOR1-3XGFP cells grown YDP. Merged GFP (green; GFP channel) autofluorescence images TB50a (control) lacking GFP <br /> cassette TOR1-3XGFP (VA34) strains shown. Strains grown overnight YPD, diluted, imaged log phase after <br /> centrifugation suspension synthetic medium. The arrow indicates dot near vacuole, feathered arrow indicates dot near <br /> the vacuolar membrane. (B) The TOR1-3XGFP signal overlaps FM4-64 staining (see Materials Methods) vacuolar membrane. <br /> Control, strain TB50a lacking GFP cassette; TOR1-3XGFP, strain VA102. The exposure settings used follows: DIC, 300 ms; GFP, <br /> 10,000 ms; Fm4-64 (Cy3), 10,000 ms. <br />  <br />  <br />  <br /> (YPD). We confirmed perivacuolar localization               contains guanine-nucleotide exchange activity Arf <br /> TOR1-3XGFP signal comparing FM4-64,             proteins involved Golgi function (8). Sec7-dsRed marker <br /> specifically stains vacuolar membrane steady state (Fig.        trans-Golgi (28). The localization TOR1-3XGFP was <br /> 6B) (38). A portion TOR1-3XGFP overlapped                  qualitatively different Sec7-dsRed. First, Sec7- <br /> expected ring-like staining FM4-64 marking vacuolar             dsRed vesicles numerous dots TOR1- <br /> membrane.                                                              3XGFP, second, punctate signals TOR1-3XGFP <br />    The predominant localization KOG1-GFP vac-                (Fig. 7) did exactly correspond. A closer correspondence <br /> uolar membrane reported previously (2, 37). We                observed TOR1-3XGFP FYVE-dsRed <br /> compared TOR1-3XGFP Kog1-GFP strains strain                   marker. <br /> containing GFP fusions (VA121) (see Fig. S2                   The FYVE-dsRed fusion protein binds phosphatidylino- <br /> supplemental material). The perivacuolar signal GFP in <br />                                                                        sitol-3-phosphate (for review, reference 23) is <br /> the double-GFP strain dramatically increased (see Fig. <br />                                                                        generally marker early endosomes found <br /> S2 supplemental material) compared either <br />                                                                        near vacuole. FYVE-dsRed localizes &#226;&#8364;&#339;to punctate struc- <br /> single-GFP strain. TOR2-3XGFP, studied comparison, <br />                                                                        tures adjacent vacuole, weakly vacuole-limiting <br /> did perivacuolar localization (see Fig. S2 the <br /> supplemental material).                                                membrane, cases vacuole&#226;&#8364;? (18). We <br />    The localization TOR1-3XGFP compared              emphasize description TOR1-3XGFP localiza- <br /> Sec7-dsRed FYVE-dsRed (Fig. 7) (31). Cells grown            tion similar description regard vacu- <br /> selective synthetic medium maintain plasmid encoding            ole. TOR1-3XGFP appeared localize near to, if <br /> the dsRed marker. Sec7 high-molecular-weight protein              within, FYVE-dsRed punctate structures near the <br />  1826     STURGILL ET AL.                                                                                                EUKARYOT. CELL <br />  <br />  <br />  <br />  <br />   FIG. 7. TOR1D330-3XGFP localization distinct Sec7-dsRed similar FYVE-dsRed. We performed microscopy with <br /> the following strains SD SD-leu (top bottom): TB50a (control strain, SD), TOR1D330-3XGFP (V66, SD), TOR1D330-3XGFP plus <br /> FYVE-dsRed (VA66 transformed pTPQ127, SD-leu), TOR1D330-3XGFP plus Sec7-dsRed (VA66 transformed pTPQ128, in <br /> SD-leu). The arrow shows example colocalization TOR1D330-3XGFP FYVE domain marker punctate structure. The exposure <br /> settings used follows: DIC, 100 ms; GFP, 2,000 ms; dsRed, 500 ms. <br />  <br />  <br />  <br /> vacuole. However, TOR1-3XGFP localization distinct                candidate complete 3XGFP cassette prin- <br /> from FYVE-dsRed localization TOR1-3XGFP                   cipal product &#226;&#172;&#402;2.3-kb band chosen further <br /> also diffusely cytoplasmic dot-like concen-           study (becoming VA102). The candidates 2XGFP <br /> trations TOR1-3XGFP near vacuole.                   1XGFP. TOR2-3XGFP VA102 expressed &#226;&#172;&#381;250- <br />   TOR2 functional 3XGFP inserted replace                kDa protein Western blotting anti-GFP antibodies <br /> N321. Our success TOR1D330-3XGFP encouraged                (Fig. 8C). These results demonstrate TOR2 re- <br /> generate GFP fusion alleles TOR2 (Fig. 8). We targeted             mains functional despite insertion 3XGFP N321 <br /> the N terminus residue N321 TOR2. Like D330                 TOR2 3XGFP N terminus nonfunc- <br /> TOR1, N321 TOR2 corresponds variable region                tional. <br /> the noncatalytic domain TOR2. The TOR2-targeted                       TOR2-3XGFP localizes punctate structures near the <br /> 3XGFP cassette introduced diploid strains            plasma membrane. The localization TOR2-3XGFP (en- <br /> TOR2 essential. To assess functionality GFP             coded TOR2N321-3XGFP VA102) compared that <br /> fusion proteins, diploids containing desired TOR2N-3XGFP       Sec7-dsRed FYVE-dsRed conditions similar to <br /> TOR2N321-3XGFP allele sporulated, dissected, ger-            used TOR1-3XGFP (Fig. 9). TOR2-3XGFP did not <br /> minated. The TOR2N-3XGFP allele nonfunctional be-                 colocalize Sec7-dsRed FYVE-dsRed, and <br /> cause spores viable 13/13 tetrads dissected          TOR2-3XGFP concentrated near vacuolar <br /> (see Fig. S3 supplemental material). In contrast,              membrane. Instead, TOR2-3XGFP detectable the <br /> TOR2N321-3XGFP functional (see Fig. S3 supple-             background punctate structures. These structures most <br /> mental material). Nineteen 23 dissected tetrads                apparent beneath plasma membrane. <br /> TOR2N321-3XGFP produced viable spores. An expected                  To better resolve structures, used confocal micros- <br /> diagnostic PCR product (441 nt) observed 2:2                 copy (Fig. 10; Fig. S4 supplemental material). <br /> segregation tetrads analyzed, product             The TOR2-3XGFP signal cytoplasmic concentrated in <br /> absent control cells (Fig. 8A).                                    punctate structures near plasma membrane. <br />   The 441-nt product expected reverse primer         TOR1-3XGFP cytoplasmic concentrated near vac- <br /> amplifying closest site GFP sequence. To          uole. The identities TOR structures near plasma <br /> further assess this, performed colony PCR primers           membrane unknown, resemble eisosomes in <br /> chosen close flanking N321 site (Fig. 8B). One             location appearance numerous eiso- <br />  VOL. 7, 2008                                                                                      TOR1-3XGFP AND TOR2-3XGFP                 1827 <br />  <br />  <br />  <br />  <br />    FIG. 8. TOR2 functional N321 replaced 1X-, 2X-, 3XGFP. (A) A 441-nt product (asterisk) establishes insertion GFP in <br /> germinated spores 2:2 segregation (see text). Primers TOR2/&#226;&#171;&#185;711 GFP/Rev spores 2A 2D (template) used colony <br /> no. 1 (TB50a/&#226;?&#163; background). (B) Results PCR consistent 2XGFP replacing N321 recombination event JK9 colony no. 1, 1XGFP <br /> in TB50 colony no. 2, 3XGFP TB50 colony no. 1 (see text). Primers TOR2/Fwd/&#226;&#171;&#185;931 TOR2/Rev/&#226;&#171;&#185;990 used. The templates <br /> used follows: 2XGFP-JK9 lanes, 2A, 2B, none, 20C; 1XGFP-TB50 lanes, 18A, 18B, 20A, 20B (colony no. 2); 3XGFP-TB50 <br /> lanes, 2A, 2B, 7C, 7D (colony no. 1). Arrows indicate diagnostic bands 1X-, 2X-, 3XGFP (see text). (C) Western blot anti-GFP <br /> of 40 &#226;?&#174;g lysate protein (see Materials Methods) control (Ctl), TB50a, VA102 strains. <br />  <br />  <br />  <br /> somes. Eisosomes recently defined character-                 functional tagged proteins complementary imag- <br /> ized punctate structures plasma membrane involved                ing proteins fixed cells antibodies. Fixation cause a <br /> in early steps endocytosis (36, 39).                                loss vesicular localization proteins, example, Rheb <br />                                                                            (34). The various localization patterns TORs provide a <br />                                                                            molecular basis large number processes controlled by <br />                           DISCUSSION <br />                                                                            TOR explain different studies observed <br />   We constructed functional TOR1-3XGFP TOR2-                           TOR different cellular locations. The localization TOR in <br /> 3XGFP proteins localize TOR living cells. Importantly,               mammals ambiguous, possibly mTOR <br /> this required insertion GFP TOR1 TOR2                (also known literature FRAP [FKBP12-rapamycin-asso- <br /> ORFs N C terminus. TOR1-3XGFP                    ciated protein[) different locations. mTOR has <br /> cytoplasmic concentrated prevacuolar compartment                  reported principally cytoplasmic shuttle <br /> and vacuolar membrane. TOR2-3XGFP cyto-                    cytoplasm nucleus (20), principally <br /> plasmic and, strikingly, concentrated plasma mem-              nuclear cancer cell lines (42), cytoplasmic and <br /> brane. We did detectably observe TOR1-3XGFP                     associated Golgi bodies ER (26), asso- <br /> nucleus, nucleus qualitatively correlated               ciated regulatory subunit protein kinase A (RI&#226;?&#163;) on <br /> decreased staining versus surrounding cytoplasm (data              late endosomes autophagosomes (30). Only study <br /> shown). More studies required address this. Imaging             used GFP-tagged mTOR, GFP-tagged mTOR over- <br />  1828     STURGILL ET AL.                                                                                                 EUKARYOT. CELL <br />  <br />  <br />  <br />  <br />    FIG. 9. TOR2N321-3XGFP localizes punctate structures near plasma membrane. We performed microscopy following strains in <br /> SD SD-leu (top bottom): TB50a (control strain, SD), TOR1-3XGFP (V102, SD), TOR1-3XGFP plus FYVE-dsRed (VA102 <br /> transformed pTPQ127, SD-leu), TOR2-3XGFP plus Sec7-dsRed (VA102 transformed pTPQ128, SD-leu). The VA102 strain <br /> lost pSH47 (URA3) plasmid 5-fluoroorotic acid treatment (Table 3). <br />  <br />  <br /> expressed N terminally tagged 1XGFP func-             plicated endocytosis actin dynamics (14). Furthermore, <br /> tionality remains established.                                   TOR2 activates AGC family kinase Ypk2 (17). Ypk2 phos- <br />    The concentration TOR1-3XGFP near vacuolar                   phorylates Pil1 Lsp1, proteins involved eisosome for- <br /> membrane consistent reports literature.            mation function (29). Moreover, TORC2 required for <br /> The loss TOR1 synthetically lethal loss class C VPS      sphingolipid biosynthesis (4) Ypk2 activated by <br /> (vacuolar protein sorting) genes (43). This genetic interaction        sphingolipids (for review, reference 25). These findings <br /> is likely specific TOR1 overexpression TOR2          make eisosomes interesting candidate location of <br /> fails rescue synthetic lethality. As second correlation,      TOR2 plasma membrane. We attempted address this <br /> the TOR1 interactors Kog1 Tco89 detected near             introducing mCherry tag C terminus LSP1 or <br /> vacuolar membrane imaging Kog1-GFP live cells           SUR7 TOR2-3XGFP strain (i) Lsp1 <br /> immunogold staining Tco89 fixed cells (2, 32, 37). Be-          Sur7 mCherry signals brighter the <br /> cause strain differences affect TOR1-related phenotypes            TOR2-3XGFP signal (ii) TOR2-3XGFP signal showed <br /> (9, 32), confirmed Kog1-8XGFP concentrated            partial colocalization markers (data shown). <br /> at vacuolar membrane TB50 (see supplemental                 Avo3 shows partial colocalization Pil1 (an eisosome <br /> material). As correlation, TOR1 interacts genetically          marker) immunofluorescence fixed cells (R. Shioda, un- <br /> and biochemically Gtr2 Ego complex, near             published data). The specific localization TOR2-3XGFP to <br /> the vacuolar membrane (12). Finally, phosphorylation            eisosomes remains defined. <br /> Sch9 TORC1 occur near vacuole Sch9                     Our findings provide insight structure TOR. It is <br /> localizes near organelle artificial Sch9 substrate         remarkable TOR, strongly conserved protein &#226;&#172;&#402;2,500 <br /> tethered vacuolar membrane phosphorylated               amino acids, retains partial function insertion <br /> TORC1-dependent manner (37). If connection               748 amino acids (3XGFP cassette) noncatalytic do- <br /> TORC1 vacuole, derive role               main. The functionality internally tagged TOR1-3XGFP <br /> vacuole <span id='am-4' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> supply regulation autophagy (19).            TOR2-3XGFP proteins placement of <br /> The physiological relevance TOR1 cytoplasm         3XGFP subdomains. The majority N terminus <br /> plasma membrane remains determined.                         TOR consists repeated HEAT motifs (21). D330 and <br />    The localization TOR2-3XGFP discrete sites near           N321 gap HEAT repeats (see Fig. S5 the <br /> plasma membrane similar eisosomes, recently              supplemental material). Furthermore, recent electron mi- <br /> described protein complexes important endocytosis (39).         croscopy structure TOR1 suggests noncatalytic do- <br /> Interestingly, TOR2 mainly TORC2, im-             main forms N-terminal head, turn, arm (1). We <br />  VOL. 7, 2008                                                                                                   TOR1-3XGFP AND TOR2-3XGFP                          1829 <br />  <br />                                                                                     4. Aronova, S., K. Wedaman, P. A. Aronov, K. Fontes, K. Ramos, B. D. Ham- <br />                                                                                        mock, T. Powers. 2008. Regulation ceramide biosynthesis TOR <br />                                                                                        complex 2. Cell Metab. 7:148&#226;&#8364;&#8220;158. <br />                                                                                     5. Cardenas, M. E., J. Heitman. 1995. FKBP12-rapamycin target TOR2 is <br />                                                                                        vacuolar protein associated phosphatidylinositol-4 kinase activity. <br />                                                                                        EMBO J. 14:5892&#226;&#8364;&#8220;5907. <br />                                                                                     6. Cohen, A., N. Perzov, H. Nelson, N. Nelson. 1999. A novel family yeast <br />                                                                                        chaperons involved distribution V-ATPase membrane <br />                                                                                        proteins. J. Biol. Chem. 274:26885&#226;&#8364;&#8220;26893. <br />                                                                                     7. Combet, C., C. Blanchet, C. Geourjon, G. Deleage. 2000. NPS@: net- <br />                                                                                        work protein sequence analysis. Trends Biochem. Sci. 25:147&#226;&#8364;&#8220;150. <br />                                                                                     8. Deitz, S. B., A. Rambourg, F. Kepes, A. Franzusoff. 2000. Sec7p directs <br />                                                                                        transitions required yeast Golgi biogenesis. Traffic (Copenhagen) <br />                                                                                        1:172&#226;&#8364;&#8220;183. <br />                                                                                     9. Devasahayam, G., D. J. Burke, T. W. Sturgill. 2007. Golgi manganese <br />                                                                                        transport required rapamycin signaling Saccharomyces cerevisiae. <br />                                                                                        Genetics 177:231&#226;&#8364;&#8220;238. <br />                                                                                    10. Devasahayam, G., D. Ritz, S. B. Helliwell, D. J. Burke, T. W. Sturgill. <br />                                                                                        2006. Pmr1, Golgi Ca2&#226;&#171;&#185;/Mn2&#226;&#171;&#185;-ATPase, regulator target of <br />                                                                                        rapamycin (TOR) signaling pathway yeast. Proc. Natl. Acad. Sci. USA <br />                                                                                        103:17840&#226;&#8364;&#8220;17845. <br />                                                                                    11. De Virgilio, C., R. Loewith. 2006. Cell growth control: little eukaryotes <br />                                                                                        make big contributions. Oncogene 25:6392&#226;&#8364;&#8220;6415. <br />                                                                                    12. Dubouloz, F., O. Deloche, V. Wanke, E. Cameroni, C. De Virgilio. 2005. <br />                                                                                        The TOR EGO protein complexes orchestrate microautophagy yeast. <br />                                                                                        Mol. Cell 19:15&#226;&#8364;&#8220;26. <br />                                                                                    13. Durr, G., J. Strayle, R. Plemper, S. Elbs, S. K. Klee, P. Catty, D. H. Wolf, and <br />                                                                                        H. K. Rudolph. 1998. The medial-Golgi ion pump Pmr1 supplies yeast <br />                                                                                        secretory pathway Ca2&#226;&#171;&#185; Mn2&#226;&#171;&#185; required glycosylation, sorting, <br />                                                                                        endoplasmic reticulum-associated protein degradation. Mol. Biol. Cell <br />                                                                                        9:1149&#226;&#8364;&#8220;1162. <br />                                                                                    14. Fadri, M., A. Daquinag, S. Wang, T. Xue, J. Kunz. 2005. The pleckstrin <br />                                                                                        homology domain proteins Slm1 Slm2 required actin cytoskele- <br />                                                                                        ton organization yeast bind phosphatidylinositol-4,5-bisphosphate and <br />                                                                                        TORC2. Mol. Biol. Cell 16:1883&#226;&#8364;&#8220;1900. <br />                                                                                    15. Gauss, R., M. Trautwein, T. Sommer, A. Spang. 2005. New modules for <br />                                                                                        repeated internal N-terminal epitope tagging genes Saccharo- <br />                                                                                        myces cerevisiae. Yeast (Chichester) 22:1&#226;&#8364;&#8220;12. <br />                                                                                    16. Guldener, U., S. Heck, T. Fielder, J. Beinhauer, J. H. Hegemann. 1996. <br />                                                                                        A new efficient gene disruption cassette repeated use budding yeast. <br />                                                                                        Nucleic Acids Res. 24:2519&#226;&#8364;&#8220;2524. <br />    FIG. 10. TOR1 TOR2 distinct localization patterns                   17. Kamada, Y., Y. Fujioka, N. N. Suzuki, F. Inagaki, S. Wullschleger, R. <br /> confocal microscopy. The control (TB50), TOR1-3XGFP (VA66),                        Loewith, M. N. Hall, Y. Ohsumi. 2005. Tor2 directly phosphorylates the <br /> TOR2-3XGFP strains grown YPD imaged confocal micros-                        AGC kinase Ypk2 regulate actin polarization. Mol. Cell. Biol. 25:7239&#226;&#8364;&#8220; <br /> copy (see Materials Methods). The exposure settings used                   7248. <br /> follows: Tor1, 400 ms GFP 50% laser intensity; Tor2, 800 ms GFP                18. Katzmann, D. J., S. Sarkar, T. Chu, A. Audhya, S. D. Emr. 2004. <br /> and 70% laser intensity; TB50, 800 ms GFP 70% laser inten-                     Multivesicular body sorting: ubiquitin ligase Rsp5 required modi- <br /> sity.                                                                                  fication sorting carboxypeptidase S. Mol. Biol. Cell 15:468&#226;&#8364;&#8220;480. <br />                                                                                    19. Khalfan, W. A., D. J. Klionsky. 2002. Molecular machinery required for <br />                                                                                        autophagy cytoplasm vacuole targeting (Cvt) pathway S. cer- <br />                                                                                        evisiae. Curr. Opin. Cell Biol. 14:468&#226;&#8364;&#8220;475. <br /> predict D330 N321 near gap                           20. Kim, J. E., J. Chen. 2000. Cytoplasmic-nuclear shuttling FKBP12- <br />                                                                                        rapamycin-associated protein involved rapamycin-sensitive signaling <br /> these subdomains. The preservation function internal                           translation initiation. Proc. Natl. Acad. Sci. USA 97:14340&#226;&#8364;&#8220;14345. <br /> tagging TOR GFP fact                       21. Kunz, J., U. Schneider, I. Howald, A. Schmidt, M. N. Hall. 2000. HEAT <br /> N terminus C terminus GFP near                               repeats mediate plasma membrane localization Tor2p yeast. J. Biol. <br />                                                                                        Chem. 275:37011&#226;&#8364;&#8220;37020. <br /> (see Protein Data Bank entry 1EMM), minimizing displace-                           22. Lee, W. L., J. R. Oberle, J. A. Cooper. 2003. The role lissencephaly <br /> ment point insertion.                                                        protein Pac1 nuclear migration budding yeast. J. Cell Biol. 160: <br />                                                                                        355&#226;&#8364;&#8220;364. <br />                                                                                    23. Lemmon, M. A. 2003. Phosphoinositide recognition domains. Traffic (Co- <br />                          ACKNOWLEDGMENTS                                               penhagen) 4:201&#226;&#8364;&#8220;213. <br />                                                                                    24. Li, H., C. K. Tsang, M. Watkins, P. G. Bertram, X. F. Zheng. 2006. <br />   We thank colleagues Spang Hall laboratories                 <span id='am-5' about='Thesaurus:Nutrient' typeof='owl:Thing'>Nutrient</span> regulates Tor1 nuclear localization association rDNA <br /> help. We thank Christiana Walch-Solimena Yukifumi Uesonso                      promoter. Nature 442:1058&#226;&#8364;&#8220;1061. <br /> important reagents.                                                                25. Liu, K., X. Zhang, C. Sumanasekera, R. L. Lester, R. C. Dickson. 2005. <br />   This research supported Swiss National Science Foun-                      Signalling functions sphingolipid long-chain bases Saccharomyces <br /> dation Canton Basel (M.N.H.) National Insti-                     cerevisiae. Biochem. Soc. Trans. 33:1170&#226;&#8364;&#8220;1173. <br /> tutes Health (DK052753) (T.W.S.).                                               26. Liu, X., X. F. Zheng. 2007. Endoplasmic reticulum Golgi localiza- <br />                                                                                        tion sequences mammalian target rapamycin. Mol. Biol. Cell 18:1073&#226;&#8364;&#8220; <br />                                                                                        1082. <br />                                 REFERENCES                                         27. Loewith, R., E. Jacinto, S. Wullschleger, A. Lorberg, J. L. Crespo, D. Bonen- <br />  1. Adami, A., B. Garcia-Alvarez, E. Arias-Palomo, D. Barford, O. Llorca.          fant, W. Oppliger, P. Jenoe, M. N. Hall. 2002. Two TOR complexes, only <br />     2007. Structure TOR complex KOG1. Mol. Cell 27:509&#226;&#8364;&#8220;516.            rapamycin sensitive, distinct roles cell growth control. <br />  2. Araki, T., Y. Uesono, T. Oguchi, E. A. Toh. 2005. LAS24/KOG1,                Mol. Cell 10:457&#226;&#8364;&#8220;468. <br />     component TOR complex 1 (TORC1), needed resistance            28. Losev, E., C. A. Reinke, J. Jellen, D. E. Strongin, B. J. Bevis, B. S. Glick. <br />     local anesthetic tetracaine normal distribution actin cytoskeleton       2006. Golgi maturation visualized living yeast. Nature 441:1002&#226;&#8364;&#8220;1006. <br />     yeast. Genes Genet. Syst. 80:325&#226;&#8364;&#8220;343.                                          29. Luo, G., A. Gruhler, Y. Liu, O. N. Jensen, R. C. Dickson. 2008. The <br />  3. Aronova, S., K. Wedaman, S. Anderson, J. Yates III, T. Powers. 2007.           sphingolipid long-chain <span id='am-163' about='xsp:base' typeof='owl:Thing'>base</span>-Pkh1/2-Ypk1/2 signaling pathway regulates <br />     Probing membrane environment TOR kinases reveals functional             eisosome assembly turnover. J. Biol. Chem. 283:10433&#226;&#8364;&#8220;10444. <br />     interactions TORC1, actin, membrane trafficking Saccharo-       30. Mavrakis, M., J. Lippincott-Schwartz, C. A. Stratakis, I. Bossis. 2007. <br />     myces cerevisiae. Mol. Biol. Cell 18:2779&#226;&#8364;&#8220;2794.                                    mTOR kinase regulatory subunit protein kinase A (PRKAR1A) <br />  1830         STURGILL ET AL.                                                 <br /> </body></html>