<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The EMBO Journal Vol. 22 No. 18 pp. 4826&#194;&#177;4835, 2003 <br />  <br />  <br /> A novel eukaryotic factor cytosolic Fe&#194;&#177;S cluster <br /> assembly <br />  <br />  <br />  <br /> Amit Roy1, Natalia Solodovnikova,                                       cluster interconverts IRP1 cytosolic aconitase, there- <br /> Tracy Nicholson, William Antholine2                                 regulating ability bind iron responsive elements <br /> William E.Walden3                                                       (IREs; Eisenstein, 2000). By binding IREs located <br />                                                                         5&#194;&#162; 3&#194;&#162; untranslated region, IRP1 regulates the <br /> Department Microbiology Immunology, University Illinois at <br />                                                                         translation stability, respectively, mRNAs encoding a <br /> Chicago, Chicago, IL 60612 2Biophysics Research Institute, <br /> Medical College Wisconsin, Milwaukee, WI 53226, USA                  variety proteins involved transport storage of <br /> 1 <br />                                                                         iron intermediary metabolism (Eisenstein, 2000; Theil <br />  Present address: Centre Biotechnology, Visva-Bharati University, <br /> Santiniketan, 731 235, West Bengal, India <br />                                                                         Eisenstein, 2000). <br />                                                                            The molecular mechanisms animal cells <br /> 3 <br />  Corresponding author                                                   regulate Fe&#194;&#177;S cluster assembly/disassembly IRP1/ <br /> e-mail: wwalden@uic.edu                                                 c-aconitase remains elucidated. Low intracellular <br /> A.Roy N.Solodovnikova contributed equally work              iron favors IRE-binding form IRP1, resulting in <br />                                                                         stronger translational repression stabilization target <br /> Iron regulatory protein 1 (IRP1) regulated                   mRNAs. High intracellular iron promotes assembly of <br /> the assembly/disassembly [4Fe&#194;&#177;4S] cluster,                   [4Fe&#194;&#177;4S] cluster converting IRP1 c-aconitase, <br /> interconverts IRP1 cytosolic aconitase. A genetic                  inhibiting IRE binding relieving translational repres- <br /> screen isolate Saccharomyces cerevisiae strains                      sion destabilizing target mRNAs (Eisenstein, 2000). <br /> bearing mutations genes required conver-                     Recent evidence suggests Fe&#194;&#177;S cluster disassembly in <br /> sion IRP1 c-aconitase led identi&#194;&#174;cation                 c-aconitase initiated oxidants produced normal <br /> a previously uncharacterized, essential gene,                  cellular metabolism (Brown et al., 1998, 2002). Further, <br /> call CFD1 (cytosolic Fe&#194;&#177;S cluster de&#194;&#174;cient). CFD1                       oxidative stress promotes conversion c-aconitase <span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span> <br /> encodes highly conserved, putative P-loop ATPase.                     IRP1 (Hentze Kuhn, 1996) phosphorylation of <br /> A non-lethal <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span>mutation</span> CFD1 (cfd1-1) reduced                          IRP1 serine residue 138 appears enhance the <br /> c-aconitase speci&#194;&#174;c activity IRP1-transformed yeast                  sensitivity Fe&#194;&#177;S cluster c-aconitase oxidants <br /> by &gt;90%, IRP1 cells readily                  (Brown et al., 1998, 2002). Taken together, results <br /> converted c-aconitase vitro incubation                  argue cluster disassembly c-aconitase dynamic, <br /> iron alone. IRP1-transformed cfd1-1 yeast lacked                        oxidant-initiated process proceeds 3Fe <br /> EPR-detectable Fe&#194;&#177;S clusters c-aconitase, pointing                   intermediate apo-IRP1, rate disassembly being <br /> to defect Fe&#194;&#177;S cluster assembly. The speci&#194;&#174;c                       affected metabolic activity (Eisenstein, 2000). We still <br /> activity cytosolic Fe&#194;&#177;S protein, Leu1p,                  know little apo-IRP1 generated <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> also inhibited &gt;90% cfd1-1 yeast,                         oxidized, 3Fe c-aconitase, however. <br /> activity mitochondrial Fe&#194;&#177;S proteins inhibi-                    Recent studies bacteria yeasts led the <br /> ted. Consistent cytosolic site activity, Cfd1p                identi&#194;&#174;cation number proteins implicated the <br /> was localized cytoplasm. To knowledge,                       biogenesis Fe&#194;&#177;S clusters (for review Lill Kispal, <br /> Cfd1p &#194;&#174;rst cytoplasmic Fe&#194;&#177;S cluster assembly                     2000). Surprisingly, Fe&#194;&#177;S cluster assembly factors <br /> factor described eukaryotes.                                         identi&#194;&#174;ed date mitochondria of <br /> Keywords: aconitase/ATPase/Fe&#194;&#177;S/IRP/YIL003W/DRE3                        eukaryotic cells (Lill Kispal, 2000). That iron- <br />                                                                         responsive Fe&#194;&#177;S cluster assembly IRP1 cytoplasm <br />                                                                         appears facile process animal cells raises the <br />                                                                         question factors involved there. Here report <br /> Introduction                                                            identi&#194;&#174;cation evolutionarily conserved gene that <br /> Fe&#194;&#177;S proteins participate variety cellular processes,           CFD1, required cytosolic Fe&#194;&#177;S cluster <br /> including intermediary metabolism, electron transfer                    biogenesis conversion mammalian IRP1 to <br /> reactions gene regulation. The presence Fe&#194;&#177;S                 c-aconitase yeast. Cfd1p &#194;&#174;rst eukaryotic Fe&#194;&#177;S <br /> cluster, type cluster oxidation state            cluster assembly factor located <br /> important determinants protein activity,                 predominately cytosol. <br /> utilized control point number regulated systems <br /> (for recent reviews Johnson, 1998; Beinert Kiley, <br />                                                                         Results <br /> 1999). For example, regulated assembly disassembly <br /> of Fe&#194;&#177;S clusters recognized central                 Isolation yeast mutant impaired for <br /> control gene expression Fe&#194;&#177;S proteins bacteria             c-aconitase-dependent growth <br /> animals (Beinert Kiley, 1999). In case RNA               Aconitase-de&#194;&#174;cient (aco1) yeast auxotrophic for <br /> binding protein, iron regulatory protein 1 (IRP1), iron-                glutamate inability generate a-keto- <br /> responsive assembly disassembly [4Fe&#194;&#177;4S]                       glutarate TCA cycle (Ogur et al., 1964; <br />  <br /> 4826                                                                                     &#195;&#163; European Molecular Biology Organization <br />                                                                                                                     Fe&#194;&#177;S cluster assembly IRP1 <br />  <br />  <br /> Gangloff et al., 1990). We animal IRP1                        defect leucine-free medium YPD. Strain AR27/IRP1 <br /> can rescue aco1 yeast glutamate auxotrophy,                              showed growth defects media, growth <br /> provided suf&#194;&#174;cient IRP1 converted c-aconitase                      particularly poor glutamate-free medium. Growth <br /> (Brown et al., 1998; Narahari et al., 2000). To isolate yeast                 AR27/IRP1 glutamate-free medium detect- <br /> mutants defects interconversion IRP1                       able 4 days incubation 30&#194;&#176;C (Figure 1), although <br /> c-aconitase, relied requirement c-aconitase                      minimal growth detected a <br /> activity glutamate-dependent growth IRP1-trans-                        week. <br /> formed aco1 yeast. Aconitase-de&#194;&#174;cient yeast transformed <br /> with IRP1 gene multi-copy plasmid (0615d/IRP1)                        Analysis c-aconitase activity AR27/IRP1 <br /> were mutagenized ethyl methane sulfonate (EMS)                           Aconitase activity extracts prepared AR27/IRP1 <br /> and allowed grow individual colonies rich                          reduced 85% compared activity in <br /> (YPD) medium. (We used high copy vector                                 extracts 0615d/IRP1 (Figure 2A). The poor growth of <br /> expression IRP1 bias selection mutations                    AR27/IRP1 glutamate-free medium correlated with <br /> in IRP1 itself.) Approximately 5 3 104 resulting                       depressed IRP1-dependent aconitase activity. This <br /> colonies replica-plated glutamate-free medium                       lowered c-aconitase activity resulted a <br /> and screened ability grow. In order enhance                     defect conversion IRP1 c-aconitase, from <br /> selection strains mutations genes affecting                        reduced stability c-aconitase Fe&#194;&#177;S cluster vivo, or <br /> cytosolic Fe&#194;&#177;S proteins, colonies replica-plated                    reduction total IRP1 protein. Results from <br /> onto leucine-free medium. Leucine biosynthesis requires                       protein immunoblot analysis exclude possibility. <br /> the activity cytosolic Fe&#194;&#177;S protein Leu1p (Ryan et al.,                  IRP1 protein present AR27/IRP1 level higher <br /> 1973; Kispal et al., 1999). As screen, monitored                   (by ~5-fold) parental 0615d/IRP1 strain <br /> the growth colonies YPD medium. Twenty-two                              (Figure 2B). Together ~6-fold reduction in <br /> colonies identi&#194;&#174;ed failed grow grew poorly                    c-aconitase activity, results indicate c-aconitase <br /> on glutamate-free medium, showed growth                            speci&#194;&#174;c activity reduced &gt;95% AR27/IRP1. <br />                                                                                  We investigated IRP1 expressed strain <br />                                                                               AR27 functional. We IRP1 expressed <br />                                                                               parental strain 0615d converted to <br />                                                                               c-aconitase vitro incubation iron DTT <br />                                                                               (Figure 2C). Typically, aconitase activity stimulated <br />                                                                               5- 9-fold incubation 0615d/IRP1 extracts with <br />                                                                               iron (compare activity Figure 2A&#194;&#177;C). After incubation <br />                                                                               iron DTT, aconitase activity AR27/IRP1 <br /> Fig. 1. IRP1-transformed AR27 mutant strain displays severe growth          extracts increased 1.09 3 103 (667.8) mU/mg. This <br /> defect glutamate-free medium. Fresh overnight cultures 0615d         represents increase ~250-fold activity in <br /> AR27, transformed IRP1, harvested, washed twice           untreated AR27/IRP1 extracts (4.3 6 0.1 mU/mg). The <br /> sterile water, resuspended sterile water OD600 0.5.         level activity achieved AR27/IRP1 nearly 5- <br /> Ten microliters suspension 10-fold serial dilutions were <br /> spotted SD medium lacking uracil, (+) (&#194;&#177;)                fold higher reached 0615d/IRP1 cell extracts <br /> glutamate. Cells allowed grow 30&#194;&#176;C 4 days               (245.9 6 2.9 mU/mg; Figure 2C), consistent the <br /> photographing.                                                                higher IRP1 protein AR27/IRP1 (Figure 2B). <br />  <br />  <br />  <br />  <br /> Fig. 2. AR27/IRP1 reduced c-aconitase activity. Strains 0615d/IRP1 AR27/IRP1 grown SD-ura medium OD600 0.8, cells <br /> were harvested extracts prepared described previously (Brown et al., 1998). Extracts used measure endogenous c-aconitase activity (A), <br /> IRP1 protein immunoblotting (B), incubated 0.5 mM ferrous ammonium sulfate 50 mM DTT room temperature 1 h to <br /> activate c-aconitase followed aconitase assay (C). <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>Unit</span> activity mmol cis-aconitate produced min mg total extract protein using 20 mM <br /> isocitrate substrate (Kennedy et al., 1983). The results enzyme assays representative independent experiments. <br />  <br />  <br />                                                                                                                                                4827 <br />  A.Roy et al. <br />  <br />  <br />                                                                         Table I. Cytosolic aconitase activity IRP1-transformed yeast strains <br />                                                                         Strain                  CFD1 allele                  Aconitase activity <br />                                                                                                 pRS313                    (mU/mg) <br />  <br />                                                                         0615d                   None                         11.3   6   1.0 <br />                                                                         AR27                    None                          1.0   6   0.3 <br />                                                                         AR27                    CFD1                          8.8   6   0.4 <br />                                                                         AR27                    cfd1-1                        1.5   6   0.2 <br />                                                                         0615d                   cfd1-1                       11.0   6   0.4 <br />                                                                         NS1-16                  cfd1-1                        0.1   6   0.1 <br />                                                                         NS1-16                  CFD1                          3.1   6   0.1 <br />                                                                         NS1-18                  cfd1-1                        0.4   6   0.1 <br />                                                                         NS1-18                  CFD1                          3.9   6   0.4 <br />  <br />                                                                         Yeast, transformed IRP1 CFD1 indicated, grown to <br />                                                                         mid-log phase selective SD medium, harvested, extracts were <br />                                                                         prepared aconitase assay. NS1-16 NS1-18 two <br />                                                                         independently isolated cfd1D strains. <br /> Fig. 3. Wild-type CFD1 restores glutamate-dependent growth AR27/ <br /> IRP1. Mutant strain AR27 cfd1-disrupted strain NS1 transformed <br /> with indicated CFD1 allele carried pRS313 transformed        mutation change AUG start codon an <br /> with IRP1. Approximately 104 transformed cells spotted 10 ml    AUA codon. We conclude CFD1 YIL003W/ <br /> of water selective SD medium, (+) (&#194;&#177;) glutamate <br /> as indicated, allowed grow 4 days 30&#194;&#176;C. <br />                                                                         DRE3 identical. Because CFD1 informative <br />                                                                         name, use nomenclature refer allele <br />                                                                         carried AR27 cfd1-1. <br />                                                                            Mutation predicted AUG translation start codon <br /> These results indicate potential IRP1 function <br />                                                                         AUA cfd1-1 strongly inhibit synthesis of <br /> as c-aconitase unimpaired AR27/IRP1, consistent <br />                                                                         Cfd1p. Isolation strain carrying mutation the <br /> with view consequence genetic <br />                                                                         CFD1 gene surprising reported that <br /> de&#194;&#174;ciency AR27 assembly maintenance <br /> of Fe&#194;&#177;S cluster IRP1. We gene                 disruption YIL003W/DRE3 lethal (Entian et al., <br />                                                                         1999; Giaever et al., 2002). To explore issue of <br /> responsible <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> CFD1, cytosolic Fe&#194;&#177;S <br />                                                                         lethality further, CFD1 disrupted 0615d. The <br /> cluster de&#194;&#174;cient 1. <br />                                                                         viability resulting cfd1 null strain, called NS1, was <br />                                                                         dependent ectopic expression CFD1, con&#194;&#174;rming that <br /> Cloning CFD1 complementation                               CFD1 essential 0615d. IRP1-transformed NS1 <br /> glutamate-dependent growth phenotype                                 harboring cfd1-1 allele viable glutamate- <br /> AR27/IRP1                                                               supplemented media, failed grow vigorously on <br /> The mutation caused c-aconitase de&#194;&#174;ciency                   glutamate-free media, showing similar growth restriction <br /> AR27/IRP1 recessive. To isolate gene responsible                 AR27/IRP1 (Figure 3). NS1 carrying wild-type <br /> for cytosolic Fe&#194;&#177;S cluster de&#194;&#174;ciency, AR27/IRP1                CFD1 gene transformed IRP1 grew on <br /> transformed genomic library wild-type yeast               media lacking glutamate (Figure 3). Taken together, these <br /> strain transformants screened restored growth              results demonstrate cfd1 de&#194;&#174;ciency lethal and <br /> on glutamate-free medium. This yielded genomic                      cfd1-1 allele, suf&#194;&#174;cient viability, is <br /> clones containing inserts BET1&#194;&#177;PAN1 intergenic                 insuf&#194;&#174;cient conversion IRP1 c-aconitase. <br /> region chromosome IX (see Materials methods <br /> for details library screen). A SacI&#194;&#177;XbaI fragment                Rescue c-aconitase activity wild-type CFD1 <br /> isolated clones complemented AR27/                    Since CFD1 rescued AR27/IRP1 growth restric- <br /> IRP1 full-<span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span> genomic clone,                      tion glutamate-free medium, to <br /> included genes, BET1 uncharacterized                    determine restored c-aconitase activity. <br /> gene designated YIL003W/DRE3. Since BET1 encodes                      Cell extracts prepared AR27/IRP1 carried <br /> membrane protein involved vesicular transport                extra-chromosomal copy wild-type CFD1 the <br /> the endoplasmic reticulum Golgi (Newman                     cfd1-1 allele. Extracts AR27/IRP1 transformed with <br /> Ferro-Novick, 1987; Ferro-Novick et al., 1991), chose                wild-type CFD1 showed nearly 9-fold increase in <br /> to focus YIL003W/DRE3, predicted encode                  aconitase activity seen AR27/IRP1 trans- <br /> a nucleotide binding protein. YIL003W/DRE3 PCR                      formed vector (Table I). Moreover, the <br /> ampli&#194;&#174;ed genomic DNA 0615d AR27. The                         presence wild-type CFD1 raised aconitase activity to <br /> PCR products cloned CEN/ARS vector                      level close seen 0615d/IRP1. Transformation of <br /> used transform AR27/IRP1. Transformation AR27/                    AR27/IRP1 cfd1-1 did signi&#194;&#174;cantly enhance <br /> IRP1 YIL003W/DRE3 isolated 0615d restored                     c-aconitase activity (Table I). The presence cfd1-1 <br /> vigorous growth glutamate-free medium,                   allele did depress c-aconitase activity 0615d/ <br /> YIL003W/DRE3 gene isolated AR27 itself did                     IRP1, consistent recessive phenotype (Table I). <br /> (Figure 3). Comparison YIL003W/DRE3 sequence                     Routinely, slight increase c-aconitase activity <br /> showed AR27 gene differed 0615d wild-                 seen cfd1-1-transformed AR27/IRP1 compared to <br /> type gene single nucleotide, G&#194;&#174;A transition                   cells transformed vector. We believe this <br /> nucleotide 3 open reading frame (ORF). The result                effect gene dosage, cfd1-1 allele should <br />  <br /> 4828 <br />                                                                                                                   Fe&#194;&#177;S cluster assembly IRP1 <br />  <br />  <br />  <br />  <br /> Fig. 4. Comparative alignment yeast Cfd1p. Yeast Cfd1p (Sc Cfd1p) aligned human Cfd1p (Hu Cfd1p), mouse Cfd1p (Mu Cfd1p), <br /> Salmonella enterica serovar typhimurium ApbC (St Mrp) Archaeoglobus fulgidis MinD (Af MinD) using CLUSTAL_X. The human mouse <br /> proteins called NuBP2 (Nakashima et al., 1999). Dark shading alignment identities light shading shows conserved residues. <br /> Residues form Walker P-loop, Switch I Switch II (Walker B motif) elements, comprise important nucleotide binding intra- <br /> molecular signaling domains P-loop ATPases, indicated beneath alignment (Cordell Lowe, 2001; Leipe et al., 2002). The location the <br /> CX2CX2C motif (residues 201&#194;&#177;207) indicated beneath alignment CXXCXXC. Regions form b-strands (black rectangles) a-helices <br /> (gray rectangles) MinD protein aligned sequence (Cordell Lowe, 2001). <br />  <br />  <br /> encode functional protein lower expression                   The predicted nucleotide binding elements Cfd1p, <br /> level wild-type gene. Similarly, CFD1-dependent                     particularly Walker P-loop Walker B motif <br /> conversion IRP1 c-aconitase observed NS1                    (Walker et al., 1982), comprise highly conserved <br /> strain (Table I). These results demonstrate non-                    regions (Figure 4). Unique Cfd1p CX2CX2C motif <br /> lethal mutation cfd1-1 allele impairs ability                  conserved Cfd1p homologs, aligns <br /> yeast convert IRP1 c-aconitase, likely                    region forms surface exposed loop MinD <br /> an inhibition cytosolic Fe&#194;&#177;S cluster assembly.                            (Figure 4; Cordell Lowe, 2001; Sakai et al., 2001). In <br />                                                                              Cfd1p, CX2CX2C motif large insertion, <br /> Characteristics protein encoded CFD1                               relative MinD sequence (Figure 4). <br /> and mutational analysis                                                         The CX2CX2C motif Cfd1p holds a <br /> CFD1 encodes predicted 31 922 Da Mr protein                        potential metal binding domain. To investigate the <br /> highly conserved shares sequence homology                       importance element Cfd1p function, of <br /> family P-loop ATPases (Leipe et al., 2002). The Mrp                       cysteine residues independently changed serine <br /> and MinD proteins bacteria closely related,                    CFD1 fusion gene encoding N-terminal His6 <br /> the NifH Fe-protein ArsA ATPase. Figure 4 shows                      tagged Cfd1p (see Materials methods); N-terminal <br /> an alignment yeast Cfd1p human mouse Cfd1p                       His6 tag did alter function (Figure 5). The resulting <br /> (also called NuBP2), bacterial Mrp MinD                         mutant genes tested ability rescue the <br /> proteins. Structural features bacterial cell                    glutamate-dependent growth defect AR27/IRP1. While <br /> division protein MinD, crystal structure                      CFD1C207S supported growth, CFD1C201S CFD1C204S <br /> solved (Cordell Lowe, 2001; Hayashi et al., 2001;                        defective, showing visible improvement in <br /> Sakai et al., 2001), aligned MinD sequence.                    growth compared cells transformed vector <br />  <br />                                                                                                                                              4829 <br />  A.Roy et al. <br />  <br />  <br />  <br />  <br /> Fig. 5. Cys201 Cys204 CX2CX2C element required Cfd1p function. AR27/IRP1 transformed vector (none) with <br /> vector carrying His6-tagged wild-type, CFD1C201S, CFD1C204S CFD1C207S fusion genes (see Materials methods). (A) Glutamate-dependent <br /> growth. Approximately 104 cells spotted 10 ml media glutamate, indicated. Photographs taken 4 days of <br /> growth 30&#194;&#176;C. (B) Cytosolic aconitase activity. Aconitase assays performed described legend Figure 2A using extracts from <br /> logarithmically growing AR27/IRP1 transformed indicated CFD1 genes. <br />  <br />  <br /> (Figure 5A). Analysis c-aconitase activity yeast               80% 4Fe form prior treatment H2O2. <br /> gave results consistent growth data. Wild-type                    These results demonstrate little 4Fe c-aconitase exists <br /> CFD1 CFD1C207S restored c-aconitase activity                        AR27 cells strongly support conclusion that <br /> AR27/IRP1, CFD1C201S CFD1C204S did                           Cfd1p plays important role assembly Fe&#194;&#177;S <br /> (Figure 5B). These results strongly implicate                          clusters cytoplasm. <br /> CX2CX2C motif, especially &#194;&#174;rst cysteine <br /> residues, important Cfd1p function.                                  Effect cfd1-1 mutation activity of <br />                                                                            Fe&#194;&#177;S proteins <br /> EPR analysis Fe&#194;&#177;S cluster c-aconitase                            We examined effect cfd1-1 mutation other <br /> in vivo                                                                    Fe&#194;&#177;S cluster proteins, including cytosolic Leu1p and <br /> Cfd1p play role Fe&#194;&#177;S cluster assembly                        mitochondrial succinate dehydrogenase (SDH) and <br /> disassembly cytosolic Fe&#194;&#177;S proteins. To investigate                  aconitase (Aco1p; yeast mitochondrial ACO1 gene <br /> what process Cfd1p function, subjected IRP1-                      transformed 0615d AR27 experi- <br /> transformed yeast EPR analysis. A [3Fe&#194;&#177;4S] cluster,                     ments). Like IRP1-dependent aconitase activity, Leu1p <br /> representing stable breakdown intermediate                        activity reduced &gt;4-fold AR27/ACO1 strain <br /> [4Fe&#194;&#177;4S] cluster present ~20% c-aconitase                      compared 0615d/ACO1 (Figure 7A). A similar <br /> logarithmically growing 0615d/IRP1, form                   difference Leu1p activity seen extracts from <br /> detected intact cells EPR (Brown et al., 2002). More                 NS1 strains, NS1/cfd1-1 having 25% the <br /> than 90% c-aconitase converted 3Fe                    Leu1p activity NS1/CFD1 (data shown). Analysis of <br /> form treatment cells H2O2 prior EPR                          Leu1p protein strains showed AR27/ACO1 <br /> analysis. If defect associated cfd1-1 affects Fe&#194;&#177;S                ~4-fold Leu1p did 0615d/ACO1 <br /> cluster disassembly, increase 3Fe EPR signal                     (Figure 7B), indicating &gt;90% reduction speci&#194;&#174;c <br /> obtained cells evident. On hand,                  activity Leu1p AR27/ACO1 cells. <br /> if cfd1-1 defect precludes Fe&#194;&#177;S cluster assembly                   In contrast, activities mitochondrial Fe&#194;&#177;S <br /> &#194;&#174;rst place, 3Fe EPR signal expected. When                      enzymes diminished AR27/ACO1 relative to <br /> subjected whole-cell EPR, AR27/IRP1 failed                    0615d/ACO1 (Figure 7A). The elevation aconitase <br /> detectable 3Fe signal c-aconitase, typical 3Fe                   activity seen AR27/ACO1 due, part, <br /> c-aconitase signal did appear following treatment                 higher Aco1p yeast compared 0615d/ <br /> with 0.6 mM H2O2 (Figure 6). Note overlapping,                    ACO1 (Figure 7B). While SDH activity modestly <br /> weak, signal seen subtracted spectrum H2O2-                      enhanced, general consequence the <br /> treated AR27/IRP1 distinct 3Fe c-aconitase                     cfd1-1 mutation mitochondrial oxidative metabolism. <br /> signal. As previously reported, 0615d/IRP1 gave easily                  For example, activity NAD+-dependent isocitrate <br /> detectable 3Fe signal, characteristic peaks g                  dehydrogenase (IDH), enzyme lacks Fe&#194;&#177;S <br /> values 2.015 2.033 (Kennedy et al., 1992; Brown                     cluster, increased (Figure 7A). We currently <br /> et al., 2002). This signal enhanced ~5-fold                         investigating basis phenomenon. In case, <br /> treatment cells 0.6 mM H2O2 (Figure 6),                         results demonstrate selective effect cfd1-1 <br /> predicted 20% c-aconitase 3Fe form                    mutation cytosolic Fe&#194;&#177;S proteins, suggest the <br />  <br /> 4830 <br />                                                                                                                 Fe&#194;&#177;S cluster assembly IRP1 <br />  <br />  <br />  <br />  <br /> Fig. 6. IRP1-transformed cfd1-1 yeast lack [3Fe&#194;&#177;4S] c-aconitase. Strains 0615d AR27 transformed IRP1 transformants grown <br /> to mid-log phase selective SD medium, harvested, prepared whole-cell EPR described Materials methods. Non-transformed yeast <br /> were grown SD-complete medium. Where indicated, cells exposed 0.6 mM H2O2 15 min prior harvest. Spectra shown the <br /> difference spectra resulting subtracting non-transformed cell spectra IRP1-transformed cell spectra. <br />  <br />  <br />  <br />  <br /> Fig. 7. Inhibition caused cfd1-1 mutation speci&#194;&#174;c cytosolic Fe&#194;&#177;S proteins. (A) Strains 0615d AR27 transformed ACO1 and <br /> transformants grown mid-log phase selective SD medium, harvested, extracts prepared assay aconitase (ACO1), succinate <br /> dehydrogenase (SDH), Leu1p, malate dehydrogenase (MDH) NAD+-dependent isocitrate dehydrogenase (IDH) described Materials and <br /> methods. (B) Protein immunoblotting performed 50 mg total extract protein using antiserum speci&#194;&#174;c mitochondrial aconitase (Aco1p) or <br /> Leu1p. <br />  <br />  <br />                                                                                                                                           4831 <br />  A.Roy et al. <br />  <br />  <br />                                                                            co-fractionation Cfd1p porin Aco1p provide <br />                                                                            strong evidence Cfd1p localized cytoplasm <br />                                                                            associated mitochondria. <br />  <br />                                                                            Discussion <br />                                                                            We utilized yeast Saccharomyces cerevisiae a <br />                                                                            model organism investigate process Fe&#194;&#177;S cluster <br />                                                                            assembly/disassembly, focusing dynamic process <br />                                                                            interconversion IRP1 c-aconitase. In IRP1- <br />                                                                            transformed aco1 yeast, conversion IRP1 c-aconitase <br />                                                                            essential growth glutamate-free medium. Using <br />                                                                            trait basis genetic screen, identi&#194;&#174;ed <br />                                                                            CFD1, previously uncharacterized gene yeast, play <br /> Fig. 8. Cfd1p localizes cytoplasm. Mid-log phase yeast sub-    crucial role cytosolic Fe&#194;&#177;S cluster assembly. Recent <br /> jected subcellular fractionation described Materials          studies, primarily yeast, identi&#194;&#174;ed number of <br /> methods. The presence individual proteins fraction          eukaryotic Fe&#194;&#177;S cluster assembly factors (Knight et al., <br /> assessed protein immunoblotting. S100, soluble cytoplasmic              1998; Land Rouault, 1998; Garland et al., 1999; <br /> material 100 000 g centrifugation; P100, material pelleted from <br /> crude cytoplasmic fraction 100 000 g centrifugation; N1, &#194;&#175;oating <br />                                                                            Kispal et al., 1999; Li et al., 1999; Schilke et al., 1999; <br /> material Nycodenz gradient; N2, light membrane vesicles; M,           Jensen Culotta, 2000; Kaut et al., 2000; Pelzer et al., <br /> mitochondria-enriched fraction; N4, large aggregates heavy mem-        2000; Tong Rouault, 2000; Lange et al., 2001; Voisine <br /> brane vesicles. The fractionation patterns shown NS1/CFD1         et al., 2001; Muhlenhoff et al., 2002; Sipos et al., 2002). <br /> (Cfd1p), 0615d/myc tagged-IRP1 (IRP1) AR27/ACO1 transformed            These factors localized predominantly mitochondria <br /> with 6HisCFD1 (Aco1p porin; note 6HisCfd1p showed a <br /> similar fractionation <span id='am-240' about='xsp:pattern' typeof='owl:Thing'>pattern</span> native Cfd1p). Relative fraction   comprise cluster assembly machinery is <br /> of S100 material loaded, ~10-fold material P100, N1          conserved prokaryotes (Zheng et al., 1998; Muhlenhoff <br /> N2, 100-fold material M N4 loaded. The              Lill, 2000). Cfd1p &#194;&#174;rst cytosolic Fe&#194;&#177;S cluster <br /> results shown representative &#194;&#174;ve independent experiments.           assembly factor identi&#194;&#174;ed. <br />                                                                               Cfd1p shares sequence homology large family <br />                                                                            P-loop ATPases falls subfamily P-loop <br /> <span id='am-162' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-163' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span>lethal phenotype</span> associated CFD1 deletion                         ATPases include NifH, Fe-protein nitrogenase <br /> relate localized function.                                         (Dean et al., 1993; Leipe et al., 2002). NifH itself an <br />                                                                            Fe&#194;&#177;S protein; however, cluster-coordinating residues <br /> Subcellular localization Cfd1p                                          NifH conserved Cfd1p. Cfd1p more <br /> The Fe&#194;&#177;S cluster assembly factors identi-                   closely related bacterial Mrp proteins, have <br /> &#194;&#174;ed date eukaryotes predominantly                       recently shown play role Fe&#194;&#177;S cluster <br /> mitochondria, minor fractions                     assembly (Figure 4; Skovran Downs, 2003). <br /> found subcellular compartments. To determine                  Therefore, role P-loop ATPases Fe&#194;&#177;S cluster <br /> intracellular location Cfd1p, subcellular fractions                biogenesis conserved evolu- <br /> prepared log phase yeast analyzed                         tion. Interestingly, homolog Cfd1p yeast, called <br /> presence Cfd1p various organelle marker                      Nbp35, localized primarily nucleus (Vitale et al., <br /> proteins (see Materials methods). Cfd1p abundant                   1996), raising possibility function Fe&#194;&#177;S <br /> in fractions containing cytoplasm (S100 N1                          cluster assembly there. That eukaryotes evolved <br /> Figure 8; N1 fraction contains residual soluble material               unique Fe&#194;&#177;S cluster assembly machineries distinct <br /> from cytoplasm, &#194;&#175;oats Nycodenz                            subcellular compartments underscores importance of <br /> gradient) present lesser extent high-                  Fe&#194;&#177;S proteins varied aspects cell physiology. <br /> speed pellet cytoplasm (P100) fraction                        Mutations genes encoding mitochon- <br /> containing light organelles membranes (N2). Based                   drial cluster assembly factors inhibit <br /> the percentage subcellular fraction loaded                biogenesis clusters mitochondrial and <br /> onto gel experiment, estimate &gt;90%                  cytosolic Fe&#194;&#177;S proteins, prompting proposal all <br /> Cfd1p fractionated cytosolic S100 (see Figure 8                       Fe&#194;&#177;S cluster assembly occurs mitochondria (Lill and <br /> legend). Importantly, Cfd1p absent fractions                 Kispal, 2000). This led view preassembled <br /> were rich mitochondria (M, Figure 8). The presence                   Fe&#194;&#177;S clusters exported mitochondria insertion <br /> the mitochondrial proteins Aco1p porin                          cytosolic proteins. At time, CFD1 three <br /> mitochondrial fraction, absence cytosolic                   genes identi&#194;&#174;ed genetically affect cytosolic Fe&#194;&#177;S <br /> S100 N1 fractions (Figure 8) con&#194;&#174;rms ef&#194;&#174;cacy                    proteins exclusively. The genes, ATM1 and <br /> this fractionation regimen. Further, small                   ERV1, encode mitochondrial inner-membrane ABC-type <br /> porin P100 fraction, absence S100,                 transporter mitochondrial inter-membrane space <br /> indicates minimal breakage mitochondria occurred                   sulfhydryl oxidase, respectively. These proteins are <br /> during cell fractionation procedure. As expected,                      thought act export Fe&#194;&#177;S clusters and/or cluster <br /> IRP1 present exclusively fractions containing                       components mitochondria cytosol (Kispal <br /> cytoplasm (Figure 8), Leu1p showed cytoplasmic                    et al., 1999; Lange et al., 2001). Since Cfd1p does not <br /> localization experiments (not shown). The similar                 stably associate mitochondria, does appear <br /> fractionation patterns Cfd1p IRP1 lack                  likely directly involved export cluster <br />  <br /> 4832 <br />                                                                                                        Fe&#194;&#177;S cluster assembly IRP1 <br />  <br />  <br /> components organelle. Interestingly, small           Materials methods <br /> percentage Cfd1p fractionated subcellular <br /> organelles membranes. At present time,         Media, strains growth conditions <br />                                                                  The 0615d yeast strain (MATa, ura3-52, trp1-D63, his3-D200, aco1-1, <br /> clear small fraction Cfd1p associated         ade2, IDP2up) used study described <br /> speci&#194;&#174;cally subcellular compartment     (Brown et al., 1998; Narahari et al., 2000). Yeasts were <br /> result cytosolic contamination fractions. In         routinely transformed plasmid DNA using lithium acetate <br /> either case, Cfd1p membrane-enriched            method (Ito et al., 1983). Transformed yeast cells grown 30&#194;&#176;C in <br />                                                                  minimal medium supplemented 2% dextrose (SD; Sherman, 1991) <br /> fractions represents &lt;10% total Cfd1p yeast cells,     lacking speci&#194;&#174;c nutrients necessary selection maintenance <br /> with remaining &gt;90% Cfd1p soluble                 speci&#194;&#174;c plasmids. For testing aconitase function vivo, glutamate was <br /> cytoplasmic protein.                                             omitted medium. Non-transformed yeast grown rich <br />    The ability convert IRP1 c-aconitase extracts     YPD medium containing 2% dextrose, SD-complete medium, as <br />                                                                  indicated (Sherman, 1991). URA3-containing plasmids shuf&#194;&#175;ed out <br /> cfd1-1 yeast iron leaves open question         ura3 yeast selection SD-complete medium supplemented with <br /> what required mitochondria cytosolic Fe&#194;&#177;S            5-&#194;&#175;uoroorotic acid (5-FOA; Sikorski Boeke, 1991). Yeast were <br /> cluster assembly. We favor view Cfd1p functions         transformed IRP1 carried 2 mm plasmids, pYADFRP <br /> to make iron available cytosolic Fe&#194;&#177;S cluster assembly,      (Narahari et al., 2000) pYESHIS-IRP. These plasmids identical <br />                                                                  URA3 gene marker pYADFRP replaced a <br /> whether iron exported mitochondria                HIS3 gene marker pYESHIS-IRP. The level IRP1 expression <br /> acquired cellular pools. An intriguing possi-         obtained plasmids particular yeast strain was <br /> bility Cfd1p iron chaperone, ferrying iron     indistinguishable. <br /> cytosolic Fe&#194;&#177;S cluster biogenesis. We speculate the <br /> CX2CX2C motif Cfd1p metal binding site        Mutagenesis <br />                                                                  IRP1-transformed 0615d grown overnight SD medium lacking <br /> process. Alignment Cfd1p amino acid sequence         uracil (SD-ura) OD600 0.8&#194;&#177;1.0. Cells pelleted, washed and <br /> that homologous proteins structural informa-        treated EMS described (Lawrence, 1991). EMS <br /> tion available predicts CX2CX2C motif             treatment gave ~40&#194;&#177;50% viability. Mutagenized cells plated onto <br /> surface exposed (Figure 4). By analogy known                  YPD solid medium cells allowed grow 30&#194;&#176;C colonies <br />                                                                  appeared. Cells replica-plated SD-glu SD-leu media. <br /> metallochaperones (Arnesano et al., 2002), expect             Colonies failed grow grew poorly SD-glu medium, <br /> that metal binding element iron chaperone        grew poorly SD-leu YPD media picked tested <br /> be near protein surface. That change of    directly growth glutamate-free SD-ura medium. AR27/IRP1 was <br /> the &#194;&#174;rst cysteine residues CX2CX2C motif           picked displayed severe growth defect glutamate-free <br /> serine impaired Cfd1p activity consistent         medium, showed growth defects leucine-free medium YPD. <br /> metal binding role motif.                               Library screening <br />    Cytosolic aconitase IRP1-transformed 0615d exists       AR27/IRP1 transformed wild-type yeast genomic library <br /> both active [4Fe&#194;&#177;4S] (80%) inactive [3Fe&#194;&#177;4S] forms           cloned YCp50 (average insert size equal 12 kb; kindly provided by <br /> (20%; Brown et al., 2002). The 3Fe form enzyme         Andy Dancis, University Pennsylvania). To identify cells been <br /> a relatively long-lived intermediate oxidant-initiated    transformed library clones complement strain&apos;s <br />                                                                  glutamate-dependent growth restriction, ~105 transformants plated <br /> Fe&#194;&#177;S cluster disassembly process, detected         SD medium lacking uracil, histidine glutamate incubated at <br /> cells EPR (Brown et al., 2002). The cfd1-1 strain lacks       30&#194;&#176;C. Three colonies appeared 5 days growth. A large <br /> any detectable 3Fe c-aconitase transformed             majority colonies acquired genomic clone containing the <br /> IRP1. This suggests little [4Fe&#194;&#177;4S] c-aconitase     mitochondrial aconitase gene, ACO1, rescue AR27 from <br />                                                                  glutamate auxotrophy. To identify clones picked wild-type <br /> ever formed IRP1-transformed cfd1-1 yeast                 complement relevant mutant gene AR27, 300 colonies were <br /> provides strong support view Cfd1p essential     replica-plated glutamate-free medium supplemented cell- <br /> for novo assembly Fe&#194;&#177;S clusters cytosol.            impermeant iron chelator ferrozine. Addition cell-impermeant iron <br />    The ability convert IRP1 c-aconitase vitro        chelators glutamate-free medium strongly inhibits growth yeast that <br />                                                                  dependent IRP1 aconitase activity, ACO1+ yeast are <br /> incubation iron suggests IRP1 expressed       resistant inhibition (Narahari et al., 2000). Two <br /> cfd1-1 yeast contains bound sulfur. One possibility      transformants highly sensitive ferrozine. These transformants <br /> is sul&#194;&#174;de held IRP1 protein-bound              carried genomic fragment BET1&#194;&#177;PAN1 region yeast <br /> polysul&#194;&#174;de (Cys-SSxS-Cys). Apo-mitochondrial aconitase           chromosome IX, subjected analysis described in <br />                                                                  Results. <br /> generated oxidant-mediated Fe&#194;&#177;S cluster disassembly <br /> retains sul&#194;&#174;de, apparently Cys-SSxS-Cys (Kennedy          PCR cloning CFD1, gene disruption site-directed <br /> Beinert, 1988). It suggested formation      mutagenesis <br /> polysul&#194;&#174;des mitochondrial aconitase            To clone CFD1 gene, yeast genomic DNA prepared previously <br /> apo-protein structure Fe&#194;&#177;S cluster      described (Narahari et al., 2000) used ampli&#194;&#174;cation CFD1 <br />                                                                  genes polymerase chain reaction (PCR) using Expand High <br /> ef&#194;&#174;ciently reconstituted iron (Kennedy            Fidelity PCR System (Roche), according manufacturer&apos;s recom- <br /> Beinert, 1988). Given Fe&#194;&#177;S cluster assembly IRP1         mendation. The CFD1 gene plus 456 bp upstream 168 bp downstream <br /> is acutely responsive cellular iron status, believe   ORF ampli&#194;&#174;ed using following primer set: 3F, 5&#194;&#162;- <br /> IRP1 behaves similarly mitochondrial aconitase        ATGGAAGAAAGAGCCGTGAA-3&#194;&#162; (5&#194;&#162; end); 3R, 5&#194;&#162;-CACTTGAA- <br />                                                                  GAACATGCTGCA-3&#194;&#162; (3&#194;&#162; end). This yielded 1506 bp CFD1 and <br /> form polysul&#194;&#174;de bonds absence iron;            cfd1-1 genes used complementation studies described text. A <br /> being ready accept excess iron cytoplasm. By           940 bp fragment comprising CFD1 cfd1-1 ORF was <br /> making iron available cluster assembly, Cfd1p          ampli&#194;&#174;ed using following primers: 2F, 5&#194;&#162;-AGCACCCAAAACTAA- <br /> play key role regulation IRP1 animals.            TACGAC-3&#194;&#162; (nucleotides &#194;&#177;35 &#194;&#177;15 relative ORF); 2R, 5&#194;&#162;- <br />                                                                  ACGTACATGCATAGGAGAGGA-3&#194;&#162; (nucleotides +905 +885 <br /> Nevertheless, results support view Cfd1p plays      relative ORF). <br /> a critical indispensable role cytosolic Fe&#194;&#177;S cluster         The ORF extends nucleotide 1 882. The DNA fragments <br /> assembly.                                                        resulting PCR ampli&#194;&#174;cations initially cloned pGEM using <br />  <br />  <br />                                                                                                                                    4833 <br />  A.Roy et al. <br />  <br /> the pGEM-T Easy Vector System I (Promega) following manufac-             protein porin detected monoclonal antibody (Molecular <br /> turer&apos;s recommendation. A total wild-type mutant PCR           Probes). Aco1p Leu1p detected polyclonal antiserum <br /> clones (four 940 bp 1506 bp products each) sequenced     speci&#194;&#174;c protein. <br /> UIC DNA Sequencing Facility. For transformation yeast, wild-type <br /> or mutant 1506 bp CFD1 genes cloned pRS313 pRS316 <br /> (Sikorski Hieter, 1989). Three independent wild-type mutant PCR <br /> clones CFD1 tested complementation assays.                        Acknowledgements <br />    The chromosomal CFD1 gene disrupted strain 0615d one-           The authors thank David Ucker critical reading manuscript and <br /> step gene disruption (Rothstein, 1991). TRP1 cloned          Violeta Chavez technical assistance. We thank Andy Dancis for <br /> unique ClaI sites 940 bp CFD1 ORF. The resulting cfd1::TRP1       generous gift yeast genomic library Aco1p antiserum, <br /> gene ampli&#194;&#174;ed PCR using 2F 2R primer set. This PCR            Roland Lill gift Leu1p antiserum acknowledge the <br /> product used transform 0615d, extra copy         assistance Bob Lee Protein Research Laboratory UIC <br /> wild-type CFD1 carried pRS316 (pRS316-CFD1). TRP+ trans-                  Research Resources Center anti-Cfd1p antibody production. This work <br /> formants tested growth medium containing 5-FOA                supported grant DK47281 National Institutes Health to <br /> identify cells chromosomal CFD1 gene disrupted.        W.E.W. We acknowledge use facilities National <br /> Since CFD1 gene essential yeast, transformants disruption   Biomedical ESR Center Medical College Wisconsin, was <br /> of chromosomal CFD1 gene grow media containing              supported National Institutes Health Grant RR01008. <br /> 5-FOA loss pRS316-CFD1 lethal. Disruption the <br /> chromosomal copy CFD1 gene con&#194;&#174;rmed PCR <br /> using primers speci&#194;&#174;c chromosomal gene. The resulting strain was <br /> called NS1/CFD1. To replace pRS316-CFD1 plasmid disrupted             References <br /> strains plasmids carrying wild-type CFD1 cfd1-1, NS1/ <br />                                                                              Arnesano,F., Banci,L., Bertini,I., Cio&#194;&#174;-Baffoni,S., Molteni,E., Huffman, <br /> CFD1 transformed genes HIS3 plasmid HIS+ <br />                                                                                 D.L. O&apos;Halloran,T.V. (2002) Metallochaperones metal- <br /> transformants plated media containing 5-FOA. Colonies that <br />                                                                                 transporting ATPases: comparative analysis sequences and <br /> grew medium collected transformed IRP1 for <br />                                                                                 structures. Genome Res., 12, 255&#194;&#177;271. <br /> further study. <br />    6HisCfd1p generated subcloning CFD1 ORF            Beinert,H. Kiley,P.J. (1999) Fe&#194;&#177;S proteins sensing regulatory <br /> NdeI EcoRI sites pET-28a (Novagen). This placed CFD1 ORF             functions. Curr. Opin. Chem. Biol., 3, 152&#194;&#177;157. <br /> downstream in-frame sequence encoding His6                 Brown,N.M., Anderson,S.A., Steffen,D.W., Carpenter,T.B., Kennedy, <br /> polypeptide. For expression yeast, XbaI&#194;&#177;SalI fragment containing          M.C., Walden,W.E. Eisenstein,R.S. (1998) Novel role of <br /> the 6HisCFD1 fusion gene subcloned downstream yeast ADH1             phosphorylation Fe&#194;&#177;S cluster stability revealed phospho- <br /> promoter pRS313, generating p6HisCFD1. All site-directed mutagen-            mimetic mutations Ser-138 iron regulatory protein 1. Proc. <br /> esis performed using p6HisCFD1 Transformer Site-directed            Natl Acad. Sci. USA, 95, 15235&#194;&#177;15240. <br /> Mutagenesis kit (Clontech) following manufacturer&apos;s protocol,        Brown,N.M., Kennedy,M.C., Antholine,W.E., Eisenstein,R.S. and <br /> was con&#194;&#174;rmed sequencing.                                                     Walden,W.E. (2002) Detection [3Fe&#194;&#177;4S] cluster intermediate of <br />                                                                                 cytosolic aconitase yeast expressing iron regulatory protein 1. <br /> Whole-cell EPR analysis                                                         Insights mechanism Fe&#194;&#177;S cluster cycling. J. Biol. Chem., <br /> Whole-cell EPR analysis performed described (Brown             277, 7246&#194;&#177;7254. <br /> et al., 2002) Bruker ELEXSYS series spectrometer                 Cook,R.A. Sanwal,B.D. (1969) Isocitrate dehydrogenase (NAD- <br /> temperature 10&#194;&#177;11K using standard methods. Conditions spectro-            speci&#194;&#174;c) Neurospora crassa. Methods Enzymol., 13, 42&#194;&#177;47. <br /> scopy follows: microwave power, 1 mW; microwave frequency,           Cordell,S.C. Lowe,J. (2001) Crystal structure bacterial cell <br /> 9.45 GHz; modulation amplitude, 10G; modulation frequency, 100 kHz;             division regulator MinD. FEBS Lett., 492, 160&#194;&#177;165. <br /> time constant, 0.082 s; scanning time, 83 s. Each spectrum represents     Dean,D.R., Bolin,J.T. Zheng,L. (1993) Nitrogenase metalloclusters: <br /> average scans. Subtraction spectrum             structures, organization synthesis. J. Bacteriol., 175, 6737&#194;&#177;6744. <br /> performed using software supplied spectrometer. Where               Diekert,K., Kroon,A.I., Kispal,G. Lill,R. (2001) Isolation and <br /> indicated, yeast treated H2O2 15 min growth media              subfractionation mitochondria yeast Saccharomyces <br /> prior harvest EPR analysis.                                              cerevisiae. Methods Cell Biol., 65, 37&#194;&#177;51. <br />                                                                              Eisenstein,R.S. (2000) Iron regulatory proteins molecular control <br /> Enzyme assays protein analysis                                              mammalian iron metabolism. Annu. Rev. Nutr., 20, 627&#194;&#177;662. <br /> Preparation yeast cytoplasmic extract, aconitase assay protein        Entian,K.D. et al. (1999) Functional analysis 150 deletion mutants in <br /> immunoblotting performed described (Brown et al.,             Saccharomyces cerevisiae systematic approach. Mol. Gen. <br /> 1998; Narahari et al., 2000). Leu1p assays performed using freshly         Genet., 262, 683&#194;&#177;702. <br /> prepared extracts citraconic acid substrate described    Ferro-Novick,S., Newman,A.P., Groesch,M., Ruohola,H., Rossi,G., <br /> (Kohlhaw, 1988). SDH, IDH MDH enzyme assays performed                  Graf,J. Shim,J. (1991) An analysis BET1, BET2 BOS1. <br /> on freshly prepared whole-cell extracts (SDH) mitochondrial extracts         Three factors mediating ER Golgi transport yeast. Cell Biophys., <br /> (IDH MDH) following published methods (Cook Sanwal, 1969;               19, 25&#194;&#177;33. <br /> Williamson Corkey, 1969; Robinson et al., 1991). All enzyme assays       Gangloff,S.P., Marguet,D. Lauquin,G.J. (1990) Molecular cloning of <br /> were carried triplicate representative           yeast mitochondrial aconitase gene (ACO1) evidence a <br /> independent experiments. When comparing enzyme activities               synergistic regulation expression glucose plus glutamate. Mol. <br /> wild-type mutant strains, yeast cultures grown day         Cell. Biol., 10, 3551&#194;&#177;3561. <br /> to OD600 0.7&#194;&#177;0.8.                                                      Garland,S.A., Hoff,K., Vickery,L.E. Culotta,V.C. (1999) <br />    Subcellular fractionation yeast performed following               Saccharomyces cerevisiae ISU1 ISU2: members well- <br /> procedure Diekert et al. (2001). Crude, whole-cell extracts             conserved gene family iron-sulfur cluster assembly. J. Mol. Biol., <br /> centrifuged 12 000 g 10 min 4&#194;&#176;C separate mitochondria              294, 897&#194;&#177;907. <br /> and organelles (P12) crude cytoplasmic fraction (S12).        Giaever,G. et al. (2002) Functional pro&#194;&#174;ling Saccharomyces <br /> The S12 separated S100 P100 centrifugation 100 000 g         cerevisiae genome. Nature, 418, 387&#194;&#177;391. <br /> for 2 h 4&#194;&#176;C. Mitochondria membrane vesicles P12          Hayashi,I., Oyama,T. Morikawa,K. (2001) Structural functional <br /> were separated Nycodenz density gradient fractions             studies MinD ATPase: implications molecular recognition <br /> collected containing residual soluble material (N1), light membrane             bacterial cell division apparatus. EMBO J., 20, 1819&#194;&#177;1828. <br /> vesicles (N2), mitochondria (M) aggregated heavy membrane            Hentze,M.W. Kuhn,L.C. (1996) Molecular control vertebrate iron <br /> vesicles (N4). All fractions generated protocol               metabolism: mRNA-based regulatory circuits operated iron, nitric <br /> analyzed presence Cfd1p marker proteins. Cfd1p             oxide oxidative stress. Proc. Natl Acad. Sci. USA, 93, 8175&#194;&#177;8182. <br /> was detected polyclonal anti-peptide antibody raised            Ito,H., Fukuda,Y., Murata,K. Kimura,A. (1983) Transformation of <br /> KLH-conjugated DKHEQPQIESPK Cfd1p sequence. The                    intact yeast cells treated alkali cations. J. Bacteriol., 153, 163&#194;&#177; <br /> IRP1 used analysis functional fusion myc epitope          168. <br /> tag C-terminus detected anti-myc epitope              Jensen,L.T. Culotta,V.C. (2000) Role Saccharomyces cerevisiae <br /> monoclonal antibody (Sigma). The outer mitochondrial membrane                   ISA1 ISA2 iron homeostasis. Mol. Cell. Biol., 20, 3918&#194;&#177;3927. <br />  <br />  <br /> 4834 <br />                                                                                                                    Fe&#194;&#177;S cluster assembly IRP1 <br />  <br /> Johnson,M.K. (1998) Iron-sulfur proteins: new roles old clusters.           localization leucine biosynthetic enzymes yeast. J. <br />   Curr. Opin. Chem. Biol., 2, 173&#194;&#177;181.                                          Bacteriol., 116, 222&#194;&#177;225. <br /> Kaut,A., Lange,H., Diekert,K., Kispal,G. Lill,R. (2000) Isa1p        Sakai,N., Yao,M., Itou,H., Watanabe,N., Yumoto,F., Tanokura,M. and <br />   component mitochondrial machinery maturation cellular           Tanaka,I. (2001) The three-dimensional structure septum site- <br />   iron-sulfur proteins requires conserved cysteine residues             determining protein MinD Pyrococcus horikoshii OT3 in <br />   function. J. Biol. Chem., 275, 15955&#194;&#177;15961.                                   complex Mg-ADP. Structure, 9, 817&#194;&#177;826. <br /> Kennedy,M.C. Beinert,H. (1988) The state cluster SH S2-         Schilke,B., Voisine,C., Beinert,H. Craig,E. (1999) Evidence a <br />   aconitase cluster interconversions removal. A convenient           conserved iron metabolism mitochondria of <br />   preparation apoenzyme. J. Biol. Chem., 263, 8194&#194;&#177;8198.                     Saccharomyces cerevisiae. Proc. Natl Acad. Sci. USA, 96, 10206&#194;&#177; <br /> Kennedy,M.C., Emptage,M.H., Dreyer,J.L. Beinert,H. (1983) The               10211. <br />   role iron activation&#194;&#177;inactivation aconitase. J. Biol. Chem.,   Sherman,F. (1991) Getting started yeast. Methods Enzymol., 194, 3&#194;&#177; <br />   258, 11098&#194;&#177;11105.                                                             21. <br /> Kennedy,M.C., Mende-Mueller,L., Blondin,G.A. Beinert,H. (1992)            Sikorski,R.S. Boeke,J.D. (1991) In vitro mutagenesis plasmid <br />   Puri&#194;&#174;cation characterization cytosolic aconitase beef liver       shuf&#194;&#175;ing: cloned gene mutant yeast. Methods Enzymol., 194, <br />   relationship iron-responsive element binding protein.          302&#194;&#177;318. <br />   Proc. Natl Acad. Sci. USA, 89, 11730&#194;&#177;11734.                                 Sikorski,R.S. Hieter,P. (1989) A shuttle vectors yeast <br /> Kispal,G., Csere,P., Prohl,C. Lill,R. (1999) The mitochondrial              host strains designed ef&#194;&#174;cient manipulation DNA in <br />   proteins Atm1p Nfs1p essential biogenesis cytosolic Fe/        Saccharomyces cerevisiae. Genetics, 122, 19&#194;&#177;27. <br />   S proteins. EMBO J., 18, 3981&#194;&#177;3989.                                         Sipos,K., Lange,H., Fekete,Z., Ullmann,P., Lill,R. Kispal,G. (2002) <br /> Knight,S.A., Sepuri,N.B., Pain,D. Dancis,A. (1998) Mt-Hsp70                 Maturation cytosolic iron-sulfur proteins requires glutathione. <br />   homolog, Ssc2p, required maturation yeast frataxin                 J. Biol. Chem., 277, 26944&#194;&#177;26949. <br />   mitochondrial iron homeostasis. J. Biol. Chem., 273, 18389&#194;&#177;18393.           Skovran,E. Downs,D.M. (2003) Lack ApbC ApbE protein <br /> Kohlhaw,G.B. (1988) Isopropylmalate dehydratase yeast. Methods             results defect Fe&#194;&#177;S cluster metabolism Salmonella enterica <br />   Enzymol., 166, 423&#194;&#177;429.                                                       serovar typhimurium. J. Bacteriol., 185, 98&#194;&#177;106. <br /> Land,T. Rouault,T.A. (1998) Targeting human iron-sulfur              Theil,E.C. Eisenstein,R.S. (2000) Combinatorial mRNA regulation: <br />   cluster assembly enzyme, nifs, different subcellular compartments          iron regulatory proteins iso-iron-responsive elements (Iso-IREs). <br />   regulated alternative AUG utilization. Mol. Cell, 2, 807&#194;&#177;          J. Biol. Chem., 275, 40659&#194;&#177;40662. <br />   815.                                                                        Tong,W.H. Rouault,T. (2000) Distinct iron-sulfur cluster assembly <br /> Lange,H., Lisowsky,T., Gerber,J., Muhlenhoff,U., Kispal,G. Lill,R.          complexes exist cytosol mitochondria human cells. <br />   (2001) An essential function mitochondrial sulfhydryl oxidase          EMBO J., 19, 5692&#194;&#177;5700. <br />   Erv1p/ALR maturation cytosolic Fe/S proteins. EMBO Rep.,          Vitale,G., Fabre,E. Hurt,E.C. (1996) NBP35 encodes essential <br />   2, 715&#194;&#177;720.                                                                   evolutionary conserved protein Saccharomyces cerevisiae with <br /> Lawrence,C.W. (1991) Classical mutagenesis techniques. Methods                  homology superfamily bacterial ATPases. Gene, 178, 97&#194;&#177;106. <br />   Enzymol., 194, 273&#194;&#177;281.                                                     Voisine,C., Cheng,Y.C., Ohlson,M., Schilke,B., Hoff,K., Beinert,H., <br /> Leipe,D.D., Wolf,Y.I., Koonin,E.V. Aravind,L. (2002) Classi&#194;&#174;cation          Marszalek,J. Craig,E.A. (2001) Jac1, mitochondrial J-type <br />   evolution P-loop GTPases related ATPases. J. Mol. Biol.,           chaperone, involved biogenesis Fe/S clusters in <br />   317, 41&#194;&#177;72.                                                                   Saccharomyces cerevisiae. Proc. Natl Acad. Sci. USA, 98, 1483&#194;&#177;1488. <br /> Li,J., Kogan,M., Knight,S.A., Pain,D. Dancis,A. (1999) Yeast              Walker,J.E., Saraste,M., Runswick,M.J. Gay,N.J. (1982) Distantly <br />   mitochondria <br /> </body></html>