<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Zhang et al. Genome Biology 2010, 11:R81 <br /> http://genomebiology.com/2010/11/8/R81 <br />  <br />  <br />  <br />  <br />  RESEARCH                                                                                                                                   Open Access <br />  <br /> Multiple distinct small RNAs originate the <br /> same microRNA precursors <br /> Weixiong Zhang1,2*&#226;&#8364;&#160;, Shang Gao3&#226;&#8364;&#160;, Xuefeng Zhou1, Jing Xia1, Padmanabhan Chellappan3, Xiang Zhou1, <br /> Xiaoming Zhang3, Hailing Jin3* <br />  <br />  <br />  <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />  Background: MicroRNAs (miRNAs), originate precursor transcripts stem-loop structures, are <br />  essential gene expression regulators eukaryotes. <br />  Results: We report 19 miRNA precursors Arabidopsis yield multiple distinct miRNA-like RNAs addition <br />  miRNAs miRNA*s. These miRNA precursor-derived miRNA-like RNAs arranged phase form <br />  duplexes approximately two-nucleotide 3&#226;&#8364;&#8482;-end overhang. Their production depends same <br />  biogenesis pathway sibling miRNAs does require RNA-dependent RNA polymerases RNA <br />  polymerase IV. These miRNA-like RNAs methylated, associated Argonaute proteins. <br />  Some miRNA-like RNAs differentially expressed response bacterial challenges, more <br />  abundant cognate miRNAs. Computational expression analyses demonstrate these <br />  miRNA-like RNAs potentially functional target protein-coding genes silencing. The function of <br />  miRNA-like RNAs supported target cleavage products published small <br />  RNA degradome data. Our systematic examination public small-RNA deep sequencing data additional <br />  plant species (Oryza sativa, Physcomitrella patens, Medicago truncatula Populus trichocarpa) animals <br />  (Homo sapiens, Mus musculus, Caenorhabditis elegans Drosophila) shows miRNA-like RNAs exist <br />  broadly eukaryotes. <br />  Conclusions: We demonstrate multiple miRNAs derive miRNA precursors sequential processing <br />  Dicer Dicer-like proteins. Our results suggest pool miRNAs larger previously <br />  recognized, miRNA-mediated gene regulation broader complex previously thought. <br />  <br />  <br /> Background                                                                         structures recognized RNase III-type <br /> MicroRNAs (miRNAs) small regulatory RNAs                                  enzymes, Drosha (as animals) Dicer Dicer-like <br /> play fundamental role gene expression regulation                           proteins (DCLs; plants), sequentially cleave <br /> eukaryotes mRNA cleavage, RNA degradation,                                 precursors liberate miRNA-miRNA* duplexes <br /> translation inhibition, DNA methylation [1-7]. miR-                             hairpins (miRNA* small RNA the <br /> NAs belong large repertoire regulatory small                               opposite arm miRNA hairpin partial <br /> RNAs, includes small interfering RNAs (siR-                             complementarity miRNA) [3,6,14]. The mature <br /> NAs) [8-11]. Most miRNA genes (MIR) transcribed                                miRNAs subsequently incorporated Argonaute <br /> by RNA polymerase II (Pol II) [12,13]. The resulting sin-                          (AGO) family proteins, target mRNAs <br /> gle-stranded miRNA precursors fold stem-loop                                  perfect partially complementary <span id='am-5' about='xsp:base' typeof='owl:Thing'>base</span> pairing <br />                                                                                    [15]. miRNAs normally abundant miR- <br /> * Correspondence: weixiong.zhang@wustl.edu; hailing.jin@ucr.edu                    NA*s [3,6,14], cases miRNA* <br /> &#226;&#8364;&#160; Contributed equally                                                              sequences abundant interact with <br /> 1 <br />  Department Computer Science Engineering, Washington University <br /> in Saint Louis, Campus Box 1045, Saint Louis, MO 63130, USA <br />                                                                                    AGO proteins exert function [16]; the <br /> 3 <br />  Department Plant Pathology Microbiology, Center Plant Cell             abundances miRNAs miRNA*s comparable, <br /> Biology, Institute Integrative Genome Biology, 900 University Ave,             called miR-5p miR-3p, depending their <br /> University California, Riverside, CA 92521, USA <br /> Full list author information available end article <br />                                                                                    positions relative 5&#226;&#8364;&#8482;-end sequences [17,18]. <br />  <br />                                      &#194;&#169; 2010 Zhang et al.; licensee BioMed Central Ltd. This open access article distributed terms Creative Commons <br />                                      Attribution License (http://creativecommons.org/licenses/by/2.0), permits unrestricted use, distribution, reproduction in <br />                                      medium, provided original work properly cited. <br />  Zhang et al. Genome Biology 2010, 11:R81                                                                    Page 2 18 <br /> http://genomebiology.com/2010/11/8/R81 <br />  <br />  <br />  <br />  <br /> Arabidopsis contains Dicer-like proteins, DCL1        partners similar miRNA-miRNA* duplexes a <br /> DCL4. The biogenesis Arabidopsis miRNAs depends            two-nucleotide 3&#226;&#8364;&#8482;-end overhang arranged in <br /> mainly DCL1, relying DCL4            phasing. Moreover, depend the <br /> [8,19]. Arabidopsis miRNAs stabilized 3&#226;&#8364;&#8482;-         biogenesis pathway known miRNAs. <br /> end methylation RNA methyltransferase HEN1,            Furthermore, multiple lines evidence suggest that <br /> which protects uridylation subsequent           miRNA-like RNAs authentic miRNAs. <br /> RNA degradation [20,21].                                      First, differentially expressed upon <br />   In contrast miRNAs, siRNAs derived dou-         bacterial challenges, abundant than <br /> ble-stranded RNA molecules multiple sources          sibling miRNAs. Second, miRNA- <br /> of origin [6,8]. Four classes siRNAs       like RNAs associated AGO proteins. Third, <br /> in plants. The class includes natural antisense tran-   predicted protein-coding targets <br /> script (nat)-siRNA derived cis-natural          similar functions, target clea- <br /> antisense transcripts, so-called nat-siRNAs. They     vage products present performing parallel ana- <br /> often induced abiotic biotic stresses, gener-      lysis RNA ends (PARE) degradome data [36-38]. <br /> ated DCL1 and/or DCL2, dependent             Fourth, expression analysis using Dicer mutants further <br /> on RNA-dependent RNA polymerase (RDR) 6 Pol               supports miRNA-like RNAs silence <br /> IV [22-25]. The second class comprises endogenous             predicted target genes. Moreover, systematic <br /> trans-acting siRNAs (tasiRNAs), encoded          genome-wide survey publically available small-RNA <br /> TAS genes [8]. miRNA-mediated cleavage TAS               deep sequencing data shows miRNA-like <br /> transcript serves template RDR6 synthesize      RNAs broadly exist plants (Oryza sativa, Physcomi- <br /> double-stranded RNA, subsequently cleaved            trella patens, Medicago truncatula Populus tricho- <br /> into approximately 21-nucleotide phased tasiRNAs           carpa) animals (Homo sapiens, Mus musculus, <br /> DCL4. The class siRNAs comprises hetero-         Caenorhabditis elegans Drosophila melanogaster). <br /> chromatic siRNAs (hc-siRNAs) [10]. hc-siRNAs nor- <br /> mally arise transposon repeat regions         Results <br /> genome, silence mobile repeat elements          <span id='am-4' about='protege:TO' typeof='owl:Thing'>To</span> study role small RNAs response bacterial <br /> via DNA methylation chromatin modification. The           challenge, prepared 13 small-RNA libraries <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> formation hc-siRNAs requires DCL3, RDR2 Pol            Arabidopsis infected various Pseudomonas syringae <br /> IV. The fourth class comprises long siRNAs (lsiRNAs),         pv. tomato (Pst) DC3000 strains sequenced them <br /> which 30 40 nucleotides <span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> [26]. The bio-       using Illumina SBS deep-sequencing platform. <br /> genesis lsiRNAs requires DCL1 partially        Sequencing data collected 6 14 hours post- <br /> dependent RDR Pol IV. Therefore, effective          inoculation (hpi) 10 mM MgCl2 (mock), type III <br /> way distinguish miRNAs various siRNAs           secretion mutated strain Pst DC3000 hrcC, a <br /> examine major distinctive components bio-        virulent strain Pst DC3000 carrying vector <br /> genesis. For example, biogenesis miRNAs does           (EV), avirulent strain Pst DC3000 (avrRpt2). <br /> not require RDRs Pol IV.                                   Pst DC3000 (avrRpt2) induces hypersensitive response <br />   A structural property miRNAs precur-       (HR) Arabidopsis Col-0 carries cognate resis- <br /> sors form foldback hairpin structures. One miRNA-             tance gene RPS2 leads cell death symptoms (the <br /> miRNA* duplex typically expected arise           hypersensitive response), usually 15 16 hpi. Our <br /> miRNA precursor [3,14,27]. Nevertheless, early           samples collected 14 hpi, right hyper- <br /> work observed additional small RNAs miR-          sensitive response visualized. From total of <br /> NAs miRNA*s, small RNAs normally            24.6 million sequencing reads all <br /> considered byproducts Dicer activities      libraries, 13,985,938 reads perfectly matched Arabi- <br /> never systematically investigated [19,28-32]. Recent     dopsis genome cDNAs, 2,578,531 <br /> studies animals identified miRNA-offset RNAs (moR-         unique. After excluding reads shorter 17- <br /> NAs) chordate [33], human [34], herpesvirus        nucleotide matched tRNAs, rRNAs, small <br /> [35], biogenesis possible functions      nuclear RNAs (snRNAs), small nucleolar RNAs <br /> small RNAs remain determined. In deep-sequen-         (snoRNAs), remaining reads kept further <br /> cing-based study small RNAs bacterial-chal-           analysis. We detected expression 191 207 <br /> lenged Arabidopsis thaliana, identified substantial      Arabidopsis miRNAs listed miRBase. The 13 libraries <br /> number sequencing reads map perfectly             sequencing reads deposited NCBI <br /> onto miRNA precursors            Gene Expression Omnibus (GEO) database [GEO: <br /> correspond mature miRNA miRNA*                      GSE19694] summary sequencing data is <br /> sequences. Most small RNAs form pairing              given Table S1 Additional file 1. <br />  Zhang et al. Genome Biology 2010, 11:R81                                                                                            Page 3 18 <br /> http://genomebiology.com/2010/11/8/R81 <br />  <br />  <br />  <br />  <br /> Multiple distinct miRNA-like RNAs arise single                     including evolutionarily conserved young non- <br /> miRNA precursor                                                           conserved miRNAs (Table 1). The sequences 35 <br /> A key observation sequencing data                        newly identified small RNAs blasted against <br /> multiple unique small-RNA reads generated                        Pst DC3000 genome, homologue &gt; 30% <br /> from miRNA precursor. Specifically,                     identity them. This result means <br /> a substantial number reads originated                    9.1% (19 207) known Arabidopsis <br /> double-stranded stem regions miRNA precur-                        miRNA precursors produce type small RNA. <br /> sors mature miRNA                       Table 1 lists miRNA precursors corre- <br /> miRNA* sequences. In cases, number                      sponding miRNA-like RNAs identified small <br /> small-RNA reads comparable greater                     RNA sequence libraries. As shown table, pre- <br /> the number reads mapped mature miRNA                         cursor (that is, pre-miR822) generate as <br /> miRNA* sequences. Furthermore, using set strin-                      distinct miRNA-like RNAs (with seven having more <br /> gent criteria (see Materials methods), observed                    reads) sides stem-loop struc- <br /> that sequencing reads map miRNA pre-                       ture (Figure 3a). Additional file 2 displays alignment <br /> cursor arranged phase, unique small-                     sequencing reads precursors, Table 1 <br /> RNA reads followed tandem some-                         includes numbers sequencing reads these <br /> times separated gap 21 22 nucleotides                     miRNA-like RNAs. <br /> along precursor [39]. Figures 1, 2, 3, 4                        In rest section, provide slew geno- <br /> this type phasing <span id='am-6' about='xsp:pattern' typeof='owl:Thing'>pattern</span> precursors                         mic molecular evidence these <br /> miR159a, miR169 m, miR319a/b, miR447, miR822                          miRNA-like RNAs authentic functional miR- <br /> miR839. It important note                        NAs. Following miRNA nomenclature [17,18], name <br /> such miRNA-like RNA appear fold-back struc-                      miRNA-like RNAs miRn.k, integer n speci- <br /> ture. In total, using minimum 5 sequencing reads                     fies particular miRNA family precursor (for exam- <br /> as cutoff, identified 35 miRNA-like RNAs 19                     ple, 159a miR159a) integer k denotes specific <br /> miRNA precursors 10 Arabidopsis miRNA families,                        miRNA miRNA-like RNA. To minimize possible <br />  <br />  <br />  <br />  <br />  Figure 1 Four miRNA precursors generate multiple miRNA-like RNAs. (a-d). Three miRNA precursors miRNA-like RNAs the <br />  upper arms close loops hairpins (a, c, d), miRNA precursor miRNA-like RNAs lower arm hairpin (b). Note <br />  miRNA-miRNA* duplexes miRNA-like RNAs, approximately two-nucleotide 3&#226;&#8364;&#8482;-end overhangs, appear miR159, miR169m and <br />  miR319b precursors. The previously annotated miRNAs named miRn.1 (see main text detail) miRNA-like RNAs having less <br />  reads named. For clarity, miR169m.2* miR319b.2* indicated numbers reads mapped them <br />  cutoff threshold 5. <br />  Zhang et al. Genome Biology 2010, 11:R81                                                                                        Page 4 18 <br /> http://genomebiology.com/2010/11/8/R81 <br />  <br />  <br />  <br />  <br />  Figure 2 miRNA-like RNAs miR447a precursor. (a) The precursor fold-back structure sequencing reads mapped miR447a.1 <br />  (that is, miR447a) individual miRNA-like RNAs. For clarity, miR447a.3* miR447a.1* indicated numbers reads <br />  mapped cutoff threshold 5. (b) Distribution sequencing reads precursor. (c) Expression miR447a.1 <br />  miR447a.3 various Arabidopsis mutants small RNA pathways. (d) Expression miR447a.3, miR447a.1, miR447a.2-3p the <br />  challenge Pst strains (hrcC, avrRpt2 EV) mock control 6 14 hpi. <br />  <br />  <br /> confusion, reserve miRn.1 known miRNA,                       opposite arm precursor fold-back struc- <br /> and newly identified miRNA-like RNAs                        ture, analogous pairing partnership of <br /> miRn.2, miRn.3, on, starting 5&#226;&#8364;&#8482;-end                  miRNA miRNA*. More critically, pairing part- <br /> the miRNA precursor. Following notation                         ners typically approximately two-nucleotide 3&#226;&#8364;&#8482;- <br /> miRNA*, miRNA-like RNA opposite                             end overhang, reflects RNase III activities [39]. <br /> miRNA-like RNA (miRn.k), lower abundance                     For example, miR159a.2-5p paired miR159a.2-3p <br /> than latter, labeled miRn.k*. However,                 two-nucleotide 3&#226;&#8364;&#8482;-end overhang (Figure 1a). Simi- <br /> abundances miRn.k miRn.k* comparable,                   lar examples miRNA precur- <br /> are named miRn.k-5p miRn.k-3p, depending                      sor structures shown Figures 1, 2, 3, 4. <br /> their relative positions [17,18]. For example,                  Second importantly, miRNA-like <br /> miRNA-like RNAs miR159a precursor                           RNAs generated biogenesis pathway as <br /> passed selection criteria labeled miR159a.2-                 cognate miRNAs. We experimentally studied some <br /> 5p, miR159a.3 miR159a.2-3p, respectively, starting                  miRNA-like RNAs miR447a miR822 pre- <br /> from 5&#226;&#8364;&#8482;-end precursor (Figure 1a).                           cursors using various mutants small RNA pathway <br />                                                                         components. As shown Figure 2c, accumulation <br /> The identified miRNA-like RNAs generated                     miR447a (which renamed miR447a.1) <br /> miRNA biogenesis pathway                                                miR447a.3 depended DCL1. The biogenesis of <br /> The newly identified miRNA-like RNAs known                      mature miR822 (that is, miR822.1) miR822.2 <br /> miRNAs share common characteristics. First,                  depended DCL4 (Figure 3c), consistent with <br /> individual miRNA-like RNA pairing partner                   previously published results [19]. Therefore, miR447a.3 <br />  Zhang et al. Genome Biology 2010, 11:R81                                                                                            Page 5 18 <br /> http://genomebiology.com/2010/11/8/R81 <br />  <br />  <br />  <br />  <br />  Figure 3 miRNA-like RNAs miR822 precursor. (a) The precursor fold-back structure sequencing reads mapped miR822.1 <br />  (that is, miR822) individual miRNA-like RNAs. (b) Distribution sequencing reads precursor. For clarity, miR822.5* indicated <br />  number reads mapped cutoff threshold 5. (c) Expression miR822.1 miR822.2 various Arabidopsis <br />  mutants small RNA pathways. (d) Expression miR822.2, miR822.1, miR822.3, miR822.1* miR822.3* challenge Pst <br />  strains (hrcC, avrRpt2 EV) mock control 6 14 hpi. <br />  <br />  <br /> and miR822.2 generated Dicer-like                        production miR447a.3 miR822.2 did need <br /> proteins cognate miRNAs. tasiRNAs endo-                      RDR proteins, Pol IV, Pol V DCL3. Therefore, <br /> genous phased siRNAs generated RDR6 DCL4                           small RNAs generated miRNA <br /> [40]. miR447a.3 miR822.2 did require RDR (Fig-                    pathway sequential DCL cleavages long hair- <br /> ures 2c 3c), ruled possibility                     pin stem regions; surely siRNAs. <br /> these phased miRNA-like RNAs tasiRNAs.                             Third, examined effect HEN1 these <br /> Furthermore, determine miRNA-like                        miRNA-like RNAs. In plants, small RNAs, including <br /> RNAs hc-siRNAs, examined accumu-                        miRNAs, siRNAs lsiRNAs, methylated their <br /> lation mutants RDR2, DCL3 largest subu-                     3&#226;&#8364;&#8482;-ends HEN1 [21,26]. Methylation stabilizes the <br /> nits Pol IV (NRPD1) Pol V (NRPE1),                       small RNAs distinguishes RNA degrada- <br /> required hc-siRNA formation function                              tion products. The accumulation miR447a.3 and <br /> [8,10,11,15]. As shown Figures 2c 3c,                          miR822.2 dependent HEN1 (Figure 5), indicating <br />  Zhang et al. Genome Biology 2010, 11:R81                                                                                             Page 6 18 <br /> http://genomebiology.com/2010/11/8/R81 <br />  <br />  <br />  <br />  <br />  Figure 4 miRNA-like RNAs miR839 precursor. (a) The precursor fold-back structure sequencing reads corresponding the <br />  individual miRNA-like RNAs miR839.1 (that is, miR839) miR839.1* (that is, miR839*). Note miR839.1*, miR839.2 miR839.3 more <br />  sequencing reads miR839.1. For clarity, miR839.3* number reads mapped cutoff threshold 5. (b) <br />  Distribution sequencing reads precursor. (c) Expression miR839.1, miR839.2 miR839.3 challenge Pst strains <br />  (hrcC, avrRpt2 EV) mock control 6 14 hpi. <br />  <br />  <br /> that small RNAs methylated. Collectively,                       non-pathogenic strain Pst DC3000 hrcC. However, <br /> these results miRNA-like RNAs pro-                     virulent strain Pst DC3000 EV induce only <br /> duced miRNA pathway cognate                           miR447a.3 miR447a.2-3p. Neither Pst DC3000 <br /> known miRNAs.                                                              EV Pst DC3000 hrcC induced miR447a (that is, <br />                                                                            miR447a.1). In addition, miR447a.1 expressed a <br /> The identified miRNA-like RNAs differentially                          lower level miR447a.2-3p miR447a.3. Similarly, <br /> expressed                                                                  miR822.3 induced Pst DC3000 EV Pst <br /> To investigate potential functions newly iden-                  (avrRpt2) 6 hpi, strains tested 14 <br /> tified miRNA-like RNAs pathogen response,                            hpi, miR822.2 induced Pst <br /> examined expression using North-                       (avrRpt2) 14 hpi. miR822.3* barely detected <br /> ern blotting. We miRNA-like                         conditions (Figure 3d). miR839.2 and <br /> RNAs profiled, homologue                        miR839.3 induced Pst (avrRpt2) 14 hpi <br /> identity &gt; 30% bacterial genome, differen-                     expressed low level condi- <br /> tially expressed challenge different strains                  tions, miR839.1 constitutively expressed at <br /> of Pst, exhibited different expression patterns                   similar level conditions (Figure 4c). <br /> their cognate miRNAs miRNA*s (Figures 2d, 3d                          The identified miRNA-like RNAs differ- <br /> 4c). As shown Figure 2d, instance,                             entially expressed different tissues. One example <br /> miR447a.2-3p miR447a.3 strongly induced                        seen comparing results miR839 <br /> the avirulent strain Pst (avrRpt2) weakly induced                   precursor Figure 4a Figure 2b [19]. <br />  Zhang et al. Genome Biology 2010, 11:R81                                                                   Page 7 18 <br /> http://genomebiology.com/2010/11/8/R81 <br />  <br />  <br />  <br />  <br /> Table 1 Nineteen Arabidopsis miRNA precursors 10 miRNA families generate total 35 miRNA-sibling RNAs <br /> MIR             Loci    miRNA ID           miRNA sequence               Reads       AGO        PARE <br /> miR159a           +     miR159a.1*         GAGCTCCTTAAAGTTCAAACA          9 <br />                         miR159a.2-5p       AGCTGCTAAGCTATGGATCCC          36         2,7 <br />                         miR159a.3          TAAAAAAGGATTTGGTTATA           6 <br />                         miR159a.2-3p       ATTGCATATCTCAGGAGCTTT          9         1,2,7      At5g24620 <br />                         miR159a.1*         TTTGGATTGAAGGGAGCTCTA         6,587 <br /> miR519b           +     miR159b.2          AGCTGCTAAGCTATGGATCCC          36          7 <br />                         miR159b.2*         ATGCCATATCTCAGGAGCTTT          14        1,2,7 <br />                         miR159b.1          TTTGGATTGAAGGGAGCTC *         2481 <br /> miR168a           +     miR168a.1          TCGCTTGGTGCAGGTCGGGAA        266,020 <br />                         miR168a.2          ATTGGTTTGTGAGCAGGGATTGGAT      10          2 <br />                         miR168a.1*         CCCGCCTTGCATCAACTGAAT         1,497 <br /> miR169b           -     miR169b.2          TGAAGTGGAGTAGAGTATAATG         7                    At4g17420 <br />                         miR169b.1          CAGCCAAGGATGACTTGCCGG         4,444 <br />                         miR169b.1*         GGCAAGTTGTCCTTCGGCTACA         8 <br /> miR169f           -     miR169f.2          TGAAGGAATAACGAATGGAAT         108          1 <br />                         miR169f.1          TGAGCCAAGGATGACTTGCCG         5,779 <br />                         miR169f.1*         GCAAGTTGACCTTGGCTCTGC         2,505 <br /> miR169i           -     miR169i.2-5p       TGAATAGAAGAATCATATTTGG         32 <br />                         miR169i.1          TAGCCAAGGATGACTTGCCTG        44,477 <br />                         miR169i.1*         GGCAGTCTCCTTGGCTATC           360 <br />                         miR169i.2-3p       TTATATGTTCTTCTCTTTCATC         9                    At5g02710 <br /> miR169j           -     miR169j.1          TAGCCAAGGATGACTTGCCTG        44,458 <br />                         miR169j.1*         AATCTTGCGGGTTAGGTTTCA          9 <br />                         miR169j.2          GGCAGTCTCCTTGGCTATC           224          4        At5g48300 <br /> miR169l           -     miR169l.1          TAGCCAAGGATGACTTGCCTG        44,392 <br />                         miR169l.1*         AATCTTGCGGGTTAGGTTTCA          9 <br />                         miR169l.2          AGGCAGTCTCTTTGGCTATC          366 <br /> miR169m           -     miR169m.2          TGAATAGAAGAATCATATTTGG         32 <br />                         miR169m.1          TAGCCAAGGATGACTTGCCTG        39,650 <br />                         miR169m.1*         GGCAGTCTCCTTGGCTATC           361 <br /> miR169n           -     miR169n.2*         TGGCGGAAAGCGTCATGTTTAG         10          4 <br />                         miR169n.1          TAGCCAAGGATGACTTGCCTG        44,458 <br />                         miR169n.1*         AATCTTGCGGGTTAGGTTTCA          9 <br />                         miR169n.2          AGGCAGTCTCTTTGGCTATC          366 <br /> miR319a           +     miR319a.2          AATGAATGATGCGGTAGACAAA         8        1,2,4,5 <br />                         miR319a.1          TTGGACTGAAGGGAGCTCCCT          27 <br /> miR319b           +     miR319b.1*         GAGCTTTCTTCGGTCCACTC           28 <br />                         miR319b.2          AATGAATGATGCGAGAGACAA         491         1,2 <br />                         miR319b.1          TTGGACTGAAGGGAGCTCCCT          30 <br /> miR447a           -     miR447a.2-5p       ACCCCTTACAATGTCGAGTAA         106        2,4,5 <br />                         miR447a.1          TTGGGGACGAGATGTTTTGTTG        198 <br />                         miR447a.2-3p       ACTCGATATAAGAAGGGGCTT          94       1,2,4,5,7 <br />                         miR447a.3          TATGGAAGAAATTGTAGTATT          96       1,2,4,5,7 <br /> miR447b           -     miR447b.1*         AGTAAACGAAGCATCTGTCCCC         8 <br />                         miR447b.1          TTGGGGACGAGATGTTTTGTTG        198 <br />                         miR447b.2          ACTCGATATAAGAAGGGGCTT          94        2,5,7 <br />                         miR447b.3          TATGGAAGAAATTGTAGTATT          96        2,4,7 <br /> miR775            +     miR775.1*          GCACTACGTGACATTGAAAC            8 <br />                         miR775.2           TTTGGTTTGTTCAAAGACATT          10          5 <br />                         miR775.1           TTCGATGTCTAGCAGTGCCA          3,136 <br /> miR822           +/-    miR822.2*          CGACCTTAAGTATAAGTAGAT          6 <br />  Zhang et al. Genome Biology 2010, 11:R81                                                                                                         Page 8 18 <br /> http://genomebiology.com/2010/11/8/R81 <br />  <br />  <br />  <br />  <br /> Table 1 Nineteen Arabidopsis miRNA precursors 10 miRNA families generate total 35 miRNA-sibling RNAs <br /> (Continued) <br />                              miR822.3*              GATGTAACGCATGTTGTTTTCT                              149            2,4,7 <br />                              miR822.1               TGCGGGAAGCATTTGCACATGT                             4,153 <br />                              miR822.4-5p            TTTCGTGGAGAATGAAATCAC                               10              1,4       At1g62030, At2g04680 <br />                              miR822.5               CATACATGAATAATAATTACC                                9              1,5 <br />                              miR822.4-3p            TATGATTTTATCCTCCATAAAA                              11               5 <br />                              miR822.1*              ATGTGCAAATGCTTTCTACAG                               693 <br />                              miR822.3               AAACAATATACGTTGCATCCC                              1,691          1,2,4,7 <br />                              miR822.2               ATCTACTTACACTTAAGGTCG                               363           1,2,4,5 <br /> miR839              +/-      miR839.2               TCATGTGAGCAGAAAGAGTAG                               10               1 <br />                              miR839.1               TACCAACCTTTCATCGTTCCC                                5 <br />                              miR839.3               TGCAAAACCGTGATAGTGCTGA                              13            1,2,4,7     At1g65960 <br />                              miR839.1*              GAACGCATGAGAGGTTGGTAAA                              33 <br /> miR841               -       miR841.2               TGTTCTTAAGTTGCTTGTGAA                                8               1 <br />                              miR841.1               TACGAGCCACTTGAAACTGAA                               59 <br />                              miR841.1*              ATTTCTAGTGGGTCGTATTCA                              3,904 <br /> miR846               +       miR846.1*              CATTCAAGGACTTCTATTCAG                               59 <br />                              miR846.2               AATTGGATATGATAAATGGTAA                              34 <br />                              miR846.2*              ACTTTTATCATATCCCATCAG                               18 <br />                              miR846.1               TTGAATTGAAGTGCTTGAATT                               37 <br /> Nineteen Arabidopsis miRNA precursors (the MIR column) 10 miRNA families generate total 35 miRNA-sibling RNAs (the miRNA ID miRNA sequence <br /> columns). For particular precursor, positions newly identified miRNA-like RNAs relative cognate miRNA indicated &#226;&#8364;&#732;Position&#226;&#8364;&#8482; column, <br /> where plus sign (+) means miRNA-sibling small RNAs (msRNAs) upper arm hairpin close loop, minus sign (-) indicates they <br /> reside lower portion near base hairpin, &#226;&#8364;&#732;+/-&#226;&#8364;&#8482; means multiple msRNAs appear sides sibling miRNA, included <br /> here. The cognate (known) miRNAs named miRn.1 (see main text). The Argonaute proteins miRNA-like RNA associate listed the <br /> &#226;&#8364;&#732;AGO&#226;&#8364;&#8482; column. The &#226;&#8364;&#732;PARE&#226;&#8364;&#8482; column lists mRNA genes miRNA-like RNA corresponding small RNA target product PARE/degradome <br /> datasets considered. <br />  <br />  <br /> The peak reads deep-sequencing data                                 gap 2 shown Figure 2b [19]. A major differ- <br /> [19] exhibited phasing pattern, agree-                         ence deep-sequencing datasets that <br /> ment deep-sequencing data (Figures 4a,b;                                 total RNA extracted seedlings, flowers, <br /> Additional file 2). It important note                               rosette leaves, siliques [19], used only <br /> sequencing read small-RNA libraries mapped                              matured rosette leaves profiling. <br /> gap 2 Figure 4a, sequencing reads                                As final note expression levels, these <br />                                                                                   miRNA-like RNAs abundant their <br />                                                                                   cognate miRNAs (Table 1). For example, miR319b.2 has <br />                                                                                   491 reads miR319b (that is, miR319b.1) 30 <br />                                                                                   reads (Table 1 Figure 1d), 10- <br />                                                                                   fold difference. Similarly, miR839.2 miR839.3 <br />                                                                                   reads miR839 (that is, miR839.1) (Figure <br />                                                                                   4a). It possible miRNA-like RNAs <br />                                                                                   induced certain developmental stages or <br />                                                                                   specific conditions regulate gene expression. <br />  <br />                                                                                   The identified miRNA-like RNAs potentially functional <br />                                                                                   We present pieces evidence that <br />                                                                                   newly identified miRNA-like RNAs have <br />                                                                                   functional mRNA targets. First, miRNA- <br />                                                                                   like RNAs identified associated AGO <br />                                                                                   proteins. In general, miRNAs loaded AGO pro- <br />                                                                                   teins silence target genes RNA cleavage, RNA <br />   Figure 5 Accumulation miR447a.3 miR822.2 mutant                     degradation, translation inhibition. Thus, searched <br />   small-RNA methyltransferase gene HEN1. WT, wild type.                    Arabidopsis datasets AGO-associated small RNAs <br />   U6, control, shows sRNA equal loading.                                      [41,42] miRNA-like RNAs identified. We found <br />  Zhang et al. Genome Biology 2010, 11:R81                                                                  Page 9 18 <br /> http://genomebiology.com/2010/11/8/R81 <br />  <br />  <br />  <br />  <br /> that 25 (71.4%) 35 miRNA-like RNAs 14             glycosyl hydrolase family. Different miRNA-like <br /> (73.7%) 19 precursors associated 5       RNAs miRNA precursor tar- <br /> AGO proteins (AGO column Table 1). In particular,        gets gene family. One pronounced exam- <br /> 14, 15, 12, 8 11 miRNA-like RNAs asso-           ple miR822 precursor (Figure 4a). Three <br /> ciated AGO1, AGO2, AGO4, AGO5 AGO7,                miRNA-like RNAs (miR822.3*, miR822.4-5p, and <br /> respectively. This result suggests iden-   miR822.5), cognate miR822 <br /> tified miRNA-like RNAs potentially function             (miR822.1), potentially target total 60 distinct <br /> through AGO proteins. Further, nucleotide    DC1 domain containing proteins, are <br /> small RNA critical determining AGO pro-        targeted multiple miRNA-like RNAs. Interestingly, <br /> teins associate consequently dictate    miRNA-like RNAs different miRNA families may <br /> its mode operation [41,42]. The nucleotides     targets protein family. For exam- <br /> the unique sequences 35 miRNA-like RNAs          ple, miR159a.2-3p, miR169j.2, miR319a.2, miR447a.3, <br /> preferentially A (45.7% total) U (42.9%),        miR447b.3, miR822.4-5p, miR839.2 targets <br /> which account nearly 90% total. As compar-     leucine-rich repeat family. These relationships <br /> ison, 75.8% 12.6% known Arabidopsis miR-         miRNA-like RNAs targets are <br /> NAs start U A, respectively. Although    reminiscent miRNAs targets, also <br /> nucleotides shifted preferential U miRNAs      allude possible origins inverted gene dupli- <br /> a nearly equal preference U A 35 miRNA-       cation [30,44]. In short, experimental compu- <br /> like RNAs, U A dominant         tational results indicate miRNA-like RNAs <br /> nucleotides miRNAs miRNA-like RNAs.             identified potential silence target <br />    Second, miRNA-like RNAs identified           genes, common related <br /> have putative mRNA targets coherent func-         functions. <br /> tions. We predicted putative targets using tar-     Third, miRNA-like RNAs mediate target <br /> get prediction method version 2 CleaveLand        silencing mRNA cleavage. Since identified <br /> software analyzing small RNA degradomes [43].           miRNA-like RNAs characteristics as <br /> With alignment score cutoff 4.5, total 33        miRNAs associated AGO pro- <br /> (94.3%) 35 miRNA-like RNAs identified           teins, hypothesized directly <br /> putative targets (Table S2 Additional file 1). We rea-   cleave mRNA targets. To test hypothesis, we <br /> soned miRNA-like RNAs silence       searched for, using version 2 CleaveLand degra- <br /> target genes, de-suppression targets        dome software [43], small RNA target signatures of <br /> expected Dicer mutants, miRNA-like          mRNA cleavage products data Arabidopsis <br /> RNAs longer produced. Thus, exam-            PARE small RNA degradomes collected three <br /> ined, using real-time RT-PCR, expression        labs different tissues various condi- <br /> of predicted targets miR169i.2-3p (At5g02710),       tions [36-38]. With alignment-score cutoff 4.5 <br /> miR169j.2 (At5g48300), miR447a.3 (At1g54710             P-value threshold 0.2, small RNA <br /> At1g06770), miR839.2 (At4g31210), miR839.3              cleavage products seven mRNA genes targeted by <br /> (At1g65960) dcl1-9 mutant wild type         miRNA-like RNAs identified (miR159a.2- <br /> (Figure 6a), predicted target miR822.4-     3p, miR169b.2, miR169i.2-3p, miR169j.2, miR822.4-5p, <br /> 5p (At1g62030) dcl4-2 mutant wild           miR839.3; PARE column Table 1). Detailed <br /> type (Figure 6b). Indeed, targets accumu-        information miRNA-like RNAs tar- <br /> lated higher level mutants wild     gets supported degradome data Table S3 <br /> type studied (Figures 6a,b). Further,       Additional file 1; alignments of these <br /> miR447a.3 miR839.2 induced Pst                  pairs miRNA-like RNAs targets, with <br /> (avrRpt2), examined expression         pairs tested, shown Figure 6d. <br /> three target genes Pst (avrRpt2) treatment.       Furthermore, miRNA-like RNAs <br /> As shown Figure 6c, targets repressed         (miR159a.2-3p, miR169j.2, miR822.4-5p, miR839.3) <br /> during Pst (avrRpt2) challenge, showing negative cor-     associated AGO proteins (Table 1), <br /> relation expression corresponding           indicating that, mechanistically, small RNAs can <br /> miRNA-like RNAs. Furthermore, similar miR-          function canonical miRNA pathway. <br /> NAs, miRNA-like RNAs identified target             Indeed, ablation miRNA-like <br /> multiple protein-coding genes (Table S2 Additional       RNAs (miR169i.2-3p, miR169j.2, miR839.3) the <br /> file 1). In addition, miRNA-like RNAs       dcl1-9 mutant miR822.4-5p dcl4-2 <br /> have multiple targets common closely related        mutant led elevated expression tar- <br /> functions. For example, miR775.2 targets genes       gets (Figures 6a,b). The relatively small number the <br />  Zhang et al. Genome Biology 2010, 11:R81                                                                                         Page 10 18 <br /> http://genomebiology.com/2010/11/8/R81 <br />  <br />  <br />  <br />  <br />  Figure 6 Negative correlation expression selected miRNA-like RNAs targets. (a) The expression targets of <br />  miR169i.2-3p, miR169j.2, miR447a.3, miR839.2 miR839.3 dcl1-9 mutant relative Ler wild type (WT), measured realtime <br />  RT-PCR. (b) The expression level miR822.4-5p target dcl4-2 mutant relative Col-0 wide type. (c) The expression two <br />  miR447a.3 targets miR839.2 target challenge Pst DC3000 (avrRpt2) relative mock treatment. Actin used <br />  internal control delta Ct calculation. Error bars correspond standard deviation data independent reactions. The <br />  experiments repeated sets biological samples similar results obtained. (d) Alignments selected miRNA-like RNAs <br />  mRNA targets expression compared Dicer mutants wide type (a, b) bacterial infection <br />  mock infection (c). Included alignment scores P-values target signatures miRNA-like RNAs target degradation <br />  products small RNA degradome datasets. The arrows target cleavage sites detected degradome data. <br />  <br />  <br /> miRNA-like RNAs mRNA cleavage products                         Distribution miRNA-like RNAs precursor fold- <br /> may reasons. First, miRNA-like                         structures <br /> RNAs typically expressed low abundance; thus,                    A remarkable characteristic miRNA-like RNAs <br /> their cleavage products low detected.                     Arabidopsis appear on <br /> Second, different tissues used experiments                   known miRNA-miRNA* duplex a <br /> (mature leaves) PARE data collection (floral                     precursor hairpin close base or <br /> tissues, including inflorescence meristem early                  loop hairpin. Two miRNA-like RNAs <br /> stage floral buds, EIN5 mutant). This tissue differ-                 reside sides miRNA-miRNA* <br /> ence explain target cleavage product                    duplex. A summary location distribution the <br /> was detected known miRNAs listed                        miRNA-like RNAs given &#226;&#8364;&#732;Position&#226;&#8364;&#8482; column of <br /> Table 1 (miR447a.1/b.1, miR822.1 miR839.1)                     Table 1, plus sign (+) means miRNA-like <br /> the expression miRNAs miRNA-like RNAs                          RNAs appear exclusively miRNA-miRNA* and <br /> often tissue-specific. Nevertheless, degradome ana-                 loop hairpin, minus sign (-) indicates that <br /> lysis provided evidence miRNA-like                      miRNA-like RNAs occur exclusively miRNA- <br /> RNAs identified experiments function                          miRNA* base hairpin, &#226;&#8364;&#732;+/-&#226;&#8364;&#8482; means <br /> through mRNA target cleavage.                                            miRNA-like RNAs sides the <br />  Zhang et al. Genome Biology 2010, 11:R81                                                                                                     Page 11 18 <br /> http://genomebiology.com/2010/11/8/R81 <br />  <br />  <br />  <br />  <br /> miRNA-miRNA* duplex. As shown, 19                                     Specifically, searched miRNA precursors are <br /> miRNA precursors identified, 7 harbored miRNA-like                              capable generating multiple miRNA-like RNAs in <br /> RNAs exclusively loops hairpins.                              publicly available small-RNA deep-sequencing datasets <br /> Examples include MIR159a, MIR319a MIR319b                                O. sativa (rice), P. patens (moss), M. truncatula <br /> Figures 1a,c,d, respectively. This consistent                       P. trichocarpa; Materials methods infor- <br /> recent discovery DCL cleavage produces                            mation sources datasets. In total, we <br /> mature miR159 miR319 starts loop ends                              identified 75, 37, 9 11 miRNA precursors, respec- <br /> of fold-back structures [45,46]. Another pre-                         tively, plant species harbor <br /> cursors produced miRNA-like RNAs near hairpin                               miRNA-like RNAs addition miRNAs. Table 2 lists <br /> bases - example, MIR169m MIR447a Figures                             miRNA precursors Additional files 3, 4, 5 and <br /> 1b 2a. This miRNA-like RNA distribution                             6 contain alignments miRNA-like RNAs to <br /> supported conventional model miRNA biogen-                            corresponding miRNA precursors. Note the <br /> esis [3,14], subsequent DCL cleavage activ-                        sequencing data rice moss Solexa <br /> ities produce miRNA-miRNA* duplex, releasing                            sequencing, data Medicago Populus <br /> the precursor miRNA liberating duplex.                             mixture Solexa 454 sequencing. <br /> The remaining precursors - is, MIR822                              Therefore, sequencing depths Medicago and <br /> MIR839 Figures 3a 4a - miRNA-like RNAs                               Populus deep rice moss, <br /> on sides cognate miRNA-miRNA*                                     resulting fewer miRNA-like RNA-bearing miRNA <br /> duplexes. These precursors contain long stems,                              precursors identified species. <br /> which likely processed continuous in-phase                           As shown Table 2, individual miRNA-like <br /> dicing activity DCL1 DCL4. This process similar                        RNAs identified conserved multiple plants. In the <br /> to biogenesis tasiRNAs DCL4.                                          plant species studied, miRNA-like RNAs <br />    Another interesting observation slot                       appeared well-conserved miRNA families, is, <br /> in phasing pattern precursor filled                             miR159 miR319 (Table 2). miRNA-like RNAs <br /> sequencing reads, shown MIR319b MIR839                                appeared miR319 precursors five <br /> (Figures 1d 4a). Such peculiar pattern miRNA-                          plants, miRNA-like RNAs occurred miR159 pre- <br /> like RNA expression suggests potential condition- tis-                       cursors bar moss. miR159 miR319 <br /> sue-specific expression, variable metabolism rates                       belong MIR family based evolution- <br /> these miRNA-like RNAs.                                                          ary origin [30,44], playing important roles plant <br />                                                                                 development [47]. Importantly, close inspection <br /> Conservation sequential DCL cleavage long MIR                             showed individual miRNA-like RNAs the <br /> hairpins eukaryotes                                                          miR159 miR319 precursors highly con- <br /> To determine miRNA-like RNAs                                served sequence level. Figure 6a displays the <br /> identified Arabidopsis exist plant spe-                        miR159a precursors Arabidopsis, rice, Medicago and <br /> cies, studied additional plant organisms.                               Populus Figure 6b shows miR319b precursors in <br />  <br />  <br /> Table 2 miRNA precursors miRNA-like RNA miRNA families plants animals <br /> Species           Number        Number miRNA precursors <br />                   precursors         families <br /> Arabidopsis                 19               10 159a/b, 168a, 169b/f/i/j/l/m/n, 319a/b, 447a/b, 775, 822, 839, 841, 846 <br /> Rice                        75               42 156i, 159a/b/c/d/e/f, 169a, 171i, 319a/b, 396e/f, 437, 441b, 444b/c/d, 445a/c/d/e/g/h/i, 806h, 807a, <br />                                                 809e/f, 812h/i/j, 820a/b/c, 821a/b/c, 1318, 1319, 1423b, 1427, 1428e/g, 1430, 1432, 1439, 1440, 1441, <br />                                                 1442, 1850, 1851, 1858b, 1862b/d, 1864, 1871, 1874, 1875, 1881, 1883b, 1884a/b, 2121b, 2122, 2123a/b, <br />                                                 2124a/b/c/e/f/g/h/i <br /> Moss                        37               27 160i, 319a/b/c/e, 390a/b, 408b, 533a, 534a, 535d, 536d, 538a/c, 893, 896, 898b, 899, 902k, 904a/b, <br />                                                 1023a, 1030a/f, 1047, 1051, 1063b/d/g, 1068, 1069, 1070, 1072, 1219a/d, 2079, 2083 <br /> Medicago                     9                7 159a, 169d/l, 319a/b, 1510b, 2087, 2088, 2089 <br /> Populus                     11                5 159a/b/c, 319c/d/f/g, 394a/b, 482, 1450 <br /> Human                       14               14 7-1, 15b, 34a, 124-2, 181a-2, 187, 193a, 204, 221, 331, 498, 518c, 769, 1977 <br /> Mouse                       27               21 7a-1, 20a, 24-2, 30b, 96, 101b, 138-1, 328, 329, 331, 337, 341, 434, 466l, 669a-1/-2/-3, 669b/e/m-1/m-2, <br />                                                 674, 685, 708, 1199, 2134-3 <br /> C. elegans                  17               16 51, 60, 61, 63, 64, 70, 75, 78, 80, 232, 244, 257, 258-1/-2, 353, 354, 1817 <br /> Drosophila                   9                9 33, 277, 279, 283, 964, 976, 988, 997, 999 <br /> miRNA precursors miRNA-like RNA miRNA families plant species (Arabidopsis, rice, moss, Medicago Populus) animal organisms (human, <br /> mouse, C. elegans Drosophila). Entries bold miRNA precursors produce miRNA-like RNAs species. <br />  Zhang et al. Genome Biology 2010, 11:R81                                                                      Page 12 18 <br /> http://genomebiology.com/2010/11/8/R81 <br />  <br />  <br />  <br />  <br /> Arabidopsis, rice, moss Medicago. As shown,          Discussion <br /> miRNA-like RNAs, particularly expressed               We reported 19 miRNA precursors Arabidopsis that <br /> relatively abundant level, conserved         able produce multiple distinct miRNA-like RNAs <br /> flanking sequences cognate miRNAs                potential function. Our analysis 13 libraries <br /> miRNA*s. For example, sequences miR319b.2             deep-sequencing data Arabidopsis characterized <br /> miR319b.2* comparable level conservation            important features miRNA-like RNAs. <br /> miR319b* (that is, miR319b.1*); specifically, miR319a,           First, miRNA-like RNAs opposite arms a <br /> miR319a*, miR319b.2, miR319b.2* 19, 12, 12              miRNA precursor hairpin usually form duplex a <br /> and 11 identical nucleotides species,            two-nucleotide 3&#226;&#8364;&#8482;-end overhang, key property <br /> respectively. Furthermore, miRNA-like              miRNA-miRNA* duplexes reflects activities <br /> RNAs, including miR319b.2, miR159a.2-5p,                     RNase III proteins [39]. Second, miRNA-like <br /> miR159a.2-3p, incorporated multiple AGO              RNAs arranged phase, reflecting sequential <br /> proteins Arabidopsis (Table 1). We examined          dicing activities RNase III proteins. Third, the <br /> target conservation miRNA-like RNAs                    nucleotide discovered miRNA-like RNAs has <br /> miR159a miR319b precursors Arabidopsis,             strong preference (approximately 90%) A U, <br /> dicotyledonous plant, rice, monocotyledonous               similar approximately 88% known <br /> plant. Both ath-miR159a.2-3p osa-miR159a.2-3p                miRNAs. <br /> have targets pentatricopeptide repeat (PPR) pro-             Moreover, obtained lines molecular evi- <br /> tein family (At2G40720, Os2G05720, Os10G40920 and,               dence support notion miRNA- <br /> Os5G50690). Collectively, results suggest         like RNAs identified authentic potentially <br /> these highly conserved miRNA-like RNAs plants             functional miRNAs. First, miRNA-like RNAs are <br /> functional.                                                      generated miRNA pathway, the <br />   To understand miRNA-like RNAs,                 pathways tasiRNAs hc-siRNAs (Figures 2 3). <br /> searched miRNA-like RNAs large collection            Second, miRNA-like RNAs likely methylated <br /> small-RNA deep sequencing data animal spe-             (Figure 5). Third, 25 (71.4%) 35 miRNA-like <br /> cies, H. sapiens (human), M. musculus (mouse), C. ele-           RNAs identified pools AGO-asso- <br /> gans D. melanogaster (see Materials methods).            ciated small RNAs (Table 1) [41,42], suggesting that <br /> We identified 14, 27, 17 9 miRNA precursors               potentially function AGO effec- <br /> human, mouse, C. elegans Drosophila, respectively,           tors. Fourth, miRNA-like RNAs can <br /> that produce miRNA-like RNAs addition              induce target mRNA cleavage, cleavage pro- <br /> known miRNAs miRNA*s (Table 2). These miRNA-                 ducts present data PARE database <br /> like RNA-bearing miRNA precursors distributed             small-RNA degradome (Figure 6) [36-38]. Fifth, <br /> intergenic regions introns, has-miR34a            miRNA-like RNAs identified pre- <br /> cel-miR354, reside 3&#226;&#8364;&#8482; untranslated regions.             dicted targets, potentially target <br /> These miRNA-like RNAs immediately adjacent            genes gene family common <br /> annotated miRNA-miRNA* duplexes, shown                 functions (Table S2 Additional file 1), simi- <br /> alignments originating precursor hairpins        lar miRNAs. Sixth, miRNA-like RNAs <br /> four animals Additional files 7, 8, 9, 10, respec-        abundant cognate miRNAs (Figures 1 <br /> tively. Different results plants,      4), miRNA-like RNAs differ- <br /> one small RNA pairing duplex present              entially expressed pathogen infections (Figures <br /> known miRNA-miRNA* duplex animal miRNA                   2, 3 4), suggesting potential regulatory func- <br /> precursor identified. This difference plants         tions response environmental stresses. The <br /> animals mainly relatively short miRNA pre-         results gene expression analysis using dcl1 dcl4 <br /> cursors animals. The newly identified miRNA-like              mutants wild type Arabidopsis plants showed that <br /> RNAs animals near precursor bases.               loss miR169i.2-3p, miR169j.2, miR447a.3, <br /> Notice miRNA-like RNAs miR7 precursors              miR839.2, miR839.3, miR822.4-5p elevate the <br /> in human mouse conserved (Table 2; Figure 7c;            expression predicted target mRNAs we <br /> Additional files 7 8). It interesting note   tested (Figure 6). In addition, miR447a.3 miR839.2 <br /> miR7 Drosophila diverged human           induced Pst (avrRpt2), detected <br /> mouse significantly (data shown), miR7           down-regulation targets Pst (avrRpt2) <br /> been reported C. elegans. These results indicate low        challenge (Figure 6). <br /> conservation miRNA-like RNAs based animal                 Moreover, miRNA-like RNAs exist evolu- <br /> small-RNA sequencing data examined.                           tionarily distant plant species animal <br />  Zhang et al. Genome Biology 2010, 11:R81                                                                                                 Page 13 18 <br /> http://genomebiology.com/2010/11/8/R81 <br />  <br />  <br />  <br />  <br />  Figure 7 Conservation miRNA-like RNAs miRNAs. The newly identified miRNA-like RNAs red annotated miRNAs and <br />  miRNA*s blue. Also included consensus sequences. (a) Conservation miR159a.2-5p/miR159a.2-3p miR159a.1/miR159a.1* <br />  (that is, miR159a/miR159a*) plants, A. thaliana (ath), O. sativa (osa), P. patens (ppt) M. truncatula (mtr). (b) Conservation miR319.2/ <br />  miR319b.2* miR319.1/miR319b.1* (that is, miR319/miR319b*) plants, Arabidopsis, rice, Medicago P. trichocarpa (ptc). (c) <br />  Conservation miR7-1 human miR7a-1 mouse. <br />  <br />  <br /> organisms examined - Arabidopsis, rice, moss, Medi-                          We did observe clear correlation deep <br /> cago Populus human, mouse, C. elegans                          sequencing read counts Northern blot results Ara- <br /> and Drosophila. This result suggests mechanism                       bidopsis, couple highly expressed known <br /> underlying miRNA-like RNAs conserved                            miRNAs. Similar observations some <br /> within plants animals. The fact                           recent studies [48,49]. The inconsistency sequen- <br /> these miRNA-like RNAs appear moss, ancient                              cing read counts Northern blot results likely to <br /> land plant, alludes possible evolutionary origin                     differences techniques. Sample pre- <br /> in plants. Some miRNA-like RNAs con-                            paration deep sequencing involves steps RNA <br /> served miRNA families, is, miR159 miR319                          adaptor ligation step PCR amplification, which <br /> plants, conserved sequence level (Figure 7),                       introduce bias final data. Small RNAs different <br /> which adds layer evidence                               5&#226;&#8364;&#8482;- 3&#226;&#8364;&#8482;-end structures modification differ- <br /> miRNA-like RNAs potentially functionally important                        ent efficiencies ligation, PCR tends amplify <br /> in plants. In addition, miRNA-like RNAs identi-                        highly abundant sequences efficiently less <br /> fied appear occur evolutionarily &#226;&#8364;&#732;old&#226;&#8364;&#8482; miRNA pre-                       abundant ones. On hand, hybridization-based <br /> cursors plants animals based                               Northern blotting hybridize small RNAs the <br /> publicly available small-RNA sequencing data                               sequence regardless end modification or <br /> examined.                                                                     structure, cross-hybridization side <br />  Zhang et al. Genome Biology 2010, 11:R81                                                                    Page 14 18 <br /> http://genomebiology.com/2010/11/8/R81 <br />  <br />  <br />  <br />  <br /> effect depending stringency hybridization       Conclusions <br /> conditions. In light observations, relied    To sum results in-depth genomic <br /> on conventional Northern blot analysis high strin-       molecular analysis Arabidopsis, results <br /> gency conditions quantify expression            extensive survey additional plant <br /> miRNA-like RNAs studied.                                   animal species, data early <br />   Some early studies, including using deep              studies plants [19,28-30] animals [31,32], is <br /> sequencing, observed small RNAs              evident continuously sequential cleavage Dicer <br /> miRNAs miRNA*s miRNA precursors                 Dicer-like proteins common theme, rather <br /> plants [19,28-30] animals [31,32]. For example,           exception, plants animals, gives <br /> sequencing reads miRNA-like RNAs             rise phased miRNA-like RNAs MIR genes. In <br /> the ath-miR839 precursor [19]. Interestingly,            Arabidopsis, DCL4 shown capable of <br /> miRNA-like RNAs mirtron          sequential cleavage generate phased small <br /> mouse [50]. However, small RNAs              RNAs tasiRNAs [8]. Our results suggest that <br /> plants animals analyzed regard          DCL1, generates miRNAs miRNA- <br /> their biogenesis potential functions. Chiang et al. [50]   like RNAs, act sequential cleavage <br /> argued single discovery mirtron precursor     DCL4 does. Further, limited conservation of <br /> in mouse coincidence spliceosome activity        miRNA-like RNAs evolutionarily &#226;&#8364;&#732;old&#226;&#8364;&#8482; MIR genes <br /> releasing particular mirtron precursor. Indeed,       indicates subject evolutionary selec- <br /> extensive data MIR genes animal           tion. The resulting small RNAs subject dif- <br /> plant species convincingly continuous sequen-       ferent rates degradation. Which miRNAs or <br /> tial cleavage activities common action Dicer        miRNA-like RNAs accumulate response to <br /> proteins animal species Dicer-like proteins plant    developmental and/or environmental cues be <br /> organisms. Another line related work covers            determined conservation tight <br /> moRNAs chordate, human herpesvirus                 regulation post-biogenesis protection degrada- <br /> [33-35]. Some miRNA-like RNAs imme-           tion. For example, miRNAs miRNA-like <br /> diately adjacent miRNA miRNA* located near          RNAs stabilized binding AGO proteins <br /> the base miRNA precursors classified         RNA-binding proteins, degraded <br /> moRNAs. However, moRNAs reported chor-            exoribonucleases [51]. <br /> date Ciona intestinalis fundamentally different          In summary, results extensive molecular <br /> from miRNA-like RNAs animal genomes           experiments Arabidopsis systematic examina- <br /> that studied. In particular, miRNA-like RNAs        tion large quantity deep-sequencing datasets <br /> identified miRNA precursor sequences           evolutionarily diverse plant species four <br /> within double-stranded RNA regions, ones          evolutionarily distant animal organisms suggest the <br /> we tested Arabidopsis dependent DCL1             multiple distinct miRNA-like RNAs identified <br /> DCL4. In contrast, moRNAs identified C. intes-        broadly exist eukaryotic species authentic <br /> tinalis - example, adjacent ci-miR124-1      functional miRNAs. Our results suggest that <br /> ci-miR124-2 shown Figure 2b [33,34] - span          pool miRNAs larger previously recog- <br /> substantially double-stranded stem region          nized, miRNA-mediated gene regulation be <br /> and outside miRNA precursors,           broader complex previously thought. <br /> be recognized processed Dicer. Thus, although <br /> moRNAs likely cut Drosha end,         Materials methods <br /> the end moRNAs generated            Plant materials, small-RNA library construction, deep <br /> determined [33-35]. More work required better           sequencing <br /> understand moRNAs animals, particularly bio-         Thirteen small-RNA libraries prepared 4- to <br /> genesis functions. In comparison, study showed        5-week-old short-day grown Arabidopsis Col-0 inocu- <br /> that miRNA-like RNAs Arabidopsis produced          lated 10 mM MgCl2 (mock), type III secretion sys- <br /> by sequential DCL cleavage long hairpin stem           tem mutated strain Pst DC3000 hrcC, virulent strain Pst <br /> regions generated sides             DC3000, avirulent strain Pst DC3000 (avrRpt2). <br /> miRNA-miRNA* duplex. Many incorpo-             Bacteria infiltration carried Arabidopsis <br /> rated AGO proteins, mediate target mRNA              wild-type mutant plants described previously <br /> cleavage. Some differentially expressed        [24]. Infiltrated leaves harvested 6 14 hpi. A <br /> response environmental stresses. All results      large number Arabidopsis mutants (dcl1-7, dcl1-9, <br /> suggested potential biological functions.               dcl2-1, dcl3-1, dcl4-2, hen1, rdr2-1, rdr1-1, rdr6-15, <br />  Zhang et al. Genome Biology 2010, 11:R81                                                                 Page 15 18 <br /> http://genomebiology.com/2010/11/8/R81 <br />  <br />  <br />  <br />  <br /> nrpd1-3, nrpe1-11) corresponding wild-type        Identifying miRNA-like RNAs determining their <br /> ecotypes used study. Assays performed     phasing patterns <br /> on 4-week-old plants grown 22&#194;&#176;C 12 h light.         We mapped sequence reads, allowing mismatch, <br />    Total RNA isolated using Trizol reagent (Invitro-    miRNA precursors. We retained have <br /> gen, Carlsbad, CA, USA) infiltrated leaves         reads arranged block patterns analysis. To <br /> fractionated 15% denaturing polyacrylamide gel.          end, applied Blockbluster software [33,34] to <br /> RNA molecules ranging 18 28 nucleotidess            identify blocks sequencing reads anno- <br /> were excised ligated 5&#226;&#8364;&#8482;- 3&#226;&#8364;&#8482;-RNA adaptors         tated miRNA precursors. The abundant sequence <br /> using T4 RNA ligase followed RT-PCR gel puri-        read detected block taken repre- <br /> fication described instructions Illumina     sentative sequence block. The total number of <br /> [52,53]. The small RNA libraries sequenced          sequence reads block sum copy <br /> Illumina Inc. University California, Riverside   number representative read copy num- <br /> (UCR) core facility.                                        bers sequence reads fall repre- <br />                                                             sentative sequence overlap the <br /> Processing deep sequencing data                          representative read nucleotides <br /> Raw sequence reads parsed remove 3&#226;&#8364;&#8482; adap-       representative sequence end. We <br /> tors. The sequencing reads small RNA       allowed overhangs tolerate imprecise dicing <br /> libraries, adaptors trimmed, mapped        activities RNase III enzymes. The remaining blocks <br /> Arabidopsis nuclear, chloroplast mitochondrial gen-     inspected; spanning two <br /> ome sequences cDNA sequences,            neighboring blocks ignored. Those miRNA precur- <br /> retrieved TAIR database [54]. The reads       sors blocks reads arranged phase were <br /> matched sequences mismatches (the row         kept analysis. When reporting results <br /> labeled &#226;&#8364;&#732;mapped&#226;&#8364;&#8482; Table S1 Additional file 1)     using Arabidopsis deep sequencing data, ignored <br /> retained analysis. Sequencing reads        blocks total sequencing <br /> aligned precursors annotated Arabidopsis      reads reduce potential false positive miRNA-like <br /> miRNAs miRBase [55] Novealign [56]. Those           RNAs sequencing error. <br /> sequencing reads mapped miRNA <br /> precursor zero mismatches retained            Northern blot analysis small RNAs <br /> further analysis.                                           Total RNA isolated using Trizol reagent (Invitro- <br />                                                             gen) infiltrated leaves 80 &#206;&#188;g resolved on <br /> Small RNA deep-sequencing data additional plant     15% denaturing polyacrylamide gel. RNA cross- <br /> and animal species                                     linked membrane using EDC (1-ethyl-3-(3-dimethyla- <br /> We collected small-RNA deep-sequencing data, gener-         minopropyl) carbodiimide (N-(3-Dimethylaminopropyl)- <br /> ated Illumina/Solexa 454 sequencing platforms,        N&#226;&#8364;&#8482;-ethylcarbodiimide hydrochloride (Sigma, St. Louis, <br /> for additional plants GEO database. In        MO, USA))) [65]. We used locked nucleic acid <br /> particular, analyzed total 18 small-RNA             (LNA) probe small RNA Northern blot analysis to <br /> sequencing datasets P. patens (moss) [57]; data-     detect miR839, DNA oligos detecting miR447 <br /> sets Solexa sequencing), O. sativa (rice [25];       miR822. Blots exposed phosphorscreens, <br /> another datasets, GEO accession number [GEO:            scanned using PhosphorImager (Molecular <br /> GSE14462]; Solexa sequencing), M. truncatula         Dynamics (Amersham Pharmacia Biotech Inc, Piscat- <br /> (two datasets Solexa sequencing [58]         away, NJ, USA)). <br /> datasets 454 sequencing [59], P. trichocarpa          The LNA probes used detection were: mirt839- <br /> (four datasets 454 sequencing [60,61]). Small          AT1G67481.2, c+tac+tct+ttc+tgc+tca+cat+ga; mirt839- <br /> RNA deep sequencing data collected       AT1G67481.1, g+gga+acg+atg+aaa+ggt+tgg+ta; and <br /> animal species: H. sapiens (human [62,63]; data-       mirt839-AT1G67481.3, t+cag+cac+tat+cac+ggt+ttt+gca <br /> sets); M. musculus (mouse [32]; datasets);            +aa. <br /> C. elegans (C. elegans [64]; datasets) D. mela-       We used following oligo probes miR447 and <br /> nogaster (Drosophila [16]; datasets). The initial      miR822: miR447a.1-anti, CAACAAAACATCTCG <br /> processing sequencing libraries followed       TCCCCAA; miR447a.2-3p-anti, AAGCCCCTTCTTA- <br /> same steps 13 Arabidopsis datasets. In sum-      TATCGAGTC; miR447a.3-anti, AATACTACAATTTCT <br /> mary, collected, processed analyzed total       TCCATA; miR822.1-anti, CATGTGCAAATGCTT <br /> 52 small RNA deep-sequencing datasets current        CCCGCA; miR822.1*-anti, TGTAGAAAGCATTTGCA- <br /> study.                                                      CATG; miR822.3*-anti, AGAAAACAACATGCGT <br />  Zhang et al. Genome Biology 2010, 11:R81                                                                                Page 16 18 <br /> http://genomebiology.com/2010/11/8/R81 <br />  <br />  <br />  <br />  <br /> TACATCC; miR822.3-anti, GGGATGCAACGTA- <br />                                                           Additional material <br /> TATTGTTTC; miR822.2-anti, ACGACCTTAAGTGT <br /> AAGTAGAT.                                                  Additional file 1: Tables S1, S2, S3. Table S1: number of <br />                                                            sequencing reads 13 small RNA libraries Arabidopsis seedlings at <br /> miRNA target cleavage product analysis target          6 14 hours pathogen infections, their <br />                                                            corresponding mock infection control. Shown table the <br /> prediction                                                 total number reads (total), number reads map the <br /> We applied version 2 CleaveLand software [43]    Arabidopsis genome cDNA sequences (mapped), nuclear genome <br /> the data PARE analysis small RNA degradomes        (nuclear), transposons repeats (repeats/mobile), cDNA sequences <br />                                                            (cDNAs). No mismatch allowed mapping table. The <br /> [36-38] characterize signatures mRNA cleavage    second number condition (column) percent reads to <br /> products miRNA-like RNAs studied. We        mapped reads. The sequencing reads available GEO <br /> considered target cleavage products align-     database accession number [GEO:GSE19694]. Table S2: putative <br />                                                            targets identified miRNA-like RNAs identified miRNA target <br /> ment score CleaveLand cutoff      finding method, developed Michael Axtell, CleaveLand software <br /> threshold 4.5. Putative targets miRNA-like RNAs      small RNA degradome analysis. Results alignment <br /> were predicted miRNA target finding            scores cutoff threshold 4.5. Table S3: miRNA-like RNAs <br />                                                            current study mRNA targets identified small <br /> method CleaveLand software package. This target     RNA degradome data. In table, columns list miRNA- <br /> finding method used scoring scheme charges        like RNAs targets, fifth columns list major <br /> penalty mismatch penalty 0.5     quantitative measures results, is, alignment scores, the <br />                                                            number raw reads target degradation products P-values <br /> wobble base pairing, doubles penalties    quantifying enrichment degradation products, respectively. The <br /> seed region, two- seven-nucleotide         column indicates pair miRNA-like RNA target tested <br /> region near 5&#226;&#8364;&#8482;end miRNA.                          current study. <br />                                                            Additional file 2: Supplemental File S1. This file sequencing <br />                                                            reads mapped aligned miRNA precursors produce <br /> Quantitative RT-PCR analysis small RNA targets          miRNA-sibling small RNAs (msRNAs) A. thaliana (ath). The sequencing <br /> For quantitative RT-PCR analysis, 5 &#206;&#188;g total RNA    data collected current study available GEO under <br /> used synthesizing cDNA. DNA contamination          accession number [GEO:GSE19694]. <br /> removed using DNase I (Invitrogen). Amplification    Additional file 3: Supplemental File S2. This file sequencing <br />                                                            reads mapped aligned miRNA precursors produce <br /> miRNA 169f miR447f targets (see AT numbers             miRNA-sibling small RNAs (msRNAs) rice (osa). The sequencing data <br /> below) carried using realtime PCR machine        obtained GEO; Materials methods details. <br /> (iQ5, Bio-Rad, Hercules, CA, USA). The following oligos    Additional file 4: Supplemental File S3. This file sequencing <br /> were used perform RT-PCR analysis: At5g02710        reads mapped aligned miRNA precursors produce <br />                                                            miRNA-sibling small RNAs (msRNAs) moss (ppt). The sequencing data <br /> (miR169i.2-3p)-QRTF, TCGACAACCCTGATGATGTT-                 obtained GEO; Materials methods details. <br /> GAGCTG; At5g02710(miR169i.2-3p)-QRTR, CCTTGT               Additional file 5: Supplemental File S4. This file sequencing <br /> TGAATTTCTTTTCTTCAATCC; At5g48300(miR169j.2)                reads mapped aligned miRNA precursors produce <br /> -QRTF, AATGGGAGCTGATTATTACGAGACTGC;                        miRNA-sibling small RNAs (msRNAs) Medicago (mtr). The sequencing <br />                                                            data obtained GEO; Materials methods details. <br /> At5g48300(miR169j.2)-QRTR, CTTGCACGTTGTC <br />                                                            Additional file 6: Supplemental File S5. This file sequencing <br /> GCTGTTTATGATC; At1g54710(miR447a.3)-QrtF,                  reads mapped aligned miRNA precursors produce <br /> GGTATATGATAATTTTCACAGTGTGTATACCAG;                         miRNA-sibling small RNAs (msRNAs) Populus (ptc). The sequencing <br /> At1g54710(miR447a.3)-QrtR, ATCTTCGGGGTAAAC-                data obtained GEO; Materials methods details. <br /> CATACCATTGAAT; At1g06770(miR447a.3)-QrtF, CGA-             Additional file 7: Supplemental File S6. This file sequencing <br />                                                            reads mapped aligned miRNA precursors produce <br /> GAGTGAAAATGAGATAGAGAT;                       At1g06770     miRNA-sibling small RNAs (msRNAs) H. sapiens (has). The sequencing <br /> (miR447a.3)-QrtR,      AGACAAGAACCATCGTAAA                 data obtained GEO; Materials methods details. <br /> ATCCT; At4g31210(miR839.2)-QrtF, GTTATTC-                  Additional file 8: Supplemental File S7. This file sequencing <br /> TAAAGTGTGGACCCTATGGGC;                       At4g31210     reads mapped aligned miRNA precursors produce <br />                                                            miRNA-sibling small RNAs (msRNAs) M. musculus (mms). The <br /> (miR839.2)-QrtR, CATCTTCAGGGTGGGTTCCCAG                    sequencing data obtained GEO; Materials methods <br /> TG; At1g65960(miR839.3)-QRTF, TCATTCACTCT-                 details. <br /> CAATTTCTCCAAGGGA; At1g65960(miR839.3)-QRTR,                Additional file 9: Supplemental File S8. This file sequencing <br /> TTCTCTATACCTTCTTTGAGAACCAC; At1g62030                      reads mapped aligned miRNA precursors produce <br />                                                            miRNA-sibling small RNAs (msRNAs) C. elegans (cel). The sequencing <br /> (miR822.4-5p)-QRTF, GCCTGGTTCATCTTGGA-                     data obtained GEO; Materials methods details. <br /> TAAGCTTCTCC; At1g62030(miR822.4-5p)-QRTR,                  Additional file 10: Supplemental File S9. This file sequencing <br /> TTGTAGATATTCGCGATGACCAGC.                                  reads mapped aligned miRNA precursors produce <br />                                                            miRNA-sibling small RNAs (msRNAs) D. melanogaster (dme). The <br />                                                            sequencing data obtained GEO; Materials methods <br />                                                            details. <br />  Zhang et al. Genome Biology 2010, 11:R81                                                                                                           Page 17 18 <br /> http://genomebiology.com/2010/11/8/R81 <br />  <br />  <br />  <br />  <br /> Abbreviations                                                                      16. Okamura K, Liu N, Eric C, Lai EC: Distinct mechanisms microRNA <br /> AGO: Argonaute; DCL: Dicer-like protein; EV: vector; GEO: Gene                   strand selection Drosophila Argonautes. Mol Cell 2009, 36:431-444. <br /> Expression Omnibus; hc-siRNA: heterochromatic small interfering RNA; hpi:          17. Meyers BC, Axtell MJ, Bartel B, Bartel DP, Baulcombe D, Bowman JL, Cao X, <br /> hours post-inoculation; lsiRNA: long small interfering RNA; MIR: miRNA gene;           Carrington JC, Chen X, Green PJ, Griffiths-Jones S, Jacobsen SE, Mallory AC, <br /> miRNA: microRNA; moRNA: miRNA-offset RNA; PARE: parallel analysis RNA               Martienssen RA, Poethig RS, Qi Y, Vaucheret H, Voinnet O, Watanabe Y, <br /> ends; Pol: RNA polymerase; RDR: RNA-dependent RNA polymerase; siRNA:                   Weigel D, Zhu JK: Criteria annotation plant MicroRNAs. Plant Cell <br /> small interfering RNA; tasiRNA: trans-acting small interfering RNA.                    2008, 20:3186-3190. <br />                                                                                    18. Ambros V, Bartel B, Bartel DP, Burge CB, Carrington JC, Chen X, Dreyfuss G, <br /> Acknowledgements                                                                       Eddy SR, Griffiths-Jones S, Marshall M, Matzke M, Ruvkun G, Tuschl T: A <br /> We thank David Baulcombe, Jim Carrington, Herve Vaucheret, Jian-Kang Zhu,              uniform microRNA annotation. RNA 2003, 9:277-279. <br /> Steve Jacobsen, John Clarke, Xuemei Chen, Adam Vivian-Smith Zhixin             19. Rajagopalan R, Vaucheret H, Trejo J, Bartel DP: A diverse evolutionarily <br /> Xie providing seeds various mutants; Stephen Cole Jian-Kang Zhu             fluid set microRNAs Arabidopsis thaliana. Genes Dev 2006, <br /> for commenting manuscript; David Langenberger Peter Stadler                 20:3407-3425. <br /> for Blockbuster software; Charles Addo-Quaye Michael Axtell        20. Li J, Yang Z, Yu B, Liu J, Chen X: Methylation protects miRNAs siRNAs <br /> the CleaveLand Software. This research supported NSF grant              3&#226;&#8364;&#8482;-end uridylation activity Arabidopsis. Curr Biol 2005, <br /> DBI-0743797, NIH grants RC1AR058681-01 U54AI057160-07, grant                15:1501-1507. <br /> the Alzheimer&#226;&#8364;&#8482;s Association, grant Monsanto WZ, NSF           21. Yu B, Yang ZY, Li JJ, Minakhina S, Yang MC, Padgett RW, Steward R, <br /> Career Award MCB-0642843, NIH R01GM093008-01, University                       Chen XM: Methylation crucial step plant microRNA biogenesis. <br /> California Discovery Grant Bio06-10566 AES-CE Research Allocation               Science 2005, 307:932-935. <br /> Award PPA-7517H HJ.                                                             22. Borsani O, Zhu J, Verslues PE, Sunkar R, Zhu JK: Endogenous siRNAs <br />                                                                                        derived pair natural cis-antisense transcripts regulate salt <br /> Author details                                                                         tolerance Arabidopsis. Cell 2005, 123:1279-1291. <br /> 1 <br />  Department Computer Science Engineering, Washington University             23. Jin H, Vacic V, Girke T, Lonardi S, Zhu JK: Small RNAs regulation of <br /> in Saint Louis, Campus Box 1045, Saint Louis, MO 63130, USA. 2Department               cis-natural antisense transcripts Arabidopsis. BMC Mol Biol 2008, 9:6. <br /> of Genetics, Washington University School Medicine, Campus Box 8232,            24. Katiyar-Agarwal S, Morgan R, Dahlbeck D, Borsani O, Villegas A, Zhu JK, <br /> Saint Louis, MO 63110, USA. 3Department Plant Pathology                         Staskawicz BJ, Jin HL: A pathogen-inducible endogenous siRNA plant <br /> Microbiology, Center Plant Cell Biology, Institute Integrative Genome          immunity. Proc Natl Acad Sci USA 2006, 103:18002-18007. <br /> Biology, 900 University Ave, University California, Riverside, CA 92521, USA.   25. Zhou X, Sunkar R, Jin H, Zhu JK, Zhang W: Genome-wide identification <br />                                                                                        analysis small RNAs originated natural antisense transcripts <br /> Authors&#226;&#8364;&#8482; contributions                                                                 Oryza sativa. Genome Res 2009, 19:70-78. <br /> XuefengZ HJ initiated research; HJ WZ designed                     26. Katiyar-Agarwal S, Gao S, Vivian-Smith A, Jin H: A novel class bacteria- <br /> experiments; JX, WZ, XiangZ, XuefengZ carried computational                induced small RNAs Arabidopsis. Genes Dev 2007, 21:3123-3134. <br /> experiments data analyses; SG PC performed biological                  27. Winter J, Jung S, Keller S, Gregory RI, Diederichs S: Many roads maturity: <br /> experiments; XiaomingZ participated biological experiments; WZ              microRNA biogenesis pathways regulation. Nat Cell Biol 2009, <br /> HJ analyzed integrated results, directed coordinated                   11:228-234. <br /> research wrote paper.                                                      28. Lacombe S, Nagasaki H, Santi C, Duval D, Pi&#195;&#169;gu B, Bangratz M, Breitler JC, <br />                                                                                        Guiderdoni E, Brugidou C, Hirsch J, Cao X, Brice C, Panaud O, Karlowski WM, <br /> Received: 2 June 2010 Revised: 22 July 2010 Accepted: 9 August 2010                    Sato Y, Echeverria M: Identification precursor transcripts 6 novel <br /> Published: 9 August 2010                                                               miRNAs expands diversity genomic organisation and <br />                                                                                        expression miRNA genes rice. BMC Plant Biol 2008, 8:123. <br /> References                                                                         29. Kurihara Y, Watanabe Y: Arabidopsis micro-RNA biogenesis Dicer- <br /> 1. Baulcombe D: RNA silencing plants. Nature 2004, 431:356-363.                     like 1 protein functions. Proc Natl Acad Sci USA 2004, 101:12753-12758. <br /> 2. Mallory AC, Bouche N: MicroRNA-directed regulation: cleave         30. Fahlgren N, Howell MD, Kasschau KD, Chapman EJ, Sullivan CM, Cumbie JS, <br />     cleave. Trends Plant Sci 2008, 13:359-367.                                         Givan SA, Law TF, Grant SR, Dangl JL, Carrington JC: High-throughput <br /> 3. Kim VN: MicroRNA biogenesis: coordinated cropping dicing. Nat Rev               sequencing Arabidopsis microRNAs: evidence frequent birth and <br />     Mol Cell Biol 2005, 6:376-385.                                                     death MIRNA genes. PLoS ONE 2007, 2:e219. <br /> 4. Ambros V: The functions animal microRNAs. Nature 2004, 431:350-355.          31. Ruby JG, Stark A, Johnston WK, Kellis M, Bartel DP, Lai EC: Evolution, <br /> 5. Flynt AS, Lai EC: Biological principles microRNA-mediated regulation:            biogenesis, expression, target predictions substantially <br />     shared themes amid diversity. Nat Rev Genet 2008, 9:831-842.                       expanded set Drosophila microRNAs. Genome Res 2007, 17:1850-1864. <br /> 6. Carthew RW, Sontheimer EJ: Origins mechanisms miRNAs                 32. Babiarz JE, Ruby JG, Wang Y, Bartel DP, Blelloch R: Mouse ES cells express <br />     siRNAs. Cell 2009, 136:642-655.                                                    endogenous shRNAs, siRNAs, microprocessor-independent, <br /> 7. Khraiwesh B, Arif MA, Seumel GI, Ossowski S, Weigel D, Reski R, Frank W:            Dicer-dependent small RNAs. Genes Dev 2008, 22:2773-2785. <br />     Transcriptional control gene expression microRNAs. Cell 2010,            33. Shi W, Hendrix D, Levine M, Haley B: A distinct class small RNAs arises <br />     140:111-122.                                                                       pre-miRNA-proximal regions simple chordate. Nat Struct Mol <br /> 8. Chapman EJ, Carrington JC: Specialization evolution endogenous               Biol 2009, 16:183-189. <br />     small RNA pathways. Nat Rev Genet 2007, 8:884-896.                             34. Langenberger D, Bermudez-Santana C, Hertel J, Hoffmann S, Khaitovich P, <br /> 9. Jin HL: Endogenous small RNAs antibacterial immunity plants.                 Stadler PF: Evidence human microRNA-offset RNAs small RNA <br />     Febs Lett 2008, 582:2679-2684.                                                     sequencing data. Bioinformatics 2009, 25:2298-2301. <br /> 10. Matzke M, Kanno T, Daxinger L, Huettel B, Matzke AJ: RNA-mediated              35. Umbach JL, Cullen BR: In-depth analysis Kaposi&#226;&#8364;&#8482;s sarcoma-associated <br />     chromatin-based silencing plants. Curr Opin Cell Biol 2009, 21:367-376.         herpesvirus microRNA expression provides insights mammalian <br /> 11. Vazquez F: Arabidopsis endogenous small RNAs: highways byways.                 microRNA-processing machinery. J Virol 2010, 84:695-703. <br />     Trends Plant Sci 2006, 11:460-468.                                             36. Addo-Quaye C, Eshoo TW, Bartel DP, Axtell MJ: Endogenous siRNA and <br /> 12. Lee Y, Kim M, Han J, Yeom KH, Lee S, Baek SH, Kim VN: MicroRNA genes               miRNA targets identified sequencing Arabidopsis degradome. <br />     transcribed RNA polymerase II. EMBO J 2004, 23:4051-4060.                   Curr Biol 2008, 18:758-762. <br /> 13. Zhou X, Ruan J, Wang G, Zhang W: Characterization identification        37. Gregory BD, O&#226;&#8364;&#8482;Malley RC, Lister R, Urich MA, Tonti-Filippini J, Chen H, <br />     microRNA core promoters model species. PLoS Comput Biol 2007,              Millar AH, Ecker JR: A link RNA metabolism silencing <br />     3:e37.                                                                             affecting Arabidopsis development. Dev Cell 2008, 14:854-866. <br /> 14. Bartel DP: MicroRNAs: genomics, biogenesis, mechanism, function.           38. German MA, Pillay M, Jeong DH, Hetawal A, Luo SJ, Janardhanan P, <br />     Cell 2004, 116:281-297.                                                            Kannan V, Rymarquis LA, Nobuta K, German R, De Paoli E, Lu C, Schroth G, <br /> 15. Vaucheret H: Plant ARGONAUTES. Trends Plant Sci 2008, 13:350-358.                  Meyers BC, Green PJ: Global identification microRNA-target RNA pairs <br />                                                                                        parallel analysis RNA ends. Nat Biotechnol 2008, 26:941-946. <br />  Zhang et al. Genome Biology 2010, 11:R81                                                                                                        Page 18 18 <br /> http://genomebiology.com/2010/11/8/R81 <br />  <br />  <br />  <br />  <br /> 39. MacRae IJ, Doudna JA: Ribonuclease revisited: structural insights       64. Ruby JG, Jan C, Player C, Axtell MJ, Lee W, Nusbaum C, Ge H, Bartel DP: <br />     ribonuclease III family enzymes. Curr Opin Struct Biol 2007, 17:138-145.         Large-scale sequencing reveals 21U-RNAs additional microRNAs and <br /> 40. Xie Z, Allen E, Wilken A, Carrington JC: DICER-LIKE 4 functions trans-        endogenous siRNAs C. elegans. Cell 2006, 127:1193-1207. <br />     acting small interfering RNA biogenesis vegetative phase change       65. Pall GS, Codony-Servat C, Byrne J, Ritchie L, Hamilton A: Carbodiimide- <br />     Arabidopsis thaliana. Proc Natl Acad Sci USA 2005, 102:12984-12989.              mediated cross-linking RNA nylon membranes improves the <br /> 41. Mi SJ, Cai T, Hu YG, Chen Y, Hodges E, Ni FR, Wu L, Li S, Zhou H, Long CZ,       detection siRNA, miRNA piRNA northern blot. Nucleic Acids Res <br />     Chen S, Hannon GJ, Qi YJ: Sorting small RNAs Arabidopsis                 2007, 35:e60. <br />     argonaute complexes directed 5 &#226;&#8364;&#732; terminal nucleotide. Cell <br />     2008, 133:116-127.                                                            doi:10.1186/gb-2010-11-8-r81 <br /> 42. Montgomery TA, Howell MD, Cuperus JT, Li DW, Hansen JE, Alexander AL,         Cite article as: Zhang et al.: Multiple distinct small RNAs originate <br />     Chapman EJ, Fahlgren N, Allen E, Carrington JC: Specificity                microRNA precursors. Genome Biology 2010 11:R81. <br />     ARGONAUTE7-miR390 interaction dual functionality TAS3 trans- <br />     acting siRNA formation. Cell 2008, 133:128-141. <br /> 43. Addo-Quaye C, Miller W: CleaveLand: pipeline using degradome <br />     data cleaved small RNA targets. Bioinformatics 2009, 25:130-131. <br /> 44. Allen E, Xie ZX, Gustafson AM, Sung GH, Spatafora JW, Carrington JC: <br />     Evolution microRNA genes inverted duplication target gene <br />     sequences Arabidopsis thaliana. Nat Genet 2004, 36:1282-1290. <br /> 45. Ad <br /> </body></html>