<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Influence Protein Abundance High-Throughput <br /> Protein-Protein Interaction Detection <br /> Joseph Ivanic, Xueping Yu, Anders Wallqvist, Jaques Reifman* <br /> Biotechnology HPC Software Applications Institute, Telemedicine Advanced Technology Research Center, US Army Medical Research Materiel Command, Ft. <br /> Detrick, Maryland, United States America <br />  <br />  <br />  <br />     <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />     Experimental protein-protein interaction (PPI) networks increasingly exploited diverse ways biological <br />     discovery. Accordingly, vital discern underlying natures identifying classifying various types of <br />     deterministic (specific) probabilistic (nonspecific) interactions detected. <span id='am-3' about='protege:TO' typeof='owl:Thing'>To</span> end, analyzed PPI networks <br />     determined using range high-throughput experimental techniques aim systematically quantifying any <br />     biases arise varying cellular abundances proteins. We confirm PPI networks determined using <br />     affinity purification methods yeast Eschericia coli incorporate correlation protein degree, number of <br />     interactions, cellular abundance. The observed correlations small statistically significant occur both <br />     unprocessed (raw) processed (high-confidence) data sets. In contrast, yeast two-hybrid yields networks that <br />     contain relationship. While previously commented based mRNA abundance, extensive analysis based <br />     protein abundance confirms systematic difference PPI networks determined technologies. We <br />     additionally demonstrate centrality-lethality rule, implies higher-degree proteins likely be <br />     essential, misleading, protein abundance measurements identify essential proteins prevalent than <br />     nonessential proteins. In fact, generally degree/abundance correlation, degree distributions <br />     nonessential essential proteins disparate. Conversely, degree/abundance correlation, the <br />     degree distributions nonessential essential proteins different. However, essentiality manifests <br />     itself biological property yeast PPI networks investigated enrichments interactions between <br />     essential proteins. These findings provide valuable insights underlying natures various high-throughput <br />     technologies utilized detect PPIs lead effective strategies inference analysis high- <br />     quality PPI data sets. <br />  <br />   Citation: Ivanic J, Yu X, Wallqvist A, Reifman J (2009) Influence Protein Abundance High-Throughput Protein-Protein Interaction Detection. PLoS ONE 4(6): <br />   e5815. doi:10.1371/journal.pone.0005815 <br />   Editor: Eugene I. Shakhnovich, Harvard University, United States America <br />   Received February 27, 2009; Accepted May 7, 2009; Published June 5, 2009 <br />   This open-access article distributed terms Creative Commons Public Domain declaration stipulates that, placed public <br />   domain, work freely reproduced, distributed, transmitted, modified, built upon, used lawful purpose. <br />   Funding: The authors supported, part, Military Operational Medicine research program U.S. Army Medical Research Materiel Command, <br />   Ft. Detrick, Maryland. This effort supported U.S. Army&#226;&#8364;&#8482;s Network Science Initiative. The opinions assertions contained private views <br />   authors construed official reflecting views U.S. Army U.S. Department Defense. This paper approved <br />   public release unlimited distribution. The funders role study design, data collection analysis, decision publish, preparation the <br />   manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: jaques.reifman@us.army.mil <br />  <br />  <br />  <br /> Introduction                                                                              Due potential significance delineating biological <br />                                                                                        organization, topologies PPI networks explored <br />    The accurate modeling cellular processes requires knowledge                      using variety graph-theoretical techniques [15&#226;&#8364;&#8220;17]; however, <br /> of underlying components practical descriptions                      recent investigations resemble <br /> of interactions [1]. Proteins make                         probabilistic, random, frameworks [18,19]. PPI networks are <br /> the cellular machinery; however, act individually,                         routinely exploited discovery biological traits, where <br /> parts dynamic pathway, elements multi-component                          correlations topological properties biological attributes <br /> complexes behave individual functional entities [2]. The                       probed for. Examples inferred relationships include those <br /> elucidation protein roles enhanced discovery                         degree (number interactions) essentiality [20&#226;&#8364;&#8220;23], <br /> interactions proteins cell. Recent advances                       connectivity evolutionary rate [24&#226;&#8364;&#8220;26]. Other studies aim <br /> experimental high-throughput (HT) technologies, notably                        identify biological entities, functional modules [27&#226;&#8364;&#8220;29] <br /> the forms yeast two-hybrid (Y2H) [3] tandem-affinity-                       pathways [30,31], networks. More recently, PPI <br /> purification (TAP) [4] platforms, enabled large-scale protein-                    network information used augment gene expression <br /> protein interaction (PPI) screens subsequent constructions                      measurements identify condition-specific response complexes <br /> corresponding PPI networks. A number HT data sets,                             [32,33]. PPI networks prospective roles drug discovery <br /> these experimental platforms, available yeast                    [34]. <br /> Saccharomyces cerevisiae [5&#226;&#8364;&#8220;9] small number                            It clear PPI networks potential considerably <br /> species, including Escherichia coli [10,11], Drosophila melanogaster                   supplement areas biological research. However, well <br /> [12,13], Caenorhabditis elegans [14].                                              known data sets different studies small <br />  <br />  <br />        PLoS ONE | www.plosone.org                                                  1                                    June 2009 | Volume 4 | Issue 6 | e5815 <br />                                                                                                                    Abundance Protein Networks <br />  <br />  <br /> numbers coincident interactions [6,35]. These small overlaps              results unexpected, quantify levels of <br /> have led skepticism suggestions bias regarding          abundance effects persistence the <br /> authenticities [36,37]. The aforementioned observations                  data sets. More interestingly, analysis high-confidence (HC) <br /> reservations primarily based analyses experimen-            interaction data sets inferred affinity purification studies <br /> tal studies yeast proteins, using Y2H screens [7,9]               they, too, statistically significant correlation <br /> another using HT mass spectrometric protein complex                        degree abundance. As mentioned earlier, Y2H <br /> identification (HMS-PCI) technique [6], based                 data sets, including HC, correlation degree and <br /> affinity purification procedure. More recently, large indepen-           abundance. Therefore, substantiate systematic differ- <br /> dent yeast PPI data sets determined using nearly identical TAP               ence PPI networks determined Y2H affinity <br /> methodologies available [5,8]. While number               purification methods. <br /> mutually detected TAP interactions modest, overlaps               In light discovered associations degree and <br /> TAP-observed interactions Y2H HMS-PCI data sets                 abundance, reinvestigated centrality-lethality rule [20&#226;&#8364;&#8220;23], <br /> are small. A small interaction overlap exists         implies higher-degree proteins likely be <br /> two TAP data sets E. coli [10,11].                                        essential. We find, strict statistical analyses degree <br />    For PPI networks effectively utilized, authenticities         distributions essential nonessential proteins, raw <br /> must established. Platform-dependent high-quality interaction             HC Y2H data sets correlation degree and <br /> maps yeast recently deduced TAP [38] Y2H               essentiality, HMS-PCI TAP PPI networks, one <br /> [39] methodologies. However, major step extracting              exception, contain substantial correlations. However, also <br /> verifying credible interactions raw experimental data                   essential proteins generally abundant than <br /> requires comprehension distinct systematic biases present             nonessential proteins and, therefore, correlations may <br /> in various experimental platforms. Previous investigations           artificially induced. In fact, generally degree/ <br /> yeast suggested protein abundance important factor           abundance degree/essentiality correlations occur tandem <br /> for detecting interactions affinity purification studies       present absent. As such, the <br /> Y2H screens [35,40]. Von Mering et al. [35] showed PPI               centrality-lethality rule misleading. In effort identify <br /> data sets deduced affinity purification studies (TAP [41]           nonrandom signatures interaction data sets determined, <br /> and HMS-PCI [6]), proteins having interactions                comparisons strict randomized simulations, propensity <br /> likely larger corresponding messenger RNA (mRNA)                     essential proteins selectively interact other. We <br /> abundances bias detected PPI data set                 yeast PPI datasets contain significant enrichments of <br /> deduced Y2H screens. Bjo&#194;&#168;rklund et al. [40] showed PPIs            essential-essential interactions. While propensity essential <br /> detected more-recent TAP studies [5,8] enriched             proteins involved essential complex biological modules has <br /> highly-abundant (.6000 molecules/cell) proteins, Y2H                 realized previously HC networks [23], demonstrate the <br /> data set contained significant enrichment. Although study            general case essential proteins prefer interact with <br /> confirmed abundance effect affinity purification experiments           other. <br /> for yeast, did perform comprehensive studies investigating             These findings provide valuable insights underlying <br /> the total extent abundance influence. Simply considering <br />                                                                              natures differences various HT <br /> the impact highly-abundant proteins insufficient to <br />                                                                              technologies utilized detect PPIs. This knowledge lead <br /> ascertain scope abundance effects. Most proteins a <br />                                                                              effective strategies inference analysis high- <br /> cell high abundances; therefore, useful to <br />                                                                              quality PPI data sets. <br /> probe relative levels promiscuity, possibly stemming <br /> <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> varying abundances proteins, perceivable a <br /> variety affinity purification data sets, including         Materials Methods <br /> inferred high-quality.                                                          We analyzed protein interaction networks yeast E. coli <br />    The influence protein abundance method                     determined Y2H [7,9], HMS-PCI [6], TAP [5,8,10,11] <br /> interaction detection reinvestigated here. We analyzed PPI data           platforms. These studies provide lists experimentally <br /> sets encompassing different platforms incorporating                 observed interactions. These unprocessed data sets referred to <br /> cellular protein mRNA abundance levels measured using <br />                                                                              raw. In cases, raw binary interactions (non-self and <br /> three diverse technologies: western blot (WB) [42], flow cytometry <br />                                                                              undirected) subsequent PPI networks assembled by <br /> (FC) [43], gene expression (GE) [44]. Together yeast PPI <br />                                                                              tabulation bait-prey pairs. Some studies additionally <br /> data sets examined previous study [35], investigated <br />                                                                              attempt identify substantive interactions using range of <br /> more recent TAP data sets yeast [5,8] E. coli [10,11]. <br />                                                                              methodologies, including experimental reproducibility, removal of <br /> Correlations protein degree, number detected <br />                                                                              suspect promiscuous proteins, assignment confidence scores <br /> interactions, cellular protein mRNA abundances were <br />                                                                              using computational techniques. These data sets referred <br /> determined averaging binning data. Additionally, to <br />                                                                              high confidence. We investigated raw PPI networks <br /> gauge potential artificial correlations arising irregular <br />                                                                              corresponding HC data sets concurrently <br /> abundance distributions computed distributions proteins by <br />                                                                              inferred. <br /> degree. We TAP HMS-PCI PPI data sets yeast <br /> and E. coli contain statistically significant correlation between <br /> protein degree cellular abundance, Y2H data sets               Yeast Data <br /> show relationship. The findings confirm affinity                  Raw Y2H data sets investigations, labeled Ito [7] and <br /> purification methods influenced probabilistic interactions            Uetz [9], downloaded IntAct database [45] (http:// <br /> due differences protein concentrations. While known         www.ebi.ac.uk/intact/site/index.jsf). Ito et al. [7] additionally <br /> the nature affinity purification methods induce retrieval              provide core, HC, data set contains interactions that <br /> nonspecific contaminants, promiscuous prey proteins,              experimentally detected times was <br /> that promiscuity related high abundance. While             downloaded (http://itolab.cb.k.u-tokyo.ac.jp/Y2H). <br />  <br />  <br />        PLoS ONE | www.plosone.org                                        2                              June 2009 | Volume 4 | Issue 6 | e5815 <br />                                                                                                                      Abundance Protein Networks <br />  <br />  <br />    Purification data HMS-PCI study, labeled Ho [6],              nonessential proteins. Comparable P-values determined to <br /> acquired original publication. Ho et al. [6] infer            test correlations protein abundance essentiality. <br /> HC data set removal suspect promiscuous prey proteins                  The enrichment essential-essential protein interactions a <br /> this downloaded IntAct.                                              network evaluated computing P-value, calculation of <br />    Purification data TAP investigations [5,8]                   Z-score assuming normal distribution, difference <br /> acquired original publications. The raw data set                actual number average obtained from <br /> Gavin et al. [5], labeled Gavin, used matrix-assisted laser                   randomly selecting proteins essential. However, when <br /> desorption/ionization-time-of-flight mass spectroscopy (MALDI-                randomly selecting proteins considered essential, ensured <br /> TOF MS) identify co-purifying proteins. A corresponding HC                 total number degree distribution chosen <br /> data set inferred study determining &#226;&#8364;&#732;socio-             essential proteins matched actual essential proteins. <br /> affinity&#226;&#8364;&#8482; scores pair proteins followed iterative           This ensures artificial differences arising changes in <br /> clustering procedure refined using curated set protein          degree distributions essential proteins observed whilst <br /> complexes [46]. Two raw data sets taken study                simultaneously conserving network structure. Results were <br /> Krogan et al. [8]. The first, labeled Krogan-TOF, used MALDI-                 deduced 1000 simulations. <br /> TOF MS identify co-purifying proteins, second, <br /> labeled Krogan-LCMS, used liquid chromatography tandem mass                   Results <br /> spectrometry (LCMS) protein identifications. Two types HC <br /> data sets study Krogan et al. [8] downloaded                 Influence Protein Abundance Degree Raw PPI <br /> from (http://tap.med.utoronto.ca/downloads.php). Both                Data Sets <br /> HC data sets inferred removing 44 nonspecific                       Experimental PPI data sets derived techniques that <br /> contaminants nearly cytoplasmic ribosomal subunits               attempt detect presence protein associations cellular <br /> the raw data. The HC data set, labeled Krogan-INT, contains             environment. Therefore, investigation influences the <br /> the remaining interactions identified detection             cellular concentration protein detected <br /> methods. The second, labeled Krogan-CORE, derived using                   interactions based protein abundance measurements. <br /> machine-learning algorithms trained curated protein complex-               Recent determinations yeast cellular protein abundances during <br /> es MIPS reference database [47].                                       normal growth achieved tagging open reading frames <br />    Yeast cellular protein mRNA abundances normal                   (ORFs), allowing expressions natural chromo- <br /> aerobic growth taken investigations,               somal locations, measuring absolute levels WB [42] FC <br /> used different measurement methodology: WB [42], FC [43],                   [43] techniques. We utilized protein abundance measurement <br /> and GE analysis [44]. Essential yeast proteins obtained                  data sets studies. Although proteins that <br /> from Saccharomyces Genome Deletion Project (http://                       overwhelmingly responsible various cellular functions, <br /> www-sequence.stanford.edu/group/yeast_deletion_project/Essential_             expression levels precursors, mRNA molecules, provide <br /> ORFs.txt) Munich Information Center Protein                       considerable insights internal states cell. In fact, <br /> Sequences (MIPS) (ftp://ftpmips.gsf.de/yeast/catalogues/gene_                 measurement mRNA expression levels microarray experi- <br /> disruption). Only proteins annotated essential datasets            ments [51&#226;&#8364;&#8220;53] far simpler widespread than <br /> were considered essential here.                                         measurement protein abundances. Accordingly, have <br />                                                                               enormous number GE studies determined <br /> Escherichia coli Data                                                         yeast mRNA abundances normal growth. The findings <br />    Raw TAP-determined PPI networks investigations,                   reported utilized GE measurements Holstege et al. <br /> labeled Butland [10] Arifuzzaman [11], acquired           [44]; however, stressed comparable results were <br /> original publications. Gene expression measurements                    obtained using GE measurements more-recent <br /> normal aerobic growth taken studies [48&#226;&#8364;&#8220;50].                  studies [54&#226;&#8364;&#8220;56]. The previously noted trend degree and <br />                                                                               abundance [35] based data Holstege et al. [44]. As <br /> Computational Analyses                                                        noted above, raw yeast PPI data sets investigated here <br />    Correlations protein degree abundance                     encompassing diverse HT technologies: TAP (three data <br /> evaluated determining Pearson Spearman rank correlation                sets), HMS-PCI (one data set), Y2H (two data sets). We have <br /> coefficients log(degree) vs. log2(abundance). In case           investigated raw TAP data sets E. coli used GE <br /> two coefficients similar, Spearman&#226;&#8364;&#8482;s correlation               measurements studies [48&#226;&#8364;&#8220;50]. <br /> coefficient generally slightly smaller magnitude. Corre-                 Table 1 shows correlations yeast protein degree <br /> lation analyses performed pairs individual data sets              abundance, WB, FC, GE measurements. <br /> (PPI network vs. abundance measurement set) averaging                 We raw TAP PPI data sets contain statistically <br /> of data, i.e., protein included separate data entity       significant correlations protein degree abundance. <br /> its degree abundance known. To illustrate general                The Pearson correlation coefficients test linear relationship <br /> trends correlations, lack of, generated plots                 log values, lying 0.23 0.33, similar but <br /> log(degree) vs. ,log2(abundance)., quantity              large corresponding Spearman rank correlation <br /> determined averaging log2(abundance) values, particular              coefficients close value. However, P-values less <br /> abundance measurement set, proteins having degree.                0.0001, suggesting relationships significant, i.e., <br />    To gauge possibility artificial correlations arising           represent random events. We stress these <br /> irregular abundance distributions proteins PPI data sets,           correlations obtained averaging data, i.e., each <br /> we computed abundance distributions proteins grouped                   protein&#226;&#8364;&#8482;s degree abundance included single data entity. <br /> degree. These illustrated color maps (Figures 1, 2, 3).               Merging Gavin Krogan-TOF data sets produces PPI <br />    Correlations essentiality degree evaluated             network contains similar degree/abundance correlations <br /> calculating P-values two-sample Kolmogorov-Smirnov (KS)                   individual data sets. Figure 1A, showing data points, <br /> test differences degree distributions essential            illustrates degree/abundance relationship raw TAP <br />  <br />  <br />        PLoS ONE | www.plosone.org                                         3                               June 2009 | Volume 4 | Issue 6 | e5815 <br />                                                                                                                  Abundance Protein Networks <br />  <br />  <br />  <br />  <br /> Figure 1. Relationship protein degree abundance raw yeast TAP Gavin PPI network [5] using western blot <br /> abundance measurements Ghaemmaghami et al. [42]. (A) All data points, i.e., protein&#226;&#8364;&#8482;s degree abundance plotted; (B) averaged <br /> data log2(abundance) values averaged proteins having degree; (C) total normalized abundance distribution, binned in <br /> integer values log2(abundance), proteins appearing PPI abundance measurement data sets; (D) normalized abundance <br /> distributions degree frequencies shown color: yellow signifies smallest nonzero value blue represents values <br /> larger 0.25. Best-fit line data (A) shown (D). <br /> doi:10.1371/journal.pone.0005815.g001 <br />  <br /> Gavin PPI network using WB abundance measurements [42],                    The observed correlations artificial abundance <br /> and Figure 1B shows general trend log2(abundance)                 distributions proteins degree skewed. However, <br /> values averaged proteins having degree. The              notion discounted finding statistically <br /> Pearson correlation coefficient averaged data 0.52       significant correlations non-averaged data. Nonetheless, we <br /> and corresponding P-value 0.0001. Similar plots        investigate abundance distributions degree further <br /> obtained raw yeast TAP PPI data sets            establish authenticity degree/abundance relationship in <br /> using FC (protein) GE (mRNA) abundance measurements.                raw TAP PPI networks. For proteins appearing raw <br /> The results E. coli TAP data sets identical           Gavin PPI WB abundance measurement data sets, show <br /> those yeast TAP sets (Table 2), Pearson Spearman            total abundance distribution (binned integer values of <br /> correlation coefficients lying 0.11 0.46                log2(abundance)) Figure 1C degree (as color map) <br /> associated P-values 0.0001. Figures 2A 2B            Figure 1D. The total abundance distribution seen very <br /> non-averaged averaged data, respectively, Butland PPI           close normal distributions degree also <br /> network using GE measurements Covert et al. It clear         reasonably symmetric averages. Note Figure 1D <br /> there definite propensity, slight, proteins            reflects general trend averaged data Figure 1B. These <br /> increasing degree higher abundances.                                observations echoed E. coli TAP Butland PPI network <br />  <br />  <br />       PLoS ONE | www.plosone.org                                        4                              June 2009 | Volume 4 | Issue 6 | e5815 <br />                                                                                                                    Abundance Protein Networks <br />  <br />  <br />  <br />  <br /> Figure 2. Relationship protein degree abundance raw E. coli TAP Butland PPI network [10] using gene <br /> expression measurements Covert et al. [48]. (A)&#226;&#8364;&#8220;(D), Figure 1 legend. <br /> doi:10.1371/journal.pone.0005815.g002 <br />  <br /> and GE measurement data set Covert et al. (Figure 2C              However, express tagged ORFs native <br /> D), degree/abundance relationship appears                     environments, TAP studies, Ho et al. express tagged <br /> pronounced. Therefore, possible TAP method                 ORFs plasmids containing GAL1 promoters. It unclear <br /> detecting interactions influenced cellular                   non-native delivery affects cellular <br /> concentrations proteins. While finding                abundances tagged baits expect prey proteins <br /> surprising, TAP method expresses tagged bait potential            concentrations similar normal growth <br /> prey ORFs natural chromosomal locations, does                  conditions. Not surprisingly, perhaps, statistically <br /> imply raw TAP-determined PPI networks incorporate                     significant correlations degree abundance Ho <br /> probabilistic, random, element. The higher cellular                   PPI data set. Correlation coefficients modest, lying <br /> abundance protein, likely detected           0.17 0.28 (Table 1), P-values than <br /> in purifications and, therefore, interactions            0.0001, suggesting statistically significant relationship between <br /> construed involved in. Although correlation coefficients           degree abundance Ho data set. Therefore, would <br /> given large, known mRNA protein                  general affinity purification methods detecting <br /> abundance measurements contain sources variation                 interactions somewhat mediated proteins&#226;&#8364;&#8482; cellular <br /> to technical biological factors.                                         abundances. <br />    The HMS-PCI technique isolate identify co-purifying                   Without counter-example PPI data sets no <br /> proteins [6] similar used TAP studies [5,8],       correlation degree abundance, expect <br /> it uses affinity purification method isolate complexes.                findings biologically relevant, i.e., average, <br />  <br />  <br />        PLoS ONE | www.plosone.org                                        5                              June 2009 | Volume 4 | Issue 6 | e5815 <br />                                                                                                                   Abundance Protein Networks <br />  <br />  <br />  <br />  <br /> Figure 3. Relationship protein degree abundance raw yeast Y2H Ito PPI network [7] using western blot <br /> abundance measurements Ghaemmaghami et al. [42]. (A)&#226;&#8364;&#8220;(D), Figure 1 legend. <br /> doi:10.1371/journal.pone.0005815.g003 <br />  <br />  <br />  <br /> the number interactions protein involved related       network using WB protein abundance measurements. It is <br /> cellular concentration. Such interpretation suggest           clear plots relationship exists degree <br /> a salient probabilistic element exists interactome.                  abundance. <br /> However, regardless extent probabilistic behavior,               In order clarify differences PPI networks <br /> it established exist stable protein complexes            determined Y2H affinity purification methods we <br /> [47]. Therefore, raw Y2H          analyzed abundance distributions degree proteins appearing <br /> PPI data sets, individual Ito, Uetz, combined,           raw Ito PPI WB abundance measurement data <br /> no correlation degree abundance (Table 1). All                  sets. Figures 3C 3D total abundance distribution and <br /> correlations coefficients small, lying 20.07           distributions degree, respectively. The total abun- <br /> 0.04, P-values, Pearson correlation coefficients,           dance distribution essentially normal notably similar to <br /> greater 0.02, individual Ito Uetz data            TAP Gavin PPI data set (Figure 1C). Additionally, <br /> sets range 0.06 0.92. When using GE WB                 Figure 3D shows distributions degree are <br /> abundance measurements, correlation coefficients absolute              symmetric averages clearly illustrates lack <br /> values 0.05 P-values greater 0.22. P-             degree/abundance correlation. Therefore, compared <br /> values smallest FC abundance measurements               results raw TAP HMS-PCI data sets, we <br /> reasons immediately clear. Figures 3A 3B               conclude raw Y2H PPI data sets contain degree/ <br /> the non-averaged averaged data, respectively, Ito PPI           abundance relationship. Furthermore, affirm the <br />  <br />  <br />       PLoS ONE | www.plosone.org                                        6                              June 2009 | Volume 4 | Issue 6 | e5815 <br />                                                                                                                            Abundance Protein Networks <br />  <br />  <br />  <br />  Table 1. Pearson correlation coefficients (r) corresponding P-values tests linear association log(degree) and <br />  log2(abundance) raw yeast PPI data sets. <br />  <br />  <br />  PPI data set           Western blot (Ghaemmaghami et al.)             Flow cytometry (Newman et al.)          Gene expression (Holstege et al.) <br />  <br />                         NPa          r                   Pb            NPa       r                   Pb        NPa        r                   Pb <br />  <br />  TAP <br />  Gavin                  2146         0.29 (0.28)         ,             1585      0.29 (0.26)         ,         2496       0.33 (0.30)         , <br />  Krogan-TOF             2291         0.24 (0.21)         ,             1523      0.25 (0.20)         ,         2701       0.23 (0.21)         , <br />  Krogan-LCMS            3543         0.26 (0.21)         ,             2314      0.28 (0.22)         ,         4707       0.26 (0.20)         , <br />  Gavin+Krogan-TOFc      2845         0.33 (0.31)         ,             1925      0.32 (0.27)         ,         3410       0.32 (0.30)         , <br />  HMS-PCI <br />  Ho                     1326         0.26 (0.23)         ,             938       0.28 (0.25)         ,         1630       0.18 (0.17)         , <br />  Y2H <br />  Ito                    2107         0.01 (0.04)         0.5820        1357      20.05 (20.04)       0.0646    2910       ,0 (0.01)           0.9212 <br />  Uetz                   929          20.01 (0.01)        0.8470        599       20.07 (20.06)       0.0686    1221       20.04 (20.03)       0.2214 <br />  Ito+Uetzd              2403         20.01 (0.01)        0.6066        1560      20.06 (20.04)       0.0277    3311       20.01 (20.01)       0.5689 <br />  <br />  Spearman rank correlation coefficients provided parentheses. <br />  a <br />    Number proteins PPI data set having abundance measurements. <br />  b <br />    The symbol &#226;&#8364;&#732;&#226;&#8364;&#732;,&#226;&#8364;&#8482;&#226;&#8364;&#8482; signifies P,0.0001 Pearson Spearman correlation coefficients. <br />  c <br />   Combined Gavin Krogan-TOF interaction data sets. <br />  d <br />    Combined Ito Uetz interaction data sets. <br />  doi:10.1371/journal.pone.0005815.t001 <br />  <br />  <br /> degree/abundance correlations observed TAP HMS-                           Influence Protein Abundance Degree HC PPI <br /> PCI PPI networks result skewed abundance                              Data Sets <br /> distributions, total individual degrees. It clear                Some yeast PPI studies utilized work inferred <br /> the Y2H methodology distinct affinity purification                       HC interaction data sets raw data. These HC PPI <br /> methods protein expression levels influence                       data sets meant contain interactions most <br /> observed interactions technique.                                      reproducible resolute. The methods used infer HC <br />    The findings consistent considering Y2H                       interactions varied (see Materials Methods). Therefore, <br /> screen bait prey proteins expressed similar                        discover sets contain any <br /> plasmids. A protein expressed plasmid likely                     relationship degree abundance. We the <br /> different cellular concentration expressed                  yeast HC PPI networks deduced raw affinity purification <br /> native chromosomal location. Additionally, different proteins                        data (TAP HMS-PCI) contain statistically significant <br /> expressed plasmid presumed similar                         correlations (Table 3). All correlation coefficients modest, <br /> expression levels. However, true,                  lying 0.13 0.39, similar observed <br /> possible Y2H screen influenced                       raw TAP HMS-PCI data sets. However, P-values <br /> abundance factor related post-transcriptional modification.                       0.0001, suggesting nontrivial relationships <br /> Investigation speculation straightforward                  exist degree abundance. These results are <br /> possible translational efficiencies proteins                    surprising Krogan-CORE, Krogan-INT, HC Ho <br /> estimated.                                                                           data sets, inferring methodologies included, first <br />                                                                                      step, removal promiscuous proteins. Krogan et al. <br />                                                                                      removed 44 nonspecific contaminants nearly cytoplasmic <br />  Table 2. Pearson Spearman rank (in parentheses)                                 ribosomal subunits raw data preliminary step, <br />  correlation coefficients (r) tests linear association                        HC Ho PPI data set subset raw data in <br />  log(degree) log2(mRNA expression) raw E. coli                        nonspecifically binding proteins subtracted. In <br />  TAP PPI data sets.                                                                  spite removals, inferring procedures, <br />                                                                                      HC data sets contain degree/abundance relationship, <br />                                                                                      albeit weak. Therefore, influence protein cellular <br />  PPI data set        Covert et al.       Kang et al.         Salmon et al.           abundance limited small proportion highly- <br />                      NPa   r             NPa    r            NPa   r                 abundant promiscuous proteins. Rather, effect seems <br />                                                                                      subtly ingrained data. The TAP HC data set of <br />  Butland             1277 0.46 (0.45) 1278      0.45 (0.42) 856    0.24 (0.21)       Gavin et al. inferred determining &#226;&#8364;&#732;socio-affinity&#226;&#8364;&#8482; <br />  Arifuzzaman         2918 0.21 (0.19) 2919      0.21 (0.20) 1764   0.13 (0.11)       scores pair proteins, quantified propensity <br />  Butland+            3158 0.30 (0.28) 3159      0.30 (0.27) 1904   0.17 (0.13)       occur purifications. However, degree/ <br />  Arifuzzmanb                                                                         abundance correlation coefficients lie 0.31 0.39. In <br />                                                                                      fact, HC Gavin data set larger correlation coefficients <br />  All P,0.0001 Pearson Spearman correlation coefficients. <br />  a <br />   Number proteins PPI data set having abundance measurements. <br />                                                                                      corresponding values raw PPI network. These <br />  b <br />   Combined Butland Arifuzzaman interaction data sets.                            findings HC TAP HMS-PCI data sets suggest <br />  doi:10.1371/journal.pone.0005815.t002                                               degree/abundance relationship is, discussed earlier, <br />  <br />  <br />          PLoS ONE | www.plosone.org                                              7                              June 2009 | Volume 4 | Issue 6 | e5815 <br />                                                                                                                                Abundance Protein Networks <br />  <br />  <br />  <br />  Table 3. Pearson correlation coefficients (r) corresponding P-values tests linear association log(degree) and <br />  log2(abundance) high-confidence yeast PPI data sets. <br />  <br />  <br />                                                                                                                                Gene expression (Holstege et <br />  PPI data set                        Western blot (Ghaemmaghami et al.)                  Flow cytometry (Newman et al.)        al.) <br />  <br />                                      NPa               r                      Pb         NPa         r                   Pb    NPa      r        Pb <br />  <br />  Gavin [TAP]                         1286              0.31 (0.30)            ,          1012        0.38 (0.37)         ,     1470     0.39     , <br />                                                                                                                                         (0.37) <br />  Krogan-CORE [TAP]                   2208              0.20 (0.18)            ,          1461        0.16 (0.14)         ,     2578     0.22     , <br />                                                                                                                                         (0.21) <br />  Krogan-INT [TAP]                    1081              0.15 (0.13)            ,          778         0.16 (0.15)         ,     1186     0.20     , <br />                                                                                                                                         (0.19) <br />  Ho [HMS-PCI]                        1240              0.26 (0.26)            ,          874         0.26 (0.26)         ,     1535     0.21     , <br />                                                                                                                                         (0.21) <br />  Ito [Y2H]                           596               20.02 (20.01)          0.6514     380         20.06 (20.06)       0.2393 727     0.02     0.6726 <br />                                                                                                                                         (0.03) <br />  <br />  Spearman rank correlation coefficients provided parentheses. <br />  a <br />   Number proteins PPI data set having abundance measurements. <br />  b <br />    The symbol &#226;&#8364;&#732;&#226;&#8364;&#732;,&#226;&#8364;&#8482;&#226;&#8364;&#8482; signifies P,0.0001 Pearson Spearman correlation coefficients. <br />  doi:10.1371/journal.pone.0005815.t003 <br />  <br />  <br /> biologically significant. Alternatively, observed correlations                     degree distributions. We degree distributions of <br /> for HC data sets result inferring procedures                       essential proteins TAP HMS-PCI data sets are <br /> that completely effectual.                                                     significantly different nonessential proteins, PKS- <br />    The Y2H HC data set Ito et al. subset raw PPI                        values 0.0001 (Table 4). These differences are <br /> network includes interactions experi-                         illustrated Figures 4A 4B, degree distributions <br /> mentally detected times. We interaction                  essential nonessential proteins raw Gavin and <br /> set shows correlation degree abundance (Table 3).                       Krogan-TOF PPI data sets, respectively. <br /> This surprising raw data set contains                                In stark contrast, degree distributions essential <br /> relationship. The correlation coefficients small, ranging                     nonessential proteins raw Y2H networks not <br /> 20.06 0.03, P-values Pearson correlation coefficients                   significantly different, PKS-values 0.9915 0.1542 the <br /> are larger 0.23.                                                                  raw Ito Uetz data sets, respectively. Degree distributions of <br />    The results HC PPI data sets identical                  essential nonessential proteins Ito Uetz PPI <br /> raw data sets. Those derived affinity purification experiments                    networks shown Figures 4C 4D, respectively, is <br /> show weak, statistically significant correlations degree                   clear data sets curves similar. Therefore, <br /> and abundance, Y2H PPI data sets relationship                        conclude raw Y2H data sets correlation <br /> between degree abundance. These findings exemplify                         degree essentiality. In fact, raw Ito data set a <br /> the difference interactions detected Y2H screens                        P-value near one. It previously reported the <br /> affinity purification procedures.                                                      Ito data set weak correlation degree essentiality <br />                                                                                        [20]; however, difference degree distributions <br /> Relation Essentiality Topology Raw PPI                                  essential nonessential proteins data set. <br /> Data Sets                                                                                  The degree/essentiality relationships discussed raw <br />    The correlation degree essentiality, proteins                   PPI data sets curious (weak) degree/ <br /> having interactions likely essential,                     abundance correlation, degree/essentiality relation- <br /> noted previously [20&#226;&#8364;&#8220;23] generally accepted precept                          ship. These tandem correlations observed raw <br /> known centrality-lethality rule. Recent analysis curated,                    interaction data sets determined affinity purification methods <br /> inferred, HC yeast networks essential proteins                       (TAP HMS-PCI). The converse true, that, no <br /> more likely involved essential complex biological modules                     degree/abundance relationship, degree/essentiality <br /> and, therefore, degrees average higher [23].                          correlation (Table 4). These tandem non-correlations observed <br /> However, findings presented here, PPI networks                                Y2H interaction data sets. Insights correlations <br /> determined using affinity purification procedures (TAP                             associated obtained looking relationship between <br /> HMS-PCI) statistically significant correlations                           essentiality abundance. In fact, essential proteins are <br /> degree abundance Y2H PPI networks not, warrants                           abundant nonessential proteins yeast <br /> another look degree/essentiality property.                                      abundance measurements utilized here. P-values, assuming normal <br />    Table 4 gives average degrees essential nonessential                         distributions, tests difference average log2(abundance) <br /> proteins raw yeast PPI networks. We essential                         essential nonessential proteins WB [42], FC [43], and <br /> proteins higher average degrees nonessential proteins                     GE [44] measurements 10219, 1027, 10237 respectively. P- <br /> all raw PPI networks. However, known degree                          values two-sample KS tests similar, 10218, 1028, and <br /> distributions PPI networks normal; rather, resemble                    10231 WB, FC, GE measurements, respectively. Therefore, <br /> power-law scaling [22,57]. Therefore, order determine                        correlation degree essentiality raw TAP and <br /> significance difference degrees essential                        HMS-PCI networks artificial essential proteins <br /> nonessential proteins use two-sample KS test compare                         generally abundant. This explain no <br />  <br />  <br />        PLoS ONE | www.plosone.org                                                  8                                 June 2009 | Volume 4 | Issue 6 | e5815 <br />                                                                                                                                  Abundance Protein Networks <br />  <br />  <br />  <br />  Table 4. Tests difference beween degree distributions essential nonessential proteins raw high-confidence yeast <br />  PPI data sets. <br />  <br />  <br />                                                                                                                         Correlation degree/ <br />  PPI data set                            Essentiala               Nonessentialb                  PKSc                   abundance <br />  <br />  Raw <br />  Gavin [TAP]                             20.79                    11.02                          ,                      yes <br />  Krogan-TOF [TAP]                        18.02                    10.63                          ,                      yes <br />  Krogan-LCMS [TAP]                       28.49                    14.36                          ,                      yes <br />  Ho [HMS-PCI]                            12.24                    8.18                           ,                      yes <br />  Ito [Y2H]                               3.04                     2.60                           0.9915                 no <br />  Uetz [Y2H]                              2.59                     1.96                           0.1542                 no <br />  High-confidence <br />  Gavin [TAP]                             5.88                     5.90                           0.5160                 yes <br />  Krogan-CORE [TAP]                       7.82                     4.20                           ,                      yes <br />  Krogan-INT [TAP]                        5.21                     2.95                           ,                      yes <br />  Ho [HMS-PCI]                            6.04                     3.89                           ,                      yes <br />  Ito [Y2H]                               2.13                     1.84                           0.9994                 no <br />  <br />  a <br />    Average degree essential proteins. <br />  b <br />    Average degree nonessential proteins. <br />  c <br />   P-value two-sample Kolmogorov-Smirnov test difference degree distributions essential nonessential proteins. The symbol &#226;&#8364;&#732;&#226;&#8364;&#732;,&#226;&#8364;&#8482;&#226;&#8364;&#8482; signifies <br />   P,0.0001. <br />  doi:10.1371/journal.pone.0005815.t004 <br />  <br />  <br /> correlation degree essentiality Y2H data sets                 HC Ho data set (HMS-PCI). Up point, findings for <br /> they contain correlations degree abundance.                     HC data sets lend support notion identified <br /> Therefore, common notion essential proteins generally                 correlation degree essentiality PPI network be <br /> higher degrees nonessential proteins misleading.                        artificially induced, essential proteins generally abundant <br />    In effort identify deterministic, nonrandom, signatures              nonessential. In stark contrast, however, HC Gavin data set, <br /> the raw yeast PPI networks, quantified enrichment                         P-value 0.52, shows correlation degree and <br /> essential-essential interactions data sets comparing                  essentiality does contain degree/abundance associa- <br /> observed numbers strict randomized simulations.                     tion. Of PPI networks investigated work, HC Gavin <br /> For given PPI network, proteins considered essential                data set contains degree/abundance correlation <br /> selected random constraint degree distribution              degree/essentiality relationship. The reasons are <br /> the selected proteins matched actual essential proteins.               immediately clear presumably related steps the <br /> This ensures results perturbed varying degree                   HC interaction inferring procedure. <br /> distributions &#226;&#8364;&#732;essential&#226;&#8364;&#8482; proteins whilst simultaneously                        We HC yeast PPI networks enriched <br /> conserving network structures. Data deduced 1000                      interactions essential proteins (Table 5). All P-values are <br /> realizations results given Table 5. We consistently             0.0001, indicating observed numbers essential- <br /> that numbers actual essential-essential interactions larger              essential interactions significantly larger strict <br /> than randomized simulations standard                        randomized simulations. Our test enrichment strict in <br /> deviations relatively small. Accordingly, P-values,              freeze network structure degree distributions of <br /> than 0.0001, indicated significant nonrandom enrichments. There-                    essential (actual randomly chosen) proteins and, therefore, is <br /> fore, biological signatures evident raw yeast PPI data sets,            difficult form extensive topological insights. However, our <br /> including Y2H despite networks correlation                       tests indicate, question, raw HC yeast PPI <br /> between degree essentiality.                                                    networks propensity essential proteins prefer to <br />                                                                                     interact other. We deduce biological signature in <br /> Relation Essentiality Topology HC Data                               PPI network does appear commonly acknowledged <br /> Sets                                                                                form degree/essentiality correlation; manifests itself <br />    Table 4 shows average degrees essential                          enhancing interactions essential proteins. While a <br /> nonessential proteins HC yeast PPI networks                    recent study concludes HC PPI networks essential proteins <br /> P-values two-sample KS tests. The Y2H HC Ito data set,                  likely involved essential complex biological <br /> P-value essentially one, correlation degree                   modules [23], general case essential <br /> essentiality. This result identical raw Ito PPI            proteins prefer interact other. <br /> network, indicating Y2H method does bias essential <br /> proteins interacting partners. The HC Krogan data                      Discussion <br /> sets, like raw counterparts, significant correlations (all P- <br /> values 0.0001) degree essentiality. This                    It shown raw HC TAP HMS-PCI PPI networks <br /> result unsurprising data sets relationship                  contain statistically significant correlations protein degree <br /> between degree abundance. Similar findings obtained                     abundance. The previously noted trend protein <br />  <br />  <br />        PLoS ONE | www.plosone.org                                               9                                   June 2009 | Volume 4 | Issue 6 | e5815 <br />                                                                                                                 Abundance Protein Networks <br />  <br />  <br />  <br />  <br /> Figure 4. Degree distributions essential (red dashed) nonessential (black) proteins raw yeast PPI networks. (A) Gavin (TAP) <br /> [5], (B) Krogan-TOF (TAP) [8], (C) Ito (Y2H) [7], (D) Uetz (Y2H) [9]. <br /> doi:10.1371/journal.pone.0005815.g004 <br />  <br />  <br /> degree mRNA abundance [35] confirmed using                        The TAP method extracts tagged bait proteins, expressed <br /> protein abundances extensive analyses. The results            native genome locations, determines other <br /> consistent yeast (three TAP HMS-PCI) E. coli (two          proteins, preys, co-purified, complexed, them. <br /> TAP) data sets. For yeast, correlations similar          The HMS-PCI method similar tagged bait proteins <br /> diverse protein mRNA abundance measurement technologies:               expressed plasmids. In techniques prey <br /> western blot, flow cytometry, gene expression. For E. coli,        proteins expressed natural conditions their <br /> results consistent using gene expression measurements             native environments. As such, degree/abundance relation- <br /> during normal aerobic growth studies. Although                  ship TAP HMS-PCI PPI data sets wholly <br /> correlation coefficients modest, observations highly           unexpected. If protein pairs similar binding <br /> significant. Furthermore, protein abundance gene expression            affinities, probability theory dictates number of <br /> measurements known variable. Yet, identified                 detected interactions proteins correlate roughly with <br /> correlations degree abundance consistently                 concentrations, abundances. Nonrandom influences in <br /> observed indicate inherent nontrivial property           forms differing expression times cellular locations will <br /> data.                                                                      remove probabilistic elements. While known that <br />  <br />  <br />       PLoS ONE | www.plosone.org                                      10                              June 2009 | Volume 4 | Issue 6 | e5815 <br />                                                                                                                             Abundance Protein Networks <br />  <br />  <br />  <br />  Table 5. Enrichment interactions essential                               proteins relatively high vivo abundances [58]. Taken <br />  proteins raw high-confidence yeast PPI data sets.                            together, results imply proteins having high <br />                                                                                      abundances detected Y2H approach few <br />                                                                                      interactions, interactions likely specific. <br />  PPI data set                       Actual       Randoma        sb                   Conversely, proteins having low abundances have <br />                                                                                      Y2H-detected interactions, likely non- <br />  Raw                                                                                 specific. Since vivo abundances proteins less <br />  Gavin [TAP]                        4692         4310           32.6                 tagged Y2H experiments, detected interactions <br />  Krogan-TOF [TAP]                   3416         3203           30.9                 necessarily occur natural environment. However, <br />  Krogan-LCMS [TAP]                  6566         6235           52.7                 associations detected proteins having high vivo <br />                                                                                      abundances, guaranteed relatively many, are <br />  Ho [HMS-PCI]                       1470         1391           20.2 <br />                                                                                      likely specific naturally occurring. Therefore, while <br />  Ito [Y2H]                          315          248            12.4                 vivo abundances influence total number Y2H- <br />  Uetz [Y2H]                         163          119            7.8                  detected interactions protein, intrinsically impact the <br />  High-confidence                                                                     numbers specific. <br />  Gavin [TAP]                        892          829            14.1                    In light observed (weak) correlations degree and <br />                                                                                      abundance TAP HMS-PCI PPI networks, we <br />  Krogan-CORE [TAP]                  1742         1397           24.0 <br />                                                                                      reinvestigated centrality-lethality rule, implies that <br />  Krogan-INT [TAP]                   906          765            13.4 <br />                                                                                      proteins having interactions likely essential. <br />  Ho [HMS-PCI]                       716          652            14.9                 From analysis diverse yeast protein mRNA <br />  Ito [Y2H]                          97           68             6.2                  abundance measurement data sets essential proteins <br />                                                                                      prevalent nonessential proteins. We observed <br />  All P,0.0001. <br />                                                                                    generally degree/abundance relationships occur tandem <br />   Average number interactions observed essential proteins 1000 <br />   simulations essential proteins selected randomly            degree/essentiality correlations. Additionally, there <br />   proteins occurring network. In simulation, total number           degree/abundance association, degree/ <br />   selected proteins degree distribution constrained             essentiality correlation. Therefore, degree/essentiality correla- <br />   identical actual essential proteins.                               tions TAP HMS-PCI data sets artificial. The lack <br />  b <br />    Standard deviation simulations described above. <br />  doi:10.1371/journal.pone.0005815.t005                                               degree/essentiality correlation Y2H data sets <br />                                                                                      supports notion. The results imply caution accepting the <br />                                                                                      generally acknowledged centrality-lethality rule. <br /> the TAP method induces retrieval nonspecific contaminants, <br />                                                                                         Biological, nonrandom, signatures identified the <br /> or promiscuous prey proteins, promiscuity may <br />                                                                                      PPI networks form enrichments interactions between <br /> be artificial property induced high abundances. <br />                                                                                      essential proteins. This propensity essential proteins interact <br /> Statistically significant correlations degree abun-                      deduced comparisons strict <br /> dance observed inferred HC TAP HMS-PCI data                          randomized simulations. Therefore, deduce essentiality <br /> sets. While inferring procedures involved steps                       does manifest itself biological property commonly <br /> eliminate contaminant nonspecifically binding proteins,                      acknowledged form degree/essentiality correlation; rather, it <br /> resulting HC interaction data sets contain degree/abun-                        actualized enhancement interactions essential <br /> dance relationships. Therefore, influence protein cellular                    proteins. <br /> abundance subtly ingrained data                               As demonstrating systematic differences PPI <br /> limited small proportion highly abundant                                 networks determined using Y2H affinity purification <br /> promiscuous proteins.                                                                methodologies, discern nature probabilistic element <br />    In direct contrast TAP HMS-PCI data sets, raw                      approaches. These findings provide insights <br /> and HC yeast Y2H PPI networks correlation                            design effective strategies deduce specific <br /> degree abundance. These results identify systematic                            invariable interactions raw TAP HMS-PCI data <br /> difference PPI networks determined Y2H                          sets. Such unbiased, untrained, procedures vital are <br /> affinity purification platforms. In hindsight, consistent               infer HC PPI networks organisms yeast and <br /> the experimental design. The Y2H approach expresses pair                        exploit discern genuine biological traits features. <br /> bait prey proteins, tested interaction,                        One avenue discovery receiving recent attention the <br /> engineered plasmids. Therefore, expression levels likely                development analyses combine gene expression PPI <br /> be different natural environments. That say                  data sets. For given conditions, environmental or <br /> that Y2H method influenced protein abundance                        physiological, changes mRNA levels relative reference <br /> some way. It generally accepted proteins expressed                  state mapped PPI networks order identify <br /> same plasmid similar abundances. While expressions                        response-type modules sub-networks. In respect, vital <br /> may similar, translational efficiencies and,                  comprehend underlying nature PPI data set. Highly <br /> so, possible Y2H screens affected plasmid-induced                  abundant proteins likely larger fluctuations their <br /> abundances. However, investigation premise                            expression levels and, therefore, utilizing network <br /> straightforward. Nonetheless, Y2H PPI data                         deduced affinity purification procedure, care be <br /> sets way mediated protein cellular abundance.                      taken interpreting results. <br />    The lack degree/abundance correlations Y2H PPI data <br /> sets related findings Zhang et al. [58],                   Author Contributions <br /> that interactions Y2H data Ito et al. [7] likely               Conceived designed experiments: JI AW JR. Performed the <br /> be biologically functional (i.e., independently reported                   experiments: JI XY. Analyzed data: JI XY AW JR. Contributed <br /> more publications using non-Y2H techniques) participating                     reagents/materials/analysis tools: JI XY. Wrote paper: JI AW JR. <br />  <br />  <br />  <br />        PLoS ONE | www.plosone.org                                               11                               June 2009 | Volume 4 | Issue 6 | e5815 <br />                                                                                                                                               Abundance Protein Networks <br />  <br />  <br />  <br />  <br /> References <br />  1. Joyce AR, Palsson BO (2006) The model organism system: integrating                    31. Przulj N, Wigle DA, Jurisica I (2004) Functional topology network protein <br />     &#226;&#8364;&#732;omics&#226;&#8364;&#8482; data sets. Nat Rev Mol Cell Biol 7: 198&#226;&#8364;&#8220;210.                                           interactions. Bioinformatics 20: 340&#226;&#8364;&#8220;348. <br />  2. Aloy P, Russell RB (2006) Structural systems biology: modelling protein                    32. Chuang HY, Lee E, Liu YT, Lee D, Ideker T (2007) Network-based <br />     interactions. Nat Rev Mol Cell Biol 7: 188&#226;&#8364;&#8220;197.                                                classification breast cancer metastasis. Mol Syst Biol 3: 140. <br />  3. Fields S (2005) High-throughput two-hybrid analysis. The promise peril.            33. Ulitsky I, Shamir R (2007) Identification functional modules using network <br />     FEBS J 272: 5391&#226;&#8364;&#8220;5399.                                                                         topology high-throughput data. BMC Syst Biol 1: 8. <br />  4. Puig O, Caspary F, Rigaut G, Rutz B, Bouveret E, et al. (2001) The tandem                  34. Nikolsky Y, Nikolskaya T, Bugrim A (2005) Biological networks analysis of <br />     affinity purification (TAP) method: general procedure protein complex                     experimental data drug discovery. Drug Discov Today 10: 653&#226;&#8364;&#8220;662. <br />     purification. Methods 24: 218&#226;&#8364;&#8220;229.                                                         35. von Mering C, Krause R, Snel B, Cornell M, Oliver SG, et al. (2002) <br />  5. Gavin AC, Aloy P, Grandi P, Krause R, Boesche M, et al. (2006) Proteome                        Comparative assessment large-scale data sets protein-protein interactions. <br />     survey reveals modularity yeast cell machinery. Nature 440: 631&#226;&#8364;&#8220;636.                    Nature 417: 399&#226;&#8364;&#8220;403. <br />  6. Ho Y, Gruhler A, Heilbut A, Bader GD, Moore L, et al. (2002) Systematic                    36. Mrowka R, Patzak A, Herzel H (2001) Is bias proteome research? <br />     identification protein complexes Saccharomyces cerevisiae mass                        Genome Res 11: 1971&#226;&#8364;&#8220;1973. <br />     spectrometry. Nature 415: 180&#226;&#8364;&#8220;183.                                                         37. Sprinzak E, Sattath S, Margalit H (2003) How reliable experimental protein- <br />  7. Ito T, Chiba T, Ozawa R, Yoshida M, Hattori M, et al. (2001) A comprehensive                   protein interaction data? J Mol Biol 327: 919&#226;&#8364;&#8220;923. <br />     two-hybrid analysis explore yeast protein interactome. Proc Natl Acad Sci           38. Collins SR, Kemmeren P, Zhao XC, Greenblatt JF, Spencer F, et al. (2007) <br />     USA 98: 4569&#226;&#8364;&#8220;4574.                                                                             Toward comprehensive atlas physical interactome Saccharomyces <br />  8. Krogan NJ, Cagney G, Yu H, Zhong G, Guo X, et al. (2006) Global landscape                      cerevisiae. Mol Cell Proteomics 6: 439&#226;&#8364;&#8220;450. <br />     protein complexes yeast Saccharomyces cerevisiae. Nature 440:                    39. Yu H, Braun P, Yildirim MA, Lemmens I, Venkatesan K, et al. (2008) High- <br />     637&#226;&#8364;&#8220;643.                                                                                       quality binary protein interaction map yeast interactome network. Science <br />  9. Uetz P, Giot L, Cagney G, Mansfield TA, Judson RS, et al. (2000) A                             322: 104&#226;&#8364;&#8220;110. <br />     comprehensive analysis protein-protein interactions Saccharomyces                    40. Bjorklund AK, Light S, Hedin L, Elofsson A (2008) Quantitative assessment of <br />     cerevisiae. Nature 403: 623&#226;&#8364;&#8220;627.                                                               structural bias protein-protein interaction assays. Proteomics 8: <br /> 10. Butland G, Peregrin-Alvarez JM, Li J, Yang W, Yang X, et al. (2005) Interaction                4657&#226;&#8364;&#8220;4667. <br />     network containing conserved essential protein complexes Escherichia                41. Gavin AC, Bosche M, Krause R, Grandi P, Marzioch M, et al. (2002) <br />     coli. Nature 433: 531&#226;&#8364;&#8220;537.                                                                     Functional organization yeast proteome systematic analysis protein <br /> 11. Arifuzzaman M, Maeda M, Itoh A, Nishikata K, Takita C, et al. (2006) Large-                    complexes. Nature 415: 141&#226;&#8364;&#8220;147. <br />     scale identification protein-protein interaction Escherichia coli K-12.              42. Ghaemmaghami S, Huh WK, Bower K, Howson RW, Belle A, et al. (2003) <br />     Genome Res 16: 686&#226;&#8364;&#8220;691.                        <br /> </body></html>