<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>CEP3 Encodes Centromere Protein <br /> of Saccharomyces cerevisiae <br /> A l e x n d e r V. S t r u n n k o v , Jeffrey K n g s b u r y , n d D o u g l s K o s h l n d <br /> Carnegie Institution Washington, Department Embryology, Baltimore, Maryland 21210 <br />  <br />  <br />  <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span>. We designed screen identify mu-                                 binding domain (binuclear Zn-cluster). At nonpermis- <br /> tants specifically affecting kinetochore function                            sire temperature cep3 cells arrest un- <br /> yeast Saccharomyces cerevisiae. The selection proce-                                divided nucleus short mitotic spindle. At per- <br /> dure based generation &quot;synthetic acen-                                missive temperature cep3 cells unable <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> <br /> tric&quot; minichromosomes. &quot;Synthetic acentric&quot; minichro-                               support segregation minichromosomes muta- <br /> mosomes contain centromere locus, lack                                        tions central element III yeast centro- <br /> centromere activity combination mutations                                 mere DNA. These minichromosomes, isolated <br /> in centromere DNA chromosomal gene (CEP)                                   cep3 cultures, fail bind bovine microtubules in <br /> encoding putative ce__ntromere p_rotein. Ten condi-                               vitro. The sum genetic, cytological biochemical <br /> tional lethal cep mutants isolated, seven                                 data lead suggest Cep3 protein a <br /> found alleles NDCIO (CEP2) encoding                                    DNA-binding component yeast centromere. Molecu- <br />  l l0-kD protein yeast kinetochore. Three mutants                                lar mass sequence comparison confirm Cep3p <br /> defined novel essential gene CEP3. The CEP3 prod-                                 p64 component centromere DNA binding <br /> uct (Cep3p) 71-kD protein potential DNA-                                complex Cbf3 (Lechner, 1994). <br />  <br />  <br />  <br />        HROMOSOMEsegregation mitosis requires pres-                           Ris, 1976). These facts outline unique status S. <br />  <br /> C       ence specialized chromosomal structure that <br />         facilitates binding chromosomes microtu- <br /> bules (mt) ~ mitotic spindle. Functionally, struc- <br />                                                                                     cerevisiae centromere list model segregation loci. <br />                                                                                        Upon identification centromere locus yeast <br />                                                                                     chromosomes (Clarke Carbon, 1980), efforts sev- <br /> ture defined linker DNA microtu-                          eral groups focused identification proteins as- <br /> bules, known centromere, kinetochore (used                         sociated centromere DNA vivo, conjunction with <br /> interchangeably here). Structural organization centro-                           dissection structural properties CenDNA itself. It <br /> meres varies dramatically evolutionary remote spe-                            suggested yeast CenDNA subdivided <br /> cies extremely complex (Brinkley et al., 1992;                           distinct structural elements (recently reviewed in <br /> Grady et al., 1992). This complexity makes structure-                           Hegemann Fleig, 1993). The first, CDEI associated <br /> function analysis centromere difficult challenge.                       CP1/Cbflp/Cpflp protein (Bram Kornberg, <br /> Fortunately, centromere budding yeast, Sac-                              1987; Baker et al., 1989; Cai Davis, 1990; Mellor et al., <br /> charomyces cerevisiae, (Clarke Carbon, 1980) provides                           1990). Both CDEI CP1 dispensable centromere <br /> an excellent model studying centromere organization                         function standard laboratory conditions. The sec- <br /> activity. The centromere DNA (CenDNA) S. cerevisiae                              ond element, CDEII, A- T-rich region protected <br /> is 150 bp <span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> (Bloom Carbon, 1982),                             nuclease digestion and/or modification vivo (Bloom <br /> drastically smaller CenDNA studied eu-                           et al., 1984; Densmore et al., 1991). The CDEII element is <br /> karyotes (Pluta et al., 1990). The CenDNA yeast pro-                        required proper centromere function, no <br /> vides space binding limited number protein                         data, far, indicating directly binds specific <br /> molecules. Also, single microtubule attached                           polypeptide. Finally, CDEIII element absolutely es- <br /> each kinetochore cells budding yeast (Peterson                        sential centromere assembly activity, vivo <br />                                                                                     vitro (Hegemann et al., 1988; Lechner Carbon, <br /> Address correspondence A. V. Strunnikov, Carnegie Institution              1991; McGrew et al., 1986; Panzeri et al., 1985). The mul- <br /> Washington, Department Embryology, 115 West University Parkway,                  tisubunit CDEIII-binding complex Cbf3 purified <br /> Baltimore, MD 21210. Ph.: (410) 554-1224. Fax: (410) 243-6311.                      partially characterized (Lechner Carbon, 1991). <br />  1. Abbreviations used paper: CenDNA, centromere DNA; ha, <br />                                                                                     The Cbf3 complex major components: p58, p64, <br /> hemagglutinin; rot, microtubules; ORF, open reading frame; ts, tempera-             p110. As subunits purified individu- <br /> ture-sensitive.                                                                     ally, primary biochemical activity of un- <br />  <br />  <br /> &#194;&#169; The Rockefeller University Press, 0021-9525/95/03/749/12 $2.00 <br /> The Journal Cell Biology, Volume 128, Number 5, March 1995 749-760                749 <br />  known. The genes encoding p58 (CTF13) (Doheny et al.,                           blot hybridization. After sporulation, segregation viable (His3-) to <br /> 1993) pll0 (NDC10/CBF2/CTF14) (Doheny et al., 1993;                         inviable (His + ) spores observed tetrad. The His3 + spores <br />                                                                                 recovered transformation AS270 pAS452 prior to <br /> Goh Kilmartin, 1993; Jiang et al., 1993a) recently                     tetrad analysis. Haploid His3+ strains (cep3-Al) carried the <br /> been characterized. The p58 pll0 proteins (henceforth,                      pAS452 (CEP3) plasmid. <br /> referred Ctfl3p Ndcl0p, respectively) in- <br /> dispensable viability yeast cell. The cells                       Nomenclature <br /> temperature-sensitive (ts) ndclO-1 mutant exhibit cell <br />                                                                                 Standard nomenclature (Jones et al., 1992) yeast genes proteins was <br /> cycle arrest nonpermissive temperature, instead un-                     used. As yeast centromere factors corresponding genes are <br /> dergo abortive mitosis, leaving chromosomal                          known redundant names (CP, Cep, Cpf, Cbf, Ctf, Ndc) chose the <br /> DNA unsegregated associated asymmetric spin-                        existing &quot;CEP&quot; acronym (Baker Masison, 1990) genes isolated <br /> dle (Goh Kilmartin, 1993). In contrast, known                      screen. Centromere mutations designated using locus <br />                                                                                 name, CDE number, particular <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span>mutation</span> name, e.g., cen6-(llI)15t. The <br /> ts allele ctf13 (Doheny et al., 1993) causes cells arrest                 CDEIII mutations used shown Fig. 1 B. <br /> at nonpermissive temperature G2-M undivided <br /> nucleus short mitotic spindle.                                            The Screening Procedure Isolation cep Mutants <br />    Clearly, identification additional protein compo- <br /> nents yeast kinetochore necessary understand mo-                   A schematic view selection procedure given Fig. 1. The first <br />                                                                                 stage screening included monitoring plasmids different reporter <br /> lecular structure function. We expect kinetochore                      CenDNA identify suitable large scale mutagenesis. Strain <br /> proteins characteristic features exemplified                    YPH102 transformed corresponding plasmid containing leu2-d <br /> by CP1, Ctfl3p Ndcl0p. First, physically                     gene CenDNA interest. The cultures mntagenized EMS <br /> associated directly indirectly CenDNA and/                       (50-90% viability). Cells plated form single colonies ab- <br />                                                                                 sence selection leucine prototrophs. The plates replica- <br /> or spindle microtubules. Second, cells&apos; deprived                       printed media lacking leucine incubated 48 h. 23&#194;&#176;C. The <br /> kinetochore protein cytological defects                        frequency Leu + colonies determined reporter CenDNA <br /> consistent inability segregate chromosomes                          tested, corresponding colonies picked master plates. The <br /> properly. Third, trans-mutations genes encoding                      trans-mntations distinguished mutant classes retrans- <br /> centromere components genetic interaction                           formation minichromosome. The plasmid pAS97 (see be- <br />                                                                                 low), exhibited highest rate induced trans-mutations and, at <br /> with cis-CenDNA mutations (cen). These features provide                         time, lowest rate spontaneous Leu +, chosen large- <br /> at criteria recognize particular protein                  scale mutagenesis. Only mutants reproducibly showing Leo + <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span>phenotype</span> <br /> kinetochore component. Some proteins satisfy                     temperature sensitivity chosen analysis. All mutant <br /> one criteria so, potential candidates                  alleles crossed twice, demonstrate generated <br />                                                                                 mutations single mutations nuclear origin, temperature sen- <br /> kinetochore components: p64 (Lechner Carbon, 1991),                         sitivity cosegregates meiosis accumulation pAS9Z <br /> Mckl (Shero Hieter, 1991), Csel Cse2 (Xiao et al., <br /> 1993), Cbf5 (Jiang et al., 1993b), Top2 (Jiang et al., 1993b),                  Plasmid Construction <br /> Kar3 (Middleton Carbon, 1994), Mif2 (Brown <br /> et al., 1993). However, DNA binding proteins essential                      Yeast-E. coli shuttle vectors, used cloning purposes pRS vectors <br />                                                                                 (Christianson et al., 1992; Sikorski Hieter, 1989). pT7blue (Novagen, <br /> for assembly functional kinetochore remain unidenti-                         Inc., Madison, WI) used cloning vector PCR-generated frag- <br /> fied. Particularly, proteins make direct contacts                    ments. <br /> CDEIII important, CDEIII absolutely required                         Plasmid pAS93, backbone of minichromosomes used in <br /> centromere function. Based belief yeast cen-                    mutagenesis experiments, constructed FAT-RS303 (leu2-d, HIS&amp; <br /> tromere complex multisubunit structure,                          21zm-ORl, Amp~) (contribution D. Gottschling, University Chicago, <br />                                                                                 Chicago, IL). The H1S3 marker removed digestion BamHI and <br /> structural components found, designed                       religation; URA3 gene (SmaI-I-lindlII fragment) ARSH4 (HindIII- <br /> genetic screen aimed directly identifying proteins involved                  XhoI fragment) inserted PvuII-XhoI sites make pAS93, pAS93 <br /> in formation yeast centromeres.                                          unique BamHI, SalI, Sinai sites, used cloning CenDNA frag- <br />                                                                                 ments, giving following minichromosomes: pAS94 (cen3-(ll)X35 <br />                                                                                 BamHI-BamHI fragment); pAS95 (GALI:CEN3 BamHI-BamHI fragment); <br /> Materials Methods                                                           pAS96 (cen6-(l)Sc SalI-BamHI fragment); pAS97 (cen6-(lll)15t SalI- <br />                                                                                 BamHI fragment); pAS98 (cen3-(lll)BCTl SalI-SalI fragment); pASI12 <br /> Strains Genetic Techniques                                                  (cen3-(lll)BCT2 SalI-SalI fragment); pASll3 (cen6-(lll)lgt20 SalI-BamHI <br />                                                                                 fragment); pASll4 (cen6-(lll)14a SalI-BamHI fragment); pAS122 (CEN6 <br /> S. cerevisiae strains ($288C background) listed Table I. Strain          SalI-BamHI fragment), pAS76 CEN4 XhoI-Seal fragment inserted <br /> YPH102 used mutagenesis, YPH499 used backcrosses             SalI-SmaI sites YEpFAT7 (Runge Zakian, 1989). The corre- <br /> centromere proteins mutants (cep). Strain $42 (Doheny et al., 1993)         sponding pDK minichromosomes (without leu2-d 2tun-ORl) been <br /> used tester ndclO allelism. Media, incubation conditions, strain   described (Kingsbury Koshland, 1991): pDK381 contains CEN6 <br /> manipulations according published protocols (Sherman et al.,            SalI-BamHI fragment, pDK377 contains cen6-(lll)lSt SalI-BamHI frag- <br /> 1986). Escherichia coli strains DH5t~ (BRL), TOP10 (Invitrogen, San             ment, pDK371 contains cen3-(lll)BCTl SalI-SalI fragment, pDK378 con- <br /> Diego, CA) SCSI10 (Stratagene, La Jolla, CA) used plasmid          tains cen6-(lll)-19t20 SalI-BamHI fragment, pDK374 +contains cen3-(ll) <br /> propagation. Standard methods yeast genetics described          X78 BamHI-BamHI fragment pDK380 contains cen6-(l)8c SalI- <br /> (Guthrie Fink, 1991; Sherman et al., 1986). Mitotic stability plas-      BamHI fragment. Descriptions CenDNA fragments used pub- <br /> mids, defined fraction cells culture retain plasmid     lished previously (Hegemann et al., 1988; Hill Bloom, 1987; McGrew <br /> when cells grown selective conditions, estimated de-       et al., 1986). <br /> scribed previously (Strunnikov et al., 1993). As parameter correlates          Several plasmids, containing CEP3 gene original isolate <br /> well efficiency plasmid transmission cell division (Hieter et   pAS300, constructed, pAS409 (pRS414 backbone) contains the <br /> al., 1985a; Koshland et al., 1985), used routine estimate   entire genomic insert pAS300 9-kb EagI-BamHI fragment, pAS420 <br /> efficiency minichromosome transmission. The disruption allele (cep3-         pRS414 3.2-kb BssHII-EcoRI fragment pAS300. HincII-HincII <br /> A/) cep3 constructed transforming AS260 diploid strain       fragment pAS420 (Fig. 2 A) introduced pR406, pRS425, and <br /> pAS450 (see below) digested EagI Xbal, giving AS270. Disruption        pRS416 cut Pvull producing pAS451, pAS452 pAS461, respec- <br /> of copy CEP3 gene HIS3 marker confirmed Southern             tively. To construct deletion CEP3, PCR fragments corresponding <br />  <br />  <br />  <br />  <br /> The Journal Cell Biology, Volume 128, 1995                                    750 <br />  Table L <br /> Strain                                                                    Genotype                                                                 Source <br />  <br /> YPH102                                  MATa ade2 his3 leu2 lys2 ura3                                                                           Ph. Hieter <br /> YPH499                                  MATa ade2 his3 teu2 lys2 trpl ura3                                                                      Ph. Hieter <br /> $42                                     MATer cq74(ndc10)-42 ade2 leu2 lys2 ura3 his3                                                           Ph. Hieter <br /> AS260                                   MATa/MATc~ ade2 his3 leu2 lys2 trpl /TRP1 ura3                                                          This study <br /> AS270                                   MATa/MATc~ cep3-A1/CEP3 ade2 his3 leu2 lys2 trpl /TRP1 ura3                                             This study <br /> ldAS255                                 MATa cep3-1 ade2 his3 leu2 lys2 trpl ura3                                                               This study <br /> 6bAS255                                 MATa cep3-1 ade2 his3 leu2 lys2 trpl ura3                                                               This study <br /> lcAS281                                 MATa cep3-1 ade2 his3 leu2 lys2 trpl ura3                                                               This study <br /> 2bAS282                                 MATa cep3-2 ade2 his3 leu2 lys2 trpl ura3                                                               This study <br /> 3dAS282                                 MATa cep3-2 ade2 his3 leu2 lys2 trpl ura3                                                               This study <br /> 2aAS257                                 MATc~ cep3-3 ade2 his3 leu2 lys2 ura3                                                                   This study <br />  <br />  <br />  <br /> to upstream noncoding downstream noncoding regions CEP30RF                    antibody 12CA5 (BAbCO, Richmond CA) used monitor tagged <br /> were cloned pT7blue, making pAS446/1. In plasmid CEP3 open             Cep3p Western blots. The pooled monoclonal antibodies 90- <br /> reading frame (ORF) substituted single BamHl recognition site,              KD component spindle pole body (Rout Kilmartin, 1990), were <br /> which HIS3 gene (BamHI-BamHI fragment) inserted,                     used undiluted indirect immunofluoreseent staining. <br /> pAS450. <br />    To construct epitope-tagged versions CEP3 pAS451 plasmid               Minichromosome-Microtubule Binding Assays <br /> was cut SpeI HindIII (Fig. 2 A). In-frame insertion tan- <br /> dem tag sequences encoding c-myc epitope (Strunnikov et al., 1993)               The microtubule-minichromosome binding assays performed essen- <br /> into SpeI site produced pAS454 (cep3::myc allele). In-flame insertion             tially described (Kingsbury Koshland, 1991) minor modifica- <br /> a triple tag sequence (contribution M. Rose, Princeton University,                tions (Kingsbury Koshland, 1993). Binding experiments for <br /> Princeton, NJ; HindIII-ends introduced PCR) encoding hemagglutinin              parallel cultures, including YPHI02/pDK381 strain as <br /> epitope (ha) HindIII site pAS451 produced pAS464 (CEP3::ha al-               control. For minichromosome/strain combination (Fig. 6) inde- <br /> lele). Correct boundaries CEP3 tag-encoding fragments                pendent binding experiments performed twice. <br /> were confirmed DNA sequencing. <br />  <br /> Cloning, Mapping, SequenceAnalysis                                    Results <br /> CEP3 Gene <br /> The genomic DNA library used isolate CEP3 gene                    Isolation cep Mutants <br /> LEU2/CEN library (Spencer et al., 1988) pSB32 vector. Two ts strains, <br /> 14-YPH102 (cep3-1), 10-YPH102 (cep3-3) transformed li- <br />                                                                                  The extrachromosomal replicon identified yeast was <br /> brary plasmid DNA. Cells spread plates leucine in-           endogenous 2 #m plasmid (Beggs, 1978). It circular <br /> cubated 23&#194;&#176;C 24 h. The plates shifted 37&#194;&#176;C,              multicopy (20-30) double-stranded DNA molecule which <br /> temperature-resistant colonies picked. Total DNA extracted          uses specialized partition effectively during <br /> used transform E. coli cells. Two plasmids enable 14-           mitotic divisions (Broach Volkert, 1991). In contrast, <br /> YPH102 10-YPH102 grow 37&#194;&#176;C. These plasmids com- <br /> plemented ts-<span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span>lethal phenotype</span> back-crossed cep3 strains. Restriction      artificial minichromosomes centromere locus, segre- <br /> mapping showed, clones contain largely overlapping inserts           gate using mitotic apparatus designed chromosomes <br /> genomic DNA. The plasmid chosen subcloning designated pAS300.            maintained single copy cell. The screen for <br />     Mapping cloned fragment chromosome accomplished              genes encoding putative centromere proteins based on <br /> using EagI-BamHI fragment pAS300 hybridization probe for <br /> filters containing yeast chromosomes separated PFGE. A single hybrid- <br />                                                                                  observation presence 2#m plasmid repli- <br /> ization signal, corresponding chromosome XIII, detected. Prelimi-         cation/segregation locus (2#M-ORI) yeast centromere <br /> nary sequencing starting vector unique internal re-       circular plasmid (2#m/CEN plasmid) creates <br /> striction sites used accurate mapping clone. The cloned      epistatic relationship loci (Tschumper <br /> DNA fragment contained SMP2 (Irie et al., 1993) MLHI (Prolla et al., <br /> 1994), known genes chromosome XIII. CEP3 gene immediate <br />                                                                                  Carbon, 1983). Such plasmid behaves single- <br /> neighbor MLH1 order open reading frames fragment            copy minichromosome multicopy 2#m plasmid <br /> is follows: CEP3&gt;,&lt;MLttl . . . . . SMP2&gt;.                                     (Tschumper Carbon, 1983). Several reports been <br />     To determine nucleotide sequence CEP3, nested deletions        published compromising centromere function <br /> pAS420 plasmid constructed (Henikoff, 1984). The resulting plasmids         minichromosome convert multicopy <br /> were sequenced using fluorescent ddNTP PCR sequencing reaction and <br /> automated DNA sequencer (model 370A; Applied Biosystems, Foster City,            equivalent 2#m plasmid, cells the <br /> CA), recommended manufacturer. Both strands CEP3 se-           population (Apostol Greer, 1988; Chlebowicz-Sled- <br /> quenced. Conflicting regions resequenced using custom synthe-           ziewska Sledziewski, 1985; Schulman Bloom, <br /> sized 18-24-met primers.                                                         1993). These results provided primary basis design <br />                                                                                  screen (Fig. 1 A). <br /> Antibodies Cytological Methods                                                  The second rationale cep screen use a <br /> Indirect immunofluorescence performed described (Kilmartin            dose-dependent marker follow changes copy num- <br /> Adams, 1984). Yeast nuclear DNA stained DAPI included                ber 2#m/CEN plasmid. Such markers used <br /> mounting media (Mowiol; Cal Biochem, La Jolla, CA). Microtubules            previously applications genetic screens (Hieter <br /> detected mouse monoclonal antibody YOLl/34 (1:200) (Kilmartin <br /> et al., 1982) goat anti-mouse antibodies conjugated rhodamine (Cap-       et al., 1985b; Larionov et al., 1989; Runge et al., 1991; <br /> pel Laboratories, Cochranville, PA). Mouse monoclonal anti-c-myc anti-           Smith et al., 1990; Toh-E, 1981). In screen cep mu- <br /> body 9El0 (Evan et al., 1985) mouse monoclonal anti-hemagglutinin            tants used leu2-d gene marker. The leu2-d <br />  <br />  <br />  <br />  <br /> Strunnikov et al. CEP3/Genefor Centromere Protein                                  751 <br />                                                                                              Figure 1. (A) Design screen for <br />                                                                                             cep mutants. The schematic map of <br />                                                                                             minichromosome used select <br />                                                                                             cep mutants shown cen6- <br />                                                                                             (III)15t reporter. The &quot;cep locus&quot; <br />                                                                                             designation stands putative <br />                                                                                             target mutagenesis yeast chro- <br />                                                                                             mosome. After cep mutation is <br />                                                                                             acquired cell, defective <br />                                                                                             product (Cep-) mutated gene <br />                                                                                             interferes compromised <br />                                                                                             reporter cen6 DNA, allowing ac- <br />                                                                                             cumulation minichromosomes <br />                                                                                             appearance Leu+ cells. <br />                                                                                             (B) CDEIII mutations used assess <br />                                                                                             function cep genes. Nucleo- <br />                                                                                             tides numbered starting <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br />                                                                                             <span id='am-239' about='xsp:base' typeof='owl:Thing'>base</span> 1 CDEIII. The CDEIII palin- <br />                                                                                             drome marked half- <br />                                                                                             arrows central base out- <br />                                                                                             lined. The mutant centromere DNA <br />                                                                                             fragments resequenced to <br />                                                                                             confirm structure. <br />  <br />  <br />  <br /> fragment truncation LEU2 promoter, making       wild-type CenDNA retain centromere activity <br /> expression single leu2-d gene insufficient comple-     allow proper chromosome segregation. <br /> ment chromosomal leu2 mutation. Approximately 100                To identify CenDNA mutations appropriate level <br /> copies leu2-d gene Leu 2- cell required        partial centromere activity screen, constructed <br /> confer Leu &#195;&#183; phenotype. That copy number easily         strains containing 2#rn/CEN minichromosomes with <br /> achieved yeast cells amplification 2#m based        different CenDNA mutations. We tested strains <br /> leu2-d plasmid, cells selected Leu + pheno-     generation Leu + clones mutagen- <br /> type (Toh-E, 1981). This amplification possible        esis. Mutagenesis strains harboring 2#m/CEN plasmids <br /> flexibility 2/zm plasmid copy number control, allowing        cen6-(1)Sc (dysfunctional CDEI) did signifi- <br /> transition normal 20-30 copies cell 100-150          numbers Leu + clones mutagenesis. The ac- <br /> copies (under leu2-d selection) deleterious effect   tivity reporter centromere apparently similar <br /> on cell physiology. Due epistatic effect mentioned        wild-type hence, discussed above, failed cep <br /> above, 2#m/CEN/leu2-d minichromosome un-             mutants. Strains 2/zm/CEN plasmids GALl:: <br /> dergo amplification copy cell 100 copies      CEN3 (tanscriptionally inactivated centromere), cen3-(lll)- <br /> cell abolishing centromere function completely           BCT1, cen3-(lll)BCT2, cen3-(lI)X35, cen6-(IIl)14a had <br /> (Schulman Bloom, 1993). This loss function         high rate spontaneous Leu + clones. These re- <br /> plasmid-born CEN locus (reporter centromere) result      porter centromeres apparently compromised much <br /> unlinked (chromosomal) mutation cep locus (Fig. 1 A)        allowed reporter plasmid achieve high <br /> is essence screen described below.                    copy number absence cep mutants. However strains <br />    Mutagenesis YPH102 cells contained 2/~m/          contained 2#m/CEN plasmids cen6-(lll) <br /> CEN minichromosome wild-type CenDNA (CEN4              15t cen6-(lll)19t20 undetectable levels spontaneous <br /> CEN6) failed produce trans-mutations inacti-       Leu + clones numerous Leu + clones mutagene- <br /> vate reporter centromere. The likely explanation     sis. These induced Leu + clones good candidates for <br /> this failure cep mutations inactivated    cep mutants, indicating CenDNA muta- <br /> the reporter centromere plasmid inactivated      tions work reporter screen. <br /> the chromosomal centromeres. Such cep mutants              The minichromosome pAS97 (Fig. 1 A), having cen6- <br /> be recovered loss viability. To overcome prob-     (III)15t reporter CEN, primarily used screen <br /> lem, decided compromise reporter centromere         described report (See Materials Methods). When <br /> using CenDNA mutations retain partial centro-      YPH102/pAS97 strain mutagenized, estimated <br /> mere activity vivo. We rationalized modification    frequency induced Leu &#195;&#183; clones carrying cep mutations <br /> would allow isolation putative cep mutants displaying     9 &#195;&#8212; 10-4. The unwanted Leu + clones, including the <br /> synergistic loss reporter centromere activity (&quot;syn-      cis-mutants clones rearranged minichromosomes, <br /> thetic acentric&quot; phenotype) compound effect cis    originated similar frequency. More 30 cep mu- <br /> and trans mutations. However, cep mutants         tants isolated. 10 conditional lethal mu- <br /> viable chromosomal centromeres            tants recessive temperature-sensitive growth. These 10 <br />  <br />  <br /> The Journal Cell Biology, Volume 128, 1995                    752 <br />  A                                                                                                                                                                                                                  mutations fell complementation groups designated <br />                                                                                                                                                                                                      SO <br />                                                                                                                                                                                                                    cep2 (seven alleles: cep2-1I cep2-17) cep3 (three al- <br />                                                                                                                                                                                                      145 <br />                                                                                                                                                                                                                    leles: cep3-1 cep3-3). All seven cep2 alleles failed to <br />                                                                                                                                                                                                      1) <br />                                                                                                                                                                                                                    complement ndc10-42 temperature sensitivity, indicating <br />      1 4 ~ &#226;&#8364;&#162; V ~ T I t R K V                                                              ~ t m <br />                                                                                                                                                                                                      a05 <br />                                                                                                                                                                                                      )3            cep2 allelic ndclO gene. Assuming, the <br />                                                                                           B    T        ~   L       I <br />                                                                                                                                                                                                      a65 <br />                                                                                                                                                                                                      53 <br />                                                                                                                                                                                                                    NDC10 gene encodes centromere protein, isolation mul- <br />      3 4 R K I t                                                         MK <br />                                                                                                                                                                                                                    tiple ndclO alleles cep-screen indicates approach <br />                                                                                                             M       Y            I           Y           W       I               T                   73 <br />                                                                                                                                                                                                                    selective centromere-specific genes. <br />      7 4 N I G L Y K T                                                   I t V                                                                                                                       93 <br />  <br />                                                                                                                                                                                                      44S           CEP3 Is Novel Essential Gene <br />      ~ 4 T D T E I t C ~                                F          W     MN <br />  <br />                                                                                                                                                                                                      505           Both CEP2 CEP3 genes cloned complementa- <br />                                                                                           L    Y        r       ~       S            I           ~           D           I                           133 <br />                                                                                                                                                                                                                    tion corresponding ts-mutations. Plasmids carrying <br />  134                       L Y L R V                                     Y M                                R       A        D           K           N                                               iS) <br />                                                                                                                                                                                                                    inserts putative wild type genes conferred tempera- <br />                                                                                           S    V            ~       T        ~           C           I       Y               ~                                     ture resistant growth mutant strains. We isolated two <br />     I?4NP                                                          l g &#226;&#8364;&#162;                                                                                                                             19) <br />                                                                                                                                                                                                                    independent DNA fragments complemented the <br />                                                                                                                                                                                                      745 <br />                                                                                                                                                                                                                    cep3-1 cep3-3 mutations. Standard genetic procedures <br />     tgl                               L     1~                     p     &#226;&#8364;&#162;           It    O       ],       v           W                                        l~                  r    A <br />                                                                                                                                                                                                                    performed plasmids con- <br />                                                                                                                                                                                                      SOS <br />                                                                                                                                                                                                      253           tained CEP3 gene. First, showed integration <br />     231                    &amp;    Y     L     I                                        T     P       P        &#194;&#165;           D                                        I,                  L    A <br />                                                                                                                                                                                                      8;5 <br />                                                                                                                                                                                                      a53 <br />                                                                                                                                                                                                                    URA3 marker chromosomal locus corresponding <br />                                                                                                                                                                                                      9~5 <br />                                                                                                                                                                                                                    cloned fragment segregated away cep3 <br />     2S4 N &amp; L L T                           &apos;0   C       l         :4    T           P     X       N        F           R                V           D           D                   Ie   R          :73 <br />                                                                                                                                                                                                                    mutation meiotic progeny diploid cells, heterozy- <br />     274 P            L     L    It    D     D                                        X     A       K        v           T                                        R                   I    1e         2~3           gous insertion cep3. This result shows the <br />     .294 <br />       . . . . . ~t   !~C~C cC&apos;I~~ &apos;l&apos;O&quot;g <br />                                   CD C~ L        .D. . . . Y       bCT&apos;T ~&apos;J~ ~TO~ ~               PG~&apos;AR~C ~                            CC~ <br />                                                                                                                                          &quot; I A&quot;PI&apos; <br />                                                                                                                                              I &apos; T~ , ~                                   .H. . . . . , 1 ,        gene cloned corresponds chromosomal locus of <br />                                                                                                                                                                                                                    cep3 ts mutation, ectopic suppressor gene. Sec- <br />     a~4 T            s     v     s    &#226;&#8364;&#162;     L    L       Q         ~     A           k     y       L        Q           D                                                                 D           333 <br />                                                                                                                                                                                                                    ond, using stepwise subcloning procedure, localized the <br />     334 V            V     R     B    B     el    $      ,r           v           L     Y       w        K           :[               x               S       L                   0    It          3S~          complementing activity single ORF (see below). Minor <br /> ..... T .... <br />  35~ D  L   O <br />                                  CAA TAC TT~ ~ c <br />                                  Q   Y   L   N <br />                                                          . <br />                                                          K <br />                                                                .   . . <br />                                                                    $   ~1 <br />                                                                                      KAA COT C~C TTA AA~ ACA TI~ C~. . . . . . . <br />                                                                                      X p P L         It T    L   D    &#226;&#8364;&#162;      I                                                                        3?3 <br />                                                                                                                                                                                                               25   truncations putative CEP30RF 5&apos; 3&apos; ends <br />                                                                                                                                                                                                                    abolished complementing activity, indicating had <br />     374 R                        t.   h     ~     v      o         ~     X           v     ~                            R                R               D                                            393 <br />                                                                                                                                                                                                                    identified authentic CEP3 ORE The cloned gene also <br /> 1286 AAT ~                 AGC Aft&amp; TIT CAA ~            . . . .                     LCTT FTTT . . . . . . .                                             VGTT ~                      T~G KAVA 1345 <br />     391 ~                      ~    &#226;&#8364;&#162;                    ~   &#226;&#8364;&#162;   A                             Q   I    ~   T                                                 S                      w        ~13                  complemented &quot;synthetic acentric&quot; phenotype three <br /> ..... <br />  414 L <br />                      FTTT KAAG ATG 7. . . . . . . . . . . . . <br />                                14  ~&apos;    L       I      Y     Y          O <br />                                                                                      TAlC A~CA . . . . . . . . . . . . . . . . . . . <br />                                                                                                D      s      L      L      K      v                                                  1    H           g33 <br />                                                                                                                                                                                                                S   cep3 mutants. <br /> z,o~ r ~             rer   ~a    ~^   ~*e ^rr     mr     err       ~rr   m-r ~             ~       M-re c A , ~ e ~ ~ . ~ m . r e ~                                                  rr&apos;r rrc         ,~              The DNA sequence 2,074-bp fragment containing the <br />     434 ~            $     K     v    1     l     s      L         I     v           ~     &#226;&#8364;&#162;       ~        H           A                x               s           t               ~    P           *S~ <br />                                                                                                                                                                                                                    CEP3 gene determined. The sequence CEP3 frag- <br /> ......               RAGA BCAT ~C~ A . . . .             ~tA TACCATT . . . . . .                                        TCT pT~C . . . . . . . . . .                                                      , <br />     ~54 N                      P   N    v    M                   ~[ T      v   v                                               I      8      ~e y                                                   4~1~         ment revealed 608-codon ORF, potentially encoding a <br />     474 0            X     r                                             X           Q     L                            O                L                                                            493 <br />                                                                                                                                                                                                                    novel protein molecular mass 71 kD (Fig. 2 A). The <br />                                                                                                                                                                                                                    amino terminus predicted polypeptide contained binu- <br />     49d A            /~    L     ~    T     ~     p      ~ $             Ic          L     ~       ~                    v                Y               ~           T               ~    R           513 <br />  <br /> I~. . . . . .                    &apos;i&apos;?A A A ~ C&apos;L&apos;? &apos;l~rA TGC; f t . . . .                  CA~ CTT CTA &#226;&#8364;&#162;AT TCA GGT GAT T~G TTT                                                                        1905 <br />                                                                                                                                                                                                                    clear zinc-cluster motif Zn(II)2Cys~ (Pan Coleman, <br />  ~14 L   S    K                  L     K     L     L     W    D     K    v                 ~ L L       S   SF G D D                                                                                   533           1990) (Fig. 2 B) common DNA-binding proteins <br />     S~         Y     H                      K     I       L        O     x           D     l       It       l           l                I~                                                           55) <br />                                                                                                                                                                                                                    (Dhawale Lane, 1993). The rest Cep3p did show <br />                                                                                                                                                          h~r~IH <br />                                                                                                                                                                                                                    significant similarity known protein sequences. Ac- <br />      S54 E           I(    r     S    L      I    K       o f ,          C       S         ~       vN       ~~                                           t           L               &apos;~   Q           5~3 <br />                                                                                                                                                                                                                    cording secondary structure prediction (Rost Sander, <br />      574 E           S     L     L                        D         |        M       ~     T        &#226;&#8364;&#162;       P           $                D                                           T     I          593           1994) 56% polypeptide aminoacid residues in <br />  ..... <br />   594 v <br />         &amp;. . . . . <br />          g   E  &#226;&#8364;&#162;                     U <br />                                             $&apos;~ ~         T&amp;T . . . . . . . . . <br />                                                           Y        II              s     D        I       L <br />                                                                                                                         STCC T . . . . . . . . . . . <br />                                                                                                                                                                                                       609 <br />                                                                                                                                                                                                                    alpha-helix, 37 % form loops. Only short region of <br /> 1950 TATCCTT&apos;I~AA&amp;~GAAT(~T~_~___~-~_%~-AT&amp;CA~T~AAC-AC-AATC&amp;CA~r~AOq~AAT                                                                                                                               2029         protein predicted (Lupas et al., 1991) form coiled- <br />  203~ ~CAAAAAAC&apos;FI~TATTA~.~C~sC A~%AC~TX&apos;~TATJ~GW~TA~C                                                                                                                                                 207~        coil structure probability higher 0.3 (Fig. 2 A). <br />                                                                                                                                                                                                                        The chromosomal copy CEP3 gene disrupted <br />                                                                                                                                                                                                                    substituting CEP30RF HIS3 marker. We <br />                                                                                                                                                                                                                     failed recover resulting allele, cep3-Al, haploid <br />                                                                                                                                                                                                                     strain, showing CEP3 essential gene. The es- <br />                                                                                                                                                                                                                     sential nature CEP3 presence DNA-binding <br />                                                                                                                                                                                                                     motif suggest Cep3p new DNA-binding component <br /> Figure 2. Sequence analysis CEP3 gene. (,4) The primary                                                                                                                                                     yeast centromere. <br /> DNA sequence BssHII-Hincll fragment plasmid pAS420                                                                                                                                                           To assess expression level CEP3, introduced two <br /> the putative polypeptide product shown. Cysteine residues                                                                                                                                                    alternative in-frame epitope tags (See Materials Meth- <br /> the putative DNA-binding domain italicized. Recognition sites <br /> for SpeI HindlII endonucleases, used construct tagged ver-                                                                                                                                                  ods) CO&#226;&#8364;&#162;H-terminal Cep3p (Fig. 2 A). The <br /> sions Cep3p, shown. Protein residues 367-402, able form                                                                                                                                                   cep3::myc construct did complement cep3-1, cep3-2, or <br /> coil-coil structure, underlined. These sequence data avail-                                                                                                                                                 cep3-Al defects. This result indicaties essential re- <br /> able GenBank/EMBL/DDBJunder accession number U12339.                                                                                                                                                           gion Cep3p limited putative Zn-binding clus- <br /> (B) Alignment metal-binding cluster Cep3p known fun-                                                                                                                                                   ter. However alternative CEP3::ha construct did comple- <br /> gal proteins: Mal2p (Kopetzki et al., 1989), Pdr3p (Genbank/                                                                                                                                                        ment cep3-1 cep3-Al mutations. The Cep3p rare <br /> EMBL/DDBJ accession number X79803), Gal4p (Marmorstein et                                                                                                                                                           protein Cep3p-ha protein detected West- <br /> al., 1992), Haplp (Zhang Guarente, 1994). The identical                                                                                                                                                         ern blots concentrated immunoprecipitation (data <br /> residues conserved cysteins highlighted.                                                                                                                                                                    shown). <br />  <br />  <br /> Strunnikov                 et   al.   CEP3/Gene f o r Centromere Protein                                                                                                                                         753 <br />                                          ,o <br /> A                                                                       Cytological Analysis cep3 Mutants <br />                                                                         Detailed cytological analysis cep3 mutants limited <br />                                                                         cep3-1 cep3-2, cep3-3 cells showed practically <br />                                                                         identical characteristics preliminary tests. When the <br />                                                                         wild-type cep3 cells shifted 37&#194;&#176;C (the non- <br />  ~   20                                 20 <br />                                                                         permissive temperature cep3), viability remained <br />                                                                         4 h. However, 8 h viability cep3 cells <br />                                                                         dropped 20 %. Time-course observations showed, that <br />       0   ,      ,      ,     ,          0           ,       ,     , <br />                                                                         6-h shift nonpermissive temperature cep3 <br />           o     eyeT         6P                 o                 6P    cells reach arrest ofceU division (Fig. 3 A). In course <br />                  cep3-1           []   25&quot;c              cep3-2         incubation 37&#194;&#176;C, late anaphase cells small- <br />                                                                         budded cells, present 25&#194;&#176;C (Fig. 4 A), disappear. The <br />                                   &#226;&#8364;&#162;    3TC (6 hr.) <br />                                                                         large-budded (dumbbell) cell undivided nucleus at <br /> B                                                                       bud neck short spindle (Fig. 4 B) the <br />                                                                         dominating class cep3 population (Fig. 3 A). Most of <br />                                                                         cep3 cells arrested 37&#194;&#176;C diploid DNA content <br />                                                                         (Fig 3 B). This complex features highly suggestive of <br />                                                                         G2-M phase arrest. The virtual absence elongated mitotic <br />                                                                         spindles, verified spindle pole body staining (Fig. <br />                                                                         5 C). Therefore phenotype cep3 similar to <br />                                                                         ctfl3-30 cells (Doheny et al., 1993) ndclO-1 mutant <br />                                                                         (Goh Kilmartin, 1993). <br />                                                                            The staining spindle microtubules cep3 cells lack- <br />                                                                         ing mitochondrial DNA revealed dominant <br />                                                                         class cells arrested cep3 population homoge- <br />                                                                         neous. We indications abnormal distribu- <br />                                                                         tion nuclear DNA, including separation chro- <br />                                                                         mosomal DNA mitotic spindle, nuclear DNA cut <br />                                                                         cytokinesis, nuclear DNA unevenly spread be- <br />                                                                         tween mother daughter cell (Fig. 5, A B). The <br />                                                                         phenotype probably spindle movements <br />                                               ..41 <br />                                                                         known phenomenon nuclear transits (Palmer et al., <br />                                                                          1989), accompanied failure majority chro- <br />                                                                         mosomes attach mitotic spindle. The presence of <br />                                                                         cells spindles nuclear DNA positioned the <br />                                                                         different cell bodies (Fig. 5, A B) suggests ar- <br />                                                                         rested cep3 ceils chromosomes fail maintain attachment <br />                                                                         microtubules mitotic spindle. <br />                                                                            Interestingly, restrictive temperature ex- <br />                                                                         amined cep2 (ndclO) mutants arrest like cep3 cells (data <br />                                                                         shown). This finding brings possibility the <br />                                                                         cep3, ctfl3 (Doheny et al., 1993) ndclO (Goh Kil- <br />                                                                         martin, 1993) cells display kind mitotic ar- <br />                                                                         rest consequence centromere inactivation. <br />  <br />                                                                         Interaction cep3 Mutations Centromere <br />                                                                         Mutations In Vivo <br />                                                                         Given synthetic destabilization reporter centro- <br />                                                                         mere, cen6-(IIl)15t, cep3 mutants presence of <br /> Figure 3. Characterization cep3cultures arrested nonper-      DNA binding motif Cep3p, reasonable postu- <br /> missive temperature. (A) The cell types cep3-1 (6bAS255)         late Cep3p CDEIII-specific DNA-binding protein. <br /> cep3-2 (2bAS282, 3dAS282) cultures grouped               To investigate possibility determined specificity <br /> classes according morphology DAPI-stained cells. The          interaction cep3 mutations different <br /> first classes present wild type cells. The fourth     CenDNA mutations, cep3 mutants transformed with <br /> class (large-budded cells undivided nucleus) generated       pDK minichromosomes (URA3, ARS1, CEN minlchromo- <br /> cep3culture exposure high temperature. Distribution of <br /> cell types cep3cultures permissive temperature different <br /> from population CEP3cells, presence large <br /> budded cells minor fraction. (B) Flow cytometry analysis           RNAse treatment. The accumulation G2-M cells double <br /> (FACS) ofa cep3-1culture 0, 2, 4, 6 h shift 37&#194;&#176;C.    (2c) content DNA evident. The &quot;lc~ peak corresponds G1 <br /> Cells haploid cep3-1strain (6bAS255) fixed ethanol       cells. FACS profiles cep3-2 strains look identical (not <br /> at time points shown, stained propidium iodide       shown) profile cep3-1 strain. <br />  <br />  <br />  <br />  <br /> The Journal Cell Biology,Volume 128, 1995                            754 <br />  Figure 4. Micrograph cep3 cells permissive restrictive temperatures. (A) The logarithmic population cep3-2 cells <br /> (3dAS282) 25&#194;&#176;C. DNA stained DAPI microtubules stained anti-tubulin antibodies. Differential interference <br /> contrast (DIC) images general cell shape. The late anaphase cells elongated mitotic spindles, small-budded cells <br /> are present population. (B) The cep3-2 cells (2bAS282 strain) 6 h 37&#194;&#176;C. The dominating class large budded cells with <br /> an undivided nucleus short spindle evident. Bar, 10 #m. <br />  <br />  <br /> somes 2#m DNA), differed                  Therefore, Ndcl0p directly indirectly partici- <br /> CenDNA sequences (Table II). We tested mitotic              pate CDEIII binding (see Discussion). On hand, <br /> stability plasmids means assay function         smcl-1 mutation, affects minichromosome <br /> of centromeres (Table II). It evident cep3        transmission (Strunnikov et al., 1993), did exhibit the <br /> mutations inactivated centromeres trans mini-            synthetic acentric phenotype centromeres harbor- <br /> chromosome CenDNA contained CDEIII mutations                     ing CenDNA mutations (data shown). Therefore, <br /> (cen6-(III)15t cen3-(llI)BCT1). However, cep3 muta-          synthetic acentic phenotype general property of <br /> tions did inactivate significantly centromeres an-          mutation affects minichromosome transmission. <br /> other CDEIII mutation (cen6-(lll)lgt20), CDEI mutation             The similar <span id='am-240' about='xsp:pattern' typeof='owl:Thing'>pattern</span> interaction centromere DNA mu- <br /> (cen3-(1)8c) CDEII mutation (cen3-(ll)X78). There-              tations permissive temperature displayed cep2 <br /> fore, cep3 mutations appear interact synergistically          cep3 mutants suggests corresponding proteins <br /> only centromeres specific mu-          act close proximity vivo. <br /> tations symmetrically flanking central base CDEIII <br /> palindrome (Fig. 1 B). This specific interaction supports the <br /> hypothesis CEP3 product CDEIII binding                 The cep3 Mutation Interferes Centroraere <br /> protein.                                                             Function In Vitro <br />    As cep2 (ndclO) mutants identified               The kinetochore S. cerevisiae acts tripartite sys- <br /> screen cep3 mutants, analyze            tem (CenDNA-kinetochore proteins-microtubules) ac- <br /> mitotic stability minichromosomes mutant                 quires motility time mitosis. Based concept, <br /> CenDNA cep2 mutants well. Similar inactivation              types biochemical assays applied previ- <br /> centromeres specific CenDNA mutations observed              ously testing yeast centromere function vitro: pro- <br /> in cep2 mutants observed cep3 mutants (Table II).          tein-DNA interaction (Doheny et al., 1993; Ng Carbon, <br />  <br />  <br />  <br /> Strunnikov et al. CEP3/Gene f o r Centromere Protein                755 <br />  Figure 5. Arrested population cep3-1 cells lacking mitochondria. (A) The cep3-1 (1cAS281) culture lacking mitochondrial DNA (rho&#194;&#176;) <br /> after 6 h 37&#194;&#176;C. The cells stained DAPI anti-tubulin antibodies. The cell marked arrow underwent partial <br /> separation chromosomal DNA; bulk chromosomal DNA spindle separated positioned different <br /> cell bodies, phenotype observed wild type ceils. (B) The rho&#194;&#176; cep3-1 cells (lcAS281) representing abnormal subtypes of <br /> the large budded cells undivided nucleus. From bottom: separation chromosomes mitotic spindle; cut-like <br /> phenotype; formation potential aploid cell. Bar, 10 #m. (C) Spindle pole body staining cep3-2 cells. 2bAS282 cells stained <br /> with antibodies 90-kD SPB component 6 h. 37&#194;&#176;C. The position SPBs shows corresponding mitotic spindles <br /> remain bipolar (no spindle collapse) elongate, cells chromosomal DNA spread mother and <br /> daughter cells. Bars, 10 #m. <br />  <br />  <br /> 1987), centromere-microtubule interaction (Kingsbury               assay described previously (Kingsbury Kosh- <br /> Koshland, 1991; Kingsbury Koshland, 1993), motil-              land, 1991, 1993). It involves quantitative precipitation of <br /> ity assays (Hyman et al., 1992; Middleton Carbon,                  minichromosomes isolated yeast cells (in form of <br /> 1994). To reveal molecular basis &quot;synthetic acen-           chromatin) association taxol-stabilized bovine <br /> tric&quot; phenotype, carried experiments testing fidel-         microtubules. All experimental data presented report <br /> ity centromere binding bovine microtubules cell-            obtained Cep3 &#195;&#183; Cep3- cells arrested with <br /> free system. This minichromosome-microtubule binding                   nocodazole prior preparation extracts contain- <br />  <br />  <br />  <br />  <br /> The Journal Cell Biology, Volume 128, 1995                          756 <br />  Table II. <br />                                                            Mitotic stability minichromosomes reporter centromere: <br />  <br /> Strain               CEN6                cen6- (111)15t              cen3- (III)BCTI                    cen6- (11I)19t20                                    cen3- (11)X78                   cen6- (I)8c <br /> Cep +             88.9   +    4.3        85.4   +    6.2              77.0   +    5.0                   84.8       +      3.8                               86.3      +    4.5            87.9     5-    0.9 <br /> cep3-1            86.3   +    4.2         6.0   5-   3.8               3.6   5-   3.5                   78.7       5-     7.0                               80.3      5-   1.8            80.0     5-    4.0 <br /> cep3-2            87.4   5-   0.8         9.3   5-   2.1              11.8   5-   5,1                   81.3       5-     2.7                               83.3      5-   4.5            65.4     -t-   13.2 <br /> cep2-12           81.8   5-   3.9        10.3   5-   2.6              15.8   5-   2.2                   63.9       5-     17.8                              83.7      5-   4.3            65.8     5-    5.0 <br />  <br />  <br />  <br />  <br /> ing minichromosomes. Arrest nocodazole (G2-M tran-                                 failure centromere function; (d) amino acid se- <br /> sition), previously shown (Kingsbury Koshland,                               quence Cep3 protein identical sequence of <br /> 1991), confers highest mt-binding potential wild type                           p64 Cbf3 complex (Lechner, 1994). The fact, <br /> cultures. This provides wide range comparison                              Cep3p Cep2p, subunits Cbf3 com- <br /> experimental data obtained mutant wild type extracts                            plex, shows screen selective genes en- <br /> and CenDNA. The G2-M transition phase                                  coding components yeast centromere. <br /> the function centromere actually required progres- <br /> sion normal cell division cycle mitosis, i.e., <br /> when kinetochore active state. In fact, com- <br />                                                                                         Discussion <br /> parison mt-binding activity cis-mutant centromeres <br />                                                                                         Developing Selective Primary Screen for <br /> from asynchronous culture (Kingsbury Koshland,                                  Kinetochore Components <br /> 1991) activity nocodazole arrested cells (Fig. 6) sug- <br /> gests centromeres assembled cis-mutant Cen-                              Centromere proteins previously identified cells <br /> DNA (e.g., cen6-(IIl)15t) achieve wild-type ac-                             vertebrates centromere (or kinetochore) antigens <br /> tivity cells arrested G2-M.                                                   (Compton et al., 1991; Earnshaw et al., 1987a, b; Yen et al., <br />    Four types cultures (grown 23&#194;&#176;C) used mi-                             1992). The yeast centromere proteins identified by <br /> crotubule-binding experiments: Cep3 &#195;&#183; cells wild type                              biochemical properties (CenDNA association) (Jiang et <br /> CEN 6 minichromosome, Cep3 &#195;&#183; cells mutant Cen-                                     al., 1993a; Lechner Carbon, 1991) secondary <br /> DNA (cen6-(lll)15t) minichromosome, Cep3- cells                                    screening existing collections temperature-sensitive <br /> wild type CEN6 minichromosome Cep3- cells                                      mutants (Goh Kilmartin, 1993) mutants frequent <br /> mutant CenDNA (cen6-(Ill)15t) minichromosome (Fig. 6).                                  chromosome loss (Doheny et al., 1993). However, identi- <br /> These combinations analogous ones used assaying                              fication additional centromere components ap- <br /> minichromosome stability vivo (Table U). For cep3/                              proaches proved limited, mainly labor <br /> CEN6 CEP3/cen6 combinations, slight difference                                intensive. In report described new approach for <br /> in capacity minichromosome-microtubule binding                                   identifying centromere proteins selecting mutants dis- <br /> was observed 23&#194;&#176;C (Fig. 6). However, combining cep3                              playing &quot;synthetic acentric&quot; phenotype. Using ap- <br /> (trans) cen (cis) mutation cell produced                              proach isolated multiple mutations genes en- <br /> striking effect: microtubule-minichromosome binding <br /> became indistinguishable background level (i.e., of <br /> acentric plasmid precipitation; Kingsbury Koshland,                                 O <br />                                                                                                                                                                     ~            CEP3/CEN6 <br />  1991). This result indicates intimate link be-                              40-                                    ............... o              ....o...     CEP3/cen6-(lll)         15t <br /> tween structure CDEIII Cep3p function. These <br /> cell-free experiments agreement data ob-                             .ta 30- <br />                                                                                                            ....&quot; . . . . . . . . . . . . .                    &lt;3 ,,.&lt;3 ....      cep3-1/CEN6 <br /> tained vivo minichromosomes. Results                              E <br /> experiments provide basis hypothesis addressing                             o     20. <br />                                                                                         O <br /> mechanism &quot;synthetic acentric&quot; phenotype: syn-                               E <br /> thetic acentrics probably generated vivo,                             2 <br />                                                                                         c- 10-                                               ~,t,,, &#226;&#8364;&#162; <br />                                                                                                                                                         ...... []   .,.o.,.      cep3-1/cen6-(lll)15t <br /> inability cen6-(lll)15t minichromosomes bind yeast                                o <br />                                                                                         c <br /> microtubules.                                                                           ~      o <br />    The structural properties Cep3p, cytologi-                         ~           o           2&apos;.s                     ~               T&apos;.s <br />  cal, genetic, biochemical data obtained cep3 mu-                                                          mt (xl0~/ml) <br />  tants, provide basis assuming Cep3 protein the <br />  CDEIII-binding subunit yeast centromere, indispensable                              Figure 6. Minichromosome-microtubule binding experiments. <br />  kinetochore assembly function. To summarize,                              Four types extracts used microtubule binding experiments: <br />  properties Cep3p suggest centromere                            CEP3/CEN6, strain YPHI02 transformed pDK381; CEP3/ <br />  protein: (a) mutant forms protein allele specific <br />                                                                                         cen6-(lll)15t, YPH102 pDK377 minichromosome; cep3-1/ <br />                                                                                         CEN6, ldAS255 pDK381 minichromosome; cep3-1/cen6- <br />  interactions particular CenDNA mutations; (b) mini-                               (lll)15t, ldAS255 pDK377 minichromosome. Binding mt <br />  chromosomes isolated cep3 cells exhibit defect                           concentrations, higher required saturation binding in <br />  centromere-dependent binding microtubules vitro; (c)                             wild type strain (Kingsbury Koshland, 1991), ex- <br />  Cep3 protein essential cell viability, cep3                         clude possible rescue binding activity cep3 mutant extracts <br />  mutants exhibit complex mitotic defects consistent                              excess rot. <br />  <br />  <br />  <br /> Strunnikov et al, CEP3/Gene Centromere Protein                                    757 <br />  coding probable centromere components;               CDEHI mutations cep3 alleles (this study). One possi- <br /> genes, NDCIO (CEP2), identified previously (Goh          ble explanation phenomenon Ndcl0p also <br /> and Kilmartin, 1993; Jiang et al., 1993a) other,        binds nucleotides CDEIII does Cep3p. <br /> CEP3, novel gene.                                            However, unlikely, Ndcl0p Cep3p not <br />    The improved efficiency approach relies        share significant sequence similarity Ndcl0p lacks any <br /> major principles. First, requiring reporter minichro-      known DNA-binding motif. More likely, Ndcl0p co- <br /> mosome achieve high copy number, selected            factor Cep3p binding CDEIII. Alternatively, Ndcl0p <br /> only mutants completely inactivate reporter        unconventional DNA-binding protein affecting <br /> centromere, eliminating possible background mu-       Cep3p close proximity Cep3p Ndcl0p bind- <br /> tants, able accumulate extra copies minichro-      ing sites. These models verified biochemical <br /> mosome (Larionov et al., 1989). Second, used reporter        studies, however, genetic data provide evidence <br /> centromere partially impaired cis-          Cep3p Ndcl0p physically interact vivo. <br /> mutation. In way mutant centromere proteins, inac-         What hypothetical way construct func- <br /> tivated reporter centromere, active         tional centromere, taking account Cep3p the <br /> chromosomal centromeres, allowing mutant cells sur-        known DNA-binding motif Cbf3 compo- <br /> vive. Third, assumed kinetochore pro-          nents? One possibility is, Cep3p Zn-cluster solely <br /> teins essential viability, imposed re-        responsible tethering centromere components to <br /> quirement cep mutants conditionally lethal.           CenDNA. This unlikely binding of <br />    The &quot;synthetic acentric&quot; approach obvious     kinetochore proteins CenDNA special re- <br /> virtues future studies centromere proteins. First,     quirements withstand pulling forces exerted <br /> it provides rapid efficient means isolate mutant    mitosis. The interaction CDEHI rel- <br /> alleles genes encoding pll0 p64 components          atively simple DNA-binding motif Cep3p, likely, would <br /> the kinetochore. These mutations serve powerful            sufficient purpose. Also, CDEII, CDEIII <br /> tools dissecting molecular organization cellu-    flanking sequences form large special chromatin struc- <br /> lar role proteins. For example, new alleles      ture (Bloom et al., 1984), likely DNA- <br /> ndclO (cep2) isolated synthetic acentric approach (see     binding components exist. Based fact, CDEIII <br /> below) phenotypically distinct known ndclO alleles.      proteins bound key elements the <br /> Second, approach saturated           assembly entire complex (mutations CDEUI can <br /> cen6-(lll)15t reporter, evidenced isolation multi-   abolish centromere function [Hegemann et al., 1988; <br /> ple alleles ndclO cep3, reporter CenDNA se-          McGrew et al., 1986] prevent formation centromere <br /> quences CDEII CDEIII mutations used             chromatin [Saunders et al., 1988]), suggest the <br /> identify genes encoding additional centromere proteins. For       DNA-binding motif Cep3p recognizes CDEIII during <br /> example, cen6-(llI)19t20 mutation synthetically        replication centromere chromatin (&quot;centromere priming&quot; <br /> acentric combined ndclO cep3 alleles,        role) nucleates assembly centromere pro- <br /> potential cep mutants obtained centromere       teins. These proteins (including Cep3p) locked <br /> a reporter (Strunnikov, A. V., unpublished results).              CenDNA using special mechanism, be <br />                                                                   identified. For example, relative richness Cep3p in <br />                                                                   cysteine residues (beyond Zn cluster) provides possibil- <br /> The CEP3 Gene Encodes Putative                                  ity disulfide bonds covalently lock Cep3p as <br /> CenDNA-binding Protein That Is Necessary                      subunits wrapped CenDNA. Partial <br /> Microtubule-binding Activity                                   DNA-wrapping documented Gal4 protein <br /> the Centroraere                                                   previously (Marmorstein et al., 1992). <br /> DNA sequence analysis revealed CEP3 encodes pro-              In addition putative DNA-binding properties of <br /> tein predicted NH2-terminal DNA-binding domain             Cep3p vivo, examined role Cep3p organizing <br /> Zn(II)2Cys6 (Pan Coleman, 1990). In fungal pro-         microtubule-binding activity kinetochore. We showed <br /> teins domain mediates DNA binding sequence-             minichromosomes isolated cep3 cells exhibit <br /> specific manner transcription          dramatic defect centromere-dependent binding mini- <br /> factors-like proteins, dispensable standard laboratory      chromosomes microtubules vitro. This observation has <br /> conditions. The characterized member family,         important implications. First, validates use iso- <br /> Gal4p, binds palindromic DNA sequence dimer (Mar-          lated minichromosome means identify charac- <br /> morstein et al., 1992). Interestingly, CenDNA mutations           terize structural components kinetochore, required for <br /> that interact cep3 alleles lie symmetrical positions   microtubule binding. Second, cep3-1 described <br /> the CDEHI palindrome, suggesting Cep3p            mutation Cbf3 component disrupts microtubule <br /> bind DNA dimer. Previously, limited functional sym-        binding activity centromere, corroborating impor- <br /> metry CDEIII described (Jehn et al., 1991) based        tance Cbf3 complex centromere structure function <br /> data obtained cis mutations.                                  (Lechner Carbon, 1991; Hyman et al., 1992). Third, the <br />     Of major proteins Cbf3 complex, Cep3p     microtubule-binding data suggest molecular model the <br /> only subunit evident DNA-binding motif. Therefore,        &quot;synthetic acentric&quot; phenomenon. While theoretically <br /> it reasonable assume Cep3p mediates binding     possible Cep3p directly binds microtubules, more <br /> Cbf3 complex CDEIII. Surprisingly, tested cep2             likely presence Cep3p needed assemble a <br /> (ndclO) mutations similar pattern interaction        multi-protein complex mt-binding subunits, ac- <br />  <br />  <br />  <br />  <br /> The lournai Cell Biology, Volume 128, 1995                     758 <br />   cordance model published previously (Kingsbury                    References <br />  Koshland, 1993). This hypothetical &quot;centromere prim-                   Apostol, B., C. L. Greer. 1988. Copy number stability yeast 2m- <br />  ing&quot; role Cep3 protein, verified additional bio-                 based plasmids carrying transcription-conditional centromere. Gene. <br />  chemical experiments, help identify centro-                     67:59-68. <br />                                                                             Baker, R. E., D. C. Masison. 1990. Isolation geae encoding Sac- <br />  mere proteins physically associated Cep3p.                                charomyces cerevisiae centromere-binding protein CPI. Mol. Cell. Biol. <br />                                                                                 16:2458-2467. <br />  Cellular Response Loss Centromere Function                       Baker, R. E., M. Fitzgerald-Hayes, T. C. O&apos;Brien. 1989. Purification of <br />                                                                                 yeast ceatromere binding protein CP1 mutational analysis its <br />  The cells multicellular organisms developed sensitive                     binding site. J. Biol. Chem. 264:10843-10850. <br />                                                                             Beggs, J. D. 1978. Transformation yeast replicating hybrid plasmid. <br />  mechanism delays anaphase chromosome                         Nature (Lond.). 275:104-109. <br />  fails achieve metaphase position (Nicklas Arana,                Bloom, K. S., J. Carbon. 1982. Yeast centromere DNA unique and <br />  1992). It suggested delay reflects &quot;check-                highly ordered structure chromosomes small circular minichromo- <br />                                                                                 somes. Cell. 29:305-317. <br />  point&quot; monitors chromosome congression kineto-                    Bloom, K. S., E. Amaya, J. Carbon, L. Clarke, A. Hill, E. Yeh. 1984. <br />  chore-microtubule attachment. As aspects cell cycle                   Chromatin conformation yeast centromeres. J. Cell Biol. 99:1559-1568. <br />  regulation S. cerevisiae quite different mul-               Brain, R. J., R. D. Komberg. 1987. Isolation Saccharomyces cerevisiae <br />                                                                                centromere DNA-binding protein, human homolog, possible role <br />  ticellular organisms (Nasmyth, 1993), possible loss                transcription factor. MoL Cell. Biol. 7:403-409. <br />  ofkinetochore function generate cell cycle delay.              Brinkley, B. R., I. Ouspenski, R. P. Zinkowski. 1992. Structure molec- <br />                                                                                ular organization centromere-kinetoehore complex. Trends Cell Biol. <br />  In fact delay observed ndclO-1 cells, mu-                 2:15-21. <br />  tants demonstrated defective essential yeast cen-              Broach, J. R., F. C. Volkert. 1991. Circular DNA plasmids yeasts. In <br />  tromere protein (Goh Kilmartin, 1993). At restrictive                     The Molecular Cellular Biology Yeast Saccharomyces. J. R. <br />                                                                                Broach, J. R. Pringle, E. W. Jones, editors. Cold Spring Harbor Labora- <br />  temperature cells undergo asymmetric elongation                      tory Press, Cold Spring Harbor, NY. 297-331. <br />  mitotic spindle leaving chromosomes unseg-                 Brown, M. T., L. Goetsch, L. H. Hartwell. 1993. MIF2 required for <br />  regated. However, experiments CenDNA mutations                           mitotic spindle integrity anaphase spindle elongation Saccharo- <br />                                                                                myces cerevisiae. J. Cell Biol. 123:387--403. <br />  (Spencer Hieter, 1992) analysis c~3-30 mutant                   Cai, M., R. W. Davis. 1990. Yeast centromere binding protein CBFI, of <br />  suggested yeast cells respond loss centro-               helix-loop-helix protein family, required chromosome stability and <br />                                                                                methionine prototrophy. Cell. 61:437--446. <br />  mere function classical fashion. The inactivation               Chlebowicz-Sledziewska, E., A. Z. Sledziewski. 1985. Construction of <br />  Ctfl3p results metaphase-like delay elongation                   multicopy yeast plasmids regulated centromere function. Gene. 39: <br /> of mitotic spindle. Since CTH3 encodes essential                        25-31. <br />                                                                            Christianson, T. W., R. S. Sikorski, M. Dante, J. H. Shero, P. Hieter. <br />  Cbf3 component, c0~13-30arrest reflect authen-                    1992. Multifunctional yeast high-copy-number shuttle vectors. Gene. 110: <br /> tic &quot;checkpoint&quot; response loss centromere in-                     119-122. <br /> tegrity (Dobeny et al., 1993). Our data obtained                 Clarke, L., J. Carbon. 1980. Isolation yeast centromere construc- <br />                                                                                tion functional small circular chromosomes. Nature (Lond.). 287:504- <br /> independent alleles ofcep3 support idea. The arrest                 509. <br /> in cep3 mutant cells phenomenologically close pub-               Compton, D. A., T. J. Yen, D. W. Cleveland. 1991. Identification novel <br /> lished description ctfl3-arrested cells. Moreover                 centromere/kinetochore-associated proteins using monoclonal antibodies <br />                                                                               generated human mitotic chromosome scaffolds. J. Cell Biol. 112: <br /> an indication arrest cep3 cells probably                 1083-1097. <br /> the separation chromosomes mitotic spindle. If                 Densmore, L., W. E. Payne, M. Fitzgerald-Hayes. 1991. In vivo genomic <br />                                                                               footprint yeast ceatromere. Mol. Cell. Biol. 11:154-165. <br /> arrest observed cep3 c0&apos;13 cells checkpoint acti-              Dhawale, S. S., A. C. Lane. 1993. Compilation sequence-specific DNA- <br /> vated inability chromosomes establish stable                     binding proteins implicated transcriptional control fungi. Nucleic Acids <br /> contacts kinetochore microtubules, ndclO-1 mutant                    Res. 21:5537-5546. <br />                                                                            Doheny, K., P. Sorger, A. Hyman, S. Tugendreich, F. Spencer, P. Hieter. <br /> could retain partial kinetochore activity, resulting                1993. Identification essential componenets S. cerevisiae <br /> catastrophic anaphase. This interpretation makes control                  kinetochore. Cell. 73:761-774. <br /> of chromosome segregation S. cerevisiae relevant                Earnshaw, W. C., P. S. Machlin, B. J. Bordwell, N. F. Rothfield, D. W. <br />                                                                               Cleveland. 1987a. Analysis anticentromere autoantibodies using cloned <br /> the metaphase-anaphase control cells Metazoa.                       autoantigen CENP-B. Proc. Natl. Acad. Sci. USA. 84:4979-4983. <br />    In conclusion, described effective non-laborious              Earnshaw, W. C., K. F. Sullivan, P. S. Machlin, C. A. Cooke, D. A. Kaiser, <br />                                                                               T. D. Pollard, N. F. Rothfield, D. W. Cleveland. 1987b. Molecular <br /> screen cep genes. This screen allowed isolation                    cloning cDNA CENP-B, major human centromere autoantigen. <br /> CEP3 gene, new essential gene encoding putative centro-                   J. Cell Biol. 104:817-829. <br /> mere component yeast, new alleles CEP2                Evan, G. I., G. K. Lewis, G. Ramsey, J. M. Bishop. 1985. Isolation of <br />                                                                               monoclonal antibodies specific human c-myc proto-oncogene product. <br /> (NDCIO) gene, encoding known centromere protein.                            Mol. Cell. Biol. 5:3610-3616. <br /> These results bring closer understanding centro-                 Goh, P. Y., J. Kilmartin. 1993. NDCIO: gene involved chromosome <br /> mere structure, however, proteins need identified.                 segregation S. cerevisiae. J. Cell Biol. 121:503-512. <br />                                                                            Grady, D. L., R. L. Ratliff, D. L. Robinson, E. C. McCanlies, J. Meyne, and <br /> Our based generation synthetic acentric mu-                  R. K. Moyzis. 1992. Highly conserved repetitive DNA sequences pres- <br /> tants useful identification un-                ent human centromeres. Proc. Natl. Acad. Sci. USA. 89:1695-1699. <br />                                                                            Guthrie, C., G. R. Fink, editors. 1991. Guide Yeast Genetics Molec- <br /> known centromere proteins.                                                    ular Biology. Academic Press, Inc., New York. 933 pp. <br />                                                                            Hegemann, J. H., U. N. Fleig. 1993. The centromere budding yeast. <br />                                                                               BioEssays. 15:451-460. <br /> We like thank Tami Kingsbury, Vincent Guacci, Ayumu <br />                                                                            Hegemann, J. H., J. H. Shero, G. Cottarel, P. Philippsen, P. Hieter. 1988. <br /> Yamamoto critical reading manuscript. We thank Johannes       Mutational analysis centromere DNA chromosome VI Sac- <br /> Lechner communicating p64 sequence prior publication. Finally,      charomyces cerevisiae. Mol. Cell. Biol. 8:2523-2528. <br /> thank Cristine Norman help preparation manuscript.           Henikoff, S. 1984. Unidirectional digestion exonuclease III creates tar- <br />                                                                              geted breakpoints DNA sequencing. Gene. 28:351-359. <br />    This work supported grant National Institutes Health <br />                                                                            Hieter, P., C. Mann, M. Snyder, R. Davis. 1985a. Mitotic stability yeast <br /> (GM-41718) D. Koshland.                                                   chromosomes: A colony color assay measures nondisjanction chro- <br />                                                                              mosome loss. Cell. 40:381-392. <br /> Received publication 20 September 1994 revised form 6 Decem-    Hieter, P., D. Pridmore, J. H. Hegemann, M. Thomas, R. W. Davis, P. <br /> ber 1994.                                                                    Philippsen. 1985b. Functional selection analysis yeast centromeric <br />  <br />  <br />  <br />  <br /> Strunnikov et al. CEP3/Gene Centromere Protein                       759 <br />      DNA. Cell. 42:913-921.                                                           Cell Sci. 102:681-690. <br /> Hill, A., K. Bloom. 1987. Genetic manipulation centromere function.        Pan, T., J. E. Coleman. 1990. GAL4 transcription factor &quot;zinc <br />     Mol. Cell. Biol. 7:2397-2405.                                                    finger&quot; forms Zn(II)2Cys6 binuclear cluster. Proc. Natl. Acad. Sci. <br /> Hyman, A. A., K. Middleton, M. Centola, T. J. Mitchison, J. Carbon.              USA. 87:2077-2081. <br />     1992. Microtubule-motor activity yeast centromere-binding protein        Palmer, R. E., M. Koval, D. Koshland. 1989. The dynamics chromo- <br />     complex. Nature (Lond.). 359:533-536.                                            movement budding yeast Saccharomyces cerevisiae. J. CellBioL <br /> Irie, K., M. Takase, H. Araki, Y. Oshima. 1993. A gene, SMP2, involved           109:3355-3366. <br />     plasmid maintenance respiration Saccharomyces cerevisiae encodes    Panzeri, L., L. Landonio, A. Stotz, P. Philippsen. 1985. Role conserved <br />     highly charged protein. Mol. Gen. Genet. 236:283-288.                          sequence dements yeast centromere DNA. EMBO (Eur. Mol. Biol. Or- <br /> Jehn, B., R. Niedenthal, J. H. Hegemann. 1991. In vivo analysis           gan.) J. 4:1867-1874. <br />     Saccharomyces cerevisiae centromere CDEIII sequence: Requirements         Peterson, J. B., H. Ris. 1976. Electron microscopic study spindle <br />     mitotic chromosome segregation. MoL Cell. Biol. 11:5212-5221.                    chromosome movement yeast Saccharomyces cerevisiae. J. Cell <br /> Jiang, W., J. Lechner, J. Carbon. 1993a. Isolation characterization       Sci. 22:219-242. <br />     gene (CBF2) specifying protein component budding yeast             Pluta, A. F., C. A. Cooke, W. C. Earnshaw. 1990. Structure human <br />     kinetochore. J. Cell Biol. 121:513-519.                                          centromere metaphase. Trends Biochem. Sci. 15:181-185. <br /> Jiang, W., K. Middleton, H.-J. Yoon, C. Fouquet, J. Carbon. 1993b. An         Prolla, T. A., D. Cristie, R. M. Liskay. 1994. Dual requirement yeast <br />     essential yeast protein, CBF5p, binds vitro centromeres microtu-       DNA mismatch repair MLItl PMS1, homologs bacterial <br />     bules. Mol. Cell. Biol. 13:4884-4893.                                            mutL gene. Mol. Cell. Biol. 14:407--415. <br /> Jones, E. W., J. R. Pringle, J. R. Broach, editors. 1992. The Molecular       Rost, B., C. Sander. 1994. Combining evolutionary information neural <br />     Cellular Biology yeast Saccharomyces. Cold Spring Harbor Press,       networks predict protein secondary structure. Proteins Struct. Funct. Ge- <br />     Cold Spring Harbor, NY.                                                          net. 19:55-72. <br /> Kilmartin, J., A. Adams. 1984. Structural rearrangements tubulin       Rout, M. P., J. V. Kilmartin. 1990. Components yeast spindle and <br />     actin cell cycle yeast Saccharomyces. J. Cell Biol.            spindle pole body. J. Cell Biol. 111 : 1913-1927. <br />     98:922-933.                                                                   Runge, K. W., V. A. Zakian. 1989. Introduction extra telomeric DNA <br /> Kilmartin, J. V., B. Wright, C. Milstein. 1982. Rat monoclonal antitubulin       sequences Saccharomyces cerevisiae results telomere elongation. <br />     antibodies derived using new nonsecreting rat cell line. J. Cell Biol.      Mol. Cell. BioL 9:1488-1497. <br />     93:576--582.                                                                  Runge, K. W., R. J. Wellinger, V. A. Zakian. 1991. Effects excess cen- <br /> Kingsbury, J., D. Koshland. 1991. Centromere-dependent binding yeast          tromeres excess telomeres chromosome loss rates. Mol. Cell. Biol. <br />     minichromosomes microtubules vitro. Cell. 66:483-495.                       11:2919-2928. <br /> Kingsbury, J., D. Koshland. 1993. Centromere Function Minichromo-          Sannders, M., M. Fitzgerald-Hayes, K. Bloom. 1988. Chromatin structure <br />     somes Isolated Budding Yeast. Mol. Biol. Cell. In press.                    altered yeast eentromeres. Proc. Natl. Acad. Sci. USA. 85:175-179. <br /> Kopetzki, E., E. Zellner, G. Schumacher, F. K. Zimmermann. 1989.              Schulmen, I. G., K. Bloom. 1993. Genetic dissection centromere func- <br />     Nucleotide sequence Saccharomyces cerevisiae positive regulator mu-       tion. Mol. Cell. Biol. 13:3156-3166. <br />     tant gene MAL2-Scp. Nucleic Acids Res. 17:5390.                               Sherman, F., G. Fink, J. Hicks. 1986. Methods Yeast Genetics. Cold <br /> Koshland, D., J. C. Kent, L. H. Hartwell. 1985. Genetic analysis          Spring Harbor Laboratory, Cold Spring Harbor, NY. 186 pp. <br />     mitotic transmission minichromosomes. Cell. 40:393-403.                    Shero, J., P. Hieter. 1991. A suppressor centromere DNA mutation <br /> Larionov, V. L., N. Y. Kouprina, A. V. Strunnikov, A. V. Vlasov. 1989.           encodes putative protein kinase (MCK1). Genes Dev. 5:549-560. <br />     A direct selection procedure isolating yeast mutants impaired     Sikorski, R. S., P. Hieter. 1989. A shuttle vectors yeast host <br />     segregation artificial minichromosomes. Curt. Genet. 15:17-25.                strains designed efficient manipulation DNA Saccharomyces <br /> Lechner, J. 1994. A zinc finger protein, essential chromosome segregation,       cerevisiae. Genet. 122:19-27. <br />     constitutes putative DNA binding subunit Saccharomyces cerevisiae    Smith, D. R., A. P. Smyth, D. T. Moir. 1990. Amplification large <br />     kinetocbore complex, Cbf3. EMBO (Fur. Mol. Biol. Organ.). 13:5203-               artificial chromosomes. Proc. Natl. Acad. Sci. USA. 87:8242-8246. <br />     521 I.                                                                        Spencer, F., P. Hieter. 1992. Centromere DNA mutations induce mitotic <br /> Lechner, J., J. Carbon. 1991. A 240 kd Multisubunit Protein Complex,             delay Saccharomyces cerevisiae. Proc. Natl. Acad. Sci. USA. 89:8908- <br />     CBF3, Major Component Budding Yeast Centromere. Cell.                8912. <br />     64:717-725.                                                                   Spencer, F., C. Connelly, S. Lee, P. Hieter. 1988. Isolation cloning <br /> Lupas, A., M. VanDyke, J. Stock. 1991. Predicting coiled coils pro-         conditionally lethal chromosome transmission fidelity genes Sac- <br />     tein sequences. Science (Wash. DC). 252:1162-1164.                               charomyces cerevisiae. In Cancer Cells. Vol. 6: Eukaryotic DNA Replica- <br /> Marmorstein, R., M. Carey, M. Ptashne, S. C. Harrison. 1992. DNA rec-            tion. T. Kelly, B. Stillman, editors. Cold Spring Harbor Laboratory, <br />     ognition GAL4: structure protein-DNA complex. Nature (Lond.).            Cold Spring Harbor, NY. 441-452. <br />     356:408-414.                                                                  Strunnikov, A. V., V. L. Larionov, D. Koshland. 1993. SMCI: An essen- <br /> McGrew, J., B. Diehl, M. Fitzgerald-Hayes. 1986. Single base-pair muta-          tial yeast gane encoding putative head-rod-tail protein required nu- <br />     tions centromere element III cause aberrant chromosome segregation         clear division defines new ubiquitous protein family. J. Cell Biol. <br />     Saccharomyces cerevisiae. Mol. Cell. Biol. 6:530-538.                             123:1635-1648. <br /> Mellor, J., W. Jiang, M. Funk, J. Rathjan, C. A. Barnes, T. Hinz, J. H.           Toh-E, A. t981. Curing 2m DNA plasmid Saccharomyces <br />      Hegemann, P. Philippsen. 1990. CPFI, yeast protein functions        cerevisiae. J. Bacteriol. 145:1421-1424. <br />      centromeres promoters. EMBO (Eur. Mol. Biol. Organ.) J. 8:            Tschumper, G., J. Carbon. 1983. Copy number control yeast centro- <br />     4017--4026.                                                                      mere. Gene. 23:221-232. <br />  Middleton, K., J. Carbon. 1994. KAR3 kinesin minus-end-directed mo-     Xiao, Z., J. T. McGrew, A. J. Schroeder, M. Fitzgerald-Hayes. 1993. <br />     tor functions centromere binding proteins (CBF3) vitro         CSE1 CSE2, new genes required accurate mitotic chromosome <br />      yeast kinetochore. Proc. Natl. Acad. Sci. USA. 91:7212-7216.                     segregation Saccharomyces cerevisiae. Mol. Cell. Biol. 13:4691-4702. <br />  Nasmyth, K. 1993. Control yeast cell cycle Cdc28 prote <br /> </body></html>