<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR AND CELLULAR BIOLOGY, June 1996, p. 3094&#226;&#8364;&#8220;3105                                                                            Vol. 16, No. 6 <br /> 0270-7306/96/$04.0010 <br /> Copyright q 1996, American Society Microbiology <br />  <br />  <br />  <br />               TEL2, Essential Gene Required Telomere <span id='am-6' about='xsp:length' typeof='owl:Thing'>Length</span> <br />                     Regulation Telomere Position Effect in <br />                              Saccharomyces cerevisiae <br />                                          KURT W. RUNGE*          AND     VIRGINIA A. ZAKIAN&#226;&#8364;&#160; <br />                                  Division Basic Sciences, Fred Hutchinson Cancer Research Center, <br />                                                        Seattle, Washington 98104 <br />                       Received 7 December 1995/Returned modification 25 January 1996/Accepted 26 March 1996 <br />  <br />              The DNA-protein complexes ends linear eukaryotic chromosomes called telomeres. In <br />           Saccharomyces cerevisiae, telomeric DNA consists variable length short repeated sequence C1&#226;&#8364;&#8220;3A. The <br />           length yeast telomeres altered <span id='am-87' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-88' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span>mutation</span>, changing levels telomere binding proteins, or <br />           increasing C1&#226;&#8364;&#8220;3A DNA sequences. Cells bearing tel1-1 tel2-1 mutations, known pre- <br />           viously short telomeres, did respond perturbations caused telomere lengthening wild-type <br />           cells. The transcription genes placed near yeast telomeres reversibly repressed, phenomenon called the <br />           telomere position effect. The tel2-1 mutation reduced position effect did affect transcriptional re- <br />           pression silent mating type cassettes, HMRa HMLa. The TEL2 gene cloned, sequenced, and <br />           disrupted. Cells lacking TEL2 function died, cells arresting large cells small pro- <br />           trusions &#226;&#8364;&#732;&#226;&#8364;&#732;blebs.&#226;&#8364;&#8482;&#226;&#8364;&#8482; <br />  <br />  <br />    Telomeres, ends linear eukaryotic chromosomes,                  elongation chromosomal plasmid telomeres <br /> specialized chromosomal domains involved rep-                 introduces C1&#226;&#8364;&#8220;3A sequences, cause <br /> lication protection chromosomal termini (for reviews,                  lengthening. However, bootstrap effect does <br /> see references 48 63). Telomeres associate an-                   occur. Rather, telomeres reach final equilibrium length <br /> other nuclear envelope, organisms                   stable long excess C1&#226;&#8364;&#8220;3A sequences <br /> they visualized, packaged heterochro-                 cell (46). This new equilibrium does represent an <br /> matin (29, 63). Genes placed near telomeres transcription-                upper limit telomere length. Cells bearing certain mutations <br /> ally repressed (18, 28), similar reduced expression               DNA polymerase C1&#226;&#8364;&#8220;3A tracts long 2 kb <br /> occurs euchromatic genes placed near centromeric                     (10), cells bearing mutations eliminate carboxy <br /> heterochromatin (for reviews, references 18, 42, 48,                  terminus Saccharomyces cerevisiae telomere binding fac- <br /> 56).                                                                          tor RAP1 protein, Rap1p, C1&#226;&#8364;&#8220;3A tracts 4 kb <br />    In organisms, chromosome end capped                       (rap1t alleles [24]). These data suggest wild-type yeast cells <br /> simple, repeated DNA sequence; example, wild-type yeast                   regulate telomere length response excess C1&#226;&#8364;&#8220;3A <br /> cells ;350 bp C1&#226;&#8364;&#8220;3A DNA ends chro-                     sequences achieve new equilibrium state. <br /> mosome (for review, reference 63). These C1&#226;&#8364;&#8220;3A se-                         Additional data suggesting yeast cells regulate telo- <br /> quences appear necessary sufficient telomere                    mere length comes experiments involving new telomere <br /> function mitosis (38, 41, 49) meiosis (60). Although               formation. When linear plasmids formed yeast cells by <br /> lengths terminal C1&#226;&#8364;&#8220;3A tracts different telomeres vary,             using short long stretches C1&#226;&#8364;&#8220;3A, final length C1&#226;&#8364;&#8220;3A <br /> average Saccharomyces telomere length appears regu- <br />                                                                               sequence plasmid termini similar chro- <br /> lated length terminal C1&#226;&#8364;&#8220;3A tracts 350 6 <br />                                                                               mosomal telomeres host strain (37, 41, 46, 59, 61). This <br /> 75 bp. When C1&#226;&#8364;&#8220;3A sequences wild-type yeast <br />                                                                               result obtained input plasmid starts 250 <br /> cells increased ;30-fold using high-copy plasmids, chro- <br />                                                                               350 bp C1&#226;&#8364;&#8220;3A sequence (46, 61). A hypothesis consistent <br /> mosomal telomeres double length 500 700 bp (46). <br /> Extra C1&#226;&#8364;&#8220;3A DNA affects telomere length extra                     data yeast cells measure the <br /> tracts introduced telomeres internal tracts                 length terminal C1&#226;&#8364;&#8220;3A tract boundary the <br /> circular plasmid. When C1&#226;&#8364;&#8220;3A DNA introduced ends                    C1&#226;&#8364;&#8220;3A non-C1&#226;&#8364;&#8220;3A sequences physical end main- <br /> of linear plasmids, plasmid chromosomal telomeres                    tain length certain range. <br /> lengthen (46). The fact excess C1&#226;&#8364;&#8220;3A sequences cause                        In S. cerevisiae, average length telomeric C1&#226;&#8364;&#8220;3A <br /> telomere elongation plasmids chromosomes                          tract altered mutations different genes. <br /> would lead predict telomere length increase                These mutations fall classes. Some cause telomeres <span id='am-7' about='protege:TO' typeof='owl:Thing'>to</span> <br /> without bounds. This expectation based consideration                lengthen shorten continuously. For example, cdc17-1 <br />                                                                               mutation, alters catalytic subunit DNA polymer- <br />                                                                               ase a, causes telomeres lengthen continuously cells are <br />                                                                               grown semipermissive temperature (8, 10). The est1-1 <br />    * Corresponding author. Present address: Cleveland Clinic Founda- <br />                                                                               mutation causes telomeres shorten continuously cells <br /> tion Research Institute, Department Molecular Biology, NC20, <br /> 9500 Euclid Ave., Cleveland, OH 44195. Phone: (216) 445-9771. Fax:            die (31), does deletion TLC1 (55). Other mutations cause <br /> (216) 444-0512. Electronic mail address: rungek@ccsmtp.ccf.org.               cells maintain telomeres new average length. For ex- <br />    &#226;&#8364;&#160; Present address: Department Molecular Biology, Princeton              ample, recessive tel1-1 tel2-1 mutations cause telo- <br /> University, Princeton, NJ 08544-1014.                                         meres maintained shorter lengths (32), the <br />  <br />                                                                        3094 <br />  VOL. 16, 1996                                                                                                    TEL2 IS AN ESSENTIAL GENE                       3095 <br />  <br />  <br />                                                                    TABLE 1. Yeast strains <br />    Strain                                                            Genotype                                                                Reference source <br />  <br /> KR14-86             ade2-1 ade8-18 ura3-52 trp1D1 leu2-3,112 his4 cyh2          r <br />                                                                                                                                          46 <br /> KR36-6L             ade2(-1 -101) ade8-18 ura3-52 trp1D1 his3D leu2D-RC                                                             47 <br /> AHF                 ade1 ade2 URA1 ura3-a his7 lys2 tyr1 gal1                                                                          A. Lustig <br /> AJL210              ade1 ade2 URA1 ura3-b his7 lys2 tyr1 gal1 cdc27 tel1-1                                                             A. Lustig <br /> AJL1291d            ade1 ade2 URA1 ura3-b his7 lys2 tyr1 gal1 tel2-1                                                                   A. Lustig <br /> AHFA8L              ade1 ade2 ade8D-SX ura3-a leu2D-RC his7 lys2 tyr1 gal                                                              A. Lustig <br /> KR75-1D             ade1 ade2 ade8D-SX ura3-b leu2D-RC his7 lys2 tyr1 gal1 cdc27 tel1-1                                                This work <br /> KR75-2A             ade1 ade2 ade8-DSX ura3-a leu2D-RC his7 lys2 tyr1 gal1 cdc27 tel1-1                                                This work <br /> KR76-2B             ade1 ade2 leu2D-RC his7 lys2 tyr1 gal1 tel2-1                                                                      This work <br /> KR76-2D             ade1 ade2 ade8D-SX ura3-a ura1 his7 lys2 tyr1 gal1 tel2-1                                                          This work <br /> KR78-6C             ade1 ade2 ade8D-SX ura3-a leu2D-RC his7 lys2 tyr1 gal1 tel2-1                                                      This work <br /> KR78-11D            ade1 ade2 ade8D-SX ura3-a leu2D-RC his7 lys2 tyr1 gal1 TEL2                                                        This work <br /> KR78-2B             ade1 ade2 ade8D-SX URA3 leu2-DRC his7 lys2 tyr1 gal1 tel2-1                                                        This work <br /> KR90-1C             ade2(-1 -101) ade8-18 ura3-52 trp1D1 leu2-3,112 his3D cyh2r TEL2::LEU2                                          This work <br /> 1422-7D             ura3 leu2 his3 lys2 esp1-1                                                                                         J. McGrew B. Byers <br /> KR95t               a/a ade2-1/ade2-101 ade8-18/ade8-18 ura3-52/ura3-52 trp1D1/trp1D1 leu2-3,112/leu2-DRC                                This work <br />                       his3D/HIS3 his4/HIS4 cyh2r/cyh2r tel2D::TRP1/TEL2 <br /> KR95-1A             ade2(-1 -101) ade8-18 ura3-52 trp1D1 leu2(-3,112 -DRC) his3D cyh2r tel2D::TRP1                               This   work <br /> KR95-2A             ade2(-1 -101) ade8-18 ura3-52 trp1D1 leu2(-3,112 -DRC) his3D cyh2r TEL2                                      This   work <br /> KR95-2D             ade2(-1 -101) ade8-18 ura3-52 trp1D1 leu2(-3,112 -DRC) his3D cyh2r tel2D::TRP1                               This   work <br /> KR97-56             ade2(-1 -101) ade8-18 ura3-52 trp1D1 leu2(-3,112 -DRC) his3D cyh2r tel2D::TRP1 VIIL                          This   work <br />                       adh4::URA3-TEL <br />  <br />  <br />  <br /> introduction large excess C1&#226;&#8364;&#8220;3A sequences (46) over-                       plasmids pade8-DSX (<span id='am-5' about='protege:FROM' typeof='owl:Thing'>from</span> R. Wellinger) pLUL. The pade8-DSX plas- <br /> expression deletion derivative RAP1 protein con-                          mid contains in-frame deletion DNA internal SalI and <br />                                                                                       XhoI sites ADE8 gene (62) 0.9-kb insertion Salmonella hisG <br /> taining carboxy terminus, Rap1DBBp, causes telomeres                           DNA (1) BglII site downstream ADE8. This deletion greatly reduces <br /> be maintained twice normal length (14). The                      ADE8 gene function. The leu2-DRC allele (constructed pLUL) contains a <br /> group mutations includes genes play role                           deletion DNA internal ClaI EcoRV sites LEU2 <br /> telomere replication: CDC17 structural gene sub-                         eliminates LEU2 function. Strain AHFA8L crossed strains AJL210 <br /> <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> DNA polymerase (8), TLC1 encodes yeast telomer-                             AJL1291d tel1-1 strains KR75-1D KR75-2A, tel2-1 strains <br />                                                                                       KR76-2B, KR76-2D, KR78-6C, wild-type strain KR78-11D. <br /> ase RNA (55), EST1 protein play role telom-                             Strains AHFA8L, KR75-2A, KR78-6C converted URA3-TEL <br /> erase activity (see references 30 31). The second group                        strains transformation pADH4-UCAIV, replaces distal 17 kb <br /> mutations, tel1-1 tel2-1, cause perturbations result                      left arm chromosome VII URA3 gene short tract of <br /> telomeres reaching new steady-state length. Cells bearing                       C1&#226;&#8364;&#8220;3A elongates vivo form full-length yeast telomere (as refer- <br />                                                                                       ence 18). KR97-56 bearing YRpH3T2 similar KR95-1A (see below) also <br /> tel1-1 mutation terminal C1&#226;&#8364;&#8220;3A tracts 50 100 bp,                           URA3 telomere left arm chromosome VII. In these <br /> while tel2-1 cells tracts 100 150 bp (32). If TEL1                     strains, URA3 gene transcribed telomere. <br /> and TEL2 genes involved regulating number                                 The KR112 (tel2-1/tel2D) diploid constructed mating KR95-2D bearing <br /> telomere repeats, mutations genes set                           YEpT2 KR78-2B. This diploid cured YEpT2 grown 100 <br />                                                                                       generations express tel2-1 <span id='am-9' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-10' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span>phenotype</span>. The KR112 diploid used the <br /> new equilibrium telomere length. If TEL1 TEL2 reg-                            position effect test HMLa HMRa (see Table 3). <br /> ulators, telomere length tel1-1 tel2-1 cells                        Chromosome stability measured previously described assay for <br /> refractory perturbations alter telomere length,                     chromosome III (45). In work, strains KR78-11D (TEL2) KR78-6C <br /> example, excess C1&#226;&#8364;&#8220;3A overproduction Rap1DBBp.                                  (tel2-1) separately mated tel2D::TRP1 strain KR95-1A/YEpT2 to <br />                                                                                       form diploid strains KR109 (TEL2/tel2D) KR107 (tel2-1/tel2D). The <br />    In paper, telomeres tel1-1 tel2-1                     plasmid YEpT2 cured diploids, cyh2R/cyh2R versions each <br /> cells did lengthen additional C1&#226;&#8364;&#8220;3A sequences                         diploid patching diploid yeast extract-peptone-dextrose <br /> were introduced cell deletion derivative                      (YEPD) medium containing cycloheximide picking single colony. The <br /> RAP1 containing C terminus, Rap1DBBp, expressed.                              resulting cycloheximide-resistant (CyhR) diploids transformed pH4:: <br />                                                                                       CUC-2 introduce CYH2s gene copy chromosome III within <br /> The tel2-1 mutation reduced telomere position effect                             HIS4 (as described reference 45). Two separate transformants diploid <br /> (TPE) repression URA3 transcription silencing                           CYH2s gene integrated chromosome MATa were <br /> the silent mating type cassettes. Both tel1-1 tel2-1 cells                   chosen, grown YEpT2 total 100 generations, used to <br /> short telomeres grew slowly 378C rich medium. The                               measure rate chromosome III loss fluctuation analysis 20 colony <br />                                                                                       tests (45). The recombination rates diploids 1.0 3 1024 both <br /> TEL2 gene cloned suppression slow-growth                               strains, loss rates given Results. <br /> phenotype, sequenced, shown essential.                                         All yeast media previously described (66). Yeast transformation was <br />                                                                                       performed lithium acetate method single-stranded salmon sperm <br />                        MATERIALS AND METHODS                                          DNA carrier (52). <br />                                                                                          Introduction excess C1&#226;&#8364;&#8220;3A sequences overexpression Rap1p C <br />    Yeast strains methods. The strains used study summarized        terminus. The telomeric C1&#226;&#8364;&#8220;3A sequences cells increased by <br /> Table 1. The wild-type strains KR14-86 KR36-6L described                using circular plasmid YEpFAT7.5 (46). This plasmid originally cloned <br /> previously (45, 46). The A364a background strains AHF, AJL210, AJL1291d           yeast cells contained C1&#226;&#8364;&#8220;3A tract ;450 bp. Upon propagation in <br /> were A. Lustig. The ura1 mutations AHF AJL210 reverted            Escherichia coli, length C1&#226;&#8364;&#8220;3A tract reduced ;270 bp (reference <br /> selecting uracil prototrophy, ura3 mutations independently           16a data shown). The plasmid shorter C1&#226;&#8364;&#8220;3A tract used in <br /> selected 5-fluoro-orotic acid (FOA) medium (31a). The ura3-a ura3-b            experiments study. For experiments reported here, YEpFAT7 or <br /> alleles distinguishable Southern blotting digesting PstI       YEpFAT7.5 transformed yeast strains described Results and <br /> probing 1.1-kb URA3 HindIII fragment (data shown). AJL210            Ura1 transformants selected. Individual transformants selected and <br /> AJL1291d progeny fourth backcross original tel1-1 tel2-1        streaked single colonies medium lacking uracil (YC-ura) leucine <br /> isolates (32) A364a wild-type strain.                                           (YC-leu). Transformants serially streaked times YC-ura or <br />    The wild-type strain AHF converted AHFA8L transformation            YC-leu strain grown 100 consecutive generations of <br />  3096       RUNGE AND ZAKIAN                                                                                                                        MOL. CELL. BIOL. <br />  <br />  <br /> these media. Cells treated manner used start 5-ml YC-ura            types yeast transformants streaked single colonies YC-ura me- <br /> YC-leu liquid cultures, appropriate, prepare yeast DNA analysis           dium 308C. A single colony strain picked restreaked on <br /> Southern hybridization (as reference 46).                                           YC-ura medium. This process repeated times strain so <br />    Overexpression RAP1 protein C terminus, Rap1DBBp, accom-                 tel2-1 transformants grown 100 generations presence the <br /> plished using YEpFAT4-DBB (14). This plasmid encodes truncated RAP1               YEpT2-XhoI linker insertion plasmid. The strains tested telomere <br /> gene codons 1 19 fused codons 499 827 deletion          length temperature sensitivity. Two plasmids failed complement tel2-1 <br /> internal BamHI fragment (14). Cells transformed YEpFAT4 YEp               mutation, insertion plasmid internal deletion plasmid (which shared <br /> FAT4-DBB, Ura1 transformants selected. Individual transformants               end point noncomplementing insertion). The 3.2-kb NsiI fragment <br /> were grown 40 consecutive generations YC-ura YC-leu allow            (see below) subcloned PstI site pVZ-1 (21), deletions <br /> maximal telomere lengthening (14). Cells treated way used start        restriction enzyme digestion. The DNA sequence determined <br /> 5-ml YC-ura YC-leu liquid cultures, appropriate, used prepare       dideoxy sequencing method (50) using Sequenase 2.0 kit (Amersham) <br /> DNA Southern hybridization described above.                                     single-stranded double-stranded templates automated <br />    Plasmid copy numbers determined preparing genomic DNA                  sequencing dye terminators. The locations following restriction sites <br /> yeast cells, digesting XhoI, SalI, NsiI, analyzing products     based numbering Fig. 4: ScaI, nucleotide 551; HindIII, 1665; PstI, <br /> Southern hybridization. The LEU2 EcoRV-AccI fragment used probe.              1869; XbaI, 2418. <br /> All sequences fragment present plasmid             The 3.2-kb NsiI fragment, lacked XhoI linker insertion, cloned <br /> chromosome. The restriction enzyme digest 20-fold dilution            PstI site pVZ-1 form plasmid pT2Na, used to <br /> loaded adjacent gel lanes, intensities plasmid chromosomal       construct yeast vectors. YIpL2T2 pT2Na plus 3.0-kb BglII fragment <br /> bands compared. The fraction cells plasmid determined              containing LEU2 gene BamHI site pT2Na. The XhoI site this <br /> plating 200 CFU (determined counting hemocytometer) strain           plasmid upstream LEU2 gene, site used target the <br /> onto complete medium medium lacking leucine. The plasmid copy number               plasmid LEU2 locus homologous recombination (43) (see Results). <br /> in wild-type, tel1-1, tel2-1 cells bearing YEpFAT4, YEpFAT4-RAP1DBB,               YRpL2T2 yeast replicating vector pT2Na plus 2.6-kb SalI-BglII <br /> and YEpFAT7.5 grown medium lacking leucine approximately 25 40               fragment bearing LEU2 gene XbaI-NsiI fragment yeast 2mm <br /> per cell. This copy number lower expected selection leu2-d         circle, contains origin replication 2mm REP3 plasmid <br /> allele, reason unclear. The copy number plasmid     stabilizing region (6). YRpH3T2 similar plasmid contains HIS3 gene <br /> tel1-1 tel2-1 cells (25 45/cell) high higher seen    instead LEU2 gene. <br /> wild-type cell.                                                                           Two types tel2 gene disruption plasmids constructed: transplacement <br />    Re-replica plate assay slow growth 37&amp;C. To score weak temperature       vector (44), pT2::TRP1, integration vector (43), YIpt2-DL2. pT2::TRP1 <br /> sensitivity tel2-1 strains, colonies patches cells taken     contains 0.6-kb NsiI-ScaI fragment upstream TEL2 open reading <br /> consecutive replica platings. First, patches cells grown rich        frame (ORF), XmnI-BglII TRP1 fragment YCp19 (57), XbaI- <br /> medium selects plasmid. A wild-type strain tel2-1 strain       NsiI fragment downstream TEL2 ORF BamHI site 59 XbaI <br /> were usually included controls. Second, cells replica plated YEPD         site pVZ-1 polylinker. These fragments cloned pVZ-1 PstI <br /> plate standard methods (34). Third, new replica plate immediately           site order (PstI NsiI)-(ScaI XmnI)-TRP1-(BglII BamHI)-XbaI-(NsiI PstI), <br /> used new master plate. The new master plate replica plated          sites parentheses fusions destroy restriction sites. The <br /> YEPD plate preincubated 378C 2 h second         integrating vector YIpt2-DL2 contains 400-bp fragment TEL2 XhoI <br /> YEPD plate (usually room temperature 48C). The YEPD plates          linker insertion PstI site, TEL2 ORF, cloned inte- <br /> the re-replica plates. The re-replica plate, prewarmed 378C,          grating vector pRS305 (54) cut XhoI PstI. The only <br /> incubated 378C 2 days, plate incubated 308C       HindIII site plasmid TEL2 DNA, HindIII site ;200 bp <br /> for length time. No growth differences observed              XhoI site PstI site. <br /> standard replica plate assays.                                                            Sequence similarity searches performed BLAST program (2) on <br />    The tel1-1 cells work bear Ts2 cdc27 mutation. Three cdc27         National Center Biotechnology Information (NCBI) servers. The TEL2 <br /> mutation revertants selected tested re-replica plate test           DNA sequence compared nonredundant DNA protein se- <br /> described Results.                                                                  quences databases. The sequences predicted ORFs also <br />    Cloning TEL2 gene. KR78-6C cells (bearing tel2-1 mutation)             compared database sequences. Separate comparisons also <br /> transformed (52) yeast genomic library cloned YEp24 (9),               expression sequence tag (EST) database. No significant similar- <br /> transformants selected plating YC-ura plates. Each plate con-           ities database sequences sequences encoded <br /> tained approximately 1,300 transformants; construct pool transformants,        3.2-kb NsiI fragments, exception overlap ESP1 <br /> 2 ml sterile H2O placed plate, cells suspended           sequence, 11 March 1996. <br /> spreader, suspension transferred sterile test tube. The plate        Construction tel2 disruption strains. The tel2D::TRP1 allele by <br /> then washed additional 2 ml sterile H2O, subsequently             disrupting copy gene diploid strain, sporulating strain, and <br /> added suspension. The 4-ml suspensions centrifuged pellet       isolating haploid strain mutant allele. The diploid KR95 formed <br /> cells. The H2O removed, cells resuspended 5 ml YC-ura           mating KR14-86 KR36-6L (45), selecting His1 diploids, selecting <br /> liquid medium grown overnight 308C. Each culture vortexed vigor-            CyhR clones patching individual diploid colony YEPD containing <br /> ously 15 s, 0.1-ml aliquot cells suspension diluted    cycloheximide (10 mg/ml). KR95 transformed pT2::TRP1 been <br /> 5 ml YC-ura liquid medium. The cultures grown 308C 2 days        digested SphI BamHI. Trp1 colonies selected analyzed by <br /> saturation (;108 cells ml), approximately 10 generations. Each culture         Southern hybridization strain wild-type copy TEL2 and <br /> diluted (0.05 ml) 5 ml fresh YC-ura medium grown 24 h 308C.         disrupted copy, KR95t. KR95t transformed YEpT2, induced to <br /> This procedure repeated times. After seventh consecutive culture,      sporulate, dissected produce KR95-1A KR95-2A (Table 1), both <br /> cells diluted YEPD liquid medium prewarmed 378C grown              bearing YEpT2. Both strains derived 4-spored tetrads MAT, <br /> 378C 3 days. Only seven cultures grew saturation. The number         trp1, his4, his3 segregated 2:2. <br /> of CFU (single cells budded cells) milliliter dense culture            The tel2-DL2 disruption constructed KR95-2A bearing YEpT2. YIpt2- <br /> determined counting hemocytometer, 200 CFU culture             DL2 linearized digestion HindIII, cuts TEL2 DNA <br /> was plated 10 YEPD plates prewarmed 378C. The            sequences, used transform KR95-2A cells, selecting Leu1 transfor- <br /> plates incubated 378C 2 days. Only 2 3 large colonies grew      mants. Transformants screened ability grow YEpT2 by <br /> plate, 14 analyzed telomere length temperature sensitivity.      replica plating FOA medium. Four 100 transformants grow on <br /> All 14 strains telomeres wild-type length Ts1. The YEp24 library       FOA plates, strains transformed YRpH3T2 give <br /> plasmid isolated strains. All strains contained      His1 Leu1 strains. Three new strains grow FOA plates or <br /> plasmid determined restriction enzyme mapping. This plasmid desig-           FOA medium lacking leucine, indicating presence YRpH3T2 <br /> nated YEpT2. These results consistent single YEpT2 transformant            resulted loss YEpT2 plasmid transformants. Southern <br /> being amplified pool positive transformant obtained         analysis strains indicated possessed tel2-DL2 disrup- <br /> the initial 7,800 transformants.                                                       tion-integration TEL2 locus. These strains able grow without <br />    Recombinant DNA manipulations TEL2 gene. YEpT2 contained                   presence normal TEL2 gene plasmid cell. <br /> 7.8-kb genomic yeast DNA insert XhoI sites. To localize TEL2              Cloning tel2-1 gene. The NsiI fragment containing tel2-1 gene was <br /> gene insert, YEpT2 partially digested RsaI fragment       isolated PCR primers end sequenced NsiI fragment <br /> corresponding size linearized plasmid isolated gel electro-       (TEL2PCR5a CGGATCCAGTACTTGTGGACAGGTTGCC TEL2 <br /> phoresis. The linearized plasmid ligated 8-bp XhoI linker, cut     PCR3a CGGATCCATTCTCGATCCTTGCC) genomic DNA the <br /> XhoI, religated conditions led circularization plasmid;      tel2-1 strain KR78-6C template, using 35 cycles 1 min 948C, 1 min at <br /> the circularized plasmid transformed E. coli MC1066 (r2 m1               558C, 3 min 728C. The 3.2-kb fragment digested BamHI and <br /> pyr::Tn5 trpC9830 leuB600 lacDX74 strA galU galK). Plasmids isolated         cloned BamHI site pVZ-1, single clone sequenced both <br /> Ura1 E. coli transformants, XhoI linker insertions mapped. Five           strands nucleotides 176 2820 TEL2 sequence (see Results). The <br /> insertions different places insert internal deletion isolated   difference detected nucleotide 1006, G-to-A transition <br /> and separately transformed KR78-6C (tel2-1) cells. Three isolates      causing serine-to-asparagine mutation predicted TEL2 protein. A region <br />  VOL. 16, 1996                                                                                                    TEL2 IS AN ESSENTIAL GENE                         3097 <br />  <br />  <br /> from nucleotide 870 1120 TEL2 sequence (using primers T2873S           mating type cassettes (12) telomeres (data shown) (16b, 31a). The <br /> CGCCGGATCCGCGATGGACTGCAAGAG T21120AS CGCCGAATT                                   plasmid pKR60 pCTC23 (12) BglII-XbaI HIS3-2mm ori portion <br /> CCCCTAAAAACCCAGGTGG) amplified PCR separate reactions                      removed replaced BamHI-XbaI URA3-2mm ori fragment from <br /> with genomic DNA tel2-1 strains (AJL1291d, KR76-2B, KR76-2D,           YEp24. This plasmid encodes fusion GAL4 protein (amino acids 1 to <br /> KR78-6C), wild-type strains (AHF AHFA8L), tel1-1 strains           147) Sir4p (amino acids 1280 1394) expressed ADC1 upstream <br /> (KR75-1D, KR75-2A, AJL210) templates. All amplification reactions         activation sequence (UAS) promoter (12). <br /> sequencing consisted denaturation 5 min 948C followed 35 cycles         The KR112 diploids bearing plasmids grown overnight 308C 5 ml of <br /> 1 min 948C, 1 min 558C, 1 min 728C cycle 6 min 728C.      YC-ura liquid medium, diluted 1:4 YEPD liquid medium, grown overnight <br /> The PCR products reactions sequenced directly, using      308C, induced sporulate. Briefly, aliquot 0.1 ml cells washed <br /> T2931S primer (GAACCGGGATGTTTGAC), automated sequencing                    twice sterile water resuspended 2 ml 0.5% potassium acetate <br /> CCF automated sequencing core. Two separate, independent amplifications          (KOAc) 100 mg adenine, uracil, tryptophan, leucine, histidine <br /> sequencing reactions performed DNA AJL1291d AHFA8L                 ml. Cell suspensions rotated 5 days 218C ensure <br /> with results. All tel2-1 amplified products contained G-to-A       complete sporulation. Aliquots cells viewed phase-contrast mi- <br /> transition nucleotide 1006 original tel2-1 clone, wild-    croscope 4003 magnification, sporulated cells (tetrads, triads, dyads) <br /> type tel1-1 products contained wild-type TEL2 sequence (G nucleotide      mitotic cells counted. The range sporulation 5 30% for <br /> 1006).                                                                               unmodified KR108 diploids ,0.1% matD/MAT(a a) diploids (data <br />    Physical genetic mapping TEL2 gene. Filters containing ordered     shown). <br /> phage clones yeast physical map provided Linda Riles                 Nucleotide sequence accession number. The <span id='am-3' about='Thesaurus:GenBank_Accession_Number' typeof='owl:Thing'>GenBank accession number</span> for <br /> Maynard Olson (42a). A 423-bp XhoI-PstI fragment TEL2 coding                TEL2 sequence U38538. <br /> region labelled random hexamers (47) used probe according to <br /> the directions provided L. Riles. Three overlapping phage clones de- <br /> tected, placing TEL2 right arm chromosome VII, approximately 35 kb <br />                                                                                                                        RESULTS <br /> proximal SPT6. Comparison restriction map TEL2 high-              TEL1 TEL2 functions required telomere length <br /> resolution EcoRI-HindIII double digest map provided L. Riles indicated that <br /> TEL2 oriented chromosome transcription goes the <br />                                                                                      control. Whereas wild-type cells telomeres 350 6 75 <br /> centromere telomere.                                                      bp, tel1-1 tel2-1 mutations cause cells maintain <br />    The genetic map S. cerevisiae (36) indicates TEL2 ;10           telomeres lengths of, respectively, 50 100 100 to <br /> centimorgans (cM) ESP1. The TEL2 gene marked inserting               150 bp C1&#226;&#8364;&#8220;3A (32). If TEL1 TEL2 genes in- <br /> 2.2-kb LEU2 SalI-XhoI fragment TEL2 XhoI linker insertion that <br /> the XhoI site LEU2 closest TEL2 C terminus. This TEL2::LEU2 <br />                                                                                      volved regulation telomere length, muta- <br /> insertion provide TEL2 function terms cell growth telomere        tions cause short telomeres altering regulation. <br /> length. A strain bearing LEU2 insertion, KR90-1C, mated strain         For example, tel1-1 tel2-1 mutations cause cells to <br /> bearing Ts2 lethal mutation ESP1, 1422-7D (from J. McGrew B. Byers).        ignore signals normally result telomere elonga- <br /> The resulting diploid sporulated, 182 randomly selected cycloheximide- <br /> resistant haploid spores scored mating type, temperature sensitivity, and <br />                                                                                      tion miscount number telomeric repeats chro- <br /> leucine auxotrophy. Of classes possible recombinant progeny, 1 Ts1        mosome ends shorter length normal main- <br /> Leu2 10 Ts2 Leu1 spores observed. The disparity             tained. Under hypothesis, cellular perturbations that <br /> classes large to random sampling error (P , 0.01). Assuming        cause telomere lengthening wild-type cells cause <br /> one recombinant class spores lost unknown differences between <br /> the parental-strain backgrounds, distance (in centimorgans) these <br />                                                                                      elongation tel1-1 tel2-1 cells. <br /> markers approximately equal yeast genetic map (36). The centimorgan-          Two types perturbations cause cells increase their <br /> per-kilobase measurements Results obtained using 4.5 6.1 kb     equilibrium telomere length introduction long <br /> distance LEU2 insertion esp1-1 mutation.                         stretches telomeric C1&#226;&#8364;&#8220;3A DNA (46) overproduc- <br />    TPE assay. Assays performed haploid strains bearing URA3-TEL <br /> (AHFA8L, KR75-2A, KR78-6C) diploid strains bearing copy of <br />                                                                                      tion Rap1DBBp, deletion derivative known yeast <br /> URA3-TEL formed mating KR78-6C KR78-11D KR97-56               telomere binding protein Rap1p (14). In vivo experi- <br /> or KR95-2A/tel2-DL2. In case diploid strains, Southern hybridization      ments, telomere length presumably altered com- <br /> was used screen loss YRpH3T2. Plasmid-free cells grown 100        petition chromosomal telomeres factors involved in <br /> generations allow expression tel2-1 mutant phenotype prior TPE <br /> assays. Assays performed streaking cells single colonies YEPD <br />                                                                                      telomere length regulation. <br /> medium 308C (diploid strains) 238C (haploid strains). After single         An excess C1&#226;&#8364;&#8220;3A sequences construct produc- <br /> colonies formed, colony strain picked streaked         ing large amounts Rap1DBBp introduced cells on <br /> single colonies synthetic complete (YC) medium, incubation 30      plasmids copy number controlled nutritional <br /> or 238C. For strain, single colonies individually picked agar <br /> plugs using Pasteur pipette, colony placed 1.0 ml sterile <br />                                                                                      selection, YEpFAT plasmids. YEpFAT plasmids carry the <br /> H2O. Each colony suspension vortexed highest setting 15 s.            promoter defective leu2-d allele selectable marker with <br /> Tenfold dilutions suspension plated (one plate             normal promoter, URA3 TRP1. Plasmids be <br /> dilution) determine titers cells grow YC (viable cells),     maintained multiple copies selecting URA3 or <br /> YC-ura (Ura1 cells), FOA (cells URA3 expression repressed by <br /> TPE). The position effect colony percentage cells grow <br />                                                                                      TRP1 gene, maintained higher levels by <br /> on FOA medium. In strains used here, .90% viable cells           selecting leu2-d allele (see, e.g., reference 46 ref- <br /> Ura1. The level position effect strain determined placing     erences therein). YEpFAT plasmids bearing insert (YEp <br /> values colonies order highest lowest choosing    FAT7 YEpFAT4), long stretches C1&#226;&#8364;&#8220;3A (YEpFAT7.5), <br /> the median value. In cases, median mode close the <br /> average. <br />                                                                                      gene expressing RAP1DBBp (YEpFAT4-DBB) were <br />    HMLa HMRa position effect assays. Silencing position effect HMLa        introduced congenic wild-type tel1-1 tel2-1 cells. All <br /> or HMRa tested monitoring sporulation MATD/MAT(a a) diploids.           transformants grown conditions previ- <br />    The TEL2/tel2D MATD/MAT(a a) tel2-1/tel2D MATD/MAT(a a)                 ously shown allow maximum telomere elon- <br /> diploids constructed using KR108 (TEL2/tel2D) KR112 (tel2-1/ <br /> tel2D). The KR112 diploid cured YEpT2 grown 100 generations. <br />                                                                                      gation presence excess C1&#226;&#8364;&#8220;3A sequences extra <br /> Each diploid transformed pKAN591 digested             RAP1 protein C termini (14, 46). Telomere length deter- <br /> completion PvuII HindIII, selecting Leu1 transformants. The plas-       mined Southern hybridization. <br /> mid pKAN591 MATax8 pBR322 (58) containing 2.2-kb LEU2 XhoI-SalI               The results analysis cells bearing plasmids encod- <br /> fragment inserted XhoI linker (a gift L. Breeden). This plasmid <br /> replaces MATa MATa LEU2 gene. Transformants tested for <br />                                                                                      ing excess telomere DNA sequences RAP1DBBp <br /> mating type determine copy MAT eliminated,              showed telomeres wild-type cells elongated while <br /> structures matD::LEU2, MATa MATa, HMLa loci confirmed             cells bearing tel1-1 tel2-1 mutations did not <br /> by Southern hybridization. For TEL2/tel2D KR108 strains, MATa/MATa           (Fig. 1). The number plasmids cell approximately <br /> diploid, matD/MATa diploid, MATa/matD diploid tested. For <br /> the tel2-1/tel2D KR112 strains, MATa/MATa diploid, matD/MATa dip- <br />                                                                                      wild-type, tel1-1, tel2-1 cells (see Materials and <br /> loid, MATa/matD diploid transformed YEp24 pKR60.           Methods). These data indicate TEL1 TEL2 functions <br /> The presence pKR60 cells derepress position effect silent       required telomere elongation response per- <br />  3098       RUNGE AND ZAKIAN                                                                                                                      MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />    FIG. 1. Telomere elongation perturbed tel1-1 tel2-1 cells. Wild-type (AHFA8L), tel1-1 (KR75-1D), tel2-1 (KR78-6C) cells analyzed telomere <br /> length presence excess C1&#226;&#8364;&#8220;3A sequences (A) plasmid expressing deletion derivative Rap1p (B). The excess C1&#226;&#8364;&#8220;3A sequences introduced by <br /> transforming cells circular plasmid YEpFAT7.5, contains 270 bp C1&#226;&#8364;&#8220;3A sequences, growing cells 100 generations described Materials <br /> and Methods (46). Rap1DBBp overproduced transforming cells YEpFAT4 bearing truncated RAP1 gene, RAP1DBB (see Materials Methods) (14), <br /> and growing cells 50 generations described (Materials Methods). In panels, genomic DNAs digested XhoI analyzed Southern <br /> hybridization C1&#226;&#8364;&#8220;3A sequences probe. The lowest band corresponds terminal restriction fragment telomeres contain Y9 element, arrows <br /> indicate telomeres different panels did hybridize Y9 probe likely X telomeres Y9 elements (see reference 11). <br />  <br />  <br />  <br />  <br /> turbations. These data consistent hypothesis                       5.3 5.4 h, respectively, wild-type cells re- <br /> TEL1 TEL2 involved telomere length regulation.                            quired 10.3 h double. The doubling times cell <br />   A new phenotype associated presence large                              types bearing vector ;5.4 h. Thus, large <br /> amounts Rap1DBBp discovered course                              amounts Rap1DBBp caused cold sensitivity wild-type cells <br /> these experiments. When wild-type, tel1-1, tel2-1 cells                          tel1-1 tel2-1 cells. <br /> bearing YEpFAT4-DBB (Fig. 1B) grown YC-leu me-                                 TEL2 function essential viability S. cerevisiae. The <br /> dium 218C, numbers tel1-1 tel2-1 cells doubled                         original description tel1-1 tel2-1 alleles suggested <br />  VOL. 16, 1996                                                                                             TEL2 IS AN ESSENTIAL GENE                    3099 <br />  <br />  <br />                                                                                  short-telomere phenotype (assayed Fig. 2; Materials <br />                                                                                  Methods). Cells bearing tel2-1 mutation trans- <br />                                                                                  formed yeast genomic library vector YEp24 (9). <br />                                                                                  To ensure TEL2 gene expressed of <br />                                                                                  time sufficient reverse tel2-1 phenotype, ;7,800 <br />                                                                                  transformants divided pools grown about <br />                                                                                  70 generations selection library plasmids. Each <br />                                                                                  pool challenged rapid growth 378C rich <br />                                                                                  medium. One pool yielded fast-growing colonies that <br />    FIG. 2. tel2-1 cells impaired growth 378C. Wild-type (AHFA8L) <br />                                                                                  normal-length telomeres. The library plasmid was <br /> and tel2-1 (KR78-6C) cells grown YEPD plate tested            isolated colonies pool. All plasmids <br /> re-replica plate assay (see Materials Methods). The re-replica plates   identical restriction enzyme mapping (data shown). <br /> grown 30 378C time periods indicated.                          The complementing library plasmid designated YEpT2. <br />                                                                                  The suppression tel2-1 phenotypes YEpT2 depen- <br />                                                                                  dent: loss plasmid gave rise strains short <br /> that cells required 150 generations growth telo-                    telomeres grew poorly 378C. These results suggested <br /> meres attain shortest lengths, phenomenon known                    complementing library plasmid contained TEL2 <br /> phenotypic lag. There indications mutations                   gene multicopy suppressor gene. When YEpT2 was <br /> had effect cell growth (32). During course                    retransformed tel2-1 strain, Ts2 short-telomere <br /> study, cells bearing tel2-1 mutation                    phenotypes reverted ;50 generations growth the <br /> growth disadvantage, weak Ts2 phenotype, 378C rich                       presence plasmid. <br /> medium (YEPD; Materials Methods) (Fig. 2). The                              YEpT2 contained 7.8-kb insert yeast genomic DNA. <br /> weak Ts2 phenotype cosegregated tel2-1 mutation                      The region plasmid responsible complementing the <br /> four tetrads. This slow-growth phenotype observed                    tel2-1 mutation localized XhoI linker insertion mutagen- <br /> tel2-1 cells short telomeres. When tel2-1 spores                    esis (see Materials Methods). Five XhoI linker insertions <br /> first isolated tetrads, rise cells normal                 yeast genomic DNA isolated. One inser- <br /> telomere lengths gradually shorten cells                  tions failed complement slow-growth short-telomere <br /> repeated divisions (32). The slow growth 378C detect-                     phenotypes tel2-1 mutation (Fig. 3 4). This insertion <br /> able tel2-1 cells grown 80 100 gener-                  near center 3.2-kb NsiI fragment. The 3.2-kb NsiI <br /> ations, showing shorter telomeres associated                   fragment complemented tel2-1 mutation copy per <br /> slow-growth phenotype. The tel1-1 mutation linked Ts2                    cell integrated LEU2 locus (see Materials and <br /> allele cdc27. Three independent Ts1 revertants ob-                       Methods), suggesting NsiI fragment contained the <br /> tained tested weak Ts2 phenotype (Fig. 2). These                     entire TEL2 gene. <br /> three tel1-1 strains showed weak Ts2 phenotype                          The sequence 3.2-kb NsiI fragment revealed the <br /> tel2-1 cells (see Materials Methods).                                        XhoI linker insertion disrupted 688-amino-acid ORF (Fig. 4). <br />    The TEL2 gene cloned complementation weak                       The XhoI linker creates frameshift 20% the <br /> Ts2 phenotype scoring complementation                        ORF, majority protein translated. <br />  <br />  <br />  <br />  <br />    FIG. 3. TEL2 gene disruptions. Diagrams wild-type TEL2 gene tel2D::TRP1 tel2-DL2 gene disruptions shown. The tel2D::TRP1 deletion <br /> removes 89 amino acids TEL2 ORF upstream sequences. The tel2-DL2 insertion does remove genomic sequences creates <br /> a 39 deletion followed 5.5-kb insertion 59 deletion TEL2 gene (see Materials Methods). <br />  3100      RUNGE AND ZAKIAN                                                                                                                  MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />    FIG. 4. Sequence TEL2 locus. The DNA sequence TEL2 locus amino acid sequence TEL2 ORF shown. The short ORF overlapping <br /> the TEL2 ATG short stretch amino acid sequence similarity TEL2 ARD1 proteins underlined. The site XhoI linker insertion in <br /> the TEL2 ORF tel2-1 mutation (G A nucleotide 1006, causing S129N substitution) indicated. The stars indicate termination codons. The tel2-1 <br /> mutation tel2-1 DNA samples tested (see text). The Tel2 protein predicted molecular mass 79 kDa pI 6.2. <br />  <br />  <br />  <br />    Two disruptions 688-amino-acid ORF con-                            tion recessive tel2-1 mutation. Cells bearing TRP1 <br /> structed. The disruption, tel2D::TRP1, replaced 1.9 kb                   disruption containing plasmid bearing 3.2-kb NsiI <br /> DNA TRP1 gene (Fig. 3). This deletion removes 599                        fragment mated congenic TEL2 tel2-1 cells (see <br /> amino acids N terminus ORF (see below) (Fig.                      Materials Methods). Diploids lost plasmid <br /> 4). Haploid cells bearing disruption unable grow                     isolated, strains grown 100 generations <br /> unless ORF provided trans plasmid. The                      overcome phenotypic lag. A tel2-1/tel2-1 diploid also <br /> disruption, tel2-DL2, placed 5.5 kb plasmid DNA, including                     constructed. All diploids tested telomere length <br /> the LEU2 gene, middle ORF deleting                      growth 378C. We noted telomeres wild-type <br /> chromosomal DNA (Fig. 3). This disruption lethal                         mutant diploid strains slightly longer telomeres <br /> (see Materials Methods). These results ORF                      similar haploid strains (Fig. 5). However, length differ- <br /> that complemented tel2-1 mutation single copy                     ence tel2-1 mutations maintained hap- <br /> essential viability S. cerevisiae.                                         loids diploids (Fig. 5 data shown). Compared with <br />    These disruption mutations tested complementa-                        TEL2/tel2-1 diploid, tel2-1/tel2D::TRP1 diploid had <br />  VOL. 16, 1996                                                                                                   TEL2 IS AN ESSENTIAL GENE                     3101 <br />  <br />  <br />                                                                                       previously measured loss rates chromosome III 0.39 3 <br />                                                                                       1024 similar wild-type diploid (45), suggesting having <br />                                                                                       copies TEL2 gene cell affect <br />                                                                                       chromosome loss. Thus, TEL2/tel2D tel2-1/tel2D <br />                                                                                       strains compared, tel2-1 mutation had, best, small <br />                                                                                       effect chromosome stability. A similar small effect chro- <br />                                                                                       mosome stability recently demonstrated tel1D mu- <br />                                                                                       tation (19). <br />                                                                                          As described above, disruption TEL2 gene lethal in <br />                                                                                       haploid cell. To determine cells lacking TEL2 gene <br />                                                                                       product die interesting way, diploid strain bearing the <br />                                                                                       tel2D::TRP1 disruption normal TEL2 gene sporu- <br />                                                                                       lated resulting tetrads dissected. As expected, <br />                                                                                       tetrads contained viable spores viable <br />                                                                                       spores Trp2, confirming TEL2 gene essen- <br />                                                                                       tial. Microscopic examination tel2D::TRP1 spores that <br />                                                                                       failed produce colonies revealed spores germi- <br />                                                                                       nated gave rise microcolonies cells. Many <br />                                                                                       cells large misshapen. <br />    FIG. 5. The tel2D mutation fails complement tel2-1 telomere length <br />                                                                                          To generate larger population cells lacking TEL2 <br /> and temperature sensitivity phenotypes. Genomic Southern blots different       gene, YRpH3T2, unstable plasmid containing TEL2 <br /> diploids C1&#226;&#8364;&#8220;3A probe shown. Two different isolates diploid        gene, introduced tel2D::TRP1 strain. YRpH3T2 is <br /> that grown 100 generations TEL2 plasmid analyzed. The         expected lost high rate, generating plasmid- <br /> genotype strain indicated lane described the <br /> text. The lanes left haploid strains. The temperature sensitivity <br />                                                                                       free cells (39). The plasmid-free cells continue divide <br /> of strain re-replica plate assay indicated        run TEL2 gene product. Thus, proportion <br /> Southern blot lane. 1, wild-type growth 378C re-replica plate assay (Fig.   cells manifesting phenotype complete lack Tel2 <br /> 2); 2, slow tel2-1 growth 378C. Arrows indicate X telomeres.                       protein expected small. Examination cells such <br />                                                                                       culture phase-contrast microscopy revealed fraction <br />                                                                                       cells novel phenotype. There large spherical <br /> short telomeres showed weak Ts2 growth, just tel2-                         cells small &#226;&#8364;&#732;&#226;&#8364;&#732;blebs&#226;&#8364;&#8482;&#226;&#8364;&#8482; &#226;&#8364;&#732;&#226;&#8364;&#732;knobs&#226;&#8364;&#8482;&#226;&#8364;&#8482; (Fig. 6). These <br /> 1/tel2-1 diploid did (Fig. 5). In contrast, TEL2/tel2D::TRP1                      blebs smaller yeast buds typical mutants that <br /> diploid phenotypically wild type (data shown). Identi-                        arrest budded cells. The presence multiple blebs an <br /> cal results obtained tel2-1/tel2-DL2 diploid (data                        unprecedented phenotype. When tel2D::TRP1 cells were <br /> not shown). As shown below, tel2D::TRP1 allele failed                        stained DAPI (49,6-diamidino-2-phenylindole), single <br /> to complement TPE phenotype tel2-1 mutation                                nucleus observed frequently associated one <br /> (Table 2).                                                                            blebs (data shown). Since cells small blebs were <br />    To confirm ORF Fig. 4 encodes TEL2 gene,                           detected parallel cultures congenic TEL2 tel2-1 <br /> the 3.2-kb NsiI fragment isolated PCR tel2-                           cells, probably cells run TEL2 <br /> 1 strain KR78-6C. The entire ORF sequenced                                gene product. Similar multiply budded cells seen in <br /> strands, difference TEL2 se-                          cells overproducing Las1p, essential nuclear protein in- <br /> quence G-to-A transition nucleotide 1006, encoding                         volved cell morphogenesis (15), surface projec- <br /> a serine-to-asparagine mutation amino acid 129 (S129N).                            tions appear somewhat larger blebs Fig. 6. <br /> PCR primers flanking mutation used amplify                             Structure location TEL2 gene. The TEL2 ORF <br /> region DNA tel2-1 strains (AJL1291d, KR76-2B,                            significant homology sequences the <br /> KR76-2D, KR78-6C), wild-type strains (AHF AH                              NCBI Blast databases (see Materials Methods). A short <br /> FA8L), tel1-1 strains (AJL210, KR75-1D, KR75-                           stretch sequence similarity yeast gene <br /> 2A) (see Materials Methods) (Table 1), PCR                                ARD1, encodes component N-terminal acetylase <br /> products sequenced. All tel2-1 strains contained                        affects TPE (3, 40) (Fig. 4, underlined sequence). How- <br /> tel2-1 mutation, A nucleotide 1006 (Fig. 4), wild- <br /> type tel1-1 cells contained TEL2 sequence. These re- <br /> sults, XhoI linker insertion, single-copy sup-                                         TABLE 2. Reduction TPE tel2-1 <br /> pression tel2-1, failure deletion complement <br /> the tel2-1 short-telomere length, weak Ts2, reduced TPE                                                                       Frequency FOAr <br />                                                                                                 Strain <br /> phenotypes, indicate 688-amino-acid ORF encodes                                                                    Median                    Range <br /> the TEL2 gene. Since lack TEL2 function lethal, tel2-1 <br /> allele loss-of-function mutation.                                            Haploid <br />    The tel2D::TRP1 allele used test effect tel2-1                    AHFA8L (wild type)           1.0 3 1022                  (0.36&#226;&#8364;&#8220;3.8) 3 1022 <br /> mutation chromosome stability yeasts. The rate loss                        KR78-6C (tel2-1)             8.9 3 1024 (0.089)a           (6.0&#226;&#8364;&#8220;13) 3 1024 <br /> yeast chromosome III measured TEL2/tel2D::TRP1                             Diploid <br /> tel2-1/tel2D::TRP1 diploids fluctuation analysis (45) (see                           TEL2/tel2D::TRP1            8.8 3 1022                  (8.6&#226;&#8364;&#8220;14.3) 3 1022 <br /> Materials Methods). Loss rates 6.1 3 1024 loss events                            tel2-1/tel2D::TRP1          3.7 3 1024 (0.0042)b          (3.0&#226;&#8364;&#8220;28) 3 1024 <br /> per cell division TEL2/tel2D::TRP1 diploid 15 3                             tel2-1/tel2D-L2             1.2 3 1023 (0.014)b         (0.20&#226;&#8364;&#8220;2.8) 3 1023 <br /> 1024 loss events cell division tel2-1/tel2D::TRP1                           a <br />                                                                                           Value parentheses fold difference versus median frequency FOAr in <br /> diploid (with standard deviations fluctu-                         wild-type haploid. <br /> ation test 2.0 3 1024 7.0 3 1024, respectively;                              b <br />                                                                                           Value parentheses fold difference versus median frequency FOAr in <br /> Materials Methods). Both loss rates higher                          TEL2/tel2D::TRP1 diploid. <br />  3102       RUNGE AND ZAKIAN                                                                                                                      MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />    FIG. 6. Unusual cells tel2D cultures. Phase-contrast micrographs cells culture tel2D::TRP1 cells bearing YRpL2T2, unstable plasmid bearing <br /> the TEL2 gene, shown. A budded cell wild-type morphology shown panel C large cell blebs. Large cells blebs make ;2% the <br /> culture. Such cells seen cultures TEL2 tel2-1 cells. <br />  <br />  <br />  <br /> ever, sequence similarity limited short region.                      FOA. Cells express URA3 gene sensitive FOA, <br /> When short region used search databases,                        cells lack URA3 expression resistant FOA <br /> proteins containing short stretches acidic amino acids                       (FOAR). In wild-type cells, copy URA3 the <br /> found, extent sequence similarity proteins                     cell near telomere, fraction cells rise FOAR <br /> was limited region, suggesting overall functional                         colonies (18). The fraction cells exhibit TPE varies from <br /> similarity. TEL2 showed similarity LAS1.                                       strain strain, ranging 1022 9 3 1021 (18, 63) (data <br />    The TEL2 ORF did structural similarity EST1                           shown). <br /> gene. The EST1 locus contains short overlapping ORFs                            To determine tel2-1 mutation altered TPE, URA3 <br /> 3 18 amino acids 80 bp upstream ATG                       gene placed left telomere chromosome VII the <br /> of EST1 ORF (31). The TEL2 locus 4-amino-acid                            congenic strains AHFA8L (wild type) KR78-6C (tel2-1) <br /> ORF overlaps ATG TEL2 ORF (Fig. 4). Small                            (see Materials Methods) fraction FOAR cells <br /> ORFs 59 region yeast genes rare. In                          determined (Table 2). The fraction FOAR <br /> genes occur, GCN4 CPA1, small ORFs                              tel2-1 cells 10 times lower congenic wild- <br /> are involved translational control expression (for re-                       type strain (Table 2). In parallel experiment, AHFA8L and <br /> view, reference 22). The TEL2 gene high proportion                         KR78-6C mated tel2D strains produce TEL2/tel2D:: <br /> of rare amino acid codons, like EST1 gene, suggesting                       TRP1, tel2-1/tel2D::TRP1, tel2-1/tel2-DL2 diploids, each <br /> the TEL2 EST1 gene products present low levels                         bearing copy URA3 VIIL (see Materials Meth- <br /> vivo.                                                                                ods). These diploids assayed TPE. The fractions of <br />    The TEL2 gene placed yeast physical genetic                        FOAR tel2-1/tel2D::TRP1 tel2-1/tel2-DL2 diploids were, re- <br /> maps. A 443-bp fragment TEL2 ORF used                                spectively, ;200 ;70 times lower wild- <br /> probe ordered phage clone set constitutes yeast                         type diploid (Table 2). Thus, ORF disruption mutations <br /> physical map (42a). The TEL2 gene mapped right arm                            failed complement reduced-TPE phenotype tel2-1 <br /> of chromosome VII, ;35 kb proximal SPT6, near CLY8,                           mutation. In contrast, strains bearing tel1-1 mutation have <br /> ;10 cM distal ESP1. Transcription TEL2 gene                             reported exhibit normal TPE levels (18), result con- <br /> oriented telomere.                                                        firmed study congenic tel1-1 strain (data not <br />    The meiotic recombination TEL2                              shown). Since tel1-1 cells shorter telomeres than <br /> ESP1 kilobase DNA 1.5 1.6 times higher                            tel2-1 cells, reduction TPE tel2-1 cells unlikely be <br /> expected (see Materials Methods). While searching                            secondary consequence short telomeres. <br /> homology 138-amino-acid ORF, 727 bp                             The yeast silent mating type cassettes HMLa HMRa <br /> TEL2 sequence overlap ESP1 sequence (33).                           contain copies MAT locus transcriptionally <br /> The esp1-1 mutation, resides 1.4 kb                         silenced (reviewed reference 27). Many trans-acting <br /> 4.5-kb ESP1 ORF (32a), mapped marked TEL2                              factors required silencing HMLa HMRa also <br /> gene (see Materials Methods). On basis data                           required TPE (3). The diploid strains described above <br /> and yeast genetic map (36), recombination                          allowed test tel2-1 mutation allowed expres- <br /> was 1.6 2.1 cM/kb compared average 0.67 cM/kb                         sion silent mating type cassettes. <br /> for chromosome VII. Therefore, region TEL2                              MATa/MATa diploids sporulate form tetrads, but <br /> ESP1 appears hotspot meiotic recombination.                              diploids heterozygous MAT cannot. In order for <br />    The tel2-1 mutation reduces position effect telomeres                      MATD/MAT(a a) diploid sporulate, information from <br /> but silent mating type cassettes. When gene                          HMRa HMLa expressed. TEL2/tel2D tel2-1/ <br /> placed near telomere Drosophila melanogaster S. cerevi-                      tel2D diploids transformed plasmid delete either <br /> siae, expression gene repressed manner                      MATa MATa (see Materials Methods). The tel2-1/tel2D <br /> is similar position effect repression heterochromatin                     MATD/MAT(a a) diploids transformed with <br /> (18, 28). To determine tel1-1 tel2-1 mutation affected                     YEp24 pKR60. The plasmid pKR60 2mm-URA3 plas- <br /> other aspects telomere function length, levels                     mid bearing GAL4-SIR4 gene fusion expression de- <br /> TPE examined. In yeast cells, TPE URA3 gene                             represses silent mating type cassettes (12) allows <br /> usually quantitated determination resistance drug                       MATD/MAT(a a) diploids sporulate (see Ma- <br />  VOL. 16, 1996                                                                                         TEL2 IS AN ESSENTIAL GENE               3103 <br />  <br />  <br />                  TABLE 3. Effect tel2-1 mutation                      generations required chro- <br />                   MATD/MAT(a a) diploid sporulation                          mosomes lose telomere function lost, leading to <br />   tel2-1/tel2D           No. spore        No. cells          Fraction <br />                                                                                 cell death. For example, cells lacking yeast telomerase RNA or <br />      diploid               figuresa           counted             sporulated    bearing est1D mutation 40 100 cell divisions <br />                                                                                 die (31, 55). However, tel2-1 tel1-1 cell telo- <br /> MATa/MATa                                                                       meres shorten reach new equilibrium length, sug- <br />  YEp24                       20                 1,112               0.018       gesting telomere length regulation defect opposed fail- <br />  pKR60b                      10                 1,161               0.0086 <br /> MATa/MATD <br />                                                                                 ure replicate telomeres. In addition, defect telomere <br />  YEp24                        0                10,392           ,9.6 3 1025     replication does readily account large-cell-with- <br />  pKR60b                      27                   876             0.031         small-bleb phenotype tel2D cells. We suggest Tel2p is <br /> MATD/MATa                                                                       involved assembling chromatin structure telomere <br />  YEp24                        0                10,486           ,9.5 3 1025     adjacent DNA. In respect, Tel2p resemble the <br />  pKR60b                      37                   792             0.047         known telomere binding protein Rap1p, essential gene <br />  a <br />      Sum dyads, triads, tetrads. <br />                                                                                 product roles silencing, transcriptional activation, <br />  b <br />      Presence plasmid derepresses silencing telomeres HM loci.   telomere maintenance, nuclear organization (7, 23, 53). <br />                                                                                    The yeast telomeric region unusual chromatin struc- <br />                                                                                 ture. Yeast terminal C1&#226;&#8364;&#8220;3A repeats nonnucleosomal <br /> terials Methods). YEp24 similar vector insert.                 chromatin structure called telosome (64), Rap1p <br /> These diploids sporulated liquid medium,                        major component (14, 65). The DNA adjacent the <br /> numbers tetrads counted (Table 3). The MATa/MATa                        C1&#226;&#8364;&#8220;3A repeats organized nucleosomes (64), unlike <br /> tel2-1/tel2D diploids sporulated 1 5% cells                nucleosomes regions genome, histones <br /> tetrads, triads, dyads. Different TEL2/tel2D diploids showed                 nucleosomes hypoacetylated (5). Genes placed in <br /> 5 30% sporulation conditions (see Materials                      region subject TPE (18) protected from <br /> and Methods). No sporulated cells observed tel2-                    methylation dam methylase (17). All effects are <br /> 1/tel2D MATD/MAT(a a) diploids bearing YEp24. However,                       considered to special chromatin structure the <br /> these cells able sporulate silencing HMLa                      telomeric region (3, 5, 16, 20, 48, 56). We hypothesize that <br /> HMRa relieved presence pKR60. These results                       Tel2p required chromatin assembly telomeres <br /> indicate position effect HMRa HMLa re-                       genome. In hypothesis, tel2-1 cells <br /> lieved fewer 1% tel2-1/tel2D cells. As tel2-1/tel2D              form altered telosome turn alters telomere length <br /> diploid relieved TPE approximately 100-fold (Table 2),                       TPE. A requirement hypothesis genes <br /> these data suggest tel2-1 mutation alters silencing               affecting cell morphology particularly sensitive changes <br /> at telomeres far greater effect telomeres                   Tel2p levels. In model, tel2D cells run Tel2p, <br /> other loci.                                                                     transcriptional regulation genes altered, result- <br />                                                                                 ing knobbed-cell phenotype. The high proportion rare <br />                                                                                 amino acid codons TEL2 ORF observation that <br />                           DISCUSSION                                            GAL1-TEL2 fusions complement tel2D galactose <br />    The tel2-1 mutation affected telomeres ways. First,                 complement glucose (data shown) suggest only <br /> tel2-1 cells maintained shorter-than-normal telomeric tracts                    low levels Tel2p required carry function. Thus, <br /> (32). Second, telomeres tel2-1 cells remained short                 role Tel2p chromatin enzymatic as <br /> cells carried plasmids contained C1&#226;&#8364;&#8220;3A DNA expressed                    opposed structural one. Under hypothesis, TEL2 <br /> a deletion derivative Rap1p (Fig. 1), caused                   resembles Drosophila Suvar genes, encode pro- <br /> telomere lengthening congenic wild-type cells (Fig. 1). In                   teins alter chromatin structure essential (for a <br /> addition, tel2-1 mutation prevented slow growth                      review, reference 42). <br /> 218C caused expression large amounts carboxyl                         Analysis tel1-1 tel2-1 cells sequencing the <br /> terminus RAP1 protein. Third, tel2-1 mutation re-                    TEL2 gene revealed common features among <br /> duced TPE detectable effect silencing HMLa                     mutations affecting telomere length. First, cells short telo- <br /> or HMRa. The tel2-1 mutation overexpression portion                    meres tel1-1, tel2-1 (Fig. 2), est1-1 mutation <br /> of TLC1 gene perturbations date                   (31, 44a, work) impaired growth high temper- <br /> been alter TPE affecting HM loci (3, 25,                   atures. This weak Ts2 phenotype observed these <br /> 55). Thus, absence Tel2p affects various aspects                strains telomeres long (see Results) <br /> telomere behavior, likely play important role                    (44a). This weak Ts2 phenotype result in- <br /> telomere structure function. The TEL2 gene, en-                       herent temperature lability essential telomere DNA-protein <br /> codes 688-amino-acid protein striking similarity                   interactions apparent telomeres short. <br /> any sequences NCBI database, essential                    The short-telomere phenotype associated tel1-1 and <br /> growth. Cells ran TEL2 gene product arrested                   tel2-1 mutations consistent hypothesis TEL1 <br /> a novel phenotype, large round cells small blebs.                          TEL2 regulators telomere length. The tel1-1 and <br />    Although multiple-telomere phenotype tel2-1 cells                     tel2-1 mutations change range allowable telomere <br /> suggests TEL2 protein (Tel2p) plays direct role                   lengths cells. The TEL2 sequence reveals presence a <br /> telomeres, rapid-death small-bleb phenotypes tel2                    short ORF overlaps TEL2 ATG, structural feature <br /> null alleles suggest Tel2p probably functions                 shared EST1 gene (31) yeast genes <br /> the cell telomeres. Failure telomere repli-                 (13, 22). The presence upstream ORFs, sug- <br /> cation expected result gradual shortening termi-                gesting translational control mechanism similar of <br /> nal C1&#226;&#8364;&#8220;3A sequences telomere function lost. Cellular                   GCN4 (22), raises possibility expression several <br /> DNA polymerases require RNA primer ini-                        genes affecting telomere length coordinately regulated. <br /> tiate synthesis. The gradual loss telomeric DNA                      Like tel2-1 cells, tel1-1 cells did respond perturbations <br /> failure replace RNA primer lead predict                     caused telomere lengthening (Fig. 1). The TEL1 gene has <br />  3104      RUNGE AND ZAKIAN                                                                                                                         MOL. CELL. BIOL. <br />  <br />  <br /> recently cloned encodes protein similarity                        7. Buchman, A. R., N. F. Lue, R. D. Kornberg. 1988. Connections between <br /> known phosphoinositide 3-kinases. TEL1 strongest se-                            transcriptional activators, silencers, telomeres revealed functional <br />                                                                                         analysis yeast DNA-binding protein. Mol. Cell. Biol. 8:5086&#226;&#8364;&#8220;5099. <br /> quence similarity known proteins ataxia telangi-                       8. Campbell, J. L., C. S. Newlon. 1991. Chromosomal DNA replication, p. <br /> ectasia gene product similar S. cerevisiae                           41&#226;&#8364;&#8220;146. In J. R. Broach, J. R. Pringle, E. W. Jones (ed.), The molecular <br /> MEC1 Schizosaccharomyces pombe rad3 cell cycle check-                               cellular biology yeast Saccharomyces: genome dynamics, protein <br /> point genes (19, 35, 51). These data suggest signal trans-                         synthesis energetics. Cold Spring Harbor Laboratory Press, Cold Spring <br />                                                                                         Harbor, N.Y. <br /> duction play role regulating monitoring telomere                         9. Carlson, M., D. Botstein. 1982. Two differentially regulated mRNAs <br /> length. The sequence TEL2 gene similar                         different 59 ends encode secreted intracellular forms yeast invertase. <br /> any known gene coding product involved signal trans-                           Cell 28:145&#226;&#8364;&#8220;154. <br /> duction cell cycle checkpoints; however, tel1-1 tel2-1 double                    10. Carson, M. J., L. Hartwell. 1985. CDC17: essential gene prevents <br />                                                                                         telomere elongation yeast. Cell 42:249&#226;&#8364;&#8220;257. <br /> mutants telomeres length tel1-1                           11. Chan, C. S. M., B.-K. Tye. 1983. Organization DNA sequences and <br /> single mutants, suggesting genes act                          replication origins yeast telomeres. Cell 33:563&#226;&#8364;&#8220;573. <br /> same pathway (32).                                                                  12. Chien, C. T., P. L. Bartel, R. Sternglanz, S. Fields. 1991. The two-hybrid <br />    Telomere length appears regulated S. cerevisiae.                            system: method identify clone genes proteins interact with <br />                                                                                         protein interest. Proc. Natl. Acad. Sci. USA 88:9578&#226;&#8364;&#8220;9582. <br /> When C1&#226;&#8364;&#8220;3A sequences wild-type yeast cells                         13. Cigan, A. M., T. F. Donahue. 1987. Sequence structural features <br /> is increased ;30-fold using high-copy-number plasmids,                               associated translational initiator regions yeast&#226;&#8364;&#8221;a review. Gene 59: <br /> chromosomal telomeres reach new equilibrium length in-                                1&#226;&#8364;&#8220;18. <br /> stead increasing length bounds (46) (see                          14. Conrad, M. N., J. H. Wright, A. J. Wolf, V. A. Zakian. 1990. RAP1 <br />                                                                                         protein interacts yeast telomeres vivo: overproduction alters telomere <br /> introduction). When linear plasmids formed yeasts                             structure decreases chromosome stability. Cell 63:739&#226;&#8364;&#8220;750. <br /> using short long stretches C1&#226;&#8364;&#8220;3A, final length C1&#226;&#8364;&#8220;3A                   15. Doseff, A. I., K. T. Arndt. 1995. LAS1 essential nuclear protein <br /> sequence plasmid termini similar length                         involved cell morphogenesis cell surface growth. Genetics 141:857&#226;&#8364;&#8220; <br /> the chromosomal telomeres host strain (37, 41, 46, 59,                           871. <br />                                                                                     16. Eissenberg, J. C. 1989. Position effect variegation Drosophila: a <br /> 61). This result obtained input plasmid starts                         genetics chromatin assembly. Bioessays 11:14&#226;&#8364;&#8220;17. <br /> with 250 350 bp C1&#226;&#8364;&#8220;3A sequence (46, 61). The terminally                       16a.Gasser, S. Personal communication. <br /> repeated DNA sequences, proteins bind them,                             16b.Gottschling, D. Personal communication. <br /> appear required telomere replication protec-                      17. Gottschling, D. E. 1992. Telomere-proximal DNA Saccharomyces cerevi- <br />                                                                                         siae refractory methyltransferase activity vivo. Proc. Natl. Acad. Sci. <br /> tion functions yeasts (14, 38, 41, 49). If telomere DNA                              USA 89:4062&#226;&#8364;&#8220;4065. <br /> length short, telomere function                         18. Gottschling, D. E., O. M. Aparicio, B. L. Billington, V. A. Zakian. 1990. <br /> lost. In Tetrahymena Trypanosoma spp., telomere                            Position effect yeast telomeres: reversible repression polII transcription. <br /> length continuously increase mitotic growth,                             Cell 63:751&#226;&#8364;&#8220;762. <br />                                                                                     19. Greenwell, P. A., S. L. Kronmal, S. E. Porter, J. Gassenhuber, B. Obermaier, <br /> subset cells average telomere length takes                             T. D. Petes. 1995. TEL1, gene involved controlling telomere length <br /> population (4, 26). These considerations suggest                        S. cerevisiae, homologous human ataxia telangiectasia gene. Cell <br /> growth advantage maintaining telomere length cer-                           82:823&#226;&#8364;&#8220;829. <br /> tain range. Thus, role telomere length regulation                         20. Hecht, A., T. Laroche, S. Strahl-Bolsinger, S. M. Gasser, M. Grunstein. <br />                                                                                         1995. Histone H3 H4 N-termini interact SIR3 SIR4 proteins: a <br /> ciliates yeasts maintain telomere length                              molecular model formation heterochromatin yeast. Cell 80:583&#226;&#8364;&#8220; <br /> sufficient accomplish telomere functions imposing                          592. <br /> burden cell growth. The results presented suggest                      21. Henikoff, S., M. K. Eghtedarzadeh. 1987. Conserved arrangement of <br /> TEL2 essential gene product yeast                            nested genes Drosophila Gart locus. Genetics 117:711&#226;&#8364;&#8220;725. <br />                                                                                     22. Hinnebusch, A. G., S. W. Liebman. 1991. Protein synthesis trans- <br /> system maintains telomere length given range,                             lational control Saccharomyces cerevisiae, p. 627&#226;&#8364;&#8220;735. In J. R. Broach, J. R. <br /> perhaps altering chromatin structure telomere                             Pringle, E. W. Jones (ed.), The molecular cellular biology the <br /> elsewhere.                                                                              yeast Saccharomyces: genome dynamics, protein synthesis energetics. <br />                                                                                         Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y. <br />                                                                                     23. Hofmann, J. F., T. Laroche, A. H. Brand, S. M. Gasser. 1989. RAP-1 <br />                          ACKNOWLEDGMENTS                                                factor necessary DNA loop formation vitro silent mating type <br />    We thank Bob Levis, Linda Breeden, Meng Chao Yao, mem-                           locus HML. Cell 57:725&#226;&#8364;&#8220;737. <br />                                                                                     24. Kyrion, G., K. A. Boakye, A. J. Lustig. 1992. C-terminal truncation of <br /> bers Zakian lab helpful discussion Alex Wolf Kurt <br />                                                                                         RAP1 results deregulation telomere size, stability, function in <br /> Hotmire technical assistance course project.                     Saccharomyces cerevisiae. Mol. Cell. Biol. 12:5159&#226;&#8364;&#8220;5173. <br />    This work supported NIH grants GM43265 GM26938                     25. Kyrion, G., K. Liu, C. Liu, A. J. Lustig. 1993. RAP1 telomere <br /> V.A.Z. GM50752 K.W.R. K.W.R. American Cancer So-                           structure regulate telomere position effects Saccharomyces cerevisiae. <br /> ciety Junior Faculty Research Professor (JFRA-512).                                     Genes Dev. 7:1146&#226;&#8364;&#8220;1159. <br />                                                                                     26. Larson, D. D., E. A. Spangler, E. H. Blackburn. 1987. Dynamics of <br />                                 REFERENCES                                              telomere length variation Tetrahymena thermophila. Cell 50:477&#226;&#8364;&#8220;483. <br /> 1. Alani, E., L. Cao, N. Kleckner. 1987. A method gene disruption      27. Laurenson, P., J. Rine. 1992. Silencers, silencing, heritable tran- <br />    allows repeated use URA3 selection construction multiply                scriptional states. Microbiol. Rev. 56:543&#226;&#8364;&#8220;560. <br />    disrupted yeast strains. Genetics 116:541&#226;&#8364;&#8220;545.                                   28. Levis, R., T. Hazelrigg, G. M. Rubin. 1985. Effects genomic position <br /> 2. Altschul, S. F., W. Gish, W. Miller, E. W. Myers, D. J. Lipman. 1990.            expression transduced copies white gene Drosophila. <br />    Basic local alignment search tool. J. Mol. Biol. 215:403&#226;&#8364;&#8220;410.                        Science 229:558&#226;&#8364;&#8220;561. <br /> 3. Aparicio, O. M., B. L. Billington, D. E. Gottschling. 1991. Modifiers     29. Lima-de-Faria, A. 1983. Molecular evolution organization chro- <br />    position effect shared telomeric silent mating-type loci S.       mosome. Elsevier, Amsterdam. <br />    cerevisiae. Cell 66:1279&#226;&#8364;&#8220;1287.                                                   30. Lin, J. J., V. A. Zakian. 1995. An vitro assay Saccharomyces <br /> 4. Bernards, A., P. A. M. Michels, G. R. Lincke, P. Borst. 1983. Growth          telomerase requires EST1. Cell 81:1127&#226;&#8364;&#8220;1135. <br />    chromosomal ends multiplying trypanosomes. Nature (London) 303:592&#226;&#8364;&#8220;           31. Lundblad, V., J. W. Szostak. 1989. A mutant defect telomere <br />    597.                                                                                 elongation leads senescence yeast. Cell 57:633&#226;&#8364;&#8220;643. <br /> 5. Braunstein, M., A. B. Rose, S. G. Holmes, C. D. Allis, J. Broach. 1993.      31a.Lustig, A. Personal communication. <br />    Transcriptional silencing yeast associated reduced nucleosome         32. Lustig, A. J., T. D. Petes. 1986. Identification yeast mutants with <br />    acetylation. Genes Dev. 7:592&#226;&#8364;&#8220;604.                                                   altered telomere structure. Proc. Natl. Acad. Sci. USA 83:1398&#226;&#8364;&#8220;1402. <br /> 6. Broach, J. R., F. C. Volkert. 1991. Circular DNA plasmids yeasts,         32a.McGrew, J. Personal communication. <br />    p. 297&#226;&#8364;&#8220;331. In J. R. Broach, J. R. Pringle, E. W. Jones (ed.), The           33. McGrew, J. T., L. Goetsch, B. Byers, P. Baum. 1992. Requirement for <br />    molecular cellular biology yeast Saccharomyces: genome dynam-             ESP1 nuclear division Saccharomyces cerevisiae. Mol. Biol. Cell <br />    ics, protein synthesis energetics. Cold Spring Harbor Laboratory Press,          3:1443&#226;&#8364;&#8220;1454. <br />    Cold Spring Harbor, N.Y.                                                         34. Miller, J. H. 1972. Experiments molecular genetics, p. 56. Cold Spring <br />  VOL. 16, 1996                                                                                                   TEL2 IS AN ESSENTIAL GENE                        3105 <br />  <br />  <br />     Harbor Laboratory, Cold Spring Harbor, N.Y.                                         chain-terminating inhibitors. Proc. Natl. Acad. Sci. USA 74:5463&#226;&#8364;&#8220;5467. <br /> 35. Morrow, D. M., D. A. Tagle, Y. Shiloh, F. S. Collins, P. Hieter. 1995.      51. Savitsky, K., A. Bar-Shira, S. Gilad, G. Rotman, Y. Ziv, L. Vanagaite, D. A. <br />     TEL1, S. cerevisiae homolog human gene mutated ataxia tel-             Tagle, S. Smith, T. Uziel, S. Sfez, M. Ashkenazi, I. Pecker, M. Frydman, R. <br />     angiectasia, functionally related yeast checkpoint gene MEC1. Cell        Harnik, S. R. Patanjali, A. Simmons, G. A. Clines, A. Sartiel, R. A. Gatti, L. <br />     82:831&#226;&#8364;&#8220;840.                                                                         Chessa, O. Sanal, M. F. Lavin, N. G. J. Jaspers, A. M. R. Taylor, C. F. Arlett, <br /> 36. Mortimer, R. K., C. R. Contopoulou, J. S. King. 1991. Genetic               T. Miki, S. M. Weissman, M. Lovett, F. S. Collins, Y. Shiloh. 1995. A <br />     physical maps Saccharomyces cerevisiae, p. 737&#226;&#8364;&#8220;752. In J. R. Broach, J. R.       single ataxia telangiectasia gene product similar PI-3 kinase. Sci- <br />     Pringle, E. W. Jones (ed.), The molecular cellular biology           ence 268:1749&#226;&#8364;&#8220;1753. <br />     yeast Saccharomyces: genome dynamics, protein synthesis energetics.         52. Schiestl, R. H., R. D. Gietz. 1989. High e <br /> </body></html>