<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR AND CELLULAR BIOLOGY, May 2008, p. 3336&#226;&#8364;&#8220;3343                                                                           Vol. 28, No. 10 <br /> 0270-7306/08/$08.00&#226;&#171;&#185;0 doi:10.1128/MCB.00567-07 <br /> Copyright &#194;&#169; 2008, American Society Microbiology. All Rights Reserved. <br />  <br />  <br />  <br />          Cdc1p Is Endoplasmic Reticulum-Localized Putative Lipid <br />             Phosphatase That Affects Golgi Inheritance Actin <br />                    Polarization Activating Ca2&#226;&#171;&#185; Signaling&#228;&#338;&#164;&#226;&#8364;&#160; <br />                Eugene Losev, Effrosyni Papanikou, Olivia W. Rossanese, Benjamin S. Glick* <br />                Department Molecular Genetics Cell Biology, The University Chicago, Chicago, Illinois 60637 <br />                          Received 30 March 2007/Returned modification 19 June 2007/Accepted 27 February 2008 <br />  <br />             In budding yeast Saccharomyces cerevisiae, mutations essential gene CDC1 cause defects Golgi <br />           inheritance actin polarization. However, biochemical function Cdc1p unknown. Previous work <br />           showed cdc1 mutants accumulate intracellular Ca2&#226;&#180;&#8482; display enhanced sensitivity extracellular <br />           Mn2&#226;&#180;&#8482; concentration, suggesting Cdc1p regulate divalent cation homeostasis. By contrast, data <br />           indicate Cdc1p Mn2&#226;&#180;&#8482;-dependent protein affect Ca2&#226;&#180;&#8482; levels. We identified cdc1 allele that <br />           activates Ca2&#226;&#180;&#8482; signaling does enhanced sensitivity Mn2&#226;&#180;&#8482; concentration. Furthermore, our <br />           studies Cdc1p endoplasmic reticulum-localized transmembrane protein putative <br />           phosphoesterase domain facing lumen. cdc1 mutant cells accumulate unidentified phospholipid, sug- <br />           gesting Cdc1p lipid phosphatase. Previous work showed deletion plasma membrane <br />           Ca2&#226;&#180;&#8482; channel Cch1p partially suppressed cdc1 growth <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span>phenotype</span>, deletion Cch1p also <br />           suppresses Golgi inheritance actin polarization phenotypes. The combined data fit model which <br />           cdc1 mutant phenotypes result accumulation phosphorylated lipid activates Ca2&#226;&#180;&#8482; signaling. <br />  <br />  <br />    We identified CDC1 gene screen temperature-                     buds (14). Subsequent work revealed CDC1 essential <br /> sensitive mutants defects inheritance late Golgi                 gene implicated function processes mating, <br /> cisternae (33). All mutants showed normal Golgi                     spindle-pole body duplication, intrachromosomal recombina- <br /> structure reproducible defects Golgi inheritance                     tion, divalent cation homeostasis (6, 13, 28, 30, 39). De- <br /> carried mutations CDC1. These cdc1 mutants de-                      spite connections, biochemical function of <br /> polarized actin. However, actin depolarization solely                   Cdc1p remained unknown. <br /> responsible Golgi inheritance phenotype treat-                     The strongest evidence function Cdc1p came <br /> ment wild-type cells actin-depolymerizing drug                      study linking Ca2&#226;&#171;&#185; regulation. Whole-cell Ca2&#226;&#171;&#185; <br /> latrunculin A (2) caused mild Golgi inheritance defect.                  levels cdc1 mutants highly elevated (29). cdc1 mutations <br /> In addition, Golgi inheritance affected act1-&#226;&#338;&#172;DSE                    activate calcineurin, Ca&#226;&#171;&#185;2/calmodulin-dependent protein <br /> cells, contain mutant actin poor substrate                phosphatase mediates Ca2&#226;&#171;&#185; signaling activation the <br /> myosin-driven transport (8, 33). On hand,                         transcription factor Crz1p (9). Moreover, deletion cal- <br /> myo2-66 <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span>mutation</span>, compromises actin-binding mo-                       cineurin regulatory subunit CNB1 synthetically lethal with <br /> tor domain type V myosin Myo2p (24), caused pro-                       cdc1 mutations (29). In cdc1 mutants, deletion plasma <br /> nounced defect late Golgi inheritance. These observations                    membrane Ca2&#226;&#171;&#185; channel Cch1p regulatory subunit <br /> led following model inheritance late Golgi <br />                                                                                 Mid1p restores intracellular Ca2&#226;&#171;&#185; wild-type levels par- <br /> cisternae (31, 33): late Golgi elements reach bud actin- <br />                                                                                 tially rescues growth defect cdc1 mutants (29). These <br /> dependent -independent pathways, Myo2p tethers <br />                                                                                 data suggested Cdc1p affects intracellular Ca2&#226;&#171;&#185; <br /> them bud. cdc1 mutations probably disrupt the <br />                                                                                 levels growth defect cdc1 mutants part <br /> actin-dependent transport Myo2p-dependent tether- <br />                                                                                 Ca2&#226;&#171;&#185; influx (29). <br /> ing, resulting strong defect Golgi inheritance. <br />                                                                                    There evidence linking Cdc1p Mn2&#226;&#171;&#185;. cdc1 mutants <br />    Testing model require knowing biochemical <br />                                                                                 rescued Mn2&#226;&#171;&#185; addition sensitive the <br /> function Cdc1p. The original cdc1 mutants isolated by <br /> Hartwell colleagues screen cell division cycle                     divalent cation chelator EGTA (25, 28). The EGTA sensitivity <br /> defects (15). The mutant cells initially arrest                   alleviated overexpression Smf1p (25), a <br /> with small buds, 2N DNA, undivided nuclei, later                        metal ion transporter high affinity Mn2&#226;&#171;&#185; (39). Fur- <br /> studies showed cdc1 cells actually arrested                   thermore, SMF1 deletion, makes cells EGTA sensi- <br />                                                                                 tive (39), rescued CDC1 overexpression (28). Sim- <br />                                                                                 ilarly, CDC1 overexpression rescued EGTA sensitivity of <br />    * Corresponding author. Mailing address: Department Molecular             cells carrying deletion PMR1, encodes Golgi <br /> Genetics Cell Biology, The University Chicago, 920 East 58th             body-localized Ca2&#226;&#171;&#185; Mn2&#226;&#171;&#185; transporter (28). PMR1 over- <br /> Street, CLSC 801, Chicago, IL 60637. Phone: (773) 702-5315. Fax:                expression exacerbated cdc1 growth defect (28). Lastly, <br /> (773) 702-3172. E-mail: bsglick@uchicago.edu. <br />                                                                                 work frost gene, Neurospora homolog CDC1, re- <br />    &#226;&#8364;&#160; Supplemental material article http://mcb <br /> .asm.org/.                                                                      vealed connection Mn2&#226;&#171;&#185; Ca2&#226;&#171;&#185; (37). These observa- <br />    &#228;&#338;&#164; <br />      Published ahead print 10 March 2008.                                 tions led hypothesis Cdc1p regulates Mn2&#226;&#171;&#185; ho- <br />  <br />                                                                          3336 <br />  VOL. 28, 2008                                                      cdc1 PHENOTYPES RESULT <span id='am-1' about='protege:FROM' typeof='owl:Thing'>FROM</span> ACTIVATING Ca2&#226;&#171;&#185; SIGNALING                                          3337 <br />  <br />  <br /> meostasis (28). This model predicted intracellular Mn2&#226;&#171;&#185; <br /> levels cdc1 mutants different levels wild- <br /> type cells. However, whole-cell Mn2&#226;&#171;&#185; content cdc1 <br /> mutants examined, difference relative wild type was <br /> observed (27). Thus, link Cdc1p Mn2&#226;&#171;&#185; re- <br /> mained elusive. <br />    A genetic screen cdc1 suppressors revealed cells are <br /> able survive CDC1 PER1 (COS16) deleted <br /> (27). Per1p endoplasmic reticulum (ER)-localized pro- <br /> tein (17) contains putative transmembrane domains. <br /> When whole-cell Mn2&#226;&#171;&#185; content per1&#226;&#338;&#172; mutants ex- <br /> amined, difference relative wild-type cells, <br /> again arguing role Cdc1p Mn2&#226;&#171;&#185; homeostasis <br /> (27). <br />    An alternative hypothesis states Cdc1p Mn2&#226;&#171;&#185;-de- <br /> pendent protein affects Ca2&#226;&#171;&#185; levels (10). This idea was <br /> supported sequence analysis suggesting Cdc1p a <br /> phosphoesterase motif related phosphatases use <br /> Mn2&#226;&#171;&#185; cofactor (34). Here, sought test alterna- <br /> tive hypothesis. In addition, asked link between <br /> Cdc1p Ca2&#226;&#171;&#185; explain effect cdc1 mutations on <br />                                                                                         FIG. 1. Two cdc1 alleles different sensitivities Mn2&#226;&#171;&#185; but <br /> Golgi inheritance actin polarization. <br />                                                                                      exhibit activated Ca2&#226;&#171;&#185; signaling restrictive temperature. (a) <br />                                                                                      Wild-type (WT), cdc1-310, cdc1-314 mutant strains grown <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> <br />                                                                                      stationary phase serially diluted (1:10) times. The strains were <br />                         MATERIALS AND METHODS <br />                                                                                      replica plated YPD medium 25&#194;&#176;C 37&#194;&#176;C, 37&#194;&#176;C 2 mM <br />    Yeast strains reagents. Saccharomyces cerevisiae strains grown rich   MnCl2, 25&#194;&#176;C 2 mM EGTA. (b) The activation Ca2&#226;&#171;&#185; <br /> glucose medium (yeast extract, peptone, dextrose [YPD]) minimal glu-          signaling measured using CDRE-lacZ reporter. Wild-type, cdc1- <br /> cose medium (synthetic dextrose [SD]) (35). SD-URA SD medium lacking              310, cdc1-314 cells grown SD-URA medium early log <br /> uracil. The experiments shown Fig. 1 performed haploid strain       phase shifted 33.5&#194;&#176;C indicated times. Reporter activity <br /> BGY103, genotype leu2-3,112 ura3-52 rme1 trp1 his4 GAL&#226;&#171;&#185; HMLa           measured hour 3 h. <br /> SEC7-EGFP bar1::URA3 (33). All experiments performed de- <br /> rivatives haploid strain JK9-3d, genotype leu2-3,112 ura3-52 <br /> rme1 trp1 his4 (21). <br />    The cdc1-314 allele contains T377I mutation (ACT ATT), cdc1-            &#226;?&#164;-Galactosidase Golgi inheritance assays. &#226;?&#164;-Galactosidase reporter as- <br /> 310 allele contains L356F mutation (ACT ATT). Our CDC1 wild-type               says performed using CDRE-lacZ (38) MPK1-lacZ (18) reporter <br /> allele serine codon (ATC) position 441 instead proline codon           plasmids. The assays performed using yeast &#226;?&#164;-galactosidase assay kit <br /> (ACC), indicated Saccharomyces Genome Database. To make cdc1-            (Pierce, Rockford, IL). Error bars figures represent standard errors the <br /> 314 derivative given strain, full-<span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span> CDC1 gene T377I mutation    means. <br /> was integrated CDC1 locus create gene duplication. Successful inte-         Golgi inheritance quantified previously described (33). In brief, we <br /> grant colonies identified PCR. One copy CDC1 popped          measured percentage buds contained late Golgi membranes a <br /> using 5-fluoroorotic acid (Toronto Research Chemicals, Toronto, Canada),         function bud size. Bud size classes III, IV, V combined class, <br /> the resulting colonies screened temperature sensitivity.                    termed III-V. <br />    To delete CCH1 gene, fragment PCR amplified cch1&#226;&#338;&#172; strain            EndoH analysis. Total cellular protein extracted adjusting culture <br /> created Saccharomyces Genome Deletion Project (http://www-sequence            10% trichloroacetic acid, incubating 60&#194;&#176;C 5 min, chilling ice 5 <br /> .stanford.edu/group/yeast_deletion_project/deletions3.html). This fragment,    min, centrifuging 5 min 3,000 &#226;&#171;&#187; g, resuspending sodium dodecyl <br /> included kanMX gene flanked 400 bp upstream CCH1 coding sequence       sulfate-polyacrylamide gel electrophoresis (SDS-PAGE) sample buffer supple- <br /> and 500 bp downstream CCH1 coding sequence, integrated           mented 50 mM Na&#226;&#171;&#185;-PIPES [piperazine-N,N&#226;&#172;&#732;-bis(2-ethanesulfonic acid); <br /> genome homologous recombination. A similar strategy used delete        pH 7.5]. The samples split endoglycosidase H (EndoH)-treated and <br /> MPK1 gene. The PER1 GUP1 genes deleted replacement LEU2             -untreated samples. EndoH-treated samples incubated 10 &#226;?&#174;l EndoH <br /> using PCR-based homologous recombination (3).                                        buffer (Prozyme, San Leandro, CA), boiled 10 min, chilled ice 5 min, <br />    The inhibitor FK506 (LC Laboratories, Woburn, MA) stored &#226;&#171;&#186;20&#194;&#176;C          treated 2.5 &#226;?&#174;l EndoH (30 &#226;?&#174;U/&#226;?&#174;l) (Prozyme), incubated 37&#194;&#176;C over- <br /> ethanol 10 mg/ml. The final concentration FK506 cell cultures 1         night, boiled 10 min, examined using SDS-PAGE. Untreated samples <br /> &#226;?&#174;g/ml.                                                                               boiled 10 min SDS-PAGE buffer examined using SDS-PAGE. <br />    Microscopy. Immunofluorescence microscopy performed previously             The separated proteins transferred polyvinylidene difluoride mem- <br /> described (33). Rabbit polyclonal anti-Kar2p antibody (a gift Jack Rose)        branes, tagged Cdc1p-myc detected SuperSignal West Femto kit <br /> was used 1:5,000 dilution. Mouse monoclonal anti-myc antibody (clone            (Pierce, Rockford, IL) using peroxidase-conjugated anti-c-myc antibody (clone <br /> 9E10; Roche, Indianapolis, IN) used 2 &#226;?&#174;g/ml.                                  9E10; Roche, Indianapolis, IN) 1:1,000. <br />    Lipid analysis. Phospholipid radiolabeling, extraction, separation two- <br /> dimensional thin-layer chromatography (TLC) performed previously <br /> described (36). The lipid extracts dried nitrogen                                            RESULTS <br /> deacetylation step needed. After extraction, lipids resuspended <br />                                                                                                       2&#226;&#180;&#8482; <br /> in solvent A (95% ethanol, water, diethyl ether, pyridine, concentrated ammo-           Role Mn Cdc1p function. We reasoned Cdc1p <br /> nium hydroxide [15:15:5:1:0.018, volume]) resolved TLC plates (Silica      Mn2&#226;&#171;&#185;-dependent protein affects Ca2&#226;&#171;&#185; levels, the <br /> Gel 60-precoated plates TLC; EMD, Germany). The results quantified          Mn2&#226;&#171;&#185; Ca2&#226;&#171;&#185; links potentially uncoupled some <br /> using Molecular Dynamics Storm 860 PhosphorImager (GE Healthcare, Pis- <br /> cataway, NJ) ImageQuant, version 1.2, software. For lipid marked with <br />                                                                                      cdc1 mutants. The previously described cdc1 strains ele- <br /> the arrows Fig. 6a, signal measured fraction total radio-     vated intracellular Ca2&#226;&#171;&#185; restrictive temperature (25, 28). <br /> activity present labeled species migrated origin.       In addition, strains rescued supplementing the <br />  3338     LOSEV ET AL.                                                                                                        MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />   FIG. 2. The putative phosphoesterase domain Cdc1p functionally important faces lumen ER. (a) Sequence analysis revealed <br /> a putative phosphoesterase domain (red box) hydrophobic putative transmembrane domains (green boxes). The conserved amino acid <br /> residues labeled red predicted chelate divalent metal ion. (b) Cdc1p mutants alanine substitutions conserved residues <br /> indicated panel expressed CEN plasmid cdc1-314 strain 25&#194;&#176;C 37&#194;&#176;C. A mutant His287, predicted to <br /> chelate divalent metal ion, used control. Lack growth 37&#194;&#176;C shows mutated residues important Cdc1p function. <br /> (c) Cells expressing myc-tagged Cdc1p immunolabeled anti-myc antibody (red) anti-Kar2p antibody (green). Scale bar, 5 &#226;?&#174;m. (d) <br /> The potential N-linked glycosylation sites Cdc1p mutated separately combination myc-tagged Cdc1p protein, was <br /> expressed second copy wild-type (WT) strain. The cell extracts treated EndoH examined using SDS-PAGE. The lack of <br /> a mobility shift EndoH treatment N215Q N233Q double mutant indicates residues N glycosylated in <br /> the lumen secretory pathway. (e) A diagram showing inferred topology Cdc1p. <br />  <br />  <br />  <br />  <br /> medium Mn2&#226;&#171;&#185; sensitive divalent cation                EGTA sensitivity activated Ca2&#226;&#171;&#185; signaling. Fourteen <br /> chelator EGTA (28). Even EGTA specific                    different cdc1 mutants (33) tested ability grow <br /> Mn2&#226;&#171;&#185; chelator, effect cdc1 mutants thought                  presence 2 mM MnCl2 37&#194;&#176;C presence of <br /> be Mn2&#226;&#171;&#185; related EGTA sensitivity rescued                 2 mM EGTA 25&#194;&#176;C. These mutants showed wide range of <br /> by Mn2&#226;&#171;&#185; addition overexpression Mn2&#226;&#171;&#185; trans-                  sensitivities EGTA, rescued by <br /> porter Smf1p (28). Thus, Mn2&#226;&#171;&#185; rescue EGTA sensitivity                 Mn2&#226;&#171;&#185;. One mutant, cdc1-310, showed best rescue response <br /> are hallmarks link Mn2&#226;&#171;&#185; CDC1. We looked                Mn2&#226;&#171;&#185; severe sensitivity responses to <br /> for new cdc1 alleles did exhibit Mn2&#226;&#171;&#185; rescue                 EGTA (Fig. 1a). Conversely, cdc1-314 mutant not <br />  VOL. 28, 2008                                            cdc1 PHENOTYPES RESULT FROM ACTIVATING Ca2&#226;&#171;&#185; SIGNALING                                3339 <br />  <br />  <br />  <br />  <br />   FIG. 3. Mpk1p activated cdc1-314 mutant the <br /> major cause Ca2&#226;&#171;&#185; signaling activation. (a) An MPK1-lacZ reporter <br /> was used measure Mpk1p activity cdc1-314 mutant cells. Wild- <br /> type (WT) mutant cells grown SD-URA medium log <br /> phase shifted 33.5&#194;&#176;C indicated times. Reporter <br /> activity measured hour 3 h. (b) A CDRE-lacZ reporter <br /> was used measure activation Ca2&#226;&#171;&#185; signaling. Wild-type, cdc1- <br /> 314, mpk1&#226;&#338;&#172;, mpk1&#226;&#338;&#172; cdc1-314 cells grown SD-URA me- <br /> dium early log phase shifted 33.5&#194;&#176;C indicated times. <br /> Reporter activity measured hour 3 h. <br />  <br />                                                                            FIG. 4. Deletion Ca2&#226;&#171;&#185; channel Cch1p completely suppresses <br /> rescued Mn2&#226;&#171;&#185; exhibited mild growth defect               Golgi inheritance actin polarization phenotypes cdc1- <br /> the presence EGTA (Fig. 1a). Despite having different            314 mutant. (a) The inheritance late Golgi cisternae measured <br />                                                                          assessing presence late Golgi marker protein Sec7p-GFP <br /> responses Mn2&#226;&#171;&#185; EGTA, cdc1-314 cdc1-310                    buds wild-type (WT), cdc1-314, cch1&#226;&#338;&#172; cdc1-314, cch1&#226;&#338;&#172; <br /> strains similarly elevated Ca2&#226;&#171;&#185; signaling pathway activa-            cells shift 33.5&#194;&#176;C 3 h. (b) Actin visualized staining <br /> tion levels restrictive temperature measured            Alexa 594-phalloidin. Wild-type, cdc1-314, cch1&#226;&#338;&#172; cdc1-314, <br /> calcineurin-dependent response element-lacZ fusion reporter,             cch1&#226;&#338;&#172;, mpk1&#226;&#338;&#172;, mpk1&#226;&#338;&#172; cdc1-314 cultures grown 25&#194;&#176;C, <br /> CDRE-lacZ (38) (Fig. 1b). The assays performed                   shifted 33.5&#194;&#176;C 3 h, fixed, stained. Fluorescence images are <br />                                                                          shown. Most wild-type, cch1&#226;&#338;&#172; cdc1-314, cch1&#226;&#338;&#172;, mpk1&#226;&#338;&#172; cells <br /> 33.5&#194;&#176;C avoid heat shock-related responses. The results            contained polarized actin patches cables, the <br /> indicate connections Mn2&#226;&#171;&#185; Ca2&#226;&#171;&#185; sepa-              cdc1-314 mpk1&#226;&#338;&#172; cdc1-314 cells contained depolarized patches and <br /> rated favor hypothesis Cdc1p Mn2&#226;&#171;&#185;-depen-               visible cables. Scale bar, 5 &#226;?&#174;m. <br /> dent protein affects Ca2&#226;&#171;&#185; signaling. <br />    This conclusion supported sequence alignment <br /> data suggesting Cdc1p metal ion-dependent phos-               high Mn2&#226;&#171;&#185; levels partially overcome destabilization for <br /> phoesterase domain (34). We set test importance            alleles cdc1-310 alleles cdc1-314. To <br /> this domain function Cdc1p. The residues Asp95,               avoid complications Mn2&#226;&#171;&#185; sensitivity, used cdc1- <br /> Asp144, Asn183, His184, His259, His316 predicted              314 allele remainder study. <br /> chelate divalent cation active site, mutated               Cdc1p localization topology. What sub- <br /> these residues individually alanine (Fig. 2a). Whereas wild-          strate Cdc1p? To address question, examined the <br /> type CDC1 CEN plasmid rescued cdc1-314 mutant                localization myc-tagged Cdc1p. Surprisingly, Cdc1p-myc co- <br /> the restrictive temperature, plasmids carrying mutated ver-          localized ER marker Kar2p (32) (Fig. 2c). A hydro- <br /> sions CDC1 failed rescue (Fig. 2b). As control, growth           phobicity analysis revealed Cdc1p predicted <br /> was rescued plasmid carrying mutation His287,            transmembrane domains, implying Cdc1p integral <br /> is predicted chelate divalent cation. The wild-type           membrane protein ER. The phosphoesterase domain is <br /> and mutant Cdc1p proteins similar localizations (see be-             second hydrophobic domains (Fig. 2a). <br /> low) expression levels (see Fig. S1 supplemental              To determine phosphoesterase domain faces the <br /> material). These results confirm putative divalent cat-         cytosol ER lumen, took advantage fact that <br /> ion-chelating residues important Cdc1p function, sug-            N-linked glycosylation occurs ER lumen (1). If a <br /> gesting Cdc1p phosphoesterase.                          protein glycosylated, treating EndoH cleave off <br />    The known Cdc1p temperature-sensitive mutations                N-linked glycan (26) reduce molecular weight. To <br /> the putative phosphoesterase domain (33) (see Materials              determine potential N-linked glycosylation site is, fact, <br /> Methods). Presumably, mutations destabilize Cdc1p,             glycosylated, mutate site ask EndoH <br />  3340      LOSEV ET AL.                                                                                                             MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />    FIG. 5. Treatment cdc1-314 cells calcineurin inhibitor FK506 does suppress growth actin polarization phenotypes. (a) <br /> Activation Ca2&#226;&#171;&#185; signaling measured using CDRE-lacZ reporter. Wild-type (WT) cdc1-314 cells grown 10 ml SD-URA <br /> medium early log phase shifted 33.5&#194;&#176;C indicated times. Immediately shift, 10 &#226;?&#174;l dimethyl sulfoxide 1 mg/ml of <br /> FK506 dimethyl sulfoxide added. The final concentration FK506 cultures 1 &#226;?&#174;g/ml. Reporter activity measured hour for <br /> 3 h. (b) Cell growth measured taking absorbance 600 nm. Cultures treated described panel a. Readings taken every <br /> hour 3 h. (c) Actin visualized staining Alexa 594-phalloidin. Cultures treated described panel a. After 3 h, cells were <br /> fixed stained. Fluorescence images shown. Most treated untreated wild-type cells contained polarized actin patches cables, <br /> whereas treated untreated cdc1-314 cells contained depolarized patches. Scale bar, 5 &#226;?&#174;m. <br />  <br />  <br />  <br /> treatment resulting protein shifts molecular weight.              function cause cell stress activate Mpk1p, which <br /> Cdc1p potential N-linked glycosylation sites. When               activate Ca2&#226;&#171;&#185; signaling. This interpretation seemed <br /> simultaneously mutated Asn5, Asn434, Asn461 Gln                   reasonable proposed some <br /> then treated EndoH, saw shift molecular                  stress conditions, Mpk1p activates plasma membrane Ca2&#226;&#171;&#185; <br /> weight seen wild-type protein (Fig. 2d). When              channel Cch1p (4). To test Mpk1p activates Cch1p in <br /> we individually mutated Asn215 Asn233, located                  cdc1 mutant, deleted MPK1 monitored Ca2&#226;&#171;&#185; <br /> in phosphoesterase domain, observed smaller shift                   signaling activation using CDRE-lacZ reporter. Surpris- <br /> (Fig. 2d). When Asn215 Asn233 simultaneously mu-                    ingly, deletion MPK1 caused small decrease cdc1- <br /> tated, molecular weight shift disappeared completely,                induced Ca2&#226;&#171;&#185; signaling activation (Fig. 3b), implying in <br /> the EndoH-treated -untreated mutant proteins                     cdc1 cells, Cch1p activated primarily Mpk1p-indepen- <br /> same mobility EndoH-treated wild-type protein (Fig.                   dent pathway. These results argue hypothesis that <br /> 2d). These results indicate phosphoesterase do-                generalized cell stress response explain Ca2&#226;&#171;&#185; signal- <br /> main residues Asn215 Asn233 glycosylated, implying                   ing activation cdc1 mutant cells. <br /> that phosphoesterase domain faces lumen se-                      Relationship Ca2&#226;&#180;&#8482; signaling activation cdc1 mu- <br /> cretory pathway. Our topology assignment supported                     tant phenotypes. Because cdc1 mutations activate Ca2&#226;&#171;&#185; <br /> previous data showing C terminus Cdc1p lume-                  signaling Mpk1p, wanted know path- <br /> nally oriented (20). Together, findings demonstrate               ways mediates additional cdc1 phenotypes defective <br /> the phosphoesterase domain Cdc1p faces ER lumen                       Golgi inheritance depolarization actin. For pur- <br /> (Fig. 2e).                                                                   pose, deleted Ca2&#226;&#171;&#185; channel Cch1p performed a <br />    Mpk1p activation cdc1 cells. Our data suggest inhib-              Golgi inheritance assay. The results showed loss Cch1p <br /> iting enzymatic function Cdc1p activates Ca2&#226;&#171;&#185; signaling.              completely suppressed Golgi inheritance actin <br /> One possibility inhibition Cdc1p triggers cell stress           polarization phenotypes (Fig. 4a b). By contrast, we <br /> response. This idea fits actin depolarization seen               deleted MPK1 cdc1 mutant, actin depolarized <br /> cdc1 mutants (33) actin depolarized cells                 (Fig. 4b). We unable test effect MPK1 deletion <br /> experiencing stress (23). We tested cdc1 mutants                Golgi inheritance small-budded cells in <br /> experiencing cell stress examining activity Mpk1p,               mpk1&#226;&#338;&#172; mutant lysed. Nevertheless, results <br /> key component cell stress signal transduction pathway                 clearly indicate cdc1 mutants, Cch1p-dependent activa- <br /> (18). These experiments employed MPK1-lacZ reporter (18,                  tion Ca2&#226;&#171;&#185; signaling leads defective Golgi inheritance and <br /> 23). The results showed Mpk1p activated cdc1                    depolarization actin. <br /> mutant restrictive temperature, consistent cell                   In test importance Ca2&#226;&#171;&#185; signaling activa- <br /> stress response (Fig. 3a).                                                   tion cdc1 mutant phenotypes, introduced per1&#226;&#338;&#172; <br />    The results presented far suggested loss Cdc1p               mutation, strongly suppresses growth defect cdc1 <br />  VOL. 28, 2008                                                cdc1 PHENOTYPES RESULT FROM ACTIVATING Ca2&#226;&#171;&#185; SIGNALING                                   3341 <br />  <br />  <br />  <br />  <br />   FIG. 6. A phospholipid species accumulates cdc1-314 mutant restrictive temperature. (a) Wild-type (WT) mutant strains were <br /> grown overnight YPD medium supplemented 32Pi. Upon reaching log phase, cells shifted 33.5&#194;&#176;C 3 h. Lipids extracted <br /> and run dimensions TLC plates. These images representative plates area major radioactive lipid species <br /> migrated. Arrows mark unidentified phosphorylated lipid consistently present higher levels cdc1 cells wild-type cells. (b) <br /> The level unidentified radioactive lipid measured indicated strains fraction total radioactive lipid, results <br /> were normalized defining signal wild-type cells 1.0. <br />  <br />  <br />  <br /> strains (27). As expected, per1&#226;&#338;&#172; cdc1 double mutant showed                       Accumulation phospholipid cdc1 cells. To understand <br /> no Ca2&#226;&#171;&#185; signaling activation restrictive temperature (see                Cdc1p affects Ca2&#226;&#171;&#185; signaling, important clarify the <br /> Fig. S2 supplemental material).                                          enzymatic function Cdc1p. The topology assignment con- <br />    One effector Cch1p-dependent Ca2&#226;&#171;&#185; influx Ca2&#226;&#171;&#185;/                     strains possible substrates phosphoesterase domain. <br /> calmodulin-dependent phosphatase calcineurin (9, 29). By ac-                    Proteins unlikely substrates because, knowledge, <br /> tivating transcription factor Crz1p, calcineurin turns                   protein kinases phosphatases ER lumen. <br /> many genes (9). We tested growth actin po-                      However, lipid phosphatases present com- <br /> larization phenotypes cdc1 cells caused calcineurin                   partment (11, 19). We hypothesized phosphor- <br /> activation. This experiment employed potent calcineurin                     ylated lipid substrate Cdc1p. To test idea, <br /> inhibitor FK506 (9). When cdc1 cells incubated 33.5&#194;&#176;C,                  radiolabeled wild-type cdc1 cells 32Pi, extracted <br /> the addition FK506 did rescue growth defect, did                 radioactive lipids, performed two-dimensional TLC <br /> it prevent actin depolarization (Fig. 5b c). As control,                  analysis. A phospholipid species reproducibly accumulated in <br /> FK506 added cdc1 cells, completely suppressed                   cdc1 mutant restrictive temperature (Fig. 6a). Quan- <br /> the elevation CDRE-lacZ activity (Fig. 5a). Thus, growth                 titation indicated 3 h 33.5&#194;&#176;C, unidentified lipid <br /> and actin polarization phenotypes seen cdc1 mutants                  approximately twice concentrated cdc1 cells in <br /> due activation calcineurin mediated                  wild-type cells (Fig. 6b). <br /> separate Ca2&#226;&#171;&#185;-dependent pathway.                                                   We postulate cdc1 mutants, abnormal accumulation <br />  3342     LOSEV ET AL.                                                                                                     MOL. CELL. BIOL. <br />  <br />  <br /> of lipid leads hyperactivated Cch1p-dependent Ca2&#226;&#171;&#185; sig- <br /> naling, turn causes toxicity. This model predicts a <br /> cdc1 strain carrying cch1&#226;&#338;&#172; suppressor mutation still <br /> accumulate unidentified lipid. Indeed, cch1&#226;&#338;&#172; cdc1 <br /> double mutant, unidentified lipid twice con- <br /> centrated cch1&#226;&#338;&#172; single mutant (Fig. 6b). Interestingly, <br /> in per1&#226;&#338;&#172; strain unidentified lipid twice as <br /> concentrated wild-type cells, concentration was <br /> not enhanced per1&#226;&#338;&#172; cdc1 double mutant (Fig. 6b). It <br /> therefore likely per1&#226;&#338;&#172; cch1&#226;&#338;&#172; mutations <br /> suppress cdc1 phenotypes different mechanisms. <br />  <br />                         DISCUSSION <br />    Our findings add evidence Cdc1p Mn2&#226;&#171;&#185;- <br /> dependent protein. Residues predicted chelate divalent <br /> metal ion active site crucial Cdc1p function. This <br />                                                                        FIG. 7. A model Cdc1p function link Cdc1p <br /> interpretation explains genes affect Mn2&#226;&#171;&#185; levels      Golgi inheritance. (a) Cdc1p presumed lipid phos- <br /> genetic interactions CDC1. For example, ability         phatase, suggest Per1p needed generate substrate <br /> CDC1 overexpression rescue EGTA sensitivity            Cdc1p. According model, inactivating Cdc1p causes its <br /> smf1&#226;&#338;&#172; strain explained Cdc1p Mn2&#226;&#171;&#185;-            substrate accumulate toxic levels causes accumulation of <br />                                                                     nontoxic precursor lipid. When Per1p inactivated, non- <br /> sensitive essential protein ER loss Smf1p <br />                                                                     toxic precursor accumulates, per1&#226;&#338;&#172; mutation suppresses le- <br /> would inhibit Cdc1p reducing Mn2&#226;&#171;&#185; concentration           thality cdc1 mutations. This nontoxic precursor correspond to <br /> the ER. However, analysis complicated effect       unidentified lipid visualized experiment shown in <br /> Cdc1p Ca2&#226;&#171;&#185; signaling. In cases genetic interactors      Fig. 6. (b) In cdc1 cells, accumulation toxic phosphorylated lipid <br /> of CDC1 influence Ca2&#226;&#171;&#185; Mn2&#226;&#171;&#185;. For example,              activates Mpk1p- Cch1p-dependent Ca2&#226;&#171;&#185; signaling. The <br />                                                                     Ca2&#226;&#171;&#185; signaling activates calcineurin. In calcineurin-independent <br /> genetic interaction Golgi body-localized transporter       pathway, strong Ca2&#226;&#171;&#185; signaling depolarizes actin. We speculate that <br /> Pmr1p reflect ability Pmr1p transport          strong Ca2&#226;&#171;&#185; signaling inhibits Golgi tethering. These effects of <br /> Ca2&#226;&#171;&#185; Mn2&#226;&#171;&#185; (22). Here, cdc1 allele does       strong Ca2&#226;&#171;&#185; signaling synergize cause pronounced defect Golgi <br /> not cause enhanced sensitivity Mn2&#226;&#171;&#185; levels exhibits    inheritance. <br /> Ca2&#226;&#171;&#185; accumulation, indicating Ca2&#226;&#171;&#185;- Mn2&#226;&#171;&#185;-re- <br /> lated phenotypes uncoupled. <br />    We propose Cdc1p Mn2&#226;&#171;&#185;-dependent ER-localized           strong cdc1 suppressor GUP1 (Stephen Garrett, personal <br /> phosphatase, active site facing lumen. The sub-        communication), identified independently using an <br /> strate Cdc1p remains identified. An ER-localized          insertional mutagenesis screen suppression cdc1 temper- <br /> phospholipid attractive candidate phosphory-          ature sensitivity (data shown). Like Per1p, Gup1p a <br /> lated lipids dolichol phosphate sphingosine phos-       transmembrane ER protein (16, 17). Gup1p reportedly an <br /> phate present ER lumen (11, 19). The enzymes        enzyme adds C26 fatty acids GPI anchor (5), a <br /> dephosphorylate lipids&#226;&#8364;&#8221;Lcb3p Cwh8p, respec-               gup1&#226;&#338;&#172; mutation comparable per1&#226;&#338;&#172; mutation ability <br /> tively&#226;&#8364;&#8221;resemble Cdc1p multispanning ER membrane            suppress cdc1 temperature sensitivity phenotype (see <br /> proteins phosphatase domains facing lumen (11, 19).        Fig. S3 supplemental material). These observations im- <br /> Our preliminary experiments dolichol sphingosine       ply putative lipid phosphatase activity Cdc1p is <br /> metabolic pathways did yield clear link Cdc1p.           linked manner GPI remodeling. <br /> However, TLC analysis revealed cdc1 cells accumulate           The results presented allow formulate model of <br /> least unidentified lipid. The substrate Cdc1p      cdc1 mutations affect Golgi inheritance. In cdc1 cells, <br /> a lipid detected yeast. A specu-      accumulation toxic phosphorylated lipid activates Cch1p- <br /> lative candidate ceramide phosphate, present         dependent Ca2&#226;&#171;&#185; signaling. One possible activation mechanism <br /> animal cells described yeast cells (7).         accumulated lipid leaves ER directly stim- <br />    Ongoing work groups shed light             ulates Cch1p plasma membrane. Alternatively, ac- <br /> biochemical activity Cdc1p. One strongest genetic         cumulated lipid act locally promote Ca2&#226;&#171;&#185; release <br /> interactors CDC1 PER1 (27). Deletion PER1 able       ER, depleting ER-associated Ca2&#226;&#171;&#185; trig- <br /> suppress CDC1 deletion (27; unpublished results). PER1       gering Ca2&#226;&#171;&#185; influx Cch1p. In case, Ca2&#226;&#171;&#185; signaling <br /> is involved processing proteins glycosylphos-      activation leads defect actin polarization. We speculate <br /> phatidylinositol (GPI) anchor (Davis Ng, personal communi-          Ca2&#226;&#171;&#185; signaling activation inhibits Golgi tethering. <br /> cation), recent paper showed PER1 required        Together, effects Ca2&#226;&#171;&#185; signaling disrupt in- <br /> GPI-phospholipase A2 activity (12). We suggest PER1         heritance late Golgi cisternae (Fig. 7b). <br /> needed generate substrate Cdc1p (Fig. 7a). Thus,            It entirely clear screen Golgi inheri- <br /> inactivating Cdc1p leads accumulation toxic phosphor-       tance defects uniquely repeatedly identified cdc1 mutants <br /> ylated lipid, simultaneous deletion PER1 blocks activity     activation Ca2&#226;&#171;&#185; signaling means as <br /> earlier pathway prevents toxicity. Another     hyperosmotic stress (23) did cause Golgi inheritance de- <br />  VOL. 28, 2008                                                      cdc1 PHENOTYPES RESULT FROM ACTIVATING Ca2&#226;&#171;&#185; SIGNALING                                         3343 <br />  <br />  <br /> fects (data shown). We propose cdc1 mutants,                         17. Huh, W. K., J. V. Falvo, L. C. Gerke, A. S. Carroll, R. W. Howson, J. S. <br />                                                                                          Weissman, E. K. O&#226;&#8364;&#8482;Shea. 2003. Global analysis protein localization in <br /> Ca2&#226;&#171;&#185; signaling activation exceptionally strong                          budding yeast. Nature 425:686&#226;&#8364;&#8220;691. <br /> results multiple downstream defects (Fig. 7b).                                    18. Jung, U. S., A. K. Sobering, M. J. Romeo, D. E. Levin. 2002. Regulation <br />                                                                                          yeast Rlm1 transcription factor Mpk1 cell wall integrity MAP <br />                           ACKNOWLEDGMENTS                                                kinase. Mol. Microbiol. 46:781&#226;&#8364;&#8220;789. <br />                                                                                      19. Kihara, A., T. Sano, S. Iwaki, Y. Igarashi. 2003. Transmembrane topol- <br />    We thank Stephen Garrett, Davis Ng, Kyle Cunningham, mem-                         ogy sphingoid long-chain <span id='am-238' about='xsp:base' typeof='owl:Thing'>base</span>-1-phosphate phosphatase, Lcb3p. Genes <br /> bers Glick lab helpful discussion. We grateful Mark                    Cells 8:525&#226;&#8364;&#8220;535. <br /> Rose, David Levin, Martha Cyert providing reagents.                          20. Kim, H., K. Melen, G. von Heijne. 2003. Topology models 37 <br />    This work supported NIH grant GM-61156 (B.S.G.) NIH                        Saccharomyces cerevisiae membrane proteins based C-terminal reporter <br />                                                                                          fusions predictions. J. Biol. Chem. 278:10208&#226;&#8364;&#8220;10213. <br /> training grant 5-20942 (E.L.).                                                       21. Kunz, J., U. Schneider, M. Deuter-Reinhard, N. R. Movva, M. N. Hall. <br />                                                                                          1993. Target rapamycin yeast, TOR2, essential phosphatidylino- <br />                                  REFERENCES <br />                                                                                          sitol kinase homolog required G1 progression. Cell 73:585&#226;&#8364;&#8220;596. <br />  1. Abeijon, C., C. B. Hirschberg. 1992. Topography glycosylation reac-       22. Lapinskas, P. J., K. Cunningham, X. F. Liu, G. R. Fink, V. C. Culotta. <br />     tions endoplasmic reticulum. Trends Biochem. Sci. 17:32&#226;&#8364;&#8220;36.                   1995. Mutations PMR1 suppress oxidative damage yeast cells lacking <br />  2. Ayscough, K. R., J. Stryker, N. Pokala, M. Sanders, P. Crews, D. G.              superoxide dismutase. Mol. Cell. Biol. 15:1382&#226;&#8364;&#8220;1388. <br />     Drubin. 1997. High rates actin filament turnover budding yeast         23. Levin, D. E. 2005. Cell wall integrity signaling Saccharomyces cerevisiae. <br />     roles actin establishment maintenance cell polarity revealed           Microbiol. Mol. Biol. Rev. 69:262&#226;&#8364;&#8220;291. <br />     using actin inhibitor latrunculin-A. J. Cell Biol. 137:399&#226;&#8364;&#8220;416.              24. Lillie, S. H., S. S. Brown. 1994. Immunofluorescence localization the <br />  3. Baudin, A., O. Ozier-Kalogeropoulos, A. Denouel, F. Lacroute, C. Cullin.         unconventional myosin, Myo2p, putative kinesin-related protein, <br />     1993. A simple efficient method direct gene deletion Saccharomyces        Smy1p, regions polarized growth Saccharomyces cerevisiae. <br />     cerevisiae. Nucleic Acids Res. 21:3329&#226;&#8364;&#8220;3330.                                         J. Cell Biol. 125:825&#226;&#8364;&#8220;842. <br />  4. Bonilla, M., K. Cunningham. 2003. Mitogen-activated protein kinase           25. Loukin, S., C. Kung. 1995. Manganese effectively supports yeast cell- <br />     stimulation Ca2&#226;&#171;&#185; signaling required <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-6' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span>survival</span> endoplasmic retic-         cycle progression place calcium. J. Cell Biol. 131:1025&#226;&#8364;&#8220;1037. <br />     ulum stress yeast. Mol. Biol. Cell 14:4296&#226;&#8364;&#8220;4305.                              26. Maley, F., R. B. Trimble, A. L. Tarentino, T. H. J. Plummer. 1989. <br />  5. Bosson, R., M. Jaquenoud, A. Conzelmann. 2006. GUP1 Saccharo-                 Characterization glycoproteins associated oligosaccharides <br />     myces cerevisiae encodes O-acyltransferase involved remodeling          use endoglycosidases. Anal. Biochem. 180:195&#226;&#8364;&#8220;204. <br />     GPI anchor. Mol. Biol. Cell 17:2636&#226;&#8364;&#8220;2645.                                        27. Paidhungat, M., S. Garrett. 1998. Cdc1 vacuole coordinately <br />  6. Byers, B., L. Goetsch. 1974. Duplication spindle plaques integra-         regulate Mn2&#226;&#171;&#185; homeostasis yeast Saccharomyces cerevisiae. Genetics <br />     tion yeast cell cycle. Cold Spring Harbor Symp. Quant. Biol. 38:123&#226;&#8364;&#8220;          148:1787&#226;&#8364;&#8220;1798. <br />     131. <br />                                                                                      28. Paidhungat, M., S. Garrett. 1998. Cdc1 required growth Mn2&#226;&#171;&#185; <br />  7. Chalfant, C. E., S. Spiegel. 2005. Sphingosine 1-phosphate ceramide <br />                                                                                          regulation Saccharomyces cerevisiae. Genetics 148:1777&#226;&#8364;&#8220;1786. <br />     1-phosphate: expanding roles cell signaling. J. Cell Sci. 118:4605&#226;&#8364;&#8220;4612. <br />                                                                                      29. Paidhungat, M., S. Garrett. 1997. A homolog mammalian, voltage- <br />  8. Cook, R. K., W. T. Blake, P. A. Rubenstein. 1992. Removal the <br />                                                                                          gated calcium channels mediates yeast pheromone-stimulated Ca2&#226;&#171;&#185; uptake <br />     amino-terminal acidic residues yeast actin. Studies vitro vivo. <br />                                                                                          exacerbates cdc1(Ts) growth defect. Mol. Cell. Biol. 17:6339&#226;&#8364;&#8220;6347. <br />     J. Biol. Chem. 267:9430&#226;&#8364;&#8220;9436. <br />  9. Cyert, M. S. 2003. Calcineurin signaling Saccharomyces cerevisiae:        30. Reid, B., L. H. Hartwell. 1977. Regulation mating cell cycle of <br />     yeast crazy response stress. Biochem. Biophys. Res. Commun.                 Saccharomyces cerevisiae. J. Cell Biol. 75:355&#226;&#8364;&#8220;365. <br />     311:1143&#226;&#8364;&#8220;1150.                                                                   31. Reinke, C. A., P. Kozik, B. S. Glick. 2004. Golgi inheritance Saccha- <br /> 10. Eguez, L., Y. S. Chung, A. Kuchibhatla, M. Paidhungat, S. Garrett.               romyces cerevisiae depends ER inheritance. Proc. Natl. Acad. Sci. USA <br />     2004. Yeast Mn2&#226;&#171;&#185; transporter, Smf1p, regulated ubiquitin-dependent             101:18018&#226;&#8364;&#8220;18023. <br />     vacuolar protein sorting. Genetics 167:107&#226;&#8364;&#8220;117.                                  32. Rose, M. D., L. M. Misra, J. P. Vogel. 1989. KAR2, karyogamy gene, <br /> 11. Fernandez, F., J. S. Rush, D. A. Toke, G. S. Han, J. E. Quinn, G. M. Carman,         yeast homolog mammalian BiP/GRP78 gene. Cell 57:1211&#226;&#8364;&#8220;1221. <br />     J. Y. Choi, D. R. Voelker, M. Aebi, C. J. Waechter. 2001. The CWH8           33. Rossanese, O. W., C. A. Reinke, B. J. Bevis, A. T. Hammond, I. B. Sears, J. <br />     gene encodes dolichyl pyrophosphate phosphata <br /> </body></html>