<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br />                            Genetic analysis of the requirements for SOS induction by <br />                            nalidixic acid in Escherichia coli <br />  <br />  <br />                            <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />                                 Nalidixic acid, the prototype antibacterial quinolone, induces the SOS response by a mechanism that <br />                                 requires the RecBCD nuclease/helicase. A key step inferred for this induction pathway is the <br />                                 conversion of a drug-induced gyrase cleavage complex into a DNA break that can be processed by <br />                                 RecBC. We tried <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> clarify the nature of this step by searching for additional gene products that are <br />                                 specifically necessary for SOS induction following nalidixic acid treatment. A transposon library of <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                 approximately 19,000 insertion mutants yielded 18 mutants that were substantially reduced for SOS <br />                                 induction following nalidixic acid but not UV treatment, and which were also hypersensitive to <br />                                 nalidixic acid. All 18 mutants turned out to have insertions in recB or recC. As expected, recA <br />                                 insertion mutants were uncovered as being uninducible by either nalidixic acid or UV treatment. <br />                                 Insertions in 11 other genes were found to cause partial defects in SOS induction by one or both <br />                                 pathways, providing possible leads in understanding the detailed mechanisms of SOS induction. <br />                                 Overall, these results suggest that nalidixic acid-induced DNA breaks are generated either by RecBC <br />                                 itself, by redundant activities, and/or by an essential protein that could not be uncovered with <br />                                 transposon mutagenesis. <br />  <br />  <br />                            Keywords <br />                                 DNA gyrase; RecBCD; DNA damage; quinolones; transposon mutagenesis <br />  <br />  <br />                            1. Introduction <br />                                               Type II DNA topoisomerases are essential enzymes involved in numerous cellular processes, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               including DNA replication, DNA segregation and transcription. These enzymes catalyze <br />                                               changes in DNA topology by creating a transient double strand break in one segment of duplex <br />                                               DNA and passing a second duplex segment through that break. Many clinically important <br />                                               antibacterial and antitumor agents target type II topoisomerases by altering the equilibrium of <br />                                               the reaction cycle to stabilize the normally transient intermediate called the cleavage complex <br />                                               (Drlica and Zhao, 1997; Burden and Osheroff, 1998). These include the antibacterial <br />                                               quinolones (e.g. nalidixic acid) that target DNA gyrase (and topoisomerase IV in certain <br />                                               bacteria), and antitumor drugs such as mitoxantrone, adriamycin and etoposide, which target <br />                                               mammalian type II topoisomerase. In each case, the drug-stabilized cleavage complex consists <br />                                               of the topoisomerase linked via phosphotyrosine bonds to both 5&apos; ends of the staggered double- <br />  <br />  <br />                            * Corresponding author: Kenneth N. Kreuzer, Email: kenneth.kreuzer@duke.edu, Phone: 919 684 6466, Fax: 919 684 2790. <br />                            1Present address: bioM rieux, 100 Rodolphe St., Durham, NC 27712 <br />                            Publisher&apos;s Disclaimer: This is a PDF file of an unedited manuscript that has been accepted for publication. As a service to our customers <br />                            we are providing this early version of the manuscript. The manuscript will undergo copyediting, typesetting, and review of the resulting <br />                            proof before it is published in its final citable form. Please note that during the production process errors may be discovered which could <br />                            affect the content, and all legal disclaimers that apply to the journal pertain. <br />                             Newmark et al.                                                                                               Page 2 <br />  <br />  <br />                                             strand break. Results <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> numerous systems demonstrate that the cytotoxicity of these <br />                                             inhibitors is dependent on formation of cleavage complexes rather than inhibition of <br />                                             topoisomerase catalytic activity (for reviews, see Chen and Liu, 1994; Drlica and Zhao, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             1997). <br />  <br />                                             Although much research has been devoted to understanding how stabilized cleavage complexes <br />                                             lead to cell death, the detailed mechanism(s) remains unclear. Formation of cleavage <br />                                             complexes is necessary but not sufficient for cytotoxicity. The 5&apos; ends of the broken DNA <br />                                             within the cleavage complex are safely linked to protein, and cleavage complexes are readily <br />                                             reversible upon drug removal both in vivo and in vitro (Hsiang and Liu, 1989; also see Drlica <br />                                             and Zhao, 1997). The energy poison dinitrophenol blocks cytotoxicity of mammalian type II <br />                                             topoisomerase inhibitors but does not prevent cleavage complex formation (Kupfer et al., <br />                                             1987). Likewise, dinitrophenol (as well as chloramphenicol) protects Escherichia coli from <br />                                             the cytotoxic action of nalidixic acid (Cook et al., 1966). Evidently, a cellular processing event <br />                                             must occur to convert a subset of the cleavage complexes into cytotoxic lesions. What is this <br />                                             event, and what is the exact nature of the cytotoxic lesion? <br />  <br />                                             A number of observations strongly suggest that the cytotoxic lesion is some type of double- <br />                                             strand break (for review, see Chen and Liu, 1994; Drlica and Zhao, 1997). For example, <br />                                             mutational inactivation of functionally conserved recombination proteins in phage, bacterial <br />                                             and eukaryotic systems leads to drug hypersensitivity. The gene products required for the repair <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             of topoisomerase-mediated DNA damage are similar or identical to those required for the repair <br />                                             of endonuclease-generated double-strand breaks. Furthermore, there is evidence for overt <br />                                             chromosomal breaks after nalidixic acid treatment, but the location of the breaks and the <br />                                             possibility of covalently attached protein were not determined (Chen et al., 1996). <br />  <br />                                             A variety of models could in principle explain the relationship between drug-stabilized <br />                                             cleavage complexes and cytotoxicity. First, a nuclease such as SbcCD might directly recognize <br />                                             the covalent protein-DNA complex and cleave the DNA nearby (Connelly et al, 2003). Second, <br />                                             the replication complex or associated helicase may be able to extract the lagging strand template <br />                                             from the cleavage complex upon collision (Howard et al., 1994). Third, DNA breaks might <br />                                             result as &#226;&#8364;&#339;collateral damage&#226;&#8364;&#65533; from recombination nucleases that act after replication fork <br />                                             blockage by the cleavage complex, as in the phage T4 system (Hong and Kreuzer 2003). We <br />                                             have recently found that quinolone-stabilized gyrase cleavage complexes block E. coli <br />                                             replication forks on plasmid pBR322 in vivo (Pohlhaus and Kreuzer, 2005). In that study, some <br />                                             of the blocked forks were broken, consistent with the collateral damage model. <br />  <br />                                             To approach the mechanism of cytotoxicity, we have taken advantage of an SOS reporter <br />                                             system and screened for E. coli mutants specifically deficient in SOS induction upon nalidixic <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             acid treatment. The primary target of nalidixic acid in E. coli is DNA gyrase (see Maxwell and <br />                                             Critchlow, 1998), and this drug was one of the first inducers of the SOS regulon studied in <br />                                             detail. The SOS regulon consists of about 30 different genes, many of which are involved in <br />                                             damage repair or bypass reactions (Friedberg et al., 1995). The LexA protein normally <br />                                             represses SOS genes, but is cleaved to trigger SOS as a result of DNA damage. Cleavage of <br />                                             LexA depends on the activated form of the RecA protein, bound to single-stranded DNA. <br />  <br />                                             DNA gyrase cleavage complexes stabilized by nalidixic acid are necessary but not sufficient <br />                                             for induction of the SOS response. There is conflicting evidence as to whether DNA replication <br />                                             is required for induction of SOS by nalidixic acid (Gudas and Pardee, 1976; Sassanfar and <br />                                             Roberts, 1990). Aside from RecA, the only known protein that is required for SOS induction <br />                                             by nalidixic acid is the multifunctional RecBCD enzyme (for review, see Myers and Stahl, <br />                                             1994). Since RecBCD generally requires a DNA end to gain entry to DNA, its involvement in <br />  <br />  <br />  <br />                                                    Gene. Author manuscript; available in PMC 2008 January 15. <br />                             Newmark et al.                                                                                             Page 3 <br />  <br />  <br />                                             SOS induction suggests that a free DNA break is somehow generated from the nalidixic acid- <br />                                             stabilized cleavage complex. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             The above results provide strong parallels between the mechanism of cytotoxicity and the <br />                                             mechanism of SOS induction by nalidixic acid, particularly since recA, recB and recC mutants <br />                                             are all hypersensitive to the drug (McDaniel et al., 1978). It seems highly likely that whatever <br />                                             mechanism creates DNA breaks to induce the SOS pathway is also a mechanism that leads to <br />                                             cytotoxicity. In this report, we identify and analyze E. coli mutants with deficiencies in the <br />                                             SOS response to nalidixic acid. <br />  <br />                               2. Materials and methods <br />                               2.1. Materials and media <br />                                             Kanamycin and nalidixic acid were purchased from Sigma, while 5-bromo-4-chloro-3-indolyl- <br />                                             &#206;&#178;-D-galactopyranoside (X-gal) was purchased from Gold Biotechnology Inc. L-broth (LB) <br />                                             contained Bacto-Tryptone (10 g/L), yeast extract (5 g/L) and sodium chloride (10 g/L) while <br />                                             LB plates had the same composition plus Bacto-agar (14 g/L). Top agar contained Bacto- <br />                                             Tryptone (13 g/L), sodium chloride (8 g/L), sodium citrate dihydrate (2 g/L), glucose (5 g/L) <br />                                             and Bacto-agar (6.5 g/L). <br />  <br />                               2.2. Transposon mutagenesis and primary nalidixic acid screening <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             A transposon library was created in E. coli strain JH39 [F- sfiA11, thr-1, leu-6, hisG4, argE3, <br />                                             ilv(Ts), galK2, srl(?) rpsL31, lac U169, dinD1::MudI(Apr lac); Heitman and Model, 1991; <br />                                             O&#226;&#8364;&#8482;Reilly and Kreuzer, 2004] using the EZ::TN&#226;&#8222;&#162; &lt;KAN-2&gt; Tnp Transposome&#226;&#8222;&#162; Kit from <br />                                             Epicentre (Madison). Electrocompetent cells were pulsed at 1.8 kV in 0.1-cm cuvettes, <br />                                             followed by outgrowth in TB for one hour at 37&#194;&#176;C with aeration. The cells were then plated at <br />                                             appropriate dilutions on LB plates containing kanamycin (60 &#206;&#188;g/ml). Following overnight <br />                                             incubation at 37&#194;&#176;C, the colonies were lifted using 85-mm circular nitrocellulose filters <br />                                             (Schleicher and Schuell) and placed, colony-side up, onto LB plates containing nalidixic acid <br />                                             (50 &#206;&#188;g/ml) and X-gal (60 &#206;&#188;g/ml). The lifted colonies were incubated overnight at 37&#194;&#176;C and the <br />                                             plates were checked for color development at approximately 4 and 12 hours. Colonies that <br />                                             turned darker blue than the wild-type were marked as &#226;&#8364;&#339;dark blue&#226;&#8364;&#65533; while those that were white <br />                                             or lighter than wild-type were marked as &#226;&#8364;&#339;white&#226;&#8364;&#65533; and &#226;&#8364;&#339;intermediate&#226;&#8364;&#65533;, respectively. <br />  <br />                               2.3. DNA techniques and sequencing <br />                                             Genomic DNA was purified using the MasterPure&#226;&#8222;&#162; DNA Purification Kit of Epicentre <br />                                             (Madison) essentially as described by the manufacturer (a phenol-chloroform extraction <br />                                             followed by an ethanol precipitation was added after the RNase treatment). The final purified <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             DNA pellet was resuspended in 30 &#206;&#188;l of T5E1 buffer (5 mM Tris-HCl, pH 7.8; 1 mM EDTA). <br />                                             The DNA samples were sequenced by the Duke University Cancer Center DNA Analysis <br />                                             Facility using a modification of the automated sequencing protocol (60 cycles, 60oC annealing <br />                                             temperature). The sequencing primer (10 pmol/&#206;&#188;l) was KAN-2 FP-1 (5&#226;&#8364;&#178;&#226;&#8364;&#8220; <br />                                             ACCTACAACAAAGCTCTCATCAACC&#226;&#8364;&#8220;3&#226;&#8364;&#178;), the forward primer from the EZ::TN&#226;&#8222;&#162; <br />                                             &lt;KAN-2&gt; Tnp Transposome&#226;&#8222;&#162; Kit. <br />  <br />                                             PCR-amplification of the recB gene was performed with primers recB-FP (5&#226;&#8364;&#178;&#226;&#8364;&#8220; <br />                                             CGGCAGGATGTTCATCTCCCGCC&#226;&#8364;&#8220;3&#226;&#8364;&#178;) and recB-RP (5&#226;&#8364;&#178;&#226;&#8364;&#8220; <br />                                             TCTTGTGCTGCACGAGTCAGCC&#226;&#8364;&#8220;3&#226;&#8364;&#178;). <br />  <br />                               2.4. Phage P1 transductions <br />                                             Transductions were performed by the method of Silhavy et al. (1984), selecting for the <br />                                             kanamycin-resistance gene of the transposon. <br />  <br />                                                    Gene. Author manuscript; available in PMC 2008 January 15. <br />                             Newmark et al.                                                                                               Page 4 <br />  <br />  <br />                               3. Results <br />                               3.1. Isolation of transposon mutants with reduced SOS induction in response to nalidixic <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               acid <br />                                             As described in the Introduction, DNA breaks are very likely generated from nalidixic acid- <br />                                             stabilized cleavage complexes as a requisite step in SOS induction. To search for genes that <br />                                             may be necessary for this step, we used the EZ::TN&#226;&#8222;&#162; &lt;KAN-2&gt; Tnp Transposome&#226;&#8222;&#162; Kit <br />                                             (Epicentre) to create a library of transposon mutants in the JH39 strain of E. coli. This strain <br />                                             has a dinD1::lacZ fusion that provides a convenient reporter for the SOS response, allowing <br />                                             non-inducible mutants to be identified by a simple color screen (Heitman and Model, 1991). <br />                                             The strain also has an sfiA (sulA) <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span>mutation</span>, which reduces filamentation and inhibition of cell <br />                                             division upon SOS induction (Friedberg et al., 1995). <br />  <br />                                             The transposon-protein complexes were transformed into JH39 by electroporation, and <br />                                             approximately 19,000 colonies with random transposon insertions were selected based on <br />                                             kanamycin resistance. The colonies were lifted onto nitrocellulose filters, which were then <br />                                             transferred to plates containing nalidixic acid and X-gal. After 4-hour and overnight incubation, <br />                                             a small fraction of the colonies displayed unusual color intensities, indicating different amounts <br />                                             of &#206;&#178;-galactosidase induction. We collected 25 &#226;&#8364;&#339;white mutants&#226;&#8364;&#65533; with no apparent induction and <br />                                             105 &#226;&#8364;&#339;intermediate mutants&#226;&#8364;&#65533; with reduced induction relative to JH39. We also isolated 66 &#226;&#8364;&#339;dark <br />                                             blue mutants&#226;&#8364;&#65533; with increased induction relative to JH39; these are detailed elsewhere (O&apos;Reilly <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             and Kreuzer, 2004). <br />  <br />                               3.2. Secondary screens for induction with nalidixic acid or UV <br />                                             We began analyzing the white and intermediate mutants, which have apparent defects in SOS <br />                                             induction after nalidixic acid. As a secondary screen for SOS induction we performed a disk <br />                                             test in which a lawn of each mutant was exposed to three levels of nalidixic acid during growth <br />                                             on X-gal plates (Figure 1A). The zones of inhibition were also measured to allow for a <br />                                             determination of drug sensitivity (Table 1). All of the 25 white mutants showed normal-sized <br />                                             zones of inhibition and again displayed no blue color in the presence of the drug (e.g., compare <br />                                             W16 to wild-type JH39 in Figure 1A and Table 1). Sixty of the intermediate mutants looked <br />                                             indistinguishable from the parental JH39 in this test, both for nalidixic acid sensitivity and the <br />                                             intensity of &#206;&#178;-galactosidase induction. These were therefore judged to be false positives and <br />                                             were discarded. The remaining 45 intermediate mutants again showed reduced induction <br />                                             compared to JH39, with 20 displaying noticeably less color than the others. Each of these 20 <br />                                             mutants was also modestly hypersensitive to nalidixic acid, as exemplified by mutants I78 and <br />                                             I40 (Figure 1A and Table 1). Of the remaining 25 intermediate mutants, one (I28) appeared to <br />                                             be hypersensitive to nalidixic acid (Figure 1A and Table 1), while the other 24 were either <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             noticeably resistant or similar to the wild-type (Tables 1 and 2; also see Figure 1A, I104). <br />  <br />                                             As an additional secondary screen, all 25 white mutants and the remaining 45 intermediate <br />                                             mutants were tested for their SOS response to UV. Like nalidixic acid, UV treatment also <br />                                             induces the SOS response, but the pathway for induction requires recFOR rather than recBC, <br />                                             presumably because UV damage leads to single-stranded DNA rather than double-strand <br />                                             breaks (McPartland et al., 1980). Cells were spotted twice onto X-gal plates, and one of the <br />                                             spots was exposed to UV. In the wild-type control, strong induction was evident after four <br />                                             hours incubation (Figure 1B, JH39). <br />  <br />                                             All of the 25 white mutants were also uninducible in this UV test (e.g., Figure 1B, W16). <br />                                             Previous analysis of the JH39 genetic background has shown a background level of &#206;&#178;- <br />                                             galactosidase expression without any SOS-inducing agent, suggesting a low level of SOS- <br />                                             independent transcription of the dinD1::lacZ fusion (Heitman and Model, 1991; also see <br />  <br />  <br />                                                    Gene. Author manuscript; available in PMC 2008 January 15. <br />                             Newmark et al.                                                                                               Page 5 <br />  <br />  <br />                                             Kenyon and Walker, 1980 and Figure 1B, JH39 minus UV). Particularly with the discovery <br />                                             of recA, recB and recC mutants in the intermediate class (see below), we surmised that the <br />                                             white mutants are completely unable to express &#206;&#178;-galactosidase, likely due to transposon <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             insertions in the dinD1::lacZ fusion, and did not analyze them further. <br />  <br />                                             The intermediate class of 45 mutants turned out to be more interesting. Twenty-nine of these <br />                                             showed apparently wild-type levels of SOS induction in response to UV (e.g., Figure 1B, I40), <br />                                             while the other 16 showed reduced induction (e.g., Figure 1B, I78, I104 and I28; Table 2). <br />                                             Mutants that are unable to process drug-stabilized cleavage complexes into an SOS signal <br />                                             should be among the 29 with a normal SOS response to UV. Those deficient in SOS response <br />                                             to both nalidixic acid and UV apparently have a more general defect in SOS induction. <br />  <br />                                             The small amount of &#206;&#178;-galactosidase expression shown by intermediate mutants in response <br />                                             to nalidixic acid might indicate that the mutated gene is not strictly required for the SOS <br />                                             response and/or that the dinD1::lacZ fusion is expressed at a low level without an SOS response <br />                                             (as mentioned above). <br />  <br />                               3.3. Analysis of insertions by DNA sequencing and P1 transduction <br />                                             We were most interested in the 29 intermediate mutants that showed normal UV responses, <br />                                             particularly those 18 that were hypersensitive to nalidixic acid. Nonetheless, to avoid missing <br />                                             any potentially interesting genes, we decided to sequence all of the intermediate mutants. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             We used the MasterPure&#226;&#8222;&#162; DNA Purification Kit (Epicentre) to purify genomic DNA from <br />                                             each mutant and then used the forward primer from the EZ::TN&#226;&#8222;&#162; transposon kit to sequence <br />                                             from one end of the transposon. Using a modified automatic sequencing protocol, we were <br />                                             able to obtain at least 100 bp of unambiguous sequence from each mutant. Using those <br />                                             sequences, the E. coli K12 genome was then searched using BLAST to determine which gene <br />                                             had been interrupted by the transposon (Table 2 and Table 3). <br />  <br />                                             Two of the insertion mutants were found to have the transposon within the trpB gene, which <br />                                             is part of the dinD1::lacZ fusion (Casadaban and Cohen, 1980). The SOS phenotypes caused <br />                                             by these insertions presumably result from polar effects on lacZ expression, and these two <br />                                             mutants were therefore discarded from the collection (not included in the Tables). <br />  <br />                                             Two of the transposon insertions from the intermediate class were in the recA gene. Both <br />                                             recA mutants were hypersensitive to nalidixic acid and were very light blue in both the nalidixic <br />                                             acid and UV tests. Given that JH39 produces a small amount of &#206;&#178;-galactosidase without <br />                                             exposure to a DNA-damaging agent (Heitman and Model, 1991), the very weak level of &#206;&#178;- <br />                                             galactosidase in the two recA insertion mutants is probably unrelated to the SOS pathway. We <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             found that a previously constructed recA56 derivative of JH39, called JH59 (Heitman and <br />                                             Model, 1991), also turned very light blue in a nalidixic acid disk test. Furthermore, Kenyon <br />                                             and Walker (1980) found that RecA was required for induced but not for basal expression from <br />                                             several din fusion constructs (the dinD1 fusion was not tested). In summary, the two new <br />                                             recA mutants had the expected <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span>phenotype</span>: defective in the SOS response to both nalidixic acid <br />                                             and UV, and hypersensitive to nalidixic acid. <br />  <br />                                             We expected to find recB and recC mutants among the mutants that were specifically defective <br />                                             in the SOS response to nalidixic acid (see Introduction). Indeed, sequencing revealed eight <br />                                             insertions in recB and eight in recC, and all 16 were hypersensitive to nalidixic acid (see Tables <br />                                             1 and 2). Finding recB and recC mutants in this group validated our mutant hunt, and also <br />                                             focused our attention on other mutants with the same phenotype in our search for additional <br />                                             genes necessary for the SOS response from nalidixic acid-stabilized cleavage complexes. <br />  <br />  <br />  <br />                                                    Gene. Author manuscript; available in PMC 2008 January 15. <br />                             Newmark et al.                                                                                             Page 6 <br />  <br />  <br />                                             Besides the 16 recBC mutants, the only other intermediate mutants that fit this category of <br />                                             interest had an insertion in ompA, the gene for a major outer membrane protein, or yhcL, the <br />                                             product of which has been suggested to be involved in dicarboxylate transport (Janausch and <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Unden, 1999). Because neither protein function suggested a role in the processing of cleavage <br />                                             complexes and because each gene was found only once in the screen, we were suspicious that <br />                                             the transposon insertions in these two strains might not be causing the observed phenotype. To <br />                                             resolve this uncertainty, we performed phage P1 transductions to move each transposon <br />                                             <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span>insertion mutation</span> into a clean JH39 background, using the kanamycin selectable marker. The <br />                                             transductants behaved like wild-type JH39 in both the nalidixic acid and UV tests, indicating <br />                                             that the transposon mutations in ompA and yhcL were not causing the interesting phenotype. <br />                                             Because we had screened about 19,000 colonies for the phenotype of interest, it seemed quite <br />                                             possible that these two had acquired a spontaneous mutation that inactivated either recB or <br />                                             recC. Indeed, PCR amplification of genomic DNA from the ompA and yhcL strains revealed <br />                                             that their recB genes were approximately 1000 bp longer than the wild-type recB gene from <br />                                             JH39. It seems very likely that an endogenous insertion sequence had interrupted the recB <br />                                             genes in both strains, giving them the recBC-like phenotype. The ompA and yhcL insertions <br />                                             were therefore discarded from the collection and are not included in the Tables. <br />  <br />                                             These results with the ompA and yhcL insertions raised the concern that other transposon <br />                                             insertions in our collection might not be causing the phenotype of interest. Insertions in 10 <br />                                             different genes, including recA, recB and recC, were isolated multiple times with the same <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             phenotype, arguing very strongly that the transposon is causative in those cases. However, in <br />                                             addition to ompA and yhcL, we isolated single-hit insertions in 6 additional genes/reading <br />                                             frames. To test whether these insertions were causing the SOS phenotype, we again performed <br />                                             P1 transductions using the insertion mutants as donors and JH39 as recipient. Kanamycin- <br />                                             resistant transductants from the hns, intQ, moaD and mobA donors recapitulated the SOS <br />                                             phenotypes of their donor parents, confirming that the transposon was causative in each case. <br />                                             Two other insertion mutants failed this transduction test and were therefore eliminated from <br />                                             the collection (not included in Tables). <br />  <br />                               4. Discussion <br />                                             We are interested in the pathway(s) that leads to DNA breaks after treatment with inhibitors <br />                                             that stabilize topoisomerase cleavage complexes. Such DNA breaks are thought to be involved <br />                                             in cytotoxicity and DNA rearrangements caused by inhibitors of type II topoisomerases (see <br />                                             Introduction). Elucidation of the detailed mechanism of DNA break formation may reveal <br />                                             important aspects of replication fork dynamics, and might also have implications for <br />                                             antibacterial and anticancer chemotherapy with topoisomerase inhibitors. We approached this <br />                                             issue experimentally by taking advantage of the SOS response, isolating mutants with reduced <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             SOS induction upon nalidixic acid treatment. <br />  <br />                                             Mutational inactivation of recB or recC blocks SOS induction by nalidixic acid but not by UV <br />                                             (McPartland et al., 1980), so we focused our attention on mutants with this phenotype. Out of <br />                                             19,000 transposon insertion mutants, we found 18 that were defective for induction by nalidixic <br />                                             acid but normal for induction by UV. Sixteen of these had a transposon insertion in the recB <br />                                             or recC gene, which confirms the validity of the genetic screen. Another two had insertions <br />                                             elsewhere, but we found that these two strains had an approximately 1-kb insertion in the <br />                                             recB gene, which explains the induction defect. E. coli has numerous insertion sequences that <br />                                             are roughly 1-kb in size (Galas and Chandler, 1989), so we surmise that these mutants had <br />                                             undergone a transposition event that inactivated the recB gene at some time prior to our genetic <br />                                             screen. Since the competent cells for generating the 19,000 transposon insertion mutants were <br />                                             grown as a single batch, these two strains may have started out as siblings with the insertion <br />                                             sequence in recB prior to the transformation of the transposon. <br />  <br />  <br />                                                    Gene. Author manuscript; available in PMC 2008 January 15. <br />                             Newmark et al.                                                                                               Page 7 <br />  <br />  <br />                                             Thus, we found a total of 18 mutants with the desired phenotype, but all 18 were defective in <br />                                             recB or recC. Since we screened 19,000 colonies and isolated so many recBC mutants, the <br />                                             screen was quite thorough, and it is unlikely that any novel gene would be uncovered by <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             screening additional mutants using this method and these criteria. Because this was a <br />                                             transposon mutagenesis strategy, one possibility is that RecBCD is the only non-essential <br />                                             protein involved in SOS induction by nalidixic acid but not by UV. Only the helicase activity <br />                                             of RecBCD is necessary for SOS induction after nalidixic acid treatment (Chaudhury and <br />                                             Smith, 1985). Therefore, the role of RecBCD is probably limited to generation of single- <br />                                             stranded DNA from the break, and the enzyme is not likely involved in break formation. <br />  <br />                                             These results are consistent with the possibility that essential cellular proteins are involved in <br />                                             DNA break formation from the cleavage complex. For example, transcription and/or DNA <br />                                             replication could be necessary. One specific model is the replication fork run-off model, in <br />                                             which the replisome directly causes DNA breaks upon collision with the cleavage complex <br />                                             (see Introduction). If transcription and/or replication are necessary for break formation, then <br />                                             some of our mutants might have reduced SOS induction simply because of slower growth. We <br />                                             measured the growth rate of one insertion mutant for each of the genes uncovered here, and <br />                                             found that several did indeed have significantly decreased growth rates: cyaA, icdA, mcrA/ <br />                                             elbA, crp, and tufA (data not shown; we did not test tufB, but it would presumably grow similarly <br />                                             to the tufA mutant). While any of these mutants could have reduced SOS induction due to poor <br />                                             growth, we note that the other mutants we isolated had little or no growth defect, and we would <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             have expected to isolate many other poor-growth mutants if that was all that was necessary to <br />                                             reduce SOS induction. <br />  <br />                                             A second possible explanation for finding only recBC mutants with the desired phenotype is <br />                                             that multiple pathways process the cleavage complex into an SOS-inducing signal. Some <br />                                             candidates for genes that might be involved in redundant pathways were uncovered in mutants <br />                                             that were modestly defective for SOS induction after nalidixic acid but seemingly normal after <br />                                             UV, namely icdA, intQ, mcrA/elbA, moaD, mobA, tufA/B and trkH (Table 2). The only mutants <br />                                             in this group with altered sensitivity to nalidixic acid were those in trkH, which were modestly <br />                                             resistant to the drug (Table 1). The trkH gene encodes an integral membrane protein involved <br />                                             in potassium uptake (Silver, 1996) and trkH mutations might somehow impact nalidixic acid <br />                                             uptake or efflux. Thus, the decreased SOS response of trkH mutants could simply reflect a <br />                                             lower effective level of inhibitor inside the cell. None of the other genes mentioned above has <br />                                             a function that clearly suggests an involvement in cleavage complex processing. However, two <br />                                             of the gene products have known or suspected DNA cleavage activities: McrA, which is a <br />                                             restriction endonuclease, and IntQ, which is encoded by a cryptic prophage and is related to <br />                                             the integrase family of proteins. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             An inherent limitation of our genetic screen raises a third possible explanation for finding only <br />                                             recBC mutants with a strong SOS defect that is specific for nalidixic acid treatment. Perhaps <br />                                             the mutants of interest were missed in our screen because they were partially constitutive for <br />                                             the SOS response. That is, even in the absence of nalidixic acid, the desired mutants might <br />                                             have levels of &#206;&#178;-galactosidase similar to or greater than the levels in wild-type cells with <br />                                             nalidixic acid. Insertions in a variety of important DNA metabolism genes cause an SOS <br />                                             constitutive phenotype (O&#226;&#8364;&#8482;Reilly and Kreuzer, 2004), and we are currently analyzing these to <br />                                             see if any are defective for SOS induction after nalidixic acid treatment. <br />  <br />                                             Our collection also included insertions in four genes that caused SOS defects after either <br />                                             nalidixic acid or UV treatment. Mutations in each of these genes are known to cause pleiotropic <br />                                             effects by altering nucleiod structure and gene expression (hns), protease function (clpP) or <br />                                             the cyclic AMP regulatory system (cyaA and crp) (Dorman, 2004; Saier et al., 1996; Gross, <br />                                             1996). Although the SOS defects in each of these four mutants is likely to be caused by some <br />  <br />  <br />                                                    Gene. Author manuscript; available in PMC 2008 January 15. <br />                             Newmark et al.                                                                                                                 Page 8 <br />  <br />  <br />                                             indirect effect of their pleiotropy, these results may provide clues about subtleties of the SOS <br />                                             regulatory system. The cAMP system, involving the products of the crp and cyaA genes, has <br />                                             previously been linked to the SOS system in the context of cAMP-dependent SOS induction <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             in resting cells (Taddei et al., 1995). <br />  <br />  <br />  <br />                               Abbreviations <br />                                             LB <br />                                                            L-broth <br />  <br />                                             UV <br />                                                            ultraviolet light <br />  <br />                                             X-gal <br />                                                            5-bromo-4-chloro-3-indolyl-&#206;&#178;-D-galactopyranoside <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                     Gene. Author manuscript; available in PMC 2008 January 15. <br />                             Newmark et al.                                                                                            Page 10 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Fig. 1. <br />                                             Nalidixic acid disk tests and UV tests of wild type and insertion mutants. A. Overnight cultures <br />                                             of each strain in LB were diluted to an OD560 of approximately 1.0 and 200 &#206;&#188;L of the diluted <br />                                             cultures were added to 3.0 mL of top agar. The top agar mixture was then poured onto an LB <br />                                             plate containing X-gal at 60 &#206;&#188;g/mL. Four circular glass fiber filters (GF/C; approximately 1 <br />                                             cm in diameter) were placed onto the cooled plate. A solution of nalidixic acid was then <br />                                             delivered to the filter disks (0, 2, 4 or 8 uL of a 5 mg/ml stock in 10 mM NaOH, corresponding <br />                                             to 0, 10, 20 or 40 &#206;&#188;g), starting from the top left position and continuing clockwise on each <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             plate. The plates were incubated overnight at 37&#194;&#176;C and photographed the next morning. The <br />                                             strain name is given above each plate, along with the gene interrupted by the transposon in <br />                                             parentheses (except for parental JH39, which has no transposon, and W16, which has not been <br />                                             sequenced). B. Overnight cultures of each strain were spotted (8 &#206;&#188;L) in duplicate onto LB <br />                                             plates containing X-gal at 60 &#206;&#188;g/mL. After four hours of incubation at 37&#194;&#176;C, half the plate was <br />                                             exposed to UV (52.5 J/m2, from a General Electric G15T8 germicidal bulb) while the other <br />                                             half was protected with a sheet of cardboard. The plates were then wrapped in foil to protect <br />                                             them from light and incubated for an additional four hours at 37&#194;&#176;C. The plates were then stored <br />                                             at 4&#194;&#176;C and photographed the next day. <br />  <br />  <br />  <br />  <br />                                                       Gene. Author manuscript; available in PMC 2008 January 15. <br />                                 Newmark et al.                                                                                                                               Page 11 <br />  <br />                                                                                                     Table 1 <br />                                Zones of inhibition from nalidixic acid. <br />  <br />                                                                                                              Zones of Inhibition (mm)a <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                 Nalidixic Acid (&#206;&#188;g):                         10                               20                               40 <br />                                         JH39                             14.6 (&#194;&#177;1.0)                      17.2 (&#194;&#177;1.3)                      19.8 (&#194;&#177;1.4) <br />                                         recA                                17.0                             20.5                             24.0                   HS <br />                                         recB                             17.8 (&#194;&#177;0.7)                      20.8 (&#194;&#177;0.9)                      23.6 (&#194;&#177;0.7)               HS <br />                                         recC                             18.3 (&#194;&#177;1.0)                      21.6 (&#194;&#177;1.3)                      24.5 (&#194;&#177;0.9)               HS <br />                                          hns                                17.0                             20.0                             23.0                   HS <br />                                         clpP                                14.0                             16.5                             20.0                   N <br />                                         cyaA                             13.3 (&#194;&#177;2.5)                      17.0 (&#194;&#177;1.7)                      20.3 (&#194;&#177;1.5)               N <br />                                         icdA                                14.0                             16.0                             18.0                   N <br />                                         intQ                                14.0                             16.0                             20.0                   N <br />                                     mcrA/elbA                               13.0                             16.0                             18.5                   N <br />                                        moaD                                 15.0                             17.0                             21.0                   N <br />                                        mobA                                 15.0                             17.0                             20.0                   N <br />                                        tufA/B                               14.5                             16.5                             19.5                   N <br />                                         crpb                             11.7 (&#194;&#177;0.6)                      15.7 (&#194;&#177;0.6)                      19.7 (&#194;&#177;0.6)               r <br />                                         trkH                                12.0                             14.0                             17.0                   R <br />  <br />                            a <br />                             The diameter of the zones of inhibition was measured on plates prepared identically to those depicted in Figure 1A. HS, hypersensitive, R, markedly <br />                            resistant, r, slightly resistant, N, normal (similar to parental JH39). The values shown represent the average measurements of independent transposon <br />                            insertion mutants. Standard error is shown when three or more transposon insertion mutants were available (see Table 3). The JH39 values represent the <br />                            average and standard deviation of 18 different measurements of this parental strain. <br />                            b <br />                             Although they did not significantly differ, the measurements of the crp promoter insertion mutant (I42) are not included in this calculation. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                                  Gene. Author manuscript; available in PMC 2008 January 15. <br />                                                                   NIH-PA Author Manuscript                                                  NIH-PA Author Manuscript                                               NIH-PA Author Manuscript <br />                                                                                                                                                                Table 2 <br />                                                              Summary of intermediate mutants. <br />  <br />                                                                  Genea                       #                 NAL Colorb                         UV Colorc                     NAL Sensitivityd            Putative or known functione <br />  <br />                                                                  recA                        2                      +                                 +                                 HS                  Homologous pairing and strand exchange protein <br />                                                                  recB                        8                      +                                +++                                HS                  RecB subunit of the helicase-nuclease ExoV <br />                                                                  recB (IS)f                  2                      +                                +++                                HS                  RecB subunit of the helicase-nuclease ExoV <br />                                                                  recC                        8                      +                                +++                                HS                  RecC subunit of the helicase-nuclease ExoV <br />                                                                                                                                                                                                                                                                                   Newmark et al. <br />  <br />  <br />  <br />  <br />                                                                  hns                         1                      ++                               ++                                 HS                  DNA-binding protein H-NS <br />                                                                  clpP                        2                      ++                               ++                                 N                   ATP-dependent serine protease <br />                                                                  cyaA                        3                      ++                                +                                 N                   Adenylate cyclase <br />                                                                  icdA                        2                      ++                               +++                                N                   Isocitrate dehydrogenase (TCA) <br />                                                                  intQ                        1                      ++                               +++                                N                   Putative integrase gene within defective prophage Qin <br />                                                                  mcrA/elbAg                  2                      ++                               +++                                N                   Restriction endonuclease (McrA)/isoprenoid biosynthesis (elbA) <br />                                                                  moaD                        1                      ++                               +++                                N                   Molybdopterin converting factor, small subunit <br />                                                                  mobA                        1                      ++                               +++                                N                   Molybdopterin-guanine dinucleotide biosynthesis protein A <br />                                                                  tufA/B                      2                      ++                               +++                                N                   Protein chain elongation factor EF-Tu <br />                                                                  crph                        4                      ++                                +                                  r                  cAMP receptor protein <br />                                                                  trkH                        2                      ++                               +++                                R                   Membrane protein, K+ transport <br />  <br />                                                                     a <br />                                                                      For each gene isolated only once, P1 transductions demonstrated that the transposon insertion is causative. <br />                                                                     b <br />                                                                      Intensity of blue color in response to nalidixic acid, as determined by disk test. +++ = normal, ++ = light blue, + = very light blue. These descriptions are relative to JH39 and are not directly <br />                                                                     comparable to the UV tests. <br />                                                                     c <br />                                                                      Intensity of blue color in response to UV radiation, as determined by UV test. +++ = normal, ++ = light blue, + = very light blue. These descriptions are relative to JH39 and are not directly comparable <br />                                                                     to the nalidixic acid disk tests. <br />                                                                     d <br />                                                                      Drug sensitivity to nalidixic acid, as determined by disk test (Table 1). N = normal (similar to parental JH39), HS = hypersensitive, r = slightly resistant, R = markedly resistant <br />                                                                     e <br />                                                                      Functions as defined by the NCBI website for E. coli K12. Italicized function is putative. <br />                                                                     f <br />                                                                      These are the two strains with a transposon in yhcL or ompA and an apparent IS element in recB. <br />                                                                     g <br />                                                                         The transposon insertion is located between these two genes. <br />  <br />  <br />  <br />  <br /> Gene. Author manuscript; available in PMC 2008 January 15. <br />                                                                     h <br />                                                                         Three of the crp insertions are within the coding sequence, the fourth is located 115 nucleotides upstream of the start codon. <br />                                                                                                                                                                                                                                                                                   Page 12 <br />                                   Newmark et al.                                                                                                                             Page 13 <br />  <br />                                                                                                      Table 3 <br />                                  Locations and directions of transposon insertions. <br />  <br />                            genea                   strain number                           9 bp direct repeatb                        last nucleotide transcribed <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            clpP                    I35                                     ACTTTGCAC                                  34 <br />                                                    I75                                     ACATGGCTA                                  163 <br />                            crp                     I42                                     TCACATTAC                                  &#226;&#710;&#8217;114c <br />                                                    I100                                    GTGCTTGGC                                  12 <br />                                                    I104                                    GGGTAAAGA                                  176 <br />                                                    I105                                    CGGTAAAGA                                  608 <br />                            cyaA                    I65                                     GTGTGGAAG                                  451 <br />                                                    I102                                    GCATATACT                                  542 <br />                                                    I103                                    CCGCAGGAT                                  1242 <br />                            hns                     I28                                     AACCTGGAC                                  329 <br />                            icdA                    I37                                     GTTTGAAGC                                  1010 <br />                                                    I107                                    CTCCAAGCC                                  418 <br />                            intQ                    I6                                      GGGTTAAGC                                  2 <br />                            mcrA/elbA               I24                                     GTTCAGAGT                                  +361d <br />                                                    I44                                     GGGTTGGAC                                  +248d <br />                            moaD                    I19                                     ACCATCCGC                                  196 <br />                            mobA                    I22                                     GCCTTACAT                                  314 <br />                            recA                    I7                                      GTCCAACAC                                  563 <br />                                                    I78                                     GGTGAAGAC                                  99 <br />                            recB                    I40                                     GGTATGTGC                                  1568e <br />                                                    I57                                     GTTCGATGG                                  1940 <br />                                                    I68                                     GGATGTGGT                                  1922 <br />                                                    I73                                     GATTTACAC                                  3470 <br />                                                    I79                                     GGTATGTGC                                  1568e <br />                                                    I83                                     TTCTTGCAC                                  2868 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                    I93                                     ACCAGGCCT                                  491 <br />                                                    I94                                     ATGGTGAAC                                  1365 <br />                            recC                    I32                                     GATAGCAGT                                  1119 <br />                                                    I36                                     CCCGCCACC                                  1558 <br />                                                    I50                                     GCTGGGAAT                                  2039 <br />                                                    I52                                     GTCTGGACG                                  34 <br />                                                    I54                                     ACATGATGG                                  3241 <br />                                                    I85                                     GGGTAAACA                                  699 <br />                                                    I90                                     GCCTGGCAC                                  442 <br />                                                    I97                                     GTTGTTGGG                                  1601 <br />                            trkH                    I23                                     TAACTACGG                                  761 <br />                                                    I27                                     GATTATCCC                                  71 <br />                            tufA                    I29                                     CACGTAGAC                                  959 <br />                            tufB                    I17                                     GGCGTAGAG                                  1047 <br />  <br />                            a <br />                                Alternate gene names: elbA (ycgW), intQ (b1579) <br />                            b <br />                             The 9 bp repeat was determined using a single primer from within the transposon. The last <span id='am-238' about='xsp:base' typeof='owl:Thing'>base</span> of the inferred repeat shows the last intact nucleotide of <br />                            the gene, reading in the 5&#226;&#8364;&#178; to 3&#226;&#8364;&#178; direction with respect to transcription. <br />                            c <br />                             This insertion is located 115 nucleotides upstream of the start codon of the crp gene. <br />                            d <br />                             These insertions lay 361 and 248 nucleotides downstream of the stop codon of the mcrA gene. <br />                            e <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             These insertions, while identical, were isolated independently and are therefore not clones. <br />  <br />  <br />  <br />  <br />                                                                    Gene. Author manuscript; available in PMC 2008 January 15. <br />   <br /> </body></html>