<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Genome-Wide Analysis Reveals Novel Genes Essential for <br /> Heme Homeostasis Caenorhabditis elegans <br /> Scott Severance1,2., Abbhirami Rajagopal1,2., Anita U. Rao1,2, Gustavo C. Cerqueira2,3, Makedonka <br /> Mitreva4, Najib M. El-Sayed2,3, Michael Krause5, Iqbal Hamza1,2* <br /> 1 Department Animal Avian Sciences, University Maryland, College Park, Maryland, United States America, 2 Department Cell Biology Molecular <br /> Genetics, University Maryland, College Park, Maryland, United States America, 3 Center Bioinformatics Computational Biology, University Maryland, College <br /> Park, Maryland, United States America, 4 The Genome Center, Department Genetics, Washington University School Medicine, St. Louis, Missouri, United States of <br /> America, 5 Laboratory Molecular Biology, National Institute Diabetes Digestive Kidney Diseases, National Institutes Health, Bethesda, Maryland, United <br /> States America <br />  <br />  <br />  <br />      <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />      Heme cofactor proteins function sub-cellular compartments diverse biological <br />      processes. Heme produced conserved biosynthetic pathway highly regulated prevent accumulation of <br />      heme&#226;&#8364;&#8221;a cytotoxic, hydrophobic tetrapyrrole. Caenorhabditis elegans related parasitic nematodes synthesize <br />      heme, instead require environmental heme grow develop. Heme homeostasis auxotrophs is, therefore, <br />      regulated accordance available dietary heme. We capitalized auxotrophy C. elegans study gene <br />      expression changes associated precisely controlled dietary heme concentrations. RNA isolated cultures <br />      containing 4, 20, 500 mM heme; derived cDNA probes hybridized Affymetrix C. elegans expression arrays. We <br />      identified 288 heme-responsive genes (hrgs) differentially expressed conditions. Of genes, 42% <br />      putative homologs humans, genomes medically relevant heme auxotrophs revealed homologs 12% in <br />      Trypanosoma Leishmania 24% parasitic nematodes. Depletion of 288 hrgs RNA&#226;&#8364;&#8220;mediated <br />      interference (RNAi) transgenic heme-sensor worm strain identified genes regulated heme homeostasis. In <br />      addition, seven membrane-spanning transporters involved heme uptake identified RNAi knockdown studies <br />      using toxic heme analog. Comparison genes positive RNAi screens resulted identification <br />      genes common vital organismal heme homeostasis C. elegans. Collectively, results provide a <br />      catalog genes essential metazoan heme homeostasis demonstrate power C. elegans genetic <br />      animal model dissect regulatory circuits mediate heme trafficking vertebrate hosts parasites, <br />      depend environmental heme <span id='am-8' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-9' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span>survival</span>. <br />  <br />   Citation: Severance S, Rajagopal A, Rao AU, Cerqueira GC, Mitreva M, et al. (2010) Genome-Wide Analysis Reveals Novel Genes Essential Heme Homeostasis in <br />   Caenorhabditis elegans. PLoS Genet 6(7): e1001044. doi:10.1371/journal.pgen.1001044 <br />   Editor: Andrew D. Chisholm, University California San Diego, United States America <br />   Received January 14, 2010; Accepted June 30, 2010; Published July 29, 2010 <br />   This open-access article distributed terms Creative Commons Public Domain declaration stipulates that, placed public <br />   domain, work freely reproduced, distributed, transmitted, modified, built upon, used lawful purpose. <br />   Funding: This work supported funding National Institutes Health (R01DK74797 IH, R01AI081803 MM, NRSA F32DK080603 SS), the <br />   March Dimes Birth Defects Foundation (1-FY06-1123 IH), NIH Intramural Research Program, National Institute Diabetes Digestive Kidney <br />   Diseases (MK). The funders role study design, data collection analysis, decision publish, preparation manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: hamza@umd.edu <br />   . These authors contributed equally work. <br />  <br />  <br />  <br /> Introduction                                                                            understood [4]. Heme transport molecules animals likely <span id='am-7' about='protege:TO' typeof='owl:Thing'>to</span> <br />                                                                                         divergent bacterial yeast proteins genetic level; <br />   <span id='am-4' about='protege:FROM' typeof='owl:Thing'>From</span> nutritional perspective, heme readily bioavailable                        bacterial yeast heme-binding proteins obvious <br /> source iron human consumption [1,2]. From cellular                             orthologs mammals [5&#226;&#8364;&#8220;8]. This demonstrated the <br /> perspective, heme iron-containing porphyrin serves                     identification heme exporter, feline leukemia virus subgroup <br /> prosthetic group diverse biological processes ranging gas-                      C cellular receptor (FLVCR), does obvious <br /> sensing microRNA processing [3]. In eukaryotes, heme                         similarities known bacterial heme transport proteins [9,10]. <br /> synthesized mitochondrial matrix defined biosynthetic                       Genetic ablation FLVCR mice resulted severe macrocytic <br /> pathway subsequently exported needed heme-containing                         anemia proerythroblast maturation arrest. That viral <br /> proteins cytoplasm membrane-bound                             receptor potential heme exporter developing erythroid <br /> organelles [3]. Given hydrophobicity cytotoxicity associated                    cells underscores divergence heme transport proteins <br /> with free heme, likely specific intracellular transport                      emphasizes importance implementing unbiased genetic <br /> pathways exist deliver heme assimilation hemoproteins                       approaches elucidate heme homeostasis pathways tractable <br /> found various subcellular compartments [4].                                          model systems. <br />   Although pathway intermediates heme biosynthesis                             Progress understanding heme homeostasis eukaryotic <br /> and degradation defined, intracellular networks                      systems hampered inability separate heme biosynthesis <br /> that mediate heme homeostasis eukaryotes remain poorly                               downstream intracellular transport pathways. To circumvent <br />  <br />  <br />         PLoS Genetics | www.plosgenetics.org                                        1                                July 2010 | Volume 6 | Issue 7 | e1001044 <br />                                                                                                            Genomic Specification Heme C. elegans <br />  <br />  <br />  <br />   Author Summary                                                                genomic studies universally applicable result the <br />                                                                                 discovery heme homeostasis pathways metazoans. <br />   Heme iron-containing cofactor proteins involved <br />   critical cellular processes. However, free heme                    Results <br />   toxic cells, suggesting heme synthesis, acquisition, <br />   transport highly regulated. Efforts understand                      Strategy profile genes transcriptionally <br />   heme trafficking multicellular organisms failed                       regulated heme C. elegans <br />   primarily inability separate processes                      C. elegans lacks highly conserved genes heme biosynthesis but <br />   endogenous heme synthesis heme uptake                                acquires heme environment growth development <br />   transport. Caenorhabditis elegans unique model                       [11]. Worms cultured axenic liquid mCeHR-2 medium the <br />   organisms synthesize heme instead <br />                                                                                 presence different amounts heme revealed characteristic <br />   eats environmental heme grow develop normally. <br />                                                                                 growth curve [11]. The optimal concentration worm growth and <br />   Thus, worms ideal genetic animal model study <br />   heme homeostasis. This work identifies novel list 288                    reproduction 20 mM heme, animals grew <br />   heme-responsive genes (hrgs) C. elegans number                    reproduced concentrations ranging $1.5 mM to <br />   related genes humans medically relevant parasites.                     ,800 mM heme. Worms grown absence exogenous heme <br />   Knocking function of hrgs reveals                      arrested L4 larval stage, concentrations heme <br />   roles heme uptake, transport, detection                    $800 mM caused worms arrest L2/L3 larval stages, <br />   organism. Our study provides insights                         possibly heme cytotoxicity. These results consistent with <br />   metazoan regulation organismal heme homeostasis.                           metabolic labelling experiments fluorescent heme <br />   The identification parasite-specific hrg homologs                      analog, zinc mesoporphyrin IX (ZnMP), used demonstrate <br />   permit selective design screening drugs                       heme uptake regulated C. elegans [12]. <br />   specifically target heme uptake pathways parasites                            To determine transcriptionally regulated compo- <br />   affecting host. Thus, work therapeutic                   nents heme uptake, wild-type N2 worms grown 4, 20, or <br />   implications treatment human iron deficiency,                      500 mM heme axenic liquid mCeHR-2 medium; 20 mM served <br />   mortality factors world-wide.                              reference sample. We chose 4 500 mM heme because <br />                                                                                 concentrations biphasic growth <br /> this issue, established genetically tractable nematode                   curve. More importantly, worms grown heme <br /> Caenorhabditis elegans animal model ideally suited            concentrations exhibited 16 h growth delay, were <br /> conduct heme studies. We previously demonstrated                 morphologically indistinguishable worms grown 20 mM <br /> roundworm does synthesize heme instead relies                        heme. In order reduce variability carryover maternal <br /> environmental heme survival [11]. Moreover, analyses                     heme P0 hermaphrodites, worms grown their <br /> available genomes related parasitic nematodes suggest                 respective heme concentrations successive generations <br /> these helminths heme auxotrophs [11]. The C. elegans                   (Figure 1). Synchronized, late L4 larvae F2 generation <br /> genome encodes repertoire hemoproteins vertebrate                harvested RNA isolation, corresponding cDNA <br /> orthologs. It likely pathways heme trafficking              probes generated hybridized Affymetrix C. elegans <br /> incorporation conserved C. elegans, parasitic worms,                 expression microarray chips. Three biological replicates were <br /> vertebrates [4]. The validity C. elegans model                prepared heme concentration. <br /> recently underscored discovery HRG-1 proteins that <br /> transport heme [12]. We identified C. elegans hrg-1 paralog             Identification hrgs C. elegans <br /> hrg-4 microarray experiments genes highly                         Statistical analyses microarray data initially <br /> upregulated low heme [12]. Expression genes               performed using Affymetrix MAS 5.0 suite software (see <br /> human homolog, HRG-1, Xenopus oocytes resulted strong                     Materials Methods). Of 22,627 probe sets, 835 probe sets <br /> heme-induced electrophysiological currents &#226;&#8364;&#8220; indication                 revealed changes 4 500 mM heme compared the <br /> the corresponding proteins heme transporters. Additionally,                control data 20 mM heme. We identified 288 genes a <br /> depletion hrg-1 worms led aberrant heme homeostasis.                   $1.6-fold change expression. To improve augment these <br /> Transient knockdown hrg-1 zebrafish caused severe                         analyses, subjected microarray results Robust <br /> impairment erythropoiesis brain skeletal defects;             Multichip Average method (RMA R package) goal <br /> these phenotypes fully rescued worm hrg-1 [12].                         combining results obtained MAS 5.0. The <br /> Collectively, studies validated advantage C.               RMA analysis (minimum change expression $1.2 fold) <br /> elegans model par excellence dissect pathways responsible           identified additional 82 hrgs. The MAS 5.0 RMA analyses <br /> for heme transport homeostasis mammals. Moreover, C.                     yielded total 370 candidate genes. Subsequently, duplicate <br /> elegans bridges evolutionary divide heme auxotrophic                     genes eliminated, minimum cut-off value RMA <br /> parasitic species provides insight helminthic-specific                 analysis increased $1.6 fold, average fold- <br /> vulnerabilities heme uptake utilization exploited            change values calculated replicates. This resulted a <br /> for drug design [13,14].                                                        list candidate genes consisting 266 genes identified using <br />    The current study specifically seeks explain draw                     MAS 5.0 22 genes selected using RMA method. The <br /> conclusions genomic data generated                   expression 288 genes, previously <br /> microarray analysis. This expression array analysis using C. elegans            identified germline genes [15], revealed $1.6-fold change at <br /> wild-type worms grown axenic liquid medium                       4 500 mM heme compared 20 mM controls. <br /> different concentrations heme performed step           Consequently, 288 genes classified hrgs (Table S1). <br /> genome-wide identification genes involved heme homeostasis.                   Normalized signal intensity values graphed visualize <br /> Our results identified heme-responsive genes (hrgs),       quality microarray data generated replicate <br /> some evolutionarily conserved metazoa                 (Figure 2A). The value colored lines cross thin <br /> others nematodes. We anticipate results             vertical line value normalized signal replicate. <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                                 2                            July 2010 | Volume 6 | Issue 7 | e1001044 <br />                                                                                                      Genomic Specification Heme C. elegans <br />  <br />  <br />  <br />  <br /> Figure 1. Profiling strategy analysis heme-responsive genes. Worms grown axenic liquid mCeHR-2 medium containing 4, 20, <br /> or 500 mM hemin chloride synchronized generations, late L4 larvae harvested extract RNA microarray analyses. cDNA was <br /> synthesized reverse transcription hybridized Affymetrix C. elegans Genome Arrays containing 22,627 probe sets chip. Affymetrix MAS <br /> 5.0 software RMA utilized analyze data. Data worms grown 4 500 mM heme compared control data worms <br /> grown 20 mM heme. The expression 288 genes increased decreased response heme 1.6 fold. <br /> doi:10.1371/journal.pgen.1001044.g001 <br />  <br /> Accurate replicates nearly horizontal lines (all values           The 288 hrgs assigned categories based on <br /> approximately equal) condition                   gene expression upregulated, downregulated, <br /> decrease increase condition change.           unchanged samples obtained worms grown 4 or <br /> In experiment, analysis of 288 hrgs revealed         500 mM heme compared 20 mM reference samples <br /> individual biological replicates nearly equal values little       (Figure 3). Eighty genes upregulated 4 mM heme (Table <br /> variation particular heme concentration, indicating          S2). Seventy-five genes upregulated 500 mM heme <br /> changes heme-dependent gene expression uniform. A                  (Table S3). Quantitative real-time PCR analysis (qRT-PCR) of <br /> principal components analysis (PCA) hrgs showed that,         representative genes categories was <br /> one exception, quality microarray data consistent           performed ensure changes observed the <br /> across biological replicates heme concentrations. The        microarray reproducible. As determined signifi- <br /> data obtained 4 mM heme replicates showed               cance (P,0.0001) Pearson&#226;&#8364;&#8482;s correlation coefficient, the <br /> inconsistent global gene expression <span id='am-167' about='xsp:pattern' typeof='owl:Thing'>pattern</span> compared           qRT-PCR confirmed changes observed the <br /> other replicates was, therefore, excluded             microarray results consistent and, therefore, reliable <br /> analysis (Figure S1).                                                      (Figure 2B; Table S4). <br />  <br />  <br />       PLoS Genetics | www.plosgenetics.org                             3                           July 2010 | Volume 6 | Issue 7 | e1001044 <br />                                             Genomic Specification Heme C. elegans <br />  <br />  <br />  <br />  <br /> PLoS Genetics | www.plosgenetics.org   4   July 2010 | Volume 6 | Issue 7 | e1001044 <br />                                                                                                             Genomic Specification Heme C. elegans <br />  <br />  <br />  <br /> Figure 2. Confirmation quality microarray results. (A) Gene expression profile 288 hrgs identified microarrays. Changes <br /> in expression 288 hrgs plotted biological replicate (thin vertical lines) growth conditions respect heme <br /> concentration (4 mM, 20 mM, 500 mM) visualized GeneSpring software (v7.2). Signal intensity values genes biological replicate <br /> and growth condition (as indicated) normalized median value array setting values ,0.01 0.01 using GeneSpring <br /> software resulting values plotted log10 scale. Values single gene connected lines slope indicating change in <br /> value samples; line color coding reflects direction change relative mean (as indicated color bar graph) hue <br /> intensity reflecting statistical confidence value (confidence increases increased brightness). (B) Validation microarray results. Microarray <br /> data verified qRT-PCR RNA 24 genes worms grown mCeHR-2 medium supplemented 4, 20, 500 mM heme. RNA 20 <br /> mM heme used reference sample. Data compared internal GAPDH (gpd-2) control fold change obtained using the <br /> 2 (-DDCt) method. The significance determined using GraphPad Instat (v. 3.06). The values qRT-PCR analysis microarray <br /> experiment provided Table S4. <br /> doi:10.1371/journal.pgen.1001044.g002 <br />  <br />  <br /> Comparative genome analyses hrgs vertebrates                          human parasitic genomes (Figure 4A). These <br /> parasites                                                                       results indicate heme-regulated genes C. elegans have <br />    Since identification hrgs common C. elegans               commonality humans heme prototrophs and <br /> mammals provide unique insights evolutionary                     protozoan parasites rely environmental heme. <br /> conservation heme homeostasis pathways metazoans,                         A small percentage 288 hrgs homologs parasitic <br /> performed reciprocal BLAST searches identify putative human                  nematodes (Figure 4B). To date, draft genomes parasitic <br /> orthologs of 288 genes (Figure 3). Searches using                   nematodes available [18&#226;&#8364;&#8220;21], addition the <br /> protein sequences revealed 121 putative human                   partial genomes available 30 parasitic species. For a <br /> orthologs (minimum E-value = 1024) C. elegans hrgs. The hrgs                 summary available genomes, [22]. Using available <br /> with human homologs present upregulated                     sequence data divided taxonomically distinct clades [23], we <br /> both extreme heme concentrations. Forty-four upregulated                   identified homologs 62 288 hrgs clade V nematodes <br /> at 4 mM heme 42 upregulated 500 mM heme,                      (C. elegans belongs clade V) homologs 10 genes the <br /> 28 downregulated 4 mM heme 36 downregu-                        clade I nematodes (where basal nematode, zoonotic <br /> lated 500 mM heme (Table S1).                                                parasite Trichinella spiralis, resides). While number identified <br />    We previously demonstrated biochemical enzyme assays                 putative orthologs higher crown lineages in <br /> and genomic analyses parasitic nematodes               basal nematodes reside root nematode <br /> sequenced genomes lack genes heme synthesis enzymes and,                evolutionary tree, categories (categories 1 3) <br /> therefore, likely rely environmental heme sustain growth              homologs parasitic species. Categories 1 3 <br /> development [11]. Similarly, genomes Trypanosoma                     represented 13 10 sequences C. elegans, respectively. <br /> Leishmania appear lack genes heme synthesis <br /> [16,17]. This suggests protozoa acquire heme                hrgs enriched regulators development- and <br /> from parasitized host. Figure 3 identifies hrg homologs            transport-related processes <br /> Trypanosoma brucei, Trypanosoma cruzi, Leishmania major. Of 288            Gene ontology (GO) analysis [24] indicated hrgs <br /> hrgs, 12 genes exclusive heme auxotrophs. Thirty-            identified microarray study involved processes <br /> seven genes homologs humans, 84 genes                      varied embryonic development, electron transport, lipid <br />  <br />  <br />  <br />  <br /> Figure 3. Heme-dependent changes gene expression. 1The 288 hrgs arranged categories based up- <br /> regulated, down-regulated, unchanged 4 mM 500 mM heme compared control data 20 mM heme. 2C. elegans protein sequences <br /> for 288 hrgs used perform reciprocal BLAST searches identify putative homologs (E-value cut-off $1024) humans protozoans. <br /> doi:10.1371/journal.pgen.1001044.g003 <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                                 5                              July 2010 | Volume 6 | Issue 7 | e1001044 <br />                                             Genomic Specification Heme C. elegans <br />  <br />  <br />  <br />  <br /> PLoS Genetics | www.plosgenetics.org   6   July 2010 | Volume 6 | Issue 7 | e1001044 <br />                                                                                                         Genomic Specification Heme C. elegans <br />  <br />  <br />  <br /> Figure 4. Comparative analysis heme-responsive genes. (A) Summary overlap hrgs human protozoan genomes. <br /> The sequences proteins encoded 288 hrgs obtained Wormbase used search homologs human genome and <br /> genomes Trypanosoma brucei, Tryoanosoma cruzi, Leishmania major. (B) Orthologs hrgs genomes parasitic nematodes. The 288 C. <br /> elegans hrg gene products used identify homologs available parasitic nematode sequences. Based 18S rRNA sequences, phylum <br /> Nematoda divided major clades; clades include parasites. Homologs identified 69 genes using amino acid sequences (at <br /> BLAST cut-off 35 bits 55% identity), summarized clade level. A total 440,012 peptides 29 parasitic nematode species was <br /> used (clade I 29,203 peptides; clade III 145,044; clade IVA 13,636; clade IVB 92,514 clade V 159,615 peptides). Nematode sequences used this <br /> analysis available parasitic nematodes website (http://www.nematode.net). (C) Gene ontology (GO) enrichment analysis hrgs <br /> upregulated 4 mM heme. Genes upregulated 4 mM heme analyzed using Fisher&#226;&#8364;&#8482;s exact test topGO package R. The most <br /> significant GO terms associated parent terms used construct hierarchical graph specificity terms increased <br /> from bottom. The text rectangle provides GO ID ratio number genes annotated GO term tested <br /> subset total gene set. The shade green rectangle corresponds significance GO result. The complete table P- <br /> values Table S6. Full GO terms provided solely genes P,0.005. <br /> doi:10.1371/journal.pgen.1001044.g004 <br />  <br /> metabolism, iron-sulfur cluster assembly. Of 288 genes            acid, fatty acids. A majority hrgs identified were <br /> the study, 115 annotated biological process (Table S5).          uncharacterized assigned biological pathway. <br /> Using Fisher&#226;&#8364;&#8482;s exact test, hierarchical graph constructed             Genome sequencing demonstrated chromosomes I, II, <br /> with significant GO terms associated parent               III, IV, X C. elegans contain roughly equivalent numbers <br /> terms [25]. Highly significant GO terms (P,0.005) associated            genes (13&#226;&#8364;&#8220;17%), chromosome V genes <br /> the subset genes upregulated 4 mM heme                  (25%) [27]. Furthermore, co-regulated functionally related genes, <br /> &#226;&#8364;&#732;embryonic development&#226;&#8364;&#8482;, &#226;&#8364;&#732;lipid transport&#226;&#8364;&#8482;, &#226;&#8364;&#732;proteolysis&#226;&#8364;&#8482;                especially essential interactions environment, tend <br /> (Figure 4C; Table S6); &#226;&#8364;&#732;responses stress&#226;&#8364;&#8482; environmental               reside local clusters chromosome [27]. We that <br /> stimuli&#226;&#8364;&#8482; associated genes downregulated               Chr I Chr III contained just 6% hrgs, 35% all <br /> 4 mM heme (Figure S2 Tables S7, S8, S9).                                 hrgs Chr V (Figure 5). Additionally, 129 hrgs on <br />    The Kyoto Encyclopedia Genes Genomes (KEGG)                     Chr V, 43 genes upregulated 4 mM heme 41 genes <br /> also frequently used analyze complex microarray data make             upregulated 500 mM heme (Figure 6). Our analysis suggests <br /> functional predictions [26]. Only 10 hrgs (,3%)                    genomic distribution hrgs non-random, reveals gene <br /> mapped KEGG pathways (Table S10). These hits included                     clustering, indicates common biological response an <br /> genes transporters metabolism sugars, amino           environmental stimulus heme. <br />  <br />  <br />  <br />  <br /> Figure 5. Depiction number hrgs chromosome relative number megabases chromosome. <br /> The chromosome highest number hrgs chromosome V. Red brown regions indicate expression gene increased at <br /> 4 500 mM heme, respectively. Light blue dark blue bars represent decrease gene expression 4 500 mM heme, respectively. <br /> doi:10.1371/journal.pgen.1001044.g005 <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                              7                             July 2010 | Volume 6 | Issue 7 | e1001044 <br />                                             Genomic Specification Heme C. elegans <br />  <br />  <br />  <br />  <br /> PLoS Genetics | www.plosgenetics.org   8   July 2010 | Volume 6 | Issue 7 | e1001044 <br />                                                                                                       Genomic Specification Heme C. elegans <br />  <br />  <br />  <br /> Figure 6. Chromosomal location fold-change heme-responsive gene. hrgs arranged chromosomes, <br /> which depicted white black bars. The transcript levels genes placed white bars altered 4 mM heme, expression of <br /> genes black bars significantly altered 500 mM heme. Vertical bars drawn scale represent genes up-regulated (red) or <br /> down-regulated (green). <br /> doi:10.1371/journal.pgen.1001044.g006 <br />  <br />    If genomic distribution hrgs purposeful, reasoned            expression, (Figure 7A). A significant upregulation GFP was <br /> that response promoters hrgs directed        observed 5 mM hrgs encoded putative <br /> a cis-acting element cluster elements common            membrane-spanning proteins (F36H1.5/HRG-4, F14F4.3/MRP- <br /> to hrgs specific category. First analyzed Categories 1       5, F58G6.3/CTR-1, F22B5.4/unnamed protein) putative <br /> 2 overrepresented transcription factor binding sites using           lysosomal cysteine protease (F32H5.1/cathepsin-L) depleted. <br /> sequences control set random promoter sequences             In contrast, GFP downregulated F46E10.11, which <br /> failed identify common cis elements. We reiterated search          encodes uncharacterized protein proposed bind metals <br /> encompass presumptive promoters ($2 kb upstream) 288              cysteine residues, depleted. <br /> hrgs using TRANSFAC [28]. Once again, common elements                        To identify potential heme transporters, identified hrgs which <br /> were identified.                                                             encoded proteins transmembrane domains (TMD). <br />                                                                              TMHMM analysis predicted 41 288 hrgs encoded for <br /> Functional validation reveals novel hrgs essential                  proteins putative TMD (Table S12). Among these <br /> for heme homeostasis C. elegans                                           41 genes encoding aquaglyceroporin-related proteins <br />     A number genome-wide RNA-mediated interference (RNAi)                 (aqp-1 aqp-8 TMD, respectively) that <br /> experiments performed C. elegans, data             transport small molecules glycerol, urea, water; cyp- <br /> all experiments available Wormbase (http://www.                 33C9 (one TMD) belongs cytochrome P450 family of <br /> wormbase.org/). Forty-six hrgs (16%) reported RNAi                     heme binding proteins; heme permeases (hrg-1 hrg-4 4 <br /> <span id='am-86' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-87' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> (Table S11). RNAi knockdown genes                    TMD); ABC transporters (mrp-5 pgp-1 $12 TMD). <br /> often resulted developmental defects sterility                   To narrow list candidate heme transporters, used <br /> embryonic lethality. These phenotypes expected                  RNAi deplete 41 hrgs encoded TMD proteins and <br /> heme essential growth reproduction C. elegans. The             exposed worms gallium protoporphyrin IX (GaPP), toxic <br /> relatively small fraction genes yielding reported RNAi                  heme analog causes severe defects worm growth and <br /> phenotype probably reflects redundancy function                development [11]. We reasoned knockdown putative <br /> of hrgs limited phenotypic assays performed date.             heme transporter presence GaPP result a <br />     We previously reported transgenic worms expressing             concomitant increase worm survival reduced ability to <br /> the hrg-1::gfp transcriptional fusion (strain IQ6011) specifically           transport toxic GaPP [12]. We identified seven hrgs which, when <br /> respond heme growth medium. Thus, strain IQ6011                depleted RNAi, revealed greater survival F1 progeny at <br /> be used animal heme sensor interrogate changes              1.5 mM GaPP, concentration lethal wild-type worms <br /> organismal heme homeostasis [12]. To identify function            (Figure 7B). These seven genes included F36H1.5/hrg-4, F14F4.3/ <br /> hrgs heme homeostasis, established functional RNAi screen            mrp-5, K08E7.9/pgp-1, Y51A2D.4/hmit1.1, Y37A1A.2, T21C9.1, <br /> using IQ6011 (see Materials Methods details). First,              F22B5.4. Three genes &#226;&#8364;&#8220; hrg-4, mrp-5, F22B5.4 &#226;&#8364;&#8220; were <br /> generated sequence-confirmed hrg mini-library E. coli               positive RNAi screens (Figure 7C). Taken together, <br /> feeding strain HT115(DE3) expressed double-stranded RNA                 genomic studies identified small subset genes not <br /> (dsRNA) 288 hrgs. Second, established               regulated heme mRNA level essential <br /> sensitive GFP-based assay conditionally screened genetic            heme transport homeostasis functional level. <br /> modulators heme homeostasis simultaneously presence                To better understand role hrg-4, mrp-5, F22B5.4 in <br /> low (5 mM) high (25 mM) heme. Third, verified positive             heme homeostasis, determined mRNA expression in <br /> candidate genes secondary screen eliminate false positives         response heme ability transport heme function <br /> using vha-6::gfp transgenic worm does respond heme             ZnMP accumulation [12]. qRT-PCR results indicated all <br /> and served negative control. Fourth, confirmed                   genes upregulated heme magnitude of <br /> authenticity candidate gene simultaneously measuring              change mRNA expression 4 mM heme significantly <br /> the GFP fluorescence intensity IQ6011 vha-6::gfp               greater hrg-4 mrp-5 F22B5.4 (8.5-fold versus 4.5- 2- <br /> COPAS Biosort instrument sorts worm time                 fold) (Figure 8A). Heme uptake assays ZnMP revealed that <br /> flight (axial <span id='am-5' about='xsp:length' typeof='owl:Thing'>length</span> object) extinction (optical density           hrg-4 RNAi resulted abrogation ZnMP accumulation the <br /> object).                                                                     worm intestine compared wild-type control worms (Figure 8B), <br />     Synchronized IQ6011 worms grown mCeHR-2                          result consistent previous studies [12]. By contrast, <br /> medium supplemented 10 mM heme repress GFP                       ZnMP accumulation dramatically increased knock- <br /> subsequently transferred NGM agar plates exposure                  mrp-5 F22B5.4. Although HRG-4 been <br /> dsRNA produced E. coli grown presence 5 mM               implicated intestinal heme transport C. elegans, function <br /> or 25 mM heme NGM agar plates. These experiments                     attributed F22B5.4 MRP-5 WormBase. <br /> performed duplicate, GFP levels patterns worms fed             Membrane topology algorithms predicted that, unlike F22B5.4, <br /> bacteria expressing 288 hrgs analyzed eye.               predicted contain single TMD, HRG-4 MRP-5 <br /> RNAi depletion 288 hrgs resulted identification 32          contain TMD respectively, characteristic feature <br /> genes specifically upregulated downregulated GFP                     membrane transporters. <br /> expression IQ6011 heme-sensor strain vha-                 To correlate intestinal ZnMP uptake studies membrane <br /> 6::gfp control strain. These 32 genes selected              transport, examined gene expression pattern hrg-4 and <br /> analysis COPAS BioSort. We identified hrgs                  mrp-5. We generated transgenic worms expressed hrg-4::gfp <br /> caused two-fold increase two-fold decrease GFP              mrp-5::gfp transcriptional fusions. hrg-4::gfp expressed <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                              9                           July 2010 | Volume 6 | Issue 7 | e1001044 <br />                                                                                                      Genomic Specification Heme C. elegans <br />  <br />  <br />  <br />  <br /> Figure 7. Functional validation heme-responsive genes. (A) RNAi depletion transgenic heme-sensor strain. GFP quantification by <br /> COPAS BioSort strain IQ6011 fed HT115(DE3) bacteria grown 5 mM heme induced synthesize dsRNA. RNAi 288 hrgs identified genes <br />  <br />  <br />       PLoS Genetics | www.plosgenetics.org                            10                           July 2010 | Volume 6 | Issue 7 | e1001044 <br />                                                                                                             Genomic Specification Heme C. elegans <br />  <br />  <br />  <br /> which caused 2-fold increase 2-fold decrease GFP levels. Each data point represents mean 6 SEM. Y-axis represents average GFP values of <br /> all gravid worms ($30 worms well) duplicate wells normalized value GFP worms fed bacteria transformed <br /> with vector. (B) RNAi depletion presence toxic heme analog. Forty-one candidate genes encoding proteins TMD were <br /> screened feeding RNAi strain IQ6011 assessing survival progeny presence toxic heme analog GaPP. Both total number <br /> of eggs number viable larvae counted 5 days exposure 1.5 mM GaPP described Materials Methods. Each data <br /> point represents mean 6 SEM separate experiments depicted percentage survival compared control plates GaPP. <br /> Knockdown 7 genes (shown) resulted animal survival presence GaPP. Asterisk indicates genes altered GFP levels strain <br /> IQ6011 worms (Figure 7A). (C) Summary overlap genes identified (A,B). Six 288 hrgs identified interfering heme <br /> homeostasis, evidenced altered GFP levels 5 mM. Seven 41 hrgs encode proteins putative TMD protect GaPP <br /> toxicity, evidenced increase progeny survival. When knocked RNAi, hrgs alter ability sense heme and <br /> protect GaPP toxicity. <br /> doi:10.1371/journal.pgen.1001044.g007 <br />  <br /> specifically intestinal cells larvae adults (Figure 8C),           human homologs suggests study provide genetic <br /> and regulated exogenous heme (not shown). Unlike hrg-                     insights mammalian heme regulation. This underscored by <br /> 4::gfp, mrp-5::gfp expressed worm                presence human homologs genes positive in <br /> tissues examined. Altogether, studies identify HRG-4                   functional RNAi screen. Indeed, recent studies using C. elegans <br /> MRP-5 membrane transporters essential intestinal                 model led identification HRG-1 the <br /> heme homeostasis C. elegans.                                                  bona fide metazoan heme importer conserved in <br />                                                                                  vertebrates [12]. <br /> Discussion                                                                          Analysis presumptive promoters 288 hrgs eight <br />                                                                                  categories identified common cis elements [28]. A detailed <br />    A major impediment identification heme uptake                   analysis 67 genes Category 2, hrg-1 hrg-4 <br /> transport pathways inability disassociate tightly            belong, overrepresented transcription factor binding sites <br /> regulated process heme synthesis downstream                          using sequences control set random promoter <br /> pathways heme transport [3]. C. elegans unique                  sequences. These silico results corroborate experimental <br /> model organisms does synthesize heme but, instead,                studies (Chen, Sinclair, Hamza, unpublished results) and <br /> relies solely exogenous heme normal growth                            support concept regulation organismal heme <br /> development [11]. Thus, worms allow study heme                            homeostasis complex, multi-tiered, effected diverse cellular <br /> homeostatic mechanisms response fluctuations environ-                   modulators. <br /> mental heme context intact, live animal.                           Studies demonstrated infectivity hookworms, <br />    Our study genome-wide transcriptional changes associated               feed blood host, significantly lower severely <br /> with heme availability represents, best knowledge,             anemic hamsters fed low-iron diet [13]. Furthermore, filarial <br /> first study <span id='am-2' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span>-gene interactions C. elegans exploiting axenic        nematodes, causative agent elephantiasis, harbor <br /> liquid growth medium. The mCeHR-2 medium permits fine                            Wolbachia &#226;&#8364;&#8220; intracellular bacterial symbiont contains the <br /> control organismal heme levels function heme                   intact heme biosynthesis pathway [14,36]. Thus, nematodes may <br /> growth medium, allowing identify 288 heme-responsive genes (hrgs).         adapted heme auxotrophy evolving pathways acquire <br /> Some genes identified predictable,                   heme host (extracellular) symbiotic <br /> encode known heme-binding proteins permeases                       relationship bacteria (intracellular). This auxotrophy be <br /> transport small molecules. Other genes sense                    exploited develop drugs block parasite-specific heme uptake <br /> retrospect, glutathione transferases (GST). A recent                     utilization. Indeed, genome database searches heme <br /> proteomic analysis C. elegans identified GST-19 highly                   auxotroph parasites led identify 12 hrg homologs protozoans <br /> abundant protein proposed sequester heme                        62 hrgs clade V nematodes. This finding significant because <br /> intracellular heme excess [29]. GSTs shown               genes encode proteins involved heme uptake and <br /> bind heme helminths hookworms Barber pole worms                   sequestration parasitized host. Further studies aimed at <br /> [30&#226;&#8364;&#8220;32]. Our genomic analysis indicates gst-22 gst-16              elucidating role hrgs heme metabolism validate <br /> upregulated 500 mM heme. Whether GST proteins                      novel anti-parasitic drug targets. <br /> bind heme remains determined.                                                 We seven 41 hrgs encode proteins <br />    GO KEGG pathway analyses reveal hrgs represent                   contain putative TMDs showed different levels resistance <br /> full spectrum biological processes. Interestingly, hrgs            GaPP toxicity. Among heme permease <br /> are enzymes proteins known bind heme. We                          (HRG-4), ABC transporters (PGP-1 MRP-5), Major <br /> speculate transcriptional regulation heme primarily                  Facilitator Superfamily transporters (HMIT-1.1 Y37A1A.2). <br /> targets cellular pathways involved heme homeostasis,                      The remaining 247 hrgs encoded proteins predicted <br /> including uptake sequestration, genes                  TMDs. These proteins encode soluble effectors heme <br /> encode target hemoproteins. The vast majority hrgs                    transport chaperones sequestering proteins. In support <br /> known function and, therefore, biological                        concept, cellular iron stored ferritin, cytosolic multi- <br /> processes pathways attributed them. Furthermore, pheno-                    subunit protein; cytoplasmic copper delivered membrane <br /> types RNAi studies involving 288 hrgs reported growth                   bound P-type ATPases secretory pathway copper <br /> and developmental defects, plausibly disruption heme                  chaperone Atox1 [37,38]. We propose similar network for <br /> homeostasis affect hemoprotein function diverse biological               trafficking intracellular heme maintaining homeostasis likely <br /> pathways ranging miRNA processing (DGCR8) gas                            exist C. elegans metazoa [3]. <br /> sensing (soluble guanylyl cyclases) circadian clock control (Rev-                Interestingly, HRG-4, MRP-5, F22B5.4 only <br /> erba) [33&#226;&#8364;&#8220;35].                                                                   positive candidates identified heme-sensor GaPP <br />    The 288 hrgs identified provide insight                functional RNAi screens. RNAi studies implicated HRG-4 as <br /> metazoan heme regulation. The fact .40% hrgs                        heme transporter C. elegans intestine [12], the <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                                 11                             July 2010 | Volume 6 | Issue 7 | e1001044 <br />                                                                                                     Genomic Specification Heme C. elegans <br />  <br />  <br />  <br />  <br /> Figure 8. Characterization candidate genes identified functional RNAi screens. (A) Heme-dependent expression. <br /> qRT-PCR performed triplicate hrg-4, mrp-5, F22B5.4 indicated heme concentrations, fold change (mean 6 SEM) is <br /> reported gene concentration heme. (B) Accumulation ZnMP worms depleted candidate genes RNAi. Synchronized wild- <br />  <br />  <br />       PLoS Genetics | www.plosgenetics.org                            12                           July 2010 | Volume 6 | Issue 7 | e1001044 <br />                                                                                                            Genomic Specification Heme C. elegans <br />  <br />  <br />  <br /> type strain worms L1 stage transferred mCeHR-2 medium containing 2 mM heme plates seeded lawn bacteria producing <br /> dsRNA vector, hrg-4, mrp-5, F22B5.4 allowed develop &lt;72 h late L4 stage. At point, worms incubated mCeHR- <br /> 2 medium containing 5 mM ZnMP overnight. Images region worm intestine captured using Leica DMIRE2 microscope equipped a <br /> Rhodamine filter CCD camera. (C) Expression hrg-4 mrp-5 GFP transcriptional reporters. Transgenic worms expressing GFP the <br /> control 3 kb hrg-4 (bottom) mrp-5 (top) putative promoters. Transgenic worms grown mCeHR-2 medium containing 4 mM heme <br /> for generation. Representative images worms obtained Leica DMIRE2 microscope using CCD camera. <br /> doi:10.1371/journal.pgen.1001044.g008 <br />  <br /> function MRP-5 protein encoded F22B5.4                         gravid adults M9 buffer containing 4, 20, 500 mM hemin. <br /> currently unknown. We speculate MRP-5, member                        Equal numbers F2 larvae L1 stage inoculated in <br /> family membrane effluxers [39], export heme                      mCeHR-2 medium supplemented 4, 20, 500 mM hemin. <br /> intestinal cells extra-intestinal cells. These results consistent,        The F2 worms allowed develop late L4 stage, <br /> in part, ubiquitous expression mrp-5::gfp worm                    harvested, flash frozen liquid nitrogen, stored 280uC. <br /> tissues, RNAi studies mrp-5 depletion               Frozen worm pellets ground fine powder, total <br /> results accumulation ZnMP worm intestine                        RNA extracted using Trizol (Invitrogen, Carlsbad, CA). RNA <br /> resistance GaPP toxicity. Unlike HRG-4 MRP-5                    obtained subjected RNase-free DNase treatment for <br /> transporters multiple TMD, F22B5.4 encodes predicted                      1 h 37uC purified using RNeasy kit (Qiagen, <br /> Type II membrane protein single TMD. Although                         Germantown, MD). Total RNA biological replicates <br /> results clearly implicate role F22B5.4 essential                     used make cDNA, hybridized C. elegans <br /> component heme homeostasis C. elegans, unclear                   Whole Genome Arrays (Affymetrix, Santa Clara, CA). <br /> this protein function heme homeostasis. Excitingly, <br /> microarray RNAi studies identified F22B5.4 gene                    cDNA synthesis quantitative real-time PCR <br /> is highly upregulated hypoxia-inducible factor (HIF)                         First strand cDNA synthesized using 2 mg total RNA <br /> transcription complex, master regulator hypoxia response                    using Superscript II First Strand cDNA synthesis kit (Invitrogen). <br /> [40&#226;&#8364;&#8220;42]. HIF regulated degradation hydroxylation                   For quantitative real-time PCR (qRT-PCR), primers spanning at <br /> of proline residues, process requires presence                    intron designed using Primer Express (Applied <br /> oxygen, 2-oxoglutarate, iron [43]. Given dependence C.                Biosystems) Beacon designer 4 (Premier Biosoft) programs. <br /> elegans heme oxygen binding sensing [44&#226;&#8364;&#8220;46]                  PCR performed using iCycler iQ Real-time PCR <br /> nutritional source iron [11], conceivable F22B5.4              Detection System (BioRad) 0.12 U/ml Taq DNA polymerase, <br /> play important role coordinating heme transport                        40 nM fluorescein (Invitrogen), SYBR Green I Nucleic Acid <br /> availability oxygen metabolism.                                             Gel Stain (Invitrogen) diluted 1:10. The PCR amplification was <br />    In current study identified novel catalog genes              run 40 cycles. The PCR products 150 and <br /> that responsive heme C. elegans. Although unclear                200 bp length. Quality PCR products determined by <br /> mechanistically worms respond heme mRNA level,                   dissociation curve analysis gel electrophoresis. Each experi- <br /> thorough study identify cis regulatory elements                   ment performed triplicate. Average CT values used for <br /> corresponding trans acting factors significantly accelerate             22DDCt calculations relative fold changes gene expression <br /> understanding C. elegans adapts environmental                      [49]. <br /> nutritional changes. Using facile genetically tractable C. <br /> elegans model system, RNAi screen hrg mini-library              Microarray data analysis <br /> be easily adapted genome screens identify regulatory                   Expression data normalized analyzed using MAS 5.0 <br /> pathways govern metazoans sense respond                         suite software (Affymetrix). Data worms grown mCeHR-2 <br /> heme organismal level.                                                     medium 4 500 mM hemin compared data from <br />                                                                                  worms grown medium containing 20 mM hemin (baseline <br /> Materials Methods                                                            samples). Microarray data verified Robust Multichip <br />                                                                                  Average Method (RMA, R package). Quantile normalization and <br /> Biological materials, strains, worm culture                                  background corrections performed using perfect match <br />    C. elegans wild-type N2 strain worms grown                  probe intensities. Using initial cut-off $1.2-fold change in <br /> axenic liquid mCeHR-2 medium [47] NGM agar plates                          mRNA expression RMA $1.6-fold change MAS 5.0 <br /> spotted E. coli OP50 HT115(DE3) strains [48].                            resulted identification 370 genes. Increasing the <br /> Synchronized, L1 larvae obtained bleaching P0 gravid                     stringency $1.6-fold change RMA MAS 5.0 <br /> worms grown mCeHR-2 medium supplemented hemin                            reduced number genes identified heme responsive 288 <br /> chloride [11]. Hemin chloride gallium protoporphyrin IX                      genes. To identify putative human orthologs, worm protein <br /> were purchased Frontier Scientific, Inc (Logan, UT).                        sequences used query human genome databases NCBI <br /> Plasmids cloning injecting worms                   reciprocal BLAST analysis E-value cut-off $1024. <br /> Fire Vector Kit (Addgene, Cambridge, MA). Primers designed                    Sequences 288 genes obtained from <br /> PCR amplify worm open reading frames based                               WormBase analyzed topology (TMHMM 2.0, <br /> Wormbase predictions ordered IDT (Coralville, IA).                      SOSUI), motifs (ELM, BLOCKS, Pfam), pathway classifica- <br /> The PCR products TA cloned L4440 plasmid.                          tion (GO KEGG). <br />  <br /> Total RNA isolation                                                              Generation hrg mini-library <br />   Equal numbers F1 larvae L1 stage inoculated                      The Ahringer Vidal feeding libraries replicated to <br /> mCeHR-2 medium 4, 20, 500 mM hemin chloride                          individual 96-well plates [50,51]. Thirty-four clones initial <br /> grown gentle shaking 20uC. Synchronized, F2 larvae                list 370 hrgs absent libraries. To complete hrg <br /> L1 stage obtained hatching eggs obtained F1                     mini-library, PCR amplified missing genes N2 worm <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                                 13                            July 2010 | Volume 6 | Issue 7 | e1001044 <br />                                                                                                        Genomic Specification Heme C. elegans <br />  <br />  <br /> genomic DNA cloned PCR fragments TA cloning                  COPAS BioSort detects low levels GFP worms. <br /> the RNAi feeding vector pL4440. Only 19 34 RNAi clones                The mean values sample determined, the <br /> were final list 288 hrgs. DNA 288 hrgs                 average duplicate calculated. This mean was <br /> sequenced confirm authenticity.                                           normalized average value GFP obtained the <br />                                                                              vector-only sample, reported arbitrary units 6 SEM for <br /> Analysis RNAi GFP expression IQ6011                         clone analyzed. <br /> vha-6::gfp strains <br />    NGM agar plates containing IPTG, carbenicillin,                       GaPP toxicity assays <br /> tetracycline seeded HT115(DE3) bacteria expressing                    Synchronized, F1 wild-type worms L1 larval stage were <br /> double-stranded RNA (dsRNA) clone hrg                    obtained P0 worms grown mCeHR-2 containing 1.5 mM <br /> mini-library. Duplicate bacterial cultures clone            hemin. Equal numbers F1 worms placed NGM agar <br /> grown 5.5 h LB containing carbenicillin tetracycline              plates containing 2 mM IPTG, 50 mg/mL carbenicillin, 12 mg/mL <br /> and 5 mM 25 mM heme. Plates seeded lawn                    IPTG plated lawn HT115(DE3) RNAi feeding bacteria <br /> bacteria dsRNA induction occurred &lt;20 h room                      harboring respective L4440 plasmid grown LB <br /> temperature. Subsequently, L1 larvae gravid IQ6011                broth carbenicillin tetracycline [12]. Worms fed the <br /> worms grown liquid media supplemented                 RNAi bacteria &lt;60 h allowed develop late L4 stage. <br /> 10 mM heme added 12-well plates. Each               At point, worms transferred fresh RNAi plates containing <br /> 12-well plate 10 wells seeded experimental clones               1.5 mM GaPP. Worms developed gravid stage laid eggs. <br /> one seeded control clones &#226;&#8364;&#8220; vector hrg-4.          After 24 h egg-laying, P0 worms (all gravid stage) were <br /> The plates incubated 15uC overnight incubated               discarded order prevent additional eggs laid. On day <br /> at 20uC additional days. The GFP levels gravid adults           5, total number surviving larvae number of <br /> were observed visually using Leica Microsystems MZ16FA                     unhatched eggs counted. P values statistical significance were <br /> stereoscope. The intensity pattern GFP gravid worms                calculated using one-way ANOVA Student&#226;&#8364;&#8220;Newman&#226;&#8364;&#8220; <br /> feeding bacteria producing dsRNA hrg                     Keuls multiple comparisons test using GraphPad InStat v. 3.06 <br /> compared intensity pattern GFP same-stage                   (GraphPad, San Diego, CA). <br /> worms feeding bacteria transformed vector. <br /> Worms displayed altered GFP replicates                     ZnMP uptake assays <br /> designated potential modulators. Potential modulators                   Equal numbers synchronized N2 L1 larvae obtained P0 <br /> screened strain produced GFP control                worms grown mCeHR-2 plus 2 mM hemin exposed the <br /> promoter responsive heme (vha-6::gfp). Any clone             RNAi bacteria NGM plates containing 2 mM IPTG 72 h. <br /> that altered GFP levels vha-6::gfp strain worms removed           This followed exposure 5 mM ZnMP plus 1.5 mM hemin <br /> from list modulators, change GFP              chloride 16 h mCeHR-2 medium. ZnMP fluorescence <br /> response heme.                                                            intensity measured described previously [12]. <br />  <br /> GFP quantification IQ6011 strain                                          Generation GFP reporter constructs <br />    A COPAS BioSort worm sorter (Union Biometrica, Holliston,                    GFP reporter fusion constructs created using Gateway <br /> MA) used measure GFP levels live worms. Plates,                    cloning (Invitrogen, Frederick, MD). The promoter of <br /> bacteria, worms prepared treated described            interest, gfp gene, 39 untranslated region unc-54 gene <br /> previous section. After 84 h RNAi plates, P0 gravid F1 L1-            cloned recombination entry vectors pDONR P4- <br /> stage worms washed 600 mL M9 <br />                                                                              P1R, pDONR 221, pDONR P2R-P3, respectively, using the <br /> buffer containing 0.01% Tween-20, transferred 1.5-mL <br />                                                                              Gateway BP Clonase kit. Sequence verified entry clones then <br /> microcentrifuge tube, allowed settle 5 min. The <br />                                                                              recombined destination vector pDEST R4-R3 using the <br /> supernatant removed discarded. Each worm pellet was <br />                                                                              Gateway LR Clonase II plus enzyme kit produce final <br /> transferred individual 96-well plate. Duplicate <br />                                                                              recombinant plasmid. <br /> samples transferred successive wells 96-well plate <br /> and separated samples well, which <br /> served flush flow cell worms analyzed prevent           Production transgenic worm strains <br /> contamination subsequent samples. The contents                  For microparticle bombardment, &lt;56106 unc-119 (ed3) gravid <br /> were washed, aspirated, analyzed COPAS BioSort worm                 worms co-bombarded 10 mg Gateway reporter <br /> sorter. The GFP gain set 2, GFP PMT setting               construct 5 mg unc-119 rescue plasmid (pDM016B) using <br /> 400. Using highly synchronized worms gravid stage,             PDS-1000 particle delivery (Bio-Rad, Hercules, CA). <br /> previously defined gate settings order ensure data        Worms washed bombardment plates transferred to <br /> obtained P0 gravid animals easily quickly                  plates seeded lawn E. coli strain JM109. After two-weeks <br /> separated data obtained worms                         25uC, multiple wild-type F2 worms screened gene <br /> developmental stages. Text file data imported Microsoft             integration PCR transgene expression. Individual <br /> Excel sorted based gating parameters recorded              transgenic lines isolated transferred axenic liquid <br /> &#226;&#8364;&#732;&#226;&#8364;&#732;Status Select&#226;&#8364;&#8482;&#226;&#8364;&#8482; column. The worm sorter records fluorescence             mCeHR-2 medium supplemented antibiotics. After two <br /> profile worm form curve, reflects              weeks serial passages, worms bleached maintained as <br /> intensity GFP mouth tail. The &#226;&#8364;&#732;&#226;&#8364;&#732;Green&#226;&#8364;&#8482;&#226;&#8364;&#8482; column            transgenic strains axenic liquid mCeHR-2 medium. <br /> recorded GFP value area curve, reduced a <br /> factor 40,000. The background levels GFP subtracted               Gene Expression Omnibus information <br /> from values used generate Figure 7A. The background level               The microarray data submitted GEO Aug 6, 2007. <br /> of GFP equal GFP levels IQ6011 worms feeding                The GEO accession number GSE8696 available http:// <br /> HT115(DE3) bacteria transformed gfp RNAi vector. The                www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc = GSE8696. <br />  <br />  <br />       PLoS Genetics | www.plosgenetics.org                              14                            July 2010 | Volume 6 | Issue 7 | e1001044 <br />                                                                                                        Genomic Specification Heme C. elegans <br />  <br />  <br />  <br /> Supporting Information                                                       categories designed expression gene <br />                                                                              increased, decreased, did change given heme <br /> Figure S1 Heat map heme microarrays. A compilation                   concentration compared 20 mM control. <br /> of heat maps generated following normalization data (see              Found at: doi:10.1371/journal.pgen.1001044.s006 (1.15 MB TIF) <br /> Figure 2 legend) using GeneSpring (v7.2) category the <br /> 288 hrgs data chips represented. The               Table S5 Heme-responsive genes known Gene Ontology <br /> down arrows indicate upregulation downregulation 4                  terms. Of 288 hrgs expression changed significantly in <br /> 500 mM heme compared 20 mM heme. Yellow represents                   response heme, results gene ontology analysis used <br /> no change signal intensity, blue indicates decrease, red            assign known biological process molecular function 63 <br /> indicates increase signal intensity. The data           genes. <br /> replicate sample 4 mM heme, indicated asterisk           Found at: doi:10.1371/journal.pgen.1001044.s007 (1.02 MB TIF) <br /> top column, inconsistent data            Table S6 Gene Ontology analysis heme-responsive genes <br /> two biological replicates determined principal                    upregulated 4 mM heme. Each GO ID assigned unique <br /> components analysis K-means clustering data.                      function association. Both listed here, GO ID <br /> Found at: doi:10.1371/journal.pgen.1001044.s001 (2.43 MB TIF)                used GO analysis figure. Green shading indicates <br /> Figure S2 Gene ontology (GO) enrichment analysis heme-                    term included corresponding GO enrichment <br /> responsive genes. (A) hrgs downregulated 4 mM heme. (B) hrgs              figures. <br /> upregulated 500 mM heme. (C) hrgs downregulated 500 mM                 Found at: doi:10.1371/journal.pgen.1001044.s008 (0.06 MB <br /> heme. Of 288 hrgs identified study, 115 annotated            PDF) <br /> with biological process. Genes analyzed using Fisher&#226;&#8364;&#8482;s            Table S7 Gene Ontology analysis heme-responsive genes <br /> exact test topGO package R. The significant                downregulated 4 mM heme. Each GO ID assigned unique <br /> GO terms associated parent terms used construct            function association. Both listed here, GO ID was <br /> a hierarchical graph specificity terms increased        used GO analysis figure. Green shading indicates that <br /> as moved bottom. The text rectangle                   term included corresponding GO enrichment figures. <br /> provides GO ID ratio number genes                      Found at: doi:10.1371/journal.pgen.1001044.s009 (0.06 MB <br /> annotated GO term tested subset total         PDF) <br /> gene set. The shade green rectangle corresponds the <br /> significance GO result. Full GO terms provided solely             Table S8 Gene Ontology analysis heme-responsive genes <br /> for genes P,0.005. The complete table P-values            upregulated 500 mM heme. Each GO ID assigned unique <br /> description GO term associated gene                  function association. Both listed here, GO ID <br /> found Tables S7, S8, S9.                                              used GO analysis figure. Green shading indicates <br /> Found at: doi:10.1371/journal.pgen.1001044.s002 (0.08 MB PDF)                term included corresponding GO enrichment <br />                                                                              figures. <br /> Table S1 The 288 heme-responsive genes identified the <br />                                                                              Found at: doi:10.1371/journal.pgen.1001044.s010 (0.09 MB <br /> microarray. Data collected using Affymetrix C. elegans               PDF) <br /> whole genome array analyzed Affymetrix MAS 5.0 <br /> software RMA. Each entry table represents gene                  Table S9 Gene Ontology analysis heme-responsive genes <br /> whose expression changed 1.6 fold             downregulated 500 mM heme. Each GO ID assigned a <br /> experimental heme concentrations. The table columns              unique function association. Both listed here, the <br /> each hrg. The &#226;&#8364;&#732;&#226;&#8364;&#732;Description&#226;&#8364;&#8482;&#226;&#8364;&#8482; column lists unique Gene ID                GO ID used GO analysis figure. Green shading <br /> assigned Wormbase gene C. elegans genome. The             indicates term included corresponding GO <br /> &#226;&#8364;&#732;&#226;&#8364;&#732;Gene name&#226;&#8364;&#8482;&#226;&#8364;&#8482; column provides gene,               enrichment figures. <br /> been assigned. The &#226;&#8364;&#732;&#226;&#8364;&#732;4 mM&#226;&#8364;&#8482;&#226;&#8364;&#8482; column gives value              Found at: doi:10.1371/journal.pgen.1001044.s011 (0.09 MB <br /> change expression gene, second &#226;&#8364;&#732;&#226;&#8364;&#732;4 mM&#226;&#8364;&#8482;&#226;&#8364;&#8482;                   PDF) <br /> column indicates gene expression increased (up) <br />                                                                              Table S10 Heme-responsive genes assigned biological <br /> or decreased (down). If column blank, change was <br />                                                                              pathway KEGG analysis. The algorithms available the <br /> less 1.6-fold. The pattern &#226;&#8364;&#732;&#226;&#8364;&#732;500 mM&#226;&#8364;&#8482;&#226;&#8364;&#8482; columns the <br />                                                                              Kyoto Encyclopedia Genes Genomes website used to <br /> same &#226;&#8364;&#732;&#226;&#8364;&#732;4 mM&#226;&#8364;&#8482;&#226;&#8364;&#8482; columns. <br />                                                                              make functional predictions 288 hrgs identified the <br /> Found at: doi:10.1371/journal.pgen.1001044.s003 (0.10 MB PDF)                microarray. Ten hrgs mapped KEGG pathways. <br /> Table S2 Heme-responsive genes expression upregu-                   Found at: doi:10.1371/journal.pgen.1001044.s012 (0.59 MB TIF) <br /> lated greater 1.6 fold worms grown 4 mM heme. The <br />                                                                              Table S11 Previously reported RNAi phenotypes heme- <br /> gene ID (description), gene name, change 4 mM <br />                                                                              responsive genes. Phenotypes observed hrgs knocked <br /> compared control (20 mM) provided gene <br />                                                                              experiments performed laboratories and <br /> whose expression increased 4 mM. <br />                                                                              compiled Wormbase. <br /> Found at: doi:10.1371/journal.pgen.1001044.s004 (0.08 MB PDF) <br />                                                                              Found at: doi:10.1371/journal.pgen.1001044.s013 (0.85 MB TIF) <br /> Table S3 Heme-responsive genes expression upregu- <br />                                                                              Table S12 Heme-responsive genes predicted TMDs. Worm <br /> lated greater 1.6 fold worms grown 500 mM <br />                                                                              protein sequences obtained Wormbase analyzed using <br /> heme. The gene ID (description), gene name, of <br />                                                                              TMHMM 2.0 SOSUI identify 41 proteins putative <br /> change 500 mM compared control (20 mM) provided <br />                                                                              hydrophobic membrane-spanning domains (TMDs). The 41 genes <br /> for gene expression increased 500 mM. <br />                                                                              putative TMDs arranged according number of <br /> Found at: doi:10.1371/journal.pgen.1001044.s005 (0.73 MB TIF) <br />                                                                              TMDs. The change levels gene expression 4 500 mM <br /> Table S4 Heme-responsive genes used corroborate                       heme indicated. Negative fold change implies regulation. <br /> microarray results. Three genes selected               Found at: doi:10.1371/journal.pgen.1001044.s014 (0.74 MB TIF) <br />  <br />  <br />       PLoS Genetics | www.plosgenetics.org                              15                            July 2010 | Volume 6 | Issue 7 | e1001044 <br />                                                                                                                               Genomic Specification Heme C. elegans <br />  <br />  <br />  <br /> Acknowledgments                                                                               Author Contributions <br /> We thank Tamika Samuel help ZnMP studies, George Poy                         Conceived designed experiments: SS AR AUR IH. Performed the <br /> expertise microarrays, Jonathan Walston helping construct hrg                    experiments: SS AR AUR GCC NMES IH. Analyzed data: SS AR <br /> mini-library, Caiyong Chen Diana Newsome qRT&#226;&#8364;&#8220;PCR analysis,                            AUR GCC MM NMES MK IH. Contributed reagents/materials/analysis <br /> Apratim Mitra Jiuzhou Song GO analysis, John Hanover                          tools: GCC MM NMES MK IH. Wrote paper: SS MM NMES MK <br /> for access COPAS worm sorter. Worm strains provided                        IH. <br /> Caenorhabditis Genetics Center. <br />  <br /> References <br />  1. Uzel C, Conrad ME (1998) Absorption heme iron. Semin Hematol 35: 27&#226;&#8364;&#8220;34.                27. Thomas JH (2006) Analysis <span id='am-164' about='Thesaurus:Homologous_Gene' typeof='owl:Thing'>homologous gene</span> clusters Caenorhabditis <br />  2. West AR, Oates PS (2008) Mechanisms heme iron absorption: Current                          elegans reveals striking regional cluster domains. Genetics 172: 127&#226;&#8364;&#8220;143. <br />     questions controversies. World J Gastroenterol 14: 4101&#226;&#8364;&#8220;4110.                         28. Matys V, Kel-Margoulis OV, Fricke E, Liebich I, Land S, et al. (2006) <br />  3. Severance S, Hamza I (2009) Trafficking Heme Porphyrins Metazoa.                    TRANSFAC module TRANSCompel: transcriptional gene regulation in <br />     Chem Rev.                                                                                     eukaryotes. Nucleic Acids Res 34: D108&#226;&#8364;&#8220;110. <br />  4. Hamza I (2006) Intracellular trafficking porphyrins. ACS Chem Biol 1:                  29. Perally S, Lacourse EJ, Campbell AM, Brophy PM (2008) Heme Transport and <br />     627&#226;&#8364;&#8220;629.                                                                                      Detoxification Nematodes: Subproteomics Evidence Differential Role of <br />  5. Stojiljkovic I, Perkins-Balding D (2002) Processing heme heme-containing               Glutathione Transferases. J Proteome Res. <br />     proteins bacteria. DNA Cell Biol 21: 281&#226;&#8364;&#8220;295.                                          30. Schuller DJ, Liu Q, Kriksunov IA, Campbell AM, Barrett J, et al. (2005) Crystal <br />  6. Weissman Z, Kornitzer D (2004) A family Candida cell surface haem-binding                  structure new class glutathione transferase model human <br />     proteins involved haemin haemoglobin-iron utilization. Mol Microbiol                   hookworm nematode Heligmosomoides polygyrus. Proteins 61: 1024&#226;&#8364;&#8220;1031. <br />     53: 1209&#226;&#8364;&#8220;1220.                                                                            31. van Rossum AJ, Jefferies JR, Rijsewijk FA, LaCourse EJ, Teesdale-Spittle P, <br />  7. Protchenko O, Rodriguez-Suarez R, Androphy R, Bussey H, Philpott CC (2006)                    et al. (2004) Binding hematin new class glutathione transferase from <br />     A screen genes heme uptake identifies FLC family required import               blood-feeding parasitic nematode Haemonchus contortus. Infect Immun 72: <br />     FAD endoplasmic reticulum. J Biol Chem 281: 21445&#226;&#8364;&#8220;21457.                          2780&#226;&#8364;&#8220;2790. <br />  8. Protchenko O, Shakoury-Elizeh M, Keane P, Storey J, Androphy R, et al. (2008)             32. Zhan B, Liu S, Perally S, Xue J, Fujiwara R, et al. (2005) Biochemical <br />     Role PUG1 inducible porphyrin heme transport Saccharomyces                       characterization vaccine potential heme-binding glutathione transferase <br />     cerevisiae. Eukaryot Cell 7: 859&#226;&#8364;&#8220;871.                                                         adult hookworm Ancylostoma caninum. Infect Immun 73: 6903&#226;&#8364;&#8220;6911. <br />  9. Keel SB, Doty RT, Yang Z, Quigley JG, Chen J, et al. (2008) A heme export                 33. Faller M, Matsunaga M, Yin S, Loo JA, Guo F (2007) Heme involved in <br />     protein required red blood cell differentiation iron homeostasis.                  microRNA processing. Nat Struct Mol Biol 14: 23&#226;&#8364;&#8220;29. <br />     Science 319: 825&#226;&#8364;&#8220;828.                                                                     34. Yin L, Wu N, Curtin JC, Qatanani M, Szwergold NR, et al. (2007) Rev-erba, a <br /> 10. Quigley JG, Yang Z, Worthington MT, Phillips JD, Sabo KM, et al. (2004)                       Heme Sensor That Coordinates Metabolic Circadian Pathways. Science <br />     Identification human heme exporter essential erythropoiesis. Cell            318: 1786&#226;&#8364;&#8220;1789. <br />     118: 757&#226;&#8364;&#8220;766.                                                                             35. Schmidt PM, Rothkegel C, Wunder F, Schroder H, Stasch JP (2005) Residues <br /> 11. Rao AU, Carta LK, Lesuisse E, Hamza I (2005) Lack heme synthesis                      stabilizing heme moiety nitric oxide sensor soluble guanylate cyclase. <br />     free-living eukaryote. Proc Natl Acad Sci U S A 102: 4270&#226;&#8364;&#8220;4275.                               Eur J Pharmacol 513: 67&#226;&#8364;&#8220;74. <br />                                                                                               36. Foster J, Ganatra M, Kamal I, Ware J, Makarova K, et al. (2005) The <br /> 12. Rajagopal A, Rao AU, Amigo J, Tian M, Upadhyay SK, et al. (2008) Haem <br />                                                                                                   Wolbachia genome Brugia malayi: endosymbiont evolution human <br />     homeostasis regulated conserved concerted functions HRG-1 <br />                                                                                                   pathogenic nematode. PLoS Biol 3: e121. doi:10.1371/journal.pbio.0030121. <br />     proteins. Nature 453: 1127&#226;&#8364;&#8220;1131. <br />                                                                                               37. De Domenico I, McVey Ward D, Kaplan J (2008) Regulation iron acquisition <br /> 13. Held MR, Bungiro RD, Harrison LM, Hamza I, Cappello M (2006) Dietary <br />                                                                                                   storage: consequences iron-linked disorders. Nat Rev Mol Cell Biol 9: <br />     iron content mediates hookworm pathogenesis vivo. Infect Immun 74: <br />                                                                                                   72&#226;&#8364;&#8220;81. <br />     289&#226;&#8364;&#8220;295. <br />                                                                                               38. Kim BE, Nevitt T, Thiele DJ (2008) Mechanisms copper acquisition, <br /> 14. Wu B, Novelli J, Foster J, Vaisvila R, Conway L, et al. (2009) The Heme <br />                                                                                                   distribution regulation. Nat Chem Biol 4: 176&#226;&#8364;&#8220;185. <br />     Biosynthetic Pathway Obligate Wolbachia Endosymbiont Brugia <br />                                                                                               39. Borst P, Elferink RO (2002) Mammalian ABC transporters health and <br />     malayi Potential Anti-filarial Drug Target. PLoS Negl Trop Dis 3: e475. <br />                                                                                                   disease. Annu Rev Biochem 71: 537&#226;&#8364;&#8220;592. <br />     doi:10.1371/journal.pntd.0000475. <br />                                                                                               40. Shen C, Nettleton D, Jiang M, Kim SK, Powell-Coffman JA (2005) Roles the <br /> 15. Reinke V, Smith HE, Nance J, Wang J, Van Doren C, et al. (2000) A global                      HIF-1 hypoxia-inducible factor hypoxia response Caenorhabditis <br />     profile germline gene expression C. elegans. Mol Cell 6: 605&#226;&#8364;&#8220;616.                       elegans. J Biol Chem 280: 20580&#226;&#8364;&#8220;20588. <br /> 16. El-Sayed NM, Myler PJ, Bartholomeu DC, Nilsson D, Aggarwal G, et al. (2005)               41. Shao Z, Zhang Y, Powell-Coffman JA (2009) Two distinct roles EGL-9 the <br />     The genome sequence Trypanosoma cruzi, etiologic agent Chagas disease.                  regulation HIF-1-mediated gene expression Caenorhabditis elegans. <br />     Science 309: 409&#226;&#8364;&#8220;415.                                                                         Genetics 183: 821&#226;&#8364;&#8220;829. <br /> 17. Sah JF, Ito H, Kolli BK, Peterson DA, Sassa S, et al. (2002) Genetic rescue            42. Gort EH, van Haaften G, Verlaan I, Groot AJ, Plasterk RH, et al. (2008) The <br />     Leishmania deficiency porphyrin biosynthesis creates mutants suitable                  TWIST1 oncogene direct target hypoxia-inducible factor-2alpha. <br />     analysis cellular events uroporphyria photodynamic therapy. J Biol              Oncogene 27: 1501&#226;&#8364;&#8220;1510. <br />     Chem 277: 14902&#226;&#8364;&#8220;14909.                                                                    43. Kaelin WG, Jr., Ratcliffe PJ (2008) Oxygen sensing metazoans: central <br /> 18. Abad P, Gouzy J, Aury JM, Castagnone-Sereno P, Danchin EG, et al. (2008)                      role HIF hydroxylase pathway. Mol Cell 30: 393&#226;&#8364;&#8220;402. <br />     Genome sequence metazoan plant-parasitic nematode Meloidogyne                      44. Gray JM, Karow DS, Lu H, Chang AJ, Chang JS, et al. (2004) Oxygen <br />     incognita. Nat Biotechnol 26: 909&#226;&#8364;&#8220;915.                                                        sensation social feeding mediated C. elegans guanylate cyclase <br /> 19. Abubucker S, Martin J, Yin Y, Fulton L, Yang SP, et al. (2008) The canine                     homologue. Nature 430: 317&#226;&#8364;&#8220;322. <br />     hookworm genome: analysis classification Ancylostoma caninum survey                45. McGrath PT, Rockman MV, Zimmer M, Jang H, Macosko EZ, et al. (2009) <br />     sequences. Mol Biochem Parasitol 157: 187&#226;&#8364;&#8220;192.                                                Quantitative mapping digenic behavioral trait implicates globin variation in <br /> 20. Ghedin E, Wang S, Spiro D, Caler E, Zhao Q, et al. (2007) Draft genome                 C. elegans sensory behaviors. Neuron 61: 692&#226;&#8364;&#8220;699. <br />     filarial nematode parasite Brugia malayi. Science 317: 1756&#226;&#8364;&#8220;1760.                    <br /> </body></html>