<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                            NIH Public Access <br />                             Author Manuscript <br />                             Mol Genet Metab. Author manuscript; available PMC 2010 April 1. <br />                            Published final edited form as: <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             Mol Genet Metab. 2009 April ; 96(4): 183&#226;&#8364;&#8220;188. doi:10.1016/j.ymgme.2008.12.006. <br />  <br />  <br />  <br />                            N- C-terminal domains human holocarboxylase synthetase <br />                            participate substrate recognition <br />  <br />                            Yousef I. Hassan1, Hideaki Moriyama2, Lars J. Olsen3, Xin Bi3, Janos Zempleni1,4 <br />                            1Department Nutrition &amp; Health Sciences, University Nebraska-Lincoln, NE 68583-0806 <br />  <br />                            2Department       Chemistry, University Nebraska-Lincoln, NE 68588-0304 <br />                            3Department       Biology, University Rochester, NY 14627-0211 <br />  <br />                            <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />                                 Holocarboxylase synthetase (HCS) catalyzes binding vitamin biotin carboxylases and <br />                                 histones. Carboxylases mediate essential steps <span id='am-2' about='Thesaurus:Macronutrient' typeof='owl:Thing'>macronutrient</span> metabolism. For example, propionyl- <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                 CoA carboxylase (PCC) catalyzes carboxylation propionyl-CoA metabolism odd- <br />                                 chain fatty acids. HCS comprises putative domains, i.e., N-terminus, biotin transfer/ATP <br />                                 binding domain, putative linker domain, C-terminus. Both N- C-termini essential <br />                                 biotinylation carboxylases HCS, exact functions domains enzyme <br />                                 catalysis unknown. Here tested hypothesis N- C-termini play roles substrate <br />                                 recognition HCS. Yeast-two-hybrid (Y2H) assays used study interactions the <br />                                 domains human HCS p67, PCC-based polypeptide HCS substrate. Both N- and <br />                                 C-termini interacted p67 Y2H assays, biotin transfer/ATP-binding linker <br />                                 domains did interact p67. The essentiality N- C-termini interactions with <br />                                 carboxylases confirmed rescue experiments mutant Saccharomyces cerevisiae, using <br />                                 constructs truncated human HCS. Finally, computational biology approach used model <br />                                 3D structure human HCS identify amino acid residues interact p67. In silico <br />                                 predictions consistent observations Y2H assays yeast rescue experiments, and <br />                                 suggested docking p67 near Arg508 Ser515 central domain HCS. <br />  <br />  <br />                            Keywords <br />                                 BirA; holocarboxylase synthetase; domains; p67; propionyl-CoA carboxylase <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            Introduction <br />                                               Biotin water-soluble vitamin serves coenzyme acetyl-CoA carboxylases 1 and <br />                                               2 (ACC1 ACC2), 3-methylcrotonyl-CoA carboxylase (MCC), pyruvate carboxylase (PC), <br />                                               propionyl-CoA carboxylase (PCC). These carboxylases play pivotal roles human <br />                                               metabolism [1]. Both ACC1 ACC2 catalyze incorporation bicarbonate malonyl- <br />                                               CoA, key regulatory step fatty acid synthesis (ACC1) mitochondrial fatty acid transport <br />  <br />  <br />                            &#194;&#169; 2008 Elsevier Inc. All rights reserved. <br />                            4Correspondence <span id='am-7' about='protege:TO' typeof='owl:Thing'>to</span>: J. Zempleni, Department Nutrition Health Sciences, University Nebraska Lincoln, 316 Ruth Leverton <br />                            Hall, Lincoln, NE 68583-0806, USA, Fax: + 1 (402) 472 1587, Tel.: + 1 (402) 472 3270, E-mail: jzempleni2@unl.edu <br />                            Publisher&apos;s Disclaimer: This PDF file unedited manuscript accepted publication. As service customers <br />                            providing early version manuscript. The manuscript undergo copyediting, typesetting, review resulting <br />                            proof published final citable form. Please note production process errors discovered could <br />                            affect content, legal disclaimers apply journal pertain. <br />                             Hassan et al.                                                                                               Page 2 <br />  <br />  <br />                                            (ACC2). 3-Methylcrotonyl-CoA carboxylase involved catabolism branched- <br />                                            chain amino acid leucine. Pyruvate carboxylase catalyzes carboxylation pyruvate to <br />                                            produce oxaloacetate gluconeogenesis. Finally, PCC involved conversion of <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            propionyl-CoA methylmalonyl-CoA metabolism odd-chain fatty acids. <br />  <br />                                            Biotin attached covalently 11 distinct lysine residues histones H2A, H3, <br />                                            H4 [2-5]. Biotinylation histones participates cross-talk various histone <br />                                            modifications gene regulation [6-9]. For example, K12-biotinylated histone H4 co- <br />                                            localizes K9-dimethylated histone H3 pericentrometric heterochromatin repressed <br />                                            genes [9,10]. In addition role heterochromatin structures gene repression, <br />                                            biotinylation histone H4 appears participate cellular response DNA damage <br />                                            [11], repression retrotransposons [12]. <br />  <br />                                            The attachment biotin carboxylases histones mediated holocarboxylase <br />                                            synthetase (HCS, EC 6.3.4.10). The open reading frame full-<span id='am-6' about='xsp:length' typeof='owl:Thing'>length</span> HCS encodes 726 amino <br />                                            acids [13] existence splicing variants (76, 82, 86 kDa) been <br />                                            confirmed [14]. HCS detected cytoplasm, mitochondria, cell nuclei, nuclear <br />                                            lamina [14,15]. <br />  <br />                                            BirA microbial ortholog HCS investigation provided valuable insights into <br />                                            HCS structure catalysis, amino acid sequences proteins 21% identical. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            The three-dimensional structure E. coli BirA solved x-ray crystallography <br />                                            [16] consists domains: N-terminal domain, central domain, C-terminal <br />                                            domain [17]. The central domains BirA HCS contain binding sites ATP and <br />                                            biotin functions catalysis understood [18-20]. In contrast, amino acid <br />                                            sequence N-termini BirA human HCS fairly distinct, preventing <br />                                            predictions HCS structure solely based analogies BirA N-terminus. The N-terminus <br />                                            BirA contains 60 amino acid residues, human HCS 446 amino acids <br />                                            long, depending splicing. It speculated N-terminal domain human HCS <br />                                            play roles gene regulation, protein-protein interactions (PPI), substrate recognition <br />                                            [14,20]. The C-terminal domain BirA essential biotinyl ligase activity [17,20] and <br />                                            participate substrate recognition [17]. It remains determined the <br />                                            C-terminus HCS plays similar role, given BirA HCS distinct biotinylations <br />                                            targets: biotin carboxyl carrier protein BirA, distinct carboxylases major <br />                                            classes histones HCS. <br />  <br />                                            The three-dimensional structure HCS determined. Here provide novel <br />                                            insights structure human HCS functions domains using molecular <br />                                            modeling approaches, yeast-two-hybrid (Y2H) assays, rescue experiments yeast <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            mutants. <br />  <br />                               Materials &amp; Methods <br />                               Cloning p67 fusion Gal4 activation domain <br />                                            The polypeptide p67 comprises 67 C-terminal amino acids human PCC (GenBank <br />                                            accession #AAA60035), including biotin-binding site K669 [18]. p67 well-established <br />                                            substrate studies HCS [18,20]. Here used p67 investigate domain-domain <br />                                            interactions substrate HCS. p67 cloned human liver cDNA the <br />                                            cloning vector pSTBlue (Novagen; San Diego, CA) using forward primer 5&#226;&#8364;&#178;- <br />                                            gaattcctgcgttccccgatg-3&#226;&#8364;&#178; reverse primer 5&#226;&#8364;&#178;-ctgctcgtggagctggaatgaggatcc-3&#226;&#8364;&#178;. The resulting <br />                                            plasmid named &#226;&#8364;&#339;p67-pSTBlue.&#226;&#8364;? p67-pSTBlue sequenced DNA core facility of <br />                                            University Nebraska-Lincoln, digested using EcoRI BamHI. The insert encoding <br />  <br />  <br />  <br />                                                Mol Genet Metab. Author manuscript; available PMC 2010 April 1. <br />                             Hassan et al.                                                                                          Page 3 <br />  <br />  <br />                                            p67 fused Gal4 activation domain, using vector pGADT7 (Clontech; Mountain View, <br />                                            CA); plasmid named &#226;&#8364;&#339;pGADT7-p67&#226;&#8364;? identity confirmed sequencing. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Cloning HCS different domains Gal4 binding domain vector <br />                                            Plasmid pUC-CAGG encodes full-length human HCS (GenBank accession #D23672) was <br />                                            kindly provided Y. Suzuki (Department Biochemical Genetics, Tohoku University <br />                                            School Medicine, Sendai, Japan) [13]. Using pUC-CAGG template, full-length human <br />                                            HCS PCR amplified using following primers: 5&#226;&#8364;&#178;-actgaattcctcatggaagatagactccac-3&#226;&#8364;&#178; <br />                                            (forward) 5&#226;&#8364;&#178;-actgtcgacttaccgccgtttggggaggatg-3&#226;&#8364;&#178; (reverse). The PCR product purified, <br />                                            digested EcoRI SalI, sub-cloned directly linearized pGBKT7 vector <br />                                            (Clontech, cat.# K1612-1) produce plasmid &#226;&#8364;&#339;pGBKT7-HCS.&#226;&#8364;? Plasmid identity was <br />                                            confirmed sequencing. <br />  <br />                                            In silico analysis human HCS using Pfam database [21] predicted major HCS <br />                                            domains (Fig. 1). The following HCS domains PCR amplified studies interactions <br />                                            p67 Y2H assays: N-terminal domain (M1-F446) amplified using primers <br />                                            5&#226;&#8364;&#178;-catggaggccgaattaatggaagatagactccac-3&#226;&#8364;&#178; (forward) 5&#226;&#8364;&#178;- <br />                                            gccgctgcaggtcgaccgaaggcctccatgttggtc-3&#226;&#8364;&#178; (reverse); biotin transfer/ATP binding (central) <br />                                            domain (F471-S575) amplified using primers 5&#226;&#8364;&#178;-ccatggaggccgaattatttgccgaagtgacccc-3&#226;&#8364;&#178; <br />                                            (forward) 5&#226;&#8364;&#178;-gccgctgcaggtcgagcgtcactgtaataaatatc-3&#226;&#8364;&#178; (reverse); putative linker domain <br />                                            (T610-V668) amplified using primers 5&#226;&#8364;&#178;-catggaggccgaattaaccatctgcatcaacg-3&#226;&#8364;&#178; (forward) <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            5&#226;&#8364;&#178;-gccgctgcaggtcgaccgtggacccagtatcgg-3&#226;&#8364;&#178; (reverse); C-terminal domain (H669- <br />                                            R718) amplified using primers 5&#226;&#8364;&#178;-catggaggccgaattccacagtggtcagcaagtc-3&#226;&#8364;&#178; (forward) and <br />                                            5&#226;&#8364;&#178;-gccgctgcaggtcgacctctcagcatgtcgaaggag-3&#226;&#8364;&#178; (reverse) primers. PCR products fused a <br />                                            Gal4 BD, using pGBKT7 vector linearized EcoRI SalI infusion <br />                                            cloning (Clontech, cat.# 631774); plasmids denoted pGBKT7-HCS1-446, <br />                                            pGBKT7-HCS471-575, pGBKT7-HCS610-668, pGBKT7-HCS669-718. Identities clones <br />                                            verified sequencing. <br />  <br />                               Functional complementation pGBKT7-HCS Saccharomyces cerevisiae biotin protein <br />                               ligase1 (BPL1) <br />                                            We used functional complementation approach verify HCS correctly folded and <br />                                            retained biotin ligase activity fused Gal4 BD. The BPL1 gene codes HCS <br />                                            ortholog BPL1 Saccharomyces cerevisiae. The open reading frame BPL1 plus 500 bp 5&#226;&#8364;&#178; <br />                                            500 bp 3&#226;&#8364;&#178; flanking sequences PCR-amplified SalI-NotI fragment. This SalI- <br />                                            BPL1-NotI fragment inserted pRS416 (CEN-ARS-URA3), resulting plasmid <br />                                            pLO70. The sequence BPL1 gene pLO70 confirmed DNA sequencing. All <br />                                            yeast strains used work derived W303-1B (MAT&#206;&#177; ade2-1 can1-100 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            his3-11,15 leu2-3,112 trp1-1 ura3-1). W303-1B transformed Ura+ plasmid pLO70, <br />                                            making strain YLO48 (W303-1B + pLO70). <br />  <br />                                            Strain YLO49 (W303-1B, bpl1 &#206;&#8221;::kanMX + pLO70) geneticin-resistant by <br />                                            transforming YLO48 PCR-generated fragment composed kanMX flanked 5&#226;&#8364;&#178; and <br />                                            3&#226;&#8364;&#178; flanking sequences BLP1 coding sequence. YLO49 transformed Trp+ with <br />                                            pGBKT7 (Gal4 BD), pGBKT7-HCS (Gal4 BD fused full-length HCS), pGBKT7- <br />                                            HCS1-446 (Gal4 BD fused HCS N-terminal domain), pGBKT7-HCS471-575 (Gal4 BD <br />                                            fused HCS central domain), pGBKT7-HCS610-668 (Gal4 BD fused HCS linker <br />                                            domain), pGBKT7-HCS669-718 (Gal4 BD fused HCS C-terminal domain) to <br />                                            produce strains YLO50 (W303-1B, bpl1 &#206;&#8221;::kanMX + pLO70 + pGBKT7), YLO51 (W303-1B, <br />                                            bpl1 &#206;&#8221;::kanMX + pLO70 + pGBKT7-HCS), YLO52 (W303-1B, bpl1 &#206;&#8221;::kanMX + pLO70 + <br />                                            pGBKT7-HCS1-446), YLO53 (W303-1B, bpl1 &#206;&#8221;::kanMX + pLO70 + pGBKT7-HCS471-575), <br />                                            YLO54 (W303-1B, bpl1 &#206;&#8221;::kanMX + pLO70 + pGBKT7-HCS610-668), YLO55 <br />  <br />  <br />                                                Mol Genet Metab. Author manuscript; available PMC 2010 April 1. <br />                             Hassan et al.                                                                                             Page 4 <br />  <br />  <br />                                            (W303-1B, bpl1 &#206;&#8221;::kanMX pLO70 + pGBKT7-HCS669-718) respectively. Ten-fold serial <br />                                            dilutions independent clones strain plated SC (synthetic complete), <br />                                            SC minus tryptophan (SC-Trp) SC+FOA plates grown days. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            In additional rescue studies, following truncated forms HCS used. HCS lacking <br />                                            N-terminal domain (M1-F446) PCR amplified using pUC-CAGG plasmid a <br />                                            template following primers: 5&#226;&#8364;&#178;-tggaggaattcatgtcatcagaacatttcaacttag-3&#226;&#8364;&#178; (forward) and <br />                                            5&#226;&#8364;&#178;-tatatgtcgaccttaccgccgtttggg-3&#226;&#8364;&#178; (reverse). The PCR products cloned pCR-Blunt II- <br />                                            TOPO vector (Invitrogen; Carlsbad, CA) digestion EcoRI SalI sub-cloning <br />                                            pGBKT7 vector fusion Gal4 BD; resulting vector denoted pGBKT7- <br />                                            HCS_Ndel. A HCS fusion construct lacking C-terminal domain (H669-R718) generated <br />                                            described pGBKT7-HCS_Ndel, using following primers: 5&#226;&#8364;&#178;- <br />                                            tggaggaattcatggaagatagactccacatggataatggactggtaccccaaaagattg-3&#226;&#8364;&#178; (forward) 5&#226;&#8364;&#178;- <br />                                            tatatgtcgaccttaccgccgtttggggaggatgaggttgacccagtatcggtaataaag-3&#226;&#8364;&#178; (reverse). This plasmid is <br />                                            denoted pGBKT7-HCS_Cdel. Plasmids pGBKT7-HCS_Ndel pGBKT7-HCS_Cdel were <br />                                            used make strains YLO56 (W303-1B, bpl1 &#206;&#8221;::kanMX + pLO70 + pGBKT7-HCS_Ndel) <br />                                            YLO57 (W303-1B, bpl1 &#206;&#8221;::kanMX + pL070 + pGBKT7-HCS_Cdel) determine whether <br />                                            BPL1 mutants rescued N- C-truncated HCS. <br />  <br />                               Characterization interactions p67 HCS <br />                                            Interactions p67 HCS investigated using Y2H assays co-transforming <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Saccharomyces cerevisiae strain AH109 (Clontech, cat.# 630444) various HCS-Gal4 BD <br />                                            fusion plasmids pGADT7-p67. After transformation, yeast plated SD/-Leu, -Trp, <br />                                            +Kan incubated colonies visible. Colonies re-suspended 10 &#206;&#188;L <br />                                            YPDA broth spotted SD/-Leu, -Trp, -His, -Ade, + Kan incubated 30&#194;&#176;C inverted <br />                                            position 21 days. Plates monitored daily appearance colonies; strong interactors <br />                                            visible earlier weak interactors. After 21 days monitoring, recovered clones <br />                                            streaked SD/-Leu, -Trp, -His, -Ade, + Kan supplemented X-&#206;&#177;-gal (200 &#206;&#188;l of <br />                                            2 mg/ml DMF) screen LacZ gene activation X-alpha-Gal assay (Clontech, cat.# <br />                                            630407). <br />  <br />                               Modeling HCS p67 interactions computational biology <br />                                            The 3D structure full-length human HCS modeled using PHYRE [22]. The 3D model <br />                                            human HCS spanning amino acid sequences 393 715 based crystal <br />                                            structure E. coli BirA [23]. The sequences BirA human HCS FASTA E-value <br />                                            4.2e-31 considered significant [24]. Similar results obtained using 3D- <br />                                            JIGSAW server [25]. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            A 3D-model amino acids 1 66 p67 obtained 3D-JIGSAW using biotin carboxyl <br />                                            carrier protein Pyrococcus horikoshi template [26]. The 3D structure p67 was <br />                                            independently confirmed PHYRE amino acids 2 65. <br />  <br />                                            In order assess interactions human HCS p67, docking proteins <br />                                            analyzed PATCHDOCK refined FireDock [27]. PATCHDOCK <br />                                            calculates spatial shape complementation structures provides list potential <br />                                            complexes. In docking calculation, protein surfaces treated facing an <br />                                            aqueous phase. The likelihood unrealistic predictions decreased conducting <br />                                            analogous docking calculations BirA substrate biotin carboxyl carrier protein [23]. <br />                                            Only docking interactions similar HCS p67 versus BirA biotin carboxyl <br />                                            carrier protein considered analyses. Data docking models refined <br />                                            using data site-directed mutagenesis experiments BirA [17]. 3D modeling HCS/ <br />  <br />  <br />  <br />                                                Mol Genet Metab. Author manuscript; available PMC 2010 April 1. <br />                             Hassan et al.                                                                                           Page 5 <br />  <br />  <br />                                            p67 interactions completed visual inspection representations generated PyMol <br />                                            [28] Deep View [29]. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Results <br />                               Functional characterization N C termini HCS <br />                                            Full-length HCS interacted p67 Y2H (Fig. 2). Colonies visible plates after <br />                                            14-15 days culture, consistent transient interactions typically observed enzymes <br />                                            substrates [30,31]. Of individual HCS domains tested, M1-F446 (N- <br />                                            terminus) H669-R718 (C-terminus) produced detectable interactions p67. The C- <br />                                            terminal domain interacted p67 strongly N-terminal domain. In response <br />                                            transformation pGBKT7-HCS669-718, yeast colonies visible plates only <br />                                            4-5 days culture. In response transformation pGBKT7-HCS1-466, yeast colonies <br />                                            visible plates 8-9 days culture. The observation individual domains <br />                                            interacted stronger length HCS explained fact amino acids <br />                                            involved establishing HCS-p67 interaction exposed readily accessible in <br />                                            constructs coding individual domains. <br />  <br />                               Biological activity hHCS HCS domain deletions yeast <br />                                            Human HCS biologically active yeast. Deletions YDL141W (the gene coding BPL1) <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            known lethal Saccharomyces cerevisiae. In studies transformed BPL1 <br />                                            mutants plasmid pGBKT7-HCS human HCS fused Gal4 BD. <br />                                            Transformation pGBKT7-HCS rescued BPL1 mutants, non-transformed cells <br />                                            died (Fig. 3). This consistent notion hHCS functionally complements yeast <br />                                            ortholog BPL1, results Y2H assays valid. Transformation BPL1 mutant <br />                                            yeast individual HCS domains (pGBKT7-HCS1-446, pGBKT7-HCS471-575, pGBKT7- <br />                                            HCS610-668, pGBKT7-HCS669-718) truncated forms HCS (pGBKT7-HCS_Ndel and <br />                                            pGBKT7-HCS_Cdel) sufficient rescue mutants. This consistent the <br />                                            theory N-terminal C-terminal domains HCS crucial substrate <br />                                            recognition. <br />  <br />                               3D modeling HCS <br />                                            We modeled 3D structure human HCS using published structure BirA <span id='am-5' about='protege:FROM' typeof='owl:Thing'>from</span> <br />                                            E. coli template [32]. The proposed structure HCS spans amino acids 393 715, <br />                                            corresponding N-terminal, central, C-terminal domains BirA. Only small <br />                                            fragment N-terminal domain human HCS included model the <br />                                            absence conserved sequence BirA template. The overall root mean square deviation <br />                                            (RSMD) predicted HCS structure BirA template 3.2 &#195;&#8230;, improved <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            1.2 &#195;&#8230; biotin-binding site HCS region spanning Thr489 Asn606. The <br />                                            comparably large RSMD overall model caused insertion human HCS, <br />                                            predicting &#206;&#177;- helix region Asn606 Asn621 compared loop region <br />                                            Asn134 Tyr141 BirA. <br />  <br />                                            Human HCS BirA superimposed identify conserved amino acid residues are <br />                                            likely important HCS catalysis. In central domain human HCS, Arg508- <br />                                            Ser515 loop corresponded biotin binding site (Arg118-Ser125) BirA. Our model <br />                                            predicted existence short helix HCS (Asn606-Asn621), valine substituted <br />                                            tryptophan corresponding helix BirA (Met211-Trp223). Valine668 linker <br />                                            region HCS corresponded Ile272 BirA. Histidine-675 Lys683 C-terminal <br />                                            domain HCS corresponded Lys277 Lys283, respectively, BirA. Although Arg718 <br />                                            HCS included structure predictions, sequence comparisons suggest this <br />                                            residue correspond Arg317 BirA. <br />  <br />  <br />                                                Mol Genet Metab. Author manuscript; available PMC 2010 April 1. <br />                             Hassan et al.                                                                                               Page 6 <br />  <br />  <br />                                            Next, predicted structure HCS substrate p67 subsequent docking calculations. <br />                                            For structure predictions, used BirA substrate biotin carboxyl carrier protein a <br />                                            template. The FASTA E-value p67 biotin carboxyl carrier protein 4e-19 with <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            excellent RSMD permitting reliable prediction p67 structure 0.3 &#195;&#8230; level of <br />                                            resolution. In biotin carboxyl carrier protein, Glu119 Glu147 important for <br />                                            biotinylation BirA [17]. These residues conserved Glu29 Glu58 p67 (Glu691 <br />                                            Glu720 PCC). <br />  <br />                                            The observations consistent conservation individual amino acids and <br />                                            entire domains biotin protein ligases substrates. Our docking calculations suggested <br />                                            p67 molecule spatially complemented cleft Arg588, Lys579, Ser515 <br />                                            including potential biotin binding site human HCS (Fig. 4). Arg508 Ser515 human <br />                                            HCS appear equally important binding p67. <br />  <br />                               Discussion <br />                                            HCS broad substrate specificity interacts with different carboxylases <br />                                            classes histones [15,33-35]. Carboxylases histones share any <br />                                            apparent sequence similarities, raising question HCS recognizes biotinylation <br />                                            targets. While preparation pure bioactive recombinant HCS characterization <br />                                            X-ray crystallography imminent, data publicly available yet. In absence <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            data, reasonable approach make predictions HCS structure substrate <br />                                            interaction based analogies BirA. This paper offers novel insights structure of <br />                                            HCS potential interactions p67 substrate using unique combination in <br />                                            silico modeling, Y2H assays, yeast rescue studies. A similar computational approach was <br />                                            recently used predict structure biotinidase, enzyme involved biotin <br />                                            metabolism [36]. <br />  <br />                                            This study makes number valuable contributions elucidation HCS structure <br />                                            catalysis. First, study provides experimental evidence N-terminus domain of <br />                                            HCS participates p67 binding, based data Y2H assays. This agreement with <br />                                            previous studies [13,14,20], suggest amino acids 128 446 HCS N-terminus <br />                                            important biotin ligase activity. The specific roles amino acids 128 446 not <br />                                            elucidated previous studies HCS catalysis. <br />  <br />                                            Second, study confirms previous theories Chapman-Smith et al. C-terminus of <br />                                            HCS important substrate recognition initiation biotin transfer process <br />                                            [17]. Those investigators showed C-terminal domain BirA essential biotin <br />                                            carboxyl carrier protein. Likewise, truncation removal C-terminus human HCS <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            abolishes biotinyl ligase activity HCS biotin carboxyl carrier protein p67 [20]. <br />  <br />                                            Third, studies demonstrate central linker domains HCS not <br />                                            participate substrate recognition. This conclusion based absence detectable <br />                                            interactions domains p67 Y2H assays. <br />  <br />                                            Fourth, laboratory heavily invested identification proteins <br />                                            interact HCS order target enzyme distinct regions chromatin [37-39]. <br />                                            Initial screens HCS-interacting proteins based Y2H assays. Our observation that <br />                                            human HCS rescue BPL1 mutant yeast suggests human HCS Y2H fully functional. <br />                                            Thus, risk encountering false positive clones Y2H assays reasonably small. <br />  <br />                                            Fifth, paper reveals potential docking sites p67 human HCS, i.e., cleft along <br />                                            Arg588, Lys579, Ser515, act concert Arg508 Ser515. Our prediction <br />                                            putative docking sites based X-ray crystallography data studies BirA and <br />  <br />                                                Mol Genet Metab. Author manuscript; available PMC 2010 April 1. <br />                             Hassan et al.                                                                                                              Page 7 <br />  <br />  <br />                                            biotin carboxyl carrier protein. It noteworthy R508W <span id='am-8' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-9' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span>mutation</span> most <br />                                            common HCS mutations human patients various ethnicities [40]. Patients carrying <br />                                            mutations respond exceptionally biotin supplementation, consistent role of <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            residue binding substrate biotin [41]. <br />  <br />                                            Sixth, model reveals existence flexible loops human HCS, e.g., loops Glu412- <br />                                            Gln417, Asn608-Thr618, Gly701-Val704, Phe487-Pro490 turn (Fig. 5). These loops <br />                                            (in particular Gly701-Val704) confer comparably great degree structural flexibility and, <br />                                            thus, particular light fairly broad substrate specificity HCS <br />                                            (various carboxylases histones). <br />  <br />                                            Seventh, silico model highlighted arginine residue (R718) C-terminal domain <br />                                            HCS major role enzyme-substrate recognition. Our model predicts that <br />                                            R718 likely corresponds highly conserved R317 BirA, known critical <br />                                            substrate recognition [17]. In previous studies suggested R317 BirA <br />                                            corresponds R725 R718 HCS [17]. We believe case, given <br />                                            that construct lacking R725 physically interacted p67 Y2H assays. Our <br />                                            modeling studies identified conserved positively charged residue (His675) might <br />                                            important substrate alignment. Previous studies aligned His675 lysine residue <br />                                            (K277) BirA, showed K277 critical correct alignment biotin carboxyl <br />                                            carrier protein active site BirA [17]. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            We recognize moderate uncertainties associated study. For example, acknowledge <br />                                            simple interaction single amino acid residues charged surfaces does not <br />                                            provide sufficient parameters build model enzyme-substrate orientation. However, if <br />                                            coupled exploration tertiary fold orientation, speculate which <br />                                            amino acid residues HCS critical substrate binding analogy BirA studies <br />                                            [42,43]. <br />  <br />                                            In conclusion, paper provides important insights structure human HCS. Our <br />                                            laboratory currently pursuing characterization HCS X-ray crystallography to <br />                                            address uncertainties associated model outlined here. <br />  <br />                               Acknowledgment <br />                                            A contribution University Nebraska Agricultural Research Division, supported funds provided <br />                                            Hatch Act. Additional support provided NIH grants DK063945, DK077816, GM62484 and <br />                                            ES015206, USDA grant 2006-35200-17138, NSF grants EPS-0701892 MCB6552870. <br />  <br />  <br />                               Abbreviations <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            ACC, acetyl-CoA carboxylase <br />                                            Ade, adenine <br />                                            Arg, arginine <br />                                            Asn, asparagine <br />                                            BPL1, biotin protein ligase1 <br />                                            FOA, 5-fluorouracil-6-carboxylic acid monohydrate <br />                                            Gal4 BD, Gal4 DNA binding domain <br />                                            Glu, glutamic acid <br />                                            HCS, holocarboxylase synthetase <br />                                            His, histidine <br />                                            Kan, kanamycin <br />                                            Leu, leucine <br />                                            Lys, lysine <br />  <br />  <br />                                                Mol Genet Metab. Author manuscript; available PMC 2010 April 1. <br />                             Hassan et al.                                                                                                    Page 8 <br />  <br />  <br />                                            MCC, 3-methylcrotonyl-CoA carboxylase <br />                                            PC, pyruvate carboxylase <br />                                            PCC, propionyl-CoA carboxylase <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            PPI, protein-protein interactions <br />                                            SD, synthetic defined <br />                                            Ser, serine <br />                                            Thr, threonine <br />                                            Trp, tryptophan <br />                                            Tyr, tyrosine <br />                                            Y2H, yeast hybrid <br />  <br />                               References <br />                                            1. Camporeale, G.; Zempleni, J. Biotin. In: Bowman, BA.; Russell, RM., editors. Present Knowledge in <br />                                               Nutrition. Vol. 9th ed.. International Life Sciences Institute; Washington, D.C.: 2006. p. 314-326. <br />                                            2. Camporeale G, Shubert EE, Sarath G, Cerny R, Zempleni J. K8 K12 biotinylated human <br />                                               histone H4. Eur. J. Biochem 2004;271:2257&#226;&#8364;&#8220;2263. [PubMed: 15153116] <br />                                            3. Kobza K, Camporeale G, Rueckert B, Kueh A, Griffin JB, Sarath G, Zempleni J. K4, K9, K18 in <br />                                               human histone H3 targets biotinylation biotinidase. FEBS J 2005;272:4249&#226;&#8364;&#8220;4259. [PubMed: <br />                                               16098205] <br />                                            4. Kobza K, Sarath G, Zempleni J. Prokaryotic BirA ligase biotinylates K4, K9, K18 K23 histone <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               H3. BMB Reports 2008;41:310&#226;&#8364;&#8220;315. [PubMed: 18452652] <br />                                            5. Chew YC, Camporeale G, Kothapalli N, Sarath G, Zempleni J. Lysine residues N- C-terminal <br />                                               regions human histone H2A targets biotinylation biotinidase. J. Nutr. Biochem <br />                                               2006;17:225&#226;&#8364;&#8220;233. [PubMed: 16109483] <br />                                            6. Chew YC, Raza AS, Sarath G, Zempleni J. Biotinylation K8 K12 co-occurs acetylation <br />                                               mono-methylation human histone H4. FASEB J 2006;20:A610.abstract <br />                                            7. Camporeale G, Giordano E, Rendina R, Zempleni J, Eissenberg JC. Drosophila holocarboxylase <br />                                               synthetase chromosomal protein required normal histone biotinylation, gene transcription <br />                                               patterns, lifespan heat tolerance. J. Nutr 2006;136:2735&#226;&#8364;&#8220;2742. [PubMed: 17056793] <br />                                            8. Camporeale G, Zempleni J, Eissenberg JC. Susceptibility heat stress aberrant gene expression <br />                                               patterns holocarboxylase synthetase-deficient Drosophila melanogaster caused decreased <br />                                               biotinylation histones, carboxylases. J. Nutr 2007;137:885&#226;&#8364;&#8220;889. [PubMed: 17374649] <br />                                            9. Gralla M, Camporeale G, Zempleni J. Holocarboxylase synthetase regulates expression biotin <br />                                               transporters chromatin remodeling events SMVT locus. J. Nutr. Biochem 2008;19:400&#226;&#8364;&#8220;408. <br />                                               [PubMed: 17904341] <br />                                            10. Camporeale G, Oommen AM, Griffin JB, Sarath G, Zempleni J. K12-biotinylated histone H4 marks <br />                                                 heterochromatin human lymphoblastoma cells. J. Nutr. Biochem 2007;18:760&#226;&#8364;&#8220;768. [PubMed: <br />                                                 17434721] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            11. Kothapalli N, Sarath G, Zempleni J. Biotinylation K12 histone H4 decreases response DNA <br />                                                 double strand breaks human JAr choriocarcinoma cells. J. Nutr 2005;135:2337&#226;&#8364;&#8220;2342. [PubMed: <br />                                                 16177192] <br />                                            12. Chew YC, West JT, Kratzer SJ, Ilvarsonn AM, Eissenberg JC, Dave BJ, Klinkebiel D, Christman <br />                                                 JK, Zempleni J. Biotinylation histones represses transposable elements human mouse cells <br />                                                 cell lines, Drosophila melanogaster. J. Nutr 2008;138:2316&#226;&#8364;&#8220;2322. [PubMed: 19022951] <br />                                            13. Suzuki Y, Aoki Y, Ishida Y, Chiba Y, Iwamatsu A, Kishino T, Niikawa N, Matsubara Y, Narisawa <br />                                                 K. Isolation characterization mutations human holocarboxylase synthetase cDNA. Nat. <br />                                                 Genet 1994;8:122&#226;&#8364;&#8220;128. [PubMed: 7842009] <br />                                            14. Hiratsuka M, Sakamoto O, Li X, Suzuki Y, Aoki Y, Narisawa K. Identification holocarboxylase <br />                                                 synthetase (HCS) proteins human placenta. Biochim. Biophys. Acta 1998;1385:165&#226;&#8364;&#8220;171. <br />                                                 [PubMed: 9630604] <br />                                            15. Narang MA, Dumas R, Ayer LM, Gravel RA. Reduced histone biotinylation multiple carboxylase <br />                                                 deficiency patients: nuclear role holocarboxylase synthetase. Hum. Mol. Genet 2004;13:15&#226;&#8364;&#8220;23. <br />                                                 [PubMed: 14613969] <br />  <br />  <br />                                                Mol Genet Metab. Author manuscript; available PMC 2010 April 1. <br />                             Hassan et al.                                                                                                         Page 9 <br />  <br />  <br />                                            16. Wilson KP, Shewchuk LM, Brennan RG, Otsuka AJ, Matthews BW. Escherichia coli biotin <br />                                                holoenzyme synthetase/bio repressor crystal structure delineates biotin- DNA-binding <br />                                                domains. Proc Natl Acad Sci U S A 1992;89:9257&#226;&#8364;&#8220;9261. [PubMed: 1409631] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            17. Chapman-Smith A, Mulhern TD, Whelan F, Cronan JE Jr. Wallace JC. The C-terminal domain of <br />                                                biotin protein ligase E. coli required catalytic activity. Protein Sci 2001;10:2608&#226;&#8364;&#8220;2617. <br />                                                [PubMed: 11714929] <br />                                            18. Leon-Del-Rio A, Gravel RA. Sequence requirements biotinylation carboxyl-terminal <br />                                                fragments human propionyl-CoA carboxylase alpha subunit expressed Escherichia coli. J. Biol. <br />                                                Chem 1994;269:22964&#226;&#8364;&#8220;22968. [PubMed: 8083196] <br />                                            19. Chapman-Smith A, Cronan JEJ. Molecular Biology Biotin Attachment Proteins. J. Nutr <br />                                                1999;129:477S&#226;&#8364;&#8220;484S. [PubMed: 10064313] <br />                                            20. Campeau E, Gravel RA. Expression Escherichia coli N- C-terminally deleted human <br />                                                holocarboxylase synthetase. Influence N-terminus biotinylation identification a <br />                                                minimum functional protein. J. Biol. Chem 2001;276:12310&#226;&#8364;&#8220;12316. [PubMed: 11124959] <br />                                            21. Bateman A, Coin L, Durbin R, Finn RD, Hollich V, Griffiths-Jones S, Khanna A, Marshall M, Moxon <br />                                                S, Sonnhammer EL, Studholme DJ, Yeats C, Eddy SR. The Pfam protein families database. Nucleic <br />                                                Acids Res 2004;32:138&#226;&#8364;&#8220;141. <br />                                            22. Bennett-Lovsey RM, Herbert AD, Sternberg MJ, Kelley LA. Exploring extremes sequence/ <br />                                                structure space ensemble fold recognition program Phyre. Proteins 2008;70:611&#226;&#8364;&#8220;25. <br />                                                [PubMed: 17876813] <br />                                            23. Weaver LH, Kwon K, Beckett D, Matthews BW. Competing protein:protein interactions proposed <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                control biological switch E. coli biotin repressor. Protein Sci 2001;10:2618&#226;&#8364;&#8220;2622. <br />                                                [PubMed: 11714930] <br />                                            24. Pearson WR, Lipman DJ. Improved tools biological sequence comparison. Proc. Natl. Acad. Sci. <br />                                                USA 1988;85:2444&#226;&#8364;&#8220;2448. [PubMed: 3162770] <br />                                            25. Bates PA, Kelley LA, MacCallum RM, Sternberg MJ. Enhancement protein modeling human <br />                                                intervention applying automatic programs 3D-JIGSAW 3D-PSSM. Proteins 2001;5 <br />                                                (Suppl):39&#226;&#8364;&#8220;46. [PubMed: 11835480] <br />                                            26. Bagautdinov B, Matsuura Y, Bagautdinova S, Kunishima N. Protein biotinylation visualized a <br />                                                complex structure biotin protein ligase substrate. J. Biol. Chem 2008;283:14739&#226;&#8364;&#8220;50. <br />                                                [PubMed: 18372281] <br />                                            27. Schneidman-Duhovny D, Inbar Y, Nussinov R, Wolfson HJ. PatchDock SymmDock: servers for <br />                                                rigid symmetric docking. Nucleic Acids Res 2005;33:363&#226;&#8364;&#8220;367. <br />                                            28. DeLano, WL. The Pymol Molecular Graphics System. DeLano Scientific; San Carlos, CA: 2002. <br />                                            29. Guex N, Peitsch MC. SWISS-MODEL Swiss-PdbViewer: environment comparative <br />                                                protein modeling. Electrophoresis 1997;18:2714&#226;&#8364;&#8220;2723. [PubMed: 9504803] <br />                                            30. Fields S, Song O. A novel genetic detect protein-protein interactions. Nature 1989;340:245&#226;&#8364;&#8220; <br />                                                246. [PubMed: 2547163] <br />                                            31. Estojak J, Brent R, Golemis EA. Correlation two-hybrid affinity data vitro measurements. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Mol Cell Biol 1995;15:5820&#226;&#8364;&#8220;5829. [PubMed: 7565735] <br />                                            32. Weaver LH, Kwon K, Beckett D, Matthews BW. Corepressor-induced organization assembly of <br />                                                biotin repressor: model allosteric activation transcriptional regulator. Proc. Natl. Acad. <br />                                                Sci. USA 2001;98:6045&#226;&#8364;&#8220;6050. [PubMed: 11353844] <br />                                            33. Hymes J, Fleischhauer K, Wolf B. Biotinylation biotinidase following incubation biocytin. <br />                                                Clin. Chim. Acta 1995;233:39&#226;&#8364;&#8220;45. [PubMed: 7758201] <br />                                            34. Hymes J, Fleischhauer K, Wolf B. Biotinylation histones human serum biotinidase: assessment <br />                                                biotinyl-transferase activity sera normal individuals children biotinidase <br />                                                deficiency. Biochem. Mol. Med 1995;56:76&#226;&#8364;&#8220;83. [PubMed: 8593541] <br />                                            35. Hymes J, Wolf B. Human biotinidase isn&#226;&#8364;&#8482;t just recycling biotin. J. Nutr 1999;129:485S&#226;&#8364;&#8220;489S. <br />                                                [PubMed: 10064314] <br />                                            36. Pindolia K, Jensen K, Wolf B. Three dimensional structure human biotinidase: modeling <br />                                                functional correlations. Mol. Genet. Metab 2007;92:13&#226;&#8364;&#8220;22. [PubMed: 17629531] <br />                                            37. Hassan YI, Zempleni J. Epigenetic regulation chromatin structure gene function biotin. J. <br />                                                Nutr 2006;136:1763&#226;&#8364;&#8220;1765. [PubMed: 16772434] <br />  <br />  <br />                                                Mol Genet Metab. Author manuscript; available PMC 2010 April 1. <br />        <br /> </body></html>