<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>RNA Editing Genes Associated Extreme Old Age in <br /> Humans Lifespan C. elegans <br /> Paola Sebastiani1.*, Monty Montano2.*, Annibale Puca3, Nadia Solovieff1, Toshio Kojima4, Meng C. <br /> Wang5, Efthymia Melista6, Micah Meltzer2, Sylvia E. J. Fischer5, Stacy Andersen7, Stephen H. Hartley1, <br /> Amanda Sedgewick8, Yasumichi Arai9, Aviv Bergman10, Nir Barzilai11, Dellara F. Terry7, Alberto Riva12, <br /> Chiara Viviani Anselmi3, Alberto Malovini3, Aya Kitamoto4, Motoji Sawabe13, Tomio Arai13, Yasuyuki <br /> Gondo14, Martin H. Steinberg6, Nobuyoshi Hirose9, Gil Atzmon11, Gary Ruvkun5, Clinton T. Baldwin6&quot;, <br /> Thomas T. Perls7&quot; <br /> 1 Department Biostatistics, Boston University School Public Health, Boston, Massachusetts, United States America, 2 Department Medicine Sections Infectious <br /> Diseases, Boston University School Medicine, Boston, Massachusetts, United States America, 3 Department Genetics, IRCCS Multimedica, Milan, Italy, <br /> 4 Computational Systems Biology Research Group, Advanced Science Institute, RIKEN, Yokohama, Kanagawa, Japan, 5 Department Genetics, Massachusetts General <br /> Hospital Harvard Medical School, Boston, Massachusetts, United States America, 6 Center Human Genetics, Boston University School Medicine, Boston, <br /> Massachusetts, United States America, 7 Geriatrics Section, Department Medicine, Boston University School Medicine, Boston, Massachusetts, United States of <br /> America, 8 Department Bioengineering, Boston University School Engineering, Boston, Massachusetts, United States America, 9 Division Geriatric Medicine, <br /> Department Internal Medicine, Keio University School Medicine, Tokyo, Japan, 10 Department Systems &amp; Computational Biology, Albert Einstein College of <br /> Medicine, Bronx, New York, United States America, 11 Institute Aging Research Diabetes Research Center, Departments Medicine Genetics, Albert Einstein <br /> College Medicine, Bronx, New York, United States America, 12 Department Molecular Genetics Microbiology, University Florida Gainesville, Florida, <br /> United States America, 13 Department Pathology, Tokyo Metropolitan Geriatric Hospital, Tokyo, Japan, 14 Clinical Thanatology Geriatric Behavioral Science, <br /> Graduate School Human Sciences, Osaka University, Suita, Osaka, Japan <br />  <br />  <br />  <br />      <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />      Background: The strong familiality living extreme ages suggests human longevity genetically regulated. The <br />      majority genes far associated longevity primarily function lipoprotein metabolism insulin/ <br />      IGF-1 signaling. There likely genetic modifiers human longevity remain discovered. <br />  <br />      Methodology/Principal Findings: Here, 18 single nucleotide polymorphisms (SNPs) RNA editing <br />      genes ADARB1 ADARB2 associated extreme old age U.S. based study centenarians, New England <br />      Centenarian Study. We replications findings independently conducted centenarian studies with <br />      different genetic backgrounds (Italian, Ashkenazi Jewish Japanese) collectively support association ADARB1 <br />      ADARB2 longevity. Some SNPs ADARB2 replicate consistently populations suggest strong effect <br />      independent different genetic backgrounds environments. <span id='am-5' about='protege:TO' typeof='owl:Thing'>To</span> evaluate functional association these <br />      genes lifespan, demonstrate inactivation orthologues adr-1 adr-2 C. elegans reduces median <br />      <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span>survival</span> 50%. We demonstrate inactivation argonaute gene, rde-1, critical regulator RNA <br />      interference, completely restores lifespan normal levels context adr-1 adr-2 loss function. <br />  <br />      Conclusions/Significance: Our results suggest RNA editors important regulator aging humans and <br />      that, evaluated C. elegans, pathway interact <span id='am-240' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-241' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-244' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-249' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-250' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-254' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-258' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-267' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-271' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-284' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-289' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-302' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-307' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-309' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-316' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span>RNA interference</span> machinery regulate lifespan. <br />  <br />   Citation: Sebastiani P, Montano M, Puca A, Solovieff N, Kojima T, et al. (2009) RNA Editing Genes Associated Extreme Old Age Humans Lifespan <br />   C. elegans. PLoS ONE 4(12): e8210. doi:10.1371/journal.pone.0008210 <br />   Editor: Joanna Mary Bridger, Brunel University, United Kingdom <br />   Received July 22, 2009; Accepted November 3, 2009; Published December 14, 2009 <br />   Copyright: &#195;&#376; 2009 Sebastiani et al. This open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: This work supported National Institutes Health grants: R01 HL087681 (to M.H.S), K24 AG025727 (to T.T.P), R01 AR055115 (to M.M.), RO1 AG027216 <br />   (to C.B.), Italian Ministry Research University (to A. P. A. M), following grants G.A, A.B. N.B.: AG-027734, AG-18728, RR-12248, DK-20541, <br />   M01-RR12248 grant Glenn Foundation. M.C.W. supported Life Sciences Research Foundation fellowship. S.E.J.F. supported MGH <br />   ECOR Fund Medical Discovery. The funders role study design, data collection analysis, decision publish, preparation manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: sebas@bu.edu (PS); mmontano@bu.edu (MM) <br />   . These authors contributed equally work. <br />   &quot; These authors contributed equally work. <br />  <br />  <br />  <br /> Introduction                                                                              broad public health significance including marked delay <br />                                                                                           age-related disability [9&#226;&#8364;&#8220;11] certain age-related diseases <br />   Exceptional longevity (EL) humans, defined living                              [12&#226;&#8364;&#8220;14]. Genetically, exceptional longevity presumed a <br /> extreme old ages 100 years older, strongly familial                        complex trait [15&#226;&#8364;&#8220;19]. Several candidate gene association <br /> [1&#226;&#8364;&#8220;8] factors facilitate exceptional survival                           studies successful discovering longevity-associated <br />  <br />  <br />         PLoS ONE | www.plosone.org                                                    1                             December 2009 | Volume 4 | Issue 12 | e8210 <br />                                                                                                                           RNA Editing Oldest Age <br />  <br />  <br /> genes humans. However, variants mainly                   Subjects Selected Association Study <br /> related lipoprotein metabolism [20&#226;&#8364;&#8220;22], FOXO proteins                       <span id='am-2' about='protege:FROM' typeof='owl:Thing'>From</span> NECS, genotype data obtained 281 males, <br /> [23,24], insulin/IGF-1 signaling [25] It likely            aged 96&#226;&#8364;&#8220;114 years 596 females, aged 100&#226;&#8364;&#8220;119 years. We <br /> more genetic modifiers human aging discovered             selected cutoff ages 96 years males 100 years females <br /> [25].                                                                       NECS focus extreme 1% survival based the <br />     In study, investigate genes A(adenosine) I        U.S. Social Security Administration cohort life table (http://www. <br /> (inosine) RNA editing pathway, post-transcriptional process            ssa.gov/OACT/NOTES/as116/as116LOT.html). NECS refer- <br /> which adenosine residues converted inosine resulting            ent cohort subjects consisted 270 spouses centenarian <br /> change gene expression protein function. Targets RNA               offspring children parents died mean age 73 <br /> editing include large number genes micro RNA.               years (average life expectancy parents&#226;&#8364;&#8482; birth cohort). <br /> Thus, surprising non-specific cellular process        Additional referent subjects selected Illumina <br /> would involved general maintenance cellular health           iControlDB database using genome-wide genetic matching as <br /> lifespan. However, implication previously              detailed methods (n = 1635). Note approximately 100 <br /> demonstrated.                                                               male centenarians included pooling-based genome screen- <br />     Here, report significant association genes       ing overlap second set. Given overlap relatively <br /> EL centenarian studies include New England                 small (,10%) subsequent analysis uses different <br /> Centenarian Study (NECS), 1,500 US individuals               analytic approach (genotype data individual subjects), do <br /> of primarily North European ancestry, aged 90 119               think overlap significant concern. <br /> years; Southern Italian Centenarian Study (SICS) &#226;&#8364;&#8220;a study               From SICS, used genotype data 271 males, ages <br /> nonagenarians centenarians closed population                  90&#226;&#8364;&#8220;108 years 188 females, aged 90&#226;&#8364;&#8220;109 years (total = 459). <br /> Cilento, South Italy; Ashkenazi Jewish Centenarian Study                Data 200 male 132 female SICS referent cohort subjects <br /> (AJCS), study approximately 300 nonagenarians                      aged 18&#226;&#8364;&#8220;48 years used analysis. From AJCS, <br /> centenarians founder population North Eastern                     genotype data obtained 299 oldest subjects (108 males <br /> European background, resident US; Japanese               aged 95 older 191 females aged 99 older) 269 <br /> Centenarian Study (JCS), study Japanese centenarians              younger referent cohort subjects (spouses offspring of <br /> has focused &#226;&#8364;&#732;&#226;&#8364;&#732;semi-supercentenarians&#226;&#8364;&#8482;&#226;&#8364;&#8482; subjects living past 105          centenarians, aged 85 younger, evidence parental <br /> years [26]. The characteristics populations allow         longevity). Four seventy oldest old subjects (82 males <br /> assess robustness associations varying genetic                aged 100&#226;&#8364;&#8220;110 years 388 females, aged 100&#226;&#8364;&#8220;116 years) and <br /> background environment.                                                 538 referent cohort subjects (randomly selected Japanese subjects, <br />     We evaluate functional significance RNA              aged 19&#226;&#8364;&#8220;89 years) constituted Japanese association study. <br /> editing candidate genes C. elegans lifespan studies        Table 1 reports summaries subjects&#226;&#8364;&#8482; characteristics. Ages <br /> silencing orthologs genes reduces median survival 50%.          extreme old validated birth certificates (in case <br /> We life span fully restored additional <br /> knockdown RNA interference gene, supporting                        Table 1. Study Subjects characteristics. <br /> functional role genes determining lifespan and <br /> implicating novel axis future aging studies. <br />                                                                                              Males     Females     All          Males   Females All <br /> Results Discussion                                                                       NECS oldest old                    NECS controls <br />                                                                              Sample Size     281       596         877          149     121         270 <br /> Selection Candidate Genes <br />    We selected genes study multiple reasons. First,        Median Age      102       103         103          75      74          75 <br /> a preliminary genome wide screening using pooled DNA samples                 Age Range       96&#226;&#8364;&#8220;114    100&#226;&#8364;&#8220;119     96&#226;&#8364;&#8220;119       58&#226;&#8364;&#8220;85   53&#226;&#8364;&#8220;85       53&#226;&#8364;&#8220;85 <br /> from approximately 130 male centenarians 130 younger male                                SICS oldest old                    SICS controls <br /> controls NECS [27], identified single                    Sample Size     271       188         459          200     132         332 <br /> nucleotide polymorphisms (SNPs) RNA-editing genes                     Median Age      94        98          96           34      32          33 <br /> ADARB1 (21q22.3), ADARB2 (10p15.3) associated <br />                                                                              Age Range       90&#226;&#8364;&#8220;109    90&#226;&#8364;&#8220;109      90&#226;&#8364;&#8220;109       18&#226;&#8364;&#8220;48   18&#226;&#8364;&#8220;48       18&#226;&#8364;&#8220;48 <br /> with extreme old age. ADARB1 exhibited strongest evidence <br /> for genetic association 5 SNPs met genome-wide                                     NA                                 Illumina controls <br /> significance, posterior odds allelic association .1,500          Sample Size                                        418     1217        1635 <br /> [27]. The probability 5 SNPs simultaneously associated              Median Age                                         47      46          47 <br /> under null hypothesis association 10213 based             Age Range                                          30&#226;&#8364;&#8220;75   30&#226;&#8364;&#8220;75       30&#226;&#8364;&#8220;75 <br /> a hyper-geometric distribution. Second, gene lies chromo-                            AJCS oldest old                    AJCS controls <br /> some 21q21 trisomy 21 (Down syndrome) resembles <br />                                                                              Sample Size     108       191         299          118     151         269 <br /> accelerated aging, premature age-related changes including <br /> in skin hair, increased frequency premature cataracts,            Median Age      99        101         100          77      73          73 <br /> hearing loss, menopause Alzheimer&#226;&#8364;&#8482;s disease [28] suggesting              Age Range       95&#226;&#8364;&#8220;108    99&#226;&#8364;&#8220;112      95&#226;&#8364;&#8220;112       54&#226;&#8364;&#8220;85   46&#226;&#8364;&#8220;85       46&#226;&#8364;&#8220;85 <br /> that genes chromosome 21 affect lifespan. Third,                              JCS oldest old                     JCS controls <br /> the genes identified preliminary genetic screen, RNA            Sample Size     82        388         470          178     360         538 <br /> editing represents general cellular process expected         Median Age      104       106         106          21      72          69 <br /> to improve cellular health; RNA editing activity been <br />                                                                              Age Range       100&#226;&#8364;&#8220;111 100&#226;&#8364;&#8220;116       100&#226;&#8364;&#8220;116      19&#226;&#8364;&#8220;89   19&#226;&#8364;&#8220;89       19&#226;&#8364;&#8220;89 <br /> associated innate immune response [29,30] age-related <br /> syndromes include dementia amyotrophic lateral sclerosis            Reported summaries contact ages. <br /> (ALS) [31].                                                                  doi:10.1371/journal.pone.0008210.t001 <br />  <br />  <br />  <br />       PLoS ONE | www.plosone.org                                        2                          December 2009 | Volume 4 | Issue 12 | e8210 <br />                                                                                                                                       RNA Editing Oldest Age <br />  <br />  <br /> of JCS, Basic Resident Registration Card). All subjects                      errors suggesting real associations not <br /> were enrolled studies Institutional Review Board approval                    artifacts. <br /> and oversight.                                                                          To test generalizability results other <br />                                                                                      independent groups, evaluated associations the <br /> Association ADARB1 Exceptional Longevity                                     AJCS JCS, using combination proxy SNPs typed <br />    We examined associations 31 SNPs ADARB1                          Affymetrix platforms SNPs Table 3 typed with <br /> NECS SICS samples using recessive dominant models                            traditional techniques (See methods). Table 4 summa- <br /> with Bayesian logistic regression [32]. The details statistical               rizes results replication study 4 5 SNPs and <br /> analysis Methods significant results                          additional proxy SNP Table 5. None SNPs in <br /> summarized Table 2 provides physical positions                       ADARB1 replicates results NECS SICS samples <br /> allele frequencies derived HapMap SNPs,                       significant association SNP rs17004734 <br /> Table 3 (rows 14&#226;&#8364;&#8220;15). Five SNPs ADARB1 strongly                               2Kb rs414743 consistent the <br /> associated extreme old age NECS, association                     presence longevity associated variants region. Because <br /> of SNP rs414743 remains significant imposing                              SNPs used NECS SICS chosen best <br /> stringent corrections multiple comparisons (Bayesian signif-                     capture genetic variations Caucasians the <br /> icance ,0.05/145 = 0.00035 145 overall number                        HapMap, correct choice founder <br /> SNPs included analysis). The SNPs tag region                     population Figure 1 shows different <span id='am-319' about='xsp:pattern' typeof='owl:Thing'>pattern</span> of <br /> strong linkage disequilibrium (LD) gene (Figure 1). None                   LD ADARB1 AJCS subjects. All ADARB1 SNPs in <br /> the SNPs reached statistical significance SICS                   Table 3 additional proxy SNPs genotyped the <br /> three SNPs rs2838809, rs2838810 rs2838816 exhibited                              JCS subjects Tables 6 7 summarize results. The <br /> consistent associations terms odds ratios allele                           SNPs Table 6 effects consistent with <br /> frequencies and, NECS SICS data combined,                          NECS SICS subjects reach statistical <br /> the SNPs remained statistically significant. These                       significance, data studies are <br /> SNPs extreme minor allele frequencies NECS                               aggregated. Figure 3 shows posterior densities ORs <br /> centenarians (MAF,0.01), allele frequencies                         rare SNPs suggestive association but <br /> controls close referent allele frequencies                      need larger sample sizes reach statistical <br /> HapMap (Table 3). Figure 2 displays scatter plot genotype                     significance. The association proxy SNPs rs2838816 <br /> intensities generated BeadStudio rules genotyping                      Table 7 consistent presence longevity <br />  <br />  <br />  <br />  <br />  Table 2. Summary selected SNPs. <br />  <br />  <br />                                                                                                         Risk versus <br />  Row                  SNP                              Chr                  position                    referent alleles                 CEPH                  JPT <br />  <br />  1                    rs10903420                       10                   1333726                     AA v AG/GG                       0.327                 0.058 <br />  2                    rs1007147                        10                   1341088                     AA v AG/GG                       0.312                 0.091 <br />  3                    rs2805562                        10                   1357514                     AA v AG/GG                       0.15                  0.058 <br />  4                    rs884949                         10                   1361610                     AA v AC/CC                       0.124                 0 <br />  5                    rs2805533                        10                   1374633                     AA/AG v GG                       0.77                  0.92 <br />  6                    rs2387653                        10                   1397826                     AA v AG/GG                       0.097                 0 <br />  7                    rs2805535                        10                   1450432                     AA v AG/GG                       0.699                 0.151 <br />  8                    rs2805543                        10                   1454892                     AA v AG/GG                       0.46                  0.105 <br />  9                    rs3898610                        10                   1474759                     AA v AG/GG                       0.841                 0.686 <br />  10                   rs1533484                        10                   1481339                     AA/AG v GG                       0.442                 0.791 <br />  11                   rs2676192                        10                   1495474                     AA v AG/GG                       0.301                 0.419 <br />  12                   rs2387952                        10                   1657365                     AA v AG/GG                       0.69                  0.616 <br />  13                   rs17294019                       10                   1659347                     AA/AG v GG                       0.196                 0.012 <br />  14                   rs3788157                        21                   45335136                    AA v AG/GG                       0.63                  0.65 <br />  15                   rs414743                         21                   45336503                    AA/AG v GG                       0.47                  0.31 <br />  16                   rs2838809                        21                   45445866                    AA v AG/GG                       0.009                 0 <br />  17                   rs2838810                        21                   45447751                    AA/AG v GG                       1                     1 <br />  18                   rs2838816                        21                   45454470                    AA v AG/GG                       0.009                 0 <br />  <br />  List 18 SNPs &#226;&#8364;&#8221;13 gene ADARB2 (10p15.3) 5 gene ADARB1 (21q22.3) &#226;&#8364;&#8221; associated exceptional longevity dominant or <br />  recessive models. The table reports sequential number easy identification tables figures (column 1), SNP identifier dbSNP (column 2), <br />  chromosome physical position human genome NCBI Build 36.3 (columns 3 4), risk versus referent alleles associated exceptional <br />  longevity using dominant recessive models (column 5), frequencies risk allele HapMap CEPH JPT. Note alleles Japanese <br />  group allele frequencies change substantially CEPH, SNPs rows 4 6 monomorphic. <br />  doi:10.1371/journal.pone.0008210.t002 <br />  <br />  <br />  <br />        PLoS ONE | www.plosone.org                                                3                            December 2009 | Volume 4 | Issue 12 | e8210 <br />                                                Table 3. SNPs ADARB2 (10p15.3) ADARB1 (21q22.3) associated exceptional longevity NECS SICS subjects. <br />  <br />  <br />                                                                                         Risk versus <br />                                                             Row      SNP                referent alleles        CEPH       NECS (877 oldest old, 1808 controls)                   SICS (459 oldest old, 429 controls)                 NECS+SICS <br />  <br />  <br />  <br />  <br /> PLoS ONE | www.plosone.org <br />                                                                                                                            OR                      Bayes sig     p(A)             OR                   Bayes sig     p(A)             OR                     Bayes sig        p(A) <br />  <br />                                               ADARB2        1        rs10903420         AA v AG/GG              0.327      1.28(1.07;1.53)         0.0048        0.28/0.23        1.14(0.84;1.54)      0.1212        0.28/0.25        1.25(1.07;1.45)        0.0020           0.28/0.24 <br />                                                             2        rs1007147          AA v AG/GG              0.312      1.35(1.13;1.64)         0.0015        0.27/0.22        1.25(0.92;1.66)      0.2040        0.29/0.25        1.34(1.15;1.59)        0.0003           0.27/0.22 <br />                                                             3        rs2805562          AA v AG/GG              0.15       1.22(0.96;1.54)         0.0500        0.15/0.12        1.38(0.97;1.97)      0.0384        0.20/0.15        1.32(1.09;1.59)        0.0022           0.16/0.13 <br />                                                             4        rs884949           AA v AC/CC              0.124      1.19(0.92;1.53)         0.0911        0.12/0.10        1.31(0.87;1.96)      0.0978        0.14/0.11        1.24(1.01;1.54)        0.0211           0.12/0.10 <br />                                                             5        rs2805533          AA/AG v GG              0.77       0.91(0.75;1.12)         0.1904        0.78/0.81        0.83(0.60;1.14)      0.1217        0.76/0.79        0.86(0.72;1.02)        0.0381           0.79/0.81 <br />                                                             6        rs2387653          AA v AG/GG              0.097      1.17(0.91;1.50)         0.1053        0.14/0.12        1.12(0.78;1.62)      0.2699        0.170.16         1.21(0.99;1.49)        0.0343           0.15/0.13 <br />                                                             7        rs2805535          AA v AG/GG              0.699      1.36(1.02;1.83)         0.0249        0.75/0.69        1.31(0.97;1.79)      0.0395        0.72/0.66        1.37(1.11;1.68)        0.0017           0.74/0.68 <br />                                                             8        rs2805543          AA v AG/GG              0.46       1.23(1.04;1.45)         0.0055        0.54/0.49        1.36(1.03;1.76)      0.0156        0.51/0.43        1.24(1.08;1.42)        0.0008           0.53/0.48 <br />  <br />  <br />  <br />  <br /> 4 <br />                                                             9        rs3898610          AA v AG/GG              0.841      1.42(1.11;1.77)         0.0015        0.88/0.83        1.19(0.80;1.74)      0.1973        0.87/0.85        1.36(1.12;1.67)        0.0015           0.87/0.84 <br />                                                             10       rs1533484          AA/AG v GG              0.442      0.81(0.70;0.97)         0.0109        0.35/0.40        0.79(0.63;1.01)      0.0428        0.43/0.49        0.86(0.75;0.98)        0.0147           0.38/0.42 <br />                                                             11       rs2676192          AA v AG/GG              0.301      0.81(0.67;0.98)         0.0096        0.26/0.30        0.95(0.74;1.31)      0.3869        0.26/0.27        0.83(0.71;0.96)        0.0086           0.26/0.30 <br />                                                             12       rs2387952          AA v AG/GG              0.69       1.37(1.15;1.66)         0.0004        0.75/0.69        1.05(0.80;1.37)      0.3484        0.66/0.65        1.20(1.04;1.40)        0.0073           0.72/0.68 <br />                                                             13       rs17294019         AA/AG v GG              0.196      0.69(0.55;0.86)         0.0005        0.14/0.19        0.80(0.59;1.10)      0.0826        0.22/0.27        0.78(0.65;0.93)        0.0030           0.17/0.21 <br />                                               ADARB1        14       rs3788157          AA v AG/GG              0.63       1.23(1.05;1.45)         0.0075        0.65/0.62        0.90(0.68;1.19)      0.2318        0.65/0.67        1.16(1.00;1.34)        0.0204           0.65/0.62 <br />                                                             15       rs414743           AA/AG v GG              0.47       0.72(0.61;0.85)         ,1025         0.40/0.48        1.08(0.83;1.45)      0.2689        0.49/0.46        0.83(0.72;0.96)        0.0053           0.43/0.48 <br />                                                             16       rs2838809          AA v AG/GG              0.009      0.27(0.05;0.93)         0.0096        0.003/0.009      0.58(0.15;1.97)      0.1973        0.011/0.016      0.46(0.19;1.08)        0.0297           0.005/0.01 <br />                                                             17       rs2838810          AA/AG v GG              1          3.71(1.12;19.29)        0.0136        0.997/0.990      2.29(0.63;10.33)     0.1088        0.991/0.984      2.73(1.11;8.25)        0.0136           0.995/0.99 <br />                                                             18       rs2838816          AA v AG/GG              0.009      0.28(0.05;0.93)         0.0116        0.003/0.009      0.58(0.15;1.98)      0.1934        0.011/0.016      0.48(0.18;1.04)        0.0309           0.005/0.01 <br />  <br />                                               The 13 SNPs gene ADARB2 (10p15.3) SNPs gene ADARB1 (21q22.3). Columns 1&#226;&#8364;&#8220;4 provide details SNPs table 2. Columns 5&#226;&#8364;&#8220;7 report results association NECS and <br />                                               referent subjects based Bayesian logistic regression dominant recessive models. Specifically, column 5 reports Bayesian estimate odds ratio 95% credible interval brackets, columns 6 reports the <br />                                               Bayes significance defined 1-p(OR.1) posterior estimate OR .1 1-p(OR,1) posterior estimate OR ,1. This number posterior probability null hypothesis OR#1 when <br />                                               estimate OR.1 (or OR$1 estimate OR,1) small values provide strong evidence null hypothesis similar Bayes p-value proposed Althman. Column 7 reports posterior probability of <br />                                               risk allele cases controls. Highlighted bold significant associations (Bayes significance ,0.05). Columns 8&#226;&#8364;&#8220;10 report replications SICS highlighted bold 4 SNPs significant this <br />                                               analysis. Columns 11&#226;&#8364;&#8220;12 report results analysis data studies aggregated. Although 15 SNPs reach statistical significance NECS 4 SNPs SICS, 18 consistent <br />                                               effects data studies aggregated, SNPs remain significant significance 11 stronger. <br />                                               doi:10.1371/journal.pone.0008210.t003 <br />  <br />  <br />  <br />  <br /> December 2009 | Volume 4 | Issue 12 | e8210 <br />                                                                                                                                                                                                                                                                                              RNA Editing Oldest Age <br />                                                                                                                     RNA Editing Oldest Age <br />  <br />  <br />  <br />  <br /> Figure 1. Pattern LD SNP ADARB1 (chromosome 21) associated exceptional longevity. The plots <br /> display pattern LD captured SNPs associated exceptional longevity ADARB1 (chromosome 21) using data NECS, SICS, <br /> AJCS JCS. The intensity red represents strength LD measured D9. <br /> doi:10.1371/journal.pone.0008210.g001 <br />  <br />  <br /> associated variants gene captured              exhibit consistent associations terms odds ratios allele <br /> our SNP selection Japanese population.                             frequencies combining data NECS SICS made <br />                                                                           associations stronger plus additional other <br /> Association ADARB2 Exceptional Longevity                          SNPs statistically significant. These SNPs tag region of <br />    We examined associations 114 SNPs ADARB2              approximately 160Kb ADARB2 includes blocks LD <br /> NECS SICS samples using recessive dominant               (Figure 4). <br /> models. Ten SNPs strongly associated extreme old age               Tables 4 5 summarize results replication 10 of <br /> in NECS, remains significant correcting        SNPs AJCS subjects. Two SNPs reach statistical <br /> multiple comparisons (SNPs rs2387952, Bayesian significance               significance set (rows 3 13) opposite effects <br /> 0.0004,0.05/145). Four significant SNPs (rs2805562;              compared NECS SICS (rows 13). However, SNPs in <br /> rs2805533; rs2805543; rs1533484) replicated          rows 1, 5&#226;&#8364;&#8220;9, 11 12 effects consistent the <br /> SICS (Bayes significance ,0.05) (Table 3). The remaining              NECS SICS and, data 3 studies are <br /> SNPs did reach statistical significance SICS did           aggregated, significant (columns 12&#226;&#8364;&#8220;14). The SNPs in <br />  <br />  <br />       PLoS ONE | www.plosone.org                                      5                        December 2009 | Volume 4 | Issue 12 | e8210 <br />                                                                                                                        RNA Editing Oldest Age <br />  <br />  <br />  <br />  <br /> Figure 2. Result genotype cluster algorithm BeadStudio. The plots normalized intensities polar coordinate the <br /> cluster <span id='am-3' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span> BeadStudio NECS subjects. The clear separation suggests genotype calls robust. <br /> doi:10.1371/journal.pone.0008210.g002 <br />  <br /> rows 2, 4, 11 typed proxy SNPs typed              elegans [31,35]. For lifespan analysis, focused C. elegans, <br /> Affymetrix 6.0 array confirm strong associations variants         orthologues ADARB1 ADARB2 RNA <br /> same region EL. These SNPs summarized Table 5               editing activity, adr-1 adr-2 [36] (see phylogenetic tree in <br /> are good proxy SNPs originally typed NECS              Figure S1). Because C. elegans average lifespan of <br /> SICS shown proximity terms physical distance              approximately 20 days, influence candidate genes on <br /> ,10kb linkage disequilibrium measured D9 r2.                     lifespan readily tested. Therefore, evaluate adr <br />    Only SNP rs2805533 reached statistical significance JCS           loss-of-function influences C. elegans lifespan, monitored the <br /> set, effect consistent observed effect         lifespan single double mutants adr-1(gv6) adr-2(gv42). <br /> the NECS, SICS AJCS samples (Table 6). The SNP                          Both alleles deletions remove the <br /> rs1533484 borderline significant (Bayesian significance                 coding sequence presumed null alleles [36] (Figure 6a and <br /> ,0.06) demonstrates consistent effects NECS                Figure S2). Strains carrying mutations adr-1 adr-2 displayed <br /> SICS results, allele frequencies substantially different.       shorter lifespan wild-type control N2 worms (log-rank <br /> The SNP rs10903420 significant opposite                test p,1028). Remarkably, aside decline lifespan, <br /> effect compared NECS, SICS AJCS subject sets. Note               obvious defects, contrast gain-of- <br /> however substantial differences allele frequencies          function studies noted lethality Drosophila [37]. <br /> pattern LD explain different patterns                        The insulin-like growth factor (IGF) pathway known <br /> associations ethnically distinct sample. Genotype data         lifespan regulatory axis worms [38], flies [39,40], mice [41], and <br /> of additional SNPs Table 7 provide evidence              humans [42]. Inactivation C. elegans insulin like receptor <br /> existence variants region physical positions              gene daf-2 dsRNA increased lifespan wild type N2 <br /> 1340K 1500K chromosome 10 associated                   worms, confirming previous studies. Notably, knockdown daf-2 <br /> exceptional longevity.                                                      dsRNA increased lifespan background adr-1 and <br />                                                                             adr-2 null mutations, resulting lifespan phenocopy similar to <br />                                                                             wild type N2 worms (Figure 6a). Similar extensions lifespan <br /> Age Trends <br />                                                                             seen single adr-1 adr-2 mutants, (Figure S2). These <br />    For SNPs, clear monotonic pattern associated with <br />                                                                             data suggest IGF axis mediated increases lifespan to <br /> increasing age observed (Figure 5). This monotonic pattern is <br />                                                                             daf-2 remain active presence adr-1 adr-2 background <br /> consistent genetic effect results alleles positively       genotypes, potency N2 wild type <br /> associated EL frequent older individuals,             background. From data, exclude possibility <br /> while alleles negatively associated EL            knockdown RNA editing genes adr-1 adr-2 mutants <br /> frequent. This pattern observed ApoE alleles [33]         results increased RNAi activity. In scenario, increased <br /> and consistent phenomenon demographic selection              RNAi target genes downstream daf-16 (seven daf-16 gene <br /> [34]. These increases allele frequencies age illustrate        targets identified comparative analysis daf-2 versus <br /> the increasing gain power conferred studying centenarians             daf-2::daf-16 strains [43]), reducing potency daf-2 <br /> and so, subjects age 105+ years, genetic studies            dependent increases lifespan. <br /> exceptional survival.                                                           In previous study, mutations adr-1 adr-2 resulted in <br />                                                                             increased GFP reporter transgene silencing, suggesting that <br /> Functional Studies                                                          declines ADAR function associated increase in <br />    To statistical association, chose investigate        RNA interference (RNAi), account GFP <br /> possible functional role genes regulating lifespan           silencing experiments. When argonaute gene rde-1, <br /> evaluating influence lifespan nematode, C. elegans,       essential RNA induced silencing complex (RISC) <br /> robust model organism candidate lifespan gene discovery. The            formation, introduced adr-1;adr-2 worms containing the <br /> A-to-I RNA editing gene family enzymatic editing                  transgenes, increased GFP silencing ADAR knockdown <br /> activity conserved broad array species                suppressed [44]. This suggests cross-regulation RNA <br /> including humans, mice, flies, zebrafish, xenopus notably, C.           editors RNA interference supported results <br />  <br />  <br />       PLoS ONE | www.plosone.org                                        6                        December 2009 | Volume 4 | Issue 12 | e8210 <br />                                                Table 4. Replication AJCS subject set. <br />  <br />  <br />                                                                                           Risk versus <br />                                                            Row         SNP                referent alleles       CEPH       NECS+SICS                                             AJCS (299 oldest old, 269 controls)                NECS+SICS+AJCS <br />  <br />  <br />  <br />  <br /> PLoS ONE | www.plosone.org <br />                                                                                                                             OR                      Bayes sig       p(A)          OR                   Bayes sig     p(A)            OR                     Bayes sig       p(A) <br />  <br />                                               ADARB2       1           rs10903420         AA v AG/GG             0.327      1.25(1.07;1.45)         0.0020          0.28/0.24     1.37(0.87;2.18)      0.0861        0.27/0.21       1.26(1.09;1.45)        0.0010          0.28/0.24 <br />                                                            2           rs1007147          AA v AG/GG             0.312      1.34(1.15;1.59)         0.0003          0.27/0.22 <br />                                                            3           rs2805562          AA v AG/GG             0.15       1.32(1.09;1.59)         0.0022          0.16/0.13     1.88(1.10;3.22)      0.0095        0.21/0.17       1.39(1.16;1.65)        ,1025           0.17/0.13 <br />                                                            4           rs884949           AA v AC/CC             0.124      1.24(1.01;1.54)         0.0211          0.12/0.10 <br />                                                            5           rs2805533          AA/AG v GG             0.77       0.86(0.72;1.02)         0.0381          0.79/0.81     0.73(0.45;1.18)      0.0955        0.76/0.81       0.84(0.72;0.98)        0.0129          0.78/0.81 <br />                                                            6           rs2387653          AA v AG/GG             0.097      1.21(0.99;1.49)         0.0343          0.15/0.13     1.40(0.83;2.38)      0.1030        0.20/0.15       1.27(1.05;1.53)        0.0082          0.17/0.13 <br />                                                            7           rs2805535          AA v AG/GG             0.699      1.37(1.11;1.68)         0.0017          0.74/0.68     1.27(0.82;1.95)      0.1476        0.81/0.77       1.28(1.06;1.54)        0.0050          0.73/0.70 <br />                                                            8           rs2805543          AA v AG/GG             0.46       1.24(1.08;1.42)         0.0008          0.53/0.48     1.26(0.85;1.87)      0.1217        0.51/0.45       1.22(1.08;1.39)        0.0001          0.52/0.47 <br />  <br />  <br />  <br />  <br /> 7 <br />                                                            9           rs3898610          AA v AG/GG             0.841      1.36(1.12;1.67)         0.0015          0.87/0.84     0.63(0.35;1.16)      0.0685        0.87/0.91       1.29(1.07;1.55)        0.0035          0.87/0.84 <br />                                                            10          rs1533484          AA/AG v GG             0.442      0.86(0.75;0.98)         0.0147          0.38/0.42 <br />                                                            11          rs2676192          AA v AG/GG             0.301      0.83(0.71;0.96)         0.0086          0.26/0.30     0.77(0.52;1.15)      0.0986        0.27/0.32       0.83(0.72;0.95)        0.0037          0.26/0.30 <br />                                                            12          rs2387952          AA v AG/GG             0.69       1.20(1.04;1.40)         0.0073          0.72/0.68     0.95(0.61;1.44)      0.3977        0.67/0.69       1.17(1.01;1.34)        0.0146          0.71/0.68 <br />                                                            13          rs17294019         AA/AG v GG             0.196      0.78(0.65;0.93)         0.0030          0.17/0.21     1.42(0.95;2.13)      0.0451        0.31/0.24 <br />                                               ADARB1       14          rs3788157          AA v AG/GG             0.63       1.16(1.00;1.34)         0.0204          0.65/0.62     0.73(0.50;1.05)      0.0463        0.58/0.65 <br />                                                            15          rs414743           AA/AG v GG             0.47       0.83(0.72;0.96)         0.0053          0.43/0.48 <br />                                                            16          rs2838809          AA v AG/GG             0.009      0.46(0.19;1.08)         0.0297          0.005/0.01    1.19(0.44;3.29)      0.3662        0.040/0.034 <br />                                                            17          rs2838810          AA/AG v GG             1          2.73(1.11;8.25)         0.0136          0.995/0.99    0.99(0.39;2.53)      0.4951        0.956/0.956 <br />                                                            18          rs2838816          AA v AG/GG             0.009      0.48(0.18;1.04)         0.0309          0.005/0.01    1.06(0.38;3.06)      0.4573        0.035/0.034 <br />  <br />                                               List SNPs ADARB2 ADARB1 reach statistical significance NECS SICS attempted replicated AJCS set. The 7 columns report details SNPs described legend of <br />                                               Table 3. Columns 8&#226;&#8364;&#8220;11 report results 14 18 SNPs Table 3 genotyped 255 oldest old 227 younger controls AJCS. Only 3 SNPs reach statistical significance set (rows 3, 13 14) two <br />                                               opposite effects compared NECS SICS (rows 13 14). However, SNPs rows 1, 5&#226;&#8364;&#8220;9, 11 12 effects consistent NECS SICS and, data 3 studies aggregated, <br />                                               significant (columns 12&#226;&#8364;&#8220;14). The SNPs rows 2, 4, 10 11 typed proxy SNPs ongoing genome wide association study conducted Affymetrix 6.0 array suggest strong associations <br />                                               region. <br />                                               doi:10.1371/journal.pone.0008210.t004 <br />  <br />  <br />  <br />  <br /> December 2009 | Volume 4 | Issue 12 | e8210 <br />                                                                                                                                                                                                                                                                                          RNA Editing Oldest Age <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                         RNA Editing Oldest Age <br />  <br />  <br />  <br />  <br />                                                                           referent allele <br />                                                                                                                                                                                                                                                                                                                                                                                                                               Tonkin et al [45], demonstrated mild <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                 Additional proxy SNPs genotyped AJCS set tag region shown proximity terms physical distance ,10kb linkage disequilibrium measured D9 r2 (See Figure 1). <br />                                                                           Risk versus <br />  <br />  <br />  <br />  <br />                                                                                                                                                                               AA/AG v GG <br />                                                                                                                                           AA/AT v TT <br />                                                                                                                                                             CC v CT/TT <br />                                                                                                                                                                                                                                                                                                                                                                                                                               chemotaxis defect adr-1;adr-2 double mutant be <br />                                                                                                                                                                                                                                                                                                                                                                                                                               rescued rde-1 mutant [45]. Therefore, evaluate the <br />                                                                                                                                                                                                                                                                                                                                                                                                                               potential cross-regulation RNA editing RNA <br />                                                                                                                                                                                                                                                                                                                                                                                                                               interference context lifespan, evaluated adr-1; adr-2 <br />                                                                                                                                                                                                                                                                                                                                                                                                                               mediated declines lifespan presence RNAi defective <br />  <br />  <br />  <br />  <br />                                                                                                                                           0.76/0.85 <br />                                                                                                                                                             0.23/0.16 <br />                                                                                                                                                                               0.95/0.98 <br />                                                                                                                                                                                                                                                                                                                                                                                                                               strain, rde-1(ne-219). Remarkably, consistent previous <br />  <br />  <br />                                                                                    AJCS (255 oldest old, 227 controls) <br />  <br />                                                                                                                            p(A) <br />                                                                                                                                                                                                                                                                                                                                                                                                                               results, loss rde-1 completely restored lifespan declines <br />                                                                                                                                                                                                                                                                                                                                                                                                                               associated adr-1; adr-2 loss-of-function (Figure 6b). We <br />  <br />                                                                                                                            Bayes sig <br />                                                                                                                                                                                                                                                                                                                                                                                                                               interpret data expanding interaction these <br />  <br />  <br />  <br />                                                                                                                                                             0.0269 <br />                                                                                                                                                                               0.0259 <br />                                                                                                                                                                                                                                                                                                                                                                                                                               RNA regulatory pathways include lifespan determination. <br />                                                                                                                                           0.01 <br />                                                                                                                                                                                                                                                                                                                                                                                                                                  Our experiments C. elegans raise question the <br />                                                                                                                                           0.56(0.33;0.91) <br />                                                                                                                                                                                                                                                                                                                                                                                                                               precise role A-to-I RNA editing gene activity human aging, <br />                                                                                                                                                             1.62(1.00;2.73) <br />                                                                                                                                                                               0.29(0.06;1.00) <br />                                                                                                                                                                                                                                                                                                                                                                                                                               remains unknown. However, demonstration this <br />                                                                                                                                                                                                                                                                                                                                                                                                                               family genes implicated regulation aging other <br />                                                                                                                                                                                                                                                                                                                                                                                                                               organisms warrants validation species, particularly <br />                                                                                                                                                                                                                                                                                                                                                                                                                               humans, provides novel regulatory axis future studies <br />                                                                                                                            OR <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                               regulatory pathways influence aging process. We <br />                                                                                                                                                                                                                                                                                                                                                                                                                               speculate unidentified ADAR variants delay age <br />                                                                                                                                                             1.00/0.58 <br />                                                                                                                                                                               1.00/0.07 <br />                                                                                                                                           0.63/0.3 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                               associated declines ADAR activity. We note reduction in <br />                                                                                                                          Distance D9/r2 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                               ADAR enzymatic activity associated Dementia, ALS and <br />                                                                                                                                                                                                                                                                                                                                                                                                                               Alzheimers disease normal aging individuals [46]. Consistent <br />                                                                                                                                                                                                                                                                                                                                                                                                                               interpretation, analysis published transcriptional <br />                                                                                                                                           11041 <br />                                                                                                                                                             2224 <br />                                                                                                                                                                               2383 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                               profiles aging C. elegans indicate adr-1 adr-2 <br />                                                                                                                                                                                                                                                                                                                                                                                                                               expression peak early adulthood decline age rather <br />                                                                                                                                                                                                                                                                                                                                                                                                                               precipitously. The declines observed study compatible <br />                                                                                                                                                                               45345886 <br />                                                                                                                                           1352129 <br />                                                                                                                                                             1361386 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                               protective role ADAR alleles aging. <br />                                                                                                                            Position <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                  Prior study, RNA editing directly <br />                                                                                                                                                                                                                                                                                                                                                                                                                               implicated regulation aging humans C. elegans. <br />                                                                                                                                                                                                                                                                                                                                                                                                                               Although impact lifespan observed C. elegans <br />                                                                                                                                                                                                                                                                                                                                                                                                                               appears independent insulin signaling (Figure 6a), the <br />                                                                                                                                                             rs10903426 <br />                                                                                                                                                                               rs17004734 <br />                                                                                    Proxy SNPs <br />  <br />  <br />  <br />                                                                                                                                           rs2804097 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                               interaction RNA editing RNA interference likely to <br />                                                                                                                                                                                                                                                                                                                                                                                                                               complex decreased insulin signaling C. elegans also <br />                                                                                                                            SNP <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                               affect RNA interference [47] suggest threshold effects <br />                                                                                                                                                                                                                                                                                                                                                                                                                               associated declining levels insulin signaling. Future studies <br />                                                                                                                                                                                                                                                                                                                                                                                                                               needed identify characterize targets RNA editing <br />                                                                                                                                           0.27/0.22 <br />                                                                                                                                                             0.12/0.10 <br />                                                                                                                                                                               0.43/0.48 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                               potential role(s) modulating RNA interference activity, <br />                                                                                                                            p(A) <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                               context aging age-related diseases. <br />                                                                                                                                                                                                                                                                                                                                                                                                                                  Our analysis provides strong evidence association of <br />                                                                                                                            Bayes sig <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                               ADARB1 ADARB2 extreme old age. Our findings of <br />                                                                                                                                           0.0003 <br />                                                                                                                                                             0.0211 <br />                                                                                                                                                                               0.0053 <br />          Table 5. Replication AJCS subject set using proxy SNPs. <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                               strongest association NECS sample consistent that <br />                                                                                                                                                                                                                                                                                                                                                                                                                               sample largest oldest studies. The <br />                                                                                                                                                                                                                                                                                                                                                                                                                               lack reproducibility SNPs to <br />                                                                                                                                                                                                                                                                                                                                                                                                                               differences overall genetic background (ethnicity), size and <br />                                                                                                                                           1.34(1.15;1.59) <br />                                                                                                                                                             1.24(1.01;1.54) <br />                                                                                                                                                                               0.83(0.72;0.96) <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                               younger ages oldest old samples. Nonetheless, associations <br />                                                                                    NECS+SICS <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                               noted different study populations suggesting that <br />                                                                                                                                                                                                                                                                                                                                                                                                                               associations ADARB1 ADARB2 EL robust <br />                                                                                                                            OR <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                               different genetic backgrounds environmental exposures. <br />                                                                                                                                                                                                                                                                                                                                                                                                                               ADARB2 large gene spanning 500Kb in <br />                                                                                    CEPH <br />  <br />  <br />  <br />                                                                                                                                           0.312 <br />                                                                                                                                                             0.124 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                               chromosome 10, analysis narrows association a <br />                                                                                                                                                                               0.47 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                               region approximately 100Kb followed-up fine <br />                                                                                                                                                                                                                                                                                                                                                                                                                               mapping sequencing discovering functional variants to <br />                                                                           referent alleles <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                               provide better understanding function genes in <br />                                                                                                                                                                                                 doi:10.1371/journal.pone.0008210.t005 <br />                                                                           Risk versus <br />  <br />  <br />  <br />  <br />                                                                                                                                                                               AA/AG v GG <br />                                                                                                                                           AA v AG/GG <br />                                                                                                                                                             AA v AC/CC <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                               human aging. <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                               Materials Methods <br />                                                                                                                                                                                                                                                                                                                                                                                                                               Ethic Statement <br />                                                                                                                                           rs1007147 <br />                                                                                                                                                             rs884949 <br />                                                                                                                                                                               rs414743 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                 Subjects included NECS, SICS, AJCS JCS provided <br />                                                                                                                                                                                                                                                                                                                                                                                                                               written informed consent, research involving human <br />                                                                                    SNP <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                               subjects approved Institutional Review Boards of <br />                                                                                                                                                                                                                                                                                                                                                                                                                               Boston University, Boston, USA (NECS), &#226;&#8364;&#732;&#226;&#8364;&#732;Istituto di Ricovero <br />                                                                                    Row <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                               e Cura Carattere Scientifico &#226;&#8364;&#732;&#226;&#8364;&#732;Multimedica, Milano, Italy (SICS), <br />                                                                                                                                                                               15 <br />                                                                                                                                           2 <br />                                                                                                                                                             4 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                               Albert Eistein College Medicine, Bronx, USA (AJCS), Keio <br />  <br />  <br /> PLoS ONE | www.plosone.org                                                                                                                                                                                                                                                                                                                                                                                                8                        December 2009 | Volume 4 | Issue 12 | e8210 <br />                                                                                                                                          RNA Editing Oldest Age <br />  <br />  <br />  <br />  Table 6. Replication JCS subject set. <br />  <br />  <br />                                        Risk versus <br />               Row     SNP              referent alleles      CEPH      NECS+SICS                                      JPT     JCS (470 oldest old, 538 controls) <br />  <br />                                                                        OR                    Bayes sig p(A)                   OR                 Bayes sig p(A) <br />  <br />  ADARB2       1       rs10903420       AA v AG/GG            0.327     1.25(1.07;1.45)       0.0020      0.28/0.24    0.058   0.51(0.23;1.07) 0.0397         0.03/0.045 <br />               2       rs1007147        AA v AG/GG            0.312     1.34(1.15;1.59)       0.0003      0.27/0.22    0.091   1.10(0.68;1.81)    0.3435      0.08/0.075 <br />               3       rs2805562        AA v AG/GG            0.15      1.32(1.09;1.59)       0.0022      0.16/0.13    0.058   0.67(0.26;1.56)    0.1813      0.02/0.03 <br />               4       rs884949         AA v AC/CC            0.124     1.24(1.01;1.54)       0.0211      0.12/0.10    0 <br />               5       rs2805533        AA/AG v GG            0.77      0.86(0.72;1.02)       0.0381      0.79/0.81    0.92    0.59(0.37;0.92) 0.0110         0.88/0.93 <br />               6       rs2387653        AA v AG/GG            0.097     1.21(0.99;1.49)       0.0343      0.15/0.13    0 <br />               7       rs2805535        AA v AG/GG            0.699     1.37(1.11;1.68)       0.0017      0.74/0.68    0.151   1.02(0.69;1.49)    0.4690      0.14/0.14 <br />               8       rs2805543        AA v AG/GG            0.46      1.24(1.08;1.42)       0.0008      0.53/0.48    0.105   0.86(0.57;1.30)    0.2381      0.10/0.12 <br />               9       rs3898610        AA v AG/GG            0.841     1.36(1.12;1.67)       0.0015      0.87/0.84    0.686   1.09(0.83;1.43)    0.2619      0.66/0.64 <br />               10      rs1533484        AA/AG v GG            0.442     0.86(0.75;0.98)       0.0147      0.38/0.42    0.791   0.79(0.58;1.07) 0.0641         0.76/0.80 <br />               11      rs2676192        AA v AG/GG            0.301     0.83(0.71;0.96)       0.0086      0.26/0.30    0.419   0.92(0.71;1.19)    0.2542      0.40/0.42 <br />               12      rs2387952        AA v AG/GG            0.69      1.20(1.04;1.40)       0.0073      0.72/0.68    0.616   0.98(0.76;1.28)    0.4576      0.59/0.60 <br />               13      rs17294019       AA/AG v GG            0.196     0.78(0.65;0.93)       0.0030      0.17/0.21    0.012   0.54(0.12;2.03)    0.1846      0.01/0.02 <br />  ADARB1       14      rs3788157        AA v AG/GG            0.63      1.16(1.00;1.34)       0.0204      0.65/0.62    0.65    1.03(0.79;1.36)    0.4080      0.67/0.66 <br />               15      rs414743         AA/AG v GG            0.47      0.83(0.72;0.96)       0.0053      0.43/0.48    0.31    1.14(0.86;1.52)    0.183       0.29/0.26 <br />               16      rs2838809        AA v AG/GG            0.009     0.46(0.19;1.08)       0.0297      0.005/0.01   0       0.81(0.36;1.81)    0.2959      0.028/0.034 <br />               17      rs2838810        AA/AG v GG            1         2.73(1.11;8.25)       0.0136      0.995/0.99   1       1.14(0.50;2.60)    0.3791      0.974/0.970 <br />               18      rs2838816        AA v AG/GG            0.009     0.48(0.18;1.04)       0.0309      0.005/0.01   0       0.81(0.36;1.79)    0.3032      0.029/0.034 <br />  <br />  Lists SNPs ADARB2 ADARB1 reach statistical significance NECS SICS attempted replicated JCS set. The 7 columns <br />  report details SNPs described legend Table 3. Columns 8&#226;&#8364;&#8220;11 report results 14 18 SNPs Table 3 genotyped 470 oldest <br />  old 538 younger controls JCS. Only 3 SNPs reach statistical significance set (rows 1, 5 10) opposite effects compared the <br />  NECS SICS (row 1). We did attempt merge results different populations substantial differences allele frequencies. <br />  doi:10.1371/journal.pone.0008210.t006 <br />  <br />  <br /> University, Tokyo, Japan (JCS). All data analyzed anony-                          showed NECS Illumina controls are <br /> mously. Control data Illumina iControlDB database                        comprised major clusters correspond northwest, <br /> anonymized.                                                                            northeast southwest Europeans, Illumina controls <br />                                                                                        sample subjects different levels admixture <br /> SNP Genotyping                                                                         clusters Figure S3. We randomly <br />    For NECS SICS samples, 1 ug genomic DNA                              sampled 1,538 subjects major clusters create a <br /> analyzed Illumina 370 CNV chip (Illumina, San Diego,                            control set matched genetic background NECS <br /> CA) samples 93% rate used                       extreme old sample set suggested [49]. We use same <br /> the analysis. For AJCS JCS, genotyping originally                          procedure identify 81 female 16 male subjects added <br /> performed Affymetrix 6.0 chip 5.0 chips, respectively                     set SICS controls. The random-selection procedure was <br /> with required rates 99% greater. Affymetrix Birdseed                        repeated twice lead results. <br /> algorithm Illumina Beadstudio used genotype calling. <br /> Non overlapping SNPs approximated SNPs                           Genetic Association Analysis (Pooled DNA Samples) <br /> substantial LD (D9.0.8) genotyped Sequenom (AJCS)                               The statistical analysis pooling based genome-wide genotype <br /> and BigDye Terminator cycle sequencing kit ABI Prism                            data described [27]. Briefly, method uses Bayesian <br /> 3730xl DNA analyzer (Applied Biosystems, CA, USA). The                                 association tests score evidence allelic associations <br /> sequence data analyzed ABI PRISM SeqScape                                    centenarians controls. Prior distributions represent <br /> Software version 2.6 (Applied Biosystems).                                             prior knowledge expected number genes may <br />                                                                                        implicated trait correct multiple <br /> Creation Genetically Matched Control Set                                          comparisons. The analysis uses linkage disequilibrium (LD) <br />    A referent cohort sample NECS subjects constructed                      based filters retain associations supported clusters of <br /> utilizing genotype data Illumina iControlDB database                          SNPs LD. <br /> and principal components analysis used match cases and <br /> controls genetic background. To reduce chances stratifica-                       Genetic Association Analysis (Individual DNA Samples) <br /> tion, identified 2,077 Caucasian referent subjects                            The genotype data 31 SNPs genes ADARB1 and <br /> Illumina iControlDB, genotype Illumina arrays,                           114 SNPs ADARB2 individually analyzed using Bayesian <br /> known age enrollment 30 75 years, used                           logistic regression [32] fit dominant recessive models of <br /> the principal component analysis implemented program                            inheritance adjusted gender. The marginal posterior distribu- <br /> EIGENSTRAT [48] examine structure referent                              tions ORs estimated using implementation of <br /> group compared NECS SICS subjects. The analysis                             Gibbs sampling WinBugs 1.4 [50], 2.5th 97.5th <br />  <br />  <br />        PLoS ONE | www.plosone.org                                                  9                            December 2009 | Volume 4 | Issue 12 | e8210 <br />                                                                                                                                                                                                                                                                                                                                                                                                                   RNA Editing Oldest Age <br />  <br />  <br />  <br />  <br />                                                                     referent alleles <br />                                                                                                                                                                                                                                                                                                                                                                       percentile used estimate 95% credible intervals (CI) the <br />                                                                                                                                                                                                                                                                                                                                                                       ORs. The 50th percentile used estimate OR, the <br />  <br />  <br />  <br />  <br />                                                                     Risk versus <br />  <br />  <br />  <br />  <br />                                                                                                                                CC/CG v GG <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                      CC/CG v GG <br />                                                                                                                                                                    CC v CG/GG <br />  <br />  <br />                                                                                                                                                                                                        AA/AT v TT <br />                                                                                                                                                                                                                          CC/CT v TT <br />                                                                                                                                                  CC v CT/TT <br />                                                                                                                                                                                                                                                                                                                                                                       frequency OR.1 used estimate posterior probability <br />                                                                                                                                                                                                                                                                                                                                                                       p(OR.1). The Gibbs sampler run 10,000 <br />                                                                                                                                                                                                                                                                                                                                                                       iterations 10,000 simulated values used to <br />                                                                                                                                                                                                                                                                                                                                                                       estimate parameters. We used prior distributions the <br />                                                                                                                                                                                                                                                                                                                                                                       regression coefficients logit function normal distributions <br />  <br />  <br />  <br />                                                                                                                                0.91/0.94 <br />                                                                                                                                                  0.09/0.06 <br />                                                                                                                                                                    0.08/0.05 <br />                                                                                                                                                                                      0.91/0.96 <br />                                                                                                                                                                                                        0.50/0.43 <br />                                                                                                                                                                                                                          0.89/0.93 <br />                                                                                                                                                                                                                                                                                                                                                                       mean 0 standard deviation 3.2 determine normal <br />  <br />                                                                                                                    p(A) <br />                                                                                                                                                                                                                                                                                                                                                                       prior distribution log(OR) mean 0 (no association) and <br />                                                                                                                                                                                                                                                                                                                                                                       variance ranges 10 genetic effect 40 with <br />                                                                              JPCS (432 oldest old, 346 controls) <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                       gene6gender effect. This set prior distributions was <br />                                                                                                                    Bayes sig <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                       determined make analysis robust rare alleles (frequen- <br />                                                                                                                                0.0422 <br />                                                                                                                                                  0.0332 <br />                                                                                                                                                                    0.0234 <br />                                                                                                                                                                                      0.0031 <br />                                                                                                                                                                                                        0.0202 <br />                                                                                                                                                                                                                          0.0059 <br />                                                                                                                                                                                                                                                                                                                                                                       cy,0.10) searched largest variance allowed <br />                                                                                                                                                                                                                                                                                                                                                                       successful execution Gibbs sampler. These prior distribu- <br />                                                                                                                                                                                                                                                                                                                                                                       tions bias analysis null hypothesis reduce false <br />                                                                                                                                                                                                                                                                                                                                                                       positive associations. <br />                                                                                                                                0.61(0.34;1.07) <br />                                                                                                                                                  1.70(0.97;3.05) <br />                                                                                                                                                                    1.84(1.01;3.58) <br />                                                                                                                                                                                      0.44(0.23;0.80) <br />                                                                                                                                                                                                        1.36(1.03;1.82) <br />                                                                                                                                                                                                                          0.54(0.30;0.86)                                                                                                                                 The Bayes significance defined 1- p(OR.1) the <br />                                                                                                                                                                                                                                                                                                                                                                       posterior estimate OR .1, 1-p(OR,1) the <br />                                                                                                                                                                                                                                                                                                                                                                       posterior estimate OR ,1, association was <br />                                                                                                                                                                                                                                                                                                                                                                       deemed significant NECS, SICS, data aggregated <br />                                                                                                                    OR <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                       studies, Bayes significance smaller than <br />                                                                                                                                                                                                                                                                                                                                                                       0.05. This measure significance posterior probability of <br />                                                                                                                                                                                                                                                                                                                                                                       null hypothesis OR#1 (or OR$1) small values denotes <br />                                                                                                                                                                                                                          0.08/0.03 <br />                                                                                                                                                  1.00/0.58 <br />                                                                                                                                                                    0.91/0.22 <br />                                                                                                                                                                                      0.88/0.20 <br />                                                                                                                                                                                                        0.08/0.03 <br />                                                                                                                                0.63/0.3 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                       strong evidence null hypothesis [32]. This analysis <br />                                                                              D9/r2 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                       identified 18 significant SNPs (Table 3), twice the <br />                                                                                                                                                                                                                                                                                                                                                                       number expected chance 145 independent tests two <br />                                                                                                                                                                                                                                                                                                                                                                       SNPs remained significant correcting threshold for <br />                                                                              Distance <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                       number tests. Furthermore, probability 18 SNPs <br />                                                                                                                                                                                                                          25343 <br />                                                                                                                                                                                                        24675 <br />                                                                                                                                                  2224 <br />                                                                                                                                5232 <br />  <br />  <br />                                                                                                                                                                    8646 <br />                                                                                                                                                                                      2726 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                       simultaneously significantly associated the <br />                                                                                                                                                                                                                                                                                                                                                                       null hypothesis association 0.0002, using binomial <br />                                                                                                                                                                                                                                            Additional SNPs genotyped JCS set corroborate associations NECS SICS subjects sets. <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                       distribution n = 145, x = 18 p = 0.05. An association that <br />                                                                                                                                                                                                                          45479813 <br />                                                                                                                                                                                                        45479145 <br />                                                                              position <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                       significant aggregated NECS SICS data was <br />                                                                                                                                1346320 <br />                                                                                                                                                  1361386 <br />                                                                                                                                                                    1406472 <br />                                                                                                                                                                                      1453158 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                       deemed replicated AJCS JCS studies same <br />                                                                                                                                                                                                                                                                                                                                                                       SNP significant (Bayes significance ,0.05) same <br />                                                                                                                                                                                                                                                                                                                                                                       genetic model consistent effects; SNP did not <br />                                                                                                                                                  rs10903426 <br />                                                                                                                                                                    rs17221652 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                       reach statistical significance (Bayes significance $0.05), the <br />                                                                                                                                                                                                                          rs2838824 <br />                                                                                                                                rs2804099 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                      rs4543904 <br />                                                                                                                                                                                                        rs6518219 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                       ORs different studies direction when <br />                                                                              SNP <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                       data studies aggregated, association was <br />                                                                                                                                                                                                                                                                                                                                                                       significant. The rationale second condition the <br />                                                                                                                                                                                                        0.005/0.01 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                       AJCS JC smaller sample sizes, less <br />                                                                                                                                0.27/0.22 <br />                                                                                                                                                  0.12/0.10 <br />                                                                                                                                                                    0.15/0.13 <br />                                                                                                                                                                                      0.74/0.68 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                       power compared NECS. However, consistent effects and <br />                                                                                                                    p(A) <br />     Table 7. Replication JCS subject set using proxy SNPs. <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                       increased significance aggregated data analyzed <br />                                                                                                                                                                                                                                                                                                                                                                       lack association replication study to <br />                                                                                                                    Bayes sig <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                       lack power effects similar different studies. This <br />                                                                                                                                0.0003 <br />                                                                                                                                                  0.0211 <br />                                                                                                                                                                    0.0343 <br />                                                                                                                                                                                      0.0017 <br />                                                                                                                                                                                                        0.0309 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                       strategy used increase power genetic association <br />                                                                                                                                                                                                                                                                                                                                                                       studies, example [51]. The results Tables 4, 5, 6 <br />                                                                                                                                                                                                                                                                                                                                                                       7. <br />                                                                                                                                                                                                                                                                                                                                                                          We conducted similar analysis stratified gender the <br />                                                                                                                                1.34(1.15;1.59) <br />                                                                                                                                                  1.24(1.01;1.54) <br />                                                                                                                                                                    1.22(0.98;1.48) <br />                                                                                                                                                                                      1.37(1.11;1.68) <br />                                                                                                                                                                                                        0.48(0.18;1.04) <br />                                                                              NECS+SICS <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                       limited sample sizes did produce strongly significant results. <br />  <br />                                                                                                                                                                                                                                                                                                                                                                       Linkage Disequilibrium (LD) Heatmaps <br />                                                                                                                    OR <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                          We used HaploView 4.1 create LD heatmaps LD <br />                                                                                                                                                                                                                                                                                                                                                                       displays generated using D9 color scheme white <br />                                                                                                                                                                                                        45454470 <br />                                                                              position <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                            doi:10.1371/journal.pone.0008210.t007 <br />                                                                                                                                1341088 <br />                                                                                                                                                  1361610 <br />                                                                                                                                                                    1397826 <br />                                                                                                                                                                                      1450432 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                       represents D9 = 0, red represents D9 = 1, different shades of <br />                                                                                                                                                                                                                                                                                                                                                                       red represent 0,D9,1 (Figures 1 4). <br />                                                                              Chr <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                       Lifespan Measurements C. elegans <br />                                                                                                                                10 <br />                                                                                                                                                  10 <br />                                                                                                                                                                    10 <br />                                                                                                                                                                                      10 <br />                                                                                                                                                                                                        21 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                          To synchronize worms lifespan, eggs isolated (N2, adr- <br />                                                                                                                                rs1007147 <br />  <br />  <br />                                                                                                                                                                    rs2387653 <br />                                                                                                                                                                                      rs2805535 <br />                                                                                                                                                                                                        rs2838816 <br />                                                                                                                                                  rs884949 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                       1, adr-2, adr-1;adr-2, rde-1, rde-4, adr-1;adr-2;rde-1) synchronized <br />                                                                                                                                                                                                                                                                                                                                                                       hatching overnight absence food 20C. Synchronized <br />                                                                              SNP <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                       L1 larvae counted plated (10 worms/plate, n = 60) on <br />                                                                                                                                                                                                                                                                                                                                                                       Escherichia coli bacterial lawns (OP50) NGM media and <br />                                                                              Row <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                       allowed develop L4-stage larvae 20C. 5-fluorodeoxyur- <br />                                                                                                                                                                                                        18 <br />                                                                                                                                2 <br />                                                                                                                                                  4 <br />                                                                                                                                                                    6 <br />                                                                                                                                                                                      7 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                       idine (FudR) solution added final concentration 0.1mg/ <br />  <br />  <br /> PLoS ONE | www.plosone.org                                                                                                                                                                                                                                                                                                                                       10                         December 2009 | Volume 4 | Issue 12 | e8210 <br />                                                                                                                          RNA Editing Oldest Age <br />  <br />  <br />  <br />  <br /> Figure 3. Posterior densities ORs. Posterior densities ORs 3 SNPs ADARB1 rare alleles moderate effects data <br /> aggregated NECS, SICS JCS. Significant associations results posterior densities overlapping 1 definite evidence an <br /> OR,1 OR.1, densities heavy tails provide definite evidence null hypothesis association. <br /> doi:10.1371/journal.pone.0008210.g003 <br />  <br /> ml prevent reproduction. Worms kept 20C lifespan               behavior, consistent previous reports adr mutant strains <br /> monitored counting alternate days. Lifespan defined              [36]. We did observe altered chemotaxis routine passage of <br /> the day adulthood (adult lifespan = 0) death. Aside          worms maintain stocks, previously noted [36]. We also <br /> reduced lifespan, worms appeared normal size general               noted reduced progeny viability (data shown). However, these <br />  <br />  <br />  <br />  <br /> Figure 4. Pattern LD SNP ADARB2 (chromosome 10) associated exceptional longevity. The plots <br /> display pattern LD captured SNPs associated exceptional longevity ADARB2 (chromosome 10) NECS, SICS, AJCS JCS <br /> data. The intensity red cells represents strength LD measured D9. The LD pattern NECS, SICS AJCS subjects similar but <br /> differ substantially pattern LD JCS subjects SNPs monomorphic (rs884949 rs2387653). <br /> Highlighted red SNPs replicate results AJCS JCS subjects. <br /> doi:10.1371/journal.pone.0008210.g004 <br />  <br />  <br />        PLoS ONE | www.plosone.org                                        11                        December 2009 | Volume 4 | Issue 12 | e8210 <br />                                                                                                                              RNA Editing Oldest Age <br />  <br />  <br />  <br />  <br /> Figure 5. Age related trend allele frequencies. The barplots age related trend allele frequencies SNPs rs17294019 <br /> (ADARB2, SNP # 98 Table 1) rs3788157 (ADARB1, SNP # 135 Table 1) NECS (n = 1,023). The frequencies common allele both <br /> SNPs stratified age groups 90&#226;&#8364;&#8220;99; 100&#226;&#8364;&#8220;105, 106 higher. Trends allele frequencies increasing age groups consistent a <br /> strong correlation genotype <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span>phenotype</span> results substantial enrichment protective alleles older subjects. <br /> doi:10.1371/journal.pone.0008210.g005 <br />  <br /> are unlikely influenced lifespan measurements,                RNAi Lifespan Measurement <br /> the adult worms sterile using FudR                         Eggs isolated gravid worms synchronized by <br /> transferred course lifespan assay.                            hatching overnight absence food. The synchronized L1 <br />  <br />  <br />  <br />  <br /> Figure 6. ADAR mediated decline lifespan, daf-2 influence, rde-1 rescue. a) Lifespan using mutant strains adr-1;adr-2 context <br /> of dsRNA mediated gene inactivation daf-2. Synchronized worms larval stage 4 (L4) sterilized FudR allowed feed on <br /> bacterial lawns contained dsRNA daf-2. Note: adr-1; adr-2 double mutant (red solid), adr-1; adr-2 double mutant dsRNA daf-2 (red <br /> hatched), N2 wild type (blue solid), N2 dsRNA daf-2 (blue hatched). Note decline lifespan adr-1; adr-2 compared N2 wildtype. <br /> Also note increases lifespan N2 adr-1; adr-2 presence dsRNA daf-2. The 50% survival time adr-1; adr-2 mutant <br /> animals 10 days (95% limits 9 12 days) compared 20 days (95% limits 18 20 days) N2 wild-type control worms. RNAi daf-2 <br /> increases lifespan 34 days (95% limits 32 40 days), compared 20 days wild type (N2 worms fed vector (RNAi)). daf-2 gene <br /> inactivation, background adr-1 adr-2 null mutations restored lifespan 18 days (95% limits 16 20 days), compared 10 <br /> days adr-1;adr-2 double mutant strain. b) Lifespan using mutant strains adr-1;adr-2 (solid red), N2 wildtype (solid blue), rde-1 (grey <br /> hatched), adr-1; adr-2; rde-1 (grey solid) demonstrate declines lifespan using mutant strains rescue lifespan RNAi defective (rde-1) <br /> background. The adr-1; adr-2 mutant half long lived wild-type (median survival time 9 days adr-1;adr-2 strain (95% limits 9 <br /> and 11 days), median survival time 21 days (95% limits 18 21 days), N2 wild-type worms. The survival distribution triple mutant adr- <br /> 1;adr-2; rde-1 median lifespan 21 days (95% limits 18 21 days), significantly different adr-1;adr-2, median lifespan 9 days <br /> (95% limits 9 11 days). The lifespan rde-1 modestly reduced compared wild-type N2, reported previously29. Inset boxes <br /> displays 50% survival (days) condition demonstrates daf-2 gene inactivation increases lifespan, wild type adr-1;adr-2 <br /> mutant strains (a) RNAi <span id='am-162' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-163' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span>knockout</span> (rde-1) restores lifespan. <br /> doi:10.1371/journal.pone.0008210.g006 <br />  <br />  <br />        PLoS ONE | www.plosone.org                                          12                         December 2009 | Volume 4 | Issue 12 | e8210 <br />                                                                                                                                               RNA Editing Oldest Age <br />  <br />  <br /> larvae placed OP50-containing agar plates                              Figure S3 Population structure NECS centenarians and <br /> allowed develop L4-stage larvae 20C. The L4-stage larvae                       controls, SICS centenarians controls, Illumina controls. <br /> were washed thoroughly, placed Escherichia coli                           Population structure NECS centenarians (blue) controls <br /> HT115 RNAi vector Escherichia coli HT115                                  (red), SICS centenarians (green) controls (orange), Illumina <br /> expressing double-stranded RNA (dsRNA) daf-2. Briefly,                              controls (grey). Each scatter plot shows principal <br /> dsRNA -expressing bacteria grown overnight LB                              components estimated using genotype data than <br /> 50 ug/ml ampicillin seeded RNAi NGM plates                                300K SNPs NECS, SICS Illumina subjects using the <br /> containing 5 mM isopropylthiogalactoside (IPTG). The RNAi                               program Eigenstrat. From bottom, left right: (Blue) scatter <br /> bacteria induced overnight room temperature dsRNA                           plot principal components centenarians the <br /> expression. About 30 synchronized L4-stage animals added                           NECS. The principal components (PC1 displayed x-axis <br /> to allowed develop adults, followed                          PC2 y-axis) identify 3 major clusters based the <br /> addition FudR. Worms kept 20C, lifespan                        ancestry NECS centenarians labeled NW <br /> monitored. Worms feeding bacteria carrying vector                          Europeans (PC1,0.005 20.0125,PC2,0.0125), Ashkenazi <br /> were used negative control. Log-rank test R package                         Jews (PC1.0.005 PC2,20.0125) SW Europeans/ <br /> survival used statistical analysis.                                         Italians (PC1.0.005 PC2.0.0125). The thresholds the <br />                                                                                         principal components identified splitting components <br /> Supporting Information                                                                  using mixture models. (Green) scatter plot principal <br /> Figure S1    Phylogenetic clustering alignment ADARs                             components centenarians SICS. In agreement the <br /> form multiple species. Phylogenetic clustering alignment                         analysis NECS subjects, centenarians SICS SW <br /> ADARs form multiple species, including Human, chimp, bull, cat,                         European genetic background. (Red) scatter plot two <br /> rat, chicken, wolf nematode (C. elegans). Sequences                            principal components controls NECS display <br /> aligned using neighbor-joining algorithm Clustal X,                         approximately population substructure centenarians; <br /> gap-stripped corrections multiple substitutions,                           (Grey) scatter plot principal components the <br /> bootstrap analyzed 1,000 bootstrap resamplings. Phylograms                         Illumina controls. The plots controls a <br /> were generated NJ-plot. The tree unrooted                               population substructure similar NECS cases controls <br /> generated using Neighbor Joining algorithm implemented                             larger level admixture European <br /> Clustal X. Note bootstrap values defining main ADAR                           subgroups. (Orange) scatter plot principal <br /> branches.                                                                               components controls SICS exhibit SW <br /> Found at: doi:10.1371/journal.pone.0008210.s001 (0.07 MB TIF)                           European genetic background SICS centenarians. Note the <br />                                                                                         plot PC1 PC2 NECS controls (red) repeated twice to <br /> Figure S2 C. elegans lifespan results using individual gene <br />                                                                                         facilitate comparisons NECS subjects between <br /> mutant strains. C. elegans lifespan results using individual gene <br />                                                                                         NECS Illumina controls. <br /> mutant strains, indicated. Eggs isolated gravid worms <br />                                                                                         Found at: doi:10.1371/journal.pone.0008210.s003 (0.12 MB TIF) <br /> and synchronized hatching overnight absence food. <br /> The synchronized L1 larvae placed OP50- <br /> containing agar plates allowed develop L4-stage larvae                        Acknowledgments <br /> at 20C. The L4-stage larvae washed thoroughly, placed                          We grateful subjects families centenarian <br /> on Escherichia coli expressing double-stranded RNA (dsRNA)                          studies. We thank Elixir Pharmaceuticals Fondazione Longevita for <br /> daf-2. Briefly, dsRNA -expressing bacteria grown overnight                         DNA referent cohort subjects residing Cilento, Italy and <br /> in LB 50 ug/ml ampicillin seeded RNAi NGM                            Illumina access control subject genotype database, Ms Yukiko <br /> plates containing 5 mM isopropylthiogalactoside (IPTG). The                             Abe, Ms Sanae Fujimaki (Division geriatric medicine, Department <br /> RNAi daf-2 bacteria induced overnight room temperature                          Internal medicine, Keio University School medicine) their <br />                                                                                         technical assistance excellent secretarial work. <br /> for daf-2 dsRNA mediated knockdown expression. About 30 <br /> synchronized L4-stage animals added and <br /> allowed develop adults, followed addition FudR.                         Author Contributions <br /> Worms kept 20C, lifespan monitored.                               Conceived designed experiments: PS MM MHS GR CB TP. <br /> Worms feeding bacteria carrying vector used                        Performed experiments: PS MM TK MCW EM MM SEJF YA CVA <br /> a negative control. Animals scored 1 2 days                               AK GA CB. Analyzed data: PS MM NS SHH AS AM TP. <br /> subsequently scored dead longer responded                           Contributed reagents/materials/analysis tools: PS MM AAP TK SLA <br /> to gentle prodding.                                                                     SHH YA AB NB DT AR MS TA YG MHS NH GA GR CB TP. Wrote <br />                                                                                         paper: PS MM NS MHS GR CB TP. <br /> Found at: doi:10.1371/journal.pone.0008210.s002 (0.05 MB TIF) <br />  <br />  <br /> References <br /> 1. Christensen K, Johnson TE, Vaupel JW (2006) The quest genetic                     6. Perls TT, Wilmoth J, Levenson R, Drinkwater M, Cohen M, et al. (2002) Life- <br />    determinants human longevity: challenges insights. Nat Rev Genet 7:               long sustained mortality advantage siblings centenarians. Proc Natl Acad <br />    436&#226;&#8364;&#8220;448.                                                                                 Sci USA 99: 8442&#226;&#8364;&#8220;8447. <br /> 2. Gudmundsson H, Gudbjartsson DF, Frigge M, Gulcher JR, Stefansson K (2000)             7. Schoenmaker M, Craen AJ, Meijer PH, Beekman M, Blauw GJ, et al. <br />    Inheritance human longevity Iceland. Eur J Hum Genet 8: 743&#226;&#8364;&#8220;749.                   (2006) Evidence genetic enrichment exceptional survival using family <br /> 3. Hjelmborg JV, Iachine I, Skytthe A, Vaupel JW, McGue M, et al. (2006)                    approach: Leiden Longevity Study. European Journal Human Genetics <br />    Genetic influence human lifespan longevity. Hum Genet 119: 312&#226;&#8364;&#8220;                   14: 79&#226;&#8364;&#8220;84. <br />    321.                                                                                  8. Terry DF, Wilcox MA, McCormick MA, Pennington JY, Schoenhofen EA, <br /> 4. Kerber RA, O&#226;&#8364;&#8482;Brien E, Smith KR, Cawthon RM (2001) Familial excess                        et al. (2004) Lower all-cause, cardiovascular, cancer mortality in <br />    longevity Utah genealogies. J Gerontol A Biol Sci Med Sci 56: B130&#226;&#8364;&#8220;139.               centenarians&#226;&#8364;&#8482; offspring. J Am Geriatr Soc 52: 2074&#226;&#8364;&#8220;2076. <br /> 5. Perls T, Shea-Drinkwater M, Bowen-Flynn J, Ridge SB, Kang S, et al. (2000)            9. Christensen K, McGue M, Petersen I, Jeune B, Vaupel JW (2008) Exceptional <br />    Exceptional familial clustering extreme longevity humans. J Am Geriatr            longevity does result excessive levels disability. Proc Natl Acad Sci U S A <br />    Soc 48: 1483&#226;&#8364;&#8220;1485.                                                                       105: 13274&#226;&#8364;&#8220;13279. <br />  <br />  <br />  <br />       PLoS ONE | www.plosone.org                                                   13                              December 2009 | Volume 4 | Issue 12 | e8210 <br />                                                                                                                                                    RNA Editing Oldest Age <br />  <br />  <br /> 10. Hitt R, Young-Xu Y, Silver M, Perls T (1999) Centenarians: older get,             31. Maas S, Kawahara Y, Tamburro KM, Nishikura K (2006) A-to-I RNA editing <br />     healthier been. Lancet 354: 652.                                                 human disease. RNA Biol 3: 1&#226;&#8364;&#8220;9. <br /> 11. Terry DF, Sebastiani P, Andersen SL, Perls TT (2008) Disentangling roles           32. Balding DJ (2006) A tutorial statistical methods population association <br />     disability morbidity survival exceptional old age. Arch Intern Med 168:             studies. Nat Rev Genet 7: 781&#226;&#8364;&#8220;791. <br />     277&#226;&#8364;&#8220;283.                                                                                  33. Schachter F, Faure-Delanef L, Guenot F, Rouger H, Froguel P, et al. (1994) <br /> 12. Atzmon G, Schechter C, Greiner W, Davidson D, Rennert G, et al. (2004)                        Genetic associations human longevity APOE ACE loci. Nat <br />     Clinical phenotype families longevity. J Am Geriatr Soc 52: 274&#226;&#8364;&#8220;277.                  Genet 6: 29&#226;&#8364;&#8220;32. <br /> 13. Evert J, Lawler E, Bogan H, Perls T (2003) Morbidity profiles centenarians:            34. Vaupel JW, Carey JR, Christensen K, Johnson TE, Yashin AI, et al. (1998) <br />     survivors, delayers, escapers. J Gerontol A Biol Sci Med Sci 58: 232&#226;&#8364;&#8220;237.                 Biodemographic trajectories longevity. Science 280: 855&#226;&#8364;&#8220;860. <br /> 14. Schoenhofen EA, Wyszynski DF, Andersen S, Pennington J, Young R, et al.                   35. Morse DP, Aruscavage PJ, Bass BL (2002) RNA hairpins noncoding regions <br />     (2006) Characteristics 32 supercentenarians. J Am Geriatr Soc 54:                          human brain Caenorhabditis elegans mRNA edited adenosine <br />     1237&#226;&#8364;&#8220;1240.                                                                                    deaminases act RNA. Proc Natl Acad Sci U S A 99: 7906&#226;&#8364;&#8220;7911. <br /> 15. Finch CE, Tanzi RE (1997) Genetics aging. Science 278: 407&#226;&#8364;&#8220;411.                        36. Tonkin LA, Saccomanno L, Morse DP, Brodigan T, Krause M, et al. (2002) <br /> 16. Salvioli S, Olivieri F, Marchegiani F, Cardelli M, Santoro A, et al. (2006) Genes,            RNA editing ADARs important normal behavior Caenorhabditis <br />     ageing longevity humans: problems, advantages perspectives. Free                   elegans. EMBO J 21: 6025&#226;&#8364;&#8220;6035. <br />     Radic Res 40: 1303&#226;&#8364;&#8220;1323.                                                                  37. Keegan LP, Brindle J, Gallo A, Leroy A, Reenan RA, et al. (2005) Tuning of <br /> 17. Sorkin J, Post W, Pollin TI, O&#226;&#8364;&#8482;Connell JR, Mitchell BD, et al. (2005) Exploring               RNA editing ADAR required Drosophila. EMBO J 24: 2183&#226;&#8364;&#8220;2193. <br />     genetics longevity Old Order Amish. Mech Ageing Dev 126:                    38. Kenyon C, Chang J, Gensch E, Rudner A, Tabtiang R (1993) A C. elegans <br />     347&#226;&#8364;&#8220;350.                                                                                      mutant lives twice long wild type. Nature 366: 461&#226;&#8364;&#8220;464. <br /> 18. Perls T (2001) Genetic phenotypic markers centenarians.                         39. Clancy DJ, Gems D, Harshman LG, Oldham S, Stocker H, et al. (2001) <br />     J Gerontol A Biol Sci Med Sci 56: M67&#226;&#8364;&#8220;70.                                                     Extension life-span loss CHICO, Drosophila insulin receptor substrate <br /> 19. Perls T, Kunkel L, Puca A (2002) The genetics aging. Curr Opin Genet Dev                   protein. Science 292: 104&#226;&#8364;&#8220;106. <br />     12: 362&#226;&#8364;&#8220;369. <br />                                                                                               40. Tatar M, Kopelman A, Epstein D, Tu MP, Yin CM, et al. (2001) A mutant <br /> 20. Barzilai N, Atzmon G, Schechter C, Schaefer EJ, Cupples AL, et al. (2003) <br />                                                                                                   Drosophila insulin receptor homolog extends life-span impairs <br />     Unique lipoprotein phenotype genotype associated exceptional <br />                                                                                                   neuroendocrine function. Science 292: 107&#226;&#8364;&#8220;110. <br />     longevity. JAMA 290: 2030&#226;&#8364;&#8220;2040. <br />                                                                                               41. Holzenberger M, Dupont J, Ducos B, Leneuve P, Geloen A, et al. (2003) IGF-1 <br /> 21. Atzmon G, Rincon M, Schechter CB, Shuldiner AR, Lipton RB, et al. (2006) <br />                                                                                                   receptor regulates lifespan resistance oxidative stress mice. Nature 421: <br />     Lipoprotein genotype conserved pathway exceptional longevity in <br />                                                                                                   182&#226;&#8364;&#8220;187. <br />     humans. PLoS Biol 4: e113. <br />                                                                                               42. Suh Y, Atzmon G, Cho M-O, Hwang D, Bingrong L, et al. (2008) Functionally <br /> 22. Atzmon G, Pollin TI, Crandall J, Tanner K, Schechter CB, et al. (2008) <br />                                                                                                   significant insulin-like growth factor 1 receptor mutations centenarians. Proc <br />     Adiponectin levels genotype: potential regulator life span humans. <br />     J Gerontol A Biol Sci Med Sci 63: 447&#226;&#8364;&#8220;453.                                                    Natl Acad Sci U S A 105: 3438&#226;&#8364;&#8220;3442. <br /> 23. Willcox BJ, Donlon TA, He Q, Chen R, Grove JS, et al. (2008) FOXO3A                       43. Lee SS, Kennedy S, Tolonen AC, Ruvkun G (2003) DAF-16 target genes that <br />     genotype strongly associated human longevity. Proc Natl Acad Sci U S A                control C. elegans life-span metabolism. Science 300: 644&#226;&#8364;&#8220;647. <br />     105: 13987&#226;&#8364;&#8220;13992.                                                                         44. Knight S, Bass B (2002) The role RNA editing ADARs RNAi. Mol Cell <br /> 24. Flachsbart F, Caliebe A, Kleindorp R, Blanche H, von Eller-Eberstein H, et al.                Biol 10: 809&#226;&#8364;&#8220;817. <br />     (2009) Association FOXO3A variation human longevity confirmed                  45. Tonkin LA, Bass BL (2003) Mutations RNAi rescue aberrant chemotaxis of <br />     German centenarians. Proc Natl Acad Sci U S A 106: 2700&#226;&#8364;&#8220;2705.                                 ADAR mutants. Science 302: 1725. <br /> 25. Vijg J, Campisi J (2008) Puzzles, promises cure ageing. Nature 454:             46. Lund J, Tedesco P, Duke K, Wang J, Kim SK, et al. (2002) Transcriptional <br />     1065&#226;&#8364;&#8220;1071.                                                                                    profile aging C. elegans. Curr Biol 12: 1566&#226;&#8364;&#8220;1573. <br /> 26. Kojima T, Kamei H, Aizu T, Arai Y, Takayama M, et al. (2004) Association                  47. Wang D, Ruvkun G (2004) The insulin pathway regulates RNAi C. elegans. <br />     analysis longevity Japanese population polymorphic variants                Cold Spring Harbor Symposium Quantitative Biology, 69th edition 69: <br />     genes involved insulin insulin-like growth factor 1 signaling pathways.             429&#226;&#8364;&#8220;433. <br />     Exp Gerontol 39: 1595&#226;&#8364;&#8220;1598.                                                               48. Price AL, Patterson NJ, Plenge RM, Weinblatt ME, Shadick NA, et al. (2006) <br /> 27. Sebastiani P, Zhao Z, Abad-Grau MM, Riva A, Hartley SW, et al. (2008) A                       Principal components analysis corrects stratification genome-wide <br />     hierarchical modular approach discovery robust associations                  association studies. Nat Genet 38: 904&#226;&#8364;&#8220;909. <br />     genome-wide association studies pooled DNA samples. BMC Genetics 9: 6.               49. Hinds DA, Stokowski RP, Patil N, Konvicka K, Kershenobich D, et al. (2004) <br /> 28. Yang Q, Rasmussen SA, Friedman JM (2002) Mortality associated Down&#226;&#8364;&#8482;s                     Matching strategies genetic association studies structured populations. <br />     syndrome USA 1983 1997: population-based study. Lancet 359:                  Am J Hum Genet 74: 317&#226;&#8364;&#8220;325. <br />     1019&#226;&#8364;&#8220;1025.                                                                                50. Spiegelhalter DJ, Myles JP, Jones DR, Abrams KR (1999) Methods health <br /> 29. Yang JH, Luo X, Nie Y, Su Y, Zhao Q, et al. (2003) Widespread inosine-                        service research. An introduction bayesian methods health technology <br />     containing mRNA lymphocytes regulated ADAR1 response                              assessment. BMJ 319: 508&#226;&#8364;&#8220;512. <br />     inflammation. Immunology 109: 15&#226;&#8364;&#8220;23.                                                      51. Thorleifsson G, Walters GB, Gudbjartsson DF, Steinthorsdottir V, Sulem P <br /> </body></html>