<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL OF BACTERIOLOGY, Mar. 1992, p. 1750-1759                                                                   Vol. 174, No. 6 <br /> 0021-9193/92/061750-10$02.00/0 <br /> Copyright X) 1992, American Society Microbiology <br />  <br />  <br />     Replacement folC Gene, Encoding Folylpolyglutamate <br /> Synthetase-Dihydrofolate Synthetase Escherichia coli, Genes <br />                        Mutagenized In Vitro <br />                                           CARON PYNE AND ANDREW L. BOGNAR* <br />                       Department Microbiology, University Toronto, Toronto, Ontario, Canada MSS 1A8 <br />                                            Received 26 July 1991/Accepted 13 January 1992 <br />  <br />               The folylpolyglutamate synthetase-dihydrofolate synthetase gene (foiC) Escherichia coli deleted <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> <br />            bacterial chromosome replaced selectable Kmr marker. The deletion strain required a <br />            complementing gene expressing folylpolyglutamate synthetase encoded plasmid viability, indicating <br />            thatfoiC essential gene E. coli. The complementingfoiC gene cloned vector pPM103 <br />            (pSC101, temperature sensitive replication), segregated spontaneously 42&#194;&#176;C absence of <br />            selection. This complementing plasmid replaced thefoiC deletion strain compatible pUC plasmids <br />            containing foiC genes mutations generated vitro, producing strains express mutant <br />            folylpolyglutamate synthetase. MutantfolC genes expressing insufficient enzyme activity complement <br />            chromosomal deletion, resulting retention pPM103 plasmid. Some mutant genes expressing low <br />            levels enzyme activity replaced complementing plasmid, strains produced auxotrophic for <br />            products folate-dependent pathways. The folylpolyglutamate synthetase gene Lactobacilus casei, which <br />            lack dihydrofolate synthetase activity, replaced complementing plasmid, strain was <br />            auxotrophic folate end products. <br />  <br />   The Escherichia coli foiC gene product, folylpolygluta-           The recent report deletion folA gene encoding <br /> mate synthetase (FPGS)-dihydrofolate synthetase, en-          dihydrofolate reductase poLA thyA strain integration <br /> zyme catalyzing distinct reactions folate biosyn-        recombinant plasmids containing ColEl origin and <br /> thetic pathway. The dihydrofolate synthetase activity adds          subsequent segregation (14) suggested con- <br /> L-glutamate dihydropteroate form dihydrofolate. Dihy-         structing deletion foiC similar method be <br /> drofolate reduced tetrahydrofolate cell dihy-       successful. <br /> drofolate reductase. The FPGS activity foiC product             The SF4 mutant strain contains folC gene point <br /> then adds second glutamate y-linkage              <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span>mutation</span> causing alani, e-<span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span>-threonine amino acid change <br /> tetrahydrofolates form polyglutamates. A second enzyme           codon 309 (16) re,,dires methionine growth (11). <br /> catalyzes addition glutamates a-link-        Plasmid-encoded mutant foiC genes express little <br /> age triglutamate (10, 12).                                   1% wild-type activity FPGS dihydro- <br />   Ferone et al. reported isolation foiC mutant             folate synthetase complemented SF4 methionine prototro- <br /> (W1485/I-21 [11] SF2 SF4 [5]) auxotrophic               phy (16, 17). The Lactobacillus casei FPGS gene, has <br /> methionine stimulated glycine absence              little dihydrofolate synthetase activity, com- <br /> vitamin B12. This mutant leaky cells            plement SF4 (31). We constructed number folC <br /> dihydrofolate synthetase FPGS activity levels 1 3%        mutants multicopy plasmids vitro mutagenesis, but <br /> of levels wild-type strain possess 10 30%      study properties hampered back- <br /> normal total folate pools (11). Because dihydrofolate           ground enzyme activity SF4 strain. In report, we <br /> synthetase activity required folate biosynthesis,          demonstrate deletion chromosomal folC gene in <br /> nonleaky mutant having residual enzyme activity            presence complementing gene plasmid temper- <br /> have folates. Such mutant require folate      ature sensitive replication. The complementing plasmid <br /> end products (thymine, purines, glycine, methionine,            replaced second plasmid containing foiC <br /> pantothenate), lethal, folate            mutant gene, provided gene product expresses <br /> required synthesis formyl methionine initiation       sufficient enzyme activity. This allows the&apos; study many <br /> protein synthesis. We wished obtain afolC mutant           folC mutants synthesized vitro absence back- <br /> completely lacks FPGS-dihydrofolate synthetase activity          ground FPGS-dihydrofolate synthetase activity encoded by <br /> investigate <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span>phenotype</span>.                                          chromosome. <br />   A number methods gene disruption allele <br /> replacement E. coli reported previously (3, 14, <br /> 15, 33). Double-stranded linear DNA fragments containing                             MATERIALS AND METHODS <br /> the folC gene Kmr cassette inserted coding <br /> sequence failed transform recBC sbcB strain kana-                 Materials. [a-32P]dATP (7,000 Ci/mmol), [a-32P]dCTP <br /> mycin resistance. In hands methods did work               (7,000 Ci/mmol), para-amino[carboxyl-14C]benzoic acid (54 <br /> even presence complementing wild-type plasmid.              mCi/mmol), L-[U-14C]glutamic acid (10 mCi/mmol) were <br />                                                                         obtained Amersham Corp. Tran35S-<span id='am-2' about='rdfs:label' typeof='owl:Thing'>label</span> (methionine) <br />                                                                         obtained ICN Pharmaceuticals. Fluorography re- <br />                                                                         agents (i3n3Hance), Aquasol, nylon membranes were <br />   * <br />       Corresponding   author.                                           New England Nuclear. Nitrocellulose obtained <br />                                                                  1750 <br />  VOL. 174, 1992                                    REPLACEMENT OF foiC WITH IN VITRO-MUTAGENIZED GENES                         1751 <br />  <br />                                               TABLE 1. Plasmids used study <br />              Plasmid(s)                                                   Description                                  Reference(s) <br /> pUC19                                    Cloning vector; Ampr                                                          24 <br /> pH2A-1                                   pKC7 usg, folC, dedD genes 10-kb fragment; Ampr Kmr             5 <br /> pCP1                                     pH2A-1 deletion 1-kb SphI fragment inactivating Km&apos; gene; Ampr        This study <br /> pCP2                                     pCP1 deletion folC dedD; Ampr                                     This study <br /> pCP3                                     pCP2 Kmr gene fromn pUC4K StuI site; Ampr Kmr                     This study <br /> pAC3                                     pUC8 3.5-kb fragment containing usg, foiC, dedD genes; Ampr          4 <br /> pPM103                                   pSC101 temperature sensitive replication; segregates 30&#194;&#176;C; Tetr     15, 20 <br /> pPMpac3                                  pPM103 containing 3.5-kb insert pAC3; Tetr                               This study <br /> pPMusg                                   pPMpac3 deletion dedD foiC; Tet&apos;                                  This study <br /> pPMdedD                                  pPMpac3 deletion usg foiC; Tetr                                   This study <br /> pAC5                                     pUC9 folC gene downstream lac promoter; Ampr                          4 <br /> pAC6                                     pUC19 PvuII fragment containing folC gene pAC5; Ampr                This study <br /> pPMfolC/amp                              pPM103 containing pAC6 EcoRI site; Ampr Tetr                           This study <br /> pPMfolC                                  pPM103 folC gene downstream lac promoter; Tetr                        This study <br /> pUGBgl                                   pUC19 usg linker correct 3&apos; end usg; Ampr                      6 <br /> pUC4K                                    pUC4 containing Kmr cassette; Ampr                                       Pharmacia <br /> pGly, pIleu, pArg, pGlu, pVAl, pLeu      pUC19 dedD gene foiC gene codon 309 mutation               16 <br />                                            indicated amino acid; Amnpr <br /> pSF4                                     pUC19 foIC dedD genes cloned SF4 mutant strain; Ampr            16 <br /> pGT3-8.1                                 pEMBL18+ L. casei FPGS gene; Ampr                                        31 <br /> pL2, pL4, pL5, pL6, pL8, pLlO            pAC5 6-bp linker inserted various sites; Ampr                         17 <br />  <br />  <br />  <br /> from Schleicher &amp; Schuell. Tetrahydrofolate, folinic acid,          DNA isolated E. coli strains method of <br /> adenosine, thymidine, ATP, methionine, glycine, glutamic            Driver Lawther (8) DNA spooled <br /> acid, antibiotics kanamycin, tetracycline, chlor-       glass rod overlay 2 volumes ice-cold <br /> amphenicol purchased Sigma Chemical Co.                   ethanol phenol-chloroform extraction DNA <br /> Ampicillin proteinase K Boehringer Mann-              redissolved TE buffer overnight gentle agitation <br /> heim. The Geneclean kit purchased Bio 101.                 prior digestion restriction endonuclease. Southern <br /> Restriction endonucleases, RNases, DNA modifying                blots performed following transfer DNA nitrocel- <br /> enzymes purchased Boehringer Mannheim, Phar-              lulose nylon membranes hybridizations described <br /> macia, Bethesda Research Laboratories.                           Southern (28) appropriate probes 42&#194;&#176;C 50% <br />   Bacterial strains, phage, plasmids, media. Strain             formamide. DNA probes labelled [ac-32P]dCTP by <br /> MM383 [thyA polA12(Ts) Rha- Lac- Strrl (22) gen-              random prinier procedure Feinberg Vogelstein (9) <br /> erous gift J. Brunton, Mount Sinai Research Institute,           using Klenow fragment DNA polymerase I. Transfor- <br /> Toronto, Canada. SF2 (foiC strA) previously                mation method Hanahan (13), for <br /> described (5). SF4 (folC strA recA srlC::TnlO) (5) used         temperature-sensitive MM383 cells strains were <br /> as recipient strain verify insertion foiC gene     transformed pPM103-derived plasmids, which <br /> some plasmid constructions. JF1754 (hsdR lac gal metB leuB          instance cells grown 30&#194;&#176;C heat shock step <br /> hisB) (19) used highly competent host plasmid           37&#194;&#176;C. Selection integrants YT <br /> constructions. DR1984 (25) host strain used             plates supplemented 25 p,g ampicillin ml (on <br /> maxicell studies. Phage P1 (cml clrlOO) (21) gift M.       cells grown 37&#194;&#176;C) followed streaking YT <br /> Herrington, Concordia University, Montreal, Canada. The             plates containing 25 ,ug kanamycin ml. Although the <br /> plasmids used study listed Table 1. Plasmid          pPM103 plasmid reported temperature sensitive <br /> pPM103 gift M. Toth, M.I.T.                                replication (15, 20), transformants grew in <br />   E. coli strains grown 2YT LB (21) medium <br /> supplemented ampicillin (50 ,ug/ml), kanamycin (50             presence tetracycline 42&#194;&#176;C. The plasmid segregated <br /> ,ug/ml), tet-racycline (10 ,ug/ml), chloramphenicol (12.5        high frequency, however, grown absence of <br /> ,ug/ml) indicated. SF4 grown Vogel-Bonner              selection 37&#194;&#176;C higher (but low frequency at <br /> minimal medium (32) supplemented methionine (50                30&#194;&#176;C). In order maintain complementing pPM103- <br /> jig/ml) glycine (50 ,ug/ml). Unsupplemented minimal             derived plasmids appropriate E. coli strains, cells <br /> medium solidified 1.5% agar used complemen-             grown presence tetracycline 30&#194;&#176;C. <br /> tation analysis. Strain MM383 grown 2YT medium                  Transduction foiC deletion Pol+ strains. The <br /> supplemented thymidine (200 ,g/ml). The following              MM383AfolC/pPMpac3a strain lysogenized P1 (cml <br /> folate end products added minimal medium                clrlOO), lysates prepared standard methods (21). <br /> growth SF2AfolC strain transformed mutant               Cells E. coli SF2 JF1754 containing plasmid <br /> derivatives folC gene: methionine, 50 ,ug/ml; glycine,       pPMpac3a infected lysate, plated kanamy- <br /> 50 ,ug/ml; adenosine, 20 ,ug/ml; thymidine, 200 ,ug/ml; pan-        cin resistance-selective plates, incubated 30&#194;&#176;C. <br /> tothenate, 1 ,ug/ml; serine, 50 ,ug/ml.                            Enzyme extraction assay. Cells grown 1-liter <br />    Plasmid isolation, DNA modification analysis,            cultures YT medium supplemented 100 ,ug ampi- <br /> bacterial transformation. Plasmid preparations         cillin ml. Cells harvested centrifugation and <br /> the alkaline lysis procedure (2). Restriction endonuclease          lysed sonication Branson sonifier. Solid ammonium <br /> digestions, nuclease digestions, ligations carried         sulfate added 60% (wt/vol) cleared lysate (50 <br /> out according manufacturer&apos;s instructions. Genomic           ml), precipitate collected centrifugation. This <br />  1752     PYNE AND BOGNAR                                                                                         J. BACTERIOL. <br />  <br /> fraction resuspended 5 10 ml 50 mM Tris HCI,         ampicillin selection, AmpS KmS segregants, <br /> pH 7.5.                                                           verified wild type Southern blots (not <br />    FPGS activity measured incorporation             shown), Amp&apos; Km&apos; segregants, would <br /> [&apos;4C]glutamate (1 mM) folate products described           possess deletions folC dedD genes, obtained. <br /> previously (7, 27). Both 5,10-methylenetetrahydrofolate (100         The results described suggested foIC an <br /> ,uM) 10-formyltetrahydrofolate (100 ,uM) used         essential gene viable MM383 derivatives a <br /> folate substrates. Dihydrofolate synthetase activity          deletion chromosomal folC gene therefore <br /> measured dihydropteroate (25 ,uM) substrate          require complementing gene plasmid could <br /> previously described (5).                                         replicate polA strain. To examine genes <br />    Incorporation labelled para-aminobenzoic acid cellu-   affected deletion, constructed <br /> lar folates. 2YT medium pretreated activated             recombinant plasmids containing folC gene (pPMfolC) <br /> charcoal filtered remove folates para-ami-          expressed lac promoter, upstream gene <br /> nobenzoic acid. para-Amino[carboxyl-&apos;4C]benzoic acid (54          (pPMusg), dedD gene (pPMdedD), genes <br /> mCi/mmol) added 20 ml medium (12,000                (pPMpac3a) DNA fragments inserted pPM103 <br /> cpm/ml) concentration 0.2 FtM. The medium             vector replicate MM383 poL4 strain (Fig. 4 <br /> inoculated appropriate E. coli strains shaken    5). The structures plasmids verified by <br /> 24 h. The cells washed twice centrifugation           restriction endonuclease analysis, plasmid-depen- <br /> counted using Aquasol scintillation fluid.                     dent expression appropriate gene products veri- <br />   Analysis plasmid-encoded gene products maxicells. E.      fied maxicell experiments (data shown). Expression <br /> coli DR1984 transformed various plasmids             folC pPMpac3a pPMfolC verified by <br /> irradiated UV light, plasmid-encoded proteins       complementation strain SF4 methionine prototrophy. <br /> labelled Tran3 S-label (methionine cysteine)             Strain MM383::pCP3 transformed tetracycline re- <br /> described Sancar et al. (26). Sodium dodecyl sulfate-          sistance plasmids described above. The Tetr <br /> polyacrylamide gel electrophoresis performed 13%         transformants grown absence ampicillin to <br /> polyacrylamide gels according method Laemmli            screen segregants integrated pCP3 plasmid. Wild- <br /> (18), gels fluorographed.                            type segregants obtained transformants, but <br />                                                                   Amp&apos; Kmr segregants obtained transfor- <br />                           RESULTS                                 mants containing pPMfolC pPMpac3a plasmids. <br />                                                                   These strains designated MM383AfolC/pPMfolC and <br />    Construction plasmid pCP3. The construction pCP3      MM383AfolC/pPMpac3a, respectively. No Amp&apos; Kmr seg- <br /> shown Fig. 1. All plasmid constructs verified          regants observed strains transformed with <br /> restriction enzyme analysis, including redigestion pCP3        pPMusg pPMdedD, cells grown and <br /> with StuI (data shown).                                       selected rich medium supplemented folate end <br />    The gene products expressed pCP3 analyzed         products methionine, glycine, thymidine, adenosine, and <br /> maxicells (Fig. 2, lane 3). The pCP3 plasmid expressed        pantothenate. The Amp&apos; Kmr segregants containing <br /> labelled protein products expressed parent plasmid         pPMfolC grew normally, dedD gene was <br /> H2al (lane 4) did express products foiC        deleted present complementing plasmid. No <br /> gene, dedD gene, TnS Kmr gene (kan 2 Fig. 2),       difference phenotype observed result the <br /> since genes deleted construction pCP3.       deletion dedD gene. <br /> ThefolC product (molecular mass, 47 kDa) shown lanes           The deletion chromosomal foiC gene strain <br /> 2, 4, 5. The dedD product (35 kDa) shown lane 4     MM383AfolC/pPMpac3 verified Southern blotting. <br /> is present faint band lane 2. The product     Figure 3B, lane 7, shows MM383::pCP3 integrant 1 contain- <br /> Kmr gene pUC4K (lane 1) appears             ing plasmid pPMpac3a. The plasmid-specific bands hybrid- <br /> mobility ,-lactamase visible discrete      izing probe shown lane 9. Lane 8 shows the <br /> band.                                                             corresponding segregant, MM383AfolC/pPMpac3a, which <br />    Integration pCP3 deletion chromosomal se-            pPMpac3a-specific bands seen 5-kb <br /> quences. Our strategy disruption chromosomal           fragments integrant absent. Also present lane 8 <br /> folC gene shown Fig. 3A. pCP3 contains selectable         8.3- 2.7-kb fragments expected segregant in <br /> Kmr marker replacing deletedfolC dedD genes,          chromosomal region containing foiC dedD <br /> the marker flanked extensive E. coli genomic DNA            genes replaced Kmr marker (Fig. 3A, left). <br /> sequences required homologous recombination. pCP3                Replacement thefoiC gene vitro-generated mutant <br /> was used transform polA(Ts) strain MM383 ampi-          genes. One objectives deleting folC gene to <br /> cillin kanamycin resistance. We tried times       express mutantfolC genes produced vitro mutagenesis <br /> were unable isolate plasmid transformants          strain chromosomally encoded background <br /> MM383, suggesting plasmids directly inte-        enzyme activity. Since mutants constructed <br /> grated chromosome. Integration intact pCP3       ColEl replicons, necessary transduce chro- <br /> the chromosomes Ampr Kmr transformants verified            mosomal foiC deletion PolA+ strain using bacte- <br /> by Southern blot analysis (Fig. 3B). The pCP3 probe hybrid-       riophage P1 described Materials Methods. We <br /> ized 9-kb 2.7-kb EcoRI fragment host strain     selected Kmr transductants strain SF2 containing <br /> (lanes 1 2). The recombination event occur           pPMpac3a plasmid. We obtained Kmr transductants if <br /> either deletion (Fig. 3A), possible          recipient strain did contain complementing <br /> integrants obtained. Fig. 3B, lane 3, shows integrant 2,     pPMpac3a plasmid. The loss chromosomal foiC gene <br /> in hybridizing fragments 2.7, 5, 5.7, 8.3 kb.   strains verified Southern blot analysis. For <br /> Integrant 1 shown lanes 4 6 hybridizing             blot shown Fig. 3C BamHI-HindIII fragment the <br /> fragments 2.7 9 kb 5 kb. Following growth       plasmid pACS (Fig. 5) used afolC-specific probe. Lanes <br /> of integrants, designated MM383::pCP3, absence         1 2 strain SF2AfolC/pPMpac3a. The probe hybrid- <br />  VOL. 174, 1992                                          REPLACEMENT OF folC WITH IN VITRO-MUTAGENIZED GENES                                1753 <br />  <br />  <br />  <br />                                               foIC dedD <br />                                  usg                        Stul <br />                               EcoRi <br />  <br />  <br />                            EcoRI <br />                            EcoRI                     pH2al <br />                              Hindlil <br />  <br />  <br />                                         Sphi                        EcoRil                         pUC4K <br />                                    Sphl digestion <br />                                           Ligation                                         BamHI                BamHI <br />                                                      pCP1                                              KAN <br />                                  Stul digestion <br />                                        Ugation <br />  <br />                                          usg                         Stul                  uS <br />                                                                                             911 <br />                                             /EcoRI \                                 EcoRI         / <br />  <br />                                                                                      usg <br />                            EcoR         I        pCP2                                                  pUGBgl <br />  <br />  <br />  <br />                                        indill                        ind &gt;l <br />  <br />                                                         Stul digestion                  Stul digestion <br />  <br />                                                             AGGCCTGAGATCC           Ligation <br />  <br />  <br />                                                               ECOfi S u/                   Stul <br />                                                                      gusg <br />  <br />  <br />                                                       EcoRI                  pCP3 <br />                                                       EcoRI <br />                                                       Hindlill <br />                                                           Sphl <br />                                                                             c o <br />                                                                                   AM     indlill <br />                                                                                    R 1~EoR <br />    FIG. 1. Construction pCP3. Plasmid pH2A-1 (5) digested SphI religated form pCP1. This procedure removed 1-kb <br /> fragment pKC7 vector portion plasmid inactivated kanamycin resistance gene. pCP1 digested StuI and <br /> religated form pCP2. This procedure removed 2-kb DNA fragment containing folC gene adjacent downstream gene (dedD) <br /> (23). The deletion removed 5 bases, including termination codon usg. A Km&apos; cassette inserted pCP2 <br /> was constructed follows: plasmid pUC4K (Pharmacia) digested BamHI, 1.3-kb DNA fragment containing kanamycin <br /> phosphotransferase gene isolated agarose gel ligated BglII site pUGBgl (6). pUGBgl linker consisting the <br /> BglII site flanked StuI sites, contains DNA sequences provide proper termination upstream gene allowing <br /> insertions BglII site (6). The kanamycin resistance cassette cloned BglII site pUGBgl excised StuI digestion that <br /> it flanked linker sequences. The DNA fragment gel purified ligated StuI site pCP2 form pCP3. pCP3 was <br /> transformed competent E. coli cells, Kmr Ampr colonies selected. DNA sequences follows:             , vector; _, E. coli; <br /> EL1, open reading frames specific genes. The DNA sequence StuI BglII-BamHI sites adjacent usg shown. The TGA <br /> stop codon frame usg provide correct translation termination highlighted. <br />  1754            PYNE AND BOGNAR                                                                                              J. BACTERIOL. <br />  <br />                   1   2     3      4      5                             transformants grown 37&#194;&#176;C absence tetracy- <br />                                                                         cline replica plated tetracycline-supplemented <br />        93-                                                              medium screen segregation pPMpac3a plasmid. <br />        68..                                                             Most colonies pAC5-transformed cells were <br />                                                  4 pur F                tetracycline sensitive, indicating pPMpac3a plasmid <br />        43-                                       -4folC                 segregated, pUC19-transformed cells <br />                                                  w hisT-usg             tetracycline sensitive. The pPMpac3a plasmid segre- <br />        30._                                      .-ded D                gated transformants containing codon 309 point <br />                                                   wusg                  mutants plated complex medium. The <br />                                                   * aamp  kan 1         SF2AfolC strains containing pVal, pArg, pLeu, pGlu <br />                                                      kan h <br />        21   -                                                           grew slowly wild type. SF2AfolC strains <br />                                                                         transformed linker insertion mutants (all which <br />        14                                                               &lt;5% wild-type FPGS dihydrofolate synthetase <br />                                                                         activity) (17) unable segregate pPMpac3a plas- <br />    FIG. 2. Expression plasmid-dependent proteins maxicells.       mid grown complex medium. <br /> DR1984 cells transformed pUC4K (lane 1), pAC3 (lane 2),          In experiment, pPMpac3a plasmid segregated <br /> pCP3 (lane 3), pH2al (lane 4), pAC5 (lane 5). All plasmids      SF2AfolC/pGT3-8.1, (containing L. casei FPGS <br /> described Table 1. Sodium dodecyl sulfate-polyacrylamide gels     gene). SF2AfolC/pGT3-8.1 grew complex medium <br /> total cell extracts autoradiographed detect proteins labelled   stably maintained, able to <br /> with Tran35S-label (methionine cysteine). The positions the <br /> molecular weight markers indicated left. The positions    repeat transformation segregation experiment that <br /> identified gene products shown right. kan 1, kanamycin       produced strain. The loss chromosomal and <br /> resistance gene pUC4K derived Tn9O3; kan 2, kanamycin         pPMpac3a-encoded folC genes Tets strains from <br /> resistance gene, derived TnS, present pH2al; hisT-usg,          pPMpac3a segregated verified Southern <br /> product gene upstream hisT (23). The sample lane 5       blotting. Figure 3B, lanes 4 6, shows thefolC-specific <br /> from separate experiment higher background labelled         probe hybridized 4- 4.7-kb DNA fragments spe- <br /> cellular proteins.                                                      cific replacing plasmids 2.6-kb <br />                                                                         pPMpac3a plasmid-specific fragment (lane 2) chromo- <br />                                                                         somal integrant. Lane 4 contained DNA <br /> ized 4.5-kb EcoRI 2.7-kb EcoRI-Smal                 lanes 5 6 did, L. casei FPGS gene-specific <br /> fragments corresponding pPMpac3a-encoded folC                    DNA hybridized weakly folC probe. To verify <br /> gene 9-kb chromosomal EcoRI fragment                SF2AfolC/pGT3-8.1 contained L. casei FPGS <br /> the parental strain, SF2 (shown lane 3). There SmaI           gene, extracts strain controls assayed for <br /> sites chromosomal EcoRI fragment (23).                      FPGS dihydrofolate synthetase activity. Table 2 shows <br />    SF2AfolC/pPMpac3a transformed ampicillin resis-               SF2AfolC/pGT8-1 high expression FPGS <br /> tance plasmids pUC19 pAC5 (wild-type folC gene                 activity,withsixfold-higheractivitywith5,10-methylenetetra- <br /> in pUC19) plasmids containing mutatedfolC genes pUC               hydrofolate 10-formyl tetrahydrofolate, similar to <br /> vectors (Table 1). pL2, pIA, pL5, pL6, pL8, pLlO               specificity enzyme purified L. casei. The <br /> derived pAC5 contain 2-amino-acid linker inser-                dihydrofolate synthetase specific activity of <br /> tions various points foiC coding sequence (17).               wild-type E. coli, indicating chromosomal and <br /> Plasmids pSF4, pGly, pIleu, pVal, pArg, pGlu, pLeu                  pPMpac3a foiC genes absent. The apparent dihydrofo- <br /> contain point mutations codon 309 foiC (the site           late synthetase activity L. casei FPGS in <br /> mutation SF4), DNA inserts contain                  traces folate pteroate substrate. Table 3 <br /> entire dedD gene (16). Plasmid pGT3-8.1 contains L.                 shows incorporation 14C-para-aminobenzoic acid the <br /> casei FPGS gene (30). All mutant plasmids tested,                cells, measure folate biosynthesis and <br /> except pArg, pLeu, pGlu, transformed strain SF4                  dihydrofolate synthetase activity. The incorporation label <br /> methionine prototrophy. The SF2AfolC/pPMpac3a Ampr                      SF2AfolC/pGT3-8.1 50-fold lower the <br />  <br />  <br />    FIG. 3. Integration pCP3 segregation E. coli chromosome. (A) Above pCP3 plasmid EcoRI restriction map <br /> of E. coli chromosomal DNA vicinity foiC gene. The region containing foiC dedD genes deleted the <br /> E. coli DNA pCP3 boxed. The recombination events (1 2) gave rise integrants shown. Immediately plasmid <br /> are EcoRI restriction maps E. coli chromosomal DNA integrants 1 2. The lowermost level shows maps following <br /> segregation plasmid: map left shows chromosomal DNA foiC gene deleted replaced Kmr marker; <br /> the map right shows wild-type segregant (WT). ThefoiC, dedD, Kmr, Ampr genes boxed. The sizes restriction fragments <br /> that hybridized probe linearized radiolabelled pCP3 plasmid indicated restriction maps host strain; the <br /> integrants thefolC-deleted segregant indicated divergent arrowheads. The restriction fragment hybridized thefolC-specific <br /> gene probe (the BamHI-HindIII fragment pAC5, shown Fig. 5) indicated restriction map wild-type segregant. (B) <br /> Southern hybridizations wild-type, integrant, segregant genomic DNA. Chromosomal DNA digested EcoRI probed with <br /> pCP3 plasmid DNA. Lanes 1 2, MM383; lane 3, MM383::pCP3 integrant 2; lanes 4 6, MM383::pCP3 integrant 1; lane 7, <br /> MM383::pCP3/pPMpac3a (same lane 6 containing plasmid pPMpac3a); lane 8, MM383AfolC/pPMpac3a; lane 9, pPMpac3a plasmid <br /> DNA (see Fig. 4 restriction map). The arrowhead indicates position 8.3-kbp band lane 8 present the <br /> chromosomal folC deleted. (C) Southern hybridizations wild-type folC-deleted strains containing complementing plasmids. <br /> Chromosomal DNA digested EcoRI (lane 1) EcoRI SmaI (lanes 2 6) probed polyacrylamide gel-purified and <br /> radiolabelled pAC5 insert DNA (Fig. 5), specific thefolC coding sequence. Lanes 1 2, SF2AfolC/pPMpac3a; lane 3, SF2; lane <br /> 4, SF2AfolC/pGT3-8.1; lane 5, SF2AfolC/pSF4; lane 6, SF2AfolC/pAC5. pPMpac3a described Fig. 4 legend. No additional bands <br /> corresponding wild-type genomic fragment integrated plasmids observed sixfold longer exposure autoradiogram. <br />  VOL. 174, 1992                                                  REPLACEMENT OF folC WITH IN VITRO-MUTAGENIZED GENES                                      1755 <br />  <br />  <br />  <br />  <br />                                                            -2.7 kbE     - <br />                                                                                         9 kb <br />  <br />  <br />                                      A <br />  <br />  <br />  <br />  <br />                   EE       E <br />                                    5.0 kb 5.0 kb <br />                                          E <br />                                                  1 <br />  <br />                                                      EE   -EK <br />                                                                   9kb <br />                                                                             E <br />                                                                                     I          E E <br />                                                                                                      5.7kb <br />  <br />                                                                                                              E <br />                                                                                                                   5.0kb <br />                                                                                                                        2 <br />  <br />                                                                                                                             EE   E <br />                                                                                                                                          8.3kb <br />  <br />                                                                                                                                                  E <br />                   II -S     a_          Am                -t            I                          111A <br />                                                                                                    TO kIJ <br />  <br />  <br />  <br />  <br />                        E <br />                        II <br />                                     2.7kb <br />                                              E <br />                                                           8.3kb <br />                                                                                 E <br />                                                                                     I          E E <br />                                                                                                1 <br />                                                                                                              E <br />                                                                                                              I <br />                                                                                                                                                      E <br />  <br />  <br />  <br />                                                                                                                  -qc             9.0kb <br />  <br />                                                  A FOL                                                                 WT <br />  <br />                            B l                                                             cl 2              3             456 <br />                  23 kb. <br />                   9.4-_1 <br />                   6.5 _. <br />                                                                                                                                             _23kb <br />                                                                                                                                             - 9.4 <br />                   4.3                                                                                                                       - 6.5 <br />  <br />                                                                                                                                             - 4.0 <br />  <br />                   2.3          - <br />  <br />  <br />  <br />                                                                                                                                             -    2.3 <br />                                                                                                                                             -    2.0 <br />  1756     PYNE AND BOGNAR                                                                                                       J. BACTrERIOL. <br />  <br />  <br />  <br />  <br />                                                                                             Xhol <br />                                                                                             paral Sail <br />                                                                                             religate <br />  <br />  <br />  <br />  <br />   FIG. 4. Construction plasmids pPM103. Plasmid pAC3 digested SmaI HindIII. The cohesive ends produced HindIII <br /> were filled Klenow fragment DNA polymerase I, DNA ligated pPM103 digested SmaI dephosphorylated <br /> with calf intestine alkaline phosphatase. The ligated plasmids used transform E. coli JF1754 selecting tetracycline resistance and <br /> ampicillin sensitivity. The resultant plasmid, designated pPMpac3, obtained possible orientations, pPMpac3a pPMpac3b. To <br /> construct pPMusg, pPMpac3a digested XhoI partially Sall religated. Transformants retained tetracycline <br /> resistance, indicating SalI site tet gene intact, selected. The digestion removes dedD gene half folC <br /> gene. The religated plasmid pPMusg retained upstream gene promoter. To construct pPMdedD, pPMpac3b digested XhoI <br /> and partially Sall religated. The upstream gene half folC gene deleted. The religated plasmid pPMdedD <br /> contained dedD gene promoter, located 3&apos; half foiC gene. DNA sequences follows:           , vector; LII, open <br /> reading frames E. coli genes; &apos;-, antibiotic resistance genes. <br />  <br />  <br /> SF4 mutant strain 500-fold lower wild-type            grown minimal medium, liquid solidified, <br /> E. coli. These results consistent absence                containing various nutritional supplements folate end <br /> dihydrofolate synthetase activity SF2AfoiC strain             products. SF2AfoiC strains containing plasmids pAC5 (wild- <br /> with dihydrofolate synthetase activity L. casei FPGS            typefolC), pGly, pSF4, plleu, pVal able grow on <br /> gene product 0.2 12% present              minimal medium, growth rates two <br /> wild-type E. coli cells.                                                 lower (16). SF2AfoiC/pArg produced microcolonies on <br />   The SF2AfolC strains containing mutant folC genes                 solidified minimal medium required methionine and <br /> which complementing pPM103 plasmid segregated                    glycine growth liquid medium. The SF2AfoIC/pGlu <br />  VOL. 174, 1992                                       REPLACEMENT OF folC WITH IN VITRO-MUTAGENIZED GENES                              1757 <br />  <br />  <br />  <br />  <br />   FIG. 5. Construction pPMfolC. Plasmid pAC5 digested PvuII, fragment containing folC gene downstream the <br /> lac promoter isolated agarose gel ligated pUC19 digested Sma I. The plasmids isolated Ampr transformants, <br /> and orientation insert excised EcoRI fragment selected. This plasmid, pAC6, digested <br /> with EcoRI ligated pPM103 linearized EcoRI. Tetr transformants E. coli JF1754 selected pooled, plasmid <br /> DNA isolated. The plasmids pool used transform E. coli SF2 methionine prototrophy, selecting plasmids <br /> containing functional folC gene. The Tetr colonies obtained Ampr. Restriction analysis plasmids isolated these <br /> transformants showed hybrid plasmid, pPMfolC/amp, caused ligation linearized plasmids EcoRI <br /> digest pAC6 partial digest. pPMfolC/amp digested PvuI religated delete Ampr gene. The resultant plasmid, <br /> pPMfolC, replicate MM383 poL4 strain E. coli complement folC mutation strain SF4. Transformants of <br /> pPMfolC Tetr AmpS. DNA sequences follows:            , pPM103; _, pUC19; El, open reading frames E. coli genes; &apos;, <br /> antibiotic resistance genes. Open arrowheads indicate lac promoter. <br />  <br /> strain required supplements methionine, glycine, thy-             gene viable unless deleted gene comple- <br /> midine, adenosine growth. The SF2AfolC/pLeu                    mented functional plasmid-encoded gene. Transduc- <br /> strain SF2AfolC/pGT3-8.1 strain required            tion chromosomal deletion using Kmr marker <br /> folate end products (including pantothenate) growth.               required presence complementing gene the <br />                                                                        recipient strain. The presence thyA mutation E. coli <br />                          DISCUSSION                                    allows deletion thefoLA gene encoding dihydrofolate <br />                                                                        reductase (14), deletion offolC lethal thyA <br />    Our results indicate thefoiC gene essential gene         strain. It sufficient include end products of <br /> in E. coli. Segregants deleted chromosomal folC              folate-dependent biosynthetic pathways medium to <br />  1758        PYNE AND BOGNAR                                                                                                                J. BACTIEkRIOL. <br />  <br />        TABLE 2. FPGS dihydrofolate synthetase E. coli                         biosynthesis, biosynthesis nucleotides pantoth- <br />                   strains transformantsa                                         enate sensitive. Studies measure levels of <br />                                                 Relative sp act&apos;                     folate strains described progress will <br />  Strain/transformant                                                                 allow determine folate concentrations limit <br />                               H2Pte      5,10-CH2-H4Pteglu     10-CHO-H4Pteglu       growth cell functioning folate- <br /> W1485C                         1                   1                     1           dependent biosynthetic pathway. <br /> SF2AfolC/pPMpac3               1.3                 1.1                   5.3            The strain containing L. casei gene, SF2AfolC/pGT3- <br /> SF2AfolC/pAC5                180                 134                   139           8.1, expresses levels FPGS activity 1.8-fold <br /> SF2AfolC/pGT3-8.1              0.12            1,438                   248           higher SF2AJolC/pAC5 E. coli foiC <br />   Experimental details described Materials Methods.                     gene vector. However, SF2AfolCIpGT3-8.1 is <br />   b Abbreviations: H2Pte, dihydropteroate; 5,10-CH2-H4Pteglu, 5,10-methyl-           auxotrophic folate end products. This auxotro- <br /> enetetrahydrofolate; 10-CHO-H4Pteglu, 10-formyl-tetrahydrofolate.                    phy probably extremely low levels dihydrofolate <br />   c Typical specific activities extracts 1, 1.8, 5.9 nmol/h/mg     synthetase activity L. casei enzyme. The incorpora- <br /> with H2Pte, 5,10-CH2H4Pteglu, 10-CHO-H4Pteglu, respectively.                     tion ofpara-aminobenzoic acid, measure folate synthe- <br />                                                                                      sis, 0.2% wild type SF2AfolC/pGT3-8.1. <br />                                                                                      The dihydrofolate synthetase-specific activity L. casei <br /> complement folC function, FPGS-deficient                             enzyme (Table 2) 1/10,000 E. coli enzyme, <br /> Chinese hamster ovary cells (29). The folate requirement                         high expression L. casei gene, total <br /> synthesis formyl-methionyl-tRNA initiation                             enzyme activity sufficient account the <br /> protein synthesis satisfied supple-                            viability SF2AfolC/pGT3-8.1. Preliminary results from <br /> ments (11). Since biosynthesis folate requires dihy-                          extraction high-performance liquid chromatography of <br /> drofolate synthetase, dihydrofolate synthetase activity                       cellular folates suggest SF2AfolC/pGT3-8.1 does contain <br /> the folC gene product essential. We                                  folates (30), favoring interpretation. The L. casei gene in <br /> obtained folC mutants dihydrofolate syn-                              strain undergone mutation some <br /> thetase activity FPGS activity,                               dihydrofolate synthetase activity, been <br /> determine dihydrofolate synthetase activity                         able repeat experiment constructed this <br /> sufficient cell viability. The loss FPGS                            strain. An alternate possibility E. coli genome in <br /> simply result methionine auxotrophy (11).                                         recipient SF2AfolC transformed pGT3-8.1 <br />    A major application strains constructed                         undergone second mutation, making able initiate <br /> as recipient strains mutant foiC genes, synthesized                           protein synthesis formyl methionine. Such mutant <br /> vitro, allow examination mutants absence                         reported literature (1), been <br /> of background activity chromosomal gene. We                               subsequently lost. <br /> have constructed strains strain the <br /> FPGS gene L. casei replaced folC gene. We                                              ACKNOWLEDGMENTS <br /> be able purify mutant enzymes study                                   This work supported grant MA-9822 Medical <br /> properties. The mutant genes present high copy                           Research Council Canada. <br /> number, enzymes expressed 4%                                  We thank J. B. McNeil S. McCracken reading the <br /> soluble cellular protein, strains complemented                         manuscript. <br /> the pGlu, pArg, pLeu mutant genes, enzyme-specific                                                      REFERENCES <br /> activity cells lower wild type <br /> (16). The lower specific activity enzyme was,                          1. Baumstark, B. R., L. L. Spremulli, U. L. R#jBhandary, and <br /> more supplements folate end products required                                G. M. Brown. 1977. Initiation protein synthesis without <br /> growth. The decrease enzyme activity IB1 (SF2                              formylation mutant Escherichia coli grows the <br />                                                                                          absence tetrahydrofolate. J. Bacteriol. 129:457-471. <br /> SF4) strain resulted 70% decrease intracellular folate                        2. Birnboim, H. C., J. Doly. 1979. A rapid alkaline extraction <br /> pools (11). The level enzyme activity lower                               procedure screening recombinant plasmid DNA. Nucleic <br /> the SF2AfolC/pArg, SF2AfolC/pLeu, SF2AfolC/pGlu                                      Acids Res. 7:1513-1523. <br /> strains SF2, suggesting folate pools                              3. Blum, P., D. Holzschu, H.-S. Kwan, D. Riggs, S. Artz. 1989. <br /> lower strains, resulting lower growth rates                          Gene replacement retrieval recombinant M13mp bac- <br /> auxotrophies. The methionine biosynthetic pathway appears                                teriophages. J. Bacteriol. 171:538-546. <br /> to sensitive low folate, followed glycine                               4. Bognar, A., C. Osborne, B. Shane. 1987. Primary structure <br />                                                                                          Escherichia coli folC gene folylpolyglutamate <br />                                                                                          synthetase-dihydrofolate synthetase product regulation of <br />                                                                                          expression upstream gene. J. Biol. Chem. 262:12337- <br />   TABLE 3. Incorporation para-amino[carboxyl-&apos;4C]benzoic                              12343. <br />                   acid E. coli strainsa                                          5. Bognar, A. L., C. Osborne, B. Shane, S. Singer, R. Ferone. <br />                                                                     pmol pABA <br />                                                                                          1983. Escherichia coli folylpolyglutamate synthetase-dihydrofo- <br />         Strain                                                     incorporated/mg       late synthetase, gene cloning high expression, purification <br />                                                                        cellsb         properties. J. Biol. Chem. 260:5625-5630. <br />                                                                                       6. Bognar, A. L., C. Pyne, M. Yu, G. Basi. 1989. Transcription <br /> W1485C .287                                                                              folC gene coding folylpolyglutamate synthetase- <br /> SF4 ........................................                            24               dihydrofolate synthetase Escherichia coli. J. Bacteriol. 171: <br /> SF2AfolC/pGT3-8.1 ........................................               0.53            1854-1861. <br />   Experimental details described Materials Methods.                      7. Bognar, A. L., B. Shane. 1983. Purification properties of <br />   b The <br />          SF2AfoIC/pGT3-8.1 cells grew 40% density             Lactobacillus casei folylpolyglutamate synthetase. J. Biol. <br /> strains; 1 ml cells optical density (at 550 nm) 1 assumed          Chem. 258:12574-12581. <br /> 1 mg cells. pABA, pam-aminobenzoic acid.                                           8. Driver, R. P., R. P. Lawther. 1985. Physical analysis of <br />   c The wild-type cells incorporated 68% labelled pABA                 deletion mutants ilvGEDA operon Eschenchia coli <br /> course experiment.                                                                K-12. J. Bacteriol. 162:598-606. <br />  VOL. 174, 1992                                         REPLACEMENT OF folC WITH IN VITRO-MUTAGENIZED GENES                                1759 <br />  <br />  9. Feinberg, A. P., B. Vogelstein. 1983. A technique            21. Miller, J. H. 1972. Experiments molecular genetics, p. <br />     radiolabelling DNA restriction endonuclease fragments high            203-205, 431-435. Cold Spring Harbor Laboratory, Cold Spring <br />     specific activity. Anal. Biochem. 132:6-13.                              Harbor, N.Y. <br /> 10. Ferone, R., M. H. Hanlon, S. C. Singer, D. F. Hunt. 1986.        22. Monk, M., J. Kinross. 1972. Conditional lethality recA <br />     a-Carboxyl linked glutamates folylpolyglutamates               recB derivatives strain Escherichia coli K-12 a <br />     Escherichia coli. J. Biol. Chem. 261:16356-16362.                        temperature-sensitive deoxyribonucleic acid polymerase I. J. <br /> 11. Ferone, R., S. C. Singer, M. H. Hanlon, S. Roland. 1983.             Bacteriol. 109:971-978. <br />     Isolation characterization E. coli mutant affected      23. Nonet, M. L., C. C. Marvel, D. Tolan. 1987. The hisT-purF <br />     dihydrofolate- folylpolyglutamate-synthetase, p. 585-589.            region Escherichia coli K12 chromosome. J. Biol. Chem. <br />     In J. A. Blair (ed.), Chemistry biology pteridi <br /> </body></html>