<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR AND CELLULAR BIOLOGY, May 2008, p. 3446&#226;&#8364;&#8220;3456                                                                            Vol. 28, No. 10 <br /> 0270-7306/08/$08.00&#226;&#171;&#185;0 doi:10.1128/MCB.02246-07 <br /> Copyright &#194;&#169; 2008, American Society Microbiology. All Rights Reserved. <br />  <br />  <br />  <br />       p23/Sba1p Protects Hsp90 Inhibitors Independently Its <br />                       Intrinsic Chaperone Activity&#228;&#338;&#164;&#226;&#8364;&#161; <br />                       Fedor Forafonov,1 Oyetunji A. Toogun,2&#226;&#8364;&#160; Iwona Grad,1 Elena Suslova,1 <br />                                     Brian C. Freeman,2 Didier Picard1* <br /> De&#194;&#180;partement Biologie Cellulaire, Universite&#194;&#180; Gene`ve, Sciences III, 30 Quai Ernest-Ansermet, CH-1211 Gene`ve 4, Switzerland,1 <br />                 Department Cell Developmental Biology, University Illinois, Urbana, Illinois 618012 <br />                       Received 19 December 2007/Returned modification 30 January 2008/Accepted 11 March 2008 <br />  <br />              The molecular chaperone Hsp90 assists subset cellular proteins essential eukaryotes. A cohort <br />           cochaperones contributes regulates multicomponent Hsp90 machine. Unlike biochemical <br />           activities cochaperone p23, vivo functions structure-function relationship remain poorly <br />           understood, genetically tractable model organism Saccharomyces cerevisiae. The SBA1 gene that <br />           encodes p23 ortholog species essential gene. We absence p23/Sba1p, <br />           yeast mammalian cells hypersensitive Hsp90 inhibitors. This protective function Sba1p depends <br />           abilities bind Hsp90 block Hsp90 ATPase inhibitor binding. In contrast, protective <br />           function Sba1p does require Hsp90-independent molecular chaperone activity Sba1p. The <br />           structure-function analysis suggests Sba1p undergoes considerable structural rearrangements upon <br />           binding Hsp90 large size p23/Sba1p-Hsp90 interaction surface facilitates maintenance of <br />           high affinity despite sequence divergence evolution. The large interface contribute preserv- <br />           ing protective function environment Hsp90 inhibitory compounds produced various <br />           microorganisms. <br />  <br />  <br />    Sba1p ortholog (4, 15) Hsp90 cochaperone p23,                  conformation lid ATP binding pocket its <br /> a small acidic eukaryote-specific protein, budding yeast                 closed conformation, providing explanation stabiliz- <br /> Saccharomyces cerevisiae (reviewed references 16 46).                    ing effects Sba1p (2). <br /> The molecular chaperone Hsp90 highly conserved                            Despite regulatory interactions p23/Sba1p and <br /> abundant cytosolic nuclear protein required                     Hsp90, Hsp90 absolutely essential viability. Deletion <br /> folding, assembly, maintenance subset proteins (23,                 mutants p23 orthologs budding fission yeasts are <br /> 44&#226;&#8364;&#8220;46, 62). The activity N-terminal ATPase domain                     viable (4, 15, 38). Similarly, p23-null mice initially develop <br /> regulated cochaperones. Although biochemical                     relatively normally dying birth retarded <br /> function ATP hydrolysis Hsp90 substrates under-                   lung development (22). Overall, vivo functions p23/ <br /> stood, genetic experiments budding yeast unambiguously                       Sba1p remain poorly understood. For S. cerevisiae, early re- <br /> demonstrated important                        ports role SBA1 &#226;&#8364;&#339;general control response&#226;&#8364;? <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> <br /> substrates essential viability (42). p23 binds                 amino acid starvation (14) maintaining chromosome <br /> ATP-bound conformation molecular chaperone                               stability (39) investigated. Most re- <br /> Hsp90, inhibits ATP hydrolysis, and, result stabilizing                 ported defects &#226;&#338;&#172;sba1 cells relate functions verte- <br /> the ATP-bound state, prolongs interaction                           brate substrates Hsp90 assayed heterologous envi- <br /> Hsp90 substrates (11, 24, 26, 32, 33, 50&#226;&#8364;&#8220;52,                    ronment (4, 7, 8, 13, 15, 17, 20, 28, 40). Indeed, name <br /> 56, 58, 60). The effects Hsp90 inhibitors geldana-                   gene SBA1 (sensitivity benzoquinone ansamycins) <br /> mycin (GA) radicicol, compete ATP                            derives fact mammalian steroid receptors are <br /> binding, compounded interfering binding                      sensitive inhibition Hsp90 inhibitors chem- <br /> p23/Sba1p (15, 26). The recently reported crystal struc-                   ical class tested &#226;&#338;&#172;sba1 strain, genuine yeast <br /> ture Sba1p-Hsp90 complex shows intimate contacts in-                     functions proliferation examined initial <br /> volving multiple regions Sba1p N-terminal                   report (4). An essential role Sba1p maintaining telomeres <br /> middle domains Hsp90. In complex, consists                      promoting dynamic interactions telomerase <br /> two Sba1p monomers Hsp90 dimer, Sba1p favors Hsp90                       telomeric repeats recently recognized <br />                                                                                 (59). This explain previously reported chromosome <br />                                                                                 instability cells overexpressing Sba1p. However, not <br />    * Corresponding author. Mailing address: De&#194;&#180;partement Biologie            understood Sba1p requirement, manifested im- <br /> Cellulaire, Universite&#194;&#180; Gene`ve, Sciences III, CH-1211 Gene`ve 4,            mediately following deletion SBA1 gene, somehow <br /> Switzerland. Phone: 41 22 379 6813. Fax: 41 22 379 6928. E-mail:                compensated seemingly long-term estab- <br /> didier.picard@cellbio.unige.ch.                                                 lishment &#226;&#338;&#172;sba1 strains. <br />    &#226;&#8364;&#160; Passed away 3 August 2007. <br />    &#226;&#8364;&#161; Supplemental material article http://mcb <br />                                                                                    Hence, role Sba1p yeast biology itself the <br /> .asm.org/.                                                                      contributions different Sba1p domains functions remain <br />    &#228;&#338;&#164; <br />      Published ahead print 24 March 2008.                                 poorly understood. For example, relevance Hsp90- <br />  <br />                                                                          3446 <br />  VOL. 28, 2008                                                                                    p23 PROTECTS AGAINST Hsp90 INHIBITORS                                3447 <br />  <br />  <br /> independent molecular chaperone function,                               4 12% NuPAGE gels (Invitrogen) morpholineethanesulfonic acid (MES) <br /> described human p23 (5, 19, 61), remains unclear. It                           buffer analyzed immunoblotting mouse monoclonal anti- <br />                                                                                        Flag M2 antibody (Sigma) chicken polyclonal antibody preparation <br /> contribute maturation specific Hsp90 substrates (40),                        raised yeast Hsp82p (30). <br /> but importance endogenous yeast processes                                 Production purification recombinant proteins. The wild-type point <br /> been addressed. We set identify new phe-                            mutant Sba1p proteins expressed tag Rosetta (Novagen) <br /> notype S. cerevisiae strains lacking Sba1p character-                       bacteria using pET expression constructs. The proteins purified essentially <br />                                                                                        described human p23 (19). Briefly, crude clarified protein extracts were <br /> ize role Hsp90 regulation Sba1p chaperone activity <br />                                                                                        resolved DEAE FF resin (GE Healthcare) using linear gradient 200 to <br /> for <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span>phenotype</span>.                                                                    650 mM NaCl. The appropriate fractions pooled, diluted threefold with <br />                                                                                        TEN0.0 buffer (10 mM Tris-HCl [pH 6.9], 1.0 mM EDTA, 1.0 mM phenylmeth- <br />                                                                                        ylsulfonyl fluoride [PMSF], 1.0 &#226;?&#174;g/ml pepstatin, 1.0 &#226;?&#174;g/ml leupeptin, 0.25 &#226;?&#174;M <br />                          MATERIALS AND METHODS                                         aprotinin), resolved ResourceQ resin (GE Healthcare) linear <br />    Yeast strains. The strain BY4742 (MAT&#226;?&#163; his3&#226;&#338;&#172;1 leu2&#226;&#338;&#172;0 lys2&#226;&#338;&#172;0 ura3&#226;&#338;&#172;0) (6)             gradient 100 400 mM NaCl. The appropriate fractions pooled, the <br /> was used wild-type strain. This strain, &#226;&#338;&#172;sba1 (MAT&#226;?&#163; his3&#226;&#338;&#172;1 leu2&#226;&#338;&#172;0       total volume reduced &#226;&#172;&#402;200 &#226;?&#174;l using microconcentrator device a <br /> lys2&#226;&#338;&#172;0 ura3&#226;&#338;&#172;0 sba1&#226;&#338;&#172;::kanMX4), &#226;&#338;&#172;hsc82, &#226;&#338;&#172;hsp82 derivatives obtained             10-kDa cutoff (Amicon Ultrafree-4; Millipore), resolved Superdex- <br /> <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> Research Genetics used generate &#226;&#338;&#172;pdr5 derivatives BYP1                 200 gel filtration column (GE Healthcare) equilibrated TEN0.1 buffer (10 mM <br /> (BY4742 pdr5&#226;&#338;&#172;::HIS3) BYP2 (BY4742 pdr5&#226;&#338;&#172;::HIS3 sba1&#226;&#338;&#172;::kanMX4), re-                  Tris-HCl [pH 6.9], 1.0 mM EDTA, 100 mM NaCl). <br /> spectively. To produce PCR product substitute HIS3 coding region             An N-terminal polyhistidine-tagged Hsp82p fusion protein expressed in <br /> that PDR5 gene homologous recombination, HIS3 sequences                 Rosetta (Novagen) bacteria using pET expression construct (pET-His6HSP82), <br /> amplified plasmid pRS313 using primers 5&#226;&#172;&#732;-agacccttttaagttttcgtatc         kind gift J. Buchner (Technische Universita&#194;&#168;t, Garching, Ger- <br /> cgctcgttcgaaagactttagacaaaaatgACAGAGCAGAAAGCCCT-3&#226;&#172;&#732; 5&#226;&#172;&#732;-tctttcgga                    many). To purify protein near homogeneity, crude clarified protein <br /> cattgaactttgatttatcagagctggtaaaACTCAACCCTATCTCGGTCT-3&#226;&#172;&#732;. The lower-                     extract batch adsorbed Talon metal affinity resin (Clontech). The resin <br /> case letters represent flanking sequences PDR5 open reading frame,              washed 1&#226;&#171;&#187; Talon binding/wash buffer (50 mM sodium phosphate [pH <br /> including ATG (boldface) 5&#226;&#172;&#732; nucleotides &#226;&#171;&#185;4686 &#226;&#171;&#185;4725             7.0], 300 mM NaCl, 10% glycerol, 1.0 mM PMSF, 1.0 &#226;?&#174;g/ml pepstatin, 1.0 &#226;?&#174;g/ml <br /> located 160 bp 3&#226;&#172;&#732; stop codon. Uppercase letters correspond             leupeptin, 0.25 &#226;?&#174;M aprotinin), adsorbed protein eluted 1&#226;&#171;&#187; Talon <br /> sequences matching beginning HIS3 open reading frame se-                elution buffer (50 mM sodium phosphate [pH 7.0], 150 mM imidazole, 300 mM <br /> quences downstream plasmid pRS313.                                            NaCl, 10% glycerol, 1.0 mM PMSF, 1.0 &#226;?&#174;g/ml pepstatin, 1.0 &#226;?&#174;g/ml leupeptin, <br />    Plasmids. Wild-type Sba1p truncation point mutants ex-             0.25 &#226;?&#174;M aprotinin). As indicated Sba1p, eluted sample diluted <br /> pressed yeast Flag-tagged derivatives plasmid pYES/Flag (1). This           threefold TEN0.0 buffer, resolved ResourceQ resin, concentrated, and <br /> plasmid URA3-marked 2&#226;?&#174;m episome galactose-inducible promoter             Superdex-200 gel filtration column TEN0.1 buffer protease <br /> GAL1. SBA1 sequences generated PCR inserted BamHI                  inhibitors. <br /> and EcoRI sites pYES/Flag. For introduction single point mutations              Protein-refolding assay. The protein-refolding assays performed de- <br /> and deletion codons 102 113, SBA1 sequences cloned               scribed previously (18). Briefly, firefly luciferase (Sigma) prepared 4 <br /> BluescriptM13&#226;&#171;&#185; mutagenized QuikChange site-directed mutagenesis             mg/ml glycylglycine (pH 7.4) denatured 37&#194;&#176;C 30 min dilution <br /> method (Stratagene) directed manufacturer.                                   unfolding buffer (25 mM HEPES [pH 7.5], 50 mM KCl, 5.0 mM MgCl2, 5.0 <br />    Growth assays. For growth assays, strains precultured minimal selec-        mM mercaptoethanol, 6 M guanidinium-HCl). For molecular chaperone <br /> tive medium 2% glucose. Cultures washed diluted               activity assays, 1.0-&#226;?&#174;l aliquot (312.5 ng) denatured luciferase solution was <br /> optical density 600 nm 0.1 serially diluted times. Four microliters     added 124 &#226;?&#174;l refolding buffer (25 mM HEPES [pH 7.5], 50 mM KCl, 5.0 <br /> of dilution spotted row plates minimal selective medium          mM MgCl2, 10 mM DTT, 1.0 mM ATP) supplemented 3.2 &#226;?&#174;M <br /> containing 2% galactose. Plates supplemented radicicol GA 10         bovine serum albumin (BSA) wild-type point mutant Sba1p. The reactions <br /> mM stocks ethanol dimethyl sulfoxide (DMSO), respectively,              incubated 30 min 37&#194;&#176;C. Refolding triggered addition of <br /> solvent alone. Images acquired growth 5 days 30&#194;&#176;C.                   Hsp70 Hdj1 1.6 &#226;?&#174;M 3.2 &#226;?&#174;M, respectively, allowed proceed for <br />    Preparation yeast extracts. All Flag-tagged variants Sba1p ex-           2 h. Aliquots (1.0 &#226;?&#174;l) removed indicated time points mixed with <br /> pressed strain BYP2. Transformants precultured overnight selective          49 &#226;?&#174;l Bright-Glo luciferase assay reagent (Promega), luciferase activity <br /> minimal medium 2% glucose, washed water, incubated             determined using Ultra Evolution plate reader (Tecan). The percent <br /> for 6 h yeast extract-peptone (YEP) 2% galactose. Cell pellets 150 &#226;?&#174;l       luciferase activity calculated using equivalent native luciferase <br /> were resuspended volume lysis buffer (20 mM Tris-HCl [pH 8.0],          diluted extent refolding buffer supplemented 3.2 &#226;?&#174;M BSA. <br /> 1 mM EDTA, 100 mM NaCl, 20 mM Na2MoO4, 20% glycerol) containing yeast                   ATPase assay. The ATPase activity Hsp82p measured using Enz- <br /> protease inhibitor cocktail recommended supplier (Sigma). Cells         Chek phosphate assay kit (Molecular Probes) previously described (9, 52). <br /> mechanically broken Mini-BeadBeater (BioSpec Products) using 500 &#226;?&#174;l          Briefly, 1 &#226;?&#174;M Hsp82p monomer supplemented indicated <br /> 0.5-mm zirconia-silica beads. For cell lysis, tubes placed precooled         levels Sba1p point mutants 1 mM ATP presence or <br /> Mini-BeadBeater aluminum rack violently shaken 5 min 4&#194;&#176;C. The               absence 60 &#226;?&#174;M radicicol buffer C (40 mM HEPES [pH 7.5], 150 mM KCl, <br /> crude extracts cleared centrifugation 45,000 rpm 30 min 4&#194;&#176;C,         5 mM MgCl2, 2 mM DTT) total volume 150 &#226;?&#174;l. The reactions were <br /> and protein concentrations adjusted 1, 3, 5 mg/ml lysis buffer    incubated 37&#194;&#176;C 90 min diluted fivefold 100-&#226;?&#174;l phosphate <br /> the experiments Fig. 2B 3, 2C, 6, respectively.                             assay manufacturer&#226;&#8364;&#8482;s instructions. A standard curve generated <br />    Co-IP immunoblotting experiments. Flag-tagged proteins immuno-             using inorganic phosphate solution provided manufacturer. The radi- <br /> precipitated crude yeast extracts follows. One milligram (experiments       cicol control samples serve confirm phosphate release Hsp90 dependent <br /> Fig. 2B 3) 5 mg protein extracts supplemented ATP 10 mM          and, course, Sba1p itself displays ATPase activity all. <br /> and Triton X-100 0.2% incubated 20 &#226;?&#174;l packed EZview red                    Limited proteolytic digestion. Limited proteolytic digestion Sba1p pro- <br /> anti-Flag M2 affinity gel (Sigma) 3 h 4&#194;&#176;C. The resin collected          teins accomplished incubating (10 &#226;?&#174;g each) 50 ng trypsin in <br /> washed times buffer (lysis buffer protease inhibitors,         TEN0.1 buffer (10 mM Tris-HCl [pH 6.9], 1.0 mM EDTA, 100 mM NaCl). <br /> ATP, Triton X-100) unless indicated otherwise. Wash steps included change          Aliquots removed 0 15 min resolved sodium dodecyl <br /> of tubes ensure removal Hsp90 nonspecifically bound         sulfate-polyacrylamide gel electrophoresis, products visualized by <br /> plastic wall. The resin-bound immunoprecipitates incubated 1&#226;&#171;&#187; LDS            Coomassie blue staining. <br /> protein sample buffer (Invitrogen) 10 min 50&#194;&#176;C eluted protein               GA binding assay. Increasing levels recombinant Hsp82p (1 1,000 nM) <br /> supernatants adjusted 100 mM dithiothreitol (DTT) boiled 10            incubated indicated Sba1p protein (16 &#226;?&#174;M) 12.5 &#226;?&#174;l <br /> min. The coimmunoprecipitation (co-IP) experiment testing release           binding buffer (40 mM HEPES-KOH [pH 7.5], 150 mM KCl, 5.0 mM MgCl2, <br /> Hsp90 presence radicicol follows. The resin-bound immu-          2.0 mM DTT, 200 &#226;?&#174;M ATP). The chaperone reactions mixed 12.5 <br /> noprecipitates split parts wash: eluted   &#226;?&#174;l fluorescein isothiocyanate (FITC)-GA (InvivoGen) (8.0 nM FITC-GA, 40 <br /> and processed mentioned above, parts incubated IP         mM HEPES-KOH [pH 7.5], 150 mM KCl, 5.0 mM MgCl2) low-binding, <br /> buffer containing 10 mM ATP 10 &#226;?&#174;M radicicol 4&#194;&#176;C 20 min            black, 96-well plate (HE Microplate; Molecular Devices) incubated at <br /> then washed times ATP. Protein samples resolved             4&#194;&#176;C 120 min. The anisotropy values determined using Ultra <br />  3448       FORAFONOV ET AL.                                                                                                            MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />    FIG. 1. &#226;&#338;&#172;sba1 strains hypersensitive radicicol (Rd). Shown is <br /> a growth assay 10-fold serial dilutions cells isogenic <br /> strains indicated genotypes. <br />  <br />  <br /> Evolution plate reader (Tecan, Inc.). The apparent Hsp82 affinities GA in <br /> the absence presence Sba1p protein derivatives determined by <br /> fitting curve data replicates according formula <br /> m1 &#226;&#171;&#185; m2 &#226;&#171;&#185; m0 &#226;&#171;&#186; &#229;&#8224;&#8216;&#229;&#8230;&#177;m1 &#226;&#171;&#185; m2 &#226;&#171;&#185; m0&#229;&#8230;&#178;2 &#226;&#171;&#186; 4 &#226;&#171;&#187; m2 &#226;&#171;&#187; m0 <br />                                                                    &#226;&#171;&#187; m3 where <br />                               2 &#226;&#171;&#187; m2 <br /> m1 &#226;&#171;&#189; calculated dissociation constant (Kd); m2 &#226;&#171;&#189; total [FITC-GA]; m3 &#226;&#171;&#189; <br /> the maximal change anisotropy, set experiment (59). <br />    Assays MEFs. The isolation culture mouse embryo fibroblasts <br /> (MEFs) wild type corresponding p23 gene disruption mouse line <br /> were described (22). For proliferation assays, cells seeded into <br /> six-well plates triplicates density 100,000 cells (about 20% <br /> confluence). About 12 h later, day 0, number cells verified a <br /> CASY cell counter model TT 40 nM GA added. Cells counted 12 h <br /> later (day 0.5) 24-h intervals. To determine effects GA <br /> on levels Raf-1, cells seeded 4 &#226;&#171;&#187; 106 cells six-well      FIG. 2. p23-negative MEFs hypersensitive GA. (A) Growth <br /> plate 4 h prior treatment 60 nM GA. After time intervals, cells     curves wild-type (WT) p23-null (&#226;&#171;&#186;/&#226;&#171;&#186;) cells cultured or <br /> were lysed mixture containing 10 mM Tris-HCl (pH 8.0), 137 mM NaCl, 2          40 nM GA (added day 0). (B) Ratios numbers treated <br /> mM EDTA, 10% glycerol, 1% Triton X-100, protease inhibitors. Equal              untreated cells. Cell numbers experimental series were <br /> amounts total protein analyzed immunoblotting anti-Raf-1         standardized experimentally determined values day 0 (set to <br /> polyclonal antibody (C-12 Santa Cruz). The filter probed     100%). Error bars standard deviations (for clarity one <br /> monoclonal antibody ab8245 GAPDH (glyceraldehyde-3-phosphate de-            data points). <br /> hydrogenase; Abcam), served loading control. <br />  <br />  <br />                                  RESULTS <br />                                                                                     relatively low dose GA, p23-null cells rapidly stopped <br />    A novel &#226;&#338;&#172;sba1 phenotype. To explore vivo functions                     growing died, wild-type p23 cells continued <br /> Sba1p budding yeast S. cerevisiae, screened spe-                      grow 2 days survived longer (Fig. 2). <br /> cific growth defects &#226;&#338;&#172;sba1 strain panel different                      Numerous studies published demonstrating that <br /> growth media presence different drugs. The                       Hsp90 inhibitors induce apoptosis wide variety different <br /> defect observe simple assay                           cell types (see, example, references 10, 35, 62). It is <br /> hypersensitivity &#226;&#338;&#172;sba1 strain Hsp90 inhibitors                        likely MEFs died apoptosis, we <br /> radicicol GA (Fig. 1A; Fig. 3D). High concentra-                       confirmed assumption directly. The dramatic <br /> tions radicicol inhibitory, albeit weakly, iso-                  difference effects GA wild-type versus p23- <br /> genic wild-type strain, deletion PDR5 gene,                          null cells supports notion p23 fulfills protective role <br /> which encodes membrane-associated drug export pump, fur-                          Hsp90 inhibitors mammalian cells. <br /> ther accentuates hypersensitivity cells lack Sba1p. In                     Sba1p interacts Hsp90 isoforms. S. cerevisiae ex- <br /> yeast, radicicol consistently slightly potent                      presses Hsp90 isoforms, Hsp82p Hsc82p, needs <br /> than GA (48; data shown). In subsequent vivo ex-                         considered exploring phenotype &#226;&#338;&#172;sba1 cells. The <br /> periments yeast, used &#226;&#338;&#172;pdr5 strain                          amino acid sequences isoforms 95% identical <br /> increased radicicol sensitivity &#226;&#338;&#172;sba1 phenotype.                             (674 709 amino acids). Although half differences <br />    The mammalian Sba1p ortholog p23 protects mammalian                              map evolutionarily poorly conserved dispensable <br /> cells GA. In view extensive conservation                         (30) charged domain N-terminal middle do- <br /> Hsp90 chaperone machine, wished assess generality                         mains, direct evidence literature Sba1p <br /> of protective function p23/Sba1p newly identified                         interacts Hsc82p, exception reference to <br /> yeast. Having previously generated p23-null mouse line (22),                      unpublished results (15). Therefore, decided verify ex- <br /> we access wild-type p23-null MEFs. Cells                       perimentally Sba1p interacts isoforms. <br /> seeded low density (about 20%) allow vigorous                           Sba1p N-terminal Flag tag (Fig. 3A) expressed in <br /> growth period days. In absence GA,                         wild-type strain strains lacking other <br /> p23-null cells grew slightly slowly wild-type cells                    Hsp90 isoform. A co-IP experiment revealed Sba1p inter- <br /> previously reported (31). However, presence                     acts similarly Hsc82p Hsp82p (Fig. 3B). <br />  VOL. 28, 2008                                                                       p23 PROTECTS AGAINST Hsp90 INHIBITORS                     3449 <br />  <br />  <br />  <br />  <br />    FIG. 3. Hsp90 interaction vivo complementation full-<span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span> truncated forms Sba1p. (A) Schematic representation Sba1p <br /> truncation mutants. The N-terminal Flag tag removed residues indicated. The p23/Sba1 signature motif underlined. (B) Sba1p interacts <br /> with Hsp82p Hsc82p. An immunoblot Hsp90 co-IP experiment shown. Flag-tagged Sba1p expressed isogenic strains <br /> expressing Hsp90 isoforms used co-IP Hsp90 anti-Flag resin. The immunoblot probed polyclonal Hsp90 <br /> antibodies recognizing isoforms (see lanes labeled &#226;&#8364;&#339;input&#226;&#8364;?; uppermost bands correspond full-length Hsp82p Hsc82p, the <br /> lower bands degradation products). (C) Expression Hsp90 interaction Sba1p truncation mutants. Flag-tagged Sba1p derivatives shown <br /> in panel A expressed &#226;&#338;&#172;sba1 strain. Total extracts (40 &#226;?&#174;g each) anti-Flag immunoprecipitates immunoblotted anti-Flag <br /> and anti-Hsp90 antibodies indicated. In case full-length Flag-Sba1b &#226;&#338;&#172;131&#226;&#8364;&#8220;216 mutant, times immunoprecipitate was <br /> loaded facilitate comparison poorly expressed mutants. (D) The 130 N-terminal amino acids Sba1p sufficient for <br /> conferring radicicol (Rd) resistance. A growth assay fivefold serial dilutions &#226;&#338;&#172;sba1 &#226;&#338;&#172;pdr5 cells expressing Sba1p proteins indicated on <br /> the left performed increasing concentrations radicicol GA 30 37&#194;&#176;C. <br />  <br />  <br />  <br />    The N-terminal domain Sba1p sufficient Hsp90                   truncation mutants allow direct comparison the <br /> binding complementation. To investigate mode                     full-length Sba1 construct co-IP experiment, results <br /> action Sba1p vivo, generated panel Sba1p trun-                suggest N-terminal truncation internal deletion <br /> cation mutants analyzed Hsp90 binding                      mutants defective Hsp90 binding. The growth <br /> complementation radicicol hypersensitivity. Both wild-type               assay displayed Fig. 3D demonstrates N-terminal <br /> and mutant proteins expressed N-terminal Flag                  130 amino acids Sba1 (sba1&#226;&#338;&#172;131&#226;&#8364;&#8220;216) sufficient for <br /> tag control galactose-inducible GAL1 pro-                  conferring radicicol resistance &#226;&#338;&#172;sba1 strain. This evident <br /> moter. Neither Flag tag overexpression effect               normal temperature (30&#194;&#176;C) mild heat stress <br /> on complementation. Note toxic effects               (37&#194;&#176;C), sensitizes cells Hsp90 inhibitors. Indeed, <br /> of overexpression Sba1p growth strains used            N-terminal domain consistently complements better <br /> study (data shown), despite previous report                 wild type presence Hsp90 inhibitors <br /> working strains different genetic background                   radicicol GA. <br /> slightly elevated temperatures (40). Of mutants shown                 Differential effects Sba1p point mutants Hsp90 bind- <br /> in Fig. 3A, &#226;&#338;&#172;131&#226;&#8364;&#8220;216 C-terminal truncation mutant               ing complementation. The aforementioned truncation ex- <br /> stably expressed. This mutant lacks poorly conserved                periments suggested N-terminal 15 residues the <br /> apparently unstructured, highly acidic tail, carry             signature motif WPRLTKEK important folding <br /> contribute chaperone activity (60, 61). As judged             and/or activity Sba1p. We decided mutate systematically <br /> co-IP experiment Fig. 3C, portion Sba1p comprising                Sba1p residues conserved patches N-ter- <br /> the N-terminal 130 amino acids sufficient bind Hsp90.                 minal domain additional conserved ones just C <br /> This observation agrees previous reports,                 terminal signature motif assess ability to <br /> C-terminal tail human Sba1p ortholog p23                bind Hsp90 complement &#226;&#338;&#172;sba1 strain (Fig. 4). The <br /> be required chaperone activity Hsp90 binding (36,               proline residues regions altered to <br /> 60, 61). Removal N-terminal 15 amino acids Sba1p               avoid major structural perturbations. In contrast the <br /> of internal p23 signature motif WPRLTKEK severely de-                   truncation mutants, point mutants expressed similar <br /> stabilizes protein. Even low levels                 levels compared wild-type Sba1p, exception of <br />  3450      FORAFONOV ET AL.                                                                                                 MOL. CELL. BIOL. <br />  <br />  <br />                                                                               When mutants tested ability to <br />                                                                            complement &#226;&#338;&#172;sba1 strain, classes apparent <br />                                                                            (Fig. 4 Table S1 supplemental material). Those <br />                                                                            unable bind Hsp90 high affinity (W12A and <br />                                                                            W104A) failed confer radicicol resistance. L107S and <br />                                                                            Y116A bind (albeit L107S binds somewhat well) yet <br />                                                                            complement. A13S K113A, display some <br />                                                                            reduction Hsp90 binding, R106A, binds <br />                                                                            Hsp90 like wild type, appear complement slightly <br />                                                                            better wild type. No complementing mutant was <br />                                                                            bind Hsp90 all. We tentatively con- <br />                                                                            cluded point Hsp90 binding necessary but <br />                                                                            sufficient protection radicicol. <br />                                                                               Correlating vitro vivo activities Sba1p mutants. <br />                                                                            In attempt correlate biochemical vivo comple- <br />                                                                            mentation activities Sba1p point mutants, performed <br />                                                                            vitro assays recombinant proteins. We selected set of <br />                                                                            mutants represent distinct classes mu- <br />                                                                            tants (see Table S1 supplemental material). Wild-type <br />                                                                            mutant Sba1p proteins produced bacteria, purified <br />                                                                            homogeneity, analyzed known p23 family func- <br />                                                                            tions&#226;&#8364;&#8221;in vitro chaperone activity inhibition Hsp82p <br />                                                                            ATPase. The molecular chaperone capacities various <br />                                                                            Sba1 proteins determined utilizing established vitro <br />                                                                            assay. It assesses ability test protein maintain <br />                                                                            chemically denatured firefly luciferase folding-competent <br />                                                                            state refolding molecular chaperone couple Hsp70- <br />                                                                            Hdj1 (18). Thirty-eight percent total luciferase activity <br />                                                                            recovered presence wild-type Sba1p (Fig. <br />                                                                            5A). Note C-terminal truncation mutants Sba1p are <br />                                                                            completely defective function better BSA <br />                                                                            maintaining luciferase refolding (36, 60, 61; data not <br />    FIG. 4. Hsp90 interaction vivo complementation Sba1p <br /> point mutants. Both wild-type Sba1p (WT) point mutants            shown) assay complete absence of <br /> expressed Flag-tagged versions &#226;&#338;&#172;sba1 &#226;&#338;&#172;pdr5 strain. (A) Ex-       Hsp90. In contrast, point mutants retain some, albeit re- <br /> pression Hsp90 interaction Sba1p point mutants. The immuno-         duced, chaperone activity fall general activity <br /> blots represent different co-IP experiments. ATP present           classes: W12A, A13S, W104A, Y116A &#226;&#172;&#402;60% of <br /> during incubation anti-Flag resin experiments, <br /> but co-IP experiment 1 (IP1), ATP omitted buffer used      wild-type activity, R106A K113A retain <br /> to wash immunoprecipitates. Note doublet observed        &#226;&#172;&#402;32%. It interesting alterations reduce <br /> Flag-Sba1 experiments, presumably C-terminal            chaperone activity Sba1p despite fact map to <br /> cleavage. The weaker band Flag-Sba1 immuno-            predicted surface residues located outside acidic tail, <br /> globulin light chain. For details, Materials Methods and <br />                                                                            required chaperone activity. However, reduced <br /> the legend Fig. 3C. (B) Radicicol (Rd) sensitivity &#226;&#338;&#172;sba1 &#226;&#338;&#172;pdr5 <br /> strains expressing Sba1p point mutants. Growth assays         chaperone activities point mutants correlate with <br /> described legend Fig. 3D.                                        vivo activities. <br />                                                                               The experiments Fig. 4A indicated Sba1p <br />                                                                            point mutants reduced ability co-IP Hsp90 from <br /> W104A, consistently slightly abundant (Fig. 4A)              crude extracts, albeit different degrees. The relative inter- <br /> (data shown). Co-IP experiments performed ex-                  action affinities W12A, A13S, W104A, R106A, K113A <br /> plore interaction Hsp90 whole-cell extracts. In            explored purified components monitoring <br /> yeast system, co-IP Hsp90 Sba1p observed             Sba1p-dependent inhibition ATPase activity of <br /> in presence ATP (15, 25; data shown). During                Hsp82p. As expected, wild-type Sba1p reduces <br /> incubation anti-Flag resin, ATP present              ATP hydrolysis rate Hsp82p &#226;&#172;&#402;50% (Fig. 5B), which <br /> the buffer favor assembly maintenance Sba1p-Hsp90                consistent stabilization ATP-bound form (9, 33, <br /> complexes. Immunoprecipitates washed                      52, 56). Using observed ATPase rates Sba1p titration, <br /> buffer ATP (Fig. 4A, upper panels labeled IP1)              calculated Sba1p-Hsp82p dissociation constant &#226;&#172;&#402;2.2 <br /> or ATP (lower panels labeled IP2). Two Sba1p mutants                  &#226;?&#174;M, good agreement prior report &#226;&#172;&#402;1.8 <br /> (W12A W104A) severely defective Hsp90 binding.                  &#226;?&#174;M (56). Three point mutants, A13S, R106A, and <br /> In absence ATP washing, mutants                     K113A, functionally interact Hsp82p, albeit lower <br /> (A13S K113A) lose Hsp90 quantitatively,                 affinities &#226;&#172;&#402;33.1, 21.5, 4.2 &#226;?&#174;M, respectively (Fig. 5B). In <br /> four mutants (Q14A, R15A, R106A, L107S)                      contrast, W12A W104A apparently unable inhibit <br /> intermediate retaining Hsp90.                                           ATPase activity Hsp82p (Fig. 5C). In fact, increasing <br />  VOL. 28, 2008           p23 PROTECTS AGAINST Hsp90 INHIBITORS                        3451 <br />  <br />  <br />                 concentrations mutants stimulated the <br />                 ATPase rate, suggesting alternative mode interaction <br />                 effect Hsp82p. Thus, ability bind to <br />                 regulate Hsp90 ATPase activity necessary comple- <br />                 mentation &#226;&#338;&#172;sba1 phenotype. The A13S, R106A, and <br />                 K113A mutants inhibit Hsp90 ATPase test tube <br />                 protect high radicicol concentrations living cells. <br />                 The mutant Y116A, affinity slightly <br />                 higher (1.1 &#226;?&#174;M) wild-type Sba1p, appears a <br />                 special case discussed below. Importantly, <br />                 Y116A results point binding inhibition the <br />                 ATPase sufficient vivo complementation. <br />                    The point mutants W12A W104A structurally loos- <br />                 ened. Unlike mutants, W12A W104A proved to <br />                 defective binding regulating Hsp90 confer- <br />                 ring radicicol resistance cells, despite fact W12 and <br />                 W104 appear recently published <br />                 Sba1p-Hsp82p interface (2). In order under- <br />                 standing differences classes mutants, <br />                 global structures investigated limited proteolysis. <br />                 With light trypsin treatment, observed slight digestion of <br />                 wild-type Sba1p point mutants A13S, R106A, and <br />                 K113A appearance faster-migrating species (&#226;&#172;&#402;25 <br />                 kDa) (Fig. 5D). In contrast, W12A W104A almost <br />                 completely digested small digestion products (&#226;&#172;?5 <br />                 kDa) apparent. Hence, W12A W104A mutants <br />                 adopt distinct loosened conformations relative other <br />                 Sba1p mutants. Because loosened conformation, <br />                 Hsp90 recognize substrates, albeit a <br />                 poor binding affinity, explain stimulatory effect <br />                 ATPase. It noteworthy W12A W104A are <br />                 clearly grossly misfolded accumulate appre- <br />                 ciable levels Escherichia coli yeast retain <br />                 chaperone activity. Their structural biochemical defects, <br />                 however, appear compromise ability complement <br />                 &#226;&#338;&#172;sba1 cells. <br />                    Sba1p reduces GA binding Hsp90 vitro p23 pro- <br />                 tects substrate vivo. It appeared point study <br />                 Sba1p protect Hsp90 inhibitors. Complex for- <br />                 mation intrinsic chaperone activity Sba1p ap- <br />                 peared necessary, albeit sufficient, function. <br />                 This called direct test Sba1p prevents an <br />  <br />  <br />  <br />                    FIG. 5. In vitro characterization biochemical activities Sba1p <br />                 point mutants. (A) Sba1p point mutants retain molecular chaperone <br />                 activity. The bar graph shows recovered luciferase activities, are <br />                 indicative ability wild-type Sba1p (WT) point mutants <br />                 maintain denatured firefly luciferase refoldable state refold- <br />                 ing Hsp70 Hdj1. BSA used negative control. The data <br />                 represent average values independent experiments. (B <br />                 C) The Sba1p point mutants display differential effects ATP <br />                 hydrolysis Hsp82p. ATPase activity expressed function of <br />                 Sba1p concentrations. The data Sba1p mutants pre- <br />                 sented separate graphs facilitate reading represent the <br />                 average values independent experiments. (D) The Sba1p <br />                 mutants inhibit ATPase Hsp82p distinct con- <br />                 formational states. The structures Sba1p mutants probed using <br />                 limited proteolytic digestion trypsin. After digestion 0 (lanes <br />                 marked &#226;&#8364;&#339;&#226;&#171;&#186;&#226;&#8364;?) 15 min (lanes marked &#226;&#8364;&#339;&#226;&#171;&#185;&#226;&#8364;?), samples resolved <br />                 sodium dodecyl sulfate-polyacrylamide gel electrophoresis vi- <br />                 sualized Coomassie blue staining. <br />  3452     FORAFONOV ET AL.                                                                                                    MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />                                                                            FIG. 7. The Y116A-Hsp90 interaction insensitive radicicol <br />                                                                          (Rd). Shown immunoblots co-IP experiment antibodies to <br />                                                                          Flag-tagged wild-type (WT) mutant (Y116A) Sba1p expressed a <br />                                                                          &#226;&#338;&#172;sba1 yeast strain. A fraction crude yeast extract loaded as <br />                                                                          &#226;&#8364;&#339;input,&#226;&#8364;? anti-Flag immunoprecipitate analyzed <br />                                                                          directly incubation indicated. <br />  <br />  <br />  <br />                                                                          Figure 8 shows endogenous Hsp90 substrate Raf-1 <br />                                                                          (54) degraded rapidly given GA concentration in <br />                                                                          MEF cells devoid p23. <br />    FIG. 6. Sba1p reduces binding affinity Hsp90 GA. The graph <br /> displays change fluorescence anisotropy fluorescein-la-                               DISCUSSION <br /> beled GA (FITC-GA) undergoes binds Hsp82p. The data <br /> points (averages replicates) fitted curves indicate         We revisited phenotype S. cerevisiae cells lacking <br /> changes absence presence fixed wild-type <br />                                                                          SBA1 gene discovered display increased <br /> (WT) representative mutant Sba1p proteins. The Kds calculated <br /> from fitted curves shown legend. Curves addi-     sensitivity Hsp90 inhibitors. Using phenotype, have <br /> tional mutants omitted clarity. Their Kds A13S, R106A,      determined known Sba1p activities important <br /> and L107S 60.8 &#226;&#171;&#190; 6.3, 51.1 &#226;&#171;&#190; 6.7, 23.6 &#226;&#171;&#190; 4.0 nM, respectively.   complementation. Hsp90 binding inhibition the <br />                                                                          ATPase activity required, intrinsic Sba1p <br />                                                                          chaperone function dispensable conferring relative resis- <br /> inhibitor binding Hsp90. The effect various Sba1p                tance Hsp90 inhibitors. The results particular <br /> derivatives binding affinity Hsp82p fluorescein-         Sba1p point mutant, Y116A, indicate abilities bind <br /> labeled version GA (FITC-GA) determined                 inhibit ATPase sufficient protection and <br /> using fluorescence anisotropy (Fig. 6). Strikingly, wild-type            inhibition ATPase protection Hsp90 inhib- <br /> Sba1p complementing mutants A13S, R106A,                     itors distinct activities p23/Sba1p. Thus, the <br /> K113A impair GA binding twofold more,                  exception intrinsic chaperone activity Sba1p, the <br /> complementation-defective mutants L107S Y116A barely                 assessed biochemical activities necessary protect cells. <br /> have effect all.                                                  While protective mutants qualitatively retain activi- <br />    Intriguingly, despite fact Y116A able bind             ties, quantitative differences inter- <br /> Hsp90 high affinity, Y116A fails protect Hsp90           esting follow up. Alterations components the <br /> GA vitro radicicol vivo, interaction Hsp90        Hsp90 complex, notably Hsp90 itself, Sti1, Cpr7m, Cdc37, <br /> is insensitive radicicol. Y116A readily coimmunopre-           previously reported render cells sensitive to <br /> cipitated Hsp90 crude yeast extracts supplemented <br /> with ATP, then, unlike wild-type Sba1p, retains Hsp90 <br /> even addition radicicol (Fig. 7) GA various GA <br /> derivatives (data shown). All biochemical experi- <br /> ments suggest unusual mutant, able to <br /> inhibit Hsp90 ATPase, prevent inhibitors from <br /> binding possibly binds presence inhibitors. <br /> Whether mutant, L107S, retains ability <br /> to bind Hsp90 does complement, shares additional <br /> biochemical characteristics Y116A remains further <br /> analyzed. <br />    A prediction experiments Hsp90 <br /> substrates vulnerable GA-induced degrada- <br /> tion absence p23/Sba1p. While experiments               FIG. 8. p23 protects Hsp90 substrate Raf-1 GA-in- <br />                                                                          duced degradation. Shown representative immunoblot extracts <br /> difficult perform large-scale effects GA            wild-type (WT) p23-null MEFs treated 60 nM GA for <br /> treatment cell growth <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-6' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span>survival</span>, decided try              indicated times. Raf-1 levels standardized GAPDH, and <br /> illustrate point examining model substrate.         starting level set 100% (see numbers micrograph). <br />  VOL. 28, 2008                                                                  p23 PROTECTS AGAINST Hsp90 INHIBITORS                 3453 <br />  <br />  <br /> Hsp90 inhibitors (3, 12, 48, 49, 57). In contrast cases,      findings yeast mammalian cells, influence <br /> we hypersensitivity cells lacking Sba1p/p23        p23 levels GA sensitivity normal cancer <br /> not merely synthetic effect attributed        cells interesting reexamine. Intriguingly, cancer <br /> to loss direct protective effect cochaperone binding.      cells, particular metastatic cells, reported have <br />    p23/Sba1p protects cells Hsp90 inhibitors. S. cerevi-       higher levels p23 (29, 37), overexpression p23 in <br /> siae &#226;&#338;&#172;sba1 strains established appear fully           MCF7 breast cancer cells stimulates ability invade a <br /> viable. Although loss SBA1 compromises mainte-              fibronectin membrane (41). It tempting speculate that <br /> nance telomeres (59) lead genomic instability            cancer cells up-regulate p23 cope increased <br /> (39) defective response amino acid starvation (14),           demand handling Hsp90 substrates protect of <br /> possibly strain-specific fashion, cells eventually        Achilles heels, Hsp90, inhibitors pro- <br /> adapt life Sba1p grow wild-type cells        duced metabolism taken food. It conceiv- <br /> (4, 15) (Fig. 1). In fission yeast Schizosaccharomyces pombe,      able p23 functions including protective be <br /> the SBA1 ortholog WOS2 required thermotolerance (38),           posttranslationally regulated influenced nature the <br /> but indication case S. cerevisiae. A   cochaperones substrate Hsp90 <br /> previous survey factors influence sensitivity     complex. The phenotype Y116A mutant indication <br /> Hsp90 inhibitors missed Sba1p (48), likely            subtle changes kind able affect the <br /> their &#226;&#338;&#172;sba1 strain deleted PDR5. Similarly,         ability wild-type p23 protect Hsp90. In context of <br /> large-scale chemical genetic screens Hsp90 inhibitors,        cancer therapy, predict tumor cells be <br /> missed &#226;&#338;&#172;sba1 (64), did score &#226;&#338;&#172;sba1 hit,         rendered sensitive GA anticancer drug by <br /> but did highlight discuss finding (43). Al-        p23-targeted drugs small interfering RNA. <br /> though observed hypersensitivity different            The increased growth inhibition and/or induction cell <br /> strain backgrounds (data shown), exclude            death response Hsp90 inhibitors or <br /> there strain-specific penetrance trait. The deletion      Hsp90 substrates fail. When Bohen originally re- <br /> of gene encoding export pump Pdr5p clearly sensitized          ported deletion SBA1 gene renders mam- <br /> our strains Hsp90 inhibitors facilitated display         malian glucocorticoid receptor assayed yeast strain <br /> differential sensitivities (Fig. 1). Remarkably,         hypersensitive Hsp90 inhibitor, absence <br /> p23 plays similar protective role MEFs, suggesting      Sba1p impact accumulation exogenous <br /> may conserved function p23 regulatory               substrate (4). We later confirmed observation p23- <br /> functions Hsp90 cochaperone.                                     null MEFs (22). Despite functionally defective, levels <br />    Possible mechanism function protection. In contrast          glucocorticoid receptor similar in <br /> to molecular chaperone function Sba1p, ability           p23-null wild-type cells. Thus, suspect loss of <br /> Sba1p bind Hsp90 inhibit ATPase              functionality Hsp90 substrates absence p23/ <br /> crucial, albeit sufficient, protecting cells Hsp90     Sba1p GA-induced degradation mechanisti- <br /> inhibitors. The biochemical correlate phenomenon            cally distinct events. Nevertheless, analysis Raf-1 sug- <br /> likely equilibrium Sba1p-Hsp90 complex           gests Hsp90 substrates indirectly protected from <br /> that favored ATP binding Hsp90 inhibitor-             Hsp90 inhibitor-induced degradation protective <br /> bound Hsp90 unable bind Sba1p (24, 26, 50, 60). This        effect p23/Sba1p Hsp90. <br /> suggests binding Sba1p Hsp90-ATP complex               Sba1p structure function. Our findings reveal Sba1p <br /> might protect Hsp90 binding inhibitor molecule,          biochemical functions protein surfaces required <br /> is compatible tight intimate nature ternary        vivo complementation. This time an <br /> complex (2) ability human p23 stabilize              extensive mutagenesis analysis performed. The rea- <br /> nucleotide-bound state Hsp90 (58). Most importantly,             son focusing mutagenesis separate regions <br /> have demonstrated Sba1p directly reduces affinity          shown Fig. 9C residues lie the <br /> Hsp90 GA. Thus, Sba1p reduces affinity Hsp90         patch surface free p23 (60) (Fig. 9A). More- <br /> inhibitors pushes equilibrium ATP-bound             over, previous publications pinpointed resi- <br /> form, explain Sba1p exert effect despite         dues right patch important Hsp90 <br /> being 10-fold abundant Hsp90. In case           binding: I117 Sba1p (40) equivalent F121 and <br /> yeast, interesting speculate protective function    W124 human p23 (63). <br /> of Sba1p importance natural environment                  Although published crystal structures free form <br /> where Hsp90 inhibitors radicicol GA produced           human p23 (60) Hsp90-bound Sba1p (2) largely <br /> by microorganisms. There evidence             identical overall fold, significant differences the <br /> type pharmacological interaction species. A rhizo-          details reinterpretation currently available <br /> sphere fungus shown produce large amounts               structural information (Fig. 9). We modeled Sba1p <br /> Hsp90 inhibitors influence Hsp90 stress re-           polypeptide sequence human p23 structure com- <br /> sponse higher plants associated fungus (34).              pare free bound forms. The comparison the <br />    In apparent contradiction findings yeast,     structures suggests Sba1p undergoes structural rear- <br /> been suggested cancer cells higher affinity GA         rangements binding Hsp90, presumably favor to <br /> because Hsp90 complexes extensively associ-             stabilize tight packing apparent complex. This <br /> ated cochaperones p23 (27). However, view            particularly pronounced N-terminal portion beginning <br /> has recently questioned (21), and, therefore, light        W12. In free form, residues 12 15 (in red Fig. 9) <br />  3454     FORAFONOV ET AL.                                                                                                      MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />    FIG. 9. Model structural rearrangements Sba1p induced Hsp90 binding. (A) Model crystal structure N-terminal <br /> domain Sba1p (residues 12 121). To model Sba1p sequence known structure unbound human p23 (60), generated a <br /> multiple-sequence alignment submitted fully automated protein structure homology-modeling server SWISS-MODEL (55). Residues <br /> 1 11 omitted resolved published structure Sba1p-Hsp82p complex (2). (B) Experimental crystal structure <br /> of Sba1p complex Hsp82p (2). Rendering Sba1p structure PDB file 2CG9 PyMOL molecular graphics <br /> system (W. L. DeLano, 2002; http://www.pymol.org). (C) Sequence alignment mutagenized sequence motifs. Colors red, purple, <br /> and blue used residues conserved, green gray the similar ones, respectively. (D) Surface <br /> representations structures shown panels A B. Black portions model unbound Sba1p correspond loops are <br /> not resolved published structure complex. <br />  <br />  <br />  <br /> pack surface vicinity N-terminal         bind Hsp90, rationalized differently. It conceivable <br /> the signature motif WPRLTKE, remainder looping                    N-terminal portion mutant properly <br /> out. Upon binding Hsp90, residues &#226;&#8364;&#339;pulled through&#226;&#8364;?               tethered core protein residue 12, nor- <br /> the interior structure A13 R15 come lie far           mally close proximity hydrophobic L107, therefore <br /> away W104 form large interface,               unable undergo large structural rearrangement upon <br /> with K113 Y116, packs lid Hsp90. Part              binding Hsp90. Despite somewhat loosened structure, both <br /> of loop comes lie inside structure contrib-            W12A W104A retain chaperone activity. Although this <br /> utes small &#226;?&#164;-sheet.                                                      activity depends C-terminal domain Sba1, not <br />    The structures interesting consider interpreting          known domain sufficient chaperone activity <br /> the defects point mutants W12A W104A.                       extent N-terminal domain contributes. <br /> Whereas aromatic chains tryptophan resi-                  Surprisingly, Sba1p mutations map known <br /> dues close proximity free form, far                Sba1p-Hsp90 interface effect minor <br /> apart Sba1p binds Hsp90. In Sba1p-Hsp90 complex,                  effect Hsp90 binding. A13, Q14, R15, K113, Y116 are <br /> W12 constitutes N terminus defined structure,              localized large interface packs lid of <br /> protrudes away core, involved pro-              Hsp90 consists Sba1p residues 13 16 113 118 <br /> tein-protein contacts. The fact bulky chain               (2), A13S K113A slightly reduced Hsp90 <br /> W104 points inwards account increased protease               binding vitro IP experiments performed under <br /> sensitivity observed mutant W104A. The behavior                certain conditions. Single point mutants unable to <br /> the W12A mutant, similarly loosened unable                 weaken interaction sufficiently large size of <br />  VOL. 28, 2008                                                                                    p23 PROTECTS AGAINST Hsp90 INHIBITORS                              3455 <br />  <br />  <br /> this particular interface. The specific characteristics                              inducible kinase &#226;?&#163; subunit eukaryotic translation initiation factor <br />                                                                                             kinase Gcn2. Mol. Cell. Biol. 19:8422&#226;&#8364;&#8220;8432. <br /> amino acid substitution mutant I117N explain                         15.   Fang, Y., A. E. Fliss, J. Rao, A. J. Caplan. 1998. SBA1 encodes yeast <br /> severely defective Hsp90 binding (40). Y116                              Hsp90 cochaperone homologous vertebrate p23 proteins. Mol. <br /> interface, aromatic chain faces away Hsp90.                               Cell. Biol. 18:3727&#226;&#8364;&#8220;3734. <br />                                                                                       16.   Felts, S. J., D. O. Toft. 2003. p23, simple protein complex <br /> While explain Y116A mutant binds                                     activities. Cell Stress Chaperones 8:108&#226;&#8364;&#8220;113. <br /> Hsp90, unusual binding characteristics presence                         17.   Freeman, B. C., S. J. Felts, D. O. Toft, K. R. Yamamoto. 2000. The p23 <br /> Hsp90 inhibitors understood detailed                                molecular chaperones act late step intracellular receptor action to <br />                                                                                             differentially affect ligand efficacies. Genes Dev. 14:422&#226;&#8364;&#8220;434. <br /> structural information.                                                               18.   Freeman, B. C., M. P. Myers, R. Schumacher, R. I. Morimoto. 1995. <br />    Interestingly, residues interface abso-                           Identification regulatory motif Hsp70 affects ATPase activity, <br />                                                                                             substrate binding interaction HDJ-1. EMBO J. 14:2281&#226;&#8364;&#8220;2292. <br /> lutely conserved (Fig. 9C) (data shown). While yeast                          19.   Freeman, B. C., D. O. Toft, R. I. Morimoto. 1996. Molecular chaperone <br /> Sba1p mutant A13S partially defective Hsp90 binding,                               machines: chaperone activities cylophilin Cyp-40 steroid <br /> serine natural residue Drosophila melanogaster. The                               aporeceptor-associated protein p23. Science 274:1718&#226;&#8364;&#8220;1720. <br />                                                                                       20.   Freeman, B. C., K. R. Yamamoto. 2002. Disassembly transcriptional <br /> human p23 mutant K95A, equivalent Sba1p                                     regulatory complexes molecular chaperones. Science 296:2232&#226;&#8364;&#8220;2235. <br /> mutant K113A, completely defective Hsp90 binding (47).                         21.   Gooljarsingh, L. T., C. Fernandes, K. Yan, H. Zhang, M. Grooms, K. <br /> Moreover, Sba1p bind yeast mammalian Hsp90,                                    Johanson, R. H. Sinnamon, R. B. Kirkpatrick, J. Kerrigan, T. Lewis, M. <br />                                                                                             Arnone, A. J. King, Z. Lai, R. A. Copeland, P. J. Tummino. 2006. A <br /> but yeast Hsp90 bind mammalian p23 (53). This sug-                                   biochemical rationale anticancer effects Hsp90 inhibitors: slow, <br /> gests large interface coordinate evolution                            tight binding inhibition geldanamycin analogues. Proc. Natl. Acad. <br /> may contribute ensuring sufficient affinity protein-                             Sci. USA 103:7625&#226;&#8364;&#8220;7630. <br />                                                                                       22.   Grad, I., T. A. McKee, S. M. Ludwig, G. W. Hoyle, P. Ruiz, W. Wurst, T. <br /> protein interaction functionality molecular chap-                              Floss, C. A. Miller III, D. Picard. 2006. The Hsp90 cochaperone p23 is <br /> erone complex.                                                                              essential perinatal survival. Mol. Cell. Biol. 26:8976&#226;&#8364;&#8220;8983. <br />                                                                                       23.   Grad, I., D. Picard. 2007. The glucocorticoid responses shaped by <br />                                                                                             molecular chaperones. Mol. Cell. Endocrinol. 275:2&#226;&#8364;&#8220;12. <br />                           ACKNOWLEDGMENTS                                             24.   Johnson, J., R. Corbisier, B. Stensgard, D. O. Toft. 1996. The involve- <br />                                                                                             ment p23, hsp90, immunophilins assembly progesterone <br />    We grateful Natasha Kralli Keith Yamamoto                            receptor complexes. J. Steroid Biochem. Mol. Biol. 56:31&#226;&#8364;&#8220;37. <br /> yeast strains. We thank Marc Fischer Olivier Donze&#194;&#180;                25.   Johnson, J. L., A. Halas, G. Flom. 2007. Nucleotide-dependent interac- <br /> efforts early project Martin Hessling preliminary                        tion Saccharomyces cerevisiae Hsp90 cochaperone proteins Sti1, <br /> biochemical experiments mutant. We thank anonymous                              Cpr6, Sba1. Mol. Cell. Biol. 27:768&#226;&#8364;&#8220;776. <br /> reviewer thorough editing manuscript.                                      26.   Johnson, J. L., D. O. Toft. 1995. Binding p23 hsp90 during <br />    The Freeman lab University Illinois supported                         assembly progesterone receptor. Mol. Endocrinol. 9:670&#226;&#8364;&#8220;678. <br /> grant National Institutes Health (DK074270). Support                  27.   Kamal, A., L. Thao, J. Sensintaffar, L. Zhang, M. F. Boehm, L. C. Fritz, and <br />                                                                                             F. J. Burrows. 2003. A high-affinity conformation Hsp90 confers tumour <br /> the Picard lab University came Canton Gene`ve, the <br />                                                                                             selectivity Hsp90 inhibitors. Nature 425:407&#226;&#8364;&#8220;410. <br /> Swiss National Science Foundation, Fondation Medic.                           28.   Knoblauch, R., M. J. Garabedian. 1999. Role Hsp90-associated <br />                                                                                             cochaperone p23 estrogen receptor signal transduction. Mol. Cell. Biol. <br />                                  REFERENCES                                                 19:3748&#226;&#8364;&#8220;3759. <br />  1. Abbas-Terki, T., O. Donze&#194;&#180;, P.-A. Briand, D. Picard. 2001. Hsp104             29.   Krebs, J., P. Saremaslani, R. Caduff. 2002. ALG-2: Ca2&#226;&#171;&#185;-binding <br />     interacts Hsp90 cochaperones respiring yeast. Mol. Cell. Biol. 21:              modulator protein involved cell proliferation cell death. Biochim. <br />     7569&#226;&#8364;&#8220;7575.                                                                              Biophys. Acta 1600:68&#226;&#8364;&#8220;73. <br />  2. Ali, M. M., S. M. Roe, C. K. Vaughan, P. Meyer, B. Panaretou, P. W. Piper,        30.   Louvion, J.-F., R. Warth, D. Picard. 1996. Two eukaryote-specific re- <br />     C. Prodromou, L. H. Pearl. 2006. Crystal structure Hsp90-nucle-               gions Hsp82 dispensable viability signal transduction func- <br />     otide-p23/Sba1 closed chaperone complex. Nature 440:1013&#226;&#8364;&#8220;1017.                          tions yeast. Proc. Natl. Acad. Sci. USA 93:13937&#226;&#8364;&#8220;13942. <br />  3. Bandhakavi, S., R. O. McCann, D. E. Hanna, C. V. Glover. 2003. A              31.   Lovgren, A. K., M. Kovarova, B. H. Koller. 2007. cPGES/p23 required <br />     positive feedback loop protein kinase CKII Cdc37 promotes               glucocorticoid receptor function embryonic growth prosta- <br />     activity multiple protein kinases. J. Biol. Chem. 278:2829&#226;&#8364;&#8220;2836.                     glandin E2 synthesis. Mol. Cell. Biol. 27:4416&#226;&#8364;&#8220;4430. <br />  4. Bohen, S. P. 1998. Genetic biochemical analysis p23 ansamycin          32.   McLaughlin, S. H., H. W. Smith, S. E. Jackson. 2002. Stimulation the <br />     antibiotics function Hsp90-dependent signaling proteins. Mol. Cell.           weak ATPase activity human Hsp90 client protein. J. Mol. Biol. <br />     Biol. 18:3330&#226;&#8364;&#8220;3339.                                                                     315:787&#226;&#8364;&#8220;798. <br />  5. Bose, S., T. Weikl, H. Bu &#194;&#168;gl, J. Buchner. 1996. Chaperone function        33.   McLaughlin, S. H., F. Sobott, Z. P. Yao, W. Zhang, P. R. Nielsen, J. G. <br />     Hsp90-associated proteins. Science 274:1715&#226;&#8364;&#8220;1717.                                       Grossmann, E. D. Laue, C. V. Robinson, S. E. Jackson. 2006. The <br />  6. Brachmann, C. B., A. Davies, G. J. Cost, E. Caputo, J. Li, P. Hieter, J. D.         co-chaperone p23 arrests Hsp90 ATPase cycle trap client proteins. J. <br />     Boeke. 1998. Designer deletion strains derived Saccharomyces cerevi-               Mol. Biol. 356:746&#226;&#8364;&#8220;758. <br />     siae S288C: useful set strains plasmids PCR-mediated gene            34.   McLellan, C. A., T. J. Turbyville, E. M. Wijeratne, A. Kerschen, E. Vierling, <br />     disruption applications. Yeast 14:115&#226;&#8364;&#8220;132.                                    C. Queitsch, L. Whitesell, A. A. Gunatilaka. 2007. A rhizosphere fungus <br />  7. Cox, M. B., C. A. Miller III. 2002. The p23 co-chaperone facilitates                enhances Arabidopsis thermotolerance production HSP90 <br />     dioxin receptor signaling yeast model system. Toxicol. Lett. 129:13&#226;&#8364;&#8220;21.            inhibitor. Plant Physiol. 145:174&#226;&#8364;&#8220;182. <br />  8. Cox, M. B., C. A. Miller III. 2003. Pharmacological genetic analysis      35.   Mitsiades, C. S., N. S. Mitsiades, C. J. McMullan, V. Poulaki, A. L. Kung, <br />     90-kDa heat shock isoprotein-aryl hydrocarbon receptor complexes. Mol.               F. E. Davies, G. Morgan, M. Akiyama, R. Shringarpure, N. C. Munshi, P. G. <br />     Pharmacol. 64:1549&#226;&#8364;&#8220;1556.                                                                Richardson, T. Hideshima, D. Chauhan, X. Gu, C. Bailey, M. Joseph, T. A. <br />  9. Cox, M. B., C. A. Miller III. 2004. Cooperation heat shock protein 90            Libermann, N. S. Rosen, K. C. Anderson. 2006. Antimyeloma activity of <br />     p23 aryl hydrocarbon receptor signaling. Cell Stress Chaperones                  heat shock protein-90 inhibition. Blood 107:1092&#226;&#8364;&#8220;1100. <br />     9:4&#226;&#8364;&#8220;20.                                                                           36.   Mollerup, J., M. W. Berchtold. 2005. The co-chaperone p23 degraded <br /> 10. Cullinan, S. B., L. Whitesell. 2006. Heat shock protein 90: unique                caspases proteasome apoptosis. FEBS Lett. 579:4187&#226;&#8364;&#8220; <br />     chemotherapeutic target. Semin. Oncol. 33:457&#226;&#8364;&#8220;465.                                      4192. <br /> 11. Dittmar, K. D., D. R. Demady, L. F. Stancato, P. Krishna, W. B. Pratt.        37.   Mollerup, J., T. N. Krogh, P. F. Nielsen, M. W. Berchtold. 2003. Prop- <br />     1997. Folding glucocorticoid receptor heat shock protein (hsp)            erties co-chaperone protein p23 erroneously attributed ALG-2 <br />     90-based chaperone machinery. J. Biol. Chem. 272:21213&#226;&#8364;&#8220;21220.                           (apoptosis-linked gene 2). FEBS Lett. 555:478&#226;&#8364;&#8220;482. <br /> 12. Dolinski, K. J., M. E. Cardenas, J. Heitman. 1998. CNS1 encodes            38.   Mun &#203;&#339; oz, M. J., E. R. Bejarano, R. R. Daga, J. Jimenez. 1999. The <br />     essential p60/Sti1 homolog Saccharomyces cerevisiae suppresses cy-              identification Wos2, p23 homologue interacts Wee1 Cdc2 <br />     clophilin 40 mutations interacts Hsp90. Mol. Cell. Biol. 18:7344&#226;&#8364;&#8220;              mitotic control fission yeasts. Genetics 153:1561&#226;&#8364;&#8220;1572. <br />     7352.                                                                             39.   Ouspenski, I. I., S. J. Elledge, B. R. Brinkley. 1999. New yeast genes <br /> 13. Donze&#194;&#180;, O., T. Abbas-Terki, D. Picard. 2001. The Hsp90 chaperone                    important chromosome integrity segregation identified dosage <br />     complex facilitator repressor dsRNA-dependent kinase             effects genome stability. Nucleic Acids Res. 27:3001&#226;&#8364;&#8220;3008. <br />     PKR. EMBO J. 20:3771&#226;&#8364;&#8220;3780.                                                        40.   Oxelmark, E., R. Knoblauch, S. Arnal, L. F. Su, M. Schapira, M. J. <br /> 14. Donze&#194;&#180;, O., D. Picard. 1999. Hsp90 binds regulates Gcn2, ligand-            Garabedian. 2003. Genetic dissection p23, Hsp90 cochaperone, reveals <br />  3456         FORAFONOV ET AL.                                                                                                                       MOL. CELL. BIOL. <br />  <br />       distinct surface involved estrogen receptor signaling. J. Biol. Chem.       52. Richter, K., S. Walter, J. Buchner. 2004. The co-chaperone Sba1 con- <br />       278:36547&#226;&#8364;&#8220;36555.                                                                     nects ATPase reaction Hsp90 progression chaperone <br /> 41.   Oxelmark, E., J. M. Roth, P. C. Brooks, S. E. Braunstein, R. J. Schneider,           cycle. J. Mol. Biol. 342:1403&#226;&#8364;&#8220;1413. <br />       M. J. Garabedian. 2006. The cochaperone p23 differentially regulates         53. Scheibel, T., T. Weikl, R. Rimerman, D. Smith, S. Lindquist, J. Buchner. <br />       estrogen receptor target genes promotes tumor cell adhesion inva-            1999. Contribution N- C-terminal domains function Hsp90 <br />       sion. Mol. Cell. Biol. 26:5205&#226;&#8364;&#8220;5213.                                                 Saccharomyces cerevisiae. Mol. Microbiol. 34:701&#226;&#8364;&#8220;713. <br /> 42.   Panaretou, B., C. Prodromou, S. M. Roe, R. O&#226;&#8364;&#8482;Brien, J. E. Ladbury, P. W.         54. Schulte, T. W., M. V. Blagosklonny, C. Ingui, L. Neckers. 1995. Disrup- <br />       Piper, L. H. Pearl. 1998. ATP binding hydrolysis essential        tion Raf-1-Hsp90 molecular complex results destabilization of <br />       function Hsp90 molecular chaperone vivo. EMBO J. 17:4829&#226;&#8364;&#8220;4836.             Raf-1 loss Raf-1-Ras association. J. Biol. Chem. 270:24585&#226;&#8364;&#8220;24588. <br /> 43.   Parsons, A. B., A. Lopez, I. E. Givoni, D. E. Williams, C. A. Gray, J. Porter,   55. Schwede, T., J. Kopp, N. Guex, M. C. Peitsch. 2003. SWISS-MODEL: an <br />       G. Chua, R. Sopko, R. L. Brost, C. H. Ho, J. Wang, T. Ketela, C. Brenner,            automated protein homology-modeling server. Nucleic Acids Res. 31:3381&#226;&#8364;&#8220; <br />       J. A. Brill, G. E. Fernandez, T. C. Lorenz, G. S. Payne, S. Ishihara, Y. Ohya,       3385. <br />       B. Andrews, T. R. Hughes, B. J. Frey, T. R. Graham, R. J. Andersen, C.       56. Siligardi, G., B. Hu, B. Panaretou, P. W. Piper, L. H. Pearl, C. Prodromou. <br />       Boone. 2006. Exploring mode-of-action bioactive compounds chem-            2004. Co-chaperone regulation conformational switching Hsp90 <br />       ical-genetic profiling yeast. Cell 126:611&#226;&#8364;&#8220;625.          <br /> </body></html>