<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Stu2p: A Microtubule-Binding Protein Is <br /> an Essential Component Yeast Spindle Pole Body <br /> Peijing Jeremy Wang Tim C. Huffaker <br /> Section Biochemistry, Molecular Cell Biology, Cornell University, Ithaca, New York 14853 <br />  <br />  <br />  <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span>. Previously isolated tub2-423, cold-sensi-                     bules. Localization SPB dependent the <br /> tive allele Saccharomyces cerevisiae gene encod-                      presence microtubules, indicating Stu2p in- <br /> ing b-tubulin confers defect mitotic spindle                       tegral component SPB. Stu2p binds microtu- <br /> function. In attempt identify additional proteins                      bules vitro. We localized microtubule-bind- <br /> that important spindle function, screened                     ing domain Stu2p highly basic 100-amino acid <br /> suppressors cold sensitivity tub2-423 ob-                      region. This region contains imperfect repeats; <br /> tained alleles novel gene, STU2. STU2 es-                     repeats appear contribute microtubule bind- <br /> sential gene encodes protein sequence                         ing similar extents. These results suggest Stu2p <br /> similar proteins identified variety organisms.                    play role attachment, organization, and/or <br /> Stu2p localizes primarily spindle pole body                           dynamics microtubule ends SPB. <br /> (SPB) lesser extent spindle microtu- <br />  <br />  <br />  <br />  <br />         microtubule-organizing center (MTOC)1 eu-                      (Moritz et al., 1995). g-Tubulin required centrosome <br />  <br /> T       karyotic cells controls number, polarity, or- <br />         ganization cellular microtubules. In yeast <br /> Saccharomyces cerevisiae, MTOC spindle pole <br />                                                                              function vivo (Joshi et al., 1992) vitro (Felix et al., <br />                                                                              1994; Stearns Kirschner, 1994) essential the <br />                                                                              viability fungi (Oakley et al., 1990; Horio et al., 1991; <br /> body (SPB), embedded nuclear envelope                        Stearns et al., 1991; Sobel Snyder, 1995; Marschall et al., <br /> throughout cell cycle nucleates microtubules                    1996; Spang et al., 1996). The phenotypes S. cerevisiae <br /> extend nucleus cytoplasm. The SPB tri-                     g-tubulin mutants consistent role pro- <br /> laminar disk consisting central plaque flanking in-                 tein microtubule nucleation (Marschall et al., 1996; <br /> ner outer plaques nuclear cytoplasmic sur-                    Spang et al., 1996). <br /> faces, respectively (Byers, 1981). The inner outer                          The relationship microtubule nucleation and <br /> plaques sites microtubule association vivo                 microtubule anchoring MTOC clear. Microtu- <br /> are likely involved nucleation microtubule                   bules dynamic polymers mitosis. The observation <br /> polymerization anchoring microtubules SPB.                     poleward microtubule flux mitotic spindles (Mitchi- <br />    The nucleation microtubules thought accom-                    son, 1989; Sawin Mitchison, 1991) implies at- <br /> plished action g-tubulin complex. g-Tubu-                 tachment microtubules MTOCs arranged in <br /> lin localized MTOC animal cells (Stearns et al.,                way allows exchange tubulin subunits the <br /> 1991; Zheng et al., 1991) fungi (Oakley et al., 1990;                    microtubule ends. One way MTOC accomplish <br /> Horio et al., 1991; Sobel Snyder, 1995; Marschall et al.,                task make lateral attachments mi- <br /> 1996; Spang et al., 1996). g-Tubulin present                  crotubules. Such attachments anchor microtubules <br /> cytoplasm animal cells component large, ring-                   MTOC simultaneously allow subunit exchange <br /> shaped structure capable nucleating microtu-                      microtubule ends. In report protein, <br /> bules (Stearns Kirschner, 1994; Zheng et al., 1995). A                   Stu2p, play role anchoring organizing <br /> similar g-tubulin&#226;&#8364;&#8220;containing ring structure iden-                   microtubules S. cerevisiae MTOC. Stu2p inte- <br /> tified MTOCs <span id='am-317' about='xsp:base' typeof='owl:Thing'>base</span> nucleated microtubules                        gral component SPB capable binding later- <br />                                                                              ally microtubules. <br />  <br /> Address correspondence Tim Huffaker, Section Biochemistry, <br /> Molecular Cell Biology, Cornell University, Ithaca, NY 14853. Tel.:      Materials Methods <br /> (607) 255-9947. Fax: (607) 255-2428. E-mail:tch4@cornell.edu <br />                                                                              Yeast Strains, Media, Plasmids <br /> 1. Abbreviations used paper: DAPI, 49,6-diamidino-2-phenylindole; <br /> GFP, green fluorescent protein; HA, hemagglutinin, MTOC, microtubule-        The yeast strains used study listed Table I. Yeast growth me- <br /> organizing center; ORF, open reading frame; SPB, spindle pole body.          dia prepared described Sherman (1991). <span id='am-4' about='protege:TO' typeof='owl:Thing'>To</span> depolymerize mi- <br />  <br />  <br />  <br />  <br /> &#239;&#163;&#169; The Rockefeller University Press, 0021-9525/97/12/1271/10 $2.00 <br /> The Journal Cell Biology, Volume 139, Number 5, December 1, 1997 1271&#226;&#8364;&#8220;1280 <br /> http://www.jcb.org                                                            1271 <br />  Table I. Yeast Strains <br /> Strain                                                                                       Genotype <br />  <br /> CUY26                                       MATa his3-D200 leu2-3,112 ura3-52 <br /> CUY30                                       MATa his3-D200 leu2-3,112 ura3-52 ade2 lys2-801 <br /> CUY502                                      MATa tub2-423::URA3 ACT1::HIS3 TUB1::LEU2::TUB1 met2 his3-D200 leu2-D1 lys2-801 ura3-52 <br /> CUY546                                      MATa/MATa ade2/ADE2 his3/D200/his3-D200 leu2-3, 112/leu2-3, 112 ura3-52/ura3-52 <br /> CUY696                                      MATa tub2-423::URA3 ade2-101 his3-D200 leu2-D1 lys2-801 ura3-52 <br /> CUY935                                      MATa tub2-423::URA3 STU1::HIS3 ade2-101 his3-D200 leu2-D1 lys2-801 ura3-52 <br /> CUY936                                      MATa tub2-423::URA3 STU1::HIS3 ade2-101 his3-D200 leu2-D1 lys2-801 ura3-52 <br /> CUY1042                                     MATa tub2-423::URA3 ACT1::HIS3 stu2-1 ade2-101 his3 leu2 lys2-801 ura3-52 <br /> CUY1045                                     MATa tub2-423::URA3 stu2-2 ade2-101 his3 leu2 lys2-801 ura3-52 <br /> CUY1046                                     MATa/MATa stu2-D1::HIS3/STU2 ade2/ADE2 his3-D200/his3-D200 leu2-3, 112/leu2-3, 112 ura3-52/ura3-52 <br /> CUY1047                                     MATa/MATa tub2-423::URA3/tub2-423::URA3 stu2-1/STU2 ACT1::HIS3/ACT1 TUB1::LEU2::TUB1/TUB1 <br />                                              met2/MET2 leu2/leu2 his3/his3 lys2-801/lys2-801 ura3-52/ura3-52 <br /> CUY1060                                     MATa/MATa stu2-D1::HIS3/stu2-D1::HIS3 leu2-D1/leu2-D1 ura3-52/ura3-52 ade2-101/ADE2 <br />                                              his3-D200/his3-D200 (pWP89 pWP90) <br /> CUY1065                                     MATa/MATa stu2-D1::HIS3/stu2-D1::HIS3 leu2-D1/leu2-D1 ura3-52/ura3-52 ade2-101/ADE2 <br />                                              his3-D200/his3-D200 (pWP90) <br /> CUY1066                                     MATa/MATa stu2-D1::HIS3/stu2-D1::HIS3 leu2-D1/leu2-D1 ura3-52/ura3-52 ade2-101/ADE2 <br />                                              his3-D200/his3-D200 (pWP89) <br /> CUY1067                                     MATa stu2-D1::HIS3 his3-D200 leu2-3,112 ura3-52 (pWP70) <br /> CUY1068                                     MATa STU2-GFP::URA3 his3-D200 ura3-52 leu2-3, 112 <br /> CUY1069                                     MATa/MATa stu2-D2::HIS3/STU2 ade2/ADE2 his3-D200/his3-D200 leu2-3, 112/leu2-3,112 ura3-52/ura3-52 <br />  <br />  <br /> crotubules, cells grown 20 mg/ml nocodazole 1 h 308C       erage size 8.2 kb. The total genomic DNA content library z31 <br /> then 1 h 48C.                                                              times large yeast genome. <br />    Selected plasmids listed Table II.                                     The stu2-1 allele cloned complementation cold-sensitive <br />                                                                               <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span>phenotype</span> tub2-423 <span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span>mutation</span>. The yeast genomic library described <br /> Isolation Spontaneous Suppressors tub2-423                              transformed CUY696. After 2 d 308C, 11,500 transformants <br />                                                                               replica plated YPD plates incubated 168C 3 d. Then, <br /> 100 individual colonies (z107 cells/colony) strain CUY696        cells replica plated YPD plates incubated 168C <br /> resuspended 100 ml sterile water spread separate YPD           5 d. Three transformants able grow 168C. Plasmids recovered <br /> plates. Colonies arose incubation 168C 8 d retested    strains able suppress tub2-423 phenotype be- <br /> for growth 168C. Those retested mated CUY502 deter-        ing retransformed CUY696. The plasmids, referred pS2, <br /> mine suppression dominant recessive. The resulting         pS20, pS28, contained 6.7-, 6.5-, 10.5-kb DNA inserts, respectively, <br /> diploids sporulated, tetrads dissected determine        5-kb overlapping fragment, shown restriction mapping. <br /> whether suppression mutation single locus. These crosses        To verify authentic stu2-1 locus cloned, 6.7-kb <br /> were used establish mutations linked    XhoI&#226;&#8364;&#8220;XbaI fragment pS2 (see Fig. 2) subcloned integrating <br /> tubulin genes (TUB1, TUB2, TUB3) marked ACT1 locus          plasmid pRS305 (Sikorski Hieter, 1989) make plasmid pWP5. pWP5 <br /> closely linked TUB2 locus marked TUB1 locus linked       linearized BamHI, cuts insert, trans- <br /> the TUB3 locus. The extragenic suppressors linkage groups       formed CUY696 cells. Leu1 transformants crossed CUY1042 <br /> by crossing pairwise combinations. A suppressor strain      resulting diploids sporulated. In tetrads dissected, paren- <br /> linkage group crossed CUY935 CUY936 determine               tal genotypes observed; spores tetrad grew 168C. <br /> whether mutation linked STU1 locus (Pasqualone Huf-           To physically map STU2 gene, 3-kb PstI&#226;&#8364;&#8220;PstI fragment the <br /> faker, 1994).                                                                 genomic insert pS2 used probe l phage containing clones of <br />                                                                               yeast genomic DNA (Riles et al., 1993). The probe hybridized two <br /> Cloning Sequencing stu2-1 Allele                                      clones, American Type Culture Collection (Rockville, MD) numbers 6122 <br />                                                                               4488. These clones z2 kb apart chromo- <br /> A centromere-based yeast genomic library CUY1042 ge-            XII near PDC1. <br /> nomic DNA protocol Rose colleagues (Rose et al., 1987;             To localize stu2-1 gene cloned DNA fragment, series of <br /> Rose Broach, 1991) following modifications. High molecular       nested deletions starting ends pS2 insert created us- <br /> weight genomic DNA isolated partially digested Sau3A.            ing combination exonuclease III nuclease S1 (Heinrich, 1991). <br /> DNA fragments 6 9 kb recovered preparative agarose          Each resulting deletions transformed CUY696 test its <br /> gel electroelution ligated BamHI site pRS315 (Sikorski     ability suppress cold-sensitive phenotype tub2-423. The deletion <br /> and Hieter, 1989). The ligated DNA transformed ElectroMAX            constructs determined span stu2-1 locus sequenced. <br /> DH10B&#226;&#8222;&#162; competent cells (GIBCO BRL, Gaithersburg, MD) electro- <br /> poration. 66,000 transformants recovered pooled. 12 15 <br /> plasmids examined inserts ranging size 5.5 15 kb av-   Cloning Sequencing STU2 stu2-2 Alleles <br />                                                                               Both STU2 stu2-2 alleles recovered plasmid gap repair <br />                                                                               (Rothstein, 1991). To prepare gapped plasmid, pS2 digested with <br /> Table II. Plasmids                                                            NsiI delete 4.3-kb fragment containing entire stu2-1 coding se- <br /> Plasmids                                          Relevant markers            quence, z1.2 kb upstream sequence z450 bp downstream se- <br />                                                                               quence. The remaining fragment religated create pWP42. pWP42 <br /> pS2                                      stu2-1 LEU2 CEN6 ARSH4               linearized NsiI, dephosphorylated, gel purified, transformed <br /> pWP44                                    stu2-2 LEU2 CEN6 ARSH4               CUY1045 (stu2-2) diploid strain CUY1046 (STU2/stu2-D1:: <br /> pWP45                                    STU2 LEU2 CEN6 ARSH4                 HIS3) recovery stu2-2 STU2 alleles, respectively. Plas- <br /> pWP70                                    STU2-HA3 LEU2 CEN6 ARSH4             mids isolated Leu1 transformants, transformed Escherichia <br /> pWP89                                    STU2-GFP LEU2 CEN6 ARSH4             coli DH10BTM cells electroporation, analyzed restriction mapping. <br />                                                                               Both STU2 (pWP45) stu2-2 (pWP44) alleles obtained. As ex- <br /> pWP90                                    STU2-GFP URA3 2 mm origin <br />                                                                               pected, stu2-D1::HIS3 allele recovered CUY1046 cells. <br />  <br />  <br />  <br />  <br /> The Journal Cell Biology, Volume 139, 1997                                 1272 <br />     To narrow location mutations suppressor alleles,      (STU2/stu2-D1::HIS3). Leu1 transformants sporulated tetrads <br /> interchanged 3.3-kb SpeI fragment pS2 (stu2-1) pWP45           dissected. In case, viable His1 spores Leu1, demonstrat- <br /> (STU2). This fragment contains 1.4 kb carboxy-terminal coding se-       ing tagged STU2 constructs complement stu2-D1::HIS3 dis- <br /> quence STU2. The sequence-swapping experiment be-             ruption. <br /> tween pWP44 (stu2-2) pWP45 (STU2). The resulting hybrid plasmids <br /> were transformed CUY696 tested ability suppress the <br /> tub2-423 phenotype. This experiment showed that, stu2-1           Fluorescence Microscopy <br /> stu2-2 alleles, suppressor mutations reside 3.3-kb SpeI fragment.   GFP visualized living cells cells fixed with <br /> The 1.4-kb carboxy-terminal coding regions STU2 stu2-2        formaldehyde described below. The green fluorescence Stu2p&#226;&#8364;&#8220;GFP <br /> sequenced strands synthetic sequencing primers. The com-        sensitive fixation recovers fluorescence reduced intensity af- <br /> plete wild-type STU2 sequence submitted EMBL/GenBank/              ter removal formaldehyde. Cells grown SD medium pres- <br /> database (accession number U35247).                                            ence 1 mg/ml 49,6-diamidino-2-phenylindole (DAPI) 2 4 h visu- <br />                                                                                alize nuclear DNA living cells. <br /> Disruption STU2                                                                 Immunofluorescent staining yeast cells performed described <br />                                                                                previously (Pasqualone Huffaker, 1994). To visualize microtubules <br /> The STU2 gene disrupted one-step gene replacement method            alone, cells fixed adding formaldehyde final concentration of <br /> (Rothstein, 1991). pWP2 (see Fig. 2) cut BamHI BglII, de-         3.7% incubating 308C 1 h. To visualize HA- GFP-tagged <br /> phosphorylated gel purified. A 1.7-kb BamHI fragment pCU34            Stu2p g-tubulin, cells fixed 25 min. Rat monoclonal anti&#226;&#8364;&#8220;yeast <br /> containing HIS3 gene ligated pWP2 backbone. In re-        a-tubulin antibody, YOL1/34 (Kilmartin et al., 1982), gift J. <br /> sulting plasmid, pWP39, 541-bp segment stu2-1 encoding amino acids        Kilmartin (Medical Research Council, Cambridge, UK). Mouse mono- <br /> 57&#226;&#8364;&#8220;235 replaced 1.7-kb fragment containing HIS3 gene (see        clonal antibody 12CA5, recognizes HA epitope, purchased <br /> Fig. 2). A 4.3-kb DNA fragment liberated pWP39 digestion           Berkeley Antibody Co.(Berkeley, CA). Rabbit anti&#226;&#8364;&#8220;yeast g-tubulin <br /> with EcoRI XbaI, gel purified, transformed diploid       antibody, TUB4-1-4 (Marschall et al., 1996), gift T. Stearns <br /> strain CUY546. This disruption allele referred stu2-D1::HIS3.         (Stanford University, Stanford, CA). Rhodamine-conjugated goat anti&#226;&#8364;&#8220; <br />    An exact deletion STU2 open reading frame (ORF)          rabbit goat anti&#226;&#8364;&#8220;rat fluorescein-conjugated goat anti&#226;&#8364;&#8220;mouse sec- <br /> replacement HIS3 gene. The primers used PCR amplifi-          ondary antibodies obtained Cappel Research Products (Dur- <br /> cation HIS3 59-TTACAGTGTAAAGTATTTTTGAGTTTTTATT-                        ham, NC). Before using visualize HA tag, 12CA5 and <br /> AAAGAGTTTGAAGTTGACTGAGCTTGGTGAGCGCTAGGAG -39                                   fluorescein-conjugated goat anti&#226;&#8364;&#8220;mouse antibodies preabsorbed <br /> and 59-AGTTGAAGACTATATATTTTATTGAGTTTATGTTATG-                                  wild-type fixed yeast spheroplasts overnight 48C. <br /> GGGAGGCTACCCTCTCGTTCAGAATGACACGAT-39. Full <span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> <br /> primers obtained HPLC purification. The resulting 1.2-kb PCR <br /> product 50-bp sequences end identical se-          Plasmid Constructions In Vitro Transcription <br /> quences immediately STU2 ORF, respectively,           Translation <br /> therefore direct homologous recombination HIS3 marker with <br /> the STU2 locus. His1 transformants obtained transforming           The entire STU2 coding sequence amplified PCR primers <br /> diploid strain CUY546 PCR product. The resulting disruption al-       SP10 SP11. SP10: 59-GGGTCTAGAACCATGTCAGGAGAAGAA- <br /> lele referred stu2-D2::HIS3 (see Fig. 2).                             GAAGTA-39; SP11: 59-GGGGTCGACTTACGTCCTGGTTGTGCCT- <br />                                                                                TCC-39. SP10 introduces XbaI site (underlined) Kozak consen- <br />                                                                                sus context start codon, critical optimal translation <br /> Tagging Stu2p Green Fluorescent Protein                           (Kozak, 1986). SP11 introduces SalI site (underlined) stop <br /> Influenza Hemagglutinin Epitope                                                codon (bold). The PCR product cloned TA cloning vector <br />                                                                                pCRTMII (Invitrogen). A plasmid STU2 oriented under <br /> A BclI site engineered just stop codon STU2 PCR           control T7 promoter identified called pWP79. The ATG <br /> with following primers: 59-ACGATTTCATCATACTCC-39                   codons lying T7 promoter STU2 start codon were <br /> 59-GGAGGCTACCCTTTATTGATCAGTCCTGGTTGTCCC-39 (the stop                           eliminated removing 59-bp XbaI fragment religating gener- <br /> codon shown boldface, inserted BclI site underlined).       ate pWP82. The carboxy-terminal truncation constructs C505 C466 <br /> The 384-bp fragment amplified pWP45 primers            digesting pWP82 KpnI SpeI, respectively, self <br /> subcloned TA cloning vector pCR&#226;&#8222;&#162;II (Invitrogen, Carlsbad, CA)         ligation resulting large fragment. All carboxy-terminal <br /> to rise pWP59. The 171-bp EcoRV&#226;&#8364;&#8220;EcoRV fragment pWP59            truncations PCR SP10 upstream primer spe- <br /> subcloned EcoRV site pWP58 contains 2.6-kb               cific downstream primers containing in-frame stop codons sub- <br /> BamHI&#226;&#8364;&#8220;SphI fragment pWP45 pTZ18U plasmid (Bio Rad, Rich-             cloned pCR&#226;&#8222;&#162;II pCR&#226;&#8222;&#162;2.1 (Invitrogen). Those clones with <br /> mond, CA), resulting pWP61 right insert orientation.               STU2 oriented control T7 promoter chosen vitro <br />    To make STU2&#226;&#8364;&#8220;GFP fusion construct, S65T mutant version           transcription translation. Similarly, amino-terminal trunca- <br /> green fluorescent protein (GFP) released pRSETB-S65T (Heim            tion constructs using specific upstream primers SP11 as <br /> et al., 1995) digestion BamHI cloned engineered BclI      downstream primer. All upstream primers designed put <br /> site pWP61 giving rise pWP87. The 3.3-kb BamHI&#226;&#8364;&#8220;SphI fragment          initiation codon (either endogenous exogenous ATG) consen- <br /> pWP87 used replace 2.6-kb BamHI&#226;&#8364;&#8220;SphI fragment pWP45              sus nearly consensus Kozak context. <br /> and pWP81. pWP81 stu2-1 insert pS2 YEp vector <br /> pRS426 (Sikorski Hieter, 1989). The resulting plasmids containing the <br /> STU2&#226;&#8364;&#8220;GFP fusions referred pWP89 (YCp) pWP90 (YEp).               Synthesis Radiolabeled Stu2p In Vitro <br /> STU2&#226;&#8364;&#8220;GFP used replace chromosomal copy STU2. A              Transcription Translation <br /> 1.2-kb fragment containing URA3 gene cloned SphI site <br /> of pWP89 produce pXC277. The 5.5-kb SpeI&#226;&#8364;&#8220;SpeI fragment pXC277            In vitro transcription translation STU2 constructs carried <br /> was transformed wild-type haploid strain.                               using TNT T7 coupled reticulocyte lysate kit (Promega, <br />    DNA encoding tandem copies hemagglutinin (HA)                  Madison, WI). The reaction combining following re- <br /> epitope amplified PCR plasmid GTEPI (a gift B. Fut-         agents: 25 ml TNT rabbit reticulocyte lysate, 2 ml TNT reaction buffer, 1 ml <br /> cher, Cold Spring Harbor Laboratory, Cold Spring Harbor, NY)          TNT T7 RNA polymerase, 1 ml TNT amino acid mixture lacking methio- <br /> following primers: 59-GAAGATCTTGGCCGCATCTTTTACCCA-                         nine, 1 ml (40 U) RNasin ribonuclease inhibitor (Promega), 5 ml (50 mCi) <br /> 39 59-GAAGATCTCCGCACTGAGCAGCGTAAT-39. The PCR-                             &#226;&#8364;&#339;cell-labeling&#226;&#8364;? grade [35S]methionine (Amersham Intl., Arlington <br /> amplified HA3 DNA BglII site end (underlined). The 120-          Heights, IL), 15 ml H2O 1 ml (1 mg) plasmid DNA. The reaction mix- <br /> bp BglII-restricted PCR product ligated BclI site pWP61     ture incubated 308C 60 80 min. Depending tran- <br /> produce pWP67. The 1.81-kb SpeI&#226;&#8364;&#8220;SphI fragment pWP67 sub-                scription translation efficiency, 2&#226;&#8364;&#8220;10 vol PEM-TDTG buffer (0.1 M <br /> cloned SpeI&#226;&#8364;&#8220;SphI site pWP20. The resulting YCp plasmid con-        Pipes, 2 mM EGTA, 1 mM MgSO4, 0.1% Triton X-100, 4 mM DTT, 20 mM <br /> taining STU2-HA3 designated pWP70.                                          Taxol, 1 mM GTP) added reaction, reactions were <br />    To address tagged Stu2p constructs functional, pWP70,       clarified centrifugation 57,000 g 20 min 308C. The supernatants <br /> pWP89, pWP90 transformed independently CUY1046                   removed used microtubule cosedimentation assay. <br />  <br />  <br />  <br />  <br /> Wang Huffaker Stu2p Is Component Yeast Spindle Pole Body          1273 <br />  In Vitro Microtubule Cosedimentation Assay                                   tub2 alleles, constructed haploid strains con- <br /> Taxol-stabilized bovine brain microtubules prepared according to <br />                                                                              taining stu2-1 cold-sensitive tub2 alleles <br /> Vallee (1986). Tubulin concentration determined using protein assay      (120, 209, 401&#226;&#8364;&#8220;407) haploid strains containing <br /> kit (Bio Rad) BSA reference protein. To generate micro-     stu2-2 seven different cold-sensitive tub2 alleles <br /> tubule ends keeping tubulin concentration constant, microtubules       (418, 421, 429, 434, 438, 445, 451). We observed that <br /> were sheared repetitive passage 26-gauge hypodermic needle <br />                                                                              stu2-2 suppressed cold sensitivity tub2-418, but <br /> attached 1-ml syringe. Microtubule lengths determined pel- <br /> leting 12-mm-round glass coverslip visualizing indirect   tested tub2 alleles suppressed ei- <br /> immunofluorescence using DM1A (Blose et al., 1984) primary anti-      ther stu2 suppressors. Therefore, suppression by <br /> body. Random fields photographed polymer lengths measured           stu2-1 stu2-2 allele specific. In addition, double <br /> on negatives converted actual lengths (Mitchison Kirsch-      mutant strain containing stu2-1 tub2-404 inviable <br /> ner, 1984). <br />     The vitro microtubule cosedimentation assay performed de- <br />                                                                              308C. At temperature, yeast strains carrying either <br /> scribed Yang et al. (1989) minor modifications. Triton X-100     mutations grows wild-type rates. Thus, le- <br /> added taxol-stabilized microtubules 0.1%. A constant         thality results combination unlinked muta- <br /> clarified Stu2p (3 ml) incubated 308C 20 min increasing      tions, phenomenon referred synthetic lethality <br /> amounts microtubules 40-ml reaction volume supplemented with <br />                                                                              (Huffaker et al., 1987). <br /> PEM-TDTG buffer necessary. Microtubules pelleted cen- <br /> trifugation 40-ml 60% glycerol PEM-TDTG cushion 40,000 g <br /> for 20 min 308C. Both supernatant glycerol cushion re-   STU2 Encodes Novel Essential Gene <br /> moved mixed 20 ml 53 SDS sample buffer glycerol <br /> (Laemmli, 1970). The microtubule pellet resuspended 100 ml SDS        We cloned dominant suppressor allele, stu2-1, its <br /> sample buffer. All samples boiled 3 min, 10 ml     ability suppress tub2-423 mutation. We obtained <br /> was subjected SDS-PAGE analysis. After electrophoresis, gel       independent clones contain z5 kb overlap- <br /> fixed, dried, quantitated using PhosphoImager (Molecular Dynam-        ping fragment. Genomic DNA insert of <br /> ics Inc., Sunnyvale, CA). <br />                                                                              plasmids directed integration LEU2 marker <br />                                                                              STU2 locus, demonstrating plasmid con- <br /> Results                                                                      tains stu2-1 allele. The STU2 gene maps chromo- <br />                                                                              XII, near PDC1 (see Materials Methods). <br /> Identification STU2 Gene                                                The smallest fragment DNA tested contains <br />                                                                              suppression activity 4.8-kb insert carried plasmid <br /> tub2-423 cold-sensitive allele yeast gene encod- <br />                                                                              pWP9 (Fig. 2). Sequence analysis showed frag- <br /> ing b-tubulin causes specific defect spindle function <br /> at 168C (Reijo et al., 1994). Spontaneous suppressors of <br /> this mutation isolated plating tub2-423 cells at <br /> 168C. Cold-resistant colonies arose frequency of <br /> z1027. We obtained 29 independent mutants grew at <br /> wild-type rates 168C. For 15 mutants, suppres- <br /> sion segregated single <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span>gene mutation</span> backcrosses to <br /> a tub2-423 strain. 10 mutations tightly <br /> linked TUB2 assumed intragenic suppressors. <br /> Three extragenic suppressors tightly linked <br /> to STU1 locus (Pasqualone Huffaker, 1994). The <br /> remaining mutations tightly linked other <br /> and dominant suppression (Fig. 1). We called the <br /> gene identified mutations STU2 (suppressor <br /> of tubulin). Neither stu2-1 stu2-2 confers conditional- <br /> <span id='am-239' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-240' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-242' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-264' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-270' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-271' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-278' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-292' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-296' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-307' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span>lethal phenotype</span> tub2-423 TUB2 background. <br />    To test genetic interactions stu2sup alleles <br />  <br />                                                                              Figure 2. STU2 subclones plasmid constructs. (A) The bar <br />                                          Figure 1. Suppression            represents 6.7-kb genomic insert pS2 contains stu2-1. <br />                                          tub2-423 stu2-1. Yeast           The stu2-1 open reading frame indicated wide arrow. A <br />                                          cells spotted YPD           smaller ORF, YLR406c, upstream stu2-1 indicated the <br />                                          plates incubated 308C        arrow. Subclones 6.7-kb insert digestion <br />                                          2 d 168C 5 d. The       Exonuclease III Nuclease S1. Right column indicates <br />                                          relevant genotypes strains       construct (1) (2) suppress the <br />                                          follows: CUY30,              cold sensitivity tub2-423. (B) The stu2-D1::HIS3 allele cre- <br />                                          TUB2 STU2; CUY696, tub2-            ated replacing 541-bp BamHI&#226;&#8364;&#8220;BglII fragment 1.7-kb <br />                                          423 STU2; CUY1042, tub2-            HIS3 gene. The stu2-D2::HIS3 allele replacing the <br />                                          423 stu2-1; CUY696/pWP45,           entire STU2 ORF HIS3. (C) A BclI restriction site in- <br />                                          tub2-423 STU2 (STU2              troduced just stop codon STU2. A 120-bp fragment <br />                                          YCp plasmid); CUY696/pS2,           encoding HA3 739-bp fragment encoding GFP was <br />                                          tub2-423 STU2 (stu2-1            cloned site. Abbreviations restriction sites: B, <br />                                          YCp plasmid); CUY1047,              BamHI; Bg, BglII; E, EcoRI; H, HindIII; N, NsiI; O, XhoI; P, <br />                                          tub2-423/tub2-423      stu2-1/      PstI; S, SpeI; V, EcoRV; X, XbaI; Z, SphI. Sites bold the <br />                                          STU2.                               vector. <br />  <br />  <br />  <br />  <br /> The Journal Cell Biology, Volume 139, 1997                                1274 <br />  ment contains 2,664-bp ORF. pWP9 contains 812-bp <br /> ORF, YLR406c, upstream larger ORF. Because <br /> pWP2, contains entire YLR406c gene, does not <br /> suppress tub2-423 allele, conclude larger <br /> ORF encode Stu2p. Both wild-type STU2 gene <br /> and stu2-2 allele cloned plasmid gap re- <br /> pair sequenced. STU2 encodes 888-amino acid, <br /> 101-kD basic protein (isoelectric point 5 8.6). Stu2p is <br /> predicted coiled-coil region (Lupas et al., 1991) <br /> <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> residues 658 764. Two potential phosphorylation <br /> sites Cdc28 kinase (S/TP[X]R/K consensus sequence) <br /> are highly basic region positions 603 645. <br />    Stu2p shows modest level similarity other <br /> proteins (Fig. 3). Stu2p 22% identical S. pombe <br /> p93dis1 (Nabeshima et al., 1995). It similar hu- <br /> man ch-TOG (Charrasse et al., 1995) C. elegans <br /> ZYG-9 (Matthews, 1997) proteins. Both proteins <br /> are twice size Stu2p. Stu2p similar the <br /> amino-terminal halves ch-TOG ZYG-9 (22% and <br /> 27% respectively). The S. cerevisiae genome does con- <br /> tain sequence encode protein signifi- <br /> cant similarity carboxy-terminal halves pro- <br /> teins. <br />    Sequence comparison STU2 sup- <br /> pressor alleles revealed base pair differences both <br /> stu2-1 (A1540G, C2561T) stu2-2 (G1537T, C2561T). <br /> The appearance T position 2561 stu2-1 stu2-2 <br /> alleles suggested nucleotide difference be <br /> due polymorphism CUY696 both <br /> suppressor mutations derived CUY1046 from <br /> which STU2 cloned. We sequenced STU2 gene <br /> from CUY696 strain does contain T at <br /> position 2561. Therefore, mutations responsible the <br /> suppressing activity stu2-1 stu2-2 A1540G                Figure 3. Alignments S. cerevisiae Stu2p, S. pombe p93dis1, <br /> G1537T, respectively. These mutations produce T514A                   human ch-TOG. Note amino-terminal half of <br /> amino acid substitution stu2-1 D513Y amino acid                ch-TOG shown figure; length protein 1,972 <br /> substitution stu2-2.                                                 amino acids. The sequence STU2 available GenBank/ <br />    To determine STU2 required mitotic                    EMBL/DDBJ accession number U35247. <br /> growth, replaced stu2-1 sequence encoding amino <br /> acids 57-235 DNA fragment bearing HIS3 <br /> marker (Fig. 2 B, stu2-D1::HIS3). The resulting construct               STU2 deletion. The GFP-tagged Stu2p visualized in <br /> was transformed wild-type His2 diploid. A His1&#195;&#8222;                  stu2-D1::HIS3 cells carrying plasmid-borne STU2&#226;&#8364;&#8220;GFP or <br /> transformant shown PCR contain copy                    cells chromosomal copy STU2 replaced by <br /> the disrupted STU2 gene (data shown). The His1 trans-               STU2&#226;&#8364;&#8220;GFP. The fluorescence <span id='am-318' about='xsp:pattern' typeof='owl:Thing'>pattern</span> observed of <br /> formant sporulated tetrads dissected. Each              situations similar, staining intensity was <br /> 20 tetrads contained His2 spores able form             somewhat greater STU2&#226;&#8364;&#8220;GFP plasmids. In living <br /> colonies spores unable form colonies,              cells, bright fluorescent dot observed unbudded <br /> demonstrating STU2 gene essential viability.            cells, dots generally observed budded cells <br /> Similar results obtained using precise deletion               (Fig. 4). In small budded cells, dots ad- <br /> the entire STU2 coding region (Fig. 2 B, stu2-D2::HIS3).                jacent separated z1 mm. In large budded cells, the <br /> The stu2-D1::HIS3 spores germinated YPD medium                       fluorescent dots segregated cell <br /> and (n 5 6) gave rise average 12 progeny                  body. The intensity staining did appear change <br /> cells. The ability stu2-D1::HIS3 cells undergo           course cell cycle. When cells grown in <br /> four cell divisions attributable supply            presence DAPI, observed Stu2p&#226;&#8364;&#8220;GFP <br /> wild-type Stu2p inherited diploid parent.                    dots resided periphery nuclear DNA. <br />                                                                            We examined localization HA3-tagged <br />                                                                         Stu2p. This construct able complement STU2 de- <br /> Stu2p Is Component SPB                                         letion, Stu2p-HA3 visualized stu2-D1::HIS3 cells <br /> A fusion STU2 coding region GFP                  carrying plasmid-borne STU2-HA3. The localization pat- <br /> constructed allow visualization Stu2p. This construct             tern Stu2p-HA3 observed for <br /> on YCp plasmid, YEp plasmid, integrated single                  Stu2p&#226;&#8364;&#8220;GFP; resided dots nuclear <br /> copy yeast genome able complement                     periphery cells (Fig. 5 C). This localization pattern <br />  <br />  <br />  <br />  <br /> Wang Huffaker Stu2p Is Component Yeast Spindle Pole Body   1275 <br />                                                                       Figure 5. (A) Colocalization Stu2p-GFP g-tubulin. <br />                                                                      CUY1060 cells stained anti-Tub4p antibody (middle) <br />                                                                      DAPI (right). The left frame shows fluorescence of <br />                                                                      Stu2p&#226;&#8364;&#8220;GFP cell. (B) CUY1060 cells treated <br />                                                                      nocodazole depolymerize microtubules stained with <br />                                                                      anti-tubulin antibody (middle) DAPI (right). The left frame <br />                                                                      shows fluorescence Stu2p&#226;&#8364;&#8220;GFP cell. (C) Local- <br /> Figure 4. Localization Stu2p&#226;&#8364;&#8220;GFP fusion protein. GFP fluo- <br />                                                                      ization Stu2p-HA3. CUY1066 cells stained anti- <br /> rescence visualized diploid cells, homozygous stu2- <br />                                                                      hemaglutinin antibody (left) DAPI (right). <br /> D1::HIS3 disruption, contain plasmid carrying STU2-GFP <br /> (CUY1065). Cells grown presence 1 mg/ml DAPI <br /> for 2 h SD media. Left column shows fluorescence of <br /> Stu2p-GFP; right column shows DNA staining DAPI            Stu2p Binds Microtubules In Vitro <br /> and differential interference contrast image cells. Bar, 5 mm. <br />                                                                      Two lines evidence suggested Stu2p directly <br />                                                                      interact microtubules. First, mutations STU2 dis- <br /> suggested Stu2p likely component SPB.              play allele-specific suppression tub2-423; second, <br /> To confirm hypothesis, compared Stu2p&#226;&#8364;&#8220;GFP                Stu2p localizes SPB microtubule ends re- <br /> localization g-tubulin, known component            lesser extent length microtu- <br /> the SPB (Sobel Snyder, 1995; Marschall et al., 1996;             bules. To determine Stu2p binds microtubules <br /> Spang et al., 1996). We observed Stu2p&#226;&#8364;&#8220;GFP colocal-             directly, performed vitro microtubule-cosedimen- <br /> ized g-tubulin (Fig. 5 A).                                      tation assay. 35S-labeled Stu2p synthesized vitro <br />   In addition presence SPBs, minor amounts             transcription translation incubated large <br /> of Stu2p appeared localize microtubules. Weak          excess taxol-stabilized bovine brain microtubules (tubu- <br /> Stu2p&#226;&#8364;&#8220;GFP staining observed straight line             lin/Stu2p .100:1). When microtubules pel- <br /> between SPBs anaphase cells, indicating Stu2p                leted centrifugation, Stu2p pelleted <br /> along spindle microtubules (Fig. 4, row). Before              microtubules (Fig. 6 A, lane). On hand, <br /> anaphase, Stu2p appeared extend away                Stu2p remained supernatant absence of <br /> SPBs center spindle, diffi-           microtubules (Fig. 6 A, lane). The binding activity <br /> cult distinguish SPB staining           abolished treatment 0.4 M NaCl (data not <br /> relatively large size fluorescent dot SPBs         shown). <br /> the relatively short distance poles. Finally,            To determine microtubule-binding affinity Stu2p, <br /> occasionally observed weak Stu2p staining cytoplas-            constant 35S-labeled Stu2p mixed with <br /> mic fibers extended SPBs, consistent              variable amounts microtubules. As seen Fig. 6, A and <br /> some Stu2p localized cytoplasmic microtubules. How-            B, binding Stu2p microtubules concentration <br /> ever, vast majority Stu2p appeared associated              dependent saturable. The apparent dissociation con- <br /> with SPB cell cycle.                              stant, Kd, equal concentration polymerized tubu- <br />   To determine association Stu2p             lin required cosediment half Stu2p reaction <br /> SPB depends presence microtubules, cells con-            5.4 3 1027 M. <br /> taining STU2&#226;&#8364;&#8220;GFP treated microtubule de-                  The localization Stu2p SPB raises prospect <br /> polymerizing drug, nocodazole. After treatment, 90%             Stu2p bind ends microtubules g-tu- <br /> of cells contained microtubules de-             bulin does. To examine possibility, sheared micro- <br /> tected immunofluorescence. Nocodazole treatment did               tubules generate polymer ends given tubulin <br /> not affect number cells containing Stu2p&#226;&#8364;&#8220;GFP stain-           concentration. The sheared microtubules average <br /> ing (.95%) noticeably change intensity staining            length 3.4 6 1.2 mm nearly times shorter than <br /> compared untreated cells (Fig. 5 B).                              unsheared microtubules average length 9.3 6 <br />  <br />  <br />  <br />  <br /> The Journal Cell Biology, Volume 139, 1997                        1276 <br />  Figure 6. Measurement dissociation constant (Kd)        Figure 7. (A) Microtubules (left) (right) shearing <br /> length Stu2p. A constant 35S-labeled vitro translated      visualized immunofluorescence. (B) The binding curves for <br /> Stu2p (z10211 M final concentration) incubated vari-           Stu2p sheared unsheared microtubules. <br /> able amounts taxol-stabilized bovine brain microtubules. Mi- <br /> crotubules pelleted centrifugation, bound Stu2p <br /> (in pellets) unbound Stu2p (in supernatants) subjected <br /> to SDS-PAGE analysis. Band intensities quantitated using           microtubule-binding activity. These results localize mi- <br /> the phosphoimager software. (A) Autoradiographs Stu2p             crotubule-binding domain Stu2p 100 amino acid <br /> the supernatants (S) corresponding pellets (P). Micro-       region amino acids 557 658. <br /> tubule concentration ranging 0 5 mM shown            To obtain quantitative binding data of <br /> lane. (B) The binding curve generated data.          constructs, measured fraction polypeptide <br /> The Kd equal concentration polymerized tubulin re-         bound microtubules range microtubule con- <br /> quired cosediment 50% Stu2p reaction.                  centrations calculated apparent Kd described <br />                                                                         length Stu2p (Fig. 9 B). The Kd&#226;&#8364;&#8482;s the <br /> 5.2 mm (Fig. 7 A). If Stu2p binds exclusively ends            C657 N558 polypeptides 1.6- 1.4-fold <br /> microtubules, expect threefold decrease                higher, respectively, Kd length protein <br /> in Kd value measured sheared microtubules. As                  confirming 231 carboxy-terminal 557 amino- <br /> shown Fig. 7 B, Kd value measured sheared mi-               terminal amino acids contribute significantly the <br /> crotubules (0.56 6 0.07 mM) nearly identical                microtubule-binding affinity Stu2p. However, Kd&#226;&#8364;&#8482;s <br /> with unsheared microtubules (0.54 6 0.19 mM). Thus,                 C557 N658 polypeptides z30-fold and <br /> affinity Stu2p microtubules does depend               .50-fold higher length protein. <br /> concentration microtubule ends, indicating Stu2p                   The 558&#226;&#8364;&#8220;657 binding domain highly basic pre- <br /> binds laterally microtubules.                                        dicted isoelectric point 10.7 rich serine (18%), <br />                                                                         threonine (8%), proline (7%), basic amino acids (16%). <br /> The Microtubule-binding Domain Stu2p                                 In addition, region consists imperfect repeats <br /> To define microtubule-binding domain Stu2p,                   named R1 (558&#226;&#8364;&#8220;607) R2 (612&#226;&#8364;&#8220;657) (Fig. <br /> measured relative microtubule-binding affinities               9 A). The 20 amino acids R1 R2 40% iden- <br /> series amino- carboxy-terminal truncation con-                   tical (regions R1a R2a), 20 amino acids of <br /> structs (Fig. 8). Initially, measured fraction Stu2p          R1 R2 40% identical (regions R1b R2b). The <br /> that cosedimented microtubules single microtu-                sequences b regions differ length and <br /> bule concentration (16.5 mM tubulin). Truncations lack             display amino acid similarity. Interestingly, <br /> up 231 amino acids carboxy terminus Stu2p                putative Cdc28 phosphorylation site of <br /> (Fig. 8, C846, C730, C657) bind microtubules nearly              b repeats. <br /> as length Stu2p. Further deletion 100 amino <br /> acids residue 557 (C557) abolishes binding <br />                                                                         Contribution Individual Repeats Binding <br /> activity. Truncations lack 557 amino acids from <br />                                                                         Affinity Stu2p <br /> the amino terminus Stu2p (N485 N558) micro-                 We determined microtubule-binding affinities of <br /> tubule-binding activities comparable length             Stu2p polypeptides carboxy- amino-terminal end- <br /> Stu2p. However, construct lacking additional 100 amino             points located repeat domains (Fig. 9 B). A <br /> acids amino terminus (N658) residual                  carboxy-terminal truncation removes R2 (C611) a <br />  <br />  <br />  <br />  <br /> Wang Huffaker Stu2p Is Component Yeast Spindle Pole Body   1277 <br />  Figure 8. Identification Stu2p microtubule-binding domain. <br /> Various truncated Stu2p polypeptides translated vitro <br /> and tested microtubule-binding activity cosedimenta- <br /> tion assay using constant microtubules (16.5 mM tu- <br /> bulin). The percentage Stu2p pelleted microtubule is <br /> indicated (1), .90%; (1/2), 70&#226;&#8364;&#8220;90%; (2), ,30%. The num- <br /> bers adjacent endpoints represent corresponding <br />                                                                     Figure 9. Dissection microtubule-binding domain Stu2p. <br /> amino acid positions. <br />                                                                     (A) Sequences bipartite repeats microtubule- <br />                                                                     binding domain Stu2p. The identical residues these <br />                                                                     repeats indicated vertical lines. (B) The Kd&#226;&#8364;&#8482;s Stu2p <br /> nearly sixfold higher Kd length protein. An           polypeptides determined Fig. 6 presented the <br /> amino-terminal truncation removes R1 element               average measured values. The full-length Stu2p as- <br /> (N612) 3.5-fold higher Kd. Thus, R1 R2           sayed times standard error mean shown. All <br /> regions contribute microtubule binding similar            truncated polypeptides assayed twice range is <br /> extents.                                                            included. The line represents full-length Stu2p. The num- <br />   We generated truncations removed half           bers adjacent endpoints represent positions cor- <br /> repeat. In cases, removing half repeat       responding amino acids. <br /> has nearly effect removing repeat. <br /> For example, C637 lacks element R2b nearly <br /> the binding affinity C611, lacks el-         interacts directly microtubules. Alleles STU2 were <br /> ements R2a R2b. Similarly, C587, contains            identified suppressors tub2-423, cold-sensitive mu- <br /> element R1a, N638, contains element R2b,             tation yeast b-tubulin gene. Thus, stu2sup alleles <br /> do bind microtubules better C557               compensate microtubule defect caused tub2- <br /> N658 constructs lack entire binding region. The            423 mutation. The simplest explanation result is <br /> one exception construct N583 lacks            tub2-423 mutation compromises ability of <br /> element R1a binding affinity equivalent           Stu2p associate microtubules, situation is <br /> to N558, contains complete R1 R2             remedied corresponding alterations Stu2p. This <br /> elements. We determined R1a re-             hypothesis supported observation suppres- <br /> quired microtubule binding absence R2.                sion allele specific; 16 tub2 mutations <br />                                                                     tested suppressed stu2sup alleles. In <br />                                                                     addition, stu2sup alleles dominant suppressors ex- <br /> Discussion                                                          pected reciprocal interaction. Finally, syn- <br /> We report identification Stu2p, yeast microtubule-         thetic lethality observed stu2-1 tub2-404 also <br /> binding protein located SPB. Both Stu2p-             indicates products genes interact. <br /> HA3 -GFP fusion proteins localize primarily                 Consistent genetic data, shown that <br /> SPB vegetative cells stages cell cycle      Stu2p binds microtubules vitro. We performed as- <br /> a lesser extent length microtubules. We be-            says using bovine brain microtubules, yeast micro- <br /> lieve fusion proteins mimic authentic Stu2p, be-          tubules difficult obtain large quantities can <br /> cause rescue stu2 null mutation. SPB localization          stabilized taxol (Barnes et al., 1992). Yeast and <br /> is dependent presence detectable microtu-             bovine tubulins share .70% identity shown <br /> bules, indicating Stu2p integral component        bind profile proteins yeast extracts <br /> SPB bind SPB component.                 (Barnes et al., 1992). Therefore, assume bind- <br /> This experiment caveat small poly-         ing properties Stu2p bovine brain microtubules re- <br /> mer undetectable immunofluorescence re-              flect binding properties Stu2p yeast microtubules. <br /> main near SPB treatment cells nocodazole.         Stu2p binds microtubules apparent Kd 0.54 <br />    Both genetic biochemical evidence indicate Stu2p        mM tubulin. This value times greater than <br />  <br />  <br />  <br />  <br /> The Journal Cell Biology, Volume 139, 1997                       1278 <br />  that obtained neuronal microtubule-associated pro-              Thus, Stu2p unique integral component of <br /> tein, tau, using assay (Goode Feinstein, 1994).            SPB associates laterally microtubules. <br /> Thus, affinity Stu2p microtubules                 One potential role Stu2p tethering microtubules <br /> range expected bona fide microtubule-associated                   SPB. The minus ends microtubules animal cells <br /> protein.                                                                shown exchange tubulin subunits the <br />    In vitro binding assays using truncations Stu2p              process poleward microtubule flux (Mitchison, 1989; <br /> defined z100 amino acid region necessary mi-              Sawin Mitchison, 1991). Because Stu2p binds laterally <br /> crotubule binding. This region does signifi-               microtubules, maintain attachment mi- <br /> cant sequence similarity related microtubule-binding             crotubules pole subunit exchange the <br /> domains tau, MAP2, MAP4 (Lee et al., 1988; Lewis                 ends. In addition, Stu2p involved organiza- <br /> et al., 1988; Chapin Bulinski, 1991) unrelated mi-           tion microtubule ends SPB. We evidence via <br /> crotubule-binding domain MAP1B (Noble et al., 1989).                 two-hybrid Stu2p dimerize (Chen, P.X., <br /> However, like domains, positively charged               T.C. Huffaker, unpublished observations) may <br /> contains repeated elements. The sequence alignment                   allow crosslink microtubules. Thus, Stu2p bun- <br /> Stu2p S. pombe homologue p93dis1 shows                 dle microtubules near SPB help generate paral- <br /> microtubule-binding region Stu2p corresponds re-                lel array microtubules observed yeast spindles. Finally, <br /> gion p93dis1 falls reported microtu-            Stu2p influence microtubule dynamics its <br /> bule-binding domain (Nakaseko et al., 1996). However,                   binding near minus ends. These roles Stu2p not <br /> the microtubule-binding regions proteins                mutually exclusive, Stu2p act struc- <br /> show sequence similarity remainder                 tural regulatory component yeast mitotic spindle. <br /> the proteins, p93dis1 microtubule-binding domain                   Homologues Stu2p variety organ- <br /> does contain repeated elements. The fact                   isms, indicating Stu2p evolutionarily conserved <br /> Stu2p microtubule-binding region highly basic                functions. In addition S. pombe p93dis1 (Nabeshima <br /> carboxyl-terminal portion tubulins acidic suggests                et al., 1995), C. elegans ZYG-9 (Matthews, 1997), and <br /> that Stu2p interact microtubules elec-               human ch-TOG (Charrasse et al., 1995), Xenopus pro- <br /> trostatic interactions. This model supported find-            tein XMAP215 (Gard Kirschner, 1987) appears <br /> ing Stu2p binding microtubules, like              member family. Partial sequencing of <br /> microtubule-associated proteins (Vallee, 1986), sensitive            XMAP215 shows similar ch-TOG (Charrasse, <br /> to salt. The amino acid substitutions stu2-1 stu2-2           S., M. Schroeder, C. Gauthier-Rouviere, L. Cassimeris, D.L. <br /> not lie microtubule-binding domain Stu2p               Garrd, C. Larroque. 1996. Mol. Biol. Cell. 7:222a). <br /> might expected. However, residues close               Like Stu2p, p93dis1 (Nabeshima et al., 1995), ZYG-9 (Mat- <br /> to microtubule-binding domain play role                thews, 1997), ch-TOG (Charrasse, S., M. Schroeder, C. <br /> altering Stu2p conformation enable interact                Gauthier-Rouviere, L. Cassimeris, D.L. Garrd, C. <br /> more effectively tub2-423 microtubules.                            Larroque. 1996. Mol. Biol. Cell. 7:222a), XMAP215 <br />    The microtubule-binding region Stu2p imper-               (Gard et al., 1995) localize spindle poles along <br /> fect bipartite repeats. Truncations Stu2p lacking             spindle microtubules mitosis. However in- <br /> repeat bind microtubules fivefold decrease              terphase, p93dis1 localizes cytoplasmic microtubules <br /> affinity, indicating repeat capable binding             ch-TOG colocalizes endoplasmic reticulum <br /> microtubules independently. This indicates Stu2p               markers. p93dis1 (Nakaseko et al., 1996), ch-TOG (Char- <br /> bind adjacent tubulin subunits microtubules.              rasse, S., M. Schroeder, C. Gauthier-Rouviere, L. Cassim- <br /> There potential phosphorylation site Cdc28 ki-               eris, D.L. Garrd, C. Larroque. 1996. Mol. Biol. Cell. 7: <br /> nase repeat, suggesting Stu2p phos-               222a), XMAP215 (Gard Kirschner, 1987) all <br /> phorylated cell cycle-dependent manner. It known                capable binding microtubules vitro, XMAP215 <br /> that phosphorylation tau reduces vitro micro-             shown promote microtubule assembly and <br /> tubule-binding activity (Biernat et al., 1993; Bramblett et al.,        turnover (Vasquez et al., 1994). Therefore, rea- <br /> 1993). By analogy, phosphorylation Stu2p regu-                 sonable conclude proteins family per- <br /> late ability bind microtubules.                                  form overlapping functions. Further study these <br />    In addition Stu2p, proteins abil-            homologues provide insights specific roles <br /> ity bind microtubules shown located               play spindle function. <br /> the SPB. Two these, kinesin-related protein Kar3p                We thank (from Cornell University) William Brown Ken Kemphues <br /> (Meluh Rose, 1990; Page et al., 1994; Saunders et al.,              comments manuscript, Daniel Starr David Wolfgang for <br /> 1997) dynein heavy chain protein Dhc1p (Yeh et al.,             performing preliminary experiments led work, Xiaoyue <br /> 1995), microtubule motors. The association Dhc1p               Peter Chen providing CUY1068, Roger Y. Tsien (University of <br /> the SPB dependent presence microtubules;                   California, San Diego) permission use mutant GFP. <br /> whether Kar3p localization SPB depends micro-                    This work supported grant National Institutes of <br /> tubules known. The S. cerevisiae g-tubu-            Health (GM40479). <br /> lin Tub4p (Sobel Snyder, 1995; Marschall et al., 1996;              Received publication 29 July 1997 revised form 18 September <br /> Spang et al., 1996). Like Stu2p, localization Tub4p              1997. <br /> to SPB dependent presence microtu- <br /> bules. However, unlike Stu2p, g-tubulin human                  References <br /> Xenopus shown bind specifically minus                Barnes, G., K.A. Louie, D. Botstein. 1992. Yeast proteins associated with <br /> ends microtubules (Li Joshi, 1995; Zheng et al., 1995).            microtubules vitro vivo. Mol. Biol. Cell. 3:29&#226;&#8364;&#8220;47. <br />  <br />  <br />  <br />  <br /> Wang Huffaker Stu2p Is Component Yeast Spindle Pole Body   1279 <br />  Biernat, J., N. Gustke, G. Drewes, E.M. Mandelkow, E. Mandelkow. 1993.                 lated centrosomes. Nature. 312:232&#226;&#8364;&#8220;237. <br />    Phosphorylation Ser262 strongly reduces binding tau microtubules:           Moritz, M., M.B. Braunfeld, J.W. Sedat, B. Alberts, D.A. Agard. 1995. Mi- <br />    distinction PHF-like immunoreactivity microtubule binding.                  crotubule nucleation g-tubulin&#226;&#8364;&#8220;containing rings centrosome. Na- <br />    Neuron. 11:153&#226;&#8364;&#8220;163.                                                                     ture. 378:638&#226;&#8364;&#8220;640. <br /> Blose, S.H., D.I. Meltzer, J.R. Feramisco. 1984. 10-nm filaments in-            Nabeshima, K., H. Kurooka, M. Takeuchi, K. Kinoshita, Y. Nakaseko, M. <br />    duced collapse living cells microinjected monoclonal poly-               Yanagida. 1995. p93dis1, required sister chromatid separation, a <br />    clonal antibodies tubulin. J. Cell Biol. 98:847&#226;&#8364;&#8220;858.                            novel microtubule spindle pole body-associating protein phosphorylated <br /> Bramblett, G.T., M. Goedert, R. Jakes, S.E. Merrick, J.Q. Trojanowski,                 Cdc2 target sites. Genes Dev. 9:1572&#226;&#8364;&#8220;1585. <br />    V.M. Lee. 1993. Abnormal tau phosphorylation Ser396 Alzheimer&#226;&#8364;&#8482;s                Nakaseko, Y., K. Nabeshima, K. Kinoshita, M. Yanagida. 1996. Dissection <br />    disease recapitulates development contributes reduced microtubule                fission yeast microtubule associating protein p93Dis1: regions implicated in <br />    binding. Neuron. 10:1089&#226;&#8364;&#8220;1099.                                                          regulated localization microtubule interaction. Genes Cells. 1:633&#226;&#8364;&#8220;644. <br /> Byers, B. 1981. Cytology yeast life cycle. In The Molecular Biology       Noble, M., S.A. Lewis, N.J. Cowan. 1989. The microtubule binding domain <br />    Yeast Saccharomyces. J.N. Strathern, E.W. Jones, J.R. Broach, editors.              microtubule-associated protein MAP1B contains repeated sequence <br />    Cold Spring Harbor Laboratory, Cold Spring Harbor, New York. 59&#226;&#8364;&#8220;96.                     motif unrelated MAP2 tau. J. Cell Biol. 109:3367&#226;&#8364;&#8220;3376. <br /> Chapin, S.J., J.C. Bulinski. 1991. Non-neuronal 210 3 103 Mr microtubule-           Oakley, B.R., C.E. Oakley, Y. Yoon, M.K. Jung. 1990. g-tubulin com- <br />    associated protein (MAP4) contains domain homologous microtu-                  ponent spindle pole body essential microtubule function in <br />    bule-binding domains neuronal MAP2 tau. J. Cell Sci. 98:27&#226;&#8364;&#8220;36.                   Aspergillus nidulans. Cell. 61:1289&#226;&#8364;&#8220;1301. <br /> Charrasse, S., M. Mazel, S. Taviaux, P. Berta, T. Chow, C. Larroque. 1995.          Page, B.D., L.L. Satterwhite, M.D. Rose, M. Snyder. 1994. Localization of <br />    Characterization cDNA pattern expression new gene over-              Kar3 kinesin heavy chain&#226;&#8364;&#8220;related protein requires Cik1 interating pro- <br />    expressed human hepatomas colonic tumors. Eur. J. Biochem. 234:                  tein. J. Cell Biol. 124:507&#226;&#8364;&#8220;520. <br />    406&#226;&#8364;&#8220;413.                                                                             Pasqualone, D., T.C. Huffaker. 1994. STU1, suppressor b-tubulin <br /> Felix, M.A., C. Antony, M. Wright, B. Maro. 1994. Centrosome assembly               mutation, encodes novel essential component yeast mitotic spin- <br />    vitro: role g-tubulin recruitment Xenopus sperm aster formation. J. Cell          dle. J. Cell Biol. 127:1973&#226;&#8364;&#8220;1984. <br />    Biol. 124:19&#226;&#8364;&#8220;31.                                                                     Reijo, R.A., E.M. Cooper, G.J. Beagle, T.C. Huffaker. 1994. Systematic <br /> Gard, D.L., M.W. Kirschner. 1987. A microtubule-associated protein                mutational analysis yeast b-tubulin gene. Mol. Biol. Cell. 5:29&#226;&#8364;&#8220;43. <br />    Xenopus eggs specifically promotes assembly plus end. J. Cell            Riles, L., J.E. Dutchik, A. Baktha, B.K. McCauley, E.C. Thayer, M.P. Leckie, <br />    Biol. 105:2203&#226;&#8364;&#8220;2216.                                                                    V.V. Braden, J.E. Depke, M.V. Olson. 1993. Physical maps six <br /> Gard, D.L., B.J. Cha, M.M. Schroeder. 1995. Confocal immunofluores-                    smallest chromosomes Saccharomyces cerevisiae resolution 2.6 ki- <br />    cence microscopy microtubules, microtubule-associated  <br /> </body></html>