<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Published online 2 October 2008                                              Nucleic Acids Research, 2008, Vol. 36, No. 19 6269&#226;&#8364;&#8220;6283 <br />                                                                                                                 doi:10.1093/nar/gkn636 <br />  <br />  <br /> Evolutionary origins human apoptosis <br /> and genome-stability gene networks <br /> Mauro A. A. Castro1,2,*, Rodrigo J. S. Dalmolin1, Jose&#194;&#180; C. F. Moreira1, <br /> Jose&#194;&#180; C. M. Mombach3 Rita M. C. Almeida4 <br /> 1 <br />  Bioinformatics <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>Unit</span>, Department Biochemistry, Federal University Rio Grande Sul (UFRGS), Rua Ramiro <br /> Barcelos 2600-anexo, Porto Alegre 90035-003, 2Department Biological Sciences, Lutheran University Brazil, <br /> Gravata&#196;&#177;&#194;&#180; 94170-240, 3Department Physics, Federal University Santa Maria (UFSM), Santa Maria 97105-900 <br /> and 4Institute Physics, Federal University Rio Grande Sul (UFRGS), Avenida Bento Gonc&#194;&#184;alves 9500, Porto <br /> Alegre 91501-970, Caixa Postal 15051, Brazil <br />  <br /> Received May 23, 2008; Revised September 14, 2008; Accepted September 15, 2008 <br />  <br />  <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span>                                                                        INTRODUCTION <br /> Apoptosis essential complex multicellular                                The concept apoptosis associated mainte- <br /> organisms failure associated                                    nance tissue homeostasis (1). The programmed cell <br /> genome instability cancer. Interactions                             death (PCD) perspective multicellular organisms <br /> apoptosis genome-maintenance mechanisms                                     guarantees substitution old and/or dysfunctional <br />                                                                                 cells, impaired accumulation cellular <br /> have extensively documented include <br />                                                                                 damages environmental insults, partici- <br /> transactivation-independent -dependent func-                                pates directly tissue development (2). According <span id='am-7' about='protege:TO' typeof='owl:Thing'>to</span> <br /> tions, tumor-suppressor protein p53                                KEGG (3), reference pathway database, up <br /> works &#226;&#8364;&#732;molecular node&#226;&#8364;&#8482; DNA-damage                                   100 genes coordinately working apoptosis. Removing <br /> response. Although apoptosis genome stability                               components a&#239;&#172;&#8364;ects it <br /> have identified ancient pathways eukary-                             impair pathway. In complex metazoan <br /> ote phylogeny, biological evolution underlying                              organisms, defective apoptosis associated orga- <br /> the emergence integrated remains lar-                              nogenesis disorders uncontrolled cell growth, <br /> gely unknown. Here, using computational methods,                                typically neoplastic diseases (4). In the <br /> we reconstruct evolutionary scenario linked                            perspective cancer cell, suppressed apoptosis a <br /> apoptosis genome stability pathways func-                             requirement order enhance cell &#239;&#172;?tness (5). In <br />                                                                                 extent, thought apoptosis related to <br /> tional human gene/protein association network. We <br />                                                                                 genome instability sense <span id='am-320' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-321' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-322' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-323' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-324' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-325' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-326' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-327' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-328' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-329' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-330' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-331' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-332' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-333' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-334' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-335' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-336' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-337' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-338' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-341' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-342' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-343' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-344' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-345' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-347' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-348' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-349' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-350' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-351' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-352' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-353' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-354' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-355' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-356' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-357' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-358' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-359' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-362' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-363' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-364' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-365' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-366' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-367' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-368' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-369' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-370' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-372' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-373' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-374' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-375' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-376' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-378' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-380' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-381' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-382' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-383' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-384' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-385' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-386' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-389' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-390' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-391' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-392' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-393' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-394' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-395' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-396' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-397' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span>mutation</span> prone <br /> found entanglement DNA repair, chromo-                              clones, containing aberrant genetic content (i.e. high <br /> some stability apoptosis gene networks                                      number chromosome aberrations DNA point- <br /> appears caspase gene family anti-                              mutations), need defective apoptosis escape cell <br /> apoptotic gene BCL2. Also, critical nodes                               death (6&#226;&#8364;&#8220;8). <br /> that entangle apoptosis genome stability                                   Genome-maintenance mechanisms intimately linked <br /> cancer genes (e.g. ATM, BRCA1, BRCA2, MLH1,                                     apoptotic components, indicates high number <br /> MSH2, MSH6 TP53), orthologs                                  proteins interact tumor-suppressor <br /> have arisen different points evolution. Our                               protein p53. In fact, protein interacts four <br /> results demonstrate genome stability                                    major DNA repair mechanisms: nucleotide excision repair <br /> apoptosis co-opted evolution recruiting                             (NER), <span id='am-476' about='xsp:base' typeof='owl:Thing'>base</span> excision repair (BER), mismatch repair <br />                                                                                 (MMR) recombinational repair (RER)&#226;&#8364;&#8221;homologous <br /> genes merge systems. We provide <br />                                                                                 recombinational repair (HRR) nonhomologous end- <br /> several examples exploit evolutionary plat- <br />                                                                                 joining (NHEJ). Concerning NER MMR, p53 can <br /> form, judiciously extended informa-                               act transactivation-independent -dependent <br /> tion gene essentiality inferred model                                   manner (9). Furthermore, DNA repair proteins <br /> organisms human.                                                             stimulate apoptosis response DNA lesions, <br />  <br /> *To correspondence addressed. Tel: +55 51 3308 5577; Fax: +55 51 3308 5540; Email: mauro@ufrgs.br <br /> Correspondence addressed Rita M.C. Almeida. Tel: +55 51 3308 6521; Fax: +55 51 3308 7286; Email: rita@if.ufrgs.br <br />  <br /> The authors wish known that, opinion, authors regarded joint First Authors. <br />  <br /> &#195;&#376; 2008 The Author(s) <br /> This Open Access article distributed terms Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/ <br /> by-nc/2.0/uk/) permits unrestricted non-commercial use, distribution, reproduction medium, provided original work properly cited. <br />  6270 Nucleic Acids Research, 2008, Vol. 36, No. 19 <br />  <br />  <br /> as example BER-associated protein poly(ADP-            human gene network. As result obtained gene <br /> ribose) polymerase-1 (PARP1) (10) MMR                  network possible discriminate ancient, less <br /> proteins MSH2, MSH6 MLH1 (11). Indeed, over-           plastic essential regions earlier, plas- <br /> lapping apoptosis DNA repair genes renders           tic essential ones. Furthermore, cancer <br /> di&#239;&#172;&#402;cult precise de&#239;&#172;?nition functional boundaries           genes identi&#239;&#172;?ed gene network located the <br /> among systems, characteristic complex        earlier, plastic essential region. We antici- <br /> biological networks (12).                                      pate analyses applied study origins <br />    On hand, apoptosis genome-stability net-      broad range neoplastic diseases. <br /> works di&#239;&#172;&#8364;erent evolutionary roots. For instance, the <br /> core machinery eukaryotic repair systems be <br /> conserved domains life,         MATERIALS AND METHODS <br /> expressive number eukaryotic proteins counter- <br /> part archaea bacteria (13). Likewise, metazoan apo-      Human gene/protein-association network <br /> ptosis contains components identi&#239;&#172;?ed       The protein&#226;&#8364;&#8220;protein interaction network associating 180 <br /> in ancient organisms prokaryotes unicellular       human genes apoptosis genome-stability pathways <br /> eukaryotes. However, molecular sources             extensively described Ref. (19). Brie&#239;&#172;&#8218;y, the <br /> eukaryotic apoptosis network inherited         network generated using database <span id='am-4' about='xsd:string' typeof='owl:Thing'>STRING</span> (24) <br /> <span id='am-5' about='protege:FROM' typeof='owl:Thing'>from</span> prokaryotes horizontal gene transfer (HGT)          input options &#226;&#8364;&#732;databases&#226;&#8364;&#8482;,&#226;&#8364;&#8482;experiments&#226;&#8364;&#8482; 0.700 <br /> di&#239;&#172;&#8364;erent events, exapted new functions form        con&#239;&#172;?dence level. STRING integrates di&#239;&#172;&#8364;erent curated, <br /> apoptosis network (14).                                        public databases containing information direct and <br />    Notwithstanding components networks        indirect functional protein&#226;&#8364;&#8220;protein associations. Each <br /> having extensively identi&#239;&#172;?ed eukaryote phylogeny       protein identi&#239;&#172;?ed according gene HUGO ID <br /> (15,16), data available evolutionary         (25) Ensembl Peptide ID (26) (Supplementary <br /> scenario functionally linked apoptosis genome-         Table S1). The results search saved data <br /> stability gene network (5,17,18). One approach assess       &#239;&#172;?les describing links genes handled <br /> the role component given interacting network      Medusa software (27). <br /> is comparative genomics. Using well-studied <br /> models, yeast mouse, comparative genomics pro-          Parsimony analysis: inferring evolutionary roots human <br /> vides powerful tools draw evolutionary inferences       apoptosis genome-stability genes <br /> poorly studied organisms (16). <br />    In previous paper characterized entanglement       The parsimony analysis divided major steps in <br /> among apoptosis genome-stability pathways             order construct parsimonious scenarios individual <br /> human protein&#226;&#8364;&#8220;protein-association network (19). Here,          sets orthologous, given species tree. We &#239;&#172;?rst built a <br /> we extend characterization build platform         consensus phylogeny eukaryotes listed STRING <br /> transfer functional information organisms         database (22). The eukaryote phylogeny based a <br /> to human. The idea based consensus         manual integration variety phylogenies (28&#226;&#8364;&#8220;33). <br /> component gene/protein interaction network         We determined presence homologs the <br /> present living organisms origin point   organisms species tree 180 genes apoptosis <br /> evolution. Thus scenario gives rise present    genome-stability networks. Likely homology was <br /> network tracked-down searching root           inferred using orthology information eukary- <br /> each component given species tree.                        otic clusters orthologous groups proteins (KOGs) <br />    Our goal create orthology map        (21), retrieved orthology assign- <br /> species tree human apoptosis genome-stability      ments STRING server; STRING augmented <br /> gene/protein-association network order transfer       KOG orthology information adding additional <br /> humans information described eukaryotes.         species (currently 35 eukaryotes) creating more <br /> We searched orthologs [i.e. homologous genes derived       groups (NOGs, nonsupervised orthologous groups) as <br /> from single ancestral gene common ances-         giving direct association three-domain <br /> tor (LCA) compared species (20)] 35 fully             phylogeny. In total, 142 eukaryotic orthologous groups <br /> sequenced eukaryotic genomes. Likely orthology             identi&#239;&#172;?ed (Supplementary Table S1). To benchmark <br /> inferred orthologous groups using STRING data-            analysis, retrieved orthologous groups <br /> base (21,22), set orthologs       set genes using Inparanoid database, as <br /> most parsimonious scenario eukaryote phylogeny          discussed later. <br /> (23). To verify orthology data, reconstructed         The second major step reconstruction the <br /> entire analysis using Inparanoid database di&#239;&#172;&#8364;erent        evolutionary scenario individual set ortho- <br /> data source, essentially obtained results         logous genes. This problem previously formu- <br /> (see Supplementary materials). As network char-        lated follows (23): given species tree set of <br /> acterizations, estimated gene plasticity measuring       orthologs particular phyletic <span id='am-477' about='xsp:pattern' typeof='owl:Thing'>pattern</span>, &#239;&#172;?nd the <br /> gene abundance distribution orthologous            parsimonious mapping set orthologs <br /> groups extant species, considered essenti-       tree. Precisely, concerning problem, this <br /> ality data available yeast mouse orthologs. Both       question restated as: orthologous <br /> plasticity essentiality information transferred    group associated human apoptosis and <br />                                                                          Nucleic Acids Research, 2008, Vol. 36, No. 19 6271 <br />  <br />                                                                              P <br /> genome-stability genes, &#239;&#172;?nd earliest ortholog          p&#195;&#176;i;  &#195;&#382; &#194;&#188; 1. The normalized Shannon infor- <br /> eukaryote phylogeny.                                              mation function H  de&#239;&#172;?ned as <br />   The incongruence evolutionary scenario is <br /> resolved according gain/penalty approach (23),                       1 X <br />                                                                   H  &#194;&#188;            p&#195;&#176;i;  &#195;&#382; ln p&#195;&#176;i;  &#195;&#382;;                  3 <br /> where parsimonious scenario presence/absence                  ln M i <br /> of genes ancestral nodes tree obtained <br /> by using inconsistency function de&#239;&#172;?ned                      divided ln(M) order normalize the <br />                                                                   quantities, guaranteeing 0   H    1. Observe if <br /> S &#194;&#188; l &#195;&#190; g ;                                                  1    gene organism, N  = M, p(i, ) = 1/M, <br />                                                                   H  = 1. In fact, H  re&#239;&#172;&#8218;ects spread distribu- <br /> where l number gene losses,   number          tion s(i, ), i.e. measures diversity exists the <br /> gene gains g gain penalty. For di&#239;&#172;&#8364;erent           ath OG. H  near 0 indicates poor diversity, H  <br /> scenario function S calculated parsimo-         close 1 suggests high diversity. As complementary <br /> nious scenario chosen yields mini-         quantity, estimate abundance D  ath <br /> mum value S. The relative costs evolutionary            OG simply obtaining ratio number of <br /> events consider cost units gene birth gene             items (orthologous genes) number organisms. <br /> acquisition (i.e. g = 2), cost <span id='am-3' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> gene loss. <br /> This ratio proposed Mirkin coworkers (23).              Transference functional information yeast and <br /> Subsequently, works validate 2:1 ratio pro-          mouse human gene/protein-association network <br /> karyotes (34,35) used similar <br /> analysis eukaryotes prokaryotes (36&#226;&#8364;&#8220;38). Further           To predict developmental essentiality human gene, we <br /> details corresponding evolutionary scenario       used mammalian <span id='am-242' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-243' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-251' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-257' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-258' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-259' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-264' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-272' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-277' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-278' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-294' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-297' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-315' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-317' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-318' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-319' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span>phenotype</span> information corre- <br /> orthologous groups presented Supplementary                 sponding mouse orthologs. In analysis, gene is <br /> Figures S14&#226;&#8364;&#8220;S49 provided spreadsheet                  de&#239;&#172;?ned &#226;&#8364;&#732;essential&#226;&#8364;&#8482; organism development a <br /> format (Supplementary Table S3).                                  <span id='am-398' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-399' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-400' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-401' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-402' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-403' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-404' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-405' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-406' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-407' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-408' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-409' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-410' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-411' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-412' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-413' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-414' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-415' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-416' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-417' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-418' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-419' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-420' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-421' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-422' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-423' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-424' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-425' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-426' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-427' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-428' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-429' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-430' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-431' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-432' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-433' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-434' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-435' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-436' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-437' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-438' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-439' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-440' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-441' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-442' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-443' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-444' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-445' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-446' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-447' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-448' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-449' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-450' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-451' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-452' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-453' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-454' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-455' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-456' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-457' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-458' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-459' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-460' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-461' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-462' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-463' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-464' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-465' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-466' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-467' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-468' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-469' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-470' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-471' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-472' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-473' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-474' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-475' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span>knock-out</span> mouse ortholog confers embryonic or <br />   To verify robustness orthology analysis           perinatal lethality (44). We obtained mouse pheno- <br /> compared gene evolutionary scenario corre-            type data curated knock-out collection available <br /> sponding obtained using di&#239;&#172;&#8364;erent data source. In            Mouse Genome Database (MGD) (http://www. <br /> this case, reconstructed entire evolutionary analysis      informatics.jax.org) (45). To predict cellular essentiality <br /> considering Inparanoid database (39). In contrast          human gene, used phenotype information of <br /> KOG algorithm, Inparanoid designed &#239;&#172;?nd orthologs            corresponding yeast orthologs. In analysis, a <br /> and in-paralogs species separate in-           human gene de&#239;&#172;?ned &#226;&#8364;&#732;essential&#226;&#8364;&#8482; cellular level if <br /> paralogs out-paralogs. KOG Inparanoid orthol-            knock-out ortholog confers lethality yeast. <br /> ogy analysis lead roughly conclusions. We             The yeast knock-out data obtained the <br /> present discuss results Supplementary                Saccharomyces SGD project &#226;&#8364;&#732;Saccharomyces Genome <br /> Material Online (Supplementary Figures S3&#226;&#8364;&#8220;S6, S50&#226;&#8364;&#8220;S94             Database&#226;&#8364;&#8482; (http://www.yeastgenome.org/) (46). Human <br /> and Table S4).                                                    yeast orthology veri&#239;&#172;?ed using data source <br />                                                                   Inparanoid database (47) provided in <br />                                                                   Supplementary Table S1. In analysis, essential <br /> Diversity analysis orthologous groups                          genes, 32, listed orthologs using <br />                                                                   Inparanoid (these genes presented Figure 5A with <br /> An orthologous group (OG) corresponds set                 asterisk names). <br /> genes di&#239;&#172;&#8364;erent extant species common <br /> gene ancestor. To obtain quantitative expression of <br /> the orthologous distribution (i.e. distribution            Human <span id='am-86' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-87' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span>gene mutation</span> statistics <br /> items orthologous group), measured                  The data analysis CAN genes obtained from <br /> the information content di&#239;&#172;&#8364;erent databases                 Cancer Gene Census (48). Both germline-mutated and <br /> (STRING Inparanoid) using Shannon Information                 somatic-mutated CAN genes retrieved then <br /> Theory (7,40&#226;&#8364;&#8220;43) de&#239;&#172;?ned follows. Consider n             crossed list 180 genes study. We identi- <br /> number selected OGs, representing ortho-           &#239;&#172;?ed 25 CAN genes placed network-based model of <br /> logous groups. Each OG labeled (a = 1, . . ., n)      apoptosis genome stability (Supplementary Table S1). <br /> has N  items (orthologous genes), distributed M                Genotype statistics germline CAN genes located r <br /> possible organisms. Consequently, given OG           module analyzed XP mutation database <br /> de&#239;&#172;?ne s(i,a) number items given organ-       (http://www.xpmutations.org). The representativeness of <br /> ism i, (i = 1, . . ., M), sum given adds      sample tested second database [Human <br /> N  The probability p(i,a) that, N  items         gene Mutation Database&#226;&#8364;&#8221;HGMD (49)] regarded <br /> a-OG, randomly chosen belongs organism          reference mutation database (published) gene <br /> written                                                        lesions responsible human inherited diseases. Table 1 <br />                                                                   shows equivalent samples obtained from <br />             s&#195;&#176;i;  &#195;&#382;                                               HGMD XP database. However, contains <br /> p&#195;&#176;i;  &#195;&#382; &#194;&#188;           ;                                        2 <br />               N                                                   limited gene information comparing (50). <br />  6272 Nucleic Acids Research, 2008, Vol. 36, No. 19 <br />  <br />  <br /> Table 1. Allelic distribution CAN genes placed r module according XP mutation database (Panel A). Sample representativeness compared to <br /> a second databases (Panel B) <br />  <br /> Panel A             Number Genotypes (%)a                                            Total genotypes          (Panel B) Entriesb <br />  <br /> CAN gene            null/non-null         non-null/non-null          null/null                                   XP database            HGMD <br />  <br /> ERCC2               20   (43.5)            26   (56.5)               0   (0.0)           46                       76c                    48 <br /> ERCC3                3   (60.0)             2   (40.0)               0   (0.0)            5                        8                     11 <br /> ERCC4                0   (0.0)              7   (100.0)              0   (0.0)            7                       18d                    17 <br /> ERCC5                0   (0.0)              5   (100.0)              0   (0.0)            5                       10                     12 <br /> XPA                  6   (6.0)             94   (94.0)               0   (0.0)          100                      128e                    25 <br /> XPC                  0   (0.0)             13   (100.0)              0   (0.0)           13                       28f                    42 <br /> DDB2                 0   (0.0)              5   (100.0)              0   (0.0)            5                        8g                     8 <br />                     29   (16.0)           153   (84.1)               0   (0.0)          182                      276                    163 <br /> a <br />   Data obtained XP mutations database (http://www.xpmutations.org) compiled according absence (null) presence (non-null) of <br /> CAN gene alleles. Null/non-null genotypes heterozygous, non-null/non-null genotypes include heterozygous homozygous. <br /> b <br />   The number allelic records present XP mutations database compared second human inherited mutation database [Human gene <br /> Mutation Database &#226;&#8364;&#8221; HGMD (49)] order attest sample representativeness. <br /> c <br />   One allele duplicated database (the XP1BR entry). <br /> d <br />   Three alleles mutation data (XP80TO, XP81TO XP89TO entries). <br /> e <br />   One allele zygosity information (XP10OS entry). <br /> f <br />  Four alleles zygosity information (XP6BR, XP4BR, XP3BE XP22BE entries). Polymorphisms considered analyses. <br /> g <br />   One allele duplicated (XP25PV entry). <br />  <br />  <br />  <br /> Indeed, successfully retrieve zygosity infor-                  clusters orthologous groups proteins (KOGs) (21), <br /> mation accessing XP database.                                      using orthology assignments STRING server <br />                                                                             (22). In total, apoptosis genome-stability genes are <br />                                                                             distributed 142 KOGs ortho- <br /> RESULTS <br />                                                                             logous groups parsimonious mapping <br /> Apoptosis genome-stability gene set                                     eukaryote phylogeny. In Figure 2A present <br /> Our analysis begins list 180 genes participating               topology species tree used analysis <br /> human apoptosis genome-stability functions pre-                      (28&#226;&#8364;&#8220;33), arranged 17 subdivisions (monophy- <br /> viously de&#239;&#172;?ned (19) provided supplementary mate-                     letic groups) based phylogenetic relationships. Every <br /> rial online (Supplementary Table S1). To de&#239;&#172;?ne gene                    species-tree node (STN) labeled according ascend- <br /> set characterized genome-maintenance                        ing subtree, referred LCA subset. <br /> mechanisms interactions com-                        To quantitative view evolutionary roots <br /> ponents. In Figure 1A reproduce interactions                    inferred 180 human genes studied here, plotted <br /> illustrate links apoptosis genome-stability                 number human apoptotic genome-stability <br /> gene networks, collectively referred                    orthologs STN (Figure 2B). Accordingly, this <br /> genome-maintenance gene network. Each node corre-                           distribution suggests sequential enlargement net- <br /> sponds gene-network node (GNN), lines                        work, progressive increase apoptosis. In con- <br /> represent direct (physical) and/or indirect (functional)                    trast, genome-stability orthologs mainly rooted in <br /> associations according STRING database human.                        STN-P (at base eukaryote species tree), suggesting <br /> They derived high-quality systematic protein&#226;&#8364;&#8220;                      orthologs involved apoptosis recent. <br /> protein interaction mapping (22). Note position                      Furthermore, order assess robustness our <br /> TP53 gene network topology connecting apoptosis                      orthology analysis reconstructed entire evolution- <br /> to 18 genome-stability components (Figure 1A, arrow,                        ary scenarios using Inparanoid database di&#239;&#172;&#8364;erent <br /> and Figure 1B). This functional overlap empha-                   data source, essentially obtained results. In <br /> sized Figure 1C complete network, shows                    contrast KOG algorithm, Inparanoid designed &#239;&#172;?nd <br /> the number links distributed gene set.                          orthologs in-paralogs species to <br /> Although apoptosis genome stability equivalent                     separate in-paralogs out-paralogs (39). We used <br /> number components network (i.e. 86:100),                     second approach construct evolutionary incon- <br /> connectivity 2-fold, indicated                sistence score (R) estimates divergence between <br /> the Venn diagram. Such di&#239;&#172;&#8364;erence arises mainly                   scenarios (i.e.  STN). We present discuss <br /> large number associations NER, MMR                             results Supplementary Material Online (Supple- <br /> chromosome stability components, yielding highly con-                     mentary Figures S3&#226;&#8364;&#8220;S6, S50&#226;&#8364;&#8220;S94 Supplementary <br /> nected gene module (Figure 1A, r).                                          Table S4). Brie&#239;&#172;&#8218;y, apoptosis genes, R = 1.709 STNs <br />                                                                              0.224 (SE) genome-stability genes R = 0.807 <br /> Construction parsimonious evolutionary scenarios                         STNs  0.202 (SE) (Figure 2C). It means each <br /> In order infer ancestral states human apoptosis                   root inferred analyses, estimated error apo- <br /> and genome-stability genes considered eukaryotic                         ptosis approximately STNs the <br />                                                                                                                        Nucleic Acids Research, 2008, Vol. 36, No. 19 6273 <br />  <br />  <br />                                                                                                       TNFRSF10D <br />  A                                                                                                                                        B <br />                                                                                                               TNFRSF10C                                            CHUK <br />                                                   DCAMKL1 <br />                                                                                                                                                                            HMGB1 <br />                 PIK3CG                                                     TNFRSF10A <br />  CAPN2   NGFB                                                     PAK7                                                                                                               BRCA2 <br />                                                                                FAS                                                                                   ATR <br />                                                                                            TNFSF10                                         NFKBIA                           XRCC6 <br />                              CAST                                                                                                                                       ATM <br />                                                                    FASLG                                                                                  IKBKB <br />                                        IRAK4 <br />                                                IRAK2    MYD88 <br />                                                             TRADD                                                                         BCL2L1                        PARP1 BRCA1 PRKDC    APEX1 <br />                          NTRK1                                           RIPK1                 DFFB <br />             CAPNS1                                         IRAK1CASP10       BIRC4 TNFRSF10B                 SHFM1                                 BCL2                          CHEK2 <br />                    PIK3R5         PIK3CB <br />                                             PIK3C3 <br />                                                    PIK3R1 <br />                                                          TNF            BIRC2 <br />                                                                                                                     PNKP <br />                                                                                                                                                                         TP53 <br />                                      IL1RAP                                                                                                                 CHEK1                    RPA1 RAD51 <br />                                                              IL1A CASP8        FADD CASP6                              PMS2 <br />                  PIK3R3        CAPN1                                  BIRC3                                                                         BAX <br />                                         PIK3CA           IKBKG                                                              LIG3 <br />                                                MAP3K14                                                  LIG4 <br />                                                                                                                                  PMS2L3 <br />                                                                                                                                                                   ERCC6 <br />                               PIK3C2A                  TRAF2 NFKB2NFKB1 CHUK             PARP2 RAD1         DFFA <br />                     PIK3CD                   IL1B                                 CASP7             MBD4              DCLRE1C <br />                                                              IL1R1                         HMGB1               BRAP                                                     MNAT1 ERCC3 <br />                                PIK3R4                                   CFLAR <br />                                                                                   CASP3                                       OGG1 EME1 <br />                                             AKT1 AKT2 BID                                             RAD50 <br />                                                                                                                      XRCC4                                         GTF2H1 <br />                     PIK3R2                                      TNFRSF1A                    NBN                                                                                ERCC2 <br />                                                  AKT3                      CASP9                     BRCA2                         MLH3 <br />                                                            APAF1                  ATR     XRCC6            XRCC1           EXO1 <br />                  PIK3C2G      PIK3C2B                                RELA <br />                                             IL3RA                                  ATM            PRKDC          MLH1                                                     CCNH <br />                  CSF2RB                               NFKBIA   IKBKB                       BRCA1         MSH2                 RAD54L <br />                           PRKAR2B                                                                                  TOPBP1 <br />                                                                BCL2                            RAD17                              SMUG1 <br />                                                       BCL2L1                CYCS PARP1               LIG1 XRCC5 APEX1 MUS81 <br />                            PRKAR1B <br />                                        IL3 BAD                                        MRE11A               MSH6                    MPG    C <br />                  PRKACB                                            CHEK1                          CHEK2          RAD51 <br />                             PRKACA                                       PPP3CA                                         MSH3 <br />                                                               BAX                                          UNG                MSH4 <br />                     PRKACG                                                                   RFC1                         FEN1 RAD54B <br />                               PRKAR2A <br />                                                                                                 RPA2 RPA1 POLE                                                          Genomestability <br />                                                                    PDCD8                RFC3                      MUTYH ORC1L <br />                                                                                                    PCNA                           TDG         Apoptosis <br />                                            PPP3CC                                                           POLD3 <br />                                                             PPP3R1              ERCC6 RFC5                                 APEX2 <br />                                                     PPP3CB           TP53                    RFC4 RPA3 <br />                                                                                                         POLD2 <br />                                                                                                                 POLD4 <br />                                                                                                                        NTHL1 <br />  <br />                                                                                                  &#207;?               RAD52 DMC1 MSH5 <br />                                                                                 ERCC8 <br />                                                                                       MNAT1ERCC3          POLD1 <br />                 Apoptosis                                                                                     CDC6 <br />                                                                                   GTF2H1             ERCC5            RAD23B <br />                                                                                           CCNHERCC2      XPA <br />                 Chromosome stability                                                                              XPC <br />                                                                                     GTF2H3                  XAB2 <br />                                                                                        CDK7      GTF2H2          DDB2 <br />                 Recombinational repair                                                     ERCC4 <br />                                                         Genome                                      GTF2H4 DDB1                                    355            112             708 <br />                 Mismatch repair                                                               ERCC1                               CETN2 <br />                                                          stability <br />                 Base-excision repair <br />                                                                                  GTF2H5 <br />                 Nucleotide-excision repair                                                                                 RAD23A <br />  <br />  <br />  <br />  <br /> Figure 1. Human apoptosis genome-stability gene network. (A) Graph interactions genes involved apoptosis DNA repair <br /> pathways, previously characterized Castro et al. (19). Di&#239;&#172;&#8364;erent pathways represented di&#239;&#172;&#8364;erent colors. Network nodes one <br /> color represent genes participating pathway. Gene IDs pathway provided Supplementary Table S1. (B) Magni&#239;&#172;?cation <br /> of TP53 gene position network topology. It highlights functional overlap TP53, linking apoptosis genome-stability <br /> components. (C) Venn diagram showing distribution links apoptosis genome-stability pathways. The overlapped area corresponds <br /> to links connecting systems. The large number associations NER, MMR chromosome-stability components designed as <br /> r module. <br />  <br />  <br /> rooting point species tree, genome stability                                              nodes according roots inferred species tree <br /> the error approximately STN down.                                                            (Figure 3D). Note gene-network node <br />    In order test phylogeny Caenorhabditis ele-                                                  (GNN) represents ortholog gene human <br /> gans root metazoa included                                                         apoptosis genome-stability gene network. <br /> Nematostella vectensis, changes base                                                        The orthology information regarding STNs is <br /> metazoa (Supplementary Figure S9). We chose organ-                                                    provided Supplementary Table S1. As quantitatively <br /> ism (i) Nematostella cnidarian; (ii) idea                                                 showed Figures 2B&#226;&#8364;&#8220;D, recent STNs concen- <br /> that cnidarians base metazoa con-                                                trate apoptosis roots (round GNNs Figure 3A B). <br /> troversial nematodes; (iii) switching taxon                                                 However, qualitative di&#239;&#172;&#8364;erence: observe the <br /> like goes way testing e&#239;&#172;&#8364;ect phylo-                                                 pooled origins inferred components apo- <br /> geny used. The result process roots                                              ptosis extrinsic (Figure 3A) intrinsic (Figure 3B) <br /> the human genes remain (the complete                                                       pathways. <br /> analysis available Supplementary Table S5)                                                          To analyze result important consider the <br /> further discussed supplements (section 1.4: deep                                                    biochemical signature apoptosis, is, caspase <br /> root metazoans).                                                                                        activation, triggered intrinsic or <br />                                                                                                            extrinsic apoptosis pathways. The intrinsic pathway is <br /> From species-tree nodes gene-network nodes                                                              associated mitochondrial outer membrane permeabi- <br /> To assess details evolutionary scenario described                                               lization cytochrome c (CYCS) release response pri- <br /> earlier context known predicted gene func-                                                   marily developmental cues cellular damage. It <br /> tions, used network-based model presented                                                        triggers apoptosis Bcl-2 gene family the <br /> Figure 1A (19).                                                                                            initiator protease caspase-9. In contrast, extrinsic <br />    Starting complete network graph generated                                                   pathway characterized ligation cell surface <br /> three relevant orthology projections characterize                                                   receptors speci&#239;&#172;?c death ligands, TNF gene prod- <br /> functional di&#239;&#172;&#8364;erences apoptosis genome sta-                                                    uct, generate catalytically active caspase-8 (51,52). The <br /> bility (Figure 3A&#226;&#8364;&#8220;C). In graphs highlighted                                                   protein encoded TNF gene multifunctional <br />  6274 Nucleic Acids Research, 2008, Vol. 36, No. 19 <br />  <br />  <br />                                       A                                                                          A     a1 Homo sapiens <br />                                                                                                              B         a2 Pan troglodytes <br />                                                 STN          LCA                                         C             b Macaca mulatta <br />                                                                                                         D <br />                                                 B            Primate                                E                  c1 Rattus norvegicus <br />                                                                                              F <br />                                                 F            Mammalia                       G                          c2 Mus musculus <br />                                                                                            H <br />                                                 I            Vertebrata                                                d Canis familiaris <br />                                                 J            Chordata                  I                               e   Bos taurus <br />                                                                                    J <br />                                                 L            Metazoa                                                   f   Monodelphis domestica <br />                                                 M            Opisthokonta LK                                           g Gallus gallus <br />                                                 P-Q          Eukarya                                                   h Xenopus tropicalis <br />                                                                                                                        i1 Takifugu rubripes <br />                                                                                                                        i2 Tetraodon nigroviridis <br />                                                                                                                        i3 Danio rerio <br />                                                               M                                                        j   Ciona intestinalis <br />                                                              N                                                         k1 Drosophila melanogaster <br />                                                         O                                                              k2 Anopheles gambiae <br />                                                   P                                                                    k3 Apis mellifera <br />                                                                                                                        l   Caenorhabditis elegans <br />                                                 Q                                                                      m1 Kluyveromyces lactis <br />                                                                                                                        m2 Saccharomyces cerevisiae <br />                                                                                                                        m3 Candida glabrata <br />                                                                                                                        m4 Eremothecium gossypii <br />                                                                                                                        m5 Debaryomyces hansenii <br />                                                                                                                        m6 Yarrowia lipolytica <br />                                                                                                                        m7 Aspergillus fumigatus <br />                                                                                                                        m8 Schizosaccharomyces pombe <br />                                                                                                                        m9 Filobasidiella neoformans <br />                                                                                                                        m10 Encephalitozoon cuniculi <br />                                                                                                                        n Dictyostelium discoideum <br />                                                                                                                        o1 Arabidopsis thaliana <br />                                                                                                                        o2 Cyanidioschyzon merolae <br />                                                                                                                        p1 Plasmodium falciparum <br />                                                                                                                        p2 Cryptosporidium hominis <br />                                                                                                                        p3 Thalassiosira pseudonana <br />                                                                                                                        q Giardia lamblia <br />  <br />                                        B 125                                                                                    C <br />                                                                                                                                 inconsistency score (R) <br />                                                                                                                                                           2.0 <br />                                                               A-D E-G        H,I           J KL <br />                                                        100                                               MN                                               1.6 <br />                                        Orthologs (n) <br />  <br />  <br />  <br />  <br />                                                                                                                                     Evolutionary <br />                                                                                                                   O <br />                                                                                                                    P <br />                                                         75 A-D E-G          H,I                                                                           1.2 <br />                                                                                            JK <br />                                                         50                                      L                        Q                                0.8 <br />                                                                     Apoptosis                            MN <br />                                                         25                                                 OP                                             0.4 <br />                                                                Genome stability                                          Q <br />                                                          0                                                                                                0.0 <br />                                                                                                                                                                          b. <br />                                                                                                                                                                 is <br />  <br />  <br />  <br />                                                          0.00      0.05    0.10    0.15   0.20   0.25 <br />                                                                                                                                                                os <br />  <br />                                                                                                                                                                         ta <br />                                                                                                                                                               pt <br />  <br />                                                                                                                                                                     .S <br />  <br />  <br />  <br />  <br />                                                                 Evolutionary distance STNs <br />                                                                                                                                                                    en <br />                                                                                                                                                           po <br />                                                                                                                                                                 G <br />                                                                                                                                                           A <br />  <br />  <br />  <br />  <br /> Figure 2. Inferring evolutionary roots human apoptosis genome-stability genes. (A) Eukaryote species tree topology used parsimony <br /> analysis. The phylogenic relationship 35 eukaryotes based manual integration variety phylogenies (28&#226;&#8364;&#8220;33). STNs the <br /> corresponding LCA indicated. (B) Distribution apoptotic genome-stability orthologs according roots inferred species tree <br /> and plotted function divergence STNs (based branch-<span id='am-6' about='xsp:length' typeof='owl:Thing'>length</span> estimates). In Supplementary Material Online exempli&#239;&#172;?ed the <br /> parsimony analysis. The evolutionary distances computed using protein families regarded conserved distant taxa and <br /> described able reconstruct three-domain phylogeny: 40S ribosomal proteins, translation initiation factor 5A proteins Flap structure- <br /> speci&#239;&#172;?c endonuclease 1 proteins (73). All proteins used analysis aligned Supplementary Figures S10&#226;&#8364;&#8220;S12. The distances expressed as <br /> the fraction sites di&#239;&#172;&#8364;er branches multiple alignment, approximation branch-length separates STNs. <br /> (C) Divergence KOG Inparanoid-derived scenarios. For apoptosis genes, R = 1.709 STNs  0.224 (SE) genome-stability genes <br /> R = 0.807 STNs  0.202 (SE). It means root inferred analyses, estimated error apoptosis approximately STNs up <br /> and rooting point species tree, genome stability error approximately STN down. <br />  <br />  <br />  <br /> proin&#239;&#172;&#8218;ammatory cytokine belongs tumor                                                                extrinsic pathway, ligands receptors are <br /> necrosis factor (TNF) superfamily, includes                                                           rooted STN-I projection, later (e.g. IL1A, IL3RA, <br /> the ligands FAS (FASLG) TRAIL (TNFSF10).                                                                     IL3 TNFRSF10D genes observed mam- <br /> These ligands bind members TNF-                                                                    mals, is, STN-F later, evinced comparing <br /> receptor superfamily (e.g. TNFRSF1A, TNFRSF10A,                                                                  STN-I projection versus complete human network; details <br /> TNFRSF10B FAS receptors) involved                                                             orthologs presented explicit parsimony <br /> regulation wide spectrum biological processes,                                                      analysis&#226;&#8364;&#8221;Supplementary Figures S46, S48 S49). <br /> as immune surveillance, innate immunity, haematopoiesis                                                             In STN-P projection (Figure 3C), however, small <br /> and tumor regression [for review, (53)].                                                                     fraction genes belongs apoptosis. Instead, graph <br />    Accordingly, noticeable components                                                          remarkable large presence genome-stability <br /> intrinsic pathway rooted mainly STN-L earlier                                                          components (triangular GNNs), quantitatively <br /> (e.g. CYCS deeply rooted eukaryote species tree&#226;&#8364;&#8221;                                                           addressed Figure 2B. <br /> Figure 3C). In contrast, subsequent enlargement                                                          Taking results together, evolutionary scenario of <br /> network graph provided mainly orthologs                                                             genome-maintenance mechanisms marked three <br />                                                                                                           Nucleic Acids Research, 2008, Vol. 36, No. 19 6275 <br />  <br />  <br />   A                                                                                                     B <br />                                                    TNFRSF10A <br />       NGFB                                           FAS <br />                                                                TNFSF10 <br />                                            FASLG <br />                                         TRADD                                                                  NTRK1 <br />                                                       TNFRSF10B                                                                    PIK3R1CASP10                          SHFM1 <br />                                    TNF <br />                                                                                                         PIK3R3       IL1RAP               CASP8 <br />                                                      FADD                                                                                                  CASP6 <br />                                                                                                                                                    CHUK    CASP7 <br />                                                                          DFFA <br />                            IL1B       IL1R1                                                                                                       CFLAR <br />                                           TNFRSF1A <br />                                                                                 XRCC4                       PIK3R2 <br />                                                                                                                                                   CASP9 CASP3 <br />                                                                                                                                              IKBKB <br />                                                                                                                                     BCL2L1 <br />  <br />  <br />                             BAD                                                             MPG                                          BCL2 <br />                                                      TP53 <br />  <br />  <br />  <br />  <br />                                                                  &#207;?                                                                                                 &#207;? <br />                   GS APO                                                                                      GS APO <br />                            GNNs rooted STN-I                                                                          GNNs rooted STN-L <br />                            GNNs present STN-I                                                                         GNNs present STN-L <br />  <br />  <br />  <br />  <br />   C                                                                                                     D <br />   C APN2 <br />                                                                                                                                  Species-treenodes <br />                                                                                                                                  (STNs) <br />                                                                                                                                                                    a1 <br />                                                                                                                                                                    a2 <br />                                                                                                                                                                    b <br />                                                                                                                                                                    c1 <br />                                                                                    PNKP                                                     STN-I                  c2 <br />                      C APN1                                                                                                                 (Vertebrata)           d <br />                                                                          LIG4                                                          STN-L                       e <br />                                                                                                                                                                    f <br />                                                              HMGB1                 DCLRE1C                                             (Metazoa)                   g <br />                                                                             BRAP         OGG1 <br />                             AKT1 AKT2                                                                                                                              h <br />                                                                         BRCA2                                                                                      i1 <br />                                                                XRCC6                          MLH3                                                                 i2 <br />                                AKT3                                                                                                                                i3 <br />                                                                                           RAD54L                                                                   j <br />                                                     CYCS            RAD17                                                                                          k1 <br />                                                                                         MUS81                                                                      k2 <br />                                                                  CHEK2                                                   STN-P                                     k3 <br />                                                                              MSH3                                                                                  l <br />                                                                       UNG       MSH4 <br />                                                                                     RAD54B                             (Eukarya)                                   m1 <br />                                                                  RPA2    MUTYH                                                                                     m2 <br />                                                                     RPA1        ORC1L                                                                              m3 <br />  <br />                                                                     &#207;? <br />                                                                                                                                                                    m4 <br />                                                                         POLD2                 MSH5                                                                 m5 <br />                                                 ERCC8       MNAT1                                                                                                  m6 <br />                                                                     ERCC5                                                                                          m7 <br />                   GS APO                           GTF2H1    CCNH                    RAD23B                                                                        m8 <br />                                                                          XAB2                                                                                      m9 <br />                            GNNs rooted STN-P                            XPC                                                                                     m10 <br />                                                     GTF2H3                DDB2                                                                                     n <br />                            GNNs present STN-P                        DDB1                                                                                       o1 <br />                                                                                                                                                                    o2 <br />                                                             ERCC1 GTF2H4                                                                                           p1 <br />                                                                                           RAD23A                                                                   p2 <br />                                                                                                                                                                    p3 <br />                                                                                                                                                                    q <br />  <br />  <br /> Figure 3. From STNs gene-network nodes (GNNs). Orthology projection genes rooted STN-I (A), STN-L (B) STN-P(C). Roots an <br /> ortholog: color nodes; presence ortholog: white nodes. The location STNs species tree indicated (D). In Supplementary <br /> Material Online provide examples orthology projection approach (Supplementary Figure S2) compared Inparanoid <br /> evolutionary scenarios di&#239;&#172;&#8364;erent orthology data source (Supplementary Figures S3&#226;&#8364;&#8220;S6). <br />  <br />  <br />  <br /> major functional increments: &#239;&#172;?rst evolution                                          GNNs placed periphery networks <br /> genome-stability gene network, components origi-                                             represent recent evolutionary innovations. There- <br /> nate basal position species tree [STN-P,                                            fore, striking feature graphs increasing <br /> inconsistency datasets R = 0.63 STNs  0.22                                                 association apoptosis genome-stability func- <br /> (SE)]; second appearance apoptotic                                           tions emergence entangled gene network, <br /> intrinsic components, rooted near metazoan divergence                                              fully consistent evolutionary strategy <br /> [STN-L, inconsistency datasets R = 1.23 STNs                                               used eukarya adding complexity existing core <br />  0.23 (SE)]; consists network enrichment                                          systems (54,55). (Inparanoid database essentially produces <br /> with apoptotic extrinsic members happens                                               evolutionary scenario; Supplementary <br /> near chordate-vertebrata root [STN-I, inconsistency                                                Figure S6.) <br /> between datasets R = 0.35 STNs  0.16 (SE)]. The net-                                                  Also, additional evidence ancestral roots of <br /> work core apoptosis genome-stability systems                                            genome stability inferred considering likely <br /> rooted tree divergence metazoans,                                            origin ancestral eukaryotic KOGs identifying <br />  6276 Nucleic Acids Research, 2008, Vol. 36, No. 19 <br />  <br />  <br /> their closest prokaryotic orthologous groups (COGs).           abundance (Figure 4C). The &#239;&#172;?rst class (a) refers genes <br /> The KOG-to-COG correspondence presented                  placed orthologous groups low diversity low <br /> Supplementary Figure S8, shows 77.0%           abundance (Figure 4A B, white GNNs; Figure 4C, <br /> genome-stability orthologs identi&#239;&#172;?able prokaryotic        white diamonds). It means organisms present <br /> orthologous groups, 39.5% apoptotic ortholo-       orthologs, associated orthologous groups <br /> gous genes.                                                    components. This implies recent origin <br />    Despite organisms consid-        GNNs, (i) present humans, the <br /> ered, construction gene network directed      end species tree; (ii) present in <br /> human. Therefore, interpretation evolutionary       extant species. For example, TP53 FAS have <br /> scenarios ultimately linked characterization   origins STN-I, shown Figures S38 S40 in <br /> human gene network. It means infer         Supplementary Material Online. This class genes must <br /> the gene network actual organism root     located region network plastic <br /> eukaryotes smaller. As stated introduc-     accept new genes. The second (b) refers to <br /> tion section, goal create orthology map    genes placed orthologous groups high diversity <br /> the species tree order transfer human informa-    low abundance (Figure 4A B, black GNNs; <br /> tion described eukaryotes. This one-way         Figure 4C, black diamonds), indicating small number <br /> strategy, explored subsequent sections.        genes organism, present di&#239;&#172;&#8364;erent spe- <br />                                                                cies. These genes located ancient region of <br /> Plasticity analysis                                            network. It implies poorly plastic genes, highly con- <br />                                                                served species. The class (c) refer those <br /> Genetic plasticity understood ability       genes placed orthologous groups high diversity <br /> functional gene network tolerate changes compo-      high abundance (Figure 4A B, red GNNs; <br /> nents. There di&#239;&#172;&#8364;erent sources changes (gene       Figure 4C, red diamonds), clearly requires high <br /> duplication, gene loss, mutations horizontal gene          plasticity. Note red white GNNs (plastic <br /> transfers), di&#239;&#172;&#8364;erent causes e&#239;&#172;&#8364;ects. These changes     GNNs) segregated black GNNs (poorly plas- <br /> in genome naturally selected,            tic) network. This segregation expected <br /> depending e&#239;&#172;&#8364;ect cell &#239;&#172;?tness      plasticity characteristic set inter- <br /> organism viability, case complex organisms. The      acting genes characteristic individual <br /> result evolutionary dynamics genetic varia-      gene. Figure 4D supports &#239;&#172;?nding showing the <br /> bility organisms species or, ultimately,     relative presence classes genes the <br /> speciation. Gene networks equally plastic          STNs: recent genes network emerge at <br /> hence equally respond variation pressures:     highly plastic regions network, more <br /> depending gene, function, in&#239;&#172;&#8218;uence         ancient ones located poorly plastic regions. <br /> genes, relevance, changes likely          Observe inhomogeneous distribution white, <br /> to tolerated selected others.                       red black GNNs network graph re&#239;&#172;&#8218;ects in <br />    Focusing networks general, expect        function performed genes. While white red <br /> gene networks tolerant variation         GNNs clearly populating apoptosis network, black <br /> present larger variability inside species        GNNs placed mainly genome stability. This result <br /> species. Focusing individual genes, organisms           suggests high evolutionary conservation genome- <br /> should tolerant drastic changes (e.g. gene          stability orthologs (i.e. class b, orthologs present many <br /> knock-out) change performed genes located       organisms variants), contrasting apo- <br /> at plastic network. These characteristics,      ptosis GNNs concentrate plasticity network <br /> gene variability di&#239;&#172;&#8364;erent genomes organ-         (i.e. class c orthologs variants organisms). <br /> ism response knock-out single genes, allow inde- <br /> pendent measures estimate gene network plasticity. One      Essentiality Saccharomyces cerevisiae. A second, inde- <br /> possible plasticity measure estimating number       pendent plasticity measure obtained assessing cell <br /> the distribution orthologs di&#239;&#172;&#8364;erent organisms.        lethality data. Here considered eukaryotic model <br /> A second, independent plasticity measure obtained       Saccharomyces cerevisiae available Saccharomyces <br /> by assessing cell lethality data. In follows present   Genome Database (SGD) (46). We transferred infor- <br /> and discuss plasticity measures.                     mation STN representing LCA yeast and <br />                                                                human (i.e. STN-M), projected cor- <br /> Diversity abundance analysis. We evaluated diver-      responding human network topology. The yeast results <br /> sity abundance orthologous groups estimate       showed Figure 5A. Observe essential genes <br /> the plasticity gene human apoptosis         concentrated speci&#239;&#172;?c portion network <br /> genome-stability gene network (precise de&#239;&#172;?nition        (blue GNNs) corresponding lower plasticity area <br /> Materials methods section exempli&#239;&#172;?ed           showed Figure 4 (black GNNs there). Furthermore, <br /> in Supplementary Material Online).                             likely orthology inferred LCA yeast human <br />    The network graph presented Figure 4A B              indicates yeast lost genes course <br /> incorporates diversity abundance statistics, allow-        evolution, mainly apoptotic genes (white GNNs <br /> ing discrimination distinct classes genes      Figure 5A). Such loss, presence <br /> based distribution diversity function        essential genes overlaid genome-stability area <br />                                                                                                                                          Nucleic Acids Research, 2008, Vol. 36, No. 19 6277 <br />  <br />  <br />  <br />                                                                                                                                                                        TNFRSF10D <br />                          A                              Apoptosis <br />                                                                                                                                                                                 TNFRSF10C <br />                                                                                                                DCAMKL1 <br />  <br />                                             PIK3CG                                                                                     TNFRSF10A <br />                      CAPN2           NGFB                                                                                    PAK7 <br />                                                                                                                                            FAS <br />                                                              CAST                                                                                        TNFSF10 <br />                                                                                                                              FASLG <br />                                                                                                IRAK4 <br />                                                                                                        IRAK2         MYD88 <br />                                                                                                                          TRADD <br />                                                         NTRK1                                                                       RIPK1                    DFFB <br />                                          CAPNS1 <br />                                                                                                IRAK1 CASP10        BIRC4 TNFRSF10B                    SHFM1 <br />                                               PIK3R5                PIK3CB            PIK3R1                BIRC2 <br />                                                                               PIK3C3         TNF                                                            PNKP <br />                                                                        IL1RAP                         IL1A                                                                    B <br />                                                                                                        CASP8          FADD                                     PMS2 <br />                                               PIK3R3                                            IKBKG                         CASP6 <br />                                                                  CAPN1 <br />                                                                             PIK3CA                           BIRC3                                                  LIG3 <br />                                                                                    MAP3K14                                  CASP7              LIG4 <br />                                                                PIK3C2A                                NFKB2       CHUK           PARP2 RAD1        DFFA                   PMS2L3 <br />                                                                                           TRAF2 <br />                                                                                IL1B                       NFKB1                            MBD4 <br />                                                  PIK3CD                                           IL1R1                                                       DCLRE1C <br />                                                                                                                                   HMGB1                BRAP <br />                                                                 PIK3R4                               TNFRSF1A CFLAR CASP3                                             OGG1 EME1 <br />                                                                                 AKT1 AKT2 BID                     CASP9               NBN RAD50 <br />                                                      PIK3R2                                     APAF1                                                        XRCC4 <br />                                                                                     AKT3                                                     BRCA2                          MLH3 <br />                                                                                          NFKBIA                         ATR        XRCC6          XRCC1           EXO1 <br />                                                  PIK3C2G       PIK3C2B                                     RELA <br />                                                                               IL3RA                                                        PRKDC        MLH1 <br />                                                                                                                           ATM <br />                                                  CSF2RB                                             IKBKB                          BRCA1        MSH2                     RAD54L <br />                                                           PRKAR2B                          BCL2L1                                                         TOPBP1 <br />                                                                                                    BCL2                                RAD17                                 SMUG1 <br />                                                                                                                    CYCS PARP1                LIG1XRCC5 <br />                                                             PRKAR1B                                                                                       APEX1 MUS81 <br />                                                                          IL3 BAD                                              MRE11A              MSH6                       MPG <br />                                                  PRKACB                                                 CHEK1                                           RAD51 <br />                                                              PRKACA                                                                       CHEK2                 MSH3 <br />                                                                                                               PPP3CA <br />                                                                                                                         TP53                                           MSH4 <br />                                                      PRKACG                                       BAX                               RFC1          UNG <br />                                                                PRKAR2A                                                                                           FEN1       RAD54B <br />                                                                                                                                         RPA2 RPA1 <br />                                                                                                                                 RFC3               POLE            ORC1L <br />                                                                                                        PDCD8                                               MUTYH <br />                                                                                                                                            PCNA                            TDG <br />                                                                               PPP3CC                                   ERCC6 RFC5                  POLD3             APEX2 <br />                                                                                                  PPP3R1                             RFC4 <br />                                                                                        PPP3CB                                            RPA3 POLD2 POLD4 NTHL1 <br />                                                                                                                       ERCC8                                                  MSH5 <br />                                                                                                                                                                &#207;? <br />                                                                                                                              MNAT1 ERCC3                 RAD52 DMC1 <br />                                                                                                                                                  POLD1 <br />  <br />                                                                                                                                              GTF2H1              ERCC5 CDC6 RAD23B <br />                                                                                                                                                        CCNH ERCC2   XPA <br />  C                                                               D                                                                                                          XPC <br />                                                                                                                                                GTF2H3                 XAB2 <br />                    1.2                                                                                                                            CDK7       GTF2H2 <br />                              (b)                                                                                                                                           DDB2 <br />                                                                  Orthologs (fraction) <br />  <br />  <br />  <br />  <br />                                                                                         1.00 <br />                                                                                                                                                       ERCC4 <br />   Diversity (H&#206;&#177;) <br />  <br />  <br />  <br />  <br />                                             (c)                                                                                                                GTF2H4 DDB1 <br />                    0.8                                                                  0.75                                                             ERCC1                                  CETN2 <br />                                                                                         0.50 <br />                    0.4 (a)                                                                                                                    GTF2H5 <br />                                                                                         0.25                                                                                                RAD23A <br />  <br />                    0.0                                                                  0.00 <br />                                                                                                A B C D E F G H I J K L MN O P Q                                    Genome stability <br />                       1.0               10.0            100.0 <br />                                    Abundance (D&#206;&#177;)                                                        Species-treenodes <br />  <br />  <br /> Figure 4. Plasticity analysis orthologous groups. (A B) Diversity H  abundance D  orthologous groups overlaid apoptosis and <br /> genome-stability gene network according categories de&#239;&#172;?ned C. (C) Distribution H  function D : (a) orthologous groups low <br /> diversity low abundance (white); (b) orthologous groups high diversity low abundance (black); (c) orthologous groups high <br /> diversity high abundance (red). (D) Fraction orthologous groups present STNs: orthologous groups low diversity low <br /> abundance (white dashed line); orthologous groups high diversity low abundance (black solid line); orthologous groups high <br /> diversity high abundance (red solid line). In Supplementary Material Online provide examples diversity analysis. <br />  <br />  <br />  <br /> (blue triangular GNNs), indicates evolutionary                                                                       day 2; thus, knock-out alleles indicate &#226;&#8364;&#732;developmental <br /> scenario consistent plasticity measures shown                                                                     lethality&#226;&#8364;&#8482; &#226;&#8364;&#732;essentiality&#226;&#8364;&#8482; embryonic stem cells. Evidence <br /> in Figure 4: lost genes represented plastic GNNs                                                                   mouse lethality obtained according frequency <br /> (red white symbols Figure 4).                                                                                          expected Mendelian genetics (i.e. zygosity allelic <br />                                                                                                                               distribution observed o&#239;&#172;&#8364;spring): signi&#239;&#172;?cant <br /> Lethality Mus musculus. In order complement                                                                        deviation expected frequency knock-out <br /> lethality measure complex multicellular eukaryotic                                                                     allele indicates lethality. Therefore, putative 178 <br /> model, assessed Mus musculus lethality data Mouse                                                                       Mus musculus orthologs identi&#239;&#172;?ed analysis, &#239;&#172;?nd <br /> Genome Database&#226;&#8364;&#8221;MGD (45). The phenotypic statistics                                                                           124 genes knock-out data available (Supple- <br /> in MGD database consider lethal allele causes                                                                        mentary Table S1). While majority produced viable <br /> death anytime fertilization postnatal                                                                    phenotypes, 39 knock-out alleles associated <br />  6278 Nucleic Acids Research, 2008, Vol. 36, No. 19 <br />  <br />  <br /> A                                                                                                                                                             B                 CAPN2 <br />                                CAST* <br />  <br />  <br />                                                                                                                                                                                            CAPNS1 <br />                                                                                                                                                                                                                    PIK3CB <br />                                                                     IKBKG*                                                                                                                                                                              CASP8FADD <br />                                                                                                                                                                                                                        PIK3CA                                              LIG4      LIG3 <br />                                                     MAP3K14* <br /> Projection functional                                                                                                                                      Projection functional                                                  TRAF2              CHUK CASP7 <br /> data LCA human                                                                                                                                          data LCA human                                                                                HMGB1 <br />                                                                                                                                                                                                                                                         CFLAR         NBN RAD50 <br /> and yeast                                                                                                                                                     mouse                                                                                                           XRCC4 <br />                                                                                              ATR                                                                          STN-C                                                                           CASP9 CASP3 <br />                                                                                                                                                                                                                                                                            BRCA2 <br />                                                                                                         PRKDC*                                                                                                                                    APAF1         ATR           XRCC1 <br />                                                                                                                                                                                                                                                     IKBKB <br />                                                                                                                              TOPBP1*                                                     Mouse <br />                                                                                                            LIG1                                                                          data                                                                       CYCS   BRCA1 <br />     STN-M                                                                                                                                                                                input                                      BCL2L1 <br />                                                                                                                                                                                                                                                                               RAD17 <br />                                                                                                       CHEK2                                                                                          PRKACA                                                 CHEK1 <br />                                                                                                   RFC1                                                                                                                                                                                  RAD51 <br />                       Yeast                                                                           RPA2 RPA1 <br />                                                                                                RFC3            POLE                                                                                                                                                                          FEN1 <br />                       data                                                                                                           ORC1L <br />                       input                                                                   RFC5 RFC4 PCNA                                                                                                                                      PPP3R1 <br />                                                                                                             POLD2 <br />                                                                                                  ERCC3       POLD1                                                                                                                                                     MNAT1 <br />                                                                                            MNAT1 <br />                                                                                                                CDC6 <br />                            GS APO                                                       GTF2H1 CCNH ERCC2                                                                                   GS APO                                                                                    XPA RAD23B <br />                                                                                                               XAB2                                                                                                                                                                      XAB2 <br />                                                                      Essentiality <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                          Essentiality <br />                                                                                         GTF2H3 <br />                                                 +               +                   +                                                                                                                              +                +                   + <br />                                                     Knock-out <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                         Knock-out <br />                                                                                                    GTF2H2                                                                                                                                                                  ERCC2 <br />                                     Orthology <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                        Orthology <br />                                                 +               +                   &#226;&#710;&#8217;       CDK7                                      CETN2*                                                                       +                +                   &#226;&#710;&#8217; <br />                                                                                                         GTF2H4                                                                                                                                                                        DDB1 <br />                                                 &#226;&#710;&#8217;               &#226;&#710;&#8217;                   &#226;&#710;&#8217;                                                                                                                              +                &#226;&#710;&#8217;                   ?                    ERCC1 <br />  <br />  <br />  <br />  <br />                                                                           C <br />                                                                                                                                                                           FAS <br />  <br />  <br />  <br />                                                                                                              NTRK1 <br />  <br />                                                                                                                                                                   BIRC3 <br />                                                                                                                                                                                                       PMS2 <br />                                                                                                                                      PIK3CA <br />                                                                           Projection human                                                                 NFKB2 <br />                                                                           functional data                                                                                           NBN <br />                                                                                                                                                       AKT2 <br />                                                                                                 Human                                                                                      BRCA2 <br />                                                                                                 data <br />                                                                                                 input                                                                       ATM                 MLH1 <br />                                                                                                                                                                                    BRCA1    MSH2 <br />                                                                                                                                                              BCL2 <br />  <br />                                                                                                                                                                                         CHEK2 MSH6 <br />                                                                                                                                                                            TP53 <br />  <br />                                                                                                                                                                                                      MUTYH <br />  <br />  <br />                                                                                                                                                                                   ERCC3 <br />                                                                                                         GS APO <br />                                                                                                                                                                                   ERCC2     ERCC5 <br />                                                                                                                                  +                    -                                              XPC <br />                                                                                                                                       Germline mut. <br />                                                                                                                   Somatic mut. <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                            XPA <br />                                                                                                                                  -                    +                           ERCC4             DDB2 <br />  <br />                                                                                                                                  +                    + <br />                                                                                                                                  ?                    ? <br /> Figure 5. Integrating evolutionary functional data. (A) Projection yeast lethality data human apoptosis genome-stability gene <br /> network: essential (blue GNNs) nonessential yeast orthologs (grey GNNs) according SGD database (46). The graph presents orthologs <br /> inferred LCA yeast human (i.e. rooted present STN-M). White GNNs correspond genes present branch absent in <br /> yeast, predicted parsimony analysis (see &#226;&#8364;&#732;Materials Methods&#226;&#8364;&#8482; section). Asterisks identify GNNs orthology predicted by <br /> orthologous groups con&#239;&#172;?rmed Inparanoid database (47). (B) Projection mouse lethality data human apoptosis genome- <br /> stability gene network: essential (red GNNs) nonessential (grey GNNs) mouse orthologs according MGD database (45). The graph presents <br /> only GNNs orthologs inferred LCA mouse human (i.e. rooted present STN-C). GNNs lack knock-out data in <br /> MGD database indicated white GNNs (mainly r module). (C) Projection genes causally implicated human cancer&#226;&#8364;&#8221;CAN genes&#226;&#8364;&#8221; <br /> according Cancer Gene Census (48). Colors indicate gene somatically mutated cancer (red GNNs) mutated germline <br /> predisposing cancer (blue GNNs) both. White GNNs indicate genes mentioned Cancer Gene Census. <br />  <br />  <br />  <br /> with embryonic-perinatal lethality. The data trans-                                                                                                   cellular expansion viable. Such reading complements <br /> ferred STN representing LCA mouse                                                                                                            results yeast, nonessential to <br /> human (i.e. STN-C) projected correspond-                                                                                                       yeast nonessential mouse cellular level. A pic- <br /> ing human network topology (Figure 5B). This data pro-                                                                                                         torial consequence complementarity results for <br /> jection shows homogeneous distribution lethal alleles                                                                                                     yeast mice set blue symbols Figure 5A <br /> among nonlethal ones (red grey GNNs, respectively),                                                                                                        overlap red symbols Figure 5B. <br /> and concentration genomic stability network of <br /> genes lacking knock-out data (white GNNs). Figure 5B <br />                                                                                                                                                                Correlating plasticity cancer statistics <br /> highlights essentiality apoptosis genome- <br /> stability gene network organism development. How-                                                                                                       The systematical, available data functional <br /> ever, genes knock-out information                                                                                                     impairment human genome-maintenance mechanisms <br /> (mainly placed r module), mouse statistics indicate                                                                                                    comes cancer statistics. According global <br /> the vast majority knock-out alleles nonessential                                                                                                     human disease network described Goh et al. (44), <br /> cellular level, given gene disruption                                                                                                      180 genes listed genome-maintenance <br />                                                                         Nucleic Acids Research, 2008, Vol. 36, No. 19 6279 <br />  <br />  <br /> gene network, 51 associated human disor-           gene loci. We attempt obtain proportion null <br /> der. From these, &gt;50% implicated cancer. As            non-null alleles human following strategy <br /> application plasticity estimates presented        used mouse infer lethality according expected <br /> previous sections, consider cancer statistics data.       frequency Mendelian distribution. We focus the <br />    Genes causally implicated cancer collectively          analysis set CAN genes placed r module, <br /> identi&#239;&#172;?ed cancer genes&#226;&#8364;&#8221;CAN genes (48), share            collectively represented locus-speci&#239;&#172;?c muta- <br /> common feature: potentially lethal             tion database&#226;&#8364;&#8221;XP mutation database (http://www. <br /> organism disruption tissue architecture, muta-         xpmutations.org). These CAN genes associated <br /> tions genes lead cancer lethal   DNA repair function (nucleotide-excision <br /> cell. These mutations types: somatic germ-         repair) related rare autosomal recessive <br /> line. While &#239;&#172;?rst arise organism development        human clinical disorders (Xeroderma pigmentosum, <br /> in cells, second inherited&#226;&#8364;&#8221;present con-       Cockayne Syndrome Trichothiodystrophy), which <br /> ception&#226;&#8364;&#8221;and continue cell. In           turns reliable obtaining representative <br /> fact, germline mutations CAN genes cause cancer pre-          sample (XP database repository XP mutations iden- <br /> disposition, cancer se, contrasting somatic         ti&#239;&#172;?ed patients worldwide). We retrieved 182 mutated <br /> mutations large extent primary cause        genotypes available database, then <br /> cancers (56).                                                    pooled according zygosity presence of <br />    Mutations lead cancer increase cell &#239;&#172;?tness (5,57),    null nonnull alleles (Table 1, Panel A). Sample <br /> implying gene network tolerate (and cell        number compared second database order <br /> may bene&#239;&#172;?t from) genetic change (58).                  attest representativeness database (Table 1, <br /> Consequently reasonable expect CAN genes           Panel B) (see Supplementary Material Online further <br /> are located plastic gene networks.                            details). Given data, case null/null patients exist in <br />    We assessed cancer statistics available Cancer     extent human population, strong <br /> Gene Census Cancer Genome Project&#226;&#8364;&#8221;CGP                     argument essentiality genes located the <br /> (http://www.sanger.ac.uk/genetics/CGP). The graph             poorly plastic region network. As pointed in <br /> Figure 5C shows projection mutations causally             Table 1, case. There total absence of <br /> implicated human cancer retrieved census.           null alleles homozygous. Therefore, considering equiva- <br /> Observe CAN genes polarized distribution          lent criteria human, mouse yeast infer leth- <br /> the network topology. Those presenting exclusively               ality, data consistent lethality germline CAN <br /> somatic mutations associated apoptotic functions        genes network projection, allowing less-plastic <br /> (red GNNs), plastic portion network,       area regarded essential human. <br /> while presenting exclusively germline ones asso- <br /> ciated genome stability (blue GNNs), poorly <br /> plastic region. Conversely, CAN genes             DISCUSSION <br /> mutation types in-between overlap apoptosis <br /> and genome-stability networks.                                   We presented orthology map order locate the <br />    The location germline mutations poses chal-          eukaryotic genes human apoptosis genome- <br /> lenge evolutionary scenario. How explain           stability gene/protein-association network. According <br /> germline mutations human genes, given         scenario, apoptosis genome stability have <br /> are located poorly plastic region? Also, care        di&#239;&#172;&#8364;erent origins evolution, spite the <br /> be taken order consider results        complex interaction systems observed in <br /> yeast mouse di&#239;&#172;&#8364;erences statistical data.        human gene network (see Figure 6 summary). <br /> For instance, CAN gene statistics comes mainly epi-         The genome-stability network emerged ear- <br /> demiological data shows exclusively genes           lier eukaryotic evolution. <br /> mutations causally implicated oncogenesis               Our results consistent scenarios <br /> have described independent reports,         described di&#239;&#172;&#8364;erent authors. For instance, position <br /> showing mutations primary patient material (48).              genome stability base eukaryotic species tree is <br /> According CGP census, underlying rationale            highly consistence DNA repair functions <br /> interpreting mutated gene causal cancer develop-         described prokaryotes [DNA repair Escherichia coli <br /> ment number pattern mutations          extensively recognized served paradigm for <br /> gene likely selected confer      investigation organisms: NER (59), BER (60), <br /> growth advantage cell population           MMR (61) RER (62)]. Also, root BCL2 the <br /> cancer developed (48). Also, contrast mouse        base LCA metazoans consistent identi&#239;&#172;?ed <br /> yeast knock-out alleles, CAN gene range            pro-<span id='am-8' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-9' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span>survival</span> functioning Bcl-2 protein family members <br /> mutations, single nucleotide substitution com-       C. elegans (63,64). Likewise, position caspases <br /> plete transcript disruption (i.e. null alleles       base LCA metazoans previously <br /> severe situation, equivalent mouse yeast knock-           described (14), consistent origins of <br /> out data).                                                       intrinsic pathway components predate TNF-like cyto- <br />    In order circumvent data limitations              kines (65). These TNF extrinsic pathway core components <br /> improve analysis investigated human           described vertebrates (47) corroborate <br /> statistics assessing genotypic pro&#239;&#172;?le CAN         scenario, line mammalian-like functioning <br />  6280 Nucleic Acids Research, 2008, Vol. 36, No. 19 <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                             Inferred changes genetic systems <br />                                                                                                                                                                                                                                                                                                                                          Root 76% genome stability components mapped network <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                             STN-P/Q <br />                                                                                                                                                                                                                                                                                                                                          (46% BER; 81% CS; 87% MMR; 89% NER; 68% RER) <br />     Distribution orthologs                                                                                                                                                                                                                                                                                                            Root 39% apoptotic components mapped network; <br />                                                                                                                                                                                                                                                                                                                                          Some earliest apoptotic core genes (e.g. Cytochrome c) <br />                                   Apoptosis <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                   Q <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                       P <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                               STN-L <br />                                                                                                                                                                                                                                                                                                                                                                                                                             Network enrichment apoptotic core <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                  O <br />                                   Genome stability                                                                                                                                                                                                                                                                                                                                                                          components intrinsic pathway <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                         N <br />                                                                                                                                                                                                                                                                                                                                                                                                                             (e.g. BCL2 initiator caspase-9) <br />                                                                                                                                                                                                                             Eukarya <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                         M <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       Network enrichment with <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                       STN-I <br />                                                                                                                                                                                                                                                                                                                                                                                                              L <br />     Distribution plasticity data                                                                                                                                                                                                                                                                                                                                                                                                                                                   apoptotic core components extrinsic <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                             K <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       pathway (e.g. TNF superfamily members) <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                           J <br />                                                      Class-a orthologs <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                       I <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     STN-F <br />                                                                                                                                                                                                                                                Opisthokonta                                                                                                                                                                                                                                                                      Presence 100% genome <br />                                                      Class-b orthologs <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  H <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  stability components mapped <br />                                                      Class-c orthologs <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                G <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  network <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     F <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  E <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             Presence 99% the <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     STN-B <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      D <br />                                                                                                                                                                                                                                                                                                                                                  Metazoa                                                                                                                                                                                                    apoptotic components <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               C <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             mapped network <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    B <br />                                                                                                                                                                                                                                                                                                                                                                              Chordata <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             A <br />                                                                                                                                                                                                                                                                                                                                                                               Vertebrata <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              18 <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               1. <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 0 <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               0 <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         15 <br />                                                                                                                                                                                                                                                                                                                                                                                                                Mammalia <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             0 <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    12 <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      0. <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               ) <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             (n <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       0 <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          5 <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                90 <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        gs <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              60 <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          0. <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     lo <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  0 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   ho <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                Primate <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          30 <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               rt <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      0 <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               O <br />      G. lamblia <br />                   T. pseudonana <br />                                   C. hominis <br />                                                P. falciparum <br />                                                                C. merolae <br />                                                                             A. thaliana <br />                                                                                           D. discoideum <br />                                                                                                           E. cuniculi <br />                                                                                                                         F. neoformans <br />                                                                                                                                         S. pombe <br />                                                                                                                                                    A. fumigatus <br />                                                                                                                                                                   Y. lipolytica <br />                                                                                                                                                                                   D. hansenii <br />                                                                                                                                                                                                 E. gossypii <br />                                                                                                                                                                                                               C. glabrata <br />                                                                                                                                                                                                                              S. cerevisiae <br />                                                                                                                                                                                                                                              K. lactis <br />                                                                                                                                                                                                                                                          C. elegans <br />                                                                                                                                                                                                                                                                       A. mellifera <br />                                                                                                                                                                                                                                                                                      A. gambiae <br />                                                                                                                                                                                                                                                                                                   D. melanogaster <br />                                                                                                                                                                                                                                                                                                                     C. intestinalis <br />                                                                                                                                                                                                                                                                                                                                       D. rerio <br />                                                                                                                                                                                                                                                                                                                                                  T. nigroviridis <br />                                                                                                                                                                                                                                                                                                                                                                    T. rubripes <br />                                                                                                                                                                                                                                                                                                                                                                                  X. tropicalis <br />                                                                                                                                                                                                                                                                                                                                                                                                  G. gallus <br />                                                                                                                                                                                                                                                                                                                                                                                                              M. domestica <br />                                                                                                                                                                                                                                                                                                                                                                                                                             B. taurus <br />                                                                                                                                                                                                                                                                                                                                                                                                                                         C. familiaris <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                         M. musculus <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                       R. norvegicus <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       M. mulatta <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    P. troglodytes <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     H. sapiens <br /> Figure 6. Summary inferred changes genetic systems. The histograms distribution 180 human orthologs according roots <br /> inferred eukaryote species tree (for details, Figures 2 3). STNs corresponding LCA indicated. Inset graph shows the <br /> presence fraction orthologs STNs (for details, Figure 4D). Diverse important events related roots sets genes pointed <br /> along STNs. Chromosome stability (CS). <br />  <br />  <br /> of extrinsic apoptosis pathway described Danio rerio                                                                                                                                                                                                                                                                                                            contrasting early origin genome-stability <br /> and absence TNF TNF receptor superfamily                                                                                                                                                                                                                                                                                                                genes. We identi&#239;&#172;?ed expansion apoptotic compo- <br /> members C. elegans (52).                                                                                                                                                                                                                                                                                                                                        nents KOG Inparanoid-derived data. <br />    However, novelty results                                                                                                                                                                                                                                                                                                             This numerical expansion apoptotic components <br /> the genome-maintenance mechanisms whole, net-                                                                                                                                                                                                                                                                                                            related origin cell functions. <br /> work-based model, produce unique evolutionary sce-                                                                                                                                                                                                                                                                                                            Such assumption illustrated TP53 appear- <br /> nario. This point view allows investigating                                                                                                                                                                                                                                                                                                                 ance transition later evolutionary scenarios: p53 <br /> sequential events led entanglement apoptosis                                                                                                                                                                                                                                                                                                        protein regulates apoptosis, key <br /> and genome-stability gene networks.                                                                                                                                                                                                                                                                                                                                regulator cellular senescence, de&#239;&#172;?ned permanent <br />    In course human genome-maintenance network                                                                                                                                                                                                                                                                                                               cell cycle arrest (66). Senescence alternative tumor <br /> evolution, major functional increments remark-                                                                                                                                                                                                                                                                                                           suppressor mechanism, damaged cells prevented <br /> able summarized Figure 6. The &#239;&#172;?rst associated                                                                                                                                                                                                                                                                                                       dividing (67). If senescence functionally <br /> the base species tree comprises genome-stability                                                                                                                                                                                                                                                                                                        emerged TP53 gene, second tumor-suppressor <br /> genes. The second evolves gradually, especially near                                                                                                                                                                                                                                                                                                           mechanism relaxed selective pressure on <br /> metazoan origin, gene components added                                                                                                                                                                                                                                                                                                                apoptosis, increasing tolerance nonadaptive <br /> apoptosis intrinsic pathway, BCL2 cas-                                                                                                                                                                                                                                                                                                             processes (e.g. genetic drift, mutation recombination) <br /> pase gene family members. The continues apo-                                                                                                                                                                                                                                                                                                             favoring evolution. Our results consistent with <br /> ptosis enrichment addition extrinsic                                                                                                                                                                                                                                                                                                           emergence major mechanisms tumor con- <br /> components, TNF superfamily members.                                                                                                                                                                                                                                                                                                                       trol metazoan evolution, regards <br />    Furthermore, macroevolutionary perspective                                                                                                                                                                                                                                                                                                            senescence genes taken account to <br /> these conclusions considered                                                                                                                                                                                                                                                                                                             draw safe conclusion. <br /> estimated evolutionary error (i.e. species tree nodes                                                                                                                                                                                                                                                                                                             Likewise, TP53 exemplify evolution genome- <br /> up rooting point species tree),                                                                                                                                                                                                                                                                                                        stability gene network. Acting transcription factor, <br /> is conceivable genes actually recent                                                                                                                                                                                                                                                                                                       p53 protein able modulate DNA-repair processes <br /> one think. Nevertheless, conclusions point                                                                                                                                                                                                                                                                                                            (9,68). Such DNA-repair gene response p53 protein in <br /> that course human genome-maintenance gene                                                                                                                                                                                                                                                                                                                line evidences showing conserved gene <br /> network evolution dramatic                                                                                                                                                                                                                                                                                                                  functions subject substantial evolution regu- <br /> increase number apoptotic components,                                                                                                                                                                                                                                                                                                                    latory level (69). <br />                                                                           Nucleic Acids Research, 2008, Vol. 36, No. 19 6281 <br />  <br />  <br />    The plasticity analysis pointed genes           plasticity emerges follows trends in <br /> evolution su&#239;&#172;&#8364;ered duplication,             unicellular organisms. <br /> poorly abundant widely distributed extant                   A test putative evolutionary scenario the <br /> species. The results locate conserved genes             human genome-maintenance network given loca- <br /> mainly genome-stability network,              tion human CAN genes. In complex organ- <br /> the ancient portion network. In contrast, cer-         isms natural selection acts di&#239;&#172;&#8364;erent levels (organism <br /> tain pairs genes known function human            &#239;&#172;?tness cell &#239;&#172;?tness), stem con&#239;&#172;&#8218;icting selective <br /> are placed di&#239;&#172;&#8364;erent distribution abundance (e.g.            pressures: fast proliferating cell clone naturally <br /> ATM BRCA1, MUS81 EME1, PCNA                            selected unicellular organism, fast proliferating cell <br /> RPA1, RPA1 RPA2&#226;&#8364;&#8221;Figure 4). Analyzing together,                 clone complex organism represent tumor that <br /> it indicate enlargement network            end killing organism. In complex organ- <br /> also occur addition new nodes even-            isms, apoptosis genome-stability networks work also <br /> tually evolve work ancient ones.                  tissue-maintenance mechanisms, favoring natural selec- <br />    Lethality measures performed complemen-             tion acting organism level. As disruption a <br /> tary ways: assessing knock-out data yeast genes             mechanism favor natural selection acting cell level, <br /> and second regarding essentiality mice. These           stands reason CAN genes located <br /> measures complementary following reasons:              plastic, cell-essential region genome- <br /> yeast unicellular organism lethality concerns             maintenance network. <br /> only cell viability, mouse multicellular animal,           Speci&#239;&#172;?cally, concerning human functional data, least <br /> with complex ontogeny. In later case, viable              questions emerge evolutionary analysis of <br /> embryo implies survival egg implantation rele-         cancer statistics: (i) distribution CAN genes is <br /> vant cell expansion. As consequence, organism            polarized major segments described the <br /> is labeled viable, certainly cell viable   evolutionary scenario? (ii) CAN genes implicated <br /> organism. However, organism viable,            types cancers (somatic germline) overlap <br /> experimental procedure does discriminate                apoptosis genome-stability networks? While addi- <br /> whether problem occurred cell organism                tional work needed fully characterize rele- <br /> level. In summary, lethality data unicellular organisms         vance results, clear this <br /> as yeast sound information genes essen-           evolutionary perspective bring insights in <br /> tial cell viability, multicellular organisms       understanding cancer origins. <br /> mice sound information genes not <br /> essential cell level. Transferring cell essentiality infor- <br /> mation mice yeast human apoptosis              SUPPLEMENTARY DATA <br /> genome-stability gene network revealed essential              Supplementary Data available NAR Online. <br /> genes cell level located ancestral <br /> region network. <br />    The integration information ancestrality, plas-       ACKNOWLEDGEMENTS <br /> ticity essentiality poses challenging questions. We <br /> found ancient, plastic essen-          We thank D. Jung technical assistance. We acknowl- <br /> tial genes located genome stability,          edge STRING, Inparanoid, MGD, SGD, CGP XP <br /> apoptosis network comprises recent, plastic          databases providing public access data. <br /> and essential genes. Genome stability required to <br /> guarantee information transference parental <br /> genome o&#239;&#172;&#8364;spring provides essen-         FUNDING <br /> tial ingredients natural selection act: memory. It       Brazilian Agencies FAPERGS, CAPES CNPq (grant <br /> not surprising genome-stability network rooted          140947/2006-0, partially). Funding open access charge: <br /> early possible species tree. It reasonable       grant 40947/2006-0. <br /> that crucial function performed highly con- <br /> served genes, gene duplication favored         Con&#239;&#172;&#8218;ict statement. None declared. <br /> the high possibility disrupting essential pathway, <br /> yielding poorly plastic network. Ancestrality, plasticity <br />                                                                    REFERENCES <br /> and essentiality pointed correlated features in <br /> typical prokaryotes (70). On hand, multicel-          1. Danial,N.N. Korsmeyer,S.J. (2004) Cell death: critical control <br /> lular organisms complex ontogeny,                 points. Cell, 116, 205&#226;&#8364;&#8220;219. <br />                                                                    2. Lettre,G. Hengartner,M.O. (2006) Developmental apoptosis in <br /> Mus musculus, available literature reports having             C. elegans: complex CEDnario. Nat. Rev. Mol. Cell Biol., <br /> found correlations (71,72). Here &#239;&#172;?nd cell gene               7, 97&#226;&#8364;&#8220;108. <br /> essentiality correlated ancestrality plasticity     3. Kanehisa,M., Goto,S., Hattori,M., Aoki-Kinoshita,K.F., Itoh,M., <br /> in unicellular complex multicellular organisms.              Kawashima,S., Katayama,T., Araki,M. Hirakawa,M. (2006) <br /> The point discriminate cell lethality            From genomics chemical genomics: new developments KEGG. <br />                                                                       Nucleic Acids Res., 34, D354&#226;&#8364;&#8220;D357. <br /> organism lethality: isolating data essential genes         4. Hipfner,D.R. Cohen,S.M. (2004) Connecting proliferation and <br /> for <span id='am-164' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-165' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-239' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span>cell survival</span> essential genes organism viabil-           apoptosis development disease. Nat. Rev. Mol. Cell Biol., <br /> ity, correlation cell essentiality, ancestrality          5, 805&#226;&#8364;&#8220;815. <br />  6282 Nucleic Acids Research, 2008, Vol. 36, No. 19 <br />  <br />  <br />  5. Crespi,B. Summers,K. (2005) Evolutionary biology cancer.        28. Ciccarelli,F.D., Doerks,T., von Mering,C., Creevey,C.J., Snel,B. <br />     Trends Ecol. Evol., 20, 545&#226;&#8364;&#8220;552.                                           Bork,P. (2006) Toward automatic reconstruction highly <br />  6. Yan,B., Wang,H., Peng,Y., Hu,Y., Wang,H., Zhang,X., Chen,Q.,               resolved tree life. Science, 311, 1283&#226;&#8364;&#8220;1287. <br />     Bedford,J.S., Dewhirst,M.W. Li,C.Y. (2006) A unique role        29. Letunic,I. Bork,P. (2007) Interactive Tree Of Life (iTOL): an <br />     DNA fragmentation factor maintaining genomic stability.             online tool phylogenetic tree display annotation. <br />     Proc. Natl Acad. Sci. USA, 103, 1504&#226;&#8364;&#8220;1509.                                 Bioinformatics, 23, 127&#226;&#8364;&#8220;128. <br />  7. Castro,M.A.A., Onsten,T.G.H., Moreira,J.C.F.                    30. Pennisi,E. (2003) Drafting tree. Science, 300, 1694. <br />     Almeida,R.M.C. (2006) Chromosome aberrations solid tumors           31. Baldauf,S.L. (2003) The deep roots eukaryotes. Science, <br />     stochastic nature. Mutat. Res., 600, 150&#226;&#8364;&#8220;164.                       300, 1703&#226;&#8364;&#8220;1706. <br />  8. Zhivotovsky,B. Kroemer,G. (2004) Apoptosis genomic             32. Katinka,M.D., Duprat,S., Cornillot,E., Metenier,G., Thomarat,F., <br />     instability. Nat. Rev. Mol. Cell Biol., 5, 752&#226;&#8364;&#8220;762.                        Prensier,G., Barbe,V., Peyretaillade,E., Brottier,P., Wincker,P. <br />  9. Sengupta,S. Harris,C.C. (2005) p53: Tra&#239;&#172;&#402;c cop cross-            et al. (2001) Genome sequence gene compaction the <br />     roads DNA repair recombination. Nat. Rev. Mol. Cell Biol.,          eukaryote parasite Encephalitozoon cuniculi. Nature, 414, 450&#226;&#8364;&#8220;453. <br />     6, 44&#226;&#8364;&#8220;55.                                                              33. Delsuc,F., Brinkmann,H. Philippe,H. (2005) Phylogenomics <br /> 10. Alano,C.C., Ying,W. Swanson,R.A. (2004) Poly(ADP-ribose)               reconstruction tree life. Nat. Rev. Genet., <br />     polymerase-1-mediated cell death astrocytes requires NAD+               6, 361&#226;&#8364;&#8220;375. <br />     depletion mitochondrial permeability transition. J. Biol. Chem.,   34. Snel,B., Bork,P. Huynen,M.A. (2002) Genomes &#239;&#172;&#8218;ux: the <br />     279, 18895&#226;&#8364;&#8220;18902.                                                          evolution archaeal proteobacterial gene content. Genome <br /> 11. Duckett,D.R., Bronstein,S.M., Taya,Y. Modrich,P. (1999)                Res., 12, 17&#226;&#8364;&#8220;25. <br />     hMutSalpha- hMutLalpha-dependent phosphorylation p53            35. Kunin,V. Ouzounis,C.A. (2003) The balance driving forces <br />     response DNA methylator damage. Proc. Natl Acad. Sci.                genome evolution prokaryotes. Genome Res., <br />     USA, 96, 12384&#226;&#8364;&#8220;12388.                                                      13, 1589&#226;&#8364;&#8220;1594. <br /> 12. Barabasi,A.L. Oltvai,Z.N. (2004) Network biology: under-           36. Campillos,M., von Mering,C., Jensen,L.J. Bork,P. (2006) <br />     standing cell&#226;&#8364;&#8482;s functional organization. Nat. Rev. Genet.,             Identi&#239;&#172;?cation analysis evolutionarily cohesive functional <br />     5, 101&#226;&#8364;&#8220;113.                                                                modules protein networks. Genome Res., 16, 374&#226;&#8364;&#8220;382. <br /> 13. Aravind,L., Walker,D.R. Koonin,E.V. (1999) Conserved               37. Itoh,M., Nacher,J., Kuma,K.i, Goto,S. Kanehisa,M. (2007) <br />     domains DNA repair proteins evolution repair systems.            Evolutionary history functional implications protein <br />     Nucleic Acids Res., 27, 1223&#226;&#8364;&#8220;1242.                                         domains combinations eukaryotes. Genome Biol., <br /> 14. Koonin,E.V. Aravind,L. (2002) Origin evolution                  8, R121. <br />     eukaryotic apoptosis: bacterial connection. Cell Death Di&#239;&#172;&#8364;er.,     38. Pal,C., Papp,B. Lercher,M.J. (2005) Adaptive evolution <br />     9, 394&#226;&#8364;&#8220;404.                                                                bacterial metabolic networks horizontal gene transfer. <br /> 15. Lin,Z., Kong,H., Nei,M. Ma,H. (2006) Origins evolution             Nat. Genet., 37, 1372&#226;&#8364;&#8220;1375. <br />     recA/RAD51 gene family: evidence ancient gene dupli-        39. Remm,M., Storm,C.E.V. Sonnhammer,E.L.L. (2001) <br />     cation endosymbiotic gene transfer. Proc. Natl Acad. Sci. USA,         Automatic clustering orthologs in-paralogs pairwise <br />     103, 10328&#226;&#8364;&#8220;10333.                                                          species comparisons. J. Mol. Biol., 314, 1041&#226;&#8364;&#8220;1052. <br /> 16. Aravind,L., Dixit,V.M. Koonin,E.V. (2001) Apoptotic molec-         40. Shannon,C.E. (1948) A mathematical theory communication. <br />     ular machinery: vastly increased complexity vertebrates revealed        Bell Syst. Tech. J., 27, 379&#226;&#8364;&#8220;423. <br />     genome comparisons. Science, 291, 1279&#226;&#8364;&#8220;1284.                        41. Kendal,W.S. (1990) The use information theory analyze <br /> 17. Merlo,L.M.F., Pepper,J.W., Reid,B.J. Maley,C.C. (2006)                 genomic changes neoplasia. Math. Biosci., 100, 143&#226;&#8364;&#8220;159. <br />     Cancer evolutionary ecological process. Nat. Rev. Cancer,    42. Castro,M.A.A., Onsten,T.T.G., Almeida,R.M.C. and <br />     6, 924&#226;&#8364;&#8220;935.                                                                Moreira,J.C.F. (2005) Pro&#239;&#172;?ling cytogenetic diversity with <br /> 18. Greaves,M. (2007) Darwinian medicine: case cancer. Nat. Rev.         entropy-based karyotypic analysis. J. Theor. Biol., 234, 487&#226;&#8364;&#8220;495. <br />     Cancer, 7, 213&#226;&#8364;&#8220;221.                                                    43. Gatenby,R.A. Frieden,B.R. (2004) Information dynamics in <br /> 19. Castro,M.A.A., Mombach,J.C.M., Almeida,R.M.C.                       carcinogenesis tumor growth. Mutat. Res., 568, 259&#226;&#8364;&#8220;273. <br />     Moreira,J.C.F. (2007) Impaired expression NER gene network       44. Goh,K.I., Cusick,M.E., Valle,D., Childs,B., Vidal,M. and <br />     sporadic solid tumors. Nucleic Acids Res., 35, 1859&#226;&#8364;&#8220;1867.                  Barabasi,A.L. (2007) The human disease network. Proc. Natl Acad. <br /> 20. Koonin,E.V. (2005) Orthologs, paralogs, evolutionary                   Sci. USA, 104, 8685&#226;&#8364;&#8220;8690. <br />     genomics. Annu. Rev. Genet., 39, 309&#226;&#8364;&#8220;338.                              45. Eppig,J.T., Blake,J.A., Bult,C.J., Kadin,J.A., Richardson,J.E. and <br /> 21. Tatusov,R., Fedorova,N., Jackson,J., Jacobs,A., Kiryutin,B.,               Mouse Genome Database Group (2007) The mouse genome <br />     Koonin,E., Krylov,D., Mazumder,R., Mekhedov,S.,                            database (MGD): new features facilitating model system. Nucleic <br />     Nikolskaya,A. et al. (2003) The COG database: updated version           Acids Res., 35, D630&#226;&#8364;&#8220;D637. <br />     includes eukaryotes. BMC Bioinformatics, 4, 41.                        46. Hirschman,J.E., Balakrishnan,R., Christie,K.R., Costanzo,M.C., <br /> 22. von Mering,C., Jensen,L.J., Kuhn,M., Cha&#239;&#172;&#8364;ron,S., Doerks,T.,                Dwight,S.S., Engel,S.R., Fisk,D.G., Hong,E.L., Livstone,M.S., <br />     Kruger,B., Snel,B. Bork,P. (2007) STRING 7&#226;&#8364;&#8220;recent develop-             Nash,R. et al. (2006) Genome Snapshot: new resource the <br />     ments integration prediction protein interactions.           Saccharomyces Genome Database (SGD) presenting overview <br />     Nucleic Acids Res., 35, D358&#226;&#8364;&#8220;D362.                                         Saccharomyces cerevisiae genome. Nucleic Acids Res., <br /> 23. Mirkin,B.G., Fenner,T.I., Galperin,M.Y. Koonin,E.V. (2003)             34, D442&#226;&#8364;&#8220;D445. <br />     Algorithms computing parsimonious evolutionary scenarios       47. O&#226;&#8364;&#8482;Brien,K.P., Remm,M. Sonnhammer,E.L.L. (2005) <br />     genome evolution, universal common ancestor domi-             Inparanoid: comprehensive database eukaryotic orthologs. <br />     nance horizontal gene transfer evolution prokaryotes.         Nucleic Acids Res., 33, D476&#226;&#8364;&#8220;D480. <br />     BMC Evol. Biol., 3, 2.                                                 48. Futreal,P.A., Coin,L., Marshall,M., Down,T., Hubbard,T., <br /> 24. von Mering,C., Jensen,L.J., Snel,B., Hooper,S.D., Krupp,M.,                Wooster,R., Rahman,N. Stratton,M.R. (2004) A census of <br />     Foglierini,M., Jou&#239;&#172;&#8364;re,N., Huynen,M.A. Bork,P. (2005)                   human cancer genes. Nat. Rev. Cancer, 4, 177&#226;&#8364;&#8220;183. <br />     STRING: known predicted protein-protein associations,              49. Stenson,P.D., Ball,E.V., Mort,M., Phillips,A.D., Shiel,J.A., <br />     integrated transferred organisms. Nucleic Acids Res.,           Thomas,N.S.T., Abeysinghe,S., Krawczak,M. Cooper,D.N. <br />     33, D433&#226;&#8364;&#8220;D437.                                                             (2003) Human gene mutation database (HGMD (R)): 2003 update. <br /> 25. Wain,H.M., Lush,M.J., Ducluzeau,F., Khodiyar,V.K. Povey,S.             Hum. Mutat., 21, 577&#226;&#8364;&#8220;581. <br />     (2004) Genew: Human Gene Nomenclat <br /> </body></html>