<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br />       Chlamydia trachomatis Serovar L2 Can Utilize Exogenous Lipoic <br />         Acid through the Action of the Lipoic Acid Ligase LplA <br />   <br />             Lipoic acid is an essential protein bound cofactor that is vital for the functioning of several important <br />          enzymes involved in central metabolism. Genomes of all sequenced chlamydiae show the presence of two genes <br />          encoding lipoic acid ligases and one gene encoding a lipoate synthase. However, the roles of these proteins in <br />          lipoic acid utilization or biosynthesis have not yet been characterized. The two distinct lipoic acid ligases in <br />          Chlamydia trachomatis serovar L2, LplA1Ct and LplA2Ct (encoded by the open reading frames ctl0537 and <br />          ctl0761) display moderate identity with Escherichia coli LplA (30 and 27%, respectively) but possess amino acid <br />          sequence motifs that are well conserved among all lipoyl protein ligases. The putative lipoic acid synthase <br />          LipACt, encoded by ctl0815, is ca. 43% identical <span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span> the E. coli LipA homolog. We demonstrate here the presence <br />          of lipoylated proteins in C. trachomatis serovar L2 and show that the lipoic acid ligase LplA1Ct is capable of <br />          utilizing exogenous lipoic acid for the lipoylation Therefore, host-derived lipoic acid may be important for <br />          intracellular growth and development. Based on genetic complementation in a surrogate host, our study also <br />          suggests that the C. trachomatis serovar L2 LipA homolog may not be functional in vivo. <br />  <br />  <br />    Members of the family Chlamydiaceae are Gram-negative                   host for a variety of metabolites, including purines, pyrimi- <br /> obligate intracellular pathogens that cause significant ocular,            dines, cofactors, and amino acids (27) and have evolved several <br /> respiratory, and sexually transmitted disease (STD) in humans              mechanisms, such as the use of specialized transporters, traf- <br /> and animals. In humans, chlamydiae cause a wide range of dis-              ficking of host organelles, and modulation of signaling path- <br /> eases, including psittacosis, pneumonia, trachoma, and STDs.               ways to successfully tap into the <span id='am-1' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span>-rich host environment <br /> Chlamydia trachomatis-mediated genital infections are among                (36). Characterizing the metabolic capabilities of intracellular <br /> the most prevalent of all STDs and result in a wide range of               pathogens is important because it enhances our understanding <br /> sequelae, including pelvic inflammatory disease, infertility, and          of the pathogen and its interactions with the host and enables <br /> ectopic pregnancy, thus posing an enormous public health                   the design of better drugs and vaccines. <br /> problem throughout the world (4). C. trachomatis also causes                  Lipoic acid is a sulfur-containing cofactor that is universally <br /> trachoma, one of the leading causes of preventable blindness in            required for aerobic metabolism (39). It is required for the <br /> the developing world (9).                                                  activity of enzymes involved in oxidative and single carbon <br />    Chlamydiae exhibit a biphasic developmental cycle that in-              metabolism such as pyruvate dehydrogenase (PDH), 2-oxoglu- <br /> volves two morphologically distinct forms (1). The elementary              tarate dehydrogenase (2-OGDH), and branched-chain keto- <br /> body (EB) is the metabolically inactive form that initiates in-            acid dehydrogenase (BCKDH). These enzyme complexes typ- <br /> fection by attaching to and invading host epithelial cells. After          ically consist of three subunits (E1, E2, and E3), and lipoic acid <br /> invasion, the EBs differentiate into the metabolically active but          is attached to a conserved lysine residue in the E2 subunit via <br /> noninfectious reticulate body (RB) within a vacuole known as               an amide bond (34). Another protein that undergoes lipoyla- <br /> the inclusion. After multiple rounds of binary fission ranging             tion is the GcsH protein of the glycine cleavage system (GCS), <br /> <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> 24 to 72 h, the RBs redifferentiate back into EBs. Even-              which converts glycine to carbon dioxide, ammonia, and 5,10- <br /> tually, cell lysis results in the release of EBs that can infect           methylenetetrahydrofolate (11). Protein-bound lipoic acid <br /> surrounding cells or be transmitted to a new host.                         then serves as a carrier of reaction intermediates between the <br />    As obligate intracellular bacteria, chlamydiae depend on the <br />                                                                            different subunits of the multienzyme complexes. <br /> host for much of their <span id='am-2' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> requirements and in general <br />                                                                               Details regarding the biosynthesis of lipoic acid and its uti- <br /> have undergone significant genome reduction (29). Consistent <br />                                                                            lization were largely obtained from studies conducted on Esch- <br /> with this lifestyle, bioinformatic analyses of chlamydiae ge- <br />                                                                            erichia coli (11). Posttranslation modification of all apo-en- <br /> nomes show that while they possess pathways for the biosyn- <br />                                                                            zymes can occur by two distinct pathways in bacteria and <br /> thesis of some nutrients, other pathways are incomplete or <br />                                                                            plants. In the first, exogenous or salvaged lipoic acid is attached <br /> entirely absent. Chlamydiae are known to be dependent on the <br />                                                                            to unlipoylated E2 subunits by the action of a lipoic acid ligase <br />                                                                            (LplA) in an ATP-dependent manner (30) (Fig. 1). The path- <br />                                                                            way involving de novo biosynthesis of the cofactor is primarily <br />          mediated through the action of two enzymes, LipA and LipB. <br />  <br />  <br />                                                                            LipB is an octanoyl-ACP transferase that transfers an octanoyl <br />          group from acyl carrier protein (ACP) to the lipoyl-accepting <br />  <br />                        domain of the apoenzyme, which subsequently becomes the <br />  <br />  <br />   FIG. 1. Lipoylation pathways demonstrated in E. coli. Lipoic acid ligase (LplA) activates exogenous lipoic acid in an ATP-dependent manner <br /> and catalyzes its transfer to the apo E2-subunit of enzyme complexes that require lipoic acid to form the holo enzyme. De novo biosynthesis of lipoic <br /> acid involves the action of two enzymes, LipB and LipA. LipB is an octanoyl-ACP (acyl carrier protein) transferase that transfers the octanoyl <br /> moiety from octanoyl-ACP to the apo-E2 subunit. LipA uses the octanoylated-E2 domain as the substrate for the synthesis of lipoylated E2. <br /> Another pathway for the biosynthesis of lipoic acid involves the transfer of free octanoic acid by LplA to the apo-E2 domain, which then serves <br /> as a substrate for LipA to form lipoylated-E2. Enzymes predicted by genome annotation that are present (enclosed in solid line box) or absent <br /> (enclosed in dashed line box) in C. trachomatis are also indicated. <br />  <br />  <br />  <br /> substrate for sulfur insertion by LipA to form lipoylated ho-                       follows. Strains from exponential-phase cultures were washed three times in 1&#226;&#171;&#187; <br /> loenzyme (28, 45) (Fig. 1). Lipoic acid synthesis can also occur                    phosphate-buffered saline (PBS; 1.05 mM KH2 PO4, 154 mM NaCl, 5.6 mM <br />                                                                                     Na2HPO4), serially diluted, and plated on M9 mannitol medium without exog- <br /> through the action of LplA which can transfer octanoic acid in                      enous lipoic acid or M9 mannitol with arabinose and lipoic acid (10 &#226;&#65533;&#174;M). <br /> an ATP-dependent manner to the apo E2 domain, which then                            Efficiency of plating was calculated as the percentage ratio between the number <br /> becomes the substrate for the action of LipA (21) (Fig. 1).                         of colonies obtained on plates without lipoic acid and those growing on plates <br />    Although lipoic acid is a vital cofactor, details regarding its                  with lipoic acid. <br /> utilization by intracellular pathogens have started to emerge                          For growth curves, single colonies from appropriate LB plates were inoculated <br />                                                                                     into M9 mannitol minimal with 0.2% arabinose, 10 mM sodium acetate, and 10 <br /> only recently (2, 10, 14, 24, 41). The obligate intracellular                       mM sodium succinate and then cultured overnight at 37&#194;&#176;C. About 1 ml of the <br /> bacterium, C. trachomatis serovar L2, encodes for four proteins                     overnight culture was then washed three times in PBS and inoculated into fresh <br /> that are predicted to undergo lipoylation. Its genome also                          M9 mannitol medium containing 0.2% arabinose with or without 10 &#226;&#65533;&#174;M lipoic <br /> encodes for two putative lipoic acid ligases designated LplA1Ct                     acid. Growth curves were performed by using a Bioscreen growth curve analyzer. <br />                                                                                     The standard deviation represents the growth measurements of five replicates. <br /> and LplA2Ct and a LipACt homolog but no putative LipB <br />                                                                                        Strain and plasmid construction. ATM967 (E. coli &#226;&#338;&#172;lplA &#226;&#338;&#172;lipB::Kanr) and <br /> homolog, suggesting that it possesses an incomplete pathway                         ATM1102 (E. coli &#226;&#338;&#172;lplA &#226;&#338;&#172;lipA::Kanr) were constructed by allelic exchange as <br /> for de novo lipoic acid biosynthesis. Transcriptome studies of                      described previously (12). Briefly, the kanamycin resistance cassette region (kan) <br /> C. trachomatis serovar D have shown that both putative ligases                      from pKD4 (Table 1) with 5&#226;&#172;&#732; and 3&#226;&#172;&#732; overhangs identical to the 5&#226;&#172;&#732; and 3&#226;&#172;&#732; regions <br /> genes and the lipoic acid synthase genes are expressed during                       of E. coli lplA, lipA, and lipB (Table 1) was amplified by PCR. All primers are <br />                                                                                     listed in Table 2. E. coli BW25113 containing pKD46 was used as the recipient <br /> growth and development (5, 6). <br />                                                                                     for linear transformation for mutant construction. Kanamycin-resistant (Kanr) <br />    In this study we show the presence of lipoylated proteins in                     recombinants were purified and screened via PCR to ensure proper insertion of <br /> C. trachomatis serovar L2 and characterize the activity of the                      the cassette. P1L4 lysates grown on the &#226;&#338;&#172;lplA::kan strain and &#226;&#338;&#172;lipB::kan strain <br /> chlamydial lipoic acid ligase, LplA1Ct. Based on the present                        were used to transduce the standard prototroph, MG1655 with selection for <br /> data, we propose that chlamydiae are capable of utilizing ex-                       Kanr. To construct the double mutant, the &#226;&#338;&#172;lplA::kan allele was transduced into <br />                                                                                     MG1655 and the transductant was transformed with pCP20 which expresses the <br /> ogenous lipoic acid. However, LipACt was found to be non- <br />                                                                                     Flp recombinase and mediates recombination between FRT sequences to excise <br /> functional in vivo in a heterologous system and did not allow us                    the kan cassette. The Kans strain thus obtained was transduced with the <br /> to determine its role in de novo biosynthesis of lipoic acid in the                 &#226;&#338;&#172;lipB::kan allele and selected on M9 minimal medium supplemented with man- <br /> bacterium.                                                                          nitol, sodium acetate, sodium succinate, and kanamycin. The &#226;&#338;&#172;lplA &#226;&#338;&#172;lipB::kan <br />                                                                                     double mutant was then stocked as ATM967. A P1L4 lysate grown on the <br />                                                                                     BW25113 &#226;&#338;&#172;lipA strain was used to transduce the MG1655 Kans &#226;&#338;&#172;lplA strain to <br />                         MATERIALS AND METHODS                                       generate the &#226;&#338;&#172;lplA &#226;&#338;&#172;lipA::kan double mutant, which was stocked as ATM1102 <br />    Bacterial strains and growth conditions. The strains used in the present study   (&#226;&#338;&#172;lplA &#226;&#338;&#172;lipA::kan). <br /> are listed in Table 1. Strains were grown in Luria-Bertani (LB) medium or M9           All plasmids used in the present study are described in Table 1. Plasmids <br /> minimal salts supplemented with 0.5% mannitol at 37&#194;&#176;C. M9 minimal medium            pLplAEc, pLipAEc, and pLipBEc contain an optimized E. coli ribosome binding <br /> was supplemented with 0.5% mannitol as the carbon source, because gene              site (RBS) and complete open reading frames of E. coli lplA, lipB, and lipA, <br /> expression under the ara promoter in pBAD18 used as the vector for comple-          respectively, under the control of the ara promoter in pBAD18. Similarly, <br /> mentation assays is repressed in the presence of glucose. Growth medium was         pLplA1Ct, pLplA2Ct, and pLipACt contain the same optimized RBS and com- <br /> supplemented with antibiotics for selection at the following concentrations: am-    plete open reading frames of ctl0537, ctl0761, and ctl0821, respectively, from C. <br /> picillin (100 &#226;&#65533;&#174;g/ml), kanamycin (50 &#226;&#65533;&#174;g/ml), and chloramphenicol (25 &#226;&#65533;&#174;g/ml).         trachomatis serovar L2 under the control of the ara promoter in pBAD18. All <br /> When required, media were also supplemented with arabinose (0.2%), IPTG             constructs were verified by sequencing at the Biomedical Instrumentation Center <br /> (isopropyl-&#226;&#65533;&#164;-D-thiogalactopyranoside; 1 mM), sodium acetate (10 mM), sodium         of the Uniformed Services University. <br /> succinate (10 mM), or lipoic acid (10 &#226;&#65533;&#174;M). Lipoic acid was obtained from Sigma.        Protein expression and purification. LplA1Ct and LplA2Ct were amplified by <br /> The lipoylated tripeptide DKLA was purchased from AnaSpec.                          PCR from C. trachomatis serovar L2 genomic DNA using Vent (New England <br />    Quantitative determination of plating efficiency of the ligases was done as      Biolabs) and ligated into the XhoI site of the expression vector pGEX-6-P1 (GE <br />  6174         RAMASWAMY AND MAURELLI                                                                                                                     J. BACTERIOL. <br />  <br />  <br />                                                  TABLE 1. Bacterial strains and plasmids used in this study <br />                                                                                                                                                            Source or <br />          Strain or plasmid                                                          Descriptiona <br />                                                                                                                                                            reference <br />  <br /> Strains <br />   C. trachomatis serovar L2             Biovar lymphogranuloma venereum L2/434/Bu                                                                   H. Caldwell <br />   E. coli MG1655                        F&#226;&#171;&#186; &#226;&#65533;&#173;&#226;&#171;&#186; ilvG rfb50 rph-1                                                                                      16 <br />      BL21(DE3)                          F&#226;&#171;&#186; ompT hsdSB(rB&#226;&#171;&#186; mB&#226;&#171;&#186;) gal dcm (DE3)                                                                        Novagen <br />      DH5&#226;&#65533;&#163;                               fhuA2 &#226;&#338;&#172;(argF-lacZ)U169 phoA glnV44 &#226;&#65533;&#190;80 &#226;&#338;&#172;(lacZ)M15 gyrA96 recA1 relA1 endA1                                  18 <br />                                           thi-1 hsdR17 <br />        BW25113                          &#226;&#338;&#172;(araD-araB)567 &#226;&#338;&#172;lacZ4787(::rrnB-3) &#226;&#65533;&#173;&#226;&#171;&#186; rph-1 &#226;&#338;&#172;(rhaD-rhaB)568 hsdR514                                        12 <br />        ATM967                           MG1655 &#226;&#338;&#172;lplA &#226;&#338;&#172;lipB::Kanr                                                                                    This   study <br />        ATM1098                          ATM967 transformed with pLplAEc                                                                             This   study <br />        ATM1099                          ATM967 transformed with pLipBEc                                                                             This   study <br />        ATM1100                          ATM967 transformed with pLplA1Ct                                                                            This   study <br />        ATM1101                          ATM967 transformed with pLplA2Ct                                                                            This   study <br />        ATM1102                          MG1655 &#226;&#338;&#172;lplA &#226;&#338;&#172;lipA::Kanr                                                                                    This   study <br />        ATM1103                          ATM1102 transformed with pLipAEc                                                                            This   study <br />        ATM1104                          ATM1102 transformed with pLipACt                                                                            This   study <br />        ATM974                           ATM967 transformed with pGEX-6-P1::LplA1Ct                                                                  This   study <br />        ATM975                           ATM967 transformed with pGEX-6-P1::LplA2Ct                                                                  This   study <br />        ATM1108                          ATM967 transformed with pGEX-6-P1::BCKDH-E2                                                                 This   study <br />        ATM921                           DH5&#226;&#65533;&#163; transformed with E. coli LplA expression construct, pYFJ16 (pQE2::LplA)                                22 <br />        ATM1112                          BL21(DE3) transformed with pYFJ16                                                                           This   study <br />  <br /> Plasmids <br />   pBAD18                                Arabinose-inducible expression vector, pBR322 ori; Ampr                                                     17 <br />   pKD4                                  ori R6K bla aphA-3                                                                                          12 <br />   pGE-X-6P1                             Protein expression vector, N-terminal GST fusion; Ampr                                                      GE Life Sciences <br />   pLplAEc                               pBAD18::lplA, contains coding sequence of E. coli lipoic acid ligase under the control                      This study <br />                                           of the ara promoter <br />   pLipAEc                               pBAD18::lipA, contains coding sequence of E. coli lipoic acid synthase under the                            This study <br />                                           control of the ara promoter <br />   pLipBEc                               pBAD18::lipB, contains coding sequence of E. coli lipoyl transferase under the                              This study <br />                                           control of the ara promoter <br />   pLipACt                               pBAD::ctl0821, contains coding sequence of lipoic acid synthase from C. trachomatis                         This study <br />                                           serovar L2 under the control of the ara promoter <br />   pLplA1Ct                              pBAD18::ctl0537, contains coding sequence of putative lipoic acid ligase from                               This study <br />                                           C. trachomatis serovar L2 under the control of the ara promoter <br />   pLplA2Ct                              pBAD18::ctl0761, contains coding sequence of putative lipoic acid ligase from                               This study <br />                                           C. trachomatis serovar L2 under the control of the ara promoter <br />   pGEX-LplA1Ct                          pGEX-6-P1::ctl0537, generates an N-terminal GST-tagged version of CTL0537                                   This study <br />   pGEX-LplA2Ct                          pGEX-6-P1::ctl0761, generates an N-terminal GST-tagged version of CTL0761                                   This study <br />   pGEX-BCKDHE2                          pGEX-6-P1::ctl0657, generates an N-terminal GST-tagged version of BCKDH-E2                                  This study <br />   pYFJ16                                E. coli lplA cloned into pQE-2 with an N-terminal His tag for overexpression                                22 <br />   a <br />       Ampr, ampicillin resistance; Kanr, kanamycin resistance. <br />  <br />  <br />  <br />  <br /> Life Sciences), verified by sequencing, and transformed into ATM967 (&#226;&#338;&#172;lplA             minimal medium supplemented with mannitol, acetate, and succinate at 37&#194;&#176;C. <br /> &#226;&#338;&#172;lipB::kan). ATM967 containing pGEX-6-P1::LplA1Ct or pGEX-6-P1::LplA2Ct                The cells were pelleted and resuspended in 12 ml of 0.1 M potassium phosphate <br /> was induced with 1 mM IPTG at room temperature overnight. Glutathione                  (pH 7), lysed by sonication, and centrifuged for 30 min at 39,000 &#226;&#171;&#187; g at 4&#194;&#176;C. <br /> S-transferase (GST)-tagged proteins were then purified from bacterial lysates by       The supernatant was then centrifuged at 236,000 &#226;&#171;&#187; g in a Beckman 45 Ti rotor <br /> using GST beads (GE Life Sciences) according to the manufacturer&#226;&#8364;&#8482;s instruc-            for 8 h. The resulting pellet was resuspended in 5 ml of 50 mM potassium <br /> tions for batch purification. The N-terminal GST tag was cleaved from the              phosphate buffer and centrifuged for an additional 8 h at 236,000 &#226;&#171;&#187; g. The <br /> recombinant protein by using PreScission protease (GE Life Sciences) and dia-          supernatant was discarded, and the pellet was resuspended in 1 ml of potassium <br /> lyzed against buffer containing 50 mM sodium phosphate (pH 7.6), 2 mM EDTA,            phosphate buffer containing 10% glycerol and stored at &#226;&#171;&#186;80&#194;&#176;C. Concentrations <br /> 100 mM NaCl, 10 mM MgCl2, and 10% glycerol. The BCKDH-E2 subunit                       of all recombinant proteins were measured by using a BCA protein assay kit <br /> encoded by ctl0657 was amplified from C. trachomatis serovar L2 by PCR and             (Pierce). <br /> also cloned into pGEX-6-P1 by using the XhoI restriction site. Recombinant                Western blot analysis. For Western blot analyses, samples were resolved by <br /> BCKDH-E2 subunit was purified as described above, and the cleaved protein              SDS-PAGE and blotted. Lipoylated proteins were detected using &#226;&#65533;&#163;-lipoic acid <br /> was dialyzed into 1&#226;&#171;&#187; PBS plus 10% glycerol. LplAEc was purified as a His-tagged        antibody (1:10,000; Calbiochem), followed by the addition of donkey anti-rabbit <br /> protein according to a previously described report (22). Briefly, E. coli BL21DE3      IgG antibody conjugated to horseradish peroxidase (1:1,000; GE Life Sciences). <br /> expressing LplAEc with an N-terminal His tag was grown at 37&#194;&#176;C until reaching          Western blots were developed using the Pico Super Signal system (Pierce) and <br /> an optical density at 600 nm (OD600) of &#226;&#172;&#402;0.5 and induced with 1 mM IPTG for            visualized by using a Fuji Intelligent Dark Box and Image Reader LAS-3000 <br /> 3 h at the same temperature. LplAEc was then purified under native conditions          software. For equal loading of samples, the protein concentration (A280) was <br /> using HisPur cobalt resin (Pierce) according to the manufacturer&#226;&#8364;&#8482;s recommen-           determined by using a NanoDrop ND-1000 spectrophotometer. <br /> dations. Purified LplAEc was then dialyzed into buffer containing 20 mM sodium            Lipoic acid ligase assays. For lipoic acid ligase activity, purified LplA1Ct (1 <br /> phosphate (pH 7.6), 2 mM EDTA, and 5% glycerol. All purified enzymes were              &#226;&#65533;&#174;M) or LplAEc (0.3 &#226;&#65533;&#174;M) was incubated in 10 mM sodium phosphate (pH 7) <br /> stored at &#226;&#171;&#186;80&#194;&#176;C.                                                                       containing 5 mM ATP, 0.3 mM dithiothreitol, and 5 mM MgCl2. E. coli apo- <br />    The purification of apo-PDH from ATM967 (&#226;&#338;&#172;lplA &#226;&#338;&#172;lipB::kan) was adapted              PDH/2-OGDH (1 &#226;&#65533;&#174;g) and chlamydial BCKDH-E2 (2 &#226;&#65533;&#174;M) were used as sub- <br /> from that of an earlier study (23). Briefly, ATM967 was grown in 100 ml of M9          strates in the reaction, and lipoic acid was used at a concentration of 1 mM. The <br />  VOL. 192, 2010                                                       C. TRACHOMATIS SEROVAR L2 AND ACTION OF LIGASE LplA1                          6175 <br />  <br />  <br />                                                               TABLE 2. Primers used in this study <br />                                                                                                Sequence (5&#226;&#172;&#732;&#226;&#8364;&#8220;3&#226;&#172;&#732;)a <br />             Primer function <br />                                                                        Forward                                                Reverse <br />  <br /> Inactivation of lplA                       CCC ATT ACT ACA AGA AAG GAA ATC GTT                                AAA GTT GCC CGC ATG GGC GGG TAA <br />                                             ATG TCC ACA TTA CGC CTC TGT GTA GGC                                CTA CCT TAC AGC CCC CGC CAT <br />                                             TGG AGC TGC TTC                                                    CAT ATG AAT ATC CTC CTT AGT TCC <br /> Inactivation of lipB                       CTC ATT CTC CAC GGA GAT ACC GTT TTG                                AAA TTG GGC CAT TGA TGT ATG GAA <br />                                             TAT CAG GAT AAA ATT CTT TGT GTA GGC                                TTA AGC GGT AAT ATA TTC GAA <br />                                             TGG AGC TGC TTC                                                    CAT ATG AAT ATC CTC CTT AGT TCC <br /> Inactivation of lipA                       ATT CGC AAC TGG AAC ACG CAC GCT ATG                                CAG ATG TAA GTA ATT ATT ACA GGA <br />                                             AGT AAA CCC ATT GTG ATG TGT GTA GGC                                TTA CTT AAC TTC CAT CCC TTT CAT <br />                                             TGG AGC TGC TTC                                                    ATG AAT ATC CTC CTT AGT TCC <br /> Clone lplA into pBAD18                     AGGAGGAATTCACCATGTCCACATTAC GCCT                                   ACGCGTCGACCTACCTTACAGCCCCCG C <br />                                             GCTC <br /> Clone lipA into pBAD18                     AGGAGGAATTCACCATGAGTAAAC CCATTGTG                                  ACG CGT CGA CTT ACT TAA CTT CCA <br />                                             ATGG AA                                                            TCC CTT TCG C <br /> Clone lipB into pBAD18                     AGG AGG AAT TCA CCT TGT ATC AGG ATA                                ACG CGT CGA CTT AAG CGG TAA TAT <br />                                             AAA TTC TTG TC                                                     ATT CGA AGT C <br /> Clone lplA1Ct into pBAD18                  AGG AGG AAT TCA CCA TGA GAA CGC GTG                                ACG CGT CGA CTT AAA TCC CCT CCT <br />                                             TCA TTG AT                                                         GCA TAA A <br /> Clone lplA2Ct into pBAD18                  AGG AGG AAT TCA CCA TGC TCA TTA ATT                                ACG CGT CGA CTT ATA GGA TTT GTG <br />                                             GCG TTT TTG TT                                                     TAG CTT TTC G <br /> Clone lipACt into pBAD18                   AGG AGG AAT TCA CCA TGA CCG ATT CAG                                ACG CGT CGA CTT A AT CTT TAT TCG <br />                                             AAT CTC CTA TT                                                     GAA GTA GCG C <br /> Clone CTL0537 into pGEX6P1                 CCGCTCGAGATGAGAACGCGTGTCATTGATTCT                                  CCGCTCGAGTTAAATCCCCTCCTGCATA <br />                                                                                                                AAGAT <br /> Clone CTL0761 into pGEX6P1                 CCGCTCGAGATGCTCATTAATTGCGTTTTTGTTC                                 CCGCTCGAGTTATAGGATTTGTGTAGCT <br />                                                                                                                TTTCGGTG <br /> Clone BCKDH-E2 (SucB2) into                CCGCTCGAGATGAAAGGGCAAAAGTATTAT                                     CCGCTCGAGCTAGGGTTCTAAAGA <br />   pGEX6P1                                                                                                      AAAG <br />   a <br />       Restriction sites are indicated in boldface. Optimized RBS sites in each primer are underlined. <br />  <br />  <br />  <br /> reaction was incubated at 37&#194;&#176;C for 1 h and analyzed by SDS-PAGE, followed by          acid, including the E2 subunits of PDH, 2-OGDH, and <br /> Western blotting with anti-lipoic acid antibody as described above.                   BCKDH (encoded by ctl0499, ctl0311, and ctl0657). All of the <br />                                                                                       enzyme subunits are predicted to possess a single lipoyl do- <br />                                   RESULTS                                             main (13). In order to examine whether C. trachomatis serovar <br />    Sequence alignments of LipACt, LplA1Ct, and LplA2Ct.                               L2 possesses lipoylated proteins, renografin purified RBs and <br /> Genes encoding for putative enzymes involved in de novo lipoic                        EBs were analyzed with a polyclonal antibody specific for li- <br /> acid biosynthesis (lipA) and lipoic acid salvage (lplA) were                          poylated proteins (37). Western blot analysis showed that the <br /> identified by BLASTp using the E. coli homologs against the C.                        anti-lipoic acid antibody recognized lipoylated proteins in both <br /> trachomatis serovar L2 genome (42). The LipA homolog                                  RB and in EB fractions (Fig. 3). The sizes of three of these <br /> (LipACt) shares 43 and 42% identity with lipoyl synthases from                        lipoylated proteins are close to the predicted sizes of the chla- <br /> E. coli and Bacillus subtilis, respectively. LipACt possesses both                    mydial E2 subunits of PDH (&#226;&#172;&#402;46 kDa), 2-OGDH (&#226;&#172;&#402;40 kDa), <br /> sets of conserved cysteine residues involved in coordinating                          and BCKDH (&#226;&#172;&#402;42 kDa). The lipoylated protein (&#226;&#172;&#402;14 kDa) <br /> iron-sulfur (Fe-S) clusters, the first of which is common to all                      present only in RBs is consistent with the predicted size of the <br /> radical SAM superfamily enzymes and the second of which is                            GcsH subunit encoded by ctl0534. <br /> unique to lipoic acid synthases (40) (Fig. 2A).                                          Functionality of lipACt, lplA1Ct, and lplA2Ct. Currently, the <br />    C. trachomatis serovar L2 also possesses two lipoic acid                           necessary tools to generate targeted <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span>knockout</span> mutants in chla- <br /> ligases, LplA1Ct and LplA2Ct (encoded by ctl0537 and ctl0761),                        mydiae is lacking; thus, E. coli is used as the surrogate host for <br /> that are only 30 and 27% identical to E. coli LplA (Fig. 2B) as                       functional characterization of relevant genes. Toward this end, <br /> determined by BLASTp analyses. These enzymes possess the                              we first constructed a &#226;&#338;&#172;lplA &#226;&#338;&#172;lipA mutant of E. coli that was <br /> typical BPL_LplA_LipB domain, which is characteristic to this                         deficient in lipoic acid synthesis (&#226;&#338;&#172;lipA) and utilization of ex- <br /> family of enzymes (35). Although LplA1Ct showed homology                              ogenous lipoic acid (&#226;&#338;&#172;lplA) but possesses a functional lipB. The <br /> to ligases from only prokaryotic organisms, LplA2Ct appeared                          resulting strain, ATM1102 (&#226;&#338;&#172;lplA &#226;&#338;&#172;lipA::kan) has an absolute <br /> to be more closely related to plant lipoic acid ligases (8).                          requirement for acetate and succinate which provide metabolic <br /> However, the two ligases did not show any similarity to the E.                        bypasses to enzymes that require lipoic acid and served as <br /> coli LipB transferase. Orthologs of both ligases have been                            the background for determining the functionality of the <br /> identified in all of the sequenced chlamydiae, suggesting that                        chlamydial lipoic acid synthase (lipACt). Because ATM1102 <br /> they are a well-conserved family of genes (Table 3).                                  (&#226;&#338;&#172;lplA &#226;&#338;&#172;lipA::kan) possess a functional lipB gene, comple- <br />    Evidence of lipoylation in C. trachomatis serovar L2 EBs and                       mentation with E. coli lipA (lipAEc) or functional chlamydial <br /> RBs. Chlamydiae possess at least three proteins that are pre-                         lipoyl synthase (lipACt) should restore de novo lipoic acid bio- <br /> dicted to undergo posttranslational modification with lipoic                          synthesis, thereby allowing bacteria to grow on M9 mannitol <br />  6176      RAMASWAMY AND MAURELLI                                                                                                        J. BACTERIOL. <br />  <br />  <br />  <br />  <br />    FIG. 2. CLUSTAL W sequence alignments of chlamydial proteins predicted to be involved in lipoate utilization and synthesis with represen- <br /> tative proteins. (A) The putative chlamydial LipA (LipACt) was aligned with lipoic acid synthases from Escherichia coli (E.c), Bacillus subtilis (B.s), <br /> Saccharomyces cerevisiae, (S.c), plant Arabidopsis thaliana (A.t), and Homo sapiens (H.s). Residues common to SAM radical super family of <br /> enzymes (I) and residues specific for lipoic acid synthases (II) are highlighted in gray. (B) Alignment of the putative lipoic acid ligases from C. <br /> trachomatis serovar L2 (LplA1Ct and LplA2Ct) with ligases from E. coli (E.c.) and Thermoplasma acidophilum (T.a.). Conserved regions are <br /> highlighted in gray, and residues involved in lipoyl-AMP binding are indicated by an inverted triangle (&#194;&#65533;). <br />  <br />  <br />  <br /> minimal medium in the presence of arabinose. ATM1102 was                      plement the mutant strain (100% by efficiency of plating), <br /> transformed with pLipAEc or pLipACt expressing lipAEc and                     whereas no complementation was observed with lipACt <br /> lipACt, respectively, under the control of the arabinose-induc-               (&#226;&#172;&#65533;10&#226;&#171;&#186;4% by efficiency of plating) under similar induction con- <br /> ible promoter in pBAD18. Transformants were selected on M9                    ditions. Similar results were obtained when complementation <br /> mannitol medium supplemented with acetate, succinate, and                     analysis was repeated with lipACt cloned into a high-copy vec- <br /> ampicillin. Select transformants were subsequently examined                   tor (data not shown). <br /> for growth in the presence of arabinose but in the absence of                    Next, we sought to assess the functionality of the two chla- <br /> exogenous lipoic acid. Our complementation studies showed                     mydial ligase genes, lplA1Ct and lplA2Ct. For this, a second <br /> that upon induction with arabinose, lipAEc was able to com-                   double mutant of E. coli (&#226;&#338;&#172;lplA &#226;&#338;&#172;lipB) was generated, and it <br />  VOL. 192, 2010                                                    C. TRACHOMATIS SEROVAR L2 AND ACTION OF LIGASE LplA1                              6177 <br />  <br />  <br />   TABLE 3. Amino acid sequence identities of chlamydial lipoic <br />      acid ligases and lipoic acid synthase with the respective <br />                          E. coli homologs <br />                                                 % Sequence identitya to: <br />  <br />               Species                           LplAEc <br />                                                                        LipAEc <br />                                         LplA1            LplA2 <br />  <br /> Chlamydia trachomatis L2                  27              30               43 <br /> Chlamydia muridarum                       25              27               44 <br /> Chlamydia pneumoniae                      25              26               44 <br /> Chlamydia caviae                          22              25               41 <br /> Chlamydia felis                           23              23               41 <br /> Protochlamydia amoebophila                25              26               44 <br />   a <br />       Sequence identities were determined by using BLASTp. <br />  <br />  <br />  <br /> was designated ATM967. This strain lacks the ability to syn- <br /> thesize lipoic acid (&#226;&#338;&#172;lipB), as well as utilize exogenous lipoic <br /> acid (&#226;&#338;&#172;lplA), and provides a better background for assessing <br /> ligase activity. To assay for putative ligase activity, ATM967 <br /> (&#226;&#338;&#172;lplA &#226;&#338;&#172;lipB::kan) was transformed with pLplAEc, pLplA1Ct, <br /> and pLplA2Ct expressing lplAEc, lplA1Ct, and lplA2Ct, respec- <br /> tively, under the control of arabinose-inducible promoter in <br /> pBAD. Transformants were screened for their ability to grow                        FIG. 4. Growth of lipoic acid ligase mutant and transformed strains <br /> on M9 minimal medium in the presence of mannitol, 10 &#226;&#65533;&#174;g of                      in the presence or absence of lipoic acid. Strains (see inset) were grown <br /> lipoic acid/ml, and 0.2% arabinose. LplAEc was able to restore                  in M9 mannitol medium containing 0.2% arabinose at 37&#194;&#176;C in the <br />                                                                                 absence (A) or presence of 10 &#226;&#65533;&#174;M lipoic acid (B). The absorbance <br /> the ability to utilize exogenous lipoic acid (100% by efficiency                (OD600) was measured in a Bioscreen growth curve analyzer and plot- <br /> of plating), while LplA1Ct complemented at an efficiency of                     ted against time (hours). Error bars represent the standard deviation <br /> 94% determined by the same method. However, no comple-                          from five replicates. <br /> mentation was observed with LplA2Ct (&#226;&#172;&#65533;10&#226;&#171;&#186;5%). <br />    In order to further examine the functionality of the ligases <br /> and to see whether there was a delayed growth of the LplA2Ct <br /> transformed strain, the growth rates of ATM967 (&#226;&#338;&#172;lplA                           were identical to each other and grew faster than MG1655 <br /> &#226;&#338;&#172;lipB::kan) expressing LplAEc, LplA1Ct, and LplA2Ct were                        transformed with pBAD18, presumably due to overexpression <br /> measured in minimal medium with or without lipoic acid. As                      of the ligases. The MG1655 transformed with pBAD18 exhib- <br /> seen in Fig. 4A, in the absence of lipoic acid, the pLplA1Ct and                ited a longer lag most likely due to the presence of only a single <br /> the pLplAEc transformed strains showed growth kinetics that                     copy of the chromosomal ligase. However, the pLplAEc- and <br />                                                                                 pLplA1Ct-expressing ATM967 and the MG1655 control strains <br />                                                                                 all had the same slope in exponential phase. A similar trend <br />                                                                                 was seen for these strains even in the presence of lipoic acid <br />                                                                                 (Fig. 4B), with growth kinetics of the LplAEc- and LplA1Ct- <br />                                                                                 expressing strains being slightly faster than that of the MG1655 <br />                                                                                 control. The growth rate of the pLplA2Ct transformed strain <br />                                                                                 was similar in the presence or absence of lipoic acid, reaching <br />                                                                                 an OD600 of only &#226;&#172;&#402;0.3 even after 46 h of growth. The double <br />                                                                                 mutant, ATM967, transformed with pBAD18 showed no <br />                                                                                 growth throughout the experiment with or without lipoic acid. <br />                                                                                    In order to confirm protein lipoylation in vivo, lysates of <br />                                                                                 ATM967 (&#226;&#338;&#172;lplA &#226;&#338;&#172;lipB::kan) expressing lplAEc, lplA1Ct, and <br />                                                                                 lplA2Ctwere blotted with the anti-lipoic acid antibody. Wild- <br />                                                                                 type E. coli is known to possess three lipoylated proteins: <br />                                                                                 PDH-E2 (65 kDa), 2-OGDH-E2 (45 kDa), and GcsH protein <br />                                                                                 (&#226;&#172;&#402;15 kDa). As shown in Fig. 5, two lipoylated proteins corre- <br />                                                                                 sponding to PDH-E2 and 2-OGDH-E2 were detected in the E. <br />   FIG. 3. Detection of lipoylated proteins in C. trachomatis serovar            coli MG1655 lysates and in strains expressing lplAEc and <br /> L2 RBs and EBs. Equal concentrations of Renografin-purified EBs                 lplA1Ct. The glycine cleavage protein GcsH was not observed in <br /> and RBs were resolved by SDS-PAGE blotted and analyzed by West-                 these lysates because the cultures were not induced by glycine. <br /> ern blotting with anti-lipoic acid antibody. The sizes of E2 subunits of        Interestingly, in lysates expressing lplAEc and lplA1Ct, lipoyla- <br /> pyruvate dehydrogenase (PDH-E2), 2-oxoglutarate dehydrogenase (2- <br /> OGDH-E2), branched-chain keto-acid dehydrogenase (BCKDH-E2) <br />                                                                                 tion was also seen in the absence of exogenous lipoic acid, <br /> and that of the glycine cleavage system protein H (GcsH) are indicated          suggesting that the LplA1Ct may also be capable of utilizing <br /> on the blot.                                                                    endogenous octanoic acid as a substrate. The octanoylated <br />  6178      RAMASWAMY AND MAURELLI                                                                                                     J. BACTERIOL. <br />  <br />  <br />  <br />  <br />    FIG. 5. Detection of lipoylated proteins in lipoic acid ligase mutant and transformed strains. Lysates of transformed strains (LplAEc, LplA1Ct, <br /> and LplA2Ct), MG1655 (positive control), and ATM967 (negative control) grown in minimal medium with or without lipoic acid were analyzed <br /> for the presence of lipoylated proteins. Equal concentrations of the lysates were separated by SDS-PAGE, blotted, and probed with anti-lipoic acid <br /> antibody. Lipoylated proteins corresponding to PDH-E2 and 2-OGDH-E2 of E. coli were observed in the strains expressing LplAEc and LplA1Ct <br /> in the presence or absence of lipoic acid but not in the LplA2Ct-expressing strain. <br />  <br />  <br /> apo-domain may then be modified into lipoylated holo-domain                 PDH and 2-OGDH and recombinant chlamydial BCKDH-E2 <br /> by the functional LipA in ATM967 (&#226;&#338;&#172;lplA &#226;&#338;&#172;lipB::kan) (Fig. 1).               as lipoate acceptors. The latter was overexpressed in ATM967 <br /> This observation is consistent with our growth curve studies                as a GST-tagged protein so that it could purified in the <br /> and previous reports that have shown that LplAEc can also                   apo-form. Consistent with our in vivo complementation <br /> utilize free octanoic acid as a substrate (21, 28, 30). Therefore,          data, recombinant LplA1Ct was able to lipoylate apo-PDH <br /> LplA1Ct appears to function in a manner similar to that of                  and 2-OGDH-E2 subunits purified from E. coli ATM967 (Fig. <br /> LplAEc.                                                                     6A, left) as detected by Western blotting with anti-lipoic acid <br />    We also examined the ability of LplA2Ct to utilize exogenous             antibody. LplA1Ct was also able to lipoylate the recombinant <br /> octanoic acid. ATM967 (&#226;&#338;&#172;lplA &#226;&#338;&#172;lipB::kan) was transformed                    chlamydial BCKDH-E2 subunit (Fig. 6B, left). Purified LplAEc <br /> with pLipBEc, pLplAEc, or pLplA2Ct and examined for its                     served as a positive control for both reactions (Fig. 6A and B, <br /> ability to grow on M9 mannitol medium supplemented with 50                  right). Recombinant LplA2Ct was unable to lipoylate either <br /> &#226;&#65533;&#174;M octanoic acid. Although pLipBEc (98% efficiency of plat-                 substrate, suggesting that this enzyme was inactive under the <br /> ing) and pLplAEc (80% efficiency of plating) were able to                   conditions in which it was tested (data not shown). <br /> complement the double mutant upon induction with arabinose,                    Previous reports have described the ability of lipoic acid <br /> no complementation was observed with pLplA2Ct (&#226;&#172;&#65533;10&#226;&#171;&#186;5%                       ligases from Listeria monocytogenes to enable the utilization of <br /> efficiency of plating). In vivo lipoylation was not seen in cells           lipoylated tripeptides as a source of lipoic acid (24). In order to <br /> expressing LplA2Ct in the presence or absence of lipoic acid                determine whether the chlamydial ligases could function in a <br /> (Fig. 4) and confirms the negative complementation data.                    similar manner, we used a lipoylated tripeptide consisting of <br />    In vitro lipoylation activity of LplA1Ct. For in vitro assays, we        aspartate-lysine [lipoic acid]-alanine (DKLipA) as a source for <br /> used lysates of ATM967 (&#226;&#338;&#172;lplA &#226;&#338;&#172;lipB::kan) enriched for apo-                 lipoic acid in the in vitro assay. However, LplA1Ct and LplA2Ct <br />  <br />  <br />  <br />  <br />    FIG. 6. In vitro lipoic acid ligase activity. Purified LplAEc (positive control) and LplA1Ct were assayed for activity in the presence or absence <br /> of various components as indicated. Purified LplA1Ct was assayed for activity using a mixture of E. coli apo-PDH/2-OGDH (panel A, left) and <br /> recombinant chlamydial BCKDH-E2 (panel B, left) as substrates at 37&#194;&#176;C for 1 h. Reactions were analyzed by SDS-PAGE and blotted, and <br /> lipoylated proteins were detected using anti-lipoic acid antibody. Purified LplAEc served as a positive control and was also tested with apo-PDH/ <br /> 2-OGDH (panel A, right) and chlamydial BCKDH-E2 (panel B, right). <br />  VOL. 192, 2010                                          C. TRACHOMATIS SEROVAR L2 AND ACTION OF LIGASE LplA1                      6179 <br />  <br />  <br /> were not able to utilize DKLipA to lipoylate the apo-enzyme          2-OGDH-E2 subunits, respectively (2). Of the two putative <br /> substrates (data not shown).                                         ligases in C. trachomatis serovar L2, LplA1Ct was found to be <br />                                                                      functional in vitro and in vivo, whereas LplA2Ct appeared to be <br />                                                                      inactive in both the surrogate E. coli host and in in vitro <br />                          DISCUSSION <br />                                                                      lipoylation assays. The detection of lipoylated proteins in <br />    Lipoic acid is an essential cofactor that is required for the     the LplA1Ct-expressing strain even in the absence of octanoic <br /> posttranslational modification of several enzyme complexes           acid suggests that it can utilize free octanoic acid-like LplAEc <br /> involved in central metabolic processes. Organisms can obtain        (21). <br /> lipoic acid either by de novo biosynthesis or by salvage of lipoic      Transcriptome studies of C. trachomatis serovar D, which <br /> acid from the environment. For example, E. coli, which has           has a similar intracellular progression as C. trachomatis L2, <br /> served as the model for unraveling the various steps involved in     have shown that lplA1Ct gene expression is detected about 8 h <br /> lipoic acid biosynthesis, is capable of synthesizing lipoic acid     postinfection, coincident with the differentiation of EBs into <br /> and utilizing exogenously available lipoic acid. Organisms           metabolically active RBs. However, the expression of lplA2Ct is <br /> such as Rickettsia spp. do not possess an LplA homolog but           detected only 16 h postinfection (6). Overall, relative expres- <br /> are predicted to synthesize lipoic acid through the action of        sion of lplA1Ct is slightly higher than that of lplA2Ct throughout <br /> LipA/B homologs (3). Others, such as Listeria spp., are lipoic       the developmental cycle (6), suggesting that it may play a more <br /> acid auxotrophs (24). Genes involved in the synthesis or utili-      active role. As described earlier, LplA1Ct is more similar to <br /> zation of salvaged lipoic acid appear to play an important role      other bacterial ligases, whereas LplA2Ct is more related to <br /> in the growth and replication of some intracellular bacterial        plant lipoic acid ligases and could be regarded as a remnant of <br /> pathogens such as Burkholderia pseudomallei (33) and Listeria        this ancestral relationship between chlamydiae and cyanobac- <br /> monocytogenes (32). Parasites such as Toxoplasma and Plasmo-         terial endosymbionts of chloroplasts (8). Inactivity of LplA2Ct <br /> dium have an absolute requirement for lipoic acid from host          in our system could be the result of incorrect assay conditions <br /> cells even though they possess a functional pathway for the de       or the lack of a lipoic acid donor or substrate specific to <br /> novo biosynthesis of the cofactor, which is produced and se-         LplA2Ct. Inherent difficulties associated with genetic manipu- <br /> questered in the apicoplast (10, 14, 15). In Mycobacterium           lation of chlamydiae precluded gene inactivation experiments <br /> tuberculosis, the lipB gene is considerably upregulated in mul-      that could have elucidated the function of LplA2Ct. <br /> tiple drug-resistant strains, and attempts to construct a lipB          The de novo pathway leading to the formation of lipoic acid <br /> mutant proved unsuccessful, suggesting that lipB is indispens-       involves the action of two enzymes, LipA and LipB. LipA <br /> able for growth (25). These examples underscore the impor-           belongs to the radical SAM family of enzymes that also in- <br /> tance of lipoic acid in the growth and replication of bacterial      cludes enzymes such as biotin synthase (31). LipA binds to <br /> pathogens.                                                           iron-sulfur clusters and mediates reductive cleavage of S-ad- <br />    C. trachomatis serovar L2 possesses a putative lipoic acid        enosylmethionine to generate 5&#226;&#172;&#732;dA&#228;&#161;&#160; radical, which ultimately <br /> synthase and two predicted lipoic acid ligase homologs, and          results in the insertion of two sulfur atoms (presumably from <br /> this raised some intriguing questions about the biosynthesis         the iron-sulfur cluster) into the octanoylated apodomain to <br /> and utilization of lipoic acid. Bacteria typically possess between   form lipoic acid (11). Although LipACt shows a fairly high <br /> three and four lipoylated proteins. Western blot analyses of C.      degree of homology to the E. coli LipA and has all of the <br /> trachomatis serovar L2 EB and RB lysates show the presence           canonical residues that are typical for this enzyme family, it <br /> of lipoylated chlamydia proteins that correspond to predicted        failed to restore growth of an E. coli mutant lacking lipA. <br /> sizes of E2 subunits of PDH, 2-OGDH, and BCKDH. How-                 Homologs of LipA from other organisms have been shown to <br /> ever, we were unable to confirm their identities due to the lack     complement a similar E. coli lipA mutant, suggesting that het- <br /> of specific antibodies. RB lysates showed the presence of an         erologous enzymes are capable of functioning in vivo and com- <br /> additional small lipoylated protein (&#226;&#172;&#65533;15 kDa) that may corre-        plementing an E. coli mutant (26, 44). Bacillus subtilis pos- <br /> spond to the GcsH subunit (ctl0534) of the glycine cleavage          sesses two ligases and a lipoic acid synthase but no putative <br /> system. Interestingly, analysis of genome sequences by BLAST         lipB, an arrangement similar to that encountered in chlamyd- <br /> shows that the other subunits that form the glycine cleavage         iae (26). The B. subtilis lipA is essential for growth of the <br /> complex are absent in chlamydiae. In Saccharomyces cerevisiae,       bacteria in minimal medium without exogenous lipoic acid, <br /> lipoylated GcsH is required for the lipoylation of PDH-E2 and        suggesting that it is capable of de novo lipoic acid biosynthesis, <br /> 2-OGDH-E2 (38). Orphan GcsH proteins have also been re-              even in the absence of a putative LipB transferase (26). The <br /> ported in other systems (41). The presence of a putative lipoy-      existence of a similar pathway in chlamydiae cannot be ruled <br /> lated GcsH subunit in chlamydiae suggests that it may have a         out since we have shown that LplA1Ct expressed in ATM967 <br /> different role in metabolism. Alternately, this band may rep-        (&#226;&#338;&#172;lplA &#226;&#338;&#172;lipB::kan) can lipoylate target proteins even in the <br /> resent a breakdown product of one of the larger lipoylated           absence of exogenous lipoic acid. Due to the limitations im- <br /> subunits.                                                            posed by the lack of genetic tools in chlamydiae, we used a <br />    The presence of two lipoic acid ligases is not uncommon           lipoic acid synthase-deficient E. coli mutant as a surrogate host <br /> in bacteria and has been previously reported in lipoic acid          to examine the functionality of LipACt. Our complementation <br /> auxotrophs such as L. monocytogenes wherein the ligases are          assay in this system suggests that LipACt is not functional in <br /> paralogous, and each ligase is proposed to utilize different         vivo. However, we cannot rule out the possibility that LipACt <br /> sources of lipoic acid (24). In organisms such as Toxoplasma         may not be functionally similar to E. coli LipA or that expres- <br /> and Plasmodium, the two ligases appear to be quite divergent         sion in the surrogate host could have resulted in an unstable <br /> at the sequence level and exhibit specificity for PDH-E2 or          protein or incomplete assembly of iron-sulfur clusters required <br />  <br /> for activity. In fact, several attempts to overexpress and purify                      <br /> LipACt in the soluble form proved unsuccessful and did not                   <br /> allow us to conclusively demonstrate the presence of the de                               <br /> novo pathway that perhaps utilizes LipACt and LplA1Ct.                             <br />    Taken together, our data suggest that chlamydiae are capa-                             <br /> ble of utilizing exogenous lipoic acid during growth and differ-                        <br /> entiation. Given the relatively low concentration of free lipoic                    <br /> acid in the intracellular environment (39), questions still re-                    <br /> main as to the exact mechanism by which chlamydiae are able                        <br /> to gain access to and sequester the cofactor. Chlamydiae are                      <br /> known to alter mitochondrial function (19) and interact with                             <br /> the host mitochondria through transient or indirect interac-                          <br /> tions (36) and may therefore have access to the limited con-                             <br /> centrations of free lipoic acid. Lipoic acid is a fairly small                      <br /> molecule (206 Da) and could thus freely diffuse into the inclu-                   <br /> sions (20). Another source for lipoic acid, as hypothesized for                  <br /> Toxoplasma (10), could be lipoylated peptides from breakdown                       <br /> of host enzymes, which could be taken up by the bacterium                    <br /> through an oligopeptide transporter and subsequently cleaved                             <br /> by nonspecific amidases to release free lipoic acid for further                     <br /> utilization.                                                                           <br />    Although antibiotic therapies are currently available for the                           <br /> treatment of ocular and genital tract infections caused by chla-                   <br /> mydiae, reports of treatment failure and demonstration of in                            <br /> vitro resistance to antibiotics suggest that development of re-                    <br /> sistance to commonly used antibiotics against chlamydiae is                      <br /> possible (7, 43). Given the importance of lipoic acid and its role                        <br /> in the growth and virulence of other intracellular pathogens,                        <br /> genes involved in lipoic acid biosynthesis and utilization make                            <br /> attractive candidates for the development of novel therapeu-                              <br /> tics (25). In the present study, we have demonstrated the ac-                        <br /> tivity of the chlamydial lipoic acid ligase, LplA1Ct, which relies                        <br /> on exogenous lipoic acid obtained from the host. Therefore,                             <br /> LplA1Ct presents a potential target that can be exploited for                     <br /> the development of novel anti-infectives against chlamydiae.                             <br /> </body></html>