<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br /> Impact of Extracellularity on the Evolutionary Rate of <br /> Mammalian Proteins <br />  <br />  <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br /> It is of fundamental importance <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> understand the determinants of the rate of protein evolution. Eukaryotic extracellular <br /> proteins are known to evolve faster than intracellular proteins. Although this rate difference appears to be due to the lower <br /> essentiality of extracellular proteins than intracellular proteins in yeast, we here show that, in mammals, the impact of <br /> extracellularity is independent <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> the impact of gene essentiality. Our partial correlation analysis indicated that the impact <br /> of extracellularity on mammalian protein evolutionary rate is also independent from those of tissue-specificity, expression <br /> level, gene compactness, and the number of protein&#226;&#8364;&#8220;protein interactions and, surprisingly, is the strongest among all the <br /> factors we examined. Similar results were also found from principal component regression analysis. Our findings suggest that <br /> different rules govern the pace of protein sequence evolution in mammals and yeasts. <br /> Key words: evolutionary rate, subcellular localization, gene essentiality, gene expression level, mammal, yeast. <br />  <br />  <br />  <br /> It has been of great interest among molecular evolutionists                         ness of its genomic (Waterston et al. 2002) and transcrip- <br /> to identify factors that explain the large variation in the evo-                    tomic (Su et al. 2004) data. Based on the &#226;&#8364;&#732;&#226;&#8364;&#732;cellular <br /> lutionary rate of proteins encoded in a genome (Fraser et al.                       component&#226;&#8364;&#8482;&#226;&#8364;&#8482; terms in Gene Ontology (GO; www. <br /> 2002; Subramanian and Kumar 2004; Drummond et al.                                   geneontology.org), we treated proteins exclusively located <br /> 2005; Zhang and He 2005; Liao et al. 2006; Makino and                               outside and inside the cell membrane as extracellular and <br /> Gojobori 2006). Extracellular proteins, also known as se-                           intracellular proteins, respectively (see supplementary <br /> creted proteins, have been shown to exhibit elevated rates                          fig S1, Supplementary Material online and Materials and <br /> of nonsynonymous substitutions in both yeasts and mam-                              Methods). Following our previous work (Liao et al. 2006), <br /> mals (Winter et al. 2004; Julenius and Pedersen 2006; Dean                          we defined mouse essential genes by <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span>knockout</span> phenotypes <br /> et al. 2008), even after the control of gene expression level                       of premature death or sterility (see Materials and Method). <br /> and number of protein interactions (Julenius and Pedersen                           To study the impact of a factor on the protein evolutionary <br /> 2006). In yeast, however, the evolutionary rate is no longer                        rate, one should compare closely rated species (Zhang and <br /> significantly different between extra- and intracellular                            He 2005) because gene properties (e.g., subcellular localiza- <br /> proteins after the control for gene essentiality (Julenius and                      tion and essentiality; Liao and Zhang 2008; Qian and Zhang <br /> Pedersen 2006), suggesting that extracellularity does not di-                       2009) and evolutionary rates may change in evolution. We <br /> rectly influence protein evolutionary rate. Here, we show that                      thus used one-to-one orthologs between mouse and rat <br /> this is not the case in mammals. More importantly, the impact                       (Rattus norvegicus) to estimate the rates of synonymous <br /> of extracellularity on mammalian protein evolutionary rate is                       (dS) and nonsynonymous (dN) substitutions. Secreted <br /> not only independent from that of gene essentiality but also                        proteins often contain a rapidly evolving signal peptide <br /> the greatest among all the factors examined.                                        (Williams et al. 2000). To avoid overestimating substitution <br />     To investigate the influence of extracellularity on the evo-                    rates of extracellular proteins, signal peptides were removed <br /> lutionary rate of mammalian proteins, we chose mouse                                prior to estimating dN and dS. We found that extracellular <br /> (Mus musculus) as our focal species for the comprehensive-                          proteins are enriched with proteins related to immune <br />  <br />  <br /> &#194;&#170; 2010 The Authors <br /> This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/ <br /> 2.5), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited. <br />  <br />  <br />  <br />  <br /> Genome Biol. Evol. 2(1):39&#226;&#8364;&#8220;43. 2010 doi:10.1093/gbe/evp058 Advance Access publication January 6, 2010                                                        39 <br />  Liao et al.                                                                                                                             GBE <br />  <br />  <br />  <br />  <br />      FIG. 1.&#226;&#8364;&#8221;Comparison of (A) dN, (B) dS, and (C) dN/dS between extracellular and intracellular proteins. The upper quartile, median, and lower <br /> quartile are indicated in each box; the bars indicate semiquartile ranges. The P values (U test) for the hypothesis of no difference between extracellular <br /> and intracellular proteins are shown above each panel. <br />  <br /> response (P 5 2.54e 28, v2 test). Because many immune-                          stantially reduced after the control of other factors (table 2). <br /> related proteins are subject to positive selection (Hughes                      Partial correlation analysis may have limitations under cer- <br /> 1999) and because X-linked genes tend to be fast evolving                       tain conditions (Drummond et al. 2006; Kim and Yi <br /> (Vicoso and Charlesworth 2006), these proteins were ex-                         2007). We thus conducted a principle component regres- <br /> cluded. Our final data set for subsequent analysis included                     sion analysis of the same data. Consistent with the results <br /> 3,069 mouse&#226;&#8364;&#8220;rat orthologs with information about gene                           from the partial correlation analysis (tables 1 and 2), we <br /> essentiality and subcellular localization in mouse. Among                       found that extracellularity contributes most to the first prin- <br /> them, 1,740 are intracellular and 288 are extracellular                         ciple component that explains the variance in mouse&#226;&#8364;&#8220;rat dN, <br /> (see supplementary fig. S1, Supplementary Material online).                     dS, and dN/dS (supplementary table S1, Supplementary Ma- <br />     We found that extracellular proteins have an average                        terial online). It is possible that the evolutionary rate differ- <br /> mouse&#226;&#8364;&#8220;rat dN (0.047) 61% higher than that of intracellular                      ence between intracellular and extracellular proteins is a <br /> proteins (0.029; P 5 7.1E-12, Mann&#226;&#8364;&#8220;Whitney U test;                              by-product of different distributions of GO terms among <br /> fig. 1A). Because the average dS for extracellular (0.24)                       the two groups of proteins. However, higher dN/dS for ex- <br /> and intracellular (0.21) proteins differ by only 13.5% (fig.                    tracellular proteins than intracellular proteins was observed <br /> 1B), elevated <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span>mutation</span> rate does not fully explain the differ-                  even when we compared proteins of the same GO terms <br /> ence in dN. Significantly higher dN/dS of extracellular pro-                    (fig 2A and C) or after excluding genes with differentially <br /> teins (average 5 0.202) than intracellular proteins (0.132;                     distributed GO terms (fig. 2B and D). Twenty-three GO <br /> fig. 1C) suggests that extracellular proteins are subject to                    categories are significantly differently distributed between in- <br /> more frequent or stronger positive selection or relaxed pu-                     tracellular and extracellular proteins and each contain at least <br /> rifying selection. Mammalian essential proteins tend to                         25 essential extracellular proteins, 25 nonessential extracellu- <br /> evolve slowly (Liao et al. 2006). We found that intracellular                   lar proteins, 25 essential intracellular proteins, and 25 nones- <br /> proteins contain a higher proportion of essential genes                         sential intracellular proteins (supplementary table S2, <br /> (1104/1740 5 63.5%) than extracellular proteins (125/                           Supplementary Material online). With the exception of two <br /> 288 5 43.4%; P 5 1.13e 10, v2 test). However, dN and                            GO categories, median dN/dS of extracellular proteins are sig- <br /> dN/dS of extracellular proteins are still significantly greater                 nificantly higher than that of intracellular proteins when pro- <br /> than intracellular proteins (fig. 1A and C), even when only                     teins of the same essentiality are compared within each of the <br /> essential or only nonessential genes are considered. Clearly,                   GO categories (supplementary table S2, Supplementary Ma- <br /> extracellularity impacts the rate of mammalian protein                          terial online). We also repeated our analysis by removing pro- <br /> evolution independently from gene essentiality.                                 teins of unknown molecular functions, proteins not involved <br />     In addition to essentiality and extracellularity, determi-                  in any known biological process, and proteins located in syn- <br /> nants of the rate of mammalian protein evolution also in-                       apse and obtained essentially the same results (supple- <br /> clude expression level, tissue-specificity, number of                           mentary figs. S3 and S4, Supplementary Material online). <br /> interacting proteins, and gene compactness (i.e., <span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span>                        Together, these results indicate that extracellularity has a ma- <br /> of introns and untranslated regions, or UTRs; Liao et al.                       jor, and likely direct, impact on mammalian protein evolution. <br /> 2006; Liang and Li 2007). We compared the relative impor-                           The influence of extracellularity on protein evolutionary <br /> tance of these factors by performing Spearman&#226;&#8364;&#8482;s rank                            rate differs greatly between what we found in mammals <br /> correlation between dN and each of the above factors. Im-                       and what was reported in yeasts (Julenius and Pedersen <br /> portant factors are expected to show stronger rank correla-                     2006). To examine whether the difference is due to the dif- <br /> tions with dN (Xia et al. 2009). We found that extracellularity                 ferent analytical approaches used, we applied the same an- <br /> is the most important factor in determining dN and dN/dS                        alytical procedures to the orthologs of yeast species <br /> among all the factors examined (table 1). Furthermore,                          Saccharomyces cerevisiae and S. paradoxus (see Materials <br /> the correlation between extracellularity and dN is not sub-                     and Methods). Our results for yeasts are consistent with <br />  <br />  <br /> 40            Genome Biol. Evol. 2(1):39&#226;&#8364;&#8220;43. 2010 doi:10.1093/gbe/evp058 Advance Access publication January 6, 2010 <br />  Protein Localization and Evolutionary Rate                                                                                                    GBE <br />  <br /> Table 1                                                                                      Although the reduction in sample size may have resulted in <br /> Rank Correlations of Various Factors with dN or dN/dS                                        weaker correlations, it should not have changed the relative <br />   Gene                     r (P value) for                      r (P value) for              importance of different factors as shown in tables 1 and 2. <br /> properties               correlation with dN                correlation with dN/dS               The different impacts of extracellularity on protein evolu- <br /> Mammals                                                                                      tionary rates in yeasts and mammals can potentially be ex- <br />   Extracell               0.177   (6.28e 11)                   0.166    (1.17e 09)           plained in two ways. First, extracellularity has qualitatively <br />   5#UTR                   0.144   (9.75e 08)                   0.133    (9.71e 07)           different meanings in these species because, for yeasts, se- <br />   Essen                   0.128   (2.21e 06)                   0.101    (1.88e 4) <br />                                                                                              creted proteins are outside the organisms, whereas for <br />   TissSpcf                0.122   (6.88e 06)                   0.096    (4.23e 4) <br />   KPPI                    0.104   (1.21e 4)                    0.103    (1.42e 4) <br />                                                                                              mammals, they are largely inside the organisms. However, <br />   3#UTR                   0.085   (1.71e 3)                    0.079    (3.71e 3)            this difference implies that properties of extracellular and in- <br />   Intron                  0.075   (5.72e 3)                    0.077    (4.48e 3)            tracellular proteins should be more similar in mammals than <br />   ExpLev                  0.038   (0.165)                      0.060    (0.028)              in yeasts, which does not explain our observation on the rate <br /> Yeasts                                                                                       of protein evolution. Second, secreted proteins involved in <br />   ExpLev                  0.541   (2.65e 215)                  0.473    (1.22e 158)          the biological processes that are present in mammals but <br />   Essen                   0.197   (2.73e 26)                   0.202    (1.57e 27) <br />                                                                                              not in yeasts evolve rapidly. However, figure 2 and supple- <br />   KPPI                    0.122   (5.80e 11)                   0.151    (7.36e 16) <br />   Extracell               0.022   (0.246)                      0.022    (0.232) <br />                                                                                              mentary table S2 (Supplementary Material online) showed <br />                                                                                              that, even within the same functional categories, extracel- <br />       NOTE.&#226;&#8364;&#8221;Extracell is 1 for extracellular proteins and 0 for intracellular proteins. <br /> Essen is 1 for essential genes and 0 for nonessential genes. &#226;&#8364;&#732;&#226;&#8364;&#732;UTR&#226;&#8364;&#8482;&#226;&#8364;&#8482; is UTR length and <br />                                                                                              lular proteins evolve faster than intracellular proteins, sug- <br /> &#226;&#8364;&#732;&#226;&#8364;&#732;Intron&#226;&#8364;&#8482;&#226;&#8364;&#8482; is average length per intron. &#226;&#8364;&#732;&#226;&#8364;&#732;KPPI&#226;&#8364;&#8482;&#226;&#8364;&#8482; is the number of interacting proteins.     gesting that the faster evolution of extracellular proteins <br /> &#226;&#8364;&#732;&#226;&#8364;&#732;TissSpcf&#226;&#8364;&#8482;&#226;&#8364;&#8482; is tissue-specificity. &#226;&#8364;&#732;&#226;&#8364;&#732;ExpLev&#226;&#8364;&#8482;&#226;&#8364;&#8482; is gene expression level. P values show the <br />                                                                                              is not attributable to special functions of these proteins. Be- <br /> probabilities of the observations under the hypothesis of no correlation. The analysis is <br /> based on 1,350 mouse&#226;&#8364;&#8220;rat orthologs or 2,840 Saccharomyces cerevisiae&#226;&#8364;&#8220;S. paradoxus            cause only a small fraction of mammalian genes are subject <br /> orthologs.                                                                                   to recurrent positive selection and genes most likely to be <br />                                                                                              subject to such selection (i.e., immunity genes) have been <br /> those previously published (table 1). That is, extracellularity                              removed from our analysis, the observed evolutionary rate <br /> has no effect on yeast protein evolutionary rate after the                                   difference between extracellular and intracellular proteins <br /> control of gene essentiality (Julenius and Pedersen 2006),                                   is most likely owing to differed purifying selection acting <br /> and expression level is the most important rate determinant                                  on them. But, the exact biological factors that cause the <br /> in yeast (Drummond et al. 2006). It should be noted that,                                    difference in purifying selection remain to be explored. <br /> compared with what was reported previously (Drummond <br /> and Wilke 2008), we observed a weaker correlation be- <br /> tween expression level and dN for mammalian proteins                                         Materials and Methods <br /> (tables 1 and 2). This difference is probably due to the small-                              The annotations, sequences, and orthologous relationships of <br /> er number of genes used here, as there are fewer genes with                                  mouse and rat genes were retrieved from Ensembl version 53 <br /> all the information needed in our partial correlation analysis.                              (www.ensembl.org), whereas those of yeast genes <br />                                                                                              were obtained from the Saccharomyces Genome Database <br /> Table 2                                                                                      (www.yeastgenome.org). Based on GO, immune-related <br /> Partial Rank Correlations of Various Factors with dN or dN/dS                                proteins have the annotation of GO:0002376 (immune system <br />                                                                                              process). Extracellularity was defined by the GO terms for cel- <br /> Extracell j controlled          r (P value) for                 r (P value) for <br />       property                correlation with dN           correlation with dN/dS           lular component (see supplementary fig. S1, Supplementary <br /> Mammals <br />                                                                                              Material online). Here,a GO term includes all its child GO terms. <br />   Extracell   j   Intron        0.175   (6.50e 11)              0.163   (1.30e 09)           Essentiality mouse genes were defined based on Mouse Ge- <br />   Extracell   j   ExpLev        0.174   (9.55e 11)              0.159   (3.16e 09)           nome Informatics 4.21 (www.informatics.jax.org), following <br />   Extracell   j   3#UTR         0.172   (1.63e 10)              0.160   (2.79e 09)           Liao and Zhang (2007). Essentialities and expression levels <br />   Extracell   j   KPPI          0.166   (6.21e 10)              0.154   (1.04e 08)           of yeast genes were obtained from Zhang and He (2005). <br />   Extracell   j   5#UTR         0.166   (7.03e 10)              0.154   (9.99e 09)           A zero fitness upon gene deletion is used to define essential <br />   Extracell   j   Essen         0.160   (2.96e 09)              0.151   (1.98e 08) <br />                                                                                              genes in both the yeast and mouse. Properties of mouse <br />   Extracell   j   TissSpcf      0.158   (4.67e 09)              0.150   (2.63e 08) <br /> Yeasts <br />                                                                                              gene expression were defined based on the microarray data <br />   Extracell   j ExpLev          0.043 (0.023)                   0.040 (0.035)                of 61 mouse tissues (Su et al. 2004). Expression level was cal- <br />   Extracell   j KPPI            0.015 (0.415)                   0.015 (0.439)                culated by averaging expression signals in the 61 tissues, <br />   Extracell   j Essen           0.011 (0.562)                   0.011 (0.547)                whereas tissue-specificity (s), which ranges from 0 to 1 (high- <br />      NOTE.&#226;&#8364;&#8221;See note of table 1 for Extracell, Essen, UTR, Intron, KPPI, TissSpcf, and        er values indicate stronger tissue-specificity), was calculated <br /> ExpLev. The factor before &#226;&#8364;&#732;&#226;&#8364;&#732;j&#226;&#8364;&#8482;&#226;&#8364;&#8482; is the factor being examined and that after &#226;&#8364;&#732;&#226;&#8364;&#732;j&#226;&#8364;&#8482;&#226;&#8364;&#8482; is the     according to Liao et al. (2006). Experimentally verified yeast <br /> factor being controlled for. P values show the probabilities of the observations under <br /> the hypothesis of no correlation. The analysis is based on 1,350 mouse&#226;&#8364;&#8220;rat orthologs or <br />                                                                                              protein&#226;&#8364;&#8220;protein interaction (PPI) data were obtained from <br /> 2,840 Saccharomyces cerevisiae&#226;&#8364;&#8220;S. paradoxus orthologs.                                       Batada et al. (2007), and thosefor humanwere compiledfrom <br />  <br />  <br /> Genome Biol. Evol. 2(1):39&#226;&#8364;&#8220;43. 2010 doi:10.1093/gbe/evp058 Advance Access publication January 6, 2010                                                     41 <br />  Liao et al.                                                                                                                              GBE <br />  <br />  <br />  <br />  <br />      FIG. 2.&#226;&#8364;&#8221;Higher dN/dS of extracellular proteins than intracellular proteins are observed even after the control of biological processes (A, B) or <br /> molecular functions (C, D). (A) and (C) were performed by comparing proteins of the same GOs at level 3 (left and right are top first and second <br /> differentially enriched terms, respectively), whereas (B) and (D) were performed by removing all genes with significantly differentially enriched GO terms <br /> at level 3 (left) or level 4 (right). See legend of figure 1 for details of the boxplot and supplementary figure S2 (Supplementary Material online) for the <br /> results of dN and dS. <br />  <br /> six sources: Human Protein Reference Database (www                      <br /> .hprd.org), Munich Information center for Protein Sequences              <br /> (mips.helmholtz-muenchen.de), Molecular INTeraction data-                         <br /> <span id='am-161' about='xsp:base' typeof='owl:Thing'>base</span> (mint.bio.uniroma2.it), Reactome (www.reactome.org),              <br /> IntAct (www.ebi.ac.uk), and Database of Interacting Proteins                      <br /> (dip.doe-mbi.ucla.edu). The human ortholog&#226;&#8364;&#8482;s number of                    <br /> interacting proteins (KPPI) was used as a proxy for a mouse                      <br /> protein&#226;&#8364;&#8482;s KPPI.                                                                    <br />    To calculate mammalian or yeast protein evolutionary <br />                                                                              <br /> rate, signal peptides, annotated by SPdb (proline.bic.nus                           <br /> .edu.sg/spdb/), were removed. Orthologous coding sequen-                         <br /> ces without signal peptides were aligned following the                       <br /> protein alignment by ClustalW (www.ebi.ac.uk/clustalw/).                             <br /> When a gene has multiple isoforms, the longest isoform                              <br /> was used. Values of dN and dS between mouse and rat                              <br /> and between S. cerevisiae and S. paradoxus were computed                           <br /> using PAML 4 (Yang 2007). <br />   <br /> </body></html>