<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br />  <br />  <br /> Cooperative Assembly of CYK-4/MgcRacGAP and <br /> ZEN-4/MKLP1 <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> Form the Centralspindlin Complex <br />  <br />  <br />  <br />  <br /> Cytokinesis in metazoan cells requires a set of antiparallel microtubules that become bundled upon anaphase onset to <br /> form a structure known as the central spindle. Bundling of these microtubules requires a protein complex, centralspin- <br /> dlin, that consists of the CYK-4/MgcRacGAP Rho-family GTPase-activating protein and the ZEN-4/MKLP1 kinesin-6 <br /> motor protein. Centralspindlin, but not its individual subunits, is sufficient to bundle microtubules in vitro. Here, we <br /> present a biochemical and genetic dissection of centralspindlin. We show that each of the two subunits of centralspindlin <br /> dimerize via a parallel coiled coil. The two homodimers assemble into a high-affinity heterotetrameric complex by virtue <br /> of two low-affinity interactions. Conditional mutations in the regions that mediate complex assembly can be readily <br /> suppressed by numerous second site mutations in the interacting regions. This unexpected plasticity explains the lack of <br /> primary sequence conservation of the regions critical for this essential protein&#226;&#8364;&#8220;protein interaction. <br />  <br />  <br /> INTRODUCTION                                                           (Mollinari et al., 2002, 2005; Kurasawa et al., 2004; Verbrug- <br />                                                                        ghe and White, 2004; Verni et al., 2004) in part through the <br /> Cell division in animal cells is dependent upon a structure            ability of this factor to bind to the kinesin-4 motor protein <br /> known as the central spindle. As cells initiate anaphase, this         Kif4 (Kurasawa et al., 2004; Mollinari et al., 2005). In vitro <br /> structure assembles between the segregating chromosomes.               microtubule bundling activity has been demonstrated for both <br /> The central spindle consists of antiparallel microtubules              centralspindlin and PRC1 (Mishima et al., 2002; Mollinari et al., <br /> (MTs) that are bundled at their plus ends, through the action          2002); however, these two complexes are each required for <br /> of microtubule-associated proteins (MAPs) and kinesin fam-             the assembly of robust, organized microtubule bundles, <br /> ily motors (for review, see Glotzer, 2005). Successful com-            with these components at restricted, well-defined positions <br /> pletion of cytokinesis in animal cells requires the bundled            within the central spindle. These complexes not only act to <br /> microtubules of the central spindle (Wheatley and Wang,                organize microtubule bundles but also serve to recruit ad- <br /> 1996). In addition, the central spindle plays an important             ditional factors that regulate initiation of cytokinesis and a <br /> role in regulating cleavage plane positioning (Dechant and             variety of components that facilitate completion of cytoki- <br /> Glotzer, 2003; Bringmann and Hyman, 2005; Werner et al.,               nesis, or abscission (Gromley et al., 2005). <br /> 2007).                                                                    Centralspindlin is of particular interest among these com- <br />    Central spindle assembly is mediated by several distinct            plexes because the CYK-4 subunit contains a GAP domain <br /> protein complexes that act cooperatively to bundle antipar-            that can induce GTP hydrolysis by RhoA, Rac, and Cdc42, at <br /> allel microtubules during anaphase. These complexes in-                varying efficiencies (Toure et al., 1998; Jantsch-Plunger et al., <br /> clude centralspindlin, which contains a kinesin-6 motor pro-           2000). In addition, CYK-4 also directly binds the guanine <br /> tein, ZEN-4/MKLP1, and a Rho-family GTPase-activating                  nucleotide exchange factor (GEF) for RhoA, ECT2 (Yuce et <br /> protein (GAP) protein, CYK-4/MgcRacGAP (Mishima et al.,                al., 2005; Zhao and Fang, 2005; Kamijo et al., 2006; Nishimura <br /> 2002). Localization of centralspindlin requires the multisub-          and Yonemura, 2006), but ECT2 is not a stoichiometric com- <br /> <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> Aurora B kinase complex, which may play a direct role             ponent of this complex. Cells depleted of the kinesin com- <br /> in microtubule bundling (Kaitna et al., 2000; Severson et al.,         ponent of centralspindlin, ZEN-4/MKLP-1, often initiate <br /> 2000; Pereira and Schiebel, 2003). Finally, the PRC1 MAP is            cleavage furrow formation, but they invariably fail to com- <br /> also implicated in microtubule bundling during anaphase                plete cytokinesis. However, human cells depleted of the <br />                                                                        RhoGAP subunit of centralspindlin, CYK-4, do not even <br />                                                                        initiate cytokinesis (Yuce et al., 2005; Zhao and Fang, 2005; <br /> This article was published online ahead of print in MBC in Press       Kamijo et al., 2006; Nishimura and Yonemura, 2006). Because <br /> (http://www.molbiolcell.org/cgi/doi/10.1091/mbc.E07&#226;&#8364;&#8220; 05&#226;&#8364;&#8220; 0468) <br /> on October 17, 2007. <br />                                                                        centralspindlin controls furrow initiation by promoting <br />                                                                        RhoA activation and regulates completion of cytokinesis, it <br /> &#226;&#8211;&#161; <br /> D &#226;&#8211;&#161; V <br />       The online version of this article contains supplemental mate-   is important to define its biochemical function and the mech- <br /> rial at MBC Online (http://www.molbiolcell.org). <br />                                                                        anism by which it becomes so discretely localized on the <br /> Address correspondence to: Michael Glotzer (mglotzer@uchicago.edu).    central spindle. <br /> Abbreviations used: GAP, GTPase-activating protein; ITC, isother-         The components of centralspindlin have been purified to <br /> mal titration calorimetry; MAP, microtubule-associated protein;        near homogeneity under stringent conditions. The two sub- <br /> MT, microtubule; SPR, surface plasmon resonance; wt, wild type.        units are present in stoichiometric amounts, with the native <br />  <br /> 4992                                                                                    &#194;&#169; 2007 by The American Society for Cell Biology <br />                                                                                                                                   Assembly of Centralspindlin <br />  <br />  <br /> molecular weight of centralspindlin being &#226;&#172;&#402;300 kDa (Mishima                         SDS-polyacrylamide gel electrophoresis (PAGE) and Coomassie Blue stain- <br /> et al., 2002). Under stringent purification conditions, no addi-                    ing. <br /> tional factors copurified in significant amounts, suggesting that                   In Vitro Translation and Binding Assay <br /> the complex is a tetramer. Because each component can indi- <br />                                                                                     CYK-4 and ZEN-4 were expressed by in vitro transcription and translation <br /> vidually oligomerize, the centralspindlin tetramer is likely a                      system with reticulocyte lysate by using the TNT-coupled reticulocyte lysate <br /> heterotetramer. The complex has been reconstituted <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> re-                         system (Promega, Madison, WI) in a 20-&#226;&#65533;&#174;l reaction. Protein fragments of <br /> combinant sources and the regions required for the interaction                      CYK-4 or ZEN-4 tagged with CBD at the N terminus were expressed from the <br /> between CYK-4 and ZEN-4 have been roughly mapped. Mi-                               T7 promoter of pCBD-TEV. After translation, the binding reaction was set up <br />                                                                                     in 100 &#226;&#65533;&#174;l of buffer B and incubated at 20&#194;&#176;C for 30 min. ZEN-4 protein was <br /> crotubule bundling by centralspindlin requires the presence of                      immunoprecipitated with specific antibodies, by addition of 0.2 &#226;&#65533;&#174;g antibody <br /> both subunits and their ability to interact (Mishima et al., 2002).                 to the reactions and incubated on ice for 1 h, followed by incubation with 5 &#226;&#65533;&#174;l <br /> The nature of the interaction domain is particularly intriguing,                    of protein A-Sepharose beads at 4&#194;&#176;C for 1 h. The beads were subsequently <br /> because it is not well conserved at the primary sequence level,                     washed with buffer B. Proteins bound to the beads were analyzed by SDS- <br />                                                                                     PAGE followed by autoradiography with a phosphorimager. Buffer B consists <br /> even though the interaction between these proteins has been                         of 20 mM HEPES, pH 7.7, 10 mM EDTA, 2 mM MgCl2, 0.5% (wt/vol) Triton <br /> documented in a variety of metazoan species (Mishima et al.,                        X-100, 150 mM NaCl2, 10 &#226;&#65533;&#174;g ml&#226;&#171;&#186;1 leupeptin/pepstatin, 1 mM DTT, and 1 mM <br /> 2002; Somers and Saint, 2003).                                                      PMSF. <br />    Although the broad outlines of how the centralspindlin <br /> complex assembles and bundles microtubules have been                                Microtubule Bundling Assay <br /> defined, critical questions remain. In particular, structural                       The centralspindlin complex was reconstituted by mixing equimolar (&#226;&#172;&#402;1 &#226;&#65533;&#174;M) <br />                                                                                     amounts of recombinant CYK-4 and ZEN-4 and incubating for 20 min at room <br /> and functional studies would benefit from a description of                          temperature. For microtubule bundling assay, samples were mixed with <br /> the minimal interacting domains of CYK-4 and ZEN-4. In                              Taxol-stabilized microtubules (final 4 &#226;&#65533;&#174;M) in buffer C and incubated for 20 <br /> addition, the question of whether their interaction requires                        min at room temperature (Mishima et al., 2002). The reaction was fixed with <br /> dimeric forms of CYK-4 and ZEN-4 remains open. To ad-                               glutaraldehyde, and the microtubules were sedimented on a coverslip as <br />                                                                                     described previously (Desai and Walczak, 2001). Microtubules were fixed in <br /> dress these issues, we have conducted a detailed structure&#226;&#8364;&#8220;                         cold methanol and stained with anti-tubulin monoclonal antibody (DM1&#226;&#65533;&#163;), <br /> function analysis of centralspindlin. We demonstrate that                           and images were acquired with a 63&#226;&#171;&#187;/1.4 numerical aperture objective on a <br /> both CYK-4 and ZEN-4 form parallel coiled coils, and we                             Zeiss Axioplan II and a CoolSnapFX charge-coupled device camera that was <br /> show that their ability to independently dimerize contrib-                          controlled with MetaMorph (Molecular Devices, Sunnyvale, CA). The quan- <br />                                                                                     tification of results was performed by analyzing 10 random fields and inte- <br /> utes substantially to stabilization of centralspindlin. We                          grating pixel intensities above a fixed threshold for all images that excluded <br /> show that CYK-4 and ZEN-4 form a high-affinity complex                              single microtubules. Buffer C consists of 125 mM KCl, 80 mM PIPES, 1 mM <br /> and that both of the temperature-sensitive mutations in                             EGTA, 1 mM MgCl2, 1 mM DTT, and 2 mM adenyl-5&#226;&#172;&#732;-yl imidodiphosphate, <br /> these genes destabilize this protein complex. Through the                           pH 6.8. <br /> use of genome-wide genetic screens, we have identified <br />                                                                                     Surface Plasmon Resonance <br /> additional critical features of the interaction interface. <br />                                                                                     The affinity and the rates of interaction between ZEN-4 and CYK-4 were <br />                                                                                     measured with a BIAcore 3000 biosensor (BIAcore, Uppsala, Sweden). CYK-4, <br /> MATERIALS AND METHODS                                                               with an N-terminal GST tag, was immobilized to the sensor chip (CM5) via an <br />                                                                                     anti-GST antibody that had been amine coupled with the dextran matrix in 10 <br /> Nematode Strains and Alleles                                                        mM sodium acetate, pH 5.0. The control surface was prepared similarly <br />                                                                                     except that CYK-4 GST was not immobilized on the chip. Kinetic experiments <br /> The following strains or alleles were used in this study: N2 (Bristol), zen-        were performed at 16 and 25&#194;&#176;C, by using flow rates of 5&#226;&#8364;&#8220;20 &#226;&#65533;&#174;l min&#226;&#171;&#186;1. ZEN-4 <br /> 4(or153ts), EH135 (unc-44(e362) bli-6(sc16) IV), SU62 (unc-44(e362) lag-1 (q385)/   was added into the flow in buffer A (but with 25 &#226;&#65533;&#174;M ATP) at various <br /> zen-4(w35) IV), and MG367 (zen-4(w35) bli-6(sc16)/unc-44(el260) lag-1(q385)).       concentrations (3&#226;&#8364;&#8220;300 nM) during the binding phase. The amount of protein <br /> Some strains in this study were obtained from Caenorhabditis Genetics Center,       bound to the sensor chip was monitored by the change in refractive index <br /> which is funded by the National Institutes of Health National Center for            (given in arbitrary units). For the measurement of kinetic parameters, the <br /> Research Resources. For rescue assays, simple transgenic arrays were pre-           amount of immobilized GST-CYK-4 was minimized, to avoid saturation and <br /> pared in MG367 as described previously (Mello and Fire, 1995) by using pRF4         avidity effects. After each binding experiment, the sensor chip was regener- <br /> at 80 ng &#226;&#65533;&#174;l&#226;&#171;&#186;1 and MP66 (zen-4), VP01 (zen-4 D520N), and VP02 (zen-4 D735N)          ated by sequential washing with 10 mM glycine, pH 2.0, followed by two <br /> at 10 ng &#226;&#65533;&#174;l&#226;&#171;&#186;1.                                                                      washes with running buffer. Triplicate injections of each concentration of <br />                                                                                     ZEN-4 were performed. For data analysis, BIAevaluate software (BIAcore) <br /> Isolation of zen-4 (or153ts) Suppressors                                            was used, and the sensorgram association and dissociation curves were <br /> Suppressors of zen-4(or153ts) mutations were obtained by mutagenesis of             locally or globally fit to a two-site model. Running buffer consists of buffer A <br /> zen-4(or153ts) animals with 30 &#226;&#65533;&#174;M ethane methyl sulfonate (EMS). After              without protease inhibitors or DTT. <br /> recovery, F1 embryos were isolated by bleaching and allowed to self-fertilize <br /> at permissive temperature (16&#194;&#176;C). When F2 animals reached adulthood, the            Isothermal Titration Calorimetry <br /> population was shifted to 25&#194;&#176;C, and fertile animals were selected, a single         Binding of monomeric ZEN-4 to dimeric CYK-4 was measured using a <br /> isolate was kept from each pool of F1 embryos. Approximately 500,000 F1             MicroCal VP isothermal titration calorimeter. Protein samples used for the <br /> genomes were screened, and 14 suppressor mutations were isolated, all of            titration were prepared by dialyzing overnight against 2 l of working buffer <br /> which were viable at 25&#194;&#176;C.                                                          A without Triton X-100 and protease inhibitors. Measurements were per- <br />                                                                                     formed at 25&#194;&#176;C by using the working buffer A minus Triton X-100 and <br /> Protein Preparation                                                                 protease inhibitors as a reference. A sample cell containing dimeric CYK-4 (10 <br />                                                                                     &#226;&#65533;&#174;M) was titrated by repeated injection of monomeric ZEN-4 (100 &#226;&#65533;&#174;M). The <br /> ZEN-4 fragments were cloned into pCBD-TEV and expressed in Escherichia <br />                                                                                     collected data were analyzed with the Origin software (OriginLab, Northampton, <br /> coli as N-terminal chitin binding domain (CBD) fusion proteins. CYK-4 frag- <br />                                                                                     MA) by using a simple single-site model. <br /> ments were cloned into pGEX-4-TEV and expressed in E. coli as N-terminal <br /> glutathione transferase (GST) fusion proteins. Lysis buffer (buffer A) con- <br /> tained 10 mM HEPES, pH 7.7, 1 mM EGTA, 1 mM MgCl2, 0.1% (wt/vol)                    Binding Studies <br /> Triton X-100, 250 mM NaCl, 10 &#226;&#65533;&#174;g ml&#226;&#171;&#186;1 leupeptin/pepstatin, 1 mM dithio-             Binding of different CYK-4 truncation products was measured by a standard <br /> threitol (DTT), and 1 mM phenylmethylsulfonylfluoride (PMSF). Extracts              pull-down assay. CYK-4 fragments were purified as maltose binding protein <br /> were clarified by centrifugation at 19,000 rpm, and the recombinant proteins        (MBP) fusions, and they were retained on the amylose beads. Recombinant <br /> were affinity purified with chitin or glutathione Sepharose beads, and eluted       ZEN-4(1-585)GFP was added at different concentrations to the beads in a <br /> with tobacco etch virus (TEV) protease.                                             100-&#226;&#65533;&#174;l reaction, and it was incubated at room temperature for 2 h in buffer B. <br />    For bicistronic expression of ZEN-4 and CYK-4, the fragments were cloned         In the case of CYK-4(1-120) and CYK-4(1-36)GCN-4, the fluorescence remain- <br /> into pGEX-6, as N-terminal CBD fusions of ZEN-4 and N-terminal GST                  ing in the supernatant after incubation was measured with a Safire2 (Tecan, <br /> fusions of CYK-4 (Sessa et al., 2005). After E. coli expression, GST-CYK-4 was      Reading, United Kingdom) microplate reader. The bound fraction was calcu- <br /> purified on glutathione Sepharose beads and eluted with reduced glutathi-           lated by subtracting the unbound fraction from the total. In the case of <br /> one, followed by incubation of the eluates with chitin beads. The recombinant       CYK-4(1-36), the bound fraction was analyzed by SDS-PAGE and Coomassie <br /> proteins were eluted from chitin beads with TEV protease and analyzed by            Blue staining, followed by quantification of band intensities with the Odyssey <br />  <br />  <br /> Vol. 18, December 2007                                                                                                                                        4993 <br />  V. Pavicic-Kaltenbrunner et al. <br />  <br /> (Li-Cor, Lincoln, NE) infrared imaging system. In both cases, a graph of          the primary function of centralspindlin is to bundle micro- <br /> bound fractions versus increasing concentrations of ligand was generated,         tubules of the central spindle. We performed a series of in <br /> and the KD was estimated using nonlinear regression in which the data were <br /> fit to an equation describing a rectangular hyperbola (y &#226;&#171;&#189; Bmax &#228;&#161;&#160; x/(KD &#226;&#171;&#185; x)).    vitro MT bundling assays, comparing ZEN-4(1-603) and <br /> In addition, the data were plotted on a Scatchard plot, and the KD value was      ZEN-4(1-603)D520N in complexes with CYK-4(1-232). As <br /> estimated from the slope (&#226;&#171;&#186;1/KD). Similar results were obtained by the two        shown previously, a complex containing ZEN-4(1-603) and <br /> methods. Protein concentrations were estimated by the Bradford assay (Bio-        CYK-4(1-232) is sufficient to bundle microtubules (Mishima <br /> Rad, Hercules, CA) using bovine serum albumin as a standard. Binding of <br /> different ZEN-4 products was also measured by a standard pull-down assay.         et al., 2002). In contrast, a complex containing ZEN-4 D520N <br /> CYK-4 fragments were purified as GST fusions and retained on the glutathi-        and CYK-4(1-232) cannot bundle microtubules (Figure 1B). <br /> one-Sepharose beads. Recombinant wild-type ZEN-4(1-585), ZEN-4(1-                 This assay requires complex formation between CYK-4 and <br /> 585)D520N, ZEN-4(1-539), and ZEN-4(1-539)GCN4 were added to the beads             ZEN-4, because CYK-4 S15L, the product of the tempera- <br /> in a 100-&#226;&#65533;&#174;l reaction and incubated for 2 h at 25&#194;&#176;C in buffer A without protease <br /> inhibitors or DTT. The beads were subsequently washed two times, and the          ture-sensitive allele cyk-4(t1689ts), which is defective in <br /> bound fraction was analyzed by SDS-PAGE and Coomassie Blue staining.              ZEN-4 binding, is similarly defective in the in vitro MT <br />                                                                                   bundling assay (Mishima et al., 2002). These data indicate <br /> Gel Filtration Chromatography                                                     that the <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> of zen-4(or153ts) is likely to be directly <br /> The diffusion coefficients of different recombinant ZEN-4 fragments were          linked to its inability to bundle microtubules. <br /> analyzed by size exclusion chromatography using a Superdex 200 column (30 <br /> ml) on an A&#194;&#180; KTA FPLC system (GE Healthcare, Chalfont St. Giles, United <br />                                                                                   ZEN-4 D520N Is a Dimer <br /> Kingdom) in buffer A (250 mM KCl and 25 &#226;&#65533;&#174;M ATP, minus Triton X-100, <br /> leupeptin/pepstatin, and PMSF). The fractions were analyzed by SDS-PAGE           In previous studies, binding of CYK-4 and ZEN-4 was not <br /> and Coomassie Blue staining.                                                      separable from ZEN-4 dimerization (Mishima et al., 2002). <br />                                                                                   Because D520N maps within the previously predicted coiled <br /> Cross-Linking                                                                     coil domain (as predicted by COILS; Lupas et al., 1991), we <br /> Recombinant ZEN-4 and CYK-4 fragments containing cysteine residues at N           tested whether ZEN-4 D520N is a dimer or a monomer <br /> or C termini were incubated in a buffer A (minus ATP, Triton X-100, leupep- <br /> tin/pepstatin, and PMSF), containing 5 mM oxidized and 4 mM reduced               (Figure 1C). Therefore, we compared the behavior of ZEN- <br /> glutathione. The reaction was quenched with 100 mM iodoacetamide and              4(1-603), a dimer, ZEN-4(1-603)D520N, and ZEN-4(1-434), <br /> analyzed by SDS-PAGE and Coomassie Blue staining.                                 which is a monomer, on size exclusion chromatography. We <br />                                                                                   found that ZEN-4(1-603)D520N elutes in the same fraction <br /> RESULTS                                                                           as ZEN-4(1-603), indicating that D520N does not prevent <br />                                                                                   dimerization (Figure 2A). Thus, the primary defect in zen- <br /> Characterization of zen-4(or153ts)                                                4(or153ts) embryos is not due to the failure of ZEN-4 to form <br /> To gain further insight into the nature of centralspindlin, we                    dimers. <br /> took a genetic approach starting with the previously identified <br /> temperature-sensitive loss-of-function allele zen-4(or153ts). zen-                ZEN-4 D520N Is Defective in CYK-4 Binding <br /> 4(or153ts) animals exhibit penetrant embryonic lethality at the                   Thus far, we have established that ZEN-4 D520N is defective <br /> restrictive temperature (25&#194;&#176;C), but they are completely viable                    in MT bundling, yet it is still able to self-associate into <br /> and have a high brood size at the permissive temperature                          dimers. Given this, we turned our attention to the ability of <br /> (16&#194;&#176;C) (Severson et al., 2000). Because this is a conditional,                    the mutant ZEN-4 to bind CYK-4, because D520N maps near <br /> loss-of-function allele, we reasoned that suppressors of the                      the previously determined ZEN-4/CYK-4 binding interface. <br /> high temperature lethality could lead to the identification of                    We compared the CYK-4 binding activity of ZEN-4(434-775) <br /> novel factors that act in conjunction with centralspindlin or                     and ZEN-4(434-775)D520N. Whereas ZEN-4(434-775) inter- <br /> identify key residues within the complex that stabilize its ac-                   acts strongly with CYK-4, ZEN-4(434-775)D520N does not <br /> tivity, either directly or indirectly.                                            (Figure 2B). To gain further insight into how structure affects <br />    Because the zen-4(or153ts) allele contains two mutations,                      the function of centralspindlin, we measured the binding <br /> each of which encodes for a Asp (D)-to-Asn (N) substitution                       affinities of wild-type and mutant centralspindlin complex, <br /> at positions 520 and 735 (Severson et al., 2000), we first                        by using surface plasmon resonance (SPR) technology. CYK- <br /> wanted to ascertain whether one of the two mutations is                           4(1-120) was immobilized on the chip matrix, and ZEN-4(1- <br /> primarily responsible for the phenotype. We generated mul-                        603) flowed free in solution. We measured binding kinetics <br /> tiple independent transgenic lines expressing either wild-                        at 25&#194;&#176;C [the nonpermissive temperature for zen-4(or153ts)] <br /> type ZEN-4, ZEN-4 D520N, or ZEN-4 D735N, in a back-                               and at 16&#194;&#176;C (the permissive temperature). The affinity of <br /> ground containing the null zen-4(w35) allele, which causes                        wild-type (wt) ZEN-4 for CYK-4 was 10 nM at 25&#194;&#176;C, com- <br /> embryonic lethality associated with cytokinesis defects; zen-                     pared with ZEN-4 D520N that had an affinity of 668 nM for <br /> 4(w35) was linked to the visible recessive marker bli-6. We                       CYK-4 (Figure 2C). At the permissive temperature, ZEN-4 <br /> assayed for the ability of the transgene to rescue the null                       D520N bound CYK-4 with an affinity of 118 nM, revealing <br /> allele, which could be conveniently scored by counting blis-                      that the interaction of this ZEN-4 variant and CYK-4 is <br /> tered animals. We found that both ZEN-4 and ZEN-4 D735N                           temperature-sensitive in vitro. This result strongly indicates <br /> could rescue zen-4(w35) at 25&#194;&#176;C, whereas ZEN-4 D520N                              that the destabilization of the complex formation between <br /> could not (Figure 1A). Therefore, the D520N <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span>point mutation</span>                        ZEN-4 D520N and CYK-4 is the primary defect in zen- <br /> in zen-4(or153ts) is responsible for causing the mutant phe-                      4(or153ts) animals. <br /> notype. Additionally, ZEN-4 D520N could rescue the null <br /> allele at 16&#194;&#176;C, the permissive temperature for zen-4(or153ts),                    Isolation of zen-4(or153ts) Suppressors <br /> indicating that this substitution is solely responsible for the                   To identify mutations in the C. elegans genome that are <br /> conditional nature of this allele.                                                capable of restoring viability to zen-4(or153ts) animals, we <br />                                                                                   mutagenized large numbers of zen-4(or153ts) animals and <br /> ZEN-4 D520N Is Defective in MT Bundling                                           propagated them for two generations at the permissive tem- <br /> Before performing the screen, we wanted to identify the                           perature (16&#194;&#176;C). When the F2 animals reached adulthood, <br /> biochemical nature of the defect caused by the D520N sub-                         the plates were shifted to 25&#194;&#176;C to impose a selection for <br /> stitution <span id='am-239' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-240' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-241' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-243' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-253' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-257' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-273' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-277' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-278' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-284' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-299' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span>mutation</span>. The first question we asked was whether                       fertile animals carrying putative suppressor mutations (Fig- <br /> ZEN-4 D520N is defective in microtubule bundling, because                         ure 3A). Fourteen independent suppressors were identified. <br />  <br /> 4994                                                                                                                 Molecular Biology of the Cell <br />                                                                                                                    Assembly of Centralspindlin <br />  <br />  <br />  <br />  <br /> Figure 1. ZEN-4 D520N impairs microtubule bundling in vitro and is responsible for the in vivo phenotype of zen-4(or153ts). (A) <br /> zen-4(or153ts) D520N fails to rescue zen-4(w35) at 25&#194;&#176;C. Simple transgenic arrays marked by rol-6(su1006) were prepared and the efficiency <br /> of zen-4 D520N and D735N versus wt ZEN-4 to rescue zen-4(w35) was assayed. The rescue efficiency was scored by counting the blistered <br /> [bli-6(sc16)] animals, both at the restrictive (25&#194;&#176;C; gray) and the permissive temperature (16&#194;&#176;C; black) as described previously (Mishima et al., <br /> 2004). At 25&#194;&#176;C ZEN-4 D520N could not rescue the null, whereas both wt and ZEN-4 D735N efficiently rescued the null. (B) The D520N point <br /> mutation in zen-4(or153ts) inhibits MT bundling. MT bundling assays were performed with constituents of centralspindlin alone, and with <br /> reconstituted centralspindlin complex, comparing wt and mutant complexes. The quantification of results was performed by the analysis of <br /> 10 random fields, followed by integrating pixel intensities above a fixed threshold. (C) Schematic representation of ZEN-4 (light gray) and <br /> CYK-4 (dark gray) proteins. The CYK-4/ZEN-4 interaction domains are indicated with a dotted line above the illustrations. The plots <br /> represent the predictions of the coiled coils of ZEN-4 and CYK-4, by COILS (Lupas et al., 1991) and PAIRCOILS (Berger et al., 1995). <br />  <br />  <br /> Before performing crosses to map the putative suppressors,                 from this screen provide further proof that the interaction <br /> we sequenced the ZEN-4 and CYK-4 loci from PCR products                    between ZEN-4 and CYK-4 is fundamental for the in vivo <br /> amplified from the suppressor strains to determine whether                 function of centralspindlin. Interestingly, two mutations, <br /> any of the suppressors contain intragenic mutations in zen-4               D31N in CYK-4 and E502K within ZEN-4, were found as <br /> or extragenic mutations in cyk-4, as a related screen using                suppressors of both cyk-4(t1689ts) and zen-4(or153ts). Be- <br /> cyk-4(t1689) resulted in many such mutations (Mishima et al.,              cause we have determined that the primary defect in both of <br /> 2002). Surprisingly, each of the suppressor mutants con-                   these alleles is a defect in complex formation, we expect that <br /> tained a single <span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span>missense mutation</span> in either the CYK-4 inter-               these mutations may stabilize the complex. To directly test this <br /> action region of ZEN-4 or the ZEN-4 interaction region of                  conjecture, we assessed whether some of the extragenic sup- <br /> CYK-4 (Figure 3B). The use of EMS as a mutagen likely                      pressor mutations in cyk-4 that rescue zen-4(or153ts) restore <br /> prevented the isolation of simple revertants of the ZEN-4                  binding to ZEN-4 D520N. Indeed, the binding of ZEN-4 <br /> D520N point mutation, because it induces G/C-A/T transi-                   D520N to CYK-4 G12S and CYK-4 G12D was significantly <br /> tions (Anderson, 1995). Several criteria suggest that these                improved compared with wild-type CYK-4 (Figure 3C). Inter- <br /> mutations were responsible for the genetic suppression                     estingly, the binding of these variants of CYK-4 to wild-type <br /> rather than bystander mutations from the mutagenesis.                      ZEN-4 was unimpaired (Figure 3C). <br /> First, each strain contained a single substitution in either <br /> cyk-4 or zen-4; second, all mutations were substitution mu- <br /> tations, no silent mutations or mutations in introns were                  ZEN-4 Dimerization Is Separable from CYK-4 Binding <br /> identified; and third, all of the mutations fell near the region           The results from this screen prompted a reevaluation of the <br /> previously implicated in complex formation. As evidence                    coiled coil boundaries within ZEN-4. In previous studies we <br /> that the screen was extensive, three of six individual muta-               used the COILS algorithm to predict that the ZEN-4 coiled <br /> tions were independently isolated multiple times. The data                 coil began at residue 508 (Lupas et al., 1991). The finding that <br />  <br /> Vol. 18, December 2007                                                                                                                      4995 <br />  V. Pavicic-Kaltenbrunner et al. <br />  <br />  <br />                                                                        D520N impairs complex formation yet retains the ability to <br />                                                                        dimerize suggests that parts of the coiled coil may be di- <br />                                                                        rectly involved in CYK-4 binding, or possibly the prediction <br />                                                                        of the coiled coil domain could have been incorrect. To test <br />                                                                        this latter possibility, we constructed a series of deletion <br />                                                                        alleles of ZEN-4 and examined their oligomeric state by <br />                                                                        performing gel filtration chromatography. We found that <br />                                                                        ZEN-4(1-539), lacking the predicted coiled coil region, <br />                                                                        eluted significantly later from a gel filtration column com- <br />                                                                        pared with ZEN-4(1-555), which behaved similarly to ZEN- <br />                                                                        4(1-603), which is a dimer (Figures 2A and 4A). ZEN-4(1- <br />                                                                        539) and smaller truncations eluted from the column at the <br />                                                                        position predicted for a monomeric globular protein (Figure <br />                                                                        4B). Interestingly, the PAIRCOILS algorithm predicts that <br />                                                                        the coiled coil starts at residue 537, rather than at residue 507 <br />                                                                        as predicted by COILS (Lupas et al., 1991; Berger et al., 1995) <br />                                                                        (Figure 1C). These data suggest that the coiled coil is likely <br />                                                                        to start at approximately position 540, and this places the <br />                                                                        critical residue, D520, within the noncoiled coil CYK-4 in- <br />                                                                        teraction domain. <br />                                                                           We next sought to test whether the monomeric form of <br />                                                                        ZEN-4, ZEN-4(1-539), is competent to bind to CYK-4. We <br />                                                                        therefore coexpressed GST-CYK-4 in a bicistronic operon <br />                                                                        with CBD-ZEN-4 in E. coli, and performed sequential affinity <br />                                                                        chromatography steps. Monomeric ZEN-4(1-539) copurified <br />                                                                        with stoichiometric amounts of CYK-4 as was observed with <br />                                                                        dimeric ZEN-4(1-555) or ZEN-4(1-601) (Figure 4C). Al- <br />                                                                        though monomeric ZEN-4 binds to CYK-4 with lower affin- <br />                                                                        ity compared with dimeric ZEN-4 (see below), these data <br />                                                                        establish that ZEN-4 dimerization is not strictly essential for <br />                                                                        CYK-4 binding. <br />  <br />                                                                        CYK-4 and ZEN-4 Are Parallel Coiled Coil Dimers <br />                                                                        The arrangement of the individual subunits within the na- <br />                                                                        tive centralspindlin complex is a feature that may govern <br />                                                                        mechanistic attributes of central spindle assembly, such as <br />                                                                        the mechanism of bundling antiparallel microtubules. We <br />                                                                        therefore wanted to determine whether the coiled coils in <br />                                                                        ZEN-4 and CYK-4 are in a parallel or an antiparallel config- <br />                                                                        uration. The fact that ZEN-4(1-555) is a dimer indicates that <br />                                                                        the coiled coil in ZEN-4 is in a parallel configuration, be- <br />                                                                        cause this truncation roughly bisects the coiled coil region <br />                                                                        and were it in an antiparallel configuration, and in register, <br />                                                                        this truncation would eliminate the overlap. To indepen- <br />                                                                        dently address the question, we used disulfide bridge for- <br />                                                                        mation under oxidizing conditions followed by nonreducing <br />                                                                        gel electrophoresis to assay for dimer formation and thereby <br />                                                                        infer the arrangement of the coiled coils. ZEN-4 contains a <br />                                                                        cysteine at position 547 that is predicted to be in the &#226;&#8364;&#339;a&#226;&#8364;&#65533; <br />                                                                        register (i.e., internal) of the coiled coil. If the helices in the <br />                                                                        ZEN-4 dimer are antiparallel, there is only a low probability <br />  <br />  <br />  <br />                                                                        proteins by in vitro translation. Also shown (far right lanes) are <br />                                                                        parallel reactions of ZEN-4 translation reactions with [35S]methi- <br />                                                                        onine, as controls for the efficiency of translation. Unlabeled ZEN- <br />                                                                        4(434-775) wt and D520N fragments, and the negative control <br />                                                                        &#226;&#338;&#172;ZEN-4(475-775), were added and incubated for 1 h. ZEN-4 was <br />                                                                        subsequently immunoprecipitated with an anti-ZEN-4 antibody. <br />                                                                        Whereas wt ZEN-4 can bind wt CYK-4, ZEN-4 D520N cannot. <br />                                                                        Therefore, ZEN-4 D520N is defective in CYK-4 binding, yet is a <br /> Figure 2. The primary defect of zen-4(or153ts) is due to the failure   dimer (A). (C) The affinity and the rates of interaction between <br /> of ZEN-4 D520N to form a stable complex with CYK-4. (A) The            ZEN-4 and CYK-4 were measured through SPR. Sensorgrams of <br /> ability of ZEN-4 D520N to form a dimer was tested by size exclusion    triplicate runs representing 100 nM ZEN-4 injected are shown. The <br /> chromatography. ZEN-4(1-603)D520N eluted in the same fraction as       affinity of ZEN-4 D520N for CYK-4 is reduced by 60-fold from wt <br /> wt ZEN-4(1-603), thus ZEN-4 D520N is a dimer. (B) CYK-4(1-120)         ZEN-4/CYK-4 affinity at 25&#194;&#176;C. However, at 16&#194;&#176;C the mutant&#226;&#8364;&#8482;s af- <br /> and &#226;&#338;&#172;CYK-4(34-120) fragments were expressed as 35S-labeled             finity for CYK-4 improves. <br />  <br /> 4996                                                                                                        Molecular Biology of the Cell <br />                                                                                                         Assembly of Centralspindlin <br />  <br />  <br />  <br />  <br />                                                                                       Figure 3. Isolation of zen-4(or153ts) suppres- <br />                                                                                       sors. (A) Schematic view of the zen-4(or153ts) <br />                                                                                       suppressor screen. Homozygous zen-4(or153ts) <br />                                                                                       animals were mutagenized at 16&#194;&#176;C. After recov- <br />                                                                                       ery, the F1 embryos were isolated by bleach- <br />                                                                                       ing and allowed to self-fertilize at permissive <br />                                                                                       temperature (16&#194;&#176;C). When F2 animals reached <br />                                                                                       adulthood, the population was shifted to 25&#194;&#176;C <br />                                                                                       and fertile animals were selected. Star depicts <br />                                                                                       a suppressor mutation. (B) Approximately <br />                                                                                       500,000 F1 genomes were screened and 14 <br />                                                                                       suppressor mutations (point mutations indi- <br />                                                                                       cated in green) were isolated, all of which <br />                                                                                       were viable at 25&#194;&#176;C. Some mutations were <br />                                                                                       isolated multiple times. The D520N mutation <br />                                                                                       of zen-4(or153ts) is indicated in red. (C) CYK-4 <br />                                                                                       fragments purified as GST fusions, and re- <br />                                                                                       tained on the glutathione Sepharose beads, <br />                                                                                       were used in a standard pull-down assay. Re- <br />                                                                                       combinant wild type or D520N ZEN-4(1-585) <br />                                                                                       were added to the beads in a 100-&#226;&#65533;&#174;l reaction, <br />                                                                                       and incubated for 2 h at 25&#194;&#176;C. The beads were <br />                                                                                       subsequently washed and the bound fraction <br />                                                                                       was analyzed by SDS-PAGE and Coomassie <br />                                                                                       Blue staining. CYK-4 fragments containing the <br />                                                                                       suppressor substitutions (G12S and G12D) <br />                                                                                       bound to both ZEN-4 D520N and wild-type <br />                                                                                       ZEN-4, whereas wild-type CYK-4 could only <br />                                                                                       bind to wild-type ZEN-4. The negative con- <br />                                                                                       trol, GST, did not bind either form of ZEN-4. <br />  <br />  <br />  <br /> that cysteine 547 of one monomer would be sufficiently close        ZEN-4(1-556)C547VGGC, that contains a Gly-Gly-Cys exten- <br /> to cysteine 547 of the adjacent monomer to allow disulfide          sion at the C terminus and assaying for dimer formation <br /> bridge formation under oxidizing conditions (Falke et al., 1988),   (Oakley and Kim, 1998). This ZEN-4 derivative also forms <br /> whereas if they are in a parallel configuration these cysteine      covalent dimers under oxidizing conditions, whereas, ZEN- <br /> residues would be predicted to pack against each other. Under       4(1-539)GGC does not form covalent dimers under oxidizing <br /> oxidizing conditions, a substantial fraction of ZEN-4(1-603) and    conditions (Supplemental Figure S1). We conclude that the <br /> ZEN-4(1-555) form covalent dimers, yet ZEN-4(1-539) does not,       coiled coil of ZEN-4 is in a parallel configuration. <br /> suggesting that the coiled coil is in a parallel configuration         We used a similar methodology to assess the orientation <br /> (Figure 4D). This was confirmed by engineering a derivative,        of the CYK-4 coiled coil. The region of CYK-4 predicted to <br />  <br /> Vol. 18, December 2007                                                                                                           4997 <br />  V. Pavicic-Kaltenbrunner et al. <br />  <br />  <br />  <br />  <br /> Figure 4. CYK-4 and ZEN-4 form parallel coiled coil dimers, yet dimerization is not strictly required for their interaction. (A) Purified <br /> fragments of ZEN-4(1-603) and (1-539) were subjected to size exclusion chromatography (refer to Figure 2A, because these samples were <br /> analyzed in the same experiment). ZEN-4(1-539) eluted in a much earlier fraction than ZEN-4(1-603), thus it behaves as a monomer. (B) <br /> Summary of size exclusion chromatography with a panel of ZEN-4 deletion derivatives. The peak of the elution fractions of ZEN-4 derivatives <br /> are plotted versus log (MW) and compared with standards. (C) CBD ZEN-4 fragments (1-539), (1-555), and (1-603) and GST CYK-4(1-120) <br /> were coexpressed from a bicistronic vector. GST-CYK-4 was purified on glutathione Sepharose beads and eluted with reduced glutathione, <br /> followed by incubation of the eluates with chitin beads and elution with TEV protease. Both monomeric ZEN-4(1-539) and dimeric <br /> ZEN-4(1-555 or 1-603) copurified with CYK-4. (D) Recombinant ZEN-4 fragments containing a cysteine residue at position 547 were incubated <br /> under oxidizing conditions. The reaction was quenched and the ability of ZEN-4 to form disulfide bridges was analyzed. ZEN-4(603) as well <br /> as ZEN-4(1-555) efficiently formed dimers, whereas ZEN-4(1-539) did not. Thus, ZEN-4 forms a parallel coiled coil dimer. (E) CYK-4 <br /> preparations (36-119 with an N-terminal cysteine shown here), were incubated under oxidizing conditions. The reaction was quenched and <br /> the ability of CYK-4 to form disulfide bridges was analyzed. CYK-4 also forms a parallel coiled coil dimer. <br />  <br />  <br /> form a coiled coil comprises residues 36-120. We engineered            tion at the C terminus or in the &#226;&#8364;&#339;c&#226;&#8364;&#65533;-orientation at the N <br /> a series of expression constructs of the CYK-4 coiled coil             terminus. CYK-4 derivatives containing either a C-terminal <br /> region, and we added a cysteine in the predicted &#226;&#8364;&#339;b&#226;&#8364;&#65533;-posi-             cysteine, an N-terminal cysteine, or a cysteine at both the N <br />  <br /> 4998                                                                                                         Molecular Biology of the Cell <br />                                                                                                          Assembly of Centralspindlin <br />  <br />  <br /> and C termini were assayed for disulfide bridge formation          of the ZEN-4 dimerization on centralspindlin stability. We <br /> under oxidizing conditions. All the CYK-4 derivatives, C39-        constructed CBD fusions of ZEN-4(1-585), ZEN-4(1-539), <br /> 115, 39-115C, and C39-115C, formed covalent dimers under           which lacks the coiled coil, and ZEN-4(1-539)GCN4, which <br /> oxidizing conditions (Figure 4E; data not shown). In addi-         contains an exogenous parallel coiled coil. As in the afore- <br /> tion, the efficiency of dimer formation was independent of         mentioned scenario, ZEN-4(1-539)GCN4 showed signifi- <br /> protein concentration over at least a fourfold range (Figure       cantly improved binding to CYK-4 compared with mono- <br /> 4E), providing further evidence for specificity. These data        meric ZEN-4(1-539) (Figure 5C). <br /> establish that CYK-4 dimerizes via a parallel coiled coil. <br /> Thus, both CYK-4 and ZEN-4 form parallel coiled coil               DISCUSSION <br /> dimers. <br />                                                                    The centralspindlin complex plays myriad roles in cytokine- <br /> Organization and Function of the Coiled Coil                       sis, including bundling antiparallel microtubules, regulating <br /> To further probe the relationship between the structure and        furrow formation, and promoting abscission (Powers et al., <br /> function of centralspindlin, we wanted to assess whether           1998; Raich et al., 1998; Jantsch-Plunger et al., 2000; Gromley <br /> and how the dimerization of each component contributes to          et al., 2005; Yuce et al., 2005; Werner et al., 2007). We have <br /> complex formation. Because our genetic and biochemical             used a combination of biochemistry and genetics to more <br /> data establish that a high-affinity centralspindlin complex is     precisely define the interactions between CYK-4 and ZEN-4 <br /> required in vivo, we wanted to determine how coiled coil-          and to establish the molecular organization of this complex <br /> mediated dimerization of individual subunits contributes to        that lies at the nexus of cytokinesis. Our results indicate that <br /> the overall stability of the centralspindlin complex. To mea-      low-affinity interactions serve as the interface between cen- <br /> sure the affinity constants of monomeric ZEN-4(1-539) for          tralspindlin subunits. However, each subunit of centralspin- <br /> dimeric CYK-4(1-120), we used isothermal titration calorim-        dlin contributes two such interfaces that are linked through <br /> etry (ITC). This method was chosen instead of SPR, because         a parallel coiled coil. This results in the cooperative assem- <br /> it measures binding of proteins free in solution and would         bly of a stable, high-affinity heterotetramer. These observa- <br /> therefore eliminate avidity affects that can be induced by         tions refine our understanding of this critical complex and <br /> immobilization of ligands on the chip matrix used for SPR          provide insight into how it bundles microtubules. <br /> experiments. We measured an 800 nM dissociation constant              Because centralspindlin localizes to overlapping antipar- <br /> for monomeric ZEN-4(1-539) for dimeric CYK-4(1-120) (Fig-          allel microtubules, and it is required for their bundling <br /> ure 5A), or &#226;&#172;&#402;80-fold weaker than the affinity of dimeric           (Jantsch-Plunger et al., 2000), it is of particular interest to <br /> ZEN-4(1-603) for dimeric CYK-4(1-120). Thus, dimerization          establish how this complex specifically recognizes microtu- <br /> of ZEN-4 greatly impacts the overall stability of centralspin-     bules arranged in this configuration. One possibility is that <br /> dlin.                                                              the complex is organized in an antiparallel configuration, as <br />    We next investigated the extent to which dimerization of        is thought to be the case for the tetrameric form of the Eg5 <br /> CYK-4 affects centralspindlin stability. We constructed a          kinesin complex that plays an analogous role, stabilizing the <br /> MBP fusion of CYK-4(1-36), which lacks the coiled coil,            bipolar mitotic spindle before anaphase. Therefore, we used <br /> CYK-4(1-120), and CYK-4(1-36)GCN4, which contains an               multiple approaches to establish the orientation of the coiled <br /> exogenous parallel coiled coil (O&#226;&#8364;&#8482;Shea et al., 1991). The latter   coils in CYK-4 and ZEN-4. First, we used a deletion ap- <br /> fusion protein was generated to assess whether the coiled          proach. If a protein is arranged in a parallel coiled coil, it <br /> coil contributes solely through dimerization. In semiquanti-       could retain its ability to dimerize when the coiled coil <br /> tative binding experiments with 1 &#226;&#65533;&#174;M ZEN-4(1-585)GFP and           region is truncated by greater than half, whereas an antipa- <br /> 0.5 &#226;&#65533;&#174;M CYK-4 derivatives, we found that CYK-4(1-120)               rallel coiled coil would not dimerize if the coiled coil region <br /> bound well to ZEN-4 as did CYK-4(1-36)GCN4, but the                were bisected. We found that both CYK-4 and ZEN-4 could <br /> construct lacking a coiled coil did not bind significantly         dimerize even when the regions predicted to form the coiled <br /> more ZEN-4 than the negative control, MBP (Figure 5B). To          coil were reduced to less than half their original lengths. To <br /> obtain quantitative measures of binding affinities, we per-        unambiguously demonstrate that CYK-4 and ZEN-4 each <br /> formed pull-down assays after incubation with ZEN-4(1-             form parallel coiled coils, we assessed cross-linking between <br /> 585)GFP and measured the fluorescence remaining in the             either a native cysteine close to one end of the coiled, be- <br /> supernatant. As seen previously using SPR, these experi-           tween an engineered cysteine residue at the extreme ends of <br /> ments show that CYK-4(1-120) and ZEN-4(1-585) form a               the predicted coiled coils, or both, and we demonstrated that <br /> complex with a dissociation constant of &#226;&#172;&#402;10 nM (Figure 5D).        these cysteine residues could efficiently form a disulfide <br /> Interestingly, the affinity of the CYK-4(1-36)GCN4 induced         bridge (Oakley and Kim, 1998). Formation of a disulfide <br /> dimer was only twofold lower that wild-type CYK-4 dimer.           bridge requires that the two sulfhydryl groups be within 0.7 <br /> The binding affinity of monomeric CYK-4(1-36) for ZEN-4(1-         nm, which is unlikely to occur unless the proteins formed a <br /> 585)GFP was outside the range of this assay, therefore titra-      parallel coiled coil in register (Falke et al., 1988). Further- <br /> tion binding assays with CYK-4 monomer constructs were             more, in the case of CYK-4 we found that the efficiency of <br /> performed. CYK-4(1-36) at 1 &#226;&#65533;&#174;M was incubated with in-              dimer formation was independent of protein concentration, <br /> creasing concentrations of ZEN-4(1-585)GFP, ranging from 0         suggesting that the molecules exist as dimers in their native <br /> to 3 &#226;&#65533;&#174;M, followed by a pull-down assay. Monomeric CYK-             state, as expected for a coiled coil dimer. Finally, the parallel <br /> 4(1-36) bound to ZEN-4(1-585) with a dissociation constant         nature of the CYK-4 coiled coil is substantiated by the fact <br /> of &#226;&#172;&#402;500 nM, or 50-fold higher than of the CYK-4 dimer for          that CYK-4(1-120) and CYK-4(1-36)GCN4, which contains a <br /> the ZEN-4 dimer. Thus, the CYK-4 coiled coil domain con-           well characterized parallel coiled coil (O&#226;&#8364;&#8482;Shea et al., 1991), <br /> tributes significantly to complex formation. However, be-          bind to ZEN-4 with near equal affinity. Similarly, the im- <br /> cause the CYK-4 coiled coil can be effectively substituted by      proved CYK-4 binding of ZEN-4(1-539)GCN4 relative to <br /> an exogenous dimerization domain, it is not likely to directly     ZEN-4(1-539), further confirms the parallel nature of the <br /> participate in binding; rather, dimerization may primarily         ZEN-4 coiled coil. <br /> serve to increase the avidity of the complex. To extend these         In previous work, we defined ZEN-4 (434-603) as the <br /> findings, we used a similar methodology to assess the effect       minimal region capable of binding to CYK-4 (Mishima et al., <br />  <br /> Vol. 18, December 2007                                                                                                         4999 <br />  V. Pavicic-Kaltenbrunner et al. <br />  <br />  <br />  <br />  <br /> Figure 5. Coiled coils in CYK-4 and ZEN-4 play a stabilizing role in centralspindlin complex formation. (A) Binding of monomeric <br /> ZEN-4(1-539) to dimeric CYK-4(1-120) was measured using ITC. Measurements were performed at 25&#194;&#176;C using the working buffer as a <br /> reference. The protein concentration was 10 &#226;&#65533;&#174;M, and the ligand concentration (ZEN-4) was 100 &#226;&#65533;&#174;M. Using a simple one-site model, the <br /> affinity of monomeric ZEN-4 for dimeric CYK-4 was calculated at 800 nM. (B) CYK-4 fragments purified as MBP fusions and retained on the <br /> amylose beads were used in a standard pull-down assay. Recombinant ZEN-4(1-585) GFP was added to the beads in a 100-&#226;&#65533;&#174;l reaction, and it <br /> was incubated at room temperature for 2 h. The beads were subsequently washed, and the bound fraction was analyzed by SDS-PAGE and <br /> Coomassie Blue staining. CYK-4 containing a coiled coil, able to form dimers, could bind ZEN-4, whereas CYK-4 lacking a coiled coil could not, <br /> nor could the negative control, MBP. S, supernatant; P, pellet. (C) CYK-4 fragments purified as GST fusions and retained on the glutathione <br /> Sepharose beads were used in a standard pull-down assay. Recombinant ZEN-4(1-585), ZEN-4(1-539) GCN4, and ZEN-4(1-539) were added to the <br /> beads in a 100-&#226;&#65533;&#174;l reaction, and they were incubated at room temperature for 2 h. The beads were subsequently washed, and the bound fraction <br /> was analyzed by SDS-PAGE and Coomassie Blue staining. ZEN-4 containing a coiled coil, either native or engineered, could bind CYK-4, whereas <br /> ZEN-4 lacking a coiled coil could not. The negative control, GST, did not bind any of the ZEN-4 fragments. (D) Comparison of binding affinities <br /> between monomeric and dimeric ZEN-4/CYK-4 constructs. Symbols indicate the method by which the binding constant was determined: &#226;&#189;&#164;, ITC; <br /> &#226;&#189;&#167;, pull-down assay; &#226;&#189;&#166;, pull-down assay with fluorescence intensity measured; &#226;&#189;&#165;, SPR; and 1, approximate KD. <br />  <br />  <br /> 2002). This region contained a coiled-coil region shown to                two biochemical functions of ZEN-4, which previously <br /> mediate homodimerization. In this current work, using                     seemed linked: dimerization and CYK-4 binding. We have <br /> more sensitive assays, we have refined the CYK-4 binding                  shown, however, that monomeric ZEN-4 binds to CYK-4 <br /> region of ZEN-4 to (434-539). Significantly, this separates               dimer &#226;&#172;&#402;80&#226;&#171;&#187; less well compared with dimeric ZEN-4. Like- <br />  <br /> 5000                                                                                                            Molecular Biology of the Cell <br />                                                                                                                 Assembly of Centralspindlin <br />  <br />  <br /> wise, we previously demonstrated that the minimal ZEN-4                  high-affinity binding of CYK-4(1-36)GCN4 to ZEN-4 demon- <br /> binding region of CYK-4 includes residues 1-120. Using                   strates that the difference in affinity between these two CYK-4 <br /> stringent assays, we were previously unable to detect an                 derivatives can largely be ascribed to the loss of dimerization <br /> interaction between the N-terminal 35 amino acids of CYK-4               as opposed to loss of specific binding interfaces. <br /> and ZEN-4. However, here we have used less stringent assays,                In an effort to further understand the molecular interac- <br /> and we show that CYK-4 (1-36) can in fact bind ZEN-4, but                tions involved in central spindle assembly, we performed a <br /> with significantly lower affinity (&#226;&#172;&#402;0.5 &#226;&#65533;&#174;M). Furthermore, the            genome-wide screen for mutations that ameliorate the tem- <br />  <br />  <br />  <br />  <br /> Figure 6. Centralspindlin: functional insights from structural aspects of the complex. (A) Sequence comparison of the CYK-4/ZEN-4 <br /> interaction domains among vertebrates and invertebrates. The original temperature-sensitive mutations are indicated by red dots; suppres- <br /> sors of zen-4(or153ts) are indicated by green dots; and suppressors of cyk-4(t1689ts) are indicated by black dots (Mishima et al., 2002). The <br /> accession numbers for the aligned sequences are as follows: ZEN-4/MKLP1[C_elegans (NP_741411.1); C_briggsae (emb CAE61601.1); <br /> H_sapiens (AAH51826.1); Danio_rerio (AAF00594.1); Strong_purp (AAG18582.1); Dros_melano (CAA12181.1); Mus_musc (NP_077207); Canis- <br /> _famil (XP_535528.2); Chinese_hamster (CAA58558.1); Xen_trop (NP_001011104.1); Gallus_gallus (NP_001025725.1); Anopheles_gamb <br /> (XP_308170.2); Apis_mellifera (XP_624886.1); Xen_laevis (AAH84928.1); Tetraodon_nigriv (CAF99883.1); Rattus_norv (XP_236313.3); Bos_taurus <br /> (XP_593824.2)]; CYK-4/MgcRacGAP [C_Briggsae (CAE71360.1); D_melanog (CAB96203.1); C_elegans (NP_499845.1); M_mus (NP_036155.1); <br /> H_sapiens (NP_037409.2); Rattus_nov (XP_235650.2); Canis_famil (XP_543675.2); Bos_taurus (XP_592496.2); Xen_laevis (AAH70771.1); Xen_trop <br /> (NP_001001236.1); Tetraodon_nigrov (CAG03964.1); Gallus_gallus (XP_424490.1); Stron_purp (XP_783360.1); Danio_rerio (AAH95799.1); Pan_trog <br /> (XP_509531.1); Apis_mellifera (XP_393627.2); Anopholes_gamb (XP_319660.2)]. (B) Schematic representation of ZEN-4 (a) and the centralspindlin <br /> complex (b). Model of possible mode of action of the complex, and the structural effects that CYK-4 may have on ZEN-4 (c). <br />  <br /> Vol. 18, December 2007                                                                                                                  5001 <br />  V. Pavicic-Kaltenbrunner et al. <br />  <br />  <br /> perature sensitivity of the zen-4(or153ts) allele. Although this    whether CYK-4 directly participates in microtubule bun- <br /> allele contained two substitution mutations, we determined          dling or whether it allosterically modulates the structure and <br /> that the defect was due to just one of the two substitutions,       biochemical properties of ZEN-4. Future biophysical analysis <br /> D520N. Previous predictions suggested that residue 520 lies         of centralspindlin will be necessary to answer this important, <br /> within the coiled coil region, and it seemed surprising that a      open question. The work presented here provides a rich con- <br /> conservative change would disrupt coiled coil formation.            text for future structural studies of centralspindlin and its com- <br /> We therefore directly defined the regions of the protein that       ponents. <br /> are sufficient to form a stable coiled coil. The data presented <br /> here suggest that D520 likely lies outside of the coiled coil <br />                                                                      <br /> region. Indeed, our experimental results fit well with the <br /> prediction of the PAIRCOILS algorithm (Berger et al., 1995).         <br />    The only mutations we found that could suppress the             <br /> zen-4(or153ts) allele were located within the ZEN-4/CYK-4            <br /> interface. This is similar to our previous characterization of       <br /> suppressors of cyk-4(t1689ts). The combined data from these <br /> two screens reveal 12 substitutions that restore the interac- <br />                                                                      <br /> tion between CYK-4 and ZEN-4 when it is perturbed by <br /> either of these substitution alleles. Interestingly, some sub-     <br /> stitution mutations were found to suppress both the cyk-4            <br /> temperature-sensitive mutation and the zen-4 temperature-            <br /> sensitive mutation, suggesting that some of these suppressor        <br /> mutations may not accommodate the altered amino acid, but          <br /> rather strengthen the interaction overall. Our genetic and          <br /> biochemical rescue of ZEN-4 D520N by mutations in CYK-4            <br /> strengthen this argument and further substantiate that the      <br /> interaction between CYK-4 and ZEN-4 is critical for the func- <br /> tion of centralspindlin in cytokinesis. Strikingly, of the two   <br /> substitutions within CYK-4, which rescued complex forma- <br /> tion, the G12D allele more effectively restored binding to         <br /> ZEN-4 D520N than did the G12S substitution. The former             <br /> substitution was identified four times in the screen, whereas <br /> the allele that more weakly restores binding was only iso-       <br /> lated twice. Additionally, the D31N allele, which was only <br /> isolated once, did not detectably restore binding under the        <br /> conditions tested (data not shown). Additionally, the sup-          <br /> pressor mutations all fall within the biochemically defined       <br /> minimal interacting regions in CYK-4 and ZEN-4 corre-                <br /> sponding to 1-36 in CYK-4 and 434-539 ZEN-4. It is notable         <br /> that two independent screens for temperature-sensitive, ma-          <br /> ternal-effect lethal mutations both identified mutations that       <br /> disrupt the CYK-4/ZEN-4 binding interface and not other             <br /> sites within either gene. Furthermore, multiple substitutions     <br /> can suppress each of these defects, suggesting remarkable <br /> plasticity in the CYK-4/ZEN-4 binding interface. These find-       <br /> ings are particularly intriguing in light of the facts that these   <br /> regions are not strongly conserved during evolution, yet <br />                                                                   <br /> there is direct evidence that the orthologous proteins interact     <br /> in C. elegans, Drosophila, and human cells (Mishima et al.,    <br /> 2002; Somers and Saint, 2003). Presumably, these regions            <br /> interact in all metazoan species, despite the poor conserva-       <br /> tion at the level of primary sequence (Figure 6A). It will be       <br /> of significant interest to obtain high-resolution structural        <br /> information for this interaction interface and the refinement       <br /> of the minimal interaction domain reported here should         <br /> facilitate these future studies.                                    <br />    The data presented here indicates that a rather weak (&#226;&#172;&#402;0.5   <br /> &#226;&#65533;&#174;M KD) interaction connects a monomer of the RhoGAP               <br /> CYK-4 to the monomer of the kinesin motor protein ZEN-4.        <br /> However, because both CYK-4 and ZEN-4 individually as-              <br /> semble into parallel coiled coil dimers, the heterotetrameric      <br /> form of centralspindlin is a highly stable (10 nM KD) com-         <br /> plex. In view of the results presented here, and assuming     <br /> that centralspindlin forms a symmetrical complex, we pro-      <br /> pose the working model depicted in Figure 6B and Supple-            <br /> mental Movie 1. In vivo, bundling of microtubules in the         <br /> central spindle requires that CYK-4 and ZEN-4 bind each              <br /> other with high affinity. At this juncture, we do not yet know       <br />  <br /> </body></html>