<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL OF BACTERIOLOGY, Aug. 1982, p. 723-728                                                    Vol. 151, No. 2 <br /> 0021-9193/82/080723-06$02.0O/0 <br />  <br />  <br />        A Gene Its Product Required Transposition of <br />                  Resistance Transposon Tn2603 <br />              MICHIYASU TANAKA, HARUKO HARAFUJI, AND TOMOKO YAMAMOTO* <br />  Division Microbial Chemistry, Faculty Pharmaceutical Sciences, Chiba University, Chiba 260, Japan <br />                                Received 17 March 1982/Accepted 26 April 1982 <br />  <br />             Tn2603 multiple-resistance transposon encoding resistance ampicillin, <br />           streptomycin, sulfonamide, mercury having molecular size 20 <br />           kilobase pairs, 200-<span id='am-4' about='xsp:base' typeof='owl:Thing'>base</span>-pair inverted repeats ends. The essential <br />           sites functions Tn2603 required transposition were <br />           determined construction characterization various deletion mutants <br />           affecting efficiency transposition. Deletions introduced plasmid <br />           pMK1::Tn2603 partial digestion restriction endonuclease EcoRI vitro. <br />           Analysis deletion mutants showed inverted repeat segments both <br />           ends trans-acting diffusible product(s) encoded right-hand of <br />           central portion required transposition Tn2603. An essential <br />           gene product revealed protein having molecular weight 110,000 by <br />           analysis polypeptides synthesized Escherichia coli minicells. This protein <br />           assumed so-called transposase. <br />  <br />    Tn2603 20 kilobases (kb) <span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span>             son original site donor replicon a <br /> encodes multiple resistance ampicillin, strep-        new copy target replicon. Resolution is <br /> tomycin, sulfonamide, mercury (19). Ampi-            mediated tnpR gene product, resolvase <br /> cillin resistance Tn2603 production         (12, 13). A transposase gene required trans- <br /> of type II ,B-lactamase (alternatively, oxacillin-     position defined transpo- <br /> hydrolyzing 3-lactamase, designated OXA-1                sons, e.g., TnS (14) TnWO (4). <br /> [17]). Tn2603 movable element capa-            <span id='am-3' about='protege:TO' typeof='owl:Thing'>To</span> determine gene(s) Tn2603 needed for <br /> ble transposing gene type II              transposition, constructed characterized <br /> enzyme known present (9). In view evi-             series deletion mutants transposon. <br /> dence showing type II enzyme gene has,          We detected polypeptides specified by <br /> in instances, recognized         Tn2603 Escherichia coli minicell system <br /> streptomycin, sulfonamide, mercury resis-            demonstrated potential transposase <br /> tance genes naturally occurring plasmids,          polypeptide Tn2603. <br /> can speculated multiple-resistance <br /> transposon Tn2603 plays major role                       MATERIALS AND METHODS <br /> the distribution type II 3-lactamase gene             Bacterial strains plamids. The bacterial strains <br /> in nature (18). This gene distinct type      used E. coli K-12 C600 (thr leu thi lac tonA <br /> I P-lactamase gene transposition medi-          supE), AB2463 (recA thr leu thi lac gal ara xyl mtl pro <br /> ated transposon specifying ampicillin          arg rpsL tsx sup-37), WA5023 Rift, rifampin- <br /> resistance, example, Tn3 (10). Recently,          resistant (Rif) mutant WA5023 (polA thy), and <br /> demonstrated Tn2603 closely related           X2207 (minA minB nat). Strains WA5023 X2207 <br /> two multiple-resistance transposons, Tn2l          kindly provided M. Yoshikawa M. Inoue, <br />                                                          respectively. The plasmids pMK1: :Tn2603-1, <br /> and Tn4, suggesting transposons          pMB9::Tn2603-1, R386, pACYC184 previ- <br /> been generated common ancestor (18).              ously described (18, 19). pMK1::Tn2603-11 plas- <br />    Recently, evidence contributed             mid Tn2603 R388::Tn2603-1 inserted into <br /> understanding mechanism transposi-             pMK1 (19). pMK1 variant plasmid ColEl, <br /> tion, models molecular mechanism              contains TnS, molecular size 12 kb. <br /> have postulated (1, 2, 5, 15). These models            In vitro formation deletions In pMK1::Tn2603-1 <br /> involve stages. The step forma-         DNA digestion EcoRI endonuclease. A 1-,ug <br /> tion cointegrate structure replicons         pMK1::Tn2603-1 DNA partially digest- <br /> joined copies transposon. In Tn3,          ed 1 U EcoRI endonuclease 15 min 37&#194;&#176;C. <br />                                                          After inactivation EcoRI endonuclease heating <br /> the cointegrate formation requires tnpA gene         15 min 65&#194;&#176;C, digested DNA ligated T4 <br /> product, transposase, specified Tn3 itself        ligase (Takara Shuzo Co., Ltd., Kyoto, Japan) and <br /> (8). The second step resolution cointe-        transformed competent E. coli C600 cells pre- <br /> grate structure resulting copy transpo-      pared method Cohen et al. (3). Ampicillin- <br />                                                        723 <br />  724      TANAKA, HARAFUJI, AND YAMAMOTO                                                                                                                J. BACTERIOL. <br />  <br />                                   IR TnS                             p                        Tn2603 <br />                           E   P           BP         PP                     E          E3E           ElBBP            IEE                 P PE <br />                                                                                                                       I I I                 I     II <br />                                   H <br />                                       I <br />                                           S <br />                                            I         I <br />                                                      H <br />                                                                              1 <br />                                                                                        11WIH         I I <br />                                                                                                     H S HH H             HN <br />                                   _            kan             ori       tr                          sul sr bis                          cot <br />  <br />                                                           El                     E4     E6110O E             2            15.91?       E3 <br />                  BWnH                      -3                        U                       .4 31       B3                     a <br />                      Mn <br />                               2           N3                                                   NS     N5 H9H         H4   HN       H2 <br />  <br />  <br />                 B. <br />                           del-28 <br />                           ded-lS <br />                           del-134 <br />                           del-44 <br />                           del-45 <br />                           del-SO <br />                           del-100 <br />                           del-170 <br />  <br />    FIG. 1. Restriction cleavage map pMK1::Tn2603-1 (A) locations deletions deletion mutants (B). <br /> Abbreviations: E, EcoRI; B, BamHI; H, Hindlll; S, Sail; P, PstI; imm, immunity colicin El; kan, kanamycin; <br /> ori, origin replication; mer, mercury; sul, sulfonamide; str, streptomycin; bla, ,B-lactamase; col, colicin El. <br /> The designations fragments indicated cleavage map (A). Notations (B): (-), deleted region; <br /> (*+), inverted region. <br />  <br /> and kanamycin-resistant transformants selected                                                       RESULTS <br /> and examined resistance streptomycin,                                        Construction deletions pMK1::Tn2603 <br /> sulfonamide, mercury chloride (Hg) colicin                                    plasmids. pMK1::Tn2603-1 derivative of <br /> production. In transformants resistant ampicdil- <br /> lin, streptomycin, sulfonamide, kanamycin, coli-                                  plasmid pMK1 Tn2603 inserted the <br /> cin production nonproduction expected                                    ColEl plasmid region molecular size <br /> due derivatives retaining intact inverted repeat                               33 kb (19). A restriction cleavage map the <br /> segments ends derivatives deleted                               plasmid constructed various re- <br /> right inverted repeat segment Tn2603, respectively.                                striction endonucleases (Fig. 1A). To make dele- <br /> Deleted molecules initially monitored agarose                                 tions span entire length Tn2603, <br /> gel electrophoresis colony lysates appropriate                                segment ampicillin, streptomycin, <br /> clones verified restriction endonuclease                                sulfonamide resistance, used EcoRI en- <br /> digestion analysis purified plasmid DNA, using                                 donuclease digestion cleavage <br /> various endonucleases. <br />    Polypeptide synthesis minicells. Minicell-produc-                               sites enzyme scattered the <br /> ing strain X2207 transformed plasmid DNA.                                    transposon. Plasmid pMK1::Tn2603-1 DNA was <br /> Isolation minicells, labeling polypeptides                                 partially digested EcoRI endonuclease, li- <br /> 14C-amino acids (45 mCi/mmol; Commissariat L&apos;En-                                    gated, transformed E. coli C600. Result- <br /> ergie Atomique, Paris, France), lysis minicells                                ant transformants examined their <br /> were carried methods Veltkamp et al. (16).                              resistance determinants colicin production <br /> Minicell lysates loaded sodium dodecyl sul-                                   (Table 1). Plasmid DNA isolated each <br /> fate-polyacrylamide gels described Laemmli                                      transformant digested EcoRI endonu- <br /> (11). After electrophoresis 16 h 12 mA, gels                              clease determine deleted fragment(s). <br /> stained Coomassie brilliant blue R-250, de- <br /> stained, soaked En3Hance (New England                                        Agarose gel electrophoresis patterns repre- <br /> Nuclear Corp., Boston, Mass.). Gels dried                                     sentative set deletion mutants isolated are <br /> heating 65&#194;&#176;C, fluorography carried 8                               shown Fig. 2. Mutant del-llS (Fig. 2, slot j) <br /> days -800C.                                                                        generated pMK1::Tn2603-11 diges- <br />    Determination Tn2603-promoted        nmsposition                             tion Sall endonuclease. pMK1::Tn2603-11 <br /> a conjugal mating assay. Determination transposi-                                  plasmid Tn2603 inserted the <br /> tion frequency carried essentially                                     inverted repeats TnS pMK1 an <br /> method described (19), WA5023                                      opposite orientation respect to <br /> Rifr used instead P3478 Riff recipient <br /> bacterium.                                                                            pMKI::Tn2603-1 (unpublished data). These de- <br />     Other methods. Plasmid DNA preparation, restric-                                  letion derivatives examined cleavage <br />  tion endonuclease digestion, agarose gel electrophore-                               analysis endonucleases determine <br />  sis, ligation, transformation                                   remaining EcoRI fragments main- <br />  methods previously described (18).                                                   tained original order (Fig. 1B). In dele- <br />  VOL. 151, 1982                                               Tn2603-ENCODED TRANSPOSASE                          725 <br />                  TABLE 1. Relative transposition frequencies complementation analysis <br />                                                                                               Relative frequencies <br />      Plasmid               Relevant markersa           Colicin               Relative         complemented <br />                                                                                                   fragment: with <br />                                                       production           transposition <br />                                                                            frequencies <br />                                                                                                 B2              H2 <br /> pMK1::Tn2603-1         Apr, Smr, Sur, Hgr, Kmr            +                       .0b           -c <br /> del-28                 Apr, Smr, Sur, Hgr, Kmr            _                    &lt;0.01          &lt;0.01            &lt;0.01 <br /> del-11S                Apr, Smr, Sur, Hgs, Kmr            +                    &lt;0.01          &lt;0.01            &lt;0.01 <br /> del-134                Apr, Smr, Sur, Hgs, Kmr            +                      0.29 <br /> del-44                 Apr, Smr, Sur, Hgr, Kmr            +                      1.1 <br /> del-45                 Apr, Smr, Sur, Hgr, Kmr            +                      1.9 <br /> del-88                 Apr, Smr, Sur, Hgs, Kmr            +                    &lt;0.01 <br /> del-100                Apr, Smr, Sur, Hgr, Kmr            +                    &lt;0.01 <br /> del-170                Apr, Smr, Sur, Hgr, Kmr            +                    &lt;0.01           0.14             0.13 <br />   Ap, Ampicillin; Sm, streptomycin; Su, sulfonamide; Km, kanamycin. <br />   b Transposition frequency 4.8 x 10&apos; experiment. <br />   c , Not tested. <br />  <br />  <br />  <br /> tion derivatives del-88 plasmid,          To determine essential function <br /> fragments retained arrangement simi-       encoded deleted region transposi- <br /> lar fragments parent plasmid        tion-deficient mutants gene product a <br /> pMK1::Tn2603-1. In del-88 plasmid, E4         site required transposition, attempted a <br /> fragment inverted original position,       complementation analysis transposition. <br /> resulting inactivation Hg resistance <br /> gene. <br />    Characterization transposition proficiency                  b   c    d       e   f h            k <br /> deletion mutants. To test ability dele- <br /> tion mutants transpose, determined trans- <br /> position frequencies conjugal mating assay <br /> as follows. Plasmid DNA parent              23. 7m- <br /> deletion mutant transformed E. coli <br /> recA strain AB2463 harboring transferrable        9.46- <br /> plasmid R386. Resultant transformants were <br /> mated E. coli polA strain WA5023 Rif,        6.67.- <br /> the R386-containing transconjugants se- <br /> lected screening tetracycline rifampin     4.26 <br /> resistance, R386::Tn transconju- <br /> gants assayed screening ampicillin, <br /> tetracycline, rifampin resistance. The fre- <br /> quency transposition R386-containing <br /> transconjugants presented relative value       2.25.- <br /> compared parental Tn2603-promot-          1.96- <br /> ed transposition (Table 1). del-28 del-11S, <br /> which lost right left inverted repeat <br /> segments, respectively, transpose. <br /> These results confirmed inverted repeat <br /> segments ends Tn2603 necessary <br /> for transposition, shown other <br /> known transposons (4, 7, 14). del-134, del-44, <br /> and del-45, lost regions the <br /> center left element, transposed      FIG. 2. Agarose gel electrophoresis EcoRl di- <br /> level close parental Tn2603. On        gests pMK1::Tn2603-1 pMK1::Tn2603-11 and <br /> other hand, del-88, del-100, del-170        deletion derivatives. Lane a, Bacteriophage X <br /> not transpose all. The deleted region common       DNA digested HindlIl endonuclease molecu- <br /> to mutants E7 fragment. These           lar size standards (molecular sizes fragments are <br />                                                       indicated kb left side). EcoRI digestion <br /> observations suggested essen-        patterns of: lane b, pMKI::Tn2603-1; lane c, del-28; <br /> tial functions encoded right-hand          lane d, del-134; lane e, del-44; lane f, del-45; lane g, <br /> region Tn2603, encompassing E7 frag-           del-88; lane h, del-100; lane i, del-170; lane j, del-llS; <br /> ment.                                                 lane k, pMKI::Tn2603-11. <br />  726    TANAKA, HARAFUJI, AND YAMAMOTO                                                                       J. BACTERIOL. <br />  <br /> pACYC184 derivatives, B2 H2            del-170. To compare transposase Tn2603 <br /> fragment contained E7 region indicated        ampicillin resistance transposon, <br /> Fig. 1, constructed transformed        Tn2602, polypeptides plasmid pTY2, a <br /> into E. coli AB2463 harboring R386           tnpA&apos; tnpR- bla- mutant constructed <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> transposition-deficient mutants, del-28,     pMK1::Tn2602-1 (20) BamHI digestion and <br /> del-11S, del-170. If region encodes     ligation, electrophoresed gel. <br /> gene product(s) required transposition,    The 110,000-molecular-weight polypeptide a <br /> mutants expected complemented         molecular weight identical transpo- <br /> in trans coexisting plasmid. The ability   sase Tn2602 (Fig. 3, slot e). Furthermore, a <br /> del-170 transpose demonstrably comple-        21,000-molecular-weight polypeptide pro- <br /> mented plasmids carrying H2 (13%) B2       duced B2 H2. A function this <br /> (14%) fragment relative wild-type plas-       polypeptide discussed. <br /> mid. These observations suggested the <br /> right-hand region encompassing E7 fragment                         DISCUSSION <br /> in element encodes gene product       In paper, demonstrated es- <br /> necessary transposition Tn2603. Howev-        sential regions Tn2603 required own <br /> er, abilities del-28 del-itS trans-    transposition genetic functional analysis <br /> pose complemented, suggesting          various deletion mutants. Moreover, de- <br /> the inverted repeats cis-acting structures       termined gene product encoded the <br /> necessary transposition.                         regions analysis polypeptides syn- <br />    Tn2603-specific polypeptides produced E.       thesized minicells. <br /> coli minicell. We attempted identify the <br /> polypeptides encoded Tn2603, especially <br /> those involved transposition, using an <br /> E. coli minicell system. Minicell-producing          A           a <br />                                                                :~.&quot;.~&quot;:!: <br />                                                                             b c d e <br />                                                                               ....1&apos;. <br />                                                                                                             B <br /> strain X2207 transformed plasmid                                 .... <br />                                                                             ... <br />                                                                             . P <br />  <br />                                                                                         .. ... <br />  <br /> pACYC184 derivative contains H2            165 K_. <br /> B2 fragments pMK1::Tn2603-1,            155Kw <br /> polypeptides produced resultant clones <br /> were analyzed (Fig. 3A). After fluorography,                                                      -_A <br /> X-ray film scanned increase sensitiv- <br /> ity; regions covering high-molecular- <br /> weight polypeptides shown Fig. 3B. The <br /> B2 fragment contains, addition Tn2603                                                             B <br /> genes, colicin El structural gene         68K-                   wl&apos;                     r <br />  <br /> inverted repeat segment TnS specifically <br /> codes transposase (14), H2 <br /> fragment contains colicin El gene      45KK- <br /> of inverted repeat. The polypeptides                                                                   top <br /> assigned colicin El gene transpo- <br /> sase encoded inverted repeat segment of <br /> Tn5 (14) shown Fig. 3. A 45,000-molecu-        25K-                                            -E <br /> lar-weight polypeptide pACYC184-H2 was <br /> assigned truncated polypeptide transpo- <br /> sase encoded TnS. The results revealed                                                                   b <br /> both H2 B2 fragments specify polypeptide <br /> having molecular weight 110,000. The gene       12.5K- <br /> encoding polypeptide re-                                                                   top <br /> gion 3.1 kb right inverted repeat <br /> segment nearest HindlIl site Tn2603.         FIG. 3. Polypeptides synthesized minicelis frac- <br /> This 3.1-kb DNA region size       tionated 10 15% sodium dodecyl sulfate-poly- <br /> account total coding capacity         acrylamide gradient gel. (A) Fluorography gel. <br />  110,000-molecular-weight polypeptide. The de-       Slot a, pACYC184; slot b, pACYC184-B2; slot c, <br /> letion E7 fragment (1.2 kb) del-170    pACYC184-H2; slot d, pMB9::Tn2603; slot e, pTY2. <br />  mutant lead loss       Marker A, Transposase Tn2602; marker C, colicin <br /> intact 110,000-molecular-weight polypeptide.         El; markers B D, transposase TnS; marker E, <br />                                                      chloramphenicol acetyltransferase. Standard molecu- <br /> The 110,000-molecular-weight polypeptide im-      lar weights x 103 (K) indicated left of <br /> plicated transposase Tn2603          gel. (B) Scanning region fractionating the <br /> pACYC184-H2 pACYC184-B2 comple-                  transposase Tn2603 lanes b c. Arrows <br />  mented trans transposition deficiency     indicate transposase Tn2603. <br />  VOL. 151, 1982                                               Tn2603-ENCODED TRANSPOSASE                           727 <br />    The right-hand central portion      molecular-weight polypeptide Tn2603 re- <br /> Tn2603 (Fig. 1) encoded function         solves cointegrate intermediate its <br /> which trans-acting, diffusible product re-       transposition. Furthermore, loss intact <br /> quired transposition. The gene product        polypeptide did lead increase the <br /> defined polypeptide having molecular           frequency formation cointegrate rela- <br /> weight 110,000. Tn2603 inverted repeat         tive transposition wild type, <br /> sequences 200 base pairs ends,       suggesting polypeptide does repress <br /> as previously determined electron micro-           weaken expression transposase <br /> scopic heteroduplex analysis precisely        gene (M. Tanaka et al., submitted publica- <br /> by sequencing (19). Deletions including ter-      tion). This observation possibly associated <br /> minal inverted repeat sequences trans-         result transposase Tn2603 <br /> pose circumstances, dele-           produced B2 fragment producing the <br /> tions internal region Tn2603 appear         21,000-molecular-weight polypeptide, although <br /> capable transposition complementation           fourfold-higher production transposase <br /> with corresponding region element.       observed H2 fragment missing the <br />    The notion transposons specify functions      21,000-molecular-weight polypeptide (Fig. 3B), <br /> involved transposition         distinct Tn3, product the <br /> proposed Tn3 Heffron et al. (7). Their         functions repressor resolvase (1, 6, 13). <br /> studies, deletion mutants affecting the <br /> transposability Tn3 used, indicated                          ACKNOWLEDGMENT <br /> its gene product, called transposase,        We grateful T. Sawai, laboratory, for <br /> and terminal inverted repeat sequences        helpful discussions. <br /> involved normal Tn3 transposition. The trans- <br /> posase specified left-hand                         LITERATURE CITED <br /> central region, intact terminal in- <br /> verted repeat structures required trans-      1. Arthur, A., D. Sherfatt. 1979. Dissection the <br />                                                           transposition process: transposon-encoded site-specific <br /> position. A similar analysis reported            recombination system. Mol. Gen. Genet. 176:267-274. <br /> for TnS TnlO (4, 14). In elements,        2. Bukharl, A. I. 1981. Models DNA transposition. Trends <br /> respective inverted repeat segments,          Biochem. Sci. 6:56-60. <br /> the pairs case, contain transpo-    3. Cohen, S. N., A. C. Y. Chang, L. Hsu. 1972. Nonchro- <br />                                                           mosomal antibiotic resistance bacteria: genetic trans- <br /> son-coded information required normal                 formation Escherichia coli R-factor DNA. Proc. <br /> transposition. TnS TnJO              Natl. Acad. Sci. U.S.A. 69:2110-2114. <br /> thought composite elements           4. Foster, T. J., M. A. Davi, D. E. Roberts, K. Takeshita, <br /> include structurally intact insertion sequence-           N. KIeckner. 1981. Genetic organization transposon <br />                                                           TnlO. Cell 23:201-213. <br /> like elements. In Tn2603, potential transpo-       5. Galas, D. J., M. Chandler. 1981. On molecular <br /> sase specified terminal inverted           mechanism transposition. Proc. Natl. Acad. Sci. <br /> repeat segments central portion,            U.S.A. 78:4858-4862. <br /> shown class elements represented         6. GiB, R. E., F. Heffron, S. Falkow. 1979. Identification <br />                                                           protein encoded transposable element Tn3 <br /> Tn3. Tn2603-encoded potential transposase             required transposition. Nature (London) <br /> a molecular weight similar            282:797-801. <br /> transposase specified Tn3 (Fig. 3). This      7. Heffron, F., P. Bedlnger, J. J. Champoux, S. Falkow. <br /> interesting observation consideration           1977. Deletions affecting transposition antibiotic <br />                                                           resistance gene. Proc. Natl. Acad. Sci. U.S.A. 69:702- <br /> divergency universality transposon-specif-          706. <br /> ic enzymes in <br /> </body></html>