<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Doyle et al. BMC Genomics 2010, 11:222 <br /> http://www.biomedcentral.com/1471-2164/11/222 <br />  <br />  <br />  <br />  <br />     RESEARCH ARTICLE                                                                                                                                 Open Access <br />  <br /> Drug target prediction prioritization: using <br /> Research article <br />  <br /> orthology predict essentiality parasite <br /> genomes <br /> Maria A Doyle1,3, Robin B Gasser*2, Ben J Woodcroft1, Ross S Hall2 Stuart A Ralph*1 <br />  <br />  <br />  <br />     <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />     Background: New drug targets urgently needed parasites socio-economic importance. Genes are <br />     essential parasite <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span>survival</span> highly desirable targets, information genes lacking, gene knockouts <br />     knockdowns difficult perform species parasites. We examined applicability large-scale <br />     essentiality information model eukaryotes, Caenorhabditis elegans, Drosophila melanogaster, Mus musculus <br />     Saccharomyces cerevisiae, discover essential genes genomes. Parasite genes lack <br />     orthologues host desirable selective targets, examined prediction essential genes within <br />     subset. <br />     Results: Cross-species analyses showed evolutionary conservation genes presence essential <br />     orthologues strong predictors essentiality eukaryotes. Absence paralogues a <br />     general predictor increased relative essentiality. By combining orthology essentiality criteria can <br />     select gene sets five-fold enrichment essential genes compared random selection. We how <br />     quantitative application criteria used predict ranked list potential drug targets Ancylostoma <br />     caninum Haemonchus contortus - blood-feeding strongylid nematodes, presently limited <br />     sequence data functional genomic tools. <br />     Conclusions: The present study demonstrates utility using orthology information multiple, diverse <br />     eukaryotes predict essential genes. The data emphasize challenge identifying essential genes among <br />     parasite absent host. <br />  <br />  <br /> Background                                                                                  potential drug targets potentially accessible many <br /> Until recently, search novel drugs parasites                                organisms. The major challenge only <br /> has carried mainly using approaches                                           identify putative targets, prioritize them, <br /> directly screen inhibition parasite growth lethal-                                available resources focused those <br /> ity. The current industry regulatory focus target-                                   likely lead effective treatments/drugs. This <br /> based drug development meant search                                        aspect pressing neglected infectious diseases <br /> new anti-parasitic compounds moved target-                                    cause disproportionate burden developing <br /> based paradigm. The completion genome projects                                       countries costs drug develop- <br /> large-scale expressed sequence tag (EST) surveys                                      ment process deterred investment pharma- <br /> range parasites means tens thousands                                      ceutical industry [1]. Thus, well-considered approaches <br />                                                                                             predict promising targets identify <br /> * Correspondence: robinbg@unimelb.edu.au, saralph@unimelb.edu.au <br /> 2 <br />                                                                                             likely essential required increase <br />   Department Veterinary Science, The University Melbourne, 250 Princes <br /> Highway, Werribee, Victoria, 3030, Australia                                                likelihood lead compounds proceed commer- <br /> 1 Department Biochemistry &amp; Molecular Biology, Bio21 Molecular Science                   cial development [2]. <br /> and Biotechnology Institute, The University Melbourne, Victoria, 3010,                     Identifying essential genes pathogens important <br /> Australia <br /> Full list author information available end article                      chemical inhibition non-essential genes is <br />                                                                                             unlikely result death infectious agent, <br />                                             &#194;&#169; 2010 Doyle et al; licensee BioMed Central Ltd. This Open Access article distributed terms Creative Commons <br />             BioMed Central Attribution License (http://creativecommons.org/licenses/by/2.0), permits unrestricted use, distribution, reproduction in <br />                                             medium, provided original work properly cited. <br />  Doyle et al. BMC Genomics 2010, 11:222                                                                              Page 2 14 <br /> http://www.biomedcentral.com/1471-2164/11/222 <br />  <br />  <br />  <br />  <br /> whereas use non-essential molecules, example,         screening uncertain compared organisms <br /> for vaccine targets rapidly select mutants        screens, future screening single molecule targets <br /> evade protective immune responses. Essentiality, how-            likely measure loss-of-function linked lethality, <br /> ever, difficult define application neces-   searching modulators agonists. There- <br /> sarily restricted experimental conditions           fore, prediction identification essential genes <br /> tested. Despite limitation, indispensability        parasitic helminths considered important step <br /> defined laboratory condition useful starting            prioritization drug targets nematodes. <br /> point identifying biological processes likely      There significant growth of <br /> be crucial survival organism natural         sequence data nematodes, enabling prediction of <br /> conditions. The proliferation genome sequencing proj-         numerous potential drug targets. Much focus been <br /> ects followed systematic analyses gene            sequencing ESTs [17-23] due, part, diffi- <br /> knockouts and/or knockdowns determine essentiality            culties producing sufficient quantities genetically <br /> in number model eubacteria eukaryotes,          homogenous material heterologous worm popula- <br /> Escherichia coli, Saccharomyces cerevisiae                   tions genomic sequencing. However, progress <br /> Caenorhabditis elegans [3-8]. Unfortunately, infec-          genome sequencing (e.g., Brugia <br /> tious agents currently amenable whole-genome              malayi) [24], program recently received fund- <br /> approaches experimental testing essentiality,         ing sequencing range socio-economically <br /> such promising candidate drug targets indi-         important nematodes order Strongylida [25]. The <br /> vidually validated genetically. This particularly true     data emerging sequencing projects have <br /> eukaryotic parasites, genetic tools char-       spurred identification drug targets parasitic <br /> acterize drug vaccine targets limited,          nematodes using bioinformatic techniques (e.g., [26]), but <br /> absent others.                                          testing essentiality targets generally <br />    A particularly challenging area development         absent. Members Strongylida prop- <br /> novel compounds combat parasitic helminths ani-            agated (through entire life cycle) culture vitro. In <br /> mals. Such helminths major socio-economic                 addition, gene silencing (double-stranded RNA interfer- <br /> importance; billion people infected         ence, RNAi) does work effectively parasitic <br /> soil-transmitted worms (= geohelminths),             nematodes order assessed far, that, cur- <br /> blood-feeding hookworms Ancylostoma duodenale                rently, major limitations conducting func- <br /> Necator americanus, common roundworm, Ascaris                tional studies different developmental stages these <br /> lumbricoides, whipworm, Trichuris trichiura              parasites [27-29] order pre-validate potential drug <br /> [9,10]. These parasites impose worldwide annual          targets. <br /> burden 39 million Disability Adjusted Life Years                The paucity experimental tools characterize drug <br /> (DALYs) cause adverse effects human               targets Strongylida intensifies need predict <br /> health, particularly children [10]. Similarly, parasitic      putative targets likely essential; in <br /> nematodes livestock, cattle sheep,           case, predictions necessarily based inference <br /> cause substantial economic losses diseases            orthologues model organisms. However, ratio- <br /> they cause, billions dollars spent annually       nal method applying knowledge essentiality <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> treatment gastro-intestinal nematodes alone. Resis-           orthologues species lacking. Taking advantage <br /> tance emerging main classes anthelmint-      recent advances genome orthology mapping <br /> ics used combat parasitism [11-14], creating        [30] availability large-scale eukaryotic essenti- <br /> need new classes compounds.                               ality datasets [31-33], created database anal- <br />    Although desirable, anthelmintic targets neces-        yse essentiality predicted existing data <br /> sarily need encoded essential genes. For example        model eukaryotes. The designation essentiality in <br /> a number anthelmintics (such aldicarb trichlo-         diverse organisms means quite different things. In <br /> rfon) target acetylcholinesterases, perturbations         yeast, essentiality refer failure grow culture, <br /> silencing C. elegans acetylcholinesterase genes           mice encompass failure embryos <br /> result viable phenotye (but drug             progress live offspring. <br /> resistant phenotypes) [15]. Likewise, levamisole targets           In present study, tested disparate <br /> acetylcholine receptors, disruption           forms essentiality predictive corresponding <br /> subunits receptors leads viable levami-       phenotypes organisms definitions <br /> sole resistant <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span>phenotype</span>[16]. Despite precedence         essentiality strongly predictive essentiality even <br /> successful drugs act agonist relationships          relatively distantly related eukaryotes. We applied les- <br /> with non-essential molecules, essentiality desirable        sons analyses prediction essential <br /> character genes pursued target-centric             genes parasitic nematodes. Members order <br /> approach. Although relative success target-based          Strongylida considered relatively closely related <br />  Doyle et al. BMC Genomics 2010, 11:222                                                                                         Page 3 14 <br /> http://www.biomedcentral.com/1471-2164/11/222 <br />  <br />  <br />  <br />  <br /> <span id='am-6' about='protege:TO' typeof='owl:Thing'>to</span> Caenorhabditis elegans, clade V [34],     genetic experiments involving mouse genes did a <br /> that considered appropriate infer gene function       higher percentage reports lethality/essentiality than <br /> from abundant functional genomic data (including         organisms studied (Figure 1), reflect <br /> whole genome RNAi knockdown) exist C. elegans         selection bias. Nonetheless, patterns observed the <br /> [26,35,36]. In addition rational prediction essenti-   prediction essentiality orthology M. mus- <br /> ality orthology essentiality data, integrated      culus dataset paralleled organisms <br /> recently developed gene network connectivity data [37]         assessed, supporting inclusion present analy- <br /> to predict key, essential genes offer         ses. <br /> promising testable drug targets. <br />   Another criterion employed predic-       Prediction essentiality based phyletic characteristics - <br /> tion drug targets infectious organisms absence    presence paralogues <br /> of corresponding host orthologue, maximise          We interrogated database determine par- <br /> opportunities pharmacological selectivity. However,        alogy useful predictor essentiality. Intuitively, <br /> many successful exceptions criterion exist;        belonging paralogous family suggest some <br /> example, genuine targets differentially regulated       degree redundancy compensate dis- <br /> between pathogen host [38], protein turnover        rupted gene; indeed, previous studies C. elegans S. <br /> more rapid host pathogen [39], targets         cerevisiae indicated genes paralogues are <br /> may active proliferating cancerous cells     likely essential [42,43]. However, mouse, a <br /> the host [40], sufficiently structurally different   recent analysis [44] revealed difference genes <br /> between host pathogen allow selectivity [41]. We        paralogues. These studies used different <br /> investigated effect excluding genes possess        criteria define paralogues and, cases, included <br /> mammalian orthologues discovered remain-          genes cause sterility essential, making dif- <br /> ing genes likely essential. Therefore,     ficult compare results studies. Therefore, to <br /> we developed flexible prioritization methods         compare effect paralogy likelihood lethal- <br /> are limited requirement genes lack host      ity different species, used single method for <br /> orthologues. In present study, applied tech-      <span id='am-3' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span> paralogue (OrthoMCL), combined <br /> niques predict essential drug targets large EST       existing large-scale essentiality data C. elegans, <br /> data sets representing blood-feeding strongylid nem-       D. melanogaster, M. musculus S. cerevisiae. The pres- <br /> atodes major socio-economic importance,              ence paralogues inferred OrthoMCL groups; <br /> Haemonchus contortus (the barber&apos;s pole worm small          gene species an <br /> ruminants) Ancylostoma caninum (canine hook-               OrthoMCL group, genes classified genes <br /> worm). We combined prioritization strategy           paralogues. The probability essentiality genes <br /> other criteria predict molecules drug targets these <br /> haematophagous nematodes proof-of-principle <br /> application using orthology predict essentiality. <br />  <br /> Results <br /> Database essentiality <br /> To enable comparison phenotypes four <br /> model organisms - Caenorhabditis elegans, Drosophila <br /> melanogaster, Mus musculus Saccharomyces cerevi- <br /> siae - database constructed allow mapping of <br /> phenotype data orthologues. Orthology was <br /> determined using OrthoMCL database [30], was <br /> populated phenotype data species. In <br /> the case C. elegans, D. melanogaster S. cerevisiae,      Figure 1 Effect paralogy probability essentiality genes. <br /> phenotype information available 94%, 75%            This graph shows probability essentiality genes and <br /> 88% protein-coding genes respectively. For M. mus-       paralogues C. elegans, D. melanogaster, M. musculus S. <br /> culus, gene knockouts/knockdowns represented 20%             cerevisiae. Presence paralogues means gene from <br />                                                                 species present OrthoMCL group. Percentages num- <br /> all protein-coding genes. Because genes chosen for <br />                                                                 bers genes lethal phenotypes expressed fraction genes <br /> <span id='am-319' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-320' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-321' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-322' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-323' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-324' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-325' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-326' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-327' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-328' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-330' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-331' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-332' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-333' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-334' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-335' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-336' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-337' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-338' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-339' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-341' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-342' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-344' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-347' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-348' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-349' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-350' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-351' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-352' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-353' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-354' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-355' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-356' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-357' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-358' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-359' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-360' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-362' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-363' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-364' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-365' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-366' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-367' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-368' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-369' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-370' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-372' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-373' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-374' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-375' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-376' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-378' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-380' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-381' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-382' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-383' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-384' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-385' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-386' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-389' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-390' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-391' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-392' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-393' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-394' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-395' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-396' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span>knockout</span> analysis mice selected address spe-         available phenotype information species. Differences in <br /> cific phenotypic questions, possible non-       essentiality genes versus paralogues highly sta- <br /> random collection introduce bias            tistically significant (P &lt; 10-04), determined using Fisher&apos;s exact test <br /> away genes expected essential. Indeed,      (Additional File 2). <br />  Doyle et al. BMC Genomics 2010, 11:222                                                                            Page 4 14 <br /> http://www.biomedcentral.com/1471-2164/11/222 <br />  <br />  <br />  <br />  <br /> with paralogues shown Figure 1              tor essentiality conservation among <br /> Additional File 1: Table S1.                                     organisms diverged recently. We also <br />   For C. elegans S. cerevisiae, genes          tested conservation orthologues among <br /> without paralogues significantly likely          multiple species improve probability essen- <br /> essential genes paralogues (Figure 1). For C.          tiality particular species. For nearly combina- <br /> elegans, 1.6-fold greater probability essenti-    tions, case (see Figure 2). For <br /> ality genes paralogues (P &lt; 10-33, Fisher&apos;s          example, 51% C. elegans genes orthologues S. <br /> exact test, Additional File 2), S. cerevisiae,       cerevisiae essential, 53% C. elegans genes that <br /> there 3.8-fold greater probability essentiality (P &lt;    orthologues S. cerevisiae D. melano- <br /> 10-51). For D. melanogaster, genes paralogues            gaster essential. There exceptions whose <br /> were likely essential (1.4-fold, P &lt; 10-12),     causes unclear- e.g., genes present species, <br /> though lesser extent C. elegans S. cerevi-     C. elegans, D. melanogaster, S. cerevisiae M. muscu- <br /> siae. The incomplete dataset M. musculus differed             lus, likely essential knocked in <br /> from species genes with-        Drosophila present for- <br /> out paralogues (0.9-fold, P &lt; 10-04) likely      mer species (Figure 2). <br /> essential genes paralogues. <br />                                                                  Prediction based orthology mapping existing <br /> Prediction essentiality based phyletic characteristics -   essentiality data species <br /> presence orthologues                                          Subsequently, tested presence an <br /> Genes shared species hypothe-           essential orthologue predictive essentiality <br /> sized previously likely          gene presence non-essential orthologue. <br /> essential [45]. However, evidence pro-     This explored previously bacterium <br /> posal tested quantitatively eukaryotes,         Pseudomonas aeruginosa, essential genes have <br /> we unaware published studies demonstrated           predicted identifying genes homologous <br /> the effect breadth extent conservation        essential genes unicellular micro-organisms, <br /> large evolutionary distances probability essenti-      Mycoplasma genitalium, Haemophilus influen- <br /> ality. Indications gene conservation used       zae, Vibrio cholerae, Staphylococcus aureus, E. coli S. <br /> a predictor essentiality eukaryotes comes           cerevisiae [47]. This set genes predicted essential <br /> study 315 essential genes Danio rerio (zebrafish)          agreement genes shown experimen- <br /> [46], essential genes likely       tally essential. However, essentiality an <br /> be conserved non-essential genes. To test effect        orthologue used predict essentiality multi- <br /> of conservation genes essentiality, analysed        cellular organisms tested. For C. elegans, D. <br /> frequency essentiality genes orthologues           melanogaster, M. musculus S. cerevisiae, assessed <br /> in C. elegans, D. melanogaster, M. musculus S. cerevi-       essentiality species used pre- <br /> siae. For species, tested effect      dictor essentiality second species. To for <br /> conservation gene species essenti-        query species (ie C. elegans, D. melanogaster, M. <br /> ality (query) species. For example, examine        musculus S. cerevisiae), assessed probability of <br /> how presence orthologue S. cerevisiae          essentiality genes lethal phenotypes the <br /> predicts essentiality C. elegans, identified         species. When comparing phenotypes linked <br /> orthology groups comprising genes spe-            particular genes species, limited analysis <br /> cies, then, C. elegans genes groups,        genes paralogues species being <br /> we calculated percent essential. The find-         compared; conducted avoid difficulty of <br /> ings shown Figure 2 Additional File 1: Table S2.      comparing phenotypes individual paralogues were <br />   For genes species, existence     associated different phenotypes. To test whether <br /> orthologue species increased probability         having essential orthologues S. cerevisiae predicted <br /> essentiality, distant source ortho-      essentiality corresponding C. elegans genes, we <br /> logue, greater effect. For C. elegans, probabil-    identified orthology groups containing non-paralogous <br /> ity essentiality gene paralogues selected       C. elegans S. cerevisiae genes subjected <br /> at random 18%; probability increased 38%          essentiality analysis. We compared frequency <br /> the gene orthologue D. melanogaster             essentiality C. elegans genes grouped with <br /> 51% orthologue existed distant relative,       essential non-essential S. cerevisiae orthologues. We <br /> S. cerevisiae (Figure 2). Conservation greater evo-       tested essentiality multiple species <br /> lutionary distance appears better indica-      improved probability essentiality. The results the <br />  Doyle et al. BMC Genomics 2010, 11:222                                                                                                          Page 5 14 <br /> http://www.biomedcentral.com/1471-2164/11/222 <br />  <br />  <br />  <br />  <br />                           	 <br />   <br />   <br />                                <br />                                <br />            	  <br />  <br />  <br />  <br />  <br />                                <br />                               <br />                               <br />                                <br />                               <br />  <br />  <br />                           	 <br />   <br />   <br />                                <br />           	 <br />     <br />    <br />  <br />  <br />  <br />  <br />                                <br />                                <br />                               <br />                               <br />                                <br />                               <br />                                                                                                                                               <br />           <br />                                                                                                                                               <br />   <br />          <br />                           	 <br />   <br />   <br />                                <br />               <br />  <br />  <br />  <br />  <br />                                <br />                                <br />                               <br />                               <br />                               <br />                                <br />  <br />  <br />                           	 <br />   <br />   <br />                                <br />                  <br />  <br />  <br />  <br />  <br />                                <br />                                <br />                                <br />                                <br />                                <br />                                 <br />  <br />                                                                                                                                     <br />  <br />  Figure 2 Effect gene conservation probability essentiality genes. This graph shows probability essentiality genes C. <br />  elegans, D. melanogaster, M. musculus S. cerevisiae respectively, modified phyletic profile. The percentages number genes with <br />  lethal phenotypes divided total number genes orthologues species compared. Specificity refers proportion genes <br />  selected subset essential sensitivity refers proportion total number essential genes species recov- <br />  ered selected subset. In cases, essentiality assessed genes paralogues. The probability essentiality genes the <br />  specified subsets versus gene selected random query genome statistically significant (P &lt; 10-05), determined using Fisher&apos;s <br />  exact test (Additional File 2). C = C. elegans, D = D. melanogaster, M = M. musculus, D = D. melanogaster, S = S. cerevisiae. <br />  Doyle et al. BMC Genomics 2010, 11:222                                                                                            Page 6 14 <br /> http://www.biomedcentral.com/1471-2164/11/222 <br />  <br />  <br />  <br />  <br /> predictions C. elegans species                     OrthoMCL group did gene the <br /> shown Figure 3 Additional File 1: Table S3, respec-                     mammals considered (i.e., Homo sapiens, M. mus- <br /> tively.                                                                        culus Rattus norvegicus). We calculated the <br />    Our analysis showed gene orthologue                          probability essentiality genes. C. elegans genes <br /> with lethal knockout/knockdown phenotype                            lacked mammalian orthologues likely <br /> likely essential orthologue non-                        essential; probability essentiality decreased <br /> essential; distant orthologue, greater                        16% 8% gene selected random, was <br /> was probability essentiality. Thus, existence                    6% gene paralogues (Table 1, Figure 3). <br /> an orthologue associated <span id='am-397' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-398' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-399' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-400' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-401' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-402' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-403' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-404' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-405' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-406' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-407' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-408' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-409' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-410' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-411' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-412' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-413' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-414' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-415' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-416' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-417' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-418' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-419' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-420' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-421' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-422' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-423' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-424' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-425' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-426' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-427' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-428' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-429' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-430' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-431' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-432' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-433' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-434' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-435' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-436' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-437' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-438' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-439' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-440' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-441' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-442' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-443' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-444' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-445' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-446' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-447' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-448' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-449' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-450' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-451' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-452' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-453' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-454' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-455' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-456' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-457' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-458' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-459' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-460' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-461' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-462' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-463' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-464' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-465' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-466' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-467' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-468' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-469' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-470' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-471' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-472' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-473' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-474' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span>lethal phenotype</span> S.                         This substantial decrease probability essentiality <br /> cerevisiae highly predictive lethality C. elegans                    important consequences drug target prediction <br /> (72%), greater presence orthologue                     explicit previously. Researchers <br /> S. cerevisiae (51%). Combining presence essen-                       quite likely examine genes exactly category <br /> tial orthologue evolutionarily distant relative                     (i.e. present pathogen absent host), but <br /> (S. cerevisiae) additional, closely-related species                    low &quot;raw&quot; probability essentiality these <br /> (i.e., D. melanogaster) increased probability -                   genes (6-8%) highlights challenge prediction of <br /> C. elegans genes orthologue D. melano-                         essential drug targets subset, we <br /> gaster S. cerevisiae, lethal phenotype,                    address here. <br /> an 81% probability lethality. <br />                                                                                Prediction drug targets Anyclostoma caninum and <br /> The effect drug-target selectivity predictions                        Haemonchus contortus: combining &apos;assayability&apos; the <br /> essentiality                                                                   prediction essentiality <br /> Anti-parasitic drugs need selectively inhibit kill                   The present findings applied prediction of <br /> parasite causing unacceptable toxicity                          drug targets haematophagous parasitic nematodes, as <br /> host. One way pursuing targets drugs                         outlined flow diagram (Figure 4). The parasites Ancy- <br /> focus parasite genes lack host orthologues. As                     lostoma caninum Haemonchus contortus were <br /> aim identify targets parasitic nematodes                       selected represent parasitic nematodes order <br /> infect mammals, examined possible effects                            Strongylida (&quot;clade V&quot; [34]), significant EST <br /> essentiality focusing parasite genes lack mam-                      sequence data exist. All available EST data A. caninum <br /> malian orthologues. To this, analysed probabil-                      H. contortus (comprising 21,975 80,551 ESTs, <br /> ity essentiality C. elegans genes                           respectively) downloaded NCBI database. <br /> possessed lacked mammalian orthologues. To obtain                           These ESTs mapped OrthoMCL orthol- <br /> the set C. elegans genes mammalian ortho-                           ogy groups BLASTX analysis individual EST <br /> logues, identified C. elegans genes                         sequences OrthoMCL proteins using the <br />                                                                                OrthoSelect method [48]. A caveat EST-based <br />                                                                                analysis incompleteness genomes the <br />                                                                                parasitic nematodes means poorer iden- <br />                                                                                tification paralogues orthologues. This analysis <br />                                                                                followed extraction C. elegans orthologues <br />                                                                                orthology groups. Previously, showed that <br />                                                                                having essential orthologue better predictor of <br />                                                                                essentiality just having orthologue (e.g., C. ele- <br />                                                                                gans, gene orthologue essential D. <br />                                                                                melanogaster 1.6-fold likely essential if <br />                                                                                C. elegans gene just orthologue D. melano- <br />                                                                                gaster). We focused subset C. elegans <br />                                                                                orthologues parasitic nematodes essen- <br />                                                                                tial (i.e., genes lethal RNAi phenotypes C. ele- <br />  Figure 3 Effect conservation essential genes probability             gans). <br />  essentiality. This graph shows likelihood essentiality C. el- <br />  egans genes essential orthologues organ-           Assayability requirement: enzymes, ion channels and <br />  isms; D. melanogaster, M. musculus S. cerevisiae. The percentages         GPCRs <br />  number genes lethal phenotypes divided total           To select targets amenable vitro <br />  number genes orthologues species. Essentiality as- <br />                                                                                screening growth inhibition lethality, focused <br />  sessed genes paralogues. C = C. elegans, D = D. melano- <br />  gaster, M = M. musculus, S = S. cerevisiae.                                   genes encoding proteins functional assays <br />  Doyle et al. BMC Genomics 2010, 11:222                                                                                          Page 7 14 <br /> http://www.biomedcentral.com/1471-2164/11/222 <br />  <br />  <br />  <br />  <br /> Table 1: Combining essentiality selectivity <br />  <br />                                                                           Probability essentiality <br />  <br />  <br />                                        Genes                 With paralogues                   Without paralogues <br />                                             paralogues <br />  <br />  <br />            C. elegans                            16%                                 11%                                18% <br />            (all genes)                    (2941/18860 genes)                   (656/5988 genes)                  (2285/12872 genes) <br />  <br />  <br />    C. elegans (excluding genes                   8%                                  6%                                 9% <br />   mammalian orthologues)             (1090/13307 genes)                   (249/4431 genes)                   (841/8876 genes) <br />   This table demonstrates effect excluding genes mammalian orthologues probability essentiality. Percentages shown <br />   number genes lethal RNAi (gene knock-down) phenotypes divided total number genes category. The <br />   numbers genes given brackets. <br />  <br />  <br /> exist. Thus, examined subset C. elegans genes                Increasing probability essentiality incorporating <br /> encoding proteins inferred enzymatic, ion chan-                information network connectivity <br /> nel, GPCR activity, determined GO term annota-                Recently, Lee et al. [37] constructed probabilistic func- <br /> tion (i.e., GO:0003824, GO:0005216 GO:0004930,                 tional interaction network C. elegans, largest such <br /> subterm thereof ). All current drugs clinical use           network date, comprising ~82% C. elegans genes. <br /> nematodes target protein catego-               This network integrates data proteins genes (e.g., <br /> ries. GPCRs targets 50% pharma-                microarray data interaction association informa- <br /> ceutical drugs [49], making attractive targets            tion) using information C. elegans genes as <br /> parasitic nematodes. We enzymes                   orthologues species. Using network, <br /> likely essential non-enzymes C. ele-               authors showed gene connectivity predictive of <br /> gans S. cerevisiae - C. elegans, 16% genes              essentiality - genes highly connected net- <br /> were essential, 21% enzymes essential;                 work likely essential; shown to <br /> for S. cerevisiae, 19% genes essential, 24%           feature conserved eukaryotes <br /> enzymes essential - so, focusing enzymes,                (including S. cerevisiae M. musculus). An indepen- <br /> further increased likelihood identifying essential              dent study Hwang colleagues showed that <br /> genes.                                                                 genes encoding proteins higher connectivity pro- <br />                                                                        tein-protein interaction networks likely be <br /> Essentiality requirement: quantitative formula                     essential E. coli S. cerevisiae [50]. As genes with <br /> maximum prediction essentiality based present                higher network connectivity scores (as assigned Lee et <br /> analysis                                                               al) likely essential lower <br /> We applied quantitative weighting scheme genes                scores [37], gave greater weighting genes with <br /> that met criterion assayability, according              higher connectivity scores. <br /> present analysis relative importance feature              We combined quantitative weighting scheme with <br /> in determining essentiality. For example, showed               network connectivity scores produce prioritized list <br /> having paralogue increased probability essenti-              putative drug targets A. caninum H. contortus <br /> ality 1.6-fold C. elegans (Table 1), having ortho-            (Additional file 3). These represent predicted C. elegans <br /> logue paralogues S. cerevisiae D.                        orthologues A. caninum H. contortus ESTs that <br /> melanogaster increased probability 3.3 2.4-fold,                chosen assayability (enzyme, ion-channel or <br /> respectively. (Figure 3). Furthermore, having ortho-                GPCR), quantitative ranking scheme applied to <br /> logue paralogues essential S. cerevi-              predict essentiality. Genes included list they <br /> siae D. melanogaster increased probability                  related lethal RNAi phenotype C. elegans. <br /> essentiality 5.2-fold. These fold increases sta-             Although list comes incomplete EST snap- <br /> tistically significant (P &lt; 0.0005), determined using               shot haematophagous helminths, nevertheless <br /> Fisher&apos;s exact test (Additional File 2). This weighting                recovers validated anthelmintic drug targets; beta- <br /> should score genes numerically order prob-              tubulin, target Benzimidazole [51]; glutamate-gated <br /> ability genes essential.                                 chloride channel, target avermectin/milbemycin <br />                                                                        anthelmintics [52] succinate dehydrogenase/fumar- <br />  Doyle et al. BMC Genomics 2010, 11:222                                                                                         Page 8 14 <br /> http://www.biomedcentral.com/1471-2164/11/222 <br />  <br />  <br />  <br />  <br />                                                                             exact probability essentiality uncharacterized <br />    Orthologue C. elegans?                                                gene, weightings estimation rel- <br />                                                                             ative likelihood essential genes being <br />                                     yes                  no <br />                                                                             considered candidate targets. <br />    Enzymes, Ion channels &amp; GPCRs? <br />                                                                             Discussion <br />                                     yes                                  Defining essential genes central strategy priori- <br />                                                                             tization drug targets infectious organisms. Although <br />    Lethal phenotype C. elegans? <br />                                                                             non-essential genes act drug targets con- <br />                                     yes                                  verting non-toxic precursors toxic products, most <br />                                                                             anti-microbials act inhibiting function essen- <br />                         Lacks paralogues?                                   tial process, leading death pathogen, either <br />                                                                             direct absence essential process or, indi- <br />               yes                                                        rectly, stress-related response. A number of <br />                   Orthologue distant relative <br />                                                                             anthelmintic drugs (e.g. levamisole [16] ivermectin <br />                                                                             [54]) appear function agonists non-essential <br />               yes                                                        genes (for review anthelmintic modes action see <br />                                                                             [55]). Although search drug targets essen- <br />             Essential orthologue distant relative                        tial genes useful approach prioritization tar- <br />                                                                             gets, inability methodology predict of <br />               yes                                      no <br />                                                                             successful anti-parasitic drug targets limitation of <br />            Presence orthologue close relative                         practice. <br />                                                                               There attempts define predict <br />               yes                                                        essential genes infectious agents; cases, where <br />                                                                             sophisticated functional genomic tools avail- <br />              Essential orthologue close relative                         able, possible systematic dis- <br />               yes                                                        ruption candidate genes, example, Streptococcus <br />                                                                             pneumoniae [56], Candida albicans [57], Aspergillus <br />                     Network connectivity score                              fumigatus [58]) Mycobacterium tuberculosis [7,8]. <br />                                                                             However, vast majority infectious organisms, <br />               yes                                                        high-throughput experimental interrogation essential- <br />                                                                             ity currently possible, efforts define essenti- <br />      Prioritized list targets parasitic nematodes                    ality based bioinformatic predictions [59- <br />                                                                             63]. <br />  Figure 4 Prioritizing essential nematode drug targets. This flow di-         Here, attempted define general characters <br />  agram outlines approach drug target prediction prioritiza- <br />  tion parasitic nematodes. Some steps clear &quot;keep reject&quot; <br />                                                                             predictive essentiality eukaryotic genes <br />  decisions genes failing criteria easily   orthology. Previous studies shown con- <br />  amenable development unacceptable probabili-         servation genomes itself predictive feature of <br />  ty essential. Other steps used quantitatively refine   essential genes. Studies Butland, [64] Hwang [50] Kro- <br />  weighting produce ranked list essential drug targets.              gan [65] colleagues showed essential genes S. <br />                                                                             cerevisiae E. coli widely phyletically <br /> ate reductase, C. elegans equivalent helminth                    distributed, connected physical <br /> target thiabendazole [53].                                               functional protein-protein interactions. Our data <br />   A likely application prioritization method                  broad evolutionary conservation genes <br /> be focus particular subset enzymes                   (among multicellular eukaryotes) predictive essenti- <br /> investigator expertise. For example, aim                  ality, essentiality orthologues a <br /> identify promising kinase drug targets, kinases                    strong predictor. Combining individual criteria are <br /> the ranked list extracted appraised. There                     independently predictive essentiality increases the <br /> are 49 predicted kinases (Figure 5) H. contortus                 specificity prediction - example, C. elegans gene <br /> A. caninum EST datasets. Those list                       selected random 16% likely essential, <br /> would expected likely essential                      C. elegans genes lack paralogues have <br /> these parasitic nematodes, based present anal-                   essential orthologues S. cerevisiae D. melanogaster <br /> yses. Although numerical weighting                       81% likely essential. This dramatic increase in <br />  Doyle et al. BMC Genomics 2010, 11:222                                                                                                                                                                                                              Page 9 14 <br /> http://www.biomedcentral.com/1471-2164/11/222 <br />  <br />  <br />  <br />  <br />                                                                                                                                                         	    <br />                                                                                                                       	                                                            <br />      	  <br />                                                                                                                                                  <br />     <br />   <br />     	  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                            <br />     <br />  <br />  <br />  <br />  <br />                                                                                                                            	      <br />  <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                               	      <br />  <br />                                                                                                                                                                                                                             	  	    <br />  <br />  <br />  <br />  <br />                                                                                                                	 <br /> 	   <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                   	 <br /> 	   <br />                                                                                                                                                            <br />   <br />  	   <br />                   	 <br />       	      <br />                                     <br />   	       	  	 <br />                                                                                                                                                   <br />      	 	    <br />  <br />  <br />  <br />  <br />                                                                                                                                               <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                  	 <br />  <br />                                                                                                                                                                                                                           	    <br />                                                                                                                                                                                                                                           <br />    <br />  <br />  <br />  <br />  <br />                                                                                                                                                                        <br />     <br />                                                                                                                                                       <br />                       	        %&amp;           %&apos;%    &amp;    &amp;   (    ) (&quot; *                                                                                                                                                    <br />             <br />    	 <br />                    <br />   		         <br />   <br />  	       +,-(*   ) (+,-*                                                                                                                                                                <br />              <br />      <br />                                              . /  0,   <br />        &amp;()     0 1  /    (  /    (    )                                                                                                                                               <br />                    <br />    	           <br />   <br />    <br />      +,-(*   ) (+,-* <br />                                                                                                                                                                               	 <br />                                     <br />           /    (    )                                                                                                                                                                    <br />           	     <br />                                     <br />          1/) 1   &amp;&apos;   /)  / (      )                                                                                                                                                                  <br /> 	 <br />                        <br />         2    <br />          $#(*   )                                                                                                                                                                                  <br />     <br />   <br />                     	 	        3 4  <br />          &apos; /)   (    ) 5( # (6  1   7(  %   / (8 2  &apos;                                                                                                                                                  <br />   <br />                        <br />             	  <br />        1/) 1/2 3  /    (    )                                                                                                                                                                         <br />                        <br />         %&amp;   <br />         %&apos;%    &amp;    &amp;   (    )                                                                                                                                                                     <br />   <br />    <br />                                     <br />         % )   (    ) (,                                                                                                                                                                                  <br />                      <br />   <br />             	  <br />       9 # &quot; #(    ) (                                                                                                                                                                                  <br />                                             % )   (    ) (,(6  )  / 0&amp;    7                                                                                                                                                                  <br />                                %&amp;   <br />         %&apos;%   (&amp;    &amp;   (    ) ( <br /> (1/2/ /:                                                                                                                                                                <br />                                %&amp;  	        %&apos;%   (% &amp;    &amp;   (    ) (&quot; *	                                                                                                                                                              <br />      <br />                      	         )    <br />         ;+(# /    (*   )                                                                                                                                                                            	 	  <br />                                     <br />         %&apos;%    &amp;    &amp;   (    ) (    <br />                                                                                                                                                                      <br />                        <br />         4.4  <br />         2  /  %(%1 %  /   ()     0 1  /    (  /    (    )                                                                                                                                              <br />                    	  <br />  <br />          )             4/)/2  (  /    (; (    ) 0(+;*    (    ) (1/2/ /:                                                                                                                                         <br />   	  <br />                                )    <br />         $+*(  /    (    ) (8 2  &apos;                                                                                                                                                                        <br />                       <br />               <br />         )     0 1  /    (    ) (6*,&lt; <br /> 0;&lt;  <br /> 0&lt; 2 <br /> ().48 2  &apos;7                                                                                                                                               <br />                    <br />             %)    <br />        % )   (    ) (6)     0 1  /    0 &apos; /)   (  /    (    ) 7                                                                                                                                     <br />     <br />                     <br />   	        &amp;    <br />         2  /:    %  3   &amp;(  /    (    ) (    ) (    )                                                                                                                                                   <br />                      <br />                        )     0 1  /    (    )                                                                                                                                                                          <br />                      <br />  	         1   <br />          1&apos;2 &amp;   (    )                                                                                                                                                                                 <br />                                     <br />          1/ 4  &amp;  :(;  0=1 (    )                                                                                                                                                        <br />                <br />                      <br />              <br />   <br />          &apos; .3   (    )                                                                                                                                                                   <br />                 <br />                    <br />  	          !&quot;    	        :     (    )                                                                                                                                                                   <br />                <br />                       	        %1   <br />         &quot; * <br />      ()     ( 1  /    (  /    (    )                                                                                                                                                         <br />                    <br />                 <br />           /    (    )                                                                                                                                                                                  <br />  <br />  <br />                     <br />                 <br />   <br />         /    (    )                                                                                                                                                                                <br />  	  <br />                        <br />                          :         &amp;(    )                                                                                                                                                                        <br />   <br />   <br />                      <br />  <br />             <br />     <br />        .%  /) &amp; (&amp;  1/) 1   (    )                                                                                                                                                   <br />                   <br />                     <br />            :)           : &apos;%/:  ()&apos;  1 ) (    )                                                                                                                                                        <br />             	    <br />  <br />                      <br />          24           &amp;.   )  % 8 %  &apos;(    <br />        &amp;(    )                                                                                                                                            <br />                 <br />   <br />                                &quot;   	  <br />       % )   (    ) (6)     0 1  /    0 &apos; /)   (  /    (    ) 7                                                                                                                       <br />                   <br />                    	             <br />            .  &amp;&apos;    (    ) 0 &amp;  &apos;    (    )                                                                                                                                               <br />               <br />   <br />   <br />                    	 	           <br />            1 &gt;/    )                                                                                                                                                                      <br />                 <br /> 	 <br />                      <br />                <br />          /.)  &amp;     (  /    (    )                                                                                                                                                     <br />                  <br />                       	             <br />          . /  0,   <br />        &amp;()     0 1  /    (  /    (    )                                                                                                                             <br />                <br />   <br />                    <br />    <br />              <br /> #    <br />     1 &gt;/    )                                                                                                                                                                      <br />              <br />    <br />  <br />                    <br />             %&amp;           %&apos;%    &amp;    &amp;   (    ) (&quot; *                                                                                                                                                    <br />                  <br />                      <br />   <br />                      % )   (    ) (,,(   1 5(%    &apos;  %().4.                                                                                                                                         <br />                   <br />                      <br />             <br />   <br />  $  <br />      1/) 1/8 .% /    )                                                                                                                                                             <br />               <br />     <br />                     <br />  	         8    <br />          /    %   / ( &apos; /)   (    )                                                                                                                                                    <br />              <br />      <br />                      <br /> 	              <br />         %$ # &amp;    &amp;   (  /    (    )                                                                                                                                                   <br />             <br />  	   <br />  <br />                      	         ):   <br />         )     0 1  /    (  /    (    )                                                                                                                                                 <br />                 <br />   <br />                                :)           )     0 1  /    (    )                                                                                                                                                         <br />                   <br />                    <br />              ) )  <br />          4/)/2  (  /    (; (    )                                                                                                                                                     <br />              <br />      <br />  <br />  <br />  <br />  Figure 5 Prioritized kinases parasitic nematodes A. caninum H. contortus. This shows C. elegans predicted orthologues A. <br />  caninum H. contortus ESTs, identified present study. All genes predicted kinases, lethal RNAi phenotype C. elegans. Targets <br />  prioritized essentiality weighting assigned present study, Wormnet connectivity score. The ranked kinases <br />  (in bold) thirty ranked targets ESTs (Additional File 3). Present mammals defined having orthologue or <br />  mammalian species OrthoMCL 2.0 database (human, rat, mouse). Essential orthologue yeast, drosophila mouse: ? = orthologue <br />  (with paralogues) associated lethal phenotype, ? = orthologue (with paralogues) non-lethal phenotypes recorded, ? = or- <br />  thologue (with paralogues) phenotype information available. Presence paralogues C. elegans determined OrthoMCL <br />  grouping. Presence paralogues A. caninum H. contortus determined described Materials methods. The network connectivity <br />  score sum interaction scores (log-likelihood scores) gene Wormnet core network [37]. <br />  <br />  <br /> the specificity prediction increases probability                                             scriptive queries. In present example, 258 <br /> essentiality dishearteningly low level,                                              candidates remained set 2941 C. elegans genes <br /> randomly chosen drug target candidate likely                                             determined experimentally essential. In practice, <br /> non-essential, reassuring level prioritized can-                                            investigator balance desired specificity of <br /> didates likely essential. However,                                                  query desired pool size selected tar- <br /> increase specificity associated predictions                                           gets, highly weight specific predictors, that <br /> also accompanied decrease sensitivity - gen-                                              genes likely essential ranked top. <br /> uinely essential genes discarded highly pre- <br />  Doyle et al. BMC Genomics 2010, 11:222                                                                           Page 10 14 <br /> http://www.biomedcentral.com/1471-2164/11/222 <br />  <br />  <br />  <br />  <br />    The discovery exclusion genes mamma-         host. We quantitatively applied conclu- <br /> lian orthologues leads sets genes       sions analysis prediction essential <br /> likely essential major implications,        genes blood-feeding parasitic nematodes, combining <br /> highlights additional challenge identification     requirement assayability identify tar- <br /> drug targets allow selectivity. The decreased         gets readily amenable investigation. Our analy- <br /> essentiality readily rationalised, considering        sis led prioritization tractable number of <br /> analyses relationship orthology        potential drug targets leading development of <br /> essentiality. In case, excluded set genes      nematocidal agents. <br /> shared orthology worms mammals. The <br /> resultant set, conserved fewer organisms, con-           Methods <br /> tained higher proportion non-essential genes.               Orthology data <br />    We focused enzymes, ion channels G-protein-            Orthology data obtained OrthoMCL [30], <br /> coupled receptors (GPCRs) vast majority           available http://www.orthomcl.org. OrthoMCL classi- <br /> all drugs target classes [49,66] chemicals             fies genes 87 genomes orthology groups. Files <br /> already exist interrogate targets. Furthermore,       containing protein sequence data orthology group <br /> found proportion essential genes greater      information downloaded OrthoMCL version <br /> enzymes protein-coding genes, C.         2. This dataset contained 20084, 14034, 24435 6718 <br /> elegans S. cerevisiae.                                       proteins C. elegans, D. melanogaster, M. musculus <br />    In study, applied semi- quantitative methods       S. cerevisiae, respectively includes genes assigned <br /> identify essential genes parasitic helminths. These           orthology groups OrthoMCL dataset also <br /> methods equally applicable prediction        assigned group (annotated OrthoMCL <br /> essential genes eukaryotic pathogens EST            &apos;no_group&apos;). Orthologues genes deriving an <br /> or genome projects. We shown genes           ancestral gene separated speciation event, <br /> essential orthologues greatly increased probability       paralogues genes separated gene <br /> of essentiality researcher identify genes      duplication event genome. Subsequent specia- <br /> their parasite essential C. elegans,    tion lead paralogous relationships genes <br /> D. melanogaster, M. musculus and/or S. cerevisiae. As            different genomes (referred out-paralogues) <br /> these species distantly related protozoan spe-        [67]. In practice, identification relationships <br /> cies, essentiality species      non-trivial robust complete genome <br /> highly suggestive essentiality protozoan parasite        data sequences useful phylogenetically informa- <br /> versus genes essential model        tive characters allow unambiguous phylogenetic trees to <br /> organisms.                                                       constructed. This currently computationally or <br />    In addition identifying drug targets, identification    practically feasible multi-eukaryote, genome <br /> promising vaccine candidates priority para-        analyses. The OrthoMCL clustering method provides a <br /> sitic diseases. Although essentiality likely         convenient, necessarily imperfect means estimat- <br /> important characteristic vaccine targets, large variety     ing orthology paralogy. In-paralogues out-paral- <br /> of characters desirable currently diffi-      ogues separated recent duplication event will <br /> cult establish bioinformatically, particularly         normally clustered OrthoMCL group may <br /> incomplete genome. Although characters                 represent gene/protein family, ancient <br /> become tractable near future, current            duplications likely result separate clusters <br /> approach appropriate identifying vaccine anti-        [68]. For purposes paper, genes different <br /> gens.                                                            species share OrthoMCL cluster con- <br />                                                                  sidered orthologues, multiple genes from <br /> Conclusions                                                      species share OrthoMCL <br /> In study, undertook quantitative assessment         cluster considered paralogues. Additional File 4 dem- <br /> essentiality inference using orthology. This analy-      onstrates example phylogenetic trees representa- <br /> sis demonstrates utility examining multiple, diverse      tive types orthologue clusters. <br /> organisms predicting essentiality, shows that <br /> essentiality gene highly predictive essentiality   Phenotype data <br /> for orthologues, organisms extremely           Phenotype information obtained C. elegans, M. <br /> different life cycles. These data highlight poten-      musculus S. cerevisiae databases Wormbase <br /> tial undesirably enriching non-essential genes           WS191 http://www.wormbase.org[69], Mouse Genome <br /> when prioritizing genes pathogen absent            Informatics (http://www.informatics.jax.org; June 2008 <br />  Doyle et al. BMC Genomics 2010, 11:222                                                                             Page 11 14 <br /> http://www.biomedcentral.com/1471-2164/11/222 <br />  <br />  <br />  <br />  <br /> [70]) Saccharomyces Genome Database (http://                   phenotype data available 20% (5008/24435) of <br /> www.yeastgenome.org; Aug 2008) [71], respectively. For             M. musculus OrthoMCL genes. <br /> D. melanogaster used results genome-wide <br /> RNAi screen [33]). In order focus phenotypes            Computational predictions essentiality <br /> result disruption/inhibition proteins, opposed          Custom Ruby scripts written query database <br /> to overexpression phenotypes, restricted analysis           Ruby Rails framework. Genes without <br /> to phenotypes following experiment/mutant                 paralogues defined genes OrthoMCL <br /> types: C. elegans (RNAi), S. cerevisiae (null, reduction        groups gene query species, or <br /> function, repressible), D. melanogaster (RNAi) M.              genes query species OrthoMCL dataset <br /> musculus (included phenotypes following                   assigned &apos;no_group&apos;. The number genes with <br /> experiments: Chemically radiation induced, Chemi-              paralogues obtained subtracting number of <br /> cally induced (ENU), Chemically induced (other), Gene              genes paralogues total number genes <br /> trapped, Radiation induced, Spontaneous, Targeted                  query species. <br /> (<span id='am-241' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-242' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-259' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-272' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-273' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-278' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-289' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-292' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-295' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-297' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-299' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-303' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-315' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-316' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-318' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span>knock-out</span>), Targeted (Reporter), Targeted (Floxed/Flt),             To analyze effect presence orthologues on <br /> Transgenic (random, gene disruption)). Essential genes             probability essentiality, example, effect hav- <br /> were defined follows; C. elegans M. musculus            ing yeast orthologue probability essentiality C. <br /> these genes phenotype description contain-             elegans genes, OrthoMCL groups contained a <br /> ing term &apos;lethal&apos; (see Additional File 5 list   gene species identified. Next the <br /> phenotypes), S. cerevisiae genes           subset groups selected contained one <br /> phenotype &apos;inviable&apos;, D. melanogaster               gene S. cerevisiae C. elegans, were <br /> genes showed lethality Adult, Pupal,               genes paralogues species. Then, for <br /> Before Pupal Eclosion life-stage [33]. We           C. elegans genes groups number genes <br /> exclude possibility phenotypes           lethal phenotypes calculated expressed a <br /> M. musculus result protein disruption/                percent total number C. elegans genes with <br /> inhibition, example, possible chemical           orthologues S. cerevisiae (numbers genes given <br /> spontaneous <span id='am-163' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-164' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-239' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span>mutation</span> result constitutive activa-          Additional File 1: Table S2). <br /> tion protein gain-of-function. However, includ-            To analyze frequency essentiality genes <br /> ing phenotypes result                  essential orthologues C. elegans, example, to <br /> conservative prediction essentiality           examine presence essential orthologue <br /> mean M. musculus genes                   S. cerevisiae predicts essentiality C. elegans, the <br /> falsely classed essential orthologues used          OrthoMCL groups contained gene spe- <br /> predict essentiality C. elegans lead lower           cies identified. Next subset groups were <br /> probability essentiality C. elegans.                        selected contained gene S. cerevi- <br />                                                                    siae C. elegans, followed identifying the <br /> Construction database whole-genome comparisons            subset groups essential S. cerevisiae ortho- <br /> of essentiality                                                    logue. Then C. elegans genes groups the <br /> To enable comparison phenotypes              number genes lethal phenotypes calculated <br /> species, relational database constructed map              expressed percent total number C. ele- <br /> phenotype data species OrthoMCL                 gans genes essential orthologue S. cerevisiae <br /> genes groups. This database PostgreSQL data-             (numbers genes given Additional File 1: Table <br /> <span id='am-553' about='xsp:base' typeof='owl:Thing'>base</span> http://www.postgresql.org used Ruby                S3). <br /> Rails framework http://www.rubyonrails.org. Custom                   The set C. elegans genes mammalian ortho- <br /> tables designed store data OrthoMCL            logues genes OrthoMCL <br /> proteins mapped genes phenotype files           group did contain gene three <br /> using gene identifiers present datasets (identifi-      mammalian species OrthoMCL (Homo sapiens, M. <br /> ers type WBGene00000001 C. elegans, CG3095              musculus Rattus norvegicus) C. elegans genes that <br /> for D. melanogaster, ENSMUSG00000020473 M.                     assigned &apos;no_group&apos; OrthoMCL. <br /> musculus YNL214W S. cerevisiae). The software <br /> used process available http://github.com/            Prediction drug targets parasitic nematodes <br /> wwood/essentiality. Phenotype data available              All available ESTs H. contortus A. caninum were <br /> 94% (18860/20084), 75% (10572/14034), 88% (5908/               downloaded National Center Biotechnology <br /> 6718) C. elegans, D. melanogaster S. cerevisiae             Information (NCBI) comprised 21,975 80,551 <br /> genes OrthoMCL dataset, respectively. In contrast,          ESTs respectively. The ESTs cleaned vector using <br />  Doyle et al. BMC Genomics 2010, 11:222                                                                                    Page 12 14 <br /> http://www.biomedcentral.com/1471-2164/11/222 <br />  <br />  <br />  <br />  <br /> SeqClean, repeats masked Repeatmasker, using        genes enzymes GO identifier catalytic <br /> default parameters programs. We initially used       activity (GO:0003824) specific sub- <br /> the CAP3 program clustering ESTs, however,            terms (e.g., GO:0016301 kinase activity). Similarly, a <br /> manual inspection results led discover   script written identify ion channels GPCRs - <br /> were getting incorrect clustering - clustering ESTs        genes GO identifier ion channel activity <br /> from similar identical genes contig     (GO:0005216) G-protein coupled receptor activity <br /> for example ESTs large gene families colla-      (GO:0004930) specific subterms <br /> gens, lectins, cysteine proteases. We tried increasing    (e.g., GO:0001584 rhodopsin-like receptor activity). The <br /> percent identity cutoff CAP3 program          software written Ruby http://www.ruby-lang.org/ <br /> default 80% 95% incorrect clus-     using RSRuby library http://rubyforge.org/projects/ <br /> tering manual inspection. The ESTs          rsruby/ communicate R http://www.r-project.org <br /> individually mapped OrthoMCL orthology groups              GO.db Bioconductor package http://www.bio- <br /> using initial steps OrthoSelect method             conductor.org/packages/data/annotation/html/ <br /> described Schreiber colleagues [48]. Briefly, ESTs     GO.db.html. Using approach, reference list GO <br /> assigned orthology group sufficient       terms did need loaded database were <br /> BLASTX e-value, averaged best hit          instead installed GO.db. The software available <br /> each species, default score cutoff used. As     GPLv3 license http://github.com/wwood/ <br /> ESTs frequently represent small fragments genes,         goruby. <br /> caveat type analysis shorter <span id='am-5' about='xsp:length' typeof='owl:Thing'>length</span>      For C. elegans genes predicted ortho- <br /> makes assignment orthologue groups robust.            logues A. caninum H. contortus ESTs iden- <br />   An OrthoMCL group defined containing paral-          tified met assayability requirement, <br /> ogues parasitic nematodes ESTs       followed identification identified those <br /> had TBLASTX hit (e-value &lt; 1e-10) other,    essential C. elegans. This resulted 421 C. <br /> an imperfect BLASTN hit (identity &lt; 95%). This conser-        elegans genes final drug target list, 163 ortho- <br /> vative e-value used avoid missing potential paralo-    logues common A. caninum H. contortus, 124 <br /> gous relationships cost flagging potential false    present H. contortus 134 pres- <br /> positive paralogues pairs manually checked.     ent larger dataset A. caninum. All genes the <br /> Additionally, OrthoMCL groups containing          list enzymes ion channels GPCRs met all <br /> C. elegans proteins considered containing paral-      selection criteria. A quantitative weighting scheme <br /> ogues. The C. elegans gene(s) group         applied according present analysis rela- <br /> classed orthologues parasite EST drug       tive importance feature determining essential- <br /> target selection criteria applied C. elegans    ity - absence paralogues, presence orthologue (with <br /> genes. A caveat OrthoSelect/OrthoMCL cluster-         paralogues) D. melanogaster M. musculus and/or S. <br /> ing approach members recently diverged gene      cerevisiae, presence essential orthologue (with par- <br /> family, genes share common domain,        alogues) D. melanogaster, M. musculus and/or S. cere- <br /> may classified paralogues,         visiae. <br /> now distinct roles functionally redun-         Network connectivity data C. elegans genes were <br /> dant. This improved preliminary EST-           obtained Wormnet v1 (available http:// <br /> clustering step, included future analy-    www.functionalnet.org/wormnet/); [37] integrated <br /> ses EST datasets amenable clustering.         database mapping genes Wormnet to <br /> The A. caninum ESTs assigned 2313                 C. elegans OrthoMCL genes using gene identifiers <br /> OrthoMCL groups, resulting 2743 C. elegans genes           Wormbase gff3 file. The &quot;core&quot; higher confi- <br /> being predicted orthologues A. caninum ESTs.        dence set linkages LLS &#226;&#8240;&#165; <br /> The H. contortus ESTs assigned 2230               0.405465108108 (ln1.5). For C. elegans gene LLS <br /> OrthoMCL groups, resulting 2650 C. elegans genes           linkages core network added this <br /> being predicted orthologues H. contortus ESTs.      resulted 10,041 genes network score greater <br />   For assayability requirement enzymes, ion channels      zero. This approach identification drug <br /> and GPCRs identified using gene ontology (GO)            targets parasitic nematodes summarised Figure 4. <br /> terms. GO terms C. elegans genes obtained from <br /> the Amigo CVS database, using version 1.127                Additional material <br /> gene_association.wb.gz included database <br /> (retrieved Feb 2009 http://cvsweb.geneontol-              Additional file 1 Numbers genes orthology analysis.doc. This file <br />                                                                shows numbers genes meeting criteria orthology <br /> ogy.org/cgi-bin/cvsweb.cgi/). A custom software library,       comparisons essentiality. <br /> named GORuby, developed classify C. elegans <br />  Doyle et al. BMC Genomics 2010, 11:222                                                                                                                                                                                                                                                                                                                       Page 13 14 <br /> http://www.biomedcentral.com/1471-2164/11/222 <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                           5.    Winzeler EA, Shoemaker DD, Astromoff A, Liang H, Anderson K, Andre B, <br />       Additional file 2 Fisher&apos;s exact test results orthology analysis.xls. <br />                                                                                                                                                                                                                                                                                                 Bangham R, Benito R, Boeke JD, Bussey H, et al.: Functional <br />       This file contains results Fisher&apos;s exact test applied comparisons of <br />                                                                                                                                                                                                                                                                                                 characterization S. cerevisiae genome gene deletion and <br />       essentiality model organisms. <br />                                                                                                                                                                                                                                                                                                 parallel analysis. Science 1999, 285(5429):901-906. <br />       Additional file 3 Prioritized list drug targets A. caninum H.                                                                                                                                                                                                            6.    Kang Y, Durfee T, Glasner JD, Qiu Y, Frisch D, Winterberg KM, Blattner FR: <br />       contortus.xls. The genes list C. elegans predicted orthologues                                                                                                                                                                                                            Systematic mutagenesis Escherichia coli genome. J Bacteriol <br />       A. caninum H. contortus ESTs, identified present study.                                                                                                                                                                                                              2004, 186(15):4921-4930. <br />       Genes correspond validated anthelmintic drug targets shown                                                                                                                                                                                                           7.    Sassetti CM, Boyd DH, Rubin EJ: Comprehensive identification of <br />       bold. All genes list predicted enzymes (by GO term)                                                                                                                                                                                                              conditionally essential genes mycobacteria. Proc Natl Acad Sci USA <br />       lethal RNAi phenotype C. elegans. Targets prioritized                                                                                                                                                                                                              2001, 98(22):12712-12717. <br />       essentiality weighting assigned present study, network                                                                                                                                                                                                             8.    Sassetti CM, Boyd DH, Rubin EJ: Genes required mycobacterial <br />       connectivity score. &quot;Present mammals&quot; defined having ortho-                                                                                                                                                                                                                   growth defined high density mutagenesis. Mol Microbiol 2003, <br />       logue mammalian species OrthoMCL 2.0 data-                                                                                                                                                                                                                   48(1):77-84. <br />       set (human, rat, mouse). Essential orthologue yeast, drosophila mouse:                                                                                                                                                                                                        9.    Bethony J, Brooker S, Albonico M, Geiger SM, Loukas A, Diemert D, Hotez <br />       ? = orthologue (with paralogues) associated lethal phenotype, ? =                                                                                                                                                                                                               PJ: Soil-transmitted helminth infections: ascariasis, trichuriasis, and <br />       orthologue (with paralogues) non-lethal phenotypes                                                                                                                                                                                                                       hookworm. Lancet 2006, 367(9521):1521-1532. <br />       recorded, ? = orthologue (with paralogues) phenotype informa-                                                                                                                                                                                                             10.   Chan MS: The global burden intestinal nematode infections--fifty <br />       tion available. Presence paralogues C. elegans determined                                                                                                                                                                                                                   years on. Parasitol Today 1997, 13(11):438-443. <br />       OrthoMCL grouping. Presence paralogues A. caninum H.                                                                                                                                                                                                             11.   Wolstenholme AJ, Fairweather I, Prichard R, von Samson-Himmelstjerna G, <br />       contortus determined described Materials methods. The net-                                                                                                                                                                                                                  Sangster NC: Drug resistance veterinary helminths. Trends Parasitol <br />       work connectivity score sum interaction scores (log-likelihood                                                                                                                                                                                                              2004, 20(10):469-476. <br />       scores) gene Wormnet core network [37].                                                                                                                                                                                                                              12.   Pomroy WE: Anthelmintic resistance New Zealand: A perspective on <br />       Additional file 4 Phylogenetic trees nematode OrthoMCL clus-                                                                                                                                                                                                                       recent findings options future. New Zealand Veterinary <br />       ters. This file contains phylogenetic trees illustrative gene clusters                                                                                                                                                                                                            Journal 2006, 54(December):265-270. <br />       include ESTs parasitic nematodes.                                                                                                                                                                                                                                              13.   Besier B: New anthelmintics livestock: time right. Trends <br />       Additional file 5 Essential phenotypes C. elegans M. muscu-                                                                                                                                                                                                                       Parasitol 2007, 23(1):21-24. <br />       lus.doc. This file contains phenotype descriptions C. elegans                                                                                                                                                                                                       14.   Gilleard JS, Beech RN: Population genetics anthelmintic resistance in <br />       M. musculus genes classed essential genes current analy-                                                                                                                                                                                                              parasitic nematodes. Parasitology 2007, 134(Pt 8):1133-1147. <br />       sis.                                                                                                                                                                                                                                                                                15.   Nguyen M, Alfonso A, Johnson CD, Rand JB: Caenorhabditis elegans <br />                                                                                                                                                                                                                                                                                                 mutants resistant inhibitors acetylcholinesterase. Genetics 1995, <br /> Abbreviations                                                                                                                                                                                                                                                                                   140(2):527-535. <br /> GPCR: G-protein coupled receptor; EST: expressed sequence tag; GO: Gene                                                                                                                                                                                                                   16.   Lewis JA, Wu CH, Levine JH, Berg H: Levamisole-resistant mutants the <br /> Ontology; LLS: log-likelihood score.                                                                                                                                                                                                                                                            nematode Caenorhabditis elegans appear lack pharmacological <br />                                                                                                                                                                                                                                                                                                 acetylcholine receptors. Neuroscience 1980, 5(6):967-989. <br /> Authors&apos; contributions                                                                                                                                                                                                                                                                    17.   Campbell BE, Nagaraj SH, Hu M, Zhong W, Sternberg PW, Ong EK, Loukas <br /> Project conceived SAR RBG. MAD BJW performed bioinfor-                                                                                                                                                                                                                       A, Ranganathan S, Beveridge I, McInnes RL, et al.: Gender-enriched <br /> matic experiments, BJW assisted creation development data-                                                                                                                                                                                                                      transcripts Haemonchus contortus--predicted functions genetic <br /> base, MAD, BJW, RSH, SAR RBG analysed data. SAR, RBG MAD wrote                                                                                                                                                                                                                      interactions based comparative analyses Caenorhabditis <br /> manuscript, authors read approved final manuscript.                                                                                                                                                                                                                                 elegans. Int J Parasitol 2008, 38(1):65-83. <br />                                                                                                                                                                                                                                                                                           18.   Hoekstra R, Visser A, Otsen M, Tibben J, Lenstra JA, Roos MH: EST <br /> Acknowledgements                                                                                                                                                                                                                                                                                sequencing parasitic nematode Haemonchus contortus <br /> The authors wish thank Wormbase, Flybase MGI teams assis-                                                                                                                                                                                                                        suggests shift gene expression transition parasitic <br /> tance data access. SAR supported Future Fellowship (FT0990350)                                                                                                                                                                                                                     stages. Mol Biochem Parasitol 2000, 110(1):53-68. <br /> from Australian Research Council (ARC). Financial support present                                                                                                                                                                                                              19.   McCarter JP, Mitreva MD, Martin J, Dante M, Wylie T, Rao U, Pape D, <br /> study provided grants RBG ARC.                                                                                                                                                                                                                                               Bowers Y, Theising B, Murphy CV, et al.: Analysis functional <br />                                                                                                                                                                                                                                                                                                 classification transcripts nematode Meloidogyne <br />                                                                                                                                                                                                                                                                                                 incognita. Genome Biol 2003, 4(4):R26. <br /> Author Details <br /> 1Department                                                                                                                                                                                                                                                                               20.   Mitreva M, Jasmer DP, Appleton J, Martin J, Dante M, Wylie T, Clifton SW, <br />               Biochemistry &amp; Molecular Biology, Bio21 Molecular Science <br />                                                                                                                                                                                                                                                                                                 Waterston RH, McCarter JP: Gene discovery adenophorean <br /> and Biotechnology Institute, The University Melbourne, Victoria, 3010, <br />                                                                                                                                                                                                                                                                                                 nematode Trichinella spiralis: analysis transcription three <br /> Australia, 2Department Veterinary Science, The University Melbourne, 250 <br />                                                                                                                                                                                                                                                                                                 life cycle stages. Mol Biochem Parasitol 2004, 137(2):277-291. <br /> Princes Highway, Werribee, Victoria, 3030, Australia 3Current address: Peter <br />                                                                                                                                                                                                                                                                                           21.   Nisbet AJ, Gasser RB: Profiling gender-specific gene expression for <br /> MacCallum Cancer Centre, St Andrew&apos;s Place, East Melbourne Victoria, 3002, <br />                                                                                                                                                                                                                                                                                                 Trichostrongylus vitrinus (Nematoda: Strongylida) microarray <br /> Australia <br />                                                                                                                                                                                                                                                                                                 analysis expressed sequence tag libraries constructed by <br /> Received: 11 September 2009 Accepted: 3 April 2010                                                                                                                                                                                                                                              suppressive-subtractive hybridisation. Int J Parasitol 2004, <br /> Published: 3 April 2010                                                                                                                                                                                                                                                                         34(5):633-643. <br /> This <br /> &#194;&#169; <br /> BMC <br />   2010 <br />      an <br />      article available <br />                    Access <br />       Genomics <br />          Doyle <br />            Open  et2010,    articleBioMed <br />                                      distributed <br />                     al; licensee <br />                            11:222 <br />                             from:           Central Ltd.the terms Creative Commons Attribution License (http://creativecommons.org/licenses/by/2.0), permits unrestricted use, distribution, reproduction medium, provided original work properly cited. <br />                                    http://www.biomedcentral.com/1471-2164/11/222 <br />                                                  under <br />                                                                                                                                                                                                                                                                                           22.   Thompson FJ, Mitreva M, Barker GL, Martin J, Waterston RH, McCarter JP, <br />                                                                                                                                                                                                                                                                                                 Viney ME: An expressed sequence tag analysis life-cycle the <br /> References                                                                                                                                                                                                                                                                                      parasitic nematode Strongyloides ratti. Mol Biochem Parasitol 2005, <br /> 1. Trouiller P, Olliaro P, Torreele E, Orbinski J, Laing R, Ford N: Drug                                                                                                                                                                                                                        142(1):32-46. <br />     development neglected diseases: deficient market public-                                                                                                                                                                                                                  23.   Yin Y, Martin J, McCarter JP, Clifton SW, Wilson RK, Mitreva M: <br />     health policy failure. Lancet 2002, 359(9324):2188-2194.                                                                                                                                                                                                                                    Identification analysis genes expressed adult filarial <br /> 2. Nwaka S, Hudson A: Innovative lead discovery strategies tropical                                                                                                                                                                                                                         parasitic nematode Dirofilaria immitis. Int J Parasitol 2006, <br />     diseases. Nat Rev Drug Discov 2006, 5(11):941-955.                                                                                                                                                                                                                                          36(7):829-839. <br /> 3. Gerdes SY, Scholle MD, Campbell JW, Balazsi G, Ravasz E, Daugherty MD,                                                                                                                                                                                                                 24.   Ghedin E, Wang S, Spiro D, Caler E, Zhao Q, Crabtree J, Allen JE, Delcher AL, <br />     Somera AL, Kyrpides NC, Anderson I, Gelfand MS, et al.: Experimental                                                                                                                                                                                                                        Guiliano DB, Miranda-Saavedra D, et al.: Draft genome filarial <br />     determination level analysis essential genes                                                                                                                                                                                                                               nematode parasite Brugia malayi. Science 2007, 317(5845):1756-1760. <br />     Escherichia coli MG1655. J Bacteriol 2003, 185(19):5673-5684.                                                                                                                                                                                                                         25.   Mitreva M, Zarlenga DS, McCarter JP, Jasmer DP: Parasitic nematodes - <br /> 4. Giaever G, Chu AM, Ni L, Connelly C, Riles L, Veronneau S, Dow S, Lucau-                                                                                                                                                                                                                     genomes control. Vet Parasitol 2007, 148(1):31-42. <br />     Danila A, Anderson K, Andre B, et al.: Functional profiling                                                                                                                                                                                                                    26.   Gilleard JS, Woods DJ, Dow JAT: Model-organism genomics veterinary <br />     Saccharomyces cerevisiae genome. Nature 2002, 418(6896):387-391.                                                                                                                                                                                                                            parasite drug-discovery. Trends Parasitol 2005, 21(7):302-305. <br />  Doyle et al. BMC Genomics 2010, 11:222                                                                                                                 Page 14 14 <br /> http://www.biomedcentral.com/1471-2164/11/222 <br />  <br />  <br />  <br /> 27. Geldhof P, Murray L, Couthier A, Gilleard JS, McLauchlan G, Knox DP,                 amino acid 200 beta-tubulin isotype 1. Mol Biochem Parasitol 1994, <br />     Britton C: Testing efficacy <span id='am-475' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-476' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-477' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-478' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-479' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-480' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-481' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-482' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-483' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-484' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-485' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-486' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-487' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-488' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-489' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-490' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-491' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-492' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-493' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-494' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-495' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-496' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-497' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-498' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-499' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-500' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-501' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-502' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-503' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-504' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-505' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-506' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-507' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-508' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-509' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-510' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-511' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-512' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-513' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-514' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-515' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-516' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-517' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-518' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-519' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-520' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-521' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-522' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-523' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-524' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-525' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-526' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-527' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-528' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-529' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-530' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-531' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-532' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-533' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-534' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-535' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-536' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-537' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-538' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-539' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-540' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-541' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-542' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-543' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-544' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-545' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-546' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-547' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-548' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-549' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-550' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-551' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-552' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span>RNA interference</span> Haemonchus                    63(2):299-303. <br />     contortus. Int J Parasitol 2006, 36(7):801-810.                                52.   Cully DF, Vassilatis DK, Liu KK, Paress PS, Ploeg LH Van der, Schaeffer JM, <br /> 28. Geldhof P, Visser A, Clark D, Saunders G, Britton C, Gilleard J, Berriman M,         Arena JP: Cloning avermectin-sensitive glutamate-gated chloride <br />     Knox D: RNA interference parasitic helminths: current situation,                  channel Caenorhabditis elegans. Nature 1994, 371(6499):707-711. <br />     potential pitfalls future prospects. Parasitology 2007, 134(Pt             53.   Prichard RK: Mode action anthelminthic thiabendazole in <br />     5):609-619.                                                                          Haemonchus contortus. Nature 1970, 228(5272):684-685. <br /> 29. Knox DP, Geldhof P, Visser A, Britton C: RNA interference parasitic         54.   Dent JA, Smith MM, Vassilatis DK, Avery L: The genetics ivermectin <br />     nematodes animals: reality check? Trends Parasitol 2007,                        resistance Caenorhabditis elegans. Proc Natl Acad Sci USA 2000, <br />     23(3):105-107.                                                                       97(6):2674-2679. <br /> 30. Chen F, Mackey AJ, Stoeckert CJ Jr, Roos DS: OrthoMCL-DB: querying           55.   Martin RJ: Modes action anthelmintic drugs. Vet J 1997, <br />     comprehensive multi-species collection ortholog groups. Nucleic                   154(1):11-34. <br />     Acids Res 2006:D363-368.                                                       56.   Thanassi JA, Hartman-Neumann SL, Dougherty TJ, Dougherty BA, Pucci <br /> 31. Kamath RS, Ahringer J: Genome-wide RNAi screening Caenorhabditis                  MJ: Identification 113 conserved essential genes using high- <br />     elegans. Methods 2003, 30(4):313-321.                                                throughput gene disruption Streptococcus pneumoniae. <br /> 32. Kamath RS, Fraser AG, Dong Y, Poulin G, Durbin R, Gotta M, Kanapin A, Le             Nucleic Acids Res 2002, 30(14):3152-3162. <br />     Bot N, Moreno S, Sohrmann M, et al.: Systematic functional analysis         57.   Roemer T, Jiang B, Davison J, Ketela T, Veillette K, Breton A, Tandia F, <br />     Caenorhabditis elegans genome using RNAi. Nature 2003,                           Linteau A, Sillaots S, Marta C, et al.: Large-scale essential gene <br />     421(6920):231-237.                                                                   identification Candida albicans applications antifungal drug <br /> 33. Mummery-Widmer JL, Yamazaki M, Stoeger T, Novatchkova M, Bhalerao S,                 discovery. Mol Microbiol 2003, 50(1):167-181. <br />     Chen D, Dietzl G, Dickson BJ, Knoblich JA: Genome-wide analysis             58.   Hu W, Sillaots S, Lemieux S, Davison J, Kauffman S, Breton A, Linteau A, Xin <br />     Notch signalling Drosophila transgenic RNAi. Nature 2009,                      C, Bowman J, Becker J, et al.: Essential gene identification drug <br />     458(7241):987-992.                                                                   target prioritization Aspergillus fumigatus. PLoS Pathog 2007, <br /> 34. Blaxter M: Caenorhabditis elegans nematode. Science 1998,                       3(3):e24. <br />     282(5396):2041-2046.                                                           59.   Aguero F, Al-Lazikani B, Aslett M, Berriman M, Buckner FS, Campbell RK, <br /> 35. Gilleard JS: The use Caenorhabditis elegans parasitic nematode                 Carmona S, Carruthers IM, Chan AW, Chen F, et al.: Genomic-scale <br />     research. Parasitology 2004, 128(Suppl 1):49-70.                                     prioritization drug targets: TDR Targets database. Nat Rev Drug <br /> 36. Zhong W, Sternberg PW: Genome-wide prediction C. elegans genetic                  Discov 2008, 7(11):900-907. <br />     interactions. Science 2006, 311(5766):1481-1484.                               60.   Joyce AR, Palsson BO: Predicting gene essentiality using genome-scale <br /> 37. Lee I, Lehner B, Crombie C, Wong W, Fraser AG, Marcotte EM: A single                 silico models. Methods Mol Biol 2008, 416:433-457. <br />     gene network accurately predicts phenotypic effects gene                    61.   Lamichhane G, Zignol M, Blades NJ, Geiman DE, Dougherty A, Grosset J, <br />     perturbation Caenorhabditis elegans. Nat Genet 2008, 40(2):181-188.               Broman KW, Bishai WR: A postgenomic method predicting essential <br /> 38. Zhang K, Rathod PK: Divergent regulation dihydrofolate reductase                  genes subsaturation levels mutagenesis: application to <br />     malaria parasite human host. Science 2002,                               Mycobacterium tuberculosis. Proc Natl Acad Sci USA 2003, <br />     296(5567):545-547.                                                                   100(12):7213-7218. <br /> 39. Bacchi CJ, Nathan HC, Hutner SH, McCann PP, Sjoerdsma A: Polyamine             62.   Seringhaus M, Paccanaro A, Borneman A, Snyder M, Gerstein M: <br />     metabolism: potential therapeutic target trypanosomes. Science                  Predicting essential genes fungal genomes. Genome Res 2006, <br />     1980, 210(4467):332-334.                                                             16(9):1126-1135. <br /> 40. Kraus JM, Verlinde CL, Karimi M, Lepesheva GI, Gelb MH, Buckner FS:            63.   Volker C, Brown JR: Bioinformatics discovery novel anti- <br />     Rational modification candidate cancer drug use                     microbial targets. Curr Drug Targets Infect Disord 2002, 2(4):279-290. <br />     Chagas disease. J Med Chem 2009, 52(6):1639-1647.                              64.   Butland G, Peregrin-Alvarez JM, Li J, Yang W, Yang X, Canadien V, <br /> 41. Silvian LF, Wang J, Steitz TA: Insights editing ile-tRNA                Starostine A, Richards D, Beattie B, Krogan N, et al.: Interaction network <br />     synthetase structure tRNAile mupirocin. Science 1999,                       containing conserved essential protein complexes Escherichia <br />     285(5430):1074-1077.                                                                 coli. Nature 2005, 433(7025):531-537. <br /> 42. Conant GC, Wagner A: Duplicate genes robustness transient               65.   Krogan NJ, Cagney G, Yu H, Zhong G, Guo X, Ignatchenko A, Li J, Pu S, <br />     gene knock-downs Caenorhabditis elegans. Proc Biol Sci 2004,                      Datta N, Tikuisis AP, et al.: Global landscape protein complexes the <br />     271(1534):89-96.                                                                     yeast Saccharomyces cerevisiae. Nature 2006, 440(7084):637-643. <br /> 43. Gu X: Evolution duplicate genes versus genetic robustness           66.   Overington JP, Al-Lazikani B, Hopkins AL: How drug targets are <br />     null mutations. Trends Genet 2003, 19(7):354-356.                                    there? Nat Rev Drug Discov 2006, 5(12):993-996. <br /> 44. Liao B-Y, Zhang J: Null mutations human mouse orthologs                 67.   Sonnhammer EL, Koonin EV: Orthology, paralogy proposed <br />     frequently result different phenotypes. Proc Natl Acad Sci USA 2008,              classification paralog subtypes. Trends Genet 2002, 18(12):619-620. <br />     105(19):6987-6992.                                                             68.   Li L, Stoeckert CJ Jr, Roos DS: OrthoMCL: identification ortholog <br /> 45. Mushegian AR, Koonin EV: A minimal gene set cellular life derived             groups eukaryotic genomes. Genome Res 2003, 13(9):2178-2189. <br />     comparison complete bacterial genomes. Proc Natl Acad Sci USA               69.   Harris TW, Antoshechkin I, Bieri T, Blasiar D, Chan J, Chen WJ, De La Cruz N, <br />     1996, 93(19):10268-10273.                                                            Davis P, Duesbury M, Fang R, et al.: WormBase: comprehensive <br /> 46. Amsterdam A, Nissen RM, Sun Z, Swindell EC, Farrington S, Hopkins N:                 resource nematode research. Nucleic Acids Res 2010:D463-467. <br />     Identification 315 genes essential early zebrafish development.         70.   Bult CJ, Kadin JA, Richardson JE, Blake JA, Eppig JT: The Mouse Genome <br />     Proc Natl Acad Sci USA 2004, 101(35):12792-12797.         <br /> </body></html>