<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>APPLIED AND ENVIRONMENTAL MICROBIOLOGY, Jan. 2009, p. 483&#226;&#8364;&#8220;492                                                                        Vol. 75, No. 2 <br /> 0099-2240/09/$08.00&#226;&#171;&#185;0 doi:10.1128/AEM.01616-08 <br /> Copyright &#194;&#169; 2009, American Society Microbiology. All Rights Reserved. <br />  <br />  <br />  <br /> Overproduction Exopolysaccharides Escherichia coli K-12 rpoS <br />               Mutant Response Osmotic Stress&#228;&#338;&#164;&#226;&#8364;&#160; <br />                                             Michael Ionescu Shimshon Belkin* <br />           Department Plant Environmental Sciences, Institute Life Sciences, The Hebrew University Jerusalem, <br />                                                     Jerusalem 91904, Israel <br />                                                Received 15 July 2008/Accepted 30 October 2008 <br />  <br />              The yjbEFGH operon implicated production exopolysaccharide unknown function and <br />           induced osmotic stress negatively regulated general stress response sigma factor RpoS. Despite <br />           obvious importance RpoS, negative selection rpoS reported place starved cultures, <br />           suggesting adaptive occurrence allowing overexpression RpoD-dependent uptake <span id='am-3' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span>- <br />           scavenging systems. The trade-off RpoS-dependent functions improved <span id='am-4' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> utilization abilities <br />           makes bacterium sensitive environmental stressors, e.g., osmotic stress. In work, addressed <br />           hypothesis overinduction genes rpoS-deficient strains indicates essentiality. Using DNA <br />           microarrays, real-time PCR, transcriptional fusions, genes wca operon, implicated in <br />           production colanic acid exopolysaccharide, previously shown induced osmotic stress, also <br />           negatively controlled RpoS. Both exopolysaccharides synthesis yjb wca involved are <br />           overproduced rpoS mutant osmotic stress. We operons essential an <br />           rpoS-deficient strain wild type; promoters operons constitutively active yjb rpoS <br />           mutants; strain produces extremely mucoid colonies, forms long filaments, exhibits reduced growth <br />           capability. In addition, wca rpoS mutant&#226;&#8364;&#8482;s growth inhibited osmotic stress. These results indicate that <br />           induced wild type, operons valuable rpoS-deficient strain, suggesting <br />           overproduction exopolysaccharides adaptive action. <br />  <br />  <br />    The role general stress response sigma factor RpoS (&#226;?&#180;38)             overexpression RpoD-dependent <span id='am-7' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span>-sensing <br /> in Escherichia coli&#226;&#8364;&#8482;s physiology extensively studied.                 uptake systems. <br /> While studies focused characterization genes                  It previously shown (24, 44) E. coli genes <br /> positively controlled RpoS physiological implica-                shown significantly induced osmotic stress, gene, <br /> tions underexpression genes rpoS-deficient                  yjbF, stood promoter drives induction <br /> strains (22, 61), refer genes negatively regulated            appeared negatively regulated RpoS. This gene, mem- <br /> RpoS consequences overexpression.                          ber four-gene operon (yjbEFGH), listed (15) among <br />    Inactivation rpoS common; mutations rpoS gene                  genes positively dependent RcsC, inner mem- <br /> have detected laboratory bacterial stocks (25, 52),              brane sensor kinase Rcs phosphorelay system. The Rcs <br /> well environmental clinical isolates pathogenic               controls variety physiological functions prokaryotes, <br /> commensal enteric bacteria (1, 43), suggesting certain              extracellular polysaccharide (EPS) synthesis (27, 42, 51, <br /> conditions, loss attenuation RpoS activity                 58), biofilm formation (10, 15, 41), cell division (3), motility <br /> adaptive value. Other reports indicated rpoS gene tends               (17, 53). It shown (16) genes the <br /> <span id='am-9' about='protege:TO' typeof='owl:Thing'>to</span> undergo frequent mutations lead loss activity           yjbEFGH operon transcribed together, operon is <br /> the mutated forms appear spread dominant                      induced growth solid surfaces, involved in <br /> glucose-limited chemostat cultures (39) incubation                production unknown EPS. <br /> stationary phase (5, 66). It shown rate rpoS                 E. coli K-12 possesses known sets genes promoting <br /> mutations spreading population decreased                   EPS production: (i) genes involved O-antigen synthesis (46); <br /> osmotic stress applied (14, 28). The selection               (ii) wca operon, responsible colanic acid (CA)-EPS <br /> pressure rpoS gene thought largely                    synthesis (19); (iii) pgaABCD operon encodes genes <br /> competition sigma factors RpoS RpoD                    involved production poly-&#226;?&#164;-1,6-N-acetyl-D-glucos- <br /> limited number RNA polymerase core subunits (13, 32). This                  amine (PGA), polysaccharide shown crucial biofilm <br /> suggests rpoS inactivation beneficial starved cell            formation (59); (iv) dfc pseudo-operon (comprised of <br />                                                                                gfcABCDE, etp, etk), responsible production type <br />                                                                                IV capsule E. coli O127:H6 (this operon nonfunctional in <br />                                                                                E. coli K-12 presence IS1 element its <br />    * Corresponding author. Mailing address: Department Plant            promoter region [40]); (v) yjbEFGH operon, paral- <br /> Environmental Sciences, Institute Life Sciences, The Hebrew Uni-            ogue dfcABCD (15). <br /> versity Jerusalem, Jerusalem 91904, Israel. Phone: 972-2-6584192.              We propose overinduction yjb operon and, possibly, <br /> Fax: 972-2-6585559. E-mail: shimshon@vms.huji.ac.il. <br />    &#226;&#8364;&#160; Supplemental material article http://aem <br />                                                                                genes involved EPS production adap- <br /> .asm.org/.                                                                     tive response osmotic stress rpoS-deficient background <br />    &#228;&#338;&#164; <br />      Published ahead print 7 November 2008.                              overinduced genes special value an <br />  <br />                                                                          483 <br />  484       IONESCU AND BELKIN                                                                                                          APPL. ENVIRON. MICROBIOL. <br />  <br />           TABLE 1. Strains plasmids used study                           Bio-X-Act DNA polymerase (Bioline) presence 200 nM template and <br />                                                                                      primers using manufacturer&#226;&#8364;&#8482;s reagents instructions. The &#226;&#172;&#402;1,500-bp prod- <br />                                                                       Source      uct gel purified using QIAquick gel extraction kit (Qiagen), digested with <br />  Strain plasmid                     Genotype <br />                                                                      reference(s)    DpnI eliminate template plasmid, desalted using kit. The <br /> Strains                                                                              PCR product transformed DY378 (64), recombination-permis- <br />   MG1655                  F&#226;&#171;&#186; &#226;?&#173;&#226;&#171;&#186;                                     Laboratory       sive strain harboring lysogenically defective lambda phage. An overnight cul- <br />                                                                       stock          ture grown 30&#194;&#176;C regrown OD600 0.6, heat shocked 15 min to <br />   QC2410                  MG1655 rpoS::Tn10                         31               induce lambda PL promoter (controlling lambda recombination promoting <br />   MK1399                  MG1655 &#226;&#338;&#172;yjbEFGH(kanR)                     This work        factors), cooled ice, washed times cold double-distilled water (DDW), <br />   MK1310                  MG1655 &#226;&#338;&#172;yjbE(kanR)                        This work        finally resuspended 1 ml DDW. Aliquots (100 &#226;?&#174;l) mixed ice with <br />   MK1351                  MG1655 &#226;&#338;&#172;Pwza(kanR) (allele                This work        50 &#226;?&#174;l purified PCR product. Electrical DNA transformation carried <br />                             introduced MS1651                                   2.5 mV (Electro Cell manipulator ECM 935; BTX, United States), the <br />                             &#229;&#8230;&#179;40&#229;&#8230;&#180;)                                                    cells grown 1 h 37&#194;&#176;C prewarmed LB broth. Positive recombi- <br />   MK2499                  QC2410 &#226;&#338;&#172;yjbEFGH(kanR)                     This work        nants selected LB agar plates supplemented kanamycin verified <br />   MK2431                  QC2410 &#226;&#338;&#172;yjbE(yjbE::kanR)                  This study       PCR sequencing. <br />   MK2411                  QC2410 &#226;&#338;&#172;Pwza(kanR)                        This work           Allele transduction P1 kc coliphage. The new inactive alleles trans- <br />   CF6343                  MG1655 &#226;&#338;&#172;lacIZ(MluI)                       8                duced MG1655 P1 kc transduction described previously (48). The <br />   SK1150                  CF6343 rcsA::kanR                         17               rpoS::Tn10 allele QC2410 transduced MK1399, MK1310, and <br />   SK1158                  CF6343 rcsB::Tn10                         8                MK1351 produce double mutants MK2499, MK2431, MK2411. <br />   SK1154                  CF6343 rcsC::Tn10                         8                Briefly, overnight donor bacterial strain culture (carrying defective allele) <br />   MS1651                  EPEC O126:H6 &#226;&#338;&#172;Pwza(kanR)                  40               regrown 10 ml LB supplemented 5 mM CaCl2 40 min the <br />   DY378                   W3110 &#226;?&#173;cI857&#226;&#338;&#172;(cro-bioA)                   64               addition 100 &#226;?&#174;l P1 kc coliphage (produced MG1655) incubated <br />                             (permissive strain gene                              37&#194;&#176;C shaking (200 rpm) 3 h culture completely lysed by <br />                             inactivation)                                            phage. Cells survived lysis killed chloroform, cell debris <br />                                                                                      removed centrifugation. The supernatant containing new phages was <br /> Plasmids                                                                             collected, various volumes incubated 37&#194;&#176;C 40 min an <br />   pDEW201                 Promoterless; luxCDABE ori                56, 57           overnight MG1655 culture suspended 10 mM MgSO4, 5 mM CaCl2. P1 <br />                              pMB1(pBR322) rop ampR                                   infection stopped adding 100 &#226;?&#174;l 0.1 M Na3 citrate sample, and <br />   pDEW/yjbF               pDEW201; yjbF&#226;&#172;&#732;::luxCDABE                  56, 57           positive transductants selected LB agar plates supplemented the <br />     (pDEW609)                (pgi-yjbF)                                              appropriate antibiotics. <br />   pDEW/wza                pDEW201; wza&#226;&#172;&#732;::luxCDABE                   This study          Cloning wza promoter. MG1655 genomic DNA (extracted a <br />   pDEW/osmY               pDEW201; osmY&#226;&#172;&#732;::luxCDABE                  56, 57           DNeasy plant mini kit [Qiagen]) used template amplifying wza <br />   pDEW/lon                pDEW201; lon&#226;&#172;&#732;::luxCDABE                   56, 57           promoter primers designed calculated annealing temperature of <br />   pKD13                   Template plasmid kanR                 11               72&#194;&#176;C target site tail containing restriction enzyme digestion site <br />                              amplification                                           followed 8 mismatched nucleotides far 5&#226;&#172;&#732; end. pDEW201 (56, 57) was <br />   pES2                    recA&#226;&#172;&#732;::gfpUV ori pUC ampR                 45               used acceptor new DNA fragment. DNAs (plasmid PCR <br />   pATC400                 pBR322 rcsA&#226;&#171;&#185; ampR                         54               product) digested EcoRI KpnI desalted using QIAquick <br />                                                                                      gel extraction kit (Qiagen). Ligation carried T4 ligase (Roche) <br />                                                                                      according manufacturer&#226;&#8364;&#8482;s instructions treating linear vector with <br />                                                                                      shrimp alkaline phosphatase (Fermentas). Chemical transformation into <br />                                                                                      AG1688 competent cells carried out. Positive plasmids selected by <br /> rpoS-deficient strain. Our results indicate alternative stress <br />                                                                                      colony PCR verified sequencing. <br /> response strategies come play absence RpoS                        DNA microarray rRT-PCR analysis. Single colonies strains MG1655 <br /> when activity diminished, allowing cells survive                   QC2410 (rpoS::Tn10) grown (37&#194;&#176;C) overnight LB broth regrown in <br /> proliferate general stress protection.                              fresh LB broth containing 0.09 0.7 M NaCl 90 180 min, respectively, <br />                                                                                      wild-type strain reached OD600 0.3. The cultures diluted <br />                                                                                      density obtained rpoS mutant (OD600 &#226;&#171;&#189; 0.15), 50 ml each <br />                         MATERIALS AND METHODS                                        strain subjected total RNA extraction using RNeasy midi kit (Qiagen) <br />    Strains plasmids. The strains used study listed Table 1. All   DNase I (Qiagen) treatment. The RNA concentration determined by <br /> strains grown Luria-Bertani medium (LB broth; 5 g liter&#226;&#171;&#186;1 yeast extract,     using ND-1000 spectrophotometer (NanoDrop Technologies, Inc., Wilming- <br /> 10 g liter&#226;&#171;&#186;1 Bacto-tryptone, 5 g liter&#226;&#171;&#186;1 NaCl) supplemented appro-      ton, DE). Twenty micrograms total RNA reverse transcribed a <br /> priate antibiotics (100 &#226;?&#174;g ml&#226;&#171;&#186;1 ampicillin, 10 &#226;?&#174;g ml&#226;&#171;&#186;1 tetracycline, 50 &#226;?&#174;g ml&#226;&#171;&#186;1      poly(T18) primer using RevertAid first-strand cDNA synthesis kit (Fermentas), <br /> kanamycin) optical density 600 nm (OD600) 1 2 (Spekol 1200;           mRNA expression assayed using Affymetrix GeneChip E. coli <br /> Analytikjene, Germany), mixed 50% glycerol, kept &#226;&#171;&#186;80&#194;&#176;C. Cells       antisense genome array (Weizmann Institute Science [Rehovot, Israel] <br /> recovered freezing overnight growth LB agar plates. Plates kept      MicroArray <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>Unit</span>) according manufacturer&#226;&#8364;&#8482;s instructions. The DNA mi- <br /> for 1 month 4&#194;&#176;C.                                                            croarray results confirmed relative real-time PCR (rRT-PCR) analysis <br />    Detection rpoS wca mutants. RpoS deficiency routinely checked          (30). Twenty-five nanograms cDNA mixed Sybr green reaction <br /> qualitatively catalase activity assay. A drop 5 &#226;?&#174;l 32% H2O2 placed    mixture (ABI) assayed ABI Prism 7000 sequence detection system <br /> on colony; immediate vigorous bubbling indicated wild-type RpoS activity (39),     (Applied Biosystems) relative quantification abundance of <br /> and lack bubbling interpreted RpoS deficiency. Deficiency wca           following genes: yjbE, yjbF, yjbG, yjbH, wzc, gmd, manC, wcaH, wcaD, and <br /> (&#226;&#338;&#172;Pwza) tested transforming strain plasmid pATC400 (rcsA&#226;&#171;&#185;)           rcsA. Two additional genes similarly amplified RpoS-depen- <br /> (54). Wild-type strains extremely mucoid (CA-EPS overproduction),         dent osmY served negative control RpoS-independent house- <br /> &#226;&#338;&#172;Pwza strains did not.                                                               keeping gene rpmA served reference. The gene yielded average <br />    Construction new inactive alleles. Nonpolar gene deletions carried    threshold cycle (CT) values (n &#226;&#171;&#189; 3) 17.8 &#226;&#171;&#190; 0.7 (mean &#226;&#171;&#190; standard deviation) <br /> as described previously (11, 64), using primers listed Table 2. Briefly,      17.5 &#226;&#171;&#190; 0.6 MG1655 QC2410, respectively. The reaction conditions <br /> pKD13 (11) used template DNA PCR primer pairs 60 bases             follows: 50&#194;&#176;C 2 min, hot start 95&#194;&#176;C 10 min, 40 cycles 95&#194;&#176;C for <br /> long designed amplify kanamycin resistance gene plasmid (20          15 s, 60&#194;&#176;C 1 min. A dissociation test (12) performed end of <br /> bases 3&#226;&#172;&#732; end primer) undergo recombination process      run ensure single PCR product reaction mixture. Results were <br /> the edges chromosomal target site (40 bases 5&#226;&#172;&#732; primer     obtained SDS analysis program (ABI) using relative quantification <br /> identical recombination sites chromosome). The &#226;&#338;&#172;Pwza allele        algorithm automatic CT calculation. A &#226;&#338;&#172;CT calculated gene as <br /> directly amplified MS1651 genome (40), extracted           &#226;&#338;&#172;CT &#226;&#171;&#189; CT(studied gene) &#226;&#171;&#186; CT(rpmA). &#226;&#338;&#172;&#226;&#338;&#172;CT calculated different <br /> DNeasy plant mini kit (Qiagen) according manufacturer&#226;&#8364;&#8482;s instructions.         ways. (i) RpoS dependency calculated equation &#226;&#338;&#172;&#226;&#338;&#172;CT &#226;&#171;&#189; <br /> PCR (TGradient; Biometra, United States) carried proofreading           &#226;&#338;&#172;CT(rpoS::Tn10) &#226;&#171;&#186; &#226;&#338;&#172;CT(MG1655). (ii) The relative expression wca genes <br />  VOL. 75, 2009                                                                                  EPS OVERPRODUCTION IN AN rpoS MUTANT                                 485 <br />  <br />  <br />                                                                 TABLE 2. Primers used study <br />       Primer name, purpose                                              Sequence (5&#226;&#172;&#732; 3 3&#226;&#172;&#732;)a                                                       Use <br />  <br /> Gene inactivation <br />  Pwza-F                                     CGTTGGTGATCGCCTGTTGACCGG                                                            Transfer Pwza allele <span id='am-5' about='protege:FROM' typeof='owl:Thing'>from</span> <br />                                                                                                                                   MS1651 MG1655 <br />   Pwza-R                                    GCCGTTCGTTCAGCTCCATCGTGG                                                            Transfer Pwza allele from <br />                                                                                                                                   MS1651 MG1655 <br />   yjbE-F                                    ATTTTTGCCATATCTGCGCTTGCGGCGACTTCTGCGT                                               Inactivation yjbEFGH yjbE <br />                                              GGGCGTGTAGGCTGGAGCTGCTTC <br />   yjbE-R                                    GTCGTGGTTGTGGTGGTCCCGGTATTAGAACCA                                                   Inactivation yjbE <br />                                              TCACCGCATTCCGGGGATCCGTCGACC <br />   yjbH-R                                    CGCACGGCTGCGTGTCGGGCCAGACGAGAAGAGAT                                                 Inactivation yjbEFGH <br />                                              CCAACGGTACATTCCGGGGATCCGTCGACC <br />  <br /> Cloning (pDEW201 based) <br />   wza-F                                     TATACATGGAATTCCGATTAACCCGGCCCAGATAGACG                                              EcoRI-mediated ligation <br />   wza-R                                     ATCAGTGGGGTACCCCCCCGCCCTCGCCTTTCAGCG                                                KpnI-mediated ligation <br />  <br /> rRT-PCR analysis <br />   yjbE-F                                    TTTGCCATATCTGCGCTTG <br />   yjbE-R                                    TGGTGGTCCCGGTATTAGAA <br />   yjbF-F                                    AAGCGACCTGCACTCATTCT <br />   yjbF-R                                    AGGCCAGCACCACAAATAAC <br />   yjbG-F                                    TATTGTCGCGTTGCTTTTGA <br />   yjbG-R                                    CCAGCTCTTCGCTAATCACC <br />   yjbH-F                                    GCTTGCTACGGCGAAACATA <br />   yjbH-R                                    GGGAAGAGTTGCACTGAAGC <br />   wzc-F                                     CCTCGATATTGCAGTGAGCA <br />   wzc-R                                     CGCTGCTCAGGGTGTAGTTT <br />  <br />   wcaD-F                                    CCCATCACCATCGTCACTTT <br />   wcaD-R                                    CCACACCATGCCAATAATGA <br />   gmd-F                                     ATGGCGACCTGAGTGATACC <br />   gmd-R                                     TTTCTTTTTCCAGACCGAGGA <br />   wcaH-F                                    AGGAAGACTTTGCCACGGTA <br />   wcaH-R                                    CTTCCAGCGTTTCGTCTTTC <br />   manC-F                                    ATGAGCAGCACCGCTTTATT <br />   manC-R                                    CCGCCAATACCAGCATTAAC <br />   rcsA-F                                    CGACATTGAAACCGTTGATG <br />   rcsA-R                                    GCAATTGCCATAAAAACGAT <br />  <br />   osmY-F                                    TGCTGGCTGTAATGTTGACC <br />   osmY-R                                    TCGGTGCTCTTGATGTTGTC <br />   rpmA-F                                    TAACGGTCGCGATTCAGAAG <br />   rpmA-R                                    GCAAACAGAGTGTGGTCACG <br />   a <br />       Underlined letters indicate restriction enzyme digestion sites. <br />  <br />  <br />  <br />  <br /> compared yjb genes calculated equation &#226;&#338;&#172;&#226;&#338;&#172;CT &#226;&#171;&#189; [averaged                NaCl-exposed cells did change. Bioluminescence data normalized <br /> &#226;&#338;&#172;CT(wzc, gmd, manC, wcaH, wcaD)] &#226;&#171;&#186; [averaged &#226;&#338;&#172;CT(yjbE, yjbF, yjbG, yjbH)].             uniform cell density dividing measured light intensity (relative light <br /> RQ (relative RNA quantification) calculated RQ &#226;&#171;&#189; 2&#226;&#171;&#186;&#226;&#338;&#172;&#226;&#338;&#172;CT                 units) OD600 value measured time point. <br /> results experiment. Results presented averaged RQ values (n &#226;&#171;&#189; 3).       Maximum bioluminescence refers peak activity obtained course <br />    Bioluminescence monitoring promoter induction. Induction promoters            300 min measurements. <br /> was routinely monitored following luminescence E. coli strains bearing         EPS-related <span id='am-10' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-11' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> characterization. Single colonies spread LB <br /> pDEW201-derived plasmids (56, 57) (Table 1) harboring promoter::luxCDABE               agar plates supplemented 0.09 0.7 M NaCl 150 &#226;?&#174;g <br /> (Photorhabdus luminescens luxCDABE bioluminescence genes) transcriptional              ml&#226;&#171;&#186;1 Congo red (16) allowed grow 24 h 37&#194;&#176;C. Plates photo- <br /> fusions 96-well microtiter plates. All experiments carried LB broth     graphed black background (without Congo red) light screen (with <br /> supplemented 0.09, 0.3, 0.5, 0.7 M NaCl. To ensure similar plasmid copy        Congo red) Olympus Stylus-770 SW digital camera. A mucoid appear- <br /> numbers (24), lag phase eliminated initiating experiments        ance indicated overproduction CA-EPS wca operon; red-stained cells <br /> growing culture. A single colony grown OD600 0.5 LB broth              indicated presence EPS driven yjb operon. <br /> supplemented appropriate antibiotic 37&#194;&#176;C, diluted 1:10 fresh              Light/fluorescence microscopy. Five microliters culture mounted a <br /> medium, regrown OD600 0.2. Fifty microliters bacterial            microscopic slide Neubauer counting chamber (400-&#226;?&#174;m2 10-&#226;?&#174;m-deep <br /> culture added wells opaque, white, clear-bottom 96-well micro-        Clay Adams; Becton Dickinson, NJ), covered coverslip, analyzed with <br /> titer plate containing 50 &#226;?&#174;l fresh LB broth supplemented           epifluorescence microscope (Axivert 135TV; Zeiss, Germany) light mi- <br /> twice required NaCl concentration 0.09 M NaCl (basal growth medium              croscope (Eclipse N100; Nikon, Japan). Photographs acquired a <br /> concentration). The plate sealed transparent cover incubated       mounted Canon PowerShot A95 digital camera. <br /> at 37&#194;&#176;C 300 min microtiter plate reader (Victor2; Wallac, Finland).             Total carbohydrate determination. Overnight colonies grown LB agar <br /> Bioluminescence (reported instrument&#226;&#8364;&#8482;s arbitrary relative light units)      plates supplemented 0.7 0.09 M NaCl collected suspended in <br /> OD600 measured intervals 15 min, following 5-s shaking. For          2% Na2SO4, vigorously mixed 10 min, calibrated uniform cell <br /> several hours following exposure NaCl microtiter plate, OD600         concentration 1010 ml&#226;&#171;&#186;1. The following analytical protocol used (65): 500 <br />  486       IONESCU AND BELKIN                                                                                                      APPL. ENVIRON. MICROBIOL. <br />  <br /> &#226;?&#174;l cells extracted addition 800 &#226;?&#174;l chloroform, vigorous mixing,    TABLE 3. Expression ratios obtained DNA microarray analysis <br /> and phase separation centrifugation (14,000 rpm 10 min). A 300-&#226;?&#174;l                   rRT-PCR gene members various EPS <br /> amount upper (aqueous) phase mixed 200 &#226;?&#174;l DDW 1                                production systemsa <br /> ml anthrone reagent (Sigma) stock solution (1 mg ml&#226;&#171;&#186;1 concentrated H2SO4 <br /> [95 98%, wt/vol]). The mixture incubated 90&#194;&#176;C 10 min,                  Microarray analysis ratio    rRT-PCR <br /> OD630 sample determined. Total carbohydrate content calculated                Osmotic-                       RpoS <br />                                                                                     Gene                     RpoS                               <span id='am-6' about='rdfs:comment' typeof='owl:Thing'>Comment</span> <br /> from trehalose (Sigma) calibration curve.                                                    stress                    dependency <br />                                                                                                           dependencyc       ratioc <br />    Characterization growth capability. Single colonies grown LB                   responseb <br /> broth supplemented appropriate antibiotics OD600 0.5, diluted <br /> 1:10 fresh LB broth supplemented final concentration 0.7   yjbE       33.2           1.7         7.1 &#226;&#171;&#190; 0.4     yjb operon <br /> 0.09 M NaCl, incubated 37&#194;&#176;C shaking (200 rpm) 6 h. The OD600        yjbF        3.9           1.6         5.3 &#226;&#171;&#190; 0.4 <br /> was monitored 2-h intervals. To verify correlation OD measurements     yjbG        1.9           1.8         4.1 &#226;&#171;&#190; 0.4 <br /> the real cell concentration, determination growth performed        yjbH        2.2           1.7         4.1 &#226;&#171;&#190; 0.3 <br /> some time points direct microscopic counts (Eclipse N100; Nikon, Japan).         wza         5             2.7            ND         wca operon <br />                                                                                     wzb         3.8           1.7            ND <br />                                                                                     wzc         4.4           2             5 &#226;&#171;&#190; 0.3 <br />                                                                                     wcaA        3.1           2.1            ND <br />                                  RESULTS                                            wcaB        9             2.7            ND <br />                                                                                     wcaC        8.8           1.9            ND <br />    Genes wca operon induced osmotic stress                        wcaD       27             2.6         4.4 &#226;&#171;&#190; 0.7 <br /> are repressed RpoS. As previously reported (24, 44),                         wcaG        4             1.8            ND <br /> promoter regulating induction yjbEFGH operon                          gmd         5.1           1.6         7.9 &#226;&#171;&#190; 0.3 <br /> activated osmotic stress repressed RpoS. In                         wcaH        5.7           2.2         8.4 &#226;&#171;&#190; 0.4 <br />                                                                                     wcaI        2.6           1.7            ND <br /> attempt identify additional EPS synthesis-related genes that <br />                                                                                     manC       18             1.4         6.1 &#226;&#171;&#190; 0.5 <br /> may coregulated yjbEFGH, screened genes                         wzxC        4.3           1.8            ND <br /> that share regulatory features: induction NaCl                         wcaK        4             1.5            ND <br /> and overinduction rpoS::Tn10 mutant.                                          wcaM        3.2           1.6            ND <br />    For purpose, MG1655 wild-type strain                            rcsA        5.9           1.9         4.1 &#226;&#171;&#190; 0.7     Transcription activator <br />                                                                                                                                           wca/yjb operons <br /> rpoS::Tn10 derivative grown LB supplemented                       osmY         6.3          0.3         0.3 &#226;&#171;&#190; 0.4     RpoS dependent <br /> either 0.09 0.7 M NaCl wild-type cultures reached an <br />                                                                                       a <br /> OD600 0.3. All samples diluted OD                          Expression ratios wild type rpoS mutant obtained fol- <br />                                                                                     lowing growth LB supplemented 0.7 0.09 M NaCl. <br /> subjected total RNA extraction, followed whole-cell DNA                         b <br />                                                                                         MG1655 (0.7 M NaCl)/MG1655 (0.09 M NaCl). <br />                                                                                       c <br /> microarray analysis. Sampling times chosen based                            rpoS::Tn10 (0.7 M NaCl)/MG1655 (0.7 M NaCl). ND, determined. <br /> activity yjbF&#226;&#172;&#732;::luxCDABE transcriptional fusion each <br /> strain NaCl dose; 180 min, order mag- <br /> nitude higher rpoS mutant wild type                    yhaL, yhaK (see Table S1 supplemental material). <br /> grown LB broth supplemented 0.7 M NaCl.                                     As previously reported (18), functional EPS produc- <br />    Three members yjb operon, yjbE, yjbF, yjbH, dis-                      tion operon, pgaABCD, induced osmotic stress. In <br /> played osmotic-stress response ratios 33 (result low                   experiment, members displayed osmotic-stress response <br /> background), 3.9, 2.2 (Table 3). In previous reports                 ratios 1.6 6.1 exhibited positive negative <br /> DNA microarray studies investigated response E.                    dependency RpoS. <br /> coli osmotic shock (e.g., reference 60), members yjb                         The apparently similar induction characteristics (osmotic <br /> operon listed osmotically regulated                         stress RpoS dependency) yjb wca confirmed by <br /> genes. This attributed lower NaCl doses (&#226;&#172;?0.5 M)                      relative real-time PCR analysis, applying sample <br /> applied studies, short duration expo-                 preparation procedure employed DNA microarray <br /> sure (&#226;&#172;?30 min). In present study, based observed                         analysis. The RpoS-dependent gene osmY served a <br /> induction characteristics yjbF&#226;&#172;&#732;::luxCDABE transcriptional                    negative control. Indeed, shown Table 3, genes <br /> fusion, experimental conditions harsher. In DNA                        tested (four members yjb operon, members the <br /> microarray, yjbE, yjbF, yjbH displayed RpoS dependency ra-                      wca operon, common regulator rcsA) displayed RpoS <br /> tios 1.7, 1.6, 1.7, respectively; RpoS dependency thresh-                dependency ratios ranging 4 8, confirming general <br /> old ratio set 1.5.                                                 trend obtained DNA microarray analysis. As expected, <br />    Among 774 genes demonstrated &#226;&#177;&#8211;2-fold overexpres-                            RpoS-dependent gene osmY induced NaCl was <br /> sion wild type rpoS mutant                            repressed rpoS mutant. <br /> presence 0.7 M NaCl, 187 displayed &#226;&#177;&#8211;1.5-fold-                           The yjb wca promoters display similar induction pat- <br /> increased level expression rpoS mutant comparison                      terns. To study induction characteristics EPS <br /> to level expression wild type (see Table S1                  biosynthesis operons, wca yjb, cloned pro- <br /> supplemental material). Also selected double criterion                      moter wza, gene wca operon (50), produce <br /> were 14 21 known genes wca operon, display-                       wza&#226;&#172;&#732;::luxCDABE transcriptional fusion. The activity this <br /> ing osmotic-shock response ratios 3.2 27 RpoS de-                         construct compared previously described <br /> pendency ratios 1.5 2.8 (Table 3). The wca operon                         (24) yjbF&#226;&#172;&#732;::luxCDABE presence different NaCl con- <br /> previously reported (49) induced osmotic stress,                   centrations, wild type rpoS mutant (Fig. <br /> negative dependence rpoS demonstrated. A                             1). As rRT-PCR analysis, RpoS-dependent pro- <br /> additional RcsC-dependent genes, defined previously (15),                        moter osmY (osmY&#226;&#172;&#732;::luxCDABE) (56, 57) served the <br /> behaved similarly, including osmB, rcsA, ugd, ykfE, ygaC, ymgD,                     control. As clearly evident panels (A D) <br />  VOL. 75, 2009                                                                     EPS OVERPRODUCTION IN AN rpoS MUTANT                       487 <br />  <br />  <br />  <br />  <br />   FIG. 1. Activities yjbF&#226;&#172;&#732;::luxCDABE (A, B), wza&#226;&#172;&#732;::luxCDABE (C, D), osmY&#226;&#172;&#732;::luxCDABE (E, F), lon&#226;&#172;&#732;::luxCDABE (G, H) transcriptional <br /> fusions response different NaCl concentrations wild-type strain (A, C, E, G) rpoS::Tn10 mutant (B, D, F, H). RLU, <br /> relative light units. <br />  <br />  <br /> of Fig. 1, yjb wca promoters exhibit similar patterns           moters similarly enhanced rpoS <span id='am-88' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-89' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span>mutation</span> and <br /> induction response different NaCl concentrations,              peaked NaCl concentration (0.7 M) (Fig. 2A). The <br /> both wild type (Fig. 1A C) rpoS::Tn10                difference observed induction promoters <br /> mutant (B D); furthermore, activation pro-                 intensity; bioluminescence exhibited wza&#226;&#172;&#732; <br />  488     IONESCU AND BELKIN                                                                                    APPL. ENVIRON. MICROBIOL. <br />  <br />  <br />                                                                         ducted temperatures, 27&#194;&#176;C 37&#194;&#176;C, previ- <br />                                                                         ously shown (49) wca operon active the <br />                                                                         lower temperature. As before, induction promoters <br />                                                                         exhibited similar patterns different intensities. Both <br />                                                                         completely inhibited rcsC rcsB mutants, <br />                                                                         displayed higher activity lower temperature <br />                                                                         wild type. In rcsA mutant, inhibition activity <br />                                                                         temperature dependent: significantly inhibited (74- <br />                                                                         fold wca 49-fold yjb) 37&#194;&#176;C moderately <br />                                                                         reduced (4.2-fold wca 2.8-fold yjb) 27&#194;&#176;C. <br />                                                                            Overproduction EPS rpoS mutants. CA-EPS produc- <br />                                                                         tion (mucoid appearance) unnoticeable wild-type colonies <br />                                                                         grown 37&#194;&#176;C LB agar plates supplemented 0.09 or <br />                                                                         0.7 M NaCl (Fig. 3A); however, 0.7 M NaCl, rpoS::Tn10 <br />                                                                         mutant produced moderate mucosity. Inactivation wca <br />                                                                         promoter rpoS mutation (&#226;&#338;&#172;Pwza rpoS::Tn10) appeared <br />                                                                         limit growth somewhat clearly abolished mucosity (Fig. <br />                                                                         3A). In parallel manner, true yjb-depen- <br />                                                                         dent EPS (Fig. 3B) characterized Congo red staining: was <br />                                                                         observed wild type apparent rpoS mutant. <br />                                                                         Inactivating yjb operon rpoS mutant (&#226;&#338;&#172;yjbEFGH <br />                                                                         rpoS::Tn10) suppressed overproduction EPS stained <br />                                                                         Congo red and, addition, gave rise extreme mucosity. <br />                                                                         The carbohydrate contents cultures (Table 4) support <br />                                                                         visual observation. The carbohydrate concentrations all <br />                                                                         strains higher strains grown LB agar plates <br />                                                                         supplemented 0.7 M NaCl. The increase carbohydrate in <br />                                                                         wild-type strain enhanced activity several <br />                                                                         EPS production systems known induced osmotic stress: <br />                                                                         PGA, yjb-dependent EPS, CA-EPS. In rpoS mutant, only <br />                                                                         expected overproduced; data in <br />                                                                         Table 4 roughly estimated mixture dominated <br />                                                                         CA-EPS. <br />   FIG. 2. (A) NaCl dose dependency yjbF&#226;&#172;&#732;::luxCDABE                  &#226;&#338;&#172;yjb rpoS::Tn10 double mutants filamentous. Upon de- <br /> wza&#226;&#172;&#732;::luxCDABE induction wild-type strain rpoS::Tn10     letion yjbEFGH, rpoS::Tn10 strain lost red <br /> mutant. (B) Induction yjbF&#226;&#172;&#732;::luxCDABE wza&#226;&#172;&#732;::luxCDABE tran-       staining described extremely mucoid <br /> scriptional fusions response 0.7 M NaCl rcsA, rcsB, rcsC   (Fig. 3A B). In addition, cells elongated filaments <br /> mutants ancestral wild type 27&#194;&#176;C 37&#194;&#176;C. Error bars <br /> show standard deviations. Max, maximum; RLU, relative light units. <br />                                                                         200 &#226;?&#174;m long (Fig. 4A). These phenotypes longer <br />                                                                         dependent osmotic stress observed during <br />                                                                         growth regular LB agar plates (0.09 M NaCl). To better <br />                                                                         visualize phenomenon, cells transformed a <br /> fusion approximately fivefold higher wild type          plasmid containing green fluorescent protein gene (pES2, <br /> and rpoS mutant. The true rRT-PCR                  recA&#226;&#172;&#732;::gfpUV) (45). Furthermore, &#226;&#338;&#172;yjbE rpoS::Tn10 double <br /> analysis; averaged RQ values (see Materials Methods)            mutant exhibit identical colony (not shown) and <br /> for wca gene expression compared expression yjb               cellular (Fig. 4B) appearance. Both phenotypes negated <br /> genes 4.7 &#226;&#171;&#190; 2.2 5.1 &#226;&#171;&#190; 1.1 wild type            strain transformed plasmid pDEW/yjbF, <br /> rpoS mutant, respectively. As expected, osmY promoter               harbors active yjbE gene, confirming nonpolar <br /> exhibited strong positive response osmotic stress,         deletion yjbE. The cellular aspect complemented <br /> activity strongly inhibited rpoS::Tn10 mutant (Fig.          phenotype shown Fig. 4B. <br /> 1E F). An RpoS-independent gene promoter (lon,                      Activity yjb wca promoters enhanced the <br /> RpoH-dependent gene) exhibited similar activity patterns            &#226;&#338;&#172;yjbEFGH rpoS::Tn10 mutant. To explain NaCl-indepen- <br /> intensities strains (Fig. 1G H).                            dent mucoid colony appearance exhibited &#226;&#338;&#172;yjbEFGH <br />    The yjb operon listed (14) genes controlled             rpoS::Tn10 mutant, activities promoters (yjb and <br /> the Rcs phosphorelay RcsA-dependent manner                 wca) assayed wild type mutants, <br /> (16). The authors pointed sequence highly          &#226;&#338;&#172;yjbEFGH, rpoS::Tn10, &#226;&#338;&#172;yjbEFGH rpoS::Tn10. The back- <br /> similar RcsAB box (62) located yjbE promoter           ground activity promoters rpoS mutant five- <br /> region. We confirmed findings assessing osmotic            sixfold higher wild-type strain, suggesting that <br /> (0.7 M NaCl) induction activity yjbF&#226;&#172;&#732;::luxCDABE              molecular mechanism promotes overinductions <br /> and wza&#226;&#172;&#732;::luxCDABE transcriptional fusions rcsA-, rcsB-,             osmotic-stress independent. As expected, back- <br /> and rcsC-inactive strains (Fig. 2B). The experiments con-          ground activity (in 0.09 M NaCl) promoters much <br />  VOL. 75, 2009                                                                EPS OVERPRODUCTION IN AN rpoS MUTANT                     489 <br />  <br />  <br />  <br />  <br />   FIG. 3. (A) Appearance culture 24 h growth LB agar plates supplemented 0.09 M 0.7 M NaCl. (B) Congo red staining <br /> of cultures grown LB agar plates supplemented 0.7 M NaCl. Representative colonies shown insets. <br />  <br />  <br />  <br /> higher &#226;&#338;&#172;yjbEFGH rpoS::Tn10 double mutant in <br /> their rpoS::Tn10 ancestor (Fig. 5). When induced 0.7 M <br /> NaCl, maximal activity similar (yjb) twice high <br /> as (wza) background activity. The observation ap- <br /> pears significant spite large standard deviation in <br /> the wza&#226;&#172;&#732;::lux background measurements. In case, the <br /> response double mutant does fully explain 10- <br /> fold-higher carbohydrate content (CA-EPS) strain <br /> grown 0.7 M NaCl (Table 4). <br />  <br />  <br />  <br /> TABLE 4. Carbohydrate content cells grown LB agar plates <br />      supplemented different concentrations NaCl <br />                                      Carbohydrate content &#226;&#171;&#190; SD <br />          Genotype                          (&#226;?&#174;g/109 cells) <br />  <br />                                 0.09 M NaCl            0.7 M NaCl <br />  <br /> Wild type                        3.4 &#226;&#171;&#190; 2.6            11.6 &#226;&#171;&#190; 6 <br /> &#226;&#338;&#172;Pwza                            2.7 &#226;&#171;&#190; 2.4             9.1 &#226;&#171;&#190; 4.5 <br /> &#226;&#338;&#172;yjbEFGH                         2.4 &#226;&#171;&#190; 1.4            16.1 &#226;&#171;&#190; 9.5 <br /> rpoS::Tn10                       0.9 &#226;&#171;&#190; 0.3             393 &#226;&#171;&#190; 124a,b <br /> &#226;&#338;&#172;Pwza rpoS::Tn10                 1.7 &#226;&#171;&#190; 1.1             136 &#226;&#171;&#190; 16b <br /> &#226;&#338;&#172;yjbEFGH rpoS::Tn10               79 &#226;&#171;&#190; 39a            806 &#226;&#171;&#190; 136a         FIG. 4. (A) Fluorescence microscopy rpoS::Tn10 &#226;&#338;&#172;yjbEFGH <br />                                                                       rpoS::Tn10 mutants harboring pES2 (recA&#226;&#172;&#732;::gfp). (B) Light microscopy <br />  a <br />      Mucoid.                                                          &#226;&#338;&#172;yjbE rpoS::Tn10 harboring pDEW201 (vector only) pDEW/yjbF <br />  b <br />      Congo red positive.                                              (containing yjbE upstream yjbF&#226;&#172;&#732;). <br />  490     IONESCU AND BELKIN                                                                                     APPL. ENVIRON. MICROBIOL. <br />  <br />  <br />  <br />  <br />   FIG. 5. Maximal activities yjbF&#226;&#172;&#732;::luxCDABE (A) and <br /> wza&#226;&#172;&#732;::luxCDABE (B) transcriptional fusions wild type and <br /> &#226;&#338;&#172;yjbEFGH, rpoS::Tn10, &#226;&#338;&#172;yjbEFGH rpoS::Tn10 mutants. Error bars <br /> show standard deviations. Max, maximum; RLU, relative light units. <br />  <br />                                                                           FIG. 6. Growth LB medium supplemented 0.09 M (A) or <br />    Inactivation yjb wca impaired growth             0.7 M (B) NaCl. The strains listed panel A. Results shown are <br />                                                                        averaged curves results repeat experiments. <br /> rpoS-deficient strain. The growth capabilities of strains <br /> were tested LB broth supplemented 0.09 M (Fig. 6A) or <br /> 0.7 M (Fig. 6B) NaCl. While wild-type strain &#226;&#338;&#172;Pwza <br /> and &#226;&#338;&#172;yjbEFGH mutants exhibited differences             detected laboratory stocks enteric bacteria and <br /> growth characteristics, rpoS mutant grew slowly        natural isolates. Since fully partially inactive rpoS alleles are <br /> saline medium. Its derived double mutants, &#226;&#338;&#172;Pwza rpoS::Tn10        common, hypothesized aspects of <br /> and &#226;&#338;&#172;yjbEFGH rpoS::Tn10, exhibited slower growth: &#226;&#338;&#172;Pwza           low-RpoS RpoS-free physiology evolved compen- <br /> affected growth presence 0.7 M NaCl,          sate loss physiological functions normally <br /> effect &#226;&#338;&#172;yjbEFGH observed low NaCl medium.           RpoS dependent. <br /> The effect osmotic stress &#226;&#338;&#172;Pwza rpoS::Tn10 growth           RpoS shown important player E. coli&#226;&#8364;&#8482;s <br /> shown Fig. 3B, mutant appears reduced           osmotic-stress response (22, 61). It activates transcription <br /> comparison strains plate.                          large number genes provide osmoprotection, as <br />    Direct microscopic counts indicated measured               cross-protection stress factors. <br /> OD600 values displayed Fig. 6 valid comparisons             The synthesis uptake compatible solutes, most <br /> between tested strains: OD600 1 represents 2.8 &#226;&#171;&#187; 108         important responses cells osmotic stress, partially <br /> (rpoS::Tn10), 2.6 &#226;&#171;&#187; 108 (&#226;&#338;&#172;Pwza rpoS::Tn10), 2.2 &#226;&#171;&#187; 108              mediated RpoS-dependent genes. These include otsA, otsB, <br /> (&#226;&#338;&#172;yjbEFGH rpoS::Tn10) cells ml&#226;&#171;&#186;1. The filamentous pheno-               treA (21), involved trehalose synthesis metabolism. <br /> type described (Fig. 4) started emerge             Nevertheless, growth rpoS mutants hyperosmotic <br /> &#226;&#338;&#172;yjbEFGH rpoS::Tn10 double-mutant culture entry              medium, slower, demonstrated attrib- <br /> to stationary phase unnoticeable             uted function organic compatible solutes trans- <br /> course experiment.                                              port systems ProP ProU, high-affinity K&#226;&#171;&#185; <br />                                                                        transport KdpFABC BetTIBA im- <br />                                                                        plicated choline metabolism (9, 63). All systems <br />                           DISCUSSION <br />                                                                        expressed RpoS-independent manner response <br />   Natural selection loss RpoS activity shown        osmotic stress. From data clear rpoS-deficient <br /> occur course <span id='am-8' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span>-limited growth            strains overproduce EPS response osmotic stress that <br />  VOL. 75, 2009                                                                  EPS OVERPRODUCTION IN AN rpoS MUTANT                                  491 <br />  <br />  <br /> the deletion yjb wca operons attenuates im-          example mechanism evolved allow cells have <br /> paired osmotolerance mutant.                                    rpoS deficient grow cope osmotic upshifts <br />    The CA-EPS reported endow cells                enjoying better nutrient-scavenging capability the <br /> some stress protection, including extreme osmotic              overexpression RpoD-dependent uptake nutrient utiliza- <br /> stress E. coli O157:H7 (6, 34, 35). It demonstrated        tion systems. <br /> (6) CA-EPS-deficient strain lost viability faster the <br />                                                                                                         REFERENCES <br /> wild-type strain course 2-day exposure 1.5 M or <br />                                                                         1. Bhagwat, A. A., J. Tan, M. Sharma, M. Kothary, S. Low, B. D. Tall, M. <br /> more NaCl. Here demonstrated CA-EPS overpro-                   Bhagwat. 2006. Functional heterogeneity RpoS stress tolerance entero- <br /> duced rpoS mutant response osmotic stress             hemorrhagic Escherichia coli strains. Appl. Environ. Microbiol. 72:4978&#226;&#8364;&#8220; <br /> its deficiency limited growth saline medium                4986. <br />                                                                         2. Bi, E. F., J. Lutkenhaus. 1991. FtsZ ring structure associated with <br /> rpoS-deficient strain. Many clinical isolates enteric bacteria          division Escherichia coli. Nature 354:161&#226;&#8364;&#8220;164. <br /> were mucoid CA-EPS production (20).                  3. Carballes, F., C. Bertrand, J. P. Bouche, K. Cam. 1999. Regulation of <br /> Furthermore, Escherichia coli O157:H7 isolates re-               Escherichia coli cell division genes ftsA ftsZ two-component <br />                                                                            rcsC-rcsB. Mol. Microbiol. 34:442&#226;&#8364;&#8220;450. <br /> ported mucoid growth medium containing               4. Charette, M., G. W. Henderson, A. Markovitz. 1981. ATP hydrolysis- <br /> high salt concentrations (26). In view increasing reports           dependent activity lon(capR) protein E. coli K12. Proc. Natl. Acad. <br />                                                                            Sci. USA 78:4728&#226;&#8364;&#8220;4732. <br /> of prevalence mutated rpoS genotypes similar               5. Chen, G., C. L. Patten, H. E. Schellhorn. 2004. Positive selection loss <br /> isolates, speculated phenomena, mu-               RpoS function Escherichia coli. Mutat. Res. 455:193&#226;&#8364;&#8220;203. <br /> cosity loss rpoS functionality, coupled,         6. Chen, J., S. M. Lee, Y. Mao. 2004. Protective effect exopolysaccharide <br />                                                                            colanic acid Escherichia coli osmotic oxidative stress. Int. J. Food <br /> the mucoid appearance consequence rpoS mutation.               Microbiol. 93:281&#226;&#8364;&#8220;286. <br />    Several studies reported varied osmotolerance clin-          7. Chung, C. H., A. L. Goldberg. 1981. The product lon (capR) gene <br /> ical E. coli isolates (29). It shown (9) E. coli          Escherichia coli ATP-dependent protease, protease La. Proc. Natl. <br />                                                                            Acad. Sci. USA 78:4931&#226;&#8364;&#8220;4935. <br /> CFT073 lacking rpoS, proP, proU simultaneously               8. Conter, A., R. Sturny, C. Gutierrez, K. Cam. 2002. The RcsCB His-Asp <br /> limited growth virulence high-osmolality human urine,             phosphorelay essential overcome chlorpromazine-induced stress <br />                                                                            Escherichia coli. J. Bacteriol. 184:2850&#226;&#8364;&#8220;2853. <br /> suggesting possesses additional osmoregulatory systems.         9. Culham, D. E., A. Lu, M. Jishage, K. A. Krogfelt, A. Ishihama, J. M. <br /> It interesting determine deletion yjb           Wood. 2001. The osmotic stress response virulence pyelonephritis <br /> wca affect CFT073&#226;&#8364;&#8482;s growth ability and/or virulence.              isolates Escherichia coli: contributions RpoS, ProP, ProU other <br />                                                                            systems. Microbiology 147:1657&#226;&#8364;&#8220;1670. <br />    Overproduction CA-EPS results mucoid colony            10. Danese, P. N., L. A. Pratt, R. Kolter. 2000. Exopolysaccharide produc- <br /> appearance reported emerge response                  tion required development Escherichia coli K-12 biofilm architec- <br /> inactivation lon (36), treatment resulted UV             ture. J. Bacteriol. 182:3593&#226;&#8364;&#8220;3596. <br />                                                                        11. Datsenko, K. A., B. L. Wanner. 2000. One-step inactivation chromo- <br /> sensitivity cell filamentation UV treatment. Lon           somal genes Escherichia coli K-12 using PCR products. Proc. Natl. Acad. <br /> ATP-dependent protease (4, 7) that, addition role             Sci. USA 97:6640&#226;&#8364;&#8220;6645. <br />                                                                        12. Dhar, A. K., M. M. Roux, K. R. Klimpel. 2001. Detection quanti- <br /> the degradation misfolded proteins following heat shock,                fication infectious hypodermal hematopoietic necrosis virus white <br /> has regulatory function (55). Both cell elongation mu-               spot virus shrimp using real-time quantitative PCR SYBR green <br /> coid colony phenotype linked lon-specific target pro-              chemistry. J. Clin. Microbiol. 39:2835&#226;&#8364;&#8220;2845. <br />                                                                        13. Farewell, A., K. Kvint, T. Nystrom. 1998. Negative regulation RpoS: <br /> teins RcsA (54) SulA (37). RcsA unstable auxiliary              case sigma factor competition. Mol. Microbiol. 29:1039&#226;&#8364;&#8220;1051. <br /> DNA binding protein Rcs (33, 54). SulA cell         14. Ferenci, T. 2008. The spread beneficial mutation experimental bac- <br /> division inhibitor induced SOS DNA repair                 terial population: influence environment genotype the <br />                                                                            fixation rpoS mutations. Heredity 100:446&#226;&#8364;&#8220;452. <br /> response (47). It binds FtsZ, key protein cell division        15. Ferrie`res, L., D. J. Clarke. 2003. The RcsC sensor kinase required for <br /> which responsible septum formation (2, 23), re-                 normal biofilm formation Escherichia coli K-12 controls expres- <br />                                                                            sion regulon response growth solid surface. Mol. Microbiol. <br /> presses cell division (38). Hence, likely yjb rpoS       50:1665&#226;&#8364;&#8220;1682. <br /> mutants, phenotypes mediated Lon protease.             16. Ferrie`res, L., S. N. Aslam, R. M. Cooper, D. J. Clarke. 2007. The <br /> We sequenced lon allele double mutant,            yjbEFGH locus Escherichia coli K-12 operon encoding proteins <br />                                                                            involved exopolysaccharide production. Microbiology 153:1070&#226;&#8364;&#8220;1080. <br /> as assayed lon&#226;&#172;&#732;::lux activity (not shown), did alter-     17. Francez-Charlot, A., B. Laugel, A. Van Gemert, N. Dubarry, F. Wiorowski, <br /> ation sequence reduced induction.                                    M. P. Castanie&#194;&#180;-Cornet, C. Gutierrez, K. Cam. 2003. RcsCDB His-Asp <br />    Our results indicate group genes            phosphorelay negatively regulates flhDC operon Escherichia <br />                                                                            coli. Mol. Microbiol. 49:823&#226;&#8364;&#8220;832. <br /> are greater importance stress resistance               18. Goller, C., X. Wang, Y. Itoh, T. Romeo. 2006. The cation-responsive <br /> growth rpoS mutant wild type. In present             protein NhaR Escherichia coli activates pgaABCD transcription, required <br />                                                                            production biofilm adhesion poly-&#226;?&#164;-1,6-N-acetyl-D-glucosamine. J. <br /> communication, demonstrated EPS produc-                 Bacteriol. 188:8022&#226;&#8364;&#8220;8032. <br /> tion operons, wca yjb. These operons similarly             19. Gottesman, S., P. Trisler, A. Torres-Cabassa. 1985. Regulation cap- <br /> regulated, expression significantly enhanced           sular polysaccharide synthesis Escherichia coli K-12: characterization of <br />                                                                            regulatory genes. J. Bacteriol. 162:1111&#226;&#8364;&#8220;1119. <br /> rpoS mutant. While stress resistance capabilities endowed          20. Grant, W. D., I. W. Sutherland, J. F. Wilkinson. 1969. Exopolysaccha- <br /> by EPS overproduction unclear (6), results clearly            ride colanic acid occurrence Enterobacteriaceae. J. Bacteriol. <br /> that growth mutants impaired EPS production                   100:1187&#226;&#8364;&#8220;1193. <br />                                                                        21. Hengge-Aronis, R., W. Klein, R. Lange, M. Rimmele, W. Boos. 1991. <br /> inhibited saline (both wca yjb) salt-free (yjb)             Trehalose synthesis genes controlled putative sigma factor en- <br /> medium.                                                                    coded rpoS involved stationary-phase thermotolerance in <br />                                                                            Escherichia coli. J. Bacteriol. 173:7918&#226;&#8364;&#8220;7924. <br />    It tempting hypothesize operons,        22. Hengge-Aronis, R., R. Lange, N. Henneberg, D. Fischer. 1993. Osmotic <br /> other stress-responsive RpoS-independent systems, evolved             regulation rpoS-dependent genes Escherichia coli. J. Bacteriol. 175:259&#226;&#8364;&#8220; <br /> in order cope prevalence low levels RpoS rpoS         265. <br />                                                                        23. Higashitani, A., N. Higashitani, K. Horiuchi. 1995. A cell division <br /> mutations necessitated need im-              inhibitor SulA Escherichia coli directly interacts FtsZ GTP <br /> proved nutrient scavenging (14). EPS overproduction              hydrolysis. Biochem. Biophys. Res. Commun. 209:198&#226;&#8364;&#8220;204. <br />  492       IONESCU AND BELKIN                                                                                                              APPL. ENVIRON. MICROBIOL. <br />  <br /> 24. Ionescu, M., A. Franchini, T. Egli, S. Belkin. 2008. Induction                 Ulber, O. Lev, T. Scheper, S. Belkin. 2003. Fluorescence biolumi- <br />     yjbEFGH operon regulated growth rate oxygen concentration.                      nescence reporter functions genetically modified bacterial sensor strains. <br />     Arch. Microbiol. 189:219&#226;&#8364;&#8220;226.                                                             Sens. Actuators 90:2&#226;&#8364;&#8220;8. <br /> 25. Jishage, M., A. Ishihama. 1997. Variation RNA polymerase sigma               46.   Samuel, S., P. Reeves. 2003. Biosynthesis O-antigens: genes and <br />     subunit composition different stocks Escherichia coli W3110. J.                 pathways involved nucleotide sugar precursor synthesis O-antigen <br />     Bacteriol. 179:959&#226;&#8364;&#8220;963.                                                                   assembly. Carbohydr. Res. 338:2503&#226;&#8364;&#8220;2519. <br /> 26. Junkins, A. D., M. P. Doyle. 1992. Demonstration exopolysaccharide           47.   Schoemaker, J. M., R. C. Gayada, A. Markovitz. 1984. Regulation cell <br />     production enterohemorrhagic Escherichia coli. Curr. Microbiol. 25:9&#226;&#8364;&#8220;17.               division Escherichia coli: SOS induction cellular location sulA <br /> 27. Kelm, O., C. Kiecker, K. Geider, F. Bernhard. 1997. Interaction                protein, key lon-associated filamentation death. J. Bacteriol. 158: <br />     regulator proteins RcsA RcsB promoter operon                      551&#226;&#8364;&#8220;561. <br />     amylovoran biosynthesis Erwinia amylovora. Mol. Gen. Genet. 256:72&#226;&#8364;&#8220;83.           48.   Silhavy, T. J., M. L. Berman, L. W. Enquist. 1984. Experiments with <br /> 28. King, T., S. Seeto, T. Ferenci. 2006. Genotyping-by-environment inter-                gene fusions. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, <br />     action influencing emergence rpoS mutations Escherichia coli pop-               NY. <br />     ulation. Genetics 172:2071&#226;&#8364;&#8220;2079.                                                    49.   Sledjeski, D., S. Gottesman. 1996. Osmotic shock induction capsular <br /> 29. Kunin, C. M., T. H. Hua, L. Van Arsdale White, M. Villarejo. 1992.                    synthesis Escherichia coli K <br /> </body></html>