<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br />               A Specific RNA Hairpin Loop Structure Binds the RNA <br />                Recognition Motifs of the Drosophila SR Protein B52 <br />  <br />              B52, also known as SRp55, is a member of the Drosophila melanogaster SR protein family, a group of nuclear <br />           proteins that are both essential splicing factors and specific splicing regulators. Like most SR proteins, B52 <br />           contains two RNA recognition motifs in the N terminus and a C-terminal domain rich in serine-arginine <br />           dipeptide repeats. Since B52 is an essential protein and is expected <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> play a role in splicing a subset of <br />           Drosophila pre-mRNAs, its function is likely to be mediated by specific interactions with RNA. To investigate <br />           the RNA-binding specificity of B52, we isolated B52-binding RNAs by selection and amplification <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> a pool <br />           of random RNA sequences by using full-<span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span> B52 protein as the target. These RNAs contained a conserved <br />           consensus motif that constitutes the core of a secondary structural element predicted by energy minimization. <br />           Deletion and substitution mutations defined the B52-binding site on these RNAs as a hairpin loop structure <br />           covering about 20 nucleotides, which was confirmed by structure-specific enzymatic probing. Finally, we <br />           demonstrated that both RNA recognition motifs of B52 are required for RNA binding, while the RS domain is <br />           not involved in this interaction. <br />  <br />  <br />    Most metazoan genes are transcribed to yield pre-mRNAs                    subset of pre-mRNA in vivo may be critically dependent on <br /> containing introns that are excised and exons that are joined to             one particular SR protein or combination of SR proteins. <br /> create a functional mRNA (for a review, see reference 39).                      The Drosophila SR protein B52 was cloned from an embry- <br /> This pre-mRNA splicing process is catalyzed by the spliceo-                  onic cDNA expression library by using a monoclonal antibody <br /> some, a multicomponent protein-RNA complex consisting of                     against a Drosophila protein associated with transcriptionally <br /> small nuclear ribonuclear protein particles (snRNPs) and non-                active loci on polytene chromosomes (7). It is also referred to <br /> snRNP proteins. The accurate selection and pairing of corre-                 as Drosophila SRp55, which was independently cloned from <br /> sponding 59 and 39 splice sites occur when these factors are                 Drosophila Schneider cells by using the monoclonal antibody <br /> assembled onto the pre-mRNA in an orderly manner (for                        mAb104 (36). The activity of both Drosophila B52 and mam- <br /> a review, see reference 32). A family of non-snRNP protein                   malian SRp55 in splicing was established by their ability to <br /> factors, termed SR proteins, have been identified as both es-                complement the splicing-deficient S100 extracts (31, 50). The <br /> sential splicing factors and modulators of alternative splicing              mammalian SRp55 was also shown to promote the use of a <br /> that function in early spliceosome assembly (for reviews, see                distal alternative 59 splice site in simian virus 40 large-T splic- <br /> references 13 and 30).                                                       ing (52). <br />    SR proteins are structurally and functionally related and evo-               In vivo, B52 is an abundant protein and plays a critical role <br /> lutionarily conserved. This family comprises at least six mem-               in Drosophila development. B52 deletion mutants are homozy- <br /> bers, designated SRp20, SRp30a, SRp30b, SRp40, SRp55, and                    gous lethal at the second-instar larval stage (35), and overex- <br /> SRp75, which are conserved from Drosophila melanogaster to                   pression of B52 protein causes severe developmental defects <br /> humans (50). Among them, the human SRp30a (SF2/ASF)                          (28), evincing that B52 is an essential protein which must be <br /> (16, 17, 25, 26) and SRp30b (SC35) (14, 15) have been the most               expressed at the appropriate level. However, little is known <br /> intensively studied. In addition, three other human SR pro-                  about the molecular events that underlie B52&#226;&#8364;&#8482;s activities. Lar- <br /> teins, SRp30c (38), p54 (8), and 9G8 (6), have been cloned re-               vae deficient in B52 can still splice the five endogenous pre- <br /> cently. For Drosophila, only two SR proteins have been cloned.               mRNAs (35) and one transgenic pre-mRNA (22) tested, indi- <br /> They are homologous to the human SRp55 and SRp20 and are                     cating that it is not required for general splicing in vivo. <br /> designated B52 (7) and RBP1 (24), respectively.                              Although mammalian SRp55 has been shown to selectively <br />    Different SR proteins appear to have overlapping but dis-                 promote U1 binding to the distal simian virus 40 large-T 59 <br /> tinct functions in constitutive and alternative splicing reactions           splicing site, the sequence determinant for this 59 splice site <br /> in vitro. Although individual SR proteins are able to comple-                selection has not been identified (52). <br /> ment splicing-deficient cytoplasmic S100 extracts (31, 50), their               The SR proteins share a similar protein structure, which has <br /> activities vary when they are presented with different pre-                  at least one RNA recognition motif (RRM) at its N terminus <br /> mRNA substrates (12). In alternative splicing, one SR protein                and a C-terminally located domain rich in arginine-serine <br /> may promote preferential usage of one splice site, whereas                   dipeptide repeats (RS domain). The RS domain has been <br /> another SR protein specifies another splice site (51). While                 postulated to mediate protein-protein interactions among <br /> members of the SR protein family are functionally redundant                  splicing factors and regulators (48). An RRM has two highly <br /> for many splicing events in vitro, the accurate splicing of a                conserved elements, referred to as RNP1 and RNP2, embed- <br />                                                                              ded in a less conserved sequence of approximately 80 residues. <br />   * Corresponding author. Mailing address: Section of Biochemistry,          It has been found in over 200 RNA-binding proteins involved <br /> Molecular and Cell Biology, Cornell University, 417 Biotechnology            in processing and transportation of RNA (for a review, see <br /> Building, Ithaca, NY 14853. Phone: (607) 255-2442. Fax: (607) 255-           reference 4). In many cases, proteins with RRMs have been <br /> 2428. E-mail: jtl10@cornell.edu.                                             shown to bind RNA with sequence specificity. Recent crystal- <br />  <br />                                                                       2649 <br />  2650       SHI ET AL.                                                                                                                            MOL. CELL. BIOL. <br />  <br />  <br /> lographic (33) and nuclear magnetic resonance (1) studies of                         twofold per cycle. The amplified DNA template was phenol extracted and eth- <br /> the U1A snRNP protein RRM with its cognate RNA have                                  anol precipitated. <br />                                                                                         (ii) Cycles of selection. RNA for each round was produced by using the <br /> provided insight into the RRM-RNA interaction. The RRM                               T7-MEGAshortscript in vitro transcription kit for large-scale synthesis of short- <br /> has a compact babbab structure which binds single-stranded                           transcript RNAs (Ambion Inc., Austin, Tex.) according to the manufacturer&#226;&#8364;&#8482;s <br /> RNA on the surface of a four-stranded b-sheet. Like many                             instructions. Gel-purified RNA was quantified by spectrophotometry and diluted <br /> other SR proteins, B52 has an extensive RS domain and two                            into 13 binding buffer (50 mM Tris-Cl [pH 7.6], 200 mM potassium acetate, 5 <br />                                                                                      mM MgCl2, 2.5 mM dithiothreitol). The pool was heated to 708C for 3 min and <br /> RRMs, one of which conforms to the RRM consensus and the                             then cooled to ambient temperature over 5 min before addition of the protein. <br /> other of which is degenerate (7).                                                    Prior to every other cycle, the folded RNA pool was also passed through an <br />    One possible mechanism by which SR proteins commit pre-                           HAWP filter (Millipore Inc., New Bedford, Mass.). The binding reaction was <br /> mRNA to splicing requires the initial binding of a specific SR                       performed in a 100-ml volume with 30 mM RNA and 30 nM B52 protein. Forty <br />                                                                                      units of rRNasin (Promega Corporation) was also included in first three cycles. <br /> protein to the RNA. Control of alternative splicing may also be                      The reaction mixture was incubated for 80 min with temperature cycling between <br /> achieved by cell- or tissue-specific factors interacting with spe-                   258C (or ambient temperature) and 378C, such that the reaction was at 258C for <br /> cific elements on pre-mRNA. Previously, SF2/ASF, or SRp30a,                          three quarters of the time (2). Protein-RNA complexes were isolated by filtration <br /> was shown to recognize purine-rich exonic splicing enhancer                          and extracted as previously described (46). The selected RNAs were amplified by <br />                                                                                      using the Superscript preamplification system for first-strand cDNA synthesis <br /> elements of the bovine growth hormone pre-mRNA (42). Re-                             and Taq DNA polymerase (Life Technologies Corporation, Gaithersburg, Md.) <br /> cently, in vitro selection of high-affinity binding sites for SF2/                   according to the manufacturer&#226;&#8364;&#8482;s instructions. <br /> ASF and SC35 (SRp30b) yielded distinct sets of consensus                                (iii) Cloning and sequencing. The pool DNAs were cloned in the vector <br /> RNA sequences (43). Similar experiments done with RBP1 led                           pGEM-3Z, between the EcoRI and HindIII sites of the polylinker region. <br />                                                                                      Individual clones were sequenced by the Taq cycle sequencing method with <br /> to the discovery of its target sequences within two regions of                       DyeDeoxy terminators on a Applied Biosystems 737A automated DNA se- <br /> the Drosophila Dsx pre-mRNA (21). Consistent with these                              quencer by the DNA Services facility at Cornell Biotechnology Programs. <br /> findings, a single <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span>missense mutation</span> in the first RRM of B52                            RNA-protein binding assay. All binding assays were performed in 20-ml vol- <br /> resulted in a dominant second-site enhancer of RNA-process-                          umes in 13 binding buffer (see above). A typical binding assay mixture with <br />                                                                                      labeled RNA contained about 20 fmol of 32P-labeled RNA probe and different <br /> ing defects (34). From these observations, it is reasonable to                       amounts (1 to 10 pmol) of B52 protein. The RNA probes were uniformly labeled <br /> assume that the specific recognition of RNA sequence or struc-                       with [a-32P]UTP (Amersham Life Science Inc.) by using the T7-MAXIscript in <br /> ture elements may be essential for the function of B52.                              vitro transcription kit (Ambion, Inc.) according to the manufacturer&#226;&#8364;&#8482;s instruc- <br />    In this study, we characterized the RNA-binding specificity                       tions. Prior to use in a binding assay, the majority of transcripts of each RNA <br />                                                                                      preparation were shown to be of the expected size by electrophoresis on an 8% <br /> of B52 by an in vitro approach. We first identified the specific                     polyacrylamide&#226;&#8364;&#8220;7 M urea gel. A typical binding assay mixture with the truncated <br /> RNA sequence recognized by B52 by using an in vitro selection                        versions of B52 contained 1 ml of the 35S-labeled translation reaction mixture <br /> technique, which yielded a relatively long consensus sequence.                       with 2 pmol of unlabeled RNA transcript (the final concentration of RNAs in this <br /> Unlike the RNAs that bind other SR proteins, which appear                            reaction was 100 nM, and the concentration of the labeled B52 proteins was <br />                                                                                      much lower). The reaction mixtures were allowed to equilibrate for 15 to 20 min <br /> to be short and unstructured (21, 43), the B52-binding site                          at ambient temperature before being subjected to filter binding, gel shift, or UV <br /> was found to have a hairpin loop structure as predicted by a                         cross-linking. Gel shifts were performed at 48C. The binding reaction mixtures <br /> computer program by using free energy minimization and as                            were set at 48C for 5 to 10 min before being loaded onto a 2.5% agarose gel in <br /> confirmed by structure-specific enzymatic probing. We also                           0.253 Tris-borate-EDTA buffer. The affinity of the RNA for the in vitro-trans- <br />                                                                                      lated polypeptides was estimated by comparing the intensities of the bands <br /> mapped the RNA-binding site on B52 and found that both                               representing the complex on the agarose gel and those of the corresponding <br /> RRMs are required for the interaction with the RNA aptamer.                          polypeptides run on SDS-PAGE. To cross-link proteins to RNA, the binding <br /> We discuss the effect of these selected B52-binding RNAs on                          reaction mixtures were irradiated for 20 min from 5 cm directly above by invert- <br /> splicing in the light of some ongoing experiments.                                   ing a short-wavelength UV transilluminator. The products of cross-linking were <br />                                                                                      analyzed by standard SDS-PAGE. <br />                                                                                         Footprinting. The procedures used for footprinting were modifications of <br />                         MATERIALS AND METHODS                                        those previously reported (9). The RNA substrate (100 ng) was incubated with or <br />    Protein expression and purification. Full-length B52 protein was expressed in     without B52 under binding conditions. RNase T2 (1 U) or RNase V1 (0.35 U) <br /> Sf9 cells by using the baculovirus expression system. The B52 cDNA used in this      was added and incubated at room temperature for 1 min. The digestion was <br /> study was described previously (28) and was cloned into the vector pJVP10Z           stopped by phenol-chloroform extraction followed by ethanol precipitation. The <br /> (47). The transfection, purification, and culturing of the recombinant baculovirus   precipitated RNAs were resuspended and analyzed by primer extension with <br />                                                                                      32 <br /> were performed as described previously (20, 41). The overexpressed B52 was              P-labeled pUC/M13 reverse sequencing primer and Superscript II reverse <br /> purified by the standard SR protein purification procedure (50).                     transcriptase (Life Technologies) and separated on a 6% (for RNase T2 diges- <br />    B52 deletion constructs were generated by PCR amplification of the corre-         tions) or 8% (for RNase V1 digestions) sequencing gel. <br /> sponding regions of the B52 cDNA. They were then cloned into the vector <br /> pGEM-3Z (Promega Corporation, Madison, Wis.). Truncated versions of B52 <br /> were made by in vitro translation with the TNT Coupled Reticulocyte Lysate                                            RESULTS <br /> System (Promega Corporation) and L-[35S]methionine (in vivo cell labeling <br /> grade; Amersham Life Science Inc., Cleveland, Ohio) according to the manu-              In vitro selection of B52-binding RNA. In order to study the <br /> facturer&#226;&#8364;&#8482;s instructions. The quality of translation products was checked by so-      RNA-binding capacity of B52, we used, with some modifica- <br /> dium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE) prior to <br /> use. <br />                                                                                      tion, an in vitro selection and amplification scheme (SELEX) <br />    Oligonucleotides. The template-primer system consisted of three oligonucle-       (46). We adapted the template-primer system used by Bartel et <br /> otides identical or similar to those used by Bartel et al. (3): Temp (108 bases),    al. (3) to produce RNA molecules having a stretch of 40 ran- <br /> 59-ACCGAGTCCAGAAGCTTGTAGTACTN40GCCTAGATGGCAGTTGA                                     dom bases sandwiched between 59 and 39 constant regions for <br /> ATTCTCCCTATAGTGAGTCGTATTAC-39 (N40 stands for 40 bases of ran- <br /> dom sequence); T7Univ (40 bases), 59-GTAATACGACTCACTATAGGGAG <br />                                                                                      primer annealing. The 59 constant sequence included a pro- <br /> AATTCAACTGCCATCTA-39; and RevUniv (22 bases), 59-ACCGAGTCCAG                         moter for T7 RNA polymerase. Five hundred sixty picomoles <br /> AAGCTTGTAGT-39. These oligonucleotides were synthesized at a quantity of             of synthetic oligodeoxynucleotide template (20 mg of a 108- <br /> 0.2 mM by Integrated DNA Technologies, Inc. (Coralville, Iowa).                      mer) was amplified by PCR to make the original pool. Assum- <br />    The templates of the deletion, <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span>mutation</span>, and antisense constructs of B52- <br /> binding sequences (BBSs) were made as oligonucleotides appended with the T7 <br />                                                                                      ing that 10% of the molecules were free of chemical lesions <br /> promoter sequence. They were synthesized as Gibco BRL Custom Primers                 that prevent second-strand synthesis and transcription (19), <br /> (Internet, PRIMERS@lifetech.com).                                                    this pool contained 3.4 3 1013 different sequences. <br />    In vitro selection. (i) Pool construction. The gel-purified synthetic oligode-       The target protein used in the selection was B52 overex- <br /> oxynucleotides Temp, T7Univ, and RevUniv were used as template and primers <br /> for a PCR (5-ml reaction mixture) carried out in 10 Eppendorf tubes. The <br />                                                                                      pressed from a baculovirus construct in insect cells. The B52 <br /> temperature cycling was performed by manual transfer among three water baths         produced by these insect cells appears to be full-length and <br /> and was stopped after four cycles when the product was still increasing nearly       properly modified, as it has the same electrophoretic mobility <br />  VOL. 17, 1997                                                                                              RNA-BINDING SPECIFICITY OF B52                        2651 <br />  <br />  <br />  <br />  <br />    FIG. 1. In vitro selection of B52-binding RNA. (A) Baculovirus-expressed B52 used as target protein in the selection. Lanes 1 and 2, protein markers (Broad Range; <br /> New England Biolabs) and the B52 preparation from baculovirus-infected cells, respectively. Lanes 3 and 4, immunoblot analysis with a B52-specific antibody and either <br /> B52 produced by baculovirus-infected cells (lane 3) or a Drosophila nuclear extract prepared as described previously (11) (lane 4). (B) RNA sequences selected by B52 <br /> from a randomized pool. The names of the individual clones are given to the left of each selected sequence. The variable region is in capital letters, and the shared <br /> sequence motif is in boldface. Nucleotides belonging to the flanking constant region are in lowercase. Nucleotides only shared by BBS 4,14,15 and BBS 8 are underlined. <br /> The full sequence of the flanking constant region is shown at the bottom. (C) Predicted secondary structures of the B52-selected RNA sequences. Nucleotides forming <br /> the shared sequence motif are in boldface. Residues derived from the constant flanking region are in lowercase. <br />  <br />  <br />  <br /> as the Drosophila protein. A Coomassie blue stain of the bac-                         occurred in three separate isolates (Fig. 1B). Since all six <br /> ulovirus B52 preparation is shown in Fig. 1A, lane 2. Immu-                           members of this family showed specific binding to B52, they <br /> noblot analysis (Fig. 1A, lane 3) of this same preparation with                       are designated B52 binding sequences (BBSs). These se- <br /> a B52-specific antibody (28) displays the identical mobility of                       quences have two absolutely &#226;&#8364;&#339;conserved&#226;&#8364;&#65533; hexamers separated <br /> B52 as when assayed in a Drosophila nuclear extract (lane 4).                         by a variable nucleotide. There is also a &#226;&#8364;&#339;conserved&#226;&#8364;&#65533; G in the <br /> In addition, baculovirus-produced B52 is fully active in an in                        flanking region of either side one <span id='am-238' about='xsp:base' typeof='owl:Thing'>base</span> from the end of the <br /> vitro splicing assay (22). Truncated versions of B52 were not                         double hexamer. This sequence motif is found in different <br /> used because we could not exclude a priori the involvement of                         sequence contexts and in different positions of the randomized <br /> the SR domain in possible sequence-specific binding.                                  region. In two of these sequences, BBS #8 and BBS 4,14,15, <br />    The pool of random RNA was carried through nine rounds                             sequence similarity extends further downstream, covering a <br /> of selection and amplification. RNA-protein complexes were                            region an additional 16 nucleotides (nt) in length. <br /> selected by binding to nitrocellulose filters. We observed a                             A second family of eight sequences was isolated, whose <br /> significant increase of the pool&#226;&#8364;&#8482;s affinity for B52 as the selection                  common feature is multiple G triplets. Members of this family <br /> progressed (data not shown). Fractions of two final pools were                        are thus called multi-G motifs (MGMs). Most sequences in <br /> cloned and sequenced. Some cloned sequences were found in                             this family have four G triplets within a 25-nt region. Unex- <br /> duplicate or triplicate, indicating that the complexity of these                      pectedly, they have no significant affinity to B52, as demon- <br /> selected pools was quite low. All sequences in the last pool                          strated below. Similar sequences were also encountered in <br /> belong to either of two families.                                                     another in vitro selection that was done simultaneously, but <br />    One family consists of four different sequences, one of which                      these sequences have not yet been fully characterized. At this <br />  2652       SHI ET AL.                                                                                                                             MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />    FIG. 2. Affinity and specificity of B52-binding RNA. (A) B52 binds tightly to selected sequences. Shown is a band shift assay on a 2.5% agarose gel of 20-ml binding <br /> reaction mixtures with increasing amounts of purified B52. MGM #1 is an RNA sequence containing the MGM (59-CGGAACTAGGGTCAAGGGATTAGGGTCA <br /> GGGGTTCCGGT-39). G0 is a sequence randomly picked from the original pool (59-GAGACCCACCGACACCTCGGCCGGCGGGGCTTTTAGCGAG-39). (B) <br /> Selected sequences compete for binding to B52. Reactions were performed as described for panel A except that an excess amount of cold competitor RNA was added <br /> prior to the radioactive probes. <br />  <br />  <br />  <br /> time, we speculate that the MGM sequences were enriched via                           quences randomly picked from the original unselected pool <br /> a structure formed by two or more RNAs or that they are the                           were used as controls. Increasing amounts of B52 were added <br /> result of contamination from this other selection done in par-                        to 32P-labeled RNA probes, which contain the 59 constant <br /> allel, since RNAs in both selections were purified on the same                        region, in the presence of a large excess of tRNA. Figure 2A <br /> gels. Because the individual MGMs tested do not bind B52                              shows a representative assay. Stable complexes were formed <br /> avidly (see Fig. 2A), this family will not be considered further                      with BBS sequences but not with the MGM sequence or the <br /> in this paper.                                                                        control sequence from the unselected pool. Since only one <br />    We used the computer program MulFold (23, 53) to exam-                             shifted complex is observed and since B52 is a monomer (27), <br /> ine the free-energy-minimized secondary structures of the                             we assume that the interaction between B52 and RNA has a <br /> BBSs with their flanking constant region (i.e., the RNA mol-                          1:1 stoichiometry. Because the concentration of the RNA <br /> ecules as they were selected). The conserved region in all four                       probes in these reactions is negligible compared to that of B52, <br /> different sequences was predicted to fold into a common sec-                          we were able to estimate the apparent dissociation constant of <br /> ondary structure element core (Fig. 1C): the double hexamer                           the complex as roughly equal to the protein concentration at <br /> forms the loop and part of the stem of a hairpin loop structure,                      which 50% of the RNA is bound. For example, lane 6 of Fig. <br /> and its flanking regions form the extended stem, which may                            2A, where 50% of RNA is bound, indicates that the Kd of the <br /> contain a bulge or internal loops. In different structures, the                       B52-BBS #14 complex is about 50 nM. The other BBS on this <br /> hairpin loop encompasses 23 to 38 nt with 6 to 12 Watson-                             gel, BBS #8, has an even higher affinity to B52, with a Kd of <br /> Crick base pairs and some G-U pairs. The loops and the tops                           approximately 20 nM. <br /> of the stems are identical in all four structures. Since the                             The specificity of the B52-RNA interaction was examined by <br /> random sequence was only 40 nt in length, it is not surprising                        subjecting it to the challenge of different specific competitors. <br /> that this structural element includes sequences from the                              As shown in Fig. 2B, a constant amount of B52 (5 pmol) was <br /> primer annealing region in some cases. Some bases in the                              incubated with increasing amounts of four different unlabeled <br /> constant region on either side of the random stretch even                             RNA competitors before the addition of either the BBS #8 or <br /> become part of the conserved sequence of different BBSs.                              BBS 14 probe. It is obvious that both BBS #8 and BBS #14 <br /> These structures suggest that the common structural element                           can compete with themselves (Fig. 2B, lanes 1 to 3 and 16 to <br /> may be selected for binding to B52.                                                   18) and with each other (lanes 4 to 6 and 13 to 15), but the <br />    Affinity and specificity of B52-binding RNA aptamers. The                          MGM and the control random sequence (G0) do not compete <br /> B52-binding RNAs were selected by repeated rounds of filter                           with these interactions (lanes 7 to 12 and 19 to 24). <br /> binding. When individual RNAs expressed from cloned se-                                  To further test the specificity of the BBS, we also performed <br /> quences were tested by this filter binding assay, the BBS RNAs                        additional binding assays. RBP1, another Drosophila SR pro- <br /> were all able to bind B52 to generate signals that were 20 times                      tein, and HSF, the Drosophila heat shock factor, did not show <br /> the background level produced by RNAs from the unselected                             any measurable affinity for the BBS RNAs (data not shown). <br /> pools. A band shift assay was also developed as an independent                        Also, as B52 has been shown previously to be able to cross-link <br /> way to assess the affinities and specificities of binding of the                      with DNA in vivo (7), we tested single-stranded DNA contain- <br /> selected sequences to B52, and these results are shown in Fig.                        ing the BBS motif for its ability to bind B52. We detected no <br /> 2. After cloning the selected sequences, we made RNA tran-                            B52-DNA complex in the band shift assay under conditions <br /> scripts without most of the 39 constant region by cutting the                         similar to those used for BBS RNA transcripts (data not <br /> template with the restriction endonuclease ScaI. For BBS #8                           shown). These results indicate that B52 possesses distinct RNA <br /> and BBS #14, this trimming did not interfere with stem for-                           binding specificity for the BBS RNA. <br /> mation. The band shift assay was first performed with these                              Defining the minimum RNA sequence requirements for <br /> sequences and some members of the MGM family. Two se-                                 binding. To determine the sequence and structure require- <br />  VOL. 17, 1997                                                                                           RNA-BINDING SPECIFICITY OF B52              2653 <br />  <br />  <br />                                                                                        to make the construct BBS-I, which consists of just the hairpin <br />                                                                                        loop structure, and it was also fully active. When we deleted <br />                                                                                        the bulged U in the stem to make the construct BBS-I/NoB- <br />                                                                                        ulge, the binding activity was still not compromised. <br />                                                                                           Another construct, BBS-II, which is part of the BBS #11 <br />                                                                                        sequence and has the conserved hairpin loop with an internal <br />                                                                                        loop in the stem, binds B52 as well as BBS-I. However, when <br />                                                                                        several nucleotides on each side of BBS-II were deleted to <br />                                                                                        decrease the stability of the stem, the affinity for B52 decreased <br />                                                                                        almost by a factor of three. This construct, bbs-II, which is <br />                                                                                        comparable to BBS-I in length and contains the sequence <br />                                                                                        shared by all members of BBS family, was predicted to have no <br />                                                                                        stable secondary structure in solution. However, we cannot <br />                                                                                        rule out the possibility that non-Watson-Crick base pairs, base- <br />                                                                                        ribose interactions, and/or base-phosphate interactions may <br />                                                                                        occur in the internal loop region to produce a compact and <br />                                                                                        stable structure or that binding to B52 may stabilize the re- <br />                                                                                        maining weak pairs to give a structure like the one in the full <br />                                                                                        sequences. The last deletion construct, AltStem, contains only <br />                                                                                        the double-hexamer region plus one of the two conserved Gs at <br />                                                                                        the 59 end. The resulting predicted structure, a smaller and <br />                                                                                        slightly different hairpin, still retained some low affinity for <br />                                                                                        B52, but much less than that of bbs-II. <br />                                                                                           The data described above suggest that both the primary <br />                                                                                        sequence and secondary structure of the RNAs contribute to <br />                                                                                        their affinity for B52. Therefore, we made substitution variants <br />                                                                                        to separate the effects of sequence and structure (Fig. 3B). <br />                                                                                        Based on the sequence of BBS-II, we made the variant <br />                                                                                        FlipBBS-II, in which part of the sequences in each half of the <br />                                                                                        stem were interchanged. Based on the sequence of BBS-I, we <br />                                                                                        made the variant TransBBS-I by two GC-to-AU transitions in <br />                                                                                        the stem. Both of these constructs, in which the conserved <br />                                                                                        flanking Gs were replaced, were poor binders even though <br />                                                                                        their predicted secondary structures were unaffected by the <br />                                                                                        changes. Then, based on bbs-II, we tried to convert the unsta- <br />                                                                                        ble internal loop region into a stem by changing sequences on <br />                                                                                        either side of the loop, resulting in the variants bbs-II/59Stem <br />                                                                                        and bbs-II/39Stem, respectively. Although both have a similarly <br />    FIG. 3. Mapping of the minimal binding elements on RNA. (A) Deletion                stable hairpin loop structure, the former, which has only one <br /> analysis defines a minimal binding site. (B) Substitution mutations define key         conserved flanking G, was a weak binder, but the latter, which <br /> features of the binding site. (C) Antisense sequences of the consensus do not <br /> bind B52. Short RNA transcripts were produced by in vitro transcription, and           has both conserved flanking Gs, was a strong one. In making <br /> their affinities for B52 were assayed by band shift. Each 1 sign indicates about a     the strong binder, bbs-II/39Stem, we generated a sequence <br /> threefold increase in affinity. The highest affinity is identical to that of BBS #8.   identical to that of BBS-I between the two endmost conserved <br /> The predicted secondary structure of each RNA is indicated under the sequence.         Gs. <br /> Pairing bases are denoted by matching parentheses, and unpaired bases marked <br /> by dashes. The stabilities of these structures are indicated by their folding energy      As a negative control, we also made antisense RNAs of <br /> (in kilocalories per mole). The substituted bases in panel B are in italic. Although   BBS-I and bbs-II (Fig. 3C). Both antisense constructs did not <br /> the base pairs represented by angle brackets were expected for the construct           bind B52, although AntiBBS-I has a hairpin loop structure <br /> bbs-II as in its parental construct BBS-II, the computer program predicted an          similar to that of BBS-I. In addition, we tested the importance <br /> alternative structure with negligible folding energy (0.2 kcal/mol). <br />                                                                                        of the single strandedness of the core BBS consensus by linking <br />                                                                                        it to its antisense sequence via a UUCG tetraloop. The BBS <br />                                                                                        consensus sequence trapped in an RNA duplex lost its binding <br /> ments for binding by B52, we designed deletion and substitu-                           activity. <br /> tion mutations and antisense constructs of BBSs in light of the                           These experiments demonstrated that the sequence between <br /> shared sequence motif and predicted secondary structures.                              and including the two endmost conserved Gs is required for <br /> These short RNA transcripts were produced by in vitro tran-                            efficient recognition by B52. While most of the conserved se- <br /> scription of synthetic templates, and their affinities to B52 were                     quence resides in the loop, these variants showed that a par- <br /> assayed by band shift.                                                                 ticular sequence of the stem (the conserved flanking Gs) is also <br />    The approximate 59 and 39 termini of the aptamer binding                            critical for this interaction. The flanking regions on both sides <br /> site were determined by deletion analysis (Fig. 3A). Based on                          of this conserved sequence segment can contribute to binding <br /> the sequence of BBS #8, we first made an RNA construct,                                in cases where they help form a more stable stem, probably by <br /> BBS-I/Long, containing the region shared by all BBS RNAs                               prepaying some entropic cost of the interaction. Thus, a strong <br /> plus the region shared only with BBS #14 but missing both 59                           B52-binding site on RNA is at least 17 nt (the core BBS, G <br /> and 39 constant regions. It is predicted to retain the BBS #8                          through G) or around 20 nt (the length of most hairpin loops <br /> hairpin loop structure, which is the shortest and most stable                          tested) in length. <br /> hairpin among the four BBSs, plus a 39 unfolded tail. This                                Probing the secondary structure and critical regions in the <br /> construct has full binding activity. Next, we deleted the 39 tail                      B52-binding site of RNA. In order to delineate the B52-binding <br />  2654       SHI ET AL.                                                                                                                                MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />    FIG. 4. Localization of the B52-binding site on RNA. (A) RNase footprinting with RNase T2 (lanes 2 to 4) or RNase V1 (lanes 9 to 11) identifies the hairpin loop <br /> of BBS #8 as the BBS. RNase was added to binding reaction mixtures lacking B52 (lanes 2 and 9), with 2 pmol of B52 (lanes 3 and 10), or with 20 pmol of B52 (lanes <br /> 4 and 11), and the resulting RNA products were analyzed by primer extension. The input RNA is shown to identify nicks in the RNA as well as reverse transcriptase <br /> stops (lane 1). The sequencing ladder (lanes 5 to 9) is used to identify the bases of interaction. Open circles indicate weak T2 protection, and closed circles and squares <br /> indicate strong T2 and V1 protection, respectively. The single-stranded regions and the sequence in the stem of the hairpin loop structure are indicated next to the <br /> sequencing ladder by thin and thick lines, respectively. (B) Partial predicted secondary structure of the BBS #8 RNA in the context of vector sequence. The sequence <br /> of BBS #8 is indicated by the nucleotides, and the constant and vector sequences are depicted by a skeletal diagram. Symbols are as in panel A. <br />  <br />  <br />  <br /> region on the selected RNA at a higher resolution, we per-                              made a set of [35S]methionine-labeled B52 deletion constructs <br /> formed RNase footprinting with the cloned BBS #8. Two                                   (Fig. 5A) by in vitro translation and tested their ability to bind <br /> RNases with different specificities were used to determine the                          the selected BBS #8 RNA in two different assays. These poly- <br /> region of binding in both single- and double-stranded areas of                          peptides also serve as a second source of target proteins in <br /> RNA. These experiments have the added benefit of confirming                             binding assays to confirm the specificity of binding to the se- <br /> the predicted hairpin loop secondary structure of the selected                          lected BBS RNA. <br /> RNA.                                                                                       The in vitro-translated polypeptides were examined by SDS- <br />    RNase T2 is a single-strand-specific endoribonuclease with                           PAGE to verify that they were of the predicted sizes before <br /> low specificity (a preference for adenines), while RNase V1                             being used in the binding assays. The integrity of two polypep- <br /> cleaves RNA predominantly at double-stranded regions with                               tides, R1 and R12, is shown in Fig. 5C (see below). Other <br /> no apparent sequence specificity (9). Both free RNA and                                 polypeptides were made in an identical way and had similar <br /> RNA-B52 mixtures were treated with RNases T2 and V1, and                                quality. Figure 5B shows a band shift assay with 35S-labeled <br /> the resulting RNA products were detected by primer exten-                               polypeptides individually or in combination. The binding reac- <br /> sion. This allows an accurate identification of the protected                           tion was set up as for the experiments described above but with <br /> nucleotides, since sequencing reactions with the same primer                            a constant amount of BBS #8 RNA transcript that was in a <br /> can be run alongside the footprinting reactions. A comparison <br />                                                                                         molar excess relative to protein. In the presence of BBS #8 <br /> of the primer extension products from free RNA with those <br />                                                                                         RNA, 35S-labeled full-length B52 (F) showed a discrete band <br /> from RNA-B52 mixtures identifies residues that are in contact <br />                                                                                         (Fig. 5B, lane 2), which has the same mobility as that of a <br /> with the protein. As the B52 concentration is increased, the <br /> intensities of several bands decrease (Fig. 4A, lanes 2 to 4 and                        B52-BBS #8 complex generated with 32P-labeled RNA (Fig. <br /> 9 to 11). Comparing the positions of these bands directly to the                        2). The polypeptide containing both RRMs (R12) gave rise to <br /> sequencing ladder identifies bases that are protected by the                            a band running faster than the full-length B52-BBS #8 com- <br /> binding of B52 (Fig. 4B). These B52-protected bases reside in                           plex (Fig. 5B, lane 4). The SR domain (S) (Fig. 5B, lane 6), <br /> the predicted hairpin loop region. In addition, the cleavage of                         RRM I (R1) (lane 10), and a polypeptide containing both <br /> the RNA substrate by these RNases in the absence of B52                                 RRM II and the SR domain (R2S) (lane 12) did not produce <br /> confirms the predicted hairpin loop secondary structure of the                          any discrete band on this native gel. When R12 and S were put <br /> selected RNA. The RNase protection results provide physical                             together in the presence of the RNA, a band (Fig. 5B, lane 8) <br /> evidence that the conserved hairpin loop of BBS RNA is the                              with mobility similar to that seen with R12 (Fig. 5B, lane 4) was <br /> target of B52 interaction.                                                              observed. No shifted band was produced with the other com- <br />    Locating the RNA-binding site of B52 in both RRMs. BBS                               bination, polypeptides R1 and R2S. These data indicate that <br /> RNAs were selected for their ability to bind full-length B52                            the SR domain of B52 does not participate in the interaction <br /> protein. It is possible that B52 binds RNA through either one                           with the RNA aptamer and that both RRMs are required for <br /> or the other RRM, the SR domain, or a combination of these                              the binding. In addition, the affinity of the RNA for both in <br /> domains. To identify the specific RNA-binding site on B52, we                           vitro-synthesized polypeptides F and R12 is comparable to that <br />  VOL. 17, 1997                                                                                           RNA-BINDING SPECIFICITY OF B52                      2655 <br />  <br />  <br />  <br />  <br />   FIG. 5. RNA-binding site on B52 localized to both RRMs. [35S]methionine-labeled B52 deletion constructs (A) (aa, amino acids) were made by in vitro translation, <br /> and their ability to bind BBS #8 RNA was examined in two different assays. The binding reactions with different 35S-labeled proteins or combinations thereof were <br /> monitored by band shift assay on a native agarose gel (B) or UV cross-linking and SDS-PAGE (C). The closed arrowhead in panel C signifies the B52&#226;&#8364;&#8220;R12-BBS #8 <br /> complex, whereas the open arrowhead points to where the B52&#226;&#8364;&#8220;R1-BBS #8 complex would be expected to migrate. <br />  <br />  <br />  <br /> measured previously (Fig. 2) by using baculovirus-produced                         i.e., the two RRMs and the RS domain. Since its large and <br /> B52 protein in excess over RNA (see Materials and Methods).                        positively charged RS domain has the potential to interact with <br />    To confirm the requirements of both RRMs for binding, we                        polyanionic RNA either specifically or nonspecifically, we used <br /> repeated these binding reactions and subjected the reaction                        full-length B52 in our selection scheme to make sure that all <br /> mixtures to UV irradiation. The UV-cross-linked species were                       epitopes on the protein surface were initially available for <br /> then analyzed by SDS-PAGE. As shown in Fig. 5C, R12 can be                         interaction with the RNA pool. The RNA aptamer that was <br /> cross-linked to the BBS 8 RNA, forming a complex with an                           eventually selected bound to both RRMs but not the RS do- <br /> apparent molecular weight the sum of those of the protein and                      main of B52, consistent with the findings for other SR proteins <br /> RNA components (lane 6). In contrast, R1 was not cross-                            such as SF2/ASF and SC35 (43). Moreover, the presence of the <br /> linked to RNA (Fig. 5C, lane 3). In these reactions, equal                         RS domain does not compromise the binding of B52 to RNA. <br /> molar amounts of protein were used. Besides, both RRMs                             However, in contrast with two previous in vitro selection ex- <br /> must be in cis to bind RNA, since R2S cannot complement R1                         periments with SR proteins, which yielded short, unstructured <br /> to restore the binding even when the RNA is at much a higher                       consensus sequences (21, 43), the BBSs are long and structur- <br /> concentration (data not shown).                                                    ally well defined. <br />                                                                                       By analogy to other RRMs such as that of the U1A protein, <br />                              DISCUSSION                                            whose structure is known, we can make some predictions about <br />                                                                                    the structure and RNA-binding properties of B52. In the crys- <br />    The SR proteins are a structurally related family of proteins                   tal structure of the U1A protein RRM-RNA complex, the <br /> that play a key role in early pre-mRNA splicing events (13, 30).                   RNA loop binds to the protein as an open structure, and the <br /> Not only are they critical in committing pre-mRNAs to splicing                     b-sheet provides a large surface of extensive interaction with <br /> in vitro, but they also control the splice site choice of alterna- <br />                                                                                    splayed-out nucleotide bases (33). Bases of seven nucleotides <br /> tively spliced transcripts. These functions are likely to be im- <br />                                                                                    in the loop are involved in direct or water-mediated hydrogen <br /> portant in normal development, although the true in vivo tar- <br />                                                                                    bonds. The core BBS is long enough to comprise two such <br /> gets of many of these factors have remained elusive. The <br />                                                                                    binding sites, interacting with both RRMs of B52 in a similar <br /> <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span>knockout</span> of the B52 gene in Drosophila (35) currently provides <br /> the only demonstration of an essential role of an SR protein                       manner. Upon binding B52, the loop region of the BBS RNA <br /> family member during development. B52 possesses two RRMs                           may be enlarged to accommodate both RRMs. Alternatively, <br /> as well as a large basic domain composed of multiple strings of                    sequence information may be read from the stem as well as the <br /> RS dipeptide repeats, and thus it is expected to interact di-                      loop. Similar to our findings, heterogeneous nuclear ribonu- <br /> rectly with nucleic acid (7). In an attempt to identify sequences                  cleoprotein particle A1, which also contains two RRMs, se- <br /> in RNA that are recognized by B52, we have isolated RNA                            lected an RNA aptamer having two nearly identical hexamers <br /> aptamers from a large population of random RNAs that bind                          separated by two variable nucleotides (5). <br /> tightly to B52. The BBS we defined is longer than many con-                           As mentioned above, the N-terminal RRM of B52 displays a <br /> sensus sequences selected by other SR proteins and non-SR                          very high similarity to the RRM consensus sequence, while the <br /> proteins containing RRMs (7, 21, 43, 45). In addition, it has                      second RRM is considerably different from the consensus se- <br /> some symmetry and forms a hairpin structure. We find that                          quence. Although single RRMs of SR proteins have been <br /> both RRMs of the B52 protein act together to bind an RNA                           shown to bind RNA or to be able to select RNA aptamers, the <br /> hairpin in a sequence-specific interaction.                                        overall binding specificity of the intact protein containing mul- <br />    Both RRMs are involved in binding. In vitro selection                           tiple RRMs may not be the sum of the specificities of the <br /> (SELEX) has been used to select oligonucleotides that bind to                      individual RRMs. The B52-binding RNA aptamer that we <br /> proteins that recognize nucleic acid as their normal function                      selected failed to bind either RRM. The two RRMs of B52 <br /> (3, 29, 37, 46) as well as to proteins that do not bind nucleic                    appear to act as a single RNA-binding composite, cooperating <br /> acid by nature (10, 44). The B52 protein consists of three                         in some way to determine both the RNA binding affinity and <br /> different domains that are potentially capable of binding RNA,                     specificity of the intact molecule. <br />  2656     SHI ET AL.                                                                                                                MOL. CELL. BIOL. <br />  <br />  <br />    Selection of the B52-binding sequences. RNA molecules are          function in vivo, by creating Drosophila fly lines designed to <br /> generally sequence-dependent folded structures rather than            overproduce BBS RNAs (40). We envision that the controlled <br /> simple strings of nucleotides. Current evidence concerning            and modulated expression of these RNA aptamers in living <br /> protein-RNA interaction suggests that proteins often make             flies will serve as a specific, reversible, fast-acting B52 inhibitor <br /> sequence-specific contacts with single-stranded residues in           that should provide insight into the function of B52 in vivo. <br /> loops, while base-paired regions are often necessary for pre- <br /> sentation of single-stranded nucleotides in the correct orien-                                   <br /> tation (49). To maximize the structural information provided           <br /> by the RNA molecule in an RNA-protein interface, the ran-           <br /> dom region used in in vitro selection should be as long as             <br /> possible. However, in most cases the complete representation          <br /> of a variable region that is longer than 25 nt would exceed the                                    <br /> practical limit of oligonucleotide synthesis. In addition, we   <br /> found that the yield and quality of the product decrease dra-      <br /> matically when its total length exceeds 100 nt in most oligonu-    <br /> cleotide synthesis facilities. Taking these factors into account,   <br /> we decided to use a template with a random region of 40 nt.            <br /> This size and the required constant flanking region would      <br /> require the synthesis of a template slightly over 100 nt in      <br /> length. Such RNAs would be able to accommodate at least one            <br /> stable hairpin loop structure or be long enough to be folded         <br /> into a pseudoknot.                                                      <br />    The core BBS site that we selected is less than 20 nt in           <br /> length. To cover the sequence space of a 20-mer, 420 or 1012             <br /> different sequences are needed. This requirement was met in               <br /> our selection, since the original pool contains at least 7.1 3          <br /> 1014 different 20-mer sequences (i.e., 3.4 3 1013 40-mers).   <br /> Therefore, the B52-binding sequences we selected may well be       <br /> the best ones available, a notion also supported by the nearly     <br /> perfect consensus we observed in unrelated sequence contexts.         <br /> Nine rounds of selection should be sufficiently rigorous to       <br /> account for the low complexity of the final selected pool.          <br />    The potential significance of B52&#226;&#8364;&#8482;s RNA-binding specificity.       <br /> By in vitro selection, we examined the inherent RNA-binding         <br /> capacity of B52 by exposing it to a vast sequence and structure     <br /> space of RNA. This does not necessarily mean that we have             <br /> found the native RNA sequence and structure with which B52                 <br /> interacts in vivo. It is possible that the native B52-binding sites       <br /> on Drosophila RNA are more degenerate or shorter than the     <br /> BBS we selected. Nonetheless, this RNA sequence can be used           <br /> to help identify potential targets of B52 in the Drosophila       <br /> genome. Indeed, we found the second hexamer of BBS (59-G       <br /> GCGAC-39) in an exon of ftz pre-mRNA, which has been used     <br /> as the splicing substrate in a functional assay of B52 (22). This        <br /> site is located near the 59 splice site in secondary structure, and   <br /> a deletion that includes this site abolishes splicing. Our pre-          <br /> liminary results showed that this pre-mRNA substrate is able    <br /> to bind B52 in a BBS-competable manner with an affinity lower     <br /> than that of the BBS. A search for BBS-like sequences in the      <br /> Drosophila sequence database yielded several Drosophila genes     <br /> matching the core BBS consensus sequence at 11 or more of       <br /> the 14 invariant positions. The pre-mRNAs encoded by these          <br /> genes are prime candidates for B52 action, and investigation of       <br /> B52&#226;&#8364;&#8482;s possible in vivo role in splicing these pre-mRNAs should        <br /> benefit from the availability of Drosophila B52 mutants (34,     <br />    Aptamers as reagents in vivo. The aptamers we selected for        <br /> B52 provide a useful tool to study B52 function both in vitro          <br /> and in vivo. RNA aptamers have been shown to have a pen-     <br /> chant to home to functional sites on protein surfaces and                 <br /> discriminate finely among these sites (18). The aforementioned         <br /> result regarding the splicing of ftz pre-mRNA suggests that the          <br /> BBS may interact with a functionally significant region of B52, <br /> and this binding may be strong enough to disrupt its normal            <br /> interaction with pre-mRNA substrates. This has encouraged us           <br /> to make use of BBS RNA as a genetic tool for dissecting B52               <br /> </body></html>