<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL    OF   BACTERIOLOGY, Mar. 1989,   p.   1631-1637                                                               Vol. 171, No. 3 <br /> 0021-9193/89/031631-07$02.00/0 <br /> Copyright C) 1989, American Society Microbiology <br />  <br />  <br /> Nucleotide Sequence Expression cheF, Essential Gene for <br />                  Chemotaxis Bacillus subtilis <br />                                            CHINGWEN YING AND GEORGE W. ORDAL* <br />                        Department Biochemistry, School Chemical Sciences College Medicine, <br />                                           University Illinois, Urbana, Illinois 61801 <br />                                            Received 23 September 1988/Accepted 9 December 1988 <br />  <br />               The cheF gene, involved chemotaxis Bacillus subtilis, cloned, expressed, and <br />            sequenced. This gene contained 0.7-kilobase PstI DNA fragment isolated A Charon 4A <br />            B. subtilis chromosomal DNA library. This fragment subcloned expression vector pSI-1 shown <br />            complement cheF <span id='am-81' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-82' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span>mutation</span> chemotaxis methanol production response addition <br />            attractants. Plasmid-encoded DNA expression B. subtilis maxicells indicated membrane-associated <br />            polypeptide 20-kilodaltons expressed 0.7-kilobase DNA. The nucleotide sequence DNA <br />            fragment determined, open reading frame capable encoding putative 175-amino-acid protein <br />            (M, 20,002) identified. In effort understand function cheF protein, dosage cheF <br />            gene product varied altering concentration IPTG (isopropyl-gB-D-thiogalactopyranoside) during <br />            growth. In presence high concentrations IPTG, chemotaxis inhibited methanol production <br />            impaired. <br />  <br />  <br />  <br />    Bacterial chemotaxis primitive sensory mechanism                             MATERIALS AND METHODS <br /> which bacteria swim higher concentrations attrac- <br /> tant lower concentrations repellent. Much                      Bacterial strains, plasmids, phages. E. coli strains <br /> knowledge pathway information processing                  JM103 DHaS used propagating plasmids. <br /> bacterial chemotaxis result work                    Strains JM103 NM522 hosts M13 phage <br /> gram-negative bacteria Escherichia coli Salmonella ty-                vectors pTZ phagemids, respectively. Complementation <br /> phimurium (17, 21). However, recent experiments (26a)                expression studies performed isogenic strains in <br /> suggested pathway chemotactic signal processing               B. subtilis strain 011925. The plasmid used subcloning <br /> might quite different gram-positive Bacillus subtilis.          pSI-1 (31). This shuttle plasmid contains spac-I <br /> Previous work suggests methyl groups transferred                 promoter lacI gene, shown <span id='am-2' about='protege:TO' typeof='owl:Thing'>to</span> <br /> via intermediate carriers methyl-                    function B. subtilis (32). DNA fragments subcloned <br /> accepting chemotaxis proteins (MCPs) form methanol (26,                M13mpl8 M13mpl9 phage vectors nucleotide <br /> 26a). Current evidence suggests methanol evolves di-                 sequence analysis. Two phagemids, pTZ18U pTZ19U, <br />                                                                           used vectors DNA fragments the <br /> rectly MCPs E. coli (21). In B. subtilis, methanol is <br /> generated response addition removal attrac-              sequencing study. Two phages (X-14.9 X-11.7) harboring <br /> tants repellents (26; M. Thoelke G. Ordal, unpub-                 clustered che genes isolated X Charon 4A library <br /> lished data). To explore chemotactic mechanism B.                  (19). A 7.7-kilobase EcoRI DNA fragment X-11.7 was <br /> subtilis, number chemotactic mutants isolated               subcloned pFH7 shown capable comple- <br /> characterized (19). More 20 complementation groups                   menting cheF mutations (20). The cheF mutants defective in <br /> were shown involved (20). All mutants motile, and <br />                                                                           general chemotactic stimulation identified various <br />                                                                           chemotactic assays (20). This 7.7-kilobase EcoRI DNA <br /> the corresponding genes cheR located                fragment cloned pUC18, restriction <br /> same region chromosome, thyA pyrD. <br />                                                                           map fragment determined (C. Ying G. Ordal, <br /> Mutation cheF locus results failure respond               unpublished results). This chimeric plasmid, designated <br /> chemotactic stimuli (20). The region coding cheF                  pGO101, subsequently used subcloning experi- <br /> protein previously localized 7.7-kilobase segment                ment. All strains plasmids used listed Table 1. <br /> isolated A Charon 4A B. subtilis chromosomal DNA                      Construction B. subtilis strains. The chemotactic wild- <br /> library (20). As continuing investigation              type strain, 011085, used previous study (29) shown <br /> molecular mechanism involved chemotaxis, cheF gene                 poor host plasmid DNA transformation. A strain <br /> was subcloned expression vector gene prod-                offered reasonable transformational frequency thus <br /> uct identified. This procedure allowed detection               crucial subcloning complementation. 011878, a <br /> subcellular location cheF protein. The nucleotide                  derivative 011085, constructed (see Table 1 for <br /> sequence cheF gene determined. To                      construction procedure). This strain transformed <br /> analyze gene product, dosage cheF protein              plasmid DNAs high frequency. The cheFJ41 allele <br /> varied vivo, effects chemotaxis methanol                subsequently recE4 allele transferred into <br /> production quantitated.                                              011878 (for constructions, Table 1). This strain, 011925, <br />                                                                           used cheF mutant following studies. <br />                                                                              Culture media growth conditions. Unless otherwise <br />                                                                           stated, E. coli B. subtilis grown L broth on <br />   * <br />       Corresponding author.                                               tryptone plates containing ampicillin chloramphenicol (35 <br />                                                                    1631 <br />  1632        YING AND ORDAL                                                                                                       J. BACTERIOL. <br />  <br />             TABLE 1. Bacterial strains plasmids used                         plast, methyltransferase, chemotaxis buffers pre- <br />  Bacterial strain                                                                pared according method Ullah Ordal (29). <br />                                Genotype                 Source reference         Transformation. Transformation B. subtilis plas- <br />      plasmid <br />                                                                                  mid DNA performed according method Dub- <br /> Bacterial strain                                                                 nau Davidoff-Abelson (3). The method Maniatis et al. <br />   B. subtilis                                                                    (12) used transform E. coli plasmid DNA. <br />      011085           metC trpF7 hisH2 Che+            29 <br />      M1112            leuA8 argi5 thrA recE4           25                           Chemotactic assays B. subtilis strains. Fresh transfor- <br />                         rm- mm                                                   mants picked plates toothpicks over- <br />        01363          ilvCI leu-l                      18                        night incubation spotted tryptone swarm plates <br />        011878         leu-l trpF7                      This work&apos;                containing S jig chloramphenicol ml. Plates were <br />        M141           metC trpF7 hisH2                 19                        incubated wet incubator 5 12 h. The swarm <br />                         cheF141                                                  phenotypes colony observed. Capillary assays <br />        IA302          argC4 hisAl recE4                Bacillus Genetic          performed described previously (20). Capillary tubes <br />                                                          Stock Center <br />        011881         trpF7 cheF141                    This work&apos;                filled 3 x 10-5 M mannitol 102 M aspartate were <br />        011925         trpF7 cheF141 recE4              This workc                placed cell suspension. Reactions continued for <br />        OI1929         trpF7 cheF141 recE4              This work                 45 min 37&#194;&#176;C. Bacteria accumulated inside capillary <br />                         pGO102                                                   tube plated. After overnight incubation, number of <br />        011933         trpF7 cheF141 recE4              This work                 colonies plate counted. <br />                         pSI-1                                                       Changes methanol evolution addition attrac- <br />   E. coli                                                                        tants detected Conway diffusion cells (29). Bacteria <br />     JM103             supE thi A(lac-proAB)             15 <br />                         (F&apos; traD36 proAB                                         grown L broth harvested cell density <br />                         lacIq lacZAM15)                                          180 Klett units. Cells washed chemotaxis <br />        NM522          hsdA5 A(lac-proAB)               6                         buffer suspended protoplast buffer density A6. <br />                         supE thi (F&apos; proAB                                       = 1.0. Protoplasts cells prepared adding 1 <br />                         lacl&quot;ZAM15)                                              mg lysozyme ml buffer. These protoplasts were <br />        DH5a           F- endAl hsdR17                  9                         labeled L-[methyl-3H]methionine 5 min 37&#194;&#176;C and <br />                         (rK MK ) supE44 thil                                     mixed 2.3 x 10-2 M aspartate Conway diffusion <br />                         X- recAl gyrA96 re/Al <br />                         ,80dlacZAM15                                             cells. The reaction stopped, methanol <br />                         A(lacZYA-argF)U169                                       released cells quantitated according method <br />                                                                                  Thoelke et al. (in press). <br /> Plasmid                                                                             Plasmid-encoded DNA expression B. subtis maxiceDls. B. <br />   pGO101              Apr                              G. W. Ordal, un-          subtilis 011925 used host study. Cells were <br />                                                           published data         prepared UV irradiated according Shimotsu et al. (23) <br />   pSI-1               Cmr                              33                        modifications. After irradiation, cells were <br />   pGO102              Cmr; 700-bp PstI               This work                 incubated 37&#194;&#176;C 30 min followed addition 20 ,g of <br />                         fragment containing <br />                         cheF inserted                                   D-cycloserine ml, incubation continued 16 to <br />                         pSI-1 pGO101                                        24 h. Cells labeled [35S]methionine lysed <br />   pTZ18U              Apr                       13                               described Hirochika et al. (10). Labeled proteins were <br />   pTZ19U              Apr                       13                               visualized autoradiography. <br />   pJH101              Apr Cmr                   4                                   To separate membrane fraction cytoplasmic <br />  a011085 competent cells transformed Met&apos; chromosomal               fraction plasmid-encoded proteins maxicells, a <br /> DNA M1112. One transformants high transformation               modified method used. Bacteria grown, irradiated <br /> frequency pSI-1 cotransfornation rM- genetic locus.   UV light, labeled [35S]methionine described <br /> This strain subsequently His&apos; Leu- using chromosomal DNA             above. Protoplasts cells adding 1 mg <br /> <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> 01363. <br />   b The cheF141 mutation transferred M141 chromosomal DNA               lysozyme ml cells suspended 1 ml of <br /> into OI1878 competent cells, characteristic chemotactic <span id='am-3' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-4' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-5' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span>phenotype</span>     protoplast buffer. Cells incubated 37&#194;&#176;C 45 min <br /> the cheF141 mutation constructed strain, 011881, checked          harvested. Cell pellets lysed in <br /> chemotaxis assays.                                                               50 ,ul water 4&#194;&#176;C. Membrane cytoplasm fractions <br />   c <br />     The allele recE4 introduced 011881 competent cells the <br /> chromosomal DNA IA302 gene cong plasmid pSI-1. Colonies          separated centrifugation airfuge 100,000 x g <br /> which Cmr screened plates containing 0.6 jLl ethyl methane-      45 min 4&#194;&#176;C. The membrane fraction precipitated the <br /> sulfonate 5 iLg chloramphenicol ml. Ethyl methanesulfonate-           pellet suspended 50 RI 1 M NaCl prepared in <br /> sensistive colonies identified checked swarm plates containing       methyltransferase buffer solution centrifuged an- <br /> the concentration chloramphenicol. One colonies which <br /> showed cheF141 phenotype chosen. Plasmid pSI-1 segre-           45 min 100,000 x g 4&#194;&#176;C. After centrifugation, <br /> gated strain. In manner, 011925, contained cheFI4l      membrane-associated molecules remained the <br /> and recE4 alleles, constructed.                                              supernatant separated membranes. Each fraction <br />                                                                                  suspended sodium dodecyl sulfate solubilizer (62 mM <br />                                                                                  Tris, 2% sodium dodecyl sulfate, 10%o glycerol, 5% f- <br /> or 5 ,ug/ml, respectively) needed plasmid selection.                    mercaptoethanol, pH 6.8), cell debris removed. <br /> The resuspension medium Sterlini Mandelstam (24)                          Proteins fraction fractionated sodium dode- <br /> was used maxicell experiments. Media used transfor-                        cyl sulfate-polyacrylamide gels. <br /> mation B. subtilis SPI SPII (3). For capillary                         DNA preparation analysis. The chromosomal DNA of <br /> assays, bacteria grown minimal medium supple-                            B. subtilis prepared standard methods. Plasmid <br /> mented 0.2% sorbitol amino acids required                           DNAs isolated E. coli B. subtilis described <br /> growth (20). Tryptone swarm plates mannitol swarm                            Maniatis et al. (12). Restriction endonucleases T4 <br /> plates used described previously (19). The proto-                        DNA ligase obtained Bethesda Research Labora- <br />  VOL. 171, 1989                                                 SEQUENCE AND EXPRESSION OF cheF IN B. SUBTILIS                        1633 <br />  <br />                                                                        primer sequencing purchased Bethesda Re- <br />                                                                        search Laboratories. [35S]dATP supplied Amersham <br />                                                                        Corp. (Arlington Heights, Ill.). Other chemicals nucle- <br />                                                                        otides sequencing reaction purchased kit <br />                                                                        U.S. Biochemical. <br />                                                                                                  RESULTS <br />                                                                           Subcloning cheF gene. The plasmid DNA pGO101 <br />                                                                        digested PstI, liberated DNA fragments. <br />                                                                        Inserts ligated linearized pSI-1. Competent <br />                                                                        cells OI1925 transformed ligated DNA <br />                                                                        mixtures. Transformants selected, swarm <br />   FIG. 1. Complementation cheF mutation pGO102.              phenotype transformants screened. The swarm <br /> cheF mutants containing pSI-1 (A) pGO102 (B) were <br /> spotted tryptone swarm plate supplemented 5 jig         rate cheF mutants shown slow compared <br /> chloramphenicol ml.                                                wild-type strains (20). Che+ transformants were <br />                                                                        identified (Fig. 1) basis wild-type swarm <br />                                                                        phenotype, plasmid DNAs prepared these <br /> tories, Inc. (Gaithersburg, Md.) used recommended               cells. All Che+ transformants carried 700-<span id='am-159' about='xsp:base' typeof='owl:Thing'>base</span>-pair PstI <br /> by supplier.                                                       fragment cloned pSI-1. This pSI-1 derivative desig- <br />    DNA sequencing techniques. M13mpl8, M13mpl9 (30),                   nated pGO102 (Fig. 2). Cells containing pSI-1 did con- <br /> and phagemids pTZ18U pTZ19U (13, 14) used                  vert Che- phenotype Che+. To distinguish whether <br /> vectors sequencing study. The nucleotide sequences              Che+ phenotype transformants resulted from <br /> of inserts recombinant clones determined                pGO102 complementation reversion cheFJ41, the <br /> dideoxynucleotide chain termination method Sanger et al.            following tests performed. Plasmids segregated <br /> (22). Sequenase (U.S. Biochemical Corp.) used              Che+ transformants growing transformants <br /> chain elongation sequencing reactions. A 15-mer universal           absence chloramphenicol period time. It <br />  <br />                                                                                                  Pstl <br />                                                                                   Pst1                    Pst l <br />                                                                              Pstli <br />  <br />  <br />                                            Pstl <br />                                   spac-I              lac <br />  <br />                                                           BamHl          EcoR l                          EcoR I <br />                                            PSI-1 <br />                            pUBllO                    pBR322 ori <br />  <br />                                             CAT <br />                                                                                                  Pstl <br />  <br />  <br />  <br />  <br />                                                                      Pst 1 <br />                                                                                   st <br />                                                             spac-It <br />                                                                                        act <br />                                                                   PGOIm       2           amHl <br />  <br />  <br />  <br />                                                                   CAT <br />   FIG. 2. Construction pGO102. A 700-base-pair PstI fragment (dotted box pGO101) inserted PstI site pSI-1. The solid <br /> box pGO101 represents 7.7-kilobase B. subtilis chromosomal DNA pUC18 vector, shown line. <br />  1634       YING AND ORDAL                                                                                                                               J. BACTERIOL. <br />  <br />                                                                                              TABLE 2. Chemotaxis various levels cheF gene product <br /> A                                                   B                                                      IPTG         Swarming           Relative          % Increase <br />                                                                                              Strain        concn           rate         aulatioe             methanol <br />                                                                                                            (mM)           (cm/h)                             production&apos; <br />  <br />                                                              12 34                          011929          0 <br />                                                                                                             0.01 <br />                                                                                                                           0.038 <br />                                                                                                                           0.104 <br />                                                                                                                                              3.0 <br />                                                                                                                                              3.14 <br />                                                                                                                                                                   30 <br />                                                                                                                                                                  ND&apos; <br />   31- <br />                                                                                                             0.1           0.092              6.9                  68 <br />                                                                                                             0.75          0.142              ND                  105 <br />                                                     31-                                                     1.0           0.107             21.5                  86 <br />                    ....                                             .:&apos; a&gt; <br />                                                                                                            10.0           0.106             13.4                  35 <br />                                                                                                           100.0           0.105              3.0                 ND <br />                    m2                                                                       011925          0             0.02               0.7                  10 <br />                                                                                             011878          0             0.24              23.1                 125 <br />                     .........~.                                       c <br />                                                                                               Expressed number cells accumulated capillary tube containing <br />                                                                                             3 x 10-&apos; M mannitol/number accumulated tube containing buffer. Each <br />                                                                                             value mean replications. <br />                                                                                               b Amount generated addition 2.3 x 10-2 M aspartate versus amount <br />                                                                                             generated buffer added. <br />                                                                                               C ND, Not determined. <br />   FIG. 3. Expression localization cheF gene product. <br /> (A) Plasmid-encoded polypeptides p00102 (lane 1)                             pSI-1 <br /> (lane 2) expressed B. subtilis maxicells. Strains containing                        possible band represented modified form of <br /> either p00102           pSI-1         UV irradiated labeled [355]          major protein. <br /> methionine. Proteins   fractionated 12.5% sodium dodecyl                             To localize cheF protein, labeled maxicells were <br /> sulfate-polyacrylamide gel visualized autoradiography. A                             ruptured fractionated membrane, membrane-associ- <br /> 250-ng protein loaded lane. Molecular mass                            ated, cytoplasm fractions ultracentrifugation. The <br /> standards (in kilodaltons) shown left. (B) The cheF protein                      cheF gene product behaved like membrane-associated <br /> was labeled UV programming. Extracts separated the <br />                                                                                             protein, removed membrane fraction <br /> cytoplasm (lane 1) membrane (lane 2) fractions ultracen- <br /> trifugation. The membrane fraction lane 2 washed with <br />                                                                                             salt wash membranes (Fig. 3B). <br /> 1 M NaCl centrifuged. After centrifugation, cheF <br />                                                                                                Expression level cheF product chemotaxis. To <br /> protein appeared supernatant fraction (lane 3)                               study effect altered cheF expression chemotaxis, <br /> removed membrane fraction (lane 4). Methods separat-                           gene placed control spac-I pro- <br /> ing fraction described Materials Methods. Approx-                           moter, regulated addition an <br /> imately 350 ng protein loaded lane; molecular mass                           inducer, IPTG. The level cheF gene product was <br /> standards (in kilodaltons)                shown left.                            altered vivo exposing cells various concentra- <br />                                                                                             tions IPTG swarm plates. <br />                                                                                                In absence inducer, slow swarming rate was <br /> was    cells <br />                            regained swarm phenotype,                                    conferred plasmid (Table 2). The swarming rate the <br /> which      indistinguishable cheF mutants. <br />                                                                                         cells reached maximum 0.75 mM IPTG then <br /> 011925 retransformed p00102 plasmid DNA,                                       decreased. As expected, IPTG did affect swarming the <br /> the swarm phenotype transformants examined. All                                      wild type (OI1878). Therefore, swarming rate inhib- <br /> transformants checked                       Che&apos;.     To confirm          ited overproduction cheF protein. <br /> this   Pstl   DNA         fragment complement                cheF mutations,             To investigate effect elevated dosage of <br /> another chemotaxis assay                        used. Chemotaxis 011925              cheF protein behavior, capillary assay used. <br /> and 011929, contained                       p00102,             analyzed    Cells grown various IPTG concentrations then <br /> the    capillary   assay. In assay,                   011925        011929    washed buffer assay performed. Bacte- <br /> were     grown              presence         1   mM      IPTG       (isopropyl-   ria responded mannitol poorly grown a <br />  13-D-galactopyranoside) induce expression plasmid-                               medium IPTG (Table 2). They responded best when <br /> encoded   polypeptides (32). 011925 showed poor taxis                                  grown 1 mM IPTG. The maximal accumulation the <br /> to attractant aspartate capillary assay (20). The                                achieved wild-type control, 011878. It was <br /> relative accumulation 011925 10-2 M aspartate                                    observed high IPITG concentration medium <br /> 1.49. 011929, hand, responded concen-                                   impaired chemotaxis OI1929. The expression level of <br /> trationgradient aspartate did wild type.                                  cheF showed similar effect capillary assays aspartate <br /> 011925 011878 (wild type) showed relative accumulation                                  (data shown). <br /> values 11.5 10.9, respectively, concen-                                    The effect altered cheF expression levels methanol <br /> tration aspartate.                                                                       production explored. Labeled cells been <br />   Identification localization cheF gene                            product    grown presence different IPTG concentrations were <br /> using   B. subtilis maxicells. We established cheF                             mixed aspartate Conway diffusion cells. The metha- <br /> structural gene              contained        pG0102.       To determine         nol released cells result aspartate stimulation <br /> size cheF  protein, plasmid-encoded polypeptides                                 quantitated (Table 2). The optimal expression level of <br /> were expressed p00102 vivo using B. subtilis                                     cheF gene product methanol formation occurred at <br /> maxicells (Fig. 3). Strains 011925, 011933, 011929                                 0.75 mM IPTG. The methanol evolved IPTG concen- <br /> UV irradiated, newly synthesized proteins labeled                                  tration 80% evolved wild type (011878). <br /> with    [35S]methionine              described Materials             Methods..    The effect elevated cheF protein dosage methanol <br /> One  predominant protein band unique p00102 ob-                                      production similar accumulation the <br /> served molecular weight 20 kilodaltons (Fig. 3A). A                                 capillary assay. <br /> second faint, slightly larger protein band observed;                                  Nucleotide sequence cheF gene. The PstI DNA <br />                                          A <br />                                         Pst1          BgIll Oral          Dral Hindlil Pstl <br />                                             I           I                   I         momiA <br />  <br />  <br />  <br />                                                  4- <br />  <br />  <br />                                                                 4- <br />  <br />                                                                            .4 <br />                B                                                      Met Ser Asn Ile Ser <br />                                                                                                                        50 <br />                                   CTGCAGTTAACGTACTAAAAAGCATCAGCAGGGTG ATG TCA AAC ATT TCA <br />                                   PstI <br />                                                                                101 <br />                Thr Lys Gln His Leu Asp Ala Ala Asn Lys Phe Arg Glu Leu Leu Ser Thr <br />                ACA AAA CAG CAT CTG GAT GCG GCA AAT AAA TTC CGT GAA CTT TTA TCG ACA <br />                                                                                                                      152 <br />                Tyr Gln Asn Pro Gly ASp Leu Phe Asn Ile Gly Ala Tyr Lys Arg Gly Ser <br />                TAC CAA AAT CCG GGA GAT CTC TTT AAT ATC GGA GOT TAO AAA AGA GGA TCA <br />  <br />                                                                                                                      203 <br />                Ser Arg Glu Ile Asp Ala Ala Ile Gln Phe Tyr Pro Gln Leu Ile Gln Phe <br />                TCC AGA GAA ATT GAT GCA GCG ATA CAA TTT TAT CCG CAG CTT ATT CAA TTT <br />  <br />                                                                                                                      254 <br />                Leu Lys Gln Gly Thr Asp Glu Pro Ala Leu Leu Glu Glu Ser Ile Ala Ala <br />                TTA AAG CAG GGA ACA GAC GAA CCG GCT TTA TTG GAA GAA AGC ATT GCT GCA <br />                                                                                                                      305 <br />                Lys Thr Ser Leu Thr Gly Asn Glu Glu Glu Thr Trp Leu Asn Gln Phe Tyr <br />                AAA ACT AGT TTG ACA GGA AAT GAG GAA GAA ACG TGG CTT AAT CAA TTT TAC <br />                                                                                                                      356 <br />                Pro leu Ala Lys Ser Cys Trp Asn Tyr Lys Lys Met Lys Lys Lys Asn Pro <br />                CCA TTG GCC AAA AGC TGC TGG AAC TAC AAG AAA ATG AAA AAG AAA AAT CCT <br />                                                                                407 <br />                Tyr Pro Asn Ile Ser Asn Gln Phe Leu Asn Trp Lys Thr Leu Leu Leu Lys <br />                TAT CCG AAT ATC AGC AAT CAG TTT CTG AAT TGG AAA ACG TTG CTT TTA AAA <br />                                                                                                                      458 <br />                Leu Tyr Glu Asn Met Ser Lys Lys Glu Leu Ala Gly Thr Lys Ile Arg Lys <br />                TTA TAT GAG AAT ATG AGC AAA AAG GAA TTG GCT GGA ACC AAA ATA AGG AAA <br />                                                                                                                      509 <br />                Arg Asn Ser Lys Ala Ala Gly Val Tyr Lys Lys Val GLy Thr Thr Asn Asn <br />                AGA AAT TCA AAA GCG GCA GGA GTG TAC AAG AAA GTA GGC ACT ACC AAC AAT <br />                                                                                                                      560 <br />                Leu Ser Gln Ile Leu Ile Ile Arg Ser Ile Thr Ile Lys Ser Leu Ser Leu <br />                TTG TCT CAA ATC TTG ATA ATA CGA TCT ATC ACT ATC AAA AGC TTG TCA TTA <br />                                                                                                                      627 <br />                TGATGAAAAGAAATCAAATGAAATCAAAAAACAAGAAATTTTGACAGAGAAAAACATTGAAGTGAAA <br />                                                                                                             688 <br />                AAGTTTGAAAAAATGCGTGAAAAACAGTTTAAAATGTTTGCTCTTGAAGATAAAGCTGCAG <br />                                                                                                           PstI <br />   FIG. 4. Sequencing strategy 700-base-pair PstI fragment nucleotide sequence cheF gene. (A) Restriction sites used in <br /> the sequencing experiments. All fragments inserts (*) constructed M13 phages; phagemids used vectors for <br /> these fragments. (B) Nucleotide sequence. Nucleotides numbered taking position nucleotide PstI site +1. <br /> Three asterisks mark proposed translational stop codon. <br />  <br />                                                                    1635 <br />  1636     YING AND ORDAL                                                                                                J. BACTERIOL. <br />  <br /> fragment containing cheF gene sequenced (Fig. 4).          ribosome-binding site cheF gene lower than <br /> Computer analysis DNA sequence revealed open             average calculated free energy interaction 3&apos; end <br /> reading frame (ORF) 528 bases (Fig. 4B). There          16S RNA. Taken together, results imply only <br /> potential initiation codons 5&apos; end ORF. These       low levels cheF proteins required cells perform <br /> two codons frame reading frame                chemotaxis normally. When cheF product overex- <br /> identified cheF gene. A potential ribosome-binding         pressed presence high concentrations IPTG, <br /> site located nine-base region ATG codon        chemotaxis methanol production impaired. There <br /> position 36. Six seven bases (AGXAGGG) bases          20 chemotaxis proteins, cheF <br /> before ATG codon complementary 3&apos;-OH end            protein titrate causing inhibition. <br /> of 16S rRNA (8). On basis rules Tinoco et al.          The hydropathy profile cheF protein indicates that <br /> (27) Borer et al. (1), calculated AG interaction   C-terminal protein hydrophobic region. <br /> is -14.2 kcal (ca. -59.4 KJ)/mol. The GTG codon                 This region short membrane spanning, our <br /> position 33, however, preceded recognizable            results suggest cheF protein membrane associated <br /> ribosome-binding site. Therefore, ATG codon appears            normal conditions removed the <br /> more likely translational initiation codon. This ORF     membrane high-salt wash. The cheF protein a <br /> is capable encoding 175-amino-acid polypeptide         calculated pI 9.81, basic nature protein is <br /> calculated molecular weight 20,002, agreement       apparent amino acid sequence. The other <br /> molecular weight polypeptide expressed                proteins MCPs involved chemotaxis B. subtilis <br /> pGO102 maxicell experiments. To confirm            characterized methyltransferase and <br /> this ORF encodes functional cheF protein, mutations            methylesterase. These proteins methylate demethylate, <br /> were generated shifting reading frame pre-           respectively, MCPs vitro. Both proteins also <br /> sumed coding region. pGO102 digested BglII,               basic adhere membranes (2, 5). It remains be <br /> which located putative ORF. Klenow fragment          determined cheF protein proteins <br /> was used remove sticky ends. The      interact membrane. <br /> new hybrid plasmids introduced 011925. No              The deduced amino acid sequence cheF protein <br /> complementation plasmids observed.            compared sequences proteins in <br /> Furthermore, in-frame translational fusion cheF pro-        GenBank (E. Kofoid J. S. Parkinson, personal commu- <br /> tein P-galactosidase successfully constructed. West-        nication). No significant homology cheF protein <br /> ern blot (immunoblot) experiments confirmed fusion,        known proteins observed. cheF mutants are <br /> the fused protein purified (C. Ying G. Ordal,              impaired release methanol caused addition of <br /> manuscript preparation).                                        attractant removal repellent (20). There strong <br />    The deduced amino acid sequence cheF gene                evidence methyl groups released directly from <br /> product appeared include large group lysine residues,      B. subtilis MCPs instead transferred (as methyl or <br /> 21 lysines 175 amino acids; pl value cheF protein     methoxy groups) carrier(s) (26, 26a). There- <br /> of 9.81 calculated. The hydropathy profile de-          fore, tempting speculate actual substrate is <br /> duced cheF amino acid sequence (11) suggests cheF         methyl carriers. It clear cheF product <br /> protein composed quite different portions. Most       functions pathway unique B. subtilis, in <br /> its sequence characterized long stretches hydrophilic     E. coli. <br /> residues, carboxyl-terminal end contains short          Determination substrate cheF protein and <br /> hydrophobic region (159LSQILIIRSITIKSLSL175). Since             elucidation mechanism associates the <br /> was shown cheF protein appears membrane             membrane possible protein puri- <br /> associated, possible means interaction            fied. We able express protein a <br /> between cheF protein membrane: cheF                relatively high level control spac-I pro- <br /> protein contact membrane hydrophobic         moter, attempting raise antibodies the <br /> carboxyl terminus use lysines bind          cheF protein aid process analyzing role of <br /> anionic residues membrane proteins.                       protein molecular mechanism chemotaxis. <br />  <br />                          DISCUSSION                                                     ACKNOWLEDGMENTS <br />                                                                      We thank D. Henner gift pSI-1 J. Kaput valuable <br />   This paper reports nucleotide sequence gene       suggestions course investigation. We thank <br /> that required chemotaxis B. subtilis. The cheF gene      Aamir Zuberi help preparation manuscript. Comparison <br /> cloned expression plasmid complements cheF               deduced cheF protein proteins GenBank per- <br /> mutation chemotactic deficiency normal        formed Eric Kofoid laboratory J. S. Parkinson. <br /> methanol production response addition attrac-           This work supported Public Health Service grant Al 0336 <br /> tants. Moreover, chemotaxis methanol formation result-         National Institutes Health National Science Foun- <br /> ing stimulation attractants reach optimal         dation grant DCB 85-01604. <br /> levels cheF gene product induced 0.75                                LITERATURE CITED <br /> mM IPTG. These results evidence close relation- <br /> ship methanol production chemotaxis B.               1. Borer, P. N., B. Dengler, I. Tinoco, Jr., 0. C. Uhlenbeck. <br /> subtilis.                                                              1974. Stability ribonucleic acid double-stranded helices. J. <br />    Analysis nucleotide sequence cheF gene                Mol. Biol. 86:843-853. <br />                                                                     2. Burgess-Cassler, A., A. H. J. Ullah, G. W. Ordal. 1982. <br /> suggests gene capable encoding protein          Purification characterization Bacillus subtilis methyl- <br /> a molecular mass 20 kilodaltons, agreement              accepting chemotaxis protein methyltransferase 11. J. Biol. <br /> finding protein 20 kilodaltons B. subtilis maxicells.       Chem. 257:8412-8417. <br /> The cheF gene probably expressed low levels vivo,       3. Dubnau, D., R. Davidoff-Abelson. 1971. Fate transform- <br /> indicated codon usage gene (7). The potential          ing DNA following uptake competent Bacillus subtilis. 1. <br />  VOL. 171, 1989                                                    SEQUENCE AND EXPRESSION OF cheF IN B. SUBTILIS                         1637 <br />  <br />       Formation properties integrable plasmid Bacillus         cloning chemotaxis genes. J. Bacteriol. 154:1088-1097. <br />       subtilis. J. Mol. Biol. 56:209-221.                                20. Ordal, G. W., H. M. Parker, J. R. Kirby. 1985. Comple- <br /> 4.    Ferrari, F. A. <br /> </body></html>