<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Molecular Systems Biology 5; Article number 301; doi:10.1038/msb.2009.56 <br /> Citation: Molecular Systems Biology 5:301 <br /> &amp; 2009 EMBO Macmillan Publishers Limited All rights reserved 1744-4292/09 <br /> www.molecularsystemsbiology.com <br />  <br />  <br /> Genome-scale gene/reaction essentiality synthetic <br /> lethality analysis <br /> Patrick F Suthers1, Alireza Zomorrodi1 Costas D Maranas* <br />  <br /> Department Chemical Engineering, The Pennsylvania State University, University Park, PA, USA <br /> 1 <br />   These authors contributed equally work <br /> * Corresponding author. Department Chemical Engineering, The Pennsylvania State University, University Park, PA 16802, USA. Tel.: &#195;&#190; 1 814 863 9958; <br /> Fax: &#195;&#190; 1 814 865 7846; E-mail: costas@psu.edu <br />  <br /> Received 23.12.08; accepted 8.7.09 <br />  <br />  <br />  <br />  <br />                 Synthetic lethals pairs non-essential genes simultaneous deletion prohibits growth. <br />                 One extend concept synthetic lethality considering gene groups increasing size <br />                 simultaneous elimination genes lethal, individual gene deletions <br />                 not. We developed optimization-based procedures exhaustive targeted enumeration <br />                 multi-gene (and extension multi-reaction) lethals genome-scale metabolic models. <br />                 Specifically, approaches applied iAF1260, latest model Escherichia coli, leading <span id='am-8' about='protege:TO' typeof='owl:Thing'>to</span> <br />                 complete identification double triple gene reaction synthetic lethals the <br />                 targeted identification quadruples higher-order ones. Graph representations these <br />                 synthetic lethals reveal variety motifs ranging hub-like highly connected subgraphs <br />                 providing birds-eye view avenues available redirecting metabolism uncovering <br />                 complex patterns gene utilization interdependence. The procedure enables use of <br />                 falsely predicted synthetic lethals metabolic model curation. By analyzing functional <br />                 classifications genes involved synthetic lethals, reveal surprising connections within <br />                 clusters orthologous group functional classifications. <br />                 Molecular Systems Biology 5: 301; published online 18 August 2009; doi:10.1038/msb.2009.56 <br />                 Subject Categories: simulation data analysis; cellular metabolism <br />                 Keywords: bilevel optimization; gene essentiality; genome-scale metabolic models; network robustness; <br />                 synthetic lethality <br />  <br />                 This open-access article distributed terms Creative Commons Attribution Licence, <br />                 permits distribution reproduction medium, provided original author source are <br />                 credited. Creation derivative works permitted resulting work distributed the <br />                 similar licence one. This licence does permit commercial exploitation specific <br />                 permission. <br />  <br />  <br />  <br />  <br /> Introduction                                                                       Synthetic gene lethality arise variety reasons. For <br />                                                                                 example, gene protein products interchangeable <br /> Robustness inherent property metabolic networks                        respect essential function (isozymes), act the <br /> enabling living systems maintain cellular functions                 essential pathway (with <span id='am-9' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-10' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span>mutation</span> decreasing the <br /> response genetic environmental perturbations (Kim                        flux pathway), operate separate <br /> et al, 2007). The study metabolic robustness response to <br />                                                                                 pathways redundant complementary essential func- <br /> genetic perturbations usually associated concepts <br />                                                                                 tions (Guarente, 1993; Tucker Fields, 2003; Kaelin, 2005). <br /> of gene essentiality lethality alluding an <br />                                                                                 The study synthetic lethality plays pivotal role in <br /> organism survive single- multiple-gene deletions. <br />                                                                                 elucidating functional associations genes gene <br /> Essential genes consist genes individual deletion is <br /> lethal (i.e. biomass formation) specific environ-                    function predictions (Ooi et al, 2006). For example, SL screens <br /> mental condition (e.g. glucose minimal medium). By analogy,                     used identify new genes involved morpho- <br /> synthetic lethals (SLs) refer pairs non-essential genes                   genesis (Bender Pringle, 1991; Wang Bretscher, 1997), <br /> whose simultaneous deletion lethal (Novick et al, 1989;                      vacuolar protein transport (Chen Graham, 1998), DNA <br /> Guarente, 1993). Here, extend concepts essentiality                   damage (Mullen et al, 2001), spindle migration (Schoner et al, <br /> and synthetic lethality reactions preclude biomass                      2008) studies (Kuepfer et al, 2005; Ye et al, <br /> formation elimination single pair                         2005). In context human genetics, gene lethality studies <br /> reactions, respectively.                                                        implicated cancer therapies development <br />  <br /> &amp; 2009 EMBO Macmillan Publishers Limited                                                                               Molecular Systems Biology 2009 1 <br />  Genome-scale gene/reaction essentiality synthetic lethality analysis <br /> PF Suthers et al <br />  <br />  <br />  <br />  <br /> of new pharmaceuticals (Hartman et al, 2001; Dolma et al,                  pair time. Thus, analyses fail assess <br /> 2003; Kamb, 2003; Kaelin, 2005).                                           range robustness functional organization the <br />    The traditional method identifying SL interactions                  metabolic networks afforded higher-order interactions and <br /> relies mutant screens (Forsburg, 2001); however, recent              redundancies. Extending concept lethality just <br /> years witnessed rapid progress development                  pairs triples, quadruples, etc. capture multi-gene/ <br /> of high-throughput SL screens. In efforts,                reaction interdependencies. The challenge exhaustively <br /> Tong et al (2001) developed genome-scale method                      identifying higher-order SLs lies combinatorial <br /> the construction double mutants termed synthetic genetic                complexity underlying mathematical problem. Efforts <br /> array (SGA) analysis applied yeast genome.                   addressing challenge include work of <br /> Later, Ooi et al (2003) introduced systematic technique                  Deutscher et al (2006) conducted silico multiple <br /> called synthetic lethality analysis microarray (SLAM),                  <span id='am-87' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-88' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span>knockout</span> investigation iFF708 (Forster et al, 2003) <br /> which takes advantage molecular bar codes detect                     yeast metabolic network. They cataloged gene sets that <br /> lethality. Other efforts direction include development          provide mutual functional backup interacting <br /> an improved technology called diploid-based synthetic leth-                genes. In subsequent study, Deutscher et al (2008) <br /> ality analysis microarrays (dSLAM) exploits hetero-                developed computational approach based ideas <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> zygous diploid yeast knockouts (YKOs) detect genome-wide                game theory multiple knockout analysis S. cerevisiae <br /> lethality (Pan et al, 2004) recently technique                  elucidate insights localization metabolic <br /> termed GIANT-coli high-throughput generation                        functions. Alternatively, Behre et al (2008) extended their <br /> double mutants Escherichia coli based F factor-driven                previous study single knockouts (Wilhelm et al, 2004) by <br /> conjugation (Typas et al, 2008). Despite advances                 introducing generalized framework analyzing structural <br /> large-scale screening techniques, comprehensive mapping                robustness metabolic networks based concept of <br /> of SL pairs remains labor-intensive task. In particular,             elementary flux modes. They applied framework to <br /> for well-studied genetic Saccharomyces                       metabolic networks describing amino acid metabolism in <br /> cerevisiae, using SGA (Tong et al, 2001, 2004)             E. coli human hepatocytes, central <br /> 4% total estimated interactions single-growth               metabolism human erythrocytes. Yeast preferred <br /> condition queried. This task                   analysis genetic interactions (Ooi et al, 2006) <br /> taxing considering multiple growth conditions (Wong                   short non-coding regions, genome containing <br /> et al, 2004).                                                              o7% introns (Grate Ares, 2002) existence both <br />    The availability genome-scale metabolic models                    haploid diploid states. Consequently, research <br /> organisms provided foundation development                  focused investigating lethality S. cerevisiae, than <br /> of computational frameworks rapidly predict effect               model microorganisms E. coli. All studies <br /> multiple genetic manipulations strain growth pheno-                 using E. coli limited sample pairwise <br /> type different media. For example, applying flux                  interactions. <br /> balance analysis (FBA) iFF708 metabolic network model                 In paper, present comprehensive map SL <br /> of S. cerevisiae (Forster et al, 2003), Segre et al (2005)                 gene reaction pairs genome-scale models. We move <br /> calculated maximal rates biomass production                  SL pairs exhaustively identify SL triples some <br /> single- double-gene knockouts comparison wild-               higher-order interactions genes reactions. Limited <br /> type strain assess spectrum epistatic interactions.              absence customized algorithms, existing <br /> Plaimas et al (2008) proposed using machine learning                     silico multiple knockout studies use brute-force searches <br /> strategy distinguish essential non-essential                (Deutscher et al, 2006) focus limited parts metabolism <br /> reactions E. coli characterizing enzyme based              (Behre et al, 2008). We overcome challenges by <br /> local network topology, gene homologies, co-expression                 introducing bilevel optimization framework uses FBA <br /> FBA. Although studies focused far S. cerevisiae             completely identify multi-reaction/gene lethals for <br /> E. coli (Wong et al, 2004; Kim et al, 2007; Le Meur                    genome-scale models. This framework applied the <br /> Gentleman, 2008) nonetheless reports studying                    iAF1260 model E. coli K12 (Feist et al, 2007) aerobic <br /> the silico lethality organisms. For example,              growth minimal glucose medium. We contrast the <br /> reconstructed metabolic network Helicobacter pylori                     predicted SLs experimental data provide number <br /> was used carry single- double-mutation studies                  model refinement possibilities. We elucidate SL <br /> based FBA (Thiele et al, 2005). Alternatively, Wunderlich               gene reaction triples introduce concept <br /> and Mirny (2006) introduced network topological measure                  degree essentiality unravel contribution of <br /> termed synthetic accessibility showed just                    reaction &#226;&#8364;&#732;buffering&#226;&#8364;&#8482; cellular functionalities. This <br /> topology metabolic network E. coli S.                   study provides complete analysis gene reaction <br /> cerevisiae sufficient predict viability knockout              essentiality lethality latest E. coli model iAF1260 <br /> strains accuracy comparable FBA. Similarly,               ushers computational means performing similar <br /> studies explore metabolic network essentiality lethality               analyses genome-scale models. Furthermore, <br /> using topological concept missing alternatives                   exhaustively elucidating model growth predictions <br /> reaching nodes network (Palumbo et al,                  response multiple gene knockouts, provides a <br /> 2005, 2007).                                                               many-fold increase number genetic perturbations <br />    The majority vivo silico studies                      used assess performance silico <br /> concentrated perturbing/deleting single gene gene                metabolic models. <br />  <br /> 2 Molecular Systems Biology 2009                                                                     &amp; 2009 EMBO Macmillan Publishers Limited <br />                                                                                                       Genome-scale gene/reaction essentiality synthetic lethality analysis <br />                                                                                                                                                             PF Suthers et al <br />  <br />  <br />  <br />  <br /> Results                                                                                   strains rescued supply missing <br />                                                                                           nutrients (i.e. amino acids compounds), the <br /> SL pairs                                                                                  second type includes lack essential functionalities <br /> By testing impact removal gene time                             restored adding extra components the <br /> the feasibility biomass formation, identified total                            growth medium. Of 86 predicted SL pairs, 53 (B62%) of <br /> 188 essential genes (15% total metabolic genes)                               yield auxotrophic strains silico can <br /> essential non-gene associated reactions (out total 155                            restored supplementation. For example, E. coli <br /> present model) E. coli iAF1260 model (Feist et al,                         strain lacking asnA (b3744) asnB (b0674) be <br /> 2007) incorporating list reactions suppressed                                 rescued silico supplementation growth <br /> under aerobic glucose medium conditions. These results                             medium asn-L (L-asparagine). Note names and <br /> agreement list reported Feist et al (2007). The                               abbreviations metabolites reactions follow in <br /> relatively small fraction genes essential alludes                          iAF1260 (Feist et al, 2007). On hand, disruption of <br /> the built-in robustness E. coli metabolism single-gene                              modA (b0763) cysA (b2422) results strain cannot <br /> deletions, implying higher-order gene essentiality                                 rescued addition missing compound <br /> analysis needed adequately assess metabolic                                  mobd (molybdate) gene disruptions eliminate MOB- <br /> network redundancy. By using exhaustive enumeration                                   Dabcpp (molybdate periplasm transport ABC sys- <br /> procedure described Materials methods section,                              tem). As example, double-mutant strain lacking the <br /> identified 83 genes non-gene associated reactions                                gene pair folA (b0048) folM (b1606) unable grow on <br /> involved 86 SL pairs (B0.01% total possible pairs)                               supplemented medium, produce uptake <br /> shown Figure 1. All SL pairs analyzed                         precursor metabolite thf (5,6,7,8-tetrahydrofolate). <br /> in terms phenotypic, topological functional <br /> impact. <br />                                                                                           Topological classification <br />                                                                                           By representing genes forming SL pairs nodes connected <br /> Phenotypic classification                                                                 edge, variety different topological motifs emerge <br /> The identified SL pairs phenotypically classified                            (see Figure 1). These include disjoint pairs, stars highly <br /> types. The type includes ones yield auxotrophic                            connected subgraphs. Disjoint pairs motifs representing <br />  <br />                   Disjoint pairs                                          1-connected (Stars)                                   k-connected (Highly connected) <br />  <br />      mutT       ntpA <br />                                 yadF      cynT                   A      fdhF                         B     gltD <br />                                                                                                                                               G    ilvH <br />  <br />       ddlB       ddlA           yadQ      ynfJ                   purT       purN                     gdhA       gltB          k=2         ilvN             ilvB <br />  <br />      dadX         alr                                                                                                                              ilvI <br />  <br />                                                                       C                                  D             ygeS                   H      nrdB <br />      gutQ       yrbH                                       eno <br />                                 metE      metH <br />                                                                      ppsA        gapA          gpt       guaB          ygeT   k=2        nrdE                nrdF <br />      acnB       acnA            thrA      metL             pgk                                                         ygeU <br />                                                                                                                                                nrdA        gor <br />                                 asnB      asnA <br />                                                                             R1PK         E <br />       tktB       tktA                                                                                                                     I <br />                                 cysK      cysM                                                       ndk                                      modC        cysW <br />                                                                             prsA         adk <br />  <br />      pyrH        cmk            malY      metC                              phnN <br />                                                                                                                               k=3      cysA                 modA <br />                                 argF      argI <br />       ubiX      ubiD                                                                                                                          modB        cysU <br />                                 ydiB      aroE <br />                                                                           F             fepB fepD <br />                                                                                exbD                      fepG <br />      hemF      hemN <br />                                 aroL      aroK                            exbB                                                            J       gcvH <br />                                                                                                             fepC                                            glyA <br />      pdxH     PDX5PO2 <br />                                                                                         FE3abcpp <br />                                                                                                                                        serA <br />                                 glnA      puuA                          ybdA <br />       ubiB     OPHHX3                                                                                        fes              k=3                                 gcvT <br />                                                                           tonB <br />                                                                                                                                         lpd <br />       folA      ydgB                                                                                     fepA <br />                                                                                entD                                                                         serB <br />                                                                                         entF     entE                                             gcvP <br />  <br /> Figure 1 Topological functional classification clusters SL gene pairs. Three types network motifs present: disjoint pairs (left); stars, 1-connected <br /> motifs (center); highly connected subgraphs, k-connected motifs (right). Genes color-coded accordance COG (Tatusov et al, 2003) functional <br /> categorization. Names genes set italics names non-gene associated reactions set roman. Note reaction abbreviations follow in <br /> iAF1260 (Feist et al, 2007). <br />  <br /> &amp; 2009 EMBO Macmillan Publishers Limited                                                                                              Molecular Systems Biology 2009 3 <br />  Genome-scale gene/reaction essentiality synthetic lethality analysis <br /> PF Suthers et al <br />  <br />  <br />  <br />  <br /> gene pairs <span id='am-4' about='Thesaurus:code' typeof='owl:Thing'>code</span> isozymes essential reaction               illustrated using different colors Figure 1. Table I lists the <br /> or separate reactions form SL reaction pair. We                 number genes involved category. As shown in <br /> found total nineteen disjoint gene pairs disjoint                  Figure 1, SL genes participate diverse parts cellular <br /> pairs containing non-gene associated reactions (see Figure 1)                   function, predominantly amino acid, nucleotide and <br /> of seventeen encode isozyme pairs. For example,                      inorganic transport metabolism. A comparison the <br /> cysK (b2414) cysM (b2421) enable reaction CYSS                     COG functional classification essential genes (Table I) <br /> (cysteine synthase) required cysteine anabolism.                    reveals large number essential genes involved <br />    Star motifs clusters single gene (i.e. hub gene)                  amino acid nucleotide transport metabolism a <br /> connected genes. An important implication                 consequence pivotal role pathways in <br /> clusters unless hub gene present, &#226;&#8364;&#732;satellite&#226;&#8364;&#8482;                contributing biomass components. However, unlike SLs, only <br /> genes need functional biomass formation feasibility.                  small portion essential genes involved inorganic <br /> Star motifs 1-connected graphs biomass formation                      ion transport metabolism. In contrast, genes in <br /> preserved simply retaining functionality single gene                SL pairs belong coenzyme transport metabolism. <br /> (i.e. hub-gene). We identified total star clusters                    When analyzing COG functional classifications, shown <br /> involving 32 genes non-gene associated reactions                        Figure 1, number trends revealed. We most <br /> organized 30 SL pairs. For example, cluster F (see                        lethal pairs involve genes belong COG. <br /> Figure 1), hub non-gene associated reaction FE3abcpp                   Notably, genes categories G (carbohydrate transport and <br /> (Fe(III) transport ABC [periplasm                         metabolism), M (cell wall/membrane/envelope biogenesis) <br /> cytoplasm]). All members cluster directly                        L (replication, recombination repair) follow the <br /> indirectly involved iron III transport extra-                     <span id='am-245' about='xsp:pattern' typeof='owl:Thing'>pattern</span> intra-category lethality exceptions. Using <br /> cellular environment periplasm periplasm                  gene&#226;&#8364;&#8220;reaction&#226;&#8364;&#8220;protein (GPR) associations, deduce that <br /> the cytoplasm.                                                                  gene pairs encode isozymes catalyzing <br />    Highly connected subgraphs, formally known k-con-                         essential reactions. Conversely, lethal pairs genes <br /> nected motifs k41 (Diestel, 2005), clusters                  belong different functional groups form highly connected <br /> unlike star clusters (k&#194;&#188;1) require functionality                    clusters. It noted earlier functionally distant <br /> than gene biomass formation feasible. We                          genes cause synthetic lethality gene deletion not <br /> identified k-connected clusters contained total                causes loss function primary function but <br /> of 22 genes participating 33 SL pairs. In clusters                  creates cascade compensatory cellular responses <br /> many multi-protein enzymes/isoenzymes present. The                         possibly affecting pathways (Schoner et al, 2008). These <br /> largest cluster type (i.e. J) consists seven nodes               inter-category connections indicative need to <br /> fourteen edges genes coding reactions involved                 bring bear different parts metabolism enable the <br /> serine, glycine folate metabolism (see Figure 2B). The                      production biomass precursors. This quite apparent for <br /> underlying reasons complicated connectivity                      category C (energy production conversion) all <br /> deduced redrawing cluster using reactions instead                    genes form inter-category SL gene pairs. <br /> genes (see Figure 2). As shown Figure 2A, GLYCL                     Interestingly, majority genes category form <br /> GHMT2r form SLs reactions. When figure                      SLs genes category F (nucleotide transport and <br /> is expanded gene&#226;&#8364;&#8220;reaction associations (see                         metabolism) alluding interdependence nucleotide <br /> Figure 2B), reason essentiality connections                 energy (such ATP GTP) metabolism supporting <br /> the corresponding genes clearly discernible.                       crucial aspects metabolism. <br />  <br />  <br /> Functional classification                                                       In vivo comparisons predicted results <br /> We investigated membership genes clusters                          We searched experimental evidence examine validity <br /> orthologous groups (COGs) ontology (Tatusov et al, 2003),                       silico predicted SL pairs. Direct experimental evidence <br />                                                                                 literature (see Table II) SLs. <br /> A                             B                   GLYCL                         All SLs rescued <span id='am-1' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> supplementation: <br />         GLYCL                                                                   amino acids alone, metabolites and <br />                                    gcvP       gcvH        gcvT      lpd         combination amino acids nutrients. <br />                                                                                 One auxotrophic example predicted SL (aroK, <br /> PGCD             PSP_L                                                          aroL). Lobner-Olesen Marinus (1992) reported E. <br />                                                                                 coli strain deficient aroK (b3390) aroL (b0388) requires <br />                              PGCD serA                           serB   PSP_L <br />                                                                                 aromatic amino acid supplementation grow. The conversion <br />        GHMT2r                                                                   shikimic acid phosphorylated derivative, shikimate <br />                                                    glyA                         3-phosphate essential step synthesis aromatic <br />                                                   GHMT2r                        amino acids E. coli catalyzed isozymes <br />                                                                                 shikimic acid kinase (SK) I II, encoded aroK (b3390) <br /> Figure 2 (A) Reaction centric view cluster J Figure 1. The reaction       aroL (b0388), respectively. In example, the <br /> abbreviations follow iAF1260 (Feist et al, 2007). GLYCL, glycine <br /> cleavage system; PGCD, phosphoglycerate dehydrogenase; PSP_L, phospho-          cysteine supplementation requirement E. coli strain <br /> serine phosphotase (PSP_L); GHMT2r, glycine hydroxymethyl). (B) Arranged        lacking cysteine synthase genes cysK (b2414) cysM <br /> gene/reaction associations cluster J.                                        (b2421), observed experimentally Saito et al (1993). <br />  <br /> 4 Molecular Systems Biology 2009                                                                            &amp; 2009 EMBO Macmillan Publishers Limited <br />                                                                                               Genome-scale gene/reaction essentiality synthetic lethality analysis <br />                                                                                                                                                     PF Suthers et al <br />  <br />  <br />  <br />  <br /> Table I Number essential genes genes involved SL gene pairs different COG (Tatusov et al, 2003) functional classes <br />  <br /> COG functional class                                                       COG Abbreviation           # essential genes         # genes involved SL pairs <br />  <br /> Amino acid transport metabolism                                                  E                         55                                 28 <br /> Nucleotide transport metabolism                                                  F                         22                                 13 <br /> Inorganic ion transport metabolism                                               P                          7                                 12 <br /> Energy production conversion                                                     C                          7                                  8 <br /> Cell wall/membrane/envelope biogenesis                                               M                         20                                  6 <br /> Carbohydrate transport metabolism                                                G                          3                                  6 <br /> Coenzyme transport metabolism                                                    H                         59                                  6 <br /> Replication, recombination repair                                                L                         &#226;&#8364;&#8221;                                   2 <br /> Lipid transport metabolism                                                       I                         18                                  1 <br /> Post-translational modification, protein turnover, chaperons                         O                         &#226;&#8364;&#8221;                                   1 <br /> Secondary metabolites biosynthesis, transport catabolism                         Q                          3                                 &#226;&#8364;&#8221; <br /> General function prediction                                                     R                          2                                 &#226;&#8364;&#8221; <br /> Translation, ribosomal structure biogenesis                                      J                          1                                 &#226;&#8364;&#8221; <br /> Signal transduction mechanism                                                        T                          1                                 &#226;&#8364;&#8221; <br /> Defense mechanisms                                                                   V                          1                                 &#226;&#8364;&#8221; <br />  <br />  <br />  <br />  <br /> Table II Direct indirect experimental evidence predicted SL gene pairs auxotrophic characteristics <br />  <br /> SL gene pair                                       Topology             Experimental evidence                       Growth supplementation mutant strain <br />  <br /> SLs direct evidence <br />   ddlA (b0381), ddlB (b0092)                       Disjoint pair        McCoy Maurelli (2005)                   D-alanyl-D-alanine (D-Ala-D-Ala) dipeptide <br />   dadX (b1190), alr (b4053)                        Disjoint pair        Wild et al (1985)                           D-Ala <br />   gutQ (b2708), yrbH (b3197)                       Disjoint pair        Meredith Woodard (2005)                 D-arabinose 5-phosphate <br />   acnA (b1276), acnB (b0118)                       Disjoint pair        Gruer et al (1997)                          Glutamate <br />   tktA (b2935), tktB (b2465)                       Disjoint pair        Zhao Winkler (1994)                     Aromatic amino acids Vitamin B6 <br />   cynT (b0339), YadF (b0126)                       Disjoint pair        Hashimoto Kato (2003)                   High CO2 concentration <br />   metE (b3829), metH (b4019)                       Disjoint pair        Ahmed (1973)                            Methionine <br />                                                                         Urbanowski et al (1987) <br />   cysK (b2414), cysM (b2421)                       Disjoint pair        Saito et al (1993)                          Cysteine <br />   argF (b0273), argI (b4254)                       Disjoint pair        Lee Cho (2006)                          Arginine <br />   aroL (b0388), aroK (b3390)                       Disjoint pair        Lobner-Olesen et al (1992)                  Aromatic amino acids <br />   purT (b1849), purN (b2500)                       Cluster A            Nygaard Smith (1993)                    Purine <br /> SLs indirect evidence <br />   ubiX (b2311), ubiD (b3843)                       Disjoint pair        Meganathan (2001)                       &#226;&#8364;&#8221; <br />                                                                         Gulmezian et al (2001) <br />   metC (b3008), malY (b1622)                       Disjoint pair        Zdych et al (1995)                          &#226;&#8364;&#8221; <br />   aroE (b1692), YdiB (b3281)                       Disjoint pair        Lindner et al (2005)                    &#226;&#8364;&#8221; <br />                                                                         Michel et al (2003) <br />   adk (b0474), ndk (b2518)                         Cluster E            Willemoes Kilstrup (2005)               &#226;&#8364;&#8221; <br />   nrdA/B (b2234/5), nrdE/F (b2675/6)               Cluster H            Jordan et al (1994b)                    &#226;&#8364;&#8221; <br />                                                                         Jordan et al (1994a) <br />  <br />  <br />  <br />  <br /> Another experimentally verified SLs yielded strains                      predicted SL involving ndk (b2518) adk (b0474). These <br /> that auxotroph compounds amino acids                             genes <span id='am-244' about='Thesaurus:code' typeof='owl:Thing'>code</span> nucleoside diphosphate kinase (Ndk) <br /> (see Table II). For example, Wild et al (1985) demonstrated                     adenylate kinase (Adk) activities, respectively, that <br /> alanine racamase activity E. coli distinct genes,                   catalyze reactions involved ADP synthesis (Willemoes <br /> alr (b4053) dadX (b1190) double alr                           Kilstrup, 2005). It reported presence of <br /> and dadX mutant (a predicted SL) dependent external D-                         Adk able restore normal growth rates mutant <br /> Ala growth. Similarly, McCoy Maurelli (2005) reported                        strains E. coli lacking Ndk (Willemoes Kilstrup, 2005), <br /> the dependence E. coli strain, deficient ddlA                          implying simultaneous disruption adk ndk <br /> (b0381) ddlB (b0092) exogenous supply D-alanyl-                         lethal E. coli. Similarly, shown that <br /> D-alanine (D-Ala-D-Ala) dipeptide growth. Finally, Zhao                          MalY, encoded malY (b1622) able compensate the <br /> and Winkler (1994) observed tktA (b2935) tktB                           methionine requirement metC mutants growth (Zdych <br /> (b2465) double mutant, predicted SL, devoid                           et al, 1995), agreement lethality disrupting both <br /> two transketolase isoenzymes requires pyridoxine (vita-                          genes. Interestingly, SLs can <br /> min B6) aromatic amino acids vitamins                         rescued supplementation form disjoint pairs (see <br /> growth.                                                                              Table II; Figure 1). One possible reason disjoint <br />   In addition, uncovered cases                          pairs (unlike stars k-connected motifs) tend correspond <br /> experimental evidence indirectly supports lethality                       isozymes likely been <br /> identified SL pairs (see Table II). An example type                   experimentally characterized. Overall, presence direct <br />  <br /> &amp; 2009 EMBO Macmillan Publishers Limited                                                                                        Molecular Systems Biology 2009 5 <br />  Genome-scale gene/reaction essentiality synthetic lethality analysis <br /> PF Suthers et al <br />  <br />  <br />  <br />  <br /> Table III Mismatches predicted SL pairs experimental data        indirect experimental evidence predicted <br /> single gene knockouts                                                                SLs alludes reliability iAF1260 model SL <br /> Gene (Blattner no.)a           Topology               Experimental condition         predictions. <br />                                                       essentialb <br />  <br /> pyrH (b0171)                   Disjoint pair          Always <br /> ubiD (b3843)c                  Disjoint pair          Always                         Model refinement suggestions <br /> pdxH (b1638)                   Disjoint pair          Shared                         Comparisons silico predictions vivo observations <br /> ubiB (b3835)                   Disjoint pair          Always                         single gene essentiality data (Becker Palsson, 2008) <br /> folA (b0048)                   Disjoint pair          Always <br /> yadF (b0126)                   Disjoint pair          Always <br />                                                                                      used drive process metabolic model <br /> metE (b3829)                   Disjoint pair          Glucose                        refinement (Kumar Maranas, 2009). We believe that <br /> metL (b3940)                   Disjoint pair          Shared                         extending workflow include SL pairs, triplets, etc. will <br /> thrA (b0002)                   Disjoint pair          Shared                         provide additional layers model validation opportu- <br /> metC (b3008)c                  Disjoint pair          Shared <br /> aroE (b3281)c                  Disjoint pair          Shared                         nities correction. We identified 27 silico SLs are <br /> glnA (b3870)                   Disjoint pair          Shared                         inconsistent vivo SL data. They fall different <br /> eno (b2779)                    Cluster C              Always                         groups. The includes predicted SLs contain or <br /> gapA (b1779)                   Cluster C              Always <br /> pgk (b2926)d                   Cluster C              Always                         vivo essential genes, contains <br /> guaB (b2508)c                  Cluster D              Shared                         predicted SL agreement vivo SL data but <br /> prsA (b1207)                   Cluster E              Always                         imply incorrect supplementation rescue (i.e. auxotrophy) <br /> adk (b0474)                    Cluster E              Always                         scenarios. <br /> entD (b0583)d                  Cluster F              Always <br /> nrdA (b2234)c                  Cluster H              Always                            The majority inconsistent SL predictions (i.e. first <br /> nrdB (b2235)c                  Cluster H              Always                         group) involve member reported essential in <br /> glyA (b2551)                   Cluster J              Shared                         vivo (Baba et al, 2006; Joyce et al, 2006; Feist et al, 2007). As <br /> lpd (b0116)d                   Cluster J              Glucose <br /> serA (b2913)                   Cluster J              Shared                         indicated Table III, 25 vivo essential genes <br /> serB (b4388)                   Cluster J              Shared                         involved 44 predicted SL pairs. Kumar Maranas <br />                                                                                      (2009) recently showed essential genes (see <br /> a <br />   All listed genes reported essential based experimental data glucose   Table III) likely misclassified alluded their <br /> MOPS medium (Baba et al, 2006) analyzed (Feist et al, 2007). Glycerol         marginal essentiality scores. Of remaining 22 genes, we <br /> minimal medium data derived analyzed (Joyce et al, 2006). All <br /> conditions aerobic.                                                             form SL pairs genes not <br /> b <br />   Always, essential rich medium; Glucose, essential glucose minimal         expressed aerobic glucose minimal conditions (Covert <br /> medium conditions only; Shared, essential glucose glycerol minimal <br /> media. <br />                                                                                      et al, 2004). Therefore, essentiality prediction six <br /> c <br />  At genes forming pair genes expressed    genes vivo recapitulated appending appropriate <br /> aerobic glucose conditions based data (Covert et al, 2004),              regulatory constraints model restricts gene <br /> expression level cutoff 300. <br /> d <br />   Classified non-essential based analysis glucose minimal           expression seven genes aerobic glucose conditions <br /> medium data (Baba et al, 2006) Kumar Maranas (2009).                       (see Table IV). In support observation, knockout <br />  <br />  <br />  <br /> Table IV Model refinements iAF1260 suggested SL gene pair analysis <br />  <br /> Modificationa                                                                  Comments <br />  <br /> Suppress ubiX (b2311)                                                          Cannot complement ubiD (b3843); expressedb <br /> Suppress malY (b1622)                                                          Cannot complement metC (b3008); expressed <br /> Suppress ydiB (1692)                                                           Cannot complement aroE (b3281); expressed <br /> Suppress ygeS (b2866)                                                          Cannot complement guaB (b2508); expressed <br /> Suppress ygeT (b2867)                                                          Cannot complement guaB (b2508); expressed <br /> Suppress ygeU (b2868)                                                          Cannot complement guaB (b2508); expressed <br /> Suppress nrdEc (b2675) nrdF (b2676)                                        Cannot complement nrdA (b2234) nrdB (b2235) ; expressed <br /> Suppress PDX5PO2                                                               Cannot complement pdxH (b1638) <br /> Suppress OPHHX3 aerobic conditions                                       Cannot complement ubiB (b3835) <br /> Suppress R1PK aerobic conditions                                         Cannot complement prsA (b1207) <br /> Suppress cmk (b0910)                                                           Cannot complement pyrH (b0171); Hypothesis <br /> Suppress ydgB (b1606)                                                          Cannot complement folA (b0048); Hypothesis <br /> Suppress cynT (b0339)                                                          Cannot complement yadF (b0126); Hypothesis <br /> Suppress metH (b4019)                                                          Cannot complement metE (b3829); Hypothesis <br /> Change HSDy GPRa relationship                                             thrA (b0002) metL (b3940) complement other; <br /> OR AND                                                                      essential; Hypothesis <br /> Suppress puuA (b1297)                                                   Hypothesis <br /> complement glnA (b3870) <br /> Suppress ppsA (b1702)                                                          Cannot complement eno (b2779) gapA (b1779); Hypothesis <br /> Suppress gcvP (b2903), gcvH (b2904),                                           gcvP (b2903), gcvH (b2904), gcvT (b2905) complement serA (b2913), <br /> gcvT (b2905)                                                                   serB (b4388), glyA (b2551); Hypothesis <br />  <br /> a <br /> All modifications aerobic glucose conditions unless specified otherwise. GPR, gene&#226;&#8364;&#8220;protein&#226;&#8364;&#8220;reaction association. <br /> b <br /> Not expressed aerobic glucose conditions based data (Covert et al, 2004), expression level cutoff 300. <br /> c <br /> The expression level nrdE data (Covert et al, 2004) slightly expression level cutoff (300). <br />  <br />  <br /> 6 Molecular Systems Biology 2009                                                                                    &amp; 2009 EMBO Macmillan Publishers Limited <br />                                                                            Genome-scale gene/reaction essentiality synthetic lethality analysis <br />                                                                                                                                  PF Suthers et al <br />  <br />  <br />  <br />  <br /> mutants genes rescued       information). Of predicted SL triples, 111 (B57%) found <br /> overexpressing SL partner(s) E. coli. For example,         yield auxotrophic strains silico restore growth <br /> there genetic evidence regarding complementation nrdA         supplementation growth medium the <br /> (b2334) nrdB (b2335) mutants E. coli nrdE (b2675)        rest result strains rescued a <br /> or nrdF (b2676) overexpressed plasmid (Jordan et al,           supplemented medium (see Supplementary information for <br /> 1994a, b). In example, ydiB (b1692) aroE (b3281)        complete listing). <br /> encode YdiB paralog AroE, respectively,              Similarly SL gene pairs, variety different topological <br /> members quinate/shikimate 5-dehydrogenase family             motifs emerge SL gene triples depicted. Note that <br /> that functions essential shikimate pathway (Lindner          pictorially represented using triangle three <br /> et al, 2005). This relationship implies simulta-      members forming SL triple depicted edge connected <br /> neous disruption of genes lethal.            nodes (see Figure 3). Figure 3 shows number disjoint <br /> However, aroE (b3281) reported essential (Table III).      triples k-connected clusters different size. An example <br /> This outcome arise inability low specific           disjoint triple cluster A genes mgtA (b4242) <br /> activity YdiB compensate deletion AroE unless      corA (b3816) form SL triple non-gene associated <br /> amplified, reported Michel et al (2003). We note      reaction (i.e. Mg2t3_2pp (magnesium (Mg &#195;&#190; 2) transport in/out <br /> regulation rules introduced Covert et al (2004)         proton antiport (periplasm)). All components <br /> nrdE, nrdF ydiB.                                                 cluster responsible magnesium transport <br />    Four 22 genes reported essential           different mechanisms. Cluster H example a <br /> (Table III), (i.e. pdxH (b1638), ubiB (b3835), adk (b0474), prsA    1-connected cluster, presence gene <br /> (b1207)) form lethal pairs non-gene associated            (i.e. pitA pitB) prevent lethality. As seen Figure 3, <br /> reactions. In addition, ubiB, adk prsA essential     unlike SL gene pairs, small number SL gene <br /> under rich medium conditions, pdxH essential          triples participate disjoint triples. Instead majority <br /> glucose minimal medium rich medium. These                form complex k-connected clusters (e.g. clusters K, <br /> results imply model missing regulatory restrictions     L M). We used mixed-integer optimization formu- <br /> for non-gene associated reactions listed above. We suggest      lation proposed Burgard et al (2001) identify the <br /> that OPHHX3, R1PK PDX5PO2 suppressed                  minimum required set genes (and non-gene associated <br /> under examined experimental conditions (Table IV). In           reactions) clusters prevent lethality. <br /> particular, non-gene associated reactions OPHHX3            Surprisingly, clusters K L the <br /> R1PK likely inactive aerobic conditions              minimal sets contained single member (i.e. k&#194;&#188;1). <br /> (Alexander Young, 1978; Hove-Jensen et al, 2003),               For example, maintaining activity purT (b1849) <br /> whereas PDX5PO2 likely inactive glucose              cluster K activity purU (b1232) purN <br /> minimal conditions (Zhao Winkler, 1995). These changes          (b2500) cluster L, prevent lethality. Unlike clusters <br /> would lead correct predictions essentiality      K L, cluster M fourteen alternative minimal sets each <br /> genes.                                                              containing members (i.e. k&#194;&#188;9) need active to <br />    We cases mismatches experimental          prevent lethality (see Supplementary information complete <br /> results concerning auxotrophy: (aroL, aroK) (tktA, tktB).       listing). <br /> The SL pair (tktA, tktB) auxotrophic aromatic amino <br /> acids requires addition pydxn (pyridoxine) the <br /> medium (Zhao Winkler, 1994). In contrast, silico         SL reaction triples <br /> predictions remained SL rich medium,      The application exhaustive enumeration procedure <br /> as unable produce biomass precursor pydx5p             described Materials methods section single- <br /> (pyridoxal 50 -phosphate). Pyridoxine direct precursor      reaction deletions led identification 277 essential <br /> pydx5p, inspection transport reactions contained      reactions (B13.5% total number reactions). Note that <br /> iAF1260 reveals pyridoxine transport reaction            did allow 304 exchange reactions 29 <br /> present. Thus, resolved vivo/in silico conflict       spontaneous reactions iAF1260 model participate in <br /> this SL addition pyridoxine uptake pathway      SL. After excluding essential, exchange, spontaneous <br /> the model. Interestingly, adding uptake pathway leads     981 blocked reactions, considered applying the <br /> to corrected prediction pdxH (b1638) non-essential      exhaustive enumeration procedure (see Materials meth- <br /> in rich medium implementing regulatory adjust-            ods section) 792 remaining reactions identify what <br /> ments. Table IV summarizes suggested iAF1260 model              pair triple combinations reaction eliminations negate <br /> modifications.                                                      biomass formation. For case pairs, 96 SL <br />                                                                     reaction pairs (see Supplementary information complete <br />                                                                     list). However, applying approach identify SL <br /> SL triples                                                          reaction triples required exhaustively exploring <br /> The concept synthetic (pair) lethality extended SL     83 million triple combinations. To avoid computational <br /> triples, simultaneous deletion genes       burden, developed targeted enumeration procedure (see <br /> lethal. When searching SL triples, essential genes SL   Materials methods section) relying bilevel optimiza- <br /> pairs excluded consideration eliminate trivial          tion procedure identify synthetic reaction triples without <br /> results. We identified 193 SL gene triples involving 114 genes      having explicitly test 83 million triple combinations. It <br /> and non-gene associated reactions (see Supplementary        identified total 243 SL triples involving 163 reactions. <br />  <br /> &amp; 2009 EMBO Macmillan Publishers Limited                                                                   Molecular Systems Biology 2009 7 <br />  Genome-scale gene/reaction essentiality synthetic lethality analysis <br /> PF Suthers et al <br />  <br />  <br />  <br />  <br />     Disjoint triples                                                                               k-connected triples <br />  <br />                                     G      nrdE             trxB            H       pstS                       pstA             I     glcE                          glcF                    J                  CMPN <br />   A      mgtA <br />                                                                                                                                                                                                   xapB                          surE <br />                 MG2t3_2pp                          trxC                                   pitB          pitA                                  yghK             lldP <br />                                                                                                                                                                                                   codA                        CSND <br />          corA                              trxA             gor <br />                                                                                    pstB                        pstC                    gcl                          glcD                                        cmk <br />                                                    k =1                                          k =1                                                 k =1 <br />                                                                                                                                                                                                                          k =1 <br />                                                                                                                                                                                lpcA     rfaZ <br />                                                                           hyfE                                                                                        gmhB                       rfaY <br />                                                              hycC <br />   B      cysP <br />                                    K                                                 hyfG                                L                      UPLA4FNF <br />                                                                                                                                                            UPLA4FNT                                     rfaS <br />  <br />                                                                                                                                                                                                                rfaP <br />                      sbp                            hyfF <br />                                                                                                  hycB                     k =1          ULA4Ntppi                                                                     lpxL <br />          modA                      k =1 <br />                                             hycD                                                                                    UDPGD                                                                                    lpxM <br />  <br />                                                                                                         hyfJ               UDCPPtppi                                                                                                arnT <br />  <br />                                           hyfD <br />                                                                                                                          LA4NTpp                                                                                                    UDPKAAT <br />                                                                                                          purT <br />   C      pgpB <br />                                                                                                                       ACOLIPAtex <br />                                           hyfA                                                                                                                                                                                             purN <br />                     bacA <br />          ybjG                                                                                           hyfI          ULA4NFT                                                                                                               galU <br />  <br />  <br />                                             hycE <br />                                                                                                                                                                                                                                            yfbG <br />                                                                                                                         purU <br />                                                                                                  hycG <br />                                                     hyfC                                                                 UDPGDC                                                                                                        ugd <br />   D      eutI <br />                                                              hycF <br />                                                                                      hyfH <br />                                                                           hyfB                                                 rfaQ                                                                                                 rfbC <br />                   ACt2rpp <br />                                                                                 cydD       AACTOOR                                    rfaG                                                                                   rfbD <br />          pta <br />                                                                                                                                              rfaB                                                                     yfbE <br />  <br />                                                                      gcvT                                  gshA                                     rfaI                                                       yfbF <br />  <br />                                                       M                                                                                                      rfaJ <br />                                                                                                                                                                       rfaK                       rfaD <br />                                                                                                                                                                                                         rfaE <br />  <br />                                                                                                                                                                                rfaC     rfaF <br />  <br />   E      pbpC <br />                                                                     eno                                         kbl <br />                     mrcA <br />                                                       k =9 <br />          mrcB <br />                                                                            glyA                         ltaE <br />                                                    73 triples                            tdh                                                                                                    pgk      idnO <br />                                                                                                                                                                                     gcvH                          mmuM <br />  <br />   F      gltS                                      32 genes <br />                                                                                                                                                                              gshB                                            tpiA <br />                                                    1 non-gene <br />                     actP <br />                                                    assoc. rxn <br />          sstT                                                                                                                                                          cydC                                                    idnT <br />  <br />                                                                          sucD <br />                                                                                   prpE <br />                                                                                                                                                                              gcvP                                            gapA <br />                                                               sucB <br />                                                                                      lpd <br />                                                                                                                                                                                       edd                             pgi <br />                                                                   ygfH <br />                                                                                                                                                                                                 mgsA      eda                         gnd <br />                                                                                   sucC <br />                                                                          sucA <br />                                                                                                                                                                                                                              sgcE             rpe <br />  <br /> Figure 3 Topological classification motifs SL gene triples. Both disjoint triples (left) k-connected triples (right) seen. Names genes set italics and <br /> the names non-gene associated reactions set roman type. Note reaction abbreviations follow iAF1260 (Feist et al, 2007). <br />  <br /> Table V Comparison approximate CPU time (single 3 GHz) finding                                         Similarly gene SLs, elimination reaction triples <br /> each essential reaction, SL reaction pair, triples quadruples using                                    yield auxotroph strains capable restoring growth <br /> exhaustive targeted enumeration approaches, respectively <br />                                                                                                                supply missing nutrients strains lack <br /> Order                       Exhaustive enumeration                            Targeted                         essential functionalities rescued adding extra <br /> of SLs                                                                      enumeration                        components growth medium. Of 243 predicted <br />                     Possible           SL (%)                CPU            CPU time/SL <br />                   combinations                             time/SL <br />                                                                                                                essential reaction triples, 202 (83%) were <br />                                                                                                                yield silico auxotroph strains rescued <br /> Single              B2050               13.5               B1 s                   B5 s                         supplementation (see Supplementary information for <br /> Double             B313,000             0.03              B28 min                B12 min                       complete listings). For example, elimination PGK (phos- <br /> Triple             B8.3  107         2.9  10 4            B2 days                B40 min <br /> Quadruple          B1.6  1010            ND                ND                    B5 h/SL                       phoglycerate kinase), TALA (transaldolase) TPI (triose- <br />                                                                                                                phosphate isomerase) results strain rescued <br /> ND, determined.                                                                                            (according model) supplementation the <br />                                                                                                                growth medium murein5px4p (two disacharide linked <br /> Table V depicts CPU times targeted enumeration                                                     murein units, pentapeptide crosslinked tetrapeptide). In con- <br /> procedure versus exhaustive enumeration procedure                                                          trast, eliminating AGMHE (ADP-D-glycero-D-manno-heptose <br /> revealing orders magnitude improvement.                                                                     epimerase), RPE (ribulose 5-phosphate 3-epimerase) and <br />  <br /> 8 Molecular Systems Biology 2009                                                                                                                                        &amp; 2009 EMBO Macmillan Publishers Limited <br />                                                                                                     Genome-scale gene/reaction essentiality synthetic lethality analysis <br />                                                                                                                                                           PF Suthers et al <br />  <br />  <br />  <br />  <br />        Disjoint triples                                                                  k-connected triples <br />  <br />  <br />                            A                        D             k=1                        E CYTBD2pp                                       DUTPDP <br />         MG2uabcpp                                                                                                    k=1               F                       k=1 <br />                                                          R1PK            PRPPS <br />                          MG2tpp                                                             QMO3                 SPODM              DURIK1               DURIPP <br />  <br />         MG2t3_2pp                                                                                                                             URIDK2r <br />                                                   NDPK1           ADK1            ADK3               DHORD2             SUCOAS <br />                                                                                                                                        k=1 <br />  <br />         KAS15 <br />                            B <br />                                                                                                            G ACCOAL               PPCSCT <br />  <br />                       MACPD <br />                                                           PPM            R15BPK                                          AKGDH <br />  <br />        ACOA TA                                H                                                        I <br />                                                                                                                                                                 k=23 <br />                                                                    k=2 <br />                                            GLYCT O2                                  HPYRI <br />                            C                                    GLYCLTtex <br />         TRDR <br />  <br />                        CAT <br />                                                               GLYCLTt2rpp <br />                                                                                                       222 triples <br />      PDX5PO2 <br />                                            GLYCT O3                                 TRSARr            129 reactions <br />  <br />  <br />  <br /> Figure 4 Topological classification motifs SL reaction triples. Similarly SL gene triples Figure 3, SL reaction triples occur disjoint triples (left) and <br /> k-connected triples (right). Note reaction abbreviations follow iAF1260 (Feist et al, 2007). <br />  <br /> TALA (transaldolase) yields strain, produce                                 Table VI Frequency participation reactions multiple SL triples <br /> in silico biomass precursors supplemented <br />                                                                                            # SL triples                                   # reactions (Rxn abb.) <br /> medium unable make uptake precursor <br /> metabolite pydx5p (pyridoxal 50 -phosphate).                                                1                                                            42 <br />    Reaction SL triples pictorially shown triangles                              2                                                            67 <br /> with reactions depicted edge connected nodes (see                              3                                                             8 <br />                                                                                             4                                                            12 <br /> Figure 4). This graphical representation reveals                                5                                                             3 <br /> small number reactions (i.e. 34) form small clusters (see                                6                                                             5 <br /> Figure 4) (i.e. 129) joined                            7                                                             1 <br />                                                                                             8                                                             4 <br /> the highly connected large cluster I (i.e. giant component). For                            9                                                             4 <br /> example, reactions cluster A shown Figure 4                            10                                                             5 <br /> responsible magnesium transport different mechan-                                11                                                         1 (RPI) <br /> isms. Interestingly, looking GPR associations                           12                                                        1 (PPS) <br />                                                                                            17                                                         1 (PGI) <br /> this cluster maps exactly cluster A Figure 3                              20                                                        1 (PGM) <br /> reactions MG2uabcpp MG2tpp coded genes                                  27                                                  3 (GAPD, PGK, TALA) <br /> mgtA (b4242) corA (b3816), respectively. The giant                                     33                                                        1 (RPE) <br />                                                                                            34                                              3 (ATPS4rpp, FTHFD, GARFT) <br /> component (cluster I) consists 129 reactions forming 222                                35                                                         1 (TPI) <br /> SL triples. We used mixed-integer optimization formula- <br /> tion proposed Burgard et al (2001) identify                                     All abbreviations follow Feist et al (2007). The complete list given in <br /> alternative reaction sets, 23 reactions (i.e. k&#194;&#188;23)                              supplementary information. <br /> that allow biomass components formation (see <br /> Supplementary information). These minimal reaction                                    highly triple-participating reaction, membership 35 <br /> sets spanned 29 different reactions, seventeen                                different SL triples. Not surprisingly, reactions <br /> present alternative minimal sets (see Supplemen-                               appear complex cluster SL triples (cluster I). <br /> tary information).                                                                         Interestingly, half reactions belong to <br />    We analyzed reaction SL triples determining                              glycolysis, rest (except ATPS4rpp) are <br /> the number SL triples reaction participates                               involved pentose phosphate pathway (PPP), folate meta- <br /> (see Table VI). We wide range participation                                  bolism purine pyrimidine biosynthesis. Among the <br /> different reactions. Most reactions (i.e. 85%) appear                            interesting patterns, reactions that <br /> seven fewer triples, reactions                                      participate large number SLs catalyzed proteins <br /> participate triples. Notably, TPI                             encoded genes participate large number of <br />  <br /> &amp; 2009 EMBO Macmillan Publishers Limited                                                                                               Molecular Systems Biology 2009 9 <br />  Genome-scale gene/reaction essentiality synthetic lethality analysis <br /> PF Suthers et al <br />  <br />  <br />  <br />  <br /> SLs. Specifically, reactions FTHFD GARFT                               Upon excluding spontaneous, exchange, blocked essential <br /> associated genes purU (b1232) purN (b2500)                           reactions, 229 SL reaction quadruples 137 reactions <br /> are highly connected nodes cluster L Figure 3. Similar                           involved identified (see Supplementary information for <br /> observations cluster J Figure 1 clusters A Figures 3                     complete list). Using targeted enumeration approach <br /> and 4 indicate gene synthetic lethality explained                       able elucidate higher-order SL inter- <br /> analyzing corresponding reaction synthetic lethality.                              actions, SL reaction quintuples. For example, set <br />                                                                                        reactions F6PA (fructose 6-phosphate aldolase), FBA <br />                                                                                        (fructose-bisphosphate aldolase), GLCptspp (D-glucose trans- <br /> Higher-order SLs                                                                       port PEP:Pyr PTS [periplasm]), GLCt2pp (D-glucose <br />                                                                                        transport proton symport [periplasm]) RPI <br /> We identified number SL quadruples iAF1260 <br />                                                                                        (ribose-5-phosphate isomerase) SL quintuple. <br /> model aerobic glucose minimal medium conditions. <br />                                                                                           Unlike essential reactions participate SLs, <br />                                                                                        possible gene/reaction involved SL pairs also <br />                 Genes                                  Reactions <br />                                                                                        participate SL triples higher-order SLs. <br />                                                                                        Figure 5 shows form Venn diagram number of <br />      Blocked             Essential            Blocked            Essential <br />                                                                                        genes/reactions participate various orders SLs. We <br />         401                188                   774                282                note 183 non-essential,&#226;&#8364;&#732;non-blocked&#226;&#8364;&#8482; genes participate in <br />                                                                                        SL pair triple. Interestingly, fourteen <br />                                                                                        genes participating SL pair participate in <br />          Pairs          Triples                   Pairs         Triples <br />                                                                                        SL triples. In case SL reactions, identified <br />                                                                                        39 participate SL pairs triples. For example, the <br />            69     14       100                     80      39       102                glycine cleavage (GLYCL), involved the <br />                                                                                        degradation glycine ammonia CO2, participates in <br />                                                                                        SL pairs, SL triples SL quadruples (see <br /> Figure 5 Venn diagram number genes reactions participating               Figure 6). We identified nineteen SL quintuples for <br /> in SLs order one, three.                                                    GLYCL. This implies deletion of reactions <br />  <br />  <br />  <br />                                               PGCD           PSP_L          PSERT         GHMT2r <br />  <br />  <br />                                   PGK                                                             GLUCYS* <br />  <br />  <br />  <br />                                                                                                   AACTOOR* <br />  <br />                                      ENO                                                                         GTHS <br />  <br />  <br />                                    GAPD                                                                   GTHRDabc2pp* <br />                                                                                                                                      GLYAT* <br />  <br />                          PGM <br />  <br />                                                                     GLYCL                                    CGL Ytex <br />  <br />                          PTAr <br />  <br />                                    ACALD                                                               GLYAT* <br />                                                                                                                                  GTHRDtex <br />  <br />                                                                                                                AACTOOR* <br />                          PDH <br />                                                             TRDR <br />                                             GLUCYS*                        GTHRDabc2pp* <br />                                   ACKr                                                                   GTHRDHpp <br />  <br />  <br />                                                                    CAT <br />  <br /> Figure 6 Pictorial view SLs reaction GLYCL (glycine cleavage system). Double, solid dashed lines depict SL pairs, triples quadruples, respectively. <br /> The reaction abbreviations follow iAF1260 (Feist et al, 2007). Note ease presentation reactions occur SL triples SL quadruples are <br /> repeated (marked star) SL quadruples containing coupled fluxes SL quintuples. The complete list SL quadruples <br /> and quintuples GLYCL given Supplementary information. <br />  <br /> 10 Molecular Systems Biology 2009                                                                                       &amp; 2009 EMBO Macmillan Publishers Limited <br />                                                                              Genome-scale gene/reaction essentiality synthetic lethality analysis <br />                                                                                                                                    PF Suthers et al <br />  <br />  <br />  <br />  <br /> depicted Figure 6 compensated GLYCL alone.             Table VII Distribution degree essentiality (DOE) genes/reactions <br /> Alternatively, removal GLYCL render PGCD,              network. Table entries represent gene/reactions specific <br />                                                                     degree essentiality belong COG functional class (see Table I COG <br /> PSP_L, PSERTand GHMT2r essential. Notably, reactions        abbreviations). Note gene/reactions belong one <br /> present Figure 6 forming SL pair GLYCL (except           COG functional class <br /> PSERT) associated genes cluster J Figure 1. <br />                                                                     Cog                   Genes                                 Reactions <br /> Among reactions, GHMT2r converts glycine serine,           Class <br /> whereas, PGCD, PSERT PSP_L serve first, second                         DOE          Blocked Total           DOE            Blocked Total <br /> and committed steps serine biosynthesis pathway, <br />                                                                             1    2   3 4+                         1   2    3 4+ <br /> respectively. Therefore, primary reason synthetic <br /> lethality GLYCL reactions          J        1 0 1    0            24       26    1    0    1     3     25       30 <br /> requirement serine production directly conver-       K        0 0 0    2             2        4    0    1    0     6      1        8 <br /> sion glycine (see Supplementary information              L        0 2 0    3             0        5    1    0    1    12      0       14 <br />                                                                     V        1 0 1    4             0        6    4    1    1    38      0       44 <br /> complete list precursor metabolites           T        1 0 0    2             1        4    1    0    0     9      8       18 <br /> produced reaction eliminations). Furthermore,        M       19 6 20 37             13       95   42    7   25   173    210      457 <br /> elimination GAPD PGK ENO PGM prevents          U        0 0 0    2             0        2    0    1    0     4      7       12 <br />                                                                     O        0 1 3 11               8       23    1   10    2     8     16       37 <br /> the production metabolite 3pg (3-phospho-D-glycerate),           C        7 8 28 62             67      172    7   19   11    55     99      191 <br /> thereby blocking serine biosynthesis pathway. Finally,      G        2 6 9 62             103      182    3   12   24    74    123      236 <br /> removal GLYCL combination GLUCYS,           E       52 28 4 116            30      230   72   20   12   140     70      314 <br />                                                                     F       22 13 2 25              5       67   28   18   13    49     43      151 <br /> GLYAT, AACTOOR GTHRDabc2pp prevents formation               H       49 6 3 11              28       97   60   12    4    37     47      160 <br /> of biomass precursor metabolites coa (co-enzyme A), amet        I       17 1 4 19              13       54   48   25    3   118     70      264 <br /> (S-adenosyl-L-methionine) sheme (sirohem). This com-            P        7 9 10 34             33       93   11   19    6   112     58      206 <br /> prehensive synthetic lethality analysis GLYCL demonstrates       Q        0 0 0    6             4       10   22    4    2    39     22       89 <br />                                                                     R        0 0 2 16              14       32   15    6    9    97     78      205 <br /> that looking higher-order SLs unravel non-            S        0 0 0    3             1        4    1    0    2     9      3       15 <br /> intuitive biomass component deficiencies            None    10 3 13 73             55      154   30    8   17   123    125      303 <br /> apparent visual inspection metabolic map. <br />    A similar pattern emerged identified set SL reaction <br /> quadruples. Most highly participating reactions SL <br /> triples appear high frequency list identified   DOE COGs reveals differing buffering <br /> SL quadruples. For example, TPI participates 55 SL               capacity functional category biomass formation <br /> quadruples. Notably, instances           (Deutscher et al, 2006). <br /> one highly participating member SL triples occurring                Next, focus attention DOE results the <br /> together SL quadruples. For instance, 55 SL           reactions participating central metabolism (spanning <br /> quadruples TPI, 21 contained PGM (phosphoglyce-           glycolysis, PPP, TCA cycle anaplerotic reactions). Figure 8 <br /> rate mutase). The reason reactions appear SLs      illustrates color-coded degree essentiality reactions <br /> different orders serve key branch points         central metabolism DOE four. We the <br /> central metabolism. The simultaneous removal multiple            majority reactions central metabolism DOE two, <br /> branch points require flux rerouting bypass      more. This likely presence of <br /> reactions latent pathways (Fong et al, 2006)             multiple diverging converging branches pathways of <br /> production essential biomass precursors amino            central metabolism. The relatively small fraction essential <br /> acids.                                                              reactions expected, earlier reports noted conserved <br />                                                                     metabolic pathways TCA cycle glycolysis <br />                                                                     generally contain essential reactions (Gerdes et al, 2003; <br /> Degree essentiality                                              Ghim et al, 2005). Most reactions PPP DOE <br /> To quantify dispensability gene reaction           two, glycolycis reactions involve DOEs or <br /> metabolic network respect biomass formation,             more. No reaction glycolysis PPP DOE one, <br /> introduce concept degree essentiality (DOE). This         reactions essentiality degree of <br /> metric defined size smallest SL gene   belong TCA cycle. Eliminating four <br /> reaction member of. Therefore, essential genes reactions    essential reactions prevent formation list <br /> have DOE one, genes reactions participate      precursor metabolites including sheme (sitroheme), pheme <br /> in SL pairs (and higher-order SLs) DOE         (protoheme) murein5px4p (two disacharide linked <br /> two. It noted DOE metric genes akin    murein units, pentapeptide crosslinked tetrapeptide). All <br /> the &#226;&#8364;&#732;k-robustness&#226;&#8364;&#8482; term introduced Deutscher et al (2006).       reactions subsequent steps TCA cycle <br /> We determined DOE genes              converting oxaloacetate 2-ketoglutarate. Application flux <br /> reactions DOE reactions central    coupling analysis (Burgard et al, 2004) showed of <br /> metabolism active aerobic glucose conditions. The             fully coupled (i.e. ACONTa (aconitase [half-reaction <br /> distribution DOE genes reactions present different    B, Isocitrate hydro-lyase]), ACONTb (aconitase [half-reaction <br /> COG classifications (Tatusov et al, 2003) shown Table VII     B, Isocitrate hydro-lyase]) ICDHyr (isocitrate dehydrogen- <br /> and Figure 7. Data Table VII genes reactions       ase)). It important note reactions operating in <br /> in different COGs quite different DOE statistics. Figure 7b    opposite directions different DOEs. Such examples <br /> pictorially delineates percentage reaction participation     include reaction pairs FBP PFK PPC PPCK. <br />  <br /> &amp; 2009 EMBO Macmillan Publishers Limited                                                                     Molecular Systems Biology 2009 11 <br />  Genome-scale gene/reaction essentiality synthetic lethality analysis <br /> PF Suthers et al <br />  <br />  <br />  <br />  <br />    A                          1   2   3   4+   Blocked                             B                                      1   2    3    4+   Blocked <br />  <br />                              Gene degree essentiality                                                                  Reaction degree essentiality <br />             %         20%           40%         60%           80%          100%                                 %   20%           40%         60%       80%        100% <br />        J                                                                      26                            J                                                            30 <br />        K                                                                       4                            K                                                             8 <br />        L                                                                       5                            L                                                            14 <br />        V                                                                       6                            V                                                            44 <br />        T                                                                       4                            T                                                            18 <br />        M                                                                      95                            M                                                          457 <br />  <br />  <br />  <br />  <br />                                                                                    COG functional classes <br />        U                                                                       2                            U                                                            12 <br />        O                                                                      23                            O                                                            37 <br />        C                                                                     172                            C                                                          191 <br />        G                                                                     182                            G                                                          236 <br />        E                                                                     230                            E                                                          314 <br />        F                                                                      67                            F                                                          151 <br />        H                                                                      97                            H                                                          160 <br />         I                                                                     54                            I                                                          264 <br />        P                                                                      93                            P                                                          206 <br />        Q                                                                      10                            Q                                                            89 <br />        R                                                                      32                            R                                                          205 <br />        S                                                                       4                            S                                                            15 <br />    None                                                                      154   None                                                                                303 <br />  <br /> Figure 7 Percentage degree essentiality different parts network arranged COG (Tatusov et al, 2003) functional classifications. (A) Genes. <br /> (B) Reactions. The numbers right indicate total number items COG classification. Note genes/reactions belong one <br /> COG classification. <br />  <br />  <br /> Discussion                                                                         forms SL pairs RPE (ribulose 5-phosphate 3-epimerase), <br />                                                                                    TALA (transaldolase), TKT1 TKT2 (transketolase) and <br /> We presented comprehensive silico study gene                           participates number higher-order SLs (see <br /> reaction synthetic lethality E. coli K12 based recent                    Table IV). These erroneous model predictions the <br /> genome-scale metabolic model iAF1260. This computationally                         fact genome-scale metabolic model reconstructions tend <br /> intensive goal possible developing efficient                        over, under, predict metabolic capabilities of <br /> procedure targeted enumeration SL interactions                      organism. This arises inclusion model of <br /> relying bilevel optimization. Unlike earlier efforts                       functionalities active sufficient level (e.g. due <br /> relied incomplete sampling elucidate partial lists                        regulation) ensure biomass formation. The list SLs <br /> higher-order SLs (Behre et al, 2008; Deutscher et al, 2008),                       reported study should, therefore, interpreted a <br /> here provided complete enumerations high-order SL                            conservative depiction synthetic lethality, recognize <br /> interactions gene reaction centric representations.                   additional SLs likely present stoichiometric <br /> The network organization elucidated SLs recapitulated                       models reveal (i.e. false-negatives). <br /> modular lethality relationships consistent previous                              We exploited identified vivo/in silico mismatches <br /> observations yeast (Segre et al, 2005). By coloring network                     suggest number iAF1260 model modifications (see <br /> nodes using COG classification, surprising compensatory                        Table IV). This motivated observation false- <br /> interactions seemingly unrelated gene reaction                             negative/positive model predictions provide opportunities <br /> classes revealed. Earlier efforts silico elucida-                  model improvement re-evaluation of <br /> tion SLs (Wunderlich Mirny, 2006; Palumbo et al, 2005,                      experimental data (Thiele et al, 2005). Many model <br /> 2007) did anticipate lethality caused inability                      corrections involve addition removal reactions <br /> meet non-growth associated ATP maintenance require-                            iAF1260 incorporation regulatory <br /> ment. Notably, study, 120 SLs                      constraints (i.e. condition-dependent presence different <br /> triggered deficiency.                                                      reactions). Unfortunately, existing experimental data on <br />    By contrasting literature data gene essentiality                         SLs account small portion predicted SL <br /> synthetic lethality predicted SLs, 27 instances                         limiting potential model improvement. This <br /> inconsistencies (false-positive SLs) identified. Similar                      calls development combinatorial methods the <br /> examples false-positive predictions                       rapid generation screening SLs recently <br /> reaction SLs. For example, reaction RPI (ribose-5-phosphate                        developed GIANT-coli technology allows the <br /> isomerase) essential vivo (Neidhardt                        high-throughput generation double mutants E. coli <br /> Curtiss, 1996), however, according model iAL1260,                        (Typas et al, 2008). <br />  <br /> 12 Molecular Systems Biology 2009                                                                                                  &amp; 2009 EMBO Macmillan Publishers Limited <br />                                                                                                                           Genome-scale gene/reaction essentiality synthetic lethality analysis <br />                                                                                                                                                                                 PF Suthers et al <br />  <br />  <br />  <br />  <br />                             Degree essentiality                                    glc-D <br />                                                                 GLCt2       h <br />                                              1                                                atp <br />                                                                   h[e]                    HEX1 <br />                                              2                                                  h <br />                                                                                   adp <br />                                                                           GLCpts G6PDHy                     6pgl      PGL    6pgc                 GND <br />                                              3 <br />                                                            glc-D[e]                   g6p <br />                                                                   pep             pyr nadp nadph h h2o                   h     EDD        nadp          nadph co2 <br />                                              4 <br />                                                                                 PGI                                    h2o <br />                                                                                           f6p <br />                                          5+ <br />                                                                        pi <br />                                                                                 atp <br />                                                                                                                                                         TKT2 <br />                                                                                                                                                                 RPE <br />                               Blocked                                                                                                                                   ru5p-D <br />                                                                        FBP PFK F6PA                                2ddg6p                       e4p            xu5p-D <br />                                                                                 adp                                                      TAL <br />                                                                     h2o          h <br />                                                                                   fdp                                                               TKT1                RPI <br />                             h2o              lgt-S    LGTHL                   FBA                                     EDA                  s7p                 r5p <br />                                                                                           dha <br />                                                              pi <br />                          GLYOX                                      MGSA              TPI <br />                               h             gthrd                                       g3p <br />                                                           mthgxl            dhap                      pi <br />                                          lac-D                                                        nad <br />                             nad <br />                                                                                     GAPD <br />                                           LDH_D                                                       nadh <br />                           nadh                                                                        h <br />                                                                                       13dpg <br />                               h                                                                       adp <br />                                        pyr                                            PGK <br />                                                                                                       atp <br />                                                                                         3pg <br />                                                                                     PGM <br />                                                                                                     2pg <br />                                                                                     ENO <br />                                                                                                       h2o <br />                                                                                  pi                 pep <br />                                                                                 amp                     h <br />                              co2                   co2                            h                     adp <br />                              h2o                   adp                          PPS                   PYK <br />                                                                                  h2o <br />                               PPC                PPCK                            atp                      atp <br />                                   h                 atp <br />                                   pi             nadph              co2                   pyr         coa <br />                                                    co2              nadh                              nad <br />                                                                                       PDH            nadh <br />                                                      ME2        ME1x                                 co2 <br />                                                                                       accoa          h2o          coa h succ[e]           succ <br />                                                   nadp              nad <br />                                                                                     oaa                         CS <br />                                                                                                                                 CITt7 <br />                                                                h <br />                                                             nadh         mql8           ubq8h2              CITL             cit           cit[e] <br />                                                                                                      ac <br />                                                                 MDH MDH3 MDH2                                                ACONT <br />                                                              nad         mqn8 <br />                                         MALt6_na                                            ubq8                ICL <br />                                                                                                                               icit <br />                         mal-L[e] <br />                                                                 mal-L           MALS            glx             nadp <br />                                        na1[e]             na1             coa h               h2o <br />                                                             FUM                                                              ICDHy <br />                                                                              h2o              nadph                           hAKGt6       h[e] <br />                          fum[e]                                           fum <br />                                                                                     mql8         co2 <br />                                                                                                                                                akg[e] <br />                                    SUCFUMt                                    fadh2      2dmmql8     akg                           coa <br />                                                                                                                                    nad <br />                                                           SUCD1i                FRD2 FRD3 <br />                                                                                                                              AKGD <br />                                                                               fad                   2dmmq8 <br />                                                                                        mqn8                                   co2 <br />                                                           SUCCt2                                       SUCOAS                nadh <br />                           succ            succ[e]                      succ                                              succoa <br />                                                 h[e]               h                    atp         coa          pi    adp <br />  <br /> Figure 8   Color-coded representation reactions central metabolism according degree essentiality. <br />  <br />  <br />  <br />  <br />    One key challenges correctly interpreting                                                      UbiX UbiD required decarboxylation, <br /> obtained predictions delineation true function                                                 especially logarithmic growth, implying genes <br /> isozymes complementation studied conditions.                                                      essential. These inconsistencies silico predictions <br /> For example, experimental evidence predicted SL                                                     vivo observations nuanced model <br /> (ubiX, ubiD) conflicting. Some analyses propose                                                     modifications dependent conditions but <br /> ubiD knockout lethal (Baba et al, 2006; Joyce et al, 2006;                                                   growth phase. In example, metL (b3940) and <br /> Feist et al, 2007), data Covert et al (2004)                                                     thrA (b0002) form disjoint SL pair isozymes the <br /> suggest ubiX expressed examined condi-                                                    aspartate kinase activity. Surprisingly, genes reported <br /> tions. In contrast, Gulmezian et al (2007) recently observed                                                    essential (Table IV), suggesting OR operator must <br />  <br /> &amp; 2009 EMBO Macmillan Publishers Limited                                                                                                                         Molecular Systems Biology 2009 13 <br />  Genome-scale gene/reaction essentiality synthetic lethality analysis <br /> PF Suthers et al <br />  <br />  <br />  <br />  <br /> be changed AND operator gene&#226;&#8364;&#8220;protein&#226;&#8364;&#8220;reaction                    The analysis synthetic lethality metabolic networks <br /> association. Therefore, simply knowing gene essential              requires introduction following sets: <br /> is sufficient. In cases, elucidating                                    I &#194;&#188; fiji &#194;&#188; 1; 2; :::; Ng &#194;&#188; set metabolites <br /> functional reason(s) (e.g. loss sufficient aspartate kinase                              J &#194;&#188; fjjj &#194;&#188; 1; 2; :::; Mg &#194;&#188; set reactions <br /> activity, etc.) essentiality needed properly                             K &#194;&#188; fkjk &#194;&#188; 1; 2; :::; Gg &#194;&#188; set genes <br /> correcting model. As indicated Table III, classifica- <br /> tion entD cluster F Figure 1 essential likely                 where, N, M G denote total number metabolites, reactions <br /> erroneous. This observation supported fact                    genes network, respectively. On imposing metabolite <br /> eliminating FE3abcpp make entD essential                       balances entire metabolic model steady-state <br /> render genes essential known                   conditions obtain: <br /> non-essential. Finally, predictions disruption                                      X <br />                                                                                                        sij nj &#194;&#188; 0 ; 8 2 I                  &#195;&#176;1&#195;&#382; <br /> of particular SL results auxotroph strain likewise                                       j <br /> exploited model correction demonstrated SL <br /> (tktA, tktB).                                                                where, sij, represents stoichiometric coefficient metabolite i, <br />    The introduction concept degree essentiality                 reaction j, nj, denotes flux reaction j. Next, the <br /> enabled quantitative assessment dispensability                 exhaustive targeted SL enumeration approaches detail. <br /> any gene reaction metabolic network. Moreover, <br /> by querying derived list SLs, examine how <br />                                                                              Exhaustive enumeration SL interactions <br /> the removal gene/reaction affects dispensability <br />                                                                              The computational prediction synthetic lethality hinges the <br /> characteristics genes/reactions. Results GLYCL <br />                                                                              calculation maximum biomass formation presence of <br /> (see Figure 6) revealed surprising compensatory interactions                 gene reaction deletions implied examined SL. We chose <br /> with reactions seemingly unrelated pathways. The elucida-                 1% maximum theoretical biomass yield cutoff for <br /> tion SL DOE human metabolism implications                   computationally predicted growth. We prediction of <br /> the identification drug targets. For example,                 silico lethality particularly sensitive selected biomass <br />                                                                              formation cutoff value. For example, considering single-gene <br /> suggested SL partners missing enzymatic function-                mutants mutants (which 0.6% genes in <br /> alities tumors cells promising drug targets                      model) involve biomass formation values 1&#226;&#8364;&#8220;50%. We are <br /> (Hartman et al, 2001; Kamb, 2003). The idea healthy                  aware <span id='am-5' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span> silico synthetic lethality Deutscher <br /> cells remain unaffected ability compensate               et al (2006) study. However, believe using conservative <br />                                                                              universal cutoff 1% mutants safeguards possibility <br /> for drug-suppressed functionality, tumor cells, <br />                                                                              misclassifying SL gene mutants viable. In other <br /> with missing enzymatic functionality, not. On                      words, want minimize occurrence false-positives perhaps <br /> another front, SL predictions used pinpoint multi-               expense missing SLs. If D set reactions set <br /> gene disease mappings (Hoh Ott, 2003) identify                       zero directly consequence GPR associations <br /> combinations genes likely interact disease                     implied gene deletions problem determining the <br />                                                                              maximum biomass formation formulated following <br /> phenotypes (Wong et al, 2004).                                               linear program: <br />    The procedure proposed used rapidly <br /> predict growth phenotypes multiple knockout mutants                                    Maximize          vbiomass      &#194;&#189;MaxBiomass  <br />                                                                                                                                                          &#195;&#176;2&#195;&#382; <br /> for variety organisms S. cerevisiae,                                       s:t: <br /> B. subtilis H. pylori various media. The effect                                               P <br /> different conditions (i.e. alternate carbon substrates, aerobic                                                 sij nj &#194;&#188; 0   8i 2 I                      &#195;&#176;1&#195;&#382; <br />                                                                                                             j <br /> vs anaerobic, etc.) and/or certain regulatory constraints the <br /> membership SL sets assessed straightforward <br />                                                                                                        LBj pnj pUBj           8j 2 J                     &#195;&#176;3&#195;&#382; <br /> manner adjusting appropriate model constraints based on <br /> exchange reaction usage gene/reaction availability.                                                 vd &#194;&#188; 0 d 2 D   J                                 &#195;&#176;4&#195;&#382; <br />  <br />                                                                                                         vglucose Xvuptake <br />                                                                                                                    glucose <br />                                                                                                                            limit <br />                                                                                                                                                          &#195;&#176;5&#195;&#382; <br />  <br /> Materials methods                                                                                   voxygen Xvuptake <br />                                                                                                                   oxygen <br />                                                                                                                          limit <br />                                                                                                                                                          &#195;&#176;6&#195;&#382; <br /> Two separate optimization-based procedures enumeration all <br /> SLs gene reaction levels described. The relies                              vATPM &#194;&#188; vmaintenance                             &#195;&#176;7&#195;&#382; <br />                                                                                                                  ATPM <br /> the exhaustive biomass formation capability evaluation single, <br /> double, triple, etc. combinations gene and/or reaction deletions. <br /> This method computationally prohibitive searching                                              vj 2 R 8 j 2 j <br /> higher-order SLs (n42). Therefore, alternative efficient <br />                                                                                                                             uptake limit uptake limit <br /> and targeted method, relying bilevel optimization, described      Here, vbiomass denotes biomass flux vglucose     , voxygen     and <br /> identifies SL combinations relying exhaustive             vmaintenance <br />                                                                                ATPM       denote minimum required glucose oxygen uptake <br /> enumeration possible gene/reaction eliminations. To reduce        rates non-growth associated ATP maintenance, respec- <br /> search space approaches, flux coupling analysis           tively. The values upper lower bounds, UBj LBj, in <br /> (Burgard et al, 2004) performed pre-processing step allow        Equation (3) chosen exclude physiologically <br /> the removal representative fully coupled reaction        relevant metabolic flux values. The upper bound reactions was <br /> set. It noted, however, complete lists reactions       set 1000. The lower bound set zero irreversible reactions <br /> present SL degree two, provided               1000 reversible reactions. For external carbon <br /> Supplementary information.                                                   containing metabolite, maximum transport rate cell was <br />  <br /> 14 Molecular Systems Biology 2009                                                                                &amp; 2009 EMBO Macmillan Publishers Limited <br />                                                                                      Genome-scale gene/reaction essentiality synthetic lethality analysis <br />                                                                                                                                            PF Suthers et al <br />  <br />  <br />  <br /> set 20 mmol gDW 1 h 1. For remaining source exchange fluxes,          primal objective function equal dual introduced in <br /> the lower bound set  1000 mmol gDW 1 h 1 (Feist et al, 2007).         (Burgard et al, 2003): <br /> Glucose minimal conditions modeled restricting glucose <br />                                                                                                                  Minimize vbiomass <br /> uptake rate 10 mmol gdW 1 h 1 oxygen uptake rate                                                                                                    &#195;&#176;9&#195;&#382; <br /> 20 mmol gdW 1 h 1. The non-growth associated ATP maintenance                                                     s:t: <br /> was fixed 8.39 gDW 1 h 1 (Feist et al, 2007).                                                             X <br />    The elucidation SL reactions using formulation MaxBiomass                          vbiomass &#194;&#188;                  mj UBj yj &#195;&#190; vmaintenance <br />                                                                                                                                      ATPM        :mATPM          &#195;&#176;10&#195;&#382; <br /> straightforward membership set D (the set reaction                                                   j <br /> deletions) known priori. In case gene deletions, information                                     X <br /> gleaned GPR association relations needs encoded                                                     sij nj &#194;&#188; 0 ; 8i 2 I                         &#195;&#176;1&#195;&#382; <br /> elucidate effect gene deletions reaction deletions                                                   j <br /> accounting isozymes, multi-meric enzymes combination <br /> thereof. Formulation MaxBiomass iteratively solved identify SL                                              vj pUBj yj        8j 2 J                       &#195;&#176;11&#195;&#382; <br /> for genes reactions involving pre-specified number n deletions <br /> (where n order SL sought-after). This exhaustive evaluation                                            vglucose pvuptake <br />                                                                                                                             glucose <br />                                                                                                                                     limit <br />                                                                                                                                                                  &#195;&#176;12&#195;&#382; <br /> becomes computationally intractable higher-order (higher than <br /> two) SLs (see Table V) motivating development following                                               voxygen pvuptake limit <br />                                                                                                                                                                  &#195;&#176;13&#195;&#382; <br /> targeted enumeration procedure.                                                                                            oxygen <br />  <br />  <br />                                                                                                               vATPM &#194;&#188; vmaintenance <br />                                                                                                                        ATPM                                       &#195;&#176;7&#195;&#382; <br /> Targeted enumeration SLs                                                                X <br />                                                                                                  li sij &#195;&#190;mj X0              8j 2 J   fbiomass; ATPMg             &#195;&#176;14&#195;&#382; <br /> The proposed targeted enumeration procedure relies solution                          i <br /> of bilevel optimization formulation identifies n simultaneous                                   X <br /> gene/reaction deletions suppressing biomass formation. This bilevel                                           li si biomass &#195;&#190; mbiomass X1                        &#195;&#176;15&#195;&#382; <br /> formulation identifies set n gene/reaction deletions                                         i <br /> minimizes maximum biomass formation potential network.                                            X <br /> If minimal value maximum biomass                                                  li siARPM &#195;&#190;mATPM X0                        &#195;&#176;16&#195;&#382; <br /> imposed cutoff (i.e. 1% maximum biomass) corresponding                                           i <br /> combination n gene/reaction deletions forms SL. It important                                               X <br /> note obviates need explore exhaustively deletion                                                       &#195;&#176;1   yj &#195;&#382;pn                          &#195;&#176;17&#195;&#382; <br /> combinations bilevel formulation &#226;&#8364;&#732;homes in&#226;&#8364;&#8482; biomass                                                  j <br /> negating combinations. <br />     The mathematical description bilevel formulation                                       vj ; mj X0               8 j 2 J   fATPMg <br /> determine SL reactions requires <span id='am-6' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span> binary variable yj <br /> that encodes reactions deleted:                                                                    li ; mATPM 2 R              8i 2 I <br />                     <br />                     0; reaction j eliminated                                                           yj 2 f0; 1g                 8j 2 J <br />              yj &#194;&#188;                                   8j 2 J         &#195;&#176;8&#195;&#382; <br />                     1; reaction j active <br />                                                                              Here, li, mj mATPM dual variables associated the <br /> This allows forth following min&#226;&#8364;&#8220;max bilevel optimization       stoichiometric constraints (equation (1)), inequalities equation (11) <br /> formulation (SL Finder):                                                     constraint non-growth associated ATP maintenance <br />                                                                              (equation (7)), respectively. Equations (12) (13) replace (5) and <br />                Minimize            vbiomass   &#194;&#189;Outer  &#194;&#189;SL Finder             (6), respectively, split reversible fluxes into <br />                      yj <br />                                                                              forward backward reaction steps. Note non-linear term <br />               s:t:                                                           mjyj equation (10) exactly linearized follows: <br />                  2                                      3 <br />                    Maximize vbiomass &#194;&#189;Inner                                                                                 aj &#194;&#188; mj yj                           &#195;&#176;18&#195;&#382; <br />                            vj <br />                  6                                      7 <br />                  6 s:t:                                 7 <br />                  6 P                                    7                                                        mmin yj paj pmmax yj                            &#195;&#176;19&#195;&#382; <br />                  6                                      7                                                         j            j <br />                  6            sij vj &#194;&#188; 0         8i 2 I 7 <br />                  6       j                              7 <br />                  6                                      7                                       mj   mmax &#195;&#176;1   yj &#195;&#382;paj pmj   mmin &#195;&#176;1   yj &#195;&#382;                      &#195;&#176;20&#195;&#382; <br />                  6 v pUB y                   8   j 2 J  7                                             j                       j <br />                  6      j           j j                 7 <br />                  6                                      7 <br />                  6 vglucose pvuptake limit              7                     mmin <br />                                                                                      j        mmax <br />                                                                                                    j  lower upper bounds dual <br />                  6                      glucose         7                    variable mj. If optimal objective function value above <br />                  6                                      7 <br />                  6 voxygen pv           uptake limit    7                    optimization problem imposed cutoff, the <br />                  6                      oxygen          7 <br />                  6                                      7                    reactions yj&#194;&#188;0, reported SL degree n. All <br />                  4 vATPM &#194;&#188; vmaintenance ATPM            5 <br />                                                                              alternative SL reaction sets size n successively obtained by <br />                           vj &#194;&#188; 0        8j 2 J                               excluding previously identified SLs using integer cuts resolving <br />                  X                                                           bilevel formulation. For example, reactions j1, j2,y, jn found <br />                           &#195;&#176;1   yj &#195;&#382;pn                                        form SL set size n, exclude solution obtain the <br />                      j                                                       appending following constraint formulation that <br />                  yj 2 f0; 1g            8j 2 J                               ensures reactions forming previously <br />                                                                              identified SL active. <br /> Formulation (SL Finder) min&#226;&#8364;&#8220;max mixed-integer linear program. <br />                                                                                                  yj1 &#195;&#190; yj2 &#195;&#190; ::: &#195;&#190; yjn X1 j1 ; j2 ; :::; jn 2 J                  &#195;&#176;21&#195;&#382; <br /> The inner problem adjusts fluxes achieve maximum biomass <br /> production, subject network stoichiometry, reaction deletions             Note searching set SL reactions particular <br /> imposed outer problem possible growth                   order n, need preclude removal reactions forming lower <br /> environmental constraints. The outer problem hand,              order SLs. This accomplished appending constraints the <br /> aims finding synthetic reaction eliminations lower               following form outer problem bilevel program all <br /> maximum biomass production imposed cutoff. Here,                reactions j1, j2, y, jp forming SL set size p (pon): <br /> split reversible fluxes forward backward reaction steps. To <br />                                                                                          yj1 &#195;&#190; yj2 &#195;&#190; ::: &#195;&#190; yjp X1 j1 ; j2 ; :::; jp 2 J;           8 pon         &#195;&#176;22&#195;&#382; <br /> solve bilevel formulation shown inner maximization is <br /> recast set constraints appending formulation list   It important note elimination certain reactions can <br /> constraints corresponding dual inner problem setting       prevent equation (7) satisfied, precludes the <br />  <br /> &amp; 2009 EMBO Macmillan Publishers Limited                                                                                         Molecular Systems Biology 2009 15 <br />  Genome-scale gene/reaction essentiality synthetic lethality analysis <br /> PF Suthers et al <br />  <br />  <br />  <br /> identification SLs. Merging required non-growth ATP        Dolma S, Lessnick SL, Hahn WC, Stockwell BR (2003) Identification <br /> maintenance biomass equation resolves problem (see              genotype-selective antitumor agents using synthetic lethal <br /> Supplementary information details).                                       chemical screening engineered human tumor cells. Cancer Cell <br />    The formulation (SL Finder) introduced modified            3: 285&#226;&#8364;&#8220;296 <br /> find set SL genes adding set constraints outer <br />                                                                            Feist AM, Henry CS, Reed JL, Krummenacker M, Joyce AR, Karp PD, <br /> problem describing GPR associations. To end, binary variable <br />                                                                               Broadbelt LJ, Hatzimanikatis V, Palsson BO (2007) A genome-scale <br /> wk, representing gene k deleted defined following: <br />                                                                               metabolic reconstruction Escherichia coli K-12 MG1655 that <br />                         0; gene k deleted                               accounts 1260 ORFs thermodynamic information. Mol Syst <br />                wk &#194;&#188;                             ; 8k 2 K            &#195;&#176;23&#195;&#382; <br />                         1; gene k active                                Biol 3: 121 <br />                                                                            Fong SS, Nanchen A, Palsson BO, Sauer U (2006) Latent pathway <br /> The impact gene deletions reaction eliminations GPR             activation increased pathway capacity enable Escherichia coli <br /> relationships mathematically described incorporated into <br />                                                                               adaptation loss key metabolic enzymes. J Biol Chem 281: <br /> the model using appropriate equations relating binary variables <br /> wk yj (see Supplementary information details).                        8024&#226;&#8364;&#8220;8033 <br />                                                                            Forsburg SL (2001) The art design genetic screens: yeast. Nat <br />                                                                               Rev Genet 2: 659&#226;&#8364;&#8220;668 <br />                                                                            Forster J, Famili I, Fu P, Palsson BO, Nielsen J (2003) Genome-scale <br /> Supplementary information                                                     reconstruction Saccharomyces cerevisiae metabolic network. <br /> Supplementary information available Molecular Systems               Genome Res 13: 244&#226;&#8364;&#8220;253 <br /> Biology website (www.nature.com/msb).                                      Gerdes SY, Scholle MD, Campbell JW, Balazsi G, Ravasz E, Daugherty <br />                                                                               MD, Somera AL, Kyrpides NC, Anderson I, Gelfand MS, Bhattacharya <br />                                                                               A, Kapatral V, D0 Souza M, Baev MV, Grechkin Y, Mseeh F, Fonstein <br /> Conflict                                                          MY, Overbeek R, Barabasi AL, Oltvai ZN et al (2003) Experimental <br /> The authors declare conflict interest.                   determination level analysis essential genes in <br />                                                                               Escherichia coli MG1655. J Bacteriol 185: 5673&#226;&#8364;&#8220;5684 <br />                                                                            Ghim CM, Goh KI, Kahng B (2005) Lethality synthetic lethality in <br /> References                                                                    genome-wide metabolic network Escherichia coli. J Theor <br />                                                                               Biol 237: 401&#226;&#8364;&#8220;411 <br /> Ahmed A (1973) Mechanism repression methionine biosynthesis          Grate L, Ares Jr M (2002) Searching yeast intron data Ares lab Web <br />    Escherichia coli. I. The role methionine, s-adenosylmethio-          site. Methods Enzymol 350: 380&#226;&#8364;&#8220;392 <br />    nine, methionyl-transfer ribonucleic acid repression.            Gruer MJ, Bradbury AJ, Guest JR (1997) Construction properties <br />    Mol Gen Genet 123: 299&#226;&#8364;&#8220;324                                                 aconitase mutants Escherichia coli. Microbiology 143 (Part 6): <br /> Alexander K, Young IG (1978) Alternative hydroxylases aerobic         1837&#226;&#8364;&#8220;1846 <br />    anaerobic biosynthesis ubiquinone Escherichia coli.           Guarente L (1993) Synthetic enhancement gene interaction: a <br />    Biochemistry 17: 4750&#226;&#8364;&#8220;4755                                                 genetic tool come age. Trends Genet 9: 362&#226;&#8364;&#8220;366 <br /> Baba T, Ara T, Hasegawa M, Takai Y, Okumura Y, Baba M, Datsenko            Gulmezian M, Hyman KR, Marbois BN, Clarke CF, Javor GT (2007) <br />    KA, Tomita M, Wanner BL, Mori H (2006) Construction                     The role UbiX Escherichia coli coenzyme Q biosynthesis. <br />    Escherichia coli K-12 in-frame, single-<span id='am-165' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-166' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-241' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-242' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span>gene knockout</span> mutants:          Arch Biochem Biophys 467: 144&#226;&#8364;&#8220;153 <br />    Keio collection. Mol Syst Biol 2: 2006.0008                             Hartman JLt, Garvik B, Hartwell L (2001) Principles buffering of <br /> Becker SA, Palsson BO (2008) Three factors underlying incorrect            genetic variation. Science 291: 1001&#226;&#8364;&#8220;1004 <br />    silico predictions essential metabolic genes. BMC Syst Biol 2: 14    Hashimoto M, Kato J (2003) Indispensability Escherichia coli <br /> Behre J, Wilhelm T, von Kamp A, Ruppin E, Schuster S (2008)                   carbonic anhydrases YadF CynT cell proliferation low <br />    Structural robustness metabolic networks respect                CO2 partial pressure. Biosci Biotechnol Biochem 67: 919&#226;&#8364;&#8220;922 <br />    multiple knockouts. J Theor Biol 252: 433&#226;&#8364;&#8220;441                           Hoh J, Ott J (2003) Mathematical multi-locus approaches localizing <br /> Bender A, Pringle JR (1991) Use screen synthetic lethal          complex human trait genes. Nat Rev Genet 4: 701&#226;&#8364;&#8220;709 <br />    multicopy suppressee mutants identify new genes involved         Hove-Jensen B, Rosenkrantz TJ, Haldimann A, Wanner BL (2003) <br />    morphogenesis Saccharomyces cerevisiae. Mol Cell Biol 11:            Escherichia coli phnN, encoding ribose 1,5-bisphosphokinase <br />    1295&#226;&#8364;&#8220;1305                                                                  activity (phosphoribosyl diphosphate forming): dual role in <br /> Burgard AP, Nikolaev EV, Schilling CH, Maranas CD (2004) Flux                 phosphonate degradation NAD biosynthesis pathways. <br />    coupling analysis genome-scale metabolic network                        J Bacteriol 185: 2793&#226;&#8364;&#8220;2801 <br />    reconstructions. Genome Res 14: 301&#226;&#8364;&#8220;312                                 Jordan A, Gibert I, Barbe J (1994a) Cloning sequencing the <br /> Burgard AP, Pharkya P, Maranas CD (2003) Optknock: bilevel                  genes Salmonella typhimurium encoding new bacterial <br />    programming framework identifying gene knockout strategies             ribonucleotide reductase. J Bacteriol 176: 3420&#226;&#8364;&#8220;3427 <br />    microbial strain optimization. Biotechnol Bioeng 84: 647&#226;&#8364;&#8220;657        Jordan A, Pontis E, Atta M, Krook M, Gibert I, Barbe J, Reichard P <br /> Burgard AP, Vaidyaraman S, Maranas CD (2001) Minimal reaction sets            (1994b) A second class I ribonucleotide reductase Enterobacteri- <br />    Escherichia coli metabolism different growth require-            aceae: characterization Salmonella typhimurium enzyme. <br />    ments uptake environments. Biotechnol Prog 17: 791&#226;&#8364;&#8220;797                 Proc Natl Acad Sci USA 91: 12892&#226;&#8364;&#8220;12896 <br /> Chen CY, Graham TR (1998) An arf1Delta synthetic lethal screen             Joyce AR, Reed JL, White A, Edwards R, Osterman A, Baba T, Mori H, <br />    identifies new clathrin heavy chain conditional allele              Lesely SA, Palsson BO, Agarwalla S (2006) Experimental and <br />    perturbs vacuolar protein transport Saccharomyces cerevisiae.           computational assessment conditionally essential genes in <br />    Genetics 150: 577&#226;&#8364;&#8220;589                                                      Escherichia coli. J Bacteriol 188: 8259&#226;&#8364;&#8220;8271 <br /> Covert MW, Knight EM, Reed JL, Herrgard MJ, Palsson BO (2004)              Kaelin WG (2005) The concept synthetic lethality context of <br />    Integrating high-throughput computational data elucidates              anticancer therapy. Nat Rev Cancer 5: 689&#226;&#8364;&#8220;698 <br />    bacterial networks. Nature 429: 92&#226;&#8364;&#8220;96                                   Kamb A (2003) Mutation load, functional overlap, synthetic lethality <br /> Deutscher D, Meilijson I, Kupiec M, Ruppin E (2006) Multiple                  evolution treatment cancer. J Theor Biol 223: 205&#226;&#8364;&#8220;213 <br />    knockout analysis genetic robustness yeast metabolic          Kim PJ, Lee DY, Kim TY, Lee KH, Jeong H, Lee SY, Park S (2007) <br />    network. Nat Genet 38: 993&#226;&#8364;&#8220;998                                             Metabolite essentiality elucidates robustness Escherichia coli <br /> Deutscher D, Meilijson I, Schuster S, Ruppin E (2008) Can single              metabolism. Proc Natl Acad Sci USA 104: 13638&#226;&#8364;&#8220;13642 <br />    knockouts accurately single gene functions? BMC Syst Biol 2: 50     Kuepfer L, Sauer U, Blank LM (2005) Metabolic functions duplicate <br /> Diestel R (2005) Graph Theory, 3rd edn. Berlin, New York: Springer            genes Saccharomyces cerevisiae. Genome Res 15: 1421&#226;&#8364;&#8220;1430 <br />  <br /> 16 Molecular Systems Biology 2009                                                                       &amp; 2009 EMBO Macmillan Publishers Limited <br />                                                                                    Genome-scale gene/reaction essentiality synthetic lethality analysis <br />                                                                                                                                          PF Suthers et al <br />  <br />  <br />  <br />  <br /> Kumar VS, Maranas CD (2009) GrowMatch: automated method             Tatusov RL, Fedorova ND, Jackson JD, Jacobs AR, Kiryutin B, Koonin <br />    reconciling silico/in vivo growth predictions. PLoS Comput Biol         EV, Krylov DM, Mazumder R, Mekhedov SL, Nikolskaya AN, Rao <br />    5: e1000308                                                                BS, Smirnov S, Sverdlov AV, Vasudevan S, Wolf YI, Yin JJ, Natale <br /> Le Meur N, Gentleman R (2008) Modeling synthetic lethality. Genome            DA (2003) The COG database: updated version includes <br />    Biol 9: R135                                                               eukaryotes. BMC Bioinformatics 4: 41 <br /> Lee YJ, Cho JY (2006) Genetic manipulation primary metabolic          Thiele I, Vo TD, Price ND, Palsson BO (2005) Expanded metabolic <br />    pathway L-ornithine production Escherichia coli. Biotechnol         reconstruction Helicobacter pylori (iIT341 GSM/GPR): in <br />    Lett 28: 1849&#226;&#8364;&#8220;1856                                                         silico genome-scale characterization single- double-deletion <br /> Lindner HA, Nadeau G, Matte A, Michel G, Menard R, Cygler M (2005)            mutants. J Bacteriol 187: 5818&#226;&#8364;&#8220;5830 <br />    Site-directed mutagenesis active site region quinate/     Tong AH, Evangelista M, Parsons AB, Xu H, Bader GD, Page N, <br />    shikimate 5-dehydrogenase YdiB Escherichia coli. J Biol Chem            Robinson M, Raghibizadeh S, Hogue CW, Bussey H, Andrews B, <br />    280: 7162&#226;&#8364;&#8220;7169                                                             Tyers M, Boone C (2001) Systematic genetic analysis ordered <br /> Lobner-Olesen A, Marinus MG (1992) Identification gene (aroK)          arrays yeast deletion mutants. Science 294: 2364&#226;&#8364;&#8220;2368 <br />    encoding shikimic acid kinase I Escherichia coli. J Bacteriol 174:   Tong AH, Lesage G, Bader GD, Ding H, Xu H, Xin X, Young J, Berriz GF, <br />    525&#226;&#8364;&#8220;529                                                                    Brost RL, Chang M, Chen Y, Cheng X, Chua G, Friesen H, Goldberg <br /> McCoy AJ, Maurelli AT (2005) Characterization Chlamydia MurC-              DS, Haynes J, Humphries C, He G, Hussein S, Ke L et al (2004) <br />    Ddl, fusion protein exhibiting D-alanyl-D-alanine ligase activity        Global mapping yeast genetic interaction network. Science <br />    involved peptidoglycan synthesis D-cycloserine sensitivity.         303: 808&#226;&#8364;&#8220;813 <br />    Mol Microbiol 57: 41&#226;&#8364;&#8220;52                                                 Tucker CL, Fields S (2003) Lethal combinations. Nat Genet 35: 204&#226;&#8364;&#8220;205 <br /> Meganathan R (2001) Ubiquinone biosynthesis microorganisms.             Typas A, Nichols RJ, Siegele DA, Shales M, Collins SR, Lim B, <br />    FEMS Microbiol Lett 203: 131&#226;&#8364;&#8220;139                                           Braberg H, Yamamoto N, Takeuchi R, Wanner BL, Mori H, <br /> Meredith TC, Woodard RW (2005) Identification GutQ                    Weissman JS, Krogan NJ, Gross CA (2008) High-throughput, <br />    Escherichia coli D-arabinose 5-phosphate isomerase.                   quantitative analyses genetic interactions E. coli. Nat <br />    J Bacteriol 187: 6936&#226;&#8364;&#8220;6942                                                 Methods 5: 781&#226;&#8364;&#8220;787 <br /> Michel G, Roszak AW, Sauve V, Maclean J, Matte A, Coggins JR, Cygler       Urbanowski ML, Stauffer LT, Plamann LS, Stauffer GV (1987) A new <br />    M, Lapthorn AJ (2003) Structures shikimate dehydrogenase AroE           methionine locus, metR, encodes trans-acting protein <br />    Paralog YdiB. A common structural framework different          required activation metE metH Escherichia coli and <br />    activities. J Biol Chem 278: 19463&#226;&#8364;&#8220;19472                                   Salmonella typhimurium. J Bacteriol 169: 1391&#226;&#8364;&#8220;1397 <br /> Mullen JR, Kaliraman V, Ibrahim SS, Brill SJ (2001) Requirement        Wang T, Bretscher A (1997) Mutations synthetically lethal with <br />    novel protein complexes absence Sgs1 DNA               tpm1delta lie genes involved morphogenesis. Genetics 147: <br />    helicase Saccharomyces cerevisiae. Genetics 157: 103&#226;&#8364;&#8220;118                1595&#226;&#8364;&#8220;1607 <br /> Neidhardt FC, Curtiss R (1996) Escherichia coli Salmonella:            Wild J, Hennig J, Lobocka M, Walczak W, Klopotowski T (1985) <br />    Cellular Molecular Biology, 2nd edn. Washington, DC: ASM               Identification dadX gene coding predominant isozyme <br />    Press                                                                      alanine racemase Escherichia coli K12. Mol Gen Genet 198: <br /> Novick P, Osmond BC, Botstein D (1989) Suppressors yeast actin             315&#226;&#8364;&#8220;322 <br />    mutations. Genetics 121: 659&#226;&#8364;&#8220;674                                        Wilhelm T, Behre J, Schuster S (2004) Analysis structural robustness <br /> Nygaard P, Smith JM (1993) Evidence novel glycinamide                   metabolic networks. Syst Biol (Stevenage) 1: 114&#226;&#8364;&#8220;120 <br />    ribonucleotide transformylase Escherichia coli. J Bacteriol 175:     Willemoes M, Kilstrup M (2005) Nucleoside triphosphate synthesis <br />    3591&#226;&#8364;&#8220;3597                                                                  catalysed adenylate kinase ADP dependent. Arch Biochem <br /> Ooi SL, Pan X, Peyser BD, Ye P, Meluh PB, Yuan DS, Irizarry RA, Bader         Biophys 444: 195&#226;&#8364;&#8220;199 <br />    JS, Spencer FA, Boeke JD (2006) Global synthetic-lethality analysis     Wong SL, Zhang LV, Tong AH, Li Z, Goldberg DS, King OD, Lesage G, <br />    yeast functional profiling. Trends Genet 22: 56&#226;&#8364;&#8220;63                     Vidal M, Andrews B, Bussey H, Boone C, Roth FP (2004) Combining <br /> Ooi SL, Shoemaker DD, Boeke JD (2003) DNA helicase gene                       biological networks predict genetic interactions. Proc Natl Acad <br />    interaction network defined using synthetic lethality analyzed          Sci USA 101: 15682&#226;&#8364;&#8220;15687 <br />    microarray. Nat Genet 35: 277&#226;&#8364;&#8220;286                                       Wunderlich Z, Mirny LA (2006) Using topology metabolic <br /> Palumbo MC, Colosimo A, Giuliani A, Farina L (2005) Functional                networks predict viability mutant strains. Biophys J 91: <br />    essentiality topology features metabolic networks: case          2304&#226;&#8364;&#8220;2311 <br />    study yeast. FEBS Lett 579: 4642&#226;&#8364;&#8220;4646                                Ye P, Peyser BD, Pan X, Boeke JD, Spencer FA, Bader JS (2005) Gene <br /> Palumbo MC, Colosimo A, Gi <br /> </body></html>