<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                            NIH Public Access <br />                             Author Manuscript <br />                             Free Radic Res. Author manuscript; available PMC 2009 May 5. <br />                            Published final edited form as: <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             Free Radic Res. 2008 January ; 42(1): 20&#226;&#8364;&#8220;29. doi:10.1080/10715760701765616. <br />  <br />  <br />  <br />                            APE1/Ref-1 regulates PTEN expression mediated Egr-1 <br />  <br />                            DAMIANO FANTINI1,*, CARLO VASCOTTO1,*, MARTA DEGANUTO1, NICOLETTA BIVI1, <br />                            STEFANO GUSTINCICH2, GABRIELLA MARCON3, FRANCO QUADRIFOGLIO1, GIUSEPPE <br />                            DAMANTE1, KISHOR K. BHAKAT4, SANKAR MITRA4, GIANLUCA TELL1 <br />                            1Department Biomedical Sciences Technologies, University Udine, 33100 Udine, Italy <br />                            2Sector Neurobiology, International School Advanced Studies (SISSA), 34014 Trieste, Italy <br />                            3Department Pathology Experimental Clinical Medicine, University Udine, 33100 Udine, Italy <br />                            4Sealy Center Molecular Medicine Department Biochemistry Molecular Biology, University <br />                            Texas Medical Branch, Galveston, TX 77555, USA <br />  <br />                            <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                 APE1/Ref-1, mammalian ortholog E. coli Xth, multifunctional protein possessing both <br />                                 DNA repair transcriptional regulatory activities, dual role controlling cellular response <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> <br />                                 oxidative stress. It rate-limiting repair oxidative DNA damage including strand breaks and <br />                                 co-transcriptional activity modulating genes expression directly regulated Egr-1 and <br />                                 p53 transcription factors. PTEN, phosphoinositide phosphatase, acts `off&apos; switch PI-3 <br />                                 kinase/Akt signalling pathway regulates cell growth <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span>survival</span>. It shown transient <br />                                 alteration APE1 level HeLa cells modulates PTEN expression acetylatable APE1 <br />                                 required activation PTEN gene. Acetylation APE1 enhances binding distinct <br />                                 trans-acting complexes involved activation repression. The acetylated protein deacetylated <br />                                 vivo histone deacetylases. It exposure HeLa cells H2O2 histone <br />                                 deacetylase inhibitors increases acetylation APE1 induction PTEN. The absence such <br />                                 induction APE1-downregulated HeLa cells confirmed APE1&apos;s role regulating inducible PTEN <br />                                 expression. That APE1-dependent PTEN expression mediated Egr-1 supported by <br />                                 experiments cells ectopically expressing Egr-1. Thus, data open new perspectives the <br />                                 comprehension functions exerted APE1 controlling cell response oxidative <br />                                 stress. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            Keywords <br />                                 APE1/Ref-1; PTEN; Egr-1; siRNA; histone acetyltransferase inhibitors; oxidative stress response <br />  <br />  <br />  <br />  <br />                            Correspondence: Gianluca Tell, Professor Molecular Biology, Department Biomedical Sciences Technologies, University of <br />                            Udine, P.le Kolbe 4, 33100 Udine, Italy. Tel: ++39 432 494311. Fax: ++39 432 494301. Email: E-mail: gtell@mail.dstb.uniud.it. <br />                            *Both authors contributed equally work. <br />                            Publisher&apos;s Disclaimer: This article used research, teaching private study purposes. Any substantial systematic <br />                            reproduction, re-distribution, re-selling, loan sub-licensing, systematic supply distribution form expressly <br />                            forbidden. <br />                            The publisher does warranty express implied make representation contents complete accurate <br />                            date. The accuracy instructions, formulae drug doses independently verified primary sources. The <br />                            publisher shall liable loss, actions, claims, proceedings, demand costs damages whatsoever howsoever caused <br />                            arising directly indirectly connection arising use material. <br />                             FANTINI et al.                                                                                               Page 2 <br />  <br />  <br />                               Introduction <br />                                             The APE1/Ref-1 protein (<span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> simply indicated APE1) plays central role the <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             highly regulated process cellular response oxidative stress [1,2]. Its activation of <br />                                             complex network cellular events determines final outcome, cell growth <br />                                             arrest, death survival cells exposed oxidative stress [1,3-6]. In addition well- <br />                                             characterized toxic effects, mild oxidative stress activates survival/proliferative signalling <br />                                             [7]. Thus, tight temporal control extent APE1 activation response oxidative <br />                                             stress modulate cell growth survival. <br />  <br />                                             APE1 ubiquitous multifunctional protein possessing DNA repair transcriptional <br />                                             regulatory activities. APE1 enhances DNA binding number transcription factors <br />                                             including p53, Egr-1 NF-&#206;&#186;B [1,2,8-11] acting transcriptional co-activator. Egr-1 is <br />                                             involved regulation cell proliferation response extracellular signal as <br />                                             mitogens growth factors, oxidative stress [12,13]. Studies gene expression in <br />                                             human tumour cells tissues support Egr-1&apos;s function tumour suppressor [14,15]. Egr-1 <br />                                             regulates expression genes p53 phosphoinositol phosphatase tensin <br />                                             homologue (PTEN) control cell-growth arrest apoptosis 16,17]. PTEN negatively <br />                                             regulates phosphoinositide 3-kinase/Akt signalling pathway, shown be <br />                                             involved regulation cell growth survival cell lines [18]. Thus, induction of <br />                                             PTEN activity lead inhibition cell growth ultimately apoptosis. We previously <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             observed PTEN promoter regulated Egr-1 APE1-dependent manner <br />                                             [11], suggesting functional relationship proteins growth suppressive <br />                                             effects response oxidative stress. We shown PTEN expression activated <br />                                             oxidative stress [19]. The regulatory functions different APE1 activities be <br />                                             implemented different mechanisms: (a) APE1&apos;s relocalization cytoplasm to <br />                                             nucleus; (b) increase APE1&apos;s level transcriptional activation [1,5,11,20]; (c) <br />                                             APE1&apos;s post-translational modifications (such acetylation phosphorylation). As recently <br />                                             demonstrated [6,21], acetylation appears fine-tuning role affecting APE1&apos;s different <br />                                             activities. <br />  <br />                                             While known nuclear accumulation APE1 triggers activation transcription <br />                                             factors, Egr-1 p53, functional role acetylation barely understood. <br />                                             Acetylation histones regulatory proteins commonly catalysed histone <br />                                             acetyltransferase (HAT) p300/CBP reversed histone deacetylases (HDACs), <br />                                             control acetylation level transcription factors co-activators [22-24]. We <br />                                             observed balance acetyltransferase activity p300/CBP the <br />                                             deacetylase activity HDAC1 maintains APE1&apos;s acetylation Lys residues 6 7 (K6, K7) <br />                                             response Ca2+ levels, controlling expression target genes [6]. Inhibition HDACs <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             HDACs inhibitors causes proteins hyperacetylation [25]. A correlation established <br />                                             hyperacetylated histones transcriptionally active chromatin [26,27]. Moreover, <br />                                             recent studies indicate potential role HDACs inhibitors promoting differentiation of <br />                                             tumour cells, suggesting possible use anticancer agents [28,29]. <br />  <br />                                             The purpose present study investigate relationship APE1, its <br />                                             acetylated form PTEN expression. We observed direct correlation levels of <br />                                             APE1 PTEN suggesting functional link. Furthermore, PTEN upregulated by <br />                                             treatment H2O2 HDACs inhibitors concomitant enhanced acetylation APE1. <br />                                             APE1-mediated PTEN activation appears involve transcription factor Egr-1. <br />                                             Collectively, studies unravelled novel, indirect, regulatory function of <br />                                             APE1 PTEN activation, opening new perspectives comprehension many <br />                                             functions exerted multifunctional protein. <br />  <br />  <br />  <br />                                                  Free Radic Res. Author manuscript; available PMC 2009 May 5. <br />                             FANTINI et al.                                                                                            Page 3 <br />  <br />  <br />                               Materials methods <br />                               Chemicals <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Trichostatin A (TSA), sodium butyrate (NaBut) chemicals purchased from <br />                                             Sigma (Milan, Italy), unless specified. HeLa cells purchased ATCC and <br />                                             HCT116 isogenic cell lines expressing p53 (p53+/+) non-expressing p53 (p53-/-) gifts <br />                                             B. Vogelstein (Johns Hopkins University, Baltimore, MD). The p53 gene inactivated <br />                                             HCT116 p53-/- cells homologous recombination. Briefly, promoterless targeting <br />                                             vectors containing geneticin hygromycin resistance gene place genomic p53 <br />                                             sequences sequentially transfected HCT116 p53+/+ cells disrupt p53 alleles <br />                                             [30]. The <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span>phenotype</span> cell line stable determined periodically Western blot <br />                                             analysis. <br />  <br />                               Cell culture analysis <br />                                             HeLa cells grown Dulbecco&apos;s Modified Eagle&apos;s Medium High Glucose (DMEMHG) <br />                                             supplemented 10% (v/v) foetal bovine serum (FBS), 2 mM L-glutamine 1% penicillin- <br />                                             streptomycin (100,000 Units/L penicillin 100 mg/L streptomycin) 37&#194;&#176;C 5% CO2 and <br />                                             95% O2. HCT116p53+/+ HCT116p53-/- human colon carcinoma isogenic cell lines were <br />                                             cultured McCoy&apos;s 5A medium supplemented 10% FBS penicillin/streptomycin, as <br />                                             previously reported [6]. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Cellular extracts typically prepared lysing 3 &#195;&#8212; 105 cells 50 &#206;&#188;L lysis buffer <br />                                             containing 50 mM Tris-HCl, (pH 7.5), 150 mM NaCl, 1% w/v Triton X-100, 1 mM EDTA, 0.5 <br />                                             mM PMSF, 0.1 mM DTT, 1 mM NaVO3, 1 mM NaF, protease inhibitor cocktail (Sigma), then <br />                                             analysed protein content [31] stored aliquots -80&#194;&#176;C. <br />  <br />                               Western blot analysis <br />                                             Cell extracts analysed SDS-PAGE (10% polyacrylamide) [20] separated <br />                                             proteins transferred nitrocellulose membranes (Schleicher &amp; Schuell). After pre-incubating <br />                                             5% non-fat dry milk PBS/0.1% w/v Tween-20 1 h, membranes incubated <br />                                             anti-APE1 monoclonal antibody [20] 3 h room temperature polyclonal <br />                                             anti-PTEN antibody (Santa Cruz Biotechnology Inc., Santa Cruz, CA) overnight 4&#194;&#176;C. The <br />                                             acetylated APE1 (AcAPE1) determined using anti-acetylated APE1 peptide- <br />                                             specific polyclonal antibody, previously described [32]. After incubation, membranes <br />                                             washed times PBS/0.1% w/v Tween-20 incubated appropriate <br />                                             secondary antibodies coupled horseradish peroxidase (Sigma) 1 h room temperature <br />                                             followed washes PBS Tween-20 0.1% w/v. At end, blots developed <br />                                             using ECL chemiluminescence procedure (Amersham Pharmacia Biotech, Milan, Italy) <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             quantified using Gel Doc 2000 videodensitometer (Bio-Rad, Hercules, CA, USA). APE1, <br />                                             AcAPE1 PTEN signals normalized actin quantitated blot with <br />                                             polyclonal anti-actin antibody (Sigma). <br />  <br />                               Transfection studies <br />                                             HCT116 cells (0.3 &#195;&#8212; 106 cells/100 mm dish) cotransfected using Lipofectamine 2000 <br />                                             (Invitrogen, Milan, Italy) according manufacturer&apos;s protocol various luciferase <br />                                             reporter plasmids. A &#206;&#178;-galactosidase expression plasmid (CMV-&#206;&#178;-Gal) used internal <br />                                             control normalizing transfection efficiency (Roche Diagnostic, Milan, Italy). Luciferase <br />                                             (Luc) activity measured using chemiluminescence procedure [11] normalized to <br />                                             &#206;&#178;-galactosidase activity. In cases, promoter activity &#206;&#178;-galactosidase <br />                                             expression plasmid affected, luciferase activity normalized total protein <br />                                             extract. For studies PTEN promoter constructs, used 0.5 &#206;&#188;g plasmid <br />  <br />  <br />                                                  Free Radic Res. Author manuscript; available PMC 2009 May 5. <br />                             FANTINI et al.                                                                                               Page 4 <br />  <br />  <br />                                             containing 2 kb (-1 -1978 bp) human PTEN promoter (PTEN-Luc) cloned into <br />                                             pGL3B-P10, 5&#226;&#8364;&#178; Luc gene, described earlier [16]. The pcDNA5.1-APE1-c-FLAG <br />                                             plasmid, encoding C-terminal FLAG-tagged APE1 used over-express APE1; as <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             control used pcDNA5.1-empty vector. The Egr-1-expressing plasmid pRSV-Egr-1 was <br />                                             described earlier [33]. <br />  <br />                               Transient inducible expression APE1 siRNA <br />                                             The oligonucleotides used siRNA APE1 using Mittal&apos;s empirical rule [34] (sense, 5&#226;&#8364;&#178;- <br />                                             CCTGCCA CACTCAAGATCTGC-3&#226;&#8364;&#178;; antisense, 5&#226;&#8364;&#178;-GCAGAT CTTGAGTGTGGCAGG-3&#226;&#8364;&#178;) <br />                                             designed bind APE1&apos;s 21 nt mRNA segment 175 nt downstream initiation codon. <br />                                             The sequences cloned BglII HindIII restriction sites pSUPER identity <br />                                             resulting plasmid pSUPER/Ref-1 confirmed direct sequencing. HeLa cells were <br />                                             transfected pSUPER/Ref-1 presence lipofectamine and, 48 h later, cells were <br />                                             processed analysis. <br />  <br />                                             For inducible down-regulation APE1 expression, siRNA sequences cloned <br />                                             pTER vector [35] tetracycline- (doxycycline-) responsive promoter generate the <br />                                             pTER/Ref-1 plasmid. HeLa cells transfected pcDNA6/TR, linearized Bst1107I <br />                                             (Fermentas, St.Leon Rot, UK) incubated blasticidin (Invitrogen) 14 days to <br />                                             select resistant cells. After isolating individual clones using cloning cylinders (Sigma), the <br />                                             clones expanded 107 cells. The TR5 clone high level expression Tet- <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             repressor transfected pTER/Ref-1 pre-digested Bst1107I (Fermentas) then <br />                                             selected resistance zeocine 14-21 days&apos; culture (Invitrogen). Thirty clones then <br />                                             grown 107 cells, control, pTER-transfected cells. To induce APE1 siRNA <br />                                             cells, 1 &#206;&#188;/mL doxycycline (Sigma) added medium cells grown <br />                                             10 days analysed APE1 expression. <br />  <br />                               Statistical analysis <br />                                             Results expressed mean&#194;&#177;SD. Each data point represents average three <br />                                             measurements, performed independent cell preparations. Statistical <br />                                             significance means&#194;&#177;SD analysed Student&apos;s t-test two-way analysis of <br />                                             variance (ANOVA) differences considered significant p&lt;0.05. <br />  <br />                               Results <br />                               APE1-dependent regulation PTEN expression <br />                                             We previously shown cotransfection experiments PTEN promoter activity was <br />                                             significantly enhanced HeLa cells over-expression APE1 [11], suggesting the <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             PTEN gene regulated APE1. In order confirm direct correlation APE1 <br />                                             level PTEN expression, examined effect down-regulation endogenous APE1 <br />                                             PTEN gene. APE1-specific small interfering RNA (siRNA), expressed plasmid, <br />                                             used reduce level endogenous APE1. HeLa cells transiently transfected with <br />                                             control siRNA APE1 specific siRNA-expressing plasmid (pSUPER/Ref-1) the <br />                                             expression levels APE1 PTEN measured Western analysis 48 h after <br />                                             transfection. Expression APE1 siRNA resulted significant decrease (~80%) the <br />                                             APE1 level, compared control cells transfection plasmid (Figure <br />                                             1A, left). At time, PTEN level reduced 50% APE1 knocked-down <br />                                             cells. <br />  <br />                                             To confirm PTEN regulatory target ~APE1, over-expressed FLAG-tagged APE1 <br />                                             HeLa cells. The PTEN level significantly increased APE1-over-expressing cells <br />                                             relative control (Figure 1A, right). <br />  <br />  <br />                                                  Free Radic Res. Author manuscript; available PMC 2009 May 5. <br />                             FANTINI et al.                                                                                             Page 5 <br />  <br />  <br />                                             APE1&apos;s regulatory activity fine-tuned various mechanisms including post- <br />                                             translational modification, redox regulation target trans-acting factor nucleo- <br />                                             cytoplasmic trafficking protein itself [1,2]. We recently demonstrated that <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             acetylation APE1 specific Lys residues (i.e. K6/K7) constitutes fundamental <br />                                             mechanism modulate regulation Ca2+-dependent promoters containing negative <br />                                             Ca2+ responsive elements (nCaRE) [6]. Therefore, tested acetylatable K6/K7 <br />                                             residues APE1 involved PTEN expression. We analysed activation PTEN promoter- <br />                                             driven luciferase expression mediated wild type APE1 non-acetylatable R6/ <br />                                             R7 mutant [6]. Figure 1B (left panel) shows over-expression wild type APE1 <br />                                             protein increased activity PTEN promoter ~2-fold, expression R6/R7 mutant <br />                                             caused marginal activation conditions comparable ectopic expression wild type vs <br />                                             mutant APE1. As expected, transient over-expression non-acetylatable APE1 protein <br />                                             unable promote endogenous PTEN expression (Figure 1B, right panel). These data <br />                                             suggest acetylated lysine residues APE1 involved APE1-mediated PTEN <br />                                             expression. <br />  <br />                               PTEN activation APE1 acetylation induced H2O2 <br />                                             Expression PTEN, early regulator cellular growth suppression, enhanced thyroid <br />                                             cells oxidative stress [19]. Here, tested H2O2 treatment similarly induce <br />                                             PTEN HeLa cells. After 1 h serum starvation, HeLa cells treated various times with <br />                                             50 &#206;&#188;M H2O2. As shown Figure 2A, H2O2 treatment increased endogenous PTEN protein <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             level time-dependent manner. <br />  <br />                                             Then, tested possibility oxidative stress stimulates acetylation APE1 in <br />                                             turn activates PTEN promoter. We quantitate acetylated APE1 by <br />                                             using polyclonal antibody selectively recognizes AcAPE1. The antibody raised <br />                                             peptide containing acetylated Lys residues corresponding position 6 APE1. This <br />                                             polyclonal rabbit antibody does recognize 50-fold excess unmodified APE1 (data <br />                                             shown). Figure 2B shows H2O2 treatment significantly increased level AcAPE1, <br />                                             occurred early (30 min) treatment. Estimation total APE1, using a <br />                                             monoclonal antibody [20], indicated level significantly affected H2O2 under <br />                                             conditions. <br />  <br />                               Elevated level acetylated APE1 PTEN upregulation HDACs inhibition <br />                                             We recently demonstrated APE1 acetylation mediated p300/CBP histone <br />                                             acetyltransferase HDACs inhibition moderately enhanced level AcAPE1 in <br />                                             HCT116 cells [6]. Thus, tested HDACs inhibition affect AcAPE1 levels <br />                                             HeLa cells. We treated HeLa cells HDACs inhibitors, namely, NaBut TSA. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             The longer time-frame used respect previous experiment Figure was <br />                                             chosen basis activation PTEN induction HDACs inhibitors (data shown). <br />                                             Figure 3A shows tors significantly increased APE1, albeit different <br />                                             extent, total APE1. The increase levels TSA treatment was <br />                                             visible time points (data shown). <br />  <br />                                             Then, tested HDACs inhibitors gulate PTEN expression. HeLa cells the <br />                                             HDACs inhibitors 30 h level quantitated Western blotting. Figure shows <br />                                             NaBut TSA increased PTEN protein level. <br />  <br />                               HDACs inhibitor-induced PTEN upregulation requires APE1 expression <br />                                             The causal involvement APE1 upregulation PTEN expression tested by <br />                                             using HeLa cells expression endogenous APE1 conditionally knocked- <br />                                             stable transfection inducible APE1-specific siRNA expression [35]. After 10 <br />  <br />  <br />                                                  Free Radic Res. Author manuscript; available PMC 2009 May 5. <br />                             FANTINI et al.                                                                                             Page 6 <br />  <br />  <br />                                             days&apos; treatment doxycycline, APE1 expression reached undetectable level in <br />                                             HeLa cells expressing APE1 specific siRNA; control cells, transfected siRNA <br />                                             vector (Control) unaffected (Figure 4A, inset). To test effect HDACs <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             inhibition PTEN gene expression, measured luciferase activity control 2- <br />                                             kb PTEN promoter [16]. Figure 4A shows TSA treatment caused ~5-fold increase Luc <br />                                             activity control cells NaBut increased activity ~11-fold relative untreated <br />                                             cells. In APE1 siRNA-expressing cells, TSA treatment caused 2-fold NaBut only <br />                                             3.5-fold increase PTEN promoter activity. <br />  <br />                                             These results, showing silencing APE1 prevents activation PTEN promoter caused <br />                                             HDACs inhibition, support major role acetylated APE1 transcriptional activation of <br />                                             PTEN gene. <br />  <br />                                             We tested HDACs inhibition causes upregulation endogenous PTEN <br />                                             protein mediation APE1. We measured PTEN protein levels control APE1 <br />                                             knock-down cells treatment NaBut TSA (Figure 4B). It evident PTEN <br />                                             levels increased time-dependent fashion HDAC inhibited response <br />                                             completely blocked APE1 knock-down cells (Figure 4B). Taken together, data <br />                                             unequivocally demonstrate essential role APE1 PTEN expression. <br />  <br />                                             The unexpected observation HeLa APE1 knocked-down cells present higher basal level <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             PTEN protein explained complex transcriptional mechanisms controlling <br />                                             PTEN expression, involve transcription factors p53 [36], CBF-1 <br />                                             [37], NF-&#206;&#186;B [38] PPAR&#206;&#179; [39], excluding possible role protein stabilization. Thus, <br />                                             data clearly point role APE1 controlling inducible constitutive PTEN <br />                                             expression. <br />  <br />                               APE1-mediated PTEN activation requires Egr-1 p53 <br />                                             Transcriptional activation PTEN control transcription factors, such <br />                                             p53 Egr-1 known co-activated APE1 [8,10,16,18,36]. While p53 is <br />                                             responsible basal PTEN expression, Egr-1 primarily activates PTEN expression response <br />                                             cellular stress [16]. We tested involvement p53 Egr-1 APE1-mediated PTEN <br />                                             expression, using nearly isogenic pair human colon carcinoma cell line HCT116 <br />                                             expressing wild type p53. The PTEN-Luc promoter contains 117 bp GC-rich sequence <br />                                             (-1031 -779 bp), minimal Egr-1 responsive element, includes Egr-1 binding <br />                                             sites [16]. This promoter contains p53 response element upstream Egr-1 binding <br />                                             sites (Figure 5A). HCT116-/- HCT116+/+ cells transfected plasmids encoding <br />                                             wild type Egr-1 p53 effects PTEN promoter analysed cells <br />                                             ectopically expressing APE1 (Figure 5B). It evident APE1 over-expression caused <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             significant (~2-fold) similar activation PTEN promoter cell lines, indicating <br />                                             stimulatory activity APE1 independent p53 status depends Egr-1. <br />                                             To confirm lack involvement p53 acetylated APE1-dependent PTEN <br />                                             upregulation, effect HDACs inhibitors endogenous PTEN expression HCT116-/- <br />                                             HCT116+/+ cells examined (Figure 5C). Western blot analysis total cell extracts <br />                                             showed increase PTEN expression dependent HDACs independent of <br />                                             p53 status cell. <br />  <br />                               Discussion <br />                                             APE1 distinct functions: DNA repair transcriptional regulation [1,2]. The <br />                                             essentiality APE1 cellular survival frequent over-expression tumour cells <br />                                             strongly suggest APE1&apos;s role preventing cell death controlling cellular proliferation <br />                                             [4,21]. However, ability activate transcription factors, p53 Egr-1 [10,11,40, <br />  <br />  <br />                                                  Free Radic Res. Author manuscript; available PMC 2009 May 5. <br />                             FANTINI et al.                                                                                                   Page 7 <br />  <br />  <br />                                             41], mainly involved controlling cell-cycle arrest apoptosis, raises possibility <br />                                             opposite regulatory roles APE1 different cellular contexts. Thus, APE1 act a <br />                                             transcriptional co-activator co-repressor distinct transcription factors. It also <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             possible different functions organized hierarchical manner. <br />  <br />                                             In present study, tested PTEN, Egr-1 target gene [10,16], regulated by <br />                                             APE1 previously suggested [11]. The experiments involving ectopic APE1 <br />                                             expression down-regulation endogenous gene indicated inducible PTEN <br />                                             expression regulated APE1. As post-translational acetylation APE1 shown <br />                                             repress PTH promoter binding nCaRE [6], checked acetylation of <br />                                             APE1 control PTEN inducible activation. While over-expression wild-type <br />                                             APE1 caused PTEN promoter activation, similar ectopic expression non-acetylatable <br />                                             R6/R7 mutant marginal effect, suggesting acetylation APE1is involved PTEN <br />                                             activation. Interestingly, oxidative stress, known induce APE1 expression, enhanced APE1 <br />                                             acetylation concomitant stimulation PTEN expression. Complementarily, treatment <br />                                             HDACs inhibitors, TSA NaBut, increased acetylation level APE1 with <br />                                             enhanced PTEN expression, suggesting induction PTEN mediated APE1. These <br />                                             conclusions confirmed co-transfection studies PTEN expression was <br />                                             examined APE1 knocked-down cells. Experiments HDACs-inhibitors further <br />                                             confirmed transcriptional activation responsible PTEN over-expression. It known <br />                                             oxidative stress promotes PTEN functional inactivation cysteine oxidation [42]. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Thus, apparent paradox induction PTEN expression oxidative stress, we <br />                                             observed HeLa thyroid cells [19] osteoblasts (data shown), be <br />                                             explained terms cellular adaptive mechanisms aiming homeostatic regulation of <br />                                             endogenous PTEN functional levels. Further work needed address issue. <br />  <br />                                             The PTEN promoter controlled Egr-1 p53 [36]. However, Egr-1 <br />                                             involved primarily stress-mediated induction PTEN gene [16]. Our results, using <br />                                             HCT116 cell lines nearly isogenic p53, demonstrated APE1-mediated PTEN <br />                                             expression dependent trans-acting function Egr-1 p53. <br />                                             While work progress, Pan et al. [43] demonstrated PTEN expression was <br />                                             induced histone deacetylase inhibitor TSA treatment Egr-1 activation. Our data provide <br />                                             novel molecular basis activation, suggesting upstream regulatory role of <br />                                             acetylated APE1 Egr-1 activation. The precise mechanism APE1&apos;s action Egr-1-driven <br />                                             PTEN expression unknown. Our attempts demonstrate stable interaction between <br />                                             acetylated APE1 Egr-1 unsuccessful, similarly case observed p53 [8]. <br />                                             However, notion APE1 redox-regulate Egr-1 [10,11] supports existence a <br />                                             transient weak interaction factors. We currently investigating role <br />                                             APE1 acetylation modulating interaction Egr-1. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             The double-function APE1 gene regulation example apparent biological <br />                                             paradox. In fact, number papers clearly demonstrated APE1&apos;s anti-apoptotic roles <br />                                             positive effect cell proliferation [1,2,4,21], evidence its <br />                                             potential role controlling proapoptotic functions p53-mediated activation [8,40, <br />                                             41] p21, leading cell cycle arrest inhibiting cyclin-dependent kinase [44] cyclin <br />                                             G [45]. It evident anti-apoptotic roles APE1 ascribable DNA-repair <br />                                             functions [4], activities transcriptional co-activator (Figure 6). Thus, is <br />                                             tempting speculate function, activation transcription factors p53 and <br />                                             Egr-1, ensures efficient cell-cycle arrest, act concert repair DNA damage <br />                                             protect cells accumulation oxidative damage. Obviously, proper modulation of <br />                                             interconnected functions fine-tuning needed regulating APE1 activities. Thus, <br />                                             in-depth understanding processes controlling: (i) APE1&apos;s subcellular distribution; (ii) <br />                                             post-translational modifications; (iii) turnover (iv) recruitment diverse <br />  <br />  <br />                                                  Free Radic Res. Author manuscript; available PMC 2009 May 5. <br />                             FANTINI et al.                                                                                                             Page 8 <br />  <br />  <br />                                             interacting partners response various endogenous external signals, required unravel <br />                                             complex picture. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Interestingly, dual role proteins transcriptional regulation DNA repair diverse <br />                                             pathways common phenomenon, observed cases BRCA1, ATM p53. <br />                                             These regulatory proteins involved homologous recombinational repair (HR), non- <br />                                             homologous end joining (NHEJ), nucleotide excision repair (NER), <span id='am-316' about='xsp:base' typeof='owl:Thing'>base</span> excision repair (BER) <br />                                             mismatch repair (MMR) [46]. The existence precise regulation switching <br />                                             mechanism shifting cells DNA repair mode apoptosis central avoidance of <br />                                             cancer progression, preventing clonal expansion cells unrepaired damage would <br />                                             lead <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span>mutation</span> transformations. It interesting cross-regulation, <br />                                             terms expression, does exist partners different pathways. <br />  <br />                               Acknowledgements <br />                                             The financial support Telethon - Italy (grants GGP06268 GGP05062) MUR (PRIN 2005 GT FQ) <br />                                             gratefully acknowledged. SM KKB supported USPHS grants R01 CA53791 R01 ES08457. The <br />                                             authors wish thank E. Adamson (Burnham Institute, USA) PTEN promoter Egr-1 expressing plasmids <br />                                             Ms W. Smith secretarial editing assistance. <br />  <br />  <br />                               Abbreviations <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             AP-1, activator protein-1; APE1/Ref-1, Apurinic/apyrimidinic (AP) endonuclease/redox <br />                                             effector factor-1; Egr-1, Early growth response protein-1; HATs, Histone acetyltransferases; <br />                                             HDACs, Histone deacetylases; NaBut, sodium butyrate; NF-&#206;&#186;B, Nuclear Factor-&#206;&#186;B; PTEN, <br />                                             protein phosphatase tensin homologue; ROS, reactive oxygen species; TSA, trichostatin <br />                                             A. <br />  <br />                               References <br />                                             [1]. Tell G, Damante G, Caldwell D, Kelley MR. The intracellular localization APE1/Ref-1: than <br />                                                   passive phenomenon? Antioxid Redox Signal 2005;7:367&#226;&#8364;&#8220;384. [PubMed: 15706084] <br />                                             [2]. Mitra S, Izumi T, Boldogh I, Bhakat KK, Chattopadhyay R, Szczesny B. Intracellular trafficking and <br />                                                   regulation mammalian AP-endonuclease 1 (APE1), essential DNA repair protein. DNA Repair <br />                                                   (Amst) 2007;6:461&#226;&#8364;&#8220;469. [PubMed: 17166779] <br />                                             [3]. Robertson KA, Hill DP, Xu Y, Liu L, Van Epps S, Hockenbery DM, Park JR, Wilson TM, Kelley <br />                                                   MR. Down-regulation apurinic/apyrimidinic endonuclease expression associated the <br />                                                   induction apoptosis differentiating myeloid leukemia cells. Cell Growth Differ 1997;8:443&#226;&#8364;&#8220; <br />                                                   449. [PubMed: 9101090] <br />                                             [4]. Fung H, Demple B. A vital role Ape1/Ref1 protein repairing spontaneous DNA damage in <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   human cells. Mol Cell 2005;17:463&#226;&#8364;&#8220;470. [PubMed: 15694346] <br />                                             [5]. Ramana CV, Boldogh I, Izumi T, Mitra S. Activation apurinic/apyrimidinic endonuclease human <br />                                                   cells reactive oxygen species correlation adaptive response genotoxicity of <br />                                                   free radicals. Proc Natl Acad Sci USA 1998;95:5061&#226;&#8364;&#8220;5066. [PubMed: 9560228] <br />                                             [6]. Bhakat KK, Izumi T, Yang SH, Hazra TK, Mitra S. Role acetylated human AP-endonuclease <br />                                                   (APE1/Ref-1) regulation parathyroid hormone gene. EMBO J 2003;22:6299&#226;&#8364;&#8220;6309. <br />                                                   [PubMed: 14633989] <br />                                             [7]. Finkel T. Oxidant signals oxidative stress. Curr Opin Cell Biol 2003;15:247&#226;&#8364;&#8220;254. [PubMed: <br />                                                   12648682] <br />                                             [8]. Gaiddon C, Moorthy NC, Prives C. Ref-1 regulates transactivation pro-apoptotic functions <br />                                                   p53 vivo. EMBO J 1999;18:5609&#226;&#8364;&#8220;5621. [PubMed: 10523305] <br />                                             [9]. Xanthoudakis S, Curran T. Identification characterization Ref-1, nuclear protein that <br />                                                   facilitates AP-1 DNA-binding activity. EMBO J 1992;11:653&#226;&#8364;&#8220;665. [PubMed: 1537340] <br />  <br />  <br />  <br />  <br />                                                   Free Radic Res. Author manuscript; available PMC 2009 May 5. <br />                             FANTINI et al.                                                                                                    Page 9 <br />  <br />  <br />                                             [10]. Huang RP, Adamson ED. Characterization DNA binding properties early growth <br />                                                   response-1 (Egr-1) transcription factor: evidence modulation redox mechanism. DNA Cell <br />                                                   Biol 1993;12:265&#226;&#8364;&#8220;273. [PubMed: 8466649] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             [11]. Pines A, Bivi N, Romanello M, Damante G, Kelley MR, Adamson ED, D&apos;Andrea P, Quadrifoglio <br />                                                   F, Moro L, Tell G. Cross-regulation Egr-1 APE/Ref-1 early response to <br />                                                   oxidative stress human osteoblastic HOBIT cell line: evidence autoregulatory loop. <br />                                                   Free Radic Res 2005;39:269&#226;&#8364;&#8220;281. [PubMed: 15788231] <br />                                             [12]. Huang RP, Wu JX, Fan Y, Adamson ED. UV activates growth factor receptors reactive oxygen <br />                                                   intermediates. J Cell Biol 1996;133:211&#226;&#8364;&#8220;220. [PubMed: 8601609] <br />                                             [13]. Liu C, Rangnekar VM, Adamson E, Mercola D. Suppression growth transformation and <br />                                                   induction apoptosis EGR-1. Cancer Gene Ther 1998;5:3&#226;&#8364;&#8220;28. [PubMed: 9476963] <br />                                             [14]. Huang RP, Fan Y, Belle I, Niemeyer C, Gottardis MM, Mercola D, Adamson ED. Decreased <br />                                                   Egr-1 expression human, mouse rat mammary cells tissues correlates tumour <br />                                                   formation. Int J Cancer 1997;72:102&#226;&#8364;&#8220;109. [PubMed: 9212230] <br />                                             [15]. Krones-Herzig A, Mittal S, Yule K, Liang H, English C, Urcis R, Soni T, Adamson ED, Mercola <br />                                                   D. Early growth response 1 acts tumor suppressor vivo vitro regulation p53. <br />                                                   Cancer Res 2005;65:5133&#226;&#8364;&#8220;5143. [PubMed: 15958557] <br />                                             [16]. Virolle T, Adamson ED, Baron V, Birle D, Mercola D, Mustelin T, Belle I. The Egr-1 <br />                                                   transcription factor directly activates PTEN irradiation-induced signalling. Nat Cell Biol <br />                                                   2001;3:1124&#226;&#8364;&#8220;1128. [PubMed: 11781575] <br />                                             [17]. Baron V, Adamson ED, Calogero A, Ragona G, Mercola D. The transcription factor Egr1 direct <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   regulator multiple tumor suppressors including TGFbeta1, PTEN, p53, fibronectin. Cancer <br />                                                   Gene Ther 2006;13:115&#226;&#8364;&#8220;124. [PubMed: 16138117] <br />                                             [18]. Sansal I, Sellers WR. The biology clinical relevance PTEN tumor suppressor pathway. <br />                                                   J Clin Oncol 2004;22:2954&#226;&#8364;&#8220;2963. [PubMed: 15254063] <br />                                             [19]. Tell G, Pines A, Arturi F, Cesaratto L, Adamson E, Puppin C, Presta I, Russo D, Filetti S, Damante <br />                                                   G. Control phosphatase tensin homolog (PTEN) gene expression normal neoplastic <br />                                                   thyroid cells. Endocrinology 2004;145:4660&#226;&#8364;&#8220;4666. [PubMed: 15231710] <br />                                             [20]. Pines A, Perrone L, Bivi N, Romanello M, Damante G, Gulisano M, Kelley MR, Quadrifoglio F, <br />                                                   Tell G. Activation APE1/Ref-1 dependent reactive oxygen species generated after <br />                                                   purinergic receptor stimulation ATP. Nucleic Acids Res 2005;33:4379&#226;&#8364;&#8220;4394. [PubMed: <br />                                                   16077024] <br />                                             [21]. Izumi T, Brown DB, Naidu CV, Bhakat KK, Macinnes MA, Saito H, Chen DJ, Mitra S. Two <br />                                                   essential distinct functions mammalian abasic endonuclease. Proc Natl Acad Sci USA <br />                                                   2005;102:5739&#226;&#8364;&#8220;5743. [PubMed: 15824325] <br />                                             [22]. Grunstein M. Histone acetylation chromatin structure transcription. Nature 1997;389:349&#226;&#8364;&#8220; <br />                                                   352. [PubMed: 9311776] <br />                                             [23]. Kuo MH, Allis CD. Roles histone acetyltransferases deacetylases gene regulation. <br />                                                   Bioessays 1998;20:615&#226;&#8364;&#8220;626. [PubMed: 9780836] <br />                                             [24]. Katan-Khaykovich Y, Struhl K. Dynamics global histone acetylation deacetylation vivo: <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   rapid restoration normal histone acetylation status removal activators repressors. <br />                                                   Genes Dev 2002;16:743&#226;&#8364;&#8220;752. [PubMed: 11914279] <br />                                             [25]. Davie JR. Inhibition histone deacetylase activity butyrate. J Nutr 2003;133:2485S&#226;&#8364;&#8220;2493S. <br />                                                   [PubMed: 12840228] <br />                                             [26]. Eberharter A, Becker PB. Histone acetylation: switch repressive permissive <br />                                                   chromatin. Second review series chromatin dynamics. EMBO Rep 2002;3:224&#226;&#8364;&#8220;229. <br />                                                   [PubMed: 11882541] <br />                                             [27]. Davis PK, Brackmann RK. Chromatin remodeling cancer. Cancer Biol Ther 2003;2:22&#226;&#8364;&#8220;29. <br />                                                   [PubMed: 12673113] <br />                                             [28]. Lehrmann H, Pritchard LL, Harel-Bellan A. Histone acetyltransferases deacetylases the <br />                                                   control cell proliferation differentiation. Adv Cancer Res 2002;86:41&#226;&#8364;&#8220;65. [PubMed: <br />                                                   12374280] <br />                                             [29]. Kim DH, Kim M, Kwon HJ. Histone deacetylase carcinogenesis inhibitors anti-cancer <br />                                                   agents. J Biochem Mol Biol 2003;36:110&#226;&#8364;&#8220;119. [PubMed: 12542981] <br />  <br />  <br />                                                  Free Radic Res. Author manuscript; available PMC 2009 May 5. <br />                             FANTINI et al.                                                                                                  Page 10 <br />  <br />  <br />                                             [30]. Bunz F, Dutriaux A, Lengauer C, Waldman T, Zhou S, Brown JP, Sedivy JM, Kinzler KW, <br />                                                   Vogelstein B. Requirement p53 p21 sustain G2 arrest DNA damage. Science <br />                                                   1998;282:1497&#226;&#8364;&#8220;1501. [PubMed: 9822382] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             [31]. Bradford MM. A rapid sensitive method quantitation microgram quantities protein <br />                                                   utilizing principle protein-dye binding. Anal Biochem 1976;72:248&#226;&#8364;&#8220;254. [PubMed: 942051] <br />                                             [32]. Szczesny B, Bhakat KK, Mitra S, Boldogh I. Age-dependent modulation DNA repair enzymes <br />                                                   covalent modification subcellular distribution. Mech Ageing Dev 2004;125:755&#226;&#8364;&#8220;765. <br />                                                   [PubMed: 15541770] <br />                                             [33]. Huang RP, Darland T, Okamura D, Mercola D, Adamson ED. Suppression v-sis-dependent <br />                                                   transformation transcription factor Egr-1. Oncogene 1994;9:1367&#226;&#8364;&#8220;1377. [PubMed: 8152797] <br />                                             [34]. Mittal V. Improving efficiency <span id='am-238' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-239' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-242' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-244' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-249' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-258' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-261' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-295' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-302' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-309' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span>RNA interference</span> mammals. Nat Rev Genet 2004;5:355&#226;&#8364;&#8220; <br />                                                   365. [PubMed: 15143318] <br />                                             [35]. van Wetering M, Oving I, Muncan V, Pon Fong MT, Brantjes H, van Leenen D, Holstege FC, <br />                                                   Brummelkamp TR, Agami R, Clevers H. Specific inhibition gene expression using stably <br />                                                   integrated, inducible small-interfering-RNA vector. EMBO Rep 2003;4:609&#226;&#8364;&#8220;615. [PubMed: <br />                                                   12776180] <br />                                             [36]. Stambolic V, MacPherson D, Sas D, Lin Y, Snow B, Jang Y, Benchimol S, Mak TW. Regulation <br />                                                   PTEN transcription p53. Mol Cell 2001;8:317&#226;&#8364;&#8220;325. [PubMed: 11545734] <br />                                             [37]. Whelan JT, Forbes SL, Bertrand FE. CBF-1 (RBP-Jkappa) binds PTEN promoter and <br />                                                   regulates PTEN gene expression. Cell Cycle 2007;6:80&#226;&#8364;&#8220;84. [PubMed: 17245125] <br />                                             [38]. Kim S, Domon-Dell C, Kang J, Chung DH, Freund JN, Evers BM. Down-regulation tumor <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   suppressor PTEN tumor necrosis factor-alpha/nuclear factor-kappaB (NF-kappaB)-inducing <br />                                                   kinase/NF-kappaB pathway linked default IkappaB-alpha autoregulatory loop. J Biol Chem <br />                                                   2004;279:4285&#226;&#8364;&#8220;4291. [PubMed: 14623898] <br />                                             [39]. Patel L, Pass I, Coxon P, Downes CP, Smith SA, Macphee CH. Tumor suppressor anti- <br />                                                   inflammatory actions PPARgamma agonists mediated upregulation PTEN. Curr Biol <br />                                                   2001;11:764&#226;&#8364;&#8220;768. [PubMed: 11378386] <br />                                             [40]. Seo YR, Kelley MR, Smith ML. Selenomethionine regulation p53 ref1-dependent redox <br />                                                   mechanism. Proc Natl Acad Sci USA 2002;99:14548&#226;&#8364;&#8220;14553. [PubMed: 12357032] <br />                                             [41]. Hanson S, Kim E, Deppert W. Redox factor 1 (Ref-1) enhances specific DNA binding p53 by <br />                                                   promoting p53 tetramerization. Oncogene 2005;24:1641&#226;&#8364;&#8220;1647. [PubMed: 15674341] <br />                                             [42]. Leslie NR, Bennett D, Lindsay YE, Stewart H, Gray A, Downes CP. Redox regulation PI 3- <br />                                                   kinase signalling inactivation PTEN. EMBO J 2003;22:5501&#226;&#8364;&#8220;5510. [PubMed: 14532122] <br />                                             [43]. Pan L, Lu J, Wang X, Han L, Zhang Y, Han S, Huang B. Histone deacetylase inhibitor trichostatin <br />                                                   potentiates doxorubicin-induced apoptosis up-regulating PTEN expression. Cancer <br />                                                   2007;109:1676&#226;&#8364;&#8220;1688. [PubMed: 17330857] <br />                                             [44]. Brugarolas J, Chandrasekaran C, Gordon JI, Beach D, Jacks T, Hannon GJ. Radiation-induced cell <br />                                                   cycle arrest compromised p21 deficiency. Nature 1995;377:552&#226;&#8364;&#8220;557. [PubMed: 7566157] <br />                                             [45]. Okamoto K, Prives C. A role cyclin G process apoptosis. Oncogene 1999;18:4606&#226;&#8364;&#8220; <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   4615. [PubMed: 10467405] <br />                                             [46]. Bernstein C, Bernstein H, Payne CM, Garewal H. DNA repair/pro-apoptotic dual-role proteins in <br />                                                   major DNA repair pathways: fail-safe protection carcinogenesis. Mutat Res <br />                                                   2002;511:145&#226;&#8364;&#8220;178. [PubMed: 12052432] <br />  <br />  <br />  <br />  <br />                                                  Free Radic Res. Author manuscript; available PMC 2009 May 5. <br />                             FANTINI et al.                                                                                         Page 11 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Figure 1. <br />                                             APE1-dependent regulation PTEN expression. (A, Left) Down-regulation PTEN to <br />                                             siRNA-mediated transient reduction APE1 expression. HeLa cells transfected 1.5 <br />                                             &#206;&#188;g pSUPER (Ctrl) pSUPER/Ref-1 plasmid; 48 h later, cell lysates (15 &#206;&#188;g) were <br />                                             used Western analysis. (Right) Upregulation endogenous PTEN transient over- <br />                                             expression APE1/Ref-1. HeLa cells transfected 0.4 &#206;&#188;g pcDNA5.1-empty the <br />                                             pcDNA5.1-APE1-c-FLAG and, 48 h later, cell extracts (10 &#206;&#188;g) analysed before. (B, <br />                                             Left) Effect APE1 over-expression PTEN promoter activity. HeLa cells harvested <br />                                             cotr <br /> </body></html>