<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The Proto-Oncogene Int6 Is Essential Neddylation of <br /> Cul1 Cul3 Drosophila <br /> Sigal Rencus-Lazar1,2, Yaniv Amir1,2, Junetai Wu3, Cheng-Ting Chien3, Daniel A. Chamovitz4*, Daniel <br /> Segal1,2* <br /> 1 Department Molecular Microbiology, Tel Aviv University, Tel Aviv, Israel, 2 Department Biotechnology, Tel Aviv University, Tel Aviv, Israel, 3 Institute Molecular <br /> Biology, Academia Sinica, Taipei, Taiwan, 4 Department Plant Sciences, Tel Aviv University, Tel Aviv, Israel <br />  <br />  <br />  <br />      <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />      Int6 proto-oncogene implicated various types cancer, mechanisms underlying activity clear. Int6 <br />      encodes subunit eukaryotic translation initiation factor 3, interacts related complexes, the <br />      proteasome, activity regulated Int6 S. pombe, COP9 signalosome. The COP9 signalosome regulates <br />      activity Cullin-Ring Ubiquitin Ligases deneddylation cullin subunit. We report generation and <br />      analysis Drosophila mutants Int6. The mutants lethal demonstrating Int6 essential gene. The mutant <br />      larvae accumulate high levels non-neddylated Cul1, suggesting Int6 positive regulator cullin neddylation. <br />      Overexpression Int6 cell culture leads accumulation neddylated cullins, supporting positive role Int6 <br />      regulating neddylation. Thus Int6 COP9 signalosome play opposing roles regulation cullin neddylation. <br />  <br />   Citation: Rencus-Lazar S, Amir Y, Wu J, Chien C-T, Chamovitz DA, et al. (2008) The Proto-Oncogene Int6 Is Essential Neddylation Cul1 Cul3 in <br />   Drosophila. PLoS ONE 3(5): e2239. doi:10.1371/journal.pone.0002239 <br />   Editor: Stefan Wo&#194;&#168;lfl, Universita&#194;&#168;t Heidelberg, Germany <br />   Received November 26, 2007; Accepted April 4, 2008; Published May 21, 2008 <br />   Copyright: &#195;&#376; 2008 Rencus-Lazar et al. This open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: This work supported grant Israel Academy Sciences (<span id='am-2' about='protege:TO' typeof='owl:Thing'>to</span> DAC). <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: dannyc@ex.tau.ac.il (DC); dsegal@post.tau.ac.il (DS) <br />  <br />  <br />  <br /> Introduction                                                                             [5,16], significance interaction clear. To <br />                                                                                          elucidate roles Int6 context multi-cellular <br />    Int6/eIF3e conserved onco-protein encoded gene                              organism, analyzed mutants Drosophila Int6, demon- <br /> initially identified preferred integration site mouse                        strating essential gene organism cell <br /> mammary tumor virus [1]. Integration virus Int6 results                      viability. While did detect disruption CSN assembly <br /> in expression C-terminally truncated version Int6,                       function Int6 mutants, decrease neddylation of <br /> causing tumorigenic transformation [2,3]. Silencing Int6                           Cul1 Cul3 absence Int6. Our findings also <br /> HeLa cells results mitotic defects [4].                                               suggest Int6 essential degradation cullin <br />    Int6 common interactor related protein complexes:                       substrates. Consistently, overexpression Int6 cell culture <br /> the eukaryotic translation Initiation Factor 3 (eIF3), protea-                       resulted elevated neddylation cullins. We show <br /> some COP9 signalosome (CSN) [5]. Int6                                   novel role Int6 positive regulator cullin <br /> identified subunit eIF3 [6]. In S. pombe, Int6 essential                  neddylation. <br /> both assembly function proteasome [7]. As result, <br /> degradation proteasome substrates impaired S.                           Methods <br /> pombe mutated Int6 homolog, yin6 [7]. It clear <br /> whether regulation proteasome Int6 conserved                      D. melanogaster maintenance hybrid dysgenesis <br /> in organisms, evidence regulation                            Strains maintained crosses conducted on <br /> found mammalian cells Arabidopsis [4,8].                                        standard cornmeal-molasses medium 25uC. Oregon-R served <br />    The CSN eight-subunit protein complex involved                               wild-type control. The Nedd8172, Cul1EX [17], CSN5null [18] <br /> regulation protein degradation. The CSN shown                             Cul306430, Cul3d577 [19] Rpn10 [20] mutations been <br /> regulate ubiquitination Cullin-Ring ubiquitin Ligases                            previously described. Description balancer chromosomes and <br /> (CRLs). The CSN removes Nedd8 modification                                  markers FlyBase [21] (http://flybase.bio.indiana. <br /> cullin subunit CRL, resulting inactivation                         edu/). The CyO TM3 Ser balancers carry act-GFP <br /> CRL ubiquitin ligase [9,10] stabilization cullin                      transgene described [22]. <br /> subunit organisms [11&#226;&#8364;&#8220;13]. On hand, CSN                               The P element Bloomington strain #11748 [21] inserted <br /> is essential maintaining stability substrate                         coding sequence Int6 <span id='am-315' about='xsp:base' typeof='owl:Thing'>base</span> 204, 1 1st <br /> adaptor subunit CRLs, promoting ubiquitin ligase                        base start codon. Analysis putative excision lines was <br /> activity [14]. The CSN regulates stability                        performed primers derived Int6 sequences upstream to <br /> the activity cullin-based E3 ubiquitin ligases.                                       P element insertion (59-ATGGCCAATTTCGATCTGA- <br />    Int6 co-purifies CSN Arabidopsis [15] interacts                     CACG-39), corresponding nucleotides 1 23, downstream <br /> with CSN subunits plants mammalian cells                          P insertion site (59-CTAGTAGTACTTCCAG- <br />  <br />  <br />         PLoS ONE | www.plosone.org                                                   1                                     May 2008 | Volume 3 | Issue 5 | e2239 <br />                                                                                                                        INT6 Cullin Neddylation <br />  <br />  <br /> GAGTCGG-39) corresponding Int6 gene sequence 1462                     solution. The membrane washed 3 times 15 minutes in <br /> 1484. The P element 11748 strain carries dominant ry+            TTBS (0.1% Tween-20 1xTBS), incubated 30 minutes the <br /> eye color marker. Crosses females 11748 strain males          secondary antibody washed 3 times 10 minutes TTBS. <br /> of strain carrying 2&#226;&#8364;&#8220;3 stable source transposase                   The membrane developed using EZ-ECL (Biological industries), <br /> (Bloomington Stock Center #1610) conducted. Dysgenic                   according manufacturer&#226;&#8364;&#8482;s instructions, exposed Fuji <br /> P{ry+t7.2 = PZ}Int6[10547] ry506/TMS, P{ry[+t7.2] = Delta2-                 Medical X-Ray Film 5 minutes. Films developed <br /> 3}99B male progeny, germ line P{ry+t7.2 =                      using Kodak X-OMAT 2000. <br /> PZ}Int6[10547] P element mobilized, crossed females                Primary antibodies used immuno-blot analysis anti-Cul1 <br /> of 11748 strain. Offspring (F2) did carry                  (1:250; Zymed), anti-actin (1:5000; MP Biomedicals), anti-Cul3 <br /> transposase source screened males rosy eyes                   (1:1000), anti-Nedd8 (1:1000), anti-Ubiquitin (1:7500; Affinity), anti- <br /> indicative loss ry+ marker caused excision P               cyclin B (1:3000) [24], anti-CSN5 (1:10000) [25], anti-CSN7 <br /> element 11748. Individual putative excision males                 (1:10000) [25] anti-HA (1:2500; Covance). Secondary antibod- <br /> crossed females 11748 strain establish balanced lines.         ies Horse radish peroxidase-conjugated goat-anti-rabbit <br /> Lines established 57 independent putative excision flies,         (1:10000; Santa Cruz), Horse radish peroxidase-conjugated <br /> 6 homozygous viable.                                          goat-anti-mouse (1:10000; Jackson Immuno-research). <br />  <br /> Heat shock treatment                                                        Signal quantification <br />   Generation germ-line clones.          flies desired cross         Immuno blots displaying signal ECL linear response <br /> were allowed lay eggs twenty-four hours small tubes               scanned using ImageScanner (Amersham Pharmacia <br /> containing standard cornmeal-molasses medium. The larvae               Biotech) bands quantified using ImageMaster 1D <br /> heat-shocked 2nd 3rd instar stages. Heat shock               Image Analysis Software (Amersham Pharmacia Biotech). <br /> administered putting vials larvae 37uC water <br /> bath 1.5 hours. Heat shock performed twice day days            Cell culture methods <br /> 3 4 egg laying. After heat treatment vials              Schneider&#226;&#8364;&#8482;s 2R+ cells grown flasks 27uC in <br /> returned 25uC.                                                           Schneider&#226;&#8364;&#8482;s Drosophila medium L-Glutamine (Biological <br />    Generation wing eye disc clones. flies desired             Industries) containing 10% Fetal Calf Serum (pre-inactivated <br /> cross allowed lay eggs twenty-four hours tubes               1 hour 60uC; Biological Industries), 100 units/ml Penicillin <br /> containing standard medium. Heat shock administered                  0.1 mg/ml Streptomycin-Sulfate (Biological Industries). Cells <br /> putting vials larvae 37uC water bath 1.5 hours        transferred 1:4 3&#226;&#8364;&#8220;4 days. For trasnfection, days <br /> 5 days performing cross. The larvae dissected            regular 1:4 transfer, cells transferred 1:3 each <br /> days (,44 hours) heat shock treatment.                            6-well plate. On following day 3.5 mg each <br />                                                                             plasmid (,1.5 mg/ml) added serum- antibiotics-free <br /> Molecular biochemical procedures                                        medium total volume 100 ml. In parallel, 10 ml ESCORT <br />    Quantitative reverse transcription PCR. RNA                          IV Transfection Reagent (Sigma) added 90 ml serum- <br /> extracted ,1000 Oregon-R (wt) ,2500 Int6173-1 1st                  antibiotics-free medium. The diluted DNA added <br /> instar larvae TRIzol reagent (Invitrogen). cDNA synthesis              diluted transfection reagent DNA-liposome <br /> was carried using Superscript II (Invitrogen).                          complexes allowed form 1 hour room temperature. <br />    Four qPCR primer pairs designed generate intron-                 The cells (50&#226;&#8364;&#8220;70% confluent) washed 4 times serum- and <br /> spanning products using Primer Express Version 1.5 software:                antibiotics-free medium 800 ml serum- antibiotics-free <br />                                                                             medium added well. 200 ml DNA- <br />   Primer         Sequence                                              ESCORT mixture added dropwise <br />   Cul1-RT5            GTGCTGACCTCCAGCGACA                                   manner. The cells grown 27uC 23 hours, which <br />                                                                             serum- antibiotics-free medium changed serum- <br />   Cul1-RT6            GGTTGATGTTAATTCGGCGC <br />                                                                             antibiotics-containing medium. Cells grown for <br />   Rp49-forward        TAAGCTGTCGCACAAATGGC                                  additional 8 hours total 31 hours, cells each <br />   Rp49-reverse        ACCGATGTTGGGCATCAGATA                                 resuspended 1 ml medium transferred <br />                                                                             tube. Cells harvested centrifugation 3 min at <br />    Quantitative reverse transcription polymerase chain reaction             2000 rpm. The cells pellet frozen 280uC. Over <br /> (qRT-PCR) analysis performed ABI PrismH 7000                      expression transfected gene verified SDS-PAGE, <br /> Sequence Detection System (Applied Biosystems) using SYBR                   followed immunoblot analysis. <br /> GreenH I chemistry. Samples triplicates normalized <br /> for RNA levels based rp49 expression. Analysis performed <br />                                                                             Results <br /> with ABI Prism 7000 SDS software RQ study Application v1.1 <br /> using D-DCt method, determines fold changes gene               Int6 essential gene <br /> expression relative comparative sample.                                   A strain obtained (Bloomington #11748) contains a <br />                                                                             P element located Int6 open reading frame <br /> SDS-PAGE Immuno-blot                                                    (P{PZ}Int610547) [21]. PCR sequence analysis confirmed <br />    Protein samples mixed 1:1 volume Laemmli sample             P element genome 11748 strain inserted <br /> buffer (BioRad), separated 10% poly-acrylamide gel using          second exon Int6, 204 bp downstream 1st ATG <br /> Mighty Small System (Hoefer). Proteins transferred              codon (Figure S1). <br /> PVDF membrane (Millipore), pre-washed methanol, transfer                 The chromosome carrying insert Int6 homozygous <br /> buffer. The membrane blocked 1 hour blocking                lethal. To verify lethality P{PZ}Int610547-carrying <br /> solution (5% milk powder, 0.02% sodium-azide 1xTBS),            chromosome P insertion Int6, P element was <br /> incubated primary antibody (see below) diluted blocking           mobilized hybrid dysgenesis protocol crosses. Six <br />  <br />  <br />       PLoS ONE | www.plosone.org                                        2                                May 2008 | Volume 3 | Issue 5 | e2239 <br />                                                                                                                             INT6 Cullin Neddylation <br />  <br />  <br /> homozygous viable excision lines obtained sequence                    Int6 negative regulator Cul1 protein levels <br /> analysis confirmed excision P element restoration               Int6 physically interacts CSN subunits both <br /> the normal Int6 gene sequence. Thus, Int6 essential Drosophila          Arabidopsis [16] mammalian cell culture [5]. These <br /> development.                                                                   interactions place Drosophila (see data <br />    Homozygous lethal excision lines obtained                  shown), serve regulate assembly CSN <br /> hybrid dysgenesis. PCR sequence analysis                   (Figure S2). To explore interaction Int6 and <br /> lines, termed Int6173-1, indicated deletion base pairs 204 952         CSN, examined Int6 regulates deneddylation <br /> in Int6 gene, 1 defined nucleotide            function CSN comparing levels non-neddylated <br /> start codon (Figure S1).                                                       neddylated Cul1 larvae various genotypes. While, as <br />    Both Int6173-1 deletion P insertional <span id='am-237' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-238' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-261' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-262' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-271' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-272' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-278' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-290' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-292' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-295' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-296' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span>mutation</span> result           expected, neddylated Cul1 accumulated csn5null larvae, <br /> in essentially identical phenotypes. Homozygous larvae              compared wild type control (Figure 2A, compare lanes 1 and <br /> mutation small, compared heterozygous siblings,               3) [27], accumulation neddylated-Cul1 observed in <br /> die 3&#226;&#8364;&#8220;4 days hatching 1st larval instar        Int6 mutant, 72 hours AED (Figure 2A, compare lanes 1 <br /> stage (Figure 1A, upper panels, data shown),               5,6). This result indicates Int6 essential CSN <br /> their heterozygous siblings progressed 3rd                 deneddylation activity. Surprisingly though, high <br /> instar. The Int6 mutant larvae display obvious                accumulation Cul1 non-neddylated form both <br /> morphological abnormalities. Larvae heteroallelic Int6             P{PZ}Int610547, P-element insertion mutant Int6, in <br /> mutations <span id='am-159' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-160' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span>phenotype</span>.                                             Int6173-1, Int6 deletion mutant (Figure 2A, compare lanes 1 and <br />    To examine phenotype lack Int6 cellular                5,6). This result confirmed Int6 mutant allele at <br /> level, generated somatic germ-line clones homozy-                  additional biological repeat. The accumulation non- <br /> gous Int6 mutation. As shown Figure 1B,C, Int6-                  neddylated Cul1 different Int6 mutants demonstrates that <br /> homozygous clones induced wing eye imaginal                    Int6 negative regulator Cul1 levels Drosophila. The high <br /> discs dozen individuals smaller               level non-neddylated Cul1 Int6 mutants result of <br /> than adjacent non-mutant twin spots, indicating Int6             elevated RNA levels levels Cul1 mRNA Int6173-1 <br /> essential viability and/or proliferation disc cells. By         mutants significantly different wild type control <br /> adult stage, Int6-homozygous clones apparently elimi-                 (Figure 2B). Thus, Int6 negatively regulates Cul1 post- <br /> nated adult eyes wings did aberrant                   transcriptional mechanism. <br /> phenotype (not shown).                                                            Another possible mechanism regulation Cul1 levels by <br />    To examine roles Int6 zygotic maternally               Int6 regulation proteasome function. As Int6 is <br /> contributed Int6 absent, germ line clones homozygous                   essential function proteasome S. pombe [7], is <br /> either Int6 mutation generated [26]. While germ-line clones               possible proteasome substrates, Cul1 <br /> of control Int6+ chromosome contained fully developed egg                  [28], accumulate Drosophila Int6 mutants. To test possibility, <br /> chambers (Figure 1D), gave rise normal embryos, germ line              examined general function proteasome two <br /> clones homozygous Int6 mutation arrested stage 5&#226;&#8364;&#8220;6               Int6 mutants monitoring levels ubiquitinated proteins. No <br /> of oogenesis (Figure 1E). This suggests somatic cells, Int6         accumulation ubiquitinated proteins detected Int6 <br /> is required proliferation viability female germ cells,           mutant, csn5null nedd8 mutants (Figure 2C, left panel). <br /> thereby proper oogenesis.                                                  High accumulation ubiquitinated proteins detected in <br />  <br />  <br />  <br />  <br /> Figure 1. Int6 mutant phenotypes. A. Larvae heterozygous (upper panel) homozygous (middle panel) Int6173-1 allele shown. Larvae <br /> homozygous P{PZ}Int610547 allele display identical phenotype (not shown). Int6173-1-csn5null double mutant larvae phenotypically <br /> identical Int6173-1 single mutants (lower panel). B,C. Clones homozygous P{PZ}Int610547 mutation display growth/viability defects. Int6 clones <br /> in wing (B) eye (C) disc (for example, outlined white), shown compared markedly larger GFP/GFP twin spots (for example, outlined in <br /> yellow); D&#226;&#8364;&#8220;E. Egg chambers germ line clones positive control, developed embryos (D) P{PZ}Int610547 mutation, <br /> arrested stage 5&#226;&#8364;&#8220;6 oogenesis (E). Somatic germ line clones Int6173-1 deletion result phenotypes P{PZ}Int610547 <br /> mutation (not shown). <br /> doi:10.1371/journal.pone.0002239.g001 <br />  <br />  <br />        PLoS ONE | www.plosone.org                                          3                                 May 2008 | Volume 3 | Issue 5 | e2239 <br />                                                                                                                            INT6 Cullin Neddylation <br />  <br />  <br />  <br />  <br /> Figure 2. Int6 negative regulator Cul1 protein levels. A. Immuno-blot analysis using anti-Cul1 antibodies performed protein samples <br /> extracted homozygous larvae indicated genotypes. B. Real-time PCR analysis Cul1 mRNA levels wild type Int6173-1 mutants the <br /> same ages A. Transcript levels mutant presented relation wt. Error bars standard deviation repeats. C&#226;&#8364;&#8220;D. Immuno- <br /> blot analysis using anti-ubiquitin (C) anti-cyclin B (D) antibodies performed protein samples extracted 1st instar larvae indicated <br /> genotypes. Rpn10 mutants used positive control proteasome malfunction. Samples run single gel lane order was <br /> rearranged figure indicated dotted lines. The blot panel A repeated similar results times. <br /> doi:10.1371/journal.pone.0002239.g002 <br />  <br /> larvae mutant proteasome regulatory subunit Rpn10, used                type control (Figure 3B, compare lanes 1 3). Yet, ratio of <br /> here positive control (Figure 2C, right panel, compare lanes 1            neddylated:non-neddylated Cul1 significantly reduced both <br /> and 4), again, similar accumulation detected            Int6 mutants (Figure 3B, compare lanes 1 4, 5, biological <br /> Int6 mutant (Figure 2C, compare lanes 1 2,3). To assay             repeats shown), indicating Cul1 neddylation indeed <br /> proteasome function mutants, monitored levels               reduced absence Int6. <br /> cyclin B, specific proteasome substrate accumulates Int6               Since neddylated: non-neddylated Cul1 ratio higher than <br /> mutants S. pombe [7]. While cyclin B accumulated Rpn10                   normal csn5null mutants lower normal Int6 mutants, <br /> mutants (Figure 2D, compare lanes 1 4), did               reasoned csn5null mutation rescue reduced <br /> Int6 mutant (Figure 2D, compare lanes 1 2,3). Taken together,              Cul1 neddylation Int6 mutants. Indeed, level neddylated <br /> these results clearly indicate accumulation Cul1            Cul1 rescued Int6173-1-csn5null double mutant larvae, <br /> absence Int6 result reduced proteasome activity.              compared single mutant (Figure 3B, compare lanes 3,5 <br />                                                                                6). This result implicates Int6 regulation of <br /> Int6 positive regulator Cul1 neddylation                               neddylation Cul1, absence Int6 and <br />    Neddylation Drosophila Cul1 results destabilization [12].         CSN5, neddylation deneddylation reduced, resulting <br /> Another possible explanation accumulation high levels            intermediate neddylated: non-neddylated Cul1 ratio. <br /> non-neddylated Cul1 Int6 mutants reduced                 Phenotypically, double mutant Int6173-1-csn5null homozygous <br /> Cul1 neddylation absence Int6. To test possibility           larvae similar Int6173-1 single mutants (Figure 1A, <br /> used anti-Nedd8 antibodies, reduced intensity              lower panel), suggesting phenotypes Int6 mutants <br /> ,90 kDa bands (the expected size neddylated cullins)                solely lack Cul1 neddylation. Taken together, <br /> Int6 mutants (Figure 3A, compare lanes 1 4, 5). If reduced            results indicate Cul1 neddylation reduced the <br /> neddylation cause accumulation Cul1                    absence Int6, result Cul1 accumulates non- <br /> absence Int6, accumulation non-neddylated Cul1 Int6               neddylated form. <br /> mutants preceded reduced Cul1 neddylation. To <br /> examine possibility, examined ratio neddylated:non-             Int6 positive regulator Cul3 neddylation <br /> neddylated Cul1 48-hours-old Int6 mutants. As expected,                  We examined Int6 regulates neddylation of <br /> ratio two-fold higher csn5null mutants, compared wild            additional cullins. Immuno-blot analysis using anti-Cul3 antibod- <br />  <br />  <br />        PLoS ONE | www.plosone.org                                          4                                May 2008 | Volume 3 | Issue 5 | e2239 <br />                                                                                                                              INT6 Cullin Neddylation <br />  <br />  <br />  <br />  <br /> Figure 3. Int6 essential neddylation Cul1 Cul3 negative regulator CSN5 levels. Immno-blot analysis using anti- <br /> Nedd8 (A), anti-Cul1 (B), anti-Cul3 (C,D) anti-CSN5 (E,F) antibodies performed protein samples extracted 48 (A&#226;&#8364;&#8220;C,E) 72 (D,F) hours egg <br /> deposit indicated genotypes. In absence Int6, overall neddylation cullins (A), specifically neddylation Cul1 (B) Cul3 <br /> (D), reduced. CSN5 accumulated absence Int6 Cul3 (F). <br /> doi:10.1371/journal.pone.0002239.g003 <br />  <br />  <br /> ies revealed significant decrease Cul3 neddylation                      deneddylation activity CSN (Figure 3D, compare lanes 1 <br /> P{PZ}Int610547 homozygous larvae, compared wild type                     3). Decreased Cul3 neddylation 72-hours-old <br /> control (Figure 3C, compare lanes 1 5), slight                   larvae Int6 mutants (Figure 3D, compare lanes 1 4, 5, <br /> decrease Int6173-1 homozygous larvae (Figure 3C, compare lanes               biological repeats shown), demonstrating Int6 is <br /> 1 4). It noted 48 hours old csn5null larvae did          essential neddylation Cul3, Cul1. Nevertheless, <br /> show increase neddylated: non-neddylated Cul3 ratio                      csn5null mutation did rescue reduced Cul3 neddylation <br /> (Figure 3C, compare lanes 1 3), persistence              caused lack Int6 Cul3 neddylation Int6173-1- <br /> of maternally deposited CSN5 stage (Figure 3E, lane 3).                 csn5null double homozygous larvae similar the <br />    To examine Int6 regulates neddylation                 Int6173-1 single mutant (Figure 3D, compare lanes 5 6). <br /> Cul3, performed immuno-blot analysis 72-hours- <br /> old csn5null Int6 mutants. Second instar csn5null larvae displayed          Int6 negative regulator CSN5 levels <br /> the expected increase ratio neddylated:non-neddylated                    The levels csn5 mRNA recently shown increase a <br /> Cul3, compared wild type control, reduced                         result expression dominant negative version Int6 in <br />  <br />  <br />        PLoS ONE | www.plosone.org                                           5                                 May 2008 | Volume 3 | Issue 5 | e2239 <br />                                                                                                                              INT6 Cullin Neddylation <br />  <br />  <br /> transgenic mice [29]. An equivalent increase CSN5 protein                      4). While levels non-neddylated Cullins affected by <br /> levels Drosophila Int6 mutants explain reduced                       overexpression Int6, neddylated form Cul1 was <br /> neddylated: non-neddylated cullin ratio Int6                     increased ,40%, neddylated form Cul3 was <br /> mutants (Figure 3A, B, D), elevated CSN5 levels result                increased 89%. As opposed Cul1, Cul3 neddylation was <br /> in increased deneddylation activity CSN. However,                     elevated presence large amounts over-expressed <br /> possible explanation does apply decreased neddylation                 Int6 (Figure 4A, compare lanes 3 4). Since neddylation of <br /> of Cul3 Int6 mutants rescued csn5null              cullins destabilizes [12], elevated neddylation both <br /> mutation (Figure 4D). We levels CSN5                     cullins presence over-expressed Int6 indirect <br /> increase Int6 mutants, 72 hours old larvae                    result. In scenario, over-expressed Int6 causes elevation in <br /> (Figure 3E,F, compare lanes 1 4,5). Since Cul1 neddylation                 cullin levels, result excess cullins tagged Nedd8 for <br /> already low 48 hours old larvae (Figure 3B), accumulation                  degradation. To examine possibility over-expressed Flag- <br /> of CSN5 absence Int6 account reduction                  tagged Cul1 Schneider cell culture. Over-expression Flag- <br /> in neddylation Cul1 Cul3. It noted,               Cul1 resulted equal increase neddylated non- <br /> similar previously reported results Arabidopsis [13], Drosophila            neddylated Cul1 (Figure 4B, compare lanes 1,2 3,4), <br /> CSN5 over-accumulates absence Cul3 (Figure 3F,                          indicating mere rise Cul1 levels does result elevated <br /> compare lanes 1 7).                                                           neddylation. We conclude Int6 sufficient for <br />                                                                                   promoting neddylation Cul1 Cul3. <br /> Int6 sufficient promoting neddylation <br />    Having demonstrated Int6 essential cullin neddyla-                 Discussion <br /> tion, examined sufficient promoting                        This work aimed analyzing roles Int6 Drosophila <br /> neddylation. Over-expression HA-tagged Int6 Drosophila                      development context interactions CSN. The <br /> Schneider cell culture led specific elevation neddylated                CSN shown de-neddylate cullins, regulating <br /> form Cul1 Cul3 (Figure 4A, compare lanes 1, 2 3,                  degradation substrates ubiquitin-proteasome <br />                                                                                   pathway. We here, time, Int6 takes part <br />                                                                                   regulation process promoting neddylation Cul1 <br />                                                                                   Cul3. This clearly shown different Int6 <br />                                                                                   mutants, provide internal repeats experiments. <br />                                                                                   Therefore, Int6 CSN play opposing roles regulation of <br />                                                                                   neddylation process. <br />  <br />                                                                                   Int6 essential gene Drosophila <br />                                                                                      To examine roles Int6 context multi-cellular <br />                                                                                   organism, analyzed mutants Drosophila Int6. Both the <br />                                                                                   deletion P element insertional mutations described this <br />                                                                                   study homozygous lethal 1st instar larval stage. In <br />                                                                                   addition, Int6 mutant clones wing disc, eye disc, the <br />                                                                                   germ line, small compared non-mutant twin- <br />                                                                                   spots, eliminated adult tissue. Int6 likely an <br />                                                                                   essential general cellular regulator Drosophila. <br />                                                                                      In spite numerous attempts, though, successful in <br />                                                                                   generating effective antibodies Drosophila Int6, were <br />                                                                                   previously reported antibodies [30] reactive hands, thereby <br />                                                                                   precluding complete biochemical analysis Int6. <br />  <br />                                                                                   Int6 promotes neddylation Cul1 Cul3 <br />                                                                                      We novel role Int6 positive regulator of <br />                                                                                   cullin neddylation. Although mechanism Int6 <br />                                                                                   regulates cullin neddylation established, likely <br />                                                                                   does involve direct interaction Int6 Cullins could <br />                                                                                   evidence interaction immunoprecipitation <br />                                                                                   (not shown). This novel role Int6 shed light on <br />                                                                                   roles Int6 plays cancer development. As here <br />                                                                                   (Figures 3), absence Int6 results reduced cullin <br />                                                                                   neddylation, accumulation CSN5. The levels <br /> Figure 4. Int6 promotes neddylation Cul1 Cul3. A,B.                        CSN5 increased absence Cul3 both <br /> Immuno-blot analysis using anti-Cul1 (A,B) anti-Cul3 (A) antibodies           Arabidopsis [13] Drosophila (Figure 3), suggest be <br /> performed protein samples extracted Schneider&#226;&#8364;&#8482;s 2R+ cells                 substrate Cul3 organisms. We propose that <br /> transfected HA-Int6 (A) Flag-Cul1 (B). Over-expression          Int6 regulates degradation certain cullin substrates <br /> HA-Int6 results specific elevation neddylated Cul1 Cul3 (A),          regulation cullin neddylation. The abrogated degradation of <br /> though neddylation increased presence of <br /> high levels Int6. This elevated neddylation result mere high <br />                                                                                   various cullin substrates absence Int6 play key role <br /> cullin levels, over-expression Flag-Cul1 results equal             development cancer. To test hypothesis, levels of <br /> elevation Cul1 neddylated Cul1 (B).                                        bona fide CRL substrates examined Int6 mutants. In <br /> doi:10.1371/journal.pone.0002239.g004                                             addition, over-expression C-terminally truncated version of <br />  <br />  <br />        PLoS ONE | www.plosone.org                                             6                               May 2008 | Volume 3 | Issue 5 | e2239 <br />                                                                                                                                                  INT6 Cullin Neddylation <br />  <br />  <br /> Int6 shown result tumorigenic transformation [3,29]. It                            situation S. pombe, subset does include cyclin B, <br /> will interesting examine effect over-                                 remains tested. Nevertheless, taken together, results and <br /> expression truncated Int6 neddylation Cul1 Cul3.                           findings Int6 essential degradation securin and <br />    It noted differences                             cyclin B HeLa cells [4] positive regulator of <br /> Cul1 Cul3 respect regulation neddylation                             proteasome function Arabidopsis (A. Yahalom D.A. <br /> Int6. The reduced neddylation Cul1 leads high accumulation                             Chamovitz, unpublished) indicate regulation proteasome <br /> of non-neddylated Cul1 Int6 mutants rescued                              function Int6 conserved higher organisms. It should <br /> mutation csn5 (Figure 2,3). However, reduced neddylation                               noted, unlike Int6 S. pombe [31,32], here <br /> Cul3 detected later Cul1 (Figure 3),                           Drosophila Int6 essential gene. It plausible that <br /> non-neddylated Cul3 does accumulate prior larvae                                  mechanisms underlying Int6 function organisms are <br /> death. In addition, reduced neddylation Cul3 rescued                           different well. <br /> by csn5 mutation. Consistently, unlike Cul1, neddylation Cul3                              Int6 multi-functional protein involved multiple processes. <br /> is promoted cell culture presence high levels                           It implicated regulation translation [32,33], mitosis <br /> over-expressed Int6 (Figure 4). We difference                                [4] proteasome regulation S. pombe [7]. Here a <br /> deneddylation Cul1 Cul3 CSN5, deneddylation                               novel role Int6 positive regulator cullin neddylation. <br /> Cul1 decreased earlier Cul3 csn5null mutants.                             Further work needed assemble existing data a <br /> These differences imply regulation neddylation                            unified picture roles Int6, normal cellular function <br /> Cul1 Cul3 identical. It possible neddylation                           cancer. <br /> Cul3 regulated factors, addition Int6 CSN5, <br /> and effect status Cul3 <br /> neddylation smaller effect Cul1. <br />                                                                                              Supporting Information <br />                                                                                              Figure S1 <br /> Int6 essential proteasome function                                             Found at: doi:10.1371/journal.pone.0002239.s001 (2.35 MB TIF) <br /> Drosophila                                                                                   Figure S2 <br />    In S. pombe, Int6 regulates function proteasome                             Found at: doi:10.1371/journal.pone.0002239.s002 (2.36 MB TIF) <br /> promoting entry proteasome regulatory subunit Rpn5 <br /> into nucleus [7]. As result, S. pombe mutants lacking Int6 show <br /> both reduction general proteasome function accumula-                                Acknowledgments <br /> tion specific proteasome substrates, Cut2/securin Cdc13/                              We thank Drs. A. Udvardy D. Glover providing materials. <br /> cyclin B. In contrast, results absence of <br /> Drosophila Int6 general proteasome function                                     Author Contributions <br /> degradation specific proteasome substrate, cyclin B, are <br />                                                                                              Conceived designed experiments: DS DC SR. Performed the <br /> normal, indicating Int6 essential proteasome activity <br />                                                                                              experiments: SR YA JW. Analyzed data: DS DC SR CC. Contributed <br /> in fly (Figure 2). The possibility Int6 regulates                               reagents/materials/analysis tools: JW CC. Wrote paper: DS DC SR. <br /> degradation subset proteasome substrates, unlike <br />  <br /> References <br />  1. Marchetti A, Buttitta F, Miyazaki S, Gallahan D, Smith GH, et al. (1995) Int-6,          13. Gusmaroli G, Figueroa P, Serino G, Deng XW (2007) Role MPN subunits <br />     highly conserved, widely expressed gene, mutated mouse mammary                       COP9 signalosome assembly activity, regulatory interaction <br />     tumor virus mammary preneoplasia. J Virol 69: 1932&#226;&#8364;&#8220;1938.                                  Arabidopsis Cullin3-based E3 ligases. Plant Cell 19: 564&#226;&#8364;&#8220;581. <br />  2. Rasmussen SB, Kordon E, Callahan R, Smith GH (2001) Evidence                         14. Wee S, Geyer RK, Toda T, Wolf DA (2005) CSN facilitates Cullin-RING <br />     transforming activity truncated Int6 gene, vitro. Oncogene 20:                   ubiquitin ligase function counteracting autocatalytic adapter instability. Nat <br />     5291&#226;&#8364;&#8220;5301.                                                                                   Cell Biol 7: 387&#226;&#8364;&#8220;391. <br />  3. Mayeur GL, Hershey JW (2002) Malignant transformation                             15. Karniol B, Malec P, Chamovitz DA (1999) Arabidopsis FUSCA5 encodes a <br />     eukaryotic translation initiation factor 3 subunit p48 (eIF3e). FEBS Lett 514:               novel phosphoprotein component COP9 complex. Plant Cell 11: <br />     49&#226;&#8364;&#8220;54.                                                                                       839&#226;&#8364;&#8220;848. <br />  4. Morris C, Jalinot P (2005) Silencing human Int-6 impairs mitosis progression          16. Yahalom A, Kim TH, Winter E, Karniol B, von Arnim AG, et al. (2001) <br />     inhibits cyclin B-Cdk1 activation. Oncogene 24: 1203&#226;&#8364;&#8220;1211.                               Arabidopsis eIF3e (INT-6) associates eIF3c COP9 <br />  5. Hoareau Alves K, Bochard V, Rety S, Jalinot P (2002) Association                      signalosome subunit CSN7. J Biol Chem 276: 334&#226;&#8364;&#8220;340. <br />     mammalian proto-oncoprotein Int-6 protein complexes eIF3,                 17. Ou CY, Lin YF, Chen YJ, Chien CT (2002) Distinct protein degradation <br />     COP9 signalosome 26S proteasome. FEBS Lett 257: 15&#226;&#8364;&#8220;21.                                   mechanisms mediated Cul1 Cul3 controlling Ci stability Drosophila eye <br />  6. Asano K, Merrick WC, Hershey JW (1997) The translation initiation factor                     development. Genes Dev 16: 2403&#226;&#8364;&#8220;2414. <br />     eIF3-p48 subunit encoded int-6, site frequent integration mouse        18. Oron E, Mannervik M, Rencus S, Harari-Steinberg O, Neuman-Silberberg S, et <br />     mammary tumor virus genome. J Biol Chem 272: 23477&#226;&#8364;&#8220;23480.                                    al. (2002) COP9 signalosome subunits 4 5 regulate multiple pleiotropic <br />  7. Yen HC, Gordon C, Chang EC (2003) Schizosaccharomyces pombe Int6 Ras                     pathways Drosophila melanogaster. Development 129: 4399&#226;&#8364;&#8220;4409. <br />     homologs regulate cell division mitotic fidelity proteasome. Cell 112:       19. Mistry H, Wilson BA, Roberts I, O&#226;&#8364;&#8482;Kane CJ, Skeath JB (2004) Cullin-3 regulates <br />     207&#226;&#8364;&#8220;217.                                                                                     <span id='am-316' about='xsp:pattern' typeof='owl:Thing'>pattern</span> formation, external sensory organ development cell <span id='am-3' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-4' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-5' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span>survival</span> during <br />  8. Yahalom A, Kim TH, von Arnim AG, Chamovitz DA (2007) Arabidopsis eIF3e                       Drosophila development. Mech Dev 121: 1495&#226;&#8364;&#8220;1507. <br />     regulated COP9 signalosome impacts development protein                 20. Szlanka T, Haracska L, Kiss I, Dea&#194;&#180;k P, Kurucz E, et al. (2003) Deletion of <br />     translation. Plant J; In press.                                                              proteasomal subunit S5a/Rpn10/p54 causes lethality, multiple mitotic defects <br />  9. Schwechheimer C, Serino G, Callis J, Crosby WL, Lyapina S, et al. (2001)                     overexpression proteasomal genes Drosophila melanogaster. J Cell Sci <br />     Interactions COP9 signalosome E3 ubiquitin ligase SCFTIR1                 116: 1023&#226;&#8364;&#8220;1033. <br />     mediating auxin response. Science 292: 1379&#226;&#8364;&#8220;1382.                                        21. Grumbling G, Strelets V, The FlyBase Consortium (2006) FlyBase: anatomical <br /> 10. Cope GA, Suh GS, Aravind L, Schwarz SE, Zipursky SL, et al. (2002) Role                   data, images queries. Nucleic Acids Research 34: D484&#226;&#8364;&#8220;D <br /> </body></html>