<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Copyright &#195;&#8220; 2009 Genetics Society America <br /> DOI: 10.1534/genetics.109.109686 <br />  <br />  <br />  <br />           A Caenorhabditis elegans RNA-Directed RNA Polymerase Sperm <br />                  Development Endogenous <span id='am-240' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-241' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-243' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-249' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-261' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-264' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-272' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-279' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-296' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-299' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-305' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-307' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-316' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span>RNA Interference</span> <br />  <br />            Jonathan I. Gent,* Mara Schvarzstein,&#226;&#8364;&#160; Anne M. Villeneuve,*,&#226;&#8364;&#160; Sam Guoping Gu,&#226;&#8364;&#161; <br />                      Verena Jantsch,&#194;&#167; Andrew Z. Fire*,&#226;&#8364;&#161;,1 Antoine Baudrimont&#194;&#167; <br />              *Department Genetics, &#226;&#8364;&#160;Department Developmental Biology, &#226;&#8364;&#161;Department Pathology, Stanford University <br />                         School Medicine, Stanford, California 94305 &#194;&#167;Department Chromosome Biology, <br />                                Max F. Perutz Laboratories, University Vienna, A-1030 Vienna, Austria <br />                                                       Manuscript received September 11, 2009 <br />                                                       Accepted publication October 1, 2009 <br />  <br />  <br />                                                                <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> <br />                 Short interfering RNAs (siRNAs) class regulatory effectors enforce gene silencing through <br />              formation RNA duplexes. Although progress identifying capabilities siRNAs in <br />              silencing foreign RNA transposable elements, siRNA functions endogenous gene regulation have <br />              remained mysterious. In certain organisms, siRNA biosynthesis involves novel enzymes act RNA- <br />              directed RNA polymerases (RdRPs). Here analyze function Caenorhabditis elegans RdRP, RRF-3, <br />              spermatogenesis. We loss RRF-3 function resulted pleiotropic defects sperm <br />              development sperm defects led embryonic lethality. Notably, sperm nuclei mutants either <br />              rrf-3 component siRNA pathway, eri-1, frequently surrounded ectopic <br />              microtubule structures, spindle abnormalities subset resulting embryos. Through high- <br />              throughput small RNA sequencing, identified population cellular mRNAs spermatogenic <br />              cells appear serve templates antisense siRNA synthesis. This set genes includes majority <br />              genes known enriched expression spermatogenesis, genes previously <br />              known expressed spermatogenesis. In subset genes, RRF-3 was <br />              required effective siRNA accumulation. These data suggest working model a <br />              major role RRF-3/ERI pathway generate siRNAs set patterns gene expression through <br />              feedback repression set critical targets spermatogenesis. <br />  <br />  <br />  <br />  <br /> R    EPRESSION gene expression small RNAs of <br />        20&#226;&#8364;&#8220;30 nt <span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span> important aspects <br /> of multicellular eukaryotic development. A variety of <br />                                                                             expression. Caenorhabditis elegans, like plants fungi, <br />                                                                             utilizes RNA-copying enzymes called RNA-directed RNA <br />                                                                             polymerases (RdRPs) RNAi machinery <br /> classes small RNA distinct structural features,                     (Smardon et al. 2000; Sijen et al. 2001). While the <br /> modes biogenesis, biological functions                     C. elegans RdRPs nonessential (RRF-1 RRF-2), <br /> identified (reviewed Hutvagner Simard 2008).                         mutations remaining (EGO-1 RRF- <br /> We particularly interested class small RNAs,                    3) lead fertility defects (Smardon et al. 2000; Simmer <br /> called endogenous short interfering RNAs (siRNAs),                          et al. 2002). RRF-3 functionally distinct EGO-1 in <br /> that similar intermediates exogenously                            RRF-3 requirement fertility temperature <br /> triggered RNA interference (RNAi) perfect                          dependent. In addition, RRF-3 activity inhibitory <br /> complementarity mRNA targets. High-throughput                            effect exogenously triggered RNAi (resulting an <br /> sequencing technology provided valuable tool                      ERI, enhanced RNAi, mutant <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span>phenotype</span> rrf-3 <br /> characterization endogenous siRNA populations                            mutants). Mutants lacking RRF-3 ERI <br /> <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> diverse sources, including mouse embryonic                        factor, ERI-1, used experimental tools <br /> stem cells (Babiarz et al. 2008), Drosophila heads                          enhanced sensitivity RNAi-based <br /> (Ghildiyal et al. 2008), Arabidopsis pollen                             screens. One proposed mechanism enhance- <br /> (Slotkin et al. 2009). These siRNAs proposed                      ment RNAi rrf-3 eri mutants a <br /> <span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span> function regulation cellular processes                    competition cofactors exogenously <br /> and genome defense downregulation gene                           triggered RNAi pathway endogenous RNAi <br />                                                                             pathway. Consistent hypothesis, siRNAs corre- <br />                                                                             sponding genes shown Northern <br />   Supporting information available online http://www.genetics.org/ <br /> cgi/content/full/genetics.109.109686/DC1.                                   analysis depend RRF-3 ERI factors for <br />   1 <br />    Corresponding author: Stanford University, 300 Pasteur Dr., Room L235,   accumulation (Duchaine et al. 2006; Lee et al. 2006; <br /> Mail Stop M5324, Stanford, CA 94305-5324. E-mail: afire@stanford.edu        Yigit et al. 2006). Global microarray analyses also <br /> Genetics 183: 1297&#226;&#8364;&#8220;1314 (December 2009) <br />  1298                                                        J. I. Gent et al. <br />  <br /> been undertaken identify messenger RNAs                           Growth conditions: As stated description each <br /> expression affected RRF-3 ERI-1 (Lee et al. 2006;             experiment, animals grown 16&#194;&#176;, 23&#194;&#176;, 25&#194;&#176;. The rrf-3 <br />                                                                         eri-1 mutant fertility phenotypes strongest 25&#194;&#176;, but <br /> Asikainen et al. 2007). <br />                                                                         certain experiments carried milder tempera- <br />    A functional significance RRF-3/ERI pathway                   ture 23&#194;&#176; insufficient numbers embryos <br /> has inferred inability rrf-3, eri-1, eri-3,          produced 25&#194;&#176;. Self-fertilized rrf-3(pk1426) hermaphrodites <br /> eri-5 mutant strains propagate high growth tem-                 approximate surviving brood size 2 25&#194;&#176;, 30 at <br /> perature (Simmer et al. 2002; Duchaine et al. 2006).                    23&#194;&#176;. rrf-3(pk1426) males significantly fertile at <br /> Rather producing temperature-sensitive mutant                      23&#194;&#176;. Self-fertilized rrf-3(ok629) hermaphrodites showed <br />                                                                         temperature-sensitive decreased fertility (they exhibited <br /> protein effects, RRF-3 ERI proteins thought               temperature-independent egg-laying-defective phenotype). <br /> to act temperature-sensitive process, evidenced              Animals used RNA extraction sequencing grown <br /> the predicted truncated presumed nonfunctional                      harvested 25&#194;&#176; (see below). <br /> protein fragments result available                     Spermatogenic cell isolation: A population mixed stage <br /> deletion alleles shared temperature-sensitive              fem-3(q20) hermaphrodites grown 16&#194;&#176; treated a <br />                                                                         1.2% sodium hypochlorite, 0.5 n NaOH solution periodic <br /> phenotypes. rrf-3 mutant animals observed                  vortexing dissolve eggs. After washing M9 <br /> exhibit X-chromosome missegregation (Simmer et al.                      buffer remove hypochlorite, eggs hatched overnight in <br /> 2002) unusual persistence chromatin mark                    M9 solution allowed starve L1 larva. These put <br /> on X chromosome male spermatogenesis                         OP50-seeded, peptone-enriched plates 25&#194;&#176; mature <br /> (Maine et al. 2005). X-chromosome missegregation                    early adult stage (55 hr), animals washed off <br />                                                                         plates M9 buffer. After repeated washes remove <br /> defective spermatogenesis referred pre-                 bacteria, animals diced razor blade a <br /> vious studies eri-1 (Kennedy et al. 2004) eri-3              10-cm tissue culture dish release spermatogenic cells into <br /> eri-5 (Duchaine et al. 2006). Furthermore, eri-3 mutant                 solution (other cells remained attached carcasses). The <br /> sterility rescued insemination wild-type                 mixture carcasses released spermatogonic cells was <br /> sperm (Duchaine et al. 2006).                                           filtered double layer 10 mm Nitex bolting cloth <br />                                                                         (Wildlife Supply) washed times M9 buffer <br />    Here investigated role RRF-3                        flash freezing liquid N2. Visual inspection a <br /> spermatogenesis. We defects evident multiple                   portion transferred microscope slide prior to <br /> stages, including fertilization, defects rrf-3           freezing indicated sample contained mix of <br /> mutant sperm produce subsequent nonviable                           spermatogenic cell types, including primary spermatocytes, <br /> embryos. By using high-throughput sequencing,                        secondary spermatocytes, residual bodies, spermatids <br />                                                                         (supporting information, Figure S5). This procedure is <br /> characterized large population siRNAs present                   optimized small-scale isolation mixed-stage spermato- <br /> spermatogenic cells strong enrichment                   genic cells immediately frozen [we note studies of <br /> antisense siRNAs genes known mRNA expres-                     mature sperm function optimally utilize Percoll gradient and <br /> sion spermatogenesis. While majority                      SM buffer (L&#226;&#8364;&#8482;Hernault Roberts 1995; Machaca et al. <br /> siRNA production spermatogenesis does                        1996) M9 buffer (Klass Hirsh 1981; Miller <br />                                                                         2006)]. <br /> require RRF-3, set genes siRNA                     Male isolation: Set 1: After sodium hypochlorite treatment <br /> production dependent RRF-3. Existing data                      mixed stage animals grown 16&#194;&#176;, released eggs were <br /> indicate increased expression genes rrf-3                  hatched overnight M9 solution starve L1 larva. Fifty- <br /> and/or eri-1 mutants. Taken together, analyses                      hours OP50-seeded NGM plates at <br /> suggest working model RRF-3/ERI                          25&#194;&#176;, starved L1&#226;&#8364;&#8482;s reached adulthood, males were <br />                                                                         separated hermaphrodites filtration 35-mm <br /> pathway generates siRNAs downregulate specific <br />                                                                         Nitex bolting cloth (Wildlife Supply). Embryos passed <br /> genes spermatogenesis, regulation                      filter separated males putting both <br /> playing key role generating functional sperm.                      males embryos unseeded area plate and <br />                                                                         allowing males crawl food. Adult males washed <br />                                                                         plates EN buffer (0.1 m NaCl, 10 mm EDTA) filtered <br />                MATERIALS AND METHODS <br />                                                                         second time immediately prior freezing liquid N2, 53 hr <br /> C. elegans strains, growth conditions, spermatogenic                initially seeded plates L1&#226;&#8364;&#8482;s. Approxi- <br /> cell isolation                                                          mately 95% him-8(e1489) sample 90% the <br />                                                                         rrf-3(pk1426); him-8(e1489) sample male. <br />    Strains: Except noted, C. elegans Bristol N2 (Brenner 1974)          Set 2: After sodium hypochlorite treatment mixed stage <br /> was used wild-type strain studies. The           animals grown 16&#194;&#176;, released eggs OP50-seeded <br /> strains used study following: GE24&#226;&#8364;&#8221;pha-1(e2123)        NGM plates 25&#194;&#176; directly starved L1&#226;&#8364;&#8482;s. Fifty- <br /> I (Schnabel Schnabel 1990); GR1378&#226;&#8364;&#8221;eri-1(mg366) IV                  hours later, reach adulthood, males were <br /> (Kennedy et al. 2004); JK816&#226;&#8364;&#8221;fem-3(q20) IV (Barton et al.               separated hermaphrodites filtration 35-mm <br /> 1987); JK987&#226;&#8364;&#8221;tra-2(q276)/mnC1 dpy-10(e128) unc-52(e444) II              Nitex bolting cloth (Wildlife Supply). Embryos passed <br /> (Okkema Kimble 1991); NL2099&#226;&#8364;&#8221;rrf-3(pk1426) II (Sijen                filter separated males putting both <br /> et al. 2001); PD3303&#226;&#8364;&#8221;rrf-3(pk1426) II; pha-1(e2123) III;                males embryos unseeded area plate and <br /> PD3330&#226;&#8364;&#8221;rrf-3(pk1426) II; him-8(e1489) IV; PD3331&#226;&#8364;&#8221;him-                   allowing males crawl food. Adult males washed <br /> 8(e1489) IV; VC407&#226;&#8364;&#8221;rrf-3(ok629)/mIn1[mIs14 dpy-10(e128)] II             plates EN buffer frozen liquid N2 60 hr after <br /> (http://www.celeganskoconsortium.omrf.org); WM48&#226;&#8364;&#8221;rde-4                  initially seeded plates eggs. Approximately <br /> (ne299) III (Tabara et al. 1999); WM161&#226;&#8364;&#8221;prg-1(tm872) I              90% him-8(e1489) 95% rrf-3(pk1426); him- <br /> (Yigit et al. 2006), tra-2(q122) II (Schedl Kimble 1988).           8(e1489) sample male. <br />                                                 C. elegans RNA-Directed RNA Polymerase                                            1299 <br />  <br /> Assaying embryonic viability                                          dence microtubule wreaths surrounding sperm nuclei in <br />                                                                       reproductive tracts (gonads spermathecae) dissected <br />    Mated tra-2(q122) females allowed lay eggs             self-fertilizing hermaphrodites tra- <br /> several hours NGM plates thin, clear lawns OP50            2(q122) females mated wild-type or <br /> (#1 day seeding). After removing mothers           rrf-3(pk1426) mutant males. Quantification performed <br /> plates, newly laid embryos immediately counted. The          using animals raised maintained 23&#194;&#176;. <br /> surviving progeny counted reached L4            Microtubule wreaths frequent sperm rrf-3 <br /> adult stage. For temperature-shift experiments, matings          self-fertilizing hermaphrodites (15 worms; 237/287 sperm <br /> set L4 stage tra-2(q122) females L4 adult           nuclei microtubule wreaths) tra-2(q122) females that <br /> males raised starting temperature,             mated rrf-3 males (1 worm; 17/20 nuclei with <br /> allowed mate overnight. The gravid females               microtubule wreaths). Self-fertilized eri-1 hermaphrodite <br /> separated males shifted new temperature,          sperm microtubule wreaths 59% sperm (5 <br /> where left sufficient time flush previously      worms; 152/256 nuclei tubulin wreaths). Microtubule <br /> fertilized embryos (6 hr 23&#194;&#176; 25&#194;&#176; 12 hr 16&#194;&#176;). After       wreaths detected control N2 self-fertilizing <br /> this period allowed lay eggs hours      hermaphrodites (2 worms; 0/93 nuclei tubulin wreaths) <br /> (3 hr 23&#194;&#176; 25&#194;&#176; 6 hr 16&#194;&#176;). In experiment,             control tra-2(q122) females mated N2 males (4 worms; <br /> multiple matings set up, minimum gravid            0/86 nuclei tubulin wreaths). Although discrete tubulin <br /> hermaphrodites evaluated growth temperature.             wreaths detected control sperm, hazy/ <br /> Males raised 23&#194;&#176; 25&#194;&#176; [because matings          diffuse tubulin staining detected 4/179 control sperm. <br /> with rrf-3(pk1426) males 25&#194;&#176; inefficient terms           For experiments analyzing wild-type sperm, successful micro- <br /> embryo production]. Twenty-three degrees represents                tubule staining mitotic germ cells gonad <br /> intermediate temperature competent sperm              served internal positive control. <br /> are produced allow continued egg production many <br /> hours mating. <br />  <br />                                                                       Preparation libraries small RNAs sequencing <br />  <br /> Cytological techniques                                                   Small RNA extracted frozen tissue samples with <br />                                                                       mirVana microRNA (miRNA) isolation kit (Ambion). <br />    RAD-51, REC-8, HIM-8, SYP-1 immunostaining:                    Libraries small RNAs prepared using protocol similar <br /> Hermaphrodite gonads dissected fixed pre-                 previously described miRNA cloning (Lau et al. <br /> viously described anti-RAD-51, anti-HIM-8, anti-SYP-1             2001), small RNA species flanked adapter <br /> (Martinez-Perez Villeneuve 2005), anti-REC-8                  sequences allow PCR sequencing. We modified the <br /> (Pasierbek et al. 2001) stainings. Animals synchronized          original protocol key steps allow capture of <br /> by bleaching kill embryos. Embryos        triphosphorylated species sequencing Illu- <br /> allowed hatch overnight NGM plates seeded                  mina genome analyzer system. To remove bias 59 <br /> Escherichia coli (OP50) 16&#194;&#176;, time embryos         monophosphorylated species preserve bias 39 <br /> shifted grow 25&#194;&#176; animals ready dissection.   hydroxylated species, removed monophosphates and <br /> After fixation, samples washed times 13 PBS 5       polyphosphates 59-end small RNA with <br /> min blocked 3% bovine serum albumin (BSA)            Antarctic Phosphatase (New England Biolabs) following the <br /> 13 PBS 20 min room temperature. Antibodies                addition 39 adapter T4 RNA ligase 1 (New England <br /> diluted 13 PBS containing 3% BSA follows: 1:80 anti-RAD-        Biolabs). The resulting hydroxylated 59-ends phosphor- <br /> 51 (Alpi et al. 2003), 1:100 anti-REC-8 (Pasierbek et al. 2001),      ylated treatment T4 polynucleotide kinase (New <br /> 1:500 anti-HIM-8 (Phillips et al. 2005), 1:200 anti-SYP-1         England Biolabs) ATP form subtrate T4 RNA <br /> (MacQueen et al. 2002). Samples incubated over-             ligase 1-mediated addition 59 adapter. T4 polynucleo- <br /> night 4&#194;&#176; humid chamber. After washing times 13       tide kinase treatment addition 39 adapter is <br /> PBS plus 0.1% Tween 20, secondary antibodies applied          important 39 phosphatase 29,39 cyclic <br /> the following dilutions: anti-rabbit DyLight 547 nm (1:200),          phosphodiesterase activity, convert unwanted <br /> anti-rabbit Alexa Fluor 568 (1:500), anti-guinea pig Alexa        RNA degradation products substrate 39 adapter <br /> Fluor 488 (1:500). After 2 hr room temperature, slides        ligation reaction. After ligation 39 59 adapters, <br /> washed times 13 PBS plus 0.1% Tween 20                   adapters extended reverse transcription <br /> mounted Vectashield antifading medium (Vector Laborato-            PCR steps include complete primer binding sites for <br /> ries, Burlingame, CA) containing 2 mg/ml DAPI. Samples           subsequent amplification sequencing reactions. To pro- <br /> examined Zeiss Axio Imager M1 microscope. Images               tect cross-contamination allow pooling of <br /> were recorded Spot camera (Diagnostics Instruments).           samples, modified 59 adapter sequence include the <br /> For multicolor immunostaining, monochrome stacks images            Illumina genomic sequencing primer immediately followed <br /> were captured separately emission wavelength (Meta-          4-nt barcode. The pair barcodes used set 1 wild- <br /> Vue software; Molecular Devices, Sunnyvale, CA). Three-               type mutant samples switched set 2 wild-type <br /> dimensional stacks images deconvolved (AutoQuant              mutant samples guard potential biases <br /> software; AutoQuant Imaging, Troy, NY) projected (Heli-           introduced adapter sequences. The samples PAGE <br /> con Focus software; http://helicon.com.ua/heliconfocus/).             purified stages: initial RNA extraction and <br />    a-Tubulin immunostaining: We used fixation procedures              ligation step (except isolated spermato- <br /> optimized preservation microtubules (Gonczy et al.             genic cell sample, PAGE purification was <br /> 1999; Oegema et al. 2001), modifications previously           skipped conserve material). Additional details each <br /> described (Martinez-Perez et al. 2008). FITC-conjugated               sample provided Table S1. The following oligos were <br /> anti-a-tubulin monoclonal antibody DM 1A (Sigma) used             used: <br /> at concentration 1 mg/ml. Images acquired stacks <br /> of optical sections 0.2-mm intervals using Deltavision           39 adapter (IDT linker-1 59 adenylation 39 dideoxyC): <br /> deconvolution microscopy system. We quantified inci-                 rAppCTGTAGGCACCATCAATC; <br />  1300                                                          J. I. Gent et al. <br />  <br /> 59 adapter (i.e., AF-PP-341 DNA/RNA hybrid oligo 59                16,887 total genes, 178 increased mRNA levels rrf- <br />   amino modifier 4-nt barcode 39-end):                         3(pk1426), 46 decreased levels. Of 18 spermato- <br />   ACGCTCTTCCGATCTrGrUrUrA;                                                genesis candidate RRF-3 targets, 3 increased mRNA levels <br /> Dual reverse transcription primer PCR primer (AF-JIG-37,              rrf-3(pk1426) (15.8-fold enrichment; P-value &#194;&#188; 8.8 3 10 4); <br />   complementary IDT linker-1): CAAGCAGAAGACGGCA                        decreased levels. Of 16,887 genes, 706 had <br />   TACGAattgatggtgcctacag;                                                 increased mRNA levels eri-1(mg366), 263 de- <br /> PCR primer (AF-JIG-40, containing 59 adapter sequence):                   creased levels. Of 18 spermatogenesis candidate RRF-3 <br />   AATGATACGGCGACCACCGACACTCTTTCCCTACACGA                                  targets, 8 increased mRNA levels eri-1(mg366) (10.6-fold <br />   CGCTCTTCCGATCT.                                                         enrichment; P-value &#194;&#188; 2.9 310 7). These 8 included 3 with <br />                                                                           elevated mRNA levels rrf-3(pk1426). None 18 had <br />                                                                           decreased levels eri-1(mg366). When considered 44 <br />                                                                           candidate RRF-3 targets just 18 siRNAs <br /> Sequence analyses                                                         present spermatogenic cell sample, numbers were <br />                                                                           highly significant: fold enrichments 6.5 4.4 P- <br />    Thirty-six-nucleotide sequencing reads generated                  values 1.5 3 10 2 4.4 3 10 4 rrf-3(pk1426) eri- <br /> using Illumina Genome Analyzer system. The 39 adapter                 1(mg366), respectively. <br /> sequences trimmed silico scanning 39- <br /> end sequence instance 4 nt the <br /> adapter: CTGT. Sequence reads lacked CTGT 39                                            RESULTS <br /> most 13 nt excluded analysis. After barcode <br /> sorting, 4 nt barcode trimmed 59-ends                 Compromised spermatogenesis accounts re- <br /> yield 19- 28-nt reads corresponding captured small              duced brood size X-chromosome loss rrf- <br /> RNAs. siRNA alignment counts produced aligning                3(pk1426) hermaphrodites: We set determine <br /> reads cDNA reference set using local installation               rrf-3(pk1426) reduced brood size X- <br /> BLAT (Kent 2002), default parameters, tileSize <br />                                                                           chromosome missegregation phenotypes attribut- <br /> was set 10 stepSize 5. The term &#226;&#8364;&#732;&#226;&#8364;&#732;siRNA alignment <br /> count&#226;&#8364;&#8482;&#226;&#8364;&#8482; (above) refers number matches                     able defects sperm oocytes. C. elegans exists two <br /> individual cDNA obtained set reads. The cDNA                  sexes: males, produce sperm, hermaph- <br /> data set, derived C. elegans WS190 genome assem-                 rodites, produce sperm oocytes in <br /> bly, downloaded http://www.wormbase.org:80/                      gonad. Defects gamete line result <br /> biomart/martview. The data set known transposons sequen- <br />                                                                           sterility self-fertilizing hermaphrodites. rrf- <br /> ces downloaded http://www.sanger.ac.uk/Projects/ <br /> C_elegans/REPEATS/elegans.lib. Read counts alignment                  3(pk1426) self-fertilizing hermaphrodites produced very <br /> counts listed Table S1.                                            fertilized embryos 25&#194;&#176;, 90% non- <br />                                                                           viable (Figure 1A). Here subsequent experi- <br />                                                                           ments, defined &#226;&#8364;&#732;&#226;&#8364;&#732;nonviable&#226;&#8364;&#8482;&#226;&#8364;&#8482; failure rise to <br /> Statistical methods                                                       L4 stage larva adults. We preponderance <br />                                                                           (or all) nonviability accounted by <br />    P-value calculations identification genes RRF-3-           embryonic lethality prior hatching. To determine <br /> dependent siRNAs: Gene-by-gene siRNA alignment counts <br />                                                                           source reduced embryo count viability, we <br /> were tallied compared sample. To measure fold- <br /> changes alignment count samples particular               set matings rrf-3(pk1426) hermaphrodites <br /> gene, alignment count converted proportion             wild-type N2 males 25&#194;&#176;. Fertilization wild-type <br /> by dividing alignment count gene sum            sperm rescued rrf-3(pk1426) brood size reduction in <br /> alignments complete cDNA reference set. To obtain                terms numbers embryos produced and <br /> measure statistical significance differences between <br />                                                                           embryonic viability (Figure 1, A B), indicating that <br /> samples, assumed difference population <br /> proportions gene calculated probability             phenotypes attributed defects during <br /> (P-value) measuring difference sample proportions              spermatogenesis. <br /> least extreme observed. A one-tailed test used             To determine X-chromosome missegrega- <br /> since prior expectations proportions certain          tion linked spermatogenesis rrf-3(pk1426) <br /> genes decrease rrf-3(pk1426) samples (see File S1 <br />                                                                           hermaphrodites, utilized genetic test takes <br /> for representative calculation). The standard errors the <br /> means sample proportion differences calculated                advantage C. elegans sex determination system. <br /> the basis predicted normal distributions sample                 The ratio X-chromosomes autosomes C. elegans <br /> proportions, justified large sample size              determines sex: animals X chromosomes <br /> the central limit theorem (the proportions equivalent            sets autosomes hermaphrodites; animals <br /> to means). The sample proportion differences standard- <br />                                                                           X sets autosomes male (Nigon <br /> ized Z-scores dividing standard errors means <br /> for proportion differences.                                           1951; Madl Herman 1979). Hence, normal self- <br />    P-values fold-enrichment calculations overlap                  fertilization, hermaphrodite contributes X from <br /> between sets genes RRF-3-dependent siRNAs                     sperm ovum resulting <br /> genes elevated mRNA levels rrf-3 eri-1 mutants: P-            embryos virtually ( 99.8%) XX hermaphrodites. <br /> values derived using Fisher&#226;&#8364;&#8482;s exact test following <br />                                                                           Generation XO males XX hermaphrodite self- <br /> numbers: 16,887 genes represented cDNA data <br /> set probes microarray used Asikainen et al.                 fertilization indicative X-chromosome loss non- <br /> (2007). Of 21 genes identified spermatogenesis             disjunction sperm oocyte lineage. To test <br /> candidate RRF-3 targets, 18 included set. Of             X-chromosome loss rrf-3(pk1426) oogenesis, set <br />                                               C. elegans RNA-Directed RNA Polymerase                                              1301 <br />  <br />  <br />  <br />  <br />   Figure 1.&#226;&#8364;&#8221;Lack embryos, nonviable embryos, X-chromosome loss attributable defective spermatogenesis rrf-3 <br /> mutants. Individual hermaphrodites allowed self-fertilize crossed males 25&#194;&#176; allowed lay <br /> eggs 6-hr window followed 18-hr window. The progeny periods counted reached L4 <br /> and adult stage. Cases matings unsuccessful, evidenced embryos lack male progeny, not <br /> counted. Multiple matings set progeny gravid hermaphrodites counted case. (A) <br /> Rescue rrf-3(pk1426) embryo production viability wild-type sperm (6-hr window).The embryos laid 6 <br /> hr counted addition surviving progeny estimate number nonviable embryos. Error bars denote standard <br /> errors means. (B) Rescue rrf-3(pk1426) brood size (24-hr window). The number surviving progeny 6- <br /> and 18-hr windows combined. Error bars denote standard errors means. (C) Diagram assay loss X chro- <br /> mosome oogenesis. pha-1(e2123) temperature-sensitive, recessive, embryonic lethal <span id='am-162' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-163' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span>mutation</span> ensures only <br /> outcrossed progeny survive 23&#194;&#176;. The tra-2(q276) mutation causes XX animals, normally hermaphrodites, to <br /> develop partially fertile males. (D) Wild-type levels X-chromosome loss rrf-3(pk1426) oogenesis. Multiple matings were <br /> set up, hermaphrodites males, exception rrf-3(pk1426) self-fertilizations, were <br /> done individual hermaphrodites. The number male progeny scored reached L4 adult stage. The <br /> total number progeny counted, n, smaller expected wild-type matings inefficiency tra- <br /> 2(q276) males. These crosses set 23&#194;&#176; 25&#194;&#176; increased success. <br />  <br />  <br /> up matings tra-2(q276) XX-transformed males,                     exhibited extremely low numbers spontaneous males <br /> which contribute X sperm (like her-                    (wild type: 1 479; rrf-3(pk1426): 2 721) (Figure 1D). <br /> maphrodite). Unless X chromosome lost                   These results indicate oogenesis source of <br /> oogenesis, hermaphrodites mated tra-2(q276) males                X-chromosome loss rrf-3(pk1426) propagation, impli- <br /> give rise entirely XX hermaphrodite progeny                 cating spermatogenesis affected process. While <br /> (Figure 1C) (Okkema Kimble 1991). We set                       RRF-3 additional functions oogenesis, these <br /> matings rrf-3(pk1426) wild-type her-                results indicate fertility and <br /> maphrodites tra-2(q276) males. To select out-                X-chromosome segregation mutant phenotypes be <br /> crossed progeny, hermaphrodites carried                    accounted roles RRF-3 spermatogenesis. <br /> temperature-sensitive mutation pha-1 causes                     RRF-3 required spermatocyte cell division: To <br /> self-fertilized progeny die embryos (Schnabel                   investigate nature sperm defect, we <br /> and Schnabel 1990). The progeny cross                       examined germlines rrf-3(pk1426) hermaphrodites <br />  1302                                                      J. I. Gent et al. <br />  <br />                                                                                          Figure 2.&#226;&#8364;&#8221;RRF-3 required normal <br />                                                                                       spermatocyte cell division. (A) Abnormal <br />                                                                                       nuclear structures lack sperm visible <br />                                                                                       DAPI staining fixed hermaphrodite <br />                                                                                       gonads undergoing spermatogenesis at <br />                                                                                       25&#194;&#176;. Nuclei wild-type mutant <br />                                                                                       spermatocytes (white arrows) appeared vis- <br />                                                                                       ibly normal, nuclei subsequent <br />                                                                                       stages appeared fragmented or <br />                                                                                       bridged mutant (blue arrowheads). <br />                                                                                       A wild-type dividing spermatocyte nuclei <br />                                                                                       (blue arrow) spermatid nuclei (or- <br />                                                                                       ange arrow) indicated. Bar, <br />                                                                                       10 mm. (B) Multiple misshapen <br />                                                                                       Hoechst-staining structures mutant <br />                                                                                       spermatocytes 25&#194;&#176;. Unfixed sperm and <br />                                                                                       spermatocytes released rrf-3(pk1426) <br />                                                                                       dissected gonads stained Hoescht <br />                                                                                       33342 multiple nuclear structures <br />                                                                                       individual cells (bottom pan- <br />                                                                                       els), inconsistent sizes shapes <br />                                                                                       compared N2 control sperm nuclei <br />                                                                                       (top). Mutant worms produced sperm <br />                                                                                       visibly similar N2 (e.g., up- <br />                                                                                       left corner panel). (C) <br />                                                                                       Quantification DAPI-staining structures <br />                                                                                       fixed hermaphrodite gonads 25&#194;&#176;. <br />                                                                                       Nuclear structures counted the <br />                                                                                       proximal gonad expected timing <br />                                                                                       spermatogenesis completion (after the <br />                                                                                       appearance oocyte prior <br />                                                                                       ovulation; stage sperm have <br />                                                                                       moved spermatheca). <br />                                                                                       Compact, sperm-like nuclei equiv- <br /> alent cell counts presence multiple DNA structures individual cells mutant. Values shown <br /> are averages obtained 9 N2 gonads 10 rrf-3(pk1426) gonads. Error bars denote standard errors means. <br />  <br />  <br /> prior completion spermatogenesis                  numbers wild type. Additional DAPI staining of <br /> 25&#194;&#176;. We investigated chromosome organization                   released gonads revealed production com- <br /> meiotic prophase using antibodies labeling RAD-51,                    pact sperm-like nuclei delayed decreased. <br /> which functions recombination decorates chias-                 At expected time spermatogenesis completion <br /> mata (Alpi et al. 2003; Colaiacovo et al. 2003);                  (prior ovulation appearance oocytes), rrf- <br /> synaptonemal complex components REC-8 SYP-1                       3 mutants fivefold reduction compact, sperm- <br /> (Pasierbek et al. 2001; MacQueen et al. 2002); HIM-               like nuclei relative N2 (Figure 2C). By time the <br /> 8, marks specific domain X chromosome                  ovulation, number rrf-3 mutants sperm-like <br /> (Phillips et al. 2005). This analysis did reveal              nuclei relative N2 increased half N2 levels <br /> overt defects meiotic prophase (Figure S1 Figure               (data shown), gonads contained cells <br /> S2). In addition, DAPI-stained nuclei process               multiple misshapen nuclei. Together, these <br /> transitioning prophase meiotic                  phenotypes indicate defects developmental progres- <br /> division appeared normal rrf-3 mutants (Figure 2A).                sion cell division processes spermatogenesis <br /> Cytological abnormalities rrf-3 mutant germ cells            rrf-3(pk1426). <br /> started evident primary spermato-                    An rrf-3 paternal effect embryogenesis: Since rrf- <br /> cyte stage, DAPI staining revealed nuclear abnor-               3(pk1426) animals elevated temperatures yield high <br /> malities small bridged structures (Figure 2A).             percentage nonviable embryos, asked whether <br /> Further analysis unfixed late-stage spermatocytes               RRF-3 functions directly embryos spermato- <br /> DIC Hoechst staining revealed unusual cell                   genesis process linked embryogenesis. To test this, we <br /> morphology presence multiple nuclear                       set matings rrf-3(pk1426) males tra- <br /> structures individual mutant spermatocytes (Fig-               2(q122) females, genetically XX hermaphro- <br /> ure 2B). Other spermatocytes gonads                   dites feminized germlines produce only <br /> appeared develop obvious morphological                     oocytes (Schedl Kimble 1988). We rrf- <br /> abnormalities produce residual bodies                      3(pk1426) males produced sperm capable of <br /> sperm normal size shape, smaller               fertilization, resulting embryos were <br />                                           C. elegans RNA-Directed RNA Polymerase                                          1303 <br />  <br />  <br />  <br />                                                                              Figure 3.&#226;&#8364;&#8221;RRF-3 involved temperature- <br />                                                                           sensitive spermatogenic process effects on <br />                                                                           embryonic viability. (A) The viability embryos <br />                                                                           sired rrf-3(pk1426) males determined tem- <br />                                                                           perature spermatogenesis. rrf-3(pk1426) N2 <br />                                                                           males raised indicated temperatures were <br />                                                                           mated tra-2(q122) females. The level em- <br />                                                                           bryonic viability estimated counting both <br />                                                                           total number eggs laid (n) num- <br />                                                                           ber surviving progeny (as described in <br />                                                                           materials methods). (B) Diagram assay <br />                                                                           embryonic requirement paternally <br />                                                                           delivered, wild-type copy rrf-3. rrf-3(pk1426) ho- <br />                                                                           mozygous hermaphrodites mated with <br />                                                                           males heterozygous rrf-3(pk1426) <br />                                                                           mIn1 inversion (which carries wild- <br />                                                                           type rrf-3 locus transgene driving pharyn- <br />                                                                           geal GFP expression). Inheritance mutant <br />                                                                           wild-type rrf-3 allele tracked GFP <br />                                                                           expression. No marker outcrossed progeny <br />                                                                           necessary sperm defect in <br />                                                                           rrf-3(pk1426) hermaphrodites. Animals were <br />                                                                           raised matings set 25&#194;&#176;. (C) A wild-type <br />                                                                           copy rrf-3 dispensable embryogenesis. <br />                                                                           GFP- non-GFP-expressing progeny were <br />                                                                           counted L4 adult stage determine the <br />                                                                           proportion paternally inherited mIn1. To <br />                                                                           provide control potential effects of <br />                                                                           inversion dominant transgene <br />                                                                           marker, 1/mIn1 males used. A total of <br />                                                                           526 progeny resulting matings were <br />                                                                           counted 1/mIn1, total 675 progeny <br />                                                                           resulting matings counted rrf- <br />                                                                           3(pk1426)/mIn1. Error bars denote standard er- <br />                                                                           ror means. <br />  <br />  <br />  <br />  <br /> nonviable ( 37% 23&#194;&#176;). This effect embryos de-           used scheme Figure 3B, setting matings <br /> pended temperature spermatogenesis            rrf-3(pk1426) homozygous hermaphrodites and <br /> occurred independent temperature            rrf-3(pk1426)/mIn1 heterozygous males. mIn1 chro- <br /> which embryogenesis occurred (Figure 3A). The pater-          mosome inversion chromosome II containing the <br /> nally derived lethality presence      wild-type rrf-3 locus transgene insertion drives <br /> second, unrelated recessive mutation trans-           pharyngeal GFP expression (Edgley Riddle 2001), <br /> heterozygous males bearing rrf-3(pk1426)          enabling determine paternal allele is <br /> deletion allele, rrf-3(ok629), produced nonviable        inherited progeny. We <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span>survival</span> was <br /> embryos. Such males heterozygous            affected physical inheritance wild-type rrf-3 <br /> extraneous mutations outside rrf-3, contribution       allele (on mIn1), indicating rrf-3 expression not <br /> of mutations, any, minimized. In        required separation joined secondary sperma- <br /> setting test rrf-3 requirement, utilized    tocytes spermatids embryogenesis (Figure 2 <br /> trans-heterozygote ok629 homozygotes be-          Figure 3C). Together early temperature- <br /> cause male-tail-defective phenotype strain        sensitive period (prior fertilization embryo as <br /> (which definitively assigned ok629). In        shown Figure 3A), result implicates spermato- <br /> matings tra-2(q122) females ok629/pk1426          genesis crucial timing RRF-3 expression <br /> trans-heterozygote males 25&#194;&#176;, resulting embryos        activity. <br /> showed similar levels nonviability embryos sired           Roles rrf-3 eri-1 preventing microtubule <br /> by pk1426 homozygous males: 84% nonviability             accumulation sperm abnormal microtubule <br /> pk1426/pk1426 72% nonviability ok629/pk1426,         structures one-cell embryos: To examine basis <br /> contrasted 2% nonviability N2 (Figure S3).          embryonic lethality, stained wild-type rrf- <br />    To distinguish paternal effect embryonic         3(pk1426) self-fertilized embryos antibody <br /> requirement paternally delivered rrf-3(1) allele,       a-tubulin. One particularly notable defect was <br />  1304                                                     J. I. Gent et al. <br />  <br />  <br />  <br />  <br />    Figure 4.&#226;&#8364;&#8221;Abnormal microtubule structures rrf-3 mutant embryos sperm. An embryo (A) sperm (B) stained with <br /> Hoechst <span id='am-3' about='rdfs:label' typeof='owl:Thing'>label</span> chromosomes (red) immunostained antitubulin antibody label microtubules (green). (A) One-cell <br /> embryo (prior meeting male female pronuclei) laid rrf-3(pk1426) hermaphrodite 23&#194;&#176;. Although asters are <br /> normally associated male pronucleus stage wild-type embryos, mutant embryo asters (marked by <br /> arrowheads) associated male pronucleus. An arrow marks polar bodies, located anterior end the <br /> embryo. Bar, 10 mm. (B) Sperm wild-type hermaphrodite (top row) rrf-3(pk1426) hermaphrodites (bottom three <br /> rows), raised indicated temperatures. Arrowheads highlight sperm nuclei surrounded tubulin wreaths observed <br /> in rrf-3 mutant sperm 23&#194;&#176;. The insets middle row enlarged view tubulin wreath encasing nucleus an <br /> rrf-3 mutant sperm. Bar, 5 mm. <br />  <br />  <br /> the presence abnormal microtubule structures                 observed wild-type sperm (n &#194;&#188; 179). The fact that <br /> subset mutant embryos one-cell stage (i.e.,            tubulin largely undetectable expected basis <br /> from formation maternal meiotic spindle                       previous observations microtubules secondary <br /> through metaphase mitotic division). Spe-               spermatocytes effectively segregated enucle- <br /> cifically, detected supernumerary microtubule asters              ated cytoplast called residual body (Roberts et al. <br /> associated sperm pronucleus tripolar spin-               1986; Ward 1986; Ward et al. 1986). At 16&#194;&#176;, we <br /> dles 5 35 rrf-3 mutant one-cell embryos (Figure 4A),           did dead embryos, mutant sperm lacked the <br /> whereas structures observed control               microtubule wreath phenotype. <br /> embryos (0 34; P-value: 0.029).                                      Because meiotic DNA damage resulting com- <br />    Immunofluorescence analysis revealed striking                promised small RNA machinery Drosophila results in <br /> microtubule-based abnormalities rrf-3 mutant sperm.               microtubule polarization defects (Chen et al. 2007; <br /> At 23&#194;&#176;, observed bright wreaths microtubules                   Klattenhoff et al. 2007; Pane et al. 2007), tested <br /> surrounding nuclei hermaphrodite                     possibility microtubule wreaths rrf-3 <br /> male-derived sperm (Figure 4B Figure S4). These                  mutant sperm consequence unrepaired <br /> microtubule wreaths clearly visible 83% (n &#194;&#188;                 meiotic DNA double-strand breaks (DSBs). (Although <br /> 307) rrf-3 mutant sperm. In contrast, 2% wild-            normal kinetics RAD-51 foci rrf-3 mutants <br /> type sperm exhibited detectable tubulin staining                 suggests successful DSB repair, data not <br /> (which diffuse appearance) sperm                exclude possibility persistent meiotic DNA dam- <br /> centrioles. Microtubule wreath structures                 age.) Specifically, looked microtubule wreaths <br />                                           C. elegans RNA-Directed RNA Polymerase                                        1305 <br />  <br />  <br />                                                                              Figure 5.&#226;&#8364;&#8221;Abnormal microtubule structures <br />                                                                           eri-1 mutant embryos sperm paternal- <br />                                                                           effect nonviability eri-1 mutant males. <br />                                                                           (A) eri-1(mg366) sperm (B) embryo her- <br />                                                                           maphrodites grown 23&#194;&#176; stained with <br />                                                                           Hoechst label chromosomes (red) antitu- <br />                                                                           bulin antibody label microtubules (green). <br />                                                                           (A) Some eri-1(mg366) sperm contain abnormal <br />                                                                           microtubule wreath structures (arrowheads) sim- <br />                                                                           ilar observed rrf-3 mutant sperm. Bar, <br />                                                                           5 mm. (B) One-cell eri-1 mutant embryo tri- <br />                                                                           polar spindle. Arrowheads mark the <br />                                                                           spindle poles. Arrows mark polar bod- <br />                                                                           ies, indicate anterior embryo. <br />                                                                           Bar, 10 mm. (C) eri-1(mg366) males sire nonviable <br />                                                                           embryos elevated temperatures. The fraction <br />                                                                           nonviable embryos estimated counting <br />                                                                           number eggs laid resulting <br />                                                                           number surviving progeny. Matings set <br />                                                                           23&#194;&#176; 25&#194;&#176;. Error bars denote stan- <br />                                                                           dard errors means. <br />  <br />  <br /> around sperm nuclei 23&#194;&#176; double-mutant worms             Ward 1995, describes dicing method, from <br /> lacking SPO-11 (the enzyme required inducing              Miller 2006, uses M9 buffer does require a <br /> meiotic DSBs) (Keeney et al. 1997; Dernburg et al.            Percoll step obtain sperm biochemical analysis). fem- <br /> 1998) RRF-3. Microtubule wreaths present          3(q20) animals produce excess sperm oocytes <br /> sperm double mutant, indicating           embryos 25&#194;&#176; (Barton et al. 1987). Functionality of <br /> structures depend meiotic DSBs induced           fem-3(gf) XX sperm demonstrated previously by <br /> SPO-11 (data shown).                                      ability sire live embryos following artificial in- <br />    Given similarities fertility RNAi      semination (Lamunyon Ward 1994). The popula- <br /> phenotypes rrf-3 eri-1 mutants, examined            tion cells isolated used sequencing shown <br /> tubulin distribution sperm embryos eri-           Figure S5. <br /> 1(mg366) hermaphrodites raised 23&#194;&#176;. We             We prepared sequencing libraries using method that <br /> microtubule wreaths sperm (59%, n &#194;&#188; 256)               allows capture small RNAs independently 59 <br /> microtubule/spindle defects embryos (4/20) pro-            phosphorylation status. In C. elegans, majority of <br /> duced eri mutants (Figure 5, A B). We          RdRP products 59 triphosphate (Ruby et al. 2006; <br /> confirmed role ERI-1 producing functionally          Pak Fire 2007; Sijen et al. 2007), standard high- <br /> normal sperm showing mating eri-1(mg366)              efficiency library preparation methods (e.g., Lau et al. <br /> males tra-2(q122) females resulted nonviable          2001) rely 59 monophosphate, character- <br /> embryos elevated temperatures (Figure 5C).                 istic microRNAs DCR-1 products. The <br />    Identification siRNAs present spermatogenic          ability capture 59-triphosphorylated siRNAs crit- <br /> cells: During progression distinct stages     ical analysis, terminus structure is <br /> of spermatogenesis premeiotic syncitium           expected present initial RdRP transcript <br /> mature sperm, large number genes choreo-         retained unless covalent 59-end <br /> graphed ensure proper genes expressed         processing. In materials methods, describe <br /> at appropriate level developmental stage (e.g.,       procedure preparing libraries using a <br /> Johnston et al. 2008). Since RNAi-related factors     modification Lau et al. (2001) addition a <br /> have implicated spermatogenesis               dephosphorylation single phosphate addition <br /> mutant studies, expected RNA-based regulatory         step following 39 adapter ligation. The resulting libraries <br /> machinery contribute levels cho-     small RNAs diverse sequence composition, <br /> reography. Consequently, characterized small RNAs      obtained large numbers distinct small RNA <br /> present male germline. We took sequencing-           species including microRNAs, piwi-interacting RNAs <br /> based approach this, adapting sperm isolation          (also called piRNAs 21U-RNAs), siRNAs. These <br /> optimizing high-throughput sequencing methods gen-         sequences available NCBI short read archive <br /> erate extensive data set small RNA sequences       (GEO accession number GSE18429). We aligned the <br /> isolated spermatogenic cells. Spermatogenic cells        small RNA sequences cDNA data set downloaded <br /> isolated adult fem-3(q20) XX animals using tech-       http://www.wormbase.org generate list of <br /> nique yields large proportion sperm precursors   siRNA alignment counts annotated gene <br /> addition mature sperm (adapted Lamunyon           (where &#226;&#8364;&#732;&#226;&#8364;&#732;siRNA alignment count&#226;&#8364;&#8482;&#226;&#8364;&#8482; gene the <br />  1306                                                     J. I. Gent et al. <br />  <br />  <br />  <br />  <br />   Figure 6.&#226;&#8364;&#8221;siRNA alignments mRNA: antisense predominance diversity intragenic locations. A modified UCSC <br /> Genome Browser Display illustrates C05C12.5 annotation siRNAs spermatogenic cell sample. For C05C12.5 <br /> annotation, solid end segments denote untranslated regions, solid segments denote exons, line segments denote <br /> introns. White arrows exons indicate gene&#226;&#8364;&#8482;s orientation. siRNA alignments represented short black segments. All <br /> but siRNA alignments outlined dashed red box antisense C05C12.5. Note siRNAs span exon&#226;&#8364;&#8220;exon <br /> boundaries produce genomic alignments. <br />  <br /> number alignments produced                       expression patterns multiple growth conditions, <br /> complete set small RNA sequences) These data                  developmental time points, mutant backgrounds <br /> tabulated File S1. Also included NCBI short                (Kim et al. 2001). For gene categorized having <br /> read archive alignment files psl format,            siRNAs spermatogenic cells, required the <br /> be directly uploaded UCSC Genome Browser                    number alignments mRNA isolated <br /> (http://www.genome.ucsc.edu) viewing. Two notable                spermatogenic cell siRNAs three. A total of <br /> features siRNA alignments bias                   4723 21,133 genes cDNA reference set <br /> antisense orientation relative annotated mRNA                 met criterion. These 4723 genes 3.7- <br /> and diversity positions individual target            3.5-fold greater-than-expected overlap the <br /> genes, illustrated siRNAs gene C05C12.5                 spermatogenesis-enriched mountain 4 expression <br /> (Figure 6). The antisense orientation spermato-               category, respectively (accounting 84% total <br /> genesis siRNAs suggests synthesis RdRP                     spermatogenesis-enriched category 77% the <br /> consistent potential modulatory roles small            mountain 4 category, Figure 7A). Conversely, genes <br /> RNAs &#226;&#8364;&#732;&#226;&#8364;&#732;standard&#226;&#8364;&#8482;&#226;&#8364;&#8482; gene-silencing model                  spermatogenesis-enriched mountain 4 <br /> which Argonuate-bound short antisense RNA seg-                    categories significantly underrepresented among <br /> ment catalytically locates silences mRNAs                   group mRNAs identified isolated sper- <br /> extended regions homology (reviewed Hutvagner                  matogenic cell sample siRNA alignment counts of <br /> and Simard 2008).                                                    fewer 3 (4.8- 3.3-fold less-than-expected, <br />    The small RNA-ome sperm large, com-                  respectively). These results indicate siRNA abun- <br /> plex, likely reflect numerous distinct regulatory             dance spermatogenic cells correlates mRNA <br /> processes. We began analyzing small RNAs likely              expression spermatogenesis siRNA <br /> roles regulation specific mRNAs compiling               production general feature gene activity during <br /> siRNA alignment counts annotated mRNA-                      spermatogenesis, involving majority genes with <br /> encoding gene data set. While comprehensive                enriched expression spermatogenesis. <br /> list spermatogenesis-expressed mRNAs unavailable                  Identification RRF-3-dependent siRNAs: To iden- <br /> for C. elegans, studies categorized mRNAs               tify siRNAs depend RRF-3 pro- <br /> with higher relative expression spermatogenic cells               duction, analyzed siRNA expression profiles from <br /> than tissues. We compared list genes            rrf-3 mutant samples. To avoid possible confound- <br /> siRNAs present spermatogenic cell sample                 ing effects fem-3 mutant background (since FEM-3 <br /> two gene lists, &#226;&#8364;&#732;&#226;&#8364;&#732;spermatogenesis-enriched&#226;&#8364;&#8482;&#226;&#8364;&#8482; &#226;&#8364;&#732;&#226;&#8364;&#732;moun-             affects control spermatogenesis), carried these <br /> tain 4.&#226;&#8364;&#8482;&#226;&#8364;&#8482; The spermatogenesis-enriched category                  experiments using populations karyotypically <br /> derived comparison mRNA expression                        normal XO males. For rrf-3 mutant wild-type <br /> mutant hermaphrodites masculinized                  control samples, him-8(e1489) mutation compro- <br /> to produce sperm feminized produce                   mises X-chromosome pairing primarily oocytes was <br /> oocytes (Reinke et al. 2004). Mountain 4 derived                 used generate populations large numbers of <br /> from study grouped genes according correlated              males (Hodgkin et al. 1979). <br />                                              C. elegans RNA-Directed RNA Polymerase                                       1307 <br />  <br />                                                                   chose track 49 genes expression likely <br />                                                                   dependent RRF-3 criterion threefold- <br />                                                                   reduced alignment count P-value ,0.005 in <br />                                                                   sets experiments (Table 1; materials and <br />                                                                   methods). <br />                                                                      This list 49 genes intended exemplary rather <br />                                                                   complete: genes appear similar <br />                                                                   properties quite reaching arbitrary statisti- <br />                                                                   cal threshold criteria, produce <br />                                                                   siRNAs expressed levels insufficient for <br />                                                                   statistical significance analyses. In addition, <br />                                                                   alignment small RNA sequences genome <br />                                                                   regard predicted coding status revealed <br />                                                                   additional unannotated loci small RNAs dependent <br />                                                                   RRF-3, previously identified RRF-3- <br />                                                                   dependent locus X chromosome called X <br />                                                                   cluster (Ambros et al. 2003; Duchaine et al. 2006). The <br />                                                                   genes list 49 highest siRNA levels, <br />                                                                   C44B11.6 K02E2.6, previously shown by <br />                                                                   siRNA Northern analysis require RRF-3 siRNA <br />                                                                   accumulation, providing independent confirmation <br />   Figure 7.&#226;&#8364;&#8221;Overlap siRNA- mRNA-based <br />                                                                   experimental approach (Duchaine et al. 2006; <br /> gene expression categories. (A) The genes siRNAs pre- <br /> sent spermatogenic cells majority genes       Lee et al. 2006). <br /> mRNAs enriched expression spermatogenesis.               For genes regulated RRF-3- <br /> Venn diagrams display numbers genes shared be-      driven mechanism spermatogenesis, would <br /> tween categories exclusive category. Genes       expect mRNAs siRNAs be <br /> with siRNAs present spermatogenic cells siRNA <br />                                                                   expressed spermatogenesis. We 5.2-fold <br /> count isolated spermatogenic cell <br /> sample. Genes mountain 4 (Kim et al. 2001) spermato-       enrichment mountain 4 expression category and <br /> genesis-enriched (Reinke et al. 2004) categories associated   4.6-fold enrichment spermatogenesis-enriched <br /> with enriched mRNA expression spermatogenesis           category list 49 candidate RRF-3 target genes, <br /> measured comparative microarray analyses. Only genes           indicating substantial fraction RRF-3-regulated <br /> that probes microarrays present the <br />                                                                   genes expressed spermatogenesis (Table 1; <br /> cDNA data set used aligning small RNA sequences were <br /> counted category. (B) The candidate RRF-3 regula-         Figure 7B). We used siRNA alignment count from <br /> tory targets enriched genes siRNAs present        spermatogenic cells means ranking candidate <br /> spermatogenic cells. Relative siRNA expression levels        gene list terms likelihood spermatogenesis <br /> measured siRNA sequencing sets him-                gene regulation. As before, split list two <br /> 8(e1489) adult males (mutant wild type rrf-3). Forty- <br />                                                                   categories using spermatogenesis siRNA alignment <br /> nine genes siRNA alignment counts decreased <br /> by threefold P-value ,0.005 rrf-            count 3 threshold, resulting 21 genes with <br /> 3(pk1426) sets categorized candidate RRF-3        spermatogenesis siRNAs 28 (Table 1). These <br /> regulatory targets.                                               21 genes spermatogenesis siRNAs dependency <br />                                                                   RRF-3 represent sample candidate RRF-3 <br />    siRNA profiles him-8(e1489) males               regulatory targets spermatogenesis. An impor- <br /> and isolated fem-3(q20) hermaphrodite sper-              tant result analysis RRF-3 little no <br /> matogenic cells generally similar genes         effect gene-specific siRNA counts majority of <br /> abundant siRNAs sample tended abundant               4723 genes spermatogenesis siRNAs (Figure 8): <br /> in other, number genes did genotype/            ,4% genes reproducible, greater-than-two- <br /> tissue specificity (Figure S6). To increase stringency        fold decreases siRNA levels rrf-3(pk1426) relative to <br /> of analysis, prepared sets males            control him-8(e1489) males (69 1784 genes; exclude <br /> genotype 25&#194;&#176; using different population synchroni-             background noise genes low siRNA alignment <br /> zation methods set. We aligned sequences             counts, genes counts 25 sets <br /> from sample cDNA data set compared                control males included). This result consistent <br /> the siRNA alignment count gene rrf-3              existence RNAi-related factors <br /> mutant control male sample. For vast majority          substantial germline phenotypes (Cox et al. 1998; <br /> genes, saw little effect rrf-3(pk1426) siRNA       Smardon et al. 2000; Knight Bass 2001; Tops et al. <br /> levels assays: ,5% genes reproduc-            2005; Chen et al. 2007). <br /> ible, greater-than-twofold decreases siRNAs levels. A             In addition mRNA-encoding genes, examined <br /> subset genes showed dramatic effects, however,             small RNAs corresponding large number repeated <br />                                                                  TABLE 1 <br />                                                                                                                                                1308 <br />  <br />  <br />                                                     Candidate RRF-3 regulatory targets <br />  <br />                                                                                                                       Fold increase rrf-3 <br />                                                                                                                        eri-1 mutants <br />                                                     siRNA alignment         Expression                               (Asikainen et al. 2007) <br />                                                    count isolated       mountain         Expression category <br /> Gene                    Annotation            spermatogenic cells    (Kim et al. 2001)   (Reinke et al. 2004)     rrf-3         eri-1 <br /> K09G1.2                                                   141                 4       Unassigned                     5.44     5.16 <br /> Y47D3A.30 (elo-8)    Polyunsaturated fatty acid elongase   91                 4       Mixed spermatogenesis: somatic CND      CND <br /> C05C12.5                                                   31           Unassigned    Spermatogenesis-enriched       2.44     2.91 <br /> R10F2.1 (cdh-1)      Cadherin                              29            9, 10, 3, 0  Unassigned                     CND      CND <br /> C36H8.1              MSP domain                            25                 4       Spermatogenesis-enriched       CND      CND <br /> B0523.1 (kin-31)     Protein kinase                        22                 4       Spermatogenesis-enriched       CND      CND <br /> W09G10.6 (clec-125)  C-type lectin                         22                26       Male soma: enriched            CND      CND <br /> C44B11.6                                                   17                &#226;&#8364;&#8221;                       &#226;&#8364;&#8221;                &#226;&#8364;&#8221;        &#226;&#8364;&#8221; <br /> B0348.5                                                    15               7, 1      Unassigned                     CND      CND <br /> F18C5.4                                                    12                 4       Spermatogenesis: enriched      CND      CND <br /> F37E3.3              Protein kinase                        11                 4       Intrinsic                      CND      1.92 <br /> K02E2.6              Retroviral aspartyl protease          11                 6       Unassigned                     CND      1.86 <br /> Y41C4A.13                                                  11           Unassigned    Unassigned                     CND      CND <br /> F55H12.1 (snf-2)     Sodium:neurotransmitter symporter     10                 4       Spermatogenesis-enriched       CND      3.10 <br /> E01G4.7                                                     8                &#226;&#8364;&#8221;                       &#226;&#8364;&#8221;                &#226;&#8364;&#8221;        &#226;&#8364;&#8221; <br /> K11C4.1              Protein kinase                         7                 4       Spermatogenesis-enriched       CND      2.15 <br /> K08D10.7 (scrm-8)    Phospholipid scramblase                6                 4       Spermatogenesis-enriched       CND      2.16 <br /> W05H12.2                                                    5           Unassigned    Unassigned                     CND      CND <br />                                                                                                                                                J. I. Gent et al. <br />  <br />  <br />  <br />  <br /> Y82E9BL.18           Cyclin-like F-box, FTH domain          4                &#226;&#8364;&#8221;                       &#226;&#8364;&#8221;                &#226;&#8364;&#8221;        &#226;&#8364;&#8221; <br /> F36A2.11                                                    3                 4       Spermatogenesis-enriched       4.06     4.13 <br /> H16D19.4                                                    3                 2       Unassigned                     CND      CND <br /> ------------------------------------------------------------------------------------------------------------------------------------ <br /> Y37E11B.2                                                   2                 6       Unassigned                     CND      CND <br /> Y17D7C.1                                                    1                 3       Unassigned                     CND      CND <br /> T08B6.2              7-Transmembrane receptor               1                 0       Unassigned                     CND      CND <br /> Y17D7B.4                                                    1                 8       Unassigned                     CND      CND <br /> C36A4.11                                                    1                &#226;&#8364;&#8221;                       &#226;&#8364;&#8221;                &#226;&#8364;&#8221;        &#226;&#8364;&#8221; <br /> K06B9.6                                                     1                &#226;&#8364;&#8221;                       &#226;&#8364;&#8221;                &#226;&#8364;&#8221;        &#226;&#8364;&#8221; <br /> W04B5.2                                                     1                 0       Unassigned                     CND      CND <br /> F52D2.6                                                     1                 7       Mixed oogenesis-somatic        CND      CND <br /> C01G8.2 (cln-3.2)    Transmembrane protein                  1                 2       Unassigned                     CND      CND <br /> W04B5.1                                                     1           Unassigned    Unassigned                     CND      CND <br /> E01G4.5              Protease inhibitor I4, serpin          1                 1       Mixed oogenesis-somatic        CND      CND <br /> Y116A8A.3 (clec-193) C-type lectin                          1           Unassigned    Male sex- enriched             CND      CND <br /> Y116A8A.1 (clec-192) C-type lectin (pseudogene)             1                10       Unassigned                     CND      CND <br /> F39E9.7              Staufen double-stranded RNA binding    0                12       Unassigned                     CND      CND <br />                                                                                                                                 (continued ) <br />                                                                                    TABLE 1 <br />                                                                                  (Continued) <br />  <br />                                                                                                                                                     Fold increase rrf-3 <br />                                                                                                                                                      eri-1 mutants <br />                                                                    siRNA alignment            Expression                                           (ASIKAINEN et al. 2007) <br />                                                                   count isolated          mountain              Expression category <br /> Gene                            Annotation                   spermatogenic cells       (KIM et al. 2001)        (REINKE et al. 2004)             rrf-3        eri-1 <br /> F55A4.4                 Staufen double-stranded RNA binding                  0                 Unassigned       Unassigned                           CND          CND <br /> Y43F8B.9                                                                     0                     15           Unassigned                           CND          CND <br /> Y7A5A.8                                                                      0                      0           Male sex-enriched                    CND          CND <br /> C18D4.4                                                                      0                     15           Unassigned                           CND          CND <br /> F39E9.6                 Neprilysin                                           0                     12           Male sex-enriched                    CND          CND <br /> Y7A5A.3                                                                      0                    3, 0          Male soma: enriched                  CND          CND <br /> F14F7.5                 WSN domain                                           0                      0           Unassigned                           CND          CND <br /> F07G6.6 (fbxa-52)       FTH domain                                           0                      0           Unassigned                           CND          CND <br /> Y46D2A.1                                                                     0                     &#226;&#8364;&#8221;            Unassigned                           CND          CND <br /> Y26D4A.17                                                                    0                      3                         &#226;&#8364;&#8221;                       &#226;&#8364;&#8221;            &#226;&#8364;&#8221; <br /> Y9D1A.1                                                                      0                     12           Unassigned                           CND          CND <br /> H09G03.1                                                                     0                      3           Unassigned                           CND          CND <br /> ZK380.5                                                                      0                     &#226;&#8364;&#8221;                          &#226;&#8364;&#8221;                       &#226;&#8364;&#8221;            &#226;&#8364;&#8221; <br />                                                                                                                                                                              C. elegans RNA-Directed RNA Polymerase <br />  <br />  <br />  <br />  <br /> M142.1 (unc-119)        GMP phosphodiesterase, D-subunit                     0                      1           Unassigned                           CND          CND <br />   The 49 genes reduced siRNA alignment counts rrf-3(pk1426) sets whole-male samples ranked according siRNA alignment count isolated <br /> spermatogenic cell sample. The 21 genes siRNA alignment counts spermatogenesis isolate (above dashed line) categorized can- <br /> didate spermatogenesis RRF-3 targets. Categorization KIM et al. (2001) REINKE et al. (2004) based mRNA expression measured microarrays. Mountain 4 genes <br /> tend spermatogenesis enrichment. Increases mRNA levels rrf-3 eri-1 mutants based mRNA microarray analysis ASIKAINEN et al. (2007) S. <br /> ASIKAINEN G. WONG (personal communication). &#226;&#8364;&#8221;, included microarray; CND, change detected. <br />                                                                                                                                                                              1309 <br />  1310                                                    J. I. Gent et al. <br />  <br />                                                                     RNAi-like processes previously observed in <br />                                                                     exogenous RNAi transgene silencing (Sijen et al. 2001; <br />                                                                     Simmer et al. 2002). <br />                                                                        siRNAs produced RRF-3 ERI-1 function to <br />                                                                     downregulate mRNA levels spermatogenesis: In <br />                                                                     absence RRF-3 ERI-1, putative <br />                                                                     target, K02E2.6, shown increased <br />                                                                     mRNA levels (Duchaine et al. 2006). To test whether <br />                                                                     RRF-3 ERI-1 generally act downregulate their <br />                                                                     spermatogenesis targets, compared RRF-3 regu- <br />                                                                     latory candidates global microarray analysis of <br />                                                                     RRF-3 ERI-1 effects mRNA levels hermaphro- <br />                                                                     dites undergoing spermatogenesis (Asikainen et al. <br />                                                                     2007; S. Asikainen G. Wong, personal communi- <br />                                                                     cation). We note limitations compari- <br />                                                                     son microarray analysis: Three 21 genes <br />                                                                     identified differentially regulated males <br />                                                                     included microarrays used Asikainen <br />                                                                     et al. (2007). Furthermore, Asikainen et al. (2007) <br />                                                                     reported data L4 hermaphrodites, which <br />                                                                     sperm sperm precursors relatively <br />                                                                     rare component (so substantial differences in <br />                                                                     spermatogenic mRNA abundance been <br />                                                                     masked pools unaffected somatic RNA). Similarly, <br />                                                                     spermatogenic cell-type-specific silencing hard <br />                                                                     detect background sperm precursor <br />                                                                     cells expressing transcript normal levels. <br />                                                                     Despite limitations, considerable interest <br />                                                                     compare siRNA data mRNA abundances <br />    Figure 8.&#226;&#8364;&#8221;Global RRF-3 effects spermatogenesis siRNA          determined Asikainen et al. (2007; Table 1). Of 18 <br /> accumulation. Scatter plots depict siRNA alignment count <br /> per gene him-8(e1489) control males (x-axis)          genes included microarray analysis and <br /> rrf-3(pk1426); him-8(e1489) (y-axis). Blue dots correspond       RRF-3-dependent spermatogenesis siRNAs, 3 <br /> the 4723 genes spermatogenic cell siRNAs. Red dots cor-        showed increased mRNA levels rrf-3 eri <br /> respond 21 genes spermatogenic siRNAs          mutants, additional 5 showed increased mRNA <br /> dependency RRF-3. Genes siRNA alignment counts of <br />                                                                     levels eri-1 mutants (15.8-fold enrichment a <br /> zero assigned value 0.5 avoid exclusion the <br /> logarithmically scaled plots. The trendlines indicate ex-       P-value 8.8 3 10 4 rrf-3 mutant; 10.6-fold <br /> pected siRNA alignment counts gene samples         enrichment P-value 2.9 3 10 7 eri-1 <br /> derived population. On linear scaled plot,          mutant; materials methods). <br /> the slopes deviate 45&#194;&#176; differences sample <br /> sizes. <br />  <br />                                                                                          DISCUSSION <br /> and/or transposable elements (http://www.sanger. <br /> ac.uk/Projects/C_elegans/REPEATS/elegans.lib). From                    Evidence developmental role RRF-3 during <br /> this analysis transposons, especially            spermatogenesis: Small RNA pathway functions are <br /> Helitron (264 siRNAs aligned HELICOP1 sequence)                commonly categorized defensive (e.g., suppression <br /> and Tc1/mariner transposons (87 siRNAs aligned Tc1             transposable elements viruses) regulatory <br /> sequence), associated high siRNA alignment                (e.g., downregulation mRNA levels develop- <br /> counts spermatogenic cell sample. We did               ment). Either function essential, exemplified <br /> find transposons required RRF-3 criteria         critical roles microRNA developmental regu- <br /> a threefold reduction siRNA alignment count P-             lators (reviewed Flynt Lai 2008) the <br /> value ,0.005 mutant control                roles piRNAs transposon control (reviewed in <br /> sets samples. Intriguingly, cases        Klattenhoff Theurkauf 2008). These develop- <br /> siRNAs corresponding transposon significantly             mental/defense roles small RNAs overlap, as <br /> more abundant rrf-3 mutants: CELE1, LINE2,                   illustrated human cellular microRNA targets a <br /> TURMOIL2. Of these, TURMOIL2 siRNAs                        viral RNA (Lecellier et al. 2005) recent <br /> present spermatogenic cell sample. The increased             description C. elegans small RNA family (21U- <br /> siRNAs transposons reflect activation           piRNAs) apparently modulates transpo- <br />                                          C. elegans RNA-Directed RNA Polymerase                                    1311 <br />  <br /> sons cellular transcripts spermatogenesis         nucleate mitotic spindles) polarity <br /> (Batista et al. 2008; Das et al. 2008; Wang Reinke       cues. Numerous genes expression enriched or <br /> 2008). Although products cellular RdRPs apparently        specific spermatogenesis indicate sperm pro- <br /> differ piRNAs structure biosynthetic        duction involves induction gene expression pro- <br /> mode, RdRPs participate responses             grams (Kim et al. 2001; Reinke et al. 2004; Johnston <br /> foreign nucleic acids, including quelling Neurospora      et al. 2008). As virtually biological process, it <br /> crassa (Cogoni Macino 1999) viral defense         expected biological functions also <br /> Arabidopsis (Mourrain et al. 2000). On basis          need downregulated sperm differentiation <br /> developmental phenotypes mutant strains,            (Lamitina L&#226;&#8364;&#8482;Hernault 2002; Muhlrad and <br /> RdRP components hypothesized carry              Ward 2002; Khalil et al. 2004). In systems in- <br /> out endogenous regulatory roles (Smardon et al. 2000;        teracting potentially cooperating controls, loss a <br /> Shiu Metzenberg 2002).                                   single regulatory modality frequently leads condi- <br />    Several characteristics RRF-3 (the temperature-        tional (rather absolute) effect. For example, null <br /> dependent character requirement RRF-3            mutations certain regulators produce specific tem- <br /> in defined developmental pathway spermatogen-      perature-sensitive somatic phenotypes larval <br /> esis) facilitate detailed analysis siRNAs associated    development C. elegans (Golden Riddle 1984; <br /> with RRF-3 function potential regulatory roles.    Melendez Greenwald 2000). Spermatogenesis in <br /> Using high-throughput sequencing tool charac-       C. elegans particularly temperature-sensitive in <br /> terizing RNA populations, antisense            wild-type worms (which sterile spermatogen- <br /> siRNAs corresponding variety transposable       esis attempted 27&#194;&#176; (Shapira Tan 2008). Null <br /> elements cellular mRNAs present spermato-         mutants C. elegans small RNA-related genes, <br /> genic cells. Among large diverse set cellular     including Piwi-related gene prg-1, share apparent <br /> mRNAs reflected pool siRNAs,            null-mutant temperature-sensitive spermatogenesis char- <br /> strong enrichment expression           acter (Batista et al. 2008; Das et al. 2008; Wang and <br /> spermatogenesis (in particular, mRNAs known          Reinke 2008). <br /> have elevated expression spermatogenesis; Kim            Models negative regulation gene expression <br /> et al. 2001; Reinke et al. 2004). By comparing         RRF-3: A &#226;&#8364;&#732;&#226;&#8364;&#732;simple&#226;&#8364;&#8482;&#226;&#8364;&#8482; model RRF-3 function during <br /> males mutant wild type rrf-3,          spermatogenesis entail RRF-3 association a <br /> identified exemplary cellular mRNAs siRNA          set transcripts, specific synthesis small RNA pools <br /> production spermatogenesis depends RRF-3.          transcripts, subsequent downregulation of <br /> These cellular mRNAs strong enrichment              new synthesis corresponding protein products, <br /> previously identified genes mRNA levels            contribution downregulation providing <br /> elevated populations animals rrf-3       balanced protein product levels allow efficient <br /> and eri-1 mutants (Asikainen et al. 2007).                   developmental process (spermatogenesis) di- <br />    RRF-3 paternal contributions embryogenesis:        verse conditions. <br /> In principle, paternal-effect lethality associated         Despite attractiveness model, note <br /> RRF-3 ERI-1 mutants result           data establish direct connection be- <br /> lack sperm-provided factors essential normal          tween RRF-3, production siRNAs specific <br /> embryogenesis pathological effects inappro-       mRNA transcripts, observed RRF-3-dependent down- <br /> priate sperm content. We favor possibility    regulation, spermatogenic phenotypes resulting in <br /> several reasons parsimony. First, RNAi pathways       mutant background. Certainly, observation of <br /> known function broadly downregulation gene          similar sets template mRNAs, siRNA products, mRNA <br /> expression, loss-of-function mutations result      modulation, spermatogenic/embryonic phenotypes <br /> in overexpression spermatogenic factors. Second,      member (ERI-1) ERI protein <br /> presence ectopic microtubule structures           supercomplex RRF-3 known reside argues for <br /> observed mutant sperm (and later defective micro-         concerted participation complex defined <br /> tubule structures embryos) suggests failure elim-      regulatory process sperm. We note small RNA <br /> inate structure (microtubules)           biogenesis regulatory mechanisms C. elegans and <br /> budding spermatids. Whether sequestration          organisms sufficiently complex multi- <br /> defects ectopic microtubule polymerization          modal, assuming siRNAs identified here <br /> budding, plausible interpretation microtubules     immediate products RRF-3 an <br /> or associated factors mutant sperm           oversimplification. <br /> pathological effect embryos.                                 From observed defects sperm form and <br />    Production functionally normal sperm requires        function, somewhat surprising observe a <br /> highly specialized cellular differentiation program       small number genes siRNA production <br /> generate cells motile, fertilization competent,     requires RRF-3. Given requirement alternative <br /> and capable contributing centrioles (which           RNA-directed RNA polymerase EGO-1 spermato- <br />  1312                                                   J. I. Gent et al. <br />  <br /> genesis (Smardon et al. 2000), plausible         costly mechanism generate self-regulated <br /> distinct classes cis-acting RNA signal (one             system. A key requirement negative <br /> EGO-1 RRF-3) responsible func-                 regulator accumulation given time de- <br /> tional RdRP recruitment subsequent genetic re-                 pends concentration product. An RdRP <br /> pression. Incomplete loss RRF-3 target siRNAs rrf-3          allow gene engage feedback <br /> mutants suggests redundancy RdRPs               regulation addition RdRP recruitment signal to <br /> other mechanisms trigger siRNA production. EGO-               transcript. This provide capacity to <br /> 1, RRF-3, C. elegans RdRPs                  convert &#226;&#8364;&#732;&#226;&#8364;&#732;off-to-on&#226;&#8364;&#8482;&#226;&#8364;&#8482; temporal spike, <br /> overlap recognition target RNAs          strong RdRP recruitment result system <br /> overlapping signal specificity RdRP recruitment              initial burst gene expression self- <br /> differential timing RdRP expression relative de-             silencing. Importantly potential contribution to <br /> velopmental events. We stress lists regulated          gene regulation, RdRP present a <br /> genes (at siRNA mRNA levels) exemplary                 relatively ubiquitous component cellular machinery, <br /> and complete: addition genes missed             message autoregulated simply needing to <br /> our detection siRNA expression redun-               make docking sites available RdRP acquire <br /> dantly triggered, failed detect un-               negative regulation following burst synthesis. <br /> annotated genes, genes siRNA expression                    We thank Anton Gartner anti-RAD-51, Josef Loidl anti-REC-8, <br /> limited hermaphrodites, genes siRNAs              Abby Dernburg anti-HIM-8; Ayelet Lamm flow cell <br /> at level low transient detected          preparation; Cheryl Smith, Ziming Weng, Phil Lacroute, Anton <br /> analysis.                                                          Valouev, Arend Sidow flow cell preparation, sequencing, and <br />                                                                    data processing; Weng-Onn Lui, Julia Pak, Poornima Parameswaran, <br />    Despite incompleteness identified regu-              Jay Maniar ideas preparation small RNA libraries for <br /> lated gene set, important note majority       sequencing; Virginia Walbot comments manuscript; the <br /> genes showed difference mRNA siRNA                 Caenorhabditis Genetics Center strains; Suvi Asikainen, Gary <br /> expression rrf-3 mutant animals. Thus RRF-3                 Wong, Derek Pavelec, Scott Kennedy sharing data. This work <br /> system appears somewhat exclusive choice                 supported National Institutes Health (NIH) grant <br />                                                                    R01GM37706 A.Z.F.; NIH grant R01GM53804 A.M.V.; a <br /> &#226;&#8364;&#732;&#226;&#8364;&#732;client.&#226;&#8364;&#8482;&#226;&#8364;&#8482; Misregulation small set genes,         Vienna Science Technology Fund WWTF grant LS05009 an <br /> of single gene, certainly drastic effects          Austrian Science Fund FWF Elise Richter grant V.J. J.I.G. was <br /> spermatogenesis. Single-gene requirements C. elegans            supported Stanford Department Biological Sciences by <br /> spermatogenesis extensively investigated              Stanford Genome Training Program (NHGRI-T32-HG00044), and <br /> the literature, hundreds conditional nu-               M.S. supported Canadian Institutes Health Research <br />                                                                    postdoctoral fellowship. <br /> merous nonconditional spermatogenesis mutants in <br /> several dozen genes (reviewed L&#226;&#8364;&#8482;Hernault 2009). <br /> While single-gene process tempting to <br />                                                                                         LITERATURE CITED <br /> propose, multigenic requirements developmental <br /> regulation demonstrated small               Alpi, A., P. Pasierbek, A. Gartner J. Loidl, 2003 Genetic and <br />                                                                        cytological characterization recombination protein RAD- <br /> RNA-regulated systems; case RRF-3 regulation,                51 Caenorhabditis elegans. Chromosoma 112: 6&#226;&#8364;&#8220;16. <br /> the existence dozen RRF-3-dependent            Ambros, V., R. C. Lee, A. Lavanway, P. T. Williams D. Jewell, <br /> siRNA templates suggests observed phenotypic                  2003 MicroRNAs tiny endogenous RNAs C. ele- <br />                                                                        gans. Curr. Biol. 13: 807&#226;&#8364;&#8220;818. <br /> syndrome involve target.                       Asikainen, S., M. Storvik, M. Lakso G. Wong, 2007 Whole ge- <br />    Our work, combined recent analyses,                      nome microarray analysis C. elegans rrf-3 eri-1 mutants. <br /> suggests potential negative regula-              FEBS Lett. 581: 5050&#226;&#8364;&#8220;5054. <br /> tory mechanisms use small RNA effectors                Babiarz, J. E., J. G. Ruby, Y. Wang, D. P. Bartel R. Blelloch, <br />                                                                        2008 Mouse ES cells express endogenous shRNAs, siRNAs, and <br /> spermatogenesis involve microRNAs (Ro et al.                  microprocessor-independent, Dicer-dependent small <br /> 2007; Marcon et al. 2008), piRNAs (reviewed                         RNAs. Genes Dev. 22: 2773&#226;&#8364;&#8220;2785. <br /> Klattenhoff Theurkauf 2008), RRF-3-dependent                   Barton, M. K., T. B. Schedl J. Kimble, 1987 Gain-of-function <br />                                                                        mutations fem-3, sex-determination gene Caenorhabditis el- <br /> siRNAs (Pavelec et al. 2009), RRF-3-independent                    egans. Genetics 115: 107&#226;&#8364;&#8220;119. <br /> siRNAs. Each mechanisms appears poised               Batista, P. J., J. G. Ruby, J. M. Claycomb, R. Chiang, N. Fahlgren <br /> to control specific set RNA targets manner              et al., 2008 PRG-1 21U-RNAs interact form piRNA <br />                                                                        complex required fertility C. elegans. Mol. Cell 31: 67&#226;&#8364;&#8220;78. <br /> would depend expression patterns specific               Brenner, S., 1974 The genetics Caenorhabditis elegans. Genetics <br /> mRNAs relevant populations small RNA effectors.                 77: 71&#226;&#8364;&#8220;94. <br />    RdRPs providers inexpensive feedback regula-              Chen, Y., A. Pane T. Schupbach, 2007 Cutoff aubergine <br />                                                                        mutations result retrotransposon upregulation check- <br /> tion: For conventional feedback regulation (e.g., em-                  point activation Drosophila. Curr. Biol. 17: 637&#226;&#8364;&#8220;642. <br /> ploying transcription factor), biological          Cogoni, C., G. Macino, 1999 Gene silencing Neurospora <br /> be devised specific repressive interaction                 crassa requires protein homologous RNA-dependent RNA <br /> between gene product (or metabolite depends                   polymerase. Nature 399: 166&#226;&#8364;&#8220;169. <br />                                                                    Colaiacovo, M. P., A. J. MacQueen, E. Martinez-Perez, K. <br /> on gene product) corresponding RNA                      McDonald, A. Adamo et al., 2003 Synaptonemal complex as- <br /> protein occurs. RdRP-based mechanisms provide                        sembly C. elegans dispensable loading strand-exchange <br />                                                      C. elegans RNA-Directed RNA Polymerase                                                     1313 <br />  <br />     proteins critical proper completion recombination.            LaMunyon, C. W., S. Ward, 1994 Assessing viability mu- <br />     Dev. Cell 5: 463&#226;&#8364;&#8220;474.                                                        tant manipulated sperm artificial insemination Caeno- <br /> Cox, D. N., A. Chao, J. Baker, L. Chang, D. Qiao et al., 1998 A                  rhabditis elegans. Genetics 138: 689&#226;&#8364;&#8220;692. <br />     novel class evolutionarily conserved genes defined piwi            LaMunyon, C. W., S. Ward, 1995 Sperm precedence her- <br />     essential stem cell self-renewal. Genes Dev. 12: 3715&#226;&#8364;&#8220;               maphroditic nematode (Caenorhabditis elegans) com- <br />     3727.                                                                        petitive superiority male sperm. Experientia 51(8): 817&#226;&#8364;&#8220;823. <br /> Das, P. P., M. P. Bagijn, L. D. Goldstein, J. R. Woolford, N. J.             Lau, N. C., L. P. Lim, E. G. Weinstein D. P. Bartel, 2001 An <br />     Lehrbach et al., 2008 Piwi piRNAs act upstream en-                 abundant class tiny RNAs probable regulatory roles in <br />     dogenous siRNA pathway suppress Tc3 transposon mobility                Caenorhabditis elegans. Science 294: 858&#226;&#8364;&#8220;862. <br />     Caenorhabditis elegans germline. Mol. Cell 31: 79&#226;&#8364;&#8220;90.                Lecellier, C. H., P. Dunoyer, K. Arar, J. Lehmann-Che, S. Eyquem <br /> Dernburg, A. F., K. McDonald, G. Moulder, R. Barstead, M.                        et al., 2005 A cellular microRNA mediates antiviral defense in <br />     Dresser et al., 1998 Meiotic recombination C. elegans ini-                human cells. Science 308: 557&#226;&#8364;&#8220;560. <br />     tiates conserved mechanism dispensable homolo-           Lee, R. C., C. M. Hammell V. Ambros, 2006 Interacting endog- <br />     gous chromosome synapsis. Cell 94: 387&#226;&#8364;&#8220;398.                                  enous exogenous RNAi pathways Caenorhabditis elegans. <br /> Duchaine, T. F., J. A. Wohlschlegel, S. Kennedy, Y. Bei, D. Conte,               RNA 12: 589&#226;&#8364;&#8220;597. <br />     Jr. et al., 2006 Functional proteomics reveals biochemical           L&#226;&#8364;&#8482;Hernault, S. W., 2009 The genetics cell biology spermato- <br />     niche C. elegans DCR-1 multiple small-RNA-mediated path-               genesis nematode C. elegans. Mol. Cell. Endocrinol. 306: <br />     ways. Cell 124: 343&#226;&#8364;&#8220;354.                                                     59&#226;&#8364;&#8220;65. <br /> Edgley, M. L., D. L. Riddle, 2001 LG II balancer chromosomes             L&#226;&#8364;&#8482;Hernault, S. W., T. M. Roberts, 1995 Cell biology nem- <br />     Caenorhabditis elegans: mT1(II;III) mIn1 set dom-              atode sperm. Methods Cell Biol. 48: 273&#226;&#8364;&#8220;301. <br />     inantly recessively marked inversions. Mol. Genet. Genomics          Machaca, K., L. J. DeFelice S. W. L&#226;&#8364;&#8482;Hernault, 1996 A novel <br />     266: 385&#226;&#8364;&#8220;395.                                                                chloride channel localizes C. elegans spermatids chloride <br /> Flynt, A. S., E. C. Lai, 2008 Biological principles microRNA-             channel blockers induce spermatid differentiation. Dev. Biol. <br />     mediated regulation: shared themes amid diversity. Nat. Rev.                 176: 1&#226;&#8364;&#8220;16. <br />     Genet. 9: 831&#226;&#8364;&#8220;842.                                                       MacQueen, A. J., M. P. Colaiacovo, K. McDonald A. M. <br /> Ghildiyal, M., H. Seitz, M. D. Horwich, C. Li, T. Du et al.,                      Villeneuve, 2002 Synapsis-dependent -independent mech- <br />     2008 Endogenous siRNAs derived transposons                           anisms stabilize homolog pairing meiotic prophase C. <br />     mRNAs Drosophila somatic cells. Science 320: 1077&#226;&#8364;&#8220;1081.                    elegans. Genes Dev. 16: 2428&#226;&#8364;&#8220;2442. <br /> Golden, J. W., D. L. Riddle, 1984 A pheromone-induced                    Madl, J. E., R. K. Herman, 1979 Polyploids sex determina- <br />     developmental switch Caenorhabditis elegans: temperature-                  tion Caenorhabditis elegans. Genetics 93: 393&#226;&#8364;&#8220;402. <br />     sensitive mutants reveal wild-type temperature-dependent               Maine, E. M., J. Hauth, T. Ratliff, V. E. Vought, X. She et al., <br />     process. Proc. Natl. Acad. Sci. USA 81: 819&#226;&#8364;&#8220;823.                             2005 EGO-1, putative RNA-dependent RNA polymerase, re- <br /> Gonczy, P., S. Pichler, M. Kirkham A. A. Hyman,                              quired heterochromatin assembly unpaired DNA during <br />     1999 Cytoplasmic dynein required distinct aspects MTOC             C. elegans meiosis. Curr. Biol. 15: 1972&#226;&#8364;&#8220;1978. <br />     positioning, including centrosome separation, cell stage      Marcon, E., T. Babak, G. Chua, T. Hughes P. B. Moens, <br />     Caenorhabditis elegans embryo. J. Cell Biol. 147: 135&#226;&#8364;&#8220;150.                   2008 miRNA piRNA localization male mammalian <br /> Hodgkin, J., H. R. Horvitz S. Brenner, 1979 Nondisjunction                   meiotic nucleus. Chromosome Res. 16: 243&#226;&#8364;&#8220;260. <br />     mutants nematode Caenorhabditis elegans. Genetics 91: 67&#226;&#8364;&#8220;94.      Martinez-Perez, E., A. M. Villeneuve, 2005 HTP-1- <br /> Hutvagner, G., M. J. Simard, 2008 Argonaute proteins: key                    dependent constraints coordinate homolog pairing synapsis <br />     players RNA silencing. Nat. Rev. Mol. Cell Biol. 9: 22&#226;&#8364;&#8220;32.                promote chiasma formation C. elegans meiosis. <br /> Johnston, D. S., W. W. Wright, P. Dicandeloro, E. Wilson, G. S.                  Genes Dev. 19: 2727&#226;&#8364;&#8220;2743. <br />     Kopf et al., 2008 Stage-specific gene expression fundamen-          Martinez-Perez, E., M. Schvarzstein, C. Barroso, J. Lightfoot, <br />     tal characteristic rat spermatogenic cells Sertoli cells. Proc.       A. F. Dernburg et al., 2008 Crossovers trigger remodeling <br />     Natl. Acad. Sci. USA 105: 8315&#226;&#8364;&#8220;8320.                                         meiotic chromosome axis composition linked two- <br /> Keeney, S., C. N. Giroux N. Kleckner, 1997 Meiosis-specific                  step loss sister chromatid cohesion. Genes Dev. 22: 2886&#226;&#8364;&#8220;2901. <br />     DNA double-strand breaks catalyzed Spo11, member                Melendez, A., I. Greenwald, 2000 Caenorhabditis elegans lin-13, <br />     widely conserved protein family. Cell 88: 375&#226;&#8364;&#8220;384.                      member LIN-35 Rb class genes involved vulval de- <br /> Kennedy, S., D. Wang G. Ruvkun, 2004 A conserved siRNA-                      velopment, encodes protein zinc fingers LXCXE <br />     degrading RNase negatively regulates RNA interference C.                  motif. Genetics 155: 1127&#226;&#8364;&#8220;1137. <br />     elegans. Nature 427: 645&#226;&#8364;&#8220;649.                                            Miller, M. A., 2006 Sperm oocyte isolation methods bio- <br /> Kent, W. J., 2002 BLAT: BLAST-like alignment tool. Genome                    chemical proteomic analysis. Methods Mol. Biol. 351: 193&#226;&#8364;&#8220;201. <br />     Res. 12: 656&#226;&#8364;&#8220;664.                                                        Mourrain, P., C. Beclin, T. Elmayan, F. Feuerbach, C. Godon et al., <br /> Khalil, A. M., F. Z. Boyar D. J. Driscoll, 2004 Dynamic his-                 2000 Arabidopsis SGS2 SGS3 genes required post- <br />     tone modifications mark sex chromosome inactivation reac-                transcriptional gene silencing natural virus resistance. Cell <br />     tivation mammalian spermatogenesis. Proc. Natl. Acad.                 101: 533&#226;&#8364;&#8220;542. <br />     Sci. USA 101: 16583&#226;&#8364;&#8220;16587.                                               Muhlrad, P. J., S. Ward, 2002 Spermiogenesis initiation Cae- <br /> Kim, S. K., J. Lund, M. Kiraly, K. Duke, M. Jiang et al., 2001 A gene             norhabditis elegans involves casein kinase 1 encoded spe-6 <br />     expression map Caenorhabditis elegans. Science 293: 2087&#226;&#8364;&#8220;                gene. Genetics 161: 143&#226;&#8364;&#8220;155. <br />     2092.                                                                    Nigon, V., 1951 Polyplo&#196;&#177;&#194;&#168;die expe&#194;&#180;rimentale chez nematode libre. <br /> Klass, M. R., D. Hirsh, 1981 Sperm isolation biochemical                 Bull. Biol. Fr. Belg. 85: 187&#226;&#8364;&#8220;225. <br />     analysis major sperm protein Caenorhabditis elegans.             Oegema, K., A. Desai, S. Rybina, M. Kirkham A. A. Hyman, <br />     Dev. Biol. 84: 299&#226;&#8364;&#8220;312.                                                      2001 Functional analysis kinetochore assembly Caeno- <br /> Klattenhoff, C., W. Theurkauf, 2008 Biogenesis germ-                     rhabditis elegans. J. Cell Biol. 153: 1209&#226;&#8364;&#8220;1226. <br />     line functions piRNAs. Development 135: 3&#226;&#8364;&#8220;9.                          Okkema, P. G., J. Kimble, 1991 Molecular analysis tra-2, sex <br /> Klattenhoff, C., D. P. Bratu, N. McGinnis-Schultz, B. S.                         determining gene C. elegans. EMBO J. 10: 171&#226;&#8364;&#8220;176. <br />     Koppetsch, H. A. Cook et al., 2007 Drosophila rasiRNA pathway            Pak, J., A. Fire, 2007 Distinct populations primary sec- <br />     mutations disrupt embryonic axis specification activation            ondary effectors RNAi C. elegans. Science 315: 241&#226;&#8364;&#8220; <br />     ATR/Chk2 DNA damage response. Dev. Cell 12: 45&#226;&#8364;&#8220;55.                     244. <br /> Knight, S. W., B. L. Bass, 2001 A role RNase III enzyme          Pane, A., K. Wehr T. Schupbach, 2007 Zucchini squash <br />     DCR-1 RNA interference germ line development                       encode putative nucleases required rasiRNA production <br />     Caenorhabditis elegans. Science 293: 2269&#226;&#8364;&#8220;2271.                              Drosophila germline. Dev. Cell 12: 851&#226;&#8364;&#8220;862. <br /> Lamitina, S. T., S. W. L&#226;&#8364;&#8482;Hernault, 2002 Dominant mutations               Pasierbek, P., M. Jantsch, M. Melcher, A. Schleiffer, D. <br />     Caenorhabditis elegans Myt1 ortholog wee-1.3 reveal                 Schweizer et al., 2001 A Caenorhabditis elegans cohesion pro- <br />     novel domain controls M-phase entry spermatogen-                 tein functions meiotic chromosome pairing disjunc- <br />     esis. Development 129: 5009&#226;&#8364;&#8220;5018.                                            tion. Genes Dev. 15: 1349&#226;&#8364;&#8220;1360. <br />  1314                                                             J. I. Gent et al. <br />  <br /> Pavelec, Derek M., J. Lachowiec, T. F. Duchaine, H. E. Smith             Sijen, T., F. A. Steiner, K. L. Thijssen R. H. Plasterk, <br />      Scott Kennedy, 2009 Requirement ERI/DICER complex                        2007 Secondary siRNAs result unprimed RNA synthesis <br />      endogenous RNA interference sperm development                      form distinct class. Science 315: 244&#226;&#8364;&#8220;247. <br />      Caenorhabditis elegans. Genetics 183: 1283&#226;&#8364;&#8220;1295.                        Simmer, F., M. Tijsterman, S. Parrish, S. P. Koushika, M. L. Nonet <br /> Phillips, C. M., C. Wong, N. Bhalla, P. M. Carlton, P. Weiser et al.,             et al., 2002 Loss putative RNA-directed RNA polymerase <br />      2005 HIM-8 binds X chromosome pairing center                      RRF-3 makes C. elegans hypersensitive RNAi. Curr. Biol. 12: <br />      mediates chromosome-specific meiotic synapsis. Cell 123:                     1317&#226;&#8364;&#8220;1319. <br />      1051&#226;&#8364;&#8220;1063.                                                              Slotkin, R. K., M. Vaughn, F. Borges, M. Tanurdzic, J. D. Becker <br /> Reinke, V., I. S. Gil, S. Ward K. Kazmer, 2004 Genome-wide                    et al., 2009 Epigenetic reprogramming small RNA silencing <br />      germline-enriched sex-biased expression profiles Caeno-               transposable elements pollen. Cell 136: 461&#226;&#8364;&#8220;472. <br />      rhabditis elegans. Development 131: 311&#226;&#8364;&#8220;323.                            Smardon, A., J. M. Spoerke, S. C. Stacey, M. E. Klein, N. Mackin <br /> Ro, S., C. Park, K. M. Sanders, J. R. McCarrey W. Yan,                        et al., 2000 EGO-1 related RNA-directed RNA polymerase <br />      2007 Cloning expression profiling testis-expressed                    functions germ-line development RNA interference <br />      microRNAs. Dev. Biol. 311: 592&#226;&#8364;&#8220;602.                                          C. elegans. Curr. Biol. 10: 169&#226;&#8364;&#8220;178. <br /> Roberts, T. M., F. M. Pavalko S. Ward, 1986 Membrane                 Tabara, H., M. Sarkissian, W. G. Kelly, J. Fleenor, A. Grishok <br />      cytoplasmic proteins transported organelle                   et al., 1999 The rde-1 gene, RNA interference, transposon <br />      complex nematode spermatogenesis. J. Cell. 102:                       silencing C. elegans. Cell 99: 123&#226;&#8364;&#8220;132. <br />      1787&#226;&#8364;&#8220;1796.                                                              Tops, B. B., H. Tabara, T. Sijen, F. Simmer, C. C. Mello et al., <br /> Ruby, J. G., C. Jan, C. Player, M. J. Axtell, W. Lee et al.,                      2005 RDE-2 interacts MUT-7 mediate RNA interference <br />      2006 Large-scale sequencing reveals 21U-RNAs additional                  Caenorhabditis elegans. Nucleic Acids Res. 33: 347&#226;&#8364;&#8220;355. <br />      microRNAs endogenous siRNAs C. elegans. Cell 127:                Wang, G., V. Reinke, 2008 A C. elegans Piwi, PRG-1, regulates <br />      1193&#226;&#8364;&#8220;1207.                                                                   21U-RNAs spermatogenesis. Curr. Biol. 18: 861&#226;&#8364;&#8220;867. <br /> Schedl, T., J. Kimble, 1988 fog-2, germ-line-specific sex deter-       Ward, S., 1986 The Asymmetric Localization Gene Products During the <br />      mination gene required hermaphrodite spermatogenesis                  Development Caenorhabditis elegans Spermatozoa. A. R. Liss, New <br />      Caenorhabditis elegans. Genetics 119: 43&#226;&#8364;&#8220;61.                                 York. <br /> Schnabel, H., R. Schnabel, 1990 An organ-specific differenti-            Ward, S., T. M. Roberts, S. Strome, F. M. Pavalko E. Hogan, <br />      ation gene, pha-1, Caenorhabditis elegans. Science 250:                 1986 Monoclonal antibodies recognize polypeptide anti- <br />      686&#226;&#8364;&#8220;688.                                                                     genic determinant shared multiple Caenorhabditis elegans <br /> Shapira, M., M. W. Tan, 2008 Genetic analysis Caenorhab-                   sperm-specific proteins. J. Cell Biol. 102: 1778&#226;&#8364;&#8220;1786. <br />      ditis elegans innate immunity. Methods Mol. Biol. 415: 429&#226;&#8364;&#8220;442.         Yigit, E., P. J. Batista, Y. Bei, K. M. Pang, C. C. Chen et al., <br /> Shiu, P. K., R. L. Metzenberg, 2002 Meiotic silencing un-                  2006 Analysis C. elegans Argonaute family reveals that <br />      paired DNA: properties, regulation suppression. Genetics                 distinct Argonautes act sequentially RNAi. Cell 127: <br />      161: 1483&#226;&#8364;&#8220;1495.                                                              747&#226;&#8364;&#8220;757. <br /> Sijen, T., J. Fleenor, F. Simmer, K. L. Thijssen, S. Parrish et al., <br />      2001 On role RNA amplification dsRNA-triggered <br />      gene silencing. Cell 107: 465&#226;&#8364;&#8220;476.                                                                      Communicating editor: K. Kemphues <br />                         Supporting Information <br />   http://www.genetics.org/cgi/content/full/genetics.109.109686/DC1 <br />  <br />  <br /> A Caenorhabditis elegans RNA-Directed RNA Polymerase Sperm <br />        Development Endogenous RNA Interference <br />  <br />  <br />   Jonathan I. Gent, Mara Schvarzstein, Anne M. Villeneuve, Sam Guoping Gu, <br />            Verena Jantsch, Andrew Z. Fire Antoine Baudrimont <br />  <br />  <br />           Copyright &#194;&#169; 2009 Genetics Society America <br />                   DOI: 10.1534/genetics.109.109686 <br />  2 SI                                                                             J. I. Gent et al. <br />  <br />  <br />  <br />  <br />              &amp;IGURE 3  <br />              ! <br />  <br />  <br />  <br />  <br />              &quot;                                                      .                                                                                          RRF <br />   PK    	 <br />              PERCENTOFGONAD <br />  <br />  <br />  <br />  <br />                                                                                                  PERCENT <br />                                                   120                                                                                  120 <br />  <br />                                                   100                                                                                  100 <br />  <br />                                                    80                                                                                    80 <br />  <br />                                                    60                                                                                    60 <br />  <br />                                                    40                                                                                    40 <br />                                                                                                                                       % classes gonad <br />  <br />  <br />  <br />                                                    20                                                                                    20 <br />  <br />                                                     0                                                                                      0 <br />                                                                                                                                                                                  <br />                                                                      ZONE                                                                                           ZONE <br />  <br />  <br />                      2!$ <br />   CATEGORY <br />                                                   &gt;12       ~9-12    ~5-8       ~1-4         0 <br />  <br />  <br />    FIGURE S1.&#226;&#8364;&#8221;Meiotic chromosome organization&#226;&#8364;&#8221;quantification RAD-51 foci wildtype rrf-3 mutant hermaphrodite <br /> gonads undergoing spermatogenesis 25&#194;&#176;C. (A) Partitioning gonads zones scoring. Gonads L4 larvae were <br /> stained with DAPI (blue) anti-RAD-51 antibody (red). The 30 nuclei enter meiosis (the proximal ones, <br /> which rise sperm) identified region occupied divided zones equal length, zone 5 most <br /> proximal. The number RAD-51 foci individual nuclei zone counted. Asterisks denote proximal ends. Scale <br /> bars 10 &#194;&#181;m. (B) Similar patterns N2 rrf-3(pk1426) RAD-51 foci counts. Zones 1 5 gonad were <br /> categorized according number RAD-51 foci. Six gonads genotype scored. Error bars denote standard <br /> errors means. <br />                                                            J. I. Gent et al.                                                   3 SI <br />  <br />  <br />  <br />  <br />         &amp;IGURE3  <br />                    %&quot;1* <br />         !          3&amp;$   <br />  <br />  <br />  <br />                                                                                                                          <br />                    /                                         %&quot;1*                                 3&amp;$   <br />  <br />  <br />  <br />  <br />                                                                                                                          <br />                    rrf-3(pk1426) <br />  <br />                   N2 <br />         &quot; <br />                                             *                       DAPI          *                  HIM-8         * <br />                   HIM-8 DAPI <br />                   rrf-3(pk1426) <br />  <br />  <br />  <br />                                                  *                                    *                                * <br />                   /                                   %&quot;1*                                4:1   <br />        #          %&quot;1* <br />                   4:1                                                                                            <br />  <br />  <br />  <br />  <br />                    SSG  (pk1426)                                                                             <br />  <br />    FIGURE S2.&#226;&#8364;&#8221;Meiotic chromosome organization&#226;&#8364;&#8221;similar patterns rrf-3 mutant wildtype anti-REC-8 (A) anti-HIM- <br /> 8 antibody (B) anti-SYP-1 antibody (C) staining spermatogenic prophase I nuclei 25&#194;&#176;C. Spermatogenic nuclei L4 <br /> larvae identified based cells enter meiosis proximal localization gonad (proximal ends <br /> marked asterisks). The scale bars 10 &#194;&#181;m. (A) In wildtype mutant, REC-8 (magenta) localized continuous <br /> tracks interface parallel aligned DAPI-stained chromosomes (blue). (B) Both wildtype mutant nuclei have <br /> either single focus closely spaced foci HIM- 8 (green) indicating pairing X-chromosomes (WT: 100%, n=50; rrf-3: <br /> 100%, n=91). (C) In wildtype mutant, SYP-1 (magenta) localized continuous tracks interface between <br /> parallel aligned DAPI-stained chromosomes (blue). <br />  4 SI                                                        J. I. Gent et al. <br />  <br />  <br />  <br />  <br />           &amp;IGUR  <br />                                          1 <br />            FRACTIONNONVIABLEEMBRYOS <br />  <br />  <br />  <br />  <br />                                         0.8 <br />  <br />                                         0.6 <br />  <br />                                         0.4 <br />  <br />                                         0.2 <br />  <br />                                          0 <br />                                                     PK     PK     OK    PK             OK      <br />  <br />  <br />  <br />                                                           PATERNAL GENOTYPE <br />  <br />    FIGURE S3.&#226;&#8364;&#8221;pk1426/ok629 trans-heterozygote males sire nonviable embryos. Matings set tra2(q122) females and <br /> ok629/pk1426 heterozygote males 25&#194;&#176;C. The proportion dead embryos estimated counting newly laid embryos and <br /> the resulting survivors. ok629 homozygous males tested defective male tail development inability mate (not <br /> shown). pk1426 weak effect, any, embryogenesis delivered wildtype heterozygous males (Figure 3C). <br /> Error bars denote standard errors means. <br />                                                            J. I. Gent et al.                                                   5 SI <br />  <br />  <br />  <br />  <br />    &amp;IGURE3  <br />  <br />                  Tubulin DNA                                                         Tubulin <br />     wild type <br />  <br />  <br />  <br />  <br />     rrf-3 <br />  <br />  <br />  <br />  <br />     FIGURE S4.&#226;&#8364;&#8221;Microtubule accumulation spermatid nuclei rrf-3 mutant males. Spermatids wildtype <br /> male gonad (top panels) rrf-3 mutant male gonad (bottom panels) stained Hoechst label chromosomes (Red) <br /> and anti-tubulin antibody label microtubules (Green). Males grown 25&#194;&#176;C. The insets panels a <br /> spermatocyte gonad undergoing meiotic division; serves internal control demonstrating <br /> the effectiveness tubulin immunostaining wildtype sample. Arrows highlight fraction tubulin wreaths observed in <br /> rrf-3 mutant spermatids. Scale bar 5 &#194;&#181;m. <br />  6 SI                                                     J. I. Gent et al. <br />  <br />  <br />  <br />  <br />               &amp;IGURE3  <br />                                                 SECONDARY <br />                                                 SPERMATOCYTES <br />                                                  LINKEDTOGETHER	 <br />  <br />                                  PRIMARY <br />                                  SPERMATOCYTE <br />  <br />  <br />  <br />  <br />                    <br />                   SPERAMOTOCYTE <br />                                                                              SPERMATIDS <br />  <br />  <br />                                                                     <br />                                                                  <br />                                                                     SPERMATIDS <br />  <br />  <br />  <br />  <br />    FIGURE S5.&#226;&#8364;&#8221;A mix cell stages isolated spermatogenic cell sample. A small portion isolated spermatogonic cells <br /> in M9 buffer immediately slide rest flash frozen liquid N2. The  <br /> </body></html>