<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Positive Feedback Transcriptional Kinase <br /> Suppression Nematodes Extraordinary Longevity <br /> and Stress Resistance <br /> C&#194;&#184;agda&#195;&#190; Tazearslan1.&#194;&#164;a, Srinivas Ayyadevara2,3., Puneet Bharill1, Robert J. Shmookler Reis1,2,3* <br /> 1 Department Biochemistry Molecular Biology, University Arkansas Medical Sciences, Little Rock, Arkansas, United States America, 2 Department of <br /> Geriatrics, University Arkansas Medical Sciences, Little Rock, Arkansas, United States America, 3 Central Arkansas Veterans Healthcare Service, Little Rock, Arkansas, <br /> United States America <br />  <br />  <br />  <br />      <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />      Insulin/IGF-1 signaling (IIS) regulates development metabolism, modulates aging, Caenorhabditis elegans. In <br />      nematodes, mammals, IIS understood operate kinase-phosphorylation cascade inactivates the <br />      DAF-16/FOXO transcription factor. Situated center pathway, phosphatidylinositol 3-kinase (PI3K) <br />      phosphorylates PIP2 form PIP3, phospholipid required membrane tethering activation signaling <br />      molecules. Nonsense mutants age-1, nematode gene encoding class-I catalytic subunit PI3K, produce a <br />      truncated protein lacking kinase domain, confer 10-fold greater longevity second-generation (F2) <br />      homozygotes, comparable gains stress resistance. Their F1 parents, like weaker age-1 mutants, far robust&#226;&#8364;&#8221; <br />      implying maternally contributed trace amounts PI3K activity PIP3 block extreme age-1 phenotypes. We find <br />      F2-mutant adults ,10% wild-type kinase activity vitro ,60% normal phosphoprotein levels vivo. <br />      Inactivation PI3K disrupts PIP3-dependent kinase signaling, surprisingly attenuates transcripts of <br />      numerous IIS components, upstream PI3K, signaling molecules cross-talk IIS. The age-1(mg44) <br />      <span id='am-243' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-244' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-258' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-267' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-277' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-279' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-297' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-300' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-307' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-309' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-318' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-319' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-320' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span>nonsense mutation</span> results, F2 adults, changes kinase profiles expression levels multiple transcripts that <br />      distinguish mutant F1 age-1 homozygotes, weaker age-1 mutant, wild-type adults. Most those <br />      changes reversed second <span id='am-321' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-322' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-323' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-324' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-325' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-326' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-327' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-328' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-329' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-330' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-331' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-332' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-333' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-334' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-335' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-337' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-338' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-341' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-342' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-343' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-344' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-347' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-348' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-349' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-350' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-351' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-352' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-353' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-354' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-355' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-356' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-357' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-358' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-359' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-362' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-363' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-364' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-365' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-366' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-367' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-368' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-369' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-370' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-372' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-373' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-374' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-375' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-376' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-378' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-379' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-380' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-381' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-382' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-383' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-384' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-385' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-386' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-387' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-388' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-389' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-390' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-391' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-392' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-393' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-394' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-395' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-396' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-397' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-398' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span>mutation</span> daf-16, implicating DAF-16/ FOXO&#226;&#8364;&#8220;dependent &#226;&#8364;&#8220;independent <br />      mechanisms. RNAi, silencing genes downregulated long-lived worms, improves oxidative-stress resistance of <br />      wild-type adults. It plausible attenuation genes age-1(mg44)-F2 adults contributes their <br />      exceptional <span id='am-9' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-10' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span>survival</span>. IIS nematodes (and presumably species) involves transcriptional kinase <br />      regulation positive-feedback circuit, favoring survival reproduction. Hyperlongevity strong age-1(mg44) <br />      mutants result inability reset molecular switch reproductive mode. <br />  <br />    Citation: Tazearslan C&#194;&#184;, Ayyadevara S, Bharill P, Shmookler Reis RJ (2009) Positive Feedback Transcriptional Kinase Suppression Nematodes with <br />    Extraordinary Longevity Stress Resistance. PLoS Genet 5(4): e1000452. doi:10.1371/journal.pgen.1000452 <br />    Editor: Susan E. Mango, Huntsman Cancer Institute, United States America <br />    Received April 14, 2008; Accepted March 11, 2009; Published April 10, 2009 <br />    This open-access article distributed terms Creative Commons Public Domain declaration stipulates that, placed public <br />    domain, work freely reproduced, distributed, transmitted, modified, built upon, used lawful purpose. <br />    Funding: This work supported grant P01-AG20641 RJSR U.S. National Institutes Health (National Institute Aging) Research <br />    Career Scientist Award (<span id='am-8' about='protege:TO' typeof='owl:Thing'>to</span> RJSR) Department Veterans Affairs. The funders role study design, data collection analysis, decision to <br />    publish, preparation manuscript. <br />    Competing Interests: Three authors (CT, AS RJSR) listed inventors patent filed University Arkansas Medical Sciences, relating <br />    novel aspects second-generation homozygotes age-1(mg44) (m333) mutant alleles, underlying extreme longevity stress resistance. <br />    * E-mail: rjsr@uams.edu <br />    &#194;&#164; Current address: Department Genetics, Albert Einstein College Medicine, Bronx, New York, United States America <br />    . These authors contributed equally work. <br />  <br />  <br />  <br />  <br /> Introduction                                                                               recruits phosphorylates AGE-1 catalytic subunit of <br />                                                                                            phosphatidylinositol 3-kinase (PI3K), turn phosphorylates <br /> IIS depends critically presence PIP3                                             regulatory subunit. Activated AGE-1 adds phosphate to <br />    The IIS pathway, governing developmental arrest, metabolism                             phosphatidylinositol 4,5-diphosphate [PI(4,5)P2] inositol- <br /> and life span Caenorhabditis elegans [1&#226;&#8364;&#8220;3], highly conserved                    ring 3-position, converting phosphatidylinositol 3,4,5- <br /> invertebrates mammals. The single IIS pathway nematodes                              triphosphate [PI(3,4,5)P3 PIP3]. <br /> corresponds structure function distinct pathways                             PIP3 plays dual role canonical insulin/IGF-1 pathway. <br /> mammals signal metabolic responses insulin, growth                             The pivotal role membrane tethering signaling <br /> response insulin-like growth factor 1 (IGF-1), respectively [4].                        molecules including AKT-1 -2, PDK-1, GSK-3 protein <br /> IIS disruption discovered enhance longevity C.                             kinase C [13&#226;&#8364;&#8220;16]. PIP3-binding recruits retains kinases at <br /> elegans [5&#226;&#8364;&#8220;8], subsequently shown extend life D.                     cytoplasmic surface cell membrane, these <br /> melanogaster mice [9&#226;&#8364;&#8220;11]. Binding insulin-like peptides                          enzymes substrates (largely kinases) concen- <br /> DAF-2, insulin/IGF-1 receptor nematodes, modulates                                  trated and, mass action, interact efficiently. Because PIP3 <br /> receptor autophosphorylation activation [12]. Active DAF-2                             quantitatively affects multiple components IIS cascade, the <br />  <br />  <br />         PLoS Genetics | www.plosgenetics.org                                           1                                 April 2009 | Volume 5 | Issue 4 | e1000452 <br />                                                                                             Hybrid Feedback Loop Governs Insulin/IGF-1 Response <br />  <br />  <br />  <br />   Author Summary                                                            larvae slowly mature 15&#226;&#8364;&#8220;20uC adults live close ten <br />                                                                             times long near-isogenic wild-type controls, highly <br />   Insulin/IGF-1 signaling (IIS) impacts development, metab-                 resistant oxidative electrophilic stresses [30]. These <br />   olism, longevity Caenorhabditis elegans. It               exceptional worms mean maximal adult life spans at <br />   viewed cascade kinase reactions, chiefly phosphor-                times conferred longevity-extending <br />   ylation kinases, leading inactivation DAF-             mutation, adult lives appear behave <br />   16/FOXO transcription factor. PI3K, phosphatidylinositol                like wild-type worms tenth age. Addition a <br />   kinase center pathway, converts PIP2 PIP3,              second mutation daf-16 gene largely entirely reverses life- <br />   instrumental kinase docking activation. Here                    span extension phenotypes daf-2 age-1 mutations <br />   PI3K deficiency elicits transcriptional inhibition              examined date [3,6,29,30]. <br />   kinases, including IIS itself. This creates               Studies IIS-pathway mutants C. elegans taxa have <br />   positive-feedback loop, DAF-16/FOXO silences <br />                                                                             provided valuable insights genetic mechanisms regulating life <br />   expression kinases inactivated <br />   it. In resulting &#226;&#8364;&#732;&#226;&#8364;&#732;flip-flop&#226;&#8364;&#8482;&#226;&#8364;&#8482; genetic switch, kinase          span [4]. The molecular basis extreme survival phenotypes <br />   signaling transcriptional silencing predominate. We                age-1(mg44) F2 homozygotes remains unknown, be <br />   discovered transcriptional arm switch infertile            assumed differ degree molecular mechanisms that <br />   age-1(mg44) mutants, defective PI3K activity. The                     underlie 4- 5-fold lesser life extensions seen IIS mutants. <br />   absence PIP3 PIP3-dependent kinase activity gives                  The key PIP3, plays structural catalytic <br />   free rein gene silencing DAF-16/FOXO. This two-                     roles signal transduction [17,31], thought mediate both <br />   tiered response scarcely evolved benefit               DAF-16-dependent -independent signaling [32]. Strong age-1 <br />   sterile mutant; components presumably serve                     mutants, lacking class-I PI3K activity, direct route to <br />   regulatory functions normal animals, reinforcing                     generate PIP3 [31]. As result, expected deficient in <br />   switch responsive environmental internal signals.                  enzyme activities require PIP3, activation by <br />   In age-1(mg44) mutants, complete inactivation PI3K                     regulatory kinases, membrane tethering ensures <br />   &#226;&#8364;&#732;&#226;&#8364;&#732;fuses&#226;&#8364;&#8482;&#226;&#8364;&#8482; switch, locking worms longevity mode.                  proximity kinases targets [17,31]. <br />   With signaling profoundly silenced, resume                       In present study, sought evidence support a <br />   reproduction, instead acquire remarkable capacity                   broad role PIP3 unique properties age-1(mg44)-F2 <br />   individual survival.                                                  adults. This role inferred one, normal PIP3 levels <br />                                                                             (in unstressed N2 worms) low detection existing <br /> influence concentration compounded. In addition, PIP3             methods; detectable levels attained starved, peroxide- <br /> binding AKT-1 allosterically exposes cryptic site recognized           stressed wild-type worms similarly stressed age-1-mutants <br /> by PDK-1 (phosphatidylinositol-dependent kinase 1), allowing                [33]. We able document expected widespread <br /> AKT phosphorylation activation [17]. In second role,               disruption protein kinase activity age-1(mg44)-F2 worms, <br /> PIP3 act catalytically, single molecule PIP3       making unexpected observation kinases <br /> potential bind successively AKT-1 molecules, enabling            chiefly inhibited transcriptional level. Direct measure- <br /> their activation. Although AKT-1 target          ment transcripts confirms silencing kinase gene expression, <br /> allosteric role documented [17], possible         leading propose novel &#226;&#8364;&#732;&#226;&#8364;&#732;hybrid&#226;&#8364;&#8482;&#226;&#8364;&#8482; positive-feedback loop in <br /> signaling molecules possess high-affinity PIP3 binding            IIS kinase cascade inhibits DAF-16/FOXO <br /> sites (termed &#226;&#8364;&#732;&#226;&#8364;&#732;Pleckstrin homology domains&#226;&#8364;&#8482;&#226;&#8364;&#8482;) similarly             transcription factor, itself attenuated DAF-16-mediated <br /> controlled. In event, infer insulinlike signaling        transcriptional silencing upstream kinases. <br /> be exquisitely sensitive PIP3 depletion, AKT-1 action <br /> (which extends far IIS [18,19]) absolutely                    Results <br /> dependent presence trace amounts PIP3. <br />    The AKT-1/AKT-2/SGK-1 complex, constitu-                 Very long-lived age-1 mutant worms broadly <br /> ent kinases activated PDK-1 [20], phosphorylates               deficient protein kinase activity <br /> the DAF-16/FOXO transcription factor sites block                   The age-1(mg44) kinase-<span id='am-87' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-88' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span>null mutants</span> deficient in <br /> entry nucleus, activate repress                  phosphatidylinositol 3,4,5-triphosphate production. Given the <br /> transcription hundreds target genes, including              importance PIP3 molecule signal transduction events <br /> modulate metabolism, reproduction, life span, resistance             originating membrane-receptor kinases, anticipated <br /> oxidative stresses [21&#226;&#8364;&#8220;24].                                                 phosphorylation numerous proteins impaired in <br />                                                                             mutants. To initially assess breadth impairment, <br /> IIS mutations wide-ranging effects longevity                        compared vitro kinase activities respect endogenous <br />    Reduction-of-function mutations impairing C. elegans IIS             substrates age-1 mutant strains, normalized wild- <br /> pathway (e.g., daf-2 age-1 mutations) cause worms              type N2DRM stock (Figure 1A&#226;&#8364;&#8220;1C). <br /> arrest development dauer (alternative stage-3) larvae [1&#226;&#8364;&#8220;3]. If             Panels A B illustrate typical experiment, panel C <br /> allowed mature permissive temperature, temperature-                 summarizes results replicate experiments independent <br /> sensitive (ts) daf-2 mutant adults attain twice normal              expansions group. The first-discovered widely <br /> longevity [6]; life extension ranges 1.1- 2.5-fold              used age-1 allele, hx546 [5,27,28], showed 32% kinase activity <br /> different daf-2 alleles [25]. A ts mutant allele age-1, hx546,       N2DRM (Figure 1C). However, worms bearing age- <br /> discovered Klass [26] reported confer 40% 65% life            1(mg44) allele 10% wild-type kinase activity, <br /> extension 20u 25uC respectively [5,27,28]. Two constitutive          maternally protected first-generation (F1) long- <br /> age-1 alleles, m333 mg44, initially reported extend C.          lived second-generation (F2) homozygotes. The F2 worms had <br /> elegans life span 2- 2.6-fold [2,8,29]; survivals          somewhat lower kinase activity F1 (7.3 vs. 8.6% N2DRM, <br /> conducted first-generation (&#226;&#8364;&#732;&#226;&#8364;&#732;F1&#226;&#8364;&#8482;&#226;&#8364;&#8482;) homozygotes. We                P = 0.05), difference consistently greater <br /> recently observed second-generation age-1(mg44) (m333)             specific bands (see Figure 1B). Staining total protein showed <br />  <br />  <br />       PLoS Genetics | www.plosgenetics.org                              2                            April 2009 | Volume 5 | Issue 4 | e1000452 <br />                                                                                                 Hybrid Feedback Loop Governs Insulin/IGF-1 Response <br />  <br />  <br />  <br />  <br /> Figure 1. Both vitro kinase activity endogenous substrates total phosphoprotein levels diminished age-1(mg44) <br /> homozygotes. (A&#226;&#8364;&#8220;C) In vitro kinase activity post-gravid sterile adult worms, 6 days L4/adult molt. Kinase activity cleared, sonicated <br /> lysates assessed c-32P-ATP incorporation 20-mg protein sample, 1 min 30uC. Samples, quenched ice, electrophoresed 10% <br /> polyacrylamide-SDS gels (Invitrogen). (A) Gels stained SYPRO Ruby (Invitrogen) confirm constant protein loads. (B) Image 32P b-emissions <br /> (Molecular Dynamics Storm) gel A, dried vacuum. (C) Data summary 2&#226;&#8364;&#8220;3 independent expansions N2DRM, age- <br /> 1(hx546), age-1(mg44) F1 homozygotes (labeled age-1 mg44F1), age-1(mg44) F2 homozygotes (labeled age-1 mg44F2), daf-16(m26) double <br /> mutants age-1 allele. These quantitations covered <span id='am-5' about='xsp:length' typeof='owl:Thing'>length</span> lane, shown A B. (D&#226;&#8364;&#8220;F) Phosphoproteins, <br /> resolved polyacrylamide gel electrophoresis visualized Pro-Q Diamond staining. (D) Image total protein stained Coomassie blue. (E) <br /> Image fluorescence Pro-Q Diamond staining; note phosphoprotein markers detected. (F) Summary data 3 <br /> independent expansions strain adult day 6. Significance: (C, F), P values shown 2-tailed t-tests (unequal variance) comparing each <br /> age-1 strain N2DRM (values directly bars) comparing strains connected brackets (values brackets). <br /> doi:10.1371/journal.pgen.1000452.g001 <br />  <br /> similar loads samples, banding patterns differed                 To corroborate low protein-kinase activity age-1(mg44) adults, <br /> (Figure 1A). One obvious difference age-1(mg44)                distinguish deficient protein <br /> other strains mutants totally infertile F2            kinases active ones, constructed arrays 70 <br /> generation, despite presence syncytial nuclei [30]. Several             synthetic peptides comprising 50 near-consensus kinase sites <span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> controls exclude explanation mutants&#226;&#8364;&#8482; lack               C. elegans proteome 20 mouse human proteins. <br /> kinase activity. age-1(mg44) F1&#226;&#8364;&#8482;s similar kinase levels              Phosphorylation vitro observed 29 peptides, representing <br /> gravid (day 2 adulthood) post-gravid (adult day 6; Figure            potential substrates 18 distinct kinases (Figure S2 and <br /> S1). Moreover, N2DRM eggs contain half kinase                    Table S1). Protein kinase activity extracts age-1(mg44) F2 <br /> activity parents, weight protein (Figure S1),         adults reduced 1.8- .8-fold, relative isogenic <br /> absence reduce kinase activity case. Deficiency            N2DRM postgravid worms (each nominal P,0.05), 22 of <br /> kinase activity characteristic dauer larvae, exhibit         29 kinase targets phosphorylated vitro. Addition of <br /> a level comparable N2DRM adults (Figure S1).                        daf-16(mu86) mutation produced essentially complete rever- <br />    The consequences adding daf-16 mutation quite different            sion, hyper-reversion (activity.N2DRM), 17 22 <br /> for age-1 alleles: age-1(hx546) worms, daf-16(m26)              peptides. <br /> mutation doubled vitro kinase activity, 68%              In view reduced protein-kinase activity age-1(mg44) <br /> wild-type 160%, mutation restored half            worms, anticipated steady-state level protein <br /> the kinase deficiency age-1(mg44) allele. Insofar kinase         phosphorylation depressed. To assess this, <br /> suppression reversed daf-16; age-1(mg44) double mutants,              phosphoproteins separated acrylamide gel electrophoresis <br /> infer activity inhibited DAF-16/ FOXO              compared wild-type age-1-mutant strains C. <br /> transcription factor. However, reversion far complete,              elegans (Figure 1D&#226;&#8364;&#8220;1F). Total protein staining (panel D) demon- <br /> either m26 (point-mutant) mu86 (large-deletion) allele           strated loading, panel E shows gel stained <br /> daf-16 (see Figure 1C Figure S1). This implies large                Pro-Q Diamond detect quantify phosphoproteins. <br /> proportion observed kinase silencing DAF-16-independent&#226;&#8364;&#8221;                 Results replicates (independent expansions strain) <br /> perhaps reflecting direct effects PIP3 depletion kinases           summarized panel F. Relative wild-type N2DRM, age- <br /> than AKT, AKT targets DAF-16/FOXO.                               1(hx546) worms 16% phosphoprotein staining (marginally <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                                3                            April 2009 | Volume 5 | Issue 4 | e1000452 <br />                                                                                                  Hybrid Feedback Loop Governs Insulin/IGF-1 Response <br />  <br />  <br /> significant P,0.05), age-1(mg44) homozygous F2 adults                  inactivates DAF-16/FOXO. Kinases require PIP3 binding <br /> showed 41% reduction steady-state phosphoprotein level                     membrane tethering kinase activation [17,31], as <br /> (P,0.001). The daf-16(m26) mutation restores allele                   AKT, PDK-1, SGK-1, expected marked <br /> ,92% N2DRM level. This finding supported 2-D                  suppression activity, directly reverted a <br /> dual-fluor phosphoprotein gels (Figure 2), 72%                  daf-16 mutation. The surprising observation mutations daf- <br /> phosphoprotein spots resolved (1199/1669) reduced                 16 restore nearly half age-1 (mg44)-F2 kinase deficiency, and <br /> twofold F2 age-1(mg44) adults relative N2DRM. The                         .70% phosphoprotein deficit, implies inhibition <br /> deficiency total phosphoprotein content pronounced                   mediated DAF-16/FOXO. Such regulation <br /> than protein kinase activity, age-1(mg44)-homozygous F2              direct (DAF-16 suppresses transcription kinase <br /> adults, surprising given phosphoprotein levels                genes) indirect (DAF-16 suppresses kinases, or <br /> reflect steady state, i.e., balance kinase                    stimulates phosphatases, suppress other <br /> phosphatase activities. We present evidence (next section)                 kinases impeding opposing phosphorylation). To test <br /> the PTEN phosphatase downregulated age-1(mg44).                    direct effects DAF-16/FOXO, used real-time polymerase <br />                                                                                 chain reaction (RT-PCR) quantify effects age-1 alleles, <br /> Transcriptional suppression signal-transduction genes                        added inactivation daf-16, transcript levels <br /> in age-1(mg44) worms                                                            IIS genes panel signaling components, <br />    F2 homozygotes age-1(mg44) expected produce                  representing wide range transduction pathways. <br /> truncated class-I PI3KCS, lacking kinase domain C-                         IIS genes. We assessed genes involved insulin-like <br /> terminus protein. These worms lack main bands                 signaling. Results comparison independent biological <br /> recognized antibodies AGE-1 C-terminal region (Figure                 replicates strain shown graphically Figure 3, and <br /> S3); residual bands represent class-II -III homologs                 data additional genes tabulated (including <br /> AGE-1. PIP3, formed exclusively class-I PI3K,                 statistical significance) Table 1. Transcript levels age-1(mg44) <br /> greatly reduced absent. PIP3 strictly required PDK-1                  F2 adults markedly reduced, factors 4- to <br /> activation AKT kinase, phosphorylates                         14-fold, kinase genes IIS pathway akt-1 and <br />  <br />  <br />  <br />  <br /> Figure 2. Most phosphoproteins depleted age-1(mg44) F2 adult worms. A 2-dimensional polyacrylamide gel electrophoresis resolves <br /> dual-labeled phosphoproteins, enriched phospho-affinity columns (Qiagen), age-1(mg44) F2 worms collected 10 d L4/adult molt <br /> (red fluorescence) post-gravid, near-isogenic N2DRM controls 6 d adults (green fluorescence). A total 1,669 spots were <br /> identified quantified fluor co-electrophoresis. Red/green ratios equal greater 2 (i.e., 2-fold abundant age- <br /> 1(mg44) N2DRM adults) observed 13 spots; 30 ratios 1.2&#226;&#8364;&#8220;2.0; 122 essentially constant (ratios 0.8&#226;&#8364;&#8220;1.2); 305 fell between <br /> 0.5 0.8; 1,199 reduced twofold abundance age-1(mg44) adults. A reduction 3-fold seen 784 spots, and <br /> $5-fold 287 spots. <br /> doi:10.1371/journal.pgen.1000452.g002 <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                                 4                             April 2009 | Volume 5 | Issue 4 | e1000452 <br />                                                                                               Hybrid Feedback Loop Governs Insulin/IGF-1 Response <br />  <br />  <br />  <br />  <br /> Figure 3. Transcriptional suppression IIS genes age-1(mg44) F2 adults. Transcript levels assayed real-time polymerase chain <br /> reaction (RT-PCR). Expression histograms shown superimposed simplified schematic IIS pathway. Yellow arrows indicate protein <br /> phosphorylations (symbolized circled P&#226;&#8364;&#8482;s), orange arrows indicate binding/activation phosphatidylinositol 3,4,5-triphosphate (PIP3, red <br /> &#226;&#8364;&#732;&#226;&#8364;&#732;structural&#226;&#8364;&#8482;&#226;&#8364;&#8482; symbols). Transcriptional outputs indicated open block arrows. Within histogram, means6SEM shown log(2) scale <br /> for steady-state transcript level, comparing wild-type age-1 mutant populations dauer larvae. For strain indicated, fold changes <br /> are shown (e.g., &#226;&#8364;&#732;&#226;&#8364;&#732;36&#226;&#8364;&#8482;&#226;&#8364;&#8482;), age-1(mg44)-F2 relative N2DRM. The age-1(mg44) worms post-gravid F1 homozygotes days 8&#226;&#8364;&#8220;9 adulthood, F2 <br /> homozygotes day 10 L4/adult molt; strains (N2DRM, age-1(hx546), daf-16(mu86); age-1(mg44) double mutants) harvested <br /> as post-gravid adults (day 6 adulthood), dauer larvae (N2DRM only, starved, dense cultures 1 day .98% worms become <br /> resistant 1% sodium dodecyl sulfate). Transcript levels assayed 3&#226;&#8364;&#8220;8 independent biological replicates, cDNA syntheses two <br /> RT&#226;&#8364;&#8220;PCRs each. All results normalized mean control genes (b-actin, T08G5.3, Y71D11.3) did differ significantly <br /> among strains. Significant differences, relative N2DRM controls, indicated asterisks Table 1. <br /> doi:10.1371/journal.pgen.1000452.g003 <br />  <br /> akt-2. This silencing extends age-1 gene itself (encoding        wormbase.org). Five (including genes known IIS <br /> the class-I catalytic subunit PI3K) and, lesser extent,          antagonists INS-1 INS-18) upregulated 3- 10-fold in <br /> class-II class-III PI3KCS genes (Table 1). Three genes                   age-1(mg44) F2 adults, ins-7 ILP gene&#226;&#8364;&#8221;an established <br /> encoding transcriptional cofactors DAF-16/FOXO                       ILP-receptor agonist <span id='am-399' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-400' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-401' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-402' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-403' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-404' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-405' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-406' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-407' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-408' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-409' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-410' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-411' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-412' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-413' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-414' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-415' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-416' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-417' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-418' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-419' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-420' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-421' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-422' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-423' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-424' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-425' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-426' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-427' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-428' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-429' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-430' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-431' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-432' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-433' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-434' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-435' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-436' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-437' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-438' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-439' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-440' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-441' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-442' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-443' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-444' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-445' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-446' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-447' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-448' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-449' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-450' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-451' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-452' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-453' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-454' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-455' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-456' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-457' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-458' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-459' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-460' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-461' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-462' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-463' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-464' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-465' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-466' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-467' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-468' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-469' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-470' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-471' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-472' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-473' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-474' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-475' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-476' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span>RNA interference</span> (RNAi) extends <br /> significantly inhibited age-1(mg44): smk-1 [34] sir-2.1 [35],         life [39]&#226;&#8364;&#8221;had roughly half wild-type transcript level. In <br /> encoding coactivators reinforce unphosphorylated DAF-16/                addition, positive targets known induced DAF-16/ <br /> FOXO, par-5, encoding member &#226;&#8364;&#732;&#226;&#8364;&#732;14-3-3&#226;&#8364;&#8482;&#226;&#8364;&#8482; family             FOXO [24,40] serve confirm enhanced activity age- <br /> that binds inactivates IIS-phosphorylated DAF-16/FOXO                    1(mg44) adults: sod-3, encoding mitochondrial superoxide dismu- <br /> [36]. In addition, daf-18 gene encoding PTEN phosphatase                 tase-3 (up 86), R11A5.4, encodes phosphoenolpyr- <br /> (opposing PI3K) [37,38] attenuated .12-fold.                             uvate carboxykinase (PEPCK), key activator gluconeogenesis <br />    The C. elegans genome contains 38 genes encoding                 (up 8.56). <br /> putative insulinlike peptides (ILPs), functional roles           For part, profile IIS transcripts consistent with <br /> initiation IIS established ILPs [12].            concerted silencing IIS pathway age-1(mg44), leaving <br /> Genes encoding ILPs, known presumed ligands DAF-               DAF-16 active state, unphosphorylated sites would <br /> 2 insulin/IGF-1 receptor, selected study based prior             exclude nucleus. Thus, age-1(mg44) F2 adults, <br /> evidence biological function, cited WormBase (www.                insulinlike peptide genes antagonize IIS upregulated while <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                              5                            April 2009 | Volume 5 | Issue 4 | e1000452 <br />                                                                                                Hybrid Feedback Loop Governs Insulin/IGF-1 Response <br />  <br />  <br />  <br /> Table 1. Expression signal-transduction genes C. elegans strains, assessed real-time polymerase chain reaction. <br />  <br />  <br /> Pathway       Gene           Protein Function / Notes          DAF-16 sites{   hx546 daf-16; hx546 F1mg44      F2 mg44     daf-16; mg44   dauers <br />  <br /> Insulin/IGF-1 Signaling (IIS) pathway <br />               ins-1          DAF-2/IIS antagonist              1, 1            16.0* 4.0           2           10.0***     3.5u           80.0**** <br />               ins-5          neuronal expression;           1, 3            1.9     0.90        24.0**      4.5*        1.0            7.4* <br />                              effect RNAi <br />               ins-6          poss. agonist; effect          0, 2            1.9     1.4         10.7**      3.2*        1.2            2.2 <br />                              RNAi <br />               ins-7          DAF-2/IIS agonist; RNAi           0, 2            5.3*    5.3*        0.95        0.55u       1.2            1.6 <br />                              extends life <br />               ins-14         possible DAF-2 / IIS agonist      0. 0            1.5     1.1         26.7*       6.8****     1.8*           22.0***** <br />               ins-18         IIS antagonist; structure similar 1, 1            1.4     1.5         3.0u        5.0***      1.1            16.1**** <br />                              INS-1 <br />               daf-2          insulin/IGF-1 receptor            1, 2            0.38u   0.90        1.6u        0.20*****   0.19*****      0.55u <br />               ist-1          insulin-receptor substrate 1      3, 0            2.0     0.55        1.3u        1.8*        0.90           6.5 <br />                              (IRS-1) <br />               age-1          PI3Kcs (class-I)                  0, 0            1.6     0.80        0.06***     0.07*****   0.63           0.17** <br />               F39B1.1        PI3Kcs (class-II)                 2               0.90    0.80        0.90        0.25***     0.90           0.90 <br />               vps-34         PI3Kcs (class-III); vesicular     0, 2            1.7     0.90        0.90        0.37**      1.0            0.44 <br />                              trafficking <br />               daf-18/pten    PIP3 phosphatase (opposes         0, 0            1.8     1.0         0.35u       0.08*****   1.0            0.03***** <br />                              AGE-1) <br />               sgk-1          serum/glucocorticoid-             2               1.0     0.80        0.18*       0.25**      0.85           0.15* <br />                              dependent kinase 1 <br />               pdk-1          phosphoinositide-dependent        2, 1            0.44    0.63        1.4         0.18**      0.52u          0.50u <br />                              kinase 1 <br />               akt-1          ortholog S/T kinase            3, 1            1.0     1.0         1.3         0.60        1.2            0.46u <br />                              AKT/PKB (RAC-a) <br />               akt-2          homolog S/T kinase             2               1.90    0.64        5.2u        2.6u        0.50           3.2 <br />                              AKT/PKB (RAC-c) <br />               daf-16         FOXO1 / FOXO3 forkhead            3, 3            0.90    0.40        0.70        0.60u       0.50u          1.0 <br />                              transcr&#226;&#8364;&#8482;n factor <br />               sod-3          Mn-superoxide dismutase           2               0.57    0.55        3.0u        9.0***      0.90           7.0* <br />                              (mitoch. SOD) <br />               pepck          phosphoenolpyruvate               0, 1            0.90    1.1         0.80        8.5*        0.95           0.90 <br />                              carboxykinase <br /> DAF-16-interacting transcriptional regulators <br />               cst-1          DAF-16 kinase, responds        0, 0            2.3     0.90        1.3         1.0         1.2            2.6u <br />                              oxidat.-stress <br />               bar-1          b-catenin transcript&#226;&#8364;&#8482;n factor     0, 3            0.60    1.1         2           0.60        0.70           2 <br />                              (Wnt pathway) <br />               smk-1          transcr&#226;&#8364;&#8482;l coactivator DAF-16   1, 3            0.72    1.1         0.70        0.28**      0.87           0.18*** <br />                              PHA-4 <br />               par-5          14-3-3 family; sequesters         1, 3            1.5     1.1         0.48        0.28**      1.1            0.13*** <br />                              pDAF-16 <br />               ftt-2          14-3-3 family; sequesters         2               1.5     0.90        0.62        0.86        1.0            0.63 <br />                              pDAF-16 <br />               sir-2.1        transcriptional coactivator    0, 1            0.91    1.1         0.95        0.26**      0.89           0.29** <br />                              DAF-16 <br /> TGF-b signaling <br />               daf-7          TGF-b family member (ligand,      2               7.0*    2.6         3.0         5.3**       1.2            19.7** <br />                              agonist) <br />               daf-1          TGF-b family receptor type I      1, 0            2.2u    1.5u        0.50u       0.22*****   1.6            1.1 <br />               daf-4          TGF-b family receptor type II     1, 3            1.5     1.0         0.60        0.24***     1.5            1.2 <br />               daf-12         VDR homolog, controlled        2               1.6     1.0         0.70        1.6         0.80           3.1u <br />                              TGF-b, IIS <br />               daf-3          SMAD transcription factor         2, 1            1.5u    0.90        0.30***     0.30**      1.9**          1.7* <br />                              c TGF-beta signaling outputs p38 ERK MAPK pathways <br />  <br />  <br />  <br />  <br />      PLoS Genetics | www.plosgenetics.org                                  6                             April 2009 | Volume 5 | Issue 4 | e1000452 <br />                                                                                                          Hybrid Feedback Loop Governs Insulin/IGF-1 Response <br />  <br />  <br />  Table 1. Cont. <br />  <br />  <br />  Pathway         Gene             Protein Function / Notes         DAF-16 sites{      hx546 daf-16; hx546 F1mg44          F2 mg44      daf-16; mg44      dauers <br />  <br />  AMPK/TOR pathway (<span id='am-2' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> sensing) <br />                  aak-1            AMP-dependent kinase 1           2, 1               1.2    0.80             0.47        0.17***      1.0               0.05* <br />                  aak-2            AMP-dep&#226;&#8364;&#8482;t. kinase 2 (activates   2                  1.7u   1.5*             0.60**      2.2**        0.57              6.4* <br />                                   DAF-16) <br />                  let-363          FRAP/mTOR ortholog; PIK1, 0                 0.70   0.70             1.0         0.25**       0.60              0.47 <br />                                   family <br />                  daf-15           ortholog RAPTOR, mTOR reg. 2, 0                  0.80   0.90             0.80        0.15***      0.80              0.22** <br />                                   subunit <br />                  rsks-1           S6K; ribosomal S6 protein kinase 1, 0               1.6    0.90             1.0         1.5          1.2               3.9* <br />                                   1 <br />  p38-MAPK (stress response, immune response) <br />                  pmk-1            p38 MAPK 1 (mitogen activated 3, 0                  2      1.0              3.8u        0.60         0.70              1.0 <br />                                   prot. kinase) <br />                  pmk-2            p38 MAPK 2 (mitogen activated 2, 0                  2      0.20*            0.40        0.20*        0.18*             0.50 <br />                                   prot. kinase) <br />                  pmk-3            p38 MAPK 3 (mitogen activated 2, 0                  2      0.40             1.2         0.56         0.20*             1.5 <br />                                   prot. kinase) <br />                  atf-2            cAMP-dep. transcription factor   0, 1               2.5    1.1              44.1***     3.9*         2.6u              9.6** <br />                                   family (+) <br />                                   c p38 MAPK signaling activates DAF-3 <br />  JNK-MAPK (immune signaling, stress response) <br />                  jnk-1            Jun N-terminal kinase 1          0, 0               1.2    1.4              0.95        3.5***       1.2               13.2***** <br />                  jun-1            JUN subunit, AP-1 transcription 2, 2                0.74   1.0              1.1         0.6u         0.95              2.2* <br />                                   factor (+) <br />                                   c JNK signaling converges DAF-3 DAF-16/FOXO <br />  ERK-MAPK (growth-factor immune responses) <br />                  let-60 RAS       small membrane GTPase co-        0, 0               0.50   0.30             0.18*       0.16***      0.50              0.30 <br />                                   receptor <br />                  lin-45           Ser/Thr kinase MEK-2, activ by0, 1               1.0    0.80             1.1         0.20**       0.90              0.33** <br />                                   LET-60 <br />                  mek-1            stress-response ERK JNK-1,   3, 1               0.75   0.50             0.30*       0.23*****    0.38u             0.07**** <br />                                   MPK-1 <br />                  mpk-1            MAPK, transduces develop&#226;&#8364;&#8482;l RAS 0, 2                 0.80   1.8              2.1         0.23*        1.0               0.21* <br />                                   signals <br />                  gsk-3            glycogen-synthase kinase 3       0, 0               0.56   0.50             0.28u       0.20**       0.90              0.13* <br />                  skn-1            oxid.-damage-response TF,        0, 0               1.3    1.0              0.55        0.30*        1.6               0.60 <br />                                   inhib&#226;&#8364;&#8482;d IIS <br />  <br />  Key: 2, assessed. Significance difference N2DRM (by 2-tailed t-test): u, nominally significant P,0.05. <br />  *, P,0.01; **, P,0.001; ***, P,1E24; ****, P,1E25, *****, P,1E26. <br />  { <br />   Number exact consensus DAF-16 sites upstream 5-kb span, (a, b), = GTAAA(C/A)AA, b = CTTATCA. <br />  For gene tested, wild-type N2DRM adults compared age-1 mutant populations dauer larvae. The age-1(mg44) worms F1 homozygotes at <br />  days 8&#226;&#8364;&#8220;9 adulthood, time post-gravid, F2 homozygotes day 10 (18 days post-hatch); groups (N2DRM, age-1(hx546), daf-16(mu86); <br />  age-1(mg44) double mutants) harvested post-gravid (days 6&#226;&#8364;&#8220;8 adulthood), dauer larvae (N2DRM only) starved, dense cultures 1 day .98% <br />  worms resistant lysis 1% sodium dodecyl sulfate. Transcript levels assayed 3&#226;&#8364;&#8220;8 independent biological replicates, cDNA <br />  syntheses RT-PCRs each. Numbers shown transcript ratios group indicated (in column header) relative transcript levels genes in <br />  near-isogenic N2DRM controls. All C(t) data (threshold cycle numbers) normalized mean values control genes (b-actin, T08G5.3, Y71D11.3) that <br />  did change strains/groups tested. Changes nominally significant (P,0.05) emphasized bold font. <br />  doi:10.1371/journal.pgen.1000452.t001 <br />  <br />  <br /> ins-7 (the confirmed IIS agonist) downregulated. Genes                        phosphatase PTEN, par-5, encoding 14-3-3 <br /> supplying IIS kinase cascade inactivates DAF-16/FOXO                         proteins sequester IIS-phosphorylated DAF-16&#226;&#8364;&#8221;genes that <br /> (daf-2, age-1, sgk-1, pdk-1) downregulated, ist-1,                arguably superfluous absence PIP3 IIS-mediated <br /> encoding homolog mammalian IRS-1 IRS-2 [41]                            phosphorylation DAF-16, respectively. <br /> serve convergence points inhibitory inputs                        Other signal transduction genes. In view global <br /> signaling pathways [41,42], upregulated. Moreover, aak-2,                      suppression kinase activities (Figure 1), extended the <br /> sole AMP-dependent kinase (AMPK) gene implicated DAF-16/                           transcriptional assay panel include non-IIS kinases other <br /> FOXO activation [43,44], shown extend lifespan                            signal-transduction genes (Table 1). The TGF-b ERK/ <br /> over-expressed [45], upregulated age-1(mg44) F2 adults                          MAPK (mitogen-activated protein kinase) signaling pathways, <br /> (Table 1). Apparent exceptions include low transcript levels                      known cross-talk extensively IIS [46], thoroughly <br /> for genes antagonize IIS: daf-18, encoding PIP3 3-                       silenced, 3 6 genes (respectively) downregulated more <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                                       7                                 April 2009 | Volume 5 | Issue 4 | e1000452 <br />                                                                                                   Hybrid Feedback Loop Governs Insulin/IGF-1 Response <br />  <br />  <br /> than 3-fold. In contrast, transcripts elevated daf-7 gene,           differences age-1(mg44) F2 N2DRM adults, 25 were <br /> encoding known TGF-b peptides C. elegans. The                     completely reversed (to 20% N2DRM level, over- <br /> <span id='am-3' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span>-responsive AMPK/TOR pathway largely silenced,                        reverted) mutations inactivating daf-16; 9 genes showed <br /> although translational output, rsks-1/S6K, not. The                   partial reversion, two, daf-2 pmk-2, did revert all. <br /> AMPK genes, aak-1 aak-2, clearly serve distinct functions (in                Four instances partial negligible reversion (ins-14, daf-2, pmk-2 <br /> view differing RNAi effects), diverge roles             pmk-3) entail expression levels daf-16(mu86); age-1(mg44) adults <br /> respect inhibition TOR complex IIS (via IST-1/IRS-                 differed significantly N2DRM (each P,0.01; combined <br /> 1/2), activating phosphorylation DAF-16/FOXO [43&#226;&#8364;&#8220;45].                      P,1029). Results similar m26 mu86 alleles daf-16; <br /> The p38/MAPK pathway, interacts IIS [47],                     Table 1 shows data mu86 allele daf-16 <br /> ambiguous p38/MAPK genes                        exons deleted. These data demonstrate age-1(mg44) F2 effects <br /> significantly downregulated (pmk-2, 5-fold, P,0.005),                 transcription primarily mediated DAF-16/FOXO, are <br /> the affected atf-2, mediating                        supplemented DAF-16-independent mechanisms&#226;&#8364;&#8221; <br /> transcriptional output pathway, induced (Table 1). A                paralleling conclusions drawn earlier data daf-2 <br /> similar conflict effects evident JNK/MAPK pathway,                  mutants [50]. <br /> wherein jnk-1 (encoding Jun N-terminal kinase 1) induced age-                Expression changes differ F2 F1 age-1(mg44) <br /> 1(mg44) F2 adults, targets, jun-1 encoding JUN                homozygotes. Transcriptional modulation seen first- <br /> transcription factor, modestly downregulated. This suggests              generation (F1) age-1(mg44) adults, inductions <br /> the critical activation target JNK-1 (in context age-1                 occasionally exceeded level seen F2. In fact, three <br /> longevity) JUN, DAF-16/ FOXO (see [48]).                         highest induction factors observed F1 adults, ins-5 <br />    Altogether, targeted gene survey indicates widespread                    (246), ins-14 (276), atf-2 (446), ins-6 (116) showing <br /> silencing signal-transduction genes age-1(mg44) F2 adults,            substantially higher transcript levels F1 worms F2. <br /> significant declines relative N2DRM (P,0.01) 23 47                 However, expression 21 genes altered 2- 8-fold F2 <br /> genes tested, increases 12 genes. The declines imply                     worms F1 (e.g., ist-1, vps-34, daf-18, sod-3, smk-1, sir2.1, aak- <br /> attenuation IIS, TGF-b, ERK/MAPK pathways,                     1 daf-15), 6 genes roughly equal two <br /> silencing redirection AMPK/TOR <span id='am-6' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span>-sensing                        generations (age-1, sgk-1, daf-3, let-60, mek-1, gsk-3). Clearly, the <br /> pathway. Two pathways implicated immune stress responses,                 consequences maternal protection differ genes. <br /> p38/MAPK JNK/MAPK, appear outputs                           Comparison IIS               mutants           dauer <br /> inhibited transcript level. The bias inhibition               larvae. F2 age-1(mg44) adults            generally far more <br /> induction reflection global transcriptional silencing,            pronounced transcriptional effects weaker age-1 allele, <br /> because data normalized mean control genes              hx546 (Figure 3 Table 1). Only 47 tested genes (4%) <br /> (b-actin, T08G5.3, Y71D11.3) did change                   changed significantly (at P,0.01) age-1(hx546) adults: ins-1 (up <br /> strains tested. However, bias consistent results            166 N2DRM) ins-7 (up 5.36 hx546, 2-fold <br /> microarray survey (Ayyadevara et al., submitted) indicated                 mg44-F2 adults). This stark contrast 35 significantly <br /> that differential gene expression strong age-1 mutants reflects               differential genes (74%) seen mg44-F2 adults&#226;&#8364;&#8221;underscoring the <br /> primarily transcriptional suppression targeting genes,             quite atypical properties stronger age-1 allele. The widely <br /> largely entirely specific alleles.                                   studied daf-2(e1370) strain tested transcript levels 17 <br />   Expression changes age-1(mg44) F2 adults                             genes, 4 (24%) differed significantly N2DRM <br /> simply recapitulate younger adult state wild-type                           (at P,0.01; Table S2). These genes (ins-1, sgk-1, daf-7 and <br /> worms. Might observed differences gene expression                         aak-2) upregulated daf-2 mutant. Like age-1(hx546), <br /> reflect considerable differences physiological age (or                 daf-2(e1370) temperature-sensitive mutant weaker <br /> chronological age) young-adult worms                             effect age-1 nonsense mutants; life extension of <br /> compared? Gene expression Drosophila lines, varying                        twofold hands, outcrossing N2DRM <br /> longevity threefold, reported fall                      background (data shown). Both daf-2(e1370) age-1(hx546) <br /> classes: genes alter transcript levels time, independent            upregulated signaling components tested IIS <br /> lifespan, expression patterns scale                  TGF-b pathways, nearly genes (all except <br /> lifespan [49]. Of genes appeared strongly modulated                  ins-1, encoding IIS antagonist, daf-7, encoding four <br /> age-1(mg44) F2 adults, subset chosen represent                     TGF-b ligands) strongly suppressed age-1(mg44) F1 and <br /> signaling pathways, assessed wild-type N2DRM adults                    especially F2 adults (Table S2). <br /> several adult ages, ages age-1(mg44) F2                     F2 age-1(mg44) adults differed significantly dauer larvae for <br /> adults. This time series (Figure 4) demonstrates              43% (20/46) signal-transduction genes tested both; the <br /> the expression changes observed long-lived age-1 worms                   number (20) transcript levels twofold of <br /> could interpreted reflections physiologically youthful            dauer larvae (Table 1)&#226;&#8364;&#8221;suggesting very <br /> state relative wild-type worms. In particular, expression age-             long-lived populations achieve resistance aging distinct <br /> 1(mg44) F2 worms 8 16 days adult age (5 10%                     partially overlapping gene-expression strategies. <br /> their mean adult life spans) did overlap levels seen                      The transcript-level changes observed age-1(mg44) F2 adults <br /> N2DRM worms 1, 3 6 days adult age (6, 18 37% life                    cross-checked previously reported daf- <br /> span), genes ins-1, ins-14, daf-18, pepck, daf-1, daf-4, aak-2,       2(e1370) adults, wild-type dauer larvae, [21,24,51&#226;&#8364;&#8220;53]. Few <br /> let-363. Thus, differences expression mirror genotype               commonalities present data gene <br /> rather aging. In contrast, scaling lifespan largely eliminates           lists, comprise highly significantly altered genes <br /> the difference strains genes ERK/MAPK                     microarray surveys. The differences reflect the <br /> pathway (daf-15 mek-1), intermediate outcomes                     greater sensitivity RT-PCR detect quantify low- <br /> seen pdk-1, daf-3, lin-45.                                               abundance transcripts; however, data Tables 1 Table S2 <br />   Silencing involves daf-16&#226;&#8364;&#8220;dependent &#226;&#8364;&#8220;independent                      imply transcriptional shifts age-1(mg44) F2 adults differ both <br /> routes. Of 35 nominally significant (P,0.01) transcriptional                     qualitatively quantitatively seen mutants. <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                                  8                            April 2009 | Volume 5 | Issue 4 | e1000452 <br />                                                                                               Hybrid Feedback Loop Governs Insulin/IGF-1 Response <br />  <br />  <br />  <br />  <br /> Figure 4. Most transcriptional changes age-1(mg44) F2 adults remain scaling lifespan. Transcript levels assayed by <br /> real-time polymerase chain reaction (RT&#226;&#8364;&#8220;PCR) described legends Figure 3 Table 1. Within histogram, means6SEM shown on <br /> a log(2) scale steady-state transcript level, comparing wild-type N2DRM adults ages age-1(mg44) F2 homozygotes adult <br /> ages. All transcript values normalized b-actin mRNA (to adjust variation RNA inputs), biological group was <br /> normalized mean value N2DRM day-6 adults (to correct run-to-run variation C(t) values). Ages measured L4/adult molt; a <br /> table (inset) indicates percent adult lifespan represented age. To monitor longitudinal changes, biological preparations each <br /> group assessed (fewer used Figure 3 Table 1). Significance differences ascertained two-tailed Behrens-Fisher t-tests, <br /> appropriate samples unequal unknown variance. The transcript levels observed experiment differ means seen previous <br /> experiments (e.g., Table 1), differences significant. <br /> doi:10.1371/journal.pgen.1000452.g004 <br />  <br />  <br /> Functional consequences transcriptional suppression                       wild-type levels double mutants daf-16, note that <br /> in age-1(mg44) worms                                                         reversion quite complete, using weaker daf- <br />    F2 age-1(mg44) adults, 4- 5-fold longer-lived           16(m26) allele [30] daf-16(mu86) deletion allele (Figure 5A), <br /> F1 adults (comparing data [8,29] [30]), outperform          indicating stress-resistance traits mediated by <br /> parents respect resistance oxidative electrophilic            DAF-16-independent pathway. These results parallel the <br /> stresses (Figure 5A 5B). Relative N2DRM controls, survival            incomplete reversion, daf-16; age-1(mg44) double mutants, seen <br /> in 5% hydrogen peroxide extended 2-fold F1 adults, 10-             vitro kinase activity phosphoprotein levels (Figure 1) and <br /> fold F2 progeny. Because age-1(mg44)-F2 adults barely               transcript levels genes (Table 1). <br /> reached 20% mortality 24 h, time 100% worms                   Most age-1(mg44)-downregulated genes essential <br /> had died groups, survival time compared            nematode growth development. That is, double-stranded <br /> threshold 20% mortality. Survival electrophilic stress, 4-          RNAs (dsRNAs) targeting them, administered developing C. <br /> HNE (similarly defined time 20% mortality) increased 1.6-              elegans, produce embryonic lethality larval arrest [54&#226;&#8364;&#8220;56]. The <br /> fold age-1(mg44) F1 worms 5-fold F2, reference             impact knockdown, however, far remained largely <br /> N2DRM. Although resistance stresses restored           untested adults. Because resistance oxidative stresses a <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                              9                            April 2009 | Volume 5 | Issue 4 | e1000452 <br />                                                                                                   Hybrid Feedback Loop Governs Insulin/IGF-1 Response <br />  <br />  <br />                                                                                   common feature long-lived C. elegans mutants [57], in <br />                                                                                   particular parallels longevity age-1 allele set studied here <br />                                                                                   (Figure 5 [30]), employed short-term assay to <br />                                                                                   evaluate contribution individual-gene downregulation, to <br />                                                                                   exceptional survival age-1(mg44)-F2 adults benign <br />                                                                                   toxic environments. <br />                                                                                      Hydrogen peroxide resistance measured duplicate <br />                                                                                   experiments, wild-type N2DRM worms exposed <br />                                                                                   dsRNA-expressing bacteria targeting 10 genes which <br />                                                                                   transcript levels markedly reduced age-1(mg44)-F2 adults. <br />                                                                                   Genes selected directly involved the <br />                                                                                   IIS pathway, representing variety signaling <br />                                                                                   pathways, RNAi constructs available from <br />                                                                                   Ahringer library [56]. E. coli, harboring empty-vector <br />                                                                                   control expressing 10 gene-targeted dsRNA species, were <br />                                                                                   fed mature adults (days 3 6 L4/adult molt) so <br />                                                                                   preclude effects development. Survival curves, during <br />                                                                                   subsequent exposure 5-mM H2O2, shown Figure 5C. <br />                                                                                   RNAi genes (encoding transcription factor and <br />                                                                                   components distinct protein-kinase signaling cascades) <br />                                                                                   produced highly significant gains peroxide survival (each <br />                                                                                   P,0.001), fifth dsRNA exposure offered marginally <br />                                                                                   significant protection (vps-34, P,0.03). The remaining dsRNA <br />                                                                                   treatments discernible effect survival, compared to <br />                                                                                   worms exposed expression vector. The above <br />                                                                                   results reproduced independent experiment, the <br />                                                                                   genes attaining P,0.001, vps-34 achieved P,0.08. <br />                                                                                      Genes (and encoded proteins) RNAi knock-down <br />                                                                                   conferred protective effect daf-3 (SMAD transcription <br />                                                                                   factor) daf-4 (TGF-b receptor, Ser/Thr kinase), both <br />                                                                                   involved TGF-b signaling; aak-1 (AMP-dependent protein <br />                                                                                   kinase 1), AMPK/TOR pathway; let-60 (RAS-family <br />                                                                                   GTPase activating MAPK), ERK-MAPK pathway, <br />                                                                                   vps-34 (class-III PI3KCS), involved vesicular trafficking and <br />                                                                                   autophagy. None individual RNAi effects matched the <br />                                                                                   peroxide survival untreated age-1(mg44) F2 adults 62 days of <br />                                                                                   adult age (large diamond symbols, Figure 5C). These data <br />                                                                                   demonstrate transcript-level changes seen age-1(mg44) F2 <br />                                                                                   homozygotes favor oxidative-stress survival. They also <br />                                                                                   contribute incrementally greatly enhanced longevity F2 <br />                                                                                   homozygotes, able confirm effects. <br />                                                                                   When begun end larval development, RNAi aak-1 and <br />                                                                                   daf-4 extended survival 7&#226;&#8364;&#8220;11%, let-60 dsRNA reduced it <br />                                                                                   ,12% (data shown). Such small effects life span require <br /> Figure 5. Survival oxidative electrophilic stresses.                       large groups reach significance; moreover, significance one <br /> N2DRM worms exposed (A) 5-mM hydrogen peroxide (H2O2),                    experiment provides little assurance independent replicates <br /> or (B) 10-mM 4-hydroxynonenal (4-HNE). Stress survivals began adult            attain significance. This reflect low statistical power <br /> day 3&#226;&#8364;&#8220;4 worms maintained liquid medium, 2060.3uC, <br />                                                                                   inherent survivals modest size, and/or inability control <br /> without bacteria. Results shown replicated independent <br /> experiments. Statistics (log-rank test, n = 40&#226;&#8364;&#8220;50, A B):            environmental variance experiments. <br /> age-1(mg44) F2 vs. F1 generation: P,1027; age-1(mg44) F1 vs. any <br /> other strain: P,1023; age-1(mg44) F2 vs. strain group:               Discussion <br /> P,1029; age-1(hx546) vs. N2DRM: P,0.004; daf-16(mu86); age-1(mg44) <br /> vs. N2DRM: P,0.005. (C) Functional consequences suppressing                       First-generation homozygotes age-1(mg44) mutation <br /> signaling genes downregulated age-1(mg44) F2 adults. N2DRM                     develop normally fertile adults 15&#226;&#8364;&#8220;25uC, display stress <br /> worms picked day 1 adulthood, maintained dsRNA-                 resistance life extension typical IIS mutants [8,14,29]. <br /> expressing E. coli, 20uC, 3 days. RNAi extension 50th-percentile <br /> H2O2 survival, percent gain N2DRM fed bacteria carrying                In contrast, progeny&#226;&#8364;&#8221;second generation homozygotes&#226;&#8364;&#8221; <br /> empty expression vector (significance, Gehans log-rank test),            develop slowly 15&#226;&#8364;&#8220;20uC, form infertile adults far <br /> follows: vps-34, 26% (P,0.03); let-60/RAS, 40% (P,0.001); daf-3, 49%              stress resistant four-fold longer lived other <br /> (P,0.0003); aak-1, 49% (P,0.00003); daf-4, 57% (P,0.0001). All                    IIS mutants [30]. Maternal protection (oocyte carryover age-1 <br /> significant effects vps-34 confirmed (each P,0.001)                mRNA, AGE-1 kinase PIP3 product, synthesized the <br /> independent experiments. Peroxide survival F2 age-1(mg44) day-62 <br /> adults, RNAi, exceeded N2DRM controls 5.6-fold <br />                                                                                   heterozygous parent) thought blunt stress-resistance <br /> (P,1026), 60th percentile (the lowest survival observed            longevity traits approximately weaker age-1 or <br /> F2 group).                                                                        daf-2 allele [30]. We total kinase activity also <br /> doi:10.1371/journal.pgen.1000452.g005                                             attenuated severely age-1(mg44) adults F2 the <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                                  10                          April 2009 | Volume 5 | Issue 4 | e1000452 <br />                                                                                               Hybrid Feedback Loop Governs Insulin/IGF-1 Response <br />  <br />  <br /> F1 generation (see Figure 1C Figure S2),                downstream consequences confer uniformly pro-longevity <br /> kinase substrates remain unaffected mutation (Table S1               effects; gene likely serve &#226;&#8364;&#732;&#226;&#8364;&#732;masters&#226;&#8364;&#8482;&#226;&#8364;&#8482;, level <br /> and Figure S2). At expression level, situation               expression depend genetic, environmental, and <br /> complex: majority tested genes strongly affected          signaling context. In keeping perspective, down- <br /> the F2 generation age-1(mg44), transcript             stream manifestations longevity assurance genes far less <br /> effects F1 adults equal, exceed, oppose                conserved, evolution distinct physiological <br /> observed F2 adults (Table 1).                                             states given species, over-arching pathways and <br />                                                                              functions serve [67]. <br /> Consequences altered gene expression IIS                             Improved stress resistance survival age-1(mg44) F2 worms, <br /> other pathways                                                               apparently arising transcriptional attenuation signaling <br />    Transcriptional effects IIS pathway fully                 pathways presumed protective, poses intriguing paradox. <br /> consistent impaired insulinlike signaling,               These pathways, activated <span id='am-7' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> deficiency, pathogens, or <br /> expected augment survival mecha-                 growth factors, reported cross-talk IIS diverse <br /> nisms employed weaker IIS mutations. Moreover, repression              levels [19,43&#226;&#8364;&#8220;47,68,69]. This suggests complex fabric of <br /> class-II class-III PI3K catalytic-subunit genes (Table 1)          signaling interactions, impact silencing multiple <br /> impede formation PI(3)P PI(3,4)P2, suppressing alternative            components predicted. Moreover, signaling that <br /> routes PI(3,4,5)P3. Increased expression aak-2 contributes          promotes survival variable hostile setting entail <br /> activation DAF-16/FOXO, opposing IIS (which inhibits              energy costs harmful side-effects unwarranted <br /> this transcription factor) increasing life span [43,44].                 constant, pathogen-free environment abundant food. An <br />    In addition effects IIS genes, however, age-1(mg44)-F2              organism avoids deleterious aspects surveillance <br /> adults striking transcriptional attenuation             systems reap survival benefits benign conditions. <br /> other signal transduction pathways interact IIS              In instances, expression changes seen strong-age-1 <br /> one another. TGF-b endocrine/paracrine signaling active                mutants appear oppose longevity stress-resistance, <br /> development, modulates signaling pathways                  based effects down- upregulation previously reported <br /> including p38/MAPK ERK/MAPK [46]. Both daf-1 daf-                    genes. For example, age-1(mg44) F2 adults <br /> 4, encoding type-I -II TGF-b receptors, respectively,                downregulate daf-18, encodes PIP3 3-phosphatase, <br /> downregulated 4- 5-fold age-1(mg44)-F2 adults. Expression           PTEN. This expected elevate steady-state level of <br /> also reduced 3-fold daf-3, encoding co-SMAD transcription              PIP3, enhancing IIS reducing longevity normal <br /> factor deployed pathways including TGF-b. Perhaps              worms. However, absence AGE-1/PI3KCS kinase <br /> partial compensation signaling downregulation, daf-7            activity, little PIP3 substrate PTEN <br /> gene encoding TGF-b-family ligand/agonist 5-fold upregu-                act. A second example downregulation age-1(mg44) F2 <br /> lated. Silencing TGF-b signaling, RNAi directed daf-3            adults skn-1, encoding transcription factor responsive to <br /> daf-4, improves survival presence hydrogen peroxide,               oxidative damage regulated IIS [70&#226;&#8364;&#8220;72]. Reduced <br /> consistent prior observation daf-1, -4 -7 mutants            expression skn-1 odds increased oxidative-stress <br /> are long-lived [46].                                                         resistance longevity; however, long-lived worms <br />    AMPK/TOR signaling implicated innate immunity                 generate lower levels reactive oxygen species, thereby <br /> and stress responses. Although remains controversial              reducing skn-1 induction. RNAi vps-34 (encoding class-III <br /> the primary response microbe stress causes            PI3KCS required vesicular trafficking autophagy [73]) was <br /> [47], secondary role nutrient sensing                recently shown block life extension eat-2(ad1116) daf- <br /> [37,43]. AAK-1 AAK-2 regulated PAR-4                          2(mu150) mutants, wild-type worms [58]. <br /> transcription factor [37,58,59], aak-1 knockdown RNAi                 Autophagy induced TOR deficiency [58], several <br /> confers resistance oxidative stress (Figure 5C). Inhibition        TOR signaling components downregulated F2 worms <br /> C. elegans TOR pathway confers stress resistance extends life            (Table 1). Considering this, autophagy least <br /> span [60,61].                                                                moderately induced worms, absence not <br />    ERK/MAPK signal-transduction essential devel-                 account low expression vps-34. These results argue any <br /> opmental processes; constituent genes                   direct role vps-34 attenuation exceptional longevity age- <br /> expressed adult nematode tissues, presumed               1(mg44) F2 worms. The possibility remains, however, vps-34 <br /> post-developmental functions defined [62&#226;&#8364;&#8220;64]. All                downregulation reinforce PIP3-depleting effects strong age- <br /> genes tested pathway markedly downregulated, 3-            1 mutation. Downregulation age-1(mg44) worms, transcripts for <br /> .6-fold (Table 1), RNAi inhibition let-60 (encoding RAS             let-60/RAS members ERK-MAPK cascade, <br /> membrane co-receptor initiates ERK/MAPK signaling)                      expected oppose additional life extension that <br /> significantly improves survival oxidative stress (Figure 5C).             typical IIS mutants, let-60 gain-of-function mutation <br />                                                                              enhances daf-2 life extension [74]. RNAi targeting smk-1, encoding <br /> Rationale, anomalies, resolutions                                        transcriptional coactivator shared DAF-16 PHA-4 [75], <br />    The dramatic effects gene mutations life span             reduces stress-resistance lifespan daf-2(e1370) worms [76], <br /> involved hypomorphic (loss-of-function) mutations, genes             effect wild-type worms controversial [34,76]. <br /> affected termed aging &#226;&#8364;&#732;&#226;&#8364;&#732;master genes&#226;&#8364;&#8482;&#226;&#8364;&#8482;. The genes                  Although smk-1 knockdown impairs sod-3 expression daf-2 worms <br /> encoding IIS components provide best-studied example. IIS,            [34], 9-fold elevation sod-3 transcripts face a <br /> common &#226;&#8364;&#732;&#226;&#8364;&#732;master genes&#226;&#8364;&#8482;&#226;&#8364;&#8482; essentially signaling              72% drop smk-1 expression age-1(mg44) (Table 1). Finally, sir- <br /> pathways, regulates numerous genes. In case IIS,              2.1 overexpression reported extend lifespan, knock-down <br /> number modulated ways protective,                shorten [77,78], 4-fold downregula- <br /> otherwise conducive long life, upregulation GSTs           tion sir-2.1 age-1(mg44)-F2 adults. <br /> other detoxification genes, &#226;&#8364;&#732;&#226;&#8364;&#732;foot soldiers&#226;&#8364;&#8482;&#226;&#8364;&#8482;            Although contradictory context extreme stress <br /> longevity assurance [65,66]. However, expect               resistance longevity, &#226;&#8364;&#732;&#226;&#8364;&#732;exceptions&#226;&#8364;&#8482;&#226;&#8364;&#8482; also <br />  <br />  <br />       PLoS Genetics | www.plosgenetics.org                              11                            April 2009 | Volume 5 | Issue 4 | e1000452 <br />                                                                                               Hybrid Feedback Loop Governs Insulin/IGF-1 Response <br />  <br />  <br /> mirrored, cases lesser degree, dauer larvae (Table 1),        cipitated antibody DAF-16/FOXO [52], possible that <br /> a robust state developmental arrest endure months             precise motif matches necessary sufficient DAF- <br /> without reducing adult life span [1,79,80]. This raises                   16 binding. In words, near-match sequences able <br /> possibility genes, effects downregulation           bind DAF-16, perfect-match motifs require <br /> context-dependent, beneficial worms                    additional features nearby DNA. <br /> already highly protected stress aging. Alternatively, these <br /> expression changes follow regulatory mechanisms                      Implication hybrid feedback loop kinase and <br /> shared age-1(mg44) adults N2 dauers, work                   transcriptional components <br /> opposition robustness. This plausible case               It surprising hyperactivation DAF-16/FOXO age- <br /> severe loss-of-function mutant, effects orchestrat-          1(mg44) F2 adults silences essentially entire IIS pathway. This <br /> ed, difficult reconcile highly-evolved alternative           implies positive feedback loop, DAF-16/FOXO <br /> developmental state dauer larva. Perhaps,             imposes transcriptional silencing kinases would <br /> a single coherent program, patterns observe reflect                    inhibit nuclear localization access target <br /> aberrant triggering adult regulatory                    genes (Figure 6). We propose second-generation age-1(mg44) <br /> mechanisms normally utilized developmental                     homozygotes trapped nonadaptive state, incapable of <br /> metabolic regulation. This particular combination mechanisms               responding diverse environmental internal signals. This <br /> could serendipitous result profound alteration PIP3            apparent paradox, failure adaptive mechanisms greatly <br /> levels turn impacts multiple pathways.                               extends lifespan, easily explained mechanisms <br />                                                                               maximize Darwinian fitness &#226;&#8364;&#8220; transmission genetic alleles to <br /> PI3Kcs-mutant effects largely, entirely,                          ensuing generations &#226;&#8364;&#8220; individual survival [83,84]. <br /> mediated DAF-16/FOXO                                                          When IIS kinase signaling predominates (the reproductive state), it <br />    The expression profile age-1(mg44) worms depends largely             suppresses DAF-16/FOXO activity. Activation PI3K favors <br /> DAF-16/FOXO, consistent prior evidence C. elegans IIS               PIP3 production AKT activation, promote cell <br /> operates mainly transcription factor, impacting                  proliferation [18,31,85]. However, IIS switch second, <br /> several target genes [2,21,22,29,52,81]. Although DAF-                functionally distinct, state: kinase signaling weak, DAF- <br /> 16/FOXO regarded largely transcriptional activator              16/FOXO activated. As demonstrated, active <br /> [21,82], effects negative regulation genes [24]. In           DAF-16/FOXO transcriptionally silences upstream <br /> our selected panel genes, two-thirds (22/33) DAF-16-                regulatory kinases, impeded DAF- <br /> mediated effects age-1(mg44) mutation involve reduced tran-                16/FOXO action preventing nuclear localization. There- <br /> script levels, indicating silencing prevails gene transcripts        fore, low-signaling, longevity state IIS self-sustaining. <br /> that encode kinases mediators intracellular signaling.           Biologically, state promotes dauer formation develop- <br /> Twenty-eight genes (of 33 transcripts appear              ment, life-extension delayed reproduction adult <br /> primarily regulated DAF-16/FOXO) mapped DAF-                     (reviewed [4]). Signals inhibit IIS kinases augment DAF- <br /> 16 binding sites 5 kb upstream initiation codon                 16/FOXO action, sufficient, trigger switch reproductive <br /> (Table 1). Of these, 21 (75%) exact matches            longevity state DAF-16/FOXO promotes somatic <br /> the known consensus sites, GTAAA(C/A)AA CTTATCA.                      protective mechanisms (Figure 6). However, exiting stable <br /> Genes lacking sites indirect targets DAF-16/                   longevity mode requires shift balance inputs govern <br /> FOXO, considering motifs occur              positive feedback loop. Such inputs include insulin-like <br /> frequency genome large [81], DNA immunopre-                   peptide agonists antagonists, hormones, pheromones, tran- <br />  <br />  <br />  <br />  <br /> Figure 6. Model IIS &#226;&#8364;&#732;&#226;&#8364;&#732;molecular switch.&#226;&#8364;&#8482;&#226;&#8364;&#8482; Three proposed states depicted, reflecting balance kinase signaling and <br /> transcriptional feedback suppression: reproductive mode, kinase signaling predominates, activated insulin/IGF-1 receptor (DAF-2, here <br /> labeled Ins-R); longevity mode, FOXO (DAF-16) prevails suppresses kinase transcription; hyperlongevity mode&#226;&#8364;&#8221;in switching <br /> between modes blocked, strongly favoring survival possibility resuming reproduction. State transitions are <br /> normally triggered signaling modulators (e.g., insulin-like peptides [12] SIR-2/14-3-3 complex [77]), age-1(mg44)-F2 mutants <br /> (&#226;&#8364;&#732;&#226;&#8364;&#732;hyperlongevity&#226;&#8364;&#8482;&#226;&#8364;&#8482; state) respond. <br /> doi:10.1371/journal.pgen.1000452.g006 <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                              12                            April 2009 | Volume 5 | Issue 4 | e1000452 <br />                                                                                               Hybrid Feedback Loop Governs Insulin/IGF-1 Response <br />  <br />  <br /> scriptional co-activators co-repressors DAF-16/FOXO                    In vitro phosphorylation assay <br /> such SIR-2 14-3-3 proteins, nutrient- stress-sensors              Worms grown 20uC quickly frozen liquid nitrogen to <br /> signaled kinase pathways (e.g., MAPK, JNK                    preserve endogenous kinase activity. Worms suspended 50-mM <br /> AMPK) cross-talk IIS. Combined, normal                    Tris pH 7.5, 80-mM b-mercaptoethanol, 2-mM EDTA, 1-mM <br /> states IIS pathway (reproductive longevity) constitute           PMSF, Protease Inhibitor Cocktail I (CalBiochem), were <br /> &#226;&#8364;&#732;&#226;&#8364;&#732;flip-flop&#226;&#8364;&#8482;&#226;&#8364;&#8482; circuit opposing kinase-cascade transcriptional        ground 278uC sonicated (VIRTIS Virsonic 475, setting <br /> signals (Figure 6).                                                           2.5, 0uC) 10-s bursts interspersed 2-min cooling periods. <br />     The concept &#226;&#8364;&#732;&#226;&#8364;&#732;genetic switch&#226;&#8364;&#8482;&#226;&#8364;&#8482; dauer formation            Kinase activity endogenous substrates assessed in <br /> new [1,86,87], demonstrated constitute                 cleared supernatants centrifugation (10 min, 11,000 g), <br /> bistable feedback loop [21,87]. Nevertheless, dual-level                    representing 20 mg protein 100 ml buffer containing 50- <br /> (kinase/transcriptional) feedback mechanism previously                mM Tris pH 7.5, 12.5-mM MgCl2 (for endogenous <br /> been proposed described. Any &#226;&#8364;&#732;&#226;&#8364;&#732;flip-flop&#226;&#8364;&#8482;&#226;&#8364;&#8482; circuitry allows           substrates) 8&#226;&#8364;&#8220;10 mCi c-32P-ATP (NEN). After 1 min 30uC, <br /> the organism simple binary choice response                        quenched samples electrophoresed 10% SDS-polyacryl- <br /> environment: early reproduction benign conditions,                   amide gels (Invitrogen), stained SYPRO Ruby <br /> postponed reproduction extended survival harsher                       (Invitrogen), dried vacuum. 32P b-emissions bands <br /> conditions. Mutational disruption IIS forces dauer formation,              migrating slower 25-kDa protein marker (Invitrogen), were <br /> irrespective environment, larvae temperature-                imaged quantified lane 6-h phosphor-screen <br /> sensitive mutations mature lower temperatures long-               exposure (Storm, Molecular Dynamics). Peptide arrays were <br /> lived adults. Recovery dauer state requires pro-                incubated above, 60 min 30uC, addition of <br /> reproductive-state kinases retain partial function,                   phosphatase inhibitors 1-mM cold ATP 32P- <br /> favorable signals (restoration food, absence stress                 ATP. Arrays stained Pro-Q Diamond (Invitrogen), <br /> crowding) reset switch reproductive mode;                 phosphorylation quantified fluorescence imaging <br /> requirement demonstrated impaired post-dauer                        (excitation/emission 550/580 nm) ScanArray 5000 (GSI <br /> recovery IIS-defective mutants [86]. Nonsense mutations                    Lumonics). <br /> truncating AGE-1 produce extreme <span id='am-165' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-166' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-239' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span>phenotype</span> forfeits <br /> this option, acquiring distinctive transcriptional profile            In vivo phosphoprotein detection quantitation <br /> and greatly enhanced survival. Details mechanism                       Total protein (20 mg), extracted strain above, was <br /> mechanisms, elimination PI3K activity blocks exit                 loaded NuPAGE 4&#226;&#8364;&#8220;12% gradient gels (Invitrogen) and <br /> from longevity mode promotes extreme longevity,                       electrophoresed 1 hour 200 V. Phosphoproteins quanti- <br /> remain elaborated. Features described report,             fied Pro-Q Diamond (Invitrogen) fluorescence, depends <br /> may contribute, include transcriptional silencing upstream                 linearly protein concentration (.1000-fold range). Protein load <br /> and collateral signaling components, accompanying loss                 assessed Coomassie Blue (BioRad) staining. Phosphorylated <br /> multiple kinase activities. Infertile mutants reveal new             (23.6, 45.0 kDa) unphosphorylated (14.4, 18.0, 62.6, <br /> strategies extend life limits imposed natural           116.2 kDa) protein standards (BioRad) furnished positive and <br /> selection, course requires reproduction. In view              negative controls. <br /> striking evolutionary conservation IIS pathway, the <br /> emerging parallels inter-pathway cross-talk nema-                  Transcript quantitation RT&#226;&#8364;&#8220;PCR <br /> todes mammals [16,19,47,68,69], mechanistic insights                     Expression selected genes assessed real-time <br /> afforded long-lived worms likely apply                 polymerase chain reaction initial round reverse <br /> insulin IGF-1 pathways mammals.                                        transcription. Total RNA purified strain (RNeasy, <br />                                                                               Qiagen), cDNAs reverse-transcribed (SuperScript III, Invitro- <br /> Materials Methods                                                         gen), followed RT-PCR (Opticon2, MJ Research, using SYBR <br />                                                                               Green, Roche). <br /> Strains <br />   Nematode strains, supplied Caenorhabditis Genetics                   Supporting Information <br /> Center (CGC, Minneapolis), derived laboratory from <br /> CGC strains, maintained 20uC 0.6% peptone NGM-                     Figure S1 In vitro kinase activity endogenous substrates is <br /> agar plates seeded E. coli strain OP50, described [88&#226;&#8364;&#8220;90].            reduced age-1(mg44) homozygotes, dauer larvae. <br />                                                                               Kinase activity assessed N2DRM day-6 adults eggs <br /> Stress resistance                                                             produce, eggs laid age-1(mg44) F1 adults, F1 adults days <br />    Assays survival presence 5-mM hydrogen peroxide               2.5 6.5 adulthood, age-1(mg44) F2 day-1 adults, daf- <br /> or 10-mM 4-hydroxynonenal modified Ayyadevara et                    16(mu86); age-1(mg44) adults, N2DRM dauer larvae. Kinase <br /> al. [91]. Wild-type (N2DRM) worms assayed day 3&#226;&#8364;&#8220;4                  activity sonicated lysates assessed described legend <br /> adulthood (,6 d post-hatch). For RNAi experiments, day-1 adults               Figure 1. Only single biological sample assessed each <br /> were washed S-buffer [92] transferred nutrient-agar                 group (hence error bars shown); technical replicates agreed <br /> plates seeded dsRNA-expressing E. coli [56]. After 3 days             620%, replicate experiments consistent with <br /> 20uC, 20 worms RNAi treatment transferred                   results shown. <br /> 24-well plates containing 300 ml S Buffer plus 5 mg/ml                     Found at: doi:10.1371/journal.pgen.1000452.s001 (1.42 MB TIF) <br /> cholesterol, supplemented, indicated, 5-mM H2O2                Figure S2   Phosphorylation vitro peptide arrays. Fluores- <br /> (freshly diluted 30% H2O2, Sigma) 10-mM 4-HNE                    cence images displayed peptide arrays (JPT Peptide <br /> (freshly obtained acid hydrolysis 4-HNE dimethylacetal                  Technologies GmbH, Berlin) shown printed duplicate blocks, <br /> which synthesized according [93]. Survival scored               stained Pro-Q Diamond (Invitrogen) 1-h incubation <br /> described [30,89].                                                            worm homogenates containing phosphatase protease <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                              13                           April 2009 | Volume 5 | Issue 4 | e1000452 <br />                                                                                                                  Hybrid Feedback Loop Governs Insulin/IGF-1 Response <br />  <br />  <br /> inhibitors 1-mM ATP. Equal protein concentrations                                    Found at: doi:10.1371/journal.pgen.1000452.s004 (0.08 MB <br /> each strain (N2DRM, age-1(mg44), daf-16(mu86); age-1(mg44))                               DOC) <br /> were incubated slides, 60 min 30uC. Note &#226;&#8364;&#732;&#226;&#8364;&#732;reversion&#226;&#8364;&#8482;&#226;&#8364;&#8482; <br />                                                                                               Table S2 Gene expression data, quantified real-time <br /> by daf-16(mu86) enhances kinases evident the <br />                                                                                               polymerase chain reaction, comparing age-1 mutant groups <br /> wild-type (N2DRM) sample. <br />                                                                                               daf-2(e1370) mutants. Worms age-1(mg44) F2 group are <br /> Found at: doi:10.1371/journal.pgen.1000452.s002 (0.77 MB TIF) <br />                                                                                               sterile adults; post-gravid adults. Significance of <br /> Figure S3    Adult age-1(mg44) F2 homozygotes deficient                                difference N2DRM (t-test): u, nominally significant at <br /> PI3Kcs protein. Western blots shown, denaturing (A)                              P,0.05; *, P,0.01; **, P,0.001; ***, P,1E24; ****, P,1E25, <br /> native (B) polyacrylamide gel electrophoresis. Electrochemilumi-                              *****, P,1E26. <br /> nescence (GE-Amersham &#226;&#8364;&#732;&#226;&#8364;&#732;ECL-plus&#226;&#8364;&#8482;&#226;&#8364;&#8482;) detected secondary anti-                                  Found at: doi:10.1371/journal.pgen.1000452.s005 (0.05 MB <br /> body (goat anti-rabbit IgG coupled horseradish peroxidase)                              DOC) <br /> rabbit monoclonal antibodies binding C-terminal region C. <br /> elegans phosphatidylinositol 3-kinase p110 catalytic subunit,                           Acknowledgments <br /> cytoplasmic b-actin loading control (Santa Cruz Biotech., <br /> Santa Cruz CA). Each lane contains 10 mg total protein.                                    We thank Ramani Alla Rajani Ayyadevara expert technical <br /> Found at: doi:10.1371/journal.pgen.1000452.s003 (0.25 MB TIF)                                 assistance, Eric Siegel statistical consultation, Drs. Piotr Zimniak <br />                                                                                               Joan McEwen helpful suggestions. <br /> Table S1 Peptide phosphorylation C. elegans kinases vitro. <br /> Mean6SD given ProQ Diamond fluorescence (in arbitrary <br />                                                                                               Author Contributions <br /> units) synthetic peptides spotted 2&#226;&#8364;&#8220;4 arrays, subtraction <br /> of mean background measured negative-control spots. The                                    Conceived designed experiments: C&#194;&#184; T SA RJSR. Performed the <br /> kinases phosphorylate peptides known;                                experiments: C <br />                                                                                                            &#194;&#184; T SA PB. Analyzed data: SA RJSR. Wrote paper: <br />                                                                                               RJSR. <br /> listed column 2 phosphorylate residues underlined, <br /> based consensus sites corresponding mammalian kinase. <br />  <br /> References <br />  1. Riddle DL, Swanson MM, Albert PS (1981) Interacting genes nematode                     20. Hertweck M, Gobel C, Baumeister R (2004) C. elegans SGK-1 critical <br />     dauer larva formation. Nature 290: 668&#226;&#8364;&#8220;671.                                                   component Akt/PKB kinase complex control stress response life <br />  2. Larsen PL, Albert PS, Riddle DL (1995) Genes regulate development                   span. Dev Cell 6: 577&#226;&#8364;&#8220;588. <br />     longevity Caenorhabditis elegans. Genetics 139: 1567&#226;&#8364;&#8220;1583.                         21. Murphy CT, McCarroll SA, Bargmann CI, Fraser A, Kamath RS, et al. (2003) <br />  3. Dorman JB, Albinder B, Shroyer T, Kenyon C (1995) The age-1 daf-2 genes                   Genes act downstream DAF-16 influence lifespan Caenorhabditis <br />     function common pathway control lifespan Caenorhabditis elegans.               elegans. Nature 424: 277&#226;&#8364;&#8220;283. <br />     Genetics 141: 1399&#226;&#8364;&#8220;1406.                                                                  22. McElwee J, Bubb K, Thomas JH (2003) Transcriptional outputs the <br />  4. Kenyon C (2005) The plasticity aging: insights long-lived mutants. Cell               Caenorhabditis elegans forkhead protein DAF-16. Aging Cell 2: 111&#226;&#8364;&#8220;121. <br />     120: 449&#226;&#8364;&#8220;460.                                                                             23. Lee SS, Kennedy S, Tolonen AC, Ruvkun G (2003) DAF-16 target genes that <br />  5. Friedman DB, Johnson TE (1988) A mutation age-1 gene Caenorhabditis                 control C. elegans life-span metabolism. Science 300: 644&#226;&#8364;&#8220;647. <br />     elegans lengthens life reduces hermaphrodite fertility. Genetics 118: 75&#226;&#8364;&#8220;86.          24. Murphy CT (2006) The search DAF-16/FOXO transcriptional targets: <br />  6. Kenyon C, Chang J, Gensch E, Rudner A, Tabtiang R (1993) A C. elegans                         approaches discoveries. Exp Gerontol 41: 910&#226;&#8364;&#8220;921. <br />     mutant lives twice long wild type. Nature 366: 461&#226;&#8364;&#8220;464.                        25. Gems D, Sutton AJ, Sundermeyer M, Albert PS, King K, et al. (1998) Two <br />  7. Lin K, Dorman JB, Rodan A, Kenyon C (1997) daf-16: An HNF-3/forkhead                          pleiotropic classes daf-2 mutation affect larval arrest, adult behavior, <br />     family member function double life-span Caenorhabditis elegans.            reproduction longevity Caenorhabditis elegans. Genetics 150: 129&#226;&#8364;&#8220;155. <br />     Science 278: 1319&#226;&#8364;&#8220;1322.                                                                   26. Klass MR (1983) A method isolation longevity mutants the <br />  8. Tissenbaum HA, Ruvkun G (1998) An insulin-like signaling pathway affects                      nematode Caenorhabditis elegans initial results. Mech Ageing Dev 22: 279&#226;&#8364;&#8220;286. <br />     longevity reproduction Caenorhabditis elegans. Genetics 148: 703&#226;&#8364;&#8220;717.         27. Friedman DB, Johnson TE (1988) Three mutants extend mean and <br />  9. Clancy DJ, Gems D, Harshman LG, Oldham S, Stocker H, et al. (2001)                            maximum life span nematode, Caenorhabditis elegans, define age-1 gene. <br />     Extension Life-Span Loss CHICO, Drosophila Insulin Receptor                        J Gerontol 43: B102&#226;&#8364;&#8220;B109. <br />     Substrate Protein. Science 292: 104&#226;&#8364;&#8220;106.                                                  28. Johnson TE (1990) Increased life-span age-1 mutants Caenorhabditis elegans <br /> 10. Tatar M, Kopelman A, Epstein D, Tu MP, Yin CM, et al. (2001) A Mutant                         lower Gompertz rate aging. Science 249: 908&#226;&#8364;&#8220;912. <br />     Drosophila Insulin Receptor Homolog That Extends Life-Span Impairs <br />                                                                                               29. Morris JZ, Tissenbaum HA, Ruvkun G (1996) A phosphatidylinositol-3-OH <br />     Neuroendocrine Function. Science 292: 107&#226;&#8364;&#8220;110. <br />                                                                                                   kinase family member regulating longevity diapause Caenorhabditis elegans. <br /> 11. Bartke AJ, Wright JC, Mattison JA, Ingram DK, Miller RA, et al. (2001) <br />                                                                                                   Nature 382: 536&#226;&#8364;&#8220;539. <br />     Extending lifespan long-lived mice. Nature 414: 412. <br />                                                                                               30. Ayyadevara S, Alla R, Thaden JJ, Shmookler Reis RJ (2008) Remarkable <br /> 12. Pierce SB, Costa M, Wisotzkey R, Devadhar S, Homburger SA, et al. (2001) <br />                                                                                                   longevity stress resistance nematode PI3K-null mutants. Aging Cell 7: <br />     Regulation DAF-2 receptor signaling human insulin ins-1, member <br />                                                                                                   13&#226;&#8364;&#8220;22. <br />     unusually large diverse C. elegans insulin gene family. Genes Dev 15: <br />     672&#226;&#8364;&#8220;686.                                                                                  31. Hawkins PT, Anderson KE, Davidson K, Stephens LR (2006) Signalling <br />                                                                                                   Class I PI3Ks mammalian cells. Biochem Soc Trans 34: 647&#226;&#8364;&#8220;662. <br /> 13. Paradis S, Ailion M, Toker A, Thomas JH, Ruvkun G (1999) A PDK1 homolog <br />     necessary sufficient transduce AGE-1 PI3 kinase signals regulate           32. Vellai T, McCulloch D, Gems D, Kovacs AL (2006) Effects sex insulin/ <br />     diapause Caenorhabditis elegans. Genes Dev 13: 1438&#226;&#8364;&#8220;1452.                                  insulin-like growth factor-1 signaling performance associative learning <br /> 14. Paradis S, Ruvkun G (1998) Caenorhabditis elegans Akt/PKB transduces insulin                  paradigm Caenorhabditis elegans. Genetics 174: 309&#226;&#8364;&#8220;316. <br />     receptor-like signals AGE-1 PI3 kinase DAF-16 transcription factor.           33. Weinkove D, Halstead JR, Gems D, Divecha N (2006) Long-term starvation and <br />     Genes Dev 12: 2488&#226;&#8364;&#8220;2498.                                                                      ageing induce AGE-1/PI 3-kinase-dependent translocation DAF-16/FOXO <br /> 15. Hemmings BA (1997) PtdIns(3,4,5)P3 gets message across. Science 277: 534.                 cytoplasm. BMC Biol 4: 1. <br /> 16. Singh SS, Chauhan A, Brockerhoff H, Chauhan VP (1993) Activation protein               34. Wolff S, Ma H, Burch D, Maciel GA, Hunter T, et al. (2006) SMK-1, an <br />     kinase C phosphatidylinositol 3,4,5-trisphosphate. Biochem Biophys Res                     essential regulator DAF-16-mediated longevity. Cell 124: 1039&#226;&#8364;&#8220;1053. <br />     Commun 195: 104&#226;&#8364;&#8220;112.                                                                      35. Tissenbaum HA, Guarente L (2001) Increased dosage sir-2 gene extends <br /> 17. Stokoe D, Stephens LR, Copeland T, Gaffney PRJ, Reese CB, et al. (1997) Dual                  lifespan Caenorhabditis elegans. Nature 410: 154&#226;&#8364;&#8220;155. <br />     role phosphatidylinositol-3,4,5-trisphosphate activation Protein             36. Wang Y, Oh SW, Deplancke B, Luo J, Walhout AJ, et al. (2006) C. elegans 14-3-3 <br />     Kinase B. Science 277: 567&#226;&#8364;&#8220;570.                                                               proteins regulate life span interact SIR-2.1 DAF-16/FOXO. Mech <br /> 18. Shtilbans V, Wu M, Burstein DE (2008) Current overview role Akt                  Ageing Dev 127: 741&#226;&#8364;&#8220;747. <br />     cancer studies applied immunohistochemistry. Ann Diagn Pathol 12:                     37. Narbonne P, Roy R (2008) Inhibition germline proliferation C. elegans <br />     153&#226;&#8364;&#8220;160.                                                                                      dauer development requires PTEN, LKB1 AMPK signalling. Development <br /> 19. Gami MS, Iser WB, Hanselman KB, Wolkow CA (2006) Activated AKT/PKB                            123: 611&#226;&#8364;&#8220;619. <br />     signaling C. elegans uncouples temporally distinct outputs DAF-2/insulin-           38. Mihaylova VT, Borland CZ, Manjarrez L, Stern MJ, Sun H (1999) The PTEN <br />     like signaling. BMC Dev Biol 6: 45.                                                           tumor suppressor homolog Caenorhabditis elegans regulates longevity dauer <br />  <br />  <br />  <br />         PLoS Genetics | www.plosgenetics.org                                             14                                April 2009 | Volume 5 | Issue 4 | e1000452 <br />                                                                                                                         Hybrid Feedback Loop Governs Insulin/IGF-1 Response <br />  <br />  <br />       formation insulin receptor-like signaling pathway. Proc Natl Acad Sci U S A             66. Ayyadevara S, Engle MR, Singh SP, Dandapat A, Lichti CF, et al. (2005) <br />       96: 7427&#226;&#8364;&#8220;7432.                                                                                    Lifespan stress resistance Caenorhabditis elegans increased expression <br /> 39.   Kawano T, Ito Y, Ishiguro M, Takuwa K, Nakajima T, et al. (2000) Molecular                        glutathione transferases capable metabolizing lipid peroxidation <br />       cloning characterization new insulin/IGF-like peptide nematode                    product 4-hydroxynonenal. Aging Cell 4: 257&#226;&#8364;&#8220;271. <br />       Caenorhabditis elegans. Biochem Biophys Res Commun 273: 431&#226;&#8364;&#8220;436.                              67. McElwee JJ, Schuster E, Blanc E, Piper MD, Thomas JH, et al. (2007) <br /> 40.   Chakravarty K, Cassuto H, Reshef L, Hanson RW (2005) Factors control                         Evolutionary conservation regulated longevity assurance mechanisms. <br />       tissue-specific transcription gene phosphoenolpyruvate carbox-                     Genome Biol 8: R132. <br />       ykinase-C. Crit Rev Biochem Mol Biol 40: 129&#226;&#8364;&#8220;154.                                             68. Kondo M, Yanase S, Ishii T, Hartman PS, Matsumoto K, et al. (2005) The p38 <br /> 41.   Wolkow CA, Munoz MJ, Riddle DL, Ruvkun G (2002) Insulin receptor                                  signal transduction pathway participates oxidative stress-mediated <br />       substrate p55 orthologous adaptor proteins function Caenorhabditis                     translocation DAF-16 Caenorhabditis elegans nuclei. Mech Ageing Dev 126: <br />       elegans daf-2/insulin-like signaling pathway. J Biol Chem 277: 49591&#226;&#8364;&#8220;49597.                       642&#226;&#8364;&#8220;647. <br /> 42.   Denley A, Carroll JM, Brierley GV, Cosgrove L, Wallace J, et al. (2007)                       69. Matsumoto M, Han S, Kitamura T, Accili D (2006) Dual role transcription <br />       Differential activation insulin receptor substrates 1 2 insulin-like                    factor FoxO1 controlling hepatic insulin sensitivity lipid metabolism. <br />       growth factor-activated insulin receptors. Mol Cell Biol 27: 3569&#226;&#8364;&#8220;3577.                           J Clin Invest 116: 2464&#226;&#8364;&#8220;2472. <br /> 43.   Greer EL, Dowlatshahi D, Banko MR, Villen J, Hoang K, et al. (2007) An                        70. An JH, Blackwell TK (2003) SKN-1 links C. elegans mesendodermal specification <br />       AMPK-FOXO pathway mediates longevity induced novel method                                 conserved oxidative stress response. Genes Dev 17: 1882&#226;&#8364;&#8220;1893. <br />       dietary restriction C. elegans. Curr Biol 17: 1646&#226;&#8364;&#8220;1656.                                   71. Tullet JM, Hertweck M, An JH, Baker J, Hwang JY, et al. (2008) Direct <br /> 44.   Apfeld J, O&#226;&#8364;&#8482;Connor G, McDonagh T, DiStefano PS, Curtis R (2004) The AMP-                          inhibition longevity-promoting factor SKN-1 insulin-like signaling C. <br />       activated protein kinase AAK-2 links energy levels insulin-like signals                    elegans. Cell 132: 1025&#226;&#8364;&#8220;1038. <br />       lifespan C. elegans. Genes Dev 18: 3004&#226;&#8364;&#8220;3009.                                              72. Bishop NA, Guarente L (2007) Two neurons mediate diet-restriction-induced <br /> 45.   Curtis R, O&#226;&#8364;&#8482;Connor G, DiStefano PS (2006) Aging networks Caenorhabditis                        longevity C. elegans. Nature 447: 545&#226;&#8364;&#8220;549. <br />       elegans: AMP-activated protein kinase (aak-2) links multiple aging                        73. Roggo L, Bernard V, Kovacs AL, Rose AM, Savoy F, et al. (2002) Membrane <br />       metabolism pathways. Aging Cell 5: 119&#226;&#8364;&#8220;126.                                                       transport Caenorhabditis elegans: essential role VPS34 nuclear <br /> 46.   Shaw WM, Luo S, Landis J, Ashraf J, Murphy CT (2007) The C. elegans TGF-                          membrane. EMBO J 21: 1673&#226;&#8364;&#8220;1683. <br />       beta Dauer pathway regulates longevity insulin signaling. Curr Biol 17:                   74. Nanji M, Hopper N, Gems D (2005) LET-60 RAS modulates effects insulin/ <br />       1635&#226;&#8364;&#8220;1645.                                                                                        IGF-1 signaling development aging Caenorhabditis elegans. Aging Cell 4: <br /> 47.   Troemel ER, Chu SW, Reinke V, Lee SS, Ausubel FM, et al. (2006) p38 MAPK                          235&#226;&#8364;&#8220;245. <br />       regulates expression immune response genes contributes longevity C.              75. Panowski SH, Wolff S, Aguilaniu H, Durieux J, Dillin A (2007) PHA-4/Foxa <br />       elegans. PLoS Genet 2: e183. doi:10.1371/journal.pgen.0020183.                                    mediates diet-restriction-induced longevity C. elegans. Nature 447: 550&#226;&#8364;&#8220;555. <br /> 48.   Oh SW, Mukhopadhyay A, Svrzikapa N, Jiang F, Davis RJ, et al. (2005) JNK                      76. Samuelson A, Carr C, Ruvkun G (2007) Gene activities mediate increased <br />       regulates lifespan Caenorhabditis elegans modulating nuclear translocation               life span C. elegans insulin-like signaling mutants. Genes Dev 21: 2976&#226;&#8364;&#8220;2994. <br />       forkhead transcription factor/DAF-16. Proc Natl Acad Sci U S A 102:                           77. Berdichevsky A, Viswanathan M, Horvitz HR, Guarente L (2006) C. elegans SIR- <br />       4494&#226;&#8364;&#8220;4499.                                                                                        2.1 interacts 14-3-3 proteins activate DAF-16 extend life span. Cell <br /> 49.   Helfand SL, Blake KJ, Rogina B, Stracks MD, Centurion A, et al. (1995)                            125: 1165&#226;&#8364;&#8220;1177. <br />       Temporal patterns gene expression antenna adult Drosophila                   78. Wang Y, Tissenbaum HA (2006) Overlapping distinct functions a <br />       melanogaster. Genetics 140: 549&#226;&#8364;&#8220;555.                                                              Caenorhabditis elegans SIR2 DAF-16/FOXO. Mech Ageing Dev 127: 48&#226;&#8364;&#8220;56. <br />                                                                                                     79. Klass M, Hirsh D (1976) Non-ageing developmental variant Caenorhabditis <br /> 50.   Yu H, Larsen PL (2001) DAF-16-dependent independent expression targets <br />                                                                                                         elegans. 260, 523&#226;&#8364;&#8220;525. Nature 260: 523&#226;&#8364;&#8220;525. <br />       DAF-2 insulin receptor-like pathway Caenorhabditis elegans include FKBPs. <br />                                                                                                     80. Cherkasova V, Ayyadevara S, Egilmez N, Shmookler Reis RJ (2000) Diverse <br />       J Mol Biol 314: 1017&#226;&#8364;&#8220;1028. <br />                                                                                                         Caenorhabditis elegans genes upregulated dauer larvae show <br /> 51.   McElwee JJ, Schuster E, Blanc E, Thomas JH, Gems D (2004) Shared <br />                                                                                                         elevated transcript levels long-lived, aged, starved adults. J Mol Biol 300: <br />       transcriptional signature Caenorhabditis elegans Dauer larvae long-lived daf-2 <br />                                                                                                         433&#226;&#8364;&#8220;448. <br />       mutants implicates detoxification longevity assurance. J Biol Chem <br />                                                                                                     81. Kenyon C, Murphy CT (2006) Enrichment regulatory motifs upstream of <br />       279: 44533&#226;&#8364;&#8220;44543. <br />                                                                                                         predicted DAF-16 targets. Nat Genet 38: 397&#226;&#8364;&#8220;398. <br /> 52.   Oh SW, Mukhopadhyay A, Dixit BL, Raha T, Green MR, et al. (2006) <br />                                                                                                     82. Lamitina ST, Strange K (2005) Transcriptional targets DAF-16 insulin <br />       Identification direct DAF-16 targets controlling longevity, metabolism and <br />                                                                                                         signaling pathway protect C. elegans extreme hypertonic stress. Am J Physiol <br />       diapause chromatin immunoprecipitation. Nat Genet 38: 251&#226;&#8364;&#8220;257. <br />                                                              <br /> </body></html>