<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Copyright &#195;&#8220; 2008 Genetics Society America <br /> DOI: 10.1534/genetics.107.075275 <br />  <br />  <br />  <br />             cin-4, Gene With Homology Topoisomerase II, Is Required <br />               Centromere Resolution Cohesin Removal <span id='am-2' about='protege:FROM' typeof='owl:Thing'>From</span> Sister <br />                              Kinetochores During Mitosis <br />  <br />                                             Gerald Stanvitch Landon L. Moore1 <br />                   Department Genetics Genomics, Boston University School Medicine, Boston, Massachusetts 02118 <br />                                                        Manuscript received May 5, 2007 <br />                                                   Accepted publication November 9, 2007 <br />  <br />  <br />                                                                 <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> <br />                  The back-<span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span>-back geometry sister kinetochores essential preventing loss damage chromosomes <br />               mitosis. Kinetochore orientation generated process resolving kinetochores the <br />               centromere (centromere resolution) prior spindle interactions. Because genes required for <br />               centromere resolution known, used Caenorhabditis elegans screen conditional mutants defective in <br />               orienting sister kinetochores mitosis. C. elegans ideal screens chromosomes are <br />               holocentric. Here identified essential gene, cin-4, required centromere resolution removal of <br />               cohesin sites near sister kinetochores mitosis. Given compromised cohesin function restores <br />               centromere resolution absence cin-4, CIN-4 likely acts remove cohesin CENP-A chromatin <br />               enabling centromere resolution. CIN-4 high amino acid identity catalytic domain topoisomerase <br />               II, suggesting partial gene duplication C. elegans topoisomerase II gene, top-2. Similar CIN-4, TOP-2 <br />               required centromere resolution; however, loss TOP-2 phenotypically distinct loss of <br />               CIN-4, suggesting CIN-4 TOP-2 topoisomerase II isoforms perform separate essential <br />               functions centromere structure function. <br />  <br />  <br />  <br />  <br /> A     failure accurately segregate mitotic chromo- <br />       somes results genome instability, severe conse- <br /> quences development, cancer progression <br />                                                                           resolution) early step positioning sister kinetochores <br />                                                                           back. <br />                                                                              The mechanism centromere resolution known, <br /> (Lengauer et al. 1998). Centromere structure par-                      independent kinetochore microtubule inter- <br /> ticularly important accurate segregation                       actions (Brenner et al. 1981; He Brinkley 1996; <br /> chromosomes. At centromere, sister kinetochores                       Moore Roth 2001). Both centromere resolution <br /> assemble face opposite directions (a                    kinetochore assembly depend centromere <br /> back-to-back orientation). This back-to-back geometry                     protein CENP-C, mechanisms not <br /> aids proper chromosome segregation restricting at-                     understood (Moore Roth 2001). However, is <br /> tachment kinetochore centrosome,                    known requirement CENP-C centromere <br /> preventing incorrect attachments (Rieder Salmon                       resolution bypassed genetic disruption cohesin <br /> 1998; Cimini et al. 2001). The back-to-back geometry                   function (Moore et al. 2005). Cohesin complex of <br /> sister kinetochores thought involved                    proteins mediate cohesion sister chro- <br /> correction inappropriate spindle&#226;&#8364;&#8220;kinetochore in-                       matids (Nasmyth 2001). The cohesin complex consists <br /> teractions anaphase progression (Zhang                       structural maintenance chromosomes (SMC) <br /> Walczak 2006; Baumann et al. 2007). The establish-                        proteins, SMC1 SMC3, non-SMC proteins, <br /> ment sister kinetochores back-to-back begins G2/                    SCC3 SCC1/Mcd1/Rad21 (Losada et al. 1998; <br /> early prophase centromere proteins initially                     Toth et al. 1999). Cohesin forms ring structure that <br /> detected single &#226;&#8364;&#732;&#226;&#8364;&#732;dots&#226;&#8364;&#8482;&#226;&#8364;&#8482; later resolve paired                 capable encircling sister chromatids, may <br /> dots (Brenner et al. 1981). The centromere protein,                       form basis chromosome cohesion (Gruber et al. <br /> CENP-A, histone H3 variant forms inner-                     2003; Huang Moazed 2006). Cohesin abun- <br /> most layer kinetochore required                         dantly present nucleus interphase is <br /> recruiting kinetochore proteins (Howman et al.                       sequentially removed chromatin mitosis. <br /> 2000; Moore Roth 2001; Oegema et al. 2001;                            In vertebrates, bulk cohesin removed during <br /> Blower et al. 2002). Thus, separation resolution                   prophase (Losada et al. 2002; Sumara et al. 2002). This <br /> of CENP-A chromatin (also referred centromere                       prophase removal pathway requires wings apart-like <br />                                                                           (WAPL) protein leads separation chro- <br />   1 <br />    Corresponding author: Boston University School Medicine, 715        mosome arms (Gandhi et al. 2006; Kueng et al. 2006). <br /> Albany St., E642, Boston, MA 02118. E-mail: lmoore@bu.edu                 Centromere resolution occurs G2/early prophase, <br />  <br /> Genetics 178: 83&#226;&#8364;&#8220;97 ( January 2008) <br />  84                                               G. Stanvitch L. L. Moore <br />  <br /> when cohesin abundantly present, suggesting            NGM plates Escherichia coli OP50 described Brenner <br /> novel mechanism removes excludes cohesin               (1974). The following marker mutations chromosomal <br /> from CENP-A chromatin. However, loss CENP-C does              rearrangements, listed linkage group, used genetic <br />                                                                  mapping: LGI&#226;&#8364;&#8221;dpy-5(e61); LGII&#226;&#8364;&#8221;rol-6(e187), unc-4(e120), mnDf99, <br /> not lead retention cohesin, suggesting          mnDf88, mnDf105, mnDf30, mnC1, lin-26(mc15), unc-104(e1625); <br /> factors involved resolution sister kineto-         LGIII&#226;&#8364;&#8221;unc-32(e189); LGIV&#226;&#8364;&#8221;unc-5(e53); LGV&#226;&#8364;&#8221;dpy-11(e224); <br /> chores (Moore et al. 2005).                                      LGX&#226;&#8364;&#8221;lon-2(e678). All temperature-sensitive stocks were <br />    To identify potential factors required centro-      maintained permissive temperature 15&#194;&#176; shifted a <br /> mere resolution, performed genetic screen using         restrictive temperature 25&#194;&#176; 6 hr prior analysis. The <br />                                                                  smc-1(e879) mutant previously characterized (Chan et al. <br /> nematode Caenorhabditis elegans. C. elegans ideal iden-   2003). The double mutants cin-4(mr127);smc-1(e879) cin- <br /> tifying mutants centromere organization defects,            4(mr127);unc-119(ed3) produced using standard genetic <br /> because chromosomes holocentric               cross techniques. <br /> early embryo easily studied immunofluorescence                  Screen chromosome instability mutants: The collection <br /> (Albertson Thomson 1982; Moore et al. 1999). Here            254 temperature-sensitive embryonic lethal mutants pre- <br />                                                                  viously described (Stear Roth 2002) screened for <br /> we identified gene, cin-4 (chromosome instabil-           chromosome instability (cin) phenotypes using 49,6-diami- <br /> ity) 89% identical central catalytic domain       dino-2-phenylindole (DAPI) stain DNA. We identified 22 <br /> of topoisomerase II. DNA topoisomerases catalyze             mutants nonpermissive temperature showed an <br /> removal topological complexities breaking, manip-          aberrant distribution DNA embryo. This aberrant <br /> ulating, rejoining DNA strands. Topological com-             DNA distribution characteristic defects chromosome <br />                                                                  segregation (Moore et al. 1999), suggesting 22 <br /> plexities, supercoils catenanes, arise        mutants defective aspect chromosome segre- <br /> DNA replication and, removed, chromo-            gation following shift permissive nonper- <br /> some segregation inhibited. All topoisomerases            missive temperature (15&#194;&#176; vs. 25&#194;&#176;). Because interested <br /> classified basis number DNA strands        genes required mitotic chromosome structure and <br /> they cleave. Odd-numbered topoisomerases,                function, mutants embryos displaying defects only <br />                                                                  meiosis cytokinesis considered further. We also <br /> topoisomerase I III, cleave single strand duplex        stained embryos mutant following shift the <br /> DNA, generating gap DNA                  nonpermissive temperature antibody tubulin. <br /> strand passed. In contrast, even-numbered             Mutants defects spindle organization not <br /> topoisomerases, topoisomerase II, cleave            considered further. We linkage mapped out-crossed three <br /> strands DNA duplex, generating double-strand              times remaining mutants reexamining them. <br />                                                                  Mutants linkage group tested for <br /> break; DNA topology altered second DNA                 noncomplementation. During out-crossing, mutants <br /> duplex passed double-strand break.                longer demonstrated embryonic lethality reex- <br /> Thus, topoisomerase II largely responsible deca-          amination longer showed chromosome instability. From <br /> tenating separating entangled DNAs. Topoisomer-              screen, identified chromosome instability genes, <br /> ase II abundantly present centromere            named chromosome instability (cin-1&#226;&#8364;&#8220;cin-5). <br />                                                                     Genetic mapping cin-4(mr127): A combination three- <br /> mitosis, suggesting role centromere function                point mapping deletion mapping used map the <br /> (Rattner et al. 1996). At anaphase, topoisomerase II             cin-4(mr127) <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span>mutation</span>. Briefly, heterozygous cin-4(mr127/1) <br /> may act cleave DNA connections sister                 males crossed hermaphrodites MT3751 &#194;&#189;dpy-5(e61)I, <br /> kinetochores tension-sensitive manner (Baumann              rol-6(e187)II, unc-32(e189)III  MT464 &#194;&#189;unc-5(e53)V, dpy- <br /> et al. 2007). These DNA connections accentuated           11(e224)V, lon-2(e678)X . F2 progeny homozygous each <br />                                                                  morphological class scored temperature-sensitive em- <br /> loss cohesins, suggesting functional interaction            bryonic lethality. Seven 36 dpy-5(e61), 0 35 rol-6(e187), 5 of <br /> between cohesin topoisomerase II centromere               35 unc-32(e189), 4 34 unc-5(e53), 5 26 dpy-11(e224), and <br /> structure cell cycle progression. In addition             6 34 lon-2(e678) cin-4(mr127), indicating the <br /> decatenation, topoisomerase II function includes reg-            cin-4 gene linkage group II. Three-point mapping was <br /> ulating gene expression centromere cohesion                  used map cin-4(mr127); heterozygote, the <br />                                                                  number total recombinant progeny selected a <br /> (Bachant et al. 2002; Ju et al. 2006; Lyu et al. 2006).          crossover event given interval parentheses between <br />    In report, demonstrate cin-4 C.          markers flanking interval&#226;&#8364;&#8221;lin-26; unc-4/cin-4: cin-4 <br /> elegans topoisomerase II gene, top-2, required           (0/10) lin-26 (10/10) unc-4 unc-104; rol-6/cin-4: unc-104 <br /> centromere resolution. The cin-4 gene appears            (4/21) cin-4 (17/21) rol-6. <br /> C. elegans unique partial gene duplication top-2. In         To define interval containing cin-4, determined <br />                                                                  known deletions interval complement cin- <br /> cin-4 mutant, top-2(RNAi) animals, cohesin is <br />                                                                  4(mr127) nonpermissive temperature. We crossed male <br /> present mitotic chromosomes, suggesting            cin-4(mr127) hermaphrodites containing deletions the <br /> two genes represent topoisomerase II isoforms               target interval scored heterozygous progeny for <br /> distinct roles centromere structure function.             complementation. The cin-4(mr127) cross mnDf1 mnDf88 <br />                                                                  produced 10/10 14/14 viable F1 progeny, respectively, <br />                                                                  indicating deletions complemented cin-4(mr127). The <br />               MATERIALS AND METHODS                              cin-4(mr127) mnDf30 cross did complement 14 15 F1 <br />                                                                  progeny sterile embryonic lethal. <br />    C. elegans strains culture conditions: The wild-type C.      Transgenic rescue cin-4(mr127): To perform transgenic <br /> elegans Bristol strain (N2) used. Strains cultured   rescue experiments, cloned 4.8-kb Bam HI/Xba I <br />                                               CIN-4 Removal Cohesin During Mitosis                                               85 <br />  <br /> fragment cosmid ZK1127 vector pJN254,             generated vitro using templates derived PCR from <br /> which contains wild-type unc-119 gene. The plasmid              plasmids isolated genome RNAi library. The T7 pro- <br /> pGS4.8ZK1127 contained entire ZK1127.7 gene             moter sequence added oligos gene generate <br />  1 kb upstream downstream sequences necessary               templates vitro synthesis previously described (Moore <br /> gene expression regulation. The plasmid pGS4.8ZK1127            et al. 1999). Oligonucleotides used PCR 59-GGACGC <br /> was bombarded cin-4(mr127)II;unc-119(ed3)III            GACAAATATGGAGT 59-TGTTCGTGGACCATCCAGTA <br /> strain previously explained Praitis et al. (2001). Trans-     ZK1127.7/cin-4; 59-GCGTGAAGTCAAAGTAGCTCAAT and <br /> genic lines identified rescue unc-119 <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span>phenotype</span>      59-CATCTTGATCAAAACATCGACAA K12D12.1/top2; and <br /> and shifted nonpermissive temperature test            59-GGCGCCAAATATGTGAAAGT 59-TGGGTATGGATC <br /> rescue cin-4(mr127) embryonic lethality phenotype.           TCGTGTCA R08C7.10/wapl-1. <br />    DNA sequencing: All sequencing performed using ABI              Indirect immunofluorescence microscopy: The cin-4(mr127) <br /> Prism 3730xl DNA sequencing (SeqWright). While sequenc-             mutant exhibited decrease brood size extended <br /> ing N2 wild-type cin-4 locus, identified difference        incubation nonpermissive temperature culminating in <br /> between DNA sequence DNA sequence reported              sterility (data shown). To optimize embryo production for <br /> for C. elegans genome (Wormbase release WS182). We              microscopy, incubation adult cin-4(mr127) <br /> found insertion G nucleotide 7040775 chro-         hermaphrodites 6 16 hr nonpermissive tem- <br /> mosome II, giving sequence 5-ctcgcgagaaagctgagtctaag            perature produced sufficient numbers embryos displaying <br /> gcgaac G cgacgcgatggtatgcgatgatatgtcag. The G insertion             Cin phenotype analysis. Embryos immunofluores- <br /> changes predicted reading frame CIN-4 include             cence fixed stained using N, N-dimethylforma- <br /> additional amino acids C terminus. To identify           mide protocol previously described (Moore et al. 1999). <br /> mr127 lesion, sequenced cin-4 locus cin-4(mr127)          Primary antibodies used work anti-HCP-3/ <br /> animals. Because DNA identity cin-4 top-2,       CENP-A (Buchwitz et al. 1999), anti-HCP-4/CENP-C (Moore <br /> we used PCR amplify cin-4 locus independently         Roth 2001), anti-SCC1/COH-2 (Pasierbek et al. <br /> top-2 locus topoisomerase II pseudogenes C.        2001); monoclonal antibody, mAb414 (Davis Blobel <br /> elegans. We performed PCR oligos 5-CGTGCATGCG                  1986), directed nuclear pore proteins; anti- <br /> TATTCCAGGG 5-CGAAATGAGCTGTGACGTCATAG                         b-tubulin antibody, YL 1/2 (Amersham Life Science, Little <br /> obtain 3.3-kb fragment corresponding cin-4 locus. The      Chalfont, Buckinghamshire, UK). Staining detected using <br /> G-to-A transition mutation changes glutamate 304 new         AlexaFluor 594- 488-conjugated secondary antibodies <br /> predicted amino acid sequence glycine.                           (Molecular Probes, Eugene, OR). The monoclonal antibody <br />    RACE cin-4 mRNA: Messenger RNA isolated              mAb414 used determine cell-cycle stage previously <br /> animals using RNeasy Oligotex mRNA isolation kit            described (Moore Roth 2001). Slides visualized <br /> (QIAGEN, Valencia, CA). Reverse transcription using Accu-           Zeiss AxioPlan 2E microscope equipped a <br /> Script (Stratagene, La Jolla, CA) random hexamers           Sensys CCD camera (Photometric) three-dimensional <br /> used generate cDNA. For 59-RACE, oligos SL1 5-               multiple wavelength fluorescence microscopy using Deltavi- <br /> GGTTTAATTACCCAAG, 5-CAGAATCAATGTACAC                       sion microscope (Applied Precision). Deltavision images were <br /> used PCR cDNA. For 39-RACE, oligos dT 5-           obtained specified wavelengths using 0.2-mm optical sec- <br /> TTTTTTTTTTTTTTTTTT 5-ATGTCTGAATTGACAAG                          tions. These sections deconvolved using Softworks <br /> CAGGA used. Longer cin-4 cDNA product obtained             software (Applied Precision) examined single <br /> using oligos 5-GAAGATGCGAACGACGCTG 5-AAGCT                      sections projections entire stack optical sections. <br /> TTCGCATACCATCGCGT PCR cDNA. The PCR pro-                      Real-time quantitative PCR assays: Real-time quantitative <br /> ducts sequenced compared genomic sequence           PCR assay (RT&#226;&#8364;&#8220;qPCR) used assay RNAi efficiency well <br /> to determine intron&#226;&#8364;&#8220;exon structure. The cin-4 sequence          potential secondary targets RNAi. Messenger RNA was <br /> be GenBank (EU191083).                                     isolated animals using RNeasy Oligotex mRNA <br />    Bioinformatics: The C. elegans genome searched using         isolation kit (QIAGEN). RNA quantity determined using a <br /> the translated basic local alignment search tool (TBLAST)           Spectrophotometer (NanoDrop). RT&#226;&#8364;&#8220;qPCR performed <br /> tool (NCBI) amino acid sequence human top-               using Brilliant II SYBR Green kit (Stratagene) MX- <br /> oisomerase II a. The following loci identified having       4000 real-time PCR machine (Stratagene). Reactions were <br /> significant identity human topoisomerase II a: K12D12.1,         normalized nonvariable control gene, F22B2.13 <br /> ZK1127.7, Y46H3C.4, F31E8.6, F32A11.5, K08E5.1, R05D3.1.        (Link et al. 2003). Oligos RT&#226;&#8364;&#8220;qPCR following: <br /> Predicted amino acid sequences aligned using Clustal       cin-4, 5-cgtggagttcgatgaagg 5-aactaaccgctttaatcc; top- <br /> W package LaserGene sequence analysis software               2, 5-ccatcaataagtaaggag 5-aagatgttagaagagttcc; R05D3.1, <br /> package (DNASTAR). F31E8.6, F32A11.5, K08E5.1 amino             5-gcgctcgctccaacaactc 5-ggttccacaaccattccc; F22B2.13, <br /> acid sequences contained frequent nonsense codons           5-catttgtggagaatgcc 5-tggaatagcgatttgacg. The DDCT a <br /> likely pseudogenes. R05D3.1 amino acid sequence 43%             reference gene method used calculate fold decrease in <br /> identical human topoisomerase II 64% identical C.       transcript (Real-Time PCR Applications Guide, Stratagene). <br /> elegans top-2/K12D12.1 catalytic domains, suggesting   Results obtained independent experi- <br /> topoisomerase II-related gene. The human WAPL amino acid            ments experiment performed triplicate. <br /> sequence used search C. elegans genome. Three <br /> predicted alternative products R08C7.10 gene (e-values <br /> of 3e&#226;&#8364;&#8220;56) identified likely C. elegans WAPL gene.                                  RESULTS <br /> Clustal W alignment R08C7.10a human WAPL showed <br /> 24% identity two. This identity present         Chromosomal instability mutant cin-4(mr127) is <br /> in C-terminal portion protein.                          required centromere resolution: To better under- <br />    <span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span>RNA interference</span> assay: RNA-mediated interference (RNAi)         stand mechanisms involved maintaining chromo- <br /> was performed established feeding soaking procedures <br /> (Kamath et al. 2001). Bacterial strains RNAi feeding       stability, screened 254 temperature-sensitive <br /> obtained whole-genome RNAi library (Kamath               embryonic lethal mutants identified chromosome <br /> Ahringer 2003). For RNAi soaking, double-strand RNA            instability mutants (cin-1&#226;&#8364;&#8220;cin-5), demonstrated defects <br />  86                                               G. Stanvitch L. L. Moore <br />  <br />                                                                                    Figure 1.&#226;&#8364;&#8221;Chromosome segregation defec- <br />                                                                                 tive chromosome instability (cin) mutant, <br />                                                                                 cin-4(mr127). Two-cell embryos (A B) <br />                                                                                 wild-type (C D) cin-4(mr127) hermaphrodites <br />                                                                                 incubated nonpermissive temperature for <br />                                                                                 6 hr. (A C) DNA visualized DAPI stain- <br />                                                                                 ing. Cell division early C. elegans embryos well <br />                                                                                 described cell-cycle position inferred ex- <br />                                                                                 amination formation orientation the <br />                                                                                 spindle (Strome 1993). (B D) Microtubules <br />                                                                                 visualized staining antitubulin antibodies <br />                                                                                 indicated mitotic spindles bipolar and <br />                                                                                 allowed inference cell-cycle position. Mitotic <br />                                                                                 spindle orientation indicated double-ended <br />                                                                                 arrows. In wild-type embryos, chromosomes seg- <br />                                                                                 regate equally opposite poles cin- <br />                                                                                 4(mr127) embryos chromosomes observed <br />                                                                                 single mass, indicating defect chromo- <br />                                                                                 segregation. Polar bodies (PB) resulting from <br />                                                                                 prior meiotic divisions indicated. The second <br />                                                                                 PB wild-type embryo plane of <br />                                                                                 focus. Bar, 5 mm. <br />  <br />  <br /> in mitotic chromosome structure function. One             CENP-A/HCP-3 failure resolve <br /> the mutants, cin-4(mr127), failed properly segregate          sister kinetochores (Moore Roth 2001). Previ- <br /> chromosomes anaphase, mitotic spindles            ously, determined loss cohesin, RNAi or <br /> were oriented bipolarly (Figure 1, C D). Together,           genetic mutation, suppress failed centromere res- <br /> these results suggested mitotic chromosome structure        olution hcp-4(RNAi) embryos (Moore et al. 2005). The <br /> and function defective cin-4(mr127) mutant.          smc-1(e879) mutation viable allele cohesin <br />    To investigate structure mitotic chro-         component, SMC1, able reestablish centro- <br /> mosomes cin-4(mr127) embryos nonpermissive             mere resolution absence CENP-C/HCP-4 <br /> temperature, embryos stained cell-cycle            (Moore et al. 2005). To determine sister kinetochores <br /> marker antibodies anti-CENP-A/HCP-3 mAb414.                  present, unresolved cin-4(mr127) embryos, we <br /> The mitotic chromosomes cin-4(mr127) em-                constructed double mutant cin-4(mr127);smc-1(e879). <br /> bryos partially condensed compared                We cin-4(mr127);smc-1(e879) double <br /> wild-type chromosomes (Figure 2, A B, DNA). In               mutant nonpermissive temperature mitotic <br /> wild-type embryos, mitotic chromosomes holocen-              chromosomes condensed CENP-A/HCP-3 <br /> tric and, such, centromeric histone CENP-A/               localization opposite sides mitotic chromosomes <br /> HCP-3 present <span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> sister chro-           frequently observed (Figure 2, C D). This <br /> matid, visualized set parallel lines (Figure    result suggested duplicated, unresolved, <br /> 2A, CENP-A) (Buchwitz et al. 1999). Unlike wild-              sister kinetochores present cin-4(mr127) embryos <br /> type embryos, mitotic chromosomes cin-4(mr127)            nonpermissive temperature. <br /> embryos, nonpermissive temperature, showed                   Cohesin associated mitotic chromosomes in <br /> only single line CENP-A/HCP-3 staining (Figure 2B,          cin-4(mr127) embryos: The observation loss cohesin <br /> CENP-A). To determine abnormal CENP-A/HCP-3               function smc-1(e879) allele restored parallel lines <br /> localization <span id='am-239' about='xsp:pattern' typeof='owl:Thing'>pattern</span>, observed cin-4(mr127) embryos,          CENP-A/HCP-3 cin-4(mr127) embryos suggested <br /> affects kinetochore structure, examined proteins              cohesin inhibiting centromere resolution. <br /> involved kinetochore assembly. Similar wild-type         To examine possibility, determined cohesin <br /> embryos, kinetochore protein CENP-C/HCP-4 over-              present mitotic chromosomes cin-4(mr127) <br /> lapped CENP-A/HCP-3 cin-4(mr127) embryos,                embryos nonpermissive temperature. In wild-type <br /> indicating CENP-C/HCP-4 localization CENP-             embryos, cohesin detected mitotic chromo- <br /> A/HCP-3 defective (Moore Roth 2001)                   somes following nuclear envelope breakdown (Chan <br /> (Figure 2B). Other kinetochore proteins                et al. 2003; Mito et al. 2003; Moore et al. 2005). We <br /> found localized CENP-A/HCP-3 (data shown).              stained cin-4(mr127) embryos, shifted nonpermis- <br /> These results addition observation mitotic        sive temperature, antibodies cohesin protein, <br /> chromosomes segregate (Figure 1) suggest              SCC1/COH-2, determined that, unlike wild-type <br /> kinetochore assembly affected cin-4(mr127)             embryos, SCC1/COH-2 present mitotic chromo- <br /> embryos.                                                         somes cin-4(mr127) embryos (Figure 3, A B). In- <br />    The failure cin-4(mr127) mitotic blastomeres        terestingly, SCC1/COH-2 staining did completely <br /> nonpermissive temperature form parallel lines              colocalize mitotic chromosomes, suggesting that <br />                                          CIN-4 Removal Cohesin During Mitosis                                         87 <br />  <br />  <br />  <br />  <br />                                                                                            Figure 2.&#226;&#8364;&#8221;cin-4 necessary <br />                                                                                         centromere resolution. Nuclei <br />                                                                                         prometaphase one- or <br />                                                                                         two- cell embryos (A) wild type, <br />                                                                                         (B) cin-4(mr127), (C) cin- <br />                                                                                         4(mr127);smc-1(e879). Cell-cycle <br />                                                                                         position inferred staining <br />                                                                                         mAb414 antibody (not <br />                                                                                         shown) mark nuclear envelope <br />                                                                                         breakdown. Embryos ob- <br />                                                                                         tained cin-4(mr127) her- <br />                                                                                         maphrodites 6-hr shift to <br />                                                                                         nonpermissive temperature <br />                                                                                         stained DNA, CENP-A/ <br />                                                                                         HCP-3 (red merged image), <br />                                                                                         CENP-C/HCP-4 (green in <br />                                                                                         merged image). Colocalization be- <br />                                                                                         tween CENP-A/HCP-3 CENP- <br />                                                                                         C/HCP-4 observed yellow <br />                                                                                         merged image. (D) Quanti- <br />                                                                                         tative analysis incidence of <br />                                                                                         resolved sister kinetochores. The <br />                                                                                         number chromosomes exam- <br />                                                                                         ined parentheses. Bar, 5 mm. <br />  <br />  <br />  <br />  <br /> cohesin cin-4(mr127) embryos localized specific      sites further, examined cohesin present on <br /> sites. To determine localization SCC1/COH-2      mitotic chromosomes cin-4(mr127) embryos. Wild- <br /> cin-4(mr127) mitotic blastomeres required cohesins,     type embryos cin-4(mr127) embryos raised the <br /> we compromised cohesin complex formation produc-           permissive temperature similar CENP-C/HCP-4 <br /> ing cin-4(mr127);smc-1(e879) double mutant. We reasoned     SCC1/COH-2 localization patterns (data not <br /> that, observed staining cohesin complex     shown); CENP-C/HCP-4 formed parallel lines along <br /> localization, compromising SMC-1 abrogate         chromosomes cohesin absent the <br /> SCC1/COH-2 staining mitotic chromosomes. Unlike            mitotic chromosomes. In cin-4(mr127) embryos, at <br /> the cin-4(mr127) embryos shifted nonpermissive         nonpermissive temperature, SCC1/COH-2 local- <br /> temperature, cin-4(mr127);smc-1(e879) embryos          ized prometaphase chromosomes (Figure 4, A <br /> SCC1/COH-2 localization mitotic chromosomes            B). Curiously, cohesin observed centrosomes <br /> not observed (Figure 3C). Detection cohesin          cin-4(mr127) embryos nonpermissive tempera- <br /> components using immunofluorescence showed similar            ture (&#226;&#8364;&#732;&#226;&#8364;&#732;c&#226;&#8364;&#8482;&#226;&#8364;&#8482; Figure 4B). The cohesin localized mitotic <br /> localization patterns SCC1/COH-2 cin-4(mr127)       chromosomes centrosomes present in <br /> and cin-4(mr127);smc-1(e879) embryos (data shown).        metaphase blastomeres (Figure 4, C D). Further <br /> These results support idea cohesin complex    examination SCC1/COH-2 localization pattern <br /> retained mitotic chromosomes cin-4(mr127) embryos       cin-4(mr127) mitotic chromosomes indicated that <br /> at nonpermissive temperature cin-4 needed     cohesin partially overlapped DNA (Figure 4, B <br /> to prevent cohesin localization.                         D, DNA SCC1/COH-2) instances, <br />    Mitotic cohesin staining adjacent unresolved         SCC1/COH-2 localization did overlap with <br /> sister kinetochores: Our results suggested mitotic       DNA, overlapped CENP-C/HCP-4 localization <br /> cohesin enriched specific sites mitotic         (Figure 4,B D, CENP-C SCC1/COH-2). When <br /> chromosomes cin-4(mr127) embryos. To characterize          SCC1/COH-2 CENP-C/HCP-4 localization did not <br />  88                                            G. Stanvitch L. L. Moore <br />  <br />  <br />  <br />  <br />                                                                                 Figure 3.&#226;&#8364;&#8221;Cohesin localized mitotic <br />                                                                              chromosomes cin-4(mr127). Metaphase nuclei <br />                                                                              two-cell embryos obtained (A) wild- <br />                                                                              type, (B) cin-4(mr127), (C) cin-4(mr127); <br />                                                                              smc-1(e879) hermaphrodites shifted non- <br />                                                                              permissive temperature 6 hr. Embryos were <br />                                                                              stained DNA (red merged image) and <br />                                                                              SCC1/COH-2 (green merged image). Bar, 2 mm. <br />  <br />  <br />  <br />  <br /> overlap, SCC1/COH-2 adjacent CENP-C/              The cohesin localization metaphase chromo- <br /> HCP-4. In cin-4(mr127) embryos nonpermissive           somes wapl-1(RNAi) suggests cohesin be <br /> temperature, SCC1/COH-2 localization mitotic           retained anaphase. Cohesin detected SCC1/ <br /> chromosomes, centrosomes, absent            COH-2 staining present mitotic chromosomes <br /> anaphase blastomeres (Figure 4, E F). These results       prometaphase metaphase (Figure 5, C and <br /> suggested cohesin present near sister            D) completely overlapped DNA prometa- <br /> kinetochores cin-4(mr127) embryos removed          phase metaphase blastomeres (merged images in <br /> from segregating chromosomes anaphase.                     Figure 5, C D). Cohesin absent anaphase <br />    Cohesin removal, centromere resolution,         chromosomes (Figure 5E). Thus, concluded the <br /> inhibited wapl-1(RNAi) embryos: The presence            mechanism remove cohesin mitotic chromo- <br /> cohesin mitotic chromosomes cin-4(mr127) em-            somes prior metaphase conserved C. <br /> bryos suggested CIN-4 function remove             elegans humans. <br /> cohesin entry mitosis. In vertebrates, cohesin         The retention cohesin vertebrate chromosomes <br /> is removed chromatid arms prophase            known inhibit sister-chromatid resolution; in <br /> requires Aurora kinase B, Polo kinase, WAPL           C. elegans possible retention cohesin on <br /> (Losada et al. 2002; Sumara et al. 2002; Gandhi et al.        mitotic chromosomes inhibits centromere resolution <br /> 2006; Kueng et al. 2006). Previously shown        (Losada et al. 2002). To address WAPL-1 was <br /> embryos reduced levels Aurora kinase B            required centromere resolution, examined ki- <br /> homolog (AIR-2) did retain cohesin mitotic             netochores cohesin wapl-1(RNAi) embryos. In <br /> chromosomes C. elegans (Moore et al. 2005). To             wapl-1(RNAi) embryos, SCC1/COH-2 present on <br /> further address related pathway cohesin         prometaphase chromosomes, sister kinetochores, <br /> removal exists C. elegans, used RNAi reduce          detected CENP-C staining, resolved (Figure <br /> expression C. elegans gene similarity    5C). This result suggested WAPL-1 neces- <br /> vertebrate gene WAPL. Others suggested              sary centromere resolution cohesin associ- <br /> the C. elegans gene R08C7.10 similarity WAPL           ation mitotic chromosomes sufficient to <br /> (Gandhi et al. 2006). We 24%            block centromere resolution. <br /> amino acid identity human WAPL C.                    ZK1127.7 cin-4 gene: Genetic mapping placed <br /> elegans R08C7.10 gene product. We determined          cin-4 unc-104 lin-26 genes chromosome <br /> the R08C7.10(RNAi) embryos cohesin present              II (Figure 6). We defined interval containing <br /> mitotic chromosomes (Figure 5, A B). Similar           mr127, using deletion mapping, the <br /> cin-4(mr127) embryos, R08C7.10(RNAi) embryos              deletion mnDf30 did complement cin-4(mr127) at <br /> contained SCC1/COH-2 localized metaphase chro-             nonpermissive temperature. Because breakpoint <br /> mosomes (Figure 5B, arrows). Unlike cin-4(mr127)              mnDf30 determined cosmid <br /> embryos, SCC1/COH-2 localization centrosomes           ZK1127 left him-14 (Zalevsky et al. 1999), we <br /> infrequently observed (&#226;&#8364;&#732;&#226;&#8364;&#732;c&#226;&#8364;&#8482;&#226;&#8364;&#8482; Figure 5B). These results     examined candidate genes unc-104 him-14 <br /> suggest R08C7.10 C. elegans ortholog WAPL      RNAi. RNAi ZK1127.7 produced embryos with <br /> and referred wapl-1.                       uneven distributions DNA anaphase bridging <br />                                          CIN-4 Removal Cohesin During Mitosis                                             89 <br />  <br />  <br />  <br />  <br />                                                                                      Figure 4.&#226;&#8364;&#8221;Mitotic cohesin chromo- <br />                                                                                   somes cin-4(mr127) embryos localized <br />                                                                                   near sites unresolved sister kinetochores. <br />                                                                                   Nuclei indicated cell-cycle positions <br />                                                                                   embryos obtained (A, C, and <br />                                                                                   E) wild-type (B, D, F) cin-4(mr127) <br />                                                                                   hermaphrodites shifted nonpermis- <br />                                                                                   sive temperature 6 hr. Prometaphase <br />                                                                                   metaphase nuclei two-cell <br />                                                                                   embryos anaphase nuclei from <br />                                                                                   four-cell embryos. Embryos stained <br />                                                                                   DNA (red merged image), the <br />                                                                                   kinetochore protein CENP-C (red sec- <br />                                                                                   ond merged image), SCC1/COH-2 <br />                                                                                   (green merged images). Centro- <br />                                                                                   (&#226;&#8364;&#732;&#226;&#8364;&#732;c&#226;&#8364;&#8482;&#226;&#8364;&#8482;) staining SCC1/COH-2 <br />                                                                                   antibody (B, D, F) highly repro- <br />                                                                                   ducible dependent nonpermis- <br />                                                                                   sive temperature. Bar, 2 mm. <br />  <br />  <br />  <br />  <br /> (data shown). In ZK1127.7(RNAi) embryos, cohesin,         sequences obtained RT&#226;&#8364;&#8220;PCR products the <br /> as detected SCC1/COH-2 staining, present            genomic sequence cin-4 wild-type animals. Se- <br /> mitotic chromatin (Figure 6B). To test ZK1127.7       quence analysis cin-4 locus indicates single <br /> is cin-4, determined cin-4(mr127) phenotype         G nucleotide reported genome database <br /> could rescued wild-type ZK1127.7 gene. A          (Wormbase release WS182) present wild-type <br /> 4.8-kb stretch DNA containing ZK1127.7 gene            cin-4 genomic sequence (at nucleotide 7040775). This <br /> with upstream downstream regulatory elements          single nucleotide results additional amino acids <br /> excised cosmid ZK1127. This 4.8-kb fragment          present C terminus predicted CIN-4 protein. <br /> was introduced cin-4(mr127)II;unc119(ed3)III ani-        The cin-4 mRNA SL1 trans-spliced and <br /> mals bombardment protocol (Praitis et al. 2001).        containing open reading frames (ORFs) (Figure 7A). <br /> We obtained independent lines stably inte-         The longer ORF predicted encode protein of <br /> grated wild-type ZK1127.7 gene genome. All       842 amino acids references CIN-4 <br /> three transgenic lines partially rescued cin-    amino acid sequence concern ORF. Sequencing <br /> 4(mr127) phenotypes (data shown). Together,           genomic locus cin-4 mr127 mutant strain <br /> results indicate cin-4 ZK1127.7 re-       identified single A-to-G transition results in <br /> ferred such.                                            glutamate 304 changed glycine (E304G) of <br />    cin-4 homology catalytic domain TOP-2:       predicted CIN-4 amino acid sequence (Figure 7D). <br /> We determined gene structure cin-4 comparing        Comparing predicted CIN-4 protein both <br />  90                                            G. Stanvitch L. L. Moore <br />  <br />  <br />  <br />  <br />                                                                                             Figure 5.&#226;&#8364;&#8221;The cohesin disso- <br />                                                                                          ciation pathway conserved in <br />                                                                                          C. elegans. (A B) Embryos <br />                                                                                          (A) wild type (B) wapl- <br />                                                                                          1(RNAi) stained DNA and <br />                                                                                          SCC1/COH-2. Arrows B indi- <br />                                                                                          cate metaphase chromosomes <br />                                                                                          SCC1/COH-2 staining. Bars, <br />                                                                                          5 mm. (C&#226;&#8364;&#8220;E) Closer view wapl- <br />                                                                                          1(RNA) nuclei different cell- <br />                                                                                          cycle stages stained DNA (red <br />                                                                                          merged image), CENP-C, and <br />                                                                                          SCC1/COH-2 (green merged <br />                                                                                          image). Overlap DNA and <br />                                                                                          SCC1/COH-2 staining shown <br />                                                                                          merged image yellow. Cohe- <br />                                                                                          sin staining centrosomes was <br />                                                                                          rare, observable wapl-1 <br />                                                                                          (RNAi) embryos. Bar, 2 mm. <br />  <br />  <br />  <br />  <br /> human C. elegans genomes BLAST analysis                  A TBLAST search C. elegans genome the <br /> showed CIN-4 89.1 54% amino acid identity        TOP-2 protein sequence identified loci in <br /> to central portion ORF K12D12.1             addition top-2 cin-4 genes. Five loci <br /> human topoisomerase II, respectively. K12D12.1 enco-          appear gene duplications. The remaining <br /> des protein 52% identity entire length    ORF R05D3.1, annotated mitochondrial <br /> to topoisomerase II humans. Given K12D12.1         topoisomerase II, does high degree con- <br /> the likely homolog topoisomerase II C. elegans,      servation DNA sequence top-2 cin-4, <br /> refer K12D12.1 top-2. These findings suggest       suggesting closely related, distinct, topo- <br /> cin-4 encodes protein resembling catalytic domain       isomerase II gene. The C. briggsae genome contains two <br /> of topoisomerase II. Comparing genomic DNA                predicted genes, CBG02746 CBG18156, share <br /> sequences cin-4 top-2 showed           53.9 35% human topoisomerase II, respec- <br /> share similar intron&#226;&#8364;&#8220;exon structure major            tively. CBG02746 79.5% amino acid identity C. <br /> difference large deletion cin-4 covering exon      elegans TOP-2 likely C. briggsae topoisomerase <br /> 2, exon 3, exon 4 top-2       II gene CBG18156 75.1% amino acid <br /> (Figure 7B). Aligning intron&#226;&#8364;&#8220;exon boundaries           identity R05D3.1. These results suggest C. <br /> cin-4 top-2 revealed high degree identity         briggsae lacks cin-4 gene. <br /> (Figure 7C). These results suggest cin-4           TOP-2 required centromere resolution: CIN-4 <br /> arose duplication top-2 gene.                     TOP-2 proteins closely resemble other; thus, <br />                                            CIN-4 Removal Cohesin During Mitosis                                       91 <br />  <br />  <br />  <br />  <br />                                                                              Figure 6.&#226;&#8364;&#8221;The ZK1127.7 open reading frame <br />                                                                            encodes cin-4 gene. (A) Summary deletion <br />                                                                            mapping. (B) Two-cell embryos mock(RNAi) <br />                                                                            ZK1127.7(RNAi) hermaphrodite stained <br />                                                                            DNA SCC1/COH-2. Arrowheads indicate <br />                                                                            mitotic DNA metaphase. Centrosomes la- <br />                                                                            beled &#226;&#8364;&#732;&#226;&#8364;&#732;c.&#226;&#8364;&#8482;&#226;&#8364;&#8482; (B) Bar, 5 mm. <br />  <br />  <br />  <br />  <br /> we determined loss top-2 RNAi affected centro-        embryos, examined centromere resolution smc- <br /> mere resolution cohesion localization. RNAi top-2         1(e879);top-2(RNAi) embryos. In smc-1(e879);top-2(RNAi) <br /> resulted embryonic lethality, indicating top-2    embryos, 2 302 (0.7%) mitotic chromosomes <br /> essential gene (Kamath Ahringer 2003). Because              showed paired sister kinetochores, suggesting loss <br /> cin-4 top-2 genes share high degree DNA           cohesin function did result centromere res- <br /> sequence identity, used qRT&#226;&#8364;&#8220;PCR examine               olution top-2(RNAi) embryos (Figure 8C). We exam- <br /> reduction respective mRNA various RNAi ex-           ined cohesin localization top-2(RNAi) embryos and <br /> periments. In RNAi experiments, reduced cin-4            that, unlike cin-4(mr127) embryos non- <br /> or top-2 gene expression 9.5- 11.5-fold, respectively        permissive temperature, cohesin detected on <br /> (Table 1). In cin-4(RNAi) experiments, top-2 mRNA               mitotic chromosomes (Figure 8, D&#226;&#8364;&#8220;F). Furthermore, <br /> levels reduced 4-fold, top-2(RNAi) did        observed cohesin localized mitotic chromo- <br /> reduce cin-4 mRNA levels (Table 1). As expected, nei-           somes cin-4(mr127);top-2(RNAi) embryos, suggesting <br /> ther cin-4(RNAi) top-2(RNAi) affected R05D3.1 mRNA          loss TOP-2 function did suppress loss cin-4 <br /> levels, reduced 8-fold R05D3.1(RNAi)              function (Figure 8G). These combined results suggest <br /> experiments. As observed (Kamath             cin-4 top-2 unique separate essential <br /> Ahringer 2003), detectable phenotype                functions. <br /> associated R05D3.1(RNAi) embryos did not <br /> further investigate R05D3.1 gene. As cin-4(mr127) <br />                                                                                       DISCUSSION <br /> embryos, top-2(RNAi) embryos contained single line of <br /> CENP-A/HCP-3 staining (Figure 8, A B)                 The holocentric nature C. elegans chromosomes is <br /> 3 212 (1.4%) mitotic chromosomes examined             useful identifying genes controlling chromosome <br /> showed indication resolved sister kinetochores.          structure function (Moore Roth 2001). <br /> Similar cin-4(mr127) embryos, CENP-C/HCP-4               Properly formed chromosomes distinct two- <br /> adjacent CENP-A/HCP-3, suggesting kineto-               line staining pattern centromere kinetochore <br /> chore assembly compromised top-2(RNAi) em-           proteins. We identified temperature-sensitive mutant, <br /> bryos (data shown). To determine compromised             cin-4(mr127), failed generate two-line kinet- <br /> cohesin function restore paired sister kineto-            ochore structure nonpermissive temperature. <br /> chores top-2(RNAi) embryos, does cin-4(mr127)       The CIN-4 protein shares 89% identity central <br />  92   G. Stanvitch L. L. Moore <br />                                              CIN-4 Removal Cohesin During Mitosis                                               93 <br />  <br />                                                                TABLE 1 <br />                                     Efficiency RNAi experiments determined RT&#226;&#8364;&#8220;qPCR <br />  <br />                                                             Fold decrease mRNA levels 6SE (P-value) <br />          Primary target locus                     cin-4                          top-2                        R05D3.1 <br />          cin-4(RNAi)                        9.5 6 1.2 (0.01)               4.0 6 1.1 (0.03)               1.2 6 0.1 (0.14) <br />          top-2(RNAi)                        1.0 6 0.1 (0.5)               10.9 6 0.8 (0.01)               1.1 6 0.1 (0.18) <br />          R05D3.1(RNAi)                      1.0 6 0.1 (0.14)               1.0 6 0.3 (0.08)               8.0 6 1.0 (0.02) <br />             CT values target cDNAs normalized control cDNA, F22B2.13, correct minor differences <br />          cDNA concentrations. The fold decrease expression target cDNAs relative mock(RNAi) calcu- <br />          lated using DDCT method. P-values calculated using Student&#226;&#8364;&#8482;s t-test. SE, standard error. <br />  <br />  <br /> domain C. elegans TOP-2 protein. Our conclusion               cohesion sister chromatids estab- <br /> that cin-4 functional gene based criteria.        lished DNA replication (Ciosk et al. 2000). The <br /> First, cin-4(mr127) mutation mapped posi-               majority cohesin removed prior metaphase. One <br /> tion chromosome II, contained ORF                       protein implicated cohesin dissociation WAPL. In <br /> ZK1127.7. Second, RNAi ZK1127.7 phenocopied                       absence WAPL, cohesin retained mitotic <br /> the cin-4(mr127) mutation. Third, cin-4(mr127) res-              chromosomes vertebrates (Gandhi et al. 2006; Kueng <br /> cued 4.8-kb fragment containing ZK1127.7                    et al. 2006). Likewise, cohesin retained mitotic <br /> ORF. Finally, identified G-to-A transition              chromosomes C. elegans WAPL-1 removed <br /> predicted change glutamate 304 predicted                   RNAi, indicating cohesin dissociation <br /> ZK1127.7 protein sequence glycine. This change               pathway conserved C. elegans vertebrates. <br /> in highly conserved residue present topoisomer-             Given dissociation pathway present C. <br /> ase II molecules examined.                                           elegans, differences WAPL-1 <br />    cin-4 partial gene duplication top-2: The                 CIN-4 function, suggesting function in <br /> observations CIN-4 protein high sequence                  different aspects cohesin dynamics. In wapl-1(RNAi), <br /> similarity catalytic region TOP-2             embryos resolved sister kinetochores, suggesting that <br /> genomic loci cin-4 top-2 highly conserved                WAPL-1 necessary centromere resolution. <br /> suggest cin-4 resulted gene-duplication event.           Although possible efficiency RNAi was <br /> Others suggested partial gene duplications             sufficient block centromere resolution, did <br /> frequent C. elegans (Katju Lynch 2003). The C.                observe cohesin mitotic chromosomes, indicating <br /> briggsae genome contains orthologs top-2 R05D3.1              presence cohesin mitotic chromosomes is <br /> but does contain cin-4 ortholog, suggesting               sufficient inhibit centromere resolution. This <br /> the gene duplication giving rise cin-4 arose            result contrasted results obtained cin-4(RNAi) <br /> divergence C. elegans C. briggsae. Interestingly, cin-4       cin-4(mr127) animals. In organisms monocentric <br /> is essential gene, indicating cin-4 ac-           chromosomes, populations cohesin: <br /> quired new essential function (neofunctionalization)               dissociated prophase second <br /> or top-2 lost essential function             population protected Shugoshin (Salic et al. <br /> provided cin-4 (subfunctionalization).                            2004; McGuinness et al. 2005). Whereas cohesin com- <br />    cin-4 required cohesin dynamics centro-                pletely overlapped mitotic chromosomes wapl- <br /> mere resolution: In cin-4(mr127) embryos non-                 1(RNAi) embryos, subset chromosome <br /> permissive temperature, cohesin present mitotic               cohesin staining cin-4(mr127) embryos. These re- <br /> chromosomes centromere resolution failed, sug-                   sults suggested majority cohesin removed <br /> gesting link processes. Cohesin                  WAPL-1 (Figure 9). Interestingly, cohesin popula- <br /> associates chromosomes prior DNA replication,                tion present cin-4 function compromised <br />  <br />  <br />    Figure 7.&#226;&#8364;&#8221;cin-4 duplication top-2 gene. (A) The cin-4 mRNA sequence following SL1 spliced leader sequence. A <br /> potential upstream open reading frame shown longer boxed region predicted cin-4 start codon shown boxed <br /> ATG. (B) Comparison structure top-2 cin-4 basis DNA sequence alignment. A cartoon depicts results <br /> from alignment top-2 cin-4 loci aligned using DNA Strider Blocks alignment tool. A representation topo- <br /> isomerase II functional domains DNA alignment graphic. (C) DNA sequence alignment intron&#226;&#8364;&#8220;exon boundary <br /> sequences top-2 cin-4. The intron DNA sequences (lowercase) cin-4 aligned introns 1, 4, 5 top-2. <br /> Non-intronic DNA shown uppercase. (D) Amino acid sequence alignment TOP-2 CIN-4. Amino acid sequences were <br /> aligned using Clustal W package Lasergene sequence analysis software package (DNASTAR). Amino acid identity is <br /> against solid background. The box indicates catalytic tyrosine present HsTOP-2a TOP-2 serine residue pre- <br /> sent CIN-4. An asterisk indicates glutamate 304-to-glycine change identified mr127. <br />  94   G. Stanvitch L. L. Moore <br />  <br />                      removed solely WAPL-1. As Shugoshin- <br />                      protected centromeric cohesin, cohesin mitotic <br />                      chromosomes cin-4(mr127) embryos localized on <br />                      near sister kinetochores. Unlike cohesin popula- <br />                      tion removed WAPL-1, cohesin near sister <br />                      kinetochores interfere centromere resolution. <br />                      Consistent idea, loss cohesin function via <br />                      genetic mutant bypassed need CIN-4 in <br />                      centromere resolution. The role CIN-4 centromere <br />                      resolution remove centromere-associated <br />                      cohesin population enable centromere resolution. <br />                      Alternatively, cohesin excluded CENP-A <br />                      chromatin CIN-4-dependent manner. Thus, the <br />                      absence functional CIN-4, cohesin retained near <br />                      sister kinetochores protected WAPL-1-medi- <br />                      ated dissociation. <br />                         In Saccharomyces cerevisiae, evidence exists topo- <br />                      isomerase II cohesin interact regulate centromere <br />                      structure function. The region near centromeres is <br />                      enriched cohesin proteins, able separate and <br />                      rejoin repeatedly prior anaphase (Blat Kleckner <br />                      1999; Goshima Yanagida 2000; He et al. 2000; <br />                      Laloraya et al. 2000; Tanaka et al. 2000). Furthermore, <br />                      centromeric cohesion negatively regulated SUMO- <br />                      1 modification TOP-2 (Bachant et al. 2002). As <br />                      evidence SUMO-1 modification of <br />                      TOP-2 acting centromere resolution C. elegans <br />                      (L. Moore, unpublished results). Thus possible that <br />                      CIN-4 C. elegans co-opted essential function <br />                      topoisomerase II. It interesting examine <br />                      TOP-2 related organisms C. briggsae, which <br />                      lack cin-4 gene. <br />                         TOP-2 CIN-4 function: Both cin-4 top-2 genes <br />                      share significant identity topoisomerase II are <br />                      essential, indicating perform necessary <br />                      topoisomerase II functions. RNAi TOP-2 inhibited <br />                      centromere resolution, did use topoisomerase <br />                      II inhibitor drugs (L. Moore, unpublished data). How- <br />                      ever, CIN-4 TOP-2 function differently centro- <br />                      mere resolution genetic disruption cohesin <br />                      function did restore centromere resolution top- <br />                      2(RNAi) embryos. Topoisomerase II function necessary <br />                      remove catenations DNA. Two observations <br />                      suggest CIN-4 does function decatenation of <br />                      DNA. First, CIN-4 protein resembles 92-kDa <br />  <br />  <br />  <br />                         Figure 8.&#226;&#8364;&#8221;TOP-2 necessary centromere resolution. <br />                      Prometaphase nuclei two-cell embryos (A D) wild <br />                      type, (B F) top-2(RNAi), (C) top-2(RNAi);smc-1(e879), (E) <br />                      cin-4(mr127), (G) top-2(RNAi);cin-4(mr127). Cell-cycle po- <br />                      sition inferred staining mAb414 antibody <br />                      (not shown), identifies nuclear envelope. Embryos <br />                      cin-4(mr127) hermaphrodites obtained 6-hr <br />                      shift nonpermissive temperature. Mitotic chromosomes <br />                      stained (A&#226;&#8364;&#8220;C) DNA CENP-A/HCP-3 (D&#226;&#8364;&#8220;G) <br />                      DNA SCC1/COH-2. SCC1/COH-2 centrosome stain- <br />                      ing indicated &#226;&#8364;&#732;&#226;&#8364;&#732;c.&#226;&#8364;&#8482;&#226;&#8364;&#8482; Bar, 2 mm. <br />                                           CIN-4 Removal Cohesin During Mitosis                                                     95 <br />  <br />                                                                interact cohesin induce prevent a <br />                                                                protein modification affecting cohesin localization. <br />                                                                   In conclusion, identified essential gene, <br />                                                                cin-4, appears partial duplication the <br />                                                                C. elegans topoisomerase II gene top-2. The cin-4 gene is <br />                                                                required prevent aberrant cohesin localization and <br />                                                                inhibition centromere resolution. As cin-4-like <br />    Figure 9.&#226;&#8364;&#8221;Model CIN-4 role centromere resolution     gene present organisms, speculate that <br /> and cohesin dynamics. A cartoon cross section mi-    functions performed topoisomer- <br /> totic chromosome (large circle). The majority cohesin       ase II organisms. In organisms, single <br /> (shading) removed WAPL-1; however, second popula- <br /> tion cohesin near sister kinetochores removed by <br />                                                                topoisomerase II needed. However, in <br /> WAPL-1 (stippled oval). CIN-4 removes reorganizes      mammals, topoisomerase II proteins, b, with <br /> second cohesin population enable centromere resolution.     distinct biochemical localization properties are <br /> Sister kinetochores depicted small solid circles.       present, suggesting division function (Chung <br />                                                                et al. 1989; Drake et al. 1989). The topoisomerase II a <br />                                                                isoform primarily responsible passage through <br /> fragment yeast TOP-2, able decatenate       mitosis, partially substituted topoisomer- <br /> DNA (Lindsley Wang 1991; Roca 2004). Second,               ase II b (Sakaguchi Kikuchi 2004). Topoisomer- <br /> examination aligned topoisomerase II protein            ase II b appears function regulating developmental <br /> sequences indicates CIN-4 serine residue            gene expression ( Ju et al. 2006; Lyu et al. 2006). Sim- <br /> instead tyrosine residue contained TOP-2         ilarly, CIN-4 represent division function for <br /> topoisomerase II. Others shown tyrosine         topoisomerase II C. elegans represents unique <br /> residue forms covalent bond topoisomerase            opportunity investigate different roles top- <br /> II DNA catalytic cycle essential     oisomerase II centromere function. <br /> decatenation (Worland Wang 1989; Jensen et al.                The authors thank M. Roth temperature-sensitive embryonic <br /> 1996). Thus, essential decatenation function likely     lethal mutant bank, J. Loidl gift SCC1/COH-2 antibody, and <br /> to performed TOP-2 C. elegans. Given             P. Padilla critical reading manuscript. Some strains used here <br /> unresolved catenation DNA acts form co-             provided Caenorhabditis Genetics Center. This work was <br />                                                                supported departmental funds L.L.M. <br /> hesion, unresolved catenations present top-2(RNAi) <br /> embryos explain inhibiting cohesin function <br /> does restore centromere resolution TOP-2 is <br /> absent.                                                                              LITERATURE CITED <br />    As CIN-4 likely alter DNA topology, CIN-4         Albertson, D. G., J. N. Thomson, 1982 The kinetochores of <br /> may regulate cohesin function association              Caenorhabditis elegans. Chromosoma 86: 409&#226;&#8364;&#8220;428. <br />                                                                Bachant, J., A. Alcasabas, Y. Blat, N. Kleckner S. J. Elledge, <br /> DNA. Topoisomerase II known associate                 2002 The SUMO-1 isopeptidase Smt4 linked centromeric <br /> variety DNA structures involving four-way junction            cohesion SUMO-1 modification DNA topoisomerase <br /> (Roca et al. 1993; West Austin 1999; Trigueros                 II. Mol. Cell 9: 1169&#226;&#8364;&#8220;1182. <br />                                                                Baumann, C., R. Korner, K. Hofmann E. A. Nigg, 2007 PICH, <br /> et al. 2004). In models cohesin function,             centromere-associated SNF2 family ATPase, regulated Plk1 <br /> DNA structures potentially block trans-                  required spindle checkpoint. Cell 128: 101&#226;&#8364;&#8220;114. <br /> location cohesin DNA, resulting aberrant           Blat, Y., N. Kleckner, 1999 Cohesins bind preferential sites <br />                                                                    yeast chromosome III, differential regulation along <br /> cohesin localization preventing resolu-            arms versus centric region. Cell 98: 249&#226;&#8364;&#8220;259. <br /> tion sister DNAs. Recruitment topoisomerase II        Blower, M. D., B. A. Sullivan G. H. Karpen, 2002 Conserved <br /> CIN-4 C. elegans necessary resolve               organization centromeric chromatin flies humans. Dev. <br />                                                                    Cell 2: 319&#226;&#8364;&#8220;330. <br /> these DNA structures restore resolution. In           Brenner, S., 1974 The genetics Caenorhabditis elegans. Genetics <br /> model, presence tyrosine, serine CIN-4,        77: 71&#226;&#8364;&#8220;94. <br /> the catalytic site necessary form DNA-        Brenner, S., D. Pepper, M. W. Berns, E. Tan B. R. Brinkley, <br />                                                                    1981 Kinetochore structure, duplication, distribution in <br /> binding sites (Roca et al. 1996). Alternatively, CIN-4         mammalian cells: analysis human autoantibodies sclero- <br /> recruit factors needed cohesin removal. An inter-              derma patients. J. Cell Biol. 91: 95&#226;&#8364;&#8220;102. <br /> esting observation cohesin readily detected        Buchwitz, B. J., K. Ahmad, L. L. Moore, M. B. Roth S. Henikoff, <br />                                                                    1999 A histone-H3-like protein C. elegans. Nature 401: 547&#226;&#8364;&#8220; <br /> at centrosomes cin-4(mr127) embryos. In humans,                 548. <br /> cohesin detected centrosomes              Chan, R. C., A. Chan, M. Jeon, T. F. Wu, D. Pasqualone et al., <br /> required centrosome function (Gregson et al.                   2003 Chromosome cohesion regulated clock gene pa- <br />                                                                    ralogue TIM-1. Nature 423: 1002&#226;&#8364;&#8220;1009. <br /> 2001). In wild-type C. elegans embryos, cohesin         Chung, T. D., F. H. Drake, K. B. Tan, S. R. Per, S. T. Crooke et al., <br /> detected centrosomes mitotic chromosomes                  1989 Characterization immunological identification of <br /> (Chan et al. 2003; Mito et al. 2003; Moore et al. 2005).           cDNA clones encoding human DNA topoisomerase II iso- <br />                                                                    zymes. Proc. Natl. Acad. Sci. USA 86: 9431&#226;&#8364;&#8220;9435. <br /> Given absence cin-4 function, localization      Cimini, D., B. Howell, P. Maddox, A. Khodjakov, F. Degrassi et al., <br /> of cohesin sites suggested cin-4 directly         2001 Merotelic kinetochore orientation major mechanism <br />  96                                                         G. Stanvitch L. L. Moore <br />  <br />      aneuploidy mitotic mammalian tissue cells. J. Cell Biol. 153:     Lyu, Y. L., C. P. Lin, A. M. Azarova, L. Cai, J. C. Wang et al., <br />      517&#226;&#8364;&#8220;528.                                                                    2006 Role topoisomerase IIbeta expression devel- <br /> Ciosk, R., M. Shirayama, A. Shevchenko, T. Tanaka, A. Toth et al.,               opmentally regulated genes. Mol. Cell. Biol. 26: 7929&#226;&#8364;&#8220;7941. <br />      2000 Cohesin&#226;&#8364;&#8482;s binding chromosomes depends sepa-                McGuinness, B. E., T. Hirota, N. R. Kudo, J. M. Peters K. <br />      rate complex consisting Scc2 Scc4 proteins. Mol. Cell 5:             Nasmyth, 2005 Shugoshin prevents dissociation cohesin <br />      243&#226;&#8364;&#8220;254.                                                                    centromeres mitosis vertebrate cells. PLoS Biol. <br /> Davis, L. I., G. Blobel, 1986 Identification characteriza-               3: 433&#226;&#8364;&#8220;449. <br />      tion nuclear pore complex protein. Cell 45: 699&#226;&#8364;&#8220;709.               Mito, Y., A. Sugimoto M. Yamamoto, 2003 Distinct develop- <br /> Drake, F. H., G. A. Hofmann, H. F. Bartus, M. R. Mattern, S. T.                  mental function Caenorhabditis elegans homologs of <br />      Crooke et al., 1989 Biochemical pharmacological proper-                 cohesin subunit Scc1/Rad21. Mol. Biol. Cell 14: 2399&#226;&#8364;&#8220; <br />      ties p170 p180 forms topoisomerase II. Biochemistry               2409. <br />      28: 8154&#226;&#8364;&#8220;8160.                                                          Moore, L. L., M. B. Roth, 2001 HCP-4, CENP-C-like protein <br /> Gandhi, R., P. J. Gillespie T. Hirano, 2006 Human Wapl                  Caenorhabditis elegans, required resolution sister cen- <br />      cohesin-binding protein promotes sister-chromatid resolu-              tromeres. J. Cell Biol. 153: 1199&#226;&#8364;&#8220;1208. <br />      tion mitotic prophase. Curr. Biol. 16: 2406&#226;&#8364;&#8220;2417.                    Moore, L. L., M. Morrison M. B. Roth, 1999 HCP-1, protein <br /> Goshima, G., M. Yanagida, 2000 Establishing biorientation oc-                involved chromosome segregation, localized centro- <br />      curs precocious separation sister kinetochores,             mere mitotic chromosomes Caenorhabditis elegans. J. Cell <br />      arms, early spindle budding yeast. Cell 100:              Biol. 147: 471&#226;&#8364;&#8220;480. <br />      619&#226;&#8364;&#8220;633.                                                                Moore, L. L., G. Stanvitch, M. B. Roth D. Rosen, 2005 HCP- <br /> Gregson, H. C., J. A. Schmiesing, J. S. Kim, T. Kobayashi, S. Zhou               4/CENP-C promotes prophase timing centromere res- <br />      et al., 2001 A potential role human cohesin mitotic spin-            olution enabling centromere association HCP-6 in <br />      dle aster assembly. J. Biol. Chem. 276: 47575&#226;&#8364;&#8220;47582.                        Caenorhabditis elegans. Mol. Cell. Biol. 25: 2583&#226;&#8364;&#8220;2592. <br /> Gruber, S., C. H. Haering K. Nasmyth, 2003 Chromosomal                   Nasmyth, K., 2001 Disseminating genome: joining, resolving, <br />      cohesin forms ring. Cell 112: 765&#226;&#8364;&#8220;777.                                    separating sister chromatids mitosis meiosis. <br /> He, D., B. R. Brinkley, 1996 Structure dynamic organiza-                 Annu. Rev. Genet. 35: 673&#226;&#8364;&#8220;745. <br />      tion centromeres/prekinetochores nucleus mamma-            Oegema, K., A. Desai, S. Rybina, M. Kirkham A. A. Hyman, <br />      lian cells. J. Cell Sci. 109: 2693&#226;&#8364;&#8220;2704.                                    2001 Functional analysis kinetochore assembly Caeno- <br /> He, X., S. Asthana P. K. Sorger, 2000 Transient sister chroma-               rhabditis elegans. J. Cell Biol. 153: 1209&#226;&#8364;&#8220;1226. <br />      tid separation elastic deformation chromosomes            Pasierbek, P., M. Jantsch, M. Melcher, A. Schleiffer, D. Schweizer <br />      mitosis budding yeast. Cell 101: 763&#226;&#8364;&#8220;775.                                et al., 2001 A Caenorhabditis elegans cohesion protein with <br /> Howman, E. V., K. J. Fowler, A. J. Newson, S. Redward, A. C.                     functions meiotic chromosome pairing disjunction. <br />      MacDonald et al., 2000 Early disruption centromeric chro-                Genes Dev. 15: 1349&#226;&#8364;&#8220;1360. <br />      matin organization centromere protein A (Cenpa) null mice.           Praitis, V., E. Casey, D. Collar J. Austin, 2001 Creation of <br />      Proc. Natl. Acad. Sci. USA 97: 1148&#226;&#8364;&#8220;1153.                                   low-copy integrated transgenic lines Caenorhabditis elegans. <br /> Huang, J., D. Moazed, 2006 Sister chromatid cohesion silent               Genetics 157: 1217&#226;&#8364;&#8220;1226. <br />      chromatin: sister ring. Genes Dev. 20: 132&#226;&#8364;&#8220;137.         Rattner, J. B., M. J. Hendzel, C. S. Furbee, M. T. Muller D. P. <br /> Jensen, S., A. H. Andersen, E. Kjeldsen, H. Biersack, E. H. Olsen                Bazett-Jones, 1996 Topoisomerase II alpha associated with <br />      et al., 1996 Analysis functional domain organization DNA              mammalian centromere cell cycle- species-specific <br />      topoisomerase II humans Saccharomyces cerevisiae.                  manner required proper centromere/kinetochore <br />      Mol. Cell. Biol. 16: 3866&#226;&#8364;&#8220;3877.                                             structure. J. Cell Biol. 134: 1097&#226;&#8364;&#8220;1107. <br /> Ju, B. G., V. V. Lunyak, V. Perissi, I. Garcia-Bassets, D. W. Rose et al.,   Rieder, C. L., E. D. Salmon, 1998 The vertebrate cell kineto- <br />      2006 A topoisomerase IIbeta-mediated dsDNA break required                   chore roles mitosis. Trends Cell Biol. 8: 310&#226;&#8364;&#8220; <br />      regulated transcription. Science 312: 1798&#226;&#8364;&#8220;1802.                        318. <br /> Kamath, R. S., J. Ahringer, 2003 Genome-wide RNAi screen-                Roca, J., 2004 The path DNA dimer interface of <br />      ing Caenorhabditis elegans. Methods 30: 313&#226;&#8364;&#8220;321.                         topoisomerase II. J. Biol. Chem. 279: 25783&#226;&#8364;&#8220;25788. <br /> Kamath, R. S., M. Martinez-Campos, P. Zipperlen, A. G. Fraser                Roca, J., J. M. Berger J. C. Wang, 1993 On simultaneous <br />      J. Ahringer, 2001 Effectiveness specific RNA-mediated                binding eukaryotic DNA topoisomerase II pair double- <br />      interference ingested double-stranded RNA Caeno-                 stranded DNA helices. J. Biol. Chem. 268: 14250&#226;&#8364;&#8220;14255. <br />      rhabditis elegans. Genome Biol. 2: RESEARCH0002.0001&#226;&#8364;&#8220;0002.              Roca, J., J. M. Berger, S. C. Harrison J. C. Wang, 1996 DNA <br />      0010.                                                                       transport type II topoisomerase: direct evidence two- <br /> Katju, V., M. Lynch, 2003 The structure early evolution                  gate mechanism. Proc. Natl. Acad. Sci. USA 93: 4057&#226;&#8364;&#8220;4062. <br />      recently arisen gene duplicates Caenorhabditis elegans        Sakaguchi, A., A. Kikuchi, 2004 Functional compatibility be- <br />      genome. Genetics 165: 1793&#226;&#8364;&#8220;1803.                                            tween isoform alpha beta type II DNA topoisomerase. <br /> Kueng, S., B. Hegemann, B. H. Peters, J. J. Lipp, A. Schleiffer et al.,          J. Cell Sci. 117: 1047&#226;&#8364;&#8220;1054. <br />      2006 Wapl controls dynamic association cohesin              Salic, A., J. C. Waters T. J. Mitchison, 2004 Vertebrate shu- <br />      chromatin. Cell 127: 955&#226;&#8364;&#8220;967.                                               goshin links sister centromere cohesion kinetochore micro- <br /> Laloraya, S., V. Guacci D. Koshland, 2000 Chromosomal                        tubule stability mitosis. Cell 118: 567&#226;&#8364;&#8220;578. <br />      addresses cohesin component mcd1p. J. Cell Biol. 151:            Stear, J. H., M. B. Roth, 2002 Characterization HCP-6, C. <br />      1047&#226;&#8364;&#8220;1056.                                                                  elegans protein required prevent chromosome twisting and <br /> Lengauer, C., K. W. Kinzler B. Vogelstein, 1998 Genetic in-                  merotelic attachment. Genes Dev. 16: 1498&#226;&#8364;&#8220;1508. <br />      stabilities human cancers. Nature 396: 643&#226;&#8364;&#8220;649.                      Strome, S., 1993 Determination cleavage planes. Cell 72: 3&#226;&#8364;&#8220;6. <br /> Lindsley, J. E., J. C. Wang, 1991 Proteolysis patterns epitopi-       Sumara, I., E. Vorlaufer, P. T. Stukenberg, O. Kelm, N. Redemann <br />      cally labeled yeast DNA topoisomerase II suggest allosteric              et al., 2002 The dissociation cohesin chromosomes in <br />      transition enzyme induced ATP binding. Proc. Natl.                prophase regulated Polo-like kinase. Mol. Cell 9: 515&#226;&#8364;&#8220;525. <br />      Acad. Sci. USA 88: 10485&#226;&#8364;&#8220;10489.                                         Tanaka, T., J. Fuchs, J. Loidl K. Nasmyth, 2000 Cohesin en- <br /> Link, C. D., A. Taft, V. Kapulkin, K. Duke, S. Kim et al., 2003 Gene             sures bipolar attachment microtubules sister centromeres <br />      expression analysis transgenic Caenorhabditis elegans Alz-             resists precocious separation. Nat. Cell Biol. 2: 492&#226;&#8364;&#8220; <br />      heimer&#226;&#8364;&#8482;s disease model. Neurobiol. Aging 24: 397&#226;&#8364;&#8220;413.                       499. <br /> Losada, A., M. Hirano T. Hirano, 1998 Identification Xen-             Toth, A., R. Ciosk, F. Uhlmann, M. Galova, A. Schleiffer et al., <br />      opus SMC protein complexes required sister chromatid cohe-              1999 Yeast cohesin complex requires conserved protein, <br />      sion. Genes Dev. 12: 1986&#226;&#8364;&#8220;1997.                                             Eco1p(Ctf7), establish cohesion sister chromatids <br /> Losada, A., M. Hirano T. Hirano, 2002 Cohesin release                     DNA replication. Genes Dev. 13: 320&#226;&#8364;&#8220;333. <br />      required sister chromatid resolution, conden-           Trigueros, S., J. Salceda, I. Bermudez, X. Ferna <br /> </body></html>