<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>  <br /> Salty dog, an SLC5 symporter, modulates Drosophila response <span id='am-7' about='protege:TO' typeof='owl:Thing'>to</span> salt stress <br />  <br />    Stergiopoulos K, Cabrero P, Davies S, Dow JAT. Salty dog, an                yeast (24), where the high osmolarity glycerol stress pathway <br /> SLC5 symporter, modulates Drosophila response to salt stress.                  (HOG) is under control of the Hog1 kinase (24). Trehalose is <br /> Physiol Genomics 37: 1&#226;&#8364;&#8220;11, 2009. First published November 18,                  another key osmolyte in bacteria, yeast and plants (17); and in <br /> 2008; doi:10.1152/physiolgenomics.90360.2008.&#226;&#8364;&#8221;To regulate their                C. elegans, osmotic stress increases expression of the synthetic <br /> internal environments, organisms must adapt to varying ion levels in their     enzyme trehalose 6-phosphate synthase (tps1 and tps2), with a <br /> diet. Adult Drosophila were exposed to dietary salt stress, and their <br /> physiological, <span id='am-8' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-9' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span>survival</span>, and gene expression responses monitored. Insects <br />                                                                                corresponding increase in trehalose levels (26). <br /> continued to feed on NaCl-elevated diet, although levels &#226;&#172;&#381;4% wt/vol               In insects, mechanisms of salt tolerance are less well under- <br /> ultimately proved fatal. Affymetrix microarray analysis of flies fed on        stood. In the aquatic larvae of the mosquito Aedes aegypti, <br /> diet containing elevated NaCl showed a phased response: the earliest           there is an interplay between salinity, feeding, and hemolymph <br /> response was widespread upregulation of immune genes, followed by              levels of the diuretic amine 5-hydroxytryptamine (12). Most <br /> upregulation of carbohydrate metabolism as the immune response was             larval mosquitoes live in fresh or brackish water, and so they <br /> downregulated, then finally a switch to amino acid catabolism and              are specialized for uptake of ions as a scarce resource. The size <br /> inhibition of genes associated with the reproductive axis. Significantly,      of the ion-scavenging anal papillae varies inversely with the <br /> the online transcriptomic resource FlyAtlas reports that most of the           ionic content of the water, suggesting that these are key <br /> modulated genes are predominantly expressed in hindgut or Malpighian           osmoregulatory organs (13, 31, 42). By contrast, some species <br /> (renal) tubule, implicating these excretory tissues as the major responders    of insect survive in and around saline lakes and show particular <br /> to salt stress. Three genes were selected for further study: the SLC5          specializations for handling of ions (like magnesium) that are <br /> symporter CG2196, the GLUT transporter CG6484, and the transcription <br />                                                                                present in massive excess (7). The insect body plan is thus <br /> factor sugarbabe (previously implicated in starvation and stress re- <br /> sponses). Expression profiles predicted by microarray were validated by        adaptable to a wide range of environmental salt levels. <br /> quantitative PCR (qPCR); expression was mapped to the alimentary                  Although Drosophila does not normally encounter extremes of <br /> canal by in situ hybridization. CG2196::eYFP overexpression constructs         salinity, its advantages as a genetic model in which gene expres- <br /> were localized to the basolateral membrane of the Malpighian (renal)           sion can both be studied and manipulated (15, 16) make it an <br /> tubules, and RNAi against CG2196 improved survival on high-salt diet,          attractive species to study. Previously, <span id='am-164' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-165' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-241' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span>mutation</span> of the inebriated <br /> even when driven specifically to just principal cells of the Malpighian        putative neurotransmitter/osmolyte transporter (ine) was shown to <br /> tubule, confirming both this tissue and this transporter as major determi-     confer reduced tolerance to dietary salt (21). Overexpression of <br /> nants of survival upon salt stress. Accordingly, CG2196 was renamed            either of the two isoforms of ine (ine-P1 or ine-P2) rescued this <br /> salty dog (salt).                                                              osmotic stress response defect. The osmotic stress-sensitive <br /> Malpighian tubule; microarray; stress response                                 <span id='am-86' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-87' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span>phenotype</span> of the ine mutants was suggested to result <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> an <br />                                                                                inability to accumulate osmolytes within the Malpighian tubule <br />                                                                                and hindgut. Osmotic stress can also be induced by dehydra- <br /> SALT STRESS CAN DISRUPT INTERNAL         homeostasis, so most                  tion and rehydration. In Drosophila populations selected for <br /> organisms can robustly defend their internal milieu against                    enhanced desiccation resistance (2, 19), flies excrete some <br /> fluctuations in external salt over a limited range. Work,                      sodium during desiccation but retain &#226;&#172;&#402;85% of the whole body <br /> particularly in yeast (1, 5, 6, 29, 34, 35) and in plants (3, 20,              sodium content, 83% of potassium, and 60% of chloride (19), <br /> 23, 35, 36, 41), suggests a variety of mechanisms, particu-                    implying that defense against desiccation is not purely by loss <br /> larly the synthesis or regulation of osmolytes, such as                        of ionic osmolytes. <br /> glycerol; the increase of K&#226;&#171;&#185; uptake and Na&#226;&#171;&#185; efflux at the                         The aim of this study was to investigate the response of adult <br /> plasma membrane; and Na&#226;&#171;&#185; accumulation at the vacuole                           Drosophila to acute salt stress, using Affymetrix microarrays <br /> (involving proton pumping ATPases, Na&#226;&#171;&#185;-efflux pumps,                           and subsequent molecular, transgenic, and physiological vali- <br /> plasma membrane proteins that operate as sensors of turgor                     dation of genes implicated in the response. <br /> loss and signaling cascades that regulate these responses). <br />    The worm, Caenorhabditis elegans, is a simple multicellular                 MATERIALS AND METHODS <br /> organism compatible with FACS-like high-throughput screen-                        Drosophila. Flies were kept on standard medium in tubes at 25&#194;&#176;C, <br /> ing. In C. elegans, massive salt loading caused upregulation of                12:12 h photoperiod, and 55% relative humidity. Wild-type (Oregon <br /> glycerol 3-phosphate dehydrogenase (25), so allowing the                       R) flies were used for obtaining genomic DNA, cDNA, and for <br /> accumulation of glycerol as a protective osmolyte, exactly as in               performing fluid secretion assays. Microinjections to produce trans- <br />  <br />  <br />    Address for reprint requests and other correspondence: J. A. T. Dow,           The costs of publication of this article were defrayed in part by the payment <br /> Integrative &amp; Systems Biology, Faculty of Biomedical and Life Sciences,        of page charges. The article must therefore be hereby marked &#226;&#8364;&#339;advertisement&#226;&#8364;&#65533; <br /> Univ. of Glasgow, Glasgow G12 8QQ, UK (e-mail: j.a.t.dow@bio.gla.ac.uk).       in accordance with 18 U.S.C. Section 1734 solely to indicate this fact. <br />  <br />                                         1094-8341/09 $8.00 Copyright &#194;&#169; 2009 the American Physiological Society                                               1 <br />  2                                                               SALT STRESS IN DROSOPHILA <br />  <br /> genic lines were performed in a w1118 background. Transgenes under                       Microarrays. Seven-day-old flies were transferred either into food <br /> control of yeast upstream activating sequences (UAS) were driven                      with 4% NaCl added or normal food. The flies were left for 4, 8, 16, <br /> ubiquitously by crossing to the Act5C-GAL4/CyO line (line 4414                        or 32 h to feed, and after that 100 flies from each treatment were <br /> from Bloomington Drosophila stock center) and in Malpighian tubule                    anaesthetized on ice and homogenized in 300 &#226;&#65533;&#174;l of TRIzol. This <br /> principal cells with the c42-GAL4 line (33).                                          whole procedure was repeated two more times to produce three <br />    Generation of transgenic <span id='am-242' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-243' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-249' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-254' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-261' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-264' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-271' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-290' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-292' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-295' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-296' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-303' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-305' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-318' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-319' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span>RNA interference</span> flies. Inserts of &#226;&#172;&#402;500                   biological replicates for each sample. All the samples were stored <br /> bp, directed against nonconserved regions of genes of interest, were                  directly at &#226;&#171;&#186;80&#194;&#176;C and processed according to standard protocols. <br /> cloned into the pWIZ vector (27) and germ-line transformed into                       Samples of 20 &#226;&#65533;&#174;g total RNA were reverse-transcribed, then in vitro <br /> w1118 embryos, according to standard procedures. This produced                        transcribed, according to Affymetrix standard protocols. The quality <br /> transgenic flies, in which hairpin ds-RNA could be expressed in cells                 of the complementary RNA (cRNA) was also checked on an Agilent <br /> of choice under control of appropriate GAL4 driver lines (8).                         RNA Bioanalyzer, with a sample in which the broad cRNA peak <br />    Dietary salt substitution. For the preparation of the &#226;&#8364;&#339;salt&#226;&#8364;&#65533; food,                 exceeded the height of the low-molecular-weight degradation peak <br /> the required salt (NaCl or KCl) was dissolved in 100 ml of normal                     taken to be satisfactory. Samples were then run on the Affymetrix <br /> food just after its preparation, mixed, and the diet left to set. Where               Drosophila genome I array under standard conditions: the experiment <br /> appropriate, indigo carmine dye (at a concentration of 1.66 g/l) was                  thus comprised 24 arrays. Quality control was performed as described <br /> also added immediately after the diet preparation, but before the                     in a previous paper from our lab (39). <br /> food set, to provide a visible marker of ingestion. Diet was freshly                     Bioinformatics. As well as routine analysis by Affymetrix propri- <br /> prepared to avoid any changes in the concentration of the salts due                   etary software (MAS 5.0), array data were analyzed using FunAlyse, <br /> to evaporation. For dissection, we anaesthetized flies by chilling                    a pipeline based on the Bioconductor package, and using robust <br /> them briefly on ice before dissecting out tubules in Schneider&#226;&#8364;&#8482;s                      multichip average for low-level normalization with subsequent calcu- <br /> medium (Invitrogen).                                                                  lation of rank products under random permutations. This method <br />  <br />  <br />  <br />  <br /> Fig. 1. Continued ingestion an survival rates on NaCl-supplemented diet. A: example of Indigo carmine in gut after 3 h of feeding with 4% NaCl food (left) <br /> compared with 6 h gut that would be scored as clear (right). B: ingestion of high-salt food over time. Flies in 0, 4, and 5% NaCl food, which contained the blue <br /> dye Indigo Carmine, were dissected after 1, 3, and 6 h and scored for blue gut contents. C: survival rates in days for flies fed with 0, 3, 4, 5, 6, 7.5, and 9% NaCl <br /> compared with starvation. Increasing NaCl significantly reduces survival [Kaplan-Meier analysis with log-rank test for trend, taking P &#226;&#171;&#189; 0.05 as the critical level <br /> (2-tailed)]. *Significantly different from the starvation control. <br />  <br />                                                      Physiol Genomics &#226;&#8364;&#162; VOL    37 &#226;&#8364;&#162;   www.physiolgenomics.org <br />                                                           SALT STRESS IN DROSOPHILA                                                                        3 <br /> provides reliable estimates of fold change, significance, and false             RESULTS <br /> discovery rate (FDR), and a sharply improved performance in exper- <br /> iments with limited numbers of arrays (10). Significant changes were               Do flies eat salty food? Before studying the effects of dietary <br /> assessed for pairs of samples (salt vs. normal diet) at each time point,        salt loading, it is important to establish that the diet remains <br /> and gene ontology terms overrepresented in gene lists identified by             palatable; otherwise, the effects of salt loading will be ob- <br /> iterative group analysis (iGA) (9).                                             scured by those of starvation or desiccation. Accordingly, flies <br />    Quantitative PCR. Real-time quantitative reverse transcription               were exposed to diet labeled with blue dye and containing <br /> PCR (qPCR) was used for further validation of the microarray results.           various loadings of NaCl, for various times and scored for blue <br /> Seven-day-old adult flies were collected and placed 1) in normal food,          color in their guts. The results of this experiment can be seen <br /> 2) in 4% NaCl food, and 3) in vials empty of food. After 8 h, flies were        in Fig. 1B. After 1 h with the food containing no extra NaCl, <br /> collected and anaesthetized in ice, and the tissue of interest was              approximately half of the flies had blue dye in their gut (Fig. <br /> dissected. Total RNA was extracted (Qiagen RNeasy Kit) and cDNA                 1A). Flies in the food containing 4% or 5% NaCl also appeared <br /> prepared according to standard protocols (Invitrogen, Superscript II).          with blue dye in their gut, which means that they had also eaten <br /> Experiments were controlled against genomic DNA contamination                   the food. After 3 h, almost all of flies had blue dye in their guts. <br /> with a no-reverse-transcriptase sample and were normalized against              After 6 h, all the flies in the food without NaCl supplementa- <br /> rp49. The samples were replicated independently six to eight times.             tion, and at least half of those on 4 and 5% NaCl food show a <br /> Data were then expressed as fold difference of stress-treated samples <br />                                                                                 full blue gut (Fig. 1A), confirming that, although food intake is <br /> compared with controls &#226;&#171;&#190; SE. <br />                                                                                 reduced on high-salt diet, unpalatability is not a major issue. <br />    Cell type-specific RNA interference knockdown of gene expression. <br /> For downregulation of CG2196 expression, CG2196-RNA interfer- <br />                                                                                    Survival on high-NaCl diet. Having established that flies <br /> ence (RNAi) flies were crossed to the c42-GAL4 (tubule principal cell           continue to ingest diet containing high levels of NaCl, it is <br /> specific) or actin-GAL4/CyO (ubiquitous expression) driver lines, and           appropriate to examine survival. These results are shown in <br /> the offspring from the crosses were used to validate the extent of              Fig. 1C. From the graph it is evident that flies fed with 3 or 4% <br /> knockdown by qPCR. Six repeats were used for each of the different              NaCl food survive much better than those that are starved. <br /> fly lines, and the experiments ware repeated twice more (three                  These results are comparable to those reported previously (21), <br /> biological repeats) in total. Normalization of the results was per-             although our work reports a greater range of salt concentrations <br /> formed against rp49 controls.                                                   and monitors survival over a time course, rather than just at 4 <br />    In situ hybridizations. An expressed sequence tag (EST) clone of             days. Additionally, our wild-type stock survived marginally <br /> the CG2196 gene was obtained (GH19680 EDM1 133-6918917                          better on high salt; at 5% NaCl (wt/vol) equivalent to 0.85 M, <br /> Drosophila Gene Collection Clone 1) and used to prepare digoxyge-               50% of flies survived to 4 days compared with 0% reported <br /> nin-labeled DNA probes as described previously (18). In situ hybrid-            previously for 0.8 M (21). Although the difference is signifi- <br /> ization was performed as described previously (18).                             cant (probably reflecting strain differences), both experiments <br />    Survival of RNAi flies. RNAi male flies were crossed with c42                show a similar <span id='am-322' about='xsp:pattern' typeof='owl:Thing'>pattern</span> of reduction of survival with increasing <br /> virgin female flies in cages and for 10 days, eggs were collected each          NaCl concentration. Flies fed with NaCl food up to 5% survive <br /> morning, counted, and transferred to vials with food containing 4%              significantly better than the starved flies, but those exposed to <br /> NaCl. The percentage of flies surviving long enough to reach the adult          higher salt levels die almost as fast as flies deprived of food <br /> stage was compared with the parental lines. At least 12 vials from              altogether. This suggests that short-term salt-handling mecha- <br /> each line were collected. Two repeats of the full experiment were <br />                                                                                 nisms can offer partial protection against salt levels elevated by <br /> performed. <br />                                                                                 up to 5% for at least a few days. Together with the previous <br />    Generation of CG2196::eYFP-overexpressing flies. Flies overex- <br /> pressing a construct of the CG2196 gene tagged with enhanced yellow <br />                                                                                 results, a behavioral adaptation to high salt is apparent: insects <br /> fluorescent protein (eYFP) were generated using the Gateway system.             provided with high-salt diet continue to feed (and thus survive <br /> The entry and destination vectors used were obtained from the                   better than insects with no food) but feed less avidly than flies <br /> Drosophila Gateway Vector collection (37). The open reading frame               on normal diet. In the subsequent microarray time-course <br /> of the gene was obtained from an EST clone and inserted in-frame <br /> into the Gateway destination vector pTWV, so encoding fluorescently <br /> COOH-terminal tagged CG2196, and flies germ-line transformed as <br /> described previously. The construct is under control of the UAS <br /> promoter, so allowing cell-specific expression under control of appro- <br /> priate GAL4 driver lines. <br />    Expression in S2 cells. Drosophila S2 cells were grown in Schnei- <br /> der&#226;&#8364;&#8482;s medium according to standard protocols. Genes of interest were <br /> cloned the destination vector pAWV (Drosophila Gateway Vector <br /> Collection). This vector contains a constitutive ActinC5 promoter. S2 <br /> cells were transfected according to the Invitrogen protocol as de- <br /> scribed previously (32). Approximately 3 &#226;&#171;&#187; 106 cells were used in <br /> each transfection. The cells were harvested in PBS and were allowed <br /> to attach to poly-L-lysine-treated glass slides. Cells were counter- <br /> stained with the nuclear stain DAPI and washed repeatedly before <br /> being viewed in a Zeiss 510 Meta confocal microscope. <br />    Statistics. For fluid secretion experiments, significance of results <br /> was determined with an unpaired Student&#226;&#8364;&#8482;s t-test, taking the critical <br /> value of P as 0.05 (two-tailed). For microarray experiments, the FDR            Fig. 2. Numbers of genes up- and downregulated at different times from start <br /> for significance was taken as 5%.                                               of salt exposure, at 1 and 5% false discovery rate (FDR). <br />  <br />                                                 Physiol Genomics &#226;&#8364;&#162; VOL   37 &#226;&#8364;&#162;   www.physiolgenomics.org <br />  4                                                          SALT STRESS IN DROSOPHILA <br />  <br /> Table 1. Iterative Group Analysis of gene families up- and downregulated by dietary salt stress <br />                                                                                                                   % Gene Family Changed at Sample Time, h <br />  <br />                              Gene Ontology <span id='am-4' about='Thesaurus:code' typeof='owl:Thing'>Code</span>                                             Group            4               8             16               32 <br />  <br />                                                               A. Upregulated gene families <br /> 0008745-N-acetylmuramoyl-L-alanine amidase activity                                       immune             25 <br /> 0006955-immune response                                                                   immune             11 <br /> 0045087-innate immune response                                                            immune             14 <br /> 0042742-defense response to bacterium                                                     immune             14 <br /> 0050829-defense response to Gram-negative bacterium                                       immune             18 <br /> 0050830-defense response to Gram-positive bacterium                                       immune              9 <br /> 0019731-antibacterial humoral response                                                    immune             25 <br /> 0009253-peptidoglycan catabolic process                                                   immune             25 <br /> 0042834-peptidoglycan binding                                                             immune             25 <br /> 0030342-1-a,25-dihydroxyvitamin D3 (1,25-(OH)2D3) 24-hydroxylase activity                                   100 <br /> 0004558-alpha-glucosidase activity                                                        sugar              25              42             42               50 <br /> 0043169-cation binding                                                                                       10              10             16               14 <br /> 0004613-phosphoenolpyruvate carboxykinase (GTP) activity                                                    100             100            100              100 <br /> 0004611-phosphoenolpyruvate carboxykinase activity                                                          100             100            100              100 <br /> 0006094-gluconeogenesis                                                                   sugar                              40 <br /> 0017076-purine nucleotide binding                                                                                            40 <br /> 0004364-glutathione transferase activity                                                                                     13 <br /> 0008643-carbohydrate transport                                                            sugar                               9              9 <br /> 0005355-glucose transmembrane transporter activity                                        sugar                              18             24 <br /> 0005351-sugar:hydrogen ion symporter activity                                                                                11             14 <br /> 0006006-glucose metabolic process                                                         sugar                              27             27               27 <br /> 0005792-microsome                                                                                                                            4 <br /> 0009063-amino acid catabolic process                                                        AA                                              17 <br /> 0016798-hydrolase activity, acting on glycosyl bonds                                      sugar                                             11 <br /> 0005996-monosaccharide metabolic process                                                  sugar                                             22               17 <br /> 0009113-purine <span id='am-320' about='xsp:base' typeof='owl:Thing'>base</span> biosynthetic process                                                                                                    75               75 <br /> 0006144-purine base metabolic process                                                                                                        9               15 <br /> 0003939-L-iditol 2-dehydrogenase activity                                                                                                  100              100 <br /> 0006164-purine nucleotide biosynthetic process                                                                                              40               50 <br /> 0006189-&#226;&#8364;&#8482;de novo&#226;&#8364;&#8482; IMP biosynthetic process                                                                                                  43               57 <br /> 0006012-galactose metabolic process                                                       sugar                                             50               50 <br /> 0008652-amino acid biosynthetic process                                                     AA                                                               10 <br /> 0004477-methenyltetrahydrofolate cyclohydrolase activity                                                                                                    100 <br /> 0009072-aromatic amino acid family metabolic process                                        AA                                                               38 <br /> 0004488-methylenetetrahydrofolate dehydrogenase. (NADP&#226;&#171;&#185;) activity                                                                                           100 <br /> 0006559-L-phenylalanine catabolic process                                                   AA                                                               50 <br />                                                             B. Downregulated gene families <br /> 0005730-nucleolus                                                                                           12 <br /> 0008143-poly(A) binding                                                                                     20 <br /> 0005616-larval serum protein complex                                                                        33 <br /> 0005344-oxygen transporter activity                                                                         14 <br /> 0045735-<span id='am-1' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> reservoir activity                                                                         25 <br /> 0046870-cadmium ion binding                                                                                 67              67 <br /> 0005786-signal recognition particle, endoplasmic reticulum targeting                                                         27 <br /> 0004035-alkaline phosphatase activity                                                                                        23 <br /> 0005316-high affinity inorganic phosphate:sodium symporter activity                                                          22 <br /> 0005048-signal sequence binding                                                                                              75 <br /> 0006621-protein retention in ER                                                                                             100 <br /> 0050830-defense response to Gram-positive bacterium                                       immune                             17 <br /> 0045087-innate immune response                                                            immune                             15            10 <br /> 0006955-immune response                                                                   immune                             13             7 <br /> 0042742-defense response to bacterium                                                     immune                             16            11 <br /> 0050829-defense response to Gram-negative bacterium                                       immune                             27            18 <br /> 0019731-antibacterial humoral response                                                    immune                             32            21 <br /> 0005784-translocon complex                                                                                                   43            43 <br /> 0042562-hormone binding                                                                                                      67                             67 <br /> 0007321-sperm displacement                                                                 reprod.                           50                             75 <br /> 0045861-negative regulation of proteolysis                                                                                   50                             63 <br /> 0007610-behavior                                                                                                             13                             23 <br /> 0004182-carboxypeptidase A activity                                                                                                        17 <br /> 0007305-vitelline membrane formation in chorion-containing eggshell                        reprod.                                          57 <br /> 0007343-egg activation                                                                     reprod.                                         33 <br /> 0008316-structural constituent of vitelline membrane                                       reprod.                                         100 <br /> 0004181-metallocarboxypeptidase activity                                                                                                   33 <br />                                                                                                                                                    Continued <br />  <br />                                                   Physiol Genomics &#226;&#8364;&#162; VOL   37 &#226;&#8364;&#162;   www.physiolgenomics.org <br />                                                                  SALT STRESS IN DROSOPHILA                                                                        5 <br /> Table 1. &#226;&#8364;&#8221;Continued <br />                                                                                                                       % Gene Family Changed at Sample Time, h <br />  <br />                                 Gene Ontology <span id='am-321' about='Thesaurus:code' typeof='owl:Thing'>Code</span>                                              Group             4              8              16              32 <br />  <br /> 0004263-chymotrypsin activity                                                                                                                   18              23 <br /> 0004806-triacylglycerol lipase activity                                                                                                         11              16 <br /> 0008970-phospholipase A1 activity                                                                                                               30              20 <br /> 0016045-detection of bacterium                                                                immune                                                            33 <br /> 0004867-serine-type endopeptidase inhibitor activity                                                                                                            12 <br /> 0016298-lipase activity                                                                                                                                         16 <br /> 0046692-sperm competition                                                                     reprod.                                                           50 <br /> 0042600-chorion                                                                               reprod.                                                           38 <br /> 0005213-structural constituent of chorion                                                     reprod.                                                           78 <br /> 0007306-eggshell chorion formation                                                            reprod.                                                           36 <br />   Gene families that are significantly (by iterative group analysis, Ref. 9) overrepresented in either the up- or downregulated groups are ordered by time of onset <br /> and are highlit with the following names: immune, immune/defence response; reprod, reproduction; sugar, carbohydrate metabolism; AA, amino acid <br /> metabolism. Note that, because of the limitations and hierarchical nature of gene ontology annotation, there is substantial overlap between the genes reported <br /> under related headings. Nonetheless, the trends should be clear. <br />  <br />  <br /> experiment, shorter times (0 &#226;&#8364;&#8220; 8 h) can thus be taken as repre-                       expression of the synthetic enzyme trehalose 6-phosphate <br /> sentative of acute salt exposure, whereas longer time points                          synthase (tps1 and tps2), with a corresponding increase in <br /> (16 &#226;&#8364;&#8220;32 h) might be expected to show additional effects of                            trehalose levels. Interestingly, although no early changes in <br /> reduced food intake.                                                                  trehalose metabolism are found in the array, there are two <br />    Microarray of acute NaCl stress. From these results, a                             significant changes at 16 h: the synthetic enzyme trehalose <br /> microarray experiment was devised comparing matched sets of                           6-phosphate synthase 1 (tps1) is downregulated, whereas the <br /> flies transferred from normal diet to either normal or 4% salt                        catabolic &#226;&#65533;&#163;-trehalase (CG16965) is upregulated. Accord- <br /> diet and sampled after 4, 8, 16, and 32 h. The numbers of genes                       ingly, trehalose does not appear to be a key osmolyte in <br /> both up- and downregulated increased steadily with time, as                           response to acute salt stress in this insect. Rapid conversion <br /> summarized in Fig. 2. The changed genes are listed in Supple-                         of glucose to trehalose on passage through the insect midgut <br /> mentary Table S1,1 and iGA of all changed gene families is                            has classically been seen as a way of maintaining a favor- <br /> shown in Table 1. The iGA gene list shows some compelling                             able concentration gradient for glucose absorption from the <br /> features. When viewed as a time series, the response can be                           gut (38), so it is possible that, in insects, trehalose has lost <br /> resolved into several phases. Initially, there is a prominent                         its primitive role as an osmolyte. <br /> immune response, with multiple genes (metchnikowin; droso-                               Another key change in osmotically stressed worms is the <br /> cin; diptericin; attacin; immune-induced molecules 1, 10, and                         upregulation of chaperones of the heat shock protein (hsp) <br /> 23; PGRP-SB1) upregulated at 4 h and downregulated at 8 and                           family, presumably to maintain protein folding under changing <br /> 16 h. At 8 and 16 h, the major response is an upregulation of                         ionic strength. However, hsps are underrepresented in the <br /> genes associated with carbohydrate metabolism. Conspicu-                              change list in Drosophila (Supplementary Table S1): only <br /> ously, about half of the Drosophila &#226;&#65533;&#163;-glucosidase (maltase)                           hsp68 is found to change significantly and then only at a <br /> gene family is upregulated, together with an &#226;&#65533;&#163;-amylase and the                        single time point. The impression is thus gained that Dro- <br /> zinc finger transcription factor sugarbabe (sug), a transcription                     sophila, with its relatively sophisticated osmoregulatory <br /> factor that has previously been implicated in starvation and                          system, can defend its internal environment effectively <br /> sugar stress responses (45). Sugarbabe upregulation after feed-                       against dietary salt loading for an extended period (of <br /> ing is thought to downregulate sugar metabolism (45), so its                          several days), before death ensues. <br /> downregulation here is consistent with upregulation of carbo-                            Ion transport and salt loading. Ion transport genes, perhaps <br /> hydrate metabolism.                                                                   the most obvious candidates for change, are relatively mod- <br />    Sugar metabolism as a general response to salt loading. The                        estly represented; one putative sodium/halide cotransporter is <br /> marked changes in genes involved in carbohydrate metabolism                           upregulated, while another does not change hugely. However, <br /> are evocative of studies of high-salt diet in other species. In                       the major insect epithelial transport genes [Na&#226;&#171;&#185;, K&#226;&#171;&#185; ATPase, <br /> both yeast and C. elegans, salt loading causes the accumulation                       V-ATPase, Na&#226;&#171;&#185;/H&#226;&#171;&#185; exchangers, Na&#226;&#171;&#185;/K&#226;&#171;&#185;/2Cl&#226;&#171;&#186; cotransporters <br /> of glycerol as a protective osmolyte (25) (24). In yeast (24) the                     (40)] do not feature in the list. <br /> HOG stress pathway is under control of the Hog1 kinase (24).                             Which tissues are critical in salt response?. These mi- <br /> Although the Drosophila Hog1 homolog mpk2 does not change                             croarray experiments were performed on whole flies; al- <br /> significantly in our experiment (Supplementary Table S1), this <br />                                                                                       though this design reduces the sensitivity of the experiment <br /> does not preclude a posttranscriptional modulation of glycerol <br />                                                                                       to tissue-specific changes of expression (11), it also avoids <br /> accumulation, so we cannot absolutely exclude it as a mecha- <br />                                                                                       preconceptions as to the major tissues involved in salt <br /> nism in insects. <br />                                                                                       response. However, since the work was performed, the <br />    Trehalose is another key osmolyte in bacteria, yeast, and <br />                                                                                       FlyAtlas online resource has become available, allowing <br /> plants (17); and in C. elegans, osmotic stress increases <br />                                                                                       basal expression levels to be ascertained for any Drosophila <br />                                                                                       gene across 14 tissues and life stages (11). So, although our <br />   1 <br />       The online version of this article contains supplemental material.              original data were obtained from whole flies, it is possible to <br />                                                       Physiol Genomics &#226;&#8364;&#162; VOL   37 &#226;&#8364;&#162;   www.physiolgenomics.org <br />  6                                                                     SALT STRESS IN DROSOPHILA <br />  <br /> Table 2. The genes modulated by dietary salt are predominantly expressed in tubule and hindgut <br />                                                             Number of Genes With Maximal Expression in Tissue <br />  <br />     Treatment     Brain     Head      Crop      Midgut        Tubule      Hindgut      Ovary         Testis      Acc       l_fat         tag    car    Fly       Total       &#226;&#65533;&#185;2 <br />  <br /> All FlyAtlas     1,783      463      1,134      1,324           791           571      2,563         2,655     2,386       1,518         900   1,856   144      18,088 <br /> 4 h up               0        0          0          2            25            22          0             0         1           0           5       7     2          64        426 <br /> 8 h up               0        0          0          2            25            22          0             0         1           0           4       7     2          63        432 <br /> 16 h up              0        2          1         10            33            40          5             0         2           0          16      26     0         135        467 <br /> 32 h up              1        7          1         14            37            54          6             0         4           0          26      43     0         193        534 <br /> 4 h down             1        1          1          0             0             6          2             0         0           0           1       4     0          16         52 <br /> 8 h down             1       12          2          3            36            23         15             8         4          32          24      52   129         341      3,273 <br /> 16 h down            1        7          1          1            48            12          3             8         0          10           1       8     4         104        638 <br /> 32 h down            0        6          0          0            49            10          3            33         0          19           1       6     5         132        545 <br />   At each time point, each probe set with significantly changed (false discovery rate &#226;&#172;&#65533; 5%) expression was screened against the Flyatlas.org expression dataset, <br /> and the tissue with maximum expression of that probe set was identified. The total scores for each tissue were then compared with the profile obtained from all <br /> genes in FlyAtlas, and &#226;&#65533;&#185;2 was calculated. For 12 d.f., the critical value of &#226;&#65533;&#185;2 is 21, so the genes up- and downregulated at all time points are significantly <br /> nonrandomly drawn from the different tissues: specifically, excretory system (tubule and hindgut, shown in bold) genes are massively overrepresented. <br /> Abbreviations for tissues: tag, thoracicoabdominal ganglion; acc, accessory gland; car, carcass; l_fat, larval fat body; fly, whole fly. Details of the FlyAtlas dataset <br /> have been published previously (11). <br />  <br />  <br /> see from expression in FlyAtlas whether the 19 most sig-                                      sively upregulated in midgut, and sugarbabe is rather more <br /> nificantly changed genes (as called by MAS 5.0) are house-                                    widely expressed. The expression patterns of CG2196 and <br /> keeping (generally expressed), or whether they are tissue                                     CG6486 were validated further by qPCR, confirming the mas- <br /> specific, and whether any particular tissues emerge as com-                                   sive expression of these genes in the tubules and midgut <br /> mon candidates for salt response. The results (Table 2) are                                   respectively (Table 3B). Whole fly quantitative PCRs for the <br /> remarkable; not only are both up- and downregulated genes all                                 three genes were performed for the 8 h time point and com- <br /> highly tissue-specifically expressed, but they almost all show                                pared with the time-course microarray data (Fig. 3). In all <br /> highest expression in either the hindgut or Malpighian (renal)                                cases, there was reasonable agreement. CG2196 and CG6484 <br /> tubules, the two key osmoregulatory tissues of the excretory                                  were consistently and increasingly upregulated over time, <br /> cycle (4), with some others specific to the midgut. Our model                                 whereas sugarbabe was consistently downregulated. Thus, <br /> for response to high levels of salt, therefore, is a massive                                  based on this limited sample the array, FlyAtlas and qPCR data <br /> upregulation of carbohydrate metabolism (perhaps reflecting                                   are concordant. <br /> sharply increased energy demands) in the hindgut and Mal-                                        Role of CG2196/salty dog. CG2196 was selected for further <br /> pighian tubules, perhaps under control of sugarbabe (a tran-                                  study, as it was predicted to represent a sodium/halide sym- <br /> scription factor previously implicated in dietary stress re-                                  porter by the computer annotation of the Drosophila genome, <br /> sponse, and which is predominantly expressed in the midgut,                                   and because it was highly specifically expressed in the Mal- <br /> tubules, hindgut, and fat body), while other tissues are rela-                                pighian tubules, one of the key tissues for response to salt <br /> tively unaffected.                                                                            (Table 2). According to Drosophila tradition for whimsical but <br />    Validation of selected genes. To encompass a range of the                                  informative names, the gene was named salty dog (salt). <br /> responses induced by salt loading, three genes were selected                                  Firstly, the expression of salt within the tubule was established <br /> for further study: the SLC5 cotransporter CG2196 (annotated                                   by in situ hybridization (Fig. 4). Expression was seen in the <br /> in FlyBase as a sodium/halide symporter), the GLUT4/8-like                                    principal cells (not the stellate cells) along the <span id='am-5' about='xsp:length' typeof='owl:Thing'>length</span> of the <br /> sugar transporter CG6484, and the transcription factor sugarb-                                tubule but particularly in the main segment. The initial segment <br /> abe. According to FlyAtlas (Table 3A), basal expression of                                    was not stained. This localization is reasonable, because the <br /> CG2196 is virtually tubule-specific, whereas CG6484 is mas-                                   principal cells are the major active ion-transporting cell type in <br />  <br /> Table 3. Genes selected for further study <br />                                                                A. FlyAtlas expression levels across multiple tissues <br />  <br />      Gene         Brain       Head        Tag        Crop           Midgut          Tubule          Hindgut        Ovary           Testis      Acc     Car       L_fat        Fly <br />  <br /> CG2196              7           4          5            7                 8         7,248             70               4             6          11      21          5         275 <br /> CG6484              6          22          4            7             2,738            12             32               4             8         523     161          6         178 <br /> sugarbabe          10         197         10            8               584           455             95               5             6           8     371        550         137 <br />  <br />                                        B. qPCR validation of CG2196 and CG6484 expression in Malpighian tubules, guts, and heads <br />  <br />                                                                                              Relative Expression Level (w.r.t. rp49) <br />  <br />     Gene                                        Head                                                          Midgut                                         Malpighian tubules <br />                                                 &#226;&#171;&#186;5 <br /> CG2196                                 4.8&#226;&#171;&#187;10 &#226;&#171;&#190;1.2&#226;&#171;&#187;10&#226;&#171;&#186;5                                             3.6&#226;&#171;&#187;10&#226;&#171;&#186;5&#226;&#171;&#190;4.5&#226;&#171;&#187;10&#226;&#171;&#186;6                                        0.129&#226;&#171;&#190;0.008 <br /> CG6484                               2.38&#226;&#171;&#187;10&#226;&#171;&#186;7&#226;&#171;&#190;1.8&#226;&#171;&#187;10&#226;&#171;&#186;8                                             8.7&#226;&#171;&#187;10&#226;&#171;&#186;5&#226;&#171;&#190;1.5&#226;&#171;&#187;10&#226;&#171;&#186;5                                    3.3&#226;&#171;&#187;10&#226;&#171;&#186;8&#226;&#171;&#190;4.2&#226;&#171;&#187;10&#226;&#171;&#186;9 <br />   In A, the FlyAtlas expression profile across multiple tissues is shown. The maximal signal for each gene is shown in bold. Details of the FlyAtlas data set have <br /> been published previously (11). In B, data are shown as means &#226;&#171;&#190; SE (n &#226;&#171;&#189; 3 independent biological replicates). <br />  <br />                                                          Physiol Genomics &#226;&#8364;&#162; VOL       37 &#226;&#8364;&#162;   www.physiolgenomics.org <br />                                                                   SALT STRESS IN DROSOPHILA                                                                          7 <br />                                                                                        pressed. So at least in the S2 cells, the CG2196-eYFP <br />                                                                                        protein is a membrane protein that appears mainly in the <br />                                                                                        plasma membrane. <br />                                                                                           Homozygous CG2196-eYFP flies (insertion in the first chro- <br />                                                                                        mosome) were crossed with c42-GAL4 or Act5C-GAL4 flies, <br />                                                                                        and expression of the chimeric protein was assessed (Fig. 5). <br />                                                                                        These GAL4 lines drive expression of UAS constructs in either <br />                                                                                        tubule principal cells [where salt is normally expressed (Fig. <br />                                                                                        4)] or ubiquitously, respectively. When CG2196-eYFP is <br />                                                                                        driven in tubule principal cells of the main and lower segment <br />                                                                                        by c42-GAL4, it is directed to basolateral membrane infoldings <br />                                                                                        (Fig. 5, B and C). Since this promoter does not drive expression <br />                                                                                        in stellate cells or in the initial segment of the tubule, no <br />                                                                                        staining is seen in these. <br />                                                                                           With the Act5C-Gal4 promoter, the staining in the prin- <br />                                                                                        cipal cells is exactly the same as for the c42 promoter. The <br />                                                                                        progeny of the cross (Act5C-GAL4 flies) &#226;&#171;&#187; (CG2196-eYFP <br />                                                                                        flies) expresses the CG2196-eYFP construct in the principal <br />                                                                                        cells of the main and lower segment again in the foldings of <br />  <br />  <br />  <br />  <br /> Fig. 3. Time course of expression of selected genes and validation at 8 h. For <br /> each gene of interest (CG2196, sugarless, and CG6844), mean fold change <br /> (salt relative to normal diet) were plotted at 4, 8, 16, and 32 h. Bars with heavy <br /> lines are significantly different from no-change hypothesis at FDR &#226;&#172;&#65533;5%. Array <br /> fold changes are also compared with qPCR measurements (shaded boxes) at <br /> 8 h (means &#226;&#171;&#190; SE, n &#226;&#171;&#189; 3; where error bars are not visible, they are too small <br /> to plot). <br />  <br />  <br />  <br /> this tissue (15). No staining was seen in other tissues or with <br /> the sense control. <br />    Salt localizes to the basolateral plasma membrane. Dro-                             Fig. 4. In situ hybridizations for CG2196 in Malpighian tubules of adult <br /> sophila S2 cells were transiently transfected with a                                   Oregon R flies. Staining with anti-sense probe. A and B: clear staining appears <br /> salt::eYFP construct using the calcium phosphate transfec-                             in the ureter and in the principal cells of the main and the lower segment of <br /> tion method. The S2 cells present a clear staining around the                          both the posterior and anterior Malpighian tubules. In most cases, the staining <br />                                                                                        in the main segment of the tubule was stronger compared with that of the lower <br /> plasma membrane (Fig. 5A). Also, there seems to be stain-                              segment. The initial segment was not stained in both pairs of tubules. C and <br /> ing in vesicles or conglomerations of the protein inside the                           D: sense controls. In all photographs, the diameter of the tubule can be taken <br /> cell, as frequently observed when proteins are overex-                                 as 35 &#226;&#65533;&#174;m. <br />  <br />                                                       Physiol Genomics &#226;&#8364;&#162; VOL    37 &#226;&#8364;&#162;   www.physiolgenomics.org <br />  8                                                               SALT STRESS IN DROSOPHILA <br />  <br />  <br />  <br />  <br /> Fig. 5. Expression pattern of CG2196-eYFP in vitro and in vivo. A: Drosophila S2 cells were transfected with the CG2196::eYFP construct under control of <br /> an actin 5C promoter. There is clear staining in the plasma membrane and in vesicles or conglomerations of the protein inside the cell. B&#226;&#8364;&#8220;G: the CG2196::eYFP <br /> construct in flies. B: overview of expression driven in tubule principal cells by the c42 GAL4 line. C: the construct under the c42 promoter appears in the principal <br /> cells in the infoldings of the basolateral plasma membrane. Nuclei were counterstained blue with DAPI in A&#226;&#8364;&#8220;C. D&#226;&#8364;&#8220;F: the CG2196::eYFP construct under the <br /> Act5C-GAL4 line, which drives ubiquitous expression. Stellate cells of the middle segment do not express this construct (D). In E and F the focus on the cells <br /> differs; E was focused midway through the tubular epithelium, whereas F was focused at the top of the tubule. In all photographs, the diameter of the tubule <br /> can be taken as 35 &#226;&#65533;&#174;m. <br />  <br />  <br /> the basolateral membrane and perhaps in a part of the                                 iment are shown in Fig. 6. The construct produces a signif- <br /> endoplasmic reticulum, while in the stellate cells of the                             icant (Student&#226;&#8364;&#8482;s t-test, taking P &#226;&#172;&#65533; 0.05 as significant) <br /> middle segment no staining is observed (Fig. 5D). Thus salt                           knockdown in salt expression, in both Act5C&#226;&#172;&#381;salt-RNAi <br /> is expressed in the ion-transporting cells of the Malpighian                          (18.8 times lower than parental RNAi flies), and c42&#226;&#172;&#381;salt- <br /> tubule, on the basolateral membrane, consistent with its                              RNAi (17.7 times less than the salt expression in RNAi flies <br /> presumed function as an uptake symporter.                                             and 10.7 times less than the salt expression in c42 flies). <br />    Modulation of salt expression in tubules impacts on                                   Salt is upregulated in response to salt challenge, so does <br /> organismal survival after salt challenge. Flies transgenic for                        RNAi-mediated salt knockdown in just the tubule reduce <br /> a UAS-salt-RNAi construct were generated, and extent of                               survival of the whole organism when challenged with di- <br /> knockdown was assessed by qPCR. UAS-salt-RNAi flies                                   etary salt? The progeny of both overexpression and knock- <br /> were crossed with the Act5C-GAL4 and c42-GAL4 driver                                  down crosses to the principal cell-specific driver c42 was <br /> lines, and expression levels of salt in Malpighian tubules                            raised on food containing 4% NaCl, and survival compared <br /> compared with parental controls. The results of this exper-                           with parental controls. The results are shown in Fig. 7. As <br />                                                      Physiol Genomics &#226;&#8364;&#162; VOL    37 &#226;&#8364;&#162;   www.physiolgenomics.org <br />                                                                 SALT STRESS IN DROSOPHILA                                                          9 <br />                                                                                         Salt is a highly tubule-specific gene, expressed on the <br />                                                                                      basolateral plasma membrane of the tubule, and which is <br />                                                                                      strongly up regulated in response to dietary salt. However, <br />                                                                                      this response is inappropriate, because c42&#226;&#172;&#381;salt-RNAi flies <br />                                                                                      actually survive better on high salt. The most probable <br />                                                                                      reason for this behavior is that the overexpression of salt <br />                                                                                      under salt diet is a miscalculated response of the organism. <br />                                                                                      Such inappropriate responses are commonly encountered in <br />                                                                                      human health and disease: for example, the cytokine storm <br />                                                                                      elicited in healthy humans exposed to influenza can be more <br />                                                                                      dangerous than the original infection. We speculate that flies <br />                                                                                      might sense increasing Na&#226;&#171;&#185; levels associated with dietary <br />                                                                                      salt stress and increase Na&#226;&#171;&#185; excretion by the tubule by <br />                                                                                      upregulating basolateral Na&#226;&#171;&#185; uptake. However, it is possible <br />                                                                                      that the osmolyte cotransported with Na&#226;&#171;&#185; by salt is not well <br />                                                                                      handled by the tubule, and so RNAi against salt actually <br />                                                                                      improves the situation. <br />                                                                                         What does salt/salty dog actually encode? Although it is <br />                                                                                      evidently a member of the well-characterized SLC5 (sodi- <br />                                                                                      um/solute symporter) gene family (43, 44), the answer is <br /> Fig. 6. Quantification of CG2196-RNA interference (RNAi) induced knock-              surprisingly vague. The Drosophila genome project (fly- <br /> down of CG2196 mRNA levels. UAS-CG2196-RNAi flies were crossed with <br /> a ubiquitous (Act5C-GAL4) and Malpighian tubule principal cell-specific              base.bio.indiana.edu) annotates it as a sodium/iodide sym- <br /> (c42) GAL4 driver and tubule mRNA levels measured by qPCR relative to the            porter. An NCBI BLASTP search against human sequences <br /> rp49 reference gene for both parentals and progeny. Although Act5C-GAL4              produces a closest match (E value 3e-87) to SLC5A12, and <br /> flies have expression levels rather higher, and c42 slightly lower, than the         a second extremely close match (E value 2e-82) to SLC5A8. <br /> RNAi parents, both drive RNAi highly efficiently: CG2196 expression in <br /> Act5C-GAL4&#226;&#172;&#381;progeny is 19&#226;&#171;&#187; less, and in c42-GAL4&#226;&#172;&#381;RNAi progeny is 18&#226;&#171;&#187; <br />                                                                                      The former cotransports short chain fatty acids, lactate, and <br /> less, than in RNAi flies. RNAi/CyO control progeny express CG2196 at levels          nicotinate with sodium and is found in kidney, small intes- <br /> almost exactly the same as the RNAi flies. Data are expressed relatively to the      tine, and skeletal muscle; whereas the latter cotransports <br /> CG2196 expression of the CG2196-RNAi parental line and normalized against            short chain fatty acids, lactate, nicotinate, and iodide with <br /> the rp49 reference gene. *Significant change from control (the CG2196-RNAi           sodium and is found in thyroid, kidney, and intestine (43, <br /> parental line) (Student&#226;&#8364;&#8482;s t-test, taking P &#226;&#172;&#65533; 0.05 as significant). <br />                                                                                      44). However, CLUSTALX alignment of the protein se- <br />                                                                                      quences with the human SLC5 members (Fig. 8) places salt <br />                                                                                      between SLC5A6, a sodium/vitamin cotransporter, and <br /> can be seen from these results, in both repeats overexpres-                          SLC5A7, a sodium/choline cotransporter. Even this list is <br /> sion of salt had no consistent effect on survival. Surpris-                          not exhaustive, as SLC5 members are also thought to <br /> ingly, c42&#226;&#172;&#381;salt-RNAi flies survive better in food containing                         facilitate the transfer of water and organic solutes such as <br /> 4% NaCl than either parental line (RNAi or c42). The                                 urea. So, based on structure, salt can be considered to be <br /> percentage of RNAi-c42 flies surviving long enough is more                           a Na&#226;&#171;&#185;/solute cotransport but cannot be ascribed a particular <br /> than double of that of any of the parents, and this result is                        transport substrate. Clearly, then, the exact function of salty <br /> statistically significant (Student&#226;&#8364;&#8482;s t-test, two-tailed). So,                        dog will require extensive further work, as functional <br /> surprisingly, the upregulation of salt on salt challenge ob-                         workup of SLC5 family members is known to be problem- <br /> served by microarray and qPCR appears to be counteradap-                             atic (43, 44), However, as SLC5 members are uptake trans- <br /> tive.                                                                                porters, and salty dog is localized to the basolateral mem- <br />                                                                                      brane of the Malpighian tubule, its upregulation in response <br /> DISCUSSION                                                                           to dietary salt would clearly increase sodium clearance from <br />                                                                                      the blood. Although this overexpression does not appear <br />    These results provide the most comprehensive view on                              sufficient to confer enhanced survival on the organism on <br /> salt stress in an insect to date. Previous studies associated                        salt challenge (Fig. 7), the observation that knockdown of <br /> salt stress (NaCl and KCl) with the inebriated neurotrans-                           the gene is actually beneficial (Fig. 7) is intriguing. <br /> mitter/osmolyte transporter (ine) (21), and the studies of                              Perhaps the most compelling aspect of the work is the <br /> dehydration and rehydration (2, 19) demonstrated that Dro-                           clear phasing of different classes of response over the <br /> sophila melanogaster is a very strong osmoregulator. Here,                           experimental period (Table 2), particularly the early induc- <br /> dietary salt stress induced a clear and reproducible sequence                        tion, then repression, of an immune response. The tubule, <br /> of changes in gene expression involving immune/stress                                although classically considered to be fluid-transporting and <br /> response, carbohydrate metabolism, and ion transport path-                           osmoregulating tissue (14), actually plays multiple roles in <br /> ways. In the longer term (&#226;&#172;&#381;16 h), effects analogous to                               the organism (16). Recently, it has been shown to mount a <br /> starvation were seen, as the reproductive axis was shut down                         robust, autonomous immune response to pathogens (28), <br /> in both males and females. It is interesting to see sugarbabe,                       mediated by the canonical Toll/imd immune pathways (22). <br /> a transcription factor associated with dietary sugar loading,                        So the machinery of the immune response is already in <br /> also changing in this study; it suggests that sug may play a                         place in the tubules; it is intriguing that it is invoked so <br /> general role in response to dietary stress.                                          early in the salt stress response. We speculate that this find- <br />                                                     Physiol Genomics &#226;&#8364;&#162; VOL    37 &#226;&#8364;&#162;   www.physiolgenomics.org <br />  10                                                                SALT STRESS IN DROSOPHILA <br />  <br />  <br />  <br />  <br /> Fig. 7. Effect of modulation of tubule CG2196 <br /> expression on whole fly survival after salt chal- <br /> lenge. Flies were allowed to lay eggs on 4% NaCl <br /> diet, then the numbers of progeny reaching adult- <br /> hood were counted and compared with survival of <br /> RNAi, eYFP, or c42 parental lines. The c42 line <br /> drives expression selectively in tubule principal <br /> cells. For each experimental condition, 2 indepen- <br /> dent replicates are shown. A and B: tubule-specific <br /> RNAi against CG2196 enhances survival on 4% <br /> salt. C and D: tubule-specific overexpression of <br /> eYFP-tagged CG2196 has no significant impact on <br /> expression relative to parental controls. *Significant <br /> change from control (Student&#226;&#8364;&#8482;s t-test, taking P &#226;&#172;&#65533; <br /> 0.05 as significant). <br />  <br />  <br />  <br />  <br /> ing implies a generality in stress signaling, at least in this                              Of course, in nature Drosophila does not face quite the <br /> tissue; irrespective of the precise nature of the stress, a set                          problems of excess salt as, for example, do mosquito larvae <br /> of key adaptive genes is quickly upregulated, then selec-                                that live in hypersaline lakes. But, importantly, the results <br /> tively turned off as the nature of the stress becomes app-                               provide a baseline against which to assess the specialized <br /> arent.                                                                                   responses of insects whose evolutionary history has equipped <br />  <br />  <br />  <br />  <br /> Fig. 8. Which human SLC5 sodium/solute sym- <br /> porter is most similar to CG2196? The reference <br /> protein sequence for each of the 12 human SLC5 <br /> members was downloaded from NCBI, together with <br /> that for CG2196. They were aligned by CLUSTALX <br /> with default parameters, and a PHYLIP tree gener- <br /> ated with 1,000 bootstrapping. The tree was viewed <br /> in Treeview (30). <br />  <br />                                                                      11 <br /> them with the ability to osmoregulate and thrive under contin-                       <br /> uous osmotic stress.                                                                   <br />   <br /> </body></html>