<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Early Embryonic Gene Expression Profiling Zebrafish <br /> Prion Protein (Prp2) Morphants <br /> Rasoul Nourizadeh-Lillabadi, Jacob Seil&#195;&#184; Torgersen&#194;&#164;, Olav Vestrheim, Melanie Ko&#194;&#168;nig, Peter Alestro&#194;&#168;m, <br /> Mohasina Syed* <br /> Department Basic Sciences Aquatic Medicine, Norwegian School Veterinary Science, Oslo, Norway <br />  <br />  <br />  <br />      <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />      Background: The Prion protein (PRNP/Prp) plays crucial role transmissible spongiform encephalopathies (TSEs) like <br />      Creutzfeldt-Jakob disease (CJD), scrapie mad cow disease. Notwithstanding importance human animal <br />      disease, fundamental aspects PRNP/Prp function transmission remains unaccounted for. <br />  <br />      Methodology/Principal Findings: The zebrafish (Danio rerio) genome contains Prp encoding genes assigned prp1, <br />      prp2 prp3. Currently, second paralogue believed similar mammalian PRNP gene structure <br />      function. Functional studies PRNP gene ortholog addressed prp2 morpholino (MO) knockdown <br />      experiments. Investigation Prp2 depleted embryos revealed high mortality apoptosis 24 hours post fertilization <br />      (hpf) impaired brain neuronal development. In order elucidate underlying mechanisms, genome- <br />      wide transcriptome analysis carried viable 24 hpf morphants. The resulting changes gene expression profiles <br />      revealed 249 differently expressed genes linked biological processes like cell death, neurogenesis embryonic <br />      development. <br />  <br />      Conclusions/Significance: The current study contributes understanding basic Prp functions demonstrates that <br />      zebrafish excellent model address role Prp vertebrates. The gene knockdown prp2 indicates an <br />      essential biological function zebrafish ortholog morphant <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span>phenotype</span> suggests neurodegenerative <br />      action gene expression effects apoptosis related effects gene networks controlling neurogenesis and <br />      embryo development. <br />  <br />   Citation: Nourizadeh-Lillabadi R, Seil&#195;&#184; Torgersen J, Vestrheim O, Ko&#194;&#168;nig M, Alestro&#194;&#168;m P, et al. (2010) Early Embryonic Gene Expression Profiling Zebrafish Prion <br />   Protein (Prp2) Morphants. PLoS ONE 5(10): e13573. doi:10.1371/journal.pone.0013573 <br />   Editor: Sharyn Jane Goldstien, University Canterbury, New Zealand <br />   Received June 21, 2010; Accepted September 26, 2010; Published October 22, 2010 <br />   Copyright: &#195;&#376; 2010 Nourizadeh-Lillabadi et al. This open-access article distributed terms Creative Commons Attribution License, which <br />   permits unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: Financial support provided The FUGE Program, Research Council Norway (via Grant 159329/S10 KO-ZFISH), EU project: TSE IN FISH (nr. <br />   QLK5-2002-0086). The funders role study design, data collection analysis, decision preparation manyscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: Mohasina.Syed@nvh.no <br />   &#194;&#164; Current address: Nofima Marin, Aas, Norway <br />  <br />  <br />  <br />  <br /> Introduction                                                                           functional relationship binding partner(s) remains <br />                                                                                        unaccounted for. A list PRNPC candidate binding partner <br />    The molecular mechanisms responsible prion pathogenesis                         proteins major proposed functions PRNPC <br /> share similarities wider group neurodegenerative                             itself summarized review Westergard et al. [4], <br /> disorders. The genetic tractability zebrafish (Danio rerio) makes                   including protection apoptosis oxidative stress, copper <br /> it suitable model addressing fundamental molecular                           ions uptake binding, synaptic signal transduction linker <br /> mechanisms underlying neurodegenerative disorders prion                        extracellular matrix. <br /> disease [1], Alzheimer disease [2] Parkinson&#226;&#8364;&#8482;s disease [3].                           The PRNP gene phylogenetically conserved, teleost <br />    In prion theory Prusiner [1] transmissible spongiform                        genomes harbor multiple prp paralogues extra genome <br /> encephalopathies proposed caused proteinaceous                          duplication. Of zebrafish PRNP genes, prp3 more <br /> infectious agent, pathological variant normal cellular prion                  divergent variant [5], prp1 prp2 closely resemble <br /> protein, PRNP scrapie (PRNPSc; named PrPSc), acts                        mammalian variant prp2 likely closest <br /> a template conversion normal cellular prion protein                         ortholog [6]. Miesbauer et al. [7] demonstrated PrP-related <br /> PRNPC (also named PrPC) PRNPSc. There however, major                            proteins zebrafish complexly glycosylated contain a <br /> unresolved issues prion research relating                               glycosylphosphatidylinositol anchor. It appears nomen- <br /> fundamental biological questions prion gene function                        clature prion protein variants corresponding genes can <br /> mechanisms transmission disease. PRNP                                  confusing (Table S1). In paper chosen use prp1, <br /> glycosylated glycosylphosphatidylinositol (GPI) anchored mem-                          prp2, prp3 Prp1, Prp2, Prp3 zebrafish genes and <br /> brane protein, function, cellular distribution polarized                   proteins respectively. For mammalian homolog denote the <br /> sorting cell remain definitively established. Also                    normal variant gene protein PRNP PRNP (PrPC) <br />  <br />  <br />        PLoS ONE | www.plosone.org                                                  1                              October 2010 | Volume 5 | Issue 10 | e13573 <br />                                                                                                                       Zebrafish Prp2 Morphants <br />  <br />  <br /> respectively. The misfolded pathological PRNP scrapie variant          120 249 genes genes mammalian ortholog <br /> termed PRNPSc (PrPSc).                                                    identifiers (Table S2). The remaining 129 genes either <br />    <span id='am-162' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-163' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-239' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span>Knockout</span> mice lacking neuronal cell surface PRNP protein           zebrafish specific non-annotated genes hypothetical or <br /> show normal development behavior [8]. On contrary, prp1           unknown function (Table S3). The upregulated gene was <br /> mRNA knockdown zebrafish results strong embryonal                 cyclin G1 (CCNG1) 4.2 fold change. Leucine rich repeat <br /> phenotype, characterized absence embryonic cell adhesion            neuronal 1 gene (LPRN1) downregulated gene a <br /> and arrested gastrulation [9]. Similarly, prp2 morphant zebrafish         minus 2.46 fold change. The majority differentially <br /> also developed <span id='am-318' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-319' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-320' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-321' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-322' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-323' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-324' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-325' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-326' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-327' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-328' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-330' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-331' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-332' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-333' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-334' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-335' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-336' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-337' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-338' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-341' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-342' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-344' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-345' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-346' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-347' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-348' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-349' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-350' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-351' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-352' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-353' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-354' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-355' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-356' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-357' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-358' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-359' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-362' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-363' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-364' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-365' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-366' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-367' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-368' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-369' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-370' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-371' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-372' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-373' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-374' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-375' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-376' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-378' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-379' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-380' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-381' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-382' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-383' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-384' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-385' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-386' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-387' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-389' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-390' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-391' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-392' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-393' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-394' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-395' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span>lethal phenotype</span> later larval stage 7        expressed genes encoding enzymes transcription regula- <br /> days post-fertilization (dpf) [9]. In study          tors location cytoplasm nucleus, respectively. <br /> demonstrated zebrafish Prp mouse PRNP mRNAs                 The IPA organization differentially expressed genes into <br /> could rescue knockdown phenotype, indicating evolution-            biological function disease clusters included &#226;&#8364;&#732;&#226;&#8364;&#732;cell death&#226;&#8364;&#8482;&#226;&#8364;&#8482;, <br /> ary conserved Prion protein function.                                     &#226;&#8364;&#732;&#226;&#8364;&#732;embryonic development&#226;&#8364;&#8482;&#226;&#8364;&#8482; &#226;&#8364;&#732;&#226;&#8364;&#732;nervous development and <br />    The apparent difference mouse <span id='am-240' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-241' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-246' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-249' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-253' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-267' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-273' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-279' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-290' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-309' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-316' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span>gene knockout</span>            function&#226;&#8364;&#8482;&#226;&#8364;&#8482; (Fig. S1, Table 2). The &#226;&#8364;&#732;&#226;&#8364;&#732;cell death/apoptosis&#226;&#8364;&#8482;&#226;&#8364;&#8482; related <br /> zebrafish models suggests important aspects Prion             gene cluster largest sub-cluster total 32 genes, of <br /> protein biology revealed zebrafish model, using             5 related apoptosis neural cells: TP53, TP73, and <br /> morpholino (MO) mediated translation blocking [10]. Aiming             BAX (upregulated) SNPR1 SNCB (downregulated, Fig. 4A, <br /> uncovering function zebrafish prp2 gene, MOs (prp2-            Table S2). The cluster &#226;&#8364;&#732;&#226;&#8364;&#732;embryonic development&#226;&#8364;&#8482;&#226;&#8364;&#8482; (Fig. S1, <br /> MO1 prp2-MO2) targeting prp2 mRNA designed               Table 2) contained 11 genes, IPA join 9 each <br /> and injected cell stage zebrafish embryos. The resulting         network sub-clusters SHH only <br /> phenotypes characterized microarray studies                 downregulated gene BMP4 highest score the <br /> conducted 24 hpf zebrafish embryos microinjected              upregulated (Fig. S2). Both SHH TP53 included of <br /> most potent MOs, prp2-MO2. In order confirm             sub-clusters directly involved developmental process of <br /> microarray data demonstrate MOs showed             embryonic cell lines. Analysis genes related &#226;&#8364;&#732;&#226;&#8364;&#732;nervous system <br /> specificity, quantitative real-time PCR (qRT-PCR) performed           development function&#226;&#8364;&#8482;&#226;&#8364;&#8482; (Fig. S1, Table 2) revealed 5 sub- <br /> on mRNA selected genes prp2-MO1 prp2-MO2                 functional clusters involved developmental processes. <br /> injected embryos.                                                         Among clusters, development brain, forebrain, neuroep- <br />                                                                           ithelial cells, neurons central nervous (CNS) were <br /> Results                                                                   present (Fig. 4B). Here TP73, AKTIP BMP4 (upregulated) with <br />                                                                           SHH (downregulated) play central role TP53 acting on <br />    Optimization MOs specific 59UTR                  CNS BAX neuron development (Fig. 4B). In cluster <br /> GenBank sequence AJ620614.1 (Fig. 1) injections revealed             &#226;&#8364;&#732;&#226;&#8364;&#732;nervous development function&#226;&#8364;&#8482;&#226;&#8364;&#8482;, IPA connected 13 of <br /> 0.2 mM prp2-MO2 resulted 71% mortality 24 hpf 74%               differentially expressed genes network of <br /> at 48 hpf. In order reduce mortality, 0.15 mM prp2-MO2             different sub-function clusters. The prominent network was <br /> was chosen led mortality rate 50%. Phase            sub-function cluster &#226;&#8364;&#732;&#226;&#8364;&#732;neurogenesis&#226;&#8364;&#8482;&#226;&#8364;&#8482;, contained TP53, <br /> contrast microscopy Prp2 morphants revealed clear                    BMP4, MEF2C BAX (upregulated) SOX11, <br /> phenotype midbrain hindbrain developmental defects               GFAP, ACHE SHH (downregulated) (Fig. S2). Also, the <br /> as compared control group (Fig. 2 B&#226;&#8364;&#8220;D). In order        &#226;&#8364;&#732;&#226;&#8364;&#732;differentiation neuroglia&#226;&#8364;&#8482;&#226;&#8364;&#8482; &#226;&#8364;&#732;&#226;&#8364;&#732;migration neurons&#226;&#8364;&#8482;&#226;&#8364;&#8482; <br /> characterize phenotypic effects, wholemount immunofluores-            harbored 5 genes each. <br /> cence carried using antibodies Prp2 HNK-1.               In order explicitly investigate differentially <br /> Fluorescence microscopy analysis wild type embryos 24           expressed genes prp2-MO2 injected embryos be <br /> hpf unveiled low significant levels Prp2 proteins           directly associated prion gene (PRNP/prp2) functions, a <br /> telencephalon, eye trigeminal ganglion (Fig. 3A). Investigation       customized IPA network prion prion associated genes <br /> of HNK-1 showed strong staining trigeminal ganglion            generated include 18 upregulated 13 <br /> neuronal extensions (Fig. 3B). In morphants Prp2 activity          downregulated genes, mammalian prp2 ortholog PRNP <br /> could detected Zn12 antibody visualized altered                direct relation MMP2 indirect relation BAX and <br /> trigeminal ganglion morphology reduced number periph-              TP53 (Fig. 5). However, PRNP (prp2) itself other <br /> eral neurons head (Fig. 3C). Also, nuclear staining           key molecules predicted network the <br /> DAP1 revealed apoptotic cells hyper-condensed nuclei          differentially expressed genes submitted microarray <br /> trigeminal ganglion, eye, olfactory placode, midbrain-         dataset (Fig. 5) showed change expression qRT-PCR <br /> hindbrain area (Fig. 3D).                                                 (Fig. 6; See discussion clarification). <br />    Microarray analysis using 16k oligonucleotide library revealed          Validation microarray results qRT-PCR showed a <br /> that 249 genes significantly differentially expressed         high Pearson&#226;&#8364;&#8482;s correlation coefficient (0.998) micro- <br /> prp2-MO2 samples (Table 1). IPA analysis identified mapped            array qRT-PCR data (Fig. 6). Further, corresponding <br />  <br />  <br />  <br />  <br /> Figure 1. Morpholino sequences. Morpholino sequences (prp2-MO1and prp2-MO2 sequences underlined) used knockdown zebrafish <br /> PrP2 gene function. <br /> doi:10.1371/journal.pone.0013573.g001 <br />  <br />  <br />       PLoS ONE | www.plosone.org                                      2                         October 2010 | Volume 5 | Issue 10 | e13573 <br />                                                                                                                             Zebrafish Prp2 Morphants <br />  <br />  <br />  <br />  <br /> Figure 2. prp2-MO2 morphant 24 hpf larvae phenotypes. Larvae immobilized CyGEL prior microscopy. A) Non-injected wt control. <br /> B&#226;&#8364;&#8220;D) individual morphants displaying defective midbrain hindbrain development. <br /> doi:10.1371/journal.pone.0013573.g002 <br />  <br />  <br />  <br />  <br /> Figure 3. Whole mount immunofluorescence analysis 24 hpf wild type (A B) Prp2 morphants (C D). Neuronal <br /> structures visualized Zn12 antibody (Yellow), Prp2 detected rabbit anti serum (Red). A) In wild type 24 hpf Prp2 is <br /> observed trigeminal ganglion telencephalon, present neuronal tissue. B) Neuronal structures visualized HNK-1 staining <br /> (Zn-12 antibody) specimen A. C) Aberrant morphology trigeminal ganglion Prp2 morphants (arrow) reduced <br /> number peripheral neurons visualized HNK-1 staining. D) Non deconvolved single plane image morphant condensed nuclei (inside <br /> stippled ring). Abbreviations: eye (e), telencephalon (t) trigeminal ganglion (tg). <br /> doi:10.1371/journal.pone.0013573.g003 <br />  <br />  <br />        PLoS ONE | www.plosone.org                                         3                         October 2010 | Volume 5 | Issue 10 | e13573 <br />                                                                                                                                     Zebrafish Prp2 Morphants <br />  <br />  <br />  <br />  Table 1.                                                                              qRT-PCR expression data strongly support assumption a <br />                                                                                        gene specific action prp2 specific MOs used. In <br />                                                                                        addition, change prp1 gene expression observed qRT- <br />  A                                                                                     PCR analysis control and/or Prp2 morphants (data <br />                                                                                        shown). <br />  Total number genes                        249 145q 104Q <br />  B                                                                                     Discussion <br />  Mapped IPA                                120 69q 51Q <br />                                                                                           In current study, targeted prp2 mRNA translation <br />  Not present IPA                           129 76q 53Q <br />                                                                                        using different targeting MOs, prp2-MO1and prp2-MO2. <br />  A) The number significantly differentially expressed genes prp2-MO2            The significantly weaker effect mortality which <br />  injected 24hpf zebrafish embryos. B) The number differentially expressed IPA       attributed different binding site prp2 mRNA <br />  mapped genes (mammalian homologs) unmapped IPA (no mammalian                   (Fig. 1). A difference prp2-MO induced mortality compared <span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span> <br />  homologs found). Arrows represent up- down-regulated genes respectively. <br />                                                                                        Malaga-Trillo reported explained different <br />  doi:10.1371/journal.pone.0013573.t001 <br />                                                                                        morpholinos [9]. Quantification gene expression the <br />                                                                                        morphants prp2-MO1 prp2-MO2 qRT-PCR (Fig. 6) <br /> qRT-PCR analysis prp2-MO1 morphant transcriptome,                               implies observed changes gene expression reliable <br /> using 14 gene probe sets, resulted comparable                            respect technical quality microarray data se <br /> <span id='am-396' about='xsp:pattern' typeof='owl:Thing'>pattern</span> gene expression mentioned results                              respect gene target specificity prp2-MO2 chosen <br /> (Pearson&#226;&#8364;&#8482;s correlation coefficient 0.92). Hence,                            microarray experiment. Investigations corresponding <br />  <br />  Table 2. List IPA biological function clusters differentially expressed genes 24hpf prp2-MO2 injected zebrafish morphant <br />  embryos. <br />  <br />  <br />  Category                                             P-value                               Molecules <br />  <br />  Cellular Development                                 1,77E-05-9,52E-03                     SHH, BMP4, CTGF, TP73, AKTIP, SDC4, FOXD3, CD164, STMN1, HBB <br />                                                                                             (includes EG:3043), HMGB1 (includes EG:3146), ASPM, ANXA1, <br />                                                                                             CRIP2, HNF4A, COL18A1, ECT2, RASGRP2, TP53, TNC, PTP4A3, BHLHB5, <br />                                                                                             ACHE, MMP2, BAX, MLLT1, TYRP1, INSM1, PRKCI, CD9, ABCC5, MEF2D, <br />                                                                                             IHH, ALDOA, MEF2C, NPM1 (includes EG:4869), HMGB3 <br />  Developmental Disorder                               2,15E-05-7,67E-03                     TP53, SHH, BMP4, SLC31A1, DKC1, HSPB8, KAL1, ACHE, MMP2, BAX, <br />                                                                                             COL1A2, PRKCI, LAMA4, CRYBA4, PHGDH, ACSL4, IHH <br />  Cell Death                                           2,39E-05-9,8E-03                      SHH, CTGF, BMP4, SLC31A1, CCNI, TP73, AKTIP, HSPB8, SDC4, MDH1, <br />                                                                                             MYLK, CCNG1, EGLN1, STMN1, HMGB1 (includes EG:3146), ANXA1, <br />                                                                                             ACSL4, IGFBP1, SNCB, COL18A1, RASGRP2, TP53, TNC, ACHE, MMP2, BAX, <br />                                                                                             PRKCI, SWAP70, NDEL1, MCM2, CD9, ABCC5, MEF2D, S1PR1, ALDOA, <br />                                                                                             IHH, MEF2C, NPM1 (includes EG:4869), HMGB3 <br />  Cellular Assembly Organization                   3,34E-05-8,36E-03                     CTGF, CKMT1B, SDC4, CKB, STMN1, HMGB1 (includes EG:3146), <br />                                                                                             GFAP, HNF4A, COL18A1, TP53, TNC, CKM, PTP4A3, ACHE, MMP2, BAX, <br />                                                                                             PRKCI, SWAP70, MCM2, CD9, LAMA4, H2AFX, S1PR1, GPM6A, NPM1 <br />                                                                                             (includes EG:4869) <br />  Cardiovascular System Development                4,76E-05-7,67E-03                     TP53, SHH, CTGF, BMP4, TP73, MMP2, COL1A2, EGLN1, LAMA4, S1PR1, <br />  Function                                                                                   IHH, MEF2C, COL18A1, ECT2 <br />  Gene Expression                                      5,06E-05-7,67E-03                     TP53, PRKCI, HMGB1 (includes EG:3146), TP73, MEF2D, ATF7IP, EIF5, <br />                                                                                             MEF2C, BAX, NPM1 (includes EG:4869), HNF4A <br />  Cell Cycle                                           5,83E-05-9,15E-03                     TP53, SHH, BMP4, CCNI, TP73, BAX, CCNG1, SESN1, STMN1, MCM2, <br />                                                                                             ANXA1, S1PR1, HNF4A, ECT2, NPM1 (includes EG:4869) <br />  Immunological Disease                                5,83E-05-8,41E-03                     TP53, SHH, SWAP70, CD9, HMGB1 (includes EG:3146), TP73, AKTIP, <br />                                                                                             ANXA1, ACHE, BAX <br />  Embryonic Development                                1,74E-04-7,67E-03                     TP53, SHH, SLC31A1, BMP4, ABCC5, PTP4A3, IHH, MMP2, BAX, HNF4A, <br />                                                                                             COL18A1 <br />  Nervous System Development Function              1,74E-04-8,23E-03                     TP53, SHH, BMP4, TP73, AKTIP, ACHE, BAX, STMN1, EGLN1, CD9, NDEL1, <br />                                                                                             HMGB1 (includes EG:3146), LAMA4, PHGDH, S1PR1, IHH, COL18A1 <br />  Organ Development                                    1,74E-04-8,36E-03                     TP53, SHH, BMP4, TP73, AKTIP, MMP2, BAX, FOXD3, TYRP1, EGLN1, HBB <br />                                                                                             (includes EG:3043), S1PR1, PHGDH, IHH, MEF2C, IGFBP1 <br />  Neurological Disease                                 5,74E-04-8,71E-03                     RNF10, SHH, CTGF, TNNI2, HSPB8, SDC4, MDH1, EEF1G, TUBB2B, CKB, <br />                                                                                             SPON1, STMN1, ASPM, PHGDH, LAMB1, ACSL4, GFAP, SNCB, HNF4A, <br />                                                                                             TP53, ACHE, MMP2, BAX, LRRN1, SLC14A1, S1PR1, PDLIM1, UCK2, NPM1 <br />                                                                                             (includes EG:4869) <br />  Organismal Development                               6,42E-04-7,67E-03                     TP53, SHH, BMP4, SLC31A1, TP73, PTP4A3, ACHE, MMP2, BAX, PRKCI, <br />                                                                                             HMGB1 (includes EG:3146), SLC14A1, IHH, IGFBP1, COL18A1 <br />  Cell Signaling                                       7,34E-04-4,17E-03                     TP53, CKM, ANXA1, CKMT1B, BAX, COL18A1 <br />  <br />  Unbolded molecules upregulated bolded molecules downregulated. <br />  doi:10.1371/journal.pone.0013573.t002 <br />  <br />  <br />  <br />        PLoS ONE | www.plosone.org                                                  4                         October 2010 | Volume 5 | Issue 10 | e13573 <br />                                                                                                                       Zebrafish Prp2 Morphants <br />  <br />  <br />  <br />  <br /> Figure 4. IPA cluster analysis significantly differentially expressed genes. IPA cluster analysis significantly differentially expressed <br /> genes 24 hpf zebrafish prp2-MO2 morphant embryos. The clusters reveal genes involved (A) cell death apoptosis functions focused <br /> on apoptosis neural cells (B) genes involved nervous development. Red color indicates up- green downregulation. <br /> doi:10.1371/journal.pone.0013573.g004 <br />  <br /> prp1 mRNA levels qRT-PCR showed unaltered levels               involved cell death, 32 sub-cluster of <br /> paralogue morphants, suggests Prp1 2            apoptosis (Fig. S1and 4A). Cyclin G1 (CCNG1) gene the <br /> proteins gained different main functions zebrafish. The           highest level upregulated expression experiment known <br /> lethal effect prp2 knockdown agreement proposal         control TP53 TP73 involved cell cycle control <br /> of apoptosis-protective role PRNPC protein [11],        induction apoptosis [12], processes are <br /> confirmed hyperdense nuclei observed DAP1 stained                   augmented present study. Furthermore, TP53 TP73 <br /> morphants (Fig. 3D). The 24 hpf prp2-MO2 morphant tran-                   S1PR, BAX SNCB directly influence apoptosis <br /> scriptome (Table S2 S3) reveals cluster 39 genes            cortical neurons, neuroepithelial, neuroblastoma, neural <br />  <br />  <br />       PLoS ONE | www.plosone.org                                      5                         October 2010 | Volume 5 | Issue 10 | e13573 <br />                                                                                                                          Zebrafish Prp2 Morphants <br />  <br />  <br />  <br />  <br /> Figure 5. A network connecting differentially expressed genes. A network connecting differentially expressed genes zebrafish <br /> 24 hpf prp2-MO2 morphant embryos identified IPA PRNP, APP, heparin EP300 occupy central role. The uncolored protein <br /> symbols, including PRNP, added IPA network gaps analyzed dataset differentially expressed genes. Red color <br /> indicates up- green downregulation. A significant prp2 mRNA down-regulation experimentally demonstrated qRT-PCR (see Fig. 6). <br /> doi:10.1371/journal.pone.0013573.g005 <br />  <br />  <br />  <br />  <br /> Figure 6. Comparison microarray data qRT-PCR data. Microarray data (blue staples), qRT-PCR RNA 24 hpf prp2-MO2 <br /> injected embryos (red staples) 24 hpf prp2-MO1 injected embryos (yellow staples) presented fold change expression 14 genes. Mean <br /> fold change (6SD) qRT-PCR based triplicate embryo pools. <br /> doi:10.1371/journal.pone.0013573.g006 <br />  <br />  <br />        PLoS ONE | www.plosone.org                                       6                         October 2010 | Volume 5 | Issue 10 | e13573 <br />                                                                                                                         Zebrafish Prp2 Morphants <br />  <br />  <br /> precursor nervous tissue cells (Fig. 4A). The altered                       It conceivable used prp2-MO gene knockdown <br /> expressions genes role apoptosis neural           strategy used mimics loss normally folded prion <br /> cells [13,14] support putative role Prp2 control            protein (PRPN/Prp2), prion induced pathogen- <br /> apoptosis neural cells zebrafish development. For example,            esis. It does resemble increase misfolded prion <br /> the SNCB protein reported decrease apoptosis                protein aggregation high molecular weight fibers <br /> TSM1 neurons [13] S1PR-null mice dramatic increase               characterize prion disease. In study sporadic Creutzfelt- <br /> in apoptosis decrease mitosis developing nervous           Jakob disease (sCJD), 79 upregulated 275 downregulated <br /> system [14].                                                                genes identified [28]. The major alterations sCJD brain <br />    Both observed phenotype morphology (Fig. 2B C)               samples included upregulation genes encoding immune and <br /> impaired neurogenesis trigeminal ganglion peripheral             stress-response factors elements involved cell death cell <br /> neurons (Fig. 3C D) morphants correlated           cycle, prominent downregulation genes encoding <br /> observed changes gene expression, clustered functions                 synaptic proteins. There obvious common pattern of <br /> associated embryo development development brain,                differentially expressed genes respective clusters or <br /> forebrain central nervous (Fig. 4B S2). Two               gene lists providing evidence loss the <br /> these key molecules, sonic hedgehog (SHH) bone morphoge-                normally folded prion protein function major cause of <br /> netic protein 4 (BMP4), downregulated. SHH acts              prion induced pathogenesis [28]. Of 230 genes sCJD gene <br /> signal molecule tissues including midline structures         list, 4 common Prp2 morphants: GFAP, <br /> the brain, spinal cord thalamus zona limitans                    phosphoglycerate dehydrogenase (PHGDH), SWAP-70 protein <br /> intrathalamica [15&#226;&#8364;&#8220;17]. SHH reported ability act           (SWAP70) TP53. SWAP70, PHGDH TP53 included in <br /> axonal guidance [18] neurogenesis                    IPA apoptosis cluster (Fig. 4A B). When extending the <br /> influences proliferation neuroblasts granule cell precursors         comparisons differentially expressed genes include two <br /> [19,20] (Fig. S2). Similar SHH, BMP4 involved                 mouse scrapie models [29,30] 1 gene (GFAP) common for <br /> biological processes including negative control neurogenesis          4 studies comparing 3 mammalian (GFAP, <br /> olfactory epithelial cells brain/forebrain development [21].         ABCA1). GFAP proposed IPA indirectly linked PRNP <br /> However, SHH BMP4 involved                       neurogenesis (Fig. 5; S2). Comparing prp2 morphant with <br /> biological processes, exemplified presence 11 9             mouse scrapie model [29] revealed 5 genes common: GFAP, <br /> respectively 14 IPA defined biological function clusters          CD9 antigen (CD9), solute carrier family 14 member 1 (SLC14A1), <br /> this study (Table 2), investigations needed pinpoint         heat shock 22kDa protein 8 (HSPB8) syndecan 4 (SDC4). An <br /> more specific effect-relationships. In addition SHH BMP4,            IPA cluster analysis link 5 genes neurological <br /> several genes associated processes neurogenesis,          disorders (Fig. S3). CD9 HSPB8 included IPA apoptosis <br /> SOX11, GFAP ACHE, downregulated prp2 knockdown                  cluster (Fig. 4A). When comparing mouse study against <br /> embryos (Fig. S2). SOX11 suggested function                    sCJD data [29,30] overlapping gene numbers 11&#226;&#8364;&#8220;14 <br /> developing nervous lack glial fibrillary acidic             far 4&#226;&#8364;&#8220;5 zebrafish-sCJD zebrafish- <br /> protein (GFAP) gene function <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span>knock-out</span> mice increases                mouse, taken consideration fact 120 249 the <br /> outgrowth axons neurons spinal cord [22] (GFAP            zebrafish genes known mammalian homologs (Table 1). With <br /> central IPA cluster related neurological disorders. See            background fair state zebrafish prp2 morphants <br /> below Fig. S3). ACHE reported essential              provide good model shed light normal biological <br /> normal dendrite axon formation hippocampal neurons               function prion protein. Our gene expression study based <br /> may function excitatory synapse development, plasticity              depleting mammalian prion protein ortholog the <br /> remodeling [22&#226;&#8364;&#8220;24]. The present microarray qRT-PCR                      developing zebrafish embryo able link the <br /> analyses demonstrate upregulation BAX gene,            zebrafish prp2 gene biological processes including cell death, <br /> pro-apoptotic protein required neurotrophin-deprived neuro-             embryo development neurogenesis. Although genes <br /> nal apoptosis [25]. Overall, prp2 knockdown zebrafish embryos            overlap comparing 3 mammalian scrapie related transcrip- <br /> suggests impaired neurogenesis (Fig.&#226;&#8364;&#8482;s 2, 3 4B)             tomes prp2 gene knockdown zebrafish transcriptome, is <br /> concert proposed function PRNP [26,27]. IPA PRNP                interesting gene linked variety of <br /> network analysis applied differentially expressed genes          neurological disorders common 4 studies (Fig. S3). Many <br /> <span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span> prp2-MO2 24 hpf transcriptome revealed                 genes associated processes uncovered still <br /> direct indirect connections PRNP (Fig. 5). The postulated            distinct mechanism action Prp2 remains found. <br /> connection PRNP observed upregulation                We conclude data study, paper <br /> pro apoptotic genes BAX TP53 indicate apoptosis                  Malaga-Trillo et al. [9], place zebrafish highly relevant <br /> protective role Prp2, proposed Aguzzi et al [11]            model address largely unanswered questions the <br /> agreement pronounced mortality effect observed             molecular mechanisms underlying prion protein function, aggre- <br /> study. The proposed IPA network includes connec-               gation corresponding disease pathology. <br /> tions PRPN, heparin phosphatidylserine 6 <br /> genes present dataset associated          Materials Methods <br /> with 30 differentially expressed genes (Fig. 5). With the <br /> microarray technology used (a 16k library coverage 20&#226;&#8364;&#8220;25 k           Zebrafish prp2-MO microinjections <br /> genes), obvious gene products affected MO            Newly fertilized embryos obtained zebrafish reared <br /> treatment included presented data set. Next generation           Alestrom Zebrafish Lab http://zebrafish.no/ (http://zebrafish. <br /> sequencing (RNA-seq) provide powerful tool            no) according husbandry standards described Alestrom <br /> used uncover magnitude details complete               Lab Standard Operating Procedures. As AAALAC (http:// <br /> transcriptome, including transcripts, alternative splice isoform        www.aaalac.org) accredited laboratory, activities <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> are <br /> variants microRNAs, allow deeper understanding gene             subject evaluation IACUC. In experiments <br /> networks involved prion disease.                                         presented, zebrafish embryos age 24 hpf used. <br />  <br />  <br />       PLoS ONE | www.plosone.org                                        7                         October 2010 | Volume 5 | Issue 10 | e13573 <br />                                                                                                                           Zebrafish Prp2 Morphants <br />  <br />  <br /> For titration dose response, MOs specific 59UTR              Data deposition <br /> GenBank sequence AJ620614.1 (Fig. 1) injected single cell                Microarray data submitted European <br /> embryos following concentrations: 0.1, 0.15, 0.5 1mM.               Bioinformatics Institutes Array Express accessible exper- <br /> The mortality rate negative control embryos (non-injected              iment accession number E-MEXP-2365. <br /> mock injected 16 Danieu solution) 20% (data not <br /> shown).                                                                        Linear RNA amplification, target labeling and <br />                                                                                hybridization <br /> Method immobilizing embryos microscopy                                    One mg total RNA linearly amplified labeled, using an <br />    Prp2 morphants larvae embedded CyGEL SustainTM                      Amino Allyl MessageAmpTM II aRNA amplification kit (Ambion <br /> (Biostatus Limited, Leicestershire, UK) investigated                   TX, U.S.A). Five mg resulting amplified RNA (aRNA) from <br /> phenotype using Nikon AZ 100 fitted Nikon Digital Sight                 MO-treated control groups labeled Cy3- <br /> DS Ri1 camera. Morphants wild type embryo fixed                    dUTP Cy5-dUTP (GE Healthcare CT, U.S.A). The labeled <br /> 4% PFA transferred methanol immunofluorescence.                     targets examined amplification yield incorporation <br />                                                                                efficiency measuring aRNA concentration 260 nm, Cy3 <br /> Whole mount immunofluorescence                                                 incorporation 550 nm Cy5 incorporation 650 nm using <br />    Wild type morphant embryos dechorinated fixed                  Nanodrop ND-1000. One 5 mg labeled target aRNA <br /> in 4% PFA dehydrated 70% methanol. Fluorescent                     mixed 9 mL 256 fragmentation buffer (Agilent <br /> immunohistochemistry carried rehydrated                         Technologies) final volume adjusted 225 mL with <br /> permeabilized embryos (10 min 1% Triton X-100, Sigma)                    RNAse free H2O followed incubation 30 min 60uC. The <br /> a 2 hr blocking 5% dry milk dissolved 16Phosphate Buffered               hybridization solution prepared adding 220.5 mL 26 <br /> Saline Tween 20 (PBST; 16 PBS, 0.1% Tween). Antibodies                         hybridization buffer (Agilent) 4.5 mL (10 mg/mL) sonicated <br /> applied included HNK-1 specific Zn12 [31] (Developmental                   herring sperm DNA (Promega WI, U.S.A) labeled target <br /> studies Hybridoma Bank, IA, USA) rabbit serum                    aRNA. Microarray slides pre-hybridized 42uC, 60 min <br /> Prp2 [32]. The used 200 fold dilution 2% dry                using 0.1% Bovine Serum Albumin Fraction V, 56SSC 0.1% <br /> milk 1% DMSO 16 PBST, diluted                   SDS. Hybridization performed 60uC 16 hours using <br /> 1006. After night incubation 4uC primary                      Agilent gasket slides G2534-60003, hybridization chamber and <br /> antibodies, embryos washed thoroughly 16PBST                   oven according manufacturer (Agilent Technologies). <br /> incubated Alexa conjugated secondary antibodies diluted                   Microarray slides washed 36 5 min 0.56 SSC and <br /> 2006 (Invitrogen, Ca, USA) 2 hours. Final 16 PBST washes                   0.01% SDS wash 42uC room <br /> was carried glycerol mounting microscopy. All                   temperature. Finally, slides washed 3 times room <br /> images captured using Zeiss Axioplan Z1 fitted                 temperature 0.066 SSC dried immediately by <br /> Apotome structured illumination microscopy. The                     centrifuging 1000 rpm 1 minute. <br /> images post processed deconvolved visualized 3D <br /> using Zeiss Axiovison software.                                                Zebrafish oligonucleotide library microarray <br />                                                                                construction <br /> RNA isolation                                                                     The Zebrafish OligoLibraryTM (Compugen, Ontario, Canada) <br />    Total RNA injected control (16 Danieau solution)                   used composed 16,399 (16k) 65-mer oligonucleotide probes, <br /> MO injected 24 hpf zebrafish embryo isolated microarray                originally representing 16,228 clusters predicted representing <br /> and qRT-PCR analysis. Briefly, pools 50 embryos treated                unique genes plus 171 positive control probes. All probe sequences <br /> with 1 mL Trizol reagent (Invitrogen, CA, U.S.A). Homoge-                   selected 39 UTR order cover maximum <br /> nization carried using MagNA Lyser Beads (Roche                        number splice variants. The oligonucleotides printed on <br /> Diagnostics Gmbh, Mannheim, Germany) total RNA                             CMT UltraGAPS slides Norwegian Microarray Consortium <br /> isolation performed according TRIzol manufacturer                   (NMC, www.microarray.no ) Microgrid II (BioRobotics, <br /> (Invitrogen), followed 15 min DNaseI treatment (Qiagen                    Oslo). Probe annotation information Unigene ID, <br /> Hilden, Germany) 25uC. Further RNA purification                         Unigene description, Gene symbol, GO ontology human and <br /> conducted using RNAeasy mini kit (Qiagen). After purification,                 mouse homologue ID descriptions obtained the <br /> the samples eluted 50 mL RNase-free water                       Unigene build release, using Genome institute Singapore <br /> aliquoted microarray qRT-PCR analyses. RNA yield                   Unigene &amp; Gene Ontology Annotation Tool Genome institute of <br /> integrity determined using NanoDrop ND-1000 instrument                  Singapore (http://giscompute.gis.a-star.edu.sg/,govind/unigene_ <br /> (NanoDrop Technologies, DE, U.S.A) Agilent 2100 bioana-                    db/). GenBank accession numbers used query. In <br /> lyzer (Agilent Technologies, CA, U.S.A.) respectively. None             present study annotations using Danio rerio Build <br /> samples showed signs degradation impurities (260/280                 #107-58. <br /> 260/230 .1.8, RIN .8.0). <br />                                                                                Pathway analysis functional profiling <br /> Microarray strategy                                                               For pathway biological function analysis significantly <br />    The total RNA zebrafish embryo morphant pools                      differentially expressed genes, Ingenuity Pathway Analysis <br /> compared total RNA Danieau-injected control pools. The                 (IPA) program used. The results analyzed using the <br /> replicate embryo pools analyzed sets dye-swaps making           Unigene &amp; Gene Ontology Annotation Tool identification of <br /> a total hybridizations. Normalized microarray datasets did         mammalian (human, mouse rat) orthologs IPA <br /> result significantly altered gene expression zebrafish housekeep-       software analysis. The output results comprise gene networks, <br /> ing gene b-actin2 (172 probes arrays). Like microarray            pathways functional clusters, given scores p-values <br /> gene expression profiling studies, reproducible high correla-         based numbers uploaded genes cluster network <br /> tion microarray qPCR data [33].                                    size cluster network IPA knowledge database. <br />  <br />  <br />        PLoS ONE | www.plosone.org                                          8                        October 2010 | Volume 5 | Issue 10 | e13573 <br />                                                                                                                                    Zebrafish Prp2 Morphants <br />  <br />  <br />  <br />  Table 3. qRT-PCR primers (59-39) used validate microarray b-actin. <br />  <br />  <br />  GenBank            Symbol         Unigene Descp                                                        Forward Primer               Reverse Primer <br />  <br />  AI959372           ccng1          Cyclin G1                                                            attgaggatcagcacgagtatg       aaccaggtctccagctttaaca <br />  U60804             Tp53           Tumor protein p53                                                    ctgaagtggtccgcagatg          ttgccctccactcttatcaaa <br />  BI891654           bax            Bcl2-associated X protein                                            ccgtgagatcttctctgatgg        gtcaggaaccctggttgaaa <br />  BI878036           sesn1          Sestrin 1                                                            gctgctcaccaaagaacaca         cacagcatggatcagctcag <br />  AI353083           hbae3          Hemoglobin alpha embryonic-3                                         atcggccgtgagactcttt          ggagagttggggcttaggtc <br />  U14590             ascl1b         Achaete-scute complex-like 1b (Drosophila)                           ccgatgaaggaagctacgag         caatctttagtagccattgaagcac <br />  BG302998           aldh9          Similar Aldehyde dehydrogenase 9 family, member A1 like 1         ataacacaacctttggcttgg        gttcgcagcaactctgtgg <br />  AI964216           rps9           Ribosomal protein S9                                                 ggaggttgcagactcaggtc         gggatgttcacaacctgctta <br />  BE693186           hbae1          Hemoglobin alpha embryonic-1                                         aaagtcatccttccacaatgagt      ggggtagacaatcaacatcctg <br />  AF082662           hbbe1          Hemoglobin beta embryonic-1                                          ctccacgtagatcctgacaactt      tgaacttcaggtgtgaatccag <br />  AF116539           iclp2          Invariant chain-like protein 2                                       agtgttgggcccttaattcc         ttcactcgccatttttcctc <br />  BI840762           gpm6aa         Similar Glycoprotein M6A                                          aatcacaggacaaggcaaca         tcagacagcactccatacatcc <br />  AF180888           pvalb2         Parvalbumin 2                                                        gacaacagcggcttcattg          actctagcgttggccttgaa <br />  AY438684.1         prp2           Prion protein 2                                                      caatcgcccaagagaggac          caagagccagacagagcaatag <br />  AY438683.1         prp1           Prion protein 1                                                      cgcttcttcaacctttttatgg       gcctttctttcctcctgatgt <br />  AF025305.1         b-actin        b-actin                                                              aaggccaacagggaaaagat         gtggtacgaccagaggcatac <br />  <br />  doi:10.1371/journal.pone.0013573.t003 <br />  <br />  <br /> Fisher&#226;&#8364;&#8482;s exact test used determine statistical significance              GenePix imported Bioconductor package LIMMA <br /> and likelihood random clustering pathway. Scores 2                   corrected background. For array array <br /> or higher 99% likelihood generated                  normalization, print tip Loess scale used. An empirical <br /> chance alone. Mammalian orthologs IPA displayed using                      Bayes moderated t-test [34&#226;&#8364;&#8220;37] applied detect differentially <br /> human homolog identifiers zebrafish genes proteins                      expressed genes treated control samples. P-values were <br /> are annotated according standard zebrafish nomenclature                          corrected multiple testing using Benjamini&#226;&#8364;&#8220;Hochberg (BH) <br /> guidelines (Table S1).                                                              method [38] p-values,0.05 selected differentially <br />                                                                                     expressed genes. The generated gene list filtered for <br /> Quantitative real-time PCR                                                          genes low intensity small changes expression. In <br />    The accuracy microarray results validated analyzing               averaged normalized MA-Plot, majority genes were <br /> 14 differentially expressed genes qRT-PCR using                         clustered M-values 60.4 (fold change 61.3) and <br /> samples total RNA. The intron spanning qRT-PCR primers                      selected threshold criteria differentially expressed <br /> designed using &#226;&#8364;&#732;&#226;&#8364;&#732;Universal Probe Library Assay Design Center/                       gene list. Another threshold criterion A-values,6, represent- <br /> ProbeFinder version 2.40&#226;&#8364;&#8482;&#226;&#8364;&#8482; ((http://www.roche-applied-science.                      ing weak intensity spots, excluded the <br /> com/sis/rtpcr/upl/adc.jsp). Primer sequences summarized                      final gene list. Relative gene expression values (qRT-PCR) for <br /> Table 3. Template cDNAs produced 1 mg total RNA                           target gene expressed mean fold change obtained <br /> using Superscript III Reverse Transcriptase according manufac-                   3 replicate samples. Correlation microarray and <br /> turer (Invitrogen). qRT-PCR analysis carried                           qRT-PCR data 14 genes analyzed using Pearson product- <br /> LightCycler 2.0 instrument using LightCyclerHFastStart DNA                          moment correlation coefficient. <br /> MasterPLUS SYBR Green I (Roche). All reactions performed <br /> in triplicate 0.5 mL respective primers (10 pmol)                   Supporting Information <br /> 2.0 mL enzyme mix total reaction volume 10 mL. The PCR <br /> conditions were: 95uC 10min followed 40 cycles 10 s                    Figure S1 IPA generated functional clusters differentially <br /> 95uC, 10 s 60uC 6 s 72uC. Melting curve analysis                      expressed genes. IPA generated functional clusters differentially <br /> agarose gel electrophoresis carried confirm PCR products.                expressed genes 24 hpf zebrafish prp2 morphant (prp2-MO2) <br /> Data analysis crossing point (Cp) determined use                 embryos. All clusters p-values,0.05 ranked decreasing <br /> maximum-second-derivative function LightCyclerHSoftware                      p-values. Corresponding p-values illustrated Table 3. The <br /> version 4.0 (Roche). Relative gene expression levels obtained                  numbers pie diagram corresponds numbers genes with <br /> using beta-actin internal reference gene efficiency                      significantly different expression cluster. The clusters appear in <br /> correction external standards. The mean gene expression values                 order diagram ID pane. <br /> for group (exposed versus control) compared p-values                  Found at: doi:10.1371/journal.pone.0013573.s001 (0.24 MB TIF) <br /> calculated using unpaired Student&#226;&#8364;&#8482;s t-test.                                         Figure S2 IPA cluster analyses significant differentially <br />                                                                                     expressed genes. IPA cluster analyses significant differentially <br /> Data analysis statistical methods                                               expressed genes 24 hpf zebrafish Prp2 morphant embryos. The <br />    Microarray slides scanned using GenePix 4000B (Axon                       cluster reveals genes involved nervous function. Red <br /> instrument, Foster City, CA, U.S.A) threshold level just                       color indicates up- green downregulation. <br /> before saturation spots. Raw data generated                         Found at: doi:10.1371/journal.pone.0013573.s002 (2.60 MB TIF) <br />  <br />  <br />        PLoS ONE | www.plosone.org                                               9                          October 2010 | Volume 5 | Issue 10 | e13573 <br />                                                                                                                                                      Zebrafish Prp2 Morphants <br />  <br />  <br /> Figure S3 Comparative analysis differentially expressed                                   Acknowledgments <br /> genes. Comparative analysis differentially expressed genes <br /> between prp2-MO2 morphant mouse scrapie model                                      We acknowledge Dr. Evgenia Salta (Department Pharmacology, <br />                                                                                              Aristotle University Thessaloniki, Thessaloniki, Greece) kindly <br /> [29] reveals 5 common genes: GFAP, CD9, HSPB8, SDC4 and <br />                                                                                              providing rabbit antiserum Prp2 Dr. Jon Hildahl for <br /> SLC14A1. The functional clustering IPA indicates central                                constructive remarks. <br /> role genes neurological disorders: Huntington disease, <br /> Parkinson disease, retinopathy, Alexander disease, motor neuron <br /> disease, hydrocephalus hypertrophy astrocytes. <br />                                                                                              Author Contributions <br /> Found at: doi:10.1371/journal.pone.0013573.s003 (0.42 MB TIF)                                Conceived designed experiments: RNL JST PA MS. Performed <br />                                                                                              experiments: RNL JST OV MK. Analyzed data: RNL. <br /> Table S1     Nomenclature relationship prion protein                             Contributed reagents/materials/analysis tools: PA MS. Wrote paper: <br /> genes proteins. The arguments prp2/Prp2                                    RNL. Conceived study: MS. Participated experimental design: <br /> mammalian PRNP/PRP homolog summarized. To simplify                                        MS. Discussions analysis results preparation and <br /> reading used old conventional names (in bold)                             discussions manuscript: MS. Carried microarray qRT-PCR <br /> article Table S1.                                                                     analysis (vet lab): RN. Design microarray experiment, statistical <br /> Found at: doi:10.1371/journal.pone.0013573.s004 (0.04 MB                                     analysis data microarray qRT-PCR: RN. Interpretation the <br /> DOC)                                                                                         biological functions based differentially expressed genes drafted the <br />                                                                                              manuscript: RN. Contributed design morpholino: JST. <br /> Table S2 Fold change differentially expressed genes mapped                                Experimental set-up: JST. Sequencing Danio rerio Prion Protein (PrP) <br /> by IPA 24 hpf prp2-MO2 MO-injected zebrafish embryos.                                     gene: JST. Whole mount Immunofluorescence: JST. Discussions of <br /> Found at: doi:10.1371/journal.pone.0013573.s005 (0.15 MB                                     manuscript: JST. Participated microinjection titration of <br /> DOC)                                                                                         morpholino: OV. Participated microinjection, titration morpho- <br />                                                                                              lino, immobilizing embryos, microscopy performance of <br /> Table S3    Fold change differentially expressed genes                              microarray qRT-PCR analysis: MK. Participated discussions <br /> were mapped IPA 24 hpf prp2-MO2 injected zebrafish                                 analysis results preparation and <br /> embryos. These genes annotated zebrafish gene symbols                               discussions manuscript: PA. <br /> obtained Genome Institute Singapore. <br /> Found at: doi:10.1371/journal.pone.0013573.s006 (0.15 MB <br /> DOC) <br />  <br />  <br /> References <br />  1. Prusiner SB (1982) Novel proteinaceous infectious particles cause scrapie.               17. Scholpp S, Wolf O, Brand M, Lumsden A (2006) Hedgehog signalling the <br />     Science 216: 136&#226;&#8364;&#8220;144.                                                                        zona limitans intrathalamica orchestrates patterning zebrafish dienceph- <br />  2. Newman M, Musgrave IF, Lardelli M (2007) Alzheimer disease: amyloidogen-                     alon. Development 133: 855&#226;&#8364;&#8220;864. <br />     esis, presenilins animal models. Biochim Biophys Acta 1772: 285&#226;&#8364;&#8220;297.             18. Charron F, Stein E, Jeong J, McMahon AP, Tessier-Lavigne M (2003) The <br />  3. Bretaud S, Allen C, Ingham PW, Bandmann O (2007) p53-dependent neuronal                      morphogen sonic hedgehog axonal chemoattractant collaborates with <br />     cell death DJ-1-deficient zebrafish model Parkinson&#226;&#8364;&#8482;s disease.                       netrin-1 midline axon guidance. 1. Cell 113: 11&#226;&#8364;&#8220;23. <br />     J Neurochem 100: 1626&#226;&#8364;&#8220;1635.                                                              19. Dahmane N, Altaba A (1999) Sonic hedgehog regulates growth and <br />  4. Westergard L, Christensen HM, Harris DA (2007) The cellular prion protein                    patterning cerebellum. Development 126: 3089&#226;&#8364;&#8220;3100. <br />     (PrP(C)): physiological function role disease. Biochim Biophys Acta           20. Dahmane N, Sanchez P, Gitton Y, Palma V, Sun T, Beyna M, et al. (2001) The <br />     1772: 629&#226;&#8364;&#8220;644.                                                                               Sonic Hedgehog-Gli pathway regulates dorsal brain growth tumorigenesis. <br />  5. Cotto E, Andre M, Forgue J, Fleury HJ, Babin PJ (2005) Molecular                             1. Development 128: 5201&#226;&#8364;&#8220;5212. <br />     characterization, phylogenetic relationships, developmental expression               21. Shou J, Rim PC, Calof AL (1999) BMPs inhibit neurogenesis mechanism <br />     patterns prion genes zebrafish (Danio rerio). FEBS J 272: 500&#226;&#8364;&#8220;513.                     involving degradation transcription factor. 2. Nat Neurosci 2: 339&#226;&#8364;&#8220;345. <br />  6. Rivera-Milla E, Oidtmann B, Panagiotidis CH, Baier M, Sklaviadis T, et al.               22. Menet V, Prieto M, Privat A, Ribotta M (2003) Axonal plasticity functional <br />     (2006) Disparate evolution prion protein domains distinct origin               recovery spinal cord injury mice deficient glial fibrillary acidic <br />     Doppel- prion-related loci revealed fish-to-mammal comparisons.                       protein vimentin genes. 1. Proc Natl Acad Sci U S A 100: 8999&#226;&#8364;&#8220;9004. <br />     FASEB J 20: 317&#226;&#8364;&#8220;319.                                                                     23. Jay P, Goze C, Marsollier C, Taviaux S, Hardelin JP, Koopman P, et al. (1995) <br />  7. Miesbauer M, Bamme T, Riemer C, Oidtmann B, Winklhofer KF, et al. (2006)                     The human SOX11 gene: cloning, chromosomal assignment tissue <br />     Prion protein-related proteins zebrafish complex glycosylated                   expression. Genomics 29: 541&#226;&#8364;&#8220;545. <br />     contain glycosylphosphatidylinositol anchor. Biochem Biophys Res Commun                24. Olivera S, Rodriguez-Ithurralde D, Henley JM (2003) Acetylcholinesterase <br />     341: 218&#226;&#8364;&#8220;224.                                                                                promotes neurite elongation, synapse formation, surface expression of <br />  8. Bueler H, Fischer M, Lang Y, Bluethmann H, Lipp HP, et al. (1992) Normal                     AMPA receptors hippocampal neurones. 1. Mol Cell Neurosci 23: 96&#226;&#8364;&#8220;106. <br />     development behaviour mice lacking neuronal cell-surface PrP                  25. Deckwerth TL, Elliott JL, Knudson CM, Johnson EM, Jr., Snider WD, et al. <br />     protein. Nature 356: 577&#226;&#8364;&#8220;582.                                                                (1996) BAX required neuronal death trophic factor deprivation and <br />  9. Malaga-Trillo E, Solis GP, Schrock Y, Geiss C, Luncz L, et al. (2009) Regulation             development. Neuron 17: 401&#226;&#8364;&#8220;411. <br />     embryonic cell adhesion prion protein 2. PLoS Biol 7: e55.                     26. Steele AD, Emsley JG, Ozdinler PH, Lindquist S, Macklis JD (2006) Prion <br /> 10. Nasevicius A, Ekker SC (2000) Effective targeted gene &#226;&#8364;&#732;knockdown&#226;&#8364;&#8482; zebrafish.              protein (PrPc) positively regulates neural precursor proliferation during <br />     8. Nat Genet 26: 216&#226;&#8364;&#8220;220.                                                                    developmental adult mammalian neurogenesis. 1. Proc Natl Acad <br /> 11. Aguzzi A, Baumann F, Bremer J (2008) The prion&#226;&#8364;&#8482;s elusive reason being. 2.                Sci U S A 103: 3416&#226;&#8364;&#8220;3421. <br />     Annu Rev Neurosci 31: 439&#226;&#8364;&#8220;477.                                                           27. Santuccione A, Sytnyk V, Leshchyns&#226;&#8364;&#8482;ka I, Schachner M (2005) Prion protein <br /> 12. Jost CA, Marin MC, Kaelin WG, Jr. (1997) p73 simian [correction                      recruits neuronal receptor NCAM lipid rafts activate p59fyn to <br />     human] p53-related protein induce apoptosis. 3. Nature 389: 191&#226;&#8364;&#8220;194.                enhance neurite outgrowth. 1. J Cell Biol 169: 341&#226;&#8364;&#8220;354. <br /> 13. da Costa CA, Masliah E, Checler F (2003) Beta-synuclein displays                      28. Xiang W, Windl O, Westner IM, Neumann M, Zerr I, et al. (2005) Cerebral <br />     antiapoptotic p53-dependent phenotype protects neurons 6-hydro-                     Gene Expression Profiles Sporadic Creutzfeldt&#226;&#8364;&#8220;Jakob Disease. Ann Neurol <br />     xydopamine-induced caspase 3 activation: cross-talk alpha-synuclein                 58: 242&#226;&#8364;&#8220;257. <br />     implication Parkinson&#226;&#8364;&#8482;s disease. 1. J Biol Chem 278: 37330&#226;&#8364;&#8220;37335.                    29. Hwang D, Lee IY, Yoo H, Gehlenborg N, Cho J-H, et al. (2009) A systems <br /> 14. Mizugishi K, Yamashita T, Olivera A, Miller GF, Spiegel S, et al. (2005)                     approach prion disease. Molecular Systems Biology 5: Article number 252; <br />     Essential role sphingosine kinases neural vascular development. 8. Mol            doi:10.1038/msb. <br />     Cell Biol 25: 11113&#226;&#8364;&#8220;11121.                                                               30. Sorensen G, Medin <br /> </body></html>