<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>A Newly Identified Essential Complex, Dre2-Tah18, <br /> Controls Mitochondria Integrity Cell Death after <br /> Oxidative Stress Yeast <br /> Laurence Vernis1*, Ce&#194;&#180;line Facca1, Emmanuelle Delagoutte1, Nicolas Soler1, Roland Chanet1, Bernard <br /> Guiard2, Ge&#194;&#180;rard Faye3, Giuseppe Baldacci1 <br /> 1 Re&#194;&#180;gulation la re&#194;&#180;plication l&#226;&#8364;&#8482;ADN des eucaryotes, UMR2027 CNRS/Institut Curie, INSERM, Universite&#194;&#180; Paris-Sud, Orsay, France, 2 CNRS, Centre Ge&#194;&#180;ne&#194;&#180;tique <br /> Mole&#194;&#180;culaire, Gif-sur-Yvette, France, 3 Stabilite&#194;&#180; des ge&#194;&#180;nomes et stress ge&#194;&#180;notoxiques, UMR2027 CNRS/Institut Curie, Orsay, France <br />  <br />  <br />  <br />      <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />      A mutated allele essential gene TAH18 previously identified laboratory genetic screen new proteins <br />      interacting DNA polymerase delta yeast [1]. The present work shows Tah18 plays role response <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> <br />      oxidative stress. After exposure lethal doses H2O2, GFP-Tah18 relocalizes mitochondria controls <br />      mitochondria integrity cell death. Dre2, essential Fe/S cluster protein homologue human anti-apoptotic <br />      Ciapin1, identified molecular partner Tah18 absence stress. Moreover, Ciapin1 able replace yeast <br />      Dre2 vivo physically interacts Tah18. Our results favour oxidative stress-induced cell death yeast <br />      involves mitochondria controlled newly identified Dre2-Tah18 complex. <br />  <br />   Citation: Vernis L, Facca C, Delagoutte E, Soler N, Chanet R, et al. (2009) A Newly Identified Essential Complex, Dre2-Tah18, Controls Mitochondria Integrity and <br />   Cell Death Oxidative Stress Yeast. PLoS ONE 4(2): e4376. doi:10.1371/journal.pone.0004376 <br />   Editor: Martin G. Marinus, University Massachusetts Medical School, United States America <br />   Received October 7, 2008; Accepted December 25, 2008; Published February 5, 2009 <br />   Copyright: &#195;&#376; 2009 Vernis et al. This open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: This work supported financially french CNRS, INSERM Institut Curie. NS recipient French CNRS post-doctoral fellowship. The funders <br />   role study design, data collection analysis, decision publish, preparation manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: laurence.vernis-beringue@curie.u-psud.fr <br />  <br />  <br />  <br /> Introduction                                                                          prevented cycloheximide, potent translation inhibitor [4]. <br />                                                                                       Such apoptotic markers described exposure to <br />    Apoptosis cell death program allowing homeostatic                             external stresses NaCl [7,8], UV [9], high levels of <br /> equilibrium multicellular organisms. It key process                        yeast mating pheromone alpha factor [10], the <br /> eliminate damaged cells aging following cellular injuries,                  absence external stress, mutated cdc48 context. CDC48 <br /> or extra cells developmental processes. It subject                     encodes ATPase involved retrotranslocation of <br /> debate form cell death program exists                            ubiquitinated proteins cytosol processing the <br /> unicellular organisms, benefit single cell kill               proteasome, particular cdc48 mutant shown to <br /> itself. In nature, single cell organisms yeasts bacteria                   die exhibiting typical apoptotic markers [11,12]. <br /> develop colonies biofilms conditions, cells                     In yeast, orthologues key regulators late mammalian <br /> are contact other. Several reports shown                   apoptotic events far identified. The yeast protease <br /> death cells benefit rest colony                       Yor197wp (Yca1) behaves like mammalian caspase: it <br /> confers growth advantage competitors. In yeast colonies,                   cleaved presence hydrogen peroxide gaining <br /> an ammonia signal emitted oxidative stress triggers                      caspase activity vitro, H2O2-induced apoptosis being <br /> yeast death centre colony [2]. Moreover                       abrogated increased YCA1 deleted overexpressed <br /> phenomenon advantageous younger cells periphery                       respectively [13]. Also, Apoptosis Inhibiting Factor (AIF) <br /> physically removing central dying cells reduces growth outer                   homologue yeast, Ynr074cp, reported translocates <br /> margin colony. Also yeast cells containing killer viruses                 mitochondria nucleus able degrade yeast <br /> been reported induce genetically programmed cell death                       nuclear DNA plasmid DNA similar manner to <br /> non-infected cells population [3] allowing <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span>survival</span>                            mammalian AIF [14]. Finally, yeast Nma111p be <br /> advantage viruses containing versus non-containing cells                       homologous mammalian mediator apoptosis HtrA2/Omi <br /> chronologically aged cultures.                                                        [15] presence apoptotic markers dead yeast cells <br />    <span id='am-2' about='protege:FROM' typeof='owl:Thing'>From</span> molecular point view indications yeast                    exposed H2O2 dependent Nma111p [16]. <br /> cells circumstances die biochemical markers                              Nevertheless, despite accumulating data showing apoptotic <br /> apoptosis. After exposure hydrogen peroxide (H2O2) acetic                       phenotypes unicellular organisms presence of <br /> acid, yeast cells exhibit DNA strand breakage, chromatin                              orthologues key apoptotic regulators, unclear <br /> fragmentation exposure phosphatidylserine outer                         fully integrated cell death pathways exist yeast. In <br /> plasma membrane [4&#226;&#8364;&#8220;6]. These molecular changes associated                             particular, early steps defined cell death program are <br /> with death require active protein synthesis                            unknown, heterologous expression active forms <br />  <br />  <br />        PLoS ONE | www.plosone.org                                                 1                               February 2009 | Volume 4 | Issue 2 | e4376 <br />                                                                                                                     Role Dre2-Tah18 H2O2 <br />  <br />  <br /> of proapoptotic mammalian Bax yeast induces cell death               [27,28]; alternatively removal CPT complexes requires <br /> apoptotic phenotypes, targets mitochondria [17,18]. No Bax               repair enzymes. This original tah18 mutant exhibited sensitivity to <br /> orthologue yeast. Interestingly, recently           CPT, suggesting role protein DNA repair drug <br /> reported that, similarly human cohesin Rad21                  treatment. <br /> involved apoptosis [19,20], yeast homologue hRad21,                   Thus, tested sensitivity tah18-5H8 tah18-5I5 <br /> Mcd1, cleaved caspase-like protease Esp1 exposure            mutants chronic exposure CPT shown figure 1-B. We <br /> to lethal doses hydrogen peroxide [21]. Further, C-terminal           assessed sensitivity mutants gamma ray <br /> part Mcd1 translocates nucleus mitochondria               irradiation creates DNA damage, mainly form <span id='am-239' about='xsp:base' typeof='owl:Thing'>base</span> <br /> and cell death enhanced conditions.                           damage double single strand breaks [29]. Using drop <br />    In study investigated role                    tests, mutants showed significant sensitivity to <br /> uncharacterized yeast protein Tah18 previously identified           gamma rays CPT compared wild-type cells (Fig. 1- <br /> in laboratory genetic screen synthetic mutants             B&#226;&#8364;&#8220;C). Thus, mutants impaired DNA repair <br /> with pol3-13, thermosensitive allele POL3 encodes             exposure CPT gamma rays. <br /> catalytic subunit DNA polymerase delta [1]. We generated               Blast analysis Tah18 sequence revealed presence of <br /> thermosensitive tah18 mutants proved highly resistant          conserved domains suggesting redox activity (Fig. 1-D). The <br /> an acute exposure H2O2. We demonstrated H2O2-                   N-terminus includes Flavodoxin-like domain consists of <br /> induced cell death dependent Tah18 abundance cell.              170 residues flavin mononucleotide (FMN)-binding <br /> In presence H2O2, observed GFP-Tah18 relocalization                site. This domain involved electron transfer reactions, has <br /> to mitochondria, detectable using fluorescent              various species. In central C-terminal part <br /> mutated versions Tah18. Next, showed mitochondria              protein, blast analysis identified potential FAD- NAD- <br /> tah18 mutants protected H2O2-induced damage,                binding domains compatible electron transfer activity. A <br /> cytochrome c degraded wild-type tah18 cells.                 search proteins domain organisation (Conserved <br /> DRE2 identified genetically interacting TAH18,              Domain Architecture Retrieval Tool, CDART) revealed many <br /> as mutated alleles synthetic lethal, Dre2 overexpression             potential oxydo-reductases involved various processes, no <br /> can suppress tah18 growth defects 35uC. Dre2 recently                 particular class emerged analysis. <br /> characterized Fe/S protein [22] human homologue,                     We investigated sensitivity tah18 mutants agents that <br /> Ciapin1, anti-apoptotic protein [23]. Ciapin1             modify redox balance medium. We used hydrogen <br /> suppress Ddre2 lethality interact physically Tah18           peroxide (H2O2) generate oxidative stress tested acute <br /> the absence oxidative stress. Finally, expressing Tah18               exposure lethal doses compound. Exponentially growing <br /> Dre2 single fusion protein allowed viability. Moreover,             cells complete glucose medium (YPD) exposed 2 mM <br /> fusion protein did delocalize mitochondria prevented              H2O2 6 hours plated complete <br /> cell death presence H2O2. From work conclude               medium assess viability counting number colony <br /> that Tah18 plays &#226;&#8364;&#732;&#226;&#8364;&#732;pro-death&#226;&#8364;&#8482;&#226;&#8364;&#8482; role response oxidative stress.       forming units (CFU). Both tah18-5H8 tah18-5I5 mutants were <br /> After exposure lethal doses H2O2, Tah18 targets                    highly resistant H2O2 exposure compared wild-type from <br /> mitochondria promotes cell death yeast                  2 hours shown Fig. 1-E. Similar results obtained using <br /> component cell death program connexion                      1mM 3mM H2O2 (not shown). This argue Tah18 <br /> between Tah18 cell death anticipated. As Bax/                involved regulating intracellular H2O2 detoxification <br /> Bcl2 homologues described far organism,              negatively positively, or/and controlling cell death in <br /> discuss significance findings respect yeast             presence H2O2. <br /> physiology. <br />                                                                              2. Cell death H2O2 exposure dependent the <br /> Results                                                                      Tah18 <br />                                                                                 We asked Tah18 overexpression affects sensitivity <br /> 1. Identification temperature sensitive tah18 mutants                     H2O2. We used strains expressing Tah18 N- <br /> which resistant wild-type hydrogen                          terminal GFP tag, control Gal1-10 <br /> peroxide DNA damaging agents                                promoter (overexpression YPGal, strain 10H8, Table 1) or <br />    TAH18 uncharacterized gene yeast, essential             control endogenous TAH18 promoter (endoge- <br /> for viability. A tah18 mutant originally identified               nous expression, strain 11B3, Table 1). Both constructs fully <br /> laboratory genetic screen synthetic mutants               viable compared wild-type strain shown drop tests <br /> pol3-13, thermosensitive allele POL3 [1]. For studying TAH18            (Fig. 1-F), indicating GFP-Tah18 functional that <br /> in created collection thermosensitive tah18              overexpression GFP-Tah18 fusion protein toxic. <br /> mutants (see Materials Methods) using previously described             Relative amounts fusion protein strains shown in <br /> method based mutagenic PCR [24]. On basis                    Fig. 1-G. When grown galactose exposed 2 mM H2O2, <br /> growth capacity 28uC absence growth 37uC,                   strain overexpressing GFP-Tah18 showed significantly higher <br /> mutants (tah18-5H8 tah18-5I5) selected                  sensitivity H2O2 compared control strain expressing <br /> characterization (Fig. 1-A). Sequencing analysis revealed          GFP-Tah18 native TAH18 promoter wild-type <br /> mutant contains mutations shown Fig. 1-D.                      strain (Fig. 1-H). Thus, overexpressing GFP-Tah18 protein <br />    A previous study yeast identified tah18 mutant              renders cells sensitive H2O2 galactose medium. <br /> through genetic screen conditional tah (top1T722A-hyper-                  It noteworthy control strains (wild-type 8C2 and <br /> sensitive) mutants exhibiting thermo-sensitive growth 36uC             GFP-Tah18 11B3) resistant H2O2 grown in <br /> the presence camptothecin mimetic mutant top1T722A                    galactose (Fig. 1-H) compared glucose (Fig. 1-E). A previous <br /> [25,26]. Camptothecin (CPT) needs ongoing replication                    report indicated yeast sensitivity H2O2 dependent on <br /> specifically freezing covalent TopoI-DNA complex                  carbon source yeast cells grown respirable glycerol- <br /> thought inhibit DNA replication cytotoxic DNA lesions                 containing medium exhibit higher resistance H2O2 cell killing <br />  <br />  <br />        PLoS ONE | www.plosone.org                                        2                          February 2009 | Volume 4 | Issue 2 | e4376 <br />                                                                                                                        Role Dre2-Tah18 H2O2 <br />  <br />  <br />  <br />  <br /> Figure 1. A&#226;&#8364;&#8220;C. Tah18-5H8 tah18-5I5 thermosensitive mutants involved DNA repair exposure gamma <br /> rays camptothecin. Ten-fold dilutions exponentially grown yeast cells spotted YPD medium (A&#226;&#8364;&#8220;B) YPD containing <br /> Camptothecin indicated (C). Gamma rays irradiation performed using 137Cs dose rate 0.850 Gy/s. Growth assessed 3 days <br /> at 28uC unless indicated. A D rad52 mutant used control. D. Tah18 exhibits domains potentially involved oxydo-reduction reactions. <br /> Flavodoxin, FAD-binding NAD-binding domains detected using Blast program. Mutations tah18-5H8 tah18-5I5 indicated. E. <br /> Tah18-5H8 tah18-5I5 resistant H2O2 acute exposure TAH18. Exponentially growing cells YPD (1&#226;&#8364;&#8220;5.106 cells milliliter of <br /> culture) exposed 2 mM H2O2 1 6 hours plated YPD. After 3 days 28uC, number colony forming units <br /> (CFU) counted expressed percentage number CFU time zero assess survival rate. Points means independent <br /> experiments. Error bars +/2standard error. F&#226;&#8364;&#8220;G. Expressing GFP-TAH18 control endogenous TAH18 promoter overexpressing GFP- <br /> TAH18 control Gal1-10 promoter does affect cell growth. F. Ten-fold dilutions exponentially grown yeast cells strains <br /> 11B3 expressing GFP-Tah18 control endogenous TAH18 promoter (GFP-TAH18), 8C2 wild-type strain (TAH18) 10H8 expressing GFP- <br /> Tah18 control Gal1-10 promoter (GFP-TAH18 overexp.) spotted YPGal (inducing conditions Gal1-10 driven expression) or <br /> YPD medium, growth assessed 3 days 28uC. G. Western blot analysis GFP-Tah18 expression. Lane1: strain 11B3 expressing GFP- <br /> Tah18 control endogenous TAH18 promoter, Lane2: wild-type strain 8C2, Lane3: strain 10H8 expressing GFP-Tah18 control of <br /> Gal1-10 promoter. Protein extracts prepared using classical TCA method starting yeast cultures grown YPGal density close to <br /> 5.106 cells milliliter culture. Equal amounts total proteins loaded 10% acrylamide gel analysed Western blot using anti-GFP <br />  <br />  <br />  <br />        PLoS ONE | www.plosone.org                                         3                           February 2009 | Volume 4 | Issue 2 | e4376 <br />                                                                                                                                  Role Dre2-Tah18 H2O2 <br />  <br />  <br />  <br /> antibodies. Anti-PCNA antibodies used loading control. H. Overexpressing GFP-TAH18 renders cells sensitive H2O2. Exponentially <br /> growing cells YPGal medium (1&#226;&#8364;&#8220;5.106 cells milliliter culture) exposed 2 mM H2O2 1 6 hours plated YPGal <br /> plates. After 3 days 28uC, number colony forming units (CFU) counted expressed percentage number CFU time zero <br /> to assess survival rate. Points means independent experiments. Error bars +/2standard error. <br /> doi:10.1371/journal.pone.0004376.g001 <br />  <br />  <br /> compared yeast cells grown fermentable glucose-containing                     pressed. Moreover, location DAPI-stained mitochondria we <br /> medium [30]. This result explained increased                           noticed absence green fluorescence (black spots <br /> expression detoxification systems catalase, superoxide                   indicated arrows) suggesting GFP-Tah18 absent from <br /> dismutase glutathione peroxidases respirable compared                     mitochondria shown magnification panel (Fig. 2-B). <br /> fermentable carbon sources [31].                                                       After 4 hour H2O2 exposure, localization GFP- <br />    To summarize, shown mutated Tah18 renders                           Tah18 fluorescence changed dramatically revealed a <br /> cells resistant H2O2 exposure, overexpression                    mitochondrial <span id='am-240' about='xsp:pattern' typeof='owl:Thing'>pattern</span>, shown colocalization DAPI- <br /> GFP-Tah18 renders cells sensitive treatment.                       stained mitochondria (Fig. 2-C). Thus, fluorescence microscopy <br /> This result indicates cell death H2O2 exposure                        suggests GFP-Tah18 relocalizes mitochondria after <br /> dependent Tah18 tah18-5H8 tah18-                      exposure H2O2. <br /> 5I5 mutants gain-of-function mutants regarding cell                      Gal-controlled expression high yeast provoke <br /> death <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> H2O2. Thus, Tah18 negatively regulate                        additional phenotypes protein overexpression though <br /> intracellular H2O2 detoxification, and/or positively promote cell                   cell viability absence H2O2 strain similar to <br /> death.                                                                              seen wild-type (Fig. 1-F). For reason next <br />                                                                                     investigated yeast strain expressing GFP-Tah18 the <br /> 3. GFP-Tah18 translocates mitochondria H2O2                                 control endogenous TAH18 promoter (11B3). This strain <br /> treatment GFP-Tah18 mutated versions                                        exhibited obvious phenotype compared wild-type in <br />    To gain better insight Tah18 function yeast,                   exponentially growing cultures acute exposure to <br /> investigated Tah18&#226;&#8364;&#8482;s intracellular localization. We used live cell                  H2O2. By Western blotting using anti-GFP antibody, low <br /> fluorescent microscopy examine localization GFP-                      amounts GFP-Tah18 protein detected compared the <br /> Tah18 fusion protein strain 10H8 (overexpression)                        strain 10H8 grown galactose (Fig. 1-G). In accordance, we <br /> 11B3 (endogenous expression). When cells grown exponen-                        detect fluorescence microscopy cells <br /> tially galactose, GFP-Tah18 fluorescence pattern diffuse                  grown absence H2O2 (Fig. 2-D). However, after <br /> in cell, contrast DAPI fluorescence pattern                    4 hours exposure H2O2 2 mM GFP-Tah18 exhibited a <br /> specifically shows nucleus mitochondria places (Fig. 2-A),                      mitochondrial distribution (Fig. 2-E) suggesting acute <br /> suggesting GFP-Tah18 mainly cytosolic overex-                          exposure H2O2 provokes Tah18 relocalization mito- <br />  <br />  Table 1. Strains table. <br />  <br />  <br />  Strain                     Genotype                                                                                                 Source <br />                                                                  S <br />  5C6                        MATa ura3 leu2 his3 dre2D::TRP1-CYH2 pRS416-DRE2-URA3                                                    This study <br />  5C8                        MATa ura3 leu2 his3 tah18D::TRP1-CYH2S pRS416-TAH18-URA3                                                 This study <br />  5H8                        MATa ura3 leu2 trp1 his3 tah18-5H8 (#31)                                                                 This study <br />  5I5                        MATa ura3 leu2 trp1 his3 tah18-5I5 (#43)                                                                 This study <br />  8C1                        MATa/alpha ura3/ura3 leu2/leu2 trp1/trp1                                                                 Ge&#194;&#180;rard Faye <br />  8C2                        MATalpha ura3 leu2 trp1 lys2 cyh2R                                                                       Ge&#194;&#180;rard Faye <br />  8C4                        MATa/alpha ura3/ura3 leu2/leu2 trp1/trp1 cyh2R/cyh2R                                                     Ge&#194;&#180;rard Faye <br />  8D1                        MATalpha ura3 his3 dre2D::TRP1-CYH2S pRS315-hCiapin1-LEU2                                                This study <br />  9A2                        MATalpha ura3 his3 dre2D::TRP1-CYH2S pRS425-hCiapin1-LEU2                                                This study <br />  9D9                        MATa ura3 leu2 trp1 dre2D::DRE2-GFP HIS3-Mx6                                                             This study <br />  10G3                       MATa ura3 leu2 trp1 his3 dre2P221S                                                                       This study <br />  10H8                       MATa ura3 leu2 his3 TRP1-GAL1-GFP-TAH18                                                                  This study <br />  11B3                       MATa ura3 his3 tah18D::TRP1-CYH2S pRS305-TAH18Promoter-GFP-TAH18-LEU2                                    This study <br />  12C2                       MATalpha ura3 leu2 his3 DRE2::TRP1-GAL-3HA-DRE2                                                          This study <br />  12E3                       MATa ura3 his3 dre2D::TRP1-CYH2S tah18D::TRP1-CYH2S pRS315-LEU2-DRE2Promoter-DRE2-TAH18                  This study <br />  12E6                       MATalpha; ade2-1; ura3-1; his3-11,15; trp1-1; leu2-3,112; can1-100; UBR1::HIS3-GAL-HA-UBR1 (background   This study <br />                             W303) <br />  12E8                       MATa his3 leu2::GFP-TAH18-pRS305-LEU2 tah18D::TRP1-CYH2S DRE2::TRP1-GAL-3HA-DRE2                         This study <br />                                                                                                                S <br />  12F2                       MATa ura3 UBR1::HIS3-GAL-HA-UBR1 leu2::GFP-TAH18-pRS305-LEU2 tah18D::TRP1-CYH2                           This study <br />  EGY48                      MATa his3 trp1 ura3 LexAop-LEU2                                                                          Clontech <br />  <br />  All strains used study congenic excepted EGY48 (Clontech) 12E6 (W303). <br />  doi:10.1371/journal.pone.0004376.t001 <br />  <br />  <br />  <br />         PLoS ONE | www.plosone.org                                              4                              February 2009 | Volume 4 | Issue 2 | e4376 <br />                                                                                                                         Role Dre2-Tah18 H2O2 <br />  <br />  <br />  <br />  <br /> Figure 2. GFP-Tah18 relocalizes mitochondria H2O2 exposure. A&#226;&#8364;&#8220;E: Fluorescence microscopy live cells expressing GFP-Tah18 <br /> under control Gal promoter exponential culture YPGal (strain 10H8, panels A&#226;&#8364;&#8220;C) control endogenous TAH18 <br /> promoter YPD (strain 11B3, panels D&#226;&#8364;&#8220;E). Panel B magnification single cell A. Mitochondria indicated white arrows. Live cells <br /> were directly stained DAPI 0.05 mg/ml mounting solution visualize nuclear mitochondrial DNA. GFP fluorescence detected <br /> using FITC filterset. When indicated, cells treated 2 mM H2O2 4 hours direct observation. P: Phase contrast; D: DAPI; F: FITC, <br /> M: Merge. F. Western blot analysis purified mitochondria. Crude extracts (CE) prepared using classical TCA method starting yeast <br /> cultures grown YPD density close 5.106 cells/mL. Cells treated 2 mM H2O2 4 hours processing. <br /> Mitochondrial extracts (Mitos.) fractionated described Materials Methods section using sucrose gradients. 10 mg extract <br /> was loaded. Anti-Mdh1 antibodies used loading control mitochondrial extracts anti-PCNA antibodies used control for <br /> nucleo-cytosolic contamination mitochondrial extracts. G. Western blot analysis crude extracts. Crude extracts prepared using classical TCA <br /> method starting yeast cultures strain 11B3 (expressing endogenous levels GFP-Tah18) grown YPD density close 5.106 cells/mL, <br /> and treated 2 mM H2O2 2 4 hours processing. <br /> doi:10.1371/journal.pone.0004376.g002 <br />  <br />  <br />        PLoS ONE | www.plosone.org                                         5                            February 2009 | Volume 4 | Issue 2 | e4376 <br />                                                                                                                          Role Dre2-Tah18 H2O2 <br />  <br />  <br /> chondria GFP-Tah18 expressed native TAH18                   2 mM H2O2 1 hour. As shown figure 3-A, heme staining of <br /> promoter.                                                                     purified mitochondria revealed holo-cytochrome c still <br />    To confirm microscopy data, used cell fractionation               intact mitochondria tah18-5H8 tah18-5I5 cells after <br /> approach. We isolated mitochondria 11B3 cells exponentially              1 hour exposure 1 mM H2O2 mitochondria from <br /> grown glucose medium presence absence H2O2                    wild-type cells. After 1 hour exposure 2 mM H2O2, holo- <br /> using sucrose density centrifugation described previously [32].            cytochrome c starts destabilized mitochondria <br /> The presence GFP-Tah18 fusion protein assessed                  tah18 mutants. The mitochondrial holo-cytochrome c thus <br /> Western blotting using anti-GFP antibody. Results figure 2-F            stable mitochondria tah18 mutants those <br /> show absence GFP-Tah18 mitochondria purified                    wild-type cells. This result correlates higher survival <br /> from untreated cells, GFP-Tah18 detected                        rate tah18 mutants compared wild-type cells acute <br /> mitochondria cells exposed H2O2.                                      exposure H2O2. <br />    Finally, Figure 2-G shows GFP-Tah18 maintained                   Furthermore, Western blotting analysis using anti-cytochrome c <br /> steady state crude extract exposure H2O2, arguing               antibody showed Cyc1 disappears mitochondrial <br /> that GFP-Tah18 appearing mitochondria                  extracts kinetics similar loss peroxidase activity (as <br /> general increase GFP-Tah18 level induced H2O2.                          detected heme staining) (Fig. 3-A). We unable detect <br />    Taken together, results GFP-Tah18 localized             Cyc1 cytosolic extracts experiment (not <br /> outside mitochondria exponentially growing cells,                  shown) indicating Cyc1 released mitochondria the <br /> relocalizes mitochondria exposure H2O2.                       cytosol H2O2, rapidly degraded. <br />    In order address tah18-5H8 tah18-5I5 mutants <br /> showed altered localization pattern H2O2 exposure, GFP-              5. DRE2 multi copy suppressor tah18 <br /> tagged versions mutants constructed. Growth                      thermosensitivity, dre2 tah18 mutants are <br /> capacity resistance H2O2 similar tagged                     synthetic lethal <br /> untagged tah-5H8 tah18-5I5 versions (not shown). Using live                  In attempt identify Tah18 partners, tried to <br /> cell fluorescence microscopy unable detect GFP              identify genetic interactions tah18 mutants. We transformed <br /> mitochondria 6 hours exposure H2O2, indicating               tah18-5I5 mutant strain multicopy 2m-based yeast DNA <br /> GFP-Tah18-5H8 GFP-Tah18-5I5 affected delocal-                     library YEp13 vector (LEU2 marker). Transformants <br /> ization mitochondria oxidative stress (not shown).               selected SC&#226;&#8364;&#8220;LEU medium permissive temperature <br />                                                                               (28uC) replica plated complete medium various <br /> 4. Mitochondria integrity preserved tah18 mutants                       restrictive temperatures ranging 32uC 37uC. Growing <br /> after H2O2 exposure                                                           colonies selected plasmid DNA recovered E. <br />    H2O2 exposure results mammalian cell death                      coli. More 250 plasmids assessed re-transformation of <br /> accompanied cytochrome c release mitochondria                  mutant selection thermoresistance analysis with <br /> cytosol degradation [33]. In yeast, cytochrome c (Cyc1)                restriction enzymes sequencing. <br /> release mitochondria cytosol shown                When colonies grown 37uC, 20 plasmids were <br /> some extent acetic acid treatment cells [34]. We used heme           recovered, containing TAH18. <br /> staining assess holo-cytochrome c status mitochondria                  At 32uC 35uC, addition 100 TAH18-containing <br /> [35]. Mitochondria purified cells treated 1 mM              plasmids, recovered 142 plasmids containing same <br />  <br />  <br />  <br />  <br /> Figure 3. Mitochondria integrity H2O2 stress preserved tah18 mutants wild-type TAH18 cells. Heme staining <br /> and Western blot analysis cytochrome c purified mitochondria reveal cytochrome c degradation prevented tah18 mutants. <br /> Mitochondrial extracts fractionated described Materials Methods section, starting exponentially grown cells (5.106 cells/ml or <br /> less) YPD medium. The concentration extract assessed using Bradford assay 10 mg loaded well. For heme staining of <br /> mitochondrial cytochromes c c1 (respectively Cyc1 Cyt1), samples incubated Laemmli buffer containing dithiothreitol 50 mM for <br /> 30 min 4uC migration lithium dodecyl sulfate-PAGE electro-transfer nitrocellulose membrane. Heme-associated peroxidase <br /> activity revealed using ECL plus Western blotting detection (GE Healthcare) directly nitrocellulose membrane. Anti-Mdh1 <br /> antibodies used loading control mitochondrial extracts. <br /> doi:10.1371/journal.pone.0004376.g003 <br />  <br />  <br />        PLoS ONE | www.plosone.org                                         6                             February 2009 | Volume 4 | Issue 2 | e4376 <br />                                                                                                                       Role Dre2-Tah18 H2O2 <br />  <br />  <br />  <br />  <br /> Figure 4. DRE2 TAH18 interact genetically products interact physically vivo. A. DRE2 multicopy suppressor tah18- <br /> 5H8 tah18-5I5. Ten-fold dilutions exponentially grown yeast cells spotted YPD medium growth assessed 3 days. When <br /> indicated, DRE2 expressed multicopy plasmid YEp13 tah18-5I5 strain (two lanes). B. Dre2 tah18 mutants synthetic lethal. <br /> Dissection plate sporulated diploid dre2-P221S/DRE2 tah18-5I5/TAH18 shows spores harbouring dre2-P221S tah18-5I5 mutated <br /> alleles unable grow. C&#226;&#8364;&#8220;D. Dre2 Tah18 interact physically vivo. C. Two-hybrid assayed according manufacturer&#226;&#8364;&#8482;s instructions (see <br /> Materials Methods). D. Crude extracts co-IP HA coated beads prepared described M&amp;M section. Anti-HA (Sigma) anti-GFP <br /> (Roche) used Western blot analysis. Lane1: 8C2 WT, Lane2: 12E8 HA-Dre2 GFP-Tah18, Lane3: 12C2 HA-Dre2, Lane4: 11B3 GFP-Tah18, Lane5: <br /> HA-Ubr1 GFP-Tah18. E. Dre2 highly conserved protein. Alignments using ClustalW Boxshade EMBnet. Identities are <br /> shaded black similarities shaded grey. <br /> doi:10.1371/journal.pone.0004376.g004 <br />  <br />  <br />  <br /> ORF, YKR071C/DRE2. No ORF appeared complement                          DRE2 essential gene vegetative growth remained <br /> the tah18 growth defect restrictive temperature 32uC              uncharacterized recent study showed Dre2 involved <br /> 35uC. Thus, DRE2 overexpression able suppress tah18                    cytosolic Fe/S protein biogenesis [22]. It noteworthy a <br /> thermosensitivity 35uC (see Fig. 4-A).                                    dre2 allele previously identified laboratory was <br />  <br />  <br />        PLoS ONE | www.plosone.org                                        7                            February 2009 | Volume 4 | Issue 2 | e4376 <br />                                                                                                                     Role Dre2-Tah18 H2O2 <br />  <br />  <br /> synthetic lethal pol3-13, thermo-sensitive allele DNA             Resistance improved slightly overexpressing <br /> polymerase delta [1]. Sequencing allele indicated                 Dre2 (not shown), indicating stabilizing Dre2-Tah18 complex <br /> Phenylalanine 221 substituted Serine.                               prevent H2O2-induced cell death. <br />    We asked dre2-P221S mutated allele                    We created permanent interaction Tah18 and <br /> tah18-5I5 allele exhibit genetic interaction. Tah18-5I5 dre2-         Dre2 fusing DRE2 frame TAH18, strain deleted both <br /> P221S haploid mutant strains (strains 5I5 10G3, Table 1)           endogenous DRE2 TAH18 (strain 12E3, Table 1 Fig. 5- <br /> crossed diploids selected. After sporulation,          B). Surprisingly, strain fully viable, suggesting the <br /> spores harbouring dre2 tah18 mutated alleles able          dissociation Dre2 Tah18 wild-type cells be <br /> grow, spores single <span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span>mutation</span> TAH18              essential viability. Immunodetection using anti-Dre2 <br /> DRE2 growing, showing combination tah18               antibody crude extracts indicated fusion protein is <br /> and dre2 alleles synthetic lethal (see Fig. 4-B).                        detected single band expected size 110 kDa and <br />                                                                             behaves like Dre2 absence presence H2O2, small <br /> 6. Tah18 protein interacts physically Dre2 protein                     fusion protein detected mitochondria both <br />    We used co-immunoprecipitation test Tah18                 conditions does relocalise mitochondria (Fig. 5-C <br /> Dre2 interact vivo. As shown figure 4-D, Hemagglutinin-tagged         lanes 3, 4, 7 8, Fig. 5-D lanes 3, 4, 7 8). Furthermore, when <br /> Dre2 (HA-Dre2) able specifically co-immunoprecipitate GFP-            assaying sensitivity H2O2, DRE2-TAH18 fusion expressing <br /> Tah18 indicating HA-Dre2 GFP-Tah18 interact yeast               strain significantly resistant wild-type (as shown in <br /> extracts, interaction mediated tags.            figure 5-E). This result suggests forcing interaction between <br />    We confirmed interaction using LexA-based two-             Dre2 Tah18 vivo prevents Tah18 relocalization the <br /> hybrid technology (Clontech, Materials Methods). Full               mitochondria decreases H2O2-induced cell death. <br /> <span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> proteins expressed fusion, either <br /> with Binding domain (Tah18) Activating domain (Dre2).            8. Human anti-apoptotic Ciapin1 able complement a <br /> The expressed proteins showed clear interaction, indicated          dre2 deletion interacts Tah18 <br /> growth blue colour compared controls selective              Database searching Dre2 homologues gave significant <br /> medium (Fig. 4-C).                                                          hits yeast protein, revealed high level of <br />                                                                             conservation species C-terminus portion the <br /> 7. Promoting Dre2-Tah18 interaction reduces H2O2-                           protein. This similarity shared 46 proteins Interpro <br />                                                                             database protein families, defines entry signature <br /> induced cell death Tah18 localization to <br />                                                                             DUF689 shown figure 4-E. The human homologue Dre2p <br /> mitochondria                                                                appeared Ciapin1 (Cytokine Induced Apoptosis Inhibitor <br />    Since Dre2 interacts Tah18, overexpressing Dre2             1) protein, initially described inhibitor apoptosis whose <br /> suppress tah18 growth defect, tested Dre2           expression regulated growth factors related <br /> able translocate mitochondria absence presence          Bcl2 caspases [23]. We investigated human <br /> of H2O2.                                                                    Ciapin1 complement absence Dre2 protein yeast. <br />    We used live cell fluorescent microscopy examine                  Ciapin1 subcloned centromeric (pRS315, strain <br /> localization Dre2-GFP fusion protein expressed             8D1 Table 1) multicopy (pRS425, strain 9A2 Table 1) vector <br /> native DRE2 promoter (strain 9D9 Table 1). In absence stress,        control TEF promoter (promoter gene <br /> the Dre2-GFP fusion protein showed diffuse cellular pattern       encoding translation elongation factor 1 alpha [37]). Both <br /> in accordance previously described cytosolic Dre2-GFP                  constructs confer growth capacity absence <br /> localisation [36]. No obvious change Dre2-GFP distribution               DRE2 containing plasmid pRS316 (Fig. 5-F). This result in <br /> was observed exposure H2O2 (Fig. 5A), indicating           accordance recent work [22]. <br /> massive Dre2 relocalization mitochondria occurred.                      We asked hCiapin1 interacts yeast Tah18. As <br />    We used anti-Dre2 antibody (see Materials                    previously described, used two-hybrid technology full <br /> Methods) raised Dre2 1-133 N-terminus                  length proteins. Like Dre2 (Fig. 4), hCiapin1 able interact <br /> part. Dre2 detected single band expected size           physically yeast Tah18 (Fig. 5-G). <br /> 48-kDa cell crude extract (Fig. 5-C lanes 1 2, Fig. 5-            We conclude hCiapin1 Dre2 share one <br /> H) degraded increased 2             common function, essential yeast be <br /> 4 hours exposure H2O2 (Fig 5-H). Very small amounts Dre2              dependent interaction Tah18. <br /> were detected mitochondrial extracts wild-type cells in <br /> the absence presence H2O2 (Fig. 5-C lanes 5 6). Figure 5-         Discussion <br /> D shows overexposure Western blot analysis using anti- <br /> Dre2 antibody increasing amounts mitochondrial extracts,                 In study evidenced previously uncharacterized protein <br /> indicating small fraction Dre2 localized              complex, Dre2-Tah18, controls yeast cells death response to <br /> mitochondria previously described using HA-tagged Dre2                 high doses hydrogen peroxide. In absence exogenous <br /> [22] mitochondrial Dre2 amounts increase              oxidative stress, Tah18 Dre2 physically interact outside the <br /> exposure H2O2 (Fig. 5-D lanes 1, 2, 5 6). This consistent         mitochondria. In presence oxidative stress, Tah18 targeted to <br /> with fact cytosolic Dre2, unlike Tah18, does relocalize        mitochondria controls mitochondrial integrity cell death. <br /> to mitochondria oxidative stress.                                 Together, results argue existence mitochondria- <br />    It possible Dre2-Tah18 interaction affect              dependent cell death program yeast involving Tah18. <br /> Tah18 localization mitochondria oxidative stress. We <br /> therefore tested Dre2 overexpression improve                  Tah18 targets mitochondria pro-death factor in <br /> resistance H2O2. Dre2 expressed control             presence oxidative stress <br /> Gal1-10 promoter (strain 12C2, Table 1), cells exposed               We observed relocalization GFP-Tah18 fusion to <br /> to 2 mM H2O2 6 hours previously described.                     mitochondria exposure high doses H2O2. This <br />  <br />  <br />       PLoS ONE | www.plosone.org                                        8                           February 2009 | Volume 4 | Issue 2 | e4376 <br />                                                Role Dre2-Tah18 H2O2 <br />  <br />  <br />  <br />  <br /> PLoS ONE | www.plosone.org   9   February 2009 | Volume 4 | Issue 2 | e4376 <br />                                                                                                                        Role Dre2-Tah18 H2O2 <br />  <br />  <br />  <br /> Figure 5. A&#226;&#8364;&#8220;D. Neither Dre2p Dre2-Tah18 fusion protein relocalize mitochondria H2O2 treatment. A. Fluorescence <br /> microscopy live cells expressing Dre2-GFP exponential culture YPD (strain 9D9, panel A). Live cells directly stained DAPI at <br /> 0.05 mg/ml mounting solution visualize nuclear mitochondrial DNA. GFP fluorescence detected using FITC filterset. When <br /> indicated, cells treated 2 mM H2O2 4 hours direct observation. P: Phase contrast; D: DAPI; F: FITC, M: Merge. B. Dre2 and <br /> Tah18 expressed fusion protein. C. Western blot analysis crude mitochondrial extracts using anti-Dre2 antibody. Crude extracts were <br /> prepared using classical TCA method starting yeast cultures grown YPD density close 5.106 cells/mL absence (2) presence <br /> (+) 2 mM H2O2.. Mitochondrial extracts fractionated described Materials Methods section using sucrose gradients cells <br /> grown absence (2) presence (+) 2 mM H2O2. 10 mg yeast crude extracts 50 mg mitochondria loaded well. WT: Wild-type <br /> strain, DRE2-TAH18: 12E3 strain expressing fusion Dre2-Tah18. Anti-Mdh1 antibodies used loading control mitochondrial extracts <br /> and anti-PCNA antibodies used control nucleo-cytosolic contamination mitochondrial extracts. D. Western blot analysis increasing <br /> amounts mitochondrial extracts. The white star shows non specific band revealed anti-Dre2 antibody. 50 mg (16) 250 mg (56) of <br /> mitochondrial extracts loaded well. E. Forcing Dre2-Tah18 interaction improves resistance H2O2. Exponentially growing cells YPD (1&#226;&#8364;&#8220; <br /> 5.106 cells milliliter culture) exposed 2 mM H2O2 1 6 hours plated YPD. After 3 days 28uC, number of <br /> colony forming units (CFU) counted expressed percentage number CFU time zero assess survival rate. Points means of <br /> three independent experiments. Error bars +/2standard error. F&#226;&#8364;&#8220;G. Human Ciapin1 complements Ddre2 deletion interacts vivo with <br /> Tah18. F. The human Ciapin1 expressed URA3-containing centromeric (pRS315) multicopy plasmid (pRS425) control of <br /> the TEF promoter. Growth assessed 3 days 28uC, 5-FOA YPD plates. G. Two-hybrid assayed according manufacturer&#226;&#8364;&#8482;s <br /> instructions (see Materials Methods). H. Dre2 degraded exposure H2O2. Western blot analysis Dre2 crude extracts after <br /> exposure H2O2. Crude extracts prepared using classical TCA method starting yeast cultures grown YPD density close 5.106 <br /> cells/mL absence (2) presence (+) 2 mM H2O2, 2 4 hours. 10 mg yeast crude extracts loaded well. Anti-PCNA <br /> antibody used loading control. <br /> doi:10.1371/journal.pone.0004376.g005 <br />  <br /> relocalization affected tah18-5H8 tah18-5I5 mutants.                mitochondria release sequestration partner has <br /> Moreover, cytochrome c rapidly destabilized               described Bax [43]. To knowledge, such <br /> degraded Tah18-dependent manner                           mechanism characterized date yeast proteins. <br /> conditions. These findings oxidative stress conditions              We identified DRE2, interacts genetically TAH18 in <br /> partly reminiscent properties Bax mammalian cells.               absence oxidative stress, showed gene products <br /> Bax pro-apoptotic protein evolutionary conserved                physically interact vivo. Forcing Dre2-Tah18 interaction by <br /> Bcl2 family, members regulate apoptosis positively            expressing fusion results increased resistance to <br /> negatively. Bax inserted outer membrane mitochondria             H2O2-induced cell death, fusion protein relocalized <br /> at early stage apoptosis [38,39]. This provokes mitochon-              mitochondria. Also overexpressing Dre2 yeast renders the <br /> drial membrane leakage release mitochondrial factors                cells resistant H2O2 arguing promoting the <br /> the cytosolic compartment (e. g. AIF, cytochrome c, endoG&#226;&#8364;&#166;),                  interaction Tah18 Dre2 helps prevent H2O2-induced <br /> which turn responsible caspases activation, DNA                    cell death Dre2 considered &#226;&#8364;&#732;&#226;&#8364;&#732;anti-death&#226;&#8364;&#8482;&#226;&#8364;&#8482; <br /> degradation, cell death.                                                  factor. This view reinforced fact Ciapin1 been <br />    Blast analysis reveals significant sequence similarity          identified anti-apoptotic factor human cells [23], is <br /> Bax yeast protein, Bax functional homologue                able replace Dre2 efficiently yeast interact with <br /> been characterized date yeast. Heterologous expression               Tah18. <br /> human Bax yeast able induce efficient mitochondria                      As described recently, Dre2 essential Fe/S cluster protein, <br /> targeting cell death Bax conformation stably             containing [2Fe-2S] [4Fe-4S] clusters involved in <br /> changed mutations coding sequence                     Fe/S cluster biogenesis [22]. It shown bacteria that <br /> presence C-terminal tag [17,18]. We Tah18                 DNA binding dimerization Fe/S-containing FNR <br /> galactose-driven overexpression induces Tah18 targeting               protein Escherichia coli regulated oxidation [44]. More <br /> to mitochondria toxicity yeast absence H2O2.                 generally, particular Fe/S clusters, [4Fe-4S] be <br /> Thus, Tah18 targeting mitochondria accurately                       seen oxygen-responsive molecular switches very <br /> controlled redox balance. We artificially targeted GFP-                sensitive oxygen ROS unstable oxidative <br /> Tah18 mitochondrial matrix using Mitochondrial                       conditions [45,46]. A likely hypothesis Dre2 destabilized <br /> Localization Sequence (MLS) absence oxidative stress.               presence H2O2, turn affect Dre2-Tah18 <br /> This strain viable presence wild-type               complex stability. <br /> Tah18 exhibited strong mitochondrial GFP fluorescence                      Thus, suggest Tah18&#226;&#8364;&#8482;s pro-death function yeast is <br /> (not shown), showing Tah18 toxic located              promoted oxidative stress destabilizing interaction <br /> mitochondria absence oxidative stress, suggesting               Dre2 allowing efficient targeting mitochondria. <br /> that Tah18 need modified oxidative stress                        Tah18 mutants resistant wild-type H2O2. <br /> relocalize mitochondria toxic, similarly Bax.                Interestingly, yeast pretreated low doses H2O2 an <br />    From molecular point view, signal triggers Tah18             increased resistance subsequent higher concentration of <br /> relocalization mitochondria unknown prediction                  hydrogen peroxide [47] activation detoxification <br /> software did detect Mitochondrial Localization Sequence               genes controlled transcription factor Yap1 [48]. <br /> in Tah18. Different mechanisms described lead               Mutants defective adaptation hydrogen peroxide were <br /> non-resident protein translocation mitochondria                selected exhibited constitutive sensitivity to <br /> stress. In case human CDK11 p110 isoform [40],                     chronic H2O2 higher resistance acute H2O2 probably due <br /> human Bid (another pro-apoptotic Bcl2 family member) [41,42],                 unbalanced NADPH production, overproduction of <br /> or yeast Mcd1 [21], protein cleavage responsible mitochon-             reduced glutathione [49]. It possibility tah18-5H8 and <br /> drial translocation. It unlikely Tah18 generates cleavage           tah18-5I5 mutants&#226;&#8364;&#8482; redox balance affected though, <br /> product oxidative stress, size GFP-Tah18                  unlike previously described mutants Ng et al. [49], are <br /> detected Western blotting mitochondrial extracts H2O2             sensitive chronic H2O2 exposure (unpublished data). <br /> exposure length protein. Translocation               Increased detoxification hydrogen peroxide stabilize <br />  <br />  <br />        PLoS ONE | www.plosone.org                                        10                           February 2009 | Volume 4 | Issue 2 | e4376 <br />                                                                                                                    Role Dre2-Tah18 H2O2 <br />  <br />  <br /> Dre2-Tah18 interaction tah18 mutants avoid Tah18                  Strains media <br /> relocalization mitochondria, observed using fluorescent                All yeast strains congenic, excepted EGY48 (Clontech) and <br /> mutated Tah18 versions.                                                       12E6 (W303). <br />    We investigated Tah18&#226;&#8364;&#8482;s pro-death function                 The TRP1-CYH2S disruption cassette described <br /> be revealed using kinds stresses. No GFP-Tah18                       previously [24]. <br /> relocalization mitochondria observed gamma-                  Tah18-5H8 tah18-5I5 strains 5H8 5I5: The TAH18 <br /> UV irradiation, camptothecin treatment (not shown). Also pol3-             gene amplified PCR mutagenic conditions as <br /> 13 mutants shifted non-permissive temperature (37uC)               described previously [54]. The amplified TAH18 DNA fragments <br /> 4 hours showed GFP-Tah18 relocalization,                     integrated yeast genome TAH18 locus by <br /> these conditions, 80% pol3-13 cells die, arguing             transformation 5C8 strain selection cycloheximide- <br /> that Tah18 pro-death function involved death cells           resistant transformants 28uC described previously [24]. <br /> exposed gamma rays, UV, CPT pol3-13 cells (not shown).               Transformants replica plated tryptophan minus (trp-) <br />    It informative identify molecular                 plates verify cassette replacement, plates <br /> mechanisms leading Tah18 relocalization mitochondria                containing 5-fluoroorotic acid counter select pRS416- <br /> after exposure H2O2, affected tah18 mutants.             URA3-TAH18 plasmid. Temperature-sensitive mutants were <br />                                                                               selected replicating complete medium plates 28uC and <br /> What essential Dre2-Tah18 function absence                      37uC. <br /> of oxidative stress?                                                             Dre2P221S strain: mutated allele dre2P221S amplified by <br />    The Tah18 essential function affected mutants             PCR original strain [1] using oligos 85 86, was <br /> used study tah18-5H8 tah18-5I5, unable             transformed directly strain 5C6 CycloR selection and <br /> grow 37uC. This function necessary case impaired              DRE2 plasmid shuffling 5-FOA containing plates. Mutated dre2 <br /> DNA replication mutants synthetic lethal pol3-13             allele sequenced control. <br /> (unpublished data). As mentioned above, tah18 mutant                     Strain 9D9: PCR product using oligos 162 163 (see Table 2) <br /> was identified genetic screen sensitivity mutated allele        pFA6a-GFP(S65T)-His3MX6 vector [55] used create <br /> of TOPI, top1T722A, mimicking cytotoxic action campto-                 DRE2-GFP fusion; expression verified Western blot <br /> thecin [25,50] indicate role Tah18 DNA                    using anti-GFP antibody (Roche). <br /> repair/DNA integrity check. Tah18-5H8 tah18-5I5 mutants                      Strain 10H8: PCR product using oligos 233 234 (see <br /> are sensitive CPT permissive temperature (this work             Table 2) pFA6a-TRP1-PGAL1-10-GFP [55] used to <br /> Fig. 1-B). Still, possible function Tah18           replace endogenous TAH18 promoter. GFP-Tah18 expression was <br /> required overcome stress generated presence             verified Western blot using anti-GFP antibody (Roche). <br /> top1T722A 35uC pol3-13 mutants. This function                       Strain 12C2: PCR product using oligos 235 236 (see <br /> compatible essential redox activity suggested               Table 2) pFA6a-TRP1-PGAL1-10-HA [55] used to <br /> functional blocks identified Tah18 (Fig. 1-D) remains            replace endogenous TAH18 promoter. HA-Dre2 expression was <br /> identified.                                                                   verified Western blot using anti-HA antibody (Roche). <br />    The Tah18-interacting protein Dre2 reported                         Strain 12E3: PCR product containing DRE2 promoter plus <br /> involved cytosolic Fe/S protein biogenesis [22]. Interestingly,            DRE2 gene using oligos 166 269 (see Table 2) and <br /> the authors suggest possible role Dre2 iron binding              subcloned replicative pRS415 vector using XhoI-PstI restric- <br /> iron reduction need source electrons reductase            tion sites. PCR product containing HA-TAH18 using oligos 278 <br /> activity. It tempting speculate Tah18 able provide           279 (see Table 2) subcloned previous vector using <br /> electrons reaction, shown Tah18 Dre2             PstI-NotI restriction sites obtain pRS415-DRE2TAH18. 5C6 <br /> interact vivo. Therefore, possible Dre2-Tah18               strain transformed plasmid pRS416- <br /> complex involved iron reduction. This raises             DRE2-URA3 plasmid chased 5-FOA medium, before <br /> possibility release bound iron Dre2                 crossed 5C8 strain. pRS416-TAH18-URA3 chased <br /> affected interaction Tah18, resulting increased             diploid strain sporulation, Dtah18:: TRP1- <br /> production hydroxyl radicals Fenton reaction. A separate           CYH2S dre2D:: TRP1-CYH2S pRS415-DRE2TAH18 spore was <br /> study link DNA replication Fe/S clusters                   selected. Dre2Tah18 expression verified Western blot using <br /> [51], genetic link POL3 NBP35,                anti-HA antibody (Roche). <br /> another Fe/S cluster protein involved Fe/S proteins                      Strain 12E8 obtained crossing strains 12C2 11B3. <br /> biogenesis cytosol [1,52]. Still, exact relationship <br />                                                                                  Strain 12F2 obtained crossing strains 12E6 11B3. <br /> between Dre2-Tah18 DNA metabolism largely unknown <br />                                                                                  Yeast strains grown rich YP medium (1% yeast extract, <br /> despite preliminary evidence [1,25,26,53] further <br />                                                                               Difco, 2% bacto peptone, Difco) supplemented glucose (2%, <br /> explored. <br />                                                                               YPD) galactose (2%, YPGal) minimum medium YNB <br />    In summary, study shows Tah18 deleterious effects <br />                                                                               (0.67% yeast nitrogen base, Difco) supplemented appropriate <br /> on mitochondria presence lethal doses H2O2, and <br />                                                                               amino acids (40 mg/ml each) glucose (2%) galactose (2%). <br /> promotes cell death. The Dre2-Tah18 complex seen a <br />                                                                               Agar (2%) (Difco) added growth plates. H2O2 CPT <br /> molecular sensor high oxidative stress levels accurately <br />                                                                               purchased Sigma. <br /> controls oxidative induced cell death pathway yeast. <br />                                                                               Cloning <br /> Materials Methods <br />                                                                                  All vectors used study described [55,56]. <br />   Sequence analysis performed ncbi server (http://                   Human Ciapin1 ORF amplified using oligos 160 161 <br /> www.ncbi.nlm.nih.gov/), search potential mitochondrial tar-               (see Table 2) using human cDNA library Hela cells <br /> geting sequence performed using TOMPRED (http://ihg.gsf.                  (kind gift Olga Grigorieva) subcloned pRS315-Tef1 <br /> de/ihg/mitoprot.html).                                                        pRS425-Tef1 promoter using SpeI-NotI. <br />  <br />  <br />        PLoS ONE | www.plosone.org                                        11                         February 2009 | Volume 4 | Issue 2 | e4376 <br />                                                                                                                Role Dre2-Tah18 H2O2 <br />  <br />  <br />  <br />  Table 2. Oligos table. <br />  <br />  <br />  Oligo         Sequence <br />  <br />  85            GCATATACACTGTAAGTG <br />  86            CCAATTGACGTCATTTAC <br />  160           CGGACTAGTATGGCAGATTTTGGGATCTC <br />  161           ATAAGAATGCGGCCGCCTAGGCATCATGAAGATTG <br />  162           GGTCTTCCTGCTTTCAAGCCTGGTCAACCTATCAATTTGGACAGCATTTCAGATGACTTGCGGATCCCCGGGTTAATTAA <br />  163           AAATTCACCTTCACCAAAGTAGACCAATTGACGTCATTTACTGAAACGAATGTGCAGGGTGAATTCGAGCTCGTTTAAAC <br />  166           CCGCTCGAGCTGGAAAAATACTCATTAGGC <br />  169           GCTCTAGAACTACGGTAAGCGTAACCACGACCAGGACCTTGACCTTGACCTTGACCCTCACCTTTGTATAGTTCATCCATGCC <br />  172           CCGCTCGAGGTCGATAGCTTTACGCATCCG <br />  173           AACTGCAGGCAGCGATTAATGGAAACACAG <br />  174           AACTGCAGATGAGTAAAGGAGAAGAACTTTTCACTGG <br />  176           GCTCTAGATCATCGAGCAAGAAAATCGTCATCC <br />  177           ATAAGAATGCGGCCGCCTACCAAGTTTCCTGGATATATC <br />  214           CGGAATTCCGGATGTCACAATACAAAACTG <br />  215           CCGCTCGAGCGGTTACAAGTCATCTGAAATG <br />  216           CGGAATTCCGGATGTCATCGAGCAAGAAAATCG <br />  217           CCGCTCGAGCGGCTACCAAGTTTCCTGGATATATC <br />  233           ATGACTACATAAAAAAAAAAAATTCGAAAAATTTCATGAGCCTTATTAACGAATTCGAGCTCGTTTAAAC <br />  234           GCGTTACCTGTCTCCGATCCATAGAGGATGACGATTTTCTTGCTCGATGACATTTTGTATAGTTCATCCATGC <br />  235           GATTAGTAATCATTTAATTTCATGACATATTAAAAAGGTATTCTAAGCAGGAATTCGAGCTCGTTTAAAC <br />  236           GTAGTCACCGCCGGATGTATTAAAAGTAAACCAGTTTTGTATTGTGACATTTTGTATAGTTCATCCATGC <br />  269           CGCCTGCAGCAAGTCATCTGAAATGCTGTCC <br />  278           CGGCTGCAGTACCCATACGATGTTCCAGATTACGCTGGTGAGGGTCAAGGTCAAGGTCAAGGTCCTGGTCGTGGTTACGCTTACCGTAGTATGTC <br />                ATCGAGCAAGAAAATCG <br />  279           ATAAGAATGCGGCCGCCTACCAAGTTTCCTGGATATATC <br />  313           CGGAATTCCGGATGGCAGATTTTGGGATCTC <br />  314           CCGCTCGAGCGGCTAGGCATCATGAAGATTGC <br />  <br />  doi:10.1371/journal.pone.0004376.t002 <br />  <br />  <br />    PRS305-PromTAH18-GFP-TAH18 constructed ampli-                   detected using various antibodies (anti-GFP 1/1000 (Roche), anti- <br /> fying independently TAH18 promoter region, GFP ORF,               Cyc1 anti-Mdh1 (kind gifts Delphine Bernard from <br /> and TAH18 ORF PCR. The TAH18 promoter region                   Pfanner lab respectively). Anti-PCNA serum Anti-Dre2 sera <br /> amplified using oligos 172 173 (see Table 2) cloned          raised rabbit (Agrobio company) using 66His-tagged <br /> pRS305 using XhoI PstI sites. The TAH18 ORF amplified             PCNA 66His-tagged Dre2 1-133 N-terminus overexpressed <br /> using oligos 176 177 cloned previous vector using        E. coli purified nickel column (Qiagen). Primary <br /> XbaI NotI sites. The GFP gene amplified using oligos 174          antibodies detected secondary horseradish peroxidase- <br /> and 169 pFA6a-GFP(S65T)-His3MX6 vector [55], PCR                 conjugated antibody (Jackson). The product reaction was <br /> product restricted PstI XbaI subcloned            detected enhanced chemiluminescence (Amersham) accord- <br /> previous vector. The resulting vector linearized TAH18         ing manufacturer&#226;&#8364;&#8482;s instructions. <br /> promoter using SpeI integrated TAH18 locus the <br /> Dtah18:: TRP1-CYH2S pRS416-TAH18 strain 5C8. The URA3-                    Protein extracts <br /> plasmid containing wild-type TAH18 shuffled 5-                 Mitochondria purified using sucrose gradients previ- <br /> FOA plates strain verified southern blot. GFP-             ously described [32], DTT-containing wash buffer was <br /> Tah18 expression verified Western blot using monoclonal          used. <br /> anti-GFP antibody (Roche).                                                   For co-immunoprecipitation experiments, cells grown <br />    Multicopy 2m-based yeast DNA library YEp13 vector              exponentially YPD washed PBS buffer glass <br /> (LEU2 marker) kind gift B. Daignan-Fornier.                    beads lysis co-IP buffer (50 mM HEPES-KOH pH 7.5, <br />                                                                           140 mM NaCl, 1 mM EDTA, 1% Triton X-100, 0.1% Na- <br /> Western blotting                                                          Deoxycholate, proteases inhibitors (Roche)). The lysate was <br />   Protein extracts prepared using classical TCA methods              clarified centrifugation 15 000 rpm 10 min. 20 ml of <br /> [57]. Proteins transferred 10% acrylamide gels             HA-coated agarose beads (Sigma) added incubation <br /> ECL membrane according manufacturer&#226;&#8364;&#8482;s instructions                 slow agitation 4uC 4 hours. Beads washed three <br /> (Amersham). The membrane blocked 5% dried milk                times co-IP buffer resuspended Laemmli buffer before <br /> Tris buffered saline containing 0.1% Tween 20. Proteins              Western blot analysis. <br />  <br />  <br />        PLoS ONE | www.plosone.org                                    12                         February 2009 | Volume 4 | Issue 2 | e4376 <br />                                                                                                                                              Role Dre2-Tah18 H2O2 <br />  <br />  <br />  <br /> Fluorescence microscopy                                                                      lacZ]. EGY48[p8op-lacZ] co-transformed the <br />    Cells washed PBS mounted directly glass                                different fusion plasmids used study clones were <br /> slides covered polylysine (Sigma) drop mounting                                 assayed growth blue colour GR GRLX media, <br /> solution (75% glycerol PBS containing DAPI 0.05 mg/ml). Data                              according manufacturer&#226;&#8364;&#8482;s instructions. <br /> were acquired using Leica DM RXA microscope, equipped with <br /> a piezoelectric translator (PIFOC; PI) placed base 1006                          Acknowledgments <br /> PlanApo N.A. 1.4 objective, 5 MHz Micromax 1300Y <br />                                                                                              We thank Sarah Flocan Danny Pirotte excellent expertise media <br /> interline CCD camera (Roper Instruments). For acquisition of <br />                                                                                              preparation, Serge Urbach anti-PCNA antibody, Stephane Hermans <br /> Z-series, stacks fluorescence images collected automatically                         designing custom devices laboratory, Fabrice Cordelie`res for <br /> at 0.2 mm Z-distance (Metamorph software; Molecular Devices).                                invaluable help using 3D fluorescence microscope. We thank Baldacci and <br /> Data processed using Image J software.                                                  Huang&#226;&#8364;&#8482;s lab members helpful discussions work, and <br />                                                                                              Janet Hall Jason Martin critically reading manuscript. We <br /> Yeast Two-hybrid                                                                             thank INSERM, CNRS Institut Curie continuous financial <br />   We used LexA-based two-hybrid technology (Clontech). DRE2,                                 support. <br /> Ciapin1 TAH18 ORFs amplified PCR using oligos <br /> 214, 215, 313 314 (see Table 1), cloned Activating                              Author Contributions <br /> domain containing plasmid pB42 using EcoRI-XhoI. TAH18                                   Conceived designed experiments: LV CF. Performed the <br /> amplified using oligos 216 217 cloned Binding                                   experiments: LV CF ED NS. Analyzed data: LV CF ED NS RC <br /> domain containing plasmid pGilda using EcoRI-XhoI. Strain                                    BG GF GB. Contributed reagents/materials/analysis tools: LV CF ED NS <br /> EGY48 transformed p8op-lacZ create EGY48[p8op-                                   GF. Wrote paper: LV. <br />  <br /> References <br />  1. Chanet R, Heude M (2003) Characterization mutations synthetic                21. Yang H, Ren Q, Zhang Z (2008) Cleavage Mcd1 Caspase-like Protease <br />     lethal pol3-13, mutated allele DNA polymerase delta Saccharomyces               Esp1 Promotes Apoptosis Budding Yeast. Mol Biol Cell 19: 2127&#226;&#8364;&#8220;2134. <br />     cerevisiae. Curr Genet 43: 337&#226;&#8364;&#8220;350.                                                      22. Zhang Y, Lyver ER, Nakamaru-Ogiso E, Yoon H, Amutha B, et al. (2008) <br />  2. Vachova L, Palkova Z (2005) Physiological regulation yeast cell death                  Dre2, Conserved Eukaryotic Fe/S Cluster Protein, Functions Cytosolic Fe/ <br />     multicellular colonies triggered ammonia. J Cell Biol 169: 711&#226;&#8364;&#8220;717.                    S Protein Biogenesis. Mol Cell Biol 28: 5569&#226;&#8364;&#8220;5582. <br />  3. Ivanovska I, Hardwick JM (2005) Viruses activate genetically conserved cell            23. Shibayama H, Takai E, Matsumura I, Kouno M, Morii E, et al. (2004) <br />     death pathway unicellular organism. J Cell Biol 170: 391&#226;&#8364;&#8220;399.                           Identification cytokine-induced antiapoptotic molecule anamorsin essential <br />  4. Madeo F, Frohlich E, Ligr M, Grey M, Sigrist SJ, et al. (1999) Oxygen stress:              definitive hematopoiesis. J Exp Med 199: 581&#226;&#8364;&#8220;592. <br />     regulator apoptosis yeast. J Cell Biol 145: 757&#226;&#8364;&#8220;767.                               24. Simon M, Giot L, Faye G (1993) A random mutagenesis procedure: application <br />  5. Ludovico P, Sousa MJ, Silva MT, Leao C, Corte-Real M (2001) Saccharomyces                    POL3 gene Saccharomyces cerevisiae. Gene 127: 139&#226;&#8364;&#8220;144. <br />     cerevisiae commits programmed cell death process response acetic              25. Reid RJ, Fiorani P, Sugawara M, Bjornsti MA (1999) CDC45 DPB11 are <br />     acid. Microbiology 147: 2409&#226;&#8364;&#8220;2415.                                                           required processive DNA replication resistance DNA topoisomerase I- <br />  6. Ludovico P, Sansonetty F, Silva MT, Corte-Real M (2003) Acetic acid induces                mediated DNA damage. Proc Natl Acad Sci U S A 96: 11440&#226;&#8364;&#8220;11445. <br />     programmed cell death process food spoilage yeast Zygosaccharomyces               26. Fiorani P, Reid RJ, Schepis A, Jacquiau HR, Guo H, et al. (2004) The <br />     bailii. FEMS Yeast Res 3: 91&#226;&#8364;&#8220;96.                                                             deubiquitinating enzyme Doa4p protects cells DNA topoisomerase I <br />  7. Huh GH, Damsz B, Matsumoto TK, Reddy MP, Rus AM, et al. (2002) Salt                          poisons. J Biol Chem 279: 21271&#226;&#8364;&#8220;21281. <br />     causes ion disequilibrium-induced programmed cell death yeast plants.             27. Stewart L, Redinbo MR, Qiu X, Hol WG, Champoux JJ (1998) A model the <br />     Plant J 29: 649&#226;&#8364;&#8220;659.                                                                         mechanism human topoisomerase I. Science 279: 1534&#226;&#8364;&#8220;1541. <br />  8. Wadskog I, Maldener C, Proksch A, Madeo F, Adler L (2004) Yeast lacking              28. Pourquier P, Pommier Y (2001) Topoisomerase I-mediated DNA damage. Adv <br />     SRO7/SOP1-encoded tumor suppressor homologue increased suscepti-                        Cancer Res 80: 189&#226;&#8364;&#8220;216. <br />     bility apoptosis-like cell death exposure NaCl stress. Mol Biol Cell 15:        29. Burkart W, Jung T, Frasch G (1999) Damage pattern function radiation <br />     1436&#226;&#8364;&#8220;1444.                                                                                   quality factors. C R Acad Sci III 322: 89&#226;&#8364;&#8220;101. <br />  9. Del Carratore R, Della Croce C, Simili M, Taccini E, Scavuzzo M, et al. (2002)           30. Maris AF, Kern AL, Picada JN, Boccardi F, Brendel M, et al. (2000) <br />     Cell cycle morphological alterations indicative apoptosis promoted              Glutathione, transcription factor Yap1, required carbon source- <br />     UV irradiation S. cerevisiae. Mutat Res 513: 183&#226;&#8364;&#8220;191.                                     dependent resistance oxidative stress Saccharomyces cerevisiae. Curr Genet <br /> 10. Severin FF, Hyman AA (2002) Pheromone induces programmed cell death S.                    37: 175&#226;&#8364;&#8220;182. <br />     cerevisiae. Curr Biol 12: R233&#226;&#8364;&#8220;235.                                                      31. Macierzynska E, Grzelak A, Bartosz G (2007) The effect growth medium on <br /> 11. Madeo F, Frohlich E, Frohlich KU (1997) A yeast mutant showing diagnostic                    antioxidant defense Saccharomyces cerevisiae. Cell Mol Biol Lett [Epub <br />     markers early late apoptosis. J Cell Biol 139: 729&#226;&#8364;&#8220;734.                               ahead print]. <br /> 12. Ye Y, Meyer HH, Rapoport TA (2003) Function p97-Ufd1-Npl4 complex                 32. Meisinger C, Pfanner N, Truscott KN (2006) Isolation yeast mitochondria. <br />     retrotranslocation ER cytosol: dual recognition                        Methods Mol Biol 313: 33&#226;&#8364;&#8220;39. <br />     nonubiquitinated polypeptide segments polyubiquitin chains. J Cell Biol              33. Stridh H, Kimland M, Jones DP, Orrenius S, Hampton MB (1998) Cytochrome <br />     162: 71&#226;&#8364;&#8220;84.                                                                                  c release caspase activation hydrogen peroxide- tributyltin-induced <br /> 13. Madeo F, Herker E, Maldener C, Wissing S, Lachelt S, et al. (2002) A caspase-                apoptosis. FEBS Lett 429: 351&#226;&#8364;&#8220;355. <br />     related protease regulates apoptosis yeast. Mol Cell 9: 911&#226;&#8364;&#8220;917.                      34. Ludovico P, Rodrigues F, Almeida A, Silva MT, Barrientos A, et al. (2002) <br /> 14. Wissing S, Ludovico P, Herker E, Buttner S, Engelhardt SM, et al. (2004) An                  Cytochrome c release mitochondria involvement programmed cell death <br />     AIF orthologue regulates apoptosis yeast. J Cell Biol 166: 969&#226;&#8364;&#8220;974.                       induced acetic acid Saccharomyces cerevisiae. Mol Biol Cell 13: <br /> 15. Suzuki Y, Imai Y, Nakayama H, Takahashi K, Takio K, et al. (2001) A serine                   2598&#226;&#8364;&#8220;2606. <br />     protease, HtrA2, released mitochondria interacts XIAP,              35. Dutta C, Henry HL (1990) Detection hemoprotein peroxidase activity on <br />     inducing cell death. Mol Cell 8: 613&#226;&#8364;&#8220;621.                                                    polyvinylidene difluoride membrane. Anal Biochem 184: 96&#226;&#8364;&#8220;99. <br /> 16. Fahrenkrog B, Sauder U, Aebi U (2004) The S. cerevisiae HtrA-like protein                36. Huh WK, Falvo JV, Gerke LC, Carroll AS, Howson RW, et al. (2003) Global <br />     Nma111p nuclear serine protease mediates yeast apoptosis. J Cell Sci               analysis protein localization budding yeast. Nature 425: 686&#226;&#8364;&#8220;691. <br />     117: 115&#226;&#8364;&#8220;126.                                                                            37. Mumberg D, Muller R, Funk M (1995) Yeast vectors controlled <br /> 17. Manon S, Chaudhuri B, Guerin M (1997) Release cytochrome c decrease                   expression heterologous proteins different genetic backgrounds. Gene 156: <br />     cytochrome c oxidase Bax-expressing yeast cells, prevention               119&#226;&#8364;&#8220;122. <br />     effects coexpression Bcl-xL. FEBS Lett 415: 29&#226;&#8364;&#8220;32.                                 38. Wolter KG, Hsu YT, Smith CL, Nechushtan A, Xi XG, et al. (1997) Movement <br /> 18. Ligr M, Madeo F, Frohlich E, Hilt W, Frohlich KU, et al. (1998) Mammalian                    Bax cytosol mitochondria apoptosis. J Cell Biol 139: <br />     Bax triggers apoptotic changes yeast. FEBS Lett 438: 61&#226;&#8364;&#8220;65.                               1281&#226;&#8364;&#8220;1292. <br /> 19. Pati D, Zhang N, Plon SE (2002) Linking sister chromatid cohesion                    39. Gross A, McDonnell JM, Korsmeyer SJ (1999) BCL-2 family members the <br />     apoptosis: role Rad21. Mol Cell Biol 22: 8267&#226;&#8364;&#8220;8277.                                       mitochondria apoptosis. Genes Dev 13: 1899&#226;&#8364;&#8220;1911. <br /> 20. Chen F, Kamradt M, Mulcahy M, Byun Y, Xu H, et al. (2002) Caspase                        40. Feng Y, Ariza ME, Goulet AC, Shi J, Nelson MA (2005) Death-signal-induced <br />     proteolysis cohesin component RAD21 promotes apoptosis. J Biol Chem                   relocalization cyclin-dependent kinase 11 mitochondria. Biochem J 392: <br />     277: 16775&#226;&#8364;&#8220;16781.                                                                            65&#226;&#8364;&#8220;73. <br />  <br />  <br />  <br />         PLoS ONE | www.plosone.org                                                      13                                February 2009 | Volume 4 | Issue 2 | e4376 <br />                                                                                                                                              Role Dre2-Tah18 H2O2 <br />  <br />  <br /> 41. Luo X, Budihardjo I, Zou H, Slaughter C, Wang X (1998) Bid, Bcl2                            NADPH-generating systems SKN7 transcription factor. Free Radic Biol <br />     interacting protein, mediates cytochrome c release mitochondria                       Med 44: 1131&#226;&#8364;&#8220;1145. <br />     response activation cell surface death receptors. Cell 94: 481&#226;&#8364;&#8220;490.               50.   Fiorani P, Bjornsti MA (2000) Mechanisms DNA topoisomerase I-induced cell <br /> 42. Gross A, Yin XM, Wang K, Wei MC, Jockel J, et al. (1999) Caspase cleaved                      killing yeast Saccharomyces cerevisiae. Ann N Y Acad Sci 922: 65&#226;&#8364;&#8220;75. <br />     BID targets mitochondria required cytochrome c release, BCL-           51.   Klinge S, Hirst J, Maman JD, Krude T, Pellegrini L (2007) An iron-sulfur <br />     XL prevents release tumor necrosis factor-R1/Fas death. J Biol                   domain eukaryotic primase essential RNA primer synthesis. Nat <br />     Chem 274: 1156&#226;&#8364;&#8220;1163.                                                                          Struct Mol Biol 14: 875&#226;&#8364;&#8220;877. <br /> 43. Sawada M, Sun W, Hayes P, Leskov K, Boothman DA, et al. (2003) Ku70                     52.   Hausmann A, Aguilar Netz DJ, Balk J, Pierik AJ, Muhlenhoff U, et al. (2005) <br />     suppresses apoptotic translocation Bax mitochondria. Nat Cell Biol 5:               The eukaryotic P loop NTPase Nbp35: essential component cytosolic <br />     320&#226;&#8364;&#8220;329.                                                                                      nuclear iron-sulfur protein assembly machinery. Proc Natl Acad Sci U S A <br /> 44. Lazazzera BA, Beinert H, Khoroshilova N, Kennedy MC, Kiley PJ (1996) DNA                      102: 3266&#226;&#8364;&#8220;3271. <br />     binding dimerization Fe-S-containing FNR protein Escherichia            53.   Ben-Aroya S, Coombes C, Kwok T, O&#226;&#8364;&#8482;Donnell KA, Boeke JD, et al. (2008) <br />     coli regulated oxygen. J Biol Chem 271: 2762&#226;&#8364;&#8220;2768.                                     Toward comprehensive temperature-sensitive mutant repository the <br /> 45. Flint DH, Smyk-Randall E, Tuminello JF, Draczynska-Lusiak B, Brown OR                         essential genes Saccharomyces cerevisiae. Mol Cell 30: 248&#226;&#8364;&#8220;258. <br />     (1993) The inactivation dihydroxy-acid dehydratase Escherichia coli               54.   Zhou YH, Zhang XP, Ebright RH (1991) Random mutagenesis gene-sized <br />     treated hyperbaric oxygen occurs destruction Fe-S                  DNA molecules use PCR Taq DNA polymerase. Nucleic Acids Res <br />     cluster, enzyme remains cell form reactivated.                19: 6052. <br />     J Biol Chem 268: 25547&#226;&#8364;&#8220;25552.                                                           55.   Longtine MS, McKenzie A 3rd, Demarini DJ, Shah NG, Wach A, et al. (1998) <br /> 46. Flint DH, Tuminello JF, Emptage MH (1993) The inactivation Fe-S cluster                    Additional modules versatile economical PCR-based gene deletion and <br />     containing hydro-lyases superoxide. J Biol Chem 268: 22369&#226;&#8364;&#8220;22376.                          modification Saccharomyces cerevisiae. Yeast 14: 953&#226;&#8364;&#8220;961. <br /> 47. Collinson LP, Dawes IW (1992) Inducibility response yeast cells            56.   Sikorski RS, Hieter P (1989) A shuttle vectors yeast host strains <br />     peroxide stress. J Gen Microbiol 138: 329&#226;&#8364;&#8220;335.                                                designed efficie <br /> </body></html>