<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The Plant Cell, Vol. 20: 1862&#226;&#8364;&#8220;1878, July 2008, www.plantcell.org &#195;&#163; 2008 American Society Plant Biologists <br />  <br />  <br />  <br />  <br /> Sphingolipid Long-Chain <span id='am-396' about='xsp:base' typeof='owl:Thing'>Base</span> Hydroxylation Is Important for <br /> Growth Regulation Sphingolipid Content and <br /> Composition Arabidopsis                                          W <br />  <br />  <br />  <br />  <br /> Ming Chen,1 Jonathan E. Markham,1 Charles R. Dietrich, Jan G. Jaworski, Edgar B. Cahoon2 <br /> Donald Danforth Plant Science Center, Saint Louis, Missouri 63132 <br />  <br />  <br />  <br /> Sphingolipids structural components endomembranes function metabolites bioactive regulators <br /> of cellular processes programmed cell death. A characteristic feature plant sphingolipids high content of <br /> trihydroxy long-chain bases (LCBs) produced LCB C-4 hydroxylase. <span id='am-5' about='protege:TO' typeof='owl:Thing'>To</span> determine functional significance <br /> of trihydroxy LCBs plants, T-DNA double mutants <span id='am-318' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-319' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-320' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-321' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-322' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-323' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-324' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-325' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-326' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-327' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-328' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-330' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-331' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-332' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-333' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-334' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-335' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-337' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-338' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-341' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-342' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-344' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-347' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-348' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-349' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-350' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-351' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-352' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-353' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-354' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-355' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-356' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-357' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-358' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-359' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-361' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-362' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-363' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-364' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-365' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-366' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-367' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-368' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-369' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-370' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-372' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-373' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-374' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-375' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-376' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-377' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-378' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-380' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-381' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-382' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-383' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-384' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-385' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-386' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-387' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-389' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-390' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-391' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-392' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-393' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-394' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-395' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span>RNA interference</span> suppression lines generated two <br /> Arabidopsis thaliana LCB C-4 hydroxylase genes Sphingoid Base Hydroxylase1 (SBH1) SBH2. These plants displayed <br /> reductions growth dependent content trihydroxy LCBs sphingolipids. Double sbh1 sbh2 mutants, <br /> which completely lacked trihydroxy LCBs, severely dwarfed, did progress vegetative reproductive growth, <br /> and enhanced expression programmed cell death associated&#226;&#8364;&#8220;genes. Furthermore, total content sphingolipids <br /> on dry weight basis increased relative amounts trihydroxy LCBs decreased. In trihydroxy LCB&#226;&#8364;&#8220;<span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span>null mutants</span>, <br /> sphingolipid content ;2.5-fold higher wild-type plants. Increases sphingolipid content resulted the <br /> accumulation molecular species C16 fatty acids very-long-chain fatty acids, more <br /> commonly enriched plant sphingolipids, accompanied decreases amounts C16-containing species of <br /> chloroplast lipids. Overall, results indicate trihydroxy LCB synthesis plays central role maintaining growth and <br /> mediating total content fatty acid composition sphingolipids plants. <br />  <br />  <br />  <br />  <br /> INTRODUCTION                                                                    More 200 different sphingolipid molecules occur plants <br />                                                                              (Markham et al., 2006; Markham Jaworski, 2007). These <br /> Sphingolipids structurally diverse molecules major              molecules differ composition head groups, <br /> components endomembrane recently                      degree hydroxylation, numbers positions double <br /> been estimated compose &gt;40% plasma membrane lipids                  bonds. The functional significance immense structural <br /> plants (Sperling et al., 2005). Not sphingolipids provide            heterogeneity plant sphingolipids begun be <br /> structural integrity membranes, believed                 addressed. For example, increases relative amounts cis <br /> contribute organization membrane microdomains                   C-8 unsaturated LCBs shown enhance aluminum <br /> lipid rafts contain proteins glycosylphosphatidyl-              tolerance Arabidopsis thaliana (Ryan et al., 2007). One char- <br /> inositol-anchored proteins, function cell surface&#226;&#8364;&#8220;related           acteristic feature sphingolipids high degree hydrox- <br /> activities including cell wall deposition (Mongrand et al., 2004;            ylation fatty acids LCBs compose ceramide <br /> Borner et al., 2005). Furthermore, sphingolipids function            backbone (Lynch Dunn, 2004). The fatty acids plant <br /> their metabolites regulators key cellular physiological            sphingolipids typically mixture C16 very-long-chain <br /> processes plants. For example, ceramide component                  (C20 C26) moieties contain a- C-2 hydroxy group <br /> sphingolipids shown function mediator                    (Lynch Dunn, 2004). LCBs, derived the <br /> programmed cell death (PCD) (Liang et al., 2003),                    condensation palmitoyl-CoA Ser, contain 18 carbon <br /> sphingolipid long-chain base (LCB) phosphates sphingosine-                   atoms hydroxyl groups. The C-1 C-3 <br /> and phytosphingosine-1-phosphate implicated                     hydroxyl groups arise Ser palmitoyl-CoA precur- <br /> signaling molecules abscisic acid&#226;&#8364;&#8220;dependent guard cell clo-               sors, respectively, hydroxyl group C-4 <br /> sure (Ng et al., 2001; Coursol et al., 2003, 2005).                          position added following synthesis LCB. The C-1 hy- <br />                                                                              droxyl group substituted polar residues form <br />                                                                              complex sphingolipids. In plants, include glucosylcer- <br /> 1 These  authors contributed equally work.                           amides (GlcCers) abundant glycosylinositolphos- <br /> 2 Address  correspondence ecahoon@danforthcenter.org.                     phoceramides (GIPCs) (Kaul Lester, 1978). While dihydroxy <br /> The author responsible distribution materials integral         LCBs, lack C-4 hydroxyl group, typically more <br /> findings presented article accordance policy described   enriched GlcCers GIPCs, trihydroxy LCBs predominate <br /> in Instructions Authors (www.plantcell.org) is: Edgar B. Cahoon <br /> (ecahoon@danforthcenter.org). <br />                                                                              classes Arabidopsis compose nearly 90% the <br /> W <br />   Online version contains Web-only data.                                     LCBs total sphingolipid extract Arabidopsis leaves (Chen <br /> www.plantcell.org/cgi/doi/10.1105/tpc.107.057851                             et al., 2006; Markham et al., 2006). <br />                                                                                                      Sphingolipid LCB Hydroxylation       1863 <br />  <br />  <br />  <br />    In Saccharomyces cerevisiae, C-4 hydroxyl group intro-           <span id='am-162' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-163' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span>phenotype</span> LCB C-4 hydroxylation, change in <br /> duced activity diiron-oxo&#226;&#8364;&#8220;type hydroxylase encoded             sphingolipid LCB composition detected plants <br /> by SUR2 (also known SYR2) gene (Cliften et al., 1996;               (Imamura et al., 2007). To determine function LCB C-4 <br /> Haak et al., 1997; Grilley et al., 1998). A similar enzyme occurs       hydroxylation plants, generated double mutants for <br /> plants, LCB C-4 hydroxylase genes (At1g69640               C-4 hydroxylase genes Arabidopsis completely <br /> At1g14290; Figure 1) Arabidopsis shown restore             lack trihydroxy LCBs generated RNAi lines with <br /> trihydroxy LCB biosynthetic ability S. cerevisiae sur2D mutants         intermediate reductions trihydroxy LCBs. The results pre- <br /> (Sperling et al., 2001). In mammals, C-4 hydroxyl group             sented LCB C-4 hydroxylation critical the <br /> LCBs arises bifunctional desaturase/hydroxylase ac-               growth viability Arabidopsis. The results show <br /> tivity DES2 (Degenerative Spermatocyte2) polypeptide,               synthesis trihydroxy LCBs plays central role the <br /> which distantly related S. cerevisiae plant C-4         maintenance sphingolipid fatty acid composition total <br /> hydroxylases (Ternes et al., 2002; Omae et al., 2004).                     sphingolipid content Arabidopsis. A model presented that <br />    The significance sphingolipid LCB C-4 hydroxylation              rationalizes contributions trihydroxy LCBs growth and <br /> been studied S. cerevisiae recently filamen-            regulation sphingolipid biosynthesis. <br /> tous fungus Aspergillus nidulans (Li et al., 2007). S. cerevisiae <br /> mutants lack LCB C-4 hydroxyl group display <br /> growth phenotypes, indicating hydroxyl moiety          RESULTS <br /> essential structural feature S. cerevisiae sphingolipids (Haak <br /> et al., 1997). Loss LCB C-4 hydroxylation, however, does                Expression SBH1 SBH2 Subcellular Localization <br /> confer resistance Pseudomonas syringae toxin syringo-               SBH1 SBH2 Polypeptides <br /> mycin (Cliften et al., 1996; Grilley et al., 1998). This resistance <br /> apparently arises alterations physical properties           Arabidopsis contains homologs S. cerevisiae LCB C-4 <br /> the plasma membrane mitigate channel-forming activity             hydroxylase, At1g69640 At1g14290, designated Sphingoid <br /> of syringomycin (Idkowiak-Baldys et al., 2004). In contrast S.        Base Hydroxylase1 (SBH1) SBH2, respectively. Each gene <br /> cerevisiae, LCB C-4 hydroxylation essential viability        shown previously restore trihydroxy LCB synthesis <br /> A. nidulans, trihydroxy LCBs shown contribute             expressed S. cerevisiae LCB C-4 hydroxylase mutants <br /> to cell wall formation transition asexual sexual           (Sperling et al., 2001). The organ-specific expression patterns of <br /> growth organism (Li et al., 2007). In mammals, trihydroxy          SBH1 SBH2 examined step address- <br /> LCBs relatively minor components sphingolipids             ing possible functional redundancy genes. RNA gel <br /> formed dual activity DES2 (Ternes et al., 2002; Omae             blot analyses indicated SBH1 SBH2 ubiqui- <br /> et al., 2004). As result, function trihydroxy LCBs         tously expressed Arabidopsis highly expressed in <br /> be dissected C-4 unsaturated LCBs mammals.                 flowers roots (Figure 2A). Of genes, SBH1 more <br />    Prior studies described here, functional significance        highly expressed Arabidopsis plant. Consistent <br /> of LCB C-4 hydroxylation plants character-            this, higher b-glucuronidase (GUS) activity detected in <br /> ized. A recent report, example, showed RNA interference           leaves cotyledons plants transformed SBH1 <br /> (RNAi) antisense downregulation LCB C-4             promoter&#226;&#8364;&#8220;GUS fusion compared transformed an <br /> hydroxylase genes rice (Oryza sativa) result reduced fertility       SBH2 promoter&#226;&#8364;&#8220;GUS fusion (Figures 2B 2L). <br /> (Imamura et al., 2007). However, possible link             Although similarity expression patterns SBH1 and <br />                                                                            SBH2 suggests redundancy organ level, possible that <br />                                                                            SBH1 SBH2 polypeptides reside different locations <br />                                                                            cell contribute, example, hydroxylation of <br />                                                                            LCBs different sphingolipid classes. To address possi- <br />                                                                            bility, SBH1 SBH2 fused C termini cyan <br />                                                                            fluorescent protein (CFP), transiently expressed in <br />                                                                            tobacco (Nicotiana tabacum) leaves. Analyses Agrobac- <br />                                                                            terium tumefaciens&#226;&#8364;&#8220;infiltrated leaves confocal microscopy <br />                                                                            revealed SBH1 SBH2 colocalize fluorescent <br />                                                                            protein marker containing endoplasmic reticulum (ER) chi- <br />                                                                            tinase signal peptide (CSP) HDEL ER retention sequence <br />                                                                            (CSP-YFP-HDEL) (Chen et al., 2006), plasma <br />                                                                            membrane dye FM4-64. This indicates Arabidopsis <br />                                                                            LCB C-4 hydroxylases ER-localized (Figure 3; Supple- <br />                                                                            mental Figure 1 online). Both N- C-terminal fusions SBH1 <br /> Figure 1. Example Reaction Catalyzed LCB C-4 Hydroxylase.          SBH2 yellow fluorescent protein (YFP) CFP able <br /> The conversion dihydroxy LCB sphinganine (d18:0) trihydroxy LCB      restore C-4 hydroxylation S. cerevisiae sur2D mutant <br /> 4-OH-sphinganine (or phytosphingosine; t18:0) activity LCB C-4   containing <span id='am-240' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-241' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-250' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-251' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-256' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-258' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-259' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-262' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-272' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-290' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-316' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-317' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span>knockout</span> LCB C-4 hydroxylase gene, indi- <br /> hydroxylase. Arabidopsis contains genes, SBH1 (At1g69640)          cating hydroxylase fusions retain enzymatic activ- <br /> SBH2 (At1g14290), encode distinct LCB C-4 hydroxylase isoforms.       ities (see Supplemental Figure 2 online). <br />  1864      The Plant Cell <br />  <br />  <br />  <br />  <br /> Figure 2. Gene Expression Analyses SBH1 SBH2. <br /> (A) Gene expression SBH1 SBH2 different organs wild-type plant (Col-0) revealed RNA gel blot analysis. RNA loading lane <br /> is indicated ethidium bromide&#226;&#8364;&#8220;stained rRNA. Shown representative blots technical replicates. <br /> (B) (L) Localization GUS activity Arabidopsis plants transformed SBH1 SBH2 promoter-GUS fusions. <br /> (B) (F) Promoter-GUS activities SBH1. <br /> (B) Ten-day-old seedling grown MS plates. <br /> (C) Cotyledon. <br /> (D) Emerging young leaf. <br /> (E) Root. <br /> (F) Rosette leaf 4-week-old soil-grown plant. <br /> (G) (L) Promoter-GUS activities SBH2. <br /> (G) Ten-day-old seedling grown MS plates. <br /> (H) Cotyledon. <br /> (I) Emerging young leaf. <br /> (J) (K) Roots. <br /> (L) Rosette leaf 4-week-old soil-grown plant. <br /> Bars = 1 mm (B), (G), (F), (L) 500 mm (C) (E) (H) (K). <br />  <br />  <br />  <br />    Overall, constitutive expression LCB C-4 hydrox-              SALK_047916, designated sbh2-1, SALK_032139, designated <br /> ylase genes Arabidopsis localization peptides             sbh2-2, SALK_024105, designated sbh2-3 (Figure 4A). Each <br /> in ER suggest SBH1 SBH2 functionally redun-                 mutant determined <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span>null allele</span> based RT-PCR <br /> dant.                                                                        (Figure 4B). Double mutants SBH genes were <br />                                                                              obtained crossing sbh1-1 sbh2-1 crossing of <br /> Relative Contributions SBH1 SBH2 LCB                               sbh1-2 sbh2-2, followed self-fertilization progeny <br /> C-4 Hydroxylation                                                            crosses. As expected, expression SBH1 SBH2 <br />                                                                              detected double mutants, determined RT- <br /> The LCB composition T-DNA mutants SBH1 SBH2                       PCR analysis (see Supplemental Figure 3 online). <br /> was measured assess relative contributions                       LCB compositions measured HPLC following hydro- <br /> gene C-4 hydroxylation Arabidopsis plants. For                   lysis total sphingolipid extract fluorescent derivatiza- <br /> studies, multiple independent T-DNA mutants obtained                    tion released LCBs. Initially, relative content of <br /> for gene: (1) SBH1, SALK_090881, designated sbh1-1,                 trihydroxy dihydroxy LCBs total sphingolipid extract <br /> and SAIL_1292_E09, designated sbh1-2; (2) SBH2,                      different organs 8-week-old wild-type, sbh1, sbh2 <br />                                                                                                       Sphingolipid LCB Hydroxylation    1865 <br />  <br />  <br />  <br />  <br /> Figure 3. Subcellular Localization SBH1 SBH2. <br />  <br /> (A) Distribution SBH1-CFP. <br /> (B) Distribution ER marker CSP-YFP-HDEL. <br /> (C) Merge images (A) (B), shows colocalization SBH1-CFP ER marker. <br /> (D) Distribution SBH2-CFP. <br /> (E) Distribution ER marker CSP-YFP-HDEL. <br /> (F) Merge images (D) (E), shows colocalization SBH2-CFP ER marker. <br /> Images additional marker controls shown Supplemental Figure 1 online. Bars = 10 mm. <br />  <br />  <br />  <br /> plants compared. Trihydroxy LCBs, principally form             F2 progeny obtained self-fertilization sbh1-1+/2 sbh2- <br /> of cis/trans isomers 4-hydroxy-8-sphingenine (t18:1) 4             1+/2 plant markedly smaller siblings. Similar <br /> hydroxysphinganine (t18:0), accounted 80 90% total          alterations growth observed segregating F2 <br /> LCBs sphingolipids roots, leaves, flowers, stems,               population crosses sbh1-1 sbh1-2 allele the <br /> siliques wild-type plants (Figure 5A). In sbh1-1 mutant,           sbh2-1, sbh2-2, sbh2-3 allele. Double mutants displayed <br /> the relative content trihydroxy LCBs reduced 50 60%          reduced vigor usually died expansion true <br /> of LCBs sphingolipids organs analyzed               leaves grown soil. In addition reduced size, these <br /> (Figure 5A). By contrast, reductions relative content           plants developed lesions near center cotyledons <br /> trihydroxy LCBs sbh2-1 mutant limited primarily            (Figure 6A, arrowhead). <br /> roots, stems, siliques, little decrease leaves             Despite inability transition reproductive growth, <br /> compared wild-type plants (Figure 5A). A similar <span id='am-398' about='xsp:pattern' typeof='owl:Thing'>pattern</span>          double mutants obtained segregating populations <br /> alterations trihydroxy LCB content observed            progeny sbh1-1+/2 sbh2-12/2 parental line. These <br /> sbh1-2, sbh2-2, sbh2-3 mutants. The larger contribution            plants displayed extended viability maintained agar <br /> SBH1 LCB C-4 hydroxylation different organs                  plates nonetheless reduced size relative their <br /> Arabidopsis consistent higher expression           siblings, necrotic lesions observed cotyle- <br /> Arabidopsis, demonstrated RNA gel blot analyses (Figure             dons (Figure 6B, arrow). These plants small, curled leaves <br /> 2A). The sbh1-1 sbh2-1 double mutant contained detectable              short petioles resulted compact, epinastic mor- <br /> amounts trihydroxy LCBs (Figure 5B) instead en-                phology rosette (Figures 6C 6D). The inability bolt <br /> riched cis/trans isomers D8-dihydrosphingenine (d18:1)          observed double mutants maintained plates 6 <br /> sphinganine (d18:0). This result indicates combined              weeks more, wild-type plants typically bolted 4 <br /> expression SBH1 SBH2 accounts LCB C-4 hydrox-              weeks growth conditions. The addition trihydroxy <br /> ylation Arabidopsis.                                                   LCB phytosphingosine (t18:0) medium concentrations <br />                                                                           200 mM unable restore growth sbh1-1 <br /> Growth Phenotypes Arabidopsis sbh Mutants                              sbh2-1 plants, higher concentrations t18:0 used <br />                                                                           adverse effects growth viability wild- <br /> No obvious growth defects observed sbh1-1                    type plants (Abbas et al., 1994). However, growth sbh1-1 <br /> sbh2-1 mutant plants grown soil standard                sbh2-1 mutants appeared restored wild-type <br /> growth conditions. However, sbh1-1 sbh2-1 double mutants                  plants genetic complementation wild-type copy of <br /> displayed severe alterations growth. Approximately 1/16th           SBH1. <br />  1866      The Plant Cell <br />  <br />  <br />  <br />                                                                            istematic zones roots wild-type plants <br />                                                                            (Figures 6I 6J). To compare meristem growth between <br />                                                                            wild-type sbh1-1 sbh2-1 plants, number cortical cells <br />                                                                            initial cell rapidly elongating cell (indicated by <br />                                                                            stars Figures 6I 6J) counted using previously <br />                                                                            described methodology (Casamitjana-Martinez et al., 2003). <br />                                                                            According criterion, root meristems 5- 10-d-old <br />                                                                            sbh1-1 sbh2-1 seedlings contained ;20 16 cortical cells, <br />                                                                            respectively, wild-type 5- 10-d-old seedlings ;38 <br />                                                                            47 cortical cells, respectively (Figure 6H). These data sug- <br />                                                                            gest cell division reduced root meristem the <br />                                                                            sbh1-1 sbh2-1 mutant. Collectively, results indicate that <br />                                                                            defects cell elongation division contribute the <br />                                                                            reduced size sbh1-1 sbh2-1 mutant. <br />                                                                               The presence necrotic lesions cotyledons double <br />                                                                            mutant plants premature death plants are <br />                                                                            reminiscent phenotypes associated accelerated cell <br />                                                                            death (acd) Arabidopsis mutants, including sphingolipid- <br />                                                                            related acd11 acd5 mutants (Brodersen et al., 2002; Liang <br />                                                                            et al., 2003). To gain clues type cell death present <br />                                                                            sbh1 sbh2 double mutants, expression PCD marker <br /> Figure 4. Gene Structures Characterization Mutant Alleles for <br />                                                                            genes examined. For studies, gene expression was <br /> SBH1 SBH2. <br />                                                                            assessed RT-PCR using leaves 2-week-old plants <br /> (A) Gene structures SBH1 SBH2 T-DNA insertion sites          maintained Murashige Skoog (MS) plates. The marker <br /> mutants obtained SALK SAIL collections. Solid bars            genes chosen previously used examine cell <br /> represent exons, lines indicate introns. The open triangles indicate <br />                                                                            death acd11 mutants (Brodersen et al., 2002). Similar to <br /> the T-DNA insertion sites. <br />                                                                            previous observations acd11 plants (Brodersen et al., <br /> (B) RT-PCR analyses using gene-specific primers T-DNA <br /> lines used null mutants SBH1 SBH2. The UBC gene <br />                                                                            2002), expression PCD related hypersensitive response <br /> (At5g25760) used internal control. Data shown represen-      marker genes detected substantially higher levels in <br /> tative independent analyses.                                      hydroxylase double mutants relative wild-type plants <br />                                                                            (Figure 8). These included SENESCENCE-ASSOCIATED GENE13 <br />                                                                            (SAG13; At2g29350), FLAVIN-CONTAINING MONOXYGENASE <br />     In addition, RNAi suppression lines targeted SBH             (FMO; At1g19250), PEROXIDASE C (PRXc; At3g49120), <br /> genes generated examine effects range                  PATHOGENESIS-RELATED PROTEIN2 (PR-2; At2g14610), and <br /> trihydroxy LCB contents growth sphingolipid content. As             PR-3 (At3g57260). By contrast, expression senescence- <br /> shown Figure 7A, reduced growth observed plants             associated gene SAG12 (At5g45890) detected LCB C-4 <br /> a relative content trihydroxy LCBs # ;35% total LCBs          hydroxylase double mutant, observed acd11 <br /> leaves, growth progressively reduced plants                (Brodersen et al., 2002). These results suggest that, like acd11, a <br /> lower relative amounts trihydroxy LCBs. Plants containing            hypersensitive response&#226;&#8364;&#8220;type PCD activated sbh1 sbh2 dou- <br /> little 18% trihydroxy LCBs, dwarfed, retained              ble mutants. <br /> ability bolt set seed. <br />     Additional studies conducted determine             Effects Altered LCB C-4 Hydroxylation on <br /> reduced growth sbh1-1 sbh2-1 plants defects               Sphingolipid Content <br /> cell division cell elongation. When plants grown vertically <br /> on agar plates conditions etiolation, hypocotyl               The effects altered trihydroxy LCB composition sphin- <br /> <span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span> sbh1-1 sbh2-1 mutant ;25%                golipid content mutants RNAi suppression lines were <br /> wild-type plants (Figures 6E 6F). In addition, length           examined analysis LCBs following hydrolysis total <br /> epidermal cells middle hypocotyls sbh1-1         sphingolipid extract. Quantitative measurements sphingo- <br /> sbh2-1 mutant shorter cells corresponding          lipids leaves 2-week-old agar-grown seedlings revealed <br /> part wild-type plants (see Supplemental Figures 4A 4B               ;1.5-fold increase total LCB content sbh1-1 relative <br /> online). When grown vertically light, roots sbh1-1           wild-type control (Figure 5C). This increase reversed <br /> sbh2-1 single mutants approximately length                complementation sbh1-1 wild-type copy SBH1 <br /> those wild-type plants. However, roots 5-d-old sbh1-1                gene (Figure 5C). No change sphingolipid content was <br /> sbh2-1 double mutant seedlings ;66% length roots               detected sbh2-1 sbh2-2 seedlings, not <br /> of wild-type seedlings age, 10 d, roots          detectable alterations total trihydroxy LCB composition <br /> sbh1-1 sbh2-1 seedlings ;34% length wild-type                  relative wild-type seedlings (Figure 5C). Most strikingly, the <br /> seedling roots (Figure 6G). Microscopic analyses roots re-              sbh1-1 sbh2-1 double mutant, lacks detectable amounts <br /> vealed sbh1-1 sbh2-1 plants shorter thinner mer-              trihydroxy LCBs, displayed twofold threefold increases in <br />                                                                                                              Sphingolipid LCB Hydroxylation          1867 <br />  <br />  <br />  <br />  <br /> Figure 5. Sphingolipid LCB Content Composition sbh1 sbh2 Knockout Mutants. <br />  <br /> (A) The relative content trihydroxy LCBs total sphingolipid extracts different organs wild-type plants sbh1-1 sbh2-1 mutants. <br /> Values means 6 SD (n = 5). <br /> (B) Profiles LCBs total sphingolipid extracts wild-type plants sbh1-1 sbh2-1 double mutant plants obtained C18 reverse-phase HPLC <br /> analysis fluorescent derivatives LCBs following barium hydroxide/dioxane hydrolysis sphingolipids. Trihydroxy LCBs labeled asterisks. <br /> The identities labeled peaks follows: 1, t18:1(8Z)-Glc (psychosine form); 2, t18:1(8E)-Glc; 3, t18:1(8Z); 4, t18:1(8E); 5, d16:1 (internal <br /> standard); 6, t18:0; 7, d18:1(8Z)-Glc; 8, d18:1(8E)-Glc; 9, d18:1(8Z); 10, d18:1(8E); 11, 1,4-anhydro-t18:1(8Z); 12, 1,4-anhydro-t18:1(8E); 13, d18:0; 14, <br /> 1,4-anhydro-t18:0. d, dihydroxy; t, trihydroxy; x:y, x, number carbon atoms, y, number double bonds. <br /> (C) Total content LCBs wild-type (Col-0) plants, single double mutants sbh1-1 sbh2-1, sbh1-1 mutant complemented a <br /> wild-type copy SBH1 [sbh1-1(com)]. Sphingolipids extracted rosettes 14-d-old plants grown MS agar plates. Data shown are <br /> averages independent extractions analyses 6 SD. A Student&#226;&#8364;&#8482;s t test indicated total LCB content sbh1-1 sbh1-1 sbh2-1 <br /> was significantly increased relative wild-type control (** P &lt; 0.01). <br />  <br />  <br />  <br /> the total LCB content leaves relative wild-type              HPLC coupled electrospray ionization-tandem mass spec- <br /> control (Figure 5C). In addition, soil-grown RNAi suppression                  trometry (ESI-MS/MS) methods allow detection and <br /> lines ;20% trihydroxy LCBs total sphingolipid extract                  quantification molecular species (or exact pairings fatty <br /> contained nearly twofold higher levels sphingolipids wild-             acids LCBs) different sphingolipid classes (Markham and <br /> type plants grown similar conditions (Figure 7B). These                  Jaworski, 2007). The goals studies determine <br /> results indicate reductions LCB C-4 hydroxylation             effects partial complete loss LCB C-4 hydroxylation <br /> affect relative content trihydroxy LCBs lead                sphingolipid metabolism examine SBH1 and <br /> increases total sphingolipid content.                                       SBH2 contribute preferentially hydroxylation different <br />                                                                                sphingolipid classes. Four major classes sphingolipid (ceram- <br /> Sphingolipidomic Analyses Mutant RNAi                                   ides, hydroxyceramides, GlcCers, GIPCs) analyzed <br /> Suppression Lines                                                              free LCBs phosphorylated LCBs (LCBPs) in <br />                                                                                2-week-old T-DNA mutant seedlings maintained MS plates. <br /> Comprehensive analyses complete complement sphin-                       Free LCBs LCBPs, normally present low <br /> golipids (sphingolipidomic analyses) T-DNA mutants RNAi                 levels, twofold fourfold higher sbh1-1 sbh2-1 <br /> suppression lines conducted using recently developed                  mutants &gt;60-fold higher sbh1-1 sbh2-1 double mutant <br />  1868      The Plant Cell <br />  <br />  <br />  <br />  <br /> Figure 6. Growth Phenotypes sbh1 sbh2 Double Mutants. <br /> (A) Ten-day-old seedlings soil. Arrows indicate sbh1-1 sbh2-1 mutant segregated sbh1-1+/ sbh2-1 / parent. Necrotic lesions are <br /> indicated arrowhead. <br /> (B) One week-old seedling grown 13 MS plates. The arrow indicates sbh1-1 sbh2-1 double mutant. <br /> (C) (D) Three-week-old seedlings MS plates: wild type (C) sbh1-1 sbh2-1 double mutant (D). Bars = 0.5 cm (A) (D). <br /> (E) (F) Reduced hypocotyl growth 7-d-old etiolated sbh1-1 sbh2-1 plants relative wild-type plants. Plants shorter hypocotyls were <br /> confirmed sbh1-1 sbh2-1 mutants (arrowheads) use PCR genotyping. The data shown experiments. <br /> (G) Root length wild type sbh1-1 sbh2-1 double mutant 5 10 d germination. <br /> (H) Number cells root meristem zone wild-type sbh1-1 sbh2-1 plants. Values means 6 SD (n = 20) (F) (H). <br /> (I) (J) Meristem size wild-type plants (I) sbh1-1 sbh2-1 double mutants (J). Stars indicate upper lower ends meristem zone (see <br /> text <span id='am-2' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span> root meristem zone). Bars = 100 mm. <br />  <br />  <br /> than wild-type plants (Figure 9A; Supplemental Figure 5A                   Increases content substrate pools LCBs for <br /> online). The increase free LCBs LCBPs double                     ceramide synthases lead higher levels ceramides <br /> mutant largely accumulation free d18:0 (60 6                 ultimately increased amounts complex sphingolipids. <br /> 16 nmol/g dry weight). This substantial increase free d18:0              Consistent this, ceramide content sbh1-1 sbh2-1 mu- <br /> the double mutant explained simply reduced C-4                       tants (150 6 12 nmol/g dry weight) nearly twofold higher <br /> hydroxylation, wild-type plants contained 1.6 6 0.6 nmol/              wild-type plants (86 6 4 nmol/g dry weight) (see <br /> g dry weight free t18:0. Instead, accumulation d18:0                Supplemental Figure 5A online). In addition, fatty acid com- <br /> likely bottleneck incorporation LCB               position ceramides double mutant distinctly <br /> ceramides. For example, enhanced production d18:0                        different wild-type plants. Most notably, palmitic acid <br /> sbh1-1 sbh2-1 plants exceed capacity ceramide                       (16:0) accounted ;85% fatty acids ceramides the <br /> synthase incorporate LCBs ceramides.                             double mutant 13% fatty acids ceramides of <br />                                                                                                          Sphingolipid LCB Hydroxylation          1869 <br />  <br />  <br />  <br />                                                                                 Examination GlcCers sbh1-1 sbh2-1 double <br />                                                                              mutant showed that, expected, contained exclusively <br />                                                                              dihydroxy-LCBs, 2OH-palmitic acid content their <br />                                                                              ceramide backbones increased 76 6 1% 45 6 <br />                                                                              1% GlcCers wild-type plants (Figures 9F 9G). <br />                                                                              Similarly, GIPCs sbh1-1 sbh2-1 double mutant con- <br />                                                                              tained dihydroxy-LCBs twice 2OH- <br />                                                                              palmitic acid GIPCs wild-type plants (Figures 9H and <br />                                                                              9I). Consistent enhanced pool sizes ceramides the <br />                                                                              double mutant, total content GlcCers increased by <br />                                                                              nearly threefold GIPCs increased nearly <br />                                                                              fivefold sbh1-1 sbh2-1 plants relative wild-type plants <br />                                                                              (see Supplemental Figure 5A online). Overall, increase total <br />                                                                              sphingolipid content detected sbh1-1 sbh2-1 double mutants <br />                                                                              similar magnitude (twofold threefold) detected <br />                                                                              HPLC analysis LCBs hydrolyzed total complement <br />                                                                              sphingolipids described (Figure 5C). <br />  <br />  <br />  <br />  <br /> Figure 7. Growth LCB Content SBH RNAi Lines. <br />  <br /> (A) Four-week-old wild-type seedlings T2 seedlings independent <br /> SBH RNAi lines varying levels trihydroxy LCB content. Bar = 1 cm. <br /> (B) An RNAi line (SBH RNAi) trihydroxy LCB content ;20% total <br /> sphingolipids grown soil, total LCBs measured <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> leaves 6-week-old plants HPLC analysis o-phthaldialdehyde <br /> derivatives LCBs following barium hydroxide/dioxane hydrolysis total <br /> sphingolipids. The RNAi-suppressed plants showed increased levels of <br /> total LCBs (4.02 6 0.04 nmol/mg dry weight [dw]) compared wild-type <br /> plants grown conditions (1.65 6 0.20 nmol/mg dry weight). <br /> Results averages independent extractions analyses 6 SD. <br />  <br />  <br /> wild-type plants (Figures 9B 9C). Accumulation ceramides <br /> in single mutants SBH1 SBH2 pronounced; <br /> however, palmitic acid content sbh1-1 ceramides was <br /> elevated relative wild-type ceramides (44 versus 13%) <br /> (see Supplemental Figures 6A 6B online). <br />    Ceramides containing 2- (or a-) hydroxy fatty acids (hydroxy- <br /> ceramides) similarly affected. The hydroxyceramide con- <br /> tent sbh1 sbh2 double mutant (150 6 12 nmol/g dry weight) <br /> was nearly fivefold higher wild-type plants (31 6 <br /> 1 nmol/g dry weight) (Figures 9D 9E). In addition, relative          Figure 8. RT-PCR Analysis Expression PCD&#226;&#8364;&#8220; Hypersen- <br /> content 2OH-palmitic acid increased 18%                       sitive Response (HR)&#226;&#8364;&#8220;Related Genes Wild-Type (Col-0) sbh1-1 <br /> hydroxyceramides wild-type plants 89% hydroxycer-                   sbh2-1 Double Mutant Plants. <br /> amides sbh1-1 sbh2-1 plants (Figures 9D 9E). Again, like <br />                                                                              The genes chosen FMO (At1g19250), ERD11 (At1g02930), PRXc <br /> ceramides single mutants, hydroxyceramides unaf- <br />                                                                              (At3g49120), SAG13 (At2g29350), SAG12 (At5g45890), PR2 <br /> fected sbh2-1 mutant. Increases total content               (At3g57260), PR3 (At3g12500), reported previously (Brodersen <br /> hydroxyceramides relative content 2OH-                  et al., 2002). UBC (At5g25760) used internal positive control. <br /> palmitic acid detectable sbh1-1 mutant             Shown representative results biological replicates. Oligo- <br /> severe double mutant (see Supplemental                     nucleotide sequences used analyses provided Supple- <br /> Figures 6D 6F online).                                                   mental Table 3 online. <br />  1870     The Plant Cell <br />  <br />  <br />  <br />  <br /> Figure 9. Measurements Free LCBs LCBPs [LCB(P)s]) Molecular Species Composition Sphingolipid Classes Extracts Leaves of <br /> Wild-Type (Col-0) sbh1-1 sbh2-1 Plants Determined HPLC-ESI-MS/MS. <br /> (B), (D), (F), (H) measurements obtained wild-type plants, (C), (E), (G), (I) measurements obtained sbh1-1 sbh2-1 <br /> double mutants. The measurements shown LCBPs (A), ceramide ([B] [C]), hydroxyceramide ([D] [E]), glucosylceramide ([F] [G]), <br /> and GIPC ([H] [I]). Results averages analyses independent samples 6 SD. <br />  <br />  <br />    GlcCer GIPC levels sbh1-1 single mutant                 total content lipids measured. The observation that <br /> increased contained higher levels 2OH-palmitic acid         SBH1 mutants small alterations composition all <br /> the wild-type controls, extent changes con-           sphingolipid classes suggests corresponding hydroxylase <br /> siderably detected sbh1-1 sbh2-1 mutants             contributes globally synthesis trihydroxy LCBs than <br /> (Figures 9F 9I; Supplemental Figures 6G, 6H, 6J, 6K             synthesis trihydroxy LCBs specific sphingolipid classes. <br /> online). Modest increases content 2OH-palmitic acid            Analyses soil-grown RNAi lines 20% total sphin- <br /> detected GlcCers GIPCs sbh2-1 mutant (Figures 9F          golipid LCBs trihydroxy form revealed similar phenotypes as <br /> 9I; Supplemental Figures 6H 6K online), little change       double mutant, including increased total content of <br />                                                                                                             Sphingolipid LCB Hydroxylation          1871 <br />  <br />  <br />  <br /> each sphingolipid class accumulation C16 fatty acid&#226;&#8364;&#8220;                Glycerolipid Analyses Double Mutant RNAi <br /> containing sphingolipids (Figure 10; Supplemental Figure 5B                Suppression Lines <br /> online).                                                                       The content molecular species composition glycerolipids <br />   Overall, results demonstrate profound redirection                 sbh1-1 sbh2-1 double mutant SBH RNAi line were <br /> sphingolipid metabolism accompanying reductions LCB C-4                     analyzed ESI-MS/MS. The purpose analyses to <br /> hydroxylation sphingolipids, leading increased sphingolipid              determine large increases total content sphingo- <br /> content enhanced C16 fatty acid content sphingolipids.                  lipids content C16 fatty acid&#226;&#8364;&#8220;containing species <br />  <br />  <br />  <br />  <br /> Figure 10. HPLC-ESI-MS/MS Analysis Sphingolipid Composition Leaves 6-Week-Old Soil-Grown Wild-Type SBH RNAi Suppression <br /> Plants. <br />  <br /> Approximately 20% total LCBs RNAi line used analyses trihydroxy form. Panels measurements wild-type <br /> plants ([B], [D], [F], [H]) SBH RNAi suppression line ([C], [E], [G], [I]). Each panel shows measurements individual molecular species <br /> for different sphingolipid classes, follows: LCBPs (A), ceramides ([B] [C]), hydroxyceramides ([D] [E]), glucosylceramides ([F] [G]), and <br /> GIPCs ([H] [I]). Values means 6 SD (n = 3). <br />  1872      The Plant Cell <br />  <br />  <br />  <br /> resulted changes glycerolipids. These studies per-                major extraplastidic glycerolipids, including phosphatidyl- <br /> formed using plants grown conditions                   ethanolamine phosphatidylinositol, wild-type plants, <br /> similar age used sphingolipidomic analyses                  C-4 hydroxylase double mutant, RNAi suppression <br /> described above. The analytical technique used anal-               line (Figures 11E 11F). In addition, molecular species <br /> yses allows measurement 144 molecular species                 compositions glycerolipids similar these <br /> has widely applied comprehensive measure-                   lines (see Supplemental Data Set 1). By contrast, significant <br /> ment glycerolipids plants (Devaiah et al., 2006; Fritz et al.,         differences detected content composition of <br /> 2007; Welti et al., 2007). Using methods, small                 chloroplast-type lipids. In particular, amounts major leaf <br /> reduction detected content phosphatidylcholine              lipid monogalactosyldiacylglycerol (MGDG) ;33% lower in <br /> the sbh1-1 sbh2-1 double mutant relative wild-type control            double mutant wild-type control 17% lower in <br /> (Figure 11D). Otherwise, little difference measured amounts           RNAi line wild-type plants (Figure 11B). In cases, <br />  <br />  <br />  <br />  <br /> Figure 11. Glycerolipid Content Composition Leaves Wild-Type Plants, sbh1-1 sbh2-1 Double Mutant, SBH RNAi Lines. <br />  <br /> (A) (F) Content major glycerolipids leaves 2-week-old seedlings wild type (Col-0), sbh1-1 sbh2-1 double mutant, SBH RNAi <br /> lines, determined ESI-MS/MS analysis. As indicated, double mutant corresponding wild-type control seedlings maintained MS <br /> plates, RNAi control plants grown soil. Values averages 6 SD (n = 5). <br /> (G) (I) Lipid species DGDG, MGDG, PG. Values averages 6 SD (n = 5). <br /> * Significant P &lt; 0.01 compared wild type based Student&#226;&#8364;&#8482;s t test. <br />                                                                                                    Sphingolipid LCB Hydroxylation       1873 <br />  <br />  <br />  <br /> most differences result reductions C18/     relative amounts C16 fatty acids corresponding de- <br /> C16-containing molecular species 34:4 34:6 (Figure 11H). In         creases relative amounts very-long-chain fatty acids (Zheng <br /> addition, ;25 9% digalactosyldiacylglycerol (DGDG)         et al., 2005). These mutants display reduced growth, although <br /> 43 30% phosphatidylglycerol (PG) detected          severe reductions growth LCB C-4 <br /> sbh1-1 sbh2-1 double mutant RNAi lines, respectively, com-          hydroxylase null mutants (Zheng et al., 2005). The growth phe- <br /> pared wild-type control plants (Figures 11A 11C). As           notype cer10 mutants attributed defects sphingo- <br /> observed MGDG, lower levels PG DGDG             lipid-mediated endocytic protein trafficking (Zheng et al., 2005). <br /> primarily reductions amounts C18/C16-containing            Indeed, sphingolipids major structural components of <br /> species 34:4 PG 34:3 34:6 DGDG (Figures 11G         plasma membrane tonoplast plant cells (Verhoek et al., <br /> 11I). Overall, results point possibility in-      1983; Yoshida Uemura, 1986), alterations sphingo- <br /> creased demand palmitic acid (16:0) support sphingolipid         lipid composition undoubtedly affect physical properties of <br /> biosynthesis LCB C-4 hydroxylase mutants met reducing          membranes organization membrane <br /> the synthesis chloroplast glycerolipids, particularly molecular      microdomains lipid rafts, linked cellular <br /> species containing C16 fatty acids.                                     functions cell wall formation cell&#226;&#8364;&#8220;cell recognition <br />                                                                         (Borner et al., 2005). In addition, sphingolipid metabolites are <br />                                                                         believed involved regulation certain physiological <br /> DISCUSSION                                                              processes plant cells. For example, LCB-1-phosphate <br />                                                                         derivative phytosphingosine, trihydroxy LCB, appears be <br /> Plant sphingolipids enriched trihydroxy LCBs, trihy-         important abscisic acid&#226;&#8364;&#8220;dependent signaling events lead- <br /> droxy LCBs compose nearly 90% sphingolipids Arabi-            ing guard cell closure (Coursol et al., 2005). As such, is <br /> dopsis leaves(Markham et al., 2006). In report,                possible loss phytosphingosine-1-phosphate the <br /> physiological metabolic significance sphingolipid trihy-         LCB C-4 hydroxylase mutants affects physiolog- <br /> droxy LCBs examined characterization LCB C-4                  ical cellular processes. Furthermore, ceramides known to <br /> hydroxylase mutants RNAi suppression lines              serve components glycosylphosphatidylinositol-type an- <br /> detectable trihydroxy LCBs intermediate levels LCB           chors cell surface proteins, including arabinogalactan <br /> C-4 hydroxylation. We complete loss trihydroxy             proteins (Svetek et al., 1999). Although quantitative signifi- <br /> LCBs results severe reductions size plants          cance proteins ceramide-containing anchors plant cells <br /> defects cell expansion division, trihydroxy LCB&#226;&#8364;&#8220;        known, functions proteins transport <br /> deficient plants unable transition vegetative            plasma membrane altered plants deficient in <br /> reproductive growth. In addition, examination RNAi suppres-          trihydroxy LCBs. <br /> sion lines revealed degree dwarfing correlated         An additional phenotype observed sbh1 sbh2 double mu- <br /> the loss trihydroxy LCBs, reduced growth observed            tants presence necrotic lesions cotyledons and <br /> in lines trihydroxy LCB content levels # ;35%               premature death (Figure 6A). Indeed, accumulation cer- <br /> total LCBs leaf extracts. By use recently developed ESI-      amides free LCBs, observed hydroxylase double <br /> MS/MS protocols sphingolipid molecular species analysis             mutant, shown induce PCD (Abbas et al., 1994; <br /> (Markham Jaworski, 2007), determined reduc-             Spassieva et al., 2002; Liang et al., 2003; Townley et al., 2005). In <br /> tions trihydroxy LCBs accompanied accumulation         addition, acd11 mutant, defective putative LCB <br /> sphingolipids C16 fatty acids typical         transport protein, shown constitutively upreg- <br /> very-long-chain fatty acids. In addition, total content sphin-   ulated marker genes hypersensitive response&#226;&#8364;&#8220;type PCD <br /> golipids classes increase relative content   (Brodersen et al., 2002). Although vivo role putative <br /> of trihydroxy LCBs decreased. In mutants completely lack           LCB transport protein sphingolipid metabolism be <br /> trihydroxy LCBs, total content sphingolipids leaves       demonstrated, possibility lesions ACD11 gene <br /> twofold threefold higher leaves wild-type plants.    result increased levels LCBs ceramides plant cells that <br /> Overall, results demonstrate C-4 hydroxylation LCBs       trigger PCD (Brodersen et al., 2002). Interestingly, observed <br /> is critical structural modification maintenance sphin-     number PCD-associated genes expressed <br /> golipid content fatty acid composition ultimately       high levels acd11 mutants upregulated sbh1-1 <br /> growth reproductive potential Arabidopsis.                       sbh2-1 double mutant (Figure 8). This finding suggests the <br />    Several formal explanations account growth               necrotic lesions premature death LCB C-4 hydroxy- <br /> phenotypes associated loss LCB C-4 hydroxylation            lase null mutant arise PCD, likely caused buildup of <br /> in Arabidopsis seen studies. Among possibilities     sphingolipids, including free LCBs ceramides. <br /> the altered growth LCB C-4 hydroxylase mutants               The observation LCB C-4 hydroxylase null mutants <br /> direct result shift LCB composition trihydroxy        RNAi suppression lines twofold threefold higher <br /> dihydroxy moieties. Alternatively, growth defects            amounts sphingolipids, primarily C16 fatty acid&#226;&#8364;&#8220;containing <br /> to secondary alterations sphingolipid profiles, including in-        species, raises question large alterations in <br /> creases total sphingolipid content accumulation            sphingolipid production affect synthesis lipids in <br /> sphingolipids containing fatty acids shorter chain lengths.          plant cells. Comprehensive analyses glycerolipids ESI-MS/ <br /> Notably, Arabidopsis cer10 mutants, reduced enoyl-           MS conducted address question. These analyses <br /> CoA reductase activity, contain sphingolipids increased            revealed amounts chloroplast lipids, particularly MGDG <br />  1874     The Plant Cell <br />  <br />  <br />  <br /> and PG, significantly lower sbh1-1 sbh2-1 double <br /> mutant RNAi suppression line relative wild-type <br /> control plants. The decreased amounts MGDG PG were <br /> due largely reduced content C16-containing molecular <br /> species. Conversely, small changes significant <br /> changes detected content composition ma- <br /> jor ER-type lipids (e.g., phosphatidylcholine phosphatidyl- <br /> ethanolamine). Although number explanations be <br /> evoked data, possibility plant cells reduce <br /> the synthesis chloroplast lipids meet increased require- <br /> ment 16:0 support enhanced sphingolipid production in <br /> plants defective LCB C-4 hydroxylation. This implies metabolic <br /> coordination ER chloroplasts regulation of <br /> the biosynthesis sphingolipids chloroplast-specific lipids. <br /> The existence metabolic network awaits inves- <br /> tigation. It noted ESI-MS/MS analyses sphin-        Figure 12. Model Showing Central Role Trihydroxy LCB Synthesis <br /> golipids glycerolipids conducted using independent           Growth Sphingolipid Metabolism. <br /> methodologies, limits direct comparisons absolute <br />                                                                       The accumulation dihydroxy LCBs (d18:0) resulting impaired C-4 <br /> amounts lipids data sets.                            hydroxylation causes sphingolipids (SL) accumulate enriched <br />    Our results indicate combined activities SBH1      C16 fatty acids (C16FA). This rationalized presence of <br /> SBH2 account detectable synthesis trihydroxy LCBs       classes acyl-CoA&#226;&#8364;&#8220;dependent ceramide synthases: CS I, which <br /> Arabidopsis leaves. However, ruled sphin-       preferentially uses dihydroxy LCBs CoA esters C16 fatty acids <br /> golipid D4 desaturases (Ternes et al., 2002), distantly     (FA) substrates, CS II, preferentially uses trihydroxy LCBs <br /> related LCB C-4 hydroxylases, introduce portion       (t18:0) CoA esters very-long-chain ($C20) fatty acids (VLCFA) as <br /> the C-4 hydroxy groups trihydroxy LCBs, particularly      substrates. As revealed phenotypes sbh1-1 sbh2-1 mutants <br /> plants tomato (Solanum lycopersicum) maize (Zea           RNAi suppression lines, LCB C-4 hydroxylation (as indicated by <br />                                                                       SBH) critical growth (as indicated shaded oval), the <br /> mays), contain high levels D4 unsaturated LCBs (Dunn <br />                                                                       synthesis trihydroxy LCB&#226;&#8364;&#8220;containing sphingolipids likely regulated, <br /> et al., 2004; Markham et al., 2006). It notable mammals <br />                                                                       Ser palmitoyltransferase (SPT), meet demands of <br /> and certain fungi, D4 desaturase homologs DES2            growth. Loss LCB C-4 hydroxylation appears uncouple regu- <br /> display level LCB C-4 hydroxylase activity (Ternes et al.,    lation, resulting accumulation sphingolipids dihydroxy LCB/ <br /> 2002; Omae et al., 2004). The contributions sphingolipid D4    C16 fatty acid&#226;&#8364;&#8220;containing ceramide backbones. 3KR, 3-ketosphinganine <br /> desaturase LCB hydroxylation difficult assess Arabi-      reductase. <br /> dopsis. The Arabidopsis sphingolipid D4 desaturase gene is <br /> expressed flowers, indicated publicly available <br /> microarray data (Zimmermann et al., 2004), LCB products        sphingolipid molecular species enriched C16 fatty acids <br /> D4 desaturation detectable nonfloral tissues            likely reflection substrate specificity acyl-CoA&#226;&#8364;&#8220; <br /> Arabidopsis (Columbia; Sperling et al., 2005; Markham et al.,         dependent ceramide synthases (or sphinganine N-acyltransfer- <br /> 2006). It reported previously that LCB C-4 hydrox-      ases), catalyze condensation LCBs fatty <br /> ylation crude microsomes maize leaves use free            acyl-CoAs. The data presented consistent with <br /> LCBs LCB components ceramides substrates (Wright             occurrence classes acyl-CoA&#226;&#8364;&#8220;dependent ceramide <br /> et al., 2003). It suggested discrete hydroxylases cata-      synthases (designated CS I CS II Figure 12) Arabidopsis. <br /> lyze reactions, free LCBs ceramides          The putative CS I class preferentially links dihydroxy LCBs with <br /> act competitors assays (Wright et al., 2003). It       C16 fatty acyl-CoAs, CS II class preferentially links <br /> possible different activities arise LCB C-4       trihydroxy LCBs very-long-chain fatty acyl-CoAs. Indeed, in <br /> hydroxylase sphingolipid D4 desaturase. In regard,       wild-type Arabidopsis, dihydroxy LCBs frequently <br /> D4 unsaturated LCBs entirely GlcCers       bound C16 fatty acids (Markham Jaworski, 2007), which <br /> than GIPCs (Markham et al., 2006; Markham Jaworski,            accentuated LCB C-4 hydroxylase mutants. In addition, <br /> 2007), suggesting D4 desaturases function directly            acyl-CoA&#226;&#8364;&#8220;dependent ceramide synthases distinct specific- <br /> ceramides GlcCers separate pool ceramides used          ities acyl-CoAs different chain lengths described <br /> in synthesis GlcCers. Conversely, consistent          mouse human cells (Guillas et al., 2003; Mizutani et al., <br /> activity free LCBs, C-4 hydroxylase&#226;&#8364;&#8220;derived trihydroxy LCBs        2005; Pewzner-Jung et al., 2006). For example, mouse <br /> are sphingolipid classes, including pools free LCBs   LASS6 ceramide synthase active, acyl-CoAs con- <br /> and LCB-1 phosphates (Markham Jaworski, 2007).                    taining fatty acid chains 12 18 carbon atoms, it <br />    The results presented indicate trihydroxy LCBs           displays little activity C20 C26 acyl-CoAs (Mizutani et al., <br /> resulting LCB C-4 hydroxylation play central role       2005). Conversely, mouse LASS2 ceramide synthase most <br /> maintenance growth, fatty acid composition, total      active, acyl-CoAs containing fatty acids 22, 24, and <br /> content sphingolipids. A model rationalizes obser-      26 carbon atoms, low activity shorter chain <br /> vations proposed Figure 12. As indicated, accumulation      acyl-CoAs (Mizutani et al., 2005). The occurrence plants of <br />                                                                                                        Sphingolipid LCB Hydroxylation            1875 <br />  <br />  <br />  <br /> ceramide synthases differing substrate specificities        hypocotyls response etiolation, seeds sown MS agar <br /> been suggested research conducted Alternaria stem             plates exposed light 6 8 h stratification promote <br /> canker-1 (asc-1) mutant tomato (Spassieva et al., 2002). In            germination. Plates wrapped aluminum foil maintained <br />                                                                           7 d 228C vertical orientation. The ability exogenous LCBs to <br /> these studies, expression Asc-1&#226;&#8364;&#8220;encoded ceramide syn- <br />                                                                           complement C-4 hydroxylase sbh1-1 sbh2-1 double mutants ex- <br /> thase yeast lacking endogenous genes ceramide <br />                                                                           amined addition phytosphingosine (t18:0; Matreya) methanol <br /> synthase created new sphingolipid species, suggesting differ- <br />                                                                           varying concentrations 200 mM MS medium supplemented <br /> ent substrate specificity tomato ceramide synthase                0.2% (w/v) Tergitol Nonidet P-40 (Sigma-Aldrich) (Abbas et al., 1994). <br /> (Spassieva et al., 2002). <br />    As indicated model Figure 12 supported the <br />                                                                           RNA Isolation RNA Gel Blotting <br /> phenotypes C-4 hydroxylase mutants RNAi suppression <br /> lines, trihydroxy LCB production required growth. The              Total RNA isolated 10-d-old soil-grown seedlings. For analyses <br /> model does distinguish contributions trihy-            organ-specific expression SBH1 SBH2, 6- 8-week-old wild- <br /> droxy LCBs very-long-chain fatty acids growth. Regard-             type plants used sources plant material. RNA extraction was <br />                                                                           performed using RNeasy Plant Kit (Qiagen) according manu- <br /> less, LCB C-4 hydroxylation important mediating the <br />                                                                           facturer&#226;&#8364;&#8482;s protocol. Total RNA (15 mg) treated glyoxal (Ambion) <br /> incorporation very-long-chain fatty acids ceramides <br />                                                                           electrophoresed 1% (w/v) agarose gel transferred <br /> of sphingolipids. In absence LCB C-4 hydroxylation, a <br />                                                                           Hybond nylon membranes (GE Bioscience) using reagents and <br /> preponderance C16 fatty acid&#226;&#8364;&#8220;containing sphingolipids ac-              protocol supplied NorthernMax kit (Ambion). Hybridization and <br /> cumulates. It likely regulation sphingolipid synthesis     detection conducted according protocol DIG High <br /> and growth tightly intertwined. Consistent this,              Primer DNA Labeling Detection Starter Kit II (Roche). The DNA <br /> recently showed plants adjust growth compensate               probe labeled PCR DIG Probe Synthesis Kit (Roche). Gene- <br /> reduced sphingolipid synthesis Arabidopsis RNAi suppression            specific probes obtained PCR products corresponding the <br /> lines LCB1 subunit Ser palmitoyltransferase (Chen              39-untranslated region SBH1 SBH2. Oligonucleotides P9 P10 <br /> et al., 2006). We propose reduction elimination LCB C-4        used generation SBH1-specific probe, P11 P12 <br />                                                                           used generate SBH2-specific probe (see Supplemental Table <br /> hydroxylation uncouples regulation sphingolipid synthesis <br />                                                                           2 online sequences oligonucleotides). <br /> from demands growth, resulting enhanced flux through <br /> the sphingolipid biosynthetic pathway increased accu- <br /> mulation sphingolipids, observed mutants RNAi                RT-PCR Verification T-DNA Mutants <br /> suppression lines. Regulation sphingolipid synthesis yeast          Total RNA isolated 10-d-old wild-type mutant seedlings <br /> and animals generally believed mediated Ser              grown 13 MS plates. Total RNA (1 mg) treated DNase <br /> palmitoyltransferase (designated SPT Figure 12),          (Roche), first-strand cDNA subsequently synthesized using <br /> regulation sphingolipid synthesis systems        SuperScript III reverse transcriptase (Invitrogen) oligo(dT) primer, <br /> well defined (Hanada, 2003). The regulatory interactions be-              according manufacturer&#226;&#8364;&#8482;s instructions. A 2-mL aliquot first-strand <br /> tween formation trihydroxy LCBs SPT likely          cDNA used template amplify SBH1, SBH2, gene the <br />                                                                           ubiquitin-conjugating enzyme (UBC; At5g25760) 20-mL reaction with <br /> fruitful topic research aimed understanding sphingo- <br />                                                                           30 cycles amplification 558C annealing temperature. PCR ampli- <br /> lipid synthesis controlled plants. <br />                                                                           fication conducted Taq DNA polymerase (New England Biolabs). <br />    Overall, characterization T-DNA mutants RNAi <br />                                                                           Primers used amplification SBH1-derived cDNA P9 P10, <br /> suppression lines C-4 hydroxylase genes studies              used SBH2 P11 P12 (see Supplemental Table 2 <br /> has provided unexpected insights significance sphin-          online). Amplification products analyzed electrophoresis 1% <br /> golipid structure plant growth development. These re-             agarose (w/v) detected ethidium bromide staining. The gene the <br /> sults highlight structural modifications, C-4            UBC (At5g25760) used internal control (Czechowski et al., <br /> hydroxylation, LCBs dictate metabolic flow sphin-           2005). Primers P13 P14 (see Supplemental Table 2 online) used <br /> golipid biosynthetic pathway Arabidopsis. It expected          amplification UBC-derived cDNA. <br /> the results presented provide direction future studies <br /> of sphingolipid function metabolism plants.                        Expression Analysis PCD Marker Genes <br />  <br />                                                                           Total RNA extracted 2-week-old control sbh1-1 sbh2-1 <br /> METHODS                                                                   double mutant plants grown MS plates, first-strand cDNA was <br />                                                                           prepared described above. RT-PCR analysis conducted with <br /> Plant Material Growth Conditions                                      equal amounts first-strand cDNA template. Oligonucleotides and <br />                                                                           numbers PCR cycles used target gene provided in <br /> Wild-type mutant Arabidopsis thaliana lines used studies    Supplemental Table 3 online. UBC (At5g25760) amplified used <br /> were Columbia (Col-0) ecotype. Multiple T-DNA insertion mutants    internal positive control. <br /> for SBH1 SBH2 (Figure 4A) obtained ABRC (Alonso <br /> et al., 2003). Primers used PCR genotyping mutants are <br />                                                                           Promoter-GUS Assay SBH1 SBH2 <br /> provided Supplemental Table 1 online. <br />    Plants grown soil MS agar described previously (Chen     A SBH1 promoter&#226;&#8364;&#8220;GUS reporter construct generated amplifica- <br /> et al., 2006). For growth long-day conditions, plants main-    tion ;2.0-kb sequence upstream start codon SBH1 <br /> tained 228C 50% humidity 16-h-light (100 mmol&#194;&#183;m22&#194;&#183;s21)/     (At1g69640) using sense antisense oligonucleotides P19 P20 <br /> 8-h-dark cycle. Root elongation measurements conducted            (see Supplemental Table 2 online). The product digested HindIII <br /> plants grown plates vertical orientation. To measure elongation   XbaI cloned corresponding sites binary vector pBI121 <br />  1876      The Plant Cell <br />  <br />  <br />  <br /> (Clontech) generate transcriptional fusion GUS coding          quences, Supplemental Table 2 online). The amplified product was <br /> region. The resulting plasmid designated ProSBH1:GUS. ProSBH2:           digested AscI PacI cloned corresponding <br /> GUS generated similar way using primer pair P21 P22 (see        restriction sites binary vector pMDC123 (Curtis Grossniklaus, <br /> Supplemental Table 2 online). Homozygous lines containing               2003) place cloning cassette produce pMDC123_SBH1g. <br /> ProSBH1:GUS ProSBH2:GUS constructs isolated T3             Transformation sbh1-1 sbh1-12/+ sbh2-12/2 pMDC123_SBH1g <br /> generation. Histochemical assay GUS activity performed            performed floral dip method (Clough Bent, 1998), and <br /> described previously (Gallagher, 1992).                                      transformants selected resistance glufosinate (10 mg/L; <br />                                                                              Sigma-Aldrich). <br /> Subcellular Localization SBH1 SBH2 <br />                                                                              Generation SBH RNAi Suppression Plants <br /> CFP fusion proteins SBH1 SBH2 prepared amplification <br /> of SBH1 SBH2 open reading frames oligonucleotides P3/P4         A SBH1 RNAi suppression construct generated using Kannibal <br /> and P6/P7 (see Supplemental Table 2 online), respectively. PCR products      RNAi vector (Helliwell Waterhouse, 2003). A 356-bp fragment <br /> were cloned pENTR/D-TOPO vector (Invitrogen). The             containing base pairs 25 380 SBH1 open reading frame was <br /> resulting plasmids combined destination vector pEarley-        amplified using pairs oligonucleotides, P15/P16 P17/P18 (see <br /> gate 102 (Earley et al., 2006) attL 3 attR recombination reaction   Supplemental Table 2 online), products reactions were <br /> generate CFP C-terminal fusion constructs pSBH1-CFP pSBH2-           cloned sequentially pKannibal vector. The amplified portion of <br /> CFP. Similarly, YFP-SBH1 YFP-SBH2 fusion proteins YFP            SBH1 contains regions absolute homology open reading frame <br /> their N termini generated using primer pair P3/P5 P6/P8 (see    SBH2. The resulting hairpin construct cauliflower <br /> Supplemental Table 2 online), respectively, destination vector       mosaic virus 35S promoter ocs terminator released and <br /> pEarleygate 104.                                                             inserted NotI site binary vector pART27 generate pKanSB- <br />    Agrobacterium tumefaciens&#226;&#8364;&#8220;mediated infiltration tobacco (Nicotiana     H1i. Binary vectors introduced Agrobacterium GV3101 by <br /> tabacum) leaves pSBH1-CFP pSBH2-CFP confocal laser              electroporation, transgenic plants generated floral dip of <br /> scanning microscopy conducted using Carl Zeiss LSM 510 laser          Arabidopsis (Col-0) (Clough Bent, 1998) screened MS plates <br /> scanning microscope described previously (Chen et al., 2006). The ER-     containing 50 mg/L kanamycin monosulfate. Four- 6-week-old T2 <br /> specific marker used studies YFP fused ER signal        plants grown soil used analyze total LCBs sphingolipid <br /> peptide basic chitinase N terminus HDEL ER retention    molecular species. <br /> sequence C terminus (Chen et al., 2006). <br />    The localizations SBH1&#226;&#8364;&#8220; SBH2&#226;&#8364;&#8220;fluorescent protein fusions      Microscopy <br /> also examined relative plasma membrane marker dye. For these <br /> studies, leaves tobacco transiently expressing SBH1-CFP SBH2-          Imaging wild-type mutant plants performed using Nikon <br /> CFP immersed 20 mM FM4-64 (Invitrogen) 10 min stain       SMZ1500 dissection microscope attached digital camera (Retiga <br /> plasma membrane cells. CFP excited 458-nm argon laser          1300; Qimaging), images processed IPLab software. <br /> line 480- 520-nm band-pass emission filter, FM4-64        Hypocotyls cleared Hoyer&#226;&#8364;&#8482;s solution observed Nikon <br /> excited 543-nm argon laser line 565- 615-nm band-pass        Eclipse E800 microscope equipped differential interference con- <br /> emission filter.                                                             trast apparatus (Nomarski optics) described previously (Chen et al., <br />                                                                              2006). Observations root meristem zones conducted whole <br />                                                                              seedlings stained propidium iodide (10 mg/mL [w/v]; <br /> Complementation Yeast sur2D Mutant SBH&#226;&#8364;&#8220;Fluorescent <br />                                                                              Sigma-Aldrich) 2 5 min rinsed water. Stained roots <br /> Protein Fusions <br />                                                                              analyzed Carl Zeiss LSM 510 laser scanning microscope with <br /> An LCB C-4 hydroxylase mutant (sur2D) Saccharomyces cerevisiae            excitation 543 nm emission 590 nm. <br /> strain BY4741 prepared disruption SUR2 gene LEU2 <br /> marker using methods described previously (Haak et al., 1997). The           Analysis Sphingolipid LCBs <br /> coding sequences N-terminal fusions SBH1 SBH2 YFP and <br /> C-terminal fusions proteins CFP amplified PCR          The total content composition sphingolipid LCBs determined <br /> from plant expression vectors used transient localization studies. The    following strong alkaline hydrolysis extracted complex sphingolipids, <br /> oligonucleotides used amplification SBH1- SBH2-CFP nucleotide      fluorescent derivatization released LCBs, subsequent reverse- <br /> sequences P23/P24 P24/P25, respectively, used         phase HPLC analysis described previously (Morrison Hay, 1970; <br /> amplification YFP-SBH1 YFP-SBH2 sequences P26/P27             Chen et al., 2006). For analysis total content sphingolipid, plants <br /> P26/P28, respectively (see Supplemental Table 2 online). PCR products        grown 14 d MS plates supplied 1% (w/v) sucrose. For <br /> were digested HindIII XbaI cloned yeast expression     analysis sphingolipid LCB composition different organs, plants <br /> vector pYES2 (Invitrogen). The resulting plasmids transformed      grown long-day conditions (see above) harvested 8 <br /> the sur2D yeast mutant, fusion proteins expressed          weeks age. <br /> galactose induction 3-mL cultures using previously described methods <br /> (Cahoon Kinney, 2004). The total LCB composition yeast        Sphingolipidomic Analyses <br /> determined heating cell pellets barium hydroxide/dioxane <br /> followed HPLC analysis fluorescent derivatives released LCBs,       Wild-type mutant plants grown 13 MS agar plates supplied <br /> as described previously (Morrison Hay, 1970; Chen et al., 2006).         1% sucrose 14 d, aerial portions seedlings were <br />                                                                              collected lyophilized. For RNAi lines wild-type controls, plants <br />                                                                              maintained soil using growth conditions described above, <br /> Genetic Complementation sbh1-1 <br />                                                                              leaves collected 6-week-old plants lyophilized. Global <br /> For genomic complementation sbh1-1 sbh1-1 sbh2-1 plants,            analyses molecular species different sphingolipid classes were <br /> ;2.7-kb fragment SBH1 amplified Arabidopsis (Col-0)              conducted reverse-phase HPLC coupled ESI-MS/MS de- <br /> genomic DNA using oligonucleotides P1 P2 (for primer se-             scribed Markham Jaworski (2007). <br />                                                                                                            Sphingolipid LCB Hydroxylation         1877 <br />  <br />  <br />  <br /> Glycerolipid Analyses                                                          Received December 28, 2007; revised June 3, 2008; accepted June 24, <br />                                                                                2008; published July 8, 2008. <br /> Comprehensive glycerolipid analyses conducted sbh1-1 sbh2-1 <br /> double mutant RNAi suppression lines grown harvested as <br /> described sphingolipidomic analyses. Lipid extractions, ESI- <br /> MS/MS analyses, lipid quantification conducted described           REFERENCES <br /> (Devaiah et al., 2006) minor modifications. For analysis DGDG, <br /> 341.11 [M + NH4]+ collision energy 24 V used. For analysis      Abbas, H.K., Tanaka, T., Duke, S.O., Porter, J.K., Wray, E.M., <br /> MGDG, 179.08 [M + NH4]+ collision energy 21 V used. The              Hodges, L., Sessions, A.E., Wang, E., Merrill, A.H., Jr., Riley, <br /> ESI-MS/MS analyses performed Kansas Lipidomics Re-                   R.T. (1994). Fumonisin- AAL-toxin-induced disruption sphin- <br /> search Center.                                                                   golipid metabolism accumulation free sphingoid bases. Plant <br />                                                                                  Physiol. 106: 1085&#226;&#8364;&#8220;1093. <br />                                                                                Alonso, J.M., et al. (2003). Genome-wide insertional mutagenesis of <br /> Accession Numbers                                                                Arabidopsis thaliana. Science 301: 653&#226;&#8364;&#8220;657. <br /> Sequence data article Arabidopsis Genome         Borner, G.H., Sherrier, D.J., Weimar, T., Michaelson, L.V., Hawkins, <br /> Initiative GenBank/EMBL databases following accession               N.D., Macaskill, A., Napier, J.A., Beale, M.H., Lilley, K.S., and <br /> numbers: SBH1, NM_105632; SBH2, NM_101295.                                       Dupree, P. (2005). Analysis detergent-resistant membranes in <br />                                                                                  Arabidopsis. Evidence plasma membrane lipid rafts. Plant Physiol. <br />                                                                                  137: 104&#226;&#8364;&#8220;116. <br /> Supplemental Data                                                              Brodersen, P., Petersen, M., Pike, H.M., Olszak, B., Skov, S., Odum, <br /> The following materials available online version article.     N., Jorgensen, L.B., Brown, R.E., Mundy, J. (2002). Knockout <br />                                                                                  Arabidopsis ACCELERATED-CELL-DEATH11 encoding sphingo- <br />   Supplemental Figure 1. Comparison ER Localization SBH1- <br />                                                                                  sine transfer protein causes activation programmed cell death and <br />   CFP SBH2-CFP Versus Plasma Membrane Marker Dye. <br />                                                                                  defense. Genes Dev. 16: 490&#226;&#8364;&#8220;502. <br />   Supplemental Figure 2. LCB C-4 Hydroxylase Activity Fluorescent           Cahoon, E.B., Kinney, A.J. (2004). Dimorphecolic acid synthe- <br />   Protein Fusions SBH1 SBH2.                                              sized coordinate activities divergent D12-oleic acid <br />   Supplemental Figure 3. Expression Analysis SBH1 SBH2                 desaturases. J. Biol. Chem. 279: 12495&#226;&#8364;&#8220;12502. <br />   sbh1 sbh2 Double Mutants.                                                    Casamitjana-Martinez, E., Hofhuis, H.F., Xu, J., Liu, C.M., Heidstra, <br />                                                                                  R., Scheres, B. (2003). Root-specific CLE19 overexpression and <br />   Supplemental Figure 4. Epidermal Cell Length Arabidopsis Col-0              sol1/2 suppressors implicate CLV-like pathway control of <br />   Hypocotyls Versus That sbh1-1 sbh2-1 Double Mutant.                     Arabidopsis root meristem maintenance. Curr. Biol. 13: 1435&#226;&#8364;&#8220;1441. <br />   Supplemental Figure 5. Content Sphingolipid Classes sbh1-1             Chen, M., Han, G., Dietrich, C.R., Dunn, T.M., Cahoon, E.B. <br />   sbh2-1 Double Mutant RNAi Suppression Lines.                               (2006). The essential nature sphingolipids plants revealed by <br />                                                                                  functional identification characterization Arabidopsis <br />   Supplemental Figure 6. Sphingolipid Composition Leaves from <br />                                                                                  LCB1 subunit serine palmitoyltransferase. Plant Cell 12: 3576&#226;&#8364;&#8220;3593. <br />   sbh1-1, sbh2-1, SBH1-Complemented sbh1-1 sbh2-1 Double <br />                                                                                Cliften, P., Wang, Y., Mochizuki, D., Miyakawa, T., Wangspa, R., <br />   Mutant. <br />                                                                                  Hughes, J., Takemoto, J.Y. (1996). SYR2, gene necessary for <br />   Supplemental Table 1. Oligonucleotide Primer Sequences Used                syringomycin growth inhibition Saccharomyces cerevisiae. Micro- <br />   Confirmation T-DNA Insertion Sites sbh1 sbh2 Mutants.                biology 142: 477&#226;&#8364;&#8220;484. <br />   Supplemental Table 2. Oligonucleotide Primer Sequences Used              Clough, S.J., Bent, A.F. (1998). Floral dip: A simplified method for <br />   RT-PCR, Complementation, Subcellular Localizations, Promoter-GUS               Agrobacterium-mediated transformation Arabidopsis thaliana. Plant <br />   Assays, RNA Gel Blot Analyses.                                             J. 16: 735&#226;&#8364;&#8220;743. <br />                                                                                Coursol, S., Fan, L.M., Le Stunff, H., Spiegel, S., Gilroy, S., and <br />   Supplemental Table 3. Oligonucleotides Numbers PCR <br />                                                                                  Assmann, S.M. (2003). Sphingolipid signalling Arabidopsis guard <br />   Cycles Used Expression Analysis Cell Death Hypersensitive <br />                                                                                  cells involves heterotrimeric G proteins. Nature 423: 651&#226;&#8364;&#8220;654. <br />   Response&#226;&#8364;&#8220;Related Genes. <br />                                                                                Coursol, S., Le Stunff, H., Lynch, D.V., Gilroy, S., Assmann, S.M., and <br />   Supplemental Data Set 1. ESI-MS/MS Analysis Glycerolipid                    Spiegel, S. (2005). Arabidopsis sphingosine kinase effects of <br />   Molecular Species Mutant RNAi Lines.                                    phytosphingosine-1-phosphate stomatal aperture. Plant Physiol. <br />                                                                                  137: 724&#226;&#8364;&#8220;737. <br />                                                                                Curtis, M.D., Grossniklaus, U. (2003). A Gateway cloning vector <br /> ACKNOWLEDGMENTS                                                                  set high-throughput functional analysis genes planta. Plant <br />                                                                                  Physiol. 133: 462&#226;&#8364;&#8220;469. <br /> We thank colleagues Daniel Lynch Teresa Dunn helpful               Czechowski, T., Stitt, M., Altmann, T., Udvardi, M.K., Scheible, <br /> comments, Jia Li technical assistance, R. Howard Berg assistance         W.R. (2005). Genome-wide identification testing superior <br /> with confocal laser microscopy, ABRC providing seeds             reference genes transcript normalization Arabidopsis. Plant <br /> SALK SAIL mutants. We indebted Mary Roth Ruth                Physiol. 139: 5&#226;&#8364;&#8220;17. <br /> Welti Kansas Lipidomics Research Center expert profiling      Devaiah, S.P., Roth, M.R., Baughman, E., Li, M., Tamura, P., <br /> of glycerolipids. The Kansas Lipidomics Research Center supported          Jeannotte, R., Welti, R., Wang, X. (2006). Quantitative profiling <br /> the National Science Foundation (Grants EPS 0236913, MCB 0455318,                polar glycerolipid species organs wild-type Arabidopsis and <br /> and DBI 0521587), Kansas Technology Enterprise Corporation,              PHOSPHOLIPASE Da1 knockout mutant. Phytochemistry 67: 1907&#226;&#8364;&#8220; <br /> K-IDeA Networks Biomedical Research Excellence National                1924. <br /> Institutes Health (Grant P20 RR-16475), Kansas State University.        Dunn, T.M., Lynch, D.V., Michaelson, L.V., Napier, J.A. (2004). A <br /> This work supported National Science Foundation Arabidopsis             post-genomic approach understanding sphingolipid metabolism in <br /> 2010 Grant J.G.J. E.B.C. (Grant MCB 0312559).                             Arabidopsis thaliana. Ann. Bot. (Lond.) 93: 483&#226;&#8364;&#8220;497. <br />  1878      The Plant Cell <br />  <br />  <br />  <br /> Earley, K.W., Haag, J.R., Pontes, O., Opper, K., Juehne, T., Song, K.,         Morrison, W.R., Hay, J.D. (1970). Polar lipids bovine milk. II. <br />   Pikaard, C.S. (2006). Gateway-compatible vectors plant                 Long-chain bases, normal 2-hydroxy fatty acids, isomeric cis <br />   functional genomics proteomics. Plant J. 45: 616&#226;&#8364;&#8220;629.                      trans monoenoic fatty acids sphingolipids. Biochim. <br /> Fritz, M., Lokstein, H., Hackenberg, D., Welti, R., Roth, M., Zahringer,         Biophys. Acta 202: 460&#226;&#8364;&#8220;467. <br />   U., Fulda, M., Hellmeyer, W., Ott, C., Wolter, F.P., Heinz, E.           Ng, C.K., Carr, K., McAinsh, M.R., Powell, B., Hetherington, A. <br />   (2007). Channeling eukaryotic diacylglycerol biosynthesis       M. (2001). Drought-induced guard cell signal transduction involves <br />   plastidial phosphatidylglycerol. J. Biol. Chem. 282: 4613&#226;&#8364;&#8220;4625.                sphingosine-1-phosphate. Nature 410: 596&#226;&#8364;&#8220;599. <br /> Gallagher, S.R., ed (1992). GUS Protocols: Using GUS Gene             Omae, F., Miyazaki, M., Enomoto, A., Suzuki, M., Suzuki, Y., and <br />   Reporter Gene Expression. (San Diego, CA: Academic Press).                  Suzuki, A. (2004). DES2 protein responsible phytoceramide <br /> Grilley, M.M., Stock, S.D., Dickson, R.C., Lester, R.L., Takemoto,           biosynthesis mouse small intestine. Biochem. J. 379: 687&#226;&#8364;&#8220;695. <br />   J.Y. (1998). Syringomycin action gene SYR2 essential sphingo-         Pewzner-Jung, Y., Ben-Dor, S., Futerman, A.H. (2006). When do <br />   lipid 4-hydroxylation Saccharomyces cerevisiae. J. Biol. Chem. 273:         Lasses (longevity assurance genes) CerS (ceramide syn- <br />   11062&#226;&#8364;&#8220;11068.                                                                   thases)? Insights regulation ceramide synthesis. J. Biol. <br /> Guillas, I., Jiang, J.C., Vionnet, C., Roubaty, C., Uldry, D., Chuard, R.,       Chem. 281: 25001&#226;&#8364;&#8220;25005. <br />   Wang, J., Jazwinski, S.M., Conzelmann, A. (2003). Human                  Ryan, P.R., Liu, Q., Sperling, P., Dong, B., Franke, S., Delhaize, <br />   homologues LAG1 reconstitute acyl-CoA-dependent ceramide                    E. (2007). A higher plant D8 sphingolipid desaturase preference <br />   synthesis yeast. J. Biol. Chem. 278: 37083&#226;&#8364;&#8220;37091.                           (Z)-isomer formation confers aluminum tolerance yeast and <br /> Haak, D., Gable, K., Beeler, T., Dunn, T. (1997). Hydroxylation           plants. Plant Physiol. 144: 1968&#226;&#8364;&#8220;1977. <br />   Saccharomyces cerevisiae ceramides requires Sur2p Scs7p. J.              Spassieva, S.D., Markham, J.E., Hille, J. (2002). The plant disease <br />   Biol. Chem. 272: 29704&#226;&#8364;&#8220;29710.                                                  resistance gene Asc-1 prevents disruption sphingolipid metabolism <br /> Hanada, K. (2003). Serine palmitoyltransferase, key enzyme sphin-           AAL-toxin-induced programmed cell death. Plant J. 32: <br />   golipid metabolism. Biochim. Biophys. Acta 1632: 16&#226;&#8364;&#8220;30.                        561&#226;&#8364;&#8220;572. <br /> Helliwell, C., Waterhouse, P. (2003). Constructs methods           Sperling, P., Franke, S., Luthje, S., Heinz, E. (2005). Are gluco- <br />   high-throughput gene silencing plants. Methods 30: 289&#226;&#8364;&#8220;295.                 cerebrosides predominant sphingolipids plant plasma mem- <br /> Idkowiak-Baldys, J., Grilley, M.M., Takemoto, J.Y. (2004). Sphin-            branes? Plant Physiol. Biochem. 43: 1031&#226;&#8364;&#8220;1038. <br />   golipid C4 hydroxylation influences properties yeast detergent-           Sperling, P., Ternes, P., Moll, H., Franke, S., Zahringer, U., and <br />   insoluble glycolipid-enriched membranes. FEBS Lett. 569: 272&#226;&#8364;&#8220;276.              Heinz, E. (2001). Functional characterization sphingolipid C4- <br /> Imamura, T., Kusano, H., Kajigaya, Y., Ichikawa, M., Shimada,                hydroxylase genes Arabidopsis thaliana. FEBS Lett. 494: 90&#226;&#8364;&#8220;94. <br />   H. (2007). A rice dihydrosphingosine C4 hydroxylase (DSH1) gene,             Svetek, J., Yadav, M.P., Nothnagel, E.A. (1999). Presence a <br />   abundantly expressed stigmas, vascular cells apical            glycosylphosphatidylinositol lipid anchor rose arabinogalactan <br />   meristem, involved fertility. Plant Cell Physiol. 48: 1108&#226;&#8364;&#8220;1120.     proteins. J. Biol. Chem. 274: 14724&#226;&#8364;&#8220;14733. <br /> Kaul, K., Lester, R.L. (1978). Isolation novel phosphoinositol-     Ternes, P., Franke, S., Zahringer, U., Sperling, P., Heinz, E. <br />   containing sphingolipids tobacco leaves. Biochemistry 17: 3569&#226;&#8364;&#8220;           (2002). Identification characterization sphingolipid D4- <br />   3575.                                                                          desaturase family. J. Biol. Chem. 277: 25512&#226;&#8364;&#8220;25518. <br /> Li, S., Bao, D., Yuen, G., Harris, S.D., Calvo, A.M. (2007). basA          Townley, H.E., McDonald, K., Jenkins, G.I., Knight, M.R., and <br />   regulates cell wall organization asexual/sexual sporulation ratio       Leaver, C.J. (2005). Ceramides induce programmed cell death in <br />   Aspergillus nidulans. Genetics 176: 243&#226;&#8364;&#8220;253.                                   Arabidopsis cells calcium-dependent manner. Biol. Chem. 386: <br /> Liang, H., Yao, N., Song, J.T., Luo, S., Lu, H., Greenberg, J.T.             161&#226;&#8364;&#8220;166. <br />   (2003). Ceramides modulate programmed cell death plants. Genes            Verhoek, B., Haas, R., Wrage, K., Linscheid, M., Heinz, E. (1983). <br />   Dev. 17: 2636&#226;&#8364;&#8220;2641.                                                            Lipids enzymatic activities vacuolar membranes isolated via <br /> Lynch, D.V., Dunn, T.M. (2004). An introduction plant sphingo-           <br /> </body></html>