<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Modeling Spinal Muscular Atrophy Drosophila <br /> Howard Chia-Hao Chang1., Douglas N. Dimlich1., Takakazu Yokokura1., Ashim Mukherjee2, Mark W. <br /> Kankel1, Anindya Sen1, Vasanthi Sridhar1, Tudor A. Fulga1, Anne C. Hart3, David Van Vactor1, Spyros <br /> Artavanis-Tsakonas1,4* <br /> 1 Department Cell Biology, Harvard Medical School, Boston, Massachusetts, United States America, 2 Department Molecular Human Genetics, Banaras Hindu <br /> University, Varanasi, India, 3 Massachusetts General Hospital, Cancer Center Department Pathology, Harvard Medical School, Boston, Massachusetts, United States <br /> of America, 4 Colle`ge France, Paris, France <br />  <br />  <br />  <br />      <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />      Spinal Muscular Atrophy (SMA), recessive hereditary neurodegenerative disease humans, linked mutations <br />      <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span>survival</span> motor neuron (SMN) gene. SMA patients display early onset lethality coupled motor neuron loss and <br />      skeletal muscle atrophy. We used Drosophila, encodes single SMN ortholog, survival motor neuron (Smn), model <br />      SMA, reduction Smn function leads defects mimic SMA pathology humans. Here a <br />      normal neuromuscular junction (NMJ) structure depends SMN expression SMN concentrates post- <br />      synaptic NMJ regions. We conducted screen genetic modifiers Smn <span id='am-239' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-240' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-249' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-257' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-259' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-261' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-271' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-273' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-290' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-296' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-297' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-300' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-308' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-316' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span>phenotype</span> using Exelixis collection of <br />      transposon-induced mutations, affects approximately 50% Drosophila genome. This screen resulted the <br />      recovery 27 modifiers, expanding genetic circuitry Smn include genes previously known <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> <br />      associated locus. Among identified modifiers wishful thinking (wit), type II BMP receptor, was <br />      shown alter Smn NMJ phenotype. Further characterization additional members BMP signaling pathway, <br />      Mothers dpp (Mad) Daughters dpp (Dad), modify Smn NMJ phenotype. The NMJ defects caused <br />      loss Smn function ameliorated increasing BMP signals, suggesting increased BMP activity SMA <br />      patients help alleviate symptoms disease. These results confirm genetic approach likely identify <br />      bona fide modulators SMN activity, especially regarding role neuromuscular junction, consequence, <br />      identify putative SMA therapeutic targets. <br />  <br />   Citation: Chang HC-H, Dimlich DN, Yokokura T, Mukherjee A, Kankel MW, et al. (2008) Modeling Spinal Muscular Atrophy Drosophila. PLoS ONE 3(9): e3209. <br />   doi:10.1371/journal.pone.0003209 <br />   Editor: Alfred Lewin, University Florida, United States America <br />   Received June 11, 2008; Accepted August 18, 2008; Published September 15, 2008 <br />   Copyright: &#195;&#376; 2008 Chang et al. This open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: SMA foundation, NIH <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: artavanis@hms.harvard.edu <br />   . These authors contributed equally work. <br />  <br />  <br />  <br /> Introduction                                                                          SMN activity increased NMJ the <br />                                                                                       organism. <br />    Spinal Muscular Atrophy (SMA) second common                               SMA results loss SMN1 function [6,21], however, the <br /> autosomal recessive genetic disease humans leading                      clinical severity disease correlates SMN2 copy number, <br /> cause genetically linked infant mortality, incidence rate                  varies individuals [22]. As small of <br /> of approximately 1 6000 births [1,2,3]. Clinical manifestation                  functional SMN2 protein produced copy capable of <br /> SMA shows degeneration spinal cord motor neurons                               partially compensating loss SMN1 gene function, <br /> muscle atrophy [4]. SMA linked nearly                            higher copy numbers SMN2 result generally milder forms of <br /> identical genes located chromosome 5, survival motor neuron 1                      SMA. Given severity SMA depends levels of <br /> (SMN1) survival motor neuron 2 (SMN2) [5]. SMN2 differs                      functional SMN, genetic modifiers capable altering SMN <br /> SMN1 10% SMN2 transcripts produce functional                          cellular activity define useful therapeutic targets. This <br /> Smn protein (SMN) <span id='am-395' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-396' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-397' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-398' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-399' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-400' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-401' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-402' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-403' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-404' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-405' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-406' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-407' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-408' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-409' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-410' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-411' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-412' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-413' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-414' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-415' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-416' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-417' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-418' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-419' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-420' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-421' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-422' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-423' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-424' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-425' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-426' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-427' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-428' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-429' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-430' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-431' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-432' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-433' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-434' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-435' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-436' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-437' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-438' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-439' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-440' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-441' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-442' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-443' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-444' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-445' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-446' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-447' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-448' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-449' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-450' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-451' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-452' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-453' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-454' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-455' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-456' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-457' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-458' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-459' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-460' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-461' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-462' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-463' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-464' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-465' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-466' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-467' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-468' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-469' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-470' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-471' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-472' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span>mutation</span> results aberrant                      reasoning prompted explore genetic circuitry capable <br /> splicing [6,7,8].                                                                     affecting SMN activity Drosophila, experimental model <br />    Elegant biochemical studies established importance                      amenable sophisticated genetic manipulations, investigate the <br /> SMN protein ubiquitous, multimeric complex involved                       role SMN system. <br /> assembly spliceosomal small nuclear ribonucleoproteins                                The Drosophila genome harbors single copy Smn gene, <br /> (snRNPs) [9,10,11,12,13,14]. Despite seemingly fundamental                        encodes highly conserved homologue SMN. The Smn <br /> and indispensable role cellular metabolism, reduction SMN                       loss function allele, Smn73Ao, results recessive larval lethality <br /> leads specific neurodegenerative profile associated                    and, importantly, neuromuscular junction abnormalities <br /> disease [1,15,16,17,18]. Though recent studies indicate                       [15,18,23]. In study, characterized additional Smn alleles <br /> that SMN influences motor neuron axonal morphology [19,20],                        demonstrate display NMJ defects. To analyze <br /> remains unclear SMN specific neuromuscular                              tissue-specific requirements SMN, used <span id='am-551' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-552' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-553' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-554' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-555' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-556' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-557' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-558' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-559' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-560' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-561' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-562' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-563' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-564' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-565' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-566' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-567' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-568' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-569' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-570' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-571' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-572' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-573' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-574' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-575' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-576' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-577' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-578' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-579' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-580' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-581' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-582' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-583' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-584' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-585' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-586' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-587' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-588' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-589' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-590' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-591' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-592' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-593' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-594' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-595' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-596' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-597' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-598' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-599' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-600' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-601' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-602' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-603' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-604' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-605' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-606' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-607' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-608' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-609' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-610' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-611' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-612' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-613' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-614' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-615' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-616' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-617' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-618' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-619' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-620' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-621' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-622' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-623' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-624' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-625' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-626' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-627' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-628' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span>RNA interference</span> <br /> junction (NMJ) function, functional requirement                       (RNAi) create series loss function Smn alleles, whose <br />  <br />  <br />         PLoS ONE | www.plosone.org                                                1                            September 2008 | Volume 3 | Issue 9 | e3209 <br />                                                                                                                      A Drosophila Model SMA <br />  <br />  <br /> phenotypes mimic dosage dependent nature SMA                        examined, including muscle (Figure 1A&#226;&#8364;&#8220;D) neurons <br /> pathology. By using muscle (mesoderm) neuronal drivers              (Figure 1F). However, Drosophila NMJ, SMN concentrated <br /> direct expression Smn RNAi constructs, determined           post-synaptic regions muscle. <br /> SMN function required tissues, appears to <br /> be higher sensitivity loss SMN function muscle.         Mutations Smn compromise viability <br />    To identify enhancers suppressors SMN activity              Previous studies determined loss Smn function results in <br /> genetic circuitry Smn, carried genetic screen              larval lethality [15,18]. We examined additional Smn alleles <br /> modifiers Smn73Ao allele using Exelixis collection           Exelixis collection, Smnf01109 Smnf05960 <br /> insertional mutations, affects approximately 50%              [25,26]. Sequence analysis strains indicates allele <br /> Drosophila genome [24,25,26]. Of 17 enhancers 10                   harbors transposon insertion Smn coding region (at <br /> suppressors uncovered screen, significant subset              amino acids I93 Smnf01109 K136 Smnf05960, see <br /> shown capable affecting Smn-related NMJ phenotypes,               Figure 2A) predicted introduce premature stop codon. <br /> validating approach. Amongst Smn modifiers wishful           (Figure 2A). Unlike Smn73Ao allele [15,18], 100% <br /> thinking (wit), encodes type II BMP receptor [27,28].              lethal homo- hemizygous (Smn73Ao/Df(Smn)) backgrounds <br /> Further experiments defined genetic interactions Smn           (Figure 2B), Smnf05960 allele produces small percentage of <br /> other members BMP signaling pathway. We                        escapers (3.3%) mutant larvae isolated cultured at <br /> demonstrated modulation BMP signaling rescues Smn-                 low density. On hand, Smnf01109 allele semi-viable <br /> related NMJ phenotypes, validating genetic approach           (67.7%) (Figure 2B), indicating Smnf01109 Smnf05960 <br /> as means identify novel targets SMN function. Moreover,         alleles null mutations previously suggested [18]. By <br /> seems likely novel targets provide potential          examining viability various Smn allelic combinations <br /> therapeutic value.                                                         (Figure 2B), determined Smnf01109 weakly hypomorphic <br />                                                                            retains degree viability cases tested, while <br /> Results                                                                    Smnf05960 appears act strong loss-of-function allele it <br />                                                                            fails complement Smn73Ao small deficiency that <br /> SMN concentrates post-synaptic regions                       uncovers Smn, Df(3L)SmnX7 (Figure 2B). Ubiquitous (tubulinGAL4, <br /> NMJ                                                                        actinGAL4) expression UAS-FLAG-Smn rescued Smnf05960 lethal- <br />    The dichotomy ubiquitous housekeeping function              ity, demonstrating lethality associated loss Smn <br /> of Smn specific neuromuscular SMA phenotype raises            activity. This consistent earlier studies showing ectopic <br /> the question Smn functions differently neuromus-            SMN expressed control ubiquitous driver <br /> cular junction (NMJ) tissue types. Specifically,             (tubulinGAL4) rescued Smn73Ao lethality [15]. <br /> whether SMN differential expression <span id='am-629' about='xsp:pattern' typeof='owl:Thing'>pattern</span> neurons and <br /> muscle SMN concentrates particular cellular             Constructing RNAi-based hypomorphic Smn alleles <br /> compartments NMJ remain open questions.                                Since clinical severity SMA correlates of <br />    To determine tissue(s) SMN expressed Drosophila          SMN expression, sought better model disease by <br /> we raised antibodies full-<span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> Drosophila SMN (See               generating set Smn alleles varying degrees SMN <br /> Materials Methods) monitored expression pattern                activity using RNAi. A GAL4-inducible vector used to <br /> particularly NMJ. In Western blots performed lysates             produce different double-stranded RNAi transgenic con- <br /> derived S2 cells, 3rd instar larvae wild-type adult heads         structs targeted full-length SMN protein (FL) as <br /> the antibody recognizes single ,28 kD band [18], correspond-             amino-terminal (N) (the entire 59 portion protein to <br /> ing predicted molecular weight Drosophila SMN (Figure S1         including Tudor domain) carboxy-terminal (C) (the 39 <br /> and data shown). Moreover, FLAG-tagged Smn                      portion protein after, including, Tudor domain) <br /> transgenic construct (UAS-FLAG-Smn) expressed                SMN regions (Fig 2A). <br /> control vestigialGAL4 driver, SMN FLAG staining                    Ten independent transgenic strains type construct <br /> overlapped dorsal-ventral (DV) boundary 3rd instar larval        (C, N FL) generated examined effects on <br /> wing discs. In addition, vestigialGAL4-directed expression           lethality SMN activity reduced eliminated using <br /> inducible RNAi allele Smn (see below) abolished SMN                 tubulinGAL4 actinGAL4, ubiquitous GAL4 drivers. It <br /> staining pattern DV boundary larval wing disc             difficult differentiate lethal phases many <br /> (Figure S1). Together, results indicate specificity       strains tubulinGAL4 background, presumably its <br /> antibody raised SMN.                                            higher levels expression. Instead, able use timing <br />    Using antibody probed SMN expression NMJ             lethality presence actinGAL4 choose lines <br /> found antigens clearly concentrated post-synaptic             ([UAS-Smn-RNAi]N4 (N4), [UAS-Smn-RNAi]C24 (C24) [UAS- <br /> regions muscle, co-localizing post-synaptic marker         Smn-RNAi]FL26B (FL26B)) define set alleles representing <br /> Discs Large (DLG) (Figure 1A&#226;&#8364;&#8220;D) [29]. Under conditions,              broadest range detectable lethality analysis <br /> we did detect antigens pre-synaptic region motor         (Figure 3A). <br /> neuron terminal (as defined horseradish peroxidase (HRP)                   Of strains generated, N4 displayed severe <br /> staining) NMJ (Figure 1A&#226;&#8364;&#8220;D). SMN staining                  phenotype, causing mortality early pupal stage. C24 was <br /> observed muscle fibers discrete foci muscle nuclei        severe results lethality later pupal stage N4, <br /> (Figure 1C E), presumably reflect SMN localization            FL26B semi-viable weakest allele <br /> Cajal bodies (gems) demonstrated mammalian cells [9],           (Figure 3A). Under control tubulinGAL4 <br /> in Drosophila ovarian nurse cells oocytes [30]. This post-             driver, N4 caused similar phenotype observed the <br /> synaptic NMJ expression pattern SMN abolished muscle-             Smn73Ao Smnf05960 mutations, suggesting N4 strong <br /> specific Smn RNAi knockdown, demonstrating specificity           hypomorphic Smn allele (data shown). The efficiency RNAi <br /> of anti-SMN antibodies (Figure S2). Consistent general        N4 C24 strains precluded testing whether <br /> role snRNP assembly, SMN detected tissues                    ectopic SMN expression rescue RNAi-induced lethality. <br />  <br />  <br />       PLoS ONE | www.plosone.org                                       2                        September 2008 | Volume 3 | Issue 9 | e3209 <br />                                                                                                                            A Drosophila Model SMA <br />  <br />  <br />  <br />  <br /> Figure 1. SMN localizes post-synaptic region Drosophila NMJ. (A&#226;&#8364;&#8220;D) SMN expression NMJ muscle fibers 6 7. <br /> (A) Pre-synaptic anti-HRP staining (red), (B) post-synaptic anti-DLG staining (blue), (C) anti-SMN staining (green) (D) merge (A&#226;&#8364;&#8220;C). SMN <br /> expression co-localizes DLG post-synaptic region NMJ. (E) SMN staining observed muscle fibers discrete foci nuclei <br /> (arrow). (F) Though pre-synaptic SMN staining observed, robust levels SMN expression seen larval brain. Scale bars (D), (E), (F) <br /> represent 10 mm, 20 mm, 50 mm. <br /> doi:10.1371/journal.pone.0003209.g001 <br />  <br /> However, note fully penetrant pupal lethality                  important genetic tools allow examine the <br /> induced expression tubulinGAL4-directed FL26B                    requirement SMN muscle neurons. <br /> completely rescued addition UAS-FLAG-Smn <br /> construct genetic background (data shown).                        Loss Smn causes neuromuscular junction defects <br />    Consistent results, examination protein derived                 SMA patients experience motor neuron degeneration and <br /> <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> 3rd instar larvae strains presence          muscle atrophy [1,4]. Consistent this, previous work has <br /> actinGAL4 driver revealed significant reductions SMN expres-               shown loss Smn function results defects Drosophila <br /> sion levels (Figure 3D), suggesting observed lethality         NMJ [15]. To confirm extend results, examined the <br /> the direct result SMN protein attenuation. Though                NMJ phenotype observed various Smn genetic backgrounds by <br /> strains did display apparent differences degree                 quantitatively assessing morphology NMJ through <br /> reduction SMN conditions, genetic results             examination synaptic bouton numbers muscles 6 7 <br /> respect viability subsequent experiments investigating NMJ             3rd instar larval NMJs. These boutons visualized by <br /> morphology (see below) strongly suggest RNAi-induced Smn                using antibodies Synaptotagmin (SYT) (pre-synaptic) <br /> strains result varying degrees SMN activity therefore,              DLG (post-synaptic) proteins, respectively (Figure 4A&#226;&#8364;&#8220;G) <br /> alleles different strengths. Importantly, reagents provide           (Materials Methods [29,31,32]). The following Smn genotypes, <br />  <br />  <br />        PLoS ONE | www.plosone.org                                         3                          September 2008 | Volume 3 | Issue 9 | e3209 <br />                                                                                                                           A Drosophila Model SMA <br />  <br />  <br />  <br />  <br /> Figure 2. Smn mutations cause lethality. (A) Schematic representation SMN protein location mutations corresponding Smn <br /> alleles used study. The conserved Tudor domain YG box indicated. Insertion sites transposon induced Smnf05960 Smnf01109 <br /> alleles denoted triangles. Regions Smn transcript targeted RNA interference (RNAi) illustrated lines SMN protein <br /> schematic. (B) Loss Smn function elicits lethality. For individuals given phenotypes, percentages surviving individuals shown are <br /> normalized wild-type. Smn73Ao Smnf05960 homozygotes die late 2nd/early 3rd larval pupal stages, Smnf05960 escapers <br /> are detected. In contrast, 67% Smnf01109 homozygotes survive adulthood. Smnf01109/Smn73Ao Smnf05960/Smn73Ao trans-heterozygous <br /> combinations viable. In addition, small deficiency uncovering entire Smn transcript generated (Df(3L)SmnX7). We crossed three <br /> Smn alleles Df(3L)SmnX7 Smn73Ao/Df(3L)SmnX7 Smnf05960/Df(3L)SmnX7 heterozygotes die 2nd 3rd instar <br /> larval stages, ,60% Smnf01109/Df(3L)SmnX7 viable. Therefore, using lethality criterion, alleles behave loss-of-function <br /> mutations Smnf01109 displaying weakest phenotype three. No obvious maternal paternal effect observed different alleles. <br /> m: maternal contribution, p: paternal contribution. WT wild-type (Canton-S). At 100 individuals examined genotype. <br /> doi:10.1371/journal.pone.0003209.g002 <br />  <br /> which capable reaching 3rd instar larval stages                   nature (Figure 4C). Surprisingly, strong loss function Smnf05960 <br /> (Smn73Ao/Smnf01109, Smnf05960/Smnf01109 Smnf01109/Smnf01109)              mutation, homozygous lethal, failed exhibit detectable <br /> and amenable dissection, examined.                          change NMJ bouton numbers Smn73Ao background. <br />    The severe reduction NMJ bouton numbers observed               However, increase pre-synaptic ghost bouton numbers <br /> in Smn73Ao/Smnf01109 genetic background (Figures 4A&#226;&#8364;&#8220;D F).               [33,34] (where pre-synaptic SYT accompanied post- <br /> The semi-viable Smnf01109 mutation displayed moderate reduction             synaptic DLG) observed individuals (Figure S3), <br /> in NMJ bouton numbers, consistent weakly hypomorphic                 indicating Smnf05960 allele does, indeed, disrupt NMJ <br />  <br />  <br />  <br />        PLoS ONE | www.plosone.org                                         4                         September 2008 | Volume 3 | Issue 9 | e3209 <br />                                                                                                                              A Drosophila Model SMA <br />  <br />  <br />  <br />  <br /> Figure 3. Lethality strongly associates loss Smn function muscle. Survival rates animals expressing N4, C24 FL26B <br /> transgenic UAS-Smn-RNAi constructs control actinGAL4 (A), how24BGAL4 (B), elavGAL4 (C) drivers measured the <br /> following developmental stages: embryo (day 0), 1st instar larva (day 2), 3rd instar larva (day 5), early pupa (day 7), late pupa (day 9), 2-day old adult <br /> (day 12). Each experiment performed triplicate. The pWIZ RNAi vector served control. The survival rates animals calculated <br /> and subtracted control values. The N4, C24 FL26B transgenic animals displayed graded viability drivers tested. Ubiquitous SMN <br /> knockdown (A) leads pupal lethality. Muscle-specific SMN knockdown (B) leads late pupal lethality animals harboring stronger alleles <br /> (N4 C24), greater 90% FL26B individuals survive adulthood. In contrast, reduction SMN neurons using N4 C24 (C) <br /> causes mild lethality (7%) compared control animals. (D) Western blots using anti-SMN polyclonal antibody reduction of <br /> SMN protein 3rd instar larvae UAS-Smn-RNAi transgenic strains combination ubiquitous actinGAL4 driver. The panel <br /> shows graded effect SMN protein levels constructs consistent effects lethality. The panel shows anti-a tubulin <br /> levels, served loading controls. <br /> doi:10.1371/journal.pone.0003209.g003 <br />  <br />  <br /> morphology. The NMJ phenotype associated Smn73Ao/Smnf01109                 SMN muscle versus neuron remains unresolved. We sought to <br /> individuals rescued partially neuronal muscle-directed                address question directly use inducible Smn <br /> expression UAS-FLAG-Smn transgene (Figure 4E&#226;&#8364;&#8220;G), suggesting                RNAi strains (N4, C24 FL26B), expressed using <br /> that SMN expression tissue sufficient restore,         tissue-specific GAL4 drivers. Therefore, chose reduce SMN <br /> partially, NMJ morphology.                                                      expression neuronal muscle lineages using pan-neuronal <br />                                                                                 elavGAL4 [35] pan-muscle how24BGAL4 drivers, respectively <br /> Loss Smn function muscles causes lethality                                (how24BGAL4 mesodermal driver expresses muscles, <br />    Though clear global reduction SMN function elicits             remainder text refer muscle driver) <br /> a larval <span id='am-473' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-474' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-475' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-476' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-477' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-478' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-479' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-480' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-481' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-482' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-483' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-484' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-485' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-486' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-487' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-488' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-489' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-490' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-491' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-492' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-493' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-494' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-495' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-496' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-497' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-498' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-499' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-500' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-501' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-502' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-503' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-504' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-505' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-506' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-507' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-508' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-509' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-510' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-511' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-512' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-513' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-514' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-515' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-516' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-517' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-518' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-519' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-520' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-521' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-522' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-523' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-524' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-525' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-526' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-527' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-528' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-529' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-530' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-531' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-532' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-533' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-534' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-535' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-536' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-537' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-538' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-539' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-540' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-541' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-542' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-543' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-544' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-545' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-546' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-547' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-548' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-549' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-550' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span>lethal phenotype</span> (Figure 2B), relative requirement              [36,37]. <br />  <br />  <br />        PLoS ONE | www.plosone.org                                           5                          September 2008 | Volume 3 | Issue 9 | e3209 <br />                                                                                                                          A Drosophila Model SMA <br />  <br />  <br />  <br />  <br /> Figure 4. Drosophila Smn mutations elicit neuromuscular junction (NMJ) defects. (A&#226;&#8364;&#8220;F) The morphology NMJ, judged bouton <br /> numbers, muscles 6 7 A2 segment observed different genetic backgrounds using pre-synaptic (Synaptotagmin) and <br /> post-synaptic (Discs large) markers, shown green red, respectively. The following genotypes examined: (A) wild-type (Canton-S), (B) <br /> Smnf05960/Smnf01109 (C) Smnf01109/Smnf01109, (D) Smn73Ao/Smnf01109. Of combinations, Smn73Ao/Smnf01109 displayed robust NMJ defect. <br /> These defects partially rescued (E) neuron-specific expression (elavGAL4) (F) muscle-specific expression (how24BGAL4) UAS-FLAG- <br /> Smn transgene. (G) More complete rescue achieved transgene expressed using drivers simultaneously. Bouton numbers <br /> were normalized ratio muscle area. Scale bars represent 20 mm. (H) Diagram bouton numbers genotypes (A&#226;&#8364;&#8220;F), normalized <br /> for muscle area. * P,0.05 determined ANOVA multiple comparisons test. For genotype 15 animals examined. <br /> doi:10.1371/journal.pone.0003209.g004 <br />  <br />  <br />  <br />  <br />        PLoS ONE | www.plosone.org                                        6                         September 2008 | Volume 3 | Issue 9 | e3209 <br />                                                                                                                        A Drosophila Model SMA <br />  <br />  <br />    Reduction SMN tissue causes lethality, however, loss        genes affect Smn-dependent processes using Exelixis <br /> of SMN expression muscle results earlier onset              collection transposon-induced mutations [25,26]. The benefits <br /> lethality, consider severe phenotype                  using collection genetic screen previously <br /> (Figure 3B&#226;&#8364;&#8220;C). In strongest Smn RNAi allele, N4, muscle-specific        described [24]. Notably, collection covers approximately 50% <br /> SMN reduction results 70% mortality (Figure 3B), neuronal          genome harbors gain- loss-of-function <br /> specific reduction results 7% mortality (Figure 3C). As RNAi          mutations exposed GAL4 presence UAS <br /> less efficient neurons, added GAL4-driven dicer construct        sequences insertional transposons [25,26]. While the <br /> increase efficacy SMN reduction conditions [38];         molecular coordinates insertion site known, gene <br /> this resulted obvious enhancement lethality Smn RNAi        assignments ambiguous, modifying transpo- <br /> and elavGAL4 backgrounds (data shown). The GAL4 repressor               son inserted genes. <br /> GAL80 expressed neurons using pan neuronal n-syb                    The screen carried stages identify both <br /> driver [39] overcome potential leakiness how24BGAL4           enhancers suppressors Smn-associated lethality (Figure 6). <br /> driver. Since lethality observed muscle specific reduction       The strong correlation observed degree lethality <br /> SMN closely resembles ubiquitous SMN reduction (compare                NMJ phenotypes using Smn RNAi lines suggested use <br /> Figure 3A B), indicate requirement SMN              lethality screening parameter successful in <br /> muscle (using how24BGAL4) important viability          identifying components SMN genetic network might <br /> requirement neurons.                                                 affect NMJ. Both phases screen utilized Smn73Ao <br />                                                                             allele, gives robust NMJ defect, importantly, contains <br /> Muscle neuronal expression required normal                       <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span>point mutation</span> YG box (Figure 2A), location <br /> NMJ morphology                                                              documented human SMN1 mutation [3]. <br />    Similar tissue-dependent lethality experiments above,             The stage F1 screen designed identify insertions <br /> sought assess impact SMN activity NMJ morphology              produced synthetic lethality semi-lethality (Materials and <br /> using UAS-Smn-RNAi strains, expressed using                Methods) Smn heterozygous background, will <br /> tissue-specific GAL4 drivers.                                               referred enhancers. Using criterion, we <br />    We selectively reduced SMN expression neuron muscle               screened entire Exelixis collection identified 17 insertions <br /> tissues crossing UAS-Smn-RNAi alleles elavGAL4            result Smn73Ao/+ lethality (Figure 7). <br /> how24BGAL4 drivers provide earliest tissue specific                In second stage screen tested ability of <br /> expression robust lethal effect (Figure 3 data             mutations suppress Smn-dependent larval lethality. This was <br /> shown). Visualized SYT (pre-synaptic) DLG (post-synaptic)            accomplished using offspring F1 screen failed to <br /> staining, NMJs Smn RNAi animals containing muscle-              generate synthetic lethality. In phase, screened 7170 strains <br /> or neuron-specific GAL4 driver revealed reduction                  (as Smn73Ao located chromosome, excluded third <br /> number synaptic boutons compared vector controls                chromosome insertions) identified suppressors homo- <br /> (Figure 5A&#226;&#8364;&#8220;M). In N4 strain, neuron muscle specific            zygous Smn73Ao lethality (Figure 7). <br /> attenuation SMN cause approximately 50% reduction in <br /> bouton numbers (Figure 5B, C, K&#226;&#8364;&#8220;M), reduction comparable               NMJ analysis Smn modifiers <br /> what observed Smn73Ao/Smnf01109 larvae (Figure 4D, H).                   To correlate modifier activity NMJ, investigated <br /> Therefore, conclude NMJ morphology dependent                 Smn modifiers (10 suppressors 17 enhancers) <br /> upon pre- post-synaptic SMN activity.                              disrupt Smn RNAi-dependent NMJ defects, using synaptic <br />    Previous studies demonstrated mutations Smn cause              GluRIIA staining assay quantify degree the <br /> decrease staining post-synaptic neurotransmitter receptor        Smn phenotype modified interacting mutation. For this <br /> subunit, GluRIIA [15]. To corroborate results extend           assay, employed C24 Smn RNAi line displays <br /> our characterization tissue-specific requirement SMN           intermediate phenotypic strength. In cases, the <br /> the NMJ, examined GluRIIA [40,41,42] expression pattern              combination modifier <span id='am-317' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-318' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-319' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-320' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-321' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-322' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-323' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-324' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-325' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-326' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-327' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-328' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-329' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-330' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-331' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-332' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-333' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-334' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-335' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-337' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-338' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-341' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-342' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-344' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-346' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-347' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-348' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-349' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-350' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-351' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-352' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-353' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-354' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-355' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-356' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-357' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-358' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-359' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-361' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-362' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-363' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-364' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-365' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-366' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-367' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-368' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-369' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-370' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-372' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-373' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-374' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-375' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-376' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-377' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-378' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-379' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-380' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-381' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-382' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-383' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-384' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-385' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-386' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-389' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-390' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-391' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-392' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-393' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-394' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span>insertion mutation</span> induced a <br /> (See Materials Methods) UAS-Smn-RNAi backgrounds.                statistically significant change C24 GluRIIA phenotype <br /> We consistent significant quantitative reduction             (Figure 7 Figure S5 S6). Amongst validated modifier <br /> synaptic GluRIIA levels Smn expression decreased using             insertions, degree enhancement suppression varied <br /> either neuron- (elavGAL4) muscle-specific (mhcGAL4) drivers.             depending locus; control crosses demonstrated there <br /> GAL4-only controls significant effect GluRIIA staining            significant Smn-independent changes GluRIIA localiza- <br /> intensity. Consistent trend observed severity       tion tested insertion lines. Three lines (f04448, d09801 &amp; <br /> lethal phenotype, strongest Smn RNAi alleles caused greatest        d00698) failed modify C24 GluRIIA staining retested <br /> reduction GluRIIA expression levels, suggesting GluRIIA             using weaker Smn RNAi strain, N13 (Strain f04448 d09801 <br /> levels sensitive dose functional SMN protein thus,        enhanced, d00698 showed interaction (data shown), <br /> would useful phenotypic metric validate potential          highlighting importance NMJ phenotype secondary <br /> modifiers Smn NMJ phenotype.                                         screening tool (Figure 7)). Thus, majority modifiers the <br />    Our analysis indicates normal NMJ morphology requires               Smn73Ao lethal phenotype confirmed second, independent <br /> SMN activity muscle neurons. However, appears                assay. All insertions modified Smn NMJ <br /> that loss SMN activity muscle causes severe lethal         phenotype, validating efficacy screen suggesting that <br /> phenotype (Figure 3B), conclusion consistent             screen prove effective tool identification of <br /> finding SMN protein concentrated post-synaptic           candidate genes relevant SMA disease state. <br /> regions muscle (Figure 1A&#226;&#8364;&#8220;D). <br />                                                                             Neuronal overexpression wishful thinking (wit) <br /> Identification genetic modifiers Smn                                  enhances Smn NMJ defects <br />  To gain insights genetic circuitry capable modulating             To validate approach, sought examine the <br /> SMN activity vivo, employed genetic approach screen          relationship wishful thinking (wit) Smn greater detail. <br />  <br />  <br />       PLoS ONE | www.plosone.org                                        7                         September 2008 | Volume 3 | Issue 9 | e3209 <br />                                                                                                                              A Drosophila Model SMA <br />  <br />  <br />  <br />  <br /> Figure 5. Muscle neuron specific Smn RNAi knockdown causes NMJ defects. (A&#226;&#8364;&#8220;I) Reduced SMN expression N4, C24 FL26B <br /> UAS-Smn-RNAi transgenic constructs elicits graded effects NMJ morphology using ubiquitous actinGAL4 (A, D, G) tissue-specific <br /> how24BGAL4 (muscle) (B, E, H) elavGAL4 (neuron) (C, F, I) drivers. Vector (pWIZ) controls shown (J, K, L). In images pre- post- <br /> synaptic tissues labeled antibodies Synaptotagmin (green) Discs large (red), respectively. (M) Bouton counts NMJs from <br /> the genotypes shown (A&#226;&#8364;&#8220;L) normalized muscle area subtracted vector controls. For genotype 15 animals were <br /> examined. * P,0.01 **P,0.05 determined ANOVA multiple comparisons test. Scale bars represent 15 mm. <br /> doi:10.1371/journal.pone.0003209.g005 <br />  <br />  <br /> wit particular previously                   function mutations cause NMJ defects, wit gain-of- <br /> implicated NMJ function [27,28] serve                    function causes obvious NMJ morphological changes. As the <br /> paradigm validating ability screen identify bona fide         wit allele identified Smn enhancer, witd02492, associated with <br /> Smn genetic modifiers.                                                          GAL4-responsive transposon, likely represented <br />    wit encodes type II BMP receptor functions                       gain-of-function mutation. Consistent notion, an <br /> retrograde signaling component neurons [27,28]. wit loss-of-                 independent UAS-wit transgene [27,28] behaved similar <br />  <br />  <br />        PLoS ONE | www.plosone.org                                           8                          September 2008 | Volume 3 | Issue 9 | e3209 <br />                                                                                                                               A Drosophila Model SMA <br />  <br />  <br />                                                                                   moderate reduction number NMJ boutons caused the <br />                                                                                   hypomorphic Mad12 allele (Figure 9D G) clearly <br />                                                                                   exacerbated mutations Smn (Figure 9E&#226;&#8364;&#8220;G). These results <br />                                                                                   suggest perturbations BMP signaling able modify <br />                                                                                   Smn-dependent phenotypes larval NMJ. <br />  <br />                                                                                   SMN activity affects BMP signaling <br />                                                                                      To validate link SMN BMP <br />                                                                                   signaling pathway examined effect reduced SMN levels <br />                                                                                   pMAD expression. Though Mad required retrograde <br />                                                                                   signaling neurons NMJ [34,45], lack detectable <br />                                                                                   pMAD staining NMJ precluded use NMJ a <br />                                                                                   means assess SMN affect expression. Instead, we <br />                                                                                   examined pMAD expression pattern adjacent anterior- <br />                                                                                   posterior compartment boundary 3rd instar larval wing discs <br /> Figure 6. Schematic representation Smn modifier screen.                    [46] (Figure 10) using engrailedGAL4 vestigalGAL4 directed <br /> Depicted crosses performed identify enhancers                      expression N4 RNAi transgene (Figure 10 Figure S4 <br /> suppressors Smn-associated lethality. In stage screen,        respectively). Regions SMN levels reduced display <br /> designed identify Smn enhancers, Smn73Ao tubulinGAL4 e/TM6B virgin <br /> females mated males Exelixis collection strains. In this <br />                                                                                   attenuated pMAD staining (Figure 10C&#226;&#8364;&#8220;E). Moreover, adult wing <br /> stage, entire Exelixis collection, affects approximately 50%         abnormalities occur regions reduced SMN expression, <br /> the Drosophila genome, tested. In F1 generation, mutations                including thicker wing veins shorter posterior cross-veins <br /> that resulted synthetic lethality reduced viability trans       (Figure 10F). These phenotypes similar phenotypes elicited <br /> Smn73Ao tubulinGAL4 e chromosome defined enhancers. In                mutations BMP pathway components thickveins <br /> second stage screen, males F1 crosses failed             (tkv) glass boat (gbb) [45,47,48]. Thus, BMP signaling in <br /> enhancement (P[Exelixis]/+; Smn73Ao tubulinGAL4 e/TM6B) mated <br /> to Smn73Ao e/TM1, Me&#194;&#180; virgin females identify mutations that <br />                                                                                   wing appears affected loss SMN activity through <br /> suppressed Smn73Ao tubulinGAL4 e/Smn73Ao, e lethal phenotype.                 regulation activated Mad, corroborating link between <br /> We performed F2 suppressor screen Exelixis mutations            Smn BMP signaling pathway. <br /> and second chromosomes testing chromosome mutations <br /> would require placing mutations cis Smn. Additional                 A Dad loss function allele capable rescuing Smn <br /> assays employed eliminate false positives (See Materials and <br /> Methods). Seventeen enhancers suppressors met criteria.             NMJ defects <br /> All 27 modifiers subsequently examined ability modify              We extended observations probing relationship <br /> the Smn NMJ phenotype GluRIIA staining (Figures S5 S6).                    BMP pathway antagonist, Daughters dpp (Dad), <br /> doi:10.1371/journal.pone.0003209.g006                                             Smn. Dad encodes Drosophila homolog mammalian anti- <br />                                                                                   Smad acts Mad antagonist [44,49,50]. Since Dad mutants <br /> fashion witd02492 conditions used screen (data                exhibit pre-synaptic overgrowth [49], tested the <br /> not shown). In addition, detected increased expression WIT                  Dad271-68 null mutation rescue Smn NMJ phenotype. <br /> in witd02492 animals containing tissue-specific GAL4 drivers (data                Consistent previous reports [49], 3rd instar larvae homozy- <br /> not shown).                                                                       gous Dad 271-68 display dispersed SYT expression the <br />    Over-expression WIT neurons using neuron-specific                    NMJ control larvae (Figure 11C). However, contrast to <br /> elavGAL4 driver Smn73Ao Smnf01109 heterozygous                 previous studies, total bouton number, determined <br /> background resulted reduced NMJ bouton numbers relative                     DLG post-synaptic staining, slightly reduced. <br /> elavGAL4 Smn73Ao/+, elavGAL Smnf01109/+ UAS-wit; elavGAL4                     Importantly, Smn73Ao/Smnf01109 NMJ phenotype sup- <br /> controls (Figure 8). This result suggests Smn-dependent                  pressed introduction Dad 271-68 (Figure 11B, D, E), <br /> NMJ phenotype sensitive elevated WIT levels.                                providing genetic evidence element BMP <br />                                                                                   pathway interacts Smn. It appears elevating BMP activity <br />                                                                                   complete loss Dad function suppresses effects of <br /> A Mad mutation enhances Smn NMJ phenotype                                     Smn mutations NMJ (Figure 11D, E). A prediction this <br />    Given involvement wit NMJ interaction               model pharmacological reagents increase BMP <br /> Smn, hypothesized Smn heterozygous background leads                    signaling ameliorate Smn-associated NMJ defects, thereby <br /> to increase sensitivity dosage BMP NMJ                     identifying set targets potential therapeutic value. <br /> development. Thus, conditions elevated levels WIT in <br /> Smn heterozygotes, possible normal BMP signaling the <br /> NMJ altered, titration BMP ligand, <br />                                                                                   Discussion <br /> thereby resulting NMJ defects. If hypothesis correct,                     SMN1 determining gene Spinal Muscular Atrophy <br /> mutations BMP components downstream wit                     (SMA) [5], devastating neurodegenerative disease humans <br /> enhance Smn NMJ phenotype. Therefore, tested                       currently available FDA-approved drug treatment. <br /> Mothers dpp (Mad) Smn interaction NMJ. Mad                     Though general biochemical function SMN snRNP <br /> encodes Drosophila homolog R-Smad, downstream effector                   assembly documented [51,52,53], remains to <br /> of pathway [34,43,44]. Pathway activation leads phosphor-                  learned action NMJ genetic circuitry <br /> ylation MAD (pMAD), subsequent translocation                    capable affecting SMN activity. Specifically, remains <br /> nucleus regulates gene expression [34,43,44]. To examine                 unclear NMJ pathology SMA the <br /> the consequences Smn/Mad interaction NMJ, used                   ubiquitous loss SMN function SMN unique <br /> hypomorphic Mad12 allele [34] combination multiple Smn                    role NMJ. Here, utilized Drosophila investigate the <br /> alleles monitor phenotypic effects NMJ. The                         tissue specificity Smn identify genes interact Smn. <br />  <br />  <br />        PLoS ONE | www.plosone.org                                             9                         September 2008 | Volume 3 | Issue 9 | e3209 <br />                                                                                                                          A Drosophila Model SMA <br />  <br />  <br />  <br />  <br /> Figure 7. Modifiers Smn phenotypes. Listed insertions enhance (top) suppress (bottom) Smn73Ao-dependent lethality. Due to <br /> the site transposon insertion, unambiguous gene assignments possible instances (shaded). Strains designations begin with <br /> &#226;&#8364;&#732;&#226;&#8364;&#732;d&#226;&#8364;&#8482;&#226;&#8364;&#8482; &#226;&#8364;&#732;&#226;&#8364;&#732;f&#226;&#8364;&#8482;&#226;&#8364;&#8482; contain GAL4 responsive elements (UAS), strains beginning &#226;&#8364;&#732;&#226;&#8364;&#732;c&#226;&#8364;&#8482;&#226;&#8364;&#8482; &#226;&#8364;&#732;&#226;&#8364;&#732;e&#226;&#8364;&#8482;&#226;&#8364;&#8482; GAL4-inducible. Gene assignments were <br /> determined using FlyBase (http://www.flybase.org/). Human homologs determined using NCBI BLAST, NCBI UniGene (NCBI) (http://www.ncbi. <br /> nlm.nih.gov/sites/entrez?db = unigene) ENSEMBL genome browser (http://www.ensembl.org). Annotated functions determined based on <br /> FlyBase, NCBI Entrez Gene SMART (http://smart.embl-heidelberg.de/). Modification NMJ morphology muscles 6 7 A2 <br /> segment assayed elavGAL4 pWIZ[UAS-Smn-RNAi]C24 background trans identified modifiers using pre-synaptic (Horseradish <br /> peroxidase) post-synaptic (GluRIIA) markers (see Materials Methods). In cases did significant phenotypic alteration, <br /> additional pWIZ[UAS-Smn-RNAi]N13 allele used (see text). The degrees change observed GluRIIA staining categorized follows: <br /> +++, strong; ++, moderate; +, weak; N.E., No Effects. <br /> doi:10.1371/journal.pone.0003209.g007 <br />  <br />  <br />  <br />  <br />        PLoS ONE | www.plosone.org                                       10                         September 2008 | Volume 3 | Issue 9 | e3209 <br />                                                                                                                        A Drosophila Model SMA <br />  <br />  <br />  <br />  <br /> Figure 8. wit overexpression neurons exacerbates Smn-dependent NMJ defects. A gain-of-function mutation wishful thinking (wit), <br /> witd02492, identified enhancer screen. To investigate interaction wit Smn NMJ, used neuron- <br /> specific driver, elavGAL4 express WIT neurons. (A&#226;&#8364;&#8220;F) The morphology NMJ, judged bouton numbers, muscles 6 7 the <br /> A2 segment observed different genetic backgrounds using pre-synaptic (Synaptotagmin) post-synaptic (Discs large) markers, shown <br /> in green red, respectively. The following genotypes examined: (A) elavGAL4/+, (B) elavGAL4, Smn73Ao/+, (C) elavGAL4, Smnf01109/+, (D) <br /> elavGAL4/UAS-wit2A, (E) elavGAL4, Smn73Ao/UAS-wit2A, (F) elavGAL4, Smnf01109/UAS-wit2A, (G) Bouton counts genotypes (A&#226;&#8364;&#8220;F wild-type). <br /> Consistent previous reports, neural induced expression UAS-wit2A transgene obvious effect NMJ bouton number. A synergistic <br /> effect observed addition single Smn allele (Smn73Ao Smnf01109) background, leading reduction NMJ bouton <br /> numbers. The phenotype severe Smnf01109 background. Smnf01109 showed approximate 50% reduction bouton numbers while <br /> Smn73Ao reduced bouton count 20%. elavGAL4, Smn73Ao/+ (B) elavGAL4, Smnf01109/+ (C) individuals display significant reduction NMJ <br /> bouton numbers compared wild-type (G). Bouton counts determined above. Error bars s.e.m.; * P,0.02 determined ANOVA <br /> multiple comparisons test wild-type controls. n 15&#226;&#8364;&#8220;20 animals genotype. Bouton numbers genotype normalized <br /> to ratio muscle areas. Scale bars represent 20 mm. <br /> doi:10.1371/journal.pone.0003209.g008 <br />  <br />  <br />       PLoS ONE | www.plosone.org                                       11                        September 2008 | Volume 3 | Issue 9 | e3209 <br />                                                                                                                      A Drosophila Model SMA <br />  <br />  <br />  <br />  <br /> Figure 9. Loss mad function enhances Smn NMJ defects. (A&#226;&#8364;&#8220;F) The morphology NMJ, judged bouton numbers, between <br /> muscles 6 7 A2 segment observed different genetic backgrounds using pre-synaptic (Synaptotagmin) post-synaptic (Discs <br /> large) markers, shown green red, respectively. The following genotypes examined: (A) wild-type, (B) Smn73Ao/+, (C) Smnf01109/+, (D) <br /> mad12/+, (E) Smn73Ao/mad12 (F) Smnf01109/mad12. (G) Bouton counts genotypes (A&#226;&#8364;&#8220;F). Introduction mad12 Smn73Ao/+ a <br /> Smnf01109/+ background dominantly reduces Smn-dependent NMJ bouton count. Error bars s.e.m.; *P,0.02 determined ANOVA <br /> multiple comparisons test wild-type controls. n 15&#226;&#8364;&#8220;20 animals genotype. Bouton numbers genotype normalized <br /> to ratio muscle areas. Scale bars represent 20 mm. <br /> doi:10.1371/journal.pone.0003209.g009 <br />  <br />  <br />  <br />  <br />       PLoS ONE | www.plosone.org                                      12                        September 2008 | Volume 3 | Issue 9 | e3209 <br />                                                                                                                        A Drosophila Model SMA <br />  <br />  <br />  <br />  <br /> Figure 10. Smn knockdown reduces pMAD signals. (A&#226;&#8364;&#8220;B) Wild-type wing discs 3rd instar larvae stained antibodies SMN <br /> (red) (A) phosphorylated MAD (pMAD) (green) (B). (C&#226;&#8364;&#8220;D) 3rd instar wing discs engrailedGAL4, pWIZ[UAS-Smn-RNAi]N4 animals stained with <br /> antibodies SMN (red) (C) pMAD (green) (D). (E) Merge (C) (D). pMAD staining reduced posterior region wing disc <br /> where SMN expression decreased (yellow line). (F) A wing engrailedGAL4, pWIZ[UAS-Smn-RNAi]N4 transgenic adult exhibits defects the <br /> posterior crossvein regions distal portions wing veins L4 L5 (arrow). Scale bars represent 40 mm. <br /> doi:10.1371/journal.pone.0003209.g010 <br />  <br /> These genes, apart intrinsic value providing insight          NMJ morphological defects [15]. We corroborated obser- <br /> into role SMN NMJ, define novel                      vations examination extant novel Smn <br /> therapeutic targets.                                                        mutations varied severities, including GAL4-inducible <br />    Previous studies based primarily analysis Smn73Ao          Smn RNAi alleles generated study. These hypomorphic <br /> allele demonstrated reduced Smn activity causes lethality          strains reduce SMN expression levels different degrees a <br />  <br />  <br />        PLoS ONE | www.plosone.org                                      13                        September 2008 | Volume 3 | Issue 9 | e3209 <br />                                                                                                                         A Drosophila Model SMA <br />  <br />  <br />  <br />  <br /> Figure 11. A dad <span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span>null allele</span> rescues Smn NMJ defects. (A&#226;&#8364;&#8220;D) The morphology NMJ, judged bouton numbers, muscles 6 <br /> and 7 A2 segment observed different genetic backgrounds using pre-synaptic (Synaptotagmin) post-synaptic (Discs large) <br /> markers, shown green red, respectively. The following genotypes examined: (A) wild-type (B), Smn73Ao/Smnf01109, (C) dad271-68 <br /> homozygotes (D) Smn73Ao dad271-68/Smnf01109 dad271-68. (E) Bouton counts genotypes (A&#226;&#8364;&#8220;D). Smn73Ao/Smnf01109 individuals display strongly <br /> reduced NMJ bouton numbers dad271-68 homozygotes greater two-fold bouton numbers relative Smn73Ao/Smnf01109 <br /> animals. The Smn73Ao dad271-68/Smnf01109 dad271-68 double mutants behave like dad271-68 homozygotes. Error bars s.e.m.; n 15&#226;&#8364;&#8220;20 animals for <br /> wild-type Smn73Ao/Smnf01109. n 30 dad271-68/dad271-68 Smn73Ao, dad 271-68/Smnf01109, dad271-68. *P,0.002 ANOVA multiple <br /> comparisons test. Bouton numbers genotype normalized ratio muscle area. Scale bars represent 15 mm. <br /> doi:10.1371/journal.pone.0003209.g011 <br />  <br />  <br />  <br /> manner formally analogous decreased SMN levels observed                   Our examination Smn NMJ structure Drosophila using pre- <br /> SMA patients. Additionally, strains model dosage-              post-synaptic markers, SYT DLG, respectively, revealed <br /> dependent nature SMA [1,2] developmental arrest                    significant losses synaptic bouton numbers multiple Smn <br /> associated animals correlates extent                  backgrounds (Figure 4). Moreover, backgrounds, also <br /> morphological abnormalities observed NMJ.                             detected reduced post-synaptic GluRIIA expression (data not <br />  <br />  <br />        PLoS ONE | www.plosone.org                                       14                        September 2008 | Volume 3 | Issue 9 | e3209 <br />                                                                                                                          A Drosophila Model SMA <br />  <br />  <br /> shown), consistent previous analyses Smn73Ao NMJ [15].            (Figure 5), reasoned identification enhancers and <br /> Together, results suggest loss SMN function                  suppressors Smn73Ao-dependent lethality likely yield <br /> Drosophila causes aberrant neuromuscular synaptic structure,                  genes function NMJ. Our genetic screen using an <br /> mimicking pathology SMA [1,54]. In addition,                     allele (Smn73Ao) encodes point mutation seen SMA <br /> structural abnormalities consistent altered electro-             patients [3] resulted identification twenty-seven modifiers <br /> physiological profile previously observed Drosophila Smn73Ao               Smn lethality. Though recognize genetic circuitry in <br /> animals [15]. It noted glutamate receptor                Drosophila differ exists humans, expect <br /> subunits display altered transcriptional profiles Smn73Ao                substantial overlap given conservation gene <br /> background; specifically GluRIIA GluRIIB transcript                   function species. <br /> levels decreased GluRIIC levels increased [55].                  Despite essential role SMN snRNP assembly [12], an <br /> Therefore, combining genetic morphological analyses                    unexpected result screen modifying <br /> pathological changes synaptic structure future electrophys-           insertions unambiguous gene assignments made <br /> iological studies necessary understand thoroughly             appear function RNA processing. Consistent notion, <br /> the synaptic consequences SMN loss SMA.                                 direct attempts identify genetic relationships SMN and <br />    A longstanding question pathology SMA relative            known components SMN multimeric complex, including <br /> neuronal muscle contribution SMN function. The RNAi                    deficiencies uncover Drosophila Gemin homologs, did not <br /> strains allowed reduce SMN function tissue-specific                affect Smn73Ao heterozygous phenotype (data shown). One <br /> fashion therefore, address issue directly. We              possible explanation removal additional components of <br /> SMN required neurons muscle normal NMJ                     SMN complex enhance Smn-related phenotypes since <br /> morphology GAL4-inducible RNAi reduction SMN                         SMN activity critical initial steps SMN complex <br /> neurons muscle decrease NMJ bouton                         assembly. Hence, altering activity &#226;&#8364;&#732;&#226;&#8364;&#732;downstream&#226;&#8364;&#8482;&#226;&#8364;&#8482; directly- <br /> numbers (Figure 5) GluRIIA staining (Figures S5 S6                interacting partners SMN SMN complex not <br /> data shown). In addition, expression SMN tissue              affect Smn-related phenotypes. <br /> is sufficient partially rescue NMJ defects associated loss               Though unambiguously identified modifier genes <br /> of Smn function (Figure 4E&#226;&#8364;&#8220;G). These results consistent                   obvious role snRNP assembly; did recover genes <br /> with previous reports Drosophila, zebrafish mouse                      (wishful thinking, fmr1 cutup) shown previously to <br /> [15,16,54,56,57,58,59] indicate interdependence                    function NMJ [27,28,63,64]. Moreover, majority the <br /> neuron muscle SMN activity.                                               remaining genes, previously known NMJ function, <br />    In contrast requirement Smn muscle neurons            modified Smn NMJ phenotypes. Thus genetic approach <br /> at NMJ, demonstrated muscle specific reduction Smn             efficient identifying genes related Smn NMJ function. <br /> causes severe lethal phenotype (Figure 3B&#226;&#8364;&#8220;C). We                This suggests similar approach utilizing hypomorphic Smn <br /> know cause lethality. It possible earlier onset        allele (e.g. UAS-Smn-RNAi) closely approximates the <br /> of lethality observed how24BGAL4 reduction SMN                 dosage dependent nature human disease condition may <br /> result leakiness driver loss SMN activity            identify additional members Smn genetic circuitry. <br /> dividing cells (the elavGAL4 driver expresses predominantly post-             An analysis interacting loci according molecular <br /> mitotic cells). However, results raise possibility           functions reveals assortment functional categories including <br /> organism vulnerable SMN reduction muscle.                   cytoskeleton interaction proteins (moe ctp), transcription factors <br /> This consistent post-synaptic concentration               (net) metabolic enzymes (CG17323 CG10561). Identified <br /> SMN NMJ (Figure 1). The functional relevance                  interactors include members signal transduction <br /> observations remains determined; however, previous report             pathways (e.g. BMP (wit), FGF (btl) Nuclear Hormone <br /> has suggested Smn specific function                    Receptor (Eip75B)), raising possibility evolutionarily <br /> Drosophila adult skeletal muscle SMN expressed                conserved signaling pathways integrate SMN targets of <br /> sarcomere shown bind a-actinin [18]. Together,                  SMN function(s). Though detailed analyses nature of <br /> these data provide plausible explanations muscle                   links (synergistic parallel) pathways SMN <br /> rendered susceptible loss Smn function.                            necessary, provide strong evidence supporting connection <br />    Current therapeutic strategies treatment SMA based              BMP signaling Smn NMJ testing additional <br /> on dosage dependent nature disease, focusing drugs              upstream downstream elements pathway. Our <br /> that increase SMN2 transcription splicing efficiency [60,61].             molecular genetic analysis clearly indicates SMN influences <br /> Though strategies ultimately prove successful treating           BMP activity. It remains determined SMN acts in <br /> SMA, complementary therapies allow delivery                  muscle influence retrograde BMP signaling the <br /> combination drugs shown successful               WIT receptor, example regulating activity WIT <br /> alleviating symptoms diseases, AIDS [62].                ligand (GBB). It possible SMN functions cell- <br /> Hence, identification and, ultimately, manipulation                autonomously neurons affect activity MAD its <br /> genetic elements affect SMN activity necessary                 antagonist, DAD. As BMP signaling pathway been <br /> treat SMA effectively. Though previous biochemical studies                    implicated neurodegenerative diseases, including Du- <br /> provide valuable fundamental knowledge SMN function,                   chenne Dystrophy Marfan Syndrome [65], probable that <br /> our current understanding SMN limited mainly               BMP signaling plays role pathology SMA in <br /> binding partners genetic modulators [12,19]. Thus,               humans. <br /> performed genome-wide genetic screen Drosophila identify                 Similar observed SMA, results confirm the <br /> novel components Smn genetic circuitry broaden                  susceptibility Drosophila NMJ lower levels SMN, and <br /> knowledge function seek potentially novel therapeutic           screen identified genes modify Smn NMJ <br /> approaches augmentation SMN2 expression.                        phenotypes. In recent studies, micro-array based approaches <br />    Since observed severity/onset Smn-dependent                 analyzed effect reduced Smn levels tissue-specific gene <br /> mortality (Figure 3) corresponds degree NMJ defects                 expression genome-wide level [52,55]. They identified genes <br />  <br />  <br />        PLoS ONE | www.plosone.org                                        15                         September 2008 | Volume 3 | Issue 9 | e3209 <br />                                                                                                                          A Drosophila Model SMA <br />  <br />  <br /> whose splicing susceptible reduced SMN function [52]              standard F2 screen exception crosses were <br /> genes involved general metabolic processes [55]. These screens            designed identify synthetic lethal interactions Smn the <br /> are clearly valuable means assess housekeeping function          F1. In screen utilized Smn73Ao allele cis a <br /> Smn. However, unlike genes recovered screen,               ubiquitously expressed tubulinGAL4 driver (Lee Luo, 1999). <br /> of affect NMJ structure, remains determined                   Initially, Smn73Ao tubulinGAL4 e/TM6B virgin females crossed <br /> whether genes identified transcriptional profiling           entire Exelixis mutant collection identify insertions that <br /> involved development and/or maintenance NMJ.                   elicit F1 synthetic lethality reduced viability. From results, <br /> Thus, genetic approach uncovered elements, revealing               arbitrarily defined strain candidate enhancer one <br /> potential NMJ-specific role Smn.                                         displayed viability 30%. In second <br />   In study, used Drosophila genetics broaden             generation, F1 males strains failed elicit synthetic <br /> understanding Smn neuromuscular junction probe             lethality crossed Smn73Ao e/TM1, Me&#194;&#180; virgins test for <br /> genetic circuitry Smn, illustrating utility genetic              ability suppress homozygous Smn73Ao larval lethality. In the <br /> approach identification novel genes impact Smn                F2 screen, candidate suppressors identified presence of <br /> activity. Given high degree genomic conservation, use              individuals bearing marker ebony (e), visible both <br /> model systems Drosophila will, view, lead           pupae adults. <br /> thorough understanding SMA pathology point potential <br /> therapeutic strategies.                                                      Secondary genetic assays <br />                                                                                 We used secondary genetic assays evaluate candidate <br /> Materials Methods                                                        modifiers. Initially, candidate enhancers crossed to <br />                                                                              tubulinGAL4 e alone, determine observed effects <br /> Drosophila stocks                                                            viability Smn-dependent. <br />    All stocks maintained standard cornmeal/yeast/                       Resultant strains candidate suppressors then <br /> molasses/agar medium 25uC. The Smn73Ao, P{EP-                             crossed stocks containing additional combinations Smn alleles <br /> gy2}EY14384, wit A12 wit B11 alleles obtained              GAL4 drivers observe effects viability. These stocks <br /> Bloomington Drosophila Stock Center (Bloomington, IN). The                   included Smnf05960 Smn73Ao combination <br /> Smnf05960 Smnf01109 alleles Exelixis collection          tubulinGAL4 actinGAL4 drivers. The actinGAL4 driver <br /> Harvard Medical School. Dad 271-68 Mad12 gifts                 tested. Twenty-seven strains modified Smn- <br /> Graeme Davis. P[UAS-wit2A] transgenic animals gifts                dependent lethality. Of these, seventeen classified as <br /> Hermann Aberle. To generate P[UAS&#226;&#8364;&#8220;FLAG-Smn] strains,                   enhancers suppressors. <br /> full-length Smn cDNA cloned amino-terminal FLAG- <br /> tagged vector, FLAG-pUAST introduced w1118 animals. <br /> Several independent strains isolated subsequently tested            NMJ analysis 27 Smn interactors <br /> for ability rescue Smn-dependent lethality homozygous             Strains containing twenty-seven interactors combination <br /> Smn73Ao e background. The majority homozygous Smn73Ao e                   pWIZ[UAS-Smn-RNAi]C24 generated. These were <br /> mutants die 2nd 3rd larval stages, 10%             tested ability modify NMJ phenotype associated with <br /> reach 3rd instar larval stage. Expression construct using        elavGAL4 (neuronal) directed expression pWIZ[UAS-Smn- <br /> the ubiquitous actinGAL4 tubulinGAL4 drivers partially                   RNAi]C24 pWIZ[UAS-Smn-RNAi]N13. GluRIIA morphometric <br /> suppressed lethality, 50% animals survived             analyses performed described previously [66]. <br /> late pupal stage 20% survived adult stage. The null <br /> allele Smn, Df(3L)SmnX7, generated imprecise excision           Lethality assays <br /> the P{EPgy2}EY14384. 1,626 excision events isolated                    Fertilized eggs collected apple juice agar plates. Before <br /> 17 failed complement Smn73Ao, including Df(3L)SmnX7.                      collection, adults allowed lay 2 hours. All F0 strains <br /> Subsequent sequence analysis Df(3L)SmnX7 determined                   balanced TM6B, Dfd:YFP, Tb CyO Dfd:YFP, the <br /> excision event removed entire SMN transcript              non-YFP embryos collected using fluorescence dissection <br /> affecting nearby loci (93 bp upstream transcription start site        scope (Zeiss). Fertilized embryos place fresh apple <br /> through final 44 bp 39 UTR). To generate              juice plates containing yeast paste. Each plate contained 20&#226;&#8364;&#8220;25 <br /> Smn RNAi constructs, different portions Smn cDNA                embryos avoid over-crowding. The animals allowed to <br /> were cloned pWIZ vector: entire cDNA (FL                        grow different developmental stages controlled temperature <br /> constructs), amino-terminal portion including              (25 uC) survival determined visual inspection. <br /> entire Tudor domain (N constructs) carboxy-terminal <br /> portion after, including, Tudor domain (C constructs)            Immunohistochemistry microscopy <br /> (Figure 2A). These constructs introduced w1118                   Primary antibodies used following dilutions: <br /> embryos germ-line transformation according standard                    monoclonal anti-DLG (1:500) (Developmental Studies Hybridoma <br /> procedures (by CBRC fly core facility, Massachusetts General             Bank), polyclonal anti-Synaptotagmin (1:1000) (a gift Hugo <br /> Hospital). Multiple independent insertions obtained            Bellen), polyclonal anti-SMN (1:250, NMJ staining), monoclonal <br /> construct, including pWIZ[UAS-Smn-RNAi]N4, pWIZ[UAS-                     anti-SMN (1:500, wing disc), polyclonal anti-pMAD (1:250) (a gift <br /> Smn-RNAi]C24 pWIZ[UAS-Smn-RNAi]FL26B transgenic strains                  Carl-Henrik Heldin). Anti-SMN monoclonal polyclonal <br /> that used analyses conducted study. Detailed            antibodies generated immunizing animals purified <br /> primer sets methods used sequencing Smnf05960                    full-length SMN protein 66His-tag fused carboxy- <br /> Smnf01109 alleles available request.                                terminus (Cocalico Biologicals, Inc.). Texas-red conjugated anti- <br />                                                                              HRP (1:250), FITC- (1:40) Cy5- (1:40) conjugated anti-rabbit <br /> Exelixis screen                                                              anti-mouse secondary antibodies purchased from <br />   The screen performed combined elements standard F1                 Jackson Immunoresearch Laboratories. For NMJ analyses, <br /> and F2 screens. This &#226;&#8364;&#732;&#226;&#8364;&#732;combination screen&#226;&#8364;&#8482;&#226;&#8364;&#8482; identical               3rd instar larvae dissected fixed 5 minutes Bouin&#226;&#8364;&#8482;s <br />  <br />  <br />       PLoS ONE | www.plosone.org                                        16                         September 2008 | Volume 3 | Issue 9 | e3209 <br />                                                                                                                                                 A Drosophila Model SMA <br />  <br />  <br /> fixative. Imaginal disc dissections performed 3rd instar                            Found at: doi:10.1371/journal.pone.0003209.s003 (0.21 MB TIF) <br /> larvae phosphate-buffered saline (PBS). Discs kept ice <br />                                                                                             Figure S4 pMAD staining vestigalGAL4, UAS-Smn- <br /> until fixation 3% paraformaldehyde PBS. Stained specimens <br />                                                                                             RNAi transgenic animals. (A&#226;&#8364;&#8220;B) 3rd instar wing discs of <br /> were mounted FluoroGuard Antifade Reagent (Bio-Rad), and <br />                                                                                             vestigalGAL4, pWIZ[UAS-Smn-RNAi]N4 animals stained with <br /> images obtained Zeiss LSM510 confocal microscope. <br />                                                                                             antibodies SMN (red) (A) pMAD (green) (B). pMAD <br /> Bouton numbers counted based Discs large and <br /> Synaptotagmin staining A2 segment muscles 6                              staining reduced dorsoventral boundary wing disc <br /> 7. The ratio muscle area various genotypes                                   SMN expression decreased (see Figure 10 wild-type <br /> normalized wild-type. GluRIIA morphometric analyses                                 control). <br /> performed described previously [66].                                                     Found at: doi:10.1371/journal.pone.0003209.s004 (1.37 MB TIF) <br />                                                                                             Figure S5    NMJ analysis Smn enhancers. Modification <br /> Supporting Information                                                                      NMJ morphology muscles 6 7 A2 <br />                                                                                             segment assayed elavGAL4 pWIZ[UAS-Smn-RNAi]C24 <br /> Figure S1 Specificity anti-SMN antibodies. (A&#226;&#8364;&#8220;C) <br />                                                                                             background trans identified modifiers using pre- <br /> Wing discs 3rd instar larvae overexpressing UAS-FLAG- <br />                                                                                             synaptic (Horseradish peroxidase (HRP)) post-synaptic <br /> Smn transgenic rescue construct using vestigalGAL driver were <br />                                                                                             (GluRIIA) markers (see Materials Methods). In three <br /> stained antibodies FLAG peptide (green) (A) and <br /> SMN (red) (B). (C) Merge (A) (B) showing overlapping                             cases (f04448, d09801 d00698) did significant <br /> expression SMN FLAG vestigal expression                                   phenotypic alteration, pWIZ[UAS-Smn-RNAi]N13 allele was <br /> domain. (D) Wild-type (E) vestigalGAL4, pWIZ[UAS-Smn-                                   used. In background, strain f04448 d09801 <br /> RNAi]N4 3rd instar wing discs stained antibodies                          enhanced, d00698 showed interaction (data not <br /> SMN (green). (F) Western blots serial dilution S2 cell                              shown Figure 7). <br /> extracts (1: 20 mg, 2: 40 mg, 3: 60 mg, 4: 80 mg total protein) using                       Found at: doi:10.1371/journal.pone.0003209.s005 (0.74 MB TIF) <br /> the polyclonal (left) monoclonal (right) antiserum SMN                          Figure S6 NMJ analysis Smn suppressors. Modification <br /> recognize single band approximately 28 kD size.                                     NMJ morphology muscles 6 7 A2 <br /> Found at: doi:10.1371/journal.pone.0003209.s001 (4.46 MB TIF)                               segment assayed elavGAL4 pWIZ[UAS-Smn-RNAi]C24 <br /> Figure S2 SMN post-synaptic staining abolished                                        background trans identified modifiers using pre- <br /> muscle specific SMN knockdown. (A&#226;&#8364;&#8220;F) The morphology                                      synaptic (Horseradish peroxidase (HRP)) post-synaptic <br /> the NMJ muscles 6 7 A2 segment observed                              (GluRIIA) markers (see Materials Methods). <br /> in different genetic backgrounds using antibodies SMN                               Found at: doi:10.1371/journal.pone.0003209.s006 (0.78 MB TIF) <br /> (green) post-synaptic marker, Discs large (red). (A&#226;&#8364;&#8220;C) <br /> Wild-type: anti-DLG (A), anti-SMN (B) (C) merge (A)                              Acknowledgments <br /> (B). (D&#226;&#8364;&#8220;F) Transgenic animals containing how24BGAL4 and <br />                                                                                             We wish thank April Duckworth, Melanie Smith, Waineshet Tefera <br /> pWIZ[UAS-Smn-RNAi]N4: anti-DLG (D), anti-SMN (E) (F)                                    technical supports. We want thank Tom Barsby, Jevede Harris, <br /> merge (D) (E). In background, SMN staining reduced                           John Satterlee, James Sleigh, Amy Walker discussion sharing <br /> (E).                                                                                        unpublished results. We wish thank Margaret L. Winberg of <br /> Found at: doi:10.1371/journal.pone.0003209.s002 (2.42 MB TIF)                               SMAF, Robert Lake, Pallavi Kshetrapal, Cecilia Lu Catherine <br />                                                                                             Dubreuil critically editing manuscript. <br /> Figure S3     Pre-synaptic ghost bouton counts elevat- <br /> ed Smn animals. The morphology NMJ between <br /> muscles 6 7 A2 segment observed different Smn                             Author Contributions <br /> backgrounds using pre-synaptic (Synaptotagmin) post-                                Conceived designed experiments: HCHC DND TY MWK DVV <br /> synaptic (Discs large) markers. Ghost bouton counts                                    SAT. Performed experiments: HCHC DND TY VS. Analyzed the <br /> determined assessing numbers boutons stained                                 data: HCHC DND TY VS DVV. Contributed reagents/materials/ <br /> positive Synaptotagmin failed stain Discs large. All                         analysis tools: HCHC AM. Wrote paper: HCHC DND MWK SAT. <br /> combinations examined (Smn73Ao/Smnf01109, Smnf05960/Smnf01109                               Read revised paper: ACH DVV. Provided unpublished results for <br />                                                                                             discussion: ACH. Intellectual contribution: TAF AS. <br /> and Smnf01109/Smnf01109) displayed elevated numbers pre- <br /> synaptic ghost boutons compared wild-type. <br />  <br /> References <br />  1. Monani UR (2005) Spinal muscular atrophy: deficiency ubiquitous                   8. Monani UR, Lorson CL, Parsons DW, Prior TW, Androphy EJ, et al. (1999) A <br />     protein; motor neuron-specific disease. Neuron 48: 885&#226;&#8364;&#8220;896.                               single nucleotide difference alters splicing patterns distinguishes SMA <br />  2. Frugier T, Nicole S, Cifuentes-Diaz C, Melki J (2002) The molecular bases                gene SMN1 copy gene SMN2. Hum Mol Genet 8: 1177&#226;&#8364;&#8220;1183. <br />     spinal muscular atrophy. Curr Opin Genet Dev 12: 294&#226;&#8364;&#8220;298.                                9. Liu Q, Dreyfuss G (1996) A novel nuclear structure containing survival of <br />  3. Wirth B (2000) An update mutation spectrum survival motor                     motor neurons protein. Embo J 15: 3555&#226;&#8364;&#8220;3565. <br />     neuron gene (SMN1) autosomal recessive spinal muscular atrophy (SMA).                10. Massenet S, Pellizzoni L, Paushkin S, Mattaj IW, Dreyfuss G (2002) The SMN <br />     Hum Mutat 15: 228&#226;&#8364;&#8220;237.                                                                      complex associated snRNPs cytoplasmic assembly <br />  4. Crawford TO, Pardo CA (1996) The neurobiology childhood spinal muscular                  pathway. Mol Cell Biol 22: 6533&#226;&#8364;&#8220;6541. <br />     atrophy. Neurobiol Dis 3: 97&#226;&#8364;&#8220;110.                                                       11. Wan L, Battle DJ, Yong J, Gubitz AK, Kolb SJ, et al. (2005) The survival of <br />  5. Lefebvre S, Burglen L, Reboullet S, Clermont O, Burlet P, et al. (1995)                     motor neurons protein determines capacity snRNP assembly: <br />     Identification characterization spinal muscular atrophy-determining                biochemical deficiency spinal muscular atrophy. Mol Cell Biol 25: <br />     gene. Cell 80: 155&#226;&#8364;&#8220;165.                                                                     5543&#226;&#8364;&#8220;5551. <br />  6. Lefebvre S, Burlet P, Liu Q, Bertrandy S, Clermont O, et al. (1997) Correlation         12. Paushkin S, Gubitz AK, Massenet S, Dreyfuss G (2002) The SMN complex, an <br />     severity SMN protein level spinal muscular atrophy. Nat Genet                assemblyosome ribonucleoproteins. Curr Opin Cell Biol 14: 305&#226;&#8364;&#8220;312. <br />     16: 265&#226;&#8364;&#8220;269.                                                                            13. Eggert C, Chari A, Laggerbauer B, Fischer U (2006) Spinal muscular atrophy: <br />  7. Lorson CL, Hahnen E, Androphy EJ, Wirth B (1999) A single nucleotide                 RNP connection. Trends Mol Med 12: 113&#226;&#8364;&#8220;121. <br />     SMN gene regulates splicing responsible spinal muscular atrophy. Proc        14. Meister G, Eggert C, Fischer U (2002) SMN-mediated assembly RNPs: a <br />     Natl Acad Sci U S A 96: 6307&#226;&#8364;&#8220;6311.                                                          complex story. Trends Cell Biol 12: 472&#226;&#8364;&#8220;478. <br />  <br />  <br />  <br />         PLoS ONE | www.plosone.org                                                     17                             September 2008 | Volume 3 | Issue 9 | e3209 <br />                                                                                                                                                   A Drosophila Model SMA <br />  <br />  <br /> 15. Chan YB, Miguel-Aliaga I, Franks C, Thomas N, Trulzsch B, et al. (2003)                   41. Elias GM, Nicoll RA (2007) Synaptic trafficking glutamate receptors by <br />     Neuromuscular defects Drosophila survival motor neuron gene mutant.                      MAGUK scaffolding proteins. Trends Cell Biol 17: 343&#226;&#8364;&#8220;352. <br />     Hum Mol Genet 12: 1367&#226;&#8364;&#8220;1376.                                                              42. Lahey T, Gorczyca M, Jia XX, Budnik V (1994) The Drosophila tumor <br /> 16. McWhorter ML, Monani UR, Burghes AH, Beattie CE (2003) Knockdown                           suppressor gene dlg required normal synaptic bouton structure. Neuron 13: <br />     survival motor neuron (Smn) protein zebrafish causes defects motor                  823&#226;&#8364;&#8220;835. <br />     axon outgrowth pathfinding. J Cell Biol 162: 919&#226;&#8364;&#8220;931.                                 43. Inoue H, Imamura T, Ishidou Y, Takase M, Udagawa Y, et al. (1998) Interplay <br /> 17. Murray LM, Comley LH, Thomson D, Parkinson N, Talbot K, et al. (2008)                         signal mediators decapentaplegic (Dpp): molecular characterization of <br />     Selective vulnerability motor neurons dissociation pre- post-                   mothers dpp, Medea, daughters dpp. Mol Biol Cell 9: <br />     synaptic pathology neuromuscular junction mouse models spinal                    2145&#226;&#8364;&#8220;2156. <br />     muscular atrophy. Hum Mol Genet 17: 949&#226;&#8364;&#8220;962.                                              44. Patterson GI, Padgett RW (2000) TGF beta-related pathways. Roles in <br /> 18. Rajendra TK, Gonsalvez GB, Walker MP, Shpargel KB, Salz HK, et al. (2007)                     Caenorhabditis elegans development. Trends Genet 16: 27&#226;&#8364;&#8220;33. <br />     A Drosophila melanogaster model spinal muscular atrophy reveals function             45. McCabe BD, Marques G, Haghighi AP, Fetter RD, Crotty ML, et al. (2003) <br />     SMN striated muscle. J Cell Biol 176: 831&#226;&#8364;&#8220;841.                                         The BMP homolog Gbb provides retrograde signal regulates synaptic <br /> 19. Oprea GE, Krober S, McWhorter ML, Rossoll W, Muller S, et al. (2008) Plastin                  growth Drosophila neuromuscular junction. Neuron 39: 241&#226;&#8364;&#8220;254. <br />     3 protective modifier autosomal recessive spinal muscular atrophy.                46. Tanimoto H, Itoh S, Dijke P, Tabata T (2000) Hedgehog creates gradient <br />     Science 320: 524&#226;&#8364;&#8220;527.                                                                         DPP activity Drosophila wing imaginal discs. Mol Cell 5: 59&#226;&#8364;&#8220;71. <br /> 20. Rossoll W, Jablonka S, Andreassi C, Kroning AK, Karle K, et al. (2003) Smn,               47. Haerry TE, Khalsa O, O&#226;&#8364;&#8482;Connor MB, Wharton KA (1998) Synergistic signaling <br />     spinal muscular atrophy-determining gene product, modulates axon growth                   BMP ligands SAX TKV receptors controls wing growth <br />     localization beta-actin mRNA growth cones motoneurons. J Cell                    patterning Drosophila. Development 125: 3977&#226;&#8364;&#8220;3987. <br />     Biol 163: 801&#226;&#8364;&#8220;812.                                                                        48. Khalsa O, Yoon JW, Torres-Schumann S, Wharton KA (1998) TGF-beta/BMP <br /> 21. McAndrew PE, Parsons DW, Simard LR, Rochette C, Ray PN, et al. (1997)                         superfamily members, Gbb-60A Dpp, cooperate provide pattern <br />     Identification proximal spinal muscular atrophy carriers patients                   information establish cell identity Drosophila wing. Development <br />     analysis SMNT SMNC gene copy number. Am J Hum Genet 60:                                125: 2723&#226;&#8364;&#8220;2734. <br />     1411&#226;&#8364;&#8220;1422.                                                                                49. Sweeney ST, Davis GW (2002) Unrestricted synaptic growth spinster-a late <br /> 22. Feldkotter M, Schwarzer V, Wirth R, Wienker TF, Wirth B (2002) Quantitative                   endosomal protein implicated TGF-beta-mediated synaptic growth regula- <br />     analyses SMN1 SMN2 based real-time lightCycler PCR: fast                        tion. Neuron 36: 403&#226;&#8364;&#8220;416. <br />     highly reliable carrier testing prediction severity spinal muscular             50. Tsuneizumi K, Nakayama T, Kamoshida Y, Kornberg TB, Christian JL, et al. <br />     atrophy. Am J Hum Genet 70: 358&#226;&#8364;&#8220;368.                                                          (1997) Daughters dpp modulates dpp organizing activity Drosophila <br /> 23. Miguel-Aliaga I, Chan YB, Davies KE, van den Heuvel M (2000) Disruption                    wing development. Nature 389: 627&#226;&#8364;&#8220;631. <br />     SMN function ectopic expression human SMN gene Drosophila.                   51. Battle DJ, Kasim M, Yong J, Lotti F, Lau CK, et al. (2006) The SMN complex: <br />     FEBS Lett 486: 99&#226;&#8364;&#8220;102.                                                                        assembly machine RNPs. Cold Spring Harb Symp Quant Biol 71: <br /> 24. Kankel MW, Hurlbut GD, Upadhyay G, Yajnik V, Yedvobnick B, et al. (2007)                      313&#226;&#8364;&#8220;320. <br />     Investigating genetic circuitry mastermind Drosophila, notch signal           52. Zhang Z, Lotti F, Dittmar K, Younis I, Wan L, et al. (2008) SMN deficiency <br />     effector. Genetics 177: 2493&#226;&#8364;&#8220;2505.                                                            causes tissue-specific perturbations repertoire snRNAs widespread <br /> 25. Parks AL, Cook KR, Belvin M, Dompe NA, Fawcett R, et al. (2004) Systematic                    defects splicing. Cell 133: 585&#226;&#8364;&#8220;600. <br />     generation high-resolution deletion coverage Drosophila melanogaster            53. Gabanella F, Butchbach ME, Saieva L, Carissimi C, Burghes AH, et al. (2007) <br />     genome. Nat Genet 36: 288&#226;&#8364;&#8220;292.                                                                Ribonucleoprotein assembly defects correlate spinal muscular atrophy <br /> 26. Thibault ST, Singer MA, Miyazaki WY, Milash B, Dompe NA, et al. (2004) A                      severity preferentially affect subset spliceosomal snRNPs. PLoS ONE 2: <br />     complementary transposon tool kit Drosophila melanogaster using P                     e921. <br />     piggyBac. Nat Genet 36: 283&#226;&#8364;&#8220;287.                                                          54. Kariya S, Park GH, Maeno-Hikichi Y, Leykekhman O, Lutz C, et al. (2008) <br /> 27. Aberle H, Haghighi AP, Fetter RD, McCabe BD, Magalhaes TR, et al. (2002)                      Reduced SMN protein impairs maturation neuromuscular junctions in <br />     wishful thinking encodes BMP type II receptor regulates synaptic growth                mouse models spinal muscular atrophy. Hum Mol Genet. <br />     Drosophila. Neuron 33: 545&#226;&#8364;&#8220;558.                                                        55. Lee S, Sayin A, Grice S, Burdett H, Baban D, et al. (2008) Genome-wide <br /> 28. Marques G, Bao H, Haerry TE, Shimell MJ, Duchek P, et al. (2002) The                          expression analysis spinal muscular atrophy model: discovery of <br />     Drosophila BMP type II receptor Wishful Thinking regulates neuromuscular                      new drug targets. PLoS ONE 3: e1404. <br />     synapse morphology function. Neuron 33: 529&#226;&#8364;&#8220;543.                                      56. Gavrilina TO, McGovern VL, Workman E, Crawford TO, Gogliotti RG, et al. <br /> 29. Zito K, Fetter RD, Goodman CS, Isacoff EY (1997) Synaptic clustering                       (2008) Neuronal SMN expression corrects spinal muscular atrophy severe <br />     Fascilin II Shaker: essential targeting sequences role Dlg. Neuron 19:             SMA mice muscle-specific SMN expression phenotypic effect. Hum <br />     1007&#226;&#8364;&#8220;1016.                                                                                    Mol Genet 17: 1063&#226;&#8364;&#8220;1075. <br /> 30. Liu JL, Murphy C, Buszczak M, Clatterbuck S, Goodman R, et al. (2006) The                 57. Hsieh-Li HM, Chang JG, Jong YJ, Wu MH, Wang NM, et al. (2000) A mouse <br />     Drosophila melanogaster Cajal body. J Cell Biol 172: 875&#226;&#8364;&#8220;884.                                 model spinal muscular atrophy. Nat Genet 24: 66&#226;&#8364;&#8220;70. <br /> 31. Littleton JT, Bellen HJ, Perin MS (1993) Expression synaptotagmin                   58. Jablonka S, Karle K, Sandner B, Andreassi C, von Au K, et al. (2006) Distinct <br />     Drosophila reveals transport localization synaptic vesicles synapse.            overlapping alterations motor sensory neurons mouse model of <br />     Development 118: 1077&#226;&#8364;&#8220;1088.                                                                   spinal muscular atrophy. Hum Mol Genet 15: 511&#226;&#8364;&#8220;518. <br /> 32. Littleton JT, Stern M, Schulze K, Perin M, Bellen HJ (1993) Mutational analysis           59. Monani UR, Sendtner M, Coovert DD, Parsons DW, Andreassi C, et al. (2000) <br />     Drosophila synaptotagmin demonstrates essential role Ca(2+)-activated               The human centromeric survival motor neuron gene (SMN2) rescues embryonic <br />     neurotransmitter release. Cell 74: 1125&#226;&#8364;&#8220;1134.                                                 lethality Smn(2/2) mice results mouse spinal muscular <br /> 33. Ataman B, Ashley J, Gorczyca D, Gorczyca M, Mathew D, et al. (2006) Nuclear                   atrophy. Hum Mol Genet 9: 333&#226;&#8364;&#8220;339. <br />     trafficking Drosophila Frizzled-2 synapse development requires              60. Avila AM, Burnett BG, Taye AA, Gabanella F, Knight MA, et al. (2007) <br />     PDZ protein dGRIP. Proc Natl Acad Sci U S A 103: 7841&#226;&#8364;&#8220;7846.                                   Trichostatin A increases SMN expression survival mouse model of <br /> 34. Eaton BA, Davis GW (2005) LIM Kinase1 controls synaptic stability                             spinal muscular atrophy. J Clin Invest 117: 659&#226;&#8364;&#8220;671. <br />     downstream type II BMP receptor. Neuron 47: 695&#226;&#8364;&#8220;708.                               61. Hua Y, Vickers TA, Okunola HL, Bennett CF, Krainer AR (2008) Antisense <br /> 35. Robinow S, White K (1988) The locus elav Drosophila melanogaster                        masking hnRNP A1/A2 intronic splicing silencer corrects SMN2 splicing <br />     expressed neurons developmental stages. Dev Biol 126: 294&#226;&#8364;&#8220;303.                      transgenic mice. Am J Hum Genet 82: 834&#226;&#8364;&#8220;848. <br /> 36. Brand AH, Perrimon N (1993) Targeted gene expression means altering               <br /> </body></html>