<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br />                            DNA damage-induced alterations in chromatin contribute <span id='am-6' about='protege:TO' typeof='owl:Thing'>to</span> <br />                            genomic integrity and age-related changes in gene expression <br />  <br />                            <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />                                 Genomic instability and alterations in gene expression are hallmarks of eukaryotic aging. The <br />                                 yeast histone deacetylase Sir2 silences transcription and stabilizes repetitive DNA, but during <br />                                 aging or in response to a DNA break, the Sir complex relocalizes to sites of genomic instability, <br />                                 resulting in the desilencing of genes that cause sterility, a characteristic of yeast aging. Using <br />                                 embryonic stem cells, we show that mammalian Sir2, SIRT1, represses repetitive DNA and a <br />                                 functionally diverse set of genes across the mouse genome. In response to DNA damage, SIRT1 <br />                                 dissociates <span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span> these loci and relocalizes to DNA breaks to promote repair, resulting in <br />                                 transcriptional changes that parallel those in the aging mouse brain. Increased SIRT1 expression <br />                                 promotes <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span>survival</span> in a mouse model of genomic instability and suppresses age-dependent <br />                                 transcriptional changes. Thus, DNA damage-induced redistribution of SIRT1 and other chromatin <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                 modifying proteins may be a conserved mechanism of aging in eukaryotes. <br />  <br />  <br />                            Keywords <br />                                 epigenetics; sirtuin; chromatin; histone; DNA repair <br />  <br />  <br />  <br />  <br />                            &#194;&#169; 2008 Elsevier Inc. All rights reserved. <br />                            * <br />                              Contact: David_Sinclair@hms.harvard.edu, Phone: 617-432-3931, Fax 617-432-6225. <br />                            7Present address: Max-Planck-Institute for Experimental Medicine, 37075 G&#195;&#182;ttingen, Germany <br />                            Publisher&apos;s Disclaimer: This is a PDF file of an unedited manuscript that has been accepted for publication. As a service to our <br />                            customers we are providing this early version of the manuscript. The manuscript will undergo copyediting, typesetting, and review of <br />                            the resulting proof before it is published in its final citable form. Please note that during the production process errors may be <br />                            discovered which could affect the content, and all legal disclaimers that apply to the journal pertain. <br />                             Oberdoerffer et al.                                                                                               Page 2 <br />  <br />  <br />                               Introduction <br />                                                  Chromosomes are arguably the most difficult structure for an organism to maintain over a <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  lifetime. Chromosomes break, mutations accumulate, and youthful gene expression patterns <br />                                                  are progressively lost. Some changes in gene expression have been interpreted as beneficial <br />                                                  responses to cellular damage (Narita et al., 2006; Niedernhofer et al., 2006). However, there <br />                                                  are numerous stochastic changes in gene expression that have no apparent long-term benefit <br />                                                  to the organism and may in fact be detrimental (Bahar et al., 2006). The &quot;Heterochromatin <br />                                                  Island Hypothesis&quot; and related hypotheses propose that alterations in chromatin and the <br />                                                  resulting gene expression changes can drive the aging process, but evidence is lacking <br />                                                  (Cutler, 1995; Imai and Kitano, 1998; Vijg, 2004; Villeponteau, 1997). <br />  <br />                                                  Potential clues about the relationship between epigenetic changes and aging come from <br />                                                  studies in Saccharomyces cerevisiae, where epigenetic changes are a primary cause of the <br />                                                  aged <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span>phenotype</span>. The SIR2 gene, encoding a <span id='am-2' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span>-responsive NAD+-dependent histone <br />                                                  deacetylase, has emerged as a key regulator of health and lifespan in yeast and other <br />                                                  organisms (Haigis and Guarente, 2006). Two major functions of Sir2 are to repress gene <br />                                                  expression at the silent mating type loci HML and HMR (Klar et al., 1979; Rine et al., 1979) <br />                                                  and to suppress recombination at the ribosomal DNA (rDNA) locus, which gives rise to <br />                                                  toxic rDNA circles (ERCs) (Sinclair and Guarente, 1997). As yeast cells age, the Sir protein <br />                                                  complex dissociates from HM loci and moves to the nucleolus in response to ERC <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  accumulation causing sterility, a hallmark of yeast aging (Kennedy et al., 1997; Sinclair and <br />                                                  Guarente, 1997; Smeal et al., 1996). Thus, a redistribution of chromatin modifying factors <br />                                                  and the resulting changes in transcription are the cause of primary yeast aging phenotypes. <br />  <br />                                                  Aging is not the only stimulus that causes yeast Sir proteins to relocalize. DNA breakage <br />                                                  causes Sir proteins to dissociate from HM loci and relocate to DNA breaks, in a DNA <br />                                                  damage checkpoint dependent manner (Martin et al., 1999; McAinsh et al., 1999; Mills et <br />                                                  al., 1999). The effect of relocalization appears to be two fold: (i) expression of HM genes <br />                                                  promotes DNA repair and (ii) Sir proteins directly modify chromatin surrounding the break <br />                                                  site, possibly to facilitate DNA repair (Lee et al., 1999; Tamburini and Tyler, 2005). <br />  <br />                                                  There is some evidence that related processes occur in mammals. First, cells damaged by <br />                                                  oxidative stress in vitro undergo stochastic transcriptional changes that parallel those in aged <br />                                                  heart tissue (Bahar et al., 2006). Second, a deficiency in the DNA repair factor ERCC1 <br />                                                  accelerates aging phenotypes and generates gene expression profiles reminiscent of aged <br />                                                  animals (Niedernhofer et al., 2006). Third, cells that senesce due to replicative aging in vitro <br />                                                  or in aged tissues in vivo exhibit alterations in heterochromatin (Herbig et al., 2006; Narita et <br />                                                  al., 2006) and secrete growth factors that can drive tumorigenesis (Campisi, 2005). Finally, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  oxidative DNA damage at promoters correlates with gene repression in the aging human <br />                                                  brain (Lu et al., 2004) and has been linked to both transcriptional and epigenetic changes <br />                                                  that may contribute to Alzheimer disease (Wu et al., 2008). <br />  <br />                                                  To date, no study has tested whether Sir2-mediated alterations in chromatin contribute to <br />                                                  aging in mammals. Several observations, however, are consistent with this possibility. The <br />                                                  mammalian ortholog of Sir2, SIRT1, regulates both the expression of individual genes <br />                                                  (Picard et al., 2004; Pruitt et al., 2006; Vaquero et al., 2004) and the formation of facultative <br />                                                  heterochromatin (Vaquero et al., 2007). SIRT1 has also been linked to the DNA damage <br />                                                  response via regulation of p53 (Luo et al., 2001; Vaziri et al., 2001) and its interaction with <br />                                                  Nbs1, a component of the DNA damage sensor complex MRN (Mre11-Rad50-Nbs1) (Yuan <br />                                                  et al., 2007). Furthermore, SIRT1 has recently been implicated in the regulation of DNA <br />                                                  methylation patterns at damaged CpG-rich DNA (O&apos;Hagan et al., 2008). Deletion of another <br />                                                  Sir2 homolog, SIRT6, reduces <span id='am-241' about='xsp:base' typeof='owl:Thing'>base</span> excision DNA repair and causes an accelerated aging <br />  <br />  <br />                                                           Cell. Author manuscript; available in PMC 2010 April 13. <br />                             Oberdoerffer et al.                                                                                             Page 3 <br />  <br />  <br />                                                  phenotype in mice (Mostoslavsky et al., 2006). In this study, we map the interaction <br />                                                  between SIRT1 and the mouse genome and identify an evolutionarily conserved DNA <br />                                                  damage response that may drive changes in gene expression during aging. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Results <br />                               Loss of Sir2 and SIRT1-dependent silencing in response to oxidative stress <br />                                                  Our previous studies on the relocalization of yeast Sir2 utilized highly artificial means to <br />                                                  induce DNA damage, such as EcoRI and the yeast HO endonuclease (Mills et al., 1999). To <br />                                                  test whether a stress more relevant to aging results in desilencing of mating-type loci, a yeast <br />                                                  strain carrying a GFP reporter at the HMR locus (HMR::GFP) was exposed to oxidative <br />                                                  stress (i.e. H2O2). There was a tight correlation between H2O2 levels and HMR derepression <br />                                                  (Figure 1A and Figure S1A). <br />  <br />                                                  An additional copy of SIR2 (2xSIR2) extends replicative lifespan (Kaeberlein et al., 1999), <br />                                                  but its effect on oxidative stress resistance and loss of silencing is unknown. When treated <br />                                                  with H2O2, strains with increased Sir2 levels maintained HMR silencing and had more stable <br />                                                  rDNA compared to wild type cells (Figures 1A and B). At non-lethal concentrations of <br />                                                  H2O2, the 2xSIR2 strain had a significantly greater replicative lifespan (Figure 1C and <br />                                                  Figure S1B). Thus, additional SIR2 suppresses the toxicity, genomic instability, and <br />                                                  desilencing caused by genotoxic stress. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  We next sought to examine whether these findings are relevant to mammals. We first tested <br />                                                  if SIRT1 associates with highly repetitive DNA such as pericentromeric major satellite <br />                                                  repeats in mouse ES cells, a cell type that has been previously used to study satellite repeat <br />                                                  silencing (Kanellopoulou et al., 2005). Histone H1 acetylation on lysine 26 (H1AcK26) <br />                                                  served as a read-out for SIRT1 deacetylase activity (Vaquero et al., 2004). Using chromatin <br />                                                  immunoprecipitation (ChIP) and quantitative PCR (q-PCR), we detected an association <br />                                                  between SIRT1 and major satellite repeats that was disrupted by the pansirtuin inhibitor <br />                                                  nicotinamide (NAM) (Figure 1D and 1E). This coincided with an increase in repeat <br />                                                  transcripts (Figure 1F) and H1AcK26 (Figure 1E). Consistent with known redundancy <br />                                                  among histone deacetylases (Zhu et al., 2004), trichostatin A (TSA), NAM and sirtinol, pan- <br />                                                  class I/II and pan-class III (sirtuin) inhibitors, respectively, caused desilencing of major <br />                                                  satellite repeats (Figure 1F and Figures S1D and E, (Kanellopoulou et al., 2005)), whereas <br />                                                  no significant increase in repeat transcripts was seen after stable knock-down of SIRT1 <br />                                                  (Figure S1C and data not shown). Ongoing work is aimed at identifying the role of other <br />                                                  HDACs that may contribute to satellite repeat silencing. <br />  <br />                                                  In accordance with our observations in yeast, treatment of cells with non-cytotoxic levels of <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  H2O2 greatly decreased the amount of SIRT1 bound to repeats, coinciding with an increase <br />                                                  in H1AcK26 (Figure 1E, Figure S2 and S3). Similar to the effect of NAM and TSA, <br />                                                  oxidative stress increased the transcription of these loci (Figure 1F), an effect that was <br />                                                  counteracted by overexpression of SIRT1 (Figure 1G). Together, these data indicate that <br />                                                  SIRT1 binds and can contribute to the silencing of major satellite repeats. <br />  <br />                               Global changes in promoter-associated SIRT1 in response to oxidative stress <br />                                                  Given the role of yeast Sir2 in silencing HM loci, we hypothesized that mammalian SIRT1 <br />                                                  might regulate a number of protein-coding genes. To test this, we used ChIP in combination <br />                                                  with a genome-wide promoter tiling array (ChIP on chip) to identify putative SIRT1 target <br />                                                  genes. Immunoprecipitated DNA from either untreated or H2O2-treated cells was hybridized <br />                                                  to a NimbleGen MM5 array to detect SIRT1- and H1AcK26-associated promoter segments <br />                                                  (Figure 2A). Based on gene ontology (GO), overrepresented groups included genes for <br />                                                  chromatin assembly and transcriptional repression (e.g. methyl-CpG binding proteins 2, -3 <br />  <br />                                                           Cell. Author manuscript; available in PMC 2010 April 13. <br />                             Oberdoerffer et al.                                                                                           Page 4 <br />  <br />  <br />                                                  and -4, SIRT7, and several histone genes), ubiquitin-regulated protein degradation, and cell <br />                                                  cycle regulation (Figure 2B, S4A and Table S1). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  Next, we determined if SIRT1-binding and H1 acetylation patterns change in response to <br />                                                  oxidative stress. Paralleling the yeast response, oxidative stress caused a redistribution of <br />                                                  SIRT1 at the chromatin level (Figure 2A), such that less than 10% of SIRT1-associated <br />                                                  promoters overlapped between untreated and H2O2-treated cells (Figure S4B), and the <br />                                                  resulting binding <span id='am-242' about='xsp:pattern' typeof='owl:Thing'>pattern</span> did not cluster into functional GO groups, indicating a shift to <br />                                                  random SIRT1 distribution (Figure 2B). There was a significant negative correlation <br />                                                  between the loss of SIRT1 binding and H1K26 acetylation (&#207;&#8225;2=12.12, P&lt;0.001, Figure 2C), <br />                                                  supporting the notion that SIRT1 regulates these genes, at least in part, through H1 <br />                                                  deacetylation. Together, these observations indicate that SIRT1 associates with considerably <br />                                                  more genes than currently known and that oxidative stress causes a major change in SIRT1 <br />                                                  distribution across the genome. <br />  <br />                               Oxidative stress causes SIRT1 dependent transcriptional deregulation <br />                                                  To determine how the redistribution of promoter-associated SIRT1 affects transcription, we <br />                                                  performed a combination of microarray-based transcriptional profiling (Table S2), and q- <br />                                                  RT-PCR analysis of SIRT1-associated and non-associated genes, comparing standard <br />                                                  growth conditions with H2O2 treatment, with and without SIRT1 overexpression. We <br />                                                  identified a diverse set of SIRT1-associated genes that significantly increased in expression <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  upon H2O2 treatment, coincident with SIRT1 release, including regulators of metabolism <br />                                                  (pisd, prkag3, gstz1), ion transport (slc9a9), cell motility (tcte3) and G-protein signaling <br />                                                  (farp2). Of these genes, 75% were repressed by modest overexpression of SIRT1 (Figure 3) <br />                                                  and H1K26 acetylation inversely correlated with SIRT1 binding (Figure S5A). <br />                                                  Demonstrating specificity, SIRT1 overexpression did not repress the induction of genes that <br />                                                  were not associated with SIRT1 (Figure 3B). The reduction in SIRT1 binding was <br />                                                  confirmed for a random selection of promoters by q-PCR using two distinct types of <br />                                                  genotoxic stress: H2O2 and methyl-methane-sulfonate (MMS) (Figure S5B). Inactivation of <br />                                                  SIRT1 resulted in transcriptional deregulation of six of the eight SIRT1-associated loci, <br />                                                  further corroborating a regulatory role for SIRT1 at these sites (Figure S5C). SIRT1 was <br />                                                  recently shown to negatively regulate HDAC1 and can, thus, function both as transcriptional <br />                                                  activator and repressor (Binda et al., 2008), which may explain why knockdown of SIRT1 <br />                                                  does not always alter gene expression in the same direction as oxidative stress. Together, our <br />                                                  results indicate that DNA damage induces a change in SIRT1 distribution that affects the <br />                                                  expression of individual genes, and that this effect can be suppressed by increasing SIRT1 <br />                                                  levels. <br />  <br />                               SIRT1 is recruited to DNA double strand breaks <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  Given that the yeast Sir complex redistributes from silent loci to sites of DNA repair (Martin <br />                                                  et al., 1999; Mills et al., 1999), we wondered if SIRT1 also behaves in this way. When <br />                                                  isolating chromatin-bound and non-chromatin-bound protein fractions from untreated or <br />                                                  H2O2-treated cells, we observed a substantial, transient, and dose-dependent increase in <br />                                                  chromatin-associated SIRT1. Treatment with MMS produced a similar effect, supporting the <br />                                                  idea that SIRT1 is recruited to damaged DNA from promoters and from the soluble nuclear <br />                                                  pool (Figure 4A, Figures S6A and S6B). <br />  <br />                                                  In yeast, the recruitment of Sir proteins to a DNA double-strand break (DSB) requires DNA <br />                                                  damage signaling through MEC1, an ortholog of the mammalian PI3-kinases ATR/ATM <br />                                                  (Martin et al., 1999; McAinsh et al., 1999; Mills et al., 1999). To test whether DNA damage <br />                                                  signaling is required for SIRT1 redistribution, cells were treated with the PI3 kinase <br />                                                  inhibitor wortmannin or the ATM inhibitor KU55933 prior to H2O2 exposure. The increase <br />  <br />  <br />                                                           Cell. Author manuscript; available in PMC 2010 April 13. <br />                             Oberdoerffer et al.                                                                                          Page 5 <br />  <br />  <br />                                                  in chromatin-associated SIRT1 was strongly reduced by both compounds (Figure 4B). We <br />                                                  then investigated whether SIRT1 recruitment is dependent on histone H2AX, one of the <br />                                                  immediate targets of ATM (Burma et al., 2001). In H2AX-deficient ES cells there was a 2&#226;&#8364;&#8220;3 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  fold reduction in the amount of chromatin-bound SIRT1 in response to H2O2 or MMS <br />                                                  compared to cells reconstituted with wild-type H2AX (Figure 4C and Figure S6C, (Xie et <br />                                                  al., 2004)). Thus, efficient recruitment of SIRT1 to damaged DNA requires DNA damage <br />                                                  signaling through ATM and H2AX. <br />  <br />                                                  Rad51, a critical component of the homologous DSB repair (HR) process, was recruited to <br />                                                  chromatin concomitant with SIRT1 in response to both H2O2 and exposure to DSB-inducing <br />                                                  ionizing irradiation (IR). Importantly, Rad51 recruitment was impaired in the absence of <br />                                                  SIRT1 (Figure 4D&#226;&#8364;&#8220;E). These data indicated that SIRT1 might physically associate with <br />                                                  DSBs and perform a key step in the DNA repair process. <br />  <br />                                                  To test this, we employed a cell-based system in which a DSB is induced by transient <br />                                                  transfection with a vector encoding the endonuclease I-SceI (Weinstock et al., 2006). This <br />                                                  system has been used previously to identify the DSB-binding patterns of a number of DNA <br />                                                  repair enzymes (Rodrigue et al., 2006). Nbs1, a component of the MRN complex served a <br />                                                  positive control for binding (Berkovich et al., 2007). Concomitant with recruitment of Nbs1, <br />                                                  SIRT1 binding was detected 24 h after transfection with I-SceI at the DSB site. Similar <br />                                                  kinetics were recently shown for the interaction between SIRT1 and DSBs in CpG-rich <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  DNA (O&apos;Hagan et al., 2008). <br />  <br />                                                  Interestingly, the association of Nbs1 with the break site was delayed and strongly reduced <br />                                                  in the absence of SIRT1 (Figures 5A and B). A similar effect was observed for recruitment <br />                                                  of Rad51 (Figure 5C). These data show that SIRT1 physically associates with sites of DNA <br />                                                  damage and agree with recent studies indicating that chromatin-modifying enzymes are <br />                                                  recruited to the DSB to prepare the site for incoming DNA repair factors (Botuyan et al., <br />                                                  2006; Tamburini and Tyler, 2005). <br />  <br />                               SIRT1 is required for efficient DSB repair and genomic stability <br />                                                  DSB repair occurs through two major pathways: HR and non-homologous end-joining <br />                                                  (NHEJ). Our Rad51 data suggested a role for SIRT1 in HR, which can be measured by <br />                                                  determining the repair frequency with which a defective GFP gene is restored to wild-type <br />                                                  upon I-SceI transfection (see Figure S7A). Inhibition of SIRT1 activity with NAM, the <br />                                                  specific SIRT1 inhibitor S91211 (Solomon et al., 2006), or stable knockdown of SIRT1 <br />                                                  resulted in a 25% to 50% reduction in GFP+ cells, indicating that SIRT1 is necessary for <br />                                                  efficient HR-mediated repair (Figure 5D, Figure S7B). Repair by NHEJ, the prominent DSB <br />                                                  repair pathway in G1 and post-mitotic cells, was also reduced in this assay system, but to a <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  lesser extent (Figure S7C). <br />  <br />                                                  To test whether SIRT1 is necessary for the maintenance of genomic stability, SIRT1 knock- <br />                                                  down ES cell lines were transiently exposed to H2O2 and analyzed for metaphase <br />                                                  aberrations. In the absence of genotoxic stress, there was no significant difference in <br />                                                  chromosomal stability between SIRT1-deficient and control cells, consistent with previous <br />                                                  findings (Chua et al., 2005). Strikingly, H2O2 treatment caused a significant increase in <br />                                                  chromosomal aberrations specifically in SIRT1 deficient cells (Figure 5E). The frequency of <br />                                                  chromatid breaks was comparable between knock-down and control ES cells, but the <br />                                                  number of chromosomal fusions, in particular dicentric chromosomes and Robertsonian <br />                                                  translocations, was significantly higher in the absence of SIRT1 (Table S3). Chromosome <br />                                                  fusions are generally a result of aberrantly repaired DNA breaks, further supporting a role <br />                                                  for SIRT1 in DSB repair. Although a general DSB defect should also increase chromatid <br />  <br />  <br />  <br />                                                          Cell. Author manuscript; available in PMC 2010 April 13. <br />                             Oberdoerffer et al.                                                                                            Page 6 <br />  <br />  <br />                                                  breaks, we predominantly detected stably inherited aberrations such as fusions as <br />                                                  metaphases were analyzed 48 h (approx. two divisions) after exposure to H2O2. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Increased SIRT1 levels protect from irradiation-induced cancer in mice <br />                                                  To test if SIRT1 promotes genomic stability in vivo, we used p53+/&#226;&#710;&#8217; mice (Jacks et al., <br />                                                  1994), which, when exposed to IR, show a high incidence of malignant thymic lymphoma <br />                                                  arising from a loss of heterozygosity (LOH) at the p53 locus (Kemp et al., 1994). To explore <br />                                                  the role of SIRT1 in damage-induced LOH, p53+/&#226;&#710;&#8217; mice were fed the SIRT1 activator <br />                                                  resveratrol (Baur and Sinclair, 2006) three weeks prior to irradiation. Resveratrol-treated <br />                                                  animals showed a 24% increase in survival (&#207;&#8225;2=5.0, P = 0.025, Wilcoxon rank sum test) and <br />                                                  a 45% reduction in the frequency of fatal thymic lymphomas (Figures 6A and B), resulting <br />                                                  in a tumor spectrum highly reminiscent of non-irradiated p53+/&#226;&#710;&#8217; mice (Donehower et al., <br />                                                  1992; Jacks et al., 1994), <br />  <br />                                                  Given that the effects of resveratrol may not be limited to SIRT1 activation, we generated a <br />                                                  SIRT1 transgenic mouse strain (SIRT1STOP), that allows for Cre-mediated temporal and <br />                                                  tissue-specific overexpression of SIRT1 via the deletion of a transcriptional STOP cassette <br />                                                  (Firestein et al., 2008). The SIRT1STOP strain was crossed to p53+/&#226;&#710;&#8217; mice carrying the <br />                                                  interferon (IFN) type I inducible Mx-cre transgene (Kuhn et al., 1995). Mice with Mx-cre- <br />                                                  dependent, IFN-inducible SIRT1 overexpression are referred to as &quot;MISTO mice&quot;. Injection <br />                                                  with the IFN inducer poly(I)poly(C), MISTO mice increased expression of SIRT1 in bone <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  marrow lymphocyte progenitors (3&#226;&#8364;&#8220;4 fold) as well as mature B and T cells (Figures 6C and <br />                                                  6D). Thymocytes had intrinsically high levels of SIRT1. Two weeks after SIRT1 induction, <br />                                                  p53+/&#226;&#710;&#8217; MISTO mice p53+/&#226;&#710;&#8217; controls were exposed to 4 Gy of &#206;&#179;-irradiation, and monitored <br />                                                  for tumor-related deaths. Deletion of the STOP cassette in tumor tissues was examined by <br />                                                  Southern blot or q-PCR (Figure S8A and B). <br />  <br />                                                  The mean survival of MISTO mice was ~46% greater than in control animals (Figure 6E, <br />                                                  &#207;&#8225;2=5.68, P=0.017, Log-Rank test; &#207;&#8225;2=4.9, P=0.027, Wilcoxon rank sum test). Furthermore, <br />                                                  the frequency of fatal thymic lymphoma was reduced by 45% in MISTO mice, consistent <br />                                                  with our finding in resveratrol-treated animals (Figure 6F). Tumor cells exhibited LOH at <br />                                                  the p53 locus both in control and MISTO mice (Figure S8C and D). Together, our data <br />                                                  indicate that increasing SIRT1 activity or quantity can increase genomic stability in vivo and <br />                                                  suppress tumorigenesis. <br />  <br />                               SIRT1-associated genes are deregulated in the aged brain <br />                                                  Our observations indicated that SIRT1, like its yeast counterpart, is recruited to DSBs in <br />                                                  response to genotoxic stress, resulting in a loss of silencing at repetitive DNA elements and <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  SIRT1-regulated genes. To test if these transcriptional changes were relevant to aging, we <br />                                                  compared the transcriptional changes caused by oxidative stress with those associated with <br />                                                  aging using q-RT-PCR and microarrays. We chose to examine neocortex because age- <br />                                                  related gene expression changes have been well characterized in this tissue (Lee et al., 2000; <br />                                                  Lu et al., 2004). Interestingly, two thirds of SIRT1-bound genes that were derepressed by <br />                                                  oxidative stress in vitro (Figure 3) were also derepressed during aging (Figures 7A and D). <br />                                                  Moreover, SIRT1-target genes identified by ChIP on chip in vitro (see Figure 2) were <br />                                                  significantly overrepresented amongst age-upregulated genes (&#207;&#8225;2=7.3, P=0.0055; Figure 7B <br />                                                  and Table S4). The abundance of major satellite repeat transcripts also increased <br />                                                  significantly with age (Figure S9A). <br />  <br />                                                  To gain mechanistic insights, we tested whether overexpression of SIRT1 in the brain could <br />                                                  delay these transcriptional changes, paralleling the ability of SIR2 to suppress the expression <br />                                                  of yeast mating type genes. The SIRT1STOP transgenic mouse was crossed to a brain- <br />  <br />  <br />                                                           Cell. Author manuscript; available in PMC 2010 April 13. <br />                             Oberdoerffer et al.                                                                                           Page 7 <br />  <br />  <br />                                                  specific Cre-driver (Nestin-cre) to generate Nestin-cre; SIRT1STOP mice, referred to as <br />                                                  &quot;NeSTO mice.&quot; Transcript levels of SIRT1-bound genes as well as age-upregulated non- <br />                                                  SIRT1 bound (control) genes were examined in NeSTO mice with comparable SIRT1 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  overexpression (~10 fold, Figure 7C, Figure S9B and C) at 8&#226;&#8364;&#8220;10 or 18&#226;&#8364;&#8220;19 months. <br />                                                  Strikingly, transcriptional derepression was almost exclusively observed for SIRT1- <br />                                                  associated genes and was completely suppressed in aged NeSTO mice (Figure 7D). The <br />                                                  only non-SIRT1 associated genes to change with ~19 months of age were inflammatory <br />                                                  markers associated with gliosis, a characteristic of brain aging (Nichols et al., 1995). This <br />                                                  upregulation was also repressed in NeSTO mice, indicating that there are secondary, <br />                                                  beneficial effects of SIRT1 overexpression. The effect of SIRT1 overexpression in mice <br />                                                  older than 19 months of age is not yet known. Together, our results confirm that SIRT1- <br />                                                  bound genes are derepressed in the aging brain and that SIRT1 overexpression can suppress <br />                                                  these age-related changes. <br />  <br />                               Discussion <br />                                                  The discovery that the yeast Sir complex relocalizes during aging and in response to DNA <br />                                                  damage (Kennedy et al., 1995; Kennedy et al., 1997; Martin et al., 1999; McAinsh et al., <br />                                                  1999; Mills et al., 1999) led us to propose a model whereby aging is caused, in part, by the <br />                                                  DNA-damage induced reorganization of chromatin, a phenomenon we have termed the <br />                                                  &quot;RCM response,&quot; for redistribution of chromatin modifiers (Imai and Kitano, 1998; <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  Oberdoerffer and Sinclair, 2007; Villeponteau, 1997). Here, we present evidence that the <br />                                                  RCM response exists in mammals, and that it may contribute to age-related changes in gene <br />                                                  expression. <br />  <br />                               SIRT1 is recruited to DSBs and is required for efficient DNA repair <br />                                                  A role for chromatin modifiers in DNA repair has been convincingly shown in yeast <br />                                                  (Tamburini and Tyler, 2005) and mammals (Bassing et al., 2002; Bhaskara et al., 2008; <br />                                                  Botuyan et al., 2006; Celeste et al., 2002; Xie et al., 2004). Here, we show that SIRT1 binds <br />                                                  to hundreds of promoters in the mouse genome, and that this binding pattern is altered in <br />                                                  response to genotoxic stress, coincident with the relocalization of SIRT1 to damaged DNA. <br />  <br />                                                  The recruitment of SIRT1 to DSBs is reminiscent of yeast Sir2 and other histone modifying <br />                                                  enzymes that bind to a DSB, resulting in epigenetic changes surrounding the break site <br />                                                  (Chen et al., 2008; Tamburini and Tyler, 2005). As shown for the methylation of H4 at <br />                                                  lysine 20, chromatin alterations surrounding a DSB can promote the recruitment of DNA <br />                                                  repair factors such as 53BP1 (Botuyan et al., 2006). DSB associated SIRT1 may serve to <br />                                                  deacetylate histones or DNA repair factors. Consistent with the latter, SIRT1 was shown to <br />                                                  directly interact with Nbs1 (Yuan et al., 2007) and we show here that both proteins coexist <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  at the break site (Figure 5B). How SIRT1 recruitment to DSBs is initiated at the molecular <br />                                                  level will require further study, but our data indicate that ATM-mediated signaling through <br />                                                  H2AX phosphorylation is important (Figure 4B and 4C). We further demonstrate SIRT1 is <br />                                                  required for efficient DSB repair and genome maintenance in response to oxidative stress. <br />                                                  The direct association with DSBs as well as the finding that SIRT1-deficient cells are <br />                                                  checkpoint proficient suggest a DNA repair defect rather than a checkpoint defect (Cheng et <br />                                                  al., 2003). Consistent with a role for SIRT1 in DNA repair, high doses of IR have been <br />                                                  reported to induce a cell cycle delay in SIRT1- deficient fibroblasts (Yuan et al., 2007), <br />                                                  reminiscent of SIRT6-deficient cells, which show a DNA repair defect and prolonged S <br />                                                  phase but no checkpoint defect (Mostoslavsky et al., 2006). <br />  <br />                                                  Based on studies showing that SIRT1 can deacetylate and inactivate p53, some researchers <br />                                                  predicted that SIRT1 will promote tumorigenesis in vivo (Lim, 2006). In contrast, p53+/&#226;&#710;&#8217; <br />                                                  mice with increased SIRT1 activity are less susceptible to irradiation-induced thymic <br />  <br />  <br />                                                           Cell. Author manuscript; available in PMC 2010 April 13. <br />                             Oberdoerffer et al.                                                                                            Page 8 <br />  <br />  <br />                                                  lymphoma with a 20&#226;&#8364;&#8220;46% greater mean lifespan (Figure 6). It is appealing to speculate that <br />                                                  increased SIRT1 levels protect from irradiation-induced LOH by increasing DSB repair <br />                                                  efficiency. Our finding that MISTO transgenic mice overexpressed SIRT1 in early <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  lymphocyte precursors but not at later stages during thymic T cell development (Figures 6C <br />                                                  and 6D) is consistent with this hypothesis and argues against a protective role for SIRT1 at <br />                                                  later stages of tumor progression. We cannot rule out, however, that additional protective <br />                                                  mechanisms contributed to the protection from tumorigenesis. <br />  <br />                               Derepression of SIRT1-associated loci in response to oxidative stress and aging <br />                                                  SIRT1 has previously been reported to contribute to the formation of facultative <br />                                                  heterochromatin (Vaquero et al., 2007). Our data show that SIRT1-mediated repression can <br />                                                  also occur at constitutive heterochromatic regions such as pericentromeric DNA, as well as a <br />                                                  number of specific genes, most prominently regulators of chromatin assembly and <br />                                                  transcription. Oxidative stress reduces the association of SIRT1 with both repetitive loci <br />                                                  (Figure 1E) and individual genes (Figure 2). Not surprisingly, this change in SIRT1 <br />                                                  localization is associated with functional consequences on silencing of heterochromatic <br />                                                  repetitive DNA (Figure 1F) as well as the expression of individual genes (Figure 3). In cells <br />                                                  and in the aging brain, the majority of these changes were counteracted by overexpressing <br />                                                  SIRT1 (Figure 3 and Figure 7D). Loss of SIRT1 binding, however, did not always lead to <br />                                                  transcriptional derepression, indicating that chromatin alterations may be necessary but not <br />                                                  always sufficient to cause transcriptional deregulation. Further work will be required to <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  identify other chromatin modifiers or transcription factors that are involved in the RCM <br />                                                  response. HDAC1 and HDAC2 are candidates for the RCM response given their increased <br />                                                  binding to chromatin after DNA damage (see Figure S10). <br />  <br />                                                  Although a wealth of data on changes in gene expression with age has been catalogued in <br />                                                  recent years, there is still much debate about their physiological relevance (Oberdoerffer and <br />                                                  Sinclair, 2007; Vijg, 2004). Transcriptional changes in the elderly may be a beneficial <br />                                                  defense response to cellular damage (Niedernhofer et al., 2006). Conversely, age-related <br />                                                  changes in gene expression may be deleterious, yet exist because of the weak forces of <br />                                                  natural selection at older ages. Thus, while a transient RCM response in young individuals is <br />                                                  likely beneficial, constitutive triggering of RCM may actually contribute to aging. This <br />                                                  duality is clearly evidenced in yeast, where the transient derepression of silent HM loci <br />                                                  increases HR and resistance to DNA damaging agents (Lee et al., 1999) yet the constitutive <br />                                                  derepression of HM loci in old cells is a cause of sterility (Smeal et al., 1996). In mammals, <br />                                                  defective DNA repair is often associated with premature aging (Lombard et al., 2005). <br />                                                  Conversely, the lack of a DNA damage response can be beneficial in situations of chronic <br />                                                  DNA damage due to telomere dysfunction (Choudhury et al., 2007; Schaetzlein et al., 2007). <br />                                                  Furthermore, exposure to genotoxic stress early in life seems to accelerate changes in gene <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  expression that have been associate with age-related diseases such as amyloidogenesis (Wu <br />                                                  et al., 2008). Interestingly, we found that constitutive overexpression of a set of age- <br />                                                  deregulated SIRT1 target genes promotes apoptosis in primary neurons (Figure S11), <br />                                                  however more work is needed to determine the physiological relevance of this observation. <br />  <br />                               Perspective <br />                                                  We have identified SIRT1 as participant in a stress response that may provide a direct link <br />                                                  between DNA damage and gene expression changes that occur during aging. While DNA <br />                                                  damage has been previously suggested to directly inhibit gene repression (Lu et al., 2004), <br />                                                  our data explains how ostensibly undamaged genes may become deregulated over time. We <br />                                                  speculate that the RCM response may also cause permanent changes to the chromatin <br />                                                  structure at sites of repair, leading to stable transcriptional changes that accumulate over a <br />                                                  lifetime (Oberdoerffer and Sinclair, 2007). Indeed, a recent report showed that SIRT1 <br />  <br />  <br />                                                           Cell. Author manuscript; available in PMC 2010 April 13. <br />                             Oberdoerffer et al.                                                                                          Page 9 <br />  <br />  <br />                                                  recruitment to a DNA break in CpG islands can result in DNA methylation changes and <br />                                                  heritable gene silencing (O&apos;Hagan et al., 2008). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  Because age-related transcriptional changes are not limited to SIRT1-regulated loci <br />                                                  (Botuyan et al., 2006; Tamburini and Tyler, 2005), multiple mechanisms involving other <br />                                                  chromatin modifiers are likely to be involved. What sets SIRT1 apart is its link to calorie <br />                                                  restriction (CR), a dietary regimen that slows aging in mammals (Sinclair, 2005). Given that <br />                                                  increased SIR2/SIRT1 expression can suppress genomic instability and gene expression <br />                                                  alterations, perhaps CR promotes genomic stability and delays aging in mammals via a <br />                                                  similar mechanism. <br />  <br />                               Experimental Procedures <br />                               Yeast experiments <br />                                                  All experiments were on log phase yeast growing in YPD (2% glucose). HMR::GFP cells or <br />                                                  HMR::GFP 2xSIR2 cells (Park et al., 1999) were exposed to H2O2 for 30 min, followed by a <br />                                                  4 h recovery period, then analyzed FACS. Replicative lifespans and rDNA recombination <br />                                                  analyses were performed as described (Lamming et al., 2005). For rDNA recombination, <br />                                                  WT or Sir2o/e W303AR cells were treated for 30 min with H2O2 (1.5 mM) prior to plating. <br />                                                  For lifespan analysis, WT or 2xSIR2 cells were plated on regular YPD agar or agar <br />                                                  supplemented with H2O2 (1 mM). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Cell culture and treatments <br />                                                  Mouse ES cells were cultured on gelatinized tissue culture dishes as described <br />                                                  (Kanellopoulou et al., 2005). Stable SIRT1 overexpressing V6.5&#226;&#8364;&#8220;C10 ES cells were <br />                                                  obtained from SIRT1STOP ES cells (Firestein et al., 2008) by Cre-mediated deletion of a <br />                                                  loxP-flanked STOP cassette. SIRT1 knock-down ES cells were generated by lentiviral <br />                                                  infection of V6.5&#226;&#8364;&#8220;C10 ES cells (Beard et al., 2006). Cells were infected with either a <br />                                                  luciferase-specific or a SIRT1-specific shRNA lentiviral vector (Araki et al., 2004). <br />                                                  Generation of SIRT1 <span id='am-163' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-164' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span>knock-out</span> ES cells and respective wild-type cells is described (Chua et <br />                                                  al., 2005). Cells were &#206;&#179;-irradiated (4 Gy, 137Cs irradiator, Shepherd and Associates) or <br />                                                  treated with H2O2 or MMS for 1 h at 37&#194;&#176;C. Treatment with 50 &#194;&#181;M wortmannin (Sigma) or <br />                                                  25 &#194;&#181;M KU55933 (AstraZeneca) was started 2&#226;&#8364;&#8220;3 h prior to other treatments. NAM (Sigma, <br />                                                  25 mM), TSA (Sigma, 0.1 &#194;&#181;M) or sirtinol (100 &#194;&#181;M) were added for the indicated times. <br />  <br />                                                  DRGFP-transgenic U2OS cells and I-SceI- or Ds-red-encoding plasmids pCBASceI and <br />                                                  pCAGGS-Dsred are described (Weinstock et al., 2006). Stable SIRT1 knock-down and <br />                                                  control lines were generated by lentiviral gene transfer using shRNA vectors from Open <br />                                                  Biosystems. Transfection with pCBASce or pCAGGS-Dsred was performed using Fugene 6 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  transfection reagent (Roche). When indicated, cells were treated with NAM (10 mM) or <br />                                                  S91211/EX-527 (50 &#194;&#181;M, (Solomon et al., 2006)) starting 2 h prior to transfection. After 48 <br />                                                  h, cells were analyzed by FACS. <br />  <br />                               Chromatin immunoprecipitation (ChIP) <br />                                                  Approximately 107 cells were cross-linked with 1% formaldehyde for 15&#226;&#8364;&#8220;20 minutes at <br />                                                  37&#194;&#176;C and quenched with glycine. Cell lysates were sonicated (Branson sonifier) and <br />                                                  incubated overnight with rabbit &#206;&#177;-Sir2&#206;&#177; (Upstate), &#206;&#177;-Nbs1 (Novus), &#206;&#177;-Rad51 (Calbiochem) <br />                                                  or &#206;&#177;-H1AcK26 (Vaquero et al., 2004). Immunoprecipitation was performed as described <br />                                                  (Upstate) and eluates were purified using QIAgen PCR purification, followed by (q-) PCR <br />                                                  analysis. <br />  <br />  <br />  <br />  <br />                                                          Cell. Author manuscript; available in PMC 2010 April 13. <br />                             Oberdoerffer et al.                                                                                       Page 10 <br />  <br />  <br />                               RNA isolation, reverse transcription, PCR analysis <br />                                                  Total RNA was isolated using Trizol reagent (Invitrogen), followed by DNase treatment <br />                                                  (Turbo Dnase free, Ambion). RNA was reverse transcribed with Invitrogen&#226;&#8364;&#8482;s Thermoscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  RT PCR system using a combination of random hexamers and oligo dT primers. PCR was <br />                                                  performed using Taq Platinum (Roche). q-PCR was performed using SYBR green-based <br />                                                  detection on a Roche Light Cycler or Roche LC480. See Table S5 for primers and PCR <br />                                                  conditions. <br />  <br />                               Microarray analysis and statistics <br />                                                  For Nimblegen promoter tiling array analysis (Nimblegen-Roche), ChIP DNA was <br />                                                  amplified by ligation-mediated PCR. IP and input DNA samples were labeled using 9mer <br />                                                  Cy3 and Cy5-labeled primers. IP and total DNAs were co-hybridized to the NimbleGen <br />                                                  MM5 minimal promoter tiling array and analyzed using NimbleScan software (NimbleGen). <br />                                                  Peak data files were generated by searching for 4 or more probes with significant <br />                                                  enrichment using a 500 bp sliding window. Each peak was assigned a false discovery rate <br />                                                  (FDR) score based on randomization. Gene ontology cluster analysis of SIRT1-associated <br />                                                  promoters (FDR &#226;&#8240;&#164; 0.1) was performed using the BiNGO plug-in in Cytoscape v2.5. <br />  <br />                                                  For Affymetrix expression analysis, total RNA was hybridized to the mouse genome 430 2.0 <br />                                                  array. CEL files were analyzed for significance and fold changes between experimental <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  groups using DChip software. For the comparison of Nimblegen and Affymetrix array data, <br />                                                  Gene IDs of both arrays were matched using DAVID. Analysis was limited to genes with <br />                                                  highly significant SIRT1 promoter enrichment (FDR &lt; 0.005) and transcriptional increase (a <br />                                                  &#226;&#8240;&#165; 10%, P &#226;&#8240;&#164; 0.005). &#207;&#8225;2-based P values were calculated using Pearson&apos;s Chi-squared test with <br />                                                  Yates&apos; continuity correction. Microarray data have been deposited in NCBI&apos;s Gene <br />                                                  Expression Omnibus and are accessible through GEO Series accession number GSE13121 <br />                                                  (http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE13121). <br />  <br />                               Cellular fractionation and Western blotting <br />                                                  Chromatin-bound protein was purified as described (Cha et al., 2005). Primary antibodies <br />                                                  were rabbit &#206;&#177;-Sir2&#206;&#177;, &#206;&#177;-histone H3, &#206;&#177;-Histone H4 (Upstate), rabbit &#206;&#177;-YY1 (Santa Cruz), <br />                                                  mouse &#206;&#177;-GAPDH (Chemicon) and rabbit &#206;&#177;-Rad51 (Dr. R. Scully); &#206;&#177;-rabbit and &#206;&#177;-mouse <br />                                                  HRP-coupled secondary antibodies were from GE Healthcare. <br />  <br />                               Metaphase analysis <br />                                                  Metaphase spreads were performed as previously described (Mostoslavsky et al., 2006). At <br />                                                  least 80 metaphases of each genotype were scored per experiment. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Mouse breeding and treatments <br />                                                  All mice were housed pathogen-free. SIRT1STOP (Firestein et al., 2008), Mx-cre (gift from <br />                                                  Dr. K. Rajewsky), Nestin-cre and p53&#226;&#710;&#8217;/&#226;&#710;&#8217; mice (Jackson Laboratory) were crossed as to <br />                                                  obtain the indicated genotypes. Experimental animals were on C57BL/6 &#195;&#8212; 129/Sv mixed <br />                                                  genetic background. For Mx-cre induction, mice were injected with 400 &#194;&#181;g poly(I)poly(C) <br />                                                  (Amersham) at 6&#226;&#8364;&#8220;10 weeks of age (Kuhn et al., 1995). For tumor studies, mice were &#206;&#179;- <br />                                                  irradiated 10&#226;&#8364;&#8220;14 days thereafter with a single dose of 4 Gy. Animals were sacrificed when <br />                                                  moribund. When not obvious, mice were submitted to necropsy to identify the cause of <br />                                                  death. Deaths not related to tumors and mice too decomposed for analysis were censored. <br />                                                  Kaplan-Meier survival curves were generated from two separate cohorts of irradiated <br />                                                  animals using JMP7 software. Resveratrol was fed at 2.4 mg/kg food as previously <br />                                                  described (Baur et al., 2006). Two cohorts were pooled for survival analysis. <br />  <br />  <br />  <br />                                                          Cell. Author manuscript; available in PMC 2010 April 13. <br />  <br />                                                  Figure 1. <br />                                                  Oxidative stress reduces Sir2/SIRT1-mediated repetitive DNA silencing in yeast and <br />                                                  mammalian cells. (A&#226;&#8364;&#8220;C). Effects of H2O2 on yeast aging. (A) H2O2 causes loss of silencing <br />                                                  at the mating type loci. Shown is the fraction of GFP+ cells in 2xSIR2 and WT cells. Unless <br />                                                  otherwise noted, P values are based on Student&#226;&#8364;&#8482;s two-tailed t test with #P&#226;&#8240;&#164; 0.1, *P&#226;&#8240;&#164; 0.05, <br />                                                  **P&#226;&#8240;&#164; 0.01, ***P&#226;&#8240;&#164; 0.001. (B) Frequency of H2O2-induced rDNA recombination in the <br />                                                  absence presence of extra Sir2 (WT and Sir2 o/e). (C) Replicative lifespan of WT and <br />                                                  2xSIR2 cells grown in the absence or presence of 1 mM H2O2. (D) Mouse SIRT1 binds to <br />                                                  repetitive genomic DNA. ChIP for SIRT1 or control immunoglobulin (Ig) at major satellite <br />                                                  repeats in the absence or presence of NAM (25 mM) (E) q-PCR analysis of ChIP <br />                                                  experiments using antibodies specific for SIRT1 or H1AcK26. ES cells were left untreated <br />                                                  or treated with H2O2 (2 mM) or NAM for 1 h. (F) Oxidative stress increases transcription of <br />                                                  satellite repeat DNA. ES cells were treated with NAM for 24 h or with H2O2 for 1 h, <br />                                                  followed by 23 h recovery. (G) Cells with a targeted extra copy of SIRT1 and control cells <br />                                                  were treated with H2O2 and analyzed as in (F). The inset shows a Western blot of SIRT1 in <br />                                                  WT (black) and SIRT1 overexpressing ES cells (white). Data are represented as mean +/&#226;&#710;&#8217; <br />                                                  SEM. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                              Cell. Author manuscript; available in PMC 2010 April 13. <br />                             Oberdoerffer et al.                                                                                       Page 16 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  Figure 2. <br />                                                  Oxidative stress causes a major redistribution of SIRT1. (A) Distribution of SIRT1 and <br />                                                  H1AcK26 along a representative chromosome. ES cells were either untreated or treated with <br />                                                  H2O2 for 1 h, followed by ChIP for SIRT1, H1AcK26 or control Ig. The x-axis depicts <br />                                                  probe sets spanning the promoters of annotated ORFs, signals across the y-axis reflect the <br />                                                  log2 change of IP DNA over input DNA. (B) Significance of overrepresentation of selected <br />                                                  gene ontology (GO) groups amongst SIRT1-associated genes before and after H2O2 <br />                                                  treatment. Asterisks indicate significance after Benjamini- Hochberg false discovery rate <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  correction. (C) Loss of SIRT1 binding correlates with increased H1K26 acetylation. <br />                                                  Promoters with &#226;&#8240;&#165; 2log2 enrichment of SIRT1 prior to H2O2 treatment, but no signal <br />                                                  thereafter were compared to non-SIRT1 associated promoters. The y-axis shows the fraction <br />                                                  of promoters with a &#226;&#8240;&#165; 2-fold increase in H1AcK26 enrichment after exposure to H2O2. <br />                                                  SIRT1 and H1AcK26 probe set binding is shown for four of these promoters. GAPDH <br />                                                  served as a negative control. <br />  <br />  <br />  <br />  <br />                                                              Cell. Author manuscript; available in PMC 2010 April 13. <br />                             Oberdoerffer et al.                                                                                       Page 17 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  Figure 3. <br />                                                  Transcriptional deregulation of SIRT1-associated genes in response to oxidative stress is <br />                                                  repressed by increasing SIRT1 levels. (A) SIRT1 expression in wild-type (WT) and SIRT1 <br />                                                  overexpressing (OE) ES cells. (B) q-RT-PCR analysis of putative SIRT1 target genes <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  (SIRT1-bound), non-SIRT1-bound control genes and &#206;&#178;-actin in wild-type (open bars) and <br />                                                  SIRT1 overexpressing ES cells (closed bars). Shown is the fold change of expression <br />                                                  compared to untreated samples. Data are represented as mean +/&#226;&#710;&#8217; SEM. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                              Cell. Author manuscript; available in PMC 2010 April 13. <br />                             Oberdoerffer et al.                                                                                      Page 18 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  Figure 4. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  SIRT1 is recruited to chromatin upon DNA damage in an ATM-dependent manner. (A) <br />                                                  Western blot analysis of SIRT1 and indicated control proteins in both chromatin-bound and <br />                                                  non-chromatin-bound protein fractions from ES cells that were either left untreated or <br />                                                  treated with H2O2 or MMS for 1 h. (B and C) Western blot analysis of chromatin-bound <br />                                                  SIRT1 in response to H2O2 with or without pretreatment with wortmannin or KU55933 (B), <br />                                                  and in the presence or absence of H2AX (C). (D and E) Recruitment of SIRT1 and Rad51 to <br />                                                  chromatin in WT or SIRT1-deficient (SIRT1-&#206;&#8221;ex4) cells in response to H2O2 or IR. <br />  <br />  <br />  <br />  <br />                                                              Cell. Author manuscript; available in PMC 2010 April 13. <br />                             Oberdoerffer et al.                                                                                        Page 19 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  Figure 5. <br />                                                  SIRT1 is recruited to DNA breaks and is required for efficient DSB repair. (A) SIRT1 <br />                                                  expression in parental U2OS-DRGFP cells, three independent shSIRT1 expressing cell lines <br />                                                  (SIRT1-KD) and shRFP expressing controls. (B) ChIP analysis of SIRT1 and Nbs1 binding <br />                                                  to a DSB in shRFP (control) and SIRT1 KD cells from (A). (C) DSB ChIP analysis of <br />                                                  Rad51 recruitment using Ds-red-transfected (&#226;&#710;&#8217;) or I-SceI transfected cells (+) from (B) 24h <br />                                                  after transfection. (D) SIRT1 knock-down results in reduced DSB repair as measured by <br />                                                  GFP rexpression (see Figure S7A). Control and SIRT1-KD cell lines from (A) were <br />                                                  transfected as in (C) and analyzed by FACS. (E) Loss of SIRT1 causes increased genomic <br />                                                  instability upon oxidative stress. Untreated or H2O2 treated shSIRT1- or control shRNA- <br />                                                  expressing ES cells were subjected to Q-FISH analysis. Telomeres are shown in red. Arrows <br />                                                  indicate a chromatid break (left) and a fused centromere (right). The fraction of aberrant <br />                                                  metaphases is shown. Data are represented as mean +/&#226;&#710;&#8217; SEM. <br />  <br />  <br />                                                              Cell. Author manuscript; available in PMC 2010 April 13. <br />                             Oberdoerffer et al.                                                                                        Page 20 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  Figure 6. <br />                                                  Increased SIRT1 activity alters tumor spectrum and increases survival of irradiated p53+/&#226;&#710;&#8217; <br />                                                  mice. (A) Survival of irradiated p53+/&#226;&#710;&#8217; mice fed normal or resveratrol-supplemented chow <br />                                                  (n=19 and 25, respectively). Tumor-related deaths were recorded in days after irradiation. <br />                                                  (B) Tumor spectrum from mice in (A), legend lists dominant tumor at time of death; #no <br />                                                  tumors detected in necropsy. (C) Western blot analysis of MACS-purified thymic CD4/CD8 <br />                                                  double-positive (DP) T cells and splenic CD8+ T cells from MISTO mice (+ Mx-cre) and <br />                                                  littermate controls (&#226;&#710;&#8217; Mx-cre). Mice were analyzed 14 days after Mx-cre induction; Thy: <br />                                                  Thymus, Spl: Spleen. (D) SIRT1 mRNA expression in lymphocyte subsets from MISTO <br />                                                  mice (closed bars) and littermate controls (open bars). BM: Bone marrow, Lin+: Lineage- <br />                                                  positive. (E) Survival of MISTO and control p53+/&#226;&#710;&#8217; mice (n=12 and 16, respectively) in <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  response to a single dose of 4 Gy &#206;&#179;-irradiation. Tumor-related deaths were recorded in days <br />                                                  after irradiation. (F) Tumor spectrum in mice from (E), legend as in (B). <br />  <br />  <br />  <br />  <br />                                                              Cell. Author manuscript; available in PMC 2010 April 13. <br />                             Oberdoerffer et al.                                                                                         Page 21 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  Figure 7. <br />                                                  Transcriptional deregulation of SIRT1 target genes occurs during normal aging. (A) <br />                                                  Expression of SIRT1-associated gene from Figure 3 in the neocortex from young (5 months) <br />                                                  and old (30 months) B6C3F1 mice (n &#226;&#8240;&#165; 6 per group), analyzed by q-PCR. Shown is the fold <br />                                                  change in expression in old relative to young mice. P values are based on student&#226;&#8364;&#8482;s one- <br />                                                  tailed t-test. (B) Microarray expression analysis of SIRT1 target genes in the neocortex of <br />                                                  young and old B6C3F1 mice (n = 5 per group). SIRT1 target genes are significantly <br />                                                  overrepresented amongst age-upregulated genes (&#207;&#8225;2 = 7.28, P = 0.0055). (C) <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  Immunofluorescence analysis of SIRT1 and NeuN expression in the neocortex of NeSTO <br />                                                  mice and Nestin-cre littermate controls. (D) q-RT-PCR analysis of SIRT1-bound genes from <br />                                                  (A) and non-SIRT1-bound genes including housekeeping genes (rps16, hprt) and genes <br />                                                  upregulated in 30 months old mice by microarray. NeSTO mice and Nestin-cre littermate <br />                                                  controls (Control) were analyzed at 8&#226;&#8364;&#8220;10 months (young, n = 3&#226;&#8364;&#8220;5 and 6, respectively) or 18&#226;&#8364;&#8220; <br />                                                  19 months of age (old, n = 4 and 5, respectively). P values are based on student&#226;&#8364;&#8482;s one-tailed <br />                                                  t-test. Data are represented as mean +/&#226;&#710;&#8217; SEM. <br />  <br />  <br />  <br />  <br />                                                              Cell. Author manuscript; available in PMC 2010 April 13. <br />   <br /> </body></html>