<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>BMC Genomics                                                                                                                             BioMed Central <br />  <br />  <br />  <br /> Research article                                                                                                                       Open Access <br /> Buffering gene duplicates: analysis molecular correlates and <br /> evolutionary conservation <br /> Kevin Hannay, Edward M Marcotte Christine Vogel* <br />  <br /> Address: Institute Cellular Molecular Biology, Center Systems Synthetic Biology, University Texas Austin, 2500 Speedway, <br /> MBB 3. 210, Austin, TX 78712, USA <br /> Email: Kevin Hannay - moleculeboy24@gmail.com; Edward M Marcotte - edward.marcotte@gmail.com; <br /> Christine Vogel* - cvogel@mail.utexas.edu <br /> * Corresponding author <br />  <br />  <br />  <br />  <br /> Published: 16 December 2008                                                    Received: 7 August 2008 <br />                                                                                Accepted: 16 December 2008 <br /> BMC Genomics 2008, 9:609    doi:10.1186/1471-2164-9-609 <br /> This article available <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span>: http://www.biomedcentral.com/1471-2164/9/609 <br /> &#194;&#169; 2008 Hannay et al; licensee BioMed Central Ltd. <br /> This Open Access article distributed terms Creative Commons Attribution License (http://creativecommons.org/licenses/by/2.0), <br /> which permits unrestricted use, distribution, reproduction medium, provided original work properly cited. <br />  <br />  <br />  <br />  <br />                  <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />                  Background: One mechanism account robustness gene knockouts knockdowns <br />                  buffering gene duplicates, extent general correlates process in <br />                  organisms matter debate. <span id='am-6' about='protege:TO' typeof='owl:Thing'>To</span> reveal general trends process, provide a <br />                  comprehensive comparison gene essentiality, duplication buffering duplicates across <br />                  seven bacteria (Mycoplasma genitalium, Bacillus subtilis, Helicobacter pylori, Haemophilus influenzae, <br />                  Mycobacterium tuberculosis, Pseudomonas aeruginosa, Escherichia coli), eukaryotes <br />                  (Saccharomyces cerevisiae (yeast), Caenorhabditis elegans (worm), Drosophila melanogaster (fly), Mus <br />                  musculus (mouse)). <br />                  Results: In organisms, duplicates significantly increase chances <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span>survival</span> upon <br />                  gene deletion (P-value &#226;&#8240;&#164; 0.05), 13%. Given duplicates make 80% of <br />                  eukaryotic genomes, small contribution surprising points dominant roles other <br />                  buffering processes, alternative metabolic pathways. The buffering capacity duplicates <br />                  appears independent degree gene essentiality tends higher genes with <br />                  high expression levels. For example, buffering capacity increases 23% highly expressed <br />                  genes E. coli. Sequence similarity number duplicates gene weak predictors of <br />                  duplicate&apos;s buffering capacity. In case study buffering gene duplicates yeast <br />                  worm somewhat similar functions non-buffering duplicates have <br />                  increased transcriptional translational activity. <br />                  Conclusion: In sum, extent gene essentiality buffering duplicates conserved <br />                  organisms does correlate organisms&apos; apparent complexity. This <br />                  heterogeneity goes expected differences experimental approaches <br />                  alone. Buffering duplicates contributes robustness organisms, small extent <br />                  &#226;&#8364;&#8220; relatively large buffering duplicates observed yeast worm be <br />                  largely specific organisms. Thus, common factor buffering duplicates <br />                  different organisms by-product duplicate retention demands high <br />                  dosage. <br />  <br />  <br />  <br />  <br />                                                                                                                                        Page 1 18 <br />                                                                                                                  (page number citation purposes) <br />  BMC Genomics 2008, 9:609                                                      http://www.biomedcentral.com/1471-2164/9/609 <br />  <br />  <br />  <br />  <br /> Background                                                       genes organism influenced factors: the <br /> Cells organisms remarkable robustness                 mutational strategy (insertion, <span id='am-241' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-242' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-243' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-246' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-249' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-250' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-251' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-253' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-258' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-262' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-264' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-277' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-290' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-295' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-315' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-318' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span>knockout</span> (deletion) or <br /> against loss genes, triggered        knockdown), growth organism clonal mixed <br /> ongoing discussion factors promoting robust-         populations, life cycle stage organism, and, for <br /> ness [1,2]. One simplest obvious mecha-          multi-cellular organisms, organism or <br /> nism buffering redundancy produced gene                simply cell line targeted. Selection pressure more <br /> duplicates [3,4]. Indeed, gene duplication major factor     stringent mixed clonal populations, we <br /> shaping prokaryotic eukaryotic genomes [5-7]. Dupli-         expect lower survival rates former. For example, a <br /> cate genes diverge sequence function [7]        mutant bacterium decreased fitness selected <br /> may ability buffer loss        mixed population, able form an <br /> respective homolog. While processes buffering         isolated colony. Insertion experiments result leaky <br /> by duplicates play important roles robustness         expression compared knockout deletion experi- <br /> gene loss, e.g. use alternative pathways [8,9], rela-     ments, identify fewer essential genes. Finally, <br /> tionship essentiality existence gene          RNAi experiments worm reasonably low <br /> duplicates attracted attention, previous            false-positive false-negative rates [25,26], would <br /> work revealed intricate picture.                              expect lower degrees gene essentiality this <br />                                                                  knockdown technique gene deletions. <br /> For example, estimates role duplicates backups <br /> for gene loss vary widely organisms.           To gain insights general principles buffer- <br /> Most yeast genes non-essential, i.e. dispensable, rich    ing gene duplicates, conducted comprehensive <br /> medium standard laboratory conditions (&gt;80%,            cross-organism comparison essentiality rela- <br /> ref. [10]). A study Gu et al. attributes 23&#226;&#8364;&#8220;59%        tionship gene duplication, analyzing prokaryo- <br /> dispensability (or survival) buffering gene dupli-         tic eukaryotic organisms &#226;&#8364;&#8220; M. genitalium, H. pylori, H. <br /> cates [11], studies quote lower             influenzae, M. tuberculosis, P. aeruginosa, B. subtilis, E. coli, <br /> range (15&#226;&#8364;&#8220;28%) [8,12-15]. Only 2% gene pairs           S. cerevisiae (yeast), C. elegans (worm), D. melanogaster <br /> synthetic sick lethal (SSL) mutant <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> yeast         (fly), M. musculus (mouse). To so, addressed <br /> show detectable similarity [16,17],              above-mentioned challenges ways. When <br /> ~20% mouse genes examined to-date buffering             selecting essentiality datasets, aimed minimize vari- <br /> duplicates observed [18,19].                            ation experimental approaches, and, possi- <br />                                                                  ble, sampled organisms specific technique <br /> Several molecular causes underlie buffering dupli-        (Table 1). We tested different definitions gene duplica- <br /> cates, relative contributions debated. For   tion, measures expression levels, (for yeast) robust- <br /> example, buffering duplicates lack functional redundancy         ness results removal genes whole- <br /> that expected backup role. Buffering         gene duplication [27,28] ribosomal genes (Addi- <br /> duplicates yeast partially overlapping expres-      tional file 1). When assessing contribution dupli- <br /> sion [20] genetic interaction profiles [13], suggesting      cates survival gene-KO/KD, normalized the <br /> their functions diverged. Alternative explanations      number essential genes. Differences technical <br /> the bias duplicates essential genes         approaches certainly influence extent essentiality <br /> been suggested. For example, disadvantageous           detected organisms; however, duplicates have <br /> for cell retain duplicates genes severe          buffering role loss gene function this <br /> (lethal) knockout phenotypes disrupt            effect observable regardless exact number <br /> their finely balanced expression dosage [21]. Further,       genes identified essential. <br /> correlation gene expression levels existence <br /> of duplicates suggests buffering gene loss       Our study reveals heterogeneity essentiality the <br /> a by-product processes retain duplicates dosage      contribution duplicates survival goes beyond <br /> amplification [12,13,22,23].                                     accountable technical differences. We show <br />                                                                  organismal complexity lifestyle, gene function, <br /> Despite availability large-scale datasets      function similarity, sequence similarity number of <br /> single gene knockouts (KO) knock-downs (KD)           duplicates gene weak predictors buffer- <br /> as double gene-KOs organisms, previous          ing capacity &#226;&#8364;&#8220; gene expression levels related measures <br /> studies mainly focused single organisms like yeast            strongest correlates. Simple relationships with <br /> [8,11-14], worm [24] mouse [18,19]. Major hin-               respect essentiality gene duplication hold true for <br /> drances cross-organism comparison differences        organisms, others. Buffering dupli- <br /> experimental approaches specific <span id='am-3' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span>           cates plays significant small heterogeneous role. <br /> essentiality used. The types numbers essential <br />  <br />  <br />                                                                                                                       Page 2 18 <br />                                                                                                 (page number citation purposes) <br />                                                                                                                                                                                                                                                          Page 3 18 <br />                                                                                                                                                                                                                                                                        (page number citation purposes) <br /> http://www.biomedcentral.com/1471-2164/9/609 <br />  <br />                                                Table 1: Essentiality gene duplicates bacterial eukaryotic organisms. <br />  <br />                                                 Organism                 Essentiality test        No. tested genes          No. essential       Number genes            Contribution             R2 P(S) vs.            R2 P(S) vs. <br />                                                                                                                                          genes         duplicates             duplicates          Effective family                   E-value <br />                                                                                                                                                                (D &#226;&#8240;&#165; 1)              buffering C                     size D <br />  <br />                                                 M. genitalium            Random insertion                           460                      364                        89                    -0.13                     0.35                     0.16 <br />                                                                          (clones) <br />  <br />                                                 H. pylori                Random insertion                          1,559                     329                       358                  0.13***                     0.26                     0.14 <br />                                                                          (population) <br />  <br />                                                 H. influenzae            Random insertion                          1,704                     631                       400                    0.01*                     0.27                     0.20 <br />                                                                          (population) <br />  <br />                                                 M. tuberculosis          Random insertion                          3,920                     614                     1,683                  0.06***                    0.63*                     0.40 <br />                                                                          (population) <br />  <br />                                                 P. aeruginosa            Random insertion                          5,566                     364                     2,689                  0.07***                   0.64**                   0.80** <br />                                                                          (clones) <br />  <br />                                                 B. subtilis              Targeted insertion                        4,105                     191                     1,857                 0.0045*                      0.37                     0.01 <br />                                                                          (clones) <br />  <br />                                                 E. coli                  Targeted knockout                         3,221                     291                     1,940                  0.06***                   0.64**                   0.82** <br />                                                                          (clones) <br />  <br />                                                 S. cerevisiae            Targeted knockout                         5,318                     952                     2,531                  0.12***                     0.00                    0.72* <br />                                                                          (clones) <br />  <br />                                                 C. elegans               Targeted knockdown                      13,915                     1,345                    9,203                  0.09***                   0.74**                  0.92*** <br />                                                                          (clones) <br />  <br />                                                 D. melanogaster          Targeted knockdown                      12,145                      318                     7,004                  0.01***                     0.00                    0.60* <br />                                                                          cell line (clones) <br />  <br />                                                 M. musculus              Collection                             4,267                    1,438                    3,664                  -0.07**                     0.03                     0.00 <br /> BMC Genomics 2008, 9:609 <br />  <br />  <br />  <br />  <br />                                                                          individual <br />                                                                          experiments <br />  <br />                                                 The table summarizes properties organisms analysis, (from left right) names organims; type KO/KD experiment; number genes tested for <br />                                                 essentiality gene-KO KD experiments; number genes resulting lethal phenotypes (essential genes); number genes duplicates (D &#226;&#8240;&#165; 1) the <br />                                                 tested genes; contribution duplicates buffering C = P(S|D &#226;&#8240;&#165; 1)/P(S|D = 0) - 1; correlation P(S) effective family size genes D (D ranges 0 8+, text); and <br />                                                 correlation P(S) distance gene nearest neighbor (measured -log(E-value), bin size 5). In experimental descriptions, &apos;clones&apos; refers clonal outgrowth plates <br />                                                 cultures; &apos;population&apos; refers (mixed) population outgrowth liquid culture. P-value thresholds 0.05, 0.01, 0.001 marked *, **, ***, respectively. <br />                                                 KD &#226;&#8364;&#8220; knockdown; KO &#226;&#8364;&#8220; knockout; P(S) &#226;&#8364;&#8220; probability survival; D &#226;&#8364;&#8220; effective gene family size (number additional gene duplicates) <br />  BMC Genomics 2008, 9:609                                                      http://www.biomedcentral.com/1471-2164/9/609 <br />  <br />  <br />  <br />  <br /> Results discussion                                           Duplicates increase chances survival &#226;&#8364;&#8220; some <br /> The extent essentiality varies widely organisms       organisms others <br /> If duplicate genes play significant role buffering          To assess contribution duplicates survival follow- <br /> against mutations, genes paralogs          ing gene-KO/KD define buffering capacity C C = <br /> should higher chances survival deletion             P(S|D &#226;&#8240;&#165; 1)/P(S|D = 0) &#226;&#8364;&#8220; 1, P(S|D = 0) prob- <br /> than singletons. This simple relationship dem-          ability survival given gene does additional <br /> onstrated yeast [11] C. elegans [24],    duplicates, i.e. singleton. P(S|D &#226;&#8240;&#165; 1) probability <br /> other organisms. To test generality prediction,      survival given gene additional <br /> we estimated families homologous genes             duplicates. C calculated organism quanti- <br /> bacterial eukaryotic organisms based BLAST [29]         fies increase probability survival gene-KO/ <br /> sequence similarity search (E-value &lt; 1.0e-10), com-         KD genes duplicate genome. <br /> pared survival knockout (KO) knockdown (KD) <br /> of genes gene families survival KO/KD         In organisms, duplicates contribute sig- <br /> of singletons (Table 1). We estimate gene expression lev-        nificantly positively survival (P-value &#226;&#8240;&#164; 0.05); with <br /> els use Codon Bias Index (Methods).                    contributions ranging 1 13% (Table 1, Figure 2). <br />                                                                  The exceptions M. genitalium mouse which <br /> We define effective family size D target gene    duplicates appear decrease chances KO survival. The <br /> number duplicates remaining KO KD. D = 0             extent buffering duplicates, i.e. value C, does <br /> denotes singletons genes; D &#226;&#8240;&#165; 1 denotes genes para-         correlate organisms&apos; complexity genome <br /> logs. The probability P(D &#226;&#8240;&#165; 1) derived fraction      size. Total C largest yeast, worm H. pylori and <br /> of genes genome dupli-            smallest H. influenzae, B. subtilis fly. While total <br /> cates (paralogs). We use probability P(S)         number fraction genes duplicates increases <br /> describes organism chances survival gene-         simpler complex organisms (Figure 1B), the <br /> deletion; P(S) derived fraction genes identi-     propensity duplicates buffer gene loss varies <br /> fied dispensable (non-essential) large-scale screens.      independently. <br /> When discussing &apos;buffering duplicates&apos; mean the <br /> enrichment duplicates non-essential genes          Next ask genes duplicates <br /> inferred statistical analysis. &apos;Essentiality/non-essenti-   chances buffering gene loss increase high <br /> ality (survival)&apos; purely based outcomes experi-         expression levels compared low expression levels. In <br /> ments.                                                           organisms, significant differences in <br />                                                                  buffering capacity C genes low high <br /> Table 1, Figure 1 2 summarize results respect       expression levels (P-value &#226;&#8240;&#164; 0.05). However, five <br /> to survival gene duplication genomes.           organisms (H. pylori, P. aeruginosa, E. coli, yeast, and <br /> Most genomes dataset relatively essential        worm), genes high expression levels dupli- <br /> genes; chances survival loss single gene       cates significantly improved chances survival; with <br /> high prokaryotes eukaryotes (P(S) &gt; 0.80),           C reaching 23% E. coli. In M. genitalium M. tubercu- <br /> except M. genitalium, H. influenzae mouse (Figure        losis, C positive highly expressed genes when <br /> 1A). Genes high expression levels likely       examining experimental expression data (Additional file <br /> essential genes low expression levels (smaller           1); B. subtilis fly survival generally high and <br /> P(S)); half (six) organisms difference sig-     distinction genes high low expression <br /> nificant (P-value &#226;&#8240;&#164; 0.01).                                       does effect. <br />  <br /> In accordance expectation complex             These results robust various methods paralog <br /> organisms tend duplicate genes, fraction        estimation, exact numbers change depending on <br /> of genes duplicates (D &#226;&#8240;&#165; 1) increases M. genital-      parameter settings. We tested, example, different E- <br /> ium bacteria, yeast animals       value cutoffs, different <span id='am-5' about='xsp:length' typeof='owl:Thing'>length</span> requirements match <br /> (Figure 1B). Compared organisms, mouse            region using methods homology estimation <br /> noticeable depletion singleton genes (D = 0) relative      completely independent particular E-value <br /> genes duplicates. In organisms, signifi-    thresholds (Additional file 1). <br /> cant increase fraction duplicates (D &#226;&#8240;&#165; 1) amongst <br /> highly expressed genes compared genes (P-value          Further correlates buffering capacity <br /> &#226;&#8240;&#164; 0.01); exception B. subtilis trend       Assuming paralogs function a <br /> inverted. When using Codon Adaptation Index experi-           deleted gene, hypothesize chances doing <br /> mental expression data obtain similar results (Addi-          increase i) number paralogs present, ii) <br /> tional file 1). <br />  <br />  <br />                                                                                                                       Page 4 18 <br />                                                                                                 (page number citation purposes) <br />  BMC Genomics 2008, 9:609             http://www.biomedcentral.com/1471-2164/9/609 <br />  <br />  <br />  <br />  <br /> Figure 1 (see legend page) <br />  <br />  <br />  <br />  <br />                                                                           Page 5 18 <br />                                                     (page number citation purposes) <br />  BMC Genomics 2008, 9:609                                                            http://www.biomedcentral.com/1471-2164/9/609 <br />  <br />  <br />  <br />  <br /> Figure 1of(see <br /> Chances                upon <br />                 previous <br />              survival      page) <br />                               gene-KO/KD vary organisms <br /> Chances survival gene-KO/KD vary organisms. While number fraction duplicate genes <br /> increases prokaryotes single- multi-cellular eukaryotes, fraction essential genes (and chances sur- <br /> vival gene-KO/KD) vary widely. The panels probability survival P(S)(A), gene family distribution and <br /> number genes duplicates (D &#226;&#8240;&#165; 1)(B). Singleton genes labeled D = 0, members two-gene families labeled D = 1, <br /> members larger gene families labeled D &#226;&#8240;&#165; 2. Red bars indicate values genes, listed Table 1. High (black) <br /> and low (white) gene expression levels estimated codon bias indices (see methods). Significant differences between <br /> genes high low expression (&#207;&#8225;2 test) marked ** (P-value &#226;&#8240;&#164; 0.01) *** (P-value &#226;&#8240;&#164; 0.001). D &#226;&#8364;&#8220; effective gene fam- <br /> ily size (number additional duplicates gene); S &#226;&#8364;&#8220; survival gene deletion (1-essentiality). Mgen &#226;&#8364;&#8220; Mycoplasma genital- <br /> ium; Hpyl &#226;&#8364;&#8220; Helicobacter pylori; Hinf &#226;&#8364;&#8220; Haemophilus influenzae; Mtub &#226;&#8364;&#8220; Mycobacterium tuberculosis; Paer &#226;&#8364;&#8220; Pseudomonas aeruginosa; <br /> Bsub &#226;&#8364;&#8220; Bacillus subtilis; Ecol &#226;&#8364;&#8220; Escherichia coli; Scer &#226;&#8364;&#8220; Saccharomyces cerevisiae (yeast); Cele &#226;&#8364;&#8220; Caenorhabditis elegans (worm); Dmel &#226;&#8364;&#8220; <br /> Drosophila melanogaster (fly); Mmus &#226;&#8364;&#8220; Mus musculus (mouse). <br />  <br />  <br /> their similarity mutant protein. We tested pre-          &#226;&#8240;&#165; 0.64 R2 &#226;&#8240;&#165; 0.80, respectively; Table 1). These correla- <br /> dictions organisms.                                     tions observed previously worm [24], are <br />                                                                       common organisms study. Yeast <br /> Only organisms, P. aeruginosa, E. coli, worm,            decent correlation distance nearest <br /> chances survival correlate significantly (P-value &#226;&#8240;&#164; 0.05)          homology (R2 = 0.72), number dupli- <br /> with number duplicates available gene             cates gene. These results change when <br /> with distance gene nearest homolog (R2              removing ribosomal genes gene pairs originating from <br />  <br />  <br />  <br />  <br /> Small <br /> Figure but2 significant buffering duplicate genes gene-KO/KD <br /> Small significant buffering duplicate genes gene-KO/KD. In organisms analysis, duplicates <br /> contribute significantly survival gene-KO/KD (P-value &#226;&#8240;&#164; 0.05), small extent. Buffering increased <br /> amongst genes high expression levels (high CBI, black bars) compared genes lower expression levels (white bars). In <br /> highly expressed genes, duplicates contribute survival 23% (E. coli). Significant enrichment duplicates amongst <br /> non-essential genes (hypergeometric distribution) significant differences genes high low expression (&#207;&#8225;2 <br /> test) marked *, **, *** P-value thresholds 0.05, 0.01, 0.001, respectively. For abbreviations Figure 1. <br />  <br />  <br />  <br />  <br />                                                                                                                             Page 6 18 <br />                                                                                                       (page number citation purposes) <br />  BMC Genomics 2008, 9:609                                                        http://www.biomedcentral.com/1471-2164/9/609 <br />  <br />  <br />  <br />  <br /> Figure <br /> some <br /> Survivalorganisms <br />           3upon single gene-KO/KD correlated number duplicates present distance gene in <br /> Survival single gene-KO/KD correlated number duplicates present distance the <br /> gene organisms. For E. coli, yeast worm, deconvolute set duplicates different effective fam- <br /> ily sizes (A), according distance respect sequence deleted gene nearest homolog (B). In E. <br /> coli worm, chances survival increase slightly increasing number duplicates present gene (D) increasing <br /> sequence similarity (as measured E-value). Yeast correlation effective family size survival (A), but <br /> chances survival higher two-gene families (D = 1) larger families (D &#226;&#8240;&#165; 2). For abbreviations Figure 1. <br />  <br />  <br />  <br /> the whole-genome duplication [28], focusing             We tested C genes different groups gene <br /> highly expressed genes (Additional file 1). Yeast partic-       function, finding strong biases (Additional file <br /> ularly enriched two-gene families (D = 1) buffer          1). <br /> for (Additional file 1). Figure 3A shows these <br /> distributions E. coli, yeast worm. <br />  <br />  <br />  <br />                                                                                                                         Page 7 18 <br />                                                                                                   (page number citation purposes) <br />  BMC Genomics 2008, 9:609                                                     http://www.biomedcentral.com/1471-2164/9/609 <br />  <br />  <br />  <br /> Two-gene families model buffering duplicates           tions [31], genetic interactions gene-KO phenotypes <br /> To better understand buffering duplicates, com-            various conditions. Buffering genes simi- <br /> pared properties subset duplicates         lar non-buffering genes these <br /> likely buffer other&apos;s function     measures genetic interactions, the <br /> not buffer other. In particular, analyzed two-       trends significant (Table 2). The lack similarity <br /> gene families tested single-         genetic interaction profiles buffering genes is <br /> double gene-KOs. Of course, members larger gene fam-          consistent recent findings Ihmels et al. [13] <br /> ilies buffer &#226;&#8364;&#8220; however,       authors included epistatic interactions <br /> difficult distinguish buffering genes         lethal double-KO phenotypes analysis. <br /> other functions. For two-gene families, double-KO <br /> of non-essential genes lethal, genes          Buffering non-buffering genes clear differences <br /> likely buffer other&apos;s function single-KOs, i.e.   terms transcriptional translational regulation <br /> we buffering duplicates. Despite generally low     (Table 2). Buffering genes higher mRNA protein <br /> contribution duplicates survival gene knock-          expression levels. Measures translation efficiency, e.g. <br /> out, two-gene families paramount candidates        protein length, molecular weight, Codon Adaptation <br /> buffering. If double-KO viable, reasons      Index (CAI), protein production rate, significantly <br /> presence duplicate explain viable single-      elevated buffering genes compared non-buffering <br /> KO phenotype. We pairs non-buffering duplicates.      ones (P-value &#226;&#8240;&#164; 0.05); protein degradation slightly <br />                                                                  decreased. Interestingly, measures (e. g. <br /> Amongst ~300,000 yeast gene pairs tested double-         length, CAI) significantly different between <br /> KO phenotypes tested large- small-scale screens           members buffering gene pair members <br /> [30], identified 50 two-gene families genetic            non-buffering gene pair (Additional file 1). <br /> interactions (buffering) two-gene families with <br /> a viable double-KO phenotype (non-buffering). These              We extracted orthologs buffering non-buff- <br /> two-gene families represent prime candidates compar-         ering yeast two-gene families fly, worm mouse <br /> ing characteristics buffering non-buffering dupli-        using InParanoid [32]. (None yeast genes had <br /> cates, respectively. Table 2 Additional file 1      orthologs E. coli). If buffering gene pair yeast a <br /> their properties tested genes. There      single-gene ortholog organism (without addi- <br /> are 551 two-gene families yeast            tional duplicates), expect ortholog essential <br /> have tested double-KO experiments; Addi-             &#226;&#8364;&#8220; single-gene orthologs non-buffering <br /> tional file 1 describes characteristics.                   gene pairs. If ortholog buffering two-gene family <br />                                                                  paralogs, expect essential. Indeed, <br /> Both buffering non-buffering two-gene families           buffering gene pairs enriched essential single <br /> defined E-value threshold (10-10, Methods);          orthologs compared non-buffering gene pairs, <br /> however, buffering genes significantly higher               trend weak significant to <br /> sequence identity members (P-value &lt; 0.05;           small numbers dataset (Table 5, P-value = 0.19; <br /> Table 2). Buffering genes conserved           Additional file 1, P-value = 0.07). There exam- <br /> non-buffering genes, i.e. slower rates evolution         ples essential single orthologs buffering gene pairs: <br /> and orthologs organisms.                             HMG1 HMG2 isozymes HMG-CoA reductase <br />                                                                  yeast (Table 3) double KO phenotype is <br /> We examined functional similarity genes           lethal. The genes ortholog worm (F08F8. 2) <br /> the sets pairs, testing buffering duplicates      mouse (HMG-CoAR, MGI96159) both <br /> more similar function non-buffering dupli-         embryonic lethal KO/KD phenotypes. SSF1 SSF2 <br /> cates. We genes buffering two-gene families       yeast proteins required ribosomal large subunit <br /> mostly identical function descriptions, descriptions         maturation (Table 3), single essential <br /> for non-buffering genes similar identical            orthologs worm (K09H9. 6, lpd-6) fly (CG5786, <br /> (Table 3, 4) &#226;&#8364;&#8220; however, finding qualitative. To     Peter Pan). <br /> quantify functional distance, measured average <br /> shortest path genes network func-            For validation, extracted 143 worm two- <br /> tional relationships [31]: buffering genes slightly          gene families tested double-RNAi knockdowns [33] <br /> shorter paths non-buffering              consist 16 pairs synthetic sick lethal (SSL) <br /> genes (not significant, Table 2), i.e. functions       phenotypes, i.e. buffering duplicates, 127 non-buffer- <br /> closer other. Other quantitative measures gene        ing duplicate gene pairs. Unfortunately, no <br /> function derived number types             experimental data available worm genes test for <br /> physical protein-protein interactions, functional interac-       measures transcriptional translational efficiency. <br />  <br />  <br />                                                                                                                      Page 8 18 <br />                                                                                                (page number citation purposes) <br />  BMC Genomics 2008, 9:609                                                           http://www.biomedcentral.com/1471-2164/9/609 <br />  <br />  <br />  <br /> Table 2: Characteristics buffering non-buffering yeast two-gene families <br />  <br />  Feature                   Source           Buffering gene       Buffering gene        Non-buffering       Non-buffering t-score <br />                                             pair &#226;&#8364;&#8220; average         pair &#226;&#8364;&#8220; count          gene pair &#226;&#8364;&#8220;    gene pair &#226;&#8364;&#8220; count <br />                                                                                             average <br />  <br />  Across genes <br />  <br />  mRNA abundance              [67]                      4.948                  91               0.906                      14       4.04* <br />  (molecules/cell) <br />  <br />  Protein abundance           [67]                     35040                   29                2116                        4       2.84 <br />  (molecules/cell) <br />  <br />  Molecular weight            [66]                   66299.9                   99             91885.0                      16       -2.33 <br />  (Da) <br />  <br />  Codon Adaptation            [66]                      0.232                  99               0.134                      16       4.97* <br />  Index <br />  <br />  Codon Bias Index            [66]                      0.187                  99               0.051                      16       5.18* <br />  <br />  Protein production          [68]                      0.632                  90               0.056                      12       3.45* <br />  rate (s-1) <br />  <br />  Proteins produced           [68]                      5.733                  85               1.388                      11       4.07* <br />  mRNA <br />  <br />  Transcription rate          [68]                      0.109                  85               0.040                      11        2.87 <br />  (s-1) <br />  <br />  Protein half-life           [69]                      108.5                  74               177.1                      13       -0.50 <br />  (min) <br />  <br />  dN/dS                       [70]                      0.056                  56               0.113                        8      -1.95 <br />  <br />  No. orthologs 14         [32]                        8.1                  94                 5.8                      15        1.52 <br />  organisms <br />  <br />  No. protein-protein         [71]                       15.2                  84                 4.3                      14       4.50* <br />  interactions <br />  <br />  Between genes <br />  <br />  Sequence similarity     BLAST output                   54.3                  50                32.5                        8      4.91* <br />  (%) <br />  <br />  Shortest path &#226;&#8364;&#8220;             [31]                       1.27                  48                1.63                        8      -1.26 <br />  Functional network <br />  <br />  Vector similarity &#226;&#8364;&#8220;         [31]                       0.15                  23                0.04                        7       2.04 <br />  Functional <br />  interactions <br />  <br />  Vector similarity &#226;&#8364;&#8220;         [30]                       0.13                  25                0.03                        8       2.01 <br />  Physical <br />  interactions <br />  <br />  Vector similarity &#226;&#8364;&#8220;     See methods                    0.01                  26                0.07                        7      -1.49 <br />  Genetic <br />  interactions <br />  <br />  <br />  <br />  <br />                                                                                                                          Page 9 18 <br />                                                                                                    (page number citation purposes) <br />  BMC Genomics 2008, 9:609                                                                     http://www.biomedcentral.com/1471-2164/9/609 <br />  <br />  <br /> Table 2: Characteristics buffering non-buffering yeast two-gene families (Continued) <br />  Vector similarity &#226;&#8364;&#8220;             [63]                         0.17                      10                     0.11                        2       0.27 <br />  KO phenotypes <br />  <br />  Worm two-gene <br />  families (subset) <br />  <br />  Length (nt)                     [66]                         1556                     254                    1359                       32        1.10 <br />  <br />  Codon Adaptation                [64]                        0.396                     254                    0.326                      32        2.46 <br />  Index <br />  <br />  dN (Ka)                   Analysis [33]                   0.34                                             0.50 <br />  <br />  The table lists selection characteristics tested sets buffering non-buffering yeast two-gene families, respectively. Also see <br />  Table 3 description data. A small number characteristics tested worm two-gene families, identified published work <br />  [33]. Due multiple hypothesis testing, t-score &gt; 3.26 considered significant adjusted P-value 0.05 (Bonferroni); significant <br />  scores marked *. An E-value &apos;0&apos; signifies E-value smaller 10-360. <br />  <br />  <br /> When calculating CAI worm sequences,                         The extremely high chances survival fly (Figure 1A) <br /> a significant bias confirming trend yeast (Table 2).                   (in part) attributed use cell line rather <br /> Buffering genes efficiently translated non-                     organism RNAi knockdowns <br /> buffering genes.                                                              instead gene deletion [35], underes- <br />                                                                               timate current technical limitations. However, in <br /> Noticeably, yeast enriched buffering gene pairs (50)                   worm, technique, RNAi-KDs, whole <br /> vs. non-buffering gene pairs (eight) compared worm                         organism produced high survival rates, much <br /> (16 143-16 = 127, respectively). This bias holds true                     higher contribution duplicates survival (see below). <br /> even regarding yeast gene pairs identified in <br /> large-scale screens: buffering non-buffering                    The low chances survival mouse likely the <br /> pairs. Previous work shown yeast enriched                     mouse dataset originating large-scale screen, <br /> buffering gene pairs originate                           individual experiments preferen- <br /> genome duplication [34]. In addition, RNAi-based                              tially targeted reported essential genes. For example, <br /> screens worms miss synthetically lethal interac-                       gene targets mouse dataset strongly enriched <br /> tions high false-negative rate                        orthologs human disease genes (OMIM data, not <br /> gene pairs non-buffering.                                         shown); dataset biased. The lack buffering by <br />                                                                               duplicate genes mouse demonstrated recently <br /> Conclusion                                                                    [18,19]; however, availability unbiased <br /> Our study provides systematic semi-quantitative                         large-scale essentiality screen mouse results may <br /> assessment essentiality gene duplication                        refined. <br /> eleven prokaryotic eukaryotic organisms revealing a <br /> heterogeneous picture. To best knowledge,                     The degree gene essentiality (or degree survival) can <br /> is organism-wide comparison.                                   influenced experimental technique def- <br />                                                                               inition essentiality used. In contrast, duplicates <br /> Chances survival gene deletion high                       contribute survival gene loss, effect <br /> most organisms (&gt;80%), i.e. essential                      detectable irrespective number essen- <br /> genes (Figure 1A). We observe variation survival                      tial non-essential genes identified (provided the <br /> that explained experimental differences                          selection unbiased). In words, expect buffering <br /> alone. The bacteria dataset analyzed                         duplicates dependent technical differ- <br /> come different experimental backgrounds (i.e. inser-                     ences essentiality alone. We introduced statistical <br /> tion vs. deletion, population vs. clonal study, Table 1). For                 tests assess significance buffering duplicates <br /> example, screens mixed populations random gene                        (Figure 2). A small P-value implies duplicates sig- <br /> insertions identify essential genes clonal stud-                    nificantly enriched non-essential genes com- <br /> ies, e.g. H. pylori, H. influenzae, M. tuberculosis vs. P.                pared random vice versa. Thus, example, H. <br /> aeruginosa, B. subtilis E. coli (Table 1); however,                 pylori genes duplicates (Figure 1B), but <br /> is general trend.                                                          duplicates exhibit significant contribution sur- <br />                                                                               vival <span id='am-319' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-320' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-321' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-322' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-323' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-324' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-325' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-326' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-327' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-328' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-330' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-331' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-332' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-333' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-334' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-335' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-337' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-338' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-339' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-341' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-342' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-344' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-345' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-347' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-348' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-349' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-350' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-351' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-352' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-353' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-354' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-355' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-356' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-357' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-358' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-359' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-362' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-363' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-364' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-365' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-366' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-367' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-368' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-369' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-370' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-372' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-373' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-374' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-375' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-376' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-378' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-380' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-381' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-382' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-383' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-384' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-385' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-386' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-388' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-389' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-390' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-391' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-392' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-393' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-394' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-395' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-396' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span>gene knockout</span> (Figure 2). Likewise, B. subtilis <br />                                                                               E. coli similar degrees gene essentiality (one <br />  <br />  <br />                                                                                                                                        Page 10 18 <br />                                                                                                                   (page number citation purposes) <br />  BMC Genomics 2008, 9:609                                                           http://www.biomedcentral.com/1471-2164/9/609 <br />  <br />  <br />  <br /> Table 3: Examples yeast buffering two-gene families (SSL double-KO phenotype) <br />  <br />  Name                Function                           Name           Function                         E-value Sequence identity (%) <br />  <br />  YIL159W BNR1        Formin, nucleates              YNL271C BNI1   Formin, nucleates              1E-82                            32 <br />                      formation linear actin                         formation linear actin <br />                      filaments, involved cell                       filaments, involved cell <br />                      processes budding                         processes budding <br />                      mitotic spindle                               mitotic spindle <br />                      orientation require                     orientation require the <br />                      formation polarized actin                      formation polarized actin <br />                      cables, functionally redundant                    cables, functionally redundant <br />                      BNI1                                         BNR1 <br />  <br />  YML075C HMG1        One isozymes             YLR450W HMG2   One isozymes                  0                           62 <br />                      HMG-CoA reductase                            HMG-CoA reductase that <br />                      catalyzes conversion                       convert HMG-CoA to <br />                      HMG-CoA mevalonate,                            mevalonate, rate-limiting <br />                      rate-limiting step                  step sterol biosynthesis; <br />                      sterol biosynthesis; localizes                    overproduction induces <br />                      nuclear envelope;                          assembly peripheral ER <br />                      overproduction induces                        membrane arrays short <br />                      formation karmellae                            nuclear-associated membrane <br />                                                                        stacks <br />  <br />  YKR067W GPT2        Glycerol-3-phosphate               YBL011W SCT1   Glycerol 3-phosphate/             2E-118                            36 <br />                      acyltransferase located                        dihydroxyacetone phosphate <br />                      lipid particles                      dual substrate-specific sn-1 <br />                      ER; involved stepwise                      acyltransferase the <br />                      acylation glycerol-3-                          glycerolipid biosynthesis <br />                      phosphate                                     pathway, prefers 16-carbon <br />                      dihydroxyacetone,                       fatty acids, similar Gpt2p, <br />                      intermediate steps lipid                       gene constitutively <br />                      biosynthesis                                      transcribed <br />  <br />  YEL042W GDA1        Guanosine diphosphatase            YER005W YND1   Apyrase wide substrate        5E-28                            27 <br />                      located Golgi, involved                    specificity, involved in <br />                      transport GDP-                          preventing inhibition of <br />                      mannose Golgi lumen                      glycosylation hydrolyzing <br />                      converting GDP GMP                          nucleoside tri- and <br />                      mannose transferred                      diphosphates are <br />                      substrate                                     inhibitors of <br />                                                                        glycotransferases; partially <br />                                                                        redundant Gda1p <br />  <br />  YKL020C SPT23       ER membrane protein                YIR033W MGA2   ER membrane protein               1E-163                            37 <br />                      involved regulation                         involved regulation of <br />                      OLE1 transcription, acts                     OLE1 transcription, acts with <br />                      homolog Mga2p; inactive ER                        homolog Spt23p; inactive ER <br />                      form dimerizes                            form dimerizes one <br />                      subunit activated                      subunit activated by <br />                      ubiquitin/proteasome-                             ubiquitin/proteasome- <br />                      dependent processing                              dependent processing <br />                      followed nuclear targeting                     followed nuclear targeting <br />  <br />  YGR038W ORM1        Evolutionarily conserved           YLR350W ORM2   Evolutionarily conserved           3E-68                            72 <br />                      protein similarity                        protein similarity to <br />                      Orm2p, required                               Orm1p, required for <br />                      resistance agents                         resistance agents that <br />                      induce unfolded protein                       induce unfolded protein <br />                      response; human ortholog                       response; human ortholog is <br />                      located endoplasmic                        located endoplasmic <br />                      reticulum                                         reticulum <br />  <br />  <br />  <br />  <br />                                                                                                                              Page 11 18 <br />                                                                                                         (page number citation purposes) <br />  BMC Genomics 2008, 9:609                                                                        http://www.biomedcentral.com/1471-2164/9/609 <br />  <br />  <br /> Table 3: Examples yeast buffering two-gene families (SSL double-KO phenotype) (Continued) <br />  YER087C-B SBH1 Beta subunit Sec61p ER              YER019C-A SBH2 Ssh1p-Sss1p-Sbh2p complex                     8E-19                           55 <br />                 translocation complex                                     component, involved in <br />                 (Sec61p-Sss1p-Sbh1p);                                     protein translocation the <br />                 involved protein                                       endoplasmic reticulum <br />                 translocation the <br />                 endoplasmic reticulum; <br />                 interacts exocyst <br />                 complex <br />  <br />  YHL003C LAG1            Ceramide synthase                 YKL008C LAC1            Ceramide synthase                  6E-169                            73 <br />                          component, involved                                    component, involved in <br />                          synthesis ceramide                                synthesis ceramide from <br />                          C26(acyl)-coenzyme A                                  C26(acyl)-coenzyme A and <br />                          dihydrosphingosine                                     dihydrosphingosine or <br />                          phytosphingosine, functionally                            phytosphingosine, <br />                          equivalent Lac1p                                       functionally equivalent to <br />                                                                                    Lag1p <br />  <br />  YHR066W SSF1            Constituent 66S pre-           YDR312W SSF2            Protein required                     0                           94 <br />                          ribosomal particles, required                             ribosomal large subunit <br />                          ribosomal large subunit                               maturation, functionally <br />                          maturation; functionally                                  redundant Ssf1p <br />                          redundant Ssf2p <br />  <br />  YPR159W KRE6            Protein required beta-1,6     YGR143W SKN1            Protein involved                      0                           68 <br />                          glucan biosynthesis; putative                             sphingolipid biosynthesis; <br />                          beta-glucan synthase; appears                             type II membrane protein <br />                          functionally redundant                               similarity Kre6p <br />                          Skn1p <br />  <br />  Two-gene families phenotypes double-KOs good model buffering gene duplicates. We distinguish &apos;buffering genes&apos; <br />  (50), i. e. gene pairs resulting synthetic sick lethal (SSL) phenotype double-KO; &apos;non-buffering genes&apos; (eight), i. e. gene pairs that <br />  result viable phenotype double gene-KO, unlikely buffer single gene-KO. <br />  Tables 3 4 list functions subset buffering non-buffering gene pairs, respectively, pair row. The buffering <br />  gene pairs table originate large-scale screens non-buffering pairs table 4. The remaining 40 buffering gene pairs <br />  originate small-scale screens, listed Additional file 2. The descriptions functions taken SGD [66]. Buffering genes <br />  (this table) described having identical functions non-buffering genes (Table 4). <br />  <br /> examined insertion, knockout experi-                            explained low number duplicate genes <br /> ments), similar fractions duplicate genes,                      lack condition-specific dispensability genes <br /> different contributions duplicates survival.                        boost survival rates normal conditions [12]. <br />                                                                                 However, reasoning apply H. pylori and <br /> Duplicates significantly positively contribute sur-                      H. influenzae genome sizes similar M. geni- <br /> vival organisms, noticeable                      talium restricted living conditions, much <br /> effects (&gt;5%; H. pylori, M. tuberculosis, P. aeru-                  higher survival rates different buffering capacities of <br /> ginosa, E. coli, yeast, worm; Figure 2). Given dupli-                      duplicates. Mouse represents exception analysis <br /> cates make 80% eukaryotic genomes (Figure 1B),                         having relatively low survival rates (Figure 1A), a <br /> the small contribution surprising points domi-                        higher ratio duplicates vs. singletons organ- <br /> nant roles buffering processes, rerouting                      isms (Figure 1B), negative contribution duplicates <br /> metabolic flux (see ref. [9] example).                                   survival (Figure 2). As explained above, conclusions in <br />                                                                                 mouse refined later. <br /> Buffering duplicates uncorrelated organismal <br /> complexity. Buffering capacity varies widely bac-                       Next examined gene characteristics been <br /> teria eukaryotes, accounting differ-                          suggested influence buffering capacity. For example, we <br /> ences experimental approaches (Table 1). M.                                  expect duplicates high sequence proximity <br /> genitalium, H. influenzae, B. subtilis, fly mouse                      (measured E-value) likely buffer loss <br /> low negative contributions duplicates buffer-                     function duplicates diverged their <br /> ing; H. pylori, yeast worm highest. M. geni-                       sequence. Similarly, expect genes many <br /> talium parasite small range host- tissue-                     duplicates (large gene families) likely be <br /> specific living conditions [36] small genome                         buffered loss function genes small families. <br /> [37](Figure 1). Its low rate survival gene-KO                           Both expectations fulfilled organ- <br />  <br />  <br />                                                                                                                                           Page 12 18 <br />                                                                                                                      (page number citation purposes) <br />  BMC Genomics 2008, 9:609                                                             http://www.biomedcentral.com/1471-2164/9/609 <br />  <br />  <br />  <br /> Table 4: Examples yeast non-buffering two-gene families (viable phenotype double-KO) <br />  <br />  Name               Function                          Name              Function                          E-value Sequence identity (%) <br />  <br />  YJR075W HOC1       Alpha-1,6-mannosyltransferase     YGL038C OCH1      Mannosyltransferase          2E-40                            27 <br />                     involved cell wall mannan                        cis-Golgi apparatus, initiates <br />                     biosynthesis; subunit                          polymannose outer chain <br />                     Golgi-localized complex                        elongation N-linked <br />                     contains Anp1p, Mnn9p,                         oligosaccharides of <br />                     Mnn11p, Mnn10p;                                 glycoproteins <br />                     identified suppressor a <br />                     cell lysis sensitive pkc1-371 <br />                     allele <br />  <br />  YGR188C BUB1       Protein kinase forms       YJL013C MAD3      Component spindle-           2E-50                            35 <br />                     complex Mad1p                              assembly checkpoint complex, <br />                     Bub3p crucial                        delays onset of <br />                     checkpoint mechanism                                anaphase cells defects <br />                     required prevent cell cycle                      mitotic spindle assembly; <br />                     progression anaphase                        interacts physically the <br />                     presence spindle                             spindle checkpoint proteins <br />                     damage, associates                             Bub3p Mad2p <br />                     centromere DNA Skp1p <br />  <br />  YHR119W SET1       Histone methyltransferase,        YJL168C SET2      Histone methyltransferase           2E-16                            30 <br />                     subunit COMPASS                              role transcriptional <br />                     (Set1C) complex                               elongation, methylates lysine <br />                     methylates histone H3                            residue histone H3; <br />                     lysine 4; required                               associates C-terminal <br />                     transcriptional silencing near                      domain Rpo21p; histone <br />                     telomeres silent                         methylation activity is <br />                     mating type loci; contains                        regulated phosphorylation <br />                     SET domain                                          status Rpo21p <br />  <br />  YDR528W HLR1       Protein involved regulation    YCL051W LRE1      Protein involved control      5E-34                            34 <br />                     cell wall composition                        cell wall structure stress <br />                     integrity response                           response; inhibits Cbk1p <br />                     osmotic stress;                                     protein kinase activity; <br />                     overproduction suppresses                         overproduction confers <br />                     lysis sensitive PKC <span id='am-163' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-164' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span>mutation</span>;                       resistance cell-wall <br />                     similar Lre1p,                             degrading enzymes <br />                     functions antagonistically to <br />                     protein kinase A <br />  <br />  YJR131W MNS1       Alpha-1,2-mannosidase             YHR204W MNL1 Alpha mannosidase-like                   9E-25                            25 <br />                     involved ER quality control;                protein endoplasmic <br />                     catalyzes removal                   reticulum required for <br />                     mannose residue                           degradation glycoproteins <br />                     Man9GlcNAc produce                        processing N- <br />                     single isomer Man8GlcNAc                    linked oligosaccharides <br />                     N-linked oligosaccharide <br />                     biosynthesis; integral ER <br />                     membrane <br />  <br />  YDR420W HKR1 Serine/threonine rich cell        YGR014W MSB2            Mucin family member          6E-12                            29 <br />               surface protein contains                             head Cdc42p- MAP <br />               EF hand motif; involved                             kinase-dependent filamentous <br />               regulation cell wall beta-                         growth signaling pathway; also <br />               1,3 glucan synthesis bud                              functions osmosensor in <br />               site selection; overexpression                            parallel Sho1p- <br />               confers resistance                                     mediated pathway; potential <br />               Hansenula mrakii killer toxin,                            Cdc28p substrate <br />               HM-1 <br />  <br />  <br />  <br />  <br />                                                                                                                                Page 13 18 <br />                                                                                                           (page number citation purposes) <br />  BMC Genomics 2008, 9:609                                                                      http://www.biomedcentral.com/1471-2164/9/609 <br />  <br />  <br /> Table 4: Examples yeast non-buffering two-gene families (viable phenotype double-KO) (Continued) <br />  YML061C PIF1            DNA helicase involved          YHR031C RRM3          DNA helicase involved            5E-102                            40 <br />                          telomere formation                                  rDNA replication Ty1 <br />                          elongation; acts catalytic                         transposition; relieves <br />                          inhibitor telomerase;                           replication fork pauses at <br />                          plays role repair                              telomeric regions; structurally <br />                          recombination                                        functionally related to <br />                          mitochondrial DNA                                       Pif1p <br />  <br />  YJL092W HPR5            DNA helicase DNA-             YOL095C HMI1          Mitochondrial inner                  2E-18                            21 <br />                          dependent ATPase involved                            membrane localized ATP- <br />                          DNA repair, required                                dependent DNA helicase, <br />                          proper timing commitment                             required maintenance <br />                          meiotic recombination                            mitochondrial genome; <br />                          transition Meiosis I                           required for <br />                          Meiosis II; potential Cdc28p                         mitochondrial transcription; <br />                          substrate                                               homology E. coli <br />                                                                                  helicase uvrD <br />  <br />  See Table 3 description. Tables 3 4 list functions subset buffering non-buffering gene pairs, respectively, one <br />  pair row. The descriptions functions taken SGD [66]. Buffering genes (Table 3) described having identical <br />  functions non-buffering genes (this table). <br />  <br />  <br /> isms (Table 1), e.g. thoroughly studied                        similarity weak indicator buffering capac- <br /> organisms yeast worm, others.                                   ity duplicates. <br />  <br /> Related sequence similarity function,                      The single best correlate buffering capacity gene <br /> dissimilar buffering duplicates naively                           duplicates (identified study) expression level. <br /> expected, measured terms expression regula-                         Genes high expression levels tend dupli- <br /> tion [20] genetic interactions [13]. When evaluating                       cates, duplicates likely buffer for <br /> function similarity terms verbal descriptions, short-                    loss gene&apos;s function. (Note subtle difference <br /> est path length network functional relationships,                      observations.) The trend holds true for <br /> and terms similarity KO-phenotype                           organisms positive buffering capacity (except for <br /> physical interaction vectors, buffering genes slightly                    M. tuberculosis) different measures expression <br /> (but significantly) similar func-                    levels (Additional file 1). For example, highly expressed <br /> tion non-buffering genes (Table 2). Thus, function                        genes E. coli, C increases 23%. Likewise, buffering <br />  <br /> Table 5: Orthologs yeast buffering non-buffering two-gene families <br />  <br />                                                                                                       Buffering pairs            Non-buffering pairs <br />  <br />  Single-gene ortholog fly, worm mouse (no duplicate) <br />  <br />  - essential                                                                                                         11                                   0 <br />  <br />  - non-essential                                                                                                     13                                   3 <br />  <br />  Multi-gene orthologs fly, worm mouse (with duplicates) <br />  <br />  - duplicates essential                                                                                           1                                   0 <br />  <br />  - duplicates non-essential                                                                                       6                                   0 <br />  <br />  Other (mix information) <br />  <br />                                                                                                                      24                                   6 <br />  <br />  This table lists number instances buffering non-buffering yeast two-gene families, respectively, single multiple <br />  orthologs fly, worm mouse KO-phenotype known. Also Table 3 description data. Orthologs are <br />  divided single-gene orthologs (no additional homologs organism) multi-gene orthologs (additional paralogs). Single- multi-gene <br />  orthologs essential non-essential organism. <br />  <br />  <br />                                                                                                                                          Page 14 18 <br />                                                                                                                     (page number citation purposes) <br />  BMC Genomics 2008, 9:609                                                      http://www.biomedcentral.com/1471-2164/9/609 <br />  <br />  <br />  <br /> two-gene families yeast higher mRNA protein           genes identified 14 analyses non-essential <br /> abundance non-buffering two-gene families, higher            yeast genes [42], resulting ~300,000 tested pairs. <br /> transcription translation rates smaller protein <br /> degradation rates (Table 2).                                      For worm extracted data large-scale double <br />                                                                   KD screens [26,61], comprise 52781 tested gene- <br /> In sum, buffering gene duplicates plays signifi-        pairs 3927 genetic interactions. Another study in <br /> cant visible role robustness gene loss          worm specifically targeted two-gene families single <br /> some organisms others. Factors influencing             ortholog yeast [33], used pairs investi- <br /> such buffering are, decreasing order approximate            gate properties two-gene families. <br /> importance, gene expression levels, sequence distance <br /> between duplicates, number duplicates available            Homology estimation <br /> per gene, gene&apos;s function type organism        We measured similarity sequences using a <br /> its lifestyle. Such ranking holds true despite differences     BLAST all-against-all search [29], required E-value <br /> experimental approaches. The lack consistency           &lt; 10-10 genes predicted homologs. This E- <br /> organisms, lack strong correlates low extent buff-      value threshold established yeast adjusted <br /> ering duplicates suggests buffering duplicates      accordingly organisms different genome size, <br /> indeed merely by-product processes. Genes              e.g. M. genitatlium (10-9) worm (3.0*10-10). This <br /> with high expression levels likely essential       threshold identified 609 two-gene families yeast. We <br /> [38] increased duplicate retention rates [12,23].        tested methods homology prediction <br /> These duplicates likely function amplify gene dos-        including different E-value thresholds, E-value-independ- <br /> age [22], supported tendency co-          ent methods use InParanoid [32], results <br /> expressed [13]. Our analysis shows relatively        qualitatively identical discussed (Additional <br /> few cases duplicates serve backup loss        file 1). <br /> gene function. <br />                                                                   Estimates gene expression levels <br /> Methods                                                           As surrogate gene expression levels, calculated the <br /> Data sets                                                         Codon Bias Index (CBI) gene using CodonW <br /> We obtained amino acid sequences genomes              server [62], standard settings parameters the <br /> (Mycoplasma genitalium; Bacillus subtilis; Helicobacter pylori;   respective organism. We calculated Codon Adap- <br /> Haemophilus influenzae; Mycobacterium tuberculosis; Pseu-         tation Index (CAI). However, requires reference <br /> domonas aeruginosa; Escherichia coli; Saccharomyces cerevi-       dataset expressed genes (which availa- <br /> siae (yeast); Caenorhabditis elegans (worm); Drosophila           ble) consider CAI appropriate measure than <br /> melanogaster (fly); Mus musculus (mouse)) collec-          CBI. Both measures expected work in <br /> tion SUPERFAMILY database [39]. Information             multi-cellular organisms tissue-specific expression <br /> gene essentiality (lethal phenotypes single gene-KO          captured sequence features. <br /> or KD) taken publications [25,35,36,40-46].              For validation, extracted literature experi- <br /> Table 1 provides overview number genes            mental expression data organisms H. pylori. <br /> tested organism (background set) number           Results CAI experimental expression data in <br /> genes identified essential. The table describes briefly     Additional file 1. For results Figure 1 2, we <br /> the experimental strategy, described publications       rank-ordered CBI values genome and <br /> and SEED database http://theseed.uchicago.edu. All         selected subsets genes highest lowest CBI; <br /> screens conducted rich medium                sizes subsets varied according organism&apos;s <br /> organisms fly (cell line). For mouse, data          genome size. See Additional file 1 details. <br /> ~4,000 individual knockout experiments obtained <br /> from Mouse Genome Database [47].                              Two-gene families characteristics <br />                                                                   In yeast, 50 two-gene families identified buffering <br /> To-date, large-scale double-KO/KD data available          (SSL phenotype) two-gene families non-buff- <br /> for yeast worm. For yeast compiled addition          ering (viable phenotype). The buffering pairs consist of <br /> the original data published Tong et al. [16,48] 13 data-       pairs identified 14 large-scale double-KO <br /> sets identified &apos;systematic screens&apos; BioGRID data-      screens (see above), 42 additional pairs identified in <br /> <span id='am-397' about='xsp:base' typeof='owl:Thing'>base</span> [30,49-60]. In parsimonious approach,              small-scale experiments listed BioGRID [30]). The <br /> included data lethal phenotypes double-KOs           non-buffering pairs originate pairs tested 14 large- <br /> study epistatic interactions. To calculate       scale screens viable phenotypes. Table <br /> background set tested gene pairs, paired 204 bait       2 describes characteristics members a <br />                                                                   gene family characteristics individual genes, aver- <br />  <br />  <br />                                                                                                                     Page 15 18 <br />                                                                                                (page number citation purposes) <br />  BMC Genomics 2008, 9:609                                                             http://www.biomedcentral.com/1471-2164/9/609 <br />  <br />  <br />  <br /> aged set. For vector comparisons,              Authors&apos; contributions <br /> constructed binary vectors (1 = observation, 0 = obser-         KH conducted experiments, analyzed results and <br /> vation) based networks functional interactions [31],         wrote paper. EMM provided valuable input sup- <br /> genetic interactions (see description datasets above),          port stages project. CV initiated guided the <br /> physical interactions (extracted BioGRID [30]),           project, conducted experiments, analyzed <br /> single gene-KO phenotypes [63]. The similarity             results wrote paper. All authors read and <br /> two vectors measured percentage shared posi-          approved final manuscript. <br /> tive interactions (Jaccard Index). More results Addi- <br /> tional file 1.                                                     Additional material <br />  <br /> As control effects WGD genes, com- <br /> pared characteristics 609 yeast two-gene fami- <br />                                                                         Additional file 1 <br />                                                                         Supplementary Notes. Additional figures comments analyses. <br /> lies split 108 501 two-gene families                  Click file <br /> without evidence origin WGD [28], respec-              [http://www.biomedcentral.com/content/supplementary/1471- <br /> tively (Additional file 1). As control, extracted            2164-9-609-S1.pdf] <br /> the 143 worm two-gene families, identified <br /> and tested Tischler et al. [33] calculated codon                 Additional file 2 <br /> adaptation indices [64](Additional file 1). Results                Supplementary Data. Data yeast gene pairs collected anal- <br /> these controls consistent yeast                 yses. <br />                                                                         Click file <br /> analysis.                                                               [http://www.biomedcentral.com/content/supplementary/1471- <br />                                                                         2164-9-609-S2.xls] <br /> We used FunSpec server [65] SGD [66] yeast <br /> protein function annotation. The SUPERFAMILY database <br /> [39] used annotation ribosomal proteins in <br /> yeast. Genes originating whole-genome duplica-            Acknowledgements <br /> tion taken directly published paper [28].            We grateful E Levy useful discussions. We also <br /> Characteristics described Table 2 obtained         thank J Pereira-Leal, M Tsechansky, SL Wong help various <br /> sources quoted table Additional file 1. For          stages project. CV acknowledges support International <br /> the ortholog analysis described Table 5, extracted           Human Frontier Science Program. EMM acknowledges support NSF, <br /> information InParanoid [32], mapped                  NIH, Welch (F15-15) Packard Foundation. <br /> against gene essentiality data described above. Infor- <br /> mation yeast two-gene families presented Addi- <br />                                                                    References <br />                                                                    1.      Hartman JL, Garvik B, Hartwell L: Principles buffering of <br /> tional file 2.                                                             genetic variation. Science 2001, 291(5506):1001-1004. <br />                                                                    2.      Pal C, Papp B, Lercher MJ: An integrated view protein evolu- <br />                                                                            tion. Nat Rev Genet 2006, 7(5):337-348. <br /> Abbreviations                                                      3.      Wilkins AS: Canalization: molecular genetic perspective. <br /> CAI: Codon Adaptation Index; CBI: Codon Bias Index; D:                     Bioessays 1997, 19(3):257-262. <br /> effective gene family size (number additional gene              4.      Tautz D: Redundancies, development flow informa- <br />                                                                            tion. Bioessays 1992, 14(4):263-266. <br /> duplicates); E-value: expectation value; KD: knockdown;            5.      Ohno S: Evolution Gene Duplication. New York: Springer- <br /> KO: knockout; MIPS: Munich Information Center Pro-                     Verlag; 1970. <br />                                                                    6.      Wolfe KH, Li WH: Molecular evolution meets genomics <br /> tein Sequences; P(S): probability survival single-                 revolution. Nat Genet 2003, 33(Suppl):255-265. <br /> or double gene-KO KD; R2: squared Pearson correlation           7.      Lynch M, Katju V: The altered evolutionary trajectories gene <br /> coefficient; SGA: Synthetic Genetic Array; SSL: synthetic                  duplicates. Trends Genet 2004, 20(11):544-549. <br />                                                                    8.      Wagner A: Robustness mutations genetic networks <br /> sick lethal (mutant); SGD: Saccharomyces Genome                         yeast. Nat Genet 2000, 24(4):355-361. <br /> Database; WGD: whole-genome duplication.                           9.      Hartman JLt: Buffering deoxyribonucleotide pool homeos- <br />                                                                            tasis threonine metabolism. Proc Natl Acad Sci USA 2007, <br />                                                                            104(28):11700-11705. <br /> Organisms: M. genitalium: Mycoplasma genitalium; H.                10.     Giaever G, Chu AM, Ni L, Connelly C, Riles L, Veronneau S, Dow S, <br /> pylori: Helicobacter pylori; H. influenzae: Haemophilus influ-             Lucau-Danila A, Anderson K, Andre B, et al.: Functional profiling <br />                                                                            Saccharomyces cerevisiae genome. Nature 2002, <br /> enzae; M. tuberculosis: Mycobacterium tuberculosis; Paer:                  418(6896):387-391. <br /> Pseudomonas aeruginosa; B. subtilis: Bacillus subtilis; E. coli:   11.     Gu Z, Steinmetz LM, Gu X, Scharfe C, Davis RW, Li WH: Role of <br /> Escherichia coli; S. cerevisiae: Saccharomyces cerevisiae                  duplicate genes genetic robustness null mutations. <br />                                                                            Nature 2003, 421(6918):63-66. <br /> (yeast); C. elegans: Caenorhabditis elegans (worm); D. mela-       12.     Papp B, Pal C, Hurst LD: Metabolic network analysis the <br /> nogaster: Drosophila melanogaster (fly); M. musculus: Mus                  causes evolution enzyme dispensability yeast. <br /> musculus (mouse).                                                          Nature 2004, 429(6992):661-664. <br />                                                                    13.     Ihmels J, Collins SR, Schuldiner M, Krogan NJ, Weissman JS: Backup <br />                                                                            redundancy: genetic interactions reveal cost of <br />                                                                            duplicate gene loss. Mol Syst Biol 2007, 3:86. <br />  <br />  <br />  <br />                                                                                                                                Page 16 18 <br />                                                                                                           (page number citation purposes) <br />  BMC Genomics 2008, 9:609                                                                      http://www.biomedcentral.com/1471-2164/9/609 <br />  <br />  <br />  <br /> 14.   Blank LM, Kuepfer L, Sauer U: Large-scale 13C-flux analysis             39.   Wilson D, Madera M, Vogel C, Chothia C, Gough J: The SUPER- <br />       reveals mechanistic principles metabolic network robust-                   FAMILY database 2007: families functions. Nucleic Acids <br />       ness null mutations yeast. Genome Biol 2005, 6(6):R49.                  Res 2007:D308-313. <br /> 15.   Kuepfer L, Sauer U, Blank LM: Metabolic functions duplicate          40.   Kobayashi K, Ehrlich SD, Albertini A, Amati G, Andersen KK, Arnaud <br />       genes Saccharomyces cerevisiae.               Genome Res 2005,             M, Asai K, Ashikaga S, Aymerich S, Bessieres P, et al.: Essential Bacil- <br />       15(10):1421-1430.                                                             lus subtilis genes. Proc Natl Acad Sci USA 2003, 100(8):4678-4683. <br /> 16.   Tong AH, Lesage G, Bader GD, Ding H, Xu H, Xin X, Young J, Berriz       41.   Gerdes SY, Scholle MD, Campbell JW, Balazsi G, Ravasz E, Daugherty <br />       GF, Brost RL, Chang M, et al.: Global mapping yeast                    MD, Somera AL, Kyrpides NC, Anderson I, Gelfand MS, et al.: Exper- <br />       genetic interaction network. Science 2004, 303(5659):808-813.                 imental determination level analysis essential <br /> 17.   Wong SL, Zhang LV, Tong AH, Li Z, Goldberg DS, King OD, Lesage                genes Escherichia coli MG1655.                    J Bacteriol 2003, <br />       G, Vidal M, Andrews B, Bussey H, et al.: Combining biological net-            185(19):5673-5684. <br />       works predict genetic interactions. Proc Natl Acad Sci USA           42.   Winzeler EA, Liang H, Shoemaker DD, Davis RW: Functional anal- <br />       2004, 101(44):15682-15687.                                                    ysis yeast genome precise deletion parallel phe- <br /> 18.   Liang H, Li WH: Gene essentiality, gene duplicability pro-                notypic characterization.            Novartis Found Symp 2000, <br />       tein connectivity human mouse. Trends Genet 2007,                      229:105-109. discussion 109&#226;&#8364;&#8220;111 <br />       23(8):375-378.                                                          43.   Salama NR, Shepherd B, Falkow S: Global transposon mutagene- <br /> 19.   Liao BY, Zhang J: Mouse duplicate genes essential sin-              sis essential gene analysis Helicobacter pylori. J Bacte- <br />       gletons. Trends Genet 2007, 23(8):378-381.                                    riol 2004, 186(23):7926-7935. <br /> 20.   Kafri R, Bar-Even A, Pilpel Y: Transcription control reprogram-         44.   Liberati NT, Urbach JM, Miyata S, Lee DG, Drenkard E, Wu G, Vil- <br />       ming genetic backup circuits. Nat Genet 2005, 37(3):295-299.               lanueva J, Wei T, Ausubel FM: An ordered, nonredundant library <br /> 21.   He X, Zhang J: Higher duplicability important genes                Pseudomonas aeruginosa strain PA14 transposon inser- <br />       yeast genomes. Mol Biol Evol 2006, 23(1):144-151.                             tion mutants. Proc Natl Acad Sci USA 2006, 103(8):2833-2838. <br /> 22.   Nowak MA, Boerlijst MC, Cooke J, Smith JM: Evolution genetic         45.   Sassetti CM, Boyd DH, Rubin EJ: Genes required mycobacte- <br />       redundancy. Nature 1997, 388(6638):167-171.                                   rial growth defined high density mutagenesis. Mol Microbiol <br /> 23.   Seoighe C, Wolfe KH: Yeast genome evolution post-                      2003, 48(1):77-84. <br />       genome era. Curr Opin Microbiol 1999, 2(5):548-554.                     46.   Akerley BJ, Rubin EJ, Novick VL, Amaya K, Judson N, Mekalanos JJ: A <br /> 24.   Conant GC, Wagner A: Duplicate genes robustness                        genome-scale analysis identification genes required for <br />       transient gene knock-downs Caenorhabditis elegans. Proc                    growth survival Haemophilus influenzae. Proc Natl Acad <br />       Biol Sci 2004, 271(1534):89-96.                                               Sci USA 2002, 99(2):966-971. <br /> 25.   Kamath RS, Fraser AG, Dong Y, Poulin G, Durbin R, Gotta M, Kana-        47.   Eppig JT, Blake JA, Bult CJ, Kadin JA, Richardson JE: The mouse <br />       pin A, Le Bot N, Moreno S, Sohrmann M, et al.: Systematic func-               genome database (MGD): new features facilitating model <br />       tional analysis Caenorhabditis elegans genome using                    system. Nucleic Acids Res 2007:D630-637. <br />       RNAi. Nature 2003, 421(6920):231-237.                                   48.   Tong AH, Evangelista M, B PA, Xu H, Bader GD, Page N, Robinson M, <br /> 26.   Lehner B, Crombie C, Tischler J, Fortunato A, Fraser AG: System-              Raghibizadeh S, Hogue CW, Bussey H, et al.: Systematic genetic <br />       atic mapping genetic interactions Caenorhabditis ele-                   analysis ordered arrays yeast deletion mutants. Sci- <br />       gans identifies common modifiers diverse signaling                         ence 2001, 294(5550):2364-2368. <br />       pathways. Nat Genet 2006, 38(8):896-903.                                49.   Pan X, P Y, Yuan DS, Wang X, Bader JS, Boeke JD: A DNA integrity <br /> 27.   Wolfe KH, Shields DC: Molecular evidence ancient dupli-                network yeast Saccharomyces cerevisiae. Cell 2006, <br />       cation entire yeast genome.                     Nature 1997,           124(5):1069-1081. <br />       387(6634):708-713.                                                      50.   Krogan NJ, Keogh MC, Datta N, Sawa C, Ryan OW, Ding H, Haw RA, <br /> 28.   Kellis M, Birren BW, Lander ES: Proof evolutionary analysis               Pootoolal J, Tong AH, Canadien V, et al.: A Snf2 family ATPase <br />       ancient genome duplication yeast Saccharomyces                      complex required recruitment histone H2A vari- <br />       cerevisiae. Nature 2004, 428(6983):617-624.                                   ant Htz1. Molecular Cell 2003, 12(6):1565-1576. <br /> 29.   Altschul SF, Gish W, Miller W, Myers EW, Lipman DJ: Basic local         51.   Lesage G, Shapiro J, Specht CA, Sdicu AM, Menard P, Hussein S, Tong <br />       alignment search tool. J Mol Biol 1990, 215(3):403-410.                       AH, Boone C, Bussey H: An interactional network genes <br /> 30.   Stark C, Breitkreutz BJ, Reguly T, Boucher L, Breitkreutz A, Tyers M:         involved chitin synthesis Saccharomyces cerevisiae. <br />       BioGRID: general repository interaction datasets.                       BMC Genet 2005, 6(1):8. <br />       Nucleic Acids Res 2006:D535-539.                                        52.   Daniel JA, Keyes BE, Ng YP, Freeman CO, Burke DJ: Diverse func- <br /> 31.   Lee I, Date SV, Adai AT, Marcotte EM: A probabilistic functional              tions spindle assembly checkpoint genes Saccharomy- <br />       network yeast genes accurate, extensive, highly                     ces cerevisiae. Genetics 2006, 172(1):53-65. <br />       modular. Science 2004, 306(5701):1555-1558.                             53.   Lesage G, Sdicu AM, Menard P, Shapiro J, Hussein S, Bussey H: Anal- <br /> 32.   Remm M, Storm CE, Sonnhammer EL: Automatic clustering                      ysis beta-1,3-glucan assembly Saccharomyces cerevisiae <br />       orthologs in-paralogs pairwise species compari-                      using synthetic interaction network altered sensitivity <br />       sons. J Mol Biol 2001, 314(5):1041-1052.                                      caspofungin. Genetics 2004, 167(1):35-49. <br /> 33.   Tischler J, Lehner B, Chen N, Fraser AG: Combinatorial RNA              54.   Zhao R, Davey M, Hsu YC, Kaplanek P, Tong A, Parsons AB, Krogan <br />       interference C. elegans reveals redundancy                    N, Cagney G, Mai D, Greenblatt J, et al.: Navigating chaperone <br />       gene duplicates maintained 80 million                    network: integrative map physical genetic interac- <br />       years evolution. Genome Biol 2006, 7(8):R69.                               tions mediated hsp90 chaperone.                       Cell 2005, <br /> 34.   Guan Y, Dunham MJ, Troyanskaya OG: Functional Analysis                     120(5):715-727. <br />       Gene Duplications Saccharomyces cerevisiae. Genetics                 55.   Friesen H, Humphries C, Ho Y, Schub O, Colwill K, Andrews B: <br />       2007, 175(2):933-943.                                                         Characterization yeast amphiphysins Rvs161p and <br /> 35.   Boutros M, Kiger AA, Armknecht S, Kerr K, Hild M, Koch B, Haas SA,            Rvs167p reveals roles Rvs heterodimer vivo. Mol <br />       Paro R, Perrimon N: Genome-wide RNAi analysis growth                       Biol Cell 2006, 17(3):1306-1321. <br />       viability Drosophila cells.                    Science 2004,     56.   Loeillet S, Palancade B, Cartron M, Thierry A, Richard GF, Dujon B, <br />       303(5659):832-835.                                                            Doye V, Nicolas A: Genetic network interactions repli- <br /> 36.   Glass JI, Assad-Garcia N, Alperovich N, Yooseph S, Lewis MR, Maruf            cation, repair nuclear pore deficiencies yeast. DNA <br />       M, Hutchison CA 3rd, Smith HO, Venter JC: Essential genes                Repair (Amst) 2005, 4(4):459-468. <br />       minimal bacterium. Proc Natl Acad Sci USA 2006, 103(2):425-430.         57.   Pan X, Yuan DS, Xiang D, Wang X, Sookhai-Mahadeo S, Bader JS, <br /> 37.   Mushegian AR, Koonin EV: A minimal gene set cellular life                 Hieter P, Spencer F, Boeke JD: A robust toolkit functional <br />       derived comparison complete bacterial genomes. Proc                     profiling yeast genome. Mol Cell 2004, 16(3):487-496. <br />       Natl Acad Sci USA 1996, 93(19):10268-10273.                             58.   Ingvarsdottir K, Krogan NJ, Emre NC, Wyce A, Thompson NJ, Emili <br /> 38.   Pal C, Papp B, Hurst LD: Genomic function: Rate evolution                  A, Hughes TR, Greenblatt JF, Berger SL: H2B ubiquitin protease <br />       gene dispensability. Nature 2003, 421(6922):496-497. dis-                 Ubp8 Sgf11 constitute discrete functional module <br />       cussion 497&#226;&#8364;&#8220;498                                                               Saccharomyces cerevisiae SAGA complex. Mol <br />                                                                                     Cell Biol 2005, 25(3):1162-1172. <br />  <br />  <br />  <br />  <br />                                                                                                                                          Page 17 18 <br />                                                                                                                     (page number citation purposes) <br />  BMC Genomics 2008, 9:609                                                                     http://www.biomedcentral.com/1471-2164/9/609 <br />  <br />  <br />  <br /> 59.   Menon BB, Sarma NJ, Pasula S, Deminoff SJ, Willis KA, Barbara KE, <br />       Andrews B, Santangelo GM: Reverse recruitment: Nup84 <br />       nuclear pore subcomplex mediates Rap1/Gcr1/Gcr2 tran- <br />       scriptional activation.         Proc Natl Acad Sci USA 2005, <br />       102(16):5749-5754. <br /> 60.   Suter B, Tong A, Chang M, Yu L, Brown GW, Boone C, Rine J: The <br />       origin recognition complex links replication, sister chroma- <br />       tid cohesion transcriptional silencing Saccharomyces <br />       cerevisiae. Genetics 2004, 167(2):579-591. <br /> 61.   Byrne AB, Weirauch MT, Wong V, Koeva M, Dixon SJ, Stuart JM, Roy <br />       PJ: A global analysis genetic interactions Caenorhabditis <br />       elegans. J Biol 2007, 6(3):8. <br /> 62.   CodonW [http://sourceforge.net/projects/codonw/] <br /> 63.   McGary KL, Lee I, Marcotte EM: Broad network-based predicta- <br />       bility Saccharomyces cerevisiae gene loss-of-function phe- <br />       notypes. Genome Biol 2007, 8(12):R258. <br /> 64.   Wu G, Culley DE, Zhang W: Predicted highly expressed genes <br />       genomes Streptomyces coelicolor Streptomy- <br />       ces avermitilis implications metabolism. <br />       Microbiology 2005, 151(Pt 7):2175-2187. <br /> 65.   Robinson MD, Grigull J, Mohammad N, Hughes TR: FunSpec: a <br />       web-based cluster interpreter yeast. BMC Bioinformatics <br />       2002, 3(1):35. <br /> 66.   Nash R, Weng S, Hitz B, Balakrishnan R, Christie KR, Costanzo MC, <br />       Dwight SS, Engel SR, Fisk DG, Hirschman JE, et al.: Expanded pro- <br />       tein information SGD: new pages proteome browser. <br />       Nucleic Acids Res 2007:D468-471. <br /> 67.   Lu P, Vogel C, Wang R, Yao X, Marcotte EM: Absolute protein <br />       expression profiling estimates relative contributions of <br />       transcriptional translational regulation. Nat Biotechnol <br />       2007, 25(1):117-124. <br /> 68.   Fraser HB, Hirsh AE, Giaever G, Kumm J, Eisen MB: Noise minimi- <br />       zation eukaryotic gene expression. PLoS Biol 2004, 2(6):e137. <br /> 69.   Belle A, Tanay A, Bitincka L, Shamir R, O&apos;Shea EK: Quantification <br />       protein half-lives budding yeast proteome. Proc Natl <br />       Acad Sci USA 2006, 103(35):13004-13009. <br /> 70.   Wall DP, Hirsh AE, Fraser HB, Kumm J, Giaever G, Eisen MB, Feld- <br />       man MW: Functional genomic analysis rates protein <br />       evolution. Proc Natl Acad Sci USA 2005, 102(15):5483-5488. <br /> 71.   Gavin AC, Bosche M, Krause R, Grandi P, Marzioch M, Bauer A, <br />       Schultz J, Rick JM, Michon AM, Cruciat CM, et al.: Functional organ- <br />       ization yeast proteome systematic analysis pro- <br />       tein complexes. Nature 2002, 415(6868):141-147. <br />  <br />  <br />  <br />  <br />                                                                                       Publish Bio Med Central every <br />                                                                                      scientist read work free charge <br />                                                                                  &quot;BioMed Central significant development for <br />                                                                                  disseminating results biomedical researc h lifetime.&quot; <br />                                                                                       Sir Paul Nurse, Cancer Research UK <br />  <br />                                      <br /> </body></html>