<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Genetic footprinting mariner-based transposition <br /> in Pseudomonas aeruginosa <br /> Sandy M. Wong*&#226;&#8364;&#160; John J. Mekalanos*&#226;&#8364;&#161;&#194;&#167; <br /> *Department Microbiology Molecular Genetics, &#226;&#8364;&#161;Shipley Institute Medicine, Harvard Medical School, 200 Longwood Avenue, <br /> Boston, MA 02115 <br />  <br /> Contributed John J. Mekalanos, June 27, 2000 <br />  <br /> The complete DNA sequence Pseudomonas aeruginosa pro-                    (Ura) used LB M9 minimal (Difco) plates 100 <br /> vides opportunity apply functional genomics major                &#226;?&#174;g&#229;&#8230;&#190;ml; 5-f luoroorotic acid (FOA) used LB plates 300 <br /> human pathogen. A comparative genomics approach combined                    &#226;?&#174;g&#229;&#8230;&#190;ml. Sucrose-resistant (Sucr) P. aeruginosa isolates were <br /> with genetic footprinting used strategy identify                screened LB plates containing 5% (vol&#229;&#8230;&#190;vol) Suc. Standard <br /> genes required viability P. aeruginosa. Use highly              molecular biology procedures used cloning and <br /> efficient vivo mariner transposition P. aeruginosa             propagation plasmids E. coli (7). Plasmids trans- <br /> facilitated analysis candidate genes class. We          ferred conjugally E. coli P. aeruginosa membrane <br /> developed rapid efficient allelic exchange using            filters early-log-phase E. coli donors (&#226;&#172;&#8225;108&#226;&#8364;&#8220;109) P. <br /> the I-SceI homing endonuclease conjunction vitro                 aeruginosa recipients (&#226;&#172;&#8225;108) grown overnight 42&#194;&#176;C. Filters <br /> mariner mutagenesis generate mutants targeted re-                 incubated 37&#194;&#176;C nonselective LB agar minimum <br /> gions P. aeruginosa chromosome genetic footprinting              5 h overnight plating mating mixture selective <br /> analyses. This technique generating transposon insertion                media. <br /> mutants widely applicable organisms are <br /> not naturally transformable lack developed vivo              Transposon, Plasmid, Strain Construction. <span id='am-2' about='protege:TO' typeof='owl:Thing'>To</span> create pFAC, <br /> transposition systems. We tested genes               Km marker pFD1 (8) replaced 808-bp MluI <br /> in P. aeruginosa putative essential homologs Hae-              fragment containing aacC1 Gm allele pBSL182 (9). <br /> mophilus influenzae. We H. influenzae                pSW(I-SceI) constructed cloning 714-bp SalI (blunt <br /> essential gene homologs needed growth P. aeruginosa,              ended)&#226;&#8364;&#8220;NdeI fragment containing I-SceI ORF pCMV(I- <br /> validating practicality comparative genomics strategy           SceI&#226;&#171;&#185;) (10) SmaI&#226;&#8364;&#8220;NdeI sites pJB658 (11). pSW(I- <br /> to identify essential genes P. aeruginosa.                               SceI) pJB658 (both carry ampicillin markers) were <br />                                                                             mobilized E. coli S17-1 PAO1SR create strains <br /> transposon &#229;&#8230;&#169; SCE jumping                                                    SW658 SWSce, respectively. Primers designed with <br />                                                                             sequence information Pseudomonas Genome Project <br />                                                                             (http:&#229;&#8230;&#190;&#229;&#8230;&#190;www.pseudomonas.com). pSW&#226;&#338;&#172;FGm constructed <br /> T    human pathogen Pseudomonas aeruginosa major <br />      cause opportunistic infections immunocompromised <br /> individuals primary cause chronic pulmonary infec- <br />                                                                             follows: 1-kilobase (kb) product amplified <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> <br />                                                                             PAO1SR primers MERF1 (5&#226;&#172;&#732;-CGCGGATCCGCCATC- <br /> tions patients cystic fibrosis leading respiratory failure       CCGAATAGAGAGAAG) 5&#226;&#172;&#732;-CGTCACGCGTGAA- <br /> and death. This organism highly resistant broad range            ATCCAGGGCGACGATGATGGG; 1.12-kb product was <br /> antibiotics complicating clinical treatment (1&#226;&#8364;&#8220;3). Identification           amplified primers 5&#226;&#172;&#732;-ACGCGTGACGCCGGCTCCGAT- <br />                                                                             TACCT MERF4 (5&#226;&#172;&#732;-CGCGGATCCTTCCGGAATCA- <br /> of P. aeruginosa genes provide important set of <br />                                                                             CATAGTCGCGT) (BamHI MluI sites underlined). <br /> candidates potential targets antimicrobial drugs. An <br />                                                                             The 1-kb 1.12-kb products used PCR primers <br /> approach developed recently allow systematic iden- <br />                                                                             MERF1 MERF4 amplify 2.18-kb product, was <br /> tification genes essential conditionally essential <span id='am-3' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-4' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-5' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span>survival</span> <br />                                                                             digested BamHI, Klenow end-filled, cloned the <br /> in Haemophilus influenzae, human respiratory pathogen whose <br />                                                                             SmaI site pEX100 (12) create pSW&#226;&#338;&#172;F. The aacC1 Gm <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                   MEDICAL SCIENCES <br /> genome sequenced completely. The approach, termed <br />                                                                             cassette cloned MluI site pSW&#226;&#338;&#172;F, followed by <br /> GAMBIT (Genomic Analysis Mapping By In vitro Trans- <br />                                                                             removal bla gene digesting ScaI&#226;&#8364;&#8220;SspI and <br /> position), exploits use mariner-family transposon <br />                                                                             Klenow end-filling create pSW&#226;&#338;&#172;FGm. pSWkan by <br /> Himar1 produce transposon insertion mutants vitro <br />                                                                             replacing 324-bp ScaI&#226;&#8364;&#8220;SspI fragment bla gene in <br /> transposition subsequent functional genomic analyses (4). In <br />                                                                             pEX100 840-bp SmaI fragment containing Km marker <br /> this report, demonstrate strategy essential gene iden- <br />                                                                             pUC18K (13). To create pSW1654 &#226;&#8364;&#8220;55, 2.9-kb product <br /> tification P. aeruginosa makes use H. influenzae <br />                                                                             amplified PAO1SR primers 5&#226;&#172;&#732;-CGCGGATC- <br /> functional genomics information. <br />                                                                             CTGGCAAGGCCCTGTCGCCGTAGA MERPA1654 <br />                                                                             (5&#226;&#172;&#732;-CGCGGATCCACAGCGACTGTCCAATCGACTCTC). <br /> Materials Methods                                                       This fragment digested BamHI, Klenow end-filled, <br /> Bacterial Strains, Plasmids, Media. Escherichia coli strains            cloned SmaI site pSWkan. To create <br /> SM10 &#226;?&#173; pir, DH5&#226;?&#163;, S17-1 (5) P. aeruginosa strains <br /> were grown LB broth maintained standard methods. <br /> P. aeruginosa strains created study derived           Abbreviations: Sm, streptomycin; Km, kanamycin; Gm, gentamicin; Cm, chloramphenicol; <br />                                                                             Cb, carbenicillin; FOA, 5-fluoroorotic acid; Ura, uracil; Suc, sucrose; r, resistant; s, sensitive; <br /> PAO1SR, spontaneous streptomycin (Sm)-resistant isolate                   kb, kilobase. <br /> derived standard laboratory strain PAO1 (6). Sm                    &#226;&#8364;&#160;Presentaddress: Department Microbiology Immunology, University Michigan <br /> provides additional marker selection. Antibiotics added               Medical School, 5641 Medical Science Building II, Ann Arbor, MI 48109. <br /> to LB medium follows (in &#226;?&#174;g&#229;&#8230;&#190;ml): E. coli,                       &#194;&#167;To   reprint requests addressed. E-mail: jmekalanos@hms.harvard.edu. <br /> ampicillin, 100; kanamycin (Km), 50; gentamicin (Gm), 5;                The publication costs article defrayed page charge payment. This <br /> chloramphenicol (Cm), 25; P. aeruginosa, carbenicillin              article marked &#226;&#8364;&#339;advertisement&#226;&#8364;? accordance 18 U.S.C. <br /> (Cb), 300; Km, 500; Gm, 100; Cm, 200; Sm, 200. Uracil                   &#194;&#167;1734 solely indicate fact. <br />  <br />  <br />  <br />                                                                                               PNAS &#229;&#8230;&#169; August 29, 2000 &#229;&#8230;&#169; vol. 97 &#229;&#8230;&#169; no. 18 &#229;&#8230;&#169; 10191&#226;&#8364;&#8220;10196 <br />  pSW&#226;&#338;&#172;1655Gm (&#226;&#338;&#172;807-bp PA1655 ORF), pSW1654 &#226;&#8364;&#8220;55                     transposon-specific primer, MarOUT (5&#226;&#172;&#732;-CCGGGGACTTAT- <br /> was used PCR amplify &#226;&#172;&#8225;8-kb product primers              CAGCCAACC), chromosomal-specific primer, MERF6 <br /> 5&#226;&#172;&#732;-CGACGCGT TGTCGTCGAGAT TGCCGATCGGGG-                                (5&#226;&#172;&#732;-AGGCTTCCAGGGTGTTCAGCATCCC), follow- <br /> TCG 5&#226;&#172;&#732;-CGACGCGTTGCCGAGCTGCCGTTGAA-                                ing conditions: 95&#194;&#176;C 2 min, followed 30 cycles 95&#194;&#176;C for <br /> GAAAGCC, followed digesting MluI cloning               30 s 68&#194;&#176;C 6 min 15 s added extension time for <br /> the aacC1 Gm cassette MluI site. pSW&#226;&#338;&#172;1654Gm                  cycle. PCR products size-fractionated 7% <br /> (&#226;&#338;&#172;688-bp PA1654 ORF) created follows: primers             denaturing polyacrylamide gel analyzed ABI377 <br /> 5&#226;&#172;&#732;-TAGGGATA ACAGGGTA ATGGATCCA AGCT T TA-                             sequencer GENESCAN DNA fragment analysis software. <br /> GGGATAACAGGGTAAT 5&#226;&#172;&#732;-ATTACCCTGTTATC-                               GENESCAN-2500 TAMRA (Applied Biosystems) used size <br /> CCTA A AGCT TGGATCCAT TACCCTGT TATCCCTA                               standards. We able read consistently 1&#226;&#8364;&#8220;1.5 kb <br /> were annealed cloned SmaI site pSWkan              resolution ranging &#226;&#171;&#190;1 bp small fragments <br /> create pSWkanBH. Primers 5&#226;&#172;&#732;-CGCGGATCCGCGGTACG-                        (&#226;&#172;?500 bp) &#226;&#171;&#190;30 bp larger fragments. To footprint the <br /> GCTCATTCTTCAC 5&#226;&#172;&#732;-CGCGGATCCCGCGTTTCT-                              PA1655, PA1654, PA906 regions, PCR performed with <br /> TCTCGCAAAGAAG used amplify 4.7-kb product                   fluorescein-labeled MarOUT primer chromosomal primers <br /> from PAO1SR, digested BamHI cloned                 MERPA1655 (5&#226;&#172;&#732;-CGCGGATCCTGGCAAGGCCCTGTCGC- <br /> into BamHI site pSWkanBH create pSW4.7. Primers             CGTAGA), MERPA1654, MER906S1 (5&#226;&#172;&#732;-TGGCCTTCAAG- <br /> 5&#226;&#172;&#732;-CGACGCGTTGCCGAGCTGCCGTTGAAGAAAGCC                                  GTGCTGGATTCGGAT), MER906G1 genomic <br /> and 5&#226;&#172;&#732;-CGACGCGTTGTCGTCGAGATTGCCGATCGG-                                DNA &#226;&#172;&#8225;106 transposon insertion mutants selected LB <br /> GGTCG used amplify &#226;&#172;&#8225;10-kb product                     medium. PCR products analyzed described above. <br /> pSW4.7, digested MluI followed cloning of <br /> the aacC1 Gm cassette MluI site. To create pSW906,           Allelic Exchange P. aeruginosa. For allelic replacement pyrF, <br /> a 4.1-kb PCR product amplified PAO1SR                   pSW&#226;&#338;&#172;FGm mobilized E. coli S17-1 SWSce or <br /> primers MER906B2 (5&#226;&#172;&#732;-CGCGGATCCTCGTGGTGTTC-                            SW658. After overnight mating 37&#194;&#176;C, transconjugants were <br /> CAGCCAGTGAAATC) MER906B3 (5&#226;&#172;&#732;-CGCGGATC-                            plated minimal medium containing Sm, Gm, Cb with <br /> CCTCCCATGGATGGAACGCCCGAATA) cloned                           Ura. To deliver transposon mutagenized regions of <br /> the BamHI site pSWkanBH. A 1.7-kb PstI fragment con-               cloned P. aeruginosa DNA genome, vitro transposition <br /> taining Gm marker pUC7Gm (a gift S. Lory,                 reactions target plasmids pSW1654&#226;&#8364;&#8220;55 pSW906 were <br /> University Washington, Seattle) cloned PstI site      performed using purified Himar1 transposase described <br /> of pSW906 create pSW906KO. pSW906KO mobilized                  (16). The pool vitro mutagenized plasmids electropo- <br /> from E. coli S17-1 SW658 generate cointegrate strains         rated E. coli S17-1 selected LB containing Gm <br /> SW129 SW323. A complementing plasmid containing               Km. This protocol generated libraries representing &#226;&#172;&#8225;104 and <br /> PA906 gene amplifying 2.65-kb product              &#226;&#172;&#8225;0.5 &#226;&#171;&#187; 103 different transposon insertion events plasmids <br /> PAO1SR primers MER906B2 5&#226;&#172;&#732;-CGCGGATC-                         pSW1654&#226;&#8364;&#8220;55 pSW906, respectively. Approximately 108 E. <br /> CCGATGGCCTTCTTCGAGGACAATGCAG. The product                             coli donors carr ying vitro mutagenized plasmid <br /> was digested BamHI cloned BamHI site             pSW1654&#226;&#8364;&#8220;55 (represents 1,000-fold excess number of <br /> pBBR1MCS (14) create pSW2.6. pBBR1MCS pSW2.6                   different insertion events) mated en masse &#226;&#172;&#8225;108 <br /> were mobilized E. coli SM10 &#226;?&#173; pir SW129            SWSce recipients overnight 37&#194;&#176;C. Transconjugants were <br /> SW323. PCR analysis Sucr isolates complemented            plated LB agar containing Gm Cb. For analysis of <br /> cointegrate strains (see Table 2) primers MER906G1               PA906 region, &#226;&#172;&#8225;109 E. coli donors containing mutagenized <br /> (5&#226;&#172;&#732;-CACATCT TCATCGAGGA ACTGCGCGCCT T)                             plasmid pSW906 (represents 1,000-fold excess in <br /> MER906G4 (5&#226;&#172;&#732;-GTGAAGGATTGGATGTATGGATCAT-                               number different insertion events) mated with <br /> TGG) yielded PCR products &#226;&#172;&#8225;5.8-kb, correlated           &#226;&#172;&#8225;109 recipients (SWSce complemented strain SWSce2.6) for <br /> gene replacement PA906 (no disruption rise           5 h 37&#194;&#176;C. Transconjugants plated LB agar contain- <br /> a predicted 4.2-kb PCR product). PCR analysis Sucr             ing Sm, Gm, Cb Sm, Gm, Cb, Cm. Transposon <br /> isolates cointegrates carrying pBBR1MCS (see Table 2)            insertion mutants pooled diluted OD600 <br /> with primers MER906S4 (5&#226;&#172;&#732;-CGCGGATCCGCAGCCG-                           &#226;&#172;&#8225;0.1 genomic DNA isolated genetic footprint <br /> GGACCCGCAT T TCATGC) MER906S5 (5&#226;&#172;&#732;-CG-                             analyses. <br /> CGGATCCTCAGTCT TCGCGAGGCT TCT TCGCCGC) <br /> yielded product size &#226;&#172;&#8225;480-bp, indicating presence         Results <br /> wild-type PA906 gene.                                                 In Vivo Transposition P. aeruginosa. The successful development <br />                                                                       vivo mariner transposon mutagenesis systems E. coli and <br /> Transposon Mutagenesis P. aeruginosa Genetic Footprinting. A   Mycobacterium smegmatis indicated likelihood trans- <br /> library transposon insertion mutants (&#226;&#172;&#8225;106) P. aeruginosa       poson working bacterium expressing transposase (8). <br /> was generated mobilizing pFAC E. coli SM10 &#226;?&#173; pir          To determine vivo mariner transposition system <br /> (&#226;&#172;&#8225;1010) PAO1SR (&#226;&#172;&#8225;109). After mating &#226;&#172;&#8225;5 h 37&#194;&#176;C,            produce high-density insertions P. aeruginosa, the <br /> transconjugants selected growth LB medium con-            suicide delivery plasmid pFAC encoding mariner transposase <br /> taining Sm Gm. Colony PCR transposon-specific              transferred conjugally E. coli PAO1SR. We <br /> primer, MarIN (5&#226;&#172;&#732;-TACGTA ACAGGT TGGCTGATA-                            obtained transposon insertion mutant library &#226;&#172;&#8225;106 colony- <br /> AGTCG), performed Gm-resistant transconju-             forming units, similar transposition efficiency seen E. <br /> gants, PCR products sequenced verify transpo-         coli, estimated frequency obtaining transposon <br /> son insertions. For pyrF genetic footprint analyses, &#226;&#172;&#8225;106         insertion mutant 1 200 recipients P. aeruginosa. <br /> colony-forming units transposon insertion library was <br /> plated selection conditions: (i) LB agar      Genetic Footprint pyrF Locus. Because vivo mariner <br /> Ura, Sm, Gm; (ii) minimal agar Sm Gm; (iii)          transposition highly efficient P. aeruginosa, tested <br /> LB agar Ura, FOA, Sm, Gm. Genomic DNA                    feasible analyze genes functionally the <br /> isolated pool insertion mutants selection       genomic scale. To develop PCR conditions genetic footprint <br /> condition used template PCR genetic footprinting        analyses P. aeruginosa, pyrF locus chosen test <br /> (15). PCR performed 6-carboxyfluorescein-labeled           region, pyrF gene product, orotidine-5&#226;&#172;&#732;-monophos- <br />  <br /> 10192 &#229;&#8230;&#169; www.pnas.org                                                                                                   Wong Mekalanos <br />  Fig. 1. Genetic footprinting pyrF locus. Locations transposon <br /> insertions pyrF region mapped PCR fluorescent-labeled <br /> transposon-specific primer, MarOUT, chromosomal primer, MERF6. <br /> GENESCAN software quantifies DNA fragments displays results an <br /> electropherogram shows fluorescence intensity (y axis) function of <br /> time, corresponds molecular weight PCR products (x axis). <br /> PCR analysis pyrF locus transposon insertion mutants isolated <br /> under selective conditions: LB plates containing Ura (A), minimal plates <br /> (B), LB plates containing Ura FOA (C). Tick marks elec- <br /> tropherogram corresponding genomic map pyrF region indi- <br /> cate position TA dinucleotides. The peaks pyrF off <br /> scale (C), &#226;&#172;&#8225;4-fold intense largest peak <br /> seen A B. <br />  <br />                                                                                  Fig. 2. Genetic footprinting H. influenzae essential gene homologs <br /> phate decarboxylase, required biosynthesis Ura                   P. aeruginosa. (A) PCR analyses PA1654 PA1655 regions with <br /> (17), provides positive negative selection. pyrF mutants                primer MarOUT chromosomal primers MERPA1654 (located 66 bp from <br /> can obtained medium containing Ura plating                        3&#226;&#172;&#732; end PA1654 ORF) (B) MERPA1655 (326 bp 3&#226;&#172;&#732; end of <br /> medium containing Ura pyrimidine analog FOA. The                         PA1655 ORF). (C) PCR analysis PA906 locus primer MarOUT and <br />                                                                                  chromosomal primer MER906S1 (286 bp 5&#226;&#172;&#732; putative start site PA906). <br /> decarboxylase enzyme converts FOA toxic product; thus, <br />                                                                                  Tick marks indicate position TA dinucleotides. <br /> pyrF mutants resistant FOA grow normally (18). <br /> Approximately 106 colony-forming units mariner trans- <br /> poson insertion library plated selection                  shift <span id='am-159' about='xsp:pattern' typeof='owl:Thing'>pattern</span> TA insertions &#226;&#172;&#8225;150 bp LB, <br /> conditions: LB Ura, Sm, Gm; M9 minimal medium                      minimal, FOA selection conditions (data shown). We <br /> Sm Gm; LB Ura, FOA, Sm, Gm. Mutants                        verified PCR products derived pyrF <br /> pooled condition, genomic DNA isolated                     region Southern blot analysis (data shown), confirming <br />  <br />  <br />  <br />  <br />                                                                                                                                                                 MEDICAL SCIENCES <br /> used template PCR fluorescent-labeled MarOUT                      genetic footprinting results accurately reflect composi- <br /> a chromosomal primer, MERF6, located 251-bp 3&#226;&#172;&#732; end                      tion mutant pool. <br /> of pyrF gene. Fig. 1 shows PCR analyses insertions the <br /> pyrF locus GENESCAN software. On LB medium supple-                          Genetic Footprint H. influenzae Essential Gene Homologs P. <br /> mented Ura, able detect transposon insertions                    aeruginosa. A BLAST search (19) available microbial <br /> in TA dinucleotides pyrF (insertion                     genome sequences revealed number essential genes <br /> between adjacent TA dinucleotides located 5&#226;&#172;&#732; end                  identified H. influenzae using GAMBIT conserved <br /> of pyrF distinguished; Fig. 1 A). We insertions                  hypothetical genes wide variety bacteria. To determine <br /> in 11 17 possible TA dinucleotide insertion sites             homologs essential H. influenzae genes, <br /> a 1.5-kb region insertions identified dinucle-                  HI1655 (putative lipoprotein), HI1654 (putative methyl trans- <br /> otides. This result demonstrates mariner transposon                     ferase), HI0906 (putative cytosine deaminase), also <br /> achieves high degree saturation target sites                essential P. aeruginosa, obtained genetic footprint data <br /> maintain site specificity seen vitro vivo                 regions P. aeruginosa (termed PA1655, PA1654, <br /> eukaryotes bacteria (8). In absence Ura minimal                    PA906, respectively). PCR analysis performed on <br /> medium, PCR products corresponding insertions                       genomic DNA &#226;&#172;&#8225;106 colony-forming units mariner <br /> pyrF detected (Fig. 1B). In contrast, footprints                     transposon insertion library selected LB medium Sm and <br /> regions flanking pyrF similar rich LB (Fig. 1 A)              Gm. We insertions 5 11 possible TA <br /> and minimal media (Fig. 1B). Selection FOA yielded                      dinucleotides PA1654 ORF (Fig. 2A) insertions <br /> expected insertions TA dinucleotide sites exclusively                  8 21 possible TA dinucleotides PA1655 <br /> pyrF (Fig. 1C). PCR analyses second primer located                        ORF (Fig. 2 A B). A second primer located 129 bp <br /> 147-bp downstream primer MERF6 showed predicted                         5&#226;&#172;&#732; 66 bp 3&#226;&#172;&#732; respect primer MERPA1654 showed the <br />  <br /> Wong Mekalanos                                                                                   PNAS &#229;&#8230;&#169; August 29, 2000 &#229;&#8230;&#169; vol. 97 &#229;&#8230;&#169; no. 18 &#229;&#8230;&#169; 10193 <br />                                                                                     analysis. Unlike H. influenzae, P. aeruginosa not <br />                                                                                    able naked DNA chromosomal integration, and <br />                                                                                    conjugation commonly used deliver DNA bacte- <br />                                                                                    rium. However, conjugation does favor gene replacement <br />                                                                                    events results formation cointegrates. We <br />                                                                                    exploited use rare cutting restriction endonuclease, <br />                                                                                    I-SceI, encoded mobile group I intron large 21S <br />                                                                                    rRNA Saccharomyces cerevisiae (20, 21). The recognition <br />                                                                                    site I-SceI (18-bp) reported bacterial genomes <br />                                                                                    date. This enzyme allows fast efficient allelic exchange <br />                                                                                    procedure, termed &#226;&#8364;&#732;&#226;&#8364;&#732;SCE jumping&#226;&#8364;&#8482;&#226;&#8364;&#8482; (see Fig. 3). <br />                                                                                       To develop SCE jumping P. aeruginosa, pyrF locus was <br />                                                                                    chosen test gene allelic exchange (Table 1). In the <br />                                                                                    presence I-SceI enzyme minimal medium, Smr, Gmr, <br />                                                                                    Cbr isolates obtained supplemented with <br />                                                                                    Ura. In presence Ura, targeted <span id='am-81' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-82' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span>knockout</span> pyrF gene <br />                                                                                    SWSce resulted gene replacement frequency 100%, <br />                                                                                    28 28 isolates screened Sucr FOAr. How- <br />                                                                                    ever, absence Ura, 4-log decrease the <br />                                                                                    frequency obtaining transconjugants SWSce. In control <br />                                                                                    recipient strain SW658, Smr, Gmr, Cbr isolates ob- <br />                                                                                    tained similar frequencies minimal medium and <br />                                                                                    Ura. In absence Ura, frequency pyrF gene <br />                                                                                    replacement 0%. Of 25 isolates screened, 25 Suc- <br />                                                                                    sensitive (Sucs) FOAs, indicating integration delivery <br />                                                                                    plasmid chromosome. However, presence Ura, <br />                                                                                    frequency pyrF gene replacement &#226;&#172;&#8225;50%. Of 25 <br />                                                                                    isolates screened, 12 Sucr FOAr, remaining <br />                                                                                    13 Sucs FOAs. These results demonstrate SCE <br />                                                                                    jumping highly efficient facilitating allelic exchange in <br /> Fig. 3. SCE jumping P. aeruginosa. Schematic diagram illustrates high-          P. aeruginosa. <br /> density mutagenesis targeted chromosomal region. Cloned regions of <br /> interest mutagenized vitro introduced P. aeruginosa. Presence <br />                                                                                    Functional Analyses Targeted Regions P. aeruginosa Chro- <br /> of I-SceI enzyme prevents cointegrate formation. The mating mixture is <br /> plated appropriate antibiotics, growth selects insertions <br />                                                                                    mosome. We used SCE jumping (Fig. 3) create mutant pools <br /> in regions containing essential genes.                                             containing random transposon insertions cloned regions <br />                                                                                    containing PA1654, PA1655, PA906. For analyses the <br />                                                                                    PA1654 PA1655 regions, 84 Gmr Cbr transposon <br /> predicted electrophoretic mobility shift pattern PCR                     insertion mutants pooled, aliquot used PCR <br /> products (data shown). Our analysis showed PA1654                     analysis primer MarOUT chromosomal primer <br /> and PA1655 genes nonessential vitro growth P.                       MERPA1654 located near 3&#226;&#172;&#732; end PA1654 ORF. Fig. <br /> aeruginosa. Fig. 2C shows genetic footprint results PA906                  4A shows agarose gel electrophoresis PCR products <br /> region 10 28 possible TA dinucleotides                        obtained PA1654&#229;&#8230;&#190;PA1655 region. The distribution of <br /> contained insertions 1.6-kb region flanking gene.                     insertions PA1654 (PCR products mapping center the <br /> There significant PCR product peaks mapping                             gene) PA1655 (intense PCR products mapping 5&#226;&#172;&#732; end <br /> insertions TA dinucleotides PA906                    gene) correlated pattern fluorescence <br /> ORF. Lack insertions PA906 suggested gene                      intensity PCR products seen GENESCAN results (Fig. <br /> is required growth P. aeruginosa rich medium.                            2 A B). PCR analysis pool mutants a <br />                                                                                    second set primers, MarOUT, chromosomal primer <br /> Development Allelic Exchange System P. aeruginosa. The                    located 84-bp downstream PA1655 ORF yielded PCR <br /> utility high-density insertional mutagenesis discrete (&#226;&#172;&#8225;10-                  pattern consistent patterns seen Figs. 2B 4A <br /> kb) chromosomal regions demonstrated recently                         (data shown). <br /> functional genomic studies naturally transformable organisms                       To analyze PA906 region, 308 Smr, Gmr, Cbr colonies <br /> (4). We sought develop genetic rapidly delivering                  pooled, genomic DNA isolated PCR analysis <br /> transposon insertions discrete regions chromosome                     primer MarOUT chromosomal primer MER906G <br /> P. aeruginosa allow recovery specific mutants                   located &#226;&#172;&#8225;2 kb upstream PA906 ORF. Fig. 4B (lane 1) <br />  <br />  <br />             Table 1. Allelic replacement pyrF using SCE jumping <br />                                                           &#226;&#171;&#186;Ura minimal                                           &#226;&#171;&#185;Ura minimal <br />             Recipient <br />             strain                          SmrGmrCbr                       SucrFOAr                   SmrGmrCbr                      SucrFOAr <br />  <br />             SW658                             1.7 &#226;&#171;&#187; 104                          0&#229;&#8230;&#190;25                   3.4 &#226;&#171;&#187; 104                       12&#229;&#8230;&#190;25 <br />             SWSce                                 0                               0                     0.9 &#226;&#171;&#187; 104                       28&#229;&#8230;&#190;28 <br />  <br />                Suicide plasmid pSW&#226;&#338;&#172;FGm transferred conjugally E. coli donor P. aeruginosa recipients SWSce (contains I-SceI- <br />             expressing plasmid) SW658 (contains parent plasmid pJB658). After overnight mating 37&#194;&#176;C, transconjugants plated onto <br />             minimal medium containing Sm, Gm, Cb Ura. Data recorded number Smr, Gmr, Cbr isolates 108 <br />             recipients. Representative isolates tested Suc FOA sensitivity. <br />  <br />  <br /> 10194 &#229;&#8230;&#169; www.pnas.org                                                                                                                  Wong Mekalanos <br />                                                                                   Table 2. Frequency Sucr isolates plasmid cointegrates <br />                                                                                  Cointegrate strain                        Frequency sucrose resistance <br />  <br />                                                                                  SW129 (pBBR1MCS)                                      1.9 &#226;&#171;&#187; 10&#226;&#171;&#186;6 <br />                                                                                  SW323 (pBBR1MCS)                                      1.0 &#226;&#171;&#187; 10&#226;&#171;&#186;6 <br />                                                                                  SW129 (pSW2.6)                                        2.2 &#226;&#171;&#187; 10&#226;&#171;&#186;4 <br />                                                                                  SW323 (pSW2.6)                                        1.8 &#226;&#171;&#187; 10&#226;&#171;&#186;4 <br />  <br />                                                                                     Complementing plasmid pSW2.6 containing PA906 parent plasmid <br />                                                                                  pBBR1MCS mated cointegrates SW129 SW323 contain <br />                                                                                  plasmid pSW906KO integrated chromosome. Resultant Cbr Gmr, Kmr, <br />                                                                                  Cmr isolates plus minus complementation grown overnight at <br />                                                                                  37&#194;&#176;C LB broth containing Gm Cm Km. Dilutions plated <br />                                                                                  LB agar containing Gm Cm 5% (vol&#229;&#8230;&#190;vol) sucrose. <br />                                                                                  Frequency sucrose resistance calculated ratio number of <br />                                                                                  isolates agar containing 5% (vol&#229;&#8230;&#190;vol) sucrose number isolates on <br />                                                                                  agar sucrose. <br />  <br />  <br />  <br /> Fig. 4. Genetic footprinting targeted regions chromosome. <br />                                                                                  analyzed group Kmr Sucs, consistent with <br /> Plasmid libraries vitro generated transposon insertions pSW1654 &#226;&#8364;&#8220;55     plasmid integration chromosome. <br /> and pSW906 mobilized E. coli SWSce. After selection           Site-directed knockout PA906 yielded transconjugants <br /> appropriate antibiotics, mutant pools used PCR analysis. Locations   Gmr, Cbr, Sucr mating input &#226;&#172;&#8225;108 SWSce <br /> the transposon insertions aligned corresponding gene position.      recipients. This result consistent SCE jumping data <br /> Arrows indicate direction transcription. (A) Genetic footprinting      Fig. 4B, indicating PA906 essential growth rich <br /> PA1655 locus PA1654 gene. Colony PCR performed          medium. In control strain SW658, did obtain <br /> primers MarOUT MERPA1654 located near 3&#226;&#172;&#732; end PA1654                transconjugants containing gene replacement PA906. Of the <br /> mutant pools. (B) Genetic footprinting PA906 region. PCR analysis     150 Gmr Cbr isolates screened, Kmr Sucs, <br /> performed primers MarOUT MER906G1 located &#226;&#172;&#8225;2.0-kb 5&#226;&#172;&#732; <br />                                                                                  consistent plasmid integration chromosome. How- <br /> end putative start site PA906 (lane 1) genomic DNA isolated from <br /> mutant pools. The inferred putative start site based sequence the <br />                                                                                  ever, using recipient strain SWSce2.6, carries com- <br /> PA906 homolog, cumB Pseudomonas putida (26). The pSW906                plementing plasmid, pSW2.6 containing wild-type copy of <br /> mutagenized plasmid library introduced strain SWSce2.6 comple-          PA906, able obtain gene replacement PA906 the <br /> mented wild-type copy PA906. Genomic DNA mutant pools         chromosome. There 24 transconjugants obtained from <br /> used template PCR primers MarOUT MER906G1 (lane 2).               mating pSW906KO SWSce2.6. PCR analysis per- <br />                                                                                  formed isolates chromosomal primers <br />                                                                                  (sequences located complementing plasmid or <br /> shows agarose gel electrophoresis PCR products                        suicide delivery plasmid), indicated disruption the <br /> obtained PA906 region. We observed insertions TA                     chromosomal copy PA906 (data shown). <br /> dinucleotides spanning regions encoding cumA tyrP <br /> homolog intergenic regions. No visible                     Gene Replacement sacB Counterselectable Marker. We next <br /> insertions detected PA906 ORF, consistent                   evaluated essentiality PA906 gene method <br /> the whole-genome genetic footprint results Fig. 2C. However,                  independent SCE jumping approach. We examined <br /> in strain carrying wild-type copy PA906 comple-                    sacB counterselectable marker used to <br /> menting plasmid, transposon insertions pool 102 Smr,                   promote gene replacement event PA906 independent <br /> Gmr, Cbr, Cmr mutants observed TA                       cointegrate strains carrying complementing plasmid <br /> dinucleotides sites chromosomal PA906 allele (Fig.                    pSW2.6 parent plasmid pBBR1MCS. PCR analysis indi- <br /> 4B, lane 2). Thus, disruption PA906 possible                  cated cointegrates SW129 SW323 contain plasmid <br /> presence complementing plasmid.                                           integration pSW906KO chromosome (data shown). <br />  <br />  <br />  <br />  <br />                                                                                                                                                                  MEDICAL SCIENCES <br />                                                                                  Table 2 shows frequency obtaining Sucr isolates the <br /> Targeted Knockout SCE Jumping. To verify functional                     presence absence complementing plasmid LB <br /> analyses genes PA1655, PA1654, PA906, insertional                         medium 5% (vol&#229;&#8230;&#190;vol) Suc. In presence of <br /> mutagenesis frequencies respective gene deter-                      complementing plasmid, frequency obtaining Sucr <br /> mined. We used SCE jumping allelic exchange con-                        isolates &#226;&#172;&#8225;10&#226;&#171;&#186;4 cointegrates. In contrast, fre- <br />                                                                                  quency obtaining Sucr isolates 2-log lower (&#226;&#172;&#8225;10&#226;&#171;&#186;6) for <br /> structs pSW&#226;&#338;&#172;1655Gm, pSW&#226;&#338;&#172;1654Gm, pSW906KO de- <br />                                                                                  cointegrates absence complementing plasmid. <br /> signed produce site-directed knockouts PA1655, PA1654, <br />                                                                                  PCR analysis chromosomal primers contain <br /> and PA906, respectively. The Km marker sacB gene on <br />                                                                                  sequences suicide delivery plasmid pSW906KO in <br /> the constructs provide convenient assays plasmid integration                 complementing plasmid pSW2.6 (see Materials Methods) <br /> or gene replacement events. Expression sacB gene product                  performed 16 Gm, Cm, Sucr representative isolates <br /> renders cell sensitive growth Suc, providing selection                 plasmid-carrying strains. The results <br /> against cointegrates (22). Representative Gmr Cbr transcon-                  showed gene replacement PA906 occurred strains <br /> jugants mating tested ability grow                carried complementing plasmid pSW2.6 those <br /> Suc Km. Targeted disruption PA1654 PA1655                         carrying pBBRIMCS (data shown). In conclusion, three <br /> genes strain SWSce resulted gene replacement fre-                     independent strategies indicate PA906 gene be <br /> quency 100% PA1655 (87 87 isolates screened)                  disrupted, unless complemented, demonstrating PA906 is <br /> PA1654 (122 122 isolates screened). All colonies                required growth P. aeruginosa. <br /> Sucr Kms. However, control strain SW658, the <br /> frequency gene replacement PA1655 (32 49 screened                     Discussion <br /> were Sucr Kms) PA1654 (19 54 screened Sucr                       In report, complex mariner transposon insertion library <br /> and Kms) 35% 65%, respectively. The remaining isolates                   generated vivo P. aeruginosa facilitated used genetic <br />  <br /> Wong Mekalanos                                                                                    PNAS &#229;&#8230;&#169; August 29, 2000 &#229;&#8230;&#169; vol. 97 &#229;&#8230;&#169; no. 18 &#229;&#8230;&#169; 10195 <br />  footprinting strategy (15) assess biological roles                     Essentiality gene vary bacterial species, because <br /> genes unknown function. We developed simple                              gene products fulfill different roles required an <br /> efficient allelic exchange (SCE jumping) using vitro                      individual bacterium. For instance, ferric-uptake regulator <br /> generated mariner mutagenesis create high-density insertions                        fur essential P. aeruginosa bacteria, <br /> within targeted chromosomal regions. The sacB gene                            including E. coli (29). Because able obtain a <br /> traditionally used counterselectable marker facilitate                         PA906 mutant P. aeruginosa, possible require- <br /> homologous recombination; however, sacB marker                              ment PA906 gene product optimal growth greater <br /> suitable approach, transposon effi-                          P. aeruginosa respective homologs P. putida and <br /> ciently disrupt gene vitro mutagenesis. Expression                      B. japonicum. The fact cumB orf74 mutants have <br /> of I-SceI enzyme P. aeruginosa shown extremely                        growth defects inability isolate mutations PA906 <br /> effective promoting gene replacement events selecting                           despite strong selection consistent essentiality of <br /> against cointegrates. In P. aeruginosa, know precise                     HI0906 H. influenzae. <br /> mechanism I-SceI enzyme-mediated homologous recombina-                                 Our results indicate H. influenzae essential <br /> tion. Double strand breaks induced I-SceI enzyme vivo                        gene homologs needed growth P. aeruginosa. <br /> have shown promote homologous recombination                                 Because P. aeruginosa genome (&#226;&#172;&#8225;6 megabases) &#226;&#172;&#8225;70% <br /> eukaryotic prokaryotic systems (10, 23&#226;&#8364;&#8220;25) proven                         greater size H. influenzae genome (&#226;&#172;&#8225;1.8 megabases), <br /> useful strategies gene replacement. SCE jumping P.                           surprising number essential genes H. <br /> aeruginosa allows recovery pool mutants containing                         influenzae essential P. aeruginosa. HI1654 been <br /> transposon insertions specific chromosomal regions. Location                        proposed minimal gene set essential cellular <br /> of insertions determined easily mutant PCR                       function based conserved sequences Mycoplasma <br /> and prove useful generating targeted mutations                        genitalium, smallest genome sequenced (&#226;&#172;&#8225;0.6 megabases), <br /> bacterium expressing I-SceI enzyme.                                                H. influenzae, genome &#226;&#172;&#8225;3 times greater M. <br />    We exploited knowledge gained studies H. influenzae                         genitalium (30). The HI1654 homolog absent list <br /> (4) determine homologs putative essen-                         nonessential genes M. genitalium Mycoplasma <br /> tial H. influenzae genes essential P. aeruginosa.                         pneumoniae (31). It probable larger coding capability <br /> Genetic footprint analyses respective homologs PA1654,                          P. aeruginosa compared organisms smaller genomes <br /> PA1655, PA906 P. aeruginosa showed PA1654                              corresponds increase redundant gene functions. A larger <br /> PA1655 essential vitro conditions tested.                        comparative genomic scale analysis H. influenzae essential <br /> In  <br /> </body></html>