<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Copyright &#195;&#8220; 2008 Genetics Society America <br /> DOI: 10.1534/genetics.107.081836 <br />  <br />  <br />  <br /> Genetic Interaction Between Escherichia coli AcpT Phosphopantetheinyl <br />             Transferase YejM Inner Membrane Protein <br />  <br />                                          Nicholas R. De Lay*,1 John E. Cronan&#226;&#8364;&#160;,2 <br />                  *Department Microbiology &#226;&#8364;&#160;Department Biochemistry, University Illinois, Urbana, Illinois 61801 <br />                                                    Manuscript received September 11, 2007 <br />                                                  Accepted publication December 31, 2007 <br />  <br />  <br />                                                               <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> <br />                Strain LH530, mutant Escherichia coli K-12, reported increased outer membrane <br />              permeability, temperature-sensitive growth, reduced synthesis lipid A. The unmapped mutant gene <br />              suppressed high-copy-number plasmids carrying wild-type acpT gene, encodes a <br />              protein catalyzes post-translational protein modification, attachment 49-phosphopantetheine. <br />              We mapped strain LH530 <span id='am-241' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-242' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-243' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-244' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-250' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-257' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-258' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-289' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-290' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-318' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span>mutation</span> gene unknown function, yejM, known encode inner <br />              membrane protein. The mutation yejM <span id='am-163' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-164' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span>nonsense mutation</span> produces truncated protein lacking the <br />              predicted periplasmic domain. Reconstruction mutation gave strain having phenotypes as <br />              LH530. In contrast nonsense mutants, deletion entire yejM gene lethal. Suppression AcpT <br />              overexpression yejM nonsense mutants encoding truncated proteins specific AcpT. <br />              Moreover, AcpT overexpression suppressed lethality deletion entire yejM gene this <br />              suppression did require AcpT enzymatically active. The mechanism overexpression <br />              specific cytosolic protein bypasses essentiality inner membrane protein unknown. <br />  <br />  <br />  <br /> T     HE cell envelopes gram-negative bacteria consist <br />        inner membrane, outer membrane, and <br /> a rigid layer peptidoglycan located two <br />                                                                          thesis hypersensitivity hydrophobic large <br />                                                                          hydrophilic antibiotics. These mutations occur the <br />                                                                          lpxA, lpxC, lpxD genes (Normark 1970; Tsuruoka <br /> membranes. The outer membrane provides protec-                         et al. 1988; Galloway Raetz 1990; Kloser et al. 1996; <br /> tive barrier prevents diffusion hydrophobic                  Vaara Nurminen 1999). Both lpxA lpxD encode <br /> and large hydrophilic antibiotics cell con-                 enzymes transfer fatty acid, b-hydroxymyristate, <span id='am-6' about='protege:TO' typeof='owl:Thing'>to</span> <br /> sists inner leaflet composed phospholipids                  nascent lipid A species (Crowell et al. 1986; Kelly et al. <br /> an outer leaflet composed lipopolysaccharides. The                    1993) lpxC encodes enzyme deacetylates <br /> structure lipopolysaccharide divided                      precursor lipid A (Young et al. 1995). <br /> three components: lipid A, core oligosaccharide,                        Hirvas et al. (1997) isolated E. coli mutant <br /> and O-antigen. The lipid A portion lipo-                      strain, strain LH530, hypersensitive hydro- <br /> polysaccharide anchors lipopolysaccharide                     phobic large hydrophilic antibiotics showed <br /> outer leaflet outer membrane. The synthesis                    temperature-sensitive growth. Strain LH530 re- <br /> lipid A Escherichia coli studied                    ported defective lipid A biosynthesis the <br /> serves model understanding process                  mutant gene mapped isolated. However, <br /> (Raetz et al. 2007). The lipid A molecules synthesized                   attempts isolate responsible gene, these <br /> by E. coli gram-negative bacteria consist                   workers identified gene called ORF195 (or o195), but <br /> a b-19-6-linked disaccharide N-acetylglucosamine                 called acpT, encodes phosphopantetheinyl <br /> is modified attachment fatty acids, phos-                      transferase (Lambalot et al. 1996; Flugel et al. 2000; De <br /> phates, additional saccharides (Raetz et al. 2007).                  Lay Cronan 2006), suppressed temper- <br /> Additional covalent modifications core lipid A                    ature-sensitive <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> mutant strain LH530. <br /> molecule, addition palmitate modi-                   However, strain LH530 carried wild-type copy acpT <br /> fication 19- 49-phosphates l-4-aminoarabinose             growth 42&#194;&#176; occurred high levels of <br /> or phosphoethanolamine, occur outer mem-                        overexpression (Hirvas et al. 1997). Therefore, growth <br /> brane stressed (Raetz et al. 2007).                                   suppression complementation. <br />    Mutations identified lipid A bio-                Strain LH530 phenotypes consistent a <br /> synthetic genes result reduction lipid A syn-                 defect lipid A synthesis; i.e., strain leaked peri- <br />                                                                          plasmic enzyme, cytoplasmic enzyme, its <br />    1 <br />     Present address: Bldg. 37, Room 5132, National Cancer Institute,     outer membrane contained abnormally low levels the <br /> Bethesda, MD 20892. <br />    2 <br />                                                                          OmpF porin (Nurminen et al. 1997). <br />     Corresponding author: Department Microbiology, University of <br /> Illinois, B103 Chemical Life Sciences Laboratory, 601 S. Goodwin        We studied mutation in <br /> Ave., Urbana, IL 61801. E-mail: j-cronan@life.uiuc.edu                   phosphopantetheinyl transferases (Flugel et al. 2000; <br />  <br /> Genetics 178: 1327&#226;&#8364;&#8220;1337 (March 2008) <br />  1328                                                 N. R. De Lay J. E. Cronan <br />  <br />                                                                  TABLE 1 <br />                                                         Strains used study <br />  <br />           Strain                                  Relevant features                                 Source reference <br />           CAG12177         MG1655 atoSTTn10                                                    Singer et al. (1989) <br />           DH5a             recA1 endA1                                                         Strain collection <br />           HT253            W3110 pdxJ8TDTn10                                                   Strain collection <br />           JM105            F9 lacIq proAB/endA D(lac-proAB)                                    Yanisch-Perron et al. (1985) <br />           LH530            F9 lacIq proAB/endA D(lac-proAB) Ts                                 Hirvas et al. (1997) <br />           MC1061           araD139 D(ara-leu)7696                                              Strain collection <br />           Top10F9          F9 lacIq Tn10 (TetR)/f80TlacZ DM15 DlacX74 recA1 endA1              Invitrogen (San Diego) <br />           JW0757-1         DaraBAD DlacZ DbioA746Tkan                                          CGSC; Baba et al. (2006) <br />           JW0014-1         DaraBAD DlacZ DdnaJ735Tkan                                          CGSC; Baba et al. (2006) <br />           CY1783           DaraBAD DlacZ bioA746TlacZY                                         This study <br />           CY1817           DaraBAD DlacZ DdnaJ735TlacZY                                        This study <br />           NRD142           LH530 atoSTTn10                                                     This study <br />           NRD148           JM105 atoSTTn10 Ts                                                  This study <br />           NRD150           MC1061 DompCTkan                                                    This study <br />           NRD151           MC1061 DyojITkan                                                    This study <br />           NRD153           MC1061 DrcsCTkan                                                    This study <br />           NRD159           MC1061 DatoBTkan                                                    This study <br />           NRD161           MC1061 DnapBTkan                                                    This study <br />           NRD163           MC1061 DccmHTkan                                                    This study <br />           NRD169           MC1061 DyejKTkan                                                    This study <br />           NRD178a          MC1061 DyejMTkan                                                    This study <br />           NRD183           JM105 DyejMTkan carrying pNRD217                                    This study <br />           NRD184           MC1061 yejM569Tkan                                                  This study <br />           NRD187           JM105 yejM569Tkan                                                   This study <br />             CGSC, Coli Genetic Stock Center (Yale University). <br />             a <br />              This strain carries chromosomal duplication yejM region (see text). <br />  <br />  <br />  <br /> De Lay Cronan 2006). We report mapping                       sensitive phenotype. Strain NRD148 generated trans- <br /> the mutation responsible temperature-sensitive                  ducing Tn10 strain NRD142 strain JM105 and <br />                                                                         screening resulting tetracycline transductants strain <br /> phenotype phenotypes strain LH530. The <br />                                                                         acquired temperature-sensitive phenotype. The <br /> mutant strain yejM gene                  DompC strain NRD150 constructed l-Red recombinase- <br /> characterized, including demonstration yejM                    mediated gene replacement (Datsenko Wanner 2000) <br /> is essential gene. Moreover, demonstrate                     using PCR product generated template pKD4 <br /> suppression temperature-sensitive phenotype                   (Datsenko Wanner 2000) OmpCKO For and <br />                                                                         Rev primers. The strains carrying deletions yojI, rcsC, atoB, <br /> strain LH530 overexpression AcpT due <br />                                                                         napB, ccmH, yejM, yejK generated l-Red recombi- <br /> to phosphopantetheinyl transferase activity                nase-mediated gene replacement using PCR product <br /> AcpT overexpression suppresses lethality                    generated template pKD4, using pair primers <br /> yejM <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span>null mutants</span>.                                                      named gene (supplemental Table 1S). Strain NRD183 <br />                                                                         constructed transduction kanamycin resistance <br />                                                                         construct strain NRD178 strain JM105 carrying <br />                                                                         plasmid pNRD217. This transduction performed the <br />                 MATERIALS AND METHODS                                   presence arabinose, transductants selected LB <br />                                                                         plates containing kanamycin arabinose. The wild-type <br />    Bacterial strains plasmids: The bacterial strains used        yejM gene modified introducing UGA termination <br /> this study derivatives strain E. coli K-12. All        codon place codon Y190 UGA (opal) codon <br /> E. coli strains listed Table 1 Tn10 insertion     substituting kanamycin resistance cassette yejM <br /> strains used gene mapping, previously de-                sequences downstream mutated codon l-Red- <br /> scribed. (Singer et al. 1989; Nichols et al. 1998). All plasmids        mediated recombination using PCR product generated <span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> used study listed Table 2. The primers used      template plasmid pKD4 using primers YejMKO4 For and <br /> study (supplemental Table 1S) purchased Inte-                 YejMKO Rev. Kanamycin-resistant recombinants selected <br /> grated DNA Technologies. Fosmid pCC1Fos obtained                    LB plates containing kanamycin 28&#194;&#176;. Strain NRD187 was <br /> from Epicentre. Site-directed mutagenesis               generated transduction kanamycin resistance de- <br /> Quickchange PCR method (Stratagene, La Jolla, CA).                      terminant strain NRD184 strain JM105. The lacZY <br />    Strain NRD142 constructed phage P1vir transduc-               transcriptional fusions bioA dnaJ constructed as <br /> tion Tn10 strain CAG12177 strain LH530                   described Ellermeier et al. (2002) vectors with <br /> (Miller 1972) followed screening tetracycline-                improved ribosome-binding sites ( J. Slauch, personal com- <br /> resistant transductants maintenance temperature-             munication) used pCY580 (Cronan 2003) used <br />                                                         Phenotype E. coli yejM                                                 1329 <br />  <br />                                                                 TABLE 2 <br />                                                       Plasmids used study <br />  <br /> Plasmid                                         Relevant features                                          Source reference <br /> pBAD24            Ampr, araBAD promoter-based expression vector, pUC-type ori                        Guzman et al. (1995) <br /> pBAD30            Ampr, araBAD promoter-based expression vector, p15a ori                            Guzman et al. (1995) <br /> pBAD322           AmpR, araBAD promoter-based expression vector, complete pBR322 origin              Cronan (2006) <br /> pCR2.1            Ampr, Kanr, cloning vector, pUC ori                                                Invitrogen <br /> pKD4              oriRg, Kanr cassette flanked FRT sites                                          Datsenko Wanner (2000) <br /> pKD46             AmpR, RepA1019(Ts), l, g, b, exo expressed araBAD promotor             Datsenko Wanner (2000) <br /> pQE70-Sfp         Ampr, pQE70-derived plasmid encoding Sfp C-terminal His6 tag                Mofid et al. (1999) <br /> pYon113           Ampr, acpT control araBAD promoter                                 Flugel et al. (2000) <br /> pNRD28            acpS amplified JM105 using primers AcpSBAD For Rev                    This study <br />                      TOPO cloned pCR2.1 <br /> pNRD83            EcoRI&#226;&#8364;&#8220;HindIII acpS fragment pNRD28 inserted sites              This study <br />                      pBAD322 <br /> pNRD192           EcoRI&#226;&#8364;&#8220;XbaI acpS fragment pQE70-Sfp inserted sites              This study <br />                      pBAD30 <br /> pNRD203           EcoRI&#226;&#8364;&#8220;HindIII acpS acpS fragment pNRD83 inserted sites            This study <br />                      pBAD24 <br /> pNRD204           EcoRI&#226;&#8364;&#8220;XbaI sfp acpS fragment pNRD192 inserted sites            This study <br />                      pBAD18 <br /> pNRD205           Encodes AcpT D91E. Site-directed mutation acpT pYon113                   This study <br />                      generated using primers AcpTD91E For Rev. <br /> pNRD206           Encodes AcpT E137S. Site-directed mutation acpT pYon113                  This study <br />                      generated using primers AcpTE137S For Rev. <br /> pNRD207           Encodes AcpT K141A. Site-directed mutation acpT pYon113 using                This study <br />                      primers AcpTK141A For Rev. <br /> pNRD216           yejM amplified JM105 using primers YejM For Rev primers               This study <br />                      TOPO cloned pCR2.1 <br /> pNRD217           NcoI&#226;&#8364;&#8220;XbaI yejM containing fragment pNRD216 inserted                  This study <br />                      sites pBAD322 <br />  <br />  <br />  <br /> as source Flp recombinase construction. b-       marker replacing yejK, ccmH, napB, yojI, ompC, rcsC, atoB were <br /> Galactosidase activities determined according Miller          determined. More 200 kanamycin-resistant transductants <br /> (1972).                                                               resulting transduction strain LH530 phage P1vir <br />    Culture media growth conditions: Strains grown            lysates grown strains NRD169, NRD163, NRD161, NRD151, <br /> in LB liquid medium LB agar plates. LB medium               NRD150, NRD153, NRD159 screened tempera- <br /> supplemented final concentrations glucose (0.2%)           ture sensitivity described above. Once closely linked <br /> arabinose (0.08%). Antibiotics used following con-        marker identified, gene replaced marker, to- <br /> centrations: 100 mg liter 1 ampicillin, 50 mg liter 1 kanamy-         gether  2500 bp adjacent genomic DNA both <br /> cin, 5 mg liter 1 tetracycline.                                   strain LH530 parental strain JM105, sequenced <br />    Gene mapping: The mutation conferring temperature                  amplification PCR using YejK, YejL, YejM <br /> sensitivity strain LH530 initially localized E. coli    primers. Each PCR product directly sequenced both <br /> K-12 chromosome transduction strain LH530 P1vir            primers used amplification Core Sequencing <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>Unit</span> at <br /> lysates collection strains harboring Tn10 in-          Keck Center Comparative Functional Genomics at <br /> sertions spaced E. coli chromosome (Singer et al.          University Illinois. <br /> 1989; Nichols et al. 1998). P1vir lysates nearly <br /> all strains collection Tn10 insertion <br /> within 56 min E. coli chromosome. These P1vir                                  RESULTS <br /> lysates used transduce Tn10 insertion of <br /> these strains strain LH530. The transductants                  Mapping mutation responsible the <br /> acquired Tn10 selected ability strains      temperature-sensitive phenotype E. coli K-12 <br /> to grow 28&#194;&#176; LB plates containing tetracycline. Between 48 <br /> and 102 tetracycline-resistant transductants obtained         strain LH530: Hirvas et al. (1997) previously identified <br /> each transduction screened loss temperature           temperature-sensitive mutant named strain LH530 <br /> sensitivity streaking transductant LB plates con-      phenotype similar mutants defec- <br /> taining tetracycline followed incubation plate 28&#194;&#176;       tive lipid A synthesis; i.e., mutant sensitive to <br /> and 42&#194;&#176;. Two Tn10 insertions cotransduced           hydrophobic large hydrophilic antibiotics per- <br /> the temperature-sensitive mutation strain LH530. One <br /> Tn10 inserted napA inserted            missive temperatures reported sub- <br /> into atoS. To define location mutation,        normal rates lipid A synthesis temperatures. <br /> cotransduction frequencies mutation kanamycin           Hirvas et al. (1997) plasmids carrying gene <br />  1330                                            N. R. De Lay J. E. Cronan <br />  <br />  <br />  <br />  <br />    Figure 1.&#226;&#8364;&#8221;The segment E. coli chromosome that <br /> contains yejM. Transductions using strain LH530 recip- <br /> ient performed using P1vir lysates derivatives <br /> of strain MC1061, genes denoted a <br /> solid arrow replaced kanamycin resistance cassette. <br /> Transductants selected LB&#226;&#8364;&#8220;kanamycin plates incu- <br />                                                                    Figure 2.&#226;&#8364;&#8221;An illustration putative structure the <br /> bated 28&#194;&#176; 2 days. From cross 200 kanamy- <br />                                                                  full-<span id='am-5' about='xsp:length' typeof='owl:Thing'>length</span> YejM (A) truncated form YejM pro- <br /> cin-resistant transductants screened for <br />                                                                  duced strain LH530 (B) based data predictions <br /> temperature-sensitive growth. Below solid arrows <br />                                                                  Daley et al. (2005). YejM inner membrane protein <br /> is frequency kanamycin marker replacing the <br />                                                                  contains putative membrane-spanning helices and <br /> gene cotransduced gene relieved the <br />                                                                  periplasmic C-terminal domain. The periplasmic domain <br /> temperature-sensitive phenotype strain LH530. <br />                                                                  YejM predicted sulfatase/phosphatase activity <br />                                                                  UniProtKB/Swiss-Prot database. <br />  <br /> named acpT (then called ORF195) suppressed                   resistance cassette inserted place yejK. We then <br /> temperature-sensitive phenotype mutant strain,           amplified genes yejK, yejL, yejM inter- <br /> but AcpT-encoding plasmids high            genic regions genes strain LH530 <br /> copy number. However, acpT gene strain LH530              parental strain JM105 sequenced PCR <br /> had sequence parental wild-type strain,          products. The sequencing data showed single differ- <br /> indicating suppression complementation.              ence genomic sequences: G570A <br /> The mutation responsible temperature-sensitive           mutation yejM gene strain LH530. The G570A <br /> phenotype strain LH530 identified mapped.          mutation resulted replacement UGG codon <br />    We mapped mutation responsible tem-               tryptophan 190 UGA termination codon. YejM <br /> perature-sensitive phenotype strain LH530 trans-           previously shown inner membrane <br /> ducing strain LH530 phage P1vir lysates                protein predicted composed transmembrane <br /> collection strains, carried transposon        domain consisting putative membrane-spanning <br /> Tn10 antibiotic resistance element inserted known           helices large periplasmic domain (Daley et al. <br /> chromosomal location (Singer et al. 1989; Nichols                2005) (Figure 2). The nonsense mutation trun- <br /> et al. 1998). Upon screening 56 min E. coli     cate expressed protein 190 residues 586 <br /> chromosome, temperature-sensitive mutation                residues cleanly remove predicted peri- <br /> strain LH530  53 29% linked               plasmic domain leaving predicted transmem- <br /> Tn10 insertions napA atoS, respectively (Figure 1).       brane domain intact (Figure 2). <br /> To resolve location mutation, set              We linked yejM mutation strain LH530 the <br /> of nonessential genes located region E. coli      Tn10 insertions napA atoS transduction <br /> chromosome (yejK, ccmH, napB, yojI, ompC, rcsC,              used strains donors transduce <br /> atoB) replaced kanamycin resistance cassettes          strains JM105 MG1655 tetracycline resistance. <br /> via l-Red recombinase-mediated gene replacement                  The resulting transductants screened for <br /> (Datsenko Wanner 2000) linkage                temperature-sensitive growth. The temperature-sensi- <br /> mutation markers determined mea-               tivity profile observed strain NRD187, derivative <br /> suring cotransduction frequencies (Figure 1). The                strain JM105 acquired yejM(Ts) muta- <br /> temperature-sensitive mutation strain LH530               tion, identical observed strain LH530 <br /> found tightly linked (100%) kanamycin             (data shown). However, yejM(Ts) <br />                                                    Phenotype E. coli yejM                                               1331 <br />  <br />  <br />  <br />  <br />  Figure 3.&#226;&#8364;&#8221;The growth strains JM105, NRD187,                 Figure 4.&#226;&#8364;&#8221;Complementation temperature-sensitive <br /> LH530 LB plates incubated overnight 28&#194;&#176; 42&#194;&#176;.     strain LH530 plasmid-encoded copy yejM. Strain JM105 <br />                                                                  derived temperature-sensitive mutant, strain LH530, <br />                                                                  carrying vector pBAD322 plasmid containing <br />                                                                  copy yejM control araBAD promoter <br /> derivative strain MG1655 clearly temperature              (pNRD217 called pYejM figure) tested abil- <br /> sensitive, strain showed greater growth 42&#194;&#176;         ity grow 42&#194;&#176; LB plates containing glucose or <br /> did strain JM105 derivatives (data shown). This          arabinose. <br /> difference behavior attributed fact that <br /> strain MG1655 grew significantly faster strain              resulting plasmid, pNRD217, complement <br /> JM105 faster growth accentuated leakiness           growth strain LH530 42&#194;&#176; plates containing either <br /> (formation small colonies) shown strain LH530              glucose arabinose. Strain LH530 containing pNRD217 <br /> upon prolonged incubation 42&#194;&#176;.                                grew LB&#226;&#8364;&#8220;arabinose-containing plates, did not <br />    Characterization yejM(Ts) mutation: To con-            grow plates containing glucose (which represses <br /> firm premature termination codon yejM            expression araBAD promoter) (Figure 4). The <br /> gene strain LH530 responsible tempera-            introduction plasmid pNRD217 strain NRD187 <br /> ture-sensitive growth strain LH530, used l-Red             similarly allowed growth 42&#194;&#176; LB plates containing <br /> recombinase-mediated gene replacement generate                arabinose containing glucose (data <br /> the nonsense (G570A) mutation strain LH530             shown). The inability recover plasmid-encoding <br /> wild-type gene concomitantly replace yejM             yejM complementation mutation LH530 in- <br /> sequence downstream mutation kanamycin             dicates clone bank screened Hirvas et al. <br /> resistance cassette. This construct transduced          (1997) lacked yejM plasmid presumably bias in <br /> into strain JM105 generate strain NRD187. We             construction bank. <br /> compared abilities strains JM105, LH530,                 The yejM(Ts) mutation suppressed overexpression <br /> NRD187 grow LB plates incubated 28&#194;&#176; 42&#194;&#176;              active inactive forms AcpT, high- <br /> overnight. Although strains grew 28&#194;&#176;,          level expression phosphopantetheinyl <br /> only strain JM105 grew 42&#194;&#176; following overnight in-            transferases, AcpS Sfp: Hirvas et al. (1997) have <br /> cubation (Figure 3). However, prolonged incuba-             previously shown temperature-sensitive pheno- <br /> tion, strain LH530 formed small colonies 42&#194;&#176;          type strain LH530 suppressed high-copy-number <br /> strain NRD187 did not. This difference suggested            plasmids carrying acpT, encodes phosphopante- <br /> in strain LH530 basal readthrough UGA non-                theinyl transferase plays minor role E. coli <br /> sense codon (Bjornsson Isaksson 1993) resulted               K-12 lipid metabolism (De Lay Cronan 2006). Phos- <br /> in low levels full-length YejM. Strains LH530             phopantetheinyl transferases enzymes transfer <br /> NRD187 essentially identical lipid A phenotypes (see         49-phosphopantetheinyl moiety CoA con- <br /> below) and, expected previous report (Hirvas         served serine residue carrier protein the <br /> et al. 1997), strains sensitive           acyl carrier proteins lipid synthesis (Lambalot et al. <br /> rifampicin erythromycin strain JM105.                   1996). Attachment prosthetic group mandatory <br />    We performed complementation experiments                   thiol group phosphopantetheinyl <br /> assess expression wild-type copy          moiety carries intermediates fatty acid biosynthe- <br /> yejM native promoter cosmid vector          sis. We tested ability phosphopantetheinyl <br /> or araBAD promoter multicopy plasmid                transferases, AcpT, AcpS, Sfp, suppress the <br /> allowed growth strain LH530 42&#194;&#176;. A fragment             temperature-sensitive phenotype LH530 over- <br /> DNA contained yejM, upstream              expression. E. coli AcpS trimeric phosphopante- <br /> and downstream genes yejL proL, respectively,            theinyl transferase catalyzes modification acyl <br /> amplified strain JM105. This genome segment             carrier proteins fatty acid biosynthesis (Flugel et al. <br /> then inserted single-copy F-factor-based cosmid         2000) Sfp monomeric Bacillus subtilis en- <br /> pCC1Fos. The presence construct strain LH530          zyme catalyzes phosphopantetheinylation carrier <br /> allowed growth nonpermissive temperature (data            proteins involved secondary metabolic processes <br /> not shown). We placed yejM gene               nonribosomal peptide polyketide syntheses <br /> araBAD promoter pBAD322 tested ability                (Lambalot et al. 1996; Mootz et al. 2001). As previously <br />  1332                                                  N. R. De Lay J. E. Cronan <br />  <br />                                                                         2000; De Lay Cronan 2006). Upon high-level ex- <br />                                                                         pression, mutant acpT alleles lacked ability <br />                                                                         functionally replace AcpS wild-type allele <br />                                                                         showed expected strong complementation (Flugel <br />                                                                         et al. 2000; De Lay Cronan 2006) (data shown). <br />                                                                         However, plasmids encoding wild-type and <br />                                                                         mutant AcpT proteins suppressed temperature- <br />                                                                         sensitive phenotype strain LH530 (Figure 5). There- <br />                                                                         fore, suppression yejM nonsense mutations did not <br />                                                                         require AcpT enzymatically active. <br />                                                                            yejM essential gene, essentiality is <br />                                                                         suppressed AcpT overexpression: The temperature- <br />                                                                         sensitive phenotype strain LH530, the <br />                                                                         production truncated form YejM lacking the <br />                                                                         C-terminal periplasmic domain, raised question of <br />                                                                         effects deletion entire yejM gene. Repeated <br />    Figure 5.&#226;&#8364;&#8221;Suppression analyses strain LH530 plasmids           attempts delete yejM strain MC1061 l-Red <br /> encoding diverse phosphopantetheinyl transferases             recombinase-mediated gene replacement unsuc- <br /> site-directed mutants AcpT phosphopantetheinyl transfer-         cessful despite ready success deleting genes <br /> ase. (A) Derivatives strain LH530 carrying vector pBAD24         method. We eventually isolated single yejM <br /> or plasmid expressed sfp, acpS, acpTunder control of <br /> an araBAD promoter (pNRD204, pNRD83, pYon113, respec-               deletion candidate, strain NRD178. We confirmed the <br /> tively) tested ability grow 42&#194;&#176; LB plates con-    presence yejM deletion strain using PCR <br /> taining glucose inducer, arabinose. (B) The ability wild-     primers yielded product yejM deletion <br /> type AcpT site-directed mutants AcpT elim-        present. However, difficulty experienced <br /> inate phosphopantetheinyl transferase activity vivo (Table 2)        construction strain suggested yejM <br /> was tested ability suppress defective growth strain <br /> LH530 42&#194;&#176; LB plates containing glucose arabi-           deletion strain second characteristic that <br /> nose.                                                                   allowed viability deletion strain. We therefore <br />                                                                         transduced yejM deletion derivative strain <br />                                                                         JM105 carried second plasmid-borne copy yejM <br /> reported Hirvas et al. (1997), high-level overexpres-                control araBAD promoter. Transduction was <br /> sion AcpT suppressed temperature-sensitive                       performed presence arabinose trans- <br /> phenotype strain LH530 (Figure 5) low levels                 duction mixtures plated LB plates containing <br /> of overexpression did (data shown). However,                    arabinose kanamycin. Six transductants tested <br /> neither overexpression AcpS high-level expres-                   ability grow LB plates containing glucose <br /> sion Sfp suppressed temperature-sensitive phe-                   (where expression plasmid-borne yejM repressed) <br /> notype strain LH530 (Figure 5). Similar results                 arabinose (where expression plasmid-borne yejM <br /> obtained strain NRD187, carries vitro-                 induced). All transductants grew LB&#226;&#8364;&#8220;arabinose <br /> constructed yejM nonsense allele (data shown).                      plates LB&#226;&#8364;&#8220;glucose plates (Figure 6). Similar <br />    The lack suppression temperature-sensitive                 results obtained experiment per- <br /> phenotype strain LH530 high-level expression                   formed MG1655 DyejM strain carrying pNRD217, <br /> AcpS Sfp suggested phosphopantetheinyl                      yejM plasmid (data shown). Therefore, trans- <br /> transferase activity AcpT required sup-                  duction resolved DyejM allele from <br /> pression. Thus, amino acid substitutions                    character present original strain, NRD178. <br /> three key AcpT residues (D91E, E137S, K141A)                        Further analysis genome original yejM <br /> conserved phosphopantetheinyl transferases                        deletion strain NRD178, using PCR primers gener- <br /> (Lambalot et al. 1996) involved                       ated products different lengths depending the <br /> binding CoA substrate Sfp AcpS (Reuter                    presence absence wild-type copy yejM, demon- <br /> et al. 1999; Parris et al. 2000; Mofid et al. 2004). The                strated original yejM deletion strain, strain <br /> AcpT amino acid substitutions substitutions pre-                   NRD178, contained duplication region the <br /> viously shown eliminate Sfp phosphopantetheinyl                      genome includes yejM. The duplicated region con- <br /> transferase activity (Mofid et al. 2004) con-                  tained copy wild-type yejM plus copy DyejM <br /> structed pBAD24 vector. Since, agreement                 allele (data shown). We attribute chromosomal <br /> a prior report (Lambalot et al. 1996), that,                   arrangement strain NRD178 l-Red-mediated re- <br /> upon overexpression, AcpT inactive largely                      combination copy preexisting chromosomal <br /> insoluble, assayed activities proteins               duplication yejM region present cell un- <br /> the abilities plasmids support growth strain             dergoing recombination event. Such spontaneous <br /> carrying conditional mutation acpS (Flugel et al.                  duplications bacterial chromosomes docu- <br />                                                    Phenotype E. coli yejM                                              1333 <br />  <br />  <br />  <br />  <br />                                                                     Figure 7.&#226;&#8364;&#8221;Suppression lethality yejM deletion <br />                                                                  strain overexpression AcpT. The yejM deletion strain <br />                                                                  NRD178 transduced phage P1vir strain JM105 <br />                                                                  containing plasmid expressed yejM (pNRD217) <br />                                                                  acpT (pYon113) arabinose-inducible araBAD pro- <br />                                                                  moter. Transductants acquired yejM deletion se- <br />                                                                  lected LB&#226;&#8364;&#8220;kanamycin&#226;&#8364;&#8220;arabinose plates. A transductant <br />    Figure 6.&#226;&#8364;&#8221;Determination essentiality yejM.          cross, parental strains, was <br /> Strains JM105, LH530, NRD183, derivative strain         streaked plates LB&#226;&#8364;&#8220;glucose LB&#226;&#8364;&#8220;arabinose were <br /> JM105 containing deletion yejM carries plasmid         incubated overnight 30&#194;&#176;. <br /> pNRD217 encoding yejM (called pyejM above) con- <br /> trol araBAD promoter, tested ability to <br /> grow overnight 28&#194;&#176; 42&#194;&#176; LB plates containing      null mutation did require AcpT enzymatic activity. <br /> arabinose glucose.                                            When yejM null mutation transduced a <br />                                                                  derivative JM105 expressing inactive E137S AcpT <br />                                                                  encoded plasmid pNRD206, growth proceeded as <br /> mented (Roth et al. 1996). Moreover, F-factor-based           seen wild-type AcpT (Figure 7). The stain expressing <br /> cosmid generated strain NRD178 genomic DNA                  E137S AcpT grew presence arabinose, <br /> contained genome fragment spanned region              failed grow expression blocked ab- <br /> between fruA ccmF (including wild-type copy             sence arabinose (in presence absence <br /> yejM) lacked kanamycin-marked DyejM deletion.            glucose) (data shown). <br /> This cosmid complemented DyejM deletion                    Does AcpT overexpression induce heat-shock <br /> strain NRD186 (data shown). This suggests          response? One possibility suppression yejM <br /> duplicated region include genes         mutations AcpT overexpression overexpres- <br /> fruA ccmF, consistent sizes docu-           sion triggers stress response induces synthesis a <br /> mented duplications (Roth et al. 1996).                          protein replaces (or bypasses) YejM function. One <br />    To test yejM essential, DyejM     stress response known induced high-level ex- <br /> deletion constructed l-Red recombinase-                 pression insoluble proteins AcpT heat- <br /> mediated gene replacement strain DY330 (Yu et al.             shock response. Hence, asked heat-shock <br /> 2000) carrying yejM plasmid pNRD217. Two              regulon induced AcpT overexpression assay- <br /> recombinants obtained shown contain yejM             ing expression classical heat-shock protein, DnaJ, <br /> deletion PCR recombinants (one              use chromosomal dnaJ-lacZY fusion construct. <br /> is shown Figure 6) grew induction            Construction fusion eliminated DnaJ function, <br /> yejM expression plasmid.                                DnaJ functionally replaced two <br />    Given yejM essential growth,          cochaperone proteins, DjlA CbpA (Gur et al. <br /> protein lacking periplasmic domain allows growth             2004), DnaJ mutants behave normally regards heat <br /> at low-growth temperatures, question raised           shock. Use fusion provides assay DnaK <br /> whether AcpT overexpression suppress            expression encoded promoter-proximal <br /> <span id='am-475' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-476' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-477' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-478' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-479' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-480' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-481' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-482' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-483' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-484' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-485' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-486' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-487' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-488' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-489' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-490' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-491' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-492' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-493' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-494' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-495' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-496' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-497' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-498' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-499' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-500' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-501' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-502' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-503' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-504' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-505' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-506' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-507' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-508' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-509' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-510' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-511' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-512' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-513' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-514' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-515' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-516' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-517' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-518' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-519' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-520' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-521' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-522' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-523' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-524' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-525' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-526' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-527' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-528' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-529' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-530' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-531' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-532' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-533' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-534' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-535' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-536' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-537' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-538' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-539' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-540' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-541' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-542' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-543' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-544' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-545' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-546' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-547' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-548' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-549' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-550' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-551' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-552' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span>lethal phenotype</span> strains carrying total deletion         gene dnaK-dnaJ operon (Cowing et al. 1985; <br /> the yejM gene. We DyejM <span id='am-397' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-398' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-399' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-400' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-401' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-402' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-403' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-404' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-405' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-406' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-407' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-408' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-409' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-410' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-411' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-412' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-413' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-414' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-415' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-416' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-417' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-418' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-419' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-420' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-421' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-422' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-423' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-424' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-425' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-426' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-427' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-428' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-429' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-430' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-431' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-432' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-433' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-434' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-435' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-436' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-437' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-438' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-439' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-440' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-441' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-442' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-443' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-444' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-445' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-446' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-447' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-448' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-449' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-450' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-451' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-452' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-453' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-454' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-455' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-456' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-457' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-458' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-459' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-460' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-461' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-462' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-463' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-464' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-465' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-466' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-467' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-468' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-469' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-470' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-471' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-472' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-473' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-474' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span>deletion mutation</span>         Bardwell et al. 1986). <br /> was readily transduced derivative JM105              Following construction dnaJ-lacZY fusion strain <br /> carried multicopy plasmid acpT expression           CY1817, behavior strain tested classical <br /> an inducible (araBAD) promoter. The strain grew            30&#194;&#176;&#226;&#8364;&#8220;42&#194;&#176; temperature shift. The strain showed the <br /> inducing conditions conditions re-            expected rapid increase dnaK-dnaJ expression (as- <br /> pressed expression plasmid-encoded acpT (Figure           sayed b-galactosidase activity) soon tempera- <br /> 7). The observed growth similar given        ture shift reached plateau threefold greater <br /> induction plasmid-borne copy yejM AcpT          30&#194;&#176; expression level  30 min shift (Figure <br /> overproduction fully suppressed effects oth-         8A) expected increases DnaK levels seen <br /> erwise lethal mutation. As reported suppres-           temperature-shift regimen (Gross 1996). As <br /> sion temperature sensitivity strain                control, lacZY fusion housekeeping biosynthetic <br /> expressing truncated YejM, suppression yejM           gene, bioA, assayed parallel. In contrast dnaJ <br />  1334                                          N. R. De Lay J. E. Cronan <br />  <br /> fusion, bioATlacZY fusion strain showed effect       bypass loss YejM. Strain NRD183, contains a <br /> temperature; b-galactosidase activity constant         complete deletion yejM plasmid encoding AcpT <br /> (Figure 8A).                                                   arabinose control, grown presence of <br />    The dnaJ-lacZY fusion strain CY1817 trans-         arabinose 30&#194;&#176; cells early log phase cultures <br /> formed plasmids encoding wild type, E137S         recovered centrifugation. The cells then <br /> mutant AcpT, vector. Cultures            washed suspended medium lacking arabi- <br /> strains grown 30&#194;&#176;, protein expression induced         nose. This culture divided halves. One-half <br /> with 0.2% arabinose, b-galactosidase activities       received 0.2% arabinose, second received water. <br /> measured. The low-growth temperature chosen                These cultures incubated 30&#194;&#176; their <br /> to decrease heat-shock protein expression levels           growth followed turbidity. When growth the <br /> thereby increase sensitivity assay. All       culture lacking arabinose began slow depletion <br /> cultures showed constant level expression 90-      AcpT, culture split one-half supple- <br /> min time course (Figure 8B), indicating induction        mented arabinose. The time required growth <br /> the heat-shock response. A possible caveat ex-         arabinose-supplemented culture, accelerated rel- <br /> periment time course brief allow     ative culture lacking arabinose, taken a <br /> induction heat-shock regulon, induction        measure time required accumulation AcpT <br /> of heat-shock response expression            levels sufficient bypass yejM deletion. Acceleration <br /> insoluble proteins seen roughly comparable          growth addition arabinose seen within <br /> time intervals (Lesley et al. 2002; Smith 2007). However,      36 min inducer addition (Figure 8C) time <br /> each protein different case          course chosen test induction heat-shock re- <br /> asked long induction AcpT overexpression takes          sponse deemed appropriate. <br />  <br />  <br />  <br />  <br />                                                                   Figure 8.&#226;&#8364;&#8221;Effects heat shock AcpT overproduction <br />                                                                dnaJ induction. (A) Behavior dnaJTlacZY fusion in <br />                                                                heat-shock response. Cultures dnaJTlacZY fusion <br />                                                                strain CY1817 (squares) bioATlacZY fusion strain <br />                                                                CY1783 (circles) growing exponentially early log phase <br />                                                                30&#194;&#176; shifted 42&#194;&#176; time zero samples taken <br />                                                                b-galactosidase assays times given. b-Galactosidase <br />                                                                activities given Miller units (Miller 1972). (B) Deriva- <br />                                                                tives strain CY1817 carrying plasmids encoding the <br />                                                                wild-type (pYon13) mutant (pNRD207) AcpT proteins or <br />                                                                vector (pBAD24) grown 30&#194;&#176; LB&#226;&#8364;&#8220;ampicil- <br />                                                                lin. Exponentially growing cultures received arabinose <br />                                                                final concentration 0.2% induce AcpT overexpres- <br />                                                                sion samples taken b-galactosidase assays the <br />                                                                times given. Triangles, vector control; squares, pYon13; and <br />                                                                circles, pNRD207. b-Galactosidase activities given Miller <br />                                                                units ordinate scale identical A. (C) The <br />                                                                cells culture strain NRD183 grown LB&#226;&#8364;&#8220;ampicillin <br />                                                                medium containing 0.2% arabinose 30&#194;&#176; harvested <br />                                                                centrifugation, washed, resuspended LB medium. <br />                                                                The culture diluted OD600 0.05 split <br />                                                                half. One culture received 0.2% arabinose the <br />                                                                half remained unsupplemented. The cultures were <br />                                                                grown overnight 30&#194;&#176;. The washed cells cul- <br />                                                                tures recovered resuspended an <br />                                                                OD600 0.1 medium been <br />                                                                grown. The turbidities cultures growth 30&#194;&#176; <br />                                                                monitored periodic dilution cultures <br />                                                                exponential growth growth culture lacking <br />                                                                arabinose began slow (at  5 hr dilution over- <br />                                                                night cultures). Half culture added pre- <br />                                                                warmed flask containing sufficient arabinose final <br />                                                                concentration 0.2%. The growth cultures <br />                                                                monitored. Squares denote culture grown with <br />                                                                arabinose experiment; circles denote cul- <br />                                                                ture lacking arabinose experiment; triangles <br />                                                                denote culture supplemented arabinose following <br />                                                                growth absence arabinose. <br />                                                   Phenotype E. coli yejM                                              1335 <br />  <br />                       DISCUSSION                                transposon mutagenesis essentially precluded detection <br />                                                                 yejM. Growth strains carrying yejM transposon <br />    YejM inner membrane protein predicted have <br />                                                                 insertion alleles assayed 30&#194;&#176; 37&#194;&#176; (Gerdes <br /> five membrane-spanning helices large C-terminal <br />                                                                 et al. 2003; Kang et al. 2004), temperatures strains <br /> periplasmic domain (Figure 2) (Daley et al. 2005) that <br />                                                                 LH530 NRD187 grow reasonably well. It remains <br /> shares sequence similarities sulfatases phos- <br />                                                                 unclear strains carrying yejM trans- <br /> phatases (Swiss Protein database). If topological <br />                                                                 poson insertion alleles temperature sensitive for <br /> prediction accurate, G205A yejM mutant <br />                                                                 growth. <br /> allele strain LH530 encode YejM protein <br />                                                                    Suppression growth defects strains lacking <br /> containing complete transmembrane domain, but <br />                                                                 YejM periplasmic domain entire protein <br /> lacking entire C-terminal periplasmic domain. We <br />                                                                 overexpression cytosolic AcpT phosphopante- <br /> constructed yejM mutation vitro resulting        theinyl transferase presents conundrum. How a <br /> strain temperature sensitive. Moreover, original        mutation resulting defective outer membrane be <br /> and constructed yejM G205A mutant strains similar           suppressed overexpression enzyme post- <br /> phenotypes respect lipid A biosynthesis             translationally modifies cytosolic carrier protein? We <br /> hypersensitivity antibiotics. Chromatographic            shown phosphopantetheinyl transferase <br /> mass spectral analyses (C. Michael Reynolds, personal           activity AcpT involved yejM mutant suppression <br /> communication) showed strains (LH530              AcpT overexpression act indirectly. <br /> NRD187) essentially normal lipid A compositions            One possible explanation based properties of <br /> in bulk lipid A produced strains       AcpT nature suppression phenomena <br /> and parent strain JM105 hexa-acylated lipid       suppression involve stress response. In <br /> A 1,49-bis-phosphate species normally E. coli          agreement results Lambalot et al. (1996), we <br /> (Raetz et al. 2007). However, mutant strains            AcpT accumulates insoluble aggregate <br /> both synthesized small discernible levels hepta-       fairly modest overexpression conditions <br /> acylated lipid A species contains additional            (data shown). Aggregation fact <br /> palmitoyl group (C. Michael Reynolds, personal com-             AcpT evolutionarily isolated its <br /> munication). We believe presence lipid A       substrates (De Lay Cronan 2006) lacks <br /> species mutant strains secondary response        substrates aid proper protein folding. Moreover, fairly <br /> problems maintaining outer membrane integrity                high levels AcpT overexpression required full <br /> (Raetz et al. 2007). Previous studies shown           suppression temperature-sensitive phenotype of <br /> addition compounds affect outer membrane                strain LH530; low levels overexpression fail sup- <br /> integrity, cationic peptides, result synthe-     press (Hirvas et al. 1997) (data shown). If misfold- <br /> sis palmitoylated lipid A Salmonella enterica serovar     ing involved suppression YejM mutations, seems <br /> Typhimurium (Zhou et al. 1999). An LpxM-deficient               specific AcpT overexpression acpT clones <br /> mutant strain E. coli O157:H7 reported         clones isolated clone bank 8000 <br /> to accumulate lipid A species (Raetz et al. 2007).         wild-type E. coli DNA fragments high-copy-number <br />    Although nonsense mutations resulted con-           vector allowed growth strain LH530 42&#194;&#176; <br /> ditionally lethal phenotype, deletion entire yejM        (Hirvas et al. 1997). We tested specificity of <br /> gene unconditionally lethal. Together, results        AcpT overexpression temperature-sensitive phe- <br /> suggest YejM functions. The function carried       notype strain LH530 overexpression AcpH, <br /> out N-terminal transmembrane domain essential         E. coli protein involved ACP metabolism that <br /> at growth temperatures function          aggregates overexpression con- <br /> C-terminal periplasmic domain dispensable low-            ditions tested (Thomas Cronan 2005; Thomaset al. <br /> growth temperatures. Both functions important            2007). Overexpression AcpH using vector <br /> for cellular process. During work Baba et al.      employed Figure 7 failed allow growth strain <br /> (2006) reported yejM essential gene       LH530 42&#194;&#176; (data shown). Consistent ob- <br /> were unable construct strain carrying complete           served inferred specificity suppression, AcpT <br /> deletion yejM l-Red recombinase-mediated gene             overexpression failed induce heat-shock regulon. <br /> replacement. In contrast, groups reported yejM         Induction heat-shock regulon misfolded pro- <br /> is essential gene basis transposon muta-       teins frequently observed (Oh Liao 2000; Lesley <br /> genesis (Gerdes et al. 2003; Kang et al. 2004). However,     et al. 2002; Smith 2007), induction heat <br /> these reports reported transposon insertions     shock played major role AcpT overproduction <br /> were located downstream nonsense mutation             suppression phenomenon, large diverse set the <br /> strain LH530 N-terminal transmembrane             Hirvas clones (instead acpT clones) have <br /> domain translated. These mutant strains          suppressed yejM mutation. <br /> produce YejM proteins 60 residues longer             Another possible explanation (suggested re- <br /> that produced strain LH530. Hence, nature             viewer) overexpressed AcpT specifically <br />  1336                                                      N. R. De Lay J. E. Cronan <br />  <br /> bind normally abundant protein(s). This                        Crowell, D. N., M. S. Anderson C. R. Raetz, 1986 Molecular <br />                                                                                 cloning genes lipid A disaccharide synthase UDP-N- <br /> protein (or proteins) trapped                              acetylglucosamine acyltransferase Escherichia coli. J. Bacteriol. <br /> AcpT aggregates, depleting cells func-                          168: 152&#226;&#8364;&#8220;159. <br /> tional AcpT-binding protein(s). The weakness                       Daley, D. O., M. Rapp, E. Granseth, K. Melen, D. Drew et al., <br /> and possible explanations consid-                            2005 Global topology analysis Escherichia coli inner mem- <br />                                                                                 brane proteome. Science 308: 1321&#226;&#8364;&#8220;1323. <br /> ered fail provide plausible mechanism                     Datsenko, K. A., B. L. Wanner, 2000 One-step inactivation of <br /> whereby overexpression normally cytosolic protein                          chromosomal genes Escherichia coli K-12 using PCR products. <br /> allows growth E. coli cells lack essential mem-                      Proc. Natl. Acad. Sci. USA 97: 6640&#226;&#8364;&#8220;6645. <br />                                                                             De Lay, N. R., J. E. Cronan, 2006 A genome rearrangement has <br /> brane protein.                                                                  orphaned Escherichia coli K-12 AcpT phosphopantetheinyl <br />    The function YejM remains obscure. Somehow loss                           transferase cognate Escherichia coli O157:H7 substrates. <br /> of periplasmic domain inner membrane                                Mol. Microbiol. 61: 232&#226;&#8364;&#8220;242. <br />                                                                             Ellermeier, C. D., A. Janakiraman J. M. Slauch, 2002 Con- <br /> protein results increased permeability outer                       struction targeted single copy lac fusions using lambda Red <br /> membrane exacerbated increased growth tem-                           FLP-mediated site-specific recombination bacteria. Gene <br /> perature. The reported phenotype reduced lipid A                             290: 153&#226;&#8364;&#8220;161. <br />                                                                             Flugel, R. S., Y. Hwangbo, R. H. Lambalot, J. E. Cronan, Jr. and <br /> biosynthesis strain LH530 (Hirvas et al. 1997; Vaara                         C. T. Walsh, 2000 Holo-(acyl carrier protein) synthase and <br /> and Nurminen 1999) complicated possibility                         phosphopantetheinyl transfer Escherichia coli. J. Biol. Chem. <br /> translocation phospholipids outer leaflet                        275: 959&#226;&#8364;&#8220;968. <br />                                                                             Galloway, S. M., C. R. Raetz, 1990 A mutant Escherichia coli <br /> the outer membrane known occur strains carry                         defective step endotoxin biosynthesis. J. Biol. <br /> mutations affecting organization lipopolysaccha-                         Chem. 265: 6394&#226;&#8364;&#8220;6402. <br /> ride cell surface (Wu et al. 2006). Hence,                     Gerdes, S. Y., M. D. Scholle, J. W. Campbell, G. Balazsi, E. Ravasz <br />                                                                                 et al., 2003 Experimental determination level anal- <br /> lipid A synthesis assayed using phospholipid synthe-                        ysis essential genes Escherichia coli MG1655. J. Bacteriol. 185: <br /> sis standard, results previously reported                     5673&#226;&#8364;&#8220;5684. <br /> at partially increased phospholipid synthesis                  Gross, C. A., 1996 Function regulation heat shock proteins, <br />                                                                                 pp. 1382&#226;&#8364;&#8220;1399 Escherichia coli Salmonella typhimurium: Cel- <br /> rather decreased lipid A biosynthesis. Indeed,                          lular Molecular Biology, edited F. C. Neidhardt, III, J. L. <br /> the accumulation phospholipids outer leaflet                       Ingraham, E. C. C. Lin, K. B. Low, B. Magasanik, W. S. Reznikoff, <br /> thought render cells sensitive imperme-                         M. Riley, M. Schaechter H. E. Umbarger. American Society <br /> able antibiotics (Raetz et al. 2007), phenotype strain                     Microbiology, Washington, DC. <br />                                                                             Gur, E., D. Biran, N. Shechter, P. Genevaux, C. Georgopoulos <br /> LH530. Increased sensitivity antibiotics                        et al., 2004 The Escherichia coli DjlA CbpA proteins substi- <br /> seen lipid A mutants. However, known lipid A                      tute DnaJ DnaK-mediated protein disaggregation. J. Bacteriol. <br /> mutants marked alterations compositions                          186: 7236&#226;&#8364;&#8220;7242. <br />                                                                             Guzman, L. M., D. Belin, M. J. Carson J. Beckwith, 1995 Tight <br /> their lipid A species yejM mutant strains                      regulation, modulation, high-level expression vectors con- <br /> essentially normal lipid A compositions (C. Michael                             taining arabinose PBAD promoter. J. Bacteriol. 177: 4121&#226;&#8364;&#8220;4130. <br /> Reynolds, personal communication).                                          Hirvas, L., M. Nurminen, I. M. Helander, R. Vuorio M. Vaara, <br />                                                                                 1997 The lipid A biosynthesis deficiency Escherichia coli <br />   We thank C. Michael Reynolds Christian Raetz De-                   antibiotic-supersensitive mutant LH530 suppressed novel <br /> partment Biochemistry, Duke University, lipid A analyses         locus, ORF195. Microbiology 143: 73&#226;&#8364;&#8220;81. <br /> Coli Genetic Stock Center strains. This work supported           Kang, Y., T. Durfee, J. D. Glasner, Y. Qiu, D. Frisch et al., 2004 Sys- <br /> National Institutes Health grant AI15650 National                   tematic mutagenesis Escherichia coli genome. J. Bacteriol. <br /> Institute Allergy Infectious Diseases.                                   186: 4921&#226;&#8364;&#8220;4930. <br />                                                                             Kelly, T. M., S. A. Stachula, C. R. Raetz M. S. Anderson, <br />                                                                                 1993 The firA gene Escherichia coli encodes UDP-3-O-(R-3- <br />                                                                                 hydroxymyristoyl)-glucosamine N-acyltransferase. The step <br />                                                                                 endotoxin biosynthesis. J. Biol. Chem. 268: 19866&#226;&#8364;&#8220;19874. <br />                       LITERATURE CITED                                      Kloser, A. W., M. W. Laird R. Misra, 1996 asmB, suppressor <br />                                                                                 locus assembly-defective OmpF mutants Escherichia coli, is <br /> Baba, T., T. Ara, M. Hasegawa, Y. Takai, Y. Okumura et al., <br />                                                                                 allelic envA (lpxC). J. Bacteriol. 178: 5138&#226;&#8364;&#8220;5143. <br />     2006 Construction Escherichia coli K-12 in-frame, single-gene        Lambalot, R. H., A. M. Gehring, R. S. Flugel, P. Zuber, M. LaCelle <br />     <span id='am-319' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-320' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-321' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-322' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-323' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-324' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-325' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-326' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-327' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-328' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-329' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-330' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-331' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-332' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-333' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-334' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-335' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-337' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-338' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-340' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-341' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-342' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-344' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-347' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-348' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-349' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-350' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-351' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-352' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-353' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-354' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-355' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-356' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-357' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-358' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-359' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-362' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-363' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-364' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-365' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-366' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-367' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-368' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-369' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-370' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-372' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-373' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-374' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-375' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-376' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-377' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-378' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-379' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-380' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-381' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-382' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-383' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-384' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-385' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-386' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-387' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-388' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-389' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-390' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-391' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-392' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-393' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-394' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-395' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-396' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span>knockout</span> mutants: Keio collection. Mol. Syst. Biol. 2: 2006.0008.       et al., 1996 A new enzyme superfamily: phosphopantetheinyl <br /> Bardwell, J. C., K. Tilly, E. Craig, J. King, M. Zylicz et al.,                 transferases. Chem. Biol. 3: 923&#226;&#8364;&#8220;936. <br />     1986 The nucleotide sequence Escherichia coli K12 dnaJ1          Lesley, S. A., J. Graziano, C. Y. Cho, M. W. Knuth H. E. Klock, <br />     gene. A gene encodes heat shock protein. J. Biol. Chem.              2002 Gene expression response misfolded protein screen <br />     261: 1782&#226;&#8364;&#8220;1785.                                                             soluble recombinant protein. Protein Eng. 15: 153&#226;&#8364;&#8220;160. <br /> Bjornsson, A., L. A. Isaksson, 1993 UGA codon context             Miller, J. H., 1972 Experiments Molecular Genetics. Cold Spring <br />     spans codons: reversal ms2i6A37 tRNA, mutation               Harbor Laboratory Press, Cold Spring Harbor, NY. <br />     rpsD(S4) streptomycin. J. Mol. Biol. 232: 1017&#226;&#8364;&#8220;1029.                 Mofid, M. R., M. A. Marahiel, R. Ficner K. Reuter, <br /> Cowing, D. W., J. C. Bardwell, E. A. Craig, C. Woolford, R. W.                  1999 Crystallization preliminary crystallographic studies <br />     Hendrix et al., 1985 Consensus sequence Escherichia coli                Sfp: phosphopantetheinyl transferase modular peptide <br />     heat shock gene promoters. Proc. Natl. Acad. Sci. USA 82: 2679&#226;&#8364;&#8220;             synthetases. Acta Crystallogr. D Biol. Crystallogr. 55: 1098&#226;&#8364;&#8220;1100. <br />     2683.                                                                   Mofid, M. R., R. Finking, L. O. Essen M. A. Marahiel, <br /> Cronan, J. E., 2003 Cosmid-based transient expression                2004 Structure-based mutational analysis 49-phosphopante- <br />     absolute off-to-on transcriptional control Escherichia coli          theinyl transferases Sfp Bacillus subtilis: carrier protein recogni- <br />     genes. J. Bacteriol. 185: 6522&#226;&#8364;&#8220;6529.                                        tion reaction mechanism. Biochemistry 43: 4128&#226;&#8364;&#8220;4136. <br /> Cronan, J. E., 2006 A family arabinose-inducible Escherichia coli        Mootz, H. D., R. Finking M. A. Marahiel, 2001 49-Phospho- <br />     expression vectors having pBR322 copy control. Plasmid 55:                  pantetheine transfer primary secondary metabolism Ba- <br />     152&#226;&#8364;&#8220;157.                                                                    cillus subtilis. J. Biol. Chem. 276: 37289&#226;&#8364;&#8220;37298. <br />                                                                 Phenotype E. coli yejM                                                       1337 <br />  <br /> Nichols, B. P., O. Shafiq V. Meiners, 1998 Sequence analysis              Thomas, J., J. E. Cronan, 2005 The enigmatic acyl carrier pro- <br />     Tn10 insertion sites collection Escherichia coli strains            tein phosphodiesterase Escherichia coli: genetic enzymolog- <br />     used genetic mapping strain construction. J. Bacteriol.                ical characterization. J. Biol. Chem. 280: 34675&#226;&#8364;&#8220;34683. <br />     180: 6408&#226;&#8364;&#8220;6411.                                                           Thomas, J., D. J. Rigden J. E. Cronan, 2007 Acyl carrier protein <br /> Normark, S., 1970 Genetics chain-forming mutant Escheri-                   phosphodiesterase (AcpH) Escherichia coli non-canonical <br />     chia coli: transduction dominance envA gene mediat-                 member HD phosphatase/phosphodiesterase family. Bio- <br />     ing increased penetration antibacterial agents. Genet.                 chemistry 46: 129&#226;&#8364;&#8220;136. <br />     Res. 16: 63&#226;&#8364;&#8220;78.                                                           Tsuruoka, T., M. Ito, S. Tomioka, A. Hirata M. Matsuhashi, <br /> Nurminen, M., L. Hirvas M. Vaara, 1997 The outer mem-                          1988 Thermosensitive omsA mutation Escherichia coli that <br />     brane lipid A-deficient Escherichia coli mutant LH530 re-               causes thermoregulated release periplasmic proteins. J. Bacter- <br />     duced levels OmpF leaks periplasmic enzymes.                            iol. 170: 5229&#226;&#8364;&#8220;5235. <br />     Microbiology 143: 1533&#226;&#8364;&#8220;1537.                                              Vaara, M., M. Nurminen, 1999 Outer membrane permeability <br /> Oh, M. K., J. C. Liao, 2000 DNA microarray detection met-                   barrier Escherichia coli mutants defective late <br />     abolic responses protein overproduction Escherichia coli.                acyltransferases lipid A biosynthesis. Antimicrob. Agents Che- <br />     Metab. Eng. 2: 201&#226;&#8364;&#8220;209.                                                        mother. 43: 1459&#226;&#8364;&#8220;1462. <br /> Parris, K. D., L. Lin, A. Tam, R. Mathew, J. Hixon et al., 2000 Crystal       Wu, T., A. C. McCandlish, L. S. Gronenberg, S. S. Chng, T. J. <br />     structures substrate binding Bacillus subtilis holo-(acyl carrier        Silhavy et al., 2006 Identification protein complex as- <br />     protein) synthase reveal novel trimeric arrangement mole-                 sembles lipopolysaccharide outer membrane Escherichia <br />     cules resulting active sites. Structure 8: 883&#226;&#8364;&#8220;895.                   coli. Proc. Natl. Acad. Sci. USA 103: 11754&#226;&#8364;&#8220;11759. <br /> Raetz, C. R., C. M. Reynolds, M. S. Trent R. E. Bishop,                   Yanisch-Perron, C., J. Vieira J. Messing, 1985 Improved M13 <br />     2007 Lipid A modification systems gram-negative bacteria.                   phage cloning vectors host strains: nucleotide sequences of <br />     Annu. Rev. Biochem. 76: 295&#226;&#8364;&#8220;329.                                               M13mp18 pUC19 vectors. Gene 33: 103&#226;&#8364;&#8220;119. <br /> Reuter, K., M. R. Mofid, M. A. Marahiel R. Ficner,                        Young, K., L. L. Silver, D. Bramhill, P. Cameron, S. S. Eveland <br />     1999 Crystal structure surfactin synthetase-activating en-              et al., 1995 The envA permeability/cell division gene Escher- <br />     zyme Sfp: prototype 49-phosphopantetheinyl transferase                ichia coli encodes second enzyme lipid A biosynthesis. <br />     superfamily. EMBO J. 18: 6823&#226;&#8364;&#8220;6831.                                            UDP-3-O-(R-3-hydroxymyristoyl)-N-acetylglucosamine deacety- <br /> Roth, J. R., N. Benson, T. Galitski, K. Haack, J. G. Lawrence et al.,              lase. J. Biol. Chem. 270: 30384&#226;&#8364;&#8220;30391. <br />     1996 Rearrangements bacterial chromosome: formation                Yu, D., H. M. Ellis, E. C. Lee, N. A. Jenkins, N. G. Copeland et al., <br />     applications, pp. 2256&#226;&#8364;&#8220;2276 Escherichia coli Salmonella ty-         2000 An efficient recombination chromosome engi- <br />     phimurium: Cellular Molecular Biology, edited F. C. Neidhardt,          neering Escherichia coli. Proc. Natl. Acad. Sci. USA 97: 5978&#226;&#8364;&#8220; <br />     III, J. L. Ingraham, E. C. C. Lin, K. B. Low, B. Magasanik, W. S.              5983. <br />     Reznikoff, M. Riley, M. Schaechter H. E. Umbarger.                    Zhou, Z., S. Lin, R. J. Cotter C. R. Raetz, 1999 Lipid A mod- <br />     American Society Microbiology, Washington, DC.                             ifications characteristic Salmonella typhimurium induced by <br /> Singer, M., T. A. Baker, G. Schnitzler, S. M. Deischel, M. Goel                    NH4VO3 Escherichia coli K12. Detection 4-amino-4-deoxy-L- <br />     et al., 1989 A collection strains containing genetically linked             arabinose, phosphoethanolamine palmitate. J. Biol. Chem. <br />     alternating antibiotic resistance elements genetic mapping              274: 18503&#226;&#8364;&#8220;18514. <br />     Escherichia coli. Microbiol. Rev. 53: 1&#226;&#8364;&#8220;24. <br /> Smith, H. E., 2007 The transcriptional <br /> </body></html>