<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>  <br />                               Axonal transport of mitochondria requires milton <br />                               <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> recruit kinesin heavy chain and is light <br />                               chain independent <br />                           Mitochondria are distributed within cells to                                milton&#226;&#8364;&#8220;KHC complexes, and mitochondria are present <br />                                         match local energy demands. We report that the                             in klc &#226;&#710;&#8217;/&#226;&#710;&#8217; photoreceptor axons. The recruitment of KHC <br />                                         microtubule-dependent transport of mitochondria                            to mitochondria is, in part, determined by the NH2 <br />  <br />  <br />  <br />  <br />  <br />                               depends on the ability of milton to act as an adaptor                                terminus&#226;&#8364;&#8220;splicing variant of milton. A direct interaction <br />                               protein that can recruit the heavy chain of conventional                             occurs between milton and miro, which is a mitochon- <br />                               kinesin-1 (kinesin heavy chain [KHC]) to mitochondria.                               drial Rho-like GTPase, and this interaction can in&#239;&#172;&#8218;uence <br />                               Biochemical and genetic evidence demonstrate that                                    the recruitment of milton to mitochondria. Thus, milton <br />                               kinesin recruitment and mitochondrial transport are                                  and miro are likely to form an essential protein complex <br />                               independent of kinesin light chain (KLC); KLC antago-                                that links KHC to mitochondria for light chain&#226;&#8364;&#8220;independent, <br />                               nizes milton&#226;&#8364;&#8482;s association with KHC and is absent <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span>                               anterograde transport of mitochondria. <br />  <br />  <br />  <br />                               Introduction <br />                               Mitochondrial localization and transport ensure the proper                           and Saxton, 2005). In metazoans, mitochondrial motility <br />                               inheritance of mitochondria upon cell division (Pereira et al.,                      involves both actin- and microtubule-dependent mechanisms <br />                               1997; Yaffe, 1999) and position mitochondria where energy                            (Morris and Hollenbeck, 1995; Hollenbeck, 1996; Ligon and <br />                               demands or oxygen supplies are greatest (Hollenbeck and                              Steward, 2000b; Hollenbeck and Saxton, 2005). In particular, <br />                               Saxton, 2005). It is likely that the concentration of local cyto-                    plus end&#226;&#8364;&#8220;directed movement involves conventional kinesin <br />                               plasmic Ca2+ also depends on mitochondrial Ca2+ uptake                               (kinesin-1) motors (Hurd and Saxton, 1996; Tanaka et al., 1998; <br />                               (Werth and Thayer, 1994; Zucker, 1999). Consequently, mito-                          Pilling et al., 2006), although kinesin-3 motors are also impli- <br />                               chondria accumulate in subcellular regions with high metabolic                       cated (Nangaku et al., 1994; Wozniak et al., 2005). Little is <br />                               requirements and high Ca2+ influx (Morris and Hollenbeck,                            known about how mitochondrial kinesin is regulated or coupled <br />                               1993) and redistribute in response to changes in the local energy                    to the organelle (Rintoul et al., 2003; Chada and Hollenbeck, <br />                               state (Wong-Riley and Welt, 1980; Hollenbeck, 1996). The                             2004; Miller and Sheetz, 2004; Cai et al., 2005; Hollenbeck and <br />                               transport of mitochondria is particularly vital in neurons because                   Saxton, 2005; Malaiyandi et al., 2005; Minin et al., 2006). <br />                               of their extended processes, and the disruption of mitochondrial                           We recently identified a novel protein called milton, <br />                               transport is correlated with neurodegenerative disease (Hollen-                      which is required for mitochondrial transport within Drosophila <br />                               beck and Saxton, 2005).                                                              melanogaster photoreceptors (Stowers et al., 2002). Mitochondria <br />                                     The mechanisms of mitochondrial transport differ be-                           were absent from milton (milt) photoreceptor axons, but were <br />                               tween species and can require actin, microtubule attachment, or                      normally distributed and appeared to be functional in their <br />                               kinesins (Yaffe et al., 2003; Boldogh et al., 2005; Hollenbeck                       cell bodies. Although devoid of mitochondria, their axons and <br />                                                                                                                    synapses were otherwise surprisingly normal in their gen- <br />                               E.E. Glater and L.J. Megeath are co-&#239;&#172;&#65533;rst authors.                                    eral architecture, possessing microtubules, synaptic vesicles, <br />                               Correspondence to Thomas L. Schwarz: Thomas.Schwarz@childrens.harvard.edu            and active zone specializations. Thus, the transport defect <br />                               Abbreviations used in this paper: GRIF, &#206;&#179;-aminobutyric acid A receptor&#226;&#8364;&#8220;interacting   was selective for mitochondria (Stowers et al., 2002; Gorska- <br />                               factor; HEK, human embryonic kidney; KHC, kinesin heavy chain; KLC, kinesin <br />                               light chain; OGT, O-GlcNAc transferase; OIP, O-linked N-acetylglucosamine&#226;&#8364;&#8220; <br />                                                                                                                    Andrzejak et al., 2003). The mechanism of milton&#226;&#8364;&#8482;s action <br />                               interacting protein.                                                                 was unknown, but milton was associated with mitochondria <br />                               The online version of this article contains supplemental material.                   and coimmunoprecipitated with kinesin heavy chain (KHC) <br />  <br />                               &#194;&#169; The Rockefeller University Press $8.00 <br />                               The Journal of Cell Biology, Vol. 173, No. 4, May 22, 2006 545&#226;&#8364;&#8220;557 <br />                               http://www.jcb.org/cgi/doi/10.1083/jcb.200601067 <br />                                                                                                                                                                                 JCB   545 <br />        in extracts of fly heads (Stowers et al., 2002). The mammalian       provide the clearest structures in which to image axonal <br />       homologues milton 1 and 2, which are also called O-linked            mitochondria. In control larvae, numerous mitochondria were <br />       N-acetylglucosamine&#226;&#8364;&#8220;interacting protein 106 (OIP106) and             present in these axons. However, in milt92 larvae, axonal mito- <br />       &#206;&#179;-aminobutyric acid A receptor&#226;&#8364;&#8220;interacting factor-1 (GRIF-1),        chondria were absent (Fig. 1, A and B). This defect is selective <br />       also colocalize with mitochondria and coimmunoprecipitate            for mitochondria, as indicated by the continued presence of <br />       with KIF5B, which is a mammalian homologue of D. mela-               immunoreactivity for KHC, which is likely to transport many <br />       nogaster KHC (Beck et al., 2002; Iyer et al., 2003; Brickley         cargoes (Goldstein, 2001; Vale, 2003), and the synaptic vesicle <br />       et al., 2005; Gilbert et al., 2006). Therefore, we have suggested    marker synaptotagmin (Fig. 1, C&#226;&#8364;&#8220;F). <br />       that milton acts as an adaptor or regulator of the mitochondrial           We also assayed mitochondrial distribution in the ventral <br />       anterograde motor.                                                   nerve cord, which consists of two central neuropil regions that <br />              We demonstrate a protein apparatus that recruits kinesin      run the <span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> of the cord and are surrounded by a cortex of cell <br />       to mitochondria and thereby permits anterograde movement.            bodies (Fig. 1, G and H). The neuropil regions contain axons, <br />       Milton, which interacts with both KHC and the mitochondrial          dendrites, and pre- and postsynaptic endings. In control larvae, <br />       protein miro, is essential in this apparatus. In contrast, kinesin   mitoGFP was present in the cell bodies, but was most abundant <br />       light chain (KLC) is dispensable for mitochondrial transport         in the neuropil, reflecting the increased concentration of mito- <br />       in axons.                                                            chondria at the synapses. In the milt92 mutant, the mitoGFP <br />                                                                            <span id='am-161' about='xsp:pattern' typeof='owl:Thing'>pattern</span> was reversed, with little GFP remaining in the neuropil. <br />       Results                                                              Synaptotagmin localization was unchanged. Thus, the selective <br />                                                                            loss of mitochondria from axons and synapses is not restricted <br />       Milton alters mitochondrial distribution                             to photoreceptors. Moreover, milton probably also mediates <br />       Flies that are homozygous for milton die as second instar larvae,    mitochondrial transport in dendrites because at least half of <br />       and milton transcripts are broadly expressed in these flies,         the mitochondria of the neuropil are expected to derive from <br />       suggesting a wider role for milton than its reported function in     postsynaptic elements. <br />       photoreceptors (Stowers et al., 2002). Therefore, we expressed <br />       GFP fused to a mitochondrial-import signal (mitoGFP; Pilling         Milton recruits KHC to mitochondria <br />       et al., 2006) in neurons of the central nervous system to exam-      The mitochondrial transport defect in milton mutants, and the in <br />       ine mitochondrial distribution in first instar larvae that are       vivo association between milton and KHC, suggests that milton <br />       homozygous for milt92, which is a <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span>null allele</span>. The segmental         is an adaptor that links KHC to mitochondria. To test this hypo- <br />       nerves that connect the central nervous system to the body           thesis, we transfected cDNAs encoding D. melanogaster milton <br />       wall of the larva contain motor and sensory axons, and thereby       (Stowers et al., 2002) and myc-tagged rat KIF5B (myc-KHC; <br />  <br />  <br />  <br />  <br />       Figure 1. Homozygous milton larvae lack <br />       axonal and synaptic mitochondria. The dis- <br />       tributions of mitochondria, synaptic vesicles, <br />       and kinesin in &#239;&#172;&#65533;rst instar FRT40, milt92/FRT40, <br />       milt92 larvae (bottom row), and FRT40/FRT40 <br />       control larvae (top row). (A&#226;&#8364;&#8220;F) Examination of <br />       peripheral nerves that extend from the poste- <br />       rior of the nerve cord to the body-wall muscles. <br />       (A and B) GFP-labeled mitochondria are <br />       abundant in the peripheral axons in controls, <br />       but absent from milton axons. In contrast, anti- <br />       synaptotagmin (syt; C and D) and anti-KHC <br />       (E and F) immunolabeling of the same prepara- <br />       tion is indistinguishable between control and <br />       mutant larvae. (G&#226;&#8364;&#8220;J) Confocal images through <br />       the middle of the ventral nerve cord. (G) To <br />       either side of the midline, the synapse-rich neu- <br />       ropil is enriched for mitochondria (mitoGFP) <br />       in control larvae, relative to the surrounding <br />       cell-body region. (H) In milton nerve cords, the <br />       mitoGFP is primarily in cell bodies. (I and J) <br />       Synaptotagmin is concentrated in the neuropil <br />       of both genotypes. <br />  <br />  <br /> 546   JCB &#226;&#8364;&#162; VOLUME 173 &#226;&#8364;&#162; NUMBER 4 &#226;&#8364;&#162; 2006 <br />  Verhey et al., 1998), either alone or together, into COS7 cells.             In addition, mitochondrial distribution was altered by the <br /> D. melanogaster milton and its mammalian homologues func-              cotransfection of milton and KHC. When milton alone was <br /> tion identically in all of our assays (Fig. S2, available at http://   highly expressed, the mitochondria were clustered near the <br /> www.jcb.org/cgi/content/full/jcb.200601067/DC1; see Mam-               nucleus in 90% of the transfected cells (Fig. 2 B), which is a <br /> malian milton homologues); therefore, we have used the rat             phenomenon encountered in only 3% of control cells. At lower <br /> kinesin in these assays. Transfected alone, milton immuno-             expression levels, the mitochondria remained distributed as in <br /> reactivity was located exclusively on mitochondria (Fig. 2 B).         untransfected cells (not depicted). Overall, milton expression <br /> In untransfected cells, the endogenous KHC was typically cyto-         caused less clustering of mitochondria in COS7 cells than had <br /> plasmic, although in some cells KHC was also observed on               previously been observed in human embryonic kidney 293T <br /> mitochondria. Upon transfection with milton, the KHC became            (HEK293T) cells, in which all of the mitochondria become <br /> highly enriched on the mitochondria, with little remaining             localized in an aggregate near the microtubule-organizing cen- <br /> detectable elsewhere in the cell (Fig. 3 C). The phenomenon            ter (Stowers et al., 2002). In contrast, coexpression of KHC and <br /> was more dramatic when rat myc-KHC was overexpressed in                milton caused many, though not all, mitochondria to reside at <br /> these cells; myc-KHC was largely cytoplasmic, and, in some             the cell margin and to form clumps at the tips of cell processes. <br /> highly expressing cells, colocalized with microtubules (Fig. 2 C),     This redistribution was not caused by a general change in the <br /> as previously observed (Verhey et al., 1998). However, when            cytoskeleton. Neither KHC nor milton, transfected singly or in <br /> cells were cotransfected with both milton and myc-KHC, KHC             combination, altered the arrangement of microtubules in these <br /> was overwhelmingly located on mitochondria, colocalizing pre-          cells (Fig. S1, available at http://www.jcb.org/cgi/content/full/ <br /> cisely with both MitoTracker and milton (64 out of 66 cells;           jcb.200601067/DC1). The microtubules are chiefly oriented <br /> Fig. 2, D1 and D2).                                                    with plus ends toward the periphery of the cell; thus, the <br />  <br />  <br />  <br />  <br />                                                                                      Figure 2. Milton recruits KHC to mitochondria. Trans- <br />                                                                                      fected COS7 cells were immunostained with anti-milton <br />                                                                                      mAb 5A124 (green) and anti-myc to detect myc-KHC <br />                                                                                      (white). Mitochondria were labeled with MitoTracker <br />                                                                                      orange (pink). The localization of the proteins was com- <br />                                                                                      pared between sham-transfected cells (A) and those trans- <br />                                                                                      fected with milton (B), myc-KHC (C), and milton and <br />                                                                                      myc-KHC (D1 and D2). KHC is present in the cytosol and <br />                                                                                      on microtubules in the absence of milton (B), but is <br />                                                                                      recruited to mitochondria by the presence of milton <br />                                                                                      (D1 and D2). Coexpression of these proteins caused <br />                                                                                      mitochondria to form aggregates (arrows) that were fre- <br />                                                                                      quently, though not always, at the periphery of the cell. <br />                                                                                      Bar, 20 &#206;&#188;m. <br />  <br />  <br />                                                      MITOCHONDRIAL MOTILITY VIA MILTON, MIRO, AND KHC &#226;&#8364;&#162; GLATER ET AL.                            547 <br />        Figure 3. Milton recruits endogenous KHC to mitochondria. <br />       COS7 cells were immunostained with anti-KHC (white) <br />       and anti-milton (green), and mitochondria were labeled <br />       with MitoTracker orange (pink). (A and B) Control cells <br />       differed in the distribution of endogenous KHC. It was <br />       enriched on mitochondria in some (A), and had a more <br />       cytoplasmic distribution in others (B). (C) In contrast, in <br />       cells transfected with milton, endogenous KHC always <br />       colocalized with mitochondria, and little or no KHC could <br />       be detected elsewhere. Bar, 20 &#206;&#188;m. <br />  <br />  <br />  <br />  <br />       redistribution of mitochondria suggests that milton has recruited     truncated myc-KHC constructs and then immunoprecipitating <br />       and activated coexpressed KHC and thereby caused a plus end&#226;&#8364;&#8220;          with anti-milton mAb 5A124. Milton associated with both full- <br />       directed shift of many of the mitochondria. Together with the         length KHC and KHC lacking the last 64 amino acids of the tail <br />       biochemical association of milton and KHC in D. melanogaster          domain (myc-KHC 1&#226;&#8364;&#8220;891), but not with KHC lacking the entire <br />       and the mitochondrial localization of milton (Stowers et al.,         tail domain (myc-KHC 1&#226;&#8364;&#8220;810) or a larger deletion (myc-KHC <br />       2002), the ability of milton to recruit KHC to mitochondria offers    1&#226;&#8364;&#8220;682; Fig. 4 B). Therefore, the KHC tail region 810&#226;&#8364;&#8220;891, but <br />       direct support for the hypothesis that milton is a mitochondria-      not 892&#226;&#8364;&#8220;955, was necessary for associating with milton. <br />       specific adaptor protein for the kinesin-1 family.                    Consistent with this finding, we observed that myc-KHC 1&#226;&#8364;&#8220;891, <br />                                                                             but not 1&#226;&#8364;&#8220;682, could be recruited to mitochondria by milton <br />       Mapping the requirements for milton&#226;&#8364;&#8220;KHC                               overexpression (Fig. 4 D). <br />       interactions <br />       To further characterize the interaction between milton and myc-       Milton&#226;&#8364;&#8220;KHC interaction is KLC independent <br />       KHC, we mapped the region of milton that is required for their        The kinesin-1 family, including D. melanogaster KHC and <br />       association by cotransfecting HEK293T cells and assaying the          mammalian KIF5s, are generally considered to be tetramers <br />       ability of portions of milton to coprecipitate with full-length       composed of two KHCs and two KLCs. The deletion of amino <br />       myc-KHC. Milton comprises 1,116 amino acids with no recog-            acids 810&#226;&#8364;&#8220;891 of KHC diminishes KLC binding (Verhey et al., <br />       nizable structural motifs, except for a long, predicted coiled coil   1998), and because endogenous KLC was present in the afore- <br />       domain (residues 140&#226;&#8364;&#8220;380) that contains a high degree of amino        mentioned experiments, milton might interact either directly <br />       acid identity with the equivalent regions in mammalian milton         with KHC or via KLC. Therefore, rat HA-tagged KLC1 (iso- <br />       homologues (Stowers et al., 2002). Milton 1&#226;&#8364;&#8220;450 was sufficient        form C; Verhey et al., 1998) was transfected into COS7 cells <br />       to associate with myc-KHC. Milton 1&#226;&#8364;&#8220;750 also coimmuno-                both alone and in combination with KHC and milton (Fig. 5). <br />       precipitated with KHC, but the COOH-terminal domain of milton         Alone, or when transfected with milton, KLC was cytoplasmic <br />       (Flag-tagged milton 750&#226;&#8364;&#8220;1,116 and 847&#226;&#8364;&#8220;1,116) did not (Fig. 4 A).      (Fig. 5 A; Verhey et al., 1998). Even when milton, myc-KHC, <br />       We also mapped the interaction by looking for colocalization in       and HA-KLC were coexpressed, KLC was invariably cytoplas- <br />       transfected COS7 cells, whose larger cytoplasmic volume made          mic and not located on mitochondria (Fig. 5 D). Moreover, in <br />       colocalization easier to score than in HEK293T cells. Milton          these cells, myc-KHC was also always cytoplasmic (Fig. 5 E). <br />       1&#226;&#8364;&#8220;450 colocalized with myc-KHC in aggregates, but these were          Thus, KLC expression inhibited the recruitment of KHC to the <br />       not on mitochondria. In contrast, Flag-milton 750&#226;&#8364;&#8220;1,116 did           mitochondria by milton. We demonstrated by coimmunoprecip- <br />       localize to mitochondria, but in its presence myc-KHC remained        itation that KLC similarly inhibited the interaction of milton <br />       cytoplasmic (Fig. 4 C). Thus, the associations of milton with         and KHC; HA-KLC expression prevented the coprecipitation of <br />       KHC and mitochondria are separable, and the 1&#226;&#8364;&#8220;450 region of           myc-KHC and milton (Fig. 5, F and G). Although not signifi- <br />       milton is sufficient for the interaction with KHC.                    cantly homologous, the KHC-binding regions of milton and <br />              We determined the region of KHC necessary for its asso-        KLC are both predicted coiled coils and, therefore, may have <br />       ciation with milton by cotransfecting milton with each of three       similar and competing interactions for a binding site on KHC. <br />  <br />  <br /> 548   JCB &#226;&#8364;&#162; VOLUME 173 &#226;&#8364;&#162; NUMBER 4 &#226;&#8364;&#162; 2006 <br />                                                                                           These observations strongly suggested that KLC is not a <br />                                                                                   part of the milton&#226;&#8364;&#8220;KHC complex and that milton replaces KLC <br />                                                                                   when associating KHC with mitochondria; therefore, we tested <br />                                                                                   this hypothesis in vivo by biochemical and genetic means. <br />                                                                                   As we previously observed, KHC immunoprecipitated with milton <br />                                                                                   from homogenates of fly heads (Stowers et al., 2002). In these <br />                                                                                   same precipitates, however, KLC was not detected. To increase <br />                                                                                   the sensitivity of the assay, we immunoprecipitated milton from <br />                                                                                   flies overexpressing myc-KLC (Gindhart et al., 1998). Again, <br />                                                                                   KHC coimmunoprecipitated with milton, but KLC did not <br />                                                                                   (Fig. 5 H). Similarly, the immunoprecipitation of myc-KLC <br />                                                                                   with anti-myc brought down KHC, but milton remained in the <br />                                                                                   supernatant. Immunoprecipitation with antibodies to KHC, <br />                                                                                   however, brought down both myc-KLC and milton. Thus, KHC <br />                                                                                   appeared to form separate complexes either with milton or with <br />                                                                                   KLC; the latter are likely to be more plentiful because of the <br />                                                                                   greater abundance of cargoes requiring KLC. Because the <br />                                                                                   milton&#226;&#8364;&#8220;KHC complex did not contain KLC, we hypothesize <br />                                                                                   that milton substitutes for KHC in mitochondrial transport. <br />                                                                                          To further test the hypothesis that mitochondrial transport <br />                                                                                   was KLC independent, we examined mutants lacking klc, which <br />                                                                                   is the unique KLC gene in D. melanogaster (Gindhart et al., <br />                                                                                   1998). In contrast to milton larvae (Fig. 1), the peripheral nerves <br />                                                                                   of homozygous klc larvae had abundant mitochondria (not <br />                                                                                   depicted). Maternally contributed KLC, however, might account <br />                                                                                   for the absence of a <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span>phenotype</span>. Therefore, as a more rigorous <br />                                                                                   test, we made eye clones homozygous null for klc (klc8ex94) in a <br />                                                                                   heterozygous background using the EGUF/hid method (Stowers <br />                                                                                   and Schwarz, 1999). Loss of klc did not prevent the differentia- <br />                                                                                   tion and viability of photoreceptors, although eyes were some- <br />                                                                                   what small and roughened, and their axons were frequently <br />                                                                                   disordered and sometimes short and defasciculated. Neverthe- <br />                                                                                   less, axonal mitochondria were numerous, just as in controls, in <br />                                                                                   third instar larvae (Fig. 6, A&#226;&#8364;&#8220;D) and in adults (not depicted). <br />                                                                                   Thus, axonal transport of mitochondria can occur in the absence <br />                                                                                   of KLC. In contrast, the axons of milton-null photoreceptors <br /> Figure 4. The interaction of milton and KHC. (A) Coimmunoprecipitation of         lacked mitochondria (Fig. 6 D). <br /> truncated milton and myc-KHC from transfected HEK293T cells. (left) Cells <br /> were transfected with myc-tagged KHC and either full-length milton, milton <br /> 1&#226;&#8364;&#8220;450, or milton 1&#226;&#8364;&#8220;750. Total cell lysate was immunoprecipitated with anti-       Milton splice variants are functionally <br /> milton 2A108 (raised against amino acids 273&#226;&#8364;&#8220;450) and probed with                 distinct <br /> 2A108 and anti-myc. (right) Cotransfection with myc-KHC and either full- <br /> length milton or Flag-milton 750&#226;&#8364;&#8220;1,116. Total cell lysate was immunoprecipi-      Although, thus far, only a single milton cDNA; which is here- <br /> tated with anti-milton mAb 5A124 (raised against residues 908&#226;&#8364;&#8220;1,055) and          after called milton-A, has been described (Stowers et al., 2002) <br /> then probed with 5A124 and anti-myc (KHC). (B) Coimmunoprecipitation of           and used in this paper, protein and RNA analysis suggested <br /> truncated myc-KHC and milton from lysates of HEK293T cells. Cells were <br /> transfected with milton and myc-KHC deletion constructs as indicated and          greater complexity. Using cDNAs and ESTs from the Berkeley <br /> immunoprecipitated with anti-milton mAb 5A124, and then the precipitate           Drosophila Genome Project, we found that milton splice vari- <br /> was probed with 5A124 and anti-myc. (A and B) Expression of KHC in all            ants can produce at least four distinct protein products with <br /> of the lysates was con&#239;&#172;&#65533;rmed by probing the original lysate (Input) with anti- <br /> myc (KHC). In this and all subsequent &#239;&#172;&#65533;gures, the autoradiograph exposures        divergent NH2 termini. After differing 5&#226;&#8364;&#178; ends, the transcripts <br /> were chosen to show relative loading and protein expression levels, rather        converge in exon 9, at amino acid 129 of milton-A (Fig. 7 A). <br /> than to show the proportion of the protein in the lysate that was precipitated.   Anti-milton P1&#226;&#8364;&#8220;152 antibody, which was raised against amino <br /> (C) Milton 1&#226;&#8364;&#8220;450, but not milton 750&#226;&#8364;&#8220;1,116, colocalizes with KHC in <br /> COS7 cells. Anti-milton 2A108 (green) and anti-myc (KHC, white) <span id='am-2' about='rdfs:label' typeof='owl:Thing'>label</span>             acids 1&#226;&#8364;&#8220;152 of milton-A, recognized a single band on a Western <br /> identical regions of cytoplasm (arrows) when KHC is expressed with milton         blot of D. melanogaster extract, but anti-milton mAb 5A124, <br /> 1&#226;&#8364;&#8220;450, and these aggregates do not correspond to mitochondria labeled             which was raised against a domain present in all the predicted <br /> with MitoTracker orange (pink). Flag-tagged milton 750&#226;&#8364;&#8220;1,116 (visualized <br /> with anti-milton mAb 5A124; green) and myc-KHC (white) do not colocalize,         splice variants (milton-A 908&#226;&#8364;&#8220;1,055), recognized multiple <br /> as only the former appears mitochondrial, whereas KHC remains cytoplasmic.        bands (Stowers et al., 2002). These observations are consistent <br /> (D) KHC 1&#226;&#8364;&#8220;891, but not KHC 1&#226;&#8364;&#8220;682, is recruited to mitochondria by milton.         with expression of splice variants in vivo. <br /> The shorter KHC remains cytoplasmic in a milton-expressing cell, whereas <br /> KHC 1&#226;&#8364;&#8220;891 is associated with mitochondria, which tend to be aggregated                  The 129 amino acids of the NH2 terminus of milton-A are <br /> at the margin of the cell (arrows). Bars, 20 &#206;&#188;m.                                  replaced by 136 amino acids in milton-B and 269 amino acids in <br />  <br />  <br />                                                              MITOCHONDRIAL MOTILITY VIA MILTON, MIRO, AND KHC &#226;&#8364;&#162; GLATER ET AL.                           549 <br />        Figure 5. Mitochondrial transport by milton and KHC is independent of KLC. (A&#226;&#8364;&#8220;E) COS7 cells, which were transfected as indicated, were immunostained <br />       with anti-milton mAb 5A124 (green) and anti-myc to detect myc-KHC (white), or with anti-HA to detect HA-KLC (white). Mitochondria were labeled with <br />       MitoTracker orange (pink). (A and B) KLC remains diffuse and cytosolic, whether or not milton is expressed. (C) In contrast, in the presence of milton, KHC <br />       was recruited to mitochondria, and, as shown in Fig. 2, mitochondria formed aggregates at the periphery of the cell (arrows). (D and E) When KLC and <br />       KHC are both expressed with milton, neither is concentrated on mitochondria, and the mitochondria remain dispersed. Bar, 20 &#206;&#188;m. (F and G) Milton and <br />       myc-KHC do not coimmunoprecipitate in the presence of KLC. HEK293T cells were transfected with milton, myc-tagged KHC, and HA-KLC as indicated. The <br />       cell lysate was immunoprecipitated with anti-myc (KHC; F) or anti-milton mAb 5A124 (G), and immunoblots of both starting lysate (Input) and precipitates <br />       were probed with anti-myc, 5A124, and anti-HA. Milton and KHC no longer coprecipitate when KLC is coexpressed. (H) KLC is not part of the milton&#226;&#8364;&#8220;KHC <br />       complex in vivo. Transgenic &#239;&#172;&#8218;ies expressing myc-KLC were homogenized and immunoprecipitated with milton, anti-myc and anti-KHC. The starting homoge- <br />       nate and the precipitates (as labeled for each lane) were assayed as in F and G. Although milton and KHC coprecipitated with one another, KLC was associ- <br />       ated with KHC, but not with milton. All samples were from one experiment and processed together, although they were run on more than one gel. <br />  <br />  <br />       milton-C (Fig. 7 A). These domains have little homology to either               contained within the sequences that are shared by milton-A and <br />       milton-A, or to one another. In milton-D, the NH2 terminus do-                  the shorter milton-D. Together with our earlier data (Fig. 4 A), <br />       main is replaced by an untranslated region, such that a translation             these results indicate that the KHC association domain resides <br />       start site corresponding to Met 138 of milton-A is predicted within             within the region corresponding to 138&#226;&#8364;&#8220;450 of milton-A. Because <br />       exon 9. All of the predicted variants contain the predicted coiled              this sequence is also present in milton-B and -C, these isoforms <br />       coil domain. The National Center for Biotechnology Information                  were likewise expected to associate with KHC and recruit it to <br />       databases of GenBank mRNAs and ESTs for human milton                            mitochondria when equivalent amounts were expressed. Surpris- <br />       1/OIP106 predict alternative NH2 termini that converge at the                   ingly, this was only true for milton-B (Fig. 7, B and F); milton-C <br />       same point as the D. melanogaster variants, but no alternative NH2              neither recruited cotransfected myc-KHC to mitochondria nor <br />       termini are predicted for mammalian milton 2/GRIF-1 as yet.                     coimmunoprecipitated with myc-KHC (Fig. 7, B and G). <br />              Do all of the splice variants function equally in recruiting <br />       KHC to mitochondria? Each variant localized to mitochondria                     Mammalian milton homologues <br />       when transfected into COS7 cells (Fig. 7, C&#226;&#8364;&#8220;E). They differed,                  Milton&#226;&#8364;&#8482;s mammalian homologues function similarly to <br />       however, in their interactions with KHC. Milton-D recruited                     D. melanogaster milton, localizing to mitochondria, coprecipi- <br />       myc-KHC to the mitochondria and coimmunoprecipitated with                       tating with KHC, and recruiting KHC to mitochondria (Fig. S2; <br />       it (Fig. 7, B and H). Thus, milton&#226;&#8364;&#8482;s KHC association domain is                  Brickley et al., 2005). <br />  <br />  <br /> 550   JCB &#226;&#8364;&#162; VOLUME 173 &#226;&#8364;&#162; NUMBER 4 &#226;&#8364;&#162; 2006 <br />  Figure 6. Mitochondria are present in axons of klc null, but not in milton null, photoreceptors. Eyes were engineered to derive solely from clones homozy- <br /> gous for either klc8ex94 or milt92. Third instar larvae were dissected and photoreceptor axons were labeled with anti-chaoptin (pink) and mitoGFP expressed <br /> selectively in the photoreceptors by ey-GAL4. The bundled photoreceptor axons defasciculated upon innervation of the optic lobe in both controls (A and C) <br /> and mutants (B and D). (B) Although klc axons appear disordered, they still contain numerous mitochondria. (D) In contrast, milt axons project into the optic <br /> lobe more normally, but almost all of the axons lack mitochondria. A few axons do contain mitochondria, but they are likely to be heterozygous axons that <br /> persist at this developmental stage. Bars, 20 &#206;&#188;m. <br />  <br />  <br />       Therefore, we examined the distribution of endogenous                      Mutations in D. melanogaster miro were recently isolated <br /> miltons in mammalian cells by means of the P1&#226;&#8364;&#8220;152 antiserum,                     (Babcock et al., 2003; Guo et al., 2005) and found to lack axonal <br /> which could recognize milton 1 and 2 because of the high con-                    mitochondria (Guo et al., 2005). Thus, miro is likely to be another <br /> servation of the epitope (milton-A 1&#226;&#8364;&#8220;152). In rat cerebellar                     essential component of the machinery for mitochondrial trans- <br /> granule neurons, endogenous milton colocalized with the                          port, and, therefore, we examined its relationship with milton. <br /> mitochondrial marker cytochrome c oxidase (Fig. S2 E).                                 We have confirmed the interaction of miro and milton that <br /> In COS7 cells, endogenous milton was also observed on mito-                      was predicted by the two-hybrid screen by coimmunoprecipi- <br /> chondria (Fig. S2 F). Thus, D. melanogaster milton and its                       tation. HEK293T cells were transfected with milton-A or -D <br /> mammalian homologues are likely to play equivalent roles in                      and either D. melanogaster miro that was tagged with the T7 <br /> mitochondrial transport.                                                         epitope or a control T7-tagged protein. Both milton isoforms <br />                                                                                  were found in anti-T7 immunoprecipitates only when T7-miro <br /> Miro binds to milton and can affect                                              was coexpressed (Fig. 9 B). When expressed in COS7 cells, <br /> its association with mitochondria                                                D. melanogaster miro invariably localized to mitochondria and <br /> Despite its mitochondrial localization, milton has neither a                     induced a redistribution of mitochondria into aggregates <br /> mitochondrial import sequence nor a transmembrane domain.                        (Fig. 9 D). This aggregation was more severe when milton was <br /> To examine how milton associates with mitochondria, we                           coexpressed, but was not accompanied by a change in microtu- <br /> expressed partially deleted forms of milton. Milton&#226;&#8364;&#8482;s COOH ter-                  bule structure (Fig. S1). <br /> minus (expressed as either amino acids 847&#226;&#8364;&#8220;1,116 or 750&#226;&#8364;&#8220;1,116)                         Because miro has a transmembrane domain, we hypothe- <br /> had a mitochondrial distribution in COS7 cells (Fig. 8, A and B),                sized that miro might be important for the mitochondrial local- <br /> although some remained cytoplasmic in highly expressing cells.                   ization of milton and that miro lacking the transmembrane <br /> In contrast, milton 1&#226;&#8364;&#8220;450 was primarily nuclear at low expres-                   domain (miro&#226;&#710;&#8224;TM; amino acids 1&#226;&#8364;&#8220;574) might have dominant- <br /> sion levels (Fig. 8 C) or filled the cytoplasm in higher expressing              negative effects. Unlike full-length miro, miro&#226;&#710;&#8224;TM, when ex- <br /> cells (not depicted). Milton 1&#226;&#8364;&#8220;750 was also cytoplasmic,                         pressed in COS7 cells, was diffusely distributed throughout the <br /> although it was occasionally enriched near the nucleus (Fig. 8 D                 cytoplasm and was also nuclear when very highly expressed. <br /> and Fig. 9 H). Thus, the COOH terminus of milton must contain                    Moreover, miro&#226;&#710;&#8224;TM did not alter mitochondrial distribution <br /> a domain that is sufficient to be targeted to mitochondria.                      (Fig. 9 E). However, miro&#226;&#710;&#8224;TM could still bind to milton, as <br />        A mitochondrial protein that might interact with milton                   indicated by their coprecipitation when cotransfected (Fig. 9 A). <br /> was identified in a catalog of yeast two-hybrid interactions of                  In contrast to the strictly mitochondrial localization of milton <br /> D. melanogaster proteins (Giot et al., 2003). This protein, miro,                when expressed with full-length miro, milton was displaced <br /> contains two GTPase domains, a pair of EF hands, and a COOH-                     from mitochondria in most cells by expression of miro&#226;&#710;&#8224;TM <br /> terminal transmembrane domain, and localizes to mitochondria                     (Fig. 9, F and G). Thus, miro appears to be important for the as- <br /> (Fransson et al., 2003). When mutated, both human miro (Frans-                   sociation of milton with mitochondria, perhaps by serving as a <br /> son et al., 2003) and the yeast orthologue, Gem1p (Frederick                     receptor for milton on the mitochondrial surface. Consistent <br /> et al., 2004), alter the subcellular localization of mitochondria                with this hypothesis, a truncated milton (1&#226;&#8364;&#8220;750), which did not <br /> in a manner reminiscent of milton overexpression (Fig. 2 B).                     associate with mitochondria when expressed alone (Fig. 9 H), <br />  <br />  <br />                                                            MITOCHONDRIAL MOTILITY VIA MILTON, MIRO, AND KHC &#226;&#8364;&#162; GLATER ET AL.                                     551 <br />                                                                                           was recruited to mitochondria when D. melanogaster miro was <br />                                                                                          overexpressed in COS7 cells (Fig. 9 I). Milton 1&#226;&#8364;&#8220;750 was also <br />                                                                                          able to coimmunoprecipitate with miro&#226;&#710;&#8224;TM (Fig. 9 C). Notably <br />                                                                                          this milton construct does not contain the mitochondrial <br />                                                                                          association domain we identified in the COOH terminus; it <br />                                                                                          is therefore likely that milton associates with mitochondria <br />                                                                                          through at least two regions: residues 847&#226;&#8364;&#8220;1,116 bind to an <br />                                                                                          unidentified protein and 1&#226;&#8364;&#8220;750 bind to miro (Fig. 10). <br />  <br />                                                                                          Discussion <br />                                                                                          We have examined the involvement of milton in kinesin- <br />                                                                                          mediated mitochondrial motility and, thus, in the essential pro- <br />                                                                                          cess of distributing mitochondria within the cell. From these <br />                                                                                          studies we have derived a model of a protein complex that <br />                                                                                          includes kinesin and adaptor proteins that link kinesin to the <br />                                                                                          mitochondrion (Fig. 10). These proteins are also likely to serve <br />                                                                                          as a focal point for regulating mitochondrial motility. <br />  <br />                                                                                          The mechanistic basis of the milton <br />                                                                                          phenotype <br />                                                                                          In vivo, milton is required for the axonal transport of mito- <br />                                                                                          chondria throughout the nervous system (Fig. 1; Stowers et al., <br />                                                                                          2002). Milton associates with kinesin-1 via a highly conserved <br />                                                                                          domain located between residues 138 and 450. This associa- <br />                                                                                          tion can recruit kinesin to mitochondria in COS7 cells and <br />                                                                                          appears to activate plus end&#226;&#8364;&#8220;directed transport of mitochon- <br />                                                                                          dria, as judged by their redistribution to aggregates in the <br />                                                                                          periphery of many cells transfected with both milton and KHC. <br />                                                                                          These findings provide a mechanistic explanation for the <br />                                                                                          absence of mitochondria from milton axons and terminals. <br />                                                                                          Consistent with this model, the motors that endogenously <br />                                                                                          associate with milton, KHC in D. melanogaster (Fig. 5 H; <br />                                                                                          Stowers and Schwarz, 2002), and KIF5 members in mammals <br />                                                                                          (Fig. 5, F and G; Brickley et al., 2005) have previously been <br />                                                                                          implicated in the axonal transport of mitochondria (Hurd and <br />                                                                                          Saxton, 1996; Tanaka et al., 1998). <br />       Figure 7. Alternative splicing of milton regulates its association with KHC. <br />       (A) Genomic structure of milton and its alternatively spliced transcripts. <br />       Milton has at least four predicted protein products (A&#226;&#8364;&#8220;D) that derive              The function of miro in mitochondrial <br />       from &#239;&#172;&#65533;ve transcripts. The transcripts share coding exons 9&#226;&#8364;&#8220;11                      motility <br />       (corresponding to amino acids 129&#226;&#8364;&#8220;1,116 of the A isoform described by <br />       Stowers et al. [2002]), but differ in their 5&#226;&#8364;&#178; ends, which are predicted to        The association of milton with mitochondria appears to be <br />       encode divergent NH2 termini. Milton-D derives from two transcripts that           mediated, in part, by its interactions with miro, and this proba- <br />       differ only in their 5&#226;&#8364;&#178; untranslated region, and its translational start site is   bly accounts for the failure of mitochondrial transport in the <br />       predicted to fall within exon 9 at Met138 of milton-A. The coiled coil <br />       domain (marked CC above milton-A) is present in each variant. Trans-               axons of miro mutants (Guo et al., 2005). This proposal is sup- <br />       lated regions are black and untranslated regions are gray. (B) Immuno-             ported (a) by the ability of a truncated cytosolic form of miro to <br />       precipitates by anti-milton mAb 5A124 of HEK293T cells transfected with            act as a dominant negative and displace milton from mitochon- <br />       myc-KHC and milton-A, -B, -C, or -D as indicated. Precipitates were <br />       probed with 5A124 and anti-myc (KHC). KHC coprecipitated with milton-A,            dria, and (b) by the ability of overexpressed full-length miro to <br />       -B, and -D, but not -C. (C&#226;&#8364;&#8220;E) COS7 cells were transfected with milton vari-        recruit to mitochondria a truncated milton (residues 1&#226;&#8364;&#8220;750) that <br />       ants, as indicated, and anti-milton mAb 5A124 immunoreactivity (green)             could not independently localize there. However, additional in- <br />       colocalized in each case with mitochondria labeled with MitoTracker or- <br />       ange (pink). Low expressing cells are shown to demonstrate milton&#226;&#8364;&#8482;s colo-          teractions for tethering milton to mitochondria are likely, as a <br />       calization with mitochondria, but when milton-B, -C, and -D are highly             COOH-terminal portion of milton (residues 847&#226;&#8364;&#8220;1,116) also <br />       expressed, they induce aggregation of mitochondria near the nucleus,               localizes to the organelle. The difficulty of purifying mitochon- <br />       just like milton-A. (F&#226;&#8364;&#8220;H) COS7 cells were cotransfected with milton vari- <br />       ants and myc-KHC, as indicated, and immunostained with 5A124 (green)               dria from limited numbers of homozygous miro larvae prevents <br />       and anti-myc (white). Milton-B and milton-D, but not milton-C, recruited           a direct determination of the amount of milton on mitochondria <br />       KHC to the mitochondria and caused aggregation of mitochondria.                    that lack miro. <br />       (G) The aggregation of mitochondria near the nucleus re&#239;&#172;&#8218;ects the high <br />       levels of milton-C expression in these cells, rather than a consistent conse-             The role of miro in kinesin-mediated transport does not <br />       quence of KHC coexpression with milton-C.                                          preclude additional roles for miro. Indeed, such functions are <br />  <br />  <br /> 552   JCB &#226;&#8364;&#162; VOLUME 173 &#226;&#8364;&#162; NUMBER 4 &#226;&#8364;&#162; 2006 <br />                                                                                               Figure 8. Truncated milton can localize to <br />                                                                                              mitochondria. COS7 cells were transfected with <br />                                                                                              either COOH-terminal regions of milton-A <br />                                                                                              (amino acids 750&#226;&#8364;&#8220;1,116 and 847&#226;&#8364;&#8220;1,116, <br />                                                                                              and also carrying a &#239;&#172;&#8218;ag epitope tag) or NH2- <br />                                                                                              terminal regions (amino acids 1&#226;&#8364;&#8220;450 and <br />                                                                                              1&#226;&#8364;&#8220;750). Cells were labeled with MitoTracker <br />                                                                                              orange and anti-milton mAb 5A124 (A and B) <br />                                                                                              or 2A108 (C and D). Constructs containing the <br />                                                                                              region 847&#226;&#8364;&#8220;1,116 were localized to mito- <br />                                                                                              chondria, but those that lacked this region <br />                                                                                              were not. Bar, 10 &#206;&#188;m. <br />  <br />  <br />  <br />  <br /> likely because a miro homologue, GEM1p, is found in yeast,          The regulation of mitochondrial movement <br /> where mitochondrial motility is chiefly actin-based (Yaffe, 1999;   Mitochondria are not static. In dividing cells they go through <br /> Boldogh et al., 2005), and GEM1 mutants have abnormal mito-         orchestrated movements to distribute themselves between the <br /> chondrial distributions (Frederick et al., 2004). In addition, it   daughter cells (Yaffe, 1999). Within axons they typically alter- <br /> will be of interest to determine the relationship of milton and     nate between stationary and moving states and can reverse their <br /> miro to syntabulin, which is another protein that has recently      direction (Hollenbeck, 1996; Ligon and Steward, 2000a). They <br /> been proposed to link kinesin to mitochondria (Cai et al., 2005).   arrest in the presence of elevated Ca2+, including Ca2+ that is <br />                                                                     derived from the activation of synaptic receptors (Li et al., <br /> KLC is not needed for KHC to transport                              2004), and respond to the activation of neurotrophin receptors <br /> mitochondria                                                        and various intracellular signals (Rintoul et al., 2003; Chada and <br /> Unexpectedly, we found that axonal transport of mitochondria        Hollenbeck, 2004; Miller and Sheetz, 2004; Malaiyandi et al., <br /> did not require the light chains of the kinesin-1 motors and that   2005). It is noteworthy that, in addition to linking kinesin to the <br /> light chains were, indeed, absent from the milton&#226;&#8364;&#8220;kinesin           mitochondria, the milton&#226;&#8364;&#8220;miro complex provides several possi- <br /> complex. When expressed in COS7 and HEK293T cells, the associ-      ble mechanisms for the regulation of transport. These include <br /> ation between milton and KHC was inhibited by KLC. In fly           the alternative splicing of milton, the posttranslational modifica- <br /> homogenates, KLC was not detected in immunoprecipitates of          tion of milton, and the modulation of the state of miro. <br /> the milton&#226;&#8364;&#8220;KHC complex. Mitochondria were abundant in the                  We have shown that the choice of NH2 terminus splicing <br /> axons of klc&#226;&#710;&#8217;/&#226;&#710;&#8217; photoreceptors. Thus, this mitochondrial motor      variant can influence KHC&#226;&#8364;&#8482;s association with the adjacent <br /> provides an exception to the conventional tetrameric structure      region of milton. In particular, KHC did not associate with <br /> of kinesin-1. Precedent for KHC-based transport that is KLC         milton-C, although it contains the KHC-association domain <br /> independent has been reported in Neurospora crassa (Steinberg       that is common to all the isoforms. The NH 2 terminus of <br /> and Schliwa, 1995), sea urchins (Skoufias et al., 1994), neuro-     milton-C presumably inhibits the interaction with KHC and <br /> nal dendrites (Setou et al., 2002), and the transport of RNA        might, thereby, reserve a pool of mitochondria for retention in <br /> particles (Palacios and St. Johnston, 2002; Kanai et al., 2004;     the cell body. Alternatively, the inhibition may not be constitu- <br /> Ling et al., 2004).                                                 tive in vivo, but, instead, might undergo regulation by additional <br />       The interaction of milton with KHC was not only KLC           factors and thereby control the recruitment of kinesin. In this <br /> independent, but was inhibited by KLC overexpression in trans-      context, it may be noteworthy that multiple bands of milton are <br /> fected COS7 and HEK293T cells. Therefore, a pool of KHC             detected on immunoblots from fly heads. Most of the milton <br /> without KLC is required for milton to associate with KHC            isoforms in these homogenates are in an association with KHC, <br /> in vivo. Previous studies have found evidence for such a pool       as determined by immunodepletion with anti-KHC. However, <br /> in bovine brain (Hackney et al., 1991). In light of our findings    there is one major band, representing nearly half of the endoge- <br /> and those cited in the preceding paragraph, it may be appropri-     nous milton, which does not appear to be associated with KHC <br /> ate to consider the light chains as one of several cargo adaptors   (Stowers et al., 2002). Thus, additional motors may associate <br /> for kinesin-1, of which milton is another.                          with milton, and particularly with milton-C. Milton may also be <br />  <br />  <br />                                                   MITOCHONDRIAL MOTILITY VIA MILTON, MIRO, AND KHC &#226;&#8364;&#162; GLATER ET AL.                            553 <br />        Figure 9. Milton associates with miro, and a dominant- <br />       negative miro can displace milton from mitochondria. <br />       (A&#226;&#8364;&#8220;C) Coprecipitation of milton and T7-tagged miro con- <br />       structs from transfected HEK293T cells with antibodies <br />       to T7. (A) Milton copuri&#239;&#172;&#65533;ed with full-length miro, as well as <br />       miro lacking the COOH-terminal transmembrane domain <br />       (Miro&#226;&#710;&#8224;TM). (B) Both milton-A and -D coprecipitated <br />       with miro. (C) Milton 1&#226;&#8364;&#8220;750 coimmunoprecipitates with <br />       miro&#226;&#710;&#8224;TM. (D and E) Full-length miro localizes to mitochon- <br />       dria when transfected into COS7 cells (D), but miro&#226;&#710;&#8224;TM <br />       does not (E). (F&#226;&#8364;&#8220;I) Cotransfection of full-length and trun- <br />       cated milton and miro constructs. Milton and miro colocal- <br />       ize with mitochondria when simultaneously transfected <br />       (F), but milton is displaced from mitochondria when co- <br />       transfected with miro&#226;&#710;&#8224;TM (G). Milton 1&#226;&#8364;&#8220;750 is not mito- <br />       chondrially associated when expressed alone (H), <br />       however, coexpression with miro induces mitochondrial <br />       localization (I). Bar, 10 &#206;&#188;m. <br />  <br />  <br />  <br />  <br />       involved in such processes as mitochondrial fission and elonga-      testes library; therefore, milton-D may correspond to the male- <br />       tion, and such a role might explain the clustering of mitochon-      specific milton transcripts on Northern blots (Stowers et al., <br />       dria when milton and miro are overexpressed.                         2002) and be necessary for the elongation of mitochondria <br />             The alternative splicing of milton may also represent an       along the axoneme of sperm (Siegenthaler et al., 2003). <br />       adaptation of the complex to the needs of particular cell types.           Posttranslational modifications are also likely to regulate <br />       Antiserum P1&#226;&#8364;&#8220;152, which binds only to milton-A, labels a sub-        mitochondrial motility. In particular, the COOH-terminal por- <br />       set of the structures in the D. melanogaster brain that are recog-   tions of the mammalian miltons bind to, and are substrates for, <br />       nized by antibodies to the common regions (Stowers et al.,           the cytosolic glycosylating enzyme O-GlcNAc transferase <br />       2002). Thus, there is tissue specificity in the choice of splicing   (OGT; Iyer et al., 2003). We have identified D. melanogaster <br />       variant. To date, ESTs for milton-D have only been found in a        OGT by mass spectroscopy in immunoprecipitates of milton <br />  <br />  <br /> 554   JCB &#226;&#8364;&#162; VOLUME 173 &#226;&#8364;&#162; NUMBER 4 &#226;&#8364;&#162; 2006 <br />                                                                                  expressed in photoreceptors (Fig. 6). To visualize mitochondria in culture, <br />                                                                                 cells were incubated with 100&#226;&#8364;&#8220;300 nM MitoTracker orange (Invitrogen) <br />                                                                                 for 15 min. <br />  <br />                                                                                 Constructs <br />                                                                                 Milton-A deletion constructs were constructed as follows: milton-A 1&#226;&#8364;&#8220;450 <br />                                                                                 and milton-A 1&#226;&#8364;&#8220;750 were made from full-length milton-A in pCMV Tag1 <br />                                                                                 (Stowers et al., 2002) that was partially digested by Sal1; milton-A <br />                                                                                 608&#226;&#8364;&#8220;1,116, 750&#226;&#8364;&#8220;1,116, 608&#226;&#8364;&#8220;942, and 847&#226;&#8364;&#8220;1,116 were made by PCR <br />                                                                                 with 5&#226;&#8364;&#178; primers containing a BamHI site and 3&#226;&#8364;&#178; primers with HindIII sites <br />                                                                                 and were cloned into pCMV Tag1 (Stratagene) with in-frame NH2 terminus <br />                                                                                 Flag tags. <br />                                                                                        Milton-B (LD33316), -C (LD28289), and -D (AT08952 and AT28977, <br />                                                                                 which differ only in the 5&#226;&#8364;&#178; untranslated region) were obtained from the <br />                                                                                 Berkeley Drosophila Genome Project (University of California, Berkeley, <br />                                                                                 CA), and their PCR-ampli&#239;&#172;&#65533;ed NH2 termini were substituted for that of <br />                                                                                 milton-A in pCMV Tag1 milton-A. The NH2 termini of these clones were ampli- <br />                                                                                 &#239;&#172;&#65533;ed by PCR with 5&#226;&#8364;&#178; primers containing a BamHI site and a 3&#226;&#8364;&#178; primer with <br />                                                                                 an XhoI site and cloned into BamHI&#226;&#8364;&#8220;XhoI&#226;&#8364;&#8220;digested pCMV Tag1 milton-A. <br />                                                                                 pCMV Tag1 milton-C was made similarly, but using NotI at the 5&#226;&#8364;&#178; instead <br /> Figure 10. Schematic representation of the protein complex that mediates        of BamHI. Human xpress-OIP106 (milton 1) and rat xpress-GRIF-1 (milton 2) <br /> anterograde mitochondrial movement. Miro is anchored to the outer mem-          were provided by G. Hart (Johns Hopkins University, Baltimore, MD; <br /> brane by its COOH-terminal transmembrane domain. The association of             Iyer et al., 2003). Rat myc-KHC, myc-KHC1-682, myc-KHC1-810, myc- <br /> milton with the mitochondrion is caused, at least in part, by the interaction   KHC1-891, and HA-KLC constructs were provided by K. Verhey (University <br /> of milton and miro, although an additional association via the COOH ter-        of Michigan, Ann Arbor, MI; Verhey et al., 1998). An EST, RE01164, cor- <br /> minus of milton is also likely. Milton is associated with OGT, which is a       responding to D. melanogaster miro (CG5410-PE), was obtained from the <br /> likely regulatory enzyme, and is responsible for recruiting KHC to the mito-    Berkeley Drosophila Genome Project and cloned between the BamHI and <br /> chondrial surface. The indicated GTPase domains and EF hands of miro            NotI sites of pA1T7 (Shamah et al., 2001), thereby placing a T7 epitope <br /> are also likely to regulate mitochondrial movement, as is the alternatively     tag at the NH2 terminus. Miro&#226;&#710;&#8224;TM was generated by digesting the miro <br /> spliced NH2 terminus of milton.                                                 construct with EcoRI and NotI, and then ligating an oligo-encoding a stop <br />                                                                                 codon between these sites. <br />  <br />                                                                                 Immunostaining <br /> from fly homogenates (unpublished data). In addition, we have <br />                                                                                 COS7 and HEK293T cells were cultured in DME supplemented with 10% <br /> determined that GlcNAc-modified milton is associated with                       FCS, L-glutamine, and penicillin/streptomycin. Rat cerebellar neurons were <br /> kinesin in vivo in D. melanogaster (unpublished data), although                 cultured as previously described (Sivasankaran et al., 2004). Cells were <br />                                                                                 transfected with calcium phosphate and immunostained 24&#226;&#8364;&#8220;36 h later. <br /> the physiological consequences of this conserved modification <br />                                                                                 In all cotransfection experiments, 1:1 ratios of DNA were used, except for <br /> are not known.                                                                  miro&#226;&#710;&#8224;TM, which was transfected in a 250-fold excess. Immunocytochemis- <br />       Miro, however, may be of the greatest potential interest as               try was performed as previously described (Stowers et al., 2002) and used <br />                                                                                 either anti-milton mAbs 2A108, 4A75, or 5A124, or anti-milton antiserum <br /> a regulator of mitochondrial motility because it contains two <br />                                                                                 P1&#226;&#8364;&#8220;152. Other primary antibodies used in this study include: chick anti- <br /> predicted calcium-binding EF hands, which are flanked by two                    myc (Invitrogen), 9E10 (Santa Cruz Biotechnology, Inc.), chick anti-HA <br /> GTPase domains (Fransson et al., 2003; Frederick et al., 2004).                 (GTS, Inc.), anti-Xpress (Invitrogen), goat anti-T7 (Bethyl Laboratories, Inc.), <br />                                                                                 mouse anti-T7 (Novagen), anti-kinesin (AKIN01; Cytoskeleton, Inc.), anti- <br /> Because Ca2+ stops mitochondrial movement, and thereby <br />                                                                                 cytochrome c oxidase (BD Biosciences), rabbit anti-KLC (a gift from J. Gind- <br /> concentrates mitochondria near areas of high energy demand,                     hart, University of Richmond, Richmond, VA; Gindhart et al., 1998), mouse <br /> such as active synapses, the EF hands of miro are likely to be                  anti-HSP60 (Stressgen Bioreagents), and rabbit anti-HA (Novus Biologi- <br />                                                                                 cals, Inc.). The following &#239;&#172;&#8218;uorescently tagged reagents were used: goat <br /> particularly important. <br />                                                                                 anti&#226;&#8364;&#8220;mouse Alexa Fluor 488, goat anti&#226;&#8364;&#8220;chick Alexa Fluor 647, donkey <br />       Mitochondrial motility is a feature of most, perhaps all,                 anti&#226;&#8364;&#8220;mouse Alexa Fluor 647, donkey anti&#226;&#8364;&#8220;goat Alexa Fluor 633, and don- <br /> eukaryotic cells. In neurons, much of this motility is microtu-                 key anti&#226;&#8364;&#8220;mouse Alexa Fluor 647 (all from Invitrogen), and goat anti&#226;&#8364;&#8220;mouse <br />                                                                                 Cy3, horseradish peroxidase Cy5, and donkey anti&#226;&#8364;&#8220;rabbit FITC (Jackson <br /> bule based, with kinesin as the plus end&#226;&#8364;&#8220;directed motor. This <br />                                                                                 ImmunoResearch Laboratories). <br /> motility, and its regulation by a variety of signals, permits the                       Cells were imaged at room temperature (25&#194;&#176;C) with a 63&#195;&#8212;, NA <br /> mitochondria to be distributed in accordance with local energy                  1.4, oil Plan-Apochromat objective lens on a laser scanning confocal <br />                                                                                 microscope (LSM 510 META/NLO; Carl Zeiss MicroImaging, Inc.) with <br /> use. Inadequate mitochondrial function in axons or dendrites <br />                                                                                 LSM software 3.2 (Carl Zeiss MicroImaging, Inc.). Images were assembled <br /> can result in decreased synapse formation (Li et al., 2004),                    into &#239;&#172;&#65533;gures with Photoshop 8.0 (Adobe) using only linear adjustments of <br /> a failure to maintain synaptic transmission (Verstreken et al.,                 contrast and color. <br />                                                                                         First or third instar larvae were dissected in PBS, &#239;&#172;&#65533;xed in 4% formal- <br /> 2005), or axonal degeneration (Ferreirinha et al., 2004). The <br />                                                                                 dehyde for 20&#226;&#8364;&#8220;30 min in PBS, washed three times in PBT (PBS, 0.3% Triton <br /> identification of milton and miro as key components of the                      X-100, and 0.5% bovine albumin serum), incubated in PBTS (PBT with 5% <br /> mechanism for mitochondrial transport by KHC should lead                        normal donkey serum) for 30&#226;&#8364;&#8220;60 min, and then incubated with 24B10 <br />                                                                                 anti-chaoptin (Developmental Studies Hybridoma Bank) at 1:200 overnight <br /> to a greater mechanistic understanding of the regulation of <br />                                                                                 in PBTS at 4&#194;&#176;C. The preparations were washed three times in PBT, then <br /> mitochondrial movement.                                                         incubated in goat anti&#226;&#8364;&#8220;mouse Cy3 (Jackson ImmunoResearch Laboratories, <br />                                                                                 Inc.) 1:400 in PBTS, washed three times in PBT and two times in PBS with <br />                                                                                 0.3% Triton X-100 and mounted in Vectashield (Invitrogen). Confocal <br /> Materials and methods                                                           images of photoreceptor axons were taken with a 40&#195;&#8212;, NA 1.3, objective <br />                                                                                 (Carl Zeiss MicroImaging, Inc.). <br /> Visualization of mitochondria <br /> Mitochondria were visualized in D. melanogaster using transgenic stocks         Coimmunoprecipitation <br /> containing mitoGFP (Pilling et al., 2006) and placed under the control of a     Cells were lysed in 5 mM EDTA, 300 mM NaCl, and 50 mM Tris-HCl, <br /> UAS promoter. Expression in selective tissues was driven by D42-Gal4,           pH 7.5, and a protease inhibitor cocktail set III (Calbiochem) was used at <br /> which is expressed in a subset of neurons (Fig. 1), or ey-Gal4, which is        1:1,000, 0.1 mg/ml PMSF (Sigma-Aldrich), and 1% Triton X-100 when <br />  <br />  <br />  <br />        precipitating milton with KHC, KLC, or 0.5% Triton X-100 for miro and                <br />       milton experiments. Lysates were precleared with irrelevant antibodies                       <br />       and protein A, incubated with anti-milton antibodies mAb 9E10 or anti-T7,                 <br />       and protein A&#226;&#8364;&#8220;Sepharose beads (GE Healthcare) for 2&#226;&#8364;&#8220;3 h at 4&#194;&#176;C. Immuno-               <br />       precipitates were separated by SDS-PAGE and transferred to nitrocellulose                  <br />       membranes. For immunodetection, anti-milton mAb 5A124 and anti-milton                    <br />       mAb 2A108 were used at 1:40; rabbit anti-KLC was used at 1:100; mAb                  <br />       9E10, rabbit anti-HA, and rabbit anti-KHC were used at 1:1,000; and                   <br />       donkey anti&#226;&#8364;&#8220;rabbit HRP and goat anti&#226;&#8364;&#8220;mouse HRP were used at 1:10,000                       <br />       (Jackson ImmunoResearch Laboratories, Inc.) <br />                                                                                            <br />       D. melanogaster stocks                                                                     <br />                                                                                                    <br />       The stocks used for these experiments were: <br />              y,w; FRT40A, milt92/CyO, y+ and yw; FRT40A and yw; FRT40A,                   <br />       GMR-hid, CL/CyO, y+;;Ey-Gal4, UAS FLP/TM6, y+ (Stowers et al., 2002).               <br />              y,w;; MYC-KLC Df (3L) 8ex94/TM3, Sb and Df (3L) 8ex94/TM3, <br />       Sb (klc null; provided by L. Goldstein, University of California, San Diego,       <br />       CA; Gindhart et al., 1998).                                                                <br />              y,w;; D42-Gal4UAS-MitoGFP/TM6, y+ and y,w; P(w+mc, UAS                   <br />       mitoGFP)/CyO, y+ and y,w;; P(w+mc, UAS mitoGFP)/TM6, y+ (A. Pilling                          <br />       and W. Saxton, Indiana University, Bloomington, Indiana).                                   <br />              y,w;; FRT79D (III) and y,w; Ey-Gal4, UAS Flp/CyO, y+; FRT79D, GMR-                  <br />       hid, CL/TM6, y+ (Stowers and Schwarz, 1999; Bloomington Stock Center).               <br />              The klc-null mutant, Df (3L) 8ex94, was recombined onto FRT79D                      <br />       chromosome. Recombinants were con&#239;&#172;&#65533;rmed by PCR analysis and by lethal-                     <br />       ity complementation analysis with klc alleles (klc8ex25, klc99 and klc8ex27 <br />       were provided by J. Gindhart). The following stocks were generated for              <br />       use in assaying mitochondria in klc&#226;&#710;&#8217;/&#226;&#710;&#8217; photoreceptors: w; p(UAS mitoGFP)/             <br />       CyO, Actin-GFP; FRT79D; and w; p(UAS mitoGFP)/CyO, actin-GFP; <br />       FRT79D, Df 3L (8ex94))/TM3, Ser, actin-GFP; and w: Ey-Gal4, UAS Flp/               <br />       CyO, actin-GFP; FRT79D, GMR-hid, CL/TM3, Ser, actin-GFP.                           <br />   <br /> </body></html>