<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>An essential component C-terminal domain phosphatase that <br /> interacts transcription factor IIF in <br /> Saccharomyces cerevisiae <br />  <br />  <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span>         One essential components phos-                                       MATERIALS AND METHODS <br /> phatase specifically dephosphorylates Saccharomyces <br /> cerevisiae RNA polymerase II (RPII) large subunit C-terminal                            Plasmid Construction Preparation Recombinant <br /> domain (CTD) novel polypeptide encoded essential                          yFcp1. Yeast growth manipulations performed es- <br /> gene termed FCP1. The Fcp1 protein localized                               sentially described (20). Yeast transformed the <br /> nucleus, binds largest subunit yeast general                       lithium acetate method (21). <br /> transcription factor IIF (Tfg1). In vitro, transcription factor                         Plasmid JA739 used express yeast complete <br />                                                                                      Fcp1 ORF control GAL10 promoter. A <br /> IIF stimulates phosphatase activity presence Fcp1 <br />                                                                                      BglII-cut PCR fragment encoding yFcp1 (amplified using the <br /> and second complementing fraction. Two distinct regions of <br />                                                                                      primers 59-CCCAGATCTATGACCACACAAATAAGGT- <br /> Fcp1 capable binding Tfg1, C-terminal Tfg1 <br />                                                                                      CTCCC-39 59-CCCAGATCTGATACGGCATCTGAG- <br /> binding domain dispensable activity vivo vitro. <br />                                                                                      CTGCTAATC-39; ATG Stop codons underlined) <br /> Sequence comparison reveals residues 173&#226;&#8364;&#8220;357 Fcp1 <br />                                                                                      inserted BamHI site plasmid pYGal. PYGal is <br /> correspond amino acid motif present proteins of <br />                                                                                      derivative pFL39 (TRP1 CEN6 ARS; 22) carries the <br /> unknown function predicted organisms. <br />                                                                                      GAL10 promoter (inserted EcoRI SmaI sites) <br />                                                                                      PGK transcription termination signal (between the <br /> Promoter-dependent transcription RNA polymerase II                                SphI HindIII sites) (obtained James Friesen, Uni- <br /> (RPII) requires general transcription factors (reviewed                       versity Toronto). Plasmid JA754 [expresses functional Fcp1 <br /> ref. 1). The form RPII binds preferentially promoters                     protein GAL10 promoter protein tagged at <br /> is extensively phosphorylated C-terminal heptapep-                        N terminus histidine residues followed the <br /> tide-repeat domain (CTD) largest subunit (2&#226;&#8364;&#8220;4). Instead,                      hemagglutinin (HA) epitope] contained double-stranded <br /> the CTD phosphorylated shortly                               oligonucleotide (59-ATGGCCCATCATCATCATCATCAT- <br /> initiation, elongating RPII generally phosphorylated                       GCCATGGCTTACCCATACGATGTTCCAGATTACGC- <br /> CTD (5, 6). CTD phosphorylation regulated process:                          TG-39 complement) inserted SmaI site of <br /> heat shock induction Drosophila hsp70 gene causes RPII,                       JA739 immediately upstream yFcp1 ORF. Plasmid <br /> paused downstream promoter unphosphory-                               JA756 encoding truncated Fcp1 protein lacking C-terminal <br /> lated CTD, transcribe downstream sequences phos-                           66 amino acids, constructed replacing SpeI-SalI <br /> phorylated CTD (7). Activator-induced CTD phosphorylation                            yFCP1 fragment JA754 SpeI-XhoI cut PCR product <br /> also facilitate RNA chain elongation (8&#226;&#8364;&#8220;11). Prior                         encoding amino acids 457&#226;&#8364;&#8220;666 Fcp1 synthetic TAG <br /> following transcriptional termination, CTD dephosphoryla-                            Stop codon downstream codon 666. Plasmid JA798 encod- <br /> tion presumably occur regenerate enzyme capable                           ing truncated Fcp1 protein lacking C-terminal 275 amino <br /> reinitiating transcription.                                                          acids, contained palindromic oligonucleotide 59- <br />    Transcription factor IIF (TFIIF) (RAP30y74) binds directly                        CTAGCCTAGGCCTAGG-39 encoding in-frame stop <br /> <span id='am-6' about='protege:TO' typeof='owl:Thing'>to</span> RPII helps assemble RPII preinitiation com-                          codon (underlined) inserted SpeI site plasmid <br /> plex promoter (12&#226;&#8364;&#8220;14). Elongating RPII interacts                      JA754. <br /> TFIIF, stimulates rate chain elongation (15, 16).                          Plasmid JA782 encoding amino acids 1&#226;&#8364;&#8220;732 Fcp1 fused to <br /> TFIIF stimulates activity CTD phosphatase                              HA epitope constructed isolating NcoI-HindIII <br /> purified human cells (17, 18) yeast (19). Two                               yFCP1 fragment plasmid JA754 inserting down- <br /> components required yeast CTD phosphatase activity                           stream T7 promoter, NcoI HindIII sites <br /> (19), described here, encoded essential                        plasmid pET23-d (Novagen). Plasmid pJA785 encoding <br /> gene, closely related human protein necessary                      amino acids 1&#226;&#8364;&#8220;666 Fcp1 downstream T7 promoter was <br /> human CTD phosphatase activity (J.A., H. Xiao, G. Pan, G.                            constructed inserting NcoI-HindIII fragment <span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> Dahmus, M.C., S. Zhang, R. G. Roeder, M. Dahmus, J.G.,                           pJA756 NcoI HindIII sites pET23d. <br /> unpublished work). That TFIIF modulate CTD phos-                             Plasmid pJA801 encoding amino acids 1&#226;&#8364;&#8220;457 Fcp1 con- <br /> phatase activity highlights importance phosphoryla- <br /> tion cycle RPII regulating initiation elongation                      Abbreviations: RPII, RNA polymerase II; CTD, C-terminal domain of <br /> reactions transcription.                                                          repeated heptapeptide sequence largest subunit RNA <br />                                                                                      polymerase II; TFIIF, transcription factor IIF; TR, thioredoxin; HA, <br /> The publication costs article defrayed page charge           hemagglutinin; GST, glutathione S-transferase. <br />                                                                                      Data deposition: The sequence reported paper been <br /> payment. This article marked &#226;&#8364;&#732;&#226;&#8364;&#732;advertisement&#226;&#8364;&#8482;&#226;&#8364;&#8482; in <br />                                                                                      deposited GenBank database (accession no. Z49704). <br /> accordance 18 U.S.C. &#194;&#167;1734 solely indicate fact.                        &#226;&#8364;&#160;J.A. R.S.C. contributed equally work. <br />                                                                                      i <br /> &#194;&#169; 1997 The National Academy Sciences 0027-8424y97y9414300-6$2.00y0               To reprint requests addressed. e-mail: kanecm@ <br /> PNAS available online http:yywww.pnas.org.                                       uclink4.berkeley.edu. <br />  <br />                                                                              14300 <br />           Biochemistry: Archambault et al.                                        Proc. Natl. Acad. Sci. USA 94 (1997)      14301 <br />  <br /> tained palindromic oligonucleotide 59-CTAGCCTAGG-             AAAGTGATGACGAAAACAAC-39 59-GGGGAATT- <br /> CCTAGG-39 encoding in-frame stop codon (underlined)            CTAATCATCCAGCATATCCATCAA-39 amino acids <br /> inserted SpeI site pJA782. A derivative JA782      667&#226;&#8364;&#8220;732 [TR-yFCP1 (667&#226;&#8364;&#8220;732)]; 59-GGGGGATCCGTT- <br /> (T7-HA-yFcp1) expressing polyhistidine- HA-tagged           GATGACGATGATGAACTATCA-39 59-GGGGAAT- <br /> Fcp1 protein (21 additional amino acids MHHHHHHMAY-               TCTAGTCGTGGTCGTCATCTTCGTC-39 amino acids <br /> PYDVPDYAGGS) constructed inserting following           458&#226;&#8364;&#8220;666 [TR-yFCP1 (458&#226;&#8364;&#8220;666)]. Stop codons underlined. <br /> double-stranded oligonucleotide NcoI site JA782       The human FCP1-expressing plasmid [hFCP1a(443&#226;&#8364;&#8220;842)] en- <br /> (59-CATGCACCACCACCACCACCA-39 59-CATGTG-                       codes 400 amino acid C-terminal fragment inserted the <br /> GTGGTGGTGGTGGTG-39; initiator ATG codon un-                BamHI site pRSETC (Invitrogen), isolation and <br /> derlined).                                                        construction described elsewhere. Proteins pro- <br />    Plasmids pDB3 pDB4, expressing amino acids 1&#226;&#8364;&#8220;732           duced vitro described below. Plasmid pJA728 encoding <br /> and 1&#226;&#8364;&#8220;626, respectively, endogenous FCP1 promoter,       C-terminal 87 amino acids yeast RAP74 (TFG1) fused <br /> were constructed cloning fragments FCP1 pDB2.          glutathione S-transferase (GST) constructed insert- <br /> Plasmid pDB2 contained FCP1 promoter region (2303          ing BglII-cut TFG1 PCR fragment (amplified yeast <br /> 210 relative nucleotide start codon    genomic DNA using primers: 59-CCCAGATCTCGGCATC- <br /> ORF) region downstream stop codon           GAATACAGTGCC TTCGCC-39 59-CCCAGATCTC- <br /> ORF (14 1295 relative nucleotide stop      TACTCTTTCTTTAATTCCATGTGGTC-39; codon encod- <br /> codon) inserted EcoRI SpeI sites, respectively,   ing Ala-649 stop codon underlined) BamHI <br /> the low copy plasmid pRS313 (23). The promoter region         site pGEX-3X (Pharmacia). Plasmid pJA8 encoding the <br /> amplified using PCR primers: 59-TCGGATCCAACGT-               C-terminal 82 amino acids human RAP74 fused GST will <br /> CTAGATGCCGCAACC-39 59-CGCAGATCTGCTTG-                         described elsewhere. <br /> TGTTTACACGTTCGC-39. The downstream region am-                    Protein Affinity Chromatography. Columns (40 ml) con- <br /> plified using primers: 59-TCGGATCCCTCAGATGCCGTA-                  taining GST, GST-yRAP74 (649 &#226;&#8364;&#8220;735), GST-hRAP74 <br /> TCTTTCC-39 59-CGCAGATCTAGCAGCTAATGGC-                         (436&#226;&#8364;&#8220;517) immobilized glutathione-Sepharose concen- <br /> TCGAGAG-39. Amplification carried Amer-          tration 3&#226;&#8364;&#8220;4 mgyml prepared (25) using immobilized <br /> ican Type Culture Collection Cosmid 8021. The NcoIyXhoI           proteins E. coli transformed pGEX-3X, pJA728, or <br /> fragment pJA782 containing full-<span id='am-5' about='xsp:length' typeof='owl:Thing'>length</span> ORF FCP1         pJA8. Columns equilibrated 300 ml ACB buffer <br /> was cloned SmaI site pDB2 make pDB3. The           (10 mM Hepes, pH 7.9y1 mM EDTAy1 mM DTTy10% <br /> NcoIyNdeI fragment pJA782 containing residues 1&#226;&#8364;&#8220;626         glycerol) containing 0.1 M NaC1 5 mgyml BSA, next <br /> the FCP1 ORF cloned SmaI site pDB2 make        400 ml ACB buffer containing 0.1 M NaC1 1 mgyml <br /> pDB4.                                                             BSA. Columns loaded 20 ml rabbit reticulocyte <br />    Plasmids pJA794 pJA796 expressing, respectively, <br />                                                                   lysate TNT transcription translation (Pro- <br /> yFcp1(1&#226;&#8364;&#8220;732) yFcp1(1&#226;&#8364;&#8220;666) endogenous yFCP1 <br />                                                                   mega) programmed 0.4 mg various plasmid DNAs <br /> promoter contained 738-bp, NcoI-cut PCR fragment encod- <br />                                                                   [T7-HA-yFcp1, pJA785, pJA801, TR-yFCP1(458&#226;&#8364;&#8220;732), TR- <br /> ing yFCP1 promoter upstream region (amplified from <br />                                                                   yFCP1(667&#226;&#8364;&#8220;732), TR-yFCP1(458 &#226;&#8364;&#8220; 666), hFCP1a(443&#226;&#8364;&#8220; <br /> yeast genomic DNA using primers: 59-GGGGCCATGGTC- <br />                                                                   842)], reticulocyte lysate diluted 10 times <br /> CCATGTTTAGACAGATGCTCACC-39 59-GGGGCC- <br />                                                                   ACB buffer containing 0.1 M NaC1 1 mgyml BSA. <br /> ATGGTCTGCTTTGCTTGTGTTTACACGTTCG-39) in- <br />                                                                   After loading, columns washed 400 ml ACB buffer <br /> serted NcoI site plasmids pJA754 pJA756. <br /> Plasmid pJA804 expressing yFcp1(1&#226;&#8364;&#8220;457) contained pal-         containing 0.1 M NaC1. Bound proteins eluted sequen- <br /> indromic oligonucleotide 59-CTAGCCTAGGCCTAGG-39                   tially 120 ml ACB containing 1 M NaC1 120 ml of <br /> encoding in-frame stop codon (underlined) inserted        ACB buffer containing 1% SDS. Aliquots input proteins <br /> the SpeI site plasmid pJA794.                                  NaC1 SDS eluates analyzed SDSy <br />    Plasmids used express recombinant Fcp1 deletion     PAGE autoradiography. <br /> derivatives transformed Escherichia coli BL21              Disruption Yeast FCP1 Gene. Plasmid JA744 used to <br /> (DE3), cells induced according Novagen&#226;&#8364;&#8482;s          disrupt endogenous FCP1 gene constructed two <br /> instructions. Cells harvested, washed cold distilled      steps. In step, BglII cut PCR fragment corresponding <br /> water, resuspended 50 mM Tris-acetate, pH 7.8y20% glyc-        genomic region located immediately upstream the <br /> eroly0.5 M potassium acetatey5 mM imidazoley0.1% Triton           FCP1 ORF (amplified yeast genomic DNA using the <br /> X-100y5 mM 2-mercaptoethanoly1 mM phenylmethylsulfonyl            following oligonucleotides: 59-CCCAGATCTGTCCCAT- <br /> fluoride, cells lysed sonication. The insoluble   GTTTAGACAGATGCTCACC-39 59-CC CAGATCTA- <br /> recombinant yFcp1 protein recovered centrifugation      ATCTGCTTTGCTTGTGTTTACACGTTC-39; portion <br /> the extract 12,000 3 g 20 min pellet           oligonucleotide hybridizes genomic DNA is <br /> resolubilized buffer 6 M urea.          underlined) inserted upstream LEU2 gene the <br /> Insoluble material removed centrifugation          BamHI site plasmid JJ250 (26). The resulting plasmid was <br /> supernatant loaded Ni-nitrilotriacetate (NTA)-         modified inserting HindIII cut PCR fragment cor- <br /> agarose column (Qiagen, Chatsworth, CA). The column           responding sequences located downstream FCP1 <br /> washed buffer containing 6 M urea 20 mM         ORF (amplified yeast genomic DNA using following <br /> imidazole, recombinant Fcp1 eluted        oligonucleotide primers: 59-CCCCAAGCTTCAGCTCA- <br /> buffer 100 mM imidazole. Recombinant Fcp1 rena-          GATGCCGTATCTTTCCACA-39 59-CCCCAAGCTT- <br /> tured described (24).                                          GAGCTCCT TAGA ATGGA ATCTATGATCGTGT-39) <br />    Plasmids encoding amino acids 458&#226;&#8364;&#8220;732, 667&#226;&#8364;&#8220;732,            downstream LEU2 gene HindIII site. The <br /> 458&#226;&#8364;&#8220;666 Fcp1 fused E. coli thioredoxin (TR)            resulting plasmid, pJA744, carries modified version the <br /> constructed inserting BamHI- EcoRI-cut PCR frag-           FCP1 genomic locus FCP1 ORF replaced with <br /> ments BamHI EcoRI sites pET32-a (Novagen).        LEU2 gene. This disrupted allele FCP1 released <br /> In case, plasmid JA782 used template          plasmid backbone digestion SmaI SacI <br /> amplifying primers follows: 59-GGGGGATCCGTT-              used disrupt FCP1 alleles diploid yeast <br /> GATGACGATGATGAACTATAC-39 59-GGGGAAT-                          strain W303 (MatayMata can1&#226;&#8364;&#8220;100ycan1&#226;&#8364;&#8220;100 his3&#226;&#8364;&#8220;11,15y <br /> TCTAATCATCCAGCATATCCATCAA-39 amino acids                      his3&#226;&#8364;&#8220;11,15 leu2&#226;&#8364;&#8220;3,112yleu2&#226;&#8364;&#8220;3,112 trp1&#226;&#8364;&#8220;1ytrp1&#226;&#8364;&#8220;1 ura3yura3 <br /> 458&#226;&#8364;&#8220;732 [TR-yFCP1 (458&#226;&#8364;&#8220;732)]; 59-GGGGG ATCCGACG-                  ade2&#226;&#8364;&#8220;1yade2&#226;&#8364;&#8220;1) using one-step gene disruption protocol <br />  14302     Biochemistry: Archambault et al.                                                Proc. Natl. Acad. Sci. USA 94 (1997) <br />  <br /> (27). Disruption confirmed PCR (data shown),                   DTGDD, amino acids 38&#226;&#8364;&#8220;52; KEFFA, amino acids 254&#226;&#8364;&#8220;258; <br /> creating strain JA830.                                                    KLXXXLATAXE, amino acids 405&#226;&#8364;&#8220;415) present a <br />    Indirect Immunof luorescence. Cells prepared im-              732 amino acid ORF chromosome XIII S. cerevisiae. <br /> munofluorescence (28, 29) minor modifications. Fixed                 Expression yeast gene E. coli N- <br /> and washed cells incubated 2 hr room temperature             terminally fused influenza virus HA epitope polyhisti- <br /> a 1:20 dilution preabsorbed (30) MMS101R monoclonal                    dine tag led, expected, production recombinant <br /> anti-HA antiserum (Babco, Richmond, CA; provided Kevin                 protein slightly mobile natural protein an <br /> Madden, Yale University, New Haven, CT). After washing,                   SDSypolyacrylamide gel (Fig. 1 A, lanes 2, 3), able to <br /> preabsorbed (30) secondary antibody (CY3-conjugated goat                  functionally replace second component required yeast <br /> anti-mouse antibody Jackson ImmunoResearch, 1:40                     CTD phosphatase activity (Fig. 1B, lanes 1, 2). <br /> dilution, provided Kevin Madden) applied. After                       This novel yeast protein similar amino acid sequence <br /> washing, cells stained 49,6-diamidino-2-phenylindole            length (32% identity 54% similarity) a <br /> and observed microscopy.                                               human protein cloned basis interaction with <br />                                                                           human TFIIF (J.A., H. Xiao, G. Pan, G. Dahmus, M.C., S. <br />                                                                           Zhang, R. G. Roeder, M. Dahmus, J.G., unpublished <br />                             RESULTS                                       work). The yeast protein associates TFIIF we <br />   Identification Essential Component Yeast CTD                 below. Thus, new yeast protein named <br /> Phosphatase. Purification CTD phosphatase extracts                Fcp1 TFIIF-associating component CTD phosphatase. <br /> of Saccharomyces cerevisiae led resolution com-             This phosphatase type 2C activity requires divalent <br /> ponent fractions essential phosphatase activity (19).             cations resistant okadaic acid vanadate (19). Yet, <br />                                                                           amino acid sequences yeast human Fcp1 (Fig. 2A) <br /> One fractions contained yeast TFIIF, be <br />                                                                           resemble known phosphatases outside the <br /> replaced highly purified TFIIF. When essential <br />                                                                           PP2C family. Fcp1 unrelated phosphatase, <br /> component purified homogeneity, polypeptide an <br />                                                                           Sit4, implicated transcription control (31). Therefore, either <br /> apparent molecular mass 100y103 kDa SDSyPAGE <br />                                                                           Fcp1 itself phosphatase activates CTD phospha- <br /> copurified activity (see Fig. 1A, lane 3). Excision           tase present fraction needed activity. Curi- <br /> this polypeptide gel, followed removal SDS             ously, Fcp1 protein contains BRCT motif many <br /> and renaturation, led recovery CTD phosphatase                  proteins involved DNA repair (32) (Fig. 2B). <br /> activity combination TFIIF-containing fraction                   Three regions yeast human Fcp1 exhibit highest <br /> (ref. 19; Fig. 1B, lane 3). As reported, phosphatase          degrees similarity: N-terminal region (36% identity) <br /> assay, enzyme specifically removes phosphate                 corresponds amino acids 124&#226;&#8364;&#8220;352 yeast Fcp1, middle <br /> CTD 32P-labeled IIo form largest subunit RPII,           region (32% identity) amino acids 425&#226;&#8364;&#8220;593 yeast Fcp1, <br /> converting dephosphorylated IIA form (19).                      C-terminal region (25% identity) amino acids <br />   The 100y103-kDa polypeptide excised SDS gel                610&#226;&#8364;&#8220;728 yeast Fcp1 (see Fig. 2B). The separation these <br /> digested protease lysC resulting peptides           regions long sequences missing yeast human <br /> resolved HPLC Applied Biosystems C8 Aquipore-300                 protein suggests functional domain. The <br /> column. Partial amino acid sequences obtained              C-terminal conserved region contains binding site <br /> well-resolved peptides. All sequences (KFWYLVERAS-                  TFIIF, central conserved region contains part <br />                                                                           second TFIIF-binding site (see below). The N-terminal <br />                                                                           conserved region known function, it <br />                                                                           contains short sequence motifs related sequences 15 other <br />                                                                           ORFs unknown function species (see Fig. 2C). <br />                                                                              Fcp1 localized cells S. cerevisiae expressing an <br />                                                                           HA-tagged FCP1 gene low copy number plasmid. <br />                                                                           Immunofluorescence anti-HA mAb revealed that <br />                                                                           HA-Fcp1 (Upper Left, Fig. 3) colocalizes DNA (49,6- <br />                                                                           diamidino-2-phenylindole staining Lower Left, Fig. 3) is, <br />                                                                           therefore, yeast nuclear protein. <br />                                                                              Yeast Fcp1 Interacts Tfg1, Yeast Homologue of <br />                                                                           RAP74. Yeast TFIIF stimulates activity yeast phosphatase <br />                                                                           (19), human TFIIF stimulates activity human phos- <br />                                                                           phatase (18). To test yeast Fcp1 human FCP1a were <br />                                                                           likely similar functions, binding yeast Fcp1 yeast <br />                                                                           TFIIF subunit Tfg1 (33, yeast RAP74) examined (Fig. 4A). <br />                                                                           35S-labeled proteins containing various portions yeast <br />  <br />                                                                           Fcp1 translation vitro applied microaf- <br />                                                                           finity columns containing C-terminal amino acids 649&#226;&#8364;&#8220;735 of <br />    FIG. 1. Reconstitution CTD phosphatase activity recom-         yeast RAP74 (33) fused GST. <br /> binant yeast Fcp1. (A) Yeast Fcp1 purified yeast cell             Indeed, yeast Fcp1 bound C-terminal region yeast <br /> extracts [yFCP1, Fraction 62, MonoQ Fraction (19)] recombinant        RAP74 eluted partly salt partly SDS <br /> Fcp1 (ryFCP1) produced E. coli purified Ni-NTA-garose           (Fig. 4A, line 1). Moreover, C-terminal amino acids <br /> chromatography separated SDSy7.5% polyacrylamide gel            667&#226;&#8364;&#8220;732 Fcp1, synthesized vitro fusion protein with <br /> and stained Coomassie blue. An arrow indicates position       TR, sufficient interaction (Fig. 4A, compare lines <br /> ryFcp1 contains 21 extra amino acids: histidines HA tag.   5 7). However, amino acids 1&#226;&#8364;&#8220;666 yeast Fcp1 bound <br /> (B) Assay yFcp1 ryFcp1 CTD phosphatase activity. The           RAP74, amino acids 1&#226;&#8364;&#8220;457 (Fig. 4A, lines 2, <br /> assay (19) measures removal 32P largest subunit RPII. <br /> Lane 1, autoradiographic signal phosphorylated form the <br />                                                                           3), implying second RAP74-binding site amino <br /> largest subunit yeast RPII. Lanes 2 3, signal remaining      acids 458&#226;&#8364;&#8220;666 yeast Fcp1. When region Fcp1 was <br /> treatment renatured recombinant (&apos;0.2 mg) purified         produced vitro fusion protein TR, also <br /> yeast (&apos;0.02 mg) Fcp1, respectively; amounts protein        sufficient bind RAP74 (Fig. 4A, line 6). Therefore, yeast <br /> saturating assay.                                                  Fcp1 RAP74-binding regions (Fig. 2B). Binding <br />            Biochemistry: Archambault et al.                                                     Proc. Natl. Acad. Sci. USA 94 (1997)            14303 <br />  <br />  <br />  <br />  <br />    FIG. 2. Sequence structure yeast Fcp1. (A) Comparison yeast human Fcp1. Amino acid sequences yeast Fcp1 (top line) and <br /> human FCP1a (J.A., H. Xiao, G. Pan, G. Dahmus, M.C., S. Zhang, R. G. Roeder, M. Dahmus, J.G., unpublished work) (bottom line) were <br /> aligned using sequence alignment program BESTFIT (Wisconsin sequence analysis package, GCG). Identical amino acids indicated a <br /> vertical line, highly similar similar amino acids colon dot, respectively. (B) Regions highest similarity approximate locations <br /> of functional motifs yeast human Fcp1. Shaded areas similar yeast human proteins. &#226;&#8364;&#732;&#226;&#8364;&#732;FCP1 homology&#226;&#8364;&#8482;&#226;&#8364;&#8482; refers <br /> to region similarity presented C. (C) Multiple alignment amino acid sequences homologous yeast Fcp1 residues 173&#226;&#8364;&#8220;362 identified <br /> by BLAST searches GenBank. Bold letters: residues identical sequences. Shaded letters: regions similarity. 1. S. cerevisiae FCP1, Z49704; <br /> 2. S. cerevisiae ORF, X90564; 3. S. cerevisiae ORF, U39205; 4. S. cerevisiae ORF, U10555; 5. S. cerevisiae ORF, Z73115; 6. Schizosaccharomyces <br /> pombe ORF, Z50142; 7. Brassica campestris EST, L46538; 8. Arabidopsis thaliana EST, T44887; 9. Brugia malayi EST, H48204; 10. Toxoplasma gondii <br /> EST, W35521; 11. Caenorhabditis elegans EST, U29536; 12. Mus musculus EST, W29399; 13. Homo sapiens EST, H24417; 14. Homo sapiens EST, <br /> H66914; 15. Homo sapiens EST, H95720; 16. Homo sapiens EST, H84869; 17. Homo sapiens FCP1. <br />  <br /> between central region RAP74 somewhat weaker                     <span id='am-7' about='xsp:pattern' typeof='owl:Thing'>pattern</span> viable nonviable spores characteristic an <br /> and salt-sensitive interactions involving                   essential gene (Fig. 5A). Nonviable spores gave rise micro- <br /> C-terminal region Fcp1 (Fig. 4 data shown).                        colonies 32&#226;&#8364;&#8220;64 cells, indicating excess <br />    The ability TFIIF stimulate CTD phosphatase                       Fcp1 protein wild-type cell. As expected, viable <br /> species specific (19). Yet, yeast Fcp1 bind C-terminal             spores leu22 (data shown). Viable LEU21 spores <br /> 83 amino acids human RAP74 (Fig. 4B, line 1)                      recovered disrupted diploid strain trans- <br /> interaction depends RAP74-binding region yeast                      formed plasmid expressing FCP1 control of <br /> Fcp1 (Fig. 4B, line 2). Human FCP1a bind                      GAL10 promoter, grown medium containing <br /> C-terminal 87 amino acids yeast RAP74 (Fig. 4A, line 8).                   galactose (Fig. 5B). Therefore, FCP1 essential gene, and <br /> Thus, Fcp1-RAP74 interaction likely involves amino acids                  dephosphorylation RPII CTD likely essential <br /> conserved yeast human proteins.                               function S. cerevisiae. However, FCP1 one <br />    The FCP1 Gene Is Essential Growth S. cerevisiae.                component CTD phosphatase, remains formally possible <br /> RPII dephosphorylated CTD initiates transcription                      essential nature regulating phosphorylation <br /> efficiently promoters vitro (2, 3). Thus, dephos-                  additional substrate vivo. <br /> phorylation CTD prior following termination                     Truncations Fcp1 That Lack RAP74 Binding Region <br /> transcription critical gene expression vivo. To               Are Still Functional. When FCP1 expressed own <br /> test FCP1 essential, gene disrupted                     promoter low copy number plasmid, viability rescued <br /> insertion LEU2 FCP1 loci. Disruption                   full-length FCP1 deletions [FCP1 (1&#226;&#8364;&#8220;666) and <br /> confirmed PCR (data shown). After meiosis                          FCP1 (1&#226;&#8364;&#8220;626)] lacking C-terminal RAP74-binding site <br /> sporulation, dissection asci revealed 2:2 segregation                  (Fig. 5C). However, deletion lacking RAP74-binding <br />  14304     Biochemistry: Archambault et al.                                                 Proc. Natl. Acad. Sci. USA 94 (1997) <br />  <br />  <br />  <br />  <br />    FIG. 3. Fcp1 localized nucleus. Cells asynchronous <br /> cultures strain JA830 (fcp1 DLEU2 pHA::FCP1), contains <br /> pJA794 producing HA-Fcp1 FCP1 promoter, W303 (no <br /> HA tag) fixed stained anti-HA mAb. Diffuse back- <br /> ground staining detected non-HA containing W303 (&#226;&#8364;&#732;&#226;&#8364;&#732;no tag&#226;&#8364;&#8482;&#226;&#8364;&#8482;); <br /> this frame overexposed intentionally detect background fluo- <br /> rescence. HA-Fcp1-specific staining colocalized nucleus         FIG. 5. Yeast FCP1 essential. (A) Diploid strain JA830 disrupted <br /> revealed 49,6-diamidino-2-phenylindole staining (HA::FCP1 pan-        copy FCP1 induced sporulate, spores each <br /> els). Both 49,6-diamidino-2-phenylindole anti-HA images     tetrad tested growth. (B) Diploid JA830 transformed with <br /> superimposed Nomarski images stained cells.                  pJA754 produce HA-tagged Fcp1 GAL10 promoter. Sporu- <br />                                                                          lation induced, spores tested growth medium <br /> sites [FCP1 (1&#226;&#8364;&#8220;457)] rescue viability ex-               containing galactose. (C) Deletion constructs Fcp1 tested for <br /> pressed high copy (Fig. 5C). Both FCP1 (1&#226;&#8364;&#8220;666) FCP1               complementation strain disrupted FCP1 vivo phos- <br />                                                                          phatase activity vitro. Amino acids construct locations of <br /> (1&#226;&#8364;&#8220;626) function vitro dephosphorylate CTD <br />                                                                          RAP74 interacting regions indicated. Low copy refers cells <br /> (summarized Fig. 5C). Because proteins vitro           heterozygous FCP1 disruption transformed plasmids con- <br /> assays recombinant Fcp1 recovered following denatur-              taining FCP1 control endogenous promoter (pJA794, <br /> ation renaturation, accurate, comparative activity deter-            pDB3, pJA796, pDB4, pJA804); cells sporulated growth of <br /> minations possible. However, cells harboring FCP1                haploids tested selective media. High copy refers cells het- <br /> (1&#226;&#8364;&#8220;626) instead wild-type FCP1, viable, cold                erozygous FCP1 disruption transformed plasmids containing <br /> sensitive (data shown). Thus, truncation likely func-           FCP1 control GAL10 promoter (pJA739, pJA756, and <br />                                                                          pJA798); cells sporulated growth haploids tested on <br /> tions vivo does wild-type Fcp1. The protein            medium containing galactose. (1) refers complementation viabil- <br /> encoded FCP1 (1&#226;&#8364;&#8220;457) insoluble               ity. The vitro reactions (19) carried Fcp1 proteins <br /> tested vitro activity.                                            overproduced E. coli purified described Experimental Proce- <br />                                                                          dures; (1) indicates phosphatase activity detected. nd, done. <br />                          DISCUSSION <br /> A yeast nuclear protein, Fcp1, identified               enzyme. Fcp1 known phosphatase motifs, but <br /> necessary CTD phosphatase activity. The importance                phosphatases diverse group proteins few <br /> this phosphatase vivo highlighted finding FCP1          primary sequence motifs common (34). Therefore, the <br /> an essential gene. It known Fcp1                 short, conserved amino acid sequences residues 173&#226;&#8364;&#8220; <br /> phosphatase enzyme essential regulatory component               357 yeast Fcp1 (see Fig. 2) signature new <br />                                                                          family protein phosphatases similar sequences found <br />                                                                          human Fcp1 15 predicted proteins unknown <br />                                                                          function organisms. Alternatively, short, <br />                                                                          conserved motifs define domain used interact a <br />                                                                          target site particular phosphatase family phospha- <br />                                                                          tases remains identified. If case, then <br />                                                                          Fcp1 signaling molecule couples phospha- <br />                                                                          tase enzyme RPIIyTFIIF complex. <br />                                                                             Yeast Fcp1 binds C-terminal evolutionarily <br />                                                                          conserved domain RAP74. An adjacent portion this <br />                                                                          domain binds RPII (35); RAP74 stabilizes posi- <br />                                                                          tion Fcp1 relative polymerase associated <br />                                                                          phosphatase dephosphorylate CTD. The phosphatase <br />                                                                          activity processive removing phosphate groups the <br />                                                                          CTD (18, 19), TFIIF itself necessary this <br />    FIG. 4. Binding Fcp1 RAP74. (A) Various 35S-labeled       processivity, human phosphatase (18). Thus, <br /> portions yeast Fcp1 human FCP1a, indicated, synthe-       stimulation phosphatase activity seen presence of <br /> sized rabbit reticulocyte lysates (input), applied GST         TFIIF likely reflects stabilized binding interaction between <br /> GST-yRAP74 (649&#226;&#8364;&#8220;735) microaffinity columns, processed             phosphatase polymerase. <br /> described Materials Methods. Volumes corresponding 5%          Interaction RAP74 Fcp1 likely involves amino <br /> the input 25% eluate loaded gel. Only the <br />                                                                          acids conserved yeast human proteins, yeast <br /> region autoradiogram containing full-length vitro synthesis <br /> products shown. (B) Various 35S-labeled portions yeast Fcp1    Fcp1 bind human RAP74 human FCP1a bind <br /> human FCP1a, indicated, chromatographed GST               yeast RAP74. Nevertheless, requirements enzyme <br /> GST-hRAP74 (436&#226;&#8364;&#8220;517) microcolumns. The columns run              activity complex simple binding assays would <br /> the eluates analyzed described A.                                 indicate, yeast human phosphatase <br />  <br />  <br /> can function RAP74 RPII derived heterol-                    <br /> ogous species (19). Moreover, S. cerevisiae does survive     <br /> when yeast FCP1 replaced cDNA encoding human                     <br /> FCP1a (J.A., unpublished data).                               <br />    The RPII recruited promoters dephosphorylated <br />                                                                               <br /> largest subunit (2, 3, 7), RPII complex specialized          <br /> initiation (36&#226;&#8364;&#8220;39) expected contain CTD phospha-           <br /> tase. There ample evidence CTD phosphorylation                <br /> important regulatory modification transition                   <br /> preinitiation efficient elongation (reviewed refs. 4 40).          <br /> Many transcription activators, including HIV-1 Tat, herpes sim-               <br /> plex virus VP16, human p53 E2F1, bind TFIIH (8,                 <br /> 9, 41) CDK7 subunit CTD kinase RPII (reviewed                     <br /> in refs. 42 43). Moreover, HIV-1 Tat protein,       <br /> stimulates elongation RPII (44, 45), stimulates kinase <br />                                                                              <br /> activity (11, 46, 47), phosphorylation CTD TFIIH           <br /> may important elongation RPII vivo (10, 40). A                  <br /> second CTD kinase, P-TEFb, required HIV Tat                <br /> transactivation (48, 49), kinase essential               <br /> productive elongation (48, 50).                             <br />    Further, RAP74 subunit mammalian TFIIF                   <br /> phosphorylated TFIIH (51) largest subunit                  <br /> TFIID (52), phosphorylated form TFIIF stimulates              <br /> elongation RPII vitro (53). Perhaps phosphorylation       <br /> of RAP74 context RPII holoenzyme leads the <br />                                                                           <br /> dissociation Fcp1, reducing eliminating CTD phospha-               <br /> tase activity elongation. Alternatively, phosphory-                 <br /> lation RAP74 simply inhibit CTD phosphatase                 <br /> activity. As well, TFIIB inhibitor CTD phosphatase                 <br /> activity (18) prevent dephosphorylation CTD                  <br /> during transition initiation elongation                 <br /> process. Activators like Tat, stimulate elongation            <br /> RPII, stimulating CTD kinase (11, 46, 47)                      <br /> and inhibiting CTD phosphatase, maintaining highly              <br /> phosphorylated form polymerase elongation <br />                                                                             <br /> complex emerges promoter. The sensitivity                     <br /> the polymerase elongation complexes action                <br /> phosphatase tested, regulating phosphor-                   <br /> ylation state polymerase necessary efficient              <br /> elongation transcription <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span>.                                    <br />   <br /> </body></html>