<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL OF BACTERIOLOGY, June 1994, p. 3790-3799                                                                                Vol. 176, No. 12 <br /> 0021-9193/94/$04.00 + 0 <br /> Copyright &#194;&#169;) 1994, American Society Microbiology <br />  <br />  <br />           The Iron Superoxide Dismutase Legionella pneumophila Is <br />                             Essential Viability <br />         ALESIA B. SADOSKY,&apos; JAMES W. WILSON,&apos; HOWARD M. STEINMAN,2                                    AND  SHUMAN&quot;* <br />                                                                                                             HOWARD A. <br />        Department Microbiology, College Physicians Surgeons Columbia University, New York, New York 10032, <br />                     Department Biochemistry, Albert Einstein College Medicine, Bronx, New York 104612 <br />                                               Received 21 January 1994/Accepted 23 March 1994 <br />  <br />             Legionella pneumophila, causative agent Legionnaires&apos; disease, contains superoxide dismutases <br />           (SODs), cytoplasmic iron enzyme (FeSOD) periplasmic copper-zinc SOD. <span id='am-4' about='protege:TO' typeof='owl:Thing'>To</span> study role the <br />           FeSOD L. pneumophila, cloned FeSOD gene (sodB) inactivated Tn903dlllacZ, forming a <br />           sodB::kacZ gene fusion. By using fusion, expression sodB shown unaffected variety of <br />           conditions, including influence sod expression Escherichia coli: aeration, oxidants, redox <br />           cycling compound paraquat, manipulation iron levels medium, stage growth. A reproducible <br />           twofold decrease sodB expression growth agar medium containing charcoal, potential <br />           scavenger oxyradicals, comparison growth medium charcoal. No induction was <br />           seen growth human macrophages. Additional copies sodB+ trans increased resistance to <br />           paraquat. Construction sodB mutant attempted allelic exchange sodB:ilacZ fusion the <br />           chromosomal copy sodB. The mutant isolated, allelic exchange possible if <br />           wild-type sodB present trans. These results indicate periplasmic copper-zinc SOD cannot <br />           replace FeSOD. The data strongly suggest sodB essential gene FeSOD required for <br />           viability L. pneumophila. In contrast, Sod- mutants E. coli Streptococcus mutans grow aerobically <br />           SOD required viability species. <br />  <br />    During aerobic growth, virtually organisms generate toxic               scavenging enzymes include peroxidase, SOD, possi- <br /> by-products respiration like hydroxyl radical (OH . ), hydro-               bly catalase (22, 38, 51, 56). In addition periplasmic <br /> gen peroxide (H202), superoxide radical (02-) (16, 20).                    CuZnSOD, L. pneumophila contains FeSOD located the <br /> These organisms, however, possess mechanisms combat                         bacterial cytoplasm (47a). The gene encoding L. pneumo- <br /> potentially destructive reactive oxygen intermediates (ROTs).                  phila FeSOD, sodB, recently cloned ability confer <br /> The principal enzymatic mechanisms procaryotes eu-                      paraquat resistance Escherichia coli AsodA AsodB strain <br /> caryotes superoxide dismutases (SODs) protect                   (51). Paraquat (methyl viologen) redox cycling agent that <br /> the harmful effects 02 catalases peroxidases                   generates 2- bacterial cytoplasm (8, 10). <br /> protect organisms effects H202 perox-                       In addition important detoxifying function bacterial <br /> ides.                                                                          SOD aerobic growth, SOD established as <br />    SODs metalloenzymes decompose superoxide rad-                      virulence factor bacterial pathogens (2, 21). For <br /> icals hydrogen peroxide molecular oxygen (28). Three <br /> types SOD exist procaryotes eucaryotes,                     example, SOD shown protect Nocardia asteroides <br /> contains different metal cofactor activity, i.e., iron (Fe),             bactericidal effects neutrophils infection <br /> manganese (Mn), copper zinc (CuZn) (6, 15, 49). The                    (2). Similarly, FeSOD mutants Shigella flexneri shown <br /> FeSOD commonly procaryotes,                       extremely sensitive killing effects host macro- <br /> MnSOD procaryotes mito-                         phages polymorphonuclear leukocytes (14). Because L. <br /> chondria eucaryotes (43). With exceptions, CuZn-                   pneumophila does trigger respiratory burst entry <br /> SOD primarily cytosol eucaryotes (1).                       phagocytes (4, 36), known FeSOD is <br /> Exceptions include CuZnSOD procaryotes Photobacte-                      important L. pneumophila&apos;s ability grow and/or <br /> rium leiognathi (41), Caulobacter crescentus (48), Haemophilus                 kill host cells. <br /> influenzae (25), Brucella abortus (3), Pseudomonas diminuta                       To determine role L. pneumophila FeSOD plays <br /> and Pseudomonas maltophilia (50), Legionella pneumo-                       growth organism aerobic conditions and <br /> phila (29, 51).                                                                host macrophages, cloned sodB gene mu- <br />    L. pneumophila, causative agent Legionnaires&apos; dis-                   tagenized Tn9O3dIIlacZ, derivative transposon <br /> ease, facultative intracellular pathogen humans                   Tn903. The transposon insertion generated sodB::lacZ fusion <br /> infects alveolar macrophages phagocytes, causing                  used attempts generate sodB mutant L. <br /> acute fatal pneumonia (58). L. pneumophila                       pneumophila study sodB gene expression. Our results <br /> requires 02 growth shown ROI-                         indicate like E. coli, L. pneumophila sodB gene <br />                                                                                largely constitutive induced oxidative stress. We <br />    * <br />      Corresponding author. Mailing address: Department Microbi-             unable generate sodB mutant strain allelic <br /> ology, College Physicians Surgeons Columbia University, 701          exchange fusion chromosomal sodB, suggesting that <br /> West 168th St., New York, NY 10032. Phone: (212) 305-6913. Fax:                FeSOD essential L. pneumophila viability. This contrasts <br /> (212) 305-1468. Electronic mail address: shuman@cuccfa.ccc.colum-              bacterial species Sod- strains viable <br /> bia.edu.                                                                       aerobically. <br />                                                                         3790 <br />  VOL. 176, 1994                                                               L. PNEUMOPHILA sodB GENE IS ESSENTIAL                   3791 <br />  <br />                                       TABLE 1. Bacterial strains plasmids used study <br />      Strain plasmid                                         Genotype features                                          Source or <br /> Strains <br />   L. pneumophila <br />      JR32                Homogeneous salt-sensitive isolate AM511                                                     42 <br />      AM511               Wild-type L. pneumophila Philadelphia-1 Smr (r- m+)                                             27 <br />      AB1156              AM511 pig::Tn9O3dIIlacZ                                                                         57 <br />      AB1267              JR32 pAB1                                                                                  This study <br />      AB1688              JR32 pAB100 sodB::lacZ integrated sodB locus                                            This study <br />      AB1701              JR32 pAB100                                                                                This study <br />      AB1703              AB1688 pAB101                                                                              This study <br />      AB1704              AB1688 pAM6                                                                                This study <br />      AB1716              sodB::lacZ mutant JR32 pAB101                                                           This study <br />      AB1717              sodB::lacZ mutant JR32 pAB101                                                           This study <br />      AB1778              pAB102 wild-type JR32                                                                        This study <br />      AB1782              pBC SK(+) wild-type JR32                                                                     This study <br />      JW1028              JR32 pJW26                                                                                 This study <br />   E. coli <br />     OX326A                E. coli /sodAsodB                                                                              51 <br />     CAR                   OX326A pCAR1                                                                              H. Steinman <br />     DH5cs                 F- endA41 hsdR17 (r- m+) supE44 thi-1 X- recAl reL41A(argF-lacZYA)U169 +80dlacZAM15            59 <br />                             deoR gyrA96 Nalr <br />     LE392Alac             e14- (r- m+) hsdR514 supE44 supF58 A(lacIZY)6 galK2 galT22 metBl trpRSS-                       26 <br />     LW211                 LE392 integrated RP4 (ACbr Tcr::Mu) transferred Pl transduction SM10, Mob+     L. Wiater <br />                              Kmnr <br />     LW252                 LW211 (KMr::Tn7) Kms Trir Spcr Smr                                                             L. Wiater <br />     SMio                  thi thr leu tonA lacY supE recA integrated RP4-2-(ACbr Tcr::Mu) KMr                    46 <br />  <br /> Plasmids <br />     pBluescript II KS(+) oriR(fl) MCS orR(ColEl) CbW                                                                     44 <br />     pBC SK(+)            oriR(fl) MCS oniR(ColE1) Cmr                                                                    Stratagene <br />     pAM6                 2.9-kb HindIII fragment (encoding Gmr) pLB41 cloned HindIII site pBSK II KS(+)     A. Marra <br />     pLB41                2.9-kb HindIII DNA Gmr DNA fragment pBR322                                                   L. Babiss <br />     pAB1                 trpA&apos;-lacZ+ 3.9-kb EcoRI-XmnI DNA pMLB1109 EcoRI-SmaI site pMMB207                This study <br />     pMLB1109             pBR322 rmB Ti term-(4X)-trpA&apos;-lacZ+                                                        M. Berman <br />     pAB100               sodB::lacZ 9-kb EcoRI DNA fragment EcoRV site pLAW344 (Cmr Cbr Kmr Lac+)               This study <br />     pAB101               sodB 4.5-kb EcoRI DNA cloned EcoRI site pAM6 (Gmr)                                     This study <br />     pAB102               sodB 4.5-kb EcoRI DNA fragment EcoRI site pBC (Cmr)                                    This study <br />     pAB104               orfl::lacZ 9-kb EcoRI DNA fragment EcoRV site pLAW344 (Cmr Cb, Kmr Lac&apos;)               This study <br />     pJW2                 sodB 4.5-kb EcoRI DNA fragment EcoRI site pMMB207 (Cmr)                                This study <br />     pJW26                sodB::lacZ 9-kb EcoRI DNA fragment EcoRI site pMMB207 (Cmr)                            This study <br />     pJW28                orfl::lacZ 9-kb EcoRI DNA fragment EcoRI site pMMB207 (Cmr Kmr Lac+)                      This study <br />     pCAR1                sodB contained 29-kb DNA fragment EcoRI site pLAFR1 (Tc&apos;)                              H. Steinman <br />     pKD368               tnpA Tn9O3dIIlacZ (&apos;lacZ Kmr) onR(ColEi) Apr                                                    K. Derbyshire <br />     pLAW344              sacB MCS oriT(RK2) Cmr loxP oriR(ColEi) Cbr loxP                                                57 <br />     pLAFR1               21.6-kb pRK290 derivative, Tcr (mob+ tra mutant)                                                17 <br />     pMMB207              RSF1010 derivative, IncQ lacP cat(Cmr) Ptac                                                     32 <br />  <br />  <br />                   MATERUILS AND METHODS                               tained healthy volunteers stored 5-ml aliquots at <br />                                                                       -80&#194;&#176;C. DNA restriction enzymes supplied New En- <br />    Bacterial strains, plasmids, cloning, DNA manipula-            gland Biolabs Boehringer Mannheim Biochemicals. M63 <br /> tions. The bacterial strains plasmids employed work       M9 media prepared described Miller (30). <br /> are listed Table 1. Preparation plasmid DNA DNA             Fructose sterilized filtration added M9 medium <br /> cloning procedures performed described              final concentration 0.2% (wt/vol). The substrate 5-bro- <br /> (26). Bacterial mating performed method Miller          mo-4-chloro-3-indolyl-,B-D-galactopyranoside (X-Gal) was <br /> (30).                                                                 used indicator ,B-galactosidase activity. The antibiotic <br />    Media reagents. E. coli strains grown overnight in <br /> Luria-Bertani broth (LB) LB agar plates described            concentrations used E. coli follows: tetracycline, 20 <br /> previously (30). L. pneumophila grown ABYE broth               ,ug/ml; kanamycin, 50 ,g/ml; carbenicillin ampicillin, 100 <br /> (23) ABCYE medium (12). Reagents, chemicals,                ,ug/ml; chloramphenicol, 25 ,ug/ml; gentamicin, 15 jig/ml. <br /> fetal calf serum purchased Sigma. Iron added         Antibiotics used L. pneumophila follows: strepto- <br /> ferric nitrate ABYE ABCYE medium 250 ug/nml,            mycin, 50 ,ug/ml; kanamycin, 25 jig/ml; chloramphenicol, 5 <br /> L-cysteine added 400 ,ug/ml. For cell culture, RPMI 1640       ,ug/ml; gentamicin, 5 ,ug/ml. <br /> was obtained JRH Biosciences L-glutamine (Gln)                  Determination stress conditions ABCYE medium. The <br /> was obtained Cellgro. Normal human serum ob-                 chemical compounds H202 1,1&apos;-dimethyl-4,4&apos;-bipyri- <br />  3792     SADOSKY ET AL.                                                                                                 J. BACTERIOL. <br />  <br />  dinium dichloride (paraquat; Sigma) added increasing       verified Southern blot analysis (data shown). The Sucr <br />  amounts (50 ,uM 10 mM) ABCYE medium prior             <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span>phenotype</span> associated bacteria containing integrated <br />  addition wild-type L. pneumophila JR32. After 3 days         copy pLAW344 sequences likely resulted sacB&apos;s <br />  37&#194;&#176;C, extent bacterial growth stress plates      inability confer sensitivity Suc single copy. Alterna- <br />  compared strain JR32 grown ABCYE medium            tively, sacB gene mutated integra- <br />  (data shown). The compound           tion pLAW344. Considering frequency random <br />  resulted slight decrease growth wild-type strain    <span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span>mutation</span>, believe scenario unlikely Sucr <br /> JR32 used stress concentration measuring effects     isolates obtained procedure contain integrated copy <br /> on sodB expression. These concentrations 5 mM              pLAW344. <br /> H202 0.5 mM paraquat.                                          DNA sequencing. DNA sequencing performed the <br />    Transposon mutagenesis screen SOD activity. The          DNA Synthesis Sequencing Facility Comprehensive <br /> L. pneumophila sodB gene mutagenized Tn9O3             Cancer Center, College Physicians Surgeons Colum- <br /> derivative, Tn903dIIlacZ. The transposon contains gene            bia University. The primer used determine nucleotide <br /> encoding resistance kanamycin (Kmr) carries 5&apos;             sequence lacZ fusion joint prepared the <br /> truncated lacZ gene capable generating lacZ fusions.     DNA Synthesis Sequencing Facility, mentioned above. <br /> The plasmid pJW2 encodes L. pneumophila FeSOD activity           The DNA sequence primer, 5&apos;-CCCAGTCAC <br /> the 4.5-kb EcoRI insert DNA cloned RSF1010               GACGTTG-3&apos;, corresponds nucleotides 31 46 of <br /> derivative pMMB207. Plasmid pMMB207 encodes chloram-                lacZ coding region. DNA sequence analysis by <br /> phenicol resistance (Cmr) mob&apos;. pJW2 introduced          using MacDNASIS version 1.0 program. Nucleic acid and <br /> into strain LW252, selection Cmr, generate strain       amino acid similarity searches GenBank/EMBL and <br /> JW1030. LW252 contains mobilization (tra) genes RK2          SwissProt data bases performed using Sequence <br /> on integrated copy RK2. Tn903dIIlacZ introduced           Analysis Software Package obtained Genetics Computer <br /> into strain JW1030 pBR322 derivative pKD368,            Group, Inc. <br /> selection Cmr (encoded pMMB207), carbenicillin resis-           HL-60 cell culture. The human leukemia cell line HL-60 (9) <br /> tance (Cbr) (encoded pKD368 vector sequences), Kmr           maintained RPMI 1640 medium supplemented 2 <br /> (encoded Tn903dIIlacZ pKD368). Independent Cmr Cbr            mM Gln 10% fetal calf serum. HL-60 cells differen- <br /> Kmr colonies isolated subsequently used donors       tiated macrophages incubating 2 days 10 <br /> transfer pJW2 nalidixic acid-resistant (Nalr) recipient    ng phorbol 12-myristate 13-acetate ml RPMI con- <br /> DH5oS bacterial mating. One Lac&apos; Nalr Cmr KMr            taining 2 mM Gln plus 10% normal human serum. Adherent <br /> transconjugants isolated, Lac- Nalr Cmr       cells washed times RPMI containing 2 mM <br /> Kmr transconjugants. Plasmid DNA prepared             Gln incubated RPMI containing 2 mM Gln plus 10% <br /> of strains introduced E. coli AsodAsodB          normal human serum prior infection. <br /> strain OX326A, selection growth LB containing              Measurement I8-galactosidase activity. 1-Galactosidase <br /> chloramphenicol. Cmr colonies purified LB              activity measured quantitating hydrolysis o-ni- <br /> containing chloramphenicol streaked M9 con-           trophenyl-o-D-galactopyranoside (ONPG) described by <br /> taining chloramphenicol plus 0.2% fructose. After 2-day           Miller (30). During growth L. pneumophila HL-60 cells, <br /> incubation 37&#194;&#176;C, growth bacterial transformants        3-galactosidase activity determined described Wiater <br /> was compared OX326A containing pJW2          et al. (57). <br /> pMMB207. Unlike pMMB207 (Sod-), pJW2 (Sod&apos;) demon-                     Southern blotting. Genomic DNA prepared according <br /> strated healthy colony formation M9 medium containing            method Silhavy et al. (45), digested EcoRI, <br /> fructose 2 days. For 28 isolates, sites Tn903dIIlacZ   separated electrophoresis 0.7% agarose gels, trans- <br /> insertion determined restriction analysis.                  ferred Hybond-N nylon membranes (Amersham) the <br />    Allelic exchange. Allelic exchange accomplished L.        method Southern (47). DNA probes prepared by <br /> pneumophila previously described (57) using vector        random-primed labelling [32P]dATP large Kle- <br /> pLAW344, contains Cmr marker counter-               fragment DNA polymerase I (13). <br /> selectable sacB gene Bacillus subtilis. The sodB::lacZ            Determination FeSOD aconitase activities. SOD <br /> fusion contained pJW26 downstream open reading           activity visualized nondenaturing electrophoresis in <br /> frame (orfl) orfl::lacZ fusion contained pJW28,             polyacrylamide gels riboflavin-nitroblue tetrazolium <br /> marked Km&apos; gene Tn903dIIlacZ, isolated            method (50). SOD activity measured solution the <br /> 9-kb EcoRI DNA fragments inserted EcoRV site           pyrogallol method (40). Aconitase activity quantitated, <br /> of pLAW344. The resulting plasmids, pAB100 pAB104,              citrate substrate, coupling isocitrate dehydro- <br /> respectively, introduced L. pneumophila JR32           genase measuring reduction NADP+ (18). One <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> of <br /> electroporation, grown ABYE 5 h, plated               aconitase activity defined 1 nmol NADPH formed per <br /> ABCYE containing kanamycin. Kmr transformants                  min. <br /> streaked plates containing ABCYE plus kanamycin plus 2% <br /> (wt/vol) sucrose (Suc) select cells longer                                   RESULTS <br /> contained pLAW344 sequences. Kmr Sucr colonies were <br /> streaked nonselective ABCYE medium and, 3 days,               Subcloning transposon mutagenesis sodB with <br /> single isolates patched plates containing ABCYE plus        Tn903dIlkacZ. Because repeated attempts mutagenize sodB <br /> kanamycin, ABCYE plus chloramphenicol, ABCYE plus               Tn9O3dIIlacZ pCAR1, 29-kb cosmid containing the <br /> 2% Suc. One set ABCYE plates overlaid 0.8%              L. pneumophila sodB gene (47a, 51), unsuccessful (data <br /> agarose containing 0.7 mg X-Gal ml, Lac              shown), sodB gene isolated smaller DNA <br /> phenotype patches determined. The total number        fragment. Plasmid pCAR1 digested EcoRI, five <br /> Kmr Sucr Cms Kmr Sucr Cmr isolates exchange            DNA fragments (21.6, 20, 4.5, 3, 1.5 kb) identified. <br /> was counted, presence transposon (Kmi)           Four 5 EcoRI fragments (the 21.6-kb fragment not <br /> pLAW344 vector sequences (Cmr) genomes             isolated corresponded pLAFR1 vector DNA) <br />  VOL. 176, 1994                                                                   L. PNEUMOPHILA sodB GENE IS ESSENTIAL                                             3793 <br />  <br /> were cloned RSF1010 derivative pMMB207 tested           1                                                               66 <br />                                                                      ATGACTTTTACCCTGCCGCAATTACCTTATGCATTGGATGCACTGGCTCCTCATGTTTCCAAAGAA sodB <br /> for Sod phenotype E. coli AsodA AsodB strain              M T F T L P Q L P Y A L D A L A P H V S K E <br /> OX326A (see Materials Methods). The Sod&apos; phenotype <br />                                                                                                                                      132 <br /> was encoded 4.5-kb EcoRI DNA fragment; hence,             67 <br />                                                                      ACCTTGGAATACCATTATGGTAAACACCATAACACTTACGTCACCAACTTAAATAAATTAATTCCC sodB <br /> corresponding plasmid pJW2 used Tn903dIIlacZ mu-              T L E Y H Y G K H H N T Y V T N L N K L I P <br /> tagenesis.                                                           133                                                            198 <br />    Following mutagenesis pJW2, location                GGAACAGAATTTGAATCCATGACTCTGGAAGAGATCATCATGAAAGCTAAAGGCGGTATTTTTAAC sodB <br /> Tn9O3dIIlacZ plasmid determined 51 Lac&apos;            G T E F E S M T L E E I I M K A K G G I F N <br />  <br /> and 5 Lac- mutants. Of 28 insertions 4.5-kb insert        199                                                                                     264 <br /> DNA, 1 (pJW26) exhibited Sod- Lac&apos; phenotype. The             AACGCCGCTCAGGTTTGGAATCACACCTTTTATTGGCACAGCATGAGCCCTAATGGTGGAGGTGAG sodB <br />                                                                       N      A   A   Q   V   W   N   H   T   F   Y   W   H   S   M   S   P   N   G   G   G    E <br /> Lac&apos; phenotype suggested lacZ gene <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> Tn903dIIlacZ fused L. pneumophila sodB sequences.           265                                                                                     330 <br />  <br /> The low frequency Sod- mutants (1 28, 3.6%) reflect         P      K G     R   L A     E   A   I   N   K   S   F   G   S   F   A A     F   K   E Q <br /> a small target size transposition cold spot                                  26 <br />  <br /> Tn903dIIlacZ transposition. <br />                                                                      331                     v <br />                                                                      TTTTCACAAACAGCTGCTACAACATTCGGTTCCGGGTGGGCATGGCTAGTCCAGGATCAGTCAGGG <br />                                                                                                                                                              396 <br />                                                                                                                                                                    sodB <br />    DNA sequencing sodB::lacZ fusion joint. To determine        F  S Q T A A T       T  F G S G W A W L V           Q D Q S G <br />  <br /> if Sod- phenotype associated pJW26               397                                                                                     462 <br /> Tn903dIIlacZ insertion sodB structural gene,          GCTTTAAAAATTATCAACACAAGCAATGCTGGAACACCAATGACTGAAGGATTAAATGCCTTATTA sodB <br />                                                                       A L   K   SI   N T S N A G T P M T E G L N A L L <br /> DNA sequence lacZ fusion joint determined. <br /> The nucleotide sequence showed 100% identity            463                                                             528 <br />                                                                      ACTTGTGACGTATGGGAGCACGCTTATTACATCGATTATAGAAATAGGCGCCCGGATTATATTGAA                             sodB <br /> the L. pneumophila sodB gene submitted J. Amemura-                 T C D V W E H A Y Y I D Y R N R R P D Y I E <br /> Maekawa H. Watanabe Tokyo, Japan (GenBank acces-              529                                                                                     594 <br /> sion no. D12922). With complete sodB sequence               GCCTTTTGGTCATTAGTTAATTGGGATTTTGCGAGCAGTAATCTGAAATAAAATAACAAGCTGGAT                            sodB <br /> reference, Tn903dIIlacZ insertion pJW26 (sodB-1)           A      F   W   S   L   V   N   W   D   F   A   S   S   N   L   K <br />  <br /> mapped nucleotide 346 (codon 116) sodB coding              595                                                            660 <br /> region, shown Fig. 1. Examination L. pneumophila        TTGTTGCAATCCCTACCGGATAATTTTATGTTTATGGAGACATCNATGAGTTTAATTACAAGTTAC orfl <br />                                                                                                                                      M   S   L   I   T   S    Y <br /> sodB::lacZ fusion joint shown Fig. 2A revealed sodB was <br /> fused frame respect lacZ gene                  661 <br />                                                                      AGCCCAATGCCTGTAACCTTCACGCACGGAGAAGGCGTGTGGTTGTATGATGAGCAGGGGAAAAAA <br />                                                                                                                                                              726 <br />                                                                                                                                                                    orf1 <br /> Tn903dIIlacZ, resulting transcriptional fusion.                  S    P     M   P   V   T   F   T   H   G   E   G   V   W   L   Y   D   E   Q   G   K    K <br />    Identification expressed ORF. To consider possi- <br /> bility Sod- phenotype associated pJW26         727 <br />                                                                      TACCTCGATGGCTTAAGCGGGATAGCAGTTTGTGGATTAGGTCATGCACACCCTGATGTCACCCGT <br />                                                                                                                                                              792 <br />                                                                                                                                                                    orf1 <br /> due insertion sodB structural gene itself,       Y L D G L S G I A V C G L G H A H P D V T R <br /> rather failure express open reading frame           793                                                            858 <br /> (ORF) downstream sodB, number Lac&apos; Lac-                ACCATACAGGAGCAGGCAGCCAGGCTAATACATACTTCTAATGTCTTTCATATTAAAGAACAAGAA orfl <br />                                                                       T I Q E Q A A R L          I H T S N V F H I K E Q E <br /> Tn903dIIlacZ insertions pJW2 examined. One <br /> Lac&apos; Sod&apos; Tn903dIIlacZ insertion (pJW28) mapped                 859                                                            924 <br /> close Lac&apos; Sod- insertion pJW26, suggesting              TTATTAGCCGCCAAACTCACTTCAATGGCACGAATGGAGCAGGTTTTTTTTGCTAATTCTGGAGCA <br />                                                                       L      L   A   A   K   L   T   S   M   A   R   M   E   Q   V   F   F   A   N   S   G   A <br />                                                                                                                                                                    orf1 <br />  <br /> Tn9O3dIIlacZ insertion sodB lost SOD activity. <br /> The sequence lacZ fusion joint showed identity       925                                                            990 <br />                                                                      GAAGCTAATGAAGCAGCCATTAAACTTACTCGTCTGTTTGGTCATAAAAAGGGAATTGAAACGCCT orf1 <br /> the 3&apos; end sodB coding region. Rather               E A   N E A A I K L T R L F G H K K G I E T P <br /> inserted sodB, however, Tn903dIIlacZ transposed           991                                                            1056 <br /> into ORF began 60 nucleotides downstream            TCTAtTATTGTAATGGAAAAAGCATTTCATGGTCGCACTATGGCCACCTTAACTGCATCAGGCAGT orf1 <br /> stop codon sodB, illustrated Fig. 1. The transposon      S <br />                                                                       28 <br />                                                                            I     I   V   M   E   K   A   F   H   G   R   T   M   A   T   L   T   A   S   G    S <br />  <br />  <br /> situated 419 nucleotides downstream putative start          V <br /> site orfl, examination fusion joint revealed   CGTAAAGTT <br />                                                                       R K V <br />                                                                                      orf1 <br /> contrast sodB, translational fusion generated <br /> between orfl lacZ Tn903dIIlacZ, shown Fig. 2B.             FIG. 1. Location Tn903dIIlacZ sodB orfl. The DNA <br /> Because orfl::lacZ insertion able complement              sequence   shown predicted amino acid sequence (shown in <br /> OX326A Sod&apos;, believe sequences downstream            single-letter abbreviation) encoded sodB orfl. Methionines <br /> the sodB gene required expression Sod&apos;            represent potential start sites sodB orfl translation, the <br /> phenotype E. coli. We exclude possibility     asterisk marks putative stop codon sodB. The sites of <br />                                                                      Tn903dIIlacZ insertion sodB orfl shown open triangles, <br /> orfl::lacZ fusion retains orfl downstream gene activity      number symbol identifies plasmid which <br /> required Sod&apos; phenotype, 3&apos; end orfl        nucleotide sequence derived: pJW26 sodB pJW28 for <br /> coding region obtained.                                 ocrl. <br />    The L. pneumophila sodB gene essential viability. We <br /> attempted construct sodB mutant L. pneumophila to <br /> study role FeSOD oxidative damage. The <br /> sodB::lacZ fusion pJW26 (conferring Sod- phenotype           details), Km&apos; Sucr colonies screened Cmr <br /> OX326A) transferred 9-kb EcoRI DNA fragment                 ,-galactosidase expression. L. pneumophila naturally <br /> into EcoRV site pLAW344 5&apos; cohesive ends            Lac-; therefore, lacZ expression entirely the <br /> had filled DNA polymerase I, Klenow fragment.           sodB::lacZ fusion. Of 22 Kmr Sucr isolates tested, 22 were <br /> The resulting plasmid pAB100, introduced wild-type          Lac&apos; Cmr, indicating pLAW344 vector DNA was <br /> L. pneumophila JR32, selection Cmr (from vector)        present genomes putative exchange mutants. <br /> and Kmr (from transposon). Kanamycin-resistant colonies          Southern blot analysis genomic DNA prepared 6 the <br /> were isolated streaked ABCYE medium containing                22 Kmr Sucr Cmr isolates verified presence the <br /> kanamycin plus 2% Suc (see Materials Methods                 wild-type sodB mutant sodB::lacZ alleles (data shown). <br />  3794        SADOSKY ET AL.                                                                                                                           J. BACTERIOL. <br />  <br />                                                                                                                    cmr <br /> A       sodB::1acZ fusion joint:                                             A <br />                                                 kwLZ <br />         K   E   Q   F   S   Q   T   A       A   L   L   N   K                                                  1        2 <br /> ...    AAA GAA CAA TTT TCA CAA ACA GCT G*C GCT TTG TTG AAT AAA <br />                                                                                                            /       sodB       \ <br />  <br />  <br />                                                                            KmrLac*      cmr                                                    cmr       KmrLac <br />  <br />                                                                           sodB::lacZ                sodB                      sodB                        sodB::lacZ <br /> B       orfl::IacZ fusion joint: <br />  <br />                        647                      krZ <br />         T   L   T   A   S   G   S    R   A   L   L   N   K <br /> ...    ACC TTA ACT GCA TCA GGC AGT CG*C GCT TTG TTG AAT AAA       .. <br />  <br />  <br />   FIG. 2. Nucleotide sequence Tn9O3dIIlacZ fusion joint for <br /> sodB (A) orfl (B). The DNA sequence determined using a <br />                                                                            B <br /> primer derived noncoding strand lacZ gene (see <br /> Materials Methods). The fusion joints sodB orfl                     cm,                                            Cmr <br /> lacZ marked asterisk. <br />  <br />  <br /> These results contrasted obtained orfl,                       sodB::lacZ                                       sodB <br /> all allelic exchange mutants tested contained                                                         or <br /> mutant orfl::lacZ allele genomes (data shown). <br />    The inability generate sodB <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span>null allele</span> suggested                                                                       sodB::bacZ <br /> that FeSOD essential viability a <br /> recombination block sodB gene, making                FIG. 3. (A) Integration sodB-containing plasmid pAB100 <br /> impossible obtain double crossover wild-type           chromosome strain JR32 homologous recombination. <br /> and mutant sodB alleles. The scenario does appear         The possible outcomes, shown 1 2, integration of <br /> be limitation homologous DNA                  pAB100 (circle) wild-type strain JR32 depend site at <br /> flanking transposon, Tn9O3dIIlacZ situated near            crossover wild-type sodB (open rectangles) and <br /> the middle 4.5-kb EcoRI DNA fragment (data                   mutant sodB::lacZ (black-centered rectangles) alleles occurs. (B) <br /> shown). In addition, problem              Excision pLAW344 DNA sequences homologous recombination <br />                                                                         sodB sodB::lacZ. Illustrated possible scenarios for <br /> exchange orfl, located immediately downstream sodB,             excision, dependent site recombination inte- <br /> there does appear block ability recombine         grated sodB sodB::lacZ genes. Plasmid sequences (encoding Cmr) <br /> within flanking DNA sequences. Similarly, unlikely           represented lines, chromosomal DNA sequences are <br /> the insertion Tn9O3dlIlacZ sodB polar expression        represented lines. <br /> of downstream gene essential cell viability, because <br /> the chromosomal orfl::Tn9O3dIIlacZ insertion mutant L. <br /> pneumophila viable. <br />    Loss sodB+ sodB+IsodB::JacZ merodiploid. All            plated ABCYE medium, 3 days 37&#194;&#176;C, the <br /> the sodB exchange mutants contain wild-type            bacteria replica plated ABCYE plates containing <br /> a mutant copy sodB genomes, pABIOO              chloramphenicol kanamycin. For independent experi- <br /> had integrated homologous recombination L.                  ments, sodB::lacZ mutant allele lost 1.4 1.8% the <br /> pneumophila chromosome sodB locus. Possible cointe-              time, shown Table 2. No Kmr Cms Lac&apos; colonies were <br /> grate structures result recombination between <br /> the wild-type chromosomal copy sodB mutant <br /> (sodB::lacZ) copy pAB100 shown Fig. 3A. <br />    The loss pLAW344 vector sequences                   TABLE 2. Loss mutant wild-type sodB alleles the <br /> wild-type sodB mutant sodB::lacZ gene occur                                      cointegrate AB1688 <br /> homologous recombination frequency. Possible out-                               No. of <br />                                                                                                  No. Km&apos;                 No. Km&apos;S <br /> comes resulting recombination wild-type                flto.      o.GinCs <br />                                                                                        colonies&quot;      Lac&apos; <br />                                                                                                   Cm&apos; Lac+                  Gins Lac-                 Frequency loss <br />                                                                                                                                                      sodB::IacZ  of <br /> mutant sodB alleles cointegrate subsequent loss <br /> of plasmid sequences strains illustrated Fig.             1            212                   0                   3                     0.014 <br /> 3B. Loss sodB::lacZ mutant allele result KmS              2            222                   0                   3                     0.014 <br /> Cm&apos; Lac- phenotype, loss wild-type sodB allele               3            899                   0                  15                     0.017 <br /> would confer KMr Cm&apos; Lac&apos; phenotype. If sodB                         4            965                   0                  17                     0.018 <br /> essential, isolates lost mutant allele (Kms             L. pneumophila AB1688 grown absence antibiotic selection as <br /> Cms Lac-) anticipated.                                         described text, total number colonies experiment was <br />    To test hypothesis sodB essential, Kmr Sucr Cmr        counted. <br /> isolate contained wild-type mutant copy sodB,             b The number bacterial colonies lost pLAW344 (encoding Cmr) <br /> L. pneumophila AB1688, tested loss            wild-type sodB+ allele strain AB1688, having Km&apos; Cms Lac&apos; <br /> wild-type mutant sodB allele. Strain AB1688 grown                phenotype. <br />                                                                            The number bacterial colonies lost pLAW344 (encoding Cmr) <br /> overnight ABYE selection, diluted 1:100 fresh           mutant sodB::IacZ allele strain AB1688, having Kmn Cms Lac- <br /> culture medium, grown saturation. The bacteria             phenotype. <br />  VOL. 176, 1994                                                                                  L. PNEUMOPHILA sodB GENE IS ESSENTIAL                           3795 <br />  <br /> TABLE 3. Loss chromosomal sodB+ gene                presence            TABLE 4. Loss pABIOI (sodB&apos; ) sodB::lacZ mutants <br />                  additional copy sodB+                                                                L. pneumophila <br />                                                        No. colonies                                               No.           No. Gms colonies that <br />                                          No.          allele <br />                                                                   present           Strain           No. of <br />                                                                                                        colonies&apos;           Gm&apos;               contain chromosomal <br />                                                                                                                                                   sodB::lacZ&apos; <br />  Strain      Plasmid <br />                            No   of <br />                           colonies        cmsl        bacterial chromosome                                               colonies&apos; <br />                                         colonies&apos; <br />                                                      sodB+&apos;         sodB::IacZd       AB1703             2,166              83                     4 20 <br />                                                                                       AB1704             2,445              72                     30 32 <br /> AB1703       pABIOI         487             3           1                   2         AB1716             2,288               6                     0 6 <br /> AB1704       pAM6           397             0           0                   0         AB1717             1,594               8                     0 8 <br />   &quot; L. pneumophila strains grown described text, total           &quot; L. pneumophila strains grown described text, number <br /> number Gmr colonies containing pABIOI (sodB+ pAM6) pAM6               bacterial colonies counted. <br /> were counted. <br />    &quot;The number Cms isolates presumably lost pLAW344 DNA <br />                                                                                         &quot;The number isolates lost pABIOI (sodB+ pAM6) or <br />                                                                                       pAM6 DNA sequences Gms. <br /> sequences (encoding Cmr) chromosome strain AB1688. <br />                                                                                         &apos; The number Gms isolates tested contain sodB::lacZ mutant allele <br />   &apos;CThe number Cms colonies lost pLAW344 DNA sequences               genomes, denoted Km&apos; Lac&apos; phenotype. <br /> the sodB::lacZ mutant allele chromosome parent strain AB1688, <br /> having Km&apos; Cms Lac- phenotype. <br />   &quot; The number Cms colonies lost pLAW344 DNA sequences and <br />  <br /> the wild-type sodB+ allele parent strain AB1688, having Kmr Cm, <br /> Lac&apos; phenotype.                                                                       antibiotic. After 24 h, cultures diluted 1:100 <br />                                                                                       fresh ABYE grown saturation allow for <br />                                                                                       loss plasmid DNA. The procedure repeated, and <br />                                                                                       dilutions bacteria plated ABCYE medium and <br /> observed, indicating wild-type sodB gene lost                       incubated 37&#194;&#176;C 4 days. To look cells lost <br /> L. pneumophila AB1688 frequency &lt;0.04%.                                       pABI01 pAM6, bacteria replica plated ABCYE <br />    If wild-type sodB gene lost strain AB1688                         medium ABCYE containing gentamicin incubated <br /> because essential, supplying second copy sodB+                        overnight 37&#194;&#176;C, total numbers Gmr Gms <br /> should permit recovery recombinants chromo-                           CFU determined strain. In cases, Gms <br /> somal sodB gene lost. This tested transforming strain                       isolates analyzed patching plates containing <br /> AB1688 pAM6, gentamicin-resistant (Gmr)                                 ABCYE ABCYE plus kanamycin check pres- <br /> pBluescript II derivative, pAB101 (sodB+ pAM6)                              ence Tn9O3dIIlacZ. After 2 days 37&#194;&#176;C, ABCYE plates <br /> selecting growth ABCYE medium containing gentami-                              overlaid X-Gal determine Lac phenotype of <br /> cin. Gmr transformants AB1703 (AB1688 pAB101)                                patched colonies. <br /> AB1704 (AB1688 pAM6) grown selection                                Approximately 2,000 colonies screened of <br /> previously described plated ABCYE medium 3 days                            strains AB1703, AB1704, AB1716, AB1717, the <br /> at 37&#194;&#176;C, bacterial colonies replica plated ABCYE                          frequency obtaining Gms isolates determined be <br /> plates containing gentamicin, gentamicin plus chlorampheni-                           3.8% strain AB1703 2.9% strain AB1704 (Table 4). <br /> col, gentamicin plus kanamycin. For strains AB1703                           Gms isolates sodB::lacZ mutant strains AB1716 and <br /> and AB1704, approximately 450 colonies tested loss                        AB1717 observed lower frequencies, 0.3 0.5%, <br /> either mutant wild-type sodB allele.                                           respectively (Table 4). Plasmid loss occurred frequently, <br />    For isolates strain AB1688 containing pAB101, loss                       therefore, strains contained chromosomal copy <br /> chromosomal sodB::lacZ allele occurred 0.2% time,                           wild-type sodB gene. Interestingly, Gms derivatives of <br /> identified Gmr Km&apos; Cms Lac- bacterial colonies. Loss                            strains AB1716 AB1717 Kms Lac-, suggesting <br /> the wild-type sodB+ allele chromosome, indicated                          mutant sodB::lacZ allele lost the <br /> a Gmr KMr Cms Lac&apos; phenotype, observed 2 487                           bacterial chromosome replaced wild-type copy of <br /> isolates (0.4%) (Table 3). No loss wild-type mutant                         sodB prior loss plasmid. Exchange mutant <br /> sodB allele observed isolates AB1688 containing                            sodB::lacZ allele wild-type sodB allele verified, in <br /> only vector pAM6 (Table 3). The Gmr KMr Cms Lac&apos;                              cases, Southern blot analysis genomic DNA prepared <br /> isolates AB1688 containing pABl01 saved strains                            representative number isolates (data shown). <br /> AB1716 AB1717. As determined plasmid DNA                                   From results, conclude sodB gene essential <br /> Southern blot analyses, strains contained wild-type copy                       growth L. pneumophila standard bacteriological <br /> of sodB+ pAB101, mutant sodB::lacZ allele                            medium. <br /> within chromosomes (data shown). These results                                 The effect sodB copy number FeSOD activity, paraquat <br /> demonstrate loss wild-type chromosomal locus                              resistance, aconitase activity. Because L. pneumophila <br /> could occur second copy sodB+ provided trans.                          sodB gene essential, possible generate null <br />    Failure observe loss plasmid copy sodB+                          mutant study effects oxidative damage strain <br /> two sodB::lacZ mutants. If sodB essential, bacteria                           lacking FeSOD activity. In E. coli, resistance paraquat is <br /> maintain functional copy gene. For wild-type L.                              strongly dependent SOD. Strains elevated SOD levels <br /> pneumophila JR32 merodiploid AB1688, copy                             increased resistance paraquat. Therefore, level of <br /> located bacterial chromosome, strains                          paraquat resistance depend number copies of <br /> AB1716 AB1717, copy maintained episomally                              sodB L. pneumophila. To test increased copy number <br /> pAB101. It possible, therefore, lose pAB101                         sodB results increased FeSOD production and, corre- <br /> strain AB1703 (pABIOI AB1688), impossible lose                              spondingly, increased paraquat resistance, pBC SK(+) and <br /> pABIOI strains AB1716 AB1717. Because pres-                              pAB102 (sodB+) introduced wild-type L. pneumo- <br /> ence pAB101 confers Gmr strains L. pneumophila,                          phila JR32 strains tested SOD activity and <br /> loss pAB101 easily monitored.                                               paraquat resistance. <br />    Strains AB1703, AB1704 (pAM6 AB1688), AB1716,                                  Bacterial strains grown saturation ABYE contain- <br /> AB1717 grown overnight 37&#194;&#176;C ABYE medium                                    ing chloramphenicol measured SOD activity and <br />  3796                 SADOSKY ET AL.                                                                                                 J. BACTERIOL. <br />  <br />       A                        B                                               difference aconitase activity observed, 0.024 0.027 <br />                                                                                units/mg protein strain, respectively. This result <br />                                                                                suggests wild-type (single-copy) levels FeSOD are <br />                                                                                adequate protect aconitase inactivation that <br />                                                                                multiple copies sodB provide protection. In <br />                                                                                addition, observed differences paraquat resistance between <br />  F.,1,LS,L,:   --- <br />                                                                                strains attributable differences in <br />                                                                                aconitase, major iron-containing protein L. pneumo- <br />    FceS&apos;[. <br />        )i.                                         I                           phila. <br />                                                                                   Effect growth conditions sodB expression. 1-Galacto- <br />                                                                                sidase activity measured strains AB1688 (chromosomal <br />                                                                                sodB::lacZ) JW1028 (plasmid sodB::lacZ) determine <br />    FIG. 4. (A) Effect multiple copies sodB FeSOD activity.            effect different parameters FeSOD gene expression. <br /> Extracts strains AB1778 (pAB102 JR32) AB1782 [pBC                  Hydrogen peroxide paraquat concentrations pro- <br /> SK(+) JR32] prepared, protein concentrations deter-               duced slight growth inhibition (5 0.75 mM, respectively) <br /> mined bicinchoninic acid method (Pierce Chemical),                  produced change sodB expression (data shown). <br /> approximately equal amounts protein electrophoresed                1-Galactosidase activity measured following growth on <br /> nondenaturing polyacrylamide gel described text. Bands               plates stationary-phase log-phase (1- 2-h) <br /> migrating L. pneumophila CuZnSOD FeSOD shown.                   cultures liquid medium. <br /> (B) Effect multiple copies sodB paraquat resistance. The left <br /> and right panels represent strains AB1778 AB1782 described                 The effect iron availability tested, E. coli sodB <br /> above. Briefly, L. pneumophila strains grown overnight satu-           control fur (ferric uptake regulation) (35) and <br /> ration 37&#194;&#176;C roller drum. Approximately 108 bacteria               L. pneumophila requires high concentrations iron <br /> added 3 ml 0.8% agar, mixture overlaid plates         optimal growth (7). To determine sodB expression is <br /> containing ABCYE plus chloramphenicol. After agar solidified,              dependent levels iron growth medium, <br /> sterile discs placed center plate 5 ,ul freshly    P-galactosidase activity measured cultures grown 2 h <br /> prepared 10-mg/ml stock solution paraquat dichloride added           liquid medium 250 ,ug ferric nitrate per <br /> each disc. The plates incubated 48 h 37&#194;&#176;C, zone         ml. For strains AB1688 JW1028, ,B-galactosidase <br /> clearing, i.e., zone growth inhibition, observed.                       activity significantly different control versus treated <br />                                                                                cultures (data shown). Moreover, change seen for <br />                                                                                strain addition 1,10-phenanthroline, iron <br /> sensitivity paraquat described above. The SODs L.                 chelator, concentration (50 ,uM) reduces growth an <br /> pneumophila easily resolved nondenaturing polyacryl-                    overnight culture 50%. <br /> amide gel electrophoresis, FeSOD migrates                       Expression sodB dependent stage of <br /> the anodic SOD (51). Figure 4A illustrates wild-type                 growth liquid medium (Fig. 5) strains AB1688 and <br /> strain JR32 multiple copies sodB+ (strain AB1778)                      JW1028. The plasmid-borne fusion generated ,B-galacto- <br /> demonstrated increase FeSOD activity comparison                       sidase activity (an average 95 units strain JW1028 <br /> with JR32 containing pBC SK(+) (strain AB1782). This                           compared 15 units AB1688), likely the <br /> contrasted slight decrease CuZnSOD activity                        increased copy number fusion. The increased level of <br /> observed wild-type JR32 containing multiple copies sodB                   3-galactosidase activity strain JW1028 permitted to <br /> rather sodB single copy (Fig. 4A, compare strain                       measure sodB expression growth human macro- <br /> AB1778 AB1782). Quantitative assays SOD activity                    phages, bacterial titers low. Similar expres- <br /> extracts strains AB1778 AB1782 indicated FeSOD                     sion growth culture, sodB gene expressed <br /> activity increased factor 1.4, 35 50%                 constitutively growth L. pneumophila macrophage- <br /> the total SOD activity (data shown).                                       like HL-60 cells (Fig. 6). The levels ,-galactosidase CFU <br />    This increased FeSOD activity correlated increased                     comparable measured strain JW1028 grown <br /> resistance paraquat (Fig. 4B). Similar results obtained                culture (data shown). <br /> with cells grown saturation plates liquid culture.                    In contrast expression sodB growth L. <br /> The zone growth inhibition strain JR32 pAB102                      pneumophila culture human macrophages, the <br /> (AB1778) 15.8 mm diameter, smaller                        expression sodB influenced charcoal growth <br /> 23.8-mm-diameter zone observed strain JR32 containing                      ABCYE agar plates. Charcoal frequently added agar <br /> pBC SK(+) (AB1782). Similar pronounced effects                         media L. pneumophila, presumably diminish concen- <br /> were observed redox cycling compounds like plum-                     tration toxic substances scavenge free radicals. A two- <br /> bagin menadione (data shown), demonstrating                        threefold decrease sodB expression observed for <br /> additional copies sodB increase FeSOD activity confer                   bacteria grown presence charcoal (Table 5). A fusion <br /> greater resistance paraquat redox cycling agents.                  lacZ pig, L. pneumophila gene required pigmen- <br /> No difference observed bacteria exposed H202 (data                   tation, resulted change gene expression these <br /> not shown).                                                                     conditions. Expression E. coli lacZ gene tac <br />    Aconitase E. coli 4Fe-4S enzymes                     promoter slightly increased presence charcoal. <br /> activities inversely related SOD activity are <br /> rapidly inactivated O2 (19). The gene encoding L.                                                DISCUSSION <br /> pneumophila aconitase (acn) highly <span id='am-239' about='Thesaurus:Homologous_Gene' typeof='owl:Thing'>homologous gene</span> <br /> encoding E. coli aconitase (29), likely L.                    To examine role FeSOD oxidative stress, <br /> pneumophila aconitase sensitive 02. To determine                      L. pneumophila sodB gene isolated mutagenized <br /> higher levels FeSOD resulted increased aconitase activity,               intention generating sodB mutant. To generate a <br /> measurements strains containing single copy                    sodB mutant L. pneumophila, sodB::lacZ gene fusion on <br />  (AB1782) multiple copies (AB1778) sodB gene. No                     pJW26 transferred allelic exchange vector <br />  VOL. 176, 1994                                                                              L. PNEUMOPHILA sodB GENE IS ESSENTIAL                                 3797 <br />  <br />  <br />        A                                                                                                                                              .10 7 <br />  <br />                                                                       10            e <br />                                                                                              15 - <br />                                                      - <br />                                                          e---o <br />  v <br />        80- <br />                                                                       S..) <br />                                                                       :   U-S <br />                                                                                     Q <br />                                                                                                                                                                     FX <br />                                                                                                                                                                     a <br />  En                                                                                i <br />                                                                              C. <br />  <br />                                                                                              10 e <br />                                                                                                                                                            ~i66Im0 <br />                                                                                                                                                          -10 <br />  U     60-                                                                                                                                                       C <br />                                                                                   as <br />  .w <br />  0)                                                                          Ln.    i <br />  C <br />        40- <br />                                                                              0 <br />                                                                                                5.-                                                                  0 <br />  <br />        20- <br />                                                                              _-     c <br />                                                                                                      \;u. <br />                                                                                                                                                          In   5 <br />                                                                       0.01                                6         10           14        18       22 <br />                 O        4       8          12           16      20 <br />                                                                                                                  Time (hour) <br />                                  Time (hour) <br />                                                                                       FIG. 6. L. pneumophila sodB expression macrophage infec- <br />                                                                                    tion. Monolayers containing 1.5 x 106 differentiated HL-60 cells were <br />        B                                                                           infected one-to-one ratio L. pneumophila JW1028 (pJW26 in <br />                                                                                    JR32). Units ,3-galactosidase activity CFU bacterial CFU/ <br />                                                                                    determined period 20 h described text. <br />                                                                                    Data plotted mean independent experiments. One <br />                                                                                    <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> P-galactosidase activity defined 1 x 10&apos; mol ONPG <br />                                                                                    hydrolyzed min. <br />       400   - <br />  <br />  <br />  <br />       300 -                                                                        AB1717, isolated means, shown to <br /> -0 <br />                                                                                    contain mutant sodB::lacZ gene chromosomes <br />  6                                                                                 plasmid-borne copy wild-type sodB+ gene. In no <br />  .: <br />       200   -                                                                      instance possible lose plasmid copy sodB+ from <br />                                                                                    strains AB1716 AB1717 strains maintain the <br />                                                                                    mutated copy genomes. In 14 cases the <br />       100   -                        --*--4                                        plasmid copy sodB+ lost strains (Table 4), a <br />                                                                                    wild-type copy sodB gene replaced mutant <br />                                                                                    chromosomal copy, presumably homologous recombina- <br />         n.          I1   I   I         I     I   I <br />                                                      - <br />                                                          I                         tion. These results support idea sodB an <br />                 0        4       8          12           16                        essential gene L. pneumophila. <br />                                  Time (hour)                                          Mutants E. coli (8) Streptococcus mutans (34) lacking <br />    FIG. 5. Expression L. pneumophila sodB gene growth                SOD viable. E. coli contains cytoplasmic SODs, an <br /> in ABYE. (A) ,-Galactosidase activity measured chromo-                    FeSOD (60) MnSOD (24). S. mutans contains single <br /> somal copy sodB::lacZ strain AB1688. Every 2 h, bacterial growth             cytoplasmic MnSOD (34). There possible explana- <br /> was measured reading A600 (open circles) ,3-galactosidase                   tions viability Sod- mutant species. <br /> activity (closed circles) measured sodB::lacZ described             Alternative antioxidants compensate absence of <br /> the text. (B) ,B-Galactosidase activity measured plasmid-                 SOD oxyradical scavenger, targets ROIs less <br /> borne (pJW26) copy sodB::lacZ strain LW1028. Open squares                    sensitive L. pneumophila, generation of <br /> denote bacterial growth measured Awo, closed squares repre- <br /> sent ,B-galactosidase activity measured sodB::lacZ.                           oxyradicals attenuated SOD lacking. It is <br />  <br />                                                                                          TABLE 5. Growth medium-dePendent sodB gene expression <br /> pLAW344 introduced wild-type strain JR32. Re- <br /> peated attempts generate sodB mutant failed, suggesting                                                         Units f3-galactosidase <br /> that sodB gene essential viability.                                    L. pneumophila                            activita <br />    If sodB essential, inability mutate chromosomal                          stri <br />                                                                                           strain      Genotype                  &apos;ii~            ABYE/ABCYE6 <br />                                                                                                                       ABYE               ABCYE <br /> copy sodB suggested single functional copy sodB <br /> exists L. pneumophila genome. This verified                            AB1688      sodB::lacZ 17.8 + 1.62 6.79 + 0.43                   2.6 <br /> Southern blot analysis genomic DNA prepared wild-                                AB1156      pig::lacZ 35.53 &#194;&#177; 0.74 29.35 &#194;&#177; 1.84                  1.1 <br /> type strain JR32, single DNA fragment hybridized                             AB1267      lacZ+      7.13 &#194;&#177; 0.43 11.78 &#194;&#177; 0.90                  0.6 <br /> sodB sequences digestion variety different                            a <br />                                                                                         L. pneumophila strains grown 3 days ABYE ABCYE agar <br /> restriction enzymes (data shown).                                              media, scraped 1 x M63 salts, assayed immediately ,-galactosidase <br />    It possible replace chromosomal copy sodB                      activity. Measurements performed triplicate, average and <br />                                                                                    standard error mean shown. <br /> mutant sodB bacteria provided second                            b Ratio ,3-galactosidase activity measured L. pneumophila strains grown <br /> wild-type copy sodB+. Two mutant strains, AB1716                            ABYE medium versus grown ABCYE medium. <br />  3798     SADOSKY ET AL.                                                                                                      J. BAcrERIOL. <br />  <br /> interesting L. pneumophila obligate aerobe E.      example, subject posttranslational control activity <br /> coli S. mutans able grow anaerobically          relating availability manganese iron (55). <br /> avoid formation oxygen-derived free radicals. <br />    In contrast E. coli S. mutans, L. pneumophila contains                          ACKNOWLEDGMENTS <br /> a periplasmic CuZnSOD addition cytoplasmic SOD.               We thank K. Derbyshire gift Tn903dIIlacZ contained on <br /> The inability construct sodB mutant strain L. pneumo-       pKD368. We thank members Shuman laboratory their <br /> phila indicates periplasmic CuZnSOD unable           helpful discussions suggestions, grateful to <br /> compensate absence cytoplasmic FeSOD. This               Carmen Rodriquez providing clean working conditions. <br /> suggests superoxide pools generated periplasm         This work supported NIAID National Research Service award <br /> cytoplasm exchangeable. This consistent         AI-08304 A.B.S., National Institutes Health grant AI-23549 to <br /> known impermeability inner membrane superoxide            H.A.S., National Science Foundation grant MCB-9220055 to <br /> and implies L. pneumophila SODs serve function-        H.M.S. H.A.S. recipient Faculty Research Award (FRA- <br />                                                                     357) American Cancer Society. <br /> ally discrete roles. <br />    Although possible construct sodB mutant,                                   REFERENCES <br /> were able demonstrate role FeSOD protection              1. Asada, K., S. Kanematsu, S. Okaka, T. Hayakawa. 1980. <br /> against ROIs comparing wild-type strain strain            Phylogenetic distribution types superoxide dismutase in <br /> harboring multiple copies plasmid-encoded sodB. The strain           organisms cell organelles, p. 136-153. In J. V. Bannister and <br /> with multiple copies sodB showed increased FeSOD activity            H. A. Hill (ed.), Chemical biochemical aspects superoxide <br /> and increased resistance redox cycling compounds                 superoxide dismutase. Elsevier, North-Holland, New York. <br /> paraquat, plumbagin, menadione.                                  2. Beaman, L., B. L. Beaman. 1984. The role oxygen its <br />                                                                         derivatives microbial pathogenesis host defense. Annu. <br />    Using sodB::lacZ fusion, demonstrated                Rev. Microbiol. 38:27-48. <br /> expression L. pneumophila sodB, like E. coli sodB         3. Beck, B. L., L. B. Tabatabai, J. E. Mayfield. 1990. A protein <br /> (11, 52), largely constitutive. The E. coli cytoplasmic Mn-          isolated Brucella abortus Cu-Zn superoxide dismutase. <br /> SOD induced variety oxidative stresses (11,        Biochemistry 29:372-376. <br /> 31, 52-54). Although FeSOD L. pneumophila         4. Bellinger-Kawahara, C. G., M. A. Horwitz. 1990. Complement <br /> cytoplasmic SOD present, induced like E. coli Mn-             component C3 fixes selectively major outer membrane <br /> SOD. No change expression seen different degrees            protein (MOMP) Legionella pneumophila mediates phago- <br />                                                                         cytosis liposome-MOMP complexes human monocytes. J. <br /> of aeration presence various oxidants redox             Exp. Med. 172:1201-1210. <br /> cycling compounds. In addition, manipulation iron concen-         5. Bey, R. F., R. C. Johnson. 1978. Protein-free low-protein <br /> tration medium failed produce significant change         media cultivation Leptospira. Infect. Immun. 19:562-569. <br /> L. pneumophila sodB expression. Finally, expression did          6. Beyer, W., J. Imlay, I. Fridovich. 1991. Superoxide dismutases. <br /> change growth exponential stationary phase.              Prog. Nucleic Acid Res. Mol. Biol. 40:221-253. <br />    The constitutivity sodB expression suggests basal     7. Byrd, T. F., M. A. Horwitz. 1987. Intracellular multiplication <br /> level gene expression sufficient provide protection            Legionella pneumophila human monocytes iron-dependent <br /> against superoxide generated normal aerobic metabo-              capacity activated monocytes inhibit intracellular <br />                                                                         multiplication reversed iron-transferrin. Clin. Res. 35:613A. <br /> lism. It argued basal level FeSOD         8. Carlioz, A., D. Touati. 1986. Isolation superoxide dismutase <br /> high induction necessary oxidative stress.            mutants Escherichia coli: superoxide dismutase necessary for <br /> However, unlikely, total SOD activity         aerobic life? EMBO J. 5:623-630. <br /> L. pneumophila (about 150 units mg protein) compa-         9. Collins, S. J., F. W. Ruscetti, R. E. Gallagher, R C. Gallo. <br /> rable aerobically grown E. coli.                              1978. Terminal differentiation human promyelocytic leukemia <br />    A decrease sodB expression agar medium               cells induced dimethyl sulfoxide polar compounds. <br /> containing charcoal. Charcoal proposed adsor-            Proc. Natl. Acad. Sci. USA 75:2458-2462. <br />                                                                     10. Farr, S. B., R. D. D&apos;Ari, D. Touati. 1986. Oxygen-dependent <br /> bent toxic substances, including free radicals (22, 37)          mutagenesis Escherichia coli lacking superoxide dismutase. <br /> fatty acids (5, 33, 39). It charcoal decreases        Proc. Natl. Acad. Sci. USA 83:8268-8272. <br /> superoxide concentration L. pneumophila cytoplasm,           11. Fee, J. A. 1991. Regulation sod genes Escherichia coli: <br />  diminishing requirement FeSOD decreasing sodB              relevance superoxide dismutase function. Mol. Microbiol. <br />  expression. The mechanism occur                 5:2599-2610. <br />  unclear.                                                           12. Feeley, J. C., R J. Gibson, G. W. Gorman, N. C. Langford, J. K <br />    Similar L. pneumophila growth culture, growth               Rasheed, D. C. Mackel, W. B. Baine. 1979. Charcoal-yeast <br /> macrophage-like HL-60 cells occurred significant                extract agar: primary isolation medium Legionella pneumo- <br /> changes sodB expression. Although L. pneumophila enters              phila. J. Clin. Microbiol. 10:437-441. <br />  host cells triggering respiratory-burst production     13. Feinberg, A. P., B. Vogelstein. 1983. A technique radiola- <br />                                                                         belling DNA restriction endonuclease fragments high specific <br /> of oxyradicals, bacterium likely encounter ROIs               activity. Anal. Biochem. 132:6-13. <br /> during growth human macrophages. It possible         14. Franzon, V. L., J. Arondel, P. J. Sansonetti. 1990. Contribu- <br />  extracellular ROIs scavenged periplasmic              tion superoxide dismutase catalase activities Shigella <br />  CuZnSOD. In contrast, FeSOD probably serves decom-              flexneri pathogenesis. Infect. Immun. 58:529-535. <br />  pose intracellular, cytoplasmic ROIs generated aerobic      15. Fridovich, I. 1978. The biology oxygen radicals. Science 201: <br />  metabolism. The lack induction sodB macrophage            875-880. <br />  infection consistent proposal.                        16. Fridovich, I. 1983. Superoxide radical: endogenous toxicant. <br />     Finally, noted results demon-        Annu. Rev. Pharmacol. Toxicol. 23:239-257. <br />                                                                     17. Friedman, A. M., S. R. Long, S. E. Brown, W. J. Buikemma, and <br />  strate constitutivity sodB expression, control        F. M. Ausubel. 1982. Construction broad host-range cosmid <br />  FeSOD activity posttranslationally. Posttranslational control        cloning vector use genetic analysis Rhizobium <br />  enzyme function particularly feasible metalloen-            mutants. Gene 18:289-296. <br />  zyme, conversion apo- holoenzyme        18. Gardner, P. R, I. Fridovich. 1991. Superoxide sensitivity of <br />  rate-limiting regulatory step. The MnSOD E. coli,             Eschenichia coli aconitase. J. Biol. Chem. 266:19323-19333. <br />  VOL. 176, 1994                                                                     L. PNEUMOPHILA sodB GENE IS ESSENTIAL                      3799 <br />  <br /> 19. Gardner, P. R., I. Fridovich. 1992. Inactivation-reactivation    41. Puget, K., A. M. Michelson. 1974. Isolation new copper- <br />     aconitase Escherichia coli. J. Biol. Chem. 267:8757-8763.                containing superoxide dismutase bacteriocuprein. Biochem. Bio- <br /> 20. Halliwell, B., J. M. C. Gutteridge. 1984. Oxygen toxicity, oxygen       phys. Res. Commun. 58:830-838. <br />     radicals, transition metals disease. Biochem. J. 219:1-14.          42. Sadosky, A. B., L. A. Wiater, H. A. Shuman. 1993. Identifica- <br /> 21. Hassett, D. J., M. S. Cohen. 1989. Bacterial adaptation              tion Legionella pneumophila genes required growth within <br />     oxidative stress: implications pathogenesis interaction        killing human macrophages. Infect. Immun. 61:5361-5373. <br />     phagocytic cells. FASEB J. 3:2574-2582.                                 43. Salin, M. L., S. M. Bridges. 1980. Isolation iron- <br /> 22. Hoffman, P. S., L. Pine, S. Bell. 1983. Production superox-          containing superoxide dismutase eucaryote, Brassica <br />      ide hydrogen peroxide medium used culture Legionella             campestris. Arch. Biochem. Biophys. 201:369-374. <br />     pneumophila: catalytic decomposition charcoal. Appl. Environ.        44. Short, J. M., J. M. Fernandez, J. A. Sorge, W. D. Huse. 1988. <br />      Microbiol. 45:784-791.                                                     XZAP: bacteriophage X expression vector vivo excision <br /> 23. Horwitz, M. A., S. C. Silverstein. 1983. Intracellular multipli-        properties. Nucleic Acids Res. 16:7583-7600. <br />      cation Legionnaires&apos; disease bacteria (Legionella pneumophila)      45. Silhavy, T. J., M. L. Berman, L. W. Enquist. 1984. Experi- <br />      human monocytes reversibly inhibited erythromycin             ments gene fusions. Cold Spring Harbor Laboratory, Cold <br />      rifampin. J. Clin. Invest. 71:15-26.                                       Spring Harbor, N.Y. <br /> 24. Keele, B. B. J., J. M. McCord, I. Fridovich. 1970. Superoxide       46. Simon, R., U. Priefer, A. Puhler. 1983. A broad host range <br />      dismutase E. coli B: new manganese containing enzyme. J.            mobilization vivo genetic engineering: transposon <br />      Biol. Chem. 245:6176-6181.                                                 mutagenesis gram negative bacteria. Bio/Technology 1:784-791. <br /> 25. Kroll, J. S., P. R. Langford, B. M. Loynds. 1991. Copper-zinc       47. Southern, E. M. 1975. Detection specific sequences among <br />      superoxide dismutase Haemophilus influenzae H. parainflu-           DNA fragments separated gel electrophoresis. J. Mol. Biol. <br />     enzae. J. Bacteriol. 173:7449-7457.                                         98:503-517. <br /> 26. Maniatis, T., E. F. Fritsch, J. Sambrook. 1982. Molecular          <br /> </body></html>