<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL OF BACTERIOLOGY, May 2008, p. 3444&#226;&#8364;&#8220;3455                                                                            Vol. 190, No. 10 <br /> 0021-9193/08/$08.00&#226;&#171;&#185;0 doi:10.1128/JB.00141-08 <br /> Copyright &#194;&#169; 2008, American Society Microbiology. All Rights Reserved. <br />  <br />  <br />  <br />          An Ortholog OxyR Legionella pneumophila Is Expressed <br />             Postexponentially Negatively Regulates Alkyl <br />                  Hydroperoxide Reductase (ahpC2D) Operon&#228;&#338;&#164; <br />                            Jason J. LeBlanc,1,2 Ann Karen C. Brassinga,4 Fanny Ewann,4 <br />                                  Ross J. Davidson,1,2,3 Paul S. Hoffman2,3,4,5* <br />               Department Pathology Laboratory Medicine, Queen Elizabeth II Health Sciences Center, Halifax, <br />                  Nova Scotia, Canada1; Department Microbiology Immunology2 Medicine,3 Faculty of <br />                   Medicine, Dalhousie University, Halifax, Nova Scotia B3H 4H7, Canada; Department of <br />                      Medicine, Division Infectious Diseases International Health4 Department of <br />                                     Microbiology,5 University Virginia School Medicine, <br />                                                Charlottesville, Virginia 22908-1340 <br />                                            Received 28 January 2008/Accepted 11 March 2008 <br />  <br />             Legionella pneumophila expresses peroxide-scavenging alkyl hydroperoxide reductase systems (AhpC1 and <br />          AhpC2D) expressed differentially bacterial growth cycle. Functional loss postexponentially <br />          expressed AhpC1 compensated increased expression exponentially expressed AhpC2D <br />          system. In study, used acrylamide capture DNA-bound complexes (ACDC) technique mass <br />          spectrometry identify proteins bind promoter region ahpC2D operon. The major protein <br />          captured ortholog OxyR (OxyRLp). Genetic studies indicated oxyRLp essential gene expressed <br />          postexponentially partially complemented Escherichia coli oxyR mutant (GS077). Gel shift assays <br />          confirmed specific binding OxyRLp ahpC2D promoter sequences, promoters ahpC1 oxyRLp; <br />          however, OxyRLp weakly bound E. coli OxyR-regulated promoters (katG, oxyR, ahpCF). DNase I protection <br />          studies showed OxyRLp binding motif spanned promoter transcriptional start sequences ahpC2 <br />          protected region unchanged treatments reducing agents hydrogen peroxide (H2O2). <br />          Moreover, OxyRLp (pBADLpoxyR)-mediated repression ahpC2-gfp reporter construct E. coli GS077 (the <br />          oxyR mutant) reversed H2O2 challenge. Alignments OxyR proteins revealed amino acid <br />          substitutions predicted ablate thiol oxidation conformational changes required activation. We suggest these <br />          mutations locked OxyRLp active DNA-binding conformation, permitted divergence of <br />          function regulator oxidative stress cell cycle regulator, controlling gene expression during <br />          postexponential differentiation. <br />  <br />  <br />    Legionella pneumophila aquatic moist soil                phagocytes (3, 6, 8, 19, 27) important for <br /> environments intracellular parasite protozoa (12, 14,              environmental persistence (25). <br /> 15). Legionnaires&#226;&#8364;&#8482; disease results aerosol transmission                We previously characterized peroxide-scavenging alkyl <br /> bacteria environment susceptible humans. In nat-                hydroperoxide reductase systems L. pneumophila: AhpC1, <br /> ural hosts (and mammalian cell lines), L. pneumophila                  similar AhpC-thioredoxin Helicobacter pylori <br /> displays dimorphic lifestyle vegetative replicating              (30, 49), AhpC2D, similar AhpC AhpD system <br /> bacilli differentiate metabolically dormant cyst-like plank-           mycobacteria (30, 42). These studies demonstrated <br /> tonic forms (17). Analogous small-cell variants Cox-              systems contributed efficient scavenging H2O2 <br /> iella burnetii elementary bodies Chlamydia spp.,            organic peroxides re- <br /> highly resilient cyst-like forms L. pneumophila enable               quired viability. Moreover, genes inversely ex- <br /> organism survive extracellularly extended periods                 pressed growth cycle; ahpC2D induced early in <br /> time highly infectious state. The cyst form likely contrib-            exponential phase repressed postexponentially, concomi- <br /> utes noted resilience L. pneumophila actions            tant induction ahpC1. Finally, compensatory increase <br /> oxidizing agents (chlorine), chloramines, biocides, used <span id='am-9' about='protege:TO' typeof='owl:Thing'>to</span> <br />                                                                             expression ahpC2D observed ahpC1 mutant, <br /> control infection man-made aquatic systems (17). Several <br />                                                                             suggesting ahpC2D operon upregulated in <br /> lines evidence suggest antioxidant enzymes play crucial <br />                                                                             response changes cellular redox status increases in <br /> roles early stages infection protozoa human <br />                                                                             intracellular peroxides. However, previous studies anti- <br />                                                                             oxidant defense genes L. pneumophila (sodB, sodC, katA, <br />                                                                             katB, ahpC1, ahpC2D) concluded these <br />   * Corresponding author. Mailing address: Division Infectious           genes activated response oxidative stress (4, 5, 30, <br /> Diseases International Health, Room 2146, MR-4 Bldg., 409 Lane          39, 43). In contrast, studies revealed these <br /> Road, University Virginia Health Systems, Charlottesville, VA <br />                                                                             antioxidant defense genes, exception sodB (consti- <br /> 22908-1340. Phone: (434) 924-2893. Fax: (434) 924-0075. E-mail: <br /> psh2n@virginia.edu.                                                         tutive), growth stage regulated. <br />   &#228;&#338;&#164; <br />     Published ahead print 21 March 2008.                                 The emerging <span id='am-89' about='xsp:pattern' typeof='owl:Thing'>pattern</span> growth stage-regulated antioxidant <br />  <br />                                                                      3444 <br />  VOL. 190, 2008                                                               OxyRLp REGULATION OF ahpC2D IN LEGIONELLA                                 3445 <br />  <br />                                           TABLE 1. Strains, genotypes, plasmids used study <br /> Bacterial strain, genotype, <br />                                                                  Relevant property(ies)                                                 Source reference <br />         plasmid <br />  <br /> E. coli <br />   DH5&#226;?&#163;                        F&#226;&#171;&#186; &#226;?&#190;80dlacZ&#226;&#338;&#172;M15 &#226;&#338;&#172;(lacZYA-argF)U169 endA1 recA1 hsdR17 deoR thi-1 supE44 gyrA96 relA1                       Clontech <br />   BL21 Codon Plus             F&#226;&#171;&#186; ompT hsdS (rB&#226;&#171;&#186; mB&#226;&#171;&#186;) dcm&#226;&#171;&#185; Tetr gal&#226;?&#173; (DE3) endA Hte &#229;&#8230;&#179;argU ileY leuW Camr&#229;&#8230;&#180;                                 Stratagene <br />   MG1655                      K-12, F&#226;&#171;&#186; &#226;?&#173;&#226;&#171;&#186;                                                                                                ATCC 47076 <br />   GS077                       MG1655-derived oxyR::kan                                                                                   51 <br />  <br /> L. pneumophila <br />   Lp02                        Philadelphia-1 derivative, rpsL hsdR thyA (Smr) mutant                                                     7 <br />   ahpC1::kan                  Lp02 ahpC1::kan mutant                                                                                     30 <br />   ahpC2D::kan                 Lp02 ahpC2D::kan mutant                                                                                    30 <br />   rpoS::kan                   Lp02 strain MB 379::kan                                                                                    2 <br />   letA::kan                   Lp02 strain MB 414::kan                                                                                    22 <br />   himA::gm himB::kan          Lp02 integration host factor double mutant (lacking &#226;?&#163; &#226;?&#164; subunits); Kanr Gmr                       This laboratory <br />  <br />  <br /> Plasmids <br />   pET29b                      IPTG-inducible T7 expression vector                                                                        Novagen <br />   petLpoxyR                   pET29b-derived overexpression vector His6-OxyRLp                                                       This study <br />   pBH6119                     RSF1010 ori, promoterless gfpmut3 td&#226;&#338;&#172;i (Ampr)                                                              21 <br />   pC2gfp                      PahpC2 region (&#226;&#171;&#186;199 &#226;&#171;&#185;27 bp upstream ahpC2) cloned upstream gfpmut3 pBH6119                     30 <br />   pC2(&#226;&#171;&#186;179/&#226;&#171;&#185;27)gfp            PahpC2 region (&#226;&#171;&#186;179 &#226;&#171;&#185;27 bp upstream ahpC2) cloned upstream gfpmut3 pBH6119                     This study <br />   pC2(&#226;&#171;&#186;156/&#226;&#171;&#185;27)gfp            PahpC2 region (&#226;&#171;&#186;156 &#226;&#171;&#185;27 bp upstream ahpC2) cloned upstream gfpmut3 pBH6119                     This study <br />   pC2(&#226;&#171;&#186;132/&#226;&#171;&#185;27)gfp            PahpC2 region (&#226;&#171;&#186;132 &#226;&#171;&#185;27 bp upstream ahpC2) cloned upstream gfpmut3 pBH6119                     This study <br />   pC2(&#226;&#171;&#186;109/&#226;&#171;&#185;27)gfp            PahpC2 region (&#226;&#171;&#186;109 &#226;&#171;&#185;27 bp upstream ahpC2) cloned upstream gfpmut3 pBH6119                     This study <br />   pC2(&#226;&#171;&#186;91/&#226;&#171;&#185;27)gfp             PahpC2 region (&#226;&#171;&#186;91 &#226;&#171;&#185;27 bp upstream ahpC2) cloned upstream gfpmut3 pBH6119                      This study <br />   pC2(&#226;&#171;&#186;66/&#226;&#171;&#185;27)gfp             PahpC2 region (&#226;&#171;&#186;66 &#226;&#171;&#185;27 bp upstream ahpC2) cloned upstream gfpmut3 pBH6119                      This study <br />   poxyRgfp                    oxyR promoter region cloned upstream gfpmut3 pBH6119                                                 This study <br />   pBAD22                      Arabinose-inducible expression vector (Ampr)                                                               18 <br />   pBADEcoxyR                  pBAD22-derived expression vector E. coli oxyR                                                          This study <br />   pBADLpoxyR                  pBAD22-derived expression vector L. pneumophila oxyR                                                   This study <br />  <br />  <br />  <br />  <br /> defense genes L. pneumophila postexpo-                weak binding OxyR-regulated promoters E. coli gel <br /> nential expression transmission (virulence) traits consis-           shift experiments, protein longer redox active and <br /> tent dimorphic lifestyle organism (17). In sup-           repression ahpC2D reversed H2O2. Fi- <br /> port view, microarray analyses comparing sessile               nally, apparent essentiality oxyRLp post- <br /> planktonic cells L. pneumophila biofilm conditions                exponential expression, concomitant repression of <br /> revealed ahpC2 ahpD highly induced genes                ahpC2D, suggests regulatory roles unrelated oxida- <br /> in sessile cells (25). Related studies similarly shown           tive stress. <br /> expression ahpC2D operon increases significantly during <br /> intracellular growth protozoa (8). Despite findings,                                      MATERIALS AND METHODS <br /> little known regulatory systems controlling ahpC2D                  Bacterial strains growth conditions. Bacterial strains plasmids used in <br /> expression antioxidant defense genes L. pneumo-                study listed Table 1 oligonucleotides Table 2. L. pneumophila <br /> phila. With exception OxyR homolog (described                    strains grown aerobically 37&#194;&#176;C buffered charcoal yeast extract agar or <br />                                                                            buffered yeast extract (BYE) broth, supplemented &#226;?&#163;-ketoglutaric acid (1 <br /> herein), typical regulators oxidative stress defenses (such as <br />                                                                            mg/ml), ferric pyrophosphate (250 &#226;?&#174;g/ml), L-cysteine (40 &#226;?&#174;g/ml), thymidine (100 <br /> SoxRS, OhrR, PerR) absent L. pneumophila                    &#226;?&#174;g/ml), antibiotics required. E. coli strains DH5&#226;?&#163;, MG1655, and <br /> genome. Growth phase-dependent regulation genes associ-                 GS077 grown 37&#194;&#176;C Luria-Bertani (LB) agar LB broth supple- <br /> ated virulence transmission traits attributed            mented appropriate antibiotics. For studies, E. coli GS077 was <br /> in global regulators, stationary-phase sigma           grown 37&#194;&#176;C anaerobic conditions using BD GasPak EZ (Becton <br />                                                                            Dickinson, Oakville, Ontario) AnaeroGen anaerobic atmosphere generation <br /> factor (RpoS) (2, 20), carbon storage regulator (CsrA) (13,            (Oxoid, Ltd., Nepean, Ontario). Antibiotics (Sigma-Aldrich Canada Ltd., <br /> 33), LetAS, CpxRA, PmrBA two-component reg-                    Oakville, Ontario) added media following concentrations, when <br /> ulatory systems (13, 16, 22, 32, 52); however, predicted               appropriate: streptomycin (100 &#226;?&#174;g/ml), kanamycin (40 &#226;?&#174;g/ml), chloramphenicol <br /> hierarchal regulatory cascade incomplete,             (20 &#226;?&#174;g/ml), ampicillin (100 &#226;?&#174;g/ml). All strains stored &#226;&#171;&#186;85&#194;&#176;C in <br />                                                                            <span id='am-3' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> broth containing 10% dimethyl sulfoxide. <br /> regulators linked oxidative stress defenses. <br />                                                                               Standard molecular techniques. Preparation DNA, restriction enzyme di- <br />    Here report use acrylamide capture DNA-                 gestions, molecular techniques described (40). <br /> bound complexes (ACDC) technique (35) tandem mass                      Plasmid isolation performed QIAprep spin miniprep kit (Qiagen <br /> spectrometry ahpC2D promoter-bound proteins identify                 Inc., Mississauga, Ontario) using standard alkaline lysis method (40). Puri- <br /> a previously uncharacterized putative regulator oxidative               fication DNA fragments agarose gels subcloning carried with <br />                                                                            QIAquick gel purification kit (Qiagen). PCR performed conditions <br /> stress L. pneumophila, designated OxyRLp. These studies                 described LeBlanc et al. (30) using oligonucleotides synthesized Integrated <br /> will oxyRLp able partly complement               DNA Technologies, Inc. (Coralville, IA). DNA protein sequencing was <br /> oxyR mutant Escherichia coli purified OxyRLp exhibited              carried DalGen Microbial Genetics Center (Halifax, Nova Scotia, <br />  3446       LEBLANC ET AL.                                                                                                                            J. BACTERIOL. <br />  <br />  <br />         TABLE 2. Oligonucleotide primers used study                        addition 1.25 &#226;?&#174;l 5% ammonium persulfate 1.25 &#226;?&#174;l diluted <br />                                                                                    N,N,N&#226;&#172;&#732;,N&#226;&#172;&#732;&#226;&#172;&#732;-tetramethylethylenediamine (1:30 buffer D). The reactions were <br />        Primer                                  Sequence (5&#226;&#172;&#732;&#226;&#8364;&#8220;3&#226;&#172;&#732;) <br />                                                                                    immediately loaded wells precast nondenaturing 5% polyacrylamide <br /> Ac-FC2PROM ......................GGATCCACCAATATTCCTCCAC                            gel (0.5&#226;&#171;&#187; Tris-borate-EDTA [TBE], Bio-Rad Laboratories). After polymeriza- <br />                                            AGGC                                    tion (approximately 5 min), electrophoresis performed 0.5&#226;&#171;&#187; TBE 1 h <br /> RC2PROM ............................GAATTCCTGGAAATTTGTTGCCT                        125 V remove nonspecific proteins. Each excised gel, <br />                                            ACCG                                    transferred microtube containing 100 &#226;?&#174;l 10% sodium dodecyl sulfate <br /> FECOXYRPBAD ................GGCGAATTCATTATGAATCGTGAT                               (SDS) sample buffer (NEB) 5% &#226;?&#164;-mercaptoethanol heated 3 min at <br />                                            CTTGAG                                  94&#194;&#176;C remove captured proteins. After brief centrifugation, 50-&#226;?&#174;l aliquots <br /> RECOXYRPBAD................GGCAAGCTTAAACCGCCTGTTTTA                                loaded precast 4 15% Tris-HCl polyacrylamide gel (Bio-Rad <br />                                            AAACTT                                  Laboratories) subjected SDS-polyacrylamide gel electrophoresis (SDS- <br /> FLPOXYRPBAD .................GGCGAATTCATTATGAATTTAAGA                              PAGE). After silver staining, proteins distinct no-DNA controls were <br />                                            GATTTACATTAT                            excised submitted mass spectrometry analysis (DalGen Microbial <br /> RLPOXYRPBAD ................GGCAAGCTTCTATGATAATTTGGA                               Genomics Center). <br />                                            TTGAACATT                                  Cloning overexpression OxyRLp E. coli GS077. For complementation <br /> FBLA......................................TCCGTGTCGCCCTTATTCC                      E. coli strain GS077 (oxyR::kan) (generously donated Gisela Storz, <br /> RBLA .....................................AACTACGATACGGGAGGGC                      National Institutes Health, Bethesda, MD), E. coli strain K-12 oxyR (oxyREc) <br /> FLPOXYRPET.....................CATATGAATTTAAGAGATTTACA                             oxyRLp expressed using arabinose-inducible expression systems <br />                                            TTAT                                    pBAD22 (18). Amplicons generated PCRs using primer pairs <br /> RLPOXYRPET ....................CTCGAGTGCTAATTTGGATTGAAC                            FECOXYRPBAD RECOXYRPBAD oxyREc FLPOXYRPBAD and <br />                                            ATTTT                                   RLPOXYRPBAD oxyRLp, digested EcoRI HindIII subcloned <br /> FC1PROM.............................GAATTCGTGCCATTACTGAGCG                         pBAD22. The resulting constructs (pBADEcoxyR pBADLpoxyR) were <br />                                            ATTC                                    transformed E. coli DH5&#226;?&#163; verified PCR DNA sequencing. <br /> RC1PROM ............................GGATCCGAGTGCCTTTCATGTA                         Plasmids extracted transformed electrocompetent E. coli GS077. <br />                                            GAGC                                    Ampicillin-resistant GS077 transformants obtained anaerobically con- <br /> FLPOXYRPROM ................GAATTCGATGGCACAAAGAGT                                  firmed PCR analysis termed GS077 pBADEcoxyR GS077 pBADL- <br />                                            TGCA                                    poxyR. The pBAD22 vector transformed wild-type MG1655 and <br /> RLPOXYRPROM ...............CTGGGATCCTGCTTTACATCTGC                                 GS077 strains, verified PCR analysis using primers ampicillin resis- <br />                                            CAGG                                    tance cassette FBLA RBLA, termed MG1655 pBAD22 GS077 <br /> FECAHPCPROM ................GGCACTGAAGATACCAAAGG                                   pBAD22, respectively. <br /> RECAHPCPROM................TCGATGAGATGTAAGGTAACC                                      Disk diffusion assays. Strains grown anaerobic conditions mid- <br /> FECKATGPROM................CGAAATGAGGGCGGGAAAAT                                    exponential phase (optical density 620 nm [OD620] 0.4), expression of <br /> RECKATGPROM ...............CGATACACAGCGTTAGAGAG                                    oxyREc oxyRLp respective pBAD22-derived plasmids induced with <br /> FECOXYRPROM ...............GGTGCCGCTCCGTTTCTG                                      various concentrations arabinose (0, 0.02, 0.2%) repressed using 0.2% <br /> RECOXYRPROM...............GCTGGCTAACGTGGCAGG                                       glucose. Samples analyzed SDS-PAGE confirm expression of <br /> FC2PROM179.......................GGCGAATTCCCTTCTTTAAAAGTA                          OxyREc OxyRLp. Cells corresponding 0.1 OD620 inoculated 4 ml <br />                                            GTTTATAAAA                              prewarmed LB 0.75% agar, mixed inversion, poured LB <br /> FC2PROM156.......................GGCGAATTCAAACATAAAAGCCA                           media proper antibiotics presence arabinose glucose. Once <br />                                            AATATTTTGC                              solidified, 10 &#226;?&#174;l 30% H2O2 placed sterile one-fourth-inch antibiotic <br /> FC2PROM132.......................GGCGAATTCAAAATGGATTAATA                           disks center plate. Following overnight incubation 37&#194;&#176;C under <br />                                            ATTCTTATATTG                            anaerobic conditions, diameters zones inhibition measured <br /> FC2PROM109.......................GGCGAATTCTTGATTGACTTTATT                          reported independent experiments. <br />                                            TATCGTCTT                                  Overproduction purification OxyRLp. The primer pair FLPOXYRPET <br /> FC2PROM91.........................GGCGAATTCCGTCTTTATAAAAAC                         RLPOXYRPET used generate amplicon containing NdeI and <br />                                            AATTGATTTT                              XhoI restriction sites, respectively. Following digestion, NdeI-XhoI fragment <br /> FC2PROM66.........................GGCGAATTCATTTATTAAATGATT                         subcloned corresponding sites pET29b, generating pETLpoxyR. <br />                                            TCACCTAAATT                             The in-frame fusion L. pneumophila oxyR C-terminal hexameric <br /> RGFP .....................................ACCATAACCGAAAGTAGTGAC                    histidine (His6) tag confirmed DNA sequencing. For overexpression, an <br />                                                                                    overnight culture E. coli BL21(DE3) Codon Plus harboring pETLpoxyR was <br />                                                                                    inoculated 1% (vol/vol) 200 ml fresh LB broth grown OD620 of <br />                                                                                    0.4. Isopropyl-&#226;?&#164;-D-thiogalactopyranoside added final concentration 1 <br /> Canada). Protein concentrations estimated using Bio-Rad protein assay       mM, incubation continued 60 min 37&#194;&#176;C. Bacteria were <br /> (Bio-Rad, Hercules, CA).                                                           harvested centrifugation (5,800 &#226;&#171;&#187; g 6 min), resuspended (5 ml/g [wet <br />    ACDC. To identify transcription factors bind promoter region     weight]) suspension buffer (20 mM Tris-HCl, pH 7.9; 500 mM NaCl; 5 mM <br /> ahpC2D (PahpC2), ACDC performed according modified version          imidazole; 1 mM PMSF), lysed ice sonication (six bursts 10 s at <br /> the methods described Nelson et al. (35). Briefly, PCRs performed using    200 W). Following clarification centrifugation (10,000 &#226;&#171;&#187; g 15 min, 4&#194;&#176;C), <br /> forward primers containing 5&#226;&#172;&#732; Acrydite (Ac) moiety, Ac-FC2PROM, un-            His6-OxyRLp purified nickel interaction chromatography previously <br /> modified reverse primer RC2PROM. Amplicons purified using MinElute          described (41). Samples taken step purification analyzed by <br /> gel extraction kit (Qiagen). Protein lysates obtained follows: bacteria    SDS-PAGE, purified protein fractions pooled, aliquoted, and <br /> were harvested (4,800 &#226;&#171;&#187; g, 6 min, 4&#194;&#176;C) 500-ml, 24-h culture L.           stored &#226;&#171;&#186;85&#194;&#176;C. A single band excised submitted mass spectrometry <br /> pneumophila. The pellet washed cold 50 mM Tris-HCl, pH 7.5,           analysis. Purified OxyRLp protein used study considered wholly <br /> resuspended buffer consisting 50 mM Tris-HCl, pH 7.5; 1 mM EDTA, pH          active. <br /> 8.0; 50 mM NaCl; 10% glycerol 0.1 mM phenylmethylsulfonyl fluoride               EMSA. For electrophoretic mobility shift assay (EMSA), amplicons spanning <br /> (PMSF); 1 mM dithiothreitol (DTT). Cells lysed using French press       promoter regions ahpC1, ahpC2, oxyR (PahpC1, PahpC2, PoxyR) were <br /> and subjected ultracentrifugation 100,000 &#226;&#171;&#187; g. Protein lysates ali-     generated PCR amplification using primers sets FC1PROM RC1PROM, <br /> quoted stored &#226;&#171;&#186;85&#194;&#176;C. Before use, protein lysates diluted buffer   FC2PROM RC2PROM, FLPOXYRPROM RLPOXYRPROM, <br /> consisting 100 mM Tris-HCl, pH 7.5, 10 mM MgCl2 treated 15          respectively. For binding assays E. coli ahpCF, katG, oxyREc promot- <br /> min room temperature 100 mM acetyl phosphate (phosphorylate             ers, primer pairs FECAHPCPROM RECAHPCPROM, FECKATGPROM <br /> DNA-binding proteins). Binding reactions ACDC assay performed         RECKATGPROM, FECOXYRPROM RECOXYRPROM were <br /> in 25-&#226;?&#174;l volume consisted 0 (no DNA control), 2, 4 8 &#226;?&#174;g         used generate respective amplicons. Fifty nanograms DNA probes was <br /> Ac-DNA, 1 &#226;?&#174;g protein lysate, 7.5 &#226;?&#174;l buffer D (20 mM HEPES, pH 7.8; 100          mixed various amounts His6-OxyRLp (0, 3.9, 7.8, 15.6, 31.3, 62.5, 125, 250, <br /> mM KCl; 10% glycerol; 1 mM DTT), 1 &#226;?&#174;g poly(dI-dC), 3.5 &#226;?&#174;l 30%           500 ng) 20-&#226;?&#174;l reaction mixture containing 20 mM Tris-HCl, pH 8.0; 5 <br /> acrylamide. Reactions incubated 15 min room temperature         mM MgCl; 20 mM KCl; 50 &#226;?&#174;g bovine serum albumin; 20% glycerol; 1 mM <br />  VOL. 190, 2008                                                                        OxyRLp REGULATION OF ahpC2D IN LEGIONELLA                                 3447 <br />  <br /> PMSF; 0 200 mM DTT (Promega, Madison, WI) oxidizing reducing           obtained. Then, glucose arabinose added final concentration of <br /> conditions, respectively. After 30 min incubation room temperature, pro-       0.02%, cultures incubated additional hour. In cases, strains <br /> tein-DNA complexes separated electrophoresis approximately 1 h        challenged 30 min 100 &#226;?&#174;M H2O2. Fluorometric detection <br /> 125 V 5% nondenaturing polyacrylamide gel 0.5&#226;&#171;&#187; TBE buffer. Gels         samples taken LB-grown bacteria performed described previously <br /> then soaked 10,000-fold diluted Sybr green I nucleic acid stain (Invitrogen, La   (30). Values expressed RFU OD620 represent triplicate values <br /> Jolla, CA) washed twice distilled water, DNA visualized using       obtained independent experiments. <br /> Typhoon 9410 (GE Healthcare) blue light excitation 488 nm and <br /> using 520-nm emission filter. Densitometry performed using ImageQuant <br /> version 5.2 (Molecular Dynamics, Sunnyvale, CA).                                                                      RESULTS <br />    Mapping PahpC2. An EMSA performed described using 500 ng <br /> of His6-OxyRLp 50 ng variously sized 5&#226;&#172;&#732;-end deletion DNA fragments               Acrylamide capture ahpC2D DNA-bound complexes. We <br /> generated PCR using common reverse primer, RC2PROM, different              employed ACDC technique (Fig. 1) screen transcrip- <br /> forward primers: FC2PROM, FC2PROM179, FC2PROM156, FC2PROM132,                        tion factors bound promoter region ahpC2D. The <br /> FC2PROM109, FC2PROM91, FC2PROM66. All promoter fragments were <br /> also subcloned pBH6119 electroporated L. pneumophila Lp02 and <br />                                                                                      principle method relies capture DNA-binding <br /> the ahpC1::kan mutant (30) evaluate promoter activity using green fluo-       proteins L. pneumophila lysates interaction with <br /> rescence protein (GFP) reporter assay.                                               ahpC2D promoter DNA sequences immobilized poly- <br />    DNase I footprinting. DNase I protection experiments performed using         acrylamide matrix covalent linkage 5&#226;&#172;&#732; Ac moiety. <br /> a 226-bp DNA fragment PahpC2 generated PCRs using primer pair                Proteins bind Ac-DNA removed by <br /> FC2PROM RC2PROM (containing EcoRI BamHI restriction sites, <br /> respectively). The digested DNA (EcoRI strand BamHI the <br />                                                                                      standard nondenaturing electrophoresis. Following SDS- <br /> bottom strand) treated calf intestinal phosphatase (New England        PAGE, silver staining, mass spectrometry analysis pro- <br /> BioLabs, Ipswich, MA), reactions purified using QIAquick gel ex-          teins captured wells native acrylamide gel, pro- <br /> traction kit (Qiagen). End-labeled DNA prepared using [&#226;?&#165;-32P]dATP            teins able bind target DNA sequences identified. <br /> Ready-To-Go T4 polynucleotide kinase (GE Healthcare) described the <br />                                                                                      Using increasing concentrations Ac-labeled ahpC2D pro- <br /> manufacturer purified QIAquick nucleotide removal kit (Qiagen). <br /> Binding reactions performed incubating approximately 20,000 cpm           moter DNA (Ac-PahpC2) ACDC assay (Fig. 1 inset), a <br /> labeled probe 10 &#226;?&#174;g OxyRLp protein assay buffer consisting 20       distinct protein captured immobilized Ac-DNA, <br /> mM Tris-HCl, pH 8.0; 10 mM MgCl2; 100 mM KCl; 1 mM CaCl2; 10% glycerol;              bands seen absence Ac-DNA. <br /> 0.05 mg/ml bovine serum albumin; 4 &#226;?&#174;g/ml poly(dI-dC); H2O2 (0 50 mM)          Peptides identified tandem mass spectroscopy analysis cor- <br /> for oxidizing 200 mM DTT reducing conditions. In cases peroxi- <br /> dation abolished DNA binding, replicates treated 200 mM DTT to <br />                                                                                      responded 181 296 amino acids (61% sequence coverage) <br /> restore DNA binding activity. After 20 min incubation, 0.5 1 U DNase        previously uncharacterized protein annotated hy- <br /> I (Sigma) added. The reaction continued 90 s 100 &#226;?&#174;l stop     drogen peroxide-inducible gene activator, OxyR, L. pneu- <br /> solution (200 mM EDTA, pH 8.0; 5 M ammonium acetate; 100 &#226;?&#174;g salmon            mophila Philadelphia-1. This protein, designated OxyRLp, <br /> sperm DNA) added. DNA ethanol precipitated following phenol-chlo- <br />                                                                                      shared 45 50% identity 63 69% similarity well- <br /> roform extraction. Dry pellets mixed formamide loading dye and <br /> loaded 6% sequencing gel. The DNA ladder performed cycle               characterized OxyR regulators Xanthomonas campestris, E. <br /> sequencing using &#226;?&#163;-35S-dATP <span id='am-6' about='rdfs:label' typeof='owl:Thing'>label</span> according SequiTherm Excel II            coli, Salmonella enterica serovar Typhimurium, Haemophilus <br /> DNA sequencing kit (Epicenter Technologies).                                         influenzae, C. burnetii BLAST analysis. <br />    Primer extension. Total RNA obtained exponentially grown L.                 The identification OxyRLp putative regulator of <br /> pneumophila automated Maxwell-16 protocol (Promega), primer ex- <br /> tension reactions performed follows. Briefly, 2 pmol [&#226;?&#165;-32P]ATP <br />                                                                                      ahpC2D unexpected, OxyR mediates oxidative <br /> end-labeled RC2PROM primer added 25 &#226;?&#174;g total RNA allowed               stress induction alkyl hydroperoxide reductase catalase <br /> anneal dropping temperature 80&#194;&#176;C 41&#194;&#176;C period 30 min.        genes organisms (1, 26, 31, 36). Moreover, those <br /> cDNA extension products generated 41&#194;&#176;C 1 h reaction mixture         organisms expressing ahpC2D operon, including close rel- <br /> containing 1 &#226;?&#174;l 200 U/&#226;?&#174;l SuperScript reverse transcriptase (Invitrogen) and <br />                                                                                      ative C. burnetii, oxyR usually located immediately upstream <br /> subsequently treated RNase A (Sigma) eliminate residual RNA. Samples <br /> were loaded 6% polyacrylamide urea sequencing gel              shares promoter regulatory sequences. However, the <br /> corresponding DNA sequence ladder obtained cycle sequencing using              oxyRLp gene located upstream ahpC2D, rather <br /> &#226;?&#163;-35S-dATP label appropriate primer accordance Sequi-           shares divergent promoter region gene annotated a <br /> Therm Excel II DNA sequencing kit protocol.                                          major facilitator transporter, downstream gene of <br />    GFP reporter assay. A GFP-transcriptional fusion PoxyR constructed <br />                                                                                      unknown function oriented opposite direction. Our at- <br /> using primer pair FLPOXYRPROM RLPOXYRPROM cloned the <br /> EcoRI BamHI sites pBH6119 (21). Plasmid constructs trans-          tempts knock oxyRLp allelic replacement drug <br /> formed E. coli DH5&#226;?&#163; strains, correct orientation determined          resistance cassette (kanamycin) methods previously used to <br /> PCR using respective combination FLPOXYRPROM RGFP primers.                obtain ahpC1 ahpC2D mutants (30) proved unsuccessful. <br /> The pBH6119 plasmid derivatives electroporated wild-type,           Since catalase supplementation media enabled recovery <br /> ahpC1, ahpC2D, rpoS, letA, himAB mutants L. pneumophila. Transformants <br /> capable growth thymidine verified respective plasmids by <br />                                                                                      oxyR mutants Pseudomonas aeruginosa (23), buffered <br /> PCR analysis using primer pairs FLPOXYRPROM RLPOXYRPROM                       charcoal yeast extract agar supplemented catalase <br /> FBLA RBLA vector controls. Fluorometric detection               (10,000 units); however, oxyRLp mutants recovered. <br /> samples taken BYE-grown bacteria performed described previously          The apparent essentiality oxyRLp unexpected, the <br /> (30). Values expressed relative fluorescence units (RFU) OD620 and <br />                                                                                      genomic organization uncomplicated overlapping reg- <br /> represent triplicate values obtained independent experiments. <br />    OxyRLp regulation ahpC2-gfp E. coli GS077. DNA fragments containing         ulatory elements downstream polarity effects. Possible func- <br /> gfp (no promoter) PahpC2gfp excised EcoRI PstI plasmids        tional divergence considered, previous studies had <br /> pBH6119 pC2gfp, respectively, subcloned pMMB206. The               indicated genes typically regulated response oxidative <br /> resulting constructs, pgfp pahpC2gfp, respectively, electroporated     stress bacteria (ahpC1, ahpC2D, katA, katB, sodB, and <br /> either GS077 pBAD22 GS077 pBADLpoxyR. Ampicillin- chloramphen- <br /> icol-resistant transformants confirmed PCR analysis subjected the <br />                                                                                      sodC) similarly regulated L. pneumophila (4, 5, 30, <br /> GFP reporter assay (30). Overnight cultures, grown 37&#194;&#176;C LB broth         38, 39, 43). Rather, genes exhibited growth <br /> anaerobic conditions, inoculated 1% fresh media OD620 0.4      stage-dependent expression (4, 5, 30, 43). <br />  3448     LEBLANC ET AL.                                                                                                                    J. BACTERIOL. <br />  <br />  <br />  <br />  <br />    FIG. 1. Acrylamide capture proteins interacting PahpC2. The ACDC assay performed adding 1 &#226;?&#174;g protein (cell extract) <br /> prepared BYE-grown L. pneumophila various amounts Ac-DNA (0, 2, 4, 8 &#226;?&#174;g). Nonspecific proteins removed nondena- <br /> turing electrophoresis 0.5&#226;&#171;&#187; TBE-5% polyacrylamide gel. Captured proteins (inset) excised wells, heated SDS sample buffer <br /> containing &#226;?&#164;-mercaptoethanol, following SDS-PAGE, visualized silver staining. dsDNA, double-stranded DNA; APS, ammonium persul- <br /> fate; TEMED, N,N,N&#226;&#172;&#732;,N&#226;&#172;&#732;&#226;&#172;&#732;-tetramethylethylenediamine. <br />  <br />  <br />  <br />    Complementation E. coli oxyR mutant. Functional                  bound state. No differences binding ob- <br /> complementation assessed cloning (pBAD22) ex-               served oxidizing (no added reducing agent) reducing <br /> pressing oxyRLp gene E. coli oxyR mutant strain GS077.          conditions (200 mM DTT) (see Fig. 2B). Similar experiments <br /> Studies conducted anaerobic conditions              carried PahpC1 PoxyR; however, DNA <br /> this strain highly susceptible peroxide killing (compared        binding observed (data shown). EMSAs also <br /> with wild-type strain MG1655) loss OxyR regula-            performed amplicons containing promoter regions of <br /> tory functions. Under inducing conditions (arabinose) re-           known members E. coli OxyR regulon (ahpCF, katG, <br /> pressing conditions (glucose), presence pBAD22 did          oxyR) oxidizing conditions. As seen Fig. 2C, <br /> alter peroxide susceptibility (Table 3). Complementation of <br /> GS077 pBADEcoxyR fully restored peroxide resistance to <br /> wild-type E. coli MG1655 levels inducing conditions, but <br />                                                                         TABLE 3. Zones inhibition (mm) disk diffusion assay using <br /> not repressing conditions. In contrast, overexpression                              10 &#226;?&#174;l 3% H2O2 <br /> pBADLpoxyR partially rescued peroxide sensitivity E. <br />                                                                                                                  Zones inhibition (mm) with: <br /> coli GS077, suggesting OxyRLp recognizes OxyR-regu- <br /> lated promoters. The noted partial complementation                            Straina                    Glucose (%)              Arabinose (%)b <br /> OxyRLp DNA sequence differences OxyR                                                    0           0.2          0.02           0.2 <br /> binding motifs organisms. <br />    Interaction OxyRLp promoter region ahpC2.            K-12&#226;&#8364;&#160;                            21 &#226;&#171;&#190; 1       23 &#226;&#171;&#190; 1       21 &#226;&#171;&#190; 1         21 &#226;&#171;&#190; 1 <br />                                                                        K-12 pBAD22&#226;&#8364;&#161;                     21 &#226;&#171;&#190; 1       22 &#226;&#171;&#190; 1       22 &#226;&#171;&#190; 1         22 &#226;&#171;&#190; 1 <br /> To confirm ACDC results extend functional studies,            GS077&#194;&#167;                           38 &#226;&#171;&#190; 2       40 &#226;&#171;&#190; 1       38 &#226;&#171;&#190; 1         37 &#226;&#171;&#190; 2 <br /> Ni2&#226;&#171;&#185;-nitrilotriacetic acid-purified OxyRLp (overexpression             GS077 pBAD22&#226;&#8364;&#161;                    39 &#226;&#171;&#190; 1       41 &#226;&#171;&#190; 1       38 &#226;&#171;&#190; 1         38 &#226;&#171;&#190; 1 <br /> <span id='am-5' about='protege:FROM' typeof='owl:Thing'>from</span> pETLpoxyR) assessed EMSA DNA-binding                   GS077 pBADEcoxyR**               37 &#226;&#171;&#190; 2       40 &#226;&#171;&#190; 1      19* &#226;&#171;&#190; 1        18* &#226;&#171;&#190; 1 <br /> specificity promoter regions (PahpC2, PahpC1, PoxyR)        GS077 pBADLpoxyR**               39 &#226;&#171;&#190; 1       39 &#226;&#171;&#190; 1      29* &#226;&#171;&#190; 2        29* &#226;&#171;&#190; 2 <br /> of genes. As seen Fig. 2A, OxyRLp bound 226-bp              a <br />                                                                             Triplicate experiments performed anaerobic conditions Muel- <br /> DNA fragment containing PahpC2 (&#226;&#171;&#186;199 &#226;&#171;&#185;27 relative            ler-Hinton agar antibiotics (&#226;&#8364;&#160;) supplemented 100 &#226;?&#174;g/ml ampi- <br />                                                                        cillin (&#226;&#8364;&#161;), 40 &#226;?&#174;g/ml kanamycin (&#194;&#167;), 100 &#226;?&#174;g/ml ampicillin 40 &#226;?&#174;g/ml <br /> ahpC2 coding sequence). This evident sample con-             kanamycin (**). <br /> taining 500 ng His6-OxyRLp, 84% DNA probe                 b <br />                                                                             *, significance (P &#226;&#172;? 0.01) determined relation strain GS077. <br />  VOL. 190, 2008                                                                OxyRLp REGULATION OF ahpC2D IN LEGIONELLA                         3449 <br />  <br />  <br />  <br />  <br />    FIG. 2. Interaction OxyRLp PahpC2. (A) Fifty ng 226-bp PahpC2 fragment spanning region &#226;&#171;&#186;199 &#226;&#171;&#185;27 (relative L. <br /> pneumophila ahpC2 initiation codon) incubated presence various amounts His6-OxyRLp. Lanes: 1, low-mass ladder (Invitrogen); <br /> 2, DNA protein; 3 10, DNA presence 3.9, 7.8, 15.6, 31.3, 62.5, 125, 250, 500 ng His6-OxyRLp, respectively. Arrows indicate <br /> bands representing DNA probe (P) mobility shift (S). Densitometry analysis reported percentage DNA probe (P [%]) shift (S <br /> [%]) relative total intensity lane. (B) Mobility shift assay performed presence 25 ng PahpC2 OxyRLp reducing <br /> (200 mM DTT, OxyRLp-red) oxidizing (no DTT, OxyRLp-ox) conditions. (C) Mobility shift assay performed presence 50 ng E. <br /> coli PahpCF, PkatG, PoxyR presence (&#226;&#171;&#185;) absence (&#226;&#171;&#186;) 500 ng His6-OxyRLp nonreducing conditions. <br />  <br />  <br />  <br />  <br /> OxyRLp weakly bound promoters highest                  albeit lower expression level (data shown). In parallel <br /> concentrations protein used (500 ng). The EMSA results                    series experiments, mobility shift assays performed <br /> indicate partial complementation E. coli oxyR                 various PahpC2 fragments (Fig. 3C). Deletion nucleo- <br /> mutant oxyRLp combination OxyRLp                     tides &#226;&#171;&#186;199 &#226;&#171;&#186;109 effect OxyRLp binding (Fig. 3C, <br /> overexpression partial activation OxyR-regulated pro-                 lanes 2 5). However, deletion nucleotides &#226;&#171;&#186;91 to <br /> moters GS077.                                                             &#226;&#171;&#186;66 (which ablated transcription) did completely abol- <br />    Mapping OxyR binding site PahpC2. To identify               ish DNA binding, suggesting OxyRLp binding motif(s) <br /> upstream DNA sequences necessary ahpC2 expression,                       remained. An alignment DNA sequences &#226;&#171;&#186;105 to <br /> PCR-generated 5&#226;&#172;&#732;-end deletion fragments PahpC2                       &#226;&#171;&#186;55 region PahpC2 genome sequences L. <br /> cloned upstream gfpmut3, encoding GFP. The promoter                       pneumophila sequences OxyR-regulated <br /> fragments shared common 3&#226;&#172;&#732; end &#226;&#171;&#185;27 (relative                     genes E. coli species reveals conservation <br /> ahpC2 initiation codon) variable 5&#226;&#172;&#732; end ranging                   E. coli OxyR consensus (Fig. 3D). Taken together, <br /> &#226;&#171;&#186;199 &#226;&#171;&#186;66 (Fig. 3A). The ability PahpC2 fragments                    findings suggest OxyRLp-protected region con- <br /> express GFP assessed ahpC1 mutant (Fig. 3B),                tains OxyR motif include promoter sequences. <br /> ahpC2 levels increased mutant (30). The 5&#226;&#172;&#732;-end                      DNase I footprinting primer extension. To de- <br /> deletions PahpC2 <span id='am-88' about='xsp:base' typeof='owl:Thing'>base</span> pairs &#226;&#171;&#186;179 &#226;&#171;&#186;109 showed                      lineate specific DNA binding sequence PahpC2 recog- <br /> transcriptional activity similar previously studied           nized OxyRLp, untreated reduced (200 mM DTT) <br /> full-<span id='am-7' about='xsp:length' typeof='owl:Thing'>length</span> fragment (&#226;&#171;&#186;199 &#226;&#171;&#185;27) (30). However, PahpC2                     forms OxyR (OxyRLp-ox OxyRLp-red, respectively) were <br /> deletions &#226;&#171;&#186;91 &#226;&#171;&#186;66 longer expressed GFP. It                used footprinting assay. OxyRLp-red OxyRLp-ox showed <br /> noted similar results obtained wild-type strain,            similar DNA footprints (Fig. 4A B) confirmed the <br />  3450     LEBLANC ET AL.                                                                                                            J. BACTERIOL. <br />  <br />  <br />  <br />  <br />    FIG. 3. Promoter deletion analysis OxyRLp binding site PahpC2. (A) Schematic representation 5&#226;&#172;&#732;-end deletions PahpC2 <br /> cloned upstream promoter-less gfp. (B) Fluorescence detected ahpC1::kan mutants harboring plasmids PahpC2 promoter DNA-GFP <br /> constructs. (C) Mobility shift assay performed 5&#226;&#172;&#732;-end deletions PahpC2. Each lane contained 50 ng DNA fragments incubated with <br /> (right panel) (left panel) 500 ng His6-OxyRLp. For gels, lanes follows (symbols shown panel B): 1, low-mass ladder <br /> (Invitrogen); 2, PahpC2 fragment &#226;&#171;&#186;199 &#226;&#171;&#185;27 (f); 3, PahpC2 fragment &#226;&#171;&#186;179 &#226;&#171;&#185;27 (&#227;&#8364;&#171;); 4, PahpC2 fragment &#226;&#171;&#186;156 &#226;&#171;&#185;27 (&#197;&#8217;); 5, PahpC2 fragment <br /> &#226;&#171;&#186;132 &#226;&#171;&#185;27 (*); 6, PahpC2 fragment &#226;&#171;&#186;109 &#226;&#171;&#185;27 (E); 7, PahpC2 fragment &#226;&#171;&#186;91 &#226;&#171;&#185;27 (&#226;&#8212;?); 8, PahpC2 fragment &#226;&#171;&#186;66 &#226;&#171;&#185;27 (&#226;&#8364;&#353;); 9, no-DNA <br /> control (}). DNA fragments annotated relative ahpC2 initiation codon. (D) DNA alignments ahpC2 upstream region (&#226;&#171;&#186;115 to <br /> &#226;&#171;&#186;55) L. pneumophila Philadelphia-1, L. pneumophila Paris, L. pneumophila Lens; ahpC Xanthomonas campestris (Xc) P. <br /> aeruginosa (Pa); ahpC, katG, oxyR, oxyS E. coli. The consensus sequence E. coli provided, matching motifs are <br /> underlined. <br />  <br />  <br />  <br /> protected region included &#226;&#171;&#186;91 &#226;&#171;&#186;66 regions. Primer                  purification process, typical OxyR <br /> extension analysis (Fig. 4C) revealed transcription               proteins (47, 48), oxidation H2O2 <br /> start site &#226;&#171;&#186;10 hexamer located               reveal typical redox-mediated DNA-binding dif- <br /> OxyRLp-protected region (Fig. 4D). The location pro-                ferences. <br /> moter region transcription start site OxyRLp-                 To test possibility, treated OxyRLp range of <br /> protected region unusual, E. coli, reduced OxyR                peroxide concentrations (10 &#226;?&#174;M 50 mM) determined <br /> binds upstream promoter region OxyR-regulated                    1 mM completely abolished DNA binding (Fig. 4A, lane <br /> genes represses transcription overlapping &#226;&#171;&#186;35 hex-              4). While shown, treatment OxyRLp 500 &#226;?&#174;M H2O2 <br /> amer (9, 31, 34, 36, 45, 46). In case PahpC2, &#226;&#171;&#186;35               reduced DNA binding 50% did alter protected <br /> hexamer outside OxyRLp-protected region, suggesting              area relative untreated OxyRLp controls. To distinguish <br /> a different regulatory mechanism reported                    reversible irreversible oxidation OxyRLp, we <br /> OxyREc (46, 47). However, OxyRLp fully oxidized                 followed 1 mM H2O2 treatment 200 mM DTT treatment <br />  VOL. 190, 2008                                                               OxyRLp REGULATION OF ahpC2D IN LEGIONELLA                        3451 <br />  <br />  <br />  <br />  <br />    FIG. 4. OxyRLp binding PahpC2 ahpC2 transcript start site. Shaded bars indicate OxyRLp DNA footprints strand (end labeled <br /> at BamHI) (A) strand (end labeled EcoRI) (B). OxyRLp binding PahpC2 occurred following conditions: lanes 1 6, no <br /> protein control reactions; lane 2, oxidizing conditions (no DTT); lane 3, 200 mM DTT; lanes 4, 1 mM H2O2; lane 5, treated 1 mM H2O2 and <br /> then 200 mM DTT recover DNA-binding activity. In panels A C, asterisk denotes ahpC2 transcript start site indicated alignment <br /> of primer extension product corresponding DNA sequence ladder. (D) Schematic PahpC2. Arrows indicate FC2PROM RC2PROM <br /> primers, open boxes denote OxyRLp sites determined DNase I protection assays shown panels A B, boldface indicates open reading <br /> frames ahpC2 lpg2351, ribosome-binding site (RBS) ahpC2 underlined. The transcription start site indicated asterisk, <br /> and &#226;&#171;&#186;10 sequence indicated bold. The numbers parentheses depict numbering sites deletions shown Fig. 3. <br />  <br />  <br />  <br /> to reduce disulfide bonds, seen Fig. 4A, lane 5,              polymerase complex, sought confirm OxyRLp was <br /> subsequent reduction restore DNA binding activity.                repressor. We introduced pahpC2gfp E. coli <br /> These studies indicate OxyRLp footprint unchanged               GS077 (the oxyR mutant) noted constitutive expression of <br /> by treatments H2O2 concentrations                    GFP (Fig. 5). We introduced pBADLpoxyR pBAD22 <br /> sufficient fully oxidize protein.                                    vector controls GS077 strains harboring pahpC2gfp and <br />   OxyRLp represses ahpC2-gfp expression. Since OxyRLp                   pgfp controls induced arabinose glu- <br /> footprint ahpC2 appears block binding RNA                 cose. As seen Fig. 5, arabinose-dependent induction of <br />  3452     LEBLANC ET AL.                                                                                                       J. BACTERIOL. <br />  <br />  <br />  <br />  <br />   FIG. 5. Regulation ahpC2-gfp E. coli GS077 OxyRLp. <br /> Strains grown LB broth subjected repressing (glucose) <br /> or inducing (arabinose) conditions. Treatment 100 &#226;?&#174;M H2O2 is <br /> indicated appropriate. RFUs normalized 1.0 OD620 <br /> <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span>. Results means &#226;&#171;&#190; standard deviations indepen- <br /> dent experiments. <br />  <br />  <br />  <br /> OxyRLp pBADLpoxyR repressed fluorescence from <br /> pahpC2gfp, consistent hypothesis binding of <br /> OxyRLp promoter ahpC2-gfp blocked transcription. <br /> Moreover, change ahpC2-gfp expression fol- <br /> lowing challenge 100 &#226;?&#174;M H2O2 (oxidative stress), further <br /> evidence OxyRLp redox-active regulator. <br />    Cell cycle expression oxyRLp. To determine temporal <br /> expression oxyR vitro growth, GFP reporter assay          FIG. 6. Cell cycle-dependent oxyR expression L. pneumophila. <br />                                                                        (A) GFP reporter assays (B) bacterial growth curves PoxyR were <br /> was performed. Figure 6 shows expression oxyRLp         performed BYE broth, fluorescence determined sam- <br /> growth cycle-dependent, increasing mid- postexpo-           ples taken 3 h, following growth wild type (}) the <br /> nential phases decreasing slightly late-stationary          ahpC2D (&#226;&#8212;?), rpoS (&#226;&#8364;&#353;), letA (&#197;&#8217;), himAB (encoding integration host <br /> phase. Similar trends observed peroxide-sensitive              factor) (E) mutant strains. RFUs normalized 1.0 OD620 <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span>. <br /> ahpC1 ahpC2D mutants. In attempt identify possible           Results means &#226;&#171;&#190; standard deviations independent <br />                                                                        experiments. <br /> regulators oxyR expression, poxyRgfp reporter construct <br /> was transformed mutant strains lacking known regulators <br /> of L. pneumophila virulence, rpoS, letA, himAB (encoding <br /> integration host factor). No differences oxyR expression            genes, including oxyRLp, aligned growth stage, increas- <br /> were observed mutants analyzed (Fig. 6), indi-           ing dramatically mid-exponential phase peaking in <br /> cating postexponential expression OxyRLp            early stationary phase, consistent view oxidative <br /> activated RpoS LetA/S regulatory cascade.                    stress acute stationary-phase bacteria (4, 39). By <br />                                                                        demonstrating OxyRLp repressed transcription from <br />                           DISCUSSION                                   ahpC2D, partly resolved basis reciprocal <br />                                                                        gene expression noted ahpC1 ahpC2D. Studies <br />    Our laboratory characterized alkyl hydroperoxide            progress identify regulatory mechanisms associated <br /> reductases (AhpC1 AhpC2D) expressed differ-            postexponential activation ahpC1 oxyR. <br /> ent stages bacterial growth provide essential peroxidatic          OxyRLp appears functionally different other <br /> functions L. pneumophila. Loss AhpC1 function results         OxyR proteins shown function redox- <br /> increased expression ahpC2D, suggesting compensatory              active repressors (47, 48). Here present lines of <br /> response cellular oxidative stress (30). Using ACDC              evidence suggest OxyRLp longer function a <br /> assay, identified OxyRLp regulatory factor controlling         global regulator antioxidant defenses: (i) oxyRLp expression <br /> expression ahpC2D operon. OxyRLp bound ahpC2D                growth stage dependent, (ii) OxyRLp function apparently <br /> promoter sequences did bind promoter regions            essential, (iii) DNA-binding properties OxyRLp re- <br /> of ahpC1 itself (autoregulation). Members OxyR               dox dependent, (iv) OxyRLp contains key amino acid substitu- <br /> family well-characterized, redox-active transcriptional ac-        tions likely ablate disulfide-bond formation conforma- <br /> tivators genes associated defense oxidative            tional changes required activation, (v) H2O2 <br /> stress. In contrast, antioxidant defense genes             treatment did increase ahpC2-gfp levels L. pneumophila <br /> (sodB, sodC, katA, katB, ahpC1 ahpC2D) L. pneu-              E. coli oxyR mutant (GS077) expressing oxyRLp. How- <br /> mophila appear activated response oxidative stress         ever, findings exclude possibility OxyRLp <br /> (4, 5, 30, 39, 43). Moreover, expression of          retains DNA sequence recognition OxyR-regulated <br />  VOL. 190, 2008                                                        OxyRLp REGULATION OF ahpC2D IN LEGIONELLA                  3453 <br />  <br />  <br />  <br />  <br />   FIG. 7. Clustal W alignment selected OxyR amino acid sequences. The alignments L. pneumophila Philadelphia-1 (LpnOxyR <br /> [OxyRLp]), Xanthomonas campestris (XanOxyR [OxyRXc]), E. coli (EcoliOxyR [OxyREc]), C. burnetii (CbOxyR [OxyRCb]). The DNA binding <br /> helix-turn-helix (HTH) shaded, relevant amino acid positions numbered bolded. <br />  <br />  <br /> promoters E. coli, indicated complementation            form disulfide bonds (redox potential &#226;&#171;&#186;185 mV) during <br /> EMSA studies. Our studies indicate OxyRLp                protein purification (1, 50). In organisms, Deino- <br /> locked activated DNA-binding conformation (29), inde-          coccus radiodurans, oxidation single (peroxidatic) cysteine <br /> pendent activation peroxide sufficient ac-      residue OxyR sufficient promote activation (10). Thus, <br /> tivate antioxidant defense E. coli.                    regardless single cysteine oxidized or <br />    In E. coli, OxyR binds DNA dimer dimers        disulfide bond formed, studies suggest the <br /> reducing oxidizing conditions, promotes      redox-sensing domain OxyRLp longer signals conforma- <br /> expression target genes (47). Treatment E. coli OxyR       tional changes promote activation. <br /> with millimolar amounts DTT shown promote                 Members LysR family (which includes OxyR) use a <br /> binding OxyR contact sites blocked &#226;&#171;&#186;35 sequences       ligand-binding conformational change mechanism mediate <br /> OxyR-regulated promoters, peroxide oxidation              DNA binding (11). <span id='am-10' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-11' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span>Mutation</span>-based studies identified key <br /> OxyR led conformational change protein con-         amino acids required function OxyR. For example, <br /> tracted DNA footprint permitted strong induction          E. coli OxyR H125I H218D mutations resulted wild- <br /> these promoters (44, 46, 47). In case L. pneumophila,         type DNA binding loss redox-mediated regulation <br /> treatment OxyRLp micromolar concentrations H2O2           (11). These amino acid substitutions occur naturally the <br /> did alter footprint, suggested OxyRLp      OxyRLp proteins strains L. pneumophila whose <br /> was fully oxidized protein purification DNA           genomes sequenced (125 218) (Fig. 7). While <br /> binding independent oxidation state. While millimolar         redox-active cysteine residues involved OxyR acti- <br /> concentrations H2O2 abolished DNA binding,             vation (Cys199 Cys208) conserved OxyRLp, is <br /> result considered physiologically relevant biological   substantial amino acid sequence divergence activation <br /> activity recoverable treatments 200 mM DTT.          domain affect redox activity. For example, key <br /> In regard, functional studies shown redox-        arginine position 220 OxyREc (Fig. 7) predicted to <br /> active thiols (C199 C208) OxyREc spontaneously oxidize        interact catalytic cysteines peroxide (28) changed to <br />  3454     LEBLANC ET AL.                                                                                                                J. BACTERIOL. <br />  <br />  <br /> threonine OxyRLp proteins. Interestingly, OxyR pro-           upregulation ahpC2D ahpC1 mutant postexpo- <br /> teins close relative C. burnetii Francisella tularensis,      nential activation ahpC1 (30). In L. pneumophila, transition <br /> Arg220 replaced lysine, suggest                exponential phase stationary phase/cyst differentiation <br /> traditional function organisms. Further comparisons            controlled postexponential regulators RpoS, LetA, <br /> revealed amino acid sequence variation helix-turn-           HimAB (2, 13, 20, 22, 32, 33, 52). However, oxyRLp-gfp <br /> helix region account differences DNA-             gene expression studies rpoS, letA, himAB mutants were <br /> binding specificity noted OxyRLp OxyREc                  indistinguishable wild type. While di- <br /> complementation experiments EMSAs. We suggest                  rectly examined, carbon storage regulator CsrA, important <br /> the loss redox-sensing function OxyRLp occurred           repressor stationary-phase genes exponential phase <br /> naturally specific amino acid substitutions           (13, 33), involved regulating oxyRLp, CsrA <br /> been demonstrated empirical studies alter disulfide          repressed postexponential expression oxyRLp in <br /> bond formation conformational changes required ac-              letA rpoS mutants. Future studies employ ACDC <br /> tivation.                                                               strategy capture putative regulatory factors associated with <br />    In contrast bacteria, F. tularensis, C. burnetii,   control oxyRLp expression. <br /> and Streptomyces species, oxyR located upstream                In summary, shown OxyRLp represses ahpC2D <br /> and divergently expressed ahpC2D system, oxyRLp            expression oxyRLp induced postexponentially. Our <br /> gene located chromosome. It conceivable          studies establish OxyRLp longer functions an <br /> that present oxyRLp acquired laterally, con-            oxidative response regulator L. pneumophila, con- <br /> comitantly deletion loss ancestral upstream           sistent previous observations L. pneumophila mounts <br /> ortholog, necessary event associated evolu-          response oxidative stress bacteria are <br /> tion dimorphic lifestyle.                                          resistant oxidative damage, phagocytic cells <br />    It unlikely function OxyRLp            (30). We propose OxyRLp adapted an <br /> repress ahpC2D growth transition stationary               oxidative stress response regulator growth stage-specific <br /> phase cyst morphogenesis. While downregulation              regulator genes mediating transition vegetative to <br /> ahpC2D consistent metabolic changes shift          resilient cyst-like transmissible forms. <br /> NADPH-thioredoxin AhpC1 occur the <br /> transition stationary phase, ahpC2D deletion mutants                                      ACKNOWLEDGMENTS <br /> both viable infectious (30). Moreover, high-level ahpC2D <br />                                                                            We thank Gisela Storz Michele Swanson providing bacterial <br /> expression, noted ahpC1 deletion mutant, sug-             strains plasmids. <br /> gested postexponential presence AhpC2D                 This work supported CIHR grant MOP 14443 NIH grant <br /> toxic. Thus, OxyRLp provide essential regulatory             AI066058 P.S.H. CIHR postdoctoral fellowship A.K.C.B. <br /> functions L. pneumophila, mutant be <br /> obtained methods routinely used generate mutants                                                  REFERENCES <br /> (30). We used pattern search option L. pneumophila            1. Aslund, F., M. Zheng, J. Beckwith, G. Storz. 1999. Regulation the <br />                                                                             OxyR transcription factor hydrogen peroxide cellular thiol-disul- <br /> Paris Lens genome website (http://genolist.pasteur.fr                   fide status. Proc. Natl. Acad. Sci. USA 96:6161&#226;&#8364;&#8220;6165. <br /> /LegioList/) screen additional genes, using motif patterns        2. Bachman, M. A., M. S. Swanson. 2001. RpoS co-operates other <br /> deduced comparisons E. coli L. pneu-                factors induce Legionella pneumophila virulence stationary phase. <br />                                                                             Mol. Microbiol. 40:1201&#226;&#8364;&#8220;1214. <br /> mophila OxyR binding motif. While motifs relatively            3. Bandyopadhyay, P., B. Byrne, Y. Chan, M. S. Swanson, H. M. Steinman. <br /> degenerate, binding sites observed upstream approxi-                2003. Legionella pneumophila catalase-peroxidases required proper <br />                                                                             trafficking growth primary macrophages. Infect. Immun. 71:4526&#226;&#8364;&#8220; <br /> mately 50 genes, including dps homolog, recently              4535. <br /> been prevent Fenton-mediated DNA damage se-                  4. Bandyopadhyay, P., H. M. Steinman. 2000. Catalase-peroxidases of <br /> questering iron (37). Others included promoters genes en-                Legionella pneumophila: cloning katA gene studies KatA func- <br />                                                                             tion. J. Bacteriol. 182:6679&#226;&#8364;&#8220;6686. <br /> coding efflux pumps (that exclude redox-cycling compounds                5. Bandyopadhyay, P., H. M. Steinman. 1998. Legionella pneumophila <br /> and organic solvents); metal ion transporters (which regulate               catalase-peroxidases: cloning katB gene studies KatB function. <br /> rates Fenton chemistry); components respiratory                   J. Bacteriol. 180:5369&#226;&#8364;&#8220;5374. <br />                                                                          6. Bandyopadhyay, P., H. Xiao, H. A. Coleman, A. Price-Whelan, H. M. <br /> chain, cytochrome oxidases, DNA repair, modifica-                Steinman. 2004. Icm/Dot-independent entry Legionella pneumophila into <br /> tion enzymes; known virulence factors, zinc met-                amoeba macrophage hosts. Infect. Immun. 72:4541&#226;&#8364;&#8220;4551. <br />                                                                          7. Berger, K. H., R. R. Isberg. 1993. Two distinct defects intracellular <br /> alloproteinase (shown inhibit oxidative burst); sub-             growth complemented single genetic locus Legionella pneumophila. <br /> strates Dot/Icm type IVB secretion (SidG, SdhB,               Mol. Microbiol. 7:7&#226;&#8364;&#8220;19. <br /> and SdeA). Finally, OxyR binding motifs up-              8. Bruggemann, H., A. Hagman, M. Jules, O. Sismeiro, M. A. Dillies, C. <br />                                                                             Gouyette, F. Kunst, M. Steinert, K. Heuner, J. Y. Coppee, C. Buchrieser. <br /> stream genes encoding numerous transcriptional regulators,               2006. Virulence strategies infecting phagocytes deduced vivo <br /> like Fur, FleQ, FleN, members LysR family,                 transcriptional program Legionella pneumophila. Cell. Microbiol. 8:1228&#226;&#8364;&#8220; <br /> suggesting cross talk regulatory             1240. <br />                                                                          9. Charoenlap, N., W. Eiamphungporn, N. Chauvatcharin, S. Utamapongchai, <br /> pathways. Interestingly, unlike E. coli, fur essential gene        P. Vattanaviboon, S. Mongkolsuk. 2005. OxyR mediated compensatory <br /> in L. pneumophila (24). Further studies required                 expression ahpC katA significance ahpC protec- <br />                                                                             tion hydrogen peroxide Xanthomonas campestris. FEMS Microbiol. <br /> dissect additional regulatory functions associated essenti-            Lett. 249:73&#226;&#8364;&#8220;78. <br /> ality.                                                                  10. Chen, H., G. Xu, Y. Zhao, B. Tian, H. Lu, X. Yu, Z. Xu, N. Ying, S. Hu, and <br />    Our studies predict participation additional regulatory           Y. Hua. 2008. A novel OxyR sensor regulator hydrogen peroxide <br />                                                                             stress cysteine residue Deinococcus radiodurans. PLoS ONE <br /> elements controlling expression ahpC2D, re-                 13:e1602. <br /> pression ahpC2D OxyRLp does fully explain                 11. Choi, H., S. Kim, P. Mukhopadhyay, S. Cho, J. Woo, G. Storz, S. Ryu. <br />  VOL. 190, 2008                                                                             OxyRLp REGULATION OF ahpC2D IN LEGIONELLA                                   3455 <br />  <br />       2001. Structural basis redox switch OxyR transcription factor.        32. Lynch, D., N. Fieser, K. Gloggler, V. Forsbach-Birk, R. Marre. 2003. <br />       Cell 105:103&#226;&#8364;&#8220;113.                                                                       The response regulator LetA regulates stationary-phase stress response <br /> 12.   Cirillo, J. D., S. Falkow, L. S. Tompkins. 1994. Growth Legionella               Legionella pneumophila required efficient infection Acan- <br />       pneumophila Acanthamoeba castellanii enhances invasion. Infect. Immun.               thamoeba castellanii. FEMS Microbiol. Lett. 219:241&#226;&#8364;&#8220;248. <br />       62:3254&#226;&#8364;&#8220;3261.                                                                       33. Molofsky, A. B., M. S. Swanson. 2003. Legionella pneumophila CsrA a <br /> 13.   Fettes, P. S., V. Forsbach-Birk, D. Lynch, R. Marre. 2001. Overexpress-             pivotal repressor transmission traits activator replication. Mol. <br />       sion Legionella pneumophila homologue E. coli regulator csrA                Microbiol. 50:445&#226;&#8364;&#8220;461. <br />       affects cell size, flagellation, pigmentation. Int. J. Med. Microbiol. 291:     34. Mongkolsuk, S., W. Whangsuk, P. Vattanaviboon, S. Loprasert, M. <br />       353&#226;&#8364;&#8220;360.                                                                                Fuangthong. 2000. A Xanthomonas alkyl hydroperoxide reductase subunit C <br /> 14.   Fields, B. S., R. F. Benson, R. E. Besser. 2002. Legionella Legion-             (ahpC) mutant showed altered peroxide stress response complex <br />       naires&#226;&#8364;&#8482; disease: 25 years investigation. Clin. Microbiol. Rev. 15:506&#226;&#8364;&#8220;526.           regulation compensatory response peroxide detoxification enzymes. <br /> 15.   Fraser, D. W., T. R. Tsai, W. Orenstein, W. E. Parkin, H. J. Beecham, R. G.             J. Bacteriol. 182:6845&#226;&#8364;&#8220;6849. <br />       Sharrar, J. Harris, G. F. Mallison, S. M. Martin, J. E. McDade, C. C.               35. Nelson, C., S. Hendy, K. Reid, J. Cavanagh. 2002. Acrylamide capture <br />       Shepard, P. S. Brachman. 1977. Legionnaires&#226;&#8364;&#8482; disease: description             DNA-bound complexes: electrophoretic purification transcription fac- <br />       epidemic pneumonia. N. Engl. J. Med. 297:1189&#226;&#8364;&#8220;1197.                                  tors. BioTechniques 32:808&#226;&#8364;&#8220;810, 812, 814&#226;&#8364;&#8220;815. <br /> 16.   Gal-Mor, O., G. Segal. 2003. Identification CpxR positive regu-         36. Ochsner, U. A., M. L. Vasil, E. Alsabbagh, K. Parvatiyar, D. J. Hassett. <br />       lator icm dot virulence genes Legionella pneumophila. J. Bacteriol.           2000. Role Pseudomonas aeruginosa oxyR-recG operon oxidative <br />       185:4908&#226;&#8364;&#8220;4919.                                                                          stress defense DNA repair: OxyR-dependent regulation katB-ankB, <br /> 17.   Gardun  &#203;&#339; o, R. A., E. Garduno, M. Hiltz, P. S. Hoffman. 2002. Intracellular        ahpB, ahpC-ahpF. J. Bacteriol. 182:4533&#226;&#8364;&#8220;4544. <br />       growth Legionella pneumophila gives rise differentiated form dissim-        37. Park, M., S. T. Yun, S.-Y. Hwang, C.-I. Chun, T. I. Ahn. 2006. The dps <br />       ilar stationary-phase forms. Infect. Immun. 70:6273&#226;&#8364;&#8220;6283.                            gene symbiotic &#226;&#8364;&#339;Candidatus Legionella jeonii&#226;&#8364;? Amoeba proteus re- <br /> 18.   Guzman, L. M., D. Belin, M. J. Carson, J. Beckwith. 1995. Tight regu-               sponds hydrogen peroxide phagocytosis. J. Bacteriol. 188:7572&#226;&#8364;&#8220;7580. <br />       lation, modulation, high-level expression vectors containing             38. Rankin, S., Z. Li, R. R. Isberg. 2002. Macrophage-induced genes of <br />       arabinose PBAD promoter. J. Bacteriol. 177:4121&#226;&#8364;&#8220;4130.                                   Legionella pneumophila: protection reactive intermediates solute <br /> 19.   Halablab, M. A., M. Bazin, L. Richards, J. Pacy. 1990. Ultra-structure              imbalance intracellular growth. Infect. Immun. 70:3 <br /> </body></html>