<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Adenosine Kinase T. b. rhodesiense Identified the <br /> Putative Target 4-[5-(4-phenoxyphenyl)-2H-pyrazol-3- <br /> yl]morpholine Using Chemical Proteomics <br /> Sabine Kuettel1, Marc Mosimann2, Pascal Ma&#194;&#168;ser2, Marcel Kaiser3, Reto Brun3, Leonardo Scapozza1*, <br /> Remo Perozzo1* <br /> 1 Pharmaceutical Biochemistry Group, School Pharmaceutical Sciences, University Geneva, University Lausanne, Geneva, Switzerland, 2 Institute Cell Biology, <br /> University Bern, Bern, Switzerland, 3 Parasite Chemotherapy, Swiss Tropical Institute, Basel, Switzerland <br />  <br />  <br />  <br />      <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />      Background: Human African trypanosomiasis (HAT), major parasitic disease spread Africa, urgently needs novel targets <br />      new efficacious chemotherapeutic agents. Recently, discovered 4-[5-(4-phenoxyphenyl)-2H-pyrazol-3- <br />      yl]morpholine (compound 1) exhibits specific antitrypanosomal activity IC50 1.0 mM Trypanosoma brucei <br />      rhodesiense (T. b. rhodesiense), causative agent acute form HAT. <br />  <br />      Methodology/Principal Findings: In work adenosine kinase T. b. rhodesiense (TbrAK), key enzyme the <br />      parasite purine salvage pathway vital parasite <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span>survival</span>, putative intracellular target compound 1 <br />      using chemical proteomics approach. This finding confirmed <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span>RNA interference</span> experiments showing down- <br />      regulation adenosine kinase counteracts compound 1 activity. Further chemical validation demonstrated compound <br />      1 interacts specifically tightly TbrAK nanomolar affinity, vitro activity measurements showed that <br />      compound 1 enhancer TbrAK activity. The subsequent kinetic analysis provided strong evidence observed <br />      hyperactivation TbrAK abolishment intrinsic substrate-inhibition. <br />  <br />      Conclusions/Significance: The results suggest TbrAK putative target compound, hyperactivation <br />      TbrAK represent novel therapeutic strategy development trypanocides. <br />  <br />   Citation: Kuettel S, Mosimann M, Ma&#194;&#168;ser P, Kaiser M, Brun R, et al. (2009) Adenosine Kinase T. b. rhodesiense Identified Putative Target 4-[5-(4- <br />   phenoxyphenyl)-2H-pyrazol-3-yl]morpholine Using Chemical Proteomics. PLoS Negl Trop Dis 3(8): e506. doi:10.1371/journal.pntd.0000506 <br />   Editor: Thomas Seebeck, University Bern, Switzerland <br />   Received December 17, 2008; Accepted July 23, 2009; Published August 25, 2009 <br />   Copyright: &#195;&#376; 2009 Kuettel et al. This open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: This work supported Swiss National Science Foundation grant no. 3100A0-120566/1 European Union (Prokinase Network <br />   #503467). The funders role study design, data collection analysis, decision publish, preparation manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: leonardo.scapozza@unige.ch (LS); remo.perozzo@unige.ch (RP) <br />  <br />  <br />  <br /> Introduction                                                                          Health Organization (WHO) private partnership. The type of <br />                                                                                       treatment depends stage disease. Pentamidine and <br />    Trypanosomiases belong major parasitic diseases spread                      suramin used stage treatment T. b. gambiense T. <br /> throughout world. Human African Trypanosomiasis (HAT;                             b. rhodesiense sleeping sickness, respectively. The drugs capable <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> <br /> sleeping sickness) vector-borne parasitic disease transmitted                 treat second stage infections toxic complicated to <br /> a protozoan parasite genus Trypanosoma bites                        administer, need cross blood-brain barrier reach <br /> infected tsetse flies. Two different forms HAT known. The                      parasites central nervous system. Melarsoprol, <br /> chronic form caused Trypanosoma brucei gambiense (T. b.                         effective forms HAT, derives arsenic <br /> gambiense) infection Trypanosoma brucei rhodesiense (T. b.                    severe effects, dramatic (prevalence 5 <br /> rhodesiense) responsible acute form disease. Both                   15%) reactive encephalopathy (encephalopathic <br /> forms HAT develop stages. After bite, parasites                   syndrome) fatal 3&#226;&#8364;&#8220;10% affected patients. An <br /> distribute blood, lymph peripheral organs (stage 1),                  increase resistance drug observed foci <br /> spread central nervous (stage 2) cause                       particularly central Africa. Eflornithine toxic than <br /> serious neurological disorders. Leaving infected people untreated,                    melarsoprol, effective T. b. gambiense sleeping <br /> HAT invariably fatal. Today, sleeping sickness threatens                           sickness, strict complicated regimen applied. It <br /> millions people 36 countries sub-Saharan Africa [1,2].                       evident therapy HAT relies drugs are <br /> The estimated number deaths annually currently                          associated severe effects. There revival drug <br /> 50&#226;&#8364;&#8482;000 70&#226;&#8364;&#8482;000 [3].                                                                research development regarding HAT compared 15 <br />    Four drugs (pentamidine, melarsoprol, eflornithine sura-                       years, number drug development projects currently <br /> min) registered treatment sleeping sickness                        ongoing. Unfortunately, development compound <br /> provided free charge endemic countries World                          (pafuramidine) having advanced phase III clinical trials [4,5] for <br />  <br />  <br />         www.plosntds.org                                                          1                                  August 2009 | Volume 3 | Issue 8 | e506 <br />                                                                                                    TbrAK Putative Target Novel Mechanism <br />  <br />  <br />  <br />   Author Summary                                                            validation suggested hyperactivation adenosine kinase as <br />                                                                             molecular mechanism underlying biological activity. <br />   Human African trypanosomiasis (HAT), devastating and <br />   fatal parasitic disease endemic sub-Saharan Africa,                    Methods <br />   urgently needs novel targets efficacious chemother- <br />   apeutic agents. Recently, discovered 4-[5-(4-                     Preparation T. b. rhodesiense lysate <br />   phenoxyphenyl)-2H-pyrazol-3-yl]morpholine exhibits spe-                      The parasites collected mouse blood lysed lysis <br />   cific antitrypanosomal activity T. b. rhodesiense,             buffer consisting 20 mM Hepes, 150 mM NaCl, 1% Triton X- <br />   causative agent acute form HAT. Here applied                 100, 2 mM tris(2-carboxyethyl)phosphine (TCEP), 10% glycerol, <br />   chemical proteomics approach cellular target                1 ml/ml protease inhibitor cocktail, pH 7.5. To end, the <br />   compound. Adenosine kinase, key enzyme                   suspension purified cells (2.566109 trypanosomes) 160 ml <br />   parasite purine salvage pathway, isolated                         PBS treated 40 ml lysis buffer concentrate (five fold). <br />   identified compound binding partner. Direct binding <br />                                                                             After short intervals sonication mixture centrifuged for <br />   assays using recombinant protein, tests an <br />                                                                             10 min 14&#226;&#8364;&#8482;000 g, supernatant recovered stored in <br />   adenosine kinase knock-down mutant parasite <br />   produced RNA interference confirmed TbrAK                       aliquots -20uC. The total protein concentration measured <br />   putative target. Kinetic analyses showed title                   using Bradford dye assay [8], lysate diluted to <br />   compound activator adenosine kinase                 desired concentration using lysis buffer. All protocols and <br />   observed hyperactivation TbrAK abolish-                  procedures used current study reviewed approved <br />   ment intrinsic substrate-inhibition. Whereas hyper-                local veterinary authorities Canton Basel-Stadt <br />   activation mechanism action known                    (Switzerland). <br />   drugs targeting cell signaling, novel hitherto <br />   unexplored concept compounds targeting metabolic                      Affinity chromatography identification mass <br />   enzymes, suggesting hyperactivation TbrAK                     spectrometry <br />   represent novel therapeutic strategy develop- <br />                                                                                Compounds 2, 3, 4 (Fig. 1) immobilized the <br />   ment trypanocides. <br />                                                                             primary secondary amino groups (Fig. S1) epoxy-activated <br />                                                                             agarose providing 12-atoms spacer (1,4-bis(2:3-epoxypropoxy)- <br /> stage treatment discontinued [6]. Thus, novel targets           butane). Swollen thoroughly washed matrix resuspended <br /> new efficacious chemotherapeutic agents urgently needed.                volumes 12.5&#226;&#8364;&#8220;50 mM ligand dissolved 50% <br />    Recently reported synthesis evaluation new 4-[5-           DMF/50 mM Na2CO3, 50 mM NaCl, pH 9.5. Coupling was <br /> (4-phenoxy-phenyl-2H-pyrazol-3-yl]morpholine              derivatives       performed 16&#226;&#8364;&#8220;72 h 25&#226;&#8364;&#8220;40uC. After 4 5 washes 50% <br /> against parasites [7]. One compounds, 4-[5-(4-               DMF/50 mM Na2CO3, 50 mM NaCl, pH 9.5, remaining <br /> phenoxy-phenyl-2H-pyrazol-3-yl]morpholine (Fig. 1, compound                 reactive groups blocked 1 M ethanolamine (pH 8) <br /> 1), exhibited good activity T. b. rhodesiense IC50        thoroughly washed low/high pH buffers. A control <br /> 1 mM low cytotoxicity [7]. This finding prompted                  matrix prepared ligand treated described <br /> address question regarding cellular target                  above. Direct absorbance scans immobilized ligand the <br /> molecular mechanism underlying observed toxicity                 matrix resuspended 50% glycerol solution (v/v) clearly <br /> the parasite. To end, report chemical proteomics             confirmed successful coupling (data shown). The of <br /> approach led identification adenosine kinase          inhibitor bound matrix determined calculation <br /> putative target. Subsequent biochemical biophysical charac-             of compound applied recovered UV <br /> terization respect compound 1 binding drug               determination. Routinely 20&#226;&#8364;&#8220;45 mmol/ml compound bound. <br /> sensitivity tests corresponding knock-down strain allowed        The resin incubated 2.5 volumes T. b. rhodesiense total cell <br />  <br />  <br />  <br />  <br /> Figure 1. Chemical structures compounds used study. The title compound 4-[5-(4-phenoxyphenyl)-2H-pyrazol-3-yl]morpholine (1) <br /> and amino substituted derivatives 4-[4-(5-morpholine-4-yl-1H-pyrazol-3-yl)-phenoxy]phenylamine (2), 4-[5-morpholine-4-yl-3-(4-phenoxyphenyl)- <br /> pyrazol-1-yl]phenylamine (3) 1-[5-(4-phenoxyphenyl-2H-pyrazol-3-yl]piperazine (4) presented. 3-(4-phenoxyphenyl)-1H-pyrazole (5) showed <br /> very low antiparasitic activity (IC50 .65 mM) general toxicity (IC50 .136 mM), used negative control the <br /> biochemical, biophysical compound sensitivity tests. <br /> doi:10.1371/journal.pntd.0000506.g001 <br />  <br />  <br />        www.plosntds.org                                                 2                              August 2009 | Volume 3 | Issue 8 | e506 <br />                                                                                                   TbrAK Putative Target Novel Mechanism <br />  <br />  <br /> lysate (2 mg/ml protein) incubated 4uC 2 hours. After            10 mM stock solution DMSO final concentration of <br /> washing lysis buffer (four times), matrix heated            33.4 mM 0.34% DMSO. The concentration [2-3H]aden- <br /> 95uC 5 min Laemmli sample buffer, directly loaded              osine (0.1&#226;&#8364;&#8220;12 mM) chosen consideration Michaelis&#226;&#8364;&#8220; <br /> separated 12% SDS-PAGE. Resolved proteins visualized                Menten conditions initial velocity measurements. The reaction <br /> by silver staining. The control matrix incubated          started addition 1 ng (0.88 nM) TbrAK. The <br /> amount lysate treated equally. Proteins bands excised           reactions incubated 5 ml aliquots spotted 5 mm <br /> <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> gels fragments resulting trypsin digestion           diameter DE81-cellulose disks placed 96-well plate stop the <br /> analyzed LC/ESI/MS/MS-QTOF mass spectrometry. Data-                      reaction. The disks washed 3 times 250 ml 5 mM <br /> <span id='am-238' about='xsp:base' typeof='owl:Thing'>base</span> searches performed using ProteinLynx Global                ammonium formate, H2O, transferred scintillation <br /> Server (all species) Mascot (other eukaryotes) search programs.         vials, soaked 2 ml displacement solution <br /> A score .46 indicates identity (p,0.05).                                    (100 mM HCl, 200 mM KCl) gently shaken 1 minute to <br />                                                                             elute phosphorylated products [9]. After adding 10 ml <br /> HPLC activity assay TbrAK                                               scintillation liquid, samples counted scintillation <br />    TbrAK catalyzes ATP-dependent phosphorylation                 counter. Km Vmax respect adenosine Ki for <br /> 59-hydroxyl moiety adenosine form AMP ADP. The                    adenosine substrate-inhibition absence presence of <br /> activity assay based HPLC separation substrate                compound 1 determined non-linear fit data to <br /> products (adenosine, AMP, ADP, ATP). The mobile phase                   substrate-inhibition model described [10]. The <br /> consisted 35 mM KH2PO4, 6 mM tetrabutylammonium                          results based independent series data point <br /> hydrogensulfate, (pH 7.0, adjusted KOH), 125 mM EDTA,                  measured triplicates. <br /> and 1% (v/v) acetonitrile. The separation performed <br /> isocratically flow-rate 1.5 ml/min monitored                 Genetic knock-down TbAK drug sensitivity tests <br /> 254 nm. The standard reaction 75 ml total volume containing                 The construction T. b. brucei adenosine kinase gene knock- <br /> 0.5 mM ATP, 0.5 mM adenosine, 0.5 mM MgCl2, 1% DMSO                      mutant described recently [11]. Expression a <br /> buffer E1 (20 mM Hepes, 150 mM NaCl, pH 7.0) started                    stem-loop construct targeting TbAK induced addition of <br /> with addition 2 mg TbrAK (final concentration 0.7 mM)            10 mg/ml tetracycline medium. Down-regulation TbAK <br /> incubated 37uC 10 min shaking vigorously (600 rpm).            expression verified Northern blotting. Parasites were <br /> To measure influence compound 1&#226;&#8364;&#8220;5 TbrAK activity,                 cultivated drug sensitivity tested using Alamar blue <br /> DMSO replaced assay mixture compound 1&#226;&#8364;&#8220;5                   test [12]. IC50 values calculated nonlinear fitting the <br /> that final concentration 1% DMSO maintained                sigmoidal dose-response curve using OriginH software. The assays <br /> the concentration compound varied range 0&#226;&#8364;&#8220;            performed triplicates times. <br /> 1 mM depending solubility. All reactions stopped by <br /> adding 75 ml mobile phase. 50 ml sample volume                  Other methods <br /> injected analysis resulting ADP/ATP ratios                    Origin supplies protocols purification of <br /> calculated used measure activity. For comparative               recombinant proteins, spectroscopic assay TbrGAPDH, <br /> reasons activity recorded absence compound set             radiometric assay TbrAK, thermal denaturation <br /> 100%. The mean independent measurements                        assay described Supporting Information (Text S1). <br /> reported. <br />                                                                             Results <br /> Calorimetric measurements <br />    Isothermal titration calorimetry (ITC) employed                   Compound choice immobilization <br /> elucidate binding affinity compound 1 TbrAK. All                 In order isolate identify intracellular target(s) of <br /> solutions degassed 5 min gentle stirring                compound 1 T. b. rhodesiense decided immobilize the <br /> vacuum. TbrAK dialyzed buffer E2 (20 mM Tris,                      compound isolate target(s) affinity chromatog- <br /> 150 mM NaCl, pH 7.5, 5% glycerol), diluted 7 mM                      raphy approach. The binding mode 1 putative target(s) <br /> supplemented 1% DMSO, filtered filled sample              unknown, immobilized derivatives compound 1 <br /> cell. The ligand solution prepared diluting stock solution         (Fig. 1, compound 2&#226;&#8364;&#8220;4), containing additional primary or <br /> of compound 1 5 DMSO buffer E2 final                   secondary amino group different positions [7], linked them <br /> concentration 150 mM 1% DMSO. The final ligand                       epoxy-activated agarose. The primary aromatic amine in <br /> solution filtered use. A titration experiment consisted       compound 2 3, secondary amino group the <br /> a control injection 1 ml followed 27 injections,        piperazine moiety compound 4 react terminal epoxy <br /> 10 ml 20 s duration, 4-min interval between. Raw              group activated agarose, providing uncharged, <br /> data collected, corrected ligand heats dilution,                hydrophilic stable 12-atoms spacer (see Fig. S1). The <br /> integrated fit two-set non-interacting binding sites             direct linkage compound 1 epoxy-activated agarose the <br /> model using OriginH software supplied instrument.              pyrazole moiety likely occur applied <br /> The measurements performed duplicate.                      experimental conditions weak nucleophilicity the <br />                                                                             heterocycle epoxide-based alkylation general requires <br /> Radiometric assay kinetic study TbrAK                                drastic conditions, i.e. prolonged heating high temper- <br />   Kinetic constants adenosine phosphorylation TbrAK                  ature concomitant use strong base [13]. Direct <br /> were obtained monitoring conversion [2-3H]adenosine            absorbance scans immobilized ligand matrix <br /> [2-3H]AMP. Reactions executed 37uC final volume             resuspended 50% glycerol solution (v/v) confirmed successful <br /> 30 ml containing 20 mM Hepes (pH 7.0), 50 mM NaCl, 167 mM                   coupling (data shown). Routinely 20&#226;&#8364;&#8220;45 mmol/ml compound <br /> ATP, 167 mM MgCl2, 0.34% BSA, 0.34% DMSO. To                            bound. Our recent evaluation compound 1 its <br /> measure influence compound 1 kinetic parameters,                  derivatives clearly indicated morpholine moiety be <br /> DMSO omitted assay mixture 1 added                important biological effect [7] corroborated by <br />  <br />  <br />       www.plosntds.org                                                  3                             August 2009 | Volume 3 | Issue 8 | e506 <br />                                                                                                     TbrAK Putative Target Novel Mechanism <br />  <br />  <br /> the fact compound 5 (morpholine removed) exhibits             trypsin digestion revealed peptide DIESTVLATK be <br /> low antiparasitic activity (IC50 .65 mM). Therefore compound              unambiguously allocated Q584S0 sequence, isoform <br /> 5 chosen negative control.                                            TbrAK cloned, expressed purified homogeneity and <br />                                                                              resulted 40&#226;&#8364;&#8220;50 mg soluble highly pure (.99%) TbrAK <br /> Immobilized compounds bind TbrGAPDH TbrAK                                liter culture (Fig. S2B). Both TbrGAPDH TbrAK were <br />    The total parasite lysate prepared, loaded matrices            active used subsequent analyses. <br /> and unbound material removed outlined Methods <br /> section. The bound proteins separated SDS-PAGE                   Compound 1 enhances TbrAK activity <br /> visualized silver-staining (Fig. 2). The use compound 2                   TbrAK activity absence presence compounds was <br /> derived matrix led detection distinct protein bands,         analyzed using HPLC protocol allowed separation of <br /> two higher molecular weights present           adenosine, AMP, ADP, ATP. Surprisingly, assay revealed <br /> control (Fig. 2A). Trypsin digestion peptide sequencing               2.5-fold (absolute value: 24563%) 1.7-fold (17465%) <br /> LC/ESI/MS/MS-QTOF mass spectrometry identified spe-                      increase enzyme activity presence 50 mM compound 1 <br /> cific band ,40 kDa adenosine kinase T. b. rhodesiense               2, respectively (Fig. 3; Fig. S3). A pronounced still <br /> (TbrAK, expected size 38.1 kDa; score 141),             significant effect (two-tailed t-test, p,0.01) observed for <br /> two corresponded human keratin (score 392 752). Affinity              compound 3 4 50 mM apparent increase activity <br /> matrix prepared compound 3 did isolate protein                    11061% 10961%, respectively. The negative control <br /> different ground signals identified human keratin             (compound 5) did activating effect TbrAK activity <br /> (Fig. 2B). In contrast, immobilized derivative 4 behaved                     (see Fig. S3). To validate activity measured single <br /> differently, distinct proteins isolated. Subsequent            concentration, concentration dependence effect was <br /> protein analysis human keratin cases. One distinct             analyzed. The hyperactivation effect concentration <br /> protein band visible control identified glycosomal         dependent compounds 1, 2 4 (Fig. 3). The results clearly <br /> T. b. rhodesiense glyceraldehyde-3-phosphate dehydrogenase                   experimental conditions half maximal <br /> (TbrGAPDH, expected size 39.3 kDa; score 87) (Fig. 2C).                   effective concentration (EC50) 38.960.9 mM compound 1 is <br /> Protein bands isolated control matrices considered            determined. Although solubility limits prevented EC50 <br /> bind unspecifically examined.                           determination regarding compounds 2 4 (Fig. 3), the <br />                                                                              concentration dependence, specificity effect is <br /> Production recombinant TbrGAPDH TbrAK                                 proven compounds. Interestingly, com- <br />    For chemical validation TbrGAPDH TbrAK,                       pound 3 appeared activate TbrAK low level (11061%) it <br /> proteins produced recombinantly sufficient amounts. The              does exhibit concentration dependence assayed up <br /> gapdh gene, kindly provided Prof. Paul A. M. Michels (University          concentrations corresponding maximum solubility, thus <br /> of Louvain, Brussels, Belgium), subcloned expression             behaved like inactive control compound 5 (Fig. 3). <br /> vector yielded 10&#226;&#8364;&#8220;15 mg highly pure (.99%) TbrGAPDH                     The EC50 compound 1 contrast IC50 <br /> per liter culture (Fig. S2A). Since ak tandem gene coding       1.0 mM parasite. We reasoned experimental <br /> two identical (99% identity, 4 345 amino acids                 conditions HPLC assay origin high <br /> different) adenosine kinases (SwissProt entries Q584S0                   EC50 value, analyzed activating effect compound 1 <br /> Q584S6) necessary determine                     using sensitive radiometric assay, allowed the <br /> isolated. The detailed analysis fragments resulting          analysis low substrate enzyme concentrations (Supporting <br />  <br />  <br />  <br />  <br /> Figure 2. SDS-PAGE (12%) analysis proteins retained affinity matrices. Panel A: Matrix linked derivative 2. Panel B: Matrix <br /> linked derivative 3. Panel C: Matrix linked derivative 4. Samples T. b. rhodesiense lysate (200 mg total protein content) incubated <br /> with 40 ml control beads (lanes a) 40 ml affinity beads (lanes b). Proteins retained matrices directly separated SDS-PAGE. <br /> Arrows indicate protein bands resulting compound derived matrices. AK: adenosine kinase T. b. rhodesiense; GAPDH: glycosomal <br /> glyceraldehyde-3-phosphate dehydrogenase T. b. rhodesiense; K: human keratin. Molecular mass markers shown left. <br /> doi:10.1371/journal.pntd.0000506.g002 <br />  <br />  <br />        www.plosntds.org                                                  4                              August 2009 | Volume 3 | Issue 8 | e506 <br />                                                                                                        TbrAK Putative Target Novel Mechanism <br />  <br />  <br />  <br />                                                                                 Table 1. Thermal stability assay regarding TbrAK absence <br />                                                                                 presence compounds substrates. <br />  <br />  <br />                                                                                                                     Tm [uC]a             DTm [uC] <br />  <br />                                                                                 TbrAK                               43.160.4             &#226;&#8364;&#8220; <br />                                                                                 TbrAK+compound 1                    47.960.1             4.8 <br />                                                                                 TbrAK+compound 2                    45.960.4             2.8 <br />                                                                                 TbrAK+compound 3                    42.860.2             20.3 <br />                                                                                 TbrAK+compound 4                    43.260.1             0.1 <br />                                                                                 TbrAK+compound 5                    43.460.1             0.3 <br />                                                                                 TbrAK+ATP                           42.860.1             20.3 <br />                                                                                 TbrAK+adenosine                     50.760.2             7.6 <br />  <br />                                                                                 a <br />                                                                                  Values represent average independent experiments. <br />                                                                                 doi:10.1371/journal.pntd.0000506.t001 <br />  <br />  <br />                                                                                   As orthogonal method CD used isothermal titration <br /> Figure 3. Concentration dependence activation effect                    calorimetry (ITC) analyze binding compound 1 TbrAK. <br /> for compounds 1 5 range 1&#226;&#8364;&#8220;750 mM. Increasing                      The titration TbrAK compound 1 revealed complex and <br />                                 N <br /> concentrations compound 1 ( ) gives sigmoid saturation curve for <br /> TbrAK activation, yielding EC50 value 38.960.9 mM. A similar trend <br />                                                                                enthalpy driven binding mode molecules 1 binding to <br />                                                                                molecule enzyme. Exothermic binding heats corrected <br /> is observed compound 2 (m), limited solubility               heats dilution, integrated plotted molar ratio <br /> concentrations .50 mM EC50 determined. Compound <br /> 4 (&amp;) started activating TbrAK concentration dependent manner             1 enzyme (trace I Fig. 4). The binding isotherm best <br /> without reaching maximum solubility limiting concentration         described using non-linear square fit assuming two-sites <br /> 750 mM. Compound 3 (&#194;&#164;) control compound 5 (w) did                  non-interacting binding model. Subsequent analysis revealed a <br /> activate TbrAK identical conditions. Values reported % <br /> activity derived transformation rate. For comparative reasons <br /> the activity recorded absence compound set 100%. <br /> Standard deviations represented vertical bars. The mean three <br /> independent measurements reported. <br /> doi:10.1371/journal.pntd.0000506.g003 <br />  <br /> Information Text S1), orthogonal method HPLC <br /> assay. Indeed, hyperactivation effect concentration <br /> dependent EC50 38612 nM compound 1 (Fig. <br /> S4). This value line observed affinity (KD of <br /> 75620 nM 497634 nM high low affinity binding <br /> site, respectively) compound 1 TbrAK, the <br /> observed IC50 1.0 mM in-vitro cell assay. <br />    In contrast adenosine kinase, TbrGAPDH activity not <br /> influenced compounds respect forward and <br /> reverse reaction (Fig. S5) measured concentrations to <br /> 50 mM (limit given strong absorption wavelength <br /> applied), suggesting TbrGAPDH considered a <br /> false positive hit. <br />  <br /> Compound 1 binding specifically TbrAK <br />    Specific binding compound 1 respect isolated <br /> proteins analyzed methods. Thermal stability of <br /> proteins protein complexes evaluated CD <br /> spectroscopy. Due fact complex ligand bound               Figure 4. Specific binding compound 1 TbrAK measured <br /> to native conformation protein higher thermal             ITC. The panel shows heat signals 27 injections of <br />                                                                                compound 1 (trace I) control compound 5 (trace II) sample <br /> stability protein [14], increased melting point              cell containing 7 mM TbrAK. The binding isotherm obtained by <br /> (Tm) protein presence ligand evidence             integration normalization raw data correction for <br /> specific binding. Indeed, apo TbrAK melted 43.160.4uC                   heat ligand dilution shown lower panel. The solid line <br /> was stabilized control compound adenosine DTm                 represents non-linear square fit based two-sites non- <br /> 7.6uC (Tm 50.760.2uC) (Table 1). In similar way, compound 1                  interacting binding model. Compound 1 binds TbrAK high <br /> and 2 able increase thermal stability DTm 4.8uC                affinity binding site KD 75620 nM DHbind of <br />                                                                                23.0560.77 kcal/mol, low affinity site exhibiting KD of <br /> (Tm 47.960.1uC) DTm 2.8uC (Tm 45.960.4uC),                         497634 nM DHbind 21.1360.24 kcal/mol. No specific heat <br /> respectively, confirming specific binding compound 1               release shown negative control (trace II). The mean three <br /> the enzyme (Table 1). Compound 3, 4, 5 did stabilize               independent experiments reported. <br /> protein added 50 mM (Table 1).                                         doi:10.1371/journal.pntd.0000506.g004 <br />  <br />  <br />        www.plosntds.org                                                    5                               August 2009 | Volume 3 | Issue 8 | e506 <br />                                                                                                                 TbrAK Putative Target Novel Mechanism <br />  <br />  <br /> high affinity binding site KD 75620 nM DHbind of <br /> 23.0560.77 kcal/mol, low affinity site exhibited KD <br /> of 497634 nM DHbind 21.1360.24 kcal/mol. The <br /> negative control using compound 5 showed specific heat release <br /> (see trace II Fig. 4). <br />    In contrast TbrAK, TbrGAPDH significantly <br /> stabilized compounds, presence the <br /> substrate NAD+ DL-GAP thermal stability increased (see Table <br /> S1). In agreement TbrGAPDH activity measurement, <br /> no binding compounds protein observed, clearly <br /> indicating TbrGAPDH false positive hit. Therefore, this <br /> enzyme validated. <br />  <br /> Compound 1 interferes substrate-inhibition of <br /> TbrAK <br />    A common property adenosine kinases various <br /> organisms control substrate-inhibition mechanism <br /> [15&#226;&#8364;&#8220;18], recently shown T. brucei AK inhibited <br /> by high adenosine concentrations prevent non-physiologically <br /> high intracellular purine nucleotide levels [19]. This prompted us <br /> to analyze influence compound 1 TbrAK                                Figure 5. Representative plots TbrAK kinetics (AMP <br />                                                                                         formation) respect adenosine absence (#) and <br /> respect substrate transformation determining kinetic <br /> parameters absence presence activator. TbrAK <br /> activity measured increasing adenosine concentrations and <br />                                                                                                     N <br />                                                                                         presence ( ) compound 1 (33 mM). Adenosine strongly inhibits <br />                                                                                         TbrAK concentrations .2 mM follows typical substrate-inhibition <br />                                                                                         kinetics. In contrast, compound 1 abolishes substrate-inhibition. The <br /> a fixed concentration 167 mM ATP. All data given                              solid lines indicate fit raw data substrate-inhibition <br /> Table 2. Adenosine kinetics absence compound 1 displayed                          model (see Table 2 fitting results). <br /> non-hyperbolic progress plots (Fig. 5). After increasing activity                    doi:10.1371/journal.pntd.0000506.g005 <br /> to maximum 2&#226;&#8364;&#8220;3 mM adenosine, enzyme activity declined <br /> at higher substrate concentrations typical finding                       activity counteracts compound 1 toxicity. This hypothesis was <br /> substrate-inhibition. Thus observed kinetic data fit                    addressed RNAi-mediated silencing TbAK expression, using <br /> substrate-inhibition model [10]. Indeed, good fit obtained                        bloodstream-form T. b. brucei express tetracycline- <br /> for data points, yielding Km 0.9960.05 mM Vmax                        inducible manner stem-loop construct targeted TbAK <br /> 19.6060.33 nM/min. kcat catalytic efficiency                          [11]. TbAK knock-down confirmed recently RNA <br /> be 0.3760.01 s21 0.3860.01 mM21 s21, respectively.                                  protein level [11,19]. The addition tetracycline reduced the <br /> TbrAK inhibited adenosine Ki 6.161.4 mM.                               sensitivity TbAK RNAi cells compound 1, raising IC50 <br /> In contrast, presence compound 1 substrate-inhibition                         131643 nM 271625 nM (two-tailed t-test, p,0.05; <br /> strongly reduced (Fig. 5), adenosine inhibited TbrAK                         Fig. 6). The negative control (compound 5) did exhibit any <br /> more ten-fold increased Ki value (78.462.2 mM). While Km                           IC50 difference induced non-induced TbAK RNAi <br /> (0.6560.04 mM),         Vmax     (14.2060.14 mM)             kcat                   cells. <br /> (0.0.2760.01 s21) slightly reduced presence compound <br /> 1, catalytic efficency kcat/Km (0.4260.01 mM21 s21) did not <br /> change significantly (Table 2). <br />  <br /> Down-regulation adenosine kinase T. b. brucei <br /> counteracts compound 1 activity <br />   The proposed mechanism action compound 1, hyper- <br /> activation TbrAK, implies down-regulation TbrAK <br />  <br />  Table 2. Parameters derived TbrAK kinetics respect <br />  adenosine absence presence compound 1. <br />  <br />  <br />                                  compound             compound 1 <br />  <br />  Km [mM]                         0.9960.05               0.6560.04 <br />  Vmax [nM/min]                   19.6060.33              14.2060.14 <br />  kcat [s21]                      0.3760.01               0.2760.01 <br />  kcat/Km [mM21 s21]              0.3860.01               0.4260.01 <br />  Ki adenosine [mM]               6.161.4                 78.462.2                       Figure 6. Dose-response curve parasite growth presence <br />                                                                                         compound 1. RNAi-mediated silencing TbAK expression by <br />  The substrate-inhibition model applied fit raw data; values represent <br />  average independently recorded series data point <br />                                                                                                      N <br />                                                                                         tetracycline ( ) reduces sensitivity TbAK RNAi cells compound 1 <br />                                                                                         compared non-induced cells (#), raising IC50 from <br />  measured triplicate.                                                                131643 nM 271625 nM (two-tailed t-test, p,0.05). <br />  doi:10.1371/journal.pntd.0000506.t002                                                  doi:10.1371/journal.pntd.0000506.g006 <br />  <br />  <br />        www.plosntds.org                                                             6                               August 2009 | Volume 3 | Issue 8 | e506 <br />                                                                                                        TbrAK Putative Target Novel Mechanism <br />  <br />  <br />  <br /> Discussion                                                                      conditions control compound 5 did bind enzyme and <br />                                                                                 produced unspecific heat signals related protein ligand <br />     A chemical proteomics approach applied isolate                   dilution sample cell (Fig. 4). <br /> identify cellular target compound 1. To end,              On target level evident additional amino group <br /> amino-derivatives immobilized, corresponding affinity             derivative 2 moderately interferes activating effect, <br /> matrixes used pull potential targets. Indeed,                 binding TbrAK disturbed productive. In <br /> approach afforded multiple proteins. Most detected             contrast, replacing oxygen morpholine moiety by <br /> in control isolation experiment (Fig. 2)              nitrogen (compound 4) strongly increases EC50 20 fold, <br /> could considered unspecific bound protein. Amongst other,                 suggesting binding TbrAK severely impaired this <br /> human keratin predominant contamination turned                derivative. Finally, substitution pyrazole moiety <br /> to present circumstances, new             (compound 3) leads low concentration independent <br /> batches matrix stored sealed containers use.       activity. These observations line fact none <br /> Remarkably, observed differences regarding inter- intrabatch             derivatives stabilized TbrAK thermal stability <br /> reproducibility. Experiments reproducible respect              assay (Table 1). On parasite level situation different. <br /> to background batch activated matrix used                  Regarding derivative 3, low activity isolated <br /> while changing new batch led altered background (see lane          enzyme exhibiting IC50 10.3 mM parasite [7], <br /> Fig. 2A,B,C). However, distinct proteins appearing               leads conclusion toxicity conferred TbrAK but <br /> control reaction identified TbrGAPDH TbrAK,                     determined target. For compound 4 the <br /> trypanosomal enzymes involved parasite glycolysis               situation appears complex. The observed concentration <br /> purine salvage, respectively. Both enzymes cloned                      dependence gives strong evidence acting TbrAK, but <br /> recombinantly expressed.                                                        potentially low affinity low activation capacity (EC50 <br />     The subsequent chemical validation enzymes revealed                 .750 mM) target lead reduced trypanocidal <br /> that TbrGAPDH activity affected compound 1                       activity. However, derivative 4 highly potent parasite <br /> (Fig. S5) specific binding 1 enzyme observed              assay (IC50 1.1 mM) [7]. It likely the <br /> (Table S1), suggesting TbrGAPDH false positive hit. As                trypanocidal effect compound 4 based combination of <br /> matter fact, highly charged nature high                      target specific activity, and/or possible off-target activity, and <br /> abundance parasite particular target strong                   increased general toxicity [7]. In addition, increased accumulation <br /> tendency isolated affinity chromatography [20&#226;&#8364;&#8220;22].                     parasite compared compound 1 contribute to <br />     In contrast unexpectedly, HPLC activity assay revealed              strong effect, reduced cell uptake explain the <br /> that compound 1 strongly activated TbrAK activity. A similar                    difference potency derivative 2 isolated target. Thus, <br /> effect measured compound 2, derivatives 3 4                   taking consideration observations complete <br /> only exhibited low significant activity TbrAK                lack activity control compound 5 experiments, we <br /> (Fig. S3). Compound 1, 2 4 activated TbrAK                             state intact morpholine/pyrazole moiety represents an <br /> concentration dependent manner, yielding EC50                             important pharmacophore, introduction of <br /> 38.960.9 mM compound 1 (Fig. 3). While EC50 2                        basic amino groups impair activating effect altered <br /> and 4 determined solubility problems,                physico-chemical properties target and/or parasite level. <br /> be estimated higher observed                  To validate TbrAK potential target compound <br /> compound 1 (Fig. 3). The observed data compound 1 2                     1 investigated parasite sensitivity compound 1 under <br /> correlate determined in-vitro data showed               conditions reduced intracellular adenosine kinase levels. To this <br /> an IC50 1.0 mM 17.7 mM blood stage T.b. rhodesiense              end, parasite viability knock-down mutant measured. <br /> [7], fact compound 2 able bind                Indeed, expected mechanism action based on <br /> target. TbrAK activation derivative 4 concentration                      overactivation TbrAK compound 1, sensitivity of <br /> dependent (EC50.750 mM, Fig. 3) corresponds                  TbrAK silenced parasite cells decreased shown IC50 <br /> the low activation capacity 50 mM (10961%, Fig. S3), (see text               raising 131643 nM 271625 nM (two-tailed t-test, <br /> below discussion issue).                                            p,0.05). Although applied tetracycline-inducible is <br />     Specific binding verified compounds 1 2             leaky capable provide complete <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span>knock-out</span> [11] the <br /> thermal stability assay shown Tm increase (Table 1). For              observed difference statistically significant, demonstrating that <br /> the active compounds (3 4) stabilization observed.             toxic effect compound 1 adenosine kinase dependent. <br /> While finding line lack activity                  Although rule compound 1 interact with <br /> concentrations tested compound 3, contrast                  targets cell amenable pull down <br /> concentration dependence derivative 4. However, taking                  approach (e.g. cytoskeleton, DNA, interference mitochondrial <br /> consideration small activating effect, concentration                    electron transport), line evidence strongly supports TbrAK <br /> dependence unchanged Tm presence compound 4,                      represent putative cellular target. <br /> it likely assume derivative binds low affinity             A step elucidation mechanism action <br /> the target. Unfortunately, limited solubility prevented             respect activating effect accomplished by <br /> investigations concentrations .750 mM regarding Tm                determination analysis kinetic parameters respect to <br /> binding affinity technical limits given CD                        substrate transformation absence presence activator <br /> spectroscopy ITC. As expected negative control                          1. TbrAK activity strongly inhibited substrate adenosine <br /> (compound 5) did activate TbrAK improve thermal                     (Ki 6.161.4 mM), follows substrate-inhibition kinetics. <br /> stability, giving evidence specific binding compound             This common characteristic adenosine kinases isolated from <br /> 1 2. In addition, binding compound 1 TbrAK                   various sources [15&#226;&#8364;&#8220;18] described TbrAK <br /> confirmed ITC revealed enthalpy driven high affinity                   recently [19]. Under experimental conditions applied, the <br /> interaction. Interestingly, appears molecules                Ki adenosine increased ten-fold value of <br /> activator bound molecule enzyme. Under                      78.462.2 mM compound 1 present, resulting strongly <br />  <br />  <br />        www.plosntds.org                                                     7                              August 2009 | Volume 3 | Issue 8 | e506 <br />                                                                                                                  TbrAK Putative Target Novel Mechanism <br />  <br />  <br /> reduced substrate-inhibition (Fig. 5) kinetic parameters                    10262% negative control (compound 5). Values are <br /> remained identical. Taken together, results suggest                   reported % activity derived ADP/ATP ratios. For <br /> the mechanism trypanocidal activity functions hyperactiva-                    comparative reasons activity recorded absence compound <br /> tion adenosine kinase. There principally different                         (column labeled TbrAK) set 100%. The mean three <br /> explanations toxic effect hyperactivation. Uncontrolled                    independent experiments reported. <br /> TbrAK activity lead purine imbalance parasite,                    Found at: doi:10.1371/journal.pntd.0000506.s003 (0.02 MB PDF) <br /> thus interfering vital purine salvage pathway, the <br />                                                                                       Figure S4 Concentration dependence activation effect of <br /> nucleotide pool subsequently nucleic acid formation. Prece- <br /> dence mechanism comes E. coli, high adenine                       compound 1 analyzed radiometric assay. Increasing <br /> concentrations cause cytotoxic [ATP]/[GTP] imbalance [23].                          concentrations compound 1 yield sigmoid saturation curve <br /> Alternatively, excessive adenosine kinase activity use                     TbrAK activation EC50 value 38612 nM. Values <br /> existing adenosine/ATP pools lead adenosine depletion                          reported % activity derived transformation rate. <br /> and ATP burn-out. Interestingly, cell death hydrolysis                        For comparative reasons activity recorded absence of <br /> ATP reserves mislocalization glycosomal hexokinase                       compound set 100%. The mean independent <br /> the cytosol observed T. brucei [24,25].                          measurements reported. <br />    Whereas hyperactivation mechanism action                           Found at: doi:10.1371/journal.pntd.0000506.s004 (0.02 MB PDF) <br /> known drugs targeting cell signaling, e.g. acetylcholine                         Figure S5 UV-spectroscopic analysis TbrGAPDH activity <br /> receptor agonists, represents novel hitherto unexplored                      (reverse reaction) absence presence compounds 1 5 at <br /> concept compounds targeting metabolic enzymes. The                                50 mM concentration. TbrGAPDH (60 nM) incubated for <br /> cytotoxic hyperactivation adenosine kinase does                           10 min 25uC absence presence compound. None <br /> provide opportunity chemotherapy sleeping sickness,                     compounds showed significantly increased or <br /> but explored pathogens tumor cells,                             decreased activity concentrations 50 mM. Values are <br /> hyperactivation metabolic key enzymes                        reported % activity derived transformation rate. For <br /> pharmacological applications.                                                         comparative reasons activity recorded absence compound <br />                                                                                       (column labeled TbrGAPDH) set 100%. The mean three <br /> Supporting Information                                                                independent experiments reported. <br />                                                                                       Found at: doi:10.1371/journal.pntd.0000506.s005 (0.02 MB PDF) <br /> Figure S1 Affinity matrix preparation. 4-[4-(5-morpholine-4-yl- <br /> 1H-pyrazol-3-yl)-phenoxy]-phenylamine (2), 4-[5-morpholine-4-                         Table S1 Thermal stability assay regarding TbrGAPDH in <br /> yl-3-(4-phenoxyphenyl)-pyrazol-1-yl]phenylamine (3) 1-[5-(4-                      absence presence compounds substrates. <br /> phenoxy-phenyl-2H-pyrazol-3-yl]piperazine (4) coupled                         Found at: doi:10.1371/journal.pntd.0000506.s006 (0.07 MB PDF) <br /> epoxy-activated agarose consisting 12-atoms spacer (1,4- <br />                                                                                       Text S1 Supporting information respect origin of <br /> bis(2:3-epoxypropoxy)butane) form affinity matrices A, B, and <br />                                                                                       supplies, protocols purification recombinant proteins, the <br /> C, respectively. The agarose beads shown spheres. <br />                                                                                       spectroscopic assay TbrGAPDH, radiometric assay for <br /> Found at: doi:10.1371/journal.pntd.0000506.s001 (0.04 MB PDF) <br />                                                                                       TbrAK, thermal denaturation assay. <br /> Figure S2     SDS-PAGE analysis expression purification                     Found at: doi:10.1371/journal.pntd.0000506.s007 (0.09 MB PDF) <br /> recombinant proteins. Panel A: TbrGAPDH, panel B: TbrAK. Lane <br /> 1: marker proteins; lane 2: soluble fraction crude extract; lane 3:                Acknowledgments <br /> final pure protein. The expected molecular weights TbrGAPDH <br /> (including His6-tag) TbrAK 41.2 kDa 38.0 kDa,                             We grateful Prof. Dr. Paul A. M. Michels (University Louvain, <br /> respectively. Molecular mass markers shown left.                    Brussels, Belgium) providing T. b. rhodesiense gapdh clone, Olga <br />                                                                                       Menang contributing project form diploma work, the <br /> Found at: doi:10.1371/journal.pntd.0000506.s002 (0.14 MB PDF) <br />                                                                                       team Dr. R. Brunisholz Functional Genomics Center operated <br /> Figure S3 Activation properties compounds 1 5 measu <br /> </body></html>