<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL OF BACTEIUOLOGY, Apr. 1976, p. 38-47                                                     Vol. 126, No. 1 <br /> Copyright &#194;&#169;) 1976 American Society Microbiology                                           Printed U.S.A. <br />  <br />  <br />   Method Isolation Escherichia coli K-12 Mutants <br />                 Deficient Essential Genes <br />                        KAREN A. ARMSTRONG&apos;* AND ROBERT K. HERMAN <br />       Department Genetics Cell Biology, University Minnesota, St. Paul, Minnesota 55108 <br />                                   Received publication 6 November 1975 <br />  <br />              We developed general procedure induction identification of <br />           mutations chromosomal essential genes located diploid region of <br />           Escherichia coli K-12. The partial diploidy conferred episome is <br />           temperature sensitive replication mutant strain form microco- <br />           lonies 42 C complete media essential chromosomal gene diploid <br />           region defective. Mutations identified procedure classified into <br />           cistrons complementation method devised purpose. <span id='am-3' about='protege:TO' typeof='owl:Thing'>To</span> verify that <br />           procedure works practice, fused episome covering rif region <br />           Fjlac+ used resulting temperature-sensitive episome iden- <br />           tify chromosomal mutations essential functions near rif. As expected, a <br />           certain proportion mutations rif gene, essential gene that <br />           codes ,3 subunit ribonucleic acid polymerase. <br />  <br />    Essentials genes Escherichia coli             some. However, suppressors general sup- <br /> whose functions essential growth                 press low efficiency, suppressor <br /> on complete broth media (5, 22). Because                tRNA insert amino acid unable to <br /> their obvious importance,             confer proper function protein product. <br /> to produce analyze mutations essential                 Thus, procedure Beckman Cooper <br /> genes. In haploid organisms, common                  permit isolation mutations <br /> method isolating mutants affected essen-               essential genes. <br /> tial genes involved use conditionally                 Murgola Adelberg (27) used streptomy- <br /> lethal mutations. In conditionally lethal                   cin suppression isolate mutations essen- <br /> strain, mutant gene product functions                tial genes E. coli. Streptomycin causes mis- <br /> under certain (permissive) conditions in-              reading nonspecific suppression three <br /> active (nonpermissive) circum-                    nonsense codons (31, 38). Therefore, isolates <br /> stances. It likely, however, genes               lesion essential gene grew on <br /> cannot easily mutated form                  complete medium streptomycin was <br /> <span id='am-2' about='Thesaurus:code' typeof='owl:Thing'>code</span> product functions                  added complete medium without <br /> experimentally desirable conditions. For in-                  streptomycin. Although mutants defi- <br /> stance, Edgar Lielausis (11) that,               cient syntheses protein, lipid, and <br /> a large sample temperature-sensitive mu-                   deoxyribonucleic acid (DNA) isolated, <br /> tants bacteriophage T4D, mutations                grew poorly permissive <br /> not distributed randomly: genes mu-                 condition partial nature the <br /> tated times, mutated                     phenotypic suppression. <br /> only once. This suggests                       To avoid problems inherent studies <br /> some genes conditionally lethal muta-                discussed above, used partially diploid <br /> tions induced. Furthermore,                    strain E. coli K-12 isolate essential gene <br /> when conditionally lethal mutations ob-                   mutants. Because F&apos; episome carries du- <br /> tained, low levels activity gene product            plicate set genes, functions essential <br /> even permissive conditions obscure                  chromosomal genes diploid region be <br /> analyses (15).                                                completely abolished loss viability of <br />    Beckman Cooper (5) used strain E.                 mutant strain. Partial diploids been <br /> coli carrying temperature-sensitive suppres-                used, example, obtain recessive lethal <br /> sor transfer ribonucleic acid (tRNA) (14) ob-              mutations ribosomal protein genes (28) and <br /> tain essential gene mutations chromo-                  gene, called rif (3), specifies /- <br />      Present address: Department Biology, University of <br />                                                               polypeptide RNA polymerase (17, 30). In both <br />  California, San Diego, La Jolla, Calif. 92093. (Send reprint cases, known phenotypic responses to <br />  requests University Minnesota address.)            particular drugs used select the <br />                                                        38 <br />  VOL. 126, 1975                                            MUTATIONS IN ESSENTIAL GENES                     39 <br /> desired mutants. We proposing gen-           Plates complete media (lac-tet Pen- <br /> eral approach wild-type alleles         assay agar) stored dark use to <br /> the episome removed unmask            avoid lethality RecA- strains (36). When re- <br /> any recessive lethal <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span>mutation</span>.                          quired, media supplemented L-amino <br />    To this, chose use F&apos; factor     acids, purines, pyrimidines, 20 ,tg/ml. <br />                                                         The exception thymine, added <br /> temperature sensitive replication.          50 ,ug/ml strains requiring high concentra- <br /> Cells mutations essential genes          tions thymine. Rifampin streptomycin were <br /> chromosome dependent              added sterile media final concentrations 100 <br /> continued presence F&apos; factor growth.         250 ,ug/ml, respectively. All media containing <br /> At 28 C cells grow normally         rifampin stored dark. <br /> temperature-sensitive episome replicates effi-              Chemicals. ICR-191, {3-chloro-7-methoxy-9-(3- <br /> ciently. However, episomal replication          [chloroethyllamino propylamino) acridine dihydro- <br /> inhibited incubation 42 C, mutants            chloride} generous gift H. J. Creech. <br /> should grow poorly form microcolonies.          Rifampin (B grade, Calbiochem) rifamycin SV <br />                                                         sodium salt (Schwarz/Mann) used parts <br /> Nonmutant bacteria grow normally              work. Since results obtained both <br /> both temperatures, F&apos; factor             drugs same, designation rifampin will <br /> maintained 28 C lost 42 C. The micro-         used text apply antibiotic. N- <br /> colonies formed mutant strains 42 C           methyl-N&apos;-nitro-N-nitrosoguanidine (NG) ob- <br /> should analogous minute colonies pro-         tained Aldrich Chemical Co., trimeth- <br /> duced unilinear transmission bac-          oprim Sigma Chemical Co. <br /> teriophage abortive transduction (29; B. A.              Nomenclature. Abbreviations symbols ge- <br /> D. Stocker, Heredity, 9:290-291, 1955).                  netic markers accordance proposed <br />     To determine method mutant           Demerec et al. (10) used Taylor Trot- <br />                                                          ter (35). ess used signify mutation an <br /> isolation successful, selected F&apos;         essential gene. Alleles rif gene denoted by <br /> factor carried known essential gene,           convention used Austin Scaife (3): rif <br /> that certain chromosomal mu-                confers sensitivity drug rifampin, rif confers <br /> tants defective essential gene          resistance drug, rif&#194;&#176; results complete <br />  recovered. We chose F&apos; factor             loss rif gene function. <br />  F110 carries essential gene rif.             Bacterial strains. The bacterial strains used in <br />  Strains deficient rif gene character-       study listed Tables 1 2. <br />  ized previously (4) known rir mu-                Mutagenesis procedures. For construction of <br />  tants. rif&quot; mutants inviable haploids,         various strains (Table 1), auxotrophs induced <br />                                                          treating exponentially growing cultures NG <br />  characteristic expected essential mutant.         described Hoess Herman (20). After muta- <br />     To make F110 temperature sensitive,           genesis, cells collected centrifugation, <br />  fused vivo well-character-           washed twice M63 buffer-salts, suspended <br />  ized temperature-sensitive episome, Fu52lac+            glucose minimal medium. When culture <br />  (8). We mutagenized strain ICR-191,          reached stationary phase, penicillin selection (16) <br />  frameshift mutagen usually produces mu-            performed enrich auxotrophs. <br />  tations virtually abolish gene function (1,            Mutations essential genes induced by <br />  32). We isolated mutants defective chro-        treating exponential culture strain KAF1/ <br />  mosomal essential genes covered tem-             KA17 (Table 2) ICR-191 (19) 4 8 h in <br />                                                          lactose-threonine-tryptophan-methionine medium <br />  perature-sensitive episome. As predicted,           28 C using 0, 10, 15, 20 jig ICR-191/ml. <br />  mutants deficient chromo-              After 4 8 h, cells collected washed as <br />  somal rif gene function.                                 NG mutagenesis suspended M63 <br />     We developed method transfer            buffer-salts. The resuspended cells diluted <br />  chromosomal mutant rif allele muta-           1:100 lactose-threonine-tryptophan-methionine <br />  tions essential genes F&apos; factor re-         medium incubated shaking 28 C until <br />  combination. This procedure important be-            stationary phase reached. <br />  cause allowed perform complementation            The efficiency ICR-191 mutagenesis was <br />  tests classify mutations cistrons.          monitored examining number colonies of <br />                                                          strain KAF1/KA17 grew minimal plates <br />                                                          containing rifampin. For purpose, station- <br />            MATERIALS AND METHODS                         ary cultures segregated cells diluted 500-fold <br />    Media. M63 minimal liquid medium agar,            plated incubated 28 C. The cul- <br />  lactose-tetrazolium agar (lac-tet), Penassay        tures showed highest frequency rifampin- <br />  broth described previously (18, 19). Penassay      resistant colonies-this varied somewhat ex- <br />  agar contains 25.5 g Antibiotic medium 2 (Difco)     periment experiment - screened mutants <br />  5 mg thiamine liter. Dilution medium,        deficient essential genes. <br />  M63 buffer-salts, contained following liter:       Thy- mutants. Spontaneous Thy- mutants were <br />  KH2PO4, 5.3 g; K2HPO4 3H20, 13.93 g; (NH4)2SO4,         selected using trimethoprim described by <br />  2.0 g; MgSQ4-7H20, 0.2 g; FeSO4 7H20, 0.5 mg.       Hoess Herman (20). <br />  40        ARMSTRONG AND HERMAN                                                                        J. BACTERIOL. <br />                                             TABLE 1. Bacterial strains <br />          Strain             Sex                    Genotype&quot;                                   Derivation <br /> Ft,62jac+I20OPS            F&apos;       F.,26ac+/lacZ+, lacY-                       F. Jacob (8) <br /> Ft,2ac+/KA9                F&apos;       Ft.621ac+largE, met, pur, str, thy+, his,   FU62Jac+/200PS     x KA9 <br />                                       pro, lac, recAl <br /> JC5029                     Hfr      ilv-38, thr-300, recA+                      CGSCb <br /> JC5029 Thy-                Hfr      ilu-38, thr-300, recA+, thy                 Spontaneous mutant JC5029 <br /> JC5029 Thy- Leu-           Hfr      ilv-38, thr-300, recA+, thy, leu            NG-induced mutant JC5029 <br />                                                                                   Thy- <br /> KAlc                       F-    argE, met                                      NG-induced mutant 254 <br /> KA2c                       F-    argE, met, str                                 Spontaneous mutant KA1 <br /> KA3C                       F-    argE, met, str, thy                            Spontaneous mutant KA2 <br /> KASC                       F-    argE, met, pur, str, thy, his, pro, lac        NG-induced mutant KA3 <br /> KA9c                       F-    argE, met, pur, str, thy+, his, pro, lac,      KA8 x KL16-99 Thy- <br />                                     recAl <br /> KA12c                   F-       argE, met, pur, lac                       NG-induced mutant KA1 <br /> KA13r                   F-       argE, met, pur, lac, thy                  Spontaneoqs mutant KA12 <br /> KA14c                   F-       argE, met, pur, lac, thy+, recAl          KA13 x KL16-99 Thy- <br /> KA14 RiF c              F-       argE, met, pur, lac, recAl, rif&apos;          Spontaneous mutant KA14 <br /> KA14 Thy-c              F-       argE, met, pur, lac, recAl, thy           Spontaneous mutant KA14 <br /> KA16c                   F-       argE, met, pur, lac, thy, thr, trp        NG-induced mutant KA13 <br /> KA17c                   F-       argE, met, pur, lac, thy+, thr, trp, KA16 x KL16-99 Thy <br />                                     recAl <br /> KL16-99 Thy-            Hfr      thy, recAl                                J. A. Clark (24) <br /> KLF10/JC1553            F&apos;       FilO metB+IargG6, metBi, his-1,            CGSC&amp; <br />                                     leu-6, recAl, str-104, malB16 <br /> KLF23/KL181             F&apos;        F123 trp+IpyrD34, his-68, trp-45,        &apos;CGSCb <br />                                     recAl <br /> 254                     F-       argE                                       P. Rogers (12) <br /> 254Rifr                  F-      argE, rif&apos;                                 Spontaneous mutant 254 <br />   Only genetic-markers relevant current study listed. <br />   b*E. coli Genetic Stock Center, Department Human Genetics, Yale University, New Haven, Conn. <br />     The met pur markers strains selected complemented the <br /> episome F110. <br />                             TABLE 2. Construction Ft,6Aact+-FIO rif met (KAFJ) <br />               Strain               Sex                    Genotype&quot;                              Derivation <br /> F110 rifr/254 Rifr                  F&apos;    F110 argE+, rifr/argE, rifr               KLF10/JC1553 x 254 RiFr <br /> F110 rif/KA14 Rif                   Ft    F110 argE+, rifilargE, met, pur, lac,     F110 rifd/254 Rip x DA14 Rifr <br />                                              reCAl, rif <br /> Ft6a21ac+-F110 rif&apos;/KA9             F&apos;     Ft0bac+-F110 argE+, rif largE,           F110 rifr/KA14 Rifr x F- <br />                                              met, pur, strr, thy+, his, pro, lac,     phenocopies Fr,6.Iac+/ <br />                                              recAl                                      KA9 <br /> Ftb621aC+-F110 rif&apos;/KA1             F&apos;     Fta621ac+-F110 argE+, rifr/argE, met,Ft62lac+-F110 rifKA9 x <br />                                              recA +                               KA1 <br /> FtS62jac+tF110 rif met/KA1          F&apos;     F,621ac+-F110 argE+, rifr, metlargE, Homogenote formation <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> <br />                                              met, recA +                          preceeding strain <br /> FtI62lac+-F110 rif&apos; metP/KA9        F&apos;     Ft,621ac+-F110 argE+, rifr, met/ Ft1621ac+-F110 rif met/KAl x <br />                                              argE, met, pur, thy+, his, pro, lac,       KA9 <br />                                              recAl <br /> FtSunbac+tF110 rif met/KA17         F&apos;     Ft,621ac+-F110 argE+, rifg, met/ Ft,6Sac+-F110 rifr met/KA9 <br />                                             argE, met, pur, lac, thy+, thr, trp,        x KA17 <br />                                             recAl <br />      Only genetic markers relevant current study listed. <br />      b <br />        Ftlac+-F110 rif&apos; met designated KAF1. <br />    Identification recA+ recA strains. The              Mating procedures. Aside exceptions de- <br />  Rec- <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span>phenotype</span> detected sensitivity          scribed below, standard mating procedure to <br />  ultraviolet light, using published procedures (20).        dilute overnight cultures donors recipients, <br />     Preparation F- phenocopies. Male strains             grown minimal medium broth, sepa- <br />  converted F- phenocopies standard           rately 1:40 Penassay broth, incubate with <br />  procedure (26).                                            shaking 37 C 2.5 4 h, mix a <br />  VOL. 126, 1975                                              MUTATIONS IN ESSENTIAL GENES                  41 <br /> 1:1 ratio. These mating mixtures incubated           chromosomal rir gene, the <br /> without shaking 1 2 h 37 C.                     wild-type allele, mutated. We thus <br />    Three mating protocols employed        began experiments F&apos; factor F110 <br /> because involved temperature-sensitive        carries rif gene (3). <br /> F&apos; factors need screening large numbers of <br /> F&apos;-containing colonies.                                      An F&apos;riflrif strain usually sensitive to <br />    Donors containing F.l,,21ac+ fused episomes     rifampin (3). However, chromosomal rir <br /> derived F&apos; factor grown stationary      allele mutated rifr rio, re- <br /> phase 28 C Penassay broth (Ft1621ac+)     sulting F&apos;riflrif F&apos;rif/rift strain pheno- <br /> liquid minimal medium (the temperature-sensitive,         typically rifampin resistant (3). Thus, con- <br /> fused episomes). They diluted 1:20         struct F&apos;rifr/rif strain, use ap- <br /> Penassay broth, shaken 34 C 5 h,         pearance Rif&apos; phenotype score directly <br /> mated shaking 2 h 34 C described        chromosomal mutations rifr rip; more- <br /> above. Plates spread mating mixtures           over, use frequency appearance of <br /> were incubated 28 C. <br />    Replica matings used screen large num-         Rifr mutants estimate success the <br /> bers isolates ability donate episome   mutagenesis ICR-191. Since F110 carries <br /> with normal replication machinery strain       rif allele, step convert this <br /> carrying episome temperature sensitive        allele rifr. This transferring <br /> in replication. Stationary donor cultures            F110 cell carrying rif gene its <br /> grown minimal medium 37 C diluted          chromosome selecting homogenate <br /> and plated minimal plates approximately        formation. <br /> 100 colonies plate. The plates incubated         Our identification mutations chromo- <br /> 37 C colonies approximately 1.5 mm         somal essential gene required episome <br /> in diameter (about 48 56 h) replica plated <br /> onto selective plates spread 0.15 ml culture    carrying gene temperature sensitive for <br /> of recipient strain containing temperature-       replication microcolony formation at <br /> sensitive episome. The recipient grown        42 C used identify mutants. <br /> Penassay broth 28 C 5 7 h. The selective        Therefore, second step fuse F110 rif <br /> plate contained minimal medium selecting        Ft.621ac+ obtain episome rif <br /> the growth parents permitting             gene F110 temperature-sensitive <br /> growth recipients received non-tem-       replication machinery Ft,621ac+. The tech- <br /> perature-sensitive episome. It crucial        nique fusing F&apos; factors (37) ob- <br /> success procedure donor plates      tain Fts62lac+-F110 rif allowed isolate a <br /> and replica mating plates fresh, i.e., dried either <br /> overnight 2 days room temperature.              temperature-sensitive episome relatively easily <br />    Spot matings used test ability      compared alternative mutagenizing <br /> one strain transfer episomes recipient         isolating temperature-sensitive mutants of <br /> conjugation. Bacteria used crosses          F110 rif. Ft,62lac+ used replica- <br /> grown minimal medium 37 C stationary             tion shown thermolabile <br /> phase cells did contain temperature-         Fts,14lac+, episome com- <br /> sensitive episome. If bacteria did possess tem-     monly used (8, 26). <br /> perature-sensitive episome, cells grown          Finally, wanted F&apos; factor carry an <br /> 28 C Penassay broth matings. Approxi-          auxotrophic marker carried chromo- <br /> mately 0.01 ml culture strain spotted <br /> onto selective plate. After spot dried, 0.01    selection used replace <br /> ml strain deposited top,      resident F&apos; factor F&apos; factor <br /> mixed spot permitted dry plate      carrying wild-type allele marker. <br /> incubated.                                                Such replacement important the <br />                                                           complementation tests mapping mu- <br />                                                           tations chromosomal essential genes (see be- <br />                     RESULTS                               low accompanying paper [2]). We chose <br />    Outline construction F,1621ac+-              met marker purpose used homo- <br /> F110 rifr met episome. As discussed above,             genate formation transfer met marker <br /> wished develop method isolating mu-              KA1 episome. Strain KA1 is <br /> tants deficient essential genes E. coli. Our        recA + carries met marker as <br /> plan induce mutations strains               KA17, chromosome be <br /> that carried duplicate copy essential            used search ess mutants. <br /> gene F&apos; factor temperature sensitive               In summary, episome construction neces- <br /> replication. As test procedure, chose         sary test procedure isolating <br /> to determine isolate mutant            mutants deficient essential genes involved: <br /> deficient specific chromosomal essential             (i) isolating F110 episome carrying rifl <br /> gene, rif. One reason choosing rif gene           mutation, (ii) fusing F&apos; factor with <br /> was simple tests used deter-             Ft,621ac+, (iii) isolating met variant the <br />  42      ARMSTRONG AND HERMAN                                                              J. BACTERIOL. <br /> fused episome met allele          glucose plates containing methionine. During <br /> same the chromosome. These               subsequent growth, recombination occur <br /> procedures outlined Table 2 dis-        produce met homogenates, iden- <br /> cussed below.                                tified follows. Single-colony isolates the <br />    Construction FilO rifr. F110 trans-          selective plates grown stationary phase <br /> ferred conjugation strain 254 Rifr (argE       28 C liquid medium containing glucose <br /> rif recA+; Table 2). Rif&quot; merodiploids            methionine. To prevent recombina- <br /> formed recombination subsequent              tion occurring episome and <br /> growth identified ability         chromosome, episome transferred to <br /> grow rifampin-containing plates. To prevent         KA9, F-recA strain carries same <br /> the occurrence changes epi-          met allele strains KA1 KA17. In the <br /> some recombination chromosomal                 conjugation transfer, streptomycin was <br /> markers, transferred KA14 Rifr,            used select donor. Also, selec- <br /> recA strain. The resulting strain resistant        tion medium contained lactose defi- <br /> to rifampin, expected. Proof F&apos; fac-       cient arginine adenine select against <br /> tor carried rifr allele (and rift)       KA9. After incubation 28 C, plates were <br /> the segregation rifampin-resistant recombi-         replicated similar plates lacking methionine, <br /> nants episome transferred             colonies didn&apos;t grow replica <br /> KA2, rifs recA + strain.                             plates identified met homozygous mero- <br />    Fusion F110 rir Ft,621ac+. F110 rif/        diploids. The colonies purified twice by <br /> KA14 rifr recA grown stationary phase           streaking obtain single colonies selective <br /> in glucose minimal medium, diluted 1:40           plates containing methionine 28 C. These <br /> Penassay broth, grown exponential phase          isolates contained desired episome <br /> 37 C, mated 34. C 2 h F- pheno-        Ft,621ac+-FllO rifr met, desig- <br /> copies Ft,621ac+/KA9 grown stationary            nated KAF1. Any mutations essential genes <br /> phase 28 C. Undiluted aliquots mating        identified using fused episome in <br /> mixture spread lactose plates con-           chromosomal genes duplicated F110 <br /> taining streptomycin select do-         chromosomal genes carried by <br /> nor. The plates supplemented             FtS621ac+ dispensable (7). <br /> histidine proline, amino acids required           Chromosomal genotype strain used <br /> by recipient. Colonies grew incu-       isolating mutations essential genes. <br />  bation 28 C candidates containing         The chromosomal genotype constructed for <br />  episome produced fusion FilO rill        experiments (Table 1) contained follow- <br />  Flsf2lac+ growth medium           ing mutations: argE, met, pur, thr, trp, and <br />  required argE+, met+, pur+ genes         recAl. The recA mutation needed pre- <br />  FilO rif lac+ gene Fts62lac+,     vent loss ess mutations chromo- <br />  F&apos; factor exist         recombination episome, <br />  stably cell (9, 33). Although deletion        carries ess+ alleles. Also, neces- <br />  DNA occur episomal fu-           sary chromosome strain carry <br />  sion, probably incompatibility      auxotrophic alleles genes carried the <br />  sets F replication genes (37), rifr gene   episome selection continued pres- <br />  fused episome rif lies       ence episome exerted. KAF1 is <br />  argE+ pur+, genes required            lost unless episomal genes required <br />  growth selective plates. The putative      growth. The argE, pur, lac muta- <br />  Ft,62lac+-F 10 rifr/KA9 strains grew      tions served purpose. The functions of <br />  minimal plates purified twice streak-         chromosomal markers (met, thr, trp) <br />  ing selective medium           apparent later comple- <br />  tested mating 34 C KA17, F-             mentation procedure discussed. Thus, the <br />  strain. The mating mixtures plated          final strain used mutant isolation <br />  selective medium test transfer           KAF1l81ac+ argE+ rift met pur+/argE met <br />  FtS61lac+-F110 rip/KA9 genes lac+, argE+,      pur lac thr trp recAl. This strain obtained <br />  met+, pur+. In way obtained            conjugating KAF1/KA9 KA17 (Table <br />  episome did        2). <br />  genes temperature sensitive              Isolation mutants defective essential <br />  replication. The fused episome FtS621ac+-F110          genes. Strain KAF1/KA17 mutagenized <br />  rifr carried KA9 (his pro) transferred       ICR-191 grown minimal medium at <br />  KA1 (argE met recA +) conjugation. Dilu-            28 C allow chromosomal segregation. Sta- <br />  tions mating mixture spread           tionary cultures segregated cells were <br />  VOL. 126, 1975                                         MUTATIONS IN ESSENTIAL GENES                 43 <br /> diluted M63 buffer-salts 100        parent strain recipient consistently gave <br /> colonies plate lac-tet agar. The plates       numerous Trp+ merodiploids spot matings <br /> were incubated 28 C 32 34 h       strain KLF23/KL181. Any mutant isolates <br /> replicated medium          showed parental phenotype forma- <br /> prewarmed 42 C. The replica plates           tion viable Trp+ merodiploids concluded <br /> incubated 42 C approximately 50 h. Most       defective chromosomal essential <br /> colonies, including formed unmuta-          genes examined further. <br /> genized bacteria, grew normally             Isolation chromosomal rite mutants. <br /> master replica plates. Rare colonies        Since KAF1 carries rifr gene, expected <br /> formed, however, grew normally           chromosomal mutations essen- <br /> 28 C master plate formed tiny clusters      tial genes rir mutations. Theoretically, it <br /> of microcolonies, called spotty colo-       possible mutated chromosomal <br /> nies, 42 C plate. These colonies tenta-    rip gene rifr rifo, would <br /> tively identified having mutations essen-      identified rif&apos; type essential <br /> tial genes chromosome. Such mutants           mutant rif&apos; class mutants <br /> would dependent presence          accept F123 trp+ place KAF1. <br /> episome KAF1 continued growth,           The F&apos;rifrir mutants easily identi- <br /> replication episome temperature sensi-     fied rifampin-resistant phenotype. <br /> tive.                                                Therefore, mutant strains unable <br />    To verify bacteria spotty colo-   form Trp+ merodiploids transferred <br /> nies defective chromosomal          minimal plates rifampin <br /> essential gene diploid region, cells     test rifampin resistance. The plates were <br /> purified corresponding colony        incubated 28 C. Two rifampin-resistant <br /> 28 C master plate spot-mated         strains independently <br /> strain carrying F123 trp+ episome (strain        isolated mutants KAF1 re- <br /> KLF23/KL181, Table 1). Trpt merodiploids             placed F110 met+ F123 trp+. When <br /> were selected glucose-arginine-methionine-        KAF1 replaced FllO met+ rifs, mero- <br /> adenine-threonine minimal plates incubated        diploids sensitive rifampin expected <br /> 37 C. The failure isolate form viable       rif mutation chromosomal rif gene. <br /> Trp+ colonies mating suggested       Furthermore, mutations two <br /> KAF1 replaced F123 trp+,       strains mapped rif portion of <br /> two F&apos; factors stably maintained        diploid region (2). We conclude these <br /> the cell; therefore, isolate pre-     mutant strains type had <br /> sumably required continued presence           predicted isolate. The seven mu- <br /> KAF1 isolate contained lesion       tants discussed accompanying <br /> some essential chromosomal gene func-          paper (2). <br /> tion coded episome.                     Construction F110 episomes carrying es- <br />    An alternative possibility strain      sential mutations. To able classify the <br /> that formed spotty colony 42 C         essential gene mutations complementation <br /> longer accept F&apos; factor conjugation. To       groups, wanted transfer F110 <br /> ascertain true, strain        met+ episomes. We shall illustrate proce- <br /> forming spotty colony 42 C mated       dure (Fig. 1) using rir muta- <br /> a strain containing F110 met+ episome.           tions. As described, tests for <br /> F110 met+ carries genes KAF1 and,      identification mutations essential genes <br /> therefore, able replace KAF1       involved replacement KAF1 F110 <br /> if episome carried gene essential   met+. The resulting strain following <br /> growth. Replacement KAF1 accom-               genotype: F110 met+ ess+/argE met pur lac thr <br /> plished mating strain KLF10/JC1553           trp ess recAl, designated strain C Fig. 1. Our <br /> spotty colony isolates KAF1/KA17 select-      basic procedure introduce recA + gene <br /> ing Met+ merodiploids. The existence          strain episome pick <br /> Met+ colonies mating indicated       ess mutation chromosome by <br /> the inability form Trp+ colonies mating     recombination: F110 met+ ess +/met ess <br /> with F123 trp+ F123 trp+ did         recAl heterogenotes (strain C) into <br /> <span id='am-160' about='Thesaurus:code' typeof='owl:Thing'>code</span> wild-type allele mutant chro-      F- phenocopies mated recA+ <br /> mosomal essential gene KAF1/         Hfr strain, JC5029 Thy- Leu- (Table 1). It was <br /> KA17 unable accept F&apos; factors          matings 100- <br /> mating.                                              fold efficient recipient grown at <br />    Control matings using unmutagenized           28 C instead 37 C liquid minimal medium <br />  44           ARMSTRONG AND HERMAN                                                                        J. BACTERtIOL. <br /> Strain <br /> desig-                                       Genotype                                              Derivation <br /> nation <br />   A               KAF1 met ess+largE met pur lac thr trp ess+ recAl                      Strain KAF1/KA17 (Table 2) <br />  <br />   B                              KAF1 met esst/(a) ess recAl                             NG ICR-191 mutagenesis; <br />                                                                                            selection Ess- mutants <br />  <br />   C <br />                                                  1r <br />                                  F110 mett ess+l(a) ess recAl <br />                                                                                                     (see text) <br />                                                                                         Strain B x KLF10/JC1553; se- <br />                                                                                            lection Met+ His+ Leu+ <br />                                                                                                      colonies <br />      D                                    F- phenocopies                                              (26) <br />  <br />      E                                         &apos;r <br />                   F11O mett esst/argE met pur lac thr trpt ess recAt+                   Strain D x JC5029 Thy- Leu-; <br />                                                                                             selection Trp+, Thy+, <br />                                                                                           Leu+ recombinants; UV irra- <br />                                                                                             diation identify RecA+ <br />                                                                                                      isolates <br />                            1 <br />                            4,                                           1 <br />   F                        (1)                                         (2)               After growth &#194;&#177; FUdR treat- <br />                       F&apos;met+                                         F&apos;met+                          ment <br />                        ess+                                           ess <br />  <br />  <br />      G                     (1)                                      (2)                 Strain F (1 + 2) x KA14 Thy- <br />                    F&apos;met+/ (b)                                 F&apos;mett+          (b)         recAl; selection Arg+ <br />                    ess+ / recAI                                ess      IrecAl              Mett Thr+ Pur+ isolates <br />  <br />  <br />      H               (1)                                (2)                               Strain G (1 + 2) x strain A; <br />                                                                                            selection Met+ Thy+ colo- <br />                                                                                                         nies <br />               F&apos;met+ / (a) esst+              F&apos;met+     /   (a) ess+ <br />                 ess+ / recAI                     ess    /   recAl <br />                   VIABLE                              VIABLE <br />                                                                         - <br />                                                                             &lt; <br />          I                           (1)                                (2)               Strain G (1 + 2) x strain B; <br />                             F&apos;met+  (a) ess <br />                                       /           F&apos;met+ / (a) ess            selection Met+ Thy+ colo- <br />                          esst / recALl              ess / recAl                            nies <br />                             VIABLE                  NONVIABLE <br />   FIG. 1. Procedure transfer chromosomal ess mutations F110 episomes. The symbol (a) denotes the <br /> markers argE met pur lac thr trp, (B) denotes argE met pur lac thy. <br />  <br /> before F- phenocopies. After                         episome further, Tlpt RecA+ <br /> mating 37 C 2 h, Trp+ Leu+ Thy+ recom-                        strains usually treated fluorodeoxyu- <br /> binants selected minimal agar 37 C.                       ridine (FUdR) 90 150 min method <br /> Since recA+ gene enters trp+                       Gallant Spottswood (13). Before re- <br /> transfer DNA Hfr strain (6), approxi-                      combinant ess episomes identified, a <br /> mately 50% Trp+ recombinants inherit                          sample episomes culture was <br /> the recA + gene Hfr. These Trp+ recom-                      transferred F-KA14 Thy- (recAl) conju- <br /> binant strains twice purified streaking                      gation prevent recombination. These <br /> onto type selective plates, grown                     crosses immediately FUdR <br /> minimal liquid medium 37 C, irradiated                        treatments. After 60 90 min mating at <br /> with ultraviolet light (UV) identify UV-re-                       37 C, mating mixtures cells strains F-1 <br /> sistant (RecA+) isolates (strain E).                                 F-2 KA14 Thy- recA diluted 1:100 <br />   To increase recombination chromo-                          liquid minimal glucose-thyniine medium <br />  VOL. 126, 1975                                          MUTATIONS IN ESSENTIAL GENES                 45 <br />  enrich KA14 Thy- recA cells          described Hong Ames (21), although <br />  received episome donor strain            method involved temperature-sensitive <br /> (strains G-1 G-2). After growth station-       mutations, advantage the <br /> ary phase 37 C, culture diluted        method used mutations can <br /> spread glucose-thymine plates form sin-       simply lead inactive gene products. F&apos; factors <br /> gle colonies. Each colony strains G-1 G-2      available desired region of <br /> was tested presence F&apos;argE+        E. coli chromosome (25), applicability <br /> met+ pur+ ess episome. This mat-          method obviously extended. <br /> ing original ess mutant parent:            Naturally, like able iden- <br /> KAF1 met ess+IargE met pur lac thr trp recAl          tify biochemical defects essential <br /> ess (strain B) selecting Met+ Thy+ (Arg+      gene mutants. We suggest possible <br /> Pur+) merodiploids. Nonrecombinant (ess I) epi-       approaches doing this. All involve mak- <br /> somes carried strain G-1 replace           ing guess particular proteins might <br /> resident episome recipient viable      involved. Such guesses based on <br /> Met+ Thy+ merodiploids (strain I-1), ess          genes known map the <br /> episomes carried strain G-2 (strain I-      region studied. One procedure involves <br /> 2). Because ess episomes rare (1 3%        assaying specific protein mutant strain <br /> some FUdR-treated cultures 0. 1%        shift 42 C. At 42 C the <br /> in untreated cultures), replica matings          differential rate synthesis protein <br /> used screen large numbers colonies        reduced compared essen- <br /> for inability produce Met+ Thy+ merodip-       tial proteins chromosomal gene that <br /> loids. Candidates carrying ess episome         protein inactive. At 42 C culture mutant <br /> identified test twice purified        cells does completely lack essential gene <br /> selective plates spot-mated        function. Some cells culture contain an <br /> the chromosomal ess + parent (strain A), KAF1         unreplicating episome, presumably in <br /> met ess+/ess+ KA17, chromosomal ess           cells ess + gene product continues be <br /> mutant derivative (strain B). Isolates gave      synthesized. F-, cells continually segre- <br /> Met+ Thy+ colonies mated chro-          gated, however. These cells contain a <br /> mosomal ess+ strain, Met+ Thy+ colonies        nearly normal ess+ gene product <br /> when mated chromosomal ess strain,           segregated, growth of <br /> were deduced following desired gen-       cells descendants gradu- <br /> otype: F&apos;argE+ met+pur+ esslargE met pur lac          ally slow ess+ gene product be- <br /> recA1 thy.                                            comes diluted rate-limiting growth. <br />    By method, constructed F&apos;met+            A second type biochemical analysis that <br /> rif&apos; episomes KAFl/met rif&quot; strains. As          applied mutants isolated is <br /> predicted, episomes form Met+ Thy+ mer-         immunological technique described by <br /> odiploids matings ess+ parent             Horvitz (23). Briefly, supernatant extract is <br /> KAFl/met rir matings rif&quot;         obtained radioactively labeled cells and <br /> mutant parent.                                        incubated antibody purified protein. <br />                                                       The antigen-antibody complex analyzed <br />                   DISCUSSION                          sodium dodecyl sulfate-polyacrylamide slab <br />    In paper, delineated procedure      gel electrophoresis. Horvitz (23) used proce- <br /> for isolating mutants deficient chromosomal        dure demonstrate sequential phosphoryla- <br /> essential genes partially diploid strain E.   tions subunit RNA polymerase. In <br /> coli K-12. Knowing episome con-         extracts mutants, look poly- <br /> ferred partial diploidy carried essential         peptide fragments, frameshift mutations <br /> gene rif, predicted essential     frequently lead premature termination of <br /> mutants isolated defective         translation (32). The gel <span id='am-161' about='xsp:pattern' typeof='owl:Thing'>pattern</span> extract <br /> chromosomal rif gene. Indeed, mutants          mutant strain extra <br /> this type isolated.                              band, polypeptide fragment, not <br />    A feature method developed       present gel pattern wild-type <br /> it concentrates limited region ge-        strain. This approach would, course, not <br /> nome-the region covered F&apos; used. Be-           work polypeptide fragment al- <br /> cause method enables classify          tered longer precipitated by <br /> mutations genes complementation               antibody. <br /> tests, feasible saturate particu-      A possible approach involves isola- <br /> lar chromosomal region mutations es-          tion partial revertant strains viable <br /> sential genes. The idea localized mutagenesis      absence fused episome. These hap- <br />  46       ARMSTRONG AND HERMAN                                                                                   J. BACTERIOL. <br /> loid strains assayed enzyme activ-                         transfer ribonucleic acid synthetase mutants ofEsch- <br /> ity exhibit specific activity lower                        erichia coli. J. Bacteriol. 112:102-113. <br />                                                                 13.   Gallant, J., T. Spottswood. 1965. The recombino- <br /> than -wild-type F- parent. These                            genic effect thymidilate starvation Escherichia <br /> partial revertants, carry in-                        coli merodiploids. Genetics 52:107-110. <br /> tragenic intergenic suppressor mutations,                    14.   Gallucci, E., G. Pacchetti, S. Zangrossi. 1970. <br /> would analogous leaky mutants de-                             Genetic studies temperature sensitive nonsense <br />                                                                         suppression. Mol. Gen. Genet. 106:362-370. <br /> scribed Schlessinger (34).                                   15.   Gefter, M. L., Y. Hirota, T. Kornberg, J. A. Wechsler, <br />    In conclusi <br /> </body></html>